<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol.18 No.21 pp.5972&#226;&#8364;&#8220;5982, 1999 <br />  <br />  <br />  <br /> Eps1, novel PDI-related protein involved ER <br /> quality control yeast <br />  <br />  <br />  <br /> Qiongqing Wang Amy Chang1                                    tion, ubiquitination degradation cytoplasmic pro- <br />                                                                  teasomes (Brodsky McCracken, 1997; Kopito, 1997; <br /> Department Anatomy Structural Biology, Albert Einstein    Sommer Wolf, 1997). <br /> College Medicine, 1300 Morris Park Avenue, Bronx, NY 10461, <br /> USA                                                                 While general concepts ER quality control have <br /> 1Corresponding <br />                                                                  broadly outlined, molecular details the <br />                author <br /> e-mail: achang@aecom.yu.edu <br />                                                                  process unknown. Genetic methods used <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                  identify components ER-associated degradation <br /> PMA1 essential gene encoding yeast plasma              machinery yeast (Hampton et al., 1996; Knop et al., <br /> membrane [H1]ATPase. A pma1-D378N mutant                   1996; McCracken et al., 1996). While novel genes <br /> dominant-negative effect cell growth             identified, including DER1, HRD1 HRD3 <br /> newly synthesized mutant wild-type Pma1 molec-               (Hampton et al., 1996; Knop et al., 1996), known <br /> ules retained degraded endoplasmic                gene products participate initial events <br /> reticulum (ER). Like substrates ER-associ-             quality control, e.g. identification substrate, or <br /> ated degradation, Pma1-D378N stabilized mutants            subsequent events, e.g. retrograde transport the <br /> defective components ubiquitination                    cytoplasm degradation. Also unanswered ques- <br /> machinery. A genetic selection performed eps             tion quality control machinery discriminates <br /> (ER-retained pma1 suppressing) mutants              normal proteins process folding and <br /> growth defect caused D378N allele suppressed.          misfolded proteins degraded, molecular <br /> In eps1 mutant, mutant wild-type Pma1                chaperones suggested participate substrate <br /> molecules allowed travel plasma mem-               selection ER-associated degradation (ERAD) (Brodsky <br /> brane; however, normal retention resident ER pro-             McCracken, 1997; Kopito, 1997; Sommer Wolf, <br /> teins Shr3 Kar2 perturbed. Eps1 novel            1997). <br /> membrane protein belonging protein disulfide                 We studied ER quality control Saccharomyces <br /> isomerase (PDI) family, Eps1 co-localizes               cerevisiae examining <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> PMA1, encoding <br /> Pma1-D378N ER. In absence Pma1-                    plasma membrane [H1]ATPase, causes defect <br /> D378N, ER export wild-type Pma1 affected               ER export newly synthesized molecule. Pma1 is <br /> by eps1 deletion, export plasma membrane              major polytopic cell surface protein physiological <br /> protein Gas1 delayed. Because Eps1 required            role regulate intracellular pH generate electro- <br /> retention degradation Pma1-D378N, propose              chemical gradient pumping protons cell. <br /> a model Eps1 acts novel membrane-                  Pma1 belongs P-type ATPase family ion trans- <br /> bound chaperone ER quality control.                           porters members conserved aspartate residue <br /> Keywords: ATPase/ER-associated degradation/molecular             undergoes phosphorylation catalysis. Previous <br /> chaperone/plasma membrane/protein disulfide isomerase            work study functional consequences mutating <br />                                                                  predicted catalytic phosphorylation site S.cerevisiae <br />                                                                  Pma1 (Asp378) led discovery substitution <br />                                                                  residue causes defective delivery newly <br /> Introduction                                                     synthesized protein plasma membrane. Instead, the <br /> As site entry secretory pathway,             mutant protein misfolded retained ER (Harris <br /> endoplasmic reticulum (ER) initiates proper folding,         et al., 1994; DeWitt et al., 1998; Maldonado et al., <br /> assembly modification newly synthesized proteins          1998; Nakamoto et al., 1998). Indeed, entire region <br /> (Hammond Helenius, 1995). Molecular chaperones               surrounding Asp378, lying central cytoplasmic <br /> such BiP calnexin assist folding assembly          domain bounded 10 transmembrane segments, appears <br /> preventing inappropriate interactions and/or aggregation         important proper folding ER export (DeWitt et al., <br /> between nascent polypeptides. In addition, protein folding       1998). Because Pma1 cell surface essential for <br /> in ER catalyzed enzymes peptidyprolyl          growth, defect Pma1 biogenesis causes growth arrest. <br /> isomerase protein disulfide isomerase (PDI) (Gething         The pma1-D378N allele, presence wild- <br /> and Sambrook, 1992; Ferrari Soling, 1999). Together,         type PMA1, dominant-negative effect growth <br /> the chaperones folding enzymes comprise component          (Harris et al., 1994; DeWitt et al., 1998; Maldonado et al., <br /> of ER quality control designed identify     1998; Nakamoto et al., 1998). <br /> immature abnormal proteins, helping ensure           We report Pma1-D378N undergoes degradation <br /> that correctly folded assembled proteins            ER ubiquitin-dependent manner. In the <br /> released secretory pathway. Those newly synthe-         presence mutant allele, wild-type Pma1 also <br /> sized proteins fail fold properly ultimately         undergoes retention degradation. A genetic selection <br /> targeted degradation, primarily retrograde transloca-     mutants suppress dominant-negative growth <br />  <br /> 5972                                                                              &#194;&#169; European Molecular Biology Organization <br />                                                                                                                       ER quality control yeast <br />  <br />  <br />  <br />  <br /> Fig. 1. Dominant-negative effect mutant Pma1-D378N. (A) Growth plates (synthetic complete medium containing 2% galactose) room <br /> temperature. Cells (F1105) bearing pGAL-HA-PMA1 (pXZ28) grow. Cells bearing pGAL-pma1-D378 (pRN409) grow. Cells grow with <br /> both URA3-marked plasmid bearing PMA1 (pXZ28) LEU2-marked plasmid bearing pma1-D378N (pRN409). eps1 permits growth of <br /> cells (WQY1) expressing Pma1-D378N. (B) Western blot showing induction Pma1 stability shutting synthesis. Wild-type cells <br /> (F1105) bearing pGAL-HA-PMA1 (pXZ28) pGAL-myc-pma1-D378N (pWQ2) plasmids grown log phase synthetic complete <br /> medium containing 2% raffinose (time 0) transferred medium containing 2% galactose induce synthesis epitope-tagged Pma1 for <br /> 4 h (on). Glucose added 3% shut synthesis incubation continued 4 h (off). Cell lysate prepared, normalized protein <br /> for SDS&#226;&#8364;&#8220;PAGE analyzed Western blotting using anti-HA (left panel) anti-myc antibody (right panel). Newly synthesized Pma1-D378N is <br /> degraded wild-type Pma1 stable. When plasmids bearing PMA1 pma1-D378N co-expressed, wild-type mutant <br /> molecules degraded. <br />  <br /> effect Pma1-D378N resulted identification                       chase, newly synthesized wild-type Pma1 (left panel) <br /> EPS1 (ER-retained Pma1 suppressing). Importantly,                         remains relatively stable, consistent long half- <br /> eps1&#226;&#710;&#8224; cells, retention Pma1-D378N wild-type                           life Pma1 (Benito et al., 1991; Chang Fink, 1995), <br /> Pma1 reversed, molecules delivered                    Pma1-D378N rapidly degraded (right panel). <br /> plasma membrane. Nevertheless, retention ER                           Degradation newly synthesized wild-type Pma1 also <br /> resident proteins Kar2 Shr3 appears normal. In                       accelerated significantly Pma1-D378N expressed <br /> absence Pma1-D378N, ER export wild-type Pma1                           (Figure 1B). <br /> is perturbed eps1 deletion; however, export                         Because pma1 mutants reported to <br /> wild-type membrane marker, Gas1, delayed. EPS1                            escape ER quality control undergo degradation the <br /> encodes novel membrane protein belonging PDI                        vacuole (Chang Fink, 1995), Pma1-D378N degrada- <br /> family. We propose role Eps1 recognition                        tion examined pep4&#226;&#710;&#8224; cells, defective vacuolar <br /> delivery substrate ER quality control machinery.                   proteinase activity. Figure 2A shows Pma1-D378N <br />                                                                              degradation pep4 cells inhibited respect to <br /> Results                                                                      wild-type cells. Degradation Pma1-D378N was <br />                                                                              examined ubc6 ubc7 mutant ubiqui- <br /> Retention degradation Pma1-D378N ER                                tin-conjugating enzymes Ubc6 Ubc7 required for <br /> quality control                                                              ERAD (Brodsky McCracken, 1997; Sommer and <br /> The pma1-D378N allele dominant-negative effect                         Wolf, 1997). Figure 2B shows ubc6 ubc7 cells, <br /> on growth cells expressing mutant protein                      degradation newly synthesized Pma1-D378N slowed <br /> grow chromosomal copy PMA1                         comparison wild-type cells. These data <br /> (Figure 1A Harris et al., 1994). Previously,                      reveal Pma1-D378N degradation dependent on <br /> been suggested mutant Pma1-D378N retained                         Ubc6 Ubc7, vacuolar proteases, indicating <br /> the ER consequence quality control mechanism                       substrate ER quality control. <br /> (Harris et al., 1994; Nakamoto et al., 1998). To determine <br /> whether Pma1-D378N undergoes degradation, exam- <br /> ined fate newly synthesized molecule. To                          Accumulation Pma1-D378N affects ER structure <br /> monitor mutant wild-type proteins independently,                     function <br /> we employed constructs wild-type                           The observation ER retention degradation of <br /> Pma1-D378N tagged hemagglutinin (HA)                           Pma1-D378N causes, turn, retention degradation <br /> c-myc epitopes, respectively. Expression placed                    wild-type Pma1 suggests possible explanation the <br /> the control regulated promoter; GAL1 MET25                          dominant-negative growth effect mutant allele: <br /> promoters produced similar results (see below). In                           wild-type Pma1 essential activity plasma <br /> Figure 1B, cells bearing pGAL-myc-pma1D378N                             membrane rate limiting growth. Because Pma1 <br /> shifted galactose-containing medium induce syn-                        turns cell surface half-time ~11 h <br /> thesis. Glucose added shut synthesis. Levels                   (Benito et al., 1991), effect cell growth to <br /> newly synthesized Pma1 detected Western blot                         defective delivery newly synthesized Pma1 predicted <br /> (with anti-myc anti-HA antibodies) 4 h induction                   slow onset. Indeed, growth curves confirmed <br /> as 4 h chase. Figure 1B shows                      detectable decrease rate growth first <br />  <br />                                                                                                                                              5973 <br />  Q.Wang A.Chang <br />  <br />  <br />  <br />  <br /> Fig. 2. Degradation mutant Pma1&#226;&#8364;&#8220;D378N. (A) Stability of <br /> Pma1&#226;&#8364;&#8220;D378N presence pep4&#226;&#710;&#8224; (WQY2), eps1&#226;&#710;&#8224; (WQY1) and <br /> multicopy EPS1 (F1105-pWQ29). Cells bearing pMET-HA-pma1- <br /> D378N (pWQ12) transferred methionine-free medium 2 h         Fig. 3. Effect expression Pma1-D378N. (A) Growth cells <br /> (time 0). Methionine added and, various times        expression Pma1-D378N. Cells exponentially growing in <br /> chase, Pma1-D378N levels quantitated Western blot            synthetic complete medium containing 2% raffinose transferred <br /> phosphorimaging, expressed percentage induced protein.       medium containing 2% galactose. Cultures incubated 30&#194;&#176;C <br /> Rates Pma1-D378N degradation similar wild-type (u)             OD600 measurements various times. Note there <br /> (t1/2 estimated semi-log plot ~1.28 h) pep4&#226;&#710;&#8224; (n) cells          detectable difference growth rate cells bearing pGAL-PMA1 <br /> (t1/2 ~1.1 h). Stabilization mutant Pma1 apparent eps1&#226;&#710;&#8224; cells   (pXZ28) pGAL-pma1-D378 (pWQ2) &#195;&#185;6 h shift to <br /> (d) (t1/2 ~3 h), degradation increased multicopy EPS1 (e)    galactose&#226;&#8364;&#8220;containing medium. (B) Intracellular transport CPY in <br /> (t1/2 ~0.2 h). (B) Pulse&#226;&#8364;&#8220;chase analysis wild-type (sub62) (u)     cells expressing mutant Pma1-D378N. Cells bearing pGAL-PMA1 or <br /> ubc6 ubc7 (MHY552) (r) cells bearing pMET-HA-pma1-D378N                  pGAL-pma1-D378N grown raffinose-containing medium and <br /> (pWQ12). Cells resuspended methionine-free medium 2 h        shifted galactose-containing medium 4 h. Cells then <br /> to induce expression Pma1-D378N, labeled                pulse-labeled 30&#194;&#176;C 5 min Expre35S35S chased for <br /> Expre35S35S 15 min. Cells chased presence        various times. Immunoprecipitation anti-CPY antibody was <br /> excess methionine cysteine various times. Immunoprecipitation    normalized TCA-precipitable c.p.m.. The arrowhead indicates <br /> with anti-HA antibody normalized lysate protein, analyzed      P1 CPY persisting cells accumulating Pma1-D378N. <br /> SDS&#226;&#8364;&#8220;PAGE quantitated densitometry. Degradation mutant <br /> Pma1 delayed ubc6 ubc7 cells (t1/2 estimated semi-log plot <br /> ~2.35 h) contrast wild-type cells (t1/2 ~0.85 h). <br />                                                                          (Figure 3B, arrowhead). Delivery vacuolar mem- <br />                                                                          brane protein alkaline phosphatase similarly slowed <br />                                                                          cells expressing Pma1-D378N (not shown). Perturbation <br /> apparent &#195;&#185;6 h induction Pma1-D378N                         ER export dependent accumulation Pma1- <br /> expression (Figure 3A).                                                  D378N: effect CPY export observed time <br />    Alternatively, possible growth defect caused           prior detectable accumulation (at 1 h induction; <br /> by Pma1-D378N accumulation consequence                         shown). <br /> general perturbation ER function. To test this, effect <br /> of Pma1-D378N maturation vacuolar protein                      eps1 suppresses dominant-negative growth <br /> carboxypeptidase Y (CPY) examined pulse&#226;&#8364;&#8220;chase                     defect induced Pma1-D378N <br /> analysis. Newly synthesized CPY enters secretory                     To understand better molecular basis Pma1-D378N <br /> pathway ER 67 kDa P1 form. Further glycosyl-                 retention degradation ER, genetic selection <br /> ation transit Golgi produces 69 kDa P2                     performed isolate eps mutants suppress the <br /> form, proteolytic processing arrival vacuole             growth defect caused pma1-D378N. Cells carrying <br /> yields 61 kDa mature enzyme (Stevens et al., 1982).                  pGAL-pma1-D378N transformed mutagenized <br /> Cells carrying pGAL-pma1-D378N shifted galac-                    genomic library containing random insertions lacZ and <br /> tose-containing medium 4 h, pulse-labeled                   LEU2 (Burns et al., 1994), plated galactose- <br /> with [35S]methionine cysteine. At time, growth              containing medium. Transformants grew galactose <br /> rate altered detectably (Figure 3A). Figure 3B shows              selected. Tetrad analysis used identify sup- <br /> that cells accumulating Pma1-D378N, slight                 pressors LEU2 insertion segregated 2:2, <br /> delay maturation P1 CPY Golgi P2 form,                   indicating single genetic locus, linked growth <br /> contrast cells expressing wild-type Pma1                         galactose medium. While eps mutants were <br />  <br /> 5974 <br />                                                                                                             ER quality control yeast <br />  <br />  <br />                                                                          punctate structures stained appear distinct <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                          organelle described wild-type cells. By contrast, <br />                                                                          staining antibody ER marker Kar2 <br />                                                                          revealed predominantly perinuclear <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, closely <br />                                                                          resembling ER pattern reported wild-type cells <br />                                                                          (Rose et al., 1989). In cells, myc staining is <br />                                                                          coincident Kar2 staining, indicating mutant <br />                                                                          Pma1-D378N retained ER (Figure 4A, arrowheads <br />                                                                          Harris et al., 1994). There extensive co- <br />                                                                          localization Pma1-D378N Kar2 perinuclear <br />                                                                          distribution cells expressing MET-pma1-D378N (see <br />                                                                          Figure 9), suggesting large punctate structures are <br />                                                                          reflection GAL1-induced protein proliferation. In the <br />                                                                          presence mutant allele, localization pattern of <br />                                                                          HA-tagged wild-type Pma1 punctate (Figure 4A, <br />                                                                          right panel); obvious staining mutant <br />                                                                          wild-type Pma1 plasma membrane. ER retention <br />                                                                          mutant wild-type Pma1 consistent with <br />                                                                          degradation wild-type protein presence of <br />                                                                          Pma1-D378N (Figure 1B). <br />                                                                             Co-localization newly synthesized Pma1-D378N and <br />                                                                          wild-type Pma1 ER examined by <br /> Fig. 4. Indirect immunofluorescence localization Pma1 wild-type    cell fractionation. Previous work established that <br /> and eps1 cells expressing Pma1-D378N. Exponentially growing cells <br /> (F1105) bearing pGAL-myc-pma1 (pWQ2) and/or pGAL-HA-PMA1                 Renografin density gradients efficiently resolve plasma <br /> (pXZ28) transferred medium containing 2% galactose. After        membrane intracellular (ER, Golgi vacuole) <br /> 4 h, cells fixed, spheroplasted permeabilized indirect      membranes (Schandel Jenness, 1994). As control, <br /> immunofluorescence. (A) Co-localization Pma1-D378N Kar2           fractionation pattern cells expressing pGAL1-HA- <br /> (top left middle panels). Cells expressing myc-Pma1-D378N were <br /> stained anti-myc monoclonal polyclonal anti-Kar2 antibodies, <br />                                                                          PMA1 determined (Figure 5A, panel a); HA- <br /> followed Cy3- Cy2-conjugated secondary antibodies,                Pma1 present predominantly fractions 10 11, <br /> respectively. Staining myc-Pma1-D378N frequently coincident        coincident plasma membranes [detected using the <br /> with (arrowheads), identical staining pattern ER   plasma membrane marker Gas1 (Figure 10 below) and <br /> marker Kar2. Right panel: cells expressing myc-tagged mutant        anti-Pma1 antibody absence induced protein (not <br /> and HA-tagged wild-type Pma1 stained anti-HA antibody. <br /> The distribution wild-type Pma1 appears similar Pma1-      shown)]. When wild-type Pma1 expressed with <br /> D378N. (B) Pma1 localization eps1&#226;&#710;&#8224; cells (WQY1) expressing            Pma1-D378N, HA-tagged protein shifts fractions 7 <br /> myc-Pma1-D378N HA-Pma1 stained anti-myc                8, myc-Pma1-D378N accumulates <br /> anti-HA antibodies. Staining wild-type mutant Pma1        (Figure 5A, panels b c). These fractions enriched <br /> predominantly cell surface. <br />                                                                          ER, based content ER membrane <br />                                                                          marker Sec61 (panel d). <br /> identified satisfied criteria, eps1 cells                   In eps1 cells, strikingly, localization indirect <br /> permitted substantial synthesis Pma1-D378N occur.                  immunofluorescence shows staining Pma1-D378N <br /> Therefore, quite likely eps1 affects                  wild-type Pma1 cell surface (compare Figure 4B <br /> post-translational fate mutant Pma1. Figure 1A shows                  A). In agreement immunofluorescence <br /> that cells insertion EPS1 coding                      results, fractionation Renografin density gradient <br /> sequence grow Pma1-D378N expressed. A                        shows mutant wild-type Pma1 migrating with <br /> strain constructed bearing complete deletion                   plasma membrane fractions 10 11, away from <br /> replacement EPS1 HIS3; cells suppress                 ER marker, Sec61, fraction 7 (Figure 4B). Therefore, <br /> the growth defect caused mutant Pma1 expressed                   eps1&#226;&#710;&#8224; stabilizes Pma1-D378N reversing ER reten- <br /> either GAL1 (Figure 1A) MET25 promoters (not shown),                  tion, permitting export mutant wild-type <br /> and used analysis.                                      Pma1 plasma membrane. <br />                                                                             Because Eps1 required ER retention Pma1- <br /> eps1 allows mutant Pma1 ER                               D378N, work carried determine whether <br /> plasma membrane                                                          retention ER resident proteins Shr3 Kar2 affected <br /> To address mechanism eps1 suppresses                        eps1&#226;&#710;&#8224;. Localization membrane protein Shr3 <br /> pma1-D378N, stability localization newly                      (Ljungdahl et al., 1992) lumenal protein Kar2 <br /> synthesized Pma1 examined. As shown Figure 2A,                   examined indirect immunofluorescence. As shown <br /> degradation mutant Pma1 slowed eps1&#226;&#710;&#8224; cells;                  Figure 6A, Shr3 Kar2 typical <br /> contrast, EPS1 overexpression increases mutant Pma1                      ER distribution surrounding nucleus [shown by <br /> degradation.                                                             49,6-diamidino-2-phenylindole (DAPI) staining] eps1 <br />    Localization mutant Pma1 studied indirect                   cells. Localization ER residents eps1 cells is <br /> immunofluorescence. ER retention myc-tagged Pma1-                     indistinguishable wild-type cells. Figure 6B <br /> D378N confirmed wild-type cells. Figure 4A                  shows detectable secretion Kar2 eps1&#226;&#710;&#8224; wild- <br /> (left panel, arrowheads) shows staining cells expressing              type cells, contrast erd2 cells readily secrete <br /> myc-tagged mutant Pma1 GAL1 promoter. Large                       Kar2 (Semenza et al., 1990). Therefore, eps1&#226;&#710;&#8224; prevents <br />  <br />                                                                                                                                 5975 <br />  Q.Wang A.Chang <br />  <br />  <br />                                                                           hydropathy analysis Eps1 predicts N-terminal signal <br />                                                                           sequence single transmembrane domain (underlined <br />                                                                           Figure 7A), Eps1 terminates sequence <br />                                                                           KKKNQD, fits closely consensus motif for <br />                                                                           retention transmembrane proteins ER (Jackson <br />                                                                           et al., 1990). <br />                                                                              While PDI1 essential cell viability, EPS1 and <br />                                                                           members family not. Indeed, eps1&#226;&#710;&#8224; does <br />                                                                           cause detectable growth defect (not shown). <br />                                                                           Overexpression yeast PDI family members allows <br />                                                                           cells grow absence PDI1 (Tachibana and <br />                                                                           Stevens, 1992; Tachikawa et al., 1995, 1997). A plasmid <br />                                                                           shuffle experiment performed determine whether <br />                                                                           EPS1 overexpression suppress lethality pdi1&#226;&#710;&#8224;. <br />                                                                           For purpose, strain employed a <br />                                                                           chromosomal deletion PDI1 complemented the <br />                                                                           PDI1 gene URA3-marked plasmid. These cells <br />                                                                           live plasmid grow on <br />                                                                           5-fluoro-orotic acid (5-FOA) (Boeke et al., 1987). The <br />                                                                           ability high copy EPS1 replace URA3-marked <br />                                                                           plasmid bearing PDI1 assayed growth 5-FOA. <br />                                                                           As positive control, ability EUG1 overexpression <br />                                                                           suppress lethality pdi1&#226;&#710;&#8224; tested. Figure 7B <br /> Fig. 5. Pma1 distribution wild-type eps1 cells expressing <br /> Pma1-D378N cell fractionation. Cells transferred galactose- <br />                                                                           shows EPS1 overexpression, like EUG1, <br /> containing medium 4 h induce Pma1 synthesis, described       allows pdi1&#226;&#710;&#8224; cells grow 5-FOA absence <br /> Figure 4. Cells lysed fractionation Renografin density   PDI1. <br /> gradients. (A) Distribution Pma1 wild-type cells. Panel shows        Because Eps1 replace essential function PDI, <br /> the distribution HA-Pma1 wild-type cells (F1105) expressing         assess Eps1 plays role <br /> solely pGAL-HA-PMA1 (pXZ28); protein predominantly <br /> in fractions 10&#226;&#8364;&#8220;11. In cells, ER marker Sec61          disulfide bond formation newly synthesized proteins. <br /> predominantly fractions 7&#226;&#8364;&#8220;8 (not shown). Panels b c,               Previous work demonstrated disulfide bond forma- <br /> respectively, distribution myc-Pma1-D378N (pWQ2)          tion required proper folding ER export CPY <br /> HA-Pma1 (pXZ28) molecules co-expressed wild-             (Tachibana Stevens, 1992). Therefore, examined <br /> type cells. Under circumstances, major fraction HA-tagged <br /> Pma1 fractions 7&#226;&#8364;&#8220;8, myc-tagged Pma1-D378N and <br />                                                                           intracellular transport CPY eps1 cells. Figure 8 <br /> Sec61 (panel d). (B) Distribution Pma1 eps1&#226;&#710;&#8224; cells.                 shows discernible difference kinetics <br /> Fractionation HA-Pma1 myc-Pma1-D378N eps1&#226;&#710;&#8224; (WQY1)               appearance P1, P2 M forms CPY wild- <br /> cells shown. Both proteins predominantly fractions        type eps1&#226;&#710;&#8224; cells. Specifically, accumulation P1 <br /> 10&#226;&#8364;&#8220;11, contrast Sec61, predominantly fraction 7.      ER form CPY eps1 cells detected. These <br />                                                                           data indicate Eps1 required disulfide bond <br /> retention Pma1-D378N does affect retention                  formation ER export CPY. <br /> normal ER proteins.                                                          In order characterize Eps1, employed construct <br />                                                                           tagged c-myc epitope C-terminus before <br /> Eps1 novel member protein disulfide                           KKKNQD sequence; construct behaves like wild- <br /> isomerase-related family                                                  type Eps1 ability reverse suppression pma1&#226;&#8364;&#8220; <br /> The identity disrupted eps1 gene determined                    D378N complementing eps1&#226;&#710;&#8224; <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> by cloning sequencing genomic DNA immediately                         (described below). By Western blot (Figure 9A), <br /> adjacent insertion (Burns et al., 1994). A database                myc-tagged Eps1 seen faint single band Mr <br /> search revealed eps1 contained insertion                   ~88 kDa, slightly larger predicted sequence. <br /> the open reading frame corresponding YIL005w (Cherry                   The intensity signal increases substantially when <br /> et al., 1997). BLAST analysis indicates EPS1 encodes                 tagged EPS1 expressed high copy plasmid. Upon <br /> a novel protein high sequence similarity PDI                      fractionation lysate, Eps1 associated mem- <br /> (~20% identity) proteins PDI family.                     brane pellet isolated following centrifugation 10 000 g. <br /> Eps1 contains domain common members                     Because Eps1 remains associated membranes after <br /> of family active site oxidoreductase                extraction 0.5 M NaCl 0.1 M Na carbonate, <br /> thioredoxin. The domain contains CXXC motif                    pH 11.5 (Figure 9A), conclude Eps1 behaves as <br /> participate dithiol&#226;&#8364;&#8220;disulfide exchange reactions (Luz                  integral membrane protein (Steck Yu, 1973). <br /> and Lennarz, 1996; Ferrari Soling, 1999) (asterisks,                  Localization Eps1 determined indirect <br /> Figure 7A). Figure 7A shows alignment Eps1                     immunofluorescence. Figure 9B shows staining of <br /> members PDI family identified yeast, including                  eps1&#226;&#710;&#8224; cells bearing multicopy myc-tagged EPS1. (Specific <br /> Pdi1 (Gunther et al., 1991; LaMantia et al., 1991), Eug1                  staining construct centromeric plasmid was <br /> (Tachibana Stevens, 1992), Mpd1 (Tachikawa et al.,                    detectable.) Staining myc&#226;&#8364;&#8220;Eps1 coincident with <br /> 1995) Mpd2 (Tachikawa et al., 1997). Other members                    perinuclear Kar2 staining cells (Figure 9B, <br /> of yeast PDI family ER lumenal proteins [with                     arrows). In presence Pma1-D378N, some <br /> HDEL signals C-terminus (Figure 7A)]. In contrast,                 co-localization Eps1 mutant Pma1. Additionally, <br />  <br /> 5976 <br />                                                                                                                      ER quality control yeast <br />  <br />  <br />  <br />  <br /> Fig. 6. Retention ER proteins. (A) Localization ER resident proteins Shr3 Kar2. Indirect immunofluorescence staining anti-HA and <br /> DNA staining DAPI performed wild-type (F1105) eps1 (WQY1) cells bearing HA-SHR3 (B2513). Staining anti-Kar2 antibody <br /> was performed eps1 cells. (B) Kar2 secretion assay. Western blot anti-Kar2 antibody nitrocellulose filter overlying wild-type, eps1 and <br /> erd2 (MY3331) cells. Only erd2 cells secrete Kar2. <br />  <br />  <br />  <br /> there punctate Eps1 staining cytoplasm                    ER export wild-type Pma1, facilitates export <br /> distinct Kar2 Pma1-D378N staining.                            Gas1. <br />  <br /> Effect eps1 protein export ER <br />                                                                             Discussion <br /> The effect eps1 intracellular transport wild-type <br /> Pma1 examined absence Pma1-D378N.                             The Pma1-D378N mutant misfolded, retained and <br /> Previously, shown conditions                      accumulated ER (Figures 4 5; Harris et al., <br /> which ER export Pma1 impaired, Pma1 accumulation                      1994; DeWitt et al., 1998; Maldonado et al., 1998; <br /> at ER detected (Roberg et al., 1999). Therefore,                 Nakamoto et al., 1998). Pma1-D378N substrate for <br /> the distribution wild-type Pma1 eps1 cells                        ER quality control degradation dependent on <br /> studied density gradient fractionation. Figure 10 shows                  Ubc6 Ubc7, Pep4 (Figure 2). Both mutant and <br /> that eps1 wild-type cells, Pma1                     wild-type Pma1 retained degraded concomitantly <br /> plasma membrane-enriched fractions 10 11 (panels                      (Figure 1). Recent evidence Pma1 forms hexamer <br /> and b). Similarly, indirect immunofluorescence, Pma1                     (Auer et al., 1998) supports model oligomeriz- <br /> appears localized exclusively plasma membrane                     ation mutant wild-type Pma1 ER results in <br /> both eps1 wild-type cells (not shown).                                  retention degradation molecules. <br />    Strikingly, effect eps1 wild-type                         We suspect growth defect associated Pma1- <br /> Pma1 transport detected, ER accumulation                         D378N accumulation accounted block Pma1 <br /> plasma membrane protein Gas1 observed. Insertion                        export coupled gradual depletion Pma1 enzyme <br /> of Gas1 ER membrane occurs GPI anchor                        activity cell surface. This idea supported a <br /> and accompanied core glycosylation, generating                     decrease growth rate detectable &#195;&#185;6 h <br /> Mr 105 kDa form. Movement Golgi rapid                         induction Pma1&#226;&#8364;&#8220;D378N expression (Figure 3), <br /> Gas1 shifts 125 kDa acquires glycosylation                 consistent slow rate Pma1 turnover the <br /> (Doering Schekman, 1996; Sutterlin et al., 1997).                       plasma membrane (Benito et al., 1991). Even so, time <br /> Figure 10 shows Gas1 distribution density gradient                    growth significantly affected, Pma1-D378N <br /> fractionation (panels c d). In wild-type cells, Gas1                 accumulation begins cause moderate disturbances of <br /> predominantly plasma membrane (fractions 10                      ER structure function. By 4 h induction, both <br /> 11). By contrast, eps1&#226;&#710;&#8224; cells, accumulation                  mutant wild-type Pma1 observed large punct- <br /> the 105 kDa ER form Gas1 lower density (7                         ate structures (Figure 4); structures represent <br /> 8) fractions (arrowhead, panel c). Pulse&#226;&#8364;&#8220;chase analysis                     proliferated ER membrane, reported upon <br /> revealed delay generation 125 kDa Golgi-                        overproduction various membrane proteins (reviewed <br /> modified form Gas1 eps1&#226;&#710;&#8224; cells comparison                     de-Kerchove d&#226;&#8364;&#8482;Exaerde et al., 1996). In addition, export <br /> wild-type cells, consistent idea slowed ER                    newly synthesized proteins ER perturbed <br /> export results ER accumulation (not shown). These                        cells accumulate Pma1-D378N (Figure 3). In <br /> data suggest Eps1 does play important role                      contrast complete block export wild-type <br />  <br />                                                                                                                                            5977 <br />  Q.Wang A.Chang <br />  <br />  <br />  <br />  <br /> Fig. 7. Eps1 member family proteins related protein disulfide isomerase. (A) Alignment Eps1 members yeast PDI <br /> family. Sequences aligned using MegAlign program (DNAstar, Madison, WI) using Clustal method (Higgins et al., 1992). Identical <br /> amino acid residues boxed, hyphens indicate gaps introduced maximize alignment. Asterisks overlie conserved thioredoxin-like CXXC <br /> motifs; PDI Eps1 single thioredoxin-like domain. All members family, Eps1, HDEL retention signals the <br /> C-terminus; Eps1 terminates residues KKKNQD C-terminus. The predicted transmembrane domain underlined. (B) Overexpression <br /> of EPS1 suppress lethality pdi1&#226;&#710;&#8224;. A strain chromosomal deletion PDI1 complemented PDI1 URA3-marked plasmid was <br /> transformed multicopy EPS1 (pWQ19) GAL1-EUG1 (pCT44). After growth medium containing galactose uracil, Ura&#226;&#8364;&#8220; cells were <br /> selected 5-FOA. Overexpression EUG1 EPS1 suppresses lethality pdi1&#226;&#710;&#8224;. <br />  <br />  <br /> Pma1, kinetic slowing export                    EPS1 genetic selection suppressors pma1- <br /> proteins. Indeed, effect likely consequence                 D378N. Many previous genetic screens identify <br /> of Pma1 accumulation detected early                   components yeast ER quality control apparatus have <br /> times induction.                                                      isolated mutants prevent degradation substrate, <br />    A major finding study identification                   leading identification components degradation <br />  <br /> 5978 <br />                                                                                                                     ER quality control yeast <br />  <br />  <br />  <br />  <br /> Fig. 8. Effect eps1 maturation CPY. Wild-type (F1105) and <br /> eps1&#226;&#710;&#8224; (WQY1) cells pulse-labeled 5 min room temperature <br /> with Expre35S35S chased various times. Immunoprecipitation <br /> of CPY lysates normalized acid-precipitable c.p.m. The <br /> kinetics CPY maturation wild-type eps1 cells similar. <br />  <br />  <br />                                                                            Fig. 10. Effect eps1 protein export ER. Localization of <br />                                                                            Pma1 Gas1 wild-type (F1105) eps1 (WQY1) cells. Cell <br />                                                                            lysate fractionated Renografin density gradient. Fractionation <br />                                                                            analyzed Western blotting polyclonal anti-Pma1 anti- <br />                                                                            Gas1 antibodies. In panels b, Pma1 predominantly in <br />                                                                            fraction 10 eps1 wild-type cells, respectively. In panel c, in <br />                                                                            eps1 cells, accumulation 105 kDa ER (arrowhead) form <br />                                                                            Gas1 fractions 7 8; addition, 125 kDa mature form is <br />                                                                            peak fraction 10. In panel d, majority Gas1 in <br />                                                                            wild-type cells fraction 10. <br />  <br />  <br />                                                                            machinery (Hampton et al., 1996; Knop et al., 1996; <br />                                                                            McCracken et al., 1996; Loayza et al., 1998). Because <br />                                                                            selection method suppressors pma1-D378N was <br />                                                                            based ability cells grow (by delivering <br />                                                                            Pma1 cell surface), unlikely yield mutants <br />                                                                            defective degradation se. Indeed, eps1 cells stabilize <br />                                                                            Pma1-D378N abolishing ER retention allowing <br />                                                                            export mutant wild-type Pma1 cell surface <br />                                                                            (Figures 4 5). <br />                                                                               We Eps1 required ER retention Pma1- <br />                                                                            D378N does play important role normal <br />                                                                            retention resident ER proteins (Figure 6). Moreover, <br />                                                                            Eps1 does appear play role intracellular <br />                                                                            transport wild-type Pma1 absence Pma1- <br />                                                                            D378N (Figure 10). Therefore, eps1&#226;&#710;&#8224; does appear to <br />                                                                            cause general perturbation protein trafficking. <br />                                                                               Eps1 novel ER membrane protein containing <br />                                                                            CXXC domain conserved PDI family members <br />                                                                            (Figures 7and 9). Unlike PDI1, EPS1 essential for <br />                                                                            cell viability. While essential function PDI to <br />                                                                            reshuffle non-native disulfide bonds mediated its <br />                                                                            thioredoxin domains (Luz Lennarz, 1996; Wang and <br />                                                                            Tsou, 1998; Ferrari Soling, 1999), known <br />                                                                            thioredoxin-like domain Eps1 plays <br /> Fig. 9. Eps1 ER membrane protein. (A) Western blot. Lysate       role ER quality control. If Eps1 interacts directly <br /> prepared eps1&#226;&#710;&#8224; cells (WQY1) centromeric          Pma1-D378N, probably mediated domain <br /> 2&#194;&#181; plasmid bearing myc-tagged EPS1 (pWQ28 pWQ30),                  lumenal thioredoxin domain topolo- <br /> mixed equal volume 1 M NaCl 0.2 M Na carbonate,              gical predictions place ~4% Pma1 lumen <br /> pH 11.5. After incubation ice 30 min, sample was <br /> centrifuged 100 000 g 1 h generate soluble membrane <br />                                                                            (Serrano, 1989; Auer et al., 1998), disulfide bonding <br /> fractions. The presence Eps1 total (T), supernatant (S) pellet   does appear play role Pma1 structure or <br /> (P) fractions detected anti-myc antibody. The Mr                  function (Petrov Slayman, 1995). At present, we <br /> ~88 000 kDa band present lysate plasmid (NT).            rule possibility ER export Pma1- <br /> Myc-Eps1 remains associated membranes extraction           D378N indirect effect eps1&#226;&#710;&#8224;, instance, <br /> high salt high pH buffer. (B) Localization Eps1 indirect <br /> immunofluorescence. Upper panels: eps1&#226;&#710;&#8224; cells bearing myc-EPS1             redox potential ER. Notably, however, <br /> (pWQ29) fixed double staining anti-myc anti-Kar2,        folding export redox-sensitive protein, CPY, is <br /> followed Cy3- Cy2-conjugated secondary antibodies. Arrows           perturbed detectably eps1 cells (Figure 8). <br /> indicate co-localization. Lower panels: double staining myc-Eps1           Although means proteins selected for <br /> with HA-Pma1-D378N. Cells (WQY1) bearing pMET-HA-pma1 <br /> (pWQ12) myc-EPS1 (pWQ29) growing exponentially minimal <br />                                                                            ER-associated degradation known, been <br /> medium containing 600 &#194;&#181;M methionine harvested, washed             suggested molecular chaperones participate in <br /> resuspended methionine-free medium induce synthesis mutant        identifying delivering substrates degradation <br /> Pma1 (see Materials methods). After 3 h, cells prepared       machinery (Hammond Helenius, 1995; Brodsky and <br /> staining polyclonal anti-HA monoclonal anti-myc antibodies.       McCracken, 1997; Kopito, 1997; Sommer Wolf, <br />  <br />                                                                                                                                            5979 <br />  Q.Wang A.Chang <br />  <br />  <br /> 1997). The observation Eps1 member PDI                       HindIII XhoI sites cloning FB1521, URA3-marked <br />                                                                               centromeric plasmid bearing MET25 promoter (Mumberg et al., <br /> family suggests model Eps1                              1994), generating pWQ5. To replace myc tag HA tag, the <br /> chaperone-like function recognition retention                       plasmid pND542 (derived pXZ28, provided N.Davis, Wayne <br /> substrates ER quality control. Indeed,                     State University, MI) used template PCR. A fragment of <br /> increasingly clear members PDI family                        0.8 kb amplified using oligonucleotide TCCCCCGGGAGCTA- <br /> chaperone functions addition enzymatic                            GTTAAAGAAAATC introduce SmaI site &#226;&#8364;&#8220;67 bp the <br />                                                                               start codon oligonucleotide CCTTCACCTCTCTTAACA. After <br /> activities (Luz Lennarz, 1996; Wang Tsou, 1998;                       cutting SmaI BstEII, PCR fragment used replace <br /> Ferrari Soling, 1999). Our hypothesis consistent                       corresponding fragment pWQ5. <br /> with observation ER-associated degradation                           The plasmid pWQ10 disrupt EPS1 constructed using PCR <br /> some substrates inhibited calnexin BiP mutants                      amplify genomic DNA; 1.5 kb PCR product introduced EcoRI <br />                                                                               SmaI restriction sites upstream EPS1 start codon 1.5 kb <br /> (Plemper et al., 1997; Brodsky et al., 1999). Moreover,                       product introduced SmaI&#226;&#8364;&#8220;XbaI sites downstream stop codon <br /> in regard, eps1 cells               cloned pBSII; mark disruption, 3.1 kb fragment <br /> delay export newly synthesized Gas1, resulting                          containing HIS3 kanr (from B2179, Fink laboratory collection) was <br /> in ER accumulation (Figure 10). This observation                           inserted SmaI. <br /> consistent role Eps1 facilitating ER export                        For plasmid shuffle experiment, pWQ19 constructed by <br />                                                                               amplifying genomic DNA PCR using oligonucleotide introduce <br /> of Gas1, exclude possibility                      SalI site ~1 kb upstream EPS1 start codon (AAGTCGACCCT- <br /> impairment Gas1 export eps1 cells indirect                           CGGTGAGCACCCCAC) oligo stop codon. The <br /> (Horvath et al., 1994).                                                       product subcloned LEU2-marked 2&#194;&#181; vector (B2201; Fink <br />    While molecular mechanism Eps1 action awaits                        laboratory collection) pGEM-T Easy (Promega, Madison, WI) at <br /> further work, absolute requirement retention                       SalI&#226;&#8364;&#8220;NotI. The multicopy EUG1 plasmid, pCT44, used plasmid <br />                                                                               shuffle, Tom Stevens&#226;&#8364;&#8482; laboratory (Tachibana Stevens, <br /> degradation Pma1-D378N points important func-                        1992). To place epitope tag C-terminus Eps1 prior to <br /> tion ER quality control.                                                   putative KKKNQD retrieval signal, pWQ26 constructed by <br />                                                                               amplifying genomic DNA PCR using oligonucleotide annealing <br />                                                                               ~1 kb upstream start codon oligonucleotide GCCAGAA- <br /> Materials methods                                                         GATCTGGGATC; 3.3 kb product cut SalI BglII <br />                                                                               ligated SalI&#226;&#8364;&#8220;BamHI sites pRS316 (Sikorski Hieter, <br /> Media strains                                                             1989). pWQ26 used template oligonucleotide-directed <br /> Unless specified, experiments performed using strains isogenic           mutagenesis (Kunkel et al., 1987) using TTGATTTTTTTTTTTGGAT- <br /> with W303 (MAT&#206;&#177; ura3-1 leu2-3,112 his3-11,15 trp1-1 ade2-1 can1-              CCCTCCATATTTCC introduce BamHI site tandem <br /> 100; F1105, Fink laboratory collection, Whitehead Institute, Cambridge,       copies myc epitope cloned generate pWQ28. pWQ29 and <br /> MA). WQY1 (MAT&#206;&#177; ura3-1 leu2-3,112 his3-11,15 trp1-1 ade2-1 can1-              pWQ30 2&#194;&#181; vectors marked LEU2 (B2201) URA3 (pRS202), <br /> 100 eps1::HIS3) generated transforming F1105 pWQ10.               respectively, bearing EPS1-myc. <br /> MY3331 (MAT&#206;&#177; suc2 ura3-52 leu2-3, 112 his4-519 erd2-B25) comes <br /> from M.Rose (Princeton University, NJ). MHY552 (MAT&#206;&#177;                          Genetic selection <br /> his3&#226;&#710;&#8224;200 leu2-3,112 ura3-52 lys2-801 trp1-1 ubc6&#226;&#710;&#8224;1::HIS3                      F1105 cells bearing pGAL-pma1-D378N (pRN409U) transformed <br /> ubc7::LEU2) sub62 (MATa his3&#226;&#710;&#8224;200 leu2-3, 112 ura3-52 lys2-                14 independent pools mutagenized genomic library <br /> 801 trp1-1) M.Hochstrasser D.Kornitzer (Technion,               containing random lacZ LEU2 insertions (Burns et al., 1994). Leu1 <br /> Haifa, Israel).                                                               Ura1 transformants (~88 000) plated synthetic complete media <br />    Standard yeast media genetic manipulations described           leucine uracil, containing 2% galactose. Ninety colonies <br /> (Sherman et al., 1986). To study proteins control galactose-   grew galactose medium crossed 10556-20A (MATa, <br /> dependent promoter, cells grown synthetic complete medium             ura3-1, leu2-3,112, his3-11,15, trp1-1, can1-100; Fink laboratory collec- <br /> containing 2% raffinose (in induction inhibition   tion), diploid subjected tetrad analysis determine <br /> of GAL1). Exponentially growing cells transferred synthetic      ability grow presence pGAL-pma1-D378N was <br /> complete medium containing 2% galactose. After induction, synthesis           linked single LEU2 insertion. Seventeen Leu1 Ura1 <br /> was terminated addition glucose 3%. Pma1-D378N             transformants fulfilled criteria, sequence analysis genomic <br /> control MET25 promoter (Mumberg et al., 1994) behaves like             DNA adjacent insertion strains undertaken as <br /> that GAL1 control. To study cells repressing conditions,    described (Burns et al., 1994). DNA sequencing performed the <br /> cells grown minimal medium containing 600 &#194;&#181;M methionine.              AECOM DNA sequencing facility. Sequences analyzed using <br /> To induce synthesis mutant Pma1, cells harvested, washed         BLAST (Altschul et al., 1990). Insertion lacZ eps1::LEU2 <br /> with water, placed minimal medium methionine; shut          mutant codon 1810 total 2106 codons. <br /> off synthesis, 2 mM methionine added. <br />                                                                               Indirect immunofluorescence <br /> Plasmids molecular biology                                                Indirect immunofluorescence staining performed essentially as <br /> B2513 URA3-marked centromeric plasmid containing HA-tagged               described (Rose et al., 1990). Briefly, cultures growing exponentially in <br /> SHR3 (Ljungdahl et al., 1992; Fink laboratory collection). The plasmid        medium containing 2% galactose harvested fixed 4.4% <br /> pXZ28, obtained J.Haber (Brandeis University, Waltham, MA),              formaldehyde, 0.1 M K phosphate, pH 6.5 2 h room temperature. <br /> contains URA3 marker PMA1 control galactose-             Cells spheroplasted oxalyticase (Enzogenetics, Corvallis, OR), <br /> inducible promoter; PMA1 tagged HA epitope introduced              permeabilized methanol acetone. Cells incubated <br /> after second amino acid (Harris et al., 1994). pRN409L pRN409U        overnight monoclonal polyclonal anti-HA (BAbCo, Berkeley, <br /> are LEU2- URA3-marked plasmids, respectively, containing              CA), monoclonal anti-myc (Santa Cruz Biotechnology Inc.) and/or anti- <br /> pma1-D378N mutation 3.8 kb HindIII&#226;&#8364;&#8220;SacI fragment                Kar2 antibody (provided Mark Rose, Princeton, NJ). Primary antibody <br /> pPMA1.2 (Nakamoto et al., 1998); pma1 control             staining visualized Cy2- and/or Cy3-conjugated secondary <br /> GAL1 promoter. pFT4 carries PMA1 c-myc epitope introduced              antibodies (Amersham, Arlington Heights, IL, Jackson Immuno- <br /> after second amino acid. pRN409 pFT4 provided                 Research, West Grove, PA). Experiments Figure 4 photographed <br /> C.Slayman (Yale University, New Haven, CT). pWQ2 bearing myc-                 using Zeiss Axiophot microscope. All experiments visualized <br /> pma1-D378N constructed replacing 750 bp HindIII&#226;&#8364;&#8220;BstEII             using Olympus IX70 microscope, images collected digitally, <br /> fragment pRN409L pFT4; HindIII site               adjusted Adobe Photoshop. <br /> plasmids introduced &#226;&#8364;&#8220;27 start codon PMA1 (Nakamoto <br /> et al., 1991). pWQ12 contains HA-pma1-D378N cloned MET25            Cell fractionation, Western blot metabolic labeling <br /> promoter. pWQ12 constructed cloning 3.7 kb HindIII&#226;&#8364;&#8220;            Cell fractionation Renografin density gradients performed essen- <br /> SmaI fragment bearing myc-pma1-D378N pWQ2 pBSII (KS1)               tially described (Schandel Jenness, 1994; Jenness et al., 1997). <br /> (Stratagene, La Jolla, CA), followed excision fragment using        Briefly, cell lysates prepared suspending cells buffer (50 mM <br />  <br />  <br /> 5980 <br />                                                                                                                      ER quality control yeast <br />  <br />  <br /> Tris, pH 7.5, 1 mM EDTA) presence protease inhibitor           Cherry,J.M., Adler,C., Ball,C., Dwight,S., Chervitz,S., Juvik,G., Weng,S. <br /> cocktail including 1 mM phenylmethylsulfonyl fluoride (PMSF) (Chang          Botstein,D. (1997) Saccharomyces Genome Database (http:// <br /> and Slayman, 1991) vortexing glass beads. Unbroken cells            genome-www.stanford.edu/Saccharomyces). <br /> were removed centrifugation 400 g 5 min. Lysate (0.5 ml)     de-Kerchove d&#226;&#8364;&#8482;Exaerde,A., Supply,P. Goffeau,A. (1996) Subcellular <br /> mixed 0.5 ml Renografin-76 (a gift L.Marsh, Albert Einstein        traffic plasma membrane H1-ATPase Saccharomyces <br /> College Medicine), placed centrifuge tube          cerevisiae. Yeast, 12, 907&#226;&#8364;&#8220;916. <br /> overlaid 1 ml 34, 30, 26 22% Renografin solutions (prepared    DeWitt,N.D., Tourinho dos Santos,C.F., Allen,K.E. Slayman,C.W. <br /> by diluting Renografin-76 Tris-EDTA buffer). Gradients             (1998) Phosphorylation region yeast plasma-membrane H1- <br /> centrifuged SW50.1 rotor overnight 150 000 g 4&#194;&#176;C. Fractions      ATPase. Role protein folding biogenesis. J. Biol. Chem., 273, <br /> (350 &#194;&#181;l) collected gradient. To prevent            21744&#226;&#8364;&#8220;21751. <br /> Renografin interfering subsequent Western blots, aliquots     Doering,T.L. Schekman,R. (1996) GPI anchor attachment required <br /> each fraction (150 &#194;&#181;l) diluted Tris-EDTA buffer containing         Gas1p transport endoplasmic reticulum COPII vesicles. <br /> protease inhibitors (1.5 ml), membranes pelleted centrifuga-     EMBO J., 15, 182&#226;&#8364;&#8220;191. <br /> tion 100 000 g 1 h. For Western blot cell lysate, cells     Ferrari,D.M. Soling,H.-D. (1999) The protein disulphide-isomerase <br /> broken vortexing glass beads described (Chang Slayman,        family: unraveling <span id='am-1' about='xsd:string' typeof='owl:Thing'>string</span> folds. Biochem. J., 339, 1&#226;&#8364;&#8220;10. <br /> 1991), samples normalized protein using Bradford assay     Gething,M.-J. Sambrook,J. (1992) Protein folding cell. Nature, <br /> (Bio-Rad Laboratories, Hercules, CA). After separation SDS&#226;&#8364;&#8220;PAGE           355, 33&#226;&#8364;&#8220;45. <br /> and electrophoretic transfer nitrocellulose, Western blots per-    Gunther,R., Brauer,C., Janetzky,J., Forster,H.-H., Ehbrecht,E.-M., <br /> formed using anti-HA, anti-myc, anti-Sec61 (provided R.Schekman,          Lehle,L. Kuntzel,H. (1991) The Saccharomyces cerevisiae TRG1 <br /> University California, Berkeley, CA) anti-Gas1 (provided           gene essential growth encodes lumenal endoplasmic <br /> T.Doering, Cornell University School Medicine, NY).                       reticulum glycoprotein involved maturation vacuolar <br />    For metabolic labeling experiments involving pGAL-pma1-D378N,             carboxypeptidase. J. Biol. Chem., 266, 24557&#226;&#8364;&#8220;24563. <br /> cells grown mid-log synthetic complete medium           Hammond,C. Helenius,A. (1995) Quality control secretory <br /> methionine cysteine containing 2% raffinose, resus-         pathway. Curr. Opin. Cell Biol., 7, 523&#226;&#8364;&#8220;529. <br /> pended fresh medium containing 2% galactose. For experiments       Hampton,R.Y., Gardner,R.G. Rine,J. (1996) Role 26S proteasome <br /> pMET-pma1-D378N, cells grown mid-log minimal medium               HRD genes degradation 3-hydroxy-3-methylglutaryl- <br /> containing 600 &#194;&#181;M methionine, harvested, washed resuspended           CoA reductase, integral endoplasmic reticulum membrane protein. <br /> methionine-free medium 2 h. Cells labeled Expre35S35S          Mol. Biol. Cell, 7, 2029&#226;&#8364;&#8220;2044. <br /> (2 mCi/25 OD600 cells) chased presence 10 mM methionine      Harris,S.L., Na,S., Zhu,X., Seto-Young,D., Perlin,D., Teem,J.H. and <br /> and cysteine. At various times chase, aliquots placed ice      Haber,J.E. (1994) Dominant lethal mutations plasma membrane <br /> the presence 10 mM Na azide. Cell lysate prepared vortex-          H1-ATPase gene Saccharomyces cerevisiae. Proc. Natl Acad. Sci. <br /> ing glass beads. For immunoprecipitation, lysate samples           USA, 91, 10531&#226;&#8364;&#8220;10535. <br /> normalized acid-precipitable c.p.m. Immunoprecipitations per-      Higgins,D.G., Bleasby,A.J. Fuchs,R. (1992) CLUSTAL V: improved <br /> formed RIPA buffer, analyzed SDS&#226;&#8364;&#8220;PAGE fluorography.            software multiple sequence alignment. Comp. Appl. Biosci., 8, <br /> Polyclonal anti-CPY provided T.Stevens (University Oregon,         189&#226;&#8364;&#8220;191. <br /> Eugene, OR).                                                               Horvath,A., Sutterlin,C., Manning-Krieg,U., Movva,N.R. and <br />                                                                              Riezman,H. (1994) Ceramide synthesis enhances transport GPI- <br />                                                                              anchored proteins Golgi apparatus yeast. EMBO J., 13, <br /> Acknowledgements                                                             3687&#226;&#8364;&#8220;3695. <br />                                                                            Jackson,M.R., Nilsson,T. Peterson,P.A. (1990) Identification a <br /> We thank Carolyn Slayman, James Haber, Mark Rose, Randy Schekman, <br />                                                                              consensus motif retention transmembrane proteins the <br /> Daniel Kornitzer, Tamara Doering, William Lennarz Tom Stevens <br />                                                                              endoplasmic reticulum. EMBO J., 9, 3153&#226;&#8364;&#8220;3162. <br /> for plasmids, antibodies strains. We thank Peter Arvan helpful <br />                                                                            Jenness,D.D., Li,Y., Tipper,C. Spatrick,P. (1997) Elimination of <br /> discussion comments manuscript. This work supported <br />                                                                              defective &#206;&#177;-factor pheromone receptors. Mol. Cell. Biol., 17, 6236&#226;&#8364;&#8220; <br /> by grant NIH (GM 58212). <br />                                                                              6245. <br />                                                                            Knop,M., Finger,A., Braun,T., Hellmuth,K. Wolf,D.H. (1996) Der1, <br /> References                                                                   novel protein specifically required endoplasmic reticulum <br />                                                                              degradation yeast. EMBO J., 15, 753&#226;&#8364;&#8220;763. <br /> Altschul,S.F., Gish,W., Miller,W., Myers,E.W. Lipman D.J. (1990)       Kopito,R. (1997) ER quality control: cytoplasmic connection. Cell, <br />   Basic local alignment tool. J. Mol. Biol., 215, 403&#226;&#8364;&#8220;410.                   88, 427&#226;&#8364;&#8220;430. <br /> Auer,M., Scarborough,G.A. Kuhlbrandt,W. (1998) Three-                  Kunkel,T.A., Roberts,J.D. Zakour,R.A. (1987) Rapid efficient <br />   dimensional map plasma membrane H1-ATPase open               site-specific mutagenesis phenotypic selection. Methods <br />   conformation. Nature, 392, 840&#226;&#8364;&#8220;843.                                        Enzymol., 154, 508&#226;&#8364;&#8220;519. <br /> Benito,B., Moreno,E. Losario,R. (1991) Half-life plasma             LaMantia,M., Miura,H., Tachikawa,H., Kaplan,A., Lennarz,W.J. and <br />   membrane ATPase activating resting yeast cells.          Mizunaga,T. (1991) Glycosylation site binding protein protein <br />   Biochim. Biophys. Acta, 1063, 265&#226;&#8364;&#8220;268.                                     disulfide isomerase identical essential cell viability in <br /> Boeke,J.D., Trueheart,J., Natsoulis,G. Fink,G.R. (1987)                  yeast. Proc. Natl Acad. Sci. USA, 88, 4453&#226;&#8364;&#8220;4457. <br />   5-Fluoroorotic acid selective agent yeast molecular genetics.    Ljungdahl,P.O., Gimeno,C.J., Styles,C.A. Fink,G.R. (1992) SHR3: <br />   Methods Enzymol., 154, 164&#226;&#8364;&#8220;175.                                            novel component secretory pathway specifically required for <br /> Brodsky,J.L. McCracken,A.A. (1997) ER-associated proteasome-         localization amino acid permeases. Cell, 71, 463&#226;&#8364;&#8220;478. <br />   mediated protein degradation: topologically restricted events    Loayza,D., Tam,A., Schmidt,W.K. Michaelis,S. (1998) Ste6p mutants <br />   came together. Trends Cell Biol., 7, 151&#226;&#8364;&#8220;156.                              defective exit endoplasmic reticulum (ER) reveal aspects <br /> Brodsky,J.L., Werner,E.D., Dubas,M.E., Goeckeler,J.L., Kruse,K.B.        ER quality control pathway Saccharomyces cerevisiae. Mol. <br />   McCracken,A.A. (1999) The requirement molecular chaperones             Biol. Cell, 9, 2767&#226;&#8364;&#8220;2784. <br />   endoplasmic reticulum-associated protein degradation              Luz,J.M. Lennarz,W.J. (1996) Protein disulfide isomerase: a <br />   demonstrates protein export import mechanistically            multifunctional protein endoplasmic reticulum. EXS, 77, 97&#226;&#8364;&#8220;117. <br />   distinct. J. Biol. Chem., 274, 3453&#226;&#8364;&#8220;3460.                                Maldonado,A.M., la Fuente,N. Portillo,F. (1998) Characterization <br /> Burns,N., Grimwade,B., Ross-Macdonald,P.B., Choi,E.-Y., Finberg,K.,          allele-nonspecific intragenic suppressor yeast plasma <br />   Roeder,G.S. Snyder,M. (1994) Large-scale analysis gene              membrane H1-ATPase gene (PMA1). Genetics, 150, 11&#226;&#8364;&#8220;19. <br />   expression, protein localization gene disruption Saccharomyces    McCracken,A.A., Karpichev,I.V., Ernaga,J.E., Werner,E.D., Dillin,A.G. <br />   cerevisiae. Genes Dev., 8, 1087&#226;&#8364;&#8220;1105.                                      Courchesne,W.E. (1996) Yeast mutants deficient ER-associated <br /> Chang,A. Fink,G.R. (1995) Targeting yeast plasma membrane         degradation Z variant &#206;&#177;-1-protease inhibitor. Genetics, 144, <br />   [H1]ATPase: novel gene AST1 prevents mislocalization mutant           1355&#226;&#8364;&#8220;1362. <br />   ATPase vacuole. J. Cell Biol., 128, 39&#226;&#8364;&#8220;49.                        Mumberg,D., Muller,R. Funk,M. (1994) Regulatable promoters of <br /> Chang,A. Slayman,C.W. (1991) Maturation yeast plasma              Saccharomyces cerevisiae: comparison transcriptional activity and <br />   membrane [H1]ATPase involves phosphorylation intracellular          use heterologous expression. Nucleic Acids Res., 22, 5767&#226;&#8364;&#8220; <br />   transport. J. Cell Biol., 115, 289&#226;&#8364;&#8220;295.                                    5768. <br />  <br />  <br />                                                                                                                                              5981 <br />  Q.Wang A.Chang <br />  <br />  <br /> Nakamoto,R.K., Rao,R. Slayman,C.W. (1991) Expression the <br />   yeast plasma membrane [H1]ATPase secretory vesicles: new <br />   strategy directed mutagenesis. J. Biol. Chem., 266, 7940&#226;&#8364;&#8220;7949. <br /> Nakamoto,R.K., Verjovski-Almeida,S., Allen,K.E., Ambesi,A., Rao,R <br />   Slayman,C.W. (1998) Substitutions aspartate 378 the <br />   phosphorylation domain yeast PMA1 H1-ATPase disrupt protein <br />   folding biogenesis. J. Biol. Chem., 273, 7338&#226;&#8364;&#8220;7344. <br /> Petrov,V.V. Slayman,C.W. (1995) Site-directed mutagenesis the <br />   yeast PMA1 H1-ATPase; structural functional role cysteine <br />   residues. J. Biol. Chem., 270, 28535&#226;&#8364;&#8220;28540. <br /> Plemper,R.K., Bohmler,S., Bordallo,J., Sommer,T. Wolf,D. (1997) <br />   Mutant analysis links translocon BiP retrograde protein <br />   transport ER degradation. Nature, 388, 891&#226;&#8364;&#8220;895. <br /> Roberg,K. Crotwell,J.M., Espenshade,P., Gimeno,R. Kaiser,C.A. <br />   (1999) LST1 SEC24 homologue used selective export the <br />   plasma membrane ATPase endoplasmic reticulum. J. Cell <br />   Biol., 145, 659&#226;&#8364;&#8220;672. <br /> Rose,M.D., Misra,L.M. Vogel,J.P. (1989) KAR2, karyogamy gene, <br />   yeast homolog mammalian BiP/GRP78 gene. Cell, 57, <br />   1211&#226;&#8364;&#8220;1221. <br /> Rose,M.D., Winston,F. Hieter,P. (1990) Methods Yeast Genetics. <br />   Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, <br />   pp. 198. <br /> Schandel,K.A. Jenness,D.D. (1994) Direct evidence ligand- <br />   induced internalization yeast &#206;&#177;-factor pheromone receptor. Mol. <br />   Cell. Biol., 14, 7245&#226;&#8364;&#8220;7255. <br /> Semenza,J.C., Hardwick,K.G., Dean,N. Pelham,H.R. (1990) ERD2, <br />   yeast gene required receptor-mediated retrieval lumenal <br />   ER proteins secretory pathway. Cell, 61, 1349&#226;&#8364;&#8220;1357. <br /> Serrano,R. (1989) Structure function plasma membrane ATPase. <br />   Annu. Rev. Plant Physiol., 40, 61&#226;&#8364;&#8220;94. <br /> Sherman,F.,. Hicks,J.B. Fink,G.R. (1986) Methods Yeast Genetics: <br />   A Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold <br />   Spring Harbor, NY, pp. 523&#226;&#8364;&#8220;585. <br /> Sikorski,R.S. Hieter,P. (1989) A shuttle vectors and <br />   yeast host strains designed efficient manipulation DNA in <br />   Saccharomyces cerevisiae. Genetics, 122, 19&#226;&#8364;&#8220;27. <br /> Sommer,T. Wolf,D.H. (1997) Endoplasmic reticulum degradation: <br />   reverse protein flow return. FASEB J., 11, 1227&#226;&#8364;&#8220;1233. <br /> Steck,T.L. Yu,J. (1973) Selective solubilization proteins red <br />   blood cell membranes protein perturbants. J. Supramol. Struct., 1, <br />   220&#226;&#8364;&#8220;232. <br /> Stevens,T., Esmon,B. Schekman,R. (1982) Early stages yeast <br />   secretory pathway required transport carboxypeptidase Y <br />   vacuole. Cell, 30, 439&#226;&#8364;&#8220;448. <br /> Sutterlin,C., Doering,T.L., Schimmoller,F., Schroder,S. Riezman,H. <br />   (1997) Specific requirements ER Golgi transport GPI- <br />   anchored proteins yeast. J. Cell Sci., 110, 2703&#226;&#8364;&#8220;2714. <br /> Tachibana,C. Stevens,T.H. (1992) The yeast EUG1 gene encodes <br />   endoplasmic reticulum protein functionally related protein <br />   disulfide isomerase. Mol. Cell. Biol., 12, 4601&#226;&#8364;&#8220;4611. <br /> Tachikawa,H., Funahashi,W., Takeuchi,Y., Nakanishi,H., Nishihara,R., <br />   Katoh,S., Gao,X., Mizunaga,T. Fujimoto,D. (1997) <br />   Ov <br /> </body></html>