<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Methods <br />  <br /> Predicting essential genes fungal genomes <br /> Michael Seringhaus,1,6 Alberto Paccanaro,2,6 Anthony Borneman,3 Michael Snyder,1,3 <br /> and Mark Gerstein1,4,5,7 <br /> 1 <br />  Department Molecular Biophysics Biochemistry, Yale University, New Haven, Connecticut 06520, USA; 2Department <br /> of Computer Science, Royal Holloway University London, Egham, TW20 0EX, United Kingdom; 3Department Molecular, <br /> Cellular Developmental Biology, Yale University, New Haven, Connecticut 06520, USA; 4Program Computational Biology <br /> and Bioinformatics, Yale University, New Haven, Connecticut 06520, USA; 5Department Computer Science, Yale University, <br /> New Haven, Connecticut 06520, USA <br />  <br />  <br />         Essential genes required organism&#226;&#8364;&#8482;s viability, ability identify genes pathogens crucial <br />         directed drug development. Predicting essential genes computational methods appealing it <br />         circumvents expensive difficult experimental screens. Most prediction based homology mapping <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />         experimentally verified essential genes model organisms. We present different approach, relies <br />         exclusively sequence features gene estimate essentiality offers promising way identify essential <br />         genes unstudied uncultured organisms. We identified 14 characteristic sequence features potentially associated <br />         essentiality, localization signals, codon adaptation, GC content, overall hydrophobicity. Using the <br />         well-characterized baker&#226;&#8364;&#8482;s yeast Saccharomyces cerevisiae, employed simple Bayesian framework measure the <br />         correlation of features essentiality. We employed 14 features learn parameters a <br />         machine learning classifier capable predicting essential genes. We trained classifier known essential genes in <br />         S. cerevisiae applied closely related relatively unstudied yeast Saccharomyces mikatae. We assessed <br />         predictive success ways: First, compared predictions generated homology mapping <br />         species. Second, verified subset predictions vivo knockouts S. mikatae, <br />         present experimentally confirmed essential genes species. <br />         [Supplemental material available online www.genome.org. http://www.gersteinlab.org/proj/predess/.] <br />  <br /> Essential genes that, absent, confer lethal phe-                          Several techniques exist identify essential genes. The <br /> notype organism. Such genes make excellent potential                         effective approach&#226;&#8364;&#8221;also difficult&#226;&#8364;&#8221;is large-scale <br /> drug targets (Cole 2002), ability rapidly identify                   experimental gene disruption. Such screens represent massive <br /> has described &#226;&#8364;&#339;the important task genomics-                          investment time resources, fea- <br /> based target validation&#226;&#8364;? (Chalker Lunsford 2002). However,                       sible. For instance, traditional essentiality screens difficult in <br /> experimentally screening lethal gene disruptions challeng-                    Candida albicans partial-diploid nature, mating dif- <br /> ing time consuming, well-studied species. We pro-                        ferences, lack insertional mutagenesis methods (De <br /> pose machine-learning approach predicting essential genes                       Backer et al. 2001). Furthermore, recent shotgun sequencing of <br /> in sequenced largely unstudied fungal species, de-                      microbial communities suggested &lt;1% species are <br /> pends homology comparison known essential genes                            amenable laboratory culture (Riesenfeld et al. 2004; Chen and <br /> but instead sequence features correlate essentiality                    Pachter 2005). Thus, identify essential genes vast ma- <br /> at gene level.                                                                   jority pathogens, look direct experimental <br />      Essentiality defined lethal <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> associated                    methods. <br /> with deletion disablement given gene. Essential genes                          Early comparative genomics approaches involved compar- <br /> account small subset genome (Reich 2000). In                       ing multiple genomes core conserved minimal genome <br /> fungal species account roughly fifth total                       labeling component genes essential (Mushegian Koo- <br /> genes: 17.8% Saccharomyces cerevisiae (Winzeler et al. 1999;                      nin 1996; Arigoni et al. 1998; Bruccoleri et al. 1998). Bayesian <br /> Giaever et al. 2002), 17.5% small subset Schizosac-                    statistical approaches used predict microbial essen- <br /> charomyces pombe studied date (Decottignies et al. 2003). Be-                     tial genes tandem transposon mutagenesis (Lamichhane <br /> cause phenotype product genotype environ-                           et al. 2003). Another machine learning trained <br /> ment, meaningful discuss essentiality relation                 identify essential genes S. cerevisiae integrating genomic <br /> given environmental condition; yeast studies, condition                      experimental data (Jeong et al. 2003). The approach dem- <br /> under essentiality assessed typically laboratory growth                  onstrates ability predict essential genes accurately com- <br /> on rich media.                                                                       putational methods does address applicability <br />                                                                                      methods novel genomes. A useful predictor clas- <br /> 6 <br />  These authors contributed equally work.                                     sify genes essentiality unknown; thus, system <br /> 7 <br />  Corresponding author.                                                               perform outside organism trained. <br /> E-mail mark.gerstein@yale.edu; fax (360) 838-7861.                                         Because essential genes thought evolve slowly <br /> Article published online print. Article publication date http:// <br /> www.genome.org/cgi/doi/10.1101/gr.5144106. Freely available online                   non-essential counterparts (Wilson et al. 1977; Hurst <br /> through Genome Research Open Access option.                                      Smith 1999), researchers tasked identifying in <br />  <br />  <br /> 1126       Genome Research                                    16:1126&#226;&#8364;&#8220;1135 &#194;&#169;2006 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/06; www.genome.org <br />               www.genome.org <br />                                                                                           Predicting essential genes fungal genomes <br />  <br />  <br /> newly sequenced organisms rely homology mapping                   http://bioweb.pasteur.fr/seqanal/interfaces/codonw.html) and <br /> known essential genes model organisms.                                     available virtually microbial genome. <br />       Such homology mapping depends accessibility                       To learn relationship features essentiality, we <br /> closely related organism experimentally determined essen-                require comprehensive set known essential genes. The bak- <br /> tial genes. This approach particularly limited seeking             er&#226;&#8364;&#8482;s yeast S. cerevisiae extremely studied; compre- <br /> identify drug targets inherently biased genes            hensive sustained efforts, 95% genes species <br /> conserved outside host organism. As drug targets,                    systematically deleted (Winzeler et al. 1999; Giaever et <br /> broadly conserved genes naturally useful genes                  al. 2002), yielding definitive mapping essential genes in <br /> unique parasite question. To identify essential drug                species. <br /> targets pathogenic organisms, therefore, disadvantageous                   We selected related relatively unstudied species on <br /> to rely homology mapping.                                                  test classifier. The yeast S. mikatae sequenced in <br />       We propose gene essentiality predicted se-               2003 shares high level sequence homology S. cere- <br /> quenced unstudied species using weighted combination                 visiae (Kellis et al. 2003). Because phylogenetic proximity <br /> certain hallmark features. Whereas homology mapping relies                    high conservation rate essential genes general <br /> continually model organisms&#226;&#8364;&#8221;and core reference set                (Jordan et al. 2002), expect orthologs S. cerevisiae <br /> known essential genes&#226;&#8364;&#8221;our approach requires once:                  essential genes exist&#226;&#8364;&#8221;and essential&#226;&#8364;&#8221;in S. mikatae. <br /> to initially identify relevant features train classifier.             Thus, homology mapping BLAST (Altschul et al. 1990) should <br /> Thereafter, information necessary classify gene               effective method identification essential genes in <br /> drawn sequence gene itself, freeing              S. mikatae strong standard evaluate our <br /> constant demand closely related, well-studied organisms                predictive success. Eight predictions experimentally <br /> chaperone predictions.                                                    tested vivo <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> mutagenesis, yielding ex- <br />       Broadly, genomic features divided groups:             perimental verification heterogenomic essentiality prediction. <br /> those intrinsic gene&#226;&#8364;&#8482;s sequence (e.g., GC content, <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>), <br /> those derived sequence (such localization signals and <br /> codon adaptation measures), experimental functional ge-                   Results <br /> nomics data. <br />                                                                               S. cerevisiae S. mikatae display similar essentiality profiles <br />       Certain features shown correlate with <br />                                                                               BLAST comparison <br /> gene essentiality. For example, protein&#226;&#8364;&#8220;protein interaction hubs <br /> are likely essential less-connected nodes (Dezso              A data file S. cerevisiae open reading frames (ORFs) anno- <br /> et al. 2003; Jeong et al. 2003; Yu et al. 2004). Although                tated include known essential genes, determined experi- <br /> functional genomics data mRNA expression protein inter-                 mentally Saccharomyces Genome Deletion Consortium <br /> action data useful characterizing essential genes,          (Winzeler et al. 1999; Giaever et al. 2002). Dubious ORFs and <br /> not sensible train classifier expect input                  pseudogenes excluded list. Within set 5888 <br /> no large-scale experiments organisms                 S. cerevisiae ORFs considered, 1030 (17.5%) essential, 339 <br /> interest. A classifier considers sequence-derived features          (5.8%) successfully deleted marked unknown, <br /> is best suited predicting essential genes unstudied organ-              remaining 4519 (76.7%) marked non-essential <br /> isms.                                                                         (Table 1). <br />       Our 14 features selected accessibility ge-                The S. mikatae genome contains 7047 putative ORFs, of <br /> nomic sequence data proposed link essentiality. We               subset 3939 ultimately usable predictive <br /> hypothesize translational stalling minimized essential             technique (see Methods). The complete set S. mikatae ORF <br /> genes&#226;&#8364;&#8221;if true, manifest preferential codon usage               sequences queried S. cerevisiae list described <br /> and paucity rare amino acids coding sequence. Fur-                above, E-value reporting threshold 0.0001 (1e-4). Us- <br /> ther, propose essential genes insulated non-              ing sequence homology criteria guide, S. mikatae exhibits a <br /> sense mutations likely contain in-frame             similar essentiality profile S. cerevisiae, 17.4% ORFs <br /> codons sequentially similar stop codons. Because subcellular               homologous known essential genes S. cerevisiae. Unique S. <br /> localization relevant essentiality, consider localization            mikatae ORFs (those BLAST hit S. cerevisiae this <br /> signals predicted transmembrane helices. We supplement                    E-value threshold) number 695 (9.9%). <br /> these easily accessible sequence features: GC content, gene                    Within subset 3939 S. mikatae ORFs ultimately ame- <br /> length, hydrophobicity. All features listed derived                nable classification system, putative essential genes are <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> sequence data (Drawid Gerstein 2000; Lu et al. 2004;                 enriched, 19.9% ORFs homologous known essential <br />  <br />  <br /> Table 1. Homology comparison S. mikatae S. cerevisiae <br />                                                                             Essential     Non-essential    Unknown         No Hit          Total <br />  <br /> Essentiality profile, S. cerevisiae known essential genesa                1030 (17.5%)    4519 (76.7%)     339 (5.8%)       N/A        5888 (100%) <br /> Full S. mikatae ORF set, BLAST S. cerevisiae essential genesb     1226 (17.4%)    4828 (68.5%)     298 (4.2%)    695 (9.9%)    7047 (100%) <br /> Usable S. mikatae ORF set, BLAST S. cerevisiae essential genesc    786 (19.9%)    2886 (73.3%)     190 (4.8%)     77 (2.0%)    3939 (100%) <br />  <br /> a <br />  Distribution essential genes S. cerevisiae ORF set considered work. <br /> b <br />  Results sequence homology queries S. mikatae putative ORF catalog (7047 ORFs) known essential genes S. cerevisiae. <br /> c <br />  Results sequence homology queries subset S. mikatae putative ORFs (3939 ORFs) usable predictions known essential genes <br /> in S. cerevisiae. Among subset, ORFs homologous essential genes S. cerevisiae enriched. <br />  <br />  <br />                                                                                                                   Genome Research            1127 <br />                                                                                                                      www.genome.org <br />  Seringhaus et al. <br />  <br />  <br /> genes S. cerevisiae. The number unique      Table 2.   Summary 14 genomic features training matrix <br /> ORFs (no BLAST hit S. cerevisiae 1e-4) is <br />                                                                                                 Description                             Type <br /> sharply reduced subset (2.0%). <br />                                                   LOC_mitochondria      predicted subcellular localization: mitochondria               binary <br /> Identifying features related essentiality      LOC_cytoplasm         predicted subcellular localization: cytoplasm                  binary <br />                                                   LOC_er                predicted subcellular localization: endoplasmic reticulum      binary <br /> We compiled list 14 features available       LOC_nucleus           predicted subcellular localization: nucleus                    binary <br /> from gene sequence hypothesize         LOC_vacuole           predicted subcellular localization: vacuole                    binary <br /> be related essentiality (Table 2). We pre-     LOC_other             predicted subcellular localization: compartment      binary <br /> dicted subcellular localization motifs using      Hydro                 hydrophobicity score                                           real <br />                                                   TM_HELIX              number predicted transmembrane helices (TMHMM 2.0)          integer <br /> the Proteome Analyst Specialized Subcellu-        CAI                   codon adaptation index                                         real <br /> lar Localization Server v2.5 (Lu et al. 2004)     Nc                    effective number codons                                     real <br /> and transmembrane helices ORF            GC                    GC content                                                     real <br /> using TMHMM v2.0 (Sonnhammer et al.               L_aa                  length putative protein amino acids                      integer <br />                                                   CLOSE_STOP_RATIO      percentage codons third-<span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span> away stop codon     real <br /> 1998; Krogh et al. 2001). Using               RARE_AA_RATIO         percentage rare amino acids translated ORF               real <br /> CodonW program (http://bioweb.pasteu- <br /> r.fr/seqanal/interfaces/codonw.html), Subcellular localization motifs extrapolated sequence data. Beginning 5888 ORFs, we <br /> computed measures codon adapta- included records 14 features available. The resulting training <br /> tion: effective number codons (Nc) set matrix 4648 lines (ORFs) &#226;&#179;&#175; 15 columns (14 features + essentiality target). <br /> (Wright 1990; Fuglsang 2004), the <br /> codon adaptation index (CAI) (Sharp Li <br /> 1987). We calculated frequency rare amino acids         classification scheme generates separate probability esti- <br /> number &#226;&#8364;&#339;close-to-stop&#226;&#8364;? codons (those single third-      mate essentiality. These estimates combined an <br /> base substitution removed stop codon). Finally, com-         unweighted average generate final essentiality prediction <br /> puted GC content, length translated protein prod-       ORF. <br /> uct, predicted protein hydrophobicity, CodonW.                Ten-fold cross-validation S. cerevisiae (4648-gene training <br />        To train classifier, 14 features compiled        set), average probability threshold set 0.5, yielded 88 true <br /> where available ORF S. cerevisiae (all features available  positives (TP) 875 false negatives (FN) total 963 <br /> for 4648 ORFs total) annotated known essentiality val-        actual essential; 39 false positives (FP) 3646 true negatives <br /> ues Saccharomyces Genome Database, creating           (TN) total 3685 non-essential. <br /> our training data set.                                                       Positive predictive value (PPV) measures genes <br />                                                                        predicted essential essential (i.e., precision). PPV is <br /> Individual genomic features contain information                  calculated TP / [TP+FP] and, thus, PPV = 0.69 case. Re- <br /> essentiality                                                           (R) measures true essential genes clas- <br />                                                                        sified correctly (i.e., sensitivity). Using values, R = 0.10 (given <br /> It important quantify information genomic <br />                                                                        TP / [TP+FN]). <br /> feature carries respect essentiality. To end, com- <br />                                                                              Because interested making small number high- <br /> puted correlation essentiality features train- <br />                                                                        value predictions, readily tolerate false negatives focus <br /> ing matrix (Fig. 1). Three features (predicted localization: ER, pre- <br />                                                                        instead minimizing false-positive predictions top- <br /> dicted localization: cytoplasm, length predicted protein) <br />                                                                        scoring predictions. Thus, demand Receiver Operating Char- <br /> had pairwise correlations essentiality signifi- <br />                                                                        acteristic (ROC) curve steep near origin. The ROC <br /> cant (P &gt; 0.05) set zero figure. Pre- <br />                                                                        curve learned classifier shown Figure 2. <br /> dicted nuclear localization showed strongest positive corre- <br /> lation essentiality, CAI showed weak positive cor- <br /> relation. The presence predicted transmembrane helices,         Predicting S. mikatae essential genes <br /> fraction close-stop codons, fraction rare amino acids <br />                                                                        To predict essential genes S. mikatae, 14 features <br /> in sequence exhibited strongest inverse correlation to <br />                                                                        comprising training data set compiled ORF. As <br /> essentiality. <br />                                                                        S. cerevisiae, subcellular localization server unable to <br />        To assess relative importance individual feature <br />                                                                        generate predictions subset genes; total, gathered <br /> as predictor essentiality, employed Naive Bayes <br />                                                                        feature set 3939 S. mikatae ORFs. The classifier trained <br /> technique (implemented using Orange software package <br />                                                                        S. cerevisiae applied data set. (Full results of <br /> [http://www.ailab.si/orange]). The results depicted nomo- <br />                                                                        essentiality prediction 3939 S. mikatae ORFs considered <br /> gram form Figure 1B, shows relative importance of <br />                                                                        Supplemental material online.) Notice that <br /> the features framework. <br />                                                                        learner directly exposed BLAST results these <br />                                                                        species direct information sequence homology. <br /> The classifier trained tested S. cerevisiae <br />                                                                        Instead, homology comparison used vivo knock- <br /> Several types classifiers trained tested using 14-     outs test predictions. <br /> feature data set S. cerevisiae order build best essenti- <br /> ality predictor organism. The best performance ob- <br />                                                                        Predicting essential genes species <br /> tained hybrid combined output diverse <br /> classification schemes including decision trees, Naive Bayes,      In addition S. mikatae, applied classifier two <br /> a logistic regression model (see Methods exact implementa-         additional yeast species, generating essential gene predictions for <br /> tion details). Given ORF associated 14 input variables,     S. bayanus Schizosaccharomyces pombe. Although un- <br />  <br />  <br /> 1128     Genome Research <br />             www.genome.org <br />                                                                                                 Predicting essential genes fungal genomes <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Stem plot correlation coefficients feature essentiality. Negative correlations shown left vertical axis, <br /> positive correlations right. Correlation coefficients filtered P value: Coefficients P &gt; 0.05 (deemed significant) set zero. <br /> (This occurred features LOC_cytoplasm, LOC_er, L_aa.) Higher-order correlations shown. (B) The nomogram Naive Bayes analysis, <br /> illustrating relative contributions predictive feature target class (essentiality). For feature, values drawn line according <br /> to influence target class: The longer line, important feature final prediction. The line gives log-odds ratio. Given <br /> a set feature states, prediction obtained performing vertical lookup log-odds ratio feature state summing log-odds <br /> ratio contributions 14 features. The higher sum, likely gene essential. <br />  <br />  <br /> able confirm predictions knockouts, report                   homologous correct &#226;&#8364;&#339;essential&#226;&#8364;? genes S. cerevisiae <br /> results Supplemental material available online (http://                    training set. Among 30 predictions, 17 (57%) match S. <br /> www.gersteinlab.org/proj/predess).                                                cerevisiae &#226;&#8364;&#339;essential&#226;&#8364;? genes. Thus, performance diminished con- <br />                                                                                   siderably training random subset genes. <br /> Examining predictive success comparison BLAST <br /> predictions                                                                       Predictions verified knockouts S. mikatae <br /> Our predictive success assessed BLAST comparison.                    We pursued PCR-directed knockout strategy explore es- <br /> Among 100 predicted essential ORFs S. mikatae, 75                      sential gene predictions further. <br /> (75%) homologous known essential genes S. cerevisiae,                        In total, ORFs selected study (Table 3; <br /> and 30 predicted essential S. mikatae, 25 (83%)                  Supplemental Table 1). This set contained Predicted Essential <br /> match S. cerevisiae essential genes.                                              ORFs (both homology prediction machine learner agree: <br />      To ensure classifier learned traits actually associated             gene essential), Predicted Non-Essential ORFs (both ho- <br /> with essentiality (as opposed quirks random subset                 mology machine learner agree: gene non-essential), and <br /> genes), performed permutation test (Edgington 1995). The                     Disputed Essential ORFs (machine learner predicts essential; <br /> &#226;&#8364;&#339;essential&#226;&#8364;? <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> randomly reassigned S. cerevisiae                     homology predicts non-essential). Results S. mikatae <br /> ORFs; new learner trained spurious &#226;&#8364;&#339;essentiality&#226;&#8364;?                 knockouts shown Figures 3 4. Table 4 reports the <br /> data, applied S. mikatae. Among 100 highest-                       Yeast Proteome Database (YPD) ORF designation, alias, de- <br /> scoring predicted essential ORFs S. mikatae, 45 (45%)                 scription closest S. cerevisiae homologs genes. <br />  <br />  <br />                                                                                                                            Genome Research              1129 <br />                                                                                                                               www.genome.org <br />  Seringhaus et al. <br />  <br />  <br />                                                                                    standard. Conceptually, expect approach portable <br />                                                                                    unstudied species, provided necessary features <br />                                                                                    derived organism question. <br />  <br />                                                                                    The genomic features correlations <br />                                                                                    Because aim predict essential genes largely unstud- <br />                                                                                    ied organisms, selected sequence-derived features available de <br />                                                                                    novo freshly sequenced genomes. (After all, useful to <br />                                                                                    train classifier experimental genomic data data <br />                                                                                    exist genes actually care classify.) Analysis of <br />                                                                                    correlations genomic features essentiality <br />                                                                                    suggests single feature set sufficient act alone <br />                                                                                    classifier essentiality. However, features that <br />                                                                                    display direct correlation essentiality prove useful. The <br />                                                                                    correlation coefficients shown Figure 1 calculated pair- <br />                                                                                    wise, higher-order correlations explicitly analyzed; <br />                                                                                    success training classifier demonstrates non-linear <br />                                                                                    combination features&#226;&#8364;&#8221;each having small correlation <br /> Figure 2. Receiver Operating Characteristic curve classifier            target variable&#226;&#8364;&#8221;is predictive essentiality. The <br /> the S. cerevisiae data set. This plots true positive rate versus false     nomogram (Fig. 2) illustrates relative importance these <br /> positive rate different thresholds classifier probability output. (Inset)   features simple Naive Bayes classifier. Here, features common <br /> The steep slope near origin signifies high percentage of <br /> correct essentiality assignments probability scores output <br />                                                                                    ORFs (close-stop ratio, GC content, hydrophobicity, gene <br /> by classifier. Because aim predict small subset essential           length, rare amino acid ratio) tend rank higher impor- <br /> genes high confidence, predictions subset          tance features ORFs (i.e., transmem- <br /> relevant approach.                                                          brane helices localization signals specific subcellular com- <br />                                                                                    partment). <br />       The Predicted Essential ORFs taken                             Some general trends visible features. Essential <br /> the classifier assigned highest essen-                  genes likely nuclear-localized, unlikely contain close- <br /> tiality scores (ORF 20,026, score = 0.997 ORF 20,713,                          stop codons, predicted transmembrane helices rare amino ac- <br /> score = 0.986) homologous                        ids (see Methods), tend preferential codon bias. <br /> known essential genes S. cerevisiae. (A knockout strain could <br /> not generated ORF 18,373 sequence/contig                         Our classifier: Caveats <br /> constraints.) Classifier predictions Predicted Essential                  Our results relationship exists 14 ge- <br /> genes correct: deletions &#226;&#338;&#172;20026 &#226;&#338;&#172;20713 non-                          nomic features essentiality, relationship be <br /> viable (Fig. 3A)                                                                   learned machine learning techniques. However, stress <br />       Putative ORFs 5507 18,487 chosen Predicted                       particular machine learning approach represent <br /> Non-Essential, selected pool lowest-scoring predic-                    optimal solution problem; determine best ap- <br /> tions overall homologous non-                        proach, applicable machine learning algorithms be <br /> essential genes S. cerevisiae. Deletions ORFs                assayed. <br /> viable, displaying wild-type growth. (Fig. 3B)                                           Because learner correlates sequence features target <br />       Four Disputed Essential genes selected small                   variable subset yeast genes&#226;&#8364;&#8221;specifically, fifth of <br /> subset top-scoring putative essential genes BLAST                     genes defined essential&#226;&#8364;&#8221;the risk exists learning <br /> comparison contradicted classifier. There                       features truly meaningful essentiality se, instead <br /> ORFs 30 predictions,                      merely designed potent identify quirks ge- <br /> predictions. Knockouts generated Dis-                  nomic subset exposed. Thus, control, ran- <br /> puted Essential ORFs. Although true lethality,                     domly shuffled essential label genes retrained our <br /> display growth rates significantly slower wild type (dou-                     classifier mock data. The resulting classification was <br /> bling time &#226;&#710;&#188;76 min). Specifically, &#226;&#338;&#172;3749 (94 min), &#226;&#338;&#172;7883 (83                       considerably effective classifier trained true <br /> min), &#226;&#338;&#172;6448 (135 min) exhibit varying degrees growth                    essential genes. This suggests learned <br /> arrest. The growth curves S. mikatae clones agreed                   true essential subset equally large set selected at <br /> with observed colony sizes. The fourth deletion strain (&#226;&#338;&#172;644)                  random, classifier learning traits charac- <br /> demonstrated near wild-type growth (Fig. 4).                                       teristic essential genes. <br />                                                                                          The S. pombe genome predicted contain 865 essential <br /> Discussion                                                                         genes. A recent gene-deletion pilot study assayed 80 genes drawn <br />                                                                                    single 253-kb region identified 14 essential (Decot- <br /> Our machine learning classifier able effectively identify                    tignies et al. 2003). To explore predictive success outside the <br /> genes necessary growth S. mikatae, considering input                     Saccharomyces clade, compared 14 known essential <br /> only sequenced-derived features; competitive                    genes predictions S. pombe. Our classifier designed to <br /> homology-based prediction, 25 30 predictions                    generate small number high-value positive predictions, at <br /> S. mikatae homologous essential genes S. cerevi-                 expense large number false-negative predictions. Thus, <br /> siae. Among closely related species, homology sets high                     assessing performance, makes sense evaluate its <br />  <br />  <br /> 1130       Genome Research <br />               www.genome.org <br />                                                                                            Predicting essential genes fungal genomes <br />  <br />  <br /> Table 3. Essential gene predictions S. mikatae <br />                                                                       BLAST vs. S. cerevisiae essential genesc <br />              S. mikatae       Score          Homolog in <br /> Rank          ORF_IDa         (0&#226;&#8364;&#8220;1)b         S. cerevisiae         1.00E-04           1.00E-20           1.00E-50                     Type <br />  <br />     1         18373           1.000         Multiple                   1                  1                  1             Predicted Essential <br />     2         20026           0.997         YOR046C                    1                  1                  1             Predicted Essential <br />     3         20713           0.986         YOR341W                    1                  1                  1             Predicted Essential <br />     4         10758           0.981         YIL126W                    1                  1                  1             Predicted Essential <br />     5          3115           0.974         YDL031W                    1                  1                  1             Predicted Essential <br />     6         12016           0.964         YJL050W                    1                  1                  1             Predicted Essential <br />     7           911           0.964         YBR088C                    1                  1                  1             Predicted Essential <br />     8         22659           0.962         Multiple                   1                  1                  1             Predicted Essential <br />     9         17592           0.957         YMR308C                    1                  1                  1             Predicted Essential <br />   10           3944           0.953         YDR190C                    1                  1                  1             Predicted Essential <br />   11           3749           0.946         YDR101C                    0                  0                  0             Disputed <br />   12          11901           0.944         YJL080C                    0                  0                  0             Disputed <br />   13          14387           0.944         YLL004W                    1                  1                  0             Predicted Essential <br />   14          19319           0.944         YOL010W                    1                  1                  1             Predicted Essential <br />   15           2919           0.944         Multiple                   1                  1                  1             Predicted Essential <br />   16          21382           0.943         YPL235W                    1                  1                  1             Predicted Essential <br />   17           4885           0.941         YBR247C                    1                  1                  1             Predicted Essential <br />   18          20238           0.936         YOR117W                    1                  1                  1             Predicted Essential <br />   19          20242           0.936         YOR116C                    1                  1                  1             Predicted Essential <br />   20            644           0.936         YBR158W                    0                  0                  0             Disputed <br />   21            788           0.936         YBR119W                    0                  0                  0             Disputed <br />   22           7883           0.936         YGL078C                    0                  0                  0             Disputed <br />   23          17520           0.934         YMR290C                    1                  1                  1             Predicted Essential <br />   24           8795           0.934         YGR145W                    1                  1                  1             Predicted Essential <br />   25           1400           0.932         YBL023C                    1                  1                  1             Predicted Essential <br />   26          19369           0.932         YOL021C                    1                  1                  1             Predicted Essential <br />   27          13994           0.931         YKR081C                    1                  1                  1             Predicted Essential <br />   28          21002           0.929         YOR259C                    1                  1                  1             Predicted Essential <br />   29          21414           0.929         YPL228W                    1                  1                  1             Predicted Essential <br />   30          21484           0.929         YPL211W                    1                  1                  1             Predicted Essential <br />  ...           ...             ...               ...                  ...                ...                ...                        ... <br /> 3726           5507           0.198         YDR525W-A                  0                  0                  0             Predicted Non-Essential <br />  ...           ...             ...               ...                  ...                ...                ...                        ... <br /> 3751          18487           0.193         YNL101W                    0                  0                  0             Predicted Non-Essential <br />  ...           ...             ...               ...                  ...                ...                ...                        ... <br /> 3939           2579           0.128         YDL173W                    0                  0                  0             Predicted Non-Essential <br />  <br /> a <br />   S. mikatae ORF ID listed SGD. <br /> b <br />   ORF essentiality scores assigned classifier. The probability averages output classifier normalized 0&#226;&#8364;&#8220;1 range, 1 corre- <br /> sponding strongest prediction essentiality. <br /> c <br />  Score 0 indicates ORF matches nonessential gene best hit, score 1 indicates match essential gene. <br /> The shaded bold lines denote ORFs selected knockout experiments. <br />  <br />  <br /> top &#226;&#710;&#188;30 predictions. Unfortunately, 30 essential gene                 vivo knockout mutagenesis; deletion strains <br /> predictions S. pombe contained overlap 80 genes                  proved non-viable. Conversely, low-scoring ORFs, predicted <br /> assayed aforementioned study, consequently                  non-essential, displayed wild-type growth deleted <br /> unable use data verify predictions.                           vivo. <br />  <br /> Success predictions&#226;&#8364;&#8221;homology, knockouts                                Disputed Essential knockouts <br /> In drug discovery terms, promising targets valu-               We took particular genes 30: those <br /> able hundreds questionable leads. We elected focus              classifier predicted essentiality BLAST compari- <br /> effectively identifying handful promising essential genes                son S. cerevisiae revealed homology non-essential gene. <br /> with high degree confidence, optimized learning               We examined Disputed Essential ORFs knockout <br /> penalize false-positive predictions. Our classifier in-          mutagenesis. <br /> tentionally skewed producing small number reliable,                    All Disputed Essential ORF deletion strains viable, <br /> high-value predictions.                                                       displayed varying degrees growth arrest. While <br />      This approach proved successful. Our 10&#226;&#8364;&#8221;and 25               sample size knockout pool insufficient draw reli- <br /> our 30&#226;&#8364;&#8221;S. mikatae predictions corroborated homology                able statistical conclusions, strains enriched in <br /> mapping known essential genes S. cerevisiae. Despite radi-              slow-growth phenotypes. According SGD deletion data (Win- <br /> cally different approaches, predictions generated sys-             zeler et al. 1999; Giaever et al. 2002), exclude non-viable <br /> tem agree homology-based prediction overall.                        deletions, 16% genes (755 4694 total) display slow- <br />      We performed experimental gene deletion assays S. mi-                 growth phenotypes. Thus, selecting viable knockouts random, <br /> katae interrogate predictions further. Two representative              expect roughly (&#226;&#710;&#188;17%) exhibit growth arrest. Our <br /> high-scoring predicted-essential genes selected assay             results (75%) severely impaired growth. <br />  <br />  <br />                                                                                                                     Genome Research             1131 <br />                                                                                                                        www.genome.org <br />  Seringhaus et al. <br />  <br />  <br />                                                                               dictor frustrated divergent nucleotide sequence this <br />                                                                               gene, capable generating educated guess based <br />                                                                               secondary characteristics. This hold true long these <br />                                                                               secondary features adequately determined sequence <br />                                                                               cues learned features continues to <br />                                                                               make sense target organism. <br />  <br />                                                                               Conclusions <br />                                                                               The ability consistently identify essential genes great <br />                                                                               value drug discovery operations. Experimentally determining <br />                                                                               essential genes challenging requires organism in <br />                                                                               question amenable growth culture receptive stan- <br />                                                                               dard molecular biological techniques. To limit essential gene dis- <br />                                                                               covery species cooperative laboratory study not <br />                                                                               ideal; indeed, strongest demand exists area micro- <br />                                                                               bial drug lead identification, area rich unstudied genomes <br /> Figure 3. Observed colony sizes knockout strains S. mikatae, <br />                                                                               novel pathogens. <br /> spotted duplicate. ORFs deleted PCR transformed into <br /> diploids. Sporulation induced tetrads dissected recover hap-             We demonstrate possible learn traits associated <br /> loid cells carrying deletion interest. Each column represents dif-   essential genes yeast species use features in <br /> ferent deletion strain; spots represent colonies grown       predictive manner. Our approach shows promise for <br /> deletion strains, spots wild-type colonies. Four     identification drug targets novel pathogenic spe- <br /> ORFs selected knockout mutagenesis: (A) high-scoring ORFs <br /> representing essentiality predictions; deletions predicted es-     cies. Future work involve continued study feature sets: For <br /> sential ORFs non-viable; (B) low-scoring ORFs likely represent    instance, functional genomics information available a <br /> non-essential genes; deletions predicted non-essential ORFs displayed      given species, does inclusion training data noticeably <br /> wild-type growth. (ORF) S. mikatae ORF deleted strain. (&#226;&#338;&#172;) Growth     improve predictive power? Broader applications system <br /> of knockout strain (deletion ORF listed above). (WT) Growth wild- <br /> type strain. (% dia) Percent mutant colony diameter compared          include bacterial classifier trained amalgam of <br /> WT slow growth.                                                bacterial essential genes, retraining higher <br />                                                                               eukaryotes available essentiality data Caenorhabditis <br /> This suggests predictor, directly contradicted             elegans Drosophila melanogaster. Our inherently <br /> by homology comparison, nonetheless identifying genes im-                  flexible, capable integrating available experimental se- <br /> portant cellular function.                                                 quence-derived data. As stand-alone approach, machine <br />      Table 4 reports closest S. cerevisiae homolog           learning classifier effective tool predicting essential <br /> genes knocked S. mikatae. Among these, machine                     genes. <br /> learner classified DEAD-box helicases essential (S. mikatae <br /> ORFs 20,026 7883), supported <br />                                                                               Methods <br /> by homology comparison. Looking Disputed Essential <br /> genes, interesting note words &#226;&#8364;&#339;essential&#226;&#8364;? &#226;&#8364;&#339;re-           Preparing fungal genomes study <br /> quired&#226;&#8364;? occur SGD descriptions (11901, 644) al- <br />                                                                               We used S. cerevisiae coding ORF list (orf_coding.fasta.gz, revi- <br /> though deletion strain actually inviable S. mikatae. <br />                                                                               sion 12/16/2004, available ftp://genome-ftp.stanford.edu/pub/ <br /> Anecdotally, machine learner said identify genes <br />                                                                               yeast/data_download/sequence/genomic_sequence/orf_dna/). <br /> with good likelihood essential, predic- <br /> tion disagrees experimental results extremely close <br /> sequence homologs S. cerevisiae, homology mapping <br /> trumps predictions. <br />  <br /> Portability unique genes <br /> To assay performance classifier large numbers of <br /> unique genes&#226;&#8364;&#8221;those known homolog exists&#226;&#8364;&#8221;we <br /> would need apply distantly related species. As dis- <br /> tant species considered, homology mapping our <br /> classifier likely suffer: BLAST lose efficacy more <br /> divergent sequences provide sequence traction identify <br /> essential orthologs, classifier, learned <br /> in training species decay distant relatives. Nonethe- <br /> less, believe predicting essentiality an <br /> amalgamation secondary characteristics prove ro- <br /> bust portable relying strictly comparison                  Figure 4. Four additional ORFs selected knockout mutagen- <br /> nucleotide sequence.                                                          esis examine small subset cases predictions <br />                                                                               generated disagreed homology comparison S. <br />       Consider hypothetical novel gene, unrelated                    cerevisiae (Disputed Essential ORFs). All deletions viable; three <br /> known gene sequence displaying hallmark feature-               showed significant degrees growth arrest. Experimental details the <br /> states associated essentiality. While homology-based pre-              stated Figure 3. <br />  <br />  <br /> 1132      Genome Research <br />              www.genome.org <br />                                                                                          Predicting essential genes fungal genomes <br />  <br />  <br /> This excludes 5&#226;&#172;&#732;-UTR, 3&#226;&#172;&#732;-UTR, intron sequences, bases               fr/seqanal/interfaces/codonw.html) used evaluate Kyte <br /> translated translational frameshifting. Pseudogenes              Doolittle&#226;&#8364;&#8482;s grand average hydropathicity (GRAVY) (Kyte <br /> and dubious ORFs excluded, yielding 5888 total ORFs.                   Doolittle 1982) &#226;&#8364;&#339;Hydro&#226;&#8364;?, protein length <br />      We used S. mikatae genomic ORF list (orf_genomic.fasta.            (amino acids), GC content, measures codon usage: <br /> gz, revision 12/15/2004, available ftp://genome-ftp.stanford.            effective Nc (Wright 1990; Fuglsang 2004) CAI (Sharp Li <br /> edu/pub/yeast/data_download/sequence/fungal_genomes/                        1987). (Nc calculated 4 ORFs.) Transmembrane <br /> S_mikatae/MIT/orf_dna/). This includes protein-coding re-               helices predicted hidden Markov model techniques with <br /> gion ORFs identified previously (Kellis et al. 2003)             TMHMM v2.0 Web server (http://www.cbs.dtu.dk/services/ <br /> excludes UTR sequences. We removed putative ORFs                   TMHMM/) (Sonnhammer et al. 1998; Krogh et al. 2001). <br /> under 150 nucleotides length, ORF begin-                   We calculated ratio rare amino acids sequence <br /> ning start codon (ATG) terminating stop codon              counting occurrence Cys, Trp, His, Met codons <br /> (TAA/TAG/TGA). This removed 2010 ORFs, yielding 7047 total                  expressed total ratio total ORF length. Thus, an <br /> ORFs.                                                                       ORF 200 codons long, translated sequence contains 3 <br />                                                                             Met, 2 Trp, 1 His, total RARE_AA_RATIO be <br /> Comparative genomics                                                        (3 + 2 + 1 = 6) / 200 = 0.03. <br />                                                                                  We calculated CLOSE STOP RATIO, number of <br /> We continued working S. cerevisiae ORF list detailed               codons third-base <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> removed stop <br /> above. For convenience, FASTA headers ORF list             codon. There codons: TAC TAT encoding Tyr <br /> annotated include essentiality values ORF, drawn            close TAA TAG; TGC TGT encoding Cys, TGG <br /> from Saccharomyces Genome Deletion project (Winzeler et al.             encoding Trp third-base mutation away TGA. <br /> 1999; Giaever et al. 2002). This ORF list formatted           Such codons counted ratio calculated with <br /> BLAST database. Performing homology search sequence                RARE_AA_RATIO above. <br /> against database yields results essentiality an-               Subcellular localization values predicted se- <br /> notated directly BLAST output; facilitates identifica-          quence data using Proteome Analyst Specialized Subcellular <br /> tion querying ORFs hit essential           Localization Server v2.5 (Lu et al. 2004). (The server unable <br /> gene.                                                                       generate localization predictions 1240 ORFs; removing <br />                                                                             reduced final number usable ORF records this <br /> S. cerevisiae training data set                                             array 4648.) <br /> To train classifier, training data set generated compris- <br /> ing 14 sequenced-derived features (Table 2). We compiled re- <br /> cord S. cerevisiae ORF 14 features          Compiling features S. mikatae <br /> be calculated. The targets learning given SGD           The 14 sequence-based features comprising S. cerevisiae <br /> experimentally determined essentiality result binary form                training data set compiled S. mikatae ORF. As with <br /> (0 = non-essential, 1 = essential). Because sought train          S. cerevisiae, Specialized Subcellular Localization Server (Lu et <br /> classifier ORFs essentiality clearly defined, ex-           al. 2004) unable generate localization predictions a <br /> cluded 283 ORFs essentiality unknown,              subset ORF list: 3098 ORFs excluded. Nc could <br /> also excluded 19 questionable ORFs (qORFs) clearly labeled           calculated additional 10 ORFs. This reduced the <br /> the ORF catalog. The CodonW program (http://bioweb.pasteur.                 final count usable ORFs S. mikatae 3939. <br />  <br />  <br /> Table 4. Closest S. cerevisiae homologs S. mikatae knockout genes <br />  <br />                           S. mikatae <br />          Type                ORF#                                                 S. cerevisiae homologa <br />  <br /> Predicted Essential         20026           YOR046C               Cytoplasmic ATP-dependent RNA helicase DEAD-box family involved in <br />                                               DBP5, RAT8            mRNA export nucleus <br />                             20713           YOR341W               RNA polymerase I subunit; largest subunit RNA polymerase I <br />                                               RPA190, RRN1 <br /> Predicted Non-Essential      5507           YDR525W-A             Protein unknown function, similarity Pmp3p, involved cation <br />                                               SNA2                  transport; green fluorescent protein (GFP)-fusion protein localizes the <br />                                                                     cytoplasm punctate <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                             18487           YNL101W               Vacuolar transporter, exports large neutral amino acids vacuole; member <br />                                               AVT4                  family seven S. cerevisiae genes (AVT1&#226;&#8364;&#8220;7) related vesicular <br />                                                                     GABA-glycine transporters <br /> Disputed Essential           3749b          YDR101C               Protein associated ribosomal export complex <br />                                               ARX1 <br />                                         b <br />                             11901           YJL080C               Essential RNA-binding G protein effector mating response pathway, <br />                                               SCP160                predominantly associated nuclear envelope ER, interacts in <br />                                                                     mRNA-dependent manner translating ribosomes multiple KH domains, <br />                                                                     similar vertebrate vigilins <br />                                   644       YBR158W               Protein required daughter cell separation, multiple mitotic checkpoints, and <br />                                               AMN1, ICS4, CST13     chromosome stability; contains 12 degenerate leucine-rich repeat motifs; <br />                                                                     expression induced Mitotic Exit Network (MEN) <br />                              7883b          YGL078C               Putative ATP-dependent RNA helicase DEAD-box family involved in <br />                                               DBP3                  ribosomal biogenesis <br />  <br /> a <br /> YPD ORF designations, alias(es), description (from SGD) closest S. cerevisiae homolog. <br /> b <br /> The deletion strain ORF displayed slow growth. <br />  <br />  <br />                                                                                                                  Genome Research            1133 <br />                                                                                                                     www.genome.org <br />  Seringhaus et al. <br />  <br />  <br /> Correlation features essentiality                                 PCR using primer internal KanMX cassette (KanB) (Long- <br />                                                                         tine et al. 1998) addition gene specific primer designed to <br /> To assess relative importance individual feature a <br />                                                                         &#226;&#710;&#188;200 bp upstream start codon particular ORF. <br /> predictor essentiality, employed distinct measures of <br />                                                                               Confirmed heterozygous deletion strains sporu- <br /> how feature related target column, SGD essentiality: <br />                                                                         lated according standard S. cerevisiae procedures (Winzeler <br /> correlation coefficient, information gain, information gain <br />                                                                         et al. 1999). Sixteen individual tetrads dissected YPAD <br /> ratio. These measures produced highly similar results, and <br />                                                                         plates growth scored photographed incubating for <br /> here report values correlation coefficients. Our <br />                                                                         40 h 48 h 30&#194;&#176;C. These plates replica plated onto <br /> Naive Bayes classifier, employed visualize relative contri- <br />                                                                         YPD plates containing 200 mg/mL G418 check cor- <br /> butions 14 features simple learner, implemented <br />                                                                         rect 2:2 segregation deletion marker associate the <br /> with Orange software package (http://www.ailab.si/orange), <br />                                                                         slow growth phenotype presence G418 resistance in <br /> and schema available Supplemental material avail- <br />                                                                         strains showed growth homozygous wild- <br /> able online (http://www.gersteinlab.org/proj/predess/). <br />                                                                         type deletion progeny confirm absence G418 <br />                                                                         resistance progeny deletion strains displayed ho- <br /> Classifier design <br />                                                                         mozygous lethality. <br /> Our efforts focused obtaining make                For strains showed decrease growth rate, this <br /> a high-value essential gene predictions. The objective       quantitated measuring average (n = 8) colony <br /> learning reduce number false positives,         size compared wild type following initial tetrad dissec- <br /> rather minimizing total number misclassifications        tion incubation performing growth tests these <br /> the test set. To achieve this, assigned higher cost false       strains liquid YPAD. For this, strains inoculated YPAD <br /> positives false negatives.                                      optical density (OD600) recorded hourly. The final <br />       Because training (S. cerevisiae) testing (S. mikatae)     growth rate expressed average doubling time (n = 4) of <br /> sets samples drawn distribution,        exponentially growing culture of strains. <br /> somewhat different characteristics. Thus, elected avoid be- <br /> ing overly aggressive learning positives S. cerevisiae   Predictions species <br /> chose instead impose relatively small bias false posi- <br />                                                                         The 14 sequence-based features compiled every <br /> tives form slightly higher cost. False positives were <br />                                                                         ORF S. bayanus (all features available 5447 ORFs) S. <br /> assigned cost 1.1, cost false negatives set <br />                                                                         pombe (all features available 4267 ORFs). The classifier <br /> equal 1. <br />                                                                         detailed applied data files. The genome data <br />       As previously explained, models trained S. cere- <br />                                                                         files, feature sets, predictions Supple- <br /> visiae data. To make predictions S. mikatae, trained <br />                                                                         mental material Web site (http://www.gersteinlab.org/proj/ <br /> several models chose gave best results during <br />                                                                         predess/). <br /> testing S. cerevisiae. To assess performance models <br /> on S. cerevisiae data, employed 10-fold cross-validation. All <br /> models implemented using WEKA software package                 Acknowledgments <br /> (Witten Frank 2005).                                                This work supported NIH grants Mark Gerstein and <br />       The model provided best performance combined             Michael Snyder. The authors wish thank Tom Royce, Tara <br /> the output diverse classifiers using unweighted average        Gianoulis, Joel Rozowsky helpful comments the <br /> probability estimates. Seven different learners trained,       manuscript, Andrea Sboner assistance Naive <br /> the output participated average. The component           Bayes implementation. <br /> learners comprising model are: (1) decision stump boosted <br /> using Adaboost algorithm 10 iterations (Freund and <br /> Schapire 1996); (2) random forest, constituted 10 trees            References <br /> (Breiman 2001); (3) alternating decision tree 10 boosting <br />                                                                         Altschul, S.F., Gish, W., Miller, W., Myers, E.W., Lipman, D.J. 1990. <br /> iterations (Freund Mason 1999); (4) multinomial logistic              Basic local alignment search tool. J. Mol. Biol. 215: 403&#226;&#8364;&#8220;410. <br /> regression model ridge estimator (le Cessie van Hou-         Arigoni, F., Talabot, F., Peitsch, M., Edgerton, M.D., Meldrum, E., Allet, <br /> welingen 1992), ridge regularization parameter set            E., Fish, R., Jamotte, T., Curchod, M.L., Loferer, H. 1998. A <br /> to 10&#226;&#179;&#174;8; (5) zeroR rule; (6) Naive Bayes classifier; (7)          genome-based approach identification essential bacterial <br />                                                                             genes. Nat. Biotechnol. 16: 851&#226;&#8364;&#8220;856. <br /> C4.5 decision tree (Quinlan 1993).                                      Breiman, L. 2001. Random forests. Mach. Learn. 45: 5&#226;&#8364;&#8220;32. <br />                                                                         Bruccoleri, R.E., Dougherty, T.J., Davison, D.B. 1998. Concordance <br /> S. mikatae gene disruption, sporulation, lethality scoring              analysis microbial genomes. Nucleic Acids Res. 26: 4482&#226;&#8364;&#8220;4486. <br />                                                                         Chalker, A. Lunsford, R. 2002. Rational identification new <br /> Heterozygous gene deletions constructed S. mikatae strain           antibacterial drug targets essential viability using a <br /> IFO1815 (courtesy M. Johnson, Washington University, St.                 genomics-based approach. Pharmacol. Ther. 95: 1. <br /> Louis, MO) using long-primer PCR approach developed S.            Chen, K. Pachter, L. 2005. Bioinformatics whole-genome <br /> cerevisiae (Lorenz et al. 1995). Primers designed ORF          shotgun sequencing microbial communities. PLoS Comput. Biol. <br />                                                                             1: 106&#226;&#8364;&#8220;112. <br /> such contained 60 bp immediately flanking          Cole, S.T. 2002. Comparative mycobacterial genomics tool drug <br /> start stop codon followed 20 bp sequence homologous              target antigen discovery. Eur. Respir. J. Suppl. 36: 78s&#226;&#8364;&#8220;86s. <br /> the plasmid template used deletion (see Supplemental            De Backer, M.D., Nelissen, B., Logghe, M., Viaene, J., Loonen, I., <br /> Table 1 primer sequences). The plasmid pFA6a-KanMX,                 Vandoninck, S., Hoogt, R., Dewaele, S., Simons, F.A., Verhasselt, <br />                                                                             P., et al. 2001. An antisense-based functional genomics approach for <br /> which contains KanMX gene (G418R) selectable marker,               identification genes critical growth Candida albicans. Nat. <br /> was used PCR template (Longtine et al. 1998).                        Biotechnol. 19: 235&#226;&#8364;&#8220;241. <br />       PCR products transformed S. mikatae using             Decottignies, A., Sanchez-Perez, I., Nurse, P. 2003. <br /> lithium acetate (LiAc) technique (Gietz Woods 2002). Trans-             Schizosaccharomyces pombe essential genes: A pilot study. Genome Res. <br />                                                                             13: 399&#226;&#8364;&#8220;406. <br /> formants selected growth YPD plates containing 200           Dezso, Z., Oltvai, Z.N., Barabasi, A.L. 2003. Bioinformatics analysis <br /> mg/mL G418. Strains tested correct integration               experimentally determined protein complexes yeast <br />  <br />  <br /> 1134     Genome Research <br />             www.genome.org <br />                                                                                            Predicting essential genes fungal genomes <br />  <br />  <br />     Saccharomyces cerevisiae. Genome Res. 13: 2450&#226;&#8364;&#8220;2454.                      Longtine, M.S., McKenzie III, A., Demarini, D.J., Shah, N.G., Wach, A., <br /> Drawid, A. Gerstein, M. 2000. A Bayesian integrating                   Brachat, A., Philippsen, P., Pringle, J.R. 1998. Additional <br />     expression data sequence patterns localizing proteins:               modules versatile economical PCR-based gene deletion and <br />     Comprehensive application yeast genome. J. Mol. Biol.                  modification Saccharomyces cerevisiae. Yeast 14: 953&#226;&#8364;&#8220;961. <br />     301: 1059&#226;&#8364;&#8220;1075.                                                           Lorenz, M.C., Muir, R.S., Lim, E., McElver, J., Weber, S.C., Heitman, <br /> Edgington, E.S. 1995. Randomization tests. Marcel Dekker, New York.               J. 1995. Gene disruption PCR products Saccharomyces <br /> Freund, Y. Mason, L. 1999. The alternating decision tree learning             cerevisiae. Gene 158: 113&#226;&#8364;&#8220;117. <br />     algorithm. In Proceeding Sixteenth International Conference     Lu, Z., Szafron, D., Greiner, R., Lu, P., Wishart, D.S., Poulin, B., Anvik, <br />     Machine Learning, pp. 124&#226;&#8364;&#8220;133. Morgan Kaufmann, San Francisco.                J., Macdonell, C., Eisner, R. 2004. Predicting subcellular <br /> Freund, Y. Schapire, R.E. 1996. Experiments new boosting               localization proteins using machine-learned classifiers. <br />     algorithm. In Proceedings International Conference Machine          Bioinformatics 20: 547&#226;&#8364;&#8220;556. <br />     Learning, pp. 148&#226;&#8364;&#8220;156. Morgan Kaufmann, San Francisco.                    Mushegian, A.R. Koonin, E.V. 1996. A minimal gen <br /> </body></html>