<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Aug. 2002, p. 4555&#226;&#8364;&#8220;4572                                                                                Vol. 184, No. 16 <br /> 0021-9193/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.184.16.4555&#226;&#8364;&#8220;4572.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />   <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> Genetic Footprinting Antimicrobial Drug Targets: Examples <br />                  Cofactor Biosynthetic Pathways <br />              Svetlana Y. Gerdes, Michael D. Scholle, Mark D&#226;&#8364;&#8482;Souza, Axel Bernal, Mark V. Baev, <br />                    Michael Farrell, Oleg V. Kurnasov, Matthew D. Daugherty, Faika Mseeh, <br />                           Boris M. Polanuyer, John W. Campbell, Shubha Anantha, <br />                              Konstantin Y. Shatalin, Shamim A. K. Chowdhury, <br />                                Michael Y. Fonstein, Andrei L. Osterman* <br />                                             Integrated Genomics Inc., Chicago, Illinois 60612 <br />                                               Received 17 January 2002/Accepted 14 May 2002 <br />  <br />              Novel drug targets required order design new defenses antibiotic-resistant pathogens. <br />           Comparative genomics provides new opportunities finding optimal targets previously unexplored <br />           cellular functions, based understanding related biological processes bacterial pathogens their <br />           hosts. We integrated approach identification prioritization broad-spectrum drug targets. <br />           Our strategy based genetic footprinting Escherichia coli followed metabolic context analysis of <br />           essential gene orthologs various species. Genes required viability E. coli rich medium identified <br />           whole-genome scale using genetic footprinting technique. Potential target pathways deduced from <br />           data compared panel representative bacterial pathogens using metabolic reconstructions <br />           genomic data. Conserved indispensable functions revealed analysis potentially represent <br />           broad-spectrum antibacterial targets. Further target prioritization involves comparison corresponding <br />           pathways individual functions pathogens human host. The promising targets are <br />           validated direct knockouts model pathogens. The efficacy approach illustrated using examples <br />           metabolism adenylate cofactors NAD(P), coenzyme A, flavin adenine dinucleotide. Several drug <br />           targets pathways, including distantly related adenylyltransferases (orthologs E. coli <br />           genes nadD, coaD, ribF), discussed detail. <br />  <br />  <br />   The growing number antibiotic-resistant microbial patho-                  scale high-throughput random transposon mutagenesis. The <br /> gens (44, 46; http://www.cdc.gov/narms/default.htm) presents                 major advantage technique parallel analysis <br /> serious challenge modern medicine. The majority existing                 thousands genes multiple growth conditions. <br /> antibiotics utilize limited number core chemical structures                  A transposon-based approach termed &#226;&#8364;&#339;genetic footprinting&#226;&#8364;? <br /> and target cellular functions, cell wall bio-               originally described Saccharomyces cerevisiae (84, 85). <br /> synthesis, DNA replication, transcription, translation (67).               Genetic footprinting three-step process: (i) random trans- <br /> Identification unexplored cellular functions potential tar-              poson mutagenesis large number cells, (ii) competitive <br /> gets prerequisite development novel antibiotic che-                outgrowth mutagenized population various selec- <br /> motypes. Choosing optimal target function crucial step                 tive conditions, (iii) analysis individual mutants surviv- <br /> in long expensive process drug development                      ing population using direct sequencing various hy- <br /> requires best possible understanding related biological                 bridization PCR-based techniques. Various modifications <br /> processes bacterial pathogens hosts.                              genetic footprinting recently applied several <br />   Extensive programs utilizing genomic information search                   microorganisms, including Mycoplasma genitalium Myco- <br /> for novel antimicrobial targets launched recently                 plasma pneumoniae (49), Pseudomonas aeruginosa (99), Heli- <br /> industry academia (14, 15, 69, 72, 91). Complete genome                    cobacter pylori (52), Escherichia coli (7, 45). Another ver- <br /> sequences multiple bacterial species, including major                  sion method, termed genomic analysis mapping by <br /> pathogens, available years,                   vitro transposition, developed Haemophilus <br /> many projects way (16). The abundance                       influenzae (1, 75) Streptococcus pneumoniae (1). Direct <br /> genomic data enabled development novel post-                        application technique usually limited microbial <br /> genomic experimental computational techniques aimed                     species natural competence, transposon mutagene- <br /> drug target discovery. Experimental approaches genome-                      sis performed vitro isolated DNA fragments, and <br /> wide identification genes essential cell viability           mutations introduced genome transformation <br /> microbial species reviewed recently (23, 43, 54, 59,                 linear DNA fragments followed gene conversion. By <br /> 67, 77, 79). These techniques based systematic                   targeting specific genomic region, approach increases <br /> gene inactivation directed mutagenesis whole-genome                    insertion density, improving resolution genetic footprinting. <br />                                                                                An elegant extension method naturally compe- <br />   * Corresponding author. Mailing address: Integrated Genomics, Inc., <br />                                                                                tent species described P. aeruginosa (99). <br /> 2201 W. Campbell Park Dr., Chicago, IL 60612. Phone: (312) 491-0846,              Genetic footprinting E. coli mini-Tn10 mutagenesis in <br /> ext. 213. Fax: (312) 491-0856. E-mail: andrei@integratedgenomics.com.          vivo recently reported (7, 45). In systematic <br />  <br />                                                                         4555 <br />  4556      GERDES ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br /> analysis E. coli gene essentiality, chosen ap-                       Transposome        formation       transposon      mutagenesis.     Plasmid <br /> proach based use &#226;&#8364;&#339;transposome,&#226;&#8364;? precut trans-                     pMOD&#226;&#172;?MCS&#226;&#172;&#381; containing artificial transposon EZ::TN&#226;&#172;?KAN-2&#226;&#172;&#381; (Epicentre <br />                                                                                 Technologies, Madison, Wis.) isolated strain (MG1655 or <br /> poson DNA stable complex modified Tn5 transposase                     DH10B) subsequently mutagenized avoid restriction-modification <br /> (40, 47). This approach advantages mini-Tn10                   problems. Transposon DNA released PvuII digestion, recommended <br /> mutagenesis: (i) single irreversible insertions pro-                   manufacturer, gel purified using QIAquick gel extraction columns <br /> duced, source transposase activity pro-                (Qiagen, Valencia, Calif.). Transposomes preformed incubating trans- <br />                                                                                 poson DNA (7 ng/&#226;?&#174;l) hyperactive Tn5 EZ::TN transposase (0.1 U/&#226;?&#174;l; <br /> tein transposome complex; (ii) need <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                 Epicentre Technologies generous gift W. Reznikoff) solution <br /> assemble elaborate transposon delivery vector                    containing 40 mM Tris-acetate (pH 7.5), 100 mM potassium glutamate, 0.1 mM <br /> tight regulation replication transposase expression;                 EDTA, 1 mM dithiothreitol, tRNA (0.1 mg/ml). Samples incubated for <br /> (iii) limit insertion cell achieved based                30 min 37&#194;&#176;C dialyzed 10 mM Tris-acetate, pH 7.5, plus 1 mM <br /> the ratio transposome complexes competent cells                    EDTA 0.05-&#226;?&#174;m filters (Millipore, Bedford, Mass.) 1 h. Dialyzed samples <br />                                                                                 mixed electrocompetent E. coli 1:2 ratio (vol/vol) transformed <br /> time transformation.                                                         electroporation. Cultures immediately diluted Luria-Bertani- <br />    Computational techniques identification potential                 based rich medium (see below) kanamycin incubated 37&#194;&#176;C 40 <br /> drug targets based genomic data reviewed re-                       min gentle agitation. The efficiency electroporation E. coli strains <br /> cently (35, 37, 67, 79). In studies, silico analysis               MG1655 DH10B 5 &#226;&#171;&#187; 104 2 &#226;&#171;&#187; 106 kanamycin-resistant colonies per <br /> successfully combined experimental techniques (4, 23,                      1 &#226;?&#174;g transposon DNA, respectively. <br />                                                                                    Outgrowth mutagenized population. Half mutagenized population <br /> 36). For example, Arigoni coauthors used comparative                        immediately frozen stored time zero sample. The rest the <br /> genome analysis identify previously uncharacterized genes                    culture used inoculate BIOFLO 2000 fermentor (New Brunswick Sci- <br /> as potential broad-spectrum targets emphasizing genes                        entific, Edison, N.J.) containing 950 ml following medium: tryptone, 10 <br /> which (i) broadly conserved various bacteria, including                  g/liter; yeast extract, 5 g/liter; 50 mM NaCl, 9.5 mM NH4Cl, 0.528 mM MgCl2, <br />                                                                                 0.276 mM K2SO4, 0.01 mM FeSO4, 5 &#226;&#171;&#187; 10&#226;&#171;&#186;4 mM CaCl2, 1.32 mM K2HPO4. <br /> pathogens; (ii) conserved humans; (iii) likely to <br />                                                                                 This medium supplemented following micronutrients: 3 &#226;&#171;&#187; 10&#226;&#171;&#186;6 mM <br /> encode soluble proteins. The essentiality selected genes                 (NH4)6(MoO7)24, 4 &#226;&#171;&#187; 10&#226;&#171;&#186;4 mM H3BO3, 3 &#226;&#171;&#187; 10&#226;&#171;&#186;5 mM CoCl2, 10&#226;&#171;&#186;5 mM CuSO4, <br /> further assessed directed knockouts E. coli Bacillus                  8 &#226;&#171;&#187; 10&#226;&#171;&#186;5 mM MnCl2, 10&#226;&#171;&#186;5 mM ZnSO4 (68). The following vitamins were <br /> subtilis (4). Most silico target identification techniques               added (concentrations milligrams liter): biotin, 0.12; riboflavin, 0.8; <br /> focused formal comparative sequence analysis, at-                    pantothenic acid, 10.8; niacinamide, 12.0; pyridoxine, 2.8; thiamine, 4.0; lipoic <br />                                                                                 acid, 2.0; folic acid, 0.08; p-aminobenzoic acid, 1.37. Kanamycin added <br /> tempting assess conservation overall biological con-                  10 &#226;?&#174;g/ml. Throughout fermentation temperature held 37&#194;&#176;C, dis- <br /> text various pathogens human host. We believe                   solved oxygen held 30 50% saturation, pH held 6.95 <br /> comparative analysis pathways biological subsystems                      (via titration 5% H3PO4). Media growth conditions designed to <br /> may significantly improve ability select potential targets.              minimize number genes required cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>survival</span>. Cells grown in <br />                                                                                 batch culture 23 population doublings (12 h) cell density 1.4 &#226;&#171;&#187; 109. <br />    Here approach identification pri- <br />                                                                                 Genomic DNA isolated used generate genetic footprints. <br /> oritization broad-spectrum antimicrobial targets known                     Detection transposon insertions nested PCR. Two pairs primers were <br /> functional roles. We use list essential genes inferred                  used consecutively, second pair primers nested (see <br /> genetic footprinting model microorganism (E. coli)                    Fig. 2A). Each primer pair contained transposon-specific primer one <br /> starting point analysis corresponding functional roles               chromosome-specific primer. Chromosome-specific landmark primers de- <br />                                                                                 signed ordered set unidirectional primer pairs covering entire E. coli <br /> in context metabolic pathways microbial pathogens.                    genome using custom software. Pairs separated average 3,500 bp, <br /> This kind comparative analysis based metabolic recon-                     primers pair separated shortest possible distance <br /> struction genomic data indicates suitable target pathways                  range &#226;&#171;&#186;3 900 bp. Average primer <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> 27 bp (sequences available <br /> and target functions pathways,                      request). Transposon-specific primers chosen avoid significant <br />                                                                                 similarity E. coli chromosome, using PrimerSelect software (DNA- <br /> potential range pathogens target. Comparison of <br />                                                                                 STAR, Inc., Madison, Wis.). Two pairs nested, outwardly directed transposon- <br /> microbial pathways individual target proteins                    specific primers (one end) used detect transposons inserted in <br /> human counterparts allows target prioritization. Fi-                    orientations (see Fig. 2A). The forward primer pair includes external <br /> nally, prospective targets validated directed knockouts                  primer, 5&#226;&#172;&#732;-GTTCCGTGGCAAAGCAAAAGTTCAA-3&#226;&#172;&#732;, internal <br /> in model pathogens (Staphylococcus aureus H. influenzae).                    primer, 5&#226;&#172;&#732;-GGTCCACCTACAACAAAGCTCTCATCA-3&#226;&#172;&#732;. The reverse primer <br />                                                                                 pair includes external primer, 5&#226;&#172;&#732;-CCGACATTATCGCGAGCCCATTTAT- <br /> The corresponding proteins cloned representative <br />                                                                                 3&#226;&#172;&#732;, internal primer, 5&#226;&#172;&#732;-GCAAGACGTTTCCCGTTGAATATGGC-3&#226;&#172;&#732;. <br /> pathogens human cDNA libraries, expressed analyzed                         The consecutive PCR amplifications (the external PCR) was <br /> in detail. The advanced target proteins representa-                   performed following conditions: 95&#194;&#176;C 1 min; 94&#194;&#176;C 12 s, 70&#194;&#176;C for <br /> tive bacterial sources, human counterparts,                6 min (2 cycles); 94&#194;&#176;C 12 s, 69&#194;&#176;C 6 min (2 cycles); 94&#194;&#176;C 12 s, 68&#194;&#176;C for <br /> crystallized, structures determined assist                  6 min (36 cycles); 68&#194;&#176;C 6 min. Amplification reactions contained 0.3 &#226;?&#174;g of <br />                                                                                 template DNA (equivalent 6 &#226;&#171;&#187; 107 E. coli genomes), 0.2 mM concentration <br /> further drug development.                                                       deoxynucleoside triphosphate, 0.4 &#226;?&#174;M concentration primer, <br />    We illustrate approach using examples biosyn-                  PCR buffer (40 mM Tricine-KOH [pH 9.2], 15 mM potassium acetate, 3.5 mM <br /> thesis adenylate cofactors, NAD, coenzyme A (CoA),                     magnesium acetate, bovine serum albumin [3.75 &#226;?&#174;g/ml]), 0.4 &#226;?&#174;l Advan- <br /> and flavin adenine dinucleotide (FAD), presenting                       tage cDNA Polymerase Mix (Clontech Laboratories, Palo Alto, Calif.) 20 &#226;?&#174;l. <br />                                                                                 The second internal PCR performed reaction mix, the <br /> promising targets development novel broad-spec- <br />                                                                                 DNA templates consisted products PCR diluted 103-fold. <br /> trum antibiotics.                                                               Amplification conditions internal PCR follows: 95&#194;&#176;C 1 min; 94&#194;&#176;C <br />                                                                                 12 s, 69&#194;&#176;C 6 min (2 cycles); 94&#194;&#176;C 12 s, 68&#194;&#176;C 6 min (9 cycles); 68&#194;&#176;C <br />                                                                                 6 min. The products internal PCRs (3-&#226;?&#174;l aliquots) size separated <br />                        MATERIALS AND METHODS <br />                                                                                 0.65% agarose gels. All insert detection analysis procedures per- <br />   Strains. E. coli strains MG1655 (F&#226;&#171;&#186; lambda&#226;&#171;&#186; ilvG rfb50 rph1) (53) DH10B   formed 96-well format. <br /> [F&#226;&#172;&#732; mcrA &#226;&#338;&#172;(mrr-hsdRMS-msrBC) &#226;?&#190;80lacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;lacX74 endA1 recA1 deoR                  Image capture analysis performed 1D Image Analysis Software <br /> &#226;&#338;&#172;(ara-leu)7697 araD139 galU galK nupG rpsL] (42) S. aureus strain RN4220    (Eastman Kodak Company, Rochester, N.Y.). Mapping detected Tn5 <br /> (55) used work.                                                    insertions using custom software, calculates insert positions <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                                       4557 <br />  <br /> within genome sequence using addresses internal landmark primers               TABLE 1. Genetic footprinting E. coli genes related to <br /> and size corresponding PCR products. Visualization insert locations           metabolism adenylate cofactors NAD(P), CoA, FAD <br /> was using custom software integrated ERGO database. The mi- <br /> crobial genome database WIT, predecessor ERGO, described                                                                    No. inserts <br />                                                                                        Biosynthetic                      ORF size       ORF strain: Conclusionb <br /> previously (71).                                                                                          Gene <br />                                                                                         subsystem                         (bp) <br />    Optimization experimental conditions low-noise detection transposon                                                       MG1655 DH10B <br /> insertions. Reaction conditions optimized using small set control genes <br /> with known essentiality, order detect maximum number inserts,       NAD            nadB                 1,620           9        ND           N <br /> keeping level PCR-introduced noise low. First, minimum                         nadA                 1,041          12        ND           N <br /> of genomic DNA contained representative mix mutant chromo-                            nadC                   891           1         4           N <br /> somes consistently yielded reproducible patterns bands PCR                            ushA                 1,650           6        ND           N <br /> determined. Second, products external corresponding internal PCRs                          pnuC                   717           4        ND           N <br /> were analyzed side agarose gel. This comparison used                           nadR                 1,251           6         4           N <br /> guide optimizing PCR parameters: cooling rate, number cycles                            pncA                   657           2        ND           N <br /> external versus internal PCR, rate PCR products&#226;&#8364;&#8482; dilutions. PCR                            pncB                 1,200           8        ND           N <br /> conditions fine-tuned yield minimum number false products,                             nadD (ybeN)            639           1         4           Ec <br /> namely, internal PCR bands lacking corresponding external PCR product.                            nadE                   825           0         1           E <br /> Third, genetic footprints 4-kb nadD locus generated independently                           nadF (yfiB)            876           0         0           E <br /> using identical template DNA samples different chromosome-specific <br /> nested primer pairs. All footprints yielded coherent patterns, consistently       CoA            panD                   378            2        0           N <br /> visualizing 10 transposon inserts area (data shown). Finally, 12 random                    panB                   792            1        3           N <br /> internal PCR bands gel purified sequenced. All bands originated                          panE                   909            5        2           N <br /> from expected chromosomal loci. Comparison calculated insert loca-                         panC                   849            1        5           N <br /> tions sequencing results indicated mapping error introduced gel                         coaA                   948            0        0           E <br /> electrophoresis 4.5% size band.                                               coaBC (dfp)          1,290            0        0           E <br />    Metabolic reconstruction genomic sequence data. The versions ge-                           coaD (kdtB)            477            0        0           E <br /> nomes used work follows: E. coli K-12 MG1655 (18) (GenBank                             coaE (yacE)            618            0        0           E <br /> accession no. U00096), P. aeruginosa PAO1 (88) (GenBank accession no.                                 acpS                   378            0        0           E <br /> AE004091), Bacillus anthracis Ames (http://www.tigr.org/tdb/mdb/mdbinprogress <br /> .html), Mycobacterium tuberculosis H37Rv (24) (GenBank accession no. AL123456),        FMN/FAD ribA                          588            0        0           E <br /> H. pylori J99 (3) (GenBank accession no. AE001439), S. aureus COL (http://www.tigr             ribD                        1,101            0        0           E <br /> .org/tdb/mdb/mdbinprogress.html/), S. pneumoniae (GenBank accession no.                        ribB                          651            0        0           E <br /> AE005672), M. genitalium (34) (GenBank accession no. L43967), H. influenzae Rd                 ribH                          468            0        0           E <br /> (32) (GenBank accession no. L42023), Chlamydia trachomatis (87) (GenBank accession             ribE                          639            0        0           E <br /> no. AE001273), Homo sapiens (http://www.ncbi.nlm.nih.gov/genome/guide/                     ribF (yaaC)                   939            0        0           E <br /> human/).                                                                                  a <br />                                                                                             The number transposon insertions gene strains MG1655 and <br />    Comparisons specific biochemical capabilities E. coli, relevant bac-      DH10B detected logarithmic aerobic outgrowth 23 population dou- <br /> terial pathogens, human host based metabolic reconstructions           blings rich medium shown. <br /> performed ERGO database. Metabolic reconstructions ERGO                     b <br />                                                                                             Assessments gene essentiality, based results genetic footprinting in <br /> tentative projections known biochemical pathways specific organism       strains MG1655 DH10B. E, essential; N, nonessential; ND, determined. <br />                                                                                           c <br /> with completely sequenced annotated genome (described reference 82).               nadD appeared nonessential genetic footprinting experi- <br /> These projections based presence absence orthologs specific        ments shown essential directed-<span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> technique. <br /> genes known, basis studies organisms, involved in <br /> corresponding pathways. This approach produces estimate metabolic <br /> potential given organism (many asserted pathways be <br /> actually expressed specific conditions) direct experimental <br /> data falls short defining actual metabolic fluxes.                               quence, using database size 106 high-cutoff E value 104. The <br />    Examples illustrating type analysis given Tables 2 4    resulting E values indication closely bacterial sequences are <br /> metabolic subsystems (groups pathways) related biosynthesis ade-          related remote human sequence conserved core the <br /> nylate cofactors NAD(P), CoA, FAD. We define pathway series              bacterial group (the higher E value is, higher divergence is). <br /> consecutive transformations bifurcations (82). Merging and/or alterna-            Verification gene essentiality E. coli. For direct verification gene <br /> tive pathways (numbered I II, etc., Table 2) combined higher             essentiality E. coli used pKO3-based allelic exchange (58). This briefly <br /> hierarchical blocks, ultimately form subsystem. In contrast pathways,       illustrated using nadD example. Deletion gene (predicted be <br /> subsystems formally defined boundaries, assembled             essential) attempted parallel nadR (a nonessential gene the <br /> based conventions field (such cofactor biosynthesis described          metabolic subsystem) control. The E. coli nadD (12, 63) (formerly <br /> references 5, 12, 13), biological expertise interests   ybeN) nadR genes flanking regions PCR amplified using <br /> particular user.                                                                       MG1655 genomic DNA template following primers: 5&#226;&#172;&#732;-CAGCTG <br />    Profile HMMs. The comparisons human bacterial sequences                 ATTCGTAAGCTGCCAAGCATC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GGGGTCGACTCACTCACGG <br /> were using HMMER 2.2 program (30; http://hmmer.wustl.edu/),              TGATAAGGATGGTTGGTGGTGATG-3&#226;&#172;&#732; nadD 5&#226;&#172;&#732;-TGGCCTGCCG <br /> multiple-sequence alignments obtained CLUSTAL-W (90). We built            ACTGACAATCTC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GCTGGAAAACGCCCTGCTGGAGT-3&#226;&#172;&#732; for <br /> profile hidden Markov model (HMM) target enzyme, using ortholog           nadR. Both PCR products cloned gene replacement vector pKO3 <br /> sequences pathogens interest, specified Fig. 5A. HMMs            (58). Next, 650-bp HincII-ClaI fragment nadR replaced 1.7-kb <br /> the following enzymes built using full-length protein sequences            Eco57I DNA fragment containing tetracycline resistance cassette of <br /> nicotinic acid mononucleotide adenylyltransferase (NaMNAT), NAD synthetase             pACYC184 (GenBank accession no. X06403). Similarly, 434-bp ApaLIBglII <br /> (NADS), NAD kinase (NADK), pantothenate kinase (PK), phosphopantetheine                fragment nadD excised replaced kanamycin resistance cas- <br /> adenylyltransferase (PPAT), dephospho-CoA kinase (DPCK). Protein do-               sette pUC4K (GenBank accession no. X06404). E. coli strain MG1655 was <br /> mains roughly corresponding amino acid residues 1 212 213 413          transformed deletion plasmids. Following cointegration, resolution, <br /> the E. coli CoaBC protein used phosphopantothenoylcysteine de-            elimination plasmids, Kmr Sucr colonies screened sensitivity <br /> carboxylase (PPCDC) phosphopantothenoylcysteine synthetase (PPCS)                  chloramphenicol (58). Final verification gene replacement by <br /> HMMs, respectively. Likewise, protein domains approximately corresponding           PCR analysis primers flanking region recombination. <br /> amino acid residues 1 176 177 214 E. coli RibF protein used         In case nadR, gene replacement readily achieved: 98% Kmr Sucr <br /> to build profile HMMs FAD synthase (FADS) flavokinase (FK), respec-             colonies Cms, Kmr Sucr Cms colonies tested PCR <br /> tively. Obtained HMMs run bacterial human se-               contained expected deletion chromosome. In case nadD, Kmr <br />                                                                                                                                                                                                                                                                                   4558 <br />                                                          TABLE 2. Functional reconstruction metabolic pathways NAD/NADP biosynthesisa <br />                                                                                                                                                        Presence ortholog pathway genome of: <br />  <br />  <br />                                                                                          Protein <br />                     NAD(P) biosynthesis                                EC no. <br />                                                                                          example <br />  <br />  <br />  <br />  <br />                                                                                                             E. colib <br />                                                                                                                         P. aeruginosa <br />                                                                                                                                         B. anthracis <br />                                                                                                                                                          M. tuberculosis <br />                                                                                                                                                                            H. pylori <br />                                                                                                                                                                                        S. pneumoniae <br />                                                                                                                                                                                                        S. aureus <br />                                                                                                                                                                                                                    M. genitalium <br />                                                                                                                                                                                                                                    H. influenzae <br />                                                                                                                                                                                                                                                    C. trachomatis <br />                                                                                                                                                                                                                                                                     H. sapiens <br />                                                                                                                                                                                                                                                                                  GERDES ET AL. <br />  <br />  <br />  <br />  <br /> De novo pathways NaMN                                                                                  Yes          Yes              Yes             Yes                Yes         No              No          No              No              No               Yes <br />  (I) Tryptophan quinolinate                                                                            &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  (II) Aspartate quinolinate <br />     Aspartate oxidase                                                1.4.3.16         gi&#229;&#8230;&#169;16130499         nadB         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     Quinolinate synthase                                                              gi&#229;&#8230;&#169;16128718         nadA         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />  Quinolinate NaMN <br />     Quinolinate phosphoribosyl transferase                           2.4.2.19         gi&#229;&#8230;&#169;16128102         nadC         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Salvage: niacin NaMN/NMN                                                                          Yes          Yes              Yes             Yese               No          Yes             Yes         Yes             No              No               Yes <br />   (I) Nicotinamide NaMN <br />      Nicotinamide deamidase                                          3.5.1.19         gi&#229;&#8230;&#169;140602           pncA         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />      Nicotinate phosphoribosyl transferase                           2.4.2.11         gi&#229;&#8230;&#169;16128898         pncB         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />   (II) Nicotinamide NMN <br />      Nicotinamide phosphoribosyl transferase (nadVc)                 2.4.2.12         gi&#229;&#8230;&#169;13629024         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Salvage: exogenous NMN NAD                                                                        Yes          Yes              No              No                 No          No              No          No              Yes             No               No <br />     NMN phosphohydrolase (extracellular) (nadNd)                     3.1.3.5          gi&#229;&#8230;&#169;1573165          ushA         &#226;&#171;&#186;                &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />     Pyridine nucleoside transporter                                                   gi&#229;&#8230;&#169;16273640         pnuC         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     Nicotinamide ribose kinasef                                      2.7.1.22         gi&#229;&#8230;&#169;16132207         nadR         &#226;&#171;&#185;                &#226;&#171;&#186;               &#226;&#180;&#8482;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     NMN adenylyltransferase <br />        (i) NMN specific (archaea, bacteria)                     2.7.7.1          gi&#229;&#8218;&#168;16132207         nadR         &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#180;&#8482;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />        (ii) NaMN/NMN specific (eukarya)                              2.7.7.1/18       gi&#229;&#8230;&#169;11245478         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Common pathway NAD/NADP                                                                                Yes          Yes              Yes             Yes                Yes         Yes             Yes         Yes             Yes             No               Yes <br />   NaMN NAD <br />    NaMN adenylyltransferase <br />      (i) NaMN specific (most bacteria)                               2.7.7.18         gi&#229;&#8230;&#169;1723307          nadD         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />      (ii) NaMN/NMN specific (eukarya)                                2.7.7.1/18       gi&#229;&#8230;&#169;11245478         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />    NAD synthetase                                                    6.3.5.1          gi&#229;&#8230;&#169;16129694         nadE         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />   NAD NADP <br />    NAD kinase                                                        2.7.1.23         gi&#229;&#8230;&#169;8489010          nadF         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />   a <br />      Subsystems, pathways, individual functional roles (specified Enzyme Classification [EC] numbers) involved adenylate cofactor biosynthesis relevant organisms presented nodes branches (rows) <br /> of hierarchical tree. Proteins shown representative examples (GenBank ID numbers). The presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) orthologous gene specific genome marked column corresponding to <br /> this organism, E. coli, actual gene names given. Alternative pathways subsystem indicated uppercase roman numerals. Nonorthologous genes functional role shown <br /> in separate rows indicated lowercase roman numerals. Bifunctional (fused) proteins functional domain shown boldface type. &#226;&#8364;&#339;Yes&#226;&#8364;? indicates particular subsystem asserted <br /> in given organism presence required genes (enzymatic functions) genome. &#226;&#8364;&#339;No&#226;&#8364;? indicates particular subpathway asserted organism basis genomic data alone. Functional <br /> roles inferred metabolic context analysis associated specific genes (&#226;&#8364;&#339;missing genes&#226;&#8364;?) indicated question marks. <br />    b <br />      Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />    c <br />      Identified Haemophilus ducreyi; present V factor-independent Pasteurellaceae, H. influenzae (62). <br />    d <br />      Identified H. influenzae (76); closest homolog E. coli ushA. <br />    e <br />      Experimental data suggest pathway functional M. tuberculosis (83). <br />    f <br />      The corresponding function E. coli H. influenzae catalyzed C-terminal domain multifunctional protein NadR (O. Kurnasov et al., unpublished data). <br />                                                                                                                                                                                                                                                                                  J. BACTERIOL. <br />  4559 <br /> GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS <br />  <br />  <br />  <br />  <br />                                                                                                              TABLE 3. Functional reconstruction metabolic pathways CoA biosynthesisa <br />                                                                                                                                                                                                        Presence ortholog pathway genome of: <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                          M. tuberculosis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                        S. pneumoniae <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                    C. trachomatis <br />                                                                                                                                                                                                                                                                    M. genitalium <br />                                                                                                                                                                         P. aeruginosa <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                    H. influenzae <br />                                                                                                                                           Protein <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                         B. anthracis <br />                                                                         CoA biosynthesis                                 EC no. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                     H. sapiens <br />                                                                                                                                           example <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        S. aureus <br />                                                                                                                                                                                                                            H. pylori <br />                                                                                                                                                              E. colib <br />                                                  De novo pathway pantothenate                                                                           Yes          Yes             Yes             Yes                Yes         No              Yes         No              No              No               No <br />                                                   Aspartate &#226;?&#164;-alanine <br />                                                      Aspartate 1-decarboxylase                                         4.1.1.11        gi&#229;&#8230;&#169;1786323          panD         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                   Ketovalerate pantoate <br />                                                      3-Methyl-2-oxobutanoate hydroxymethyl transferase                 2.1.2.11        gi&#229;&#8230;&#169;1786326          panB         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                      2-Dehydropantoate 2-reductase <br />                                                        (i) 2-Dehydropantoate 2-reductase                               1.1.1.169       gi&#229;&#8230;&#169;1100871          panE         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                        (ii) Ketol-acid reductoisomerase                                1.1.1.86        gi&#229;&#8230;&#169;146477           ilvC         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                   &#226;?&#164;-Alanine, pantoate pantothenate <br />                                                      Pantoate-&#226;?&#164;-alanine ligase                                         6.3.2.1         gi&#229;&#8230;&#169;1786325          panC         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                  Pantothenate (vitamin B5) salvage                                                                         Yes          Yes             Yes             Yes                Yes         Yes             Yes         No              Yes             No               Yes <br />                                                      Sodium/pantothenate symporter                                                     gi&#229;&#8230;&#169;455654           panF         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                  Common pathway CoA                                                                                     Yes          Yes             Yes             Yes                Yes         Yes             Yes         No              Yes             No               Yes <br />                                                     Pantothenate kinase                                                                                                 ?f                                                 ?f <br />                                                       (i) Most bacteria                                                2.7.1.33        gi&#229;&#8230;&#169;1790409          coaA         &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                       (ii) Eukarya                                                     2.7.1.33        gi&#229;&#8230;&#169;6320740c         &#226;&#171;&#186;            &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Phosphopantothenoylcysteine synthetase                             6.3.2.5         gi&#229;&#8230;&#169;1790070          coaBe        &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#171;&#185;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Phosphopantothenoylcysteine decarboxylase                          4.1.1.36        gi&#229;&#8230;&#169;1790070          coaCe        &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#171;&#185;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Pantetheine-phosphate adenylyltransferase <br />                                                       (i) Bacteria                                                     2.7.7.3         gi&#229;&#8230;&#169;1790065          coaD         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                       (ii) Eukarya                                                     2.7.7.3         gi&#229;&#8230;&#169;632171d          &#226;&#171;&#186;            &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#180;&#8482; <br />                                                     Dephospho-CoA kinase                                               2.7.1.24        gi&#229;&#8230;&#169;1786292          coaE         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#180;&#8482; <br />                                                    a <br />                                                      See Table 2, footnote a, details. <br />                                                    b <br />                                                      Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />                                                    c <br />                                                      Eukaryotic pantothenate kinase (21), structurally unrelated corresponding E. coli enzyme (CoaA family). <br />                                                    d <br />                                                      Eukaryotic pantetheine-phosphate adenylyltransferase distantly related corresponding bacterial enzymes (beyond recognition Psi-Blast) (27); forms fusion protein dephospho-CoA kinase in <br />                                                  higher eukaryotes. <br />                                                    e <br />                                                      In bacteria (except S. pneumoniae set), CoaB C-terminal domain, CoaC N-terminal domain bifunctional protein. <br />                                                    f <br />                                                      Pantothenate kinase &#226;&#8364;&#339;missing gene&#226;&#8364;? number bacterial pathogens. <br /> VOL. 184, 2002 <br />                                                                                                                                                                                                                                                                                                        4560 <br />                                                                                                                                                                                                                                                                                                       GERDES ET AL. <br />  <br />  <br />  <br />  <br />                                                          TABLE 4. Functional reconstruction metabolic pathways FMN/FAD biosynthesisa <br />                                                                                                                                                                           Presence ortholog pathway genome of: <br />  <br />  <br />                                                                                                            Protein <br />                               FMN/FAD biosynthesis                                         EC no.          example <br />  <br />  <br />  <br />  <br />                                                                                                                             E. colib <br />                                                                                                                                            P. aeruginosa <br />                                                                                                                                                            B. anthracis <br />                                                                                                                                                                               M. tuberculosis <br />                                                                                                                                                                                                 H. pylori <br />                                                                                                                                                                                                             S. pneumoniae <br />                                                                                                                                                                                                                             S. aureus <br />                                                                                                                                                                                                                                         M. genitalium <br />                                                                                                                                                                                                                                                         H. influenzae <br />                                                                                                                                                                                                                                                                         C. trachomatis <br />                                                                                                                                                                                                                                                                                          H. sapiens <br />  <br />  <br />  <br />  <br /> De novo pathways riboflavin                                                                                             Yes             Yes             Yes               Yes                Yes         Yes             Yes         No              Yes             Yes              No <br />  Ribuloso-5-phosphate L-3,4-dihydroxy-2-butanone-4-phosphate <br />     3,4-Dioxy-2-butanone-4-phosphate synthetase                                           4.1.2.-        gi&#229;&#8230;&#169;1789420        ribBc           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#171;&#185;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />  GTP 5-amino-6-ribitylamino-2,4-(1H,3H)-pyrimidinedione <br />                                                                                                                                        c <br />     GTP cyclohydrolase II                                                                 3.5.4.25       gi&#229;&#8230;&#169;16129238       ribA            &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#171;&#185;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine deaminase                                                                  3.5.4.26       gi&#229;&#8230;&#169;1786616        ribDd           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#353;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine reductase                                                                  1.1.1.193      gi&#229;&#8230;&#169;1786616        ribDd           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#353;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine phosphatase                                                                3.1.3                            ?               ?               ?                 ?                  ?           ?               ?           &#226;&#171;&#186;               ?               ?                &#226;&#171;&#186; <br />  L-3,4-Dihydroxy-2-butanone-4-phosphate, 5-amino-6-ribitylamino-2,4- <br />        (1H,3H)-pyrimidinedione riboflavin <br />     6,7-Dimethyl-8-ribityl-lumazine synthase                                              2.5.1.9        gi&#229;&#8230;&#169;16128400       ribH            &#226;&#171;&#185;               &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#171;&#186; <br />     Riboflavin synthase                                                                   2.5.1.9        gi&#229;&#8230;&#169;16129620       ribE            &#226;&#171;&#185;               &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#171;&#186; <br />  <br /> Riboflavin (vitamin B2) salvage                                                                                            No              No              Yes               No                 No          Yes             Yes         Yes             No              No               Yes <br />     Riboflavin transporter (ypaA)e                                                                       gi&#229;&#8230;&#169;16079362       &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#185;                 &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           ?               &#226;&#171;&#186;               &#226;&#171;&#186;                ? <br />  <br /> Common pathway FMN/FAD                                                                                                  Yes             Yes             Yes               Yes                Yes         Yes             Yes         Yes             Yes             Yes              Yes <br />    Riboflavin kinase                                                                      2.7.1.26       gi&#229;&#8230;&#169;16128019       ribFf           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#185; <br />    FAD synthetase                                                                         2.7.7.2 <br />      (i) Bacteria                                                                                        gi&#229;&#8230;&#169;16128019       ribFf           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />      (ii) Eukarya                                                                                        gi&#229;&#8230;&#169;6320159g       &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  a <br />    See Table 2, footnote a, details. <br />  b <br />    Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />  c <br />    RibA RibB form bifunctional (fused) protein bacteria monofunctional separate proteins E. coli H. influenzae. <br />  d <br />    RibD bifunctional (fused) protein bacteria. <br />  e <br />    Riboflavin transporter originally identified B. subtilis (56). <br />  f <br />    RibF bifunctional (fused) protein bacteria. <br />  g <br />    Eukaryotic FAD synthase originally identified S. cerevisiae (100) detectable sequence similarity corresponding bacterial enzymes. <br />                                                                                                                                                                                                                                                                                                       J. BACTERIOL. <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                        4561 <br />  <br /> Sucr Cms colonies recovered (out 192 Kmr Sucr colonies screened),              formed E. coli strain DH10B identical experimental <br /> suggesting nadD essential E. coli growth rich medium.                   conditions. <br />    Verification predicted gene essentiality S. aureus. Allelic exchange vector <br /> pBT2 utilized verification gene essentiality S. aureus (20). The nadD <br />                                                                                           Preliminary gene essentiality conclusions based <br /> (gi&#229;&#8230;&#169;15927174) pncA (gi&#229;&#8230;&#169;15927492) S. aureus orthologs PCR amplified            semiautomatic analysis number relative posi- <br /> using primers 5&#226;&#172;&#732;-ATTCCTTGTCGCCCGTTATGC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-AACGCGCTTC                            tions inserts retained gene selective out- <br /> ATTGTATCCT-3&#226;&#172;&#732; nadD 5&#226;&#172;&#732;-GTCCGTTAATCCCACAAGCATCA-3&#226;&#172;&#732;                               growth 23 population doublings. Failure recover inserts, <br /> and 5&#226;&#172;&#732;-CGCCGACTTTATCTTTTTCAGC-3&#226;&#172;&#732; pncA. Genomic DNA isolated <br />                                                                                        presence limited number inserts very <br /> from S. aureus strain ATCC 29213 used PCR template. Both PCR <br /> products cloned plasmid pCR2.1-TOPO (Invitrogen, Carlsbad, Calif.).          end coding sequence, suggested cells carrying trans- <br /> A 384-bp NheI-AvrII fragment nadD replaced tetracycline resis-           posons gene viable growth con- <br /> tance marker (generous gift M. Smeltzer, University Arkansas Med-          ditions. However, occur gene constituted a <br /> ical Sciences, Little Rock), resulting 4.0-kb DNA fragment containing          cold spot transposition insert polar effect on <br /> the inactivated nadD cloned pBT2 (20). This plasmid introduced <br /> into S. aureus RN4220 electroporation. Following cointegration, resolution, <br />                                                                                        essential downstream gene. We validated technology <br /> and elimination plasmid (20), tetracycline-resistant (Tetr) colonies       combination following controls: (i) genetic foot- <br /> screened sensitivity chloramphenicol. No Tetr Cms colonies        printing mutagenized population prior outgrowth <br /> the case nadD, pncA successfully inactivated control exper-        (time zero sample) number genes previously estab- <br /> iment. In case, 38% Tetr colonies Cms, Tetr <br />                                                                                        lished essential (Fig. 2B), (ii) verification consistency <br /> Cms colonies tested PCR contained correct pncA deletion chro- <br /> mosome.                                                                                preliminary essentiality assignments metabolic <br />    The S. aureus coaD ortholog (gi&#229;&#8230;&#169;15926709) flanking regions amplified       biological context gene (see below), (iii) compar- <br /> by PCR using primers 5&#226;&#172;&#732;-GATTGCCAGTTGTAGGGTTCATA-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GC                           ison observed essentiality data reported the <br /> GTTGGCTTAATCACAGAATA-3&#226;&#172;&#732;, cloned pBT2 (to produce plasmid                          literature, (iv) genetic footprinting presence a <br /> pMF32), subjected vitro transposon mutagenesis. To end an <br /> artificial transposon constructed (M. Farrell et al., unpublished data) by <br />                                                                                        complementing DNA fragment (for example, reference <br /> cloning Tet M marker derived Tn916 (38) pMOD-2&#226;&#172;?MCS&#226;&#172;&#381;                      27). <br /> (Epicentre Technologies). Plasmid pMF32 mixed 1:1 molar ratio               Global transposon mutagenesis E. coli. Libraries 2 &#226;&#171;&#187; <br /> Tet M-transposon DNA, EZ::TN transposase (Epicentre Technologies)              105 independent insertion mutants generated E. coli <br /> added 0.1-U/&#226;?&#174;l final concentration. The mixture incubated 2 h at <br />                                                                                        strains MG1655 DH10B using Tn5-based EZ::TN&#226;&#172;?KAN- <br /> 37&#194;&#176;C reaction buffer (50 mM Tris-acetate [pH 7.5], 150 mM potassium acetate, <br /> 10 mM magnesium acetate, 40 mM spermidine) used transform E.                2&#226;&#172;&#381;Tnp transposome (Epicentre Technologies) transposon <br /> coli strain DH10B. One Tetr E. coli colonies screened whole-           delivery system. It utilizes mutant hyperactive form the <br /> cell PCR, different transposon-containing pMF32 plasmids se-             Tn5 transposase, reported low inser- <br /> lected, including plasmids containing coaD inactivated single transposon      tion site specificity (41, 64). In study EZ::TN transposase <br /> insertion middle gene plasmids transposon open <br /> reading frames (ORFs) immediately upstream downstream coaD. All five <br />                                                                                        sufficiently random yield 1 14 indepen- <br /> plasmids used parallel pBT2-driven allelic exchange S. aureus           dent insertions majority E. coli genes (for example, <br /> RN4220 (see above). No Tetr Cms (knockout) colonies obtained coaD             Fig. 2). The average insertion density experimentally <br /> disruptions, neighboring ORFs (encoding gi&#229;&#8230;&#169;15926708                      determined insert 250 bp (after outgrowth). <br /> gi&#229;&#8230;&#169;15926710) successfully inactivated. <br />                                                                                        However, total number 105 independent insertion mu- <br />    Preliminary characterization bacterial targets human counter- <br /> parts. Representative bacterial target enzymes human counterparts            tants analyzed 23 cell divisions theoretically corresponds <br /> were expressed E. coli BL21(DE3) N-terminal fusions six-His tag           insert 46 bp genomic sequence (the E. coli <br /> using pProEX-HT3a (Invitrogen). Corresponding DNA fragments                 genome size 4,639 kb). We believe difference may <br /> were amplified PCR using bacterial genomic DNA (American Type Culture               slight preferences target sequence recognition <br /> Collection, Manassas, Va.) human brain cDNA (Clontech) primers gen- <br /> erating NcoI BspHI site 5&#226;&#172;&#732; end SalI PstI site 3&#226;&#172;&#732; end. <br />                                                                                        modified Tn5 transposase actually used a <br /> The DNA fragments purified, digested, cloned NcoI SalI            numerical measure bias. In data a <br /> (or PstI) restriction sites vector. The resulting constructs verified   sixfold difference observed insertion density of <br /> DNA sequencing. Recombinant proteins purified homogeneity using              250 bp versus frequency expected completely <br /> two-step procedure consisting chromatography Ni-nitrilotriacetic acid aga- <br />                                                                                        random insertions (one 46 bp). This insertion density al- <br /> rose (Qiagen) gel filtration (Superdex G-200) described previously (28). <br /> Preliminary enzymatic characterizations performed representative bac-         lowed make essentiality assessments 87% E. coli <br /> terial human enzymes involved biosynthesis NAD (NMN                  ORFs, larger 80 amino acid residues in <br /> NaMNAT), CoA (PPAT), FAD (FADS) using high-performance liquid                      length. <br /> chromatography analysis reaction products and/or coupled enzymatic                 To assess reliability approach, compared re- <br /> assays, described reference 27. Enzymatic reactions performed in <br /> the presence 0.1 mM ATP 0.1 mM concentration respective <br />                                                                                        sults 50 min E. coli chromosome gene <br /> substrates: NMN (or NaMN), 4&#226;&#172;&#732;-phosphopantetheine, flavin mononucleotide             essentiality data compiled literature Genetic <br /> (FMN). High-performance liquid chromatography analysis performed using             Resource Committee Japan (http://www.shigen.nig.ac.jp <br /> ion-pair chromatography isocratic conditions: 100 mM sodium phosphate            /ecoli/pec/Analyses.jsp). Of 111 nonessential genes listed <br /> buffer (pH 3.5) 8 mM tetrabutylammoniumbromide methanol (8% for <br />                                                                                        region chromosome, 98 genes (88%) were <br /> NaMNAT, 15% PPAT, 30% FADS), column (50 4.6 mm <br /> [inner diameter]), packed 5 &#226;?&#174;M C18 (Supelco).                                     experimentally determined nonessential (retained in- <br />                                                                                        serts) procedure. <br />                                                                                           Of 81 known essential genes located 50 <br />                                   RESULTS <br />                                                                                        min E. coli chromosome, transposon insertions were <br />   Genetic footprinting (Fig. 1) systematically applied                          detected 70 genes (86%) experiment. One gene of <br /> determine genes required logarithmic aerobic growth                            81 analyzed, encoding essential cell division protein FtsK <br /> of E. coli MG1655 enriched Luria-Bertani medium. Genetic                            (11), contained 10 insertions. However, inserts were <br /> footprinting limited chromosomal region per-                             clustered C-terminal half coding sequence, <br />  4562     GERDES ET AL.                                                                                                   J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                      FIG. 1. General scheme E. coli genetic footprinting procedure. <br />  <br />  <br />  <br /> corresponding amino acid residues 780 1329 (Fig. 2C).            genes sets data good agreement. Of 13 genes <br /> Only N-terminal domain FtsK required role         lacked transposon insertions outgrowth MG1655, <br /> cell division viability (97). This suggests genetic foot-     12 did contain inserts strain DH10B well. The <br /> printing used successfully detect (and map)          exception nadE, contained single insert the <br /> vivo essential domains ORFs, cases essen-          DH10B experiment. Six seven nonessential genes in <br /> tial region immediately adjacent translational start. In       pathways analyzed strains con- <br /> the 10 remaining essential genes single transposon insertion         tained inserts MG1655 DH10B. Only panD lacked <br /> was detected outgrowth strains            inserts DH10B, MG1655 experiment it <br /> (MG1655 DH10B). This fact              contained two. This reflect fact DH10B <br /> genes tolerate transposon inserts certain restricted        experiment transposons inserted possible <br /> loci detrimental effect corresponding gene            orientations monitored, orientations were <br /> product. Similar phenomena reported ge-             mapped MG1655 experiment. <br /> netic footprinting experiments E. coli (45) H.                Comparison MG1655 DH10B genetic footprints <br /> influenzae (1).                                                        permitted detailed mapping deletions DH10B <br />    Comparison genetic footprints E. coli                chromosome: &#226;&#338;&#172;(ara-leu)7697 (22) &#226;&#338;&#172;lacX74 (10, 89). De- <br /> strains. To test reproducibility approach, gen-          letion &#226;&#338;&#172;(ara-leu)7697 covers 25.6-kb region, corre- <br /> erated limited genetic footprint E. coli strain DH10B             sponds area 63.4 89.0 kb MG1655 genome <br /> (covering 650 kb chromosome) compared          includes genes polB fruR (Fig. 2E). <br /> with corresponding region MG1655. Unambiguous es-               Deletion &#226;&#338;&#172;lacX74 corresponds region 340.3 to <br /> sentiality data strains obtained 530 genes            369.5 kb (total 29.2 kb) MG1655 sequence in- <br /> (excluding deleted DH10B). Identical assessments           cludes genes b0324 b0347 possibly <br /> essentiality produced 487 genes (92%) group.          mhpB (not shown). <br /> The results genetic footprinting experiments MG1655                 Do transposon insertions produce polar effects? Surpris- <br /> and DH10B genes controlling biosynthesis NAD, CoA,              ingly, cases transposon insertions detected up- <br /> and FAD presented Table 1. For majority            stream known essential genes, expected to <br />  VOL. 184, 2002                                                    GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                               4563 <br />  <br />  <br />  <br />  <br />    FIG. 2. Detection mapping transposon insertions. (A) Primer strategy nested PCR. Transposon-specific primers shown gray; <br /> chromosome-specific landmark primers shown black. (B) The gel image shows analysis chromosomal loci: aspC (nonessential) <br /> and lpdA ftsJ (essential) time zero (lanes 1, 3, 5) outgrowth (lanes 2, 4, 6). ORF locations marked relative each <br /> pair lanes. Several inserts visible lpdA ftsJ time zero, detected outgrowth. The nonessential aspC contains <br /> insertions time points. (C E) Examples genetic footprints. Note scale different panel. The length direction of <br /> each gene indicated large horizontal gray arrows. Black diamonds represent transposon inserts. The width diamond corresponds <br /> to mapping error introduced gel electrophoresis. The positions landmark PCR primers shown bows crossing genes, well <br /> as arrows genes. (C) Genetic footprinting ftsK locus MG1655. Only 3&#226;&#172;&#732; half essential gene contains inserts. <br /> (D) Genetic footprinting coaD locus MG1655. A transposon insertion immediately upstream essential gene apparently does not <br /> interfere expression. (E) Mapping &#226;&#338;&#172;(ara-leu)7697 deletion DH10B. The genetic footprint corresponding region MG1655 <br /> is shown comparison. <br />  <br />  <br />  <br /> destroy promoter sequences polar effect              inserts nonessential N-terminal regions protein will <br /> expression downstream genes. This case coaD (as              tolerated interfere translation down- <br /> shown Fig. 2D), argS, frr, rplT, secA,          stream sequences. This limits subgenic resolution ge- <br /> other genes. We believe EZ::TN&#226;&#172;?KAN-2&#226;&#172;&#381; transpo-                     netic footprinting cases multifunctional proteins with <br /> son sequence inserted orientation capable initi-            essential domains proximal translational start (as shown in <br /> ating level transcription sufficient <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>         Fig. 2C). <br /> cases, specific promoter sequence added                  Analysis gene essentiality data context E. coli <br /> its structure. Analysis transposon sequence Neu-              metabolic pathways. One major advantages genetic <br /> ral Network Promoter Prediction program (95; http://www                     footprinting directed knockouts identification es- <br /> .fruitfly.org/seq_tools/promoter.html) reveals multiple putative            sential genes fact footprinting fast highly <br /> E. coli-type promoters oriented outwards directions.                parallel method. As high-throughput technique, has <br /> This probably explains polar effects transposon inser-               limitations, example, cross-feeding outgrowth <br /> tions downstream genes rarely observed study.                complex mutant population. An efficient way minimize <br /> However, translational polarity distal domains genes              erroneous conclusions respect essentiality a <br /> is likely significant limiting factor technique. In         given gene consider results context the <br /> a multifunctional protein essential C-terminal domain,              corresponding metabolic pathway subsystem. Metabolic <br />  4564     GERDES ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Simplified diagrams, illustrating biochemical transformations directly involved biosynthesis NAD(P) (A), CoA (B), and <br /> FMN/FAD (C). Most pathways genes shown present E. coli, exceptions marked dashed lines. Recycling <br /> pathways transformations related genes remain unknown (such NMN deamidase) included. <br />  <br />  <br />  <br />  <br /> context analysis allows reconciliation data genetic          chemically genetically. On hand, key <br /> footprinting experiments, literature (if available),         genes identified recently, essentiality has <br /> well predicted metabolic reconstruction. This tech-              directly confirmed. The results genetic footprinting <br /> nique efficiently applied E. coli,     majority known E. coli genes involved the <br /> best-studied model microbial systems. When contradictions                biosynthesis salvage pathways producing NAD(P), CoA, <br /> between types data encountered certain gene,          FAD summarized Tables 2 4. <br /> it possible formulate probable assertion           Genetic footprinting E. coli genes involved NAD(P) <br /> inspecting behavior E. coli genes           biosynthesis. The behavior majority NAD(P) biosyn- <br /> pathway.                                                                 thetic genes genetic footprinting experiments (Table 1) is <br />     The biosynthesis adenylate cofactors provides good ex-          consistent previously published data meta- <br /> ample utility approach, (i) cofactors         bolic reconstruction presented Fig. 3A. As <br /> are essential metabolites types organisms, (ii)            expected, genes novo salvage pathways <br /> corresponding pathways E. coli rich essential genes,           appear nonessential, experiments outgrowth oc- <br /> and (iii) subsystems thoroughly studied bio-             curred rich media containing niacinamide. Nicotinamide <br />  VOL. 184, 2002                                             GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                       4565 <br />  <br />  <br /> riboside (NmR) probably advanced NAD(P) pre-           (kdtB), previously shown essential (36, 45). As <br /> cursor transported E. coli. Therefore,      shown Table 1, genes essential in <br /> genes common pathway essential,       footprinting study, acpS, produces en- <br /> inactivation compensated NAD(P) salvage.          zyme responsible covalent attachment CoA acyl car- <br /> This consistent observations, notable ex-     rier protein, required fatty acid biosynthesis (33). <br /> ception nadD. Multiple insertions gene ob-            Genetic footprinting E. coli genes involved FMN/FAD <br /> served, suggesting nadD dispensable E. coli growth     biosynthesis. All FMN/FAD biosynthetic genes were <br /> in rich media. This contradicted genetic data Salmonella       essential genetic footprinting experiments in <br /> enterica serovar Typhimurium (48), exper-       E. coli strains (Table 1). The fact genes de <br /> iments E. coli MG1655, failed produce directed       novo riboflavin biosynthesis essential presence of <br /> deletion nadD rich medium, presence           riboflavin (0.8 mg/liter) medium consistent the <br /> 50 &#226;?&#174;M NAD NMN. In hands, nadD                 absence riboflavin transporter E. coli K-12 (6). Ribo- <br /> successfully deleted presence plasmid containing     flavin auxotrophs obtained E. coli using specific <br /> functional human nadD ortholog, pyridine nucleotide adeny-         selection steps facilitate riboflavin transport uncharacter- <br /> lyltransferase 1 (PNAT-1) (gi&#229;&#8230;&#169;12620200; O. Kurnasov, unpub-        ized mutations significantly higher concentrations of <br /> lished data).                                                      exogenous riboflavin (6, 8, 9). <br />    One possible explanation contradiction pres-        Two enzymes common flavin pathway, consecutively <br /> ence functional NadR E. coli (Table 2). The nadR gene     producing cofactors, FMN FAD, form bifunc- <br /> a transcriptional regulator novo biosynthesis        tional protein encoded essential ribF gene. Both enzy- <br /> niacin salvage genes E. coli (92). Recently NadR shown      matic activities expected indispensable, ge- <br /> to possess low levels NMN adenylyltransferase activity (74)     netic footprinting data suggest essentiality the <br /> as NmR kinase activity (O. Kurnasov et al., unpublished    C-terminal domain (encoding FK). This general limitation <br /> data). This suggests NadR play role salvaging        technique respect multifunctional proteins. <br /> exogenous NMN. The NMN adenylyltransferase activity                In summary, 13 genes encoding 16 enzymes three <br /> NadR involved recycling intracellular NMN           vitamin/cofactor biosynthetic pathways shown re- <br /> directly NAD, bypassing NAD synthase encoded             quired aerobic growth E. coli enriched media (Table <br /> nadE (Fig. 3A). However, flux PnuC-NadR sal-           1). Twelve genes identified essential genetic <br /> vage pathway unlikely sufficient compensate        footprinting agreement published data theoretical <br /> inactivation nadD nadE, responsible        analysis. The contradictory case (nadD) reconciled by <br /> bulk NAD production E. coli (73). Also, explanation     directed gene deletion strategy. In addition, 12 genes <br /> contradicts apparent essentiality nadE observed      related metabolism NAD(P), CoA, FAD ana- <br /> genetic footprinting experiments, compensatory role      lyzed genetic footprinting nonessential <br /> of PnuC-NadR bypass imply nonessentiality nadE        (Table 1). Again, results good agreement the <br /> as nadD (Fig. 3A). At present propose          available experimental data metabolic reconstructions of <br /> single unambiguous interpretation contradiction be-         corresponding pathways. <br /> tween classical genetic genetic footprinting data        Some essential genes representing potential broad-spectrum <br /> respect nadD. However, point disagree-      antibacterial targets analyzed respect the <br /> ment NAD(P) biosynthesis genetic footprinting       genomic data available selected bacterial pathogens for <br /> data types evidence.                                  human host (see below). This analysis allows extrapolation <br />    Genetic footprinting E. coli genes involved pantothe-     experimental gene essentiality data E. coli onto <br /> nate/CoA biosynthesis. In contrast complexity            bacterial pathogens, including S. aureus. <br /> NAD(P) metabolism, CoA biosynthetic pathway topo-              Selected target validation. The predicted essentiality the <br /> logically simple (Fig. 3B). All genetic footprinting data   S. aureus nadD ortholog (gi&#229;&#8230;&#169;15927174) confirmed an <br /> are consistent experimental data theoret-      indirect approach. Deletion nadD attempted parallel <br /> ical considerations (Table 1). Each enzymatic step com-     S. aureus pncA ortholog (gi&#229;&#8230;&#169;15927492), presumed be <br /> mon pathway pantothenate CoA nonredundant           dispensable presence exogenous nicotinate, using <br /> indispensable, phosphorylated intermediates      allelic exchange vector pBT2 (20). No &#226;&#338;&#172;nadD variants were <br /> can transported cell. This eliminates potential        obtained, extensive screening (2,248 colonies), while <br /> cross-feeding possibility exogenous CoA salvage      control experiment pncA successfully inactivated in <br /> transport phosphorylated precursors growth         38% colonies screened. This indicates nadD is <br /> medium. All genes pantothenate biosynthetic path-       essential S. aureus growth complex tryptic soy <br /> way nonessential bypassed exogenous pan-         broth (Difco Laboratories, Detroit, Mich.). <br /> tothenate known transporter encoded panF gene            A different strategy used verify predicted essen- <br /> (51, 93).                                                          tiality S. aureus coaD ortholog (gi&#229;&#8230;&#169;15926709). In vitro <br />    The coaA gene E. coli, encoding pantothenate kinase,     transposon mutagenesis used disrupt coaD as <br /> first enzyme common five-step pathway pantothe-        adjacent ORFs. The conclusion coaD essentiality <br /> nate CoA, previously characterized essential (86, 94).   drawn based fact coaD disrupted <br /> Genes enzymatic steps CoA biosynthesis        neighboring ORFs (gi&#229;&#8230;&#169;15926708 gi&#229;&#8230;&#169;15926710) <br /> (coaBC, coaD, coaE) recently discovered         successfully inactivated Tet M transposon, with <br /> (39, 65). Two genes, coaE (formerly yacE) coaD        frequencies 83 5%, respectively. <br />  4566     GERDES ET AL.                                                                                                          J. BACTERIOL. <br />  <br />  <br />  <br />  <br />  FIG. 4. Selection antibacterial drug targets combination genetic footprinting E. coli comparative analysis reconstructed <br /> metabolic subsystems, pathways, individual genes pathogens humans. <br />  <br />  <br />  <br />    Selected bacterial targets human counterparts               genetic footprinting identifying novel antibacterial drug tar- <br /> were overproduced E. coli, purified, characterized. The           gets (Fig. 4). <br /> respective adenylyltransferase activities following re-              Experimental gene essentiality data determined genetic <br /> combinant enzymes directly confirmed vitro:                      footprinting analyzed terms relevant E. coli met- <br /> NaMNATs E. coli, S. aureus, H. pylori (gi&#229;&#8230;&#169;1786858,              abolic subsystems pathways using ERGO database. <br /> gi&#229;&#8230;&#169;15924584, gi&#229;&#8230;&#169;2314504, respectively); isoforms hu-          Such contextual analysis refines essentiality assessment of <br /> man PNAT, i.e., PNAT-1 (gi&#229;&#8230;&#169;12620200) PNAT-2                          gene addressing following questions. Which met- <br /> (gi&#229;&#8230;&#169;11245478); PPATs S. aureus, M. tuberculosis, H. influ-          abolic pathway essential gene associated with? Does this <br /> enzae, H. pylori (gi&#229;&#8230;&#169;15924115, gi&#229;&#8230;&#169;560525, gi&#229;&#8230;&#169;1573650,            pathway yield metabolite essential cell viability? Is there <br /> gi&#229;&#8230;&#169;15612433, respectively); PPAT domain human                 route produce metabolite? Can me- <br /> PPAT/DPCK protein (27); FADS domain B. subtilis                   tabolite salvaged growth media final form? <br /> bifunctional FK/FADS protein (gi&#229;&#8230;&#169;16078730); human                    What advanced precursor (the point salvage) <br /> FADS (F. Mseeh, unpublished results).                                    cell acquire environment? This kind of <br />                                                                          analysis generates hypotheses essentiality non- <br />                                                                          essentiality known genes pathways. Com- <br />                           DISCUSSION <br />                                                                          parison models actual genetic footprinting data <br />    Our approach identifying ranking antibacterial drug            data available literature (e.g., http://www <br /> targets based combination genetic footprinting              .shigen.nig.ac.jp/ecoli/pec/Analyses.jsp) allows refinement of <br /> model (E. coli) comparative genome analysis                initial experimental assessments gene essentiality calls for <br /> schematically illustrated Fig. 4. Both major components            experimental examination (as case nadD). <br /> this approach, experimental computational, post-                    Once consistent data compiled E. coli, metabolic <br /> genomic techniques. Genetic footprinting E. coli allows               reconstruction selected pathways extended panel of <br /> identification essential genes function growth condi-         representative pathogens. A hierarchical overview illustrated <br /> tions high-throughput format. This relatively new tech-             Tables 2 4 provides convenient way perform com- <br /> nique multiple potential applications functional genom-           parative cross-genome analysis relevant pathways met- <br /> ics. For example, monitoring changes <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene           abolic subsystems. Questions addressed step the <br /> essentiality varying growth conditions and/or genetic              analysis include following. Are pathways present in <br /> background, possible assess various aspects cell         pathogens consideration? Which pathways con- <br /> metabolism. This allow functions previously uncharac-             taining essential genes conserved nonredundant within <br /> terized genes established allows functional roles          group? Which functions selected pathways are <br /> of known genes refined. An example study             encoded closely related genes? The point reflect <br /> been published recently (7). Here consider application             likelihood finding small molecule efficiently <br />  VOL. 184, 2002                                                     GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                         4567 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 6. General scheme chemical transformations catalyzed by <br />                                                                            adenylyltransferases (ATse) biosynthesis NAD(P), <br />                                                                            CoA, FAD. These enzymes&#226;&#8364;&#8221;NaMNAT (encoded E. coli by <br />                                                                            nadD), PPAT (gene coaD), FADS (ribF)&#226;&#8364;&#8221;were selected the <br />                                                                            promising antibacterial drug targets pathways. <br />  <br />  <br />  <br />                                                                            bind target protein pathogens spectrum <br />                                                                            suppressing functional activity. We use parameters as <br />                                                                            preliminary measures likelihood, P values pair- <br />                                                                            wise sequence comparisons E values produced by <br />                                                                            comparing bacterial sequence profile (HMM) built <br />                                                                            specific panel pathogens (as illustrated Fig. 5A and <br />    FIG. 5. Prioritization potential broad-spectrum antibacterial        B). With structural functional data, elaborate <br /> drug targets biosynthetic pathways NAD(P), CoA, FAD. <br /> Three criteria shown target enzyme (marked abbrevi-        productive criteria applied, conservation of <br /> ations x axis). (A) Range pathogens. A number patho-       elements active site, relative substrate preferences, <br /> gens (from representative set) containing given target enzyme      affinities range natural synthetic ligands. <br /> (and included building corresponding HMM) represented           Next, corresponding human pathways reconstructed <br /> by filled bars. Organisms containing target enzyme there- <br /> fore excluded spectrum indicated inside open bars (by <br />                                                                            genomic data assess potential effects may <br /> two-letter abbreviations). (B) Compact pathogen subsets outliers.      arise inhibition human counterparts antibac- <br /> Negative logarithms E values indicate closely bacterial    terial targets. One criterion used selecting antibacte- <br /> sequence related probabilistic consensus corresponding    rial targets absence corresponding function and <br /> profile HMM (larger values indicate higher similarity). Subsets         corresponding pathway human host (for in- <br /> pathogens target orthologs closely related consensus <br /> are shown inside dotted boxes. Outliers indicated two-letter        stance, bacterial cell wall biosynthesis). This approach, al- <br /> organism abbreviations. Organisms shown P. aeruginosa (E) (PA),        historically productive, unnecessarily rejects <br /> B. anthracis ( ) (BA), M. tuberculosis (&#198;&#8217;) (MT), H. pylori (&#226;&#8211;&#160;) (HP), S.    promising targets. Many targets universal biosynthetic path- <br /> pneumoniae (&#228;&#352;?) (PN), S. aureus (&#226;&#189;&#167;) (SA), M. genitalium (&#227;&#8364;&#171;) (MG),           ways reconsidered new opportuni- <br /> H. influenzae (&#197;&#8217;) (HI), C. trachomatis (&#226;&#8364;&#353;) (CT), E. coli (F). The <br /> E. coli proteins similar Yersinia pestis Salmo- <br />                                                                            ties provided comparative genomics parallel chemical <br /> nella enterica serovar Typhi. (C) Human counterparts: distance        synthesis. Certain features revealed level host-patho- <br /> bacterial families. The negative logarithms E values human       gen comparative genome analysis useful evaluation <br /> counterpart sequences compared corresponding bacterial              targets. For instance, pathway unique in <br /> HMMs (smaller values indicate higher divergence human en-           pathogens potentially redundant humans (as NAD <br /> zyme corresponding bacterial family). &#226;&#180;&#177;&#226;&#180;&#177;, HMM E values for <br /> these human proteins higher threshold (&#226;&#172;&#381;10,000).              biosynthesis), human functional counterpart be <br />                                                                            structurally unrelated bacterial proteins. In cases where <br />  4568     GERDES ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br /> bacterial orthologs target enzyme human version         graded nicotinamide riboside membrane-bound or <br /> share sequence similarity, prioritize targets based       periplasmic phosphohydrolases, nadN gene prod- <br /> (i) lowest P values generated pairwise compar-       uct (76). In addition, two-step niacinamide salvage pathway <br /> isons target protein homologs bacterial pathogen        gram-positive pathogens S. pneumoniae S. au- <br /> with human functional counterpart (ii) E value         reus constitute possible narrow-spectrum target. Both <br /> derived comparison human protein sequence             organisms lack genes novo NAD biosynthesis and <br /> the profile HMM built corresponding pathogen pro-            PnuC-NadR pathway nicotinamide riboside salvage <br /> teins (as Fig. 5C).                                               (Table 2). <br />    To illustrate approach, essential conserved genes           The three-step pathway NaMN NAD NADP, <br /> with known functions biosynthesis key adenylate        requiring NaMNAT, NADS, NADK, conserved the <br /> cofactors NAD(P), CoA, FAD investigated. These              majority bacterial pathogens (Table 2), representing poten- <br /> biosynthetic pathways fit criteria listed above: (i)     tially broad-spectrum targets. This pathway nonredundant, <br /> each cofactor utilized multiple enzymes pathogens;      corresponding genes essential E. coli (Table 2). <br /> (ii) bacteria unable import cofactors     NaMNAT NADS present C. trachomatis H. <br /> phosphorylated precursors prior degradation; (iii)       influenzae, excludes pathogens target <br /> last steps biosynthetic pathways universal        profile. Inhibition NADK cover broadest range of <br /> and nonredundant contain number promising targets.          pathogens, excluding C. trachomatis (see Fig. 5A). How- <br />    NAD(P) biosynthesis characterized high level           ever, NADK recently characterized H. sapiens (57), <br /> complexity diversity various organisms (60, 66), includ-      revealing high sequence similarity bacterial <br /> ing versions novo biosynthesis          enzymes (the P value 2 &#228;&#161;&#160; 10&#226;&#171;&#186;25 NADK P. aeruginosa <br /> routes salvage exogenous precursors (Table 2 Fig.         significantly better P values pairwise comparison <br /> 3A). The absence recognizable genes NAD(P) bio-           bacterial orthologs). The C-terminal synthase <br /> synthesis genome C. trachomatis (Table 2) suggests         domain human NADS ortholog (gi&#229;&#8230;&#169;10433831) reveals a <br /> that obligate intracellular parasite unique transport       lower significant sequence similarity its <br /> machinery salvage NAD NADP. Therefore, C. tra-            bacterial counterparts (the best P value 6 &#228;&#161;&#160; 10&#226;&#171;&#186;13 NADS <br /> chomatis excluded panel pathogens used           M. genitalium). Finally, human counterpart bacte- <br /> to assess targets NAD biosynthesis.                               rial NaMNAT marginal sequence similarity bac- <br />    Genes controlling novo NAD biosynthesis aspartate         terial enzymes nadD family (the best P value &#226;&#172;&#402;0.2 for <br /> and salvage niacin E. coli nonessential rich        NaMNAT H. pylori). E values derived comparing <br /> media. This conclusion projected P. aeruginosa, B.       human PNAT, NADS, NADK profile HMMs the <br /> anthracis, possibly M. tuberculosis based metabolic re-       corresponding bacterial protein families support conclu- <br /> constructions (Table 2). The cases controversial.   sions (Fig. 5C). Prioritization potentially broad- <br /> In spite fact genes required niacinamide        spectrum targets NAD biosynthesis combination of <br /> salvage (pncA pncB) present genome, M. tuber-         criteria summarized Fig. 5 yields target <br /> culosis reported strictly dependent novo        preference order follows: NaMNAT better NADS <br /> NAD biosynthesis (83). Since pncA expressed M. tubercu-        better NADK. <br /> losis (81, 102), interpretation pncB ortholog        NaMNAT activity initially characterized E. coli (26), <br /> functionally inactive M. tuberculosis, labora-     chromosomal locus mapped Salmonella (48). <br /> tory conditions.                                                     However, identity structural gene remained un- <br />    Theoretically, bacterial novo NAD biosynthesis as-        known recently. We identified gene based its <br /> partate attractive target pathway, absent       clustering chromosome nadE pncB gene or- <br /> humans, novo biosynthesis occurs five-step ox-         thologs microbial genomes (12). This gene been <br /> idative degradation tryptophan (66). However,          independently identified E. coli Mehl coworkers (63). <br /> set bacterial pathogens, H. pylori expected       We cloned expressed nadD E. coli have <br /> useful targets pathway, lacks niacin salvage        cloned expressed orthologs gram-positive (S. au- <br /> genes (Table 2). The two-step conversion aspartate             reus) gram-negative (H. pylori) pathogens (12), we <br /> quinolinate make excellent target pathway anti-         purified NadD proteins. Analysis substrate <br /> infective agents specific H. pylori (Table 2 Fig. 3A). A      specificity purified recombinant proteins revealed strong <br /> systematic analysis essential genes conserved exclusively      preferences bacterial enzymes NaMN over <br /> H. pylori (23) did recognize potential pathway    NMN (up &#226;&#172;&#381;17,000-fold case S. aureus NaMNAT). <br /> a target drug development, did          This bias consistent major flux NAD biosynthesis <br /> account metabolic context organism.                      bacteria going nicotinic acid mononucleotide <br />    The PnuC-NadR pathway nicotinamide riboside salvage            (NaMN) NaAD. <br /> in H. influenzae provides example narrow spec-             At isoforms NadD identified the <br /> trum target. The complete pathway present limited       human genome (gi&#229;&#8230;&#169;12620200, gi&#229;&#8230;&#169;11245478, gi&#229;&#8230;&#169;14029540). <br /> number bacteria, sole route NAD biosyn-         We cloned, purified, characterized forms <br /> thesis H. influenzae. H. influenzae requires so-called V fac-     designated PNAT-1 PNAT-2, them <br /> tors growth (25, 76), simplest nicotinamide      perform adenylyltransferase reaction equally efficiently <br /> riboside. All acceptable V factors, NADP, NAD,             pyridine nucleotides NMN NaMN (O. Kurnasov <br /> NMN (if present growth medium), gradually de-             et al., unpublished data). Human PNAT-2 indepen- <br />  VOL. 184, 2002                                                GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                        4569 <br />  <br />  <br /> dently identified characterized groups (31,          tholog monofunctional PPCDC. Monofunctional <br /> 80). The observed dual specificity human PNATs consis-          PPCSs S. pneumoniae B. anthracis reveal high <br /> tent previous experimental data (61) recon-         sequence similarity other, quite divergent <br /> struction putative human NAD biosynthetic pathway              PPCS domain bifunctional PPCDC/PPCS pro- <br /> from genomic data (Table 2).                                          teins. In contrast bacterial orthologs, produce a <br />    The difference substrate preferences human              low reliable similarity score compared human <br /> PNATs bacterial NaMNATs additional incentive               monofunctional PPCS (3 &#228;&#161;&#160; 10&#226;&#171;&#186;23 H. sapiens B. <br /> pursuing NadD selective, relatively broad-spectrum           anthracis). The enzyme common pathway, DPCK, is <br /> antibacterial target. The three-dimensional structures             ubiquitous pathogens set. <br /> NaMNATs E. coli (101) B. subtilis (70)                The enzymes human CoA biosynthesis were <br /> solved association NaMN. Comparison struc-           recently identified characterized (27). Two them, <br /> tures structure human PNAT-2 (103) reveals sig-           PPCDC DPCK, relatively similar bacterial <br /> nificant differences organization active sites,       counterparts (Fig. 5C). The human monofunctional PPCS is <br /> which facilitate identification selective antibacterial      distant PPCS domains bacterial bifunctional <br /> inhibitor.                                                            PPCDC/PPCS proteins, closely related the <br />    Similar comparative analysis CoA FMN/FAD                bacterial monofunctional PPCDCs. The PPAT domain the <br /> biosynthetic pathways humans bacteria produced addi-           human bifunctional PPAT/DPCK protein reveals signifi- <br /> tional broad-spectrum antibacterial targets (Fig. 5). The          sequence similarity bacterial counterpart, it <br /> novo pathway CoA biosynthesis present bacterial        likely quite dissimilar overall structure. PPAT <br /> pathogens absent humans (Table 3), corre-              probably represents attractive target common <br /> sponding enzymes attractive targets          CoA biosynthetic pathway number reasons: (i) con- <br /> the universal presence pantothenate symporter orthologs.           stitutes target broad range pathogens (Fig. 5A), (ii) all <br /> Some pathogens lack novo pantothenate biosynthetic            bacterial orthologs enzyme closely related the <br /> genes (Table 3) depend solely salvage pantothenate (S.          consensus (no outliers; Fig. 5B), (iii) human PPAT is <br /> pneumoniae H. influenzae) dephospho-CoA (M. geni-              dissimilar bacterial HMM profile (Fig. 5C). <br /> talium C. trachomatis). Bacterial pantothenate symporters         PPAT E. coli (encoded coaD, previously kdtB) was <br /> (panF family) share high sequence similarity human             previously described, three-dimensional structure was <br /> protein implicated vitamin transport (96). No panF or-             solved (39, 50). <br /> thologs present genomes M. genitalium C.                The majority bacteria (with exception M. geni- <br /> trachomatis. Specific transporters allowing salvage host de-       talium) considered conserved novo riboflavin <br /> phospho-CoA intracellular pathogens (which             biosynthetic pathway, present humans (Table 4 <br /> represent narrow spectrum targets) unknown.                 Fig. 3C). Moreover, genes novo path- <br />    All enzymatic steps common pathway CoA              way essential E. coli (Table 4). These observations have <br /> biosynthesis pantothenate encoded essential genes         brought attention gene products potential <br /> in E. coli (Table 3). Four enzymes conserved        antibacterial drug targets (17, 29, 98). However, direct exper- <br /> broad range bacterial pathogens. The enzyme           imental analysis riboflavin transport salvage these <br /> pathway, PK, encoded E. coli coaA (86) belongs             pathogens necessary order define useful these <br /> structural family different recently identified      targets be. B. subtilis known effectively salvage exog- <br /> eukaryotic PKs (21). This makes coaA-related PKs attrac-           enous riboflavin, bypassing requirement novo <br /> tive target relatively narrow subset list patho-      pathway (19). Orthologs riboflavin transporters, <br /> gens, including H. influenzae, M. tuberculosis, S. pneu-          gene ypaA, recently identified B. subtilis (56) present in <br /> moniae. S. aureus (as staphylococci                 gram-positive pathogens, including B. anthracis, S. au- <br /> enterococci) B. anthracis lack coaA orthologs,           reus, S. pneumoniae, functionality not <br /> enzymatic function likely performed remote or-             directly confirmed. Additional structurally dissimilar ri- <br /> thologs eukaryotic PK (such tr&#229;&#8230;&#169;Q99SC8 S. aureus).        boflavin transporters likely exist pathogens. For <br /> Neither known forms PK identified             example, riboflavin transport occur M. genitalium, since <br /> sequence similarity analysis P. aeruginosa, H. pylori,       novo riboflavin biosynthesis pathway asserted in <br /> few pathogens included set, suggests       organism (Table 4). However, ypaA orthologs be <br /> existence uncharacterized form enzyme.        available mycoplasma ureaplasma ge- <br />    All remaining enzymes pathway represent poten-         nomes. <br /> tial antibacterial drug targets relatively broad spectrum (Ta-        A common pathway FMN/FAD biosynthesis repre- <br /> ble 3). In bacteria, enzymatic activities, PPCS          sented bacteria bifunctional enzyme catalyzes <br /> PPCDC, located C-terminal N-terminal               consecutive reactions. The reaction formation of <br /> domains bifunctional (fused) protein (gene coaBC, previ-         FMN, catalyzed FK domain (C terminus). The second <br /> ously dfp E. coli). The exception list pathogens S.   reaction conversion FMN FAD FADS domain <br /> pneumoniae (as streptococci enterococci)            (N terminus). Both enzymes represent antibacterial targets <br /> containing separate ORFs, coaB coaC, operon            broad range (Fig. 5A). The HMM E values the <br /> (in reversed order relative fusion proteins). Interest-        group pathogens high (Fig. 5B), indicating <br /> ingly, B. anthracis (as Bacillus cereus) contains      targets divergent. However, FADS <br /> bifunctional PPCDC/PPCS monofunctional PPCS or-                 parameter improved removing outliers, <br />  4570       GERDES ET AL.                                                                                                                                  J. BACTERIOL. <br />  <br />  <br /> such H. pylori (often source divergent pro-                               quired growth survival Haemophilus influenzae. Proc. Natl. Acad. <br />                                                                                              Sci. USA 99:966&#226;&#8364;&#8220;971. <br /> teins sample group pathogens) M. genitalium                           3.   Alm, R. A., L. S. L. Ling, D. T. Moir, B. L. King, E. D. Brown, P. C. Doig, <br /> (Fig. 5B).                                                                                   D. R. Smith, B. Noonan, B. C. Guild, B. L. deJonge, G. Carmel, P. J. <br />    In contrast known bacteria, eukaryotic FK                             Tummino, A. Caruso, M. Uria-Nickelsen, D. M. Mills, C. Ives, R. Gibson, <br />                                                                                              D. Merberg, S. D. Mills, Q. Jiang, D. E. Taylor, G. F. Vovis, T. J. Trost. <br /> FADS identified S. cerevisiae (78, 100) monofunctional                                1999. Genomic-sequence comparison unrelated isolates hu- <br /> proteins. We overexpressed, purified, characterized                                 man gastric pathogen Helicobacter pylori. Nature 397:176&#226;&#8364;&#8220;180. <br /> both human enzymes common FMN/FAD biosynthetic                                   4.   Arigoni, F., F. Talabot, M. Peitsch, M. D. Edgerton, E. Meldrum, E. Allet, <br />                                                                                              R. Fish, T. Jamotte, M. L. Curchod, H. Loferer. 1998. A genome-based <br /> pathway (F. Mseeh, unpublished data). Sequence comparison                                    approach identification essential bacterial genes. Nat. Biotechnol. <br /> of human FK FADS profile HMMs clearly indi-                                     16:851&#226;&#8364;&#8220;856. <br /> cates FADS potentially selective target (Fig.                            5.   Bacher, A., S. Eberhardt, W. Eisenreich, M. Fischer, S. Herz, B. Illarionov, <br />                                                                                              K. Kis, G. Richter. 2001. Biosynthesis riboflavin. Vitam. Horm. <br /> 5C). In agreement this, reliable P values produced                               61:2&#226;&#8364;&#8220;49. <br /> in pairwise comparisons human FADS bac-                              6.   Bacher, A., S. Eberhardt, G. Richter. 1996. Biosynthesis riboflavin, <br /> terial counterparts, human FK sequence similar-                                p. 657&#226;&#8364;&#8220;664. In F. C. Neidhardt et al. (ed.), Escherichia coli Salmonella: <br />                                                                                              cellular molecular biology. ASM Press, Washington, D.C. <br /> ity bacterial FK domains higher similarity                             7.   Badarinarayana, V., P. W. Estep, J. Shendure, J. Edwards, S. Tavazoie, F. <br /> between bacterial FK domains (for example,                                   Lam, G. M. Church. 2001. Selection analyses insertional mutants <br />                                                                                              using subgenic-resolution arrays. Nat. Biotechnol. 19:1060&#226;&#8364;&#8220;1065. <br /> M. tuberculosis FK domain gives P value 10&#226;&#171;&#186;9 human <br />                                                                                         8.   Bandrin, S. V., M. Y. Beburov, P. M. Rabinovich, A. I. Stepanov. 1979. <br /> FK P value 4 &#228;&#161;&#160; 10&#226;&#171;&#186;6 H. pylori FK domain).                                  Riboflavin auxotrophs Escherichia coli. Genetika 15:2063&#226;&#8364;&#8220;2065. (In Rus- <br />    In conclusion, genetic footprinting E. coli combination                             sian.) <br />                                                                                         9.   Bandrin, S. V., P. M. Rabinovich, A. I. Stepanov. 1983. 3 linkage groups <br /> with comparative genome analysis reconstructed metabolic                                  genes riboflavin biosynthesis Escherichia coli. Genetika 19:1419&#226;&#8364;&#8220; <br /> subsystems pathways allowed identify number                                  1425. (In Russian.) <br /> of antibacterial targets involved biosynthesis ad-                       10.   Beckwith, J. R., E. R. Signer. 1966. Transposition Lac region of <br />                                                                                              Escherichia coli. I. Inversion Lac operon transduction Lac by <br /> enylate cofactors NAD(P), CoA, FAD. Among                                       &#226;?&#190;80. J. Mol. Biol. 19:254&#226;&#8364;&#8220;265. <br /> attractive targets pathways adenylyltrans-                        11.   Begg, K. J., S. J. Dewar, W. D. Donachie. 1995. A new Escherichia coli <br /> ferases: NaMNAT, PPAT, FADS (Fig. 6). Two                                       cell-division gene, ftsk. J. Bacteriol. 177:6211&#226;&#8364;&#8220;6222. <br />                                                                                        12.   Begley, T. P., C. Kinsland, R. A. Mehl, A. Osterman, P. Dorrestein. <br /> targets (NaMNAT PPAT) validated direct-                                     2001. The biosynthesis nicotinamide adenine dinucleotides bacteria. <br /> ed-knockout strategy S. aureus. Some orthologs target                              Vitam. Horm. 61:103&#226;&#8364;&#8220;119. <br /> enzymes representative pathogens, human                          13.   Begley, T. P., C. Kinsland, E. Strauss. 2001. The biosynthesis of <br />                                                                                              coenzyme A bacteria. Vitam. Horm. 61:158&#226;&#8364;&#8220;171. <br /> counterparts, overproduced, purified, characterized                           14.   Benton, B. L., J. Ving, F. Malouin, P. K. Martin, M. B. Schmid, D. Sun. <br /> by direct vitro assays. Comparative analysis adenylate                                 March 2000. Methods screening compounds active Staphylococcus <br /> cofactor biosynthesis provides illustration general ap-                              aureus target genes. U.S. patent 6,037,123. <br />                                                                                        15.   Benton, B. L., J. Ving, F. Malouin, P. K. Martin, M. B. Schmid, D. Sun. <br /> proach extended functional subsystems                                May 2001. Methods screening compounds active Staphylococcus <br /> reveal novel antibacterial targets.                                                          aureus target genes. U.S. patent 6,228,588. <br />                                                                                        16.   Bernal, A., U. Ear, N. Kyrpides. 2001. Genomes OnLine Database <br />                                                                                              (GOLD): monitor genome projects world-wide. Nucleic Acids Res. <br />                                 ADDENDUM                                                     29:126&#226;&#8364;&#8220;127. <br />                                                                                        17.   Black, M. T., L. K. Shilling, R. K. Stodola, R. L. Warren, A. L. Kosmatka, <br />    While manuscript reviewed, genome-wide                                    R. O. Nicholas, L. M. Palmer, M. A. Lonetto, J. C. Fedon, J. E. Hodgson, <br />                                                                                              D. J. C. Knowles. June 2001. RibB. U.S. patent 6,252,044. <br /> analysis essential genes H. influenzae published (2).                        18.   Blattner, F. R., G. Plunkett, C. A. Bloch, N. T. Perna, V. Burland, M. Riley, <br /> Comparing essential genes divergent organisms use-                                J. ColladoVides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, N. W. <br /> ful &#226;&#8364;&#339;for defining common essential pathways life                                 Davis, H. A. Kirkpatrick, M. A. Goeden, D. J. Rose, B. Mau, Y. Shao. <br />                                                                                              1997. The complete genome sequence Escherichia coli K-12. Science <br /> and potential targets development antimicrobial thera-                                277:1453&#226;&#8364;&#8220;1462. <br /> peutics&#226;&#8364;? (2). Our genome-scale list essential nonessen-                         19.   Bresler, S. E., E. A. Glazunov, D. A. Perumov. 1972. Study the <br /> tial genes E. coli currently preparation.                                           operon riboflavin biosynthesis Bacillus subtilis. IV. Regulation the <br />                                                                                              synthesis riboflavin synthetase. Investigation riboflavin transport <br />                                                                                              cell membrane. Genetika (USSR) 8:109&#226;&#8364;&#8220;117. <br />                           ACKNOWLEDGMENTS                                              20.   Bru&#194;&#168;ckner, R. 1997. Gene replacement Staphylococcus carnosus and <br />                                                                                              Staphylococcus xylosus. FEMS Microbiol. Lett. 151:1&#226;&#8364;&#8220;8. <br />    We grateful William Reznikoff Igor Goryshin (University                  21.   Calder, R. B., R. S. B. Williams, G. Ramaswamy, C. O. Rock, E. Campbell, <br /> of Wisconsin, Madison); Jerry Jendrisak Les Hoffman (Epicentre                           S. E. Unkles, J. R. Kinghorn, S. Jackowski. 1999. Cloning char- <br /> Technologies); Tadhg Begley (Cornell University, Ithaca, N.Y.); Zol-                         acterization eukaryotic pantothenate kinase gene (panK) Aspergil- <br /> tan Oltvai (Northwestern University, Chicago, Ill.); Henry Burd,                         lus nidulans. J. Biol. Chem. 274:2014&#226;&#8364;&#8220;2020. <br />                                                                                        22.   Casadaban, M. J., S. N. Cohen. 1980. Analysis gene control signals <br /> Niels Larsen, Gregory Kogan, Yuri Kuniver, Yuri Grechkin, Rob- <br />                                                                                              DNA fusion cloning Escherichia coli. J. Mol. Biol. 138:179&#226;&#8364;&#8220;207. <br /> ert Haselkorn (Integrated Genomics) valuable discussions, help,                    23.   Chalker, A. F., H. W. Minehart, N. J. Hughes, K. K. Koretke, M. A. Lonetto, <br /> and support project. We thank George Church (Har-                            K. K. Brinkman, P. V. Warren, A. Lupas, M. J. Stanhope, J. R. Brown, and <br /> vard Medical School, Boston, Mass.) gift pKO3, Mark                               P. S. Hoffman. 2001. Systematic identification selective essential genes in <br /> Smeltzer (University Arkansas Medical Sciences) gift                       Helicobacter pylori genome prioritization allelic replacement mu- <br /> pCRII::tet RN4220, Reinhold Bru    &#194;&#168;ckner (Universita&#194;&#168;t Tu <br />                                                              &#194;&#168;bingen,                        tagenesis. J. Bacteriol. 183:1259&#226;&#8364;&#8220;1268. <br /> Tu&#194;&#168;bingen, Germany) pBT2, Don Clewell (University Mich-                     24.   Cole, S. T., R. Brosch, J. Parkhill, T. Garnier, C. Churcher, D. Harris, S. V. <br /> igan, Ann Arbor) plasmid pAM120 containing Tn916.                                    Gordon, K. Eiglmeier, S. Gas, C. E. Barry, F. Tekaia, K. Badcock, D. <br />                                                                                              Basham, D. Brown, T. Chillingworth, R. Connor, R. Davies, K. Devlin, T. <br />                                                                                              Feltwell, S. Gentles, N. Hamlin, S. Holroyd, T. Hornby, K. Jagels, A. Krogh, <br />                                  REFERENCES                                                  J. McLean, S. Moule, L. Murphy, K. Oliver, J. Osborne, M. A. Quail, M. A. <br />   1. Akerley, B. J., E. J. Rubin, A. Camilli, D. J. Lampe, H. M. Robertson,              Rajandream, J. Rogers, S. Rutter, K. Seeger, J. Skelton, R. Squares, S. <br />      J. J. Mekalanos. 1998. Systematic identification essential genes vitro         Squares, J. E. Sulston, K. Taylor, S. Whitehead, B. G. Barrell. 1998. <br />      mariner mutagenesis. Proc. Natl. Acad. Sci. USA 95:8927&#226;&#8364;&#8220;8932.                           Deciphering biology Mycobacterium tuberculosis complete <br />   2. Akerley, B. J., E. J. Rubin, V. L. Novick, K. Amaya, N. Judson, J. J.               genome sequence. Nature 393:537&#226;&#8364;&#8220;544. <br />      Mekalanos. 2002. A genome-scale analysis identification genes re-          25.   Cynamon, M. H., T. B. Sorg, A. Patapow. 1988. Utilization me- <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING AND COFACTOR BIOSYNTHESIS                                       4571 <br />  <br />       tabolism NAD Haemophilus parainfluenzae. J. Gen. Microbiol 134:            49. Hutchison, C. A., S. N. Peterson, S. R. Gill, R. T. Cline, O. White, C. M. <br />       2789&#226;&#8364;&#8220;2799.                                                                           Fraser, H. O. Smith, J. C. Venter. 1999. Global transposon mutagenesis <br /> 26.   Dahmen, W., B. Webb, J. Preiss. 1967. The deamido-diphosphopyri-                 minimal mycoplasma genome. Science 286:2165&#226;&#8364;&#8220;2169. <br />       dine nucleotide diphosphopyridine nucleotide pyrophosphorylases           50. Izard, T., A. Geerlof. 1999. The crystal structure novel bacterial <br />       Escherichia coli yeast. Arch. Biochem. Biophys. 120:440&#226;&#8364;&#8220;450.                     adenylyltransferase reveals half sites reactivity. EMBO J. 18:2021&#226;&#8364;&#8220;2030. <br /> 27.   Daugherty, M., B. Polanuyer, M. Farrell, A. Lykidis, V. Cre&#194;&#180;cy-Lagard,        51. Jackowski, S., J. H. Alix. 1990. Cloning, sequence, expression of <br />       A. Osterman. 2002. Complete reconstitution human coenzyme A               pantothenate permease (panF) gene Escherichia coli. J. Bacteriol. <br />       biosynthetic pathway comparative genomics. J. Biol. Chem. 277:21431&#226;&#8364;&#8220;             172:3842&#226;&#8364;&#8220;3848. <br />       21439.                                                                           52. Jenks, P. J., C. Chevalier, C. Ecobichon, A. Labigne. 2001. Identifica- <br /> 28.   Daugherty, M., V. Vonstein, R. Overbeek, A. Osterman. 2001. Archaeal             tion nonessential Helicobacter pylori genes using random mutagenesis <br />       shikimate kinase, new member GHMP-kinase family. J. Bacteriol.              loop amplification. Res. Microbiol. 152:725&#226;&#8364;&#8220;734. <br />       183:292&#226;&#8364;&#8220;300.                                                                     53. Jensen, K. F. 1993. The Escherichia coli K-12 wild types W3110 and <br /> 29.   Debouck, C., J. C. Fedon, D. D. Jaworski, J. Mooney, L. M. Palmer, C. M.             MG1655 rph <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> leads pyrimidine starvation <br />       Traini, M. Wang, R. L. Warren, Y. Y. Zhong. December 2001. RibH.                 low pyrE expression levels. J. Bacteriol. 175:3401&#226;&#8364;&#8220;3407. <br />       U.S. patent 6,326,462.                                                           54. Judson, N., J. J. Mekalanos. 2000. Transposon-based approaches to <br /> 30.   Eddy, S. R. 1998. Profile hidden Markov models. Bioinformatics 14:755&#226;&#8364;&#8220;               identify essential bacterial genes. Trends Microbiol. 8:521&#226;&#8364;&#8220;526. <br />       763.                                                                             55. Kreiswirth, B. N., S. Lofdahl, M. J. Betley, M. O&#226;&#8364;&#8482;Reilly, P. M. Schlievert, <br /> 31.   Emanuelli, M., F. Carnevali, F. Saccucci, F. Pierella, A. Amici, N. Raffaelli,       M. S. Bergdoll, R. P. Novick. 1983. The toxic shock syndrome exotoxin <br />       G. Magni. 2000. Human NMN adenylyltransferase: molecular cloning,                structural gene detectably transmitted prophage. Nature 305: <br />       chromosomal localization, tissue mRNA levels, bacterial expression,              709&#226;&#8364;&#8220;712. <br />       enzymatic properties. J. Biol. Chem. 276:406&#226;&#8364;&#8220;412.                                56. Kreneva, R. A., M. S. Gel&#226;&#8364;&#8482;fand, A. A. Mironov, Y. A. Yomantas, Y. I. Kozlov, <br /> 32.   Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness,            A. S. Mironov, D. A. Perumov. 2000. Inactivation ypaA gene in <br />       A. R. Kerlavage, C. J. Bult, J. F. Tomb, B. A. Dougherty, J. M. Merrick, K.          Bacillus subtilis; analysis resulting phenotypic expression. Russ. <br />       McKenney, G. Sutton, W. Fitzhugh, C. Fields, J. D. Gocayne, J. Scott, R.             J. Genet. 36:972&#226;&#8364;&#8220;974. <br />       Shirley, L. I. Liu, A. Glodek, J. M. Kelley, J. F. Weidman, C. A. Phillips, T.   57. Lerner, F., M. Niere, A. Ludwig, M. Ziegler. 2001. Structural and <br />       Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T.               functional characterization human NAD kinase. Biochem. Biophys. Res. <br />       Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L.              Commun. 288:69&#226;&#8364;&#8220;74. <br />       Fuhrmann, N. S. M. Geoghagen, C. L. Gnehm, L. A. McDonald, K. V.                 58. Link, A. J., D. Phillips, G. M. Church. 1997. Methods generating <br />       Small, C. M. Fraser, H. O. Smith, J. C. Venter. 1995. Whole genome               precise deletions insertions genome wild-type Escherichia coli: <br />       random sequencing assembly Haemophilus influenzae Rd. Science                 application open reading frame characterization. J. Bacteriol. 179:6228&#226;&#8364;&#8220; <br />       269:496&#226;&#8364;&#8220;512.                                                                         6237. <br /> 33.   Flugel, R. S., Y. Hwangbo, R. H. Lambalot, J. E. Cronan, C. T. Walsh.        59. Loferer, H., A. Jacobi, A. Posch, C. Gauss, S. Meier-Ewert, B. Seiz- <br />       2000. Holo-(acyl carrier protein) synthase phosphopantetheinyl trans-            inger. 2000. Integrated bacterial genomics discovery novel anti- <br />       fer Escherichia coli. J. Biol. Chem. 275:959&#226;&#8364;&#8220;968.                                 microbials. Drug Discov. Today 5:107&#226;&#8364;&#8220;114. <br /> 34.   Fraser, C. M., J. D. Gocayne, O. White, M. D. Adams, R. A. Clayton, R. D.        60. Magni, G., A. Amici, M. Emanuelli, N. Raffaelli, S. Ruggieri. 1999. <br />       Fleischmann, C. J. Bult, A. R. Kerlavage, G. Sutton, J. M. Kelley, J. L.             Enzymology NAD&#226;&#171;&#185; synthesis. Adv. Enzymol. Relat. Areas Mol. Biol. <br />       Fritchman, J. F. Weidman, K. V. Small, M. Sandusky, J. Fuhrmann, D.                  73:135&#226;&#8364;&#8220;182. <br />       Nguyen, T. R. Utterback, D. M. Saudek, C. A. Phillips, J. M. Merrick, J. F. <br />                                                                                        61. Magni, G., M. Emanuelli, A. Amici, N. Raffaelli, S. Ruggieri. 1997. <br />       Tomb, B. A. Dougherty, K. F. Bott, P. C. Hu, T. S. Lucier, S. N. Peterson, <br />                                                                                            Purification human nicotinamide-mononucleotide adenylyltransferase. <br />       H. O. Smith, C. A. Hutchison, J. C. Venter. 1995. The minimal gene <br />                                                                                            Methods Enzymol. 280:241&#226;&#8364;&#8220;247. <br />       complement Mycoplasma genitalium. Science 270:397&#226;&#8364;&#8220;403. <br />                                                                                        62. Martin, P. R., R. J. Shea, M. H. Mulks. 2001. Identification a <br /> 35.   Freiberg, C. 2001. Novel computational methods anti-microbial target <br />                                                                                            plasmid-encoded gene Haemophilus ducreyi confers NAD in- <br />       identification. Drug Discov. Today 6:S72&#226;&#8364;&#8220;S80. <br />                                                                                            dependence. J. Bacteriol. 183:1168&#226;&#8364;&#8220;1174. <br /> 36.   Freiberg, C., B. Wieland, F. Spaltmann, K. Ehlert, H. Brotz, H. La- <br />                                                                                        63. Mehl, R. A., C. Kinsland, T. P. Begley. 2000. Identification the <br />       bischinski. 2001. Identification novel essential Escherichia coli genes <br />                                                                                            Escherichia coli nicotinic acid mononucleotide adenylyltransferase gene. J. <br />       conserved pathogenic bacteria. J. Mol. Microbiol. Biotechnol. <br />                                                                                            Bacteriol. 182:4372&#226;&#8364;&#8220;4374. <br />       3:483&#226;&#8364;&#8220;489. <br /> 37.   Galperin, M. Y., E. V. Koonin. 1999. Searching drug targets           64. Meis, R. 2000. EZ::TN transposon insertions target DNA vitro are <br />       microbial genomes. Curr. Opin. Biotechnol. 10:571&#226;&#8364;&#8220;578.                               highly random. Epicentre Forum 7:5. [Online.] http://www.epicentre.com/ <br /> 38.   Gawron-Burke, C., D. B. Clewell. 1984. Regeneration insertionally             forum.asp. <br />       inactivated streptococcal DNA fragments excision transposon             65. Mishra, P. K., P. K. Park, D. G. Drueckhammer. 2001. Identification <br />       Tn916 Escherichia coli: strategy targeting cloning genes          yacE (coaE) structural gene dephosphocoenzyme A kinase in <br />       gram-positive bacteria. J. Bacteriol. 159:214&#226;&#8364;&#8220;221.                                   Escherichia coli K-12. J. Bacteriol. 183:2774&#226;&#8364;&#8220;2778. <br /> 39.   Geerlof, A., A. Lewendon, W. V. Shaw. 1999. Purification charac-         66. Moat, A. G., J. W. Foster. 1987. Biosynthesis salvage pathways of <br />       terization phosphopantetheine adenylyltransferase Escherichia coli.          pyridine nucleotides, p. 1&#226;&#8364;&#8220;20. In D. Dolphin O. Avramovich (ed.), <br />       J. Biol. Chem. 274:27105&#226;&#8364;&#8220;27111.                                                      Pyridine nucleotide coenzymes, B. John Wiley &amp; Sons, New York, <br /> 40.   Goryshin, I. Y., J. Jendrisak, L. M. Hoffman, R. Meis, W. S. Reznikoff.          N.Y. <br />       2000. Insertional transposon mutagenesis electroporation released          67. Moir, D. T., K. J. Shaw, R. S. Hare, G. F. Vovis. 1999. Genomics and <br />       Tn5 transposition complexes. Nat. Biotechnol. 18:97&#226;&#8364;&#8220;100.                             antimicrobial drug discovery. Antimicrob. Agents Chemother. 43:439&#226;&#8364;&#8220;446. <br /> 41.   Goryshin, I. Y., J. A. Miller, Y. V. Kil, V. A. Lanzov, W. S. Reznikoff.     68. Neidhardt, F. C., P. L. Bloch, D. F. Smith. 1974. Culture medium for <br />       1998. Tn5/IS50 target recognition. Proc. Natl. Acad. Sci. USA 95:10716&#226;&#8364;&#8220;              enterobacteria. J. Bacteriol. 119:736&#226;&#8364;&#8220;747. <br />       10721.                                                                           69. Nilsen, T. W. June 2001. Method identifying essential functional <br /> 42.   Grant, S. G. N., J. Jessee, F. R. Bloom, D. Hanahan. 1990. Differential          genes. U.S. patent 6,248,525. <br />       plasmid rescue transgenic mouse DNAs Escherichia coli methy-           70. Olland, A. M., K. W. Underwood, R. M. Czerwinski, M. C. Lo, A. Aula- <br />       lation-restriction mutants. Proc. Natl. Acad. Sci. USA 87:4645&#226;&#8364;&#8220;4649.                 baugh, J. Bard, M. L. Stahl, W. S. Somers, F. X. Sullivan, R. Chopra. <br /> 43.   Hamer, L., T. M. DeZwaan, M. V. Montenegro-Chamorro, S. A. Frank,                2002. Identification, characterization, crystal structure Bacillus sub- <br />       J. E. Hamer. 2001. Recent advances large-scale transposon mutagenesis.            tilis nicotinic acid mononucleotide adenylyltransferase. J. Biol. Chem. 277: <br />       Curr. Opin. Chem. Biol. 5:67&#226;&#8364;&#8220;73.                                                     3698&#226;&#8364;&#8220;3707. <br /> 44.   Hancock, R. E. W., D. Knowles. 1998. Are approaching end           71. Overbeek, R., N. Larsen, G. D. Pusch, M. D&#226;&#8364;&#8482;Souza, E. Selkov, Jr., N. <br />       antibiotic era? Curr. Opin. Microbiol. 1:493&#226;&#8364;&#8220;494.                                Kyrpides, M. Fonstein, N. Maltsev, E. Selkov. 2000. WIT: integrated <br /> 45.   Hare, R. S., S. S. Walker, T. E. Dorman, J. R. Greene, L. M. Guzman, T. J.           high-throughput genome sequence analysis metabolic re- <br />       Kenney, M. C. Sulavik, K. Baradaran, C. Houseweart, H. Y. Yu, Z. Foldes,             construction. Nucleic Acids Res. 28:123&#226;&#8364;&#8220;125. <br />       A. Motzer, M. Walbridge, G. H. Shimer, K. J. Shaw. 2001. Genetic             72. Palmer, L. M., J. M. Pratt, M. Rosenberg. October 2000. Method for <br />       footprinting bacteria. J. Bacteriol. 183:1694&#226;&#8364;&#8220;1706.                               determining gene essentiality pathogen. U.S. patent 6,139,817. <br /> 46.   Heinemann, J. A. 1999. How antibiotics cause antibiotic resistance. Drug         73. Penfound, T., J. W. Foster. 1996. Biosynthesis recycling NAD, <br />       Discov. Today 4:72&#226;&#8364;&#8220;79.                                                               p. 721&#226;&#8364;&#8220;730. In F. C. Neidhardt et al. (ed.), Escherichia coli Salmonella: <br /> 47.   Hoffman, L. M., J. J. Jendrisak, R. J. Meis, I. Y. Goryshin, W. S.               cellular molecular biology. ASM Press, Washington, D.C. <br />       Reznikoff. 2000. Transposome insertional mutagenesis direct sequenc-         74. Raffaelli, N., T. Lorenzi, P. L. Mariani, M. Emanuelli, A. Amici, S. Rug- <br />       ing microbial genomes. Genetica 108:19&#226;&#8364;&#8220;24.                                        gieri, G. Magni. 1999. The Escherichia coli NadR regulator endowed <br /> 48.   Hughes, K. T., D. Ladika, J. R. Roth, B. M. Olivera. 1983. An indis-             nicotinamide mononucleotide adenylyltransferase activity. J. Bacteriol. <br />       pensable gene NAD biosynthesis Salmonella typhimurium. J. Bacte-              181:5509&#226;&#8364;&#8220;5511. <br />       riol. 155:213&#226;&#8364;&#8220;221.                                                               75. Reich, K. A., L. Chovan, P. Hessler. 1999. Genome scanning Hae- <br />  4572        GERDES ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />       mophilus influenzae identification essential genes. J. Bacteriol. 181:    90. Thompson, J. D., D. G. Higgins, T. J. Gibson. 1994. Clustal-W: im- <br />       4961&#226;&#8364;&#8220;4968.                                                                           proving sensitivity progressive multiple sequence alignment through <br /> 76.   Reidl, J., S. Schlor, A. Kraiss, J. Schmidt-Brauns, G. Kemmer, E.                sequence weighting, position-specific gap penalties weight matrix <br />       Soleva. 2000. NADP NAD utilization Haemophilus influenzae. Mol.               choice. Nucleic Acids Res. 22:4673&#226;&#8364;&#8220;4680. <br />       Microbiol. 35:1573&#226;&#8364;&#8220;1581.                                                         91. Timberlake, W., V. Gavrias. November 1999. Identification essential <br /> 77.   Rosamond, L., A. Allsop. 2000. Harnessing power genome             survival genes. U.S. patent 5,976,828. <br />       search new antibiotics. Science 287:1973&#226;&#8364;&#8220;1976.                           92. Tritz, G. J. 1974. Characterization nadR locus Escherichia coli. Can. J. <br /> 78.   Santos, M. A., A. Jimenez, J. L. Revuelta. 2000. Molecular character-            Microbiol. 20:205&#226;&#8364;&#8220;209. <br />       ization FMN1, structural gene monofunctional flavokinase       93. Vallari, D. S., C. O. Rock. 1985. Isolation characterization of <br />       Saccharomyces cerevisiae. J. Biol. Chem. 275:28618&#226;&#8364;&#8220;28624.                            Escherichia coli pantothenate permease (panF) mutants. J. Bacteriol. 164: <br /> 79.   Schmid, M. B. 1998. Novel approaches discovery antimicrobial <br />                                                                                            136&#226;&#8364;&#8220;142. <br />       agents. Curr. Opin. Chem. Biol. 2:529&#226;&#8364;&#8220;534. <br />                                                                                        94. Vallari, D. S., C. O. Rock. 1987. Isolation characterization of <br /> 80.   Schweiger, M., K. Hennig, F. Lerner, M. Niere, M. Hirsch-Kauffmann, T. <br />                                                                                            temperature-sensitive pantothenate kinase (coaA) mutants Escherichia <br />       Specht, C. Weise, S. L. Oei, M. Ziegler. 2001. Characterization of <br />       recombinant human nicotinamide mononucleotide adenylyl transferase                   coli. J. Bacteriol. 169:5795&#226;&#8364;&#8220;5800. <br />       (NMNAT), nuclear enzyme essential NAD synthesis. FEBS Lett.                95. Waibel, A., T. Hanazawa, G. Hinton, K. Shikano, K. J. Lang. 1989. <br />       492:95&#226;&#8364;&#8220;100.                                                                          Phoneme recognition using time-delay neural networks. IEEE Transact. <br /> 81.   Scorpio, A., Y. Zhang. 1996. Mutations pncA, gene encoding                  Acoustics Speech Signal Process. 37:328&#226;&#8364;&#8220;339. <br />       pyrazinamidase/nicotinamidase, cause resistance antituberculous           96. Wang, H., W. Huang, Y.-J. Fei, H. Xia, T. L. Yang-Feng, F. H. Leibach, L. D. <br />       drug pyrazinamide tubercle bacillus. Nat. Med. 2:662&#226;&#8364;&#8220;667.                         Devoe, V. Ganapathy, P. D. Prasad. 1999. Human placental Na&#226;&#171;&#185;- <br /> 82.   Selkov, E., N. Maltsev, G. J. Olsen, R. Overbeek, W. B. Whitman. 1997.           dependent multivitamin transporter. Cloning, functional expression, gene <br />       A reconstruction metabolism Methanococcus jannaschii                  structure, chromosomal location. J. Biol. Chem. 274:14875&#226;&#8364;&#8220;14883. <br />       sequence data. Gene 197:GC11&#226;&#8364;&#8220;GC26.                                               97. Wang, L. L., <br /> </body></html>