<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Role of Pam16&#226;&#8364;&#8482;s degenerate J domain in protein <br /> import across the mitochondrial inner membrane <br />  <br /> Translocation of proteins across the mitochondrial inner mem-               the channel, hence, completing one cycle of the translocation <br /> brane is an essential process requiring an import motor having              process. <br /> mitochondrial Hsp70 (mtHsp70) at its core. The J protein partner of            Pam16 (Tim16), the most recently discovered essential com- <br /> mtHsp70, Pam18, is an integral part of this motor, serving <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>               ponent, displays some similarities with Pam18 (17, 18). Both are <br /> stimulate the ATPase activity of mtHsp70. Pam16, an essential               tightly associated with the inner membrane, presumably via their <br /> protein having an inactive J domain that is unable to stimulate             hydrophobic N-terminal regions. In addition, the C-terminal <br /> mtHsp70&#226;&#8364;&#8482;s ATPase activity, forms a heterodimer with Pam18, but              region of Pam16 has significant sequence similarity to Pam18&#226;&#8364;&#8482;s <br /> its function is unknown. We set out to test the importance of three         C-terminal J domain. However, Pam16 lacks the critical HPD <br /> properties of Pam16: (i) a stable interaction between Pam16 and             motif, having DKE in its place, and does not stimulate Ssc1&#226;&#8364;&#8482;s <br /> Pam18, (ii) the inability of Pam16&#226;&#8364;&#8482;s degenerate J domain to stim-           ATPase activity (19). Pam16 and Pam18 form a complex that has <br /> ulate Ssc1&#226;&#8364;&#8482;s ATPase domain, and (iii) the innately lower stimulatory        a reduced ability to stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity compared <br /> activity of the Pam16:Pam18 heterodimer, compared to Pam18                  to Pam18 alone. This inhibition has been suggested to play an <br /> alone. Neither substantial reduction in the ability of Pam18 to             important role in protein translocation (19, 20). <br /> stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity, nor the presence of an active J              To better understand Pam16&#226;&#8364;&#8482;s function, we asked whether this <br /> domain in Pam16, had deleterious effects on cell growth, indicating         innately lower stimulatory activity of the Pam16:Pam18 het- <br /> the lack of importance of two of these biochemical properties.              erodimer and Pam16&#226;&#8364;&#8482;s inability to stimulate Ssc1&#226;&#8364;&#8482;s ATPase <br /> However, a stable interaction between Pam16&#226;&#8364;&#8482;s degenerate J                  activity were critically important in vivo. We found that neither <br /> domain and Pam18&#226;&#8364;&#8482;s J domain was found to be critical for function.          substantial reduction in the activity of Pam18 to stimulate Ssc1&#226;&#8364;&#8482;s <br /> Alterations that destabilized the Pam16:Pam18 heterodimer had               ATPase activity nor the presence of an active J domain in Pam16 <br /> deleterious effects on cell growth and mitochondrial protein im-            had deleterious effects on cell growth. However, a stable inter- <br /> port; intragenic suppressors that restored robust growth also               action between Pam16&#226;&#8364;&#8482;s degenerate J domain and Pam18&#226;&#8364;&#8482;s J <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      CELL BIOLOGY <br /> restored heterodimer stability. Our results support the idea that           domain was found to be critical for function. <br /> Pam16&#226;&#8364;&#8482;s J-like domain strongly interacts with Pam18&#226;&#8364;&#8482;s J domain, <br /> leading to a productive interaction of Pam18 with mtHsp70 at the <br />                                                                             Materials and Methods <br /> import channel.                                                             Plasmids and Genetic Analysis. A PAM16:PAM18 chimera was <br />                                                                             constructed by introducing an AvrII site at the codons for amino <br /> mitochondria &#229;&#8230;&#169; translocation &#229;&#8230;&#169; Hsp40 &#229;&#8230;&#169; Pam18 &#229;&#8230;&#169; heterodimer                  acids 106&#226;&#8364;&#8220;107 of PAM18 and between amino acids 52 and 53 of <br />                                                                             PAM16 through site-directed mutagenesis using the <br />                                                                             QuikChange protocol (Stratagene). This <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> causes a <br /> M       itochondrial function depends on the efficient import of <br />         hundreds of proteins that are synthesized on cytosolic <br /> ribosomes and then translocated through proteinaceous chan- <br />                                                                             K107R change in Pam18. The N terminus of Pam18 was epitope <br />                                                                             tagged with three tandem copies of the hemagglutinin (HA) <br />                                                                             peptide by introducing a NotI site after PAM18&#226;&#8364;&#8482;s initiation codon <br /> nels into the matrix (1&#226;&#8364;&#8220;3). Typically, N-terminal targeting se- <br />                                                                             and inserting a NotI fragment encoding the HA tag <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> quences are driven across the inner membrane through the <br />                                                                             pGTEPI (21). <br /> presequence translocase (Tim23 complex) by the membrane <br />                                                                                To obtain temperature-sensitive (Ts) mutants, a library of <br /> potential (4&#226;&#8364;&#8220;6). Translocation of the remainder of the protein              random mutations in pRS315-PAM18 (10) was created through <br /> through the Tim23 channel requires the action of the import                 PCR amplification of the coding region of PAM18. &#226;&#338;&#172;pam18 <br /> motor that is associated with the channel at the inner membrane             carrying pRS316-PAM18 was transformed with the library and <br /> (6&#226;&#8364;&#8220;9). Five critical components of this import motor, all of which          incubated at 30&#194;&#176;C on leucine omission plates. Transformants <br /> are highly conserved among eukaryotes, have been identified                 were patched onto 5-fluoroorotic acid plates to select for <br /> (3, 6).                                                                     candidates that could grow in the absence of the wild-type copy <br />    MtHsp70 (Ssc1 in yeast), the core of the import motor,                   of PAM18 and then replicated to leucine omission plates and <br /> interacts with translocating polypeptides in an ATP-dependent               incubated at 37&#194;&#176;C. <br /> process to drive import (7&#226;&#8364;&#8220;9). Like any typical Hsp70, Ssc1                    Suppressors of the Pam18L150W Ts <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span> were selected <br /> functions in the import process with two cochaperones that                  by generating a mutagenized library of pRS315-pam18L150W as <br /> facilitate its reaction cycle. Pam18 (Tim14) serves as its J protein        described above. The haploid, &#226;&#338;&#172;pam18 carrying pRS316- <br /> partner, stimulating ATP hydrolysis and thus, stabilizing the               pam18L150W, was transformed with the library, and the result- <br /> interaction with translocating polypeptide (10&#226;&#8364;&#8220;12). Pam18, like             ing transformants were replica plated onto 5-fluoroorotic acid <br /> all J proteins, contains a J domain with a conserved histidine,             plates and incubated at 37&#194;&#176;C. Suppressors of the Ts phenotype <br /> proline, aspartic acid (HPD) tripeptide that is essential for               caused by the Pam16:Pam18 fusion protein in a &#226;&#338;&#172;pam16 strain <br /> function (10&#226;&#8364;&#8220;12). Mge1 is a nucleotide release factor that <br /> facilitates release of ADP and rebinding of ATP, thus destabi- <br /> lizing the polypeptide interaction (8, 13, 14). Tim44, the fourth           Abbreviations: HA, hemagglutinin; Ts, temperature sensitive; co-IP, coimmunoprecipita- <br /> essential motor component, is a peripheral membrane protein                 tion. <br />  <br /> that serves to tether Ssc1 to the Tim23 translocase until it binds          *P.R.D. and B.S. contributed equally to this work. <br /> to a translocating polypeptide (7&#226;&#8364;&#8220;9, 15, 16). Interaction of Ssc1           &#226;&#8364;&#160;To   whom correspondence should be addressed. E-mail: ecraig@wisc.edu. <br /> with a translocating polypeptide triggers release of Ssc1 from              &#194;&#169; 2005 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0505969102                                                PNAS &#229;&#8230;&#169; August 30, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 35 &#229;&#8230;&#169; 12419 &#226;&#8364;&#8220;12424 <br />  was performed essentially as described above, except that the <br /> template for mutagenesis was the AvrII&#226;&#8364;&#8220;XhoI fragment encod- <br /> ing the Pam18 J domain. In all cases, plasmid DNA was <br /> recovered from Ts transformants and used to transform the <br /> parent strain to verify the phenotype. All in vivo experiments <br /> were carried out in the W303 genetic background in derivatives <br /> of PJ53 (22). <br />  <br /> Protein Purification. Six histidine codons were introduced at the 5&#226;&#172;&#732; <br /> end of genes encoding Pam18 and Pam16:Pam18 and at the 3&#226;&#172;&#732; <br /> end of Pam16. All were cloned into the plasmid pET3a. Over- <br /> expression was carried out in Escherichia coli strain C41 (23) by <br /> allowing growth at 20&#194;&#176;C or 28&#194;&#176;C to an A600 of 0.6, followed by <br /> induction using 0.5 mM IPTG for 4 or 6 h. Protein was purified <br /> by standard affinity chromatography using Ni-NTA agarose. The <br /> Ssc1His (24) was purified from yeast as described. Antibodies <br /> were affinity purified and crosslinked to protein A-Sepharose <br /> beads as described (8). <br />  <br /> Coimmunoprecipitation (co-IP). Purified protein (7 &#226;&#65533;&#174;g) was incu- <br /> bated with Pam18- or Pam16-specific antibody beads in co-IP <br /> buffer (20 mM Mops-KOH, pH 7.4&#229;&#8230;&#190;250 mM sucrose&#229;&#8230;&#190;80 mM <br /> KCl&#229;&#8230;&#190;0.2% Triton X-100&#229;&#8230;&#190;1 mM PMSF) at 4&#194;&#176;C for 1 h. The <br /> Pam16- and Pam18-bound beads were washed three times in <br /> co-IP buffer and blocked with 0.1% BSA at 23&#194;&#176;C for 20 min. <br /> Approximately 90% of Pam18 or Pam16 was immunoprecipi- <br /> tated. For analysis of complex formation, 7 &#226;&#65533;&#174;g of either Pam18 <br /> or Pam16 was added, and the mixture incubated in a final volume <br /> of 200 &#226;&#65533;&#174;l at 23&#194;&#176;C for 30 min. The beads were washed four times <br /> and precipitated material analyzed by SDS&#229;&#8230;&#190;PAGE, followed by <br /> Coomassie staining. <br />    For mitochondria, 1 mg was preincubated at 30&#194;&#176;C or 37&#194;&#176;C for <br /> 15 min to induce the mutant phenotype and lysed on ice by              Fig. 1.    In vitro analysis of Pam16 &#226;&#8364;&#8220;Pam18 complex formation. (A) Size <br /> addition of 1% Triton X-100 in 20 mM Mops-KOH, pH 7.4&#229;&#8230;&#190;250              exclusion chromatography. Three separate reactions were carried out: (i) 50 <br /> mM sucrose&#229;&#8230;&#190;80 mM KCl&#229;&#8230;&#190;5 mM EDTA&#229;&#8230;&#190;1 mM PMSF. Lysates                      &#226;&#65533;&#174;g of Pam18 alone, (ii) 50 &#226;&#65533;&#174;g of Pam16 alone, and (iii) 50 &#226;&#65533;&#174;g of Pam18 and <br /> were centrifuged for 10 min at 20,800 &#226;&#171;&#187; g at 4&#194;&#176;C. Beads (15 &#226;&#65533;&#174;l         Pam16 incubated in size exclusion buffer for 30 min at 23&#194;&#176;C. Reaction mixtures <br /> bed volume) crosslinked to specific antibodies were incubated          were subjected to size exclusion chromatography. Alternate fractions were <br /> with the supernatants for 1 h at 4&#194;&#176;C, and the samples were             analyzed by SDS&#229;&#8230;&#190;PAGE followed by immunoblot analysis using Pam16- and <br />                                                                        Pam18-specific antibodies. Signals were quantified by using densitometry and <br /> processed as described above. In the case of HA-Pam18, the <br />                                                                        plotted as percent of total Pam16 or Pam18 protein. A dashed line is drawn to <br /> lysates were incubated with HA-specific antibody for 2 h before        indicate the peak fraction of Pam16. Note the shift of position of Pam16 in <br /> the co-IP by using protein A-Sepharose.                                presence of Pam18. Positions of migration of molecular standards are indi- <br />                                                                        cated by arrowheads: BSA, 66 kDa; ovalbumin, 45 kDa; lysozyme, 14.3 kDa. (B) <br /> Size Exclusion Chromatography. Superdex 200HR-10&#229;&#8230;&#190;30 was run at         Co-IP. Prebound 2 &#226;&#65533;&#174;M of either Pam16 (16) and&#229;&#8230;&#190;or Pam18 (18), 4 &#226;&#65533;&#174;M of J-like <br /> 0.5 ml&#229;&#8230;&#190;min in Sizing buffer (25 mM Hepes-KOH, pH 7.4&#229;&#8230;&#190;100               domain, amino acids 48 &#226;&#8364;&#8220;148 of Pam16 (16-J) and&#229;&#8230;&#190;or amino acids 101&#226;&#8364;&#8220;168 of <br /> mM KCl&#229;&#8230;&#190;5% glycerol&#229;&#8230;&#190;0.2% Triton X-100) at 4&#194;&#176;C. Fractions (300           Pam18 (18-J) were incubated for 30 min, subjected to immunoprecipitation <br /> &#226;&#65533;&#174;l) were collected between 7 and 30 ml; aliquots from alternate        using indicated antibodies, and analyzed by SDS&#229;&#8230;&#190;PAGE followed by staining <br /> fractions were subjected to SDS&#229;&#8230;&#190;PAGE and immunoblot anal-              with Coomassie blue. A sample containing the input (I) served as a control for <br />                                                                        co-IP efficiency. <br /> ysis using Pam16- and Pam18-specific antibodies. <br />  <br /> Miscellaneous. Affinity-purification of antibodies (8), ATPase         ing to an estimated molecular mass of 39&#226;&#8364;&#8220;40 kDa (Fig. 1 A Lower <br /> assays (8), co-IP of Ssc1-Tim44 complex (9), mitochondria              Left). These estimated molecular masses are consistent with <br /> purification (24), and in vitro import assays (25, 26) were carried    formation of homodimers by both Pam16 and Pam18. <br /> out as described. For import assays, mitochondria isolated from <br />                                                                          Previous analyses indicated that Pam16 and Pam18 could form <br /> cells grown at 30&#194;&#176;C were preincubated at 37&#194;&#176;C for 10 min to <br />                                                                        a heterodimer (18, 19). Consistent with these findings, we found <br /> induce the mutant phenotype before the addition of precursor <br /> protein at 30&#194;&#176;C. Immunoblot analysis was carried out by using          that upon mixing equimolar amounts of full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> Pam16 and <br /> the ECL system (Amersham Pharmacia) according to the man-              Pam18, &#226;&#172;&#381;75% of Pam16 was coimmunoprecipitated with anti- <br /> ufacturer&#226;&#8364;&#8482;s instructions.                                              bodies specific for Pam18 (Fig. 1B). Conversely, &#226;&#172;&#381;75% of Pam18 <br />                                                                        could be coimmunoprecipitated with antibodies specific for <br /> Results                                                                Pam16. Thus, Pam16 and Pam18 efficiently formed heteromeric <br /> Pam16 and Pam18 Can Form Homodimers. In mitochondria depleted          complexes. To assess the size of these heterodimeric complexes, <br /> of Pam16, Pam18 can be cross-linked to itself (17), suggesting         mixtures of Pam18 and Pam16 were separated by gel filtration. <br /> that Pam18 might be able to form homodimers. We assessed the           The elution of Pam16 and Pam18 was coincident (Fig. 1 A Right) <br /> quaternary structure of purified Pam18 by using size-exclusion         at a position corresponding to a molecular size of 43&#226;&#8364;&#8220;45 kDa. <br /> chromatography. Pam18, a 18-kDa protein, eluted in fractions           This estimated molecular mass is consistent with that of a <br /> 24&#226;&#8364;&#8220;30, corresponding to an estimated molecular mass of 43&#226;&#8364;&#8220;45           Pam18:Pam16 heterodimer. Based on these co-IP and chromato- <br /> kDa (Fig. 1A Upper Left). Under similar conditions, Pam16, a           graphic results, we conclude that, although Pam16 and Pam18 <br /> 16-kDa protein, eluted primarily in fractions 28&#226;&#8364;&#8220;34, correspond-       can form homodimers, heterodimers form preferentially. These <br />  <br /> 12420 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0505969102                                                                                     D&#226;&#8364;&#8482;Silva et al. <br />                                                                                    Simply replacing DKE with HPD was not sufficient to convert <br />                                                                                   Pam16&#226;&#8364;&#8482;s J-like domain to an active J domain (ref. 19 and data not <br />                                                                                   shown). Therefore, we replaced Pam16&#226;&#8364;&#8482;s J-like domain with <br />                                                                                   Pam18&#226;&#8364;&#8482;s functional J domain, forming a chimeric protein, <br />                                                                                   Pam16:Pam18, composed of the N-terminal 52 aa of Pam16 and <br />                                                                                   the J domain of Pam18 (amino acid s106&#226;&#8364;&#8220;168; Fig. 2 A). We <br />                                                                                   purified Pam16:Pam18 to determine whether it could, in fact, <br />                                                                                   stimulate the ATPase activity of Ssc1. The fusion was 70% as <br />                                                                                   active as Pam18 when added to preformed radiolabeled ATP- <br />                                                                                   Ssc1 complexes. When present at a 2.5-fold excess over Ssc1, <br />                                                                                   Pam18 and the fusion protein stimulated the ATPase activity of <br />                                                                                   Ssc1 4- and 2.8-fold, respectively. <br />                                                                                      We then tested whether the fusion could substitute for Pam16 <br />                                                                                   in vivo. &#226;&#338;&#172;pam16 cells expressing Pam16:Pam18 (&#226;&#338;&#172;pam16 <br />                                                                                   [Pam16:Pam18]) at normal levels were viable, growing as well as <br />                                                                                   wild-type cells at 30 and 34&#194;&#176;C (Fig. 2B). Thus, presence of the <br />                                                                                   J-like domain of Pam16 is not uniquely required for viability. <br />                                                                                   However, &#226;&#338;&#172;pam16 [Pam16:Pam18] did not grow at 37&#194;&#176;C. This Ts <br />                                                                                   growth could indicate that the presence of an active J domain in <br />                                                                                   Pam16 had deleterious consequences. To test whether this <br />                                                                                   stimulatory activity of the chimeric protein affected growth <br />                                                                                   in vivo, we mutated the DNA encoding the Pam18 HPD trip- <br />                                                                                   eptide such that a glutamine would replace the histidine, an <br />                                                                                   alteration we had previously shown to obviate the ability of <br />                                                                                   Pam18 to stimulate the ATPase activity of Ssc1 and rescue the <br />                                                                                   inviability of &#226;&#338;&#172;pam18 cells (10). However, the H&#229;&#8230;&#190;Q alteration in <br />                                                                                   the chimeric protein did not improve growth of the &#226;&#338;&#172;pam16 <br />                                                                                   strain (Fig. 2B). We conclude that the inability of the degenerate <br /> Fig. 2. Analysis of Pam16:Pam18 chimera and Pam18 J domain mutants. (A) <br />                                                                                   J domain of Pam16 to act as a J domain is unimportant to <br /> Predicted domains of Pam16, Pam18, and Pam16:Pam18 (16:18) based on               Pam16&#226;&#8364;&#8482;s essential in vivo function. <br /> sequence alignment. The amino acids corresponding to predicted regions are <br />  <br />  <br />  <br />  <br />                                                                                                                                                            CELL BIOLOGY <br /> highlighted. Regions: intermembrane space region (IM), membrane associa-          Pam18 with Significantly Reduced Stimulatory Activity Is Functional in <br /> tion region (M), predicted mitochondria targeting region (T), and J domain (J).   Vivo. The J domain of Pam18 is less active in stimulating Ssc1&#226;&#8364;&#8482;s <br /> (B and C) Growth phenotype. Tenfold serial dilutions of &#226;&#338;&#172;pam16 or &#226;&#338;&#172;pam18          ATPase activity when in complex with a fragment of Pam16 (19). <br /> cells expressing the indicated genes were spotted onto minimal media and          In agreement with these results, we found that full-length <br /> incubated at the indicated temperatures for 2 days. (B) &#226;&#338;&#172;pam16 cells express-     Pam16:Pam18 heterodimer was 50&#226;&#8364;&#8220;60% as active as Pam18 <br /> ing Pam16 (WT), Pam16:Pam18 (16:18), or Pam16:Pam18H141Q (16:18H141Q). (C) <br />                                                                                   alone in stimulating Ssc1&#226;&#8364;&#8482;s ATPase activity over a range of <br /> &#226;&#338;&#172;pam18 cells expressing pam18 (WT), Pam18P142A (18-P&#229;&#8230;&#190;A), and Pam18K144Q <br /> (18-K&#229;&#8230;&#190;Q). (D) Stimulation of Ssc1 ATPase activity. Ssc1:[&#226;&#65533;&#163;-32P]ATP complex <br />                                                                                   protein concentrations (Fig. 2D). To begin to assess whether this <br /> (0.25 &#226;&#65533;&#174;M) was incubated at 23&#194;&#176;C in the presence of indicated concentrations       inhibition by Pam16 of Pam18&#226;&#8364;&#8482;s stimulatory capacity is an <br /> of Pam18 or Pam18 mutants: P142A (P&#229;&#8230;&#190;A), K144Q (K&#229;&#8230;&#190;Q), D143N (D&#229;&#8230;&#190;N), and&#229;&#8230;&#190;or          important part of Pam16&#226;&#8364;&#8482;s function, as suggested (19), we wanted <br /> Pam16. The rate of ADP formation was measured and plotted as the fold             to determine whether a 50&#226;&#8364;&#8220;60% reduction in the stimulatory <br /> stimulation over the basal rate determined in the absence of any additional       ability of Pam18 had consequencess in vivo. Therefore, we <br /> component (set at one) versus the ratio of the Pam protein relative to Ssc1       constructed several PAM18 mutants encoding single amino acid <br /> concentration (molar excess).                                                     alterations in or near the crucial HPD tripeptide: P142A, <br />                                                                                   D143N, and K144Q. Pam18D143N, like the previously analyzed <br />                                                                                   Pam18H141Q, was neither able to rescue the inviability of &#226;&#338;&#172;pam18 <br /> results are consistent with Pam16 and Pam18 having a higher                       cells nor measurably stimulate the ATPase activity of Ssc1 (Fig. <br /> affinity for each other than for themselves.                                      2D). However, pam18P142A and pam18K144Q cells grew as well as <br />    To determine whether the J and J-like domains of the two                       wild-type cells under a variety of conditions (Fig. 2C and data not <br /> proteins are sufficient for heterodimer formation, we mixed                       shown). Protein import into mitochondria of these mutants was <br /> fragments containing amino acids 48&#226;&#8364;&#8220;149 of Pam16 and amino                        also unaffected (Fig. 5, which is published as supporting infor- <br /> acids 101&#226;&#8364;&#8220;168 of Pam18 together (Fig. 2A) and performed                           mation on the PNAS web site). On the other hand, the ability of <br /> immunoprecipitation with Pam18- and Pam16-specific antibod-                       Pam18P142A and Pam18K144Q to stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity <br /> ies. As with full-length protein, these smaller fragments con-                    was reduced &#226;&#172;&#8225;70 and 60%, respectively (Fig. 2D). As expected, <br /> taining the J domains coimmunoprecipitated (Fig. 1B). Pam16                       formation of heterodimers between the mutant Pam18 proteins <br /> contains a 30-aa C-terminal segment beyond the J-like domain.                     and Pam16 resulted in further reduction in ATPase stimulation <br /> A fragment containing only the J-like domain, amino acids                         (Fig. 6, which is published as supporting information on the <br /> 48&#226;&#8364;&#8220;119, could also be coprecipitated with the Pam18 J domain                      PNAS web site). Together, these results indicate that a decrease <br /> (data not shown). Thus, the J domains are sufficient for het-                     in the ability of Pam18 to stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity is well <br /> erodimer formation.                                                               tolerated. We conclude that the extent of the decrease in ATPase <br />                                                                                   stimulation caused by the interaction of Pam18 with Pam16 is not <br /> The Absence of a Functional J Domain in Pam16 Is Not Functionally                 sufficient to have significant effects on mitochondrial protein <br /> Important. Pam16 is not capable of stimulating Ssc1&#226;&#8364;&#8482;s ATPase                      import or cell growth. <br /> activity and thus, is not a functional J protein partner of Ssc1 <br /> (19). However, whether the absence of this activity is an impor-                  Suppression of the Ts Phenotype of &#226;&#338;&#172;pam16 [Pam16:Pam18]. We next <br /> tant aspect of Pam16&#226;&#8364;&#8482;s function is not known. To address this                     tested the hypothesis that heterodimer formation between <br /> question, a Pam16 molecule containing an active J domain                          Pam16 and Pam18 is critical in vivo for transport of proteins into <br /> capable of stimulating Ssc1&#226;&#8364;&#8482;s ATPase activity was needed.                         mitochondria. To approach this question, we used the <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                      PNAS &#229;&#8230;&#169; August 30, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 35 &#229;&#8230;&#169; 12421 <br />                                                                                    Pam16:Pam18 chimera. Extracts of detergent solubilized mito- <br />                                                                                   chondria from cells expressing this tagged Pam18, which sup- <br />                                                                                   ports growth like wild-type Pam18 in vivo (data not shown), were <br />                                                                                   subjected to immunoprecipitation using HA-specific antibodies. <br />                                                                                   As expected from previous results (17, 18), Pam16 was coim- <br />                                                                                   munoprecipitated with the tagged Pam18. However, in mito- <br />                                                                                   chondrial lysates from cells expressing the Pam16:Pam18 chi- <br />                                                                                   mera in place of wild-type Pam16, no coimmunoprecipitation <br />                                                                                   was observed (Fig. 3B, data not shown). <br />                                                                                      This apparent decrease in stability of the chimera and the <br />                                                                                   defect in protein translocation is consistent with the idea that the <br />                                                                                   import and growth defects of cells expressing the fusion protein <br />                                                                                   are caused by the instability of the heterodimer. To test this idea, <br />                                                                                   we carried out an intragenic suppressor analysis of &#226;&#338;&#172;pam16 <br />                                                                                   [Pam16:Pam18], selecting for growth at 37&#194;&#176;C after mutagenesis <br />                                                                                   of the DNA encoding the J domain of the chimera. Two <br />                                                                                   suppressors encoding single amino acid alterations within the J <br />                                                                                   domain were isolated: T152A and V132A (Fig. 4B). Both <br />                                                                                   partially suppressed the growth defect at 37&#194;&#176;C caused by the <br />                                                                                   chimera (Fig. 3C). Mitochondria isolated from the suppressor <br />                                                                                   strains imported Cytb2(47)-DHFR nearly as efficiently as wild- <br />                                                                                   type mitochondria, indicating that the amino acid alterations in <br />                                                                                   the J domain suppressed both growth and translocation defects <br />                                                                                   imparted by the chimeric protein (Fig. 3A). To assess the <br />                                                                                   interaction between Pam18 and the chimera containing the <br />                                                                                   suppressor amino acid substitution, co-IP assays were carried <br />                                                                                   out. Chimeric suppressor protein was coimmunoprecipitated <br />                                                                                   with Pam18, although not as efficiently as wild-type Pam16 (Fig. <br /> Fig. 3. Analysis of intragenic suppressors of PAM16:PAM18 phenotype. (A) In       3B), indicating that the suppressor mutations caused a partial <br /> vitro import. Cytb2(47)-DHFR was added to 50 &#226;&#65533;&#174;g of preincubated indicated         restoration of a stable interaction between Pam18 and the <br /> mitochondria at 37&#194;&#176;C for 15 min. Reaction mixtures were separated by SDS&#229;&#8230;&#190;PAGE     chimera. <br /> and imported i-Cytb2(47)-DHFR was quantified after immunoblotting using anti- <br /> DHFR antibodies. Import in WT mitochondria after 30 min was set to 100%. (B and   Amino Acid Substitutions in J Domain of Pam18 Destabilize the <br /> C) The following genes were used: WT, pam16:pam18 (16:18), pam16:pam18-           Pam16:Pam18 Interaction. The data presented above supports the <br /> T152A (16:18-T152A), and pam16:pam18-V132A (16:18-V132A). (B) In organellar <br />                                                                                   idea that the interaction between Pam18 and Pam16 is important <br /> analysis of Pam18 &#226;&#8364;&#8220;Pam16 complex. Mitochondria from cells expressing Pam18 <br /> with an HA-tag were lysed and subjected to co-IP using HA-specific antibodies. <br />                                                                                   for function of the translocation machinery. However, to obtain <br /> Immunoblotting was carried out by using Pam16-specific (Upper) or Pam18-          independent confirmation of this idea, we screened for Ts muta- <br /> specific (Lower) antibodies. (C) Growth phenotypes. Tenfold serial dilutions of   tions in PAM18. One PAM18 mutant was isolated that grew like <br /> &#226;&#338;&#172;pam16 cells expressing indicated genes were spotted onto minimal medium and      wild-type at the optimum growth temperature of 30&#194;&#176;C, but poorly <br /> incubated at the indicated temperatures for 2 days.                               at 37&#194;&#176;C (Fig. 4A). The growth defect was caused by a single amino <br />                                                                                   acid change in the J domain, L150W, at a site predicted to be at the <br />                                                                                   beginning of helix III, near the opposite end of the loop from the <br /> Pam16:Pam18 chimera containing the Pam18 J domain for our                         HPD tripeptide (Fig. 4B). To test for defects in protein import in <br /> initial analysis. This chimera was chosen because &#226;&#338;&#172;pam16                          pam18(L150W) mitochondria, henceforth called pam18L&#229;&#8230;&#190;W, <br /> [Pam16:Pam18] cells are Ts for growth, indicating that the                        in vitro import experiments were conducted. Import of Cytb2(47)- <br /> Pam18 J domain does not completely substitute for the Pam16                       DHFR was substantially reduced in pam18L&#229;&#8230;&#190;W mitochondria <br /> J-like domain. Although J domains of Pam16 and Pam18 are                          compared to wild-type (Fig. 4C). <br /> sufficient for dimer formation, the Pam18 homodimer is less                          To test for biochemical defects that could be responsible for <br /> stable than the Pam16:Pam18 heterodimer. We reasoned that                         the reduced import efficiency, we purified the mutant protein <br /> the Ts growth could be due to the less stable interaction between                 and tested its ability to stimulate the ATPase activity of Ssc1. <br /> the chimera and Pam18 that presumably heterodimerize by                           Pam18L150W was &#226;&#172;&#381;75% as active as wt Pam18. For example, at <br /> interaction between the two Pam18 J domains.                                      a 2.5-fold excess of Pam18 over Ssc1, Pam18L150W and wild-type <br />    As a first step, we analyzed the chimera more thoroughly.                      Pam18 stimulated Ssc1 3.1- and 4-fold, respectively. This level of <br /> Making use of an in organellar translocation assay, we asked                      activity is significantly above that required for both normal <br /> whether protein import into the matrix of the mitochondria of                     growth and rates of protein import, as indicated by the results of <br /> &#226;&#338;&#172;pam16 [Pam16:Pam18] cells was defective. Saturating amounts                      the experiments reported above. <br /> of purified Cytb2(47)-DHFR, a fusion protein in which the                            We also assessed the Pam18:Pam16 complex in mitochondrial <br /> matrix targeting sequence of cytochrome b2 is fused to dihydro-                   extracts. Co-IP of Pam16 with Pam18L150W was significantly less <br /> folate reductase (DHFR), was added to &#226;&#338;&#172;pam16 [Pam16:Pam18]                        than co-IP of wild-type Pam18. Interactions between other <br /> mitochondria that had been preincubated at 37&#194;&#176;C to induce a                       components of the import motor, Ssc1 and Tim44, and Ssc1 and <br /> mutant phenotype. Thirty minutes after addition of saturating                     Mge1, were also tested; no defects in these interactions were <br /> amounts of precursor protein, only 20% as much DHFR was                           found in mutant mitochondria (Fig. 7, which is published as <br /> imported into &#226;&#338;&#172;pam16 [Pam16:Pam18] mitochondria compared                          supporting information on the PNAS web site). To more directly <br /> to wild-type mitochondria (Fig. 3A).                                              analyze heterodimer formation, purified proteins were used. <br />    Next, we tested the stability of the Pam18:chimera complex                     Under the conditions tested, 7-fold less Pam18L150W was found <br /> in vivo. To carry out this test, we first generated a HA-tagged                   in complex with Pam16 than wild-type Pam18 (Fig. 4E). In sum, <br /> Pam18 to allow us to distinguish between Pam18 and the                            both in organellar and in vitro experiments indicate decreased <br />  <br /> 12422 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0505969102                                                                                       D&#226;&#8364;&#8482;Silva et al. <br />                                                                                     grew like wild-type at 34&#194;&#176;C, but slowly at 37&#194;&#176;C, were identified: <br />                                                                                    pam18L&#229;&#8230;&#190;W-V132A and pam18L&#229;&#8230;&#190;W-T152A (Fig. 4A). These <br />                                                                                    amino acid alterations are the same as those able to suppress <br />                                                                                    the temperature sensitivity of &#226;&#338;&#172;pam16 cells expressing <br />                                                                                    the Pam16:Pam18 chimera. V132 and T152 are predicted to <br />                                                                                    lie within helix II and helix III of the J domain, respectively <br />                                                                                    (Fig. 4B). <br />                                                                                       To determine whether the suppressor mutations compensated <br />                                                                                    for the import defect of pam18L&#229;&#8230;&#190;W, we carried out in organellar <br />                                                                                    import assays using saturating amounts of Cytb2(47)-DHFR. <br />                                                                                    Import was substantially improved in both suppressors com- <br />                                                                                    pared to pam18L&#229;&#8230;&#190;W (Fig. 4C). To test whether the defect in <br />                                                                                    heterodimer formation was also suppressed, we performed co-IP <br />                                                                                    analysis using lysates of pam18L&#229;&#8230;&#190;W-T152A and pam18L&#229;&#8230;&#190;W- <br />                                                                                    V132A mitochondria. More Pam16 was coimmunoprecipitated <br />                                                                                    by Pam18-specific antibodies from lysates of suppressors than <br />                                                                                    from lysates of pam18L&#229;&#8230;&#190;W; however, less Pam16 was coimmu- <br />                                                                                    noprecipitated than from wild-type lysates (Fig. 4D), consistent <br />                                                                                    with the partial suppression of the growth defect. In addition, the <br />                                                                                    purified suppressor Pam18 proteins were more efficiently coim- <br />                                                                                    munoprecipitated with Pam16, than was Pam18L150W (Fig. 4E). <br />                                                                                    In sum, the suppressor mutations that partially correct both the <br />                                                                                    growth and translocation defect of pam18L&#229;&#8230;&#190;W cells also partially <br />                                                                                    restore stability of the Pam16:Pam18 heterodimer. <br />  <br />                                                                                    Discussion <br />                                                                                    The idea that the ability of J proteins to stimulate the ATPase <br />                                                                                    activity of their partner Hsp70s is critically important is both well <br />                                                                                    accepted and substantiated. However, the results presented here <br />                                                                                    demonstrate that, at least in some cases, decreased stimulatory <br />                                                                                    capacity can be tolerated with little if any effects on cell growth. <br />                                                                                    A Pam18 mutant protein having only 30% of wild-type activity <br />  <br />  <br />  <br />  <br />                                                                                                                                                            CELL BIOLOGY <br />                                                                                    supported vigorous growth under all conditions tested. Whether <br />                                                                                    such reduction in activity of other J proteins would be as well <br />                                                                                    tolerated is not clear, because, to our knowledge, no studies <br />                                                                                    correlating stimulatory ability of J proteins and in vivo function <br />                                                                                    have previously been reported. Such robustness may be re- <br />                                                                                    stricted to cases such as Pam18 and Ssc1 in which both compo- <br />                                                                                    nents are tethered to their site of action, resulting in extremely <br />                                                                                    high local concentrations. Consistent with this idea, the ability of <br />                                                                                    mutant Pam18 to enhance Ssc1&#226;&#8364;&#8482;s ATPase activity approached <br />                                                                                    that of wild-type at higher concentrations, at which point the <br />                                                                                    stimulatory capacity of WT began to plateau. <br />                                                                                       The results presented here support the idea that Pam18 and <br />                                                                                    Pam16 interact via their J and J-like domains. To our knowledge, <br />                                                                                    the idea of two proteins dimerizing via their J domains is <br /> Fig. 4.      Analysis of pam18(L150W) and its intragenic suppressors. <br />                                                                                    unprecedented. Several J proteins are homodimers, but interact <br /> pam18(L150W), 18L&#229;&#8230;&#190;W; pam18(L150W-T152A), 18L&#229;&#8230;&#190;W-T152A; pam18(L150W- <br /> V132A), 18L&#229;&#8230;&#190;W-V132A. (A) Growth phenotypes. Tenfold serial dilutions of <br />                                                                                    via sequences distinct from their J domains. For example, Sis1 <br /> &#226;&#338;&#172;pam18 expressing indicated PAM18 genes were spotted onto rich media and           and Ydj1 of the yeast cytosol have been shown to homodimerize <br /> incubated at indicated temperatures for 2 days. (B) Model of Pam18 J domain.       via C-terminal coiled-coils located at their C termini, far from <br /> Amino acid sequence corresponding to the J domain of Pam18 (108 &#226;&#8364;&#8220;164) was          the N-terminal J domain (27, 28). Although E. coli DnaJ is <br /> modeled by using coordinates of HDJ1 (34) and drawn by using Micron Separa-        dimeric (29), extensive analysis indicates that its J domain is <br /> tions Web Lab VIEWERPRO software. Relevant amino acids are highlighted. (C) In     monomeric (30, 31). However, interaction between Pam16 and <br /> vitro import. Import reactions were carried out as in Fig. 3B. (D) In organellar   Pam18 through their J-like domains is supported by both bio- <br /> analysis of Pam18 &#226;&#8364;&#8220;Pam16 complex formation. A total of 25% of lysate was           chemical and genetic analyses. First, fragments of Pam16 and <br /> loaded as a control (25% I). (E) In vitro co-IP analysis of complex formation. <br />                                                                                    Pam18 containing only their J domains are competent to form <br /> Proteins (2 &#226;&#65533;&#174;M) were incubated for 30 min at 23&#194;&#176;C: Pam16 (16), Pam18 (18), Pam18 <br /> L150W (18L&#229;&#8230;&#190;W), Pam18L150W-V132A (18L&#229;&#8230;&#190;W-V132A), and Pam18L150W-T152A <br />                                                                                    a complex. Second, a single amino acid alteration within Pam18&#226;&#8364;&#8482;s <br /> (18L&#229;&#8230;&#190;W-T152A). Co-IPs were carried out by using Pam18-specific antibodies.         J domain destabilizes the heterodimer, a defect that can be <br /> Samples were analyzed by SDS&#229;&#8230;&#190;PAGE and stained with Coomassie blue. As a            suppressed by a second change in the J domain. In addition, both <br /> control (C), Pam18 and mutants were omitted. All (100%) of the input was           Pam18 and Pam16 form homodimers in vitro, consistent with <br /> loaded.                                                                            increased cross linking of Pam18 to itself upon depletion of <br />                                                                                    Pam16 and vice versa (17). <br /> stability of the Pam16:Pam18L150W heterodimer compared to                             Our data also support the idea that this interaction between <br /> wild-type heterodimer.                                                             Pam16 and Pam18 is critical in vivo. In both the case of the <br />                                                                                    Pam16:Pam18 chimera and the Pam18 Ts mutant, Pam18L150W, <br /> Intragenic Suppressors of PAM18 Ts Mutations Compensate for the                    the stability of the heterodimer was reduced compared to the <br /> Import Defect. We selected for intragenic suppressors of the                       wild-type Pam16:Pam18 interaction. Intragenic suppressors of <br /> growth defect of pam18L&#229;&#8230;&#190;W. Two suppressors, both of which                          the growth defect not only corrected the translocation defect, but <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                      PNAS &#229;&#8230;&#169; August 30, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169; no. 35 &#229;&#8230;&#169; 12423 <br />  also partially restored stability of the dimer. Although it is                           protein import. The Pam16:18 heterodimer could toggle be- <br /> conceivable that the suppressor mutations revert some unknown                            tween having an active, Pam18, and inactive, Pam16, J domain <br /> biochemical property of the mutant protein that is crucial for                           juxtaposed to Ssc1 depending on whether a translocating <br /> function, our data are most consistent with the idea that a stable                       polypeptide is in the import channel and the ATPase activity of <br /> interaction between Pam16 and Pam18 is critical for mitochon-                            Ssc1 needs to be stimulated. Other types of regulatory scenarios <br /> drial protein import and thus cell growth.                                               envisioned are much less likely in light of the data presented in <br />    Combining the data reported here with earlier results, a                              this report. For example, any model based on the innately lower <br /> picture emerges of a heterodimer of Pam16 and Pam18 formed                               stimulatory activity of the Pam18:Pam16 heterodimer compared <br /> by two distantly related proteins interacting through their most                         to Pam18 alone or the absence of an active J domain in Pam16 <br /> highly related sequences, their J-like domains. We think it likely                       is unlikely because an active J domain supports Pam16 function <br /> that PAM16 and PAM18, which are present in higher eukaryotes,                            as well as an inactive one and robust growth, even at 37&#194;&#176;C, is <br /> evolved from a single progenitor PAM gene, which encoded a                               supported by a mutant Pam18 J domain that is only 30% as active <br /> protein that formed a homodimer that functioned in the import                            as wild-type Pam18; thus, Pam16&#226;&#8364;&#8482;s binding to Pam18, which <br /> process. However, upon gene duplication, two specialized pro-                            decreases activity &#226;&#172;&#8225;50%, would not be sufficient for significant <br /> teins coevolved in a manner consistent with the subfunctional-                           regulation of activity. <br /> ization model of protein evolution (32, 33). One of them, the                               In summary, our data are most consistent with the idea that <br /> ortholog of Pam18, maintained an active J domain. The other,                             Pam16&#226;&#8364;&#8482;s degenerate J domain serves the critical function of <br /> the ortholog of Pam16, lost its functionality as a J protein, but                        forming a stable interaction with Pam18 and to position Pam18 <br /> its degenerate J domain evolved to stably interact with Pam18&#226;&#8364;&#8482;s                          precisely at the channel for optimal activity. It is quite possible <br /> J domain.                                                                                that such interactions and possible conformational changes <br />    Although the data presented here provides evidence for the                            caused by the presence of translocating polypeptides could serve <br /> functional importance of the Pam16 J-like domain, the function                           to alter the positioning of Pam18&#226;&#8364;&#8482;s J domain at the translocon, <br /> of Pam16&#226;&#8364;&#8482;s N terminus, and thus, the heterodimer as a whole,                             thus dramatically affecting its ability to stimulate Ssc1. Critical <br /> remains to be clarified. Although Pam18 itself is a transmem-                            testing of such models awaits further reconstitution of the import <br /> brane protein (10&#226;&#8364;&#8220;12), we think that the most likely function of                         machinery. <br /> the N terminus of Pam16 is to precisely position Pam18 in <br /> juxtaposition to Ssc1 at the translocon to maximize ATPase                             <br /> stimulation. The N terminus of Pam16, the region required for                          <br /> its membrane association, is essential (data not shown). The N                         <br /> terminus of Pam16 may also serve a regulatory function during                       <br />  <br /> </body></html>