<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Systems Biology 3; Article number 104; doi:10.1038/msb4100144 <br /> Citation: Molecular Systems Biology 3:104 <br /> &amp; 2007 EMBO Nature Publishing Group All rights reserved 1744-4292/07 <br /> www.molecularsystemsbiology.com <br />  <br /> REPORT <br />  <br /> Pathway redundancy protein essentiality revealed <br /> in Saccharomyces cerevisiae interaction networks <br /> Igor Ulitsky Ron Shamir* <br />  <br /> School Computer Science, Tel Aviv University, Tel Aviv, Israel <br /> * Corresponding author. School Computer Science, Tel Aviv University, Tel Aviv 69978, Israel. Tel.: &#195;&#190; 972 3 6405383; Fax: &#195;&#190; 972 3 6405384; <br /> E-mail: rshamir@tau.ac.il <br />  <br /> Received 8.1.07; accepted 11.2.07 <br />  <br />  <br />  <br />  <br />                The biological interpretation genetic interactions major challenge. Recently, Kelley and <br />                Ideker proposed method analyze genetic physical networks, explains many <br />                known genetic interactions linking different pathways physical network. Here, we <br />                extend method devise novel analytic tools interpreting genetic interactions physical <br />                context. Applying tools large-scale Saccharomyces cerevisiae data set, analysis <br />                reveals 140 between-pathway models explain 3765 genetic interactions, roughly doubling those <br />                previously explained. Model genes tend short mRNA half-lives many <br />                phosphorylation sites, suggesting stringent regulation linked pathway redundancy. <br />                We identify &#226;&#8364;&#732;pivot&#226;&#8364;&#8482; proteins physical interactions pathways our <br />                models, pivots tend essential highly conserved. Our analysis models and <br />                pivots sheds light organization cellular machinery roles individual <br />                proteins. <br />                Molecular Systems Biology 17 April 2007; doi:10.1038/msb4100144 <br />                Subject Categories: metabolic regulatory networks; computational methods <br />                Keywords: essential genes; genetic interactions; pathway analysis; protein interactions; S. cerevisiae <br />  <br />  <br />  <br />  <br /> Introduction                                                                     interact physically (Tong et al, 2001, 2004), protein <br />                                                                                  PIs likely GIs (Ozier et al, <br /> Gene <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>knockout</span> studies shown B18%                               2003). <br /> Saccharomyces cerevisiae genes essential growth                        The modular nature cellular organization been <br /> rich medium (Giaever et al, 2002). Consequently, buffering                    widely recognized (Hartman et al, 2001). Many methods <br /> the genetic level believed abundant eukaryotes                       developed detecting functional modules <br /> (Hartman et al, 2001). <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> better understand role                          PI networks. Such modules, termed pathways, <br /> nonessential genes, large-scale studies performed                        represent physically interacting proteins involved carrying <br /> double knockouts (Pan et al, 2004; Tong et al, 2004)                         particular function. Depending detection <br /> identified events synthetic lethality, mutant                    method, pathways represent molecular complexes <br /> carrying deletions nonessential genes lethal,                      (Bader Hogue, 2003) signaling cascades (Rives and <br /> synthetic sickness, mutant shows weaker                              Galitski, 2003). Kelley Ideker (2005) defined pathway <br /> <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span>. We use term genetic interaction (GI)                    group proteins densely interconnected the <br /> for interaction genetic perturbations affecting                PI network, studied frequency GIs and <br /> phenotype, lethal sick. The graph genes                   pathways. In systematic analysis large-scale <br /> nodes edges corresponding GIs called GI network.                   GI PI data, concluded between-pathway <br />    Recent technologies enable systematic mapping                       explanations GIs B3.5 times abundant than <br /> protein&#226;&#8364;&#8220;protein (Ito et al, 2001) protein&#226;&#8364;&#8220;DNA (Lee et al,                    within-pathway explanations, concluded GIs mostly <br /> 2002) interactions (physical interactions (PIs)), yielding large                 bridge redundant processes. Further arguments the <br /> PI networks. As networks larger, need                            prevalence between-pathway GIs given Ye et al <br /> computational tools dissecting mounting. The                         (2005), postulated genes pathway are <br /> integrated analysis PI GI networks compelling                        expected share common GI partners, used similarity of <br /> challenge, carry important complementary                             GI patterns successful function prediction. Similar results <br /> biological signals. Initial studies shown proteins                  established recently DNA damage (Pan <br /> the region GI network slightly likely                    et al, 2006). <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                     Molecular Systems Biology 2007 1 <br />  Pathway redundancy essential proteins <br /> I Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Results discussion                                                                connected subnetwork PI network (a connected <br />                                                                                       pathway, described Materials methods). An example <br /> Assembly GI PI networks                                                        sparse pathways presented Figure 2A. The <br /> We assembled GI network (Figure 1A) taking                              buffering mechanism DNA repair through <br /> BioGRID database version 2.0.19 (Stark et al, 2006) 13 632                            homologous recombination response oxidative <br /> synthetic lethality synthetic fitness interactions                            stress partially recovered using dense <br /> S. cerevisiae, covering 2682 genes. By focusing genes                         pathway <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> (not shown). We define between-pathway <br /> at interactions, obtained GI network 1869                           model (BPM) Kelley Ideker (2005), using new <br /> genes 12 850 interactions. Our PI network, consisting                          notion pathway (Figure 1C). The scoring models the <br /> protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220;DNA interactions multiple                            model detection algorithm defined Materials and <br /> sources (Supplementary information 1), contained 68172                                methods. A comparison models using <br /> interactions covering 6184 proteins.                                                  dense pathways interaction data (Supplementary <br />                                                                                       information A) shows construct between- <br />                                                                                       pathway explanations GIs (3765 versus 3117), while <br /> Pathway definitions between-pathway models                                        maintaining significant functional content models. Our <br />                                                                                       models allow direct interpretation specific <br /> Our starting point computational framework Kelley <br />                                                                                       buffering cases congruence methods (Supplementary <br /> and Ideker (2005) detection between-pathway inter- <br />                                                                                       information B). <br /> pretations GIs. Kelley Ideker define &#226;&#8364;&#732;pathway&#226;&#8364;&#8482; a <br /> densely connected set proteins PI network, a <br /> &#226;&#8364;&#732;between-pathway model&#226;&#8364;&#8482; disjoint pair pathways that <br /> are densely interconnected GI network (Figure 1B).                             A comprehensive model map S. cerevisiae <br /> Models defined probabilistically using                            Our BPM finding approach generated 140 models and <br /> greedy algorithm. While requirement high PI density                         provided between-pathway explanations 3765 GIs, 2.7- <br /> appropriate complexes, known biological                                fold increase 1377 interactions explained Kelley <br /> pathways (e.g. linear signaling cascades) induce dense                         Ideker (2005). This mainly incorporation of <br /> subnetworks physical network. We chose                            GIs (12 850 instead 4125) lesser extent due <br /> employ alternative <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, pathway simply                        using PIs (68172 versus 27 604), use only <br />  <br />  <br />  <br />  <br /> Figure 1 Study outline methodology. (A) Overview analysis methods reported results. (B) A BPM constructed dense pathways PI <br /> network. (C) A BPM constructed connected pathways PI network. (D, E) Examples biological explanations pivot proteins. In (D), pivots <br /> correspond shared members linear pathways, signal flow indicated arrow directions, (E), correspond shared complex <br /> members. Note pivots (E) redundant, densely connected pathways physical interactions genetic interaction <br /> between them. <br />  <br /> 2 Molecular Systems Biology 2007                                                                                            &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                         Pathway redundancy essential proteins <br />                                                                                                                                          I Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Figure 2 Model examples. Rectangles represent genes, pathways shown different colors. The blue ovals pivot nodes. Essentials genes are <br /> drawn thicker border. See main text discussion of BPMs (A&#226;&#8364;&#8220;E). In (A), clarity, 52 genetic interactions pathways shown. <br />  <br />  <br />  <br />  <br /> to create &#226;&#8364;&#732;scaffolds&#226;&#8364;&#8482; pathways scoring models.                       Po0.001; Supplementary Figure S2). The correlation the <br /> The gene content models available supplementary                      pathways BPM higher expected (0.112 versus <br /> archive. A description interactive visualization                     0.0378 expected, Po0.001; Supplementary Figure S2). <br /> the models available http://acgt.cs.tau.ac.il/bpm. <br />                                                                                     Identification pivot proteins <br /> Functional enrichment models                                                     As established pathways models frequently <br />                                                                                     represent coherent functional units, functional units <br /> We utilized TANGO algorithm (Shamir et al, 2005) order <br />                                                                                     cell share components (Krause et al, <br /> to test functional enrichment models GO <br />                                                                                     2004), tested possibility computationally detecting <br /> categories (Ashburner et al, 2000). A total 71.4 69.3% <br />                                                                                     shared components PI network. To end, <br /> of BPMs enriched functional <br />                                                                                     model, sought proteins densely connected <br /> category &#226;&#8364;&#732;biological process&#226;&#8364;&#8482; &#226;&#8364;&#732;cellular <br />                                                                                     pathways (Figure 1D E, Materials and <br /> compartment&#226;&#8364;&#8482; ontologies, respectively (Supplementary Table <br />                                                                                     methods), called pivot proteins model. <br /> S1). Of complexes annotated SGD GO-slim (Cherry et al, <br />                                                                                     Altogether, identified 124 distinct pivots 40 models. On <br /> 1998), 46.3% enriched BPM. Despite the <br />                                                                                     average, 1.09 pivots model, pivot <br /> low coverage GI network, coverage complexes is <br />                                                                                     appeared 1.22 models. <br /> comparable achievable direct analysis PI <br />                                                                                        We systematically analyzed representation proteins <br /> network (Supplementary information C). <br />                                                                                     known distinct processes the <br />                                                                                     group pivots. To end, identified proteins participat- <br />                                                                                     ing complexes pathways (see Materials and <br /> Phenotypic coherence between <br />                                                                                     methods), used curated set multicomplexed <br /> pathways                                                                            genes (Krause et al, 2004). As summarized Table I, the <br /> To extent pathways BPM                           pivots enriched sets. One example such <br /> function? To answer question, used phenotypic                           overlap BPM 96 (Figure 2B). In model containing as <br /> contribution non-essential genes S. cerevisiae,                            pathways parts SWR1 Ino80 complexes involved in <br /> measured fitness deletion mutants diverse                              chromatin remodeling, identified pivot proteins Arp4, <br /> treatments (Brown et al, 2006). We BPM pathway                         Rvb1 Rvb2, proteins known to <br /> is significantly coherent phenotypic response <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                   participate SWR1 Ino80 complexes (Shen <br /> than random connected groups size physical                       et al, 2000; Krogan et al, 2003). In BPM 97 (Figure 2C), Sus1, <br /> network (Pearson correlation 0.384 versus average 0.0382,                  shown nuclear pore <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                          Molecular Systems Biology 2007 3 <br />  Pathway redundancy essential proteins <br /> I Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Table I Multiple roles pivot proteins <br />  <br />                                                 No. proteins                      Pivots                     Expected                        Significance <br />  <br /> GO complexes                                          206                              21                          4.35                        P&#194;&#188;1.27  10 9 <br /> KEGG pathways                                         390                              11                          8.24                        P&#194;&#188;0.200 <br /> Filtered KEGG pathways                                 71                               6                          1.50                        P&#194;&#188;3.68  10 3 <br /> Known complex overlaps                                 39                               8                          0.55                        P&#194;&#188;7.49  10 9 <br />  <br /> The enrichment proteins known participate multiple physical pathways set pivot proteins. GO complexes taken SGD &#226;&#8364;&#732;macromolecular <br /> complex GO-slim&#226;&#8364;&#8482; ontology. The filtered KEGG pathways KEGG pathways 50% genes formed connected component physical network. <br /> Known complex overlaps taken Krause et al (2004). Significance evaluated using hypergeometric distribution. <br />  <br />  <br /> and SAGA complex (Rodriguez-Navarro et al, 2004)                          partially redundant (McCleland et al, 2003), are <br /> identified pivot model representing GIs                     placed different pathways. Note four <br /> two pathways. When pivots correspond known                              subunits Ndc80 complex possible GIs PIs <br /> complex pathway overlaps, frequently represent                            pairs, biologically correct partition this <br /> general purpose genes cooperating multiple pathways.                         complex pathways obtained taking con- <br /> For example, BPM 87, general transcription factor Spt15                    sideration GIs model. For example, Mad1 <br /> was identified pivot model contains components                     physically connects Tid3/Spc25 subunit and <br /> of distinct transcription-related complexes RSC, SWR1                     genetically interacts Nuf2/Spc24 subunit. These <br /> SAGA (Figure 2D).                                                                 additional external interactions cause biologically correct <br />                                                                                   partition score higher alternatives. <br />                                                                                      Mad1, highly conserved protein specific function in <br /> Essentiality evolutionary retention pivot                                  spindle cell cycle checkpoint, additional interest. At <br /> proteins                                                                          spindle checkpoint, cells arrested metaphase <br />                                                                                   chromosomes successfully attach microtubules. <br /> The pathways form model partially <br />                                                                                   Tid3 Spc25, members Ndc80 complex, which <br /> redundant function, pivots represent proteins <br />                                                                                   appear pathway Mad1, specifically linked to <br /> that active pathways, hypothesized the <br />                                                                                   spindle checkpoint organisms (summarized in <br /> pivots frequently correspond essential genes. Indeed, 72 <br />                                                                                   Bharadwaj et al, 2004). Moreover, recruitment Mad1 <br /> of pivots essential, highly significant <br />                                                                                   shown dependent Spc25 Tid3 Xenopus and <br /> fraction given total number essential genes the <br />                                                                                   human cells (McCleland et al, 2003; Bharadwaj et al, 2004) and <br /> network (22.6 essentials expected, P&#194;&#188;1.42  10 23). Although <br />                                                                                   spindle checkpoint shown defective spc25 <br /> we observed general strong correlation degree and <br />                                                                                   mutants (Wigge Kilmartin, 2001). How exactly does Mad1 <br /> essentiality (P&#194;&#188;5.87  10 85 using rank-sum test), pre- <br />                                                                                   attach kinetochore currently known. Although <br /> viously reported (Jeong et al, 2001), high prevalence of <br />                                                                                   Mad1 shows yeast two-hybrid interaction Spc25 in <br /> essential genes pivots far be <br />                                                                                   S. cerevisiae (Newman et al, 2000) Tid3 human <br /> explained degrees (39.98 essentials expected, <br />                                                                                   cells (Martin-Lluesma et al, 2002), attempts demonstrate a <br /> Po10 5). The essential pivots tend closer <br />                                                                                   biochemical interaction Ndc80 Mad1 been <br /> functions BPMs (Supplementary information D). <br />                                                                                   reported unsuccessful (Martin-Lluesma et al, 2002; McCleland <br /> Essentiality recently shown connected the <br />                                                                                   et al, 2003). <br /> evolutionary retention genes eukaryotes (Gustafson <br />                                                                                      In model, Mad1 linked pivot Smc1, member of <br /> et al, 2006). Using data Gustafson et al (2006) we <br />                                                                                   cohesin complex, required sister chromatid cohesion <br /> found pivots significantly retained evolution <br />                                                                                   mitosis. Smc1 shown localize kinetochore <br /> (P&#194;&#188;9.79  10 9), controlling large fraction <br />                                                                                   meiosis interact Tid3 yeast human <br /> of essential genes (P&#194;&#188;0.029). <br />                                                                                   cells (Zheng et al, 1999; Gregson et al, 2002). Furthermore, <br />                                                                                   Smc1 shown required proper assembly the <br />                                                                                   mitotic spindle human cells (Gregson et al, 2001), its <br /> Example: spindle checkpoint model <br />                                                                                   exact function metaphase unknown. Our findings <br /> An interesting example using models pivots                             suggest connection Mad1 kinetochore in <br /> understanding cellular mechanisms BPM 66 (Figure 2E).                          general Ndc80 complex particular, mediated <br /> This model represents buffering different compo-                          Smc1. Note use pivots provides additional <br /> nents kinetochore, complex bound centromeres                      clues BPM annotation understanding inter- <br /> of chromosomes mitosis. Together pivots,                      pathway organization. <br /> model composed members known subcomplexes <br /> of inner kinetochore: Ndc80, COMA MIND (De Wulf <br /> et al, 2003). Specifically, pivots Mtw1 Dsn1 correspond <br /> to distinct <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> MIND complex, bridges                             Physiological properties models <br /> different kinetochore subcomplexes (De Wulf et al, 2003;                          Recently, physiological properties PI network hubs <br /> Westermann et al, 2003). The subunits Ndc80                            extensively analyzed (Batada et al, 2006). As many <br /> complex, Nuf2/Spc24 Tid3/Spc25, shown                           proteins models hubs, asked whether <br />  <br /> 4 Molecular Systems Biology 2007                                                                                      &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                Pathway redundancy essential proteins <br />                                                                                                                                 I Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> their physiological properties differed                  ensemble networks degree sequence counting <br /> hubs. Here, focus analysis mRNA stability (Wang                 fraction contain interaction b. The log- <br />                                                                              odds score then <br /> et al, 2002) number putative phosphorylation sites <br /> (Obenauer et al, 2003), properties manifesting                                           P&#195;&#176;V1 ; V2 jMBPM &#195;&#382; <br />                                                                                 S&#195;&#176;V1 ; V2 &#195;&#382; &#194;&#188; log <br /> turnover regulation gene. Detailed analysis,                                          P&#195;&#176;V1 ; V2 jMnull &#195;&#382; <br />                                                                                                  Q <br /> covering additional properties, described Supplementary                                log &#195;&#176;a;b&#195;&#382;2V1  V2 bI&#195;&#176;a; b&#195;&#382; &#195;&#190; &#195;&#176;1   b&#195;&#382;&#195;&#176;1   I&#195;&#176;a; b&#195;&#382;&#195;&#382; <br /> information E Supplementary Table S2.                                                   &#194;&#188;Q <br />                                                                                                &#195;&#176;a;b&#195;&#382;2V1  V2 ra;b I&#195;&#176;a; b&#195;&#382; &#195;&#190; &#195;&#176;1   ra;b &#195;&#382;&#195;&#176;1   I&#195;&#176;a; b&#195;&#382;&#195;&#382; <br />    Our tests BPM genes behave significantly unlike <br /> other genes. The average mRNA half-life BPM genes                     The main difference score score described in <br />                                                                              Kelley Ideker (2005) structure imposed BPMs the <br /> 21.4, versus 26.2 genes (P&#194;&#188;9.97  10 10, rank-sum                PI network. Here score model density PIs within <br /> test). The average number phosphorylation sites 5.1,                  pathway, instead require connected GP. <br /> versus 4.0 genes (P&#194;&#188;3.13  10 9). Both parameters <br /> were significantly correlated pathways that <br /> constitute BPM (intraclass correlations 0.408 0.189,                Model finding algorithm <br /> Po10 4 P&#194;&#188;0.0184, respectively). This finding                   The model finding procedure described Kelley Ideker (2005) a <br /> explained high degrees model genes (Supplementary                  greedy network search algorithm uses seeds single GIs. We <br /> Table S2).                                                                   improve procedure initializing algorithm better seeds <br />                                                                              maximal bicliques GI network (a biclique disjoint <br />    Note mRNA half-lives experimentally derived,                 pair node sets node set edges nodes <br /> whereas phosphorylation sites computationally in-                    set). The following procedure performed u, v <br /> ferred sequence. On 3552 genes                   (u, v)AEP <br /> parameters available, correlated (r&#194;&#188; 0.0182 <br />                                                                              1. Identify B&#226;&#8364;&#8221;the set nodes adjacent u v GG. Proceed <br /> P&#194;&#188;0.276). However, parameters significantly corre-                      |B|Xkmin. <br /> lated genes BPMs (566 genes, r&#194;&#188; 0.132,                         2. Partition B connected components GP: B1, B2, y, Bl. <br /> P&#194;&#188;1.6  10 3). These results remain highly significant                   3. For Bi |Bi|Xkmin, identify set Ai nodes <br /> controlling key functions enriched model partici-                    adjacent nodes Bi GG. <br />                                                                              4. For Ai, partition connected components induces in <br /> pants (Supplementary Figure S4). These findings suggest                    GP, A1i , A2i , y, Am               j                           j <br />                                                                                                     add (Bi, Ai) set seeds |Ai|Xkmin. <br /> genes BPMs experience stringent regulation. A <br /> possible hypothesis pathways redundant                  This algorithm produces maximal bicliques (V1, V2) GG, that <br />                                                                              Vi induces connected component GP size Xkmin. The <br /> mechanisms available, tighter regulation allows cell               produced set seeds filtered overlaps. We used kmin&#194;&#188;2 in <br /> to switch alternatives faster. However,                     tests. Owing relatively sparse nature interaction <br /> conclusion revalidated GIs covering wider                   networks, method efficient practice. <br /> functional range available.                                              The optimization phase starts seed candidate model, <br />                                                                              greedily tries improve score current model through <br />    Our results indicate despite limitations today&#226;&#8364;&#8482;s GI <br />                                                                              addition, removal exchange nodes pathways <br /> and PI networks, integrated analysis powerful                     keeping pathway connected PI network. In order to <br /> approach understanding organization yeast                     efficiently track connectivity requirement, use the <br /> cellular system. We expect analysis provide                   notion articulation nodes. An articulation node connected graph <br /> insights large picture genetic redundancy higher              node removal disconnects graph. Articulation nodes can <br />                                                                              efficiently detected depth-first search traversal the <br /> eukaryotes well.                                                          graph, calculating &#226;&#8364;&#732;lowpoint&#226;&#8364;&#8482; values node (cf. Even, <br />                                                                              1979). The algorithm maintains connectivity pathway by <br />                                                                              dynamically updating, pathway, set articulation nodes. <br /> Materials methods                                                        This set used ban optimization moves disrupt connectivity. <br />                                                                              After optimization, filtering step removes models overlap by <br /> Scoring models                                                               450% pathways higher scoring models. An additional <br />                                                                              model filtering step computes empiric P-value sampling 1000 <br /> We build probabilistic score described Kelley Ideker <br />                                                                              random gene groups size, retains BPMs with <br /> (2005) Sharan et al (2005) identify between-pathway explana- <br />                                                                              Po0.05 (see Supplementary information F). <br /> tions GIs finding BPMs GI PI networks. Let <br /> GG&#194;&#188;(V, EG) GI network let GP&#194;&#188;(V, EP) PI network. <br /> Note nodes V represent genes products, <br /> depending context. A BPM pair disjoint sets V1, V2,      Identification pivot proteins <br /> that (a) |V1|, |V2|X2, (b) Vi induces connected subgraph GP        Given model (V1, V2), seek nodes densely connected to <br /> and (c) unusually GIs V1 V2 (Figure 1C).          pathways physical network. Specifically, vAV, <br /> We Vi pathway. Property (c) reflects assumption         denote Ni(v) nodes Vi adjacent v GP. We v a <br /> genetic buffering implies dense set GIs pathways.           pivot if, i&#194;&#188;1, 2, |Ni(v)|Xlmin |Ni(v)| significant, given the <br />    We quantify property (c). We derive log-odds score reflecting       degree v GP (PoPmax, using hypergeometric test). In actual <br /> the likelihood density GIs pathways           analysis described paper, used lmin&#194;&#188;2 Pmax&#194;&#188;0.05. In <br /> unusually high. We compare hypotheses: BPM                     addition, filter master regulator genes involved many <br /> hypothesis, pair genes, V1 V2,          processes, protein folding chaperons, considered as <br /> genetically interact high probability b, independently   pivots proteins degree o250 GP. <br /> gene pairs, likelihood model (V1, V2) thus <br /> P(a, b)A(V1   V2)bI(a, b) &#195;&#190; (1 b)(1 I(a, b)), I(a, b) equals 1 if <br /> there exists GI b equals 0; null <br /> hypothesis, pair (a, b) connected probability ra, b,           Statistical analysis <br /> representing chance observing interaction random, given       Correlation analysis performed using non-parametric Spear- <br /> the degrees b GG. We estimate ra, b generating random       man test, unless indicated. All P-values reported when <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                Molecular Systems Biology 2007 5 <br />  Pathway redundancy essential proteins <br /> I Ulitsky R Shamir <br />  <br />  <br />  <br /> controlling specific gene class (e.g. essential genes) obtained   Gregson HC, Schmiesing JA, Kim JS, Kobayashi T, Zhou S, Yokomori K <br /> by random sampling large number gene groups                (2001) A potential role human cohesin mitotic spindle aster <br /> fraction genes class. The P-values reported                    assembly. J Biol Chem 276: 47575&#226;&#8364;&#8220;47582 <br /> controlling degrees calculated binning         Gregson HC, Van Hooser AA, Ball Jr AR, Brinkley BR, Yokomori K <br /> genes 40 equal-size bins based degree, sampling <br />                                                                                  (2002) Localization human SMC1 protein kinetochores. <br /> genes bins, maintaining proportion genes <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                  Chromosome Res 10: 267&#226;&#8364;&#8220;277 <br /> each bin. <br />                                                                              Gustafson AM, Snitkin ES, Parker SC, DeLisi C, Kasif S (2006) Towards <br />                                                                                  identification essential genes using targeted genome <br />                                                                                  sequencing comparative analysis. BMC Genomics 7: 265 <br /> Supplementary information                                                    Hartman JLt, Garvik B, Hartwell L (2001) Principles buffering of <br />                                                                                  genetic variation. Science 291: 1001&#226;&#8364;&#8220;1004 <br /> Supplementary information available Molecular Systems <br />                                                                              Ito T, Chiba T, Yoshida M (2001) Exploring protein interactome <br /> Biology website (www.nature.com/msb). <br />                                                                                  using comprehensive two-hybrid projects. Trends Biotechnol 19: <br />                                                                                  S23&#226;&#8364;&#8220;S27 <br />                                                                              Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality and <br />                                                                                  centrality protein networks. Nature 411: 41&#226;&#8364;&#8220;42 <br /> Acknowledgements                                                             Kelley R, Ideker T (2005) Systematic interpretation genetic <br /> We thank members Shamir laboratory helpful                        interactions using protein networks. Nat Biotechnol 23: 561&#226;&#8364;&#8220;566 <br /> discussions. IU fellow Edmond J Safra Bioinformatics             Krause R, von Mering C, Bork P, Dandekar T (2004) Shared <br /> Program Tel-Aviv University. RS supported                  components protein complexes&#226;&#8364;&#8221;versatile building blocks or <br /> Wolfson foundation, EMI-CD project funded              biochemical artefacts? BioEssays 26: 1333&#226;&#8364;&#8220;1343 <br /> European Commission FP6 Programme, thematic             Krogan NJ, Keogh MC, Datta N, Sawa C, Ryan OW, Ding H, Haw RA, <br /> area &#226;&#8364;&#732;Life Sciences, genomics biotechnology health&#226;&#8364;&#8482;, contract            Pootoolal J, Tong A, Canadien V, Richards DP, Wu X, Emili A, <br /> number LSHG-CT-2003-503269.                                                      Hughes TR, Buratowski S, Greenblatt JF (2003) A Snf2 family <br />                                                                                  ATPase complex required recruitment histone H2A variant <br />                                                                                  Htz1. Mol Cell 12: 1565&#226;&#8364;&#8220;1576 <br />                                                                              Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, Gerber GK, <br /> References                                                                       Hannett NM, Harbison CT, Thompson CM, Simon I, Zeitlinger J, <br />                                                                                  Jennings EG, Murray HL, Gordon DB, Ren B, Wyrick JJ, Tagne JB, <br /> Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM,                 Volkert TL, Fraenkel E, Gifford DK, Young RA (2002) <br />    Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-         Transcriptional regulatory networks Saccharomyces cerevisiae. <br />    Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald            Science 298: 799&#226;&#8364;&#8220;804 <br />    M, Rubin GM, Sherlock G (2000) Gene ontology: tool                Martin-Lluesma S, Stucke VM, Nigg EA (2002) Role Hec1 spindle <br />    unification biology. The Gene Ontology Consortium. Nat Genet               checkpoint signaling kinetochore recruitment Mad1/Mad2. <br />    25: 25&#226;&#8364;&#8220;29                                                                     Science 297: 2267&#226;&#8364;&#8220;2270 <br /> Bader GD, Hogue CW (2003) An automated method finding                    McCleland ML, Gardner RD, Kallio MJ, Daum JR, Gorbsky GJ, Burke <br />    molecular complexes large protein interaction networks. BMC                DJ, Stukenberg PT (2003) The highly conserved Ndc80 complex is <br />    Bioinformatics 4: 2                                                           required kinetochore assembly, chromosome congression, and <br /> Batada NN, Hurst LD, Tyers M, Russell R (2006) Evolutionary                  spindle checkpoint activity. Genes Dev 17: 101&#226;&#8364;&#8220;114 <br />    physiological importance hub proteins. PLoS Comput Biol 2: e88         Newman JR, Wolf E, Kim PS (2000) A computationally directed screen <br /> Bharadwaj R, Qi W, Yu H (2004) Identification novel                       identifying interacting coiled coils Saccharomyces cerevisiae. <br />    components human NDC80 kinetochore complex. J Biol                     Proc Natl Acad Sci USA 97: 13203&#226;&#8364;&#8220;13208 <br />    Chem 279: 13076&#226;&#8364;&#8220;13085                                                     Obenauer JC, Cantley LC, Yaffe MB (2003) Scansite 2.0: proteome-wide <br /> Brown JA, Sherlock G, Myers CL, Burrows NM, Deng C, Wu HI,                       prediction cell signaling interactions using short sequence <br />    McCann KE, Troyanskaya OG, Brown JM (2006) Global analysis                 motifs. Nucleic Acids Res 31: 3635&#226;&#8364;&#8220;3641 <br />    gene function yeast quantitative phenotypic profiling. Mol          Ozier O, Amin N, Ideker T (2003) Global architecture genetic <br />    Syst Biol 2:1                                                                 interactions protein network. Nat Biotechnol 21: 490&#226;&#8364;&#8220;491 <br /> Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, Hester ET, Jia Y,        Pan X, Ye P, Yuan DS, Wang X, Bader JS, Boeke JD (2006) A DNA <br />    Juvik G, Roe T, Schroeder M, Weng S, Botstein D (1998) SGD:                   integrity network yeast Saccharomyces cerevisiae. Cell 124: <br />    Saccharomyces genome database. Nucleic Acids Res 26: 73&#226;&#8364;&#8220;79                    1069&#226;&#8364;&#8220;1081 <br /> De Wulf P, McAinsh AD, Sorger PK (2003) Hierarchical assembly         Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, <br />    budding yeast kinetochore multiple subcomplexes. Genes Dev               Hieter P, Spencer F, Boeke JD (2004) A robust toolkit functional <br />    17: 2902&#226;&#8364;&#8220;2921                                                                 profiling yeast genome. Mol Cell 16: 487&#226;&#8364;&#8220;496 <br /> Even S (1979) Graph Algorithms. Potomac, MD: Computer Science                Rives AW, Galitski T (2003) Modular organization cellular networks. <br />    Press                                                                         Proc Natl Acad Sci USA 100: 1128&#226;&#8364;&#8220;1133 <br /> Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S,            Rodriguez-Navarro S, Fischer T, Luo MJ, Antunez O, Brettschneider S, <br />    Lucau-Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, El-               Lechner J, Perez-Ortin JE, Reed R, Hurt E (2004) Sus1, a <br />    Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S,                      functional component SAGA histone acetylase complex and <br />    Curtiss M, Davis K, Deutschbauer A, Entian KD, Flaherty P, Foury F,           nuclear pore-associated mRNA export machinery. Cell 116: <br />    Garfinkel DJ, Gerstein M, Gotte D, Guldener U, Hegemann JH,                   75&#226;&#8364;&#8220;86 <br />    Hempel S, Herman Z, Jaramillo DF, Kelly DE, Kelly SL, Kotter P,           Shamir R, Maron-Katz A, Tanay A, Linhart C, Steinfeld I, Sharan R, <br />    LaBonte D, Lamb DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier             Shiloh Y, Elkon R (2005) EXPANDER&#226;&#8364;&#8221;an integrative program suite <br />    M, Mao R, Menard P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-            microarray data analysis. BMC Bioinformatics 6: 232 <br />    Macdonald P, Scherens B, Schimmack G, Shafer B, Shoemaker DD,             Sharan R, Ideker T, Kelley B, Shamir R, Karp RM (2005) Identification <br />    Sookhai-Mahadeo S, Storms RK, Strathern JN, Valle G, Voet M,                  protein complexes comparative analysis yeast bacterial <br />    Volckaert G, Wang CY, Ward TR, Wilhelmy J, Winzeler EA, Yang Y,               protein interaction data. J Comput Biol 12: 835&#226;&#8364;&#8220;846 <br />    Yen G, Youngman E, Yu K, Bussey H, Boeke JD, Snyder M,                    Shen X, Mizuguchi G, Hamiche A, Wu C (2000) A chromatin <br />    Philippsen P, Davis RW, Johnston M (2002) Functional profiling             remodelling complex involved transcription DNA <br />    Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391                      processing. Nature 406: 541&#226;&#8364;&#8220;544 <br />  <br /> 6 Molecular Systems Biology 2007                                                                              &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                         Pathway redundancy essential proteins <br />                                                                                                                          I Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz A, Tyers M    Wang Y, Liu CL, Storey JD, Tibshirani RJ, Herschlag D, Brown PO <br />    (2006) BioGRID: general repository interaction datasets.          (2002) Precision functional specificity mRNA decay. <br />    Nucleic Acids Res 34: D535&#226;&#8364;&#8220;D539                                         Proc Natl Acad Sci USA 99: 5860&#226;&#8364;&#8220;5865 <br /> Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N,             Westermann S, Cheeseman IM, Anderson S, Yates 3rd JR, Drubin DG, <br />    Robinson M, Raghibizadeh S, Hogue CW, Bussey H, Andrews B,              Barnes G (2003) Arch <br /> </body></html>