<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Letter <br />  <br /> Origins impact constraints evolution <br /> of gene families <br /> Boris E. Shakhnovich1,3 Eugene V. Koonin2 <br /> 1 <br />  Bioinformatics Program, Boston University, Boston, Massachusetts 02215, USA; 2National Center Biotechnology Information, <br /> National Institutes Health, Bethesda, Maryland 20894, USA <br />  <br />  <br />         Recent investigations high-throughput genomic phenomic data uncovered variety significant but <br />         relatively weak correlations gene&#226;&#8364;&#8482;s functional evolutionary characteristics. In particular, essential genes <br />         genes paralogs slight propensity evolve slowly nonessential genes singletons, <br />         respectively. However, given weakness multiplicity associations, biological relevance remains <br />         uncertain. Here, existence essential paralog used specific strong gauge of <br />         selection. We partition gene families genomes classes: include essential gene <br />         (E-families) essential genes (N-families). We weaker purifying selection causes N-families <br />         evolve dynamic regime higher rates duplicate fixation pseudogenization. Because genes <br />         E-families subject significantly stronger purifying selection N-families, survive longer and <br />         exhibit greater sequence divergence. Longer average <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> time allows divergence upstream regulatory <br />         regions, resulting change transcriptional context paralogs E-families. These findings compatible <br />         differential division ancestral functions (subfunctionalization) emergence novel functions <br />         (neofunctionalization) prevalent modes evolution paralogs E-families opposed <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />         pseudogenization (nonfunctionalization), typical fate paralogs N-families. Unlike other <br />         characteristics genes, essentiality, existence paralogs, expression level, membership E-family or <br />         N-family strongly correlates level selection appears major determinant gene&#226;&#8364;&#8482;s <br />         evolutionary fate. <br />         [Supplemental material available online www.genome.org.] <br />  <br /> The nature connections organismal molecular                           theory molecular evolution, remains open question. There <br /> evolution remains fundamental and, generally, unanswered                           substantial disagreement correlation an <br /> question. The relationship evolution gene                       artifact caused transitive relationships expression level <br /> organism characterized change fitness precipi-                      (Pal et al. 2001) abundance (Drummond et al. 2005). Although <br /> tated deletion <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span> gene (Drake et al. 1998;                       recent, sophisticated statistical analyses indepen- <br /> Keightley Eyre-Walker 1999). Although simple defi-                     dent groups suggest correlation gene&#226;&#8364;&#8482;s <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>knockout</span> <br /> nition, quantifying fitness effect gene deletions muta-                    effect purifying selection does exist; studies also <br /> tions remains hard problem. One principal difficulties                   shown correlation probably quite weak (Drummond <br /> that changes deleterious conditions neutral                      et al. 2005; He Zhang 2005; Wall et al. 2005). Most impor- <br /> or beneficial conditions (Macarthur Levins                           tantly, observed correlations taught little evo- <br /> 1964; Levins 1968). As result, notions fitness essen-                   lutionary dynamics governing gene duplication divergence. <br /> tiality inherently ambiguous.                                                          Gene duplication followed divergence pri- <br />       Driven recent availability complete genome                      mary driving forces functional innovation evolu- <br /> sequences, results genome-wide functional as-                        tion (Ohno 1970; Lynch Katju 2004). Currently, diver- <br /> says, researchers observed significant correlations                     gence paralogs duplication thought follow one <br /> functional characteristics genes, essentiality (Hurst                 routes. The common outcome duplication <br /> Smith 1999; Hirsh Fraser 2001; Yang et al. 2003), number                      event nonfunctionalization copy a <br /> protein-interaction partners (Fraser et al. 2002), expression                     pseudogene and, eventually, goes extinct (Nei Roychoud- <br /> level (Pal et al. 2001), rate evolution (equivalently                 houry 1973; Petrov Hartl 2000), second copy <br /> strength purifying selection). However, reported correla-                     retains original function. The other, common none- <br /> tions relatively weak, contribution character-                   theless crucial evolutionary scenarios neofunctionalization <br /> istic remains subject vigorous debate (Jordan et al. 2003; Wall                 subfunctionalization. In case neofunctionalization, <br /> et al. 2005; Drummond et al. 2006). In particular, validity                   paralog retains original function, one <br /> the observed negative correlation gene&#226;&#8364;&#8482;s knockout ef-                      evolves new function phase rapid, nearly neutral <br /> fect (essentiality) evolutionary rate, originally predicted                   evolution (Ohno 1970). Under subfunctionalization model, <br /> Kimura Ohta (Kimura 1981) corollary neutral                          multiple functions ancestral gene divided between <br />                                                                                      paralogs, evolve purifying selection (Force <br /> 3 <br />  Corresponding author.                                                               et al. 1999; Lynch Force 2000; He Zhang 2005). Given <br /> E-mail Borya@acs.bu.edu; fax (617) 353-4814. <br /> Article published online print. Article publication date http://   sufficiently detailed information, probability functional <br /> www.genome.org/cgi/doi/10.1101/gr.5346206.                                           divergence occurred speciation event estimated <br />  <br />  <br /> 16:1529&#226;&#8364;&#8220;1536 &#194;&#169;2006 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/06; www.genome.org                           Genome Research          1529 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />  <br />  <br /> Figure 1. The largest E- N-gene families yeast. The family left contains essential genes (N-family), family right includes <br /> two essential members (E-family). The red circles represent essential genes, blue squares represent nonessential genes. The E-family represented <br /> here consists various mitochondrial membrane transporters, N-family includes seripauperins (poorly characterized multigene family repre- <br /> sented, primarily, subtelomeric regions yeast chromosomes). <br />  <br />  <br /> using statistical models sequence evolution (Gu 1999,                       true essential genes paralogs (Yang et <br /> 2001a,b). However, general, relationship selec-                 al. 2003). Since independent line analysis indicated <br /> tion neofunctionalization understood. For ex-                  genes paralogs, average, evolve slowly than <br /> ample, paralogs experience period relaxed purify-                singletons (Davis Petrov 2004; Jordan et al. 2004), nature <br /> ing selection immediately duplication, genes paralogs               relationships variables&#226;&#8364;&#8221;the gene&#226;&#8364;&#8482;s <br /> show greater evolutionary conservation singletons (Davis                  essentiality, existence paralogs, rate evolution&#226;&#8364;&#8221; <br /> and Petrov 2004; Jordan et al. 2004). This unexpected find-             remains unclear. We sought explore essentiality or <br /> ing taken indicate that, average, duplications               paralogy necessary sufficient primary determinant of <br /> genes critical biological functions retained               purifying selection. To end, need compare strength <br /> than duplications biologically important genes.                        selection essential nonessential members the <br />       The goal present study elucidate general               gene family. <br /> relationships functional constraints, differential                           Paralogy relationships genes genomes can <br /> strengths purifying selection, gene duplication. First,              conveniently represented general form Diver- <br /> observe membership gene family good predictor                gence Diffusion Graph (DDG). The vertices represent genes, <br /> selection. For example, genes essential paralog               edges represent homology relationships weighed according <br /> under stronger selection genes essential paralogs.                sequence similarity scores (Fig. 1; Supplemental ma- <br /> We families paralogs include es-                terial). All genes partitioned paralogous families (Har- <br /> sential gene (E-families) consist entirely non-              rison Gerstein 2002; Enright et al. 2003) finding strongly <br /> essential genes (N-families) evolve dramatically different regi-            connected components DDG, is, sets vertices in <br /> mens. Although genes E-families, average, evolve substan-                path exists pair (see Supplemental mate- <br /> tially slowly genes N-families, E-families               rial). Partitioning paralogous families (strongly connected com- <br /> a greater divergence paralogs. This attrib-                ponents DDG) classes&#226;&#8364;&#8221;those include least <br /> uted significantly longer average survival time paralogs             essential gene (E-families) essential mem- <br /> in E-families compared N-families. The E-families appear              bers (N-families)&#226;&#8364;&#8221;allows compare essential nonessen- <br /> comprise reservoir genes evolution new functions               tial members family (see Methods). For species <br /> the subfunctionalization neofunctionalization routes. Fi-                  genome-wide data gene essentiality available, the <br /> nally, relationship evolution               E-family N-family sets included substantial numbers genes <br /> the open reading frames upstream regions. Specifically,                    (Table 1), providing statistical comparison evolutionary <br /> genes E-families stronger selection evolve novel             characteristics essential nonessential genes within <br /> transcriptional regulatory contexts.                                           E-families comparisons sets genes in <br />                                                                                E-families N-families. <br />  <br /> Results                                                                        Table 1. Numbers essential genes paralogs <br />                                                                                sizes E-families N-families <br /> Homogeneity strength selection paralogous <br /> families                                                                                                       Essential     Nonessential <br />                                                                                                All essential   genes       genes         Genes in <br /> While essential genes tend evolve slowly and, implication,               Species             genes       E-families     E-families       N-families <br /> appear stronger purifying selection nonessential <br /> ones (Hirsh Fraser 2001; Jordan et al. 2002), trend                S. cerevisiae      1032             95             183             656 <br />                                                                                C. elegans          861            158             119            1300 <br /> relatively weak. Furthermore, published reports suggest                   E. coli             688             24             328             521 <br /> that negative correlation selection essentiality <br />  <br />  <br /> 1530      Genome Research <br />              www.genome.org <br />                                                                                                               Evolution gene families <br />  <br />  <br /> Table 2. Average strength purifying selection essential     strength selection correlates probability pseudo- <br /> nonessential genes, E-families N-families                  genization M. leprae. Since observed E- <br />                     Essential genes   Nonessential genes       P      families N-families evolve different strengths puri- <br />                                                                       fying selection, hypothesized exhibit <br /> SFP density            0.01567              0.02158          1e-20    different pseudogenization duplication rates. Identification <br /> Ka/Ks &#226;&#8364;&#8220; Yeast          0.10                 0.13             1e-8     pseudogenes (Harrison et al. 2001, 2002; Harrison Ger- <br /> Ka/Ks &#226;&#8364;&#8220; E. coli        0.054                0.099            1.6e-6   stein 2002) derived E-families N-families S. cerevisiae <br />                                                                       showed estimated pseudogenization rate E-families <br /> Only nonessential <br /> genes families      E-families          N-families                 approximately seven times lower N-families. <br /> SFP density            0.012               0.027             1e-40    Although high-precision estimate of <br /> Ka/Ks &#226;&#8364;&#8220; Yeast          0.08                0.12              4e-11    small number pseudogenes E-families, difference in <br /> Ka/Ks &#226;&#8364;&#8220; E. coli        0.056               0.1               1e-8     pseudogenization rates highly statistically significant <br />                                                                       (Table 3). <br /> The table shows essential nonessential members E- <br /> families stronger purifying selection members N-          The larger total number paralogs (Table 1) N-families, <br /> families. The P-values calculated using two-tailed t-test.     coupled observation higher pseudogenization rate, <br />                                                                       suggests recent duplicates N-families enjoy <br />                                                                       higher duplication rate. To test prediction, identified <br />       For example, partitioning paralogous gene sets E-          pairs orthologs lineage-specific paralogs using InPara- <br /> families N-families used test essentiality   noid algorithm (Remm et al. 2001) closely related yeast <br /> a major determinant selection. If strength selection   species (S. cerevisiae, Candida glabrata, Kluyveromyces lactis, Aphis <br /> characteristic gene family membership opposed essenti-       gossypii, Debaryomyces hansenii). These data num- <br /> ality, predict genes E-families, including nonessen-   ber paralogs fixed species diverg- <br /> tial ones, subject significantly stronger purifying       ing common ancestor. For example, given gene had <br /> selection members N-families. We assessed strength        paralog S. cerevisiae, ortholog paralogs C. <br /> of purifying selection standard measures: single feature   glabrata, infer lineage-specific duplication occurred <br /> polymorphism (SFP) densities Saccharomyces cerevisiae genes        divergence C. glabrata. Using approach, we <br /> (Winzeler et al. 2003), K /K s ratios S. cerevisiae&#226;&#8364;&#8220;        calculated number duplicate fixation events mem- <br /> Saccharomyces paradoxus orthologs Escherichia coli K12/           bers E-families N-families. We N-families fix <br /> CFT073 orthologs. The genes stronger purifying         duplication events rate approximately twice E- <br /> selection expected lower SFP densities lower Ka/Ks    families (Table 4) For example, 90% E-family members <br /> ratios (Nei 1987).                                                    S. cerevisiae exactly ortholog C. glabrata, while <br />       We essential nonessential members        80% N-families group. In agreement with <br /> E-families substantially stronger purifying selection       this, 17% E-families &gt;46% N-families fixed <br /> than members N-families, independent species analyzed       duplicate (to produce extra paralog) S. cerevisiae <br /> and method used estimate selection (Table 2). Further-         divergence common ancestor D. hansenii. <br /> more, employed criteria, difference strength     The validity analysis hinges assumption the <br /> selection nonessential genes E-families N-             duplication rate E-families N-families. Alter- <br /> families considerably significant difference        natively, differences duplication rates explain the <br /> between essential nonessential genes species      observed <span id='am-475' about='xsp:pattern' typeof='owl:Thing'>pattern</span> invoking differential fixation dupli- <br /> (Table 2). Thus, strength selection        cates; however, hard imagine genes E-families <br /> salient characteristic gene family membership essential-      N-families duplicate different rates. <br /> ity. Moreover, data Table 2 suggest essentiality se         Taken together, duplication pseudogenization data in- <br /> is necessary sufficient impose purifying selection.    dicate N-families evolve significantly &#226;&#8364;&#339;dynamic&#226;&#8364;? <br /> Instead, given E-families exhibit significant biases   regime E-families. Perhaps, weaker purifying se- <br /> characteristics responsible transitive correla-     lection (Table 2), members N-families higher rate of <br /> tions strength selection, Codon Adaptation        pseudogenization (Table 3) duplicate fixation (Table 4). <br /> Index protein abundance (Hahn Kern 2005; Supplemental          Of course, observations entirely indepen- <br /> Table S4), conclude existence essential paralog     dent greater duplicate fixation rate result a <br /> be used specific marker genes subject strong purifying     higher rate pseudogenization. The relevant issue effect of <br /> selection. We systematic differences protein func- <br /> tions E-families N-families; classes fami- <br />                                                                       Table 3.     Genes pseudogenes E-families N-families in <br /> lies showed comparable functional diversity, E- <br />                                                                       yeast <br /> families included greater fraction molecular chaperones and <br /> proteins related functions, N-families en-                        No.           No.             Pseudogene/gene <br /> riched metabolic enzymes transporters (for annotated lists                       genes         pseudogenes                 ratio <br /> of E-family N-family membership, Supplemental ma-         E-families        278           4                            0.014 <br /> terial).                                                              N-families        656           62                           0.095 <br />                                                                       E/N ratio         0.42          0.06                    P-value &lt; 1e-20 <br /> Selection dynamics duplications divergence <br />                                                                       The ratio pseudogenes/genes seven times E-families prob- <br /> The implication observation sets    ability P &lt; 1e-20 difference chance. The calcula- <br /> genes related evolution share characteristics        tion comparing ratio genes types of <br /> impose selection. We recently showed (Shakhnovich 2006)          families pseudogenes. <br />  <br />  <br />                                                                                                              Genome Research             1531 <br />                                                                                                                 www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 4. Rate duplicate fixation E-families N-families              thermore, observed difference distributions synony- <br /> between S. cerevisiae yeasts                                        mous site divergence E-families N-families compatible <br />                  C. glabrata      K. lactis    A. gossypii     D. hansenii    notion characteristic half-life paralogs in <br />                                                                               E-families longer N-families. Taking Ks <br /> Exactly 1 ortholog pair                                                       value measure evolutionary time, estimated pairs <br />   E-families         0.9             0.86          0.87            0.81       paralogs E-families survive, average, times <br />   N-families         0.8             0.6           0.53            0.41       longer (mean Ks = 3.25) paralogs N-families (mean <br />                                                                               Ks = 0.15; P &lt; 1e-40). Thus, increased evolution- <br /> +1 paralog S. cerevisiae <br />   E-families          0.08           0.12          0.11            0.17       ary dynamism results shift shorter life spans genes <br />   N-families          0.17           0.33          0.41            0.46       N-families. <br />  <br /> +2 paralogs S. cerevisiae                                                  Longer life span duplicates E-families allows greater <br />   E-families        &lt; 0.01         &lt; 0.01        &lt; 0.01          &lt; 0.01       divergence sequence transcriptional regulation <br />   N-families          0.02           0.05          0.03            0.07 <br />                                                                                   The observation longer average time survival duplicates <br /> We used InParanoid software estimate number species-                E-families carries significant implications divergence of <br /> specific paralogs S. cerevisiae yeast species.       protein sequences families. In fact, visual examination of <br /> Each difference E-families N-families significant            graph representation largest paralogous families <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> P-value &lt; 1e-10 using &#226;?&#185;2 tests. <br />                                                                                   S. cerevisiae immediately reveals striking difference the <br />                                                                                   E-families N-families (Fig. 1). Although sizes both <br /> this prodigious rate evolution typical fate para-               families shown Figure 1 similar, N-family much <br /> logs. Does higher pseudogenization rate N-families (Table                  greater number connections node (node degree) the <br /> 3) offset higher duplicate fixation rate (Table 4) resulting             (&#226;&#710;&#188;10 compared &#226;&#710;&#188;1). We observed substantial <br /> shorter overall survival pairs paralogs? To test this,              difference mean clustering coefficients charac- <br /> assume synonymous sites approximately neutral                        terize density connections genes types <br /> evolve clock-like fashion. Under assumption, distri-                families (&#226;&#710;&#188;0.55 N-families compared &#226;&#710;&#188;0.21 E-families; <br /> bution synonymous substitutions paralogous gene families                    P &lt; 1e-3). Both number connections node the <br /> should mirror age distribution paralogs.                               clustering coefficient measure transitivity sequence space, sug- <br />       For N-families, distribution Ks shows best fit            gesting that, average, E-family paralogs diverged farther away <br /> an exponential decay curve. This consistent approxi-                   N-family paralogs. <br /> mately constant probability pseudogenization <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> time.                           We calculated distribution sequence divergence be- <br /> In sharp contrast, number paralogs E-families correlates                tween pairs paralogs E-families N-families using two <br /> linearly increase synonymous site divergence (Fig. 2).                standard measures nonsynonymous substitution rate, the <br /> The shape distribution Figure 2 E-families                   number nonsynonymous substitutions nonsynonymous <br /> explained model pseudogenization rate                        site (Ka), amino acid sequence identity (see Supplemental <br /> these families drops paralogs diverge (data shown). Fur-                   material). Indeed, criteria, paralogs E-families were <br />                                                                                                             characterized greater diver- <br />                                                                                                             gence (Fig. 3A,B). For example, S. ce- <br />                                                                                                             revisiae, average amino acid se- <br />                                                                                                             quence identity paralogs E- <br />                                                                                                             families &#226;&#710;&#188;40% compared &#226;&#710;&#188;73% <br />                                                                                                             N-families (Fig. 3B). Qualitatively <br />                                                                                                             similar results observed E- <br />                                                                                                             families N-families E. coli and <br />                                                                                                             Caenorhabditis elegans (Supplemental <br />                                                                                                             Table S6a,b). Furthermore, differ- <br />                                                                                                             ence function distribution <br />                                                                                                             family size shown Supplemental <br />                                                                                                             Table S7a,b. Finally, nearly identical re- <br />                                                                                                             sults obtained broad range of <br />                                                                                                             cutoffs used classify genes paralogs <br />                                                                                                             (Table 5; Supplemental Table S6a,b). <br />                                                                                                             These results unequivocally that, on <br />                                                                                                             average, paralogs E-families are <br />                                                                                                             separated protein se- <br />                                                                                                             quence space paralogs N- <br />                                                                                                             families. <br />                                                                                                                   In fact, difference sequence <br />                                                                                                             divergence distributions large <br /> Figure 2. Distributions synonymous substitution rates (Ks) pairs paralogs E-familes (gray      wanted assess pre- <br /> circles) N-families (black squares). The x-axis Ks values y-axis number pairs. <br /> The Ks bins 0.3 E-families 0.1 N-families. The black dashed line fit          dictive value differentiating between <br /> exponential decay paralogs N-families (R2 = 0.96, P &lt; 0.001). The solid gray line       E-families N-families absence <br /> linear fit Ks divergence paralogs E-families (R2 = 0.86, P &lt; 0.001).                          essentiality data. We performed re- <br />  <br />  <br /> 1532      Genome Research <br />              www.genome.org <br />                                                                                                                        Evolution gene families <br />  <br />  <br />  <br />  <br /> Figure 3. Sequence divergence E-families N-families. (A) Distributions nonsynonymous substitution rates (Ka) pairs paralogs in <br /> N-families (black squares) E-families (red circles). The mean Ka values &#226;&#710;&#188;0.5 E-families &#226;&#710;&#188;0.14 N-families (P &lt; 1e-50). (B) Distributions <br /> of sequence identity pairs paralogs E-families (red circles) N-families (black squares). The mean sequence identity &#226;&#710;&#188;40% pairs of <br /> paralogs E-families &#226;&#710;&#188;73% pairs paralogs N-families (P &lt; 1e-40). <br />  <br />  <br /> ciprocal analysis, is, examined characteristics                In contrast, paralogs N-families appeared coregu- <br /> sequence divergence differentiate E-families                  lated TFs (Fig. 5). Thus, average, E-family <br /> N-families. To end, used Receiver Operating Characteris-                members diverged substantially farther N- <br /> tic (ROC) statistics based average separation paralogs             family members sequence, regulatory <br /> families classify genes E-families N-families               context. This consistent notion longer time of <br /> explicitly invoking essentiality marker. The ROC curve                  survival paralogs E-families allows modification upstream <br /> Figure 4 shows &#226;&#710;&#188;80% E-families paralogs                regions increases chances acquisition new regula- <br /> far diverged &#226;&#710;&#188;20% N-families. An better separation                tory mechanisms. <br /> E-families N-families obtained using clustering <br /> coefficient classification criterion: 73% E-families            Discussion <br /> were captured single false positive (N-family),              We present evidence gene family membership general <br /> this analysis covered larger families (those                 reliable indicator strength purifying selection acting <br /> more members), resulting classification 11 E-families 18 <br /> N-families yeast. Thus, separation paralogs sequence <br /> space E-families N-families dissimilar classifica- <br /> tion based solely sequence divergence, mostly, reproduces the <br /> partitioning based using essential paralogs markers. <br />       Our results E-families explore protein se- <br /> quence space, duplication divergence paralogs, to <br /> a greater depth N-families. This important bear- <br /> ings impact family membership functional diver- <br /> gence. So far, presented evidence paralogs E- <br /> families survive longer, enjoy lower rate pseudogenization, <br /> and diverge sequence farther paralogs N-families. Sub- <br /> functionalization (division ancestral pleiotropy) character- <br /> ized strong purifying selection paralogs dupli- <br /> cation (Force et al. 1999; Lynch Force 2000) be <br /> aided long survival duplicates E-families. Since <br /> sequence divergence taken characteristic sub- <br /> functionalization, reasonably hypothesize paralogs <br /> from E-families preferentially evolve subfunctionaliza- <br /> tion scenario. <br />       If diverging paralogs E-families follow subfunc- <br /> tionalization (or neofunctionalization) routes, expect                Figure 4. ROC curves predicting family belongs E- or <br />                                                                                 N-class based average sequence divergence paralogs clustering <br /> to observe divergence sequence expression               coefficient. Paralogous gene families (strongly connected components of <br /> regulation paralogs adapt new biological niches. We                   DDG) sorted average sequence divergence pairs of <br /> compared extent transcription factor (TF) sharing                paralogs (black squares) clustering coefficients (green triangles). <br /> paralogs classes families. In accord notion              Only families members analyzed, families <br />                                                                                 clustering coefficient 0 excluded uninformative. This <br /> of greater functional diversification genes E-families,              results 23 E-families 63 N-families sequence divergence <br /> of paralogs families common transcription                   calculation 12 E-families 18 N-families clustering coeffi- <br /> factors binding upstream regions (Harbison et al. 2004).               cient calculation. The red diagonal line represents random expectation. <br />  <br />  <br />                                                                                                                        Genome Research              1533 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 5. Robustness sequence divergence E-families                       The magnitude observed differences two <br /> and N-families Ks cutoff used building DDG graph S.              types families sharply contrasts previous observations weak <br /> cerevisiae <br />                                                                               moderate correlations various functional evolu- <br />              No. genes      No. genes       &lt; Ka&gt;      &lt;Ka&gt;       tionary characteristics genes determined genome scale <br /> Ks cutoff    E-families     N-families     E-families    N-families     (Hurst Smith 1999; Hirsh Fraser 2001; Pal et al. 2001; <br />                                                                               Jordan et al. 2003; Drummond et al. 2005). We believe the <br /> 5                 275               658          0.509344       0.147632 <br />                                                                               case E-families N-families different this <br /> 4.4               270               659          0.509814       0.147657 <br /> 3.8               264               642          0.507339       0.144439      partitioning attains sharp separation genes classes, <br /> 3.2               246               625          0.500473       0.140768      measured characteristics evolution reflect collective <br /> 2.6               204               585          0.490572       0.128384      behaviors distinct gene sets. The tight link between <br /> 2.3               159               568          0.445489       0.124553 <br />                                                                               gene family membership strength purifying selection <br /> 2                 114               491          0.404539       0.104033 <br /> 1.7                69               434          0.307063       0.091604      likely reflect relationship function con- <br /> 1.4                34               369          0.20307        0.070387      straints. Gene families share set related functions <br />                                                                               (Todd et al. 2001; Shakhnovich Max Harvey 2004) a <br /> Average nonsynonymous sequence divergence (Ka) calculated            similar range functional constraints, translates a <br /> the pairs paralogs. The differences &lt;Ka&gt; E-families         characteristic strength purifying selection. The typical range of <br /> N-families statistically significant (P &lt; 0.001). (An expanded version <br /> is included Supplemental Table S1.)                                        functions associated certain level purifying selection, in <br />                                                                               turn, determines subsequent fate duplicates, is, <br /> on gene. Purifying selection, linked functional constraints,             likelihood gene fixed duplication, diverges a <br /> may affect course molecular evolution                 particular degree paralog, undergoes subfunctional- <br /> influencing speed divergence, affecting fate           ization neofunctionalization. Further investigations the <br /> of paralogs duplication. Specifically, paralogs            origins homogeneity selection acting members paralo- <br /> in E-families include essential genes subject                gous families uncover novel determinants selection <br /> stronger purifying selection genes N-families (Table 2)               rooted commonalities shared members family <br /> without essential paralogs. Thus, salient observation            specific individual genes. <br /> that genes families paralogs experience similar levels                 Another possible conclusion study subfunc- <br /> selection. Although clear-cut difference dis-             tionalization transient phase gene evolution, genes <br /> tribution protein functions E-families N-families,             divide ancestral functions soon undergo neofunctionaliza- <br /> appears presence essential genes corre-             tion. This hypothesis consistent observations the <br /> lates greater biological importance E-family members                  apparently decreasing rate pseudogenization passage of <br /> (Table 2). In spite stronger selective constraints, paralogs            time (Fig. 2) change transcriptional regulation paralogs <br /> E-families exhibit greater sequence divergence N-families                E-families (Fig. 5). A similar sub-, neofunctionalization model <br /> (Fig. 3). Furthermore, difference exploration se-               recently proposed He Zhang study the <br /> quence space E-families N-families large              evolution protein&#226;&#8364;&#8220;protein interaction networks (He and <br /> we use average sequence divergence paralogs cluster-              Zhang 2005). <br /> ing coefficient predictive manner classify large propor-                  The results presented uncover surprisingly strong link <br /> tion families using existence essential para-            gene&#226;&#8364;&#8482;s membership paralogous family, con- <br /> log criterion (Fig. 4).                                                  straints imposed evolution purifying selection, the <br />       The difference average sequence divergence between <br /> members E-families N-families attributed the <br /> differential dynamics molecular evolution families. <br /> Specifically, N-families evolutionarily dynamic, is, <br /> duplicates families tend pseudogenes shortly <br /> after duplication (Table 3) enjoy higher fixation rate <br /> (Table 4). These results compatible recent evidence in- <br /> dicating that, yeast, functionally important genes tend to <br /> duplicate (He Zhang 2006). The net outcome of <br /> the difference evolutionary dynamics E-families and <br /> N-families paralogs E-families longer life spans <br /> (Fig. 2). Thus, average, paralogs E-families older, is, <br /> have evolving longer time, paralogs N-families <br /> (Fig. 2). It likely that, thanks longer average life <br /> span, paralogs E-families follow subfunctionalization or <br /> neofunctionalization paths paralogs N- <br /> families. This hypothesis supported demonstration of <br /> more extensive transcriptional rewiring (divergence TF- <br /> binding sites) paralogs E-families N-families <br /> (Fig. 5). These findings robust respect method of <br /> family identification and, importantly, species              Figure 5. Shared transcription factors regulating pairs paralogs from <br />                                                                               E-families N-families. In E-families, 92% pairs TF- <br /> families analyzed&#226;&#8364;&#8221;very similar results obtained                 binding sites common, 29% pairs N-families shared <br /> yeast, nematode C. elegans, bacterium E. coli (see            sites TFs (P &lt; 1e-50). The distribution N-families is <br /> Supplemental material).                                                       characterized heavy tail pairs sharing TFs. <br />  <br />  <br /> 1534        Genome Research <br />              www.genome.org <br />                                                                                                                     Evolution gene families <br />  <br />  <br /> characteristics gene family evolution duplication. The clas-                generation protein database search programs. Nucleic Acids Res. <br />                                                                                   25: 3389&#226;&#8364;&#8220;3402. <br /> sification genes E-families N-families useful <br />                                                                               Conery, J.S. Lynch, M. 2001. Nucleotide substitutions the <br /> starting point variety future studies relationships             evolution duplicate genes. Pac. Symp. Biocomput. 2001: 167&#226;&#8364;&#8220;178. <br /> among evolution genes, genomes, phenotypes.                        Cormen, T.H. 2001. Introduction algorithms, 2nd ed. MIT Press, <br />                                                                                   Cambridge, MA. <br />                                                                               Davis, J.C. Petrov, D.A. 2004. Preferential duplication conserved <br />                                                                                   proteins eukaryotic genomes. PLoS Biol. 2: e55. <br /> Methods                                                                       Drake, J.W., Charlesworth, B., Charlesworth, D., Crow, J.F. 1998. <br />                                                                                   Rates spontaneous mutation. Genetics 148: 1667&#226;&#8364;&#8220;1686. <br /> To construct DDG species (S. cerevisiae, C.             Drummond, D.A., Bloom, J.D., Adami, C., Wilke, C.O., Arnold, F.H. <br /> elegans, E. coli), complete sets protein sequences                2005. Why highly expressed proteins evolve slowly. Proc. Natl. Acad. <br />                                                                                   Sci. 102: 14338&#226;&#8364;&#8220;14343. <br /> the respective genomes extracted GenBank data-                  Drummond, D.A., Raval, A., Wilke, C.O. 2006. A single determinant <br /> <span id='am-474' about='xsp:base' typeof='owl:Thing'>base</span>, all-against-all sequence comparison performed                    dominates rate yeast protein evolution. Mol. Biol. Evol. <br /> using BLAST program (Altschul et al. 1997; ftp://                             23: 327&#226;&#8364;&#8220;337. <br /> ftp.ncbi.nih.gov). Each protein node DDG,                 Enright, A.J., Kunin, V., Ouzounis, C.A. 2003. Protein families and <br />                                                                                   TRIBES genome sequence space. Nucleic Acids Res. 31: 4632&#226;&#8364;&#8220;4638. <br /> results comparisons, represented BLAST                     Force, A., Lynch, M., Pickett, F.B., Amores, A., Yan, Y.L., and <br /> scores, expectation (e) values, amino acid sequence identities,                Postlethwait, J. 1999. Preservation duplicate genes by <br /> are edges connecting nodes. The edges assigned                       complementary, degenerative mutations. Genetics 151: 1531&#226;&#8364;&#8220;1545. <br /> weights using BLAST scores, amino acid sequence identities,                Fraser, A.G., Kamath, R.S., Zipperlen, P., Martinez-Campos, M., <br />                                                                                   Sohrmann, M., Ahringer, J. 2000. Functional genomic analysis <br /> the ratio nonsynonymous (Ka) synonymous (Ks)                       C. elegans chromosome I systematic <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>. Nature <br /> substitution rates. To calculate Ka Ks values, amino acid                 408: 325&#226;&#8364;&#8220;330. <br /> sequence alignments traced corresponding                     Fraser, H.B., Hirsh, A.E., Steinmetz, L.M., Scharfe, C., Feldman, <br /> nucleotide sequence alignments, Ka Ks calculated                     M.W. 2002. Evolutionary rate protein interaction network. <br />                                                                                   Science 296: 750&#226;&#8364;&#8220;752. <br /> using PAML (Yang 1997; Lynch Conery 2000; Yang                        Gerdes, S.Y., Scholle, M.D., Campbell, J.W., Balazsi, G., Ravasz, E., <br /> Nielsen 2000; Conery Lynch 2001). A similar procedure                     Daugherty, M.D., Somera, A.L., Kyrpides, N.C., Anderson, I., <br /> used Enright et al. (2003).                                                    Gelfand, M.S., et al. 2003. Experimental determination system <br />       Paralogous families identified finding strongly                 level analysis essential genes Escherichia coli MG1655. J. <br />                                                                                   Bacteriol. 185: 5673&#226;&#8364;&#8220;5684. <br /> connected components DDG described Cormen                        Giaever, G., Chu, A.M., Ni, L., Connelly, C., Riles, L., Veronneau, S., <br /> (2001) (see Supplemental Table S1). We define strongly con-                     Dow, S., Lucau-Danila, A., Anderson, K., Andre, B., et al. 2002. <br /> nected component set nodes path exists                    Functional profiling Saccharomyces cerevisiae genome. Nature <br /> any pair. The principal results presented obtained us-                  418: 387&#226;&#8364;&#8220;391. <br />                                                                               Gu, X. 1999. Statistical methods testing functional divergence after <br /> ing BLAST cutoff 1e-15 dS = 5. However, nearly identical                 gene duplication. Mol. Biol. Evol. 16: 1664&#226;&#8364;&#8220;1674. <br /> results obtained broad range cutoffs (Table 3)              Gu, X. 2001a. Mathematical modeling functional divergence after <br /> After identifying strongly connected components DDG,                   gene duplication. J. Comput. Biol. 8: 221&#226;&#8364;&#8220;234. <br /> high-throughput essentiality data used divide fami-               Gu, X. 2001b. A site-specific measure rate difference gene <br />                                                                                   duplication speciation. Mol. Biol. Evol. 18: 2327&#226;&#8364;&#8220;2330. <br /> lies E- N-classes. The yeast gene essentiality data              Hahn, M.W. Kern, A.D. 2005. Comparative genomics centrality <br /> were obtained high-throughput knockout experiments                           essentiality eukaryotic protein-interaction networks. <br /> (Giaever et al. 2002); C. elegans gene essentiality data                 Mol. Biol. Evol. 22: 803&#226;&#8364;&#8220;806. <br /> from genome-wide RNAi knockdown experiments (Fraser et                    Harbison, C.T., Gordon, D.B., Lee, T.I., Rinaldi, N.J., Macisaac, K.D., <br />                                                                                   Danford, T.W., Hannett, N.M., Tagne, J.B., Reynolds, D.B., Yoo, J., et <br /> al. 2000; Kamath et al. 2003; Simmer et al. 2003), E. coli                al. 2004. Transcriptional regulatory <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> eukaryotic genome. <br /> essentiality data GenBank (ftp://ftp.ncbi.nih.gov)                      Nature 431: 99&#226;&#8364;&#8220;104. <br /> (Gerdes et al. 2003; details, Supplemental material).             Harrison, P.M. Gerstein, M. 2002. Studying genomes the <br /> The data yeast SNPs Winzeler et al. (2003). The                      aeons: Protein families, pseudogenes proteome evolution. J. Mol. <br />                                                                                   Biol. 318: 1155&#226;&#8364;&#8220;1174. <br /> transcription factor data ChIP-chip experiments (Lee                Harrison, P.M., Echols, N., Gerstein, M.B. 2001. Digging dead <br /> et al. 2002; Harbison et al. 2004).                                               genes: An analysis characteristics pseudogene <br />       We used InParanoid program (http://inparanoid.                          population Caenorhabditis elegans genome. Nucleic Acids Res. <br /> cgb.ki.se/) identify orthologs species-specific paralogs                   29: 818&#226;&#8364;&#8220;830. <br />                                                                               Harrison, P., Kumar, A., Lan, N., Echols, N., Snyder, M., Gerstein, <br /> from yeast species (S. cerevisiae, C. glabrata, K. lactis, A.            M. 2002. A small reservoir disabled ORFs yeast genome and <br /> gossypii, D. hansenii). The genomes obtained                    implications dynamics proteome evolution. J. Mol. Biol. <br /> NCBI (http://www.ncbi.nlm/nih.gov).                                               316: 409&#226;&#8364;&#8220;419. <br />                                                                               He, X. Zhang, J. 2005. Rapid subfunctionalization accompanied by <br />                                                                                   prolonged substantial neofunctionalization duplicate gene <br />                                                                                   evolution. Genetics 169: 1157&#226;&#8364;&#8220;1164. <br /> Acknowledgments                                                               He, X. Zhang, J. 2006. Higher duplicability important genes <br />                                                                                   yeast genomes. Mol. Biol. Evol. 23: 144&#226;&#8364;&#8220;151. <br /> The authors thank Eugene Shakhnovich invaluable sup-                  Hirsh, A.E. Fraser, H.B. 2001. Protein dispensability rate of <br /> port review manuscript. We thank Matthew Hahn                     evolution. Nature 411: 1046&#226;&#8364;&#8220;1049. <br /> for proposing duplication rate study helpful                 Hurst, L.D. Smith, N.G. 1999. Do essential genes evolve slowly? <br /> comments regarding manuscript. Additionally, extend                        Curr. Biol. 9: 747&#226;&#8364;&#8220;750. <br />                                                                               Jordan, I.K., Rogozin, I.B., Wolf, Y.I., Koonin, E.V. 2002. Essential <br /> our appreciation Charles DeLisi, Tim Reddy, Joe Mellor, Julian                 genes evolutionarily conserved nonessential genes <br /> Mintseris, Bioinformatics program Boston                     bacteria. Genome Res. 12: 962&#226;&#8364;&#8220;968. <br /> University discussion insights.                                       Jordan, I.K., Wolf, Y.I., Koonin, E.V. 2003. No simple dependence <br />                                                                                   protein evolution rate number protein&#226;&#8364;&#8220;protein <br />                                                                                   interactions: Only prolific interactors tend evolve <br />                                                                                   slowly. BMC Evol. Biol. 3: 1. <br /> References                                                                    Jordan, I.K., Wolf, Y.I., Koonin, E.V. 2004. Duplicated genes evolve <br />                                                                                   slower singletons despite initial rate increase. BMC Evol. <br /> Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller,       Biol. 4: 22. <br />     W., Lipman, D.J. 1997. Gapped BLAST PSI-BLAST: A new              Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., <br />  <br />  <br />  <br />                                                                                                                     Genome Research              1535 <br />                                                                                                                        www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />     Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M., et al. 2003.          Remm, M., Storm, C.E., Sonnhammer, E.L. 2001. Automatic <br />     Systematic functional analysis Caenorhabditis elegans genome         clustering orthologs in-paralogs pairwise species <br />     using RNAi. Nature 421: 231&#226;&#8364;&#8220;237.                                            comparisons. J. Mol. Biol. 314: 1041&#226;&#8364;&#8220;1052. <br /> Keightley, P.D. Eyre-Walker, A. 1999. Terumi Mukai riddle        Shakhnovich, B.E. 2006. Relative contributions structural <br />     <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>deleterious mutation</span> rates. Genetics 153: 515&#226;&#8364;&#8220;523.                       des <br /> </body></html>