<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Oct. 1997, p. 6325&#226;&#8364;&#8220;6334                                                                              Vol. 179, No. 20 <br /> 0021-9193/97/$04.0010 <br /> Copyright &#194;&#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />             Rapamycin Specifically Interferes Developmental <br />                    Response Fission Yeast Starvation <br />                                         R. WEISMAN,1* M. CHODER,1                 AND   Y. KOLTIN2 <br />               Department Molecular Microbiology Biotechnology, Faculty Life Sciences, Tel Aviv University, <br />                 Tel Aviv 69978, Israel,1 Millennium Pharmaceuticals, Inc., Cambridge, Massachusetts 021392 <br />  <br />            Rapamycin microbial macrolide belongs family immunosuppressive drugs suppress <br />          immune blocking stages signal transduction T lymphocytes. In Saccharomyces cerevisiae <br />          cells, T lymphocytes, rapamycin inhibits growth cells arrested G1 stage cell cycle. <br />          Rapamycin effective antifungal agent, affecting growth yeast filamentous fungi. Unex- <br />          pectedly, observed rapamycin apparent effect vegetative growth Schizosaccharomyces <br />          pombe. Instead, drug effective cells experience starvation. Under conditions, <br />          homothallic wild-type cells normally mate undergo sporulation. In presence rapamycin, this <br />          sexual development process strongly inhibited cells adopt alternative physiological option enter <br />          stationary phase. Rapamycin strongly inhibits sexual development haploid cells prior stage sexual <br />          conjugation. In contrast, drug slight inhibitory effect sporulation diploid cells. A genetic <br />          approach applied identify signal transduction pathway inhibited rapamycin. The results <br />          indicate rapamycin did suppress derepression sexual development strains which <br />          adenylate cyclase deleted cyclic AMP-dependent protein kinase encoded pka1 mutated. Nor <br />          did rapamycin inhibit unscheduled meiosis observed pat1-114 mutants. Overexpression ras11, an <br />          essential gene sexual development, did rescue sterility rapamycin-treated cells. However, <br />          expression activated allele, ras1Val17, antagonized effect rapamycin restored ability the <br />          cells respond mating signals presence drug. We discuss possible mechanisms the <br />          inhibitory effect rapamycin sexual development S. pombe. <br />  <br />  <br />    Rapamycin, cyclosporin A (CsA), FK506 microbial                  activity binds S. cerevisiae proteins encoded the <br /> products exhibit immunosuppressive activity (1, 5, 6, 21,             TORs. The human homologs known FRAP, RAFT, <br /> 41, 70). The importance immunosuppressive drugs                  RAPT1, mTOR (7, 13, 66, 67). The TORs their <br /> as therapeutic agents catalyzed search define                mammalian counterparts large proteins contain a <br /> molecular mechanism action. Studies mechanism                    phosphatidylinositol 3-kinase motif. It noteworthy the <br /> have carried mainly certain types mammalian                human FRAP protein rapamycin-sensitive regulator of <br /> cells budding yeast Saccharomyces cerevisiae.                  mitogen-stimulated p70 S6 kinase (p70S6k) (8). The activity of <br /> These studies suggested features mode              p70S6k itself previously reported numerous other <br /> of action drugs conserved evolution. In               studies inhibited rapamycin (11, 14, 23, 43, 64). How- <br /> all systems studied, immunosup-                 ever, clear main downstream effec- <br /> pressants bind high affinity cytoplasmic receptors                  tors rapamycin-sensitive pathway. Several proteins, <br /> termed immunophilins (reviewed references 28 69).                    activity dependent independent p70S6k <br /> CsA binds cyclophilin, FK506 rapamycin,                   activity, suggested. One line investigation sug- <br /> are structurally related, bind different immunophilin called              gests main downstream effectors involved the <br /> FKBP. Both cyclophilins FKBPs exhibit similar enzymatic                 control protein synthesis, control results inhi- <br /> activities peptidyl-prolyl cis-trans isomerization.                      bition phosphorylation ribosomal protein S6 by <br />    Complexes CsA-cyclophilin FK506-FKBP inhibit                     p70S6k (14, 36, 73). Rapamycin inhibit process of <br /> T-cell receptor signalling pathway binding inhibiting             cap-dependent translation initiation inhibition phos- <br /> the activity Ca21-dependent protein phosphatase, cal-                phorylation eIF-4E binding protein (4, 9, 47). Other <br /> cineurin. As consequence, translocation cyto-                suggestions downstream effectors rapamy- <br /> plasm nucleus T-cell transcription factor, NF-AT,             cin-sensitive pathway involved transcriptional regula- <br /> is inhibited (15, 51). In S. cerevisiae, CsA-cyclophilin                tion. For example, rapamycin inhibits phosphorylation of <br /> FK506-FKBP complexes block recovery G1 arrest                      transcription factor CREMt. As result, transcriptional acti- <br /> induced mating pheromone (25) inhibit vegetative                 vation genes, normally occurs following stimulation of <br /> growth strains (40, 74). The physiological target            adenylyl cyclase pathway, inhibited (17). CREMt a <br /> drug-immunophilin complexes yeast calcineurin (12, 25).              member cyclic AMP (cAMP)-responsive element-bind- <br /> In fission yeast Schizosaccharomyces pombe CsA causes                    ing factors. The possibility factors play role in <br /> abnormal multiseptated <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> (76, 80). This abnormal                    rapamycin-sensitive pathway supported the <br /> phenotype assumed result inhibition cal-                  finding rapamycin inhibits transcription proliferat- <br /> cineurin activity (80).                                                     ing cell nuclear antigen preventing binding cAMP- <br />    The rapamycin-FKBP complex does affect calcineurin                   responsive element-binding elements promoter (24). <br />                                                                                Rapamycin inhibits interleukin 2-induced activity of <br />                                                                             cyclin-dependent kinases (CDKs) Cdc2 Cdk2 (2, 56, 60). <br />    * Corresponding author. Fax: 972-3-6409407. E-mail: ronitt@post          This inhibition thought result inhibition the <br /> .tau.ac.il.                                                                 degradation CDK inhibitor Kip1 (60). The molecular mech- <br />  <br />                                                                      6325 <br />  6326       WEISMAN ET AL.                                                                                                                             J. BACTERIOL. <br />  <br />  <br /> anism links inhibition rapamycin TOR proteins                                           TABLE 1. Strains used study <br /> and CDKs elucidated (for review, reference                         Strain                                         Description <br /> 1). <br />                                                                                                                     2 <br />    Studies effect rapamycin S. cerevisiae led                  ED812 .............................972 h <br /> a better understanding cellular response rapamycin                     ED787a .............................975 h1 <br />                                                                                     ED666a .............................ura4-D18 leu1-32 ade6-M210 h1 <br /> the molecular level. Rapamycin causes cell growth arrest S.                      ED667a .............................ura4-D18 leu1-32 ade6-M216 h1 <br /> cerevisiae, cells arrest early G1 phase cell cycle                TA07d ...............................ura4-D18/ura4-D18 leu1-32/leu1-32 ade6-M216/ <br /> (3, 31, 39). As mammalian cells, toxic effect drug                                                            ade6-M210 h1/h2 <br /> requires formation complex yeast FKBP (31,                        TA22d ...............................leu1-32 h1 <br /> 39). Two related phosphatidylinositol 3-kinase-like proteins,                       TA23d ...............................leu1-32 h2 <br /> TOR1 TOR2, identified genetic screens,                         ED1091a ...........................pat1-114 h2 <br /> the targets rapamycin-FKBP12 complex (10, 31, 32, 42).                       ED1054a ...........................pat1-114 leu1-32 h2 <br /> The loss TOR function results inhibition transla-                      ED813a .............................h90 <br />                                                                                     SP837b...............................ura4-D18 leu1-32 ade6-M216 h90 <br /> tion initiation induces physiological changes <br />                                                                                     CHP364c ...........................pka1-216 leu1-32 ade6-M216 h90 <br /> characteristic starved cells entering stationary phase (3).                      CHP559c ...........................git2-1::LEU2 Fbp1::ura41 ura4::fbp1-lacZ leu1-32 <br /> These results suggest TOR signalling pathway regu-                                                                 ade6-M216 h90 <br /> lates translation initiation response <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> growth sig- <br />                                                                                       a <br /> nals (3, 19).                                                                           P. Fantes S. MacNeill, Edinburgh University, Edinburgh, United King- <br />                                                                                     dom. <br />    Thus far, effect rapamycin S. pombe                         b <br />                                                                                         A. Cohen, Hebrew University, Jerusalem, Israel. <br /> reported. S. pombe well-studied yeast widely used                     c <br />                                                                                         C. S. Hoffman, Boston College, Chestnut Hill, Mass. <br /> a model organism higher eukaryotes, especially studies                         d <br />                                                                                         This work. <br /> on cell cycle regulation (48, 62). Concerning evolution, is <br /> distantly related budding yeast S. cerevisiae some <br /> respects similar higher eukaryotes S. cerevi-                    tains 0.1% glucose. Cells grown experiments 25&#194;&#176;C, unless <br /> siae (65, 71).                                                                      indicated. <br />    S. pombe multiplies primarily haploid state. Haploid                         Rapamycin purchased Biomol Research Laboratories, Plymouth, <br />                                                                                     Mass. For stock solution, 1 mg rapamycin dissolved 1 ml 1:1 <br /> cells mating types, h1 (P) h2 (M) (reviewed                         dimethyl sulfoxide (DMSO)-methanol solution kept 220&#194;&#176;C. Rapamycin <br /> reference 22). Wild-type S. pombe cells homothallic (h90);                      added final concentration 0.2 mg/ml liquid- agar-containing <br /> that is, switch mating types h1 h2                       media, unless indicated. An equal volume drug vehicle solution <br /> every generation. Heterothallic strains, express                        (1:1 DMSO-methanol) used control experiments. <br />                                                                                        Standard genetic procedures performed described references 30 <br /> either h1 h2 mating type phenotypes, available.                         38. Transformation S. pombe cells performed electroporation (63). <br /> Upon starvation cells committed sexual                         General molecular techniques performed described reference 68. <br /> differentiation advance stationary phase. If                         Assays mating efficiency. Cells grown 25&#194;&#176;C EMM, appropriately <br /> sexual differentiation pathway chosen, cells opposite mat-                    supplemented amino acids required, density approximately 5 3 <br />                                                                                     106 cells/ml. Then, cultures washed times double-distilled <br /> ing types conjugate form diploid zygotes. If starvation con-                     water reinoculated nitrogen-free liquid medium (EMM-N) density <br /> ditions prevail, diploid zygotes undergo meiosis sporu-                     approximately 5 3 106 cells/ml incubated 25&#194;&#176;C. Aliquots were <br /> lation (reviewed reference 22).                                                  taken indicated time points, gentle sonication, numbers of <br />    cAMP plays critical role regulation sexual devel-                    cells, zygotes, spores counted microscope. Whenever solid <br />                                                                                     EMM-N medium used, 5 ml medium containing 5 3 106 cells spotted. <br /> opment S. pombe. Addition cAMP medium                               After 3 days incubation 25&#194;&#176;C, toothpick used pick cells <br /> mutants produce high intracellular cAMP level inhibit                       center patch, cells briefly sonicated examined <br /> sexual differentiation, attenuation cAMP path-                       microscopically. <br /> way results propensity sporulate rich <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                         The percentage mating calculated dividing number zygotes, <br />                                                                                     asci, free spores number total cells. One zygote ascus was <br /> growth conditions (49, 79). Entry sexual pathway                        counted cells, spore counted half cell. In experiment <br /> also dependent establishment mating pheromone                         500 1,000 cells counted. <br /> signalling pathway. The single S. pombe homolog mam-                            Assay sporulation efficiency diploids. A diploid strain (TA07) was <br /> malian Ras oncoprotein, ras11 gene product, essential                        constructed according method described Moreno et al. (55). Diploid cells <br />                                                                                     grown starved described haploid cells mating assay. After <br /> for sexual differentiation thought modulate sensitivity                   3 days incubation starvation medium, aliquots taken cells were <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> mating pheromones (26, 58, 59; reviewed reference                         briefly sonicated examined microscopically. The percentage sporulation <br /> 33). Remarkably, S. pombe ras11 gene product es-                         calculated dividing number asci free spores number of <br /> sential vegetative growth (26, 58). As mammals,                          total cells. One ascus counted cells, spore counted a <br />                                                                                     half cell. In experiment 500 1,000 cells counted. <br /> ras11 product participates regulation protein kinase                       Determination cell viability stationary phase heat shock. <br /> cascade ultimately regulates member mitogen-                          Cells grown EMM 25&#194;&#176;C mid-log phase presence absence <br /> activated protein kinase family. In S. pombe cascade                        rapamycin. The cells washed reinoculated EMM-N the <br /> composed products byr21, byr11, spk11 (29, 57,                        presence absence rapamycin. Aliquots sampled indicated times. <br />                                                                                     Cells counted, appropriately diluted, plated yeast extract (YE) <br /> 75).                                                                                plates. Cell viability determined capacity cells form colonies on <br />    We report rapamycin does affect vegetative growth                       supplemented YE 30&#194;&#176;C. <br /> and entrance stationary phase S. pombe                               Cells exposed heat shock 5 days incubation EMM-N. <br /> distinct effect sexual development early stage prior                    Aliquots taken incubated 47&#194;&#176;C various times. Viability was <br />                                                                                     determined described above. <br /> conjugation. <br />  <br />                                                                                                                       RESULTS <br />                         MATERIALS AND METHODS <br />                                                                                       Rapamycin does inhibit vegetative growth S. pombe. <br />   Yeast strains, media, general techniques. The S. pombe strains used for <br /> this study listed Table 1. All media used study based those <br />                                                                                     Rapamycin shows strong inhibitory effect vegetative <br /> described previously (55) described reference 72.         growth S. cerevisiae cells concentration ,0.1 mg/ml <br /> Edinburgh minimal medium (EMM)-N contains nitrogen; EMM lowG con-                (31, 39). Surprisingly, rapamycin did inhibit <br />  VOL. 179, 1997                                                      RAPAMYCIN INHIBITS SEXUAL DEVELOPMENT IN S. POMBE                        6327 <br />  <br />  <br />                                                                                 homothallic h90 cells. Upon shift EMM-N absence <br />                                                                                 drug, cells gradually conjugated final value of <br />                                                                                 48% reached. Cells starved presence of <br />                                                                                 rapamycin reached maximum mating efficiency 7%. <br />                                                                                 Microscopic examination revealed morphology the <br />                                                                                 starved cells presence rapamycin similar to <br />                                                                                 stationary (resting) haploid cells. In presence of <br />                                                                                 rapamycin cells arrested small cell size, typical starved <br />                                                                                 cells, did develop conjugation tubes (Fig. 2; compare <br />                                                                                 panels C D). In addition, effect agglutination was <br />                                                                                 observed. Cells untreated culture started agglutinate <br />                                                                                 23 h, rapamycin-treated cultures did not <br />                                                                                 agglutinate course experiment (Fig. 2; com- <br />                                                                                 pare panels C D). Sexual agglutination early indica- <br />                                                                                 tion sexual activity (reviewed reference 22). The lack of <br />                                                                                 agglutination presence rapamycin suggests rapa- <br />                                                                                 mycin inhibits early sexual differentiation stages. <br />    FIG. 1. Rapamycin does affect wild-type h1 colony growth. Cells were <br />                                                                                    Vegetative growth presence rapamycin causes mat- <br /> plated EMM YE plates containing rapamycin drug vehicle alone.      ing incompetence. Although rapamycin apparent effect <br /> Colonies larger rich (YE) plates minimal (EMM) plates,   vegetative growth S. pombe, examined possi- <br /> size affected presence rapamycin.                              bility rapamycin affects growing cells effect is <br />                                                                                 observed starvation. To end, homothallic cells <br />                                                                                 grown exponential phase presence rapamycin. <br /> the growth S. pombe. Similar-size colonies obtained                     Cells washed times reinoculated into <br /> when wild-type homothallic (h90) heterothallic (h1 h2)                    EMM-N absence rapamycin. We determined <br /> strains grown complex minimal medium                       cells underwent normal sexual differentiation once <br /> the presence (0.2 0.5 mg/ml) absence rapamycin. This                   removed drug. <br /> insensitivity drug shown cells grown range                  The exposure rapamycin growth phase a <br /> of temperatures 20 35&#194;&#176;C (Fig. 1 data                        dramatic effect ability cells undergo sexual <br /> shown). A microscopic examination did reveal distin-                    differentiation. More 20-fold reduction mating effi- <br /> guishable morphological changes associated rapamy-                     ciency EMM-N exposure drug growth <br /> cin treatment, cells divided cell size un-                   observed, compared efficiency observed un- <br /> treated cells (see Fig. 2A B). The doubling time                     treated cells (Fig. 3 4). A decrease mating <br /> wild-type h1 strain liquid complex medium 32&#194;&#176;C                     efficiency observed cells grown starved in <br /> presence rapamycin similar control with-                  presence rapamycin: 500-fold reduction in <br /> out drug (135 125 min, respectively). These results                     mating efficiency comparison cells ex- <br /> indicate that, contrast effect vegetative growth               posed rapamycin (Fig. 3 Fig. 4). Thus, pregrowth with <br /> of S. cerevisiae, rapamycin does interfere vegeta-                 rapamycin rendered cells incompetent mating. This sug- <br /> tive growth S. pombe.                                                        gests target rapamycin exists vegetative <br />    Rapamycin strongly inhibits conjugation S. pombe. In                    growth. One explanation drug blocks, an <br /> series experiments examined ability rapamycin-                     irreversible way, biochemical pathway, allowing <br /> treated cells respond conditions normally induce                    cells committed sexual differentiation path- <br /> opposite-mating-type cells undergo sexual differentiation.                   way. <br /> These conditions primarily consist <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation (re-                    Since growth presence rapamycin affects subse- <br /> viewed reference 22). Conventionally, low-<span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                  quent sexual differentiation, able examine whether <br /> medium malt extract (ME), low-nitrogen medium                         rapamycin affects cells mating types equally mat- <br /> (EMM-N), low-glucose medium (EMM lowG) used                         ing type specific. We examined mating efficiency crosses of <br /> strong induction sexual differentiation. Table 2 depicts                 wild-type heterothallic strains mating type <br /> mating efficiencies obtained opposite-mating-                    grown presence rapamycin. We that, irre- <br /> type cells starved presence 0.2 mg rapamycin                   spective mating types grown rapamy- <br /> per ml. The results indicate starvation                   cin, mating efficiency reduced 2.5-fold relative that <br /> conditions used, mating efficiency strongly reduced (9-               crosses untreated strains opposite mating types. Thus, <br /> 60-fold) presence drug.                                           function affected rapamycin needs be <br />    Mating requires opposite mating types, h1 h2,                   active mates. When mating types grown in <br /> be present close proximity. These conditions                          presence rapamycin plated drug-free plates, <br /> achieved homothallic strains, switch mat-                 mating efficiency reduced ninefold relative for <br /> ing types h1 h2, mixture opposite                        crosses untreated cells. Therefore, mating efficiency <br /> mating types heterothallic strains. When homothallic                       observed crosses mating type grown in <br /> strain heterothallic strains starved presence                 presence rapamycin intermediate one, indicating <br /> rapamycin, significant difference inhibitory ef-                 untreated cells partially suppress effect rapa- <br /> fects rapamycin homothallic strain mix-                    mycin. <br /> ture heterothallic strains observed. These results indi-                    The differences mating efficiencies above <br /> cate rapamycin does interfere process                         crosses demonstrated exposing patches crosses <br /> responsible mating type switching homothallic strains.                   iodine vapor (Fig. 5). As deprivation nutrients a <br />    Figure 3 demonstrates inhibitory effect rapamycin                  signal entry meiosis, newly formed zygotes rapidly <br /> the entry sexual differentiation pathway wild-type                  undergo meiosis sporulation. Carbohydrate accumulation <br />  6328       WEISMAN ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Rapamycin does affect morphology growing cells exerts dramatic effect morphology nitrogen-starved cells. (A B) h90 wild-type <br /> (ED813) cell samples taken exponentially growing culture EMM supplemented drug vehicle (A) rapamycin (B). (C D) h90 <br /> wild-type (ED813) cell samples taken 3 days shift EMM liquid EMM-N containing drug vehicle (C) rapamycin (D). <br />  <br />  <br />  <br /> is hallmark sporulation process detected                      later stages, sporulation, constructed diploid <br /> staining iodine vapor (45, 61). Figure 5 (left panel) dem-                      strain using haploid strains complementing alleles <br /> onstrates patch cross untreated cells stained                    ade6 gene (see reference 55). The diploid strain (TA07) <br /> dark, indicating presence spores. The patch cross                     formed crossing strain ED667 (ura4-D18 leu1-32 ade6- <br /> which mating types pretreated rapamycin ap-                           M216 h1) ED666 (ura4-D18 leu1-32 ade6-M210 h2) and <br /> pears yellow, indicating lack sporulation. The patches                     determining adenine prototrophs progeny. <br /> crosses mating type pretreated                               Because diploid resulted mating auxo- <br /> rapamycin appear greyish, indicating reduced sporulation.                            trophic mutants examined nutritional <br />    Rapamycin little effect sporulation                      markers effect sensitivity rapamycin during <br /> diploids. To determine rapamycin affects                            conjugation. When ED666 crossed ED667 on <br /> early stages sexual differentiation affect                      EMM-N plates presence rapamycin, ninefold re- <br />  VOL. 179, 1997                                                        RAPAMYCIN INHIBITS SEXUAL DEVELOPMENT IN S. POMBE                                        6329 <br />  <br />  <br />    TABLE 2. Inhibition mating treatment rapamycina <br />                                                               Mating efficiency <br /> Medium composition                  Mating type <br />                                                                    (%) <br />  <br /> EMM-N                                h90                             58 <br /> EMM-N 1 rapa                         h90                              5 <br /> EMM lowG                             h90                             52 <br /> EMM lowG 1 rapa                      h90                              6 <br /> EMM-N                                h1 3   h2                       59 <br /> EMM-N 1 rapa                         h1 3   h2                        1 <br /> ME                                   h1 3   h2                       60 <br /> ME 1 rapa                            h1 3   h2                        4 <br />     h (ED813) cells 1:1 mixture h1 (ED787) h2 (ED812) cells <br />   90 <br />  <br /> were grown absence rapamycin mid-log phase; cells then <br /> plated solid media indicated compositions presence absence <br /> of rapamycin (rapa). Mating efficiency calculated described Materials <br /> and Methods. The results mean values separate experiments. <br />  <br />  <br />                                                                                       FIG. 4. Rapamycin inhibits mating heterothallic strains. Heterothallic h1 <br /> duction mating efficiency compared                        h2 cells grown EMM presence drug vehicle (EMM) <br /> same cross absence rapamycin observed (data                      presence rapamycin (EMM1R). In mid-log phase, cells har- <br /> shown). Thus, auxotrophic traits used cross                     vested washed, opposite-mating-type cultures mixed to- <br />                                                                                    gether 1:1 ratio. A total 5 3 106 cells plated EMM-N plate <br /> effect drug sensitivity.                                                    supplemented drug vehicle (EMM-N) rapamycin (EMM- <br />    The effect rapamycin sporulation diploid TA07                      N1R). The mating efficiency determined 3 days postplating, described <br /> was determined. Interestingly, significant decrease sporu-                   Materials Methods. <br /> lation observed diploid cells induced to <br /> sporulate presence rapamycin (Fig. 6; second <br /> bars left). A microscopic examination diploid cells                    development starved haploids, exerts little effect on <br /> which sporulated presence rapamycin revealed                        sporulation starved diploids. This observation reinforces <br /> that asci containing spores formed                         conclusion drug effect specific early stage <br /> efficiency absence drug. However, small                    developmental response starvation, mating even <br /> decrease sporulation efficiency (2.4-fold) observed                    initiated, little effect later stages. <br /> the diploid cells grown presence rapamycin                         Treatment rapamycin does interfere entrance <br /> before subjecting nitrogen-limiting conditions (Fig. 6;                    stationary phase. The transition S. pombe cells the <br /> third bar left). Note results                      mitotic cycle meiotic sexual cycle occurs G1 phase <br /> experiment compared similar experi-                          (reviewed reference 22). Under nitrogen starvation condi- <br /> ments haploids, inhibitory effect                         tions, cells arrested G1 phase enter <br /> cells grown presence rapamycin                         stationary phase, analogous mammalian G0 state, or <br /> than 500-fold (Fig. 3 4). Therefore, concluded that, <br /> whereas rapamycin dramatic inhibitory effect sexual <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 5. Untreated cells partially suppress sterility rapamycin-treated <br />                                                                                    cells. Heterothallic h1 h2 cells pregrown presence either <br />                                                                                    drug vehicle rapamycin mixed plated EMM-N <br />                                                                                    plates rapamycin (left) rapamycin (right). Plates incubated <br />                                                                                    3 days stained iodine vapor. None patches the <br />                                                                                    rapamycin-containing plate (right) stained dark color, rapamycin in- <br />    FIG. 3. Inhibition mating rapamycin. h90 wild-type cells (ED813) grow-    hibits sexual development. On plate rapamycin (left) various degrees <br /> ing exponentially EMM presence drug vehicle (h, F)       dark staining detected, depending partners the <br /> rapamycin (&#226;&#8211;&#160;, E) washed times water recultured ;5 3         crosses pregrown presence rapamycin. The table plates <br /> 106 cells/ml EMM-N medium containing drug vehicle (h, &#226;&#8211;&#160;) rapamycin       indicates, patch, pretreatment applied partner cross. <br /> (F, E). At indicated time points, samples removed mating          R indicates pregrowth rapamycin. The numbers parentheses indicate <br /> efficiencies determined described Materials Methods.                mating efficiencies observed microscopic examination. <br />  6330       WEISMAN ET AL.                                                                                                                      J. BACTERIOL. <br />  <br />  <br />                                                                                    TABLE 3. Effect rapamycin treatment matinga efficiencies <br />                                                                                             strains overexpressing ras11 ras1Val17 <br />                                                                                                                                               Mating efficiency <br />                                                                                         Medium                       Plasmidb <br />                                                                                                                                                    (%) <br />  <br />                                                                                    EMM-N                      pREP1 (vector)                       63 <br />                                                                                    EMM-N 1 rapa               pREP1 (vector)                       15 <br />                                                                                    EMM-N                      pDB2489ras1                          52 <br />                                                                                    EMM-N 1 rapa               pDB2489ras1                           8.0 <br />                                                                                    EMM-N                      pDB2489ras1-Val17                     8.0 <br />                                                                                    EMM-N 1 rapa               pDB2489ras1-Val17                    ,0.01 <br />                                                                                      a <br />                                                                                        SP837 h90 cells transformed indicated plasmids grown to <br />                                                                                    mid-log phase appropriately supplemented EMM washed. A total of <br />                                                                                    ;5 3 106 cells plated appropriately supplemented EMM-N plates the <br />                                                                                    presence absence rapamycin (rapa). Mating efficiency calculated as <br />                                                                                    described Materials Methods. <br />                                                                                      b <br />                                                                                        Plasmids pDB2489ras1 pDB9ras1-Val17 kindly provided D. <br />                                                                                    Hughes, Chester Beatty Laboratories, London, United Kingdom. <br />  <br />  <br />    FIG. 6. Rapamycin small inhibitory effect sporulation a <br /> diploid. TA07 diploid cells grown EMM presence drug          rapamycin. Note shift nitrogen limiting <br /> vehicle (EMM) rapamycin (EMM1R). In mid-log phase, cells were <br /> harvested, washed, plated EMM-N drug vehicle rapamycin.         medium untreated h1 culture continued divide several <br /> The sporulation efficiencies determined 3 days postplating.                   times completely arrested (Fig. 7). It clear that <br />                                                                                    rapamycin-treated cultures divided times before <br />                                                                                    arrest, number cell divisions were <br /> initiate sexual differentiation. Some mutants                     arrested slightly smaller untreated culture. <br /> impaired sexual differentiation impaired                           The fact rapamycin does affect entrance into <br /> capability entering stationary phase. Such mutants                     stationary phase shown experimental proce- <br /> low viability starvation conditions (for examples                        dures. One characteristics S. pombe stationary cells is <br /> references 37 54).                                                             acquisition heat resistance (16). Cells stationary phase <br />    In order determine effect rapamycin ability                    tolerate heat shock 47&#194;&#176;C. Therefore, 5 days after <br /> of cells enter stationary phase, examined viability                  incubation EMM-N, rapamycin-treated h1 h90 cul- <br /> of cells starved EMM-N presence                            tures (see above) subjected heat shock 47&#194;&#176;C, cell <br /> rapamycin. h1 h90 cells grown EMM mid-log                           <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> values compared heat-shocked h1 <br /> phase presence rapamycin reinoculated                      untreated cells. The results obtained indicated cells the <br /> EMM-N presence rapamycin. Rapamycin inhibited                            different cultures equally resistant heat shock (data not <br /> the mating activity h90 culture (resulting                     shown). Thus, experiments indicate rapamycin- <br /> 0.01% mating efficiency). We monitored viability                        treated cells enter sustain stationary phase normally. <br /> cells (presumably) remained haploid, demonstrated                          An activated allele ras1, ras1Val17, restores ability of <br /> Fig. 7. The results cell viability long-term station-                 rapamycin-treated cells respond mating signals. The <br /> ary-phase cultures EMM-N significantly affected                      phenotypic effect rapamycin resembles effect loss <br />                                                                                    function known genes play role sexual <br />                                                                                    differentiation. More 20 genes required mating have <br />                                                                                    identified. One genes ras11, close homolog <br />                                                                                    mammalian ras proto-oncogene (26, 58). Haploid cells <br />                                                                                    carrying ras1 null <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> viable mate or <br />                                                                                    respond mating pheromones. In contrast, cells carrying the <br />                                                                                    constitutively activated allele, ras1Val17, hyperresponsive to <br />                                                                                    pheromones. Such cells grow normally vegetative <br />                                                                                    conditions, starvation presence both <br />                                                                                    mating types produce long conjugation tubes mate <br />                                                                                    poorly (27, 34). Taken together, results suggest a <br />                                                                                    transient pulse ras1 activity required early nitrogen <br />                                                                                    starvation (reviewed reference 22). <br />                                                                                       We examined effect overexpression wild- <br />                                                                                    type ras11 activated ras1 alleles rapamycin-treated <br />                                                                                    cells. Overexpression ras11 did effect cells <br />                                                                                    grown starved presence rapamycin, and <br />                                                                                    suppression defect conjugation observed (Ta- <br />                                                                                    ble 3). However, overexpression ras1Val17 restore <br />    FIG. 7. Rapamycin does affect cell viability long-term stationary cul- <br />                                                                                    ability cells respond mating pheromones. Under <br /> tures. Wild-type h1 h90 cells grown EMM mid-log phase        starvation conditions cells produced elongated conjugation <br /> absence presence rapamycin shifted EMM-N absence        tubes, presence rapamycin (Fig. 8). This result <br /> or presence rapamycin, respectively. After shift nitrogen-limiting   suggests rapamycin affect early step the <br /> medium, samples cultures aliquoted indicated times. Each <br /> sample counted, appropriately diluted, plated YE plates. The number <br />                                                                                    mating pathway, upstream step determined ras1. It is <br /> of colonies formed plate counted, inferred concentration    determined targets rapamycin the <br /> viable cells time point calculated.                                    ras11 protein lie pathway. Remarkably, rapamycin <br />  VOL. 179, 1997                                                       RAPAMYCIN INHIBITS SEXUAL DEVELOPMENT IN S. POMBE                                      6331 <br />  <br />  <br />  <br />  <br />   FIG. 8. Rapamycin does inhibit agglutination development conjugation tubes cells carrying ras1Val17 allele. SP837 cells carrying pDB2489ras1- <br /> Val17 grown EMM mid-log phase, washed times water recultured EMM-N presence drug vehicle (A) rapamycin <br /> (B). Cells lacking plasmid treated rapamycin shown panel C. Elongation conjugation tubes agglutination, presence the <br /> absence rapamycin, seen cells expressing ras1Val17 parental wild-type cells. <br />  <br />  <br />  <br />  <br /> does exert effect Ras1 pathway                          ATP cAMP, catalytic subunit protein kinase A <br /> mammalian cells (53). Thus, rapamycin affects                       (pka1, known git6) readily mate rich medium. <br /> directly S. pombe ras11 pathway remains intriguing                          We mating efficiency git2 git6 mu- <br /> open question.                                                                     tants complete minimal medium affected <br />    Rapamycin does inhibit phenotypes associated                       rapamycin (data shown). Therefore, evident that <br /> mutants cAMP/pka1-dependent pathway. The pat11                              rapamycin does exert effect downstream cAMP- <br /> gene (previously referred ran11) encodes protein kinase                    dependent protein kinase. <br /> which represses sexual differentiation essential for <br /> the maintenance vegetative cell division growth (35, 52, <br /> 61). Loss function pat11 causes unscheduled meiosis, <br /> irrespective ploidy, mating type, growth conditions. The <br /> pat1-114 strain temperature-sensitive mutant dis- <br /> tinctive phenotype restrictive temperature, i.e., growth <br /> cessation development irregular numbers spores. <br /> Tests conducted mutant indicate rapamycin does <br /> not suppress uncontrolled meiosis pat1-114 cells the <br /> restrictive temperature (Fig. 9). These results suggest that <br /> rapamycin involved suppression functions that <br /> are required expression pat1-114 lethal pheno- <br /> type. Conditions able suppress pat1-driven meiosis <br /> and associated growth defect include increase the <br /> level intracellular cAMP overexpression pka11, the <br /> gene cAMP-dependent kinase (49). Therefore, the <br /> inability rapamycin suppress pat1-114 lethal pheno- <br /> type suggests rapamycin involved stim- <br /> ulation cAMP/pka1 pathway. In agreement this <br /> suggestion observations rapamycin does inter- <br /> fere aspects vegetative growth meiosis (data pre- <br /> sented above). Such aspects life cycle S. pombe be <br /> affected alternations intracellular cAMP levels (reviewed <br /> in reference 79). <br />    In addition, rapamycin did suppress mu- <br />                                                                                       FIG. 9. Rapamycin does rescue pat1-114 growth defect. pat1-114 h1 <br /> tants cAMP/pka1 pathway derepressed                           cells streaked EMM presence drug vehicle (A B) or <br /> sexual development. Mutants defective genes coding                      rapamycin (C D). Incubation 25 33&#194;&#176;C. Rapamycin did not <br /> adenylate cyclase (git2, known cyr1), converts                       suppress <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> pat1-114 cells restrictive temperature. <br />  6332     WEISMAN ET AL.                                                                                                   J. BACTERIOL. <br />  <br />  <br />                          DISCUSSION                                  rapamycin does exert effect downstream the <br />                                                                      ras11 gene product. The simplest explanation data is <br />    Rapamycin specifically inhibits sexual development re-         target rapamycin acts upstream ras11 gene <br /> sponse starvation. We report here, time,        product, directly indirectly. An alternative explana- <br /> effect rapamycin fission yeast. We contrast      tion rapamycin acts pathway parallel the <br /> to effect S. cerevisiae fungi, rapamycin does   ras1 pathway shares ras1 pathway of <br /> exhibit apparent effect vegetative growth S.           functions. For example, mating signals mediated by <br /> pombe. Instead, effect drug apparent         G protein subunit, gpa1. The gpa1-mediated signals are <br /> upon starvation. Under conditions nutrient deprivation            transmitted parallel ras1 pathway converge <br /> wild-type homothallic cells enter sexual devel-       byr2 (78). <br /> opment pathway stationary phase. Rapamycin specifi-              In view findings model systems, direct effect <br /> cally inhibits sexual development pathway stage       rapamycin upstream ras11 gene product likely. <br /> of agglutination does affect entry stationary       In mammalian cells, Ras Raf appears of <br /> phase. Our conclusion rapamycin inhibits sexual develop-        pathway affected rapamycin (14, 20, 53). The RAS <br /> ment early step supported observations.      proteins S. cerevisiae unlikely mediate rapa- <br /> First, cellular agglutination, discernible stages   mycin effect. This suggestion based findings that <br /> in conjugation process, inhibited rapamycin. Second,       constitutively activated mutants RAS/cAMP pathway in <br /> zygote formation inhibited rapamycin, observed           S. cerevisiae, including resulting expression of <br /> standard mating assays. And third, rapamycin slight         dominant activated RAS2 allele, RAS2Val19, sup- <br /> effect, effect all, sporulation diploid cells,    press growth arrest induced rapamycin (3). The genetic <br /> indicating later stages sexual development, is, en-     relationships pathway mediates rapa- <br /> trance meiosis spore formation, far affected    mycin effect ras11 pathway remain determined. <br /> rapamycin.                                                              cAMP plays important role regulation sexual <br />    The lack effect rapamycin vegetative growth          differentiation S. pombe. The addition cAMP me- <br /> does reflect inability drug penetrate vegeta-       dium gene mutations lead increase cAMP <br /> tively growing cells. Our findings cells rendered sterile   level pka1 activity inhibit sexual differentiation. Our findings <br /> if grown presence rapamycin (and             support hypothesis rapamycin inhibits sexual <br /> starved absence drug) indicate drug           differentiation stimulation cAMP/pka1 pathway. <br /> enters growing cells, effect apparent       First, rapamycin does suppress unscheduled meiosis of <br /> upon starvation. It possible target rapamycin         pat1-114 mutants. In contrast, lethal phenotype sup- <br /> exists vegetatively growing cells activated     pressed processes raise cAMP intracellular level, <br /> becomes effective starvation. It noteworthy        introduction cAMP growth medium and <br /> the pronounced effect, 500-fold reduction        overexpression pka11 (49). In addition, elevated cAMP <br /> mating efficiency, observed rapamycin present dur-        levels cause impairment normal control cell <br /> ing growth starvation.                                      growth. For example, loss function regulatory subunit <br />    The inhibition rapamycin sexual development          cAMP-dependent protein kinase (cgs1) inhibits mating <br /> mating type specific. When mating type grown            meiosis, brings increase cell <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> at <br /> with drug mixed untreated opposite          division (18). Rapamycin, hand, does exert <br /> mating type, resulting mating efficiency independent      detectable effects cell growth cell size. Furthermore, <br /> the mating type grown drug. Inter-           rapamycin does exert effect downstream cAMP- <br /> estingly, resulting mating efficiency crosses        dependent protein kinase. This shown inability of <br /> intermediate; is, lower (by 2.5-fold) mating    rapamycin inhibit derepression sexual development of <br /> efficiency untreated cells higher (by 4-fold)        mutants carrying defective adenylate cyclase protein kinase <br /> mating efficiency crosses mating types         A catalytic subunit genes. <br /> grown presence rapamycin. Thus, appears           Possible mechanisms inhibitory effect rapamycin <br /> untreated cells partially suppress effect rapamycin.      sexual development. Rapamycin strongly inhibits sexual <br />    The effect rapamycin sexual development mutants. We         conjugation haploid cells, process requires condi- <br /> have initiated investigations mode action           tions starvation. In contrast, processes are <br /> rapamycin examining effect rapamycin sexual             affected starvation, is, entrance meiosis en- <br /> development mutants. Our findings agreement          trance stationary phase, slightly affected not <br /> the hypothesis rapamycin acts early stage      affected rapamycin. Thus, appears cells able <br /> sexual development pathway. However, specific pathway(s)         sense nutrient depletion, switch mitotic cycles <br /> or protein(s) affected rapamycin remains unknown. A cen-          sexual development haploid cells specifically inhibited <br /> tral element sexual differentiation pathway ras11      rapamycin. One possible explanation sterility effect <br /> gene. The ras11 gene product essential mating        rapamycin inhibits expression specific genes <br /> meiosis dispensable vegetative growth. Like mam-      required early sexual differentiation process, <br /> malian counterpart, ras11 gene product participates       genes required pheromone production re- <br /> regulation protein kinase cascade ultimately regulates     sponse. However, slight effect rapamycin sporu- <br /> a member mitogen-activated protein kinase family (29,         lation diploids argue direct effect of <br /> 57, 75). Our studies sterility associated         rapamycin pheromone production, process of <br /> rapamycin suppressed overexpression wild-type       meiosis requires intact pheromone signalling pathway (re- <br /> ras11 gene. However, expression ras11 activated allele,       viewed reference 79). Nevertheless, conjugation more <br /> ras1Val17, restores ability rapamycin-treated cells    sensitive pheromone activities meiosis (46). Hence, is <br /> agglutinate respond mating conditions, demonstrated        possible rapamycin causes reduction mating pher- <br /> by development conjugation tubes starvation, in-         omone signals reduction leads far more <br /> dependent presence drug. Therefore, clear        dramatic effect mating meiosis. <br />  VOL. 179, 1997                                                            RAPAMYCIN INHIBITS SEXUAL DEVELOPMENT IN S. POMBE                                          6333 <br />  <br />  <br />    The effect rapamycin S. pombe cells appears                                R. Abraham. 1996. Direct inhibition signalling function mam- <br /> sight substantially different effects cell                           malian target rapamycin phosphoinositide 3-kinase inhibitors, wort- <br />                                                                                              mannin LY294002. EMBO J. 15:5256&#226;&#8364;&#8220;5267. <br /> systems, does affect cell cycle progression. However,                      10.   Cafferkey, R., P. R. Young, M. M. McLaughlin, D. J. Bergsma, Y. Koltin, <br /> close look regulation cell cycle S. pombe                               G. M. Sathe, L. Faucette, W.-K. Eng, R. K. Johnson, G. P. Livi. 1993. <br /> indeed suggest effect rapamycin caused inhi-                               Dominant missense mutations novel yeast protein related mammalian <br /> bition function early G1, Start. This inhibition                              phosphatidylinositol 3-kinase VPS34 abrogate rapamycin cytotoxicity. <br />                                                                                              Mol. Cell. Biol. 13:6012&#226;&#8364;&#8220;6023. <br /> may apparent starvation. In S. pombe,                             11.   Calvo, V., C. M. Crews, T. A. Vik, B. E. Bierer. 1992. Interleukin 2 <br /> G1 phase particularly short rapidly growing cells. Under                               stimulation p70 S6 kinase activity inhibited immunosuppressant <br /> rapid-growth conditions G1-S size control cryptic,                                 rapamycin. Proc. Natl. Acad. Sci. USA 89:7571&#226;&#8364;&#8220;7575. <br /> wild-type cells end mitosis                           12.   Cardenas, M. E., R. S. Muir, T. Breuder, J. Heitman. 1995. Targets of <br />                                                                                              immunophilin-immunosuppressant complexes distinct highly conserved <br /> minimum size entry S phase (reviewed reference                                   regions calcineurin A. EMBO J. 14:2772&#226;&#8364;&#8220;2783. <br /> 44). Therefore, inhibition functions required                           13.   Chiu, M. I., H. Katz, V. Berlin. 1994. RAPT1, mammalian homolog of <br /> for control G1, specifically, functions delay cell cycle                            yeast Tor, interacts FKBP12/rapamycin complex. Proc. Natl. Acad. <br /> progression phase, cause effect                                Sci. USA 91:12574&#226;&#8364;&#8220;12578. <br />                                                                                        14.   Chung, J., C. J. Kuo, G. R. Crabtree, J. Blenis. 1992. Rapamycin-FKBP <br /> rapid growth. In contrast, G1 control limiting                                   specifically blocks growth dependent activation signaling 70 kd <br /> under conditions mitosis initiated reduced size,                            S6 protein kinases. Cell 69:1227&#226;&#8364;&#8220;1236. <br /> such nitrogen starvation. A delay G1 necessary                         15.   Clipstone, N. A., G. R. Crabtree. 1992. Identification calcineurin a <br /> under starvation conditions allow accumulation                               key signalling enzyme T-lymphocyte activation. Nature 357:695&#226;&#8364;&#8220;697. <br />                                                                                        16.   Costello, G., L. Rodgers, D. Beach. 1986. Fission yeast enters sta- <br /> of mass initiation processes required                             tionary phase G0 state mitotic G1 G2. Curr. Genet. 11:119&#226;&#8364;&#8220; <br /> mating. Thus, sterility rapamycin-treated cells in-                               125. <br /> deed result inhibition function(s) normally                       17.   Groot, R. P., L. M. Ballou, P. Sassone-Corsi. 1994. Positive regulation <br /> enables cells linger G1. In regard phenotype                              cAMP-responsive activator CREM p70 S6 kinase: alternative <br />                                                                                              route mitogen induced gene expression. Cell 79:81&#226;&#8364;&#8220;91. <br /> of rum1 mutants S. pombe relevant. The rum11 gene                            18.   DeVoti, J., G. Seydoux, D. Beach, M. McLeod. 1991. Interactions be- <br /> has central role regulation G1 progression. It en-                               tween ran11 protein kinase cAMP dependent protein kinase negative <br /> codes stoichiometric inhibitor cdc2, central CDK                                    regulators fission yeast meiosis. EMBO J. 10:3759&#226;&#8364;&#8220;3768. <br /> which controls G1-S G2-M transitions. Cells                              19.   Di Como, C. J., K. T. Arndt. 1996. Nutrients, Tor proteins, <br />                                                                                              stimulate association TAP42 type 2A phosphatases. Genes Dev. <br /> rum1 deleted exhibit defect vegetative growth                                  10:1904&#226;&#8364;&#8220;1916. <br /> are profoundly sterile (50). Rapamycin cause effect                             20.   Downward, J. 1994. Regulating S6 kinase. Nature 371:378&#226;&#8364;&#8220;379. <br /> similar rum1 mutants. This hypothesis currently                          21.   Dumont, F. J., M. J. Staruch, S. J. Koprak, M. R. Melino, N. H. Sigal. <br /> being examined laboratory.                                                            1990. Distinct mechanisms suppression murine T cell activation the <br />                                                                                              related macrolides FK506 rapamycin. J. Immunol. 144:251&#226;&#8364;&#8220;258. <br />                                                                                        22.   Egel, R. 1989. Mating type genes, meiosis sporulation, p. 31&#226;&#8364;&#8220;68. In A. <br />                           ACKNOWLEDGMENTS                                                    Nasim, P. Young, B. F. Johnston (ed.), Molecular biology fission <br />                                                                                              yeast. Academic Press, New York, N.Y. <br />    We thank P. A. Fantes S. A. MacNeill (University Edinburgh,                  23.   Ferrari, S., R. B. Pearson, M. Siegmann, S. C. Kozma, G. Thomas. 1993. <br /> Edinburgh, United Kingdom) S. pombe strains, A. Cohen (Hebrew                            The immunosuppressant rapamycin induces inactivation p70S6k through <br /> University, Jerusalem, Israel) SP837 strain, D. Hughes (Chester                      dephosphorylation novel set sites. J. Biol. Chem. 268:16091&#226;&#8364;&#8220;16094. <br /> Beatty Laboratories, London, United Kingdom) ras1 ste6 plas-                   24.   Feuerstein, N., D. Huang, M. B. Prystowsky. 1995. Rapamycin selec- <br /> mids, C. S. Hoffman (Boston College, Chestnut Hill, Mass.)                           tively blocks interleukin-2-induced proliferating cell nuclear antigen gene <br /> git2 git6 mutants. We grateful M. Mevarech (Tel Aviv                              expression T lymphocytes. Evidence inhibition CREB/ATF binding <br /> University, Tel Aviv, Israel) encouraging begin work                      activities. J. Biol. Chem. 270:9454&#226;&#8364;&#8220;9458. <br /> for helpful discussions P. A. Fantes (University Edinburgh)                  25.   Foor, F., S. A. Parent, N. Morin, A. M. Dahl, N. Ramadan, G. Chrebet, K. A. <br />                                                                                              Bostian, J. B. Nielsen. 1992. Calcineurin mediates inhibition FK506 <br /> for useful comments course work.                                          cyclosporin recovery a-factor arrest yeast. Nature 360:682&#226;&#8364;&#8220; <br />    This work supported partly Israeli Ministry Health.                         684. <br />                                                                                        26.   Fukui, Y., Y. Kaziro, M. Yamamoto. 1986. Mating pheromone-like <br />                                  REFERENCES                                                  diffusible factor released Schizosaccharomyces pombe. EMBO J. 5:1991&#226;&#8364;&#8220; <br />  1. Abraham, R. T., G. J. Wiederrecht. 1996. Immunopharmacology                       1993. <br />     rapamycin. Annu. Rev. Immunol. 14:483&#226;&#8364;&#8220;510.                                         27.   Fukui, Y., T. Kozasa, Y. Kaziro, T. Takeda, M. Yamamoto. 1986. Role <br />  2. Albers, M. W., R. T. Williams, E. J. Brown, A. Tanaka, F. L. Hall, S. L.             ras homolog life cycle Schizosaccharomyces pombe. Cell 44: <br />     Schreiber. 1993. FKBP-rapamycin inhibits cyclin-dependent kinase activity              329&#226;&#8364;&#8220;336. <br />     cyclin D1-Cdk association early G1 osteosarcoma cell line.          28.   Galat, A. 1993. Peptidyl-prolyl cis trans isomerases: immunophilins. Eur. <br />     J. Biol. Chem. 268:22825&#226;&#8364;&#8220;22829.                                                          J. Biochem. 216:689&#226;&#8364;&#8220;707. <br />  3. Barbet, N. C., U. Shneider, S. B. Helliwell, I. Stansfield, M. F. Tuite,       29.   Gotoh, Y., E. Nishida, M. Shimanuki, T. Toda, Y. Imai, M. Yamamoto. <br />     M. N. Hall. 1996. TOR controls translation initiation early G1 progres-              1993. Schizosaccharomyces pombe Spk1 tyrosine-phosphorylated protein <br />     sion yeast. Mol. Biol. Cell 7:25&#226;&#8364;&#8220;42.                                                  functionally related Xenopus mitogen-activated protein kinase. Mol. Cell. <br />  4. Berreta, L., A.-C. Gingras, Y. Svitkin, M. N. Hall, N. Sonenberg. 1996.              Biol. 13:6427&#226;&#8364;&#8220;6434. <br />     Rapamycin blocks phosphorylation 4E-BP1 inhibits cap-depen-             30.   Gutz, H., H. Heslot, U. Leupold, N. Loperino. 1974. Schizosaccharomy- <br />     dent initiation translation. EMBO J. 15:658&#226;&#8364;&#8220;664.                                      ces pombe, p. 395&#226;&#8364;&#8220;446. In R. D. King (ed.), Handbook genetics, vol. 1. <br />  5. Bierer, B. E., P. S. Mattila, R. F. Standaert, L. A. Herzenberg, S. J. Burakoff,         Plenum Publishing Corporation, New York, N.Y. <br />     G. Crabtree, S. L. Schreiber. 1990. Two distinct signal transmission           31.   Heitman, J., N. R. Movva, M. N. Hall. 1991. Targets cell cycle arrest <br />     pathways T lymphocytes inhibited complexes formed                   immunosuppressant rapamycin yeast. Science 253:905&#226;&#8364;&#8220;909. <br />     immunophilin FK506 rapamycin. Proc. Natl. Acad. Sci. USA             32.   Helliwell, S., P. Wagner, J. Kunz, M. Deuter-Reinhard, R. Henriquez, and <br />     87:9231&#226;&#8364;&#8220;9235.                                                                            M. Hall. 1994. TOR1 TOR2 structurally functionally similar but <br />  6. Brazelton, T. R., R. E. Morris. 1996. Molecular mechanisms action              identical phosphatidylinositol kinase homologues yeast. Mol. Biol. <br />     new xenobiotic immunosuppressive drugs: tacrolimus (FK506), sirolimus                    Cell 5:105&#226;&#8364;&#8220;118. <br />     (rapamycin), mycophenolate mofetil leflunomide. Curr. Immunol. 8:              33.   Hughes, D. A., M. Yamamoto. 1993. Prokaryotic simple eukaryotic <br />     710&#226;&#8364;&#8220;720.                                                                                 systems, p. 123&#226;&#8364;&#8220;146. In J. Kurjan B. L. Taylor (ed.), Signal transduction. <br />  7. Brown, E. J., M. W. Albers, T. B. Shin, K. Ichikawa, C. T. Keith, W. S. Lane,            Academic Press, New York, N.Y. <br />     S. L. Schreiber. 1994. A mammalian protein targeted G1 arresting            34.   Hughes, D. A., Y. Fukui, M. Yamamoto. 1990. Homologous activators of <br />     rapamycin-receptor complex. Nature 369:756&#226;&#8364;&#8220;758.                                          ras fission budding yeast. Nature 344:355&#226;&#8364;&#8220;357. <br />  8. Brown, E. J., P. A. Beal, C. T. Keith, J. Chen, T. B. Shin, S. L. Schreiber.   35.   Iino, Y., M. Yamamoto. 1985. Negative control initiation of <br />     1995. Control p70 S6 kinase kinase activity FRAP vivo. Nature                meiosis Schizosaccharomyces pombe. Proc. Natl. Acad. Sci. USA 82:2447&#226;&#8364;&#8220; <br />     377:441&#226;&#8364;&#8220;446.                                                                             2451. <br />  9. Brunn, G. J., J. Williams, C. Sabers, G. Wiederrecht, J. C. Lawrence, Jr.,     36.   Jefferies, H. B. J., C. Reinhard, S. C. Kozma, G. Thomas. 1994. Rapa- <br />  6334         WEISMAN ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />       mycin selectively represses translation &#226;&#8364;&#732;polypyrimidine tract&#226;&#8364;&#8482; mRNA          59. Nielsen, O., J. Davey, R. Egel. 1992. The ras1 function Schizosaccha- <br />       family. Proc. Natl. Acad. Sci. USA 91:4441&#226;&#8364;&#8220;4445.                                        romyces pombe mediates pheromone induced transcription. EMBO J. 11: <br /> 37.   Kitamura, K., T. Nakagawa, C. Shimoda. 1990. Novel sterile mutants               1391&#226;&#8364;&#8220;1395. <br />       fission yeast Schizosaccharomyces pombe defective re-            60. Nourse, J., E. Firpo, W. M. Flanagan, S. Coats, K. Polyak, M.-H. Lee, J. <br />       sponse starvation. Curr. Genet. 18:315&#226;&#8364;&#8220;321.                                          Massague, G. R. Crabtree, J. M. Roberts. 1994. Interleukin-2-mediated <br /> 38.   Kohli, J., H. Hottinger, P. Munz, A. Strauss, P. Thuriaux. 1977. Genetic            elimination p27kip1 cyclin dependent kinase inhibitor prevented by <br />       mapping Schizosaccharomyces pombe mitotic meiotic analysis                rapamycin. Nature 372:570&#226;&#8364;&#8220;573. <br />       induced haploidisation. Genetics 87:471&#226;&#8364;&#8220;489.                                        61. Nurse, P. 1985. Mutants fission yeast Schizosaccharomyces pombe <br /> 39.   Koltin, Y., L. Faucette, D. J. Bergsma, M. A. Levy, R. Cafferkey, P. L. Koser,          alter shift cell proliferation sporulation. Mol. Gen. <br />       R. K. Johnson, G. P. Livi. 1991. Rapamaycin sensitivity Saccharomy-              Genet. 198:497&#226;&#8364;&#8220;502. <br />       ces cerevisiae mediated peptidyl-prolyl cis-trans isomerase related      62. Nurse, P. 1990. Universal control mechanism regulating onset M-phase. <br />       human FK506-binding protein. Mol. Cell. Biol. 11:1718&#226;&#8364;&#8220;1723.                             Nature 344:503&#226;&#8364;&#8220;508. <br /> 40.   Koser, P. L., D. J. Bergsma, R. Cafferkey, W.-K. Eng, N. M. McLaughlin, A.          63. Prentice, H. 1992. High efficiency transformation Schizosaccharomyces <br />       Ferrara, C. Silverman, K. Kasyan, M. J. Bossard, R. K. Johnson, T. G.                   pombe electroporation. Nucleic Acids Res. 20:621. <br />       Porter, M. A. Levy, G. P. Livi. 1991. The CYP2 gene Saccharomyces            64. Price, D. J., J. R. Grove, V. Calvo, J. Avruch, B. E. Bierer. 1992. <br />       cerevisiae encodes cyclosporin A peptidyl prolyl cis trans isomerase             Rapamycin induced inhibition 70-kilodalton S6 protein kinase. Sci- <br />       N-terminal signal sequence. Gene 108:74&#226;&#8364;&#8220;80.                                             ence 257:973&#226;&#8364;&#8220;977. <br /> 41.   Kunz, J., M. N. Hall. 1993. Cyclosporin A, FK506 rapamycin:            65. Russell, P., P. Nurse. 1986. Schizosaccharomyces pombe Saccharo- <br />       just immunosuppression. Trends Biochem. 18:334&#226;&#8364;&#8220;338.                                myces  <br /> </body></html>