<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>     Conserved Fungal Genes as Potential Targets for Broad-Spectrum <br />                       Antifungal Drug Discovery&#226;&#8364;&#160; <br />  <br />              The discovery of novel classes of antifungal drugs depends <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> a certain extent on the identification of new, <br />           unexplored targets that are essential for growth of fungal pathogens. Likewise, the broad-spectrum capacity of <br />           future antifungals requires the target gene(s) to be conserved among key fungal pathogens. Using a genome <br />           comparison (or concordance) tool, we identified 240 conserved genes as candidates for potential antifungal <br />           targets in 10 fungal genomes. To facilitate the identification of essential genes in Candida albicans, we developed <br />           a repressible C. albicans MET3 (CaMET3) promoter system capable of evaluating gene essentiality on a <br />           genome-wide scale. The CaMET3 promoter was found to be highly amenable to controlled gene expression, a <br />           prerequisite for use in target-based whole-cell screening. When the expression of the known antifungal target <br />           C. albicans ERG1 was reduced via down-regulation of the CaMET3 promoter, the CaERG1 conditional mutant <br />           strain became hypersensitive, specifically to its inhibitor, terbinafine. Furthermore, parallel screening against <br />           a small compound library using the CaERG1 conditional mutant under normal and repressed conditions <br />           uncovered several hypersensitive compound hits. This work therefore demonstrates a streamlined process for <br />           proceeding <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> selection and validation of candidate antifungal targets to screening for specific inhibitors. <br />  <br />  <br />    Candidiasis, aspergillosis, and cryptococcosis are the leading             and target only two cellular structures (the cell membrane and <br /> invasive fungal infections that cause substantial morbidity and               the cell wall) (49). Therefore, there is a critical need for new <br /> mortality in nosocomial settings and among immunocompro-                      classes of broad-spectrum antifungals that bind to novel tar- <br /> mised patients (51). Candida species are considered the fourth                gets. Microbial genomics offer an unprecedented opportunity <br /> most common bloodstream isolates in the United States (15).                   to enlarge the repertoire of antimicrobial compounds with the <br /> For many years, amphotericin B and fluconazole have been the                  identification of genes that encode potential new targets for <br /> standard of therapy for treatment of severe fungal infections                 chemotherapy. To date, more than 10 fungal genomes have <br /> (13). However, serious side effects such as nephrotoxicity are                been sequenced (http://ncbi.nih.gov/RefSeq/ [accessed August <br /> associated with amphotericin B, while fluconazole suffers from                2005]), including those of Saccharomyces cerevisiae, Candida <br /> such drawbacks as frequent interactions with coadministered                   albicans, C. glabrata, Aspergillus fumigatus, Cryptococcus neo- <br /> drugs, limited spectrum (for example, it is not effective against             formans, Debaryomyces hansenii, Kluyveromyces lactis, Schizo- <br /> aspergillosis), and the increasing emergence of fluconazole-                  saccharomyces pombe, Yarrowia lipolitica, and Eremothecium <br /> resistant Candida species (for example, Candida krusei and                    gossypii. It is expected that tremendous interest and effort will <br /> some strains of Candida glabrata) (9, 13, 20). Encouragingly,                 be devoted to the identification of essential genes as potential <br /> several new drugs have recently been approved by the U.S.                     drug targets from key fungal pathogens. While hundreds of <br /> Food and Drug Administration, including voriconazole, caspo-                  essential genes have been identified in S. cerevisiae (55) and C. <br /> fungin, and micafungin. Like the other two triazoles (posacon-                albicans (11, 44, 55), target prioritization in terms of, for ex- <br /> azole and ravuconazole) that are in development, voriconazole                 ample, degree of essentiality, broad-spectrum potential, drug <br /> displays a broader antifungal spectrum, including Aspergillus                 target potential, fungal specificity, availability of functional <br /> strains and resistant Candida species (25, 30). Caspofungin and               assays, and amenability to high-throughput screening (HTS) <br /> micafungin represent the first new class (the echinocandin                    would likely present a much greater challenge to the research <br /> lipopeptides) of antifungal agents approved after several de-                 community. <br /> cades of intensive research (2, 28, 56).                                         Recognizing the ever-increasing demands for broad-spec- <br />    In spite of these recent advances, the currently marketed                  trum antifungals, we sought to identify potential antifungal <br /> antifungals and those in development represent only three <br />                                                                               targets that are conserved among the 10 fungal species men- <br /> major chemical classes (polyenes, azoles, and echinocandins) <br />                                                                               tioned above and to develop a system to identify target-specific <br />                                                                               inhibitors. To this end, we identified 240 conserved genes <br />    * Corresponding author. Mailing address: Bristol-Myers Squibb Com-         among the 10 fungi via the genome comparison tool, deter- <br /> pany Pharmaceutical Research Institute, 5 Research Parkway, Walling-          mined essentiality of selected genes in C. albicans using a <br /> ford, CT 06492. Phone: (203) 677-7015. Fax: (203) 677-6771. E-mail: <br /> ying-kai.wang@bms.com. <br />                                                                               repressible C. albicans MET3 (CaMET3) promoter, and adapted <br />    &#226;&#8364;&#160; Supplemental material for this article may be found at http://ec         the CaMET3 promoter system to a target-based whole-cell HTS <br /> .asm.org/.                                                                    platform. <br />  <br />                                                                         638 <br />  VOL. 5, 2006                                                                         IDENTIFICATION OF CONSERVED FUNGAL GENES                                     639 <br />  <br />                         MATERIALS AND METHODS                                        after digestion with SacI and KpnI (blunt ended) and ligated into the SacII <br />                                                                                      (blunt ended) and SacI sites of pGEM-URA3 to replace CaURA3. The <br />    Bioinformatics. The completed genome sequences were acquired from the <br />                                                                                      resulting plasmid was linearized with SphI and NcoI, gel purified, and ligated <br /> NCBI Reference Sequence database (http://ncbi.nih.gov/RefSeq/) for the follow- <br />                                                                                      with the CaMET3 promoter PCR product treated with SphI and NcoI, yield- <br /> ing 10 fungal and human strains: S. cerevisiae S288c (5,866 open reading frames <br />                                                                                      ing plasmid pAMP. <br /> [ORFs]), C. albicans SC5314 (13,685 ORFs), C. glabrata CBS138 (5,192 ORFs), <br />                                                                                         Construction of conditional mutants. Two steps were involved in constructing <br /> A. fumigatus Af293 (9,907 ORFs), Cryptococcus neoformans JEC21 (6,593 <br />                                                                                      conditional mutants of C. albicans. The first step was the disruption of the first <br /> ORFs), D. hansenii CBS767 (6,317 ORFs), K. lactis NRRL Y-1140 (5,336 <br />                                                                                      allele of the gene of interest by the PCR-based gene disruption method (54) in <br /> ORFs), Schizosaccharomyces pombe 972 h&#226;&#171;&#186; (5,035 ORFs), Y. lipolitica CLIB99 <br />                                                                                      strain BWP17, with subsequent verification by PCR analysis. Next, the promoter <br /> (6,544 ORFs), Eremothecium gossypii ATCC 10895 (4,718 ORFs), and Homo <br />                                                                                      of the second copy of the gene was replaced by the CaMET3 promoter in the <br /> sapiens (29,236 ORFs). Other sources of genome sequence and annotation data <br />                                                                                      heterozygous strain. To do this, the CaMET3 promoter-swapping cassette was <br /> include Genome Therapeutics Corporation, Waltham, MA, The Institute for <br />                                                                                      amplified by PCR from plasmid pUMP or pAMP. The PCR primers used <br /> Genomic Research (http://www.tigr.org), the Yeast Proteome Database (YPD) <br />                                                                                      contained both template-specific common primer sequences and gene-specific <br /> (http://www.proteome.com/), and Incyte Pharmaceuticals, Palo Alto, CA. Where <br />                                                                                      flanking sequences of 50 to 60 bp in <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>. The common primer sequences <br /> incomplete genome sequences were encountered, coverage was increased by <br />                                                                                      annealed to the plasmid template pUMP or pAMP for amplification of the <br /> adding contiguous sequences for the same organism from multiple public and/or <br />                                                                                      promoter-swapping cassette, whereas the flanking sequences directed the ampli- <br /> private sources and assembling the data using the PHRAP algorithm (http://www        fied promoter cassette fragment to its specific location on the chromosome <br /> .phrap.org/). Likely genes that were not yet annotated in available yet incomplete   during recombination. The resulting PCR product of the CaMET3 promoter- <br /> genomes were identified using the program CRITICA (4), and MAGPIE (17)               swapping cassette was then introduced into the cells that were heterozygous for <br /> was used to visualize genes in context. Similar genes were identified using the      the gene of the interest to replace the endogenous promoter of the remaining <br /> &#226;&#8364;&#339;neighbors&#226;&#8364;&#65533; function of the concordance system (7). To determine if the pre-         allele via homologous recombination. Typically, between 500- to 1,000-bp nucle- <br /> dicted gene sequences were full length and of the proper reading frame for           otides upstream of the ATG codon of the gene of interest were removed, and <br /> incomplete genomes, both BLAST searches and CLUSTAL multiple sequence                promoter constructs were verified by PCR. <br /> alignments (1, 26) were conducted.                                                      Cell-based high-throughput screening. C. albicans conditional mutants har- <br />    To identify conserved genes, whole-genome comparisons based on all-versus-        boring the gene of interest under the control of the CaMET3 promoter were used <br /> all pairwise alignments with the FASTA algorithm (41) were conducted using the       for screening of target-specific inhibitors in the absence and presence of methi- <br /> concordance function essentially as described previously (7). Briefly, all ORFs      onine or cysteine. To carry out HTS, a culture grown overnight was diluted in SD <br /> (&#226;&#172;&#381;100 amino acids) of the query genome were compared to ORFs from various            plus His broth medium to 7.5 &#226;&#171;&#187; 105 CFU/ml, 35 &#226;&#65533;&#174;l of which was dispensed into <br /> other genomes and stored. A list of ORFs and their associated amino acid             wells of 384-well plates containing 8 &#226;&#65533;&#174;l of compounds in each well. Ten micro- <br /> sequences was then generated at a specified percent protein identity cutoff.         liters of SD plus His broth containing or lacking methionine or cysteine was <br /> When a gene of interest was not found in a genome, FRAMESEARCH (24) was              added to these wells. The final compound concentration at screening was 14.2 <br /> used to determine that it was not due to a failure of the gene-finding algorithm.    &#226;&#65533;&#174;M. The concentration of methionine or cysteine used was the 50% inhibitory <br />    Strains and growth media. The C. albicans strains used were SC5314 (wild          concentration (IC50) value previously determined, such that cells would prolif- <br /> type; Bristol-Myers Squibb Culture Collection) and its derivative, BWP17             erate at half the growth rate compared to no-methionine or cysteine control <br /> (ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434/ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG),         cultures. The plates were then incubated at 30&#194;&#176;C for 18 h, and cell growth was <br /> kindly provided by A. P. Mitchell of Columbia University. The S. cerevisiae strain   determined by reading the absorbance at 590 nm on a Perkin-Elmer 7000 plate <br /> used was ATCC 201390 (MATa/MAT&#226;&#65533;&#163; his3&#226;&#338;&#172;1/his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0/leu2&#226;&#338;&#172;1 lys2&#226;&#338;&#172;0/                  reader. Cell growth inhibition was calculated by using plain medium as a &#226;&#8364;&#339;blank&#226;&#8364;&#65533; <br /> LYS2 met15&#226;&#338;&#172;0/MET15 ura3&#226;&#338;&#172;0/ura30). Escherichia coli strain DH5&#226;&#65533;&#163; was used for          and cells grown in medium with dimethyl sulfoxide only as the &#226;&#8364;&#339;total.&#226;&#8364;&#65533; To <br /> plasmid propagation. Yeast extract-Bacto peptone-dextrose medium (YEPD),             determine dose responses, the compound concentration that causes 50% growth <br /> synthetic complete (SC) medium, and synthetic dextrose (SD) were prepared            inhibition (IC50) was calculated using the following two-parameter logistic equa- <br /> according to standard procedures as previously described (46). Uridine (25 &#226;&#65533;&#174;g        tion: Y &#226;&#171;&#189; 100/[1 &#226;&#171;&#185; (IC50/I)H], where Y is the percent growth inhibition, I is the <br /> ml&#226;&#171;&#186;1) was added according to a method described previously by Fonzi and Irwin        inhibitor concentration, and H is the hill slope, as described by GraphPad <br /> (16), and other supplements such as histidine and arginine (20 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1) were         Software Inc. (San Diego, CA). <br /> added as described previously (46). In general, 5 mM of methionine and 2.5 mM <br /> of cysteine were used in media unless otherwise indicated. Both S. cerevisiae and <br /> C. albicans strains were grown at 30&#194;&#176;C, and the E. coli strain was grown at 37&#194;&#176;C.                                      RESULTS <br />    DNA manipulations. DNA manipulations were carried out according to stan- <br /> dard molecular methods (45) unless otherwise noted. Typical PCRs were carried <br />                                                                                         Identification of conserved fungal genes via concordance <br /> out in a volume of 100 &#226;&#65533;&#174;l with an initial 2-min jump start at 95&#194;&#176;C, 30 cycles of     analysis. It is generally believed in the antimicrobial drug dis- <br /> amplification (2 min at 95&#194;&#176;C, 1 min at 52&#194;&#176;C, and 5 min at 72&#194;&#176;C), and a final         covery community that conserved, essential genes have great <br /> 10-min extension step at 72&#194;&#176;C. Typical reaction mixtures contained 10 mM             potential to yield broad-spectrum antimicrobial agents. Using <br /> Tris-HCl, pH 8.3, 1.5 mM MgCl2, 50 mM KCl, 0.2 mM deoxynucleoside triphos- <br />                                                                                      the 1,049 essential S. cerevisiae genes (http://www.yeastgenome <br /> phates, 10 pmol of primers, 100 ng of genomic template DNA, and 5 U of Taq <br /> DNA polymerase (Gibco-BRL) or Pfu Turbo DNA polymerase (Stratagene).                 .org) as a query, we conducted a pairwise genome comparison <br /> Total genomic DNA from C. albicans was isolated by the glass bead lysis method       or concordance analysis across the nine other complete fungal <br /> (27). All PCR primers used in this study are listed in Table S1 in the supple-       genomes in an effort to identify genes that are conserved across <br /> mental material.                                                                     those fungi (see Materials and Methods for details). To deter- <br />    Transformation of C. albicans was carried out as described previously (54). <br />                                                                                      mine a legitimate cutoff for gene conservation, we profiled a <br /> Typically, about 20 to 80 &#226;&#65533;&#174;l of the PCR mixture was used for transformation. For <br /> disrupting the first allele of a target gene, cells were plated onto selective       list of known fungal &#226;&#8364;&#339;drug-able&#226;&#8364;&#65533; targets against the 10 fungal <br /> medium (SC minus uridine or SC minus arginine), while the second round of            genomes. This allowed us to quantitatively correlate the broad- <br /> transformation to disrupt the second allele used SC minus uridine and arginine       spectrum capacity of the currently marketed drugs to the per- <br /> medium.                                                                              cent protein identity of their corresponding drug targets. Vori- <br />    Construction of C. albicans MET3 promoter-swapping cassettes. Two plas- <br /> mids, pUMP and pAMP, which contain a CaMET3 promoter cassette, were <br />                                                                                      conazole (VFend) and caspofungin (Cancidas) are considered <br /> constructed. Plasmid pUMP contains the CaURA3 gene, and pAMP harbors                 to have broad-spectrum capacity against a number of medically <br /> CaARG4 as the selective marker. The 1.4-kb C. albicans MET3 promoter region          important fungi, including Candida species and Aspergillus spe- <br /> was amplified by PCR from genomic DNA of strain SC5314. To construct                 cies (30, 34). Voriconazole targets lanosterol 14-&#226;&#65533;&#163;-demethyl- <br /> pUMP, the CaMET3 promoter PCR product was cut with restriction enzymes               ase, encoded by the essential gene ERG11 in the yeast S. <br /> SphI and NcoI, gel purified, and ligated into pGEM-URA3 that was linearized <br /> by SphI and NcoI. This placed the CaMET3 promoter sequence adjacent to <br />                                                                                      cerevisiae (50). The target of caspofungin is an essential cell <br /> CaURA3 but in the opposite orientation to avoid transcription read-through. To       wall enzyme, &#226;&#65533;&#164;-1,3-glucan synthase, encoded by FKS1 and <br /> construct pAMP, the C. albicans ARG4 gene was excised from pRS-ARG4&#226;&#338;&#172;SpeI             FKS2 and regulated by RHO1 in S. cerevisiae (35, 36). The <br />  640                   LIU ET AL.                                                                                                                                                                                                                   EUKARYOT. CELL <br />  <br />  <br />                                                                                                                                                                                                  putative homologs of all four genes were identified in the 10 <br />  <br />  <br />                                                                                                                                   H. sapiens <br />  <br />  <br />                                                                                                                                                   None <br />  <br />                                                                                                                                                           None <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         None <br />                                                                                                                                                                                                  fungal genomes with protein sequence identity of more than <br />  <br />  <br />                                                                                                                                                    36 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  64 <br />  <br />  <br />  <br />                                                                                                                                                                          30 <br />                                                                                                                                                                          20 <br />  <br />                                                                                                                                                                                       36 <br />                                                                                                                                                                                       66 <br />                                                                                                                                                                                                  40% for Erg11p and more than 50% for each of the three gene <br />                                                                                                                                   E. gossypii                                                    products for glucan synthase (Table 1). In vitro, terbinafine <br />                                                                                                                                                   71                                             (Lamisil) has an inhibitory activity against many strains of <br />                                                                                                                                                   72 <br />  <br />                                                                                                                                                           72 <br />  <br />                                                                                                                                                                  84 <br />  <br />  <br />                                                                                                                                                                         60 <br />                                                                                                                                                                         64 <br />                                                                                                                                                                         48 <br />  <br />                                                                                                                                                                                       65 <br />                                                                                                                                                                                       92 <br />                                                                                                                                                                                                  Candida species, Aspergillus species, and Cryptococcus neofor- <br />                                                                                                                                                                                                  mans (although it is only marketed for the treatment of foot <br />                                                                                                                                                                                                  fungus) (31), and its target, squalene epoxidase (Erg1p), was <br />                                                                                                                                   Y. lipolitica <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  found to be conserved at &#226;&#177;&#8211;39% across the 10 fungi (Table 1). <br />                                                                                                                                                   60 <br />                                                                                                                                                   65 <br />  <br />                                                                                                                                                           65 <br />  <br />                                                                                                                                                                  73 <br />  <br />  <br />                                                                                                                                                                         42 <br />                                                                                                                                                                         44 <br />                                                                                                                                                                         35 <br />  <br />                                                                                                                                                                                       47 <br />                                                                                                                                                                                       85 <br />                                                                                                                                                                                                  In addition, four other known fungal targets, CHS2, RAM2, <br />                                                                                                                                                                                                  NMT1, and EFT2, which are the subject of preclinical research <br />                                                                                                                                                                                                  but lack a marketed drug, were also compared side by side <br />                                                                                                                                   S. pombe <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  across the 10 fungal genomes and were found to be conserved <br />                                                                                                                                                   46 <br />                                                                                                                                                   56 <br />  <br />                                                                                                                                                           56 <br />  <br />                                                                                                                                                                  71 <br />  <br />  <br />                                                                                                                                                                         36 <br />                                                                                                                                                                         37 <br />                                                                                                                                                                         30 <br />  <br />                                                                                                                                                                                       47 <br />                                                                                                                                                                                       79 <br />                                                                                                                                                                                                  at 30% or higher at the protein level (Table 1). Based on the <br />                                                                                                                                                                                                  above-described data, we set the benchmark in our search for <br />                                                                  % Identity to other fungi at protein level <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  conserved genes across the fungal genomes to &#226;&#177;&#8211;40% identity <br />                                                                                                                                   K. lactis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  at protein level. <br />                                                                                                                                                   75 <br />                                                                                                                                                   82 <br />  <br />                                                                                                                                                           82 <br />  <br />                                                                                                                                                                  83 <br />  <br />  <br />                                                                                                                                                                         65 <br />                                                                                                                                                                         74 <br />                                                                                                                                                                         51 <br />  <br />                                                                                                                                                                                       64 <br />                                                                                                                                                                                       93 <br />  <br />  <br />                                                                                                                                                                                                     A total of 240 genes, denoted conserved fungal genes, were <br />                                                                                                                                                                                                  identified with percent identity at &#226;&#177;&#8211;40% at the protein level <br />                                                                                                                                   D. hansenii <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  using the 1,049 essential S. cerevisiae genes as a query (see <br />       TABLE 1. Homology comparison of known antifungal targets <br />  <br />  <br />  <br />  <br />                                                                                                                                                   64 <br />                                                                                                                                                   74 <br />  <br />                                                                                                                                                           74 <br />  <br />                                                                                                                                                                  76 <br />  <br />  <br />                                                                                                                                                                         41 <br />                                                                                                                                                                         53 <br />                                                                                                                                                                         38 <br />  <br />                                                                                                                                                                                       58 <br />                                                                                                                                                                                       87 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  Table S2 in the supplemental material). To view the distribu- <br />                                                                                                                                                                                                  tion of those genes by function, we assigned them to the func- <br />                                                                                                                                                                                                  tional and cellular categories described by the YPD (http: <br />                                                                                                                                   C. neoformans <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  //www.proteome.com) for S. cerevisiae (Table 2). It was found <br />                                                                                                                                                                                                  that the 240 conserved fungal genes encode proteins involved <br />                                                                                                                                                   43 <br />                                                                                                                                                   52 <br />  <br />                                                                                                                                                           53 <br />  <br />                                                                                                                                                                  68 <br />  <br />  <br />                                                                                                                                                                         33 <br />                                                                                                                                                                         39 <br />                                                                                                                                                                         32 <br />  <br />                                                                                                                                                                                       37 <br />                                                                                                                                                                                       74 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  in all 39 cellular processes categorized by the YPD. As shown <br />                                                                                                                                                                                                  in Table 2, the top four cellular processes to which the con- <br />                                                                                                                                                                                                  served fungal genes were assigned are protein synthesis (13%), <br />                                                                                                                                   A. fumigatus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  RNA processing modification (10%), protein degradation <br />                                                                                                                                                                                                  (7%), and polymerase II (Pol II) transcription (5%), the basic, <br />                                                                                                                                                   47 <br />                                                                                                                                                   63 <br />  <br />                                                                                                                                                           63 <br />  <br />                                                                                                                                                                  72 <br />  <br />  <br />                                                                                                                                                                         36 <br />                                                                                                                                                                         39 <br />                                                                                                                                                                         31 <br />  <br />                                                                                                                                                                                       46 <br />                                                                                                                                                                                       79 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  evolutionarily conserved cellular processes. It is interesting that <br />                                                                                                                                                                                                  there are 3% of the conserved fungal genes whose biological <br />                                                                                                                                                                                                  functions were unknown in S. cerevisiae according to the Saccha- <br />                                                                                                                                   C. glabrata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  romyces Genome Database (SGD) (http://www.yeastgenome.org <br />                                                                                                                                                   83 <br />                                                                                                                                                   86 <br />  <br />                                                                                                                                                           86 <br />  <br />                                                                                                                                                                  92 <br />  <br />  <br />                                                                                                                                                                         76 <br />                                                                                                                                                                         83 <br />                                                                                                                                                                         55 <br />  <br />                                                                                                                                                                                       72 <br />                                                                                                                                                                                       93 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  [accessed October 2005]). <br />                                                                                                                                                                                                     The preferred profile for fungal targets are essential proteins <br />                                                                                                                                                                                                  conserved among fungi yet absent from the human genome so <br />                                                                                                                                   C. albicans <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  as to minimize potential toxic side effects exerted by pharma- <br />                                                                                                                                                   64 <br />                                                                                                                                                   73 <br />  <br />                                                                                                                                                           73 <br />  <br />                                                                                                                                                                  77 <br />  <br />  <br />                                                                                                                                                                         40 <br />                                                                                                                                                                         52 <br />                                                                                                                                                                         39 <br />  <br />                                                                                                                                                                                       54 <br />                                                                                                                                                                                       87 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  cological inhibition of the cellular targets. With this ideal in <br />                                                                                                                                                                                                  mind, the 240 conserved fungal genes were prioritized based <br />                                                                                                                                                                                                  on their relatedness to human genes (Fig. 1A). Interestingly, <br />                                                                                                                                                     micafungin <br />  <br />                                                                                                                                                     micafungin <br />  <br />                                                                                                                                                     micafungin <br />                                                                                     Known drug(s) <br />  <br />  <br />  <br />  <br />                                                                                                                                                   Caspofungin, <br />  <br />                                                                                                                                                   Caspofungin, <br />  <br />                                                                                                                                                   Caspofungin, <br />                                                                                     or inhibitor(s) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         Nikkomycins <br />                                                                                                                                                                         Terbinafine <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  more than half (59%) of the conserved fungal genes have <br />                                                                                                                                                                                       Sordarin <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  putative human homologs with percent identities ranging from <br />                                                                                                                                                   Azoles <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         None <br />  <br />                                                                                                                                                                                       None <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  41 to 60%, whereas each of those having the least (0 to 40%) <br />                                                                                                                                                                                                  and most (61 to 100%) identity with human genes constitutes <br />                                                                                                                                                                                                  around 20% of the conserved fungal genes identified. When <br />                                                                                                                                                   Lanosterol-14&#226;&#65533;&#163;-demethylase <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  the essential yeast gene set (1,049 genes) as well as the entire <br />                                                                                                                                                   Subunit of geranylgeranyl <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  yeast gene set (5,866 genes) were compared with the human <br />                                                                                                                                                   Subunit of 1,3-&#226;&#65533;&#164;-glucan <br />  <br />                                                                                                                                                   Subunit of 1,3-&#226;&#65533;&#164;-glucan <br />  <br />  <br />  <br />  <br />                                                                                                                                                   N-Myristoyl transferase <br />                                                                                                               Annotation in SGD <br />  <br />  <br />  <br />  <br />                                                                                                                                                   Translation elongation <br />                                                                                                                                                   GTP binding protein <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  genome, it was found that the majority of the yeast genes <br />                                                                                                                                                   Squalene epoxidase <br />                                                                                                                                                     regulating glucan <br />  <br />                                                                                                                                                   Chitin synthase II <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  (&#226;&#172;&#402;76% for the essential gene set and &#226;&#172;&#402;88% for whole gene <br />                                                                                                                                                                                                  set) do not have significant human homologs (&#226;&#172;&#381;40% protein <br />                                                                                                                                                     transferase <br />                                                                                                                                                     synthase <br />  <br />                                                                                                                                                     synthase <br />  <br />  <br />                                                                                                                                                     synthase <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  identity) (Fig. 1B; data shown for human homologs of the <br />                                                                                                                                                     factor 2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  entire yeast gene set only). Similar findings were also made <br />                                                                                                                                                                                                  when each of the remaining nine fungal genomes was com- <br />                                                                                                                                                                                                  pared with the human genome, in which between 85 to 93% of <br />                                                                                                                                                                                                  the fungal genes for any one organism were found to be unique <br />                                                                                     S. cerevisiae <br />                                                                                     Gene ID in <br />  <br />  <br />                                                                                                                                                   ERG11 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  to fungi (data not shown). This indicates that most (&#226;&#172;&#402;80%) of <br />                                                                                                                                                                                       NMT1 <br />                                                                                                                                                                         RAM2 <br />                                                                                                                                                                  RHO1 <br />  <br />  <br />  <br />                                                                                                                                                                         ERG1 <br />                                                                                                                                                                         CHS2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       EFT2 <br />                                                                                                                                                   FKS1 <br />  <br />                                                                                                                                                           FKS2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  the 240 conserved fungal genes identified here come from only <br />                                                                                                                                                                                                  7 to 15% of each fungal genome that is also conserved in the <br />  VOL. 5, 2006                                                                         IDENTIFICATION OF CONSERVED FUNGAL GENES                   641 <br />  <br />  <br />                                      TABLE 2. Functional distribution of the 240 conserved fungal genes identified <br />                                                                                                                                     No. of     % of <br />          Function category a                                                  Gene(s) of S. cerevisiae <br />                                                                                                                                     genesb    genesc <br />  <br /> Protein synthesis                        CDC95, GRS1, RIA1, RPS31, SUP35, YLL034C, RPL18B, SIS1, SUP45, DYS1, ILS1,                    50      12.8 <br />                                            MES1, NMD3, ALA1, CDC60, DED1, DED81, DPS1, EFB1, FRS1, FRS2, GLN4, <br />                                            HTS1, HYP2, KRS1, RLP24, RPL10, RPL15A, RPL17A, RPL3, RPL5, RPP0, RPS15, <br />                                            RPS2, RPS20, RPS3, RPS5, SES1, SUI2, SUI3, THS1, TIF11, TIF34, TIF5, VAS1, <br />                                            R341C, YEF3, YHR020W, YIF2, YNL247W <br /> RNA processing modification              CDC95, GRS1, HAS1, RIA1, RRP3, RVB1, RVB2, GLC7, SPB1, GSP1, ACC1, DBP5,                      37       9.5 <br />                                            MTR4, DIB1, DIS3, LSM2, PRP22, PRP43, SNU13, EMG1, BMS1, CBF5, DBP2, <br />                                            DBP9, DIM1, ERB1, FAL1, GAR1, HCA4, IMP3, IMP4, MAK16, NIP7, NOP1, <br />                                            NOP2, NOP5, SIK1 <br /> Protein degradation                      KAE1, RPS31, RPT4, RPT6, RPN11, RPN6, RPN8, RPT1, RPT2, RPT3, RPT5, SKP1,                     28       7.2 <br />                                            RSP5, UBA1, SRP1, CDC48, UBC9, PRE1, PRE10, PRE2, PRE3, PRE4, PRE5, <br />                                            PRE8, PUP1, PUP2, PUP3, SCL1 <br /> Pol II transcription                     RVB1, RVB2, RPT4, RPT6, RPN11, RPN6, RPN8, RPT1, RPT2, RPT3, RPT5, ACT1,                      19       4.9 <br />                                            SPT16, TOP2, SSL2, RPO26, RPB5, SPT15, RPO21 <br /> Cell structure                           ACT1, SSC1, CCT2, CCT3, CCT4, CCT5, CCT6, CCT7, CCT8, TCP1, TUB1, CDC42,                      15       3.8 <br />                                            ARC19, ARC35, TUB2 <br /> Protein folding                          SUP35, YLL034C, SIS1, SUP45, SSC1, CCT2, CCT3, CCT4, CCT5, CCT6, CCT7,                        15       3.8 <br />                                            CCT8, TCP1, HSP60, KAR2 <br /> Vesicular transport                      RPS31, ACT1, RHO3, ARP3, LST8, SEC13, YKT6, SEC23, COP1, GDI1, SAR1,                          15       3.8 <br />                                            SEC26, SEC27, SEC4, YPT1 <br /> Unknown                                  BRX1, FRQ1, KRE30, KRE33, KRR1, NBP35, NSA2, RLI1, RPF2, R339C, YJR072C,                      13       3.3 <br />                                            YLR243W, YOR262W <br /> Carbohrate metabolism                    GLC7, PCM1, SEC53, CDC19, TPI1, GFA1, ACS2, ENO2, FBA1, PGI1, PGK1, UGP1                      12       3.1 <br /> Chromatin chromosome                     RVB1, RVB2, RPT4, RPT6, RSP5, SSL2, POL2, RFC4, RFC5, CDC2, CDC54, RAD3                       12       3.1 <br />   structure <br /> DNA repair                               RVB1, RVB2, RPT4, RPT6, RSP5, SSL2, POL2, RFC4, RFC5, CDC2, CDC54, RAD3                       12       3.1 <br /> Protein modification                     DYS1, RSP5, UBA1, PCM1, SEC53, GPI8, PSA1, BET2, MAS1, MAS2, PMI40, STT3                      12       3.1 <br /> Other metabolism                         HEM1, NFS1, PMA1, GLN1, HEM15, HEM2, HEM12, HEM13, QNS1, RIB3, RIB5                           11       2.8 <br /> Cell cycle control                       GSP1, SKP1, SRP1, CDC48, UBC9, SPT16, TUB1, RHO1, CDC28, SDS22                                10       2.6 <br /> Cell polarity                            GLC7, ACT1, CDC42, ARC19, ARC35, RHO3, ARP3, RHO1, CDC12, PWP2                                10       2.6 <br /> DNA synthesis                            POL2, RFC4, RFC5, RFC2, RFC3, CDC2, CDC54, CDC47, MCM2, MCM6                                  10       2.6 <br /> Lipid fatty acid and sterol              ACC1, HEM1, ERG10, ERG11, ERG13, ERG20, ERG25, ERG7, IDI1, MVD1                               10       2.6 <br />   metabolism <br /> Small-molecule transport                 RSP5, UBA1, LST8, SEC13, NFS1, PMA1, ATM1, PET9, NEO1                                          9       2.3 <br /> Nuclear-cytoplasmic transport            ILS1, MES1, GSP1, ACC1, DBP5, MTR4, SRP1, CRM1                                                 8       2.0 <br /> Amino acid metabolism                    SKP1, RSP5, UBA1, GLN1, ILV3, ILV5, SAH1                                                       7       1.8 <br /> Energy generation                        CDC19, TPI1, NFS1, HEM15, HEM2, ATM1, PET9                                                     7       1.8 <br /> Pol III transcription                    RPO26, RPB5, SPT15, RPC40, RET1, RPC11, RPO31                                                  7       1.8 <br /> Mitosis                                  DIB1, DIS3, TOP2, TUB1, TUB2                                                                   6       1.5 <br /> Pol I transcription                      RPO26, RPB5, SPT15, RPC40, RPA135, RPA190                                                      6       1.5 <br /> Protein translocation                    SUP35, SSC1, HSP60, KAR2, SEC61, SRP54                                                         6       1.5 <br /> RNA splicing                             DIB1, LSM2, PRP22, PRP43, SNU13, SUB2                                                          6       1.5 <br /> Nucleotide metabolism                    ADE13, GUK1, RNR1, RNR2, URA6                                                                  5       1.3 <br /> Cell wall maintenance                    GFA1, GPI8, PSA1, RHO1                                                                         4       1.0 <br /> Cytokinesis                              YLL034C, CDC48, CDC12, PWP2                                                                    4       1.0 <br /> Signal transduction                      RSP5, RHO3, RHO1, NOG1                                                                         4       1.0 <br /> Cell stress                              GLC7, HTB1, TRR1                                                                               3       0.8 <br /> Mating response                          ACT1, CDC42, CDC12                                                                             3       0.8 <br /> Meiosis                                  GLC7, CDC28, NOP7                                                                              3       0.8 <br /> Membrane fusion                          YLL034C, CDC48, YKT6                                                                           3       0.8 <br /> Other                                    ACT1, SEC23                                                                                    2       0.5 <br /> Protein complex assembly                 EMG1, RPO26                                                                                    2       0.5 <br /> RNA turnover                             NMD3, LSM2                                                                                     2       0.5 <br /> Recombination                            SUB2, RAD3                                                                                     2       0.5 <br /> Phosphate metabolism                     IPP1                                                                                           1       0.3 <br />  <br /> Total                                                                                                                                391 <br />   a <br />       The 39 function categories as described by the YPD (http://www.proteome.com). <br />   b <br />       Total counts were over 240 since certain genes were assigned to multiple functions by the YPD. <br />   c <br />       Percentage was calculated based on the number of genes in each category divided by the total number (391). <br />  <br />  <br />  <br /> human genome, which at least partly explains the difficulty in                       necessary to verify their essentiality in the remaining fungi, <br /> developing safe and selective antifungal drugs.                                      especially in the pathogenic fungi, in order for them to be <br />    Development of a CaMET3 promoter system for essential                             considered antifungal targets. Towards this goal, we developed <br /> gene identification in C. albicans. Although the aforemen-                           a CaMET3 promoter system capable of large-scale identifica- <br /> tioned conserved fungal genes are essential in S. cerevisiae, it is                  tion of essential genes in the primary fungal pathogen C. albi- <br />  642     LIU ET AL.                                                                                                       EUKARYOT. CELL <br />  <br />  <br />                                                                        gous parental strains tested (Fig. 2C; data shown for a repre- <br />                                                                        sentative heterozygous strain). <br />                                                                           We previously identified a list of conserved genes between S. <br />                                                                        cerevisiae and C. albicans using the S. cerevisiae essential gene <br />                                                                        set (52). A dozen genes with no known biological function <br />                                                                        (denoted conserved unknown reading frames [CURFs]) were <br />                                                                        selected in this work to evaluate their essentiality in C. albicans <br />                                                                        with the hope of identifying novel, broad-spectrum antifun- <br />                                                                        gals (Table 3). Each of the 12 CURFs was evaluated by the <br />                                                                        CaMET3 promoter system, and 10 were determined to be <br />                                                                        essential in C. albicans (Table 3 and Fig. 2C). To corroborate <br />                                                                        the essentiality results obtained by the CaMET3 promoter <br />                                                                        method, all genes tested (known or unknown) were indepen- <br />                                                                        dently evaluated for essentiality by the PCR-based gene dis- <br />                                                                        ruption method (54). Comparable results were obtained be- <br />                                                                        tween the two methodologies for all genes except CaYJR072c, <br />                                                                        which was deemed to be not essential by the CaMET3 pro- <br />                                                                        moter method, yet the disruption method failed to obtain the <br />                                                                        disruption of both alleles (data not shown). <br />                                                                           Characterization of the CaMET3 promoter for HTS amena- <br />                                                                        bility. It has previously been reported that both bacterial and <br />                                                                        fungal cells can become hypersensitive to specific inhibitors <br />                                                                        when the target molecule is rate limiting for cell growth by <br />                                                                        either using heterozygous strains (11, 21) or using regulatable <br />                                                                        (either repressible/or inducible) promoters (12, 44). To assess <br />                                                                        the utility of CaMET3 promoter in this regard, all 13 essential <br />                                                                        genes (3 known genes and 10 CURFs) identified in this work <br />                                                                        were individually assessed for repression of growth via con- <br />                                                                        trolled expression of the CaMET3 promoter by the addition of <br />                                                                        methionine or cysteine. A variety of conditions such as inocu- <br />                                                                        lum size, methionine or cysteine concentration, and incubation <br />                                                                        time were explored. Figure 3 shows growth curves of two rep- <br />                                                                        resentative conditional mutants and of the parental BWP17 <br />                                                                        strain using various concentrations of either methionine or <br />   FIG. 1. (A) Illustration of identifiable human homologs of the 240   cysteine, demonstrating that the CaMET3 promoter could be <br /> conserved fungal genes (FCGs) as percent protein identity, with the    titrated by either methionine (for example, CaYDR341c) or <br /> lower left corner showing the least homology and upper right corner    cysteine (for example, CaYDR341c and CaNMT1), while nei- <br /> showing the most homology. (B) Human homologs of the genes             ther methionine nor cysteine caused any significant growth <br /> (5,866) of S. cerevisiae plotted as percent protein identity. <br />                                                                        defect on the parental BWP17 strain at concentrations up to 5 <br />                                                                        mM. To assist HTS, the concentration of either methionine or <br />                                                                        cysteine that inhibits 50% of the cell growth (IC50), was deter- <br /> cans. In this system, a conditional mutant where one allele of         mined for each of the 13 conditional mutants tested (Table 4). <br /> the gene of interest was disrupted by the PCR-based gene               As discussed below, the IC50 was the concentration arbitrarily <br /> disruption technique (54) and the second allele was placed             chosen and used to set cells that were rate limiting in growth in <br /> under the control of the CaMET3 promoter (see Fig. 2A for a            the cell-based HTS to identify hypersensitive inhibitors. We <br /> schematic illustration of the promoter-swapping approach) was          found that cysteine alone was capable of repressing all 13 <br /> constructed. Homologous recombinations were verified by                essential genes tested, while methionine was only able to re- <br /> PCR confirmation analysis using various primer sets (Fig. 2B).         press 3 (CaYDR341c, CaYOL077c, and CaERG1) at up to 10 <br /> Gene essentiality was determined by comparing phenotypes of            mM (Fig. 3 and Table 4). <br /> serially diluted conditional mutants grown on SD agar plates in           The C. albicans ERG1 gene, encoding a squalene epoxidase, <br /> the absence and presence of methionine and cysteine (Fig.              is specifically inhibited by terbinafine (13, 20). To assess drug <br /> 2C). Scores of 1 to 4 were used to assess the degree of essen-         hypersensitivity, growth inhibition by terbinafine was deter- <br /> tiality, with 1 being absolutely required and 4 being not re-          mined for the PMET3-CaERG1 conditional mutant in the pres- <br /> quired for cell growth. As expected, three known essential C.          ence and absence of methionine. As shown in Fig. 4A, a 15- <br /> albicans genes (CaERG1, CaRAM2, and CaNMT1) were de-                   fold increase in terbinafine potency (expressed as IC50) was <br /> termined to be essential in this work, with scores of 1, 3, and 1,     observed in the presence of methionine compared to the no- <br /> respectively (Fig. 2C). However, CaPFY1, whose essentiality is         methionine control. In addition, the growth inhibition in the <br /> strain dependent in S. cerevisiae, was found to be not essential       presence of methionine varied from about 25 to 90% over a log <br /> in C. albicans (Fig. 2C). It is important that no growth defect        range of inhibitor concentrations (0.06 to 1 &#226;&#65533;&#174;g/ml), whereas <br /> by methionine and/or cysteine was observed for any heterozy-           only 0.5 to 3.5% growth inhibition was observed in the absence <br />  VOL. 5, 2006                                                                IDENTIFICATION OF CONSERVED FUNGAL GENES                            643 <br />  <br />  <br />  <br />  <br />   FIG. 2. Construction of C. albicans conditional mutants by using the CaMET3 promoter-swapping cassette. (A) Scheme of promoter swapping <br /> in which the PCR-amplified CaMET3 promoter-swapping cassette is transformed to a heterozygous YFG1 strain, replacing the endogenous <br /> promoter of the second allele via homologous recombination. Primer sets used for confirmation PCR are shown as arrows and are as follows: 5&#226;&#172;&#732; <br /> promoter/3&#226;&#172;&#732; PTURA, 5&#226;&#172;&#732; PTURA/3&#226;&#172;&#732; promoter, and 5&#226;&#172;&#732; gene/3&#226;&#172;&#732; gene. (B) Verification of CaMET3 promoter conditional mutants of the CaERG1 <br /> gene, as an example, by PCR confirmation analysis. Lanes labeled &#226;&#8364;&#339;M&#226;&#8364;&#65533; contain 1-kb DNA. Lanes 1 and 2 depict examples of wrong constructs, <br /> whereas lanes 3 and 4 are examples of correct conditional mutant constructs. The expected sizes of PCR products using the primer sets 5&#226;&#172;&#732; <br /> PTURA/3&#226;&#172;&#732; promoter, 5&#226;&#172;&#732; promoter/3&#226;&#172;&#732; PTURA, and 5&#226;&#172;&#732; gene/3&#226;&#172;&#732; gene are 1.6 kb, 1.2 kb, and 2.4 kb, respectively. (C) Growth phenotypes of various <br /> CaMET3 promoter conditional mutants grown on SD plus histidine medium (SD plus histidine plus uridine for the CaERG1/Caerg1::ARG4 mutant <br /> used as a growth control) in the absence (&#226;&#171;&#186;) or presence (&#226;&#171;&#185;) of 5 mM of methionine and 2.5 mM of cysteine after incubation for 2 days at 30&#194;&#176;C. <br /> Cells were serially diluted (1 to 10) starting at 2 &#226;&#171;&#187; 106 CFU/ml for the first dilution, and 5 &#226;&#65533;&#174;l of the dilution was spotted from left to right. An <br /> arbitrary score of 1, 2, 3, or 4 was assigned to a specific conditional mutant corresponding to its degree of growth defect in SD plus methionine <br /> and cysteine, with 1 being most essential and 4 being nonessential. <br />  <br />  <br />  <br /> of methionine over the same terbinafine concentration range                    Parallel high-throughput screening. The utility of the CaMET3 <br /> (Fig. 4A). On the contrary, the experiments conducted with                  promoter for identifying hypersensitive inhibitors was further <br /> fluconazole (Fig. 4B) or sordarin (Fig. 4C), which are known                explored by conducting a cell-based parallel HTS campaign <br /> inhibitors of ERG11 and EFT2 (29), respectively, showed that                using the PMET3-CaERG1 conditional mutant. A library of <br /> sensitivities of the PMET3-CaERG1 conditional mutant to these               &#226;&#172;&#402;12,000 compounds in the Bristol-Myers Squibb Company <br /> drugs were not altered by the presence of methionine. Similar               compound collection was screened in the absence and pres- <br /> results were also obtained when a modified MIC test was                     ence of methionine (denoted Met minus and Met plus, respec- <br /> conducted for the three antifungals for comparison from the                 tively) (see Materials and Methods for details). Using &#226;&#177;&#8211;40% <br /> same cultures (data not shown) (the MIC was defined as the                  growth inhibition as a cutoff at the 14.2 &#226;&#65533;&#174;M compound con- <br /> drug concentration that gave 100% of cell growth inhibition for             centration, 37 hits were identified from the Met-plus screen, 18 <br /> terbinafine or sordarin and 80% of inhibition for fluconazole,              of which were also scored positive in the Met-minus screen <br /> respectively). Furthermore, the sensitivity of a PMET3-CaRAM2               (Fig. 5). By superimposing the 37 compounds from both the <br /> conditional mutant to these three drugs remained unchanged                  Met-plus and Met-minus screen with respect to percent inhi- <br /> in either the absence or presence of 5 mM methionine and 1                  bition of cell growth, we categorized them into three groups. <br /> mM cysteine (data not shown). Taken together, these results                 Group I has six hits with percent growth inhibition ranging <br /> established that reduced expression of CaERG1 resulted in                   from 41 to 70% in the Met-plus screen and &#226;&#171;&#186;11 to 10% in the <br /> cells that were hypersensitive only to the target-specific inhib-           Met-minus screen. Group II contains hits with &#226;&#172;&#381;40% inhibi- <br /> itor terbinafine.                                                           tion in the Met-plus screen and 11 to 40% inhibition in the <br />  644                                                                                                                                      LIU ET AL.                                                                                                                                                                                                                                                                                                                                              EUKARYOT. CELL <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 Met-minus screen. Group III consists of the 18 hits common to <br />                                                                                                                                                  D. hansenii K. lactis S. pombe Y. lipolitica E. gossypii H. sapiens                                                                                                                                                                                                                            both conditions (both &#226;&#172;&#381;40%). By referring to the dose-re- <br />                                                                                                                                                                                                                        21 <br />                                                                                                                                                                                                                        35 <br />                                                                                                                                                                                                                        45 <br />  <br />                                                                                                                                                                                                                                          36 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         56 <br />                                                                                                                                                                                                                                                         40 <br />                                                                                                                                                                                                                                                         34 <br />  <br />                                                                                                                                                                                                                                                                         15 <br />                                                                                                                                                                                                                                                                         41 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         28 <br />  <br />                                                                                                                                                                                                                                                                                                         24 <br />                                                                                                                                                                                                                                                                                                         42 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 sponse curves for terbinafine in the presence and absence of <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 methionine (Fig. 4A), those six hits in group I therefore rep- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 resent the so-called &#226;&#8364;&#339;hypersensitive&#226;&#8364;&#65533; hits that would otherwise <br />                                                                                                                                                                                                                        58 <br />                                                                                                                                                                                                                        62 <br />                                                                                                                                                                                                                        46 <br />  <br />                                                                                                                                                                                                                                          68 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         75 <br />                                                                                                                                                                                                                                                         62 <br />                                                                                                                                                                                                                                                         55 <br />  <br />                                                                                                                                                                                                                                                                         41 <br />                                                                                                                                                                                                                                                                         68 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         48 <br />  <br />                                                                                                                                                                                                                                                                                                         48 <br />                                                                                                                                                                                                                                                                                                         70 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 not be found if normal strains were used in HTS. <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                          DISCUSSION <br />                                                                                                                                                                                                                        42 <br />                                                                                                                                                                                                                        61 <br />                                                                                                                                                                                                                        50 <br />  <br />                                                                                                                                                                                                                                          63 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         68 <br />                                                                                                                                                                                                                                                         55 <br />                                                                                                                                                                                                                                                         48 <br />  <br />                                                                                                                                                                                                                                                                         37 <br />                                                                                                                                                                                                                                                                         54 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         38 <br />  <br />                                                                                                                                                                                                                                                                                                         42 <br />                                                                                                                                                                                                                                                                                                         60 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                    Identification of conserved fungal genes is a first step to- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 wards the discovery and development of future broad-spec- <br />                                                                                                                                                                                                                        35 <br />                                                                                                                                                                                                                        54 <br />                                                                                                                                                                                                                        46 <br />  <br />                                                                                                                                                                                                                                          50 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         63 <br />                                                                                                                                                                                                                                                         44 <br />                                                                                                                                                                                                                                                         39 <br />  <br />                                                                                                                                                                                                                                                                         21 <br />                                                                                                                                                                                                                                                                         45 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         42 <br />  <br />                                                                                                                                                                                                                                                                                                         31 <br />                                                                                                                                                                                                                                                                                                         43 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 trum antifungal agents. Although the inclusion of medically <br />                                                                                                                                        % Protein identity with other fungi <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 important fungi should conceivably be adequate for the anal- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 ysis, we chose to include as many available completed fungal <br />                                                                                                                                                                                                                        68 <br />                                                                                                                                                                                                                        71 <br />                                                                                                                                                                                                                        74 <br />  <br />                                                                                                                                                                                                                                          85 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         85 <br />                                                                                                                                                                                                                                                         72 <br />                                                                                                                                                                                                                                                         65 <br />  <br />                                                                                                                                                                                                                                                                         57 <br />                                                                                                                                                                                                                                                                         71 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         58 <br />  <br />                                                                                                                                                                                                                                                                                                         58 <br />                                                                                                                                                                                                                                                                                                         80 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 genomes as possible for the following reasons. First, a human <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 pathogen is a relative term from a clinical perspective. For <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 example, C. albicans and A. fumigatus are both opportunistic <br />                                                                                                                                                                                                                        61 <br />                                                                                                                                                                                                                        62 <br />                                                                                                                                                                                                                        54 <br />  <br />                                                                                                                                                                                                                                          64 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         76 <br />                                                                                                                                                                                                                                                         60 <br />                                                                                                                                                                                                                                                         57 <br />  <br />                                                                                                                                                                                                                                                                         43 <br />                                                                                                                                                                                                                                                                         68 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         46 <br />  <br />                                                                                                                                                                                                                                                                                                         49 <br />                                                                                                                                                                                                                                                                                                         72 <br />  TABLE 3. Properties of fungal genes of unknown function identified in this work <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 human pathogens that did not produce deep-organ, potentially <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 fatal diseases in humans until recently due to advances in <br />                                                                                                                                      neoformans <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 modern medicine (32, 38). Second, broad-spectrum antibiotics <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 and antifungals do not discriminate between pathogenic and <br />                                                                                                                                                                                                                        21 <br />                                                                                                                                                                                                                        45 <br />                                                                                                                                                                                                                        44 <br />  <br />                                                                                                                                                                                                                                          44 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         54 <br />                                                                                                                                                                                                                                                         42 <br />                                                                                                                                                                                                                                                         34 <br />  <br />                                                                                                                                                                                                                                                                         22 <br />                                                                                                                                                                                                                                                                         43 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         34 <br />  <br />                                                                                                                                                                                                                                                                                                         27 <br />                                                                                                                                                                                                                                                                                                         44 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 nonpathogenic microbes. In addition, we wish to emphasize <br />                                                                                                                                   C. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 that the 240 conserved fungal genes identified in this work <br />                                                                                                                         fumigatus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 should serve as a starting point for future fine tuning as more <br />                                                                                                                                                                                                                                                                                                                         Combined information taken from both the SGD (http://www.yeastgenome.org) and the CGD (http://www.candidagenome.org). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 fungal genomes are sequenced and more essential genes in <br />                                                                                                                                                                                                                        25 <br />                                                                                                                                                                                                                        50 <br />                                                                                                                                                                                                                        26 <br />  <br />                                                                                                                                                                                                                                          36 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         66 <br />                                                                                                                                                                                                                                                         47 <br />                                                                                                                                                                                                                                                         35 <br />  <br />                                                                                                                                                                                                                                                                         25 <br />                                                                                                                                                                                                                                                                         40 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         33 <br />  <br />                                                                                                                                                                                                                                                                                                         33 <br />                                                                                                                                                                                                                                                                                                         42 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 each species are identified. <br />                                                                                                                      A. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                    Although our concordance analysis is a powerful tool for <br />                                                                                                             glabrata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 multiple-genome analysis, certain genes would inevitably be <br />                                                                                                                                                                                                                        57 <br />                                                                                                                                                                                                                        63 <br />                                                                                                                                                                                                                        54 <br />  <br />                                                                                                                                                                                                                                          68 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         75 <br />                                                                                                                                                                                                                                                         65 <br />                                                                                                                                                                                                                                                         58 <br />  <br />                                                                                                                                                                                                                                                                         44 <br />                                                                                                                                                                                                                                                                         66 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         46 <br />  <br />                                                                                                                                                                                                                                                                                                         46 <br />                                                                                                                                                                                                                                                                                                         71 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 missed in our gene set. They include proteins that share con- <br />                                                                                                          C. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 served function due to structural and/or functional motif (for <br />                                                                                                 albicans <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 example, active site) similarity but not primary sequence sim- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 ilarity (14). Other genes that can be missed are those that are <br />                                                                                                                                                                                                                        59 <br />                                                                                                                                                                                                                        63 <br />                                                                                                                                                                                                                        56 <br />  <br />                                                                                                                                                                                                                                          70 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         75 <br />                                                                                                                                                                                                                                                         65 <br />                                                                                                                                                                                                                                                         58 <br />  <br />                                                                                                                                                                                                                                                                         43 <br />                                                                                                                                                                                                                                                                         69 <br />  <br />  <br />                                                                                                                                                                                                                                                                                         50 <br />  <br />                                                                                                                                                                                                                                                                                                         49 <br />                                                                                                                                                                                                                                                                                                         69 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 nonessential when evaluated individually, yet the protein com- <br />                                                                                              C. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 plex, for example, to which the gene product is assembled can <br />                                                                                    Essential <br />                                                                                     in this <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 be vital for cell growth. The fungal &#226;&#65533;&#164;-1,3-glucan synthase is <br />                                                                                      work <br />  <br />                                                                                                                                                                                                                        Yes <br />                                                                                                                                                                                                                        Yes <br />  <br />  <br />                                                                                                                                                                                                                                          Yes <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         Yes <br />  <br />                                                                                                                                                                                                                                                         Yes <br />  <br />                                                                                                                                                                                                                                                                         Yes <br />                                                                                                                                                                                                                                                                         Yes <br />  <br />  <br />                                                                                                                                                                                                                                                                                         Yes <br />  <br />                                                                                                                                                                                                                                                                                                         Yes <br />                                                                                                                                                                                                                                                                                                         Yes <br />                                                                                                                                                                                                                        No <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         No <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 such an example, where neither S. cerevisiae FKS1 (ScFKS1) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 nor ScFKS2 is required individually for cell viability, yet the <br />                                                                                                                                                                                                                        Viable/inviable <br />                                                                                    Null mutant <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                          Heterozygous <br />                                                                                                                                                                                                                        Not available <br />                                                                                                                                                                                                                        Not available <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         Not available <br />  <br />                                                                                                                                                                                                                                                         Not available <br />  <br />                                                                                                                                                                                                                                                                         Not available <br />                                                                                                                                                                                                                                                                         Not available <br />  <br />  <br />                                                                                                                                                                                                                                                                                         Not available <br />  <br />                                                                                                                                                                                                                                                                                                         Not available <br />                                                                                                                                                                                                                                                                                                         Not available <br />                                                                                    <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 disruption of both is lethal (35). Another example is transla- <br />                                                                                     in CGD <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                            viable <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 tional elongation factor 2, which is encoded by two nonessen- <br />                                                                                                                                                                                                                                                         Inviable <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 tial genes (EFT1 and EFT2) (42). <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                    To qualify as potential antifungal targets, the conserved fun- <br />                                                                                                                                                                                                                                                       5S rRNA binding in S. cerevisiae; <br />  <br />                                                                                                                                                                                                                                                          and filamentous growth in C. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 gal genes have to be essential for fungal cell growth, especially <br />                                                                                                                                                                                                                                                          cyclase-like protein involved <br />                                                                                                                                                                                                                                                          Involved in chitin synthesis <br />                                                                                                                                                                                                                                                       Unknown protein involved in <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                       Unknown protein involved in <br />  <br />  <br />                                                                                                                                                                                                                                                       Unknown protein involved in <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 for medically important fungi. Since the verification of gene <br />                                                                                                                                                                                                                                                          ribosomal large-subunit <br />                                                                                                                                                                                                                                                       RNA terminal phosphate <br />                                                                                                                                                                                                                                                       Unknown protein; rRNA <br />                                                                                                                                                                                                                                                       Arginyl-tRNA synthetase <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 essentiality is a tremendous task even in the model yeast S. <br />                                                                                                                                                                                                                        YLR100w/ERG27 ERG27/orf19.3240 3-Keto sterol reductase <br />                                                                                                                          Descriptiona <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                          in rRNA processing <br />  <br />  <br />                                                                                                                                                                                                                                                       RNA, protein binding <br />                                                                                                                                                                                                                                                          RNA metabolism <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                          rRNA processing <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 cerevisiae, we decided to initiate the conserved fungal gene <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 identification process by using the essential gene set of S. <br />                                                                                                                                                                                                                                                       Protein binding <br />  <br />                                                                                                                                                                                                                                                          processing <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                          biogenesis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 cerevisiae. We reasoned that fungal genes that are conserved <br />                                                                                                                                                                                                                                                          albicans <br />                                                                                                                                                                                                                                                       Unknown <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                       Unknown <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 with the essential S. cerevisiae genes would more likely be <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 essential in other fungi. In this regard, the conserved gene set <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 identified previously by Braun et al. (5) from six fungi (C. <br />                                                                                                                                  Gene ID in CGD <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 albicans, S. cerevisiae, Schizosaccharomyces pombe, A. niger, <br />                                                                                                                                                                                                                                      CSI2/orf19.5232 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 Magnaporthe grisea, and Neurospora crassa) would likely con- <br />                                                                                                                                                                                                                                      orf19.3341 <br />                                                                                                                                                                                                                                      orf19.2708 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                      orf19.6463 <br />                                                                                                                                                                                                                        YGR145w/ENP2 orf19.6686 <br />  <br />                                                                                                                                                                                                                                      orf19.5066 <br />                                                                                                                                                                                                                                      orf19.1886 <br />  <br />  <br />                                                                                                                                                                                                                                      orf19.3724 <br />  <br />                                                                                                                                                                                                                                      orf19.6955 <br />                                                                                                                                                                                                                        YLR009w/RLP24 orf19.4191 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 tain many genes that are not essential, even in S. cerevisiae (for <br />                                                                                                                                                                                                                        YNL132w/KRE33 orf19.512 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 example, ARO4, DAL5, and AGP2, to name a few). Those <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 nonessential genes would therefore not be considered antifun- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 gal targets. <br />                                                                                                                                                                                                                        YOR056c/NOB1 <br />                                                                                                                                                                                                                        YOL010w/RCL1 <br />                                                                                                                                                                                                                        YLR022c/SDO1 <br />                                                                                                                                                                                                                        YDR341c/RRS1 <br />  <br />  <br />                                                                                                                                                                                                                        YOL077c/BRX1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                        YJR072c/NPA3 <br />                                                                                                           Gene ID for <br />                                                                                                           S. cerevisiae <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                    In addition to finding conserved fungal genes, the identifi- <br />                                                                                                                                                                                                                        YOR004w <br />                                                                                                                                                                                                                        YDR412w <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 cation of conserved genes of unknown function would be an- <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 other way of prioritizing targets in the hope of finding novel <br />                                                                                                                                                                                                                                                                                                                         a <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                 classes of broad-spectrum inhibitors. The 12 CURFs evaluated <br />  VOL. 5, 2006                                                              IDENTIFICATION OF CONSERVED FUNGAL GENES                          645 <br />  <br />  <br />  <br />  <br />   FIG. 3. Growth curves of different C. albicans strains in media containing or lacking methionine or cysteine. (A to D) Two conditional mutants <br /> harboring genes of CaYDR341c and CaNMT1, respectively, in SD plus histidine broth containing methionine (A and C) or cysteine (B and D). <br /> (E and F) Growth of the parental control strain BWP17 in YEPD broth containing methionine (E) or cysteine (F). A culture grown overnight was <br /> inoculated with an inoculum of 1 &#226;&#171;&#187; 106 CFU/ml on a 96-well plate containing 200 &#226;&#65533;&#174;l of broth medium with various concentrations (millimolar) <br /> of methionine or cysteine as indicated. The plate was incubated at 30&#194;&#176;C, and cell growth was monitored by reading the optical density at 600 nm <br /> (OD600) at the indicated time intervals. <br />  <br />  <br />  <br />  <br /> here fall into this category. They were identified several years          of the inclusion of more genome sequences for this type of <br /> ago via the concordance tool by using the two yeast (S. cerevi-           analysis. On the other hand, unknown functionality of a gene is <br /> siae and C. albicans) genome sequences available at that time             a relative term, and as time goes on, an increasing number of <br /> (52), with the anticipation that genes conserved between these            genes will be found to possess a cellular function or role via <br /> two fungi would likely to be conserved among other fungi as               either functional characterization or genome annotation. Ex- <br /> well. Now that more fungal genome sequences are available,                amples include YLR100w and YDR341c, which recently were <br /> we found that 5 out of the 12 genes evaluated are conserved at            found to encode 3-keto sterol reductase (18) and arginyl-tRNA <br /> 40% or greater among the 10 fungal genomes (Table 2). This                synthetase (47), respectively. <br /> result suggests that around 50% of the conserved gene set                    We wish to stress that the successful development of a <br /> previously identified from S. cerevisiae and C. albicans may not          broad-spectrum antifungal drug requires knowledge and re- <br /> be conserved among other fungi, highlighting the importance               search beyond the identification of conserved fungal genes. For <br />  646       LIU ET AL.                                                                                                                      EUKARYOT. CELL <br />  <br />  <br />       TABLE 4. Repressibility of essential C. albicans genes by                     CaMET3 promoter system proven in this work to be robust and <br />                    methionine or cysteinea                                          reliable for evaluating gene essentiality in C. albicans, it may <br />                                                              IC50 (mM)              also provide additional features. By using the triply marked <br />   C. albicans gene ID,          Systematic ID <br />         this work                  in CGD                                           BWP17 strain, the availability of two promoter-swapping cas- <br />                                                      Methionine         Cysteine <br />  <br /> CaERG1                          orf19.406                0.48            NDb <br /> CaYDR341c                       orf19.3341               1.41             0.06 <br /> CaYOL077c                       orf19.5232               2.60            ND <br /> CaYLR009w                       orf19.4191               1.0             ND <br /> CaERG27                         orf19.3240             &#226;&#172;&#381;10                1.10 <br /> CaNMT1                          orf19.4641             &#226;&#172;&#381;10                1.23 <br /> CaRAM2                          orf19.4817             &#226;&#172;&#381;10                1.44 <br /> CaYNL132w                       orf19.512              &#226;&#172;&#381;10                0.73 <br /> CaYGR145w                       orf19.6686             &#226;&#172;&#381;10                0.54 <br /> CaYDR412w                       orf19.5066             &#226;&#172;&#381;10                1.25 <br /> CaYOL010w                       orf19.1886             &#226;&#172;&#381;10                0.83 <br /> CaYOR004w                       orf19.3724             &#226;&#172;&#381;10                1.19 <br /> CaYOR056c                       orf19.6955             &#226;&#172;&#381;10                0.23 <br /> BWP17 control strain                                   &#226;&#172;&#381;10              &#226;&#172;&#381;10c <br />    a <br />      A culture of the conditional mutant grown overnight was inoculated with an <br /> innoculum of 1 &#226;&#171;&#187; 106 CFU/ml on a 96-well plate containing 200 &#226;&#65533;&#174;l of SD plus <br /> histidine broth (YEPD medium for strain BWP17) in the presence of 0 mM or <br /> various concentrations of methionine or cysteine that was serially diluted (one- <br /> to twofold), ranging from 0.0097 mM to 10 mM. The plate was incubated at 30&#194;&#176;C <br /> for 30 h, and cell growth was monitored by reading the optical density at 600 nm. <br /> For calculating IC50s, cell growth in the absence of methionine or cysteine was <br /> set at 100%, while the medium only was used as a blank. <br />    b <br />      ND, not determined. <br />    c <br />      Percent inhibition at 10 mM cysteine is 43%, as calculated using the 11-h <br /> time point (Fig. 3), yet no significant inhibition was observed with &#226;&#177;&#8226;5 mM <br /> cysteine. <br />  <br />  <br />  <br /> instance, the &#226;&#65533;&#164;-1,3-glucan synthases (Fks1p and Fks2p) are <br /> well conserved, at 52 to 86% identity across the 10 fungi (Table <br /> 1), yet its inhibitor, caspofungin, is not active against Crypto- <br /> coccus neoformans, presumably due to the lack of or the low <br /> content of glucan in its cell wall. On the other hand, however, <br /> the broad-spectrum inhibitors of Erg11p, voriconazole and <br /> posaconazole (conserved at 43% or greater across the 10 fungi <br /> studied) (Table 1), have been successfully developed even <br /> though the inhibitor fluconazole lacks activity against Aspergil- <br /> lus species. <br />    In an effort to evaluate the essentiality of the conserved <br /> fungal genes identified in this work in C. albicans, we devel- <br /> oped a repressible MET3 promoter system capable of the <br /> large-scale evaluation of gene essentiality. Although the MET3 <br /> promoter as well as several other endogenous and exogenous <br /> regulatable promoters, including GAL1 (22), MAL1 (6), MAL2 <br /> (3), PCK1 (33), and MRP1 (37), have been used for evaluating <br /> the essentiality of many genes in C. albicans, only the exoge- <br /> nous tetracycline (Tet) promoter system has been proven to <br /> possess the capacity for the identification of essential genes on <br /> a genome-wide scale (44). While the recently reported PCR- <br /> based promoter (PMET3-green fluorescent protein [GFP], <br /> PGAL1-GFP, and PPCK1-GFP) cassettes (19) could conceivably <br /> be used for large-scale essential gene evaluation, the cassettes <br /> have yet to be demonstrated for this utility. In addition, it is not <br />                                                                                        FIG. 4. Dose-response curves of the PMET3-CaERG1/Caerg1::ARG4 <br /> yet known if the placement of the GFP at the amino terminus                         conditional mutant for the antifungal drugs terbinafine (A), fluconazole <br /> of the target protein would complicate the evaluation of gene                       (B), and sordarin (C) in the absence (&#226;&#171;&#186;) and presence (&#226;&#171;&#185;) of methionine. <br /> essentiality and function. Therefore, to extend the utility of the                  A culture grown overnight was inoculated with an inoculum of 1 &#226;&#171;&#187; 106 <br /> endogenous CaMET3 promoter, in this report, we constructed                          CFU/ml on a 96-well plate containing 200 &#226;&#65533;&#174;l of SD plus histidine broth <br />                                                                                     with various concentrations of drugs as indicated. Where indicated, me- <br /> two PCR-based promoter-swapping cassettes harboring either                          thionine was added to 0.2 mM, a concentration inhibiting 50% of the cell <br /> the CaURA3 or the CaARG4 selective marker for use in the                            growth. After 24 h at 30&#194;&#176;C, the optical densities were determined with a <br /> evaluation of gene essentiality in C. albicans. Not only was this                   plate reader at 600 nm. <br />  VOL. 5, 2006                                                             IDENTIFICATION OF CONSERVED FUNGAL GENES                      647 <br />  <br />  <br />                                                                          effector of the CaMET3 promoter. It should be noted, though, <br />                                                                          that a high concentration of cysteine (10 mM) did appear to <br />                                                                          slightly slow the growth of the parental BWP17 strain (Fig. 3F), <br />                                                                          albeit with the IC50 of cysteine being &#226;&#177;&#8211;10 mM (Table 4). <br />                                                                             We demonstrated that the CaERG1 conditional mutant <br />                                                                          strain displayed, in the presence of a rate-limiting amount of <br />                                                                          methionine, hypersensitivity to the specific inhibitor terbin- <br />                                                                          afine but not to other antifungals such as fluconazole and <br />                                                                          sordarin (Fig. 3). It should be noted that hypersensitivity to <br />                                                                          fluconazole and other azoles was reported with a CaERG1 <br />                                                                          conditional mutant placed under the control of the CaMET3 <br />                                                                          promoter (40). While different strain backgrounds (CAI4 ver- <br />                                                                          sus BWP17) could account for the discrepancy, different assay <br />                                                                          methods (a spot assay on solid medium [40] versus a modified <br />   FIG. 5. Superimposition of HTS hits from parallel screening of a       CLSI macrodilution method used in this work) could also be <br /> PMET3-CaERG1/Caerg1::ARG4 conditional mutant in the presence (&#226;&#8212;&#65533;)         responsible for the observed differences in drug susceptibility. <br /> (0.4 mM) and absence (e) of methionine (Met-plus and Met-minus              Traditionally, antimicrobial agents were identified via whole- <br /> screen, respectively). Based on the percent inhibition range in the <br /> Met-minus screen, the positive hits identified using a cutoff of &#226;&#172;&#381;40%    cell screening. This approach, however, does not identify the <br /> growth inhibition in the Met-plus screen were sorted into three groups   biochemical target of a lead compound. On the other hand, <br /> (group I, &#226;&#171;&#186;11 to 10%; group II, 11 to 40%; group III, &#226;&#172;&#381;40% in the        enzyme- or target-based in vitro screening often yields potent <br /> Met-minus screen), which are separated by the dashed vertical lines.     enzyme inhibitors but with little or no whole-cell activity. <br />                                                                          Moreover, there has been little success with attempts to build <br />                                                                          in &#226;&#8364;&#339;whole-cell activity&#226;&#8364;&#65533; for these enzyme inhibitors (43). Using <br /> settes with two different selective markers permits the evalua-          the down-regulatable PMET3-CaERG1 conditional mutant <br /> tion of a gene in the rare situation where three allelic copies          strain, we demonstrated, as a proof of principle, that hyper- <br /> exist (23). In this case, the first allele is deleted using the HIS1     sensitive compound hits with whole-cell activity could be iden- <br /> selective marker, followed by sequential placement of the sec-           tified in a single parallel HTS campaign. Several lines of evi- <br /> ond and third alleles under the control of the CaMET pro-                dence suggest that those hypersensitive hits are target specific. <br /> moter using either the URA3 or the ARG4 cassette. It is note-            First, hypersensitivity of the PMET3-CaERG1 conditional mu- <br /> worthy that out of the 13 essential C. albicans genes identified         tant strain in the presence of methionine was observed only <br /> here, CaYJR072c, which appeared to be nonessential in this               with terbinafine and those six hits in group I of Fig. 5. Second, <br /> study, was reported previously by Davis et al. (10) to be re-            none of the six hits were found to inhibit the growth of an <br /> quired for cell growth. While the different methodologies used           efflux-deficient C. albicans strain as well as a strain of Staphy- <br /> (Tn7-UAU1 insertional mutagenesis versus repressible pro-                lococcus aureus at the HTS screening concentration (14.2 &#226;&#65533;&#174;M) <br /> moter) may account for the discrepancy, other possibilities              tested (data not shown). Third, the presence of methionine did <br /> could exist, since our attempt to obtain double disruption of            not alter the susceptibility of the PMET3-CaERG1 conditional <br /> this gene also failed (data not shown).                                  mutant to fluconazole and sordarin (Fig. 4B and C) as well as <br />    The CaMET3 promoter was further characterized and found               several other known antifungal inhibitors (control compounds <br /> to be highly amenable to controlled gene expression. By vary-            in the HTS), including papulacandin B, ascosteroside, pradi- <br /> ing the amount of methionine or cysteine added to the growth             mycin, cycloheximide, and benomyl, that bind to targets other <br /> medium and thus varying the cellular level of the essential gene         than CaErg1p compared to the Met-minus controls (data not <br /> product (8), cell growth was correspondingly delayed or inhib-           shown). On the other hand, while the hypersensitive hits are <br /> ited (Fig. 3 and Table 4). Interestingly, we found that cysteine         potential specific inhibitors, the common hits identified from <br /> could control expression of the CaMET3 promoter more                     the Met-plus and Met-minus screens may contain specific hits <br /> tightly than methionine, because all 13 conditional mutants              as well. One possible reason for this is that the active concen- <br /> could be repressed in growth by cysteine but not all could be            trations of those compounds in the screening solution were <br /> repressed by methionine at concentrations up to 10 mM (Table             high enough to inhibit cell growth under both Met-plus and <br /> 4). It is conceivable that the CaMET3 promoter is leaky when             Met-minus conditions. Therefore, individual dose-response <br /> regulated by methionine alone, since the maximum inhibition              determinations are needed to identify which common hit(s) is <br /> of cell growth by methionine never reached beyond 80% for all            specific to the target under evaluation. <br /> conditional mutants tested (data not shown). Although the                   Finally, the construction of conditional mutants by using the <br /> CaMET3 promoter has been utilized for the evaluation of the              CaMET3 promoter makes it possible to screen against any <br /> essentiality of several genes in C. albicans (8, 40, 53), it is not      target placed under this promoter, whether or not its function <br /> known whether methionine or cysteine was the more effective              is known or if a functional in vitro assay is available. Although <br /> modulator, since methionine and cysteine were often used                 new classes of inhibitors can be identified by rescreening ex- <br /> simultaneously. Contradictory results were also reported in the          isting drug targets, the new targets identified in this work <br /> S. cerevisiae literature regarding whether methionine, cysteine,         conceivably offer greater potential for the identification of <br /> or both are the true repressors of the enzymes of sulfur me-             novel classes of antifungal agents. Admittedly, the successful <br /> tabolism (39, 48). Nevertheless, our data presented here sug-            identification of novel targets as well as inhibitors does not <br /> gest that in a C. albicans background, cysteine is a much tighter        necessarily lead to a drug due to the complexity of the drug <br />  <br />  <br />  <br /> development process. Nevertheless, it is our hope that the                              <br /> streamlined process we developed proceeding from selection                      <br /> and validation of candidate antifungal targets to screening for                          <br /> specific inhibitors can be one of many useful approaches for       <br /> future antifungal drug discovery.                                                         <br /> </body></html>