<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Genetics of Cryptopleurine Resistancein Saccharomyces cerevisiae: <br />             Expression of a Ribosomal Protein Gene Family <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                   The Saccharomyces cereuisiae CRYl gene encodes the 40s ribosomal subunit protein rp59 andconfers <br />                sensitivity <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> theprotein synthesis inhibitor cryptopleurine. A yeast strain containing the cryl-Al::URA3 <br />                <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>null allele</span> is viable, cryptopleurine sensitive (Cry&#226;&#8364;&#65533;),and expresses rp59 mRNA, suggesting that there <br />                is a second functional CRY gene. The CRY2 gene has been isolated <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> a yeast genomic library <br />                cloned in bacteriophage X, using a CRYl DNA probe. The DNA sequence of the CRY2 gene contains <br />                an open reading frame encoding ribosomal protein 59 thatdiffers at five residues from rp59 encoded <br />                by the CRYl gene. The CRY2 gene was mapped to the left arm of chromosome X , centromere- <br />                proximal to cdc6 and immediately adjacent to ribosomal protein genes RPS24A and RPL46. Ribosomal <br />                protein 59 is an essential protein; upon sporulation of a diploid doubly heterozygous for cryl-A2::TRPl <br />                cryZ-Al::LEU2 null alleles, no spore clones containing both null alleles were recovered. Several results <br />                indicate that CRY2 is expressed, but at lower levels than CRYl: (1) Introduction of CRY2 on high copy <br />                plasmids into CryR yeast of genotype cryl CRY2 confers a CryS <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Transformation of these <br />                CryR yeast with CRY2 on a low copy CEN plasmid does not confer a Cry&#226;&#8364;&#732; phenotype. (2) Haploids <br />                containing the cryl-A2::TRPl null allele have a deficit of 40s ribosomal subunits, but cryZA1::LEUZ <br />                strains have wild-type amounts of 40s ribosomal subunits. (3) CRY2 mRNA is present at lower levels <br />                than CRYl mRNA. (4) Higher levels of &amp;galactosidase are expressed from a CRYl-lacZ gene fusion <br />                than from a CRY2-lacZ gene fusion. Mutations that alter or eliminate the last amino acid of rp59 <br />                encoded by either CRYl or CRY2 result in resistance to cryptopleurine. Because CRY2 (and cry2) is <br />                expressed at lower levels than CRYl (and cryl), the CryRphenotype of cry2 mutants is only expressed <br />                in strains containing a cryl-A null allele. <br />  <br />  <br />                                                                                  gene and acry 1 resistance allele havebeen cloned and <br /> T       0 facilitate our studies of yeastribosome biogen- <br />          esis, we previously isolated a ribosomal protein <br />  gene conferring a readily assayable phenotype, resist- <br />                                                                                  shown to encode the 40s ribosomal subunit protein <br />                                                                                  rp59(LARKINand WOOLFORD           1983;HIMMELFARB, <br /> ance to the plant alkaloid cryptopleurine (LARKIN     and                        VASSAROTTIand FRIESEN      1984). <br />  WOOLFORD       1983;LARKIN, THOMPSON         and WOOL-                             Homologs ofrp59 have beenidentified in a number <br />  FORD 1987). Cryptopleurine and therelated antibiotic                            of other organisms. The inferred amino acid sequence <br /> emetine inhibit protein synthesis in yeast and other                             of yeast rp59 is approximately 80% identical to that <br />  eukaryotes by binding t o a high affinity site on the                           of its homolog rpS14 in human, hamster, rat, Dro- <br />  40s ribosomal subunit and blocking the elongation                               sophila, maize, trypanosomes, and Neurospora. Ri- <br />  factor EF-2 and GTP-dependenttranslocation of pep-                              bosomal protein 59 is 50% identical in sequence to <br />  tidyl-tRNA from the A site to the P site of the ribo-                           Halobacterium cutirubrum ribosomal protein S 1 1 and <br />  some (BARBACID,FRE~NOand V ~ Z Q U E Z1975;                                     37% identical to the Escherichia coli ribosomal protein <br />  BUCHERand SKOCERSON            1976; DOLZ,V ~ Z Q U Eand <br />                                                        Z                         S11 (CHENet al. 1986; RHOADS,DIXIT and ROUFA <br /> J I M ~ N E Z 1982). Cryptopleurine resistanceinyeast                            1986; TANAKA,     ISHIKAWA and OCATA1986; LARKIN, <br /> most often results from a recessive <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span> in cryl,                            THOMPSON     and WOOLFORD    1987; BROWNet al. 1988; <br /> a gene approximately 2 cM centromere-proximal to                                 LARKIN   et al. 1989; PERELMAN and  BOOTHROYD     1990; <br /> the mating type locus (MAT)on chromosome HI (SKO-                                TYLER   and HARRISON    1990). These striking homolo- <br /> CERSON, MCLAUCHLIN           and WAKATAMA 1973; GRANT,                           gies among proteins from widely divergent species, as <br /> S ~ N C H Eand Z J I M ~ N E Z 1974). The wild-type CRYl                         wellas the associationof cryptopleurine resistance <br />                                                                                  with mutations in genes encoding rp59, attest to the <br />    &#226;&#8364;&#8482; Present address:Department of Genetics, SK-50,University of Washing-        functional importance of rp59 in translation and sug- <br /> ton, Seattle, Washington 98195. <br />    * Present address: MerckResearch Laboratories, P.O. Box 2000, 80Y-            gest that these proteins are involved in similar con- <br /> 230,Rahway, New Jersey 07065-0900. <br />    &#226;&#8364;&#8482; Present address: Division of Biological Sciences, University of Nebraska, <br />                                                                                  served reactions required for the elongation steps of <br /> 308 Manter Hall, Lincoln, Nebraska 68588-01 18.                                  protein synthesis. <br /> Genetics 135: 7 19-730 (November, 1993) <br />  720                                                A. G. Paulovich et al. <br />  <br />    Although resistant cryl alleles are recessive t o t h e    somes was probed using methods recommended by the <br /> wild type CRYl allele in heterozygous diploids, several       supplier (Clontech Laboratories, lnc., Palo Alto, California). <br />                                                               Libraries ofyeast genomic fragments cloned in bacterio- <br /> unusualfeaturesofthegeneticsofcryptopleurine                  phage X or cosmids were kindly provided by Drs. L. Riles <br /> resistancesuggestthat      a simpleMendeliandomi-             and M. Olson, (Washington University, St. Louis, Missouri, <br /> nance/recessiveness relationship is inadequate to de-         and University of Washington, Seattle, Washington), and <br /> scribe the behavior of this system. T h e most striking       were probed as described in BENDER       and PRINGLE(1991). <br /> observation is that diploid yeast strains monosomic           The 5&#226;&#8364;&#8482; ends of the CRY2 transcripts were mapped by primer <br />                                                               extension analysis,using the oligonucleotide primer 5&#226;&#8364;&#8482;- <br /> for chromosome ZZZ are sensitive to cryptopleurine            GACTTGAGAGTTATGGCG-3&#226;&#8364;&#8482;, as described in LARKIN, <br /> even if the remaining copyof chromosome 111 carries           THOMPSON     and WOOLFORD(1987). Oligonucleotides 5&#226;&#8364;&#732;- <br /> a cryl resistance allele (MEADE, RILEY a n d MANNEY           TACAAGTACAATACATGCA-3&#226;&#8364;&#8482; and 5&#226;&#8364;&#65533;AAGCCGTAT <br />  1977; CLARKE and    CARBON1983). In an elegant series        TTCCATGCG-3&#226;&#8364;&#8482;, complementary to unique sequences in <br /> of experiments, MEADE, RILEYa n d MANNEY(1977)                the 3&#226;&#8364;&#732; untranslated regions of CRYl and CRY2, respectively, <br />                                                               were end-labeled and usedasgene-specific hybridization <br /> extended this observation toa series of aneuploid and         probes to detect CRYl and CRY2 mRNAsinRNAblot <br /> polyploid yeast strains and demonstrated that crypto-         assays. Steady-state levels of CRYl and CRY2 mRNA were <br /> pleurine resistance is a function of the ratio of the         measured by quantitative S1 nuclease protection assays that <br /> number of the cryl resistance alleles t o ploidy. O n e       distinguish CRYl and CRY2 mRNAs. The 1.1-kb HindIII- <br /> interpretation of theseresults is that the level of           Bglll fragment containing the 5&#226;&#8364;&#732; two-thirds of CRYl was <br />                                                               usedas a diagnostic probe for CRY1 and CRY2 mRNAs. <br /> resistance depends on an interaction between     cryl a n d   Because the nucleotide sequences of the 3&#226;&#8364;&#8482; exons of CRYl <br /> a geneorgenesnotlocatedonchromosome                   ZZZ.    and CRY2 diverge at their 5&#226;&#8364;&#732; ends, a shorter CRYl DNA <br /> Alternatively, the CRYl locus may represent only one          fragment is protected against S1 nuclease digestion when <br /> member of a ribosomal protein gene family encoding            hybridized to CRY2 mRNA than when hybridized to CRYl <br /> rp59 with at least one member of the family located           mRNA. A 0.9-kb XbaI-BglIl fragment was used as a hybrid- <br />                                                               ization probe for R P L l mRNA (DESHMUKH       et al. 1993). S1 <br /> o n a chromosome other than chromosome           ZZZ. Be-     nuclease protection assays were carried out as described in <br /> cause halfof the yeast ribosomal protein genes studied        OSLEYet al. (1986), using the above DNA probes labeled <br /> thusfar are duplicatedand essential(reviewedin                with 32P at the 5&#226;&#8364;&#8482; ends of their BgllI sites, and hybridized <br /> WOOLFORD and WARNER 199 1; WOOLFORD 199 I), we                to 20 Pg of total yeast RNA. <br /> favored the secondpossibility. In this report, we dem-           Isolation of CRY;!genomic clones from a yeast recom- <br />                                                               binantphagelibrary: Twenty thousand plaques from a <br /> onstrate that there is a second gene encoding rp59,           library of EcoRI partial genomic fragments of yeast DNA <br /> CRY2, located on chromosomeX , providing an expla-            cloned in bacteriophage X Charon 4A (WOOLFORDand <br /> nation for the genetic behavior of cryptopleurine re-         ROSBASH    1981) were screened by plaque hybridization with <br /> sistance in Saccharomyces cerevisiae. We note that cryl       32P-labeledplasmid pJL32 containing the 3.5-kb EcoRl frag- <br /> resistance mutations are examples of recessive muta-          ment that includes most of CRYl (LARKIN       and WOOLFORD <br />                                                               1983). Positive plaques were counterscreened with SzP-la- <br /> tions in a single member of a multigene family. T h e         beled plasmid pJL37 containing a 1.5-kb EcoRI fragment <br /> CryR phenotype conferred     by cryl is detectable incryl     that includes only the 5&#226;&#8364;&#8482; exon and 5&#226;&#8364;&#8482; end of the intron of <br /> CRY2 haploids because CRYl is expressed at higher             CRYl. <br /> levels than CRY2.                                                DNA sequencing: A series of nested deletions was gen- <br />                                                               erated across the 3.0-kb EcoRI-BamHl CRY2 genomic frag- <br />                                                               ment subcloned in the Bluescript SK+ vector (Stratagene, <br />              MATERIALS AND METHODS <br />                                                               LaJolla, CA) usingexonuclease I11 and mung bean nuclease. <br />    Strains and media:S. cerevisiae strains used in this study Single-stranded templates were prepared and DNA sequenc- <br /> are listedin Table 1. E. coli strains HBlOl and NM522         ing was performed using the Sequenase system as recom- <br /> (Stratagene, LaJolla, California) were used to propagate      mended by the manufacturer. The sequence of CRY2 has <br /> plasmids, and strain Dp50supF (MANIATISet al. 1978) was       been submitted to the GenBank database under accession <br /> used to propagate bacteriophage X. Standard methods were      number L12564. <br /> employed for yeast genetics (MORTIMERand HAWTHORNE               Construction of plasmids and cyZA or c y 2 A null al- <br /> 1969; SHERMAN,     FINKand HICKS1986). Yeast were trans-      leles: The 4.6-kb EcoRI CRY2 genomic fragment cloned in <br /> formed by the LiAc procedure of ITO et al. (1983). Cryp-      bacteriophage X was subcloned by standard methods into <br /> topleurine phenotypes were assayed on medium Containing       the EcoRl site of plasmidpBR325 (BOLIVAR      1978). The 3.0- <br /> 5-1 5 PM cryptopleurine obtained from Corkwood Enter-         kb EcoRI-BamHI fragment containing CRY2 was inserted <br /> prises, <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> F, 61 Waratah St., Kirrawee, New South Wales,    between the EcoRI and BamHl sites of plasmids pBR322 <br /> Australia.                                                    and YEpl3 (BROACH,      STRATHERN and     HICKS1979) to pro- <br />    Manipulation of nucleic acids: Methods for preparation     duce plasmids pBR322CRY2RB and YEpl3CRY2RB, re- <br /> and electrophoresis of DNA and RNA, transfer to nylon         spectively. Plasmid pCRY2 was identified by hybridization <br /> membranes (Schleicher &amp; Schuell, Keen, New Hampshire)         ofthe  3.0-kb EcoRI-BamHl CRY2 to a library of yeast partial <br /> and hybridization with radiolabeled DNA probes have been      Sau3A fragments cloned in vector YEp24 (CARLSON            and <br /> described previously (MANIATIS,    FRITSCHand SAMBROOK BOTSTEIN1982). Plasmid pJT3O is a derivative of YCp50 <br />  1982; FEINBERG   and VOGELSTEIN   1983; LARKIN and   WOOL-   that contains the cryl resistance allele and SUP1 1. <br /> FORD 1983; LAST, STAVENHAGEN         and WOOLFORD       1984;    The cryl-Al::URA3 deletion-insertion plasmidpcry 1- <br /> LARKIN,THOMPSON        and WOOLFORD1987;SAMBROOK,             AURAwas constructed by inserting the l.l-kb Hind111 <br /> FRITSCH   and MANIATIS1989).                                  fragment containing the yeast URA3 gene between the <br />    A blotof electrophoretically separated yeast chromo-       EcoRI and BglII sites internal to CRYl as follows: the 2.2-kb <br />                                                  Cryptopleurine Resistance in Yeast                               1           72 <br />                                                               TABLE 1 <br />                                           Saccharomyces cerevisiae strains used in this study <br />  <br />    Strain                                         Genotype                                                   Source <br />  <br /> A364A       MATaadel-1 ade2-1 uraltyrl-1 his7 lys2-1 gall                                       HARTWELL    (1968) <br /> BJ423       MATa cryl hisl-1                                                                    E. W. JONES <br /> DBY747      MATa his3-A1 leu2-3,112 trpl-289 ura3-52                                            Yeast Genetics Stock Center, <br />                                                                                                   Berkeley, California <br /> JLYlO     MATa cryl trpl-289 ura3-52                                                            Spore clone from BJ423 X DBY747 <br /> JLYl1     MATa <br />              leu2-3,112 <br />                      ura3-52                                                                    Spore clone from BJ423 X DBY747 <br /> JLY 12    MATa/MATacyl/CRYl leu2-3,112/LEU2 trpl-289/TRPIura3-52/ura3-52                        JLYlO X JLYll <br /> RL92      MATaleu2-3,112 <br />                       prp2-I <br />                           ura3-52                                                               LASTet al. ( 1 984) <br /> JLYS      MATa cryl leu2-3,112ura3-52 trpl                                                      This study <br /> BJ3344    MATa his3-A2OO leu2-AI lys2-801 ura3-52 trpl-A101 can1                                E. W. JONES <br /> BJ3345    MATa ade2-101 his3-A200 leu2-A1 lys2-801 trpl-A101 ura3-52                            E. W. JONES <br /> MMY1401   MATaIMATa adeP-lOI/ADEZ his3-A200/his3-A200 leu2-Al/leu2-A1 lys2-801/lys2-801         BJ3344 X BJ3345 <br />             trpl-AlOl/trpl-A101 ura3-52/ura3-52 canl/CANl <br /> MPY1402 MATa cryl-A2::TRPI CRY2 ade2-101 his3-A200 leu2-AI lys2-801 trpl-AI01 ura3-52           This study <br />  MPY1403 MATa CRYl cry2-AI::LEU2 his3-A200 leu2-AI lys2-801 trpl-AI01 ura3-52                   This Study <br />  MPY 1404 MATa/MATa cryl-A2::TRPI/CRYI cry2-Al::LEU2/CRY2 adeZ-lOI/ADEP his3-A200/              MPY 1402 X MPY 1403 <br />             his3-AZO0 leu2-Al/leu2-Al lys2-801/lys2-801 trpl-AlOl/trpl-AI01 ura3-52/ura3-52 <br /> JTY69     MATaIMATa his3-A200/his3-A200 trpl-Al/trpl-A2 leu2-A2/leu2-A2ura3-167/ura3-167        This Study <br /> JTY78     MATaIMATa  cryl-A2::TRPlICRYI      cry2-Al::LEU2/CRY2 his3-A200/his3-A200 t r f l -   This Study <br />             Al/trpl-Al leu2-A2/leu2-A2 ura3-167/ura3-167 <br /> JTY8l     MATa <br />            CRYl        cry2-Al::LEU2 his3-A200 trpl-AI leu2-A2 <br />                                                              ura3-167                           Spore clone from JTY78 <br /> JTY82     MATa <br />            cryl-A2::TRPl <br />                      CRY2             his3-A200 trpl-A1 leu2-A2 ura3-167                        Spore clone from JTY78 <br /> JTY83     MATa <br />           CRYl         CRY2 his3-A200 trpl-A1 leu2-A2 <br />                                                   ura3-167                                      Spore clone from JTY78 <br /> DBY745 cryMATa cryl leu2-3,112 <br />                       adel       ura3                                                           J. HABER <br />  <br />  <br />  <br />  <br /> CRYl HindIII fragment from plasmid pCRYl (LARKIN and                  CRY2 coding sequence in pcry2ALEU are three codons in <br /> WOOLFORD1983) was cloned into the HindIII site ofa                    the short 5&apos; exon and the final 49 codons of the 3&apos; exon. <br /> derivative ofplasmid pBR322 (BOLIVAR        et al. 1977) bearing      T h e deletion-insertion endpoints of each crylA or cry2A null <br /> a filled-in, inactivated EcoRI site to create plasmid pJL59.          allele construct were confirmed by DNA sequencing. <br /> pJL59 was digested with EcoRI and BglII. T h e resulting                 One-step replacements of the yeast CRYl or CRY2 genes <br /> ends were repaired with DNA polymerase Klenow fragment                were performed by the method of ROTHSTEIN(1983). Lin- <br /> and XhoI linkers were ligated to the ends. This DNA was               ear HindIII fragments(5-10 rg) containing cryl-AI::URA3 <br /> digested with XhoI and recircularized by ligation to create           or cryl-AZ::TRPI, and the AatII-BamHI fragment contain- <br /> the CRYl deletion plasmid pJL127. T h e 1.1-kb HindIII                ing cry2-A1::LEUZ were transformedinto yeast diploids <br /> fragment bearing URA3 was purified from plasmid YEp24,                JLY12, MMY1401, orJTY78. <br /> the Hind111 ends were filled in, and XhoI linkers were ligated           Analysis of yeast ribosomal subunits: Yeast cells were <br /> to the ends. The resulting fragment was ligated into the              grown to a density of 1.0-1.5 X lo8 cells/ml in 200 ml of <br /> XhoI site of pJL 127 to createplasmid pcryl AURA.                     YEPD medium at 30&quot;. Lysates were prepared in buffer <br />    T h e deletion-insertion allele cryl-AP::TRPI was derived          containing 0.5 M KCI. Total amounts of cellular ribosomal <br /> from pJL127 as follows. T h e EcoRI site within CRY1 had              subunits were quantified by centrifugation of cell extracts <br /> been regenerated upon construction ofpJL127. Plasmid                  at 27,000 rpm for   5 hr at 13&#194;&#176;C on  7% to 47%linear sucrose <br /> pJL127 was digested with EcoRI, andthe 1.45-kb yeast                  gradients containing 0.5 M KCI, as described in Y.-F. TSAY, <br /> TRPl EcoRI fragment purified from       plasmid YRp7 (STRUHL          G . SHANKWEILER,     J. LAKE,and J. L. WOOLFORD,      JR. (un- <br /> et al. 1979) was insertedtocreatepcryl              ATRP. T h e       published data). <br /> pcry 1AURA and pcryl ATRP constructs both          lack all of the       Construction and assay of CRYI-lac2 and CRYZ-lac2 <br /> CRYl coding sequence except the first two codons present              gene fusions: T h e 1.1-kb HindIII-BglII fragment contain- <br /> in the short 5&apos; exon and the final 29 codons. This second             ingthe 5&apos;two-thirdsof       CRYlwas ligated between the <br /> disruption of CRYl using TRPl was constructed tofacilitate            HindIII and BamHI sites of plasmid YIp356R (MYERSet al. <br /> subsequent experiments employing crylA <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>null mutants</span> con-              1986) to create an in-frame fusion of CRYl to the E. coli <br /> taining CRYl on a plasmid bearing URA3 (MORITZ et al.                 lac2 gene. Likewise, the 1.3-kb EcoRI-BglII fragment con- <br /> 1990).                                                                taining the 5&apos; two-thirds of CRY2was ligated between the <br />    The deletion-insertion allele cry2-A1::LEUZwas derived             EcoRI and BamHI sites of YIp357 (MYERSet al. 1986) to <br /> from plasmid pBR322CRY2RB by digesting this DNA with                  fuse CRY2 in-frame with l a d . Each construct was directed <br /> EcoRV and Asp718, repairing the Asp718 end with Klenow                to integrate at the ura3-167locus of yeast strain JTY83 by <br /> fragment of DNA polymerase, ligating XhoI linkers to the              linearization at theunique StuI site within the plasmid-borne <br /> ends, and recircularizing the plasmid by ligation to create           URA3 marker and transformation selecting for Ura+. Ge- <br /> the plasmid pcry2A. pcry2A was then digested with XhoI                nomic Southern hybridization confirmed the expected in- <br /> and a 2.2-kbXhoI-Sal1 yeast LEU2fragment was inserted by              tegration events. Expression of each gene fusion was assayed <br /> ligation to create plasmid pcry2ALEU. All that remains of             on solid medium containing X-gal and in extracts fromcells <br />  722                                                        A. G. Paulovich et al. <br />  <br />  <br />  <br />  <br />                                           X                                           X <br />   FIGURE 1.-Replacement of the CRYI gene with a cryI-Al::URA3 null allele. The thick horizontal arrow depicts the CRY1 gene. Black <br /> regions are noncoding sequences, white regions are coding sequences, and the shaded region is the intron (notdrawn perfectly to scale). The <br /> thin lines are adjacent genomic sequences. Restriction enzymes sites: Bg, BgllI; Hd. HindIII; RI, EcoRI; X, Xhol. DNA sequences between <br /> the EcoRI and BglII sites of CRYJ were replaced with the 1.1-kb HindIII fragment containing the yeast URA3 gene. Synthetic Xhol linkers <br /> were attached to the filled-in EcoRI and Bglll sites in CRY1 and to the filled-in Hind111 sites at the endsof the LIRA3 fragment. and the DNAs <br /> were ligated together. The 2.8-kb Hind111 cryI-A1::CJRA3 fragment from this construct was transformed into veast diploid JLY 12, selecting <br /> for Ura&#226;&#8364;&#8482;, such that one C R Y l allele was replaced with cryl-AJ::URA3. <br />  <br /> grown in liquid medium, as described in DESHMUKHet al.                    carrying a cryl resistance allele (MEADE, RILEY and <br /> (1 993).                                                                  MANNEY1977; CLARKE and CARBON 1983). The one <br />    Isolation and sequencing   of cry2 and cry2 mutant alleles <br /> conferring cryptopleurine resistance:The cryl mutant al-                  exceptional resistant transformant showed aberrant <br /> lele was cloned as described in LARKINand WOOLFORD                        segregation for all markers tested when sporulated, <br /> (1983). To isolate cry2 mutants resistant to cryptopleurine,              and was not examined further. Nineteen tetrads were <br /> 1.4 X lo9 cells of strain JTY82 (genotype cryl-A2::TRPI                   dissected from one of the CryS transformants, and <br /> C R M ) were plated on YEPD, and colonies were replica- <br /> plated to YEPD containing 10 PM cryptopleurine.Four CryR                  four viable spores were obtained from each. The Ura <br /> colonies were identified. Asymmetric polymerase chain re-                 phenotype segregated 2+:2-. Ura+ cosegregatedwith <br /> action (PCR) was employed to obtain single-stranded cry2                  cryptopleurine sensitivity, was tightly linked to MATa <br /> DNA from each strain for sequencing of cry2 alleles (GYL-                 and always segregated in repulsion to cryptopleurine <br /> LENSTEN 1989; ALLARD, ELLSWORTH and HONEYCUTT <br />                                                                           resistance (data not shown). Genomic Southern blots <br /> 1991).One rg of genomic DNA extracted from each mutant <br /> strain was used in the first PCR, together with 50 pmol of                demonstrated that the integration event produced a <br /> each of the two oligonucleotide       primers     5&#226;&#8364;&#732;-AACA-                new HindIII fragment of the expected size (2.8 kb) at <br /> ACTCCCAAT-3&#226;&#8364;&#8482; (CRY2-2) and 5&#226;&#8364;&#65533;AAGCCGTATTTCC                                 the CRYl locus that hybridized to both CRYI and <br /> ATGCG3&#226;&#8364;&#8482; (CRY2-5). Onetwentieth of the DNA produced                        URA3 probes andthatcosegregated with the Ura+ <br /> in this reaction was subjected to PCR using the same two <br /> oligonucleotides in a 50:l ratio (50 pmol CRY2-2 and 1                    phenotype in tetrads(LARKIN1985). We conclude <br /> pmol CRY2-5). Oligonucleotide CRY2-5 was subsequently                     that CRYl is not an essential gene. <br /> used as a primer for DNA sequencing reactions. The nu-                       Becauseyeast ribosomal protein genes are often <br /> cleotide sequence of each mutant allele was determined                    duplicated and essential (reviewed in WOOLFORD     and <br /> twice.                                                                    WARNER1991; WOOLFORD1991),the mostlikely <br />                                                                           explanation for the viability of haploids carrying the <br />                            RESULTS                                        cryl-Al::URA3 mutation is that there is a second gene <br />    Yeastcontaina second gene conferring a C$                              encoding functional rp59. Thispossibility is supported <br /> phenotype: Most yeast ribosomal proteins examined                         by the observation that CRYl genomic and cDNA <br /> to date are essential for viability (reviewed in WOOL-                    clones hybridize to yeast genomic DNA restriction <br /> FORD and WARNER 1991       ; WOOLFORD     199 1). To de-                  fragments in addition to those derived from theCRYl <br /> termine whether CRYI encodes an essential protein,                        locus [LARKIN andWOOLFORD         (1983) and Figure 2, <br /> we disrupted the CRYI gene by replacing sequences                         lanes 1 and 21 and that haploids containing the c r y l - <br /> between the EcoRI and BglII sites within CRYl with a                      Al::URA3 disruption allele express an RNAspecies <br /> 1. l-kb fragmentcontaining the yeast URA3 gene (Fig-                      homologous to CRYl probes that is identical in size to <br /> ure 1). T h e cryl-AI::URA3 allele lacks all of the CRYl                  the mature wild-type CRYl mRNA (Figure 3). <br /> coding sequence except the first two codons and the                          A second functional CRY gene was isolated from a <br /> final 29 codons. A HindIII fragment containing the                        library of yeast genomic DNA cloned into bacterio- <br /> cryl-AI::URA3 disruption was used to transform dip-                       phage X, using CRYI probes. Phage that hybridized to <br /> loid yeast strain JLY 12 to Ura+. JLY 12 is homozygous                    the CRYI-containing plasmid pJL32 were counter- <br /> for ura3-52 and heterozygous for cryl and CRYI and                        screened with plasmidpJL37, which contains the short <br /> therefore Ura- and CryS. T h e CRYI allele of JLY 12                      5&#226;&#8364;&#732; exon and the 5&#226;&#8364;&#8482; end of the intron of CRYI (LARKIN <br /> is linked to MATa, and the cryl allele is linked to                       and WOOLFORD1983). Because the yeastDNA in <br /> MATa. One hundred and seven of one hundred and                            pJL37 hybridizes only to the CRYI locus (LARKIN and <br /> eight stable Ura+ transformants examinedwere CryS,                        WOOLFORD1983), those phagethat hybridized to <br /> consistent with the behavior of monosomic diploids                        pJL32 but did not hybridize to pJL37 were likely to <br />                                                       Cryptopleurine Resistance in Yeast                                    723 <br />  <br />                                                                         bridized to C R Y l DNA (data not shown). <br />                                                                            Sequences within the 4.6-kb EcoRI fragment that <br />                                                                         hybridized to theC R Y l probe were localized by South- <br />                                                                         ern blot analysis to a 0.7-kb EcoRV-BglII fragment <br />           5.6 Kb -                               - CRY2                 andanadjacent         0.6-kb BglII fragment(data not <br />           4.6 Kb -                               - CRY2                 shown). The nucleotide sequences of these two frag- <br />                                                                         ments and the adjacent EcoRI-EcoRV fragment were <br />           3.5 Kb -                               - CRYl                 determined. This sequence contains an open reading <br />                                                                         frame of 414 nucleotides interrupted within the <br />                                                                         fourth codonby an intron bounded by consensus yeast <br />           2.2 Kb -                               - CRYl                 5&apos; and 3&apos; splice sites and containing the conserved <br />                                                                         branch point sequence TACTAAC (Figure 4). The <br />                                                                         ORF encodes a protein 96% identical to that of rp59 <br />                                                  - CRYl                 encoded by C R Y l (Figure 4). Therefore, we designate <br />            1.6 Kb- <br />                                                                         this gene CRY2. The C R Y l gene is interrupted within <br />              Probe: <br />                 CRY2CRYl                                                the third codon by a 307 nucleotide intervening se- <br />    FIGURE2.--GenomicDNA        fragments that hybridize to CRYI <br />                                                                         quencethat is spliced from the C R Y l pre-mRNA <br /> and CRY2 DNA.DNA was prepared from CryS yeast haploid                   (LARKIN,THOMPSON           and WOOLFORD1987). The <br /> A364A (genotype: CRYl CRY2). digested withEcoRI or HindIII,             CRY2 intron is 408 nucleotides long and is not ho- <br /> subjected to electrophoresis on I % agarose gels. transferred to        mologous to the C R Y l intron, exceptforthe         con- <br /> Nytran, andprobed with the &quot;P-labeled 2.2-kb CRYI Hind111               served intron boundaries and branchpoint sequences. <br /> fragment or 4.6-kb CRY2 EcoRI fragment. Lanes 1 and 3 contain 5 <br /> rg of DNA digested with EcoRI. and lanes 2 and 4 contain 5 rg of        Primer extension analysis indicated that the 5&apos; ends <br /> DNA digested with HindIII. DNA in lanes 1 and 2 was hybridized          of the CRY2 transcripts map to nucleotides -24, -27, <br /> to a CRY1 probe and that in lanes 3 and 4 to a CRY2 probe. The          and -33 with respect to theCRY2 translation initiation <br /> sizesin kilobases (kb) and origins of each genomic fragment are         codon. <br /> indicated. <br />                                                                             We confirmed that a functional rp59 is expressed <br />                                             Tetrad <br />                                                                         from CRY2 by demonstrating that this gene is tran- <br />              CRY1 Genotype $        I   -    -   +   +1                 scribed in vivo and that it can confer aCrySphenotype <br />                                                                         when transformedintoaCryRstrain.              The 3.0-kb <br />                                                                         EcoRI-BamHI fragment containing CRY2 and the 2.2- <br />                                                                         kb HindIII fragment containing C R Y l were hybrid- <br />                                                                         ized to a blot of RNA isolated from the temperature- <br />                                                                         sensitive splicing mutant RL92(genotype: CRYl CRY2 <br />                                                                         p r p 2 ) grown at the permissive and restrictive temper- <br />                                                                         atures. At the restrictive temperature, unspliced pre- <br />                       11-10- .I_                                        mRNA accumulates in the prp2 mutant (ROSRASH           et <br />                                         7                               al. 1981).Notethatboththe          C R Y l andthe CRY2 <br />                                                                         probes hybridized tothemRNA              expressed from <br />  <br />                       CRYI- <br />                               I                                         either gene, although not with equal efficiency. The <br />                                                                         approximately <br />                                                                         (LARKIN, <br />                                                                                        960-nucleotide <br />                                                                                      THOMPSON <br />                                                                                                               C R Y l pre-mRNA <br />                                                                                                   and WOOLFORD1987) and an <br />                                                                         approximately 1 100-nucleotide transcript comple- <br />                                                                         mentary to CRY2 accumulate in cells shifted from 23&quot; <br />   FIGURE%-RNA complementary to C R Y l DNAis present in                 to 37&quot; (Figure 5), consistent with the observed block <br /> haploids containing a cryl-AI::URA3 null allele. Total RNAwas           in step 1 of splicing in prp2 temperature-sensitive <br /> extracted from the parental wild-type diploid JLY 12 (lane I), a <br />                                                                         mutants (LIN,LUSTIGand ABELSON         1987). From these <br /> diploid heterozygous for the cryl-AI::URA3 disruption (lane 2). and <br /> from the haploid spores of one tetrad derived from the heterozy-        data w e inferthat CRY2 is transcribed as an 1100 <br /> gous diploid (lanes 3-6). The RNA was subjected to electrophoresis      nucleotide pre-mRNA from which a 408 nucleotide <br /> on a 1.2% agarose-formaldehyde gel, transferred to Nytran, and           intron is spliced to produce a mature CRY2 mRNA. <br /> probed with a &quot;P-labeled 2.2-kb C R Y l HindIII fragment. The blot          T o directly demonstratethatthe         CRY2 locus is <br /> was simultaneously probed with S2P-labeledplasmid 11-10. which <br /> encodes an abundant yeast transcript, to ensure that equal masses <br />                                                                         expressed, we tested whether introduction of extra <br /> of RNA were loaded in each lane.                                        copies of the CRY2 gene into a CryR strain conferred <br />                                                                         a CrySphenotype. The CRY2 gene cloned in the yeast <br /> contain DNA from the duplicated CRY sequence and                         high-copy number plasmid YEpl3 or YEp24 was <br /> not from C R Y l itself. All of the phage thus identified                transformed into the CryR strain   JLYS (genotype: cry1 <br /> contained a common 4.6-kb EcoRI fragment that hy-                        CRY2). T h e cells transformed with the high-copy <br />  724                                                            A. G. Paulovich et al. <br />   -580               -560               -540                -520               -500               -480 <br />  CCTTCGTCATCTTGAAACATGGTCGTAGATGTGTAAGGATATTCTGTCGA~ATTGCCGATCTGATAAAGTTTGAAACTTGTTCTAGTTGAAACAAAGCCGGTGATGTTACTTGCTGCT <br />  <br />   -460                -440               -420                -400               -380                -360 <br />  AAGTCTAAGCGACCCTTAATGGTCCCTCTCCTGGCCGTAGCACTGGCGACATAAATAAATGCACTGAGTAAAAGAACAATCTGTGAAAACATCGTCACACTGCTGTTTCACTTGTTGATC <br />                                                                                                                                ............. <br />   -340                -320                -300                 -280                -260                -240 <br />  AAATTTACTTTGATGATATTTAATGGCTTGACAAATTCGTTCATCTTAATTTCATACATAATTTCATCCTGAAATCCTCGATCATTTTCGAGACTGTTTTCGAGCTTGATTTTATATGTT <br />  <br />   -220               -200               -180               -160               -140               -120 <br />  GGTGTAAACACATCATTGTACATCCTTGCACCACACTGAAAAGAATATGGAAAAAATTTCAAACTTTTTAATGGGTGATAGACTTGAAAAAC~CGGAAT~AGATATAGTCGCAAG <br />                        ............. <br />   -100                -80                -60                -4 0               -20                  1 <br />  AACTTGCCTTCTACGAGTATAAAAGCGGTACTATTCCTTCTGTATGCATCTTCATCGCAAGAGAGAAAAAGACTGAAATCAACAACTCCCAATAACAATTAAGAATGGCTAACG~ <br />                                                                                            *           *       *                                        M   A   N <br />                                                                                                                                                             M   S <br />    20                 40                 60                  80                100                120 <br />  GAAAGGGGTGATATCCTGTTT~CCATTTAGTTATTCTTTTTTTCAGATGGAGACTGTGATCGGGTAGCGCATGATGTTATTATTGGAATTTTATTTTTTTTGCAACTTTGAAGCACT <br />  <br />   140                                                          200                 220                 240 <br />  CCTAACTATATTGCAATAGTGTTATWLGATGCTACTTATGTTTGTAAGGCCATGAAATTCTGTAATACGGAGTTAAAATTTCTGCAATAAAGCACTTCAATGCTATTGTGTTATAGAGGC <br />  <br />   2                                      300                 320                 340                 360 <br />  TCTGCGCTCACCCTTAATACTGTATCATCTGTAATAAAGAATGTTTATCTTAAAAGGATTGTAACTAAATGGTTGTTAGATAGTGACTCATTCGTTTATTTTTTAATAATCGAAAAAGCA <br />  <br />   380              400               420            60 440              4                 480 <br />  GAAGTTCTGTT~CTATACCTCTTATCATGT~CCTTGTTCAAGCTCGCGATAACTCTCAAGTCTTTGGTGTTGCCAGAATCTACGCCTCCTTTAACGATACCTTTGTAC <br />                                                  D     L   V   Q   A   R   D   N   S   Q       V   F       G       V   A   R    I   Y   A   S   F   N   D   T   F   V   H <br />                                                  N V <br />   500                                                          560                 580                 600 <br />  ATGTTACCGATTTATCTGGTAAGGAAACTATCGCCAGAGTTACTGGTGGTATGAAAGTCAAGGCCGACAGAGACGAATCATCTCCATACGCTGCTATGTTGGCTGCCCAAGATGTTGCCG <br />    V T D L S G K E T I A R V T G G M K V K A D R D E S S P Y A A M L A A Q D V A A <br />  <br />  620                 640                 660                  680                 700                 720 <br /> CTAAGTGTAAGGAAGTCGGTATCACTGCCGTTCACGTTAAGATCAGAGCTACTGGTGGTACCAGAACCAAGACCCCAGGACCAGGTGGTCAAGCCGCTTTGAGAGCTTTGGCCAGATCTG <br />   K C K E V G I T A V H V K I R A T G G T R T K T P G P G G Q A A L R A L A R S G <br />         R <br />  740                 7 60                780                  800                 820                 840 <br /> GTTTGAGGATTGGCCGTATCGAAGACGTTACTCCAGTCCCATCAGACTCCACCAGAAAGAAG~TGGTAGAAGAGGTAGAAGATTATGATTTCTTTTTTTTTTACTCTATTAACGCATGG <br />   L R I G R I E D V T P V P S D S T R K K G G R R G R R L &#226;&#8364;&#8482; <br />                                          C <br />  860                 880                 900                  920                 940                 960 <br /> AAATACGGCTTTACCTTCTCACATAAATATAGTTTGTTTAATTAAT~GCAATATAAACATTTGCATATTCGACTTCGTGCATGCAG~TGCGGCAGAT~TTCATTTCTGTCTAAT <br />  <br />  980                1000 <br /> CTTTTGACAGTTTTAAGTACAATATA <br />      FIGURE4.-DNA sequence of the CRY2 gene and inferred amino acid sequence of rp59 encoded by CRY2. The nucleotide sequence of <br /> the coding strand of CRY2 is shown. The open reading frame of CRY2 is translated into amino acids below each line of DNA sequence. The <br /> five amino acids encoded by CRY1 that differ from those encoded by CRY2 are indicated below the CRY2 amino acid sequence. The <br /> nucleotides of CRY2 are numbered with respect to thetranslation initiation codon, designated + 1. The two U A S R ~sequences <br />                                                                                                                        C        at nucleotides <br /> -377 to -364 and -206 to -193 are underlined (. ... .). Transcription initiation sites at nucleotides -24, -27, and -33 are underlined. <br /> The nucleotides within the CRY2 intron are italicized. The conserved 5&#226;&#8364;&#8482; splice site, branchpoint, and 3&#226;&#8364;&#8482; splice site sequences are underlined <br /> (-). <br />  <br />  <br /> number CRY2 plasmids are CryS when replicated to                               adjacent to ribosomal protein genes RPS24A and <br /> selectivemedium containing 5 PM cryptopleurine                                 RPL46 (LEERet al. 1985). This is the first example of <br /> (data not shown).Cells transformed with the high                               three tightlylinked    eukaryotic ribosomal protein <br /> copy YEpl3 or YEp24 vectors remain CryR.We con-                                genes (Figure 6). <br /> clude that CRY2 encodes a functional ribosomal pro-                              There are onlytwo CRY genes: The 4.6-kb EcoRI <br /> tein 59 capable of expressing the CrySphenotype.                               restriction fragment containing CRY2 and the 2.2-kb <br />    Mapping CRY2 The CRY2 gene was localized to                                 HindIII fragment containing CRYl were each hybrid- <br /> the left arm of chromosome X, centromere-proximal                              ized to blots ofyeastgenomic DNA digested with <br /> to cdc6,by hybridizationof the 1.5-kb EcoRI-SphI                               EcoRI or HindIII. Only fragments from the CRYl or <br /> restriction fragment that contains CRY2 to a blot of                           CRY2 loci were detected. The CRYl probe hybridized <br /> electrophoretically resolved yeast chromosomes and                             to 1.6-kb and 3.5-kb EcoRI fragments and a 2.2-kb <br /> then to a set of cloned <br />                       overlapping genomic restriction                          HindIIIfragment, allofwhich       include CRYI, and <br /> fragments constructed by L. RILESand M. OLSON                                  hybridized weakly to the 4.6-kb EcoRI fragment and <br /> (BENDERand PRINGLE1991). Thus the CRYl and                                     a 5.6-kb HindIII fragment containing CRY2 (Figure <br /> CRY2 genes are on different chromosomes. Compar-                               2). The cloned4.6-kb EcoRI CRY2 fragment gave <br /> ison of the nucleotide sequences flanking CRY2 with                            strong hybridization signals withthe 4.6-kb EcoRI and <br /> sequence data bases revealed that CRY2 is immediately                          5.6-kb HindIII CRY2 genomic fragments and weak <br />                                                Resistance <br />                                                  Cryptopleurine              in Yeast                                      725 <br />                       23&apos;37&apos;     37&apos; <br />                                   23&apos;                                  AI::LEU2 disruption allele (data not shown). <br />                                                                           cryl-A2::TRPI haploid MPY 1402 and cry2- <br />                                                                        Al::LEU2 haploid MPY 1403, derived from the gene <br />                                                                        disruption experiments described above, were mated <br />                P-                            -P <br />                                                                        to form diploid strain MPY 1404 that is doubly het- <br />               M-                             &quot;                         erozygous for cryl-A2::TRPI and cry2-Al::LEU2 null <br />                                              - RPS24A                  alleles. The doubly heterozygous diploid was sporu- <br />                                                                        lated and subjected to tetrad analysis. No Trp+Leu+ <br />                                                                        cells (genotype: cryl-A2::TRPlcry2-AI::LEU2) were <br />                                                                        recovered in cases predicted from the genotypes of <br />                        13 24 <br />                                                                        the spores within each tetrad. Of the 21 tetrads dis- <br />             Probe:    CRYl         CRY2 <br />                                                                        sected, 16 were tetratype, 3 non-parental ditype, and <br />   FIGURE5.-The C R Y 2 gene is transcribed into a primary tran-        2 parental ditype with respect to the cryI-A2::TRPI <br /> script containing an intron larger than that in the CRYZ gene. Total <br /> RNA was prepared from 20 ml of exponentially growing prp2 <br />                                                                        and cry2-Al::LEU2 mutations. Thisratio of tetrad <br /> mutant cells grown in YEPD at 23&quot; (lanes 1 and 3), or grown at         classes is close to the 4: 1: 1 ratio expected for the <br /> 23&quot; and shifted to the nonpermissive temperature of 37&quot; for two        segregation of two unlinked genes, as predicted for <br /> hr to accumulate unspliced intron-containing pre-mRNAs (lanes 2        CRYl and CRY2 on chromosomes 111 and X , respec- <br /> and 4). The RNA was subjected to electrophoresis on 1.2%agarose-       tively. Southern blot hybridization demonstrated that <br /> formaldehyde gels, transferred to Nytran, and hybridized with &quot;P- <br /> labeled 2.2-kb CRYZ HindIII fragment (lanes I and 2) or 3.0-kb <br />                                                                        the parental diploid and haploid spore clone DNAs <br /> CRY2 EcoRI-BamHI fragment (lanes 3 and 4). Because the CRYZ            contained the expected wild-type or mutant CRY al- <br /> and CRY2 DNAs each cross-hybridize to mRNA expressed from              leles (data not shown). Identical results were obtained <br /> the other respective CRY gene, the ratios of pre-mRNA to mRNA          upon sporulation of strain JTY78, also doubly heter- <br /> apparent from these blots are not accurate and are underestimates.     ozygous for cryl-A2::TRPI and cry2-A1::LEU2. This <br /> P, intron-containing pre-mRNA; M, spliced mRNA; RPS24A <br /> RPS24A transcript that hybridizes to the 3.0-kb EcoRI-BamHI frag-      diploid was constructed by sequential transformation <br /> ment that also contains CRY2.                                          with linear DNAs containingthe two null alleles. <br />                                                                        When JTY78 was transformed with plasmid pJT30 <br /> signals with the 3.5-kb EcoRI and 2.2-kb HindIII                       containing URA3 and cry&apos;l, sporulated, andsubjected <br /> fragments containing CRYl (Figure 2).                                  to tetrad dissection, all of the Trp+Leu+Ura+meiotic <br />    Ribosomal protein 59 encoded by the CRYl or                         products obtained were Cry&apos;. These cryl-A2::TRPI <br /> CRY2 genes is an essential yeast protein: A cryl-                      cry2-A2::LEU2 pJT30 cells were unable to grow on <br /> A2::TRPI null allele was constructed by inserting the                  medium containing 5-fluoro-oroticacid, which selects <br /> T R P l gene in place of 106 of the 137 codonsof C R Y l .              for loss of plasmids bearing URA3. Taken together, <br /> A linear 3.1-kb HindIIIfragmentcontaining cryll-                       these results indicate thatrp59encoded         by either <br /> A2::TRPI was transformed into yeast diploid                            CRYl or CRY2 is essential for mitotic growth of yeast. <br /> MMY 140 1 to replace of     onethe wild-type CRYI alleles              The cryl-A2::TRPI CRY2 or CRYl cry2-Al::LEUZ hap- <br /> with cryll-A2::TRPI. Trp+ transformants were se-                        loids recovered upon meiosis of either of the doubly <br /> lected, sporulated, and subjected to tetraddissection.                  heterozygous diploids MPY 1404 or JTY78 grew at <br /> Four viable CryS spores were recovered from all te-                     identical rates to wild-type cells, suggesting that nei- <br /> trads, all markerssegregated as expected andthe                         thermutant has a significant functional deficit of <br /> T R P l marker was tightly linked to the mating type                    rp59. The lethality of cryl-A2::TRPI cry2-AI::LEU2 <br /> locus, MATa or MATa. Southern blot hybridization                        doublemutants is consistent with there being only <br /> demonstratedthatthe        TRPI markercosegregated                      two functional CRY genes in yeast.We have also <br /> with the cryl-A2::TRPI restriction fragments (data not                  constructeda     cryl-A2::TRPIcry2-AI::LEU2GALI: <br /> shown) and confirmed the CRYl gene replacement.                         CRYl strain conditional for synthesis of rp59 to dem- <br />    A cry2-AI::LEU2 null allele was constructed by in-                   onstrate that upon cessation of synthesis of rp59, 40s <br /> serting LEU2 between the EcoRV and Asp71 8 sites                        ribosomal subunits are no longer assembled and the <br /> within CRY2, such thatcodons 4-89 were deleted                          cells stop dividing (MORITZ et al. 1990). <br /> (Figure 7). An AatII-BamHI fragment containing cry2-                       cryl-AP::TRPl mutants are modestly deficient in <br /> AI::LEU2 was used to transform the diploid strain                       40s ribosomal subunits, but cry2-Al::LEU2 mutants <br /> MMY 1401. Leu+ transformants were selected, spor-                       contain wild-type levels of 40s ribosomal subunits: <br /> ulated, and subjected to tetrad analysis and Southern                   T o examine the effect of deleting CRYl or CRY2 on <br /> blot hybridization. All tetrads yielded four viable CryS                assembly of ribosomes, we examined the translational <br /> spores, and all markerssegregated 2:2. T h e Leu+                       apparatus in cryl-A2::TRPI    CRY2, <br />                                                                                                           CRYl             cry2- <br /> phenotype cosegregatedwith replacement of the wild-                     Al::LEU2, and wild-type haploid strains JTY82, <br /> type 4.6-kb EcoRI CRY2 fragment by the predicted                       JTY8 1, and JTY83,respectively. Free ribosomal sub- <br />  2.25-kb and 5.25-kb EcoRI fragments for the CRY2-                      units, 80s monoribosomes, and polyribosomes were <br />  726                                                        A. G . Paulovich et al. <br />                                             CRY2 RPS24A             RPL46 <br />  <br />  <br />  <br />  <br />                        ..........,-11 <br />                                                     *. . <br />                                                        :..#* <br />                                                     cdc6 <br />                                                                &apos; <br />  <br />                                                                             ura2 tif2 <br />                                                                                            11-(       -..-....-   x <br />   FIGURE6.-The tightly-linked cluster of ribosomal protein genes CRY2, RPS24A and RPL46 is on the left arm of chromosome X , <br /> centromere proximal to cdc6. The parentheses indicate that the orientation of the EcoR1 fragment containing this gene cluster is unknown. <br />  <br />                       Aa R1                    RV                  Nr   A                                         0a <br />  <br />  <br />  <br />  <br />                                                                L EU2 <br />                           X                                                                              S <br />   FIGURE7.-Replacement of the CRY2 gene with the cryZAI::LEU2 deletion-insertion allele. The disruption is depicted as in Figure 2. <br /> Restriction enzyme cleavage sites: As, Asp718; Aa, AntII; Ba, BamH1; Bg, BgllI; RI, EcoRI; RV, EcoRV; Nr, NruI; S,SalI; X, Xhol. A deletion <br /> of the CRY2 gene was constructed by removing the sequences between the EcoRV site in the intron of CRY2 and the Asp718 site within <br /> codon 89 of CRY2. The 2.9-kb XhoISalI fragment containing the yeast LEU2 gene was inserted between the EcoRV and Asp718 sites in <br /> CRY2 as shown. An AatII-BamHI fragment containing the cry2-AI::LEU2 construct was transformed into diploid &quot;1401        selecting for Leu+. <br />  <br /> separated by sucrose gradient sedimentation of ex-                                 The relative expression of CRYl and CRY2 was also <br /> tracts prepared fromthese three strains. We observed                            assayed using fusions of the E. coli lacZ gene at codons <br /> a modest decrease in the relative amount of free 40s                            107 and 108 of CRYl and CRY2, respectively. Single <br /> ribosomal subunits compared to free 60s ribosomal                               copiesof each gene fusion were individually inte- <br /> subunits in the cryl-A2::TRPlCRY2 mutant strain                                 grated into the genome of yeast strain JTY83 at the <br /> compared to wild-typecells (data not shown). The                                ura3-167 locus, confirmed by genomic Southern hy- <br /> ratio of total 40s ribosomal subunits to total 60s                              bridization. Sixty units of &amp;galactosidaseper mg pro- <br /> ribosomal subunits was measured by subjecting cell                              tein were expressed in strains containing CRYl-lacZ <br /> extracts to centrifugation on high-salt sucrose gra-                            and 7 units per mg protein in strains bearing CRY2- <br /> dients that dissociate polyribosomes and monoribo-                              lacZ, consistent with the observed relative levelsof <br /> somes to subunits. The ratio of 40s to 60s ribosomal                            CRYl and CRY2 mRNAs. <br /> subunits in the cryl-A2::TRPl CRY2 mutant is 80% of                                Isolation and sequencing of my1 and c y 2 alleles <br /> wild type (Figure 8). In contrast, both of these gra-                           conferring a CfyR phenotype: Most but not all mu- <br /> dient assays revealed that the CRYlcry2-Al::LEU2                                tations that result in a CryR phenotype in yeast map <br /> mutant has no detectable deficit of 40s ribosomal                               to the cryl locus, tightly linked to the mating type <br /> subunits (Figure 8 and datanot shown).                                          locus MAT on chromosome ZZZ. Mutations that cause <br />    CRY1 isexpressed at higherlevelsthan C R Y 2                                 cryptopleurine resistance but are not linked to MAT <br /> The deficiency of 40s ribosomal subunits in the cryl-                           have been isolated, but the genes defined by these <br /> A2::TRPlCRY2 mutant but not in the CRYl cry2-                                   mutations have not been mapped or characterized <br /> Al::LEU2 mutant suggests eitherthat CRYl is ex-                                 further (A. K. HOPPER,personal communication). Be- <br /> pressed more thanCRY2, or that rp59 expressed from                              cause the rp59 products of CRYl and CRY2 are nearly <br /> CRYl is more functional than rp59 expressed from                                identical, we thoughtthat it might be possible to <br /> CRY2. The relative levels of CRYl and CRY2 mRNAs                                obtain CryR mutants resulting from mutations at <br /> in wild-type cells were measured by an S1 protection                            CRY2. However, such mutations might be difficult to <br /> assay using a DNA fragment complementary to C R Y l .                           isolate from a CRYl CRY2 strain. All cryl mutations <br /> As shownin Figure 9, CRYl mRNA is present in                                    isolated to date are recessive. The higher expression <br /> 5-10-fold excess of CRY2 mRNA. Because the CRYl                                 of CRYl compared to CRY2 might mask the phenotype <br /> probe hybridizes less well to CRY2 mRNA than to                                 of a cry2 mutation. Therefore, to determinewhether <br /> CRYl mRNA, this assay most likely provides a slight                             CryR mutants could result from mutations in CRY2, <br /> underestimate of CRY2 mRNA levels. Similar results                              we selected spontaneous CryRmutants using a haploid <br /> were obtained by hybridization to RNAblotswith                                  strain containing the cryl-A2::TRPl null allele. CryR <br /> gene-specific oligonucleotide probes complementary                              mutants were selected by plating haploid strain JTY82 <br /> to thedivergent sequences within the 3&apos; untranslated                            (cryl-A2::TRPlCRY2) on YEPD containing 10 pM <br /> regions of CRYl and CRY2 (data not shown).                                      cryptopleurine. CRY2 DNA was recovered from each <br />  <br />  <br />                                                                          FIGURE9.-CRYI mRNA is present in higher levels than CRY2 <br />                                                                       mRNA in wild-type cells. Levels of CRY1 and CRY2 m R N A were <br />        E                                                              measured by an S1 nuclease protection assay. RNA was extracted <br />        g     0.5                                                      from wild-type strain JTY83, hybridized to the &quot;P end-labeled <br />        cu                                                             Hindlll-BglI1 CRYl fragment and XbaI-Bglll RPLI fragment. di- <br />        a <br />                                                                       gested with nuclease S I , subjected to electrophoresis on a denatur- <br />                                                                       ing polyacrylamide gel, and exposed to X-ray film. The positions <br />                                                                       of protected DNAs corresponding to CRYl,CRY2 and R P L f <br />                                                                       mRNAs are indicated. A shorter CRYI DNA fragment is protected <br />              0.0 <br />                                                                       by hybridization to CRY2 mRNA (-1 50 nucleotides) compared to <br />                                                                       that protected by CRYI mRNA (3 16 nucleotides). because the CRY2 <br />                                                                       nucleotide sequence has diverged from CRY1 at several positions <br />                                                                       between the Bglll site and the intron. <br />         C                 CRYl     CfY2-Al::LEU2 <br />                                  60s <br />                                                                        mined. Three of the cry2 DNAs contain a T T A to <br />                                                                        TGA <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> in the last codon of CRY2. <br />                                                                        The fourth cry2 mutation, L138S, results in the re- <br />                                                                        placement of the last amino acid of rp59, Leu-138, <br />                                                                        with Ser. <br />                                                                           Previously, we cloned a cryl resistance allele from <br />                                                                        the CryR strainDPB745cry by the integration-excision <br />                                                                        method (LARKIN and      WOOLFORD     1983). CryS strains <br />                                                                        (genotype: CRYI CRY2) transformed with this DNA <br />                                                                        on a high copy plasmid became CryR, i.e., were able <br />                                                                        to grow on 10 ~ L cryptopleurine. <br />                                                                                            M                This cryl allele was <br />                                                                        sequenced and found to contain an L137S mutation <br />    FIGURE8.-The amount of 40s ribosomal subunits is decreased          in its last codon, identical to the cry2 missense muta- <br /> in a cryl-A2::TRP1 mutant but not in the cry2-AI::LEU2 mutant.         tion described above. <br /> CRY1CRY2 (wild type), cryl-A2::TRP1CRY2, and CRYl cry2- <br /> AI::LEU2 cells (200 ml) were grown in YEPD medium to mid-log <br /> phase, harvested, and lysate from each culture was subjected to <br />                                                                                                 DISCUSSION <br /> centrifugation on high salt sucrose gradients. The gradients were <br /> collected and OD260 monitored using an lSC0 gradient fractiona-           In this report we demonstrate that the yeast ribo- <br /> tor. The positions of 40s and 60s ribosomal subunits are indicated     somal protein rp59 is encoded not only by the CRYI <br /> above the optical density profiles.                                    gene but also by a second, unlinked gene CRY2. We <br />                                                                        cloned CRY2 based on its homology with the previ- <br /> of four independently isolated CryR strains,either by                  ously cloned CRYl gene and demonstrated that func- <br /> the integration-excision method using CRY2 cloned in                   tional rp59 is expressed from CRY2. The CRY2 locus <br /> Y Ip5 (ROEDER   and FINK 1980),or by amplification by                  encodes a protein 96% identical to that encoded by <br /> the polymerase chain reaction. The nucleotide se-                      CRYI. T w o differences between the polypeptides en- <br /> quence of each complete cry2 mutant allele was deter-                  coded by CRYI and CRY2 are conservative substitu- <br />  728                                           A. G. Paulovich et al. <br />  <br /> tions: arginine codon 72 in CRY1 is a lysine codon in      ferences in expression of CRYl and CRY2 might also <br /> CRY2, and cysteine codon 123 in CRYl is aserine            result from post-transcriptional mechanisms. <br /> codon in CRY2 (Figure 4). T h e most drastic differ-          Although CRYl contributes at least 80% of rp59 in <br /> ences betweenthe rp59polypeptides encodedby these          wild-type cells, we observe that cryl A-1 null mutants <br /> two locilie in the first three or fouramino acids          still contain 80% of wild type levels of 40s ribosomal <br /> at theiraminotermini.       T h e inferredaminoter-        subunits and grow at wild-type rates. This contrasts <br /> minal sequence of rp59     encoded        by CRYl is       with the situation previously observed for duplicated <br /> MSNVVQAR. . ., but is MANDLVQAR. . .for rp59               essential yeast ribosomal protein genes. Disruptions of <br /> encoded by CRY2 (Figure 4). Although the overall           one or the other gene of a duplicate pair typically <br /> primary structure of rp59 has been highly conserved        result in a proportional decrease in the amount of <br /> throughout evolution, some obvious divergence has          each gene product and of the ribosomal subunit of <br /> occurred at the amino termini    of rp59 homologues in     which that proteinis a component(reviewed in WOOL- <br /> other organisms. For instance, the rp59 proteins en-       FORD and WARNER1991; WOOLFORD            1991). We are <br /> coded by the yeast CRYl or CRY2 genes and the rpSl4        presently examining the coordinate regulation of the <br /> protein encoded by the mammalian RPS14 gene are            CRYlICRY2 genepair. Current evidence indicates <br /> 80% identical, but the mammalian protein contains          that expression of CRY2 increases upon disruption of <br /> an additional 14 amino acids at its amino terminus         CRYl (Z. LI, A. G. PAULOVICH,     J. R. THOMPSON,    F. <br /> that are notpresent in yeast rp59 (RHOADS,    DIXITand     HE, A. JACOBSON and J. L. WOOLFORD,       manuscript in <br /> RHOUFA1986; LARKIN,THOMPSON             and WOOLFORD       preparation). Further studyof this duplicate genepair <br /> 1987).                                                     should provide insight into the cellular mechanisms <br />    Consistent with the high degree of evolutionary         coordinating the expression of ribosomal proteins. <br /> conservation exhibited by the rp59 family, we dem-            The existence of a second yeast CRY gene encoding <br /> onstrated that rp59 is essential for viability of yeast    rp59 suggested that mutations conferring a CryR phe- <br /> and that it is required for the stable accumulation of     notype could be obtained not only in the CRYl gene <br /> 40s ribosomal subunits (MORITZet al. 1990 and this         but also in CRY2. We confirmed this expectation by <br /> work). Previously we used a cryl-A2::TRPl cry2-            isolating CryRmutants using a crylACRY2 strain. <br /> Al::LEU2 GAL2::CRYI strain conditional forsynthesis        Sequencing of cry1 and cry2 mutant alleles revealed <br /> of rp59todemonstratethat,upontermination              of   that changing the last amino acid of rp59 fromleucine <br /> synthesis of rp59, other componentsof the 40s ribo-        to serine, or eliminating this carboxy-terminal amino <br /> somal subunit continue to be synthesized at wild-type      acid by nonsense mutation causes a CryR phenotype. <br /> rates but d6 not assemble into a stable 40s subunit        Similarly, mutations in codons at the 3&#226;&#8364;&#8482; end of the <br /> and are degraded (MORITZet al. 1990). Under these          Chinese hamster ovary gene encoding the rp59hom- <br /> conditions, componentsof 60s ribosomal subunits are        ologue rpS14 result in emetine resistance for these <br /> synthesized and assemble into functional subunits. In      cultured mammalian cells (CHANGand WASMUTH <br /> this paper, we determined that disruption of either        1983; MADJARet al. 1983; RHOADS      and ROUFA1985); <br /> CRYl or CRY2 does not measurably affect cellular           a single Arg-149 to Cys-149 mutation (EmR-2)causes <br /> growth rates. Cells containing a cryl-A2::TRPI null        resistance to low levels of emetine, whereas a double <br /> allele contain a modest deficit of 40s subunits, while     Arg-149 to Cys-149 and Arg-150 to His-150 mutation <br /> those containing a cry2-AI::LEU2 null allele are not       (EmR2-2)confers resistance to high levels of emetine. <br /> noticeably deficient in 40s subunits.                      Interestingly, 40s ribosomal subunits in the CHO <br />    We determined that CRYl is expressed at signifi-        EmR-2-2double mutant are unstable, consistent with <br /> cantly higher levels than CRY2 in wild-type haploid        our observations thatrp59 is necessary for stable <br /> yeast. We previously identified by mutational analysis     accumulation of 40s subunits (MORITZ et ul. 1990; <br /> two CRYl promoter elements: UASR~G 257-nucleo- <br />                                             at             this study). <br /> tide and aT-richregion(LARKIN, THOMPSON             and       Previously DOLZ,VAZQUEZ and J I M ~ N E Z (1982) <br /> WOOLFORD     1987). Both of these sequencesare present     demonstrated the existence of a high affinity binding <br /> upstream of most yeast r p genes and, when tested,         site for cryptopleurine on the 40s ribosomal subunit. <br /> have been shown to be necessary to transcription of        Binding of cryptopleurine to ribosomes isolated from <br /> rp genes (reviewed in WOOLFORD        1991). One might     CryR yeast was reduced compared to that for       CryS <br /> expect that differences in these conserved promoters       ribosomes, but not enough to account for the CryR <br /> could resultin different levels oftranscription of CRYl    phenotype. Therefore, DOLZet al. suggested that cry1 <br /> compared to CRY2. There are two putative UASRPG            mutations would not only affect binding of crypto- <br /> sequences upstream of CRY2 at nts -377 and -206,           pleurine to ribosomes but also result in a conforma- <br /> followed by a T-rich sequence. It has not been tested      tional change in ribosomes that impaired translational <br /> whether these or other sequences upstream of CRY2          inhibition by cryptopleurine bound to the ribosome. <br /> function as transcriptional promoters. Of course, dif-     Our findings, as well as those of MEADE et al., are <br />  <br />  <br /> consistent with this hypothesis and explain two previ-                 <br />                                                                         <br /> ously confusing observations: (1) T h e original obser-                   <br /> vations that the CryR phenotype is completely reces-                      <br /> sivein cryl/CRYl diploids and that one can obtain                          <br /> recessive CryR haploid mutants seemed incompatible             <br /> with the existence of duplicate, functional CRY genes. <br /> However, it is now clear that the CryR phenotype of              <br /> cryl CRY2 haploids results from cryl being more                       <br /> highly expressed than CRY2. Binding of cryptopleu-             <br /> rine <br />    to      wild-type sensitive ribosomes effectively                    <br /> freezes polyribosomes containing them (DOLZ,Viz-                  <br /> QUEZ and J I M ~ N E Z 1977). Therefore the extent    of <br />                                                                      <br /> sensitivity to cryptopleurine is a function of the ratio                  <br /> of sensitive to resistant ribosomes. Our measurements                       <br /> of the relative expression of CRYl and CRY2 suggest                         <br /> that a cryl CRY2 strain contains 80%-90% CryR ri-               <br /> bosomes, sufficient to sustain efficient translation in <br />                                                                             <br /> the presence of cryptopleurine, while a CRYl cry2                        <br /> straincontains only 10-20% CryR ribosomes and                                 <br /> therefore is CryS. A cryl/CRYl diploid is predicted to       <br /> contain mostly CryS ribosomes and is CryS. (2) T h e            <br />                                                             <br /> existence oftwo CRY genes, CRY1 on chromosomeI11 <br /> and CRY2 on chromosome X , explains why diploid                    <br /> yeast monosomic for chromosome I11 carrying cryl                  <br /> are phenotypically CryS (MEADE, RILEYand MANNEY                         <br /> 1977; CLARK and CARBON        1983). We suggest that the            <br /> unequal expression ofthe CRYl and CRY2 genes                         <br /> explains why most of the previously isolated mutations                    <br /> conferring resistance tocryptopleurinemaptothe                    <br /> CRYl locus.                                                            <br /> </body></html>