<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 20 No. 14 pp. 3695&#194;&#177;3704, 2001 <br />  <br />  <br /> A nuclear AAA-type ATPase (Rix7p) required <br /> for biogenesis nuclear export of <br /> 60S ribosomal subunits <br />  <br />  <br /> Olivier Gadal, Daniela Strau&#195;&#187;,                                       eukaryotic ribosomal subunits (Venema Tollervey, <br /> Joris Braspenning, Dominic Hoepfner1,                                1999). However, clear assembly pre- <br /> Elisabeth Petfalski2, Peter Philippsen1,                             ribosomes closely coordinated rRNA maturation <br /> David Tollervey2 Ed Hurt3                                        transport. A succession steps leads formation <br />                                                                      particles competent release nucleolus and <br /> BZH, Biochemie-Zentrum Heidelberg, Im Neuenheimer Feld 328,          export nucleus (for recent review Kressler <br /> D-69120 Heidelberg, Germany, 1Biozentrum der Universita&#195;&#710;t Basel, <br /> Klingelbergstra&#195;&#187;e 70, CH-4056 Basel, Switzerland 2Institute   et al., 1999). <br /> Cell Molecular Biology, University Edinburgh, Swann              The mechanism ribosome biogenesis appears be <br /> Building, King&apos;s Buildings, Edinburgh EH9 3JR, UK                    conserved eukaryotes. Many components <br /> 3 <br />  Corresponding author                                                involved complex process identi&#194;&#174;ed and <br /> e-mail: cg5@ix.urz.uni-heidelberg.de                                 characterized Saccharomyces cerevisiae, &gt;60 <br />                                                                      trans-acting factors required synthesize mature <br /> Ribosomal precursor particles assembled                   60S 40S subunits. These include rRNA-modifying <br /> nucleolus export cytoplasm. Using                  enzymes, endonucleases, exonucleases, RNA helicases <br /> visual assay nuclear accumulation 60S subunits,               proteins associated small nucleolar RNAs <br /> we isolated conditional-lethal strains             (snoRNAs) (Kressler et al., 1999). The majority these <br /> defects ribosomal export (rix mutants). Here                   factors enriched nucleolus. However, are <br /> report characterization <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>mutation</span> essen-               detected cytoplasm (Kressler et al., 1999) or <br /> tial gene, RIX7, encodes novel member                 associated nuclear pore complex (NPC) (Rout <br /> AAA ATPase superfamily. The rix7-1 temperature-                      et al., 2000). <br /> sensitive allele carries <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> causes                   Saccharomyces cerevisiae proven useful for <br /> defects pre-rRNA processing, biogenesis 60S                    analysis NPC nuclear export proteins, <br /> ribosomal subunits, subsequent export                 tRNA mRNA (Amberg et al., 1992; Kadowaki et al., <br /> the cytoplasm. Rix7p, associates 60S ribo-                1992; Doye Hurt, 1997; Segref et al., 1997; Stade <br /> somal precursor particles, localizes                  et al., 1997; Hellmuth et al., 1998; Sarkar et al., 1999). The <br /> nucleus exponentially growing cells, concen-                  analysis nuclear export ribosomes aided by <br /> trates nucleolus stationary phase cells. When              development functional green &#194;&#175;uorescent protein <br /> cells resume growth shift fresh medium,                      (GFP)-tagged ribosomal protein reporters. We pre- <br /> Rix7p&#194;&#177;green &#194;&#175;uorescent protein exhibits transient                  viously reported vivo export assay uses large <br /> perinuclear location. We propose nuclear AAA                  subunit reporter Rpl25p&#194;&#177;eGFP (Hurt et al., 1999; Gadal <br /> ATPase required restructuring nucleoplasmic                   et al., 2001). Rpl25p imported nucleus and <br /> 60S pre-ribosomal particles make competent                   assembles ribosomes direct binding rRNA <br /> for nuclear export.                                                  inside nucleolus (Van Beekvelt et al., 2000). This <br /> Keywords: AAA ATPase/nuclear transport/ribosomal                     assay showed subset nucleoporins Ran <br /> biogenesis/ribosomal export                                          required nuclear 60S subunit export. <br />                                                                      Analysis using different ribosomal protein&#194;&#177;GFP fusion <br />                                                                      gave similar results (Stage-Zimmermann et al., 2000). We <br />                                                                      exploited Rpl25p&#194;&#177;eGFP assay identify new <br />                                                                      factors required nuclear 60S subunit export (Gadal <br /> Introduction                                                         et al., 2001; Milkereit et al., 2001). <br /> The nucleolus specialized subnuclear compartment                Here report characterization rix7-1 mutant, <br /> which steps ribosome biogenesis place. This             complemented YLL034c encoding previ- <br /> process starts synthesis pre-rRNA                    ously uncharacterized member AAA family of <br /> transcripts, pre-5S rRNA large RNA polymerase I                ATPases. AAA proteins act variety cellular <br /> transcript (35S pre-rRNA yeast), processed            functions, including cell cycle regulation, protein degrad- <br /> the mature 25S/28S, 18S 5.8S rRNAs (Kressler et al.,             ation, organelle biogenesis vesicular transport; hence, <br /> 1999). The 5S RNA transcribed RNA polymerase III               origin `ATPases associated different <br /> and recruited separately assembling ribosome                  activities&apos;. The AAA domain highly conserved among <br /> (Dechampesme et al., 1999). Ribosomal proteins                   family common mechanism action been <br /> synthesized cytoplasm imported                   proposed. A nucleotide-dependent conformational switch <br /> nucleus, associate newly transcribed                 proposed apply tension bound proteins, allowing <br /> pre-rRNAs generate pre-ribosomal particles (Woolford,             AAA proteins unfold polypeptides, dissociate protein&#194;&#177; <br /> 1991). In contrast pre-rRNA processing modi&#194;&#174;ca-               protein interactions generate unidirectional movement <br /> tion, little known assembly pathway            (Rouiller et al., 2000; Vale, 2000). We Rix7p is <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                                    3695 <br />  O.Gadal et al. <br />  <br />  <br /> located nucleus, required 60S biogenesis <br /> and subsequent nuclear export. <br />  <br /> Results <br /> Isolation rix7-1 mutant, impaired <br /> in export ribosomes nucleolus <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> cytoplasm <br /> The screen nuclear accumulation large subunit <br /> reporter Rpl25p&#194;&#177;eGFP (Gadal et al., 2001) identi&#194;&#174;ed <br /> rix7-1, shows slow growth permissive tempera- <br /> tures (doubling times 2.8 h rix7-1 strains 23&#194;&#176;C and <br /> of 2 h wild-type strain) ceases growth 5 h after <br /> shift restrictive temperature 37&#194;&#176;C (Figure 1A). At <br /> 23&#194;&#176;C, Rpl25p&#194;&#177;eGFP cytoplasmic nuclear exclu- <br /> sion rix7-1 strain, strong nuclear accumulation <br /> was observed 2&#194;&#177;3 h transfer 37&#194;&#176;C (Figure 1B). The <br /> rix <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> was, therefore, visible distinct <br /> growth defect observed 37&#194;&#176;C. The <span id='am-241' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of <br /> accumulation Rpl25p&#194;&#177;eGFP inside nucleus was <br /> found different various rix mutants (Gadal <br /> et al., 2001). In rix mutants, Rpl25p&#194;&#177;eGFP accu- <br /> mulates nucleoplasm, other <br /> mutants, including rix7-1, Rpl25p&#194;&#177;eGFP accumulates <br /> predominantly nucleolus (Figure 1B, insert). This <br />                                                                 Fig. 1. Nuclear accumulation Rpl25p&#194;&#177;eGFP rix7-1 cells. <br /> suggests following transfer rix7-1 cells 37&#194;&#176;C,   (A) Growth curve rix7-1 isogenic wild-type cells 37&#194;&#176;C. <br /> release pre-ribosomal particles nucleolus    (B) rix7-1 isogenic wild-type cells expressing Rpl25p&#194;&#177;eGFP <br /> nucleoplasm inhibited.                                       grown 23&#194;&#176;C, shift 5 h 35&#194;&#176;C inspection in <br />    The wild-type RIX7 gene cloned complementa-           &#194;&#175;uorescence microscope. The arrowhead points nuclear <br />                                                                 staining triangle nucleolar staining. <br /> tion rix7-1 temperature-sensitive (ts) phenotype, and <br /> corresponds locus YLL034c, encodes essential <br /> and novel protein 92 kDa (see below). Cloned YLL034c <br /> complemented growth defect rix7-1 (Figure 2A) and <br /> rescued nuclear accumulation Rpl25p&#194;&#177;eGFP.                et al., 2000; Gadal et al., 2001). As shown Figure 2B, <br /> However, percent complemented cells          rix7-1 mutation synthetically lethal rpl10-2 <br /> exhibited nuclear accumulation Rpl25p&#194;&#177;eGFP (data         allele, indicating Rix7p Rpl10p functionally <br /> shown). Similarly, pre-rRNA processing defect           interact. <br /> not completely rescued expression YLL034c (see <br /> below), indicating rix7-1 mutation semi-            Rix7p conserved nuclear member AAA <br /> dominant. To rix7-1 mutation lies          ATPase family <br /> YLL034c gene locus isolate mutant allele, DNA        A search conserved motifs Rix7p revealed the <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> chromosomal locus recovered PCR            presence AAA ATPase domains (residues 241&#194;&#177;430 <br /> rix7-1 isogenic wild-type strains. A single nucleotide      569&#194;&#177;737; Neuwald et al., 1999) putative <br /> substitution YLL034c open reading frame        bipartite nuclear localization signal (NLS) the <br /> of rix7-1, changing proline (224) leucine. The recovered     N-terminal region (residues 175&#194;&#177;195; Figure 3A, upper <br /> rix7 allele inserted ARS/CEN plasmid. When          panel). To identify functionally important regions within <br /> expressed rix7-D strain, cloned allele conferred ts    Rix7p, set N- C-terminal deletions generated. <br /> growth nuclear accumulation Rpl25p&#194;&#177;eGFP similar          A construct deleted N-terminal amino acids 1&#194;&#177;174 <br /> to original rix7-1 strain. Mild nuclear accumulation     (DN) functional, longer N-terminal deletion <br /> Rpl25p&#194;&#177;eGFP observed recovered rix7           1&#194;&#177;202, including putative NLS (DN+NLS), no <br /> allele expressed wild-type strain (data shown),    longer supported growth (Figure 3A). A short C-terminal <br /> consistent semi-dominant phenotype rix7-1.          deletion 776&#194;&#177;837 (DC) leads loss function <br /> We conclude Pro224&#194;&#174;Leu mutation YLL034c          (Figure 3A). The short C-terminal region Rix7p lies <br /> responsible rix7-1 phenotype.                           outside AAA domains is, therefore, essential cell <br />    Next, wanted know RIX7 gene             growth. <br /> functionally linked components involved                Fluorescence microscopy revealed full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> <br /> ribosomal export. Mutations ribosomal protein            Rix7p&#194;&#177;GFP nuclear exponentially growing cells <br /> Rpl10p, late nuclear assembling component 60S              (Figure 3B). The functional Rix7pDN&#194;&#177;GFP shows a <br /> ribosomal subunits, rise rix phenotype (Gadal     similar localization, removal putative NLS <br /> et al., 2001). Rpl10p acts, part, recruiting     (DN+NLS) causes cytoplasmic location (Figure 3B). <br /> Nmd3p 60S ribosomal subunit. Nmd3p, turn,       Furthermore, corresponding (N+NLS)&#194;&#177;GFP construct <br /> adapter exportin Xpo1p, exports NES-              nuclear (Figure 3B). Thus, Rix7p contains single <br /> containing proteins nucleus cytoplasm (Ho       functional NLS N-terminal region (Figure 3A). <br />  <br /> 3696 <br />                                                                             AAA ATPase required nuclear export 60S subunits <br />  <br />  <br />                                                                        reduced, 40S peak increased (Figure 3A). The <br />                                                                        polysomal peaks reduced half-mers were <br />                                                                        readily detected. <br />                                                                           To observe decrease 60S ribosomal subunits in <br />                                                                        rix7-1 cells directly, 40S 60S subunits were <br />                                                                        dissociated high salt (0.8 M KCl) sucrose <br />                                                                        density gradient centrifugation (Figure 4B). The 60S:40S <br />                                                                        ratio measured UV absorption ~2 wild-type strains, <br />                                                                        decreased 0.85 rix7-1 strain shift 37&#194;&#176;C <br />                                                                        (Figure 4B). This strong depletion 60S subunits <br />                                                                        suggests Rix7p functions large subunit biogenesis. <br />  <br />                                                                        Pre-rRNA processing defective rix7-1 <br />                                                                        mutant strains <br />                                                                        To test depletion 60S ribosomal subunits <br />                                                                        observed rix7-1 strain consequence defects in <br />                                                                        pre-rRNA processing, analysed northern <br />                                                                        hybridization (Figure 5B C), primer extension <br />                                                                        (Figure 5D) pulse&#194;&#177;chase analysis (Figure 6). <br />                                                                        Following transfer rix7-1 strain 37&#194;&#176;C 2 h, <br /> Fig. 2. rix7-1 complemented YLL034c, genetically <br />                                                                        precursors pathway 25S 5.8S rRNA <br /> linked RPL10. (A) rix7-1 complemented YLL034c. Growth         synthesis, 27SB, 7S 6S pre-rRNAs, very <br /> of wild-type yeast (RIX7+) rix7-1 ts strain harbouring      strongly reduced (Figure 5). By 4 h transfer 37&#194;&#176;C, <br /> empty plasmid (-), plasmid 10 kb genomic insert containing    mature 5.8S 25S rRNAs seen depleted, <br /> YLL034c (pRIX7) plasmid containing YLL034c. YPD              consistent reduced 60S subunit the <br /> plates incubated 4 days 23 37&#194;&#176;C. (B) Synthetic lethal <br /> relationship rix7-1 rpl10-2. Synthetic lethality       rix7-1 strain 37&#194;&#176;C. In addition, 35S pre-rRNA was <br /> analysed ability lose wild-type RPL10 present URA3-     accumulated, accompanied reduction levels <br /> containing plasmid haploid double mutant rix7-1/rpl10-2 (see    27SA2 20S pre-rRNAs. A low level 23S <br /> Table I). For comparison, growth haploid single mutants         pre-rRNA, generated 35S cleavage at <br /> rix7-1and rpl10-2 shown. Growth analysed 4 days at <br /> 23&#194;&#176;C 5-&#194;&#175;uoroorotic acid (FOA)-containing plates (left panel) or <br />                                                                        site A3 absence prior cleavage sites A0, A1 and <br /> SDC+All (minimal glucose medium plus nutrients) plates.            A2, detected. These phenotypes reveal delay in <br />                                                                        early cleavages sites A0, A1 A2 the <br />                                                                        rix7-1 strain. <br /> However, construct short C-terminal deletion                   Primer extension analysis indicated cleavage at <br /> (DC), nuclear, functional, demonstrat-           sites A2 (Figure 5D), A0 A3 (data shown) <br /> ing Rix7p plays different essential role.          continued. Primer extension stops sites B1L B1S <br />   Rix7p shows high homology members                      detected low levels, clear change in <br /> AAA ATPase superfamily (data shown). However,                      relative ef&#194;&#174;ciencies (Figure 5D). All cleavages were <br /> human NVL (Germain-Lee et al., 1997) Schizo-                       accurate nucleotide level. We conclude that <br /> saccharomyces pombe SPBC16E9.10c ortho-                     processing pre-rRNA sites A3, B1L B1S <br /> logues Rix7p additional conserved motifs               continues rix7-1 strain, followed rapid degrad- <br /> identi&#194;&#174;ed proteins, including essential               ation 27SB pre-rRNAs. In wild-type cells, low <br /> C-terminal domain, addition AAA domains                  level pre-rRNA extends site A2 site E (the <br /> (Figure 3C). No clear homology detected                 3&#194;&#162; end 5.8S rRNA) observed (Figure 5C). In the <br /> non-essential N-terminal domain Rix7p, putative               rix7-1 mutant, lost replaced species that <br /> NLS present just &#194;&#174;rst AAA domain                  extends A2 C2 (the 3&#194;&#162; end 7S pre-rRNA). We <br /> human S.pombe counterparts (data shown).                       conclude pre-rRNA cleavage site C2 also <br />                                                                        continues mutant strains. Together the <br />                                                                        continued processing B1, indicates level <br /> Rix7p involved 60S ribosomal subunit                             7S pre-rRNA continues synthesized and <br /> biogenesis                                                             rapidly degraded. Notably, intermediates the <br /> The nucleolar/nuclear accumulation Rpl25p&#194;&#177;eGFP                      pathway 25S/5.8S synthesis accumulated the <br /> observed rix7-1 ts mutant (see Figure 1B)               rix7-1 strains, making unlikely loss mature <br /> localization Rix7p (see Figure 3B) suggest role                rRNAs primarily defect pre-rRNA cleavage. <br /> Rix7p biogenesis large ribosomal subunits.                      Pulse&#194;&#177;chase labelling rix7-1 strain 2 h after <br /> Defects 60S subunit biogenesis generally result                  transfer 37&#194;&#176;C, [3H]uracil [3H]methyl <br /> decreased free 60S subunits appearance `half-               methionine, showed dramatic reduction synthesis <br /> mer&apos; polysomes (polysomes containing unjoined 40S                   25S 5.8S rRNAs (Figure 6). The 35S pre- <br /> subunit initiation site). The sucrose density gradient          rRNA synthesized showed retardation its <br /> pro&#194;&#174;le polysomes isolated rix7-1 cells grown                processing. The 27SA pre-rRNA synthesized <br /> the permissive temperature similar                  delay rix7-1 strain, 27SB 7S <br /> isogenic wild type (Figure 4A). Following transfer              pre-rRNAs detected. Synthesis 20S pre- <br /> rix7-1 strain 37&#194;&#176;C, free 60S subunit peak                   rRNA 18S rRNA continued, delay consistent <br />  <br />                                                                                                                                3697 <br />  O.Gadal et al. <br />  <br />  <br />  <br />  <br /> Fig. 3. Domain organization Rix7p. (A) Upper panel: schematic presentation Rix7p sequence consisting N-domain (N), NLS <br /> (residues 175&#194;&#177;195), consecutive AAA domains (residues 241&#194;&#177;430 569&#194;&#177;737) short C-terminal domain (residues 789&#194;&#177;824). Arrows <br /> indicate position N- C-terminal deletions. Lower panel: functional analysis Rix7p deletion constructs DN, D(N+NLS) DC. <br /> The complementing activity Rix7p&#194;&#177;GFP, DN, DN+NLS DC tested RIX7 shuf&#194;&#175;e strain plating cells 5-FOA-containing plates <br /> (4 days 23&#194;&#176;C). (B) Intracellular location Rix7p&#194;&#177;GFP derived N- C-terminal deletions. Strains expressing Rix7p&#194;&#177;eGFP or <br /> the deletion constructs DN, D(N+NLS), DC N+NLS analysed 23&#194;&#176;C exponential growth phase &#194;&#175;uorescence microscopy <br /> and Nomarski imaging. (C) Schematic presentation conserved amino acids Rix7p homologues. Black grey rectangles denote <br /> identical homologous residues, respectively. NCB.GenPep accession Nos: S.cerevisiae Rix7p (S64785), S.pombe (T39584) and <br /> Homo sapiens (U68140) homologues. <br />  <br />  <br /> with 35S processing. Some reduction                    Rix7p associates pre-ribosomes <br /> of 18S rRNA seen, probably reduced                            The observed defects 60S ribosomal biogenesis suggest <br /> growth rate mutant. Synthesis tRNAs                     Rix7p functionally interacts pre-ribosomes inside <br /> 5S rRNA continued rix7-1 strain, reduction                  nucleus. Three major ribosomal precursor particles are <br /> accumulation consistent reduced growth.                             reported yeast (Trapman et al., 1975). An early 90S <br />    Together, data indicate strains defective                   particle, contains 35S pre-rRNA associated with <br /> Rix7p primarily defective pre-rRNA cleavage,                      early assembling ribosomal proteins, subsequently <br /> but rapidly degrade 27SB, possibly 7S, pre-rRNA.                     cleaved 66S 43S pre-ribosomes, precursors to <br /> This suggests Rix7p involved structural                         mature 60S 40S subunits, respectively. The 66S <br /> rearrangement required correct assembly,                   pre-ribosomal particle contains 27S 7S pre-rRNA <br /> stability, 60S ribosomal subunit. The kinetic delay                   species, 43S particle contains 20S <br /> in processing early sites pathway 18S                       pre-rRNA. Protein complexes associated pre- <br /> rRNA probably indirect, seen                      ribosomal particles, Noc1p&#194;&#177;Noc2p Noc2p&#194;&#177; <br /> many mutants defective 60S synthesis (Kressler                      Noc3p complexes, shown co-fractionate <br /> et al., 1999; Venema Tollervey, 1999).                                   35S 27S/7S pre-rRNA, respectively, <br />  <br /> 3698 <br />                                                                                 AAA ATPase required nuclear export 60S subunits <br />  <br />  <br />  <br />  <br /> Fig. 4. rix7-1 impaired 60S subunit biogenesis. Polysomal pro&#194;&#174;les <br /> (OD260 nm) sucrose density gradient centrifugation derived from <br /> the rix7-1 mutant wild-type strain grown permissive temperature <br /> (23&#194;&#176;C) 4 h shift restrictive temperature (37&#194;&#176;C) low <br /> salt (100 mM KCl) (A) high salt (0.8 M KCl) conditions (B). The <br /> positions 40S, 60S 80S ribosomal particles, polysomes and <br /> `half-mers&apos; indicated. <br />  <br />  <br />  <br /> under extraction conditions used (Milkereit <br /> et al., 2001). <br />    The association protein A (ProtA)-tagged Rix7p with <br /> pre-ribosomal particles assessed sucrose gradient <br /> centrifugation western blotting (Figure 7). The <br /> ribosomal protein Rpl10p served marker 60S <br /> subunits, 80S ribosomes, polysomes, Noc3p <br /> protein marker 66S pre-ribosomes (see also <br /> Milkereit et al., 2001). When whole-cell lysates were <br /> prepared low salt conditions, Rix7p found <br /> on sucrose gradient. However, pool of <br /> Rix7p&#194;&#177;ProtA seen denser fractions sucrose <br /> gradient, peak zone 60S subunits are <br /> found (Figure 7A). A careful analysis fractions <br /> around 60S subunits revealed Rix7p&#194;&#177;ProtA co-peaks <br /> with 60S subunits, tails deeper fractions the <br /> gradient, pre-ribosomal particles 66S                Fig. 5. Northern analysis pre-rRNA processing rix7-1 cells. <br /> 90S particles (Figure 7B). In contrast,                 (A) Schematic diagram pre-rRNA processing pathway. The 35S <br /> the yeast cell extract treated high salt prior                pre-rRNA precursor contains sequences mature 18S, 5.8S <br />                                                                           25S rRNAs separated internal transcripted spacers, <br /> centrifugation, pool Rix7p&#194;&#177;ProtA peaking                    &#194;&#175;anked 5&#194;&#162;-ETS 3&#194;&#162;-ETS. The locations known <br /> 60S subunits disappeared, 60S 40S subunits                   processing sites rRNA precursor molecules indicated. (B) Wild <br /> readily detectable (data shown). We conclude                     type (lanes 1 2) rix7-1 (lanes 3&#194;&#177;5) analysed following <br /> under steady-state conditions, fraction Rix7p                     growth 25&#194;&#176;C (0 h samples), 2 6 h transfer 37&#194;&#176;C. <br /> associated intranuclear precursor particles 60S                   Pre-rRNA species high molecular weight detected indicated, <br />                                                                           oligonucleotides used. (C) Pre-rRNA species low <br /> large subunits. Rix7p strongly predicted enzyme               molecular weight wild type (lanes 1&#194;&#177;3) rix7-1 (lanes 4&#194;&#177;6) <br /> and expect transient interactions pre-                   analysed way. (D) Primer extension analysis of <br /> ribosomal particles.                                                      wild-type (lanes 1 2) rix7-1 (lanes 3&#194;&#177;5) cells. <br />  <br />  <br />                                                                                                                                         3699 <br />  O.Gadal et al. <br />  <br />  <br />  <br />  <br />                                                                           Fig. 7. A pool Rix7p co-fractionates pre-ribosomal particles <br />                                                                           60S large subunits. (A) Analysis polysomal ribosome fractions <br />                                                                           derived cells expressing Rix7p&#194;&#177;ProtA. Fractions analysed <br /> Fig. 6. Pulse&#194;&#177;chase analysis pre-rRNA processing rix7-1         SDS&#194;&#177;PAGE western blotting using a-Rpl10p a-ProtA <br /> strain. Wild-type (lanes 1 &#194;&#177;5) rix7-1 (lanes 6&#194;&#177;10) strains       antibodies, respectively. (B) Comparison Rix7p&#194;&#177;ProtA 66S <br /> labelled [3H]uracil 1 min [3H]methyl methionine 2    pre-ribosomal subunit marker Noc3p (Milkereit et al., 2001) sucrose <br /> min, chased unlabelled uracil methionine times   gradients. Fractions analysed SDS&#194;&#177;PAGE western blotting <br /> indicated. (A) High molecular weight RNA species. (B) Low molecular       using a-Noc3p a-ProtA antibodies, respectively. <br /> weight RNA species. <br />  <br />                                                                           perinuclear location re-induction growth <br />                                                                           (Figure 8A, re-induction). <br /> Rix7p changes intranuclear distribution                                  This change intranuclear localization Rix7p <br /> depending growth condition                                         points dynamic localization different growth <br /> As shown above, Rix7p tagged GFP expressed                       conditions. This followed time-lapse &#194;&#175;uores- <br /> from ARS/CEN plasmid exhibits generalized nuclear                    cence microscopy. Similar observations, <br /> localization clear sub-nuclear concentration (see                 Rix7p&#194;&#177;GFP concentrated distinct area the <br /> Figure 3B). We expressed Rix7p&#194;&#177;GFP construct                         nucleus starved cells (Figure 8B, 0.5 h). After re- <br /> integrated RIX7 gene locus avoid overexpression.                induction growth transfer fresh medium <br /> When cells grown stationary phase (OD600                          (Figure 8B, 3.5 h), nucleolar concentration of <br /> between 5 10), Rix7p&#194;&#177;GFP signal weaker,                    Rix7p&#194;&#177;GFP changed distribution the <br /> but concentrated distinct nuclear region, appar-                 entire nucleoplasm (Figure 8B, 3.5 8 h). No speci&#194;&#174;c <br /> ently corresponding nucleolus (Figure 8A, station-                 changes pattern Rix7p&#194;&#177;GFP distribution were <br /> ary; below). When cells stationary phase                      seen mitosis. <br /> transferred fresh medium 2 h, Rix7p&#194;&#177;GFP signal                    To Rix7p&#194;&#177;GFP concentrates nucleolus <br /> increased exhibited transient perinuclear location                  stationary phase, co-expressed nucleolar <br /> (Figure 8A, re-induction). After prolonged growth fresh                marker DsRed&#194;&#177;Nop1p (Gadal et al., 2001). The green <br /> medium, Rix7p&#194;&#177;GFP &#194;&#174;nally regained overall nuclear                      &#194;&#175;uorescent signal Rix7p&#194;&#177;GFP red &#194;&#175;uorescent <br /> distribution (Figure 8A, exponential). As control used               signal DsRed&#194;&#177;Nop1p largely overlapped starved cells <br /> a bona &#194;&#174;de nucleolar protein tagged GFP,                             (Figure 8C, stationary), indicating Rix7p&#194;&#177;GFP indeed <br /> Noc1p&#194;&#177;GFP (Milkereit et al., 2001),                   concentrates nucleolus. The peripheral nuclear <br /> similar distinct nuclear spot (i.e. nucleolus) starved             location Rix7p&#194;&#177;GFP seen re-induction growth is <br /> cells, remains nucleolar does exhibit                       extended nucleolus. Under conditions, <br />  <br /> 3700 <br />                                                                                    AAA ATPase required nuclear export 60S subunits <br />  <br />  <br />  <br />  <br /> Fig. 8. Location Rix7p&#194;&#177;GFP living cells grown different growth conditions. (A) The Rix7&#194;&#177;GFP (genomic integration) strain grown <br /> at 23&#194;&#176;C stationary phase (OD600 = 5), growth re-induced fresh YPD medium 3 h 23&#194;&#176;C, cells grown exponential phase <br /> (OD600 = 0.5). The GFP signal monitored &#194;&#175;uorescence microscopy. Cells visualized Nomarski imaging. The nucleolar protein <br /> Noc1p&#194;&#177;GFP (Milkereit et al., 2001), does exhibit transient perinuclear location growth induction, served control. (B) Time- <br /> lapse microscopy Rix7p&#194;&#177;GFP stationary cells re-induction fresh YPD medium. (C) Rix7&#194;&#177;GFP cells expressing DsRed&#194;&#177;Nop1p <br /> (pUN100-DsRed-NOP1) analysed &#194;&#175;uorescence microscopy stationary phase conditions growth fresh YPD medium. The <br /> Rix7p&#194;&#177;GFP DsRed&#194;&#177;Nop1p images merged detect co-localization. <br />  <br />  <br />  <br /> the nucleolus remains distinct small, does                       recently (for reviews Vale, 2000; Zhang et al., <br /> overlap signi&#194;&#174;cantly Rix7p&#194;&#177;GFP signal                               2000). While biological functions various <br /> (Figure 8C, re-induction).                                                   members superfamily unrelated, common <br />    We conclude studies Rix7p shuttle                     mechanism emerging structural studies. AAA <br /> between nucleolus, nucleoplasm nuclear periph-                       proteins assemble oligomers, cases forming <br /> ery. Alterations distribution Rix7p correlate                  hexameric rings. A nucleotide-dependent conformational <br /> with changes ribosome biogenesis transport                         switch applies tension bound substrates, allowing AAA <br /> response <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability.                                           proteins unfold dissociate attached proteins (Vale, <br />                                                                              2000). We direct evidence oligomerization <br />                                                                              Rix7p, rix7-1 exhibits semi-dominant phenotype <br /> Discussion                                                                   consistent incorporation intact and <br /> Here, report member superfamily AAA                      mutated subunits oligomeric complex. Similar <br /> ATPases (Rix7p) associates precursor particles                       observations NSF, ClpA regula- <br /> 60S ribosomal subunits, plays role large subunit                    tory subunits proteasome (Vale, 2000). <br /> assembly subsequent export cytoplasm.                                It reasonable assume activities can <br />    While members `DEAD-box&apos; family                            disassemble protein complexes required the <br /> ATPases (helicases) implicated ribosome                         various stages ribosome synthesis transport <br /> assembly, &#194;&#174;rst report role AAA-type                  pre-ribosomes nucleolus cytoplasm. A <br /> ATPase ribosome biogenesis. AAA proteins                        vast number trans-acting factors participate ribosome <br /> in eukaryotes, prokaryotes Archaea, organ-                      synthesis. More 50 protein factors &gt;100 <br /> ism different AAA proteins, function                       snoRNAs predicted associate subsequently <br /> diverse cellular processes. The genome S.cerevisiae                       dissociate pre-rRNA molecule its <br /> encodes ~50 different AAA ATPases. The unifying feature                      intranuclear lifespan. Moreover, pre-rRNA processing in <br /> of AAA superfamily ATPase domain 220                            yeast fast, implying rapid binding release of <br /> amino acids, structural fold solved                           factors. This presumably entails considerable free <br />  <br />                                                                                                                                             3701 <br />  O.Gadal et al. <br />  <br />  <br /> Table I. Yeast strains <br /> Name                     Genotype                                                              Origin <br />  <br /> RS453a                   MATa, ade2, leu2, ura3, his3, trp1                                    Hurt et al. (1999) <br /> FY23                     MATa, ura3, trp1, leu2                                                derived S288C <br /> FY86                     MATa, ura3, his3, leu2                                                derived S288C <br /> rix7-1 original          MATa, ura3, his3, leu2, rix7-1                                        isolated ts collection (Amberg et al., 1992) <br /> rix7-1                 MATa, ura3, his3, leu2, trp1, rix7-1                                  offspring rix7-1 original 3 FY86 <br /> rix7-1                 MATa, ura3, his3, leu2, trp1, rix7-1                                  offspring rix7-1 original 3 FY86 <br /> RIX7 Shuf&#194;&#175;e              MATa, leu2, ura3, his3, met15, lys2, rix7::KANMX4                     derived Euroscarf strain Y21522 <br />                          +pRS316-RIX7(ARS/CEN, RPL25, URA3) <br /> DEHQ1-1                  MATa, ade2, his3, trp1, leu2, ura3, can1, qsr1D1::HIS3                Eisinger et al. (1997) <br />                          +pHFF22 (ARS/CEN, URA3, QSR1) <br /> RPL10D rix7-1            MATa, ade2, his3, trp1, leu2, ura3, qsr1D1::HIS3, rix7-1              offspring rix7-1 3 DEHQ1-1 <br />                          +pHFF22 (ARS/CEN, URA3, QSR1) <br /> RIX7&#194;&#177;eGFP                MATa, leu2, ura3, his3, met15, lys2, rix7::KANMX4                     derived RIX7 Shuf&#194;&#175;e <br />                          +pRS315-RIX7-eGFP(ARS/CEN, LEU2, Rix7-eGFP) <br /> RIX7&#194;&#177;GFP                 MATa, ura3, trp1, his3, leu2, RIX7-GFP::KANMX4                        offspring FY23 3 FY86 <br /> RIX7&#194;&#177;ProtA               MATa, ura3, trp1, his3, leu2, RIX7-ProtA::TRP1                        offspring FY23 3 FY86 <br />  <br />  <br />  <br />  <br /> energy change is, therefore, likely require                    60S subunit biogenesis. Rix7p associated with <br /> substantial input energy. This need may, part,                 precursor particles 60S subunits apparently <br /> met ATP hydrolysis, Rix7p required                  accompanies nucleolus nucleoplasm. <br /> these steps. Growing yeast cells copies              Our data suggest model Rix7p acts to <br /> U3 snoRNP support ribosome synthesis ~1 min                 restructure 60S subunits nucleus prior to <br /> in absence recycling (at 2000 ribosomes/min),                transport NPCs. In absence this <br /> other pre-rRNA processing factors appear roughly                restructuring, subunits competent for <br /> comparable abundances, underlining potential import-                export, retained rapidly degraded the <br /> ance activity.                                               nucleus. The lack Rix7p activity slow <br />     The large Balbiani ring pre-mRNAs                    recycling processing factors, potentially explain- <br /> extensively studied Chironomus tentans. For these,                   ing delay early pre-rRNA processing steps seen <br /> substantial structural rearrangement seen precede             mutant strains. <br /> nuclear export, accompanied loss not <br /> all hnRNP proteins (Sun et al., 1998). It appears very <br /> probable export ribosomal subunits involve             Materials methods <br /> energy-dependent structural rearrangements. Close homo- <br /> logues Rix7p exist eukaryotes, Archaea             Yeast strains, DNA recombinant work, microbiological <br />                                                                         techniques <br /> bacteria, suggesting function ribosome synthesis              Yeast strains used study listed Table I. Microbiological <br /> is conserved eukaryotes. Preparation                  techniques, plasmid transformation recovery, mating, sporulation of <br /> ribosome subunits nuclear transport                diploids tetrad analysis carried essentially described <br /> activity. We reported previously late structural            (Santos-Rosa et al., 1998). DNA recombinant work performed <br /> rearrangement appears convert salt-labile pre-60S                    according Maniatis et al. (1982). <br /> particles mature subunits (Tollervey et al., 1993).              Plasmid constructions <br /> Rix7p obvious candidate participate              Plasmids pUN100-DsRed-Nop1, pRS316-Rpl25-eGFP, pNOPPA1L, <br /> rearrangement.                                                          pNOPGFP1L pFA6a-GFP(S65T)-KANMX6 described previ- <br />    The pre-rRNA processing phenotype observed                    ously (Hellmuth et al., 1998; Longtine et al., 1998; Gadal et al., 2001). <br />                                                                         pFA6a-(2*ProtA-TEV)-TRP1 derived pFA6a-GFP(S65T)- <br /> rix7-1 mutant strain typical defects seen                 TRP1 cloning PCR-generated fragment coding 23ProtA-TEV <br /> other mutants defective 60S subunit biogenesis.             pNOPPA1L place GFP coding sequence. <br /> In cases, early pre-rRNA cleavages sites                 RIX7 including 5&#194;&#162;-UTR ampli&#194;&#174;ed PCR yeast genomic <br /> A0&#194;&#177;A2 inhibition,                 DNA. The derived fragment cut SacI&#194;&#177;BamHI fused in- <br />                                                                         frame eGFP variant present pRS315-RPL25-eGFP, generating <br /> no clear direct relationship mutant product              pRS315-RIX7-eGFP. The pRS315-rix7-1-eGFP plasmid constructed <br /> and cleavages (see Milkereit et al., 2001;                   way, using yeast genomic DNA extracted rix7-1 <br /> reviewed Kressler et al., 1999; Venema Tollervey,                strain. <br /> 1999). Recycling processing assembly factors                    Three PCR-generated fragments containing 5&#194;&#162;-UTR RIX7 and <br /> be inhibited rix7-1 cells direct defect               different C-terminal deletions RIX7 coding sequence were <br />                                                                         generated yeast genomic DNA: N (corresponding residues <br /> disassembly nucleoplasmic accumulation                        1&#194;&#177;174), N+NLS (corresponding residues 1&#194;&#177;202) DC (correspond- <br /> inappropriately structured 60S ribosome precursors. In                  ing residues 1&#194;&#177;776). These PCR-generated fragments cut <br /> either case, depletion pool free factors available         SacI&#194;&#177;BamHI fused in-frame eGFP variant present in <br /> envisaged inhibit assembly processing newly               pRS315-RPL25-eGFP. N-terminal deletions RIX7 ampli&#194;&#174;ed in <br />                                                                         steps. The RIX7 5&#194;&#162;-UTR ampli&#194;&#174;ed PCR yeast genomic <br /> synthesized pre-rRNAs.                                                  DNA, generating NdeI site ATG codon RIX7. This fragment <br />    In summary, shown conserved member                    cloned SacI&#194;&#177;BamHI fragment pRS315-RPL25-eGFP to <br /> of AAA-type ATPase superfamily, Rix7p, involved                  generate pRS315-RIX7::eGFP. Two fragments corresponding the <br />  <br />  <br /> 3702 <br />                                                                                    AAA ATPase required nuclear export 60S subunits <br />  <br /> C-terminal RIX7 coding sequence generated PCR: DN            References <br /> (residues 174&#194;&#177;837) DN+NLS (residues 202&#194;&#177;837). These fragments <br /> were cut NdeI&#194;&#177;BamHI cloned pRS315- RIX7::eGFP. These           Amberg,D.C., Goldstein,A.L. Cole,C.N. (1992) Isolation and <br /> deletion constructs expressed control RIX7               characterization RAT1: essential gene Saccharomyces <br /> promoter using ATG start codon NdeI site.                           cerevisiae required ef&#194;&#174;cient nucleocytoplasmic traf&#194;&#174;cking of <br />                                                                                mRNA. Genes Dev., 6, 1173&#194;&#177;1189. <br /> Strain constructions                                                         Dechampesme,A.M., Koroleva,O., Leger-Silvestre,I., Gas,N. and <br /> Genomic integration GFP in-frame RIX7 obtained                  Camier,S. (1999) Assembly 5S ribosomal RNA required a <br /> described previously (Longtine et al., 1998). Genomic integration           speci&#194;&#174;c step pre-rRNA processing pathway. J. Cell Biol., 145, <br /> ProtA carried way, using pFA6a-(2*ProtA-           1369&#194;&#177;1380. <br /> TEV)-TRP1 vector.                                                            Doye,V. Hurt,E.C. (1997) From nucleoporins nuclear pore <br />                                                                                complexes. Curr. Opin. Cell Biol., 9, 401&#194;&#177;411. <br />                                                                              Eisinger,D.P., Dick,F.A. Trumpower,B.L. (1997) Qsr1p, 60S <br /> Cloning RIX7/YLL034c <br />                                                                                ribosomal subunit protein, required joining 40S 60S <br /> A yeast genomic library LEU2-containing ARS/CEN plasmid <br />                                                                                subunits. Mol. Cell. Biol., 17, 5136&#194;&#177;5145. <br /> (Gautier et al., 1997) transformed rix7-1 strain. From <br />                                                                              Gadal,O., Straub,D., Kessl,J., Trumpower,B., Tollervey,D. Hurt,E. <br /> colonies growing restrictive temperature (37&#194;&#176;C) isolated <br />                                                                                (2001) Nuclear export 60S ribosomal subunits depends Xpo1p <br /> plasmid pRIX7 genomic insert. The complementing plasmid <br />                                                                                requires NES-containing factor Nmd3p associates the <br /> contained YLL034c gene. pYLL034c harbouring RIX7 gene <br />                                                                                large subunit protein Rpl10p. Mol. Cell. Biol., 21, 3405&#194;&#177;3415. <br /> was cloned shown complement ts growth defect rix7-1 <br />                                                                              Gautier,T., Berge&#195;?s,T., Tollervey,D. Hurt,E. (1997) Nucleolar KKE/ <br /> mutant. pRS316-RIX7 generated subcloning YLL034c into <br />                                                                                D repeat proteins Nop56p Nop58p interact Nop1p are <br /> pRS316. <br />                                                                                required ribosome biogenesis. Mol. Cell. Biol., 17, 7088&#194;&#177;7098. <br />                                                                              Germain-Lee,E.L., Obie.,C. Valle,D. (1997) NVL: new member of <br /> Pulse&#194;&#177;chase northern analysis rRNA                                      AAA family ATPases localized nucleus. Genomics, 44, <br /> Pulse&#194;&#177;chase labelling rRNA analysis rRNA processing               22&#194;&#177;34. <br /> northern hybridization performed described (Tollervey, 1987;         Hellmuth,K., Lau,D.M., Bischoff,F.R., Ku&#195;&#710;nzler,M., Hurt,E.C. and <br /> Tollervey et al., 1993).                                                       Simos,G. (1998) Yeast Los1p properties exportin-like <br />   Oligonucleotides used were: 003: TGTTACCTCTGGGCCC; 004:                      nucleocytoplasmic transport factor tRNA. Mol. Cell. Biol., 18, <br /> CGGTTTTAATTGTCCTA; 007: CTCCGCTTATTGATATGC; 008:                               6374&#194;&#177;6386. <br /> CATGGCTTAATCTTTGAGAC; 013: GGCCAGCAATTTCAAGTTA;                              Ho,J.H., Kallstrom,G. Johnson,A.W. (2000) Nmd3p Crm1p- <br /> 017: GCGTTGTTCATCGATGC; 020: TGAGAAGGAAATGACGCT;                               dependent adapter protein nuclear export large ribosomal <br /> 219: GAAGCGCCATCTAGATG; 5&#194;&#162; A0L: GGTCTCTCTGCTGCCGG.                             subunit. J. Cell Biol., 151, 1057&#194;&#177;1066. <br />                                                                              Hurt,E., Hannus,S., Schmelzl,B., Lau,D., Tollervey,D. Simos,G. <br /> Fluorescence microscopy                                                        (1999) A novel vivo assay reveals inhibition ribosomal nuclear <br /> pRS315-RPL25-eGFP pRS316-RPL25-eGFP introduced                     export Ran-cycle nucleoporin mutants. J. Cell Biol., 144, <br /> yeast cells transformation, selected SDC-leu medium SDC-          389&#194;&#177;401. <br /> ura medium, respectively. Individual transformants grown liquid      Kadowaki,T., Zhao,Y. Tartakoff,A.M. (1992) A conditional yeast <br /> SDC-leu medium 23&#194;&#176;C OD600 nm ~1, shift 37&#194;&#176;C           mutant de&#194;&#174;cient mRNA transport nucleus cytoplasm. Proc. <br /> liquid YPD medium. After centrifugation, cells resuspended             Natl Acad. Sci. USA, 89, 2312&#194;&#177;2316. <br /> water, mounted slide viewed &#194;&#175;uorescence microscope.          Kressler,D., Linder,P. De La Cruz,J. (1999) Protein trans-acting <br /> In vivo, GFP signal examined &#194;&#175;uorescent channel;            factors involved ribosome biogenesis Saccharomyces cerevisiae. <br /> DsRed used fusion Nop1p examined rhodamine                  Mol. Cell. Biol., 19, 7897&#194;&#177;7912. <br /> channel Zeiss Axioskop &#194;&#175;uorescence microscope pictures         Longtine,M.S., McKenzie,A., Demarini,D.J., Shah,N.G., Wach,A., <br /> obtained Xillix Microimager CCD camera. Digital pictures           Brachat,A., Philippsen,P. Pringle,J.R. (1998) Additional <br /> processed software program Improvision (Open lab)                   modules versatile economical PCR-based gene deletion and <br /> Photoshop 4.0.1 (Adobe).                                                       modi&#194;&#174;cation Saccharomyces cerevisiae. Yeast, 14, 953&#194;&#177;961. <br />                                                                              Maniatis,T., Fritsch,E.T. Sambrook,J. (1982) Molecular Cloning: A <br /> Time-lapse microscopy                                                          Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold <br /> Time-lapse microscopy performed using RIX7&#194;&#177;GFP strain.                 Spring Harbor, NY. <br /> Exponential growing cells (interval: 120 s; exposure: 0.2 s; transmission:   Milkereit,P. et al. (2001) Maturation intranuclear transport pre- <br /> 25%; Z-Planes: 3; Optovar: 13; objective: 1003 1.3 PH3 oil)                 ribosomes requires Noc proteins. Cell, 105, 499&#194;&#177;509. <br /> stationary culture (interval: 180 s; exposure: 0.3 s; transmission: 50%;     Neuwald,A.F., Aravind,L., Spouge,J.L. Koonin,E.V. (1999) AAA+: <br /> Z-Planes: 3; Optovar: 13; objective: 1003 1.3 PH3 oil) followed        class chaperone-like ATPases associated assembly, <br /> confocal microscopy.                                                           operation, dissassembly proteins complexes. Genome Res., 9, <br />                                                                                27&#194;&#177;43. <br /> Miscellaneous                                                                Rouiller,I., Butel,V.M., Latterich,M., Milligan,R.A. Wilson-Kubalek, <br /> The search functional domains performed using pfscan               E.M. (2000) A major conformational change p97 AAA ATPase upon <br /> package (www.isrec.isb-sib.ch/software/PFSCAN). SDS&#194;&#177;PAGE                   ATP binding. Mol. Cell, 6, 1485&#194;&#177;1490. <br /> western blot analysis performed according Siniossoglou et al.        Rout,M.P., Aitchison,J.D., Suprapto,A., Hjertaas,K., Zhao,Y. and <br /> (1996), isolation ribosomes low high salt conditions       Chait,B.T. (2000). The yeast nuclear pore complex: composition, <br /> sucrose gradient centrifugation described Tollervey et al. (1993).       architecture transport mechanism. J. Cell Biol., 148, 635&#194;&#177;651. <br /> Whole-cell lysates fractions sucrose gradient separated    Santos-Rosa,H., Moreno,H., Simos,G., Segref,A., Fahrenkrog,B., <br /> by SDS&#194;&#177;PAGE analysed western blotting using indicated               Pante&#195;&#8218;,N. Hurt,E. (1998) Nuclear mRNA export requires <br /> antibodies. Polyclonal antiserum Rpl10p kind gift             complex formation Mex67p Mtr2p nuclear <br /> Dr Trumpower (Hanover, NH). Af&#194;&#174;nity-puri&#194;&#174;ed a-Noc3p antibodies                 pores. Mol. Cell. Biol., 18, 6826&#194;&#177;6838. <br /> were kindly obtained Dr Tschochner (BZH, Heidelberg, Germany).          Sarkar,S., Azad,A.K. Hopper,A.K. (1999) Nuclear tRNA <br />                                                                                aminoacylation role nuclear export endogenous tRNAs <br />                                                                                Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA, 96, <br />                                                                                14366&#194;&#177;14371. <br /> Acknowledgements                                                             Segref,A., Sharma,K., Doye,V., Hellwig,A., Huber,J., Lu&#195;&#710;hrmann,R. and <br />                                                                                Hurt,E.C. (1997) Mex67p essential factor nuclear <br /> We grateful Dr Trumpower providing antibodies               mRNA export binds poly(A)+ RNA nuclear pores. <br /> ribosomal proteins. We thank Dr K.Stra&#195;&#710;&#195;&#187;er critical reading         EMBO J., 16, 3256&#194;&#177;3271. <br /> manuscript. E.H. recipient grants Deutsche Forschungs-     Siniossoglou,S., Wimmer,C., Rieger,M., Doye,V., Tekotte,H., Weise,C., <br /> gemeinschaft (Schwerpunktprogramm `Funktionelle Architektur des                Emig,S., Segref,A. Hurt,E.C. (1996) A novel complex of <br /> Zellkerns&apos;), E.P. D.T. funded Wellcome Trust, O.G.          nucleoporins, includes Sec13p Sec13p homolog, is <br /> is holder HFSP fellowship.                                              essential normal nuclear pores. Cell, 84, 265&#194;&#177;275. <br />  <br />  <br />                                                                                                                                               3703 <br />  O.Gadal et al. <br />  <br /> Stade,K., Ford,C.S., Guthrie,C. Weis,K. (1997) Exportin 1 (Crm1p) <br />   essential nuclear export factor. Cell, 90, 1041&#194;&#177;1050. <br /> Stage-Zimmermann,T., Schmidt,U. Silver,P.A. (2000) Factors <br />   affecting nucler export 60S subunits vivo. Mol. Biol. Cell, <br />   11, 3777&#194;&#177;3789. <br /> Sun,X., Alzhanova-Ericsson,A.T., Visa,N., Aissouni,Y., Zhao,J. and <br />   Daneholt,B. (1998) The hrp23 protein Balbiani ring pre-mRNP <br />   particles released just binding particles the <br />   nuclear pore complex. J. Cell Biol., 142, 1181&#194;&#177;1193. <br /> Tollervey,D. (1987) A yeast small nuclear RNA required normal <br />   processing pre-ribosomal RNA. EMBO J., 6, 4169&#194;&#177;4175. <br /> Tollervey,D., Lehtonen,H., Jansen,R.P., Kern,H. Hurt,E.C. (1993) <br />   Temperature-sensitive mutations demonstrate roles yeast &#194;&#174;brillarin <br />   pre-rRNA processing, pre-rRNA methylation ribosome <br />   assembly. Cell, 72, 443&#194;&#177;457. <br /> Trapman,J., Retel,J. Planta,R.J. (1975) Ribosomal precursor <br />   particles yeast. Exp. Cell Res., 90, 95&#194;&#177;104. <br /> Vale,R.D. (2000) AAA proteins: lords ring. J. Cell Biol., 150, <br />   13&#194;&#177;19. <br /> Van Beekvelt,C.A., Kooi,E.A., Graaff-Vincent,M., Riet J., Venema,J. <br />   Raue,H.A. (2000) Domain III Saccharomyces cerevisiae 25 S <br />   ribosomal RNA: role binding ribosomal protein L25 60 S <br />   subunit formation. J. Mol. Biol., 296, 7&#194;&#177;17. <br /> Venema,J. Tollervey,D. (1999) Ribosome synthesis in <br />   Saccharomyces cerevisiae. Annu. Rev. Genet., 33, 26 <br /> </body></html>