<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Metabolite essentiality elucidates robustness <br /> of Escherichia coli metabolism <br /> Pan-Jun Kim*&#226;&#8364;&#160;, Dong-Yup Lee&#226;&#8364;&#161;&#194;&#167;&#194;&#182;&#229;&#8218;&#168;, Tae Yong Kim*&#194;&#167;&#194;&#182;, Kwang Ho Lee*&#194;&#167;&#194;&#182;**, Hawoong Jeong*&#226;&#8364;&#160;&#226;&#8364;&#161;&#226;&#8364;&#160;&#226;&#8364;&#160;, Sang Yup Lee*&#226;&#8364;&#161;&#194;&#167;&#194;&#182;&#226;&#8364;&#160;&#226;&#8364;&#160;&#226;&#8364;&#161;&#226;&#8364;&#161;, <br /> and Sunwon Park&#226;&#8364;&#161;&#194;&#182; <br /> *Center Systems Synthetic Biotechnology, Institute BioCentury, &#226;&#8364;&#160;Department Physics, &#226;&#8364;&#161;Bioinformatics Research Center, &#194;&#167;Metabolic <br /> and Biomolecular Engineering National Research Laboratory, BioProcess Engineering Research Center, &#194;&#182;Department Chemical Biomolecular <br /> Engineering (BK21 Program), &#226;&#8364;&#161;&#226;&#8364;&#161;Department BioSystems, Korea Advanced Institute Science Technology, 373-1 Guseong-dong, <br /> Yuseong-gu, Daejeon 305-701, Korea <br />  <br /> Edited Philip P. Green, University Washington School Medicine, Seattle, WA, approved July 2, 2007 (received review April 9, 2007) <br />  <br /> Complex biological systems robust genetic envi-                 various genotypic environmental conditions using the <br /> ronmental changes levels organization. Many biological                genome-scale Escherichia coli metabolic model consisting 762 <br /> functions Escherichia coli metabolism sustained               metabolites 932 biochemical reactions (9&#226;&#8364;&#8220;13) (see Materials <br /> single-gene multiple-gene mutations using redundant                  Methods). <br /> or alternative pathways. Thus, limited number genes have <br /> been identified lethal cell. In regard, reaction-         Results <br /> centric gene deletion study limitation understanding               Characterization Vivo Validation Metabolite Essentiality. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> <br /> metabolic robustness. Here, report use flux-sum,                explore robustness E. coli metabolism metabolite <br /> is summation incoming outgoing fluxes                    perspective, classified intracellular metabolites into <br /> particular metabolite pseudo-steady state conditions,                categories, essential nonessential metabolites, mon- <br /> good conserved property elucidating robustness E. coli              itoring cell growth consumption rate given <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> metabolite point view. The functional behavior,                  metabolite suppressed zero (Materials Methods). The <br /> well structural evolutionary properties metabolites               resultant list essential metabolites given supporting <br /> essential cell <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span>, investigated means                  information (SI) Table 1 19 different environmental <br /> constraints-based flux analysis perturbed conditions. The                 conditions specified different combinations C, P, N, <br /> essential metabolites capable maintaining steady flux-sum              S sources, aerobic vs. anaerobic conditions (SI Table 2). <br /> even severe perturbation actively redistributing                 The results obtained glucose-minimal medium aerobic <br /> relevant fluxes. Disrupting flux-sum maintenance               condition used representative examples. Interest- <br /> suppress cell growth. This approach analyzing metabolite es-                 ingly, relatively unstudied metabolites, hexadecenoyl-ACP, <br /> sentiality provides insight cellular robustness concomitant            phosphatidylglycerol, 2-isopropylmaleate, be <br /> fragility, used applications, including            essential. The metabolite essentiality does depend on <br /> development new drugs treating pathogens.                                environmental conditions 87.8% total essential <br />                                                                                 metabolites essential different growth conditions <br /> cellular robustness &#229;&#8230;&#169; flux-sum &#229;&#8230;&#169; metabolic networks <br />                                                                                 (Fig. 1a). <br />                                                                                    The essentiality given metabolite demonstrated in <br />                                                                                 vivo means multiple gene disruptions metab- <br /> A     vailability complete genome sequences well <br />       characterized organisms led reconstruction of <br /> genome-scale metabolic networks, represent complex <br />                                                                                 olite. If disrupting multiple nonlethal reactions a <br />                                                                                 particular metabolite suppresses cell growth, metabolite can <br />                                                                                 regarded essential deletion individual <br /> web metabolites interconversions catalyzed                  reaction itself nonlethal. We conducted gene <br /> gene products. Robustness, inherent property metabolic                   deletion experiments neighboring reactions around <br /> networks, enables maintenance cellular functions                   tetrahydrofolate, identified essential metabolite <br /> various internally externally perturbed conditions. This                    silico. Among reactions (genes), phosphoribosylglyci- <br /> robustness experimentally demonstrated                  namide formyltransferase (purN), glycine cleavage system <br /> the disruption considerable portion genes affect              (lpdA), glycine hydroxymethyltransferase (glyA) se- <br /> the cell viability (1, 2). Although studies topological              lected disruption targets. Each single double gene dele- <br /> functional properties metabolic networks achieved                  tion mutant (&#226;&#338;&#172;purN, &#226;&#338;&#172;lpdA, &#226;&#338;&#172;glyA, &#226;&#338;&#172;purN&#226;&#338;&#172;lpdA) able <br /> progress (3&#226;&#8364;&#8220;6), provide limited understanding of <br /> metabolic robustness. The conventional attempt study such <br /> robustness relies identification genes reactions               Author contributions: P.-J.K. D.-Y.L. contributed equally work; P.-J.K., D.-Y.L., <br />                                                                                 T.Y.K., K.H.L., H.J., S.Y.L., S.P. designed research; P.-J.K., D.-Y.L., T.Y.K., K.H.L. <br /> indispensable cell. However, universal metabolic pathways                  performed research; P.-J.K., D.-Y.L., T.Y.K., K.H.L., H.J., S.Y.L., S.P. wrote paper. <br /> across species, tricarboxylic acid (TCA) cycle or <br />                                                                                 The authors declare conflict interest. <br /> glycolytic pathways, relatively lethal reactions (1, 7). <br />                                                                                 This article PNAS Direct Submission. <br /> This fact indicates important reaction is, the <br />                                                                                 Freely available online PNAS open access option. <br /> higher chance backup pathway (7). Thus, the <br />                                                                                 &#229;&#8218;&#168;Present  addresses: Department Chemical Biomolecular Engineering, National <br /> functionally important reactions necessarily lethal, and <br />                                                                                  University Singapore, 4 Engineering Drive 4, Singapore 117576; Bioprocessing <br /> this point places limitation reaction-centric approach                  Technology Institute, Agency Science, Technology Research (A*STAR), 20 Biopolis <br /> with studying lethality observing gene deletion effects. In               Way, No. 06-01, Centros, Singapore. <br /> this regard, investigated interplay cellular                **Present address: R &amp; D Center Bioproducts, CJ Corporation, Seoul 157-724, Korea. <br /> robustness underlying metabolism metabolite                    &#226;&#8364;&#160;&#226;&#8364;&#160;To   correspondence addressed. E-mail: hjeong@kaist.ac.kr or <br /> point view, robustness accomplished                  leesy@kaist.ac.kr. <br /> level metabolites, fundamental entities (4,                This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> 8) generated, consumed, recycled metabolic pro-                      0703262104/DC1. <br /> cesses. Constraints-based flux analysis carried                   &#194;&#169; 2007 The National Academy Sciences USA <br />  <br />  <br />  <br /> 13638 &#226;&#8364;&#8220;13642 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 34                                                            www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703262104 <br />  Fig. 1. Characteristics essential nonessential metabolites E. coli metabolism. (a) Metabolic network including central cell envelope <br /> metabolism. Cofactors shown number associated reactions large visual examination. The size circle/box <br /> corresponds flux associated metabolite/reaction, thickness line denotes flux line. (b) Experimental measure <br /> of growth rate relative wild type disrupting genes essential metabolite tetrahydrofolate (blue), nonessential <br /> metabolite 1-deoxy-D-xylulose 5-phosphate (red). The deleted genes: P, purN; L, lpdA; G, glyA; PL, purN/lpdA; PLG, purN/lpdA/glyA; DX, dxs/xylB. (c) Distributions <br />                                                                                                                            k&#226;&#171;&#185;&#226;?&#166; <br /> of degree k essential nonessential metabolites. The vertical axis represents P(k) defined fraction degree, &#229;&#8230;&#176;k&#226;&#171;&#186; &#226;?&#166; P(k&#226;&#172;&#732;)dk&#226;&#172;&#732; &#226;&#171;&#189; f(k) f(k) fraction <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              BIOPHYSICS <br /> of metabolites k &#226;&#171;&#186; &#226;?&#166; k &#226;&#171;&#185; &#226;?&#166;. Such distributions follow power-law P(k) &#226;&#172;&#8364; k&#226;&#171;&#186;&#226;?&#165; &#226;?&#165; &#226;&#171;&#189; 1.97 essential metabolites &#226;?&#165; &#226;&#171;&#189; 3.06 nonessential <br /> ones. (d) The horizontal axis represents number different organisms N, vertical axis represents fraction metabolites conserved <br /> phylogenetically &#226;&#172;&#381;N different organisms. EMC/NMC denotes metabolites E. coli essential/nonessential half growth conditions. The <br /> majority EMC (66.1%) present organisms (&#226;&#177;&#8211;79.3%), contrary case NMC (only 21.2% phylogenetic range). <br />  <br />  <br />  <br /> to survive growth rate changes,                             First, number reactions (degree) participated each <br /> simultaneous deletion of genes                                        metabolite calculated (5). The degree distributions both <br /> (&#226;&#338;&#172;purN&#226;&#338;&#172;lpdA&#226;&#338;&#172;glyA) prevented cell growth completely, indicat-                           essential nonessential metabolites follow a <br /> ing tetrahydrofolate essential cell growth                      power-law distribution broad range degrees (Fig. 1c). <br /> (Fig. 1b). In contrast, 1-deoxy-D-xylulose 5-phosphate iden-                       Not surprisingly, degree distribution essential metabolites <br /> tified nonessential metabolite silico, experimental                        right-skewed compared nonessential ones, indicat- <br /> disruption of reactions producing metabolite                         ing essential metabolites connected reactions <br /> constructing &#226;&#338;&#172;dxs&#226;&#338;&#172;xylB slightly changed increased                         nonessential ones. Indeed, highly connected me- <br /> the growth rate compared wild type (Fig. 1b). These                           tabolites essential; include essential molecules <br /> results indicate multiple gene <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mutants                           cofactors, i.e., H, H2O, ATP, Pi, ADP NAD, which <br /> reactions essential metabolites suffer                             participate &#226;&#172;&#381;76 reactions. Among metabolites having the <br /> detrimental impact cellular function,                          degree &#226;&#172;?3, 34% essential. It <br /> nonessential metabolites negligible influence growth                         noticed nonessential metabolites mani- <br /> capability. Throughout experiments, measured growth                          fest inactive state fluxes metabolites <br /> rates gene deletion mutants relative wild type                   remain zeros. If inactive metabolites nonessential <br /> were consistent silico predictions (Note 1 SI                           metabolites considered, 88.6% active metabolites <br /> Appendix).                                                                             having degree &#226;&#172;?3 essential. <br />                                                                                           Because loss essential metabolites directly threatens cell <br /> Structural Evolutionary Properties Metabolites. We                         viability, expect metabolites essential <br /> investigated inherent network properties essential metab-                       various growth conditions conserved across <br /> olites elucidate correlation structural prop-                       species. We investigated evolutionary conservation the <br /> erty functional behavior metabolite perspective.                          essential metabolites 227 organisms fully sequenced <br />  <br /> Kim et al.                                                                                                  PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 13639 <br />  genomes (SI Table 3). Indeed, metabolites essential most <br /> growth conditions E. coli present different <br /> organisms, showing higher degree conservation than <br /> the nonessential ones evolutionary process (Fig. 1d). <br />  <br /> Stability Metabolite Flux-Sum. To understand robustness of <br /> the cellular metabolism quantitatively metabolite per- <br /> spective, strength fluxes metabolite <br /> was quantified. To end, flux-sum (&#226;&#338;&#189;) metabolite <br /> was defined summation incoming outgoing fluxes <br /> as follows: <br />  <br />              &#226;&#338;&#189;i &#226;&#171;&#189;   &#229;&#8224;&#732; <br />                     j&#226;?&#167;Pi <br />                            Sijvj &#226;&#171;&#189; &#226;&#171;&#186;     &#229;&#8224;&#732; <br />                                          j&#226;?&#167;Ci <br />                                                 Sijvj &#226;&#171;&#189; <br />                                                           1 <br />                                                           2 <br />                                                               &#229;&#8224;&#732; <br />                                                               j <br />                                                                   &#229;&#8230;&#169;Sijvj&#229;&#8230;&#169;, <br />  <br />  <br /> where Sij stoichiometric coefficient metabolite in <br /> reaction j, &#226;?&#175;j flux reaction j. Pi denotes set <br /> of reactions producing metabolite i, Ci denotes set of <br /> reactions consuming metabolite i. Under pseudo-steady state <br /> assumption, &#226;&#338;&#189;i mass flow contributed fluxes <br /> producing (consuming) metabolite i. <br />   The robustness E. coli metabolism examined deter- <br /> mining sensitivity flux-sum genetic perturbation <br /> around given metabolite. It quantified evaluating the <br /> relative fluctuation &#226;&#338;&#189;i response deletion active <br /> nonlethal reactions: <br />  <br />                              &#229;&#8224;&#8216;&#229;&#8230;&#183;&#226;&#338;&#189;i2&#229;&#8230;&#184; &#226;&#171;&#186; &#229;&#8230;&#183;&#226;&#338;&#189;i&#229;&#8230;&#184;2/&#229;&#8230;&#183;&#226;&#338;&#189;i&#229;&#8230;&#184; <br /> where &#229;&#8230;&#183;. . .&#229;&#8230;&#184; denotes average deletion active <br /> nonlethal reactions. It noted interested <br /> in trivial cases deletion inactive reactions. At <br /> low relative fluctuation values, number essential metab- <br /> olites greater nonessential metabolites <br /> (Fig. 2a). This result indicates flux-sums essential <br /> metabolites relatively insensitive genetic perturbation <br /> compared nonessential ones. Indeed, 94.3% of <br /> total metabolites fluctuation range &#226;&#172;?0.0875              Fig. 2. Stability metabolite flow genetic perturbations. (a) The ratio <br /> all essential, nonessential metabolites            essential metabolites metabolites function flux-sum fluctuation. <br /> twenty highest ranked ones relative fluctuations. Thus,            (b) Flux-sum fluctuation versus flux-vector fluctuation essential/ <br /> be concluded essential metabolites resistant internal            nonessential metabolites. (c d) The number essential metabolites (c) or <br /> variations maintaining basal mass flow correspond-             nonessential ones (d) metabolite takes relative change flux-sum <br />                                                                              &#229;&#8230;&#169;&#226;&#338;&#172;&#226;&#338;&#189;i&#229;&#8230;&#169;/&#226;&#338;&#189;i active nonlethal reaction j maximum contribution to <br /> ing metabolite, leading robustness cellular <br />                                                                              flux-sum (maximum &#229;&#8230;&#169;Sij&#226;?&#175;j&#229;&#8230;&#169;/&#226;&#338;&#189;i) removed. The cases (over) the <br /> metabolism.                                                                  diagonal indicate additional flux compensation (loss) reactions. <br />    What mechanism contribute resistance                     (e) Illustration neighboring reactions carbamoyl phosphate (cbp). The <br /> essential metabolites internal perturbations? To explicitly               flux-sum carbamoyl phosphate compensated carbamoyl- <br /> tackle question, monitored individual flux values                phosphate synthase (CBPS) reaction highest flux, carbamate <br /> around essential metabolites genetic perturbations. We                 kinase (CBMK), removed. Thickness arrow represents of <br /> defined flux-vector (&#226;&#338;&#191;  &#225;&#382;&#8225; ) metabolite collection           flux, shown the reaction. <br />                                                         &#225;&#382;&#8225; &#226;&#171;&#189; {Sij&#226;?&#175;j}, <br /> individual fluxes Sij&#226;?&#175;j linked reactions j, &#226;&#338;&#191; <br /> and evaluated flux-vector fluctuation, represents the <br /> relative deviation flux values given metabolite            nonessential ones. To clarify resistance of <br /> upon deleting reactions:                                                     essential metabolites internal perturbations, severe pertur- <br />                                                                              bation conducted deleting reaction contributes <br />                            &#229;&#8224;&#8216;&#229;&#8230;&#183;&#229;&#8230;&#169;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#169;2&#229;&#8230;&#184; &#226;&#171;&#186; &#229;&#8230;&#169;&#229;&#8230;&#183;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#184;&#229;&#8230;&#169;2/&#229;&#8230;&#169;&#229;&#8230;&#183;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#184;&#229;&#8230;&#169;                   flux-sum given essential metabolite. It be <br />                                                                              noted deleted reaction active nonlethal reaction linked <br /> where &#229;&#8230;&#169;. . .&#229;&#8230;&#169; denotes magnitude given vector. Apparently,       metabolite. Fig. 2 c d shows effects kind of <br /> observed variation relevant fluxes metabolite directly         severe perturbation flux-sum changes essential and <br /> contributes change flux-sum metabolite. Scatter        nonessential metabolites, respectively. Most essential metabolites <br /> plot flux-sum fluctuation corresponding                  located diagonal, indicating extent of <br /> flux-vector fluctuation nonessential metabolites clearly shows         flux-sum change flux loss caused deleting most <br /> linear relationship, indicating flux-sum metabolites         contributing reaction. Accordingly, reaction having <br /> is affected perturbed variations individual fluxes (Fig.        relatively high flux eliminated, flux-sum compen- <br /> 2b). On hand, considerable number essential me-               sated fluxes essential metabolite, recovering <br /> tabolites small fluctuation flux-sums despite          flux loss immediately. Remarkably, essential metab- <br /> the increased fluctuations individual flux values (Fig. 2b).          olites, flux loss recovered fluxes other <br /> This result implies flux-sums essential metabolites          reactions, resulting small change flux-sum, <br /> not affected flux variations them, compared               dominant reaction high flux value (6) is <br />  <br /> 13640 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703262104                                                                                                 Kim et al. <br />  Fig. 3. Phenotypic effect attenuating flux-sum level essential metabolites. (a) Changes cell growth rate flux-sum continuously decreases. <br /> The growth rate flux-sum scaled relative wild type. The growth profile type A case metabolite, phosphatidylglycero- <br /> phosphate, type B oxidized thioredoxin, type C adenosine diphosphate. (b c) Cell growth rate relative wild type reducing <br /> the flux-sum half, shown function basal flux-sum level attenuated essential metabolites. The results glucose-minimal aerobic condition <br /> (b) 19 environmental conditions (c) presented. The metabolites b colored way according growth profile. For the <br /> metabolites belong type a, refer Note 4 SI Appendix. <br />  <br />  <br />  <br /> removed (Fig. 2c). Such metabolites include carbamoyl phosphate,                          basal flux-sums, type B high basal flux-sums, and <br /> dUMP, CMP, L-glutamate 5-semialdehyde (Note 3 SI                                   type C ultra-high basal flux-sums (Fig. 3 b c). <br /> Appendix). For example, carbamoyl phosphate key metabolite                           It turns 83.8% essential metabolites belong type A. <br /> involved arginine proline metabolism purine                             These metabolites adjust cell growth rate proportionally to <br /> pyrimidine biosynthesis. The flux-sum carbamoyl phosphate                              flux-sum, act like acclimators affecting cell <br /> could maintained alteration fluxes largest                        growth; acclimator metabolites allow cell growth rate to <br /> flux reaction catalyzed carbamate kinase blocked                             finely adjusted flux-sums, provide <br /> completely (Fig. 2e); carbamoyl-phosphate syn-                          effective control cell growth. The classification essential <br /> thase compensate large flux loss caused knocking-                        metabolites according growth profile flux-sum <br /> out carbamate kinase, resulting recovery 98.9%                           attenuation described in Note 4 SI Appendix. <br /> basal flux-sum. The up-regulation carbamoyl-phosphate syn- <br /> thase response deletion carbamate kinase actually                         Discussion <br /> inferred gene expression profile data (Note 3 SI                              The functional robustness metabolic networks outcome <br /> Appendix). These results suggest maintenance                              long evolutionary process reflects resistance toward <br /> flux-sum serve good indicator metabolic robustness. This                      internal external fluctuations (14&#226;&#8364;&#8220;17). For example, the <br /> fact motivated predict efficiently candidate reactions                    existence alternative pathways flux redistributions implies <br /> activated flux-sum recovery severe <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>gene knockout</span>                        backup pathways possibly activated to <br /> perturbations. Indeed, using stoichio-similarity, developed                        perform function various genetically envi- <br /> an algorithm predict probable reaction                            ronmentally perturbed conditions (3, 18). Such fault-tolerance <br /> recover flux-sum gene knockout perturbation (Note 3                         robustness key <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>cell survival</span> these <br /> in SI Appendix). Therefore, believe cellular robustness                       perturbations. In regard, metabolite-based perspective can <br /> be elucidated functional property metabolic network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      BIOPHYSICS <br />                                                                                           provide new guidelines interpreting cellular robustness. <br /> manifesting resilience essential metabolites                           Essential metabolites substantial cell survival capable of <br /> disturbed flux conditions.                                                                rerouting metabolic fluxes sustaining usage level. <br />                                                                                           This capability essential metabolites leads quite dra- <br /> Attenuation Metabolite Flux-Sum. Essential metabolites play                          matic tolerance wide range internal disturbances. It is <br /> pivotal role cell survival, steadily maintaining mass flow                         possible essential metabolites the <br /> to produce consume metabolites internal                                characteristics presented here. This inaccuracy arise be- <br /> disturbances cell. In sense, metabolite                             cause regulatory mechanisms fully considered in <br /> perspective robustness E. coli provides cellular-level                          current analysis. We examined effects of <br /> fragility: failure maintain flux-sum single essential                     incorporating limited number regulatory mechanisms dur- <br /> metabolite drastically suppress cellular growth. The                            ing study, shown difference (Note 1 in <br /> malfunction multiple genes metabolite                                 SI Appendix). It expected better classification essential <br /> cause critical decrease flux-sum. Especially,                        nonessential metabolites performed genome- <br /> essential metabolites (85%), reducing flux-sum half                         wide regulatory mechanisms incorporated genome- <br /> basal level led suppression growth rate half                        scale flux analysis. <br /> more, 28.9% active nonessential metabolites                                  A number applications envisaged using concept <br /> showed behavior.                                                                     metabolite essentiality. For example, used develop <br />    The effects reducing flux-sum cell growth                               metabolic engineering strategies enhanced production de- <br /> examined next. When flux-sum gradually decreased, <br />                                                                                           sired bioproducts suitably implementing desired flux values. <br /> each essential metabolite exhibited characteristic profile the <br />                                                                                           It used identify new drug targets. Disruption <br /> cell growth rate, belonged types: A, B, and <br />                                                                                           (knockout) multiple nonlethal reactions (genes) an <br /> C, Fig. 3a.&#194;&#167;&#194;&#167; The growth rate sensitive extent of <br />                                                                                           essential metabolite lead fatal cell damage the <br /> flux-sum reduction types A C, type <br />                                                                                           attenuation (knockdown) reactions same <br /> B. Such characteristic growth rate correlated <br />                                                                                           effect. In case treating superbacteria resistant to <br /> with basal flux-sum values; metabolites type A low <br />                                                                                           multiple antibiotics, design drugs inhibit enzymes <br />                                                                                           catalyzing multiple nonlethal reactions essential metab- <br /> &#194;&#167;&#194;&#167;Exceptionally, <br />                ubiquinol-8, ubiquinone-8, L-malate exhibit different growth   olite. Alternatively, synthetic lethal mutations (7, 19, 20) be <br />  profile termed type D. For details, refer Note 4 SI Appendix.              systematically identified enzymes various screening <br />  <br /> Kim et al.                                                                                                 PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 13641 <br />  techniques (21), implemented siRNA anti-sense                  linked reactions producing metabolite M be <br /> techniques.                                                                     consistently taken account, preventing phenotypic effect <br />                                                                                 irrelevant essentiality given metabolite M. We <br /> Materials Methods                                                           scaled resultant change cell growth rate relative the <br /> Constraints-Based Flux Analysis. The genome-scale silico E. coli             growth rate wild type calculating essentiality the <br /> metabolic model iJR904 used slight modifications                       metabolite. When reactions metabolite were <br /> based publicly available information databases                       inactive specific growth condition, considered me- <br /> (9&#226;&#8364;&#8220;12); consists 762 metabolites (including external metab-               tabolite nonessential. Because essentiality metab- <br /> olites) 932 biochemical reactions (including transport pro-                 olites follows clear bimodal distribution (SI Fig. 5), essential <br /> cesses). Cell growth quantified biomass equation                       metabolite easily identified absence leads to <br /> derived drain biosynthetic precursors relevant                  decrease cell growth rate half wild type, <br /> cofactors E. coli biomass appropriate ratios (13).                absence nonessential metabolite minimal or <br /> The stoichiometric relationships metabolites                      effect cell growth. We tried criteria for <br /> reactions genome-scale silico E. coli model                      essential/nonessential metabolites according essentiality <br /> balanced steady-state hypothesis. The resultant bal-                  did difference. <br /> anced reaction model is, however, underdeter- <br /> mined calculating flux distribution insufficient              Construction Gene Knockout Mutant Strains. Mutant strains were <br /> measurements and/or constraints. Thus, unknown fluxes                       constructed one-step gene inactivation method (24). The <br /> within metabolic reaction network calculated linear                 wild-type E. coli W3110 strain transformed pKD46 that <br /> programming-based optimization objective function                    contains phage &#226;?&#173; recombination system. E. coli W3110 cells <br /> maximizing growth rate, subject constraints pertaining               carrying pKD46 transformed electroporation a <br /> to mass conservation, reaction thermodynamics, capacities                   PCR product produced using plasmid pKD3 <br /> as follows:                                                                     pKD4 templates. The PCR product 50- 56-bp <br />                       &#229;&#8224;&#732; <br />                       j&#226;?&#167; J <br />                              Sijvj &#226;&#171;&#189; bi,   &#226;?&#163; j &#226;&#177;&#8226; v j &#226;&#177;&#8226; &#226;?&#164; j, <br />                                                                                 homology upstream downstream DNA immediately <br />                                                                                 adjacent specific target gene knocked-out, also <br />                                                                                 contained Flp recombinase target site (FRT). Recombinant <br /> where Sij represents stoichiometric coefficient metabolite               strains selected growing cells presence chlor- <br /> i reaction j, &#226;?&#175;j flux reaction j, J set reactions,         amphenicol kanamycin, inserted cassette elimi- <br /> and bi net transport flux metabolite i. If metabolite            nated using helper plasmid pCP20. Each knockout mutant <br /> an intermediate, bi zero. &#226;?&#163;j &#226;?&#164;j lower                  confirmed PCR analysis using primers not <br /> upper bounds flux reaction j, respectively. Herein,               region gene deletion. The knockout mutants were <br /> flux irreversible reaction considered positive:                 grown Luria-Bertani (LB) broth LB agar plates 37&#194;&#176;C. <br /> the negative flux signifies reverse direction reaction. <br /> The intracellular fluxes quantified elucidate robust-               Measurement Specific Growth Rate. All strains grown M9 <br /> ness E. coli metabolism response genetic perturbations                 minimal medium containing 5 g/liter glucose determine their <br /> under various environmental conditions (SI Table 2). We                    growth kinetics using Microbiology Reader Bioscreen C <br /> performed simulation additional regulatory constraints                 analyzer (Oy Growth Curves AB Ltd, Helsinki, Finland). De- <br /> (22) optimization scheme, MOMA (minimization                     tailed procedures described Note 1 SI Appendix. <br /> metabolic adjustment) (23), qualitative difference <br /> from results presented here.                                                We thank C.-M. Ghim, S. B. Sohn, H. U. Kim, J. S. Yang, H. Yun <br />                                                                                 providing valuable information linguistic advice S. T. Kim <br /> Characterization Metabolite Essentiality. The metabolite essen-              J.-H. Pak illustrative assistance. The pKD46 plasmid used our <br />                                                                                 gene-disruption experiments kindly provided B. L. Wanner <br /> tiality defined phenotypic effect metabolite M               (Purdue University, West Lafayette, IN). This work supported by <br /> on cell growth consumption rate set zero. All fluxes             Korean Systems Biology Program Grant M10309020000-03B5002-00000 <br /> around metabolite M restricted produce                Ministry Science Technology LG Chem Chair <br /> metabolite, balancing constraint mass conservation                 Professorship. H.J. acknowledges financial support Korea <br /> is relaxed allow nonzero values incoming fluxes               Science Engineering Foundation (KOSEF) Grant R17- <br /> all outgoing fluxes limited zero. As such, metabolites             2007-073-01001-0. <br />  <br />  <br />  1. Gerdes SY, Scholle MD, Campbell JW, Bala &#194;&#180;zsi G, Ravasz E, Daugherty MD,    13. Neidhardt FC, Curtiss R, Ingraham JL, Lin ECC, Low KB, Magasanik B, <br />     Somera AL, Kyrpides NC, Anderson I, Gelfand MS, et al. (2003) J Bacteriol       Reznikoff W, Riley M, Umbarger HE (1996) Escherichia coli Salmonella <br />     185:5673&#226;&#8364;&#8220;5684.                                                                  (ASM Press, Washington, DC). <br />  2. Goebl MG, Petes TD (1986) Cell 46:983&#226;&#8364;&#8220;992.                                  14. Wagner A (2005) Robustness Evolvability Living Systems (Princeton Univ <br />  3. Papp B, Pal C, Hurst LD (2004) Nature 429:661&#226;&#8364;&#8220;664.                              Press, Princeton). <br />  4. Raymond J, Segre` D (2006) Science 311:1764&#226;&#8364;&#8220;1767.                           15. Stelling J, Sauer U, Sz <br /> </body></html>