<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>3286&#226;&#8364;&#8220;3292 Nucleic Acids Research, 1998, Vol. 26, No. 13                                                       &#239;&#8250;&#8482; 1998 Oxford University Press <br />  <br />  <br /> A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> NPS1/STH1, essential gene encoding a <br /> component novel chromatin-remodeling complex <br /> RSC, alters chromatin structure Saccharomyces <br /> cerevisiae centromeres <br /> Eiko Tsuchiya*, Tomoki Hosotani Tokichi Miyakawa <br />  <br /> Department Fermentation Technology, Faculty Engineering, Hiroshima University, Kagamiyama, <br /> Higashi-Hiroshima 739-8527, Japan <br />  <br /> Received February 2, 1998; Revised Accepted May 12, 1998 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                     interactions reconstituted nucleosomes ATP-dependent <br />                                                                              manner. The SNF/SWI complex highly conserved through <br /> The NPS1/STH1 gene encodes nuclear protein                                 evolution: homologs Snf/Swi proteins been <br /> essential progression G2/M phase                               identified eukaryotes, SNF/SWI complexes related <br /> Saccharomyces cerevisiae. Nps1p shares homology                              biochemical functions purified human cells <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> Snf2/Swi2p, subunit protein complex known                          (13&#226;&#8364;&#8220;16) Drosophila melanogaster (17&#226;&#8364;&#8220;19). <br /> as SNF/SWI complex. Recently, Nps1p                              The Snf2/Swi2 subunit SNF/SWI complex contains a <br /> to component protein complex termed RSC (3)                        domain DNA RNA helicases been <br /> essential mitotic growth, function                        shown harbor DNA-dependent ATPase activity (20). This <br /> unknown. We isolated temperature-sensitive mutant                          protein likely play key role chromatin remodeling <br /> allele NPS1, nps1-105, mutation                        activity complex. In Saccharomyces cerevisiae, novel <br /> increases sensitivity thiabendazole (TBZ). At                     SNF/SWI-like complex termed RSC (remodel structure of <br /> restrictive temperature, nps1-105 arrested G2/M                       chromatin) recently purified characterized (3). RSC an <br /> phase MAD1-dependent manner missegregated                             &#226;&#710;&#188;1 Mda complex contains 15 polypeptides. The <br /> the mini-chromosome higher frequency                           largest subunit RSC corresponds Snf2/Swi2p the <br /> wild type cells. The nuclease digestion chromatin                     SNF/SWI complex proved product NPS1/STH1 <br /> of mutant cells revealed mutation causes                        previously isolated (1,2). In addition to <br /> the alteration chromatin structure                             Nps1p/Sth1p, RSC contains related to <br /> centromeres restrictive temperature. The results                      components SNF/SWI complex: Rsc6p (related Swp73p), <br /> suggested that, nps1-105 mutant, impaired                             Rsc8p (related Swi3p) Sfh1p (related Snf5p) (3,21). Like <br /> chromatin structure surrounding centromeres                              SNF/SWI complex, RSC showed DNA-stimulated ATPase <br /> lead impairment kinetochore function                        activity able alter histone&#226;&#8364;&#8220;DNA interactions in <br /> cells arrest G2/M phase spindle-assembly                      reconstituted mononucleosomes ATP-dependent manner <br /> checkpoint system.                                                           (3). Despite fact constituents SNF/SWI <br />                                                                              complex encoded non-essential genes, Nps1p/Sth1p, <br /> INTRODUCTION                                                                 Rsc6p, Rsc8p Sfh1p encoded genes essential for <br />                                                                              mitotic growth. Moreover, LexA-Sth1p fusion protein did not <br /> The DNA eukaryotic genome packed chromatin                        activate transcription conditions analogous <br /> basically consists &#226;&#710;&#188;150 <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> pair (bp) DNA wrapped                 LexA-Snf2 protein functions transcriptional activator (2), <br /> an octamer histone proteins (H2A, H2B, H3 H4). This              indicating RSC complex play wider role the <br /> nucleoprotein structure contributes store huge DNA molecules              SNF/SWI complex. However, physiological function <br /> within nucleus but, time, confers barrier DNA         downstream targets RSC identified. As <br /> metabolism, i.e., transcription, replication, repair recombination.       previously reported, depletion Nps1p arrests cells at <br />   Current studies regulation eukaryotic gene expression            G2/M phase (1). In addition, recent finding the <br /> revealed presence protein complex, SNF/SWI complex,               temperature-sensitive sfh1-1 mutation causes accumulation of <br /> which antagonizes chromatin-mediated repression transcription             cells 2C-DNA content restrictive temperature, indicates the <br /> (reviewed 4&#226;&#8364;&#8220;8). Purified yeast SNF/SWI complex contained               requirement RSC progression G2/M phase (21). <br /> least 11 proteins (9&#226;&#8364;&#8220;12). Biochemical analysis purified                 To explore function Nps1p, isolated temperature- <br /> SNF/SWI complex revealed complex possesses                       sensitive nps1 mutant allele, nps1-105. In report describe <br /> DNA-stimulated ATPase activity destabilize histone&#226;&#8364;&#8220;DNA               characterization mutant. We nps1-105 <br />  <br />  <br /> *To correspondence addressed. Tel/Fax: +81 824 24 7868; Email: etsuchi@ipc.hiroshima-u.ac.jp <br />                                                                                                          3287 <br />  <br />                                                                             Nucleic Acids <br />                                                                             Nucleic       Research,1994, <br />                                                                                     Acids Research, 1998,Vol.  26,No. <br />                                                                                                           Vol.22,  No.113         3287 <br />  <br /> mutation causes alteration chromatin structure    Flow cytometry <br /> centromeric DNA restrictive temperature. Our results <br /> provide clue understanding essential role Nps1p   Flow cytometry analysis performed Coulter Epics Elite <br /> and RSC cell division cycle.                                 ESP. Cells prepared described previously (1). Each FACS <br />                                                                     scan performed 2 &#195;&#8212; 104 cells. <br /> MATERIALS AND METHODS <br />  <br /> Media, yeast strains manipulation                               Digestion chromatin nucleases Southern blot <br />  <br /> Both rich (YPD) minimal (SD) media basic yeast              Cells W303D (a/&#206;&#177;) WTH-1D (a/&#206;&#177; nps1-105/nps105) in <br /> manipulations described (22). YP-galactose medium       mid-log phase rapidly shifted 27 38 C the <br /> prepared YPD, glucose (2%) replaced       addition equal volume medium pre-warmed 45 C. <br /> D-galactose (2%).                                                   After 3 h incubation, cells harvested converted to <br />   Experiments performed yeast W303 (23) strain          sphearoplasts described Szent-Gyorgyi Isenberg (24) <br /> background (ade2-1 leu2-3, 112 his3-11, 15 trp1-1 ura3-1).          following modifications. Sodium azide phenylmethyl- <br />                                                                     sulfonyl fluoride (PMSF) added 20 0.5 mM, <br />                                                                     respectively, immediately prior harvesting cells. Zymolyase <br /> Generation temperature-sensitive NPS1 alleles                    100T (Seikagaku Kogyo, 50 &#194;&#181;g/ml) used instead lyticase. <br />                                                                     Sphearoplasting carried 30 min 34 38 C the <br /> Hydroxylamine mutagenesis pOS31NPS1 (NPS1, LEU2,                 cells heat treatment, respectively. Sphearoplast <br /> CEN3, ARS1) carried according Rose et al. (22). We       suspensions diluted 4 vol buffer A (1.4 M <br /> selected 6 &#195;&#8212; 103 transformants WET-1D (a/&#206;&#177; nps1&#226;&#710;&#8224;::HIS3/          sorbitol, 40 mM N-2-hydroxyethylpiperazine-N&#226;&#8364;&#178;-2 ethanesulfomic <br /> nps1&#226;&#710;&#8224;::HIS3 YCp50NPS1) SC-leucine plates transferred      acid/KOH, pH 7.5, 0.5 mM MgCl2, 20 mM NaN3, 0.5 mM <br /> SC-leucine+5FOA plates. Colonies formed SC-leucine+5FOA          PMSF) sphearoplast concentration estimated measuring <br /> plates screened temperature sensitivity 37 38 C     optical density suspension 260 nm lysis in <br /> by replica plating YPD plates. Plasmids recovered      1% sodium dodecyl sulfate. Sphearoplasts corresponding to <br /> Escherichia coli retransformed WET-1D confirm       OD260 value 150&#226;&#8364;&#8220;250 harvested centrifugation, <br /> temperature-sensitive <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. One plasmid pTH105-1               washed buffer B [1 M sorbitol, 10 mM piperazine-N, <br /> (nps1-105) showed temperature sensitivity 38 C         N-bis 2-ethanesulfonic acid (PIPES), pH 7.5, 0.2 mM CaCl2, <br /> sequenced identify mutation.                                 20 mM NaN3, 0.5 mM PMSF] resuspended 2 ml MNase <br />    The 5.8 kb XbaI fragment containing nps1-105 excised         digestion buffer (10 mM PIPES, pH 7.5, 0.2 mM CaCl2, 1 mM <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> pTH105-1 transformed WET-1 (a nps1&#226;&#710;&#8224;::HIS3               PMSF, 20 mM NaN3) DraI digestion buffer (10 mM Tris&#226;&#8364;&#8220;HCl, <br /> YCp50NPS1) replace genomic NPS1-disruption allele            pH 7.5, 10 mM MgCl2, 50 mM NaCl, 1 mM dithioerythritol, 1 mM <br /> (nps1&#226;&#710;&#8224;::HIS3) nps1-105 mutant allele. Colonies grown       PMSF, 20 mM NaN3) (25). The suspension prewarmed at <br /> on SC+5FOA plate checked histidine auxotrophy        37 C 2 min, MNase (50 U/ml, Boehringer Mannheim) <br /> temperature sensitivity. Three candidate clones nps1-105     added. Samples withdrawn various time points and <br /> mutant transformed pOS31NPS1 check             EDTA added 25 mM stop reaction. For DraI digestion, <br /> suppression temperature sensitivity crossed wild type     various amounts DraI added sphearoplast <br /> strain check segregation temperature-sensitive    suspension (300 &#194;&#181;l) reaction carried 37 C for <br /> phenotype. The temperature-sensitive growth phenotype        30 min. After digestion, DNA deproteinized purified <br /> three clones suppressed introduction pOS31NPS1        described Ling et al. (26). For experiments naked <br /> and temperature-sensitive phenotype segregated 2:2       DNA, sphearoplasts prepared exactly described for <br /> the progenies brought crosses sporulation. One   chromatin digests, immediately nuclease cleavage, <br /> of temperature-sensitive progeny crosses           DNA deproteinized purified. Southern blot analysis was <br /> selected named WTH-1 (nps1-105).                                using fluorescein-labeled probes Gene Images <br />                                                                     labeling detection kit (Amersham). Probes used detecting <br /> Plasmids, primers, plasmid constructions strain                 CEN3 CEN4 592 bp NheI&#226;&#8364;&#8220;BamHI fragment from <br /> constructions                                                       pOS31 contains CEN3 DNA sequence 1018 bp <br />                                                                     SpeI&#226;&#8364;&#8220;StuI fragment YCp50 contains CEN4 DNA <br /> The MAD1 gene cloned PCR using primers A (ttttttagcg-        sequence, respectively. <br /> gatccaagggtttac) B (tagatcaatgttaaggtcgacccgt) the <br /> genomic DNA template. The resulted fragment cut with <br /> BamHI SalI, cloned pUC119 (pUCMAD1). The               Analysis DNA topoisomers <br /> mad1&#226;&#710;&#8224;::TRP1 allele created replacing internal <br /> BglII&#226;&#8364;&#8220;SnaBI fragment MAD1 pUCMAD1 coding amino            The topoisomer distribution yeast 2 &#194;&#181;m circle plasmid <br /> acids 28&#226;&#8364;&#8220;580 BamHI&#226;&#8364;&#8220;StuI fragment YRp7 containing        DNA assayed described previously (27). Topoisomers <br /> the TRP1 gene (pUC mad1&#226;&#710;&#8224;::TRP1). Plasmid pUC mad1&#226;&#710;&#8224;::TRP1            separated electrophoresis 1% agarose gels containing <br /> was digested SmaI SalI introduced W303-1A       2.5 &#194;&#181;g/ml chloroquine detected Southern blot as <br /> WTH-1 effect one-step gene replacement chromo-          described section. The EcoRI&#226;&#8364;&#8220;SnaBI fragment from <br /> somal MAD1 locus. The resultant strains termed W-DM1           YEp24 contains 2 &#194;&#181;m DNA sequence used probe <br /> and WTH-DM1, respectively.                                          endogenous 2 &#194;&#181;m plasmids. <br />  3288 Nucleic Acids Research, 1998, Vol. 26, No. 13 <br />  <br />  <br />  <br />  <br /> Figure 1. Phenotypes nps1-105. Wild type nps1-105 grown to <br /> mid-log phase spotted YPD plates YPD plates containing 100 &#194;&#181;g/ml <br /> of TBZ incubated indicated temperatures 3 days. <br />  <br />  <br />  <br /> Mini-chromosome missegregation assay <br /> The mitotic segregation SUP11 marked mini-chromosomes <br /> (pHM410, SUP11, URA3, CEN4, ARS1) strains YTW-30D <br /> (a/&#206;&#177; ade2-101/ade2-101) YTW-n9D (a/&#206;&#177; ade2-101/ade2-101 <br /> nps1-105/nps1-105) monitored using color colony assay <br /> (28). In assay, pink colonies picked from <br /> color media plates used experiment. Cells grown <br /> to mid-log phase 27 C shifted 38 C (experiment 1) or <br /> added TBZ 60 &#194;&#181;g/ml shifted 34 C (experiment 2). <br />                                                                              Figure 2. The cell cycle arrest nps1-105 restrictive temperature <br /> After treatment (3 h), &#226;&#710;&#188;100 cells spread color media            depends MAD1. (A) Growth viability nps1-105 nps1-105 mad1&#226;&#710;&#8224; <br /> plates brief sonication (5 s). These plates incubated           mutants. Wild type (W303-1A, open square), mad1&#226;&#710;&#8224; (W-DM1, closed square), <br /> for 5 days 25 C overnight 4 C. Three independent              nps1-105 (WTH-1, open circle) nps1-105 mad1&#226;&#710;&#8224; (WTH-DM1, closed <br /> assays performed monitor segregation pHM410.                  circle) cells grown early-log phase 25 C shifted 38 C. At the <br />                                                                              times indicated figure, cell counts obtained, diluted appropriate <br />                                                                              numbers plated YPD medium 25 C. Viability presented the <br /> RESULTS                                                                      percentage cells form colonies 25 C. Values average of <br />                                                                              independent determinations; standard deviations 30%. <br /> Isolation NPS1 temperature-sensitive mutant                             (B) Flow-cytometric analysis DNA content nps1-105 nps1-105 <br />                                                                              mad1&#226;&#710;&#8224; cells. Cultures analyzed (A) stained propidium iodide and <br /> In order investigate role NPS1, generated conditional           analyzed DNA content using flow cytometer. Results plotted the <br /> NPS1 alleles employing hydroxylamine-directed vitro                    relative DNA cell versus number cells. <br /> mutagenesis. Mutagenized plasmids introduced NPS1 <br /> deletion strain carrying wild type NPS1 plasmid plasmid <br /> shuffling. One plasmid allowed growth 30 38 C             carbamate (MBC) benomyl (Fig. 1, data MBC and <br /> was chosen analysis. The mutant allele designated            benomyl shown). The nps1-105 mutant grow on <br /> as nps1-105. This allele replaced genomic NPS1                  YPD plate containing &gt;110 70 &#194;&#181;g/ml TBZ permissive <br /> locus WET-1 (nps1&#226;&#710;&#8224;::HIS3) step gene replacement                 (&lt;30 C) semipermissive temperature (35 C), respectively. <br /> make nps1-105 mutant strain (Materials Methods). Upon                The wild type strain, hand, grow YPD <br /> shift 38 C, nps1-105 mutant stopped growth                    plate containing 160 &#194;&#181;g/ml TBZ 30 35 C. <br /> accumulation large-budded cells possessing single nucleus              We tested sensitivity nps1-105 hydroxyurea, <br /> a 2C-DNA content (Fig. 2, data cellular nuclear                      methylmethane sulphonate, cycloheximide trifluoperazine. <br /> morphologies shown). These phenotypes consistent                No difference observed wild type nps1-105 in <br /> with cells carrying galactose-inducible NPS1               sensitivity drugs, indicating mutant is <br /> shifting glucose medium (1). DNA sequence determination                   specifically sensitive microtubule-destabilizing agents (data <br /> revealed nps1-105 carries single base substitution             shown). Both temperature TBZ sensitivities nps1-105 <br /> the open reading frame (ORF): G A position 2288 resulted               cosegregated haploid progenies NPS1/nps1-105 <br /> in amino acid substitution cysteine tyrosine position           diploid sporulation, complemented introduction <br /> 763. The Cys763 Nps1p conserved Snf2p located               wild type NPS1 low-copy-number plasmid (data shown). <br /> within near conserved helicase sequence motif. In case of <br /> Nps1p, amino acid residue exists near essential Lys792              MAD1 disruption causes failure cell cycle arrest and <br /> (1) serve ATP binding site association              loss viability nps1-105 restrictive temperature <br /> G-rich motif 20 amino acids upstream Cys763. <br />                                                                              The observations indicated possibility NPS1 <br /> nps1-105 shows increased sensitivity                                      affects organization and/or stability mitotic spindles. <br /> microtubule-destabilizing agents                                             Spindle malformation detected spindle-assembly checkpoint <br />                                                                              control including MAD BUB gene products components <br /> In addition temperature-sensitive growth phenotype,                   (reviewed 29). If Nps1p involved processes, cell <br /> nps1-105 mutant showed increased sensitivity microtubule-                 cycle arrest nps1-105 elevated temperature interfered with <br /> destabilizing agents thiabendazole (TBZ), methylbenz-                loss spindle-assembly checkpoint function. To test this <br />                                                                                                                                    3289 <br />  <br />                                                                                                Nucleic Acids <br />                                                                                                Nucleic       Research,1994, <br />                                                                                                        Acids Research, 1998,Vol.  26,No. <br />                                                                                                                              Vol.22,  No.113                    3289 <br />  <br />  <br />  <br />  <br /> Figure 3. Effects nps1-105 mutation chromatin structures. (A) MNase digestion <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> wild type nps1-105 chromatin. Wild type (WT) and <br /> nps1-105 cells grown mid-log phase 27 C, shifted 38 C 3 h converted sphearoplasts. Sphearoplasts (125 OD260 U/ml) digested with <br /> 50 U/ml MNase 37 C times indicated DNA deproteinized purified. DNA samples subjected electrophoresis 1.3% agarose gel <br /> and stained ethidium-bromide. (B) MNase-sensitive sites centromeric chromatin chromosome III. Sphearoplasts (75 OD260 U/ml) prepared as <br /> described (A) digested 50 U/ml MNase 80 s 37 C (lanes s). MNase digestion naked DNA (lanes n) carried 1 U/ml MNase <br /> for 50 s 37 C. After deproteinization purification, DNA samples subsequently digested BamHI subjected Southern blot analysis fluorescein <br /> labeled NheI&#226;&#8364;&#8220;BamHI fragment CEN3. The sizes molecular weight markers (M) indicated bp. At left (B), schematic map chromosome <br /> region surrounding CEN3 cited. BamHI cleavage site indicated closed circle. I, II III relative position sequence elements CDEI, CDEII <br /> and CDEIII, respectively. (C) DraI accessibility CEN3 CEN4 chromatin increases nps1-105 shift 38 C. Sphearoplasts (90 OD260 U/ml) prepared <br /> from wild type nps1-105 cells treated 3 h 38 C digested DraI 30 min 37 C. Genomic map DNA regions surrounding CEN3 CEN4 <br /> is schematically indicated panel positions probes used Southern hybridization. Relative positions CDEI CDEIII denoted by <br /> black shadowed boxes, respectively, CDEII lies elements. DraI BamHI sites indicated bars closed circle, respectively. <br /> Arrowheads 1 2 right CEN4 panel indicate 2400 2100 bp fragments, respectively. The sizes molecular weight markers (M) indicated in <br /> bp. (D) Superhelicity endogenous 2 &#194;&#181; plasmid DNA wild type nps1-105 cells. Wild type (lanes 1 3) nps1-105 (lanes 2 4) cells grown at <br /> 27 C early-log phase treated 38 C 3 h. DNA samples prepared cells (lanes 1 2) heat treatment (lanes 3 4) were <br /> separated agarose gels containing 2.5 &#194;&#181;g/ml chloroquine subjected Southern blot analysis. <br />  <br />  <br />  <br /> possibility, disrupted MAD1 gene nps1-105 mutant.                      micrococcal nuclease (MNase) DNA deproteinized and <br /> The nps1-105 mad1&#226;&#710;&#8224; double mutant arrest G2/M                            digested BamHI. Southern blot analysis DNA sample <br /> phase 5 h incubation 38 C (Fig. 2). In addition, viability                  wild type cells CEN3 probe showed 250 bp <br /> of double mutant cells decreased dependence                          centromere-core region DNA occurring 350 600 bp <br /> incubation period 38 C. In contrast, notable decrease                       BamHI recognition site protected MNase <br /> viability observed wild type, nps1-105 mad1&#226;&#710;&#8224;                             digestion (Fig. 3B). In region flanking CEN3 core region, an <br /> mutants 8 h incubation 38 C.                                                   extensive series MNase cutting sites seen &#226;&#710;&#188;160 bp <br />                                                                                      intervals described Bloom Carbon (30). In wild type <br /> The nps1-105 mutation alters chromatin structure                                 strain, cutting pattern altered 3 h incubation <br /> around centromere                                                                38 C. The cutting pattern DNA sample nps1-105 <br /> Recent finding RSC exhibits activity perturb nucleosome                   cells grown 27  indistinguishable wild type, <br /> structure vitro (3) results described above, suggested                    number faint cutting sites seen wild type <br /> the possibility relevance Nps1p chromatin                           chromatin lanes appeared centromere core the <br /> structure centromeres. Bloom Carbon showed yeast                         flanking region nps1-105 incubated 38 C lanes <br /> centromeres packaged unique chromatin structure                           (indicated arrowheads). As demonstrated Figure 3A, <br /> composed nuclease-resistant core flanked ordered arrays                      overall MNase cutting pattern chromatin 38 C-treated <br /> of nucleosomes (30). We examined chromatin structure                          wild type mutant cells showed change, indicating the <br /> CEN3 flanking sequences nps1-105 3 h incubation                      increase nuclease accessibility heat-treated nps1-105 <br /> 38 C. Permeabilized sphearoplasts partially digested                       chromatin specific region CEN3. <br />  3290 Nucleic Acids Research, 1998, Vol. 26, No. 13 <br />  <br />    Next, asked structural changes chromatin occur      topoisomer distribution introduced heat treatment <br /> other centromere regions. The perturbation structural           wild type mutant, suggesting alteration <br /> integrity centromere DNA examined increase          centromere structure mutant result loss <br /> in nuclease accessibility chromatin restriction enzyme sites    nucleosomes (Fig. 3D). <br /> within internal centromere region. Saccharomyces cerevisiae <br /> centromere DNA contains elements, CDEI, CDEII                The nps1-105 mutation affects transmission fidelity of <br /> CDEIII (31&#226;&#8364;&#8220;33). CDEI CDEIII 8 25 bp DNA                    mini-chromosome <br /> elements partial dyad symmetry, respectively. CDEII a <br /> 76&#226;&#8364;&#8220;86 bp region highly rich A T contains       Mutations CEN DNA histone H4 depletion histones <br /> DraI recognition site (TTTAAA). Both CEN3 CEN4 contain             H4 H2B affect chromatin structure centromere and <br /> three closely spaced DraI sites CDEII.                    cause chromosome missegregation (36&#226;&#8364;&#8220;41). We asked the <br />    Permeabilized sphearoplasts prepared cells treated          nps1-105 mutation affects chromosome segregation monitoring <br /> 38 C digested increasing DraI DNA           rate loss (1:0 segregation) non-disjunction (2:0 <br /> was purified deproteinization. For analysis CEN3,         segregation) events nps1-105/nps1-105 ade2-101/ade2-101 <br /> purified DNA samples digested BamHI. If DraI             diploid carrying SUP11 mini-chromosome. In assay, loss <br /> sites CEN3 CDEII accessible enzyme,              non-disjunction events occurred division after <br /> BamHI digestion deproteinization rise       plating result appearance half-pink half-red and <br /> 480 bp band. As shown Figure 3C CEN3, 480 bp cutting            half-white half-red colonies, respectively (42,43). <br /> band appeared nps1-105 chromatin lanes wild type            The rates 1:0 2:0 segregation wild type cells that <br /> ones. The result consistent MNase digestion     grown 25 C 0.1% rates remained <br /> Figure 3B.                                                             unaltered 3 h treatment 38 C. On hand, 2:0 <br />    In case CEN4, additional DraI sites located     segregation rate noticeably higher mutant cells that <br /> to left right sides flanking core centromere elements      grown 25 C rate increased heat <br /> at distance 626 611 bp, respectively. So, case,     treatment (Table 1, experiment 1). The rate 1:0 segregation in <br /> the accessibility DraI sites CDEII    mutant somewhat higher wild type not <br /> sites flanking region, accessibility sites   notably increased heat treatment. The results indicating <br /> in flanking region rise cutting fragment &#226;&#710;&#188;600    nps1-105 mutation affects segregation plasmid. <br /> or 1237 bp, respectively (Fig. 3C, CEN4). Neither 600 1237 bp         We carried assay cells treated TBZ at <br />                                                                        semipermissive temperature (34 C). Although rates 1:0 and <br /> band observed wild type chromatin, indicating DraI <br />                                                                        2:0 segregation mutant cells unaltered absence of <br /> sites CDEII site flanking region are <br />                                                                        TBZ, presence drug 3 h incubation increased <br /> protected digestion. In contrast, mutant chromatin <br />                                                                        rate 2:0 segregation (Table 1, experiment 2). The same <br /> lanes, appearance 600 bp band apparent, showing the <br />                                                                        treatment induced little increase 2:0 segregation rate wild <br /> digestion DraI sites CDEII site     type cells, suggesting missegregation mini-chromosome <br /> flanking region. In Figure 3C CEN4 panel, 2400 2100 bp             mutant caused failure reassembly of <br /> fragment (arrowheads 1 2, respectively) appeared           microtubules. <br /> wild type- nps1-105 nps1-105 chromatin lanes, respectively. <br /> In analysis, DraI site flanking right CEN4 core <br /> elements susceptible digestion wild type     DISCUSSION <br /> mutant chromatin (data shown). Database searches DraI <br />                                                                        We isolated temperature-sensitive allele NPS1, nps1-105, and <br /> cleavage sites DNA region neighboring CEN4 revealed that <br />                                                                        mutation causes alteration chromatin <br /> two additional sites located 2423 2111 bp upstream of <br />                                                                        structure surrounding centromeric DNA region elevated <br /> the right-most DraI site Figure 3C. Therefore, proximal         temperature. As centromere chromatin structure known to <br /> DraI site protected digestion wild type     maintained yeast cell cycle (39), this <br /> in mutant. The results chromosome IV,        alteration nps1-105 result G2/M arrest <br /> 1.5 kb DNA region left centromere core elements     mutant cells. Therefore, suggested NPS1 <br /> may assembled phased nucleosome array structural         mutant, impaired chromatin structure centromeres lead to <br /> integrity centromere core flanking region              loss kinetochore function cells arrest G2/M phase <br /> perturbed nps1-105 treatment 38 C. With wild type      spindle-assembly checkpoint system. Several lines of <br /> or mutant cells grown 27 C, increase sensitivity      evidence support idea. First, nps1-105 mutant is <br /> to DraI observed CEN3 CEN4 DNA (data               specifically sensitive microtubule-destabilizing agents (Fig. 1). <br /> shown).                                                                It known mutation genes encoding kinetochore <br />    Saunders et al. (34) reported nucleosome depletion alters      components causes elevated sensitivity benomyl (44&#226;&#8364;&#8220;47). <br /> the chromatin structure centromeres. A loss nucleosomes          Second, nps1-105 cell-cycle-arrest phenotype MAD1 <br /> increases positively supercoiled species closed circular       dependent (Fig. 2). Finally, nps1-105 mutation causes an <br /> DNA. To ask perturbed structural integrity centromere        increase frequency non-disjunction mini-chromosome <br /> chromatin nps1-105 mutant results loss              (Table 1). Non-disjunction events resulted segregation <br /> nucleosomes, analyzed distribution topoisomers            replicated sister chromatids pole anaphase. <br /> naturally occurring yeast 2 &#194;&#181;m plasmid. This plasmid exists       A subset non-disjunction events caused failure the <br /> covalently closed circular mini-chromosome assembled              centromeres paired sister chromatids interact properly with <br /> nucleosomal chromatin (reviewed 35). No change               microtubules. <br />                                                                                                                                       3291 <br />  <br />                                                                                                    Nucleic Acids <br />                                                                                                    Nucleic       Research,1994, <br />                                                                                                            Acids Research, 1998,Vol.  26,No. <br />                                                                                                                                  Vol.22,  No.113            3291 <br />  <br />                 Table 1. Analysis NPS1 function mitotic chromosome transmission half-sectored coloniesa <br />  <br />                  Strain             Heat              TBZ           Colonies (%)                                                    Total no. of <br />                                     treatment (h)                   P+W       R              W/P         P/R         W/R            colonies scored <br />                                                                                                          (1:0)       (2:0) <br />                  Experiment 1       38 C              60 &#194;&#181;g/ml <br />                  WT                 0                 &#226;&#8364;&#8220;             92           7.5         0.1         0.1         0.1            1628 <br />                                     3                 &#226;&#8364;&#8220;             87          12.8         0.1         0.2         0.1            1527 <br />                  nps1-105           0                 &#226;&#8364;&#8220;             84          14.0         0.9         0.3         0.6            1246 <br />                                     3                 &#226;&#8364;&#8220;             82          14.3         1.9         0.4         1.5            2372 <br />                  Experiment 2       34 C <br />                  WT                 0                 &#226;&#8364;&#8220;             87          12.4         0.1         0.1         0.1            1373 <br />                                     3                 &#226;&#8364;&#8220;             86          13.9         0.1         0.0         0.2            1900 <br />                                     3                 +             87          13.8         0.1         0.1         0.3            1141 <br />                  nps1-105           0                 &#226;&#8364;&#8220;             86          12.6         0.5         0.1         0.5            1518 <br />                                     3                 &#226;&#8364;&#8220;             85          13.2         0.5         0.1         0.7            1995 <br />                                     3                 +             82          13.9         1.9         0.3         1.6            1278 <br />  <br />                 anps1-105 (YTW-n9D, a/&#206;&#177; ade2-101/ade2-101 nps1-105/nps1-105) isogenic wild type (WT) strain (YTW-30D), harboring <br />                 pHM410 (CEN4 SUP11 URA3) subjected heat treatment. Cells removed cultures indicated times <br />                 plated color media plates. The cells allowed form colonies incubation 25 C 5 days. The percentage <br />                 colony type calculated presented table. P, W R indicate pink, white red colonies, W/P, P/R <br />                 W/R indicate half-white half-pink, half-pink half-red half-white half-red sectored colonies, respectively. Standard deviation was <br />                 10%. <br />  <br />  <br />    To explain phenotypes nps1-105 mutant,                      (calcinosis, Raynaud&#226;&#8364;&#8482;s phenomenon, esophageal dysmotility, <br /> at possibilities. The Nps1p plays direct                   sclerodactyly telagiectasia) shown centromere- <br /> role organization and/or maintenance centromere                          specific protein domain resembling histone H3 (54). <br /> chromatin structure independently RSC complex. At                                 CSE4 isolated screen genes cause increase in <br /> present, genes encoding RSC components identified                             rate loss artificial chromosome bearing partial loss <br /> including NPS1 (3,21,48). Analysis conditional mutant                           function mutation CDEII essential for <br /> alleles genes required understanding role                       mitotic growth (55). The phenotypes temperature-sensitive <br /> RSC complex chromatin structure. The second model                           CSE4 allele, cse4-1, resemble nps1-105: cse4-1 <br /> Nps1p functions RSC complex transcriptional                               mutant arrest predominance large budded cells <br /> regulator resembling SNF/SWI complex genes essential                           containing single G2 nuclei short bipolar mitotic spindles at <br /> the assembly certain chromatin structure centromeres.                            elevated temperature shows increased non-disjunction <br /> Previous studies indicate depletion histone H2B H4                          frequency chromosome bearing mutant centromere DNA <br /> alters chromatin structure centromeres causes G2 arrest                         sequence (53). Although yeast centromeric DNA packed <br /> (49&#226;&#8364;&#8220;51). Nonetheless, phenotypes reported loss                      specialized nucleosomes, question specialized <br /> histones different nps1&#226;&#8364;&#8220;105 points. First,                 nucleosomes properly localized centromeres not <br /> depletion histones caused elevated sensitivity MNase                   answered yet. Fundamentally, binding histones DNA is <br /> overall chromatin, nps1-105 mutation did (Fig. 3A).                      dependent specific DNA sequence. In nuclei growing <br /> Second, reduction superhelical density covalently closed                     cells, conventional nucleosomes much <br /> circular plasmid introduced loss histones                 abundant specialized nucleosomes. In case of <br /> by defect nps1-105 (Fig. 3D). Finally, cell cycle arrest                 human cells, targeting CENP-A centromeres requires that <br /> the histone depletion irreversible: cells carrying inducible                   CENP-A expression uncoupled histone H3 synthesis <br /> allele histone H2B H4 lost viability shift                    S phase: CENP-A mRNA accumulates later cell <br /> repressed conditions 2 h. On hand, nps1-105                       cycle histone H3, peaking G2 (56). On hand, the <br /> mutants remained viable 8 h restrictive condition                 mRNA CSE4 reported present relatively constant levels <br /> the MAD1 gene functional (Fig. 2). Accordingly, Nps1p                     cell cycle (53). In addition, Cse4p <br /> to affect expression histone genes. A possibility                possesses DNA-binding activity higher affinity to <br /> is Nps1p acts component RSC complex                                centromere DNA obscure. The simplest scenario explain the <br /> organization modulation centromeric chromatin structure.                     role RSC complex required recruiting <br /> The yeast centromeric DNA packed unique chromatin                            specialized nucleosomes, organizing and/or maintaining the <br /> structure (30). In addition, nuclease-protected region                  unique chromatin structure distinct region genome. <br /> centromere highly resistant ionic conditions known                   In regard, biochemical genetic properties SFH1, gene <br /> disrupt loose DNA&#226;&#8364;&#8220;protein complexes (52) thought                           encoding component RSC, (21). A <br /> to contain specialized nucleosomes. These nucleosomes                              temperature-sensitive SFH1 mutation, sfh1-1, causes G2/M arrest <br /> suggested contain Cse4p, yeast homolog human                               elevated temperature similar nps1-105. The Sfh1 <br /> kinetochore protein, CENP-A, place histone H3 (53). CENP-A                       protein phosphorylated G1 phase dephosphorylated <br /> was identified using sera patients suffering CREST                          progression S phase. Because Sfh1p required the <br />  3292 Nucleic Acids Research, 1998, Vol. 26, No. 13 <br />  <br /> progression G2/M, phosphorylation Sfh1p                      14 Kown,H., Imbalzano,A.N., Khavari,P.A., Kingston,R.E. Green,M.R. <br /> suggested negatively regulate Sfh1p function: thus, protein                (1994) Nature, 370, 477&#226;&#8364;&#8220;481. <br />                                                                                15 Wang,W., C&#195;&#180;t&#195;&#169;,J., Xue,Y., Zhou,S., Khavari,P.A., Bigger,S.R., <br /> is likely activated dephosphorylation S phase. The                Muchardt,C., Kalpana,G., Goff,S.P., Yaniv,M., Workman,J.L., et al. (1996) <br /> organization newly synthesized DNA properly highly                        EMBO J., 15, 5370&#226;&#8364;&#8220;5382. <br /> ordered chromatin structure occurs tight link                    16 Wang,W., Xue,Y., Zhou,S., Kuo,A., Cairns,B.R., Tijan,R. and <br /> synthesis S phase.                                                     Crabtree,G.R. (1996) Genes Dev., 10, 2117&#226;&#8364;&#8220;2130. <br />    However, results exclude possibility                      17 Tamkun,J.W., Deuring,R., Scott,M.P., Kissinger,M., Pattatucci,A.M., <br /> requirement RSC transcriptional regulation. Cairns et al.                  Kaufman,T.C. Kennison,J.A. (1992) Cell, 68, 561&#226;&#8364;&#8220;572. <br />                                                                                18 Tsukiyama,T. Wu,C. (1995) Cell, 83, 1011&#226;&#8364;&#8220;1020. <br /> reported RSC 10-fold abundant                       19 Tsukiyama,T., Daniel,C., Tamkun,J. Wu,C. (1995) Cell, 83, 1021&#226;&#8364;&#8220;1030. <br /> SNF/SWI complex (3). Immunofluorescent localization                         20 Laurent,B.C., Treich,I. Carlson,M. (1993) Genes Dev., 7, 583&#226;&#8364;&#8220;591. <br /> Nps1p anti-Nps1p antibody expressing Nps1-GFP                       21 Cao,Y., Cairns,B.R., Kornberg,R.D. Laurent,B.C. (1997) Mol. Cell. Biol., <br /> fusion protein revealed protein concentrated              17, 3323&#226;&#8364;&#8220;3334. <br /> specific region nucleus localized entire region            22 Rose,M.D., Winston,F. Hieter,P. (1990) Methods Yeast Genetics. <br /> stainable DAPI (1; E.Tsuchiya, unpublished result). The                      Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. <br />                                                                                23 Rothstein,R. (1983) Methods Enzymol., 101, 202&#226;&#8364;&#8220;211. <br /> observation indicates Nps1p RSC functions              24 Szent-Gyorgyi,C. Isenberg,I. (1983) Nucleic Acids Res., 11, 3717&#226;&#8364;&#8220;3736. <br /> organization centromere chromatin structure                 25 Veit,B.E. Fangman,W.L. (1985) Mol. Cell. Biol., 5, 2190&#226;&#8364;&#8220;2196. <br /> organization chromatin region transcriptional             26 Ling,X., Harkness,T.A.A., Schultz,M.C., Fisher-Adams,G. and <br /> regulation. The analysis questions progress                 Grunstein,M. (1996) Genes Dev., 10, 686&#226;&#8364;&#8220;699. <br /> laboratory. It spt (suppressor Ty)           27 Saavedra,R.A. Huberman,J.A. (1986) Cell, 45, 65&#226;&#8364;&#8220;70. <br />                                                                                28 Koshland,D. Hieter,P. (1987) Methods Enzymol., 155, 351&#226;&#8364;&#8220;372. <br /> mutations, spt4-138 spt6-140, ctf (chromosome <br />                                                                                29 Rudner,A.D. Murray,A.W. (1996) Curr. Opin. Cell. Biol., 8, 773&#226;&#8364;&#8220;780. <br /> transmission fidelity) phenotype thought cause altered              30 Bloom,K.S. Carbon,J. (1982) Cell, 29, 305&#226;&#8364;&#8220;317. <br /> chromatin structure centromeres (57). The spt mutants                  31 Clark,L. Carbon,J. (1980) Nature, 287, 504&#226;&#8364;&#8220;509. <br /> identified extragenic suppressors transcriptional defects                32 Fitzgerald-Hayes,M., Clark,M. Carbon,J. (1982) Cell, 29, 235&#226;&#8364;&#8220;244. <br /> caused Ty Ty delta element insertions 5&#226;&#8364;&#178; regions           33 Hegemann,J.H. Fleig,U.N. (1993) Bioessays, 15, 451&#226;&#8364;&#8220;460. <br /> HIS4 LYS2 genes (58,59). Spt4p, Spt5p Spt6p function                34 Saunders,M.J., Yeh,E., Grunstein,M. Bloom,K. (1990) Mol. Cell. Biol., <br />                                                                                   10, 5721&#226;&#8364;&#8220;5727. <br /> a complex (60) repress transcription (61). There might <br />                                                                                35 Broach,J.R. (1981) In Strathern,J.N., Jones,E.W. Broach,J.R. (eds), <br /> be protein complexes required organization and/or                 The Molecular Biology Yeast Saccharomyces: Life Cycle and <br /> alteration hi <br /> </body></html>