<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Luminescence Resonance Energy Transfer-Based High-Throughput <br />       Screening Assay for Inhibitors of Essential Protein-Protein <br />               Interactions in Bacterial RNA Polymerase <br />  <br />              The binding of sigma factors <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> core RNA polymerase is essential for the specific initiation of transcription <br />           in eubacteria and is thus critical for cell growth. Since the responsible protein-binding regions are highly <br />           conserved among all eubacteria but differ significantly <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> eukaryotic RNA polymerases, sigma factor binding <br />           is a promising target for drug discovery. A homogeneous assay for sigma binding to RNA polymerase <br />           (Escherichia coli) based on luminescence resonance energy transfer (LRET) was developed by using a eu- <br />           ropium-labeled &#226;&#65533;&#180;70 and an IC5-labeled fragment of the &#226;&#65533;&#164;&#226;&#180;&#8226; subunit of RNA polymerase (amino acid residues <br />           100 through 309). Inhibition of sigma binding was measured by the loss of LRET through a decrease in IC5 <br />           emission. The technical advances offered by LRET resulted in a very robust assay suitable for high-throughput <br />           screening, and LRET was successfully used to screen a crude natural-product library. We illustrate this <br />           method as a powerful tool to investigate any essential protein-protein interaction for basic research and drug <br />           discovery. <br />  <br />  <br />    The need for new antimicrobial drugs has become an obvi-                   tures of Thermus aquaticus holoenzyme (14, 22). Each protein <br /> ous scientific challenge since inappropriate use and clinical                 has a remarkably conserved (&#226;&#172;&#381;80%) amino acid sequence (11, <br /> conditions have favored selection for strains resistant to an                 13) among all known eubacteria within these regions. These <br /> increasing number of antibiotics. In order to accelerate the                  homologies suggest a highly conserved structure and function <br /> pace at which we compete with nature, high hopes lie in the                   within the holoenzyme form of RNAP, which is crucial for the <br /> exploitation of recent advances in genomic and proteomic re-                  correct initiation of transcription. Any inhibitor of this inter- <br /> search. Due to the increasingly detailed structural and mech-                 action can thus be expected to be a broad-spectrum antibiotic. <br /> anistic information about proteins involved in the central pro-               This is supported by the fact that the induction of expression of <br /> cesses of the life cycle of a cell, such as its replication,                  the &#226;&#65533;&#164;&#226;&#172;&#732; fragment (residues 260 through 309) stops cell growth in <br /> transcription, and translation, rationally designed assays can be             E. coli. (2). No &#226;&#65533;&#180;70 homologue has been found in archaea and <br /> developed to find inhibitors of very specific and vulnerable                  mammalian cells except for sigma factors in mitochondria (21) <br /> targets within these biochemical machines. With this paper, we                and chloroplasts (1). However, these do not show a significant <br /> illustrate the development of a luminescence resonance energy                 homology to their bacterial counterparts. This fact implies that <br /> transfer (LRET)-based high-throughput screen for antimicro-                   there is very little chance of a potential new antibiotic inter- <br /> bial drugs based on the identification of a crucial protein-                  fering with eukaryotic RNAP assembly, which could otherwise <br /> protein interaction in RNA polymerase (RNAP).                                 lead to serious side effects if it were used as a drug. <br />    Like the ribosome and DNA polymerase, the bacterial tran-                     In order to screen for inhibitors of RNAP assembly with <br /> scription machinery appears to offer an attractive target for                 sigma, an assay simpler, faster, and more reliable than the ones <br /> drug discovery and rational drug design. Any inhibitor of the                 currently available (6) was developed for the formation of the <br /> core RNAP assembly with a major sigma factor to form the                      &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; complex of E. coli. We decided to use LRET, a recent <br /> holoenzyme would inhibit the initiation of transcription and                  modification of fluorescence resonance energy transfer <br /> therefore prevent the growth and eventually the <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> of a                 (FRET) that can create the desired signal upon protein bind- <br /> cell. Researchers have previously identified by biochemical                   ing (15, 17, 18). The more general term &#226;&#8364;&#339;luminescence&#226;&#8364;&#65533; in- <br /> methods the regions in Escherichia coli RNAP (a coiled-coil                   stead of &#226;&#8364;&#339;fluorescence&#226;&#8364;&#65533; (as in FRET) indicates that lanthanide <br /> region between amino acid residues 260 and 309 in the &#226;&#65533;&#164;&#226;&#172;&#732;                      emission is technically not considered fluorescence (i.e., arising <br /> subunit of E. coli) (3, 4) and its major sigma factor &#226;&#65533;&#180;70 (region             from a singlet-to-singlet transition). The details of LRET have <br /> 2.2) that are largely responsible for sigma-core interactions                 been elegantly described in recent reviews by Selvin (8) and <br /> (10, 12). These results have now been confirmed by the struc-                 Heyduk (16) and will be covered only briefly here. A quanti- <br />                                                                               tative description of the effect is based on the Fo &#194;&#168;rster theory <br />                                                                               that describes the decrease of energy transfer as inversely pro- <br />   * Corresponding author. Mailing address: McArdle Laboratory for <br /> Cancer Research, University of Wisconsin&#226;&#8364;&#8221;Madison, 1400 University <br />                                                                               portional to the sixth power of the distance between the two <br /> Ave., Madison, WI 53706. Phone: (608) 263-2635. Fax: (608) 262-2824.          dyes and is applicable to FRET and LRET. The advantages of <br /> E-mail: burgess@oncology.wisc.edu.                                            LRET result from the prolonged fluorescent lifetimes of the <br />  <br />                                                                        1492 <br />  VOL. 69, 2003                                                                                       LRET ASSAY FOR DRUG DISCOVERY                               1493 <br />  <br />  <br /> lanthanide-based donor fluorophores like Eu and Tb (more                              LRET assay to test for the inhibition of protein-protein interaction of labeled <br /> than microseconds up to several milliseconds) compared to the                      &#226;&#65533;&#180;70 and &#226;&#65533;&#164;&#226;&#180;&#8226;. The LRET assay was performed in NTG buffer (200-&#226;&#65533;&#174;l total <br />                                                                                    volume) plus 2.5% DMSO (when library samples were used) with 40 nM &#226;&#65533;&#180;70* <br /> short lifetime of most organic-based fluorophores used in                          (labeled protein) and 30 nM &#226;&#65533;&#164;&#226;&#172;&#732;* fragment (labeled protein). All assays were <br /> FRET, like Cy5 or IC5 (picoseconds up to a microsecond), so                        performed with refolded &#226;&#65533;&#180;70*. In the standard assay, the &#226;&#65533;&#164;&#226;&#172;&#732;* fragment was <br /> that LRET offers mainly technical advantages over FRET.                            added in denatured form directly into the reaction mixture, which, by 10-fold <br />    Heyduk and coworkers used LRET to measure DNA bind-                             dilution of GuHCl to 0.15 M in the final assay, allowed instant refolding. This was <br />                                                                                    done to prevent precipitation of the refolded labeled protein upon storage before <br /> ing to &#226;&#65533;&#180;70 in holoenzyme by using an Eu chelate as a donor <br />                                                                                    the assay (5). In order to assure that refolding occur and to confirm the results <br /> and Cy5 as an acceptor (7). They were able to show its appli-                      from the experiments with denatured protein being added, some assays were <br /> cability within this system. We adapted the principle by ex-                       carried out by adding the &#226;&#65533;&#164;&#226;&#172;&#732;* fragment in its refolded state. <br /> changing the Cy5-labeled polynucleotide with an IC5-labeled                           A stock solution (200 nM) of &#226;&#65533;&#180;70* was prepared prior to the assay by a 1:200 <br /> &#226;&#65533;&#164;&#226;&#172;&#732; fragment (residues 100 through 309 N-terminally fused to a                      dilution of labeled protein (40 &#226;&#65533;&#174;M) with NTG stored at &#226;&#171;&#186;20&#194;&#176;C in storage buffer. <br />                                                                                    A stock solution (1.25 &#226;&#65533;&#174;M) of the &#226;&#65533;&#164;&#226;&#172;&#732;* fragment was prepared by the dilution of <br /> heart muscle kinase [HMK] recognition site and a His6 tag).                        labeled denatured protein (75 &#226;&#65533;&#174;M in TNTw buffer) to 1 M GuHCl with NTG and <br /> For the resulting homogenous assay, we labeled &#226;&#65533;&#180;70 with a                          NTG plus 6 M GuHCl. First, 10 &#226;&#65533;&#174;l of that &#226;&#65533;&#180;70* stock solution was mixed with <br /> europium&#226;&#8364;&#8220;diethylenetriaminepentateic acid (DTPA)&#226;&#8364;&#8220;7-ami-                            NTG buffer (amount adjusted to give a final volume of 200 &#226;&#65533;&#174;l), the potential <br /> no-4-methylcoumarin-3-acetic acid (AMCA) complex as a do-                          inhibitor (dissolved in 1 ml of methanol, diluted 1:50 in DMSO, and 5 &#226;&#65533;&#174;l of which <br /> nor and the HMK-His6-&#226;&#65533;&#164;&#226;&#172;&#732;(100-309) fragment with the Cy5                             was applied to the reaction mixture), salt, or solvent was then added, and finally, <br />                                                                                    the 5 &#226;&#65533;&#174;l of denatured, labeled &#226;&#65533;&#164;&#226;&#172;&#732;* stock solution was added. Salts or denaturants <br /> analogue IC5-maleimide (Dojindo) (purification and labeling                        were dissolved in NTG buffer according to the desired final concentration to <br /> procedures described in Bergendahl et al. [5]). It is possible to                  maintain standard buffer conditions. Mixing (pipetting up and down three times) <br /> monitor complex formation between &#226;&#65533;&#180;70 and &#226;&#65533;&#164;&#226;&#172;&#732; simply by                             after the addition of each component was very important for reproducible re- <br /> looking at the long-lived emission of the acceptor (IC5) due to                    sults. The mixture was incubated for 30 min at room temperature and measured <br />                                                                                    in a 96-well plate (Costar 3650) with a multiplate reader (VictorV2 1420; Wal- <br /> LRET as an optically measurable signal of complex formation. <br />                                                                                    lac). For this time-resolved fluorescence measurement, the manufacturer&#226;&#8364;&#8482;s pro- <br /> Measuring time-resolved fluorescence allows one to start sig-                      tocol (LANCE high count 615/665) was used (excitation occurred with 1,000 <br /> nal acquisition after the background fluorescence (potentially                     flashes at 325 nm, measurement was delayed by 100 &#226;&#65533;&#174;s, and data were acquired <br /> from a natural product) and intrinsic acceptor fluorescence                        for 50 &#226;&#65533;&#174;s at 615 and 665 nm). <br /> have decayed so that all short-lived background fluorescence                          In fluorimetric measurements, it is common to use a second emission wave- <br />                                                                                    <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> as an internal standard. This allows for the correction of instrument noise <br /> can be excluded from the measurement. This leads to a highly                       but also normalizes the signal for the actual amount of donor, in this particular <br /> favorable signal-to-noise ratio and a higher sensitivity and con-                  case. This was possible, since donor and acceptor emission wavelengths were well <br /> fidence. This is especially important when working with natu-                      separated and could be acquired separately with the multiplate reader. The IC5 <br /> ral-product samples, which often produce a high background                         emission was corrected for the very small amount of signal from the Eu emission <br />                                                                                    band (by cross-talk measurement of a standard) and then divided by the intensity <br /> fluorescence. The principle of the assay and the structures of <br />                                                                                    of the Eu signal. The normalization could be included in the overall measure- <br /> the dyes are described in Fig. 1.                                                  ment protocol and is described by the manufacturer of the multiplate reader <br />    The assay can be performed in a multiwell plate and mea-                        (Wallac). Fortunately, the nature of this method allowed us to differentiate the <br /> sured by a multiplate reader to accomplish a high-throughput                       loss of signal due to the inhibition of protein-protein binding from simple ab- <br /> screening of a large number of samples from any chemical                           sorption caused by the inner-filter effect of the substance. This helped us to <br />                                                                                    identify false positives in the actual high-throughput screen. <br /> library in an automated way. Typical reaction volumes were 10                         In vitro transcription. The conditions and procedure in the in vitro transcrip- <br /> to 200 &#226;&#65533;&#174;l where the components, including the test substances,                     tion assay were according to those published by Landick et al. (9) except for <br /> were mixed directly in the multiwell plate before the plate was                    omitting bovine serum albumin, since it could bind potential hydrophobic inhib- <br /> measured in the reading device. The very sensitive nature of                       itors. The assay conditions were as follows: 100 mM NaCl, 25 mM Tris HCl (pH <br /> such a fluorescence-based assay (typically in the low nanomo-                      7.9), 10 mM MgCl2, 1 mM dithiothreitol, 0.1 mM EDTA, 5% glycerol, 0.15 mM <br />                                                                                    ApU (dinucleotide), 20 nM template DNA, 20 nM core RNAP, and inhibitor in <br /> lar range) provides good accuracy and signal-to-noise ratio,                       final concentration of 2% DMSO, 10 nM &#226;&#65533;&#180;70, 25 &#226;&#65533;&#174;M concentrations each of <br /> avoiding false-positive hits in the measurement. We have cho-                      ATP and CTP, 10 &#226;&#65533;&#174;M GTP, and 20 &#226;&#65533;&#174;Ci of [&#226;&#65533;&#163;-32P]GTP/100 &#226;&#65533;&#174;l, added in the <br /> sen to screen a marine sponge library (19) since it has been                       order listed. For the template, we amplified a 694-bp fragment from the plasmid <br /> successfully screened for cytotoxic compounds. Due to its com-                     pCL185 by PCR with the primers 5&#226;&#172;&#732;-GTT TTC CCA GTC ACG AC-3&#226;&#172;&#732; and <br />                                                                                    5&#226;&#172;&#732;-CAG TTC CCT ACT CTC TCG CAT G-3&#226;&#172;&#732;. Omitting UTP in the reaction <br /> plexity, it demanded the high performance of the assay and was <br />                                                                                    mixture and initiating with ApU dinucleotide resulted in a halted complex at <br /> therefore well suited to evaluate its applicability. We show the                   position &#226;&#171;&#185;16 and the transcribed RNA oligonucleotide AUG GAG AGG GAC <br /> robust nature of the assay and its suitability for use as a high-                  ACG G. The assay tested for the ability of the RNAP holoenzyme (core plus <br /> throughput screen.                                                                 sigma factor) to recognize a &#226;&#65533;&#180;70-dependent promoter (T7A1; located after bp <br />                                                                                    280 of the 694-bp PCR fragment) and to initiate as well as elongate to yield the <br />                                                                                    16-mer oligonucleotide. Failure of sigma binding to the core resulted in a lack of <br />                        MATERIALS AND METHODS <br />                                                                                    product. <br />   Materials and chemicals. Overproduction, labeling, and purification of HMK- <br /> His6-&#226;&#65533;&#164;&#226;&#172;&#732;(100-309) and &#226;&#65533;&#180;70(442C) were described previously (5). The origin of the <br /> natural-product library comprises privately collected samples of marine sponges <br />                                                                                                                      RESULTS <br /> (19) extracted with acetone and ethylacetate. The samples (2 to 5 mg [dry <br /> weight]) were dissolved in 1 ml of methanol and diluted 1:50 in dimethyl sulfox- <br /> ide (DMSO) for the assay. All chemicals used were purchased from Sigma unless        Labeled as well as unlabeled proteins were shown to be able <br /> otherwise indicated in the text. A multiplate reader (VictorV2 1420; Wallac) was   to form a complex in electrophoretic mobility shift (EMS) <br /> used to perform the LRET assay. The following buffers were used: NTG buffer        assays by using native polyacrylamide electrophoresis gels (5). <br /> (50 mM NaCl, 50 mM Tris, 5% glycerol [pH 7.9]), storage buffer (50 mM              The different scanning techniques also confirmed the identities <br /> Tris-HCl, 10 &#226;&#65533;&#174;M EDTA, 0.5 M NaCl, 10% glycerol [pH 7.5]), and TNTw buffer <br /> (6 M GuHCl, 50 mM Tris-HCl [pH 7.9], 500 mM NaCl, 0.1% [vol/vol] Tween 20, <br />                                                                                    of the bands in the EMS assay. Furthermore, the EMS assay <br /> 400 mM imidazole). Core RNAP was prepared according to the method de-              showed that the unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment can compete for bind- <br /> scribed by Thompson et al. (20).                                                   ing to the labeled &#226;&#65533;&#180;70. Thus, the unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment itself <br />  1494      BERGENDAHL ET AL.                                                                                            APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Principle of the LRET assay. (A) The scheme above shows the principle of how the LRET signal in the assay is created upon binding <br /> of the labeled proteins &#226;&#65533;&#164;&#226;&#172;&#732; (residues 100 through 309) and &#226;&#65533;&#180;70. The fluorescence of the IC5-labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment has decayed during the delay of <br /> the data acquisition 50 &#226;&#65533;&#174;s after excitation at 320 nm. Only the Eu emission of labeled &#226;&#65533;&#180;70 and the sensitized IC5 emission in the complex can be <br /> observed after the delay due to the characteristic long Eu luminescence lifetime of over 1 ms. (B) In the diagram, the luminescence is plotted versus <br /> the time. The graphs shows the intensity of the luminescence emitted at 615 and 665 nm over time. Whereas the intrinsic IC5 emission has decayed <br /> long before the data acquisitions starts, the emission from the IC5 sensitized by LRET goes on for over 1 ms. IC5 has a slightly faster decay than <br /> the Eu emission. That way, only the IC5 emission due to LRET is acquired in the data acquisition window, whereas the background emission and <br /> the intrinsic IC5 fluorescence is excluded. (C) The structures of the fluorophores that were used to derivatize the proteins are shown. The Eu <br /> chelate DTPA-AMCA-maleimide was used to <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> &#226;&#65533;&#180;70. IC5-PE-maleimide served as the label for the &#226;&#65533;&#164;&#226;&#172;&#732; fragment. <br />  <br />  <br />  <br /> represents a positive control for an agent able to interfere with            beled proteins. The LRET assay provides a fast and reproduc- <br /> the binding of the labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment and &#226;&#65533;&#180;70 in the assay.                 ible alternative to the EMS assay (5) to monitor the formation <br />   Validation of the LRET assay to test for the inhibition of                 of a protein-protein interaction between the labeled &#226;&#65533;&#180;70 and <br /> protein-protein interaction of labeled &#226;&#65533;&#180;70 and &#226;&#65533;&#164;&#226;&#180;&#8226; with unla-                 &#226;&#65533;&#164;&#226;&#172;&#732; fragment as well as its inhibition. All results of the EMS <br />  VOL. 69, 2003                                                                       LRET ASSAY FOR DRUG DISCOVERY                     1495 <br />  <br />  <br />  <br />  <br />   FIG. 2. Competition of unlabeled &#226;&#65533;&#180;70, a nonbinding &#226;&#65533;&#180;70 mutant <br /> (D403R) (3), and bovine serum albumin (BSA) with labeled &#226;&#65533;&#180;70 for <br /> binding to the &#226;&#65533;&#164;&#226;&#172;&#732; fragment. Only the &#226;&#65533;&#180;70 (wild type) competes for <br /> binding to the &#226;&#65533;&#164;&#226;&#172;&#732; fragment, while the other proteins show only non- <br /> specific binding resulting in a slightly lower LRET count. <br />  <br />  <br />  <br /> assay were reproducible by the LRET assay, and as an exam- <br /> ple, we were able to show competition of labeled &#226;&#65533;&#180;70 binding <br /> to the &#226;&#65533;&#164;&#226;&#172;&#732; fragment by increasing the amounts of unlabeled &#226;&#65533;&#180;70 <br /> (Fig. 2). As a very important feature of the assay, the signal- <br /> to-noise ratio was between 10 and 11. The limit of detection <br /> within the assay, using the described instrument, was 1 nM <br /> labeled &#226;&#65533;&#180;70. A maximum of 250 nM labeled &#226;&#65533;&#180;70 could be <br /> applied to the assay before diffusion-controlled LRET oc- <br /> curred. <br />    Influence of salt and organic solvents on the LRET assay. In <br /> further experiments, the dependence of salt (NaCl and <br /> GuHCl) and solvents (methanol, ethanol, and DMSO) were <br /> characterized (Fig. 3). As can be seen, the salt concentration <br /> had a major effect on the assay, since the signal drops to 50% <br /> when the NaCl concentration was increased from 100 to 400 <br /> mM. It is known that this &#226;&#65533;&#164;&#226;&#172;&#732; fragment interaction with &#226;&#65533;&#180;70 is           FIG. 3. Dependence of salt concentration and effect of solvent con- <br /> weakened by increased NaCl concentration. On the other                centration on the LRET assay. (A) With increasing amounts of salt <br /> hand, DMSO as a common solvent for natural products to be             (NaCl, GuHCl), the LRET signal significantly decreases, which should <br /> tested had no significant effect on the assay. The signal in the      be due to the decreased amount of the &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; complex formed. (B) In- <br /> LRET assay was not critically affected by the amount of               creasing amounts of DMSO, methanol, and ethanol (0 to 5%) mixed <br />                                                                       with the assay buffer prior to the addition of proteins are shown. <br /> DMSO present, up to 2.5%. In the same experiment, ethanol <br /> and methanol showed a more significant effect over the range <br /> of 1 to 5%. <br />    Performance of the LRET assay in a preliminary screen of              Confirmation of hits by the LRET assay and by in vitro <br /> 100 sponge extracts. The concentration of inhibitor in the            transcription. The concentration of inhibitor in the extracts of <br /> extracts of marine sponges was estimated to be less than 1 &#226;&#65533;&#174;M         marine sponges was examined over the range of 30 nM to 2 <br /> (assuming a 2-mg sample was 100% of a single component                &#226;&#65533;&#174;M in the LRET assay. Samples A5, B12, E1, E12, G1, H4, <br /> having an approximate molecular mass of 1,000 Da). Out of             and H6 resulted in no or very minor loss of signal in the LRET <br /> the 100 samples tested, sample D7 turned out to be the only           assay. Sample D7 turned out to be the only extract to give a <br /> extract to reduce the signal by roughly 90% in the assay. Sam-        clear inhibition curve over the examined range (Fig. 5A). This <br /> ples E1, E12, G1, H4, and H6 showed a loss of signal of more          was confirmed by in vitro transcription assays. Samples A5, <br /> than 50%, which served as an arbitrary threshold (between 55          B12, E1, E12, G1, H4, and H6 resulted in no loss of activity in <br /> and 75%). Most samples (like A7 and B12) were significantly           the in vitro transcription assay, as judged by densitometry of <br /> less active in the inhibition of binding, judged by the loss of the   the electrophoretograms of the gels (Fig. 5B). In this assay, <br /> LRET signal (Fig. 4). The variance of the obtained values for         only D7 showed a dose-dependent inhibition of transcriptional <br /> the majority of samples spread over a wide range of up to 40%         activity, with a 50% inhibitory concentration (IC50) of around <br /> loss of signal.                                                       1 &#226;&#65533;&#174;M, assuming it had the mass and content of the sample <br />  1496      BERGENDAHL ET AL.                                                                                        APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Screen of 100 extracts from marine sponges by the LRET <br /> assay. Inhibition of &#226;&#65533;&#180;70 binding to the &#226;&#65533;&#164;&#226;&#172;&#732; fragment was tested with 100 <br /> extracts from marine sponges (G. Marriott and J. Tanaka). Binding of <br /> &#226;&#65533;&#180;70 to the &#226;&#65533;&#164;&#226;&#172;&#732; fragment could be significantly inhibited by fraction D7, <br /> while other fractions show lower inhibition under these conditions. <br /> Assuming a molecular mass of 1,000 Da and 100% active content in a <br /> 2-mg sample, concentrations used were in the low micromolar range. <br />  <br />  <br /> mentioned above. There was no significant difference between <br /> the IC50s determined by LRET and those derived from the in <br /> vitro transcription assays. <br />  <br />                            DISCUSSION <br />    The use of LRET to investigate protein-protein interactions <br /> in a homogenous assay has been described before, along with <br /> its advantages and challenges. In spite of them, there are not <br /> many publications that describe the development of an assay                  FIG. 5. Validation of the hits by LRET (A) and in vitro transcrip- <br /> based on this technique and actually document its final appli-            tion (B) assays. (A) Some candidates (D7, H4, E1, and G1) resulting <br /> cation. In this work, we describe a straightforward approach to           from the screen and a sample with no effect (A5) serving as control <br /> designing a robust high-throughput assay that can be exploited            were tested in the LRET assay over a range of 33 nM to 2 &#226;&#65533;&#174;M <br />                                                                           (estimated concentration). The y axis represents the fluorescence sig- <br /> for any other pair of proteins that appears to be a valid target          nal resulting from LRET due to binding of labeled &#226;&#65533;&#180;70 and the &#226;&#65533;&#164;&#226;&#172;&#732; <br /> for drug discovery.                                                       fragment. Data from samples E1 and G1 are represented by H4 since <br />    There are several reasons to believe that the primary pro-             they behaved very similar in the LRET assay. (B) To determine the <br /> tein-protein interaction between bacterial core RNAP and                  functional effect of an inhibitor of &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; binding, we tested the can- <br />                                                                           didates in an in vitro transcription assay. This assay requires the in- <br /> sigma factors represents a prime target for drug discovery. The           corporation of &#226;&#65533;&#180;70 into holoenzyme RNAP and tests for its ability to <br /> key to the potential of this target is the absolute requirement           initiate transcription and elongation. The autoradiogram of the se- <br /> of sigma binding to core RNAP for the initiation of transcrip-            quencing gel was used to quantify the transcription product from a <br /> tion; no bacterial cell can grow upon uptake of an inhibitor that         halted complex using a T-less cassette as a template and omitting UTP <br /> effectively blocks this interaction. In addition to a very high           in the reaction mixture (see Materials and Methods). The resulting <br />                                                                           16-mer oligonucleotide can be separated on the gel and quantified by <br /> bioactivity, a good specificity can also be expected since the            using Molecular Dynamics software. The estimated concentrations in <br /> binding region of both proteins is highly conserved among                 the assay were between 0.1 and 50 &#226;&#65533;&#174;M (data from samples A5, E1, and <br /> bacteria and is significantly different from any known eukary-            G1 are represented by H4 since they behaved very similar in the assay; <br /> otic analogue. This implies a very low probability for side               data from D7 are indicated by triangles). <br /> effects to occur due to interference with human RNAP. The <br /> site itself offers another advantage over many potential and <br /> specific targets. Since the binding site on the &#226;&#65533;&#164;&#226;&#172;&#732; subunit of             need for new antibiotics, this has recently become a major issue <br /> RNAP is suspected to interact with many if not all of the sigma           in drug discovery. <br /> factors of a bacterium, the development of resistance via point              Using LRET to measure sigma binding to core RNAP has <br /> mutations against an inhibitor that binds to the &#226;&#65533;&#164;&#226;&#172;&#732; subunit of            been shown by Heyduk and coworkers to be an effective and <br /> RNAP in the binding site is unlikely, since it may impair                 very sensitive method. We used a well-characterized <br /> binding to any of the essential sigma factors. Due to the in-             &#226;&#65533;&#180;70(442C) mutant (7) that has all-natural cysteine residues <br /> creasing incidence of antibiotic resistance and the growing               mutated to serine residues and was shown to be fully active in <br />  VOL. 69, 2003                                                                         LRET ASSAY FOR DRUG DISCOVERY                             1497 <br />  <br />  <br /> in vitro transcription assays (7). This mutant was derivatized       substance. This contributed enormously to avoiding false-pos- <br /> with a DTPA-AMCA-maleimide Eu complex that serves as a               itive readings. Also, the very good compatibility of the assay <br /> donor in the LRET assay. A fragment (residues 100 through            with the use of DMSO as a solvent to add test substances <br /> 309) of the &#226;&#65533;&#164;&#226;&#172;&#732; subunit of RNAP with an N-terminal HMK                augments its applicability as a high-throughput screen. Since <br /> recognition site and His6 tag fusion was derivatized with IC5-       many natural products and most peptides or small molecules <br /> maleimide as the LRET acceptor (5). We were able to show             from combinatorial libraries have low solubility in water, it is <br /> with EMS assays (5) and with spectrometric measurements by           necessary to use organic solvents. In this respect, DMSO rep- <br /> using time-resolved fluorescence that the labeled proteins can       resents the most versatile and potent solvent, and since it is <br /> bind to each other in all combinations with or without the           often used in libraries, its compatibility is crucial for an assay. <br /> label. As controls, the unlabeled proteins were tested to de-        One sample (D7) could be identified by the LRET assay as an <br /> termine if they could compete with their labeled counterparts.       inhibitor containing extract, and this was confirmed by an in <br /> In both assays, EMS and LRET, the unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment was          vitro functional assay. The assumptions made to estimate the <br /> able to compete with the labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment for binding to          concentration of the inhibitors should clearly overestimate the <br /> labeled &#226;&#65533;&#180;70. With this data, we concluded that the assay can be      amount of active compound present so that it is fair to expect <br /> used to monitor &#226;&#65533;&#180;70 to &#226;&#65533;&#164;&#226;&#172;&#732; binding and can be used to screen          an IC of 10 to 100 nM or less. Initial studies on the ability of <br /> for inhibitors of this protein-protein interaction. The assay        the D7 extract to inhibit the cell growth of E. coli cells on <br /> represents a fast and sensitive probe for this particular complex    culture plates were inconclusive due to limited amounts of <br /> formation. Substrates and material either are readily available      available material (data not shown). Considering the complex- <br /> or can be prepared in simple and efficient procedures. All the       ity of the samples, we plan to carry out an automated screen of <br /> labeled protein components show excellent stability during           over 1,500 more sponge extracts before deciding on which <br /> storage, a great advantage when screening large libraries with       samples to characterize in detail. <br /> 10,000 to 100,000 or more substances. Furthermore, the LRET             In the case of natural-product libraries, a positive hit could <br /> assay has a very high sensitivity so that measurements can be        represent a new class of antibiotic, since no substance with <br /> performed at very low protein concentrations of 1 to 100 nM,         such a mode of action is known. On the other hand, the screen <br /> resulting also in a very low cost per screen.                        will help to identify known antibiotics for which the mode of <br />    The suitability of the assay for the high-throughput screen-      activity has not yet been identified or that reveal more than one <br /> ing of natural and combinatorial libraries was demonstrated by       activity. Together with positive hits from combinatorial librar- <br /> a screen of 100 extracts of marine sponges. The high demand          ies, these substances can then serve as lead structures to design <br /> on the performance of an assay when screening natural-prod-          and tailor a new compound with desirable characteristics, such <br /> uct libraries such as this, containing very complex mixtures of      as high activity, specificity, stability, and ability to enter the cell <br /> possibly more than 30 compounds, becomes obvious by looking          and, on the other hand, low side effects, costs, chance, and <br /> at the variance of the obtained values in the LRET assay. This       likelihood of resistance development. In addition, this assay <br /> variance occurs mainly due to fluorophores in the mixtures that      can serve as a powerful tool to investigate the relative binding <br /> can absorb at any of the crucial wavelengths of the assay and is     of different sigma factors and sigma factor mutants to the core. <br /> generally referred to as the inner-filter effect. Such an effect <br /> could have led to the response of samples B12, E1, E12, G1,                                <br /> H4, and H6 in the LRET assay (Fig. 4 and, in part, 5A), which          <br /> in the in vitro transcription assay (Fig. 5B) did not show an        <br /> inhibitory effect on transcriptional activity. It is also conceiv-    <br /> able that one of these compounds binds to a region of the &#226;&#65533;&#164;&#226;&#172;&#732;          <br /> subunit that is inaccessible in core RNAP and thus leads to an <br />                                                                                                     <br /> impaired binding by the distortion of protein structure. This <br />                                                                      <br /> underlines the technical and biochemical differences between            <br /> the two described assays so that compounds that are active in         <br /> the LRET assay might not have the same effect in the in vitro      <br />                                                                         <br /> transcription assay. The LRET assay monitors particularly the       <br /> binding of sigma to a fragment of the &#226;&#65533;&#164;&#226;&#172;&#732; subunit from RNAP,           <br />                                                                     <br /> whereas the in vitro transcription assay tests the ability of      <br /> holoenzyme RNAP for transcriptional initiation and elonga-               <br /> tion. Due to its simplicity and suitability for use in a high-  <br />                                                                      <br /> throughput mode, the LRET assay is used to select for poten-            <br /> tial inhibitors among a large number of candidates present in            <br /> a library of compounds. This way, the number of samples that        <br />                                                                      <br /> have to be investigated by a more informative but also signif-           <br /> icantly more time-consuming assay, like the in vitro transcrip-      <br />                                                                      <br /> tion assay, can be minimized.                                         <br />    The LRET assay turned out to be very sensitive and reliable.        <br /> A very good signal-to-noise ratio of above 10 and the internal         <br />                                                                      <br /> standard method helped to distinguish between inhibition and      <br /> fluorescence quenching by an inner-filter effect of the test      <br /> </body></html>