<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF VIROLOGY, Dec. 2000, p. 11088&#226;&#8364;&#8220;11098                                                                                       Vol. 74, No. 23 <br /> 0022-538X/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         Reconstitution Marek&#226;&#8364;&#8482;s Disease Virus Serotype 1 (MDV-1) <br />          DNA Cloned Bacterial Artificial Chromosome and <br />         Characterization Glycoprotein B-Negative MDV-1 Mutant <br />         DANIEL SCHUMACHER, B. KARSTEN TISCHER, WALTER FUCHS,                                   AND   NIKOLAUS OSTERRIEDER* <br />          Institute Molecular Biology, Friedrich-Loeffler-Institutes, Federal Research Centre Virus Diseases Animals, <br />                                                     D-17498 Insel Riems, Germany <br />                                               Received 14 July 2000/Accepted 15 September 2000 <br />  <br />              The complete genome Marek&#226;&#8364;&#8482;s disease virus serotype 1 (MDV-1) strain 584Ap80C cloned Esche- <br />           richia coli bacterial artificial chromosome (BAC). BAC vector sequences introduced US2 locus <br />           MDV-1 genome homologous recombination. Viral DNA containing BAC vector used <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />           transform Escherichia coli strain DH10B, colonies harboring complete MDV-1 genome F <br />           plasmid (MDV-1 BACs) identified. DNA various MDV-1 BACs transfected chicken embryo <br />           fibroblasts, 3 days transfection, infectious MDV-1 obtained. Growth MDV-1 recovered <br />           BACs indistinguishable parental virus, assessed plaque formation and <br />           determination growth curves. In MDV-1 BAC clones, sequences encoding glycoprotein B (gB) were <br />           deleted one-step mutagenesis using linear DNA fragment amplified PCR. Mutant MDV-1 recovered <br />           transfection BAC DNA harbored 2.0-kbp deletion 2.6-kbp gB gene able grow and <br />           induce MDV-1-specific plaques cells providing MDV-1 gB trans. The gB-negative virus reported here <br />           represents MDV-1 mutant deletion essential gene demonstrates power and <br />           usefulness BACs analyze genes gene products slowly growing strictly cell-associated herpes- <br />           viruses. <br />  <br />  <br />    Marek&#226;&#8364;&#8482;s disease virus (MDV) member Alphaher-               cells allowed efficient growth MDV-1 (29). Due to <br /> pesvirinae subfamily Herpesviridae (8, 12, 36). Based           problems, multiple rounds purification virus recom- <br /> virulence chickens, ability induce T-cell lymphomas,           binants coseeding infected uninfected CEF needed <br /> antigenic properties, serotypes MDV (MDV               (1, 9, 21, 22, 23, 27, 30). <br /> serotype 1 [MDV-1], MDV-2, MDV-3) (24, 29). MDV-3                        In recent years, manipulations large herpesvirus ge- <br /> represents herpesvirus turkeys (HVT)                nomes facilitated using bacterial artificial chro- <br /> widely used vaccination MD. According             mosome (BAC) vectors. The genomes murine human <br /> recent nomenclature, MDV-1 classified gallid herpesvirus            cytomegaloviruses (HCMV) (3, 15), herpes simplex virus type <br /> 2 (GHV-2), MDV-2 classified GHV-3, HVT clas-                 1 (34), pseudorabies virus (PrV) (32, 33), Epstein-Barr <br /> sified meleagrid herpesvirus. All viruses belong          virus (10) cloned infectious BACs using this <br /> new genus Marek&#226;&#8364;&#8482;s disease-like viruses Alphaherpes-            technique. Targeted random mutagenesis herpesvirus <br /> virinae (36).                                                             genomes cloned BACs considerably faster re- <br />    Control MDV-1 infection achieved vaccination,                liable mutagenesis longer dependent growth <br /> primarily HVT. However, vaccination failures               viruses eukaryotic cells performed in <br /> description so-called &#226;&#8364;&#339;very virulent&#226;&#8364;? MDV-1, MDV-2                 Escherichia coli (3, 6, 15, 33, 34, 37). <br /> strains later attenuated MDV-1 strains (e.g., strain                     The aim study provide basis fast and <br /> Rispens CVI 988) used vaccine formulations (38,              efficient production MDV-1 recombinants cloning the <br /> 40). In recent years reported United States,             complete 180-kbp genome E. coli stable F plasmid and <br /> even virulent MDV-1, &#226;&#8364;&#339;very virulent plus&#226;&#8364;? (vv&#226;&#171;&#185;), MDV-1               apply recently developed recE- recT-based mutagen- <br /> variants appeared caused high mortality vaccinated                 esis (17, 18, 41) cloned MDV-1 DNA. Infectious <br /> flocks (39). One vv&#226;&#171;&#185; strains, 584A, passaged                 MDV-1 readily recovered transfection cloned <br /> serially chicken embryo fibroblasts (CEF) lost patho-              BAC DNA, MDV-1 BACs stable several <br /> genicity chickens (39). The reasons differences            rounds bacterial growth serial propagation CEF. Last, <br /> changes pathogenicity 584A MDV-1 strains               one-step deletion essential MDV-1 gene E. <br /> are poorly understood, molecular analyses MDV-1                coli possible, shown advantage for <br /> using recombinant virus mutants difficult perform            analysis essential nonessential MDV-1 genes may <br /> infectious virus progeny released supernatants             serve tool production biologically safe modified live <br /> cultured cells primary secondary chicken duck              virus and/or DNA vaccines. <br />  <br />                                                                                                 MATERIALS AND METHODS <br />    * Corresponding author. Mailing address: Institute Molecular <br />                                                                             Virus cells. Primary secondary CEF quail muscle cells (QM7; ATCC <br /> Biology, Friedrich-Loeffler-Institutes, Federal Research Centre Vi-   cell line CRL-1962) maintained Dulbecco&#226;&#8364;&#8482;s modified essential medium <br /> rus Diseases Animals, Boddenblick 5a, D-17498 Insel Riems, Ger-        supplemented 5 10% fetal calf serum. MDV-1 strain 584Ap80C was <br /> many. Phone: 49-38351-7266. Fax: 49-38351-7151. E-mail: klaus             kindly provided Richard L. Witter, Avian Diseases Oncology Laboratory <br /> .osterrieder@rie.bfav.de.                                                 (ADOL), East Lansing, Mich. Strain 584Ap80C represents avirulent, cell- <br />  <br />                                                                       11088 <br />  VOL. 74, 2000                                                                                  MDV-1 BAC AND gB-NEGATIVE MUTANT                               11089 <br />  <br />  <br />                                     TABLE 1. Primers used generate plasmids pDS pcMgB delete gB <br />  Primer                                                           Sequencea                                                         Fragment plasmid generated <br />  <br /> MUS21                     5&#226;&#172;&#732;-ACAggatccGTGTTTGAATACTGG-3&#226;&#172;&#732;                                                                             2.1-kb pDS <br /> MUS22                     5&#226;&#172;&#732;-ATAgtcgacTttaattaaCCGGTAGTCATTAGC-3&#226;&#172;&#732;                                                                    2.1-kb pDS <br /> MUS23                     5&#226;&#172;&#732;-ATCgcatgcttaattaaTTTGGCAAAACGGAATAGG-3&#226;&#172;&#732;                                                                 3.0-kb pDS <br /> MUS24                     5&#226;&#172;&#732;-CGCaagcttAATATGAATCTCTAAAACTTCTCGGC-3&#226;&#172;&#732;                                                                  3.0-kb pDS <br /> gB-up                     5&#226;&#172;&#732;-GATAgaattcATGCACTATTTTAGGCGG-3&#226;&#172;&#732;                                                                         pcMgB <br /> gB-low                    5&#226;&#172;&#732;-ATACctcgagTTACACAGCATCATCTTCTG-3&#226;&#172;&#732;                                                                       pcMgB <br /> gBkana                    5&#226;&#172;&#732;-TTTTCTTTCATCAATAGATGTTCGTTCCAAATCATCTGATTCCTCGCCAT                                                      Kanr gene gB deletion <br />                               AAAGCACTAAATCGGAACCCTAAAGGGAGC-3&#226;&#172;&#732; <br /> gBkanb                    5&#226;&#172;&#732;-ATATAGACAGATCACTAATCGATATACAGATAGGACGCCCGTTTCCATTG                                                      Kanr gene gB deletion <br />                               ATTGTCTCCTTCCGTGTTTCAGTTAGCCTC-3&#226;&#172;&#732; <br />   a <br />     For primers MUS21 gB-low, bold sequences indicate restriction enzyme sites, sequences italics indicate additional bases present MDV-1 <br /> sequence. For primers gBkana gBkanb, underlined sequences indicate sequences pEGFP-N1 used amplify Kanr gene, sequences bold italics <br /> indicate gB sequences allow homologous recombination recE- recT-mediated deletion gB. <br />  <br />  <br />  <br /> culture-passaged descendant vv&#226;&#171;&#185; strain 584A (39) grown primary        quently cloned pTZ18R (Pharmacia-Amersham) obtain plasmid pDS <br /> or secondary CEF cells previously described (19). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> variety perma-       (Fig. 1). A BAC vector containing Eco-gpt gene control the <br /> nent avian cells tested, QM7 cells shown support growth MDV-1.           HCMV immediate-early promoter released PacI fragment plasmid <br /> Subsequently, absence MDV-1 sequences QM7 cells tested             pHA1 (15; kindly provided M. Messerle, Ludwig-Maximilians Universita&#194;&#168;t, <br /> Southern blot hybridization PCR targeting different regions genome       Munich, Germany) inserted PacI sites 2.1- 3.0-kbp <br /> before used propagation MDV-1 (V. Zelnik, R. Riebe, N.         fragment cloned pDS (Fig. 1; Table 1). Primary CEF cells cotransfected <br /> Osterrieder, unpublished results). Virus growth curves determined de-       approximately 2 &#226;?&#174;g 584Ap80C DNA 10 &#226;?&#174;g pDS-pHA1. At 5 days <br /> scribed previously (23) slight modifications. Briefly, 100 PFU used    transfection, infected cells plated primary CEF cells pres- <br /> infect 2 &#226;&#171;&#187; 106 freshly seeded CEF cells. At various times infection (0, 12,   ence 250 &#226;?&#174;g mycophenolic acid (MPA) ml, 50 &#226;?&#174;g xanthine ml, <br /> 24, 48, 72, 96, 120 h), infected cells trypsinized titrated fresh   100 &#226;?&#174;g hypoxanthine ml. To avoid death actively dividing CEF, cells <br /> CEF cells. Virus growth curves determined independent experi-           seeded density 1.5 &#226;&#171;&#187; 107 cells 75-cm2 flask grown overnight, <br /> ments. A QM7 cell line constitutively expressing MDV-1 glycoprotein B (gB) was <br />                                                                                     selection medium applied 1 h addition virus-containing cells. <br /> obtained transfection 106 QM7 cells 10 &#226;?&#174;g plasmid pcMgB. To <br />                                                                                     The MPA-xanthine-hypoxanthine selection repeated total four <br /> obtain pcMgB, based pcDNA3 (Invitrogen), MDV-1 gB gene <br />                                                                                     times. After complete cytopathic effect developed, viral DNA prepared <br /> from strain Rispens amplified PCR using gB-specific primers (Table 1) <br /> and cloned control HCMV immediate-early promoter. Trans-           infected cells (16) 1 &#226;?&#174;g infected-cell DNA electroporated into <br /> fected QM7 cells grown presence 1 mg G418 ml,             E. coli DH10B cells. Electrocompetent bacteria prepared described <br /> gB-expressing clones identified using anti-gB monoclonal antibody (MAb)        previously (17, 18), electroporation performed 0.1-cm-wide cuvettes <br /> 2K11 (kindly provided Jean-Francois Vautherot, Institut National la Re-       1,250 V, resistance 200 &#226;?&#8364;, capacitance 25 &#226;?&#174;F (Easyject electro- <br /> cherche Agronomique, Tours, France). The resulting cell line constitutively         poration system; Eurogenentec). Transformed bacteria incubated 1 ml of <br /> expressing MDV-1 gB designated MgB1.                                            Luria-Bertani (LB) medium (28) supplemented 0.4% glucose 1 h 37&#194;&#176;C <br />    Construction MDV-1 BACs. MDV-1 DNA purified infected cells           plated LB agar containing 30 &#226;?&#174;g chloramphenicol ml (28). <br /> by sodium dodecyl sulfate-proteinase K extraction described earlier (16). For    Single colonies picked placed liquid LB medium, small-scale <br /> construction plasmid pDS-pHA1, 2.1- 3.0-kbp fragments      preparations BAC DNA performed alkaline lysis E. coli (28). <br /> the MDV-1 US2 gene (Fig. 1) amplified PCR using primers containing          Large-scale preparation BAC DNA achieved silica-based affinity chro- <br /> appropriate restriction enzyme sites (Table 1). Both fragments subse-          matography using commercially available kits (Qiagen; Macherey &amp; Nagel). <br />  <br />  <br />  <br />  <br />    FIG. 1. Schematic illustration cloning procedure transfer plasmid introduce BAC vector MDV-1 genome. The organization the <br /> approximately 180-kbp MDV-1 genome (A) BamHI restriction map (B) determined Fukuchi et al. (11) shown. The unique short region (US) and <br /> the ORFs located US shown (C D). A 2.1- 3.0-kbp fragment bordering US2 gene (grey boxes) amplified PCR cloned plasmid <br /> pTZ18R rise recombinant plasmid pDS. The 7.2-kbp BAC vector released recombinant plasmid pHA1 (15) inserted pDS resulted in <br /> plasmid pDS-pHA1 (E). Restriction enzyme sites determined Brunovskis Velicer (7) abbreviated follows: B, BamHI; E, EcoRI; P, PstI; Pa, PacI; <br /> S, SalI. The variable lengths terminal DNA fragments indicated (var). <br />  11090       SCHUMACHER ET AL.                                                                                                                             J. VIROL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Digitally scanned image DNA 584Ap80C (V) DNA isolated chloramphenicol-resistant E. coli DH10B colonies named BAC19, <br /> BAC20, BAC24. Viral BAC DNA cleaved BamHI EcoRI, separated 0.8% agarose gel electrophoresis, stained ethidium bromide (left <br /> panel). The restriction enzyme digests flanked 1-kb ladder (Gibco-BRL). Asterisks indicate additional bands size variations individual fragments for <br /> the BAC clones (in cases additional bands comigrate bands). Arrows indicate bands arising insertion BAC vector sequences (left <br /> panel). Two subfragments Bam-HI-A additional BamHI site introduced inserting BAC vector sequences additional band 5.8 kbp in <br /> EcoRI-digested BAC DNA indicated arrows. After Southern transfer DNA fragments nylon membranes, hybridization digoxigenin-labeled <br /> fragments released plasmid pDS pHA1 performed. The sizes (in kilobase pairs) reactive bands given right. <br />  <br />  <br />  <br />  <br />    Mutagenesis MDV-1 BACs. For mutagenesis MDV-1 BAC DNA E.                 (kindly provided Lucy Lee, ADOL) convalescent serum chicken <br /> coli, recE-catalyzed reactions promoting homologous recombination            infected MDV-1 (anti-MDV). <br /> linear DNA fragments, referred E/T cloning, performed (18, 41). <br /> Plasmid pGETrec (kindly provided Panos Ioannou, Murdoch Institute, Mel- <br /> bourne, Australia) harboring recE, recT, bacteriophage &#226;?&#173; gam gene                                             RESULTS <br /> transformed BAC20-containing DH10B cells (18). After induction recE, <br /> recT, gam addition 0.2% arabinose, electrocompetent cells pre- <br /> pared essentially described previously (18). To delete gB gene BAC20, <br />                                                                                         Construction analysis BACs containing complete <br /> the kanamycin resistance (Kanr) gene plasmid pEGFP-N1 (Clontech)              MDV-1 genomes. 584Ap80C viral DNA transfected into <br /> amplified PCR. The designed primers contained 50-nucleotide homology              primary CEF cells pDS-pHA1 (Fig. 1). Five days <br /> arms bordering desired deletion gB 20 nucleotides amplifica-      transfection, infected cells seeded fresh CEF and <br /> tion Kanr gene (Table 1). The resulting 1.6-kbp fragment purified <br /> from agarose gel (Qiagen) electroporated pGETrec-containing BAC20 <br />                                                                                      overlaid selection medium. This procedure repeated <br /> cells. Colonies harboring Camr Kanr genes identified plates          total times. Finally, DNA recombinant <br /> containing antibiotics (18).                                                    MDV-1 able grow presence MPA-xan- <br />    DNA analyses. BAC viral 584Ap80C DNA isolated prokaryotic              thine-hypoxanthine isolated, cleaved BamHI, and <br /> eukaryotic cells cleaved EcoRI, BamHI, BglI, StuI separated on <br /> 0.8% agarose gels. DNA fragments transferred positively charged nylon <br />                                                                                      subjected Southern blot analysis using labeled pDS a <br /> membranes (Pharmacia-Amersham), Southern blot hybridization per-             probe. In addition BamHI-A fragment, additional <br /> formed using digoxigenin-labeled BAC19 DNA individual BamHI fragments             bands approximately 17 10 kbp size specifically <br /> of MDV-1 strain GA (11, 19). In addition, gB-specific probe plasmid           detected. This hybridization <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span> indicated approxi- <br /> pcMgB probe harboring Kanr gene prepared analysis of <br /> gB-negative MDV-1 BAC. Chemoluminescence detection DNA hybrids using <br />                                                                                      mately 10% viral DNA contained BAC vector sequences <br /> CSPD according supplier&#226;&#8364;&#8482;s instructions (Roche Biochemicals).         (data shown). This DNA used transform E. coli <br />    IIF. For indirect immunofluorescence (IIF) analysis, cells grown 6-    DH10B cells. Transformed bacteria plated agar con- <br /> 24-well plates (Greiner) glass coverslips subsequently infected      taining chloramphenicol, single colonies picked. <br /> indicated. Cells fixed 90% acetone various times infection or <br /> transfection, IIF exactly described, samples analyzed by <br />                                                                                         BAC DNA isolated bacterial colonies extracted <br /> conventional fluorescence microscopy confocal laser scanning microscopy           run agarose gels. Several bacterial colonies were <br /> (14). The antibodies used anti-gB MAb 2K11, anti-pp38 MAb H19 (9)               shown contain high-molecular-weight extrachromosomal <br />  VOL. 74, 2000                                                                              MDV-1 BAC AND gB-NEGATIVE MUTANT                           11091 <br />  <br />  <br />  <br />  <br />   FIG. 3. Digitally scanned images Southern blots analyze size variations BAC19, BAC20, BAC24 DNA. Viral DNA strain 584Ap80C individual <br /> BACs cleaved BamHI EcoRI transferred nylon membranes. Sheets incubated digoxigenin-labeled BAC19 DNA, labeled BamHI-C, or <br /> BamHI-D. The positions (in kilobase pairs) size markers (1-kb ladder; Gibco-BRL) given left. The smear-like bands 584Ap80C DNA hybridized with <br /> BamHI-D sequences bracketed. Abbreviations: V, viral DNA strain 584Ap80C; 19, 20, 24, DNA BAC19, BAC20, BAC24, respectively. <br />  <br />  <br />  <br /> DNA, clones (BAC19, BAC20, BAC24)                           BAC20, BAC24 DNA, fragments 4.3, 2.8, 1.7 kbp <br /> exhibiting similar BamHI EcoRI restriction fragment pat-                     specifically reacted pDS probe, 5.8- 1.7- <br /> terns chosen analysis (Fig. 2). The selected                    kbp fragments specifically hybridized pHA1 probe <br /> BAC clones characterized restriction enzyme digestion                    (Fig. 2). These fragments corresponded exactly pre- <br /> and Southern blotting cleavage BamHI EcoRI. It                     dicted insertion pHA1 sequences (Fig. 1), it <br /> was demonstrated compared parental MDV-1                             concluded BAC vector correctly inserted <br /> strain 584Ap80C, BAC19, BAC20, BAC24 DNA exhibited                           instead US2 open reading frame (ORF) MDV-1 <br /> almost identical restriction enzyme fragment patterns (Fig. 2                    BAC clones analyzed. <br /> and 3). Two notable exceptions, however, readily recog-                        Some variation banding patterns BAC19, BAC20, and <br /> nized. The 20.8-kbp BamHI-A fragment present 584Ap80C                         BAC24 noted BamHI- EcoRI-digested DNA, <br /> DNA absent analyzed BAC clones. Instead, frag-                        e.g., additional band approximately 6.2 kbp BamHI- <br /> ments 17.4 9.8 kbp size detected DNA                      digested BAC19 DNA additional bands BamHI- or <br /> BAC19, BAC20, BAC24 (Fig. 2). These bands (Fig. 2)                       EcoRI-digested DNA BAC20 BAC24. These additional <br /> represented subfragments BamHI-A additional                       bands, comigrating fragments, <br /> BamHI site introduced insertion BAC vector                         indicated asterisks Fig. 2. To address question of <br /> sequences (Fig. 1). In EcoRI-digested BAC DNA, addi-                         variations restriction enzyme patterns, hybridization <br /> tional band 5.8 kbp (BAC vector sequences) (Fig. 2)                       labeled BamHI-D fragment (Fig. 1) performed be- <br /> minor alterations sizes fragments caused deletion                   cause size variations terminal internal repeats the <br /> of US2 gene observed (Fig. 2). The correct insertion                    unique long region (TRL IRL, respectively) common <br /> of BAC sequences various clones proven                         cell culture-adapted MDV-1 strains (35). It shown by <br /> Southern blot hybridizations using labeled inserts plasmid                    Southern blotting additional fragments observed in <br /> pDS pHA1 probe. The expected reaction pattern                         BamHI- EcoRI-digested DNA BAC19, BAC20, <br /> BamHI- EcoRI-digested DNA observed. In BamHI-                             BAC24 resulted variations TRL IRL. Two <br /> digested BAC DNA, 17.4- 9.8-kbp BamHI fragments spe-                         broad smears detected BamHI-D probe viral <br /> cifically reacted probes, parental                          584Ap80C DNA digested BamHI ranged ap- <br /> 584Ap80C DNA, 20.8-kbp BamHI-A fragment hybridized                           proximately 9 15 kbp 4 8 kbp (corresponding to <br /> only pDS probe (Fig. 2). In EcoRI-digested BAC19,                       BamHI-D -H fragments virulent MDV-1, respec- <br />  11092       SCHUMACHER ET AL.                                                                                                                          J. VIROL. <br />  <br />  <br />  <br />  <br />    FIG. 4. (A) IIF analysis representative MDV-1 plaques infection 584Ap80C recombinant viruses obtained transfection BAC19, BAC20, or <br /> BAC24 DNA. At 5 days p.i., infected cells fixed subjected IIF using anti-gB MAb 2K11. Detection bound antibodies performed anti-mouse <br /> Alexa 488 (Molecular Probes). Approximately 100 plaques induced virus scanned fluorescence microscope significant differences <br /> between virus reconstituted BAC clones parental virus reconstituted viruses observed. Magnification, &#226;&#171;&#187;250. (B) Growth curves MDV-1 <br /> strain 584Ap80C viruses recovered transfection various BACs. After infection CEF cells 100 PFU 584Ap80C transfection progeny BAC19, <br /> BAC20, BAC24, virus titers determined indicated times p.i. coseeding fresh CEF cells. Virus plaques counted immunofluorescent <br /> staining MAb 2K11. Each point represents mean independent experiments. <br />  <br />  <br />  <br />  <br /> tively; Fig. 1). Similar observations smear-like bands                      Reconstitution infectious MDV-1 cloned DNA. DNA <br /> made digestion 584Ap80C DNA EcoRI (Fig. 3).                          BAC19, BAC20, BAC24 transfected primary <br /> In contrast, distinct different bands detected                   CEF. From day 1 transfection, MDV-1-specific IIF sig- <br /> BamHI-D probe BAC clones analyzed (Fig. 3). It                          nals MDV-1-specific antibodies detected. <br /> noted BAC19 band ca. 8 kbp size                         From day 3 transfection, MDV-1-specific virus plaques <br /> apparently detected probe EcoRI digestion,                         clearly appeared demonstrated IIF using anti-MDV-1 <br /> whereas bands detected EcoRI-cleaved DNA                          MAbs. To determine growth recombinant MDV-1 <br /> BAC20 BAC24. The 8-kbp reactive band represent                           comparable parental virus, plaque sizes virus <br /> double molar band BamHI-digested BAC19 DNA                              growth curves determined. First, MDV-1 rescued after <br /> two bands reacted probe                           transfection various BACs coseeded fresh CEF <br /> relatively high intensity compared smaller                   sizes plaques compared induced pa- <br /> reactive fragments detected BACs. It                        rental 584Ap80C. As shown plaques stained day 5 <br /> must noted original blot 4.5-kbp EcoRI                     postinfection (p.i.), appreciable differences plaque sizes <br /> fragment present left terminus UL region (12)                    MDV-1 rescued transfection various <br /> visible BAC24 DNA                           MDV-1 BACs parental viruses detected (Fig. 4A). <br /> BACs parental 584Ap80C. All restriction enzyme                           Second, virus growth curves 584Ap80C MDV-1 recov- <br /> fragments different BAC clones generated cleavage                     ered BACs determined. In case BACs, 100 PFU <br /> of DNA BamHI EcoRI appeared identical                            virus harvested day 5 transfection used infect <br /> with viral 584Ap80C DNA. These observations                          fresh CEF seeded six-well plates. Similarly, 100 PFU of <br /> confirmed Southern blot analyses using labeled                           584Ap80C used infect fresh CEF way. At <br /> BAC19 DNA labeled BamHI-A, -B, -C, -I2 fragments                         various times p.i., virus-infected CEF harvested ti- <br /> probes (BAC19 BamHI-C probe shown Fig. 3).                                  trated coseeding 10-fold virus dilutions fresh CEF. The <br />  VOL. 74, 2000                                                                                 MDV-1 BAC AND gB-NEGATIVE MUTANT                             11093 <br />  <br />  <br />  <br />  <br />    FIG. 5. Digitally scanned images Southern blots analyze stability BAC vector sequences viruses recovered transfection BAC19 BAC20. <br /> Transfection progeny passaged times, viral DNA isolated passage. Virus DNA cleaved BamHI EcoRI, separated 0.8% <br /> agarose gel electrophoresis, transferred nylon membranes. Southern blot hybridization performed using digoxigenin-labeled fragments plasmid pDS or <br /> pHA1. Lanes: V, 584Ap80C; 19, BAC19; 20, BAC20; 1 4, passages 1 4 transfection BAC19 DNA, respectively; 4a, DNA isolated passage 4 after <br /> transfection BAC20 DNA. The sizes (in kilobases) reactive fragments given. Asterisks indicate reactive 1.6-kb band marker (1-kb ladder; <br /> Gibco-BRL). <br />  <br />  <br />  <br /> results experiments summarized Fig. 4B. It                     This deletion affect putative 360-nucleotide <br /> demonstrated MDV-1 reconstituted BACs exhib-                         LORF5 cause deletion 49 nucleotides 3&#226;&#172;&#732; end <br /> ited growth characteristics virtually identical                  LORF5 (12). To perform one-step mutagenesis BAC20 <br /> other parental 584Ap80C (Fig. 4B). Virus titers                    E. coli, plasmid pGETrec conferring ampicillin resistance <br /> steadily increased 12 96 h p.i., maximal titers                       transformed BAC20-containing DH10B cells. Subse- <br /> were reached (Fig. 4B). From plaque sizes growth                           quently, Kanr gene amplified primers approx- <br /> characteristics, concluded biological properties                    imately 70 nucleotides <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> allowed recA-independent <br /> MDV-1 BACs vitro indistinguishable                       homologous recombination MDV-1 gB sequences (Table <br /> parental strain.                                                                   1; Fig. 6). The resulting PCR product purified elec- <br />    To ascertain stability BAC-derived viruses, prog-                    troporated BAC20-pGETrec cells. Bacteria plated <br /> eny BAC transfections BAC19 BAC20 passaged                           LB agar containing chloramphenicol kanamycin, and <br /> four times viral DNA prepared. Isolated DNA                            colonies resistant picked. After DNA isolation of <br /> cleaved BamHI EcoRI, Southern blot hybridiza-                          individual colonies, Southern blot sequence analysis of <br /> tion performed using pDS pHA1 probe. Identical                          recombinant BAC20 harboring deletion gB gene <br /> DNA fragments described BAC DNA isolated                              (20&#226;&#338;&#172;gB) performed. A Kanr- gB-specific probe de- <br /> from DH10B cells detected probes (Fig. 5),                       tected fragments mutant BAC 20&#226;&#338;&#172;gB cleavage with <br /> and concluded BAC vector sequences remained stably                         BamHI, EcoRI, BglI, StuI perfect agreement <br /> inserted 584Ap80C genomes recovered indi-                          calculated insertion Kanr resistance <br /> vidual MDV-1 BAC clones serial passage CEF.                          gene gB-encoding sequences (Fig. 6C 7). In addition, <br /> However, hybridization BamHI-D fragment PCR                           DNA cycle sequencing using primers bind Kanr <br /> analysis indicated variability 132-bp tandem repeat                    gene allowed sequence determinations recombina- <br /> sequences virus population restored                     tion sites proved correct insertion Kanr gene within <br /> passage CEF (data shown).                                                   gB (data shown). It noted pGETrec easily lost <br />    Mutagenesis BAC20 deletion gB-encoding se-                            E. coli cells grown absence ampicillin (Fig. 7). <br /> quences. In experiments, recently developed method                      From results restriction enzyme patterns, South- <br /> for mutagenesis BACs applied remove 2.0 kbp                       ern hybridizations, nucleotide sequencing recombi- <br /> 2.6-kbp gB gene BAC20 (Fig. 6; Table 1), i.e., nucleotides                    nation sites, concluded entire gB ORF was <br /> 59867 61881 MDV-1 sequence according num-                         removed mutant BAC clone 20&#226;&#338;&#172;gB appre- <br /> bering Lee et al. (12) deleted gB ORF.                     ciable alterations regions genome present. <br />  11094       SCHUMACHER ET AL.                                                                                                                             J. VIROL. <br />  <br />  <br />  <br />  <br />    FIG. 6. (A) Schematic illustration mutagenesis BAC20 remove gB-encoding sequences. First, recombinant plasmid pGETrec encoding L-arabinose- <br /> inducible recE, recT, bacteriophage &#226;?&#173; gam gene transformed BAC20-containing DH10B cells (indicated step 1). Subsequently, PCR amplification <br /> of Kanr gene plasmid pEGFP-N1 (Clontech) primers contained 50-nucleotide homology arms bordering gB deletion, 1.6-kbp PCR <br /> amplicon electroporated DH10B cells harboring BAC20 pGETrec (indicated step 2). Bacterial suspensions plated agar containing 30 &#226;?&#174;g <br /> of kanamycin ml 30 &#226;?&#174;g chloramphenicol ml. Double-resistant colonies picked subjected analysis. (B) Schematic illustration the <br /> location gB gene 180-kbp MDV-1 genome replacement gB ORF Kanr gene homologous recombination recombinant BAC <br /> clone 20&#226;&#338;&#172;gB. (C) Schematic representation BamHI, BglI, EcoRI, StuI restriction fragment sizes 20&#226;&#338;&#172;gB generated insertion Kanr gene instead <br /> of gB ORF. Also indicated restriction fragment sizes parental BAC20. Fragment sizes given kilobase pairs calculated according to <br /> published sequences (12). <br />  <br />  <br />  <br />    Analysis gB-negative MDV-1 reconstituted 20&#226;&#338;&#172;gB.                           After primary transfection, plaque formation observed <br /> Because gB essential growth herpesviruses analyzed                     coseeding gB-expressing MgB1 cells (data not <br /> to date (reviewed reference 25), QM7 cell line                            shown). From results, concluded (i) gB essen- <br /> expressed MDV-1 gB control HCMV imme-                               tially required cell-to-cell spread MDV-1 cultured <br /> diate-early promoter generated (Table 1). IIF analyses                           cells (ii) 49 nucleotides extreme 3&#226;&#172;&#732; <br /> demonstrated 90% cells MgB1 cell                        end MDV-1-specific putative LORF5 required <br /> line constitutively expressed MDV-1 gB demonstrated using                         growth MDV-1 strain 584Ap80C, 20&#226;&#338;&#172;gB <br /> MAb 2K11 convalescent chicken serum (anti-MDV) (Fig.                            virus able grow cells provided MDV-1 gB but <br /> 8). To analyze growth BAC20 20&#226;&#338;&#172;gB various cell                             LORF5 trans. <br /> types, DNA prepared used transfect CEF, QM7, or <br /> MgB1 cells. At 3 5 days transfection, virus plaques                                                       DISCUSSION <br /> were observed cells transfected BAC20 (Fig. 9). <br /> However, transfection 20&#226;&#338;&#172;gB DNA, MDV-1-specific                                The salient findings report (i) complete <br /> plaques observed gB-expressing MgB1 cells (Fig.                         genome attenuated MDV-1 strain 584Ap80C (38) was <br /> 9). In CEF QM7 cells transfected 20&#226;&#338;&#172;gB, single cells                        cloned infectious BAC (ii) recently developed <br /> expressed MDV-1 pp38 protein demonstrated reac-                            recE-catalyzed mutagenesis protocol successfully applied <br /> tivity MAb H19 (9), plaque formation inhibited                          delete essential gB gene MDV-1 genome. To <br /> (Fig. 9). These results gB required MDV-1 cell-                         knowledge, gB-negative virus reported represents <br /> to-cell spread vitro confirmed coseeding 20&#226;&#338;&#172;gB-                           MDV-1 mutant deletion essential gene <br /> infected MgB1 cells fresh CEF, QM7, MgB1 cells.                              demonstrates power usefulness BACs ana- <br />  VOL. 74, 2000                                                                              MDV-1 BAC AND gB-NEGATIVE MUTANT                           11095 <br />  <br />  <br />  <br />  <br />    FIG. 7. Scanned image ethidium bromide-stained 0.8% agarose gel containing BAC20 20&#226;&#338;&#172;gB DNA cleaved BamHI (B), EcoRI (E), BglI <br /> (Bg), StuI (S) separated 0.8% agarose gel electrophoresis (left panel). DNA fragments transferred nylon membranes hybridized a <br /> digoxigenin-labeled Kanr- gB-specific probe. The Kanr probe prepared labeling 1,015-bp BlnI fragment pEGFP-N1, gB probe prepared <br /> by labeling gB sequences released plasmid pcMgB. <br />  <br />  <br />  <br /> lyze genes gene products slowly growing strictly                      ing infectious herpesvirus genomes independently eu- <br /> cell-associated herpesviruses.                                                   karyotic cells rapid efficient mutagenesis exploiting <br />    Although MDV important pathogen chickens                        recombination apparatus prokaryotes. The BAC cloning <br /> causes T-cell tumors high mortality infected animals                      mutagenesis methods applied various <br /> (24, 29), little known function individual genes                 herpesviruses comprising subfamilies, including the <br /> and gene products lytic, latent, tumor phase                    Alphaherpesvirinae herpes simplex virus PrV (3, 6, 15, 32, <br /> infection. Functional analyses MDV-1 genes gene prod-                     33, 34). Particularly MDV-1, however, BAC cloning and <br /> ucts using mutant virus impaired main rea-                     mutagenesis certainly major advantage. Once the <br /> sons. First, cultured cells infected MDV-1 yield                     MDV-1 genome cloned BAC stably main- <br /> free infectious virus, second, efficient growth MDV-1                  tained E. coli, generation mutants analyses essen- <br /> cultured cells appeared restricted primary second-                   tial genes relatively easy. We cloned the <br /> ary chicken duck cells (29). Hence, generation virus                       complete genome MDV-1 strain 584Ap80C infectious <br /> recombinants cotransfection eukaryotic cells, i.e., ho-                 BAC. Strain 584Ap80C descendant vv&#226;&#171;&#185; MDV-1 <br /> mologous recombination used mutagenize                         strain 584A 80 serial passages CEF (39). Analysis of <br /> Alphaherpesvirinae laborious time-consuming                         cloned MDV-1 genomes present BAC19, BAC20, and <br /> and required constant supply primary cells.                               BAC24 demonstrated despite high similarities, variations <br />    The introduction BAC cloning herpesvirus genomics                     restriction enzyme patterns obvious. This heterogene- <br /> by Messerle coworkers (15) provided basis maintain-                    ity attributed variations BamHI-D -H <br />  11096       SCHUMACHER ET AL.                                                                                                                         J. VIROL. <br />  <br />  <br />  <br />  <br />   FIG. 8. Confocal laser scan analysis MgB1 cells constitutively expressing MDV-1 gB. MgB1 QM7 cells seeded glass coverslips incubated with <br /> anti-gB MAb 2K11 convalescent chicken serum anti-MDV (&#226;?&#163;-MDV). Secondary antibodies anti-mouse anti-chicken immunoglobulin G conjugated Alexa <br /> 488 (Molecular Probes). Nuclei counterstained propidium iodide. The view shown 115 115 &#226;?&#174;m. <br />  <br />  <br />  <br /> fragments, i.e., TRL IRL region genome (11). It                     13, 31). Amplification tandem 132-bp repeats asso- <br /> is known different numbers 132-bp tandem repeats                       ciated loss oncogenicity constant low num- <br /> present different MDV-1 strains number                          ber units demonstrated virulent strains (5, 6). <br /> repeats increases serial passage cultured cells (4, 5, 11,                However, recent work widely used Rispens vaccine <br />  <br />  <br />  <br />  <br />    FIG. 9. IIF analysis MgB1, QM7, CEF cells transfection BAC20 (upper panels) 20&#226;&#338;&#172;gB (lower panels). At 4 days transfection, cells were <br /> fixed acetone incubated anti-pp38 MAb H19. The secondary antibody anti-mouse immunoglobulin G conjugated Alexa 488 (Molecular Probes). <br /> Whereas MDV-1 plaques observed cell lines transfection BAC20 DNA, viral plaques observed MgB1 cells transfection with <br /> 20&#226;&#338;&#172;gB. Only single infected cells observed QM7 CEF cells (arrowheads). Magnification, &#226;&#171;&#187;250. <br />  VOL. 74, 2000                                                                    MDV-1 BAC AND gB-NEGATIVE MUTANT                               11097 <br />  <br />  <br /> strain demonstrated direct correlation         combination analyze essential MDV-1 genes. In addition, <br /> of small numbers 132-bp repeats virulence (35). For        comparative analyses gene protein functions various <br /> MDV-1 strain 584Ap80C, hybridization cleaved viral DNA             Alphaherpesvirinae include MDV-1 use this <br /> with BamHI-D fragment gave diffuse banding patterns,              allows studies distantly related members the <br /> indicating variable number repeats present virus          viral subfamily, instance, PrV able grow on <br /> population. In contrast, single bands identified         QM7 cells. <br /> each BAC clones probe. The sizes           In summary, cloning complete MDV-1 genome an <br /> bands cleavage BamHI EcoRI varied BAC19,             infectious BAC, establishment fast efficient mutagen- <br /> BAC20, BAC24, indicating genomes containing dif-             esis system, use permanent QM7 cell line should <br /> ferent numbers 132-bp repeats cloned. This            greatly facilitate future analyses MDV-1 genes. In addition, <br /> interpretation substantiated PCR analyses targeting        possible generate novel generation MDV-1 <br /> 132-bp repeats. Whereas ladder-like appearance PCR             vaccines based BACs. At present, MDV-1 BACs <br /> products obtained DNA 584Ap80C,                containing genomes virulent strains like RB1B being <br /> typical attenuated MDV-1 strains (2), distinct bands         generated. With cloned genomes hand, detailed <br /> amplified cloned viral DNA BAC19, BAC20,                 assessment genes expressed lytic, latent, tumor <br /> BAC24 (Zelnik et al., unpublished). It con-             phases MDV-1 infection possible. <br /> cluded variations restriction enzyme patterns the <br /> different BAC clones resulted various numbers tandem                                   ACKNOWLEDGMENTS <br /> 132-bp repeats present individual clones. These varia- <br /> tions TRL IRL did influence              We gratefully acknowledge expert technical assistance Kerstin <br />                                                                       Wink. We indebted Richard L. Witter, ADOL, East Lansing, <br /> infectivity cloned DNA infectious virus was <br />                                                                       Mich., generously providing MDV-1 strains, including 584Ap80C, <br /> recovered transfection DNA isolated         Martin Messerle, LMU, Munich, Germany, providing plas- <br /> different BAC clones.                                                 mid pHA1 help support BAC cloning. We thank Panos <br />    After cloning complete MDV-1 genome proof            Ioannou, Murdoch Institute, Melbourne, Australia, providing plas- <br /> infectivity MDV-1 DNA maintained propagated F            mid pGETrec constant advice. Lucy Lee, ADOL, Jean-Fran- <br /> plasmid E. coli, recently developed mutagenesis           cois Vautherot, Institut National la Recherche Agronomique, <br /> which permits homologous recombination linear DNA frag-            Tours, France, generously provided MAbs H19 2K11, respectively. <br /> ments BACs (17, 18, 41) used delete gB-encoding             This work supported grant QLK2-CT-1999-00601 the <br /> sequences BAC20. The mutagenesis based recE, recT,           Commission European Union. <br /> and recB- recC-suppressing bacteriophage &#226;?&#173; gam gene <br />                                                                                                       REFERENCES <br /> present plasmid pGETrec (18). The big advantages this <br />                                                                        1. Anderson, A. S., M. S. Parcells, R. W. Morgan. 1998. The glycoprotein <br /> mutagenesis follow. (i) Only 30- 50-bp homology                 D (US6) homolog essential oncogenicity horizontal transmission <br /> arms needed target specific sequence deleted,              Marek&#226;&#8364;&#8482;s disease virus. J. Virol. 72:2548&#226;&#8364;&#8220;2553. <br /> i.e., deletion ORF achieved need          2. Becker, Y., E. Tabor, Y. Asher, I. Davidson, M. Malkinson, R. L. Witter. <br /> clone recombination cassettes. (ii) The method fast. (iii)        1993. PCR detection amplified 132 bp repeats Marek&#226;&#8364;&#8482;s disease virus <br />                                                                           type 1 (MDV-1) DNA serve indicator critical genomic rear- <br /> The pGETrec vector rapidly lost bacterial cells            rangement leading attenuation virus virulence. Virus Genes 7:277&#226;&#8364;&#8220; <br /> absence antibiotic selection. After electroporation gB          287. <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>knockout</span> PCR product pGETrec-containing BAC20 cells,              3. Borst, E. M., G. Hahn, U. H. Koszinowski, M. Messerle. 1999. Cloning <br /> between 10 30 Camr Kanr double-resistant colonies                 human cytomegalovirus (HCMV) genome infectious bacterial <br />                                                                           artificial chromosome Escherichia coli: new approach construction of <br /> were obtained. One clones named 20&#226;&#338;&#172;gB cho-                 HCMV mutants. J. Virol. 73:8320&#226;&#8364;&#8220;8329. <br /> sen analyses lost pGETrec immedi-           4. Bradley, G., M. Hayashi, G. Lancz, A. Tanaka, M. Nonoyama. 1989. <br /> ately plated agar containing chloramphenicol              Structure Marek&#226;&#8364;&#8482;s disease virus BamHI-H gene family: genes of <br /> and kanamycin. Southern blot analyses demonstrated success-               putative importance tumor induction. J. Virol. 63:2534&#226;&#8364;&#8220;2542. <br />                                                                        5. Bradley, G., G. Lancz, A. Tanaka, M. Nonoyama. 1989. Loss Marek&#226;&#8364;&#8482;s <br /> ful deletion gB gene insertion Kanr gene            disease virus tumorigenicity associated truncation RNAs tran- <br /> in 20&#226;&#338;&#172;gB. MDV-1 recovered transfection CEF cells                 scribed BamHI-H. J. Virol. 63:4129&#226;&#8364;&#8220;4135. <br /> with 20&#226;&#338;&#172;gB unable spread infected cells neigh-          6. Brune, W., C. Menard, U. Hobom, S. Odenbreit, M. Messerle, U. H. <br /> boring cells. However, 20&#226;&#338;&#172;gB growth MgB1 cells pro-               Koszinowski. 1999. Rapid identification essential nonessential her- <br />                                                                           pesvirus genes direct transposon mutagenesis. Nat. Biotechnol. 17:360&#226;&#8364;&#8220; <br /> vide MDV-1 gB trans similar BAC20, indi-                364. <br /> cating MDV-1 gB, like counterparts                   7. Brunovskis, P., L. F. Velicer. 1995. The Marek&#226;&#8364;&#8482;s disease virus (MDV) <br /> herpesviruses, essential cell-to-cell spread infectivity.       unique short region: alphaherpesvirus-homologous, fowlpox virus-homolo- <br /> Because MDV-1 highly cell associated cultured cells             gous, MDV-specific genes. Virology 206:324&#226;&#8364;&#8220;338. <br />                                                                        8. Cantello, J. L., A. S. Anderson, A. Francesconi, R. W. Morgan. 1991. <br /> does release infectious virus culture medium,               Isolation Marek&#226;&#8364;&#8482;s disease virus (MDV) recombinant containing lacZ <br /> were able investigate possible role MDV-1 gB               gene Escherichia coli stably inserted MDV US2 gene. J. Virol. <br /> virus entry. It noted deleting majority        65:1584&#226;&#8364;&#8220;1588. <br /> gB gene BAC20, putative MDV-specific LORF5                  9. Cui, Z. Z., D. Yan, L. F. Lee. 1990. Marek&#226;&#8364;&#8482;s disease virus gene clones <br />                                                                           encoding virus-specific phosphorylated polypeptides serological charac- <br /> (12) affected. However, 20&#226;&#338;&#172;gB able                terization fusion proteins. Virus Genes 3:309&#226;&#8364;&#8220;322. <br /> grow gB trans-complementing cells express             10. Delecluse, H. J., T. Hilsendegen, D. Pich, R. Zeidler, W. Hammer- <br /> LORF5, observed <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> 20&#226;&#338;&#172;gB virus at-                 schmidt. 1998. Propagation recovery intact, infectious Epstein-Barr <br /> tributed solely absence gB expression. The generated            virus prokaryotic human cells. Proc. Natl. Acad. Sci. USA 95:8245&#226;&#8364;&#8220; <br />                                                                           8250. <br /> gB mutant represents example MDV-1               11. Fukuchi, K., M. Sudo, Y.-S. Lee, A. Tanaka, M. Nonoyama. 1984. <br /> deletion essential gene demonstrates power               Structure Marek&#226;&#8364;&#8482;s disease virus DNA: detailed restriction enzyme map. <br /> the BAC cloning mutagenesis system, especially               J. Virol. 51:102&#226;&#8364;&#8220;109. <br /> useful MDV-1. MDV-1 BACs permanent cell line              12. Lee, L. F., P. Wu, D. Sui, D. Ren, J. Kamil, H. J. Kung, R. L. Witter. <br />                                                                           2000. The complete unique long sequence overall genomic organi- <br /> QM7, allows MDV-1 propagation which&#226;&#8364;&#8221;unlike                      zation GA strain Marek&#226;&#8364;&#8482;s disease virus. Proc. Natl. Acad. Sci. USA <br /> the quail fibroblast cell line QT35&#226;&#8364;&#8221;does harbor MDV-1                 97:6091&#226;&#8364;&#8220;6096. <br /> sequences (Zelnik et al., unpublished), represent excellent        13. Maotani, K., K. Kanamori, K. Ikuta, S. Ueda, S. Kato, K. Hirai. 1986. <br />  11098         SCHUMACHER ET AL.                                                                                                                                   J. VIROL. <br />  <br />  <br />       Amplification tandem repeat inverted repeats Marek&#226;&#8364;&#8482;s disease             short unique region viral genome essential viral <br />       virus DNA serial vitro passage. J. Virol. 58:657&#226;&#8364;&#8220;659.                         replication cell culture vaccine-induced immunity chickens. Virol- <br /> 14.   Meindl, A., N. Osterrieder. 1999. The equine herpesvirus type 1 US2                 ogy 195:140&#226;&#8364;&#8220;148. <br />       homolog encodes nonessential membrane-associated virion component.              28.   Sambrook, J., D. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />       J. Virol. 73:3430&#226;&#8364;&#8220;3437.                                                                 laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Har- <br /> 15.   Messerle, M., I. Crnkovic, W. Hammerschmidt, H. Ziegler, U. H. Ko-                  bor, N.Y. <br />       szinowski. 1997. Cloning mutagenesis herpesvirus genome            29.   Schat, K. A. 1985. Characteristics virus, p. 77&#226;&#8364;&#8220;112. In L. N. Payne <br />       infectious bacterial artificial chromosome. Proc. Natl. Acad. Sci. USA 94:              (ed.), Marek&#226;&#8364;&#8482;s disease. Kluwer Academic Publishers, Hingham, Mass. <br />       14759&#226;&#8364;&#8220;14763.                                                                      30.   Schat, K. A., B. J. van Iddekinge, H. Boerrigter, P. H. O&#226;&#8364;&#8482;Connell, G. <br /> 16.   Morgan, R. W., J. L. Cantello, C. H. McDermott. 1990. Transfection               Koch. 1998. Open reading frame L1 Marek&#226;&#8364;&#8482;s disease herpesvirus not <br />       chicken embryo fibroblasts Marek&#226;&#8364;&#8482;s disease virus DNA. Avian Dis.                   essential vitro vivo virus replication establishment of <br />       34:345&#226;&#8364;&#8220;351.                                                                             latency. J. Gen. Virol. 79:841&#226;&#8364;&#8220;849. <br /> 17.   Muyrers, J. P., Y. Zhang, G. Testa, A. F. Stewart. 1999. Rapid modifi-        31.   Silva, R. F., R. L. Witter. 1985. Genomic expansion Marek&#226;&#8364;&#8482;s disease <br />       cation bacterial artificial chromosomes ET-recombination. Nucleic                 virus DNA associated serial vitro passage. J. Virol. 54:690&#226;&#8364;&#8220;696. <br />       Acids Res. 27:1555&#226;&#8364;&#8220;1557.                                                          32.   Smith, G. A., L. W. Enquist. 1999. Construction transposon mu- <br /> 18.   Narayanan, K., R. Williamson, Y. Zhang, A. F. Stewart, P. A. Ioannou.               tagenesis Escherichia coli full-length infectious clone pseudorabies <br />       1999. Efficient precise engineering 200 kb beta-globin human/                  virus, alphaherpesvirus. J. Virol. 73:6405&#226;&#8364;&#8220;6414. <br />       bacterial artificial chromosome E. coli DH10B using inducible homol-        33.   Smith, G. A., L. W. Enquist. 2000. A self-recombining bacterial artificial <br />       ogous recombination system. Gene Ther. 6:442&#226;&#8364;&#8220;447.                                       chromosome application analysis herpesvirus pathogenesis. <br /> 19.   Osterrieder, N. 1999. Sequence initial characterization UL10                 Proc. Natl. Acad. Sci. USA 97:4873&#226;&#8364;&#8220;4878. <br />       (glycoprotein M) UL11 homologous genes serotype 1 Marek&#226;&#8364;&#8482;s Dis-             34.   Suter, M., A. M. Lew, P. Grob, G. J. Adema, M. Ackermann, K. Shortman, <br />       ease Virus. Arch. Virol. 144:1853&#226;&#8364;&#8220;1863.                                                 C. Fraefel. 1999. BAC-VAC, novel generation (DNA) vaccines: a <br /> 20.   Osterrieder, N., A. Neubauer, C. Brandmu     &#194;&#168;ller, B. Braun, O.-R. Kaaden,         bacterial artificial chromosome (BAC) containing replication-competent, <br />       J. D. Baines. 1996. Th <br /> </body></html>