<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Developmental Biology                                                                                                                BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> echinus, required interommatidial cell sorting cell death in <br /> the Drosophila pupal retina, encodes protein homology <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ubiquitin-specific proteases <br /> Jeffrey M Copeland1, Ian Bosdet2, J Douglas Freeman2, Ming Guo3, <br /> Sharon M Gorski2 Bruce A Hay*1 <br />  <br /> Address: 1Division Biology, MC 156-29, California Institute Technology, Pasadena, CA 91125, USA, 2Genome Sciences Centre, British <br /> Columbia Cancer Agency. Vancouver, British Columbia V5Z 1L3, Canada 3Department Neurology, Brain Research Institute, The David <br /> Geffen School Medicine UCLA, Los Angeles, CA 90095, USA <br /> Email: Jeffrey M Copeland - copelanj@its.caltech.edu; Ian Bosdet - ibosdet@bcgsc.ca; J Douglas Freeman - dfreeman@bcgsc.ca; <br /> Ming Guo - mingfly@ucla.edu; Sharon M Gorski - sgorski@bcgsc.ca; Bruce A Hay* - haybruce@caltech.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 5 July 2007                                                        Received: 18 January 2007 <br />                                                                               Accepted: 5 July 2007 <br /> BMC Developmental Biology 2007, 7:82   doi:10.1186/1471-213X-7-82 <br /> This article available <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-213X/7/82 <br /> &#194;&#169; 2007 Copeland et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Programmed cell death used remove excess cells ommatidia the <br />                  Drosophila pupal retina. This death required establish crystalline, hexagonal packing of <br />                  ommatidia characterizes adult fly eye. In previously described echinus mutants, <br />                  interommatidial cell sorting, precedes cell death, occurred relatively normally. <br />                  Interommatidial cell death partially suppressed, resulting adult eyes contained excess <br />                  pigment cells, ommatidia mildly disordered. These results suggested that <br />                  echinus functions pupal retina primarily promote interommatidial cell death. <br />                  Results: We generated number new echinus alleles, likely <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>null mutants</span>. Analysis these <br />                  alleles provides evidence echinus roles cell sorting cell death. echinus encodes <br />                  protein homology ubiquitin-specific proteases. These proteins cleave ubiquitin-conjugated <br />                  proteins ubiquitin C-terminus. The echinus locus encodes multiple splice forms, including two <br />                  proteins lack residues thought critical deubiquitination activity. Surprisingly, <br />                  ubiquitous expression eye versions Echinus lack residues critical ubiquitin <br />                  specific protease activity, version predicted functional, rescue echinus loss-of- <br />                  function <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span>. Finally, genetic interactions detected echinus loss gain- <br />                  of-function number known apoptotic regulators. These include Notch, EGFR, caspases <br />                  Dronc, Drice, Dcp-1, Dream, caspase activators, Rpr, Hid, Grim, caspase inhibitor <br />                  DIAP1, Lozenge Klumpfuss. <br />                  Conclusion: The echinus locus encodes multiple splice forms protein homology to <br />                  ubiquitin-specific proteases, protease activity unlikely required echinus function, at <br />                  echinus overexpressed. Characterization likely echinus null alleles genetic <br />                  interactions suggests echinus acts novel point(s) regulate interommatidial cell sorting <br />                  and/or cell death fly eye. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 15 <br />                                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                           http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br />  <br /> Background                                                        lack Hid defects Drice activation IOC cell <br /> The adult Drosophila eye consists 750&#226;&#8364;&#8220;800 individual           death [10,27], mutants proteins are <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> eyes, known ommatidia, arranged            available. Together observations suggest that <br /> hexagonal lattice. Each ommatidium consists 8 pho-             IOC death driven, part, Hid-dependent <br /> toreceptors, 4 lens-secreting cone cells 2 primary pig-       inhibition DIAP1, facilitates activation Dronc <br /> ment cells. Ommatidia separated            Drice (Fig. 4 schematic). <br /> secondary tertiary (2&#194;&#176; 3&#194;&#176;) pigment cells, by <br /> sensory bristles. Each cell types occupies stereo-     Ubiquitination, presumably deubiquitination, <br /> typic position lattice. <span id='am-475' about='xsp:pattern' typeof='owl:Thing'>Pattern</span> formation       plays important roles regulation cell <br /> eye initiated 3rd larval instar wave mor-       death pathway. DIAP1 E3 ubiquitin ligase [28-32] <br /> phogenesis sweeps epithelial cell layer         ubiquitinate inactivate Dronc [15,16]. DIAP1 <br /> eye imaginal disc. First, photoreceptor cells      promote ubiquitination degradation of <br /> lens-secreting cone cells specified sequential        pro-apoptotic proteins binds Reaper <br /> inductive interactions. During early pupal stages, cone           [33]. DIAP1 ubiquitinates itself [28-32,34] and <br /> cells come cover photoreceptors. They recruit         DIAP1 ubiquitination stimulated RHG pro- <br /> two primary pigment cells, surround cone cells.         teins [28-32,34]. Many components ubiquitin <br /> Cells specified stage form         pathway identified regulators RHG-medi- <br /> interommatidial cell (IOC) lattice, ultimately         ated cell death. Examples include ubiquitin activating <br /> be composed secondary pigment cells, tertiary pigment          enzyme (uba1), components SCF-type E3 ubiq- <br /> cells, bristles. These cells initially appear undifferenti-   uitin ligase (skpA novel F-box gene, morgue) the <br /> ated unpatterned, layers IOCs           deubiquitinating enzyme fat facets [29,31,32]. However, <br /> separating neighboring ommatidia. Reorganization                  points proteins work regulate death <br /> begins presumptive lattice cells maximizing            largely unknown. <br /> contacts primary pigment cells other <br /> lattice cells. This results lattice cell connect-   As noted above, IOCs initially arranged double <br /> ing primary pigment cells,          triple rows ommatidia pupal eye. IOCs <br /> ommatidia separated single layer lattice            rearrange end-to end configuration form a <br /> cells, arranged end-to-end chain. About two-thirds          single layer row cells separating primary pigment <br /> of cells develop secondary pigment         cells different ommatidia. Cell death occurs after <br /> cells, makes face ommatidial          rearrangement, sorting, complete. A key player in <br /> hexagon, tertiary pigment cells, make alterna-        process immunoglobulin family protein Rough- <br /> tive vertices, bristle groups making ver-       est (Rst). In absence roughest IOCs remain <br /> tices. The remainder IOCs eliminated                stacked side-by-side multiple rows IOC death does <br /> apoptotic cell death [1,2].                                       occur [2]. Rst localized IOCs border <br />                                                                   IOCs primaries [35], process that <br /> Much cell death Drosophila takes form apoptosis         requires DE-cadherin Notch [36-38]. Recent observa- <br /> [3]. Caspase proteases central executioners            tions suggest Rst promotes sorting physically inter- <br /> apoptotic cell death [4]. Dronc required cell         acting Hibris, immunoglobulin family <br /> deaths fly [5-8], including IOCs [9].         membrane protein expressed primary pigment cells <br /> Once activated interactions adaptor Ark,         required IOC sorting death [37]. <br /> Dronc cleaves activates effector caspases Drice <br /> and Dcp-1 thought bring cell death [5,6].       The EGF receptor pathway provides important anti-apop- <br /> Drice activated stages IOC death           totic signals IOCs. Loss EGFR signaling pupal <br /> occurs [10], Drice mutants lack some, all, IOC        eye results fewer IOCs [39], activation EGFR or <br /> death, highlighting importance protease               Ras promotes IOC <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>survival</span> [40]. EGFR activation pro- <br /> [11,12]. DIAP1 cell death inhibitor suppresses      motes IOC survival negatively regulating <br /> activity Dronc caspases activated Dronc         hid levels pro-apoptotic activity [10,27,41]. Pro-sur- <br /> several different mechanisms [5,6,13-19]. Reaper (Rpr)            vival signaling EGFR antagonized by <br /> [20], Head involution defective (Hid) [21], Grim [22],            Notch-mediated signals (probably IOCs), which <br /> Sickle [23-25], Jafrac2 [26], known collectively       required remove excess IOCs [10,40]. The amount <br /> RHG proteins founding members Rpr, Hid            contact IOC primary pigment cells (which <br /> Grim, bind DIAP1 short-N-terminal motif              produce EGFR-dependent survival signals) opposed to <br /> and disrupt DIAP1-caspase interactions            IOCs (which produce Notch-mediated signals that <br /> mechanisms, effect unleashing          antagonise EGFR pathway) likely important <br /> cascade apoptosis-inducing caspase activity. Flies        calculus determines IOC fate. Ubiquitina- <br />  <br />  <br />                                                                                                                      Page 2 15 <br />                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                         http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> tion plays important roles regulating signaling         novel point, mechanisms inde- <br /> pathways. The EGFR monoubiquitinated following lig-          pendent USP activity. <br /> and binding, promotes receptor endocytosis and <br /> degradation, attenuating signaling [42]. In Notch      Results <br /> pathway, monoubiquitination ligands Notch           CG2904 encodes echinus, expressed low levels <br /> by multiple E3 ligases associated endocytic events      ubiquitously pupal retina <br /> that promote signaling. Ubiquitination Notch        As step cloning echinus locus used both <br /> E3 ligases associated internalization lysosomal     EMS mutagenesis P element excision generate new <br /> degradation [43].                                               echinus alleles. We identified X chromosome-linked P <br />                                                                 element insertion line, ecPlacZ, recessive rough eye <br /> Other proteins regulate IOC survival iden-       phenotype failed complement ec1 (Fig. 1A). We <br /> tified. The Runx DNA-binding protein Lozenge required        generated number excisions element. The <br /> for IOC death [44-46]. Lozenge pro-apoptotic activity        rough eye phenotype reverted these, indi- <br /> mediated ability induce expression Argos,      cating P element insertion responsible the <br /> a secreted inhibitor EGFR activation cone cells,      echinus phenotype. The ecPlacZ transposon located <br /> 2&#194;&#176; 3&#194;&#176; cells [44]. Lozenge activates expression      CG2901 CG2904, suggesting the <br /> Klumpfuss, transcription factor similarity        genes good candidates encode echinus. <br /> Wilm&apos;s Tumor suppressor, 2&#194;&#176; 3&#194;&#176; cells [44]. Klump- <br /> fuss function required normal levels IOC death,       Ommatidia wildtype flies arranged regular <br /> and genetic evidence suggests antagonizes EGFR          hexagonal pattern (Fig. 2A), extra IOCs not <br /> signaling downstream receptor activation [47].               observed pupal eyes (Fig. 2G; Table 1). In contrast, <br />                                                                 adult eyes ec1 flies rough (Fig. 2B), extra IOCs <br /> echinus (ec), defined single allele, ec1, identified   present pupal retinas (Fig. 2H). To determine if <br /> by Calvin Bridges 1918 X-chromosome-linked,             CG2904 encodes echinus generated flies expressing <br /> recessive, rough eye mutant (as described [48]). More        dsRNA corresponding CG2904 order trigger <br /> recently, Wolff Ready showed ec1 flies             RNAi-dependent knockdown CG2904 expression, <br /> decreased levels IOC death, like rst mutants.           control eye-specific-GMR promoter <br /> However, IOC sorting failed occur rst animals,      (GMR-CG2904-RNAi). Consistent hypothesis, <br /> sorting largely (though completely) intact ec1       GMR-CG2904-RNAi flies rough eye phenotype (Fig. <br /> flies [2,49]. Expression baculovirus caspase inhibi-     2C) extra IOCs (Fig. 2I). To create deletion alleles of <br /> tor p35 prevents death sorting, giving rise     CG2904 generated excisions nearby P element <br /> a pupal retinal phenotype similarities             insertion line, EP(X)1343, wildtype respect to <br /> (though identical to) observed ec1 flies [50].      echinus (Fig. 1A). Multiple excision lines identified <br /> Together observations suggested ec acts         rough eyes homozygotes. Each failed <br /> primarily subsequent sorting, regulating cell        complement ec1 ecPlacZ. Breakpoints these <br /> death directly [49-51]. To understand role echinus          determined, delete sequences <br /> plays bringing IOC death generated number         CG2904 transcript. We focused anal- <br /> of new echinus alleles re-examined phenotypes associ-       ysis these, designated ecEP&#226;&#710;&#8224;4 (Fig. 1A; Fig 1D, J). <br /> ated ec1 following extensive outcrossing remove         Pupal eyes ecEP&#226;&#710;&#8224;4 showed extra IOCs (Fig. 2J). <br /> modifiers. Multiple alleles, including putative null alleles,   We CG2904 gene disrupted the <br /> show defects cell sorting cell death. We          breakpoints excision allele generated alter- <br /> cloned echinus locus encodes multi-       nate P element insertion (ec&#226;&#710;&#8224;9; Figure 1H. Kramer, unpub- <br /> ple isoforms protein homology ubiquitin        lished). This mutant rough eyes extra IOCs <br /> specific protease (USP) family proteases. Somewhat           (data shown). More recently, number new P ele- <br /> unexpectedly, versions Echinus lack residues            ment insertion lines surrounding genomic region <br /> thought important USP catalytic activity res-     identified [52]. Several rough eye pheno- <br /> cue echinus loss-of-function sorting cell death phe-    type fails complement ec1, is <br /> notypes. We unable detect significant interactions      located near CG2904 (Figure 1A). We identified sev- <br /> between loss- gain-of-function echinus alleles        eral new EMS alleles echinus (see methods details). <br /> number known suspected regulators IOC death.           We sequenced CG2904 coding sequences of <br /> echinus primarily regulate cell sorting, loss       identified E125-Stop change ec56 (Figure <br /> death consequence earlier defect     1A). We identified stop <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span> (L792-Stop in <br /> process. Alternatively, echinus regulate cell sorting       splice form 1) EMS-derived ec3c3 allele [53]. Finally, <br /> and cell death, regulation death occurring         sequenced CG2904 coding nearby regions from <br />                                                                 original ec1 stock. Two significant alterations were <br />  <br />  <br />                                                                                                                     Page 3 15 <br />                                                                                               (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                                                  http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br />                                                                                                                       EP&#226;&#710;&#8224;4 <br />                                                                                                                   ec <br />                   A. Splice Form 1                                                                                ec &#226;&#710;&#8224;9 <br />  <br />  <br />  <br />  <br />                                                                                                               5 <br />                                                                                                            17 <br />                                                                                   28 <br />                                         7 <br />                                      92 <br />  <br />  <br />  <br />  <br />                                                                                                         09 <br />                                KG 8 <br />  <br />  <br />  <br />  <br />                                                                                    2 <br />                                     9 <br />  <br />  <br />  <br />  <br />                                                                                    ) <br />  <br />  <br />  <br />  <br />                                                                                                        3 <br />                                                                                 10 <br />                                   00 <br />  <br />  <br />  <br />  <br />                                                                                 cZ <br />  <br />  <br />  <br />  <br />                                                                                                    KG <br />                            SU )11 <br />  <br />  <br />  <br />  <br />                                                                                                     34 <br />                                                                             KG <br />                                                                             (la <br />  <br />  <br />  <br />  <br />                                                                                                  )1 <br />                                (X <br />  <br />  <br />  <br />  <br />                                                                                                 P) <br />  <br />  <br />  <br />  <br />                                                                                                                                                           pi a <br />                                                                                               (X <br />                                                                        ec P <br />  <br />  <br />                                                                          P) <br />                              P) <br />  <br />  <br />  <br />  <br />                                                                                        SU <br />                             EP <br />  <br />  <br />  <br />  <br />                                                                                                                                                          Co <br />                                                                       SU <br />  <br />  <br />  <br />  <br />                                                                                             EP <br />                                                                                        P( <br />                                                                                                                       ec 56   ec1     ec3c3 <br />  <br />  <br />  <br />  <br />                                                                    P( <br />                          P(                 AUG                                                                       *       *       * <br />                                                 tRNA:P:3E CG2901 <br />                                   CR32785 <br />  <br />  <br />                   fd3F     1                2                                      4                                   7 8    9 <br />                                                                                                                                                   5 kb        roX1 <br />  <br />                                                                                                                                     RH68894 <br />                                                 Cys Box                                                                His Box <br />  <br />  <br />  <br />  <br />                  B. Splice Form 2 <br />                                                                             AUG <br />                                                 tRNA:P:3E CG2901 <br />                                   CR32785 <br />  <br />                                                                         3          4                                   7 8 9 <br />                   fd3F                                                                                                                                        roX1 <br />                                                                                                                                               RH68894 <br />  <br />                                                  Catalytically Inactive UCH Domain <br />  <br />  <br />  <br />  <br />                  C. Splice Form 3                                                                       AUG <br />                                                 tRNA:P:3E CG2901 <br />                                   CR32785 <br />  <br />  <br />                   fd3F                                                                              5             6   7 8 9 <br />                                                                                                                                                               roX1 <br />  <br />  <br />                                                                    Catalytically Inactive UCH Domain                                RH68894 <br />  <br />  <br />  <br />  <br /> echinus <br /> Figure gene <br />          1 structure, mutants surrounding genomic region <br /> echinus gene structure, mutants surrounding genomic region. Echinus (CG2904) exons indicated shaded boxes. <br /> Exons numbered sequentially respect 5&apos; end gene. The shaded boxes diagonal lines represent the <br /> conserved USP domain. Three different splice versions identified cDNA sequencing illustrated panels A-C. <br /> Exons common splice versions noted lines numbered exons 7,8, 9. Nearby genes indicated <br /> (open arrows), Flybase annotated P element insertions, ecPlacZ. Open triangles indicate P element insertions are <br /> wildtype respect echinus, filled triangles indicate P element insertion lines rough eyes fail complement <br /> echinus. The location EMS-induced point mutations echinus, ec56 ec3c3, mutations identified ec1 (a <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> and Copia insertion) indicated asterisks. The locations breakpoints echinus deletion alleles ecEP&#226;&#710;&#8224;4 ec&#226;&#710;&#8224;9 <br /> are indicated dotted lines top. (A) ec-SF1 encodes version echinus contains Cys His box residues impor- <br /> tant USP catalysis, noted highlighted residues. These surrounding sequences highly conserved predicted <br /> echinus homologs insect species. Highly related sequences number species. (B) The ec-SF2 <br /> transcript initiates downstream exon, contains initial methionine coding sequences lack Cys box. The <br /> 3&apos; exon containing His box sequences (exon 9) present. (C) The ec-SF3 transcript initiates distinct position further <br /> 3&apos;, contains His box sequences, lacks Cys box sequences. Sequences highly related alternative N-terminus <br /> are number species. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      Page 4 15 <br />                                                                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                             http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> noted: R295-Stop mutation coding region,                 A                   B                    C                    D             E              F <br />  <br /> a Copia element inserted 3&apos; transcript. TUNEL <br /> assays and/or anti-active caspase immunostaining con- <br /> ducted using ec&#226;&#710;&#8224;9, ecEP&#226;&#710;&#8224;4, ecPlacZ, ec56, confirmed a <br /> reduction apoptosis mid-pupal retinas (see Addi-                 G                   H                    I                    J             IK             L <br />                                                                                                     * <br />                                                                                       0 <br />                                                                                    30 2                                      * <br /> tional File 1).                                                                 B <br />                                                                                  2 <br />                                                                                   0 <br />  <br />                                                                                                                       * <br />                                                                                                                        * <br />                                                                                  20 <br />                                                                                       0 <br />                                                                                   30 2 B                    * <br />                                                                                                         * <br />                                                                                                                        *         * <br /> Together, results strongly suggest CG2904 <br /> is echinus. To test definitively asked expression                 w-                  ec 1           GMR-ec(RNAi)             ec EP&#226;&#710;&#8224;4   GMR-ec (SF1)   ec <br />                                                                                                                                                                        EP&#226;&#710;&#8224;4 <br />                                                                                                                                                                           ; GMR-ec (SF1) <br />  <br />  <br /> of CG2904 rescue echinus phenotype. We iden-                 M                   N                    O <br />  <br /> tified cDNAs CG2904 larval-pupal <br /> library (see below). One these, designated ec-SF1, was <br /> introduced flies control GMR pro-                    ec 1 (A2C1)         ec 1 (B1B1)          ec 3c3 <br /> moter (GMR-ec-SF1 flies). These flies wildtype- <br /> appearing eyes IOC number (Fig. 2E,K),                    IOC <br />                                                                        adult <br />                                                                        Figure <br />                                                                        Flies sorting, <br />                                                                               fly 2eyes <br />                                                                                     mutations <br />                                                                                    increase <br />                                                                                            various <br />                                                                                                 CG2904 <br />                                                                                                   IOC have <br />                                                                                                             number <br />                                                                                                     genotypes       (A-F) <br />                                                                                                                  rough eyes, <br />                                                                                                                           SEMdefects <br />                                                                                                                                viewsinof <br /> introduced echinus ecEP&#226;&#710;&#8224;4 background, GMR-ec-                 Flies mutations CG2904 rough eyes, defects in <br /> SF1 restored adult ommatidial patterning normal                    IOC sorting, increase IOC number (A-F) SEM views of <br /> IOC cell death (Figure 2F,L). RH68894 represents group               adult fly eyes various genotypes. (G-O) Pupal retinas of <br />                                                                        various genotypes stained anti-Dlg. (A, G) Wildtype flies <br /> of 3 kb cDNA species overlap echinus anti- <br />                                                                        regularly spaced ommatidia invariant number of <br /> sense DNA strand. RH68894 predicted non-cod-                IOCs. Cell types indicated bristle (B), 2&#194;&#176;, 3&#194;&#176;, asterisk <br /> ing RNA, based lack reading frame signif-             represent extra IOCs. (B,H) ec1 flies obtained the <br /> icant size. To test RH68894 role                    Bloomington Stock center rough eyes modest <br /> mitigating echinus phenotype, GMR-RH68894                      number extra 2&#194;&#176; 3&#194;&#176; pigment cells. (C,I) GMR-driven <br /> introduced fly (GMR-RH68894 flies). Adult eyes                RNAi CG2904 results flies rough eyes large <br /> of GMR-RH68894 flies appear wildtype. In addition,                     increase IOCs, stacked side-by-side parallel <br /> when introduced ecEP&#226;&#710;&#8224;4, GMR-RH68894 failed                     rows. (D,J) Flies homozygous deletion CG2904, <br /> rescue alter echinus rough eye phenotype. Together              ecEP&#226;&#710;&#8224;4, rough eyes, large increase IOCs, many <br /> these observations demonstrate CG2904 (hereafter                  cells stacked side-by-side parallel rows. (E,K) GMR- <br /> simply referred echinus), RH68894, encodes               dependent expression ec-SF1 effect adult <br />                                                                        eye does cause excess death IOCs. (F,L) <br /> echinus. <br />                                                                        Expression GMR-ec-SF1 restores normal levels IOC <br />                                                                        death ecEP&#226;&#710;&#8224;4 flies. (M,N) Pupal eyes independent <br /> Pupal retinas animals homozygous new alleles               stocks ec1 outcrossed 5 generations. There are <br /> echinus ecEP&#226;&#710;&#8224;4, ec&#226;&#710;&#8224;9 ec56,           increased numbers IOCs compared original ec1 <br /> wildtype flies expressing GMR-CG2904-RNAi, showed                    stock, extra cells aligned side-by-side parallel <br /> striking difference retinas mutant ec1 (as                    rows. (O) Pupal eyes ec3c3 flies modest increase <br /> obtained Bloomington Stock Center). ec1 pupal                 IOC number defects cell sorting. <br /> eyes showed significant increase IOC number subse- <br /> quent time death normally occurs, this <br /> increase associated modest level side-              by-side alignment IOCs (Fig. 2; Table 1) [2]. One the <br />                                                                        new EMS alleles isolated, ec3c3, contains stop <br /> Table 1: Average Number IOCs <br />                                                                        codon near C-terminus echinus coding region, <br />              Genotype                        Average no. IOC           showed similar phenotype (Fig. 2O). ec3c3 likely be <br />                                                                        partial loss-of-function allele echinus ec3c3/ <br />                  w-                                 9.0                Df(1)HC244 results stronger adult rough eye pheno- <br />                  ec1                            10.6 &#194;&#177; 1.5             type observed homozygous ec3c3 flies (see <br />             GMR-ecRNAi                          14.1 &#194;&#177; 2.1             Additional File 2). In contrast, pupal eyes ecEP&#226;&#710;&#8224;4 and <br />                ecEP&#226;&#710;&#8224;4                           14.1 &#194;&#177; 1.7             GMR-CG2904-RNAi (Fig. 2), ec&#226;&#710;&#8224;9 ec56 (data not <br />            GMR-ec (SF1)                          9.1 &#194;&#177; 0.3 <br />                                                                        shown) showed greatly increased number IOCs <br />        ecEP&#226;&#710;&#8224;4; GMR-ec (SF1)                      9.6 &#194;&#177; 1.0 <br />             ec1 (A2C1)                          14.8 &#194;&#177; 1.8             (Table 1) extra cells aligned side- <br />             ec1 (B1B1)                          14.4 &#194;&#177; 2.2             by-side. The Bloomington ec1 stock outcrossed to <br />                 ec3c3                           10.6 &#194;&#177; 1.0             wildtype flies 5 generations independent exper- <br />                                                                        iments. Interestingly, pupal eyes out- <br /> Table 1 shows average number IOCs genotypes             crossed lines showed increase number extra <br /> indicated. Examples adult pupal eyes genotypes   IOCs (Table 1) IOC cell stacking (Fig. 2M,N). Impor- <br /> shown Fig. 2. <br />                                                                        tantly, sorting death phenotypes rescued <br />  <br />  <br />                                                                                                                                                              Page 5 15 <br />                                                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                             http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> by GMR-dependent expression ec-SF1 multiple echi-          A                                   B <br /> nus alleles (Fig. 2). Together, observations suggest                                                                       * <br /> that original ec1 line picked sup-                                P                                 * * <br />                                                                                                P                       *         * <br /> pressor mutations true echinus null pheno- <br /> type pupal eye results extra IOCs, of <br />                                                                                          P <br /> these cells stacked side-by-side. In addition,                                     P <br /> seems direct correlation severity of <br /> defects sorting cell death. Thus, pupal ret- <br /> inas original ec1 allele ec3c3 displayed mild <br /> defects sorting IOC death, retinas          C          CC           D                        E <br /> deletion allele ecEP&#226;&#710;&#8224;4, outcrossed ec1, GMR-CG2904-RNAi,                    CC                                P <br /> and ecEP&#226;&#710;&#8224;4/Df(1)HC244 (a deficiency covers                                                    P <br /> echinus locus) (data shown), displayed more <br /> severe defects sorting IOC death. These observa-                                                   P <br />                                                                                                                P <br />                                                                                                                             **** ** <br /> tions raise question echinus decrease-                                                                    * ** <br /> in-IOC death phenotype result loss echinus func-       F                       F&apos;                       F&quot; <br /> tion death activator, secondary consequence a <br /> failure sorting, precludes death signaling (dis- <br /> cussed below). <br />  <br /> cDNAs echinus isolated early embryos <br /> [54], pupal eyes (this work), genetic inter- <br /> actions echinus genes result pheno- <br /> types tissues eye described           G                      G&apos;                        G&quot; <br /> [48,52]. Thus, likely echinus expressed in, and <br /> plays roles tissues eye, focus <br /> in work pupal eye. To determine echinus <br /> expression pattern tissue carried tissue in <br /> situ hybridizations pupal retinas antisense echi- <br /> nus cDNA probe. echinus transcripts detected at <br /> low levels cone cells, primary pigment cells IOCs          H <br /> prior to, period IOC death (Fig. 3A,B). <br /> The ecPlacZ allele carries version lacZ functions as <br /> an enhancer trap. Therefore, second, perhaps <br /> more sensitive method visualizing echinus expression, <br /> we examined &#206;&#178;-gal expression pupal retinas this <br /> line. Consistent results echinus tissue situ <br /> hybridizations, &#206;&#178;-gal expressed uniformly cone <br /> cells, primary pigment cells IOCs wildtype (heter-         the <br />                                                                   SF1 <br />                                                                   an <br />                                                                   The pupal <br />                                                                       or <br />                                                                        echinus <br />                                                                   Figure  an <br />                                                                           3 eye, <br />                                                                          important <br />                                                                                transcript <br />                                                                                   andsite <br />                                                                                        GAL4-driver-dependent <br />                                                                              ec-silencing microRNA <br />                                                                                            ec <br />                                                                                               expressed <br />                                                                                                  action suggests <br />                                                                                                                uniform <br />                                                                                                                    expression <br />                                                                                                                        pigment <br />                                                                                                           low,         ofcells <br />                                                                                                                            levels  in <br />                                                                                                                                    ec- <br /> ozygous ecPlacZ) pupal retinas (Fig. 3C&#226;&#8364;&#8220;E). These observa-        The echinus transcript expressed low, uniform levels the <br />                                                                   pupal eye, GAL4-driver-dependent expression ec-SF1 an <br /> tions exclude possibility Echinus protein      ec-silencing microRNA suggests pigment cells impor- <br /> differentially translated specific cells, sug-     tant site ec action. (A,B) Tissue situ hybridization echi- <br /> gest transcription echinus specific populations     nus antisense probe complementary splice forms 28 hr <br />                                                                   APF pupal retinas. (A) Focal plane showing primary pigment cells. <br /> IOCs critical point cell sorting cell death reg-   (B) Focal plane showing IOCs. (C-E) Pupal retinas hetero- <br /> ulation.                                                          zygous ecPlacZ/+ flies showing anti-&#206;&#178; galactosidase staining cone <br />                                                                   cells (C), primary pigment cells (D), IOCs (E). Cell types indi- <br />                                                                   cated cones (C), primaries (P), IOCs (*). (F-F&quot;) 24 hr pupal <br /> To explore question echinus expression           retina LL54-GAL4; UAS-GFP flies stained anti-Dlg to <br /> required pupal eye development took advan-              outline cell boundaries (F, F&quot;) GFP (F&apos;,F&quot;) visualize the <br /> tage GAL4-driver, LL54-GAL4, expressed pre-          LL54-GAL4 expression pattern. (G-G&quot;) 29 hr pupal retinas from <br />                                                                   LL54-GAL4; UAS-GFP flies stained above. LL54-GAL4 is <br /> dominantly, exclusively primary, secondary          expressed primarily, exclusively pigment cells, not <br /> tertiary pigment cells, cone cells bristles (Fig.      bristles cone cells. (H) 36 hr pupal eye LL54-GAL4; UAS- <br /> 3F&#226;&#8364;&#8220;G) [55]. Expression LL54-GAL4 wildtype back-           CG2904-RNAi stained anti-Dlg. Extra IOCs sorting <br /> ground, conjunction UAS-driven miRNA (UAS-              defects apparent. Arrows indicate bristles. <br /> CG2904-RNAi) targeting echinus splice forms, pheno- <br /> copied echinus phenotype (Fig. 3H). We cannot <br />  <br />  <br />                                                                                                                            Page 6 15 <br />                                                                                                     (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                          http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> exclude possibility low-level expression        Database searches identified genomic sequences if <br /> echinus-targeting miRNA cell types eye       spliced, generate similar forms echinus multi- <br /> sufficient generate phenotype. This possibility not-    ple Drosophila species insects. Out- <br /> withstanding, observations suggest echinus nor-        arthropods Echinus shows homology <br /> mally functions, part, pigment cells     mammalian DUBs USP53 USP54, with <br /> to regulate IOC fate. Interestingly, however, expression     essentially homology occuring USP <br /> either different splice forms echinus (ec-SF1     domain. Most importantly purposes report, <br /> ec-SF2; below), ecEP&#226;&#710;&#8224;4 background,         Ec-SF2 Ec-SF3 encode proteins truncated USP <br /> same promoter, failed rescue ecEP&#226;&#710;&#8224;4 phenotype (data      domains lack residues important catalysis. Specif- <br /> not shown). This observation suggests, does             ically, Ec-SF2 Ec-SF3 contain catalytic histidine <br /> prove, echinus expression required cell   aspartic acid residues USP His box, but <br /> types bring proper IOC sorting death. Anal-        lack Cys box catalytic cysteine. Ec-SF2 and <br /> ysis echinus clones required determine defin-     Ec-SF3 instead alternative N-termini con- <br /> itively cell types echinus expression           served Drosophilia sequence similarity to <br /> required. Finally, GMR-driven transgenes expressed       Cys box motif (Figure 1B, C). The homologous mam- <br /> all cell types developing eye [50,56,57]. As noted       malian DUB USP54 encodes 4 alternative splice <br /> above (Fig. 2E,K), forced expression echinus reti-    forms, contain complete USP <br /> nal cells did itself induce defects sorting        domains [59]. <br /> ectopic retinal (IOC) cell death. This, conjunction with <br /> the observation endogenous echinus expressed uni-       Echinus lacks USP activity model substrate, USP <br /> formly pupal retina, consistent model       activity required echinus-dependent death of <br /> which echinus expression sufficient induce        IOCs <br /> death IOCs, necessary.                          We generated flies expressing microRNA targets Ec- <br />                                                                 SF1 specifically, control GMR promoter. <br /> Echinus gives rise multiple splice forms encode         These flies showed echinus-like adult rough eye pheno- <br /> proteins homology ubiquitin-specific proteases          type, pupal eyes contained extra IOCs (Fig. 4A,E). In <br /> We sequenced multiple echinus cDNAs identified              contrast, GMR-driven expression microRNAs designed <br /> three splice variants (designated ec-SF1, ec-SF2, ec-       target Ec-SF2 Ec-SF3 resulted flies appeared <br /> SF3) (Fig. 1A&#226;&#8364;&#8220;C). In these, common 3&apos; coding          wildtype, extra IOCs observed (data not <br /> and UTR sequence spliced distinct 5&apos; UTR coding       shown). We exclude possibility micro- <br /> sequences. To determine splice forms expressed        RNAs targeting Ec-SF2 Ec-SF3 failed phenocopy <br /> during pupal retinal cell death, conducted RT-PCR            echinus failed induce large enough <br /> using exon-specific primers. We isoforms         decrease splice form expression (though exper- <br /> were expressed pupal retina stages       iments expression miRNAs sufficient sup- <br /> IOC death occurred (see Additional File 3). No differences      press rescue echinus GMR-dependent expression of <br /> were seen expression different splice forms         ec-SF2 ec-SF3; data shown). Nonetheless, our <br /> monitored using tissue situ hybridizations (data         results targeting Ec-SF1 demonstrate splice <br /> shown).                                                         form, least, important bringing IOC death, <br />                                                                 suggest important regulat- <br /> Each splice form encodes large protein roughly 1700        ing IOC survival. Also consistent hypothesis is <br /> aa. Blast searches identified region homology       observation GMR-driven RNAi targets all <br /> with proteins, N-terminal USP domain,                splice forms (GMR-CG2904-RNAi) results phenotype <br /> domain seven families deubiquiti-        similar observed flies ec-SF1 was <br /> nating enzymes (DUBs). USP-containing DUBs                  targeted (Fig 2C,I). USP activity se unlikely suf- <br /> cysteine proteases capable removing ubiquitin       ficient rescue, echinus sorting cell death <br /> or ubiquitin-like proteins substrates [58,59]. The         phenotypes rescued GMR-dependent <br /> USP domain features short, well-conserved motifs &#226;&#8364;&#8220;          expression yeast UBP2, ubiquitin-specific protease <br /> the Cys box, contains essential catalytic             known active multiple substrates [62,63] (data <br /> cysteine, His box, contains conserved His       shown). <br /> Asp residues thought essential catalysis. <br /> Structural studies USPs HAUSP Ubp14             To test hypothesis Ec-SF1 deubiquitinating <br /> revealed catalytic histidine aspartic acid         activity, measured ability cleave model ubiqui- <br /> deprotonate catalytic cysteine allowing nucle-          tin-linked substrate, Ub-Arg-B-Gal, fusion protein of <br /> ophilic attack [60,61]. Ec-SF1 encodes USP domain            Ubiquitin (Ub) Eschericia coli &#206;&#178;-galactosidase (&#206;&#178;-Gal), <br /> with known essential catalytic residues (Figure 1A).    separated arginine (Arg) residue [62,63]. Ub-Arg-&#206;&#178;- <br />  <br />  <br />                                                                                                                      Page 7 15 <br />                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                         http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br />  <br /> A                 B                 C                   D                      ing enzymes active assay. However, contrast <br />                                                                                yeast Ubp2, expression ec-SF1 ec-SF2 Ub-Arg-&#206;&#178;- <br />                                                                                gal cells, presence X-Gal, resulted forma- <br />                                                                                tion blue colonies (Table 2). The human proteins most <br />                                                                                homologous Ec-SF1, USP53 USP54, lack activ- <br />                                                                                ity assay [64]. These results consistent with <br />                                                                                models Echinus, particular Ec-SF1, lacks deu- <br /> E                 F                 G                   H                      biquitinating activity. However, discussed (the <br />                                                                                Discussion), failure detect cleavage model sub- <br />                                                                                strate does rule possibility Ec-SF1 has <br />                                                                                activity (unknown) substrates. <br />  <br />                       ec EP&#226;&#710;&#8224;4 <br />                                         EP&#226;&#710;&#8224;4                 EP&#226;&#710;&#8224;4 <br /> GMR-ec-SF1-RNAi                    ec      ; GMR-ec-SF2 ec      ; GMR-ec-SF3   To test hypothesis Echinus USP activity is <br />                                                                                required ability bring death excess <br /> Echinus <br /> mote <br /> Figure   4 <br />       normal <br />          does not <br />               IOCrequire <br />                    death deubiquitinating activity pro- <br /> Echinus does require deubiquitinating activity pro-                     IOCs, asked expression Echinus splice forms that <br /> mote normal IOC death. (A-D) SEMs adult eyes various                     lack critical USP catalytic residues, Ec-SF2 Ec-SF3, <br /> genotypes. (E-H) Pupal retinas various genotypes stained                    rescue echinus phenotype. Somewhat sur- <br /> with anti-Dlg. (A,E) GMR-driven expression microRNA                       prise, introduced ecEP&#226;&#710;&#8224;4 background (Fig. <br /> targeting ec-SF1 results echinus phenotype. (B,F) ecEP&#226;&#710;&#8224;4                 4B,F), expression Ec-SF2 (Fig. 4C,G) Ec-SF3 (Fig. <br /> eyes. (C,G) Eyes genotype ecEP&#226;&#710;&#8224;4; GMR-ec-SF2/+. (D,H)                       4D,H), resulted complete restoration normal IOC <br /> Eyes genotype ecEP&#226;&#710;&#8224;4; GMR-ec-SF3/+. Expression ver-                      death. These experiments involve gene overexpression <br /> sions Echinus lack essential USP catalytic residues res-               address question Ec-SF2 Ec- <br /> cues ecEP&#226;&#710;&#8224;4 phenotype.                                                     SF3 normally required IOC death. However, do <br />                                                                                demonstrate Echinus USP activity unlikely be <br />                                                                                absolutely required IOC death, splice <br /> gal stable protein, bacterial cells expressing                forms lack residues necessary activity. <br /> form blue colonies presence substrate X-Gal <br /> (5-bromo-4-chloro-3-indolyl-&#206;&#178;-D-galactopyranoside)                             Echinus does significant genetic interactions with <br /> (Table 2). Deubiquitinating enzymes remove Ub cre-                        components core apoptosis machinery, other <br /> ate Arg-&#206;&#178;-gal, unstable protein. Thus, cells expressing                     pathways implicated regulation retinal cell death <br /> Ub-Arg-&#206;&#178;-gal, active deubiquitinating                            Our observations presented Fig. 2 indicate null <br /> enzyme yeast Ubp2, rise white colonies                      alleles echinus previously unappreciated defect in <br /> the presence X-Gal (Table 2) [62]. Many deubiquitinat-                      cell sorting cell death. The cell death defects <br />                                                                                observed echinus mutants simply indirect <br /> Table 2: Bacterial Assay Deubiquitinating Activity                  result failure cell sorting. Alternatively, echinus may <br /> Echinus                                                                        play roles sorting cell death. We chosen <br />                                                                                explore echinus promoting death of <br />  Plasmid                                           Colony Color <br />                                                                                specific IOCs. We searched genetic interactions <br />  pUb-Arg-&#206;&#178;-Gal                                         Blue <br />                                                                                echinus loss- gain-of-function (overexpres- <br />  pRB105 (yUbp2)                                        White                   sion) signaling pathways known regulate IOC <br />  pRB-ec (SF1)                                          White                   death. GMR-ec phenotypes Fig. 5 refer ec-SF1. Simi- <br />  pRB-ec (SF2)                                          White                   lar phenotypes observed GMR-ec-SF2 (not <br />  pUb-Arg-&#206;&#178;-Gal                                         White                   shown). The RHG family protein Hid required nor- <br />  pRB105 (yUbp2) <br />                                                                                mal IOC death, caspases Dronc Drice. Loss <br />  pUb-Arg-&#206;&#178;-Gal                                          Blue <br />  pRB-ec (SF1)                                                                  overexpression echinus significant effect on <br />  pUb-Arg-&#206;&#178;-Gal                                          Blue                   dominant eye phenotypes associated GMR-driven <br />  pRB-ec (SF2)                                                                  overexpression of molecules, other <br />                                                                                cell death activators including Rpr, Grim, Debcl (Fig. 5) or <br />  Table 2. Echinus ec-SF1 lacks deubiquitinating activity model            caspases Dcp-1 Strica (see Additional File 4). echi- <br />  substrate bacteria. Ubiquitin-Arg-&#206;&#178;-Gal long half-life, and <br />                                                                                nus loss- gain-of-function effect a <br />  colonies expressing protein blue the <br />  presence X-gal substrate. Expression S. cerevisiae Ubp2 Arg-       small eye phenotype associated partial loss-of-func- <br />  &#206;&#178;-Gal results cleavage ubiquitin, exposing N-terminus Arg-       tion DIAP1 resulting GMR-dependent expression <br />  &#206;&#178;-Gal, short half-life (white colonies). In contrast,             dsRNA corresponding sequences diap1 <br />  expression ec-SF1 ec-SF2 does result significant cleave <br />  Ub-Arg-&#206;&#178;-Gal (blue colony color). <br />                                                                                coding region (GMR-diap1-RNAi flies) [12] (Fig. 5). <br />  <br />  <br />  <br />                                                                                                                                     Page 8 15 <br />                                                                                                               (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                            http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br />             echinus   wild type   GMR-ec                           Discussion <br />                                                                    We showed echinus, gene required normal IOC <br /> GMR-gal4, <br /> UAS-Debcl                                                          death, corresponds CG2904. CG2904 generates multi- <br />                                                                    ple transcripts, encodes protein with <br />                                                                    homology USP family ubiquitin-specific pro- <br />                                                                    teases. Echinus necessary sufficient cause IOC <br />  GMR-Rpr <br />                                                                    death overexpressed. These results consistent <br />                                                                    models echinus provides activity can <br />                                                                    modulate signals drive death specific cells <br />  GMR-Hid                                                           (or prevent survival), providing death <br />                                                                    signals specific cells. Previous analyses ec1 allele <br />                                                                    led model echinus functions subsequent to <br /> GMR-Grim                                                           IOC sorting [49,50]. However, analysis multiple <br />                                                                    newly generated echinus alleles, including multiple likely <br />                                                                    null alleles, showed presence excess IOC cells <br />                                                                    arranged side-by-side configuration respect to <br /> GMR-diap1 <br />  (RNAi)                                                            primary pigment cells, addition excess cells arranged <br />                                                                    end-to-end. These observations consistent a <br />                                                                    model echinus functions promote proper IOC <br /> GMR-Dronc                                                          sorting, failure IOC death result of <br />                                                                    incorrectly positioned IOCs unable send or <br />                                                                    receive cell death signals. An alternative possibility we <br />                                                                    explored echinus plays roles cell death signaling <br /> GMR-Drice <br />                                                                    cell sorting. We failed observe significant <br />                                                                    genetic interactions echinus known sus- <br /> Figure 5interactions <br /> Genetic <br /> regulators cell death <br />                        between <br />                           echinus <br />                                  eye     known potential    pected death regulators. While observations not <br /> Genetic interactions echinus known potential        rule possibility echinus acts regulate death <br /> regulators cell death eye. To right schematic   novel point, tend support models which <br /> depicting known suggested interactions death            echinus functions primarily regulate cell sorting. <br /> regulators fly. The question mark separating Debcl/ <br /> Buffy Ark indicates uncertainy roles      Homology searches genome sequence insects <br /> proteins play regulating Ark activation activity. GMR-       suggest echinus conserved. But known <br /> driven transgenes indicated genotype introduced        functions of genes. One splice form <br /> into ecEP&#226;&#710;&#8224;4 background, wildtype background       echinus, ec-SF1, encodes protein contains catalytic <br /> the presence GMR-ec-SF1. For death regulator <br />                                                                    residues essential USP activity. ec-SF1 expressed in <br /> tested, similar phenotypes observed presence of <br /> GMR-ec-SF2 (data shown).                                       pupal eye splice form-specific RNAi tran- <br />                                                                    script phenocopied echinus. However, Ec-SF1 inactive <br />                                                                    deubiquitination assay utilizing model ubiquitin-&#206;&#178;- <br />                                                                    gal fusion protein substrate. This reflect fact that <br />                                                                    ec-SF1 inactive USP. Alternatively, Echinus may <br /> We failed interactions loss gain-          active specific substrates, phenomenon <br /> of-function echinus mutations              observed number USPs [58,59,65]. It pos- <br /> pathways implicated IOC death. These include fol-           sible Ec-SF1, USP family mem- <br /> lowing: EGF pathway (the dominant EGFR allele                  bers, cleaves proteins modified ubiquitin- <br /> EGFREllipse, GMR-driven versions Ras, Sina              related proteins ISG15 [66] Nedd8 [67]. <br /> Yan); Runx transcription factor lozenge (lz50e             Regardless answer question, observation <br /> GMR-lozenge); Notch (GMR-GAL4-UAS-Delta, Nfa-g);               splice forms Echinus lack residues essential for <br /> JNK (GMR-GAL4, UAS-dTAK) (see Additional File 4).                  USP activity rescue echinus phenotype strongly sug- <br /> GMR-GAL4-UAS-klumpfuss lethal combination                   gests ubiquitin ubiquitin-like protease activity is <br /> with GMR-ec. However, significance interaction         essential Echinus function, bringing <br /> is unclear expression GMR-GAL4-UAS-klumpfuss              sorting death excess IOCs. This does not <br /> alone resulted rare adults (see Additional File 4).        mean Echinus functions necessarily unrelated to <br /> Finally, interactions observed loss-            regulation deubiquitination. Interestingly, like echinus, <br /> gain-of-function mutations echinus rst (see Addi-           genes USP53 USP54 human mouse a <br /> tional File 5).                                                    shorter splice form missing key catalytic residues a <br />  <br />  <br />                                                                                                                        Page 9 15 <br />                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                          http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> longer form includes residues (the longer forms      overexpressed, echinus requires cofactors in <br /> are shown Fig. 1). Also like Echinus, USP53          order act targets, relatively <br /> USP54 long forms inactive deubiquitination assay       rare rate-limiting. If true, overexpression echinus <br /> using model ubiquitin-&#206;&#178;-gal fusion protein substrate          expected little effect pheno- <br /> [63]. This conserved similarity gene structure suggests    types associated high-level expression target <br /> functional requirement multiple splice forms         proteins. Therefore, observations allow conclude <br /> echinus, despite observed absence protease activity.     echinus probably rate-limiting, or <br /> Inactive versions known USPs can, cases,       dose-dependent regulator death activators. <br /> bind ubiquitinated substrates, functioning dominant <br /> negatives block deubiquitination, facilitating     Conclusion <br /> degradation events dependent ubiquitin con-         The echinus locus encodes multiple splice versions pro- <br /> jugation [68-71]. A number components ubiqui-         teins homology USP family proteases. But is <br /> tin pathway regulate cell death fly eye, including   clear evidence regulation ubiquitination rele- <br /> ubiquitin activating enzyme uba1, E3 ubiquitin ligase       vant echinus&apos;s role promoting IOC sorting death. <br /> DIAP1, components SCF-type E3 ubiquitin               Echinus did significant genetic interactions with <br /> ligase (skpA novel F-box gene, morgue), deu-      number known death regulators, expression of <br /> biquitinating enzyme fat facets [29,31,32]. Perhaps Echi-       echinus sufficient induce ectopic <br /> nus promotes cell sorting death binding substrates       death. Together observations suggest possi- <br /> modified ubiquitin ubiquitin-like proteins,             bilities. The echinus regulates &#226;&#8364;&#8220; very <br /> thereby blocking removal modifications.            specific contexts/cell types &#226;&#8364;&#8220; unknown (or untested) <br /> Alternatively, Echinus titrate cellular inhibitors     upstream regulators effectors core cell death <br /> ubiquitin ubiquitin-like proteases. Finally, impor-    pathway. Alternatively, echinus act necessary but <br /> tant emphasize Echinus&apos;s functions sorting       sufficient component parallel death signaling or <br /> cell death unrelated ubiquitin ubiquitin-like      effector pathway. Finally, echinus function primarily <br /> proteins. Some USPs affect signaling pathways           regulate cell sorting, failure process leading <br /> which independent ability remove ubiqui-        <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>cell survival</span> IOCs unable effectively <br /> tin ubiquitin-like proteins [72]. Central addressing      transmit receive death signals. Drawing links between <br /> these questions identification proteins bound      echinus pathways requires identifica- <br /> Echinus eye.                                             tion Echinus targets. <br />  <br /> As step direction searched genetic       Methods <br /> interactions echinus mutations genes             Identification sequencing echinus alleles <br /> known suspected regulate IOC survival. EGFR            Adult males exposed EMS mated females <br /> Notch, important upstream regulators IOC survival.           carrying attached-X chromosome (XX/Y). Progeny <br /> Particularly case Echinus overexpression          males rough eyes crossed attached- <br /> experiments, observations suggest echinus       X females stocks, designated ec56 ec30, estab- <br /> sufficient regulate signaling       lished. Complementation tests used establish <br /> pathways. We searched interactions echi-       allelism echinus. Genomic DNA EMS <br /> nus number known suspected cell death effec-        allele isolated instar larvae using standard <br /> tors. These included Rpr, Hid, Grim, caspases           DNA isolation protocols. The coding portions flank- <br /> Dronc, Drice, Dcp-1. For genes,       ing DNA echinus amplified PCR using Platinum <br /> the eye-specific partial loss-of-function DIAP1 induced      Taq polymerase (Invitrogen) sequenced using Big <br /> using GMR-diap1-RNAi, expression eye results          Dye terminator cycle sequencing kit (v3.1) 3730XL <br /> ectopic cell death. However, phenotypes           sequencer (ABI). The obtained sequence compared <br /> were significantly suppressed echinus loss-of-func-      published Drosophila genomic sequence. Both <br /> tion background, enhanced echinus coexpression.           strands completely sequenced exon any <br /> Perhaps echinus does regulate components,          ambiguities mutations re-sequenced. The P ele- <br /> components pathways, downstream points.          ment ecPlacZ isolated originally rough eye mutant <br /> However, caveats conclusion. First,   mapping X chromosome failed complement <br /> echinus rare level mRNA, presumably       ec1 (BA Hay, unpublished). Excision alleles ele- <br /> at level protein well. Therefore, simply    ment, nearby element, EP(X)1343, <br /> that loss echinus, normally regulates target(s)    generated using standard techniques. Deletion alle- <br /> in context lower endogenous expression        les echinus generated excision of <br /> level, little effect phenotypes high-level tar-   EP(X)1343. Approximately 300 independent excision <br /> get expression. Second, cases Echinus          lines characterized using primers indicated Figure <br />  <br />  <br />                                                                                                                     Page 10 15 <br />                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                          http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> 1. Several deletions identified removed        idues 562&#226;&#8364;&#8220;583; cta agg gac tac tca atg gac c), ec-SF2 (resi- <br /> the echinus coding region. Two these, ecEP&#226;&#710;&#8224;4 ec&#226;&#710;&#8224;9,    dues 530&#226;&#8364;&#8220;551; cta aga agt tct cga gca aaa c), sequences <br /> indicated Fig. 1A. The ec3c3 allele generated EMS      common ec transcripts (SF1 residues 4252&#226;&#8364;&#8220;4273; <br /> mutagenesis previously described [53]. The ec&#226;&#710;&#8224;9 dele-         gca atg caa aaa tgg atg tag a). These constructs known <br /> tion allele resulted imprecise excision P element      GMR-ec-SF1-RNAi, GMR-SF2-RNAi, GMR- <br /> insertion adjacent gene roX1 (H. Kramer,              CG2904-RNAi, respectively. <br /> unpublished). <br />                                                                  Echinus RH68894 transgenes expression <br /> Isolation echinus cDNAs                                       The coding regions splice versions echinus, ec- <br /> echinus cDNAs isolated larval-pupal cDNA             SF2 ec-SF3, lack UCH domain catalytic residues, <br /> library using probe generated UCH domain           introduced GMR, generating GMR-ec-SF2 and <br /> of CG2904. Several clones isolated sequenced.           GMR-ec-SF3, respectively. The coding region ec-SF1, <br /> These encoded different splice forms echinus, ec-         contains known UCH catalytic domain residues, <br /> SF2 (Genbank AY576488) ec-SF3 (Genbank                       introduced GMR, generating GMR-ec-SF1. <br /> DQ418878). To identify echinus 5&apos; cDNA end sequences <br /> we carried 5&apos; RACE. Total RNA isolated w1118        Drosophila lines genetics <br /> pupal eye discs using RNeasy Micro Kit (Qiagen). The 5&apos;          Drosophila strains crosses performed 25&#194;&#176;C. <br /> RACE System used according manufacturer&apos;s             Pupal timing expressed hours, white prepu- <br /> protocol (Invitrogen). cDNA reverse tran-            pae stage defined 0 hours pupal formation (APF). <br /> scribing pupal RNA using echinus-specific primer          Pupal dissections performed 36 hrs APF unless <br /> (5&apos;-GGTCTGCAGCTGCTGGAAAAGTTCC-3&apos;). 5&apos; echinus                    noted otherwise. The following strains used: UAS- <br /> transcripts isolated performing PCR using       klumpfuss [74], Ellipse, ec1, Notchspl-1(Bloomington Stock <br /> anchor primer complementary dCTP cap               Center, Indiana University), GMR-&#226;&#710;&#8224;N dcp-1 [75], GMR- <br /> cDNA gene-specific primers (5&apos;-AGATACAGTCTT-                 drice [76], ecPlacZ (this work), GMR-hid, GMR-rpr [77], <br /> GGCCACCGCATA-3&apos;,                5&apos;-TTGTTGTTGGCGCTACT-            GMR-grim, GMR-dronc [6], GMR-strica GMR-Gal4- <br /> GCCATAGC-3&apos;,                                          5&apos;-    UAS-debcl [78] GMR-Gal4-UAS-dTak1 [79]. The echi- <br /> CCGCATTGATGCACCGATCCCTCTC-3&apos;). Nested PCR                        nus deletion allele mutant ecEP&#226;&#710;&#8224;4 generated impre- <br /> followed using internal echinus primers (5&apos;-GAAAC-           cise excision P element insertion line EP(X)1343. <br /> GATCGTCGAAAGGCGTCCAA-3&apos;, 5&apos;-CCAGTGGGGCAT-                        LL54-GAL4-expressing flies obtained Craig <br /> GTGGCAGCGATGT-3&apos;,                                     5&apos;-    Montell [55]. GMR-yUbp2 (GenBank M94916) was <br /> CCATTACGGCCAATTCCACGCTGCT-3&apos;)                        cloned NotI-StuI pGMR-1N. RH68894 (Research <br /> anchor primer. PCR amplicons purified using                 Genetics/Invitrogen) cloned GMR vector. <br /> QiaQuick Gel Extraction Kit (Qiagen) sequenced. <br /> This work led identification echinus splice    Echinus expression pattern <br /> form, ec-SF1 (Genbank DQ418877), encodes dis-              A ~1.2 kilobase region echinus (residues 1,532 to <br /> tinct 5&apos; noncoding coding sequences.                         2,691 respect ec-SF2) amplified PCR using <br />                                                                  T3/T7-tailed primers. Digoxigenin-labeled RNA probes <br /> RNAi-mediated knockdown echinus function                      prepared sense antisense strands <br /> We used approaches silence echinus expression. In         (Roche). In situ hybridization OreR pupal retinas was <br /> the approach cDNA fragment placed           performed essentially described [80]. Pupal retinas <br /> SympUAST vector, carries UAS elements oppo-             ec&#226;&#710;&#8224;9 used negative control staining. <br /> site strands flanking insert [73]. Transgenic flies carry- <br /> ing constructs recombined GMR-Gal4            RT-PCR analysis echinus splice form expression <br /> generate GMR-Gal4-UAS-ec-RNAi flies. Several segments            Retinas OreR pupae (26&#226;&#8364;&#8220;27 hrs APF) dissected <br /> of coding region targeted strategy: resi-     PBS transferred immediately RNAlater <br /> dues 1237&#226;&#8364;&#8220;1505, 232&#226;&#8364;&#8220;398, defined respect          (Ambion). Total RNA extracted using Trizol (Invitro- <br /> the sequence ec-SF2 cDNA. We generated flies         gen). RT-PCR Reactions performed (n = 3) using the <br /> expressing GMR-driven artificial microRNAs, using            SYBR Green One-step RT-PCR reagent kit Applied <br /> mir-6.1 backbone, designed target specific 21 bp              Biosystems 7900 Sequence Detection System. Each 15 &#194;&#181;l <br /> sequences echinus. In brief, 22 bp sequences com-         reaction included 50ng total RNA 0.1 &#194;&#181;M each <br /> plementary echinus substituted mir-6.1          primer. The echinus rp49 primer pairs designed <br /> precursor stem backbone position normally occu-           using Primer Express Version 2.0 software (Applied Bio- <br /> pied mature mir-6 miRNA (C.H. Chen B.A.               systems) constructed span intron. Gel elec- <br /> Hay, unpublished). The 22 nt sequences targeted           trophoresis melting curve analysis performed each <br /> the region surrounding catalytic cysteine ec-SF1(res-     run ensure single major product corre- <br />  <br />  <br />                                                                                                                    Page 11 15 <br />                                                                                               (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                           http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> sponding predicted size melting temperature.       Authors&apos; contributions <br /> Primer sequences: SF1 splice form: 5&apos;-TGCTTTCTCAATT-          JMC carried experiments study. IB <br /> GTGCCGT-3&apos;, SF2 splice form: 5&apos;-CAACATTGGCGCAT-               JDF isolated sequenced echinus alleles, car- <br /> TCTTTC-3&apos;, common 3&apos; primer SF1 SF2: 5&apos;-              ried tissue-in-situ hybridization RT-PCR analysis <br /> AAGATACAGTCTTGGCCACCG-3&apos;, SF3 splice form: 5&apos;-                echinus expression. MG participated data analysis, <br /> GCCTTGTGCCTGCAAAAGTT-3&apos;, 5&apos;-TCAGAGTCACAA-                     experimental design writing manuscript. SG and <br /> CATGGCAGC-3&apos;, echinus splice forms: 5&apos;-CAGCT-             BAH responsible overall experiment design, anal- <br /> GCCCTTCACCCA-3&apos;,        5&apos;-TATGTCGCCCATGTTGCC-3&apos;,             ysis data and, conjunction JMC, writing the <br /> rp49: 5&apos;-AGTCGGATCGATATGCTAAGCT-3&apos;, 5&apos;-AGA-                   manuscript. All authors read approved final man- <br /> TACTGTCCCTTGAAGCGG-3&apos;.                                        uscript. <br />  <br /> Microscopy, immunocytochemistry, antibodies               Additional material <br /> Scanning electron microscope images produced a <br /> Hitachi machine. Flies dehydrated ethanol <br /> series, incubated hexamethyldisilazane (Sigma) over-        Additional file 1 <br />                                                                echinus mutants decrease IOC apoptosis. TUNEL staining of <br /> night, dried prior use. Pupal retinas dissected    (A) OreR, (B) ecPlacZ (C) ec&#226;&#710;&#8224;9 pupal retinas (29&#226;&#8364;&#8220;30 hr APF). Anti- <br /> in PBS fixed 30 minutes 4% paraformaldehyde.        active caspase-3 immunostaining (D) OreR (E) ec56 (F) <br /> Immunostaining carried PBT (PBS + 0.1% Tri-         ecEP&#226;&#710;&#8224;4 pupal retinas (30 hr APF). Apoptosis reduced, com- <br /> ton-X100) containing 10% fetal calf serum. Antibodies          pletely absent, ec mutant pupal retinas. <br /> were used following concentration: mouse anti-Dlg       Click file <br />                                                                [http://www.biomedcentral.com/content/supplementary/1471- <br /> (1:150) mouse anti-&#206;&#178; galactosidase (1:15) (Develop- <br />                                                                213X-7-82-S1.pdf] <br /> mental Studies Hybridoma Bank, University Iowa, <br /> Iowa City, IA), rabbit cleaved caspase-3 (1:100, Cell Sign-    Additional file 2 <br /> aling Technologies). Secondary antibodies included             Phenotype ec3c3 allele trans deficiency region contain- <br /> mouse Alexa Fluor 488 (Molecular Probes) mouse          ing echinus. Scanning electron micrographs pupal retinas several <br /> rabbit IgG conjugated Cy3 (Jackson Labs). Pupal reti-       different genotypes shown. ec3c3 placed trans deficiency that <br /> nas mounted VectaShield medium (Vector,         removes echinus, Df(1)HC244, shows severe rough eye pheno- <br /> Burlingome, CA) Antifade (Molecular Probes).                type homozygous ec3c3 flies. Pupal retinas significant increase <br />                                                                number improper sorting IOCs. This genetic observation sug- <br />                                                                gests ec3c3 represents partial loss-of-function allele. <br /> TUNEL staining                                                 Click file <br /> Pupal retinas dissected PBS fixed 30 min-      [http://www.biomedcentral.com/content/supplementary/1471- <br /> utes 3% paraformaldehyde. The In situ Cell Death            213X-7-82-S2.pdf] <br /> Detection Kit (Roche Applied Science) Promega Dead- <br /> End Fluorometric TUNEL used TUNEL <br /> labeling fluorescein-dUTP. Tissues incubated at <br /> 37&#194;&#176;C 1 hour mixture enzyme <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> solu- <br /> tion rinsed PBS. Labeled tissues mounted in <br /> Antifade reagent (Molecular Probes), viewed Zeiss <br /> Axioplan 2 images captured using Retiga <br /> 1350EX digital camera (Qimaging Corp.) Northern <br /> Eclipse software (Empix Imagin, Inc.). <br />  <br /> Interommatidial cell counts <br /> Interommatidial cell counts counting the <br /> IOCs, minus bristles, surround primary pig- <br /> ment cells. Three separate areas counted pupal <br /> retina, pupal retinas counted for <br /> each genotype. <br />  <br /> Deubiquitination assay <br /> Deubiquitination assays carried described in <br /> [62]. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 12 15 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                               http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br />  <br />                                                                                   References <br />  Additional file 3                                                                1.    Cagan RL, Ready DF: The emergence order Dro- <br />  Three echinus splice forms, ec-SF1, ec-SF2 ec-SF3, expressed                sophila pupal retina. Dev Biol 1989, 136(2):346-362. <br />                                                                                   2.    Wolff T, Ready DF: Cell death normal rough eye <br />  pupal retina stage IOC death. Gel image shows the <br />                                                                                         mutants Drosophila. Develop 1991, 113:825-839. <br />  results RT-PCR analysis using primers positive control (rp49),          3.    Hay BA, Guo M: Caspase-dependent cell death Drosophila. <br />  primers specific echinus splice form (SF1, SF2, SF3), primers              Annu Rev Cell Dev Biol 2006, 22:623-650. <br />  recognize splice forms (all). M= Size Marker; &#226;&#8364;&#8220; + des-        4.    Shi Y: Caspase activation, inhibition, reactivation. Protein <br />  ignate absence presence pupal retinal RNA template; expected                 Science 2004, 13:1979-1987. <br />  product sizes follows: 91 bp (rp49), 104 bp (SF1), 111 bp (SF2),          5.    Meier P, Silke J, Leevers SJ, Evan GI: The Drosophila caspase <br />                                                                                         DRONC regulated DIAP1. Embo J 2000, 19(4):598-611. <br />  147 bp (SF3), 135 bp (all). <br />                                                                                   6.    Hawkins CJ, Yoo SJ, Peterson EP, Wang SL, Vernooy SY, Hay BA: <br />  Click file                                                                    The Drosophila caspase DRONC cleaves following gluta- <br />  [http://www.biomedcentral.com/content/supplementary/1471-                              mate aspartate regulated DIAP1, HID, and <br />  213X-7-82-S3.pdf]                                                                      GRIM. J Biol Chem 2000, 275(35):27084-27093. <br />                                                                                   7.    Chew SK, Akdemir F, Chen P, Lu WJ, Mills K, Daish T, Kumar S, Rod- <br />                                                                                         riguez A, Abrams JM: The apical caspase dronc governs pro- <br />  Additional file 4                                                                      grammed unprogrammed cell death Drosophila. <br />  Genetic interactions echinus components signal-                 Developmental Cell 2004, 7:897-907. <br />  ing pathways. The indicated genotypes introduced ec&#226;&#710;&#8224;4 back-        8.    Daish TJ, Mills K, Kumar S: Drosophila caspase Dronc required <br />  ground, wildtype background presence GMR-ec-SF1.                   specific developmental cell death pathways stress- <br />  For genotype tested, similar phenotypes observed presence             induced apoptosis. Developmental Cell 2004, 7:909-915. <br />                                                                                   9.    Xu D, Li Y, Arcaro M, Lackey M, Bergmann A: The CARD-carrying <br />  GMR-ec-SF2 (data shown). Genotypes discussed text                caspase Dronc essential most, all, developmen- <br />  indicated below. The EGFRELP mutation hypermorphic allele                  tal cell death Drosophila. Development 2005, 132:2125-2134. <br />  EGF receptor [81,82]. However, genetically behaves partial loss-of-      10.   Yu SY, Yoo SJ, Yang L, Zapata C, Srinivasan A, Hay BA: A pathway <br />  function allele eye induces expression high levels         signals regulating effector initiator caspases the <br />  EGFR inhibitor Argos [83]. Downstream Ras pathway components                       developing Drosophila eye. Development 2002, 129:3269-3278. <br />                                                                                   11.   Xu D, Wang Y, Willecke R, Chen Z, Ding T, Bergmann A: The effec- <br />  tested include RasN17 (a dominant negative version Ras driven the <br />                                                                                         tor caspases drICE dcp-1 partially overlapping func- <br />  sevenless promoter) Sina Yanact (a version Yan inhibited            tions apoptotic pathway Drosophila. Cell Death Differ <br />  MAPKinase phosphorylation). GMR-GAL4-UAS-Delta expresses                        2006, 13:1697-1706. <br />  Notch ligand Delta cell morphogenetic furrow [84].           12.   Muro I, Berry DL, Huh JR, Chen CH, Huang H, Seoul JH, Yoo SJ, Guo <br />  Nfa-g removes Notch activity specifically pigment cells [85], leading            M, Baehrecke EH, Hay BA: The Drosophila caspase Ice impor- <br />  failure 1&#194;&#176; pigment cells differentiate, decrease IOC death            tant apoptotic cell deaths spermatid indi- <br />                                                                                         vidualization, nonapoptotic process. Development 2006, <br />  [86]. Notch echinus X chromosome. To search                    133:3305-3315. <br />  interactions Nfa-g echinus loss-of-function mutations took        13.   Wang SL, Hawkins CJ, Yoo SJ, Muller HA, Hay BA: The Drosophila <br />  advantage flies carrying autosomal insertion GMR-CG2904-                      caspase inhibitor DIAP1 essential cell survival is <br />  RNAi targets transcript sequences common echinus splice                    negatively regulated HID. Cell 1999, 98(4):453-463. <br />  forms, phenocopies echinus (Fig. 2C,I). GMR-&#226;&#710;&#8224;N-DCP1                     14.   Hawkins CJ, Wang SL, Hay BA: A cloning method identify cas- <br />  expresses GMR control version caspse DCP-1 lacks               pases regulators yeast: identification Dro- <br />                                                                                         sophila IAP1 inhibitor Drosophila caspase DCP- <br />  N-terminal prodomain. GMR-GAL4-UAS-dTAK flies express GMR                        1. Proc Natl Acad Sci U S A 1999, 96(6):2885-2890. <br />  control kinase TAK1, activator JNK signaling. GMR-Strica flies         15.   Wilson R, Goyal L, Ditzel M, Zachariou A, Baker DA, Agapite J, Steller <br />  express long prodomain caspase GMR control.                                  H, Meier P: The DIAP1 RING finger mediates ubiquitination <br />  Click file                                                                    Dronc indispensable regulating apoptosis. Nature <br />  [http://www.biomedcentral.com/content/supplementary/1471-                              Cell Biology 2002, 4(6):445-450. <br />                                                                                   16.   Chai J, Yan N, Huh JR, Wu JW, Li W, Hay BA, Shi Y: Molecular <br />  213X-7-82-S4.pdf] <br />                                                                                         mechanisms Reaper-Grim-Hid-mediated suppression of <br />                                                                                         DIAP1-dependent Dronc ubiquitination. Nature Structural Biol- <br />  Additional file 5                                                                      ogy 2003, 10:892-898. <br />  echinus does interact roughest. Eye-specific Gain-(GMR-                 17.   Zachariou A, Tenev T, Goyal L, Agapite J, Steller H, Meier P: IAP- <br />                                                                                         antagonists exhibit non-redundant modes action through <br />  ecSF1) loss-of-function (GMR-ec-RNAi) echinus mutants were <br />                                                                                         differential DIAP1 binding. EMBO J 2003, 22:6642-6652. <br />  introduced gain (GMR-GAL4-UAS-rst) loss-of-function (rstCT)             18.   Yan N, Wu JW, Chai J, Li W, Shi Y: Molecular mechanisms of <br />  roughest mutant backgrounds. No significant interactions observed                 Drice inhibition DIAP1 removal inhibition by <br />  genes.                                                                   Reaper, Hid Grim. Nature Structural Molecular Biology <br />  Click file                                                                    2004, 11:420-428. <br />  [http://www.biomedcentral.com/content/supplementary/1471-                        19.   Tenev T, Zachariou A, Wilson R, Ditzel M, Meier P: IAPs func- <br />                                                                                         tionally non-equivalent regulate effector caspases <br />  213X-7-82-S5.pdf]                                                                      distinct mechanisms. Nature Cell Biology 2005, 7:70-77. <br />                                                                                   20.   White K, Grether ME, Abrams JM, Young L, Farrell K, Steller H: <br />                                                                                         Genetic control programmed cell death Drosophila. Sci- <br />                                                                                         ence 1994, 264(5159):677-683. <br />                                                                                   21.   Grether ME, Abrams JM, Agapite J, White K, Steller H: The head <br /> Acknowledgements                                                                        involution defective gene Drosophila melanogaster func- <br /> We thank Burke Judd Helmut Kramer sharing unpublished informa-                  tions programmed cell death.                  Genes Dev 1995, <br /> tion, Helmut Kramer Ting Wu echinus stocks, Marco Marra                 9(14):1694-1708. <br />                                                                                   22.   Chen P, Nordstrom W, Gish B, Abrams JM: grim, novel cell <br /> helpful discussions. SG grateful BC Cancer Foundation NSERC               death gene Drosophila. Genes Dev 1996, 10(14):1773-1782. <br /> (Discovery Grant #250125 M. Marra) financial support. Financial sup-       23.   Christich A, Kauppila S, Chen P, Sogame N, Ho SI, Abrams JM: The <br /> port provided NIH grant GM057422 B.A.H. NIH grants                     Damage-Responsive Drosophila Gene sickle Encodes a <br /> NS042580 NS048396 M.G.                                                           Novel IAP Binding Protein Similar Distinct from <br />                                                                                         reaper, grim, hid. Curr Biol 2002, 12(2):137-140. <br />  <br />  <br />  <br />  <br />                                                                                                                                             Page 13 15 <br />                                                                                                                        (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                            http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> 24.   Srinivasula SM, Datta P, Kobayashi M, Wu JW, Fujioka M, Hegde R,               types Drosophila eye control cell-spe- <br />       Zhang Z, Mukattash R, Fernandes-Alnemri T, Shi Y, Jaynes JB, Alnemri           cific transcription factors. Development 1998, 125:3681-3687. <br />       ES: sickle, Novel Drosophila Death Gene reaper/hid/             47.   Rusconi JC, Fink JL, Cagan R: klumpfuss regulates cell death in <br />       grim Region, Encodes IAP-Inhibitory Protein. Curr Biol                      Drosophila retina.            Mechanisms Development 2004, <br />       2002, 12(2):125-130.                                                           121:537-546. <br /> 25.   Wing JP, Karres JS, Ogdahl JL, Zhou L, Schwartz LM, Nambu JR: Dro-       48.   Lindsley DL, Zimm GG: The genome Drosophila mela- <br />       sophila sickle Is Novel grim-reaper Cell Death Activator.                    nogaster. San Diego , Academic Press; 1992. <br />       Curr Biol 2002, 12(2):131-135.                                           49.   Hay BA, Wolff T, Rubin GM: Expression baculovirus P35 pre- <br /> 26.   Tenev T, Zachariou A, Wilson R, Paul A, Meier P: Jafrac2 IAP             vents cell death Drosophila.                   Development 1994, <br />       antagonist promotes cell death liberating Dronc                        120(8):2121-2129. <br />       DIAP1. Embo J 2002, 21(19):5118-5129.                               50.   Reiter C, Schimansky T, Nie Z, Fischbach KF: Reorganization of <br /> 27.   Kurada P, White K: Ras promotes cell survival Drosophila                 membrane contacts prior apoptosis Drosophila ret- <br />       downregulating hid expression. Cell 1998, 95(3):319-329.                       ina: role IrreC-rst protein. Development 1996, <br /> 28.   Yoo SJ, Huh JR, Muro I, Yu H, Wang L, Wang SL, Feldman RMR, Clem               122(6):1931-1940. <br />       RJ, Muller HAJ, Hay BA: Apoptosis inducers Hid, Rpr Grim             51.   Baker Brachmann C, Cagan RL: Patterning fly eye: role of <br />       negatively regulate levels caspase inhibitor DIAP1                   apoptosis. Trends Genetics 2003, 19:91-96. <br />       distinct mechanisms. Nature Cell Biol 2002, 4:416-424.                   52.   Morris JR, Chen J, Filandrinos ST, Dunn RC, Fisk R, Geyer PK, Wu C: <br /> 29.   Hays R, Wickline L, Cagan R: Morgue mediates apoptosis                  An analysis transvection yellow locus Drosophila <br />       Drosophila melanogaster retina promoting degradation                        melanogaster. Genetics 1999, 151:633-651. <br />       DIAP1. Nature Cell Biology 2002, 4(6):425-431.                        53.   Manak JR, Dike S, Sementchenko V, Kapranov P, Biemar F, Long J, <br /> 30.   Holley CL, Olson MR, Colon-Ramos DA, Kornbluth S: Reaper elim-                 Cheng J, Bell I, Ghosh S, Piccolboni A, Gingeras TR: Biological func- <br />       inates IAP proteins stimulated IAP degradation                     tion unannotated transcription early develop- <br />       generalized translational inhibition. Nature Cell Biol 2002,                   ment Drosophila melanogaster. Nature Genetics 2006, <br />       4:439-444.                                                                     38:1151-1158. <br /> 31.   Ryoo HD, Bergmann A, Gonen H, Ciechanover A, Steller H: Regu-            54.   Wang T, Montell C: Rhodopsin formation Drosophila is <br />       lation Drosophila IAP1 degradation apoptosis                         dependent PINTA retinoid-binding protein. J Neurosci <br />       reaper ubcD1. [erratum appears Nat Cell Biol 2002                       2005, 25:5187-5194. <br />       Jul;4(7):546.]. Nature Cell Biology 2002, 4(6):432-438.                  55.   Moses K, Ellis MC, Rubin GM: The glass gene encodes zinc-fin- <br /> 32.   Wing JP, Schreader BA, Yokokura T, Wang Y, Andrews PS, Husei-                  ger protein required Drosophila photoreceptor cells. <br />       novic N, Dong CK, Ogdahl JL, Schwartz LM, White K, Nambu JR:                   Nature 1989, 340:531-536. <br />       Drosophila Morgue F box/ubiquitin conjugase domain                 56.   Ellis MC, O&apos;Neill EM, Rubin GM: Expression Drosophila glass <br />       protein important grim-reaper mediated apoptosis. Nat                      protein evidence negative regulation activity in <br />       Cell Biol 2002, 4(6):451-456.                                                  non-neuronal cells DNA-binding protein. Develop- <br /> 33.   Olson MR, Holley CL, Yoo SJ, Huh JR, Hay BA, Kornbluth S: Reaper               ment 1993, 119(3):855-865. <br />       regulated IAP-mediated ubiquitination. J Biol Chem 2002,           57.   Amerik AY, Hochstrasser M: Mechanism function deubiq- <br />       278:4028-4034.                                                                 uitinating enzymes.           Biochemica et Biophysica Acta 2004, <br /> 34.   Yoo SJ: Grim stimulates DIAP1 poly-ubiquitination binding                   1695:189-207. <br />       UbcD1. Mol Cells 2005, 20:446-451.                                    58.   Nijman SMB, Luna-Vargas MPA, Velds A, Brummelkamp TR, Dirac <br /> 35.   Ramos RGP, Igloi GL, Lichte B, Baumann U, Maier D, Schneider T,                AMG, Sixma TK, Bernards R: A genomic functional inven- <br />       Brandstatter JH, Frohlich A, Fischbach KF: The irregular chiasm C-             tory deubiquitinating enzymes. Cell 2005, 123:773-786. <br />       roughest locus Drosophila, affects axonal projec-               59.   Baker RT, Tobias JW, Varshavsky A: Ubiquitin-specific proteases <br />       tions programmed cell death, encodes novel immu-                         Saccharomyces cerevisiae: Cloning UBP2 UBP3, <br />       noglobulin-like protein. Genes Dev 1993, 7:2533-2547.                          functional analysis UBP family. J Biol Chem 1992, <br /> 36.   Gorski SM, Baker Brachmann C, Tanenbaum SB, Cagan RL: Delta                    267:23364-23375. <br />       Notch promote correct localization IrreC-rst. Cell                60.   Huang Y, Baker RT, Fischer-Vise JA: Control cell fate deu- <br />       Death Differentiation 2000, 7:1011-1013.                                   biquitinating enzyme encoded fat facets gene. Science <br /> 37.   Bao S, Cagan R: Preferential adhesion mediated Hibris                   1995, 270:1828-1831. <br />       Roughest regulates morphogenesis patterning                   61.   Quesada V, Diaz-Perales A, Gutierrez-Fernandez A, Garabaya C, Cal <br />       Drosophila eye. Developmental Cell 2005, 8:925-935.                            S, Lopez-Otin C: Cloning enzymatic analysis 22 novel <br /> 38.   Grzeschik NA, Knust E: IrreC/rst-mediated cell sorting                  human ubiquitin-specific proteases. BBRC 2004, 314:54-62. <br />       Drosophila pupal eye development depends proper local-                62.   Amerik AY, Li SJ, Hochstrasser M: Analysis deubiquitinat- <br />       ization DE-cadherin. Development 2005, 132:2035-2045.                       ing enzymes yeast Saccharomyces cerevisiae. Biol <br /> 39.   Freeman M: Reiterative use EGF receptor triggers dif-                   Chem 2000, 381:981-992. <br />       ferentiation cell types Drosophila eye. Cell 1996,         63.   Malakhov MP, Malakhova OA, Kim KI, Ritchie KJ, Zhang DE: UBP43 <br />       87:651-660.                                                                    (USP18) specifically removes ISG15 conjugated pro- <br /> 40.   Miller DT, Cagan RL: Local induction patterning pro-                    teins. J Biol Chem 2002, 277:9976-9981. <br />       grammed cell death developing Drosophila retina.                  64.   Gong L, Kamitani T, Millas S, Yeh ET: Identification novel iso- <br />       Development 1998, 125:2327-2335.                                               peptidase dual specificity ubiquitin- NEDD8- <br /> 41.   Bergmann A, Agapite J, McCall K, Steller H: The Drosophila gene                conjugated proteins. J Biol Chem 2000, 275:14212-14216. <br />       hid direct molecular target Ras-dependent survival sig-          65.   Naviglio S, Mattecucci C, Matoskova B, Nagase T, Nomura N, Di <br />       naling. Cell 1998, 95(3):331-341.                                              Fiore PP, Draetta GF: UBPY: A growth-regulated human ubiq- <br /> 42.   Dikic I: Distinct monoubiquitin signals receptor endocyto-                  uitin isopeptidase. EMBO J 1998, 17:3241-3250. <br />       sis. Trends Biochemical Sciences 2003, 28:598-604.                    66.   DeSalle LM, Pagano M: Regulation G1 S transition by <br /> 43.   Le Borgne R, Bardin A, Schweisguth F: The roles receptor                ubiquitin pathway. FEBS Lett 2001, 490:179-189. <br />       ligand endocytosis regulating Notch signaling. Development            67.   Lopez-Otin C, Overall CM: Protease degradomics: new chal- <br />       2005, 132:1751-1762.                                                           lenge proteomics. Nat Rev Mol Cell Biol 2002, 3:509-519. <br /> 44.   Wildonger J, Sosinsky A, Honig B, Mann RS: Lozenge directly acti-        68.   Li M, Chen D, Shiloh A, Luo J, Nikolaev AY, Qin J, Gu W: Deubiqui- <br />       vates argos klumpfuss regulate programmed cell                          tination p53 HAUSP important pathway p53 <br />       death. Genes Development 2005, 19:1034-1039.                               stabilization. Nature 2002, 416:648-653. <br /> 45.   Siddall NA, Behan KJ, Crew J R., Cheung TL, Fair JA, Batterham P, Pol-   69.   Malakhova OA, Kim KI, Luo JK, Zou W, Kumar KGS, Fuchs SY, Shuai <br />       lock JA: Mutations lozenge D-Pax2 invoke ectopic pat-                   K, Zhang DE: UBP43 novel regulator interferon signal- <br />       terned cell death developing Drosophila eye using                       ing independent ISG15 isopeptidase activity. EMBO J <br />       distinct mechanisms. Dev Genes Evol 2003, 213:107-119.                         2006, 25:2358-2367. <br /> 46.   Flores GV, Daga A, Kalhor HR, Banerjee U: Lozenge expressed           70.   Giordano E, Rendina R, Peluso I, Furia M: RNAi Triggered Sym- <br />       pluripotent precursor cells patterns multiple cell                      metrically Transcribed Transgenes Drosophila mela- <br />                                                                                      nogaster. Genetics 2002, 160(2):637-648. <br />  <br />  <br />  <br />                                                                                                                                          Page 14 15 <br />                                                                                                                     (page number citation purposes) <br />  BMC Developmental Biology 2007, 7:82                                                        http://www.biomedcentral.com/1471-213X/7/82 <br />  <br />  <br />  <br /> 71.   Klein T, Campos-Ortega JA: Klumpfuss, Drosophila gene <br />       encoding member EGR family transcription fac- <br />       tors, involved bristle leg development. Development <br />       1997, 124:3123-3134. <br /> 72.   Xu P, Vernooy SY, Guo M, Hay BA: The Drosophila microRNA <br />       mir-14 suppresses cell death required normal fat <br />       metabolism. Current Biology 2003, 13:790-795. <br /> 73.   Huh JR, Foe I, Chen CH, Guo M, Park JM, Hay BA: The Drosophila <br />       Inhibitor Apoptosis DIAP2 required innate <br />       immune response Gram-negative bacterial infection, and <br />       negatively regulated IAP-binding proteins the <br />       Reaper/Hid/Grim family. J Biol Chem 2006, 282:2056-2068. <br /> 74.   Hay BA, Wassarman DA, Rubin GM: Drosophila homologs of <br />       baculovirus inhibitor apoptosis proteins function block <br />       cell death. Cell 1995, 83(7):1253-1262. <br /> 75.   Vernooy SY, Chow V, Su J, Verbrugghe K, Yang J, Cole S, Olson MR, <br />       Hay BA: Drosophila Bruce potently suppress Rpr- and <br />       Grim-dependent Hid-dependent cell death. Curr Biol <br />       2002, 12:1164-1168. <br /> 76.   Takatsu Y, Nakamura M, Stapleton M, Danos MC, Matsumoto K, <br />       O&apos;Connor MB, Shibuya H, Ueno N: TAK1 participates c-Jun N- <br />       terminal kinase signaling Drosophila development. <br />       Mol Cell Biol 2000, 20:3015-3026. <br /> 77.   Tautz D, Pfeifle C: A non-radioactive situ hybridization <br />       method localization specific RNAs Drosophila <br />       embryos reveals translational control segmentation <br />       gene hunchback. Chromosoma 1989, 98(2):81-85. <br /> 78.   Baker NE, Rubin GM: Effect eye development dominant <br />       mutations Drosophila homologue EGF receptor. <br />       Nature 1989, 340:150-153. <br /> 79.   Lesokhin AM, Yu SY, Katz J, Baker NE: Several levels EGF <br />       receptor signaling photoreceptor specification in <br />       wild-type, Ellipse, null mutant Drosophila. Dev Biol 1999, <br />       205:129-144. <br /> 80.   Spencer SA, Powell PA, Miller DT, Cagan RL: Regulation EGF <br />       receptor signaling establishes pattern developing <br />       Drosophila retina. Development 1998, 125:4777-4790. <br /> 81.   Shellenbarger DL, Mohler JD: Temperature-sensitive mutations <br />       No <br /> </body></html>