<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol.17 No.15 pp.4226&#226;&#8364;&#8220;4237, 1998 <br />  <br />  <br />  <br /> Separation structural dynamic functions of <br /> the mitochondrial translocase: Tim44 crucial for <br /> the inner membrane import sites translocation of <br /> tightly folded domains, loosely folded <br /> preproteins <br />  <br /> Ulf Bo&#194;&#168;mer1,2, Ammy C.Maarse3,                                         stepwise interaction positively charged presequences <br /> Falk Martin1,4, Andreas Geissler1,4,                                   patches negatively charged amino acids the <br /> Alessio Merlin1, Birgit Scho&#194;&#168;nfisch1,                                  Tom proteins (Dietmeier et al., 1997; Schatz, 1997). <br /> Michiel Meijer3, Nikolaus Pfanner1,5                                   Translocation inner membrane driven two <br /> and Joachim Rassow1                                                    different mechanisms. The step mediated the <br />                                                                        presequences pulled translocation <br /> 1Institut <br />         fu&#194;&#168;r Biochemie und Molekularbiologie 4Fakulta&#194;&#168;t fu&#194;&#168;r       channel mitochondrial membrane potential (Martin <br /> Biologie, Universita&#194;&#168;t Freiburg, Hermann-Herder-Strasse 7,             et al., 1991; Bauer et al., 1996). The presequences <br /> D-79104 Freiburg, Germany 3Institute Molecular Cell Biology,   subsequently cleaved translocation the <br /> BioCentrum Amsterdam, 1098 SM Amsterdam, The Netherlands <br />                                                                        mature parts preproteins membrane potential- <br /> 2Present address: Institut fur Zell- und Molekularbiologie, <br />                             &#194;&#168;                                          independent. The final steps translocation mediated <br /> Schering AG, Berlin, Germany <br />                                                                        mitochondrial Hsp70 (mtHsp70) coupled <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> 5Corresponding author <br />                                                                        hydrolysis ATP (Kang et al., 1990; Scherer et al., <br /> e-mail: pfanner@ruf.uni-freiburg.de                                    1990; Hwang et al., 1991; Rassow Pfanner, 1995a; <br />                                                                        Ungermann et al., 1996). <br /> The essential gene TIM44 encodes subunit                         The inner membrane proteins Tim44, Tim23 Tim17 <br /> inner mitochondrial membrane preprotein translocase                    identified yeast Saccharomyces cerevisiae <br /> that forms complex matrix heat-shock                        components machinery mediates the <br /> protein Hsp70. The specific role Tim44 protein                   translocation presequence-targeted preproteins the <br /> import defined lack                 matrix (Maarse et al., 1992, 1994; Scherer et al., 1992; <br /> means block function. Here report                       Horst et al., 1993; Dekker et al., 1993; Emtage Jensen, <br /> Saccharomyces cerevisiae mutant allele TIM44                   1993; Ryan et al., 1994; uniform nomenclature, <br /> allows selective efficient inactivation Tim44                Pfanner et al., 1996). Tim23 Tim17 integral <br /> organello. Surprisingly, mutant mitochondria                   membrane proteins components 90 kDa complex <br /> still able import preproteins. The import rate              associates preproteins translocation <br /> reduced ~30% compared wild-type long                     inner membrane. It suggested that <br /> the preproteins carry stably folded domains.                    proteins directly participate formation the <br /> Moreover, number import sites reduced.                   inner membrane protein import channel (Dekker et al., <br /> However, mutant mitochondria strongly                          1993, 1997; Ryan Jensen, 1993; Ku&#194;&#168;brich et al., 1994; <br /> impaired pulling folded domains preproteins                      Bauer et al., 1996; Lohret et al., 1997; Ryan et al., 1998). <br /> close outer membrane promoting                        Tim44 encoded essential gene (Maarse et al., <br /> unfolding. Our results demonstrate Tim44                   1992). A mammalian homologue identified recently <br /> an essential structural component import chan-                  mice (Wada Kanwar, 1998). The elucidation its <br /> nel, crucial import folded domains.                      functions hampered lack means to <br /> We suggest concerted action Tim44                      block activities. The role Tim44 mitochondrial <br /> mtHsp70 drives unfolding preproteins acceler-                   protein import is, therefore, ambiguous. It been <br /> ates translocation loosely folded preproteins. While                suggested Tim44 participate directly the <br /> mtHsp70 essential import tightly                    formation protein import channel be <br /> loosly folded preproteins, Tim44 plays special-                 cross-linked efficiently preproteins transit the <br /> ized role translocation tightly folded domains.                  mitochondrial membranes co-purified transloca- <br /> Keywords: Hsp70/mitochondrial protein import/Tim44                     tion intermediates (Scherer et al., 1992; Blom et al., 1993; <br />                                                                        Berthold et al., 1995; Horst et al., 1995, 1996; Kanamori <br />                                                                        et al., 1997). Interactions Tim44 Tim17 Tim23 <br />                                                                        demonstrated biochemical genetic <br /> Introduction                                                           means (Berthold et al., 1995; Blom et al., 1995; Bo&#194;&#168;mer <br /> The passage nuclear-encoded preproteins                    et al., 1997). However, contribution Tim44 the <br /> cytosol mitochondrial matrix mediated Tom                 formation channel difficult conceive because <br /> proteins mitochondrial outer membrane Tim                primary structure Tim44 does putative <br /> proteins mitochondrial inner membrane (Ryan                 membrane-spanning segments, fractionations Tim44 <br /> Jensen, 1995; Schatz, 1996; Neupert, 1997; Pfanner                 behaves like peripheral membrane protein (Maarse et al., <br /> Meijer, 1997). The mechanism drives pre-                     1992; Blom et al., 1993; Berthold et al., 1995). <br /> proteins outer membrane depend                   According accessibility proteases, Tim44 is <br />  <br /> 4226                                                                                                     &#194;&#169; Oxford University Press <br />                                                                                                                   Function Tim44 <br />  <br />  <br /> exposed matrix inner membrane (Blom <br /> et al., 1993; Rassow et al., 1994). In location, Tim44 <br /> serves membrane anchor mtHsp70, the <br /> complex Tim44 mtHsp70 suggested repres- <br /> ent core mitochondrial protein import machinery <br /> (Kronidou et al., 1994; Rassow et al., 1994, 1995; <br /> Schneider et al., 1994; von Ahsen et al., 1995; Horst <br /> et al., 1996; Ungermann et al., 1996). The functional <br /> relevance interaction addressed using mutant <br /> form mtHsp70, Ssc1-2p, unable bind Tim44 <br /> but binds preproteins (Voos et al., 1996). However, <br /> more recent data indicate mtHsp70 binds to <br /> Tim44, directly Tim17&#226;&#8364;&#8220;Tim23 complex. <br /> The mutant form Ssc1-2p defective interaction <br /> with membrane-binding sites (Bo&#194;&#168;mer et al., 1997). <br /> Therefore, decided defects protein <br /> translocation observed mutant allele <br /> ssc1-2 caused lack interaction with <br /> Tim44. Up now, individual functions two <br /> membrane anchors mechanism protein transloca- <br /> tion discerned, contribution Tim44 <br /> to activity mtHsp70 unclear. <br />    In study, examined function Tim44, <br /> using temperature-sensitive mutant Tim44. For    Fig. 1. Temperature-sensitive mutant Tim44. (A) Growth defect a <br /> time, mutant allows investigation effects      yeast strain containing allele tim44-8. Wild-type TIM44 the <br /> selective inactivation functional Tim44. We       mutant allele tim44-8 cloned single-copy vector <br />                                                                 YCplac111 expressed tim44 deletion strain. The strains were <br /> the number translocation sites inner membrane      grown glycerol plates 23 37&#194;&#176;C, respectively. (B) Aggregation <br /> not reduced depletion Tim44, suggesting Tim44        mutant Tim44 heat shock. Isolated mitochondria were <br /> is constitutive component protein import chan-     incubated import buffer (including 2 mM ATP; BSA omitted) <br /> nel. The mutant mitochondria reduced rate import      37&#194;&#176;C different time periods indicated. Subsequently, all <br /> reactions, pronounced case tightly folded        samples shifted 25&#194;&#176;C 5 min. In parallel, mitochondria were <br />                                                                 kept ice incubated 25&#194;&#176;C. Triton X-100 added a <br /> protein domains. We conclude Tim23 Tim17           final concentration 0.1%, samples shaken end-over- <br /> form protein import channel absence Tim44.        end 8&#194;&#176;C 10 min. Aggregates pelleted 1 h at <br /> Tim44 separate functional <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> acts   106 000 gmax analysed SDS&#226;&#8364;&#8220;PAGE immunoblotting. <br /> dynamic element Tim machinery.                           (C) Mitochondria treated (B) tested aggregation of <br />                                                                 mtHsp70, Mge1, ADP/ATP carrier (AAC) cytochrome b2 <br />                                                                 (Cyt. b2). <br /> Results <br /> A temperature-sensitive mutant Tim44                         structure collapses higher temperature renders <br /> We generated mutated forms TIM44 low fidelity             protein prone aggregation. Hence, allele tim44-8 <br /> PCR expressed mutated genes S.cerevisiae          provides opportunity deplete mitochondria func- <br /> a single-copy vector control             tional Tim44. As mutant <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> induced <br /> promoters. The clone containing allele tim44-8          pre-incubation isolated mitochondria without <br /> isolated used study (Figure 1A). At 23&#194;&#176;C,       affecting known components protein import <br /> mutant strain grows similarly wild-type glucose       machinery (and does require temperature shift <br /> and glycerol, 37&#194;&#176;C grows             vivo), indirect effects inactivation Tim44 on <br /> poorly media. We isolated mitochondria            mitochondrial structure function minimized. <br /> both mutant wild-type strain growth at <br /> 23&#194;&#176;C. We Tim44 protein present            The number translocation sites preproteins <br /> the amounts strains. After lysis            carrying presequence reduced upon <br /> mitochondria presence detergent,        depletion functional Tim44 <br /> the mutant Tim44 differed wild-type protein         To determine role Tim44 mitochondrial protein <br /> a dramatically enhanced tendency aggregate (Figure           import, isolated mitochondria tim44-8 mutant <br /> 1B). After pre-incubation isolated mitochondria        corresponding wild-type, imported several <br /> at 37&#194;&#176;C, mutant tim44-8 protein aggregated efficiently      different preproteins. To deplete tim44-8 mitochondria <br /> (Figure 1B, lanes 9&#226;&#8364;&#8220;16) wild-type Tim44 protein       Tim44, mitochondria subjected heat shock <br /> remained soluble (Figure 1B, lanes 1&#226;&#8364;&#8220;8). The solubility         prior import reactions. We synthesized precursors <br /> of mitochondrial proteins tested       &#206;&#177;-subunit matrix-processing peptidase (MPP), <br /> affected tim44-8 <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, including Tim23 (Figure       &#206;&#178;-subunit ATP synthase (F1&#206;&#178;), AAC in <br /> 1B), matrix proteins mtHsp70 Mge1p, ADP/            reticulocyte lysate presence [35S]methionine/ <br /> ATP carrier (AAC) inner membrane                 [35S]cysteine imported proteins isolated <br /> intermembrane space protein cytochrome b2 (Figure 1C).          mitochondria (Figure 2). Complete translocation &#206;&#177;MPP <br /> The mutant Tim44 protein labile tertiary        F1&#206;&#178; mitochondrial membranes assayed <br />  <br />                                                                                                                                4227 <br />  U.Bo&#194;&#168;mer et al. <br />  <br />  <br />  <br />  <br /> Fig. 2. Mitochondria tim44-8 mutant efficient import preproteins. (A) Import &#206;&#177;-subunit matrix-localized <br /> mitochondrial processing peptidase MPP. The preprotein synthesized reticulocyte lysate presence [35S]methionine/[35S]cysteine and <br /> incubated isolated mitochondria tim44-8 mutant strain corresponding wild-type (WT) 25&#194;&#176;C times indicated. Prior to <br /> import, mitochondria strains incubated 37&#194;&#176;C 15 min induce mutant phenotype. The import reactions stopped by <br /> addition valinomycin cooling ice. The samples split halves treated proteinase K remove preproteins outside the <br /> mitochondrial membranes (1 PK) left protease (&#226;&#8364;&#8220; PK). After reisolation mitochondria centrifugation, imported proteins were <br /> separated SDS&#226;&#8364;&#8220;PAGE analysed using Phosphorimager. p, precursor protein; m, mature protein. The graphs relative amounts of <br /> processed proteins. The processed protein obtained 30 min absence proteinase K set 100% (control). (B) Import of <br /> the &#206;&#178;-subunit mitochondrial ATP synthase (F1&#206;&#178;). The experiment performed described (A). (C) Import ADP/ATP carrier <br /> (AAC). Since AAC does contain cleavable presequence, protease-protected AAC shown. For quantification &#226;&#710;&#8224;&#207;&#710;-dependent <br /> imported AAC, protease-protected AAC accumulated absence membrane potential subtracted. <br />  <br />  <br />  <br /> by resistance externally added protease. In assay,                   below, Figure 6), suggesting reduced import <br /> the import efficiencies tim44-8 mutant mitochondria                   efficiencies this. <br /> were reduced ~20&#226;&#8364;&#8220;40% (Figure 2A B, samples 1                             To study import defects mutant mitochondria <br /> PK). However, processing imported proteins reached                    detail, used hybrid proteins containing <br /> 80&#226;&#8364;&#8220;95% wild-type levels, demonstrating                       DHFR (dihydrofolate reductase) domain were <br /> inactivation Tim44 minor effects                          designed specifically characterize mechanisms of <br /> translocation presequences protein                        mitochondrial protein import. First imported hybrid <br /> import channels membranes (Figure 2A B,                           protein Su9(70)&#226;&#8364;&#8220;DHFR (containing presequence of <br /> samples &#226;&#8364;&#8220; PK). Considering mutant Tim44                         subunit 9 ATP synthase fused entire DHFR) <br /> depleted .95%, reduction import efficiencies                   used commonly characterize import of <br /> of &#206;&#177;MPP F1&#206;&#178; pronounced expected. In                        matrix-targeted proteins (Figure 3A). The import efficiency <br /> particular, insertion presequences Tim                   reduced 20&#226;&#8364;&#8220;30%, demonstrating the <br /> channel Tim44-independent. Tim44 appears                         inactivation Tim44 allows considerable activity of <br /> to act primarily late stages protein translocation.                    import machinery. <br />    A slight reduction import kinetics                           The possibility existed radiochemical amounts <br /> observed AAC, import regarded                       preprotein synthesized reticulocyte lysate <br /> as completely independent Tim17/Tim23/Tim44                           imported small number sites containing <br /> machinery (Sirrenberg et al., 1996, 1998; Dekker et al.,                     intact Tim44. We tested proximity the <br /> 1997; Koehler et al., 1998). However, import                      translocating Su9(70)&#226;&#8364;&#8220;DHFR Tim44 chemical cross- <br /> AAC requires membrane potential &#226;&#710;&#8224;&#207;&#710;                            linking (Figure 3B). Using reagent disuccinimidyl <br /> mitochondrial inner membrane. We mem-                         suberate (DSS), preprotein cross-linked efficiently <br /> brane potential slightly weaker tim44-8 mutant                    Tim44 wild-type mitochondria observed previously <br /> mitochondria wild-type mitochondria (see                         (Blom et al., 1993; Ku&#194;&#168;brich et al., 1994). Two cross- <br />  <br /> 4228 <br />                                                                                                                                  Function Tim44 <br />  <br />  <br />  <br />  <br /> Fig. 3. The hybrid protein Su9(70)&#226;&#8364;&#8220;DHFR imported absence functional Tim44. (A) Import Su9(70)&#226;&#8364;&#8220;DHFR. The preprotein <br /> contains N-terminal residues 1&#226;&#8364;&#8220;70 Su9 fused entire DHFR mouse. After synthesis 35S-labelled protein reticulocyte <br /> lysate, preprotein imported described legend Figure 2A. p, preprotein; i, processing intermediate; m, mature protein. (B) Cross- <br /> linking Su9(70)&#226;&#8364;&#8220;DHFR. The preprotein accumulated translocation sites import ATP-depleted mitochondria. In parallel samples, the <br /> membrane potential &#226;&#710;&#8224;&#207;&#710; dissipated addition valinomycin prior import. The mitochondria reisolated cross-linking reagent <br /> DSS added. For details cross-linking protocol, Materials methods. Lanes 1&#226;&#8364;&#8220;6, SDS&#226;&#8364;&#8220;PAGE fluorography total cross-linked <br /> products; lanes 7&#226;&#8364;&#8220;14, SDS&#226;&#8364;&#8220;PAGE fluorography immunoprecipitated proteins; lanes 15&#226;&#8364;&#8220;18, immunoprecipitation proteins cross-linking <br /> by EGS. Tim44* Tim44**, cross-linking products containing unprocessed processed Su9&#226;&#8364;&#8220;DHFR, respectively. <br />  <br />  <br /> linking products formed immunopre-                        mitochondrial matrix space (Koll et al., 1992). The <br /> cipitated antibodies raised Tim44, representing                    preprotein applied concentrations close saturation <br /> cross-linking preprotein processing                         mitochondrial import sites (Dekker et al., 1997). <br /> intermediate (Figure 3B, bands labelled Tim44*                            The import kinetics protein shown Figure <br /> Tim44** lanes 2 9). In contrast, Su9(70)&#226;&#8364;&#8220;DHFR                          4A. The import efficiency mutant mitochondria <br /> imported mutant mitochondria cross-                          reduced similar extent import the <br /> linked (Figure 3B, lane 5 versus lane 2, lane 13 versus                   radiochemical amounts preproteins reticulocyte <br /> lane 9). The result obtained ethylene                           lysate shown Figures 2 3. <br /> glycolbis succinimidyl-succinate (EGS; Figure 3B, lanes                          The preprotein b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR processed two <br /> 15&#226;&#8364;&#8220;18), reagent longer spacer arm DSS                           stages. Immediately translocation inner <br /> (1.61 nm versus 1.14 nm). These results confirm                     membrane, the presequence cleaved off <br /> preproteins imported absence Tim44.                          processing intermediate formed. In subsequent <br />    We asked mitochondria inactiv-                       step, ~8 additional amino acids removed, yielding the <br /> ated Tim44 able import large amounts prepro-                       intermediate i*, indicating maturation step of <br /> teins. To obtain soluble preprotein chemical amounts,                    preprotein (Laloraya et al., 1995). Interestingly, the <br /> we expressed hybrid protein b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR Esch-                        formation i* intermediate reduced drastically in <br /> erichia coli purified functional form (Dekker                       tim44-8 mitochondria formation i <br /> et al., 1997). The construct consists 167                        intermediate affected (Figure 4A, upper panel). The <br /> residues cytochrome b2 complete DHFR (Koll                             effect observed mutants the <br /> et al., 1992). Due deletion (&#226;&#710;&#8224;) segment                           mtHsp70/Mge1 (Voos et al., 1993, 1994; Laloraya <br /> within presequence, preprotein targeted                         et al., 1995), suggesting Tim44 mtHsp70 <br />  <br />                                                                                                                                                4229 <br />  U.Bo&#194;&#168;mer et al. <br />  <br />  <br />                                                                       possibility accumulating b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR translo- <br />                                                                       cation sites import presence DHFR ligand <br />                                                                       methotrexate (MTX). It shown previously that <br />                                                                       number translocation intermediates be <br />                                                                       accumulated method limited number of <br />                                                                       available import sites inner membrane (Dekker <br />                                                                       et al., 1997). We titrated maximal number of <br />                                                                       translocation intermediates parallel tim44-8 <br />                                                                       mutant mitochondria corresponding wild-type <br />                                                                       mitochondria. Surprisingly, similar numbers for <br />                                                                       mitochondria strains (Figure 4B). About 20 <br />                                                                       pmol preprotein accumulated mg of <br />                                                                       mitochondrial protein, agreement previous studies <br />                                                                       wild-type mitochondria (Vestweber Schatz, 1988; <br />                                                                       Rassow et al., 1989; Dekker et al., 1997). We conclude <br />                                                                       Tim44 essential factor formation of <br />                                                                       functional protein translocation channels. The involvement <br />                                                                       Tim44 increases efficiency protein import, but <br />                                                                       Tim44 does determine number of <br />                                                                       import sites. <br />  <br />                                                                       Function Tim44 unfolding import a <br />                                                                       tightly folded protein domain <br />                                                                       To assess features preprotein determine its <br />                                                                       dependence Tim44, investigated import of <br />                                                                       cytochrome b2&#226;&#8364;&#8220;DHFR hybrid proteins compared con- <br />                                                                       structs differ requirement unfolding of <br />                                                                       defined protein domains (Figure 5A). We tested <br />                                                                       different constructs, containing N-terminal parts of <br />                                                                       cytochrome b2, residues 1&#226;&#8364;&#8220;220 1&#226;&#8364;&#8220;85, respectively. The <br />                                                                       presequences constructs identical; however, the <br />                                                                       longer construct [b2(220)-DHFR] contains complete <br />                                                                       haem-binding domain cytochrome b2 (residues 81&#226;&#8364;&#8220;180) <br />                                                                       tightly folded requires unfolding allow <br /> Fig. 4. Import purified preprotein b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR chemical   translocation mitochondrial membranes. The <br /> amounts inactivation Tim44. (A) Import b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR      haem-binding domain missing b2(85)&#226;&#8364;&#8220;DHFR. Thus, <br /> (in absence MTX). The preprotein b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR contains the <br /> N-terminal 167 residues cytochrome b2 fused complete <br />                                                                       preproteins differ requirement energy- <br /> DHFR. b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR expressed E.coli, purified             dependent unfolding distinct protein domain; rate <br /> incubated isolated mitochondria tim44-8 mutant    import limited efficiency unfolding the <br /> corresponding wild-type (WT), pre-treated 37&#194;&#176;C, 25&#194;&#176;C    domain (Gambill et al., 1993; Glick et al., 1993; Voos <br /> times indicated. In parallel samples, valinomycin added        et al., 1993, 1996). The DHFR domain stably <br /> dissipate membrane potential &#226;&#710;&#8224;&#207;&#710;. The mitochondria were <br /> reisolated amounts imported protein analysed       folded and, absence MTX, does cause <br /> SDS&#226;&#8364;&#8220;PAGE, immunoblotting densitometry. The amounts             effective restrictions import reactions (Pfanner et al., <br /> b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR processed wild-type mitochondria 20 min        1990; Voos et al., 1996). We preproteins <br /> were set 100% (control). p, precursor protein; i, processing       imported efficiently mitochondria wild- <br /> intermediate; i*, second processing intermediate. (B) Titration of <br /> protein import sites. Mitochondria pre-treated 37&#194;&#176;C and <br />                                                                       type (Figure 5B, lanes 1&#226;&#8364;&#8220;4). In contrast, mitochondria <br /> incubated presence increasing concentrations b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;   tim44-8 mutant, import b2(220)&#226;&#8364;&#8220;DHFR was <br /> DHFR. MTX included accumulate preprotein              considerably delayed import b2(85)&#226;&#8364;&#8220;DHFR <br /> translocation intermediate spanning mitochondrial membranes.     showed minor reduction (Figure 5B, lanes 6&#226;&#8364;&#8220;9). <br /> The import reactions stopped addition valinomycin      The effect pronounced treatment the <br /> cooling ice. The mitochondria reisolated relative <br /> amounts processed translocation intermediates b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR     mitochondria protease (1 PK). The comparison of <br /> were determined SDS&#226;&#8364;&#8220;PAGE, immunoblotting densitometry. In      import efficiencies constructs demonstrates <br /> both (A) (B), import-competent preprotein added        role Tim44 unfolding cytochrome b2 <br /> excess; ,10% total added preprotein imported     haem-binding domain. The unfolding translocation of <br /> into mitochondria. <br />                                                                       loosely folded protein domains, b2(85)&#226;&#8364;&#8220;DHFR <br />                                                                       [and b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR, Figure 4A], far affected. <br /> system act translocating polypeptide similar                 The preprotein b2(220)&#226;&#8364;&#8220;DHFR contains intact <br /> stage import.                                                      cytochrome b2 presequence. This presequence directs the <br />    The remarkably high import efficiencies observed             processed proteins intermembrane space (Guiard, <br /> depletion Tim44 raise question Tim44                1985). A substitution alanine position 63 proline <br /> is intrinsic component protein import channel               entails mistargeting imported proteins the <br /> of inner membrane involved import              mitochondrial matrix (Beasley et al., 1993). We now <br /> reaction different way. We took advantage                 tested tim44-8 mitochondria defect in <br />  <br /> 4230 <br />                                                                                                                     Function Tim44 <br />  <br />  <br />                                                                           unfolding haem-binding domain observed <br />                                                                           preprotein containing modified presequence. <br />                                                                           We used hybrid protein b2(220)A63P&#226;&#8364;&#8220;DHFR (Figure <br />                                                                           5A) import tim44-8 mutant mitochon- <br />                                                                           dria inhibited similarly import b2(220)&#226;&#8364;&#8220; <br />                                                                           DHFR (Figure 5C). This result shows intact Tim44 <br />                                                                           facilitates import tightly folded cytochrome <br />                                                                           b2 haem-binding domain, independently this <br />                                                                           domain targeted intermembrane space to <br />                                                                           matrix. <br />                                                                              We wondered strong effect tim44-8 <br />                                                                           mutation translocation haem-binding domain <br />                                                                           caused weakening mitochondrial <br />                                                                           membrane potential &#226;&#710;&#8224;&#207;&#710; differential &#226;&#710;&#8224;&#207;&#710; dependence <br />                                                                           preproteins used. For purpose, compared <br />                                                                           membrane potential isolated wild-type mutant <br />                                                                           mitochondria (Figure 6A). As assay, used the <br />                                                                           quenching fluorescent dye 3,39-dipropylthiadicar- <br />                                                                           bocyanine iodide [DiSC3(5)]. We minor reduction <br />                                                                           membrane potential mutant mitochondria <br />                                                                           account slightly reduced import rate <br />                                                                           preproteins &#226;&#710;&#8224;&#207;&#710;-dependent steps (shown Figures <br />                                                                           2 3). In order exclude differential effects of <br />                                                                           tim44-8 mutation mediated indirectly a <br />                                                                           differential &#226;&#710;&#8224;&#207;&#710; dependence preproteins, deter- <br />                                                                           mined influence membrane potential the <br />                                                                           import rate preproteins (Figure 6B). We com- <br />                                                                           pared import preproteins presence <br />                                                                           increasing concentrations uncoupling reagent <br />                                                                           carbonyl cyanide m-chlorophenyl-hydrazone (CCCP; <br />                                                                           Figure 6B). The preprotein F1&#206;&#178; showed strong reduction <br />                                                                           import low concentrations CCCP, demon- <br />                                                                           strating strong dependence &#226;&#710;&#8224;&#207;&#710;. In contrast, import <br />                                                                           fusion proteins b2(220)A63P&#226;&#8364;&#8220;DHFR b2(85)&#226;&#8364;&#8220; <br />                                                                           DHFR revealed weak dependence membrane <br />                                                                           potential; AAC showed intermediate behaviour. <br />                                                                           There obvious correlation differ- <br />                                                                           ences CCCP titrations differences observed <br />                                                                           import experiments tim44-8 mitochondria <br />                                                                           shown Figures 2 5. In particular, construct <br />                                                                           b2(220)A63P&#226;&#8364;&#8220;DHFR, import experiments was <br /> Fig. 5. Import cytochrome b2 haem-binding domain inhibited      strongly affected inactivation Tim44, showed <br /> in tim44-8 mitochondria. (A) Cytochrome b2&#226;&#8364;&#8220;DHFR fusion proteins.          highest resistance reduction &#226;&#710;&#8224;&#207;&#710; CCCP <br /> In contrast b2(85)&#226;&#8364;&#8220;DHFR, preprotein b2(220)&#226;&#8364;&#8220;DHFR                   titrations. We conclude data membrane <br /> contains complete haem-binding domain (residues 81&#226;&#8364;&#8220;180). In             potential does account delayed translocation <br /> b2(220)A63P&#226;&#8364;&#8220;DHFR, intermembrane space sorting signal is <br /> inactivated substitution alanine position 63 proline. <br />                                                                           haem-binding domain tim44-8 mitochondria. <br /> (B) The preproteins b2(85)&#226;&#8364;&#8220;DHFR b2(220)&#226;&#8364;&#8220;DHFR                     The characterization import defects shown in <br /> synthesized reticulocyte lysate presence [35S]methionine/    Figures 2&#226;&#8364;&#8220;6 suggests Tim44 structural part <br /> [35S]cysteine imported isolated mitochondria times       import channel acts later steps import. <br /> indicated. To dissipate membrane potential, samples received <br /> valinomycin. After import reaction, mitochondria treated <br /> with proteinase K (1 PK) left protease (&#226;&#8364;&#8220; PK), reisolated      Function Tim44 translocation step after <br /> and analysed SDS&#226;&#8364;&#8220;PAGE fluorography. The relative amounts           insertion presequence inner <br /> of processed proteinase K-protected protein determined using     membrane <br /> a phosphorimager. The protein imported wild-type           We investigated possible role Tim44 &#226;&#710;&#8224;&#207;&#710;- <br /> mitochondria 30 min set 100% (control). p, precursor <br /> protein; i, processing intermediate; m, mature protein. (C) Import of <br />                                                                           independent steps import process. The preprotein <br /> the cytochrome b2 haem-binding domain matrix-targeted        Su9(70)&#226;&#8364;&#8220;DHFR accumulated spanning mito- <br /> preprotein. 35S-labelled b2(220)A63P&#226;&#8364;&#8220;DHFR synthesized              chondrial membranes import presence MTX <br /> reticulocyte lysate imported isolated mitochondria       (Schwarz et al., 1993; Ungermann et al., 1994). An active <br /> times indicated. Due amino acid substitution presequence,   protein import machinery capable pressing MTX- <br /> the preprotein (p) processed intermediate (i). The <br /> mitochondria treated proteinase K (1 PK) left        stabilized DHFR domain outer membrane, <br /> protease (&#226;&#8364;&#8220; PK), reisolated analysed SDS&#226;&#8364;&#8220;PAGE                  rendering intermediate resistant externally added <br /> fluorography. The relative amounts processed proteinase            proteases, subsequent dissipation mem- <br /> K-protected protein quantified.                                      brane potential (the DHFR domain hybrid protein <br />  <br />                                                                                                                                4231 <br />  U.Bo&#194;&#168;mer et al. <br />  <br />  <br />  <br />  <br /> Fig. 6. Inhibition import cytochrome b2 haem-binding domain tim44-8 mitochondria caused reduced membrane potential. <br /> (A) Assessment mitochondrial membrane potential &#226;&#710;&#8224;&#207;&#710;. Wild-type mitochondria (WT) tim44-8 mitochondria (200 &#194;&#181;g protein) pre- <br /> incubated 37&#194;&#176;C 15 min subsequently incubated membrane potential-sensitive dye DiSC3(5). The fluorescence changes were <br /> recorded fluorescence spectrophotometer 25&#194;&#176;C. Valinomycin (Val) added indicated dissipate membrane potential. The difference <br /> between fluorescence prior addition valinomycin represents relative assessment membrane potential. (B) Sensitivity of <br /> import efficiency reduction membrane potential. The preproteins imported described legend Figure 2A that <br /> increasing amounts CCCP included reduce mitochondrial membrane potential. All import reactions stopped 4 min and <br /> analysed SDS&#226;&#8364;&#8220;PAGE fluorography. The amounts processed protein quantified using phosphorimager, processed <br /> preprotein absence CCCP set 100% (control). <br />  <br />  <br /> stays outside membranes resistant proteolysis).                 spanning segment translocation intermediates. After <br /> A reduction activity import machinery leads                     dissipation membrane potential, precursor form <br /> to partial release translocation intermediate,                 Su9(70)&#226;&#8364;&#8220;DHFR processing intermediate of <br /> Su9 hybrid protein accessible                          Su9(81)&#226;&#8364;&#8220;DHFR released slowly import <br /> proteases. We followed principle determine                        sites, demonstrating &#226;&#710;&#8224;&#207;&#710;-independent retention mechan- <br /> the activity protein import machinery tim44-                    ism acts polypeptide (Figure 8, upper panel, <br /> 8 mitochondria (Figure 7). After import presence                       lane 5 versus lane 4, lane 11 versus lane 10). The <br /> of intact membrane potential MTX mitochon-                        shorter constructs released digested rapidly by <br /> dria wild-type mutant, Su9(70)&#226;&#8364;&#8220;                       added protease (Figure 8, precursor form lanes 1&#226;&#8364;&#8220; <br /> DHFR precursor protein resistant proteinase K                          3, processing intermediate lanes 4&#226;&#8364;&#8220;6). A <br /> (Figure 7B, lanes 2 8). The membrane potential                        comparison different constructs indicates that <br /> sufficient preproteins import channel.                     ~45&#226;&#8364;&#8220;50 amino acid residues required the <br /> After dissipation membrane potential, prepro-                      polypeptides import channel. In contrast, the <br /> teins remained intact wild-type mitochondria                       tim44-8 mutant mitochondria, &#226;&#710;&#8224;&#207;&#710;-independent reten- <br /> were released slowly (lanes 3&#226;&#8364;&#8220;6). In mutant                     tion polypeptides import sites observed with <br /> mitochondria, however, intermediates degraded                        constructs (Figure 8, lower panel). <br /> rapidly addition protease (lanes 9&#226;&#8364;&#8220;12). These                       The assay confirms Tim44 required in <br /> data indicate Tim44 required establish mem-                      retention polypeptides import channels, but <br /> brane potential-independent force acts polypep-                      demonstrates constructs suffi- <br /> tides transit mitochondrial membranes.                          cient <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> encounter Tim44-dependent activity. <br />    To determine location Tim44 functionally                      Since known independent studies ~50 amino <br /> gains access translocating polypeptide, con-                        acid residues required span mitochondrial <br /> structed series hybrid proteins containing N-terminal                    membranes (Rassow et al., 1990; Ungermann et al., 1994), <br /> parts different lengths subunit 9, fused DHFR                    conclude Tim44 acts preproteins have <br /> (Figure 8A). The constructs imported presence                     partially traversed inner membrane are <br /> of MTX Su9 taken molecular ruler                        accessible matrix membrane. <br /> to determine distance DHFR domain <br /> outside mitochondrial membranes location                       Discussion <br /> which Tim44 starts act translocation intermediate, <br /> using assay previous experiment. In wild-type                      Tim44 protein identified component of <br /> mitochondria, stability constructs import                   protein translocation machinery mitochondrial <br /> channels dependent length membrane-                         inner membrane (Maarse et al., 1992; Scherer et al., <br />  <br /> 4232 <br />                                                                                                                              Function Tim44 <br />  <br />  <br />  <br />  <br />                                                                           Fig. 8. Su9&#226;&#8364;&#8220;DHFR fusion proteins require membrane-spanning <br />                                                                           segments ~45 amino acids stable Tim44-dependent insertion <br />                                                                           translocation sites. (A) Su9&#226;&#8364;&#8220;DHFR fusion proteins. The constructs <br /> Fig. 7. Reduced activity protein import machinery enhanced     contain different N-terminal parts Su9 mitochondrial ATP <br /> mobility translocation intermediates tim44-8 mitochondria.          synthase fused entire DHFR mouse. The proteins contain <br /> (A) Scheme experiment. (B) As illustrated (A), hybrid       processing sites (designated 1. 2.) residues 35 36 <br /> protein Su9(70)&#226;&#8364;&#8220;DHFR (containing residues 1&#226;&#8364;&#8220;70 Su9 fused        residues 66 67, respectively. The numbers shown in <br /> entire DHFR, Figure 8A) synthesized reticulocyte lysate      figure refer C-terminal residue Su9 the <br /> in presence [35S]methionine/[35S]cysteine imported        respective construct. Both parts fusion proteins connected by <br /> mitochondria tim44-8 mutant corresponding wild-         linker (Gly, Ile) residues (Gly, Ser, Gly, Ile), as <br /> type presence MTX. Subsequently, mitochondria               indicated. (B) Resistance Su9&#226;&#8364;&#8220;DHFR fusion proteins proteinase <br /> containing accumulated translocation intermediate washed         K accumulation mitochondrial protein import sites. The fusion <br /> and incubated 25&#194;&#176;C presence valinomycin (to             proteins synthesized reticulocyte lysate presence of <br /> dissipate membrane potential &#226;&#710;&#8224;&#207;&#710;) MTX (to stabilize native     [35S]methionine imported mitochondria tim44-8 mutant <br /> folding state DHFR domain). After times indicated,         corresponding wild-type presence MTX. After <br /> mitochondria cooled ice treated proteinase K (PK).       reisolation washing, mitochondria resuspended a <br /> The mitochondria reisolated analysed SDS&#226;&#8364;&#8220;PAGE             buffer containing valinomycin (to dissipate membrane potential <br /> fluorography. The activity mitochondrial protein import            &#226;&#710;&#8224;&#207;&#710;) MTX (to stabilize native folding state DHFR). The <br /> machinery leads pressing DHFR domain outer          mitochondria incubated 25&#194;&#176;C times indicated then <br /> membrane resistance externally added protease.         cooled 0&#194;&#176;C. The samples treated proteinase K digest <br /> Reduced activity import machinery allows access            unstable translocation intermediates. The mitochondria reisolated <br /> protease. The presequence Su9 contains sites processing,    analysed SDS&#226;&#8364;&#8220;PAGE fluorography. <br /> position 35 (i-form) position 66 (m-form). p, precursor protein; <br /> i, processing intermediate; m, mature protein. The relative amounts of <br /> the protease-resistant pSu9(70)&#226;&#8364;&#8220;DHFR translocation imtermediates          import channel Tim44 essential part <br /> were determined using phosphorimager. The amounts retained        import motor. Indeed number of <br /> the incubation presence membrane potential        translocation sites affected inactivation of <br /> reisolation, prior dissipation (time 0), set 100% (control). <br />                                                                           Tim44, indicating Tim44 required the <br />                                                                           formation protein import pore. This conclusion is <br /> 1992). Since then, little information accumulated                corroborated observation formation the <br /> concerning precise function Tim44. It                     90 kDa complex Tim proteins tightly associated <br /> speculated Tim44 involved essentially                    translocating polypeptides reduced the <br /> the formation protein import pore; alternatively,               depletion Tim44 (unpublished data). The formation of <br /> has suggested Tim44 mtHsp70                       inner membrane protein import channel be <br /> might constitute motor mitochondrial protein import                function proteins Tim44; candidates are <br /> (Maarse et al., 1992; Horst et al., 1993; Glick, 1995;                    Tim23 Tim17 intrinsic components the <br /> Pfanner Meijer, 1995; Rassow Pfanner, 1995b;                      90 kDa complex (Bauer et al., 1996; Dekker et al., 1997; <br /> Neupert, 1997). In study, addressed                    Lohret et al., 1997). <br /> alternatives selective inactivation Tim44 mito-                   The insertion Tim23 Tim17 inner <br /> chondria temperature-sensitive mutant strain                   membrane does require mtHsp70 Tim44 (Bo&#194;&#168;mer <br /> the characterization consequences preprotein                   et al., 1997; Ka&#194;&#180;ldi et al., 1998). However, rule <br /> translocation.                                                            possibility Tim44 play role the <br />    Surprisingly, inactivation Tim44 allows quite                   import proteins involved in <br /> efficient import preproteins, concentra-                  initial steps channel formation. Once established, the <br /> tions close saturation import sites. This observation               maintenance functional import channels appears be <br /> thus challenges concept Tim44 component                  completely Tim44-independent. <br />  <br />                                                                                                                                            4233 <br />  U.Bo&#194;&#168;mer et al. <br />  <br />  <br />    Our functional data obtained tim44-8 mutant          folded preproteins (Glick et al., 1993; Voos et al., 1996; <br /> indicate preprotein traverse inner mem-        Matouschek et al., 1997). <br /> brane reach matrix encounter con-               The stability Su9&#226;&#8364;&#8220;DHFR translocation inter- <br /> sequences inactivation Tim44. Considering          mediates import sites shows striking dependence on <br /> Tim44 structural component protein import        presence Tim44. We suggest Tim44 strongly <br /> pore amino acid sequence Tim44 shows           facilitates transfer mtHsp70 translocating <br /> hydrophilic protein does reveal potential          polypeptide directly membrane. In presence of <br /> membrane-spanning segments, suggest Tim44 acts           Tim44, mobility accumulated translocation <br /> at inner inner membrane, cooperation          intermediates reduced drastically, suggesting poly- <br /> with Tim proteins form import channel.           peptide segments emerge import sites immedi- <br />    Excluding role Tim44 pore-forming protein,         ately receive mtHsp70 molecule. This high Tim44- <br /> what role mitochondrial protein import? After       dependent fidelity import motor appears be <br /> had observed Tim44 form complex             required unfolding import tightly folded <br /> mtHsp70, speculated Tim44 required          domains. In absence Tim44 function, mtHsp70 can <br /> to deliver mtHsp70 translocating polypeptide              drive import loosely folded preproteins though <br /> (Kronidou et al., 1994; Rassow et al., 1994; Schneider           bound mtHsp70s distant membrane <br /> et al., 1994). It known studies mtHsp70      (the model protein shows high mobility the <br /> binds translocating preproteins essential           import channel experiment Figure 7B same <br /> component import machinery (Kang et al., 1990;            preprotein imported efficiently Figure 3A). Our <br /> Scherer et al., 1990). However, possible           results suggest mitochondria functions of <br /> investigate Tim44 actually plays role          channel formation generation strong force in <br /> process. We preproteins          vectorial protein movement mediated different <br /> strictly mtHsp70-dependent translocation (Gambill       subcomplexes import machinery. Tim23 Tim17 <br /> et al., 1993; Glick et al., 1993; Voos et al., 1993; Stuart      constitute import pore; Tim44 with <br /> et al., 1994) imported reasonable efficiencies    mtHsp70 appears dynamic element the <br /> into tim44-8 mitochondria. We conclude transfer         import machinery. A similar situation emerged the <br /> of mtHsp70 polypeptides emerging import            elucidation protein transport mechanisms the <br /> sites entirely dependent Tim44. At       endoplasmic reticulum. Here, corresponding functions <br /> degree, Tim44-independent binding mtHsp70            exerted complexes Sec61p Sec63p, respect- <br /> be possible, binding appears functional        ively (Brodsky Schekman, 1993; Panzner et al., 1995; <br /> the mechanism protein import.                                 Corsi Schekman, 1997; Lyman Schekman, 1997). <br />                                                                     In summary, Tim44 structural <br />    However, testing consequences depletion <br />                                                                  constituent inner membrane protein import channel. <br /> of active Tim44 protein import, observed drastic <br />                                                                  Tim44 acts inner inner membrane <br /> reduction import tightly folded protein domain. <br />                                                                  element independent motor <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> driving <br /> In contrast import loosely folded domains, <br />                                                                  unfolding translocation preproteins. Mitochondrial <br /> the unfolding translocation tightly folded domain <br />                                                                  Tim44 plays stimulatory role import loosely <br /> require involvement Tim44. This result remarkable      folded preproteins, crucial pulling folded <br /> in context data obtained previously            domains. <br /> with mutant forms mtHsp70 (von Ahsen et al., 1995; <br /> Voos et al., 1996; Bo&#194;&#168;mer et al., 1997). The mtHsp70 <br /> which encoded allele ssc1-2 shows strong binding       Materials methods <br /> to preproteins impaired binding Tim44 to <br />                                                                  Construction plasmids manipulation S.cerevisiae <br /> Tim17&#226;&#8364;&#8220;Tim23. This defect correlates delayed import        strains <br /> of tightly folded protein domains, raising question       Temperature-sensitive alleles TIM44 generated low fidelity <br /> whether complex mtHsp70 Tim44                 PCR technique published previously (Rassow et al., 1994). The <br /> complex Tim17&#226;&#8364;&#8220;Tim23 involved import               mutant alleles cloned single-copy vector YCplac111 and <br />                                                                  introduced diploid strain MB2-22 (Maarse et al., 1992). After <br /> reaction. The results tim44-8 mutant demonstrate        random sporulation, haploid cells disrupted nuclear TIM44 gene <br /> that Tim44 required translocation   harbouring rescuing plasmid tested temperature sensitivity. <br /> tightly folded domain, mtHsp70&#226;&#8364;&#8220;           One isolated alleles designated tim44-8. A corresponding <br /> Tim17&#226;&#8364;&#8220;Tim23 complexes mitochondria substitute          strain containing wild-type TIM44 gene used wild-type <br />                                                                  strain. Hybrid proteins containing N-terminal parts Su9 fused to <br /> for mtHsp70&#226;&#8364;&#8220;Tim44 complexes inactivation           murine DHFR (Viebrock et al., 1982; Ungermann et al., 1994) were <br /> of Tim44. We conclude complexes mtHsp70              obtained vitro transcription/translation using cDNA constructs <br /> with Tim17&#226;&#8364;&#8220;Tim23 Tim44 play different roles             cloned vector pGEM4 (Promega). To assemble Su9(58)&#226;&#8364;&#8220;DHFR, <br /> in protein import mechanism. The strong &#226;&#8364;&#732;pulling&#226;&#8364;&#8482;            Su9(76)&#226;&#8364;&#8220;DHFR Su9(81)&#226;&#8364;&#8220;DHFR, corresponding Su9 parts were <br />                                                                  obtained PCR ligated pGEM4 plasmid containing DHFR <br /> function required unfolding tightly folded       cDNA, using BamHI site cDNA upstream DHFR start <br /> domains attributed mtHsp70 (Glick, 1995;         ATG EcoRI site vector. Su9 DHFR parts thereby <br /> Pfanner Meijer, 1995; Schatz, 1996; Matouschek               connected linker residues (Gly, Ser, Gly, Ile). The construct <br /> et al., 1997) exerted Tim17&#226;&#8364;&#8220;Tim23-bound            Su9(70)&#226;&#8364;&#8220;DHFR described previously (Pfanner et al., 1987). <br /> mtHsp70 mtHsp70 cooperation Tim44. Our <br />                                                                  Import preproteins isolated mitochondria cross- <br /> data exclude, however, mtHsp70&#226;&#8364;&#8220;Tim17&#226;&#8364;&#8220;            linking Tim44 <br /> Tim23 complex involved formation            Yeast cells grown YPG medium (1% Bacto-yeast extract, <br /> weaker pulling activity required import loosely      2% Bacto-peptone, 3% glycerol) 30&#194;&#176;C. Mitochondria isolated <br />  <br />  <br /> 4234 <br />                                                                                                                                    Function Tim44 <br />  <br />  <br /> essentially according protocol Daum et al. (1982),              (2 &#194;&#181;g/ml antipain, 5 &#194;&#181;g/ml aprotinin, 0.25 &#194;&#181;g/ml chymostatin, 1.25 &#194;&#181;g/ <br /> modifications published Gambill et al. (1993). To induce             ml leupeptin, 0.5 &#194;&#181;g/ml pepstatin A, 0.2 mM PMSF) shaken end- <br /> tim44-8 phenotype, mitochondria pre-incubated 37&#194;&#176;C 15          over-end 10 min 8&#194;&#176;C. Insoluble material removed by <br /> min cooled 25&#194;&#176;C subsequent import experiments.                 ultracentrifugation (30 min 100 000 g). The supernatants were <br /> Preproteins synthesized rabbit reticulocyte lysate presence     incubated 45 min 8&#194;&#176;C end-over-end shaking antibodies <br /> of [35S]methionine/[35S]cysteine (Amersham) vitro transcription       pre-bound protein A&#226;&#8364;&#8220;Sepharose. After washing cycles lysis <br /> by SP6 RNA polymerase (Stratagene). To import preproteins,             buffer, protein A&#226;&#8364;&#8220;Sepharose pellets boiled sample buffer <br /> reticulocyte lysates diluted import buffer [3% (w/v) bovine          analysed SDS&#226;&#8364;&#8220;PAGE. <br /> serum albumin (BSA, fatty acid-free; Sigma), 250 mM sucrose, 60 mM <br /> KCl, 5 mM MgCl2, 5 mM sodium malate, 2 mM ATP, 20 mM potassium <br /> phosphate, 10 mM MOPS-KOH, pH 7.2] isolated mitochondria                   Acknowledgements <br /> were added final concentration 25 &#194;&#181;g mitochondrial protein <br /> per 100 &#194;&#181;l assay. Under conditions, mitochondria           We thank Dr Oliver von Ahsen Hanne Mu&#194;&#168;ller help screen <br /> was limiting import experiments. Some samples received 1 &#194;&#181;M             suitable temperature-sensitive mutant TIM44, Nicole Zufall for <br /> valinomycin 100-fold concentrated stock solution ethanol          help construction Su9&#226;&#8364;&#8220;DHFR plasmids, Dr Bernard Guiard for <br /> dissipate membrane potential. The samples incubated           providing b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR-overproducing E.coli strain, Drs <br /> 25&#194;&#176;C 30 min. To accumulate translocation intermediates, hybrid       Michael Brunner, Bernd Bukau, Elizabeth Craig, Arthur Horwich and <br /> proteins containing DHFR domain imported presence             Wolfgang Voos helpful discussions. The work supported the <br /> 2 &#194;&#181;M MTX (Sigma). For treatment mitochondria import           Deutsche Forschungsgemeinschaft, Sonderforschungsbereich 388, <br /> reactions, proteinase K used final concentration 40 &#194;&#181;g/ml          Fonds der Chemischen Industrie Mu&#194;&#168;ller-Fahnenberg-Stiftung. <br /> for 10 min 0&#194;&#176;C. Mitochondria reisolated centrifugation at <br /> 16 000 g 10 min. Proteins analysed SDS&#226;&#8364;&#8220;PAGE and <br /> autoradiography. Relative amounts 35S-labelled proteins quanti-        References <br /> fied using storage PhosphorImaging (Molecular Dynamics <br /> Inc.). Chemical amounts soluble preprotein b2(167)&#226;&#710;&#8224;&#226;&#8364;&#8220;DHFR           Bauer,M.F., Sirrenberg,C., Neupert,W. Brunner,M. (1996) Role of <br /> purified expression E.coli published previously (Dekker               Tim23 voltage sensor presequence receptor protein import <br /> et al., 1997). The imported protein determined                   mitochondria. Cell, 87, 33&#226;&#8364;&#8220;41. <br /> immunoblotting, using antiserum raised DHFR moiety              Beasley,E.M., Mu&#194;&#168;ller,S. Schatz,G. (1993) The signal sorts yeast <br /> and comparison standard amounts purified preprotein (Dekker           cytochrome b2 mitochondrial intermembrane space contains <br /> et al., 1997). Blue native electrophoresis mitochondrial proteins          distinct functional regions. EMBO J., 12, 2303&#226;&#8364;&#8220;2311. <br /> performed following published procedure (Scha&#194;&#168;gger von Jagow,            Berthold,J., Bauer,M.F., Schneider,H.-C., Klaus,C., Dietmeier,K., <br /> 1991; Dekker et al., 1996, 1997).                                                 Neupert,W. Brunner,M. (1995) The MIM complex mediates <br />    For cross-linking experiments, hybrid protein Su9(70)&#226;&#8364;&#8220;DHFR             preprotein translocation mitochondrial inner membrane and <br /> accumulated import sites import ATP-depleted mitochondria.             couples mt-Hsp70/ATP driving system. Cell, 81, 1085&#226;&#8364;&#8220;1093. <br /> Mitochondria (in presence 20 &#194;&#181;M oligomycin inhibit F0F1-         Blom,J., Ku&#194;&#168;brich,M., Rassow,J., Voos,W., Dekker,P.J.T., Maarse,A.C., <br /> ATPase) reticulocyte lysate ATP depleted incubation              Meijer,M. Pfanner,N. (1993) The essential yeast protein MIM44 <br /> apyrase (10 U/ml; Sigma) 10 min 0&#194;&#176;C; import performed              (encoded MPI1) required early step preprotein <br /> in presence 20 &#194;&#181;M oligomycin. The mitochondria washed                 translocation mitochondrial inner membrane. Mol. Cell. <br /> by centrifugation sucrose cushion (500 mM sucrose, 1 mM                 Biol., 13, 7364&#226;&#8364;&#8220;7371. <br /> EDTA, 10 mM MOPS, pH 7.2) resuspended 250 mM sucrose,                   Blom,J., Dekker,P.J.T. Meijer,M. (1995) Functional physical <br /> 1 mM EDTA, 10 mM MOPS-KOH pH 7.2 (SEM buffer). For cross-                         interactions components yeast mitochondrial inner-membrane <br /> linking EGS (Pierce), mitochondria (50 &#194;&#181;g protein)                  import machinery (MIM). Eur. J. Biochem., 232, 309&#226;&#8364;&#8220;314. <br /> incubated 0&#194;&#176;C 20 min 1 ml SEM containing EGS final           Bo&#194;&#168;mer,U., Meijer,M., Maarse,A.C., Dekker,P.J.T., Pfanner,N. and <br /> concentration 200 &#194;&#181;M. The reaction stopped addition                  Rassow,J. (1997) Multiple interactions components mediating <br /> 100 mM Tris&#226;&#8364;&#8220;HCl pH 7.2 second incubation 20 min 0&#194;&#176;C.                  preprotein translocation inner mitochondrial membrane. <br /> Cross-linking DSS (Pierce) performed following basically               EMBO J., 16, 2205&#226;&#8364;&#8220;2216. <br /> same protocol, using DSS final concentration 400 &#194;&#181;M. Details           Brodsky,J.L. Schekman,R. (1993) A Sec63p&#226;&#8364;&#8220;BiP complex <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> of procedure published previously (Blom et al., 1993; Ku&#194;&#168;brich           yeast required protein translocation reconstituted <br /> et al., 1994). Proteins precipitated addition 10% trichloroacetic      proteoliposome. J. Cell Biol., 123, 1355&#226;&#8364;&#8220;1363. <br /> acid presence 0.0125% deoxycholate. For immunoprecipitation,         Corsi,A.K. Schekman,R. (1997) The lumenal domain Sec63p <br /> the precipitates lysed 1% SDS, 60 mM Tris&#226;&#8364;&#8220;HCl pH 6.8. The                 stimulates ATPase activity BiP mediates BiP recruitment <br /> samples diluted 40-fold addition 1% (w/v) Triton X-100,                translocon Saccharomyces cerevisiae. J. Cell Biol., 137, <br /> 0.3 M NaCl, 10 mM Tris&#226;&#8364;&#8220;HCl pH 7.5 antibodies added                   1483&#226;&#8364;&#8220;1493. <br /> bound protein A&#226;&#8364;&#8220;Sepharose (Pharmacia).                                      Daum,G., Bo&#194;&#168;hni,P.C. Schatz,G. (1982) Import proteins into <br />                                                                                   mitochondria. Cytochrome b2 cytochrome c peroxidase located <br /> Assessment mitochondrial membrane potential                                intermembrane space yeast mitochondria. J. Biol. Chem., <br /> The &#226;&#710;&#8224;&#207;&#710; isolated yeast mitochondria assessed recording               257, 13028&#226;&#8364;&#8220;13033. <br /> fluorescence decrease voltage-sensitive dye DiSC3(5) (Molecular         Dekker,P.J.T., Keil,P., Rassow,J., Maarse,A.C., Pfanner,N. Meijer,M. <br /> Probes) essentially described previously (Sims et al., 1974; Ga&#194;&#168;rtner          (1993) Identification MIM23, putative component protein <br /> et al., 1995). An aliquot 200 &#194;&#181;g mitochondrial protein used             import machinery mitochondrial inner membrane. FEBS Lett., <br /> per assay. The mitochondria pre-incubated 37&#194;&#176;C 15 min              330, 66&#226;&#8364;&#8220;70. <br /> induce phenotype tim44-8 mutant. The measurements              Dekker,P.J.T., Mu&#194;&#168;ller,H., Rassow,J. Pfanner,N. (1996) <br /> then performed 25&#194;&#176;C.                                                           Characterization preprotein translocase outer <br />                                                                                   mitochondrial membrane blue native electrophoresis. Biol. Chem., <br /> Co-immunoprecipitations                                                           377, 535&#226;&#8364;&#8220;538. <br /> All antibodies used study polyclonal antibodies raised         Dekker,P.J.T., Martin,F., Maarse,A.C., Bo&#194;&#168;mer,U., Mu&#194;&#168;ller,H., Guiard,B., <br /> rabbits isolated proteins. For co-immunoprecipitations, antibodies        Meijer,M., Rassow,J. Pfanner,N. (1997) The Tim core complex <br /> (6 &#194;&#181;l serum) pre-bound protein A&#226;&#8364;&#8220;Sepharose (10 &#194;&#181;l wet                  defines number mitochondrial translocation contact sites and <br /> volume; Pharmacia Biotech Inc.) 1 h 480 &#194;&#181;l lysis buffer [1%             hold arrested preproteins absence matrix Hsp70/Tim44. <br /> digitonin (13 recrystallized ethanol), 10% (w/v) glycerol, 50 mM             EMBO J., 16, 5408&#226;&#8364;&#8220;5419. <br /> NaCl, 2 mM EDTA, 30 mM HEPES-KOH, pH 7.4]. In parallel,                      Dietmeier,K., Ho&#194;&#168;nlinger,A., Bo&#194;&#168;mer,U., Dekker,P.J.T., Eckerskorn,C., <br /> preprotein containing DHFR domain synthesized reticulocyte             Lottspeich,F., Ku&#194;&#168;brich,M. Pfanner,N. (1997) Tom5 functionally <br /> lysate imported presence MTX isolated mitochondria             links mitochondrial preprotein receptors general import pore. <br /> (20 &#194;&#181;g mitochondrial protein). After washing SEMP buffer                  Nature, 388, 195&#226;&#8364;&#8220;200. <br /> [250 mM sucrose, 1 mM EDTA, 10 mM MOPS&#226;&#8364;&#8220;KOH, pH 7.2, 0.2 mM                     Emtage,J.L.T. Jensen,R.E. (1993) MAS6 encodes essential inner <br /> freshly added phenylmethylsulfonyl fluoride (PMSF)], mitochondria             membrane component yeast mitochondrial protein import <br /> were resuspended lysis buffer supplemented protease inhibitors            pathway. J. Cell Biol., 122, 1003&#226;&#8364;&#8220;1012. <br />  <br />  <br />                                                                                                                                                  4235 <br />  U.Bo&#194;&#168;mer et al. <br />  <br />  <br /> Gambill,D., Voos,W., Kang,P.J., Miao,B., Langer,T., Craig,E.A.            mitochondrial inner membrane protein involved protein import. <br />    Pfanner,N. (1993) A dual role mitochondrial heat shock protein         FEBS Lett., 394, 215&#226;&#8364;&#8220;221. <br />    70 membrane translocation preproteins. J. Cell Biol., 123,         Martin,J., Mahlke,K. Pfanner,N. (1991) Role energized inner <br />    109&#226;&#8364;&#8220;117.                                                                   membrane mitochondrial protein import. &#226;&#710;&#8224;&#207;&#710; drives movement <br /> Ga&#194;&#168;rtner,F., Voos,W., Querol,A., Miller,B., Craig,E.A., Cumsky,M.G.       presequences. J. Biol. Chem., 266, 18051&#226;&#8364;&#8220;10957. <br />    Pfanner,N. (1995) Mitochondrial import subunit Va cytochrome       Matouschek,A., Azem,A., Ratliff,K., Glick,B.S., Schmid,K. and <br />    c oxidase characterized yeast mutants: independence              Schatz,G. (1997) Active unfolding precursor proteins during <br />    receptors, requirement matrix hsp70 translocase function.          mitochondrial protein import. EMBO J., 16, 6727&#226;&#8364;&#8220;6736. <br />    J. Biol. Chem., 270, 3788&#226;&#8364;&#8220;3795.                                          Neupert,W. (1997) Protein import mitochondria. Annu. Rev. <br /> Glick,B.S. (1995) Can Hsp70 proteins act force-generating motors?          Biochem., 66, 863&#226;&#8364;&#8220;917. <br />    Cell, 80, 11&#226;&#8364;&#8220;14.                                                         Panzner,S., Dreier,L., Hartmann,E., Kostka,S. Rapoport,T.A. (1995) <br /> Glick,B.S., Wachter,C., Reid,G.A. Schatz,G. (1993) Import              Posttranslational protein transport yeast reconstituted purified <br />    cytochrome b2 mitochondrial intermembrane space: tightly        complex Sec proteins Kar2p. Cell, 81, 561&#226;&#8364;&#8220;570. <br />    folded heme-binding domain makes import dependent matrix            Pfanner,N. Meijer,M. (1995) Protein sorting&#226;&#8364;&#8221;pulling proteins. <br />    ATP. Protein Sci., 2, 1901&#226;&#8364;&#8220;1917.                                           Curr. Biol., 5, 132&#226;&#8364;&#8220;135. <br /> Guiard,B. (1985) Structure, expression regulation nuclear gene     Pfanner,N. Meijer,M. (1997) Mitochondrial biogenesis: Tom and <br />    encoding mitochondrial protein: yeast L (1)-lactate cytochrome       Tim machine. Curr. Biol., 7, 100&#226;&#8364;&#8220;103. <br />    c oxidoreductase (cytochrome b2). EMBO J., 4, 3265&#226;&#8364;&#8220;3272.                 Pfanner,N., Tropschug,M. Neupert,W. (1987) Mitochondrial protein <br /> Horst,M., Jeno&#194;&#168;,P., Kronidou,N.G., Bolliger,L., Oppliger,W., Scherer,P.,      import: nucleoside triphosphates involved conferring import- <br />    Manning-Krieg,U., Jascur,T. Schatz,G. (1993) Protein import       competence precursors. Cell, 49, 815&#226;&#8364;&#8220;823. <br />    yeast mitochondria: inner membrane import site protein ISP45      Pfanner,N., Rassow,J., Guiard,B., So&#194;&#168;llner,T., Hartl,F.U. Neupert,W. <br />    MPI1 gene product. EMBO J., 12, 3035&#226;&#8364;&#8220;3041.                             (1990) Energy requirement unfolding membrane translocation <br /> Horst,M., Hilfiker-Rothenfluh,S., Oppliger,W. Schatz,G. (1995)            precursor proteins import mitochondria. J. Biol. Chem., <br />    Dynamic interaction protein translocation systems inner      265, 16324&#226;&#8364;&#8220;16329. <br />    outer membranes yeast mitochondria. EMBO J., 14, 2293&#226;&#8364;&#8220;2297.       Pfanner,N. et al. (1996) Uniform nomenclature protein transport <br /> Horst,M., Opplinger,W., Feifel,B., Schatz,G. Glick,B.S. (1996) The        machinery mitochondrial membranes. Trends Biochem. Sci., <br />    mitochondrial protein import motor: dissociation mitochondrial          21, 51&#226;&#8364;&#8220;52. <br />    hsp70 membrane anchor requires ATP binding          Rassow,J. Pfanner,N. (1995a) Molecular chaperones intracellular <br />    ATP hydrolysis. Protein Sci., 5, 759&#226;&#8364;&#8220;767.                                  protein translocation. Rev. Physiol., Biochem. Pharmacol., 126, 199&#226;&#8364;&#8220; <br /> Hwang,S.T., Wachter,C. Schatz,G. (1991) Protein import           264. <br />    yeast mitochondrial matrix. A new translocation intermediate     Rassow,J. Pfanner,N. (1995b) The motor mitochondrial protein <br />    mitochondrial membranes. J. Biol. Chem., 266, 21083&#226;&#8364;&#8220;21089.         import: mitochondrial analog Sec63p&#226;&#8364;&#8220;Kar2p system. In <br /> Ka&#194;&#180;ldi,K., Bauer,M.F., Sirrenberg,C., Neupert,W. Brunner,M. (1998)        Qagliariello,E. Palmieri,F. (eds), 30 Years Mitochondrial <br />    Biogenesis Tim23 Tim17, integral components TIM              Research. Elsevier Science Publishers, Amsterdam, pp. 113&#226;&#8364;&#8220;118. <br />    machinery matrix-targeted preproteins. EMBO J., 17, 1569&#226;&#8364;&#8220;1576.       Rassow,J., Guiard,B., Wienhues,U., Herzog,V., Hartl,F.U. Neupert,W. <br /> Kanamori,T., Nishikawa,S.-I., Shin,I., Schultz,P.G. Endo,T. (1997)        (1989) Translocation arrest reversible folding precursor protein <br />    Probing environment protein import pathways yeast         imported mitochondria. A means quantitate translocation <br />    mitochondria site-specific photocrosslinking. Proc. Natl Acad. Sci.     contact sites. J. Cell Biol., 109, 1421&#226;&#8364;&#8220;1428. <br />    USA, 94, 485&#226;&#8364;&#8220;490.                                                        Rassow,J., Hartl,F.U., Guiard,B., Pfanner,N. Neupert,W. (1990) <br /> Kang,P.J., Ostermann,J., Shilling,J., Neupert,W., Craig,E.A.              Polypeptides traverse mitochondrial envelope extended state. <br />                                                                               FEBS Lett., 275, 190&#226;&#8364;&#8220;194. <br />    Pfanner,N. (1990) Requirement hsp70 mitochondrial matrix <br />                                                                             Rassow,J., Maarse,A.C., Krainer,E., Ku&#194;&#168;brich,M., Mu&#194;&#168;ller,H., Meijer,M., <br />    translocation folding precursor proteins. Nature, 348, <br />                                                                               Craig,E.A. Pfanner,N. (1994) Mitochondrial protein  <br /> </body></html>