<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>INFECTION AND IMMUNITY, June 2005, p. 3568&#226;&#8364;&#8220;3576                                                                                   Vol. 73, No. 6 <br /> 0019-9567/05/$08.00&#226;&#171;&#185;0 doi:10.1128/IAI.73.6.3568&#226;&#8364;&#8220;3576.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  Inducible Antisense RNA Expression Characterization Gene <br />                   Functions Streptococcus mutans <br />                                            Bing Wang Howard K. Kuramitsu* <br />                          Department Oral Biology, State University New York, Buffalo, New York 14214 <br />                      Received 12 November 2004/Returned modification 6 January 2005/Accepted 15 February 2005 <br />  <br />              In order examine gene function Streptococcus mutans, recently initiated antisense RNA <br />           strategy. Toward end, constructed evaluated Escherichia coli-S. mutans shuttle <br />           expression vectors fruA scrB promoters S. mutans, tetR-controlled tetO promoter <br />           Staphylococcus aureus. Among these, tetO/tetR proved tightly controlled pro- <br />           moter. By using shuttle plasmid system, modulation gene function inducible antisense RNA expres- <br />           sion demonstrated comC antisense fragments different sizes distinct gtfB antisense <br />           fragments. It demonstrated size, relative position, antisense DNA fragment is <br />           important mediating antisense phenomenon. Furthermore, constructing screening random DNA <br />           libraries tet expression shuttle system, 78 growth-retarded transformants harboring antisense DNA <br />           fragments identified. Almost corresponded homologous essential genes other <br />           bacteria. In addition, novel essential gene, coaE gene, encoding dephospho-coenzyme A kinase, is <br />           involved final step coenzyme A catabolism S. mutans, identified characterized. These results <br />           suggest antisense RNA strategy useful identifying novel essential genes S. mutans bacteria <br />           characterizing physiology (including potential virulence factors) organisms. <br />  <br />  <br />    Streptococcus mutans primary etiologic agent tooth                pared traditional gene inactivation strategies result in <br /> decay humans. One important virulence traits               total loss gene function, regulated gene expression can <br /> this pathogen sucrose-dependent colonization tooth sur-                provide quantitative information functional impor- <br /> faces (biofilm formation). This property dependent               tance gene product. It allows creation condi- <br /> expression glucosyltransferase (GTF) genes (gtf)              tional lethal phenotypes essential genes. In addition, manip- <br /> organism (22). One gene products, GtfB, important                ulating expression genes important tool for <br /> for insoluble glucan synthesis (1) shown induc-                understanding regulatory cascade mechanisms impor- <br /> ible biofilms (26, 45). S. mutans naturally transformable            tance gene regulation pathogenesis. <br /> bacterium. The comC gene product, competence-stimulating                      Interference mRNA expression antisense RNA has <br /> peptide (CSP), appears play important role facilitating             used effectively eukaryotic systems inhibit gene <br /> the natural transformation S. mutans bacteria                  expression (5). Recently, approach used with <br /> biofilm formation aciduricity organisms (23).                 Staphylococcus aureus (19), mycobacteria (32) Candida <br /> Bacteria adapt environmental changes modifying bi-               albicans (6) identify essential genes virulence factors as <br /> ological processes adjusting expression levels genes          study gene functions microorganisms (17, 46). For <br /> involved. Since numerous gene products associated             purposes, regulated promoter essential. Al- <br /> these biological processes, evaluating biological processes            variety promoters vectors devel- <br /> via modulation specific target gene expression important            oped allow quantitative modulation gene expression in <br /> evaluating role target genes physiology             gram-negative bacteria (3, 7, 10, 14, 43), systems <br /> organism. By using new tetO/tetR-regulated S. mutans shuttle               function gram-positive organisms. This likely <br /> vector, constructed series antisense strains modulating the <br />                                                                              results more-stringent control promoter usage in <br /> expression comC genes gtfB genes. <br />                                                                              gram-positive species relative gram-negative species <br /> Natural transformation comC antisense strains biofilm <br />                                                                              (28, 29). Although useful promoter systems re- <br /> formation gtfB antisense strains evaluated induc- <br />                                                                              ported S. aureus, Streptococcus pneumoniae, Bacillus <br /> tion expression antisense DNAs. <br />                                                                              subtilis (8, 20, 12, 21, 34, 36, 44), strong regulatory systems <br />    Bacterial pathogens regulate expression viru- <br />                                                                              available S. mutans. Among gram-positive bacte- <br /> lence genes coordinated manner response changes in <br />                                                                              rial promoter systems, Ptet induced tetracycline its <br /> the environment. The specific modulation target gene ex- <br />                                                                              derivative doxycycline, common components in <br /> pression powerful strategy study contribution of <br /> a given gene product bacterial growth virulence. Com-                 bacterial media allow induction promoter <br />                                                                              activity interfering bacterial metabolism. Re- <br />                                                                              cently, promoters induced sugars S. <br />                                                                              mutans identified: scrB promoter induced nearly <br />   * Corresponding author. Mailing address: Department Oral Bi- <br /> ology, State University New York Buffalo, 3435 Main Street, <br />                                                                              threefold sucrose (16) fruA promoter induced <br /> Buffalo, NY 14214. Phone: (512) 249-5901. Fax: (716) 829-3942. E-            approximately fourfold inulin relative fructose (4). In this <br /> mail: Kuramitsu@earthlink.net.                                               study, adapted tetO/tetR promoter S. aureus <br />  <br />                                                                       3568 <br />  VOL. 73, 2005                                                                                        INDUCIBLE ANTISENSE RNA EXPRESSION                                3569 <br />  <br />  <br /> expression system, fruA scrB promoters S.                         pTetE pTetR pair shuttle plasmids tetO/tetR opposite <br /> mutans, construct shuttle expression vectors evaluate                          orientation (Fig. 1). For construction S. mutans transformable shuttle <br />                                                                                           vector, plasmid pSCR749(B), derivative BglII site deleted multi- <br /> relative strengths induction properties promoters.                           cloning site (MCS) pTS749, digested NcoI EcoRI. The HpaI- <br />    The availability complete bacterial genome sequences                                digested fragment pMM223 cloned blunted fragment of <br /> offers possibility uncovering physiological functions                          NcoI-EcoRI digested vector, pSCR749(B), yielding plasmid pSM. The EcoRI- <br /> of unknown genes microorganisms. Recently, re-                                 BglII-digested tetO/tetR fragment finally cloned EcoRI-BglII-digested <br />                                                                                           vector pSM yield shuttle plasmid pSCR. <br /> ported utilization antisense RNA strategy identify <br />                                                                                              <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> clone fruA promoter pMM223 MCS, BamHI-PstI-digested <br /> common essential genes heterogeneous bacteria employ-                               fragment pKS::fruA containing fruA promoter fragment introduced <br /> ing E. coli expression clone DNA fragments S.                             BamHI-PstI sites pMM223. The resultant plasmid, pMfru, contains the <br /> mutans Escherichia coli (41). In present study,                               fruA promoter upstream aad gene (Fig. 1). To clone scrB promoter into <br /> constructed DNA libraries tetO/tetR regulatory expres-                             pMM223 MCS, PCR product scrB promoter region digested BglII <br />                                                                                           introduced EcoRI/BglII sites pMM223. The resultant plasmid pM- <br /> sion shuttle vector identify essential genes microor-                          scrB contains scrB promoter upstream aad gene (Fig. 1). <br /> ganism screening antisense transformants retarded                                To construct pTet::lacZ-expressing plasmid, lacZ reporter gene <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> growth. One previously uncharacterized genes codes                               pSBL20 (16) amplified PCR using pair primers (scrB-9, 5&#226;&#172;&#732;-GAAG <br /> for dephospho-coenzyme A kinase activity involved                             ATCTATTATTAGGAGTAGATAAACG-3&#226;&#172;&#732;, BglII site underlined; <br /> the final step coenzyme A catabolism appears                                 scrB-10, 5&#226;&#172;&#732;-CCAACAATAGGATAAAATTGTCC-3&#226;&#172;&#732;) introduced into <br />                                                                                           BglII-SmaI sites pTetE. The recombinant plasmid pTetELac then <br /> essential growth S. mutans.                                                    transformed S. mutans LT11. <br />                                                                                              Preparation screening genomic DNA libraries. S. mutans LT11 chro- <br />                              MATERIALS AND METHODS                                        mosomal DNA fragments (200 400 bp) partially digested Sau3AI were <br />    Bacterial strains plasmids. The S. mutans strains used study          cloned BglII site pTetR, transformed E. coli DH5&#226;?&#163;, grown <br /> LT11 (37), spontaneous mutant UA159 high transformation efficiency;             L-agar containing 150 &#226;?&#174;g/ml erythromycin 30&#194;&#176;C. The transformants were <br /> GS5 (13); S. mutans NG8&#226;&#338;&#172;comC (24); S. mutans LN62, gtfB mutant (33).                pooled inoculated LB containing 150 &#226;?&#174;g/ml erythromycin 30&#194;&#176;C. <br /> E. coli DH5&#226;?&#163; (supE44 hsdR17 recA1 endA1 gyrA96 thi-1 relA1) utilized             The plasmid pool extracted cells transformed S. <br /> general cloning host. S. mutans strains grown Todd-Hewitt broth (THB;             mutans LT11. The transformants plated TSA&#226;&#8364;&#8220;5 &#226;?&#174;g/ml erythromycin and <br /> Invitrogen, Carlsbad, CA) tryptic soy broth agar (TSA) plates (Difco, De-          grown 37&#194;&#176;C anaerobically. The colonies replica plated TSA&#226;&#8364;&#8220;5 <br /> troit, Mich.). E. coli strains cultured L-broth (Invitrogen), transformants   &#226;?&#174;g/ml erythromycin plates presence/absence 50 ng/ml doxycycline. The <br /> were selected L-agar plates supplemented indicated antibiotics.               transformants showed smaller colonies colonies doxycycline <br />    Plasmids used study pUC19 (43), pMM223 (40), pACL2073 (2),                plate compared plate antibiotic selected further <br /> pTS749 (35), derivative pSCR749 (unpublished data), pKS::fruA (4),                analysis. The growth-retarded <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> confirmed examining growth in <br /> pCR2.1 TOPO (Invitrogen).                                                                 broth cultures presence/absence doxycycline optical density 600 <br />    DNA manipulations sequence similarity analysis. DNA isolation, restric-            nm (OD600). <br /> tion endonuclease digestion, PCR, sequencing, Southern blotting, ligation, trans-            Construction scrA::yacE S. mutans LT11. By use pair primers (Sc-1, <br /> formation, DNA manipulations carried described previ-               5&#226;&#172;&#732;-CATTTGCAAAAAACTCC-3&#226;&#172;&#732;; Sc-2, 5&#226;&#172;&#732;-CTTGCCAATTCTTAATATC), an <br /> ously (42). Restriction endonucleases DNA-modifying enzymes                800-bp DNA fragment containing start codon promoter scrA part <br /> obtained Invitrogen, New England Biolabs Inc. (Beverly, MA), Pro-                scrB gene produced PCR. The yacE structural gene 5&#226;&#172;&#732; <br /> mega Corp. (Madison, WI) used according specifications                  terminus ATG start codon stop codon amplified PCR <br /> manufacturers. DNA fragments isolated agarose gels using                   E. coli, using following pair primers: yac-10 (5&#226;&#172;&#732;-GGAAGATCTCA <br /> QIAEXII kit (QIAGEN, Valencia, CA). Double-stranded PCR template se-                      TTACGGTTTTTCCTG-3&#226;&#172;&#732;, BglII site underlined) yac-11 (5&#226;&#172;&#732;-AG <br /> quencing performed using Sequenase 7-deaza-dGTP sequencing kit                 GTATATAGTTGCCTTAAC-3&#226;&#172;&#732;). The scrAB PCR fragment treated XbaI-T4 <br /> (USB, Cleveland, OH). Sequence similarity searches performed                    kinase ligated BglII-T4 kinase-treated yacE PCR fragment. The <br /> BLAST S. mutans strain UA159 genome sequence (found                    PCR product scrA::yacE amplified ligation reaction described <br /> University Oklahoma website [http://www.genome.ou.edu/smutans.html])               using primers yac-10 Sc-3 (5&#226;&#172;&#732;-TATCTAGACTTTGTGCTCCAACA <br /> Swiss-Prot databases.                                                                     ATAG-3, XbaII site underlined). The BglI-XbaI-digested scrA::yacE <br />    The general strategy gene inactivation S. mutans involved designing         fragment cloned BamHI-XbaI sites pUCSp, was <br /> pairs PCR primers corresponding upstream downstream fragments               derived pUC18 insertion aad gene pSCSP-1 (42) the <br /> (average size, 500 bp) target gene. The PCR products ligated             FspI sites inactivate &#226;?&#164;-lactamase gene. The recombinant plasmid pSY was <br /> with erythromycin resistance gene, erm (35). PCR amplification recombi-         transformed S. mutans LT11 select spectinomycin-resistant transfor- <br /> nant DNA fragment containing upstream fragment-erm-downstream fragment                mants containing chromosomal copy LT11 scrA::yacE (Fig. 2). <br /> was performed using pair primers corresponding 5&#226;&#172;&#732; end upstream            Construction, overexpression, purification SMU.1613c-histidine tag <br /> DNA fragment 3&#226;&#172;&#732; end downstream fragment. The resultant recombinant             fusion protein (SMU.1613c-H). A pair primers complementary the <br /> DNA fragment entire target gene replaced erm          SMU.1613c open reading frame start codon (coaE-1, 5&#226;&#172;&#732;-CCAAGC <br /> gene cloned TOPO PCR cloning plasmid (Invitrogen) screening              TTCATTCATTTTGCAACCTC-3&#226;&#172;&#732;; coaE-2, 5&#226;&#172;&#732;-CGGGATCCACAAAAATTA <br /> erythromycin-resistant (200-&#226;?&#174;g/ml) transformants. The resultant plasmid          TTGGGATTACAGGC-3&#226;&#172;&#732;) designed amplify SMU.1613c gene. The <br /> digested EcoRI transformed S. mutans LT11 (37). Following selection         PCR product digested BamHI-HindIII cloned BamHI- <br /> for erythromycin-resistant (5-&#226;?&#174;g/ml) transformants, target gene inactivated       HindIII sites pQE80L-1 (QIAGEN), contains six-His tag upstream <br /> by double-crossover recombination mechanism.                                            MCS. The recombinant plasmid pQC contained six-His SMU.1613c <br />    Construction regulatory systems shuttle plasmids pMM223 pTS749               control T5 promoter lac operator. The DNA sequence <br /> and lacZ reporter plasmid pTetELac. The tetO/tetR promoter DNA fragment                   SMU.1613c-H verified sequencing. <br /> was amplified PCR using pair primers corresponding flanking                   The E. coli transformant harboring pQC cultured 1.0 liter LB&#226;&#8364;&#8220;100 <br /> sequences tetO/tetR region pACL2073 additional EcoRI BglII             &#226;?&#174;g/ml ampicillin 30&#194;&#176;C optical density 0.5 0.6 600 nm then <br /> sites 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends, respectively (Ptet-5, 5&#226;&#172;&#732;-GGAATTCAAGCTTGCAT                   induced 0.1 mM isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG). After induction <br /> CCCTGCAG-3&#226;&#172;&#732;; Ptet-6, 5&#226;&#172;&#732;-GAAGATCTATTCGAGCTCGGTACCC-3&#226;&#172;&#732; [with                                30&#194;&#176;C 3 h aeration, bacteria harvested centrifugation at <br /> the EcoRI BglII sites, respectively, underlined). The EcoRI-BglII-digested            4,000 &#226;&#171;&#187; g 20 min resuspended 20 ml lysis buffer (50 mM NaH2PO4, <br /> tetO/tetR fragment cloned EcoRI-BglII-digested vector                   300 mM NaCl, pH 8.0). A soluble fraction obtained sonication centrifu- <br /> pMM223 transformed E. coli DH5&#226;?&#163;. &#226;&#338;&#187;he resultant plasmid pTetE                     gation applied Ni-nitrilotriacetic acid spin column (QIAGEN), and <br /> contained tetO controlling transcription downstream reporter genes, aad            SMU.1613c-H eluted according instructions supplier, slight <br /> and lacZ. Plasmid pTetR tetR gene located upstream                   modifications. Protein purity evaluated 12.5% sodium dodecyl sulfate- <br /> reporter genes constructed cloning EcoRI-PstI fragment con-             polyacrylamide gel electrophoresis (SDS-PAGE) Western blotting with <br /> taining tetO/tetR pACL2073 EcoRI-PstI sites pMM223. Plasmids               RGS.His antibody (QIAGEN). <br />  3570       WANG AND KURAMITSU                                                                                                                       INFECT. IMMUN. <br />  <br />  <br />  <br />  <br />                                                  FIG. 1. Construction inducible expression shuttle vectors. <br />  <br />  <br />    Measurement MICs. Each S. mutans strain grown overnight 37&#194;&#176;C anaer-        viously (25) following modifications. Each 1.0-ml assay mixture con- <br /> obically inoculated 105 CFU duplicate THB media containing serially     tained KCl (20 mM), MgCl2 (10 mM), ATP (10 mM), NADH (0.3 mM), phos- <br /> diluted concentrations spectinomycin growth measured presence         phoenolpyruvate (PEP) (0.4 mM), pyruvate kinase (10 U), lactic <br /> and absence doxycycline inducer. After strains grown overnight       dehydrogenase (4 U) 50 mM Tris buffer (pH 8.5). After addition the <br /> 37&#194;&#176;C anaerobically, cell density measured determining OD600. The     dephospho-coenzyme A kinase sample, reaction initiated addition <br /> MIC spectinomycin tested strain determined lowest            dephospho-coenzyme A (0.4 mM). The dephospho-coenzyme A kinase activ- <br /> concentration inhibiting bacterial growth.                                          ity determined measurement NADH oxidation OD340. <br />    Determination transformation efficiency S. mutans. Bacteria          Determination &#226;?&#164;-galactosidase activity. S. mutans LT11 strains harboring <br /> cultured THB&#226;&#8364;&#8220;10% horse serum overnight 37&#194;&#176;C inoculated (1/10)        pTetO pSMT grown THB containing 5 &#226;?&#174;g/ml erythromycin to <br /> into fresh THB&#226;&#8364;&#8220;10% horse serum presence/absence 50 ng/ml doxycy-          mid-log phase. Doxycycline added broth cells induced for <br /> cline. Bacteria incubated 2 h 10 &#226;?&#174;g/100 &#226;?&#174;l chromosomal DNA          2 h. The cells collected centrifugation assayed &#226;?&#164;-galacto- <br /> from GS5::&#226;&#338;&#172;gtfD mutant (Tetr) (15) added. The cells incubated       sidase activity described previously (16). <br /> 37&#194;&#176;C 2 h serially diluted TSA&#226;&#8364;&#8220;10 &#226;?&#174;g/ml tetracycline agar <br />                                                                                        Sucrose-mediated biofilm formation assay. Biofilm formation quantified <br /> plates. After incubation 37&#194;&#176;C anaerobically 48 h, transformation effi- <br />                                                                                     previously described (45). Flat-bottom polystyrene microtiter plates (96-well <br /> ciency determined number transformants/total cells transformed. <br />                                                                                     Easy Wash enzyme immunoassay-radioimmunoassay plates; Corning Inc., Corn- <br />    Dephospho-coenzyme A kinase activity assay. The enzymatic determination <br />                                                                                     ing, N.Y.) containing 100 &#226;?&#174;l THB&#226;&#8364;&#8220;0.5% sucrose inoculated with <br /> with myokinase (MK) (ATP:AMP phosphotransferase), pyruvate kinase (PK), <br />                                                                                     S. mutans (1.7 &#226;&#171;&#187; 105 CFU well) 24-h growth THB. After 24 h of <br /> and lactate dehydrogenase (LDH) quantitates decrease NADH, mea- <br />                                                                                     incubation 37&#194;&#176;C, 25 &#226;?&#174;l 1% (wt/vol) crystal violet solution added each <br /> sured change extinction 340 nm, follows: <br />                                                                                     well. After 15 min, wells rinsed times 200 &#226;?&#174;l distilled water <br />                                     CoaE                                            air dried. The crystal violet abiotic surfaces solubilized 95% <br />          Dephospho-coenzyme A &#226;&#171;&#185; ATP - <br />                                     |  0 CoA &#226;&#171;&#185; ADP                                  ethanol, optical density 570 nm measured. Growth deter- <br />                                                                                     mined measuring turbidities (optical densities 570 nm) parallel wells <br />                        PK                                                           following resuspension sessile organisms planktonic <br />          2 ADP &#226;&#171;&#185; 2 PEP | <br />                        -0 ATP &#226;&#171;&#185; 2 Pyruvate                                          cells. <br />                                                                                        Determination GTF activity. Relative amounts GTF protein deter- <br />                                                                                     mined measuring enzyme activity polyacrylamide gels previously <br />                                      LDH <br />                                                                                     described (38, 39). Briefly, strains grown mid- late-log stage <br />          2 Pyruvate &#226;&#171;&#185; 2 NADH &#226;&#171;&#185; 2 H&#226;&#171;&#185; - <br />                                     |  0 2 Lactate &#226;&#171;&#185; 2 NAD&#226;&#171;&#185; <br />                                                                                     determined cell density (OD600 THB medium) presence or <br />                                                                                     absence 50 ng/ml doxycycline. The cells collected washed with <br /> Dephospho-coenzyme A (CoA) kinase activity measured described pre-           phosphate-buffered saline buffer (pH 7.4). The cells resuspended in <br />  VOL. 73, 2005                                                                                INDUCIBLE ANTISENSE RNA EXPRESSION                  3571 <br />  <br />  <br />  <br />  <br />               FIG. 2. Construction LT11 scrA::yacE strain, expresses YacE E. coli chromosome S. mutans. <br />  <br />  <br /> phosphate-buffered saline buffer adjusted OD600 0.85          ml). Therefore, chose pTetEB examine promoter <br /> 0.875. Twenty microliters cell suspension run 8.75% SDS-PAGE         activities presence/absence inducer. Since MIC mea- <br /> gels. After electrophoresis, gels incubated overnight 37&#194;&#176;C 3% <br /> sucrose, 0.5% Triton X-100, 10 &#226;?&#174;g/ml dextran T10 10 mM sodium phos-          surement semiquantitative evaluation, pTet::lacZ re- <br /> phate, pH 6.8, 37&#194;&#176;C, resulting glucan bands treated periodic   porter plasmid, pTetElac, constructed resulting <br /> acid pararosanilin. The intensities stained bands reflect relative   strain examined ability express &#226;?&#164;-galactosidase <br /> amounts activities GTF proteins.                                         activity induced. Experiments examining kinetics of <br />                                                                                     induction inducer concentrations indicated steady- <br />                                  RESULTS                                            state lacZ gene expression reached maximum 3 h <br />                                                                                     induction 100 ng/ml doxycycline 50 100 <br />    Evaluation inducible expression systems S. mutans. In <br />                                                                                     ng/ml doxycycline promoted maximum induction sig- <br /> order maximize effects antisense RNA expression on <br /> the physiology S. mutans, necessary construct                    nificant inhibition growth. When fully induced, pTetE <br /> for organism shuttle plasmid allow                        expressed highest level &#226;?&#164;-galactosidase activity, <br /> for high-level induction cloned antisense fragments. A com-                      120 units, 12 times basal expression (11 units) (Fig. <br /> parison activities promoters scrB fruA                     3). Thus, pTetE selected construct in- <br /> genes S. mutans tetracycline-inducible                      ducible antisense RNA strains. <br /> promoter expression S. mutans performed using                            Modulation expression S. mutans genes induc- <br /> vector pMM223. This accomplished initially measuring                         ible antisense RNA production depends size, the <br /> spectinomycin MICs individual strains harboring                          relative position, antisense DNA fragment. To study the <br /> respective plasmids presence absence inducer.                         regulation physiological functions individual genes via <br /> pTetE expressed highest level aad gene product                        inducible antisense RNA production impact the <br /> three expression systems (the MIC strain 64 mg/ml                      position size antisense DNA fragments changes, <br /> in absence doxycycline 128 mg/ml presence                      initially, series comC DNA fragments PCR amplified <br /> doxycycline), 8 times higher pFru-controlled aad gene                          cloned KpnI-SmaI sites antisense orienta- <br /> expression (the MIC strain 16 mg/ml ab-                         tion downstream tetO promoter vector pTetE (Fig. 4). <br /> sence/presence 0.5% inulin) 32 times higher pScr                        The resultant plasmids introduced LT11, yielding <br /> controlled aad gene expression (the MIC strain 4                       comC antisense RNA strains. Transformation strains <br /> mg/ml absence/presence 0.5% sucrose), compared                         evaluated introduction chromosomal DNA an <br /> the strain harboring vector pMM223 (MIC, 500 &#226;?&#174;g/                           LT11::&#226;&#338;&#172;gtfD mutant. The results indicated location of <br />  3572     WANG AND KURAMITSU                                                                                                         INFECT. IMMUN. <br />  <br />  <br />                                                                                  TABLE 1. Transformation efficencies LT11 comC <br />                                                                                                 antisense strains <br />                                                                                                    Transformation efficiency (&#226;&#171;&#187;10&#226;&#171;&#186;10)a LT11 comC <br />                                                                        Strain                                       antisense strains <br />  <br />                                                                                                   Doxycycline (&#226;&#171;&#186;)                     Doxycycline (&#226;&#171;&#185;) <br />  <br />                                                                        Com-1                          347 &#226;&#171;&#190; 45                            19 &#226;&#171;&#190; 12 <br />                                                                        Com-2                          164 &#226;&#171;&#190; 58                            34 &#226;&#171;&#190; 23 <br />                                                                        Com-3                          493 &#226;&#171;&#190; 67                            33 &#226;&#171;&#190; 20 <br />                                                                        Com-4                          532 &#226;&#171;&#190; 138                           40 &#226;&#171;&#190; 23 <br />                                                                        Com-5                          255 &#226;&#171;&#190; 64                            46 &#226;&#171;&#190; 32 <br />                                                                        Com-6                          559 &#226;&#171;&#190; 147                          591 &#226;&#171;&#190; 132 <br />                                                                        pTetE                          653 &#226;&#171;&#190; 167                          612 &#226;&#171;&#190; 103 <br />                                                                        &#226;&#338;&#172;comC                           15 &#226;&#171;&#190; 15                            40 &#226;&#171;&#190; 21 <br />                                                                        LT11                         2,443 &#226;&#171;&#190; 782                        1,987 &#226;&#171;&#190; 819 <br />                                                                          a <br />                                                                              Values means &#226;&#171;&#190; standard deviations; (&#226;&#171;&#185;), present; (&#226;&#171;&#186;), absent. <br />  <br />  <br />  <br />  <br />                                                                        reported previously important sucrose-mediated bio- <br />                                                                        film formation strain GS5 (1). We constructed a <br />                                                                        series gtfB antisense RNA strains strategy used <br />                                                                        comC antisense strains use pSCR. A series gtfB <br />                                                                        DNA fragments PCR amplified cloned the <br />                                                                        KpnI-SmaI sites pSM. Next, tetO/tetR fragment cloned <br />   FIG. 3. LacZ activities LT11::pTetEBLac. LacZ activities (&#226;?&#164;-      EcoRI-BglII sites respective plasmids, resulting <br /> galactosidase activity, Miller units) measured wild-type    shuttle plasmids gtfB fragments antisense orien- <br /> LT11 grown presence 100 ng/ml doxycycline             tation downstream tetO promoter (Fig. 5C). The result- <br /> LT11::pTetElac grown absence/presence doxycycline. <br /> (A) Time course adding doxycycline. Triangle line, LT11::        ant plasmids transformed strain GS5, yielding a <br /> pTetElac presence 100 ng/ml doxycycline; square line,        series gtfB antisense RNA strains. Sucrose-dependent bio- <br /> LT11::pTetElac absence doxycycline; diamond line, LT11    film formation antisense strains compared that <br /> the presence 100 ng/ml doxycycline. (B) Dosage doxycycline   GS5 harboring pSCR only, gtfB null <br /> the growth LT11::pTetElac. The data averages three <br /> samples, standard errors shown. <br />                                                                        mutant LN62 (33) presence 50 ng/ml doxycycline (Fig. <br />                                                                        5A). After induction, gtfB antisense strains showed reduced <br />                                                                        biofilm formation compared GS5 GS5::pSCR ele- <br />                                                                        vated activity relative LN62. These results similar to <br /> antisense DNA fragments 3&#226;&#172;&#732; 5&#226;&#172;&#732; terminus of <br />                                                                        comC antisense strains, antisense fragments from <br /> comC resulted similar inhibitions transformation. The <br />                                                                        3&#226;&#172;&#732; 5&#226;&#172;&#732; termini gtfB produced similar inhibitions of <br /> minimal size antisense fragment achieve optimal <br />                                                                        biofilm formation (Fig. 5A B). The minimum size the <br /> inhibition estimated &#226;&#172;&#381;20 bp (Table 1). The supplemen- <br />                                                                        antisense fragment necessary significant inhibition was <br /> tation CSP restored transformation efficiency anti- <br />                                                                        &#226;&#172;&#381;100 bp. This size represents 4% intact gtfB gene. <br /> sense inhibited strains (data shown). This result confirmed <br />                                                                        Glucosyltransferase activity assays confirmed GTF activi- <br /> that inhibition transformation efficiency comC <br />                                                                        ties antisense strains lower GS5 <br /> antisense strains mediated reduced expression of <br />                                                                        harboring pSMT (Fig. 5D). GS5::gtf-3 60-bp anti- <br /> CSP presence doxycycline. <br />                                                                        sense gtfB fragment little impact GTF expression (Fig. <br />    Biofilm formation important physiological adaptation <br />                                                                        5D), minimal effects biofilm formation. <br /> of oral bacteria. The gtfB gene, approximately 4,500 bp, was <br />                                                                           Identification essential genes construction screen- <br />                                                                        ing inducible antisense RNA libraries S. mutans LT11. To <br />                                                                        construct stable antisense RNA libraries E. coli, intro- <br />                                                                        duced strain LT11 chromosomal DNA (200 400 bp) <br />                                                                        partially digested Sau3AI BglII site pTetR, <br />                                                                        downstream tetO promoter. The random genomic li- <br />                                                                        brary amplified passage E. coli provide <br />                                                                        sufficient amounts DNA plasmid pool then <br />                                                                        transformed S. mutans LT11. By screening growth- <br />                                                                        retarded transformants following replica plating TSA with/ <br />                                                                        50 ng/ml doxycycline (confirmed determination of <br />                                                                        OD600 broth cultures presence absence inducer, <br />    FIG. 4. Genetic arrangement comC gene wild-type (wt) S.   sequence analysis), total 78 growth-retarded <br /> mutans LT11 antisense strains. Ptet, promoter tetO; solid ar- <br /> rows, antisense comC fragments recombinant plasmids; open <br />                                                                        transformants harboring antisense DNA fragments pu- <br /> arrow, comC gene. The sizes antisense DNA fragments cloned      tative essential genes identified (Table 2) approxi- <br /> into vectors indicated right strain name.        mately 3,000 transformants. <br />  VOL. 73, 2005                                                                       INDUCIBLE ANTISENSE RNA EXPRESSION                      3573 <br />  <br />  <br />  <br />  <br />   FIG. 5. Modulation expression gtfB gene gtfB antisense strains. wt, wild type. (A) Bacterial growth biofilm formation S. <br /> mutans wild-type GS5, gtfB mutant LN62, GS5 harboring shuttle vector pSCR (pSCR), GS5 gtfB antisense strains (gtf-1, gtf-2, gtf-3, gtf-4) in <br /> 1/4 strength THB&#226;&#8364;&#8220;mucin&#226;&#8364;&#8220;0.5% sucrose doxycycline (50 ng/ml). Growth (white bars) biofilm formation (black bars) measured under <br /> anaerobic conditions. The data averages samples, standard errors shown. (B) Data experiment the <br /> absence doxycycline. (C) Genetic arrangement gtfB gene wild-type S. mutans GS5 antisense strains. Ptet, promoter tetO; solid <br /> arrows, antisense gtfB fragments recombinant plasmids; open arrow, gtfB gene. The sizes antisense DNA fragments cloned the <br /> vectors indicated right strain name. (D) GTF activity detected following SDS-PAGE analysis periodic acid-Schiff staining of <br /> the S. mutans strains mentioned grown presence (top panel) absence (bottom panel) 50 ng/ml doxycycline. <br />  <br />  <br />  <br />  <br />    Identification characterization essential gene en-              In order characterize essential nature the <br /> coding dephospho-coenzyme A kinase (coaE) S. mutans.                    SMU.1613c gene, gene complementation strategy car- <br /> Since growth-retarded transformants, W213, appar-               ried introducing E. coli yacE gene chromo- <br /> ently harbored novel gene unknown function time             strain LT11 downstream scrA promoter this <br /> this analysis downstream tetO promoter antisense             S. mutans strain. The resultant transformant, LT11 <br /> RNA orientation, subjected analysis. Follow-             scrA::yacE strain, subjected gene inactivation the <br /> ing sequencing insert fragment homology                   SMU.1613c gene. Erythromycin-spectinomycin-resistant trans- <br /> search S. mutans UA159 database, antisense DNA                  formants recovered compared control <br /> fragment corresponded conservative hypothetical protein,              LT11 wild-type strain. Truncation SMU.1613c double <br /> SMU.1613c, optimal Blastp hit ytaG gene                crossover transformants confirmed PCR (data <br /> Bacillus subtilis. It shares 33% identity yacE gene          shown). The successful inactivation SMU.1613c <br /> of E. coli. The gene product yacE E. coli recently          gene indicated expression yacE gene E. coli S. <br /> characterized dephospho-coenzyme A kinase, enzyme                    mutans complemented function SMU.1613c prod- <br /> involved critical step coenzyme A biosynthesis              uct. Therefore, SMU.1613c gene product function <br /> (27). Homologues yacE gene exist broad spectrum                similar yacE product, i.e., gene also <br /> of microorganisms, eukaryotic cells, indicating            expresses dephospho-coenzyme A kinase (CoaE) activity. <br /> this enzyme important cell metabolism. However, gene                  To confirm function SMU.1613c gene <br /> encoding dephospho-coenzyme A kinase gram-positive bac-                 product, histidine-tagged SMU.1613c fusion protein plasmid <br /> teria characterized. Therefore, gene es-              constructed using vector pQE80L-1. The resultant plas- <br /> sentiality SMU.1613c initially confirmed attempts             mid, pQC, transformed E. coli. The SMU.1613c-H <br /> gene inactivation. However, erythromycin-resistant trans-               expressed purified transformant. The E. coli <br /> formants recovered following transformation experi-                   CoaE-His tag fusion protein (CoaE-H) expressed and <br /> ments designed inactivate gene using transcrip-               purified E. coli strain harboring pESC124, histidine- <br /> tional terminator-negative erythromycin resistance gene                    tagged yacE fusion plasmid (31). Both proteins same <br /> replacing SMU.1613c gene. This result compat-               molecular mass, 22 kDa (data shown). The CoaE <br /> ible essential nature gene S. mutans.                   activity assay showed fusion proteins expressed sim- <br />  3574     WANG AND KURAMITSU                                                                                               INFECT. IMMUN. <br />  <br />                                TABLE 2. Essential genes S. mutans detected antisense RNA strategy <br />                                                                                                                No.          E. coli gene <br />            Gene class                     UA159 gene                     Protein encoded <br />                                                                                                                clones          homologue <br />  <br /> DNA replication,                           gyrA                       DNA gyrase A                               2               gyrA <br />  recombination, repair <br />  <br /> Translation, ribosomal structure,          rl24                       50S ribosomal protein L24                  8               rplX <br />   biogenesis <br />                                            rl19                       50S ribosomal protein L19                  4               rplS <br />                                            SMU.849                    50S ribosomal protein L27                  4               rpmA <br />                                            rl14                       50S ribosomal protein L14                  6               rplN <br />                                            rs5                        30S ribosomal protein S5                  20               rpsE <br />                                            rs18                       30S ribosomal protein S18                  6               rpsR <br />                                            r120                       50S ribosomal protein L27                  2               rplT <br />                                            rl13                       50S ribosomal protein L13                  2               rplM <br />                                            secY                       Preprotein translocase                     1               secY <br />  <br /> Transcription                              rRNA                       23S ribosomal RNA                         20 <br />                                            SMU.419                    Unknown                                    2 <br />  <br /> Cell envelope biogenesis                   Putative murG              Undecaprenyl-PP-MurNAc-                     1              murG <br />                                                                        pentapeptide-UDP-GlcNAc <br />                                                                        GlcNAc transferase <br />  <br /> Coenzyme metabolism                        SMU.1613c                  Hypothetical protein SMU.1613c              1              yacE <br />  <br />  <br />  <br /> ilar CoaE activities (Fig. 6A) compared negative control.          tion. The antisense DNA fragments 5&#226;&#172;&#732; end the <br /> In addition, ADP formation dose dependent                    target genes cloned downstream inducible promoter had <br /> SMU.1613c-H concentration (Fig. 6B). Thus, results                inhibitory effects 3&#226;&#172;&#732; fragments. <br /> confirm SMU.1613c encodes dephospho-coenzyme A ki-                 Compared location antisense DNA fragments, the <br /> nase activity S. mutans. This report iden-       size fragments appears important achieve max- <br /> tification characterization CoaE corresponding           imal antisense inhibition. A minimum &#226;&#172;&#381;20 bp required <br /> gene, coaE, gram-positive bacteria.                                  produce inhibition expression comC gene, <br />                                                                         &#226;&#172;&#381;5% <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> mRNA comC gene. <br />                            DISCUSSION                                   For gtfB, relatively large, minimum size 200 bp, <br />                                                                         4% length, required achieve <br />    The current availability gram-positive bacterium-induc-           significant antisense inhibition. Therefore, results sug- <br /> ible expression systems somewhat limited, especially re-          gest DNA fragment larger approximately 5% the <br /> gard regulated systems. There systems use with <br />                                                                         entire gene length likely required produce optimal inhi- <br /> gram-positive bacteria: Pspac (44), xylA (20), nisA (8, 21), <br />                                                                         bition gene expression. Such size limitations antisense <br /> MalR-regulated promoter (30), C1-regulated promoter (34), <br />                                                                         RNA effects reported bacteria. The <br /> and Ptet (36). Unfortunately, different species differ- <br />                                                                         resulting antisense RNA likely inhibits translation the <br /> ent responses systems (9). Most regulated <br />                                                                         target genes. <br /> by carbohydrate sources essential growth and <br />                                                                            By screening antisense library constructed the <br /> may subject carbon catabolite repression. There no <br />                                                                         tetO/tetR expression shuttle vector, categories essen- <br /> comparable inducible expression oral streptococci. <br /> We introduced tetO/tetR, scrB, fruA promoters          tial genes S. mutans identified. These essential genes <br /> streptococcal shuttle vector pMM223. The induction experi-              detected involved essential biological <br /> ments showed PtetO/tetR relatively strong inducible           processes DNA replication, recombination repair, <br /> promoter compared promoters S. mutans.              translation, ribosomal structure biogenesis, transcription, <br /> One useful feature inducer            cell envelope biogenesis, metabolism. Most of <br /> known interfere concentrations utilized            appear involved maintenance ribosomal <br /> aspects S. mutans metabolism.                                        structure transcriptional processes. The relatively low fre- <br />    One recent approaches studying gene function,              quency essential gene detection S. mutans system <br /> especially genes involved essential cell functions,     compared reported S. aureus (11, 19) to <br /> utilization antisense RNA manipulate expression               plasmid instability E. coli S. mutans antisense <br /> levels target genes (2, 18). It noticed      fragments introduced gram-negative host. For <br /> antisense fragments produced uniform inhibitory effects (18).           example, observed cloning S. mutans scrB <br /> In addition, unclear effects relative positions      promoter E. coli plasmids (42). In addition, stronger <br /> the sizes antisense RNA fragments target gene             induction promoter activity studies (11, 19) <br /> expression. The comC gtfB antisense strains constructed          allow convenient identification growth- <br /> this study produced different degrees inhibition cell func-       inhibited transformants inducing conditions. Neverthe- <br />  VOL. 73, 2005                                                                         INDUCIBLE ANTISENSE RNA EXPRESSION                                3575 <br />  <br />                                                                                                             REFERENCES <br />                                                                             1. Aoki, H., T. Shiroza, M. Hayakawa, S. Sato, H. K. Kuramitsu. 1986. <br />                                                                                Cloning Streptococcus mutans glucosyltransferase gene coding insol- <br />                                                                                uble glucan synthesis. Infect. Immun. 53:587&#226;&#8364;&#8220;594. <br />                                                                             2. Bateman, B. T., N. P. Donegan, T. M. Jarry, M. Palma, A. L. Cheung. <br />                                                                                2001. Evaluation tetracycline-inducible promoter Staphylococcus au- <br />                                                                                reus vitro vivo application demonstrating role sigB <br />                                                                                microcolony formation. Infect. Immun. 69:7851&#226;&#8364;&#8220;7857. <br />                                                                             3. Brosius, J., M. Erfle, J. Storella. 1985. Spacing &#226;&#171;&#186;10 &#226;&#171;&#186;35 <br />                                                                                regions tac promoter. Effect vivo activity. J. Biol. Chem. <br />                                                                                260:3539&#226;&#8364;&#8220;3541. <br />                                                                             4. Burne, R. A., Z. T. Wen, Y. Y. Chen, J. E. Penders. 1999. Regulation of <br />                                                                                expression fructan hydrolase gene Streptococcus mutans GS-5 by <br />                                                                                induction carbon catabolite repression. J. Bacteriol. 181:2863&#226;&#8364;&#8220;2871. <br />                                                                             5. Davies, L., N. A. Farrar, M. Satre, R. P. Dottin, J. D. Gross. 1996. <br />                                                                                Vacuolar H(&#226;&#171;&#185;)-ATPase weak <span id='am-82' about='xsp:base' typeof='owl:Thing'>base</span> action Dictyostelium. Mol. Micro- <br />                                                                                biol. 22:119&#226;&#8364;&#8220;126. <br />                                                                             6. De Backer, M. D., B. Nelissen, M. Logghe, J. Viaene, I. Loonen, S. Vandon- <br />                                                                                inck, R. Hoogt, S. Dewaele, F. A. Simons, P. Verhasselt, G. Vanhoof, R. <br />                                                                                Contreras, W. H. Luyten. 2001. An antisense-based functional genomics <br />                                                                                approach identification genes critical growth Candida albicans. <br />                                                                                Nat. Biotechnol. 19:235&#226;&#8364;&#8220;241. <br />                                                                             7. Boer, H. A., L. J. Comstock, M. Vasser. 1983. The tac promoter: a <br />                                                                                functional hybrid derived trp lac promoters. Proc. Natl. Acad. <br />                                                                                Sci. USA 80:21&#226;&#8364;&#8220;25. <br />                                                                             8. Ruyter, P. G., O. P. Kuipers, M. M. Beerthuyzen, I. van Alen-Boerrigter, <br />                                                                                W. M. Vos. 1996. Functional analysis promoters nisin gene <br />                                                                                cluster Lactococcus lactis. J. Bacteriol. 178:3434&#226;&#8364;&#8220;3439. <br />                                                                             9. Eichenbaum, Z., M. J. Federle, D. Marra, W. M. Vos, O. P. Kuipers, M. <br />                                                                                Kleerebezem, J. R. Scott. 1998. Use lactococcal nisA promoter to <br />                                                                                regulate gene expression gram-positive bacteria: comparison induction <br />                                                                                level promoter strength. Appl. Environ. Microbiol. 64:2763&#226;&#8364;&#8220;2769. <br />                                                                            10. Elvin, C. M., P. R. Thompson, M. E. Argall, P. Hendry, N. P. Stamford, P. E. <br />                                                                                Lilley, N. E. Dixon. 1990. Modified bacteriophage lambda promoter <br />                                                                                vectors overproduction proteins Escherichia coli. Gene 87:123&#226;&#8364;&#8220;126. <br />                                                                            11. Forsyth, R. A., R. J. Haselbeck, K. L. Ohlsen, R. T. Yamamoto, H. Xu, J. D. <br />                                                                                Trawick, D. Wall, L. Wang, V. Brown-Driver, J. M. Froelich, G. C. Kedar, <br />                                                                                P. King, M. McCarthy, C. Malone, B. Misiner, D. Robbins, Z. Tan, Z. Y. Zhu <br />                                                                                Zy, G. Carr, D. A. Mosca, C. Zamudio, J. G. Foulkes, J. W. Zyskind. <br />                                                                                2002. A genome-wide strategy identification essential genes in <br />                                                                                Staphylococcus aureus. Mol. Microbiol. 43:1387&#226;&#8364;&#8220;1400. <br />                                                                            12. Geissendorfer, M., W. Hillen. 1990. Regulated expression heterolo- <br />                                                                                gous genes Bacillus subtilis using Tn10 encoded tet regulatory ele- <br />                                                                                ments. Appl. Microbiol. Biotechnol. 33:657&#226;&#8364;&#8220;663. <br />                                                                            13. Gibbons, R. J., K. S. Berman, P. Knoettner, B. Kapsimalis. 1966. Dental <br />                                                                                caries alveolar bone loss gnotobiotic rats infected capsule form- <br />                                                                                ing streptococci human origin. Arch. Oral Biol. 11:549&#226;&#8364;&#8220;560. <br />                                                                            14. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu- <br />                                                                                lation, modulation, high-level expression vectors containing arab- <br />                                                                                inose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130. <br />                                                                            15. Hanada, N., H. K. Kuramitsu. 1989. Isolation characterization of <br />                                                                                Streptococcus mutans gtfD gene, coding primer-dependent soluble <br />   FIG. 6. Dephospho-coenzyme A kinase activity assays measured              glucan synthesis. Infect. Immun. 57:2079&#226;&#8364;&#8220;2085. <br /> ADP formation. (A) ADP formation catalyzed purified YacE-His            16. Hiratsuka, K., B. Wang, Y. Sato, H. Kuramitsu. 1998. Regulation of <br /> tag fusion protein E. coli purified fusion protein S. mutans       sucrose-6-phosphate hydrolase activity Streptococcus mutans: character- <br /> CoaE-H compared negative control adding CoaE.                   ization scrR gene. Infect. Immun. 66:3736&#226;&#8364;&#8220;3743. <br /> (B) Dosage effect S. mutans CoaE-H. The data averages        17. Ji, Y., A. Marra, M. Rosenberg, G. Woodnutt. 1999. Regulated antisense <br /> three samples, standard errors shown.                                  RNA eliminates alpha-toxin virulence Staphylococcus aureus infection. J. <br />                                                                                Bacteriol. 181:6585&#226;&#8364;&#8220;6590. <br />                                                                            18. Ji, Y., D. Yin, B. Fox, D. J. Holmes, D. Payne, M. Rosenberg. 2004. <br />                                                                                Validation antibacterial mechanism action using regulated antisense <br /> less, utilized present study did allow               RNA expression Staphylococcus aureus. FEMS Microbiol. Lett. 231:177&#226;&#8364;&#8220; <br /> identification number essential genes S. mutans                 184. <br />                                                                            19. Ji, Y., B. Zhang, S. F. Van Horn, P. Warren, G. Woodnutt, M. K. Burnham, <br /> the characterization novel essential gene, coaE,                  M. Rosenberg. 2001. Identification critical staphylococcal genes using <br /> organism. Furthermore, coaE gene, encoding dephospho-                      conditional phenotypes generated antisense RNA. Science 293:2266&#226;&#8364;&#8220; <br />                                                                                2269. <br /> coenzyme A kinase, identified characterized                  20. Kim, L., A. Mogk, W. Schumann. 1996. A xylose-inducible Bacillus <br /> essential gene S. mutans. Undoubtedly, screening                     subtilis integration vector application. Gene 181:71&#226;&#8364;&#8220;76. <br /> additional antisense DNA clone banks strategy                21. Kuipers, O. P., M. M. Beerthuyzen, P. G. Ruyter, E. J. Luesink, and <br />                                                                                W. M. Vos. 1995. Autoregulation nisin biosynthesis Lactococcus <br /> uncover additional novel essential genes                     lactis signal transduction. J. Biol. Chem. 270:27299&#226;&#8364;&#8220;27304. <br /> prove specific targets novel anticaries approaches.              22. Kuramitsu, H. K. 1993. Virulence factors mutans streptococci: role of <br />                                                                                molecular genetics. Crit. Rev. Oral Biol. Med. 4:159&#226;&#8364;&#8220;176. <br />                                                                            23. Li, Y., R. A. Burne. 2001. Regulation gtfBC ftf genes of <br />                       ACKNOWLEDGMENTS                                          Streptococcus mutans biofilms response pH carbohydrate. Mi- <br />                                                                                crobiology 147:2841&#226;&#8364;&#8220;2848. <br />   We thank B. Zaat, A. L. Cheung, R. A. Burne providing            24. Li, Y. H., P. C. Lau, J. H. Lee, R. P. Ellen, D. G. Cvitkovitch. 2001. <br /> some plasmids used study.                                       Natural genetic transformation Streptococcus mutans growing biofilms. <br />   This study supported National Institutes Health            J. Bacteriol. 183:897&#226;&#8364;&#8220;908. <br /> grant DE03258.                                                             25. Martin, D. P., D. G. Drueckhammer. 1993. Separate enzymes catalyze <br />  3576         WANG AND KURAMITSU                                                                                                                           INFECT. IMMUN. <br />  <br />       final steps coenzyme A biosynthesis Brevibacterium ammoni-                Integrational plasmids tetracycline-regulated expression genes in <br />       agenes: purification pantetheine phosphate adenylyltransferase. Biochem.             Streptococcus pneumoniae. Gene 226:243&#226;&#8364;&#8220;251. <br />       Biophys. Res. Commun. 192:1155&#226;&#8364;&#8220;1161.                                              37.   Tao, L., T. J. MacAlister, J. M. Tanzer. 1993. Transformation efficiency <br /> 26.   Mattos-Graner, R. O., M. H. Napimoga, K. Fukushima, M. J. Duncan,          <br /> </body></html>