<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>    Mapping and identification of essential gene <br />     functions on the X chromosome of Drosophila <br />  <br />     The Drosophila melanogaster genome consists of four                               large-scale P element insertion screen for vital gene functions <br />     chromosomes that contain 165 Mb of DNA, 120 Mb of which                           and a BAC tiling map for the X chromosome. A collection of <br />     are euchromatic. The two Drosophila Genome Projects, in                           501 X-chromosomal P element insertion lines was used <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> map <br />     collaboration with Celera Genomics Systems, have sequenced                        essential genes cytogenetically and to establish short sequence <br />     the genome, complementing the previously established                              tags (STSs) linking the insertion sites to the genome. The <br />     physical and genetic maps. In addition, the Berkeley                              distribution of the P element integration sites, the identified <br />     Drosophila Genome Project has undertaken large-scale                              genes and transcription units as well as the expression <br />     functional analysis based on mutagenesis by transposable                          patterns of the P-element-tagged enhancers is described and <br />     P element insertions into autosomes. Here, we present a                           discussed. <br />  <br />                    INTRODUCTION <br /> Drosophila melanogaster has been a model organism for <br /> genome research for almost 90 years, starting with Sturtevant&#226;&#8364;&#8482;s <br /> demonstration that recombination frequencies could be used to          Fig. 1. Diagram showing the contig distribution along the X chromosome. <br /> map genes in a linear order along the chromosome (Sturtevant,          Contigs are indicated by the boxes above the schematic representation of the <br /> 1913). Physical maps that relate genes to physical sites on the        X chromosome. Contig 1 is composed of cosmids and BACs. <br /> chromosomes (Bridges, 1935), together with the precise <br /> mapping of cloned DNA fragments by in situ hybridization to <br /> polytene chromosomes, enabled the first chromosomal maps of <br /> cloned genomic DNA segments (Wensink et al., 1974). These <br /> achievements, the development of procedures for molecular              <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> the termini of cosmid clone inserts. These sequence tags <br /> screening (Grunstein and Hogness, 1975) and the assembly of            correspond to cosmid clones of known cytological location, <br /> large chromosomal contigs (Bender et al., 1983) have facilitated       with apparently single-copy inserts that are members of contigs <br /> systematic attempts to construct P1-based physical maps of the         (Madueno et al., 1995). Some additional tags were determined <br /> Drosophila genome. These maps cover &#226;&#710;&#188;90% of the euchrom-               from isolated cosmid clones of known cytological location. <br /> atic genome and are punctuated by short sequence tag (STS)             Additional probes were derived from sequences reported in <br /> sites that are, on average, 50 kb apart (Kimmerly et al., 1996).       databases, generally corresponding to genes of known cyto- <br /> More recently, a whole genome shotgun sequencing project, a            logical map position and from sequences of P element <br /> collaborative effort between Celera Genomics and the Berkeley          insertions, again of known cytological position. Probes used <br /> Drosophila Genome Project (BDGP), has provided a 95%                   were selected partly on the basis of cytological position, in order <br /> coverage of the Drosophila genome (Adams et al., 2000; Rubin           to enhance the distribution of probe sequences along the <br /> et al., 2000b). In addition, expressed sequence tag (EST) projects     chromosome. <br /> have generated sequence information from &gt;100 000 transcripts <br /> (Rubin et al., 2000a).                                                                Example for inter-contig gaps <br />    To assign function to the DNA sequences, the BDGP has <br />                                                                        Two probes (ES00262, mapping to 7B1-2, and ES00263, <br /> undertaken large-scale functional analysis of the genome <br />                                                                        mapping to polytene chromosome bands 7B1-6) fail to hybridize <br /> through gene-disruption projects that are based on mutagenesis <br />                                                                        to BAC clones from either library. Both probes are derived from <br /> by transposable P element insertion. These projects have <br />                                                                        mapped P1 clones. No BAC clones hybridizing to probes on <br /> focused on the isolation of essential genes on autosomes. The <br />                                                                        either side of ES00262 and ES00263 were observed. The <br /> current collection of strains disrupt at least 1200 different genes, <br />                                                                        explanation may be that the sequences from which these probes <br /> representing &#226;&#710;&#188;30% of the estimated 3600 genes on the auto- <br />                                                                        were derived are incorrect or that there may be polymorphism <br /> somes that are mutable to easily scorable phenotypes, mainly <br />                                                                        between the Drosophila strains used in the construction of the <br /> sterility or lethality (Spradling et al., 1995, 1999). Here, we <br />                                                                        BAC and P1 libraries. This gap represents a separation between <br /> report a large-scale P element insertion screen to identify essen- <br />                                                                        contig 1 (1A to 7A) and contig 2 (7B to 10A). Other gaps gener- <br /> tial genes on the X chromosome, as well as the construction of <br />                                                                        ally correspond to intensely staining polytene chromosome <br /> an X-chromosomal BAC tiling map, which complements other <br />                                                                        bands (e.g. those in 10B, 11CD, 13A and 17A) and presumably <br /> ongoing studies of the Drosophila genome. <br />                                                                        represent long regions with few probes. In a number of cases, <br />                                                                        clones repeatedly failed to give signals with probes to which, on <br />          RESULTS AND DISCUSSION                                        the basis of cytological location and other hybridization data, <br />                                                                        they might be expected to hybridize. Possible explanations <br />            Tiling map of the X chromosome                              include clones being absent on certain filters, cloned DNA <br />                                                                        segments bearing segmental deletion, and other failures in <br /> A physical map of the X chromosome was constructed by                  hybridization. <br /> hybridizing 521 probes to high-density filters bearing both the <br /> EDGP and BDGP BAC libraries. 5771 BAC clones were identi-                          Generation of P element insertions <br /> fied by hybridization. Eleven contigs were assembled using a <br /> combination of manual analysis of hybridization data and in situ       The euchromatin of the Drosophila X chromosome comprises <br /> hybridization. These span the X chromosome from the subtelo-           &#226;&#710;&#188;22 Mb of DNA, containing 2182 predicted protein coding <br /> meric region of 1A to 20C near the <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span> of the chromosome arm          genes (Adams et al., 2000). As an estimated two-thirds of all <br /> (Figure 1; supplementary information available on the website          Drosophila genes show no obvious loss-of-function <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> http://www2.open.ac.uk/biology/research/molecular-genetics/            (Miklos and Rubin, 1996), &#226;&#710;&#188;800 of the X-chromosomal genes are <br /> EDGPmap.html). The longest contig, contig 1, spans divisions 1&#226;&#8364;&#8220;6       mutable to a scorable phenotype such as sterility or lethality. In <br /> and comprises both cosmid clones (Siden-Kiamos et al., 1990)           order to assess such genes in a systematic manner, we generated <br /> and BAC clones. Divisions 1&#226;&#8364;&#8220;3 were mapped to yield template            a collection of lethal P element insertions on the X chromosome. <br /> clones for sequencing (Benos et al., 2000, 2001), and in this            The crossing scheme for generating X-chromosomal P{lacW} <br /> region the BAC libraries were screened principally to link             insertion lines is depicted in Figure 2. Females with a reinsertion <br /> cosmid contigs.                                                        of the P{lacW} element, as shown by the absence of the <br />    Probe sequences were selected from several sources. The             dominant second chromosome marker Curly and the white+ eye <br /> EDGP cosmid mapping project determined many sequence tags              colour from P{lacW}, were individually mated to FM7c males. <br />  <br />  <br />                                                                                                             EMBO reports vol. 3 | no. 1 | 2002 35 <br />                                                                    scientific report <br />     A. Peter et al. <br />  <br />  <br />  <br />  <br />                                                                                        Fig. 3. P{lacW} distribution along the X chromosome. The localization is <br />                                                                                        based on the molecularly defined integration site. Open bars represent all <br />     Fig. 2. Crossing scheme for isolating lethal X-chromosomal insertion lines.        insertions in a given polytene region whereas shaded bars indicate the number <br />     The sex chromosomes (left pair in each genotype) and the two large                 of genes affected. <br />     autosomes are shown schematically with females on the left. Relevant <br />     mutations are labelled accordingly. The P{lacW} insertion is indicated by the <br />     triangle and its white+ selectable marker is represented by the darker <br />     chromosome. The transposase source is indicated by the double-headed <br />     arrow. Balancer chromosomes are hatched. Absence of the crossed-out males <br />     in the F3 generation indicates a potential lethal P{lacW} insertion. For further   that contain a second integration site and two containing three <br />     details, see Methods. <br />                                                                                        integrations where only one in situ hybridization signal was <br />                                                                                        observed. In eight such cases, the insertion sites are &lt;50 kb <br />                                                                                        apart, and thus likely to be below the limit of resolution of the in <br />                                                                                        situ hybridization technique. In four of these closely linked <br />     57 105 such crosses were initially set and 39 900 (69.9%)                          strains, the same transcription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> was hit by two P elements, <br />     produced offspring. From these crosses, 501 strains (1.3%) were                    indicative of local hopping during the remobilization of the <br />     derived that were either hemizygously lethal or semilethal                         starting P{lacW}. Collectively, these results indicate that &gt;90% <br />     (defined as &lt;20% viability compared to the sibling balancer                        of the lines contain single P element insertions. Hence, this <br />     males). Taking into account that the X chromosome represents                       collection of P element insertion strains is comparable with the <br />     roughly one-seventh of the genomic target size, 1.3% corres-                       best P element collections generated to date (Spradling et al., <br />     ponds to a frequency of &#226;&#710;&#188;9% lethal P element insertions, a rate                    1999). <br />     similar to those reported for autosomal insertions with comparable                    P{lacW} insertion and an associated <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> can be <br />     P element vector (Cooley et al., 1988; Bellen, 1999). In addition,                 causally linked if excision of the P{lacW} restores wild type <br />     73 lines were isolated that carried a visible <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> or that were                function. We tested a total of 206 lines by combining a trans- <br />     sterile in one sex.                                                                posase source P{ry+ &#226;&#710;&#8224;2-3}(99B) with the the P{lacW} element <br />                                                                                        (Bellen et al., 1989). 134 strains (65.0%) produced fully viable <br />                 Distribution of P element insertions                                   male progeny that lost the white+ gene, suggesting that the P <br />                                                                                        element excised precisely. The 65% value should be regarded as <br />     The insertion sites of 401 strains were determined by in situ                      a minimum, since the reversion tests were performed with only <br />     hybridization on polytene chromosome squashes. In addition,                        10 females bearing both the P{lacW} insertion and the trans- <br />     DNA from the insertion sites was isolated either by plasmid                        posase source. Some P elements are difficult to excise and hence <br />     rescue or by inverse PCR to generate an STS for all the insertion                  exhibit low mobilization frequencies and will almost certainly <br />     lines. Figure 3 summarizes the distribution of the P integrations                  have been missed in this type of protocol. <br />     along the X chromosome, showing a higher frequency of <br />     P{lacW} insertions in regions closer to the telomere. An observation                                Characterization of the lines <br />     that remains valid even when considering only non-redundant <br />     insertions. This skewed distribution is not mirrored by the relatively             As the &#206;&#178;-galactosidase gene in P{lacw} can report the expression <br />     even distribution of transcription units along the X chromosome.                   <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of adjacent genes, we immunohistochemically stained <br />     Hence, it is likely that an inherent insertion site preference of                  embryos with anti-&#206;&#178;-gal antibody. 398 of the 501 strains (79.4%) <br />     P elements (Liao et al., 2000) is responsible for the observed                     expressed the enzyme. This is considerably higher than the 64% <br />     distribution.                                                                      reported after random autosomal insertion of the same P element <br />        The quality of a P element collection depends largely on the                    (Bier et al., 1989) and may result from the fact that the X-chromo- <br />     proportion of strains that harbour single P{lacW} insertions. In                   somal lines are selected for insertions affecting essential genes. <br />     situ hybridizations to polytene chromosomes were carried out                       Representative patterns for each line were deposited in the <br />     for 401 strains. 89.5% (359 lines) contain single insertions, 9.7%                 FlyView database, http://flyview.uni-muenster.de/ (Janning, <br />     (39 lines) contain two insertions and 0.7% (three lines) contain                   1997), where they can be searched by various criteria. <br />     three P{lacW} insertions. However, for those 14 strains that                          The lethal phase was determined in 497 of the 501 lines. The <br />     showed a second hybridization signal in polytene region 3A, we                     most frequent stage where development stops is the larval stage, <br />     were unable to confirm the second insertion site by molecular                      with 29.2% of the analyzed lines (see Supplementary data). This <br />     means. Conversely, molecular analysis identified eight strains                     is not unexpected, since homozygous mutants can often survive <br />  <br />  <br /> 36 EMBO reports vol. 3 | no. 1 | 2002 <br />                                                      scientific report <br />                                                                                         Gene functions on Drosophila X chromosome <br />  <br /> up to this stage by relying on maternal contribution of gene           (Spradling et al., 1999). The 60 so-called warm spots (2&#226;&#8364;&#8220;5 <br /> products deposited by the heterozygous mother. In addition, the        insertions per gene; Spradling et al., 1999) correspond to 169 X- <br /> majority of the lines probably do not carry a <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null allele</span>, but         chromosomal insertions (34.5%), similar to what was observed <br /> rather hypomorphic mutations.                                          for the second and third chromosomes (35.8 and 34.2%, respec- <br />    We have generated an STS flanking the P element insertion           tively). There are 23 hot spots on the X chromosome that are hit <br /> site for 496 of the 501 lines by either plasmid rescue or by           at least six times, resulting in a total of 225 insertions (45.9%). <br /> inverse PCR (the sequences were submitted to the EMBL nucleo-          The frequencies for the second and third chromosome hot spots <br /> tide database; see Supplementary data). The integration sites          are 39.1 and 38.1%, respectively. The six genes that were hit <br /> were determined by BLAST search to the published Drosophila            most frequently are: Notch, 22 insertions; Trf2, 17; inx2, 15; ras, <br /> genomic sequence (Adams et al., 2000). In cases where the gene         14; and act5C, ctp and trol 12 each. In total, these six hot spots <br /> annotation is supported by EST data, we can unambiguously              contained 104 insertions, more than all the transcription units that <br /> identify the affected gene. For predicted genes whose open             were hit once. In summary, in this X-chromosomal collection, single <br /> reading frame is only annotated we assumed that the insertion is       hits are under-represented and hot spots are over-represented, <br /> integrated in the vicinity of a putative transcription start site      when compared to the autosomal P insertion collections. Taken <br /> upstream of the nearest translational start codon, since there is      together, of the X-chromosomal genes mutated in our screen <br /> a strong preference for P integrations in the 5&#226;&#8364;&#178;-end of genes          each is hit, on average, 2.7 times, whereas the figures for the <br /> (Spradling et al., 1995).                                              autosomal collection are 2.3 and 2.2, respectively. <br />    Since the value of this collection is, in part ,determined by the <br /> number of new genes that are affected, we will briefly discuss <br /> our mapping data. A total of 513 STS sequences were generated <br />                                                                                            CONCLUSIONS <br /> from 496 strains. In five strains, repeated attempts to isolate        Saturation mutagenesis studies give a ratio of lethal complementa- <br /> flanking sequences failed, and in 16 the identification of the         tion groups to polytene chromosomes bands of 0.81 (Ashburner <br /> integration site was uninformative, since the P element is inserted    et al., 1999). This estimate predicts &#226;&#710;&#188;820 essential genes for the <br /> in repetitive DNA. In 11 of these lines, the insertion is within a     X chromosome (1012 bands). Hence, our collection represents <br /> yoyo retrotransposon (Whalen and Grigliatti, 1998). The other          mutations in slightly &lt;25% of the X-chromosomal essential <br /> 497 STSs were generated from the remaining 480 lines and are           genes and is similar to the data obtained for the autosomal <br /> derived from unique X-chromosomal sequences. Five insertions           collection of lethal P insertions (Spradling et al., 1999). The <br /> were &gt;7 kb away from any annotated gene and it is unclear              P element screen for X-chromosomal genes required for adult <br /> whether the P integration is responsible for the observed lethal       viability and the BAC tiling map reported here represent a valu- <br /> phenotype. Three additional P insertions occurred in the vicinity      able tool for the Drosophila research community. The gene <br /> of known EP insertions but were some distance from the next            disruption collection provides the first opportunity to link &#226;&#710;&#188;130 <br /> annotated gene, suggesting that the the P elements have inserted       X-chromosomal genes with a phenotype. The majority of the <br /> into a putative promotor region of an unknown gene. Fourteen           lines are available from the Bloomington stock center, together <br /> P elements, representing five gene pairs, may affect two genes.        with their in situ hybridization data, and the enhancer trap <br /> In each of these cases, one gene is localized within a large intron    expression patterns of these strains have been deposited in the <br /> of another gene.                                                       FlyView database. Using these resources, the Drosophila <br />    183 P{lacW} insertions affect 52 different genes that were          research community can utilize the collection for the functional <br /> previously characterized at both the molecular and genetic             and molecular characterization of affected genes. The BAC tiling <br /> level, e.g. Notch, pebbled and short gastrulation. Phenotypically      map can, on the other hand, serve as a reference point for <br /> well characterized mutations of previously unidentified genes          mapping sequences and, more importantly, it will be essential as <br /> form a second group, consisting of two loci. These are trol,           a source for a particular genomic fragment, e.g. for germ line <br /> formerly known as zw1, with 12 insertions and stardust with one        transformation. The combined efforts of the community will not <br /> insertion. The largest group of insertions disrupts genes for          only increase our knowledge of the model organism Drosophila <br /> which no mutation has previously been reported. In most cases,         but will also, by virtue of the evolutionary conservation of many <br /> some molecular information is available or the genes have been         genes and processes, shed light on human gene function. <br /> predicted by computer algorithms (Adams et al., 2000). In <br /> summary, 301 insertions affect 130 genes (including the five <br /> nested gene pairs) that remain to be characterized. Among this <br />                                                                                                METHODS <br /> class are transcription units that <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> for proteins putatively        Drosophila strains and embryos. The following fly strains <br /> involved in signaling cascades, e.g. Rala, protein degradation, or     (Lindsley and Zimm, 1992; FlyBase, 1999) were used in the <br /> Rpt4, intracellular transport, or Ntf-2, as well as many other         experiments. The starting element, P{lacW} (Bier et al., 1989) <br /> cellular and developmental processes.                                  originated from two male sterile insertion strains, 1040B and <br />    To determine the frequency by which genes were hit, we              1260A, which each contain a single transposon on chromosome 2. <br /> relied on molecular information. Indeed, allelism cannot easily        The strain used to supply transposase activity was w; CyO/wgSp; <br /> be determined by genetic criteria, as complementation tests            TM6/Sb P{ry+ &#226;&#710;&#8224;2-3}(99B) (Robertson et al., 1988). The female <br /> between X chromsome lethals are not possible without first             fertile FM6 and the female sterile FM7c chromosomes were <br /> introducing a duplication that rescues the lethality. Genes on the     employed as X chromosome balancers. In addition to being <br /> X chromosome that are hit only once represent 96 out of 490            fertile in both sexes, FM6 has the advantage of carrying a white <br /> insertions (19.6%). This compares to a rate of 25.8 and 28.8%          null allele, thereby facilitating the detection of low expression of <br /> in the second and third chromosome collection, respectively            the white+ marker gene. <br />  <br />  <br />  <br />     Crossing scheme (see also Figure 2)                                           <br />     P: w/w; P{lacW}/P{lacW} &#195;&#8212; w/Y; wgSp/CyO; ry Sb P{ry+ &#226;&#710;&#8224;2-3}(99B)/              <br />     TM6B                                                                      <br />     F1: FM6/FM6 &#195;&#8212; w/Y; P{lacW}/CyO; ry Sb P{ry+ &#226;&#710;&#8224;2-3}(99B)/+                 <br />     F2: w/FM6; CyO/+ (with P{lacW} somewhere) &#195;&#8212; FM7c/Y                       <br />     The absence of non-balancer bearing sons in the F3 generation              <br />     indicates a potential lethal insertion in the X chromosome. <br />                                                                   <br />     Immunohistochemistry. Enhancer trap expression pattern was <br />                                                                                     <br />     analyzed by immunohistochemistry. Embryos were collected on <br />                                                                                   <br />     apple juice agar plates, fixed and stained with anti-&#206;&#178;-galactosidase              <br />     antibody (Cappel). The secondary peroxidase-coupled antibody                <br />     was a biotinylated anti-rabbit Ig (Organon Teknika).                             <br />     Molecular analysis. DNA was isolated from adult flies with                   <br />     QIAGEN DNeasy according to the instructions of the manufac-                    <br />     turer. For isolation of flanking sequences by plasmid rescue                     <br />     (Pirrotta, 1987), EcoRI-digested DNA was used, resulting in                <br />     DNA from the 3&#226;&#8364;&#178;-end of P{lacW}. Inverse PCR (Silver, 1991) was                   <br />     performed essentially as described in the BDGP webpage (http://          <br />     www.fruitfly.org/). The construction of the BAC tiling map and           <br />     their application is described as supplementary information on the         <br />     website ftp://ftp.ebi.ac.uk/pub/databases/edgp/200111/bac_probes.txt;           <br />     the P insertion data are stored in the same directory under ftp://           <br />      <br /> </body></html>