<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, June 2005, p. 5292&#226;&#8364;&#8220;5305                                                                                    Vol. 25, No. 12 <br /> 0270-7306/05/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.25.12.5292&#226;&#8364;&#8220;5305.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />                          Involvement Human MOF ATM Function <br />     Arun Gupta, Girdhar G. Sharma,1 Charles S. H. Young,2 Manjula Agarwal,1 Edwin R. Smith,3&#226;&#8364;&#160; <br />                      1 <br />  <br />     Tanya T. Paull,4 John C. Lucchesi,3 Kum Kum Khanna,5 Thomas Ludwig,2 Tej K. Pandita1* <br />        Department Radiation Oncology, Washington University School Medicine, St. Louis, Missouri 631081; College of <br />            Physicians Surgeons, Columbia University, New York, New York 100322; Department Biology, Emory <br />             University, Atlanta, Georgia 303223; University Texas Austin, Austin, Texas 787124; Queensland <br />                                Institute Medical Research, Brisbane, Queensland 4029, Australia5 <br />                      Received 22 October 2004/Returned modification 13 December 2004/Accepted 21 March 2005 <br />  <br />             We determined hMOF, human ortholog Drosophila MOF gene (males absent first), <br />          encoding protein histone acetyltransferase activity, interacts ATM (ataxia-telangiectasia- <br />          mutated) protein. Cellular exposure ionizing radiation (IR) enhances hMOF-dependent acetylation its <br />          target substrate, lysine 16 (K16) histone H4 independently ATM function. Blocking IR-induced <br />          increase acetylation histone H4 K16, expression dominant negative mutant &#226;&#338;&#172;hMOF <br />          <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span>-mediated hMOF knockdown, resulted decreased ATM autophosphorylation, ATM <br />          kinase activity, phosphorylation downstream effectors ATM DNA repair increasing cell <br />          killing. In addition, decreased hMOF activity associated loss cell cycle checkpoint response <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />          DNA double-strand breaks. The overexpression wild-type hMOF yielded opposite results, i.e., modest <br />          increase cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> enhanced DNA repair IR exposure. These results suggest hMOF <br />          influences function ATM. <br />  <br />  <br />    In eukaryotic cells, DNA damage activates signal transduc-               (4) suggested chromatin structural perturbations in- <br /> tion pathways rapidly affect downstream processes              duced DNA double-strand breaks (DSBs) serve trigger <br /> gene transcription, cell cycle progression, DNA replication             ATM activation. Recent studies indicate MRN <br /> (13, 25). All processes involve alterations chromatin           (Mre11, Rad50, Nbs1) complex TRF2 either <br /> structure. Posttranslational covalent modifications histones             influences activation ATM (9, 29, 52) serves modu- <br /> have emerged key regulatory events DNA damage re-                     lator/amplifier ATM activity (17, 23). Our studies DNA <br /> sponse. A widespread modification acetylation catalyzed               chromosome damage treatment IR suggested <br /> histone acetyltransferases reversed deacetylases (3, 13,             cells deficient ATM function efficient block- <br /> 50). Reversible acetylation lysines (K) positions 5, 8,          ing conversion DNA damage chromosome damage <br /> 12, 16 amino-terminal tail histone H4 occurs               (33, 38, 40), possibly indicating inherent chromatin alter- <br /> vivo eukaryotes (3). The hyperacetylation histone H4              ation (16, 39). ATM reported regulate function <br /> could lead unfolding nucleosomal fiber (50),              histone-modifying factors (22, 27). In report, we <br /> the acetylation histone H4 K16 occurs hyperactive              identified hMOF ATM-interacting protein have <br /> male X chromosome Drosophila polytene chromosomes                        provided evidence functional hMOF participates the <br /> (51). Ikura et al. (19) noted Tip60 (Tat-interacting pro-              activation ATM response DNA damage. hMOF the <br /> tein), acetylates histones H2A, H3, H4, plays role              human ortholog Drosophila MOF, histone acetyltrans- <br /> in DNA repair. More recently, Kusch et al. (28) demonstrated                ferase modifies X-chromosome chromatin males in <br /> that Drosophila Tip60 acetylates nucleosomal phospho-                   order achieve dosage compensation (15). <br /> H2Av exchanges unmodified H2Av. Bird et al. <br /> (5) reported acetylation histone H4 Esa1 (es- <br /> sential SAS2-related acetyltransferase) required DNA                                         MATERIALS AND METHODS <br /> repair yeast suggested similar modification                  Yeast two-hybrid screen. The procedure construction hMOF its <br /> function mammalian cells.                                                fragments used ATM TRF1 described <br />    ATM (ataxia-telangiectasia-mutated protein) crucial               previously (30, 33, 45, 48). The yeast two-hybrid screen performed the <br />                                                                             transformation yeast strain Y153 containing plasmid pHM677 (Y153/ <br /> the initiation signaling pathways mammalian cells follow-             pHM677) cDNA library human embryonic cells (Clontech). Primary <br /> ing exposure ionizing radiation (IR) DNA-dam-                  transformants analyzed interaction ATM selection histidine <br /> aging agents (36, 46), cells deficient ATM function             prototrophy dropout plates supplemented 20 mM 3-aminotriazole and <br /> have defective telomere chromatin (47). Bakkenist Kastan                expression &#226;?&#164;-galactosidase (&#226;?&#164;-gal) determined filter lift tests. <br />                                                                             Plasmids encoding putative binding partners ATM isolated <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             double-positive yeast clones retransformed Y153/pHM677 order to <br />                                                                             confirm interactions. Positive clones retransformation subjected to <br />   * Corresponding author. Mailing address: Department Radiation          automated sequencing homology searches using BLAST (tBLASTn) <br /> Oncology, Washington University School Medicine, 4511 Forest             search GenBank human database. Several putative cellular binding part- <br /> Park, St. Louis, MO 63108. Phone: (314) 747-5461. Fax: (314) 362-           ners ATM identified. Most isolated copies. To <br /> 9790. E-mail: pandita@wustl.edu.                                            quantify interaction ATM hMOF, interactions hMOF <br />   &#226;&#8364;&#160; Present address: Laboratory Chromatin Biology, Rockefeller           (wild type mutated) ATM fragments liquid &#226;?&#164;-gal assayed (Fig. <br /> University, 1230 York Avenue, New York, NY 10021.                           1) according previously described procedure (12, 30). <br />  <br />                                                                      5292 <br />  VOL. 25, 2005                                                                                         MOF INFLUENCES ATM FUNCTION                        5293 <br />  <br />  <br />  <br />  <br />   FIG. 1. Yeast two-hybrid-binding assays. (A) hMOF cDNA fragments generated PCR. (B) The Gal4 DNA-binding domain (DBD) was <br /> fused bait (hMOF, hMOF fragments, ATM fragments), activation domain (AD) Gal4 fused prey (ATM fragments, hMOF, <br /> or hMOF fragments). DNA-binding domain 4, lacks amino acids 1217 1239, generated described previously (10). (C) Plasmids in <br /> pairs introduced Saccharomyces cerevisiae HF7c cells, &#226;?&#164;-gal activity quantified using O-nitrophenyl &#226;?&#164;-D-galactoside assay. <br /> (D) Schematic representation domains ATM protein. NLS, nuclear localization signal. <br />  <br />  <br />  <br />    Cell culture derivation cell lines. AT221JE-TpEBS7 (A-T [ataxia-telan-   appropriate primer pair combinations. ATM&#226;&#338;&#172;LZ generated described <br /> giectasia] fibroblast vector only), AT221JE-TpEBS7-YZ5 (A-T fibro-      previously (10). The wild mutant forms tagged GFP, HA, Flag were <br /> blast complemented ATM cDNA), GM5849, 293 cells maintained           cloned pIND(SP1)/Neo vector. The final constructs verified by <br /> and transfected plasmids described previously (45). Complementary DNAs     DNA sequencing. An adenoviral construct expressing hMOF (Ad.hMOF) was <br /> encoding wild mutant hMOF (&#226;&#338;&#172;hMOF) cloned mammalian ex-            constructed described previously (44). hMOF small interfering RNA (siRNA) <br /> pression vector pIND(SP1)/Neo (Invitrogen, Carlsbad, CA) described previously   control Luc siRNA obtained Dharmacone Research (Lafayette, <br /> (18, 45). Mutant forms hMOF created using PCR approach           CO). We used hMOF siRNA vector pRNA-U6.1/Neo (GeneScript). <br />  5294        GUPTA ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. hMOF ATM interaction. The specific interaction ATM hMOF shown determined coIP experiments. Cells <br /> were harvested 1 h postirradiation. IP immunoblotting hMOF ATM- species-matched immunoglobulin G (IgG) control <br /> antibodies. Input represents 50% total protein used IPs. Cell lysates AT221JE-TpEBS7-YZ5 (A-T fibroblast ATM <br /> cDNA complementation) AT22IJE-TpEBS7 (A-T fibroblasts vector only) cells. (A) IP performed anti-hMOF antibodies. <br /> Coprecipitated ATM detected Western blot analysis anti-ATM antibody AT22IJE-TpEBS7-YZ5 cells. IR-induced ATM <br /> phosphorylation detected anti-p-Ser-1981 ATM antibody. (B) Reciprocal IP ATM detect hMOF ATM complex. Coprecipi- <br /> tated hMOF detected AT22IJE-TpEBS7-YZ5 using anti-ATM antibody. <br />  <br />  <br />  <br />  <br />    Western analysis, immunoprecipitation, HAT, ATM kinase assays. Cell              irradiated (I) unirradiated (c) cells 75% internal stan- <br /> lysates prepared according previously described procedure (41).             dard DNA left filter. <br /> Polyclonal hMOF antibody raised rabbit described previously (34). <br /> Immunoblotting detections hMOF ATM according a <br /> previously described procedure (41, 45). For immunoprecipitation (IP), cells                                             RESULTS <br /> were broken lysis buffer described previously (41). Lysates precleaned <br /> with purified immunoglobulin G protein A/G beads. Proteins immu-                  Using modified yeast two-hybrid screen (12, 30) described <br /> noprecipitated specific antibodies, immunoprecipitants washed             Materials Methods, putative cellular binding <br /> with lysis buffer described previously (41). An ATM kinase assay per- <br /> formed described previously (41). A histone acetyltransferase (HAT) assay <br />                                                                                         partners ATM isolated, including encoded by <br /> was performed according manufacturer&#226;&#8364;&#8482;s instructions (Upstate Biotech-            cDNAs identified DNA sequence analysis human or- <br /> nology).                                                                                thologs Drosophila MOF (15). The longer cDNAs <br />    Assay ionizing radiation sensitivity. Cells plated 35-mm dishes.         1.8 kb encoded 458-amino-acid protein mass <br /> The cell count determined using Coulter counter. Cells plated as <br />                                                                                         52.4 kDa containing chromodomain region, zinc finger, <br /> single cells 60-mm dishes 5 ml media, incubated 6 h, subse- <br /> quently exposed IR. The actual amounts cells dish chosen              acetyl coenzyme A (acetyl-CoA)-binding site HAT <br /> ensure 50 colonies survive particular radiation dose treat-          activity directed histones H3, H2A, H4 (34). <br /> ment. Cells exposed IR dose range 0 8 Gy room temper-              The interaction hMOF ATM examined by <br /> ature. Cells incubated 12 days fixed methanol:             liquid &#226;?&#164;-gal assays (Fig. 1). Full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> hMOF ATM frag- <br /> acetic acid (3:1) prior staining crystal violet. Only colonies containing &#226;&#172;&#381;50 <br /> cells counted. <br />                                                                                         ments containing leucine zippers (LZ) exhibited 3.4- 7.8- <br />    Metaphase chromosome spreads prepared procedures described pre-              fold higher &#226;?&#164;-gal activities observed empty <br /> viously (35). Giemsa-stained chromosomes metaphase spreads analyzed             vector control. Minimum interactions between <br /> for chromosome end-to-end associations. G1 phase-specific chromosomal aber-             ATM fragments containing PI3-kinase domain hMOF <br /> rations assessed described previously (37). Briefly, cells plateau phase <br />                                                                                         hMOF proteins. Deletion region containing <br /> were irradiated 3 Gy, allowed incubate 24 h, subcultured, and <br /> metaphases collected. Chromosome spreads prepared proce-               LZ domain ATM fragment deletion region <br /> dure described previously (35). The categories G1-type asymmetrical chromo-          containing chromodomain hMOF resulted loss of <br /> some aberrations scored included dicentrics, centric rings, interstitial deletions/     interaction, establishing association pro- <br /> acentric rings, terminal deletions.                                                 teins occurs interaction regions containing the <br />    S-phase-specific chromosomal aberrations analyzed metaphase. Expo- <br /> nentially growing cells treated 2 Gy gamma radiation, mitotic <br />                                                                                         domains (Fig. 1). <br /> cells collected 3 6 h irradiation. Both chromosome-               To confirm interactions ATM hMOF <br /> chromatid-type aberrations scored. For G2 phase-specific chromosomal               occurred mammalian cells, performed coimmunoprecipi- <br /> aberrations, cells exponential phase irradiated 1 Gy, met-             tation (coIP) experiments. Anti-ATM antibody immunopre- <br /> aphases collected 45 90 min following irradiation examined for <br />                                                                                         cipitated endogenous hMOF cells expressing func- <br /> chromatid breaks gaps metaphase described previously (33). <br />    Neutral filter elution. The level DNA DSBs cells estimated              tional ATM, association proteins did <br /> described previously (40). Cells labeled period one-and-one-half          change exposure IR (Fig. 2A). The reciprocal <br /> doublings washed free radioactive medium reincubated            immunoprecipitation yielded results (Fig. 2B). The <br /> nonradioactive medium additional 3 h prior experiment. Cells         association hMOF ATM confirmed noniso- <br /> exposed IR incubated 37&#194;&#176;C different times postirradiation. Cells <br /> were layered polycarbonated filters, lysed, eluted neutral condi- <br />                                                                                         genic cells respect ATM function (data shown). <br /> tions (pH 9.6) described previously (40). Relative elution (RE) calculated          The region ATM responsible interactions with <br /> as RE &#226;&#171;&#189; &#226;&#171;&#186;log(FI/Fc), FI Fc fractions DNA left filter        hMOF determined vitro incubating glutathione-S- <br />  VOL. 25, 2005                                                                           MOF INFLUENCES ATM FUNCTION                 5295 <br />  <br />  <br />  <br />  <br />    FIG. 3. In vitro mapping region ATM interacts hMOF. Binding detection vitro-translated hMOF (A) 293 cell <br /> extracts (B) series GST-ATM proteins performed according previously described procedures (26). (C) Mapping region of <br /> hMOF interacts ATM. Bacterially expressed GST-hMOF fusion proteins incubated 293 cell extracts, bound protein was <br /> analyzed immunoblotting ATM antibody. (D) The LZ region ATM essential interaction hMOF vivo IR-induced <br /> Ser-1981 ATM phosphorylation. AT22IJE (deficient ATM) cells transfected Flag-tagged ATM&#226;&#338;&#172;LZ Flag-tagged ATM and <br /> treated 5 Gy. ATM&#226;&#338;&#172;LZ did coimmunoprecipitate hMOF, did ATM&#226;&#338;&#172;LZ autophosphorylate IR. <br />  <br />  <br />  <br /> transferase (GST)-ATM proteins vitro transcribed          To map hMOF domain interacting ATM, GST- <br /> translated 35S-labeled hMOF protein described previously           hMOF deletion constructs, collectively spanning entire <br /> (26). hMOF interacted GST-ATM fragment                    open reading frame hMOF, generated reverse tran- <br /> (amino acids 1098 1371) containing LZ region (Fig. 3A).        scriptase PCR expressed bacterial cells. Cell extracts of <br /> To confirm interaction endogenous hMOF                293 cells mixed agarose beads coated GST- <br /> GST-ATM fusion protein, extracts 293 cells mixed            hMOF fusion proteins. The GST pull-down results identified <br /> with beads containing GST GST-ATM proteins. The bound              chromodomain region hMOF site interacting with <br /> protein analyzed immunoblotting hMOF antibody.            ATM (Fig. 3C). Subsequently, physiological relevance of <br /> The results experiments clearly delineate LZ re-         interaction LZ region ATM hMOF was <br /> gion ATM domain interacting hMOF (Fig. 3B).            established coexpressing Flag-tagged ATM&#226;&#338;&#172;LZ (ATM with <br />  5296     GUPTA ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br /> a deletion LZ region) (10) hMOF ATM-deficient <br /> (AT221JE) cells examined IR-induced ATM au- <br /> tophosphorylation. Deletion LZ region ATM resulted <br /> in loss interaction hMOF IR-induced ATM <br /> autophosphorylation (Fig. 3D). Such results consistent <br /> with fact ATM&#226;&#338;&#172;LZ retains kinase activity vitro; <br /> however, unable phosphorylate substrates vivo <br /> (10). <br />    IR enhances hMOF-mediated acetylation histone H4 at <br /> K16. To test IR exposure alters hMOF levels ac- <br /> tivity presence absence functional ATM, cells in <br /> exponential phase irradiated, hMOF protein levels <br /> were determined increasing intervals postirradiation. Expo- <br /> sure IR altered hMOF protein levels (Fig. 4A) nor <br /> its H4 peptide acetylation activity (Fig. 4B), irrespective of <br /> whether cells expressed ATM not. Like ortholog Dro- <br /> sophila (1, 49) contrast yeast Esa1p mammalian <br /> Tip60, hMOF acetylates histone H4 uniquely lysine 16 (E. <br /> R. Smith J. C. Lucchesi, unpublished observation). Using <br /> an antibody specific human H4 acetylated K16, de- <br /> tected increase acetylation histone H4 following IR <br /> exposure detectable change overall histone H4 <br /> levels (Fig. 5B). Cells (293) (GM 5849) func- <br /> tional ATM did difference histone acetylation, <br /> suggesting K16 acetylation histone H4 hMOF is <br /> independent ATM function (Fig. 5B). To determine <br /> whether increased K16 acetylation histone H4 IR <br /> exposure specifically hMOF activity, cells either <br /> expressing mutant hMOF (&#226;&#338;&#172;MOF; fragment 1 Fig. 1A) or <br /> with hMOF knockdown analyzed Western blotting for <br /> IR-induced acetylation histone H4 K16. In comparison to <br /> the levels control, cells expressing &#226;&#338;&#172;hMOF (Fig. <br /> 1A) cells hMOF knockdown (Fig. 5C) decreased <br /> acetylation histone H4 K16 IR exposure (Fig. 5B, <br /> D), irrespective presence ATM; cells overexpressing <br />                                                                       FIG. 4. Effect IR hMOF levels activity. (A) Cells with <br /> full-length hMOF (Fig. 5A) higher levels IR-induced         ATM (293) ATM (GM 5849) exposed 5 Gy and <br /> K16 acetylation histone H4 control cells did (Fig. 5B).    examined hMOF p53 levels various times postirradiation. <br /> These results suggest IR exposure enhances hMOF-de-           Note change hMOF levels; however, p53 levels increased <br /> pendent acetylation histone H4 K16 independently          293 cells GM 5849 cells postirradiation. (B) HAT assay <br /> ATM function.                                                      hMOF. Purified wild-type recombinant GST-hMOF, purified re- <br />                                                                    combinant GST-&#226;&#338;&#172;hMOF, purified recombinant GST-&#226;&#338;&#172;hMOF-ac, and <br />    hMOF inactivation abrogates ATM activation. We tested           immunoprecipitated hMOF control irradiated 293 cells (25 <br /> whether inactivation hMOF influence IR-          ng assay) added HAT reaction mixture containing acetyl <br /> induced ATM activation. The expression &#226;&#338;&#172;hMOF abro-              coenzyme A (HAT assay kit; Upstate Biotechnology). Biotinylated <br /> gated ATM autophosphorylation irradiated cells (Fig. 6A),       histone H4 peptides acetylated wild-type hMOF by <br />                                                                    &#226;&#338;&#172;hMOF &#226;&#338;&#172;hMOF-ac. Recombinant PCAF (25 ng assay) was <br /> whereas expression hMOF fragments 2 4 little         used positive control HAT assay. <br /> effect ATM autophosphorylation (data shown). To de- <br /> termine hMOF association influences ATM <br /> autophosphorylation, cells expressing HA-tagged wild-type <br /> hMOF &#226;&#338;&#172;hMOF irradiated examined ATM                tone H4 K16 influence ATM activation, cells <br /> Ser1981 phosphorylation (Fig. 6B). Interestingly, ATM com-         treated 100 nM deacetylase inhibitor <br /> plexed full-length hMOF displayed IR-induced ATM              trichostatin A (TSA) 5 mM sodium butyrate (NaB) for <br /> phosphorylation (Fig. 6Ba), phosphorylation        4 h. TSA NaB enhanced level acetylated histone H4 <br /> was observed ATM associated &#226;&#338;&#172;hMOF deletion             K16, slight effect, case TSA, ATM <br /> mutant (Fig. 6Bb). The influence hMOF IR-induced             autophosphorylation ATM kinase activity (Fig. 6E). <br /> ATM autophosphorylation consistent influence        The high levels IR-induced autophosphorylation ATM <br /> hMOF ATM monomerization (Fig. 6C). Furthermore, cells           ATM kinase activity unaffected TSA-enriched <br /> expressing &#226;&#338;&#172;hMOF cells hMOF knockdown             acetylation histone H4 K16. These results suggest that <br /> greatly reduced IR-induced ATM kinase activity, deter-          IR-induced acetylation histone H4 K16 hMOF along <br /> mined vitro kinase assay (Fig. 6D) (41).                  DNA breaks induced IR exposure induce changes <br />    To determine increase acetylation his-     chromatin structure chromatin changes <br />  VOL. 25, 2005                                                                               MOF INFLUENCES ATM FUNCTION                   5297 <br />  <br />  <br />  <br />  <br />    FIG. 5. Effect IR exposure acetylation histone H4 K16. (A) Ad.hMOF constructed described previously (44). hMOF levels <br /> were detected Western blotting using hMOF-specific antibody. Higher levels hMOF observed cells overexpressing hMOF (&#226;&#171;&#185;hMOF). <br /> (B) Analysis extracts GM5849, 293 cells infected adenovirus vector, 293 cells expression &#226;&#338;&#172;hMOF-ac, 293 cells <br /> infected Ad.hMOF levels histone H4 acetylated histone H4 K16. Histone H4 levels change cell type IR. <br /> However, cells expressing &#226;&#338;&#172;hMOF minimum increase level acetylated histone H4 K16, cells overexpressing hMOF have <br /> higher levels acetylated histone H4 K16 IR treatment, demonstrating hMOF does acetylate histone H4 K16. (C) siRNA-mediated <br /> reduction hMOF protein isogenic cells respect ATM function. (D) Reduced hMOF expression results loss IR-induced <br /> acetylation histone H4 K16 cells ATM. <br />  <br />  <br />  <br /> may transduced hMOF ATM activation. We                      hMOF influences ATM-mediated phosphorylation down- <br /> also tested hMOF influence ATM activation                stream effectors. In mammalian cells, response DNA <br /> a nonchromatin context. To address issue, exploited           damage, ATM phosphorylates various proteins control <br /> in vitro MRN-dependent substrate phosphorylation               G1, S, G2 cell cycle arrest (13, 36, 46). We examined <br /> by ATM described previously (29). Our analysis revealed                inactivation hMOF influences ATM-mediated <br /> that hMOF does alter ATM function vitro (Fig. 7),                  IR-induced phosphorylation Chk2 293 cells. ATM phos- <br /> supporting argument hMOF act trans-                     phorylates Chk2 Thr-68 response DNA damage (31), <br /> ducer chromatin structural alterations ATM DNA                activates Chk2 phosphorylate substrates including p53 <br /> damage.                                                                   Cdc25 (32). Cells (293) expressing &#226;&#338;&#172;hMOF cells <br />  5298     GUPTA ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. hMOF inactivation influences ATM activation. (A) The expression &#226;&#338;&#172;hMOF hMOF knockdown siRNA resulted failure of <br /> IR-induced ATM autophosphorylation 293 cells. (B) Functional hMOF association required IR-induced ATM autophosphorylation. 293 <br /> cells transfected HA-hMOF (a) HA-&#226;&#338;&#172;hMOF (b). ATM immunoprecipitated anti-HA tagged hMOF showed only <br /> IR-induced ATM autophosphorylation (subpanel a), ATM complexed &#226;&#338;&#172;hMOF did loss IR-induced ATM autophosphor- <br /> ylation (subpanel b). (C) hMOF inactivation influences IR-induced ATM dimer-to-monomer formation. (a) hMOF knocked 293 cells <br /> by siRNA described above. (b) Flag-tagged ATM coexpressed HA-tagged ATM cells functional hMOF treated <br /> with 10 Gy. Thirty minutes irradiation, ATM immunoprecipitated anti-Flag immunoblotted anti-HA antibody. Whole-cell <br /> extract immunoblotted anti-HA. Note loss hMOF expression &#226;&#338;&#172;hMOF abrogates IR-induced conversion ATM <br /> dimer monomer. (D) hMOF influences IR-induced ATM kinase activity. (a) hMOF inactivation expression &#226;&#338;&#172;hMOF knockdown of <br /> hMOF siRNA 293 cells results decreased IR-induced ATM kinase activity. ATM kinase assay performed described previously (41). <br /> (b) The quantitation bands performed described previously (41). (E) IR-induced acetylation histone H4 K16 correlates ATM <br /> activation. (a) I, Histone H4 levels acetylation histone H4 K16. II, ATM autophosphorylated ATM levels. III, ATM and <br /> phosphorylated ser15-p53 levels. (b) The quantitation bands panel E(a) performed described previously (41). The values shown <br /> in histograms means independent experiments. <br />  <br />  <br />  <br />  <br /> with hMOF knockdown reduced IR-induced phosphory-                       hMOF influences genomic stability IR response. Cells <br /> lation Chk2 Thr-68 compared control cells (Fig. 8).             derived A-T individuals exhibit genomic instability and <br /> These results support argument hMOF function in-                hypersensitive IR radiomimetic drugs (36, 46). A-T <br /> fluences ATM-dependent phosphorylation downstream                 fibroblasts premature senescence. Telomere loss and <br /> DNA damage response effectors.                                           chromosome end-to-end associations frequently seen in <br />  VOL. 25, 2005                                                                    MOF INFLUENCES ATM FUNCTION                5299 <br />  <br />  <br />  <br />  <br />                                                         FIG. 6&#226;&#8364;&#8221;Continued <br />  <br />  <br />  <br /> A-T cells (36, 42). These cells exhibited approximately 0.39     1). Once again, significant change chromosomal break <br /> chromosome end-to-end associations metaphase,        frequencies observed cells overexpressing wild-type <br /> the parental cells displayed 0.12 chromosome end-to-end as-      hMOF (Table 1). <br /> sociations metaphase (Table 1). The increase chromo-         Consistent differences spontaneous genomic <br /> some end-to-end association seen cells expressing &#226;&#338;&#172;hMOF       damage, ATM&#226;&#171;&#185;/&#226;&#171;&#185; cells expressing &#226;&#338;&#172;hMOF cells with <br /> is similar cells hMOF knockdown. In contrast,    hMOF knockdown exhibited modest enhancements cell kill- <br /> the overexpression hMOF effect frequency     ing response IR exposure compared parental cells, <br /> chromosome end associations. Since chromosome end-to-end         ATM&#226;&#171;&#186;/&#226;&#171;&#186; cells expressing &#226;&#338;&#172;hMOF did (Fig. 9). How- <br /> associations lead anaphase bridge formation,     ever, cells overexpressing hMOF exhibited decreased IR sen- <br /> cells analyzed anaphase bridges. Cells expressing   sitivity cell killing (Fig. 9). Thus, cells expressing &#226;&#338;&#172;hMOF <br /> &#226;&#338;&#172;hMOF cells hMOF knockdown displayed              cells hMOF knockdown display karyotypic instability <br /> threefold higher frequencies anaphase bridges compared     decreased <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>cell survival</span> IR exposure. All these <br /> parental cells (Table 1). Chromosome end associations        cellular phenotypes linked defective chromo- <br /> bridges induce spontaneous chromosome chro-       somal repair, reported cells deficient ATM <br /> matid breaks. Cells expressing &#226;&#338;&#172;hMOF cells hMOF         (36). <br /> knockdown displayed higher frequencies chromatid         One way address DNA repair affected by <br /> as chromosome breaks compared parental cells (Table      hMOF function compare cell cycle stage-specific chromo- <br />  5300      GUPTA ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. hMOF required ATM phosphorylation p53 on <br /> Ser-15 MRN vitro. A kinase assay performed described <br /> previously (29). The concentration ATM 0.125 nM, (&#226;&#171;&#185;) or <br /> without (&#226;&#171;&#186;) MRN (7.5 nM) various concentrations hMOF. The <br /> human ATM expressed human 293T cells transient                 FIG. 8. hMOF inactivation influences Chk2 phosphorylation. 293 <br /> transfection plasmids containing full-length Flag-tagged ATM         cells expressing &#226;&#338;&#172;hMOF cells hMOF knockdown treated <br /> pcDNA3. The recombinant ATM purified resin conjugated            10 Gy. Cell extracts immunoblotted Chk2 Chk2 <br /> anti-Flag antibodies washed extensively high-salt buffer be-      specific phospho-Threonine (P-T68) antibodies. <br /> fore elution Flag peptide. The recombinant MRN complex was <br /> expressed Sf21 insect cells baculovirus puri- <br /> fied nickel affinity, ion exchange, gel filtration chromatography <br /> (29). Recombinant GST-hMOF added kinase assay                cycle G1, S-phase-specific chromosome aberrations <br /> determine required purified ATM function vitro. <br /> Addition hMOF significant effect ATM-dependent            cells evaluated. Cells expressing &#226;&#338;&#172;hMOF dis- <br /> phosphorylation p53 ATM autophosphorylation in vitro        played higher frequencies S-phase-specific chromatid and <br /> assay.                                                                     chromosomal aberrations metaphase compared to <br />                                                                            parental cells (Fig. 10B). Similar results obtained when <br />                                                                            G2-phase-specific chromosome aberrations evaluated <br /> somal aberrations cells expression              (Fig. 10C). The overexpression hMOF resulted modest, <br /> &#226;&#338;&#172;hMOF according previously described procedure (45).                statistically significant, decrease residual chro- <br /> Cells expressing &#226;&#338;&#172;hMOF exhibited significant increase                 mosome aberrations phases cell cycle <br /> residual IR-induced G1 chromosomal aberrations seen met-                (Fig. 10). In addition, used previously described biochem- <br /> aphases (Fig. 10A). To determine defective repair              ical approach (38) determine influence hMOF on <br /> be documented cells expressing &#226;&#338;&#172;hMOF phases cell              DNA DSB repair IR exposure. As illustrated Fig. 11, <br />  <br />  <br />  <br />  <br />    FIG. 9. hMOF influences cell survival. 293 cells expressing different fragments (fragments 1 3, shown Fig. 1A) &#226;&#338;&#172;hMOF wild-type <br /> hMOF exposed different doses ionizing radiation. Cells expressing &#226;&#338;&#172;hMOF fragment 1 fragment 3 (shown Fig. 1A) had <br /> reduced cell survival IR exposure. An A-T cell line used positive control. The results means experiments. The cells <br /> with expression &#226;&#338;&#172;hMOF hMOF knockdown statistically significant (P &#226;&#172;? 0.01, determined chi-square test) enhancements cell <br /> killing compared control cells. Cells expressing wild-type hMOF modest reduction cell killing compared control. <br />  VOL. 25, 2005                                                                                              MOF INFLUENCES ATM FUNCTION                           5301 <br />  <br />  <br />  <br />  <br />    FIG. 10. Chromosome aberrations IR treatment. (A) G1-specific aberrations exposure 3 Gy. (B) S-phase-specific aberrations after <br /> irradiation 2 Gy. (C) G2 phase-type aberrations irradiation 1 Gy. The means represent values independent <br /> experiments. For experiment, 50 metaphases scored. <br />  <br />  <br />  <br />               TABLE 1. Spontaneous chromosomal aberrations cells expressing mutant hMOF cells hMOF knockdown <br />                                           Chromosome end                                                    No. chromosome                     Chromatid gaps &#226;&#171;&#185; <br />                                                                               Bridges/300 <br />        Cell type                           associations/200                                                  gaps &#226;&#171;&#185; breaks/200                         breaks/200 <br />                                                                               anaphases <br />                                              metaphases                                                         metaphases                             metaphases <br />  <br /> 293                                              24                               16                                  2                                     3 <br /> 293 &#226;&#171;&#185; vector                               21                               13                                  3                                     3 <br /> 293 &#226;&#171;&#185; &#226;&#338;&#172;hMOF                                      78*                              43*                                12*                                    7* <br /> 293 &#226;&#171;&#185; hMOF                                       23                               12                                  3                                     1 <br /> 293 (hMOF-siRNA)                                 63*                              51*                                10*                                   11* <br /> 293 (control-siRNA)                              19                               14                                  4                                     2 <br /> HFF                                              12                                8                                  2                                     2 <br /> GM5823 (A-T)                                     68*                              56*                                18*                                   12* <br />   a <br />     Results cells expressing mutant hMOF cells hMOF knockdown significantly different controls (293 cells). Asterisks indicate <br /> that differences significant assessed chi-square analysis (P &#226;&#172;&#381; 0.01). <br />  5302      GUPTA ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />                                                                                                   DISCUSSION <br />  <br />                                                                              Members MYST protein family associated <br />                                                                           acute myeloid leukemia (MOZ), transcriptional silencing <br />                                                                           yeast (SAS2 YBF2/SAS3), interactions HIV Tat in <br />                                                                           humans (Tip60), dosage compensation Drosophila <br />                                                                           (MOF) (55). Histone acetyltransferases exist components of <br />                                                                           multisubunit complexes involved different processes as <br />                                                                           transcription activation, gene silencing, cell cycle progres- <br />                                                                           sion DNA damage repair (8). Such complexes have <br />                                                                           reported contain ATM-related proteins as <br />                                                                           TRRAP, human PCAF (2). ATM impli- <br />                                                                           cated mitogenic signal transduction, chromosome conden- <br />                                                                           sation, meiotic recombination, cell cycle control, telomere <br />                                                                           maintenance (36). Recently, Shahrabani-Gargir et al. (43) <br />                                                                           demonstrated ATM controls IGF-IR gene expression a <br />                                                                           DNA damage response pathway mechanism(s) involving <br />                                                                           zinc finger transcription factors Sp1 WT1. <br />                                                                              The interaction hMOF ATM substantiated our <br />                                                                           coimmunoprecipitation data fact able <br />                                                                           map respective regions interaction. Using RNA <br />    FIG. 11. DNA DSB repair analysis. Cells expres-       interference-mediated knockdown hMOF cDNA en- <br /> sion &#226;&#338;&#172;hMOF wild-type hMOF treated single doses         coding hMOF protein lacks C-terminal end includ- <br /> 15 Gy lysed different periods irradiation according   ing acetyl-CoA binding site, obtained evidence <br /> previously described procedure (40). A-T cells used control. <br /> The unrepaired DNA breaks measured neutral filter elution. <br />                                                                           hMOF function necessary ATM activation. This <br /> The data represent means independent experiments.             conclusion based observation hMOF inactivation <br />                                                                           reduces IR-induced autophosphorylation ATM its <br />                                                                           kinase activity, bypassing cell cycle arrest increas- <br /> cells expressing &#226;&#338;&#172;hMOF (fragment 1 Fig. 1A) higher                ing chromosome damage. It noted although <br /> amounts residual DNA DSBs, cells overexpressing                    overexpression &#226;&#338;&#172;hMOF fragment produce non- <br /> hMOF relatively residual DNA DSBs, control                 specific effects complicate interpretation of <br /> cells. The observation expression &#226;&#338;&#172;hMOF (frag-                experimental results, numerous experiments both <br /> ment 1 Fig. 1A) results higher frequency chromosome            RNA interference-mediated knockdown hMOF and <br /> aberrations irrespective cell cycle phase analyzed           &#226;&#338;&#172;hMOF used, results obtained ap- <br /> supports idea inactivation hMOF reduces               proaches similar. <br /> cell&#226;&#8364;&#8482;s capacity global DNA repair.                                       Like ortholog Drosophila, contrast other <br />    A characteristic hallmark A-T cells loss              known histone acetyltransferases, hMOF acetylates histone H4 <br /> DNA-damage-induced cell cycle checkpoint (24, 36, 46). Cells              uniquely lysine 16. In Drosophila, particular isoform of <br /> expressing &#226;&#338;&#172;hMOF displayed significant increase              H4 restricted X chromosome males, its <br /> G1 phase cells decrease S-phase entry IR treat-            presence directly correlated transcriptional enhance- <br /> ment compared control cells showed significant de-              ment responsible dosage compensation (6). In humans, is <br /> crease S-phase entry, indicating G1 arrest (Fig. 12A). The             abundantly distributed euchromatic chromo- <br /> loss G1 checkpoint seen cells expressing &#226;&#338;&#172;hMOF               complement (21). This observation extent and <br /> similar A-T cells (53), indicating <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>         rapidity ATM phosphorylated monomerized <br /> is mimicked cells expressing &#226;&#338;&#172;hMOF. In addition G1              following IR exposure (4) suggest possibility link <br /> checkpoint, IR causes transient inhibition DNA replica-              H4 acetylation hMOF ATM activation. <br /> tion. Cells deficient ATM function exhibit radioresistant                 However, TSA- NaB-induced enhancement acetylation <br /> DNA synthesis, indicator loss S-phase check-             levels histone H4 K16 minimum influence ATM <br /> point. Cells expressing &#226;&#338;&#172;hMOF showed decreased inhibition              kinase activity, suggesting modifications component <br /> DNA synthesis, compared parental cells, following IR               chromatin sufficient ATM activation. Such re- <br /> exposure (Fig. 12B). A-T cells defective G2/M check-            sults consistent fact ATM activation not <br /> points DNA damage, ATM signaling shown                 dependent chromatin modifications alone, ATM <br /> to required G2/M checkpoint response IR (54).            kinase activity IR exposure S phase is <br /> The efficiency G2 checkpoint control evaluated mea-             similar G1- G2/M-phase cells (41). <br /> suring proportion cells mitosis (mitotic index) IR           A plausible explanation functional interaction <br /> exposure (11). Cells expressing &#226;&#338;&#172;hMOF exhibited decrease                hMOF ATM acetylation hMOF ATM itself <br /> in mitotic index approximately 25%, decrease              unknown nonhistone protein involved ATM acti- <br /> the parental control cells approximately 68% (Fig. 12C).              vation. Acetyltransferases acetylate histone non- <br /> These results suggest inactivation hMOF does lead             histone proteins include p300, CBP, PCAF, Tip60, which <br /> to defective G2 checkpoint.                                             acetylates androgen receptor addition H2A, H3, and <br />  VOL. 25, 2005                                                                                MOF INFLUENCES ATM FUNCTION                    5303 <br />  <br />  <br />  <br />  <br />    FIG. 12. Cell cycle checkpoint analysis. (A) 293 cells irradiated 8 Gy. Twelve hours following irradiation, cells pulsed with <br /> bromodeoxyuridine 30 min. The proportion S-phase cells determined. The percentage S-phase cells irradiated culture relative to <br /> that unirradiated control shown. The results represent means experiments. For independent experiment, 200 cells were <br /> examined. (B) The S-phase checkpoint determined radioresistant DNA synthesis exposure ionizing radiation. 293 cells were <br /> irradiated doses indicated. The rate DNA synthesis determined 1 h postirradiation pulse labeling [3H]thymidine 30 min. <br /> The value unirradiated control set 100% cell type. The means standard deviations triplicate experimental points are <br /> shown. (C) Mitotic index monitored 2 h treatment 2 Gy. The means represent values independent experiments. For <br /> each experiment, 200 metaphases scored. <br />  <br />  <br />  <br />  <br /> H4 (55). More specifically, Drosophila MOF shown               suggesting DSBs induce relaxed chromatin topol- <br /> acetylate MSL3 (male-specific lethal 3), component              ogy, increasing nucleosomal access. TSA-treated cells <br /> the dosage compensation MSL complex; acetylation re-                exhibited slight increase ATM autophosphorylation <br /> quired specific MSL3 functions (7). Our observation               change ATM kinase activity absence of <br /> the level H4K16-Ac increased exposure IR               irradiation. The saturation acetylation sites chromatin <br /> any apparent increase hMOF activity             cells free hMOF, leading un- <br /> and irrespective presence ATM explained                scheduled acetylation nonhistone target(s). Whether <br />  5304      GUPTA ET AL.                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br /> ATM activation acetylation hMOF                                                     REFERENCES <br /> not known present. Our analysis revealed hMOF does                   1. Akhtar, A., P. B. Becker. 2000. Activation transcription through <br /> not influence function monomeric ATM in vitro                      histone H4 acetylation MOF, acetyltransferase essential dosage <br />                                                                                 compensation Drosophila. Mol. Cell 5:367&#226;&#8364;&#8220;375. <br /> assay (Fig. 7), supporting argument hMOF act                 2. Allard, S., R. T. Utley, J. Savard, A. Clarke, P. Grant, C. J. Brandl, L. Pillus, <br /> a transducer chromatin structural alterations ATM                   J. L. Workman, J. Cote. 1999. NuA4, essential transcription adaptor/ <br />                                                                                 histone H4 acetyltransferase complex containing Esa1p ATM-re- <br /> DNA damage.                                                                     lated cofactor Tra1p. EMBO J. 18:5108&#226;&#8364;&#8220;5119. <br />    However, observation hMOF-mediated chromatin                     3. Allfrey, V. G., B. G. Pogo, V. C. Littau, E. L. Gershey, A. E. Mirsky. 1968. <br /> alteration response IR exposure correlates ATM                       Histone acetylation insect chromosomes. Science 159:314&#226;&#8364;&#8220;316. <br />                                                                              4. Bakkenist, C. J., M. B. Kastan. 2003. DNA damage activates ATM <br /> activation supports proposition chromatin structure                    intermolecular autophosphorylation dimer dissociation. Nature <br /> may represent coupling mechanism intermolecu-                       421:499&#226;&#8364;&#8220;506. <br /> lar ATM phosphorylation IR-induced alterations chro-                  5. Bird, A. W., D. Y. Yu, M. G. Pray-Grant, Q. Qiu, K. E. Harmon, P. C. Megee, <br />                                                                                 P. A. Grant, M. M. Smith, M. F. Christman. 2002. Acetylation histone <br /> matin structure, proposed Bakkenist Kastan (4).                       H4 Esa1 required DNA double-strand break repair. Nature 419: <br /> Consistent potential role hMOF ATM activation                    411&#226;&#8364;&#8220;415. <br /> is ability dominant negative hMOF mutations hMOF                   6. Bone, J. R., J. Lavender, R. Richman, M. J. Palmer, B. M. Turner, M. I. <br />                                                                                 Kuroda. 1994. Acetylated histone H4 male X chromosome associ- <br /> knockdown recapitulate cellular characteristics A-T                   ated dosage compensation Drosophila. Genes Dev. 8:96&#226;&#8364;&#8220;104. <br /> cells, especially response exposure IR. hMOF                  7. Buscaino, A., T. Kocher, J. H. Kind, H. Holz, M. Taipale, K. Wagner, M. <br /> plays role signaling DNA damage response, evi-                      Wilm, A. Akhtar. 2003. MOF-regulated acetylation MSL-3 the <br />                                                                                 Drosophila dosage compensation complex. Mol. Cell 11:1265&#226;&#8364;&#8220;1277. <br /> denced influence phosphorylation ATM down-                  8. Carrozza, M. J., R. T. Utley, J. L. Workman, J. Cote. 2003. The diverse <br /> stream effectors. Subsequent acetylation histone H4                   functions histone acetyltransferase complexes. Trends Genet. 19:321&#226;&#8364;&#8220;329. <br />                                                                              9. Carson, C. T., R. A. Schwartz, T. H. Stracker, C. E. Lilley, D. V. Lee, and <br /> at K16 deacetylation order re-                     M. D. Weitzman. 2003. The Mre11 complex required ATM activation <br /> paired DNA nucleosomal form. It elegantly                         G2/M checkpoint. EMBO J. 22:6610&#226;&#8364;&#8220;6620. <br /> demonstrated histone H4K16 deacetylated sites                 10. Chen, S., P. Paul, B. D. Price. 2003. ATM&#226;&#8364;&#8482;s leucine-rich domain and <br />                                                                                 adjacent sequences essential ATM regulate DNA damage <br /> DSBs yeast (20), supporting argument                        response. Oncogene 22:6332&#226;&#8364;&#8220;6339. <br /> hMOF plays role early DNA damage repair. Kim et                   11. Dhar, S., J. A. Squire, M. P. Hande, R. J. Wellinger, T. K. Pandita. 2000. <br /> al. (27) shown ATM interacts histone                         Inactivation 14-3-3&#226;?&#180; influences telomere behavior ionizing radiation- <br />                                                                                 induced chromosomal instability. Mol. Cell. Biol. 20:7764&#226;&#8364;&#8220;7772. <br /> deacetylase HDAC1 vitro vivo                    12. Durfee, T., K. Becherer, P. L. Chen, S. H. Yeh, Y. Yang, A. E. Kilburn, W. H. <br /> extent association increased exposure                   Lee, S. J. Elledge. 1993. The retinoblastoma protein associates the <br /> MRC5CV1 human fibroblasts ionizing radiation. Consistent                     protein phosphatase type 1 catalytic subunit. Genes Dev. 7:555&#226;&#8364;&#8220;569. <br />                                                                             13. Elledge, S. J. 1996. Cell cycle checkpoints: preventing identity crisis. <br /> with results histone H4 K16 acetylation induced IR,                  Science 274:1664&#226;&#8364;&#8220;1672. <br /> Ju Miller (22) differences H3 H4                   14. Hasan, S., M. O. Hottiger. 2002. Histone acetyl transferases: role in <br />                                                                                 DNA repair DNA replication. J. Mol. Med. 80:463&#226;&#8364;&#8220;474. <br /> histone acetylation A-T normal cells response               15. Hilfiker, A., D. Hilfiker-Kleiner, A. Pannuti, J. C. Lucchesi. 1997. mof, <br /> histone deacetylase inhibitor treatments.                                       putative acetyl transferase gene related Tip60 MOZ human <br />    Despite interaction fraction hMOF ATM,                      genes SAS genes yeast, required dosage compensation in <br />                                                                                 Drosophila. EMBO J. 16:2054&#226;&#8364;&#8220;2060. <br /> hMOF-mediated acetylation histone H4 K16 following IR                 16. Hittelman, W. N., T. K. Pandita. 1994. Possible role chromatin <br /> exposure independent ATM function. In addition,                       alteration radiosensitivity ataxia-telangiectasia. Int. J. Radiat. Biol. <br /> inactivation hMOF enhances IR-induced cell killing                   66:S109&#226;&#8364;&#8220;S113. <br />                                                                             17. Horejsi, Z., J. Falck, C. J. Bakkenist, M. B. Kastan, J. Lukas, J. Bartek. <br /> ATM&#226;&#171;&#185;/&#226;&#171;&#185; cells ATM&#226;&#171;&#186;/&#226;&#171;&#186; cells, suggesting hMOF                      2004. Distinct functional domains Nbs1 modulate timing magni- <br /> functions upstream ATM. The results indicating ATM                      tude ATM activation low doses ionizing radiation. Oncogene <br /> autophosphorylation occurs ATM associated wild-                    23:3122&#226;&#8364;&#8220;3127. <br />                                                                             18. Hwang, K. K., H. J. Worman. 2002. Gene regulation human orthologs <br /> type hMOF &#226;&#338;&#172;hMOF support argument                          Drosophila heterochromatin protein 1. Biochem. Biophys. Res. Commun. <br /> the association functional hMOF ATM critical                     293:1217&#226;&#8364;&#8220;1222. <br />                                                                             19. Ikura, T., V. V. Ogryzko, M. Grigoriev, R. Groisman, J. Wang, M. Horikoshi, <br /> IR-induced ATM activation. Hasan Hottiger (14) pro-                         R. Scully, J. Qin, Y. Nakatani. 2000. Involvement TIP60 histone <br /> posed DNA repair, chromosomal repression relieved                    acetylase complex DNA repair apoptosis. Cell 102:463&#226;&#8364;&#8220;473. <br /> by acetylation histone N-terminal lysine residues, similar           20. Jazayeri, A., A. D. McAinsh, S. P. Jackson. 2004. Saccharomyces cer- <br />                                                                                 evisiae Sin3p facilitates DNA double-strand break repair. Proc. Natl. Acad. <br /> to loss chromosomal repression allows success-                  Sci. USA 101:1644&#226;&#8364;&#8220;1649. <br /> ful transcription underlying genes. Since acetylation            21. Jeppesen, P., B. M. Turner. 1993. The inactive X chromosome female <br /> of tail domain histone H4 K16 links DNA repair                   mammals distinguished lack histone H4 acetylation, cytogenetic <br />                                                                                 marker gene expression. Cell 74:281&#226;&#8364;&#8220;289. <br /> (5) transcription (1), experiments required              22. Ju, R., M. T. Muller. 2003. Histone deacetylase inhibitors activate <br /> determine MOF-mediated modifications discriminate                       p21(WAF1) expression ATM. Cancer Res. 63:2891&#226;&#8364;&#8220;2897. <br /> signals induced DNA damage transcription.                            23. Karlseder, J., K. Hoke, O. K. Mirzoeva, C. Bakkenist, M. B. Kastan, J. H. <br />                                                                                 Petrini, T. Lange. 2004. The telomeric protein TRF2 binds ATM <br />                                                                                 kinase inhibit ATM-dependent DNA damage response. PLoS <br />                        ACKNOWLEDGMENTS                                          Biol. 2:E240. [Online.] <br />                                                                             24. Khanna, K. K. 2000. Cancer risk ATM gene: continuing debate. <br />    We thank K. D. Brown, M. B. Kastan, S. J. Elledge, B. D. Price,          J. Natl. Cancer Inst. 92:795&#226;&#8364;&#8220;802. <br /> J. Chen providing constructs antibodies. We thank S. J.      25. Khanna, K. K., S. P. Jackson. 2001. DNA double-strand breaks: signal- <br /> Elledge, S. Elgin, Y. Shiloh, S. P. Jackson, C. Hunt, N. Horikoshi, S. N.       ing, repair cancer connection. Nat. Genet. 27:247&#226;&#8364;&#8220;254. <br /> Powell, J. Eissenberg comments, suggestions, helpful      26. Khanna, K. K., K. E. Keating, S. Kozlov, S. Scott, M. Gatei, K. Hobson, Y. <br />                                                                                 Taya, B. Gabrielli, D. Chan, S. P. Lees-Miller, M. F. Lavin. 1998. ATM <br /> discussion.                                                                     associates phosphorylates p53: mapping region interaction. <br />    This investigation supported NIH (NS34746, CA10445                    Nat. Genet. 20:398&#226;&#8364;&#8220;400. <br /> [project 2]), Department Army, A-T Children&#226;&#8364;&#8482;s Society,       27. Kim, G. D., Y. H. Choi, A. Dimtchev, S. J. Jeong, A. Dritschilo, M. Jung. <br /> funds Radiation Oncology, Washington University School                  1999. Sensing ionizing radiation-induced DNA damage ATM through <br /> Medicine, T.K.P. NCI grant PO1 CA97403 (project 5) T.L.               interaction histone deacetylase. J. Biol. Chem. 274:31127&#226;&#8364;&#8220;31130. <br />  VOL. 25, 2005                                                                                               MOF INFLUENCES ATM FUNCTION                            5305 <br />  <br /> 28. Kusch, T., L. Florens, W. H. Macdonald, S. K. Swanson, R. L. Glaser, J. R.             expression deoxyribonucleic acid damage response pathway mecha- <br />     Yates III, S. M. Abmayr, M. P. Washburn, J. L. Workman. 2004.                      nisms involving zinc-finger transcription factors Sp1 WT1. Endocrinol- <br />     Acetylation Tip60 required selective histone variant exchange             ogy 145:5679&#226;&#8364;&#8220;5687. <br />     DNA lesions. Science 306:2084&#226;&#8364;&#8220;2087.                                              44.   Sharma, G. G., A. Gupta, H. Wang, H. Scherthan, S. Dhar, V. Gandhi, G. <br /> 29. Lee, J. H., T. T. Paull. 2004. Direct activation ATM protein kinase         Iliakis, J. W. Shay, C. S. Young, T. K. Pandita. 2003. hTERT associates <br />     Mre11/Rad50/Nbs1 complex. Science 304:93&#226;&#8364;&#8220;96.                                    human telomeres enhances genomic stability DNA repair. <br /> 30. Lim, D. S., D. G. Kirsch, C. E. Canman, J. H. Ahn, Y. Ziv, L. S. Newman,               Oncogene 22:131&#226;&#8364;&#8220;146. <br />     R. B. Darnell, Y. Shiloh, M. B. Kastan. 1998. ATM binds beta-adaptin      45.   Sharma, G. G., K. K. Hwang, R. K. Pandita, A. Gupta, S. Dhar, J. Parenteau, <br />     cytoplasmic vesicles. Proc. Natl. Acad. Sci. USA 95:10146&#226;&#8364;&#8220;10151.                    M. Agarwal, H. J. Worman, R. J. Wellinger, T. K. Pandita. 2003. Human <br /> 31. Matsuoka, S., G. Rotman, A. Ogawa, Y. Shiloh, K. Tamai, S. J. Elledge.             heterochromatin protein 1 isoforms HP1Hs&#226;?&#163; HP1Hs&#226;?&#164; interfere with <br />     2000. Ataxia telangiectasia-mutated phosphorylates Chk2 vivo                 hTERT-telomere interactions correlate changes cell growth and <br />     vitro. Proc. Natl. Acad. Sci. USA 97:10389&#226;&#8364;&#8220;10394.                                      response ionizing radiation. Mol. Cell. Biol. 23:8363&#226;&#8364;&#8220;8376. <br /> 32. McGowan, C. H. 2002. CHK2: tumor suppressor not? Cell Cycle 1:401&#226;&#8364;&#8220;          46.   Shiloh, Y. 2003. ATM related protein kinases: safeguarding genome <br />     403.                                                                                   integrity. Nat. Rev. Cancer 3:155&#226;&#8364;&#8220;168. <br /> 33. Morgan, S. E., C. Lovly, T. K. Pandita, Y. Shiloh, M. B. Kastan. 1997.       47.   Smilenov, L. B., S. Dhar, T. K. Pandita. 1999. Altered telomere nuclear <br />     Fragments ATM dominant-negative complementing activ-                  matrix interactions nucleosomal periodicity ataxia telangiectasia cells <br />     ity. Mol. Cell. Biol. 17:2020&#226;&#8364;&#8220;2029.                                                    ionizing radiation treatment. Mol. Cell. Biol. 19:6963&#226;&#8364;&#8220;6971. <br /> 34. Neal, K. C., A. Pannuti, E. R. Smith, J. C. Lucchesi. 2000. A new human      48.   Smilenov, L. B., W. Mellado, P. H. Rao, S. G. Sawant, C. B. Umbricht, S. <br />     member MYST family histone acetyl transferases high se-                 Sukumar, T. K. Pandita. 1998. Molecular cloning chromosomal <br />     quence similarity Drosophila MOF. Biochim. Biophys. Acta 1490:170&#226;&#8364;&#8220;                  localization Chinese hamster telomeric protein chTRF1. Its potential role <br />     174.                                                                                   chromosomal instability. Oncogene 17:2137&#226;&#8364;&#8220;2142. <br /> 35. Pandita, T. K. 1983. Effect temperature variation sister chromatid <br />                                                                                      49.   Smith, E. R., A. Pannuti, W. Gu, A. Steurnagel, R. G. Cook, C. D. Allis, and <br />     exchange frequency cultured human lymphocytes. Hum. Genet. 63:189&#226;&#8364;&#8220; <br />                                                                                            J. C. Lucchesi. 2000. The Drosophila MSL complex acetylates histone H4 at <br />     190. <br />                                                                                            lysine 16, chromatin modification linked dosage compensation. Mol. <br /> 36. Pandita, T. K. 2003. A multifaceted role ATM genome maintenance. <br />                                                                                            Cell. Biol. 20:312&#226;&#8364;&#8220;318. <br />     Expert Rev. Mol. Med. 5:1&#226;&#8364;&#8220;21. <br /> 37. Pandita, T. K., C. R. Geard. 1996. Chromosome aberrations human           50.   Tse, C., T. Sera, A. P. Wolffe, J. C. Hansen. 1998. Disruption higher- <br />     fibroblasts induced monoenergetic neutrons. I. Relative biological effec-           order fold <br /> </body></html>