<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>PCI proteins eIF3e and eIF3m define distinct translation initiation <br /> factor 3 complexes <br />  <br />                  Background: PCI/MPN domain protein complexes comprise the 19S proteasome lid, the COP9 <br />                  signalosome (CSN), and eukaryotic translation initiation factor 3 (eIF3). The eIF3 complex is <br />                  thought <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> be composed of essential core subunits required for global protein synthesis and non- <br />                  essential subunits that may modulate mRNA specificity. Interactions of unclear significance were <br />                  reported between eIF3 subunits and PCI proteins contained in the CSN. <br />                  Results: Here, we report the unexpected finding that fission yeast has two distinct eIF3 complexes <br />                  sharing common core subunits, but distinguished by the PCI proteins eIF3e and the novel eIF3m, <br />                  which was previously annotated as a putative CSN subunit. Whereas neither eIF3e nor eIF3m <br />                  contribute to the non-essential activities of CSN in cullin-RING ubiquitin ligase control, eif3m, <br />                  unlike eif3e, is an essential gene required for global cellular protein synthesis and polysome <br />                  formation. Using a ribonomic approach, this phenotypic distinction was correlated with a different <br />                  set of mRNAs associated with the eIF3e and eIF3m complexes. Whereas the eIF3m complex <br />                  appears to associate with the bulk of cellular mRNAs, the eIF3e complex associates with a far more <br />                  restricted set. The microarray findings were independently corroborated for a random set of 14 <br />                  mRNAs by RT-PCR analysis. <br />                  Conclusion: We propose that the PCI proteins eIF3e and eIF3m define distinct eIF3 complexes <br />                  that may assist in the translation of different sets of mRNAs. <br />  <br />  <br />  <br />  <br /> Background                                                                    CSN/eIF3) domains [1]. The proteasome 20S catalytic <br /> Three protein complexes that are conserved <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to                      particle and the 19S regulatory subunit cooperate in <br /> humans, the 19S proteasome lid, the CSN, and eIF3, con-                       degrading polyubiquitylated proteins (reviewed in [2]). <br /> tain subunits characterized by two protein motifs: the                        The 19S proteasome can be separated into the <span id='am-88' about='xsp:base' typeof='owl:Thing'>base</span> com- <br /> MPN (Mpr1/ Pad1 N-terminal) and the PCI (proteasome/                          plex, which binds and unfolds substrates [3], and the <br />  <br />  <br />                                                                                                                                        Page 1 of 16 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> eight subunit lid complex, which cleaves ubiquitin from        Results <br /> substrates, thus apparently facilitating the entry of sub-     Both csn6 and csn7b are essential genes <br /> strates into the catalytic proteasome barrel [4,5].            CSN complexes of higher eukaryotes typically contain <br />                                                                eight distinct subunits (two MPN and six PCI proteins). <br /> In higher eukaryotes, the subunits of the 19S lid show         However, in fission yeast only six subunits are known <br /> pair-wise similarity to the eight subunits of the CSN [6-9].   (Csn1p, 2p, 3p, 4p, 5p, 7Ap; Ref. [13]), none of which are <br /> In vivo, CSN promotes the activity of cullin-RING ubiqui-      essential for viability [40-42]. We noticed two genes in the <br /> tin ligases [10-16], multiprotein complexes containing         Schizosaccharomyces pombe genome database, originally <br /> cullins, the RING protein RBX1, and one of several hun-        annotated as csn6            (SPBC4C3.07) and csn7b <br /> dred substrate-specific adaptors [17-22]. The MPN              (SPAC1751.03), which encode MPN and PCI domain <br /> domain containing CSN subunit 5 harbors a protease             containing proteins with considerable similarity to meta- <br /> motif [23] that cleaves the ubiquitin-related peptide          zoan CSN6 and CSN7B, respectively (data not shown). In <br /> NEDD8 from cullins [24,25]. This activity, acting in con-      order to determine whether these genes might function in <br /> cert with the CSN-associated deubiquitylation enzyme           the known biochemical pathways regulated by CSN, cul- <br /> Ubp12, was proposed to promote cullin function by facil-       lin deneddylation and Ubp12p-mediated deubiquityla- <br /> itating the recruitment of labile substrate adaptors           tion [24,26], we deleted csn6 and csn7b in wild-type <br /> [11,16,26,27].                                                 diploid cells. Upon sporulation and tetrad analysis, only <br />                                                                two viable spores could be recovered in each case, both of <br /> The third PCI/MPN complex, eIF3, is more distantly             which were uracil auxotroph (Fig. 1A, and data not <br /> related to CSN and the 19S lid (reviewed in [28]).             shown). Thus, unlike the genes encoding the six known <br /> Whereas human eIF3 consists of up to 13 subunits, con-         subunits of the fission yeast CSN [41,42], csn6 and csn7b <br /> secutively named eIF3a &#226;&#8364;&#8220; l and GA17 [29,30], budding           are essential. <br /> yeast contains only six to eight subunits (depending on <br /> purification conditions). Five of these subunits are           Csn6p and Csn7Bp associate with eIF3 components <br /> orthologs of human eIF3a, b, c, g, and eIF3i [31,32] and       To elucidate the essential functions of Csn6p and Csn7Bp, <br /> appear to constitute a conserved core complex [31,33].         we sought to identify their interacting proteins. To this <br /> Fission yeast contains the same five core subunits, in addi-   end, we attempted to modify csn6 and csn7b at their <br /> tion to the non-core subunits eIF3d/Moe1p, eIF3e/Int6p,        endogenous genomic loci with five consecutive protein A <br /> and eIF3h [34-36]. A putative eIF3f ortholog was also          epitope tags (proA). The C-terminal tag also contained a <br /> identified, but biochemical evidence confirming it as an       cleavage site for Tobacco Etch Virus (TEV) protease <br /> authentic eIF3 subunit functioning in protein synthesis is     upstream of the protein A moieties. Diploid analysis indi- <br /> still outstanding [34,35,37,38].                               cated that C-terminal tagging destroyed the essential func- <br />                                                                tion of csn6 (data not shown). In contrast, a csn7b-proA <br /> eIF3 is the most complex translation initiation factor and     haploid strain was viable and did not display any obvious <br /> plays at least two important roles in protein synthesis.       phenotypes (data not shown). This indicated that the <br /> First, eIF3 binds to the 40S ribosome and facilitates load-    tagged allele was functional. Total protein lysate from the <br /> ing of the Met-tRNA/eIF2&#194;&#183; GTP ternary complex to form          csn7b-proA haploid strain was absorbed to Immunoglobu- <br /> the 43S preinitiation complex. Subsequently, eIF3 appar-       lin G (IgG) resin and specifically retained proteins were <br /> ently assists eIF4 in recruiting mRNAs to the 43S complex.     eluted with sodium dodecyl sulfate (SDS) or by cleavage <br /> A critical in vivo function of eIF3 core subunits in these     with TEV protease. The two eluates were resolved by gel <br /> processes was indicated by the lethality of the respective     electrophoresis and Csn7Bp-proA interacting proteins <br /> budding yeast deletion strains [31]. In contrast, in fission   were identified by peptide mass fingerprinting using <br /> yeast, the non-core subunits eIF3d and eIF3e are not           matrix-assisted laser desorption/ionization-time-of-flight <br /> essential for viability or global protein synthesis            (MALDI-TOF) mass spectrometry. <br /> [34,36,38,39]. It was therefore proposed that distinct sub- <br /> classes of eIF3 complexes, containing different combina-       Both eluates contained an identical set of proteins. In <br /> tions of core and non-core subunits, may regulate specific     addition to Csn6p, the Csn7Bp complex contained the <br /> subsets of mRNAs in fission yeast [34,36,38,39]. Our           eIF3 core subunits eIF3a, b, c, and i, as well as the non- <br /> study provides the first experimental evidence substantiat-    core subunit eIF3h (Fig. 1B, Table 1). The eIF3g core sub- <br /> ing this hypothesis by demonstrating that biochemically        <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> was not detected using MALDI-TOF in this purifica- <br /> distinct eIF3 complexes defined by the PCI domain pro-         tion, but was detected in an independent purification of <br /> teins eIF3e and eIF3m (a novel eIF3 protein) associate         the Csn7Bp complex using the more sensitive method of <br /> with different sets of mRNAs.                                  nano-liquid chromatography and tandem mass spectrom- <br />                                                                etry (LC-MS/MS, see Fig. 5A). However, eIF3e and eIF3d, <br />                                                                two previously described components of S. pombe eIF3 <br />  <br />  <br />                                                                                                                   Page 2 of 16 <br />                                                                                             (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                             http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br />  <br />                                                      TEV                                            SDS <br /> A                                       B <br />                                                       csn7b-                                         csn7b- <br />                                                    wt proA                                        wt proA <br />          &#226;&#710;&#8224;csn6      &#226;&#710;&#8224;csn7b <br />                                            kDa                                           kDa <br />                                                                      eIF3c               121                         eIF3c <br />  1                                         121 <br />                                                                      eIF3a                                           eIF3a <br />                                                                      eIF3b                                           eIF3b <br />  2 <br />                                             75                                            75                         Csn7Bp-ProA <br />                                                                      eIF3c* <br />  3                                                                                                                   eIF3c* <br />                                                                      IgH <br />  4                                          50                       Csn7Bp <br />                                                                                           50                         IgH <br />                                             42                       eIF3h <br />                                                                                           42                         eIF3h <br />                                                                      eIF3i <br />                                                                      Csn6p                                           eIF3i <br />                                                                                                                      Csn6p <br />                                                                                           27 <br />                                             27                       TEV, IgL <br />                                                                      ProA                                            IgL, ProA <br />  <br />  <br />                                                  10%SDS-PAGE                                     Trycine Gel <br />  <br /> Figure <br /> Analysis 1of csn6 and csn7b genes and proteins <br /> Analysis of csn6 and csn7b genes and proteins. (A) The csn6 and csn7b genes were disrupted in diploid S. pombe cells by <br /> inserting the ura4 marker. Diploids were sporulated and spore viability was examined by tetrad analysis. Only two spores were <br /> viable, indicating that csn6 and csn7b are essential. (B) Cell lysate from a strain carrying csn7b modified with five C-terminal <br /> pro-A tags and a TEV cleavage site at the endogenous genomic locus was absorbed to IgG resin, followed by elution of bound <br /> proteins with TEV protease (left panel) or SDS (right panel). Gels were stained with Coomassie Brilliant Blue, and proteins <br /> were identified by MALDI-TOF mass spectrometry. The asterisks denote degradation products of elF3c. <br />  <br />  <br />  <br />  <br /> Table 1: Summary of eIF3 subunits in various eukaryotes <br />  <br />  Unified           Domain             Human               S. cerevisiae       S. pombe Csn7Bp   S. pombe Int6p       S. pombe eIF3b <br />  Nomenclature                                             (Core subunits)     complex           complex              complex <br />  <br />  eIF3a             PCI                p170                Tif32p              p107              p107                 p107 <br />  eIF3b             RRM                p116                Prt1p               p84               p84                  p84 <br />  eIF3c             PCI                p110                Nip1p               p104              p104                 p104 <br />  eIF3d             -                  p66                 -                   -                 Moe1p                Moe1p <br />  eIF3e             PCI                p48                 -                   -                 Int6p                Int6p <br />  eIF3f             MPN                p47                 -                   Csn6p             Csn6p                Csn6p <br />  eIF3g             RRM/Zn finger      p44                 Tif35p              Tif35p            Tif35p               Tif35p <br />  eIF3h             MPN                p40                 -                   p40               -                    p40 <br />  eIF3i             WD repeat          p36                 Tif34p              Sum1p             Sum1p                Sum1p <br />  eIF3j             -                  p35                 -                   -                 -                    - <br />  eIF3k             PCI                p28                 -                   -                 -                    - <br />  eIF3l             PCI                p67                 -                   -                 -                    - <br />  eIF3m             PCI                GA17 (?)            -                   Csn7Bp            -                    Csn7Bp <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 3 of 16 <br />                                                                                                     (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                       http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> [34,36,38,39] were consistently absent from the Csn7Bp         strong reduction in protein levels within 20 to 32 hours <br /> complex when analyzed either by MALDI-TOF or by LC-            (Fig. 2B). Coincident with shut-off of eIF3f and eIF3m <br /> MS/MS (Fig. 1B and 5A).                                        expression, global cellular protein synthesis was dimin- <br />                                                                ished by ~80%, as determined by metabolic labeling with <br /> Notably, no authentic CSN subunits were found in the           35S-methionine (Fig. 2C). <br />  <br /> Csn7Bp complex, consistent with the previous demon- <br /> stration that S. pombe CSN contains only six non-essential     Further biochemical analysis by sucrose gradient velocity <br /> subunits [13]. Structural analysis indicated that Csn6p is     centrifugation revealed a strong reduction in the forma- <br /> not only related to human CSN6, but shows a slightly           tion of polysomes in eif3f and eif3m mutants following <br /> greater overall similarity to human eIF3f (data not            promoter shut-off (Fig. 2D). In contrast, as described pre- <br /> shown). Based on the functional characterization               viously [34,36,38,39], eif3e mutants showed only a minor <br /> described below (Fig. 2), fission yeast Csn6p therefore        reduction in total protein synthesis (data not shown) and <br /> appears to be the ortholog of human eIF3f.                     essentially normal polysomes (Fig. 2D). <br />  <br /> A Csn7Bp ortholog is not known in the human eIF3,              The conserved cullin deneddylation function of CSN was <br /> although the recently identified subunit GA17 [30] shows       not impaired either in eif3f and eif3m mutants or in eif3e <br /> considerable similarity to Csn7Bp (K. Hofmann, personal        mutants, since cullin 1 (Cul1p) did not accumulate exclu- <br /> communication). In addition, interactions of metazoan          sively in the neddylated form as it does in csn5 deletion <br /> CSN7 with eIF3 subunits have previously been observed          strains (Fig. 2E). Similarly, unlike in csn5 mutants (Fig. 2E; <br /> [43,44]. With regards to the unified nomenclature of eIF3      [26]), CSN/Ubp12p-mediated inhibition of Cul1p in <br /> subunits [29], we propose eIF3m as a more appropriate          vitro ubiquitin ligase activity was not impaired upon shut- <br /> name for the novel fission yeast eIF3 subunit encoded by       off of either eif3f or eif3m expression. Cul1p neddylation <br /> csn7b (see Table 1). Throughout this manuscript, we will       and activity were also unaffected in eif3e mutants (Fig. <br /> use the proposed new names for the eif3f and eif3m genes       2E). These data suggested that eIF3f and eIF3m are bona <br /> and their products. The original annotations have recently     fide subunits of fission yeast eIF3 but not CSN, perform- <br /> been revised by the curator of the S. pombe genome data-       ing essential functions in protein synthesis similar to eIF3 <br /> base, in order to reflect the findings presented here.         core subunits. <br />  <br /> The eIF3f and eIF3m proteins are primarily cytoplasmic         The eIF3m and eIF3e proteins define distinct eIF3 <br /> The evidence presented above suggested that eIF3f and          complexes <br /> eIF3m are essential subunits of S. pombe eIF3. To substan-     Although the purification of the eIF3m complex robustly <br /> tiate this possibility, we determined the subcellular local-   retrieved roughly stoichiometric amounts of known eIF3 <br /> ization of eIF3f and eIF3m. If these proteins were             subunits, two subunits, eIF3d and eIF3e, were conspicu- <br /> authentic eIF3 subunits involved in protein synthesis,         ously absent (Fig. 1B, Table 1). To exclude the possibility <br /> they should assume a subcellular localization similar to       that we had overlooked these subunits because they were <br /> known eIF3 subunits. We therefore prepared three strains       obscured by the IgG bands, we affinity-purified eIF3e- <br /> with green fluorescent protein (GFP) tagged alleles of         associated proteins from an eif3e-proA strain. The retrieved <br /> eIF3b, eIF3e, and eIF3m integrated into their respective       proteins were identified using MALDI-TOF mass spec- <br /> genomic loci. In addition, since C-terminal GFP-tagging        trometry as before. <br /> destroyed the essential function of the small eIF3f protein, <br /> we prepared a strain mildly overexpressing N-terminally        Like the eIF3m complex, the eIF3e complex contained <br /> GFP-tagged eIF3f from a pREP81 plasmid. Fluorescence           roughly stoichiometric amounts of the eIF3 core subunits <br /> microscopy of live cells revealed that all four proteins       a, b, c, g, and i (Fig. 3A, Table 1). In addition, eIF3f was <br /> were primarily localized in the cytoplasm (Fig. 2A), which     present while, unexpectedly, eIF3m and eIF3h were miss- <br /> would be consistent with a function in protein transla-        ing. We did not detect the proteasome lid subunit Rpn5p, <br /> tion. In contrast, CSN subunits are largely nuclear in fis-    which previously had been shown to associate with eIF3e <br /> sion yeast ([41], and our own unpublished observation).        and eIF3d [45]. This observation indicated that the <br />                                                                described eIF3d/eIF3e/Rpn5p interaction is either unsta- <br /> eIF3f and eIF3m are required for global protein synthesis      ble under our purification conditions or contains only a <br /> To directly assess the potential role of eIF3f and eIF3m in    minor fraction of the total eIF3e engaged in protein <br /> protein synthesis, we prepared haploid eif3f and eif3m         interactions. <br /> deletion strains maintained viable by plasmids driving the <br /> regulated expression of eif3f and eif3m, respectively.         These results raised the intriguing possibility that fission <br /> Expression from these plasmids could be turned off by the      yeast contains two distinct eIF3 complexes that comprise <br /> addition of thiamin to the growth media, resulting in a        an overlapping set of core subunits, but are distinguished <br />  <br />  <br />                                                                                                                    Page 4 of 16 <br />                                                                                              (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                                                              http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br />  <br />        A                                   eIF3b.GFP         eIF3e.GFP             B         Strain:              + Thiamin (h)      0       20 24    28    32 <br />                                                                                                                                                                   kDa <br />                                                                                               &#226;&#710;&#8224;eif3f                                                              40 <br />                                                                                          pRep81.myc-eif3f         Myc-eIF3f <br />  <br />                                                                                               &#226;&#710;&#8224;eif3m                                                               50 <br />                                                                                          pRep81.myc-eif3m        Myc-eIF3m <br />  <br />  <br />                                                                                                                               60S Polysomes <br />                                                                                                  D <br />  <br />                                                                                                                   WT                                          &#226;&#710;&#8224;eif3e <br />                                            GFP.eIF3f         eIF3m.GFP <br />                                                                                                                            40S     80S <br />  <br />        C    Background                                  wt       &#226;&#710;&#8224;eif3f   &#226;&#710;&#8224;eif3m <br />                                                                                                                  eif3f                                        eif3f <br />                                                               pRep81. pRep81.                           &#226;&#710;&#8224;eif3f <br />                                            Plasmid     pRep81 myc-eif3f myc-eif3m                                 on                                           off <br />  <br />                                            Thiamin <br />  <br />  <br />                                                                                                        &#226;&#710;&#8224;eif3m    eif3m                                        eif3m <br />                                                                                                                    on                                          off <br />        Autoradiography <br />  <br />                                                                                                                                      n5 3f     3m 3e <br />                                                                                                  E                                 cs eif &#226;&#710;&#8224;eif &#226;&#710;&#8224;eif <br />                                                                                                                    Background: wt &#226;&#710;&#8224; &#226;&#710;&#8224; <br />                                                                                                                   pRep81.myc-      eif3f eif3m <br />                                                                                                                       Thiamin <br />              Coomassie <br />               staining <br />  <br />                                                                                                                  Blot:     kDa <br />               Relative 35S incorporation <br />  <br />  <br />  <br />  <br />                                            1.2                                     On                                                                        Ub(n) <br />                                             1                                      Off             IP:            &#206;&#177;-Ub     180 <br />                                                                                                    &#206;&#177;-Cul1p                 120 <br />                                            0.8 <br />                                            0.6 <br />                                                                                                                                                              Cul1p-Ned8p <br />                                                                                                                  &#206;&#177;-Cul1p <br />                                            0.4                                                                                75                             Cul1p <br />                                            0.2                                                                                           * <br />                                                                                                    Total                      50                            Myc-eIF3m <br />                                             0                                                                    &#206;&#177;-Myc <br />                                                  Vector         eif3f     eif3m                    lysate                     40                            Myc-eIF3f <br />  <br />  <br />  <br /> Roles <br /> Figureof2eif3e and eif3min protein synthesis <br /> Roles of eif3e and eif3min protein synthesis. (A) Subcellular localization. Live cells expressing GFP-tagged alleles of eif3b, <br /> eif3e, and eif3m at the endogenous genomic loci were examined by fluorescence microscopy. N-terminally GFP-tagged eif3f <br /> was expressed at low levels from the pREP81 plasmid. (B) Shut-off strains. Diploid heterozygous eif3f and eif3m deletion <br /> strains were transformed with pREP81 plasmids driving the thiamin-repressible expression of Myc-tagged eIF3f and eIF3m, <br /> respectively. Diploids were sporulated and homozygous disruptants carrying the eif3f and eif3m plasmids were selected. Cells <br /> were grown in liquid medium to an OD595 of 0.3 in the absence of thiamin, followed by promoter shut-off by the addition of <br /> thiamin. Samples were taken at the indicated time points after promoter shut-off and analyzed for the expression of plasmid <br /> borne eIF3f and eIF3m by immunoblotting with anti-Myc antibodies. (C) Effect on total protein synthesis. The eif3f and eif3m <br /> shut-off strains were maintained in the absence or presence of thiamin as indicated. Strains transformed with empty pREP81 <br /> plasmid were included as control. Cells were metabolically labeled with 35S-methionine, and aliquots of total cellular proteins <br /> were separated by SDS-PAGE and analyzed by autoradiography. The Coomassie Blue stained gel is shown to document equal <br /> protein loading. Data were quantified by PhosphorImager analysis and results are displayed in a bar graph. (D) Effect on polys- <br /> omes. Polysome profiles were determined for the indicated strains as described in the Methods section. An eif3e deletion <br /> strain is shown for reference. (E) Effect on CSN-mediated regulation of cullin-RING ubiquitin ligases. Cul1p complexes were <br /> immunopurified from the indicated strains and employed in substrate-independent in vitro ubiquitylation reactions with puri- <br /> fied E1, the E2 Cdc34p, ubiquitin, and ATP [26]. Polyubiquitin chains formed in the reaction are indicated (top panel). In con- <br /> trast to csn5 mutants, derepression of Cul1p activity is not observed in cells lacking eif3f, eif3m, or eif3e. The neddylation state <br /> of Cul1p was determined by immunoblotting (middle panel). Hyperneddylated Cul1p only accumulates in csn5 mutants. Shut- <br /> off of eif3f and eif3m expression was verified by immunoblotting with anti-Myc antibodies (lower panel). <br />  <br />  <br />  <br />                                                                                                                                                                       Page 5 of 16 <br />  <br />  <br /> Figure 3eIF3 complexes <br /> Distinct <br /> Distinct eIF3 complexes. (A) eIF3 complexes associated with eIF3e and eIF3m were affinity-purified by absorption of the <br /> respective pro-A-tagged proteins to IgG beads. Bound proteins were eluted by cleavage with SDS (eIF3e) or TEV protease <br /> (eIF3m). (B) The eluates described in (A) were analyzed by immunoblotting with eIF3d antibodies. The asterisk refers to cross- <br /> reactivity of the secondary antibody with Ig light chains. (C) Lysates from cells expressing Myc-tagged Csn5p, eIF3e, and eIF3m <br /> from their endogenous genomic loci were immunoprecipitated with anti-eIF3d antisera. Coprecipitated proteins were identi- <br /> fied by immunoblotting as indicated. Total cell lysates are shown to document expression levels of the endogenously tagged <br /> proteins. (D) Protein lysate from cells expressing protein A-tagged eIF3b from the endogenous promoter was absorbed to IgG <br /> beads, and specifically retained proteins were identified by LC-MS/MS. <br />  <br />  <br />  <br />                                                                                                                      Page 6 of 16 <br />                                                                                                (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                       http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> by the presence of either the essential PCI protein eIF3m       analysis revealed 106 mRNAs uniquely enriched in the <br /> and the MPN domain protein eIF3h or the non-essential           eIF3e complex, whereas 2050 transcripts were uniquely <br /> PCI protein eIF3e and its binding partner eIF3d. To             enriched in the eIF3m complex (Fig. 4A,C). <br /> exclude the possibility that our mass spectrometry analy- <br /> sis missed substoichiometric amounts of eIF3d in the            A ranked list of all 106 mRNAs that were uniquely <br /> eIF3m complex, we analyzed eIF3m complexes by immu-             enriched more than three-fold in the eIF3e complex was <br /> noblotting with eIF3d antisera. Whereas eIF3d was readily       assembled ([see additional file 2]). Approximately 75% of <br /> detectable in the eIF3e complex, it was undetectable in the     these mRNAs fell into one of three categories: mRNAs <br /> eIF3m complex (Fig. 3B). In the reciprocal experiment,          encoding proteins involved in intermediary metabolism <br /> eIF3d antibodies co-immunoprecipitated eIF3e-13Myc,             (25.5%), those encoding proteins involved in protein <br /> but not eIF3m-13Myc or Csn5p-13Myc (Fig. 3C).                   metabolism (14.2%), and those encoding proteins with <br />                                                                 unknown functions (35.8%; Fig. 4B). The remainder was <br /> To exclude the possibility that our cell lysis or affinity      distributed roughly equally among four categories: <br /> purification conditions led to uncontrolled release of eIF3     mRNAs encoding proteins involved in nucleic acid metab- <br /> subunits, thus mimicking the existence of distinct sub-         olism, those encoding proteins involved in transcription, <br /> complexes, we performed a purification using the protein        those encoding transporter proteins, and those encoding <br /> A-tagged eIF3b core subunit as bait. As determined by LC-       unclassified proteins. <br /> MS/MS performed on excised bands detected in the com- <br /> plex upon PAGE separation, the eIF3b complex contained          A corresponding list of the 106 most highly enriched <br /> all known eIF3 core components as well as eIF3f and             mRNAs exclusively present in the eIF3m complex (out of <br /> eIF3m (Fig. 3D). The eIF3e and eIF3d proteins were also         a total of 2050 enriched by a factor greater than three) <br /> present, albeit in clearly substoichiometric amounts.           revealed that ~58% also fell into three categories: mRNAs <br /> These findings indicate that purification of eIF3b resulted     encoding proteins involved in nucleic acid metabolism <br /> in the copurification of the distinct eIF3m- and eIF3e-con-     (9.4%), those encoding transporter proteins (18.9%), and <br /> taining complexes. Taken together, these data strongly          those encoding proteins of unknown function (20.8%). <br /> suggested the existence of two distinct eIF3 complexes          The remainder was roughly equally distributed among <br /> defined by the PCI proteins eIF3m and eIF3e.                    four categories: mRNAs encoding proteins involved in <br />                                                                 intermediary metabolism, those mRNAs encoding pro- <br /> The eIF3m and eIF3e proteins associate with distinct sets       teins involved in protein metabolism, those encoding <br /> of mRNAs                                                        proteins involved in transcription, and those encoding <br /> The finding that eif3m is essential, whereas eif3e is dispen-   unclassified proteins (Fig. 4B, [see additional file 2]). <br /> sable, suggested the possibility that the different eIF3 <br /> complexes they define regulate different subsets of             To exclude the possibility that our affinity purification <br /> mRNAs. To approach this, we sought to identify mRNAs            protocol unspecifically enriched highly expressed mRNAs, <br /> specifically associated with the eIF3m and eIF3e com-           we averaged the data from the three competively hybrid- <br /> plexes. The complexes were affinity-purified as described       ized samples of total S. pombe cDNA to determine the rel- <br /> above in the presence of RNAse inhibitors, and the associ-      ative expression rank of each mRNA (rank 1 = most <br /> ated RNA was extracted, amplified, and converted to Cy3-        abundant mRNA, rank 5407 = least abundant mRNA; see <br /> labeled cDNA as described in the Methods section. The           legend to Fig. 4C for detail). A plot of expression rank vs. <br /> Cy3-labeled cDNA was hybridized competitively with              factor of enrichment of the 106 mRNAs most enriched in <br /> Cy5-labeled cDNA prepared from total RNA onto micro-            the eIF3e and eIF3m complexes revealed that the eIF3m <br /> arrays representing all 4988 predicted S. pombe open read-      complex contained mRNAs that were distributed equally <br /> ing frames (ORFs) and RNAs. Based on the background             over the entire range of expression levels. In contrast, the <br /> subtracted hybridization signals, eIF3-associated mRNAs         eIF3e complex was enriched for rare mRNAs (~65 % of <br /> were ranked according to their factor of enrichment in the      eIF3e-associated mRNAs had an expression rank &gt; 4000; <br /> eIF3m and eIF3e complexes over a mock purified sample           see Fig. 4C). The lack of a linear correlation between <br /> ([see additional file 1]).                                      mRNA expression level and enrichment in the eIF3e and <br />                                                                 eIF3m complexes argues against unspecific copurification <br /> The microarray analysis suggested a global role for eIF3m       of abundant mRNAs. <br /> and a more restricted function of eIF3e in translation. <br /> Using an arbitrary cut-off value of three-fold enrichment       Specific association of mRNAs with eIF3m and eIF3e <br /> over mock, eIF3m associated with 2464 different mRNAs,          complexes <br /> whereas eIF3e associated with only 520 distinct species         To validate the results from the microarray hybridizations, <br /> (Fig. 4A). We observed 414 mRNAs enriched more than             we chose several mRNAs at random from each list of 106 <br /> three-fold in both samples. (Fig. 4A). In addition, our         and used RT-PCR to confirm their interactions with the <br />  <br />  <br /> Figure 4 of mRNAs with eIF3 complexes <br /> Association <br /> Association of mRNAs with eIF3 complexes. (A) Messenger RNAs associated with eIF3e and eIF3m complexes were <br /> identified by microarray hybridization as described in the Methods section. The graph indicates the numbers of mRNAs <br /> enriched more than three-fold over mock in both complexes. (B) Classification into functional groups of the 106 mRNAs most <br /> highly and uniquely enriched in complexes with eIF3e or eIF3m. (C) The relative expression ranks of the 106 most highly and <br /> uniquely enriched mRNAs in the eIF3e and eIF3m complexeswere determined using the relative hybridization signals obtained <br /> with total S. pombe cDNA ([see additional file 1]). The expression ranks were blotted against the factor of enrichment of each <br /> mRNA in the eIF3 complexes. No correlation between expression rank and enrichment is apparent, indicating that highly <br /> expressed mRNAs were not unspecifically enriched in eIF3 complexes. The 5407 individual features on the microarray slides <br /> represent the 4988 ORFs and transcripts predicted in the Sanger Centre S. pombe genome database and various controls. To <br /> simplify the analysis, expression ranks were also assigned to these control spots, thus resulting in expression ranks for some <br /> ORFs higher than the theoretically possible number of 4988. <br />  <br />  <br />  <br /> Figure association <br /> Specific 5           of mRNAs with the eIF3e and eIF3m complexes <br /> Specific association of mRNAs with the eIF3e and eIF3m complexes. (A) The eIF3 complexes shown were affinity- <br /> purified as described in the Methods section, and separated by SDS-PAGE followed by staining with Coomassie Brilliant Blue. <br /> The labeled bands were identified by LC-MS/MS. eIF3f* refers to a band in the eIF3m complex that was not identified by mass <br /> spectrometry in this experiment, and is therefore assigned based on its comigration with eIF3f identified in the eIF3e complex <br /> (right lane). eIF3f was positively identified by MALDI-TOF mass spectrometry as a subunit of the eIF3m complex in Fig. 1B. (B) <br /> The indicated eIF3e- and eIF3m-associated mRNAs were extracted from the purified complexes and employed in RT-PCR <br /> reactions using primers specific for each mRNA (see Table 3). PCR products obtained with total S. pombe RNA are shown for <br /> reference in the left panel. The factor of enrichment of each mRNA in the respective eIF3 complex and the enrichment rank <br /> (out of all 106 mRNAs enriched in either complex) are indicated below the gels. <br />  <br />  <br />  <br />  <br /> respective eIF3 complexes. eIF3e-proA- and eIF3m-proA-                     found to be enriched in the eIF3e complex by microarray <br /> associated complexes were affinity-purified as before and                  analysis were also found preferentially associated with <br /> separated by PAGE. Both complexes showed the same dif-                     this complex by RT-PCR (Fig. 5B). Based on this result, we <br /> ferential subunit composition as described above (Fig.                     calculate a false positive rate of less than 0.11 with a 95% <br /> 5A). The associated RNA was purified, split into equal                     confidence interval of 0.00 &#226;&#8364;&#8220; 0.31. Whereas mRNAs with <br /> portions, and employed in RT-PCR reactions with gene-                      a high enrichment factor (&gt;10, [see additional file 2]) <br /> specific primer pairs.                                                     appeared to be exclusively associated with eIF3e-proA <br />                                                                            (within the limit of detection), those with a lower enrich- <br /> The RT-PCR analysis of a total of fourteen mRNAs                           ment factor (&lt;10) also displayed a low amount of associ- <br /> included eight mRNAs identified in the eIF3e complex,                      ation with the eIF3m complex (Fig. 5B). Conversely, all <br /> five mRNAs identified in the eIF3m complex, and one                        five mRNAs highly enriched in the eIF3m complex as <br /> mRNA found to be associated with both complexes with                       determined by microarray analysis ([see additional file 2]) <br /> similar enrichment factors. These mRNAs varied widely in                   were almost exclusively bound to this complex when ana- <br /> their factors of enrichment and their enrichment ranks as                  lyzed by RT-PCR (Fig. 5B). <br /> indicated in Fig. 5B. Without exception, all eight mRNAs <br />  <br />  <br />                                                                                                                                     Page 9 of 16 <br />                                                                                                               (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Notably, the mRNA encoding the proteasome subunit             viability. Two competing models were proposed to <br /> Rpn5p, which was found in both complexes by                   explain the pleiotropic <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of eif3e deletion <br /> microarray analysis with similar enrichment factors           mutants, which includes slow growth in minimal media <br /> (eIF3e = 5.1; eIF3m = 8.1) was also confirmed as being        and meiotic defects [34,36,38,39,50,51]. In the first <br /> associated with both eIF3 complexes by RT-PCR (Fig. 5B).      model, eIF3e regulates the translation of all transcripts, <br /> These results strongly suggest that the eIF3m and eIF3e       but rare mRNAs may be more affected by a fractional <br /> complexes specifically associate with distinct sets of        reduction in translation efficiency and thus give rise to <br /> mRNAs.                                                        distinct phenotypes. In the second model, eIF3e regulates <br />                                                               the translation of a specific subset of mRNAs encoding <br /> Discussion                                                    proteins whose depletion leads to distinct phenotypes but <br /> PCI proteins define distinct eIF3 complexes                   not lethality. The findings presented here strongly favor <br /> Our results show that Csn6p and Csn7Bp are subunits of        the second model. <br /> fission yeast eIF3. Csn6p was shown to be the eIF3f subu- <br /> nit conserved in all eukaryotes except budding yeast. Con-    Our microarray analysis revealed 106 mRNAs uniquely <br /> sistent with this finding, eIF3f has previously been found    enriched in the eIF3e complex, many being rare, whereas <br /> to be associated with eIF3g, although the role of eIF3f in    2050 transcripts were uniquely enriched in the eIF3m <br /> protein synthesis had not been examined [34]. Our study       complex. The latter finding is consistent with the severe <br /> further identified two distinct eIF3 complexes defined by     protein synthesis defect and lethality of eif3m deletion <br /> the PCI proteins eIF3e and eIF3m (Fig. 3A). No direct         mutants. Whereas these mRNA lists were based on an <br /> equivalent of eIF3m is apparent in eIF3 preparations from     arbitrary cut-off value of three-fold enrichment over <br /> other eukaryotes. Considering the essential function of       mock, this criterion proved stringent enough to confirm <br /> eIF3m in protein synthesis in fission yeast, it is possible   specific association with their respective eIF3 complexes <br /> that other PCI proteins substitute for eIF3m in other         of all 14 mRNAs retested by RT-PCR. Thus, since we calcu- <br /> organisms.                                                    late the false positive rate to be less than 0.11 with 95% <br />                                                               confidence interval 0.00 &#226;&#8364;&#8220; 0.31, the maximum number of <br /> Consistent with this idea is the finding that the PCI         mRNAs listed potentially false positively as enriched in <br /> domain containing GA17, a recently identified protein         the eIF3e complex would be 33 out of 106 (with 95% <br /> copurifying with human eIF3 [30], shows more similarity       confidence). <br /> to eIF3m than to any other PCI protein (K. Hofmann, per- <br /> sonal communication). In addition, the PCI domain con-        However, we consider it unlikely that all mRNAs found <br /> taining CSN subunits 1, 3, 7, and 8 were found to interact    enriched in the eIF3e complex are exclusive translational <br /> with eIF3 subunits in Arabidopsis thaliana and in human       targets of this complex, because the cellular phenotypes of <br /> cells [43,44,46]. In addition, Pci8p/Csn11p, a non-essen-     eif3e mutants are rather discrete. Although many of the <br /> tial PCI subunit of the budding yeast CSN [47,48], copu-      eIF3e-associated mRNAs encode essential proteins, viabil- <br /> rifies with essential eIF3 core subunits [49]. Although the   ity, global protein synthesis, and polysome formation are <br /> functional significance of these interactions is still        not affected in these mutants when they are grown in rich <br /> unclear, our findings raise the intriguing possibility that   media (Fig. 2D, and data not shown). It is therefore likely <br /> they define multiple subclasses of distinct eIF3 complexes.   that eIF3m or other PCI domain proteins can deputize for <br /> Whereas fission yeast appears to rely on an essential eIF3    eIF3e in many cases. In fact, our microarray study revealed <br /> complex classified by eIF3m and a non-essential complex       that a substantial portion of eIF3e-associated mRNAs was <br /> specified by eIF3e, higher eukaryotes may divide the task     also recovered in the eIF3m complex (Fig. 4A). Nonethe- <br /> of mRNA translation among a multitude of eIF3 com-            less, we strongly suspect that inefficient translation of a <br /> plexes. Since the individual PCI proteins defining these      limited set of eIF3e-associated mRNAs contributes to dis- <br /> subcomplexes are expected to be present in substoichio-       tinct cellular phenotypes found in eif3e mutants (see <br /> metric amounts relative to core components, as is eIF3e in    below). <br /> the S. pombe core complex, they may have escaped detec- <br /> tion in bulk eIF3 preparations from higher eukaryotes.        The eIF3e protein and translational control during stress <br />                                                               response <br /> Distinct eIF3 complexes and translational specificity         The eif3e mutants display marked sensitivity to a wide <br /> Although our finding of distinct eIF3 complexes that asso-    variety of cellular stresses, such as osmotic stress, <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br /> ciate with different sets of mRNAs was surprising, circum-    starvation, and low temperature [34,38,39,50,52] In addi- <br /> stantial evidence pointing to their existence was provided    tion, eif3e mutants show synthetic lethality with proteas- <br /> in previous studies. For example, fission yeast eIF3e copu-   ome mutants, which are presumed to accumulate a large <br /> rifies with eIF3 core subunits, and yet, unlike core subu-    load of misfolded proteins [45]. Conversely, overexpres- <br /> nits, eIF3e is not required for global protein synthesis or   sion of eif3e confers resistance to a broad spectrum of <br />  <br />  <br />                                                                                                                  Page 10 of 16 <br />                                                                                             (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                          http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> unrelated drugs, whose only commonality appears to be              While we have so far been unsuccessful in identifying <br /> that they induce cellular stress [38]. Overexpression of           common sequence elements within the upstream regions <br /> eIF3e also activates genes involved in stress defense, such        of mRNAs enriched in the eIF3e complex (data not <br /> as thioredoxin [38]. Interestingly, the mRNA encoding              shown), this analysis was severely hampered by the fact <br /> thioredoxin was also identified as a putative translational        that 5&apos;UTRs are poorly defined in S. pombe. It is therefore <br /> target of eIF3e ([see additional file 2]). Finally, eIF3e          still possible that critical features, including secondary <br /> undergoes relocalization into cytoplasmic foci in                  structure determinants, lie within the 5&apos;UTR or other <br /> response to heat and osmotic stress, thus implicating it in        regions of eIF3e-associated mRNAs. Substantial addi- <br /> stress regulation [35].                                            tional work will be required to delineate the exact mecha- <br />                                                                    nism of how eIF3e may contribute to the translation of <br /> Global protein synthesis is actively switched off in               specific mRNAs and the potential impact of this mecha- <br /> response to cellular stress and <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation               nism on human cancers defective in INT6/eIF3e [58,59]. <br /> (reviewed in Refs. [53,54]). This shut-off protects cells <br /> from proteotoxicity by relieving chaperones of their load          Conclusion <br /> of unfolded client proteins, conserving amino acids for            We provide biochemical evidence for the existence of two <br /> other essential functions, and attenuating the metabolic           distinct eIF3 complexes in fission yeast. These complexes <br /> consequences of protein synthesis. However, long-term              contain an overlapping set of subunits, but are distin- <br /> adaptation to stress conditions, repair and recovery               guished by the PCI proteins eIF3e and eIF3m. Based on <br /> require synthesis of new stress-induced proteins. During           the finding that the distinct eIF3 complexes associate with <br /> the stress response, some mRNAs must be translated in              different mRNAs, we propose that they have different <br /> the presence of a repressed general translation machinery,         translational specificities. <br /> suggesting that select eIFs can escape the global repres- <br /> sion. Our studies raise the intriguing possibility that the        Methods <br /> eIF3e complex has such a specialized function during               Fission yeast techniques <br /> stress response. Lack of eif3e may prevent stress-induced          Preparation of fission yeast cultures, cDNA cloning, yeast <br /> synthesis of its critical translational targets, thus leading to   transformation, PCR-based genomic epitope tagging and <br /> the known stress sensitivity of the mutant. Substantial            gene deletion, mating, and tetrad analysis were carried out <br /> additional work will be required to address this                   using standard methods as described [26]. The strains <br /> possibility.                                                       used in this study are listed in Table 2. <br />  <br /> Potential mechanisms underlying mRNA discrimination by             Affinity purification and mass spectrometry <br /> eIF3 complexes                                                     Epitope-tagged strains (eif3m-proA and eif3e-proA strains) <br /> Several lines of evidence have begun to implicate other            were grown in 12 L yeast extract and supplements (YES) <br /> general translation factors, including eIF2 and eIF4, in           medium to 1.5 optical density (OD) at 595nm <br /> translational specificity [55]. Importantly, these factors         wavelength and harvested by centrifugation. Cell lysates <br /> are downstream of important signaling cascades, includ-            were prepared by bead lysis in a lysis buffer containing 50 <br /> ing the <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing TOR pathway and a variety of              mM Tris-HCl pH 8.0, 150 mM NaCl, 0.5% Triton X-100, <br /> stress-induced kinases (reviewed in Refs. [55,56]).                1 mM dithiothreitol, supplemented with protease inhibi- <br /> Although the details of these regulatory mechanisms are            tors (10 ug/ml leupeptin, 10 ug/ml pepstatin, 5 ug/ml <br /> beginning to emerge, it is still unclear exactly how eIF2          aprotinin and 1mM phenylmethylsulfonylfluoride). <br /> and eIF4 contribute to mRNA specificity.                           Upon centrifugation at 17,000 rpm for 50 min at 4&#194;&#176;C, <br />                                                                    300 mg total protein in a volume of 50 ml was absorbed <br /> Since eIF3 cooperates with the cap-binding eIF4 complex            to 100 ul Dyna beads (Dynal Biotech, Oslo, Norway) cou- <br /> in directing the 5&apos; end of mRNAs to the 40S ribosome, and          pled to whole rabbit IgG (Jackson Immunochemicals, <br /> several eIF3 subunits contain RNA binding motifs, our              West Grove, Pennsylvania, USA). Mock purifications were <br /> data is consistent with a model in which eIF3 complexes            performed in parallel by absorbing lysate from cells lack- <br /> defined by distinct PCI proteins recognize determining             ing any tagged proteins to IgG beads. Beads were collected <br /> features of mRNAs presented to the 40S subunit in concert          by a magnetic device and washed five times in 5 ml lysis <br /> with eIF4. In agreement with this idea is the recent finding       buffer. TEV protease (Invitrogen, Carlsbad, California, <br /> that the human PCI protein eIF3a specifically facilitates          USA) cleavage or elution in 5% SDS solution were used to <br /> the translation of the mRNA encoding ribonucleotide                collect bound proteins. Eluted proteins were separated on <br /> reductase M2 [57]. In addition, A. thaliana eIF3h, a non-          trycine gels or by conventional SDS-polyacrylamide gel <br /> core subunit, was recently shown to be specifically                electrophoresis (SDS-PAGE), and visualized by Coomas- <br /> required for efficient translation of the transcription factor     sie G250 staining. Protein bands were excised from gels, <br /> ATB2 [46].                                                         digested with trypsin (Promega Corporation, Madison, <br />  <br />  <br />                                                                                                                      Page 11 of 16 <br />                                                                                                 (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                          http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Table 2: Strains used in this study <br />  <br />  Name                     Genotype                                                                                      Source <br />  <br />  DS448/1                  leu1-32 ura4-d18 ade6-704           h+                                                        Lab stock <br />  DS448/2                  leu1-32 ura4-d18 ade6-704           h-                                                        Lab stock <br />  C399/3                   leu1-32 ura4-d18 ade6-704           csn5.13myc kan h-                                         Lab stock <br />  C485/3                   leu1-32 ura4-d18 ade6-704           eif3m.13myc kan h-                                        Lab stock <br />  C617/1                   leu1-32 ura4-d18 ade6-704           eif3m.CBP.tev.5proA kan h-                                Lab stock <br />  C642/1                   leu1-32 ura4-d18                    diploid, eif3f/eif3f::ura4                                This study <br />  C642/2                   leu1-32 ura4-d18                    diploid, eif3f/eif3f::ura4                                This study <br />  C648                     leu1-32 ura4-d18 ade6-704           eif3e.cbp.tev.5proA kan h+                                This study <br />  C650/1                   leu1-32 ura4-d18 ade6-704           eif3f::ura4 pRep81.6xhis.myc.eif3f h-                     This study <br />  C650/2                   leu1-32 ura4-d18 ade6-704           eif3m::ura4 pRep81.6xhis.myc.eif3m h-                     This study <br />  C652/1                   leu1-32 ura4-d18 ade6-704           eif3e::ura4 h+                                            This study <br />  C663/1                   leu1-32 ura4-d18 ade6-704           eif3m.gfp kan h-                                          This study <br />  C663/2                   leu1-32 ura4-d18 ade6-704           eif3e.gfp kan h-                                          This study <br />  C665                     leu1-32 ura4-d18 ade6-704           eif3e.13myc kan h-                                        This study <br />  C701                     leu1-32 ura4-d18 ade6-704           eif3b.gfp kan, h-                                         This study <br />  C702                     leu1-32 ura4-d18 ade6-704           eif3b.cbp.tev.5proA kan h-                                This study <br />  <br />  <br />  <br />  <br /> Wisconsin, USA), and subjected to mass spectrometry.            RT-PCR <br /> The eIF3 complexes were analyzed by MALDI-TOF, and by           RNAs associated with the eIF3e and eIF3m complexes <br /> LC-MS/MS at the Harvard NIEHS Center Proteomics Facil-          were purified as described above. Primer design and RT- <br /> ity, according to standard procedures.                          PCR conditions were according to the manual supplied <br />                                                                 with the Platinum Quantitative RT-PCR Thermoscript kit <br /> Protein synthesis and polysome analysis                         (Invitrogen). Primers used for RT-PCR are listed in Table <br /> The eif3f and eif3m conditional shut-off strains were inoc-     3. The primer concentration for actin amplification was <br /> ulated in 10 ml Edinburgh Minimal Media (EMM) with or           0.25 uM. Other primers were used at 1 uM. Total RNA was <br /> without thiamin and cultured at 30&#194;&#176;C for 30 hrs. 100 uCi        obtained as described [19]. <br /> 35S-methionine were added to the cells, which were <br />  <br /> labeled for 30 min and then harvested by centrifugation.        Immunological techniques <br /> Cell lysates were prepared by bead disruption and equal         Immunoprecipitation, immunoblotting, and in vitro <br /> amounts of the lysates were analyzed by SDS-PAGE. 35S-          ubiquitination assays were carried out exactly as described <br /> methionine incorporation was determined by autoradiog-          previously [26]. Antibodies against protein A and ubiqui- <br /> raphy and quantified by PhosphorImager analysis (Bio-           tin were purchased from Sigma and Genzyme (Cam- <br /> Rad, Hercules, California, USA).                                bridge, Massachusetts, USA), respectively. Anti-eIF3d/ <br />                                                                 Moe1p polyclonal antibody was provided by E. Chang <br /> Polysome analysis was performed exactly as described in         (Baylor College of Medicine, Houston, Texas, USA) <br /> Ref. [60]. Briefly, cells were grown in EMM with or with- <br /> out thiamin for 30 hrs. Cycloheximide (100 ug/ml) was           Microarray analysis <br /> added to each culture for 20 min. Cells were harvested          The eIF3 complexes were affinity-purified as described <br /> and lysed in breaking buffer (20 mM Tris/HCl pH 7.4, 50         above, except that 50 U/ml SuperRNasein (Ambion, Aus- <br /> mM NaCl, 30 mM MgCl2, 1 mM DTT, 50 ug/ml cyclohex-              tin, Texas, USA) and 5 mM ribonucleoside vanadyl com- <br /> imide, 0.2 mg/heparin, 50 U/ml SuperRNasein, and pro-           plex (New England Biolab, Beverly, Massachusetts, USA) <br /> tease inhibitors), followed by centrifugation at 13,000 &#195;&#8212; g     were added to the lysis buffer. A mock purification was <br /> for 5 min. 20 A260 nm units of each supernatant in a vol-       performed under identical conditions with lysate from <br /> ume of 500 ul were fractionated on 5-45 % sucrose gradi-        cells lacking any tagged proteins. The methods for extrac- <br /> ents for 3 hrs at 30,000 rpm using a SW40-Ti rotor in a         tion of RNA, cDNA synthesis, RNA amplification, and <br /> Beckman L70 centrifuge. Polysome profiles were obtained         aminoallyl labeling were adopted from previous publica- <br /> by monitoring the absorbance at 254 nm along the gradi-         tions [61,62]. Briefly, after binding and washing, beads <br /> ent using a Bio-Rad fractionator, and the output was            were treated with 3 U/ul DNase I, followed by elution <br /> recorded using a Bio-Rad UV detector.                           with 5 % SDS, extraction with phenol/chloroform and <br />                                                                 then ethanol precipitation. The associated RNA was <br />  <br />  <br />                                                                                                                            Page 12 of 16 <br />                                                                                                       (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Table 3: RT-PCR primers                                        Control Cy5-labeled aminoallyl-cDNA was made by oligo <br />                                                                dT primed reverse transcription of total S. pombe RNA. <br /> Gene Name         Sequence                       Product <br />                                                  Size (bp) <br />                                                                Labeled cDNAs were hybridized onto glass slide microar- <br /> actinF            GAGCTTCCTGATGGTCAAGT              200        rays. Microarrays were made by spotting purified ds-PCR <br /> actinR            GGATACATAGTGGTACCACC                         products 500 to 1200 bp in <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> onto glass slides coated <br /> cdc16F            AAGTTAGCGCCCAAATCACC              150        with aminopropylsilane (Erie Scientific, Portsmouth, <br /> cdc16R            TTTTGAATGCCCCCCCACGG                         New Hampshire, USA). The PCR products used repre- <br /> gar2F             GGTGCCATTGAGAAACCTTC              150 <br />                                                                sented 4988 predicted ORFs and transcripts as annotated <br /> gar2R             CCGAACCTTCAAAGAAATCA <br /> pof6F             TCTGATCGGCCTAAGCTGTC              150 <br />                                                                by the Sanger Centre [63] and are located at the 3&apos; end of <br /> pof6R             CAATCTTGCAAATTCAACTC                         predicted genes to maximize sensitivity in detecting oligo- <br /> rpn5F             TGAGAAGCAAGTTCGTCAGG              160        dT primed reverse transcription products. Descriptions of <br /> rpn5R             TGAAACAATCGATGACAAGT                         primer pairs and the features on the microarrays are avail- <br /> rps8-1F           AAGCGTATTCACGAGGTCCG              150        able at the Longhorn Array Database [64]. <br /> rps8-1R           CAACTCGTTGTTAGAAGGGT <br /> scd1F             TCAGAGTTGGCTGCTTTCTT              150 <br />                                                                Median pixel intensities for each spot obtained from <br /> scd1R             ATCCATTGTGTGCCCTGTTC <br /> SPAC1348.05F      CTTAGTGAACAGTTTGGAAG              150        hybridizations of eIF3-associated RNA ([see additional <br /> SPAC1348.05R      TGATAAACCAACGGATCCGA                         file 1, sheet 1]) were corrected by local background sub- <br /> SPAC21E11.04F     GAGACATCACCTGCTCCAGA              150        traction and divided by the corresponding values <br /> SPAC21E11.04R     TTTGCTCCGGTGACTAGGTG                         obtained from a mock purification. This resulted in the <br /> SPBC12C2.06F      TCTGTTCCCAAACCTCAAGC              150        factor of enrichment of mRNAs bound to eIF3 complexes <br /> SPBC12C2.06R      ATCGATTTTTGCACCTTTAG                         over mock. The mRNAs were ranked according to their <br /> SPBC1683.01F      GGCCGTAAATTTGTCTACGG              150 <br />                                                                factor of enrichment ([see additional file 1, sheet 2]). An <br /> SPBC1683.01R      ACCACCAATACCAACACCAA <br /> SPBC36B7.03F      CTCTCAATTAAATTTCACCC              150        enrichment factor of three was chosen as an arbitrary <br /> SPBC36B7.03R      AGGAGTACCGTATAAAGCAT                         threshold. We found 2464 mRNAs were enriched greater <br /> SPCC70.05cF       GTACCTGGAAATAACTCTCC              150        than three-fold over mock in the eIF3m complex, while <br /> SPCC70.05cR       CATAGCCTTTTCTAAGAGAT                         we observed 520 mRNAs enriched in the eIF3e complex <br /> tf2-12F           AAGCATGTACCAGAGATAGG              150        ([see additional file 1, sheet 3]). Comparison of the data <br /> tf2-12R           TGAATCACCTAGAAGAATTA                         sets revealed 414 mRNAs that occurred in both com- <br /> ung1F             ACTTTGGAGAGTTCTTGGTT              150 <br />                                                                plexes, 2050 mRNAs uniquely enriched in the eIF3m <br /> ung1R             TGGAGTATGATGTGACCATG <br />                                                                complex and 106 mRNAs that were exclusively enriched <br /> F = Forward; R = Reverse                                       in the eIF3e complex ([see additional file 1, sheet 4; addi- <br />                                                                tional file 2]). A corresponding list of the 106 mRNAs <br />                                                                most highly and uniquely enriched in the eIF3m complex <br />                                                                was assembled ([see additional file 2]) and used to select <br />                                                                specific mRNAs for RT-PCR analysis. <br /> reversely transcribed using Superscript II reverse tran- <br /> scriptase (Gibco-BRL Division of Invitrogen) with 1 uM         The competitive Cy5-labeled total cDNA provided clear <br /> oligo-dT(15)-T7 primer (5&apos;-AAA CGA CGG CCA GTG AAT             microarray feature identification and a guide to relative <br /> TGT AAT ACG ACT CAC TAT AGG CGC-3&apos;) and 1 uM tem-              sequence abundance in the pre-purification sample from <br /> plate switch primer (5&apos;-AAG CAG TGG TAT CAA CGC                which the relative expression rank of each mRNA was <br /> AGA GTA CGC GGG-3&apos;). Full-length double stranded               determined ([see additional file 1, sheet 2]). In this study, <br /> (ds)-cDNA was synthesized using Advantage Polymerase           data were not adjusted by ratio normalization to total <br /> (Clontech, BD Biosciences, Mountain View, California,          RNA signals or local area normalization functions. The <br /> USA). Newly synthesized ds-cDNA was passed through             microarray data were deposited with the public ArrayEx- <br /> Bio-6 columns (Bio-Rad) and dried. RNA amplification           press (E-MEXP-208) [65] and GEO [66] databases. <br /> was performed with the T7 megascript kit (Ambion). <br /> Amplified RNA was purified using the RNeasy kit (Qia-          List of abbreviations <br /> gen, Hilden, Germany) and reversely transcribed with           CSN COP9 signalosome <br /> Superscript II reverse transcriptase in the presence of ami- <br /> noallyl-dUTP in reactions primed with oligo-dT (Invitro-       Cul Cullin <br /> gen). After RNAse H digestion and purification on <br /> QIAquick columns (Qiagen), the aminoallyl-cDNA was             eIF Eukaryotic initiation factor <br /> labeled with Cy3, and purified on QIAquick columns. <br />                                                                EMM Edinburgh Minimal Media <br />  <br />  <br />                                                                                                                   Page 13 of 16 <br />                                                                                              (page number not for citation purposes) <br />  BMC Biology 2005, 3:14                                                                        http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> GFP Green fluorescent protein                                                A. Houseman for assistance with microarray data analysis. This study was <br />                                                                              funded by National Institute of Health (NIH) grant GM59780 to DAW, NIH <br /> IgG Immunoglobulin G                                                         grant P40 RR16320 to JL, by the NIEHS Center Grant ES-00002, and the <br />                                                                              NIEHS Training Grant ES07155. <br />  <br /> LC MS/MS Liquid chromatography tandem mass <br />                                                                           <br /> spectrometry                                                   <br /> MALDI-TOF Matrix-assisted laser desorption ionization     <br /> time-of-flight                                         <br /> MPN Mpr1/Pad1 N-terminal                            <br /> OD Optical density                                   <br /> PAGE Polyacrylamide gel electrophoresis         <br /> PCI Proteasome, COP9 Signalosome, eIF3               <br /> ProA Protein A                                               <br /> RT-PCR Reverse transcription-polymerase chain reaction     <br /> TEV Tobacco etch virus                               <br /> YES Yeast extract and supplements (growth media)                     <br /> Authors&apos; contributions                                                  <br /> CZ performed the bulk of the experiments shown in this                     <br /> study. FA and SW prepared various strains used in this                <br /> study. SK and ARI performed mass spectrometry. AO per-                      <br /> formed the microarray hybridizations, and JL participated                  <br /> in the design of the microarray studies and in data analy-                    <br /> sis. DAW contributed to study design and drafting of the                       <br /> manuscript.                                                                    <br />   <br /> </body></html>