<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization of Protein Hubs by Inferring <br /> Interacting Motifs <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Protein Interactions <br />  <br />  <br /> The characterization of protein interactions is essential for understanding biological systems. While genome-scale <br /> methods are available for identifying interacting proteins, they do not pinpoint the interacting motifs (e.g., a domain, <br /> sequence segments, a binding site, or a set of residues). Here, we develop and apply a method for delineating the <br /> interacting motifs of hub proteins (i.e., highly connected proteins). The method relies on the observation that proteins <br /> with common interaction partners tend <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> interact with these partners through a common interacting motif. The sole <br /> input for the method are binary protein interactions; neither sequence nor structure information is needed. The <br /> approach is evaluated by comparing the inferred interacting motifs with domain families defined for 368 proteins in <br /> the Structural Classification of Proteins (SCOP). The positive predictive value of the method for detecting proteins with <br /> common SCOP families is 75% at sensitivity of 10%. Most of the inferred interacting motifs were significantly <br /> associated with sequence patterns, which could be responsible for the common interactions. We find that yeast hubs <br /> with multiple interacting motifs are more likely to be essential than hubs with one or two interacting motifs, thus <br /> rationalizing the previously observed correlation between essentiality and the number of interacting partners of a <br /> protein. We also find that yeast hubs with multiple interacting motifs evolve slower than the average protein, contrary <br /> to the hubs with one or two interacting motifs. The proposed method will help us discover unknown interacting motifs <br /> and provide biological insights about protein hubs and their roles in interaction networks. <br /> Citation: Aragues R, Sali A, Bonet J, Marti-Renom MA, Oliva B (2007) Characterization of protein hubs by inferring interacting motifs from protein interactions. PLoS Comput <br /> Biol 3(9): e178. doi:10.1371/journal.pcbi.0030178 <br />  <br />  <br />                                                                                             scopy. These methods are not always applicable and thus the <br /> Introduction <br />                                                                                             number of known 3-D atomic structures of proteins and their <br />    Protein&#226;&#8364;&#8220;protein interactions play a central role in many                                 complexes is limited. As a result, accurate and general <br /> cellular processes, ranging from signal transduction to                                     computational methods for identifying motifs involved in <br /> formation of cellular macrostructures and cell cycle control                                protein&#226;&#8364;&#8220;protein interactions are needed. <br /> [1&#226;&#8364;&#8220;3]. Recently, several techniques such as two-hybrid assays                                 Recently, several methods [19&#226;&#8364;&#8220;25] have been developed to <br /> [4&#226;&#8364;&#8220;6] and af&#239;&#172;&#65533;nity puri&#239;&#172;&#65533;cations followed by mass spectrometry                                describe protein&#226;&#8364;&#8220;protein interactions in terms of interacting <br /> [7&#226;&#8364;&#8220;9] have enabled large-scale identi&#239;&#172;&#65533;cation of protein&#226;&#8364;&#8220;                                    protein domains, as de&#239;&#172;&#65533;ned in the Structural Classi&#239;&#172;&#65533;cation of <br /> protein interactions. While these efforts provide rich lists of                             Proteins (SCOP) [26], PFAM [27], and InterPro [28] databases. <br /> interacting proteins, they do not produce information about                                 However, while these methods &#239;&#172;&#65533;nd interactions between <br /> the speci&#239;&#172;&#65533;c interfaces involved in each interaction.                                        prede&#239;&#172;&#65533;ned protein domains, interactions between unde&#239;&#172;&#65533;ned <br />    Proteins interact through a limited set of interface types                               domains remain undetected. Sequence-based methods over- <br /> [3,10,11]. These interfaces are usually key determinants of the                             come this problem by identifying sequence signatures that <br /> function. Therefore, narrowing down protein&#226;&#8364;&#8220;protein inter- <br />                                                                                             consistently co-occur in pairs of interacting protein sequen- <br /> actions to interactions between speci&#239;&#172;&#65533;c protein components <br />                                                                                             ces [29], while structure-based methods can predict the amino <br /> (e.g., a domain, sequence segments, a binding site, or a set of <br />                                                                                             acid residues that are in contact during a protein&#226;&#8364;&#8220;protein <br /> residues) is important for a more accurate characterization of <br /> the function of proteins and their complexes. Identifying the <br /> protein interfaces that mediate interactions may also be                                    Editor: Ruth Nussinov, National Cancer Institute, United States of America, and Tel <br />                                                                                             Aviv University, Israel <br /> useful for the prediction of unknown protein&#226;&#8364;&#8220;protein <br /> interactions [12,13], for homology-based protein annotation                                 Received April 19, 2007; Accepted July 27, 2007; Published September 14, 2007 <br /> methods [14], and for relating gene essentiality and network                                A previous version of this article appeared as an Early Online Release on July 30, <br /> topology [15].                                                                              2007 (doi:10.1371/journal.pcbi.0030178.eor). <br />    Traditionally, the description of protein interactions in                                Copyright: &#195;&#8220; 2007 Aragues et al. This is an open-access article distributed under <br /> terms of the interacting components has been based on                                       the terms of the Creative Commons Attribution License, which permits unrestricted <br />                                                                                             use, distribution, and reproduction in any medium, provided the original author <br /> protein structural domains [16], protein functional sites [17],                             and source are credited. <br /> and protein patches [18]. However, fully characterizing                                     Abbreviations: iMotif, interacting motif; PDB, Protein Data Bank; PIANA, Protein <br /> protein surfaces that are in contact with each other during                                 Interactions and Network Analysis; SCOP, Structural Classification of Proteins <br /> an interaction requires the determination of the structure of                               * To whom correspondence should be addressed. E-mail: mmarti@cipf.es (MAMR); <br /> protein complexes by X-ray crystallography or NMR spectro-                                  boliva@imim.es (BO) <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                            1761                                   September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                           Protein Hubs Characterization by iMotifs <br />  <br />  <br /> Author Summary                                                                structure nor sequence information is required to assign <br />                                                                               iMotifs to proteins. <br /> Recent advances in experimental methods have produced a deluge                   Two main objectives have been addressed in this work. The <br /> of protein&#226;&#8364;&#8220;protein interactions data. However, these methods do               &#239;&#172;&#65533;rst objective was to demonstrate whether protein inter- <br /> not supply information on which specific protein regions are                  actions alone can be used to infer interacting motifs. The <br /> physically in contact during the interactions. Identifying these              positive predictive value of our method in detecting proteins <br /> regions (interfaces) is fundamental for scientific disciplines that           with common SCOP families was 75% at sensitivity of 10%, <br /> require detailed characterizations of protein interactions. In this <br />                                                                               and the Spearman correlation coef&#239;&#172;&#65533;cient between the <br /> work, we present a computational method that identifies groups of <br /> proteins with similar interfaces. This is achieved by relying on the          number of iMotifs assigned to proteins and the number of <br /> observation that proteins with common interaction partners tend to            interfaces found by Kim et al. [15] was 0.57. The second <br /> interact through similar interfaces. The proposed method retrieves            objective was to examine if the conclusions on protein hubs <br /> protein interactions from public data repositories and groups                 of Kim et al. [15] hold for our iMotifs assignments. The results <br /> proteins that share a sensible number of interacting partners.                demonstrate that protein hubs with multiple iMotifs are more <br /> Proteins within the same group are then labeled with the same                 likely to be essential than hubs with one or two iMotifs and <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;interacting motif&#226;&#8364;&#8482;&#226;&#8364;&#8482; identifier (iMotif). The evaluation performed           that protein hubs with multiple iMotifs evolve slower than the <br /> using known protein domains and structural binding sites suggests             average protein in the dataset, as opposed to hubs with one or <br /> that the method is better suited for proteins with multiple <br />                                                                               two iMotifs. <br /> interacting partners (hubs). Using yeast data, we show that the <br /> cellular essentiality of a gene better correlates with the number of <br /> interacting motifs than with the absolute number of interactions.             Results <br />                                                                               Proteins with Common Interaction Partners Tend to Share <br />                                                                               a SCOP Domain <br /> interaction, but require information about the structures of                    The basic assumption behind this work is that proteins with <br /> both proteins [30&#226;&#8364;&#8220;33]. Recently, Kim et al. used known                        overlapping sets of interaction partners tend to interact with <br /> protein interactions and structures to characterize the                       those partners through a common interacting motif. The <br /> interfaces between two interacting proteins [15]. They found                  validity of this assumption was tested on a nonredundant set <br /> that some previously accepted relationships between network                   of 368 proteins with known SCOP domains (Material and <br /> topology and genomic features [34&#226;&#8364;&#8220;36] are actually more                       Methods). Although SCOP does not classify proteins by their <br /> re&#239;&#172;&#8218;ective of the number of distinct binding interfaces. For                   interfaces, SCOP domains were used as surrogates for iMotifs <br /> example, highly connected proteins in the network (i.e., hubs)                because protein interaction types can be de&#239;&#172;&#65533;ned by the <br /> with multiple interfaces are twice as likely to be essential as               domains in the interacting proteins [43]. <br /> hubs with one or two interfaces. The &#239;&#172;&#65533;ndings of Kim and                         We found the number of common interaction partners (N) <br /> coworkers clarify some previous analyses that related the                     to be a good indicator of the probability of two proteins <br /> observed essentiality of hubs with their high number of                       having a domain in the same SCOP family, especially for <br /> interacting partners [34,37] or with their interactions to other              highly connected proteins (Figure S1). For example, 73% of <br /> hubs [38]. Kim et al. also demonstrated that the evolutionary                 protein pairs with 50&#226;&#8364;&#8220;60 common interaction partners <br /> rate is signi&#239;&#172;&#65533;cantly lower for multi-interface hubs than for                  shared a SCOP domain. We also studied other metrics to <br /> the average protein, but not so for hubs with one or two                      measure the similarity between two sets of interaction <br /> interfaces.                                                                   partners, but none of them outperformed N at the <br />    Here, our basic assumption is that proteins with over-                     identi&#239;&#172;&#65533;cation of protein pairs with a common domain family <br /> lapping sets of interacting partners tend to interact with the                (Figures S1 and S2A, and Table S1). <br /> common partners through the same interacting motif, such as                     It is worth noting that our assumption relies on the binary <br /> a domain, sequence segments, a binding site, or a set of                      nature of the input interactions. Two proteins tend to have a <br /> residues. A similar assumption has been previously used to                    common interacting motif only if they share direct physical <br /> annotate protein sequences [14,39&#226;&#8364;&#8220;41]. We &#239;&#172;&#65533;rst tested this                    interactions with the same partner(s). However, the likelihood <br /> assumption based on databases of protein interactions [42]                    of two proteins sharing a SCOP domain was lower by solely <br /> and protein domains de&#239;&#172;&#65533;ned in SCOP [26], observing that the                   using yeast two-hybrid experimental data, a detection <br /> assumption holds true for highly connected proteins (i.e.,                    method that is more likely to contain binary protein <br /> hubs in a protein&#226;&#8364;&#8220;protein interaction network). Building on                   interactions than other experimental methods [44] (Figure <br /> this validation, we then developed a method for identifying                   S2B). <br /> interacting motifs (iMotifs), which has been implemented <br /> within the protein&#226;&#8364;&#8220;protein interaction framework and                          Delineating Interacting Motifs <br /> integration engine PIANA (Protein Interactions and Network                       Based on the observation that highly connected proteins <br /> Analysis) [42]. iMotifs are not required to be of any particular              with common interaction partners tend to interact with them <br /> structural type or size, thus allowing us to characterize                     through a common interacting motif, we have developed a <br /> proteins and their interactions at different levels of reso-                  method that groups proteins with similar interacting motifs <br /> lution, ranging from full proteins to small binding sites. In                 (Figure 1 and Material and Methods section). Brie&#239;&#172;&#8218;y, the <br /> contrast to other methods, our approach is not limited to                     procedure is carried out in four steps: 1) build a protein&#226;&#8364;&#8220; <br /> &#239;&#172;&#65533;nding prede&#239;&#172;&#65533;ned classes of interacting motifs, such as SCOP                  protein interaction network; 2) initialize a cluster interaction <br /> domains or PROSITE functional sites, and can be used to                       network by assigning each protein of the network to a cluster; <br /> identify unknown interacting motifs. Moreover, the sole                       3) iteratively create new clusters by fusing similar clusters <br /> input for our method is binary protein interactions; neither                  (allowing a protein to be in more than one cluster) until the <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org                1762                           September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                  Protein Hubs Characterization by iMotifs <br />  <br />  <br />  <br />  <br /> Figure 1. Assigning iMotifs to Proteins and Identifying iMotif&#226;&#8364;&#8220;iMotif Interactions <br /> First, the protein interaction network is built. Second, a cluster interaction network is created by placing each protein in a different cluster. Third, <br /> clustering is performed until the similarity score drops below a certain threshold. Fourth, an iMotif <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> is assigned to each cluster with more than one <br /> protein, and iMotif assignments and interactions are derived. <br /> doi:10.1371/journal.pcbi.0030178.g001 <br />  <br />  <br />  <br /> similarity score drops below a prede&#239;&#172;&#65533;ned threshold; and 4)                       determining the size, sequence, or structure of that feature <br /> label with a different interacting motif identi&#239;&#172;&#65533;er (iMotif) each                 (Figure 2A). Thus, an iMotif can be an interface consisting of <br /> cluster with more than one protein and derive iMotif&#226;&#8364;&#8220;iMotif                      a set of domains or only a speci&#239;&#172;&#65533;c constellation of a small <br /> interactions from the clustered network. In step 3), the                         number of residues (Figure 2B). <br /> similarity score between two clusters is their number of <br /> common interacting partners in the cluster interaction                           Method Evaluation <br /> network (N). Assigning an iMotif to a group of proteins                            The de&#239;&#172;&#65533;nition of iMotifs depends on a similarity metric <br /> simply establishes that they have a certain feature that allows                  and its threshold. Thus, different thresholds or metrics <br /> them to interact with the same set of partners, without                          produce different iMotifs, corresponding to different levels <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                  1763                              September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                  Protein Hubs Characterization by iMotifs <br />  <br />  <br />  <br />  <br /> Figure 2. <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> of an Interacting Motif (iMotif) <br /> The <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of an iMotif depends on the minimum number of common partners required in order to consider the given binary protein interactions <br /> mediated through a common interacting motif. <br /> (A) From the protein interaction network perspective, proteins with common partners (two in the example provided) are considered to interact with <br /> these partners through a similar feature, and, therefore, are classified as being of the same iMotif. <br /> (B) The same process is shown from a structural perspective: proteins interacting through a similar feature (regardless of the feature being two <br /> structural domains or a single binding site) are considered to have a common iMotif. To further illustrate the method, we also describe a sample iMotif <br /> assignment for prothrombin (UniProt <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> THRB_HUMAN) (Figure S3). <br /> doi:10.1371/journal.pcbi.0030178.g002 <br />  <br />  <br /> of resolution in the description of protein&#226;&#8364;&#8220;protein inter- <br /> actions. For example, the method can be applied at the <br /> resolution of domains from SCOP [26], and PFAM [27], or at <br /> the higher resolution of functional sites from PROSITE [45]. <br /> In this section, we have evaluated the method on a non- <br /> redundant set of proteins (Material and Methods) for three <br /> different tasks: (i) detecting proteins with common SCOP <br /> domain families; (ii) predicting SCOP domain&#226;&#8364;&#8220;domain <br /> interactions observed in the Protein Data Bank (PDB) [46]; <br /> and (iii) predicting the number of distinct binding interfaces <br /> as de&#239;&#172;&#65533;ned by Kim et al. [15]. Therefore, in the evaluation, <br /> iMotifs effectively represent SCOP family domains (for the <br /> &#239;&#172;&#65533;rst two tasks) and structural binding interfaces (for the third <br /> task). <br />  <br /> Detecting Proteins with Domains in the Same SCOP <br /> Family <br />   We evaluated the ability of the method to detect proteins <br /> with a domain in the same SCOP family (Methods). Using an N <br /> threshold of 30 common interaction partners, our method <br /> achieves a positive predictive value of ;75%, sensitivity of                    Figure 3. Performance of the Method in Detecting Proteins with <br /> ;10%, and applicability of ;20% (Figure 3). The positive                        Common SCOP Families <br /> predictive value drops to ;50% for N of 15, indicating that                     The positive predictive value, sensitivity, and applicability (Methods) are <br />                                                                                 plotted as a function of the number of common interaction partners <br /> the accuracy of our method in detecting proteins with                           threshold (N) used for the clustering. <br /> common SCOP family domains proportionally decreases with                        doi:10.1371/journal.pcbi.0030178.g003 <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 1764                               September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                              Protein Hubs Characterization by iMotifs <br />  <br />  <br />  <br />                                                                                     Table 1. Protein Essentiality and Predicted iMotifs <br />  <br />                                                                                     Protein Sets             Proteins           Essential      Percent        p-Value <br />                                                                                                              Tested for         Proteins       Essential <br />                                                                                                              Essentiality <br />  <br />                                                                                     Entire proteome          6,018              1,116          19% <br />                                                                                     All in PIANA             5,034              1,047          21% <br />                                                                                     Singlish-iMotif hubs        90                 27          30%            All-singlish: 0.04 <br />                                                                                     Multi-iMotif hubs          507                262          52%            All-multi: <br />                                                                                                                                                               2.2 3 10 16; <br />                                                                                                                                                               singlish-multi: <br />                                                                                                                                                               1.5 3 10 4 <br />  <br />  <br />                                                                                     The fraction of yeast proteins that are products of essential genes [58] was calculated for <br />                                                                                     the entire proteome, singlish-iMotif hubs (one or two iMotifs), and multi-interface hubs. <br />                                                                                     iMotifs were assigned by applying the method to all yeast hubs in PIANA with N set to 20. <br />                                                                                     The p-value of the difference between the whole dataset and singlish- and multi-iMotif <br />                                                                                     hubs (all-singlish and all-multi) and the singlish and multi-iMotif hubs (singlish-multi) was <br />                                                                                     calculated using the Fisher&#226;&#8364;&#8482;s exact test for count data. <br />                                                                                     doi:10.1371/journal.pcbi.0030178.t001 <br />  <br />  <br />  <br />                                                                                     method (Figure 4). The number of protein interfaces indeed <br />                                                                                     correlates with the number of predicted iMotifs per protein <br /> Figure 4. Correlation between the Number of Binding Interfaces and the <br /> Number of iMotifs                                                                   (e.g., for N of 20, rs is 0.57 and p-value 0.01). The number of <br /> Each point corresponds to a protein from the test set for which a number            iMotifs assigned to proteins by our method tends to be higher <br /> of binding interfaces was assigned by Kim et al. [15], and a number of              than the number of binding sites de&#239;&#172;&#65533;ned by Kim et al. This <br /> iMotifs was inferred with N set to 20. Both variables were found to be              might be attributed to two factors: (i) current structural data <br /> significantly correlated (rs is 0.57 and p-value is 0.01). The correlation <br /> between the number of interfaces and the number of iMotifs is                       do not contain all possible protein&#226;&#8364;&#8220;protein interactions, <br /> significant for all N values lower than 23 (Figure S5).                             resulting in an underestimation of the number of binding <br /> doi:10.1371/journal.pcbi.0030178.g004                                               sites assigned by the method in [15], and (ii) the lack of <br />                                                                                     coverage of the interactome space, which results in an <br />                                                                                     overestimation of the number of iMotifs per protein assigned <br /> the number of common interacting partners. Therefore, the                           by our method. The second factor is addressed by using <br /> method should be preferentially applied to assigning inter-                         sequence information to merge similar iMotifs (below). <br /> acting motifs to highly connected proteins. The growth of the <br /> interactome data [47,48] is likely to make the approach more                        iMotifs Assignments for Hub Proteins <br /> applicable in the future. Nevertheless, the applicability can                         Using an N threshold of 20, our method assigned 12,342 <br /> already be increased at the expense of the positive predictive                      iMotifs to 2,014 of the 5,571 hub proteins in PIANA (i.e., <br /> value by using other similarity metrics (Table S1). We provide                      proteins with 20 or more interaction partners), resulting on <br /> a complete list of iMotif assignments for the test set (Table S2).                  average in 8.6 iMotifs per hub. The percentage of hubs with <br />                                                                                     one or two iMotifs was 46% (241 hubs had one iMotif; 689 <br /> Predicting Domain&#226;&#8364;&#8220;Domain Interactions                                               hubs had two iMotifs). We studied the correlation between <br />   Domain&#226;&#8364;&#8220;domain interactions can be predicted from the                              the number of iMotifs assigned to a hub and its number of <br /> iMotif&#226;&#8364;&#8220;iMotif interactions found by the method (Materials                           interactions, &#239;&#172;&#65533;nding no relationship between the two <br /> and Methods). We evaluated the accuracy of these predictions                        variables (Spearman correlation coef&#239;&#172;&#65533;cient is  0.002 with p- <br /> with respect to domain interactions in the PDB. Our method                          value 0.94). A complete list of iMotif assignments for all hub <br /> achieves a positive predictive value of ;65% for ;5% of the                         proteins in PIANA is in Table S3 and the number of iMotifs <br /> proteins in the test set (Figure S4), suggesting that the method                    per hub is in Table S4. <br /> can be applied to the prediction of domain&#226;&#8364;&#8220;domain <br /> interactions when a suf&#239;&#172;&#65533;ciently large and varied sample of                          Essentiality and Number of iMotifs Are Correlated in Hub <br /> protein interactions is known. However, with the available                          Proteins <br /> interaction data, other methods that rely on both interaction                          Similarly to Kim and co-workers&#226;&#8364;&#8482; results, [15], we found <br /> networks and prede&#239;&#172;&#65533;ned domains [19&#226;&#8364;&#8220;22] may be better                                that yeast hubs with multiple iMotifs are more likely to be <br /> suited than our approach for predicting domain&#226;&#8364;&#8220;domain                               essential than those with one or two iMotifs (singlish-iMotif) <br /> interactions.                                                                       (Table 1). Furthermore, we observed a correlation (rs is 0.61 <br />                                                                                     and p-value is 1.64 3 10 5) between the number of iMotifs in <br /> Predicting the Number of Binding Interfaces                                         yeast hubs and the fraction of essential proteins (Figure 5A). <br />   Kim et al. used protein 3-D structures and binary protein                         We compared the correlation between iMotifs and essential- <br /> interactions to make inferences about the number of binding                         ity to the correlation between the number of interactions of <br /> interfaces of proteins [15]. We tested whether there is a                           hubs and essentiality to con&#239;&#172;&#65533;rm that the &#239;&#172;&#65533;rst was not a direct <br /> correlation between the number of binding interfaces found                          consequence of the second (Figure 5B). These results suggest <br /> in their work and the number of iMotifs predicted by our                            that the number of iMotifs predicted for a protein could be <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                     1765                                      September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                                          Protein Hubs Characterization by iMotifs <br />  <br />  <br />  <br />  <br /> Figure 5. Correlation between the Number of iMotifs and Protein Essentiality <br /> Proteins from PIANA were binned according to their number of iMotifs (A) and to their number of interactions (B), and the fraction of essential proteins <br /> was calculated for each bin. Bins with only one protein were not considered for calculating the correlations. <br /> (A) Correlation between the number of iMotifs assigned to yeast hub proteins ( 20 interactions) in PIANA and the fraction of essential proteins (rs is <br /> 0.61 and p-value is 1.6 3 10 5). iMotifs were assigned to yeast hubs using an N threshold of 20. <br /> (B) Correlation between the number of interactions of yeast hub proteins in Figure 5A and the fraction of essential proteins (rs is 0.51 and p-value is 1.1 <br /> 3 10 6). <br /> doi:10.1371/journal.pcbi.0030178.g005 <br />  <br />  <br /> used for selecting biologically relevant candidates for gene                                     occurrence of the <span id='am-88' about='xsp:pattern' typeof='owl:Thing'>pattern</span> in the iMotif with respect the whole <br /> deletion experiments.                                                                            dataset. As shown on Figure 6, 80% of iMotifs had a speci&#239;&#172;&#65533;c <br />                                                                                                  sequence pattern contained in at least 74% of their proteins <br /> Multi-iMotif Hubs Evolve Slower Than Other Proteins;                                             (using a p-value cutoff of 10 8). A list with the best sequence <br /> Singlish-iMotif Hubs Do Not                                                                      pattern for each iMotif is provided in Table S5. Interestingly, <br />    A common measure of evolutionary rate is the dN/dS ratio                                      a similar analysis based on Pfams assignments to iMotifs <br /> (the ratio of nonsynonymous to synonymous substitutions)                                         showed a different trend (i.e., very few iMotifs had most of <br /> [49]. Kim et al. found that multi-interface hubs have a lower                                    their proteins described by a Pfam). For example, as shown on <br /> evolutionary rate than the average protein in their data, but                                    Figure S6, only 10% of all iMotifs had a speci&#239;&#172;&#65533;c Pfam in at <br /> the same was not true for singlish-interface hubs. Our results                                   least 28% of their proteins (p-value cutoff of 10 8). Such a <br /> are in agreement with their &#239;&#172;&#65533;ndings. Multi-iMotif hubs, in <br />                                                                                                  difference can be explained by the fact that many inter- <br /> contrast to singlish-iMotif, evolve signi&#239;&#172;&#65533;cantly slower than <br />                                                                                                  actions are carried out by short sequence patches [3,51], while <br /> the average protein in our dataset (Table 2). However, the <br />                                                                                                  Pfam families usually consist of long structured protein <br /> evolutionary rate difference between multi- and singlish- <br />                                                                                                  regions. <br /> iMotif hubs (i.e., 0.062 and 0.056, respectively) was not found <br />                                                                                                    As indicated above, incompleteness in interaction data may <br /> to be signi&#239;&#172;&#65533;cant (p-value of 0.21). <br />                                                                                                  result in arti&#239;&#172;&#65533;cially high numbers of iMotifs. This over- <br /> Extracting Sequence Patterns from iMotifs                                                        estimation can be reduced by merging iMotifs with a common <br />   Sequence patterns for each iMotif were generated using the                                     sequence pattern (Material and Methods). Fusing iMotifs <br /> PRATT program [50] (Methods). Brie&#239;&#172;&#8218;y, PRATT identi&#239;&#172;&#65533;es                                            based on sequence pattern similarity decreased the average <br /> sequence patterns common to a set of sequences. In this work,                                    number of iMotifs per hub from 8.6 to 4.2. This reduction, in <br /> we selected signi&#239;&#172;&#65533;cant sequence patterns for each iMotif by                                      turn, increased the correlation between the number of <br /> maximizing the number of proteins within the iMotif that                                         binding sites from Kim et al. [15] and the number of iMotifs <br /> contained the pattern. The signi&#239;&#172;&#65533;cance (i.e., p-value) of a                                      in the test set proteins (Spearman correlation coef&#239;&#172;&#65533;cient was <br /> sequence pattern assigned to an iMotif depends on the                                            0.59 with p-value of 0.001). <br />  <br />  <br />  <br /> Table 2. Protein Evolutionary Rate and Predicted iMotifs <br />  <br /> Analysis                   Entire                All in           Singlish-iMotif              p-Value                  Multi-iMotif             p-Value               p-Value <br />                            Proteome              PIANA            Hubs                         (All-Singlish)           Hubs                     (All-Multi)           (Singlish-Multi) <br />  <br /> Evolutionary rate          0.077                 0.074            0.062                        0.12                     0.056                    8.1 3 10 11           0.21 <br />  <br />  <br /> The average evolutionary rate of yeast proteins [49] was calculated for the entire proteome, single-iMotif hubs, and multi-interface hubs. iMotifs were assigned by applying the method to <br /> all yeast proteins in PIANA with N set to 20. The p-value of the difference between the whole dataset and singlish- and multi-iMotif hubs (all-singlish and all-multi) and the singlish- and <br /> multi-iMotif hubs (singlish&#226;&#8364;&#8220;multi) was calculated using the Mann-Whitney U two-sided test. <br /> doi:10.1371/journal.pcbi.0030178.t002 <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                                 1766                                      September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                               Protein Hubs Characterization by iMotifs <br />  <br />  <br />  <br />  <br /> Figure 6. Sequence Patterns in iMotifs <br /> Relationship between the percentage of iMotifs for which a significant sequence pattern was found and the percentage of proteins within the iMotif <br /> that contained the pattern. Three different significance cutoffs were used for associating sequence patterns to iMotifs: 10 5 (long dashed line), 10 8 <br /> (solid line), and 10 10 (short dashed line). <br /> doi:10.1371/journal.pcbi.0030178.g006 <br />  <br />  <br /> Discussion                                                                     that the method works better for highly connected proteins <br />                                                                                and using high values for N. Moreover, our approach is not <br />    We described, implemented, and evaluated a method that                      limited to &#239;&#172;&#65533;nding prede&#239;&#172;&#65533;ned classes of protein components <br /> relies solely on binary protein interactions to identify <br />                                                                                and thus allows us to predict new types of interacting motifs. <br /> interacting motifs (iMotifs) and their interactions. Our <br />                                                                                For example, an iMotif can be mapped to a prede&#239;&#172;&#65533;ned class <br /> approach obtained high positive predictive value for identi- <br />                                                                                (e.g., a SCOP domain or a PROSITE functional site) by <br /> fying proteins with domains from the same SCOP family and <br />                                                                                examining the known classes assigned to proteins with that <br /> predicting domain&#226;&#8364;&#8220;domain interactions. We also analyzed <br />                                                                                iMotif. Therefore, iMotifs that remain unmapped are likely <br /> hub proteins and their properties based on the number of <br />                                                                                candidates for unknown classes. Such predictions may prove <br /> iMotif assigned to them, obtaining similar &#239;&#172;&#65533;ndings to those in <br />                                                                                useful for target selection in structural genomics. <br /> an independent approach that rely on protein structure <br />                                                                                   Relying solely on experimentally detected interactions <br /> information [15]. <br />                                                                                affects the accuracy of our method. It has been shown that <br />    Recent estimates suggested that only one-&#239;&#172;&#65533;fth of inter- <br /> action types are known [43]. Therefore, current knowledge of                   high-throughput experiments have limited reliability and <br /> protein structures is not suf&#239;&#172;&#65533;cient to describe all protein                    that many of the detected interactions are probably not <br /> interaction types. Our approach, in contrast to other                          direct (i.e., they are carried out through a third protein) or do <br /> previously described methods, accomplishes three different                     not even exist (i.e., false positives) [52]. However, we did not <br /> objectives: (i) it predicts the number of different iMotifs in a               observe an improvement by solely using interactions from <br /> protein, (ii) it classi&#239;&#172;&#65533;es proteins by their predicted iMotifs,                yeast two-hybrid assays (Figure S2B), the high-throughput <br /> and (iii) it predicts interactions between the iMotifs. The                    method that is best suited to detect direct interactions. As <br /> method can identify iMotifs independently of structural or                     more interaction data becomes available, we will reexamine <br /> sequence information; it can assign an iMotif to two                           the effects of restricting the method to employ interactions <br /> structural domains or two iMotifs to a single domain. Since                    speci&#239;&#172;&#65533;cally labeled as &#226;&#8364;&#732;&#226;&#8364;&#732;direct&#226;&#8364;&#8482;&#226;&#8364;&#8482; [6]. One way of avoiding these <br /> the resolution at which iMotifs describe protein interfaces                    limitations is to calculate similarity scores using families of <br /> depends on the similarity metric used and the threshold                        proteins instead of absolute numbers of protein partners. <br /> applied by the method, iMotif assignments can be used to                       This will prevent assigning the same iMotif to proteins that <br /> infer whether the interaction is mediated through multiple,                    have many common partners but all of them belong to a <br /> single, or partial domains. On the one hand, setting a high                    single protein family. Removal of redundancy from the sets of <br /> threshold on the number of common interaction partners (N)                     partners indeed increases the percentage of identi&#239;&#172;&#65533;ed <br /> will assign few iMotifs to reduced sets of proteins (i.e., very                protein pairs with a common domain family (Figure S7). <br /> speci&#239;&#172;&#65533;c and restrictive iMotifs). On the other hand, using low                    The combination of iMotif assignments with sequence <br /> N thresholds will assign the same iMotif to broad numbers of                   search methods identi&#239;&#172;&#65533;ed speci&#239;&#172;&#65533;c sequence patterns in <br /> proteins (i.e., very unspeci&#239;&#172;&#65533;c and general iMotifs). We showed                 iMotifs. We found that most iMotifs had a signi&#239;&#172;&#65533;cant <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                1767                             September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                     Protein Hubs Characterization by iMotifs <br />  <br />  <br /> sequence pattern that was contained in most of the iMotif                          pj) interacts with those clusters that contain a partner of pj in the <br /> proteins. These patterns, which could be responsible for the                       protein interaction network. <br />                                                                                       Third, iteratively create new clusters by fusing the most similar <br /> iMotif proteins common interactions, could then be used to:                        clusters until the similarity score drops below a prede&#239;&#172;&#65533;ned threshold. <br /> (i) localize the iMotif in the protein sequence, (ii) assign                       Two clusters are similar if they share a minimum number of common <br /> iMotifs to proteins for which no interaction data is yet                           interacting partners (N). Thus, the similarity score between two <br />                                                                                    clusters is their number of common partners in the cluster <br /> available, and (iii) predict interactions between proteins that                    interaction network. Other similarity metrics were considered, but <br /> contain patterns assigned to two interacting iMotifs.                              none outperformed the use of N (Figure S1). When fusing two <br />    Our iMotif assignments are similar to those obtained using                      clusters, the resulting cluster inherits interactions that were common <br />                                                                                    to both fused clusters. Since proteins may have multiple interfaces, all <br /> an independent approach, which relies not only on known                            initial clusters (from step 2) remain in the cluster interaction network <br /> protein&#226;&#8364;&#8220;protein interactions, but also on protein structure                        even after being fused to another cluster. Interactions between non- <br /> information [15]. In agreement with the results of Kim et al.,                     initial clusters are not considered for calculating the similarity scores. <br /> we observe different properties between hubs with multiple                            Fourth, each cluster with more than one protein is labeled with a <br />                                                                                    different interacting motif identi&#239;&#172;&#65533;er (iMotif), and that iMotif is assigned <br /> iMotifs (multi-iMotif) and hubs with one or two iMotifs                            to all proteins within that cluster. iMotif&#226;&#8364;&#8220;iMotif interactions are then <br /> (singlish-iMotif). In particular, we &#239;&#172;&#65533;nd that (i) multi-iMotif                     derived from interactions in the cluster interaction network where both <br /> hubs are more likely than singlish-iMotif hubs to be essential                     sides of the interaction have been labeled with an iMotif identi&#239;&#172;&#65533;er. <br />                                                                                       For example (Figure 1), a proteome of six proteins (namely A, B, C, <br /> for cell viability, and (ii) multi-iMotif hubs, in contrast to                     D, E, and F) forms a network of interactions that connects proteins A <br /> singlish-iMotif hubs, evolve slower than the average protein.                      with B, C, and D, and protein E with B, C, D, and F (step 1). Our <br /> Furthermore, we have also observed a correlation between                           method starts by creating a cluster interaction network from the <br />                                                                                    network of protein interactions (i.e., six clusters with seven <br /> the number of iMotifs of a hub and its essentiality for cell                       interactions) (step 2). Next, the clusters that share the largest number <br /> <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>survival</span>. The properties observed for hubs with respect to                         of common interactions are fused (i.e., clusters 1 and 5, with three <br /> their number of iMotifs may re&#239;&#172;&#8218;ect the difference between                          common interactions, are fused into a new cluster 7). This step is then <br /> proteins with multiple simultaneously possible interactions                        repeated until the maximum similarity score between the clusters <br />                                                                                    drops below a prede&#239;&#172;&#65533;ned threshold (i.e., N &#194;&#188; 2 common interactions). <br /> (multi-iMotif hubs are probably involved in permanent                              Thus, the iterative process will run for another iteration creating a <br /> complexes) and proteins with multiple exclusive interactions                       new cluster (cluster 8) by fusing clusters 2, 3, and 4, which have two <br /> (for singlish-iMotif hubs involved in transient interactions).                     common interactions (step 3). Once the iterative process is &#239;&#172;&#65533;nished, <br />                                                                                    the method assigns iMotif identi&#239;&#172;&#65533;ers to all proteins in clusters with <br /> This is in agreement with the previous observation that                            more than one protein (i.e., proteins A and E in cluster 7 share iMotif <br /> interfaces of transient protein&#226;&#8364;&#8220;protein interactions are less                      1, and proteins B, C, and D in cluster 8 share iMotif 2) (step 4). <br /> restricted in evolution than interfaces in permanent com-                          Moreover, iMotif&#226;&#8364;&#8220;iMotif interactions are then derived from the <br />                                                                                    cluster interaction network (i.e., one interaction between iMotif 1 <br /> plexes [53].                                                                       and iMotif 2). <br />    Our results extend the &#239;&#172;&#65533;ndings and conclusions of Kim and                          Figure 2 illustrates iMotif assignments from a network perspective <br /> co-workers [15] to proteins of unknown structure. Thus,                            (Figure 2A) and from a structural perspective (Figure 2B). A more <br /> inferring interacting motifs from protein interactions is                          detailed description of the algorithm is provided as pseudocode in <br />                                                                                    Figure S8. <br /> likely to be helpful for providing biological insights about                          Test set and evaluation procedure. We have evaluated the method <br /> hubs for which no structural information is available.                             on a test set created by selecting proteins (i) with at least &#239;&#172;&#65533;ve <br />                                                                                    experimentally detected interactions, (ii) with at least 80% of their <br />                                                                                    sequence covered by the domains de&#239;&#172;&#65533;ned in SCOP, and (iii) that did <br /> Materials and Methods                                                              not introduce a redundancy bias in the evaluation (i.e., if any two <br />                                                                                    sequences had a sequence identity greater than 30%, a BLAST e-value <br />    Protein interactions. Protein&#226;&#8364;&#8220;protein interactions from DIP                     smaller than 10 5, and the alignment had at least 30 residues, the <br /> 2006.01.16 [54], MIPS 2006.01 [55], HPRD 2005.09.13 [56], BIND                     shortest member of the pair was not selected). The &#239;&#172;&#65533;nal set contained <br /> 2006.01 [57], and two recent high-throughput experiments [5,6] were                368 sequences (Table S6). Due to the restrictions imposed, the test set <br /> integrated using PIANA version 1.2 [42], allowing us to work with a                contains many proteins related to the proteosome and the ribosome. <br /> large set of 363,571 interactions between 42,040 proteins. PIANA                      The SCOP family assignment was evaluated by considering as <br /> represents protein interactions as a network where the nodes are                   positive assignments those proteins found by the method to have a <br /> proteins and the edges are interactions between the proteins. In such              common iMotif with the query protein. Among these positives, we <br /> a network, a set of proteins linked to protein pj (i.e., physically                de&#239;&#172;&#65533;ne as true positives those proteins that have a common SCOP <br /> interacting with pj) is named &#226;&#8364;&#732;&#226;&#8364;&#732;partners of pj&#226;&#8364;&#8482;&#226;&#8364;&#8482;. In such a network, we            family <span id='am-87' about='Thesaurus:code' typeof='owl:Thing'>code</span> with the query protein. Moreover, we de&#239;&#172;&#65533;ne as false <br /> de&#239;&#172;&#65533;ne hubs as proteins with 20 or more partners. The average                       negatives the proteins that have the same SCOP family code as the <br /> number of interactions per hub in our dataset was ;49. PIANA builds                query protein but were not found by the method to share an iMotif. <br /> the protein interaction network by retrieving partners for an initial                 iMotif&#226;&#8364;&#8220;iMotif interaction predictions were evaluated against inter- <br /> set of proteins. To avoid a positive bias in the method evaluation,                acting SCOP families obtained from the PDB. Two SCOP domains were <br /> interactions inferred from 3-D structures were not used in this work.              considered to interact if they were co-crystallized and had at least two <br />    Structural domains and protein binding interfaces. Protein domain               atoms within 5 A&#203;&#353; distance. Because we are interested in domain <br /> assignments and classi&#239;&#172;&#65533;cation were obtained from the SCOP release                  interactions at the protein&#226;&#8364;&#8220;protein interaction level, we excluded <br /> 1.69 [26]. Here, domains are de&#239;&#172;&#65533;ned at the SCOP family level. Thus,                intrachain interactions from this set. Our method creates a list of <br /> domain&#226;&#8364;&#8220;domain interactions refer to SCOP family interactions. The                  putative domain&#226;&#8364;&#8220;domain interactions for each predicted iMotif&#226;&#8364;&#8220;iMotif <br /> number of protein binding interfaces for hub proteins was obtained                 interaction by assuming that all domains of the query protein with one <br /> from the Structural Interaction Network 2.0 [15].                                  iMotif interact with all domains of proteins with the other iMotif. In this <br />    Essential proteins and evolutionary rates. A list of ORFs essential             context, we de&#239;&#172;&#65533;ne as positive any iMotif&#226;&#8364;&#8220;iMotif interaction where the <br /> for the survival of the yeast cell was obtained from the Saccharomyces             query protein is involved. A positive is then considered a true <br /> Genome Deletion Project [58]. The evolutionary rates (dN/dS) of yeast              prediction if at least one of its putative domain&#226;&#8364;&#8220;domain interactions <br /> proteins were taken from the adjusted values given by Wall et al. [49].            is observed in the PDB. Finally, false negatives are interactions observed <br />    Assigning iMotifs to proteins and &#239;&#172;&#65533;nding iMotif&#226;&#8364;&#8220;iMotif interac-                 in the PDB for SCOP families of the query protein that do not appear in <br /> tions. The assignment of iMotifs to a set of proteins is carried out in a          any list of putative SCOP family interactions. <br /> four-step procedure (Figure 1):                                                       To avoid biases in the evaluation, only proteins from the test set <br />    First, build the protein interaction network.                                   (before removing redundancy) and their SCOP families were <br />    Second, initialize a cluster interaction network (i.e., nodes are               considered when counting positives and negatives. The positive <br /> clusters that contain one or more proteins, and edges are interactions             predictive value is de&#239;&#172;&#65533;ned as the number of true positives over the <br /> between clusters) by assigning each protein of the protein interaction             total number of positives, and sensitivity is the number of true <br /> network to a different cluster. Each cluster (containing one protein               positives over the sum of true positives and false negatives. The <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                    1768                               September 2007 | Volume 3 | Issue 9 | e178 <br />                                                                                                                    Protein Hubs Characterization by iMotifs <br />  <br /> positive predictive value and sensitivity were calculated with respect             Figure S2. The Percentage of Protein Pairs Having a Domain of the <br /> to the similarity score threshold used for stopping the clustering. We             Same SCOP Family Is Plotted as a Function of Their Similarity Scores <br /> also de&#239;&#172;&#65533;ne the applicability of the method as the percentage of                    (Grouped in Ranges of 10 Units), Using the Same Parameters as in <br /> proteins with at least one positive under a given threshold.                       Figure S1 but Introducing New Restrictions <br />    Extracting sequence patterns from iMotifs. For each group of                    (A) Proteins that have more than 70 interactions are ignored when <br /> protein sequences with a given iMotif, sequence signatures were                    performing the analysis. <br /> generated using the PRATT program [50], a software tool capable of                 (B) Only interactions from y2h are used. <br /> &#239;&#172;&#65533;nding &#239;&#172;&#8218;exible sequence patterns from a set of unaligned sequences. <br /> Parameters were set to produce patterns covering a maximum of 15                   Found at doi:10.1371/journal.pcbi.0030178.sg002 (94 KB TIF). <br /> residues with no more than three consecutive unspeci&#239;&#172;&#65533;ed positions                  Figure S3. Sample iMotif Assignment <br /> (gaps) and a maximum of one &#239;&#172;&#8218;exible (of variable <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span>) gap region. <br /> The number of nonredundant patterns for all iMotifs was 80,654.                    (A) Superposition of the prothrombin and the pancreatic trypsin <br /> Next, all patterns were searched against all proteins in PIANA with at             inhibitor structures (PDB IDs 1BTH and 2HPQ) shows an interaction <br /> least one interaction (a dataset of 42,040 sequences) using the                    through the SCOP family domain Eukaryotic proteases (in red). <br /> ps_scan program [59].                                                              (B) The structure of the anionic trypsin II interaction with the <br />    The signi&#239;&#172;&#65533;cance (i.e., p-value) of the association between a                    pancreatic trypsin inhibitor (PDB ID 1BRB) also shows an <br /> sequence pattern and an iMotif was assessed using the binomial                     interaction through the SCOP family domain Eukaryotic proteases <br /> distribution, based on the occurrence of the pattern inside the iMotif             (in red). <br /> with respect the whole dataset [60]:                                               Found at doi:10.1371/journal.pcbi.0030178.sg003 (233 KB TIF). <br />                                        <br />                                      M n                                           Figure S4. Performance of the Method in Predicting SCOP Domain&#226;&#8364;&#8220; <br />                  p   value &#195;&#176;Mn&#195;&#382; &#194;&#188;        p &#195;&#176;1   p&#195;&#382;&#195;&#176;M n&#195;&#382;                            Domain Interactions <br />                                      n <br />                                                                                    The positive predictive value, sensitivity, and applicability are plotted <br /> where M is the number of protein sequences within an iMotif, n                     as a function of the number of common interacting partners <br /> represents the number of proteins within the iMotif that contain the               threshold used for the clustering. The positive predictive value and <br /> sequence pattern, and p is the probability of &#239;&#172;&#65533;nding a protein from                sensitivity using a trivial approach are also shown (thin lines). The <br /> the whole dataset that contains the same pattern (i.e., the number of <br />                                                                                    applicability for the trivial approach is ;70%. Sensitivity is highly <br /> proteins containing the pattern divided by the number of proteins in <br />                                                                                    dependent on the group of proteins for which an iMotif&#226;&#8364;&#8220;iMotif <br /> the whole dataset). <br />                                                                                    interaction can be predicted at a given threshold: if a protein with a <br />    Best iMotif sequence pattern. In this work, the best sequence <br />                                                                                    prediction has a SCOP code with multiple interactions in the PDB, <br /> pattern for each iMotif (Table S5) was considered to be the pattern <br />                                                                                    the sensitivity obtained can vary greatly from one threshold to <br /> that maximized the number of proteins in the iMotif that had the <br />                                                                                    another. Moreover, we compared our method with the trivial <br /> pattern. Two additional considerations were taken into account for <br />                                                                                    approach of creating putative lists of domain&#226;&#8364;&#8220;domain interactions <br /> selecting the best pattern for an iMotif: (i) the sequence pattern <br />                                                                                    by assuming that all domain families of proteins in the test set <br /> should be found in at least 70% of proteins within the iMotif and (ii) <br />                                                                                    interact with all domain families of their interaction partners. The <br /> the p-value of the pattern should be lower than 10 8. <br />                                                                                    positive predictive value for this trivial approach was 33%, which is <br />    Merging iMotifs based on sequence commonalities. Interacting <br />                                                                                    below that of our method for thresholds higher than 15. <br /> motifs were merged by means of an agglomerative hierarchical <br /> clustering. Two iMotifs were considered to be similar if they had a                Found at doi:10.1371/journal.pcbi.0030178.sg004 (93 KB TIF). <br /> common sequence pattern when applying a p-value cutoff of 1 3 10 5 <br /> and requiring the pattern to be found in at least 70% of proteins in               Figure S5. Spearman Correlation Coef&#239;&#172;&#65533;cient between the Number of <br /> both iMotifs. Using more stringent p-value cutoffs did not produce                 Interfaces and the Number of iMotifs Is Plotted as a Function of <br /> any iMotif fusions for proteins from the test set.                                 Different N Thresholds <br />    Assigning Pfams to iMotifs. Hidden Markov Models from the                       Due to the limited number of iMotif assignments with stringent N <br /> Pfam-A database [61] were assigned to all proteins with at least one               thresholds, correlations become nonsigni&#239;&#172;&#65533;cant (i.e., p-value . 0.05) <br /> known interaction (a dataset of 42,040 sequences) using the HMMER                  for N thresholds higher than 22. <br /> package [62]. The p-value for each HMM in relation with each iMotif                Found at doi:10.1371/journal.pcbi.0030178.sg005 (35 KB TIF). <br /> was calculated using the binomial distribution, based on the <br /> occurrence of the Pfam inside the iMotif and in the whole dataset                  Figure S6. Relationship between the Percentage of iMotifs for Which <br /> (above).                                                                           a Signi&#239;&#172;&#65533;cant Pfam Was Detected by Sequence Search and the <br />    Statistical tests. All correlations were measured using the Spear-              Percentage of Proteins within the iMotif That Contained the Pfam <br /> man rank correlation coef&#239;&#172;&#65533;cient (rs). The assessment of whether two                Three different signi&#239;&#172;&#65533;cance cutoffs were used for &#239;&#172;&#65533;nding Pfams <br /> binomial samples of essentiality observations are signi&#239;&#172;&#65533;cantly differ-             associated with iMotifs: 10&#226;&#8364;&#8220;5 (long dashed line), 10&#226;&#8364;&#8220;8 (plain line), and <br /> ent was calculated using Fisher&#226;&#8364;&#8482;s test. The assessment of whether two              10&#226;&#8364;&#8220;10 (short dashed line). <br /> non-Gaussian samples of evolutionary rate observations come from <br /> the same distribution was calculated using the Mann-Whitney U two-                 Found at doi:10.1371/journal.pcbi.0030178.sg006 (87 KB TIF). <br /> sided test. Correlations and differences in the observations were                  Figure S7. The Percentage of Protein Pairs Having a Domain of the <br /> considered signi&#239;&#172;&#65533;cant for p-values lower than 0.05. All tests were                 Same SCOP Family Is Plotted as a Function of Their Similarity Scores <br /> performed using the implementation provided by R [63].                             (Grouped in Ranges of 10 Units), Using the Same Parameters as in <br />                                                                                    Figure S1 but Introducing a New Restriction: Redundancy Was <br />                                                                                    Removed from the Sets of Partners To Avoid Arti&#239;&#172;&#65533;cial Increase or <br /> Supporting Information                                                             Decrease of the Score Caused by Groups of Homolog Proteins <br /> Figure S1. The Percentage of Protein Pairs Having a Domain of the                  The procedure followed to remove redundancy was the same as the <br /> Same SCOP Family Is Plotted as a Function of Their Similarity Scores               one used for creating the evaluation set. We observe a signi&#239;&#172;&#65533;cant <br /> (Grouped in Ranges of 10 Units)                                                    improvement for all metrics with respect to Figure S1. <br /> To measure the likelihood of two proteins pi and pj having a common                Found at doi:10.1371/journal.pcbi.0030178.sg007 (95 KB TIF). <br /> interacting motif, we de&#239;&#172;&#65533;ned four different similarity metrics: 1) N: <br /> the number of interaction partners that are common to pi and pj                    Figure S8. Pseudocode of the Algorithm Implemented for Assigning <br /> (long dashed line); 2) Rmax: the ratio between N and the number of                 iMotifs to Proteins from a Protein&#226;&#8364;&#8220;Protein Interaction Network <br /> partners of the protein with more partners (bold line); 3) Rmin: the               Found at doi:10.1371/journal.pcbi.0030178.sg008 (345 KB TIF). <br /> ratio between N and the number of partners of the protein with fewer <br /> partners (circles); 4) Rave: the average of metrics Rmax and Rmin (dotted          Table S1. Number of Protein Pairs under Each Similarity Score Range <br /> line). For each score obtained using the similarity metrics described              for Metrics Described in Figure S1 <br /> above, the percentage of protein pairs within that score range is                  In parentheses, the number of pairs with at least one domain within <br /> plotted. For example, we observed that using N as the similarity                   the same SCOP family is indicated. We observe that metrics such as <br /> metric, 73% of proteins with 50&#226;&#8364;&#8220;60 common interaction partners                     Rmin outperform N at detecting a higher number of protein pairs <br /> shared a SCOP domain.                                                              with a domain within the same SCOP family, but this is done at the <br />                     expense of decreasing the accuracy of the method. <br />  <br />                     <br /> Table S2. Complete List of iMotifs Assignments for Proteins in the                <br /> Test Set <br /> The number of common interaction partners (N) was set to 15. First <br /> column is the iMotif identi&#239;&#172;&#65533;er. Second column is the number of                      <br /> proteins within the iMotif. Subsequent colums are the proteins <br />                                                                                      <br /> within the iMotif. Proteins are identi&#239;&#172;&#65533;ed using UniProt entry names <br />                                                                                 <br /> and NCBI GI identi&#239;&#172;&#65533;ers. In parentheses, &#226;&#8364;&#732;&#226;&#8364;&#732;yes&#226;&#8364;&#8482;&#226;&#8364;&#8482;, &#226;&#8364;&#732;&#226;&#8364;&#732;no&#226;&#8364;&#8482;&#226;&#8364;&#8482;, and &#226;&#8364;&#732;&#226;&#8364;&#732;-&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                              <br /> indicate whether the proteins had a domain within the same SCOP                      <br /> family. <br />                                                                                       <br /> Found at doi:10.1371/journal.pcbi.0030178.st002 (251 KB TXT).                         <br /> Table S3. Complete List of iMotifs Assignments for Hub Proteins in              <br /> PIANA <br />                                                                                    <br /> The number of common interaction partners (N) was set to 20. See                          <br /> legend on Table S2.                                                         <br /> Found at doi:10.1371/journal.pcbi.0030178.st003 (2.4 MB TXT).                            <br /> Table S4. Number of iMotifs Assigned to Each Hub in PIANA                         <br /> Found at doi:10.1371/journal.pcbi.0030178.st004 (38 KB TXT). <br />                                                                                    <br /> Table S5. Best Sequence Pattern for Each iMotif from Table S3                          <br /> First column is the iMotif identi&#239;&#172;&#65533;er. Second column is the pattern in                      <br /> PROSITE format. Third column is the fraction of proteins within the                       <br /> iMotif that have the pattern. Fourth column is the p-value of the                     <br /> pattern.                                                                                  <br /> Found at doi:10.1371/journal.pcbi.0030178.st005 (704 KB TXT).                         <br /> </body></html>