<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, June 1998, p. 3191&#226;&#8364;&#8220;3200                                                                          Vol. 18, No. 6 <br /> 0270-7306/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology <br />  <br />  <br />  <br />                A Chimeric Subunit Yeast Transcription Factor IIIC <br />                           Forms Subcomplex t95 <br />         NATHALIE MANAUD, ROSALI&#194;&#180;A ARREBOLA, BE <br />                                              &#194;&#180; NE <br />                                                 &#194;&#180; DICTE BUFFIN-MEYER, OLIVIER LEFEBVRE, <br />              HARTMUT VOSS,&#226;&#8364;&#160; MICHEL RIVA, CHRISTINE CONESA, AND ANDRE &#194;&#180; SENTENAC* <br />                 Service Biochimie et Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, CEA/Saclay, F-91191 Gif-sur-Yvette Cedex, France <br />                        Received 20 January 1998/Returned modification 25 February 1998/Accepted 5 March 1998 <br />  <br />              The multisubunit yeast transcription factor IIIC (TFIIIC) multifunctional protein required promoter <br />           recognition, transcription factor IIIB recruitment, chromatin antirepression. We report isolation and <br />           characterization TFC7, essential gene encoding 55-kDa polypeptide, t55, present affinity-purified <br />           TFIIIC. t55 chimeric protein generated ancient chromosomal rearrangement. Its C-terminal half is <br />           essential cell viability sufficient ensure TFIIIC function DNA binding transcription assays. <br />           The N-terminal half nonessential highly similar putative yeast protein encoded another <br />           chromosome cyanobacterial protein unknown function. Partial deletions N-terminal domain <br />           impaired t55 function high temperature media containing glycerol ethanol, suggesting link <br />           PolIII transcription metabolic pathways. Interestingly, t55 found, TFIIIC <br />           subunit t95, protein complex distinct TFIIIC play role regulation <br />           PolIII transcription, possibly relation cell metabolism. <br />  <br />  <br />    In eucaryotic cells, transcription variety small                  The molecular structure yeast TFIIIB TFIIIC has <br /> genes conducted RNA polymerase III (PolIII) re-                     investigated. Yeast TFIIIB comprises com- <br /> quires auxiliary factors. For yeast tRNA gene (tDNA)                  ponents: TBP, TATA box binding protein, also <br /> activation, preinitiation complexes assembled defined                required transcription PolI PolII, additional <br /> order upstream transcription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (18, 27,                  polypeptides, TFIIIB70/BRF1 TFIIIB90/B0, iden- <br /> 52). Transcription factor IIIC (TFIIIC) plays primary role               tified protein-DNA cross-linking (6). Together TBP, <br /> this multistep complex assembly binding intragenic                  TFIIIB70 able bind TFIIIC-tDNA complexes (29). The <br /> promoter elements tRNA genes (the A B blocks). Yeast                   resulting complex competent recruit PolIII after <br /> TFIIIC remarkably large multisubunit factor                  assembly TFIIIB90 (29, 30). Purified yeast TFIIIC com- <br /> protein subassemblies, named tA tB, distinct                    prises polypeptides, 138, 131, 95, 91, 60, 55 kDa (5, <br /> DNA binding properties, visualized electron                    16, 43). Purification TFIIIC near homogeneity, protein- <br /> microscopy free DNA-bound form (46),                  DNA cross-linking (5, 9, 16), coimmunoprecipitation ex- <br /> cleaved limited proteolysis (37). The tB domain binds <br />                                                                               periments (13, 44) suggested largest polypeptides <br /> tightly B block (37) shown display the <br />                                                                               subunits TFIIIC, suggestion confirmed by <br /> properties enhancer binding proteins (11). Binding the <br />                                                                               gene cloning (2, 34, 36, 44, 48). The 138- 95-kDa compo- <br /> tA domain A block weaker B block <br /> dependent. Once bound, TFIIIC promotes binding tran-                   nents (t138 t95), located tB tA, respectively, are <br /> scription factor IIIB (TFIIIB) upstream transcription                  thought DNA binding subunits, be <br /> start site (6, 26, 28, 31). The process similar yeast 5S               specifically cross-linked tDNA probe mapped at <br /> RNA gene activation, TFIIIC assembly depen-                    level B block A block, respectively (5, 9, 13, <br /> dent binding transcription factor IIIA (TFIIIA)                16). t91 recently shown cooperate t138 TFIIIC- <br /> the internal promoter sequence. TFIIIB itself does bind                tDNA binding (2) mapped 39 location of <br /> detectably TATA-less PolIII genes but, assembled                  TFIIIC-5S RNA gene complexes (9). t131 stands the <br /> TFIIIC, interacts intimately DNA sufficient,             TFIIIB-assembling subunit based upstream gene loca- <br /> in yeast system, directing accurate initiation PolIII              tion, shown protein-DNA cross-linking (5, 6), direct <br /> during multiple rounds transcription vitro (28, 31).                    interaction TFIIIB70 TFIIIB90 (12, 32, 45). On <br /> Hence, TFIIIB initiation factor required activa-               hand, little known smallest polypeptides, <br /> tion PolIII genes, TFIIIC TFIIIA act                    60 55 kDa, reproducibly copurify yeast <br /> assembly factors. However, shown TFIIIC                 TFIIIC activity. Both proteins six <br /> multifunctional protein, involved promoter recog-                 polypeptides isolated TFIIIC-tDNA complexes (16). A <br /> nition TFIIIB recruitment displacement                 55-kDa polypeptide located photo-cross-linking exper- <br /> nucleosomes relieve repression transcription chro-               iments t95, opposite sides DNA helix, <br /> matin (10).                                                                   vicinity A block tRNA genes (5, 6). The 60-kDa <br />                                                                               polypeptide cross-linked DNA. <br />                                                                                  We report cloning identification TFC7, an <br />   * Corresponding author. Mailing address: Service Biochimie et            essential gene encoding 55-kDa subunit TFIIIC. Anal- <br /> de Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, CEA/Saclay, F-91191 Gif-sur-Yvette Ce- <br /> dex, France. Phone: 33 1 69 08 22 36. Fax: 33 1 69 08 47 12. E-mail: <br />                                                                               ysis deletion mutants showed C-terminal half <br /> sentenac@dsvidf.cea.fr.                                                       t55 necessary TFIIIC transcriptional activity. We <br />   &#226;&#8364;&#160; Present address: LION Biosciences AG, 69120 Heidelberg, Ger-              t55 interacts t95 subunits <br /> many.                                                                         present distinct protein complexes. <br />  <br />                                                                        3191 <br />  3192       MANAUD ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />                          MATERIALS AND METHODS                                         oligonucleotides, NM8 (59-TCCTTTTCAATACATATGTATCCTTACGACGTT <br />                                                                                        CCTGATTATGCCATGGTGGTGAACAC) NM7 (59-TCAGCGGGATCCT <br />    Yeast strains methods. YNN281 3 YNN282 (22) used gene                   TACATAGGGCGGACATTGC), used mutagenesis. NM8 contains the <br /> disruption. Preparation media, tetrad dissection, yeast methods           epitope coding sequence (boldface letters) nucleotides com- <br /> were performed standard techniques (3). Plasmids harboring modified alleles         plementary TFC7 DNA create NdeI NcoI restriction sites. NM7 <br /> of TFC7 gene used transform YNM2 haploid strain (ade2-101              complementary TFC7 harbors BamHI restriction site (boldfaced) just <br /> lys2-801 ura3-52 trp1-D1 his3-D200 tfc7-D::HIS3 pNM2). The modified copies          stop codon. The amplified DNA fragment cloned pGEM-T <br /> TFC7 substituted wild-type TFC7 plasmid shuffling plates con-           vector (Promega), creating pNM8. The NdeI/BamHI NcoI/BamHI fragments <br /> taining 5-fluoro-orotic acid. Viable strains isolated 30&#194;&#176;C tested     pNM8 cloned pET28a (Novagen), pAS2, pACT2 (21), <br /> growth 37 16&#194;&#176;C.                                                                 creating pNM11, pAS-t55, pACT-t55, respectively. <br />    Purification immunoprecipitation TFIIIC. TFIIIC purified starting           An NdeI restriction site inserted TFC7 gene promoter PCR- <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> 30 g (wet weight) Saccharomyces cerevisiae cells following procedure       mediated mutagenesis plasmid pNM2 oligonucleotides NM5 (59-CA <br /> described Huet et al. (25). Immunoprecipitation performed described          GCCATTGACCCCAAAATGAGAA) NM9 (59-CGTGTTCACCACCAT <br /> by Ossipow et al. (39). Cells expressing wild-type epitope-tagged versions       ATGTATTGAAAAGGA). The resulting PCR product cloned the <br /> t138, t131, t95 (25, 34, 36) harvested exponential phase, crude     pGEM-T vector, creating pNM10. The SphI/NdeI fragment pNM10 the <br /> extracts prepared described Huet et al. (25), protease          NdeI/BamHI fragment pNM8 sequentially cloned YCplac22 (19), <br /> inhibitors (O-complete; Boehringer) extraction buffer containing 20 mM             creating pNM12. The t55-DN1, t55-DN2, t55-DN3, t55-DC mutants were <br /> HEPES (pH 7.5), 50 mM CH3COOK, 1 mM EDTA, 1 mM dithiothreitol (DTT),                   constructed deletions nucleotides 49 288, 286 834, 34 834, 835 <br /> and 10% glycerol used. Proteins precipitated ammonium sulfate,          1335, respectively, epitope-tagged version TFC7. These mutants were <br /> resuspended 5% original crude extract volume dialysis buffer (25 mM       obtained cleavage pNM8 BsaBI/XbaI, XbaI/EcoRV, StyI/EcoRV, and <br /> HEPES [pH 7.5], 100 mM KCl, 0.1 mM EDTA, 0.25 mM DTT, 10% glycerol),                   EcoRV/BamHI, respectively, treatment Klenow DNA polymerase mung <br /> and dialyzed twice 2 h time 4&#194;&#176;C 250 volumes            bean nuclease, religation, sequencing junction. The resulting NdeI/ <br /> buffer. Typically, 10 g (wet weight) yeast cells yielded 2 ml dialyzed extract   BamHI DNA fragments cloned pNM12, place wild-type <br /> containing 15 30 mg protein/ml, estimated Bradford analysis (8). Per       tagged version TFC7, creating pNM24, pNM25, pNM26, pNM15, respec- <br /> assay, 1.2 mg mouse monoclonal antihemagglutinin (HA) antibodies (53)           tively. The EcoRI/SalI DNA fragments pNM12, pNM15, pNM24, pNM25, <br /> incubated 30 min 10&#194;&#176;C 20 ml magnetic beads (8 3 108 beads/ml         pNM26 cloned pUN45 used transform YNM2 yeast cells. <br /> phosphate-buffered saline containing 0.1% bovine serum albumin [BSA]) coated              Expression TFC7 Escherichia coli. Recombinant TFC7 protein (rTFC7p) <br /> with rat monoclonal antibodies directed mouse immunoglobulin G2b               tagged N-terminal end histidines HA epitope was <br /> (Dynal M450). After extensive washing phosphate-buffered saline containing          expressed plasmid pNM11 E. coli BL21(pLysS). Crude extract prepara- <br /> 0.1% BSA dialysis buffer, beads incubated gentle             tion protein purification Ni21-nitrilotriacetic acid (NTA)&#226;&#8364;&#8220;agarose (Qia- <br /> shaking 10&#194;&#176;C 50 ml dialyzed extract. After 3 h incubation, beads     gen) native conditions performed described Chaussivert et al. <br /> were washed times 200 ml washing buffer (25 mM HEPES [pH 7.5],           (12). <br /> 50 mM KCl, 0.1 mM EDTA, 10% glycerol, 0.1% Triton X-100). Proteins                   Anti-t55, -t95, -t131 polyclonal antibodies. rTFC7p recombinant <br /> eluted incubation 30 min room temperature 16 ml washing buffer       t131DTPR2 (12) expressed hexahistidine fusions, purified Ni21- <br /> containing 2 mg synthetic peptide corresponding HA sequence ml.        NTA&#226;&#8364;&#8220;agarose, loaded preparative SDS&#226;&#8364;&#8220;8% polyacrylamide gel. Gel <br /> Immunoprecipitated proteins analyzed sodium dodecyl sulfate (SDS)-             slices containing recombinant t55 t131 derivatives 95-kDa subunit <br /> polyacrylamide gel electrophoresis (PAGE) Western blotting.                        preparative SDS-polyacrylamide gel used t55 amino acid se- <br />    Amino acid sequence determination. TFIIIC purified preparative             quence determination (see above) excised injected rabbits (three <br /> scale following immunopurification procedure described Huet et al. (25).        injections 3-week intervals) production antibodies. Preimmune <br /> Affinity-purified fractions containing TFIIIC DNA binding activity pooled         antibodies antibodies directed TFIIIC subunits purified protein <br /> (133 fractions, 66.5-ml final volume). Proteins precipitated cold tri-       A-Sepharose described previously (20), protein concentration was <br /> chloroacetic acid (10% final concentration) 40 min ice centrifuged       estimated Bradford analysis (8). <br /> 4&#194;&#176;C 4 h 17,600 3g. Pellets washed cold acetone, centrifuged (4&#194;&#176;C,        Interaction t55 35S-labeled t95. Far-Western experiments per- <br /> 20 min, 17,600 3 g, resuspended 0.6 ml SDS electrophoresis buffer (3).       formed described previously (24), rTFC7p denatured-rena- <br /> Proteins separated electrophoresis overnight 8% polyacryl-          tured filters incubation labeled probe. Plasmid pCS5, harbor- <br /> amide&#226;&#8364;&#8220;SDS gel slightly stained Coomassie blue. Polypeptides 138,           ing TFC1 gene encoding t95 (48), linearized StuI. The gene was <br /> 131, 95, 91, 80, 75, 60, 55, 50 kDa revealed. Starting 880 g (wet        transcribed vitro T7 RNA polymerase wheat germ extracts (Promega) <br /> weight) YCS7 cells expressing HA-tagged version t95 (25), 200        presence [35S]methionine. Recombinant t55 subjected SDS- <br /> 250 pmol TFIIIC obtained. A gel slice containing 95-kDa subunit         PAGE, blotted nitrocellulose, denatured-renatured according the <br /> used anti-t95 production (see below). A gel slice containing 55-kDa            method Papavassiliou Bohmann (42). The filters incubated with <br />                                                                                        35 <br /> polypeptide excised, crushed, incubated protease described              S-labeled t95, washed, autoradiographed. Recombinant t55 located by <br /> previously (48), proteinase K used instead trypsin. The re-         use anti-HA antibodies, immune complexes visualized ECL <br /> sulting peptides separated reversed-phase high-pressure liquid chroma-         kit (Amersham). <br /> tography sequenced. Seven peptide sequences (YDNPRM, EIPVY, TYIPF,                    Two-hybrid assays. The expression GAL4(1-147)-t55 GAL4(768-881)- <br /> ELAFPN, ERLVGT, FASPF, SDRKWV) determined.                                    t55 fusion proteins plasmids pAS-t55 pACT-t55, respectively, was <br />    Cloning disruption TFC7. Two degenerate oligonucleotides (Ol20               verified Western blot analysis yeast crude extracts polyclonal antibod- <br /> [59CGGAATTCRTTNGGRAANGCNARYTC] Ol8 [59NNTAYGAYAA                                   ies directed GAL4. GAL1-LacZ activation assays performed as <br /> YCCNMGNATG]) designed peptides ELAFPN YDNPRM, respec-                         described previously (12) transformation yeast strain combinations <br /> tively, used amplify yeast genomic DNA fragment &#226;&#8364;&#339;touchdown&#226;&#8364;?               plasmids. Transcriptional activation lacZ reporter gene assayed by <br /> PCR (14). A 509-bp DNA fragment obtained, cloned pBSKS (Strat-                growing transformed cells selective medium overlaying with <br /> agene), sequenced, contain continuous open reading frame                5-bromo-4-chloro-3-indolyl-b-D-galactopyranoside (X-Gal) agar. b-Galactosi- <br /> (ORF) encoding initial peptides plus others. The sequence         dase activity yeast extracts measured exactly described previously (51), <br /> entire TFC7 gene searching Munich Information Centre              30&#194;&#176;C independent transformants. The interaction between <br /> Protein Sequences (MIPS) database (GenBank accession no. Z75018).                      TFIIIB70 t131 (12) used reference. <br />    Disruption TFC7 gene performed PCR method (7). Two                     DNA binding transcription assays. The transcription factor-tDNA inter- <br /> 57-mer oligonucleotides harboring sequences complementary TFC7 gene             action monitored gel retardation assay essentially described previ- <br /> and yeast HIS3 selectable marker used amplify PCR &apos;1.1-kb         ously (35). TFIIIC partially purified wild-type mutant crude extracts <br /> DNA fragment directly introduced yeast YNN281 3 YNN282               chromatography Ultrogel-heparin A4R (Sepracor) column described <br /> strain transformation. In resulting His1 transformants, copy         previously (25). Heparin-purified TFIIIC fractions (1 ml, &apos;0.5 mg protein) <br /> whole TFC7 ORF replaced HIS3 gene, surrounded stop codon                 incubated 32P-labeled DNA probe (3 10 fmol; 3,000 10,000 <br /> modules, inserted antisense direction respect TFC7. Diploid         cpm) 15 ml binding buffer containing 10 mM Tris-HCl (pH 8), 1 mM EDTA, <br /> transformants verified PCR analysis. The heterozygous diploid strain           150 mM KCl, 10% glycerol, 50 mg BSA, 1 mg competitor DNA. The <br /> was transformed multicopy plasmid pNM2 harboring TFC7                probe 200-bp PCR-amplified fragment plasmid pUC-Glu (17), car- <br /> sporulated. One spore bearing chromosomally deleted allele TFC7             rying yeast tRNAGlu3   gene, 200-bp PCR-amplified fragment plasmid <br /> containing pNM2 plasmid chosen yield strain YNM2 used plasmid           pGE2 (4), harboring B block yeast tRNALeu      3   gene. Complexes <br /> shuffling.                                                                             analyzed nondenaturing gel electrophoresis revealed autoradiog- <br />    Construction plasmids. The 2.6-kb EcoRI/KpnI DNA fragment cos-              raphy (25). <br /> mid cospEOA273 containing coding flanking sequences TFC7                   Transcription assays performed different templates: pUC-Glu <br /> cloned plasmid YEplac195 (19), creating pNM2. The sequence encoding             (17), harboring tRNAGlu  3   gene; pRS316-SUP4, containing yeast SUP4 <br /> methionine residue followed YPYDVPDYA epitope (HA epitope) de-                  tRNA gene (a gift S. Shaaban); pGE2 (4), harboring yeast <br />                                                                                                Leu <br /> rived influenza virus HA protein (53) added just initi-        tRNA3 gene. Transcription reactions carried 45 min 25&#194;&#176;C in <br /> ation codon TFC7 PCR-mediated mutagenesis plasmid pNM2. Two                   40-ml mixtures containing 20 mM HEPES-KOH (pH 7.9); 10% glycerol; 5 mM <br />  VOL. 18, 1998                                                                TFC7 ENCODES THE 55-kDa SUBUNIT OF YEAST TFIIIC                                  3193 <br />  <br />  <br />  <br />  <br />    FIG. 1. Sequence analysis TFC7. (A) Schematic representation duplication block 50 defined Wolfe Shields (54). The locations chromosomes XIV <br /> and XV duplicated genes (TFC7 HUF) indicated shading. (B) The regions sequence similarities TFC7p, HUFp, Syn H schematically <br /> represented. (C) Sequence similarities TFC7p. The N-terminal TFC7p (amino acids 1 278) aligned following protein sequences: HUFp, S. <br /> cerevisiae 30.7-kDa hypothetical protein; Syn H, Synechocystis hypothetical protein; PGM, S. cerevisiae PGM; FbPase, S. cerevisiae FbPase; Cob C, S. typhimurium <br /> a-ribazole-59-phosphate phosphatase (GenBank accession no. Z75018, Z71385, D64002, P00950, S42124, U12808, respectively). Complete sequences shown <br /> only Syn H PGM. The flanking portions proteins, showed homology TFC7p, included. The amino acid positions each <br /> sequence indicated right. Identical residues boxed, conserved substitutions shaded. Conserved active-site residues PGM, FbPase, Cob <br /> C enzymes indicated stars. <br />  <br />  <br />  <br /> MgCl2; 90 mM KCl; 0.1 mM EDTA; 1 mM DTT; 1 U RNasin (Amersham);                  elution step DNA affinity column. The fraction con- <br /> 0.6 mM ATP, GTP, CTP; 0.03 mM [32P]UTP (2 10 Ci/mmol); 0.1 mg           taining 50- 80-kDa polypeptides RNase activity <br /> of plasmid DNA; heparin-purified TFIIIC fraction (1 ml, &apos;0.5 mg); RNA PolIII <br /> (50 ng); recombinant TFIIIB70 (150 ng); recombinant TBP (40 ng); B0             did explore further. <br /> fraction (400 ng). Protein fractions prepared according method          The 55-kDa polypeptide excised gel, sev- <br /> Huet et al. (25). RNA transcripts analyzed polyacrylamide-urea gel          eral peptides obtained proteinase K digestion. The <br /> electrophoresis revealed autoradiography (25). <br />    Nucleotide sequence accession number. The <span id='am-3' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> for <br />                                                                                     amino acid sequences seven peptides determined. De- <br /> TFC7 Z75018.                                                                     generate oligonucleotides designed amino acid se- <br />                                                                                     quences peptides used primers amplify a <br />                                                                                     genomic DNA fragment &#226;&#8364;&#339;touchdown&#226;&#8364;? PCR (14). A 509-bp <br />                                 RESULTS <br />                                                                                     DNA fragment obtained contain continu- <br />    Isolation disruption TFC7 gene. Yeast TFIIIC                       ous ORF encoded peptides. At stage, the <br /> a multisubunit protein comprises polypeptides, 138,                     sequence entire gene, named TFC7, ob- <br /> 131, 95, 91, 60, 55 kDa. In order clone gene encod-                      tained MIPS database (GenBank accession no. <br /> ing 55-kDa subunit, TFIIIC purified preparative                        Z75018). The TFC7 gene located chromosome XV. <br /> scale crude extracts immunoaffinity DNA af-                             The TFC7 ORF encodes 435-amino-acid protein a <br /> finity purification procedure (25). TFIIIC components                          predicted Mr 49,000 theoretical pI 5 that <br /> separated preparative SDS-PAGE stained Coo-                             contains seven microsequenced peptides. Comparison of <br /> massie blue. The TFIIIC fraction contained polypep-                         TFC7 protein (TFC7p) sequence National Center <br /> tides consistently affinity-purified fractions plus                  Biotechnology Information (NCBI) nonredundant data- <br /> additional polypeptides, 80, 75, 50 kDa. The 75-kDa                          <span id='am-164' about='xsp:base' typeof='owl:Thing'>base</span> use BLAST program server (1) revealed intrigu- <br /> polypeptide, occasionally purified TFIIIC fractions,                       ing similarities N-terminal half TFC7p and <br /> was recently shown immunologically related 91-                         proteins unrelated transcription. First, N-terminal <br /> kDa protein (2). The 80- 50-kDa polypeptides                         domain TFC7p shows 64% identity 80% similarity (Fig. <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> contaminants easily separated TFIIIC                         1C) S. cerevisiae 30.7-kDa protein unknown function <br /> with linear gradient ammonium sulfate instead salt                        encoded gene located chromosome XIV and <br />  3194     MANAUD ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> provisionally named HUF (for homolog unknown function). <br /> It appears TFC7 HUF genes two <br /> large blocks gene duplication chromosomes XIV <br /> and XV. As shown Fig. 1A, TFC7 HUF genes are <br /> located just border duplication block 50, defined by <br /> Wolfe Shields (54). The coding sequence N-termi- <br /> nal TFC7p belongs duplication block 50, the <br /> coding sequence C-terminal half present on <br /> chromosome XV. <br />    The N-terminal TFC7p shows regions se- <br /> quence similarity (Fig. 1B C) Synechocystis 23.7-kDa <br /> protein (Syn H), unknown function, a <br /> family phosphoglycerate mutase (PGM), fructose-2,6-bi- <br /> phosphatase (FbPase), acid phosphatase enzymes from <br /> various species. These enzymes catalyze similar phosphotrans- <br /> fer reactions involve phosphohistidine intermediate. <br /> Their activity shown dependent highly <br /> conserved histidinyl residues arginyl residue (refer- <br /> ence 40 references therein). Figure 1C shows compari- <br /> son TFC7p N-terminal half, HUF protein (HUFp), Syn <br /> H sequences sequences PGM FbPase enzymes <br /> from S. cerevisiae Cob C acid phosphatase Sal- <br /> monella typhimurium. The TFC7p N-terminal moiety showed                 FIG. 2. TFC7 encodes subunit TFIIIC. TFIIIC purified heparin <br /> approximately 20% identity 40% similarity amino           chromatography assayed gel retardation labeled tDNAGlu <br />                                                                                                                                         3   probe. <br />                                                                      Preformed TFIIIC-tDNAGlu    complexes (lane 1) incubated 250 500 <br /> acid sequence level PGM Cob C enzymes (and                                       3 <br />                                                                      ng preimmune antibodies (lanes 4 5) anti-rTFC7p antibodies (lanes 2 <br /> 15% identity 30% similarity FbPase enzyme). The           3). Complexes separated nondenaturing electrophoresis re- <br /> three highly conserved catalytic residues amino       vealed autoradiography. C1, position TFIIIC-tDNA complexes; C2, posi- <br /> acids flanking histidinyl residues conserved TFC7p.      tion complexes bound anti-rTFC7p antibodies; F, free labeled tDNA. <br /> These sequence similarities proteins clearly unrelated to <br /> transcription cyanobacterial protein raised ques- <br /> tion functional role TFC7p, any, PolIII transcrip-   DNA affinity-purified TFIIIC fraction (data shown). <br /> tion. On hand, similarity existing sequences      In order investigate TFC7p TFIIIC <br /> databases C-terminal half TFC7p,           subunits, purified anti-rTFC7p antibodies used alter <br /> which included duplication block 50 (Fig. 1A).             migration TFIIIC-tDNA complexes mobility shift <br />    To test TFC7 essential cell growth via-        assays (16, 48). Preformed TFIIIC-tDNAGlu    3   complexes were <br /> bility, DNA fragment harboring yeast HIS3 gene sur-            incubated polyclonal anti-rTFC7p antibodies (or preim- <br /> rounded stop codons inserted place            mune control immunoglobulins) analyzed electro- <br /> TFC7 ORF PCR method (7). The resulting diploid cells            phoresis polyacrylamide gel. As shown Fig. 2, anti- <br /> (YNM1 strain) chromosomal copies               rTFC7p antibodies able bind TFIIIC-tDNA <br /> TFC7 gene deleted. Analysis sporulation products re-          complexes (C1), converting larger species (C2) <br /> vealed nonviable viable (always his2) spores         migrated slowly (lanes 2 3). On hand, <br /> tetrad, showing TFC7 essential gene. To confirm          migration TFIIIC-tDNA complexes affected by <br /> this result, strain YNM1 transformed multicopy plas-        preimmune antibodies (Fig. 2, lanes 4 5). The supershift <br /> mid pNM2, harboring TFC7, sporulated. The resulting              induced anti-rTFC7p antibodies showed TFC7p part <br /> haploid strain chromosomal disruption expressing          TFIIIC-tDNA complexes corresponds t55. <br /> the plasmidic TFC7 gene viable. Thus, TFC7, like                Analysis t55 mutants. The similarity N-terminal <br /> genes encoding largest subunits TFIIIC, essen-       half TFC7p cyanobacterial protein enzymes re- <br /> tial cell viability. On hand, HUF,       lated glycolytic pathway intriguing. To define t55 <br /> the homolog unknown function TFC7 gene,                domains necessary TFIIIC activity, N-terminal <br /> expressed essential cell viability (results      C-terminal deletions (Fig. 3A) HA-tagged version of <br /> shown).                                                              t55 generated described Materials Methods. <br />    TFC7 encodes 55-kDa subunit TFIIIC. The TFC7               Centromeric plasmids harboring mutant copies TFC7 <br /> gene engineered add histidine residues             gene tested ability functionally replace a <br /> epitope derived influenza virus HA protein (HA              chromosomally disrupted copy TFC7. The resulting strains <br /> epitope) N terminus protein. The tagged protein        grown different temperatures medium containing <br /> was expressed E. coli cells purified histidine fusion    glucose glycerol carbon source. As shown Fig. <br /> under nondenaturing conditions Ni21-NTA&#226;&#8364;&#8220;agarose. West-            3A, deletion C-terminal domain (t55-DC) lethal, <br /> ern blot analysis purified protein fraction performed     suggesting domain essential TFIIIC function. In <br /> with monoclonal antibodies directed HA epitope.          contrast, deletion N-terminal domain (t55-DN3), <br /> The HA-tagged recombinant protein (predicted Mr, 50,000)             homologous HUFp, resulted wild-type <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. <br /> migrated SDS-polyacrylamide gel apparent size          3B). This vivo result confirmed vitro studies. <br /> of &apos;56 kDa, suggesting TFC7p posttranslationally         Wild-type t55-DN3 TFIIIC factors purified hep- <br /> modified.                                                            arin chromatography tested tDNA binding activity by <br />    rTFC7p injected rabbits antibody production.         gel retardation assays ability promote tran- <br /> Immunoblotting performed immune serum raised            scription different templates presence TFIIIB and <br /> rTFC7p specifically revealed 55-kDa polypeptide present          RNA PolIII. Figure 4A shows deletion form TFIIIC <br />  VOL. 18, 1998                                                                   TFC7 ENCODES THE 55-kDa SUBUNIT OF YEAST TFIIIC                                     3195 <br />  <br />  <br />  <br />  <br />    FIG. 3. Deletion analysis t55. (A) Deletion mutants constructed described Materials Methods HA-tagged version t55 (t55-WT). <br /> t55-WT t55 deletions schematically represented. The positions deleted amino acids (inclusive) construct indicated parentheses. Centromeric <br /> plasmids harboring deletion mutant copy TFC7, expressed promoter, tested ability functionally replace, different temperatures <br /> and glucose (Glu)- glycerol (Gly)-containing medium, chromosomally disrupted copy TFC7. A summary viability thermal sensitivity of <br /> the strains shown; lethal (2), wild-type (1), temperature-sensitive (ts) phenotypes indicated; ND, determined. (B) Viability thermal sensitivity of <br /> yeast strains harboring wild-type t55 deletion variants t55. Growth 30 37&#194;&#176;C glucose- glycerol-containing medium wild-type viable t55 deletion <br /> strains shown. <br />  <br />  <br />  <br /> was able form complex tRNAGlu      3  gene. The                           PolIII. As shown Fig. 4B, using of <br /> migration TFIIIC-tDNAGlu 3    complexes formed                          TFIIIC (based 95-kDa subunit), deletion form of <br /> altered anti-HA antibodies, showing epitope-tagged                              TFIIIC showed reduced levels transcriptional activity com- <br /> t55-DN3 associated subunits form TFIIIC                               pared wild type, twofold, keeping the <br /> (data shown). Deletion N-terminal half t55 no-                            twofold-lower TFIIIC-tDNA complexes formed <br /> ticeably increased rate migration TFIIIC-tDNAGlu  3                           mutant factor. The lower transcriptional activity the <br /> complexes, increase migration rate ob-                              mutant form TFIIIC accounted its <br /> served DNA fragment harboring B block                                deficiency DNA binding. Furthermore, deletion form of <br /> the tRNALeu3   gene used probe (Fig. 4A). This differ-                         TFIIIC able assemble TFIIIB correct gene position, <br /> ence migration wild-type mutant TFIIIC-                              transcripts obtained factors migrated the <br /> tDNA complexes easily explained 5% difference                             level. We concluded, therefore, N-terminal half <br /> in molecular mass wild-type mutant factors (30                              t55 play role stability and/or conformation <br /> kDa &apos;600 kDa) reflect different conformational                             TFIIIC C-terminal domain sufficient sup- <br /> states factor.                                                                   port t55 function TFIIIC-tDNA binding proper TFIIIB <br />    Even TFIIIC fractions purified wild-type                             assembly. <br /> and mutant strains shown Western blot analysis                                  In contrast deletion N-terminal domain, <br /> contain similar amounts TFIIIC, based 95-kDa sub-                             homologous HUFp, partial deletion domain t55 <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (data shown), yield complexes obtained                         (t55-DN1) resulted unexpectedly temperature-sensitive <br /> t55-DN3 TFIIIC fraction twofold lower                               phenotype (Fig. 3B). This effect specific t55-DN1 <br /> obtained wild-type factor. This result, obtained                          deletion, different partial deletion, t55-DN2, did not <br /> two distinct preparations t55-DN3 factor, suggested                         detectably affect cell growth 37&#194;&#176;C glucose-containing me- <br /> the mutant form TFIIIC slightly defective DNA                                 dium. Since N-terminal t55 regions sequence <br /> binding unstable purification wild-type                         similar enzymes related glycolytic pathway <br /> factor.                                                                                 (PGM FbPase; Fig. 1), mutant strains grown in <br />    Next, t55-DN3 factor assayed specific transcrip-                         medium containing glycerol ethanol. Cells growing glyc- <br /> tion different tRNA genes intron                         erol medium underwent glycolysis slowly did cells <br /> a reconstituted transcription presence TFIIIB                          growing glucose medium result poor uptake of <br />  3196       MANAUD ET AL.                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />                                                                                             TABLE 1. In vivo interaction t55 t95 the <br />                                                                                                            two-hybrid system <br />                                                                                                                                                  b-Galactosidase <br />                                                                                                        Fusion proteins <br />                                                                                                                                                     activitya <br />  <br />                                                                                    GAL4(1-147)                     GAL4(768-881)-t55                     6 <br />                                                                                    GAL4(1-147)-t95                 GAL4(768-881)                        10 <br />                                                                                    GAL4(1-147)-t95                 GAL4(768-881)-t55                    81 <br />                                                                                    GAL4(1-147)-TFIIIB70            GAL4(768-881)-t131                  133 <br />                                                                                       a <br />                                                                                         Nanomoles o-nitrophenyl-b-D-galactoside hydrolyzed minute mil- <br />                                                                                    ligram protein. <br />  <br />  <br />  <br />                                                                                    viously revealed interactions t131 TFIIIB70 (12) well <br />                                                                                    TFIIIB90 (45). The TFC7 ORF fused DNA <br />                                                                                    binding domain [GAL4(1-147)] transcriptional acti- <br />                                                                                    vation domain [GAL4(768-881)] yeast GAL4 protein. <br />                                                                                    All combinations t55 fusion proteins the <br />                                                                                    TFIIIB (TFIIIB70, TFIIIB90, TBP), TFIIIC (t138, t131, <br />                                                                                    t95, t91), TFIIIA, PolIII (C160, C128, C82, C53, <br />                                                                                    AC40, C34, C31, C27, AC19, ABC10a, ABC10b) comple- <br />    FIG. 4. DNA binding transcriptional activities t55-DN3 TFIIIC.           mentary fusion proteins assayed (2, 12, 51). Activation of <br /> TFIIIC purified heparin chromatography wild-type strain (WT) <br /> or mutant strain expressing C-terminal t55 (t55-DN3). (A) <br />                                                                                    lacZ reporter gene estimated b-galactosidase assays <br /> Formation TFIIIC-DNA complexes. Factor-DNA complexes formed             selected transformants. The interaction TFIIIB70 <br /> described Materials Methods incubating wild-type t55-DN3 TFIIIC       t131 (12) used reference. Significant levels of <br /> with DNA probes harboring tRNAGlu     3  gene B block     b-galactosidase activity detected t55 fused the <br /> tRNALeu    gene, indicated. Complexes analyzed electrophoresis and <br />        3 <br /> autoradiography quantified scanning. (B) Transcription various class <br />                                                                                    transcriptional activation domain assayed t95 <br /> III genes t55-DN3 TFIIIC. Wild-type t55-DN3 TFIIIC incubated        complementary fusion (Table 1). This observation suggested <br /> three tRNA genes reconstituted transcription mixtures described Mate-     t55 t95 interacted cell. The reciprocal combi- <br /> rials Methods. The different templates indicated. The RNA products         nation gave lower significant levels b-galactosidase ac- <br /> were isolated electrophoresis, revealed autoradiography, quantified by <br /> scanning. For Glu3 template, upper RNA band taken into <br />                                                                                    tivity (data shown). Next, investigated interaction <br /> account, SUP4 Leu3 templates, RNA bands, corre-           t95 deletion versions t55. The truncated frag- <br /> sponding primary mature transcripts, quantified. The different rel-    ments t55 shown Fig. 3A fused DNA binding <br /> ative yields primary mature transcripts depended level maturase   transcriptional activation domain GAL4 assayed with <br /> activity present heparin-purified TFIIIC fraction. <br />                                                                                    reciprocal t95 fusions. Deletions N-terminal domain <br />                                                                                    t55 did alter significantly t55-t95 interaction. On the <br />                                                                                    hand, deletion C-terminal domain t55 gave <br /> glycerol yeast cells, cells growing medium con-                      background b-galactosidase levels (data shown). This re- <br /> taining ethanol, respiratory substrate, did undergo gly-                     sult suggested C-terminal t55 essential its <br /> colysis. One imagine N-terminal t55                      interaction t95. <br /> has activity interferes metabolism, growth                       The interaction t55 t95 confirmed vitro <br /> N-terminal deletion mutants medium containing glyc-                      far-Western blotting experiment (24). Partially purified <br /> erol ethanol instead glucose affected. As shown                     rTFC7p subjected SDS-PAGE, transferred mem- <br /> in Fig. 3B, growth t55-DN2 mutant cells glycerol-                        branes, denatured-renatured, probed 35S-labeled t95 <br /> containing medium resulted temperature-sensitive pheno-                       antibodies directed HA epitope present at <br /> type observed glucose-containing medium. Fur-                      N-terminal end rTFC7p. As shown Fig. 5, t95 specif- <br /> thermore, doubling time t55-DN1 mutant cells grown                       ically bound level t55. No signal obtained when <br /> 30&#194;&#176;C glycerol-containing medium increased (about two-                       filters incubated labeled protein (t91) or <br /> fold) compared wild-type t55, t55-DN2, t55-DN3 <br /> mutant cells. Similar results obtained cells were <br /> grown medium contained ethanol (or ethanol plus glyc- <br /> erol) instead glucose (data shown). In cases, the <br /> wild-type phenotype restored transformation of <br /> the mutant cells centromeric plasmid harboring the <br /> wild-type TFC7 gene. We verified mutant strains <br /> were rho1. We concluded results N-termi- <br /> nal domain, partially deleted, impair t55 function. <br /> Furthermore, reduced rate growth t55-DN1 mutant <br /> cells 30&#194;&#176;C thermal sensitivity t55-DN2 <br />                                                                                        FIG. 5. t55 interacts t95. Recombinant t55, expressed HA-tagged <br /> mutant cells observed cells grown                          hexahistidine fusion, purified E. coli cells native conditions. <br /> glycerol ethanol instead glucose suggested potential link                  Eluted polypeptides (1 5 mg; respectively, lanes 1 2) subjected to <br /> between metabolic pathways PolIII transcription.                               SDS-PAGE, transferred membrane, denatured-renatured, probed with <br />                                                                                    35 <br />    t55 interacts t95. To gain insight func-                        S-labeled t95 described Materials Methods. Labeled polypeptides <br />                                                                                    revealed autoradiography (lanes 1 2). The membrane was <br /> tion 55-kDa subunit TFIIIC, used two-hybrid                       incubated anti-HA antibodies (lanes 3 4), immune complexes were <br /> system study interactions t55 com-                      visualized ECL kit. The molecular masses marker polypeptides are <br /> ponents PolIII cloned far. This method pre-                       indicated kilodaltons. <br />  VOL. 18, 1998                                                                  TFC7 ENCODES THE 55-kDa SUBUNIT OF YEAST TFIIIC                                   3197 <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 7. t55 stably associated t95. Cell extracts strains expressing <br />                                                                                     HA-t95 wild-type t95 chromatographed heparin column de- <br />                                                                                     scribed previously (25). (A) Heparin fractions eluting 200 250 mM ammo- <br />                                                                                     nium sulfate containing HA-t95 t55 pooled (30 ml) and <br />                                                                                     chromatographed 4-ml anti-HA column described previously <br />                                                                                     (25). Proteins eluted (1-ml fractions) competition synthetic HA <br />                                                                                     peptide (0.1 mg/ml). Protein samples (40 ml) subjected SDS-PAGE, <br />                                                                                     transferred nitrocellulose membrane, probed anti-HA antibodies <br />                                                                                     polyclonal antibodies directed t55. Immune complexes revealed with <br />                                                                                     ECL kit. Lanes: 1 (L), heparin fraction loaded anti-HA column; 2 <br />                                                                                     (FT), flowthrough fraction; 3 7, eluted fractions. (B) Heparin fractions (1.5 <br />                                                                                     ml) containing untagged t95 t55 chromatographed anti-HA <br />    FIG. 6. Elution profile TFIIIC subunits chromatography           column (0.15 ml) described above. Proteins eluted batchwise (0.1-ml <br /> heparin column. (A) Cell extracts wild-type strain strains expressing   fraction) competition synthetic HA peptide. Protein samples (40 ml) <br /> HA-t91, HA-t131, HA-t138 chromatographed heparin column             analyzed Western blotting antibodies directed t55. Lanes as <br /> described previously (25). Eluted fractions assayed ammonium sulfate       panel A. <br /> (1) protein (E) concentrations. The DNA binding activity TFIIIC assayed <br /> by gel retardation indicated hatched zone. The data correspond the <br /> results obtained fractions wild-type strain. Similar results were <br /> obtained fractions epitope-tagged strains. (B) Heparin fractions      contrast t91, t131, t138, t55 t95 coeluted <br /> (20 ml) subjected SDS-PAGE, proteins transferred membranes, <br /> and filters incubated antibodies. Immune complexes revealed <br />                                                                                     lower salt concentrations (200 250 mM ammonium sul- <br /> with ECL kit. Heparin fractions wild-type strain probed       fate), fractions 52 68, did contain detectable <br /> polyclonal antibodies directed t55 (row 1) polyclonal         TFIIIC-tDNA binding activity. These results suggested that <br /> antibodies directed t95 (row 2) TFIIIB70 (row 6). Heparin fractions      t55 t95 exist forms: associated TFIIIC <br /> from HA-t131 (row 3), HA-t138 (row 4), HA-t91 (row 5) probed with <br /> anti-HA antibodies. <br />                                                                                     subcomplex potentially containing proteins but <br />                                                                                     TFIIIB. TFIIIB70 (Fig. 6B) TFIIIB (data shown) <br />                                                                                     transcriptional activities coeluted fractions 68 92, well <br />                                                                                     t55-t95 complex (fractions 52 68). <br /> in similar experiments t95 used labeled probe                        In order verify physical association t55 t95, the <br /> to interact different blotted polypeptides (t95, TBP,                      putative subcomplex prepared strain expressing an <br /> TFIIIB70) (results shown).                                                      HA-tagged version t95. After heparin chromatography, pro- <br />    Existence t55-t95 subcomplex distinct TFIIIC. In                       tein fractions eluted 200 250 mM ammonium sulfate <br /> other work purification TFIIIC yeast crude                           contained t55 HA-t95 pooled and <br /> extracts, observed b-galactosidase&#226;&#8364;&#8220;t95 fusion protein                     purified immunoaffinity chromatography an- <br /> eluted peaks heparin chromatography (12a). The                        ti-HA column described previously (25). Proteins eluted by <br /> 95-kDa subunit present fractions contained                          competition HA peptide subjected SDS- <br /> tDNA binding activity TFIIIC fractions eluting                    PAGE, transferred nitrocellulose membrane, probed <br /> lower salt concentrations contained detectable                          anti-HA antibodies polyclonal antibodies directed to <br /> tDNA binding activity. We decided study precisely                       t55 (Fig. 7A). As expected, HA-tagged t95 bound an- <br /> elution profile different TFIIIC subunits heparin                     ti-HA column (Fig. 7A, lanes 1 2) eluted by <br /> chromatography correlate presence TFIIIC subunits                         competition HA peptide (lanes 3 7). t55 also <br /> to factor tDNA binding activity. Extracts wild-type                     retained antibody column (Fig. 7A, lane 2) was <br /> various epitope-tagged yeast strains chromatographed                        coeluted t95 (lanes 3 7). In order verify the <br /> heparin columns, proteins eluted ammonium                          coelution t55 t95 resulted stable association <br /> sulfate gradient. Fractions assayed salt protein                       subunits fortuitous binding <br /> concentrations DNA binding activity 32P-                        t55 anti-HA column, similar experiment per- <br /> labeled tDNA (Fig. 6A). Eluted proteins subjected                         formed control heparin fraction prepared wild- <br /> to SDS-PAGE, transferred nitrocellulose membranes,                           type strain harboring untagged t95. As shown Fig. 7B, when <br /> probed polyclonal antibodies directed t55 t95.                      t95 untagged, t55 (as untagged t95 [results not <br /> Anti-HA antibodies used reveal epitope-tagged t91,                          shown]) fully recovered flowthrough fraction (Fig. <br /> t131, t138 subunits. As shown Fig. 6B, t55, t91, t95, t131,                   7B, lane 2) detected HA-eluted fractions (lane <br /> and t138 eluted fractions (76 92) containing 300 350                       3). These results confirmed existence t55-t95 subcom- <br /> mM ammonium sulfate, tDNA binding activity. In                             plex include polypeptides. When analyzed by <br />  3198       MANAUD ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />                                                                                       significant pool free t131 subunit. In contrast, the <br />                                                                                       immunopurification procedure applied HA-t95 <br />                                                                                       crude extract, relative yields TFIIIC subunits <br />                                                                                       markedly modified. The recoveries t131 t91 <br />                                                                                       remained levels, t55 HA-t95 <br />                                                                                       greatly increased, range three- sixfold, respec- <br />                                                                                       tively. These results indicated markedly more <br />                                                                                       t95 t55 subunit yeast crude extracts relative to <br />                                                                                       t131 t91 corroborated existence t55-t95 <br />                                                                                       subcomplex yeast cells. <br />  <br />                                                                                                                DISCUSSION <br />                                                                                          Yeast TFIIIC multifunctional protein required pro- <br />                                                                                       moter binding, TFIIIB recruitment, chromatin antirepres- <br />                                                                                       sion. We pursued characterization multisub- <br />                                                                                       <span id='am-165' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> factor report isolation essential gene, <br />                                                                                       named TFC7, encoding smallest subunit. It appears this <br />                                                                                       polypeptide chimeric protein belongs different <br />                                                                                       protein complexes. <br />    FIG. 8. Coimmunoprecipitation TFIIIC subunits crude extracts. Im-             Based biochemical data gene cloning, well <br /> munoprecipitations anti-HA antibodies performed described             established largest polypeptides, 138, 131, 95, <br /> Materials Methods crude extracts wild-type (WT) strain or <br /> strains expressing HA-tagged versions t138, t131, t95 (HA-t138, HA- <br />                                                                                       91 kDa, contained highly purified TFIIIC fractions are <br /> t131, HA-t95, respectively). Proteins eluting synthetic peptides         subunits TFIIIC. The smallest polypeptides (60 55 <br /> were subjected SDS-PAGE, transferred membrane, probed succes-             kDa) consistently TFIIIC fractions <br /> sively polyclonal antibodies directed t95, t55, t131, t91. Im-       different laboratories (5, 16, 43). Using peptide sequences <br /> mune complexes revealed ECL kit quantified scanning. A <br /> value 1 arbitrarily given amounts t95, t55, t131, t91 subunits <br />                                                                                       obtained gel-purified protein, identified the <br /> immunoprecipitated HA-t138 crude extract used refer-            TFC7 gene encoding 55-kDa polypeptide. Polyclonal anti- <br /> ence. On left indicated molecular masses (in kilodaltons) polypep-     bodies directed rTFC7p able supershift TFIIIC- <br /> tides probed antibodies noted right. The 75-kDa polypeptide             tDNA complexes, confirming presence 55-kDa <br /> revealed anti-t91 antibodies immunologically related t91 (2). <br />                                                                                       polypeptide factor-tDNA complex. Like the <br />                                                                                       genes encoding components yeast PolIII transcription <br />                                                                                       isolated far, TFC7 essential gene. <br /> silver staining, eluted fractions contained HA-t95                          Searches databases revealed chimeric structure of <br /> t55 polypeptides (data shown).                           TFC7p. The N-terminal TFC7p showed intriguing sim- <br />    To determine t55-t95 subcomplex                              ilarities proteins unrelated transcription was <br /> present crude extract confirm                      highly similar yeast hypothetical protein unknown func- <br /> partial TFIIIC dissociation heparin chromatography,                            tion (named HUFp). On hand, significant se- <br /> immunoprecipitations performed crude extracts pre-                          quence similarities detected C-terminal part <br /> pared wild-type strain strains expressing                           TFC7p. In fact, coding sequences N-terminal <br /> HA-tagged copy t138 (HA-t138), t131 (HA-t131), t95                              TFC7p HUFp located just border of <br /> (HA-t95). Anti-HA antibodies used immunopurifica-                            duplication block 50 present chromosome XIV and <br /> tion, bound proteins eluted competition                            chromosome XV (54). One hypothesis chimeric <br /> specific peptide antigen. Using immunopurified proteins                           structure t55 resulted fusion ancestral <br /> from HA-t138, HA-t131, HA-t95 crude extracts, suc-                              transcription factor subunit (corresponding C-terminal <br /> ceeded reconstituting transcription tRNA gene,                        t55 included duplication block) and <br /> TFIIIC activity recovered similar efficiencies                        protein unknown function (encoded ances- <br /> three cases (data shown). Eluted TFIIIC fractions                            tral HUF corresponding N-terminal t55). <br /> analyzed Western blotting probed polyclonal an-                           This hypothesis, explain intriguing sequence <br /> tibodies directed t95, t55, t91, t131. Immune complexes                         similarities TFIIIC subunit cyanobacterial <br /> were revealed chemiluminescence (Fig. 8). As expected,                          protein (Syn H), supported results t55 deletion <br /> TFIIIC subunit detected immunopurification                               mutant analysis. Indeed, C-terminal t55 was <br /> performed wild-type crude extract containing HA-                            necessary interaction t95 sufficient tran- <br /> tagged polypeptide (Fig. 8, WT). In contrast, immunoprecipi-                          scription factor activity. Whereas deletion C-termi- <br /> tation crude extracts containing HA-tagged subunit                            nal t55 lethal, mutant form TFIIIC entirely <br /> (Fig. 8, HA-t138, HA-t131, HA-t95) resulted copu-                           deprived N-terminal half t55 (t55-DN3) supported <br /> rification 95-, 55-, 131-, 91-kDa subunits, indicating                     normal cell growth, able bind tDNA promoter se- <br /> association polypeptides other.                                    quences vitro, and, bound, recruited TFIIIB produc- <br />    The results immunopurification shown Fig. 8                         tively efficiently wild-type factor. However, al- <br /> quantified scanning. The value 1 arbitrarily given                       N-terminal domain t55 dispensable <br /> the t95, t55, t131, t91 subunit immunopuri-                          TFIIIC activity, partial deletion domain (t55-DN1) <br /> fied HA-t138 crude extract used refer-                          impaired t55 function high temperature. The residual <br /> ence. As shown Fig. 8, polypeptides recov-                         presence truncated N-terminal fragment inter- <br /> ered approximately efficiencies                            fered t55 folding led defect TFIIIC assembly or <br /> same relative yields HA-t131 crude extract. Note                        stability. <br /> HA-t131 overrepresented, suggesting absence                             The N-terminal t55 showed sequence similarities to <br />  VOL. 18, 1998                                                   TFC7 ENCODES THE 55-kDa SUBUNIT OF YEAST TFIIIC                                 3199 <br />  <br />  <br /> different enzymes catalyze similar phosphotransfer reac-                                 ACKNOWLEDGMENTS <br /> tions, specifically, enzymes related glycolytic path-       We thank Christophe Carles Franc&#194;&#184;oise Bouet peptide se- <br /> way (PGM FbPase) acid phosphatase. Since residues             quence determination Anny Ruet help raising rabbit poly- <br /> important catalytic activity enzymes con-      clonal antibodies. We grateful Janine Huet Emmanuel <br /> served t55, assayed TFIIIC rTFC7p enzymatic            Favry B0, recombinant TBP, recombinant TFIIIB70, PolIII <br /> activity. No PGM phosphatase activities detected (re-        preparations. We thank Jochen Ru   &#194;&#168;th Genevie`ve Dujardin for <br /> sults shown). Nevertheless, partial deletions N-ter-      helpful discussions Cathy Jackson improving manuscript. <br />                                                                         This work supported grant European Union <br /> minal domain t55 resulted reduced growth rate 30&#194;&#176;C        BIOTECH program (to A.S.). N.M. supported fellowship <br /> (t55-DN1) thermosensitive phenotype (t55-DN1             French Ministe`re l&#226;&#8364;&#8482;Enseignement Supe&#194;&#180;rieur et la Re- <br /> t55-DN2) glycerol- ethanol-containing medium. These            cherche, R.A. supported P.F.I. postdoctoral fellowship <br /> growth defects, revealed medium contain-          Spanish Ministerio Educacio&#194;&#180;n y Cultura. <br /> ing glycerol ethanol instead glucose, suggest possible <br /> relationship PolIII transcription metabolic path-                                        REFERENCES <br /> ways deserves investigated further.                       1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990. <br />    The direct interaction t55 t95 observed vitro          Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410. <br /> and vivo agrees photo-cross-linking mapping       2. Arrebola, R., N. Manaud, S. Rozenfeld, M. C. Marsolier, O. Lefebvre, C. <br />                                                                          Carles, P. Thuriaux, C. Conesa, A. Sentenac. 1998. t91, essential <br /> these polypeptides opposite sides DNA helix,            subunit yeast transcription factor IIIC, cooperates t138 DNA <br /> the vicinity A block tRNA genes (5, 6). Our inability          binding. Mol. Cell. Biol. 18:1&#226;&#8364;&#8220;9. <br /> to alter migration tB-tDNA complexes anti-t55             3. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. <br />                                                                          Smith, K. Struhl (ed.). 1994. Current protocols molecular biology. <br /> antibodies (data shown) argues favor               John Wiley &amp; Sons, Inc., New York, N.Y. <br /> localization t55 tA, TFIIIC domain binds        4. Baker, R. E., B. D. Hall. 1984. Structural features yeast tRNA genes <br /> A block tRNA genes. However, sequence analysis t55 did             affect transcription factor binding. EMBO J. 3:2793&#226;&#8364;&#8220;2800. <br />                                                                       5. Bartholomew, B., G. A. Kassavetis, B. R. Braun, E. P. Geiduschek. 1990. <br /> not reveal known DNA binding motifs, attempts                 The subunit structure Saccharomyces cerevisiae transcription factor IIIC <br /> demonstrate, gel retardation assays Southwestern blot-             probed novel photocrosslinking reagent. EMBO J. 9:2197&#226;&#8364;&#8220;2205. <br /> ting, t55 t95 binds DNA, nonspecifically, failed      6. Bartholomew, B., G. A. Kassavetis, E. P. Geiduschek. 1991. Two com- <br />                                                                          ponents Saccharomyces cerevisiae transcription factor IIIB (TFIIIB) are <br /> (results shown) (44, 48). One imagine in-             stereospecifically located upstream tRNA gene interact the <br /> teraction tA A block tRNA genes requires              second-largest subunit TFIIIC. Mol. Cell. Biol. 11:5181&#226;&#8364;&#8220;5189. <br /> association t95 t55. However, tDNA binding             7. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul- <br />                                                                          lin. 1993. A simple efficient method direct gene deletion Saccha- <br /> activity transcriptional activation (or inhibition)         romyces cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330. <br /> to associated partially purified t55-t95 subcomplex         8. Bradford, M. M. 1976. A rapid sensitive method quantitation of <br /> (results shown). From point view, presence            microgram quantities protein utilizing principle protein-dye bind- <br /> two TFIIIC components distinct complexes reminis-              ing. Anal. Biochem. 72:248&#226;&#8364;&#8220;254. <br />                                                                       9. Braun, B. R., B. Bartholomew, G. A. Kassavetis, E. P. Geiduschek. 1992. <br /> cent equivalent chromatographic separation             Topography transcription factor complexes Saccharomyces cerevi- <br /> of silkworm human TFIIIC distinct, complementary                 siae 5 S RNA gene. J. Mol. Biol. 228:1063&#226;&#8364;&#8220;1077. <br /> activities. TFIIIC activity silkworm cells separated     10. Burnol, A.-F., F. Margottin, J. Huet, G. Almouzni, M.-N. Prioleau, M. <br />                                                                          Me&#194;&#180;chali, A. Sentenac. 1993. TFIIIC relieves repression U6 snRNA <br /> into fractions, required form com-            transcription chromatin. Nature 362:475&#226;&#8364;&#8220;477. <br /> plex tRNA gene (41). For human TFIIIC, fractions,           11. Burnol, A. F., F. Margottin, P. Schultz, M. C. Marsolier, P. Oudet, A. <br /> named TFIIIC1 TFIIIC2, necessary reconsti-               Sentenac. 1993. Basal promoter enhancer elements yeast U6 snRNA <br />                                                                          gene. J. Mol. Biol. 233:644&#226;&#8364;&#8220;658. <br /> tute TFIIIC DNA binding transcriptional activities:         12. Chaussivert, N., C. Conesa, S. Shaaban, A. Sentenac. 1995. Complex <br /> TFIIIC2, multisubunit protein (33, 50, 56), binds strongly          interactions yeast TFIIIB TFIIIC. J. Biol. Chem. 270:15353&#226;&#8364;&#8220; <br /> itself B block tRNA genes, A block binding             15358. <br />                                                                      12a.Conesa, C. Unpublished results. <br /> gene transcription require TFIIIC1 (50, 55). Furthermore,            13. Conesa, C., R. N. Swanson, P. Schultz, P. Oudet, A. Sentenac. 1993. On <br /> TFIIIC1 chromatographically separated addi-               subunit composition, stoichiometry, phosphorylation yeast <br /> tional activity, TFIIIC0, able partially substitute       transcription factor TFIIIC/t. J. Biol. Chem. 268:18047&#226;&#8364;&#8220;18052. <br /> TFIIIC1 activity (38). The situation different here,    14. Don, R. H., P. T. Cox, B. J. Wainwright, K. Baker, J. S. Mattick. 1991. <br />                                                                          &#226;&#8364;&#732;Touchdown&#226;&#8364;&#8482; PCR circumvent spurious priming gene amplifica- <br /> t55-t95 complex (with possible associated components)             tion. Nucleic Acids Res. 19:4008. <br /> required TFIIIC activity. The existence t55-t95 com-        15. Edwards, D. R., C. L. Parfett, D. T. Denhardt. 1985. Transcriptional <br /> plex distinct TFIIIC suggests dual role             regulation serum-induced RNAs mouse fibroblasts: equivalence of <br />                                                                          species B2 repetitive elements. Mol. Cell. Biol. 5:3280&#226;&#8364;&#8220;3288. <br /> subunits. This complex possibly endowed regulatory           16. Gabrielsen, O. S., N. Marzouki, A. Ruet, A. Sentenac, P. Fromageot. <br /> functions roles unrelated transcription. The             1989. Two polypeptide chains yeast transcription factor t interact with <br /> subcellular localization complex identification          DNA. J. Biol. Chem. 264:7505&#226;&#8364;&#8220;7511. <br />                                                                      17. Gabrielsen, O. S., T. B. Oyen. 1987. The requirement A block <br /> of associated proteins, any, shed light func-         promoter tRNA gene transcription vitro depends ionic environ- <br /> tion. It uncover phenotype(s)            ment. Nucleic Acids Res. 15:5699&#226;&#8364;&#8220;5713. <br /> caused deletion HUFp, highly similar       18. Geiduschek, E. P., G. A. Kassavetis. 1992. RNA polymerase III tran- <br />                                                                          scription complexes, p. 247&#226;&#8364;&#8220;280. In S. L. McKnight K. R. Yamamoto <br /> N-terminal t55. HUFp distantly homologous en-              (ed.), (Transcriptional regulation, vol. 1. Cold Spring Harbor Laboratory <br /> zymes related glycolytic pathway. This similarity suggests        Press, Cold Spring Harbor, N.Y. <br /> a connection cell metabolism supported      19. Gietz, R. D., A. Sugino. 1988. New yeast-Escherichia coli shuttle vectors <br />                                                                          constructed vitro mutagenized yeast genes lacking six-base pair re- <br /> altered growth phenotype t55-DN1 t55-DN2 mutant                   striction sites. Gene 74:527&#226;&#8364;&#8220;534. <br /> cells glycerol- ethanol-containing medium. The level        20. Harlow, E., D. Lane. 1988. Antibodies: laboratory manual. Cold Spring <br /> PolIII transcription known vary according cell              Harbor Laboratory, Cold Spring Harbor, N.Y. <br /> growth rate human, mouse, yeast cells (15, 23, 47, 49).       21. Harper, J. W., G. R. Adami, N. Wei, K. Keyomarsi, S. J. Elledge. 1993. <br />                                                                          The p21 Cdk-interacting protein Cip1 potent inhibitor G1 cyclin- <br /> TFIIIC components play role coordinated reg-               dependent kinases. Cell 75:805&#226;&#8364;&#8220;816. <br /> ulation.                                                             22. Heinemeyer, W., A. Gruhler, V. Mohrle, Y. Mahe, D. H. Wolf. 1993. <br />  3200         MANAUD ET AL.                                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />       PRE2, highly homologous human major histocompatibility complex-                 ferent aspects DNA binding various pol III genes. Nucleic Acids Res. <br />       linked RING10 gene, codes yeast proteasome subunit necessary                 25:2440&#226;&#8364;&#8220;2447. <br />       chymotryptic activity degradation ubiquinated proteins. J. Biol.          39.   Ossipow, V., J. P. Tassan, E. A. Nigg, U. Schibler. 1995. A mammalian <br />       Chem. 268:5115&#226;&#8364;&#8220;5120.                                                                   RNA polymerase II holoenzyme containing components required for <br /> 23.   Hoeffler, W. K., R. G. Roeder. 1985. Enhancement RNA polymerase                 promoter-specific transcription initiation. Cell 83:137&#226;&#8364;&#8220;146. <br />       III transcription E1A gene product adenovirus. Cell 41:955&#226;&#8364;&#8220;963.        40.   O&#226;&#8364;&#8482;Toole, G. A., J. R. Trzebiatowski, J. C. Escalante-Semerena. 1994. The <br /> 24.   Huet, J., C. Conesa, N. Manaud, N. Chaussivert, A. Sentenac. 1994.                 cobC gene Salmonella typhimurium codes novel phosphatase in- <br />       Interactions yeast TFIIIB components. Nucleic Acids Res. 22:3433&#226;&#8364;&#8220;              volved assembly nucleotide loop cobalamin. J. Biol. Chem. <br />       3439.                                                                                  269:26503&#226;&#8364;&#8220;26511. <br /> 25.   Huet, J., N. Manaud, G. Dieci, G. Peyroche, C. Conesa, O. Lefebvre, A. Ruet,     41.   Ottonello, S., D. H. Rivier, G. M. Doolittle, L. S. Young, K. U. Sprague. <br />       M. Riva, A. Sentenac. 1996. RNA polymerase III class III transcrip-            1987. The properties new polymerase III transcription factor reveal that <br />       tion factors Saccharomyces cerevisiae. Methods Enzymol. 273:249&#226;&#8364;&#8220;267.              transcription complexes assemble pathway. EMBO J. <br /> 26.   Joazeiro, C. A. P., G. A. Kassavetis, E. P. Geiduschek. 1996. Alternative          6:1921&#226;&#8364;&#8220;1927. <br />       outcomes assembly promoter complexes: roles TBP flexible      42.   Papavassiliou, A. G., D. Bohmann. 1992. Optimization signal-to- <br />       linker placing TFIIIB tRNA genes. Genes Dev. 10:725&#226;&#8364;&#8220;739.                         noise ratio south-western assays using lipid-free BSA blocking <br /> 27.   Kassavetis, G. A., C. Bardeleben, B. Bartholomew, B. R. Braun, C. A. P.                reagent. Nucleic Acids Res. 20:4365&#226;&#8364;&#8220;4366. <br />       Joazeiro, M. Pisano, E. P. Geiduschek. 1994. Transcription RNA            43.   Parsons, M. C., P. A. Weil. 1990. Purification characterization of <br />       polymerase III, p. 107&#226;&#8364;&#8220;126. In R. C. Conaway J. W. Conaway (ed.),                  Saccharomyces cerevisiae transcription factor TFIIIC. Polypeptide composi- <br />       Transcription: mechanisms regulation. Raven Press, Ltd., New York,                 tion defined polyclonal antibodies. J. Biol. Chem. 265:5095&#226;&#8364;&#8220;5103. <br />       N.Y.                                                                             44.   Parsons, M. C., P. A. Weil. 1992. Cloning TFC1, Saccharomyces <br /> 28.   Kassavetis, G. A., B. R. Braun, L. H. Nguyen, E. P. Geiduschek. 1990. S.           cerevisiae gene encoding 95-kDa subunit transcription factor <br />       cerevisiae TFIIIB transcription initiation factor proper RNA poly-           TFIIIC. J. Biol. Chem. 267:2894&#226;&#8364;&#8220;2901. <br />       merase III, TFIIIA TFIIIC assembly factors. Cell 60:235&#226;&#8364;&#8220;245.       45.   Ru&#194;&#168;th, J., C. Conesa, G. Dieci, O. Lefebvre, A. Du &#194;&#168;sterho&#194;&#168;ft, S. Ottonello, and <br /> 29.   Kassavetis, G. A., C. A. P. Joazeiro, M. Pisano, E. P. Geiduschek, T. Colbert,         A. Sentenac. 1996. A suppressor mutations class III transcription <br />       S. Hahn, J. A. Blanco. 1992. The role TATA-binding protein               encodes component yeast TFIIIB. EMBO J. 15:1941&#226;&#8364;&#8220;1949. <br />       assembly function multisubunit yeast RNA polymerase III           46.   Schultz, P., N. Marzouki, C. Marck, A. Ruet, P. Oudet, A. Sentenac. <br />                                                                                              1989. The DNA-binding domains yeast transcription factor t as <br />       transcription factor, TFIIIB. Cell 71:1055&#226;&#8364;&#8220;1064. <br />                                                                                              observed scanning transmission electron microscopy. EMBO J. 8:3815&#226;&#8364;&#8220; <br /> 30.   Kassavetis, G. A., S. T. Nguyen, R. Kobayashi, A. Kumar, E. P. Geiduschek, <br />                                                                                              3824. <br />       M. Pisano. 1995. Cloning, expression, function TFC5, gene <br />                                                                                        47.   Sethy, I., R. D. Moir, M. Librizzi, I. M. Willis. 1995. In vitro evidence for <br />       encoding B0 component Saccharomyces cerevisiae RNA polymer- <br />                                                                                              growth regulation tRNA gene transcription yeast. A role transcrip- <br />       ase III transcription factor TFIIIB. Proc. Natl. Acad. Sci. USA 92:9786&#226;&#8364;&#8220;9790. <br />                                                                                              tion factor (TF) IIIB70 TFIIIC. J. Biol. Chem. 270:28463&#226;&#8364;&#8220;28470. <br /> 31.   Kassavetis, G. A., D. L. Riggs, R. Negri, L. H. Nguyen, E. P. Geiduschek.    48.   Swanson, R. N., C. Conesa, O. Lefebvre, C. Carles, A. Ruet, E. Quemeneur, <br />       1989. Transcription factor IIIB generates extended DNA interactions                 J. Gagnon, A. Sentenac. 1991. Isolation TFC1, gene encoding one <br />       RNA polymerase III transcription complexes tRNA genes. Mol. Cell.                   DNA-binding subunits yeast transcription factor t (TFIIIC). Proc. <br />       Biol. 9:2551&#226;&#8364;&#8220;2566.                                                                     Natl. Acad. Sci. USA 88:4887&#226;&#8364;&#8220;4891. <br /> 32.   Khoo, B., B. Brophy, S. P. Jackson. 1994. Conserved functional domains       49.   Tower, J., B. Sollner-Webb. 1988. Polymerase III transcription factor B <br />       RNA polymerase III general transcription factor BRF. Genes Dev.                 activity reduced extracts growth-restricted cells. Mol. Cell. Biol. <br />       8:2879&#226;&#8364;&#8220;2890.                                                                           8:1001&#226;&#8364;&#8220;1005. <br /> 33.   Kovelman, R., R. G. Roeder. 1992. Purification characterization       50.   Wang, Z., R. G. Roeder. 1996. Transcription factor IIIC1 (TFIIIC1) acts <br />       forms human transcription factor IIIC. J. Biol. Chem. 267:24446&#226;&#8364;&#8220;                downstream region stabilize TFIIIC2 binding RNA polymer- <br />       24456.                                                                                 ase III promoters. Mol. Cell. Biol. 16:6841&#226;&#8364;&#8220;6850. <br /> 34.   Lefebvre, O., C. Carles, C. Conesa, R. N. Swanson, F. Bouet, M. Riva,        51.   Werner, M., N. Chaussivert, I. M. Willis, A. Sentenac. 1993. Interaction <br />       A. Sentenac. 1992. TFC3: gene encoding B-block binding subunit              complex RNA polymerase III subunits 70-kDa com- <br />       yeast transcription factor TFIIIC. Proc. Natl. Acad. Sci. USA 89:10512&#226;&#8364;&#8220;                ponent transcription factor IIIB. J. Biol. Chem. 268:20721&#226;&#8364;&#8220;20724. <br />       10516.                                                                           52.   Willis, I. M. 1993. RNA polymerase III. Genes, factors transcriptional <br /> 35.   Lefebvre, O., J. Ru  &#194;&#168;th, A. Sentenac. 1994. A <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> largest             spec <br /> </body></html>