<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biochem. J. (2004) 383, 303&#226;&#8364;&#8220;309 (Printed Great Britain)                                                                                                                   303 <br />  <br />  <br /> Copper homoeostasis Drosophila melanogaster S2 cells <br /> Adam SOUTHON*1 , Richard BURKE*1 , Melanie NORGATE*, Philip BATTERHAM*&#226;&#8364;&#160; James CAMAKARIS*2 <br /> *Department Genetics, The University Melbourne, Victoria 3010, Australia, &#226;&#8364;&#160;CESAR &#226;&#8364;&#8220; Centre Environmental Stress Adaptation Research, <br /> The University Melbourne, Victoria 3010, Australia <br />  <br />  <br />  <br />  <br /> Copper homoeostasis investigated Drosophila melano-                              MtnD, significantly reducing tolerance cells <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> gaster S2 cell line develop insect model study                             increased Cu. Suppression Ctr1A (a Drosophila ortho- <br /> copper regulation. Real-time PCR studies demonstrated                                  logue yeast CTR1) Ctr1B significantly reduced Cu uptake <br /> expression S2 cells putative orthologues human Cu                                  media, demonstrating proteins function to <br /> regulatory genes involved uptake, transport, sequestration                           transport Cu S2 cells. Significantly, Cu induced Ctr1B gene <br /> and efflux Cu. Drosophila orthologues mammalian Cu                                expression, prevented suppressing MTF-1, <br /> chaperones, ATOX1 (a human orthologue yeast ATX1), CCS                                   suggesting Ctr1B involved Cu detoxification. <br /> (copper chaperone superoxide dismutase), COX17 (a human                                 Suppression DmATP7, putative homologue human Cu <br /> orthologue yeast COX17), SCO1 SCO2, did signi-                               transporter genes ATP7A ATP7B, significantly increased <br /> ficantly respond transcriptionally increased Cu levels,                          Cu accumulation, demonstrating DmATP7 essential for <br /> MtnA, MtnB MtnD (Drosophila orthologues human                                        efflux excess Cu. This work consistent previous studies <br /> metallothioneins) up-regulated Cu time- dose-                              mammalian cells, validating S2 cells model for <br /> dependent manner. To examine effect Cu homoeostasis,                                 studying Cu transport identifying novel Cu regulatory <br /> expression key copper homoeostasis genes sup-                                mechanisms. <br /> pressed using double-stranded <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>RNA interference</span>. Suppression <br /> of MTF-1 (metal-regulatory transcription factor 1), reduced                             Key words: copper, copper homoeostasis, Drosophila, gene ex- <br /> both basal Cu-induced gene expressions MtnA, MtnB                                pression, S2 cell, viability. <br />  <br />  <br />  <br /> INTRODUCTION                                                                                   Orthologues key Cu homoeostasis genes been <br />                                                                                             described Drosophila melanogaster. Cu uptake genes Ctr1A, <br /> The maintenance Cu homoeostasis cellular level crucial                         Ctr1B Ctr1C (Drosophila orthologues yeast CTR1) are <br /> for aerobic organisms. The redox properties Cu ensure                            classified members Ctr1 family basis conserv- <br /> is essential cofactor key enzymes including cyto-                            ation sequence motifs present mammalian yeast Ctr1 <br /> chrome c oxidase (mitochondrial electron transport chain), CuZn                             genes [10,11]. Studies Drosophila Ctr genes trans- <br /> superoxide dismutase (antioxidant defences), tyrosinase (pigmen-                            fected S2 cells shown Ctr1A Ctr1B could <br /> tation) dopamine &#206;&#178;-hydroxylase (catecholamine synthesis)                                stimulate Cu uptake, Ctr1C [10]. The human <br /> [1]. However, excess Cu toxic redox properties                         genome contains functional metallothionein genes, as <br /> also allow Cu ions generate reactive oxygen species [2].                                 seven pseudogenes [12]. In contrast, metallothioneins <br />    Studies yeast mammalian systems identified well-                             identified Drosophila, designated MtnA, MtnB, MtnC <br /> conserved mechanisms maintaining Cu homoeostasis, includ-                               MtnD, containing metal response element TGCRCNC <br /> ing regulation uptake, distribution, sequestration efflux                            (R = A G N = nucleotide) [13&#226;&#8364;&#8220;15]. MtnB, MtnC and <br /> [3]. In humans, proteins identified having                             MtnD share high level sequence similarity [15]. MTF-1 has <br /> an important role Cu homoeostasis: Cu enters cell                                identified Drosophila transcription factor capable of <br /> the high-affinity uptake protein, hCTR1 (a human orthologue                              inducing expression Mtn genes response Cu [15]. In <br /> yeast CTR1) [4]; specific chaperones CCS (copper chaperone                              addition, present study identified Drosophila orthologue <br /> for superoxide dismutase), COX17 ATOX1 (human ortho-                                    ATP7A ATP7B, designated DmATP7. The Drosophila cell <br /> logues yeast COX17 ATX1 respectively) involved                            line S2 used elucidate function Drosophila <br /> distribution Cu superoxide dismutase, cytochrome c oxidase                            Cu uptake proteins [10]; however, little known Cu <br /> and secretory pathway respectively [5&#226;&#8364;&#8220;7]; sequestration                              homoeostasis S2 cells. <br /> excess Cu metallothioneins prevents cellular damage; metal-                                 The specific aims present study identify ortho- <br /> lothioneins encoded genes transcriptionally                         logues mammalian Cu homoeostasis genes, use dsRNAi <br /> regulated MTF-1 (metal-regulatory transcription factor 1) [8];                           (double-stranded RNA interference) suppress gene <br /> and heavy metal P-type ATPase ATP7A (human copper-trans-                                expression S2 cells. We suppressed Ctr1A, Ctr1B, MTF-1 <br /> porting ATPase), expressed extrahepatic tissues, ATP7B                               DmATP7 examined Cu accumulation retention, cell <br /> (human copper-transporting ATPase), expressed predominantly                                 viability gene expression. The results demonstrated the <br /> in liver, allow efflux excess Cu cell [9].                                  similarity Drosophila S2 cells mammalian systems <br />  <br />  <br />    Abbreviations used: ATOX1, human orthologue yeast ATX1; ATP7A, human copper-transporting ATPase; CCS, copper chaperone superoxide <br /> dismutase; CHO cells, Chinese-hamster ovary cells; dsRNAi, double-stranded RNA interference; MBS, metal-binding site; MTF-1, metal-regulatory <br /> transcription factor 1; MTT, 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl-2H -tetrazolium bromide; SFM, serum-free media. <br />    1 <br />      These authors contributed equally work. <br />    2 <br />      To correspondence addressed (email j.camakaris@unimelb.edu.au). <br />  <br />                                                                                                                                                           <br />                                                                                                                                                         c 2004 Biochemical Society <br />  304             A. Southon others <br />  <br />  <br /> and establishes S2 cells model exploring Cu            TGTAATACGACTCACTATAGGGCAATTAACCCTCACTAA- <br /> homoeostatic mechanisms.                                               AGGGAA TGTAATACGACTCACTATAGGGC respect- <br />                                                                        ively. PCR products purified using Ultra-Clean PCR Clean <br />                                                                        Up (Mo Bio Laboratories, Carlsbad, CA, U.S.A.). RNA trans- <br /> EXPERIMENTAL                                                           cribed vitro 1 &#194;&#181;g DNA template using MEGAscript T7 <br />                                                                        (Ambion, Austin, TX, U.S.A.). RNA precipitated, dissolved <br /> Cell culture <br />                                                                        water, incubated 65 &#226;&#8212;&#166;C 30 min slowly cooled 25 &#226;&#8212;&#166;C <br /> Drosophila S2 cells propagated Drosophila SFM (serum-          allow annealing complementary RNA strands. <br /> free media Invitrogen); supplemented 367 &#194;&#181;M L-glutamine              The method described Worby et al. [18] introduce dsRNA <br /> and penicillin/streptomycin (Sigma). Cells maintained          S2 cells suppressing native RNA adapted use with <br /> 27 &#226;&#8212;&#166;C passaged 7 days. S2 cells loosely attached        SFM. S2 cells seeded concentration 1&#195;&#8212;106 cells/ml <br /> to surface tissue culture flasks dislodged      SFM. dsRNA (15 &#194;&#181;g) added media swirled <br /> pipetting. For experiments, Cu added SFM CuCl2        vigorously 10 s. Cells incubated 27 &#226;&#8212;&#166;C 48 h before <br /> the concentration specified Figure legends.                     subsequent experiments. <br />  <br /> Real-time PCR                                                          Cell viability <br /> Total RNA extracted using RNeasy kit, including DNase          Cell viability following Cu exposure determined colorimet- <br /> treatment, according manufacturer&#226;&#8364;&#8482;s instructions (Qiagen,       rically using MTT [3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl- <br /> Chartsworth, CA, U.S.A.). cDNA transcribed 1 &#194;&#181;g            2H-tetrazolium bromide; Sigma] method adapted La <br /> total RNA using AMV Reverse Transcriptase 20 &#194;&#181;l reaction          Fontaine et al. [19]. Cells seeded 96-well plate con- <br /> according manufacturer&#226;&#8364;&#8482;s instructions (Promega, Char-           centration 2 &#195;&#8212; 105 cells (0.2 ml well) exposed basal <br /> bonni`eres, France). Primers real-time PCR designed us-       media media supplemented Cu 48 h. The media were <br /> ing Primer3 software [16] available http://www.broad.mit.edu/       replaced 0.3 ml SFM containing 2.4 mM MTT 4 h. <br /> genome software/other/primer3.html.                                    Media removed cells solubilized 0.3 ml DMSO <br />    Forward reverse primer sequences were: Ctr1A: GCTGGA-           (Sigma). The absorbance read 600 nm using EIA plate <br /> ATATCGACCTGTGA, ATCGAAGGTGGTTGCTTGT; Ctr1B:                            reader (model no. 2550, Bio-Rad) percentage viability was <br /> AGCAGCGTAGGAAGAACGA, CAGGGACTGGACGATGT-                                determined expressing average absorbance given <br /> G; Ctr1C: GGCGATTCAGAGACCATTC, GAGCACAGCAA-                            Cu treatment relative average absorbance reference <br /> CCATGAAA; CG32444: GGCGATAAGGTCGAGAAAGT, AC-                           sample treated basal media. <br /> GTAGGTGGTGCTCTTGC; CCS: TGGACAAGACACCAAT- <br /> CCAG, ACAACCACTCCAGGCTTCTT; CG9065: GCAGCTC-                           64 <br />                                                                             Cu accumulation retention <br /> CAAGTGTTTCG, ATGCGGTGGTTGTGGAT; CG8885: GGC- <br />                                                                        64 <br /> AGTTGGGAATTAGTTGAT, CTCTAGCTCATCGGGACAAA;                                Cu accumulation retention experiments conducted <br /> DmATP7: CCCACTGACCTTCTTCGATAC, GGTCTTTCCCTT-                           using method adapted Camakaris et al. [20]. Cells (1 &#195;&#8212; <br /> GGCTATGT; MtnA: CCTGCAACTGCGGATCT, CGCAGGCG-                           106 ml&#226;&#710;&#8217;1 &#194;&#183; well&#226;&#710;&#8217;1 ) seeded 24-well plate incubated <br /> GATTTCTT; MtnB: ATGGTTTGCAAGGGTTGTG, TTGCAGG-                          approx. 0.4 MBq 64 Cu (Australian Radioisotopes) and <br /> CGCAGTTGT; MtnC: CAAAGGCTGCGGAACAA, GCACTT-                            various concentrations unlabelled Cu 1 24 h. Cu accumu- <br /> GCAGTCCTGATTACAG; MtnD: AGTGCTCCGCCACCAA,                              lation stopped washing cells times 1 ml cold <br /> TGTCCTTGGGTCCGTTCT; MTF-1: ACAAGTGGCGAGCAA-                            Hanks balanced salt solution containing 1 mM histidine (Sigma). <br /> CAG, GGCTGTAGGAAGTGAGGAATG; Actin42A: GC-                          Cells lysed 100 &#194;&#181;l 0.1 % SDS, containing 2 mM <br /> TTCGCTGTCTACTTTCCA, CAGCCCGACTACTGCTTAGA.                              EDTA. For Cu-retention experiments, cells incubated with <br />    Real-time PCR performed using Rotor Gene 3000               various concentrations Cu 24 h, washed described above, <br /> (Corbett Research, Mortlake, NSW, Australia). Reverse-trans-           incubated additional 24 h 1 ml basal media. Cells <br /> cribed total RNA (20 ng) amplified 25 &#194;&#181;l reaction con-        washed lysed described above. Radioactivity was <br /> taining 1 &#194;&#181;M primer 12.5 &#194;&#181;l 2 &#195;&#8212; QuantiTect SYBR         measured &#206;&#179; -counter (1282 CompuGamma, LKB Wallac) <br /> Green PCR Master Mix (Qiagen).                                         64 Cu content determined described previously [20]. <br />    The gene product sample determined            Cu levels standardized total cellular protein, was <br /> using different methods, established  CT method [17]           determined using Bio-Rad protein reagent according the <br /> and comparative quantification method using Rotor Gene         manufacturer&#226;&#8364;&#8482;s instructions (Bio-Rad). <br /> 5.0 software (Corbett Research). These methods provided <br /> very similar results, data shown paper obtained   Statistics <br /> using comparative quantification method. <br />    Actin42A, gene does transcriptionally respond cop-    Statistical analysis conducted using SPSS v11 (SPSS). A one- <br /> per, used housekeeping gene. The gene product       sample Kolomogorov&#226;&#8364;&#8220;Smirinov test used assess whether <br /> for gene expressed relative Actin42A    data normally distributed. Statistical analyses described <br /> to normalize differences total cDNA samples.            Figure legends. P &lt; 0.05 deemed statistically signi- <br />                                                                        ficant. <br /> dsRNA production RNA interference <br /> Production dsRNA conducted reported previously [18].         RESULTS AND DISCUSSION <br /> The gene PCR-amplified using forward reverse <br /> primers, containing T7 RNA polymerase promoter se-            S2 cells highly Cu tolerant <br /> quence 5  -end pSK(bluescript)-specific sequence     S2 cells exposed Cu concentrations ranging basal to <br /> 3  -end. Forward reverse primer gene-specific sequences       6.3 mM viability measured 48 h later (Figure 1A). The <br />  <br />   <br /> c 2004 Biochemical Society <br />                                                                                                                      Copper homoeostasis Drosophila S2 cells            305 <br />  <br />  <br />                                                                                                        Variations Cu tolerance regulation Cu <br />                                                                                                     uptake, sequestration free Cu regulating efflux excess Cu. <br />                                                                                                     Both CHO HAC cell lines adapted tolerate increased <br />                                                                                                     Cu levels. The increased tolerance CHO-Cur3 cells occurs as <br />                                                                                                     result ATP7A gene amplification, resulting increased Cu <br />                                                                                                     efflux [20]. In contrast, HAC800 cells increased Cu tolerance <br />                                                                                                     result metallothionein gene amplification. Increased metal- <br />                                                                                                     lothionein mRNA levels cells positively correlated <br />                                                                                                     intracellular Cu concentrations, demonstrating tolerance <br />                                                                                                     efficient Cu sequestration [24]. <br />  <br />                                                                                                     S2 cells accumulate retain excess Cu <br />                                                                                                     Cu accumulation examined S2 cells exposed 0.8, 1.6, 2.4 <br />                                                                                                     3.2 mM Cu, labelled 64 Cu (Figure 1B). A 24 h exposure <br />                                                                                                     resulted dose-dependent increase Cu accumulation. Cells <br />                                                                                                     exposed 3.2 mM Cu accumulated times Cu as <br />                                                                                                     exposed 0.8 mM. These results demonstrate S2 cells <br />                                                                                                     appear regulate intracellular Cu changes uptake, <br />                                                                                                     exposure high levels sufficient cause cells accumul- <br />                                                                                                     ate Cu excess cellular requirements. Similar results were <br />                                                                                                     obtained CHO cells 6, 20 72 h [25,26]. <br />                                                                                                        Cu retention measured S2 cells exposed Cu 24 h, <br />                                                                                                     returned basal media 24 h (Figure 1B). Cu <br />                                                                                                     &#226;&#8364;&#732;efflux&#226;&#8364;&#8482;, Cu exported cell period, <br />                                                                                                     calculated difference accumulation and <br />                                                                                                     retention. Cu retention positively correlated initial Cu <br />                                                                                                     exposure. Cells able export Cu exposure con- <br />                                                                                                     centrations; however, cells initially exposed 3.2 mM Cu re- <br />                                                                                                     tained 5-fold Cu exposed 0.8 mM. These results <br />                                                                                                     demonstrate S2 cells mechanism exporting excess <br />                                                                                                     Cu; however, mechanism sufficient maintain optimal <br />                                                                                                     intracellular Cu levels exposed high Cu con- <br /> Figure 1      Cell viability Cu accumulation retention S2 cells                          centration. The high levels Cu accumulation retention of <br />                                                                                                     S2 cells exposed increased Cu concentrations suggests the <br /> (A) Cell viability measured MTT assay 48 h exposure 0.2, 0.4. 0.8, 1.6, <br /> 2.4, 3.2, 4.8 6.4 mM Cu, expressed percentage viable cells compared         relatively high Cu tolerance cells probably result of <br /> cells exposed basal media. Values means +                                                    efficient intracellular Cu sequestration decreased uptake or <br />                                                 &#226;&#710;&#8217; S.E.M. replicates independent <br /> experiments. (B) Intracellular Cu accumulation retention cells exposed 0.8, 1.6, 2.4      increased efflux. <br /> and 3.2 mM Cu labelled 64 Cu. Accumulation measured 24 h Cu exposure. <br /> Retention measured 24 h Cu exposure additional 24 h exposure basal <br /> media. Intracellular Cu normalized total cellular protein described Experimental   S2 cells express orthologues Cu homoeostasis genes <br /> section. Values means + &#226;&#710;&#8217; S.E.M. replicates independent experiments. A <br /> one-way ANOVA Games&#226;&#8364;&#8220;Howell post-hoc test used determine statistically significant       The sequencing D. melanogaster genome facilitated the <br /> differences groups. *P &lt; 0.05 compared 0.8 mM Cu accumulation; &#226;&#8364;&#160;P &lt; 0.05               identification orthologues mammalian Cu homoeostasis <br /> compared 0.8 mM Cu retention. A Pearson correlation showed Cu exposure posi-               genes involved uptake, distribution, sequestration efflux of <br /> tively correlated Cu accumulation (r = 0.98, P &lt; 0.05) Cu retention (r = 0.99,             Cu [27]. These Drosophila orthologues significant protein <br /> P &lt; 0.05).                                                                                          sequence identity similarity (Table 1). Three copper uptake <br />                                                                                                     genes, Ctr1A, Ctr1B Ctr1C, identified [10,11]. Four <br />                                                                                                     metallothioneins, designated MtnA, MtnB, MtnC MtnD [13&#226;&#8364;&#8220; <br />                                                                                                     15], MTF-1, transcription factor capable inducing the <br /> Cu concentration needed reduce cell viability 50 % control                                 expression Mtn genes, identified [15]. <br /> (EC50 ) 2.8 mM, consistent previous study                                    A default BLASTn search Drosophila genome identified <br /> found viability S2 cells decreased 70 % control                                 putative orthologues human Cu chaperones ATOX1, CCS, <br /> 48 h exposure 2 mM CuSO4 [21]. Cu tolerance S2 cells                                      COX17 SCO1 SCO2. One putative Cu-transporting <br /> much higher seen CHO cells (Chinese-hamster ovary                                      P-type ATPase identified, designated DmATP7, which <br /> cells), 24 h EC50 approx. 0.33 mM CuCl2 [19].                                       shows strong sequence similarity ATP7A ATP7B <br /> However, comparisons different cell types difficult                                     humans (Table 1). The sequence similarity human <br /> due different measures viability, different cell-culture                                      ATP7A ATP7B recently reviewed, entire <br /> media bioavailability Cu. Both CuCl2 CuSO4                                           protein sequence, ATP7A ATP7B 54 % identical; how- <br /> are used studies Cu added com-                                       ever, sequence similarity higher key motifs <br /> plex. Comparable levels Cu tolerance                                        [9,28]. These include N-terminal MBSs (metal-binding sites) <br /> other cell lines: HL-60, human leukaemia cell line,                                        characterized CXXC motif, transmembrane domains <br /> approximate EC50 1 mM exposed copper sulphate/                                           TGES/A, DKTG, TGDN AMXGDGVND motifs in <br /> nitrilotriacetate 14 h [22], rat hepatoma cell line HAC                                 major cytosolic loop. An alignment human ATP7A and <br /> has EC50 0.4 mM exposed Cu(II) acetate 72 h                                       ATP7B putative DmATP7 protein sequences using <br /> [23].                                                                                               CLUSTAL W [29] demonstrated key motifs were <br />  <br />                                                                                                                                                        <br />                                                                                                                                                      c 2004 Biochemical Society <br />  306               A. Southon others <br />  <br />  <br /> Table 1     D. melanogaster Cu homoeostasis genes expressed S2 cells <br /> Putative D. melanogaster orthologues human copper homoeostasis genes identified default BLASTn search D. melanogaster genome (http://flybase.bio.indiana.edu). Percentage <br /> protein identity similarity determined using default Gap alignment human Drosophila sequences (http://www.accelrys.com). <br />  <br />                             Human gene                                                   Drosophila              Protein sequence               Fold induction Cu <br />                             (GenBank&#194;&#174; accession ID)            Protein function          orthologue              identity/similarity (%)        (means +&#226;&#710;&#8217; S.E.M.) <br />  <br />                             hCTR1 (Al831843)                   Cu uptake                 CG3977 Ctr1A            47.6/57.8                      1.2 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.1 <br />                                                                                          CG15551                 PA 38.1/48.4                   Not detected <br />                                                                                          Ctr1C *                 PB 37.5/47.2 <br />                                                                                          CG7459 Ctr1B            38.4/46.5                      4.3 + <br />                                                                                                                                                     &#226;&#710;&#8217; 1.6 <br />                             ATOX1 (AY165037)                   Cu chaperone              CG32444                 22.4/35.8                      1.2 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.1 <br />                             CCS (AF002210)                     Cu chaperone              CG17753 CCS             51.0/64.1                      1.1 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.1 <br />                             COX17 (L77701)                     Cu chaperone              CG9065                  57.1/60.3                      0.9 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.1 <br />                             SCO1 (AF026852)                    Cu chaperone              CG8885                  51.0/62.4                      1.1 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.1 <br />                             SCO2 (AL021683)                    Cu chaperone              CG8885                  44.7/51.6 <br />                             ATP7A (L06133)                     Cu efflux                 CG1886                  49.9/58.9                      1.7 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.6 <br />                             ATP7B (U11700)                     Cu efflux                 CG1886                  49.9/59.1 <br />                             MT &#226;&#8364;&#160;                               Cu sequestration          CG9470 MtnA             N/A                            46 + <br />                                                                                                                                                    &#226;&#710;&#8217; 12&#226;&#8364;&#161; <br />                                                                Cu sequestration          CG4312 MtnB             N/A                            1443 + <br />                                                                                                                                                      &#226;&#710;&#8217; 517*&#226;&#8364;&#161; <br />                                                                Cu sequestration          CG5097 MtnC             N/A                            Not detected <br />                                                                Cu sequestration          CG33192 MtnD            N/A                            474.4 + <br />                                                                                                                                                       &#226;&#710;&#8217; 149.3*&#226;&#8364;&#161; <br />                             MTF-1 (X78710)                     MT transcription          CG3743                  PA 32.0/38.5                   1.4 + <br />                                                                                                                                                     &#226;&#710;&#8217; 0.2 <br />                                                                factor                    MTF-1*                  PB 27.0/32.4 <br />    * FlyBase identified protein sequences, designated PA PB, Ctr1C MTF-1. Gene expression measured real-time PCR normalized Actin42A expression as <br /> described Experimental section. Gene expression measured S2 cells exposed basal media 0.8 mM Cu 24 h. Values gene expression cells exposed Cu relative cells <br /> exposed basal media means + &#226;&#710;&#8217; S.E.M. replicates independent experiments. A one-sample t test used determine gene expression significantly different <br /> with Cu exposure. <br />    &#226;&#8364;&#160; Drosophila Mtn sequences similar multiple members human MT family assigned (N/A) single putative orthologue. <br />    &#226;&#8364;&#161; P &lt; 0.05. <br />  <br />  <br />  <br />  <br /> conserved DmATP7 (results shown). The conservation                                             (Figure 2D). In contrast, Ctr1A gene expression did change <br /> of key motifs Drosophila suggests protein                                       Cu exposure (results shown). To best know- <br /> probably functional Cu-transporting ATPase. Interestingly,                                      ledge, time Ctr1 gene shown be <br /> protein sequence alignment showed DmATP7                                           up-regulated Cu. Studies using Drosophila larvae that <br /> N-terminal MBS, corresponding MBS 3&#226;&#8364;&#8220;6 human ATP7A                                               Ctr1B gene expression increased Cu depletion decreased <br /> and ATP7B.                                                                                            excess Cu [10] similar results seen yeast [30]. <br />    S2 cells embryonic origin, important                                             To investigate role Cu homoeostasis genes S2 cells, <br /> establish putative Drosophila Cu genes ex-                                         utilized dsRNAi suppress gene expression Ctr1A, Ctr1B, <br /> pressed cells. Real-time PCR used examine gene                                        MTF-1 DmATP7. S2 cells treated dsRNA 48 h <br /> expression cells basal conditions exposure                                      analysis. Real-time PCR used measure gene <br /> 0.8 mM Cu. All genes Table 1, exception Ctr1C                                      expression, expressed percentage con- <br /> and MtnC, expressed S2 cells. PCR failure unlikely                                        trol cells treated non-specific dsRNA. Mean Ctr1A gene <br /> to responsible inability detect expression                                     expression reduced 9.9 % control (S.E.M. = 2.8 %), <br /> genes expression detected using adult D. melanogaster                                       Ctr1B reduced 15.0 % (S.E.M. = 4.6 %), MTF-1 was <br /> cDNA PCR conditions (results shown). A                                             reduced 24.1 % (S.E.M. = 2.6 %) DmATP7 reduced <br /> previous study using RNA blot failed detect Ctr1B                                   6.9 % (S.E.M. = 2.0 %). Suppression maintained these <br /> or Ctr1C expression S2 cells [10]. However, using real-time                                        levels 4 days 7 days (results <br /> PCR, Ctr1B detectable expression induced                                          shown). Similar dsRNAi experiments S2 cells also <br /> Cu exposure (Table 1).                                                                                shown gene suppression lasted 7 days [21]. <br />    Orthologues mammalian Cu chaperones ATOX1, CCS, <br /> COX17 SCO1 SCO2 did significantly respond trans- <br /> criptionally increased Cu levels. As expected, expression                                      Ctr1A Ctr1B transport Cu S2 cells <br /> of MtnA, MtnB MtnD genes S2 cells significantly                                            The expression Ctr1A Ctr1B suppressed using <br /> increased Cu exposure (Table 1). Drosophila Mtn expression                                      dsRNAi, Cu accumulation S2 cells determined <br /> has shown temporally tissue-specifically induced                                       exposure 2 &#194;&#181;M Cu. As expected, Ctr1A- Ctr1B- <br /> by Cu [13&#226;&#8364;&#8220;15]. To explore Mtn gene expression profile further,                                    dsRNAi-treated cells took significantly Cu control <br /> we exposed S2 cells various Cu concentrations 12, 24                                       cells 1 h (Figure 3A), exposed 2 &#194;&#181;M Cu 24 h, <br /> 48 h measured gene expression real-time PCR (Fig-                                            accumulated half Cu control cells (Figure 3B). <br /> ures 2A&#226;&#8364;&#8220;2C). Gene expression increased time- dose-                                           Suppression Ctr1 gene did affect gene expression <br /> dependent manner Mtn genes, expression                                     (results shown), suggesting unaffected <br /> of MtnB MtnD increased MtnA                                              Ctr1 function transport Cu cell. However, ap- <br /> time points. Interestingly, Cu conditions, Ctr1B gene                                     pears Ctr1A Ctr1B necessary maximal Cu <br /> expression increased time- dose-dependent manner                                        accumulation, physiological Cu conditions. Cu tolerance <br />  <br />   <br /> c 2004 Biochemical Society <br />                                                                                                                         Copper homoeostasis Drosophila S2 cells                          307 <br />  <br />  <br />  <br />  <br />                                                                                                  Figure 3     Cu accumulation S2 cells dsRNAi treatment <br />                                                                                                  S2 cells pretreated Ctr1A , Ctr1B , DmATP7 control dsRNA. (A) Cells exposed to <br />                                                                                                  2 &#194;&#181;M Cu-labelled 64 Cu, intracellular Cu measured 1 h. Cu normalized <br />                                                                                                  total cellular protein. Values means + &#226;&#710;&#8217; S.E.M. replicates independent <br />                                                                                                  experiments. Data normally distributed one-way ANOVA Games&#226;&#8364;&#8220;Howell post-hoc <br />                                                                                                  test used determine significant differences. *P &lt; 0.05 compared control. (B) Cells <br />                                                                                                  exposed 2 &#194;&#181;M Cu labelled 64 Cu, intracellular Cu measured 24 h. <br />                                                                                                  Cu normalized total cellular protein. Values means + &#226;&#710;&#8217; S.E.M. replicates <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                  independent experiments. Data normally distributed Mann&#226;&#8364;&#8220;Whitney test <br />                                                                                                  used determine significant differences. &#226;&#8364;&#160;P &lt; 0.05 compared control. <br />  <br />  <br />  <br />  <br />                                                                                                  MTF-1 suppression reduces Cu tolerance <br />                                                                                                  Cu tolerance examined suppression MTF-1. The <br />                                                                                                  viability treated cells significantly lower control <br />                                                                                                  exposed Cu (Figure 4), probably inhibition of <br /> Figure 2     Metallothionein gene expression S2 cells Cu exposure                       Cu-induced Mtn expression. To test hypothesis, exposed <br /> S2 cells exposed basal media 2, 39, 79, 157, 393 785 &#194;&#181;M Cu 12, 24        MTF-1-dsRNAi-treated cells basal media 0.8 mM Cu <br /> 48 h, MtnA (A), MtnB (B), MtnD (C) Ctr1B (D) gene expressions measured         measured expression MTF-1 Mtn genes. Re- <br /> with real-time PCR. Expression normalized Actin42A expressed relative cells        lative basal conditions, Cu significantly increased expression of <br /> exposed basal media. Data loge -transformed values means + <br />                                                                              &#226;&#710;&#8217; S.E.M.   MtnA, MtnB MtnD control cells; however, suppression <br /> replicates. The gene expression profiles MtnA , MtnB , MtnD Ctr1B significantly      MTF-1 significantly reduced response (Table 2). After <br /> related Cu exposure concentration time Cu exposure, determined general <br /> linear model univariate analysis, P &lt; 0.05. <br />                                                                                                  MTF-1 suppression, MtnA, MtnB MtnD expressions were <br />                                                                                                  reduced basal conditions, demonstrating additional <br />                                                                                                  role transcription factor absence increased Cu. <br />                                                                                                  Expression Mtn genes tended increase slightly MTF-1- <br /> was significantly affected suppression Ctr1A                                 dsRNAi-treated cells Cu exposure, probably to <br /> Ctr1B (Figure 4). Unfortunately, attempts use dsRNAi sup-                                  ability residual MTF-1 protein induce Mtn ex- <br /> press Ctr1A Ctr1B simultaneously unsuccessful;                                     pression. Alternatively, another, unidentified <br /> hence possible determine mech-                                  mechanism capable inducing limited Mtn expres- <br /> anisms Cu uptake active cells.                                                       sion response Cu. <br />  <br />                                                                                                                                                                      <br />                                                                                                                                                                    c 2004 Biochemical Society <br />  308               A. Southon others <br />  <br />  <br />                                                                                                      DmATP7 required Cu efflux S2 cells <br />                                                                                                      Cu accumulation retention studies wild-type S2 cells <br />                                                                                                      showed cells capable effluxing Cu (Figure 1A). <br />                                                                                                      Given bioinformatic analysis suggests Drosophila has <br />                                                                                                      putative Cu translocating P-type ATPase (Table 1), it <br />                                                                                                      likely protein responsible Cu efflux from <br />                                                                                                      S2 cells, similar role ATP7A ATP7B mammalian <br />                                                                                                      cells [20]. Cu accumulation examined DmATP7-dsRNAi- <br />                                                                                                      treated cells. After exposure 2 &#194;&#181;M Cu 1 h, cells did show <br />                                                                                                      significant increase Cu accumulation (Figure 3A); however, <br />                                                                                                      24 h exposure Cu resulted 4-fold increase Cu accumu- <br />                                                                                                      lation (Figure 3B). These results demonstrate DmATP7 is <br />                                                                                                      required efflux Cu S2 cells. <br />                                                                                                         Suppressing DmATP7 gene expression did significantly <br />                                                                                                      affect S2 cell viability response Cu (Figure 4), suggesting <br />                                                                                                      major copper detoxification mechanism. <br /> Figure 4     S2 cell viability dsRNAi treatment                                                It possible complete <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> DmATP7 cause <br />                                                                                                      reduced tolerance increases copper; however, level of <br /> S2 cells pretreated MTF-1, DmATP7 , Ctr1A , Ctr1B control dsRNA. Cell viability <br /> was measured MTT assay 48 h exposure 0.2, 0.4. 0.8, 1.6, 2.4, 3.2, 4.8 and <br />                                                                                                      DmATP7 gene suppression present study sufficient <br /> 6.4 mM Cu expressed percentage viable cells compared exposed               increase intracellular Cu levels significantly sig- <br /> basal media. Values means +  &#226;&#710;&#8217; S.E.M. replicates independent experiments. <br />                                                                                                      nificant effect cell viability. Since wild-type S2 cells continue <br /> Viability significantly dependent Cu exposure concentration different groups      accumulate Cu exposed increased concentrations, is <br /> as determined general linear model univariate analysis; P &lt; 0.05. A Dunnett post-hoc          probable excess intracellular Cu efficiently se- <br /> test viability MTF-1-treated cells significantly different control cells;     questered prevent cellular damage reactive oxygen spe- <br /> P &lt; 0.05. <br />                                                                                                      cies. Unfortunately, able suppress expression <br />                                                                                                      DmATP7 MTF-1 dsRNAi simultaneously. <br />                                                                                                      We predict S2 cells lacking significant levels DmATP7 <br />    Interestingly, MTF-1 suppression reduced basal                                       MTF-1 extremely sensitive Cu exposure, there <br /> Cu-induced expressions Ctr1B (Table 2). Ctr1B contains sev-                                       does appear P-type ATPase Cu efflux, and <br /> eral copies metal-response elements bound                                         Mtn gene expression significantly induced Cu in <br /> MTF-1, suggesting transcription factor regulate                                        S2 cells reduced MTF-1 expression (Table 2). <br /> Ctr1B Mtns [10]. Studies Drosophila shown                                        We examined mechanisms Cu homoeostasis Droso- <br /> that Ctr1B [10] MTF-1 [15] knockout flies sensitive                                       phila S2 cells accumulate export <br /> both increased decreased Cu levels. Zhou et al. [10] proposed                                    Cu comparable way mammalian cells, an <br /> that sensitivity MTF-1 null flies low Cu levels                                  expression orthologues mammalian genes involved Cu <br /> loss Ctr1B regulation sensitivity Ctr1B null flies                                    uptake, transport, sequestration efflux. Although Ctr1A <br /> to high Cu levels potential role protein Cu                                  Ctr1B involved Cu uptake, Cu-induced expression <br /> detoxification. This supported studies yeast Ctr1                                     Ctr1B mediated MTF-1, suggesting dual role Cu <br /> human Ctr1 that excess Cu stimulates endocytosis                                       uptake detoxification. Sequestration appears primary <br /> of proteins [31,32]. If Ctr1B translocates response Cu                                method Cu detoxification S2 cells, preventing induction of <br /> would interesting explore occurs prevent                                  Mtn expression suppressing MTF-1 reduces copper tolerance. <br /> excess Cu uptake sequester detoxify intracellular Cu                                       However, DmATP7 important, essential efflux <br /> actively. The MTF-1-mediated up-regulation Ctr1B expression                                       excess Cu. This work consistent previous studies <br /> found S2 cells exposed Cu supports active detoxification                                    mammalian cells validates S2 cells efficient mal- <br /> role Ctr1B, protein mediate sequestration                                    leable exploring mechanisms Cu homo- <br /> excess Cu.                                                                                           eostasis. <br />  <br />  <br />  <br />  <br /> Table 2     Gene expression S2 cells MTF-1 dsRNAi treatment <br /> S2 cells pretreated MTF-1 control dsRNA exposed basal media 0.8 mM Cu 24 h. MTF-1, MtnA , MtnB MtnD gene expressions measured with <br /> real-time PCR normalized Actin42A expression. Values expressed relative control cells exposed basal media means + <br />                                                                                                                                               &#226;&#710;&#8217; S.E.M. replicates independent <br /> experiments. MTF-1, MtnA Ctr1B gene expression normally distributed one-way ANOVA Games&#226;&#8364;&#8220;Howell post-hoc test used determine significant differences. * P &lt; 0.05 <br /> compared control cells exposed basal media. &#226;&#8364;&#160; P &lt; 0.05 compared MTF-1 cells exposed basal media. MtnB MtnD gene expression normally distributed a <br /> Mann&#226;&#8364;&#8220;Whitney test used determine significant differences. &#226;&#8364;&#161; P &lt; 0.05 compared control cells exposed basal media. &#194;&#167; P &lt; 0.05 compared MTF-1 cells exposed basal <br /> media. <br />  <br />                              dsRNA           Cu (mM)           MTF-1                 MtnA                MtnB                 MtnD                   Ctr1B <br />  <br />                              Control         Basal             1.00 + <br />                                                                     &#226;&#710;&#8217; 0.05           1.00 + <br />                                                                                           &#226;&#710;&#8217; 0.04         1.00 + <br />                                                                                                               &#226;&#710;&#8217; 0.04          1.00 + <br />                                                                                                                                    &#226;&#710;&#8217; 0.10            1.00 + <br />                                                                                                                                                           &#226;&#710;&#8217; 0.03 <br />                              Control         0.8               1.87 + <br />                                                                     &#226;&#710;&#8217; 0.05*          5.79 + <br />                                                                                           &#226;&#710;&#8217; 0.23*        402 + <br />                                                                                                              &#226;&#710;&#8217; 44.6&#226;&#8364;&#160;          1326 + <br />                                                                                                                                    &#226;&#710;&#8217; 156&#226;&#8364;&#160;            2.71 + <br />                                                                                                                                                           &#226;&#710;&#8217; 0.11* <br />                              MTF-1           Basal             0.24 + <br />                                                                     &#226;&#710;&#8217; 0.01*          0.20 + <br />                                                                                           &#226;&#710;&#8217; 0.01*        0.18 + <br />                                                                                                               &#226;&#710;&#8217; 0.02&#226;&#8364;&#160;         0.34 + <br />                                                                                                                                    &#226;&#710;&#8217; 0.05&#226;&#8364;&#160;           0.32 + <br />                                                                                                                                                           &#226;&#710;&#8217; 0.05* <br />                              MTF-1           0.8               0.26 + <br />                                                                     &#226;&#710;&#8217; 0.04*          1.51 + <br />                                                                                           &#226;&#710;&#8217; 0.14*&#226;&#8364;&#161;       3.43 + <br />                                                                                                               &#226;&#710;&#8217; 0.27&#226;&#8364;&#160;&#194;&#167;        6.54 + <br />                                                                                                                                    &#226;&#710;&#8217; 1.29&#226;&#8364;&#160;&#194;&#167;          0.60 + <br />                                                                                                                                                           &#226;&#710;&#8217; 0.08* <br />  <br />  <br />  <br />   <br /> c 2004 Biochemical Society <br />                                                                                                                             Copper homoeostasis Drosophila S2 cells                            309 <br />  <br />  <br /> This work supported grants International Copper Association The                  17 Lorentzos, P., Kaiser, T., Kennerson, M. L. Nicholson, G. A. (2003) A rapid and <br /> Australian Institute Nuclear Science Engineering. R. B. recipient Peter               definitive test Charcot-Marie-Tooth 1A hereditary neuropathy liability to <br /> Doherty Fellowship National Health Medical Research Council currently                  pressure palsies using multiplexed real-time PCR. Genet. Test. 7, 135&#226;&#8364;&#8220;138 <br /> receives J. N. Peters Bequest Fellowship University Melbourne. MTF-1 cDNA              18 Worby, C. A., Simonson-Leff, N. Dixon, J. E. (2001) RNA interference gene <br /> was gift Professor Walter Schaffner (Institute Molecular Biology, University               expression (RNAi) cultured Drosophila cells, Science STKE 2001(95), PL1 <br /> Z&#194;&#168;urich, Z&#194;&#168;urich, Switzerland). Ian Gordon Statistical Consulting Centre             19 La Fontaine, S., Firth, S. D., Lockhart, P. J., Brooks, H., Parton, R. G., Camakaris, J. and <br /> University Melbourne acknowledged statistical assistance.                                     Mercer, J. F. (1998) Functional analysis intracellular localization human <br />                                                                                                         Menkes protein (MNK) stably expressed cDNA construct Chinese hamster ovary <br />                                                                                                         cells (CHO-K1). Hum. Mol. Genet. 7, 1293&#226;&#8364;&#8220;1300 <br />                                                                                                      20 Camakaris, J., Petris, M. J., Bailey, L., Shen, P., Lockhart, P., Glover, T. W., Barcroft, C., <br /> REFERENCES <br />                                                                                                         Patton, J. Mercer, J. F. (1995) Gene amplification Menkes (MNK; ATP7A) <br />  1 Linder, M. C. Hazegh-Azam, M. (1996) Copper biochemistry molecular biology.                  P-type ATPase gene CHO cells associated copper resistance enhanced <br />    Am. J. Clin. Nutr. 63, 797S&#226;&#8364;&#8220;811S                                                                     copper efflux. Hum. Mol. Genet. 4, 2117&#226;&#8364;&#8220;2123 <br />  2 Halliwell, B. Gutteridge, J. M. (1984) Oxygen toxicity, oxygen radicals, transition           21 Radyuk, S. N., Sohal, R. S. Orr, W. C. (2003) Thioredoxin peroxidases foster <br />    metals disease. Biochem. J. 219, 1&#226;&#8364;&#8220;14                                                            cytoprotection cell death response different stressors: over- under-expression <br />  3 Camakaris, J., Voskoboinik, I. Mercer, J. F. (1999) Molecular mechanisms copper               thioredoxin peroxidase Drosophila cells. Biochem. J. 371, 743&#226;&#8364;&#8220;752 <br />    homeostasis. Biochem. Biophys. Res. Commun. 261, 225&#226;&#8364;&#8220;232                                          22 Kawai, K., Liu, S. X., Tyurin, V. A., Tyurina, Y. Y., Borisenko, G. G., Jiang, J. F., St Croix, <br />  4 Zhou, B. Gitschier, J. (1997) hCTR1: human gene copper uptake identified                C. M., Fabisiak, J. P., Pitt, B. R. Kagan, V. E. (2000) Antioxidant antiapoptotic <br />    complementation yeast. Proc. Natl. Acad. Sci. U.S.A. 94, 7481&#226;&#8364;&#8220;7486                                function metallothioneins HL-60 cells challenged copper nitrilotriacetate. <br />  5 Culotta, V. C., Klomp, L. W., Strain, J., Casareno, R. L., Krems, B. Gitlin, J. D. (1997)        Chem. Res. Toxicol. 13, 1275&#226;&#8364;&#8220;1286 <br />    The copper chaperone superoxide dismutase. J. Biol. Chem. 272, 23469&#226;&#8364;&#8220;23472                    23 Freedman, J. H., Weiner, R. J. Peisach, J. (1986) Resistance copper toxicity of <br />  6 Amaravadi, R., Glerum, D. M. Tzagoloff, A. (1997) Isolation cDNA encoding               cultured hepatoma cells. Characterization resistant cell lines. J. Biol. Chem. 261, <br />    human homolog COX17, yeast gene essential mitochondrial copper recruitment.                 11840&#226;&#8364;&#8220;11848 <br />    Hum. Genet. 99, 329&#226;&#8364;&#8220;333                                                                           24 Czaja, M. J., Weiner, F. R. Freedman, J. H. (1991) Amplification the <br />  7 Klomp, L. W., Lin, S. J., Yuan, D. S., Klausner, R. D., Culotta, V. C. Gitlin, J. D. (1997)      metallothionein-1 metallothionein-2 genes copper-resistant hepatoma cells. <br />    Identification functional expression HAH1, novel human gene involved copper              J. Cell. Physiol. 147, 434&#226;&#8364;&#8220;438 <br />    homeostasis. J. Biol. Chem. 272, 9221&#226;&#8364;&#8220;9226                                                        25 Petris, M. J., Camakaris, J., Greenough, M., LaFontaine, S. Mercer, J. F. (1998) <br />  8 Ghoshal, K. Jacob, S. T. (2001) Regulation metallothionein gene expression.                   A C-terminal di-leucine required localization Menkes protein the <br />    Prog. Nucleic Acid Res. Mol. Biol. 66, 357&#226;&#8364;&#8220;384                                                       trans-Golgi network. Hum. Mol. Genet. 7, 2063&#226;&#8364;&#8220;2071 <br />  9 Voskoboinik, I., Camakaris, J. Mercer, J. F. (2002) Understanding mechanism           26 Voskoboinik, I., Strausak, D., Greenough, M., Brooks, H., Petris, M., Smith, S., Mercer, <br />    function copper P-type ATPases. Adv. Protein Chem. 60, 123&#226;&#8364;&#8220;150                                    J. F. Camakaris, J. (1999) Functional analysis N-terminal CXXC metal-binding <br /> 10 Zhou, H., Cadigan, K. M. Thiele, D. J. (2003) A copper-regulated transporter required            motifs human Menkes copper-transporting P-type ATPase expressed cultured <br />    copper acquisition, pigmentation, specific stages development Drosophila               mammalian cells. J. Biol. Chem. 274, 22008&#226;&#8364;&#8220;22012 <br />    melanogaster. J. Biol. Chem. 278, 48210&#226;&#8364;&#8220;48218                                                     27 FlyBase Consortium (2003) The FlyBase database Drosophila genome projects and <br /> 11 Lee, J., Petris, M. J. Thiele, D. J. (2002) Characterization mouse embryonic cells            community literature. Nucleic Acids Res. 31, 172&#226;&#8364;&#8220;175 <br />    deficient ctr1 high affinity copper transporter. Identification Ctr1-independent      28 Lutsenko, S. Petris, M. J. (2002) Function regulation mammalian <br />    copper transport system. J. Biol. Chem. 277, 40253&#226;&#8364;&#8220;40259                                             copper-transporting ATPases: insights biochemical cell biological approaches. <br /> 12 Garrett, S. H., Somji, S., Todd, J. H., Sens, M. A. Sens, D. A. (1998) Differential              J. Membr. Biol. 191, 1&#226;&#8364;&#8220;12 <br />    expression human metallothionein isoform I mRNA human proximal tubule                       29 Thompson, J. D., Higgins, D. G. Gibson, T. J. (1994) CLUSTAL W: improving the <br />    cells exposed metals. Environ. Health Perspect. 106, 825&#226;&#8364;&#8220;832                                      sensitivity progressive multiple sequence alignment sequence weighting, <br /> 13 Mokdad, R., Debec, A. Wegnez, M. (1987) Metallothionein genes Drosophila                      position-specific gap penalties weight matrix choice. Nucleic Acids Res. 22, <br />    melanogaster constitute dual system. Proc. Natl. Acad. Sci. U.S.A. 84, 2658&#226;&#8364;&#8220;2662                   4673&#226;&#8364;&#8220;4680 <br /> 14 Lastowski-Perry, D., Otto, E. Maroni, G. (1985) Nucleotide sequence expression            30 Labbe, S., Zhu, Z. Thiele, D. J. (1997) Copper-specific transcriptional repression <br />    Drosophila metallothionein. J. Biol. Chem. 260, 1527&#226;&#8364;&#8220;1530                                       yeast genes encoding critical components copper transport pathway. <br /> 15 Egli, D., Selvaraj, A., Yepiskoposyan, H., Zhang, B., Hafen, E., Georgiev, O.                    J. Biol. Chem. 272, 15951&#226;&#8364;&#8220;15958 <br />    Schaffner, W. (2003) Knockout &#226;&#8364;&#732;metal-responsive transcription factor&#226;&#8364;&#8482; MTF-1                 31 Ooi, C. E., Rabinovich, E., Dancis, A., Bonifacino, J. S. Klausner, R. D. (1996) <br />    Drosophila homologous rec <br /> </body></html>