<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 20 No. 15 pp. 4183&#194;&#177;4193, 2001 <br />  <br />  <br /> An essential nuclear envelope integral membrane <br /> protein, Brr6p, required nuclear transport <br />  <br />  <br />  <br /> Anne Bruyn Kops Christine Guthrie1                         karyopherin b family number unrelated <br /> Department Biochemistry Biophysics, UCSF Medical School, <br />                                                                   transport factors Ntf2 Mex67 (reviewed in <br /> 513 Parnassus Avenue, San Francisco, CA 94143, USA                Pemberton et al., 1998; Wozniak et al., 1998; Go&#195;&#710;rlich and <br /> 1 <br />                                                                   Kutay, 1999). <br />  Corresponding author <br /> e-mail: guthrie@cgl.ucsf.edu <br />                                                                      Interactions carriers nucleoporins lining <br />                                                                   channel pore thought important for <br /> Despite rapid advances understanding                transport (reviewed Fabre Hurt, 1997; Pemberton <br /> function nuclear pore complex nuclear                   et al., 1997; Wozniak et al., 1998; Go&#195;&#710;rlich Kutay, <br /> transport, little known role nuclear             1999; Wente, 2000), precise roles of <br /> envelope itself play critical process. A              particular nucleoporins understood. Our <br /> small number integral membrane proteins speci&#194;&#174;c                knowledge functional architecture yeast <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> envelope identi&#194;&#174;ed budding yeast,             nuclear pore bene&#194;&#174;ted greatly detailed study of <br /> however, reported affect transport.              isolated NPCs carried recently Rout et al. (2000). <br /> We identi&#194;&#174;ed essential gene, BRR6,                  The study produced working model NPC, <br /> product, Brr6p, behaves like nuclear envelope inte-             con&#194;&#174;rming presence ~30 yeast nucleoporins the <br /> gral membrane protein. Notably, brr6-1 mutant                 pore placing spatially respect known <br /> speci&#194;&#174;cally affects transport mRNA protein               ultrastructural features central channel and <br /> reporter containing nuclear export signal. In add-              nuclear basket. Many proteins likely to <br /> ition, Brr6p depletion alters nucleoporin distribution            function targeting transport substrates NPC <br /> and nuclear envelope morphology, suggesting              passage cytoplasm. <br /> protein required spatial organization                  In addition nucleoporins, NE NPC <br /> nuclear pores. BRR6 interacts genetically sub-             play key role transport, possibly providing <br /> set nucleoporins, Brr6-green &#194;&#175;uorescent                    binding sites transport factors anchoring pore <br /> protein (GFP) localizes punctate nuclear rim                 structures. Interestingly, antibody gp210, a <br /> <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, suggesting location near nuclear               mammalian integral membrane protein located pore <br /> pore. However, Brr6-GFP fails redistribute                membrane domain bridging inner outer NE <br /> Dnup133 mutant, distinguishing Brr6p known                   membranes (reviewed Gerace Burke, 1988; <br /> proteins pore membrane domain. We hypothe-                 Dingwall Laskey, 1992; Gant Wilson, 1997), <br /> size Brr6p located adjacent nuclear pore           shown inhibit nuclear import (Greber and <br /> and interacts functionally pore transport            Gerace, 1992). Although mechanism inhibition <br /> machinery.                                                        unknown, result suggests integral membrane <br /> Keywords: integral membrane protein/mRNA export/                  proteins affect transport events. <br /> nuclear envelope/nuclear transport                                   In yeast, integral membrane proteins speci&#194;&#174;c to <br />                                                                   NE identi&#194;&#174;ed. Two these, Pom152p and <br />                                                                   Pom34p (Wozniak et al., 1994; Strambio-de-Castillia <br />                                                                   et al., 1995; Tcheperegine et al., 1999; Rout et al., 2000), <br /> Introduction                                                      co-purify yeast nuclear pores. Both believed to <br /> The presence nuclear envelope (NE) eukaryotes             reside pore membrane domain function in <br /> divides cell nuclear cytoplasmic compart-            nuclear pore assembly anchoring. The third, Ndc1p, is <br /> ments, greatly enhancing capacity control gene             inner envelope protein interacts NPC <br /> expression. Transport proteins ribonucleoprotein           spindle pole body (Chial et al., 1998). In addition, <br /> particles (RNPs) nucleus cytoplasm                yeast proteins, Snl1p, Nem1p Spo7p, common to <br /> through nuclear pore complex (NPC) central         NE endoplasmic reticulum, been <br /> ability (reviewed Go&#195;&#710;rlich Kutay, 1999; Nakielny           shown affect NPC (Ho et al., 1998; Siniossoglou <br /> and Dreyfuss, 1999). The structure NPC                  et al., 1998). However, known yeast NE <br /> conserved yeast humans, consisting highly            integral membrane proteins reported affect <br /> symmetrical core structure organized central pore        nuclear transport. <br /> channel peripheral cytoplasmic &#194;&#174;laments               Here report characterization essential yeast <br /> nucleoplasmic basket structures thought play key roles         protein, Brr6p, properties NE integral <br /> in initiation termination transport pathways               membrane protein affects speci&#194;&#174;c nuclear transport <br /> (reviewed Franke Scheer, 1974; Davis, 1995;                pathways. The BRR6 gene interacts genetically a <br /> Stof&#194;&#175;er et al., 1999; Wente, 2000). Transport different        subset nucleoporins, loss Brr6p function causes <br /> types protein RNP cargoes NPC               redistribution Nsp1p Nup188-green &#194;&#175;uorescent <br /> mediated speci&#194;&#174;c protein carriers, including members           protein (GFP) aberrant envelope pore <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                               4183 <br />  A.de Bruyn Kops C.Guthrie <br />  <br />  <br />  <br />  <br />                                                                             Fig. 2. The brr6-1 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> causes defect NES protein export. <br />                                                                             Shown localization patterns NLS/NES(GFP)2 and <br /> Fig. 1. The brr6-1 mutant accumulates bulk poly(A) RNA               NLS(GFP)3 reporters BRR6 brr6-1 cells indicated. The NLS/ <br /> nucleus nuclear rim. Shown mRNA localization             NES construct accumulates nuclear rim brr6-1 con&#194;&#174;rmed <br /> patterns BRR6 brr6-1 cells 30&#194;&#176;C determined situ             DAPI staining (data shown). The NLS construct shows <br /> hybridization digoxygenin-labeled oligo dT50 probe. DAPI             nucleoplasmic staining mutant wild-type cells. Bar, 10 mm. <br /> staining used con&#194;&#174;rm locations cell nuclei. The inserts <br /> in brr6-1 panels dT50 DAPI staining single nucleus <br /> enlarged mRNA accumulation nuclear rim evident        shown), indicating likely loss-of- <br /> some mutant cells. Bar, 10 mm. <br />                                                                             function phenotypes. In brr6-1 cells, dT50 signal <br />                                                                             clearly concentrated nuclear rim (Figure 1, <br />                                                                             insert), suggesting BRR6 play role step of <br /> morphologies. Strikingly, brr6-1 cold-sensitive (cs)                    mRNA export occurring near nuclear pore. <br /> allele accumulates mRNA nuclear export signal <br /> (NES) protein reporter nuclear rim. Thus, Brr6p <br />                                                                             brr6-1 defective NES protein transport <br /> represents &#194;&#174;rst example yeast NE integral <br />                                                                             The factors known affect mRNA export yeast be <br /> membrane protein impacts nuclear transport. <br />                                                                             divided general categories: appear be <br />                                                                             dedicated mRNA affect protein <br />                                                                             transport pathways (reviewed Nakielny Dreyfuss, <br /> Results                                                                     1999). Our situ hybridization results suggested role for <br /> BRR6 identi&#194;&#174;ed complementation                           BRR6 mRNA export; assess BRR6 also <br /> growth defect brr6-412, original isolates                  functions protein transport, examined localiza- <br /> the brr6-1 cs mutant obtained in situ hybridization                   tion number different GFP-tagged protein transport <br /> screen cs mRNA export mutants (see Materials                        reporters living BRR6 brr6-1 cells. Reporters were <br /> methods). The 197 aa BRR6 open reading frame (ORF)                          selected known utilize different protein <br /> [Saccharomyces cerevisiae Genome Database (SGD)                             transport pathways. The set included diffusible and <br /> accession No. YGL247w] predicted encode                            non-diffusible SV40 nuclear localization signal (NLS)- <br /> essential 22.8 kDa protein unknown function.                             GFP constructs [NLS-GFP, NLS(GFP)3], SV40 NLS/ <br /> Disruption BRR6 ORF HIS3 marker                             NES-GFP reporter [NLS/NES(GFP)2], ribosomal protein <br /> con&#194;&#174;rmed gene essential. The brr6-1 allele                  NLS reporter (L25-GFP) GFP-tagged forms of <br /> found contain single, conservative arginine lysine                  known mRNA binding proteins, Npl3p Nab2p. <br /> change amino acid 110. Isogenic BRR6 brr6-1                             Of reporters tested, NLS/NES(GFP)2 <br /> strains generated integrating wild-type mutant                  construct showed change localization. In BRR6 <br /> alleles Dbrr6::HIS3 deletion strain (see Materials                   cells (Figure 2), reporter showed expected wild- <br /> and methods details). The resulting brr6-1 mutant                       type cytoplasmic distribution reported previously (Stade <br /> showed moderate growth defect 30&#194;&#176;C,                          et al., 1997). Interestingly, half brr6-1 mutant <br /> exacerbated 16&#194;&#176;C, BRR6 strain indistin-                    cells GFP signal showed pronounced accumulation <br /> guishable wild-type parent (data shown).                       reporter nuclear rim, consistent defect <br />                                                                             NES protein transport. In contrast, distribution an <br /> The brr6-1 mutant accumulates mRNA                                   NLS(GFP)3 reporter lacking NES sequence was <br /> nucleus nuclear periphery                                        unaffected brr6-1 (Figure 2). Similarly, defects <br /> Using digoxygenin-labeled dT50 probe, examined                         observed using diffusible NLS-GFP reporter (data <br /> the mRNA situ hybridization patterns BRR6                         shown) steady state experiments the <br /> brr6-1 cells maintained 30&#194;&#176;C (Figure 1) shifted                    kinetic protein import assay developed Goldfarb and <br /> 16&#194;&#176;C (data shown). At temperatures, BRR6 cells                     colleagues (Shulga, 1996). The distributions L25- <br /> showed cell dT50 staining typical wild-type                    GFP reporter, Npl3p Nab2p unchanged in <br /> cells. In contrast, brr6-1 cells clear staining cell             brr6-1 (data shown). The effect NLS/NES(GFP)2 <br /> nucleus 30&#194;&#176;C 16&#194;&#176;C. Thus, brr6-1 exhibits                 protein reporters implies defect speci&#194;&#174;c to <br /> constitutive nuclear mRNA export defect. A strain                        NES protein transport pathway. Interestingly, NES <br /> which copy BRR6 control                    protein exporter Xpo1p (reviewed Go&#195;&#710;rlich Kutay, <br /> repressible GAL1-GAL10 promoter showed                          1999) continued wild-type nucleoplasmic and <br /> growth defect nuclear mRNA accumulation                            nuclear rim distribution brr6-1 using Xpo1-GFP <br /> grown 5 h media containing glucose (data                         fusion protein (data shown). <br />  <br /> 4184 <br />                                                                                        Brr6p nuclear envelope protein transport <br />  <br />  <br />                                                                        et al., 1998), suggesting Brr6p located or <br />                                                                        near pores. Interestingly, overexpression Brr6-GFP <br />                                                                        resulted dramatic localization nuclear and <br />                                                                        cellular peripheries (Figure 3A). The distribution <br />                                                                        resembles ER markers Sec63-GFP GFP- <br />                                                                        HDEL living yeast cells (Prinz et al., 2000) most <br />                                                                        likely represents accumulation excess Brr6p the <br />                                                                        peripheral ER. Similar localization NE the <br />                                                                        ER shown Spo7p Nem1p (Siniossoglou <br />                                                                        et al., 1998) mammalian nuclear pore <br />                                                                        membrane protein, Pom121, overexpression <br />                                                                        (Soderqvist et al., 1997). <br />  <br />                                                                        BRR6 interacts genetically nucleoporin genes <br />                                                                        Consistent possibility Brr6p located or <br />                                                                        near NPC, BRR6 shows genetic interactions with <br />                                                                        nuclear pore components. A haploid double mutant <br />                                                                        carrying cs allele nucleoporin, nup188 (brr7-1; <br />                                                                        A.de Bruyn Kops C.Guthrie, unpublished results), and <br />                                                                        Dbrr6::HIS3 deletion covered brr6-1 TRP <br />                                                                        plasmid wild-type BRR6 URA plasmid viable, <br />                                                                        loss URA plasmid 5&#194;&#162;-&#194;&#175;uoroorotic <br />                                                                        acid (5-FOA)-containing medium lethal (data not <br />                                                                        shown), indicating brr6-1 synthetically lethal with <br />                                                                        nup188cs/brr7-1. Moreover, overexpression BRR6 <br />                                                                        (BRR6 2 mm plasmid) Dnup188 Dnup1 deletion <br />                                                                        strains dramatically impaired growth, overexpres- <br />                                                                        sion modest effect wild-type W303 or <br />                                                                        Dnup116 cells (Figure 3B). A Dnup2 strain a <br />                                                                        C-terminal Dnic96 deletion mutant [Dnic96(532&#194;&#177;839)] as <br />                                                                        xpo1-1 gle2-1 (RNA export factor) <br /> Fig. 3. Brr6-GFP localizes nuclear rim. (A) The localization    temperature-sensitive (ts) mutants showed synthetic <br /> patterns Brr6-GFP fusion protein expressed low copy        lethality BRR6 using assay (data shown). <br /> (CEN/ARS) vector high copy 2 mm plasmid indicated. At low <br /> copy, Brr6-GFP shows predominantly punctate nuclear rim staining. <br />                                                                        Interestingly, locations Nup1p (Rout et al., 2000) <br /> Upon overexpression, Brr6-GFP accumulates nuclear cell      portion Nic96p (Fahrenkrog et al., 1998) detected <br /> peripheries. (B) The effect BRR6 (BRR6/pRS424) overexpression       immuno-electron microscopy coincide nuclear <br /> relative vector (pRS424) growth wild-type (W303),   basket, pointing possible impact BRR6 basket <br /> Dnup116::HIS3, Dnup1::HIS3 Dnup188::HIS3 strains (all W303         function. Importantly, number strains tested contain- <br /> strain background) -TRP medium. BRR6 overexpression causes <br /> dramatic growth defects Dnup1 Dnup188 cells mild      ing deletions mutations nucleoporins [including <br /> effects wild-type Dnup116 cells. (C) Nup188-GFP Brr6-       Dnup100, Dnup133, Dnup145, Dnic96(28&#194;&#177;63), brr3/ <br /> GFP (Brr6-GFP/pRS424) localization patterns wild-type (RS456)       gle1cs ts alleles] showed effect, indicating the <br /> and Dnup133 cells. In contrast Nup188-GFP, Brr6-GFP fails        observed interaction speci&#194;&#174;c subset nucleoporins <br /> show clustering Dnup133 mutant. Bar, 10 mm. <br />                                                                        transport components. <br />  <br />                                                                        Brr6-GFP localization unchanged a <br /> Brr6p located nuclear rim                                    pore-clustering mutant <br /> The mRNA export NES protein transport defects                      Although BRR6 interacts genetically nucleoporins, <br /> observed brr6-1, apparent absence                 Brr6p identi&#194;&#174;ed recent NPC preparations <br /> effects protein reporters, argue BRR6 plays            reported Rout et al. (2000), raising possibility that <br /> role mRNA NES transport pathways. The accumu-                   Brr6p interact pore the <br /> lation mRNA NES reporter nuclear                NPC se. Another criterion frequently used to <br /> rim suggests BRR6 act NE. To determine                 evaluate protein nucleoporin its <br /> the location BRR6 gene product, examined                 distribution altered nucleoporin <br /> distribution Brr6-GFP fusion protein living cells           mutants (e.g. Dnup133; Doye et al., 1994; Pemberton <br /> &#194;&#175;uorescence microscopy. When Brr6-GFP expressed                    et al., 1995) known cause pore clustering. To test the <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> low copy plasmid, GFP signal observed                   behavior Brr6p assay, compared the <br /> predominantly nuclear rim, suggesting Brr6p                distribution Brr6-GFP tagged nucleoporin, <br /> associates NE (Figure 3A). Notably, Brr6-                 Nup188-GFP, Dnup133 strain. As predicted, the <br /> GFP rim staining pattern punctate, similar seen            pattern Nup188-GFP signal changed nuclear rim <br /> with nuclear pore components (Rout et al., 2000). The                  wild-type strain characteristic single bright <br /> pattern distinctly different uniform rim                  nuclear spot Dnup133 mutant (Figure 3C). Attempts <br /> distribution non-pore-associated NE integral membrane               localize Brr6-GFP Dnup133 nucleo- <br /> proteins, Spo7p Nem1p yeast (Siniossoglou               porin deletion mutants known cause pore clustering <br />  <br />                                                                                                                              4185 <br />  A.de Bruyn Kops C.Guthrie <br />  <br />  <br /> using low copy Brr6-GFP construct unsuccessful <br /> because GFP signal low. Efforts localize <br /> the protein immuno&#194;&#175;uorescence using number of <br /> epitope tags unsuccessful, likely <br /> because small size, membrane association and <br /> apparent low abundance protein. Similarly, have <br /> not able generate usable antibody against <br /> Brr6p itself. However, experiment per- <br /> formed using high copy Brr6-GFP construct (Figure 3C), <br /> wild-type Dnup133 cells showed indistinguishable <br /> Brr6-GFP distributions. A caveat experiments is <br /> the possibility excess Brr6-GFP remain dis- <br /> tributed NE. Nonetheless, given the <br /> dramatic nature clustering <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>phenotype</span>, would <br /> still expect intensi&#194;&#174;ed signal location if <br /> clustering occurred. The failure Brr6-GFP cluster in <br /> this assay distinguishes Brr6p nucleoporins as <br /> from pore membrane proteins, Pom152p Pom34p, <br /> which behave like canonical nucleoporins this <br /> assay NPC preparation (Rout et al., 2000). <br />  <br /> The extraction pro&#194;&#174;le Brr6p crude lysates <br /> typi&#194;&#174;es integral membrane protein <br /> The Brr6-GFP localization results indicate Brr6p is <br /> most likely membrane component concentrated the <br /> NE. As such, Brr6p peripheral membrane <br /> protein, associated envelope interactions with <br /> nucleoporins envelope-associated proteins, an <br /> integral membrane protein. One characteristics that <br /> distinguishes integral peripheral membrane proteins <br /> is relative solubility different extraction <br /> conditions. While peripheral proteins readily <br /> removed treatment high ionic strength high <br /> pH, solubilization integral membrane proteins requires <br /> detergent disrupt lipid&#194;&#177;lipid lipid&#194;&#177;protein inter- <br /> actions. We tested ability high salt, high pH   Fig. 4. The extraction pro&#194;&#174;le Brr6p crude lysates typi&#194;&#174;es an <br /> detergent solubilize Brr6p.                                  integral membrane protein. Shown analyses Brr6-GFP (A), <br />                                                                 Pom152 (B) Nup188-HA (C) extracted whole-cell lysates <br />    Whole-cell extracts prepared Dbrr6 cells           using different extraction conditions: (i) 150 mM NaCl; (ii) 1 M <br /> containing low copy BRR6-GFP construct using             NaCl; (iii) 150 mM NaCl, 100 mM sodium carbonate; (iv) 150 mM <br /> different extraction conditions (see Materials              NaCl, 4% Triton X-100. Supernatant (S) pellet (P) lanes are <br /> methods). Extracts centrifuged 100 000 g            indicated. Western blots probed antibodies GFP, <br />                                                                 Pom152 HA tag. Brr6-GFP Pom152 present the <br /> pellet supernatant samples assayed western          supernatant detergent treatment, Nup188-HA also <br /> blotting antibody speci&#194;&#174;c GFP. Brr6-GFP             evident high salt high pH supernatants. <br /> remained associated cell pellets samples <br /> extracted low salt, high salt high pH conditions <br /> but ef&#194;&#174;ciently extracted low salt buffer containing <br /> 4% Triton X-100 (Figure 4A). This pro&#194;&#174;le expected        weaker candidate, present near N-terminus (residues <br /> for integral membrane protein similar         58&#194;&#177;74) (Figure 5A). The use these <br /> observed known nuclear envelope integral mem-           sequences anchor Brr6 membrane dictate <br /> brane protein Pom152p (Figure 4B). In contrast, HA-             topology protein respect bilayer and <br /> tagged Nup188p, non-integral membrane nuclear pore            hence, availability different parts protein for <br /> component, extracted high salt high          functional interactions. We particularly interested in <br /> pH conditions (Figure 4C). Thus, crude lysates, Brr6p        determining orientation brr6-1 mutation (aa 110) <br /> exhibits properties characteristic integral membrane         respect NE lumen. The mutation located <br /> proteins.                                                       predicted transmembrane sequences and <br />                                                                 lies region BRR6 shows striking homology <br /> The C-terminus Brr6p sequestered                   (31% identical, 49% similar) Saccharomyces <br /> NE lumen                                                        cerevisiae karyopherin gene, KAP123 (residues 247&#194;&#177; <br /> Consistent solubility behavior Brr6p             312), implicated ribosomal protein import mRNA <br /> extracts, sequence analysis BRR6 ORF revealed            export (Rout et al., 1997; Schlenstedt et al., 1997; Seedorf <br /> two hydrophobic stretches potential function        Silver, 1997) (Figure 5A). The region homology <br /> as transmembrane domains. One located near               comprises ~1/3 BRR6 ORF (residues 52&#194;&#177;120) and <br /> C-terminus (residues 158&#194;&#177;174) second, somewhat            re&#194;&#175;ect interaction Brr6p Kap123p a <br />  <br /> 4186 <br />                                                                                                 Brr6p nuclear envelope protein transport <br />  <br />  <br />  <br />  <br /> Fig. 5. Trypsin digestion yields detergent-sensitive C-terminal Brr6 fragment. (A) The BRR6 ORF, showing locations brr6-1 mutation <br /> (asterisk), KAP123 homology predicted transmembrane domains. A sequence alignment homologous portions BRR6 and <br /> KAP123 shown. No signi&#194;&#174;cant similarity BRR6 karyopherin b family prototype, KAP95. (B) The results trypsin <br /> digestion experiments carried whole-cell lysates containing Brr6-GFP polyoma-Brr6 constructs tagged C- N-termini, <br /> respectively. Western blot analyses using antibodies GFP polyoma tags demonstrate protection C-terminal Brr6-GFP <br /> fragment (asterisk), sensitive detergent. No protected N-terminal fragments observed. (C) The likely topology Brr6p the <br /> membrane. The ends region KAP123 homology (arrows) site brr6-1 mutation (asterisk) indicated. <br />  <br />  <br />  <br /> common transport factor nucleoporin. For                         <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> protein (lane 3). Although possible the <br /> region KAP123 homology brr6-1 mutation                      trypsin protection results protein conformation rather <br /> accessible transport machinery, Brr6p                   membrane association, detergent sensitivity <br /> be anchored membrane C-terminal predicted                      (lane 5) argues favor protection bilayer. The <br /> transmembrane domain, C-terminus sequestered                        size protected Brr6-GFP fragment correlates <br /> in NE lumen.                                                             precisely length expected Brr6p anchored <br />    To investigate membrane topology Brr6p,                         membrane C-terminal-most predicted <br /> used trypsin digestion determine N- and/or                    transmembrane domain, C-terminus extending <br /> C-termini protein protected proteolysis                  lumen NE. <br /> the presence absence Triton X-100. We carried                        In extracts containing N-terminally tagged polyoma- <br /> trypsin digests crude lysates containing Brr6p bearing                    Brr6p protected fragments (Figure 5B). A <br /> either C-terminal GFP tag N-terminal polyoma tag.                    band ~26 kDa corresponding full-length product <br /> The results typical experiment shown Figure 5B.                  seen absence trypsin protease <br /> In absence trypsin, protein ~49 kDa corres-                      inhibitor added (lanes 1, 3 4). However, bands <br /> ponding full-length Brr6-GFP present presence                   observed trypsin included, the <br /> or absence detergent (lanes 1 4). Trypsin digestion                   absence presence detergent (lanes 2 5). These <br /> in absence detergent results protected fragment                  results consistent topology Brr6 <br /> of ~29 kDa apparent mol. wt (lane 2). The fragment                        N-terminus, including site brr6-1 mutation and <br /> speci&#194;&#174;c trypsin digestion addition                         region homology Kap123p, located outside <br /> soybean trypsin inhibitor results primarily full-                     lumen (Figure 5C). Such orientation make <br />  <br />                                                                                                                                              4187 <br />  A.de Bruyn Kops C.Guthrie <br />  <br />  <br /> interaction regions transport machinery <br /> feasible. <br />  <br /> Depletion Brr6p alters localization two <br /> nucleoporins <br /> Taken together, BRR6 genetic interactions the <br /> Brr6-GFP localization extraction data argue Brr6p <br /> is NE membrane protein located near, interacting <br /> with, NPC. To examine relationship <br /> between Brr6p NPC, tested the <br /> localization nucleoporins, Nup188p Nsp1p, is <br /> altered conditions BRR6 expression is <br /> inhibited. The localization GFP-tagged Nup188 <br /> protein examined living cells sole <br /> copy Brr6p expressed promoter <br /> (BRR6-Myc) repressible GAL1-GAL10 promoter <br /> (galBRR6-Myc) (see Materials methods). When these <br /> strains assayed growth YEP plates containing <br /> galactose glucose, BRR6-Myc grew carbon <br /> sources galBRR6-Myc grew galactose failed <br /> to grow glucose. When cells cultured YEP media <br /> containing galactose switched YEP glucose, a <br /> growth defect observed galBRR6-Myc in <br /> the BRR6-Myc strain 6 h. When Nup188-GFP local- <br /> ization examined living cells, GFP signal            Fig. 6. Depletion Brr6p causes redistribution nucleoporins. <br /> became consolidated single spot           (A) Nup188-GFP Nsp1p localizations conditions where <br /> galBRR6-Myc cells 5 h following switch glucose-         Brr6p expressed GAL1-GAL10 promoter (galBRR6-Myc) was <br /> containing medium (Figure 6A). Co-staining DAPI            depleted glucose inhibition. Nup188-GFP assayed living <br /> con&#194;&#174;rmed spot located nuclear rim          cells Nsp1p detected indirect immuno&#194;&#175;uorescence &#194;&#174;xed <br />                                                                 cells. Growth glucose medium 5 h resulted consolidation of <br /> (data shown). No change Nup188-GFP localization          Nup188-GFP Nsp1p signals (examples indicated arrows) <br /> was observed BRR6-Myc strain. Similar results       galBRR6-Myc strain BRR6-Myc BRR6 is <br /> obtained Nsp1p detected immuno&#194;&#175;uores-               control promoter. (B) Nup188-GFP Nsp1p <br /> cence microscopy &#194;&#174;xed galBRR6-Myc BRR6-Myc               distributions BRR6 brr6-1 cells. Bar, 10 mm. <br /> cells grown galactose switched glucose medium <br /> (Figure 6A). When cells maintained galactose, no <br /> change Nup188-GFP Nsp1p localization was <br /> observed BRR6-Myc majority galBRR6-Myc             Brr6p depletion, carried electron microscopy on <br /> cells (data shown). A small portion (&lt;10%)               galBRR6-Myc BRR6-Myc strains grown galactose <br /> galBRR6-Myc cells showed concentrations signal          switched glucose 5 h. <br /> few spots nuclear rim (data shown), probably         Aberrant pore envelope morphology consisting a <br /> re&#194;&#175;ecting suboptimal expression Brr6p               bulge outer NE overlying large electron-dense <br /> galactose promoter cells. In brr6-1 mutant,         mass observed &gt;50% galBRR6-Myc nuclei <br /> Nup188-GFP Nsp1p appeared decreased                following shift glucose-containing medium <br /> nuclear rim signal redistributed single spot   (Figure 7A). The abnormal structures appeared both <br /> (Figure 6B).                                                    individually clusters strongly resembled the <br />                                                                 NE `herniations&apos; reported previously gle2-1 and <br /> Brr6p depletion causes changes NPC                           Dnup116 mutants (Wente Blobel, 1993; Murphy et al., <br /> distribution envelope morphology                            1996; Bailer et al., 1998). As Nup188-GFP and <br /> The redistribution Nup188-GFP Nsp1p               Nsp1p localization phenotypes, abnormal membrane <br /> punctate rim pattern single spot observed Brr6p       morphology seen small portion (&lt;5%) the <br /> depletion reminiscent pore-clustering phenotype       galBRR6-Myc cells maintained galactose-containing <br /> exhibited Dnup133 (Doye et al., 1994; Pemberton et al.,      medium. In contrast, pore envelope morphologies <br /> 1995). Clustering seen mutant GLE2,          BRR6-Myc strain medium indistin- <br /> NPC-associated transport factor, certain          guishable seen untagged wild-type strain, <br /> nucleoporin mutants (reviewed Wente, 2000). In           BRR6, grown glucose medium (Figure 7B). It seems <br /> gle2-1 mutant, phenotype accompanied major            probable clustered herniations observed following <br /> changes NE morphology (reviewed Fabre Hurt,           Brr6p depletion ultrastructural correlate the <br /> 1997; Wente, 2000). In mutants, including Dnup133,        nucleoporin redistribution seen light microscopic <br /> pores aggregate little alteration envelope        level. Interestingly, small, isolated herniations also <br /> observed (Doye et al., 1994; Aitchison et al., 1995; Heath      evident brr6-1 cells grown glucose 30&#194;&#176;C <br /> et al., 1995; Li et al., 1995). To determine            (Figure 7B); however, clusters detected the <br /> changes pore membrane ultrastructure underlie         mutant. These results indicate Brr6p required for <br /> Nup188-GFP Nsp1p redistribution observed following          normal pore distribution NE morphology. <br />  <br /> 4188 <br />                                                                                                 Brr6p nuclear envelope protein transport <br />  <br />  <br />  <br />  <br /> Fig. 7. Brr6p depletion brr6-1 mutation cause changes NE morphology. Shown transmission electron microscope images thin <br /> sections cells prepared high-pressure freezing. (A) A single example BRR6-Myc examples galBRR6-Myc cells grown in <br /> galactose medium switched glucose medium 5 h. The upper panels examples nuclei (NU) (bar, 500 nm). Examples of <br /> typical pores marked asterisk (*) shown higher magni&#194;&#174;cation lower panels (bar, 100 nm). (B) Images BRR6 brr6-1 <br /> cells grown glucose 30&#194;&#176;C. Upper panels nuclei (bar, 500 nm), lower panels individual pores (*) higher magni&#194;&#174;cation <br /> (bar, 100 nm). <br />  <br />  <br />  <br />                                                                              nucleoporins, Nup188-GFP Nsp1p, a <br /> Discussion <br />                                                                              punctate rim pattern single spot. <br /> To date handful yeast NE integral membrane                            The redistribution nucleoporins observed Brr6p <br /> proteins identi&#194;&#174;ed,                         depletion reminiscent pore-clustering phenotype <br /> shown participate transport processes. We                         exhibited number nucleoporin mutants, including <br /> identi&#194;&#174;ed essential yeast NE component, BRR6,                       Dnup133, mutant GLE2, NPC-associated <br /> function affects mRNA NES protein transport                      factor believed function mRNA export association <br /> well spatial organization nuclear pores.                           nucleoporin NUP116 (reviewed Wente, 2000). <br />                                                                              In Dnup133 gle2-1, phenomenon been <br />                                                                              shown result lateral movement NPCs the <br /> Brr6p integral membrane protein NE                              envelope (Belgareh Doye, 1997; Bucci Wente, <br /> that interacts functionally NPC                                     1997). Interestingly, ultrastructural morphology the <br /> Several observations indicate Brr6p likely                   envelope NPC following depletion Brr6p resembles <br /> integral membrane protein nuclear envelope:                           distinctive herniations documented previously both <br /> (i) Brr6p behaves biochemically like integral mem-                        gle2 Dnup116 mutants (Wente Blobel, 1993; <br /> brane protein, resistant high pH requiring                      Murphy et al., 1996; Bailer et al., 1998). As noted <br /> detergent ef&#194;&#174;cient extraction; (ii) C-terminal Brr6p                   previously, herniations differ `grape-like <br /> fragment protected trypsin digestion absence                  aggregates&apos; NPCs nucleoporin mutants <br /> but presence detergent, consistent                       clustering, including Dnup133, Dnup145 and <br /> protein anchored membrane predicted                       Dnup120 (reviewed Pemberton et al., 1995; Murphy <br /> transmembrane domain identi&#194;&#174;ed C-terminus                      et al., 1996). The smaller envelope herniations evident in <br /> BRR6 ORF; (iii) visualization GFP-tagged Brr6p                            brr6-1 mutant re&#194;&#175;ect earlier stage the <br /> expressed low copy living cells reveals predomin-                    herniation process. <br /> antly nuclear rim distribution. In addition, results <br /> suggest Brr6p interacts functionally NPC.                      BRR6 functions nuclear transport <br /> First, BRR6 shows speci&#194;&#174;c genetic interactions                      The brr6-1 mutant accumulates mRNA NLS/ <br /> con&#194;&#174;rmed nucleoporin genes, NUP1, NIC96 NUP188                           NES-GFP fusion protein nuclear rim, consistent with <br /> (Rout et al., 2000 references therein),                 role BRR6 transport events occurring near the <br /> possible nucleoporin, NUP2 (Loeb et al., 1993). Secondly,                    nuclear pore. Importantly, defects detected a <br /> depletion Brr6p results dramatic redistribution                 number protein transport reporters brr6-1, <br />  <br />                                                                                                                                             4189 <br />  A.de Bruyn Kops C.Guthrie <br />  <br />  <br /> arguing mutation affects subset transport          recent computer-aided reconstructions serial <br /> events. In particular, absence defects NLS-GFP       sectioned yeast (Winey et al., 1997). In studies, a <br /> constructs lacking NES implies speci&#194;&#174;c effect NES        minimum pore-free space (120 nm yeast) gener- <br /> protein transport. Future experiments required        ally observed pore NPCs did not <br /> determine aspect NES protein transport               abut. The separation pores, seemingly para- <br /> pathway affected brr6-1; principle, punctate        doxical given high lateral mobility yeast pores the <br /> rim pattern re&#194;&#175;ect accumulation docked NES           envelope (Belgareh Doye, 1997; Bucci Wente, <br /> proteins pore face import export.            1997), explained area occupied each <br /> However, genetic interaction BRR6                NPC effectively expanded associated membrane <br /> NUP1 points possibility brr6-1 affect            proteins adjacent pore. <br /> transport events occurring nuclear basket.                   Peri-pore membrane proteins serve anchor <br />    Interestingly, presence mRNA NES              pore-associated structures, nuclear basket, and <br /> protein transport defects brr6-1 suggests possible          provide docking sites transport substrates. It yet <br /> overlap pathways. Such overlap            known export carriers &#194;&#174;rst encounter cargoes <br /> proposed earlier based extremely rapid nuclear            nucleus. Although cargo&#194;&#177;carrier complexes are <br /> accumulation NES reporter mRNA             envisaged dock nuclear basket evidence <br /> xpo1-1 karyopherin mutant (Stade et al., 1997),       interactions karyopherins putative basket <br /> direct role Xpo1p (Crm1p) mRNA export remains             components increasing, transport substrates could <br /> controversial (Hodge et al., 1999; Neville Rosbash,          accumulate NE. In regard interesting <br /> 1999). Overlap mRNA NES export pathways              recent immuno-electron microscopic study found <br /> is suggested &#194;&#174;ndings vertebrate              mRNPs concentrated adjacent NPC (Iborra et al., <br /> nucleoporin, Nup98, participates Crm1p-mediated Rev           2000), central axis nuclear basket, <br /> protein export (Zolotukhin Felber, 1999)       raising possibility mRNPs accumulate <br /> mRNA export involving Gle2p homolog, Rae1p                   membrane entering pore. In a <br /> (Bailer et al., 1998; Pritchard et al., 1999). The accumu-       scenario, NE integral membrane protein Brr6p <br /> lation mRNA NES reporter nuclear          important concentrating mRNP cargoes near <br /> rim brr6-1 suggests exciting possibility Brr6p,      NPC, playing key role mRNA export the <br /> as integral membrane protein, function step     nucleus. <br /> transport mRNA NES protein export pathways <br /> converge NE. Alternatively, NES accumulation <br /> could secondary effect reporter trapped the <br /> envelope herniations present brr6-1. In case, NES <br />                                                                  Materials methods <br /> protein export continue remaining normal           Strains plasmids <br /> pores, explaining absence NES nucleoplasmic               The strains used work listed Table I. Cloning carried out <br /> signal mutant. However, explain        using published bacterial yeast vectors (Bluescript, YCp50, pSE358, <br /> mRNA export defect given cells dramatic           pRS305 pRS424; references Supplementary data, available at <br />                                                                  The EMBO Journal Online). pJL602 gift Joachim Li. Nup188- <br /> nucleoplasmic mRNA accumulation brr6-1.                       GFP (URA) Npl3-GFP (LEU) constructs generated Katrin <br />                                                                  Stade Chris Siebel (K.Stade, C.Siebel C.Guthrie, unpublished <br /> Brr6p located novel membrane                         results). In addition, following constructs obtained other <br /> domain adjacent NPC                                       laboratories: Kap123-GFP (Seedorf Silver, 1997); NLS-GFP (Shulga, <br /> The punctate character Brr6-GFP rim localization          1996); NLS-(GFP)3, L25-GFP Nab2-GFP (gifts Nicole Miller, <br />                                                                  Arash Komeili, Arie Kaffman Erin O&apos;Shea); NLS/NES-(GFP)2, <br /> pattern, similar nucleoporins, strongly suggests      NLS-(GFP)2 Xpo1-GFP (Stade et al., 1997). <br /> that Brr6p located near NPC. However, unlike <br /> Pom152p Pom34p, components pore membrane              Isolation brr6-1 Dbrr6::HIS3 alleles <br /> domain bridge inner outer NE NPC             The brr6-1 allele (bad response refrigeration) isolated twice (brr6- <br /> (Rout et al., 2000 references therein), Brr6-GFP did         275, brr6-412) oligo dT50 situ hybridization screen cs mRNA <br />                                                                  transport mutants S.cerevisiae (A.de Bruyn Kops C.Guthrie, <br /> not relocate Dnup133 clustering assay,            unpublished results) similar used identify ts transport mutants <br /> Brr6p identi&#194;&#174;ed recent nuclear pore preparation           (for references Supplementary data). The BRR6 gene (SGD accession <br /> reported Rout et al. (2000). Thus, based               No. YGL247w) cloned complementation brr6-412 cs <br /> available information, Brr6p behaves differently            growth defect using genomic DNA library (Rose et al., 1987). The <br /> proteins pore membrane domain. These apparent             BRR6 ORF disrupted (Dbrr6::HIS3) replacing sequence <br />                                                                  encoding aa 7&#194;&#177;196 HIS3 coding sequence diploid YPH399 and <br /> distinctions raise intriguing possibility Brr6p      W303 cells. Tetrad analysis Dbrr6::HIS3 diploids showed that <br /> located pore membrane domain but, rather,             BRR6 essential strain backgrounds. Linkage brr6- <br /> adjacent NPC, novel peri-pore NE             275 brr6-412 mutations BRR6 gene con&#194;&#174;rmed crosses <br /> domain. To date, attempts test prediction         haploid Dbrr6::HIS3 strains containing BRR6 URA3 plasmid <br /> immuno-electron microscopy using tagged forms            subsequent loss plasmid 5-FOA. Recovery brr6-275 and <br />                                                                  brr6-412 mutations plasmids followed DNA sequence analysis <br /> of Brr6p unsuccessful owing low signal. We          showed single change aa 110 (Arg Lys) isolates, we <br /> are generating antibodies Brr6p itself continued      designated brr6-1. The mutant plasmids conferred cs growth <br /> efforts localize protein ultra-structural level.   mRNA transport defects haploid brr6::HIS3 deletion strains. <br />    Interestingly, NPC-associated membrane domain              Isogenic strains targeted insertion mutant wild-type <br />                                                                  BRR6 BRR6 downstream sequence Dbrr6::HIS3 deletion <br /> around pore explain phenomenon                  strains. The resulting strains, BRR6 brr6-1, contain wild-type and <br /> minimum pore spacing documented early ultra-struc-            brr6-1 alleles LEU2 marker &#194;&#175;anked Dbrr6::HIS3 gene <br /> tural studies (reviewed Franke Scheer, 1974)          disruption. <br />  <br />  <br /> 4190 <br />                                                                                                Brr6p nuclear envelope protein transport <br />  <br />  <br /> Table I. Yeast strains <br /> Strain                         Genotype                                                                     Source (this study unless noted) <br />  <br /> brr6-275 (yDBK22)              MATa trp1-63 leu2-1 ura3-52 brr6-1cs                                     outcrossed strain derived from <br />                                                                                                             DBY4275 (D.Botstein) <br /> brr6-412 (yDBK58)              MATa brr6-1cs                                                            outcrossed strain derived from <br />                                                                                                             DBY4412 (D.Botstein) <br /> Dbrr6::HIS3 (yDBK123)          MATa/a trp1-63/trp1-63 leu2-1/leu2-1 ura3-52/ura3-52 his3-200/his3-200       disruption BRR6 YPH399 diploid <br />                                ade2-101/ade2-101 lys2-801/lys2-801 BRR6/brr6::HIS3 <br /> Dbrr6::HIS3 (yDBK124)          MATa/a ade2-1/ade2-1 ura3-1/ura3-1 his3-11,15/his3-11,15 trp1-1/trp1-1       disruption BRR6 W303 diploid <br />                                leu2-3,112/leu2-3,112 BRR6/brr6::HIS3 <br /> Dbrr6::HIS3 haploid shuf&#194;&#175;e     MATa trp1-63 leu2-1 ura3-52 his3-200 ade2-101 lys2-801 brr6::HIS3            segregant yDBK123 <br /> strain (yDBK125)               (YCp50-URA3-BRR6) <br /> Dbrr6::HIS3 haploid shuf&#194;&#175;e     MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   segregant yDBK124 <br /> strain (yDBK126)               (YCp50-URA3-BRR6) <br /> BRR6 (yDBK165)                 MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3/BRR6(LEU2)        BRR6 integrated Dbrr6::HIS3 <br />                                                                                                             (W303 background) <br /> brr6-1 (yDBK166)               MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3/brr6-1(LEU2)      brr6-1 integrated Dbrr6::HIS3 <br />                                                                                                             (W303 background) <br /> BRR6-untagged (yDBK155)        MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   pDBK55 shuf&#194;&#175;ed yDBK126 <br />                                (pSE358-TRP1-BRR6) <br /> BRR6-GFP (yDBK175)             MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   pDBK75 shuf&#194;&#175;ed yDBK126 <br />                                (pSE358-TRP1-BRR6-GFP) <br /> BRR6-Myc (yDBK176)             MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   pDBK76 shuf&#194;&#175;ed yDBK126 <br />                                (pSE358-TRP1-BRR6-myc epitope) <br /> polyoma-BRR6 (yDBK181)         MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   pDBK81 shuf&#194;&#175;ed yDBK126 <br />                                (pSE358-TRP1-polyoma epitope-BRR6) <br /> galBRR6-Myc (yDBK192)          MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 brr6::HIS3                   pDBK92 shuf&#194;&#175;ed yDBK126 <br />                                (pJL602-LEU2-BRR6-myc epitope) <br /> Dnup188 (NP188-2.4)            MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 nup188-1::HIS3               NUP188 disruption (R.Wozniak)a <br /> brr7-1 (yDBK36)                MATa ade2-101 lys2-801 leu2-1 ura3-52 nup188 cs                              isolate dT50 screen <br />                                                                                                             (A.de Bruyn Kops C.Guthrie, <br />                                                                                                             unpublished) <br /> Dnic96 (nic96D532&#194;&#177;839)         MATa ade2 trp1 leu2 ura3 nic96::HIS3 (pUN100-LEU2-nic96D532&#194;&#177;839)             NIC96 disruption (E.Hurt)a <br /> Dnup1 (LDY460)                 MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 1-100 nup1::LEU2         NUP1 disruption (L.Davis)a <br /> Dnup2 (JLY48)                  MATa ura3-52 leu2-3,112 trp1-289 nup2-4::URA3                                NUP2 disruption (G.Fink)a <br /> Dnup116 (SWY27)                MATa ade2-1 ura3-1 his3-11,15 trp1-1 leu2-3,112 nup116-5::HIS3               NUP116 disruption (S.Wente)a <br /> Dnup133 (nup133&#194;&#177;)              MATa ade2 trp1 leu2 ura3 nup133::HIS3                                        NUP133 disruption (E.Hurt)a <br /> aComplete   reference provided Supplementary data. <br />  <br />  <br /> Epitope tagging                                                             followed staining monoclonal anti-digoxygenin (1:100; <br /> Low copy constructs (CEN/ARS) BRR6 tagged            Boehringer Mannheim) FITC-conjugated goat anti-mouse (1:400; <br /> at C-terminus GFP coding sequence Myc epitope             Cappel/ICN) antibodies mounting glycerol/gelatin containing <br /> (EQKLISEEDL) N-terminus polyoma epitope                   DAPI (0.5 mg/ml). For localization GFP-tagged proteins, cells were <br /> (MEYMPME)2. Tagged untagged constructs shuf&#194;&#175;ed                harvested, resuspended media examined immediately by <br /> haploid Dbrr6::HIS3 (W303 strain background) strains carrying     &#194;&#175;uorescence microscopy. DAPI (0.5 mg/ml) added con&#194;&#174;rm the <br /> single copy BRR6. Comparison growth tagged untagged        location cell nucleus. For immuno&#194;&#175;uorescence, cells &#194;&#174;xed and <br /> strains YEPD media showed nearly wild-type growth tagged         spheroplasted prior staining. The following localization studies were <br /> strains 30&#194;&#176;C.                                                            carried out: (i) low high copy Brr6-GFP [in Dbrr6::HIS3 cells (W303 <br />                                                                             background) carrying CEN/ARS Brr6-GFP construct W303 cells <br /> Brr6p overexpression depletion                                          containing 2 mm Brr6-GFP plasmid 30&#194;&#176;C]; (ii) Nup188-GFP and <br /> DNA fragments containing GFP-tagged untagged BRR6                  Brr6-GFP [in Dnup133 wild-type cells carrying Nup188- <br /> subcloned 2 mm plasmids (TRP1) transformed W303               GFP(URA) high copy Brr6-GFP(TRP) plasmid 25&#194;&#176;C]; <br /> cells. The GFP-tagged construct gave highly elevated GFP &#194;&#175;uorescent         (iii) Nup188-GFP Nsp1p localization galBRR6-Myc BRR6- <br /> signal relative fusion CEN plasmid, indicating BRR6   Myc cells conditions Brr6p depletion. Nup188-GFP assayed <br /> was overexpressed. Wild-type cells transformed constructs         living cells. Nsp1 localized immuno&#194;&#175;uorescence using <br /> showed mild growth defects. To examine effect overexpression      monoclonal anti-Nsp1p (1:1000; 32D6 Aris laboratory) rhodamine- <br /> selected nucleoporin mutants, strains carrying untagged BRR6 2 mm       conjugated goat anti-mouse antibodies (1:1000); (iv) selected GFP- <br /> plasmid grown dropout media (-Trp) room temperature          tagged protein import reporters (in BRR6 brr6-1 cells grown in <br /> 3 days.                                                                     dropout media 30&#194;&#176;C). <br />    A GAL1-GAL10 promoter driven, BRR6-Myc construct in <br /> the pJL602 gal vector (LEU2). Strains sole copy BRR6 <br /> was control GAL1-GAL10 BRR6 promoter         Electron microscopy <br /> were generated shuf&#194;&#175;ing Dbrr6::HIS3 haploid deletion strain     Electron microscopy carried (i) BRR6 brr6-1 cells grown <br /> (W303 background) galBRR6-Myc BRR6-Myc strains,                 YEPD 30&#194;&#176;C (ii) BRR6-Myc galBRR6-Myc cells grown at <br /> respectively. Brr6p depletion experiments carried galBRR6-      30&#194;&#176;C galactose switched glucose 5 h. Cells harvested <br /> Myc BRR6-Myc cells grown 30&#194;&#176;C harvested 0 5 h             (OD600 = 0.15&#194;&#177;0.25) rapidly &#194;&#174;ltration subjected high-pressure <br /> following change galactose- glucose-containing media.             freezing freeze substitution embedding Epon-Araldite (for <br />                                                                             reference Supplementary data). Sample preparation carried out <br /> Localization mRNA proteins                                           Kent McDonald Berkeley Electron Microscopy Facility. <br /> Cells grown OD600 = 0.15&#194;&#177;0.25 YEPD appropriate dropout        Embedded cells sectioned thickness ~65 nm post-stained <br /> media. Bulk polyA(+) mRNA localized &#194;&#174;xed, spheroplasted cells        lead citrate (0.2%, 3 min) aqueous uranyl acetate (1%, <br /> by situ hybridization digoxygenin-tailed oligo dT50 probe         10&#194;&#177;30 min). <br />  <br />  <br />                                                                                                                                                4191 <br />  A.de Bruyn Kops C.Guthrie <br />  <br /> Biochemical extraction trypsin digestion Brr6p                        Fahrenkrog,B., Hurt,E.C., Aebi,U. Pante,N. (1998) Molecular <br /> Whole-cell lysate (700 ml) prepared 100 ml BRR6-GFP cells          architecture yeast nuclear pore complex: localization Nsp1 <br /> (OD600 = 1.0) bead-beating standard buffer (20 mM HEPES&#194;&#177;KOH            subcomplexes. J. Cell Biol., 143, 577&#194;&#177;588. <br /> pH 7.9, 5 mM MgCl2, 150 mM NaCl) presence protease                 Franke,W.W. Scheer,U. (1974) Structures functions the <br /> inhibitors, incubating 10 min 30&#194;&#176;C zymolyase (100T, 1 mg/        nuclear envelope. In Busch,H. (ed.), The Cell Nucleus. Academic <br /> ml) disrupt cell wall. Aliquots (100 ml) lysate frozen      Press, New York, NY, pp. 219&#194;&#177;347. <br /> then thawed. An equal volume 23 extraction buffers        Gant,T.M. Wilson,K.L. (1997) Nuclear assembly. Annu. Rev. Cell <br /> added following &#194;&#174;nal concentrations: (i) 150 mM NaCl; (ii) 1 M     Dev. Biol., 13, 669&#194;&#177;695. <br /> NaCl; (iii) 150 mM NaCl, 100 mM sodium carbonate pH 11;                  Gerace,L. Burke,B. (1988) Functional organization nuclear <br /> (iv) 150 mM NaCl, 4% Triton X-100 (v/v). Lysates incubated            envelope. Annu. Rev. Cell Dev. Biol., 4, 335&#194;&#177;374. <br /> 30 min ice, dounced 10 times pelleted 100 000 g (4&#194;&#176;C)          Go&#195;&#710;rlich,D. Kutay,U. (1999) Transport cell nucleus and <br /> 30 min Beckman TLA-120.1 rotor. Pellet supernatant samples            cytoplasm. Annu. Rev. Cell Dev. Biol., 15, 607&#194;&#177;660. <br /> were analyzed western blotting. Brr6-GFP, Nup188-HA Pom152            Greber,U.F. Gerace,L. (1992) Nuclear protein import inhibited by <br /> were detected using following antibodies: polyclonal anti-GFP (Silver      antibody lumenal epitope nuclear pore complex <br /> laboratory), monoclonal anti-HA epitope (12CA5; Lerner laboratory)         glycoprotein. J. Cell Biol., 116, 15&#194;&#177;30. <br /> monoclonal anti-Pom152 (MAB118C3, Rout laboratory).                          Heath,C.V., Copeland,C.S., Amberg,D.C., Priore,V.D., Snyder,M. and <br />    For trypsin digestion experiments, 400 ml whole-cell lysate prepared     Cole,C.N. (1995) Nuclear pore complex clustering nuclear <br /> from BRR6-GFP polyoma-BRR6 cells (see above) pelleted              accumulation poly(A)+ RNA associated mutation the <br /> resuspended equal volume BTM (10 mM Tris pH 6.5, 0.1 mM               Saccharomyces cerevisiae RAT2/NUP120 gene. J. Cell Biol., 131, <br /> MgCl2), dounced 10 times ice pelleted 100 000 g. Pellets        1677&#194;&#177;1697. <br /> resuspended 700 ml BTM. Aliquots (80 ml) incubated ice         Ho,A.K., Raczniak,G.A., Ives,E.B. Wente,S.R. (1998) The integral <br /> for 1 h presence absence trypsin (500 mg/ml), Triton X-100        membrane protein Snl1p genetically linked yeast nuclear pore <br /> (1% v/v) soybean trypsin inhibitor. Samples collected              complex function. Mol. Biol. Cell, 9, 355&#194;&#177;373. <br /> trichloroacetic acid precipitation analyzed SDS&#194;&#177;PAGE              Hodge,C.A., Colot,H.V., Stafford,P. Cole,C.N. (1999) Rat8p/Dbp5p <br /> western blotting using polyclonal antisera GFP (Silver laboratory)      shuttling transport factor interacts Rat7p/Nup159p and <br /> or polyoma epitope (O&apos;Shea laboratory).                                       Gle1p suppresses mRNA export defect xpo1-1 cells. EMBO <br />                                                                                 J., 18, 5778&#194;&#177;5788. <br /> Supplementary data                                                           Iborra,F.J., Jackson,D.A. Cook,P.R. (2000) The path RNA <br /> Because space limitations, brief descriptions methods given      nuclear pores: apparent entry sides into <br /> here. For greater Supplementary data, available The           specialized pores. J. Cell Sci., 113, 291&#194;&#177;302. <br /> EMBO Journal Online.                                                         Li,O., Heath,C.V., Amberg,D.C., Dockendorff,T.C., Copeland,C.S., <br />                                                                                 Snyder,M. Cole,C.N. (1995) Mutation deletion the <br />                                                                                 Saccharomyces cerevisiae RAT3/NUP133 gene causes temperature- <br /> Acknowledgements                                                                dependent accumulation poly(A)+ RNA constitutive clustering <br />                                                                                 nuclear pore complexes. Mol. Biol. Cell, 6, 401&#194;&#177;417. <br /> We thank Erin O&apos;Shea, Pam Silver, John Aris Mike Rout gifts      Loeb,J.D.J., Davis,L.I. Fink,G.R. (1993) NUP2, novel yeast <br /> of antibodies protein reporter constructs, Katrin Stade Chris       nucleoporin, functional overlap proteins nuclear <br /> Siebel Nup188-GFP Npl3-GFP plasmids. We grateful             pore complex. Mol. Biol. Cell, 4, 209&#194;&#177;222. <br /> David Botstein use cold-sensitive bank mutants          Murphy,R., Watkins,J.L. Wente,S.R. (1996) GLE2, Saccharo- <br /> Richard Wozniak, Ed Hurt, Gerald Fink, Valerie Doye, Laura Davis            myces cerevisiae homologue Schizosaccharomyces pombe <br /> Susan Wente nucleoporin mutants. We thank Kent McDonald             export factor RAE1, required nuclear pore complex structure and <br /> Berkeley EM facility help preparing cells electron microscopy. We       function. Mol. Biol. Cell, 7, 1921&#194;&#177;1937. <br /> also thank Michael Rout, Susan Wente Katherine Wilson helpful        Nakielny,S. Dreyfuss,G. (1999) Transport proteins RNAs in <br /> discussions. This project supported grant NIH                 nucleus. Cell, 99, 677&#194;&#177;690. <br /> (GM21119) C.G. In addition, parts work funded             Neville,M. Rosbash,M. (1999) The NES-crm1p export pathway is <br /> fellowships A.d.B.K. Jane Cof&#194;&#174;n Childs Fund Cancer              major mRNA export route Saccharomyces cerevisiae. <br /> Research American Cancer Society, California Division. C.G.          EMBO J., 18, 3746&#194;&#177;3756. <br /> an American Cancer Society Research Professor Molecular Genetics.         Pemberton,L.F., Rout,M.P. Blobel,G. (1995) Disruption the <br />                                                                                 nucleoporin gene NUP133. Proc. Natl Acad. Sci. USA, 92, <br />                                                                                 1187&#194;&#177;1191. <br /> References                                               <br /> </body></html>