<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>et al. <br /> Freimoser <br /> 2006 <br /> Volume 7, Issue 11, Article R109                                                                                             Open Access <br /> Research <br /> Systematic screening polyphosphate (poly P) levels yeast <br />  <br />  <br />  <br />  <br />                                                                                                                                                              <span id='am-5' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br /> mutant cells reveals strong interdependence primary <br /> metabolism <br /> Florian M Freimoser*, Hans Caspar H&#195;&#188;rlimann*, Claude A Jakob&#226;&#8364;&#160;, <br /> Thomas P Werner* Nikolaus Amrhein* <br /> Addresses: *Institute Plant Sciences, ETH Zurich, 8092 Zurich, Switzerland. &#226;&#8364;&#160;Institute Microbiology, ETH Zurich, Zurich, Switzerland. <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reviews <br /> Correspondence: Florian M Freimoser. Email: ffreimoser@ethz.ch <br />  <br />  <br /> Published: 15 November 2006                                                   Received: 2 August 2006 <br />                                                                               Revised: 4 October 2006 <br /> Genome Biology 2006, 7:R109 (doi:10.1186/gb-2006-7-11-r109) <br />                                                                               Accepted: 15 November 2006 <br /> The electronic version article complete be <br /> found online http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reports <br /> &#194;&#169; 2006 Freimoser et al.; licensee BioMed Central Ltd. <br /> This open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which <br /> permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br /> involved <br /> &lt;p&gt;A <br /> Polyphosphate <br />         systematicmaintenance <br />            metabolism <br />                     analysis ofin  polyphosphate <br />                                    ofyeast <br />                                       normal polyphosphate      content <br />                                                     levels yeast <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>knockout</span>  strains for <br />                                                                          provides  insights   every <br />                                                                                           almostinto phosphate <br />                                                                                                         non-essential <br />                                                                                                                   homeostasis.&lt;/p&gt; <br />                                                                                                                          gene identified 255 genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  <br />                  Background: Inorganic polyphosphate (poly P) occurs universally organisms bacteria <br />                  man. It functions, example, phosphate energy store, involved activation <br />                  regulation proteins. Despite ubiquitous occurrence important functions, unclear <br />                  poly P synthesized poly P metabolism regulated higher eukaryotes. This work <br />                  describes systematic analysis poly P levels yeast knockout strains mutated every <br />                  non-essential gene. <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />                  Results: After consecutive screens, 255 genes (almost 4% yeast genome) found <br />                  involved maintenance normal poly P content. Many genes encoded proteins <br />                  functioning cytoplasm, vacuole transport transcription. Besides reduced poly P <br />                  content, strains exhibited reduced total phosphate content, showed altered ATP and <br />                  glycogen levels disturbed secretion acid phosphatase. <br />  <br />                  Conclusion: Cellular energy phosphate homeostasis suggested result the <br />                  equilibrium poly P, ATP free phosphate cell. Poly P serves buffer for <br />                  ATP free phosphate levels is, therefore, essential consequently                                             interactions <br />                  variable component network. However, strains reduced poly P levels only <br />                  affected ATP phosphate content, components depend ATP <br />                  free phosphate content, glycogen secreted phosphatase activity. <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br /> Background                                                                     buffer [2-4] activation enzymes [5,6] regu- <br /> Inorganic polyphosphate (poly P) linear polymer                      lation chromatin condensation, gene expression trans- <br /> consists phosphoanhydride linked phosphate residues                     lation [1,7,8]. Poly P involved bacterial pathogenicity <br /> occurs ubiquitously organisms living cells [1]. The                 [9,10], <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span> stationary phase bacteria yeast <br /> functions poly P range role phosphate store                   [9,11,12], adaptation alkaline osmotic stress <br />  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.2 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                             http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         [13-15]. In slime mold Dictyostelium discoideum, poly P                  proteins) cell homeostasis (2 proteins) biological <br />         regulates development predation behavior [16],                    function terms; transporter activity (30 proteins) the <br />         humans blood coagulation accelerated fibrinolysis                  significantly overrepresented molecular function term <br />         delayed poly P [17]. At cellular level, poly P               (Figure 1b,c). Other proteins encoded identified genes <br />         detected cytosol, nucleus, mitochondria,               participate biological processes organelle cell <br />         endoplasmic reticulum [7,18,19]. Poly P particularly                  wall organization biogenesis (24 10 proteins, respec- <br />         prominent acidocalcisomes trypanosomatids [20],                    tively), protein biosynthesis modification (17 12 pro- <br />         Saccharomyces cerevisiae entire poly P                     teins, respectively), DNA RNA metabolism (10 7 <br />         pool resides vacuole [21,22].                                         proteins, respectively) involved response <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                      stress (11 proteins). Additional molecular functions were <br />         Despite universal occurrence broad functions,               prevalent discovered proteins included activities <br />         little known poly P metabolism regulation. In               hydrolases, transferases, transcription regulators and <br />         S. cerevisiae (and higher eukaryotes) is, example,          structural molecules (18, 17, 16 14 proteins, respectively) <br />         known poly P synthesized, despite fact                   protein DNA binding proteins (15 8 proteins, <br />         20% organism&apos;s dry weight com-                      respectively) (Figure 1c). <br />         posed poly P [1]. To gain broad view poly P metabolism <br />         yeast identify pathways involved regulation              Vacuolar proteins important determining <br />         poly P levels, extracted quantified poly P knock-              poly P levels <br />         strains non-essential yeast genes.                         All 255 genes ranked according impact poly P <br />                                                                                      content. The mutant cells strongly affected poly <br />                                                                                      P content highly enriched knockouts genes that <br />         Results                                                                      encode vacuolar proteins (6 20 vacuolar proteins <br />         Altogether, 4,765 strains YKO collection [23], con-                 10 strongly affected mutants). Vacuolar func- <br />         sisting knockouts non-essential yeast genes, ini-                 tion depends acidification vacuolar lumen, <br />         tially screened. Strains hypo- hyper-                         mediated vacuolar H+-ATPase (V-ATPase) <br />         accumulated poly P subjected consecutive                       complex. In poly P screen, 9 14 V-ATPase sub- <br />         rounds screening. After round, 255 strains                 units required normal poly P content (Vma5 (14), <br />         YKO collection altered poly P levels exper-             Vma8 (10), Vma10 (13), Vma13 (34), Vph1 (1), Cup5 (5), Tfp3 <br />         iments (a complete list data available online Addi-             (15), Ppa1 (20), Vma6 (27); ranks given parentheses). In <br />         tional data file 1). Almost 255 strains reduced             addition, regulators (Rav1 (43), Vps34 (30), Fab1 (150) and <br />         poly P levels occasionally, specific time-points                 Vac14 (213)) assembly factors (Vma22 (16), Vph2 (23)) of <br />         growth, poly P hyper-accumulation relative                     V-ATPase discovered important for <br />         wild-type strain observed.                                               normal poly P levels. <br />  <br />         All cellular compartments involved maintenance                        Next VPH1 knockout, VTC4 VTC1 deletion <br />         poly P levels                                                             strains second affected mutants, and <br />         All 255 discovered genes categorized Gene                      knockout VTC2 hypo-accumulated poly P (rank <br />         Ontology (GO) Slim terminology significantly overrepre-                  219). Together Vtc3, subunits the <br />         sented terms (relative occurrence yeast                vacuolar transporter chaperon (Vtc) complex, pre- <br />         genome) determined (Figure 1a-c). The cellular com-                     viously shown required maintenance poly P levels <br />         partments highest number proteins important                  [3]. Other proteins involved hypo-accumulation poly <br />         poly P content cytoplasm (59 proteins), nucleus                 P involved membrane docking fusion Golgi- <br />         (56 proteins), mitochondrion mitochondrial                       to-endosome endosome-to-vacuole steps (Vps33, <br />         envelope (33 9 proteins, respectively), vacuole (20                  ninth affected mutant), represent alternative path- <br />         proteins) endoplasmic reticulum (13 proteins) (Fig-                  way Golgi vacuole (effected AP-3 com- <br />         ure 1a). Other genes important poly P content                plex consists Apm3, Apl5, Apl6 Aps3; ranks 21, 59, <br />         encoded membrane ribosomal proteins (12 11 pro-                      57 55, respectively). Vam3, vacuolar t-SNARE protein <br />         teins, respectively) proteins functioning Golgi appa-              traffics vacuole AP-3 complex [24], also <br />         ratus (5 proteins), peroxisome cell wall (4 proteins              important maintenance poly P levels (rank 8). <br />         each) (Figure 1a). But terms &apos;vacuole&apos; &apos;mem- <br />         brane&apos; significantly overrepresented 255 genes                   In strongly affected strains, genes encoding the <br />         important maintenance poly                  phosphofructokinase subunits (PFK1 PFK2, ranks 28 <br />         P levels. The representation cellular compartments                    17) pyruvate kinase PYK2 (rank 7) knocked <br />         reflected highly overrepresented biological                  out. In additional strains, components glycoly- <br />         molecular function terms: transport vesicle mediated                     sis identified (GCR2, GPM3, YOR283W; 63, 201 and <br />         transport (36 22 proteins, respectively), transcription (16              181, respectively). <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                     Genome Biology 2006,     Volume 7, Issue 11, Article R109      Freimoser et al. R109.3 <br />  <br />  <br />  <br />  <br />                (a) Cellular component                             (b) Biological process                         (c) Molecular function <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      comment <br />                   Cytoplasm <br />  <br />                      Nucleus <br />                                                                    *Transport <br />  <br />                                                                     Unknow n <br />                                                                                                                     Unknow n <br />  <br />                                                                                                              *Transporter act. <br />                                                                                                                                               // <br />               Mitochondrion                         Organelle org. &amp; biogen.                                   Hydrolase act. <br />                    *Vacuole                    *Vesic le --mediated transport                                Transferase act. <br />                           ER                            Protein biosynthesis                              Transcript. reg. act <br />                  *Membrane                                     *Transcription <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      reviews <br />                                                                                                                Protein binding <br />                    Unknown                              Protein modification                               Structural mol. act. <br />  <br />                     Ribsome                              Response stress                                      DNA binding <br />  <br />                          PM                          Cell wall org. &amp; biogen..                                     Ligase act. <br />  <br />       Mitochond. envelope                                  DNA metabolism                                Oxidoreductase act. <br />  <br />        Endomembrane sys.                                             Meiosis                               Protein kinase act. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      reports <br />             Golgi apparatus                       Membrane org. &amp; biogen..                               Translation reg . act. <br />  <br />                     Cell w                             RNA metabolism                                   Enzyme reg . act. <br />  <br />                  Nucleolous                                      Sporulation                                   Peptidase act. <br />  <br />                 Peroxis ome                                 Lipid metabolis m                           Signal transducer act. <br />  <br />                         Other                                          Other                                            Other <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      deposited research <br />                                                                                                                                                // <br />                                  0       30      60                              0       20         40                           0    20        60 <br />                                      Number genes                                 Number genes                              Number genes <br />                                        category                                    category                                 category <br />  <br /> All <br /> Figure <br />     cellular <br />          1 compartments involved maintenance poly P levels <br /> All cellular compartments involved maintenance poly P levels. Categorization 255 genes important poly P levels using GO <br /> Slim terminology to: (a) cellular component, (b) biological process (c) molecular function. Categories significantly overrepresented (P &#226;&#8240;&#164; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      refereed research <br /> 0.05; red bars) marked asterisk. Only 15 abundant categories shown (all genes summarized &apos;other&apos;). <br />  <br />  <br />  <br /> Poly P profiles identified mutant strains                                 higher 2 h 24 h (Figure 2b). Cluster 2 significantly <br /> For 255 discovered strains poly P data different                    enriched (compared complete genome) cytoplasmic <br /> experiments/time-points available: second                     mitochondrial proteins proteins functioning in <br /> screen (performed 4 h time-point) samples                    organelle organization biogenesis (Figure 2a). The mito- <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> screen (2 h, 4 h, 8 h 24 h samples). Hier-                    chondrial proteins cluster 2 comprised, example, a <br /> archical clustering experiments log2-trans-                  mitochondrial phosphate transporter (Mir1) mito- <br /> formed poly P data revealed 4 h samples                    chondrial ribosomal proteins (Mrpl33, Mrp51 Mrpl27).                            interactions <br /> experiments grouped (Figure 2d). The 2 h time-                          Cluster 4 contained genes caused dramatic <br /> point similar 4 h time-point 8 h                     effect poly P levels deleted: 22 genes cluster <br /> and 24 h samples (Figure 2). Clustering genes resolved                    (out total 39) 30 important genes <br /> distinct groups genes similar poly P patterns. One                       maintenance poly P levels. This group genes also <br /> cluster genes (Figure 2, cluster 1) comprised seven genes                     included V-ATPase subunits (Vma8, Ppa1, <br /> that caused elevated poly P levels 2 h 4 h time-                      Cup5, Vma10, Vph1, Vma5) discovered many <br /> points deleted. Three seven genes cluster                    additional components vacuole. Correspondingly, this <br />                                                                                                                                                                      information <br />  <br />  <br />  <br />  <br /> (KRE1, ECM33 RIM21) encode proteins function                         cluster significantly enriched vacuolar, membrane and <br /> cell wall biosynthesis organization, GO Slim cate-                     cytoplasmic proteins involved transport, cell <br /> gory significantly enriched cluster (Figure 2a). In                  homeostasis vesicle mediated transport exhibit trans- <br /> two clusters (Figure 2, clusters 2 4) relative poly                porter hydrolase activity (Figure 2a). In distinct cluster <br /> P content was, average, minimal 4 h time-point                     3, membrane proteins function transport also <br />  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.4 Genome Biology 2006,      Volume 7, Issue 11, Article R109                         Freimoser et al.                                                            http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />                                                        2nd screen (4h) <br />                                      1st screen (4h) <br />             (a) <br />                                                                                                                                            Poly P ratio (log2) <br />  <br />  <br />  <br />                                                                               24h <br />                                                                          4h <br />  <br />  <br />                                                                                     8h <br />                                 2h <br />  <br />  <br />  <br />  <br />                                                                                                                                     -3.0           0             3.0 <br />  <br />                      1 <br />  <br />                                                                                              Cell. component: cytoplasm, mitochondrium <br />                       2                                                                      Biol . process : organelle org. &amp; biogen. <br />  <br />  <br />  <br />  <br />                                                                                              Cell. component: membrane <br />                       3                                                                      Biol . process : transport <br />  <br />  <br />  <br />                                                                                             Cell. component : vacuole, membrane, cytoplasm <br />  <br />                       4                                                                      Biol . process : transport, cell homeostasis, vesicle <br />                                                                                              mediated transport <br />                                                                                              Mol. function : transporter act., hydrolase act. <br />  <br />  <br />                                                                                            (b)                                        1 <br />                                                                                                   Relative poly P content (log 2) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        Cluster 1 <br />  <br />                                                                                                                                                                                        Cluster 2 <br />                                                                                                                                       0 <br />                                                                                                                                                                                        Cluster 3 <br />                                                                                                                                      -1                                                Cluster 4 <br />  <br />                                                                                                                                      -2 <br />  <br />                                                                                                                                      -3 <br />  <br />                                                                                                                                      -4 <br />                                                                                                                                               2h       4h        8h        24h <br />  <br />  <br />         Figure 2 255 genes poly P screen resolved distinct groups similar poly P profiles <br />         Clustering <br />         Clustering 255 genes poly P screen resolved distinct groups similar poly P profiles. (a) Hierarchical clustering (uncentered Pearson <br />         correlation, complete linkage) log2-transformed data (relative wild-type). Clusters significantly enriched (P &#226;&#8240;&#164; 0.05) GO Slim category <br />         marked colored bars branches. (b) Average poly P values (log2-transformed, relative wild type) genes distinct clusters <br />         (a) time-points. <br />  <br />  <br />  <br />  <br />                                                                                          Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                              Genome Biology 2006,    Volume 7, Issue 11, Article R109   Freimoser et al. R109.5 <br />  <br />  <br />  <br />  <br /> significantly overrepresented, but, average, relative                               3). Poly P content total phosphate levels strongly <br /> poly P content varied slightly increased                               reduced 30 non-essential mutants knock- <br /> growth corresponding deletion strains (Figure 2b).                                  TFP3 (only slightly reduced poly P total phosphate <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           comment <br />                                                                                            levels) &#206;&#8221;ecm14, &#206;&#8221;mrp51,&#206;&#8221;apm3 &#206;&#8221;sir3 strains had <br /> Many strains altered poly P levels affected                                  normal content total phosphate (Figure 3). The ATP <br /> in total phosphate content, ATP levels, acid                                         levels exhibited highest variability 0 h time-point, <br /> phosphatase activity glycogen accumulation                                             cluster knockouts having increased content <br /> Comparison 255 non-essential ORFs                                       (knockouts ERG6, VTC4, VTC1, PYK2, VPH1, CUP5 and <br /> important poly P content results published                               ECM14). Strongly reduced ATP levels measured in <br /> large scale analyses revealed 56 strains affected                           knockouts dubious open reading frame (ORF) <br /> in glycogen accumulation [25], 4 strains                                YPR099c uncharacterized ORF YOL019w (Figure 3). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           reviews <br /> reduced poly P levels previously screen                                Almost 30 knockouts hyper-accumulated glycogen, <br /> for altered PHO5 (acid phosphatase (rAPase)) regulation,                                   2 h 0 h 24 h time-points. An exception <br /> which serves indicator activation state PHO                            <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> dubious ORF YPR099c, which <br /> pathway [26,27]. To verify data screen,                                undetectable glycogen levels 0 h 2 h time- <br /> test poly P <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> accompanied complex                                 points (which explained effect MRPL51, <br /> phenotypes, measured cell density, poly P levels, total                                 encoded opposite strand). Most 30 <br /> phosphate content, ATP concentrations, glycogen levels                                 knockout mutants exhibited strongly increased rAPase <br /> rAPase activity 30 affected knockout strains.                                  activity 0 h 24 h time-points. Exceptions only <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           reports <br />                                                                                            knockouts ECM14, VTC4, VTC1 VAM3 (Figure 3). <br /> Overall, cell density ATP content affected <br /> characteristics, poly P total phosphate content gen- <br /> erally reduced, glycogen levels rAPase activity                               Discussion <br /> increased (Figure 3).                                                                      Previous screen described here, yeast <br />                                                                                            mutant cells analyzed respect poly P <br /> Cell density slightly reduced (to roughly half OD600)                              content and, knowledge, time a <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           deposited research <br /> in strains, 2 h 5 h time-points (Figure                           specific metabolite, is, poly P, isolated quantified <br />  <br />  <br />                                    OD600          Poly P                 Pi tot            ATP         Glycogen       rAPase <br />  <br />                                 Time [h]              Time [h]          Time [h]           Time [h]     Time [h]      Time [h] <br />                                0   2   5 24       0    2   5 24         0 2   5 24     0     2 5 24    0 2   5 24    0   2 5 24 <br />                                                                                                                                   ypr099c <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           refereed research <br />                                                                                                                                   reg1 <br />                                                                                                                                   vma22 <br />                                                                                                                                   vps33 <br />                                                                                                                                   ykl118w <br />                                                                                                                                   vma8 <br />                                                                                                                                   vma10 <br />                                                                                                                                   vma5 <br />                                                                                                                                   yol019w <br />                                                                                                                                   tfp3 <br />                                                                                                                                   pfk2 <br />                                                                                                                                   pfk1 <br />                                                                                                                                   ecm14 <br />                                                                                                                                   mrp51 <br />                                                                                                                                   apm3 <br />                                                                                                                                   sir3 <br />                                                                                                                                   ppa1 <br />                                                                                                                                   pep5 <br />                                                                                                                                   vph2 <br />                                                                                                                                   vma6 <br />                                                                                                                                   tat2 <br />                                                                                                                                   erg6 <br />  <br />  <br />  <br />                                                                                                                                                                           interactions <br />                                                                                                                                   vtc4 <br />                                                                                                                                   vtc1 <br />                                                                                                                                   pyk2 <br />                                                                                                                                   vph1 <br />                                                                                                                                   cup5 <br />                                                                                                                                   vam3 <br />                                                                                                                                   ypt7 <br />                                                                                                                                   vps34 <br />  <br />  <br />  <br />  <br />                                        Ratio (log2) <br />  <br />                             -2.0              0                   2.0 <br />                                                                                                                                                                           information <br />  <br />  <br />  <br />  <br /> Figure <br /> Many strains <br />         3     altered poly P levels affected total phosphate content, ATP levels, rAPase activity glycogen accumulation <br /> Many strains altered poly P levels affected total phosphate content, ATP levels, rAPase activity glycogen accumulation. All <br /> measurements (OD600, total phosphate content, ATP levels, rAPase activity glycogen content) 30 strains affected poly P levels are <br /> given relative wild type (log2-transformed). The data hierarchically clustered (uncentered Pearson correlation, average linkage). <br />  <br />  <br />  <br />                                                                     Genome Biology 2006, 7:R109 <br />  R109.6 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                             http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         complete mutant cell collection. After                    data impossible conclude V-ATPase activity <br />         consecutive screens, 255 knockout mutant strains                        regulates poly P levels directly indirectly. As poly P levels <br />         altered poly P levels identified. The genes affected                 VMA4 knockout strain slightly restored by <br />         mutant cells encode proteins intracellular com-               growing cells acid buffered growth medium, Ogawa et <br />         partments components functioning processes                    al. [3] concluded V-ATPase itself essential for <br />         primary metabolism. This broad analysis defined novel                   poly P metabolism. Instead, authors suggested the <br />         biological functions 250 yeast genes allowed,                  Vtc complex (consisting Vtc1, Vtc2, Vtc3 Vtc4), which <br />         time, global view pathways processes              regulates vacuolar membrane fusion, morphology func- <br />         affecting poly P metabolism yeast. But                 tion [37,38], directly involved synthesis poly P in <br />         genes required maintenance normal poly P con-                   yeast [3]. The fact mutants affected various stages of <br />         tent, strains hypo- hyper-accumulated poly P                     retrograde forward vesicle trafficking, in <br />         reduced total phosphate content, altered ATP gly-                autophagy, indicate intact secretory pathway is <br />         cogen levels up-regulated rAPase secretion. From                 required normal poly P content supports hypoth- <br />         analysis conclude poly P content extremely sen-                esis Vtc complex influences poly P levels indirectly <br />         sitive parameter highly intertwined primary                     [38]. This secondary effect poly P metabolism be <br />         metabolism.                                                                  caused by: mislocalization misregulation vacuolar <br />                                                                                      proteins important poly P synthesis storage; <br />         However, number knockout mutant cells reduced                    impairment poly P synthesis transport the <br />         poly P levels probably underestimated: Several                   secretory pathway. Thus, pathways indirectly <br />         mutants previously identified hypo-accumulate                     affect poly P content impact vacuole secre- <br />         poly P screen. For example, knock-                tory pathway and, thereby, link seemingly unrelated path- <br />         outs VMA4, VTC3, ARG82 KCS1 discovered,                    ways poly P metabolism. The reduced poly P content in <br />         knockouts members PHO pathway (PHO3,                   mutants subunits AP-3 complex, <br />         PHO4 or, example, PHO84), shown                       mediates alternative pathway Golgi the <br />         convey hypo-accumulation poly P [3,26,28]. This                     vacuole [39] medically relevant type of <br />         indication stringency screening conditions                 Hermansky-Pudlak syndrome [40], case of <br />         selection criteria suggests true number non-                 indirect effects; mutations proteins targeted the <br />         essential genes involved poly P metabolism higher                 vacuole AP-3 complex contain putative di- <br />         255 genes reported here. The reliability high-               leucine signal AP-3 targeting hypo-accumulated poly P <br />         throughput screening procedure confirmed                     (data shown). <br />         fact 30 strains YKO collection indi- <br />         vidually tested showed strongly reduced poly P levels.                Although deletion genes encode vacuolar proteins <br />                                                                                      caused severe reduction poly P levels, frac- <br />         Although poly P observed functions differ-                tion discovered genes encoded proteins function in <br />         ent organelles [7,18,19,29], 90% 99% poly P local-              vacuole. The remaining 235 genes encode proteins func- <br />         ized vacuole [1,21,22]. Consequently, measure                      tioning cellular compartments. Assessing poly P metab- <br />         exclusively vacuolar poly P proteins                     olism larger picture phosphate energy <br />         organelles screen affect                  homeostasis help explain involvement multi- <br />         vacuolar poly P levels. Poly P storage central function            tude pathways. PolyP, constitute 50% of <br />         yeast vacuole and, thus, mutants affected vacuolar                 total phosphate content yeast (data shown), thus <br />         functions morphology likely impaired                   seen form cellular phosphate free phos- <br />         capability store polyP. Important vacuolar functions                      phate, DNA, RNA, nucleotides phospholipids (Figure 4). <br />         include maturation activation different proteins                  Considering example poly P metabolism Escherichia <br />         physiological functions storage metabolites                coli evidence ATP-dependent poly P kinase <br />         cell homeostasis [30-32] depend V-ATPase.                 activity yeast [7], assumed majority yeast <br />         This large multimeric, partly membrane-embedded complex                      poly P synthesized ATP (Figure 4). The poly P pool is <br />         conserved yeast man relevant human                 direct equilibrium ATP pool, be <br />         diseases osteopetrosis distal renal tubular acido-               hydrolyzed buffer free phosphate levels (Figure 4). ATP <br />         sis [33,34]. In screen, subunits V-ATPase               itself hydrolyzed (for example, V-ATPase to <br />         regulators identified. In addition, V-                   assure vacuolar acidification). But importantly, ATP <br />         ATPase activity glucose regulated; presence                pool directly indirectly sustains cellular activities and <br />         glucose, V-ATPase functional absence glu-                processes DNA RNA metabolism, ribosome bio- <br />         cose causes reversible dissociation V0 V1 subunits                genesis assembly, transcription protein biosynthesis, <br />         V-ATPase inactivated [35]. This phenomenon                       synthesis storage carbohydrates (Figure 4). <br />         explain decline poly P levels soon growth             Hence, pathways indirectly associated with <br />         medium depleted glucose [36]. However, based                 poly P metabolism. Ribosome biogenesis, example, claims <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                       Genome Biology 2006,   Volume 7, Issue 11, Article R109   Freimoser et al. R109.7 <br />  <br />  <br />  <br />  <br />                                                                                   Conclusion <br />            Phosphorylated                Storage                                  PolyP metabolism primary metabolism strongly <br />           sugars &amp; proteins           carbohydrates                               interdependent: On hand, poly P levels depend necessar- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   comment <br />                                                                                   ily integrity primary metabolism reflect the <br />                                                                                   physiological state cell. On hand, poly P itself <br />                                                                                   influences cell metabolism importance energy <br />  Phospholipids <br />                             ATP                          poly P                   phosphate homeostasis. This genome-wide analysis <br />                                                                                   poly P content implies poly P indirect link <br />                                                                                   different cellular pathways phosphate, <br />                                                                                   energy carbohydrate metabolism transcription and <br />                        NTPs <br />                                              Pi                                   translation activities. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   reviews <br />                 DNA / RNA <br />                                                                                   Materials methods <br />                                            Pho4                                   Strains culturing conditions <br />                                        (PHO pathway)                              The haploid yeast <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>knock-out</span> strains (YKO [23], based the <br />                                                                                   strain BY4741: MAT his3&#206;&#8221;1 leu2&#206;&#8221;0 met15&#206;&#8221;0 ura3&#206;&#8221;0) were <br />                                                                                   grown 96-well deep plates 30&#194;&#176;C YPD medium (1% <br /> Figure <br /> Poly <br /> network&apos; <br />      P links <br />          4 energy phosphate metabolism &apos;phosphate-energy-                w/v yeast extract, 2% w/v peptone, 2% w/v glucose, with <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   reports <br /> Poly P links energy phosphate metabolism &apos;phosphate-energy-              optional addition 200 mg/l G418). For screen, fresh <br /> network&apos;. Enzymatic reactions main phosphate/energy pools,            YPD medium inoculated 40 &#206;&#188;l precultured <br /> ATP poly P (shown black arrows) (at partly) hypothesized.        (3d) stationary cells cells harvested 4 h (first <br /> Flow different organic phosphate/energy pools indicated gray <br /> arrows. In contrast ATP, poly P directly interconnected       second round screen) 2 h, 4 h, 8 h 24 h <br /> other phosphate pools and, therefore, levels fluctuate more.                  (third round screen). Growth cell number moni- <br />                                                                                   tored measuring light scattering 600 nm (OD600) in <br />                                                                                   BioTek PowerWave&#226;&#8222;&#162; XS microplate spectrophotometer <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   deposited research <br />                                                                                   (BioTEK Instruments Inc., Vermont, USA). For detailed poly <br />                                                                                   P measurements selected strains, culture inocu- <br />                                                                                   lated cell density 1 OD600/ml (approximately 107 cells/ <br />                                                                                   ml), time-point 1 OD600 equivalent cells was <br /> about 60% total transcription, 90% mRNA splicing,                       collected poly P quantification. <br /> requirement RNA polymerases dedication <br /> of 200 different proteins and, thus, considerably                          Poly P purification quantification <br /> affects energy homeostasis [41,42].                                               Poly P extraction, purification quantification per- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   refereed research <br />                                                                                   formed described previously [36]. In short, cells pel- <br /> Intervention point phosphate-energy-net-                       leted deep plates centrifugation (approximately <br /> work inevitably causes effects require read-                       640 g, 15 minutes, 4&#194;&#176;C), supernatant discarded, 50 &#206;&#188;l <br /> justment allocation resources different                     1 M sulphuric acid added suspension neu- <br /> phosphate energy pools. In context, &apos;interventions&apos;                      tralized 50 &#206;&#188;l 2 M NaOH 100 &#206;&#188;l Tris-malate buffer (1 <br /> could represent genetic changes externally induced                       M, pH 7.5, 6% neutral red solution (0.1% neutral red 70% <br /> physiological changes occur, example,                      ethanol)). Cell fragments pelleted centrifugation <br /> progression different growth phases. Both types                        (approximately 640 g, 15 minutes, 4&#194;&#176;C) 100 &#206;&#188;l the <br /> changes affect poly P levels strongly,                     supernatant removed. Poly P purified, enzymati- <br /> other characteristics altered. Many mutants failed                      cally digested quantified using colorimetric assay [36].                  interactions <br /> to accumulate poly P invest accumula- <br /> tion storage carbohydrates (glycogen) longer                      Screening procedure data analysis <br /> buffer expression phosphate regulated genes (as                            For poly P measurements poly P raw data normal- <br /> indicated elevated rAPase activity). In contrast, total phos-                  ized cell density (OD600) measured corre- <br /> phate content closely followed poly P levels,                    sponding culture. The poly P content expressed <br /> case correlation coefficient relative poly P                  relative median cultures 96-well <br /> all total phosphate values (log2-transformed) 0.65.                      plate (first screen) poly P content wild-type <br />                                                                                                                                                                   information <br />  <br />  <br />  <br />  <br /> Even ATP levels se significantly altered,                      strain BY4741 (second screens). All strains a <br /> but compared characteristics meas-                         relative poly P content 1.5-fold higher relative poly P con- <br /> ured, changes smaller, confirming, partly,                    tent 0.67-fold lower selected second third <br /> the general notion that, viable growing cells, ATP levels                  round screening. In screen, data collected at <br /> remain constant [43].                                                             different time points (2 h, 4 h, 8 h, 24 h). Only strains <br />  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.8 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                                 http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         fulfilled following criteria selected          GmbH, Mannheim, Germany) relative light units emitted <br />         subsequent data analysis: relative poly P content through-               measured luminometer (Lumat LB 9507, Berthold <br />         24 h time-course varied two-fold;                 Technologies GmBH Co. KG, Bad Wildbad, Germany). <br />         time-point differed two-fold wild- <br />         type poly P content. Strains did grow excluded.                To quantify glycogen cell pellet frozen glyco- <br />         In strains, affected gene reintroduced                      gen extracted described [49]. The extracted glycogen <br />         expressed plasmid, restored poly P levels (data                 digested adding 10 U alpha-amylase 12.6 U <br />         shown).                                                                  amyloglucosidase sodium acetate buffer (pH 4.8, 220 <br />                                                                                      mM) 55&#194;&#176;C 16 h. Released glucose quantified with <br />         The poly P data selected strains screens                  D-Glucose HK kit (Megazyme International Ireland Ltd., <br />         time-points combined clustered hierarchically (with                 Bray, Ireland). <br />         Pearson uncentered Pearson correlation distance <br />         measure, respectively, complete average linkage                    Acid phosphatase activity assayed according Huang <br />         Genesis [44]). The selected strains ranked accord-                 O&apos;Shea [27] 50 &#206;&#188;l cell suspension adding 200 &#206;&#188;l p- <br />         ing strongly poly P levels affected (calculated               nitrophenyl-phosphate (20 mM). After 15 minutes room <br />         sum squares log2-transformed poly P ratios                 temperature, 200 &#206;&#188;l 10% ice-cold trichloracetic acid and <br />         time-points screen).                                400 &#206;&#188;l sodium carbonate solution (2 M) added the <br />                                                                                      OD420 measured. All results expressed relative to <br />         The genes deleted strains altered poly P               respective values wild-type cells (BY4741). <br />         content categorized studied different computa- <br />         tional tools comparison available data sets. All <br />         genes categorized according GO terminology                    Additional data files <br />         using annotation given go_slim_mapping.tab file                   The following additional data available online <br />         available Saccharomyces Genome Database [45].                       version paper. Additional data file 1 contains poly P <br />         Significantly overrepresented GOSlim terms (compared                      data screens. <br />         representation complete yeast genome)                      All <br />                                                                                      Click <br />                                                                                          poly    data <br />                                                                                               P data <br />                                                                                      Additional <br />                                                                                            file <br />                                                                                                      offile <br />                                                                                                          the1 screens. <br />                                                                                                                     screens <br />         determined using Cytoscape BiNGO plugin (set- <br />         tings: hypergeometric test statistic, false discovery rate cor-              Acknowledgements <br />         rection multiple tests, P &#226;&#8240;&#164; 0.05 confidence limit) [46,47].              We thank S Eicke, M Stadler, S Ernst, C Neupert S Clerc help at <br />         All data manipulations performed Microsoft Excel                     different stages research. M Aebi acknowledged helpful and <br />                                                                                      motivating discussions, SZeeman group providing spectropho- <br />         FileMaker.                                                               tometer help glycogen quantification, W Gruissem J F&#195;&#188;t- <br />                                                                                      terer making luminometer available. M Peter M Sohrmann are <br />                                                                                      given thanks providing yeast strains. This work supported the <br />         Measurement total phosphate, ATP, glycogen                            Swiss National Science Foundation (grant 3100A0-112083/1) ETH <br />         acid phosphatase activity                                                    Zurich. <br />         Detailed measurements metabolites activities <br />         performed selected strains various time-points (0 <br />         h, 2 h, 5 h, 24 h). Individual cultures time-point             References <br />         inoculated OD600 = 1 (approximately 107 cells/ml). At                  1.   Kornberg A, Rao NN, Ault-Rich&#195;&#169; D: Inorganic polyphosphate: a <br />                                                                                           molecule functions. Annu Rev Biochem 1999, 68:89-125. <br />         time point 1OD600 equivalent cells harvested                 2.   Neef DW, Kladde MP: Polyphosphate loss promotes SNF/SWI- <br />         used subsequent measurements. All data obtained                          Gcn5-dependent mitotic induction PHO5. Mol Cell Biol <br />                                                                                           2003, 23:3788-3797. <br />         duplicate measurements.                                                 3.   Ogawa N, DeRisi J, Brown PO: New components for <br />                                                                                           phosphate accumulation polyphosphate metabolism in <br />         For measurement total phosphate, cells pelleted,                          Saccharomyces cerevisiae revealed genomic expression <br />                                                                                           analysis. Mol Biol Cell 2000, 11:4309-4321. <br />         resuspended 200 &#206;&#188;l 1 MH2SO4 heated boiling                    4.   Thomas MR, O&apos;Shea EK: An intracellular phosphate buffer fil- <br />         water bath 20 minutes. Released phosphate quanti-                         ters transient fluctuations extracellular phosphate levels. <br />         fied molybdate malachite green described [36].                        Proc Natl Acad Sci USA 2005, 102:9565-9570. <br />                                                                                      5.   Kuroda A, Nomura K, Ohtomo R, Kato J, Ikeda T, Takiguchi N, <br />         For ATP quantification neutralized cell extract                   Ohtake H, Kornberg A: Role inorganic polyphosphate pro- <br />         poly P quantification prepared, ATP quantified                        moting ribosomal, protein degradation Lon protease <br />                                                                                           E. coli. Science 2001, 293:705-708. <br />         method adapted Hyswert et al. [48]. The neutral-                   6.   Wang L, Fraley CD, Faridi J, Kornberg A, Roth RA: Inorganic <br />         ized sample (20 &#206;&#188;l) added 80 &#206;&#188;l Tris buffer (20 mM,                       polyphosphate stimulates mammalian TOR, kinase <br />         pH8, 2 mM EDTA) 4 &#206;&#188;l phosphoenol pyruvate                                involved proliferation mammary cancer cells. Proc <br />                                                                                           Natl Acad Sci USA 2003, 100:11249-11254. <br />         added (2.5 mM, pH8, 0.125 M MgSO4, 0.312 M K2SO4). For                       7.   Kulaev IS, Vagabov VM, Kulakovskaya TV: The Biochemistry Inorganic <br />         quantification ATP, 5 &#206;&#188;l sample added 45                    Polyphopshates 2nd edition. Chichester, West Sussex: John Wiley and <br />                                                                                           Sons, Ltd; 2004. <br />         &#206;&#188;l luciferase buffer (10 mM Tris-H2SO4, pH 7.4, 3.5 mM                       8.   McInerney P, Mizutani T, Shiba T: Inorganic polyphosphate inter- <br />         MgSO4). After addition 50 &#206;&#188;l luciferase solution (Roche                    acts ribosomes promotes translation fidelity vitro <br />         ATP Bioluminescence Assay kit CLS II, Roche Diagnostics                           vivo. Mol Microbiol 2006, 60:438-447. <br />                                                                                      9.   Kim KS, Rao NN, Fraley CD, Kornberg A: Inorganic <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                     Genome Biology 2006,      Volume 7, Issue 11, Article R109      Freimoser et al. R109.9 <br />  <br />  <br />  <br />  <br />       polyphosphate essential long-term survival viru-                     JF: Characterization yeast ionome: genome-wide <br />       lence factors Shigella Salmonella spp. Proc Natl Acad Sci                analysis <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> mineral trace element homeostasis <br />       USA 2002, 99:7675-7680.                                                         Saccharomyces cerevisiae. Genome Biol 2005, 6:R77. <br /> 10.   Rashid MH, Rumbaugh K, Passador L, Davies DG, Hamood AN,                  33.   Sobacchi C, Frattini A, Orchard P, Porras O, Tezcan I, Andolina M, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       comment <br />       Iglewski BH, Kornberg A: Polyphosphate kinase essential                  Babul-Hirji R, Baric I, Canham N, Chitayat D, et al.: The mutational <br />       biofilm development, quorum sensing, virulence Pseu-                     spectrum human malignant autosomal recessive <br />       domonas aeruginosa. Proc Natl Acad Sci USA 2000, 97:9636-9641.                  osteopetrosis. Hum Mol Genet 2001, 10:1767-1773. <br /> 11.   Crooke E, Akiyama M, Rao NN, Kornberg A: Genetically altered              34.   Stover EH, Borthwick KJ, Bavalia C, Eady N, Fritz DM, Rungroj N, <br />       levels inorganic polyphosphate Escherichia coli. J Biol                   Giersch AB, Morton CC, Axon PR, Akil I, et al.: Novel ATP6V1B1 <br />       Chem 1994, 269:6290-6295.                                                       ATP6V0A4 mutations autosomal recessive distal <br /> 12.   Sethuraman A, Rao NN, Kornberg A: The endopolyphosphatase                       renal tubular acidosis new evidence hearing loss. J <br />       gene: essential Saccharomyces cerevisiae. Proc Natl Acad Sci                 Med Genet 2002, 39:796-803. <br />       USA 2001, 98:8542-8547.                                                   35.   Parra KJ, Kane PM: Reversible association V1 and <br /> 13.   Pick U, Weiss M: Polyphosphate hydrolysis acidic vacu-                   V0 domains yeast vacuolar H+-ATPase unconven- <br />       oles response amine-induced alkaline stress halo-                  tional glucose-induced effect. Mol Cell Biol 1998, 18:7064-7074. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       reviews <br />       tolerant alga Dunaliella salina. Plant Physiol 1991, 97:1234-1240.        36.   Werner TP, Amrhein N, Freimoser FM: Novel method the <br /> 14.   Rohloff P, Montalvetti A, Docampo R: Acidocalcisomes                    quantification inorganic polyphosphate (iPoP) Saccharo- <br />       contractile vacuole complex involved osmoregulation                      myces cerevisiae shows dependence iPoP content the <br />       Trypanosoma cruzi. J Biol Chem 2004, 279:52270-52281.                        growth phase. Arch Microbiol 2005, 184:129-136. <br /> 15.   Weiss M, Bental M, Pick U: Hydrolysis polyphosphates               37.   Cohen A, Perzov N, Nelson H, Nelson N: A novel family yeast <br />       permeability changes response osmotic shocks cells                     chaperons involved distribution V-ATPase and <br />       halotolerant alga Dunaliella.            Plant Physiol 1991,             membrane proteins. J Biol Chem 1999, 274:26885-26893. <br />       97:1241-1248.                                                             38.   M&#195;&#188;ller O, Neumann H, Bayer MJ, Mayer A: Role Vtc pro- <br /> 16.   Zhang H, Gomez-Garcia MR, Brown MR, Kornberg A: Inorganic                       teins V-ATPase stability membrane trafficking. J Cell <br />       polyphosphate Dictyostelium discoideum : influence                        Sci 2003, 116:1107-1115. <br />       development, sporulation, predation. Proc Natl Acad Sci               39.   Odorizzi G, Cowles CR, Emr SD: The AP-3 complex: coat of <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       reports <br />       USA 2005, 102:2731-2735.                                                        colours. Trends Cell Biol 1998, 8:282-288. <br /> 17.   Smith SA, Mutch NJ, Baskar D, Rohloff P, Docampo R, Morrissey JH:         40.   Huizing M, Boissy RE, Gahl WA: Hermansky-Pudlak syndrome: <br />       Polyphosphate modulates blood coagulation fibrinolysis.                     vesicle formation yeast man. Pigment Cell Res 2002, <br />       Proc Natl Acad Sci USA 2006, 103:903-908.                                       15:405-419. <br /> 18.   Beauvoit B, Rigoulet M, Guerin B, Canioni P: Polyphosphates          41.   Rudra D, Warner JR: What better measure ribosome <br />       source high energy phosphates yeast mitochondria: 31                    synthesis? Genes Dev 2004, 18:2431-2436. <br />       P NMR study. FEBS Lett 1989, 252:17-21.                                   42.   Warner JR: The economics ribosome biosynthesis yeast. <br /> 19.   Harold FM, Miller A: Intracellular localization inorganic                    Trends Biochem Sci 1999, 24:437-440. <br />       polyphsophate Neurospora crassa. Biochim Biophys Acta 1961,            43.   Ditzelm&#195;&#188;ller G, Wohrer W, Kubicek CP, Rohr M: Nucleotide pools <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       deposited research <br />       50:261-270.                                                                     growing, synchronized stressed cultures Saccharo- <br /> 20.   Docampo R, Souza W, Miranda K, Rohloff P, Moreno SN: Acido-                  myces cerevisiae. Arch Microbiol 1983, 135:63-67. <br />       calcisomes - conserved bacteria man. Nat Rev                      44.   Sturn A, Quackenbush J, Trajanoski Z: Genesis: Cluster analysis of <br />       Microbiol 2005, 3:251-261.                                                      microarray data. Bioinformatics 2002, 18:207-208. <br /> 21.   Indge KJ: Polyphosphates yeast cell vacuole. J Gen                 45.   Saccharomyces Genome Database                    [ftp://genome-ftp.stan <br />       Microbiol 1968, 51:447-455.                                                     ford.edu/pub/yeast/data_download/literature_curation/] <br /> 22.   Urech K, Durr M, Boller T, Wiemken A, Schwencke J: Localization           46.   Maere S, Heymans K, Kuiper M: BiNGO: Cytoscape plugin to <br />       polyphosphate vacuoles Saccharomyces cerevisiae.                       assess overrepresentation Gene Ontology categories in <br />       Arch Microbiol 1978, 116:275-278.                                               biological networks. Bioinformatics 2005, 21:3448-3449. <br /> 23.   Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K,              47.   Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin <br />       Andre B, Bangham R, Benito R, Boeke JD, Bussey H, et al.: Func-                 N, Schwikowski B, Ideker T: Cytoscape: software environment <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       refereed research <br />       tional characterization S. cerevisiae genome gene                     integrated models biomolecular interaction networks. <br />       deletion parallel analysis. Science 1999, 285:901-906.                      Genome Res 2003, 13:2498-2504. <br /> 24.   Darsow T, Burd CG, Emr SD: Acidic di-leucine motif essential              48.   Hyswert DW, Morrison NM: Studies ATP, ADP, AMP <br />       AP-3-dependent sorting restriction functional                    concentrations yeast beer. J Am Soc Brew Chem 1977, <br />       specificity Vam3p vacuolar t-SNARE. J Cell Biol 1998,                    35:160-167. <br />       142:913-922.                                                              49.   Wang Z, Wilson WA, Fujino MA, Roach PJ: Antagonistic controls <br /> 25.   Wilson WA, Wang Z, Roach PJ: Systematic identification                   autophagy glycogen accumulation Snf1p, yeast <br />       genes affecting glycogen storage yeast Sac <br /> </body></html>