<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, June 2006, p. 2178&#226;&#8364;&#8220;2184                                                                   Vol. 50, No. 6 <br /> 0066-4804/06/$08.00&#226;&#171;&#185;0 doi:10.1128/AAC.00140-06 <br /> Copyright &#194;&#169; 2006, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         Molecular Validation LpxC Antibacterial Drug Target <br />                          Pseudomonas aeruginosa <br />        Khisimuzi E. Mdluli,1&#226;&#8364;&#160; Pamela R. Witte,1&#226;&#8364;&#161; Toni Kline,2&#194;&#167; Adam W. Barb,3 Alice L. Erwin,1* <br />         Bryce E. Mansfield,1&#194;&#182; Amanda L. McClerren,3&#227;&#8250;&#179; Michael C. Pirrung,4&#194;&#167;&#194;&#167; L. Nathan Tumey,3# <br />                     Paul Warrener,1&#226;&#8364;&#160;&#226;&#8364;&#160; Christian R. H. Raetz,3 C. Kendall Stover1&#226;&#8364;&#161;&#226;&#8364;&#161; <br />         Departments Research Biology1 Chemistry,2 Chiron Corporation, 201 Elliott Avenue West, Suite 150, Seattle, <br />              Washington 98119, Departments Biochemistry3 Chemistry,4 Duke University Medical Center, <br />                                                Durham, North Carolina 27710 <br />  <br />                          Received 2 February 2006/Returned modification 2 March 2006/Accepted 24 March 2006 <br />  <br />              LpxC [UDP-3-O-(R-3-hydroxymyristoyl)-GlcNAc deacetylase] metalloamidase catalyzes first <br />           committed step biosynthesis lipid A component lipopolysaccharide. A previous study (H. R. <br />           Onishi, B. A. Pelak, L. S. Gerckens, L. L. Silver, F. M. Kahan, M. H. Chen, A. A. Patchett, S. M. Galloway, S. A. <br />           Hyland, M. S. Anderson, C. R. H. Raetz, Science 274:980&#226;&#8364;&#8220;982, 1996) identified series synthetic <br />           LpxC-inhibitory molecules bactericidal Escherichia coli. These molecules did inhibit the <br />           growth Pseudomonas aeruginosa developed antibacterial drugs. The <br />           inactivity LpxC inhibitors P. aeruginosa raised possibility LpxC activity be <br />           essential gram-negative bacteria. By placing lpxC gene P. aeruginosa tight control an <br />           arabinose-inducible promoter, demonstrated essentiality LpxC activity P. aeruginosa. It found <br />           compound L-161,240, potent inhibitor previous study, active P. aeruginosa <br />           construct endogenous lpxC gene inactivated LpxC activity supplied the <br />           lpxC gene E. coli. Conversely, E. coli construct growth dependent P. aeruginosa lpxC <br />           gene resistant compound. The differential activities L-161,240 bacterial species are <br />           result primarily greater potency E. coli enzyme differences intrinsic <br />           resistance bacteria antibacterial compounds permeability efflux. These data validate P. <br />           aeruginosa LpxC target novel antibiotic drugs help direct design inhibitors against <br />           clinically important gram-negative bacteria. <br />  <br />  <br />    Lipopolysaccharide critical function gram-nega-                    membrane permeable small molecules of <br /> tive bacterial membrane integrity resistance host                       E. coli, P. aeruginosa multidrug efflux pumps. <br /> defenses, therefore, conserved lipopolysaccharide                      As result of factors, P. aeruginosa less <br /> biosynthetic enzymes attractive targets novel anti-                    susceptible E. coli antibiotics (24). Several <br /> bacterial drugs. A drug targeting enzymes biosyn-                      laboratories focused metalloenzyme LpxC <br /> thetic pathway need active Pseudomonas                     [UDP-(3-O-acyl)-N-acetylglucosamine deacetylase], it <br /> aeruginosa nonfermenting gram-negative bacterial                     catalyzes committed step lipid A synthesis (Fig. 1) <br /> species, Escherichia coli enteric                 demonstrated essential growth E. <br /> bacteria, clinically useful. The P. aeruginosa outer                     coli (3, 12, 38). P. aeruginosa LpxC similar sequence <br />                                                                                (Fig. 2) catalyzes activity (11). While essen- <br />                                                                                tiality LpxC activity P. aeruginosa formally <br />   * Corresponding author. Present address: Microbial Pathogens Pro-            proven, lpxC gene inactivated saturating trans- <br /> gram, Seattle Biomedical Research Institute, 307 Westlake Ave. N.,             poson mutagenesis study (15). These data suggest might <br /> Seattle, WA 98109. Phone: (206) 256-7431. Fax: (206) 256-7229. E-              possible discover LpxC inhibitors active E. <br /> mail: alice.erwin@sbri.org.                                                    coli P. aeruginosa. However, early LpxC in- <br />   &#226;&#8364;&#160; Present address: Global Alliance TB Drug Discovery, New <br />                                                                                hibitors, showed antibacterial activity E. <br /> York, NY 10004. <br />   &#226;&#8364;&#161; Present address: deCODE Biostructures, Bainbridge Island, WA               coli certain organisms, able inhibit growth of <br /> 98110.                                                                         P. aeruginosa (5, 12, 27&#226;&#8364;&#8220;29). It tempting assume that <br />   &#194;&#167; Present address: Department Genome Sciences, University              reason failure intrinsic resistance P. <br /> Washington, Seattle, WA 98195.                                                 aeruginosa antibiotics. Challenging assumption, we <br />   &#194;&#182; Present address: Department Pathobiology, University Wash- <br /> ington, Seattle, WA 98195.                                                     undertook studies described evaluate basis <br />   &#227;&#8250;&#179; Present address: Department Chemistry, University Illinois,          refractory nature P. aeruginosa LpxC inhibitors <br /> Urbana-Champaign, IL 61801.                                                    effective E. coli. We focused com- <br />   &#194;&#167;&#194;&#167; Present address: Department Chemistry, University Cali-             pound L-161,240 (Fig. 1), active LpxC in- <br /> fornia Riverside, Riverside, CA 92521. <br />   # Present address: Wyeth Research, Pearl River, NY 10965. <br />                                                                                hibitors reported researchers Merck (4, 27). We found <br />   &#226;&#8364;&#160;&#226;&#8364;&#160; Present address: Rosetta Inpharmatics LLC, Seattle, WA 98109.             critical reason inactivity compound <br />   &#226;&#8364;&#161;&#226;&#8364;&#161; Present address: Pfizer Corporation, Ann Arbor, MI 48105.                 P. aeruginosa failure inhibit enzyme activ- <br />  <br />                                                                         2178 <br />  VOL. 50, 2006                                                                                                                     P. AERUGINOSA LpxC                    2179 <br />  <br />  <br />  <br />  <br />   FIG. 1. (A) LpxC catalyzes deacetylation UDP-(3-O-acyl)-N-acetylglucosamine, second step biosynthesis lipid A. Because <br /> the step reversible, LpxC reaction committed step conversion UDP-N-acetylglucosamine lipid A. (B) Structure <br /> of LpxC inhibitor L-161,240 (27). <br />  <br />  <br />  <br /> ity. These findings implications designing effective                        &#226;?&#174;g plasmid pKD6 containing E. coli lpxC gene (34). The lpxC genes were <br /> strategies identify LpxC inhibitors developed                         amplified using Pwo DNA polymerase (Roche) 100-&#226;?&#174;l reaction mixture <br />                                                                                      containing 200 &#226;?&#174;M concentration deoxynucleoside triphosphate a <br /> as novel antibacterial drugs.                                                        0.5 &#226;?&#174;M concentration primer 30 cycles (94&#194;&#176;C denaturation, 55&#194;&#176;C <br />                                                                                      annealing, 72&#194;&#176;C polymerization). The PCR products purified the <br />                         MATERIALS AND METHODS                                        QIAquick PCR purification kit QIAGEN digested NdeI and <br />                                                                                      EcoRI restriction enzymes. The bands sizes predicted lpxC genes <br />    Reagents bacterial cultivation. The bacterial strains plasmids used are <br />                                                                                      identified following gel electrophoresis excised gel. The <br /> listed Table 1. P. aeruginosa strains grown 37&#194;&#176;C Luria-Bertani (LB) <br />                                                                                      excised DNA purified using QIAquick gel extraction kit QIAGEN. <br /> broth (Difco) plated sheep blood agar (Remel). E. coli grown LB <br />                                                                                      The purified DNA cut NdeI EcoRI ligated T7 <br /> broth LB agar. EDTA, bis-Tris buffer, sucrose, arabinose, dimethyl <br /> sulfoxide (DMSO) purchased Sigma ultrapure agents. Yeast extract        expression vector (36) pET21b (Novagen) cut multiple <br /> and tryptone obtained Difco. Restriction enzymes, T4 DNA ligase,           cloning site NdeI EcoRI. The ligation mixture transformed into <br /> and reaction buffers obtained New England Biolabs. Polymyxin         DH5&#226;?&#163;, plated LB agar containing ampicillin (250 &#226;?&#174;g/ml). The <br /> B nonapeptide, tetracycline, ampicillin, carbenicillin, gentamicin, kanamycin    inserts resulting clones sequenced subcloned. The E. <br /> were purchased Sigma. Compound L-161,240 synthesized de-             coli P. aeruginosa inserts subcloned pDN19 produce plasmids <br /> scribed previously (4). Antibacterial compounds dissolved DMSO            pEC-lpxC1 pPA-lpxC1, respectively, low-copy-number complementation <br /> make stock solutions polymyxin B nonapeptide 3 mg/ml, L-161,240 10          E. coli JBK-1 subcloned pUCP30T produce plasmids pEC-lpxC2 <br /> mg/ml, tetracycline 125 mg/ml. For growth curves, DMSO added           pPA-lpxC2, respectively, complementation P. aeruginosa promoter <br /> control tubes needed DMSO concentrations             replacement <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span>. <br /> cultures experiment.                                                        Construction P. aeruginosa strain lpxC expression tightly controlled <br />    Enzyme inhibition assays. LpxC activity measured previously described      araBAD promoter. Promoter replacement carried using homol- <br /> (13, 20), using crude cell extracts (38) E. coli W3110 P. aeruginosa    ogous recombination strategy, recombination pBEM10 P. <br /> PAO1 purified enzyme E. coli BL21/DE3/pLysS/pJEJ1 (14) P. aerugi-         aeruginosa removed native lpxC promoter placed tightly regulated <br /> nosa PAO1 (16) enzyme source. Assays 25 mM phosphate             araBAD promoter upstream lpxC chromosome (Fig. 3). In preliminary <br /> buffer pH 7.4 5 &#226;?&#174;M substrate 30&#194;&#176;C, enzyme concentrations             experiments (data shown) promoter controlled expression of <br /> (typically 0.5 10 nM) adjusted conversion 10% time     luciferase gene lux (9), P. aeruginosa, low <br /> course assays.                                                                background level expression absence arabinose expression <br />    DNA manipulations. Standard recombinant DNA procedures used (30).            eliminated addition glucose E. coli (22). Background levels <br /> The primers amplification coding region lpxC genes included        promoter expression successfully reduced altering sequence the <br /> NdeI EcoRI restriction sites subsequent cloning. For E. coli gene,   ribosome binding site original sequence AGGAG CTTCT. <br /> primers 5&#226;&#172;&#732;-GGGAATTCCATATGATCAAACAAAGGACACTTAAAC                                    Plasmid pPW101 constructed ligating RP4 origin transfer oriT <br /> GT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CCGGAATTCTTATGCCAGTACAGCTGAAGGCGCT-3&#226;&#172;&#732;,                              pSP72. oriT amplified plasmid pEX100T (33) introduction <br /> for P. aeruginosa gene, 5&#226;&#172;&#732;-GGGAATTCCATATGATGATCAAA                     NdeI AatII restriction sites. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> create pBEM10, following DNA <br /> CAACGCACCTTGAAGAACAT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CCGGAATTCCTACACTGCCG                                  pieces amplified sequentially ligated pPW101: tetracycline <br /> CCGCCGGGCGCATATAG-3&#226;&#172;&#732;. These primers used PCR mixture                       resistance marker plasmid pUCP26 (26), araBAD promoter (8) <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> containing template 10 50 &#226;?&#174;g P. aeruginosa genomic DNA 1         plasmid pBAD HisB (Invitrogen), altered ribosome binding site, the <br />  2180        MDLULI ET AL.                                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />    FIG. 2. Comparison predicted LpxC sequences E. coli P. aeruginosa. An 82% similarity 57% identity shared the <br /> two sequences entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> protein. Key residues indicated color symbols residues, follows. The <br /> novel zinc binding motif characteristic known LpxC enzymes, HKXXD, shown red (stars). Four conserved histidines shown <br /> in blue (&#226;&#8211;&#160;), E. coli histidines absent P. aeruginosa shown cyan (&#228;&#352;?). The conserved phenylalanines providing a <br /> hydrophobic patch (7) shown orange (&#197;&#8217;). Analogous A. aeolicus, E. coli histidine 265 stabilize oxyanion intermediate the <br /> transition state. Significant differences enzyme fold participating residues suggested differences key residues, example, the <br /> two nonconserved histidines E. coli. <br />  <br />  <br /> araC repressor gene (17, 31), pBAD HisB, 340 <span id='am-82' about='xsp:base' typeof='owl:Thing'>base</span> pairs         culture diluted 5 &#226;&#171;&#187; 105 cells/ml. A total 200 &#226;?&#174;l diluted culture <br /> of P. aeruginosa lpxC gene PA4406. The tetracycline resistance marker               added wells containing 4 &#226;?&#174;l inhibitor. The 96-well plates incubated <br /> amplified using forward primer introduced BglII site (5&#226;&#172;&#732;-AGATCTCA              37&#194;&#176;C overnight OD600s determined using Spectramax Plus <br /> AGGGTTGGTTTGCGCA-3&#226;&#172;&#732;) reverse primer introduced EcoRI                      plate reader (Molecular Devices). <br /> site (5&#226;&#172;&#732;-GAATTCTAATTCTCATGTTTGACA-3&#226;&#172;&#732;). The araBAD promoter and <br /> araC gene amplified piece pBAD HisB vector. The forward <br /> primer introduced XhoI site (5&#226;&#172;&#732;-CTCGAGGCATGCATAATGTG CCTGT                                                             RESULTS <br /> C-3&#226;&#172;&#732;), reverse primer introduced HindIII site (5&#226;&#172;&#732;-AAGCTTCTCCTG <br /> TTAGCCCAAAAAAACG-3&#226;&#172;&#732;). A primer set used alter ribosome                         Compound L-161,240, potent LpxC inhibi- <br /> binding site introduce upstream BssHII site (5&#226;&#172;&#732;-GCGCGCGGACGAA <br />                                                                                         tors described Onishi et al., active E. coli, an <br /> AGTAAACCCACTGG-3&#226;&#172;&#732;) downstream HindIII site (5&#226;&#172;&#732;-AAGCTTATT <br /> CAGAAGGTTAGCCCAAAAAAACGGG-3&#226;&#172;&#732;). The 340 bases PAO1                              MIC 1 &#226;?&#174;g/ml, activity P. aeruginosa (MIC &#226;&#172;&#381; <br /> lpxC amplified PAO1 genomic DNA. The forward primer introduced                50 &#226;?&#174;g/ml) (27). We compound inactive <br /> a HindIII site (5&#226;&#172;&#732;-AAGCTTATGATCAAACAACGCACCTT-3&#226;&#172;&#732;), re-                          wild-type strains P. aeruginosa (PAO1 ATCC <br /> verse primer introduced XbaI site (5&#226;&#172;&#732;-TCTAGAAGCGCTGCCATCCATG                         27853), strain PAO200, genes mexAB oprM, <br /> ATCGG-3&#226;&#172;&#732;). These pieces ligated pPW101 form final <br /> product, pBEM10. Transformation pBEM10 P. aeruginosa removed the <br />                                                                                         encoding major multidrug efflux pump, deleted (32), <br /> native lpxC promoter replaced tightly regulated araBAD promoter         hypersusceptible strain Z61 (ATCC 35151) (2). <br /> just upstream chromosomal copy lpxC. Growth resulting strain,          Treatment PAO1 PAO200 polymyxin B non- <br /> designated PAO1-PBAD-lpxC, dependent arabinose.                                  apeptide (3 &#226;?&#174;g/ml) increase permeability mem- <br />    Growth curves. Bacterial cultures prepared diluting overnight cultures to <br />                                                                                         brane (23) failed render strains susceptible to <br /> an optical density 600 nm (OD600) 0.1 5 ml LB. The inhibitor L-161,240 <br /> was added bacterial cultures final concentration 50 &#226;?&#174;g/ml 10 &#226;?&#174;g/ml.   L-161,240 (data shown). These observations suggested <br /> The cultures incubated shaking, 0.8 ml taken OD600 readings       failure L-161,240 reach target P. <br /> over course experiment. DH5-&#226;?&#163;, PAO1, PAO200 grown            aeruginosa cells primary reason its <br /> 37&#194;&#176;C. In cases temperature-sensitive JBK-1 strains assayed,    failure inhibit growth P. aeruginosa. <br /> cultures grown 42&#194;&#176;C overnight time course cultures. <br />    Microtiter assay growth inhibition promoter replacement mutants by <br />                                                                                           L-161,240 38 times potent E. coli than <br /> L-161,240. Single colonies DH5&#226;?&#163;, PAO1, mutant strain picked            P. aeruginosa in vitro assay LpxC activity in <br /> and grown LB 37&#194;&#176;C, shaking approximately 4 h,              crude bacterial extracts (Table 2). For E. coli, inhibitor was <br />  VOL. 50, 2006                                                                                                       P. AERUGINOSA LpxC                2181 <br />  <br />  <br />                                                           TABLE 1. Bacterial strains plasmids <br />         Strain plasmid                                          Relevant characteristic(s)a                               Source reference(s) <br />  <br /> P. aeruginosa strains <br />   PAO1                                  Wild type; sequenced strain                                                      10, 35 <br />   PAO200                                mexAB-oprM                                                                       32 <br />   ATCC 35151 (Z61)                      Chemically mutagenized strain hypersusceptible antibiotics                    18, 39 <br />   PAO1-PBAD-lpxC                        lpxC promoter replaced araBAD promoter                                        This work <br />  <br /> E. coli strains <br />   W3110                                 K-12 derivative                                                                  E. coli Genetic Stock Center <br />   DH5&#226;?&#163;                                  Cloning host                                                                     Gibco/Life Technologies <br />   BL221/DE3/pLysS/pJEJ1                 T7 RNA polymerase-driven expression E. coli lpxC                      13 <br />   JBK-1/pKD6                            Chromosomal lpxC disrupted Km cassette; lpxC temp-sensitive              34 <br />                                           plasmid; Amp <br />  <br /> Plasmids <br />   pET21b                                Expression plasmid; Amp                                                          Novagen <br />   pDN19                                 Low-copy-number plasmid; Tet                                                     25 <br />   pUCP26                                Tet                                                                              26 <br />   pUCP30T                               Replicates E. coli P. aeruginosa; Gm                                 33 <br />   pSP72                                 Amp                                                                              Promega <br />   pBAD/His B                            araC&#226;&#171;&#185;; ParaBAD                                                                   Invitrogen <br />   pEC-lpxC1                             lpxC E. coli cloned pDN19                                              This work <br />   pPA-lpxC1                             lpxC P. aeruginosa cloned pDN19                                        This work <br />   pEC-lpxC2                             lpxC E. coli cloned pUCP30T                                            This work <br />   pPA-lpxC2                             lpxC P. aeruginosa cloned pUCP30T                                      This work <br />   pPW101                                Derived pSP72; oriT &#226;&#171;&#185;                                                       This work <br />   pBEM10                                Derived pPW101; recombination P. aeruginosa chromosome                 This work <br />                                            results placing lpxC control ParaBAD; Amp; Tet <br />  a <br />      Antibiotic resistance markers: Amp, ampicillin; Gm, gentamicin; Km, kanamycin; Tet, tetracycline. <br />  <br />  <br />  <br />  <br /> equally active purified LpxC                         resistant LpxC inhibitor temperature (Fig. 4). <br /> LpxC bacterial extract. However, P. aeruginosa, puri-                      This indicated relative insensitivity P. aerugi- <br /> fication LpxC increased susceptibility L-161,240.                         nosa enzyme inhibitor sufficient confer resistance <br />    These observations indicate P. aeruginosa LpxC                      growth inhibition. <br /> resistant L-161,240 LpxC E. coli                             We constructed mutant P. aeruginosa native <br /> reducing effect intrinsic resistance mechanisms does                     promoter lpxC replaced inducible promoter (Fig. <br /> render P. aeruginosa susceptible growth inhibition                            3). Modification E. coli araBAD promoter render it <br /> L-161,240. These data allow determine relative                     tightly regulated P. aeruginosa described Materials and <br /> contributions aspects P. aeruginosa&#226;&#8364;&#8482;s resistance                    Methods. The resulting P. aeruginosa mutant, designated <br /> to compound. In order assess effect L-161,240                      PAO1-PBAD-lpxC, fully capable growth pres- <br /> P. aeruginosa LpxC bacterial cell known allow                       ence arabinose did grow absence this <br /> entry compound, use E. coli construct                       inducer. This confirmed P. aeruginosa similar E. coli <br /> which growth 42&#194;&#176;C dependent presence a <br />                                                                                     contains functional copy lpxC that <br /> functional lpxC gene E. coli P. aeruginosa. The <br />                                                                                     activity essential growth. PAO1-PBAD-lpxC was <br /> chromosomal lpxC gene E. coli strain JBK-1/pKD6 been <br />                                                                                     transformed plasmid containing P. aeruginosa <br /> inactivated; wild-type copy E. coli lpxC provided the <br />                                                                                     lpxC E. coli lpxC (pPA-lpxC2 pEC-lpxC2, respectively). <br /> temperature-sensitive replicon pKD6, confers am- <br />                                                                                     The transformants incubated various concentra- <br /> picillin resistance (34). Since lpxC essential growth, this <br />                                                                                     tions LpxC inhibitor. In absence arabinose, P. <br /> strain viable 42&#194;&#176;C functional copy the <br /> temperature-sensitive replicon. We transformed JBK-1/pKD6                           aeruginosa strains expressing E. coli LpxC nearly as <br /> with low-copy, non-temperature-sensitive replicon carrying                        susceptible inhibitor E. coli DH5&#226;?&#163; strains, while <br /> wild-type lpxC E. coli P. aeruginosa. Selection                  transformants lpxC gene P. aeruginosa were <br /> tetracycline resistance 42&#194;&#176;C yielded ampicillin-susceptible                      resistant, parent P. aeruginosa strain PAO1 (Fig. 5). <br /> transformants E. coli P. aeruginosa con-                     When experiment carried presence 0.2% <br /> struct, designated JBK-1/pEClpxC1 JBK-1/pPAlpxC1, re-                           arabinose induce expression chromosomal lpxC gene, <br /> spectively. This result indicated lpxC P. aeruginosa                      transformant susceptible L-161,240 (Fig. 5). <br /> could expressed E. coli background capable                        This confirmed P. aeruginosa lpxC enzyme itself the <br /> of substituting inactivated chromosomal copy.                               primary factor resistance P. aeruginosa L-161,240. <br />    When incubated 42&#194;&#176;, JBK-1/pEC-lpxC1 failed grow                            The intrinsic resistance mechanisms P. aeruginosa, the <br /> in presence LpxC inhibitor L-161,240, ex-                             form membrane impermeability inhibitor efflux, play a <br /> pected. However, isogenic strain JBK-1/pPA-lpxC1                            relatively minor role. <br />  2182      MDLULI ET AL.                                                                                         ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 3. Construction PAO1-PBAD-lpxC, P. aeruginosa mutant lpxC controlled araBAD promoter. (A) The promoter <br /> replacement vector pBEM10 derived pPW101 PCR products 5&#226;&#172;&#732; region P. aeruginosa lpxC described Materials Methods. <br /> (B) Promoter replacement mutagenesis. Transformation pBEM10 P. aeruginosa removed native lpxC promoter replaced tightly <br /> regulated araBAD promoter just upstream chromosomal copy lpxC. lpxC expression occurs presence arabinose. <br />  <br />  <br />  <br />                              DISCUSSION                                         construct active lpxC gene E. coli <br />                                                                                 complementary construct growth E. coli <br />    Our data indicate LpxC P. aeruginosa refractory <br />                                                                                 depended lpxC gene P. aeruginosa. Thus, is <br /> to inhibition compound L-161,240 concentrations 10-fold <br />                                                                                 clearly difference enzymes, differences spe- <br /> higher completely inhibit LpxC E. coli. <br />                                                                                 cies membrane structure specificity efflux pumps, that <br /> This demonstrated in vitro assays enzyme ac- <br />                                                                                 primary mediator differential susceptibilities the <br /> tivity bacterial growth experiments using P. aeruginosa <br />                                                                                 bacterial species inhibitor. <br />                                                                                    There remains, however, cytoplasmic process P. <br />                                                                                 aeruginosa reduces activity L-161,240 (Table <br />   TABLE 2. L-161,240 inhibition LpxC enzyme either <br />                                                                                 2). Our experiments reveal existence of, char- <br />                   purified crude sources a <br />                                                                                 acterize, process. The inhibitor sequestered or <br />                         IC50 (&#226;?&#174;M) L-161,240 against:         IC50 ratio      inactivated P. aeruginosa. Alternatively, P. aeruginosa <br />  Enzyme source                              P. aeruginosa     (P. aeruginosa/   LpxC itself complexed molecules a <br />                         E. coli DH5&#226;?&#163;                              E. coli) <br />                                                 PAO1                            way limit access inhibitor enzyme. Nonethe- <br /> Crude extracts         0.037 &#226;&#171;&#190; 0.002        1.40 &#226;&#171;&#190; 0.07            37.8         less, different potencies L-161,240 en- <br /> Purified enzyme        0.023 &#226;&#171;&#190; 0.003        0.22 &#226;&#171;&#190; 0.03             9.6         zymes appear single factor contributes to <br />    a <br />      IC50, 50% inhibitory concentration. Data means &#226;&#171;&#190; standard deviations   lack activity inhibitor P. aeruginosa. <br /> from replicate assays.                                                       Our current knowledge structure biochemistry of <br />  VOL. 50, 2006                                                                                                      P. AERUGINOSA LpxC                2183 <br />  <br />  <br />  <br />  <br />   FIG. 4. The lpxC gene P. aeruginosa confers L-161,240 resis-        FIG. 5. Treatment promoter replacement mutant P. aerugi- <br /> tance E. coli lacking functional endogenous lpxC gene. The strain   nosa various concentrations LpxC inhibitor L-161,240 in <br /> JBK-1/pKD6 contains chromosomal lpxC gene disrupted           absence (A) presence (B) arabinose. Regardless pres- <br /> kan element wild-type copy lpxC temperature-sensitive    ence arabinose, wild-type E. coli DH5&#226;?&#163; susceptible inhibitor <br /> replicon pKD6. In strains JBK-1/pEC-lpxC1 JBK-1/pPA-lpxC1,           wild-type (wt) PAO1 resistant. PAO1-PBAD-lpxC trans- <br /> pKD6 replaced plasmids (not temperature sensitive) bear-     formed plasmid carrying lpxC gene E. coli <br /> ing wild-type lpxC E. coli P. aeruginosa, respectively. The     (pEC-lpxC2) P. aeruginosa (pPA-lpxC2). In absence arabi- <br /> bacterial strains incubated LB medium containing 10 &#226;?&#174;g/ml        nose (A), chromosomal lpxC gene expressed strain <br /> L-161,240 drug, bacterial growth determined moni-       carrying pEC-lpxC2 susceptible inhibitor strain <br /> toring change absorbance 600 nm.                               carrying pPA-lpxC2 resistant. In presence 0.2% arabinose (B), <br />                                                                          chromosomal lpxC gene PAO1-PBAD-lpxC expressed, con- <br />                                                                          ferring resistance LpxC inhibitor affected the <br /> the LpxC enzymes limited species. The relatively          plasmid-borne lpxC. <br /> high degree primary sequence similarity E. coli <br /> and P. aeruginosa enzymes (Fig. 2) demonstrates inade- <br />                                                                          P. aeruginosa gram-negative pathogens, de- <br /> quacy primary sequences accurate prediction func- <br />                                                                          tails presented (1, 16, 19). <br /> tional similarity proteins. Subtle differences, as <br /> the nonconserved histidines E. coli sequence,                                       ACKNOWLEDGMENTS <br /> have significant consequences overall fold binding <br />                                                                             This work supported PathoGenesis Corporation, Seattle, WA <br /> properties. The unexpected differences spe-              (later merged Chiron Corporation). C.R.H.R. supported by <br /> cies&#226;&#8364;&#8482; LpxC enzymes highlight importance protein struc-            NIH grant GM-51310. <br /> ture drug design. Nuclear magnetic resonance (6) X-ray                                              REFERENCES <br /> (37) structures LpxC Aquifex aeolicus de-               1. Andersen, N., J. Bowman, A. Erwin, E. Harwood, T. Kline, K. Mdluli, S. Ng, <br /> termined, zinc sites A. aeolicus P. aeruginosa            K. Pfister, R. Shawar. November 2004. Antibacterial agents. Patent <br /> enzymes studied using extended X-ray absorption                    application WO 2004/062601 A2. <br />                                                                           2. Angus, B. L., A. M. Carey, D. A. Caron, A. M. B. Kropinski, R. E. W. <br /> fine-structure spectroscopy (21). Structural studies LpxC                 Hancock. 1982. Outer membrane permeability Pseudomonas aeruginosa: <br /> from species addition A. aeolicus provide struc-               comparison wild-type antibiotic-supersusceptible mutant. Anti- <br />                                                                              microb. Agents Chemother. 21:299&#226;&#8364;&#8220;309. <br /> tural rationale differential susceptibilities spe-       3. Beall, B., J. Lutkenhaus. 1987. Sequence analysis, transcriptional orga- <br /> cies given inhibitor facilitate design               nization, insertional mutagenesis envA gene Escherichia coli. J. <br /> inhibitors broader specificity previously                 Bacteriol. 169:5408&#226;&#8364;&#8220;5415. <br />                                                                           4. Chen, M. H., M. G. Steiner, S. E. Laszlo, A. A. Patchett, M. S. Anderson, <br /> been possible.                                                               S. A. Hyland, H. R. Onishi, L. L. Silver, C. R. H. Raetz. 1999. Carbo- <br />    Our data suggest strategies identify broad-spectrum               hydroxamido-oxazolidines: antibacterial agents target lipid A biosynthe- <br /> LpxC inhibitors challenging ac-                  sis. Bioorg. Med. Chem. Lett. 9:313&#226;&#8364;&#8220;318. <br />                                                                           5. Clements, J. M., F. Coignard, I. Johnson, S. Chandler, S. Palan, A. Waller, <br /> count structural differences LpxC enzymes differ-                J. Wijkmans, M. G. Hunter. 2002. Antibacterial activities charac- <br /> ent gram-negative bacterial species. The finding nonen-                 terization novel inhibitors LpxC. Antimicrob. Agents Chemother. 46: <br />                                                                              1793&#226;&#8364;&#8220;1799. <br /> zyme components cell extracts affect activity inhibitors        6. Coggins, B. E., X. Li, A. L. McClerren, O. Hindsgaul, C. R. H. Raetz, P. <br /> within bacterial cell adds additional level complexity.            Zhou. 2003. Structure LpxC deacetylase bound substrate-analog <br /> Target-based antibiotic discovery successful               inhibitor. Nat. Struct. Biol. 10:645&#226;&#8364;&#8220;651. <br />                                                                           7. Coggins, B. E., A. L. McClerren, L. Jiang, X. Li, J. Rudolph, O. Hindsgaul, <br /> it possible evaluate separately factors               C. R. Raetz, P. Zhou. 2005. Refined solution structure LpxC-TU- <br /> contribute bacterial growth inhibition: penetration                514 complex pKa analysis active site histidine: insights the <br /> cell wall, resistance efflux, inactivation sequestration            mechanism inhibitor design. Biochemistry 44:1114&#226;&#8364;&#8220;1126. <br />                                                                           8. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu- <br /> intracytoplasmic components, potency target.                  lation, modulation, high-level expression vectors containing arabi- <br /> The use multiple molecular biochemical approaches                     nose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130. <br />                                                                           9. Hickey, M. J., T. M. Arain, R. M. Shawar, D. J. Humble, M. H. Langhorne, <br /> allowed evaluate LpxC inhibitor                  J. N. Morgenroth, C. K. Stover. 1996. Luciferase vivo expression <br /> L-161,240 case, potency                  technology: use recombinant mycobacterial reporter strains evaluate <br /> target primary factor limiting activity P. aeruginosa.           antimycobacterial activity mice. Antimicrob. Agents Chemother. 40:400&#226;&#8364;&#8220; <br />                                                                              407. <br /> This conclusion critical discovery potent small-           10. Holloway, B. W. 1955. Genetic recombination Pseudomonas aeruginosa. <br /> molecule LpxC inhibitors antibacterial activity                  J. Gen. Microbiol. 13:572&#226;&#8364;&#8220;581. <br />  2184       MDLULI ET AL.                                                                                                      ANTIMICROB. AGENTS CHEMOTHER. <br />  <br /> 11. Hyland, S. A., S. S. Eveland, M. S. Anderson. 1997. Cloning, expression,     26. Olsen, R. H., G. DeBusscher, W. R. McCombie. 1982. Development of <br />     purification UDP-3-O-acyl-GlcNAc deacetylase Pseudomonas                 broad-host-range vectors gene banks: self-cloning Pseudomonas <br />     aeruginosa: metalloamidase lipid A biosynthesis pathway. J. Bacte-          aeruginosa PAO chromosome. J. Bacteriol. 150:60&#226;&#8364;&#8220;69. <br />     riol. 179:2029&#226;&#8364;&#8220;2037.                                                             27. Onishi, H. R., B. A. Pelak, L. S. Gerckens, L. L. Silver, F. M. Kahan, M. H. <br /> 12. Jackman, J. E., C. A. Fierke, L. N. Tumey, M. Pirrung, T. Uchiyama, S. H.            Chen, A. A. Patchett, S. M. Galloway, S. A. Hyland, M. S. Anderson, and <br />     Tahir, O. Hindsgaul, C. R. H. Raetz. 2000. Antibacterial agents             C. R. H. Raetz. 1996. Antibacterial agents inhibit lipid A biosynthesis. <br />     target lipid A biosynthesis gram-negative bacteria. Inhibition diverse         Science 274:980&#226;&#8364;&#8220;982. <br />     UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylases sub-          28. Pirrung, M. C., L. N. Tumey, A. L. McClerren, C. R. H. Raetz. 2003. <br />     strate analogs containing zinc binding motifs. J. Biol. Chem. 275:11002&#226;&#8364;&#8220;             High-throughput catch-and-release synthesis oxazoline hydroxamates. <br />     11009.                                                                               Structure-activity relationships novel inhibitors Escherichia coli LpxC: <br /> 13. Jackman, J. E., C. R. H. Raetz, C. A. Fierke. 1999. UDP-3-O-(R-3-                vitro enzyme inhibition antibacterial properties. J. Am. Chem. Soc. <br />     hydroxymyristoyl)-N-acetylglucosamine deacetylase Escherichia coli           125:1575&#226;&#8364;&#8220;1586. <br />     zinc metalloenzyme. Biochemistry 38:1902&#226;&#8364;&#8220;1911.                                   29. Pirrung, M. C., L. N. Tumey, C. R. H. Raetz, J. E. Jackman, K. Snehalatha, <br /> 14. Jackman, J. E., C. R. H. Raetz, C. A. Fierke. 2001. Site-directed mu-            A. L. McClerren, C. A. Fierke, S. L. Gantt, K. M. Rusche. 2002. Inhi- <br />     tagenesis bacterial metalloamidase UDP-(3-O-acyl)-N-acetylglu-                bition antibacterial target UDP-(3-O-acyl)-N-acetylglucosamine <br />     cosamine deacetylase (LpxC). Identification zinc binding site. Bio-           deacetylase (LpxC): isoxazoline zinc amidase inhibitors bearing diverse <br />     chemistry 40:514&#226;&#8364;&#8220;523.                                                                metal binding groups. J. Med. Chem. 45:4359&#226;&#8364;&#8220;4370. <br /> 15. Jacobs, M. A., A. Alwood, I. Thaipisuttikul, D. Spencer, E. Haugen, S. Ernst,    30. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />     O. Will, R. Kaul, C. Raymond, R. Levy, L. Chun-Rong, D. Guenthner, D.                laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring <br />     Bovee, M. V. Olson, C. Manoil. 2003. Comprehensive transposon mutant             Harbor, N.Y. <br />     library Pseudomonas aeruginosa. Proc. Natl. Acad. Sci. USA 100:14339&#226;&#8364;&#8220;         31. Schleif, R. 1992. DNA looping. Annu. Rev. Biochem. 61:199&#226;&#8364;&#8220;223. <br />     14344. <br />                                                                                      32. Schweizer, H. P. 1998. Intrinsic resistance inhibitors fatty acid biosyn- <br /> 16. Kline, T., N. H. Andersen, E. A. Harwood, J. Bowman, A. Malanda, S. <br />                                                                                          thesis Pseudomonas aeruginosa efflux: application novel <br />     Endsley, A. L. Erwin, M. Doyle, S. Fong, A. L. Harris, B. Mendelsohn, K. <br />                                                                                          technique generation unmarked chromosomal mutations study <br />     Mdluli, C. R. H. Raetz, C. K. Stover, P. R. Witte, A. Yabannavar, S. Zhu. <br />                                                                                          efflux systems. Antimicrob. Agents Chemother. 42:394&#226;&#8364;&#8220;398. <br />     2002. Potent, novel vitro inhibitors Pseudomonas aeruginosa <br />                                                                                      33. Schweizer, H. P., T. T. Hoang. 1995. An improved gene <br />     deacetylase LpxC. J. Med. Chem. 45:3112&#226;&#8364;&#8220;3129. <br />                                                                                          replacement xylE fusion analysis Pseudomonas aeruginosa. Gene 158: <br /> 17. Lee, N. 1980. Molecular aspects ara regulation, p. 389&#226;&#8364;&#8220;410. In J. H. Miller <br />                                                                                          15&#226;&#8364;&#8220;22. <br />     W. S. Reznikoff (ed.), The operon. Cold Spring Harbor Laboratory, <br />     Cold Spring Harbor, N.Y.                                                         34. Sorensen, P. G., J. Lutkenhaus, K. Young, S. S. Eveland, M. S. Anderson, <br /> 18. Li, X. Z., D. M. Livermore, H. Nikaido. 1994. Role efflux pump(s)          C. R. H. Raetz. 1996. Regulation UDP-3-O-[R-3-hydroxymyristoyl]- <br />     intrinsic resistance Pseudomonas aeruginosa: resistance tetracycline,          N-acetylglucosamine deacetylase Escherichia coli. The second enzymatic <br />     chloramphenicol, norfloxacin. Antimicrob. Agents Chemother. 38:1732&#226;&#8364;&#8220;             step lipid A biosynthesis. J. Biol. Chem. 271:25898&#226;&#8364;&#8220;25905. <br />     1741.                                                                            35. Stover, C. K., X. Q. Pham, A. L. Erwin, S. D. Mizoguchi, P. Warrener, M. J. <br /> 19. McClerren, A. L., S. Endsley, J. L. Bowman, N. H. Andersen, Z. Guan, J.              Hickey, F. S. Brinkman, W. O. Hufnagle, D. J. Kowalik, M. Lagrou, R. L. <br />     Rudolph, C. R. Raetz. 2005. A slow, tight-binding inhibitor               Garber, L. Goltry, E. Tolentino, S. Westbrock-Wadman, Y. Yuan, L. L. <br />     zinc-dependent deacetylase LpxC lipid A biosynthesis antibiotic              Brody, S. N. Coulter, K. R. Folger, A. Kas, K. Larbig, R. Lim, K. Smith, D. <br />     activity comparable ciprofloxacin. Biochemistry 44:16574&#226;&#8364;&#8220;16583.                   Spencer, G. K. Wong, Z. Wu, I. T. Paulsen, J. Reizer, M. H. Saier, R. E. <br /> 20. McClerren, A. L., P. Zhou, Z. Guan, C. R. Raetz, J. Rudolph. 2005.               Hancock, S. Lory, M. V. Olson. 2000. Complete genome sequence of <br />     Kinetic analysis zinc-dependent deacetylase lipid A biosynthetic       Pseudomonas aeruginosa PA01, opportunistic pathogen. Nature 406:959&#226;&#8364;&#8220; <br />     pathway. Biochemistry 44:1106&#226;&#8364;&#8220;1113.                                                  964. <br /> 21. McClure, C. P., K. M. Rusche, K. Peariso, J. E. Jackman, C. A. Fierke,       36. Studier, F. W., A. H. Rosenberg, J. J. Dunn, J. W. Dubendorff. 1990. Use <br />     J. E. Penner-Hahn. 2003. EXAFS studies zinc sites UDP-(3-O-                T7 RNA polymerase direct expression cloned genes. Methods En- <br />     acyl)-N-acetylglucosamine deacetylase (LpxC). J. Inorg. Biochem. 94:78&#226;&#8364;&#8220;85.           zymol. 185:60&#226;&#8364;&#8220;89. <br /> 22. Miyada, C. G., L. Stoltzfus, G. Wilcox. 1984. Regulation araC         37. Whittington, D. A., K. M. Rusche, H. Shin, C. A. Fierke, D. W. Christianson. <br />     gene Escherichia coli: catabolite repression, autoregulation, effect       2003. Crystal structure LpxC, zinc-dependent  <br /> </body></html>