<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Molecular Biology                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                          Open Access <br /> Viable nonsense mutants essential gene SUP45 of <br /> Saccharomyces cerevisiae <br /> Svetlana E Moskalenko1,2, Svetlana V Chabelskaya1,2, Sergei G Inge- <br /> Vechtomov2, Michel Philippe1 Galina A Zhouravleva*1,2 <br />  <br /> Address: 1Universit&#195;&#169; Rennes 1, CNRS UMR 6061, IFR 97, 2 av. Pr. L&#195;&#169;on Bernard 35043 Rennes Cedex, France 2Department Genetics, St <br /> Petersburg State University, Universitetskaya emb. 7/1, 199034, St Petersburg, Russia <br /> Email: Svetlana E Moskalenko - smoskalenko@hotmail.com; Svetlana V Chabelskaya - sveta@btc.bio.pu.ru; Sergei G Inge- <br /> Vechtomov - inge@mailru.com; Michel Philippe - michel.philippe@univ-rennes1.fr; Galina A Zhouravleva* - galina@btc.bio.pu.ru <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 10 February 2003                                                   Received: 29 November 2002 <br />                                                                               Accepted: 10 February 2003 <br /> BMC Molecular Biology 2003, 4:2   doi:10.1186/1471-2199-4-2 <br /> This article available <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2199/4/2 <br /> &#194;&#169; 2003 Moskalenko et al; licensee BioMed Central Ltd. This Open Access article: verbatim copying redistribution article permitted in <br /> all media purpose, provided notice preserved article&apos;s original URL. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Termination protein synthesis eukaryotes involves polypeptide <br />                  release factors (eRFs) &#226;&#8364;&#8220; eRF1 eRF3. The highly conserved translation termination factor eRF1 <br />                  Saccharomyces cerevisiae encoded essential gene SUP45. <br />                  Results: We isolated sup45-n (n nonsense) mutations cause nonsense <br />                  substitutions following amino acid positions eRF1: Y53 &#226;&#8224;&#8217; UAA, E266 &#226;&#8224;&#8217; UAA, L283 &#226;&#8224;&#8217; <br />                  UAA, L317 &#226;&#8224;&#8217; UGA, E385 &#226;&#8224;&#8217; UAA. We full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> eRF1 protein present mutants, <br />                  decreased amounts. All mutations situated weak termination context. All these <br />                  sup45-n mutations viable different genetic backgrounds, viability increases <br />                  growth absence wild-type allele. Any sup45-n mutations result temperature <br />                  sensitivity (37&#194;&#176;C). Most sup45-n mutations lead decreased spore viability spores <br />                  bearing sup45-n mutations characterized limited budding germination leading <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                  formation microcolonies 4&#226;&#8364;&#8220;20 cells. <br />                  Conclusions: Nonsense mutations essential gene SUP45 isolated absence of <br />                  tRNA nonsense suppressors. <br />  <br />  <br />  <br />  <br /> Background                                                                     Release factor eRF1 recognizes stop codons [5], in <br /> Termination protein synthesis occurs ribos-                        contrast prokaryotes RF1 catalyses translation <br /> ome elongation machinery encounters in-frame termi-                         termination UAG UAA codons, RF2 UGA <br /> nation (stop) codon, UAG, UGA UAA,                            UAA codons [6]. All proteins encoded by <br /> mRNA. This stop codon located A-site ribos-                      essential genes: prfA (for RF1), prfB (for RF2) bacteria <br /> ome recognized release factor (RF1/RF2 prokary-                     [7-9] SUP45 (for eRF1) S. cerevisiae [10,11]. <br /> otes eRF1 eukaryotes), triggers release                    Although sequences prokaryotic eukaryotic <br /> nascent peptide ribosome. Termination effi-                           release factors differ significantly [see [12] review], a <br /> ciency enhanced GTPase release factor, RF3                        &quot;tRNA-mimicry&quot; model generalizing mechanisms of <br /> prokaryotes eRF3 eukaryotes [reviewed [1-4]].                        prokaryotic eukaryotic translation termination, was <br />                                                                                proposed [13]. Peptide determinants RF1 RF2 <br />  <br />  <br />                                                                                                                                           Page 1 13 <br />                                                                                                                     (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                      http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> involved specificity stop codon recognition            tants obtained. All sup45 mutant alleles together <br /> been identified: cases tripeptide (PAT RF1              wild-type SUP45 allele parent strain 1B- <br /> and SPF RF2) located homologous region              D1606 cloned using PCR. The DNA sequence for <br /> both proteins [14].                                                   cloned alleles determined. <br />  <br /> Recently crystal structures human eRF1 bacte-              The sequencing parent strain 1B-D1606 revealed 6 <br /> rial RF2 determined different               nucleotide substitutions compared published <br /> [15,17]. However, associated ribosome RF2,                sequence [10]: A69G, G519T, T687C, T702A, G927A, <br /> conformation modified mimics tRNA molecule                   T1008C. All these, T702, those <br /> [16,18]. Thus eRF1 RF2 comparable                   described earlier [10] mutant alleles obviously <br /> structures compatible tRNA mim-                different origin regarding wild-type strain. All <br /> icry model proposed previously [13]. The eRF1 com-              nucleotide substitutions localized codon <br /> posed domains. Domain 3 corresponds C-                position change amino acid residues. The <br /> terminal eRF1 necessary interaction           reason differences origin the <br /> with eRF3 discrepancies                strain Petergoff Breeding Stocks (XII race S. cerevi- <br /> precise localization region eRF1 interacts             siae) independent origin [31] compared the <br /> with eRF3 [19-22]. Domain 2 responsible pepti-             strains used laboratories [32]. <br /> dyl transferase hydrolytic activity includes GGQ <br /> motif highly conserved evolution                Comparison mutant SUP45 DNA sequences with <br /> [23]. Mutations GGQ (Gly residues) dominant-neg-               parent strain revealed cases a <br /> ative vitro [23] lethal vivo S. cerevisiae cells [15].   single-<span id='am-242' about='xsp:base' typeof='owl:Thing'>base</span> changed coding sequence. Surprisingly, <br /> The function domain 1 remains unknown                  16 mutations 5 represented nonsense mutations, <br /> was proposed domain 2, form               resulting mutations sense codon ochre <br /> a functionally active &quot;core&quot; domain [24]. The role N-              (TAA) (four mutations) (TAG) nonsense codon (one <br /> terminal domain eRF1 stop codon recognition                 <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span>) (Figure 1). The 9 missense muta- <br /> been proposed [15] supported mutational                        tions described elsewhere. Strains bearing <br /> approach [25] crosslinking experiments [26].                      sup45-101, sup45-102, sup45-104, sup45-105, sup45-107 <br />                                                                       contain, respectively, following base substitu- <br /> In eukaryotes eRF1 interacts eRF3 vivo [27,28].               tions: G796T, T159A, T848A, G1153T T950G, which <br /> This interaction mediated C-terminus eRF1                lead nonsense mutations, characterized. <br /> [20,21]. The deletion 19 C-terminal amino acids <br /> abolishes interaction eRF3, causes enhance-               Nonsense mutations SUP45 gene lead to <br /> ment nonsense suppression, does destroy via-               readthrough mutated stop codon <br /> bility. In contrast, eRF1 32 C-terminal amino acid             All mutations defined omnipotent suppres- <br /> deletion unable support viability [20]. Also             sors testing histidine, lysine, adenine tryp- <br /> been shown nonsense mutations (UAA) SUP45                 tophan auxotrophy; his7-1 (UAA), lys9-A21 (UAA,) ade1- <br /> gene lead lethality absence SUQ5 suppressor              14 (UGA) trp1-289 (UAG) mutation present in <br /> [29]. However, viable nonsense mutant iso-                 parent strain (Figure 2A). <br /> lated prfB gene S. typhimurium encoding RF2 <br /> release factor. It proposed mutation               To determine level nonsense suppression dif- <br /> reduces cellular RF2 leading inefficient             ferent SUP45 mutants vector based assay for <br /> termination translation autosuppression [30].                  quantification nonsense codon readthroughin vivo was <br />                                                                       employed [33]. &#206;&#178;-galactosidase activities measurements <br /> In present work isolated spontaneous                 mutants consistent results obtained for <br /> nonsense mutations SUP45 gene confer viabil-              auxotrophy (Figure 2A): parent strain does possess <br /> ity different genetic backgrounds. These mutations                 suppressor activity. In omnipotent mutants tested, <br /> result decreased level full-length eRF1, thermosen-           nonsense suppression levels stop codons high <br /> sitivity lethality meiotic segregation hetero-              (between 8- 28% UAA; 10- 25% for <br /> zygous diploids.                                                      UAG, 22- 42% UGA) (Figure 2B). <br />  <br /> Results                                                               Since C-terminal region eRF1 essential cell via- <br /> Isolation nonsense mutations SUP45 gene                     bility [10,11], full-length eRF1 present <br /> After selection histidine (his7-1 (UAA)) &#226;&#8364;&#8220; lysine (lys9-          strains bearing sup45 nonsense mutations (which <br /> A21 (UAA)) prototrophy, sixteen SUP45 mutants                     referred sup45-n, n equals nonsense), <br /> eighty-four SUP35 mutants 400 His+ Lys+ rever-                  probably readthrough stop codon. An affinity- <br />  <br />  <br />                                                                                                                           Page 2 13 <br />                                                                                                     (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br />  <br />                                                                suppression <br />                                                                Figure <br />                                                                Nonsense  2 mutations SUP45 gene lead omnipotent <br />                                                                Nonsense mutations SUP45 gene lead to <br />                                                                omnipotent suppression. A. Plate assays showing the <br />                                                                growth yeast strains bearing nonsense mutations the <br />                                                                SUP45 gene (101-1B-D1606, 102-1B-D1606, 104-1B-D1606, <br /> Figure <br /> present <br /> Schematic1work <br />             representation sup45-n mutations isolated    105-1B-D1606, 107-1B-D1606) synthetic medium <br /> Schematic representation sup45-n mutations iso-             histidine (SC-His), lysine (SC-Lys), adenine (SC-Ade) <br /> lated present work. Position mutations indi-      tryptophan (SC-Trp) 25&#194;&#176;C compared parent <br /> cated SUP45 gene. The numbers SUP45        strain 1B-D1606. The types nonsense mutations parent <br /> gene correspond amino acid positions codons           strain denoted media indications. SUP45 mutant <br /> were changed premature stop codons. Numbers          alleles indicated. Ten serial dilutions yeast suspension <br /> right arrows correspond size truncated eRF1       density used. Five independent clones were <br /> protein.                                                       tested, representative results shown. B. Determination <br />                                                                level nonsense suppression sup45-n mutants. <br />                                                                UAA, UAG UGA suppression levels quantified by <br />                                                                measuring &#206;&#178;-galactosidase levels strains 1B-D1606 <br /> purified anti-eRF1 antibody used Western blot <br />                                                                (wild type) control mutants transformed plas- <br /> analysis examine eRF1 levels SUP45 mutants. As           mids pUKC817, pUKC818 pUKC819 bearing specific <br /> shown Figure 2C, considerable         stop codon (TAA, TAG TGA, respectively) frame that <br /> full-length eRF1 sup45-n strains. Comparison         precedes lacZ gene. The efficiency nonsense codon <br /> wild-type strain revealed significant decrease        readthrough (%) quantified ratio &#206;&#178;-galactosidase <br /> amount eRF1 protein. Ribosome fractions isolated            activity cells harboring lacZ stop codon cells <br /> from sup45-104, sup45-105, sup45-107 contained trun-         stop codon in-frame lacZ gene in <br /> cated protein molecular mass consistent          pUKC815 plasmids. The strains Fig. 2A used. <br /> predicted (31.5, 43.0 35.2 kDa, respectively)          Results means separate experiments. C. <br /> (Figure 2C).                                                   Western blot showing synthesis full-length eRF1 (49 <br />                                                                kDa) truncated eRF1 proteins predicted molecular <br />                                                                mass 43.0 kDa, 35.2 kDa 31.5 kDa mutants 105, 107 <br /> Western blot analysis did reveal presence trun- <br />                                                                104, respectively. (*) Indicates non-specific band. A <br /> cated protein ribosome fractions, lysates      search yeast proteome N-terminal peptide <br /> prepared sup45-101 strain (Figure 2C data         sequence used antibody production, revealed <br /> shown). The inability detect truncated protein         presence protein (methionyl-tRNA synthetase, <br /> with predicted molecular mass 29.6 kDa           <span id='am-2' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> CAA24627) containing this <br /> explained proteolytic sensitivity possibly   sequence expected molecular weight near 85 <br /> the absence ribosome-binding site. The small trun-      kDa. Cultures strains (A) grown to <br /> cated protein (5.9 kDa) sup45-102 strain       mid-log phase medium selective plasmid ribosome <br /> detected. Relative level eRF1 wild-type strain    fractions prepared. The sample <br /> 1B-D1606 (set 1), full-length eRF1         loaded lane. Immunoblot analysis performed <br /> 101-D1606, 102-D1606, 104-D1606, 105-D1606                 using polyclonal antibodies directed N-terminal <br />                                                                eRF1. <br /> 107-D1606 strains 0.32, 0.08, 0.13, 0.14 0.17, <br /> respectively. The highest level eRF1 sup45-101 strain <br />  <br />  <br />                                                                                                                      Page 3 13 <br />                                                                                                (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                              http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> corresponds lowest level suppression meas-        pressor mutation parent strain plasmid, shuffle <br /> ured &#206;&#178;-galactosidase assay (Figure 2B). These results       analysis performed derivative S288C [35] <br /> suggest small eRF1 required       entire genomic sequence does contain muta- <br /> cell viability.                                                tions tRNA genes [36]. A haploid strain 1A-Y23282 <br />                                                                bearing SUP45 disruption [URA3 SUP45] plasmid <br /> The effect reducing eRF1 cell viability evaluated    pRS316/SUP45 transformed [LEU2 sup45-n] <br /> next. All sup45-n mutations (but missense muta-        plasmids resulting transformants plated on <br /> tions, data shown) lead temperature-sensitive           5-FOA medium. All transformants able grow in <br /> lethality (Figure 3). Previously, analogous <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span>      presence 5-FOA (Figure 4D). <br /> caused opal supK584 mutation RF2 gene S. <br /> typhimurium explained reduced level            The comparison data presented Figures 4A 4D <br /> mature RF2 protein higher temperature [30]. To deter-       shows efficiency plasmid shuffle varies sig- <br /> mine case yeast strains carry-   nificantly mutations sup45-105 <br /> ing sup45-n alleles, ribosomes isolated wild-        highest sup45-101. To compare efficiency <br /> type strain 1B-D1606 104-1B-D1606 strain          plasmid shuffle quantitatively maintenance plas- <br /> two hours incubation 37&#194;&#176;C. Western blot         mid wild-type SUP45 tested. The haploid yeast <br /> analysis did reveal significant difference      strain 1A-D1628 carrying SUP45 gene centromeric <br /> amount full-length eRF1 protein non-permissive       plasmid pRS315 transformed pRS316 plasmids <br /> temperature compared permissive tempera-           bearing different sup45-n mutations. Cotransformants <br /> ture mutant wild-type strains (data     plated 200 cells Petri dish YPD medium, <br /> not shown).                                                    grown 25&#194;&#176;C days replica plated SC- <br />                                                                Ura SC-Leu media. 200 500 colonies tested <br /> Do sup45-n mutations lead lethality different genetic    3 independent transformants. All cotransformants <br /> backgrounds?                                                   able lose [SUP45 LEU2] [sup45-n URA3] <br /> The viability strains bearing nonsense mutations     plasmid different efficiency. The plasmid pRS316 <br /> essential SUP45 gene explained ways. It      bearing mutant sup45-n alleles lost 15&#226;&#8364;&#8220;20% of <br /> is possible parent strain itself contains weak tRNA   tested colonies (data shown). At time very <br /> suppressor. However, level readthrough          small number colonies lost pRS315 plasmid wild- <br /> strain, measured quantitative termination codon           type SUP45. (Table 1, column). These data con- <br /> readthrough assay, negligible (Figure 2B).                  sistent result plasmid shuffle 5-FOA (Fig. <br />                                                                4A 4D): frequency wild-type plas- <br /> To examine possibility plasmid shuffle analy-     mid loss (0.3%) obtained sup45-105 high- <br /> sis performed. The haploid yeast strain 1A-D1628       est (6.3%) sup45-101. <br /> transformed pRS315 plasmids bearing different <br /> sup45-n mutations. In strain lethal SUP45 disrup-     Ura+ Leu- colonies selected tested adenine <br /> tion complemented SUP45 gene URA3             tryptophan auxotrophy. In cases sup45-n muta- <br /> centromeric plasmid, pRS316. Transformants           tions led suppression ade1-14 trp1-289 (Figure <br /> subjected plasmid shuffle analysis [34] verify           4B). Cells bearing pRS316/sup45-n plasmid re-trans- <br /> whether strains containing sup45-n alleles lose      formed pRS315-SUP45 plasmid experiment <br /> the plasmid carrying wild-type gene. Leu+ Ura+ trans-      repeated (Table 1, second column). The results that <br /> formants plated 5-FOA medium select         growth presence mutant alleles SUP45 <br /> [URA3 SUP45] plasmid. Control cells transformed           ability lose plasmid wild-type allele of <br /> [LEU2 SUP45] plasmid grew 5-FOA. Cells transformed          SUP45 increased. To determine increase <br /> with pRS315/sup45-n plasmids grew 5-FOA                explained reversions sup45-n alleles to <br /> medium different efficiencies (Figure 4A).       wild-type allele shuffle experiments yeast, the <br /> Ura- Leu+ colonies selected tested adenine        corresponding plasmids prepared yeast trans- <br /> and tryptophan auxotrophy. In cases sup45-n muta-          formants sequenced. In tested cases presence of <br /> tions lead suppression ade1-14 trp1-289 (Figure      sup45-n mutation confirmed (data shown). <br /> 4B). Again, Western blot analysis showed all <br /> SUP45::HIS3 [sup45-n] strains full-length eRF1        Do nonsense mutations SUP45 gene lead to <br /> synthesized lesser extent compared SUP45::HIS3       lethality meiosis? <br /> [SUP45] strain (Figure 4C).                                    Previously shown sup45 nonsense <br />                                                                mutants, selected presence SUQ5, viable <br /> To confirm viability sup45-n mutations         absence SUQ5 tRNA suppressor [29]. In our <br /> could explained presence weak tRNA sup-       work mutants viable distinct yeast <br />  <br />  <br />                                                                                                                   Page 4 13 <br />                                                                                             (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br />  <br /> Figure <br /> tivity 3 mutations SUP45 gene lead thermosensi- <br /> Nonsense                                                            different <br />                                                                 Nonsense <br />                                                                 Figure <br />                                                                      4 mutations <br />                                                                               genetic backgrounds <br />                                                                                       SUP45 gene confer viability in <br /> Nonsense mutations SUP45 gene lead ther-              Nonsense mutations SUP45 gene confer the <br /> mosensitivity. Thermosensitivity strains bearing sup45-n     viability different genetic backgrounds. A. <br /> mutations. The strains (2A) tested       Strain 1A-D1628 bearing SUP45 disruption co-trans- <br /> rich medium (YPD) 37&#194;&#176;C.                                      formed pRS316/SUP45 pRS315/sup45-n plasmids. The <br />                                                                 cotransformants bearing pRS316/SUP45 vector pRS315 <br />                                                                 used negative control. The transformants <br />                                                                 assayed growth plating 5-FOA medium to <br /> strains contain SUQ5. Thus main         select URA3 plasmid pRS316/SUP45 carrying a <br />                                                                 wild-type copy SUP45. Ten serial dilutions yeast suspen- <br /> difference data previously published <br />                                                                 sions density used. Five independent trans- <br /> data [29] explained different approaches           formants combination tested. Representative <br /> selection mutants used. In addition, work, plas-     results shown. B. Plate assays showing growth of <br /> mid shuffle analysis used test viability sup45-   transformants 1A-D1628 bearing pRS316/SUP45 or <br /> n mutants analysis meiotic products used         pRS316/sup45-n plasmids synthetic medium without <br /> to characterize viability sup45-0 (0 ochre)   adenine (SC-Ade) tryptophan (SC-Trp) 25&#194;&#176;C. The <br /> mutants [29].                                                   strains tested way Fig. 2A. C. West- <br />                                                                 ern blot showing expression full-length eRF1 (49 kDa) <br /> To compare results ones Stansfield et al. [29]    truncated eRF1 proteins strains (B). D. <br /> we transformed diploid strain D1628 (a parent strain        Complementation SUP45 gene disruption strain <br />                                                                 13A-Y23282 sup45-n mutations. The transformants with <br /> 1A-D1628 strain used plasmid shuffle experiment) het- <br />                                                                 vector pRS315 used negative control. The same <br /> erozygous SUP45 disruption plasmids carrying           plasmids (A) used. <br /> sup45-n alleles performed tetrad analysis. Table 2 <br /> shows His+ Ura+ segregants obtained the <br /> case sup45-105 allele (where His+ marker for <br /> the SUP45 gene disruption Ura+ &#226;&#8364;&#8220; plasmid            using diploid strain Y23282, derivative of <br /> presence). This result explained loss       S288C (data shown). <br /> the corresponding plasmid stability plasmids <br /> in tested diploids practically (Table 2).     Microscopic examination lethal spores the <br /> For sup45-n alleles appearance His+ Ura+       control strain showed unable carry out <br /> segregants registered,       mitotic divisions stayed cell stage. This <br /> less case plasmids wild-type SUP45.         result explained loss SUP45 bearing plas- <br /> The presence sup45-n alleles His+Ura+ spore clones        mids. In contrast, case diploids bearing mutant <br /> was confirmed suppression ade1-14 trp1-289            sup45-n alleles lethal spores form microcolonies 4 <br /> alleles (data shown). The results obtained        20 cells (Table 3), abnormally <br />  <br />  <br />                                                                                                                      Page 5 13 <br />                                                                                                (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                                  http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> Table 1: The efficiency maintenance pRS315/SUP45 plasmid SUP45::HIS3 [sup45-n] background <br />  <br />              Transformants 1A-D1628 bearing plasmids a:                             % Ura+ Leu- colonies rounds selection: <br />  <br />         pRS315 [LEU2] with:                  pRS316 [URA3] with:                            Ib                                   II c <br />  <br />                SUP45                                SUP45                               9.5 &#194;&#177; 1.1                            11.0 &#194;&#177; 0.7 <br />                SUP45                              sup45-101                             6.3 &#194;&#177; 0.8                            10.9 &#194;&#177; 1.1 <br />                SUP45                              sup45-102                             2.2 &#194;&#177; 0.7                            4.3 &#194;&#177; 0.9 <br />                SUP45                              sup45-104                             2.5 &#194;&#177; 0.9                            12.1 &#194;&#177; 2.2 <br />                SUP45                              sup45-105                             0.3 &#194;&#177; 0.2                            15.6 &#194;&#177; 1.7 <br />                SUP45                              sup45-107                             2.0 &#194;&#177; 0.3                            5.0 &#194;&#177; 0.6 <br />  <br />  Strain 1A-D1628 bearing disruption SUP45 gene (SUP45::HIS3) SUP45 gene pRS315 plasmid transformed pRS316/ <br />  sup45-n plasmids. b The frequency pRS315/SUP45 loss cotransformants determined (I). c Transformants bearing sup45-n plasmids <br />  selected SC- Ura medium, re-transformed pRS315/SUP45 plasmid frequency pRS315/SUP45 loss cotransformants was <br />  determined (II). At 3 independent transformants tested plasmid combinations. The average data standard errors (&#194;&#177; <br />  SE) presented. <br /> shaped buds abnormally elongated. Apparently,                           length eRF1 encoded SUP45 gene the <br /> SUP45::HIS3 [sup45-n] spores germinate cells                       readthrough product encoded sup45-n allele. It is <br /> fail divide cell cycles.                                        known interaction eRF1 eRF3 mediated <br />                                                                                C-terminus eRF1 [20,21]. Thus, absence of <br /> To determine presence SUQ5 mutation                             readthrough, truncated eRF1 proteins encoded by <br /> could increase viability sup45-n segregants, dip-                   sup45-n alleles interact eRF3. Accordingly, yeast <br /> loid strain heterozygous SUQ5 sup45-n                         eRF3 fused GAL4DB vector pGBT9 eRF1/Sup45 <br /> alleles sporulated tetrads analyzed cell                      fused GAL4AD vector pGADGH introduced into <br /> viability suppression (Table 4). As expected, SUQ5                         strain HF7C separately tested activa- <br /> mutation significantly increased frequency His+                         tion HIS3 reporter gene. In control experiment co- <br /> Ura+ segregants sup45-n mutations sup45-                        transformants harboring plasmids vec- <br /> 107. These data consistent known properties                       tor (pGBT9 pGADGH) did grow the <br /> of SUQ5 tRNA suppress UAA, UGA codon [33].                          absence histidine. The co-transformants harboring <br />                                                                                yeast (eRF3/Sup35and eRF1/Sup45) full-length pro- <br /> Nonsense mutations SUP45 recognized stop                             teins used positive control. Figure 5A shows that <br /> codons presence wild-type SUP45 allele                               co-transformants bearing length eRF1 eRF3 <br /> The readthrough sup45-n mutations explained                        proteins grew selective medium 3-amino- <br /> in ways decreased level eRF1 weak                      1,2,4-triazole (3-AT), co-transformants carrying <br /> termination context surrounding sup45-n mutations. To                          truncated variants eRF1 did not. <br /> assess possibilities determined the <br /> sup45-n alleles reassigned presence wild-                   Western blot analysis revealed presence fusion pro- <br /> type SUP45 allele. For purpose two-hybrid                      tein wild-type eRF1 protein transform- <br /> was used [37]. This approach allows distinguish full-                    ants, sup45-102 (Figure 5B). No readthrough <br />  <br /> Table 2: Nonsense mutations SUP45 lead decreased ascospore viability <br />  <br />   D1628 transformed        Plasmid      Total number tetrads   Tetrads indicated number viable spores     % His+Ura+ segregants a <br />          plasmids:             stability % <br />  <br />                                                                            4           3              2           1 <br />  <br />  pRS316/SUP45                      71                  49                  7           19            22           1                     27 <br />  pRS316/sup45-101                  74                  52                  6            6            38           2                     15 <br />  pRS316/sup45-102                  71                  30                  1            1            26           2                      5 <br />  pRS316/sup45-104                  74                  48                  0            2            42           4                      2 <br />  pRS316/sup45-105                  72                  35                  0            0            32           3                      0 <br />  pRS316/sup45-107                  69                  31                  3            5            21           2                     14 <br />  <br />  The His+Ura+ segregants characteristic sup45-n allele presence (His+ marker SUP45 gene disruption Ura+- for <br />  pRS316 plasmid presence). The data correspond sum experiments cell number nonviable colonies calculated from <br />  (Table 3). <br />  <br />  <br />                                                                                                                                         Page 6 13 <br />                                                                                                                 (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                          http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> Table 3: Nonsense mutations SUP45 lead block cell divisions spore formation <br />  <br />   D1628 transformed    Total number non-viable   Percentage non-viable spores forming microcolony following numbers cells: <br />          plasmids:                   spores <br />  <br />                                                                    5&#226;&#8364;&#8220;20                         3&#226;&#8364;&#8220;4                              1 <br />  <br />  pRS316/SUP45                           22                          0                            0                              100 <br />  pRS316/sup45-101                       68                          3                            1                               96 <br />  pRS316/sup45-102                       52                          15                           25                              60 <br />  pRS316/sup45-104                       64                          8                            28                              64 <br />  pRS316/sup45-105                       39                          13                           18                              69 <br />  pRS316/sup45-107                       25                          24                           8                               68 <br />  <br />  <br />  <br />  <br /> proteins observed plasmids bearing                    existence mutated nonsense-suppressed tRNA gene in <br /> sup45-n mutations sup45-101, small                     genetic background parent strain employed in <br /> amount readthrough fusion protein detected. Thus,                  study unlikely, codon-specificity of <br /> in presence normal wild-type eRF1                  suppressors: work ochre (UAA) opal <br /> readthrough sup45-n alleles completely repressed.                   (UGA) sup45-n mutations isolated. Further the <br />                                                                           parent strain strains used testing viability of <br /> Discussion                                                                sup45-n mutations, possess suppressor activity. <br /> In study shown yeast containing non-                    Moreover, mutations tRNA encoding genes were <br /> sense mutations SUP45 gene selected                  strain S288C [40], derivative used in <br /> viable strains. The product gene, translation termi-              plasmid shuffle experiments. <br /> nation factor eRF1, essential small deletions <br /> from C-terminus (about 30 amino acids) lethal                     Viable opal <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> (supK584) essential <br /> [20,21]. Consistent this, shown full-               gene prfB encoding bacterial release factor RF2 been <br /> length eRF1 protein present mutants bearing                     described S. typhimurium [30]. It shown that <br /> sup45-n alleles, smaller amounts.                             mutation reduces cellular RF2. Such a <br />                                                                           reduction RF2 level leads inefficient termination of <br /> There processes known overcome                   translation autosuppression. Nonsense muta- <br /> effect nonsense codons, suppression                  tions isolated essential SUP35 gene <br /> mutated natural cellular RNA [reviewed [38]]. The                   encoding translation termination factor eRF3, both <br /> readthrough in-frame stop-codon pro-                     (sup35-2 (UAG) sup35-21 (UAA)) [41,42] via- <br /> grammed weak termination context and/or RNA-sec-                       ble. Thus, different translation termination factors, such <br /> ondary structure elements [see [1] recent review].                    RF2, eRF1 eRF3, common property &#226;&#8364;&#8220; via- <br />                                                                           bility nonsense mutants encoding genes. Non- <br /> Many commonly used E. coli laboratory strains                   sense mutations SUP45 gene (sup45-0) SUQ5- <br /> well wild-type populations contain UAG suppressor                      ochre suppressing background previously iso- <br /> tRNAs [39]. For S. cerevisiae data unknown. The                  lated; nevertheless, support viability the <br />  <br /> Table 4: Nonsense mutations SUP45 support ascospore viability presence SUQ5 <br />  <br />    Strain D1638      Total number                         Number tetrads viable spores                              % His+Ura+ <br />  transformed        tetrads                                                                                                segregants <br />      plasmids: <br />  <br />                                                 4                    3                    2                    1 <br />  <br />  pRS316/SUP45              35                   8                   12                   14                    1                      32 <br />  pRS316/sup45-101          36                   11                  11                   13                    1                      33 <br />  pRS316/sup45-102          31                   9                   10                   11                    1                      35 <br />  pRS316/sup45-104          20                   1                   7                    11                    1                      17 <br />  pRS316/sup45-105          36                   6                   10                   18                    2                      30 <br />  pRS316/sup45-107          36                   2                   9                    24                    1                      18 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Page 7 13 <br />                                                                                                            (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                   http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br />                                                                    occurrence natural sites termination efficiency of <br />                                                                    tetranucleotide stop codon [44,45]. For UAA stop codon <br />                                                                    termination efficiency influenced fourth posi- <br />                                                                    tion order G &gt; A &gt; U &#226;&#8240;&#165; C (from efficient to <br />                                                                    efficient termination) frequent +4 con- <br />                                                                    text S. cerevisiae A, C frequent [45]. In <br />                                                                    accordance data relative level eRF1 full- <br />                                                                    length protein correlates strength transla- <br />                                                                    tion termination signal (Table 5). No UAAA mutations <br />                                                                    isolated screen. For UGA stop codon trans- <br />                                                                    lation efficiency influenced fourth position the <br />                                                                    order G&gt;U&gt;A&gt;C, UAG order different <br />                                                                    (A&gt;U&gt;C&gt;G) [45]. Indeed, sup45-107 (UGAA) mutation, <br />                                                                    sup45-n mutations, situated weak <br />                                                                    termination context. Thus, premature stop codons the <br />                                                                    SUP45 gene suppressed relatively high effi- <br />                                                                    ciency depending surrounding sequence. <br />  <br />                                                                    Interestingly, sup45-0 mutations, led appear- <br />                                                                    ance in-frame UAA stop codon [29], situated in <br />                                                                    strongest +4 context purine presented in <br /> Expression <br /> amount 5of eRF1 <br /> Figure        sup45-n mutations <br />                     protein            to <br />                             does notinlead presence wild-type <br />                                               readthrough          fourth position. However discrepancy our <br /> Expression sup45-n mutations presence                 data Stansfield et al. explained by <br /> wild-type eRF1 protein does lead                  different context influence viable sup45-n <br /> readthrough. A. Two-hybrid interactions. Strain HF7C           mutation completely identical sup45-18 [29] <br /> co-transformed pGADGH plasmids carrying different             isolated (Borchsenius, Zhouravleva, unpub- <br /> sup45-n mutations pGBT9/SUP35 plasmid. The interac-            lished). Also, readthrough premature stop codons <br /> tions tested extent resistance 3-AT. Ten         SUP45 gene explained weak ter- <br /> serial dilutions yeast suspensions density     mination context presence wild-type <br /> used. Five independent transformants combination          eRF1 protein readthrough fusion Gal4- <br /> were tested. Representative results shown. B. Western <br />                                                                    Sup45-n mRNAs. Interestingly, possibility isolating <br /> blot showing expression full-length eRF1+Gal4 and <br /> truncated eRF1+Gal4 proteins predicted molecular              viable nonsense mutations SUP45 gene the <br /> mass transformants (A).                          existence wild-type tRNAs yeast cytoplasm has <br />                                                                    predicted [29]. Natural nonsense suppression a <br />                                                                    widespread mechanism translation regulation <br /> absence SUQ5 mutation leading lethality               [reviewed [38]]. To date, ability different yeast <br /> crosses wild-type strain [20].                                tRNA translate stops codons shown <br />                                                                    [46-48]. Experiments progress identify precise <br /> It known relative efficiency termination       mechanism leading readthrough sup45-n mutations. <br /> depends local sequence context surrounding the <br /> termination codon [reviewed [1,2,43]]. In numerous              Although sup45-n mutations lead lethality dif- <br /> experiments E. coli shown             ferent genetic backgrounds, viability increased after <br /> important determinant termination efficiency             growth absence wild-type allele. The mainte- <br /> nucleotide immediately following stop codon (the +4            nance plasmid containing wild-type copy the <br /> base). This suggests translation termination signal       SUP45 gene changed: frequency loss very <br /> consists nucleotides [39,44]. Statistical analysis      low compared plasmid bearing sup45-n mutation, <br /> sequences surrounding stop codons natural termina-              however, frequency significantly increased after <br /> tion sites shown significant bias bases          reshuffle SUP45 bearing plasmid sup45-n mutants. <br /> around stop codon, marked             In crosses sup45-n strains wild-type <br /> nucleotide immediately following stop codon                    strain viability segregants bearing sup45-n alleles <br /> [reviewed [39,43]]. This analysis identified contexts           higher meiotic progeny diploid cells heter- <br /> that favor termination vivo different organisms,             ozygous SUP45 disruption. Possibly, additional muta- <br /> among E. coli S. cerevisiae. Although          tions selected stage selection, or, <br /> context different bacteria yeast, quite       alternatively, decreased level eRF1 influence <br /> good correlation frequency +4 nucleotide            expression components translation termi- <br />  <br />  <br />                                                                                                                        Page 8 13 <br />                                                                                                  (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                      http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> Table 5: The efficiency UAA-readthrough sup45-n mutations depends base following stop codon <br />  <br />  SUP45 allele   Termination tetranucleotide   Level eRF1 (% wild type)   Natural occurrence highly expressed genes (%) [45] <br />  <br />   sup45-102               UAAG                               8                                           35.9 <br />   sup45-104               UAAU                              13                                           16.7 <br />   sup45-105               UAAU                              14                                           16.7 <br />   sup45-101               UAAC                              32                                           1.3 <br />  <br />  <br />  <br />  <br /> nation complex. Further studies necessary identify             amino acid positions corresponding premature <br /> the precise mechanism phenomenon.                             stop codons. Indeed, sup45-n mutations situ- <br />                                                                       ated highly conserved regions eRF1 (data not <br /> It known high levels nonsense suppression               shown). <br /> yeast lethal [49]. Most sup45-n mutations lead to <br /> decreased spore viability. Spores bearing sup45-n muta-               Previously shown truncated eRF1 proteins <br /> tions characterized limited growth germina-              411, 360 359 amino acids able bind to <br /> tion leading formation micro-colonies 4 20            ribosome [29]. In experiments truncated eRF1 <br /> cells. The arrest cell division sup45-n bearing spores          proteins 384, 316 282 amino acids, 265 <br /> cannot explained presence residual                  amino acids, bind ribosome. This result suggests that <br /> eRF1 protein derived heterozygous diploid. It                eRF1 ribosome-binding site localized between <br /> has shown disruption SUP45 gene leads                amino acids 265 282. Consistent data, it <br /> to blockage cell cycle START G1, manifested          previously proposed human eRF1 a <br /> as inability spores produce microcolonies [50].             ribosome-binding site includes amino acids 246 270 <br /> Interestingly, presence mutated suppressor tRNA                [24]. <br /> also led increase spore lethality [51] arrest in <br /> cell division approximately 50-cell stage [52]. The            The eRF3-binding site S. cerevisiae eRF1 located in <br /> combination sup45-n alleles SUQ5 mutation                     C-terminal region, particular deletion from <br /> restores viability sup45-n spores sup45-0           eRF1 19 C-terminal amino acids abolish interac- <br /> mutations [29].                                                       tion eRF3 [20,21]. The two-hybrid analysis used in <br />                                                                       work confirms loss extreme C-terminus of <br /> All sup45-n mutations (but missense mutations)                eRF1 abolishes interaction eRF3. <br /> led thermosensitivity. The data concerning effi- <br /> ciency translation termination different temperatures           The studies described raise question mech- <br /> are controversial: S. typhimurium higher 42&#194;&#176;C           anisms leading viability nonsense mutations in <br /> than 37&#194;&#176;C supK584 wild-type strains [30].              essential genes. To date mutations isolated <br /> However S. cerevisiae efficiency termination did           essential genes S. cerevisiae [41,42,54,55] <br /> not change wild-type strain, lower 33&#194;&#176;C            existence different mechanisms pro- <br /> at 30&#194;&#176;C sup45 mutant [33]. We did sig-              posed. Viability pol3 nonsense mutations been <br /> nificant difference full-length eRF1 pro-            explained presence [PSI+] factor encoded the <br /> tein non-permissive temperature compared               SUP35 gene parent strain, leads to <br /> the permissive temperature mutant sup45-104               readthrough stop codons mutated POL3 gene <br /> strain wild-type strain. Thus thermosensitivity            [55]. Analysis snm1 nonsense mutations revealed that <br /> of sup45-n mutants explained                 mutations suppressed amplification the <br /> reduction eRF1 protein produced               plasmid carrying snm1 alleles. Plasmid amplification <br /> mutant high temperature proposed RF2 protein                leads increase SNM1 mRNA level, which, is <br /> in supK584 strain [30].                                               believed led partial readthrough resulting the <br />                                                                       appearance length Snm1 protein. Interestingly, <br /> The sup45-n mutations lead high level nonsense                snm1 mutations result translation infidelity sug- <br /> codon suppression (up 42% UGA stop codon                  gesting translation suppression specific the <br /> measured &#206;&#178;-galactosidase assay)                     SNM1 gene [54]. The viability sup35-2 sup35-21 <br /> decreased level eRF1. This result consistent pre-          mutations explained reduction eRF3 level that <br /> vious findings [29,53]. However, additional causes can-               causes inefficient termination translation leads to <br /> not excluded full-length eRF1 protein produced              autosuppression [41,42]. However, precise mecha- <br /> in sup45-n mutants contains missense substitutions             nism leading readthrough mutations, as <br />  <br />  <br />                                                                                                                            Page 9 13 <br />                                                                                                      (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> sup45-n mutations, remains unclear. Future studies         two-hybrid interactions assessed ability to <br /> include identification components partici-      transactivate HIS3 reporter gene (to confer growth the <br /> pate process.                                           absence histidine presence different con- <br />                                                                 centrations 3-AT). <br /> Conclusions <br /> The data presented work nonsense muta-        Bacterial strains used work JM109 (recA1 <br /> tions (sup45-n) SUP45 gene isolated      supE44 endA1 hsdR17 gyrA96 relA1 thi &#206;&#8221; (lac-proAB) F&apos; <br /> such mutations support viability different genetic back-     [traD36 proAB+lacIqlacZ&#206;&#8221;M15]) [61] XL1-Blue (recA1, <br /> grounds. All mutations lead decreased level       endA1, gyrA96, thi-1, hsdR17, supE44, relA1 [F&apos;, proAB, lacIq, <br /> full-length eRF1 protein ability suppress    &#206;&#8221;(lacZ)M15, Tn10(tet)]) [62]. <br /> other nonsense mutations. It appear low <br /> level eRF1 (up 10% relative wild type) sufficient   For quantitative characterization nonsense suppression <br /> to support viability. All sup45-n mutations situated     sup45 mutants &#206;&#178;-galactosidase reporter [33] was <br /> weak +4 termination context relative level eRF1      used. Mutants transformed pUKC815 plasmid <br /> full-length protein correlates strength    containing PGK1-lacZ fusion (control) pUKC817, <br /> nucleotide translation termination signal. Most sup45-       818, 819 plasmids carrying TAA, TAG, TGA (termination <br /> n mutations lead decreased spore viability thermo-       codons), respectively, cloned frame lacZ. &#206;&#178;-galac- <br /> sensitivity. Our data confirm site interaction      tosidase activity determined described [63]. Only <br /> with eRF3 localized C-terminus eRF1. Our           case nonsense suppression active &#206;&#178;-galac- <br /> results suggest ribosome-binding site yeast         tosidase synthesized. Efficiencies suppression were <br /> eRF1 localized amino acids 265 282.              calculated ratio &#206;&#178;-galactosidase activity cells har- <br />                                                                 boring lacZ premature termination codon in <br /> Methods                                                         cells normal allele lacZ. Values liquid &#206;&#178;-galac- <br /> Strains, media reagents                                     tosidase assay represent mean assays <br /> The Saccharomyces cerevisiae strains used study     independent transformants. <br /> listed additional file 1. All strains [psi-]. Strains <br /> BSC483/4c D1628 described earlier                 DNA/RNA methods <br /> [56,57]. Yeast cultures grown standard rich (YPD)       Plasmid DNA isolated described [62]. Yeast <br /> or synthetic (SC) media [58] 25&#194;&#176;C. Usually, omnipo-          genomic DNA prepared using lyticase (Sigma) and <br /> tent SUP45 mutations selected reversions          genomic DNA purification kit (Promega). <br /> nonsense mutations different types (for example, UAA <br /> and UGA) [see [59] review], decreases proba-       The SUP45 ORF amplified PCR genomic <br /> bility obtaining mutations tRNA genes. In work       DNA yeast strains using oligonucleotides correspond- <br /> we used reversions nonsense mutations             ing 5&apos; ORF (82 &#226;&#8364;&#8220; CATTTCGGCTTGTCTCC or <br /> same type (UAA). The yeast strain 1B-D1606 containing           99 &#226;&#8364;&#8220; TTTGTGATTCCATGAAGAGGATAACAGA CT) and <br /> nonsense mutations types constructed.          oligonucleotide, complementary 3&apos; end the <br /> For selection suppressor mutants, simultaneous spon-         ORF (83 &#226;&#8364;&#8220; TCTGGCATCTAGTGATTAAATTC). PCR reac- <br /> taneous reversions histidine lysine prototrophy          tions performed using Taq Pfu polymerase <br /> were employed. His+ Lys+ revertants crossed           (Promega) recommended manufacturer. <br /> strains bearing sup35 sup45 mutations identify            Sequences PCR primers used double strand <br /> SUP45 mutants. Standard methods yeast genetics          sequencing were: 82, 83 (see above), 45 &#226;&#8364;&#8220; CGACTGCAC- <br /> used [58]. Transformants grown media selec-         CTAAATCCAATG, 87 &#226;&#8364;&#8220; CCTCATTATCCTCGGCATC, 92 &#226;&#8364;&#8220; <br /> tive plasmid maintenance (SC-Trp, SC-Leu, SC-Ura).          GTTCGATGTCAAAAGTGAC, 93 &#226;&#8364;&#8220; GACGAAATTTC- <br /> Suppression nonsense mutations estimated              CCAGGACAC, 94 &#226;&#8364;&#8220; CAAAAACTTCGGTGCTAC, 100 &#226;&#8364;&#8220; <br /> growth 25&#194;&#176;C synthetic media lacking corre-            GACATCGAACCTTACAAACCTATC, 101 &#226;&#8364;&#8220; ctgctatcacttc- <br /> sponding amino acids. To determine frequency             cacccaac, 104 &#226;&#8364;&#8220; GAAAGGTCGCCGAAGTTGCTG, 105 &#226;&#8364;&#8220; <br /> plasmid loss transformants plated 200 cells         CAGCAACTTCGGCGACCTTTC. At independent <br /> Petri dish YPD medium, grown 25&#194;&#176;C             PCR-products sequenced subcloned pGEM-T <br /> days replica plated SC-Ura SC-Leu media.              vector (Promega). The presence mutated allele in <br /> About 200 500 colonies 3 independent trans-             pGEM-T plasmid verified sequencing. <br /> formants, tested. For plasmid shuffle, selective <br /> medium containing 1 mg/ml 5-fluoroorotic acid (5-FOA,           Plasmids <br /> Sigma) used. Yeast transformations performed           Plasmid pRS316/ySUP45 constructed subcloning <br /> as described previously [60]. 3-AT plates contained differ-     XbaI-HindIII fragment pYex-SUP45 (a gift V. <br /> ent concentrations 3-amino-1,2,4-triazole (Sigma). The       Kushnirov) sites pRS316 [64]. Plasmid <br />  <br />  <br />                                                                                                                     Page 10 13 <br />                                                                                                (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                  http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> pRS315/SUP45 described earlier [57]. Plasmids            Additional material <br /> pRS316/sup45-101, pRS316/sup45-102, pRS316/sup45- <br /> 104, pRS316/sup45-107 obtained substitution of <br /> the BglII-EcoRI fragment pRS316/SUP45 BglII- <br />                                                                   Additional File 1 <br />                                                                   Table 6. Yeast strains used study <br /> EcoRI fragments pGEM-T/sup45-101, pGEM-T/                    Click file <br /> sup45-102, pGEM-T/sup45-104 pGEM-T/sup45-107,                 [http://www.biomedcentral.com/content/supplementary/1471- <br /> respectively. Plasmid pRS316/sup45-105 obtained            2199-4-2-S1.ppt] <br /> substituting BglII-BamHI fragment pRS316/SUP45 <br /> for BglII-BamHI fragment pGEM-T/sup45-105. Plas- <br /> mids pRS315/sup45-101, pRS315/sup45-102, pRS315/ <br /> sup45-104, pRS315/sup45-105, pRS315/sup45-107            Acknowledgments <br /> were obtained substituting BglII-BamHI fragment        We grateful M. Tuite V. Kushnirov yeast strains and <br /> pRS315/SUP45 BglII-BamHI fragments pRS316/          plasmids; C. Le Goff technical assistance M.D.Ter-Avane- <br /> sup45-101,      pRS316/sup45-102,    pRS316/sup45-104,       syan helpful discussion. We thank H.B.Osborne reading the <br />                                                              manuscript. This work supported common grant CNRS (PICS <br /> pRS316/sup45-105 pRS316/sup45-107, respectively. <br />                                                              1113), S.M., S.Ch., S.I-V., G.Z. RFBR (00-04-22001NCNI_a), S.M., <br />                                                              S.Ch., S.I-V. CRDF No.ST-012-0, S.I-V., G.Z. CRDF RB1-2336- <br /> Plasmids pGBT9/SUP35 pGADGH/SUP45                   ST-02. S.M INTAS(YSF-00-19). <br /> described previously [41]. Plasmids pGADGH/sup45-101, <br /> pGADGH/sup45-102,          pGADGH/sup45-104              References <br /> pGADGH/sup45-107 obtained substituting BglII-        1.      Bertram G, Innes S, Minella O, Richardson J, Stansfield I: Endless <br /> EcoRI fragment pGADGH/SUP45 BglII-EcoRI frag-                 possibilities: translation termination stop codon recog- <br />                                                                      nition. Microbiology 2001, 147:255-269. <br /> ment pGEM-T/sup45-101, pGEM-T/sup45-102,                2.      Kisselev LL, Buckingham RH: Translational termination comes <br /> pGEM-T/sup45-104 pGEM-T/sup45-107, respectively.                 age. Trends Biochem Sci 2000, 25:561-566. <br />                                                              3.      Nakamura Y, Ito K, Ehrenberg M: Mimicry grasps reality trans- <br /> Plasmid pGADGH/sup45-105 obtained substitut-                  lation termination. Cell 2000, 101:349-352. <br /> ing BglII-BamHI fragment pGADGH/SUP45 BglII-          4.      Poole E, Tate W: Release factors role decoding <br /> BamHI fragment pGEM-T/sup45-105.                                proteins: specificity fidelity termination protein <br />                                                                      synthesis. Biochim Biophys Acta 2000, 1493:1-11. <br />                                                              5.      Frolova L, Le GX, Rasmussen HH, Cheperegin S, Drugeon G, Kress <br /> Protein procedures                                                   M: A highly conserved eukaryotic protein family possessing <br /> Yeast cultures grown YPD medium                      properties polypeptide chain release factor. Nature 1994, <br />                                                                      372:701-703. <br /> medium selective plasmid markers. Ribosomal frac-        6.      Craigen WJ, Lee CC, Caskey CT: Recent advances peptide <br /> tions prepared described [65]. From            chain termination. Mol Microbiol 1990, 4:861-865. <br />                                                              7.      Caskey CT, Forrester WC, Tate W, Ward CD: Cloning the <br /> &#206;&#188;g proteins separated 12 % SDS-polyacry-              Escherichia coli release factor 2 gene. J Bacteriol 1984, <br /> lamide gel according Laemmli [66], transferred             158:365-368. <br /> a nitrocellulose membrane (Immobilon-P). Western blot        8.      Scolnick E, Tompkins R, Caskey T, Nirenberg M: Release factors <br />                                                                      differing specificity terminator codons. Proc Natl Acad Sci <br /> analysis performed 1/100 diluted polyclonal                 U S A 1968, 61:768-774. <br /> rabbit antibodies obtained mixture      9.      Weiss RB, Murphy JP, Gallant JA: Genetic screen cloned <br />                                                                      release factor genes. J Bacteriol 1984, 158:362-364. <br /> peptides N-terminal (CLTDEYGTASNIKSRV)              10.     Breining P, Piepersberg W: Yeast omnipotent supressor SUP1 <br /> and C-terminal (VDESEDEYYDEDEGS) regions S.                   (SUP45): nucleotide sequence wild type mutant <br /> cerevisiae eRF1 purified N-terminal             gene. Nucleic Acids Res 1986, 14:5187-5197. <br />                                                              11.     Himmelfarb HJ, Maicas E, Friesen JD: Isolation SUP45 <br /> peptide eRF1. Bound antibodies detected using                omnipotent suppressor gene Saccharomyces cerevisiae and <br /> the Amersham ECF (Amersham Pharmacia Bio-                     characterization gene product. Mol Cell Biol 1985, <br /> tech) Storm 840 PhosphorImager (Molecular                    5:816-822. <br />                                                              12.     Nakamura Y, Ito K, Matsumura K, Kawazu Y, Ebihara K: Regulation <br /> Dynamics, USA). Signal intensities quantified using             translation termination: conserved structural motifs in <br /> ImageQuaNT 5.2 software (Molecular Dynamics, USA).                   bacterial eukaryotic polypeptide release factors. Biochem <br />                                                                      Cell Biol 1995, 73:1113-1122. <br />                                                              13.     Ito K, Ebihara K, Uno M, Nakamura Y: Conserved motifs in <br /> Authors&apos; contributions                                               prokaryotic eukaryotic polypeptide release factors: <br /> SM experiments described                tRNA-protein mimicry hypothesis. Proc Natl Acad Sci U S A 1996, <br />                                                                      93:5443-5448. <br /> paper, GZ SS performed initial cloning sequenc-      14.     Ito K, Uno M, Nakamura Y: A tripeptide &apos;anticodon&apos; deciphers <br /> ing sup45 mutations. SI, MP GA supervised             stop codons messenger RNA. Nature 2000, 403:680-684. <br /> work participated coordination. All authors read   15.     Song H, Mugnier P, Das AK, Webb HM, Evans DR, Tuite MF: The <br />                                                                      crystal structure human eukaryotic release factor eRF1 &#226;&#8364;&#8220; <br /> and approved final manuscript.                                   mechanism stop codon recognition peptidyl-tRNA <br />                                                                      hydrolysis. Cell 2000, 100:311-321. <br />                                                              16.     Klaholz B, Pape T, Zavialov AV, Myasnikov AG, Orlova EV, Vester- <br />                                                                      gaard B: Structure Escherichia coli ribosomal termina- <br />                                                                      tion complex release factor 2. Nature 2003, 421:90-94. <br />                                                              17.     Vestergaard B, Van LB, Andersen GR, Nyborg J, Buckingham RH, <br />                                                                      Kjeldgaard M: Bacterial polypeptide release factor RF2 is <br />  <br />  <br />                                                                                                                        Page 11 13 <br />                                                                                                   (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                                 http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br />       structurally distinct eukaryotic eRF1. Mol Cell 2001,              38.   Beier H, Grimm M: Misreading termination codons in <br />       8:1375-1382.                                                                  eukaryotes natural nonsense suppressor tRNAs. Nucleic <br /> 18.   Scarlett DJ, McCaughan KK, Wilson DN, Tate WP: Mapping func-                  Acids Res 2001, 29:4767-4782. <br />       tionally important motifs SPF GGQ decoding                   39.   Tate WP, Poole ES, Mannering SA: Hidden infidelities the <br />       release factor RF2 Escherichia coli ribosome                        translational stop signal. Prog Nucleic Acid Res Mol Biol 1996, <br />       hydroxyl radical footprinting. Implications macromo-                      52:293-335. <br />       lecular mimicry structural changes RF2. J Biol Chem           40.   Percudani R, Pavesi A, Ottonello S: Transfer RNA gene redun- <br />       press.                                                                        dancy translational selection Saccharomyces cerevisiae. <br /> 19.   Ebihara K, Nakamura Y: C-terminal interaction translational                J Mol Biol 1997, 268:322-330. <br />       release factors eRF1 eRF3 fission yeast: G-domain                41.   Cosson B, Couturier A, Chabelskaya S, Kiktev D, Inge-Vechtomov S, <br />       uncoupled binding role conserved amino acids.                      Philippe M: Poly(A)-binding protein acts translation termi- <br />       RNA 1999, 5:739-750.                                                          nation eukaryotic release factor 3 interaction does <br /> 20.   Eurwilaichitr L, Graves FM, Stansfield I, Tuite MF: The C-terminus            influence [PSI+] propagation.             Mol Cell Biol 2002, <br />       eRF1 defines functionally important domain transla-                  22:3301-3315. <br />       tion termination Saccharomyces cerevisiae. Mol Microbiol             42.   Zhou P, Derkatch IL, Uptain SM, Patino MM, Lindquist S, Liebman <br />       1999, 32:485-496.                                                             SW: The yeast non-Mendelian factor [ETA+] variant of <br /> 21.   Ito K, Ebihara K, Nakamura Y: The stretch C-terminal acidic                [PSI+], prion-like form release factor eRF3. EMBO J 1999, <br />       amino acids translational release factor eRF1 primary                 18:1182-1191. <br />       binding site eRF3 fission yeast. RNA 1998, 4:958-972.            43.   Tate WP, Mannering SA: Three, more: translational <br /> 22.   Merkulova TI, Frolova LY, Lazar M, Camonis J, Kisselev LL: C-termi-           stop signal length. Mol Microbiol 1996, 21:213-219. <br />       nal domains human translation termination factors eRF1               44.   Poole ES, Brown CM, Tate WP: The identity base follow- <br />       eRF3 mediate vivo interaction. FEBS Lett 1999,                   ing stop codon determines efficiency vivo trans- <br />       443:41-47.                                                                    lational termination Escherichia coli.          EMBO J 1995, <br /> 23.   Frolova LY, Tsivkovskii RY, Sivolobova GF, Oparina NY, Serpinsky              14:151-158. <br />       OI, Blinov VM: Mutations highly conserved GGQ motif              45.   Bonetti B, Fu L, Moon J, Bedwell DM: The efficiency translation <br />       class 1 polypeptide release factors abolish ability human               termination determined synergistic interplay <br />       eRF1 trigger peptidyl-tRNA hydrolysis.                 RNA 1999,           upstream downstream sequences Saccharo- <br />       5:1014-1020.                                                                  myces cerevisiae. J Mol Biol 1995, 251:334-345. <br /> 24.   Frolova LY, Merkulova TI, Kisselev LL: Translation termination       46.   Weiss WA, Edelman I, Culbertson MR, Friedberg EC: Physiological <br />       eukaryotes: polypeptide release factor eRF1 composed                    levels normal tRNA(CAGGln) effect partial suppres- <br />       functionally structurally distinct domains. RNA 2000,                     sion amber mutations yeast Saccharomyces cerevi- <br />       6:381-390.                                                                    siae. Proc Natl Acad Sci U S A 1987, 84:8031-8034. <br /> 25.   Bertram G, Bell HA, Ritchie DW, Fullerton G, Stansfield I: Termi-       47.   Pure GA, Robinson GW, Naumovski L, Friedberg EC: Partial sup- <br />       nating eukaryote translation: domain 1 release factor                      pression ochre mutation Saccharomyces cerevisiae by <br />       eRF1 functions stop codon recognition. RNA 2000,                           multicopy plasmids containing normal yeast tRNAGln <br />       6:1236-1247.                                                                  gene. J Mol Biol 1985, 183:31-42. <br /> 26.   Chavatte L, Frolova L, Kisselev L, Favre A: The polypeptide chain       48.   Tuite MF, McLaughlin CS: Endogenous read-through UGA <br />       release factor eRF1 specifically contacts s(4)UGA stop                    termination codon Saccharomyces cerevisiae cell-free sys- <br />       codon located A site eukaryotic ribosomes. Eur J Bio-               tem: evidence involvement mitochondrial and <br />       chem 2001, 268:2896-2904.                                                     nuclear tRNA. Mol Cell Biol 1982, 2:490-497. <br /> 27.   Zhouravleva G, Frolova L, Le Goff X, Le Guellec R, Inge-Vechtomov       49.   Liebman SW, Sherman F: Inhibition growth amber sup- <br />       S, Kisselev L, Philippe M: Termination translation eukaryo-             pressors yeast. Genetics 1976, 82:233-249. <br />       tes governed interacting polypeptide chain release            50.   Borkhsenius AS, Inge-Vechtomov SG: [The role SUP35 and <br />       factors, eRF1 eRF3. EMBO J 1995, 14:4065-4072.                            SUP45 genes controlling Saccharomycetes cell cycle]. Dokl <br /> 28.   Stansfield I, Jones KM, Kushnirov VV, Dagkesamanskaya AR, Poznya-             Akad Nauk 1997, 353:553-556. <br />       kovski AI, Paushkin SV: The products SUP45 (eRF1)            51.   Louis EJ, Haber JE: Nonrecombinant meiosis I nondisjunction in <br />       SUP35 genes interact mediate translation termination                    Saccharomyces cerevisiae induced tRNA ochre suppres- <br />       Saccharomyces cerevisiae. EMBO J 1995, 14:4365-4373.                          sors. Genetics 1989, 123:81-95. <br /> 29.   Stansfield I, Eurwilaichitr L, Akhmaloka , Tuite MF: Depletion   52.   Edelman I, Culbertson MR: Exceptional codon recognition by <br />       levels release factor eRF1 causes reduction                   glutamine tRNAs Saccharomyces cerevisiae. EMBO J <br />       efficiency translation termination yeast. Mol Microbiol                 1991, 10:1481-1491. <br />       1996, 20:1135-1143.                                                     53.   Valouev IA, Kushnirov VV, Ter-Avanesyan MD: Yeast polypeptide <br /> 30.   Kawakami K, Nakamura Y: Autogenous suppression opal                     chain release factors eRF1 eRF3 involved cytoskel- <br />       mutation gene encoding peptide chain release factor                    eton organization cell cycle regulation. Cell Motil Cytoskel- <br />       2. Proc Natl Acad Sci U S A 1990, 87:8432-8436.                               eton 2002, 52:161-173. <br /> 31.   Inge-Vechtomov SG: New genetic lines yeast Saccharomyces             54.   Cai T, Reilly TR, Cerio M, Schmitt ME: Mutagenesis SNM1, <br />       cerevisiae. Vestnik LGU (in Russian) 1963, 21:117-125.                        encodes protein component yeast RNase <br /> 32.   Mortimer RK, Johnston JR: Genealogy principal strains               MRP, reveals role ribonucleoprotein endoribonu- <br />       yeast genetic stock center. Genetics 1986, 113:35-43.                         clease plasmid segregation. Mol Cell Biol 1999, 19:7857-7869. <br /> 33.   Stansfield I, Akhmaloka , Tuite MF: A mutant allele SUP45        55.   Kokoska RJ, Stefanovic L, DeMai J, Petes TD: Increased rates of <br />       (SAL4) gene Saccharomyces cerevisiae shows temperature-                    genomic deletions generated mutations yeast gene <br />       dependent allosuppressor omnipotent suppressor phe-                       encoding DNA polymerase delta decreases cel- <br />       notypes. Curr Genet 1995, 27:417-426.                                         lular levels DNA polymerase delta. Mol Cell Biol 2000, <br /> 34.   Boeke JD, LaCroute F, Fink GR: A positive selection mutants               20:7490-7504. <br />       lacking orotidine-5&apos;-phosphate decarboxylase activity                56.   Eaglestone SS, Cox BS, Tuite MF: Translation termination effi- <br />       yeast: 5-fluoro-orotic acid resistance. Mol Gen Genet 1984,                   ciency regulated Saccharomyces cerevisiae envi- <br />       197:345-346.                                                                  ronmental stress prion-mediated mechanism. <br /> 35.   Brachmann CB, Davies A, Cost GJ, Caputo E, Li J, Hieter P: Designer           EMBO J 1999, 18:1974-1981. <br />       deletion strains derived Saccharomyces cerevisiae                  57.   Le Goff C, Zemlyanko O, Moskalenko S, Berkova N, Inge-Vechtomov <br />       S288C: useful set strains plasmids PCR-mediated                  S, Philippe M: Mouse GSPT2, GSPT1, substitute <br />       gene disruption applications.                  Yeast 1998,          yeast eRF3 vivo. Genes Cells 2002, 7:1043-1057. <br />       14:115-132.                                                             58.   Sherman F, Fink GR, Hicks JB, Cold Spring HL: Laboratory course <br /> 36.   Hani J, Feldmann H: tRNA genes retroelements yeast                 manual methods yeast genetics. Cold Spring Harbor, Cold <br />       genome. Nucleic Acids Res 1998, 26:689-696.                                   Spring Harbor Press 1986. <br /> 37.   Fields S, Song O: A novel genetic detect protein-pro- <br />       tein interactions. Nature 1989, 340:245-246. <br />  <br />  <br />  <br />                                                                                                                                       Page 12 13 <br />                                                                                                                  (page number citation purposes) <br />  BMC Molecular Biology 2003, 4:2                                                               http://www.biomedcentral.com/1471-2199/4/2 <br />  <br />  <br />  <br /> 59.   Inge-Vechtomov SG, Mironova LN, Ter-Avanesyan MD: [Ambiguity <br />       translation: eukaryotic version?].            Genetika 1994, <br />       30:1022-1035. <br /> 60.   Gietz RD, Schiestl RH, Willems AR, Woods RA: Studies the <br />       transformation intact yeast cells LiAc/SS-DNA/ <br />       PEG procedure. Yeast 1995, 11:355-360. <br /> 61.   Yanisch-Perron C, Vieira J, Messing J: Improved M13 phage clon- <br />       ing vectors host strains: nucleotide sequences the <br />       M13mp18 pUC19 vectors. Gene 1985, 33:103-119. <br /> 62.   Sambrook J, Fritsch EF, Maniatis T: Molecular Cloning: A labora- <br />       tory manual. Cold Spring Harbor, Cold Spring Harbor Press 1989. <br /> 63.   Burke D, Dawson D, Stearns T, Cold Spring HL: Methods yeast <br />       genetics. Cold Spring Harbor, Cold Spring Harbor Laboratory Press <br />       2000. <br /> 64.   Sikorski RS, Hieter P: A shuttle vectors yeast host <br />       strains designed efficient manipulation DNA Saccha- <br />       romyces cerevisiae. Genetics 1989, 122:19-27. <br /> 65.   Didichenko SA, Ter-Avanesyan MD, Smirnov VN: Ribosome- <br />       bound EF-1 alpha-like protein yeast Saccharomyces cerevi- <br />       siae. Eur J Biochem 1991, 198:705-711. <br /> 66.   Laemmli UK: Cleavage structural proteins the <br />       assembly head bacteriophage T4. Nature 1970, <br />       227:680-685. <br />  <br />  <br />  <br />  <br />                                                                                    Publish Bio Med Central every <br />                                                                                   scientist read work free charge <br />                                                                               &quot;BioMed Central significant development for <br />                                                                               disseminating results biomedical researc h lifetime.&quot; <br />                                                                                    Sir Paul Nurse, Cancer Research UK <br />  <br />                                                                                  Your research papers be: <br />                                                                                    available free charge entire biomedical community <br />                                                                             <br /> </body></html>