<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Mitotic Chromosome Condensation Requires Brn1p, <br /> the Yeast Homologue of Barren <br />  <br />           In vitro studies suggest that the Barren protein may function as an activator of DNA topoisom- <br />           erase II and/or as a component of the Xenopus condensin complex. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> better understand the role <br />           of Barren in vivo, we generated conditional alleles of the structural gene for Barren (BRN1) in <br />           Saccharomyces cerevisiae. We show that Barren is an essential protein required for chromosome <br />           condensation in vivo and that it is likely to function as an intrinsic component of the yeast <br />           condensation machinery. Consistent with this view, we show that Barren performs an essential <br />           function during a period of the cell cycle when chromosome condensation is established and <br />           maintained. In contrast, Barren does not serve as an essential activator of DNA topoisomerase II <br />           in vivo. Finally, brn1 mutants display additional phenotypes such as stretched chromosomes, <br />           aberrant anaphase spindles, and the accumulation of cells with &#226;&#172;&#381;2C DNA content, suggesting that <br />           Barren function influences multiple aspects of chromosome transmission and dynamics. <br />  <br />  <br /> INTRODUCTION                                                      toward the chromatin bridge. Because this stretched DNA <br />                                                                   <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> did not appear to result <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> global defects in <br /> Chromosomes undergo dramatic programmed changes in                chromosome condensation, a role for Barren in topoisom- <br /> higher-order structure as a prerequisite to their proper seg-     erase II function was inferred from the observation that <br /> regation in mitosis. Replicated sister chromatids must re-        Barren and topoisomerase II coimmunoprecipitate and <br /> main attached to one another until anaphase, undergo sig-         interact in a two-hybrid assay. Indeed, purified Barren <br /> nificant condensation, and become disentangled from one           influences the activity of purified topoisomerase II in <br /> another before complete segregation to opposite poles. Al-        vitro, suggesting that Barren may regulate topoisomerase <br /> though many activities are required to ensure that these          II activity in vivo (Bhat et al., 1996). Although this is an <br /> processes occur correctly, the characterization of key players    exciting possibility, the hypothesis has yet to be tested <br /> remains at an early stage. A number of specialized compo-         directly in vivo. <br /> nents have been identified, either biochemically or geneti-          A different model for Barren function was proposed from <br /> cally, and subsequently shown to be conserved among the           work in the Xenopus system based on the copurification of <br /> different experimental systems. One of the more interesting       Barren with 13S condensin. This five-component complex, in <br /> but perplexing is the Barren protein.                             which all of the proteins are conserved from Xenopus to <br />    Barren was originally discovered as a Drosophila muta-         yeast, was initially isolated from mitotic extracts of Xenopus <br /> tion that caused defects in the nuclear division of neuro-        eggs and shown to promote the condensation of chromatin <br /> nal precursor cells (Bhat et al., 1996). Further examination      in vitro (Hirano and Mitchison, 1994; Hirano et al., 1997). <br /> of the cellular phenotype of barren mutants revealed a            Two of the condensin subunits, XCAP-C and XCAP-E, are <br /> morphology similar to that exhibited by DNA topoisom-             members of the SMC (structural maintenance of chromo- <br /> erase II mutants of Saccharomyces cerevisiae (Holm et al.,        somes) family of proteins (for recent reviews, see Jessberger <br /> 1985). Dividing cells frequently contain chromosomes that         et al., 1998; Strunnikov, 1998; Yanagida, 1998; Hirano, 1999). <br /> appear unable to separate from one another during an-             In vivo, inactivation of the yeast SMC homologues of <br /> aphase; instead, the anaphase chromosomes are fre-                XCAP-C, Smc2p and cut3, leads to defects in mitotic chro- <br /> quently stretched across the two daughter cells, and cy-          mosome condensation (Saka et al., 1994; Strunnikov et al., <br /> tokinesis initiates with the plasma membrane pinching in          1995a). Because Barren is a condensin subunit in the Xenopus <br />                                                                   system, it has been postulated to be required for chromo- <br />   &#226;&#8364;&#161; <br />              some condensation. <br />  <br />  <br />  <br />  <br />  <br />    Assigning a function for Barren based on the Drosophila or <br /> Xenopus studies is complicated by several factors. Because it            Table 1. S. cerevesiae strains used in this study <br /> has not been possible in the Xenopus system to inactivate <br /> individual subunits, their specific contribution to this in              Strain                  Genotype                       Source <br /> vitro condensation activity has yet to be established. Al- <br />                                                                          CH325   MATa lys2-801 ura3-52 his4-539           Holm et al. (1985) <br /> though a role for the SMC subunits in this activity is inferred                   top2-4 <br /> from the condensation defect of yeast smc mutants (Saka et               CH335   MATa lys2-801 ura3-52 his4-539           Holm et al. (1985) <br /> al., 1994; Strunnikov et al., 1995a), the in vivo role of indi-                   TOP2 <br /> vidual non-SMC subunits in chromosome condensation has                   CH345   MAT&#226;&#65533;&#163; ade2 ura3-52 top2-4                 Holm et al. (1985) <br /> not been addressed. Thus, it remains an open question                    CH1580 MATa leu2 trp1 ura3-52                    Holm laboratory <br /> whether the non-SMC subunits mediate condensin activity                  CH1861 MATa/&#226;&#65533;&#163; ura3-52/ura3-52 leu2/leu2          Holm laboratory <br /> or are involved in auxiliary functions, such as linking con-                      trp1-63/trp1-63 <br /> densation with DNA topoisomerase II activity. At first, this             CH2518 MATa ura3-52 leu2 trp1-63                 This study <br />                                                                                   brn1&#226;&#172;&#8249;TRP1 [pCH1719 (BRN1 <br /> latter model might seem particularly appealing given both <br />                                                                                   URA3)]a <br /> the in vitro interaction between Drosophila Barren and DNA               CH2523 MATa ura3-52 leu2 trp1-63                 This study <br /> topoisomerase II and the failure to observe a condensation                        BRN1.TRP1b <br /> defect in the Drosophila barren mutants. However, studies in             CH2528 MAT&#226;&#65533;&#163; ura3-52 leu2 trp1-63                 This study <br /> other systems have not revealed interactions between the                          BRN1.TRP1b <br /> condensin components and DNA topoisomerase II (Hirano                    CH2524 MATa ura3-52 leu2 trp1-63 brn1-           This study <br /> and Mitchison, 1994; Hirano et al., 1997; Strunnikov, per-                        9.TRP1b <br /> sonal communication). Given the apparent discrepancy be-                 CH2529 MAT&#226;&#65533;&#163; ura3-52 leu2 trp1-63 brn1-           This study <br /> tween the Drosophila and Xenopus observations, it is critical                     9.TRP1b <br />                                                                          CH2525 MATa ura3-52 leu2 trp1-63 brn1-           This study <br /> to test the in vivo role of Barren in cell cycle&#226;&#8364;&#8220; dependent <br />                                                                                   16.TRP1b <br /> chromosome metabolism, particularly in chromosome to-                    CH2530 MAT&#226;&#65533;&#163; ura3-52 leu2 trp1-63 brn1-           This study <br /> pology and condensation.                                                          16.TRP1b <br />    With the development of cytological tools in budding                  CH2534 MATa ura3-52 leu2 trp1-63                 This study <br /> yeast and its extremely powerful genetics, Saccharomyces                          BRN1.TRP1 [rho0]b <br /> cerevisiae has become an excellent model system for studying             CH2535 MATa ura3-52 leu2 trp1-63 brn1-           This study <br /> the in vivo function of proteins, such as Barren, involved in                     9.TRP1 [rho0]b <br /> cell cycle&#226;&#8364;&#8220; dependent chromosome dynamics. The coupling                  CH2536 MATa ura3-52 leu2 trp1-63 brn1-           This study <br /> of these methods with biochemical analysis of DNA metab-                          16.TRP1 [rho0]b <br />                                                                          4aAS247 MATa trp1-63 ura3-52 leu2 smc1-2     Strunnikov et al. <br /> olism and topology allows a uniquely broad spectrum of <br />                                                                                                                         (1993) <br /> questions in mitosis to be addressed. Here we exploit these              2bAS283 MATa ade2 his3 leu2 lys2 ura3-52     Stunnikov et al. <br /> approaches to analyze the Barren homologue in budding                             smc2-6                                (1995a) <br /> yeast, addressing its in vivo function in eukaryotic chromo-             1aAS330 MATa smc2-8 ura3 leu2 lys2 his3 ade2 Strunnikov et al. <br /> some metabolism.                                                                                                        (1995a) <br />                                                                          CH2540 MATa trp1-63 ura3-52 leu2 smc1-2      This study <br />                                                                                   [rho0] <br /> MATERIALS AND METHODS                                                    CH2539 MATa ade2 his3 leu2 lys2 ura3-52 smc This study <br />                                                                                   2-6 [rho0] <br /> Strains and Media                                                        CH2587 MATa lys2-801 ura3-52 his4-539 top2- This study <br /> All strains used in this study have an S288c genetic background and               4 [rho0] <br /> are listed in Table 1. Standard genetic techniques were used in the      a <br /> construction and growth of these strains (Sherman et al., 1986). Cells     This strain carries a disruption of the BRN1 gene with the TRP1 <br /> were grown in YEPD (rich medium) or SD (synthetic dextrose or            gene as a marker. <br />                                                                          b <br /> selective) medium. YEPD is 1% yeast extract, 2% bactopeptone, 2%           These strains have TRP1 adjacent to a described gene but do not <br /> dextrose, with or without 2% bacto-agar; SD medium is 0.67% yeast        disrupt the gene. <br /> nitrogen <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span>, 2% bacto-agar, and 2% dextrose. For synthetic com- <br /> plete (SC) medium, 20 mg of uracil and tryptophan and 60 mg of <br /> leucine were added to 1 l of SD medium. 5-FOA plates were made <br /> as described previously (Ausubel et al., 1999).                          and flanking BamHI and XbaI sites) was amplified by PCR with the <br />    Strains lacking mitochondrial DNA ([rho0]) were used for flow         use of primers prCH1140 (TTATGGCAGCACTGCATAATTCTCT- <br /> cytometry and were created by growing [rho&#226;&#171;&#185;] strains in 25 &#226;&#65533;&#174;g/ml         TACTGCGCGGATCCTATGTGATGTTGATCACG) and prCH1148 <br /> ethidium bromide in YEPD to stationary phase. These cells were           (TAGCTCTAGAGATTTCGTAAGCCCGAAGGC). This fragment <br /> plated for single colonies on YEPD and tested for their ability to       was ligated into the BamHI&#226;&#8364;&#8220;XbaI sites of both pCH1097 (LEU2 <br /> grow on YEP plates containing 2% glycerol (a nonfermentable car-         CEN/ARS) and pCH1099 (URA3 CEN/ARS), which are also known <br /> bon source). In addition, fluorescence microscopy of DAPI-stained        as pRS315 and pRS316 vectors, respectively (Sikorski and Hieter, <br /> cells confirmed the absence of mitochondrial DNA in the [rho0]           1989). Plasmids pCH1748 (brn1-9) and pCH1749 (brn1-16) are de- <br /> strains.                                                                 rived by random in vitro mutagenesis from plasmid pCH1720; they <br />                                                                          were isolated in the screen for heat-sensitive brn1 alleles. To make <br />                                                                          plasmid pCH1721 (BRN1.TRP1; TRP1 is inserted next to BRN1), <br /> Plasmids                                                                 PCR was used to fuse the TRP1 allele with a fragment from down- <br /> To construct plasmids pCH11720 (LEU2 CEN/ARS BRN1) and                   stream of the BRN1 locus with flanking EagI and SacII sites with the <br /> pCH1719 (URA3 CEN/ARS BRN1), a 2.3-kilobase (kb) fragment                use of primers prCH1195 (GAATCGGCCGAAAGCCTTCGGGCT- <br /> (containing the entire BRN1 gene, 5&#226;&#172;&#732; and 3&#226;&#172;&#732; untranslated sequences,      TACGAA) and prCH1196 (TAGGCCGCGGTCTTGGTCGCACTA- <br />  <br />  <br /> 1294                                                                                                          Molecular Biology of the Cell <br />                                                                                                      Chromosome Condensation Requires BRN1 <br />  <br />  <br /> ATCAAC). Next, this EagI&#226;&#8364;&#8220;SacII fragment was subcloned into the              First-Cycle Arrest Experiments <br /> same sites of plasmids pCH1720, pCH1748, and pCH1749 to create <br />                                                                             Cell cycle progression was monitored at the restrictive temperature <br /> plasmids pCH1721, pCH1725, and pCH1726. <br />                                                                             in plate assays as described (Weinert and Hartwell, 1993). Strains <br />                                                                             were grown to early log phase at 25&#194;&#176;C. The cells were sonicated, <br />                                                                             diluted, and plated onto prewarmed YEPD plates and incubated at <br /> Creation of Heat-sensitive brn1 Mutants                                     25 or 37&#194;&#176;C. The number of cell divisions was monitored over time by <br /> We first showed that BRN1 is essential by disrupting one allele of          scoring the number of cell bodies (1, 2&#226;&#8364;&#8220; 4, 5&#226;&#8364;&#8220; 8, 9 &#226;&#8364;&#8220;16, and &#226;&#172;&#381;17) in 100 <br /> BRN1 with TRP1 in diploid strain CH1861. Although the control               microcolonies for up to 16 h. When incubated at 25&#194;&#176;C, 88 &#226;&#8364;&#8220;100% of <br /> cross exhibited excellent spore viability, 11 of 12 tetrads from the        the microcolonies from each tested strain contained &#226;&#172;&#381;16 cells after <br /> diploid with the BRN1 disruption exhibited 2:2 segregation for              16 h. <br /> viability; all living spores were Trp&#226;&#171;&#186;, and the inviable spores gave <br /> rise to two to eight cells. We next used the plasmid-shuffle technique <br /> (Boeke et al., 1984) to identify 16 heat-sensitive brn1 mutations. The      Viability Experiments <br /> BRN1 gene in plasmid pCH1720 was mutagenized by PCR ampli-                  We examined viability in cultures that were growing exponentially <br /> fication in 12 independent reactions; each of these reactions con-          at 25&#194;&#176;C. To examine the effects of temperature on asynchronously <br /> tained a 75% lower concentration of dATP than of the rest of the            dividing cells, each culture was divided, one aliquot was incubated <br /> nucleotides. The entire coding region was mutagenized with the use          at 25&#194;&#176;C and one at 37&#194;&#176;C, and the samples were checked for viability <br /> of primers prCH1149 and prCH1150. The amplified DNA contain-                at 3 and 6 h. To check for viability, the samples were sonicated, <br /> ing the mutagenized BRN1 gene and a 6.0-kb BamHI&#226;&#8364;&#8220;XbaI fragment              diluted, and plated on YEPD plates that were incubated at 25&#194;&#176;C for <br /> of plasmid pCH1720 lacking the BRN1 coding region were then                 3 d. To examine the effects of temperature on cells arrested in G1 or <br /> cotransformed into strain CH2518 to allow recombination during              S phase, the exponentially growing cultures were initially incubated <br /> transformation (Muhlrad et al., 1992). The transformants were rep-          for 3 h at 25&#194;&#176;C in 5 &#226;&#65533;&#174;g/ml &#226;&#65533;&#163;-factor or 0.1 M hydroxyurea in YEPD <br /> lica plated onto 5-FOA&#226;&#8364;&#8220; containing plates to select for the loss of         (pH 4.0). Each culture was then divided, an aliquot was incubated at <br /> plasmid pCH1719 (BRN1 URA3) and were simultaneously screened                25 or 37&#194;&#176;C for an additional 3 h, and the cultures were checked for <br /> for phenotypes at 37 and 14&#194;&#176;C. Among 10,000 transformants, &#226;&#172;&#402;10%             viability. <br /> contained <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>knockout</span> mutations for brn1. Although we screened for <br /> cold sensitivity at 14&#194;&#176;C, no cold-sensitive mutants were recovered. <br /> Sixteen heat-sensitive mutants were recovered from transformants            Execution Point Experiments <br /> derived from 10 of the mutagenized pools of DNA. We confirmed <br /> that the heat-sensitive mutations were brn1 mutations by isolating          To determine the time at which Barren function is required in the <br /> and purifying the mutant plasmids and then transforming them into           cell cycle, cultures growing exponentially at 25&#194;&#176;C in YEPD (pH 4.0) <br /> a brn1::TRP1 strain carrying a BRN1 URA3 plasmid (strain CH2518)            were treated for 3 h with 5 &#226;&#65533;&#174;g/ml &#226;&#65533;&#163;-factor to arrest them in G1. <br /> (Robzyk and Kassir, 1992). In each case, eviction of the BRN1               After the &#226;&#65533;&#163;-factor was washed away, the cells were resuspended in <br /> plasmid led to temperature sensitivity.                                     YEPD and divided into three aliquots. Nothing was added to the <br />    To replace the normal chromosomal BRN1 allele with mutant brn1           first aliquot, 0.1 M hydroxyurea was added to the second aliquot, <br /> alleles, the TRP1 gene was inserted just downstream of each plas-           and 10 &#226;&#65533;&#174;g/ml nocodazole was added to the third aliquot. Each <br /> mid-borne brn1 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span>, the brn1 genes plus flanking DNA were              aliquot was divided into two portions, one of which was incubated <br /> cut out of each plasmid with the use of BamHI and SacII, and the            at 25&#194;&#176;C and the other at 37&#194;&#176;C for 3 h. Samples were then taken and <br /> linear DNA was used to transfer the BRN1 gene into strain CH1580            checked for viability as described above. <br /> (trp1 BRN1). After selecting the transformants for TRP1, integration <br /> was confirmed by PCR. In each case, the strain carrying the trans- <br /> ferred brn1 gene was heat sensitive, as expected. Strains bearing           Cell Morphology and Indirect Immunofluorescence <br /> integrated brn1 alleles were used for all physiological experiments.        Yeast nuclear DNA was visualized by DAPI staining of formalde- <br />                                                                             hyde-fixed cells. Asynchronous cultures (2.5 &#226;&#171;&#187; 106 cells/ml) were <br />                                                                             grown in YEPD at 23&#194;&#176;C, shifted to 37&#194;&#176;C for 3 h, and fixed in 3.6% <br /> Drug Sensitivity and Synthetic Lethality                                    formaldehyde (60 min, room temperature). Indirect immunofluo- <br />                                                                             rescence was done essentially as described (Kilmartin et al., 1982) <br /> To assess the drug sensitivity of strains, serial dilutions were spot- <br />                                                                             with the use of the YOL1/34 mAb against &#226;&#65533;&#163;-tubulin (Serotec, Indi- <br /> ted onto YEPD plates containing 0.02 or 0.04 M hydroxyurea, 0.005 <br />                                                                             anapolis, IN) and goat anti-rat FITC (Sigma, St. Louis, MO). <br /> or 0.017% methyl methane sulfonate, and 7.5 or 15 &#226;&#65533;&#174;g/ml benomyl. <br /> Sensitivity to UV light was evaluated by placing spots of cells on <br /> YEPD plates and then exposing them to 254-nm UV radiation doses <br /> from 0 to 200 J/m2 with the use of a Stratalinker 1800 (Stratagene, La      Catenation of Plasmids <br /> Jolla, CA). For each test, the plates were incubated at 25&#194;&#176;C for 2 d.       Strains containing a 14-kb circular minichromosome (pDK243, ARS1 <br /> Strains CH2523 (BRN1), CH2524 (brn1-9), and CH2525 (brn1-16)                CEN3 LEU2 [Koshland et al., 1985]) were pregrown in selective <br /> showed no unusual sensitivity to the DNA-damaging agents hy-                medium, diluted into YEPD, and then grown to early log phase <br /> droxyurea, methyl methane sulfonate, and UV light or to the mi-             (3&#226;&#8364;&#8220; 4 &#226;&#171;&#187; 106 cells/ml) at the permissive temperature (approximately <br /> crotubule-depolymerizing drug benomyl (our unpublished results).            five divisions). Cells were synchronized in G1 by adding 3 &#226;&#65533;&#174;M final <br /> Because previous reports have suggested that Brn1p may interact             &#226;&#65533;&#163;-factor, incubating at 23&#194;&#176;C for 2.5 h, and preshifting to 37&#194;&#176;C for 30 <br /> functionally with DNA topoisomerase II or SMC proteins, we per-             min (typically, &#226;&#172;&#381;85% of cells showed a schmoo morphology). The <br /> formed crosses to determine whether brn1 mutations would exhibit            cells were then released into prewarmed YEPD with 0.1 mg/ml <br /> synthetic lethality with mutations affecting these other proteins.          pronase E in the presence of 20 &#226;&#65533;&#174;g/ml nocodazole. To keep the <br /> Synthetic lethality was tested by crossing strain CH2529 (brn1-9)           BRN1 cells arrested at early M for 3 h at the nonpermissive temper- <br /> with strain CH325 (top2-4), 4aAS247 (smc1-2), or 2bAS283 (smc2-6).          ature, an additional 10 &#226;&#65533;&#174;g/ml nocodazole was added 1 h after <br /> Brn1 mutations are not synthetic lethal with top2-4 (10 of 10 tetrads       release. Plasmid DNA was isolated from 10 ml of the &#226;&#65533;&#163;-factor&#226;&#8364;&#8220; or <br /> were 4:0 for viability), smc2-6 (7 of 7 tetrads were 4:0 for viability),    nocodazole-arrested culture, and half was separated on a 0.6% <br /> or smc1-2 (5 of 7 tetrads were 4:0 for viability). These negative results   agarose gel in 1&#226;&#171;&#187; Tris-acetate-EDTA, transferred to GeneScreen <br /> are provided for completeness, but they provide little information          (Du Pont, Boston, MA), and probed with random-primer-labeled <br /> about the function of Brn1p.                                                pBR322 by standard techniques. <br />  <br />  <br /> Vol. 11, April 2000                                                                                                                            1295 <br />  B.D. Lavoie et al. <br />  <br />  <br /> Chromosome Breakage Assay                                                 RESULTS <br /> Cultures growing exponentially in YEPD at 25&#194;&#176;C were divided, and          Heat-sensitive brn1 Mutations Were Produced in the <br /> half was incubated at 25&#194;&#176;C and half at 37&#194;&#176;C for 5 h. Chromosomal <br /> DNA was isolated with the use of the method described (Spell and          Essential BRN1 Gene <br /> Holm, 1994). Chromosomes were separated on a 1% agarose gel in            To examine the in vivo role of Barren, we generated condi- <br /> 0.5&#226;&#171;&#187; Tris-borate-EDTA in a CHEF-DR pulsed-field gel apparatus             tional alleles in the structural gene for the budding yeast <br /> (Bio-Rad, Richmond, CA). Electrophoresis was performed at 4&#194;&#176;C             homologue Brn1p (for details, see MATERIALS AND <br /> with a 60-s switch time for 16.2 h and then a 90-s switch time for 9 h.   METHODS). We first showed that BRN1 is essential by <br /> Hybridization was performed on the dried, rehydrated gel, as de-          disrupting one allele in a diploid cell; after sporulating the <br /> scribed (Spell and Holm, 1994), with the use of a URA3 probe that         heterozygous diploid, all of the haploid progeny carrying <br /> was prepared by random primer labeling. <br />                                                                           the disrupted gene were inviable, giving rise to only two to <br />                                                                           eight progeny cells. We next used the plasmid-shuffle tech- <br />                                                                           nique (Boeke et al., 1984) to isolate 16 heat-sensitive brn1 <br /> FACS Analysis                                                             mutations. Stable heat-sensitive brn1 mutants were made by <br /> To prepare the cells for flow cytometry, samples were sonicated,          introducing 5 of the 16 brn1 mutations into chromosomes in <br /> diluted, and prepared for microfluorometric analysis by staining          place of the normal BRN1 gene. We chose two alleles (brn1-9 <br /> with propidium iodide, as described (Hutter and Eipel, 1979). A           and brn1-16) for additional experimentation; these alleles <br /> Becton-Dickinson (Franklin Lakes, NJ) FACS was used to determine          caused no unusual drug sensitivity or synthetic lethality <br /> the DNA content of the cells in each sample. A total of 10,000 cells      with top2, smc1, or smc2 mutations (see MATERIALS AND <br /> of each type were analyzed for their DNA content. For cultures            METHODS). <br /> treated with drugs, the cells were grown in YEPD (pH 4.0) and <br /> treated with 5 &#226;&#65533;&#174;g/ml &#226;&#65533;&#163;-factor or 0.1 M hydroxyurea, as indicated in <br /> Figure 3. <br />                                                                           Brn1 Mutations Cause Arrest of Cell Division and <br />                                                                           Cell Cycle&#226;&#8364;&#8220; dependent Loss of Viability <br />                                                                           To examine the effects of the brn1 mutations on cell division, <br /> Chromosome Condensation and Cohesion <br />                                                                           we investigated whether brn1 strains undergo an arrest of <br /> BRN1, brn1, and smc2-8 cultures (2.5&#226;&#8364;&#8220;3 &#226;&#171;&#187; 106 cells/ml) were arrested      cell division in the first cycle after they are placed at the <br /> in G1 (3, 3, and 1 &#226;&#65533;&#174;M final &#226;&#65533;&#163;-factor, respectively) for 2.5 h at 23&#194;&#176;C,    restrictive temperature. To allow comparison of this pheno- <br /> preshifted to 37&#194;&#176;C, and released into nocodazole, as described above      type with other strains, we examined strains CH2523 <br /> for the plasmid catenation assay. In all experiments, cells were fixed <br />                                                                           (BRN1), CH2524 (brn1-9), CH2525 (brn1-16), CH325 (top2-4), <br /> with 3.6% formaldehyde for 2 h and prepared for fluorescence in <br /> situ hybridization (FISH) analysis, as described (see Guacci et al. <br />                                                                           4aAS247 (smc1), and 2bAS283 (smc2) in the same experi- <br /> [1994] for methods and probe details). Digoxigenin (DIG)-labeled          ments. Exponentially growing cultures at 25&#194;&#176;C were soni- <br /> rDNA (chromosome XII) was used to visualize the condensation of           cated and spread on prewarmed YEPD plates at 25 or 37&#194;&#176;C, <br /> repetitive DNA sequences. Condensation of unique sequences was            and the number of cells in each microcolony was counted <br /> monitored as described previously by measuring distances between          over time. The top2 and smc1 strains exhibited a first-cycle <br /> DIG-CEN16 proximal and distal probes signals, which are sepa-             arrest; none of the microcolonies contained more than 4 cell <br /> rated by 255 kb (probes 1 and 3, respectively, from Guacci et al.         bodies even when incubated at 37&#194;&#176;C for 9 h. The majority of <br /> [1994] and Strunnikov et al. [1995a]). Nuclei with a single signal (or    brn1 and smc2 cells underwent arrest in the first division, but <br /> more than two spots) were not included in our analysis because of         a small proportion of the cells exhibited a multicycle arrest; <br /> the difficulty of assigning one spot to two overlapping signals or to     &#226;&#172;&#402;25% of the microcolonies contained more than 4 cells after <br /> hybridization of only one probe. It is noteworthy, however, that          7.5 h of incubation at 37&#194;&#176;C. Even the cells in these micro- <br /> BRN1 cells contained roughly twice as many nuclei with a single           colonies ultimately arrested, however, because none of the <br /> spot than brn1-9 cells, suggesting that our calculated average dis- <br />                                                                           microcolonies from these strains contained more than 16 <br /> tance for condensed DNA is somewhat overestimated, as is pre- <br /> dicted from the distribution of the histogram (see Figure 4D). Sister <br />                                                                           cells after an overnight incubation at 37&#194;&#176;C. Thus, although a <br /> chromatid cohesion was analyzed with the use of DIG-labeled               minority of brn1 cells continue to divide, the majority of the <br /> CEN16 or CEN1 proximal probes, as described by Guacci et al.              cells cease dividing in the first generation after they are <br /> (1994). Mouse anti-DIG antibodies were from Sigma, and goat anti-         shifted to the restrictive temperature. <br /> mouse (FITC) and pig anti-goat (FITC) secondary antibodies were              brn1 mutants also exhibit a loss of viability when incu- <br /> from Jackson Immunoresearch (West Grove, PA). Nuclei were coun-           bated at the restrictive temperature. When exponentially <br /> terstained with propidium iodide in antifade mounting medium              growing cultures are shifted to the restrictive temperature, <br /> (ONCOR; Intergen, New York, NY) and visualized with the use of            &#226;&#172;&#402;75% of the brn1 cells become inviable after 3 h (1.5 gener- <br /> a Zeiss (Thornwood, NY) epifluorescence microscope. Images were           ation times) and &#226;&#172;&#402;95% become inviable after 6 h (3 genera- <br /> recorded digitally with the use of a Princeton (Trenton, NJ) charge-      tion times) (Table 2). This result is quantitatively similar to <br /> coupled device camera with Signal Analytics (Fairfax, VA) process-        results observed with smc1 and smc2 mutants, which main- <br /> ing software, which allows image superimposition. <br />                                                                           tain moderate viability after 1 generation time at the restric- <br />    For the rDNA condensation maintenance experiment, cells were <br /> arrested in G1 with the use of &#226;&#65533;&#163;-factor (as described above) and then <br />                                                                           tive temperature (Strunnikov et al., 1993, 1995a). However, <br /> released into medium containing nocodazole for 2 h at the permis-         the result is noticeably different from what is observed with <br /> sive temperature (23&#194;&#176;C). An aliquot was taken to assess the initial       top2 mutants, which lose viability much more rapidly at the <br /> state of rDNA condensation. The remainder of the culture was then         restrictive temperature (Holm et al., 1985). <br /> shifted to 37&#194;&#176;C for 1 h in the continued presence of nocodazole, after       Loss of viability at the restrictive temperature could be <br /> which the cells were harvested. Condensation was assessed with the        caused by the requirement for Brn1p at a specific time dur- <br /> use of DIG-labeled rDNA, as described above.                              ing the cell cycle, or it could be caused by a more general <br />  <br />  <br /> 1296                                                                                                         Molecular Biology of the Cell <br />                                                                                                     Chromosome Condensation Requires BRN1 <br />  <br />  <br />  <br /> Table 2. Viability at the restrictive temperature of dividing cells and cells arrested in the cell cycle <br />  <br />                                                  Asynchronousa                                                  Drug treatedb <br />  <br /> Strain                                    3h                         6h                         &#226;&#171;&#185;&#226;&#65533;&#163;-Factor                       &#226;&#171;&#185; Hydroxyurea <br />  <br /> CH2523 (BRN1&#226;&#171;&#185;)                           140%                       220%                           94%                              114% <br /> CH2524 (brn1-9)                           29%                         4%                           88%                               96% <br /> CH2525 (brn1-16)                          27%                         8%                           92%                               97% <br /> 4aAS247 (smc1-2)                          22%                         1%                           89%                              100% <br /> 2bAS283 (smc2-6)                          23%                        10%                           95%                               94% <br /> CH325 (top2-4)                            12%                         1%                          116%                               91% <br />  <br /> For each experiment, viability is expressed as a ratio of the number of colonies produced from the 37&#194;&#176;C incubation to the number of colonies <br /> produced from the 25&#194;&#176;C incubation. Values shown are averages of two experiments that produced similar results. <br /> a <br />   Cultures growing exponentially at 25&#194;&#176;C were divided into two aliquots, which were incubated at 25 and 37&#194;&#176;C for the indicated times and <br /> then plated for viability. <br /> b <br />   Cultures growing exponentially at 25&#194;&#176;C were treated with &#226;&#65533;&#163;-factor or hydroxyurea for 3 h and then divided into two portions, which were <br /> incubated at 25 or 37&#194;&#176;C for an additional 3 h and then plated for viability. <br />  <br />  <br />  <br /> requirement for Brn1p in cell metabolism. If the former is                 synchronously dividing brn1 strain is prevented from enter- <br /> true, then brn1 mutants should remain viable at the restric-               ing S phase with hydroxyurea, the viability of the culture <br /> tive temperature when they are prevented from traversing                   remains high, even though it is incubated at the restrictive <br /> the cell cycle; both smc2 and top2 mutants exhibit this phe-               temperature for 3 h (Table 3). If the strain is instead allowed <br /> notype. Exponentially growing brn1, smc1, smc2, and top2                   to pass though S and G2 phases, the viability of the culture <br /> cultures were arrested in G1 phase of the cell cycle with                  decreases to the same extent whether it is treated with <br /> &#226;&#65533;&#163;-factor or in S phase with hydroxyurea. The cultures were                 nocodazole or not. (The efficacy of the nocodazole treatment <br /> divided into two aliquots, and the cells were incubated at                 was confirmed by observations of cell morphology [our <br /> either the restrictive or the permissive temperature for an                unpublished results].) This result is in sharp contrast with <br /> additional 3 h. All cultures maintained good viability (at                 the result observed with a top2 strain, which maintains good <br /> least 84%) when cell cycle arrest was maintained (Table 2).                viability as long as it is prevented from passing through <br /> This result demonstrates that brn1 mutants must pass                       mitosis by treatment with either hydroxyurea or nocoda- <br /> through the cell cycle to incur irreversible damage. It also               zole. These results suggest that the activity or assembly of <br /> demonstrates that arrested G1- and S-phase cells do not                    Brn1p is required between early S phase and early M phase. <br /> require ongoing Brn1p activity. <br />                                                                            Brn1 Mutants Arrest Heterogeneously and Exhibit <br /> Brn1p Is Required between Early S Phase and                                Aberrant Mitotic Spindles <br /> Early M Phase                                                              Although it appears that Brn1p is required to allow cells to <br /> To determine the time in the cell cycle during which Brn1p                 traverse the cell cycle successfully, brn1 mutants, like top2 <br /> is required, we examined the viability of synchronously <br /> dividing populations of cells. Because brn1 mutants show <br /> only a moderate loss of viability in the first generation time             Table 3. Viability at the restrictive temperature of cells traversing <br /> at the restrictive temperature, we examined cell viability in              specific landmarks in the cell cycle <br /> two &#226;&#8364;&#339;windows&#226;&#8364;&#65533; of the cycle. In each case, exponentially <br /> growing cells were treated with &#226;&#65533;&#163;-factor to arrest the cells in                                     Drug added at initiation of synchronous <br /> G1; the &#226;&#65533;&#163;-factor was then removed, and the cells were                                                             cell cycle <br /> shifted to the restrictive temperature and allowed to begin <br /> cycling. In one sample, the cultures were treated with hy-                 Strain                 None         Hydroxyurea         Nocodazole <br /> droxyurea to stop the cell cycle at early S phase; in a second <br /> sample, the cells were instead treated with nocodazole to                  CH2523 (BRN1&#226;&#171;&#185;)         169%             95%                 92% <br />                                                                            CH2524 (brn1-9)         34%             98%                 37% <br /> allow them to pass through S phase and arrest at G2/M; a <br />                                                                            CH2525 (brn1-16)        48%             98%                 72% <br /> third sample was not treated with drugs and served as a                    CH325 (top2-4)           8%             96%                 93% <br /> control. After 3 h at the restrictive temperature, an aliquot of <br /> each sample was spread on rich medium at the permissive                    Cultures synchronized in G1 with &#226;&#65533;&#163;-factor were divided into ali- <br /> temperature to evaluate the viability of the cells. Because                quots, half of which were incubated at 25&#194;&#176;C and half at 37&#194;&#176;C. At <br /> viability was assessed at the permissive temperature, this                 each temperature, one aliquot was immediately treated with hy- <br /> regimen reveals when the irreversible lethal event occurs                  droxyurea, one was immediately treated with nocodazole, and one <br /> during the cell cycle, regardless of when the actual death of              was left untreated. After 3 h, the cultures were plated at 25&#194;&#176;C for <br /> the cell occurs.                                                           viability, which is expressed as a ratio of the number of colonies <br />    Our results suggest that Barren is required between the                 produced from the 37&#194;&#176;C culture to the number produced by the <br />                                                                            25&#194;&#176;C culture of each type. <br /> beginning of S phase and the beginning of mitosis. When a <br />  <br />  <br /> Vol. 11, April 2000                                                                                                                        1297 <br />  B.D. Lavoie et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Phenotypic characteristics of brn1 mutants at the restrictive temperature. (A) Distribution of cell types. Asynchronous cultures of <br /> strains CH2523 (BRN1) and CH2524 (brn1-9) were shifted to 37&#194;&#176;C for 3 h, fixed, and stained with DAPI to visualize nuclei. Cells were scored <br /> according to bud morphology and position/distribution of nuclear DNA, as indicated. (B) Spindle <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> in large-budded BRN1 and brn1-9 <br /> cells. Large-budded cells from strains grown as described for A were examined for spindle length. Short spindles refers to the 1- to 2-&#226;&#65533;&#174;m <br /> length typical of a metaphase cell, intermediate spindles are 4 &#226;&#8364;&#8220;5 &#226;&#65533;&#174;m in length, and long spindles range from 7 to 15 &#226;&#65533;&#174;m, typical of <br /> anaphase/telophase cells. At least 100 large-budded cells were scored per sample. (C) Nuclear and spindle morphology. Cells were grown <br /> as described for A, fixed, prepared for indirect immunofluorescence of tubulin (see MATERIALS AND METHODS), and stained with DAPI <br /> to visualize DNA. Micrographs were false colored in blue (DNA) and green (anti-tubulin). Arrowheads denote extended spindles in dividing <br /> BRN1 (wild-type) cells in contrast to the thick, intermediate spindles found in the large-budded brn1-9 cells. Note that in the mutant, the <br /> spindle is often associated with only one side of the bilobed DNA mass and/or grossly mispositioned (see DISCUSSION). In addition, the <br /> bilobed nuclei present in BRN1 cells are morphologically distinct from those in the mutant; in general, the BRN1 cells show very little <br /> chromosomal material joining two equal DNA masses, suggesting the rapid progression of anaphase to telophase in BRN1 cells. In the brn1 <br /> mutant, however, anaphase appears to initiate normally but proceeds with difficulty, yielding cells with two connected chromatin masses. <br /> In these early to mid anaphase cells, &#226;&#172;&#381;40% of bilobed nuclear masses showed uneven DAPI staining. DIC, differential interference contrast <br /> microscopy. <br />  <br />  <br />  <br /> and smc2 mutants, do not show a classic cell-division-cycle              through the bud neck and was frequently distributed asym- <br /> phenotype at the restrictive temperature. BRN1 and brn1                  metrically. The nuclear spindles in these cells were intensely <br /> cultures were grown exponentially at 25&#194;&#176;C and then shifted               stained, only partially extended (4 &#226;&#8364;&#8220;5 &#226;&#65533;&#174;m), and predomi- <br /> to 37&#194;&#176;C for 3 h. As expected, the BRN1 strain showed a                   nantly localized at or near the bud neck, although typically <br /> distribution of cell morphologies, representing cells at all             failing to traverse it. That the cells with stretched nuclei <br /> normal stages of the cell cycle (Figure 1A). This distribution           initiate a true anaphase was confirmed by a separate exper- <br /> was also observed with control BRN1 and brn1 cultures that               iment in which hemagglutinin-tagged Pds1p, an inhibitor of <br /> were retained at 25&#194;&#176;C (our unpublished results). In compar-              the metaphase-to-anaphase transition, was observed to be <br /> ison, the brn1 strain at 37&#194;&#176;C was somewhat depleted of cells             depleted (our unpublished results). Surprisingly, 62% of <br /> with small buds. In their place was an increased number of               brn1 cells with bilobed nuclei displayed aberrant spindle <br /> large-budded cells, almost half of which appeared to be in               localization, with tubulin staining localized to only one of <br /> early anaphase.                                                          the two DNA masses. Together, these data suggest that brn1 <br />    To characterize the morphology of the large-budded cells              cells undergo an aberrant anaphase. <br /> more thoroughly, cells from similarly grown cultures were <br /> fixed, prepared for indirect immunofluorescence of tubulin, <br /> and treated with DAPI to stain DNA. More than half of the                Brn1 Mutants Do Not Accumulate Catenated <br /> large-budded BRN1 cells contained two well-separated nu-                 Plasmids or Broken Chromosomes <br /> clei (anaphase and telophase cells) with nuclear microtu-                Although brn1 mutants appear phenotypically distinct from <br /> bules extending along the entire DNA mass (Figure 1, B and               top2 mutants both in the efficiency and morphology of their <br /> C). In contrast, roughly half of the large-budded brn1 cells             arrest and in the loss of viability at the restrictive tempera- <br /> possessed a dumbbell-shaped DNA mass that extended                       ture, the apparent interaction of Barren and DNA topoisom- <br />  <br />  <br /> 1298                                                                                                          Molecular Biology of the Cell <br />                                                                                          Chromosome Condensation Requires BRN1 <br />  <br />  <br /> erase II in Drosophila prompted us to test brn1 mutants for       bation times (our unpublished results; see also Figure 3B). <br /> two phenotypes that are defining characteristics of top2 mu-      This FACS profile differs, however, from that produced by <br /> tants, catenation of circular plasmids and the production of      top2 mutant cells, which suffer chromosome breakage and <br /> broken chromosomes.                                               nondisjunction, thereby producing cells with both &#226;&#172;&#65533;1C and <br />    Because DNA topoisomerase II is the only cellular enzyme       &#226;&#172;&#381;2C DNA contents (Figure 3A) (Holm et al., 1989). Thus, it <br /> capable of disentangling catenated circles of double-             appears that in both brn1 and smc2 mutants incubated at the <br /> stranded DNA, circular plasmids remain catenated after            restrictive temperature, a significant fraction of cells display <br /> replication when top2 strains are incubated at the restrictive    an increase in DNA content. <br /> temperature (Koshland et al., 1985). If Barren is required for       To examine the timing of this phenomenon, we arrested <br /> DNA topoisomerase II activity, as has been suggested in           exponentially growing cells in G1 with &#226;&#65533;&#163;-factor, released <br /> Drosophila, then brn1 mutants might exhibit a similar accu-       them to the restrictive temperature, and then examined <br /> mulation of catenated plasmids when they are incubated at         them by FACS at various times. For the brn1-16 strain, DNA <br /> the restrictive temperature. Wild-type and mutant strains         replication was apparent 1 h after release from &#226;&#65533;&#163;-factor, <br /> containing a 14-kb circular minichromosome were synchro-          which is similar to what is observed with BRN1 cells (Figure <br /> nized in G1 in medium containing &#226;&#65533;&#163;-factor, then released          3B). The initial appearance of &#226;&#172;&#381;2C cells occurred 2 h after <br /> into nocodazole-containing medium at the nonpermissive            release, and the population of &#226;&#172;&#381;2C cells was much more <br /> temperature. The topological state of purified minichromo-        apparent 3 h after release from &#226;&#65533;&#163;-factor (Figure 3B). This cell <br /> somes was then analyzed by agarose gel electrophoresis            type appears to be produced during the second cell cycle <br /> (Figure 2A). As expected, higher-molecular-weight cate-           after release from &#226;&#65533;&#163;-factor, because its production was en- <br /> nated circles accumulate in nocodazole-treated top2 cells. In     tirely prevented by treating the culture with either &#226;&#65533;&#163;-factor <br /> contrast, the brn1 mutants display no change in plasmid           or hydroxyurea 1.5 h after release from the initial &#226;&#65533;&#163;-factor <br /> topology at the restrictive temperature, indicating that brn1     treatment (Figure 3C; our unpublished results). A similar <br /> mutants effectively resolve the catenanes produced during         phenomenon was seen with smc2 mutants, except that <br /> replication, even at the restrictive temperature.                 progress through the cell cycle was somewhat slower with <br />    When full-length chromosomes are segregated in the ab-         this strain. Thus, it appears that brn1 and smc2 cells begin to <br /> sence of DNA topoisomerase II activity, approximately one-        exhibit this unusual staining in the second cell cycle after a <br /> third of chromosomes larger than 300 kb suffer a distinctive      shift to the restrictive temperature. One possibility is that <br /> <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of breakage (Spell and Holm, 1994). If Barren is          this staining pattern reflects aberrant excessive DNA repli- <br /> required for DNA topoisomerase II activity, then brn1 mu-         cation (see DISCUSSION). <br /> tants might be expected to exhibit the same kind of chromo- <br /> some breakage. To test this hypothesis, BRN1, brn1-9, brn1-       Brn1 Mutants Are Defective in Chromosome <br /> 16, and top2-4 strains were grown exponentially at 25&#194;&#176;C, and      Condensation <br /> the cultures were divided and incubated for an additional <br /> 5 h at either 25 or 37&#194;&#176;C. Although a clear smear of broken        The phenotypic similarities between brn1 and smc2 mutants <br /> chromosomes was visible in the lane with DNA from the             suggest that, like smc2 mutants, brn1 mutants may have <br /> top2 culture grown at 37&#194;&#176;C, no such smear was visible in the      defects in chromosome condensation. This predicted pheno- <br /> lanes containing DNA from brn1 strains (Figure 2B). Thus, it      type of brn1 mutants is also suggested from the copurifica- <br /> appears unlikely that Barren is required to activate DNA          tion of Xenopus Barren with the 13S condensin complex <br /> topoisomerase II in vivo.                                         (Hirano et al., 1997). To test this hypothesis, exponentially <br />                                                                   growing brn1 and BRN1 cultures were arrested in G1 with <br />                                                                   &#226;&#65533;&#163;-factor, shifted to the restrictive temperature, and then <br />                                                                   released into nocodazole to allow the chromosomes to attain <br /> Brn1 Mutants Accumulate an Unusual Population                     maximal condensation at the restrictive temperature. Under <br /> of Cells with an Apparent &gt;2C DNA Content at the                  this regimen, Brn1p should be inactivated during both the <br /> Restrictive Temperature                                           establishment and the maintenance of condensation. Chro- <br /> To examine the state of the DNA in brn1 mutants at the            mosome condensation was monitored with the use of FISH <br /> restrictive temperature, we used FACS analysis of cells lack-     with an rDNA probe (Guacci et al., 1994), revealing dramatic <br /> ing mitochondrial DNA. Exponentially growing cultures at          differences among the strains. Chromosome XII in BRN1 <br /> 25&#194;&#176;C were divided and placed at 25 or 37&#194;&#176;C for various            strains condensed as expected to a discrete linear array, but <br /> periods. Surprisingly, there is a broadening and flattening of    brn1 mutants revealed a disorganized rDNA array (Figure 4, <br /> the G2 peak in brn1 cells incubated at the restrictive temper-    A and B). Similarly, an smc2 mutant showed a clear defect in <br /> ature, and many cells appear to have a DNA content &#226;&#172;&#381;2C            rDNA condensation when subjected to this regimen (Figure <br /> (Figure 3A). This phenomenon is not observed in the BRN1          4B) (Strunnikov, personal communication). Thus, Barren, like <br /> control strain, and it is not observed in a variety of temper-    Smc2p, is involved in mitotic chromosome condensation. <br /> ature-sensitive yeast strains incubated at the restrictive tem-      To determine whether Barren function is limited to an <br /> perature. For example, both late-nuclear-division mutants         early stage in chromosome condensation (i.e., as for an <br /> cdc5 and cdc15 and many DNA-replication mutants (e.g.,            assembly factor) or whether ongoing Barren activity is <br /> cdc44/rfc1 and pol30) undergo cell cycle arrest with a uniform    required to maintain mitotic chromosome structure, we in- <br /> 2C DNA content (Howell et al., 1994; Amin and Holm, 1996;         activated Barren after chromosome condensation had oc- <br /> Oh and Holm, unpublished data). Notably, however, a sug-          curred. BRN1 and brn1 cultures were arrested in early mi- <br /> gestion of the same unusual phenomenon occurs when an             tosis with the use of nocodazole at the permissive <br /> smc2 strain is incubated at the restrictive temperature; this     temperature to allow rDNA condensation. The cultures were <br /> phenotype becomes more pronounced after prolonged incu-           then shifted to the restrictive temperature to inactivate Barren <br />  <br />  <br /> Vol. 11, April 2000                                                                                                          1299 <br />  B.D. Lavoie et al. <br />  <br />  <br />                                                                          specifically in mitosis, and the state of the rDNA was moni- <br />                                                                          tored by FISH with the use of an rDNA probe. As expected, <br />                                                                          both BRN1 and brn1 chromosomes were condensed at the <br />                                                                          permissive temperature. In contrast, although the BRN1 chro- <br />                                                                          mosomes remained condensed at 37&#194;&#176;C, rDNA condensation <br />                                                                          was disrupted in the brn1-9 strain (Figure 4C). These data <br />                                                                          indicate that ongoing Barren activity is required for the main- <br />                                                                          tenance of chromosome condensation during mitosis. <br />                                                                             To address whether the condensation defects in brn1 mu- <br />                                                                          tants are limited to repetitive DNA, we examined chromo- <br />                                                                          some condensation in unique sequences on chromosome <br />                                                                          XVI. Exponentially growing brn1 and BRN1 cultures were <br />                                                                          arrested with &#226;&#65533;&#163;-factor, shifted to the restrictive temperature, <br />                                                                          and then released into nocodazole to block the cells in mi- <br />                                                                          tosis. Chromosome condensation was examined by using <br />                                                                          FISH and measuring the distance between two sequences <br />                                                                          that map 255 kb apart on chromosome XVI (Figure 4D). In <br />                                                                          BRN1 nuclei, the average distance between CEN16 proximal <br />                                                                          and distal probes was 0.75 &#226;&#171;&#190; 0.08 &#226;&#65533;&#174;m (experiment 1; Figure <br />                                                                          4D) and 0.77 &#226;&#171;&#190; 0.09 &#226;&#65533;&#174;m (experiment 2; our unpublished <br />                                                                          results), which are similar to values reported previously for <br />                                                                          condensed DNA (Strunnikov et al., 1995a). In contrast, in <br />                                                                          brn1-9 nuclei, the average distance between probes was <br />                                                                          1.07 &#226;&#171;&#190; 0.11 &#226;&#65533;&#174;m (experiment 1) and 0.97 &#226;&#171;&#190; 0.11 &#226;&#65533;&#174;m (experi- <br />                                                                          ment 2); these values are very similar to what is seen in <br />                                                                          uncondensed chromosomes in G1 cells (1.0 &#226;&#171;&#190; 0.01 to 1.1 &#226;&#171;&#190; <br />                                                                          0.1 &#226;&#65533;&#174;m; Strunnikov et al., 1995a). Thus, little or no chromo- <br />                                                                          some condensation occurs on unique DNA sequences in the <br />                                                                          absence of functional Barren protein. <br />                                                                             Because a number of mutations that affect chromosome <br />                                                                          cohesion also cause alterations in chromosome condensation <br />                                                                          (Guacci et al., 1997; Strunnikov, personal communication), <br />                                                                          we tested brn1 mutants arrested in mitosis for precocious <br />                                                                          separation of sister chromatids in the absence of microtu- <br />                                                                          bules. Although chromosome condensation was impaired <br />                                                                          under these conditions, sister chromatid cohesion appeared <br />                                                                          normal when analyzed with a CEN16 probe (Figure 4E) and <br />                                                                          a CEN1 probe (our unpublished results). Thus, an underly- <br />                                                                          ing defect in cohesion is unlikely to be the source of the <br />                                                                          observed condensation defects. Instead, brn1 cells are spe- <br />                                                                          cifically impaired in their ability to establish and/or main- <br />                                                                          tain chromosome condensation at the restrictive tempera- <br />                                                                          ture. Together, these results strongly suggest that Barren <br />                                                                          plays an essential role in chromosome condensation in yeast. <br />  <br />                                                                          DISCUSSION <br /> Figure 2. Neither catenated plasmids nor broken chromosomes              To begin to unravel the biological function of Barren in <br /> are apparent in brn1 strains at the restrictive temperature. (A) Plas-   eukaryotic cells, we used genetic, cytological, and molecular <br /> mid catenation in top2 and brn1 mutants. Strains CH2523 (BRN1),          biological methods to initiate studies of the Barren homo- <br /> CH2524 (brn1-9), CH2525 (brn1-16), CH335 (TOP2), and CH325 <br /> (top2-4), each containing a 14-kb CEN3 ARS1 plasmid (pDK243), <br /> were synchronized in G1 with &#226;&#65533;&#163;-factor, shifted to the restrictive <br /> temperature, and then released into prewarmed medium containing <br /> nocodazole (37&#194;&#176;C) to rearrest cells in mitosis (M). Isolated plasmid <br /> DNA was separated by agarose gel electrophoresis and visualized          Figure 2 (Cont). CH2525 (brn1-16), and CH325 (top2-4) were <br /> by Southern blotting with the use of a pBR322-based probe. S, L, and     grown exponentially at 25&#194;&#176;C, the cultures were divided, and half of <br /> R denote the supercoiled, linear, and relaxed forms, respectively.       each culture was incubated at 25&#194;&#176;C and half at 37&#194;&#176;C for 5 h. Chro- <br /> Catenated circular minichromosomes run as a high-molecular-              mosomes were separated on a CHEF gel, and chromosome V was <br /> weight smear of topoisomers (see top2-4, lane M). The unlabeled          visualized by hybridization with a URA3 probe. The smear of <br /> M-specific band between L and R has been ascribed to two closed          hybridization that lies between the well and the chromosome V <br /> circular intertwined monomers (C forms), as described by Koshland        band is not reproducible. Characteristic fragments of chromosome <br /> and Hartwell (1987). (B) Assay for broken chromosomes in brn1 and        V appear only in strain CH325 (top2-4) at 37&#194;&#176;C. WT, wild type. STD, <br /> top2 mutants. Strains CH2523 (BRN1 TOP2), CH2524 (brn1-9),               chromosomal molecular weight standard. <br />  <br />  <br /> 1300                                                                                                          Molecular Biology of the Cell <br />                                                                                                       Chromosome Condensation Requires BRN1 <br />  <br />  <br /> Figure 3. Mutant brn1 strains <br /> produce cells containing &#226;&#172;&#381;2C <br /> DNA content after two generation <br /> times at the restrictive tempera- <br /> ture. (A) Production of &#226;&#172;&#381;2C cells <br /> in asynchronously dividing cul- <br /> tures. Strains CH2534 (BRN1), <br /> CH2587 (top2-4), CH2535 (brn1-9), <br /> CH2536 (brn1-16), and CH2539 <br /> (smc2-6) were grown exponen- <br /> tially at 25&#194;&#176;C, the cultures were <br /> divided, and one aliquot was in- <br /> cubated at 25&#194;&#176;C and one at 37&#194;&#176;C <br /> for 3 h. At 37&#194;&#176;C, strain CH2587 <br /> (top2-4) produces cells with both <br /> &#226;&#172;&#65533;1C and &#226;&#172;&#381;2C DNA content. At <br /> 37&#194;&#176;C, strains CH2535 (brn1-9), <br /> CH2536 (brn1-16), and CH2539 <br /> (smc2-6) exhibit a broadening of <br /> the 1C peak and a shift of the sec- <br /> ond peak to &#226;&#172;&#381;2C DNA content. <br /> (B) Kinetics of production of &#226;&#172;&#381;2C <br /> cells. Strains CH2534 (BRN1), <br /> CH2536 (brn1-16), and CH2539 <br /> (smc2-6) were grown exponen- <br /> tially at 25&#194;&#176;C and then arrested in <br /> G1 with &#226;&#65533;&#163;-factor. The &#226;&#65533;&#163;-factor was <br /> removed and the synchronous <br /> cultures were allowed to divide at <br /> 37&#194;&#176;C. Cells with &#226;&#172;&#381;2C DNA con- <br /> tent are produced in strains <br /> CH2536 (brn1-16) and CH2539 <br /> (smc2-6) in approximately the sec- <br /> ond generation time after initiation of the cell cycle. In a separate experiment, strain CH2535 (brn1-9) gave substantially similar results. (C) <br /> Preventing a second cell cycle prevents the production of cells with &#226;&#172;&#381;2C DNA content. Strain CH2536 (brn1-16) was induced to traverse a cell cycle <br /> synchronously, as described for B. However, &#226;&#65533;&#163;-factor was added to one sample at 1.5 h after the initial release from &#226;&#65533;&#163;-factor, when most of the cells <br /> had completed DNA replication; a second sample was left untreated as a control. Samples were taken for FACS analysis 3 h after the cells were <br /> initially released from the synchronizing treatment with &#226;&#65533;&#163;-factor. Production of cells with &#226;&#172;&#381;2C DNA content was prevented by preventing the cells <br /> from initiating a second division with the second &#226;&#65533;&#163;-factor treatment. Substantially similar results were obtained when treatment with hydroxyurea <br /> was substituted for the second treatment with &#226;&#65533;&#163;-factor. <br />  <br />  <br /> logue in budding yeast. The Barren product in budding                        the previous analysis of the role of Barren. In Xenopus, <br /> yeast is essential for viability and cell division. The analysis             Barren copurifies with the five-subunit condensin complex, <br /> of heat-sensitive brn1 mutants revealed that cells compro-                   and as such it was inferred to play an important role in the <br /> mised in Barren function exhibit partially elongated chro-                   condensation process (Hirano et al., 1997). Because the con- <br /> mosome masses similar to the anaphase defect observed for                    tribution of individual condensin subunits to the in vitro <br /> barren mutants in Drosophila, which suggests a conserved                     condensation reaction was not addressed, however, the <br /> function for Barren in chromosome transmission between                       physiological significance of Barren&#226;&#8364;&#8482;s association with con- <br /> yeast and flies. This chromosome phenotype is also very                      densin was unclear. That yeast Barren is essential for viabil- <br /> similar to the phenotype of smc2 mutants of budding yeast,                   ity and is required for chromosome condensation in vivo <br /> cut3 and cut14 mutants of Schizosaccharomyces pombe, and                     suggests that it is an integral component of condensin func- <br /> mix1 mutants of Caenorhabditis elegans (Saka et al., 1994;                   tion. Although we do not know the molecular function of <br /> Strunnikov et al., 1995a; Lieb et al., 1998). All of these genes             Barren in the condensin complex, our genetic results provide <br /> encode homologues of components of Xenopus condensin, a                      two important constraints. Because loss of Barren function <br /> complex essential for chromosome condensation in vitro.                      yields uncondensed chromosomes, Barren is likely to be <br /> Furthermore, we directly show that brn1 mutants are defec-                   critical to the core function of condensin rather than to serve <br /> tive in chromosome condensation in vivo, as has been dem-                    as a minor auxiliary factor. Second, the absence of chromo- <br /> onstrated for smc2, cut3, and cut14. Consistent with this                    some condensation when Barren activity is compromised <br /> observation, Barren performs an essential function between                   implies that the loss of Barren function inactivates the cell&#226;&#8364;&#8482;s <br /> early S phase and early M phase, the period of the cell cycle                condensation machinery, i.e., Barren is not a negative regu- <br /> during which condensation is established and maintained.                     latory subunit of condensin. Recent work with the Xenopus <br /> Together, our results reveal that the Barren product of bud-                 condensin complex has shown that, in vitro, the condensin <br /> ding yeast is critical for mitotic chromosome segregation                    complex wraps DNA in a right-handed manner (Kimura <br /> and condensation.                                                            and Hirano, 1997; Kimura et al., 1999). That this reaction <br />    The demonstration that yeast Barren functions in vivo in                  requires phosphorylated Barren and/or XCAP-D2 and/or <br /> chromosome condensation both corroborates and extends                        XCAP-G suggests that some or all of these proteins play a <br />  <br />  <br />  <br />  <br />  <br />  <br />                                                                                  Figure 4. Chromosome condensation and chro- <br />                                                                                  mosome cohesion in brn1 mutants at the restric- <br />                                                                                  tive temperature. (A) FISH analysis of rDNA mor- <br />                                                                                  phology in brn1 mutants. Exponentially growing <br />                                                                                  strains CH2523 (BRN1) and CH2524 (brn1-9) were <br />                                                                                  synchronized in G1, shifted to the nonpermissive <br />                                                                                  temperature (37&#194;&#176;C), and released into medium <br />                                                                                  containing nocodazole (37&#194;&#176;C) to rearrest the cells <br />                                                                                  in mitosis. Cells were fixed and processed for <br />                                                                                  FISH with the use of DIG-labeled rDNA as the <br />                                                                                  probe with FITC-conjugated secondary antibodies <br />                                                                                  (see MATERIALS AND METHODS). Micro- <br />                                                                                  graphs show the rDNA signal in green and the <br />                                                                                  bulk DNA (propidium iodide stain) in red (false <br />                                                                                  colors). (B) Quantitation of rDNA morphology. <br />                                                                                  Strains CH2523 (BRN1), CH2524 (brn1-9), CH2525 <br />                                                                                  (brn1-16), and 1aAS330 (smc2-8) were treated as <br />                                                                                  described for A. Amorphous rDNA signals were <br />                                                                                  scored as decondensed, and clear loop or <span id='am-1' about='xsd:string' typeof='owl:Thing'>string</span>- <br />                                                                                  like structures were scored as condensed. In each <br />                                                                                  sample, &#226;&#172;&#381;100 nuclei were counted. (C) Mainte- <br />                                                                                  nance of rDNA condensation. Strains CH2523 <br />                                                                                  (BRN1) and CH2524 (brn1-9) were arrested in <br />                                                                                  early mitosis in medium containing nocodazole <br />                                                                                  for 2 h at the permissive temperature (23&#194;&#176;C) to <br />                                                                                  allow chromosome condensation to occur. The <br />                                                                                  cultures were then shifted to the restrictive tem- <br />                                                                                  perature (37&#194;&#176;C) for 1 h. The state of rDNA con- <br />                                                                                  densation was monitored by FISH as described <br />                                                                                  for A both before (23&#194;&#176;C) and after the shift to <br />                                                                                  restrictive temperature (37&#194;&#176;C). Scoring of samples <br />                                                                                  was as described for B. More than 100 nuclei were <br />                                                                                  scored per sample. (D) Chromosome condensa- <br />                                                                                  tion of unique chromosome XVI sequences. Cells <br />                                                                                  were prepared as described for A and probed by <br />                                                                                  FISH with the use of a CEN16 proximal probe and <br />                                                                                  a second chromosome XVI probe 250 kb away (see <br />                                                                                  MATERIALS AND METHODS). Distances be- <br />                                                                                  tween FISH signals are shown as the percentage <br />                                                                                  of total measurements in each category among <br />                                                                                  nuclei exhibiting two signals. The average dis- <br />                                                                                  tance between signals is 0.75 &#226;&#171;&#190; 0.08 &#226;&#65533;&#174;m (n &#226;&#171;&#189; 83) <br />                                                                                  for BRN1 and 1.07 &#226;&#171;&#190; 0.11 &#226;&#65533;&#174;m (n &#226;&#171;&#189; 101) for brn1-9. <br />                                                                                  Nuclei with greater or fewer than two signals <br />                                                                                  were not included in the analysis. (E) FISH anal- <br />                                                                                  ysis of chromosome cohesion. Nocodazole-ar- <br />                                                                                  rested cells (37&#194;&#176;C) prepared as described for A <br />                                                                                  were analyzed with a CEN16 proximal probe, and <br />                                                                                  the number of signals per nucleus was scored. At <br />                                                                                  least 100 nuclei were counted per sample. <br />  <br />  <br /> role in the activation of condensin (Kimura et al., 1998).       chromosome condensation, probably through inactivation <br /> Consistent with this suggestion, anti-Eg7 (XCAP-D2) anti-        of the cellular condensation machinery. Because chromo- <br /> bodies were used to demonstrate a role for this subunit in       some condensation is a reversible process that is initiated in <br /> the in vitro condensation assay (Cubizolles et al., 1998).       early M and reversed in late M/early G1, the regulation of <br /> Additional experiments will be needed to assess whether          condensin activity through Brn1p could provide an effective <br /> Barren functions as a regulatory switch or as a direct medi-     mechanism for both condensing and decondensing DNA. <br /> ator of altering chromosome topology.                               An intriguing question is whether both repetitive and <br />    In vivo, ongoing Barren activity is required to maintain      unique DNA condense through similar mechanisms. Al- <br /> mitotic chromosome structure. The most simple interpreta-        though previous work with smc2 mutant cells failed to show <br /> tion of this result is that Barren plays a structural role in    a defect in rDNA condensation (Strunnikov et al., 1995a), this <br /> higher-order chromosome organization. Alternatively, the         was most likely due to a less stringent inactivation regimen <br /> maintenance of chromosome condensation could be a highly         (see Figure 4B). We now show that both Smc2p and Brn1p <br /> dynamic process involving a constant reestablishment of the      are required for the condensation of repetitive as well as <br /> condensed state. Regardless of the mechanism, it is clear that   unique DNA sequences. These data are consistent with a <br /> inactivation of Brn1p in early M causes an untimely loss of      model whereby chromosome compaction of both repetitive <br />  <br />  <br />  <br />  <br /> and unique DNA sequences occurs through a common mo-             unpublished results). It is unlikely that this phenotype is <br /> lecular machinery and thus through a conserved mecha-            caused by some aberrant binding of propidium iodide to the <br /> nism. Although defects in rDNA condensation are more             uncondensed DNA, because a sharp 2C peak is observable <br /> dramatic by FISH than those seen with unique DNA probes,         in both mutants in the first M phase. Instead, the &#226;&#172;&#381;2C FACS <br /> this difference may reflect the reiterative nature of the rDNA   profiles are reminiscent of FACS profiles produced by a cdc6 <br /> array and/or a change in the relative distribution of con-       mutant that displays promiscuous initiation of DNA repli- <br /> densin proteins. Indeed, indirect immunofluorescence of          cation (Liang and Stillman, 1997). This observation brings up <br /> Smc4p and Ycd2p, yeast homologues of two additional con-         the interesting possibility that improper condensation of the <br /> densin subunits, suggests that they are enriched on the          chromosomes in the first cell cycle subsequently leads to <br /> rDNA (Strunnikov, personal communication; Zheng and              inappropriate initiation of DNA replication. Because the <br /> Koshland, unpublished results).                                  prereplication complex is formed at the M-to-G1 transition <br />    It is noteworthy that condensation of both unique and         (Diffley et al., 1994), one possibility is that its normal assem- <br /> repetitive DNA is perturbed in the first mitosis after Brn1p     bly is affected by the condensation state of the chromosomes. <br /> inactivation. At this time, topological links between sisters       A second unexpected phenotype of brn1 mutants is the <br /> are resolved, as judged by the absence of catenated circular     absence of spindle and chromosome colocalization. One pos- <br /> minichromosomes and chromosome V breaks. Thus, con-              sible explanation is that decondensation results in intertwin- <br /> densation does not appear to be obligatory in driving out        ing that causes detachment of chromosomes from the spin- <br /> DNA tangles, as has been proposed previously (Wood and           dle when the cell enters anaphase. This possibility seems <br /> Earnshaw, 1990). Rather, anaphase itself may be sufficient to    unlikely, however, because topological links and chromo- <br /> promote DNA untangling by topoisomerase II (Holm et al.,         some breaks do not accumulate in brn1 mutants. Further- <br /> 1985; Holm, 1994). However, a modest delay in anaphase is        more, top2 mutants complete anaphase by breaking chromo- <br /> detectable in brn1 mutants at the restrictive temperature (our   somes, suggesting that kinetochore-spindle attachments <br /> unpublished results), suggesting that the kinetics of chro-      resist a challenge from topologically linked DNA (Holm et <br /> mosome untangling may be facilitated by condensation, and        al., 1989). A more attractive model is that Barren (or the <br /> this effect may be particularly important for the larger chro-   chromosome condensation it maintains) plays a novel role in <br /> mosomes of higher eukaryotes.                                    centromere structure and/or function. In support of this <br />    Although a condensation defect is the earliest observable     idea, cytological observations of mitotic chromosomes sug- <br /> phenotype of brn1 mutants, there is a later appearance of        gest an unusual condensed state of centromeric regions (Rat- <br /> additional phenotypes, including an anaphase failure that is     tner and Lin, 1985). In addition, mutations in Cep3p, an <br /> cytologically similar to DNA topoisomerase II mutants. Al-       established kinetochore component, result in a similar dis- <br /> though earlier in vitro analysis of Drosophila Barren sug-       placement of the chromosomes from the spindle, presum- <br /> gested that it might serve as an activator of topoisomerase II   ably resulting from a failure of the chromosomes to maintain <br /> function, our results suggest that yeast Barren is probably      a stable attachment to the dynamic spindle (Strunnikov et al., <br /> not an essential activator of topoisomerase II in vivo. Be-      1995b). Furthermore, preliminary evidence suggests that <br /> cause topoisomerase II is the only enzyme known to disen-        Brn1p genetically interacts with a subset of kinetochore pro- <br /> tangle catenated circular plasmids, a failure by the mutant      teins (Lavoie and Koshland, unpublished observation). It <br /> Barren protein to activate topoisomerase II should have led      will be interesting to determine whether this putative role <br /> to the accumulation of catenanes. In brn1 mutants at the         for Barren at the centromere reflects a general role for con- <br /> restrictive temperature, however, catenanes are efficiently      densin proteins. However, neither smc2 mutations nor mu- <br /> resolved and chromosomes are segregated without break-           tations affecting the homologue of XCAP-G cause an accu- <br />                                                                  mulation of mid anaphase nuclei (Lavoie and Koshland, <br /> age. Nonetheless, it remains possible that Barren could serve <br />                                                                  unpublished observation). Thus, it is tempting to speculate <br /> to regulate topoisomerase II activity in a more subtle man- <br />                                                                  that specific cellular processes may be differentially sensitive <br /> ner. Barren may serve as an inhibitor of topoisomerase II <br />                                                                  to defects in the functions of condensin proteins. Additional <br /> activity in vivo, as was observed in vitro when Barren was <br />                                                                  study will be necessary to determine the precise functional <br /> present at high concentrations. Cytological evidence sug- <br />                                                                  roles of these proteins, which play a pivotal role in the <br /> gests that Barren and topoisomerase II reside in close prox- <br />                                                                  condensation of chromosomes. <br /> imity in condensed and condensing chromosomes (Bhat et <br /> al., 1996). Perhaps when associated with chromosomes they        Note added in proof. In the accompanying paper by Ouspenski et al., <br /> are members of a large multiprotein complex and perform          brn1 mutants do not exhibit excess DNA replication during the time <br /> their functions simultaneously but independently.                course of the FACS experiment. This disparity probably reflects the <br />                                                                  roughly twofold difference in doubling times of the strains used by <br />    An unexpected phenotype of cells defective for Barren         the two laboratories. <br /> function is an accumulation of cells containing &#226;&#172;&#381;2C DNA <br /> content. This phenotype may be general for chromosome <br /> condensation mutants, because smc2 mutants also produce          ACKNOWLEDGMENTS <br /> &#226;&#172;&#381;2C cells. When Barren or Smc2p is inactivated at the be- <br /> ginning of the cell cycle, chromosome condensation is per-       We are indebted to A. Strunnikov for providing the 1aAS330 <br /> turbed at the time of the first mitosis (Figure 4; our unpub-    (smc2-8) strain and for communication of unpublished results. We <br />                                                                  thank A. Philp and members of our laboratories for critical reading <br /> lished results). In contrast, the appearance of cells with &#226;&#172;&#381;2C   of the manuscript. B.D.L. and K.M.T. contributed equally to this <br /> DNA content occurs much later (Figure 3). In fact, the ap-       work. This work was supported by grants GM36510 (to C.H.) and <br /> pearance of &#226;&#172;&#381;2C cells can be entirely abolished if the brn1      GM41718 (to D.K.) from the National Institutes of Health. B.D.L. is <br /> cells are prevented from proceeding through a second cell        the recipient of a Centennial Fellowship from the Medical Research <br /> cycle with either &#226;&#65533;&#163;-factor or hydroxyurea (Figure 3C; our        Council of Canada. <br /> </body></html>