<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>GAGA factor binding <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> DNA via a single trinucleotide <br /> sequence element <br />                                          see 4). On the hsp26 gene, the GA-rich sequences that bind GAF <br />                                                                             have been shown to be important for the maintenance of DNase I <br /> GAGA transcription factor (GAF) is an essential                             hypersensitive sites in the promoter, sites which are subsequently <br /> protein in Drosophila, important for the transcriptional                    critical for heat-induced and developmental expression of hsp26 <br /> regulation of numerous genes. GAF binds to GA                               (8&#226;&#8364;&#8220;10). It therefore appears that GAF can keep the promoter in a <br /> repeats in the promoters of these genes via a DNA-                          chromatin conformation which presumably allows for the <br /> binding domain containing a single zinc finger. While                       association of additional factors necessary for transcriptional <br /> GAF binding sites are typically composed of 3.5 GA                          competence. GAF elements have also been shown to be critical <br /> repeats, the Drosophila hsp70 gene contains much                            in the regulation of the hsp70 gene. Mutations in GAF elements <br /> smaller elements, some of which are as little as three                      in the promoter of hsp70 have been shown in vivo to specifically <br /> bases (GAG) in <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>. Interestingly, the binding of                        affect the formation of promoter-paused RNA polymerase II (pol <br /> GAF to more distant trinucleotide elements is relatively                    II), the accessibility of heat shock factor (HSF) to its target <br /> strong and not appreciably affected by the removal of                       sequences and transcriptional activation (11,12). Further analysis <br /> larger GA arrays in the promoter. Moreover, a simple                        of hsp70 sequences indicates that recombinant GAF can disrupt <br /> synthetic GAG sequence is sufficient to bind GAF in                         nucleosomes on in vitro assembled templates (13), and can <br /> vitro. Here we directly compare the affinity of GAF for                     facilitate the &#226;&#8364;&#732;remodeling&#226;&#8364;&#8482; of chromatin in an ATP-dependent <br /> different sequence elements by immunoprecipitation                          fashion with the help of a remodeling complex termed NURF <br /> and gel mobility shift analysis. Furthermore, our                           (14). It is therefore reasonable to conclude that GAF stimulates <br /> measures of the concentration of GAF in vivo indicate                       gene expression by opening chromatin and maintaining the <br /> that it is a highly abundant nuclear protein, prevalent                     promoter in a conformation which sets the stage for the binding <br /> enough to occupy a sizable fraction of correspondingly                      of other sequence-specific factors and pol II. <br /> abundant trinucleotide sites.                                                  GAF does not appear to limit its interactions to the promoter. <br />                                                                             In vivo crosslinking on hsp70 and hsp26 indicates that GAF <br /> INTRODUCTION                                                                association is restricted to promoter sequences prior to heat shock, <br />                                                                             but that after induction it associates with sequences throughout <br /> The repressive nature of chromatin has been demonstrated, and               the body of the gene (15). Interestingly, within the gene itself <br /> those factors capable of alleviating this repression have become            there are very few large GA arrays, which are characteristic of <br /> a major target of investigation (for review see 1). Not only must           GAF regulatable promoters; however, there are several smaller <br /> a mechanism exist to assemble nucleosomes on genes and                      elements of 3&#226;&#8364;&#8220;5 nucleotides which occur every 75 bases on <br /> facilitate the condensation and formation of higher order                   average (R.C.Wilkins and J.T.Lis, unpublished observation). The <br /> chromatin structures, but a means to selectively disrupt chromatin          nature of the association and the function of GAF over the length <br /> is needed to predicate the subsequent expression of these genes.            of the transcriptional <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> therefore remains an open question. <br /> One factor that plays an important role in promoter architecture               Originally, according to sequence comparison, GAF elements <br /> in Drosophila is the GAGA transcription factor (GAF). Originally            were believed to be comprised of &#226;&#710;&#188;3.5 (GA) repeats (5). <br /> identified by virtue of its ability to stimulate transcription <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span>         However, the binding domain of GAF contains a single zinc <br /> the engrailed (en) and ultrabithorax (Ubx) promoters in vitro               finger and may require no more than a single trinucleotide <br /> (2,3), GAF binds to GA-rich sequences, (CT&#194;&#183;GA)n, found in the               sequence for GAF specific binding, given that most zinc finger <br /> promoters of numerous Drosophila genes (for reviews see 4,5).               peptides require only a three <span id='am-84' about='xsp:base' typeof='owl:Thing'>base</span> pair consensus per finger <br /> Interestingly, GAF appears to exert its effect not by activating or         (16&#226;&#8364;&#8220;18). Recently, Omichinski et al. (19) have published the <br /> repressing the transcriptional machinery directly, but by mitigat-          NMR structure of the GAF DNA binding domain bound to a <br /> ing the repressive effects of histones (6,7).                               pentamer sequence (GAGAG). The zinc finger itself is responsible <br />    GAF has specifically been shown to play a critical role in the           for base contacts at the first three nucleotides, and basic regions <br /> expression of the well-characterized heat shock genes (for review           flanking the finger, BR1 and BR2, contact the last two bases of <br />  <br />  <br /> *To whom correspondence should be addressed. Tel: +1 607 255 2442; Fax: +1 607 255 2428; Email: jtl10@cornell.edu <br />                                                                                                              2673 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1998,Vol.  26,No. <br />                                                                                                               Vol.22,  No.111        2673 <br />  <br /> the pentamer, indicating that a GAGAG pentamer may indeed be            reaction (21). Approximately 30 fmol of gel purified, end labeled <br /> the consensus binding site for GAF (19). Interestingly, most            DNA was incubated in a 25 &#194;&#181;l binding reaction with His-GAF <br /> single base pair substitutions within this pentamer, except the         protein in GAGA binding buffer (GBB): 80 mM KCl, 10 mM <br /> central G, only moderately affect the affinity of GAF for these         HEPES pH 8.3, 5 mM MgCl2, 10% Glycerol, 1 mM DTT, 0.1% <br /> sequences.                                                              NP40. The upper level of His-GAF used in all footprinting <br />    Here we examine the interaction of GAF with both native and          experiments was 75 ng, and was sequentially diluted 3-fold for all <br /> synthetic binding sites, and given the predicted prevalence of low      subsequent lanes. After a 30 min incubation at room temperature, <br /> affinity GAF elements in the genome, we also examine the                samples were treated with 1 &#194;&#181;l of a 20 &#194;&#181;g/ml solution of DNase <br /> abundance of GAF in the nucleus. While GAF strongly interacts           I (Worthington) for 1 min at room temperature. The reaction was <br /> with its known high affinity binding sites, it also interacts more      stopped by adding 30 &#194;&#181;l 50 mM EDTA and 1% SDS; the DNA <br /> extensively with lower affinity sequences in the hsp70 promoter,        was phenol&#226;&#8364;&#8220;chloroform extracted and ethanol-precipitated, and <br /> around the start site, and into the beginning of the transcribed        run on a 6% polyacrylamide&#226;&#8364;&#8220;urea sequencing gel. <br /> region. Low affinity elements as small as GAG trinucleotides <br /> appear to be targets of GAF binding as assayed by DNase I <br /> footprinting. Filling of more distant minimal sites appears largely     In vitro immunoprecipitation <br /> uncooperative with the occupancy of other GAF binding sites. In <br />                                                                         In vitro immunoprecipitations were done using identical levels of <br /> addition, a trinucleotide sequence is sufficient to confer GAF <br />                                                                         His-GAF protein and the hsp70 promoter fragments described for <br /> binding to synthetic sites with an affinity only a few fold less than <br />                                                                         DNase I footprinting. Binding reactions were for 30 min on ice <br /> that of a larger &#226;&#8364;&#732;high affinity&#226;&#8364;&#8482; site. Measurements of GAF <br />                                                                         in GBB, at which point GAF/Zn+Q specific antibody (15) was <br /> abundance indicate that GAF is plentiful, and is present at a <br />                                                                         added and incubated for an additional 1 h at 4 C. Complexes <br /> concentration in the nucleus sufficient to bind minimal trinucleo- <br />                                                                         were then precipitated with Protein A&#226;&#8364;&#8220;Sepharose beads (Sigma) <br /> tide sites in vivo. <br />                                                                         pre-blocked with salmon sperm DNA. The beads were then <br />                                                                         washed four times with GBB, and the DNA was extracted by <br /> MATERIALS and METHODS                                                   boiling the samples in 1% SDS and analyzed on a 6% <br />                                                                         polyacrylamide&#226;&#8364;&#8220;urea sequencing gel. <br /> Purification of GAF fusion proteins <br /> Histidine-tagged GAGA protein (His-GAF) was overproduced                Gel mobility shifts <br /> and purified from Escherichia coli BL21/DE3pLysS cells using <br /> His-bind resin (Novagen) according to the manufacturer&#226;&#8364;&#8482;s                Probes for band-shifts were generated by kinasing the indicated <br /> recommendations. Cells were grown to an OD600 between 0.5               oligonucleotides with [&#206;&#179;-32P]ATP. The complimentary unlabeled <br /> and 1.0, and induced by addition of 1 mm IPTG (Sigma). Cells            oligo was mixed with the kinased oligo and annealed by heating <br /> were grown for an additional 3 h and harvested by centrifugation.       the mix to 95 C for 5 min., cooling to room temperature in a 75 C <br /> Prior to use, purified product was dialyzed into buffer containing      heat block, and finally cooling to 4 C on ice. Approximately <br /> 100 mM NaCl, 10 mM Tris pH 8, 10% glycerol, 0.1 mM EDTA                 30 fmol of double-strand probe was incubated for 30 min at room <br /> and 1 mM DTT. The His-GAF fusion protein was coded for by               temperature in a 25 &#194;&#181;l binding reaction with the indicated protein. <br /> the p10His-GAF plasmid which contains amino acids 5&#226;&#8364;&#8220;519 of              As with footprinting and immunoprecipitation experiments, <br /> GAF fused in frame to the 10 histidine residues of pET16b. This         binding reactions were in GBB with a maximal protein mass of <br /> construct was generated by ligating an NdeI&#226;&#8364;&#8220;HindIII fragment            75 ng. In addition, 50 ng poly(dG&#226;&#8364;&#8220;dC) was added to gel shift <br /> from the pAR-GAGA plasmid (20) to the pET16b vector treated             experiments to reduce non-zinc-finger specific binding. Samples <br /> with the same enzymes.                                                  were run out for analysis on a 1% agarose gel in 1&#195;&#8212; TBE buffer. <br />   The maltose binding protein&#226;&#8364;&#8220;zinc finger fusion protein (MBP&#226;&#8364;&#8220; <br /> Zn) was overproduced and purified from E.coli BL21 cells using <br />                                                                         Determination of relative binding constants <br /> Amylose resin (NEB) according to the manufacturer&#226;&#8364;&#8482;s recommen- <br /> dations. Cells were grown, induced and harvested as above. The          The relative binding constants of the various DNA fragments <br /> MBP&#226;&#8364;&#8220;Zn fusion protein was coded for by the pMal-Zn plasmid              analyzed were determined as described previously (22,23). <br /> which contains amino acids 244&#226;&#8364;&#8220;407, inclusive of the zinc finger        Typically, oligonucleotides were mixed together with a limiting <br /> and the basic regions BR1 and BR2 of the GAF, generated by              amount of purified MBP&#226;&#8364;&#8220;Zn or His-GAF in a single binding <br /> dropping a BamHI&#226;&#8364;&#8220;SalI cut PCR product into the pMalC2 vector            reaction as described above (&#226;&#710;&#188;10 ng of either protein was used). <br /> (NEB). The PCR product included bases 1&#226;&#8364;&#8220;1220 of GAF, and was            Bound and unbound DNA species were separated by electro- <br /> generated with the following oligos: RCW3 (5&#226;&#8364;&#178;-GGCGGGAATT-               phoresis on a 1% TBE agarose gel and eluted by crushing the gel <br /> CATGTCGCTGCCAATGAACTCGCTG-3&#226;&#8364;&#178;) and Znex (5&#226;&#8364;&#178;-TCC-                         slice in Gilbert&#226;&#8364;&#8482;s buffer and incubating overnight. Samples were <br /> ATCACTGTCGACTGTGTTGTCGCCCTC-3&#226;&#8364;&#178;).                                        analyzed side-by-side on a 10% denaturing acrylamide gel. Gels <br />                                                                         were analyzed on a Molecular Dynamics STORM 840, and <br /> In vitro footprinting                                                   species were quantitated using the Molecular Dynamics Image- <br />                                                                         Quant&#239;&#163;&#170; program. Relative binding constants of different oligo- <br /> Preparation of end-labeled DNA for DNase I footprinting was             nucleotides n and m can therefore be calculated by using the <br /> performed by PCR-amplification of fragments from plasmids               equation Krelative = Kn/Km = [Cn/Dn]/[Cm/Dm], where C is the <br /> containing the hsp70 gene with primers described previously             concentration of labeled DNA in the bound fraction and D is the <br /> (21). Either the top or bottom DNA strand was 32P-labeled by            concentration of the same labeled species in the unbound fraction <br /> kinasing the appropriate strand primer before the amplification         (22,23). <br />  2674 Nucleic Acids Research, 1998, Vol. 26, No. 11 <br />  <br /> Western analysis <br /> Purified His-GAF and S2 nuclear proteins were size-fractionated <br /> by electrophoresis on 10% SDS&#226;&#8364;&#8220;polyacrylamide gels. Nuclei <br /> from Drosophila S2 cells were prepared as described previously <br /> (24). Gels were electroblotted onto nitrocellulose membranes, <br /> probed with GAF specific antibodies (15) and detection was <br /> carried out using an ECL detection reagent kit (Amersham). <br /> Protein species in nuclei samples were quantified by comparison <br /> to known amounts of recombinant GAF protein, that itself was <br /> quantified by comparison to BSA standards by Coomassie and <br /> OD260. <br />  <br /> Immunodepletion <br /> Nuclei from Drosophila Kc cells were prepared as described <br /> previously (24) and extract was prepared by sonication. The <br /> extract was incubated on a rotary wheel for 1 h at 4 C in the <br /> presence or absence of GAF specific antibody. Antibody&#226;&#8364;&#8220;protein <br /> complexes were then precipitated by centrifugation after incuba- <br /> tion for an additional 30 min at 4 C with protein A&#226;&#8364;&#8220;Sepharose <br /> beads. The remaining supernatant was size-fractionated by <br /> electrophoresis on 10% SDS&#226;&#8364;&#8220;polyacrylamide gels and analyzed <br /> by silver staining or western blotting with an ECL detection <br /> reagent kit (Amersham). <br />  <br /> RESULTS <br /> GAF protects trinucleotides on the hsp70 promoter <br /> GAF has been shown to interact with a variety of different            Figure 1. GAF protein binding to high and low affinity sites on the hsp70 gene <br /> sequence targets on the hsp70 gene (15). Here we examine in           as assayed by DNase I digestion. Lanes 1 and 7 are the DNase I patterns in the <br />                                                                       absence of protein on the top and bottom strands of hsp70 respectively. Lanes <br /> detail the affinity of GAF for different promoter sequences using     2&#226;&#8364;&#8220;6 and 8&#226;&#8364;&#8220;12 are DNase I digestions performed in the presence of 3-fold <br /> a His-GAF construct coding for residues 5&#226;&#8364;&#8220;519 of GAF. This            decreases in the concentration of purified His-GAF protein. High affinity GAF <br /> recombinant source lacks bulky fusion moieties and includes the       sites are delineated with solid black boxes, and low affinity GAF sites are <br /> entire GAF sequence, except the first five amino acids which are      indicated by gray boxes, corresponding regions of protection are shown as open <br /> dispensible for DNA binding (25). This recombinant protein is         ovals and filled in circles, respectively. The location of HSEs and the TATA box <br />                                                                       are shown as gray stippled and black stripped boxes respectively. <br /> stable and easily purifiable by its histidine tag as a full length <br /> product (Fig. 4A). <br />    Figure 1 shows GAF protection of hsp70 sequences from              bases in length, with a subset of these comprised of GAG <br /> DNase I digestion as a function of protein concentration. At low      trinucleotides. The presence of low affinity sites within these key <br /> levels of GAF, high affinity sites within the promoter are readily    regulatory regions suggests their possible importance for factor <br /> protected, as represented by open ovals. As protein concentration     accessibility. When compared with the frequency of sites within <br /> increases, this protection spreads to sequences flanking these        the gene itself, the density of low affinity sites around the <br /> strong sites and additional distinct sites of protection become       promoter is significantly higher. <br /> evident, as indicated by closed circles in Figure 1. Spreading of        When sequences at weak and strong GAF binding sites are <br /> the GAF footprint is more prominent on the bottom stand of            compared, it is found that strong sites contain GA repeats of 5 <br /> hsp70 (Fig. 1), where it can be seen to extend from the lower         bases or longer, whereas weaker sites contain a simpler GAG <br /> affinity sites flanking the TATA box down towards the transcrip-      motif. At the highest concentration of GAF, all GAG sequences <br /> tion start site, consistent with recent reports of GAF binding in     in the fragment are occupied. Since the concentration of the DNA <br /> this region (26). In contrast, at higher GAF concentrations the top   used in these experiments is extremely low relative to that of <br /> strand of hsp70 shows several regions of hypersensitivity. These      GAF, we were able to estimate the apparent dissociation constant <br /> regions of increased accessibility to DNase I, indicated by arrows,   as equal to the protein concentration giving half occupancy of a <br /> also map to key regions of the promoter. Hypersensitive bands are     particular site. The difference in affinities of GAF for strong and <br /> seen between GAF binding sites in and around the TATA box,            weak sites is &lt;10-fold, with high affinity sites having a Kd of &#226;&#710;&#188;1.3 <br /> heat shock elements (HSEs) and the start site of transcription.       &#195;&#8212; 10&#226;&#8364;&#8220;9 M. <br /> This hypersensitivity appears to be strand-specific, as no such <br /> hyper-reactive sequences are seen on the bottom strand with           Occupancy of trinucleotides appears to be independent <br /> increasing GAF concentrations. Interestingly, regions of hyper-       of high affinity sites <br /> sensitivity on the top strand of hsp70 and extended protection on <br /> the bottom strand coincide with the presence of low affinity sites.   For many DNA binding proteins, the filling of low affinity protein <br /> Sequence inspection of these sites reveals GA repeats of only 3&#226;&#8364;&#8220;4     binding sites is highly cooperative and dependent upon nearby <br />                                                                                                                             2675 <br />  <br />                                                                                               Nucleic Acids <br />                                                                                               Nucleic       Research,1994, <br />                                                                                                       Acids Research, 1998,Vol.  26,No. <br />                                                                                                                             Vol.22,  No.111          2675 <br />  <br />  <br />                  A                                                                    sites at higher concentrations with a &lt;10-fold difference in <br />                                                                                       affinity when compared with high affinity sites, in agreement <br />                                                                                       with DNase I footprinting results for these fragments. <br />  <br />                                                                                       Relative affinity of GAF for sites of varying length <br />                                                                                       The NMR structure of the GAF DNA binding domain complexed <br />                                                                                       with DNA shows base contacts over an entire GAGAG pentamer <br />                                                                                       sequence. The zinc finger itself is responsible for base contacts <br />                                                                                       over the first three nucleotides, with basic regions flanking the <br />                                                                                       finger (BR1 and BR2) contacting the last two bases of the <br />                                                                                       pentamer (19). Most single base pair substitutions within this <br />                                                                                       pentamer, except the central guanine, only moderately affect the <br />                                                                                       affinity of GAF for these sequences, perhaps because the zinc <br />                                                                                       finger contacts alone are sufficient for GAF specific binding, and <br />                                                                                       therefore only require a single intact GAG. <br />                                                                                          To determine if a trinucleotide was indeed sufficient for GAF <br />                                                                                       specific binding, several synthetic oligos were generated (Fig. <br />                                                                                       3A) and their affinities for an MBP fusion with the zinc finger <br />                                                                                       binding domain of GAF (MBP&#226;&#8364;&#8220;Zn) were analyzed by band-shift <br />                                                                                       analysis (Fig. 3B). Sequences for oligos were derived from actual <br />                                                                                       hsp70 sequences, with minor changes. MBP&#226;&#8364;&#8220;Zn binds quite <br />                                                                                       strongly to a high affinity site [(GA)4], moderately to a pentamer, <br />                                                                                       and less so to a trinucleotide (GAG), but is nevertheless still <br />                                                                                       capable of binding all three. When the trinucleotide is changed to <br />                                                                                       TGT or AGA, GAF binding is undetectable. <br />   B                                                                                      In order to more quantitatively assess GAF sequence require- <br />                                                                                       ments for binding, the relative affinities of GAF for a high affinity <br />                                                                                       site [(GA)4], a pentamer consensus site (GAGAG) and a <br />                                                                                       trinucleotide site (GAG) were examined directly (Fig. 3C). The <br />                                                                                       method of Liu-Johnson et al. (22) was used to determine the <br />                                                                                       relative binding affinities within the context of a single binding <br />                                                                                       reaction in the presence of either limiting His-GAF or the GAF <br /> Figure 2. (A) GAF binding to low affinity sites on hsp70 in the presence and          DNA binding domain fusion MBP&#226;&#8364;&#8220;Zn. Bound and free DNA <br /> absence of high affinity sites in cis. Lanes 1 and 6 are the DNase I <span id='am-85' about='xsp:pattern' typeof='owl:Thing'>pattern</span> in the   sequences were separated by electrophoresis, as in a standard <br /> absence of protein on the bottom strand of hsp70. Lanes 2&#226;&#8364;&#8220;5 and 7&#226;&#8364;&#8220;10 are <br /> DNase I digestions performed in the presence of 3-fold decreases in the               band-shift, and the DNAs were eluted and resolved on a <br /> concentration of purified His-GAF protein on the intact hsp70 promoter, and the       denaturing sequencing gel for quantitation (Fig. 3D). The affinity <br /> same fragment digested with SalI prior to GAF addition. Site of digestion with        of either GAF source for high [(GA)4] and low affinity (GAG) <br /> SalI is indicated. (B) Immunoprecipitation (IP) of high and low affinity sites of     sites differed by 3.5-fold, with the pentamer consensus <br /> hsp70 digested with SalI. Lanes 2&#226;&#8364;&#8220;4 and 6&#226;&#8364;&#8220;8 represent the fragments found in <br /> the supernatant and pellet of IPs in the presence of increasing concentrations of <br />                                                                                       (GAGAG) having an intermediate affinity, only 1.5-fold greater <br /> His-GAF. Lane 1 is the supernatant of a protein-free control, and lane 5 is the       than that of a trinucleotide site alone (GAG). <br /> pellet. IPs were performed using a GAF specific polyclonal antibody to the <br /> C-terminal two-thirds of the protein [Zn+Q (15)]. High and low affinity site          GAF is very abundant in the nucleus <br /> fragments are labeled. <br />                                                                                       The ability of GAF to bind strongly to trinucleotides indicates that <br />                                                                                       there are a large number of potential binding sites. To determine <br /> stronger binding sites. However, with respect to GAF association                      the abundance of GAF, we examined its prevalence in Drosophila <br /> on hsp70, this appears not to be the case. When high and low                          S2 nuclei by western analysis, comparing with a recombinant <br /> affinity sites are separated before GAF binding by restriction                        GAF standard (Fig. 4A). For the most abundant GAF species (the <br /> digestion, occupancy of these trinucleotides persists. High and                       67 kDa isoform), we estimate roughly 3.0 &#195;&#8212; 105 molecules/cell <br /> low affinity sites were separated by cleavage of hsp70 promoter                       with a relatively large standard deviation of 1.4 &#195;&#8212; 105. This <br /> DNA sequences with a restriction endonuclease, and their DNase                        number is the average of 10 independent experiments. When <br /> I footprints compared with the same promoter fragments prior to                       totalling all immunoreactive species, we roughly estimate the <br /> cleavage as a function of GAF concentration (Fig 2A). The                             total number of GAF isomers to be in the order of one million. <br /> occupancy of more distant, low affinity sites is not appreciably                      Taking into consideration the diameter of a Drosophila nucleus <br /> affected by the presence and absence of high affinity sites.                          of 2 &#194;&#181;m (27), we calculate a nuclear volume of 3.4 &#195;&#8212; 10&#226;&#8364;&#8220;12 l, and <br />    Immunoprecipitation assays show that GAF is able to efficient-                     an approximate concentration of the GAF 67 kDa isomer at <br /> ly bind DNA fragments containing either high or low affinity sites                    130 nM. Assuming one nucleosome per 200 base pairs, and a total <br /> (Fig. 2B). High and low affinity sites were separated as before by                    genome size of 1.7 &#195;&#8212; 105 kb, on average one molecule of GAF67 <br /> restriction digestion, incubated with His-GAF and immunopre-                          is present for every 2.5 nucleosomes. <br /> cipitated with GAF specific antibodies. Three-fold increases in                          Quantitative western analysis indicates GAF is a very abundant <br /> protein concentrations show that GAF can bind lower affinity                          nuclear protein. One might expect a protein this abundant to be <br />  2676 Nucleic Acids Research, 1998, Vol. 26, No. 11 <br />  <br />  <br />                              A <br />  <br />  <br />  <br />  <br /> B                                                              C                                              D <br />  <br />  <br />  <br />  <br /> Figure 3. (A) Sequences of synthetic oligos used. (B) MBP&#226;&#8364;&#8220;Zn binding to various synthetic oligos. MBP&#226;&#8364;&#8220;Zn binding was tested against oligos containing either an <br /> idealized binding element [(GA)4], a pentamer (GAGAG), a trinucleotide (GAG) or alternate trinucleotide sequences (AGA or TGT). (C) Comparison of the relative <br /> affinities of His-GAF for an idealized binding element [(GA)4], a pentamer consensus, and a trinucleotide (GAG). Sequences present in the bound or free fractions <br /> are indicated. (D) Graph representing the relative affinities of both His-GAF and MBP&#226;&#8364;&#8220;Zn for each of the above sequence elements for three independent experiments <br /> of the type shown in (C). Values were normalized against binding to the [(GA)4] element which was assigned a value of one. <br />  <br />  <br />  <br /> A                                            B                                     causes a change in total nuclear protein as assayed by silver <br />                                                                                    staining with respect to several bands, indicated by closed circles <br />                                                                                    in Figure 4B, and most strikingly in the range of the most <br />                                                                                    prevalent GAF isoform at 67 kDa, indicated by an arrow. Western <br />                                                                                    analysis of duplicate samples run on the same gel verify the <br />                                                                                    identity and depletion of the 67 kDa isoform upon immunopreci- <br />                                                                                    pitation with GAF specific antibody, indicating the relative <br />                                                                                    abundance of GAF when compared with total nuclear protein. <br />  <br />                                                                                    DISCUSSION <br />                                                                                    GAF is an essential nuclear factor in Drosophila and has been <br />                                                                                    ascribed a variety of functions. GAF has a global role in <br />                                                                                    chromatin structure and is a critical component of numerous <br />                                                                                    promoters (for review see 4). Although sequence comparison of <br />                                                                                    known GAF elements shows a typical length of 3.5 GA repeats, <br /> Figure 4. (A) Quantitation of GAF molecules per cell. Representative gel of        and recent NMR studies indicate that the binding consensus is a <br /> known quantities of Drosophila S2 nuclei compared with known concentra-            pentamer (GAGAG) (19), here we show that a single trinucleo- <br /> tions of recombinant His-GAF, analyzed by western blotting with GAF specific       tide (GAG) appears sufficient to dictate GAF specific binding. <br /> antibody. (B) Immunodpeletion of the GAF 67 kDa isoform. Representative <br /> silver staining and western blotting of Drosophila Kc nulcear extract <br />                                                                                    Most zinc fingers have a core three base pair recognition sequence <br /> supernatants after a mock or GAF specific immunoprecipitation. The 67 kDa          with only minor contacts outside this core contributing to <br /> isoform is marked by an arrow, and several other interesting changes in total      specificity of the interaction (16&#226;&#8364;&#8220;18). GAF contains only one zinc <br /> protein are indicated by filled circles.                                           finger of the C2H2 variety, which, in addition to some basic <br />                                                                                    N-terminal regions, is sufficient for GAF specific binding <br />                                                                                    (19,25). These N-terminal basic regions, BR1 and BR2, have <br /> easily detected by analyzing total nuclear protein, and indeed it is               been implicated in sequence recognition, and have been shown to <br /> (Fig. 4B). Immunoprecipitation with GAF specific antibodies                        interact with the remaining two bases of the pentamer consensus; <br />                                                                                                                  2677 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1998,Vol.  26,No. <br />                                                                                                               Vol.22,  No.111                2677 <br />  <br /> however, the importance of these regions has not been fully            sequence, and can do so by means of a single zinc finger. Upon <br /> addressed. Single base pair mutations in the pentamer consensus,       comparison of the relative affinities of GAF for various sites, <br /> except the central G (GAGAG), have only moderate effects on            there is only a modest increase in affinity of GAF for a pentamer <br /> GAF binding, indicating the relatively minor contribution of BR1       consensus over a single trinucleotide. The affinity of GAF for <br /> and BR2 to the binding of GAF. With the central G in the               larger repeats [(GA)4] is, however, greater than both pentamer <br /> pentamer left untouched, any single <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> would still result in     and trinucleotide sites. The strong binding of GAF to larger sites <br /> a remaining trinucleotide for the zinc finger of GAF to bind.          may reflect a cooperative binding of GAF when sites are <br /> Surprisingly, not only can GAF bind a single trinucleotide site,       immediately adjacent. This is reminiscent of HSF binding to its <br /> but it can also do so by means of a single zinc finger which binds     targets where strong cooperativity exists for adjacent sites, but <br /> with near nanomolar affinity.                                          little cooperativity is evidenced between separated sites (30,31). <br />    Association of GAF with trinucleotides is evident on both the       Clearly, while GAF does prefer a pentamer sequence, the <br /> hsp70 and hsp26 genes in vitro (data not shown). All GAG               difference in affinity between pentamer and trinucleotide sites is <br /> trinucleotides in the promoter bind GAF in vitro when assayed by       less than a factor of two. <br /> DNase I footprinting. The frequency of GAG trinucleotides at the          Differences in GAF binding to high and low affinity sites as <br /> promoter is higher than that seen in the body of the gene, and these   assayed by footprinting and immunoprecipitation indicate a <br /> triplets are found clustered around important regulatory regions.      &lt;10-fold difference in affinity between fragments with multiple <br /> HSEs, the TATA box, and the transcription start site, become           binding sites. More precise measures of GAF affinity to single <br /> hypersensitive at high GAF concentrations, as a result of              sites shows an &#226;&#710;&#188;3.5-fold difference between elements of differing <br /> increased GAF binding to the promoter at low affinity sites. This      length. Binding to all sites assayed is in the nanomolar range. <br /> perturbation of DNA in these regions may have a role in                GAF is extremely abundant and sufficient to produce an <br /> providing access to key regulatory factors in a chromatin context,     approximate nuclear concentration of 150 nM. Though some <br /> as hypersensitive promoter regions correlate with inducibility of      percentage of GAF molecules may be multimeric (32), insoluble <br /> many genes. Indeed, mutations in the larger GAF elements have          or not available for binding, this high nuclear concentration of <br /> been shown to affect in vivo occupancy of HSEs by HSF, the             GAF makes it possible for high affinity sites, and a substantial <br /> association of TBP with the TATA box, and the establishment of         fraction of the low affinity sites, to be occupied in vivo. <br /> a promoter-paused RNA pol II molecule (11,12,28). Likewise,               Not all low affinity sites may be accessible for GAF binding <br /> GAG trinucleotides may also be functionally aiding factor              due to competition with other DNA binding proteins, chromatin <br /> association by increasing DNA accessibility in these regions.          structure and nuclear organization. Such may be the case with the <br />    GAF has also been shown to interact in vivo with internal hsp70     hsp70 gene, where GAF is only bound at the promoter prior to <br /> sequences after heat shock. No large binding sites for GAF appear      heat shock. After heat shock, GAF gains access to internal <br /> in these sequences, and presumably heat shock induced associ-          sequences and is seen to crosslink throughout the body of the gene <br /> ation of GAF with these sequences is due to the ability of GAF         (15). It is therefore possible that GAF gains access to these <br /> to interact with lower affinity DNA binding sites during               weaker internal sites only during transcription when the gene is <br /> transcription (15). Trinucleotides (GAG) can be predicted to           in an open conformation. Conversely, GAF may also be <br /> occur approximately once every 83 bp given base frequencies in         dependent upon other factors for this interaction. Gal4 protein has <br /> the Drosophila genome, and could easily explain GAF crosslink-         recently been shown to bind and activate transcription from its <br /> ing in the body of several genes. In fact, GAF elements of 3&#226;&#8364;&#8220;5 bp      own high affinity sites within nucleosomal templates, but requires <br /> occur, on average, once every 75 bp within hsp70 (R.C.Wilkins          SWI/SNF for occupancy and activation through lower affinity <br /> and J.T.Lis, unpublished observation). If accessible during            sites (33). Similarly, GAF may not be able to access low affinity <br /> transcription, these sites might provide the means by which GAF        sites in the body of the gene without the help of additional factors, <br /> associates with the body of active genes.                              or may be impeded by the same barriers that generate paused <br />    When high and low affinity sites are presented on the same, or      polymerase on these genes. Clearly, internal sites are not available <br /> separate fragments, there is little difference in the relative         for GAF binding prior to induction, and their accessibility may be <br /> occupancy of the more distant low affinity sites. GAF association      dependent upon cooperative interactions with nucleosome re- <br /> with GAG trinucleotides appears largely non-cooperative with           modeling factors or the transcriptional machinery itself. <br /> occupancy of high affinity sites. While our footprinting pattern <br /> agrees with results of GAF binding to the hsp70 gene by Weber          <br /> et al. (26), this lack of, or modest, cooperativity is in apparent <br /> contrast. The basis of this difference is not obvious, but could be    <br /> due to differences in recombinant GAF preparations, or technical       <br /> factors that are augmented by the steepness of the GAF gradient       <br /> used in those experiments. We believe that any cooperative         <br /> interactions between more distant sites must be weak in vitro,    <br /> since measures of the affinity of GAF for promoter sites in cis by <br /> footprinting is in agreement with the relative affinities of GAF <br /> sites assayed in trans. Similar types of non-cooperative loading      <br /> have been shown with zeste mediated activation in vivo (29). <br />    When looking at the capacity of single synthetic sites to bind        <br /> GAF, it is once again apparent that a trinucleotide alone is            <br /> sufficient for binding. GAF specifically recognizes a GAG               <br /> trinucleotide, as opposed to an AGA or TGT trinucleotide              <br />  2678 Nucleic Acids Research, 1998, Vol. 26, No. 11 <br /> </body></html>