<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#194;&#169; 2005 Genetics Society America <br /> DOI: 10.1534/genetics.104.036137 <br />  <br />  <br />  <br />               Systematic, RNA-Interference-Mediated Identification mus-101 <br />                           Modifier Genes Caenorhabditis elegans <br />  <br />                             Antonia H. Holway, Crystal Hung W. Matthew Michael1 <br />                                     The Biological Laboratories, Department Molecular Cellular Biology, <br />                                               Harvard University, Cambridge, Massachusetts 02138 <br />                                                     Manuscript received September 9, 2004 <br />                                                   Accepted publication December 8, 2004 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 The Mus101 family chromosomal proteins, identified initially Drosophila, widely conserved has <br />              shown function variety DNA metabolic processes. Such functions include DNA replication, DNA <br />              damage repair, postreplication repair, damage checkpoint activation, chromosome stability, chromosome <br />              condensation. Despite conservation widespread involvement chromosome biogenesis, little is <br />              known Mus101 regulated proteins required Mus101 exert its <br />              functions. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> learn Mus101, initiated analysis protein C. elegans. Here, we <br />              C. elegans mus-101 essential gene, required DNA replication, also <br />              plays important role DNA damage response. Furthermore, use <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi)- <br />              mediated reverse genetics screen genes modify mus-101 partial loss-of-function RNAi <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br />              Using systematic approach modifier gene discovery, chromosome I genes <br />              modify mus-101 RNAi phenotype, encodes E3 SUMO <br />              ligase promotes SUMO modification MUS-101 vitro. These results expand understanding <br />              MUS-101 regulation genetic interactions uncovered using screening strategies <br />              rely solely RNAi. <br />  <br />  <br />  <br /> T    HE combination RNA interference (RNAi)-medi- <br />      ated gene knockdown methods functional ge- <br /> nomics provided alternative classic forward <br />                                                                             could, example, reveal additional, nonessential func- <br />                                                                             tions gene. In addition, hypomorphic condi- <br />                                                                             tion exploited search genes modify <br /> genetics gene discovery efforts organisms                       phenotype essential gene therefore <br /> (Fraser et al. 2000; Gonczy et al. 2000; Berns et al. 2004;                 act common pathway. This assertion based <br /> Boutros et al. 2004; Paddison et al. 2004). RNAi trig-                   considerable evidence budding yeast where <br /> gered double-strand RNA (dsRNA) processed                        synthetic lethal screens used define and <br /> a manner results destruction mRNAs                      order genetic pathways. Synthetic lethal phenotypes result <br /> are homologous dsRNA (reviewed Novina                         mutations, affects viability on <br /> Sharp 2004). In nematode Caenorhabditis elegans                         own, cause decrease viability combined (re- <br /> dsRNA delivered organism meth-                    viewed Hartman et al. 2001). This occur of <br /> ods: soaking (Maeda et al. 2001), injection (Fire et al.                    ways: combining null alleles nonessential <br /> 1998), feeding (Timmons Fire 1998). The avail-                       genes, combining hypomorphic allele essen- <br /> ability complete genome sequence C. elegans                     tial gene <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>null allele</span> nonessential gene, and <br /> allowed systematic, high-throughput RNAi screens                      combining hypomorphic alleles different es- <br /> routinely performed, screens reported                         sential genes. That genetic interactions specific, <br /> using dsRNA delivery methods. While abso-                     suggest inclusion genes common or <br /> lute, trend emerged studies indi-                     similar pathways, addressed systematic <br /> cates RNAi feeding robust pene-                    genetic analysis (SGA) complete collection of <br /> trant methods, suggesting                      nonessential yeast deletion mutants. SGA involves mak- <br /> genes RNAi feeding phenotypically closer                       ing double mutants mutant and <br /> hypomorphic null condition.                                     5100 nonessential deletion mutants scoring <br />    When using RNAi analyze properties genes                       effects fitness (Tong et al. 2001). A recent study <br /> that encode essential functions, hypomorphic condi-                       reported results 132 SGA screens totaling &#226;&#181;&#8216;4000 <br /> tion useful. Partial depletion essential gene                  interactions (Tong et al. 2004). The average number of <br />                                                                             interactions target 34, demonstrating such <br />                                                                             screens reasonably specific interactions that <br />   1 <br />    Corresponding author: 2021 Biological Laboratories, 16 Divinity Ave.,    revealed. Another indication specificity the <br /> Cambridge, MA 02138. E-mail: matt@mcb.harvard.edu                           correlation function targets hits uncov- <br /> Genetics 169: 1451&#226;&#8364;&#8220;1460 (March 2005) <br />  1452                                          A. H. Holway, C. Hung W. M. Michael <br />  <br /> ered screen: 27% interactions involved                  formed described (Maeda et al. 2001). RNAi feeding <br /> pairs genes previously known function                 performed described (Timmons Fire 1998). For <br />                                                                       feeding RNAi, bacteria cultured 24 hr 37&#226;&#172;&#353; in <br /> in similar pathways. These data reveal               Terrific Broth containing 50 &#226;?&#174;g/ml ampicillin, seeded onto <br /> systematic genetic modifier screens effective tools            NGM plates containing 5 mm isopropyl-&#226;?&#164;-d-thiogalactopyrano- <br /> for gene discovery.                                                   (IPTG), allowed dry overnight. <br />    We studying DNA damage response                      Hoechst&#226;&#8364;&#8482;s 33258 staining embryos gonads: Worms <br /> C. elegans particularly interested               dissected glass microscope slides permeabilized <br />                                                                       freeze cracking. Slides fixed 10 min methanol/ <br /> mus-101 locus. mus-101 (F37D6.1) encodes 1227-amino-                formaldehyde fixative &#226;&#171;&#186;20&#226;&#172;&#353; washed PBST. DNA staining <br /> acid protein characterized presence             accomplished adding 10 &#226;?&#174;l 10 &#226;?&#174;g/&#226;?&#174;l Hoechst&#226;&#8364;&#8482;s 33258. <br /> copies BRCA1 carboxyl-terminal (BRCT) repeat.                     Antibody production: PCR primers designed amplify <br /> BRCT repeats commonly proteins involved                  fragments corresponding C-terminal 333 amino acids <br /> in DNA metabolism thought mediate protein-                 mus-101. The fragments subcloned pET30 <br />                                                                       bacterial expression vector (Novagen) used produce <br /> protein interactions BRCT-repeat-containing                   recombinant protein. Six-histidine-tagged recombinant pro- <br /> proteins phosphorylated protein-binding partners                  tein purified nickel agarose column denaturing <br /> (Manke et al. 2003; Yu et al. 2003). The mus-101 gene              conditions. Purified proteins used antigens to <br /> highly conserved evolution, homologs                  immunize rabbits. Polyclonal antibodies obtained and <br /> present humans (TopBP1; Yamane et al. 1997), Xeno-                 affinity purified according standard procedures. <br />                                                                          Propidium iodide tubulin staining embryos: Worms <br /> pus (Xmus101; Van Hatten et al. 2002), Drosophila                     mounted, fixed, washed described Hoechst&#226;&#8364;&#8482;s <br /> (mus101; Boyd et al. 1976), Arabidopsis (MEI1; Grelon                 staining. Slides incubated 200 &#226;?&#174;g/ml RNAse 30 min <br /> et al. 2003), fission budding yeast (cut5                 37&#226;&#172;&#353; followed 5-min incubation 0.1 mg/ml propidium <br /> DPB11, respectively; Saka Yanagida 1993; Araki                    iodide solution (Sigma, St. Louis). Embryos incu- <br /> et al. 1995). Mus101 family members linked                  bated antibody &#226;?&#163;-tubulin (MAb DM1A 1:100; <br />                                                                       Sigma) 2 hr room temperature, followed incubation <br /> to variety chromosomal pathways, including DNA                   donkey anti-mouse, FITC-labeled secondary antibody. <br /> replication, S-phase checkpoint activation, postreplica-              Embryos visualized Olympus BX51 microscope. <br /> tion repair damaged DNA, chromosome condensa-                      Pictures captured using SPOT RT monochrome camera <br /> tion, chromosome stability, meiosis. An important                 (Diagnostic Instruments). <br /> question regarding Mus101 function protein                    Embryo culture assays: Embryos prepared culture <br />                                                                       experiments basis published protocols (Edgar and <br /> is regulated participate multiple chromo-           McGhee 1988) deviations. Briefly, dissected adult <br /> somal pathways inducing inappropriate cross-talk              worms collected incubated 2 min 1:9 solution <br /> between different pathways. To help answer question,             6% hypochlorite (Fisher). Worms pelleted and <br /> we initiated analysis C. elegans mus-101 gene.         resuspended minimal egg growth media (EGM; Edgar <br /> RNAi soaking resulted robust depletion mus-101,              McGhee 1988), followed 2-min hypochlorite <br />                                                                       treatment. The remaining eggs pelleted, resuspended a <br /> a failure replicate DNA, embryonic lethality. RNAi             final time EGM, transferred gelatin-subbed slide (2%), <br /> using feeding protocol resulted penetrant                 covered coverslip supported half-inch adhesive <br /> depletion, viability, sensitivity DNA-damaging             transfer tape. Pressure applied coverslip attach <br /> agent methyl methanesulfonate (MMS). We took advan-                   embryos slide permeabilize vitelline mem- <br /> tage hypomorphic condition caused mus-101                   brane. Additional EGM added chamber. Finally, <br />                                                                       washes 30 &#226;?&#174;l EGM cytochalasin B flushed <br /> RNAi feeding systematically screen genes that,              chamber embryos incubated 1 hr. <br /> when codepleted mus-101, modified mus-101                    Exposure replication inhibitor hydroxyurea (HU) was <br /> phenotype. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> library 2425 chromosome I genes                  accomplished incubating permeabilized embryos EGM <br /> we isolated modifiers. Three displayed               containing 5 mm HU cytochalasin B. Embryos fixed <br /> MMS sensitivity, suggesting common function                    flushing chamber 30-&#226;?&#174;l washes fixative <br />                                                                       (Edgar McGhee 1988), followed PBS. DNA staining <br /> mus-101. Additionally, identified modifiers,               accomplished flushing chamber 30-&#226;?&#174;l <br /> gei-17, encodes presumptive E3 SUMO ligase                  washes solution containing 10 &#226;?&#174;g/&#226;?&#174;l Hoechst&#226;&#8364;&#8482;s 33258, <br /> shown stimulate SUMO modification MUS-101                    followed PBS and, finally, 30 &#226;?&#174;l mounting medium (2% <br /> vitro. These results shed new light regulation              NPG 80% glycerol). <br /> mus-101 demonstrate systematic codepletion                      MMS sensitivity assays: L4 F1 worms grown plates con- <br />                                                                       taining appropriate bacterial expression vectors were <br /> by RNAi useful method identification genetic              transferred plates containing 0.05 mg/ml MMS (Sigma) at <br /> modifiers.                                                            25&#226;&#172;&#353;. Eggs laid worms collected time and <br />                                                                       scored <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>survival</span>. <br />                                                                          Chromosome I RNAi modifier screen: The chromosome I <br />                MATERIALS AND METHODS                                  RNAi library purchased MRC geneservice. Bacteria <br />                                                                       amplified described mixed 1:1 ratio with <br />   C. elegans strains culturing: The N2 Bristol strain         mus-101 RNAi bacteria control RNAi bacteria (bacte- <br /> used experiments. Worms maintained described           ria expressing dsRNA exogenous green fluorescent <br /> (Brenner 1974). Embryonic lethality determined count-          protein, GFP). Additionally, mus-101 RNAi bacteria were <br /> ing percentage eggs failed hatch 20 hr laying.   mixed 1:1 ratio GFP RNAi bacteria establish <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span>- <br />   RNA interference assays: mus-101 RNAi soaking per-           line mus-101 RNAi lethality (which typically &#226;&#172;?5%). Bacte- <br />                                                   RNAi Modifier Screen C. elegans                                        1453 <br />  <br />                                                                     wild type feeding RNAi reduced level 53% <br />                                                                     wild type. When embryonic viability progeny <br />                                                                     treated animals assessed, RNAi by <br />                                                                     soaking caused high (78%) level embryonic lethality <br />                                                                     RNAi feeding resulted low <br />                                                                     level embryonic lethality (4%). We conclude that <br />                                                                     mus-101 essential gene, partial decrease in <br />                                                                     mus-101 expression does dramatically compromise <br />                                                                     viability. <br />   Figure 1.&#226;&#8364;&#8221;Immunoblot analysis MUS-101 protein levels <br /> in wild-type animals soaking feeding RNAi. Below          DNA replication problems embryos depleted of <br /> each RNAi lane percentage embryonic lethality (emb)          mus-101 soaking RNAi: A conserved function the <br /> in F1 (soaking) F2 (feeding) progeny RNAi-treated         Mus101 protein family DNA replication (Saka and <br /> animals. For soaking RNAi, N2 worms submerged             Yanagida 1993; Araki et al. 1995; Van Hatten et al. <br /> solution concentrated dsRNA according published pro- <br /> cedures (Maeda et al. 2001). After recovery period whole-         2002); likely explanation embryonic <br /> worm lysates prepared immunoblots. For feeding             lethality induced mus-101 RNAi soaking failure <br /> RNAi, animals cultured plates seeded bacteria          replicate. To investigate further, wild-type <br /> expressing dsRNA generations 25&#226;&#172;&#353; prior            mus-101 RNAi soaking embryos treated with <br /> preparation whole-worm lysates. Each lane blot cor- <br />                                                                     RNAse fixed stained nucleic acid <br /> responds 10 adult worms. <br />                                                                     stain propidium iodide assess DNA content. As shown <br />                                                                     Figure 2A, propidium iodide signal (&#226;&#8364;&#339;DNA&#226;&#8364;?) in <br /> rial mixes plated media containing 5 mm IPTG           mus-101 RNAi soaking embryos far intense than <br /> and allowed dry overnight. P0 worms seeded L1s           wild type, signal resulting an <br /> and cultured 25&#226;&#172;&#353;. Approximately 100 F2 embryos col-         anti-&#226;?&#163;-tubulin antibody (&#226;&#8364;&#339;tubulin&#226;&#8364;?) qualitatively equiva- <br /> lected mixture scored survival. Genes         lent. The mus-101 soaking RNAi embryos con- <br /> scored modifiers higher percentage lethality was <br /> observed gene X/mus-101 RNAi combination             tain substantially DNA wild-type embryos. A <br /> the gene X/GFP RNAi mus-101/GFP RNAi combinations                similar observation reported embryos de- <br /> after multiple rounds screening.                                 pleted DNA replication factor CDT-1 (Zhong et al. <br />    SUMOylation assay: A cDNA encoding full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> mus-101           2003). <br /> was transcribed translated presence [35S]methio- <br />                                                                        The data Figure 2A suggest role mus-101 in <br /> nine according manufacturer&#226;&#8364;&#8482;s instructions (Promega, <br /> Madison, WI). The sumoylation kit purchased LAE            DNA replication. To examine directly possibil- <br /> Biotech International vitro sumoylation reactions       ity, took advantage previous observations DNA <br /> performed according manufacturer&#226;&#8364;&#8482;s instructions         synthesis occurs normally embryos treated the <br /> the following exceptions. Recombinant C. elegans his-tagged         actin poison cytochlasin (Edgar McGhee 1988). <br /> UBC-9 substituted human protein supplied with <br /> the kit. Twenty-microliter reactions contained 5 &#226;?&#174;l trans-   Cytochlasin blocks cell division does prevent <br /> lation product, 0.36 &#226;?&#174;g C. elegans UBC-9, 0.68 &#226;?&#174;g C. elegans        DNA replication; cytochlasin-treated embryos will <br /> GEI-17, 1 &#226;?&#174;g human SUMO-I, 1 &#226;?&#174;g human SUMO III,                 perform multiple rounds DNA replication accu- <br /> 150 ng human E1. To generate recombinant C. elegans UBC-9           mulate masses DNA individual cells (Figure <br /> and GEI-17, full-length cDNAs amplified cDNA <br />                                                                     2B). This allows direct assessment involve- <br /> library cloned pET-28A expression vector (Nova- <br /> gen). Recombinant protein purified using standard his-          ment given gene DNA replication, embryos <br /> tag purification protocols. Induction GEI-17 performed       replicate DNA simply fail accumulate <br /> for 4 hr 16&#226;&#172;&#353;.                                                    masses DNA time. For experiment, four-cell- <br />                                                                     stage embryos collected, vitelline membranes <br />                                                                     permeabilized, samples treated with <br />                           RESULTS                                   cytochalasin B. After 60-min incubation period, the <br />    Differential depletion mus-101 soaking               embryos fixed DNA visualized the <br /> feeding RNAi: To initiate analysis mus-101 function           application Hoechst&#226;&#8364;&#8482;s 33258. As shown Figure 2C, <br /> in C. elegans, depleted gene product using RNAi              consistent previous reports (Edgar McGhee <br /> by soaking feeding. To assess relative effec-          1988), wild-type embryos accumulated large amounts <br /> tiveness feeding soaking RNAi mus-101 de-                 DNA individual cells hour-long incu- <br /> pletion, probed whole-worm lysates derived                  bation (Figure 2C, compare I II). By contrast, mus- <br /> treated animals affinity-purified anti-MUS-101              101 RNAi soaking embryos accumulated little <br /> antibody. RNAi soaking resulted robust depletion            DNA incubation period (Figure 2C, III). The <br /> of MUS-101 protein, RNAi feeding                DNA synthesized mus-101 RNAi soaking <br /> effective (Figure 1). Quantification band intensi-           embryos qualitatively similar synthe- <br /> ties using National Institutes Health Image revealed             sized wild-type embryos treated with <br /> that soaking RNAi reduced MUS-101 levels 10%                  replication inhibitor HU (Figure 2C, compare III <br />  1454                                     A. H. Holway, C. Hung W. M. Michael <br />  <br />  <br />  <br />  <br />                                                                              Figure 2.&#226;&#8364;&#8221;DNA replication attenuated after <br />                                                                           mus-101 RNAi soaking. (A) Wild-type mus- <br />                                                                           101 soaking RNAi embryos fixed, treated <br />                                                                           RNase I, stained propidium iodide <br />                                                                           visualize DNA (&#226;&#8364;&#339;DNA&#226;&#8364;?) anti- <br />                                                                           &#226;?&#163;-tubulin visualize tubulin ( &#226;&#8364;&#339;tubulin&#226;&#8364;?). (B) <br />                                                                           Schematic uncoupling DNA replication <br />                                                                           cell division occurs treatment of <br />                                                                           permeabilized embryos cytokinesis inhib- <br />                                                                           itor cytochlasin B. (C) Permeabilized embryos <br />                                                                           fixed stained Hoechst&#226;&#8364;&#8482;s 33258 stain <br />                                                                           immediately permeabilization (I) or <br />                                                                           1-hr incubation culture media (II&#226;&#8364;&#8220;IV). <br />                                                                           The embryos shown II IV wild type, <br />                                                                           embryo depicted III depleted <br />                                                                           mus-101 gene product soaking RNAi. The <br />                                                                           embryo IV treated HU block replica- <br />                                                                           tion. The embryos depicted I&#226;&#8364;&#8220;IV represen- <br />                                                                           tative larger (&#226;&#172;&#381;25) sample set ana- <br />                                                                           lyzed condition. <br />  <br />  <br />  <br />  <br /> to IV). From this, conclude mus-101 required        notype observation RNAi feeding depletion <br /> for DNA replication early C. elegans embryo.            replication initiation factor ORC-2 causes iden- <br />    Depletion mus-101 feeding RNAi mimics hypo-         tical sterility/abnormal nuclear morphology defect (A. <br /> morphic condition: By contrast RNAi soaking,             Holway, unpublished data). <br /> RNAi feeding did cause extensive embryonic                 In Drosophila, hypomorphic alleles mus101 that <br /> lethality (Figure 1). It did, however, result 20% steril-   retain DNA replication function DNA <br /> ity F1 progeny treated animals. Staining      damage response function protein exist (Boyd <br /> germ lines F1 fertile sterile adults revealed       et al. 1976). In C. elegans shown feeding RNAi <br /> the sterile animals abnormal gonad morphology           allows embryonic viability, DNA replication, and <br /> characterized necrotic-looking germ cell nuclei (Fig-       determine DNA dam- <br /> ure 3). This demonstrates that, low frequency, mus-         age response function mus-101 retained after <br /> 101 feeding RNAi affects proliferation germ line.       feeding RNAi. To examine this, tested MMS sensi- <br /> Although basis proliferation defect        tivity. F1 L4 stage animals plated media con- <br /> known, morphology affected nuclei reminiscent        taining 0.05 mg/ml MMS allowed lay eggs for <br /> of wild-type germ-cell nuclei blocked replica-    16 hr transferred fresh MMS plate for <br /> tion HU (MacQueen Villeneuve 2001)                 additional 16 hr egg laying. Embryonic lethality <br /> treated ionizing radiation (Gartner et al. 2000).         F2 determined counting number <br /> Consistent replication problem causing phe-        eggs failed hatch. As shown Figure 4, and <br />                                                 RNAi Modifier Screen C. elegans                                          1455 <br />  <br />  <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;Abnormal nuclear morphology animals steri- <br /> lized mus-101 feeding RNAi. Gonads F1 progeny of <br /> depleted animals dissected stained Hoechst&#226;&#8364;&#8482;s <br /> 33258 visualize DNA. Roughly 80% animals <br /> are fertile normal nuclear morphology germ <br /> line (I), remainder sterile display abnor- <br /> mal morphology (II). A single gonad arm shown, the <br /> distal tip right case. <br />                                                                      Figure 4.&#226;&#8364;&#8221;MMS sensitivity mus-101 feeding RNAi. <br />                                                                   Either wild-type mus-101 feeding RNAi animals plated <br /> by contrast wild type, mus-101 RNAi feeding re-             L4s media containing 0.05 mg/ml MMS. Embryonic <br />                                                                   lethality progeny animals determined after <br /> sulted MMS sensitivity. This result makes impor-           16 32 hr egg laying described materials and <br /> tant points: (1) mus-101 required DNA damage           methods. <br /> response C. elegans embryos (2) separation of <br /> the DNA replication DNA damage response <br /> function mus-101 achieved RNAi               ments shown reproducibility suffered the <br /> feeding.                                                          cutoff set beneath value. <br />    Identification mus-101 modifiers: The immunoblot               To isolate mus-101 modifiers, screened gene <br /> in Figure 1 shows mus-101 RNAi feeding reduces            present library containing 2425 chromosome I genes <br /> the steady-state level MUS-101 protein roughly half         (representing 89% chromosome; Fraser et <br /> of normally present. The experiments shown             al. 2000 details library). One concern raised <br /> Figures 3 4 reveal reduction gener-        performing screen mixing two <br /> ally tolerable, phenotypic consequences         different feeding vectors depletion experi- <br /> the germ line embryo. RNAi feeding there-              ment weaken eliminate ability either <br /> fore mimics hypomorphic allele mus-101. We sought            feeding vector effectively silence target gene. <br /> to exploit hypomorphic condition identify           This did occur. Fraser et al. (2000) reported <br /> genes function mus-101 systematically search-        chromosome I RNAi feeding screen using full <br /> ing genes that, codepletion, modified mus- <br /> 101 feeding RNAi phenotype. In particular, screened <br /> for genes codepleted mus-101 caused <br /> an increase embryonic lethality. Codepletions were <br /> performed mixing, 1:1 ratio, bacteria expressing <br /> dsRNA mus-101 bacteria expressing dsRNA <br /> against gene screened. The mixture then <br /> seeded media plates prior plating L1 stage <br /> worms. After generations selective media em- <br /> bryonic viability determined compared that <br /> observed control codepletion substituted mus- <br /> 101 irrelevant gene GFP. Two criteria im- <br /> posed weed weak and/or nonspecific interactions               Figure 5.&#226;&#8364;&#8221;Screening strategy isolation mus-101 mod- <br /> (Figure 5). One, percentage embryonic lethality               ifiers dual RNAi. Well A control plate contains <br /> produced codepletion mus-101 candidate              bacteria expressing dsRNA gene screened <br /> gene, twice observed               dsRNA irrelevant protein (GFP). This produces <br />                                                                   N % embryonic lethality (emb). When dual RNAi performed <br /> candidate codepleted GFP. Two, embryonic             gene screened mus-101 (well B), then <br /> lethality produced codepletion mus-101              percentage emb &#226;&#172;&#381;10 twice N for <br /> candidate gene 10% pilot experi-              gene pursued further. <br />  1456                                      A. H. Holway, C. Hung W. M. Michael <br />  <br />                                                            TABLE 1 <br />                              Genes interact mus-101 codepletion feeding RNAi <br />  <br />          Gene                   emb (GFP) (%)                   emb (mus-101) (%)                 Presumed function <br />          W01A8.1                  3.3   (9/269)                   14.7   (58/395)                 Unknown <br />          F26B1.2                  7.7   (29/379)                  14.5   (62/427)                 KH domain <br />          gei-17                  12.8   (20/156)                  28.9   (43/149)                 PIAS SUMO ligase <br />          arx-7                    4.6   (11/239)                  12.6   (17/135)                 Arp 2/3 complex <br />          let-49                   1.7   (6/356)                   13.5   (21/156)                 Transcription factor <br />            Emb determined counting number eggs failed hatch. Both percentage actual <br />          counts shown. Emb (GFP) refers embryonic lethality codepletion GFP emb (mus-101) refers <br />          embryonic lethality codepletion mus-101. <br />  <br />  <br /> dose chromosome I feeding vectors. Our screen,             presumptive ortholog C. briggsae. F26B1.2 encodes <br /> which used half dose chromosome I vectors,               protein containing heterogeneous nuclear ribonucleo- <br /> produced outcomes consistent re-             protein K homology (KH) domain. gei-17 encodes pro- <br /> ported Fraser et al. (2000) 95% time. The               tein homology mammalian protein inhibitor of <br /> 5% cases outcomes did agree             activated STAT (PIAS) family transcriptional regulators <br /> equally divided examples Fraser et al.              E3 SUMO ligases (reviewed Schmidt Muller <br /> (2000) reported phenotype given gene               2003). arx-7 encodes component Arp2/3 com- <br /> did examples observe pheno-              plex, regulates actin dynamics, let-49 a <br /> type Fraser et al. (2000) did not. Thus 5%               transcriptional regulator controls expression of <br /> discrepancy likely reflects noise inherent             genes required germ-line formation postembry- <br /> large-scale functional genomics screen. The high degree           onic development (Kwon et al. 2001). <br /> of correlation phenotypes reported Fraser                 The genes identified screen caused an <br /> et al. (2000) using complement feeding vector           increase embryonic lethality <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>- <br /> and reported using half complement shows             pleted mus-101, relative GFP. To make sure that <br /> unambiguously RNAi codepletion feeding allows             enhanced embryonic lethality synergy <br /> silencing target genes codepletion.                mus-101 RNAi unexpected <br />    Table 1 lists results screen. Five genes      suppression embryonic lethality GFP RNAi, <br /> fit criteria outlined Figure 5 isolated. Three        performed full-dose feeding RNAi experiments on <br /> of genes, gei-17, arx-7, let-49, pre-           genes. No significant differences observed <br /> viously characterized, significant detail. The         embryonic lethality dose feeding <br /> other two, W01A8.1 F26B1.2, charac-             RNAi relative observed GFP codepletions <br /> terized. W01A8.1 encodes protein strong ho-             genes. For let-49, dose feeding <br /> mology current databases, excepting            RNAi resulted 100% sterility F1, precluding <br />                                                                   analysis F2. We conclude genes <br />                                                                   identified screen false positives resulting <br />                                                                   interactions GFP RNAi. <br />                                                                      MMS sensitivity mus-101 modifiers: Four the <br />                                                                   screen hits tested MMS sensitivity. We <br />                                                                   did score let-49 assay owing sterility <br />                                                                   phenotype. Three genes tested showed sensi- <br />                                                                   tivity (Figure 6). One genes, gei-17, exception- <br />                                                                   ally sensitive MMS, embryonic lethality 100% <br />                                                                   time point. The genes, F26B1.2 <br />                                                                   arx-7, modestly sensitive. For arx-7, we <br />                                                                   did note weak eggshell phenotype; un- <br />                                                                   clear MMS sensitivity arx-7 RNAi <br />                                                                   embryos direct involvement DNA <br />                                                                   damage response weakened eggshell resulting <br />                                                                   embryos experiencing higher effective concen- <br />                                                                   tration MMS experiment. <br />    Figure 6.&#226;&#8364;&#8221;MMS sensitivity mus-101 modifiers. MMS sen-          To understand mus-101, gei-17, and <br /> sitivity assays performed described Figure 4     F26B1.2 depleted embryos MMS sensitive, stained <br /> feeding RNAi indicated genes.                         MMS-exposed embryos DNA stain Hoechst&#226;&#8364;&#8482;s <br />                                              RNAi Modifier Screen C. elegans                                        1457 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 7.&#226;&#8364;&#8221;MMS induces nuclear <br />                                                                                   morphology abnormalities mus-101 <br />                                                                                   gei-17 feeding RNAi. Wild-type (I), <br />                                                                                   mus-101 (II), gei-17 (III), F26B1.2 <br />                                                                                   (IV) feeding RNAi embryos ex- <br />                                                                                   posed MMS fixed and <br />                                                                                   stained Hoechst&#226;&#8364;&#8482;s 33258 visualize <br />                                                                                   DNA. V VI enlarged exam- <br />                                                                                   ples anaphase bridges were <br />                                                                                   commonly observed mus-101 and <br />                                                                                   gei-17 feeding RNAi samples, respec- <br />                                                                                   tively. <br />  <br />  <br />  <br />  <br /> 33258 examine morphology nuclear DNA.            transcription translation presence <br /> As shown Figure 7, nuclear morphology F26B1.2            [35S]methionine produce MUS-101 TnT. MUS- <br /> depleted embryos did differ wild type           101 TnT mixed core components the <br /> MMS exposure. By contrast, nuclear morphology               SUMOylation machinery (E1, E2, SUMO) well <br /> both mus-101 gei-17 depleted embryos affected          recombinant GEI-17, reaction products were <br /> by MMS. Specifically, nuclei discontinuous        resolved SDS-PAGE gel. This resulted ap- <br /> size shape, nuclei appeared torn             pearance high-molecular-weight bands ran above <br /> fragmented, numerous anaphase bridges            MUS-101 TnT SDS-PAGE (Figure 8). The appear- <br /> observed. Such nuclear morphology aberrations         ance bands dependent presence <br /> observed mus-101 gei-17 depleted embryos       GEI-17 SUMO reaction mixture, <br /> the absence MMS exposure. Thus, addition              demonstrating derived SUMO mod- <br /> MMS sensitivity, mus-101 gei-17 share similar nu-        ification MUS-101 GEI-17 facilitates this. We <br /> clear morphology defect MMS exposure. This sug-           conclude GEI-17 stimulates SUMO modification of <br /> gests mus-101 gei-17 components           MUS-101 vitro, suggesting GEI-17 control <br /> common pathway responds MMS-induced DNA                MUS-101 function vivo. <br /> damage embryo. <br />    GEI-17 stimulates SUMO modification MUS-101 <br /> in vitro: To strengthen connection mus-101                              DISCUSSION <br /> and gei-17, sought biochemical evidence inter-          In article used RNAi reduce expression <br /> action gene products. The PIAS do-             mus-101 gene. Soaking RNAi cleared 90% MUS- <br /> main family proteins, GEI-17 member,          101 protein, feeding RNAi removed roughly <br /> has shown stimulate SUMO modification               half. Accordingly, soaking RNAi caused high embryonic <br /> substrate proteins (Johnson Gupta 2001). SUMO              lethality, likely reduction DNA <br /> is small, ubiquitin-like polypeptide covalently      replication. For remainder work have <br /> attached substrate proteins (reviewed Johnson            focused mus-101 RNAi feeding phenotype. <br /> 2004). SUMO modification substrates alter            The mus-101 RNAi feeding phenotype: Feeding <br /> function, chromosomal proteins particular           RNAi caused low level (20%) sterility F1, <br /> subject regulation SUMO system. Attachment           DNA staining germ lines sterile animals <br /> of SUMO substrate similar ubiquitination          revealed defect germ-cell proliferation. We have <br /> in attachment mediated sequential trans-       observed phenotype RNAi DNA <br /> fer SUMO E1 enzyme E2             replication proteins, subunits origin rec- <br /> to substrate. The step, transfer SUMO         ognition complex (A. Holway, unpublished data), and <br /> E2 substrate, facilitated E3 ligase     suspect phenotype connected a <br /> as PIAS domain protein. Because gei-17 modifies          defect initiation DNA replication. It also <br /> mus-101 feeding RNAi phenotype, speculated             possible inability repair replication-induced <br /> GEI-17 stimulate SUMO modification MUS-101.             damage responsible phenotype. <br /> We asked GEI-17 protein stimulate              Unlike sterility phenotype, modestly <br /> SUMO modification MUS-101 reconstituted             penetrant, MMS sensitivity phenotype feeding <br /> vitro SUMOylation assay. MUS-101 radiolabeled              RNAi highly penetrant embryonic lethality of <br />  1458                                      A. H. Holway, C. Hung W. M. Michael <br />  <br />  <br />  <br />                                                                               Figure 8.&#226;&#8364;&#8221;GEI-17 stimulates SUMO modifica- <br />                                                                            tion MUS-101. An vitro SUMOylation assay <br />                                                                            performed described materials and <br />                                                                            methods. (&#226;&#171;&#185;) indicates given compo- <br />                                                                            nent included reaction (&#226;&#171;&#186;) indi- <br />                                                                            cates omitted. <br />  <br />  <br />  <br />  <br /> &#226;&#172;&#381;90% compared &#226;&#172;?10% wild type. This suggests           arx-7, F26B1.2) showed MMS sensitivity full <br /> that 50% reduction MUS-101 protein incompati-           dose feeding RNAi, implying participation the <br /> ble efficient embryonic DNA damage response.            DNA damage response. F26B1.2 encodes KH-domain- <br /> This, turn, suggests replication damage         containing protein. KH proteins bind single-strand nu- <br /> response functions mus-101 uncoupled               cleic acids, roles proteins the <br /> feeding RNAi. Interestingly, uncoupling           DNA damage response common, note <br /> been observed Drosophila budding            MCG10 KH protein shown a <br /> fission yeast homologs mus-101. Drosophila mus101            transcriptional target p53 promote cell cycle <br /> hypomorphic alleles MMS sensitive defec-           arrest apoptosis DNA damage human cells <br /> tive postreplication repair nonetheless viable       (Zhu Chen 2000). Our finding F26B1.2 RNAi <br /> exist (Boyd et al. 1976). In fission yeast, separation-         embryos MMS sensitive strengthens con- <br /> of-function alleles cut5 allow replication      nection KH proteins DNA damage <br /> activation DNA damage checkpoint iso-          response. <br /> lated (McFarlane et al. 1997). Finally, budding yeast,          The gene screen showed ro- <br /> the dpb11-1 allele MMS sensitive permissive tem-      bust MMS sensitivity gei-17. The gei-17 gene was <br /> perature DNA replication (Wang Elledge 2002).           initially identified 1 26 genes recovered yeast <br /> Thus common theme gene family avail-           two-hybrid screen using transcription factor gex-3 <br /> ability hypomorphic alleles. The data presented         bait (Tsuboi et al. 2002) further <br /> shed light showing alterations mus-      characterized. gei-17 highly conserved shares ho- <br /> 101 coding sequences necessary generate              mology Su(var)2-10 chromosomal regulator in <br /> hypomorphic alleles limiting expression other-     Drosophila (Hari et al. 2001) PIAS family of <br /> wise wild-type copy gene sufficient this. One   mammalian SUMO ligases (reviewed Seeler and <br /> interpretation DNA damage response          Dejean 2003). Three findings reported provide <br /> function Mus101 family requires protein          strong evidence gei-17 mus-101 act in <br /> a per-cell basis does replication function.            common pathway: <br />    The modifier screen: Having established hypomor- <br />                                                                 1. The gei-17 gene selected screen mus- <br /> phic condition using mus-101 feeding RNAi, ex- <br />                                                                    101 modifier. <br /> ploited screen genes modified pheno- <br />                                                                 2. Like mus-101, gei-17 RNAi embryos MMS sensi- <br /> type. This resulted isolation genes that <br />                                                                    tivity MMS-induced nuclear morphology de- <br /> displayed enhanced embryonic lethality codepleted <br />                                                                    fect. We observed MMS sensitivity RNAi for <br /> with mus-101. Interestingly, genes identified <br />                                                                    number DNA damage response genes yet <br /> are known encode DNA replication factors. Indeed, <br />                                                                    genes date produce nuclear <br /> when directed codepletion experiments per- <br />                                                                    morphology defect mus-101 gei-17. Thus the <br /> formed mus-101 number different DNA <br />                                                                    nuclear morphology defect relatively rare. <br /> replication factors, failed detect interactions <br />                                                                 3. We demonstrated biochemical interaction be- <br /> (A. Holway C. Hung, unpublished data). This is <br />                                                                    tween MUS-101 GEI-17 showing in vitro <br /> in contrast synthetic lethal screen performed in <br />                                                                    SUMOylation assay GEI-17 stimulates SUMO <br /> budding yeast using dpb11-1 allele (Kamimura et al. <br />                                                                    modification MUS-101. <br /> 1998). This screen identified numerous sld genes, many <br /> of DNA replication factors. One explanation           Taken together, results suggest SUMO <br /> for C. elegans MUS-101 excess,       modification MUS-101 GEI-17 regulates MUS-101 <br /> after feeding RNAi embryonic DNA replication,          function DNA damage response. The con- <br /> making difficult weaken pathway. If so,      nections Mus101 PIAS proteins not <br /> we expect replication genetic            limited reported here. We note Dro- <br /> interactions, given embryonic lethality scored     sophila gei-17 homolog Su(var)2-10 shares chromo- <br /> phenotype.                                                      somal phenotype fly mus101 genes are <br />    Interestingly, genes tested (gei-17,       required condensation heterochromatic regions <br />                                               RNAi Modifier Screen C. elegans                                                 1459 <br />  <br /> of chromosome (Gatti et al. 1983; Yamamoto et al.              We thank Craig Hunter colleagues generous expert <br /> 2000; Hari et al. 2001). How phenotype            advice working C. elegans. We grateful Tim Schedl <br /> connected MMS sensitivity reported          thoughtful discussion advice assistance the <br />                                                                 early stages work. We thank Mike Matunis help advice <br /> C. elegans orthologs? We propose mus-101          SUMO modification. The N2 Bristol strain HT115(DE3) cells <br /> gei-17 required promote DNA replication            used work provided Caenorhabditis Genetics Center, <br /> conditions replication difficult. MMS-mediated         funded National Institutes Health National Center <br /> damage DNA known stress replication (Tercero           Research Resources. The chromosome I RNAi library created <br /> and Diffley 2001), replication hetero-              Julie Ahringer distributed MRC geneservice. A.H.H. was <br />                                                                 supported funds Genetics Genomics training grant <br /> chromatin thought difficult (reviewed by <br />                                                                 National Institute General Medical Sciences (NIGMS) <br /> Maison Almouzni 2004). Thus heterochroma-               Don Wiley Award Excellence Graduate Studies (funded <br /> tin condensation defect indirect result          Merck). Additional support work provided research <br /> failure replicate heterochromatic DNA. Consistent       grants NIGMS (R01GM67735), American Cancer Society <br /> this hypothesis recent study budding yeast       (RSG-03-153), Searle Scholar Award W.M.M. <br /> isolated DPB11 screen chromosome integrity genes <br /> (Huang Koshland 2003). This work showed that <br /> DPB11 required elongation phase replica-                            LITERATURE CITED <br /> tion maintain replication fork progression. Hypomor- <br />                                                                 Araki, H., S. H. Leem, A. Phongdara A. Sugino, 1995 Dpb11, <br /> phic alleles DPB11 caused elongation failures higher          interacts DNA polymerase II(epsilon) Saccharo- <br /> frequencies did wild type elongation chal-            myces cerevisiae, dual role S-phase progression a <br /> lenged increasing distance active origins            cell cycle checkpoint. Proc. Natl. Acad. Sci. USA 92: 11791&#226;&#8364;&#8220;11795. <br />                                                                 Berns, K., E. M. Hijmans, J. Mullenders, T. R. Brummelkamp, A. <br /> on mini-chromosomes. While work re-                 Velds et al., 2004 A large-scale RNAi screen human cells <br /> quired confirm hypothesis, intriguing              identifies new components p53 pathway. Nature 428: 431&#226;&#8364;&#8220; <br /> speculate GEI-17-mediated SUMO modification                 437. <br />                                                                 Boutros, M., A. A. Kiger, S. Armknecht, K. Kerr, M. Hild et al., <br /> MUS-101 allows MUS-101 promote replication                 2004 Genome-wide RNAi analysis growth viability Dro- <br /> stressful conditions.                                               sophila cells. Science 303: 832&#226;&#8364;&#8220;835. <br />    Dual RNAi modifier screens: This article presents,        Boyd, J. B., M. D. Golino, T. D. Nguyen M. M. Green, 1976 <br />                                                                     Isolation characterization X-linked mutants Drosophila <br /> our knowledge, large-scale screen genetic             melanogaster sensitive mutagens. Genetics 84: 485&#226;&#8364;&#8220; <br /> modifiers based entirely RNAi. We screened               506. <br /> a chromosome I library covers 13% C. elegans        Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics <br />                                                                     77: 71&#226;&#8364;&#8220;94. <br /> genome isolated modifiers. Through extrapola- <br />                                                                 Edgar, L. G., J. D. Mcghee, 1988 DNA synthesis control <br /> tion estimate total 38 modifier genes              embryonic gene expression C. elegans. Cell 53: 589&#226;&#8364;&#8220;599. <br /> would isolated conditions screening           Fire, A., S. Xu, M. K. Montgomery, S. A. Kostas, S. E. Driver et <br /> the entire genome. This is, interestingly, line         al., 1998 Potent specific genetic interference double- <br />                                                                     stranded RNA Caenorhabditis elegans. Nature 391: 806&#226;&#8364;&#8220;811. <br /> average number 34 modifiers uncovered budding             Fraser, A. G., R. S. Kamath, P. Zipperlen, M. Martinez-Campos, <br /> yeast SGA. Whether dual RNAi modifier screens                M. Sohrmann et al., 2000 Functional genomic analysis C. <br /> are universally useful isolating genetic modifiers re-           elegans chromosome I systematic RNA interference. Nature <br />                                                                     408: 325&#226;&#8364;&#8220;330. <br /> mains determined. The drawbacks include               Gartner, A., S. Milstein, S. Ahmed, J. Hodgkin M. O. Hen- <br /> possibility low penetrance occur using feed-            gartner, 2000 A conserved checkpoint pathway mediates DNA <br /> ing RNAi. Additionally, method does appear iso-              damage&#226;&#8364;&#8220;induced apoptosis cell cycle arrest C. elegans. <br />                                                                     Mol. Cell 5: 435&#226;&#8364;&#8220;443. <br /> late nonspecific interactors unclear arx-7,        Gatti, M., D. A. Smith B. S. Baker, 1983 A gene controlling <br /> which encodes cytoplasmic component actin                  condensation heterochromatin Drosophila melanogaster. <br /> polymerization regulator Arp2/3 complex, par-                 Science 221: 83&#226;&#8364;&#8220;85. <br />                                                                 Gonczy, P., C. Echeverri, K. Oegema, A. Coulson, S. J. Jones et <br /> ticipate directly mus-101. Nonetheless, method             al., 2000 Functional genomic analysis cell division C. eleg- <br /> </body></html>