<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, July 2007, p. 4784&#226;&#8364;&#8220;4795                                                                           Vol. 27, No. 13 <br /> 0270-7306/07/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.00494-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />   Role Histone Deacetylase 1 Human Tumor Cell Proliferation&#228;&#338;&#164; <br />           Silvia Senese,1,2 Katrin Zaragoza,1 Simone Minardi,2 Ivan Muradore,1 Simona Ronzoni,1 <br />                   Alfonso Passafaro,1 Loris Bernard,2 Giulio F. Draetta,3 Myriam Alcalay,2 <br />                                   Christian Seiser,4 Susanna Chiocca1,2* <br />   European Institute Oncology, Department Experimental Oncology, 20141 Milan, Italy1; IFOM-IEO Campus, Via Adamello 16, <br />       20139 Milan, Italy2; Cancer Research, Merck Research Laboratories, Basic Research, 33 Avenue Louis Pasteur, Boston, <br />              Massachusetts 021153; Max F. Perutz Laboratories, Medical University Vienna, Vienna Biocenter, <br />                                           Dr. Bohr-Gasse 9/2, A-1030 Vienna, Austria4 <br />                            Received 21 March 2007/Returned modification 11 April 2007/Accepted 18 April 2007 <br />  <br />              Posttranslational modifications core histones central regulation gene expression. Histone <br />           deacetylases (HDACs) repress transcription deacetylating histones, class I HDACs crucial role <br />           mouse, Xenopus laevis, zebra fish, Caenorhabditis elegans development. The role individual class I <br />           HDACs tumor cell proliferation investigated using <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>RNA interference</span>-mediated protein knockdown. We <br />           absence HDAC1 cells arrest G1 phase cell cycle G2/M <br />           transition, resulting loss mitotic cells, cell growth inhibition, increase percentage of <br />           apoptotic cells. On contrary, HDAC2 knockdown showed effect cell proliferation unless concur- <br />           rently knocked HDAC1. Using gene expression profiling analysis, inactivation HDAC1 <br />           affected transcription specific target genes involved proliferation apoptosis. Furthermore, HDAC2 <br />           downregulation did cause significant changes compared control cells, inactivation HDAC1, <br />           HDAC1 plus HDAC2, HDAC3 resulted distinct clusters. Loss HDACs impair cell <br />           cycle progression affecting transcription specific target genes biological <br />           processes. Our data support idea drug targeting specific HDACs highly beneficial the <br />           treatment cancer. <br />  <br />  <br />    In eukaryotic cells, balance acetylation                      shown induce polyploidy human cancer cell lines <br /> deacetylation critical gene transcription func-               (55). As learn HDACs biological pro- <br /> tions different cellular proteins. Histone acetylation regu-               cesses regulate, strong idea identities the <br /> lated histone acetyltransferases histone deacetylases                relevant target deacetylases need determined aid in <br /> (HDACs), conserved yeast (Saccharomyces cer-                      development HDAC inhibitors anticancer agents is <br /> evisiae) humans. Mammalian HDACs grouped                            emerging. One clear example demonstration HDAC2 <br /> into classes according homology yeast pro-                     induction loss adenomatosis polyposis coli tumor <br /> teins. The class I enzymes HDAC1 HDAC2 ubiqui-                           suppressor colorectal tumorigenesis, providing reason for <br /> tously expressed involved more-general                         treating certain carcinomas HDAC inhibitors (58). Inter- <br /> cellular processes. According sequence, HDAC3                        estingly, demonstrated previously HDAC <br /> belongs class I family, interact               inhibitor valproic acid selectively induces polyubiquitination <br /> class I class II enzymes represent functional                   proteasomal degradation HDAC2 (35). Lately, HDAC3 <br /> link families (11). Class II enzymes                        required normal mitotic progression and <br /> tissue-specific functions, class III enzymes NAD de-                     maximal phosphorylation histone 3 (H3) Ser 10 (S10) <br /> pendent involved control life span                      mitosis (38). Phosphorylation H3 S10 well- <br /> certain organisms (15, 16, 33). Recently, class IV, in-                    characterized evolutionary conserved mitotic event (6, <br /> cludes HDAC11-related enzymes, added (12, 15).                          17, 22, 23, 45). <br /> Because connection transcriptional repression                        Recent experiments small interfering RNA (siRNA) <br /> and HDAC recruitment, HDAC inhibitors reverse si-                            implicated HDAC1 HDAC3 regulation prolifer- <br /> lenced genes (reviewed reference 30).                                         ation <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>survival</span> cancer cells (13). In particular, idea <br />    HDAC inhibitors new class drugs anticancer                      HDAC1 potential target therapeutic <br /> potential. They shown induce apoptosis effec-                       intervention certain cancers stems series studies. <br /> tively cancer cells currently clinical trials                HDAC1 shown inhibit estrogen receptor alpha <br /> variety cancers (4, 40). One example includes HDAC                        protein expression transcriptional activity, suggesting that <br /> inhibitor suberoylanilide hydroxamic acid, recently                    modulate breast cancer progression (32). Overexpres- <br />                                                                                  sion HDAC1 mRNA protein levels reported for <br />                                                                                  human gastric prostate tumors (5, 18, 34, 44), whereas <br />   * Corresponding author. Mailing address: European Institute On-             overexpression HDAC3 seen colon tumors (54). <br /> cology, Department Experimental Oncology, Via Ripamonti 435, <br /> 20141 Milan, Italy. Phone: 39-02-57489835. Fax: 39-02-57489851. E-mail: <br />                                                                                  Furthermore, published data indicate role mouse HDAC1 <br /> susanna.chiocca@ifom-ieo-campus.it.                                              regulation proliferation development. For in- <br />   &#228;&#338;&#164; <br />     Published ahead print 30 April 2007.                                   stance, expression HDAC1 induced growth <br />  <br />                                                                           4784 <br />  VOL. 27, 2007                                                            ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                                              4785 <br />  <br />  <br /> factor activation mouse T cells fibroblasts (3, 19)                              After Western blotting, primary antibodies anti-HDAC1 (monoclonal mouse <br /> HDAC1 levels elevated highly proliferative                           antibody; Upstate), anti-HDAC3 (ab 7030; Abcam), anti-caspase-3 (H-277; <br />                                                                                          Santa Cruz Biotechnology) incubated membrane overnight 4&#194;&#176;C; <br /> tissues, embryonic stem (ES) cells, transformed                              anti-HDAC2 (ab 7029; Abcam), anti-p21 (a kind gift K. Helin&#226;&#8364;&#8482;s lab), <br /> cell lines (3, 37), suggesting link HDAC1 function                             anti-acetylated H3/H4 (in-house antibody), antitubulin (DM1A; Sigma), and <br /> and proliferation. In accordance idea, disruption                           anti-H3 (ab 1791; Abcam) incubated 2 h room temperature. Proteins <br /> the HDAC1 gene resulted reduced proliferation mouse                                visualized chemiluminescence ECL Western blotting detection <br /> embryos ES cells (37), overexpression HDAC1                               reagents (Amersham Biosciences). <br />                                                                                             Growth curve. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> determine effects siRNA knockdown cell prolif- <br /> led impaired cell proliferation murine fibroblasts (3).                            eration, 24 h transfection, cells collected trypsinization, counted by <br /> Taken together, results suggest class I HDACs                             use hemacytometer trypan blue dye, plated 3,000 viable cells/well <br /> crucial controlling proliferation state mammalian                              U2OS MCF7 2,000 viable cells/well MCF10A 96-well tissue <br /> cells.                                                                                   culture dish final volume 200 &#226;?&#174;l. Every 24 h, 96-well tissue culture dish <br />    HDAC1 HDAC2 heterodimerize gener-                                   stained crystal violet solution left dry. Then, crystal violet <br />                                                                                          incorporated cellular membrane solubilized solution 10% <br /> ally large multiprotein complexes: Sin3, NuRD,                              acetic acid&#226;&#8364;&#8220;phosphate-buffered saline (PBS) absorbance 595 nm was <br /> and CoREST complexes (16). Numerous transcription factors,                               measured MRX microplate reader (Dynatech). We calculated in- <br /> including regulators cell cycle, differentiation, develop-                        crease (n-fold) absorbance ratio absorbance time i/absorbance <br /> ment, shown associate directly HDAC1                               time zero (where time 48, 72, 96 h siRNA transfection time <br /> HDAC2 HDAC1/HDAC2 complexes, mediat-                                     zero 48 h transfection). Each experimental point average three <br />                                                                                          independent experiments, respective standard deviation. <br /> ing repression specific target genes (1, 7, 42). Further-                            Colony formation assay. Twenty-four hours transfection, cells col- <br /> more, loss Sds3, component Sin3/HDAC1/                                   lected trypsinization, counted use hemacytometer trypan blue dye, <br /> HDAC2 complex putative recruiting factor, led cell                              plated 3,000 viable cells/well U2OS MCF7 2,000 viable cells/ <br /> death missegregation chromosomes mitosis                                MCF10A six-well tissue culture dish. Six days plating, cells <br /> (8). Therefore, HDAC1- HDAC1/HDAC2-mediated                                     stained crystal violet solution (1% crystal violet, 35% ethanol) and <br />                                                                                          colonies counted. We calculated relative CFU follows: number of <br /> chromatin modifications important cell cycle                              colonies sample/number colonies control. The average relative <br /> control development. In light evidence, con-                              CFU plates calculated, respective standard deviation. <br /> ducted study analyze effect depleting class I deacety-                          Fluorescence-activated cell sorter (FACS) analysis cell cycle. To detect <br /> lases human tumor cells RNA interference (RNAi). We                                HDAC1, phosphorylated S10 H3 (H3-P), DNA HDAC1, cyclin A, and <br /> show ablating HDAC1 HDAC3 protein                                 DNA, 72 h cycle transfection, attached cells superna- <br />                                                                                          tant collected resuspended 106 cells/ml 1% formaldehyde 15 <br /> expression inhibition tumor cell proliferation.                           min. After PBS washing, pellet resuspended ice-cold 70% ethanol and <br /> Clearly, HDAC1 knockdown abolishes ability tumor                                  stored 4&#194;&#176;C. For immunodetection, cells washed twice PBS per- <br /> cells proceed mitosis. On contrary, HDAC2                                 meabilized 0.1% Triton X-100 PBS 10 min. After blocked 5% <br /> knockdown shows effect, unless concurrently knock                                  normal goat serum 20 min, cells incubated anti-HDAC1 mouse <br /> down HDAC1 HDAC2. In addition, con-                                     monoclonal antibody (Upstate), 1:250, anti-H3-P rabbit polyclonal antibody <br />                                                                                          (Upstate) anti-cyclin A (Pharmingen), 1:100, PBS plus 1% bovine serum <br /> ducted analysis changes gene expression U2OS                             albumin 2 h room temperature. Cells rinsed PBS incubated for <br /> cells response knockdown deacetylases demon-                                1 h Cy5-conjugated goat anti-mouse (1:50; Jackson Immunoresearch) and <br /> strated large number HDAC target genes involved                            fluorescein isothiocyanate (FITC)-conjugated goat anti-rabbit antibodies. Cells <br /> in proliferation apoptosis.                                                          washed resuspended 1 ml solution containing 2.5 &#226;?&#174;g/ml <br />    The overall evidence points idea                           propidium iodide (PI) (Sigma) PBS 250 &#226;?&#174;g/ml RNase PBS incu- <br />                                                                                          bated overnight 4&#194;&#176;C acquisition. <br /> individual class I HDAC plays different role regulating                                Samples acquired FACSCalibur (Becton Dickinson) flow cytometer. <br /> genes involved various biological processes.                                          At 10,000 events acquired. Analysis performed using CellQuest <br />                                                                                          3.3 (Becton Dickinson). <br />                                                                                             FACS analysis activation caspase-3. For active caspase-3 analysis, 96 h <br />                          MATERIALS AND METHODS                                           cycle transfection, attached cells supernatant were <br />    Cell culture. U2OS MCF7 cell lines grown Dulbecco&#226;&#8364;&#8482;s modified              collected, fixed 1% formaldehyde PBS 70% cold ethanol, and <br /> Eagle&#226;&#8364;&#8482;s medium (Cambrex) supplemented 10% fetal bovine serum (Cam-                  permeabilized 0.1% Triton X-100 10 min. Prior incubation anti- <br /> brex), antibiotics, 2 mM L-glutamine (Cambrex). The MCF10A cell line             caspase-3 polyclonal antibody (9661L; Cell Signaling), cells incubated for <br /> maintained F12-Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s medium (1:1) supplemented                  30 min 10% goat serum PBS. After washed, cells incubated <br /> with 5% horse serum, 20 ng/ml epidermal growth factor, 0.5 &#226;?&#174;g/ml hydrocorti-             anti-rabbit FITC antibody (Sigma) 1 h room temperature (light <br /> sone, 50 ng/ml cholera toxin, 10 &#226;?&#174;g/ml insulin, antibiotics. All cell lines     protected). The labeled cells stained overnight 4&#194;&#176;C 2.5 &#226;?&#174;g/ml PI <br /> cultured humidified 37&#194;&#176;C incubator 5% CO2.                                     RNase 250 &#226;?&#174;g/ml PBS. The cellular suspension passed a <br />    Transfection cells siRNA, electrophoresis, Western blotting.              FACSCalibur (Becton Dickinson) flow cytometer. At 10,000 events were <br /> siRNAs ordered MWG BIOTECH prepared according                       acquired, analysis performed using CellQuest 3.3 (Becton Dickinson) <br /> manufacturer&#226;&#8364;&#8482;s instructions (stock concentration 20 &#226;?&#174;M molecular-biology-          ModFit LT 3.1 software. <br /> grade water). Oligofectamine used siRNA transfections (Invitrogen,              Cell synchronization. Twenty-four hours cycle siRNA trans- <br /> San Diego, CA). The following siRNA sequences used: 5&#226;&#172;&#732;-CGUACGCG                     fection (see above), cells synchronized G1/S transition treatment <br /> GAAUACUUCGATT-3&#226;&#172;&#732; siRNA LUC (control), 5&#226;&#172;&#732;-CAGCGACUGUUUG                               2 mM thymidine 14 h, released 8 h, treated with <br /> AGAACCTT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CUAAUGAGCUUCCAUACAATT-3&#226;&#172;&#732; siRNA                                    thymidine 14 h. After washes PBS, cells cultured fresh <br /> HDAC1, 5&#226;&#172;&#732;-UCCGUAAUGUUGCUCGAUGTT-3&#226;&#172;&#732; siRNA HDAC2,                                  medium medium 0.1 &#226;?&#174;g/ml nocodazole different times (as indicated <br /> 5&#226;&#172;&#732;-GAUGCUGAACCAUGCACCUTT-3&#226;&#172;&#732; siRNA HDAC3.                                             experiment) harvested. Cells synchronized prometaphase by <br />    Oligofectamine reagent diluted used according manufacturer&#226;&#8364;&#8482;s           treatment 0.1 &#226;?&#174;g/ml nocodazole 16 h released the <br /> instructions. The siRNA used optimized fixed                  drug-induced cell cycle block washed times PBS. <br /> knockdown experiment. A second cycle siRNA transfection performed                    Time-lapse microscopy. Time-lapse microscopy performed use Cell <br /> 24 h later. Cells harvested 72 h cycle transfection stored   R imaging station live-cell microscopy (Olympus) mounted IX81 <br /> at &#226;&#171;&#186;80&#194;&#176;C lysis urea buffer (8 M urea, 0.1 M NaH2PO4, 0.01 M Tris,               inverted microscope (Olympus) equipped incubation chamber (Evotec). <br /> pH 8).                                                                                   The includes Hamamatsu ORCA-ER charge-coupled-device camera <br />  4786       SENESE ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. siRNA-mediated knockdown HDAC1, HDAC2, HDAC3 U2OS, MCF7, MCF10A cells. The loss HDAC1 and <br /> HDAC2 produced hyperacetylation histones H3 H4. (A C) Relative (rel.) expression levels indicated mRNAs determined <br /> by real-time PCR, described Materials Methods. Averages independent experiments shown, standard deviations are <br /> indicated error bars. (D F) Western blots U2OS whole-cell extracts transfection siRNA HDAC1, HDAC2, HDAC3, and <br /> HDAC1 plus HDAC2. Cells transfected processed described Materials Methods, 30 &#226;?&#174;g loaded lane. As control <br /> (CTRL), used transfection antiluciferase siRNA. <br />  <br />  <br />  <br /> for image acquisition. A movie montage performed using ImageJ image            establish arbitrary threshold used discriminate HDAC1-positive (gate II) <br /> analysis software (W. Rasband, National Institutes Health, Bethesda, MD).       HDAC1-negative (gate I) cells. H3-P fluorescence distribution evalu- <br />    Statistical immunofluorescence. Forty-eight hours siRNA transfection,     ated gate I gate II fix value cells H3-P positive and <br /> cells plated four-chamber slides pretreated 15 &#226;?&#174;g/ml poly(D)lysine    negative. We evaluated gate I gate II the <br /> (Sigma) treated 0.1 &#226;?&#174;g/ml nocodazole 16 h enrich mitotic       percentages double positive cells. <br /> population. Slides fixed 4% paraformaldehyde&#226;&#8364;&#8220;1&#226;&#171;&#187; PIPES [piperazine-            Extraction RNA reverse transcription-PCR. For validation mi- <br /> N,N&#226;&#172;&#732;-bis(2-ethanesulfonic acid)] 10 min, permeabilized 0.1% Triton        croarray results, employed independent RNA preparations samples <br /> X-100 0.2% bovine serum albumin PBS, processed using standard           described above. Total RNA isolated using QIAGEN RNeasy Protect <br /> protocols. The primary antibodies used anti-HDAC1 monoclonal (Upstate)        mini kit described manufacturer. cDNA generated reverse <br /> at 1:250 anti-H3-P rabbit polyclonal (Upstate) 1:100, conjugated    transcription-PCR PE Applied Biosystem TaqMan reverse transcription <br /> secondary antibodies Cy3-conjugated goat anti-mouse (Jackson Laborato-        reagents. Relative levels specific mRNA determined 5&#226;&#172;&#732; nuclease <br /> ries) FITC-conjugated goat anti-rabbit (Sigma). DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamino-2-         assay (TaqMan) chemistry system. All PCRs performed ABI <br /> phenylindole) (Sigma) used nuclear counterstaining. Fluorescence mi-       7900HT sequence detection system. The GAPDH (glyceraldehyde-3-phosphate <br /> croscopy images acquired using BX61 (Olympus) motorized fluorescence        dehydrogenase) gene used control gene normalization. <br /> microscope. Acquired images visualized using browser developed        Microarray analysis. Biotin-labeled cRNA targets obtained 5 &#226;?&#174;g of <br /> ImageJ macro language. After acquisition DAPI, Cy3, FITC channels,          total RNA derived samples described above. cDNA synthesis per- <br /> DAPI staining employed identify cell nuclei build mask         formed Gibco SuperScript custom cDNA synthesis kit, biotin-labeled <br /> successive calculations. The mean nuclear levels DAPI, HDAC1, H3-P          antisense RNA transcribed using vitro transcription (Ambion, <br /> were stored memory allow later statistical analysis. HDAC1 fluores-      Inc., Austin, TX) including Bio-11-UTP Bio-11-CTP reaction (NEN <br /> cence distribution evaluated control- HDAC1-interfered cells order   Life Sciences, Boston, MA). GeneChip hybridization, washing, staining, and <br />  VOL. 27, 2007                                                   ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                                4787 <br />  <br />  <br />  <br />  <br />   FIG. 2. Absence HDAC1 HDAC3 causes defective proliferation. (A C) Growth curves U2OS, MCF7, MCF10A transfected <br /> with HDAC siRNAs. The increase absorbance 595 nm directly correlated cell proliferation, described Materials Methods. Each <br /> curve represents average independent experiments, standard deviation indicated error bars. (D F) CFU assay U2OS, <br /> MCF7, MCF10A cells transfected HDAC siRNAs. The ratio CFU number sample/CFU number control calculated. <br /> Each column represents average independent experiments, respective standard deviations. CTRL, control. <br />  <br />  <br />  <br /> scanning performed according Affymetrix protocols (Santa Clara, CA).   lated function HDAC1 prevent inappropriate ex- <br /> Two copies HG-U133Plus GeneChip hybridized target.       pression cyclin-dependent kinase (CDK) inhibitors p21 <br /> Full details microarray methods described supplementary data <br /> posted http://bio.ifom-ieo-campus.it/supplementary/MCB49407/. <br />                                                                                p27 (37). In accordance model, promoters of <br />                                                                                genes p21 p27 hyperacetylated HDAC1 null <br />                                                                                cells (37). In addition, HDAC1 recruited p21 <br />                                RESULTS                                         gene transcription factor SP1 (36) Tbx2 (51) and <br />    siRNA-mediated knockdown HDAC1 HDAC3. We es-                          p21 genes upregulated HDAC inhibitor <br /> tablished methodology HDAC class I knockdown                        treatment (29, 30, 47, 48). Therefore, analyzed endogenous <br /> various cell lines: U2OS (an osteosarcoma cell line), MCF7 (a                  p21 protein levels RNAi treatment. Figure 1D indicates <br /> mammary carcinoma cell line), MCF10A (an epithelial                        p21 levels increase absence HDAC1 alone <br /> mammary gland cell line). As shown Fig. 1A F,                        HDAC1 plus HDAC2 U2OS cells, accordance with <br /> expression levels HDACs efficiently knocked                      HDAC1 null embryo results (37) effect HDAC <br /> down respective siRNAs. It interesting note                  inhibitors (47). Conversely, MCF7 MCF10A, we <br /> the levels endogenous HDAC1 HDAC2 cell                     did observe changes p21 levels HDAC1 or <br /> lines increase opposition (Fig. 1D F), i.e.,                HDAC1 HDAC2 knockdown (data shown). Interest- <br /> upon knockdown increases protein                   ingly, absence HDAC1 HDAC2 did <br /> level. On hand, observed decrease                       Western blot analysis modification-specific antibodies re- <br /> HDAC1 HDAC2 HDAC3 knockdown,                              veal increased acetylation levels subset histones H3 and <br /> the protein level. We know reason                    H4 (Fig. 1D F) cell lines. This result could <br /> result, preliminary data indicate proteasome de-                indicate histones substrates HDACs <br /> pendent. Nevertheless, reduced expression HDAC1                         concurrent double knockdown HDAC1 <br /> HDAC2 consequence reduced cellular                      plus HDAC2 leads hyperacetylation. We noted <br /> viability HDAC3 knockdown (Fig. 2).                                      specific knockdown HDAC did not <br />    The absence HDAC1 induces p21 expression                     affect basal levels p53 acetylation (data shown), <br /> U2OS cell line, absence HDAC1 HDAC2                        reflecting idea multiple deacetylases involved in <br /> increases histone acetylation. One important proliferation-re-                 regulating p53 acetylation (28, 39, 52). <br />  4788     SENESE ET AL.                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />    siRNA-mediated knockdown HDAC1 HDAC3 dimin-                 The panel Fig. 4A represents biparametric anal- <br /> ishes cellular viability proliferation. It known          ysis, using anti-H3-P PI staining levels U2OS cells. <br /> HDAC inhibitors lead p21 induction. They induce cell          This analysis shows increase G1 cells absence of <br /> cycle arrest G1 G2/M apoptosis, killing tumor        HDAC1 time points thymidine release, accompa- <br /> cells (4, 29, 48). The pathway leading tumor cell death   nied lower percentage mitotic cells. <br /> unknown, unlikely single molecular pathway         Next, released U2OS cells double thymidine <br /> will identified cell types HDAC inhibitors. We      block nocodazole-containing culture medium enrich <br /> decided investigate cellular viability proliferation        mitotic population. Biparametric analysis using anti-cyclin <br /> U2OS, MCF7, MCF10A cells HDAC knockdown.                    A, G2 marker, PI staining (Fig. 4B) showed 24 h <br /> Clearly, absence HDAC1, HDAC3, HDAC1 plus                  control cells blocked mitosis expected that <br /> HDAC2 generated remarkable defect proliferation          HDAC1 knockdown cells unable remain mitosis. In <br /> U2OS MCF7 cell lines (Fig. 2A B)            fact, time, larger G1/S HDAC1- <br /> unnoticeable effect MCF10A cell line (Fig. 2C). The lack       negative cells, accompanied significant decrease mitotic <br /> of HDAC2 pronounced effect cells (Fig.         cells increase sub-G1 cells. <br /> 2A C). This inconsistent published data prob-          We analyzed proportion sub-G1 cells re- <br /> ably different cell lines used (24). We analyzed      lease nocodazole block. As shown Fig. 4C, we <br /> the effect absence HDACs life spans        observed massive apoptosis HDAC1-negative U2OS cells. <br /> of cell lines. In correlation results Fig. 2A   Taken together, data demonstrate proliferative <br /> to C, cells lacking HDAC1, HDAC3, combination                  defect HDAC1 knockdown cells partial G1 <br /> HDAC1 plus HDAC2 diminished capacity form col-               arrest cell death G2/M transition the <br /> onies plated low density (Fig. 2D F). These data         beginning mitosis. Alternatively, HDAC1-deficient <br /> suggest HDAC1 (either combination          cells insensitive nocodazole grow be <br /> HDAC2) HDAC3 important roles prolifera-               G1 tetraploids. However, followed living cells time- <br /> tion tumor cells, agreement published data indicating      lapse microscopy, observed inability HDAC1-nega- <br /> a relevant role murine HDAC1 regulation prolif-         tive cells condense enter mitosis, suggesting these <br /> eration development (37). An HDAC3-AKAP95/HA95-                   cells die G2/M transition (Fig. 4D). Our data confirm <br /> Aurora B pathway regulating mitosis demonstrated             extend published findings showing HDAC inhibitors <br /> recently, explaining mitotic defect observed HDAC3          induce G1 and/or G2/M arrest and/or apoptosis (4). <br /> siRNA (38). It interesting note cell line              Induction apoptosis HDAC1-deficient U2OS cells. The <br /> MCF10A, untransformed counterpart MCF7, ab-                observation differences numbers apop- <br /> sence HDACs effect cellular prolifer-        totic cells HDAC1 knockdown prompted confirm <br /> ation.                                                                apoptotic cell death examining caspase-3 activation, by <br />    HDAC1 knockdown cells marked reduction H3                FACS analysis (Fig. 5A) Western blot analysis (Fig. 5B) <br /> S10 phosphorylation. H3-P mitotic marker. To explore             (see Materials Methods). Our results indicate that <br /> whether defect proliferation HDAC1 knockdown                caspase-3 activation involved apoptosis HDAC1 <br /> cells mitotic defect, applied single-cell statis-     knockdown. <br /> tical analysis control-interfered HDAC1-inter-               Comparison gene expression profiles depletion of <br /> fered cells using anti-HDAC1 antibody anti-H3-P          HDAC1, HDAC2, HDAC3, HDAC1 plus HDAC2. We then <br /> antibody (see Materials Methods). We analyzed           analyzed gene expression profiles U2OS cells loss <br /> cell lines mitotic cells presence absence           HDAC1, HDAC2, HDAC3, HDAC1 plus HDAC2 by <br /> HDAC1 (Fig. 3). Figure 3A, B, C histograms report-           using HG-U133 Plus Affymetrix oligonucleotide chips, which <br /> ing distributions HDAC1 levels control-interfered           explore expression approximately 54,000 human tran- <br /> populations HDAC1-interfered populations, described            scripts. Results analyzed using GCOS elab- <br /> in Materials Methods. As shown upper panels,            orated GenePicker GeneSpring software (see the <br /> were able discriminate HDAC1-positive cells (gate          supplementary data posted http://bio.ifom-ieo-campus.it <br /> II) HDAC1-negative cells (gate I). The middle panels              /supplementary/MCB49407/) (2, 10). A detailed description of <br /> represent distributions H3-P levels gates        microarray methods, data analysis elaboration, com- <br /> order define basal level H3-P-positive cells. In        parative results supplementary data the <br /> lower panels, correlation HDAC1 H3-P levels           URL mentioned above. <br /> in gates represented. Interestingly, knockdown             Ablation HDAC1, HDAC2, HDAC3, HDAC1 plus <br /> HDAC1 led clear reduction number H3-P-                 HDAC2 resulted altered expression 987, 283, 1,317, <br /> positive cells, suggesting HDAC1 important          601 genes, respectively (see supplementary Table 1 posted <br /> proper cell cycle progression. Again, effect           http://bio.ifom-ieo-campus.it/supplementary/MCB49407/). <br /> drastic MCF10A cell line (Fig. 3C).                            There large degree overlap lists genes <br />    HDAC1 knockdown cells arrest G1 die               regulated siRNA HDAC1 regulated siRNA <br /> during G2/M transition. We tested HDAC1                  HDAC1 plus HDAC2. Ablation HDAC2 results mild <br /> knockdown affects cell cycle progression. U2OS cells             <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> and, compared conditions, associated <br /> synchronized double thymidine block G1/S transi-          deregulated expression genes, 86% are <br /> tion, cell cycle progression removal thymidine        deregulated above-mentioned con- <br /> monitored FACS analysis PI-stained DNA (Fig. 4A, top).          ditions. Only 31% genes deregulated HDAC3 siRNA are <br />  VOL. 27, 2007                                              ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                                 4789 <br />  <br />  <br />  <br />  <br />   FIG. 3. Absence mitotic marker H3-P HDAC1-deficient cells. U2OS, MCF-7, MCF10A cells analyzed presence of <br /> H3-P. (Top) Distribution HDAC1 signal siRNA-control (CTRL)-transfected siRNA-HDAC1-transfected populations. The <br /> dashed line represents arbitrary threshold used distinguish HDAC1-negative (gate I) HDAC1-positive (gate II) cells. (Middle) <br /> Distribution H3-P signals gate I gate II. The dashed line represents arbitrary threshold used distinguish positive and <br /> negative H3-P cells. (Bottom) Scatter plot HDAC1 signal versus H3-P signal. The percentages inside plot indicate percentages double <br /> positive cells. <br />  <br />  <br /> in common regulated siRNA HDAC1                        deregulated knockdown HDAC1 HDAC1 plus <br /> HDAC1 plus HDAC2 (see supplementary data sup-                      HDAC2, according Gene Ontology Biological Process cat- <br /> plementary Table 1 posted URL mentioned above).                     egories (for complete description, supplementary <br /> Hierarchical clustering using lists regulated genes          data posted http://bio.ifom-ieo-campus.it/supplementary <br /> confirms higher degree similarity gene                /MCB49407/). To investigate molecular basis HDAC1- <br /> expression profiles resulting ablation HDAC1, HDAC2,               mediated growth arrest, focused attention func- <br /> or HDAC1 plus HDAC2 resulting ablation                  tional categories: apoptosis proliferation. We 36 <br /> of HDAC3 (Fig. 6).                                                         probe sets (corresponding 28 nonredundant genes) that <br />   We performed functional classification genes                   classified relevant apoptosis 100 probe sets <br />  4790     SENESE ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. HDAC1-deficient cells arrest G1 die G2/M transition. (A) Asynchronous (As) siRNA-control (CTRL)-transfected (black) <br /> and siRNA-HDAC1-transfected (gray) U2OS cells blocked G1/S transition use thymidine released fresh medium. (Top) <br /> DNA content analyzed FACS analysis PI-stained cells different time points release fresh medium. (Bottom) Biparametric <br /> FACS analysis using anti-H3-P PI staining performed calculate percentages cells G1, S, G2, M phases time point. <br /> (B) U2OS control- HDAC1-interfered cells blocked G1/S transition use thymidine released medium containing <br /> nocodazole. (Top) DNA content analyzed FACS analysis PI-stained cells different time points. (Middle) Biparametric FACS analysis <br />  VOL. 27, 2007                                                ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                                   4791 <br />  <br />  <br />  <br />  <br />    FIG. 5. HDAC1 knockdown cells undergo apoptosis. (A) Effect of <br /> siRNA HDAC1 caspase-3 activation FACS analysis. The bars <br /> represent percentages cleaved caspase-3-positive cells control <br /> (CTRL) siRNA-HDAC1-transfected U2OS cells 96 h. The <br /> values correspond averages independent experiments, <br /> and error bars indicate associated standard deviations. <br /> (B) Western blots U2OS whole-cell extracts transfection of <br /> control siRNA HDAC1. Cells collected 120 h transfec- <br /> tion, 50 &#226;?&#174;g lysates loaded lane. As control, used <br /> an injection antiluciferase siRNA. The anti-caspase-3 antibody used <br /> was able recognize full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> inactive cleaved active <br /> forms. <br />  <br />  <br /> (corresponding 72 nonredundant genes) relevant prolif-                   FIG. 6. Gene expression profiles. Supervised hierarchical clustering <br /> eration. Three genes present categories (F2R,                 (according Pearson&#226;&#8364;&#8482;s correlation) performed using comparative <br />                                                                            expression values gene lists indicated dendrogram (for <br /> interleukin-1A [IL1A], SPP1 genes). Figure 7 shows                   gene lists, supplementary Table 1 posted http://bio.ifom-ieo-campus <br /> graphical representation behavior genes                 .it/supplementary/MCB49407/). In cases, expression value the <br /> U2OS cells lacking HDAC1 HDAC1 plus HDAC2 com-                          control cells used baseline (value 1). <br />  <br />  <br />  <br />  <br /> using anti-cyclin A (CYCA) PI staining performed calculate percentages cells G1/S, G2, M phases time point. <br /> (Bottom) DNA content analyzed FACS analysis evaluate percentages sub-G1 cells. (C) U2OS control- HDAC1-interfered <br /> cells blocked G2/M transition use nocodazole released fresh medium. (Top) DNA content analyzed FACS analysis <br /> on PI-stained cells different time points. (Bottom) DNA content analyzed FACS analysis evaluate percentages sub-G1 cells. (D) <br /> (Top) Percentages cells control- HDAC1-interfered population able undergo mitosis 30 h. (Bottom) Images of <br /> time-lapse experiments U2OS siRNA-CTRL-transfected siRNA-HDAC1-transfected cells. Control cells experience normal mitosis <br /> (white arrowheads), HDAC1 knockdown cells enter mitosis die (yellow black/white arrowheads). <br />  4792     SENESE ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Gene expression profiles quantitative real-time PCR. (A) Graphic representation expression patterns genes functionally <br /> involved apoptosis proliferation regulated ablation HDAC1 HDAC1 plus HDAC2 U2OS cells. (B) Validation microarray data <br /> by TaqMan quantitative real-time PCR. The error bars indicate standard deviations independent experiments. CTRL, control. <br />  <br />  <br />  <br />  <br /> pared behavior vector-transfected U2OS cells (con-             .it/supplementary/MCB49407/. To verify changes gene <br /> trol). Clearly, genes involved apoptosis induced            expression detected microarray analysis, using <br /> than repressed RNAi HDAC1 and/or HDAC1 plus                        independent RNA preparations described experimental <br /> HDAC2. Multiple proapoptotic genes, including FAS, BCL2-                 conditions, performed quantitative real-time PCR analysis <br /> like 1, members tumor necrosis factor (TNF) family            19 genes expression altered regulate <br /> receptors ligands (TNFRSF10B, TNFRSF11B, TNF                     proliferation apoptosis (cyclin D2 [CCND2], CCNE2, <br /> RSF12A), activated. Conversely, approximately equal                 CDKN1A [p21], HDAC9, IL6R, IL-9, MYC, ubiquitin-specific <br /> numbers genes involved cell proliferation up-             peptidase 16 [USP16], CCNG2, cell division cycle 25C [CDC25C], <br /> downregulated. The complete gene lists sup-              CDKN2C, nuclear factor 1A [NF1A], NFIB, septin 6 [SEP6], <br /> plementary Tables 1 2 posted http://bio.ifom-ieo-campus           transforming growth factor &#226;?&#164;2 [TGFB2], TNFRSF10B, <br />  VOL. 27, 2007                                            ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                        4793 <br />  <br />  <br /> TNFRSF11B, IL-24, IL1A genes). As shown Fig. 7,          lack mitotic cells absence HDAC1 (Fig. 3 and <br /> was high correlation microarray real            4). This result implies HDAC1 essential mitosis, a <br /> -time data 19 genes.                                          yet-undescribed observation. As expected published data <br />    We consistency recent study identi-         showing HDAC6 major tubulin deacetylase that <br /> fying genes apoptosis cell proliferation      HDAC1 specific tubulin deacetylase activity (25, 57), we <br /> pathways regulated different HDAC inhibitors            did observe changes tubulin acetylation following <br /> 16-h culture treatment (46). Although siRNA experi-              siRNA-mediated knockdown HDAC1 HDAC3 (data not <br /> ments difficult discern early late events,    shown). All cells interfered HDAC1 nuclear <br /> is interesting note study identified that, exam-     appearance typical interphase apoptosis. Taken together, <br /> ple, cell division cycle gene CDC25C genes encoding           data suggest human tumor cells lacking HDAC1 <br /> TNF family receptors ligands modulated               enter mitosis undergo apoptosis through <br /> HDAC inhibitors (46). Altogether, findings support          caspase-3 activation. Future work characterize the <br /> idea HDAC inhibition affects expression genes             mechanism mitotic arrest. Our data are, however, consistent <br /> within pathways regulate tumor cell growth survival.         studies pointing importance histone deacetyla- <br />                                                                       tion formation pericentric heterochromatin (9, 50) <br />                                                                       showing loss component Sin3/HDAC1/ <br />                          DISCUSSION <br />                                                                       HDAC2 complex leads cell death missegregation of <br />    HDAC inhibitors induce cell cycle arrest, apoptosis, dif-       chromosomes mitosis (8). Interestingly, loss HDAC1 <br /> ferentiation tumor cells, number entered        untransformed cells, ES cells, fibroblasts, neu- <br /> early-phase clinical trials. Based phenotype HDAC1-         rons, does induce apoptosis (37; G. Lagger, R. Grausen- <br /> deficient embryos ES cells, HDAC1 implicated               burger, C. Seiser, unpublished data), a <br /> proliferation control (37). In study, HDAC1         possible checkpoint activation cells compensation <br /> has important role proliferation human tumor cells       HDAC2. <br /> and influences capacity undergo mitosis. Al-            Based microarray results, analysis global gene <br /> though study shows absence HDAC3           expression profiles U2OS cells absence HDAC1, <br /> drastic effect cell proliferation, directed attention       HDAC2, HDAC3, HDAC1 plus HDAC2 yielded large <br /> mostly HDAC1 knockdown phenotype. Clearly,                 number genes present altered expression levels (see <br /> microarray data indicate HDAC3 favors            supplementary data posted http://bio.ifom-ieo-campus.it <br /> uniquely expressed genes, compared HDAC1 HDAC1                 /supplementary/MCB49407/). HDAC1 HDAC1 plus HDAC2 <br /> plus HDAC2 (Fig. 6) (also supplementary Table 1 posted            knockdown regulated largely overlapping set genes, in- <br /> at http://bio.ifom-ieo-campus.it/supplementary/MCB49407/),            cluding regulated HDAC2 knockdown. <br /> suggesting different specific pathways action. Recent          HDAC3 knockdown caused deregulation vast number <br /> work clearly shown HDAC3 novel HDAC             genes. Interestingly, 50% genes induced HDAC3 <br /> pathway regulating mitosis Aurora B kinase activity modu-          knockdown induced ablation HDAC1 or <br /> lation (38). The aim study define role   HDAC1 plus HDAC2 expression, 21% genes <br /> of HDAC1 tumor cells and, parallel, provide molecular        repressed HDAC3 knockdown common the <br /> insight HDAC1 functions conducting global gene              conditions. These results consequence the <br /> expression study U2OS cell line. It known          protein reduction HDAC1 HDAC2 observed after <br /> effects HDAC inhibitors cell line specific          HDAC3 knockdown (Fig. 1). <br /> HDAC inhibitor selective (14), providing ground detailed             Again, analyzed greater genes regulated by <br /> studies HDAC responsible phenotype           HDAC1. In accordance proposed role HDAC1 as <br /> the cell line interest. It demonstrated     transcriptional repressor, HDAC1 target genes were <br /> expression profiles induced HDAC inhibitors cell line          upregulated downregulated absence HDAC1. <br /> dependent that, expression patterns HDACs            Interestingly, result evened HDAC1 and <br /> vary cell line cell line, anticipate different         HDAC2 ablated. This line reports that <br /> effects <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>knockout</span> particular HDAC (4, 14). In              demonstrated independent recruitment HDAC1 <br /> U2OS MCF7 cells, absence HDAC1 remark-               HDAC2 target genes (26, 49). <br /> able effect apoptosis proliferation phe-            Because phenotype observed absence of <br /> notype exacerbated simultaneous lack HDAC2.             HDAC1 (and HDAC1 plus HDAC2), validated genes in- <br /> Interestingly, MCF10A, untransformed counterpart           volved apoptosis proliferation. Notably, number of <br /> MCF7 cells, sensitive lack HDAC1. In fact,         cyclins, cyclin-dependent kinases, cyclin-dependent ki- <br /> the effects proliferation cell cycle          nase inhibitor p21, expression tightly coordinated to <br /> evident (Fig. 2 3). These results agreement       regulate appropriate cell cycle progression, induced after <br /> idea HDAC inhibitors antiproliferative proapop-         RNAi treatment HDAC1. p21 regarded an <br /> totic properties mainly transformed cells (4, 41). We         attractive therapeutic target human cancer. Although a <br /> identified caspase-3 mediator apoptosis provoked       plethora data regarding role multifunctional <br /> the absence HDAC1. Caspase-3 proapoptotic enzyme              protein cellular pathways emerged, precise <br /> and triggers apoptosis intrinsic death       mechanism p21 regulates cell cycle progression still <br /> ligand pathways (20, 21). It good apoptosis       unknown, partly p21 act positive a <br /> indicator. However, striking effect observed          negative fashion cell proliferation (53). Given p21 <br />  4794      SENESE ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> expression increased HDAC1 knockdown U2OS                                 10. Finocchiaro, G., P. Parise, S. P. Minardi, M. Alcalay, H. Muller. 2004. <br />                                                                                          GenePicker: replicate analysis Affymetrix gene expression microarrays. <br /> cells MCF7 cells, argue mitotic                             Bioinformatics 20:3670&#226;&#8364;&#8220;3672. <br /> phenotype observed tumor cell lines linked                       11. Fischle, W., F. Dequiedt, M. J. Hendzel, M. G. Guenther, M. A. Lazar, W. <br /> to p21. Our microarray data showed repression                                    Voelter, E. Verdin. 2002. Enzymatic activity associated class II <br />                                                                                          HDACs dependent multiprotein complex containing HDAC3 and <br /> CDC25C septin 6. In Saccharomyces cerevisiae, septins                            SMRT/N-CoR. Mol. Cell 9:45&#226;&#8364;&#8220;57. <br /> are family cell division cycle regulatory proteins                     12. Gao, L., M. A. Cueto, F. Asselbergs, P. Atadja. 2002. Cloning and <br /> been associated actin stress fibers interphase cells,                        functional characterization HDAC11, novel member human <br />                                                                                          histone deacetylase family. J. Biol. Chem. 277:25748&#226;&#8364;&#8220;25755. <br /> cleavage furrow dividing cells, bud neck budding                       13. Glaser, K. B., J. Li, M. J. Staver, R. Q. Wei, D. H. Albert, S. K. Davidsen. <br /> yeast (reviewed reference 31). The human homologue                                 2003. Role class I class II histone deacetylases carcinoma cells using <br />                                                                                          siRNA. Biochem. Biophys. Res. Commun. 310:529&#226;&#8364;&#8220;536. <br /> septin 6 fused MLL cases                         14. Glaser, K. B., M. J. Staver, J. F. Waring, J. Stender, R. G. Ulrich, S. K. <br /> infant acute myeloid leukemia (43). Cdc25C phosphatase                              Davidsen. 2003. Gene expression profiling multiple histone deacetylase <br /> major cell cycle regulator mammals. Its C-terminal catalytic                          (HDAC) inhibitors: defining common gene set produced HDAC inhi- <br />                                                                                          bition T24 MDA carcinoma cell lines. Mol. Cancer Ther. 2:151&#226;&#8364;&#8220;163. <br /> phosphatase domain dephosphorylates Cdc2 promote G2/M                             15. Gregoretti, I. V., Y. M. Lee, H. V. Goodson. 2004. Molecular evolution <br /> transition (27), N-terminal regulatory domain con-                           histone deacetylase family: functional implications phylogenetic <br /> tains multiple phosphorylation sites. Multisite phosphorylation                          analysis. J. Mol. Biol. 338:17&#226;&#8364;&#8220;31. <br />                                                                                      16. Grozinger, C. M., S. L. Schreiber. 2002. Deacetylase enzymes: biological <br /> of Cdc25C different kinases recruits specific proteins, con-                          functions use small-molecule inhibitors. Chem. Biol. 9:3&#226;&#8364;&#8220;16. <br /> trolling normal cellular mitotic progression (reviewed                        17. Gurley, L. R., R. A. Walters, R. A. Tobey. 1973. Histone phosphorylation <br /> reference 56). Clearly, Septin6 Cdc25C impor-                              late interphase mitosis. Biochem. Biophys. Res. Commun. 50:744&#226;&#8364;&#8220; <br />                                                                                          750. <br /> tant roles mitosis. Finally, activation interleu-                     18. Halkidou, K., L. Gaughan, S. Cook, H. Y. Leung, D. E. Neal, C. N. <br /> kins HDAC ablation postapoptotic                             Robson. 2004. Upregulation nuclear recruitment HDAC1 hormone <br />                                                                                          refractory prostate cancer. Prostate 59:177&#226;&#8364;&#8220;189. <br /> activation inflammation response.                                              19. Hauser, C., B. Schuettengruber, S. Bartl, G. Lagger, C. Seiser. 2002. <br />    In summary, defined role HDAC1                                   Activation mouse histone deacetylase 1 gene cooperative histone <br /> proliferation human tumor cells and, importantly,                                phosphorylation acetylation. Mol. Cell. Biol. 22:7820&#226;&#8364;&#8220;7830. <br />                                                                                      20. Henderson, C., C. Brancolini. 2003. Apoptotic pathways activated by <br /> identified mitotic defect human tumoral cells knock-                          histone deacetylase inhibitors: implications drug-resistant phenotype. <br /> down HDAC1. These results confirm impor-                               Drug Resist. Updates 6:247&#226;&#8364;&#8220;256. <br /> tance characterizing biological function HDAC1,                            21. Henderson, C., M. Mizzau, G. Paroni, R. Maestro, C. Schneider, C. <br />                                                                                          Brancolini. 2003. Role caspases, Bid, p53 apoptotic response <br /> also light potential significance target cancer                       triggered histone deacetylase inhibitors trichostatin-A (TSA) sub- <br /> therapy.                                                                                 eroylanilide hydroxamic acid (SAHA). J. Biol. Chem. 278:12579&#226;&#8364;&#8220;12589. <br />                                                                                      22. Hendzel, M. J., Y. Wei, M. A. Mancini, A. Van Hooser, T. Ranalli, B. R. <br />                          ACKNOWLEDGMENTS                                                 Brinkley, D. P. Bazett-Jones, C. D. Allis. 1997. Mitosis-specific phos- <br />                                                                                          phorylation histone H3 initiates primarily pericentromeric hetero- <br />   We grateful use IFOM-IEO Campus facilities                      chromatin G2 spreads ordered fashion coincident with <br /> thank members laboratories. We especially thank Luca                          mitotic chromosome condensation. Chromosoma 106:348&#226;&#8364;&#8220;360. <br /> Giorgetti, Roberta Carbone (Sr.), Anna Antoni, Mario Faretta,                 23. Hsu, J. Y., Z. W. Sun, X. Li, M. Reuben, K. Tatchell, D. K. Bishop, J. M. <br />                                                                                          Grushcow, C. J. Brame, J. A. Caldwell, D. F. Hunt, R. Lin, M. M. Smith, and <br /> Saverio Minucci experimental suggestions. We thank Manuelo <br />                                                                                          C. D. Allis. 2000. Mitotic phosphorylation histone H3 governed by <br /> Malizia Stefano Leva technical help.                                             Ipl1/aurora kinase Glc7/PP1 phosphatase budding yeast nema- <br />   This work supported grants Associazione Italiana                       todes. Cell 102:279&#226;&#8364;&#8220;291. <br /> per la Ricerca sul Cancro (AIRC) S.C., GEN-AU program                    24. Huang, B. H., M. Laban, C. H. Leung, L. Lee, C. K. Lee, M. Salto-Tellez, <br /> of Austrian Ministry Science C.S., Italian                        G. C. Raju, S. C. Hooi. 2005. Inhibition histone deacetylase 2 in- <br /> Ministry Health.                                                                      creases apoptosis p21Cip1/WAF1 expression, independent histone <br />                                                                                          deacetylase 1. Cell Death Differ. 12:395&#226;&#8364;&#8220;404. <br />                                  REFERENCES                                          25. Hubbert, C., A. Guardiola, R. Shao, Y. Kawaguchi, A. Ito, A. Nixon, M. <br /> 1. Ahringer, J. 2000. NuRD SIN3 histone deacetylase complexes devel-              Yoshida, X. F. Wang, T. P. Yao. 2002. HDAC6 microtubule-asso- <br />    opment. Trends Genet. 16:351&#226;&#8364;&#8220;356.                                                     ciated deacetylase. Nature 417:455&#226;&#8364;&#8220;458. <br /> 2. Alcalay, M., N. Meani, V. Gelmetti, A. Fantozzi, M. Fagioli, A. Orleth, D.        26. Humphrey, G. W., Y. Wang, V. R. Russanova, T. Hirai, J. Qin, Y. Nakatani, <br />    Riganelli, C. Sebastiani, E. Cappelli, C. Casciari, M. T. Sciurpi, A. R.              B. H. Howard. 2001. Stable histone deacetylase complexes distinguished <br />    Mariano, S. P. Minardi, L. Luzi, H. Muller, P. P. Di Fiore, G. Frosina,           presence SANT domain proteins CoREST/kiaa0071 Mta-L1. <br />    P. G. Pelicci. 2003. Acute myeloid leukemia fusion proteins deregulate genes          J. Biol. Chem. 276:6817&#226;&#8364;&#8220;6824. <br />    involved stem cell maintenance DNA repair. J. Clin. Investig. 112:         27. Hutchins, J. R., P. R. Clarke. 2004. Many fingers mitotic trigger: <br />    1751&#226;&#8364;&#8220;1761.                                                                            post-translational regulation Cdc25C phosphatase. Cell Cycle 3:41&#226;&#8364;&#8220;45. <br /> 3. Bartl, S., J. Taplick, G. Lagger, H. Khier, K. Kuchler, C. Seiser. 1997.      28. Ito, A., Y. Kawaguchi, C. H. Lai, J. J. Kovacs, Y. Higashimoto, E. Appella, <br />    Identification mouse histone deacetylase 1 growth factor-inducible            T. P. Yao. 2002. MDM2-HDAC1-mediated deacetylation p53 is <br />    gene. Mol. Cell. Biol. 17:5033&#226;&#8364;&#8220;5043.                                                  required degradation. EMBO J. 21:6236&#226;&#8364;&#8220;6245. <br /> 4. Bolden, J. E., M. J. Peart, R. W. Johnstone. 2006. Anticancer activities      29. Johnstone, R. W. 2002. Histone-deacetylase inhibitors: novel drugs the <br />    histone deacetylase inhibitors. Nat. Rev. Drug Discov. 5:769&#226;&#8364;&#8220;784.                  treatment cancer. Nat. Rev. Drug Discov. 1:287&#226;&#8364;&#8220;299. <br /> 5. Choi, J. H., H. J. Kwon, B. I. Yoon, J. H. Kim, S. U. Han, H. J. Joo, D. Y.   30. Johnstone, R. W., J. D. Licht. 2003. Histone deacetylase inhibitors in <br />    Kim. 2001. Expression profile histone deacetylase 1 gastric cancer              cancer therapy: transcription primary target? Cancer Cell 4:13&#226;&#8364;&#8220;18. <br />    tissues. Jpn. J. Cancer Res. 92:1300&#226;&#8364;&#8220;1304.                                        31. Kartmann, B., D. Roth. 2001. Novel roles mammalian septins: <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> 6. Cobb, J., M. Miyaike, A. Kikuchi, M. A. Handel. 1999. Meiotic events           vesicle trafficking oncogenesis. J. Cell Sci. 114:839&#226;&#8364;&#8220;844. <br />    centromeric heterochromatin: histone H3 phosphorylation, topoisomer-          32. Kawai, H., H. Li, S. Avraham, S. Jiang, H. K. Avraham. 2003. Overex- <br />    ase II alpha localization chromosome condensation. Chromosoma 108:                pression histone deacetylase HDAC1 modulates breast cancer progres- <br />    412&#226;&#8364;&#8220;425.                                                                              sion negative regulation estrogen receptor alpha. Int. J. Cancer 107: <br /> 7. Cress, W. D., E. Seto. 2000. Histone deacetylases, transcriptional con-           353&#226;&#8364;&#8220;358. <br />    trol, cancer. J. Cell Physiol. 184:1&#226;&#8364;&#8220;16.                                      33. Khochbin, S., A. Verdel, C. Lemercier, D. Seigneurin-Berny. 2001. <br /> 8. David, G., G. M. Turner, Y. Yao, A. Protopopov, R. A. DePinho. 2003.              Functional significance histone deacetylase diversity. Curr. Opin. Genet. <br />    mSin3-associated protein, mSds3, essential pericentric heterochroma-           Dev. 11:162&#226;&#8364;&#8220;166. <br />    tin formation chromosome segregation mammalian cells. Genes Dev.           34. Kim, J. H., Y. K. Choi, H. J. Kwon, H. K. Yang, J. H. Choi, D. Y. Kim. <br />    17:2396&#226;&#8364;&#8220;2405.                                                                         2004. Downregulation gelsolin retinoic acid receptor beta expression <br /> 9. Ekwall, K., T. Olsson, B. M. Turner, G. Cranston, R. C. Allshire. 1997.           gastric cancer tissues histone deacetylase 1. J. Gastroenterol. <br />    Transient inhibition histone deacetylation alters structural func-         Hepatol. 19:218&#226;&#8364;&#8220;224. <br />    tional imprint fission yeast centromeres. Cell 91:1021&#226;&#8364;&#8220;1032.                   35. Kramer, O. H., P. Zhu, H. P. Ostendorff, M. Golebiewski, J. Tiefenbach, <br />  VOL. 27, 2007                                                           ROLE FOR HDAC1 IN HUMAN TUMOR CELL PROLIFERATION                                            4795 <br />  <br />       M. A. Peters, B. Brill, B. Groner, I. Bach, T. Heinzel, M. Gottlicher.        47. Richon, V. M., T. W. Sandhoff, R. A. Rifkind, P. A. Marks. 2000. Histone <br />       2003. The histone deacetylase inhibitor valproic acid selectively induces             deacetylase inhibitor selectively induces p21WAF1 expression gene- <br />       proteasomal degradation HDAC2. EMBO J. 22:3411&#226;&#8364;&#8220;3420.                               associated histone acetylation. Proc. Natl. Acad. Sci. USA 97:10014&#226;&#8364;&#8220;10019. <br /> 36.   Lagger, G., A. Doetzlhofer, B. Schuettengruber, E. Haidweger, E. Simboeck,        48. Richon, V. M., Y. Webb, R. Merger, T. Sheppard, B. Jursic, L. Ngo, F. Civoli, <br />       J. Tischler, S. Chiocca, G. Suske, H. Rotheneder, E. Wintersberger, C.            R. Breslow, R. A. Rifkind, P. A. Marks. 1996. Second generation hybrid <br />       Seiser. 2003. The tumor suppressor p53 histone deacetylase 1                  polar compounds potent inducers transformed cell differentiation. <br />       antagonistic regulators cyclin-dependent kinase inhibitor p21/WAF1/            Proc. Natl. Acad. Sci. USA 93:5705&#226;&#8364;&#8220;5708. <br />       CIP1 gene. Mol. Cell. Biol. 23:2669&#226;&#8364;&#8220;2679.                                         49. Sasaki, S., L. A. Lesoon-Wood, A. Dey, T. Kuwata, B. D. Weintraub, G. <br /> 37.   Lagger, G., D. O&#226;&#8364;&#8482;Carroll, M. Rembold, H. Khier, J. Tischler, G. Weitzer, B.           Humphrey, W. M. Yang, E. Seto, P. M. Yen, B. H. Howard, K. Ozato. <br />       Schuettengruber, C. Hauser, R. Brunmeir, T. Jenuwein, C. Seiser. 2002. <br />                                                                                             1999. Ligand-induced recruitment histone deacetylase negative- <br />       Essential function histone deacetylase 1 proliferation control CDK <br />                                                                                             feedback regulation thyrotropin beta gene. EMBO J. 18:5389&#226;&#8364;&#8220;5398. <br />       inhibitor repression. EMBO J. 21:2672&#226;&#8364;&#8220;2681. <br /> 38.   Li, Y., G. D. Kao, B. A. Garcia, J. Shabanowitz, D. F. Hunt, J. Qin, C. P <br /> </body></html>