<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  Preservation of Duplicate Genes by Complementary, Degenerative Mutations <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 The origin of organismal complexity is generally thought <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> be tightly coupled to the evolution of new <br />              gene functions arising subsequent to gene duplication. Under the classical model for the evolution of <br />              duplicate genes, one member of the duplicated pair usually degenerates within a few million years by <br />              accumulating deleterious mutations, while the other duplicate retains the original function. This model <br />              further predicts that on rare occasions, one duplicate may acquire a new adaptive function, resulting in <br />              the preservation of both members of the pair, one with the new function and the other retaining the old. <br />              However, empirical data suggest that a much greater proportion of gene duplicates is preserved than <br />              predicted by the classical model. Here we present a new conceptual framework for understanding the <br />              evolution of duplicate genes that may help explain this conundrum. Focusing on the regulatory complexity <br />              of eukaryotic genes, we show how complementary degenerative mutations in different regulatory elements <br />              of duplicated genes can facilitate the preservation of both duplicates, thereby increasing long-term opportu- <br />              nities for the evolution of new gene functions. The duplication-degeneration-complementation (DDC) <br />              model predicts that (1) degenerative mutations in regulatory elements can increase rather than reduce <br />              the probability of duplicate gene preservation and (2) the usual mechanism of duplicate gene preservation <br />              is the partitioning of ancestral functions rather than the evolution of new functions. We present several <br />              examples (including analysis of a new engrailed gene in zebrafish) that appear to be consistent with the <br />              DDC model, and we suggest several analytical and experimental approaches for determining whether the <br />              complementary loss of gene subfunctions or the acquisition of novel functions are likely to be the primary <br />              mechanisms for the preservation of gene duplicates. <br />                  For a newly duplicated paralog, <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> depends on the outcome of the race between entropic <br />                  decay and chance acquisition of an advantageous regulatory <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span>. <br />                                                                                          Sidow (1996, p. 717) <br />                  On one hand, it may fix an advantageous allele giving it a slightly different, and selectable, function <br />                  <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> its original copy. This initial fixation provides substantial protection against future fixation of <br />                  null mutations, allowing additional mutations to accumulate that refine functional differentiation. <br />                  Alternatively, a duplicate locus can instead first fix a <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>null allele</span>, becoming a pseudogene. <br />                                                                                                  Walsh (1995, p. 426) <br />                  Duplicated genes persist only if mutations create new and essential protein functions, an event that <br />                  is predicted to occur rarely. <br />                                                                              Nadeau and Sankoff (1997, p. 1259) <br />                  Thus overall, with complex metazoans, the major mechanism for retention of ancient gene duplicates <br />                  would appear to have been the acquisition of novel expression sites for developmental genes, <br />                  with its accompanying opportunity for new gene roles underlying the progressive extension of <br />                  development itself. <br />                                                                                        Cooke et al. (1997, p. 362) <br />  <br />  <br />  <br /> T   HE genomes of most organisms contain multiple <br />     copies of genes that are closely related in structure <br /> and function. Such gene families can arise from tandem <br />                                                                          events such as those presumed to have preceded the <br />                                                                          origin of vertebrates (Ohno 1970; Morizot et al. 1991; <br />                                                                          Lundin 1993; Holland et al. 1994; Amores et al. 1998; <br /> duplications, as in the case of the HOX, hemoglobin,                     Pe&#194;&#180;busque et al. 1998), brewer&#226;&#8364;&#8482;s yeast (Wolfe and <br /> and keratin clusters in animals, or from polyploidization                Shields 1997; Seoighe and Wolfe 1998), and many <br />                                                                          plant species (Lewis 1979). The mechanism that pre- <br />                                                                          serves a large proportion of duplicate genes for long <br />   Corresponding author: Allan Force, Department of Biology, University   time periods, however, is unclear. The classical model <br /> of Oregon, Eugene, OR 97403. E-mail: force@oregon.uoregon.edu            predicts that duplicate genes initially have fully overlap- <br /> Genetics 151: 1531&#226;&#8364;&#8220;1545 (April 1999) <br />  1532                                                      A. Force et al. <br />  <br /> ping, redundant functions, such that one copy may                   For example, in tetraploid fish lineages, 30&#226;&#8364;&#8220;75% of the <br /> shield the second copy from natural selection, if gene              duplicate protein-coding genes have avoided nonfunc- <br /> dosage is not critical. Because deleterious mutations               tionalization for time spans on the order of 50 to 100 <br /> occur much more frequently than beneficial mutations                million yr (Allendorf et al. 1975; Ferris and Whitt <br /> (Lynch and Walsh 1998), the classical model predicts                1979); in Xenopus laevis, about half of all duplicate genes <br /> that the most common fate for the duplicate pair should             have been preserved for 30 million yr (Bisbee et al. 1977; <br /> be the fixation of a null allele that prevents normal               Graf and Kobel 1991; Hughes and Hughes 1993); and <br /> transcription, translation, and/or protein function, i.e.,          for the allopolyploidization event in maize, an annual <br /> the formation of a pseudogene at one of the duplicate               plant, 72% have avoided nonfunctionalization for 11 <br /> loci (Haldane 1933; Nei and Roychoudhury 1973;                      million yr (Whitkus et al. 1992; Ahn and Tanksley <br /> Bailey et al. 1978; Takahata and Maruyama 1979;                     1993; White and Doebley 1998). The fact that most <br /> Li 1980; Watterson 1983). Under this model, first                   loci observed in these lineages appear to have a non- <br /> elucidated by Ohno (1970), the only mechanism for                   functional member in some related tetraploid species <br /> the permanent preservation of duplicate genes is the                argues against the idea that both duplicate genes are <br /> fixation of rare beneficial mutations endowing one of               retained due to constraints imposed by gene dosage <br /> the copies with a novel function, while the second copy             requirements, at least for the enzyme loci investigated. <br /> maintains the original function. The introductory quo-              Although the highest levels of duplicate gene retention <br /> tations illustrate the extent to which this model is cur-           in some fish and plant lineages may be due to the incom- <br /> rently the central paradigm in the theory of duplicate              plete transition to diploid inheritance, similar estimates <br /> gene evolution.                                                     of duplicate gene preservation have emerged for more <br />   Here we discuss difficulties in the ability of the classical      ancient polyploidization events for which disomic inher- <br /> model to explain the preservation of gene duplicates                itance has clearly been reestablished, such as duplica- <br /> in evolution and then propose a new model that can                  tions that preceded the origin of tetrapods (33%; <br /> explain duplicate gene preservation by the fixation of              Nadeau and Sankoff 1997). Second, in X. laevis, which <br /> degenerative mutations rather than by the fixation of               became tetraploid z30 mya, nucleotide substitution pat- <br /> new beneficial mutations. Next, we present several ex-              terns are consistent with the action of purifying selection <br /> amples, including original data from the zebrafish en-              on both copies of the duplicated genes (Hughes and <br /> grailed genes, consistent with the new model. Finally, we           Hughes 1993). Third, for loci that have avoided non- <br /> suggest a series of experimental approaches for testing             functionalization in both duplicate copies, there seems <br /> the new model.                                                      to be a relative paucity of null alleles segregating in <br />   Problems with the classical model for the preservation            extant populations (Ferris and Whitt 1977). Such ob- <br /> of gene duplicates: Under the simplest model for the                servations are unexpected for loci involved in an ongo- <br /> fate of duplicate genes (the double-recessive model),               ing degenerative process, and suggest the possibility that <br /> the rate at which nonfunctional genes (genes that do                the duplicate loci are stabilized in populations. <br /> not make a functional protein product) become fixed                   Several attempts have been made to explain the high <br /> in populations is largely determined by random genetic              rate of duplicate gene preservation found by empirical <br /> drift and the null mutation rate (u), provided the prod-            observation. First, surviving duplicate loci in these taxa <br /> uct of the effective population size and u is ,0.01. Under          may have been preserved because new gene functions <br /> these conditions, the frequency of individuals homozy-              evolve at a much higher rate than predicted. We are <br /> gous null at both duplicate loci is negligible, and null            not aware, however, of any convincing evidence that the <br /> mutations behave essentially as neutral alleles. The                majority of duplicate copies have acquired new func- <br /> probability that one copy will become nonfunctional is              tions that did not already exist in the ancestral genes <br /> then z1 2 e22ut, where t is the number of generations               (Ferris and Whitt 1979). A second possible explana- <br /> since the two loci have been functionally diploid with              tion is that long-term effective population sizes may have <br /> respect to meiosis (Nei and Roychoudhury 1973;                      been larger than expected, in fact large enough so that <br /> Takahata and Maruyama 1979; Li 1980; Watterson                      selection against double homozygotes prevents the fixa- <br /> 1983). This result suggests that most gene duplicates               tion of null alleles at either locus (Takahata and Maru- <br /> should become nonfunctional with high probability in                yama 1979; Li 1980; Walsh 1995). This appears not to <br /> a relatively short period of time. For example, if u is             account for the case of X. laevis (Hughes and Hughes <br /> 1026 per generation, then the mean time to nonfunc-                 1993). The population size requirements for the preser- <br /> tionalization is on the order of a few million generations          vation of gene duplicates by selection against double <br /> or less.                                                            nulls over hundreds of millions of years may be prohibi- <br />   Three general observations involving species derived              tively extreme. A third possible explanation for the dis- <br /> from polyploidization events appear to contradict the               crepancy between theory and observation is that the <br /> rapid demise of gene duplicates predicted by the classi-            rate of gene loss has been slowed by some form of <br /> cal model. First, in numerous cases, the fraction of genes          natural selection against heterozygous carriers of null <br /> preserved is higher than predicted by the classic model.            alleles (Bailey et al. 1978; Takahata and Maruyama <br />                                                 Preservation of Duplicate Genes                                         1533 <br />  <br /> 1979; Li 1980; Hughes and Hughes 1993; Clark 1994;               gene in a specific tissue, cell lineage, or developmental <br /> Nowak et al. 1997).                                              stage, or individual functional domains within the poly- <br />    Gene structure and duplicate gene preservation: An            peptide coding portion of the gene. <br /> alternative reason for the failure of the classical model          The model presented below outlines how degenera- <br /> to explain the fates of most duplicate loci may be an            tive mutations in regulatory subfunctions can facilitate <br /> overly simplistic view of gene structure. Although a gen-        the preservation of duplicate genes, in the absence of <br /> eral assumption of the classical model is that the proper-       any positive selection for beneficial mutations, by parti- <br /> ties of a gene may be adequately subsumed under a                tioning the repertoire of gene expression patterns of <br /> single function, genes often have several functions, each        ancestral alleles. This model is quite distinct from the <br /> of which may be controlled by different DNA regulatory           classical model, under which degenerative mutations <br /> elements (see the following reviews for a number of              can only lead to gene loss and beneficial mutations are <br /> examples: Piatigorsky and Wistow 1991; Hughes                    the only route to gene preservation. <br /> 1994; Kirchhamer et al. 1996; Arnone and Davidson <br /> 1997). A case in point is the cut locus in Drosophila <br /> melanogaster (Jack 1985; Liu et al. 1991; Jack and                    GENE PRESERVATION BY COMPLEMENTARY <br /> DeLotto 1995). Genetic and molecular analyses dem-                          DEGENERATIVE MUTATIONS <br />                                                                             (SUBFUNCTIONALIZATION) <br /> onstrate that a 120-kb region of DNA upstream of the <br /> cut promoter drives tissue-specific expression, and that            Following a polyploidization event, genomic redun- <br /> many spontaneous recessive mutant alleles result from            dancies exist at several levels: duplicate chromosomes, <br /> insertions of transposable elements into this region. The        duplicate genes, and duplicate regulatory regions driv- <br /> regulatory mutation alleles fall into five complementa-          ing gene expression. Each level of redundancy is subject <br /> tion classes, with varying effects on tissue-specific expres-    to processes of mutation and random genetic drift, <br /> sion (in Malpighian tubules, spiracles, central nervous          which can lead to loss of function by chromosome loss, <br /> system, specific portions of wing and leg imaginal discs,        gene inactivation, or loss of individual regulatory ele- <br /> and embryonic and adult external sensory organs), as             ments. If duplicate chromosomes lose different genes, <br /> well as on morphology and viability. Similar comple-             then for the organism to remain viable, the two chromo- <br /> mentation groups involving regulatory-region muta-               somes must complement each other by jointly retaining <br /> tions are known for other developmental genes in D.              functional copies of all genes present on the original <br /> melanogaster, including cubitus interruptus (Slusarski et        ancestral chromosome. Likewise, if duplicate genes lose <br /> al. 1995) and Ultrabithorax (Bender et al. 1983).                different regulatory subfunctions, then they must com- <br />    The widespread existence of complementation classes           plement each other by jointly retaining the full set of <br /> within eukaryotic gene loci indicates that gene expres-          subfunctions present in the original ancestral gene. We <br /> sion patterns are typically controlled by multiple (and          refer to the complementary loss of duplicate genetic <br /> often modular and independent) regulatory regions as-            elements by degenerative mutation as the duplication- <br /> sociated with distinct protein-coding domains (Arnone            degeneration-complementation (DDC) process. The <br /> and Davidson 1997). With the explicit assumption that            unique feature that distinguishes the DDC process from <br /> these principles involving complementation between al-           the classical model is that degenerative mutations facili- <br /> leles at the same locus can be extended to complementa-          tate rather than hinder the preservation of duplicate <br /> tion between two duplicate loci, we suggest that the             functional genes. In the following discussion, we focus <br /> regulatory complexity inherent in many gene classes              on duplications of entire chromosomes or genomes <br /> is an essential, but previously missing, component of            rather than tandem gene duplications because we wish <br /> models for the evolutionary fate of duplicate genes.             to exclude for now complications caused by uncertainty <br /> Further justification for this argument derives from sub-        about the extent of the original duplication and local <br /> stantial evidence showing spatial and temporal parti-            homogenization events caused by unequal crossing over <br /> tioning of expression patterns for gene duplicates in a          or gene conversions (Zhou and Li 1996). <br /> wide variety of species (Ferris and Whitt 1979;                     Under the general DDC model, the process of dupli- <br /> Hughes and Hughes 1993; Ekker et al. 1995; Lee et al.            cate gene evolution occurs in two phases (Figure 1). <br /> 1996; Raff 1996; Gerhart and Kirschner 1997). To                 During phase I, genes may experience one of three <br /> formally incorporate the issue of expression <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span>             alternative fates, the first two of which correspond to <br /> complexity into models of gene duplication, we focus             outcomes under the classical model. First, one copy <br /> here on subfunctions that affect different gene expres-          may incur a null mutation in the coding region, which <br /> sion domains during development. Here we adopt an                subsequently drifts to fixation, leading to gene loss (non- <br /> operational <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of a subfunction as a specific            functionalization). Nonfunctionalization can also occur <br /> subset of a gene&#226;&#8364;&#8482;s function that, when mutated, estab-           if all of the regulatory regions of one duplicate are <br /> lishes a distinct complementation group, as in the cut           destroyed. Second, one copy may acquire a mutation <br /> example above (Liu et al. 1991; Jack and DeLotto                 conferring a new function, which becomes fixed <br /> 1995). A subfunction might involve the expression of a           through positive Darwinian selection (neofunctionaliza- <br />  1534                                                      A. Force et al. <br />  <br />                                                                                    Figure 1.&#226;&#8364;&#8221;Three potential fates of duplicate <br />                                                                                    gene pairs with multiple regulatory regions. <br />                                                                                    The small boxes denote regulatory elements <br />                                                                                    with unique functions, and the large boxes <br />                                                                                    denote transcribed regions. Solid boxes de- <br />                                                                                    note intact regions of a gene, while open boxes <br />                                                                                    denote null mutations, and triangles denote <br />                                                                                    the evolution of a new function. Because the <br />                                                                                    model focuses on mutations fixed in popula- <br />                                                                                    tions, the diagram shows the state of a single <br />                                                                                    gamete. In the first two steps, one of the copies <br />                                                                                    acquires null mutations in each of two regula- <br />                                                                                    tory regions. On the left, the next fixed muta- <br />                                                                                    tion results in the absence of a functional pro- <br />                                                                                    tein product from the upper copy. Because <br />                                                                                    this gene is now a nonfunctional pseudogene, <br />                                                                                    the remaining regulatory regions associated <br />                                                                                    with this copy eventually accumulate degenera- <br />                                                                                    tive mutations. On the right, the lower copy <br />                                                                                    acquires a null mutation in a regulatory region <br /> that is intact in the upper copy. Because both copies are now essential for complete gene expression, this third mutational event <br /> permanently preserves both members of the gene pair from future nonfunctionalization. The fourth regulatory region, however, <br /> may still eventually acquire a null mutation in one copy or the other. In the center, a regulatory region acquires a new function <br /> that preserves that copy. If the beneficial mutation occurs at the expense of an otherwise essential function, then the duplicate <br /> copy is preserved because it retains the original function. <br />  <br />  <br /> tion). It is now thought that such mutations may often               of the subfunction from either copy may be opposed <br /> involve changes in regulatory regions (Grenier et al.                by purifying selection. <br /> 1997; Shubin et al. 1997; Palopoli and Patel 1998).                     Mutations that cause subfunctions to degrade may <br /> Assuming this new function results in the loss of an                 occur by several mechanisms, including nucleotide sub- <br /> essential ancestral function, neofunctionalization in-               stitutions, deletions, inversions, insertions of transpos- <br /> sures the preservation of the nonmutated copy. [In prin-             able elements, slippage/replication errors, and unequal <br /> ciple, neofunctionalization can also occur if one or both            crossing over between repeated transcription-factor <br /> copies acquire a new regulatory region without altering              binding sites. Transposable elements may generate <br /> existing subfunctions, as pointed out by Sidow (1996)].              many subfunctional alleles. For example, P, copia, and <br /> Third, each duplicate may experience loss or reduction               gypsy elements are known to be mutagenic when they <br /> of expression for different subfunctions by degenerative             insert into 59 regions of Drosophila genes (Kidwell <br /> mutations. In such a case, the combined action of both               and Lisch 1997). Species with a recent history of poly- <br /> gene copies is necessary to fulfill the requirements of the          ploidization, for example, maize, appear to have such <br /> ancestral locus (subfunctionalization). If this happens,             insertions commonly in the 59 and 39 regions of genes, <br /> then complementation of subfunctions between dupli-                  whereas in species lacking a recent polyploidization <br /> cate genes will preserve both partially degenerated cop-             event, such insertions are infrequent (White et al. 1994; <br /> ies. In phase II of the DDC model, duplicate genes                   Wessler et al. 1995). Such transposable element inser- <br /> preserved either by neofunctionalization or subfunc-                 tions, presumably in regulatory DNA, may be tolerated <br /> tionalization undergo random resolution of persisting                in recently evolved polyploid species because of the <br /> redundant subfunctions, as the accumulation of degen-                redundancy of their regulatory elements. <br /> erative mutations eliminates each subfunction in one                    The probability of subfunctionalization: The argu- <br /> or the other copy.                                                   ments presented above suggest that the DDC process <br />    Subfunctionalization can occur by two different                   could make both gene duplicates essential, but can it <br /> routes: qualitative or quantitative. Under qualitative sub-          account for the high levels of duplicate gene preserva- <br /> functionalization, which we model below and illustrate               tion observed in polyploid lineages? Here we consider <br /> in Figure 1, one duplicate copy goes to fixation for a               a simple model that suggests that, with reasonable pa- <br /> complete loss-of-subfunction mutation, and the second                rameter values, the DDC process can account for a sig- <br /> locus subsequently acquires a null mutation for a differ-            nificant fraction of preserved duplicate genes. <br /> ent subfunction. In contrast, quantitative subfunctional-               Consider the situation in which both members of a <br /> ization results from the fixation of reduction-of-expres-            recently duplicated gene have z independently mutable <br /> sion mutations in both duplicates. In this case, once                subfunctions, all of which are essential, at least in single <br /> the total regulatory efficiency of a subfunction in both             copy, and all of which mutate at identical rates (ur) to <br /> copies has been reduced to a threshold level determined              alleles lacking the relevant subfunction. Letting uc be <br /> by organismal requirements, any further degradation                  the rate at which null mutations arise in the coding <br />                                                        Preservation of Duplicate Genes                                              1535 <br />  <br /> region, the null mutation rate for the locus is then uc 1 <br /> zur per gene copy. We assume that conditions are such <br /> that one functional allele (of four possible allele copies) <br /> of a given duplicated gene pair is sufficient for wild- <br /> type function (the double recessive model), and that <br /> beneficial mutations are rare relative to degenerative <br /> mutations. Provided the product of population size and <br /> genic mutation rate is ,0.01 (Li 1980), the frequency <br /> of double null homozygotes will be sufficiently low such <br /> that all allele frequencies will evolve in an effectively <br /> neutral manner. The rate of fixation of a mutation in <br /> a population will then be approximately equal to the <br /> rate of mutation at the level of the gene (Kimura 1983). <br />    Now imagine that one of the duplicate gene copies <br /> experiences a fixation event. Assuming there is more <br /> than one subfunction, the probability that the gene <br /> survives this event (and does not become a pseudogene) <br /> is the total regulatory-region mutation rate divided by <br /> the total mutation rate for the two copies <br />                                                                            Figure 2.&#226;&#8364;&#8221;Combinations of relative null mutation rates to <br />                                              zur                        regulatory and coding regions (ur/uc) and number of subfunc- <br /> Prob (survival of first fixation event) 5         .            (1)      tions (z) that yield various probabilities (PS) of duplicate gene <br />                                           uc 1 zur                      preservation by the DDC process. The probablility of duplicate <br />                                                                         gene preservation increases with the number of regulatory <br />   Following the elimination of one of the z subfunctions                elements. <br /> from the first gene copy, the second copy must maintain <br /> this subfunction, because complete loss of an essential <br /> subfunction from both duplicates would be lethal. Thus,                 tionalization, PS, is obtained by summing this quantity <br /> the permissible mutation rate for the second copy be-                   over i 5 2 to z, <br /> comes (z 2 1)ur. Additional null mutations can occur                                                     z <br /> in the remaining (z 2 1) regulatory subfunctions or in <br /> the coding region in the partially degraded first copy. <br />                                                                                                  PS 5    o PS,i,                     (4) <br />                                                                                                         5 <br />                                                                                                         i 2 <br />  <br /> Therefore, the total rate (summed over both copies) <br />                                                                         and the probability of nonfunctionalization is equal to <br /> for the second mutational event is [uc 1 2(z 2 1)ur]. <br />                                                                         1 2 PS. From this expression, we see that the probability <br /> The probability of subfunctionalization upon this sec- <br />                                                                         of duplicate-gene preservation increases with the num- <br /> ond event, PS,2, is equal to the probability that the coding <br />                                                                         ber of regulatory regions and with the mutation rate per <br /> regions have survived the first hit multiplied by the <br />                                                                         regulatory region (Figure 2). More regulatory regions <br /> probability that the second mutation occurs in a comple- <br />                                                                         provide more targets for subfunctionalization that can <br /> mentary subfunction in the second copy, <br />                                                                         be hit without penalty, and an increasing mutation rate <br />  <br />                       1u 1zu zu 2 1u 1(z 2(z 2 1)u 2 <br />                                          2 1)u                          per subfunction reduces the relative probability of fixa- <br />            PS,2 5               r                 r <br />                                                      .         (2)      tion of a null mutation in the coding region before <br />                         c             r       c       r <br />                                                                         complementation. <br /> Following this logic, it can be seen that (z 2 1) distinct                 The DDC process leads to subfunctionalization with <br /> series of mutational events can lead to duplicate-gene                  high probability given reasonable parameter values. For <br /> preservation by subfunctionalization&#226;&#8364;&#8221;the first two null                 example, if there are five subfunctions and the mutation <br /> mutations in regulatory regions may occur on different                  rate per subfunction is 10% of the coding region null <br /> gene copies, two may initially occur on the same copy                   rate, then the probability of subfunctionalization is 0.1, <br /> followed by a third on the second copy, three may ini-                  and if the mutation rate per subfunction is 30% that of <br /> tially occur on the same copy followed by a fourth on                   the null rate, then the probablitity of subfunctionaliza- <br /> the second copy, and so on. The probability of each of                  tion is 30% (Figure 2). Generally, if the total rate of <br /> these additional pathways to subfunctionalization, i.e.,                subfunctional mutations (zur) exceeds the null rate in <br /> (i 2 1) consecutive regulatory-region null mutations on                 the coding region by more than approximately fourfold, <br /> one copy followed by one on the other, is given by the                  then the probability of gene preservation by subfunc- <br /> generalization of Equation 2,                                           tionalization exceeds 50%. The complexity and size of <br />                                     i22                                 regulatory regions of eukaryotic genes (Kirchhamer et <br />        PS,i 5   1u 1zu zu 2 p 1u 1(z 2(z <br />                   c <br />                         r <br />  <br />                             r <br />                                      2 j 2 1)u <br />                                     j50  2 j 2 1)u 2 <br />                                           c <br />                                                      .r <br />  <br />                                                           r <br />                                                                (3)      al. 1996; Arnone and Davidson 1997) suggests that <br />                                                                         these conditions may be met frequently. <br /> The total probability of gene preservation by subfunc-                     Time scales for subfunctionalization and resolution: <br />  1536                                                     A. Force et al. <br />  <br /> Using the model presented above, the mean time to <br /> gene preservation conditional on its actual occurrence <br /> can be obtained by treating the times to mutational <br /> events as geometrically distributed variables. The rate <br /> of occurrence of an initial regulatory-region null muta- <br /> tion is 2zur, because each of the two copies contains z <br /> mutational targets. As noted above, subsequent to this <br /> initial event, zero to (z 2 2) additional degenerative <br /> mutations may be incurred by the first-hit copy before <br /> the first mutation on the opposite copy. The mean time <br /> to subfunctionalization conditional on the occurrence <br /> of (i 2 1) consecutive regulatory-region null mutations <br /> on one copy followed by one on the other is then <br />                                            i21 <br />                 tS,i 5 <br />                          1 1 <br />                             1 <br />                          ur 2z <br />                                1 <br />                                            5 <br />                                             o <br />                                                    1 <br />                                            j 1 z 2 j <br />                                                     2.   (5a) <br />  <br /> The mean time to subfunctionalization is then <br />  <br />                                  z <br />                                       tS,i PS,i <br />                          tS 5    o       PS <br />                                                .         (5b) <br />                                 5 <br />                                 i 2 <br />  <br />    As in the classical model, these expressions indicate <br /> that the fates of duplicate genes are generally deter- <br /> mined in a relatively short period (on an evolutionary <br /> time scale; Figure 3A). For example, if ur 5 1027/yr, <br /> tS is on the order of 4 million yr or less provided the <br /> number of regulatory regions is greater than five, and <br /> even with z , 5 it does not exceed 12.5 million yr. Thus, <br /> under the DDC model, most duplicate genes that are <br /> destined to be preserved by subfunctionalization are                  Figure 3.&#226;&#8364;&#8221;(A) The mean time to subfunctionalization as <br /> expected to become so within a few million years. With             a function of the number of subfunctions (z), for the situation <br /> a regulatory-region mutation rate x times that in the              in which ur 5 1027/yr. (B) The fates of gene pairs are deter- <br /> figure, the mean time to subfunctionalization would be             mined in a short time, on an evolutionary scale, and the ratio <br /> divided by x.                                                      of the mutation rate in regulatory and coding regions is a <br />                                                                    weak determinant of the expected degree of resolution at the <br />    Unless there are only two initial regulatory regions,           time of subfunctionalization. <br /> some regulatory regions (as many as z 2 2) will likely <br /> remain to be resolved over evolutionary time after the <br /> initial subfunctionalization event. The fraction of regu-             The molecular nature of subfunctions and the preser- <br /> latory regions that is expected to be resolved at the time         vation of genetic redundancy: The preceding theory <br /> of gene preservation by subfunctionalization is                    assumes that individual regulatory subfunctions are in- <br />                                        z <br />                                             iPS,i                  dependently mutable, with single mutations being suf- <br />                      Pr(0) 5           o    zPS <br />                                                  .        (6)      ficient to eliminate a subfunction. Under this simple <br />                                       5 <br />                                       i 2 <br />                                                                    scenario, the various subfunctions within duplicate <br />    This fraction depends only weakly on the ratio of               genes preserved by the DDC process are expected to <br /> coding-region to regulatory-region mutation rates, and             be resolved randomly, with each copy retaining about <br /> is ,0.5 if the number of regulatory regions exceeds five           half of its subfunctions within the limits of binomial <br /> (Figure 3B). Thus, we anticipate that after the preserva-          sampling. However, while we define subfunctions by <br /> tion of duplicate genes by the DDC process, a substantial          their mutational properties such that they are members <br /> fraction of regulatory subfunctions will typically remain          of distinct complementation classes, this <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> does <br /> to be resolved in phase II. Assuming that the occurrence           not describe how such subfunctions are arranged on <br /> of mutations that destroy regulatory regions is a Poisson          the DNA molecule. Regulatory regions for different sub- <br /> process, for any site that is unresolved at the time of            functions are often partially overlapping or embedded, <br /> gene preservation, the probability that it is still unre-          leading to the situation where the number of expression <br /> solved after t further time units is simply P0(t) 5 e22tur.        domains exceeds the number of complementation <br /> The number of unresolved sites at time t then follows              groups (Jack and DeLotto 1995; Kirchhamer et al. <br /> a binomial distribution with parameter P0(t).                      1996). Some of the central issues are illustrated in Figure <br />                                                    Preservation of Duplicate Genes                                         1537 <br />  <br />                                                                     (Pickett and Meeks-Wagner 1995). In Figure 4C, for <br />                                                                     example, regulatory regions 1 and 3 are partially over- <br />                                                                     lapping and regulatory region 2 is embedded in the <br />                                                                     region of overlap. If in this situation, one gene duplicate <br />                                                                     suffers a degenerative mutation in the nonoverlapping <br />                                                                     portion of regulatory region 1, and the other gene dupli- <br />                                                                     cate experiences a degenerative mutation destroying <br />                                                                     the nonoverlapping portion of regulatory region 3, then <br />                                                                     regulatory region 2 will be sheltered from mutation <br />                                                                     events that destroy two or more adjacent binding <br />                                                                     sites because such a mutation in that region would <br />                                                                     completely eliminate subfunction 1 or 3 (Figure 4F). <br />                                                                     Resolution of subfunction 2 may then proceed only by <br />                                                                     small-scale mutational events that partially degrade the <br />                                                                     quantitative functionality of individual transcription-fac- <br />                                                                     tor binding sites. If on the other hand, the binding sites <br />                                                                     for regulatory region 2 are completely shared with both <br />                                                                     regulatory regions 1 and 3 (Figure 4, E and G), then <br />                                                                     mutational events that inactivate regions 1 and 3 on <br />                                                                     opposite copies will indefinitely preserve the embedded <br />                                                                     regulatory region 2 in both copies. Thus, the resolution <br />                                                                     of overlapping and/or embedded subfunctions pro- <br />                                                                     vides a potential explanation for the maintenance of <br />                                                                     some of the redundancy observed between duplicate <br />                                                                     genes without invoking positive selection for duplicate <br />   Figure 4.&#226;&#8364;&#8221;Overlapping and embedded regulatory ele-                gene expression (Nowak et al. 1997). <br /> ments. All transcription-factor binding sites shown are as-            The topology of regulatory regions may also help <br /> sumed to be essential for each expression domain. (A) Inde-         explain unidirectional and bidirectional divergence of <br /> pendent regulatory regions with independent transcription-          gene duplicates observed by Ferris and Whitt (1979). <br /> factor binding sites. (B) Overlapping regulatory regions with <br /> independent sites. (C) Overlapping and embedded regulatory          For some enzyme-encoding loci, gene duplicates show <br /> regions with independent sites. (D) Overlapping regulatory          unidirectional divergence&#226;&#8364;&#8221;the enzyme products of one <br /> regions with shared sites. (E) Embedded regulatory regions          locus predominate in all tissues in which the two loci <br /> with shared sites. (F) Resolution of overlapping regulatory         are expressed. This situation would be most common <br /> regions with independent sites (derived from C) leading to          at gene loci with overlapping and/or embedded ar- <br /> quantitative resolution of regulatory region 2 after 1 and 3 are <br /> destroyed on paralogous copies. (G) Resolution of embedded          rangements of regulatory regions, because the elimina- <br /> regulatory regions with shared sites leading to true redun-         tion of the embedded element from one gene copy <br /> dancy for regulatory region 2 after 1 and 3 are destroyed on        (element 2 in Figure 4E, for example) would prevent the <br /> paralogous copies.                                                  fixation of alleles in which the overlapping regulatory <br />                                                                     elements (elements 1 and 3 in Figure 4E) are destroyed <br />                                                                     on the other gene duplicate. Under bidirectional diver- <br /> 4. The situation modeled above is equivalent to a setting           gence, either locus may be more highly expressed in <br /> in which the spatial arrangement of transcription-factor            any given tissue. This situation should be most frequent <br /> binding sites allows the independent resolution of the              for gene loci with independently arranged regulatory <br /> subfunctions (z 5 3 in Figure 4A). Within overlapping               regions. <br /> (Figure 4, B and D) or embedded (Figure 4, C and E)                    DDC and dosage effects: In some situations, gene <br /> regulatory regions, the transcription-factor binding sites          dosage requirements might increase the probability that <br /> may be either interdigitated and acting independently               both gene duplicates are preserved. The theoretical <br /> (Figure 4, B and C) or shared, with the same DNA                    model developed above assumes that for each subfunc- <br /> binding site(s) being required for more than one ex-                tion, activity of only one of the four alleles of the two <br /> pression domain (Figure 4, D and E). In these cases,                gene duplicates is sufficient for survival. It is possible, <br /> some mutational events can knock out more than one                  however, that after gene duplication some subfunctions <br /> expression domain at a time, effectively reducing the               must remain intact in more than one of the four alleles <br /> number of subfunctions.                                             to ensure optimal fitness. For instance, consider a gene <br />    Complexities involving the physical arrangement of               with three separate subfunctions. After duplication, the <br /> regulatory regions on the DNA may help explain, with-               first subfunction may be sufficient for survival if intact <br /> out invoking positive selection, how the same expression            in a single allele, the second subfunction may be suffi- <br /> domains may be preserved by both gene duplicates                    cient in two alleles, but the third subfunction might be <br />  1538                                                   A. Force et al. <br />  <br /> required in three of the four alleles. In such a case, the <br /> first and second subfunctions could be resolved to either <br /> duplicate gene by the principles of DDC. The third <br /> subfunction, however, would have to be maintained by <br /> both gene duplicates to have at least three active alleles. <br /> In such cases, dosage requirements would provide the <br /> initial gene preservation mechanism, but complemen- <br /> tary loss of other subfunctions or acquisition of a new <br /> function could reinforce the initial preservation event. <br /> Note that this type of dosage effect provides an alterna- <br /> tive mechanism to shared embedded elements (Figure                  Figure 5.&#226;&#8364;&#8221;Phylogeny of engrailed family genes. Sequences <br />                                                                  were aligned by eye with the assistance of CLUSTAL X, and <br /> 4) for retaining a specific expression domain by both            only the unambiguous portions of the alignment were em- <br /> gene duplicates.                                                 ployed in the final analysis. Phylogenetic tree based on the <br />    In some cases, gene dosage requirements might cause           neighbor-joining method (Saitou and Nei 1987) with multi- <br /> the partitioning of subfunctions to be favored by positive       ple substitution correction. Numbers are bootstrap values <br /> selection. For example, consider a situation in which            based on 1000 runs. Lamprey enga was cloned by the polymer- <br />                                                                  ase chain reaction using two primers [forward, 59ATI AA(A <br /> activity of all four alleles of a duplicated gene pair in a      G) ATI TGG TT(T C) CA(A G) AA(T C) AA39; and reverse, <br /> certain tissue or time is deleterious. In such a case, the       59TG(G A) TT(G A) TAI A(G A)I CC(T C) TGI GCC AT39] <br /> fixation of a nonfunctional or subfunctional allele might        designed to amplify the conserved Engrailed amino acid se- <br /> be accelerated by positive selection. Note that a case           quences IKIWFQNK and MAQGLYNH, respectively. The <br /> like this differs from the formal model proposed above,          primers were used to screen a lamprey cosmid library con- <br />                                                                  structed from a single ammocete larva according to the Stra- <br /> which assumes that drift and purifying selection is usu-         tagene SuperCos cloning kit. The same primers were used to <br /> ally sufficient for the fixation of subfunctional alleles.       obtain additional sequence from zebrafish eng1b (Amores et <br /> In these cases, mutations of subfunctions that would be          al. 1998). Species abbreviations: Hsa, Homo sapiens; Mmu, Mus <br /> deleterious in the single-copy genes before duplication          musculus; Gga, Gallus gallus; Dre, Danio rerio; Pma, Petromyzon <br /> would become beneficial after duplication. Because this          marinus; Bfl, Branchiostoma floridae; Her, Heliocidaris erythro- <br />                                                                  gramma. GenBank accession nos.: Hsa EN2, J03066; Mmu <br /> might increase the rate of fixation of subfunctional al-         En2, L12705; Gga En2, L12697; Dre eng2, X68446; Dre eng3, <br /> leles while simultaneously increasing the rate of fixation       X68447; Hsa EN1, L12699; Mmu En1, Y00201, M11987; Gga <br /> of nonfunctional alleles, the overall effect on the proba-       En1, L12695; Dre eng1b, AF071237; Dre eng1, X68445; Pma <br /> bility of duplicate gene preservation is not clear. Future       enga AF129401; Bfl AmphiEn, U82487; Her HeEn, U58775. <br /> experimental and modeling work may help to define <br /> these more complex interactions between gene dosage, <br /> population size, the mutation rates to subfunctional,            1992). Zebrafish, however, has been reported to have <br /> coding null, and neofunctional alleles, and the roles            three engrailed genes (eng1, eng2, and eng3; Ekker et <br /> of purifying and positive selection in duplicate gene            al. 1992), and here we report additional sequence and <br /> preservation. It is hoped that the near-neutral DDC              expression data of a fourth eng gene that we recently <br /> model provided here can act as a null hypothesis for             cloned, eng1b (Amores et al. 1998). Phylogenetic analysis <br /> testing these and other more complex possibilities.              rooted with engrailed genes of amphioxus (Holland et <br />    Possible examples of the DDC process: Here we pre-            al. 1997) and a new lamprey engrailed gene, enga, shows <br /> sent several possible examples of the general DDC pro-           that eng1/eng1b and eng2/eng3 are ancient duplicate <br /> cess and the way in which it can account for observed            gene pairs (Figure 5). The tree shows that eng1 and <br /> patterns of duplicate gene expression. Additionally, we          eng1b are both orthologous to tetrapod En1, and that <br /> suggest experiments that could falsify the DDC model             eng2 and eng3 are both orthologous to tetrapod En2. <br /> as an explanation for these specific cases. We consider          The phylogenetic tree shows that the duplication <br /> here a pair of duplicate engrailed genes in zebrafish and        event(s) that produced the eng1/eng1b and eng2/eng3 <br /> the ZAG1 and ZMM2 gene pair in maize. Analysis of such           gene pairs occurred in the lineage of zebrafish after it <br /> cases must identify gene duplicates, determine whether           diverged from the lineage of tetrapods. The duplication <br /> they arose by tandem duplication or by duplication of            that produced the two main clades of vertebrate engrailed <br /> large chromosome regions, infer ancestral functions of           genes occurred before that divergence. <br /> the unduplicated parent gene, and finally determine                 To determine whether the zebrafish eng gene pairs <br /> whether the distribution of gene functions between du-           originated in chromosome-scale duplications or local <br /> plicated genes can be explained by the complementary             tandem duplications, we mapped the eng1b locus and <br /> sharing of ancestral functions or only by the acquisition        compared it to the genome locations of other engrailed <br /> of novel functions.                                              genes in mammals and zebrafish (Logan et al. 1989; <br />    Engrailed genes in zebrafish: Tetrapods have two mem-         Amores et al. 1998; Postlethwait et al. 1998). The <br /> bers of the engrailed gene family, called En1 and En2            results showed that the zebrafish genes eng1 and eng1b <br /> (Joyner and Martin 1987; Gardner and Barald                      are linked to dlx2 and dlx5 on two different chromo- <br />                                                 Preservation of Duplicate Genes                                         1539 <br />  <br />                                                                  fish duplicates eng1/eng1b. Therefore, En1 can be used <br />                                                                  as an outgroup to infer the ancestral shared expression <br />                                                                  domains of eng1 and eng1b. <br />                                                                     Although the expression patterns of engrailed genes <br />                                                                  are complex, here we focus on expression patterns of <br />                                                                  the engrailed-1 gene family in two groups of cells. Zebra- <br />                                                                  fish eng1 is expressed in the pectoral appendage bud, <br />                                                                  while eng1b is expressed in a specific set of neurons in <br />                                                                  the hindbrain/spinal cord (Figure 7). Is either of these <br />                                                                  expression domains due to neofunctionalization? Or <br />                                                                  were both present in the progenitor gene before dupli- <br />                                                                  cation and one domain lost by each duplicate? Examin- <br />                                                                  ing the most recent unduplicated outgroup would allow <br />                                                                  one to infer the state of the ancestral gene. In the ab- <br />                                                                  sence of information from the most recent outgroup, <br />                                                                  tetrapods can provide appropriate data. In mouse and <br />                                                                  chicken, En1 is expressed in both expression domains, <br />                                                                  the developing pectoral appendage bud, and in specific <br />                                                                  neurons of the hindbrain and spinal cord ( Joyner and <br />                                                                  Martin 1987; Davis et al. 1991; Gardner and Barald <br />                                                                  1992). Therefore it appears that following the duplica- <br />                                                                  tion event that produced eng1 and eng1b in the ray- <br />    Figure 6.&#226;&#8364;&#8221;Syntenic relationships of engrailed-1 genes. Ze-    finned lineage, the eng1 ancestor retained the pectoral <br /> brafish appears to have two copies of a single chromosome        appendage bud expression and lost the hindbrain/spi- <br /> segment present in the last common ancestor of teleosts and      nal cord neuron expression, while the eng1b ancestor <br /> mammals, one copy with eng1 and dlx2 and the other copy <br /> with eng1b and dlx5. One copy of this segment has been broken    lost the pectoral appendage bud expression and re- <br /> by a chromosome rearrangement, and appears in LG6 and            tained the hindbrain/spinal cord neuron expression as <br /> LG1. In humans, as in zebrafish LG9, this segment is in a        hypothesized in Figure 7. <br /> single chromosome arm, Hsa2q, but in mouse, this segment            Is this a case of gene preservation by subfunctionaliza- <br /> is on two chromosomes. Orthologous genes are on the same         tion? These data suggest complementary loss of expres- <br /> horizontal line. Syntenies are displayed, but gene orders are <br /> ignored to ease the comparisons of orthologues.                  sion, which is consistent with the DDC model. A defini- <br />                                                                  tive test of this hypothesis will require identification of <br />                                                                  the regulatory elements responsible for these expres- <br /> somes (Figure 5). Prior analysis had shown that the gene         sion domains in zebrafish, fish that share the eng1/eng1b <br /> duplicates dlx2 and dlx5 are both zebrafish orthologues          duplication, fish that diverged from the lineage giving <br /> of tetrapod DLX2 (Stock et al. 1996; Ellies et al. 1997),        rise to zebrafish before the duplication event, and tetra- <br /> which is linked to EN1 on human chromosome 2q                    pods, including mouse and chicken. In zebrafish, there <br /> (Ozcelik et al. 1992). Likewise, eng2 and eng3 genes             appear to be many examples similar to engrailed, includ- <br /> are linked to the gene duplicates shh and twhh on two            ing duplicates of msx genes (Ekker et al. 1997), nkx <br /> different zebrafish chromosomes (Postlethwait et al.             genes (Lee et al. 1996), and hedgehog genes (Ekker et <br /> 1998), and both of these zebrafish hedgehog genes are            al. 1995). <br /> orthologous to tetrapod SHH (Zardoya et al. 1996),                  ZAG1 and ZMM2 in maize: As a second possible exam- <br /> which is linked to EN2 on human chromosome 2q                    ple of the preservation of gene duplicates by subfunc- <br /> (Logan et al. 1989). These mapping results show that             tionalization, consider the duplicate genes known as <br /> zebrafish has two copies of large chromosome segments            ZAG1 and ZMM2 in the maize genome, which originated <br /> surrounding the zebrafish engrailed genes, and that these        via an allotetraploidization event between two closely <br /> syntenic regions are conserved with mammalian ge-                related grasses about 11 mya (Goodman et al. 1980; <br /> nomes (Figure 6). We conclude that the two gene pairs            Wendel et al. 1986; Helentjaris et al. 1988; Gaut and <br /> eng1/eng1b and eng2/eng3 arose by duplication events             Doebley 1997; White and Doebley 1998). ZAG1 and <br /> that involved large chromosome sections, consistent              ZMM2 are apparent orthologues of the single-copy flo- <br /> with their origin in a genome duplication event in ray-          ral homeotic genes known as AGAMOUS in Arabidopsis <br /> finned fish (Amores et al. 1998; Postlethwait et al.             and as PLENA in Antirrhinum, both of which are ex- <br /> 1998).                                                           pressed in carpels and stamens (Yanofsky et al. 1990; <br />    Note that two independent data sets, gene phyloge-            Coen and Meyerowitz 1991; Bradley et al. 1993). <br /> nies based on sequence information and chromosomal               ZAG1 is expressed at high levels throughout maize car- <br /> locations based on genetic mapping data, concur that             pel development, but it is expressed at only low levels <br /> the tetrapod En1 gene is an outgroup to the two zebra-           in stamen primordia (Mena et al. 1996). In contrast, <br />  1540                                                       A. Force et al. <br />  <br />  <br />  <br />  <br />                                                                                       Figure 7.&#226;&#8364;&#8221;Expression pattern of zebrafish <br />                                                                                    eng1 and eng1b. Expression of eng1 (left) and <br />                                                                                    eng1b (right) transcript in 28.5-hr zebrafish em- <br />                                                                                    bryos. Dorsal view of trunk showing the pecto- <br />                                                                                    ral appendage bud (top), which expresses <br />                                                                                    eng1, and dorsal view of head (bottom) show- <br />                                                                                    ing the expression of eng1b in a specific set of <br />                                                                                    hindbrain and spinal neurons. In situ hybrid- <br />                                                                                    ization was performed according to Jowett et <br />                                                                                    al. (1995) and Thisse et al. (1993). The eng1b <br />                                                                                    probe was derived from the 39 untranslated <br />                                                                                    region by the polymerase chain reaction using <br />                                                                                    using two primers (forward, GCTCATGGCT <br />                                                                                    CAAGGACTCTA; and reverse, AACATTG <br />                                                                                    GACTAAACGTAAAACTT) and cloned into <br />                                                                                    the TA cloning vector (Invitrogen, San Diego), <br />                                                                                    while the eng1 probe was a gift from Monte <br />                                                                                    Westerfield. The figure shows a hypothesized <br />                                                                                    evolutionary scenario for the tissue-specific <br />                                                                                    patterns of expression of the engrailed family <br />                                                                                    members in zebrafish on the left and in tetra- <br />                                                                                    pods on the right. Solid and open boxes indi- <br />                                                                                    cate full expression and lack of expression, <br />                                                                                    respectively. Blue boxes represent the pectoral <br />                                                                                    appendage regulatory element and red boxes <br />                                                                                    indicate the neuron regulatory element. <br />  <br />  <br />  <br />  <br /> ZMM2 is highly expressed in maize stamens but not at                 that the ancestral genes to ZAG1 and ZMM2 were both <br /> all in the immature carpel, although it increases in late            expressed strongly in the developing stamens and car- <br /> female flower development (Figure 8). The <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                  pels in the allotetraploid ancestor of maize shortly after <br /> of a ZAG1 null mutant is limited to early carpel develop-            the polyploidization event, as AGAMOUS and PLENA are <br /> ment, suggesting that late ZMM2 expression rescues                   today in Arabidopsis and Antirrhinum. We hypothesize <br /> later carpel development in ZAG1 mutants.                            that reciprocal regulatory mutations in the ZAG1/ <br />    The DDC model can explain these data by suggesting                ZMM2 duplicates complemented each other, thereby <br />                                                                      preserving both genes that exist in today&#226;&#8364;&#8482;s maize. After <br />                                                                      the allotetraploidization event, degenerative regulatory <br />                                                                      mutations decreased the expression of ZAG1 in stamens <br />                                                                      but not in carpel, while other regulatory mutations elim- <br />                                                                      inated the expression of ZMM2 in the early carpel but <br />                                                                      not in the stamens. If this hypothesis is correct, then, <br />                                                                      maize plants doubly homozygous for ZMM2 and ZAG1 <br />                                                                      null mutations should produce plants that phenocopy <br />                                                                      AGAMOUS and PLENA mutants in Arabidopsis and An- <br />                                                                      tirrhinum. In addition, molecular analysis of the pro- <br />                                                                      moters of this gene family in maize, its close relative <br />                                                                      sorghum, Arabidopsis, and Antirrhinum should identify <br />                                                                      conserved regulatory elements that became partitioned <br />                                                                      after gene duplication. <br />   Figure 8.&#226;&#8364;&#8221;The subfunctionalization hypothesis to explain              Hoxa1 and Hoxb1 in mouse: A third possible example <br /> the expression of duplicated genes in maize. Small solid boxes <br /> indicate active regulatory elements, gray boxes indicate regula-     of DDC in duplicate genes involves the Hoxa1 and Hoxb1 <br /> tory elements with reduced activity, and open boxes indicate         genes in mouse (Figure 9). These genes reside in dupli- <br /> null activity alleles for the regulatory element.                    cate Hox clusters, groups of closely linked genes that <br />                                                 Preservation of Duplicate Genes                                       1541 <br />  <br />                                                                  rhombomere 4. Judging from expression and functional <br />                                                                  analysis, an analogous element probably drives the early <br />                                                                  expression of Hoxa1 in rhombomere 5 and more caudal <br />                                                                  segments, but this element has yet to be defined molecu- <br />                                                                  larly. The DDC model predicts that the unduplicated <br />                                                                  ancestor of these two genes possessed regulatory ele- <br />                                                                  ments that drove expression of this Hox gene in both <br />                                                                  late r4 and early r5 expression domains, and that after <br />                                                                  duplication, these two subfunctions resolved divergently <br />                                                                  between the two paralogous copies (Figure 9). <br />                                                                     In addition to the independent roles of Hoxa1 and <br />                                                                  Hoxb1 just discussed, these two genes have early redun- <br />                                                                  dant roles, including expression in broadly overlapping <br />                                                                  territories and activation of some of the same down- <br />                                                                  stream targets (Studer et al. 1994, 1996; Dupe&#194;&#180; et al. <br />                                                                  1997; Maconochie et al. 1997; Chen and Ruley 1998). <br />                                                                  Some of the shared roles are important for their inde- <br />                                                                  pendent roles. Both genes contain a retinoic acid re- <br />   Figure 9.&#226;&#8364;&#8221;Can the DDC hypothesis explain the expression        sponse element (RARE) in their 39 region that initiates <br /> of duplicated Hox genes in mouse? Small solid boxes indicate     expression of the adjacent gene in neuroectoderm and <br /> a conserved RARE enhancer, medium boxes indicate active          mesoderm (Langston et al. 1997; Figure 9). The Hoxa1 <br /> rhombomere regulatory elements, and large boxes indicate         RARE is required not only for Hoxa1 expression in <br /> the transcribed coding regions. The &#226;&#8364;&#339;r5 enhancer?&#226;&#8364;&#65533; is an un-     rhombomere 5, but also for the proper level of Hoxa1 <br /> characterized element that is expected to drive expression <br /> from the r3/r4 border, through r5 to more caudal segments,       expression in the somitic and presomitic mesoderm <br /> and also to be necessary for glossopharyngeal nerve develop-     (Thompson et al. 1998). The requirement of a single <br /> ment. Open boxes indicate hypothetical dead regulatory ele-      conserved regulatory element (the RARE) for expres- <br /> ments. The r5 enhancer element in Hoxa1 has not yet been         sion in different domains (rhombomere 5 and meso- <br /> identified but is predicted to be present.                       derm) is a property expected of embedded regulatory <br />                                                                  elements as diagrammed in Figure 4, D, E, and G. The <br />                                                                  DDC model predicts that when subfunctions with <br /> encode a family of DNA-binding proteins that specifies           shared embedded regulatory elements become diver- <br /> fate along the anterior-posterior axis of bilaterian ani-        gently resolved, then redundant, overlapping expres- <br /> mals (Lewis 1978; Krumlauf 1994). The four tetrapod              sion domains will be retained, as observed for the <br /> Hox clusters arose from two sequential duplication               Hoxa1/Hoxb1 duplicated gene pair in portions of the <br /> events, probably whole genome duplications, before the           neuroectoderm and the mesoderm. <br /> divergence of ray-finned and lobe-finned fishes 420 mya             What experiments can distinguish whether the cur- <br /> (Duboule and Dolle&#194;&#180; 1989; Holland and Garcia-Fer-                rent subfunctions of murine Hoxa1 and Hoxb1 dupli- <br /> nandez 1996; Amores et al. 1998; Postlethwait et al.             cates arose by subfunctionalization, neofunctionaliza- <br /> 1998). The Hoxa and Hoxb clusters are probably sisters           tion, or some other model? A critical issue is whether <br /> derived from the second duplication event (Kappen and            the r4 and r5 enhancers were present in the ancestral <br /> Ruddle 1993; Zhang and Nei 1996; Amores et al. 1998).            gene before duplication. One can infer the state of the <br /> Thus, for this case, entire genes and all their regulatory       ancestral gene by examining an outgroup that diverged <br /> elements were duplicated, which might not be true of             from the lineage of tetrapods just before the duplication <br /> a tandem duplication.                                            event that produced the Hoxa1 and Hoxb1 genes. The <br />    Hoxb1 and Hoxa1 cooperate to pattern anterior ecto-           lamprey might be such an outgroup (Pendleton et al. <br /> dermal and mesodermal derivatives of vertebrate em-              1993; Amores et al. 1998; Carr et al. 1998) and experi- <br /> bryos (Figure 9). Hoxa1 is important for segment iden-           ments should be conducted to see if it is, and if so, to <br /> tity in rhombomere 5 (r5) of the hindbrain and for the           determine whether the corresponding functions were <br /> development of the glossopharyngeal nerve as well as             likely to have already been present in the ancestral gene <br /> more caudal rhombomeres (Dupe&#194;&#180; et al. 1997; Studer et            before duplication. <br /> al. 1998). Analogously, Hoxb1 is important for specifying           In summary, the examples discussed above provide <br /> the identity of rhombomere 4 (r4) and development of             data that are consistent with the DDC model, and in <br /> the facial nerve (Po&#194;&#168;pperl et al. 1995; Gavalas et al.           some cases are more readily explained by the DDC <br /> 1998). A 70-bp-long element, called the r4 autoregula-           model than the classical model. Further experiments <br /> tory enhancer, is conserved between mouse, chick, and            need to be done to firmly establish which route of dupli- <br /> pufferfish (Po&#194;&#168;pperl et al. 1995). This element functions        cate gene preservation was employed in each case. <br /> to elevate the expression of Hoxb1 and restrict it to               Testing the DDC and classical models: As we noted <br />  1542                                                  A. Force et al. <br />  <br /> earlier, even the most basic premise of the classical                                          TABLE 1 <br /> model of duplicate gene evolution&#226;&#8364;&#8221;that gene dupli-                      Critical values of zur/(uc 1 zur) required for specific <br /> cates are preserved only by the evolution of new func-                  probabilities of subfunctionalization (PS), given for <br /> tions&#226;&#8364;&#8221;has never been tested. Because deleterious mu-                      different numbers of subfunctions (z) obtained <br /> tations are much more common than beneficial                                           from Equations 3 and 4 <br /> mutations, we believe that the DDC process provides a <br /> reasonable (and parsimonious) alternative explanation                                                      PS <br /> for at least some cases of long-term preservation of gene       z                     0.1                 0.3                 0.5 <br /> duplicates. Unlike the classical model, the mutational <br /> mechanisms that lead to gene preservation by DDC are             2                  0.448                0.774               1.000 <br /> distinct from those responsible for the origin of new            3                  0.384                0.654               0.832 <br />                                                                  5                  0.351                0.598               0.759 <br /> gene functions. On the other hand, by expanding the <br />                                                                 10                  0.332                0.57                0.728 <br /> time period for which genes are exposed to selection,           &#226;&#710;&#382;                   0.316                0.548               0.708 <br /> the preservation of duplicates by the DDC process facili- <br /> tates subsequent opportunity for the evolution of new <br /> functions. If the evolution of new gene functions is the <br /> only mechanism of duplicate gene preservation, then it             or larger to achieve at least a 50% probability of <br /> should be possible to empirically reject our alternative           gene preservation (Table 1). If the relative rate of <br /> subfunctionalization hypothesis. We now consider some              formation of subfunctional alleles is not within this <br /> potentially fruitful avenues for future research.                  range, then subfunctionalization as modeled is un- <br />                                                                    likely to be a major mechanism of duplicate gene <br /> 1. Phylogenetic analysis: The subfunctionalization                 preservation. Experiments must be designed to mea- <br />    model predicts that the sum of subfunctions in pre-             sure the mutation rate to subfunctional, neofunc- <br />    served gene duplicates will be equal to the total sub-          tional, and nonfunctional alleles to test critically the <br />    functions in the ancestral gene. This prediction is             various models. If empirical studies demonstrate that <br />    clearly distinct from the position of the classical             the rate of mutation to subfunctional alleles is too <br />    model, which suggests that gene preservation is de-             low relative to the rate of coding null mutations, then <br />    pendent upon the acquisition of new cis-regulatory              this particular subfunctionalization model is falsified. <br />    regions driving novel expression patterns during de-         3. Regulatory region complexity: The subfunctionaliza- <br />    velopment (Sidow 1996; Cooke et al. 1997). To test              tion model predicts that the probability of gene pres- <br />    these alternative hypotheses, the evolutionary time             ervation should be higher for more complex genes <br />    frame must be short enough to preclude the possibil-            (with larger numbers of subfunctions), particularly <br />    ity that genes initially preserved by subfunctionaliza-         for genes in which the regulatory regions for subfunc- <br />    tion will have also subsequently acquired new func-             tions are spatially independent on the DNA because <br />    tions. This then requires the analysis of recently              more complex genes provide more targets for sub- <br />    preserved duplicates on a cladogram that also allows            function mutations. Testing this prediction requires <br />    the inference of ancestral expression patterns from             the molecular characterization of regulatory regions <br />    appropriate outgroups. For example, to explain the              for various genes in species with duplicated genes <br />    derivation of the triplicate Drosophila genes paired,           and comparison with closely related species without <br />    gooseberry, and gooseberry-neuro, which have conserved          the duplications. <br />    protein function but distinct developmental func-            4. Multiple polyploidization events: The subfunctionali- <br />    tions, Li and Noll (1994) suggested that following              zation model makes specific predictions about the <br />    duplication &#226;&#8364;&#339;genes may acquire new functions by                 probability of duplicate gene preservation after <br />    changes in their regulatory regions generating an               closely spaced polyploidization events, such as those <br />    altered expression&#226;&#8364;&#65533; without considering the possibil-           thought to have occurred early in vertebrate phylog- <br />    ity that these three genes simply result from the dif-          eny (Holland et al. 1994; Holland and Garcia- <br />    ferential loss of subsets of the expression domains of          Fernandez 1996; Nadeau and Sankoff 1997; <br />    the ancestral gene. A phylogenetic analysis of closely          Amores et al. 1998). The DDC model suggests that <br />    related outgroup species with single gene copies                duplicate loci preserved by subfunctionalization after <br />    would distinguish between the classical and DDC                 the first polyploidization event (Figure 1, right) will <br />    models.                                                         have fewer subfunctions than the parent locus before <br /> 2. Mutation rate to subfunctional alleles: The simple              duplication (Figure 1, top). Therefore, because the- <br />    subfunctionalization model discussed here requires              ory predicts that the likelihood of preservation de- <br />    that the total subfunction mutation rate relative to            pends on the number of subfunctions (Figure 2), the <br />    the total null mutation rate be on the order of 0.3             probability that both duplicate loci will be preserved <br />    or larger to achieve at least a 10% probability of              after the second round of duplication is diminished <br />    duplicate gene preservation, and on the order of 0.7            relative to the first polyploidization event. If, on the <br />  <br />  <br />    other hand, a single duplicate survives the first round      seriously as a testable model for explaining the evolu- <br />    with all of the original subfunctions intact (Figure         tionary fate of duplicate genes. <br />    1, left), then after the second round of duplication,         <br />    the probability of duplicate preservation will be ap-    <br />    proximately the same as in the first event. If the           <br />    level of gene preservation does not change between         <br />    polyploidization events, then subfunctionalization is         <br />    an unlikely explanation for the preservation of dupli- <br />    cate genes. Data from the HOX complexes of verte-            <br />    brates suggest that the level of duplicate preservation      <br />    does indeed decline with subsequent duplication          <br />    events. Assuming the (AB)(CD) model of HOX clus- <br />    ter duplication (Kappen and Ruddle 1993; Zhang <br />    and Nei 1996; Amores et al. 1998), then 11 of 14 <br />                                                                                    <br />    gene pairs (13 HOX cluster genes plus EVX; 79%) <br />    were both preserved after the first duplication      <br />    (Amores et al. 1998). After the second duplication,               <br />    in the lineage leading to mouse, 16 of 25 gene pairs         <br />    (64%) were preserved, while after the third duplica-          <br />    tion, which occurred in the ray-finned fish lineage        <br />    leading to zebrafish (Amores et al. 1998), only 13 of          <br />    45 gene pairs (29%) have been preserved to the            <br />    present in duplicate copies. With the caveat that a            <br />    few genes may remain to be discovered in the zebra-    <br />    fish, and that all genes still present in duplicate copy      <br />    at the time of the second and third duplication events      <br />    had been permanently preserved by some mecha-              <br />    nism, the results conform at least to the qualitative        <br />    expectations of the DDC model.                                   <br /> 5. Population size: Finally, because the subfunctionali-           <br />    zation model assumes that selection against mutant      <br />    alleles is negligible, the predictions for this model      <br />    should be fulfilled most closely in populations of         <br />    relatively small size, because under these conditions,          <br />    the incidence of double null homozygotes and the                <br />    appearance of beneficial mutations is minimal. On        <br />    the other hand, because neofunctionalization relies         <br />    entirely on rare beneficial mutations, it should be a         <br />    more common mechanism of gene preservation in                <br />    species with large effective sizes.                         <br />  <br />    Because it focuses on the regulatory complexity of         <br /> genes and the preservational nature of degenerative                <br /> mutations, the DDC model provides a different perspec-           <br /> tive on the evolutionary consequences of gene duplica-            <br /> tions than that of the classical model, which focuses           <br /> primarily on the nonfunctionalizing properties of de-              <br /> generative mutations in coding regions and on the neo-       <br /> functionalizing properties of beneficial mutations. The             <br /> DDC model organizes a rather disparate collection of             <br /> observations and principles on gene structure, and we          <br /> believe that most of these principles are established       <br /> firmly enough that subfunctionalization must be taken       <br /> </body></html>