<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  Mutations in SID2, a Novel Gene in Saccharomyces cerevisiae, Cause Synthetic <br />     Lethality With sic1 Deletion and May Cause a Defect During S Phase <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 SIC1 encodes a nonessential B-type cyclin/CDK inhibitor that functions at the G1/S transition and the <br />              exit <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> mitosis. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> understand more completely the regulation of these transitions, mutations causing <br />              synthetic lethality with sic1&#226;&#338;&#172; were isolated. In this screen, we identified a novel gene, SID2, which encodes <br />              an essential protein that appears to be required for DNA replication or repair. sid2-1 sic1&#226;&#338;&#172; strains and <br />              sid2-21 temperature-sensitive strains arrest preanaphase as large-budded cells with a single nucleus, a short <br />              spindle, and an &#226;&#181;&#8216;2C DNA content. RAD9, which is necessary for the DNA damage checkpoint, is required <br />              for the preanaphase arrest of sid2-1 sic1&#226;&#338;&#172; cells. Analysis of chromosomes in mutant sid2-21 cells by field <br />              inversion gel electrophoresis suggests the presence of replication forks and bubbles at the arrest. Deleting <br />              the two S phase cyclins, CLB5 and CLB6, substantially suppresses the sid2-1 sic1&#226;&#338;&#172; inviability, while stabilizing <br />              Clb5 protein exacerbates the defects of sid2-1 sic1&#226;&#338;&#172; cells. In synchronized sid2-1 mutant strains, the onset <br />              of replication appears normal, but completion of DNA synthesis is delayed. sid2-1 mutants are sensitive <br />              to hydroxyurea indicating that sid2-1 cells may suffer DNA damage that, when combined with additional <br />              insult, leads to a decrease in viability. Consistent with this hypothesis, sid2-1 rad9 cells are dead or very <br />              slow growing even when SIC1 is expressed. <br />  <br />  <br />  <br /> P    ASSAGE through the eukaryotic cell cycle is regu- <br />     lated by cyclin-dependent kinases (CDKs). CDKs are <br /> active when bound to cyclins and it appears that cyclins <br />                                                                            upon the transcription factor Swi5p, and disappearing <br />                                                                            at the G1/S transition (Knapp et al. 1996). A decrease <br />                                                                            in Clb2p-Cdc28p activity is needed for passage into G1 <br /> are responsible for much of the functional specificity                     and it appears that Sic1p assists in this process by inhib- <br /> of the cyclin-CDK complex. The activity of CDK com-                        iting the CDK complex (Toyn et al. 1997). While cells <br /> plexes is regulated at the level of expression of CDKs                     can enter G1 with elevated levels of Clb2p, overex- <br /> and cyclins, as well as post-translationally by phosphory-                 pressed Clb2&#226;&#338;&#172;DBp (destruction box deleted) is lethal, <br /> lation, regulated degradation, and CDK inhibitors. In                      causing cell cycle arrest in telophase (Surana et al. <br /> the budding yeast Saccharomyces cerevisiae, Cdc28p is the                  1993). Clb2-Cdc28 kinase activity is also decreased by <br /> main CDK involved in cell cycle control, forming a com-                    proteosome degradation of Clb2p after its ubiquitina- <br /> plex with both the G1 cyclins (Cln1-3) and the B-type                      tion by the anaphase-promoting complex (APC; Surana <br /> cyclins (Clb1-6). The G1 cyclins act upstream of the                       et al. 1993; Irniger et al. 1995). <br /> events controlling the G1/S phase transition, including                       At the G1/S transition, degradation of Sic1p releases <br /> bud formation, microtubule organizing center duplica-                      inhibition of Clb5/6p-Cdc28p, which then induce DNA <br /> tion, and DNA replication (reviewed by Lew et al. 1997).                   replication (Schwob et al. 1994; Schneider et al. 1996). <br /> One essential function of the G1 cyclins is the inactiva-                  Sic1p proteolysis is dependent on its Cln-dependent <br /> tion of the B-type cyclin-Cdc28 inhibitor, Sic1p, as sic1                  phosphorylation (Schneider et al. 1996; Verma et al. <br /> deletion rescues strains containing a cln1&#226;&#338;&#172; cln2&#226;&#338;&#172; cln3&#226;&#338;&#172;                    1997) and upon Cdc34p, an E2 ubiquitin-conjugating <br /> triple deletion that are otherwise inviable (Schneider                     enzyme (Schwob et al. 1994). Inducing GAL-CLB5&#226;&#338;&#172;DB <br /> et al. 1996). <br />                                                                            expression advances DNA replication in sic1&#226;&#338;&#172;, but not <br />    As an inhibitor of Clb-Cdc28 kinase activity, Sic1p <br />                                                                            SIC1 cells, indicating that Sic1p has a function in regulat- <br /> appears to function to allow cells to exit mitosis as well <br />                                                                            ing S phase entry (Schwob et al. 1994). Clb5p and Clb6p <br /> as to prevent premature DNA replication (Schwob et <br />                                                                            seem to have related, but not identical, roles in initiating <br /> al. 1994; Toyn et al. 1997). SIC1 transcripts are cell cycle <br />                                                                            DNA replication. clb5&#226;&#338;&#172; cells initiate DNA replication <br /> regulated, accumulating late in M phase dependent <br />                                                                            normally, but take twice as long to complete DNA syn- <br />                                                                            thesis (Epstein and Cross 1992; Ku&#194;&#168;hne and Linder <br />                                                                            1993). Cells deleted for clb6 display a normal onset and <br />   Corresponding author: Elizabeth A. Vallen, Department of Biology, <br /> Swarthmore College, Swarthmore, PA 19081.                                  duration of replication. However, when clb5 and clb6 <br /> E-mail: evallen1@swarthmore.edu                                            are both deleted, DNA synthesis is delayed, but the <br />  <br /> Genetics 159: 17&#226;&#8364;&#8220;33 (September 2001) <br />  18                                                 M. D. Jacobson et al. <br />  <br /> duration of replication, once begun, is unaffected              nation with sic1&#226;&#338;&#172;, disrupt the normal regulated process <br /> (Schwob and Nasmyth 1993). Recent evidence sug-                 of DNA replication sufficiently to render the cells invia- <br /> gests that Clb5p can activate early and late origins of         ble. To identify such genes, as well as other genes playing <br /> replication while Clb6p can activate only the early ori-        roles in the exit from mitosis, we screened for mutations <br /> gins (Donaldson et al. 1998). This result agrees with           causing synthetic lethality with sic1&#226;&#338;&#172;. Here we report <br /> the phenotypes of clb5 and clb6 single mutants. The             on the discovery of a novel gene, SID2, which appears <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of the double deletion can then be explained          to play a role in DNA replication. <br /> if the remaining Clbs (Clb1-4) can trigger both early <br /> and late origins to fire (Donaldson et al. 1998). <br />    In addition to timing DNA replication, inhibition of                        MATERIALS AND METHODS <br /> Clb5/6p-Cdc28p activity by Sic1p may function to regu-             Yeast strains and media: YP-dextrose (YPD), YP-galactose <br /> late origin loading and the DNA replication machin-             (YPGal), and synthetic complete (SC) minimal media were <br /> ery. Binding of the six-subunit origin recognition com-         made by standard techniques (Ausubel et al. 1987). Hydroxy- <br /> plex (ORC), the Mcm family (Mcm2-7), and Cdc6p is               urea and &#226;&#65533;&#163;-factor (both from Sigma Chemical, St. Louis) were <br />                                                                 used at 0.2 m and 0.1 &#226;&#65533;&#174;m, respectively. The strains and plasmids <br /> thought to make the origins competent for firing by             used in this study are listed in Tables 1 and 2, respectively. <br /> Clb5/6p-Cdc28p kinase activity (reviewed in Diffley             All yeast strains are isogenic with BF264-15D (trp1-1a leu2- <br /> 1996; Stillman 1996). Origin loading is inhibited by            3,112 ura3 ade1 his2) and are bar1&#226;&#171;&#186; unless otherwise noted. <br /> CDK activity and this is thought to be the basis for a          Standard methods were used for all strain constructions, <br /> mechanism that allows replication to occur once per             crosses, and transformations (Ausubel et al. 1987; Rose et al. <br />                                                                 1990; Guthrie and Fink 1991). A disruption of sic1 marked <br /> cell cycle (Dahmann et al. 1995). Cdc6p appears to              with TRP1, SIC1 under the control of the inducible GAL1 <br /> recruit Mcm binding to the origins. In the presence,            promoter (Nugroho and Mendenhall 1994; the gifts of M. <br /> but not in the absence of SIC1, late G1 expression of           Mendenhall) and a disruption of rad9 marked with LEU2 <br /> Cdc6p (under the control of the HO promoter) can                (Weinert and Hartwell 1990) were integrated into the <br /> promote Mcm binding (Tanaka et al. 1997). This sug-             BF264-15D background. The clb5::ARG4 and clb6::ADE1 dis- <br />                                                                 ruptions were made in the BF264-15D background and have <br /> gests that Sic1-induced delay of Clb-Cdc28 activity allows      been previously described (Epstein and Cross 1992; Schwob <br /> proper origin binding of competence factors in prepara-         and Nasmyth 1993). The CLB2HA construct (Schwab et al. <br /> tion for DNA replication. Clb-Cdc28 kinase activity and         1997), CLB5 HA, and CLB5&#226;&#338;&#172;DBHA constructs have also been <br /> Sic1p also affect DNA replication in a Cdc6- and ORC-           described (Cross et al. 1999). Yeast strain L40 and plasmids <br /> independent fashion, suggesting that the kinase may             pNIA, pNIAE2, and pNEAE2 have been previously described <br />                                                                 (Rhee et al. 2000). <br /> also have direct effects on enzymes required for DNA               Plating efficiency assays: Tenfold serial dilutions in water <br /> synthesis (Duncker et al. 1999). One possibility is that        were made from fresh stationary-phase cultures and 5 &#226;&#65533;&#174;l from <br /> the Clb-Cdc28 kinase regulates the association of DNA           each dilution was plated. Plates were incubated for 2&#226;&#8364;&#8220;4 days <br /> polymerase-primase (pol&#226;&#65533;&#163;) to chromatin (Desdouets               at 30&#226;&#172;&#353;. <br /> et al. 1998).                                                      Mutagenesis and sic1&#226;&#338;&#172; lethality screen: LY623 and MJ65 <br />                                                                 yeast cells were mutagenized using standard procedures (Rose <br />    SIC1 is not an essential gene, but sic1 cells show a         et al. 1990) to &#226;&#181;&#8216;30% viability. Mutagenized cells were plated <br /> high frequency of chromosome loss and breakage (Nug-            on YPGal (&#226;&#181;&#8216;200 colonies per plate). The colonies were then <br /> roho and Mendenhall 1994). Several genetic back-                screened by replica plating for mutants that were alive on <br /> grounds make SIC1 essential, including dbf2&#226;&#338;&#172;, GAL-CLB2,         YPGal and dead on YPD. <br /> cdh1&#226;&#338;&#172;/hct1&#226;&#338;&#172;, cdc23-1, and rsi1-1 (apc2; Schwab et al.              Library screening: LEU2 CEN4 plasmids, which comple- <br />                                                                 mented sid2-1, were isolated by transforming a sid2-1 strain <br /> 1997; Toyn et al. 1997; Kramer et al. 1998). Overexpres-        (MJ163) with American Type Culture Collection library 77162 <br /> sion of Sic1p is able to rescue cdc5, cdc14-1, cdc15, and       (constructed by P. Hieter in pBS32). Transformants were se- <br /> cdc20-1 strains (Schwab et al. 1997; Toyn et al. 1997;          lected on SCGal-Leu minimal media plates and replica plated <br /> Jasperson et al. 1998). All of these genetic interactions       to YPD and YPGal in order to isolate colonies that could <br /> appear to be related to Sic1&#226;&#8364;&#8482;s function at the exit from        grow on YPD. The plasmids were recovered from Dex&#226;&#171;&#185; strains <br />                                                                 (Hoffman and Winston 1987) and plasmid linkage was tested <br /> mitosis. Cdc23p and Apc2p are members of the APC                after retransformation. Partial sequence was obtained by The <br /> (Zachariae et al. 1998b) while Cdc20p and Cdh1p/                Rockefeller University Protein/DNA Technology Center using <br /> Hct1p seem to function as APC activators (Schwab et             primer pBRSB (ACCGCACCTGTGGCGCCG), which hybrid- <br /> al. 1997; Visintin et al. 1997). dbf2, cdc5, cdc14, and         izes to pBR322 sequences 31 <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairs upstream of the BamHI <br /> cdc15 all have terminal arrest phenotypes late in mitosis       site. <br />                                                                    Cloning and disrupting SID2: All restriction enzymes and <br /> (Byers and Goetsch 1973; Pringle and Hartwell                   DNA modifying enzymes were used according to the manufac- <br /> 1981; Johnston et al. 1990; Kitada et al. 1993) and may         turer&#226;&#8364;&#8482;s instructions. A 3.4-kb fragment (PstI to SpeI) containing <br /> activate the APC or dephosphorylate Cdc28 substrates            the entire YJR046w open reading frame (ORF) was isolated <br /> or regulators (Jaspersen et al. 1998; Visintin et al.           from a LEU2 CEN4 library plasmid that rescued sid2-1. This <br /> 1998). Taken together, these data indicate that SIC1            fragment was cloned into pRS405 to form pMJ01 and into <br />                                                                 pRS415 to form pMJ02 (Figure 1). pMJ01 was digested with <br /> plays an important role in late mitosis. If SIC1 also has       NcoI (which cuts uniquely in YJR046w) and integrated by ho- <br /> a significant role in regulating DNA replication, it is         mologous recombination into the sid2-1 haploid MJ193 to <br /> possible that genes exist which, when mutated in combi-         form MJ257. The integration was confirmed by Southern blot <br />                                                    SID2 and S Phase Cyclin Activity                                               19 <br />  <br />                                                               TABLE 1 <br />                                                      Strains used in this study <br />  <br /> Strain                                Relevant genotype                                                   Source <br /> KK1             MAT&#226;&#65533;&#163; sic1::TRP1 GAL1-SIC1::URA3 rad9::LEU2                           MJ65 <br /> KK11            MAT&#226;&#65533;&#163; SID2 sic1::TRP1 GAL1-SIC1                                       KK1 &#226;&#171;&#187; MJ163 <br /> KK17            MATa sid2-1 sic1::TRP1 GAL1-SIC1                                     KK1 &#226;&#171;&#187; MJ163 <br /> KK20            MATa SID2 sic1::TRP1 GAL1-SIC1 rad9::LEU2                            KK1 &#226;&#171;&#187; MJ163 <br /> KK23            MAT&#226;&#65533;&#163; sid2-1 sic1::TRP1 GAL1-SIC1 rad9::LEU2                          KK1 &#226;&#171;&#187; MJ163 <br /> L40             lexAoperator::HIS3 lexA operator::lacZ trp1                          Hollenberg et al. (1995) Rhee et al. (2000) <br /> LY623           MATa sic1::TRP1 GAL1-SIC1::URA3 <br /> LY677           MAT&#226;&#65533;&#163; cdc15-2 sic1::TRP1                                              K2944 (Oehlen and Cross 1994) <br /> LY699           MAT&#226;&#65533;&#163; dbf2::URA3 sic1::TRP1 GAL1-SIC1::URA3                           S7-4A (L. H. Johnston) <br /> LY907           MAT&#226;&#65533;&#163; sid2-1 sic1::TRP1 GAL1-SIC1 <br /> LY909           MAT&#226;&#65533;&#163; sic1::TRP1 GAL1-SIC1 <br /> LY914           MATa HIS2 his3 <br /> LY915           MATa/MAT&#226;&#65533;&#163; HIS2/HIS2 his3/his3 <br /> LY925           MATa HIS2 his3 SID2-PrA::his5&#226;&#171;&#185; <br /> LY985           arg4 sic1::TRP1 GAL1-SIC1::URA3 clb6::ADE1 clb5::ARG4 <br /> LY986           MATa arg4 sic1::TRP1 GAL1-SIC1::URA3 <br /> LY987           arg4 sic1::TRP1 GAL1-SIC1::URA3 sid2-1 <br /> LY988           arg4 sic1::TRP1 GAL1-SIC1::URA3 clb6::ADE1 clb5::ARG4 sid2-1 <br /> LY989           arg4 sic1::TRP1 GAL1-SIC1::URA3 clb5::ARG4 <br /> LY991           arg4 sic1::TRP1 GAL1-SIC1::URA3 clb5::ARG4 sid2-1 <br /> LY1023          MATa HIS2 his3 sid2::LEU2 pSU3[SID2-RS416] <br /> LY1036          MATa HIS2 his3 SID2                                                  LY914 <br /> LY1037          MATa HIS2 his3 sid2-21                                               LY914 <br /> LY1118          MATa sid2-1 <br /> MJ55            MATa sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ58            MATa sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ65            MAT&#226;&#65533;&#163; sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ160           MATa sid2-1 sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ163           MATa sid2-1 sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ193           MAT&#226;&#65533;&#163; sid2-1 sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ249           MAT&#226;&#65533;&#163; cdh1::LEU2 sic1::TRP1 GAL1-SIC1::URA3                           W320 (Schwab et al. 1997) <br /> MJ257           MAT&#226;&#65533;&#163; siD2::LEU2 sic1::TRP1 GAL1-SIC1::URA3 <br /> MJ282           MATa sid2-1 CLB2HA sic1::TRP1 GAL1-SIC1::URA3                        W9317 (Schwab et al. 1997) <br /> MJ288           MATa CLB2HA sic1::TRP1 GAL1-SIC1::URA3                               W9317 (Schwab et al. 1997) <br /> MJ292           MATa cdh1::LEU2 CLB2HA sic1::TRP1 GAL1-SIC1::URA3                    W320, W9317 (Schwab et al. 1997) <br /> MJ316           MATa SID2 sic1::TRP1 GAL1-SIC1 CLB5HA <br /> MJ317           MATa sid2-1 sic1::TRP1 GAL1-SIC1 CLB5HA <br /> MJ319           MATa SID2 sic1::TRP1 GAL1-SIC1 CLB5&#226;&#338;&#172;DBHA <br /> MJ321           MAT&#226;&#65533;&#163; sic1::TRP1 GAL1-SIC1::URA3 sid2-1 CLB5&#226;&#338;&#172;DBHA <br /> MJ322           MATa sic1::TRP1 GAL1-SIC1::URA3 sid2-1 <br /> MJ323           MATa sic1::TRP1 GAL1-SIC1::URA3 <br /> MJx21-4D        MAT&#226;&#65533;&#163; dbf2::URA3 GAL1-SIC1::URA3                                      S7-4A (L. H. Johnston) <br />   All strains are ade1 trp1-1 ura3 leu2-3,112 his2 bar1 and are from this study unless otherwise noted. <br />  <br />  <br /> analysis. MJ257 was crossed to a SID2 strain (MJ58). The dip-        the digested plasmid were blunted using Klenow enzyme and <br /> loid was sporulated and the tetrads were dissected for analysis.     then ligated, to form pMJ03. pMJ03 was then digested with <br /> If the sid2-1 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> was not linked to YJR046w, approxi-           XbaI, liberating a 970-bp fragment internal to SID2, and the <br /> mately half of the Leu&#226;&#171;&#186; spores should have had the Sid&#226;&#171;&#186;              ends were blunted with Klenow enzyme. A 3.5-kb fragment <br /> phenotype because the mutation would have been segregating           containing the LEU2 and Kanr genes from pJA51-&#226;&#338;&#172;P digested <br /> independently of the LEU2 integration. None of the spores            with SmaI (Cross 1997) was ligated to the digested MJ03 to <br /> (54 spores analyzed) were Sid&#226;&#171;&#186;, indicating that the sid2-1 muta-     form MJ07. pMJ07 was digested with HindIII and the resulting <br /> tion was linked to YJR046w. In addition, when MJ257 was              5-kb fragment was gel purified and used to disrupt SID2 in a <br /> crossed to sid2-1 strains, all Leu&#226;&#171;&#185; spores were Sid&#226;&#171;&#185; and all         diploid by homologous recombination. A diploid containing <br /> Leu&#226;&#171;&#186; spores were Sid&#226;&#171;&#186; (47 spores analyzed), indicating tight         the deletion (confirmed by Southern blot) was sporulated and <br /> linkage between the integrated DNA and the SID2 locus.               tetrads were dissected. <br />    SID2 was deleted in a diploid heterozygous for sic1&#226;&#338;&#172; and             Construction of the temperature-sensitive SID2 allele, sid2- <br /> GAL1-SIC1 using pMJ07, which was derived from pMJ01 in               21: SID2-HIS3 (pSH6) and SID2-URA3 (pSU3) plasmids were <br /> the following way. The XbaI polylinker site in pMJ01 was re-         made by gap repair by first isolating a 5.6-kb fragment (ScaI <br /> moved with a SpeI/NotI digest. The 5&#226;&#172;&#732; overhanging ends of            to DraIII) containing all of SID2 and flanking vector sequences <br />  20                                                        M. D. Jacobson et al. <br />  <br />                             TABLE 2                                    were used to amplify the protein A-his5&#226;&#171;&#185; fragment for C-termi- <br />                                                                        nal addition of protein A to Sid2: 046-PROTA 5&#226;&#172;&#732;, GGATA <br />                   Plasmids used in this study                          AAAACAGATTTTCTAAGCTGTTGCAAATCCACAAATCAA <br />                                                                        AACAACAAGATGGTGAAGCTCAAAAACTTAAT; 046-HIS3&#226;&#172;&#732;, <br /> Name        Vector           Relevant genes            Reference       CGTACATACACAATGCACAGTCTTCAAAGTAAAATACCAA <br />                                                                        CGTATGTATCAAGATCGTCGACGGTATCGATAAGCTT <br /> pMJ01       RS405        SID2, LEU2                    This study      where the underlined sequences correspond to those from <br /> pMJ02       RS415        SID2, LEU2                    This study      SID2. Twenty-five cycles of PCR consisting of 1 min at 95&#226;&#172;&#353;, 1 <br /> pMJ03       RS405        SID2, LEU2 (XbaI in           This study      min at 55&#226;&#172;&#353;, and 4 min (&#226;&#171;&#185;5-sec increase/cycle) at 72&#226;&#172;&#353; were <br />                             polylinker removed)                        performed. The PCR products were transformed into his3/ <br /> pMJ07       RS405        sid2::LEU2-Kanr               This study      his3 diploid cells (LY915) and His&#226;&#171;&#185; transformants were se- <br /> pSIC1       RS315        SIC1, LEU2                    M. Tyers        lected. Homologous recombination resulted in SID2-ProA fu- <br /> pSH6        RS413        SID2, HIS3                    This study      sions linked to his5&#226;&#171;&#185;. Putative protein A-tagged strains were <br /> pSU3        RS416        SID2, URA3                    This study      analyzed by Western and Southern blotting to verify tagging <br />                                                                        of the Sid2 protein. <br />                                                                           Construction of lexA-GAL4-SID2 fusions and &#226;&#8364;&#732;&#226;&#8364;&#732;one-hybrid&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                        assay: The SID2 gene was amplified with forward primer 5&#226;&#172;&#732;-AAA <br /> from pMJ02 and 3-kb PvuI vector fragments from both pRS413             GAGATCGAATACCCGGGGATCCTTATGAGTGGCACAGCC <br /> and pRS416. The pMJ02 SID2 fragment and either the pRS413              TAAT-3&#226;&#172;&#732; and reverse primer 5&#226;&#172;&#732;-TCGCCCGGAATTAGCTTGGCT <br /> or pRS416 fragment were cotransformed into LY914 and the               GCAGTTCAATCTTGTTGTTTTGAT-3&#226;&#172;&#732; using the Expand High <br /> resultant plasmids were isolated. pSH6 was then mutagenized            Fidelity PCR system (Roche, Indianapolis). The underlined se- <br /> with hydroxylamine according to standard procedures (Rose              quences correspond to those from SID2 and the italicized se- <br /> et al. 1990).                                                          quences correspond to sequences in the pNIA plasmid. Thirty- <br />    Haploid strains containing pSU3 (SID2-URA3) were deleted            three cycles of PCR each consisting of of 1 min at 94&#226;&#172;&#353;, 1 min at <br /> for SID2 using pMJ07 as described above. The sid2::LEU2 dele-          50&#226;&#172;&#353;, and 2 min at 68&#226;&#172;&#353; were performed. The PCR fragment was <br /> tion was confirmed in the resulting Leu&#226;&#171;&#185; FOAS strains by               recombined into plasmid pNIA (Rhee et al. 2000) by cotransfor- <br /> Southern blot analysis. One such strain, LY1023, was trans-            mation into yeast. Plasmids were recovered from yeast (Hoffman <br /> formed with pSH6 (SID2-HIS3) and became FOAR as expected.              and Winston 1987), electroporated into Escherichia coli, and <br /> LY1023 was then transformed with hydroxylamine-mutagen-                analyzed by restriction analysis. Six independently derived NIA- <br /> ized pSH6 and plated on SCDex-His at room temperature                  SID2 plasmids were transformed into strain L40 (Rhee et al. <br /> (&#226;&#181;&#8216;200 colonies per plate). The colonies were replica plated            2000) and liquid &#226;&#65533;&#164;-galactosidase assays were performed on log <br /> to SCDex-His and SCDex &#226;&#171;&#185; FOA at both room temperature                  phase cultures grown under selective conditions as described <br /> and 37&#226;&#172;&#353; and then screened for mutants that were dead only              (Ausubel et al. 1987). <br /> on SCDex &#226;&#171;&#185; FOA at 37&#226;&#172;&#353;. The plasmids were recovered from                   Yeast fractionation: Fractionation was performed (Rout and <br /> these strains and transformed back into LY1023 to verify the           Kilmartin 1998) using the modifications for S. cerevisiae de- <br /> phenotype. Five sid2-ts plasmids were isolated from &#226;&#181;&#8216;7700              scribed with a 1-liter culture of Wickerham&#226;&#8364;&#8482;s media grown over- <br /> colonies screened with a 1% frequency of sid2 null mutations           night to an optical density (660 nm) of 0.8. The volumes loaded <br /> (FOAS at room temperature and 37&#226;&#172;&#353;).                                    for Western blot analysis were adjusted to compensate for varying <br />    The 3.5-kb SpeI to SalI fragments containing the sid2-ts al-        total volumes of each collected fraction. <br /> leles were isolated from the five plasmids and cloned into                Phenotypic analysis of sid2-1 sic1&#226;&#338;&#172; and sid2-21 strains: SID2 <br /> pRS406 digested with XhoI and SpeI. These new sid2-ts integrat-        and sid2-1 strains (both sic1&#226;&#338;&#172; GAL1-SIC1) were grown overnight <br /> ing plasmids were digested with XhoI (which cuts uniquely              to early log phase in liquid YPGal media. The cultures were then <br /> in SID2) and integrated by homologous recombination into               split and dextrose was added to one-half of each to a final <br /> LY914. Purified Ura&#226;&#171;&#185; transformants were patched onto YPD               concentration of 2%. YPD cultures of sid2-21 and SID2 strains <br /> at room temperature and then streaked on SCDex &#226;&#171;&#185; FOA at                were grown overnight at 25&#226;&#172;&#353; to early log phase and then shifted <br /> room temperature. The resulting FOAR colonies were screened            to 37&#226;&#172;&#353;. For both experiments, samples were removed at 2-hr <br /> for temperature-sensitive growth. Temperature-sensitive strains        intervals and processed for FACS analysis, cell counting, or immu- <br /> were recovered from only one of the original five alleles (sid2-21)    nofluorescence staining as described below. For the synchroniza- <br /> although at least 48 FOAR colonies from 12 independent trans-          tion experiments, early log phase cultures of SID2, sid2-1, SID2 <br /> formants were screened for each allele. Integration after diges-       sic1&#226;&#338;&#172; GAL1-SIC1, and sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 were grown in YPGal <br /> tion of the remaining four plasmids with BstEII, which cuts            at 30&#226;&#172;&#353;. &#226;&#65533;&#163;-factor was added and incubation continued for 3 hr. <br /> upstream of SID2, was also tried, but again, no temperature-           Cells were centrifuged, washed in YP lacking sugar, and resus- <br /> sensitive recombinants were recovered after passage on FOA.            pended in fresh 30&#226;&#172;&#353; YPD. Samples were removed at 12-min <br /> One of the resulting sid2-21 temperature-sensitive strains,            intervals. <br /> LY1037, was transformed with SU3 and found to become tem-                 FACS analysis and cell counting: Flow cytometric DNA quanti- <br /> perature resistant, confirming that the temperature-sensitive          tation was performed as described elsewhere (Epstein and <br /> growth of LY1037 was due to sid2-21. Furthermore, when                 Cross 1992). Growth curve samples were fixed with 3 ml of <br /> LY1037 was crossed to MJ257 (SID2::LEU2), temperature sensi-           0.74% formaldehyde in 1&#226;&#171;&#187; PBS. The samples were sonicated <br /> tivity and Leu&#226;&#171;&#185; segregated in repulsion in all 12 tetrads ana-         for 12 sec and cell number was analyzed using a Coulter counter. <br /> lyzed.                                                                 Microscopic analysis was used to determine the percentage of <br />    Tagging Sid2 with protein A: A Sid2-protein A fusion protein        cells that were unbudded, small budded, or large budded. At <br /> was constructed by PCR amplification of the protein A gene             each time point, at least 200 cells were counted. Small-budded <br /> and an adjacent his5&#226;&#171;&#185; marker (the Schizosaccharomyces pombe            cells were those where the daughter bud was less than two-thirds <br /> homolog of the S. cerevisiae HIS3 gene) from pBXAHis5 (M.              the size of the mother bud. Large-budded cells had a daughter <br /> Rout, The Rockefeller University) using primers with homol-            bud greater than two-thirds the size of the mother bud. <br /> ogy to SID2 at their 5&#226;&#172;&#732; ends and to protein A or his5&#226;&#171;&#185; at their           Field inversion gel electrophoresis assay: Yeast strains 1036 <br /> 3&#226;&#172;&#732; ends. The pBXAHis5 plasmid was derived from pFA6a-                  (SID2) and 1037 (sid2-21) were grown to early log phase in YPD <br /> HIS3MX6 (Wach et al. 1997). The following oligonucleotides             at 23&#226;&#172;&#353;. Samples of the wild type and mutant were removed for <br />                                                       SID2 and S Phase Cyclin Activity                                              21 <br />  <br /> processing, the cultures were shifted to 37&#226;&#172;&#353;, and samples were         tion or at the exit from mitosis, we isolated mutations <br /> then removed at 2-hr intervals. For the hydroxyurea-treated con-       that caused synthetic lethality with sic1&#226;&#338;&#172;. sic1&#226;&#338;&#172; GAL1-SIC1 <br /> trol sample, hydroxyurea was added directly to an aliquot of the <br /> log phase culture of wild-type cells (final concentration, 0.2 m)      strains (LY623 and MJ65) were mutagenized and screened <br /> and incubation continued for 3 hr at 30&#226;&#172;&#353;. Chromosomal DNA              for mutants alive on YPGal (SIC1 expressed) and dead <br /> samples were prepared in agarose plugs as described (Schwartz          on YPD (SIC1 repressed). Approximately 28,000 colonies <br /> and Cantor 1984; Rose et al. 1990). Samples containing equiva-         were screened by replica plating and 21 recessive mutants <br /> lent OD600 units of cells were applied to a 1% agarose gel, electro-   were isolated. The mutants were named SID for SIC1 Indis- <br /> phoresed in 0.5&#226;&#171;&#187; Tris-borate-EDTA buffer at 4&#226;&#8364;&#8220;8&#226;&#172;&#353; at 8.3 V/cm, <br /> stained with ethidium bromide overnight, and then destained            pensable. <br /> for 1 hr. Field inversion was controlled by a PC500 SwitchBack            By complementation testing, the following groups were <br /> pulse controller (Hoefer, Amersham Pharmacia Biotech, Piscata-         established: sid1 (seven alleles), sid2 (one allele), sid3/dbf2 <br /> way, NJ) with a run time of 32 hr, a pulse time of 1&#226;&#8364;&#8220;50 sec and        (six alleles), and sid4/cdc15 (one allele). The remaining six <br /> an F/R ratio of 3.0:1. <br />                                                                        mutants do not fall into the four complementation groups <br />    Immunofluorescence staining: Immunofluorescence micros- <br /> copy was done essentially as described previously (Wente et al.        described above and define at least two more complemen- <br /> 1992). Tubulin was visualized using anti-tubulin antibody              tation groups. They remain unsorted because of difficul- <br /> (Wente et al. 1992; 1:200 dilution) followed by Cy-3 donkey            ties with backcrossing. There appears to have been sponta- <br /> conjugated anti-rat IgG (Jackson ImmunoResearch Laboratories,          neous diploidization of many of the mutants, resulting in <br /> West Grove, PA). The fluorescent DNA-specific dye 4&#226;&#172;&#732;,6-diamid- <br /> ino-2-phenylindole (DAPI) was used to visualize yeast nuclei.          minimal spore viability following tetrad dissection after <br />    Immunoprecipitation and detection of HA-tagged proteins:            crosses to wild-type haploid strains. Attempts to sporulate <br /> The immunoprecipitation protocol is based on methods de-               the diploidized strains after transformation with MAT-con- <br /> scribed previously (Levine et al. 1996). Yeast cultures (100 ml)       taining plasmids were unsuccessful. sid1 has yet to be <br /> were grown overnight to an optical density (600 nm) of 1.0.            cloned, though two LEU2 CEN libraries have been thor- <br /> Cells were collected and washed in TNN buffer (50 mm Tris pH <br /> 7.5, 250 mm NaCl, 5 mm EDTA, 10% glycerol, 0.1% Nonidet                oughly screened for rescue plasmids (39,000 transfor- <br /> P-40). TNN extraction buffer [TNN &#226;&#171;&#185; 5% aprotinin (Sigma),              mants). Preliminary analysis suggests that sid1 sic1&#226;&#338;&#172; strains <br /> 0.1 mm phenylmethylsulfonyl fluoride, 10 &#226;&#65533;&#174;g/ml leupeptin, 10           arrest as large-budded cells with segregated nuclei (data <br /> &#226;&#65533;&#174;g/ml pepstatin, 10 mm NaPPi (pH 7.4), 10 mm NaF] was used             not shown). sid3 sic1&#226;&#338;&#172; mutants failed to complement a <br /> for cell breaking, antibody incubation, and immunoprecipitation        dbf2&#226;&#338;&#172; sic1&#226;&#338;&#172; GAL1-SIC1 strain (LY699) on YPD and sid3 <br /> with the protein A-agarose slurry. The extract was incubated for <br /> 1 hr on ice with 1 &#226;&#65533;&#174;l of the monoclonal HA11 antibody (ascitis;        was meiotically linked to dbf2&#226;&#338;&#172; (MJX21-4D) in tetrad analy- <br /> Babco). Following incubation with the antibody, the extract was        sis. The sid4 sic1&#226;&#338;&#172; mutant failed to complement a cdc15-2 <br /> added to 30 &#226;&#65533;&#174;l of a protein A-agarose slurry (Sigma) prewashed         sic1&#226;&#338;&#172; strain (LY677). Linkage could not be established, how- <br /> with TNN and rotated at 4&#226;&#172;&#353; for 1 hr.                                   ever, as the sid4 mutant had spontaneously diploidized. <br />    SDS-polyacrylamide gel electrophoresis (10%) and transfer to <br /> Immobilon were done as previously described (Cross and Blake <br />                                                                        Although cdh1&#226;&#338;&#172; sic1&#226;&#338;&#172; spores are inviable (Schwab et <br /> 1993). Following the transfer, the immunoblots were blocked            al. 1997), no cdh1 mutants resulted from this screen as <br /> overnight [PBS with 0.5% Nonidet P-40, 0.1% Tween 20, 0.5%             determined by complementation testing to MJ249 and <br /> BSA, and 15% milk (Carnation)]. The blot was incubated with            MJ242. cdh1&#226;&#338;&#172; sic1&#226;&#338;&#172; spores were isolated in our strain <br /> antibodies (1.5 hr for the primary, 1 hr for the secondary) in         background using GAL1-SIC1 to suppress the lethal- <br /> PBS-Tween 20 (0.2%) with 1% milk at room temperature. The <br /> primary antibody, polyclonal rabbit anti-HA (Covance Research          ity. cdh1&#226;&#338;&#172; sic1&#226;&#338;&#172; spores that did not contain GAL1-SIC1 <br /> Products, Richmond, CA), was diluted 1:7500. The secondary             were inviable as previously reported. cdh1&#226;&#338;&#172; sic1&#226;&#338;&#172; GAL- <br /> antibody was a 1:1500 dilution of polyclonal donkey anti-rabbit        SIC1 strains were able to grow when replica plated to <br /> conjugated to horseradish peroxidase (Amersham). The samples           YP-dextrose, though they clearly grew more slowly than <br /> were washed three times for 10 min each with PBS-Tween 20              sic1&#226;&#338;&#172; GAL1-SIC1 strains (data not shown). Thus the very <br /> following each antibody incubation. The proteins were then <br /> detected using an enhanced chemiluminescence kit (Amer-                low level of expression from GAL1-SIC1 on glucose is prob- <br /> sham).                                                                 ably sufficient to partially rescue cdh1&#226;&#338;&#172; sic1&#226;&#338;&#172; GAL-SIC1 <br />    Other Western blots were processed as described above except        strains on glucose medium, which may account for our <br /> for the following modifications. The blots were blocked for 1 hr       failure to isolate cdh1 mutations in our screen. The isolated <br /> at room temperature in PBS-Tween 20 with 2.5% milk and incu- <br />                                                                        sid mutants were all found to complement cdc5 and cdc14 <br /> bated with the antibodies in PBS-Tween 20 containing 5% milk. <br /> A 1:1000 dilution of rabbit anti-mouse IgG (Organon Teknika,           strains. <br /> Durham, NC) was used to detect protein A tags. Mouse mono-                SID2 is YJR046w, a novel gene: To clone SID2, a LEU2 <br /> clonal anti-Nop1 (Aris and Blobel 1988) was diluted 1:2000             CEN4 library was screened for plasmids that could rescue <br /> and mouse monoclonal anti-PGK (Molecular Probes, Eugene,               the lethality of sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 cells on YPD. Four <br /> OR) was diluted 1:10,000. The secondary antibody was a 1:1000 <br /> dilution of polyclonal donkey anti-rabbit (sheep anti-mouse for <br />                                                                        plasmids containing the yeast ORF YJR046w/TAH11 and <br /> Nop1 and PGK) conjugated to horseradish peroxidase (Amer-              flanking regions were isolated after screening 8000 trans- <br /> sham).                                                                 formants. An insert containing only YJR046w intact was <br />                                                                        subcloned into RS415 and the resulting plasmid (pMJ02) <br />                                                                        was able to rescue the lethality of sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 <br />                             RESULTS <br />                                                                        cells on YPD (Figure 1A). LEU2 was integrated adjacent to <br />    Isolation of mutants synthetically lethal with sic1 dele-           YJR046w in a sid2-1 strain and was found to be meiotically <br /> tion: To identify factors that assist SIC1 at the G1/S transi-         linked to SID2 (materials and methods). SID2 encodes <br />  22                                                       M. D. Jacobson et al. <br />  <br />                                                                       GAL1-SIC1. Therefore, overexpression of SIC1 does not <br />                                                                       suppress the lethality caused by deletion of SID2. Most of <br />                                                                       the further analysis of SID2 was performed using sid2-1 <br />                                                                       strains. The sid2-1 mutation was backcrossed from the <br />                                                                       original mutagenized strain into the sic1&#226;&#338;&#172; GAL1-SIC1 back- <br />                                                                       ground at least four times before further analysis. <br />                                                                          SID2 on a centromere-based plasmid completely res- <br />                                                                       cued the lack of growth of the sid2-1 sic1&#226;&#338;&#172; cells (Figure 1A). <br />                                                                       Similarly, GAL1-SIC1 overexpression permitted growth of <br />                                                                       sid2-1 sic1&#226;&#338;&#172; strains on YPGal that was comparable to wild <br />                                                                       type (Figure 1A). To demonstrate that growth of sid2-1 <br />                                                                       sic1&#226;&#338;&#172; strains on YPGal was dependent on the presence of <br />                                                                       GAL-SIC1, we dissected diploids heterozygous for sid2-1, <br />                                                                       sic1&#226;&#338;&#172;, and GAL-SIC1 on YPGal. SID2::LEU2 was also segre- <br />                                                                       gating in the cross, allowing unambiguous determination <br />                                                                       of sid2-1 spores. Dissection of 65 tetrads on YPGal gave 23 <br />                                                                       spores that could unequivocally be predicted to be sid2-1 <br />                                                                       sic1&#226;&#338;&#172; and lacking GAL-SIC1; all were dead. In contrast, <br />                                                                       the viability of SID2 sic1&#226;&#338;&#172; cells lacking GAL-SIC1 was 86% <br />                                                                       (n &#226;&#171;&#189; 29). <br />                                                                          The screen that isolated sid2-1 identified mutants that <br />                                                                       were viable in the presence of high levels of SIC1 and <br />                                                                       inviable in the absence of SIC1. To determine whether <br />                                                                       sid2-1 mutant cells required the high levels of SIC1 ex- <br />                                                                       pressed from the GAL1 promoter throughout the cell <br />                                                                       cycle, we assayed the growth of sid2-1 cells in the presence <br />    Figure 1.&#226;&#8364;&#8221;(A) A CEN-based plasmid carrying YJR046w <br /> (pMJ02) rescues sid2-1 sic1 lethality. Strain MJ193 (sid2-1 sic1&#226;&#338;&#172;     of lower levels of SIC1. In contrast to the complete suppres- <br /> GAL1-SIC1) was transformed with the YJR046w-containing plas-          sion by GAL-SIC1, sid2-1 sic1&#226;&#338;&#172; strains were only partially <br /> mid pMJ02, the RS415 vector, and a RS415 CEN-based plasmid            rescued by SIC1 on a centromere-based plasmid when <br /> containing SIC1 under the control of its own promoter. MJ58           SIC1 was expressed under the control of its own promoter <br /> (sic1&#226;&#338;&#172; GAL1-SIC1) was transformed with RS415. The strains were <br />                                                                       (Figure 1A). However, a more complete rescue of sid2-1 <br /> grown overnight to stationary phase in SCGal-Leu and 5 &#226;&#65533;&#174;l <br /> of 10-fold serial dilutions were plated on YPGal and YPD and          strains by endogenous SIC1 was observed in backcrosses <br /> incubated for 3&#226;&#8364;&#8220;4 days at 30&#226;&#172;&#353;. The SIC1 plasmid-containing sid2-1     where the wild-type SIC1 gene was segregating against <br /> sic1&#226;&#338;&#172; strain shows a slight rescue compared to vector but did         sic1&#226;&#338;&#172;. When diploids heterozygous for sid2-1 and sic1&#226;&#338;&#172; <br /> not grow nearly as well as the YJR046w-containing sid2-1 sic1&#226;&#338;&#172;        were sporulated and tetrads were dissected, colonies that <br /> strain on YPD. All transformants grew equally well on YPGal <br /> where GAL1-SIC1 was expressed. (B) sid2-1 SIC1 and SID2 SIC1          were sid2-1 SIC1 were similar in size to those that were <br /> strains have similar plating efficiencies. Strains 914 (SID2 SIC1)    SID2 SIC1 (data not shown). When sid2-1 SIC1 spores were <br /> and LY1118 (sid2-1 SIC1) were grown overnight to stationary           analyzed, they gave plating efficiencies more similar to <br /> phase in YPD and plated as described above. (C) The cloned            wild-type SID2 SIC1 strains (Figure 1B), although a slight <br /> region contained in pMJ01 (RS405) and pMJ02 (RS415) that <br />                                                                       decrease in both the number of colony forming units and <br /> rescues sid2-1 and the SID2 disruption construct (pMJ07) where <br /> a Kanr/LEU2 fragment replaced &#226;&#181;&#8216;1 kb of YJR046w coding se-             colony size could still be observed in the sid2-1 SIC1 strain. <br /> quence.                                                               The observed difference between the ability of plasmid <br />                                                                       and chromosomal SIC1 to rescue may be due to a lack of <br />                                                                       regulatory regions in the SIC1-containing plasmid. The <br /> a 604-amino-acid protein lacking significant homology to              suppression of sid2-1 by wild-type levels of SIC1 demon- <br /> any known genes. Since SID2 was known to be essential                 strates that overexpression of SIC1 is not absolutely re- <br /> in wild-type cells (Huang et al. 1997), a sid2&#226;&#338;&#172; GAL1-SIC1             quired for the viability of sid2-1 mutant cells. Although <br /> strain was constructed to determine if the sid2 null muta-            the sid2-1 SIC1 mutant cells appear to grow fairly similarly <br /> tion could be rescued by elevated Sic1 levels. SID2 was               to wild type, they do have an increase in the number of <br /> deleted in a diploid heterozygous for GAL1-SIC1 and sic1&#226;&#338;&#172;             cells with a 2C DNA content and large buds compared to <br /> using a construct that removed 1 kb internal to SID2 and              SID2 SIC1 cells (Figure 7, E and F). Based on the other <br /> inserted a 3.5-kb fragment containing LEU2-Kanr (Figure               phenotypes of sid2 mutants, it is likely that this phenotype <br /> 1C). The diploid was sporulated and the tetrads were                  is due to a defect in DNA replication or induction of a <br /> dissected on galactose-containing media. All viable spores            DNA damage checkpoint. It is less likely that the sid2 <br /> were Leu&#226;&#171;&#186;; the spores predicted to contain sid2::LEU2                 mutant cells replicate their DNA prematurely compared <br /> were inviable even when they were predicted to contain                to wild type because sid2 mutant cells are actually delayed <br />                                                   SID2 and S Phase Cyclin Activity                                                 23 <br />  <br /> in DNA replication compared to wild-type cells (see <br /> below). <br />    Sid2p, while predominantly cytoplasmic, has a func- <br /> tional nuclear import signal: Determining the timing of <br /> Sid2p expression or its intracellular localization might aid <br /> our understanding of how and when Sid2p functions. To <br /> detect Sid2p, we tagged the genomic locus of SID2 with <br /> protein A and detected it by Western blot. SID2-PrA strains <br /> do not show a growth defect and have a wild-type FACS <br /> profile (data not shown). In addition, SID2-PrA sic1&#226;&#338;&#172; <br /> strains are viable. This indicates that the protein A tag <br /> does not interfere with Sid2p function. A strain containing <br /> the tagged SID2 gene was arrested with &#226;&#65533;&#163;-factor and re- <br /> leased into YPD. Protein extracts were made at 12-min <br /> intervals following the &#226;&#65533;&#163;-factor release and Sid2p levels <br /> were found to remain constant throughout the cell cycle <br /> (data not shown). <br />    We could not detect Sid2-PrA by immunofluorescent <br /> staining of either logarithmically growing or hydroxyurea <br /> (HU)-arrested cells, perhaps because the level of expres- <br /> sion is low or the localization is diffuse. Following fraction- <br /> ation of the Sid2-protein A-containing strain, Sid2p pro- <br /> tein was visualized by Western blot and found to be <br /> predominately cytoplasmic, although a small amount of <br /> nuclear Sid2p could be detected (Figure 2A). Interest- <br /> ingly, analysis of Sid2p&#226;&#8364;&#8482;s localization using a one-hybrid <br /> assay (Rhee et al. 2000) suggests that Sid2p has a functional         Figure 2.&#226;&#8364;&#8221;Sid2p is predominantly cytoplasmic, but has a <br /> nuclear localization signal (NLS; Figure 2B). A fusion             functional nuclear localization signal. (A) Strain LY925, in which <br /> protein containing a modified lexA DNA-binding domain,             SID2 was genomically tagged at its carboxyl terminus with protein <br /> the GAL4 transcriptional activation domain, and Sid2p              A, was fractionated using a sucrose gradient. Cytosolic, surface, <br />                                                                    surface to 2.01 m sucrose, 2.01&#226;&#8364;&#8220;2.10 m sucrose, 2.10&#226;&#8364;&#8220;2.30 m <br /> (NIASID2) could enter the nucleus and activate transcrip-          sucrose, and pellet fractions were collected and appropriate <br /> tion from a lexA-operator-driven lacZ reporter gene. In            amounts were loaded to adjust for varying collection volumes. <br /> contrast, a control fusion containing the cytoplasmic pro-         The fractions were examined by Western blot probing for Sid2- <br /> tein VirE2, which is of similar size to Sid2p, could not           PrA, Nop1 (a nuclear control), and PGK (3-phosphoglycerate <br /> activate transcription from the reporter gene (NIAE2, Fig-         kinase, a cytoplasmic control). (B) &#226;&#65533;&#164;-Galactosidase assays were <br />                                                                    performed on strain L40, which contains lacZ under the control <br /> ure 2B; Rhee et al. 2000). Levels of transcriptional activa-       of the lexA operator, after transformation with plasmids encod- <br /> tion from the Sid2p-containing construct were lower than           ing fusion proteins. All the plasmids contain a modified lexA <br /> that from a control construct containing the NLS from              DNA-binding domain and the GAL4 transcriptional activation <br /> SV40 T antigen (NEAE2). Taken together with the frac-              domain. Control plasmid NIA contains the lexA-GAL4 fusion, <br />                                                                    while the remaining plasmids contain fusions between that se- <br /> tionation experiments, these data suggest that Sid2p has           quence and other coding regions. Plasmid NIAE2 carries the <br /> a functional NLS, although it may be weak or subject to            sequence encoding the cytoplasmic VirE2 protein from Agrobac- <br /> regulation.                                                        terium tumefaciens, plasmid NEAE2 carries the sequence encoding <br />    sid2-1 sic1&#226;&#338;&#172; and sid2-21 cells have a preanaphase arrest:       the SV40 large T Ag NLS and VirE2, and NIASID2 carries the <br /> To characterize the effect of the sid2-1 mutation, we ana-         entire SID2 coding sequence. Error bars represent the standard <br />                                                                    error of two experiments; the data for NIASID2 is the average of <br /> lyzed the morphology of sid2-1 and SID2 cells (both sic1&#226;&#338;&#172;          the results of two experiments performed with six independently <br /> GAL1-SIC1) in the absence of SIC1 expression by shifting           isolated NIASID2 plasmids. Activity levels and standard error for <br /> cultures from YPGal (SIC1 expressed) to YPD (SIC1 re-              the NIA and NIAE2 transformants were 0.43 &#226;&#171;&#190; 0.04 units and <br /> pressed). sid2-1 sic1&#226;&#338;&#172; cells slowed proliferation and then         0.31 &#226;&#171;&#190; 0.03 units, respectively. <br /> arrested 6 hr after repression of GAL1-SIC1, showing only <br /> minimal increases in cell number at later time points              cells in the SID2 strain but otherwise had little effect on <br /> (Figure 3A). There was no decrease in viability up to 5 hr         the distribution of cell morphologies (Figure 3B). <br /> after the shift to dextrose-containing media and a slight             Similarly, we compared the morphology of wild-type <br /> decrease in viability (less than fourfold) by 10 hr after the      SID2 cells and cells containing a temperature-sensitive al- <br /> shift (data not shown). sid2-1 sic1&#226;&#338;&#172; cells accumulated with        lele, sid2-21, after incubation at the nonpermissive temper- <br /> large buds at the arrest point (Figure 3, B and D). Repres-        ature. sid2-21 cells began to die by 3 hr after shift to 37&#226;&#172;&#353; <br /> sion of SIC1 slightly decreased the number of unbudded             (Figure 4A). After 4 hr of incubation at 37&#226;&#172;&#353;, the sid2-21 <br />  24                                                   M. D. Jacobson et al. <br />  <br />  <br />  <br />  <br />                                                                                                 Figure 3.&#226;&#8364;&#8221;sid2-1 sic1&#226;&#338;&#172; cells ar- <br />                                                                                              rest preanaphase as large-budded <br />                                                                                              cells with a 2C DNA content. Strains <br />                                                                                              MJ58 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1) and <br />                                                                                              MJ193 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1) <br />                                                                                              were grown in YPGal overnight to <br />                                                                                              log phase and the cultures were <br />                                                                                              split. Dextrose was added (2%) to <br />                                                                                              half the cultures at time 0 to repress <br />                                                                                              SIC1 expression from the GAL1 pro- <br />                                                                                              moter. Cell number (A), budding <br />                                                                                              distribution (B), DNA content by <br />                                                                                              FACS analysis (C), and nuclear and <br />                                                                                              spindle morphology (D) were ex- <br />                                                                                              amined. In B and D, the cells were <br />                                                                                              fixed 6 hr after addition of dex- <br />                                                                                              trose. <br />  <br />  <br />  <br />  <br /> cells accumulated with large buds similar to the sid2-1           replicate at least most of their DNA in the absence of SIC1 <br /> sic1&#226;&#338;&#172;-arrested cells (Figure 4, B and D). The distribution        expression, they may have some defect associated with <br /> of cell morphologies of SID2 cells, in contrast, was unaf-        DNA synthesis. <br /> fected by the temperature shift (Figure 4B).                         The DNA contents of the SID2 and sid2-21 populations <br />    The arrest of sid2-1 sic1&#226;&#338;&#172; and sid2-21 cells was further       of cells were similar at 25&#226;&#172;&#353;, with cells approximately equally <br /> examined using FACS to analyze DNA content. By 6 hr               distributed between two distinct peaks at 1C and 2C (Fig- <br /> after repression of GAL1-SIC1, the DNA content of SID2            ure 4C). For the SID2 cells, this profile remained constant <br /> sic1&#226;&#338;&#172; cells was predominately 2C, though a small 1C peak          after the temperature shift to 37&#226;&#172;&#353;. In contrast, 2 hr after <br /> was also observed (Figure 3C). Cells deleted for sic1 have        the shift to the nonpermissive temperature, the sid2-21 <br /> a short G1 phase probably due to the lack of S phase              cells accumulated with a DNA content intermediate be- <br /> cyclin/CDK (Clb5/Clb6p-Cdc28p) inhibition (Schwob et              tween the 1C and 2C peaks. By 4 hr at the nonpermissive <br /> al. 1994). The DNA content of the mutant sid2-1 sic1&#226;&#338;&#172;             temperature, the DNA content of the sid2-21 cells shifted <br /> cells also shifted primarily to an &#226;&#181;&#8216;2C peak. However, in          to a broad, &#226;&#181;&#8216;2C peak (Figure 4C). These data indicate <br /> contrast to the SID2 sic1&#226;&#338;&#172; cells, the sid2-1 sic1&#226;&#338;&#172; population     that, like the sid2-1 sic1&#226;&#338;&#172; cells, the sid2-21 mutant cells <br /> at 6 hr after shift to YPD had a less distinct 1C peak, and       appear to replicate most or all of their DNA under nonper- <br /> the 2C peak was much broader, having a shoulder of cells          missive conditions. However, the accumulation of cells in <br /> with DNA content between 1C and 2C (Figure 3C). These             S phase at 2 hr suggests that sid2 mutants may have a <br /> data indicate that, although the mutant cells were able to        defect in DNA replication (see also below). <br />                                                 SID2 and S Phase Cyclin Activity                                               25 <br />  <br />  <br />  <br />  <br />                                                                                                Figure 4.&#226;&#8364;&#8221;sid2-21 cells arrest as <br />                                                                                             large-budded cells with a 2C DNA <br />                                                                                             content, a single nucleus, and a <br />                                                                                             short spindle. Strains 1037 (sid2-21) <br />                                                                                             and 914 (SID2) were grown over- <br />                                                                                             night to log phase in YPD at 25&#226;&#172;&#353; and <br />                                                                                             then shifted to 37&#226;&#172;&#353;. Colony-forming <br />                                                                                             units (A), budding distribution (B), <br />                                                                                             DNA content by FACS analysis (C), <br />                                                                                             and nuclear and spindle morphol- <br />                                                                                             ogy (D) were examined. In B and <br />                                                                                             D, the cells were fixed 4 hr after <br />                                                                                             shift to 37&#226;&#172;&#353;. <br />  <br />  <br />  <br />  <br />    To characterize the defect in the sid2 mutants more           arrest primarily in telophase with elongated spindles and <br /> completely, sid2-1 sic1&#226;&#338;&#172;, SID2 sic1&#226;&#338;&#172;, sid2-21, and SID2 cells    DNA segregated between the mother and bud (Surana <br /> were stained with DAPI and tubulin was visualized by indi-       et al. 1993). The arrest phenotype demonstrated by the <br /> rect immunofluorescence. In contrast to the SID2 sic1&#226;&#338;&#172;           sid2-1 sic1&#226;&#338;&#172; and sid2-21 cells has some similarities to pheno- <br /> and SID2 cells that were at various cell cycle stages, the       types demonstrated by mutants that affect DNA replication <br /> sid2-1 sic1&#226;&#338;&#172; and sid2-21-arrested cells appeared to be pre-      or APC activation. Of these, the slow S phase observed in <br /> anaphase with a single nucleus and a short spindle (Figures      the sid2-21 mutant strains is most consistent with a defect <br /> 3D and 4D). The arrest morphology of sid2 sic1&#226;&#338;&#172; and sid2-        in DNA replication. <br /> 21 cells suggests that, unlike other mutations that are             Both S and M phase B-type cyclins are only slightly <br /> lethal in combination with sic1&#226;&#338;&#172;, the primary defect of          stabilized in sid2-1 mutants: One possibility, based on the <br /> the sid2 mutants is not in the reduction of Clb2p-CDK            morphology of sid2-1 sic1&#226;&#338;&#172; cells, was that sid2-1 affected <br /> activity. Mutants that fail to inactivate Clb2p-CDK activity     APC activity. It has previously been demonstrated that a <br />  26                                                     M. D. Jacobson et al. <br />  <br />  <br />  <br />  <br />   Figure 5.&#226;&#8364;&#8221;Clb5p and Clb2p are only slightly stabilized in sid2-1 &#226;&#65533;&#163;-factor-arrested cells. Strains (A) MJ316 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 <br /> CLB5 HA), MJ317 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 CLB5 HA), and MJ319 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 CLB5&#226;&#338;&#172;DB HA) and (B) MJ288 (SID2 sic1&#226;&#338;&#172; GAL1- <br /> SIC1 CLB2HA), MJ282 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 CLB2HA), and MJ292 (cdh1&#226;&#338;&#172;/hct1&#226;&#338;&#172; sic1&#226;&#338;&#172; GAL1-SIC1 CLB2HA) were grown overnight to <br /> log phase and arrested with 0.1 &#226;&#65533;&#174;m &#226;&#65533;&#163;-factor for 3 hr. (A) Clb5p levels were examined at 1-hr intervals after the addition of &#226;&#65533;&#163;-factor <br /> and (B) Clb2p levels were examined before and after the arrest. The samples were processed by immunoprecipitation followed by <br /> immunoblotting. <br />  <br />  <br />  <br /> number of mutants that affect APC activity are lethal in             HAp was present at a slightly higher level in sid2-1 &#226;&#65533;&#163;-factor- <br /> combination with deletion of sic1 (Toyn et al. 1997;                 arrested cells than in SID2 &#226;&#65533;&#163;-factor-arrested cells (Figure <br /> Schwab et al. 1997; Kramer et al. 1998). Mutants that fail           5B). Cdh1p/Hct1p targets Clb2p for degradation by the <br /> to activate the APC have defects in the degradation of               APC in late mitosis, and G1 cells deleted for CDH1/HCT1 <br /> Clb2p (Irniger et al. 1995; Zachariae et al. 1996; Kramer            show greatly increased levels of Clb2p compared to wild <br /> et al. 1998). In addition, Clb5p degradation appears, at             type (Schwab et al. 1997; see also Figure 5B). Clb2-HAp <br /> least in part, also to be regulated by the APC (Irniger              was not stabilized in sid2-1 cells to the degree that it was <br /> and Nasmyth 1997; Shirayama et al. 1999). Failure to                 in cdh1&#226;&#338;&#172;/hct1&#226;&#338;&#172; cells. Taken together, these results suggest <br /> degrade the B-type cyclins (caused by a mutation in an               that while SID2 may have some role in decreasing Clb5p <br /> APC component), coupled with lack of inhibition of the               and Clb2p-CDK kinase activity, it is most likely very minor <br /> CDK kinase (caused by deletion of sic1), appears to result           as Clb stability is not affected to the degree that it is by <br /> in levels of CDK kinase that are too high to allow progres-          removing either the destruction box (Clb5p) or CDH1/ <br /> sion through mitosis. We therefore analyzed Clb5p and                HCT1 (Clb2p). This suggests that Sid2p is not a compo- <br /> Clb2p stability in sid2-1 sic1&#226;&#338;&#172; cells to determine whether           nent of the APC and that the arrest of sid2-1 sic1&#226;&#338;&#172; cells <br /> increased B-type cyclin levels could contribute to the ob-           is not likely to be due primarily to a defect in APC activity <br /> served phenotypes.                                                   (see discussion). <br />    An asynchronous culture of a genomically tagged                      sid2-1 interacts genetically with S phase cyclins: As sid2-1 <br /> CLB5HA strain was treated with &#226;&#65533;&#163;-factor, which blocks CLB5           sic1&#226;&#338;&#172; cells had a preanaphase arrest and sid2-1 did not <br /> expression (Epstein 1992). Clb5p turnover was examined               appear to affect APC function, we thought it likely that <br /> by following protein levels during the arrest. A strain in           sid2-1 was causing a defect in DNA replication. We there- <br /> which Clb5-HAp&#226;&#8364;&#8482;s destruction box was removed was used                fore analyzed the effects of deleting the S phase cyclins, <br /> as a positive control (CLB5&#226;&#338;&#172;DBHA). The levels of Clb5-               CLB5 and CLB6, on the growth of sid2-1 sic1&#226;&#338;&#172; cells. If <br /> HAp resulting from the deletion of the destruction box               Sic1p rescues sid2-1 by inhibiting the kinase activity of S <br /> do not affect the viability of cells even in the absence of          phase cyclin-CDK complexes, then deleting these genes <br /> sic1 (Figure 6). The percentage of unbudded cells at each            should mimic Sic1p expression. A deletion of CLB5 par- <br /> hourly time point was comparable for all three strains,              tially suppressed the growth defect of sid2-1 sic1 cells, and <br /> indicating that they arrested with similar kinetics (data not        the clb5 clb6 double deletion almost completely rescued <br /> shown). Clb5-HAp levels in the sid2-1 strain were slightly           sid2-1 sic1 cells (Figure 6, top). Deleting CLB6 alone did <br /> higher than in the SID2 strain, though Clb5-HAp was not              not have a detectable effect (data not shown). Since the <br /> stabilized to the degree that it was upon removal of the             removal of these S phase activators rescued the arrest <br /> destruction box (Figure 5A), suggesting that this defect             caused by sid2-1 in sic1 cells, we hypothesized that increas- <br /> is not sufficient to explain the arrest of sid2-1 cells.             ing S phase cyclin levels would have the opposite effect. <br />    To assay the stability of Clb2p in sid2-1 cells, CLB2HA           The destruction box of the more potent of these two <br /> cells were arrested with &#226;&#65533;&#163;-factor as described above. In             cyclins, CLB5, was removed (Cross et al. 1999) and sid2-1 <br /> wild-type strains, Clb2p is degraded at this G1 arrest point         sic1&#226;&#338;&#172; GAL1-SIC1 CLB5&#226;&#338;&#172;DBHA strains were constructed. <br /> (Amon et al. 1994). While the sid2-1 cells did not uniformly         The CLB5&#226;&#338;&#172;DBHA construct results in partial stabilization <br /> arrest at 1C as determined by FACS analysis, the profile             of Clb5-HAp (Figure 5A). sid2-1 cells that contained <br /> was comparable to the SID2 strain (data not shown). Clb2-            CLB5&#226;&#338;&#172;DBHA showed at least a 10-fold decrease in plating <br />                                                    SID2 and S Phase Cyclin Activity                                               27 <br />  <br />                                                                      point between 1C and 2C and are delayed in reaching a <br />                                                                      completed 2C state (Figure 7). Consistent with the profiles <br />                                                                      for the sid2-1 SIC1 strain, the replication delay for sid2-1 <br />                                                                      sic1&#226;&#338;&#172; GAL1-SIC1 cells is also observed when strains are <br />                                                                      released into YPGal where GAL1-SIC1 is expressed (data <br />                                                                      not shown). It may be that errors or DNA damage occur <br />                                                                      during DNA replication due to the sid2-1 mutation, since <br />                                                                      damage slows the rate of S phase progression due to a <br />                                                                      Mec1- and Rad53-dependent checkpoint (Paulovich and <br />                                                                      Hartwell 1995), and sufficient DNA damage can cause <br />                                                                      arrest with a nearly 2C DNA content. <br />                                                                         We were interested in determining whether replication <br />                                                                      was completed in sid2 strains and used field inversion gel <br />                                                                      electrophoresis to probe the structure of chromosomes <br />    Figure 6.&#226;&#8364;&#8221;sid2-1 interacts genetically with S phase cyclins.      in sid2 mutant strains. Chromosomes isolated from cells <br /> (Top) Strains LY986 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1), LY987 (sid2-1 sic1&#226;&#338;&#172;      blocked in replication fail to band properly on similar gel <br /> GAL1-SIC1), LY989 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 clb5&#226;&#338;&#172;), LY991 (sid2-1 <br /> sic1&#226;&#338;&#172; GAL1-SIC1 clb5&#226;&#338;&#172;), LY985 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 clb5&#226;&#338;&#172;            systems (Hennessy et al. 1991). This is most likely due to <br /> clb6&#226;&#338;&#172;), and LY988 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC clb5&#226;&#338;&#172; clb6&#226;&#338;&#172;) were grown     the presence of replication forks and bubbles, which make <br /> overnight to stationary phase in YPGal and 5 &#226;&#65533;&#174;l of 10-fold serial    the chromosomes heterogeneous in size and alter their <br /> dilutions were plated on YPGal and YPD and incubated for 3&#226;&#8364;&#8220;4         migration properties. As expected, DNA isolated from <br /> days at 30&#226;&#172;&#353;. (Bottom) Strains MJ322 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1),        cells blocked in S phase by treatment with hydroxyurea <br /> MJ321 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 CLB5&#226;&#338;&#172;DBHA), MJ323 (SID2 sic1&#226;&#338;&#172; <br /> GAL1-SIC1), and MJ319 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 CLB5&#226;&#338;&#172;DBHA)               failed to band (Figure 8, lane 9), while DNA from a wild- <br /> were grown and plated as described for the top.                      type strain in log phase demonstrated a characteristic chro- <br />                                                                      mosome banding <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Figure 8, lanes 1&#226;&#8364;&#8220;4). Under <br />                                                                      permissive conditions, DNA isolated from a sid2-21 mutant <br /> efficiency compared to strains with wild-type Clb5 levels            strain migrated similarly to wild type (lane 5). In contrast, <br /> when grown in the absence of SIC1 (Figure 6, bottom).                under nonpermissive conditions, the DNA isolated from <br /> The CLB5&#226;&#338;&#172;DBHA construct did not appear to have an effect             the sid2-21 mutant showed much less banding than the <br /> on SID2 sic1&#226;&#338;&#172; cells. Taken together, these interactions              wild-type strain (Figure 8, lanes 6&#226;&#8364;&#8220;8). When taken together <br /> suggest that Sid2p&#226;&#8364;&#8482;s major function is related to DNA                with the FACS analysis, these data suggest that, although <br /> replication, since the main (but not the only) biological            the sid2-21 mutant cells may replicate most of their DNA, <br /> function of CLB5 and CLB6 is to trigger replication                  they still have some replication forks or bubbles present <br /> (Schwob and Nasmyth 1993; Segal et al. 1998).                        at the time of arrest. In some experiments, there was <br />    sid2 mutants may accumulate DNA damage during repli-              slighly more banding in the mutant at 37&#226;&#172;&#353; than in the <br /> cation:sic1&#226;&#338;&#172; cells exhibit accelerated entry into DNA repli-         hydroxyurea-treated sample. It is likely that this is either <br /> cation presumably resulting from premature Clb5p- and                the result of a less complete arrest or the presence of <br /> Clb6p-associated Cdc28 kinase activity (Schwob et al.                fewer forks and bubbles in the sid2-21-arrested cells. <br /> 1994). It is possible, therefore, that sid2 sic1&#226;&#338;&#172; synthetic             If DNA replication is slow and fails to be completed in <br /> lethality could be the result of DNA damage or synthesis             the sid2 mutant strains because of the accumulation of <br /> defects occurring because of unregulated replication. This           damage, it is possible that sid2-1 mutants would be sensitive <br /> would be consistent with the preanaphase arrest observed             to DNA damaging agents or compounds that affect DNA <br /> for the sid2-1 sic1 mutants, since DNA damage results in             replication, since then damage would be occurring for <br /> a checkpoint-dependent preanaphase arrest. To analyze                two independent reasons. sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 strains <br /> the progression of the mutant cells in S phase, we used              were &#226;&#181;&#8216;100-fold more sensitive to HU treatment than SID2 <br /> &#226;&#65533;&#163;-factor to synchronize the cells in G1 and monitored                sic1&#226;&#338;&#172; GAL1-SIC1 strains (Figure 9A). A similar effect of <br /> DNA synthesis by FACS analysis. SID2, sid2-1, SID2 sic1&#226;&#338;&#172;             HU was found when sid2-1 SIC1 strains were grown on <br /> GAL1-SIC1, and sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 strains were re-               YPD, where GAL1-SIC1 was repressed (data not shown). <br /> leased from the &#226;&#65533;&#163;-factor arrest into YPD. All cells began            Following UV treatment, a less dramatic decrease in plat- <br /> budding and replicating their DNA at approximately the               ing efficiency, of &#226;&#181;&#8216;5- to 10-fold, was found when compar- <br /> same time, by about 40 min after the &#226;&#65533;&#163;-factor release                ing sid2-1 to SID2 strains (data not shown). In contrast, <br /> (Figure 7). Due to the speed and partial asynchrony of              neither sid2-1 nor sid2-21 strains showed any sensitivity to <br /> DNA replication, the SID2 strains do not accumulate de-              the microtubule depolymerizing drug, benomyl, when as- <br /> tectably at an intermediate stage between 1C and 2C while           sayed at a range of drug concentrations (data not shown). <br /> replicating their DNA. Only a decrease in the 1C peak                   It may be that the sid2-1 mutation is directly or indirectly <br /> and a commensurate increase in the 2C peak could be                  (by affecting DNA repair) causing DNA damage, which <br /> observed. At the same time intervals, however, sid2-1 cells,         is then exacerbated by UV or HU to a point of decreased <br /> regardless of their SIC1 genotype, do accumulate at a                colony formation even with GAL1-SIC1 expression. A de- <br />  28   M. D. Jacobson et al. <br />                                                    SID2 and S Phase Cyclin Activity                                                 29 <br />  <br />  <br />  <br />  <br />                                                                        Figure 9.&#226;&#8364;&#8221;(A) sid2-1 cells are sensitive to hydroxyurea. Strain <br />                                                                     MJ163 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1) was transformed with either a <br />                                                                     plasmid containing SID2 (pMJ02) or vector (RS415). The strains <br />                                                                     were grown overnight to stationary phase in SCGal-Leu. Strains <br />                                                                     LY909 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1) and LY907 (sid2-1 sic1&#226;&#338;&#172; GAL1- <br />                                                                     SIC1) were grown overnight to stationary phase in YPGal. Five <br />                                                                     microliters of 10-fold serial dilutions were plated on YPGal and <br />                                                                     YPGal with 200 mm hydroxyurea. (B) sid2-1 rad9 cells are slow <br />                                                                     growing or dead. Spores from a diploid strain formed by crossing <br />   Figure 8.&#226;&#8364;&#8221;Chromosomes isolated from sid2-21 strains show          KK1 (MAT&#226;&#65533;&#163; sic1&#226;&#338;&#172; GAL1-SIC1 rad9&#226;&#338;&#172;) and MJ163 (MATa sid2-1 <br /> decreased banding on an inverted field gel. Strains 1036 (SID2)     sic1&#226;&#338;&#172; GAL1-SIC1) were dissected on YPGal and incubated at 30&#226;&#172;&#353; <br /> and 1037 (sid2-21) were grown to early log phase in YPD. Aliquots   for 3&#226;&#8364;&#8220;4 days. The observed or predicted genotype of each spore <br /> of the cultures were removed, the cultures shifted to 37&#226;&#172;&#353;, and      colony is noted below the tetrad plate. <br /> incubation continued for 6 hr. The DNA replication inhibitor <br /> HU was added to an aliquot of strain 1036 and incubated for <br /> 3 hr at 30&#226;&#172;&#353;. Chromosomal DNA was isolated from each sample,         were either dead or extremely slow growing, even when <br /> separated on a field inversion gel, and stained with ethidium       GAL1-SIC1 was expressed (Figure 9B and Table 3). Fur- <br /> bromide. Numbers represent the hours the sample was incu- <br /> bated at 37&#226;&#172;&#353; before chromosome isolation.                           thermore, analysis of the sid2-1 rad9&#226;&#338;&#172; cells showed that <br />                                                                     the preanaphase Cdc&#226;&#171;&#186; arrest of sid2 sic1&#226;&#338;&#172; cells depends on <br />                                                                     RAD9 (Table 4). The demonstration that RAD9 is required <br /> fect of sid2-1 cells in a checkpoint pathway could also cause       both for the full viability of sid2-1 cells and for the cell <br /> HU and UV sensitivity, but this is less likely because of           cycle arrest of sid2 sic1&#226;&#338;&#172; is consistent with the hypothesis <br /> the delay observed during DNA replication in sid2-1 strains         that sid2-1 results in defects in DNA replication or repair. <br /> and the cell cycle arrest phenotype observed with the sid2-1 <br /> sic1&#226;&#338;&#172; strains. <br />                                                                                              DISCUSSION <br />    To determine whether sid2 mutant cells were accumu- <br /> lating DNA damage and to test the hypothesis that sid2-1               Identification of genes synthetically lethal with sic1&#226;&#338;&#172;: <br /> sic1 cells arrest because of defects in DNA replication or          SIC1 encodes a nonessential B-type cyclin/CDK inhibitor <br /> repair, we analyzed sid2 rad9 cells. RAD9 is required for           that functions at both the G1/S transition and the exit <br /> the G2 cell cycle arrest caused by DNA damage or in-                from mitosis. Sic1p decreases Clb5/6p-associated kinase <br /> complete replication (Weinert and Hartwell 1988,                    activity at the G1/S stage in the cell cycle, delaying initia- <br /> 1993). We constructed diploid strains heterozygous for              tion of DNA replication. This is thought to provide the <br /> sid2-1 and rad9&#226;&#338;&#172; and homozygous for sic1&#226;&#338;&#172; and GAL-SIC1.             cell time to prepare properly for DNA replication (load <br /> Tetrad analysis showed that sid2-1 rad9&#226;&#338;&#172; spore colonies             origins, synthesize nucleotides, etc.; Tanaka et al. 1997; <br />  <br />  <br />   Figure 7.&#226;&#8364;&#8221;DNA replication is delayed in sid2-1 mutant strains. Strains MJ55 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1), MJ160 (sid2-1 sic1&#226;&#338;&#172; GAL1- <br /> SIC1), LY914 (SID2 SIC1), and LY1118 (sid2-1 SIC1) were grown overnight to log phase and arrested with 0.1 &#226;&#65533;&#174;m &#226;&#65533;&#163;-factor for 3 hr. <br /> The cultures were then released into YPD and progression into S phase was followed by FACS analysis (A and B) and onset of budding <br /> (C and D). Asynchronous cultures of strains LY914 and LY1118 were analyzed by FACS analysis and budding index (E and F). <br />  30                                                       M. D. Jacobson et al. <br />  <br />                            TABLE 3                                                                TABLE 4 <br /> rad9 sid2-1 double mutants have a defect in growth and viability            rad9 sid2-1 double mutants fail to demonstrate the <br />                                                                             preanaphase arrest observed in RAD9 sid2-1 strains <br /> Genotype <br /> (predicted or                % slow                  % fast                                      % large-budded <br /> observed)                 growing/dead              growing                                        cells with a           % anucleate <br />                                                                        Genotype                  single nucleus              cells <br /> RAD9 SID2                 17.9   (n   &#226;&#171;&#189;   12)    82.1   (n   &#226;&#171;&#189;   55) <br /> RAD9 sid2-1               20.0   (n   &#226;&#171;&#189;   11)    80.0   (n   &#226;&#171;&#189;   44)   RAD9 SID2                        10                       0 <br /> rad9::LEU2 SID2            7.3   (n   &#226;&#171;&#189;   4)     92.7   (n   &#226;&#171;&#189;   51)   rad9::LEU2 SID2                  10                       0 <br /> rad9::LEU2 sid2-1         98.5   (n   &#226;&#171;&#189;   66)     1.5   (n   &#226;&#171;&#189;   1)    RAD9 sid2-1                      80                       5 <br />                                                                        rad9::LEU2 sid2-1                27                       1 <br />    Spores from a diploid strain formed by crossing rad9::LEU2 <br /> SID2 sic1::TRP1 GAL1-SIC1 and RAD9 sid2-1 sic1::TRP1 GAL1-                Strains KK11 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1), KK17 (sid2-1 sic1&#226;&#338;&#172; <br /> SIC1 were dissected on YPGal and incubated at 30&#226;&#172;&#353; for 4 days.          GAL1-SIC1), KK20 (SID2 sic1&#226;&#338;&#172; GAL1-SIC1 rad9::LEU2), and <br /> Fast growing and slow growing phenotypes refer to colony               KK23 (sid2-1 sic1&#226;&#338;&#172; GAL1-SIC1 rad9::LEU2) were grown over- <br /> size as can be seen in Figure 8B and were assigned before              night to log phase in YPGal and dextrose was added (2%) to <br /> scoring sid2 or rad9. The sid2-1 genotype was assigned to viable       repress the galactose-inducible promoter making the strains <br /> spores on the basis of their failure to grow on YPD media.             sic1. The cells were fixed and processed for immunofluores- <br />                                                                        cence 6 hr after the addition of dextrose. At least 200 cells <br />                                                                        were scored for each genotype. The numbers represent the <br /> Vallen and Cross 1999). As Clb5/6p-Cdc28p may affect                   percentage of cells with a given morphology. The remaining <br /> the activity of the DNA replication machinery (Duncker                 cells were those that did not have a large bud with a single <br />                                                                        nucleus. This class included unbudded cells with a single nu- <br /> et al. 1999), Sic1p might also act during S phase to alter <br />                                                                        cleus, cells with small or medium buds and a single nucleus, <br /> the rate of elongation. A major event in the exit from                 and cells with large buds in anaphase or telophase. Large- <br /> mitosis is the degradation of Clb2p. Sic1p most likely works           budded cells had buds at least two-thirds the size of the mother. <br /> in parallel to the destruction of Clb2p by inhibiting the <br /> kinase activity of any remaining Clb2p associated with <br /> Cdc28p. To understand more completely the regulation                   is minimal compared to the effect of deleting CDH1. <br /> of these two cell cycle transitions, mutations synthetically           Cdh1p targets Clb2p to the APC (Schwab et al. 1997; <br /> lethal with sic1&#226;&#338;&#172; were isolated. Mutations in two genes                Visintin et al. 1997; Zachariae et al. 1998a; Figure 5) but <br /> already known to interact genetically with SIC1, DBF2 and              even the significant stabilization of Clb2p in cdh1 cells <br /> CDC15, were recovered. The products of both of these                   is not lethal. One likely possibility is that the observed <br /> genes are thought to assist Sic1p in regulating the exit               stabilization of the B-type cyclins is due to DNA damage <br /> from mitosis, although their mechanisms of action are                  in the sid2-1 cells (Germain et al. 1997). <br /> not yet completely established (Donovan et al. 1994;                      SID2 affects DNA replication or repair: Several argu- <br /> Jaspersen et al. 1998).                                                ments suggest that Sid2p acts during DNA synthesis or <br />    We found a novel gene, SID2, in this screen. We show                repair. First, the delay in completion of S phase in sid2-1 <br /> that SID2 is an essential gene (see also Huang et al. 1997)            mutants and the terminal arrest phenotype of sid2-1 sic1&#226;&#338;&#172; <br /> encoding a protein stable throughout the cell cycle.                   and sid2-21ts cells suggest that lack of Sid2p causes defects <br /> sid2-1 sic1&#226;&#338;&#172; and sid2-21ts strains arrest as large-budded cells        in DNA replication or damage that blocks cell cycle pro- <br /> with a single nucleus, a short spindle, and DNA content                gression. Second are the observed genetic interactions <br /> that is close to 2C. This is indicative of a preanaphase               between sid2-1 and CLB5/6, which function primarily to <br /> arrest and is likely to be due to a defect in the preceding            regulate DNA replication. Third, an allele of SID2 has <br /> S phase. In contrast, cells arrested because of a failure to           previously been isolated as tah11-1, which causes a temper- <br /> exit from mitosis due to high Cdc28p/Clb2p kinase activity             ature-sensitive growth defect in the presence of a DNA <br /> have two separated nuclei and an extended spindle (Sur-                topoisomerase I mutant (top1T722A) that mimics the cy- <br /> ana et al. 1993; Jaspersen et al. 1998). This phenotype is             totoxic action of camptothecin (Fiorani and Bjornsti <br /> indeed observed with dbf2 sic1 double mutants (Toyn et                 2000; R. J. D. Reid, P. Fiorani, M. Sugawara and M.-A. <br /> al. 1997). sid2-1 is the only mutation currently known to              Bjornsti, unpublished results). The tah11-1 mutant is also <br /> be synthetically lethal with sic1&#226;&#338;&#172; that causes an earlier cell         hypersensitive to hydroxyurea and to camptothecin when <br /> cycle arrest phenotype.                                                TOP1 is overexpressed. Other mutations that result in <br />    Both S and M phase B-type cyclins (Clb5p and Clb2p)                 similar phenotypes include alleles of CDC45 and DPB11 <br /> appear to be slightly stabilized in cells containing the sid2-1        (Reid et al. 1999). Both of these genes function in DNA <br /> mutation. This minor effect is unlikely to account for the             replication (Araki et al. 1995; Owens et al. 1997; Zou et <br /> sid2-1 sic1 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>, although we cannot fully rule            al. 1997; Zou and Stillman 1998) and appear to affect <br /> this out. The complete viability of sic1 CLB5&#226;&#338;&#172;DBHA strains             Okazaki fragment maturation (Reid et al. 1999). Fourth, <br /> (Figure 6B) argues against the possibility that the minor              the preanaphase arrest observed in sid2-1 sic1&#226;&#338;&#172; cells is <br /> effect on Clb5p stability is by itself sufficient to account           dependent on RAD9, suggesting that arrest results from <br /> for sid2-1 sic1 lethality. Similarly, the effect on Clb2p levels       induction of a checkpoint due to DNA damage and/or <br />                                                 SID2 and S Phase Cyclin Activity                                                         31 <br />  <br /> incomplete replication. Fifth, sid2-1 mutant strains are         GAL-SIC1 and sid2-1 SIC1 cells and sid2-1 GAL-SIC1 cells <br /> sensitive to the ribonucleotide reductase inhibitor, HU,         are dependent on RAD9 for full viability. <br /> which blocks replication progression.                               Although fractionation experiments show the majority <br />    One possibility is that, in sid2-1 sic1&#226;&#338;&#172; cells, sid2-1 may    of Sid2p is cytoplasmic, a more indirect assay suggests the <br /> lead to defects during S phase that are enhanced by the          protein does contain a functional NLS, which is capable <br /> precocious onset of replication caused by lack of sic1.          of targeting a fusion protein to the nucleus. Although we <br /> Removing the CLB5 destruction box acts to antagonize             can not unequivocally rule out the possibility that the <br /> this process further by placing additional stress in what        fusion protein is diffusing into the nucleus, a control fu- <br /> has become a critical point in the cell cycle of sid2-1 sic1     sion with the known cytoplasmic protein VirE2, which is <br /> strains. Apparently, delaying either the initiation or com-      similar in size to Sid2, failed to enter the nucleus. In <br /> pletion of replication by GAL1-SIC1 expression or by clb5&#226;&#338;&#172;       addition, diffusion into the nucleus is known to occur <br /> clb6&#226;&#338;&#172; provides time for sid2-1 cells to correct mistakes or      generally only for proteins less than &#226;&#181;&#8216;40 kD (reviewed in <br /> to synthesize precursors required for DNA replication.           Kaffman and O&#226;&#8364;&#8482;Shea 1999). While diffusion limits likely <br />    An alternative model for the role of SID2 is that it acts     depend on the tertiary structure of the protein as well as <br /> during the preceding G1 phase, as the cell is preparing          its molecular weight, the Sid2-containing fusion protein <br /> to replicate its DNA. If sid2-1 results in the cells having      is predicted to have a molecular weight of &#226;&#181;&#8216;100 kD, mak- <br /> not prepared properly for the forthcoming replication            ing it quite unlikely that the protein can diffuse into the <br /> process, then high Clb kinase activity late in G1 may lead       nucleus. Interestingly, the levels of transcription activated <br /> to lethality. High Clb kinase activity can prevent origins       by the Sid2 fusion were lower than those observed for the <br /> from becoming competent for DNA replication (Dah-                control containing the T-antigen NLS. Consistent with the <br /> mann et al. 1995). However, we find that sid2-1 sic1&#226;&#338;&#172; and        fractionation experiment, it may be that only a portion <br /> sid2-21 cells accumulate with close to a 2C complement           of the protein is constitutively nuclear or, alternatively, <br /> of DNA, making it unlikely that the arrest we observe is         that it is transported into the nucleus during a brief period <br /> due to a severe defect in origin firing. It is also unlikely     in the cell cycle. The presence of a functional NLS in Sid2 <br /> that the early entry into DNA replication caused by sic1&#226;&#338;&#172; in     suggests that the role of the protein in DNA replication <br /> a sid2-1 background is lethal due to insufficient nucleotide <br />                                                                  or repair, while unclear at this point, could be direct. <br /> levels. We recently demonstrated that MEC1 is required <br />                                                                  The characterization of more sid2 alleles as well as the <br /> for a prereplication delay that allows the accumulation of <br />                                                                  identification of Sid2p-interacting proteins may help illu- <br /> sufficient deoxyribonucleotides for DNA synthesis (Vallen <br />                                                                  minate the role of SID2 in DNA replication. <br /> and Cross 1999). Overexpression of RNR1, the gene en- <br />                                                                     Cyclin-dependent kinase inhibitors have previously <br /> coding the limiting subunit of ribonucleotide reductase, <br />                                                                  been demonstrated to have an important role in the main- <br /> suppresses the lethality of mec1&#226;&#338;&#172;. However, overexpression <br />                                                                  tenance of genome integrity. In mammalian cells, CDK <br /> of RNR1 does not suppress the lethality of sid2-1 sic1&#226;&#338;&#172; <br />                                                                  inhibitors are induced by irradiation and are frequently <br /> strains (E. A. Vallen, unpublished results). <br />                                                                  mutated in human cancers, suggesting that they have a <br />    We also think it is unlikely that sid2-1 sic1&#226;&#338;&#172; is lethal <br />                                                                  role in ensuring genome stability. Cells deleted for sic1 <br /> because of nonspecific damage to DNA. If any insult to <br />                                                                  have an increased frequency of chromosome loss and <br /> DNA were lethal to sic1&#226;&#338;&#172; cells, we would have expected <br /> to isolate many more mutations causing this phenotype.           breakage or recombination (Nugroho and Mendenhall <br /> Although our screen was not saturated, there are a large         1994). It is likely that these defects are due in some way <br /> number of genes whose null alleles cause increased DNA           to advancing origin firing or DNA replication in sic1&#226;&#338;&#172; <br /> damage or faulty repair (Friedberg et al. 1991). A simi-         cells. Identifying loci such as SID2, which genetically inter- <br /> lar screen for mutations synthetically lethal with sic1&#226;&#338;&#172;         act with CDK inhibitors and affect DNA replication or <br /> (Kramer et al. 1998) also failed to recover mutations affect-    repair, will help elucidate the specific roles of CDK inhibi- <br /> ing DNA repair. Furthermore, we and others have found            tors in ensuring genome stability. <br /> that sic1&#226;&#338;&#172; strains are not noticeably sensitive to UV or HU          <br /> (M. D. Jacobson, unpublished observations; Nugroho                <br /> and Mendenhall 1994) and that CLB5&#226;&#338;&#172;DB and sic1&#226;&#338;&#172;                   <br /> CLB5&#226;&#338;&#172;DB strains are also not UV sensitive (M. D. Jacob-      <br /> son, unpublished observations). The spontaneous rate of          <br /> point mutations as assayed by the frequency of canavanine-      <br /> resistant colonies is also not affected by sic1&#226;&#338;&#172; (E. A.          <br /> Vallen, unpublished observations; Nugroho and Men-           <br /> denhall 1994) or sid2-1 (K. Knox and E. A. Vallen,               <br /> unpublished observations). Perhaps sid2-1 causes a very <br />                                                             <br /> specific type of damage that is lethal in combination with       <br /> sic1&#226;&#338;&#172;. Whatever the defect, it is not entirely suppressed     <br /> by SIC1 expression as DNA synthesis is still slowed in sid2-1   <br />  <br /> </body></html>