<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />      Primate Chaperones Hsc70 (Constitutive) and Hsp70 (Induced) Differ <br />          Functionally in Supporting Growth and Prion Propagation in <br />                             Saccharomyces cerevisiae <br />  <br />                                                             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Hsp70&#226;&#8364;&#8482;s are highly conserved essential protein chaperones that assist protein folding and prevent pro- <br />              tein aggregation. They have modular structures consisting of ATPase, substrate-binding, and C-terminal <br />              domains. Substrate binding and release is regulated by ATP hydrolysis and nucleotide exchange, which in <br />              turn are regulated by cochaperones. Eukaryotes have constitutive (Hsc70) and stress-inducible (iHsp70) <br />              isoforms, but their functions have not been systematically compared. Using a yeast system <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> evaluate <br />              heterologous Hsp70&#226;&#8364;&#8482;s we find that primate Hsc70 supported growth but iHsp70 did not. Plant Hsc70 and <br />              iHsp70 counterparts behaved similarly, implying evolutionary conservation of this distinction. Swapping <br />              yeast and primate Hsp70 domains showed that (i) the Hsc70&#226;&#8364;&#8220;iHsp70 distinction resided in the ATPase <br />              domain, (ii) substrate-binding domains of Hsp70&#226;&#8364;&#8482;s within and across species functioned similarly regard- <br />              ing growth, (iii) C-terminal domain function was important for growth, and (iv) Hsp70 functions impor- <br />              tant for cell growth and prion propagation were separable. Enzymatic analysis uncovered a correlation <br />              between substrate affinity and prion <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> and showed that ATPase and protein-folding activities <br />              were generally similar. Our data support a view that intrinsic activities of Hsp70 isoforms are comparable, <br />              and functional differences in vivo lie mainly in complex interactions of Hsp70 with cochaperones. <br />  <br />  <br />  <br />  <br /> H     SP70 is a ubiquitous essential protein chaperone <br />        that assists protein folding and is involved in <br /> protein synthesis, protein transport across membranes, <br />                                                                           over the substrate-binding pocket, which stabilizes sub- <br />                                                                           strate binding. Return to the ATP-bound state restores <br />                                                                           the open conformation, facilitating substrate release. <br /> and dynamics of macromolecular assemblies such as mi-                     Intrinsic ATPase activity among Hsp70&#226;&#8364;&#8482;s is weak, and <br /> crotubules and clathrin (Gao et al. 1991; Nelson et al.                   ATP hydrolysis is the rate-limiting step in the reaction <br /> 1992; McClellan et al. 1998; Oka et al. 1998). Hsp70                      cycle (Schmid et al. 1994; McCarty et al. 1995). <br /> also protects cells <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> stress by preventing protein ag-                    Fine tuning of this cycle is regulated by cochaperones. <br /> gregation and helping refold denatured proteins (see                      Hsp40&#226;&#8364;&#8482;s, universal partners of Hsp70&#226;&#8364;&#8482;s, stimulate ATP <br /> Mayer et al. 2001; Hartl and Hayer-Hartl 2002 for                         hydrolysis and coordinate it with substrate binding <br /> general reviews). Hsp70 has a high affinity for extended                  (Greene et al. 1998; Laufen et al. 1999; Wittung- <br /> hydrophobic peptides and its diverse functions depend                     Stafshede et al. 2003). Nucleotide exchange factors <br /> upon ATP-regulated binding and release of exposed                         facilitate ADP release and subsequent rebinding of ATP <br /> hydrophobic surfaces on proteins.                                         (Kabani et al. 2002; Fan et al. 2003; Harrison 2003). In <br />    Hsp70&#226;&#8364;&#8482;s have a modular structure consisting of a highly                addition to regulating Hsp70 activity, cochaperones also <br /> conserved amino-terminal ATPase domain, an adjacent                       recruit Hsp70 to various subcellular locations or protein <br /> well-conserved substrate-binding domain (SBD) that                        complexes that require its chaperoning activity (Young <br /> contains a hydrophobic pocket and a lid-like structure                    et al. 2003). The CTD of many eukaryotic cytosolic <br /> over the pocket, and a conserved but more variable                        Hsp70&#226;&#8364;&#8482;s contains a conserved octapeptide that mediates <br /> C-terminal domain (CTD). In the ATP-bound state, the                      interactions with a variety of cochaperones, including <br /> substrate-binding pocket is open and rapidly exchanges                    Hsp40&#226;&#8364;&#8482;s and others that contain tetratricopeptide re- <br /> substrate. ATP hydrolysis induces closing of the the lid                  peat (TPR) protein interaction motifs (Freeman et al. <br />                                                                           1995; Duina et al. 1996; Marsh et al. 1998; Scheufler <br />   1 <br />    Present address: Cumhuriyet University, Faculty of Art and Science,    et al. 2000). TPR cochaperones have been shown to <br /> Chemistry Department, Room K3 110, 58140, Sivas, Turkey. <br />   2 <br />                                                                           stimulate ATPase of yeast cytosolic Hsp70 (Fan et al. <br />    Present address: Department of Life Science, Liaoning University, <br /> Shenyang, 110036, China. <br />                                                                           2003; Wegele et al. 2003b; Hainzl et al. 2004). <br />   3 <br />    Corresponding author: LBG/NIDDK/NIH, Bldg. 8, Room 407, 8 Center          Although the key features of Hsp70&#226;&#8364;&#8482;s reaction cycle <br /> Dr., Bethesda, MD 20892-0851. E-mail: masisond@helix.nih.gov              are highly conserved, different isoforms have evolved to <br />  <br /> Genetics 172: 851&#226;&#8364;&#8220;861 (February 2006) <br />  852                                           Y. Tutar, Y. Song and D. C. Masison <br />  <br /> perform specialized functions, such as the yeast Ssb                                           TABLE 1 <br /> subfamily that associates with ribosomes and assists                 Percentage of amino acid identity of Hsp70 isoforms <br /> translation (Nelson et al. 1992). There is a universal <br /> dependency on Hsp70 chaperone function in cellular                                            S. oedipus            A. thaliana <br /> processes during which proteins are incompletely <br />                                                                   Hsp70 isoform        Hsc70         iHsp70     Hsc70       iHsp70 <br /> folded, and while several Hsp70 isoforms are found in <br /> the cytosol, others reside in the endoplasmic reticulum           Ssa1p                  75            70         71          71 <br /> and mitchondria. Among the cytosolic isoforms, some               S.o. Hsc70                           77         76          76 <br /> are constitutively expressed (Hsc70&#226;&#8364;&#8482;s) whereas others             S.o. iHsp70                                     70          69 <br />                                                                   A.t. Hsc70                                                  90 <br /> (iHsp70&#226;&#8364;&#8482;s) are expressed when cells are exposed to stress. <br /> Under conditions of stress the increased abundance of               S. oedipus is the New World monkey cotton-top tamarin. S.o. <br /> Hsp70 protects cells by binding exposed hydrophobic               Hsc70 is identical to human Hsc70 (Hsp70-8); S.o. iHsp70 is <br /> surfaces on partially unfolded proteins and preventing            96% identical to human iHsp70 (Hsp70-6). <br />                                                                     A. thaliana is the plant thale cress. S.o., S. oedipus. A.t., A. <br /> nonproductive interactions that lead to protein aggre-            thaliana. <br /> gation. Feedback regulation of iHsp70 expression helps <br /> maintain overall Hsp70 abundance at a level appropri- <br /> ate for the need of its chaperoning function (Craig and           respectively. Arabidopsis thaliana iHSP70 (AJ002551) was cloned <br /> Gross 1991; Rieger et al. 2005).                                  by RT-PCR from mRNA prepared from heat-shocked plants, <br />    Since the structure and enzymatic activities among             provided by Charles Guy (Sung et al. 2001), and cloned sim- <br />                                                                   ilarly, creating pAtP70. A. thaliana HSC70-1 (X74604) was am- <br /> Hsp70&#226;&#8364;&#8482;s are similar and the various isoforms differ mainly        plified from cDNA and cloned similarly, creating pAtC70. <br /> in how their expression is regulated, it is generally             Amino acid identities of Hsp70 isoforms used in this study are <br /> thought that induction of iHsp70 expression is a re-              shown in Table 1. <br /> sponse to protect cells from protein misfolding prob-                Hybrid genes were constructed by first subcloning BamHI- <br /> lems by increasing the overall amount of Hsp70. It is also        SphI fragments from plasmids pN1, pCCC, and pPPP into <br />                                                                   plasmid pGEM-T (Promega, Madison, WI). The Quick-Change <br /> possible, however, that a specialized Hsp70 activity of           kit (Stratagene, La Jolla, CA) was used to make site-specific <br /> inducible isoforms is important under adverse condi-              nucleotide mutations without altering peptide coding to in- <br /> tions. In line with this notion, a recent study showing           troduce XbaI and XhoI sites at conserved codons correspond- <br /> that iHsp70 was specifically required for growth of can-          ing to L391 and E540, respectively, of SSA1. After swapping <br /> cer cells suggests that iHsp70 can function differently           NdeI&#226;&#8364;&#8220;XbaI (ATPase domain), XbaI&#226;&#8364;&#8220;XhoI (SBD), and XhoI&#226;&#8364;&#8220;SphI <br />                                                                   (CTD) fragments, hybrid genes were returned to pN1 as BamHI&#226;&#8364;&#8220; <br /> from Hsc70 (Rohde et al. 2005). Nonetheless, although             SphI fragments. L391 is at the end of a conserved stretch of <br /> there is considerable interest in Hsp70, there is very little     leucine residues between the ATPase and peptide-binding <br /> understanding of differences in function of cytosolic             domains. E540 is at a putative hinge point on the helix that <br /> Hsp70 isoforms.                                                   extends across the substrate-binding pocket ( James et al. <br />    The lack of a useful in vivo system has limited our            1997). Structures of all alleles amplified by PCR were con- <br />                                                                   firmed by sequencing. <br /> ability to compare functions of Hsp70 isoforms within or             Rich medium (YPAD) contains 1% yeast extract, 2% pep- <br /> across eukaryotic species. To compare and dissect func-           tone, 400 mg/liter adenine, and 2% glucose. 1/2YPD, which is <br /> tions of Hsp70&#226;&#8364;&#8482;s we developed a yeast system that can be          limiting for adenine and enhances red color of [psi ] cells, is <br /> used to assess functional complementation by any Hsp70            similar but lacks supplemented adenine and contains 0.5% <br /> isoform regarding both growth and prion propagation.              yeast extract. Synthetic media were as described (Sherman <br />                                                                   1994). Counterselection medium containing 5-fluoro-orotic <br /> Using this system we identified and characterized func-           acid (5-FOA) was used as described (Boeke et al. 1984). Briefly, <br /> tional differences among Hsp70&#226;&#8364;&#8482;s that provide new                 G400-1C cells transformed by plasmids carrying Hsp70 iso- <br /> insight into Hsp70 function.                                      forms and the LEU2 selectable marker were grown on medium <br />                                                                   lacking leucine and containing uracil, which relieves selective <br />                                                                   pressure for maintenance of the URA3 marker on pRDW10. <br />                                                                   Transformants were then transferred to medium containing <br />               MATERIALS AND METHODS                               5-FOA, which kills cells that express URA3. Only cells having <br />                                                                   lost pRDW10 during growth on the uracil-containing medium <br />    Yeast strain, plasmids, and growth conditions: Strain G400-    will gow on 5-FOA. Yeast were grown at 30&#194;&#176; unless indicated <br /> 1C (MATa kar1-1 SUQ5 ade2-1 his3D202 leu2D1 trp1D63 ura3-52       othwerwise. <br /> ssa1TKanMx ssa2THIS3 ssa3TTRP1 ssa4Tura3-f1/pRDW10)                  Thermotolerance: Aliquots of 650 ml were removed from <br /> ( Jones and Masison 2003) was used. Plasmids pRDW10 ( Jung        cultures grown at 30&#194;&#176; in YPAD to an OD600 &#194;&#188; 0.15&#226;&#8364;&#8220;0.3 and <br /> et al. 2000) and pN1 (Schwimmer and Masison 2002) are             transferred to 13 3 100-mm borosilicate tubes. A zero-time <br /> single-copy URA3- and LEU2-based vectors, respectively, with      sample of 100 ml was removed and transferred to a test tube on <br /> the SSA1 coding region and 500 bp of 59- and 39-flanking DNA.     ice. The remaining cultures were placed in a 52&#194;&#176; circulating <br /> Primate HSC70 (Saguinus oedipus, the New World monkey             water bath and 100-ml aliquots were removed periodically and <br /> cotton-top tamarin, GenBank accession no. AF142571) and           transferred to tubes on ice. Samples were then titered on YPAD <br /> iHSP70 (AF142572) (Wu et al. 2000) were PCR amplified using       plates, which were incubated for 5 days at 30&#194;&#176;. <br /> primers with 59 NdeI and SphI sites and products were used           Luciferase reactivation in vivo: Reactivation of thermally <br /> to replace SSA1 from the initiator ATG to a site 280 bp 39 of     denatured luciferase was done as described (Parsell et al. <br /> the translation terminator in pN1, creating pCCC and pPPP,        1994; Jung and Masison 2001). Briefly, cells expressing a <br />                                                   Distinct Functions of Hsc70 and Hsp70                                         853 <br />  <br /> thermolabile form of bacterial luciferase (Escher et al. 1989)                                RESULTS <br /> were shifted to 37&#194;&#176; for 30 min to induce expression of heat- <br /> shock proteins and subsequently to 44&#194;&#176; for 1 hr to cause ther-           Mammalian and plant Hsc70, but not iHsp70, sup- <br /> mal denaturation of luciferase. Cultures were then incubated          ports yeast growth: The yeast Ssa family of cytosolic <br /> at 23&#194;&#176; and recovery of luciferase activity was monitored over         Hsp70&#226;&#8364;&#8482;s consists of four functionally redundant isoforms <br /> time using a Zylux FB15 luminometer. Cyclohexamide was                (Ssa1p&#226;&#8364;&#8220;Ssa4p). Ssa1p and Ssa2p are abundant, consti- <br /> added to the cultures 10 min prior to the end of the 44&#194;&#176; incu- <br /> bation to prevent synthesis of luciferase during the recovery         tutively expressed, and 97% identical. Ssa1p expression <br /> period.                                                               is partially repressed under optimal conditions, but it <br />    Protein expression and purification: The Pichia pastoris ex-       becomes fully induced by stress or depletion of Ssa2p <br /> pression system (Invitrogen, San Diego) was used to subclone          (Stone and Craig 1990). Ssa3p and Ssa4p are expressed <br /> and express Hsp70&#226;&#8364;&#8482;s. Selective purification of cloned Hsp70 <br /> by ultrasonication of lysate, DEAE-Sephacel, butyl-Sepharose, <br />                                                                       only under nonoptimal growth conditions. Ssa Hsp70 <br /> and gel filtration chromatography was done as described               function is essential for viability and any one of the four <br /> (Wegele et al. 2003a). The Hsp40 protein Ydj1p was expressed          Ssap&#226;&#8364;&#8482;s can support yeast growth if sufficiently expressed <br /> in Escherichia coli BL21(DE3) from plasmid pET11aYdj and              (Werner-Washburne et al. 1987). <br /> purified with DEAE and hydroxyapatite chromatography as                  To test the ability of Hsp70&#226;&#8364;&#8482;s from other species to <br /> described (Lu and Cyr 1998; Fan et al. 2005). Sephadex G-50 <br /> (Fung et al. 1996) or charcoal (Ha et al. 1997) treatment was         support yeast growth we used strain G400-1C, which <br /> used to remove nucleotides from protein preparations.                 lacks all chromosomal SSA genes and expresses SSA1 <br />    Luciferase refolding in vitro: Refolding of guanidinum-HCl         from plasmid pRDW10 (Jones and Masison 2003). This <br /> denatured firefly luciferase (Promega) in buffer that con-            strain grows as well as wild type under optimal or stress- <br /> tained Hsp70, with or without a twofold molar excess of Hsp40         ful conditions, showing that Ssa1p provides all Hsp70 <br /> (Ydj1p), was performed as described (Lu and Cyr 1998). <br /> Aliquots of the refolding reaction were mixed with luciferase         functions necessary for normal growth and stress pro- <br /> assay reagent (Promega) and luminescence was measured in a            tection. Since Ssa Hsp70 function is essential for viability, <br /> Zylux FB15 luminometer. Identical reactions without Hsp70             G400-1C is unable to grow if it loses pRDW10. <br /> were done to determine background refolding.                             Strain G400-1C was transformed by plasmids carrying <br />    ATP hydrolysis assays: Hsp70 protein (0.5 mm) in buffer <br />                                                                       genes encoding the primate cytosolic Hsp70 isoforms <br /> Y (25 mm Hepes/KOH, pH 7.4, 5 mm MgCl2, 50 mm KCl, and <br /> 50 mg/ml bovine serum albumin) was incubated with 100 mm              Hsc70 (constitutive) or iHsp70 (inducible), and trans- <br /> [a-32P]ATP and 2-ml aliquots were collected every 5 min and           formants were tested for their ability to grow on medium <br /> dried on polyethyleneimine-thin layer chromatography plates.          that kills cells expressing the URA3 gene, which is car- <br /> Samples were resolved with a mixture of 0.5 m lithium chloride        ried on pRDW10 (materials and methods). If the <br /> and 0.5 m formic acid. Data were quantified using a Fuji film <br /> FLA-3000 phosphoimager and background hydrolysis was <br />                                                                       primate Hsp70&#226;&#8364;&#8482;s support yeast growth, then cells will be <br /> subtracted. When Ydj1 was included, samples were analyzed             able to lose pRDW10 and survive on the counterselect- <br /> according to Lu and Cyr (1998).                                       ing medium. The extent to which the primate proteins <br />    Peptide synthesis and labeling: Peptide GOKTUG                     provide essential Hsp70 function is then reflected in the <br /> (GLQLSLTARG), a modified form of Pep70 (Fung et al.                   growth rate of cells having lost pRDW10 and expressing <br /> 1996), was synthesized by Biosource and purified by reversed <br /> phase chromatography. Purity was determined by mass spec-             only the mammalian Hsp70. Cells transformed by the <br /> troscopy. Peptide labeling by acrylodan (6-acrylol-2-dimethy-         control plasmid expressing Ssa1p lost pRDW10 readily, <br /> laminonaphtalene) and removal of excess dye were performed            those expressing Hsc70 lost pRDW10 less frequently, <br /> according to Han and Christen (2004).                                 and those with the vector alone or expressing iHsp70 were <br />    Determination of dissociation constants: Experiments were <br />                                                                       unable to lose pRDW10. Thus, Hsc70 supported yeast <br /> performed in buffer Y (above). Measurements were done in <br /> a Perkin Elmer (Norwalk, CT) LS 50 spectrophotometer.                 growth but iHsp70 did not, which uncovers an impor- <br /> Acrylodan-labeled peptide (50&#226;&#8364;&#8220;200 nm) was titrated with               tant functional distinction between them. Cells sup- <br /> increasing concentrations of Hsp70 protein (25 nm&#226;&#8364;&#8220;30 mm)              ported by Hsc70 grew more slowly than those expressing <br /> in the presence of 1 mm ATP/ADP (500 ml total volume).                Ssa1p (Figure 1B colonies, Table 2), indicating that mam- <br /> Excitation and emission wavelengths were set to 370 and 500 <br /> nm, respectively. Emission spectra of peptide-Hsp70 com- <br />                                                                       malian and yeast Hsc70&#226;&#8364;&#8482;s also have significant func- <br /> plexes were recorded between 420 and 620 nm with 0.4 and              tional differences. <br /> 0.2 slitwidths. Each set of experiments was performed twice.             The difference in the ability of Hsc70 and iHsp70 to <br /> Fluorescence signals were corrected for background, inner-            support yeast growth could reflect a divergence of func- <br /> filter, and dilution effects. Dissociation constants (Kd) were        tion related to the different conditions under which the <br /> determined by using the equation <br />                                                                       isoforms are expressed. To test if the distinction in Hsc70 <br />       , F &#194;&#188; , FM   &#195;&#176;&#195;&#176;&#194;&#189;L T 1 &#194;&#189;E T 1 Kd &#195;&#382;                               and iHsp70 function was a general one, we repeated the <br />                                                                       growth complementation test with Hsc70 and iHsp70 <br />                          &#195;&#176;&#195;&#176;&#194;&#189;L T 1 &#194;&#189;E T 1 Kd &#195;&#382;2 <br />                                                                       isoforms from the widely used plant model A. thaliana. <br />                               4   &#194;&#189;L T   &#194;&#189;E T &#195;&#382;0:5 &#195;&#382;=&#195;&#176;2   &#194;&#189;E T &#195;&#382;      As with their primate counterparts, and despite the <br />                                                                       fact that the plant iHsp70 is more similar to plant Hsc70 <br /> (Gisler et al. 1998), where ,F is the increase in fluorescence <br /> at a given concentration and ,FM is the maximum fluores-              than the plant and primate Hsc70&#226;&#8364;&#8482;s are to each other <br /> cence change, and L and E are total concentrations of peptide         (90% vs. 77%, respectively, Table 1), the plant Hsc70 <br /> and Hsp70, respectively.                                              supported yeast growth but the plant iHsp70 did not <br />  854                                            Y. Tutar, Y. Song and D. C. Masison <br />  <br />                                                                                                 TABLE 2 <br />                                                                                       1 <br />                                                                    Growth and [PSI ] phenotype of cells supported by different <br />                                                                                         Hsp70 isoforms <br />  <br />                                                                                             Growth rate <br />                                                                    Hsp70 <br />                                                                    isoforma   (min/cell div)b     No. cells/15 hrc     [PSI1]d <br />                                                                    Unmodified <br />                                                                     AAA                113                250        1111 <br />                                                                     CCC                230                 15          1 <br />                                                                     PPP              Inviable                         NA <br />                                                                     AtC70              169                40         1111 <br />                                                                     AtP70            Inviable                         NA <br />  <br />                                                                    AAA substituted <br />                                                                      CAA           128                    130         111 <br />                                                                      PAA         Inviable                              NA <br />                                                                      ACA           113                    250         111 <br />                                                                      APA           114                    240          1 <br />                                                                      AAC           146                     72        11111 <br />                                                                      AAP           174                     36        11111 <br />  <br />                                                                    Others <br />                                                                      ACC               144                76           11 <br />                                 1                                    APP               226                16           11 <br />    Figure 1.&#226;&#8364;&#8221;Growth and [PSI ] prion phenotypes of cells <br />                                                                      CAC               177                34          111 <br /> expressing various Hsp70 isoforms. (A) Diagram of Ssa1p <br /> (AAA), indicating amino acid residue numbers at junctions            PAP             Inviable                          NA <br /> of ATPase, substrate-binding (SBD), and C-terminal (CTD)             CCA               129                126           1 <br /> domains. (B) Growth of strains is supported by the indicated         PPA             Inviable                          NA <br /> Hsp70 isoform. AAA is yeast Ssa1p, and CCC and PPP are pri-          CCP               161                48           11 <br /> mate Hsc70 and Hsp70, respectively. The remaining isoforms           PPC             Inviable                          NA <br /> are hybrids with ATPase, SBD, and CTD (from left to right)            a <br />                                                                         AAA is yeast Ssa1p, CCC and PPP are S. oedipus Hsc70 and <br /> from the indicated Hsp70&#226;&#8364;&#8482;s. The relative ability of Hsp70&#226;&#8364;&#8482;s <br />                                                                    Hsp70, respectively, and hybrid proteins have ATPase, sub- <br /> to support growth is reflected in the size of colonies in the <br />                                                                    strate-binding, and C-terminal domains (from left to right) <br /> small panels. The relative &#226;&#8364;&#732;&#226;&#8364;&#732;strength&#226;&#8364;&#8482;&#226;&#8364;&#8482; of [PSI1] propagation <br />                                                                    from the indicated proteins. AtC70 and AtP70 are A. thaliana <br /> is reflected in the degree of red pigmentation and the relative <br />                                                                    Hsc70 and Hsp70, respectively. <br /> rate of growth without adenine ( ade). As [PSI1] becomes              b <br />                                                                         Growth rate was measured for [psi ] cells grown in liquid <br /> weaker, cells become redder and less able to grow without ad- <br />                                                                    YPAD at 30&#194;&#176;. Inviable, allele does not support growth. <br /> enine. The appearance of red colonies reflects mitotic loss of        c <br />                                                                         Number of cells expected after 15 hr if one cell was al- <br /> [PSI1]. AAA  and AAA1 exemplify typical [psi ] and [PSI1] <br />                                                                    lowed to grow at the rate indicated. This number helps illus- <br /> phenotypes, respectively. <br />                                                                    trate the degree to which Hsp70 isoforms support growth. <br />                                                                       d <br />                                                                         Relative [PSI1] phenotype was scored subjectively and in- <br />                                                                    cludes color on YPD, growth without adenine, and mitotic sta- <br /> (Table 2). These results suggest that the basis for the            bility (see Figure 1B). NA, not applicable as Hsp70 does not <br /> distinction in function between these constitutively ex-           support growth. <br /> pressed and stress-inducible isoforms of Hsp70 is evo- <br /> lutionarily conserved. <br />    To identify the structural domains responsible for the <br /> differences in function, we constructed hybrid proteins            for growth complementation and prion propagation <br /> by separately substituting the ATPase, substrate-binding,          (see below). <br /> and C-terminal domains of Ssa1p with those from pri-                  ATPase domains of mammalian constitutive and in- <br /> mate Hsc70 and iHsp70 (see materials and methods,                  ducible Hsp70&#226;&#8364;&#8482;s have distinct functions: The PAA hy- <br /> Figure 1A). On the basis of their three-domain struc-              brid protein did not support yeast growth but CAA <br /> ture, we refer to the Ssa1p, Hsc70, and iHsp70 proteins            supported growth relatively well (Table 2 and compare <br /> as AAA, CCC, and PPP, respectively. Hybrid proteins are            colony size in Figure 1B). Therefore, a major difference <br /> named for the isoform domains they are derived from;               between the constitutive and inducible primate Hsp70&#226;&#8364;&#8482;s <br /> e.g., the PAA hybrid has the ATPase domain of iHsp70               is the function of the ATPase domain. All hybrid pro- <br /> fused to the SBD and CTD of Ssa1p. We constructed                  teins containing the ATPase domain of iHsp70 (P    ) <br /> additional hybrids in which single domains of CCC and              failed to support growth at both optimal (30&#194;&#176;) and ele- <br /> PPP were replaced with those from AAA and hybrids in               vated but nonlethal (37&#194;&#176;&#226;&#8364;&#8220;39&#194;&#176;) temperatures (Table 2 <br /> which the CTDs of CCC and PPP were switched. These                 and data not shown). In contrast, all other Hsp70 hybrids, <br /> additional hybrids (&#226;&#8364;&#732;&#226;&#8364;&#732;others&#226;&#8364;&#8482;&#226;&#8364;&#8482; in Table 2) were tested only        including APP, supported growth. Thus, activity of the <br />                                            Distinct Functions of Hsc70 and Hsp70                                       855 <br />  <br /> mammalian iHsp70 ATPase domain was not compatible              1C has the typical [PSI1] phenotype of uniform white <br /> with Hsp70 functions required for yeast growth.                colonies and good growth without adenine (Figure 1B, <br />    Substrate-binding domains of different Hsp70 iso-           AAA1), which shows that Ssa1p supports normal [PSI1] <br /> forms function similarly in vivo: Cells expressing Ssa1p       propagation as the only Ssap in the cell ( Jones and <br /> hybrids with SBD substitutions (ACA and APA) grew at           Masison 2003). We showed earlier that a mutant form <br /> the wild-type rate (Table 2 and compare colony size in         of Ssa1p weakens [PSI1], indicating that proper Hsp70 <br /> Figure 1B). Although substrate binding is crucial for          function is necessary for normal [PSI1] propagation <br /> Hsp70 functions, these results show that there are min-        (Jung et al. 2000; Jones and Masison 2003). <br /> imal differences in the ability of the different SBDs to          In addition to its reduced ability to support yeast <br /> interact directly with cellular substrates. Therefore, dif-    growth, mammalian Hsc70 (CCC) had reduced ability <br /> ferences in Hsp70 function, both within and across spe-        to support [PSI1] propagation, which was seen as re- <br /> cies, are better explained by how this peptide-binding         duced growth without adenine, partial pigmentation of <br /> activity is regulated by the flanking domains.                 cells, and the appearance of red [psi ] colonies (Figure <br />    Hsp70 C-terminal domain function is important for           1B and data not shown). The [PSI1] phenotypes of cells <br /> growth: Cells expressing Ssa1p hybrids with substituted        expressing the hybrid Hsp70&#226;&#8364;&#8482;s, however, showed that <br /> CTDs (AAC, AAP) grew but at significantly reduced              Hsp70 functions important for prion propagation were <br /> rates (Table 2 and compare colony size in Figure 1B).          separable from those important for growth. Although <br /> Thus, although the Hsp70 substrate binding cycle is            cells expressing Ssa1p with substituted C-terminal do- <br /> regulated primarily by ATP hydrolysis and nucleotide           mains (AAC, AAP) grew more slowy than those express- <br /> exchange, the C-terminal domain can significantly              ing Ssa1p, they displayed a stronger [PSI1] phenotype, <br /> influence Ssa1p activities important for cell growth.          as indicated by whiter colony color and increased rela- <br /> Together with the observations that SBD functions are          tive growth rate without adenine (Figure 1B, Table 2, <br /> similar and that cells expressing CCA grew better than         data not shown). Thus, the CTD substitutions had oppo- <br /> those expressing AAC (Table 2), these results show that        site effects on Hsp70 functions with regard to growth <br /> the CTD is an important determinant of functional              and prion propagation. <br /> differences between yeast and mammalian Hsc70. The                In contrast, cells expressing Ssa1p hybrids with mam- <br /> CTD is the most diverged region of Hsp70&#226;&#8364;&#8482;s and among           malian SBDs (i.e., ACA and APA) grew normally but <br /> eukaryotic cytosolic isoforms it is known to interact with     displayed a weakened [PSI1] phenotype (Figure 1B, <br /> a variety of cochaperones that regulate Hsp70 function         Table 2), which was seen as increased pigmentation and, <br /> (Freeman et al. 1995; Demand et al. 1998; Scheufler            in cells expressing APA, spontaneous mitotic loss of <br /> et al. 2000). These results might therefore reflect dif-       [PSI1] (red colonies). [PSI1] propagation was more <br /> ferences in specificity regarding such cochaperone             significantly impaired in cells expressing the APA hybrid <br /> interactions.                                                  than in cells expressing ACA. Since both of these hy- <br />    Differences in Hsp70 domain functions for growth            brids supported growth as well as Ssa1p, this difference <br /> and [PSI1] prion propagation: The genotype of strain           in magnitude of effect on [PSI1] is apparently unrelated <br /> G400-1C allows straightforward monitoring of the prop-         to Hsp70 SBD activities required for essential cellular <br /> agation characteristics of the yeast prion [PSI1]. [PSI1]      processes and reveals another qualitative distinction in <br /> propagates in the cytoplasm as self-replicating aggre-         functions of constitutive and inducible Hsp70 isoforms. <br /> gates, thought to be amyloid, of the translation release       Together, these data uncover differences in functions of <br /> factor Sup35p (eRF3) (Wickner 1994; Stansfield et al.          constitutive and inducible Hsp70&#226;&#8364;&#8482;s with regard to both <br /> 1995; Zhouravleva et al. 1995; Patino et al. 1996). Ag-        growth and prion propagation and are consistent with <br /> gregation of Sup35p in [PSI1] cells reduces the ability of     our earlier work showing that Hsp70 mutations that <br /> Sup35p to function in translation termination, which           impair [PSI1] propagation are separable from those <br /> suppresses the ade2-1 nonsense allele in certain yeast         affecting cell growth (Jung et al. 2000; Jones and <br /> strains (Cox 1965). Nonsuppressed ade2-1 cells require         Masison 2003). <br /> adenine and are red on limiting adenine due to ac-                Hsp70 function in thermotolerance and refolding of <br /> cumulation of an Ade2p substrate. By suppressing ade2-1,       thermally denatured protein: In addition to acting in <br /> [PSI1] confers growth without adenine and a nor-               processes essential for cell growth under optimal con- <br /> mal white colony color (Figure 1B, compare AAA  and            ditions, Hsp70 protects cells from stress by preventing <br /> AAA1).                                                         aggregation of partially unfolded proteins. When levels <br />    A weakened [PSI1] phenotype is seen as partial red          of stress are high enough to overcome the capacity of <br /> pigmentation on limiting adenine and reduced ability           Hsp70 to prevent aggregation, <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>survival</span> of fungi, plants, <br /> to grow without adenine. Spontantous loss of [PSI1]            and other organisms also requires an Hsp100-family <br /> during mitosis, which gives rise to red colonies, also         chaperone (Hsp104 in Saccharomyces cerevisiae) that acts <br /> reflects weakened [PSI1] propagation. Conversely, a            by resolubilizing aggregated proteins (Parsell et al. 1994; <br /> stronger [PSI1] phenotype is seen as enhanced growth           Glover and Tkach 2001). Hsp70 and its cochaperone <br /> without adenine and whiter colony color. Strain G400-          Hsp40 cooperate with Hsp104 in protein disaggregation <br />  856                                            Y. Tutar, Y. Song and D. C. Masison <br />  <br />  <br />  <br />  <br />   Figure 2.&#226;&#8364;&#8221;Thermotolerance of cells expressing various <br /> Hsp70 isoforms. Survival of cells exposed to the lethal temper-       Figure 3.&#226;&#8364;&#8221;Reactivation of thermally denatured bacterial <br /> ature of 52&#194;&#176;, expressed as a percentage of viable cells in the     luciferase in cells expressing various Hsp70 isoforms. After <br /> preexposure culture, is plotted as a function of time.             causing thermolabile luciferase to aggregate by exposing cul- <br />                                                                    tures to elevated but nonlethal temperature, recovery of lu- <br />                                                                    ciferase activity, expressed as a percentage of that in the <br /> (Glover and Lindquist 1998). Yeast lacking Hsp104                  preexposure cultures, was monitored as a function of time. <br /> grow normally under optimal conditions but die much <br /> more rapidly when exposed to a lethal stress. Hsp104 <br /> disaggregation activity is also essential for [PSI1] prop-         cells to elevated but nonlethal temperature (Figure 3). <br /> agation (Chernoff et al. 1995). To further assess the              ACA and CCC showed modestly reduced ability to <br /> function of the primate and hybrid proteins in vivo, we            function in this assay. The opposite result regarding CCC <br /> compared their ability to provide thermotolerance and              in the thermotolerance assay might reflect a difference <br /> to aid in the reactivation of a thermally denatured re-            in the ability of CCC to cooperate with Hsp104 in dis- <br /> porter protein. Although these processes in yeast are              aggregating proteins important for cell growth and a <br /> dependent upon Hsp104, such comparisons can reveal                 specific substrate not normally expressed in yeast. The <br /> differences in the ability of Hsp70 to prevent protein             strains expressing AAC and AAP were least efficient in <br /> aggregation and to cooperate with other chaperones in              reactivating luciferase, which reflects a reduced ability <br /> protein disaggregation.                                            of these isoforms to function with Hsp104 in disaggre- <br />    After exposure to a lethal temperature for 15 min, a            gating this substrate. Although an altered interaction <br /> time when survival depends mainly on protein disag-                with Hsp104 is not likely to explain the growth defect of <br /> gregation, all strains except those expressing CCC had             cells expressing AAC or AAP, it might affect the process <br /> a fraction of viable cells similar to that of the stain ex-        of breaking prion aggregates in a way that could explain <br /> pressing AAA (Figure 2). Survival of cells expressing              the improved [PSI1] phenotype in these cells. <br /> CCC was reproducibly three to fivefold higher than that               Enzymatic activities of purified Hsp70&#226;&#8364;&#8482;s: In addition <br /> of cells expressing AAA, indicating that it performed              to its structure, the enzymatic functions of Hsp70 are <br /> well in the disaggregation process. In the first 2 min             highly conserved. The ability of the primate, plant, and <br /> after the heat shock, when survival relies more on pre-            hybrid proteins to support growth implies that such <br /> vention of protein aggregation, the percentage of sur-             activities of these various Hsp70&#226;&#8364;&#8482;s are comparable. Never- <br /> vivors in cells expressing CCC was reproducibly lower              theless, the variation in phenotype of cells expressing <br /> than that of cells expressing AAA while the other strains          the different isoforms reflects functional differences <br /> survived normally. These data suggest that as compo-               among them. To test if the different phenotypes of cells <br /> nents of the Hsp104 disagregation machinery the                    expressing the various Hsp70&#226;&#8364;&#8482;s were attributable to spe- <br /> different isoforms functioned at least as well as wild             cific differences in intrinsic or stimulated Hsp70 activity <br /> type and that CCC might be less efficient than AAA at              we measured enzymatic activities of purified Hsp70&#226;&#8364;&#8482;s <br /> preventing protein aggregation in vivo.                            alone or combined with Ydj1p, which is one of the two <br />    To compare functions of the different Hsp70&#226;&#8364;&#8482;s in                major cytosolic Hsp40&#226;&#8364;&#8482;s of yeast known to regulate Ssa1p. <br /> protein disaggregation in vivo we monitored reactiva-                 Figure 4 shows amounts of ATP hydrolyzed in 10 min <br /> tion of a thermolabile bacterial luciferase after exposing         by the unmodified Hsp70 isoforms and those containing <br />                                                Distinct Functions of Hsc70 and Hsp70                                          857 <br />  <br />  <br />  <br />  <br />    Figure 4.&#226;&#8364;&#8221;Intrinsic and Hsp40-simulated ATPase activity <br /> of various Hsp70 isoforms. The percentages of ATP hydro-             Figure 5.&#226;&#8364;&#8221;Intrinsic and Hsp40-simulated refolding of <br />                                                                    chemically denatured firefly luciferase by various Hsp70 iso- <br /> lyzed by the different Hsp70 proteins in 10 min in the absence <br /> (intrinsic, shaded bars) or presence (1 Ydj1, solid bars) of the   forms. Reactivation of denatured luciferase after incubating <br /> Ssa1p Hsp40 partner Ydj1p are shown.                               with Hsp70 for 15 min in the absence (intrinsic, shaded bars) <br />                                                                    or presence (1 Ydj1, solid bars) of the Ssa1p Hsp40 partner <br />                                                                    Ydj1p is shown as a percentage of that present before denatur- <br />                                                                    ing. Background activity measured in samples without Hsp70 <br /> single substitutions of Ssa1p domains. CAA stands out as           has been subtracted. <br /> having the lowest intrinsic activity but it has an Hsp40- <br /> stimulated activity similar to that of Ssa1p (AAA). AAP <br /> had the highest stimulated activity, which might account           ities (Kd) using a fluorescently labeled peptide. Hsp70 <br /> for its reduced ability to support growth. However, cells          exchanges substrate rapidly when in the open ATP- <br /> expressing AAC had a phenotype similar those express-              bound form, whereas it releases substrate much more <br /> ing AAP but the AAC protein had near normal ATPase                 slowly when in the closed ADP-bound state. Overall we <br /> activity. Additionally, PPP, which did not support growth,         found that the Hsp70&#226;&#8364;&#8482;s bound peptide  10- to 20-fold <br /> had intrinsic and Hsp40-stimulated ATPase activity                 more tightly when associated with ADP than with ATP <br /> similar to that of wild type. Therefore, the variation in          (Table 3), which is in line with previously published data <br /> growth and prion phenotypes cannot be explained                    (Greene et al. 1995). In comparing the Kd&#226;&#8364;&#8482;s of the ADP- <br /> simply by differences in intrinsic or Hsp40-stimulated             bound Hsp70&#226;&#8364;&#8482;s that supported growth, those that <br /> ATPase activity. Moreover, it appears unlikely that the            improved [PSI1] propagation (AAC and AAP) had the <br /> inability of PPP to support growth was due solely to a             lowest affinity for peptide, while the proteins that had <br /> major difference in such ATPase activity.                          the greatest inhibitory effect on [PSI1] (CCC and APA, <br />    We then tested the ability of purified Hsp70&#226;&#8364;&#8482;s, alone           Table 2 and Figure 1B) bound peptide most tightly. <br /> or combined with Ydj1p, to refold denatured luciferase             These data agree with our earlier work showing that <br /> in vitro. Figure 5 shows relative levels of reactivation of        alterations that enhance Hsp70 substrate binding in- <br /> chemically denatured luciferase after 15 min of in-                hibit [PSI1] propagation ( Jones et al. 2004). Thus, it is <br /> cubation with Hsp70. Here again the differences in                 possible that the effects of the different hybrids on <br /> enzymatic activity generally were modest. The hybrids              [PSI1] propagation can be explained by differences in <br /> with the lowest refolding activity were CAA and PAA.               affinity of the isoforms for substrates. <br /> PAA was least able to refold luciferase, which might                  Abundance of Hsp70 and Hsp104: It was possible <br /> account for its inability to support growth. However,              that differences in Hsp70 abundance in cells expressing <br /> PPP, which also failed to support growth, functioned as            the various isoforms contributed to the different phe- <br /> well as wild-type Ssa1p in this assay. Therefore, even             notypes we observed. We therefore compared relative <br /> where differences in ability to refold denatured lucifer-          abundance of Hsp70 by Western analysis, using anti- <br /> ase were detectable they did not correleate with strain            bodies that recognize iHsp70 or both Hsc70 and iHsp70 <br /> phenotypes.                                                        (Figure 6A). Although abundance of Hsp70 varied <br />    To determine if the Hsp70 isoforms differed in                  among half the strains, the only appreciable differences <br /> affinity for substrate, we measured their binding affin-           compared with AAA were a modest reduction of ACA <br />  858                                           Y. Tutar, Y. Song and D. C. Masison <br />  <br />                           TABLE 3                                 those expressing CCC, ACA, and APA, which had slightly <br />              Hsp70 substrate-binding affinities                   less Hsp104, and CAA that had an elevated amount. <br />                                                                   Therefore, except for cells expressing CAA, differences <br />                            Kdb (mm)                               in Hsp104 expression are not likely to explain the ob- <br />                                                                   served differences in thermotolerance or luciferase <br /> Hsp70 isoforma         ATP          ADP             [PSI1]c <br />                                                                   reactivation in vivo. <br /> AAA                    60.91         7.62          1111 <br /> CCC                    59.64         4.55            1 <br /> PPP                    55.34        12.87           NA <br /> CAA                    55.59         5.62          111                                 DISCUSSION <br /> PAA                    51.53        45.43           NA <br /> ACA                    67.73         8.93          111               We developed a yeast system that allows us to test the <br /> APA                    58.21         2.56           1/&#226;&#8364;&#8220;           ability of any Hsp70 isoform to functionally replace yeast <br /> AAC                    64.37        12.26         11111           Hsp70 and used it to show that growth of budding yeast <br /> AAP                    53.40        10.06         11111           can be supported by Hsp70 from species as diverse as <br />   a <br />      Names are as described in Table 2 and in the text.           mammals and plants. We further found that constitutive <br />   b <br />      Kd was determined by titrating fluorescently labeled pep-    and inducible isoforms of primate cytosolic Hsp70 func- <br /> tide with Hsp70.                                                  tion differently, a distinction that we show is conserved <br />    c <br />      Values from Table 2 are reproduced to make comparison        in plants. Using hybrid proteins we identified Hsp70 <br /> easier.                                                           structural domains that confer this and other functional <br />                                                                   differences. Among these differences we found that <br />                                                                   Hsp70 functions important for prion propagation were <br /> and more significant reduction of CCC. Aside from CCC <br />                                                                   separable from those important for growth. Biochemi- <br /> there was no correlation between Hsp70 abundance <br />                                                                   cal analysis of the Hsp70 isoforms showed that, aside <br /> and any of the phenotypes assessed. Therefore, a differ- <br />                                                                   from substrate affinities, differences in enzymatic activ- <br /> ence in abundance might explain the exceptional phe- <br />                                                                   ities were not considerable, nor did they correlate with <br /> notypes of cells expressing CCC, but it is unlikely to <br />                                                                   phenotype, suggesting that the observed phenotypic <br /> explain differences among strains expressing the other <br />                                                                   differences were due to complex regulation of Hsp70 <br /> isoforms. <br />                                                                   functions in the cell. <br />    As Hsp104 plays a major role in both thermotolerance <br />                                                                      Hsp70 interacts with numerous cofactors and is a <br /> and luciferase reactivation, we also compared abun- <br />                                                                   major component of several different protein-folding <br /> dance of Hsp104 (Figure 6B). When taking into account <br />                                                                   machines in the cell. The fact that yeast growth can be <br /> the amount of total protein in each lane, the only strains <br />                                                                   supported by our collection of diverse Hsp70 proteins is <br /> with notable differences in Hsp104 abundance were <br />                                                                   remarkable as it demonstrates conservation of Hsp70&#226;&#8364;&#8482;s <br />                                                                   many functional interactions, including those involving <br />                                                                   communication between Hsp70 domains as well as <br />                                                                   between Hsp70 and its cochaperones. The ability of <br />                                                                   Ydj1p to stimulate ATPase and protein refolding of the <br />                                                                   various Hsp70&#226;&#8364;&#8482;s to similar levels in vitro is a clear dem- <br />                                                                   onstration of this conservation for an Hsp70&#226;&#8364;&#8220;Hsp40 <br />                                                                   interaction. We attributed the inability of primate iHsp70 <br />                                                                   to support yeast growth to its ATPase domain, but its <br />                                                                   ATPase activity in vitro was similar to that of Ssa1p. <br />                                                                   Therefore, it is likely that differences in vivo are influ- <br />                                                                   enced by interactions with other cofactors, possibly <br />                                                                   those coinduced with iHsp70. <br />                                                                      Craig and colleagues found that substrate-binding <br />                                                                   domains were not responsible for differences between <br />                                                                   the functionally distinct Ssa and Ssb Hsp70 subfamilies <br />                                                                   of yeast, leading them to propose that peptide-binding <br />    Figure 6.&#226;&#8364;&#8221;Western analysis to compare the abundance of <br /> Hsp70 and Hsp104 in cells expressing various Hsp70 iso-           activity per se plays little or no role in specifying Hsp70 <br /> forms. (A) Blots containing identical samples from lysates        function in vivo ( James et al. 1997). Our finding that <br /> of cells expressing the various Hsp70 isoforms, indicated at      hybrid Ssa1 proteins containing primate SBDs from <br /> top, were probed with antibody recognizing iHsp70 (top            either Hsc70 or iHsp70 supported growth as well as <br /> row) or both Hsc70 and iHsp70 (middle row). (B) Samples           Ssa1p shows that such functional redundancy of cyto- <br /> from similar lysates were probed with anti-Hsp104 antibody <br /> (top row). The bottom rows in both A and B show portions          solic Hsp70 SBDs extends both widely across eukaryotic <br /> of membranes used for the blots stained by amido black as         species and between mammalian Hsp70 isoforms. Thus, <br /> loading and transfer controls.                                    the functional differences among Hsp70&#226;&#8364;&#8482;s appear to lie <br />                                            Distinct Functions of Hsc70 and Hsp70                                                    859 <br />  <br /> more in how the SBD-substrate interactions are regu-           that deleting the octapeptide from human Hsp70 re- <br /> lated than in the physical properties of the different         duced Hop1&#226;&#8364;&#8220;Hsp70 interactions and that extended <br /> SBDs. In fact, we find that affinity of the different Hsp70    deletion of CTD residues abolishes the interaction, <br /> isoforms for substrate was determined mostly by the do-        indicating that CTD residues other than the octapep- <br /> mains flanking the SBD. ATPase and C-terminal domains          tide affect TPR cochaperone interaction (Carrigan <br /> are known to interact with a variety of cochaperones,          et al. 2004). The yeast and mammalian Hsp70&#226;&#8364;&#8482;s analyzed <br /> which adds to the complexity of this regulation in vivo.       here differ in several CTD residues, including a conser- <br /> Our data therefore are consistent with the view that           vative change in the fourth position of the octapeptide. <br /> specificity of Hsp70 function in vivo is dictated by var-      Whereas Ssa1 has valine, the mammalian Hsp70&#226;&#8364;&#8482;s have <br /> ious cochaperone interactions (James et al. 1997; Young        isoleucine. Thus, sequences both within and near the <br /> et al. 2003; Mayer and Bukau 2005) and suggest either          octapeptide likely affect the specificity or affinity of TPR <br /> that the SBD has little influence on the specificity of        cochaperone interactions, which in turn affect func- <br /> cochaperone interaction or that such SBD&#226;&#8364;&#8220;cochaperone           tions important for growth or prion propagation. Our <br /> interactions are highly conserved.                             system will be useful for identifying and characterizing <br />    Although the regulation of SBD function was more            Hsp70&#226;&#8364;&#8220;cochaperone interactions and the mechanisms <br /> important than SBD structure regarding growth com-             underlying their physiological effects. <br /> plementation, we found that isoforms AAA, ACA, and                Although humans have at least six cytosolic Hsp70&#226;&#8364;&#8482;s, <br /> APA, which differ only in their SBD, had different [PSI1]      one of which is strongly and constitutively expressed in <br /> phenotypes. [PSI1] therefore represents a diagnostic           all cells and another that is strictly stress inducible, the <br /> that is useful for identifying differences in Hsp70 func-      degree of functional redundancy or divergence among <br /> tion that are subtle enough to be undectable by their          them is unknown. Our finding that constitutive and <br /> effects on cell growth. Our current data do not distin-        inducible cytosolic isoforms of Hsp70 are functionally <br /> guish if such effects are due to altered interactions of       distinct regarding cell growth suggests that iHsp70 lacks <br /> Hsp70 with amyloid forms of Sup35p as a substrate or           an activity required for growth under normal conditions. <br /> with other components of the chaperone machinery               The conservation of this difference in Hsc70/iHsp70 <br /> such as Hsp40 or Hsp104. Nevertheless, our finding that        function in plants testifies to its significance and raises <br /> SBD substitutions weakened prion propagation without           the possibility that a specific Hsp70 function is impor- <br /> detectably affecting cell growth points to Hsp70 as a          tant when cells are exposed to stress. In support of these <br /> possible target for therapeutic intervention in amyloid        notions, Ja&#194;&#168;a&#194;&#168;ttela&#194;&#168; and colleagues found that viability of <br /> diseases.                                                      both normal and cancer cells depends upon Hsc70 and <br />    Our data show that the Hsp70 C-terminal domain              that cancer cells have an additional requirement for <br /> plays an important role in Hsp70 functions required for        iHsp70 (Nylandsted et al. 2000; Rohde et al. 2005). <br /> cell growth. Although Hsp40 interacts with both the C-         Therefore, depending on conditions, survival can hinge <br /> terminal and ATPase domains of Hsp70 (Freeman et al.           on the particular cytosolic Hsp70 isoform being expressed. <br /> 1995; Gassler et al. 1998; Greene et al. 1998), it is not      Additionally, in numerous studies assessing effects of <br /> known if Hsp70 activity is regulated by a specific inter-      Hsp70 overproduction little or no reason is given for the <br /> action between these two domains. While it has been            choice of Hsp70 isoform used, presumably because <br /> shown that the combined SBD and CTD can influence              Hsp70&#226;&#8364;&#8482;s are considered functionally equivalent. Our data <br /> activity of the ATPase domain (Lopez-Buesa et al. 1998),       suggest that the outcome of such experiments might be <br /> if a specific cooperation between the ATPase and               influenced by the particular Hsp70 isoform used. <br /> C-terminal domains exists, our data showing that ACA            <br /> and APA support growth as well as AAA implies that the       <br /> intervening SBD does not strongly influence such               <br /> cooperation.                                                 <br />    The CTD is the least conserved region of Hsp70 but <br />                                                             <br /> most eukaryotic cytosolic Hsp70&#226;&#8364;&#8482;s from yeast to mam-           <br /> mals end with a conserved octapeptide [GPT(I/V)                 <br /> EEVD] that is known to physically interact with TPR <br /> motifs. TPRs are structural domains involved in protein&#226;&#8364;&#8220; <br /> protein interactions that define, among other subclasses,                       <br /> a family of Hsp70 and Hsp90 cochaperones (Sikorski             <br /> et al. 1990; Blatch and Lassle 1999; Scheufler et al.            <br /> 2000). Earlier we showed that this octapeptide on Ssa1p     <br /> is important for yeast growth, although it is not essen-       <br /> tial for viability, and that some TPR cochaperones can          <br /> bind, albeit weakly, to Hsp70 that lacks the octapeptide       <br /> ( Jones et al. 2004). Smith and colleagues also showed       <br /> </body></html>