<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Delineation of the high-af&#194;&#174;nity single-stranded <br /> telomeric DNA-binding domain of Saccharomyces <br /> cerevisiae Cdc13 <br />                                                                   Several strategies have been identi&#194;&#174;ed for telomere capping <br />                                                                            by telomere end-binding proteins (TEBPs) (3). For example, <br /> Cdc13 is an essential protein <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Saccharomyces                           in the hypotrichous ciliate Oxytricha nova, the heterodimeric <br /> cerevisiae that caps telomeres by protecting the                           TEBP speci&#194;&#174;cally recognizes and buries the single-stranded <br /> C-rich telomeric DNA strand from degradation and                           overhang (4,5). An end-binding protein with limited sequence <br /> facilitates telomeric DNA replication by telomerase.                       similarity <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> O.nova TEBP has been identi&#194;&#174;ed both in the <br /> In vitro, Cdc13 binds TG-rich single-stranded telo-                        &#194;&#174;ssion yeast Schizosaccharomyces pombe and in humans (6). <br /> meric DNA with high af&#194;&#174;nity and speci&#194;&#174;city. A pre-                         Deletion of this protein (Pot1) in &#194;&#174;ssion yeast results in loss of <br /> viously identi&#194;&#174;ed domain of Cdc13 encompassing                             telomeric DNA, chromosomal mis-segregation, and reduced <br /> amino acids 451&#194;&#177; 694 (the 451&#194;&#177; 694 DBD) retains the                        growth that could be bypassed by circularization of the <br /> single-stranded DNA-binding properties of the full-                        chromosome. In mammalian cells, a TRF2-mediated duplex <br /> <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> protein; however, this domain contains a                            lariat structure at the terminus of the chromosome called a <br /> large unfolded region identi&#194;&#174;ed in heteronuclear                           t-loop has been proposed to sequester the end of the <br />                                                                            chromosome (7). <br /> NMR experiments. Trypsin digestion and MALDI <br />                                                                               Cdc13 is an essential telomere-capping protein from the <br /> mass spectrometry were used to identify the min-                           budding yeast Saccharomyces cerevisiae that protects the <br /> imal DNA-binding domain (the 497&#194;&#177; 694 DBD) neces-                          C-rich telomeric strand from degradation (8&#194;&#177;11). In vivo, <br /> sary and suf&#194;&#174;cient for full DNA-binding activity.                          the temperature sensitive mutant allele cdc13-1 causes <br /> This domain was completely folded, and the                                 resection of the C-rich strand at the non-permissive tempera- <br /> N-terminal unfolded region removed was shown                               ture, along with cell-cycle arrest and lethality (12). In <br /> to be dispensable for function. Using af&#194;&#174;nity                              independent genetic studies, CDC13 has also been shown to <br /> photocrosslinking to site-speci&#194;&#174;cally modi&#194;&#174;ed telo-                        be a positive and negative regulator of telomere length (13). <br /> meric single-stranded DNA, the 497&#194;&#177; 694 DBD was                            <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>Mutation</span> of residue 252 of Cdc13 causes a failure in telomere <br /> shown to contact the entire 11mer required for high-                       replication, even though the catalytic function of telomerase is <br /> af&#194;&#174;nity binding. Intriguingly, both domains bound                          not impaired in such mutant strains (14&#194;&#177;16). These mutant <br /> single-stranded telomeric DNA with much greater                            alleles of CDC13 can be reciprocally suppressed by certain <br />                                                                            mutations of the EST1 subunit of telomerase (16), suggesting <br /> af&#194;&#174;nity than the full-length protein. The full-length <br />                                                                            that the positive regulatory role of Cdc13 in vivo is recruitment <br /> protein exhibited the same rate of dissociation as                         of the enzyme to telomeric chromatin. Consistent with these <br /> both domains, however, indicating that the                                 activities, Cdc13 is believed to be localized to the 3&#194;&#162; single- <br /> full-length protein contains a region that inhibits                        stranded overhang at the telomere as it binds single-stranded <br /> association with single-stranded telomeric DNA.                            yeast telomeric DNA with both high af&#194;&#174;nity and speci&#194;&#174;city <br />                                                                            (14,17). Cdc13 does not bind double-stranded telomeric DNA <br />                                                                            or single- or double-stranded DNA of random sequence, and it <br /> INTRODUCTION                                                               does not require a free 3&#194;&#162; end for binding. Cdc13 has the same <br /> Telomeres are nucleoprotein complexes that form the ends of                af&#194;&#174;nity for binding a free single-stranded 3&#194;&#162; end, however, <br /> eukaryotic chromosomes. They are composed of repetitive                    such that it could bind and localize to the very end of the 3&#194;&#162; <br /> tracts of DNA and a suite of proteins that speci&#194;&#174;cally                     overhang in vivo. In fact, the DNA-binding function of Cdc13 <br /> recognize both the double-stranded region and the G-rich                   in isolation has been shown to be active in vivo. Tethering of <br /> single-stranded 3&#194;&#162; overhang of telomeric DNA (1). Telomeres                the telomerase components Est1p or Est3p to the telomere by <br /> perform various functions in the cell, including capping the               fusion with the Cdc13 DNA-binding domain (DBD) restores <br /> end of the chromosome, protecting it from degradation and                  immortality to senescing mutants of Cdc13 and results in <br /> end-to-end fusion, and serving as a substrate for telomerase,              longer telomeres, respectively (18,19). In addition, fusion of <br /> the specialized reverse transcriptase that replicates                      the DBD to the end-protection factor Stn1p restores cell <br /> telomeres (2).                                                             viability in the absence of full-length, functional Cdc13, <br />  <br /> *To whom correspondence should be addressed. Tel: +1 303 492 4576; Fax: +1 303 492 5894; Email: deborah.wuttke@colorado.edu <br />  4306   Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br /> although these cells still undergo senescence since they are       protein eluted at ~60% B and was over 95% pure as estimated <br /> unable to recruit telomerase components (16). These experi-        by Coomassie-stained SDS&#194;&#177;PAGE. The yield of protein was <br /> ments indicate that Cdc13 functions to localize key proteins to    typically 15&#194;&#177;20 mg per liter of cells. <br /> the telomere that are involved in telomere end protection and <br /> replication.                                                       Production of recombinant His-tagged 497&#194;&#177; 694 DBD <br />    A DBD of Cdc13 (451&#194;&#177; 694 DBD) was previously identi&#194;&#174;ed          DNA encoding the 497&#194;&#177; 694 DBD was PCR-ampli&#194;&#174;ed and <br /> within the 924-residue full-length protein by deletion mapping     subcloned into the expression vector pET21a (Novagen), <br /> and limited proteolysis, facilitating biochemical studies of       between the NdeI and XhoI restriction sites, in-frame with the <br /> Cdc13 bound to single-stranded telomeric DNA (20,21). The          C-terminal His-tag. BL21(DE3) E.coli were transformed, <br /> DBD has no similarity to any sequence in the database.             grown, and induced as described above. Cells were pelleted by <br /> Notably, no detectable sequence similarity can be detected         centrifugation, resuspended in lysis buffer (50 mM sodium <br /> between the 451&#194;&#177; 694 DBD and the other telomeric DNA end-          phosphate, pH 8.0, 300 mM NaCl, 10% glycerol, 0.5% Tween <br /> binding proteins S.pombe and human Pot1 (6) or the                 20, 10 mM imidazole, 5 mM b-mercaptoethanol, 1 mM <br /> heterodimeric O.nova TEBP (22&#194;&#177;24). Even in the complete            PMSF, 5 mM pepstatin A, 10 mM leupeptin, 100 mM antipain <br /> absence of sequence similarity, the recent solution structure of   and 200 mM chymostatin) and lysed by French press. Cellular <br /> the Cdc13 DBD characterized here revealed that this domain         debris was removed by centrifugation, and the supernatant was <br /> adopts the same fold as both TEBP and the predicted fold of        cleared of DNA by precipitation with 0.1% polyethylenimine <br /> Pot1 (25). This result suggests that these telomere-binding        at 4&#194;&#176;C for 30 &#194;&#177;45 min with stirring, followed by centrifuga- <br /> proteins are evolutionarily related and that structure&#194;&#177;function    tion. The supernatant was puri&#194;&#174;ed by af&#194;&#174;nity chromatography <br /> studies of Cdc13 are directly relevant to telomere maintenance     using a 5 ml HiTrap Chelating HP column (Pharmacia) <br /> in other organisms.                                                charged with nickel chloride. Equilibration/wash buffer was <br />    Although the 451&#194;&#177; 694 DBD retains DNA-binding proper-           the same as lysis buffer but without protease inhibitors. Bound <br /> ties comparable to that of the full-length protein, as assessed    protein was eluted with a linear gradient of imidazole from 10 <br /> by both biochemical (20,21) and genetic (16,18) studies, it        to 500 mM. Protein was estimated to be 95% pure as measured <br /> does not represent the minimal, independently folded               by Coomassie-stained SDS&#194;&#177;PAGE. <br /> structural domain. To better understand structure&#194;&#177;function <br /> relationships governing single-stranded DNA binding at             Production of recombinant full-length Cdc13 <br /> telomeres, we have further characterized the biochemical           Frozen SF9 cells infected with baculovirus expressing full- <br /> and structural properties of the minimal DBD.                      length His6-Cdc13 (amino acids 1&#194;&#177;924) were provided by the <br />                                                                    Lundblad Laboratory. The protein was puri&#194;&#174;ed using nickel- <br />                                                                    NTA chromatography as described previously (14). <br /> MATERIALS AND METHODS <br />                                                                    Limited trypsin cleavage of the 451&#194;&#177; 694 DBD <br /> Production of recombinant DBDs                                     The 451&#194;&#177; 694 DBD (30 mM in 200 ml), alone or with 1 molar <br /> DNA encoding the Cdc13 single-stranded telomeric DBDs              equivalent telomeric 11mer (dGTGTGGGTGTG), was incu- <br /> (amino acids 451&#194;&#177; 694 and 497&#194;&#177; 694) was PCR-ampli&#194;&#174;ed from          bated with 0.4% w/w trypsin (Sigma) at room temperature in <br /> a genomic clone (generously provided by the Lundblad               50 mM potassium phosphate, pH 7.0, 350 mM NaCl, 0.25 mM <br /> laboratory, Baylor College of Medicine) and subcloned into         Na2EDTA, 0.02% NaN3 and 2 mM DTT. At various time <br /> the T7 expression vector pET21a (Novagen) between the NdeI         points, 8 ml aliquots of the reaction were withdrawn, diluted <br /> and XhoI restriction sites. Electrotransformed (1.8 V, 400 W,      with 12 ml of water and 5 ml of SDS loading dye, and run on <br /> 25 mF) BL21(DE3) Escherichia coli were grown in LB                 15% SDS&#194;&#177;PAGE visualized with Coomassie brilliant blue <br /> medium with 50 mg/l ampicillin at 37&#194;&#176;C to an OD600 of 0.6 and      stain. <br /> induced with 1 mM IPTG at 22&#194;&#176;C for 4&#194;&#177;5 h. Cells were <br />                                                                    MALDI mass spectrometry of the limited trypsin <br /> harvested by centrifugation, resuspended in buffer A (50 mM <br />                                                                    cleavage products <br /> potassium phosphate, pH 7.0, 50 mM NaCl, 0.5 mM <br /> Na2EDTA, 0.02% NaN3 and 2 mM DTT), and lysed by two                For MALDI mass spectrometric analysis, the 451&#194;&#177; 694 DBD <br /> passes through a French press (Aminco). Cell extract was           (30 mM in 200 ml) was incubated with 0.4% w/w trypsin in <br /> cleared of nucleic acids by precipitation with 0.1% poly-          10 mM Tris&#194;&#177;HCl, 2 mM DTT at room temperature. Aliquots <br /> ethylenimine at 4&#194;&#176;C for 30 min with stirring and then              were withdrawn as described above for SDS&#194;&#177;PAGE analysis. <br /> centrifugation. The addition of a PEI precipitation step in        At the 10 min time point, 2 ml of the reaction mixture was <br /> the puri&#194;&#174;cation protocol, which was included in the puri&#194;&#174;ca-       mixed with 2 ml of a-cyano-4-hydroxycinnamic acid (CHCA) <br /> tion of every protein studied here, was necessary in order to      matrix. The samples were analyzed in positive ion mode on a <br /> remove endogenous E.coli nucleic acids that were non-              Voyager-DE STR mass spectrometer (Perseptive Biosystems). <br /> speci&#194;&#174;cally bound to the recombinant protein. The amount           Internal sample calibration was achieved with a mixture of <br /> of non-speci&#194;&#174;cally bound nucleic acid could be readily             insulin, thioredoxin and myoglobin standards. <br /> followed by monitoring the A260/A280 ratio in the UV/Vis <br /> absorption spectrum. Failure to remove the bound nucleic           Preparation of NMR samples <br /> acids resulted in spurious binding features. The cleared           Uniformly 15N isotopically labeled DBD (451&#194;&#177; 694 and <br /> supernatant was puri&#194;&#174;ed by ion-exchange chromatography             497&#194;&#177; 694 DBD without a His-tag) was produced by expression <br /> over a 5 ml HiTrap SP&#194;&#177;Sepharose column (Pharmacia) by              in minimal media containing 6.7 g/l Na2HPO4, 3 g/l KH2PO4, <br /> gradient elution with buffer B (buffer A with 1 M NaCl). The       1.5 g/l NaCl, 2 g/l glucose, 10 ml/l basal medium Eagle <br />                                                                                     Nucleic Acids Research, 2002, Vol. 30 No. 19     4307 <br />  <br />  <br /> Table 1. Yields of crosslinked complexes with photoactive DNAs             removed using microspin G25 columns (Pharmacia). All <br /> Name                         Sequence                            % Yield   assays were conducted in 5 mM HEPES, pH 7.8, 75 mM KCl, <br />                                                                            2.5 mM MgCl2, 0.1 mM Na2EDTA, 1 mM DTT and <br /> WT                           dGTGTGGGTGTG                         0        0.1 mg/ml BSA. Equilibrium binding reactions were per- <br /> DNA1                         dGIUGTGGGTGTG                       19        formed with 32P-11mer at concentrations 10-fold below the <br /> DNA2                         dGTGIUGGGTGTG                       22 <br /> DNA3                         dGTGTGGGIUGTG                       40        dissociation constant and serial dilutions of protein. The <br /> DNA4                         dGTGTGGGTGIUG                       19        reactions were incubated on ice for 30&#194;&#177;60 min to equilibrate. <br />                                                                            For gel-shift assays, 5 ml of each reaction with a small amount <br /> IU,   5-iodouracil substituted for the <span id='am-82' about='xsp:base' typeof='owl:Thing'>base</span> thymine.                       of bromophenol blue tracking dye were loaded on a 20 3 <br />                                                                            20 cm 3 1.5 mm, 5% acrylamide, non-denaturing gel. Gels <br />                                                                            were equilibrated at a constant 200 V for 30&#194;&#177;45 min before the <br /> vitamin solution (Gibco-BRL), 162.2 mg/l FeCl3, 2.86 mg/l                  samples were loaded. Gels were dried and visualized by a <br /> H3BO4, 15 mg/l CaCl2&#194;&#180;2H2O, 40 mg/l CoCl2&#194;&#180;6H2O, 200 mg/l                    PhosphorImager (Molecular Dynamics). For &#194;&#174;lter binding, <br /> CuSO4&#194;&#180;5H2O, 208 mg/l MgCl2&#194;&#180;6H2O, 2 mg/l MoO3, 208 mg/l                     80 ml of each binding reaction was &#194;&#174;ltered through a 96-well <br /> ZnCl2 and 1.5 g/l (15NH4)2SO4. Growth and puri&#194;&#174;cation was                  MultiScreen MAHA N4550 &#194;&#174;lter plate using a MultiScreen <br /> as described above except for a 7 h induction time with IPTG,              Resist Vacuum Manifold (Millipore). The wells were pre- <br /> yielding typically 10&#194;&#177;15 mg protein per liter of medium. The               washed with 80 ml of binding buffer without BSA, washed <br /> 451&#194;&#177; 694 DBD was concentrated to 400 mM in 50 mM                           with 2 3 200 ml after the samples had been &#194;&#174;ltered, and <br /> potassium phosphate, pH 7.0, 50 mM NaCl, 0.02% NaN3,                       the &#194;&#174;lter allowed to dry before being exposed to a <br /> 2 mM DTT-d10 and 10% D2O. This domain was insoluble at                     PhosphorImager screen. For both assays, spots were quanti&#194;&#174;ed <br /> concentrations above 400 mM. The 497&#194;&#177; 694 DBD was                          (Imagequant) and plots were normalized and &#194;&#174;t with a <br /> concentrated to 700 mM in 50 mM imidazole-d4, pH 7.0,                      standard two-state binding model: y = (ymax) / [1 + (Kd / x)] <br /> 150 mM NaCl, 100 mM Na2SO4, 0.02% NaN3, 2 mM                               where x is the concentration of protein, y is the fraction of <br /> DTT-d10 and 10% D2O.                                                       DNA bound and ymax is total protein bound, normalized. <br /> NMR methods                                                                Equilibrium dissociation constants (Kds) are reported as an <br />                                                                            average value plus or minus the standard deviation of at least <br /> NMR data were collected at 20 or 25&#194;&#176;C on a Varian Unity                    three measurements determined on different days or with <br /> Inova 600 MHz spectrometer. 1H-15N HSQC spectra were                       different protein preparations. <br /> obtained with 2048 points and 128 t1 increments using a <br /> gradient sensitivity-enhanced pulse sequence (26). Spectra                 Off rates measured by native PAGE <br /> were processed with the NMRPipe/NMRDraw programs                           Protein (5.55 nM) was incubated for 60 min on ice with <br /> using a cosine apodization function and one round of zero                  555 pM 32P-labeled DNA in 200 ml of binding buffer <br /> &#194;&#174;lling (27).                                                               containing bromophenol blue dye. Aliquots of 18 ml were <br />                                                                            removed over the time course; 2 ml of 1 mM unlabeled DNA <br /> Modi&#194;&#174;ed photoactive DNA oligonucleotides <br />                                                                            was added to each aliquot (&#194;&#174;nal concentrations of 5 nM <br /> The 5-iodo-2&#194;&#162;deoxyuridine-containing single-stranded DNA                   protein, 500 pM 32P-labeled DNA, and 100 nM unlabeled <br /> oligonucleotides (Operon; Table 1) were resuspended in                     DNA). Time points were analyzed by a 5% native acrylamide <br /> 10 mM triethylammonium acetate buffer, pH 6.0, and puri&#194;&#174;ed                 gel run at 200 V. Plots of protein-bound 32P-labeled DNA <br /> by acetonitrile gradient on a semi-preparative C4 reversed-                versus time were &#194;&#174;t to single exponential decay curves: y = <br /> phase column at 4 ml/min (Vydac). Solutions of the puri&#194;&#174;ed                 C1[exp(&#194;&#177;koff * x)] + C2 where x is time in hours, y is the <br /> oligonucleotides were prepared in deionized water and stored               fraction of DNA bound, C1 is the span (Ymax &#194;&#177; C2), and C2 is <br /> at &#194;&#177;20&#194;&#176;C. The purity of the DNA was determined to be &gt;99%                  the asymptote or plateau of non-speci&#194;&#174;c binding. <br /> by MALDI mass spectrometry obtained in negative ion mode <br /> using hydroxypicolinic acid (HPA) as the crystallization                   Binding titrations to determine the fraction of active <br /> matrix.                                                                    protein <br />                                                                            32P-labeled DNA (1 nM) and unlabeled DNA (100 nM) were <br /> Protein&#194;&#177;DNA photocrosslinking                                              mixed and heated to 90&#194;&#176;C for 10 min and cooled quickly on <br /> Crosslinking reactions (500 ml of 100 mM protein and DNA)                  ice. This mixture was incubated on ice for 1.5 h with varying <br /> were performed in 50 mM potassium phosphate buffer, pH 7.0,                amounts of protein (full-length Cdc13, the 451&#194;&#177; 694 DBD or <br /> 50 mM NaCl and 1 mM DTT. The reactions were transferred                    the 497&#194;&#177; 694 DBD) ranging from 0 to 3 molar equivalents. The <br /> to a 1 ml polymethylmethacrylate cuvette with a 1 cm path-                 samples were analyzed by a 5% native acrylamide gel. <br /> length and irradiated at 325 nm, stirring, for 3 h with an <br /> Omnichrome Series 74 He&#194;&#177;Cd laser operating at 25&#194;&#177;27 mW. <br /> The reactions were analyzed by SDS&#194;&#177;PAGE.                                   RESULTS <br /> Equilibrium binding assays by gel shift and &#194;&#174;lter                          Limited proteolysis reveals the minimal Cdc13 DBD <br /> binding                                                                    Deletion mapping and limited proteolysis were used pre- <br /> The 11mer dGTGTGGGTGTG was 5&#194;&#162;-end labeled using T4                         viously to map a DBD (451&#194;&#177; 694 DBD) within the 924 amino <br /> DNA kinase according to the Gibco-BRL protocol, with 5 mM                  acid full-length Cdc13 protein (20,21). This domain has a <br /> DNA and 150 mCi/ml [g-32P]ATP. A 25 ml labeling reaction                   strong tendency to precipitate at high concentrations which is <br /> was incubated at 37&#194;&#176;C for 30 min. Unincorporated 32P was                   somewhat alleviated at reduced temperatures (15&#194;&#177;20&#194;&#176;C). The <br />  4308    Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison of Cdc13 DBD 1H-15N HSQC spectra. (A) 1H-15N HSQC spectrum of 400 mM 451&#194;&#177; 694 DBD at 600 MHz, 20&#194;&#176;C. The sample was prepared <br /> in 50 mM potassium phosphate, pH 7.0, 50 mM NaCl, 0.02% NaN3, 2 mM DTT-d10 and 10% D2O. (B) 1H-15N HSQC spectrum of 700 mM 497&#194;&#177; 694 DBD at <br /> 600 MHz, 25&#194;&#176;C. The sample was prepared in 50 mM imidazole-d4, pH 7.0, 150 mM NaCl, 100 mM Na2SO4, 0.02% NaN3, 2 mM DTT-d10 and 10% D2O. <br />  <br />  <br /> 1H-15N HSQC spectrum of the 451&#194;&#177; 694 DBD (Fig. 1A)                        chemical shifts clustering between 8 and 9 p.p.m. in the proton <br /> reveals that while a folded species is present with well                  dimension. The lack of chemical shift dispersion and the <br /> dispersed chemical shifts in both proton and nitrogen                     presence of sharp linewidths in these peaks indicate the <br /> dimensions, the spectrum is dominated by cross-peaks with                 presence of an unfolded species. The features of this spectrum <br />                                                                                        Nucleic Acids Research, 2002, Vol. 30 No. 19             4309 <br />  <br />  <br />                                                                              acids at the N-terminus of the 451&#194;&#177; 694 DBD are particularly <br />                                                                              susceptible to trypsin cleavage, presumably because this <br />                                                                              region is unfolded. <br />                                                                              The 497&#194;&#177; 694 DBD forms a stable structural domain <br />                                                                              Several shorter candidates of the domain were subcloned for <br />                                                                              recombinant expression in E.coli. A construct of the domain <br />                                                                              comprising amino acids 497&#194;&#177; 694 expressed in high yield as a <br />                                                                              soluble protein. This domain exhibited several favorable <br /> Figure 2. SDS&#194;&#177;PAGE time course of limited trypsin digestion of the 451&#194;&#177;694   features relative to the 451&#194;&#177;694 DBD. It was considerably <br /> DBD. Reactions were performed as described in Materials and Methods.         more soluble than the longer domain and remained soluble at <br /> Lane m, protein markers; other lanes in minutes of time of the reaction.     higher temperatures (25&#194;&#177;30&#194;&#176;C). A comparison of the 1H-15N <br />                                                                              HSQC spectrum of the parent domain (451&#194;&#177; 694 DBD) and of <br /> could be due to an unfolded region of polypeptide in an                      the minimal domain (497&#194;&#177; 694 DBD) is shown in Figure 1. <br /> otherwise folded domain or the presence of both folded and                   The NMR spectrum was dramatically improved by the <br /> unfolded proteins at equilibrium.                                            removal of the unfolded region. The dispersed resonances of <br />    To test the possibility of an unfolded region in an otherwise             the folded species are almost identical between the two <br /> folded domain, the 451&#194;&#177;694 DBD was subjected to limited                      domains. However, the random coil resonances (Fig. 1A) have <br /> trypsin digestion at room temperature. Figure 2 illustrates the              completely disappeared (Fig. 1B), consistent with removal of <br /> time course of the reaction analyzed by SDS&#194;&#177;PAGE. Upon                       a large region of unfolded polypeptide that does not affect the <br /> exposure to trypsin, the 28 kDa 451&#194;&#177; 694 DBD almost                          structure of the folded domain. <br /> immediately formed a smaller, 22 kDa stable fragment <br /> which was remarkably resistant to further cleavage. This                     The 497&#194;&#177; 694 DBD contacts the entire minimal 11-nt <br /> reaction <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span> did not change in the presence of 1 molar                   DNA <br /> equivalent of the single-stranded DNA ligand dGTGTG-                         The minimal DNA required for high-af&#194;&#174;nity binding by full- <br /> GGTGTG, indicating that the unfolded region did not become                   length Cdc13 is an 11mer, dGTGTGGGTGTG. This sequence <br /> structured upon binding DNA (data not shown).                                of DNA is complementary to the yeast telomerase RNA <br />    MALDI mass spectrometry was used to speci&#194;&#174;cally deter-                    template and is representative of yeast telomeric sequence <br /> mine the boundaries of the smaller, stable domain. An                        (20,28). Four variants of this minimal DNA were used for <br /> identical trypsin digest product was obtained in low salt                    photocrosslinking, with the chromophore 5-iodouracil substi- <br /> conditions (10 mM Tris&#194;&#177;HCl), which facilitated direct analy-                 tuted for each of the four thymine bases of the molecule <br /> sis of the reaction. The major product by MALDI (Fig. 3) is                  (Table 1). The iodine atom of 5-iodouracil is approximately <br /> shown to be a MH(+1) of 21 985 6 72 Da. Examination of the                   the same size as the methyl group of thymine; therefore, <br /> map of predicted trypsin cut sites reveals that this fragment                this substitution is not likely to perturb binding of the <br /> corresponds to one of two predicted fragments: amino acids                   protein&#194;&#177;DNA complex. Indeed, the Kd of each of the <br /> 504 &#194;&#177;692 [MH(+1) 21 971] or amino acids 496 &#194;&#177; 685 [MH(+1)                    substituted DNAs for binding to the 497&#194;&#177; 694 DBD was <br /> 21 979]. This result indicates that approximately 50 amino                   identical to the unsubstituted DNA (data not shown). The long <br />  <br />  <br />  <br />  <br /> Figure 3. MALDI-TOF mass spectrum of trypsin digest products. (a) Minor digestion product; (b &#194;&#177; e) protein internal calibration standards as follows: <br /> b, insulin +1; c, myoglobin +2; d, thioredoxin +1; e, myoglobin +1. (f) major digestion product; (g) trypsin. <br />  4310    Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br />  <br />  <br /> Figure 4. SDS&#194;&#177;PAGE of photo-crosslinked 497&#194;&#177; 694 DBD/DNA at 100 mM <br /> each. Shown are time courses for irradiation at 325 nm, with lanes labeled <br /> in minutes and the &#194;&#174;rst lane containing size markers. The conditions are <br /> stated in Materials and Methods. (A) WT DNA of sequence dGTGTG- <br /> GGTGTG; (B) DNA1; (C) DNA2; (D) DNA3; (E) DNA4. <br />                                                                              Figure 5. Fraction of DNA bound as a function of protein concentration <br />                                                                              used to determine the equilibrium dissociation binding constant. Diamonds, <br />                                                                              full-length Cdc13; squares, 451&#194;&#177; 694 DBD; and circles, 497&#194;&#177; 694 DBD <br /> wavelength of 5-iodouracil&apos;s absorption and photocross-                      bound to dGTGTGGGTGTG. Each curve is an average of at least three sep- <br /> linking chemistry (325 nm) disfavors non-speci&#194;&#174;c excitation                  arate experiments conducted by &#194;&#174;lter binding. Plots were &#194;&#174;t with a standard <br />                                                                              two-state binding model. Equilibrium dissociation constants (Kds) are 310 <br /> of the DNA and protein chromophores (29). As expected, no                    6 50, 5 6 1 and 3 6 1 pM, respectively. Full-length protein is His-tagged, <br /> covalent products were formed and protein degradation was                    451&#194;&#177; 694 DBD is not His-tagged, and 497&#194;&#177; 694 DBD was measured with <br /> not observed upon irradiation of the 497&#194;&#177; 694 DBD com-                       and without a His-tag (no difference in binding was observed). <br /> plexed with unsubstituted DNA (Fig. 4A). A study of the time <br /> course of photocrosslinking revealed that each substituted <br /> DNA formed a speci&#194;&#174;c covalent adduct consistent with 1:1 <br /> stoichiometry of the protein and DNA (Fig. 4B&#194;&#177;E). The yields <br /> of crosslinked species determined by densitometry ranged <br /> from 20 to 40% (Table 1), which are typical for systems using <br /> this chromophore (29&#194;&#177;31). Identi&#194;&#174;cation of the exact site of <br /> crosslinking on the protein was precluded by the inability to <br /> generate complete proteolytic digests of the protein&#194;&#177;DNA <br /> adduct. However, we have recently obtained more detailed <br /> information on the binding interface of the protein by NMR <br /> structural analysis using experiments that measure intermole- <br /> cular contacts (25). Several aromatic amino acids are located <br /> along the interface that would be expected to form crosslinks <br /> to 5-iodouracil. These data indicate that the 497&#194;&#177; 694 DBD <br /> contains all the contacts needed for binding the minimal DNA <br /> 11mer.                                                                       Figure 6. Fraction of labeled DNA bound as a function of time as measured <br />                                                                              by native PAGE, used to determine the dissociation rate constant. <br />                                                                              Diamonds, full-length Cdc13; squares, 451&#194;&#177; 694 DBD; and circles, 497&#194;&#177; <br /> The 451&#194;&#177; 694 DBD and the 497&#194;&#177; 694 DBD bind telomeric                         694 DBD. The experiments were conducted as stated in Materials and <br /> DNA more tightly than the full-length protein                                Methods. Full-length protein is C-terminally His-tagged, while both <br />                                                                              domains are not His-tagged. <br /> To test whether the smaller 497&#194;&#177;694 DBD was both necessary <br /> and suf&#194;&#174;cient for function, gel-shift and &#194;&#174;lter-binding assays <br /> were used to determine the equilibrium binding dissociation                  these conditions with a binding stoichiometry of 1:1 (data not <br /> constant (Kd) of this domain with the single-stranded                        shown). Therefore, the differences that we observed in binding <br /> telomeric DNA substrate dGTGTGGGTGTG. Figure 5 com- <br />                                                                              were not due to a difference in the fraction of active protein. <br /> pares the binding curves determined by &#194;&#174;lter-binding assays of <br /> full-length Cdc13, the 451&#194;&#177;694 DBD and the 497&#194;&#177;694 DBD.                      The dissociation rates of the three proteins are similar <br /> Dissociation constants (Kds) measured by gel-shift assay <br /> yielded the same results (data not shown). The measured Kd of                Dissociation rates were measured by competition assay for the <br /> 310 6 50 pM for full-length Cdc13 con&#194;&#174;rms the result of                      three proteins and are presented in Figure 6. These dis- <br /> previous studies (14,20). However, we observed that the Kd for               sociation rates are quite similar, with rate constants of 2.8 3 <br /> 451&#194;&#177; 694 DBD was 5 6 1 pM, which is substantially lower                      10&#194;&#177;4 min&#194;&#177;1 for full-length Cdc13, 3.3 3 10&#194;&#177;4 min&#194;&#177;1 for <br /> than previously reported (370 pM) (20) and ranging from 72 to                451&#194;&#177; 694 DBD, and 4.3 3 10&#194;&#177;4 min&#194;&#177;1 for 497&#194;&#177; 694 DBD. This <br /> 240 nM for similar substrates (32). The Kd for the 497&#194;&#177;                      gives a half-life for the complex of ~41 h for full-length <br /> 694 DBD (both with and without the His-tag) was 3 6 1 pM,                    Cdc13, 27 h for the 497&#194;&#177; 694 DBD, and 35 h for the 451&#194;&#177; <br /> or ~100-fold tighter than for full-length Cdc13 and similar to               694 DBD. Based on the similarity of the dissociation rates, the <br /> that of the 451&#194;&#177; 694 DBD. Binding titrations well above the Kd               large difference in Kd between full-length protein and the two <br /> indicated that protein preparations were over 75% active under               domains must be due to an association rate effect. Association <br />                                                                              Nucleic Acids Research, 2002, Vol. 30 No. 19       4311 <br />  <br />  <br /> rates were too fast to measure accurately by gel-shift assay        within the spectrum of the 451&#194;&#177;694 DBD, clearly demon- <br /> directly. From the measured Kds and dissociation rates, the         strating that the folded region is also present in the longer <br /> calculated association rates are as follows: 9.0 3 105 M&#194;&#177;1          domain. Further biochemical characterization using af&#194;&#174;nity <br /> min&#194;&#177;1 for full-length Cdc13, 6.6 3 107 M&#194;&#177;1 min&#194;&#177;1 for the            photocrosslinking revealed that the smaller, folded domain <br /> 451&#194;&#177; 694 DBD, and 1.4 3 108 M&#194;&#177;1 min&#194;&#177;1 for the 497&#194;&#177;                  contacts the entire 11mer of single-stranded DNA required <br /> 694 DBD.                                                            for high af&#194;&#174;nity and speci&#194;&#174;city binding (20). Af&#194;&#174;nity <br />                                                                     photocrosslinking has previously been used successfully as a <br />                                                                     probe of contacts between protein and single-stranded DNA <br /> DISCUSSION                                                          (38). In particular, crosslinks found between chromophore- <br /> Single-stranded nucleic acids are involved in a wide array of       containing DNA and the O.nova TEBP corresponded to sites <br /> fundamental biological functions, including telomere regula-        of protein&#194;&#177;DNA contacts in the high resolution structure <br /> tion, DNA replication, repair and recombination, transcrip-         (22,29,39). Similarly high-yielding protein&#194;&#177;DNA crosslinks <br /> tional mechanisms, translation, and RNA splicing (33&#194;&#177;35).           were obtained in this system with use of the 5-iodouracil <br /> Recognition of single-stranded nucleic acids has been impli-        chromophore. We have shown that sites throughout the DNA <br /> cated in many pathological processes in humans, ranging from        11mer crosslink and therefore interact with the 198 amino acid <br /> cancer and aging to various infectious diseases. Therefore, the     minimal binding domain de&#194;&#174;ned here. This result has been <br /> sequence-speci&#194;&#174;c and non-speci&#194;&#174;c recognition of single-             subsequently con&#194;&#174;rmed by mapping the protein&#194;&#177;DNA inter- <br /> stranded nucleic acids by proteins is crucial for maintaining,      face using NMR spectroscopy, which revealed that several <br /> manipulating and utilizing the genetic material contained in        amino acids capable of crosslinking do contact DNA (25). <br /> cells. Relatively little is known concerning the requirements,         Interestingly, both DBDs bind DNA with tighter af&#194;&#174;nity <br /> either structural or functional, for sequence recognition in a      (Kd &#194;&#187; 3&#194;&#177;5 pM) than the full-length protein (Kd = 310 pM) <br /> single-stranded context. Cdc13 is an essential protein in           (Fig. 5). Thus, the smaller, 497&#194;&#177; 694 DBD contains the <br /> S.cerevisiae that regulates telomere capping and telomeric          essential region for contacting single-stranded telomeric <br /> replication (36). The single-stranded telomeric DBD is central      DNA. The difference in af&#194;&#174;nity between the isolated domains <br /> to these functions and can substitute for full-length protein       and full-length protein could simply be an artifact of <br /> when fused to appropriate binding partners (16,18). The high        extracting the domain from the full-length protein. However, <br /> af&#194;&#174;nity and sequence speci&#194;&#174;city of single-stranded DNA              enhanced binding of functional domains relative to full-length <br /> recognition exhibited by Cdc13 (14,20) make this an ideal           protein has been observed in the other TEBPs, O.nova TEBP a <br /> system to further our understanding of sequence-speci&#194;&#174;c             subunit (40) and S.pombe Pot1p (6), perhaps indicating that <br /> single-stranded DNA binding.                                        this is a general feature of this family of TEBPs. In the case of <br />    Previous identi&#194;&#174;cation of an independent DBD of Cdc13            Pot1 protein, deletion of a large C-terminal segment increased <br /> (451&#194;&#177; 694 DBD) has facilitated biochemical studies of its           binding af&#194;&#174;nity by ~10-fold, while in O.nova TEBP a subunit <br /> single-stranded DNA-binding activity (20,32). Although the          several different truncations at the C-terminus increased <br /> DBD de&#194;&#174;ned by amino acids 451&#194;&#177;694 is competent for single-          binding. Additionally, several splice variants of human Pot1 <br /> stranded telomeric DNA binding in vitro and can function            protein (hPot1) have been identi&#194;&#174;ed and appear to interact <br /> in vivo, the NMR data presented here clearly show that it is not    differentially with single-stranded human telomeric DNA <br /> a completely folded domain. The 1H-15N HSQC &#194;&#174;ngerprint              (P.Baumann, E.Podell and T.R.Cech, personal communica- <br /> spectrum of this domain (Fig. 1A) contains several intense,         tion). The full-length protein may contain a region inhibitory <br /> poorly dispersed resonances indicative of an unstructured state     to binding which could play a regulatory role in vivo by <br /> superpositioned on the well dispersed resonances indicative of      attenuating the extremely tight binding of the DBD. <br /> a folded protein. Protein domains involved in a diverse array          Our characterization of the 451&#194;&#177; 694 DBD has generated <br /> of cellular processes have been found to be intrinsically           results that are in contrast to previous studies of the same <br /> unstructured and only fold upon binding to their protein or         domain (20,32). In our study we have used highly puri&#194;&#174;ed, <br /> nucleic acid partners (37). Thus, we considered the possibility     soluble preparations of the DBDs, taking care to ensure that <br /> that the unfolded region is directly involved in contacting         over 75% of the protein was active. We have observed that it is <br /> DNA. However, in this case the unfolded region does not fold        critical to include a small amount of BSA (or Nonidet P-40 <br /> upon binding and does not affect single-stranded telomeric          detergent) in the binding reactions to prevent loss of the <br /> DNA binding by Cdc13 directly. The 451&#194;&#177; 694 DBD also                protein to surfaces and aggregation or precipitation of the <br /> exhibits poor solubility and is therefore unsuitable for in vitro   protein. When this protocol was not followed we obtained low <br /> characterization. Presumably, a construct of this domain            and irreproducible binding, presumably due to non-speci&#194;&#174;c <br /> expressed in yeast may have unexpected, variable effects as         loss of protein and/or protein inactivation (data not shown). <br /> well.                                                               Previous studies intended to determine the binding af&#194;&#174;nity of <br />    In this work, the Cdc13 DBD was re&#194;&#174;ned by limited trypsin        the 451&#194;&#177; 694 DBD were not performed in the presence of BSA <br /> digestion and MALDI mass spectrometry (Figs 2 and 3),               or detergent (20,32; V.Lundblad, personal communication). <br /> producing a domain (497&#194;&#177; 694 DBD) that is both structurally         This may explain the discrepancy between previous results <br /> and functionally independent. This represents the true,             and the binding af&#194;&#174;nities reported here. <br /> minimal DBD. In contrast to the 451&#194;&#177; 694 DBD, the 1H-15N               The rates of dissociation of the DNA ligand for full-length <br /> HSQC spectrum of the 497&#194;&#177; 694 DBD reveals the presence of           Cdc13, 451&#194;&#177; 694 DBD and 497&#194;&#177; 694 DBD are uniformly slow, <br /> a completely folded species (Fig. 1B). Careful examination of       yet similar. The calculated rate of association of the 451&#194;&#177; <br /> the spectrum obtained on the 451&#194;&#177; 694 DBD reveals that the          694 DBD and the 497&#194;&#177; 694 DBD with DNA is two to three <br /> well dispersed resonances of the 497&#194;&#177;694 DBD are present            orders of magnitude faster than for full-length protein. Thus, <br />  4312     Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br /> the differences in binding af&#194;&#174;nity are primarily due to                       <br /> differences in the rates of association. We have not determined                  <br />                                                                              <br /> in vitro if the attenuation of binding in the full-length protein is         <br /> due to regions N- or C-terminal to the DBD. Further study of                  <br /> this phenomenon is underway to determine what effect this                    <br /> attenuation has on yeast telomeres in vivo.                                  <br />    Cdc13 performs critical functions with partner protein                      <br />                                                                               <br /> complexes both in capping the telomere, protecting it from              <br /> degradation and fusion, and in regulating telomeric replication            <br /> and length. The single-stranded telomeric DBD of Cdc13 is                   <br /> central to its function and exhibits unusually high af&#194;&#174;nity and             <br /> speci&#194;&#174;city for its DNA target. We have delineated the true                    <br />                                                                             <br /> minimal folded domain that is both necessary and suf&#194;&#174;cient                  <br /> for high-af&#194;&#174;nity binding to single-stranded telomeric DNA.                  <br /> The present work provides insights into this important mode of             <br /> DNA recognition.                                                      <br />   <br /> </body></html>