<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Schizosaccharomyces pombe Aurora&#226;&#8364;&#8220;related <br /> Kinase Ark1 Interacts with the Inner Centromere <br /> Protein Pic1 and Mediates Chromosome Segregation <br /> and Cytokinesis <br />  <br />         The chromosomal passenger proteins aurora-B, survivin, and inner centromere protein (INCENP) <br />         have been implicated in coordinating chromosome segregation with cell division. This work <br />         describes the interplay between aurora, survivin, and INCENP orthologs in the fission yeast <br />         Schizosaccharomyces pombe and defines their roles in regulating chromosome segregation and <br />         cytokinesis. We describe the cloning and characterization of the aurora-related kinase gene ark1&#226;&#171;&#185;, <br />         demonstrating that it is an essential gene required for sister chromatid segregation. Cells lacking <br />         Ark1p exhibit the cut <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>phenotype</span>, DNA fragmentation, and other defects in chromosome segre- <br />         gation. Overexpression of a kinase-defective version of Ark1, Ark1-K147R, inhibits cytokinesis, <br />         with cells exhibiting an elongated, multiseptate phenotype. Ark1p interacts physically and/or <br />         genetically with the survivin and INCENP orthologs Bir1p and Pic1p. We identified Pic1p in a <br />         two-hybrid screen for Ark1-K147R interacting partners and went on <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> map domains in both <br />         proteins that mediate their binding. Pic1p residues 925&#226;&#8364;&#8220;972 are necessary and sufficient for Ark1p <br />         binding, which occurs through the kinase domain. As with Ark1-K147R, overexpression of <br />         Ark1p-binding fragments of Pic1p leads to multiseptate phenotypes. We also provide evidence <br />         that the dominant-negative effect of Ark1-K147R requires Pic1p binding, indicating that the <br />         formation of Ark1p-Pic1p complexes is required for the execution of cytokinesis. <br />  <br />  <br /> INTRODUCTION                                                          aurora kinases has been shown to be transforming (Bischoff <br />                                                                       et al., 1998; Zhou et al., 1998; Giet and Prigent, 1999), and <br /> The segregation of replicated chromosomes in mitosis must             aurora amplification or overexpression has been associated <br /> be precisely coordinated in space and time with the process           with breast (Sen et al., 1997; Zhou et al., 1998; Tanaka et al., <br /> of cytokinesis. Failure to separate the proper complement of          1999; Miyoshi et al., 2001) and colon (Bischoff et al., 1998; <br /> chromosomes into each daughter cell before cytokinesis can            Takahashi et al., 2000; Sakakura et al., 2001) tumors. A de- <br /> result in aneuploidy, which is linked to cell transformation          tailed understanding of how these kinases regulate cell di- <br /> and the development of cancer (for review, see Marx, 2001).           vision could thus prove useful in the development of cancer <br /> The aurora-Ipl1 kinases have received much attention re-              treatments. <br /> cently for their roles in regulating chromosome dynamics                 Aurora kinases are defined by a high degree of identity in <br /> and cytokinesis (for reviews, see Bischoff and Plowman,               their respective kinase domains and can be subdivided into <br /> 1999, and Giet and Prigent, 1999). Ectopic expression of              three classes: auroras A, B, and C (for review, see Nigg, <br />                                                                       2001). The aurora-A kinases are regulators of centrosomal <br />                                                                       duplication and separation and are thus crucial for the for- <br />   Article published online ahead of print. Mol. Biol. Cell 10.1091/   mation of the bipolar mitotic spindle (Glover et al., 1995; <br /> mbc.01&#226;&#8364;&#8220; 07&#226;&#8364;&#8220; 0330. Article and publication date are at www.molbiol.    Roghi et al., 1998; Giet and Prigent, 2000; for review, see <br /> cell.org/cgi/doi/10.1091/mbc.01&#226;&#8364;&#8220; 07&#226;&#8364;&#8220; 0330. <br />                                                                       Goepfert and Brinkley, 2000). Glover and coworkers identi- <br />   * Corresponding author. E-mail address: hunter@salk.edu <br />     Abbreviations used: APC, anaphase-promoting complex; BIR,         fied the founding member of this family in Drosophila mela- <br />     baculoviral inhibitor-of-apoptosis repeat; DAPI, 4,6-diamidino-   nogaster and discovered that cells lacking aurora have circu- <br />     2-phenylindole; GST, glutathione-S-transferase; HA, hemagglu-     lar arrays of chromosomes organized around a monopolar <br />     tinin; INCENP, inner centromere protein; KD, kinase-defective;    spindle (Glover et al., 1995). Aurora-A kinases localize to <br />     YES, yeast extract with supplements.                              centrosomes and are found along mitotic spindles through- <br />  <br />  <br /> 1132                                                                                  &#194;&#169; 2002 by The American Society for Cell Biology <br />                                                                                                 The Fission Yeast Aurora-related Kinase <br />  <br />  <br /> out mitosis and into telophase (Kimura et al., 1997; Bischoff        implicated in phosphorylating kinesin-like proteins that as- <br /> et al., 1998; Roghi et al., 1998; Schumacher et al., 1998; Zhou      semble and stabilize the central spindle (Geiser et al., 1997; <br /> et al., 1998). Depletion of Caenorhabditis elegans aurora-A          Giet et al., 1999; Kaitna et al., 2000; Severson et al., 2000). C. <br /> (AIR-1) by <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>RNA interference</span> leads to severe aneuploidy and           elegans aurora-B (AIR-2) binds to the kinesin-like protein <br /> embryonic lethality (Schumacher et al., 1998a). Although             ZEN-4, and both AIR-2 and INCENP (ICP-1) are required to <br /> centrosomes are able to separate in this case, they appear           recruit ZEN-4 to the central spindle (Kaitna et al., 2000; <br /> abnormal, and the mitotic spindle is disorganized.                   Severson et al., 2000). There is also evidence that aurora-B is <br />    Members of the aurora-B subfamily function later in mi-           required to recruit the Pavarotti kinesin-like protein in D. <br /> tosis, with transcript and protein levels peaking after those        melanogaster (Giet and Glover, 2001), although these data <br /> of aurora-A (for review, see Bischoff and Plowman, 1999).            were recently contradicted by another report (Adams et al., <br /> These kinases belong to a class of proteins referred to as           2001). Finally, it has been reported that rat aurora-B can <br /> chromosomal passengers, which are thought to play key                phosphorylate the myosin II regulatory light chain (Murata- <br /> roles in coordinating chromosome segregation with cytoki-            Hori et al., 2000) and might thereby regulate contraction of <br /> nesis (for review, see Adams et al., 2001). Chromosomal              the actomyosin ring. <br /> passengers are found along the <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> of chromosomes in                 Although vertebrates have up to three aurora kinases, <br /> prophase, concentrate at the inner centromere in metaphase,          budding yeast and fission yeast each possess only one fam- <br /> and are left behind at the central spindle during anaphase,          ily member. Chan and Botstein discovered the first aurora <br /> where they are thought to function in organizing and acti-           kinase, budding yeast Ipl1p, in 1993 (Chan and Botstein, <br /> vating the cytokinetic machinery. Depletion of aurora-B ki-          1993), and since then it has been further characterized by <br /> nases (Schumacher et al., 1998b; Woollard and Hodgkin,               others (Biggins et al., 1999; Kim et al., 1999; Wei et al., 1999). <br /> 1999; Kaitna et al., 2000) or expression of dominant-negative        Cells bearing a temperature-sensitive allele of IPL1 exhibit <br /> versions (Terada et al., 1998) results in late blocks to cytoki-     spindle pole defects and severe chromosome missegrega- <br /> nesis. Interestingly, the downregulation of aurora-B is re-          tion. Ipl1p was also the first aurora kinase shown to associ- <br /> quired for the polyploidization of human megakaryocytic              ate with an INCENP, Sli15p (Kim et al., 1999). Here, we <br /> lineages, which involves multiple rounds of replication in           describe the essential role of the S. pombe counterpart, auro- <br /> the absence of cytokinesis (Katayama et al., 1998; Kawasaki          ra-related kinase 1p (Ark1p), in chromosome segregation, its <br /> et al., 2001).                                                       interactions with the survivin homolog Bir1p, and its bind- <br />    Aurora-B orthologs have been shown to bind to the inner           ing to the fission yeast inner centromere protein Pic1p. We <br /> centromere proteins (INCENPs), chromosomal passenger                 define the domains in both Ark1p and Pic1p that mediate <br /> proteins that serve to localize and perhaps activate these           their binding and provide evidence that their interaction is <br /> kinases (Kim et al., 1999; Adams et al., 2000; Kaitna et al.,        required for the completion of cytokinesis. <br /> 2000; Adams et al., 2001). HeLa cells expressing a C-termi- <br /> nally truncated INCENP mutant (INCENP1&#226;&#8364;&#8220; 405) mislocalize <br /> aurora-B (Adams et al., 2000) and exhibit defects in mitosis         MATERIALS AND METHODS <br /> and cytokinesis (Mackay et al., 1998). RNA interference ex-          Yeast Strains and Media <br /> periments have shown that D. melanogaster aurora-B is re- <br /> quired for the transfer of INCENP <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> chromosome arms               Fission yeast were grown in the rich medium yeast extract with <br />                                                                      supplements (YES) or in Edinburgh minimal medium with the <br /> to centromeres and the midbody (Adams et al., 2001). An-             appropriate supplements (Moreno et al., 1991). Diploids were gen- <br /> other passenger protein, baculoviral inhibitor-of-apoptosis          erated by crossing the haploid strains FY527 (h- ura4-D18 leu1&#226;&#8364;&#8220;32 <br /> repeat (BIR)-1/survivin, is required for the localization of C.      his3-D1 ade6-M216) and FY528 (h&#226;&#171;&#185; ura4-D18 leu1&#226;&#8364;&#8220;32 his3-D1 ade6- <br /> elegans aurora-B (AIR-2) to chromosomes (Speliotes et al.,           M210). Overexpression studies were carried out in the haploid <br /> 2000). Human survivin is reported to bind directly to both           wild-type strain FY254 (h- ura4-D18 leu1&#226;&#8364;&#8220;32 can1&#226;&#8364;&#8220;1 ade6-M210). The <br /> aurora-B and INCENP (Wheatley et al., 2001), and IN-                 integration construct pJK210-ark1&#226;&#171;&#185;-HA3 was linearized with PflMI <br /> CENP1&#226;&#8364;&#8220; 405 inhibits the transfer of survivin from chromo-            and used to transform the temperature-sensitive strain FY584 (h&#226;&#171;&#185; <br /> somes to centromeres and the central spindle. Taken to-              cdc25&#226;&#8364;&#8220;22 ura4-D18 leu1&#226;&#8364;&#8220;32 ade6-M216 ark1&#226;&#171;&#185;HA3::ura4&#226;&#171;&#185;) to yield <br />                                                                      JLY55 (h&#226;&#171;&#185; cdc25&#226;&#8364;&#8220;22 ura4-D18 leu1&#226;&#8364;&#8220;32 ade6-M216 ark1&#226;&#171;&#185;HA3::ura4&#226;&#171;&#185;). <br /> gether, these data indicate that the initial localization of <br /> aurora-B depends on INCENP and/or survivin and that the <br /> subsequent migrations of these three passenger proteins are          Cloning, ark1&#226;&#180;&#8482; Disruption, and Mutagenesis <br /> interdependent. <br />                                                                      ark1&#226;&#171;&#185; was PCR-amplified from genomic DNA using the primers <br />    Binding to INCENP and survivin may serve to target                5&#226;&#172;&#732;-AGTGGCGGCCGCTGATGGTGTTACCTCAAAATG-3&#226;&#172;&#732; and 5&#226;&#172;&#732;- <br /> aurora-B to its substrates, which include diverse proteins           TTGAGCGGCCGCCGGAAGATTCAGAACTTTTGC-3&#226;&#172;&#732;, digested <br /> involved in mitosis and cytokinesis (for review, see Giet and        with NotI, and cloned into pBluescript-KS&#226;&#171;&#185; to give pBS-ark1&#226;&#171;&#185;. To <br /> Prigent, 1999). The single budding yeast aurora kinase Ipl1p         generate an ark1&#226;&#171;&#185; disruption construct, two ark1&#226;&#171;&#185; fragments were <br /> phosphorylates the kinetochore protein Ndc10p and is                 PCR-amplified from pBS-ark1&#226;&#171;&#185; and cloned into pAF1. The first <br /> thought to mediate microtubule-kinetochore attachments               product, encompassing the first 347 bp of ark1&#226;&#171;&#185;, was amplified with <br /> (Biggins et al., 1999; Sassoon et al., 1999). Bir1p binds directly   a T3 primer and 5&#226;&#172;&#732;-ACGCGTCGACCAATATGAAATTCTCGC- <br /> to Ndc10p and could target Ipl1p in this case (Yoon and              CATTG-3&#226;&#172;&#732;, digested with SalI, and cloned into pAF1 to give pAF1- <br />                                                                      ark1&#226;&#171;&#185;-A. The second product, comprising the last 308 bp, was am- <br /> Carbon, 1999). Aurora-B orthologs also phosphorylate serine          plified with a T7 primer and 5&#226;&#172;&#732;-AACTGCAGCCACCTGAAATGG- <br /> residue 10 in histone H3 (Hsu et al., 2000; Speliotes et al.,        TGGAGGG-3&#226;&#172;&#732;, digested with PstI and SacI, and cloned into pAF1- <br /> 2000; Adams et al., 2001; Giet and Glover, 2001), an event           ark1&#226;&#171;&#185;-A to give disruption construct pAF1-ark1&#226;&#171;&#185;-B. Diploids gener- <br /> that is required for premitotic chromosome condensation              ated by crossing strains FY527 and FY528 were then transformed <br /> (Wei et al., 1999). Aurora-B family members have also been           with a 2.7-kb NotI fragment from pAF1-ark1&#226;&#171;&#185;-B. Proper integration <br />  <br />  <br /> Vol. 13, April 2002                                                                                                                1133 <br />  Leverson et al. <br />  <br />  <br /> was confirmed by probing Southern blots of BanI-digested genomic         Yeast Two-Hybrid Screens and Direct Pairwise <br /> DNA with a probe spanning the first 351 bp of ark1&#226;&#171;&#185;.                     Tests <br />    ark1&#226;&#171;&#185; was subcloned from pBS-ark1&#226;&#171;&#185; into pGEX-KG using XbaI <br /> and SacI to generate pGST-ark1&#226;&#171;&#185;. Constructs linking ark1&#226;&#171;&#185; down-          Budding yeast two-hybrid strain AH109 (Clontech) harboring the <br /> stream of the full-strength nmt1 promoter (Maundrell, 1993) were         HIS3, ADE2, and lacZ reporters downstream of heterologous GAL4- <br /> generated by digesting pBS-ark1&#226;&#171;&#185; with NotI and ligating the puri-        responsive promoter elements was first transformed with pGBT9- <br /> fied insert into either pSLF172 (Forsburg and Sherman, 1997) or          ark1K147R or pGBT9- ark1NT with a standard lithium acetate proce- <br /> pSGP72. Partial fragments of ark1&#226;&#171;&#185; were PCR-amplified from pBS-          dure. Then, 600-ml cultures of these transformants were used for <br /> ark1&#226;&#171;&#185; using primers engineered with NotI sites for cloning into          transformation with 50 &#226;&#65533;&#174;g of S. pombe Matchmaker cDNA library <br /> pSLF172. All pSLF172 constructs express proteins in-frame with           (Clontech) and plated to minimal medium containing Lys, Ade, <br /> three C-terminal hemagglutinin (HA) epitope tags, which were             Ura, Tyr, Met, and 2.5 or 10 mM 3-amino-1,2,4-triazole. Several His&#226;&#171;&#185; <br /> detected by immunoblotting with the 12CA5 anti-HA monoclonal             colonies were picked between days 4 and 8 after plating, restreaked <br /> antibody (mAb). An insert containing ark1&#226;&#171;&#185;::HA3 and the nmt1             to &#226;&#171;&#186;Trp&#226;&#171;&#186;Leu&#226;&#171;&#186;His&#226;&#171;&#186;Ade medium, and patched out for &#226;&#65533;&#164;-galacto- <br /> terminator was subcloned from pSLF172-ark1&#226;&#171;&#185; into pJK210 using            sidase colony-lift filter assays (Matchmaker System User Manual, <br /> XhoI and SacI to yield pJK210-ark1&#226;&#171;&#185;-HA3.                                 Clontech). Isolates that tested positive for &#226;&#65533;&#164;-galactosidase activity <br />    The two-hybrid bait construct pGBT9- ark1K147R was generated by       were cultured in &#226;&#171;&#186;Leu liquid medium overnight and used to pre- <br /> PCR-amplifying ark1K147R with 5&#226;&#172;&#732;-CCGGAATTCGTGTTACCT-                     pare total DNA for PCR amplification and sequencing of library <br /> CAAAATGTAAACAAC-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-TTTCTGCAGGGAAGATTCA-                           vector inserts. <br /> GAACTTTTGCGAG-3&#226;&#172;&#732;, digesting with EcoRI and PstI, and ligating <br /> into pGBT9 (Clontech, Palo Alto, CA). Bait pGBT9-ark1NT, compris- <br /> ing the N-terminal 116 residues of Ark1p, was constructed by <br /> amplifying an ark1&#226;&#171;&#185; fragment with 5&#226;&#172;&#732;-CCGGAATTCGTGTTACCT-                 RESULTS <br /> CAAAATGTAAACAAC-3&#226;&#172;&#732; and 5&#226;&#172;&#732;-TTTCTGCAGCATTCCAATAT- <br /> GAAATTCTCGCC-3&#226;&#172;&#732; and ligating into pGBT9 with EcoRI and PstI.             ark1&#226;&#180;&#8482; Is Required for Sister Chromatid Segregation <br /> pic1&#226;&#171;&#185; fragments were PCR-amplified with primers engineered with          Others had previously noted the existence of an aurora <br /> EcoRI and XhoI sites for ligation into the prey vector pGAD-GH <br /> (Clontech). <br />                                                                          kinase gene in S. pombe (accession number AL022245.2; <br />    All mutagenesis was carried out with the QuikChange Site-Di-          Bischoff and Plowman, 1999; Giet and Prigent, 1999). We <br /> rected Mutagenesis kit (Stratagene, La Jolla, CA) according to the       designed oligonucleotides to PCR-amplify ark1&#226;&#171;&#185; (aurora- <br /> manufacturer&#226;&#8364;&#8482;s instructions. The sequence of each mutant was con-        related kinase 1, named by I. Hagan, see Morishita et al., <br /> firmed by automated sequencing (Applied Biosystems Inc., Foster          2001) from genomic DNA and went on to disrupt one copy <br /> City, CA).                                                               of the gene in a diploid strain. Genomic Southern blotting <br />                                                                          confirmed that a large portion of ark1&#226;&#171;&#185;, encoding kinase <br /> Spore Germination Assay,                                                 subdomains I&#226;&#8364;&#8220;VIII, had been replaced by the his3&#226;&#171;&#185; gene <br /> 4,6-Diamidino-2-Phenylindole/Calcofluor Staining,                        (Figure 1A and our unpublished results). His&#226;&#171;&#185; heterozy- <br /> and Immunofluorescence                                                   gotes were allowed to sporulate, and tetrad dissection was <br />                                                                          carried out on YES. Of 30 tetrads examined, 29 comprised <br /> Spo&#226;&#171;&#185; diploid cells heterozygous for the ark1&#226;&#171;&#185; disruption were <br />                                                                          two viable and two nonviable spores, with all of the viable <br /> grown in YES to an OD595 of 0.8 and then allowed to sporulate for <br /> 72 h in liquid malt extract (25&#194;&#176;C). Cultures were checked for asci and   colonies being His&#226;&#171;&#186; (Figure 1B and our unpublished re- <br /> then treated with 2% glusulase overnight at 25&#194;&#176;C. Spores were            sults). The disruption was thus lethal, demonstrating that <br /> rinsed in yeast nitrogenous bases lacking ammonium sulfate and           the ark1&#226;&#171;&#185; gene is essential for viability. As expected, express- <br /> spun through a 25% glycerol cushion for 10 min at 2000 rpm. After        ing ark1&#226;&#171;&#185; from an episome rescued the lethality of haploid <br /> three additional rinses, ark1&#226;&#171;&#185;-disrupted his&#226;&#171;&#185; spores were allowed to     disruptants (our unpublished results). <br /> germinate overnight at 32&#194;&#176;C in Edinburgh minimal medium lacking             We next investigated the cause of lethality by performing <br /> histidine. Cells were then stained with 4,6-diamidino-2-phenylin-        spore germination experiments. Spores generated from het- <br /> dole (DAPI) for analysis by fluorescence microscopy. DAPI/cal- <br /> cofluor staining was performed according to previously published         erozygous ark1&#226;&#171;&#185; disruptants were grown overnight in me- <br /> methods (Moreno et al., 1991).                                           dium lacking histidine, allowing us to selectively monitor <br />                                                                          the growth of ark1&#226;&#171;&#185;-disrupted cells. Spores germinated, but <br />                                                                          DAPI staining revealed that most of the resulting cells failed <br /> Recombinant Proteins and Kinase Assays                                   to complete mitosis. A large portion exhibited the cut (cell <br /> Glutathione-S-transferase (GST)-fusion and His-tagged proteins           untimely torn) phenotype, with sister chromatids failing to <br /> were expressed in Escherichia coli and purified as described previ-      separate before cytokinesis (Figure 1C). This phenotype can <br /> ously (Leverson et al., 2000). An N-terminal fragment of Bir1p           be indicative of failures in chromosome condensation, bipo- <br /> encompassing the first 330 residues was expressed fused to eight <br /> N-terminal histidines for purification with Talon metal affinity resin <br />                                                                          lar spindle formation, microtubule-kinetochore attachments, <br /> (Clontech). Bir11&#226;&#8364;&#8220;330 fractions also contained an abundant break-        and/or sister chromatid separation/segregation (Yanagida, <br /> down product, which peptide sequencing and mass spectrometry             1998). We often observed dividing cells with unequal <br /> revealed to comprise residues 1&#226;&#8364;&#8220;208. Kinase assays were performed        amounts of DNA on either side of the septum, and many <br /> as follows: GST-Ark1p was bound to 10 &#226;&#65533;&#174;l bed-volume glutathione-         cells exhibited stretched or fragmented DNA (Figure 1C). <br /> agarose in 1 ml PBS for 30 min at 4&#194;&#176;C. Beads were washed twice           These phenotypes are very similar to the chromosome con- <br /> with PBS, three times with kinase wash (20 mM HEPES pH 7.4, 1            densation and segregation defects observed in cells deleted <br /> mM dithiothreitol), and then incubated for 20 min with or without <br />                                                                          for bir1&#226;&#171;&#185;/pbh1&#226;&#171;&#185;/cut17&#226;&#171;&#185; (Samejima et al., 1993; Rajagopalan <br /> substrates in kinase wash with 20 mM MgCl2, 25 &#226;&#65533;&#174;M ATP, and 10 <br /> &#226;&#65533;&#174;Ci [&#226;&#65533;&#165;-32P]ATP. Reactions were stopped by adding equal volumes           and Balasubramanian, 1999; Uren et al., 1999; and our un- <br /> of 2&#226;&#171;&#187; SDS-PAGE loading buffer. The samples were boiled and               published results). All of the observed ark1&#226;&#171;&#185; disruption <br /> separated on SDS-PAGE gels, which were then dried for autora-            phenotypes are consistent with a role for Ark1p in chromo- <br /> diography.                                                               some segregation. <br />  <br />  <br /> 1134                                                                                                           Molecular Biology of the Cell <br />                                                                                                       The Fission Yeast Aurora-related Kinase <br />  <br />  <br />  <br />  <br /> Figure 1. ark1&#226;&#171;&#185; is essential for chromosome segregation. (A) One copy of ark1&#226;&#171;&#185; was disrupted in a diploid strain by homologous <br /> recombination. The disruption replaces kinase subdomains I&#226;&#8364;&#8220;VIII with the his3&#226;&#171;&#185; gene. The kinase-encoding region is indicated by a thick line, <br /> and restriction sites are indicated as reference points. (B) ark1&#226;&#171;&#185;/ark1::his3&#226;&#171;&#185; heterozygotes were sporulated and analyzed by tetrad dissection. <br /> Viable:inviable spore ratios of 2:2 were observed on rich medium (YES). Replica plating to minimal medium lacking histidine demonstrated <br /> that viable spores possess wild-type ark1&#226;&#171;&#185;. (C) Purified spores bearing the ark1&#226;&#171;&#185; disruption were allowed to germinate in minimal medium <br /> lacking histidine. After 20 h at 32&#194;&#176;C, cultures were analyzed by DAPI staining. Cells lacking Ark1p exhibited the cut phenotype (arrows) and <br /> often showed stretched or fragmented DNA (bottom left panels). Wild-type cells are shown for comparison (top left panels). <br />  <br />  <br /> Ark1p Alleviates the Slow-Growth Defect of bir1&#226;&#8364;&#8220; 46                       a chromosomal passenger protein (Skoufias et al., 2000; Uren <br /> and Phosphorylates Bir1p In Vitro                                         et al., 2000; Wheatley et al., 2001). The C. elegans survivin <br /> The inhibitor of apoptosis protein survivin was originally                homolog BIR-1 has been reported to play essential roles in <br /> described as a suppressor of programmed cell death, but                   cytokinesis (Fraser et al., 1999). Like Ark1p, the fission yeast <br /> recent work has shed light on its main role as a mitotic                  survivin homolog Bir1p (Pbh1p/Cut17p) is an essential me- <br /> regulator (for reviews, see Reed and Bischoff, 2000, and Silke            diator of chromosome segregation (Samejima et al., 1993; <br /> and Vaux, 2001). In fact, survivin is now known to behave as              Rajagopalan and Balasubramanian, 1999; Uren et al., 1999; <br />  <br />  <br /> Vol. 13, April 2002                                                                                                                       1135 <br />  Leverson et al. <br />  <br />  <br />                                                                                                         Figure 2. Ark1p alleviates the <br />                                                                                                         slow-growth phenotype of bir1&#226;&#8364;&#8220; 46 <br />                                                                                                         and phosphorylates Bir1p in vitro. <br />                                                                                                         (A) Temperature-sensitive strain <br />                                                                                                         bir1&#226;&#8364;&#8220; 46 grows more slowly than <br />                                                                                                         wild-type cells at 28&#194;&#176;C. bir1&#226;&#8364;&#8220; 46 cells <br />                                                                                                         were transformed with a construct <br />                                                                                                         for expressing bir1&#226;&#171;&#185; from its endog- <br />                                                                                                         enous promoter (squares), one for <br />                                                                                                         expressing ark1&#226;&#171;&#185; from the thiamine- <br />                                                                                                         repressible nmt1 promoter (dia- <br />                                                                                                         monds), or with empty nmt1 vector <br />                                                                                                         alone (circles). Transformed cells <br />                                                                                                         were washed extensively in medium <br />                                                                                                         lacking thiamine and grown over- <br />                                                                                                         night at 28&#194;&#176;C to allow for the induc- <br />                                                                                                         tion of Ark1p. The cultures were di- <br />                                                                                                         luted to an OD595 of 0.1 and grown <br />                                                                                                         in parallel at 28&#194;&#176;C. The OD595 of each <br />                                                                                                         culture was measured at various <br /> time points as a measure of cell division. (B) Wild-type and KD Ark1p were purified as GST-fusion proteins and used to perform in vitro <br /> kinase assays. GST-Ark1p (WT) incubated alone became autophosphorylated and migrated as multiple bands (lane 1), whereas KD <br /> GST-Ark1-K147R migrated as a single band (lane 2). Both proteins were also incubated with purified fragments of Bir1p (Bir1p 1&#226;&#8364;&#8220;330 and <br /> Bir1p 1&#226;&#8364;&#8220;208). GST-Ark1p phosphorylated both fragments of Bir1p (arrows), whereas GST-Ark1-K147R did not. The positions of molecular <br /> weight markers (in kDa) are indicated at left. The schematic depicts the domain structure of the Bir1 proteins, with gray boxes representing <br /> the two BIR motifs. <br />  <br />  <br />  <br /> Huang et al., in preparation). Cells lacking Bir1p resemble               protein (our unpublished results), indicating that the kinase <br /> ark1&#226;&#171;&#185;-deleted cells, exhibiting the cut phenotype and other               is indeed defective. As expected, GST-Ark1-K147R failed to <br /> chromosome segregation defects. We thus investigated the                  phosphorylate either fragment of Bir1p (Figure 2B, lane 4). <br /> potential links between Ark1p and Bir1p. <br />    Yeast cells bearing the temperature-sensitive allele bir1&#226;&#8364;&#8220; 46 <br /> grow more slowly than wild-type cells at 28&#194;&#176;C and exhibit                 Overexpression of KD Ark1p Leads to Defects in <br /> severe defects in sister chromatid segregation at 34&#194;&#176;C                    Cytokinesis <br /> (Huang et al., in preparation). Because BIR-1 acts to localize            Because KD versions of other aurora family members have <br /> AIR-2 in C. elegans (Speliotes et al., 2000), we hypothesized             been shown to inhibit cytokinesis, we tested whether Ark1- <br /> that high levels of Ark1p might rescue the defects associated             K147R could exert dominant-negative effects in vivo. Several <br /> with bir1&#226;&#8364;&#8220; 46. Indeed, overexpression of Ark1p alleviated the             versions of Ark1p (Figure 3A) were overexpressed in wild- <br /> slow-growth phenotype observed at 28&#194;&#176;C (Figure 2A). Cells                 type strain FY254 from the nmt1 promoter, which is induced <br /> expressing Ark1p from the full-strength nmt1 promoter                     in the absence of vitamin B1/thiamine (Maundrell, 1993). <br /> grew nearly as well as those expressing wild-type Bir1p and               Each protein was expressed in-frame with three C-terminal <br /> grew much faster than cells transformed with the nmt1                     HA epitope tags to allow for its detection by immunoblot- <br /> vector alone. However, Ark1p overexpression was not suf-                  ting. Although HA-tagged Ark1p was capable of rescuing <br /> ficient to rescue bir1&#226;&#8364;&#8220; 46 at 34&#194;&#176;C (our unpublished results).             ark1&#226;&#171;&#185;-disrupted haploids, the HA-tagged kinase domain <br /> Nevertheless, the former result indicates that Ark1p, when                alone (Ark1-KIN) was not (our unpublished results). Neither <br /> produced at sufficient levels, is capable of fulfilling some              Ark1p nor Ark1-KIN overexpression had any obvious effects <br /> function that is compromised in bir1&#226;&#8364;&#8220; 46 cells.                           on colony formation (Figure 3B). Cells overproducing Ark1- <br />    To determine whether Ark1p could phosphorylate Bir1p                   K147R (KD), however, grew very slowly and could only <br /> in vitro, recombinant Ark1p was isolated as a semipurified                form microcolonies. Microscopic examination revealed that <br /> GST-fusion protein. On incubation with [&#226;&#65533;&#165;-32P]ATP, GST-                   the majority of these cells were elongated and often <br /> Ark1p was autophosphorylated and could be detected as                     branched (Figure 3C). Cells overexpressing Ark1-K147R in <br /> multiple bands on autoradiographs (Figure 2B, lane 1). Like               liquid medium were also elongated, and many possessed <br /> other aurora family members, GST-Ark1p is able to phos-                   multiple division septa (Figure 3C). An unusually large <br /> phorylate myelin basic protein but not histone H1 (our                    percentage (&#226;&#172;&#402;40% compared with &#226;&#172;&#402;10% in typical wild- <br /> unpublished results). GST-Ark1p also phosphorylated a                     type cultures) of cells were singly septated and binucleate. <br /> semipurified, His-tagged Bir1p fragment (residues 1&#226;&#8364;&#8220;330)                  Chromosome segregation appears to occur normally in these <br /> and strongly phosphorylated a shorter breakdown product                   cells, and even cells with multiple septa exhibit centrally <br /> (residues 1&#226;&#8364;&#8220;208) that retains the two BIR domains (Figure                 localized DAPI-staining masses on either side of a given <br /> 2B, lane 3). As a negative control, we constructed a kinase-              septum. Ark1-K147R thus appears to be acting after chro- <br /> defective (KD) form of Ark1p, with lysine 147 in kinase                   mosome segregation to inhibit cell division in a dominant- <br /> subdomain II mutated to arginine (Hanks and Hunter, 1995).                negative manner. <br /> When analyzed for kinase activity in vitro, GST-Ark1-K147R                   Interestingly, a KD mutant lacking the first 116 amino <br /> generated only a single weak autophosphorylated band                      acids (Figure 3A, KIN-KD) did not affect cell growth (Figure <br /> (Figure 2B, lane 2) and failed to phosphorylate myelin basic              3B), suggesting that the N-terminal, nonkinase domain of <br />  <br />  <br /> 1136                                                                                                            Molecular Biology of the Cell <br />                                                                                               The Fission Yeast Aurora-related Kinase <br />  <br />  <br /> Figure 3. The Ark1-K147R <br /> dominant-negative mutant inhib- <br /> its cytokinesis. (A) Schematic of <br /> wild-type and mutant versions of <br /> Ark1p expressed from the thia- <br /> mine-repressible nmt1 promoter. <br /> The catalytic kinase domain <br /> (CAT) is indicated as a gray box, <br /> and the putative KEN box is <br /> shown as a hatched box. Vect, <br /> empty nmt1 vector; Ark1, wild- <br /> type Ark1p; KD, KD Ark1- <br /> K147R; NT, N-terminal fragment <br /> of Ark1p (1&#226;&#8364;&#8220;116); KIN, kinase do- <br /> main of Ark1p (117&#226;&#8364;&#8220;384); KIN- <br /> KD, KD kinase domain of Ark1p. <br /> An immunoblot performed with <br /> anti-HA mAb 12CA5 (right) <br /> demonstrates that similar levels <br /> of wild-type Ark1, the KD kinase <br /> domain, and the N-terminal frag- <br /> ment were expressed. Molecular <br /> weight markers (in kDa) are in- <br /> dicated at left. (B) Ark1 proteins <br /> described in A were overex- <br /> pressed in wild-type cells grown <br /> on solid medium lacking thia- <br /> mine (&#226;&#171;&#186;B1). Cells expressing KD <br /> Ark1p grew poorly, forming only <br /> microcolonies, whereas cells expressing the N-terminal domain (NT) or the KD kinase domain alone (KIN-KD) were unaffected. (C) Cells <br /> expressing Ark1-KD on solid medium were elongated and branched (bottom left). The same cells grown in inducing liquid medium possess <br /> multiple septa (center) and multiple nuclei (right), as revealed by calcofluor and DAPI staining, respectively. <br />  <br />  <br /> Ark1p is required for the dominant-negative effect. This            384-KD and 72&#226;&#8364;&#8220;384-KD were exposing a functional degron, <br /> region could feasibly serve to mediate protein&#226;&#8364;&#8220;protein inter-       the proteins would no longer exert a negative effect simply <br /> actions, and we reasoned that expressing it alone (Figure 3A,       because they were unstable. However, 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220; <br /> NT) might be sufficient to inhibit cytokinesis. NT, however,        384-KD bearing the mutations KENKRTSNSK to <br /> did not affect cell growth (Figure 3B), although significant        AAAKRTSNSA were unable to inhibit cell growth (Figure <br /> levels of the mutant were expressed (Figure 3A). Thus, the          4B). Moreover, these mutants were stably expressed regard- <br /> KD kinase domain, along with some N-terminal sequence, is           less of whether the KEN box was mutated or not (Figure 4C <br /> required for the dominant-negative effect.                          and our unpublished results). Constitutive expression of <br />   To identify the N-terminal regions that are required for          full-length Ark1p with the KEN box mutations also had no <br /> the dominant-negative effect, we constructed a series of            effect on the ability of cells to form colonies (Figure 4B). <br /> truncation mutants, each bearing the K147R <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>mutation</span> (Fig-           Thus, the Ark1p KEN sequence does not appear to be re- <br /> ure 4A). Each mutant was again expressed from the nmt1              sponsible for the failure of 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD to act <br /> promoter in wild-type cells. As before, full-length Ark1-           as dominant-negatives. <br /> K147R exerted a dominant-negative effect, whereas its ki- <br /> nase domain alone (117-384 KD) did not (Figure 4B). Trun- <br /> cation mutant 24 &#226;&#8364;&#8220;384-KD retained dominant-negative                 Ark1p Levels Remain Constant throughout the Cell <br /> activity, but mutants 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD did not,             Cycle <br /> suggesting to us that residues 24 &#226;&#8364;&#8220; 47 play a crucial role in       Although our data indicated that the KEN sequence is not <br /> mediating the effect. However, colony formation was also            playing a role in regulating Ark1p levels, we nevertheless <br /> inhibited by mutant 88 &#226;&#8364;&#8220;384-KD, demonstrating that resi-            decided to monitor Ark1p levels throughout the cell cycle. <br /> dues 88 &#226;&#8364;&#8220;116 are necessary and sufficient to restore domi-          We engineered temperature-sensitive strain cdc25&#226;&#8364;&#8220;22 to ex- <br /> nant-negative activity to the KD kinase domain.                     press a C-terminally HA epitope&#226;&#8364;&#8220;tagged version of Ark1p <br />   The inability of 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD to act as dom-          from its endogenous promoter. The strain was blocked in G2 <br /> inant-negatives seemed to be a function of residues 48 &#226;&#8364;&#8220; 87,        at 35&#194;&#176;C and then released synchronously into mitosis by <br /> which are absent in 88 &#226;&#8364;&#8220;384-KD. The amino acids KEN-                shifting to 25&#194;&#176;C. Samples were taken every 20 min to pre- <br /> KRTSNSK (78 &#226;&#8364;&#8220; 87) are conspicuous in this region, because           pare cell extracts and to determine the percentage of sep- <br /> similar N-terminal sequences are found in most of the other         tated cells as a measure of synchrony. Although the cultures <br /> aurora family members (see Bischoff and Plowman, 1999).             progressed synchronously through the cell cycle, anti-HA <br /> This sequence also resembles the recently described &#226;&#8364;&#339;KEN            immunoblotting revealed that Ark1p levels did not vary <br /> box&#226;&#8364;&#65533; (Pfleger and Kirschner, 2000), which has been shown to         significantly over the course of our experiment (Figure 4D). <br /> act as a recognition sequence for the anaphase-promoting            Ark1p levels thus appear to remain relatively constant <br /> complex (APC) ubiquitin ligase. If the truncations in 48 &#226;&#8364;&#8220;          throughout the cell cycle. These data, along with the obser- <br />  <br />  <br /> Vol. 13, April 2002                                                                                                            1137 <br />  Leverson et al. <br />  <br />  <br />                                                                                                          Figure       4. Structure-function <br />                                                                                                          analysis of the Ark1-K147R dom- <br />                                                                                                          inant-negative mutant. (A) A se- <br />                                                                                                          ries of N-terminally truncated <br />                                                                                                          Ark1-K147R mutants was tested <br />                                                                                                          for the ability to inhibit wild-type <br />                                                                                                          cell growth. The residues present <br />                                                                                                          in each mutant are indicated at <br />                                                                                                          left. To test the role of a putative <br />                                                                                                          KEN box, comprising residues <br />                                                                                                          78 &#226;&#8364;&#8220; 87 (hatched box), the KEN- <br />                                                                                                          KRTSNSK sequence was mu- <br />                                                                                                          tated to AAAKRTSNSA (black <br />                                                                                                          dots). (B) Ark1 proteins de- <br />                                                                                                          scribed in A were overexpressed <br />                                                                                                          in wild-type cells from the thia- <br />                                                                                                          mine-repressible nmt1 promoter. <br />                                                                                                          Cells expressing KD Ark1p (KD) <br />                                                                                                          grew poorly on solid medium <br />                                                                                                          lacking thiamine, whereas cells <br />                                                                                                          expressing the KD kinase do- <br />                                                                                                          main alone (117&#226;&#8364;&#8220;384-KD) were <br />                                                                                                          not affected. Truncation mutant <br />                                                                                                          88 &#226;&#8364;&#8220;384-KD behaves identically to <br />                                                                                                          the full-length KD mutant, indi- <br />                                                                                                          cating that residues 88 &#226;&#8364;&#8220;116 are <br />                                                                                                          sufficient to restore dominant- <br />                                                                                                          negative activity to the kinase do- <br />                                                                                                          main. Mutants 48 &#226;&#8364;&#8220;384-KD and <br />                                                                                                          72&#226;&#8364;&#8220;384-KD have no effect on cell <br /> growth, whereas 24 &#226;&#8364;&#8220;384-KD again inhibits colony formation. Wild-type cells were also unaffected by the expression of KEN box&#226;&#8364;&#8220;mutated <br /> versions of 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD. (C) Extracts were prepared from liquid cultures of cells expressing HA-tagged Ark1p or truncation <br /> mutants KEN-48 &#226;&#8364;&#8220;384-KD, KEN-72&#226;&#8364;&#8220;384-KD, or 117&#226;&#8364;&#8220;384-KD, and then analyzed by immunoblotting with anti-HA mAb 12CA5. Molecular <br /> weight markers (in kDa) are indicated at left. (D) Temperature-sensitive strain cdc25&#226;&#8364;&#8220;22 was engineered to express an HA epitope&#226;&#8364;&#8220;tagged <br /> version of Ark1p from its endogenous promoter. A liquid culture of this strain was blocked in G2 by incubation at 35&#194;&#176;C, then released to <br /> undergo synchronous cell division at 25&#194;&#176;C. Samples were taken every 20 min to prepare cell extracts, which were then analyzed by <br /> SDS-PAGE and immunoblotting with anti-HA antiserum. The percentage of septated cells was determined at each time point as a measure <br /> of the culture&#226;&#8364;&#8482;s synchrony. <br />  <br />  <br />  <br /> vation that constitutive expression of Ark1p appears to have             INCENP homologues (Adams et al., 2000; Kaitna et al., 2000; <br /> no serious consequences (Figure 3B), indicate that total                 Adams et al., 2001). Ark1-K147R binding to Pic1p residues <br /> Ark1p levels do not need to be tightly regulated during the              925-1018 was confirmed in direct two-hybrid pairwise tests <br /> cell cycle.                                                              (Figure 5A), and an analysis of additional Pic1p fragments <br />                                                                          revealed that residues 925&#226;&#8364;&#8220;972 are necessary and sufficient <br /> Ark1 Binds to Pic1p, the Fission Yeast Inner                             for Ark1p binding (Figure 5A). No pic1&#226;&#171;&#185; clones were iden- <br />                                                                          tified in screens that used the pGBT9-NT bait, and consistent <br /> Centromere Protein                                                       with this observation, the N-terminal bait failed to bind the <br /> We next performed a yeast two-hybrid screen in an attempt                IN box in direct pairwise tests (Figure 5B). A reciprocal <br /> to identify potential Ark1p regulators, adaptors, and/or                 experiment in which full-length Pic1p was used as bait <br /> substrates. The full-length, KD mutant Ark1-K147R was                    demonstrated that Pic1p binds to the Ark1p kinase domain <br /> used as bait in the hope of identifying proteins related to its          (residues 117&#226;&#8364;&#8220;384) but not to the N-terminal extension alone <br /> dominant-negative effect. Because a portion of the N-termi-              (Figure 5B). The Pic1p-Ark1p kinase domain interaction ap- <br /> nal region of Ark1-K147R is clearly required for its domi-               peared to be weaker than that observed between Ark1p or <br /> nant-negative effect (Figure 4), a bait comprising residues              the Ark1p kinase domain and the minimal IN box (Figure 5B <br /> 1&#226;&#8364;&#8220;116 (pGBT9-NT) was also used. With the Ark1-K147R bait,                and our unpublished results). This may be because the IN <br /> a screen of &#226;&#172;&#381;1.5 &#226;&#171;&#187; 107 transformants yielded 42 his&#226;&#171;&#185; ade&#226;&#171;&#185;                box is less accessible in full-length Pic1p. In addition, Ark1p <br /> colonies, 16 of which also scored positive for &#226;&#65533;&#164;-galactosidase           residues 88 &#226;&#8364;&#8220;116, which are required for the dominant-neg- <br /> activity. DNA sequencing revealed that six of the library                ative effect of Ark1-K147R, could be required for strong <br /> clones expressed C-terminal fragments of the S. pombe ho-                binding to full-length Pic1p. <br /> molog of INCENP, which we will refer to as Pic1p (S. pombe <br /> inner centromere protein). Four of these clones expressed <br /> Pic1p residues 765-1018 in-frame with the GAL4 activation                The Pic1p IN Box Acts as a Dominant-Negative <br /> domain, whereas the other two expressed only residues                    Inhibitor of Cytokinesis <br /> 925-1018 (Figure 5A). The latter piece corresponds to the                Mice homozygous for an INCENP disruption die early in <br /> highly conserved &#226;&#8364;&#339;IN box&#226;&#8364;&#65533; previously described in other                  embryogenesis with severe defects in chromosome segrega- <br />  <br />  <br /> 1138                                                                                                          Molecular Biology of the Cell <br />                                                                                            The Fission Yeast Aurora-related Kinase <br />  <br />  <br /> Figure 5. Ark1-K147R inter- <br /> acts with the inner centromere <br /> protein Pic1p. (A) Two C-termi- <br /> nal fragments of the S. pombe <br /> inner centromere protein Pic1p <br /> were identified as Ark1-K147R <br /> binding partners in a yeast two- <br /> hybrid screen. The first frag- <br /> ment      comprises     residues <br /> 765-1018, whereas the second <br /> includes only residues 925-1018. <br /> Both fragments contain the IN <br /> box (indicated by a black box), <br /> which is highly conserved <br /> among INCENPs (Adams et al., <br /> 2000). The Ark1-K147R bait <br /> (DBD) and various Pic1p frag- <br /> ment prey (AD) constructs were <br /> used to transform yeast two-hy- <br /> brid strain AH109. Cotransfor- <br /> mants were streaked to selec- <br /> tive medium to test for <br /> interactions. Pic1p residues <br /> 925&#226;&#8364;&#8220;972 were necessary and <br /> sufficient for Ark1p binding. (B) <br /> Various Ark1p and Pic1p bait <br /> (DBD) and prey (AD) constructs <br /> were used to transform strain <br /> AH109. Cotransformants were <br /> streaked to selective medium to <br /> test for interactions. Although <br /> Ark1-K147R bound to Pic1p <br /> 925-1018, the N-terminal domain alone (residues 1&#226;&#8364;&#8220;116) did not. In reciprocal experiments, the wild-type kinase domain of Ark1p <br /> (residues 117&#226;&#8364;&#8220;384) was sufficient to bind full-length Pic1p. <br />  <br />  <br /> tion, microtubule bundling, and cytokinesis (Cutts et al.,        ure 6C). To test whether the dominant-negative effect of the <br /> 1999). Dominant-negative versions of INCENP have also             IN box depended on its ability to sequester endogenous <br /> been described, including the C-terminally truncated mu-          Ark1p, we next tested whether concomitant overexpression <br /> tant INCENP1&#226;&#8364;&#8220; 405, which causes defects in chromosome seg-        of wild-type Ark1p would alleviate the effect. Indeed, wild- <br /> regation and cytokinesis (Mackay et al., 1998). Another mu-       type cells overexpressing both the IN box and Ark1p grew <br /> tant, made by fusing INCENP to the centromere-targeting           normally on solid medium (Figure 6C) and appeared normal <br /> region of CENP-B, can no longer localize to the midbody late      (our unpublished results). Our interaction data and overex- <br /> in mitosis and causes defects in cytokinesis (Eckley et al.,      pression studies thus strongly indicate that the effects of <br /> 1997). The defects caused by both mutants may be a result of      Ark1-K147R are mediated through stable Pic1p binding. <br /> improperly targeting aurora-B. We reasoned that overpro-          Such an interaction would be likely to compete away wild- <br /> ducing the Pic1p IN box would also have negative conse-           type Ark1p, denying it access to crucial substrates. <br /> quences, because it is sufficient to bind Ark1p but is unlikely     To further examine this idea, the various KD Ark1p trun- <br /> to localize properly. Indeed, fission yeast overexpressing        cation mutants (Figure 4) were used as bait in the two- <br /> Pic1p fragment 925-1018 exhibited phenotypes very similar         hybrid system to assess their ability to bind the Pic1p IN <br /> to those seen with Ark1-K147R. These cells grew poorly on         box. Although dominant-negative mutants 24 &#226;&#8364;&#8220;384-KD and <br /> solid medium (Figure 6A) and were elongated and often             88 &#226;&#8364;&#8220;384-KD bound to the IN box as well as full-length Ark1p <br /> branched. Likewise, cells grown in inducing liquid medium         (Figure 6D), the other mutants bound poorly (72&#226;&#8364;&#8220;384-KD) or <br /> were elongated and branched and exhibited multiple septa          not at all (48 &#226;&#8364;&#8220;384-KD). The ability of these mutants to confer <br /> (Figure 6B). These cells also occasionally exhibited the cut      a dominant-negative effect in wild-type S. pombe thus corre- <br /> phenotype (our unpublished results), which is the primary         sponds directly to their ability to bind to Pic1p, indicating <br /> phenotype of cells deleted for pic1&#226;&#171;&#185; (our unpublished re-         that Pic1p-Ark1p complex formation is essential for execut- <br /> sults).                                                           ing cytokinesis in fission yeast. <br />    To determine whether Ark1p binding was required for the <br /> dominant-negative effect of this fragment, we next overex- <br /> pressed larger C-terminal fragments of Pic1p with or with-        DISCUSSION <br /> out the IN box (Figure 6C). As expected, Pic1&#226;&#8364;&#8220;765-1018 <br /> bound to Ark1p in two-hybrid pairwise tests and exerted a         This work describes the initial characterization of the S. <br /> dominant-negative effect on cell growth (Figure 6C). Pic1&#226;&#8364;&#8220;        pombe aurora-related kinase, Ark1p. We have shown that <br /> 765-924, which lacks the IN box, failed to bind Ark1p, and        ark1&#226;&#171;&#185; is an essential gene required for the proper segrega- <br /> fission yeast overproducing this mutant grew normally (Fig-       tion of sister chromatids in mitosis. Cells lacking Ark1p <br />  <br />  <br /> Vol. 13, April 2002                                                                                                         1139 <br />  Leverson et al. <br />  <br />  <br />                                                                                                      Figure 6. Ark1p-Pic1p complexes are <br />                                                                                                      required for the completion of cytokine- <br />                                                                                                      sis. (A) Ark1p and Pic1p mutants were <br />                                                                                                      overexpressed in wild-type cells. Vect, <br />                                                                                                      empty nmt1 vector; Ark1, wild-type <br />                                                                                                      Ark1p; KEN, Ark1p bearing four muta- <br />                                                                                                      tions in the putative KEN box (see text); <br />                                                                                                      KD, KD Ark1-K147R; NT, N-terminal <br />                                                                                                      fragment of Ark1p (1&#226;&#8364;&#8220;116); KIN, kinase <br />                                                                                                      domain of Ark1p (117&#226;&#8364;&#8220;384); KIN-KD, <br />                                                                                                      KD kinase domain of Ark1p; Pic1 IN, <br />                                                                                                      Pic1p IN box (925&#226;&#8364;&#8220;1018). Transformants <br />                                                                                                      isolated on media containing thiamine <br />                                                                                                      (B1) were streaked to medium with <br />                                                                                                      (&#226;&#171;&#185;B1) or without (&#226;&#171;&#186;B1) thiamine and <br />                                                                                                      grown at 32&#194;&#176;C for 3 d. Wild-type cells <br />                                                                                                      overexpressing the Pic1p Ark1p-bind- <br />                                                                                                      ing IN box (925&#226;&#8364;&#8220;1018) grew poorly and <br />                                                                                                      formed only microcolonies. (B) The <br />                                                                                                      Pic1p IN box was overexpressed in <br />                                                                                                      wild-type cells grown in liquid medium <br />                                                                                                      lacking thiamine. DAPI/calcofluor <br />                                                                                                      staining revealed that DNA segregated <br />                                                                                                      properly but that many cells failed to <br />                                                                                                      complete cytokinesis and were multiply <br />                                                                                                      septated. (C) Pic1p fragments 765-1018, <br />                                                                                                      925-1018, and 765&#226;&#8364;&#8220;924 were tested in <br />                                                                                                      parallel for the ability to bind Ark1- <br />                                                                                                      K147R in direct two-hybrid tests (left <br />                                                                                                      plate) and to act as dominant-negative <br />                                                                                                      mutants when overexpressed in wild- <br />                                                                                                      type S. pombe (center plate). Only frag- <br />                                                                                                      ments 765-1018 and 925-1018, which <br />                                                                                                      possess the IN box, bound to Ark1p and <br />                                                                                                      acted as dominant-negatives. Wild-type <br />                                                                                                      cells overexpressing the dominant-neg- <br />                                                                                                      ative Pic1p IN box (925&#226;&#8364;&#8220;1018) were res- <br />                                                                                                      cued by coexpressing Ark1p, but not by <br />                                                                                                      empty vector (right plate). (D) KD <br />                                                                                                      Ark1p truncation mutants (left) were <br />                                                                                                      used as bait in direct two-hybrid tests <br /> for binding to the Pic1p IN box (residues 925-1018). Yeast transformed with the indicated constructs were assayed for growth on selective <br /> media (right) as an indicator of protein&#226;&#8364;&#8220;protein interactions. Ark1p mutants that failed to bind Pic1p were incapable of exerting dominant- <br /> negative effects. <br />  <br />  <br />  <br /> contain fragmented nuclei and often exhibit the cut pheno-               Ark1/Aurora and Bir1/Survivin <br /> type, indicating that these cells are defective in some aspect           We found that ark1&#226;&#171;&#185; overexpression is capable of rescuing the <br /> of chromosome condensation and/or migration. We also                     bir1&#226;&#8364;&#8220; 46 slow-growth phenotype at 28&#194;&#176;C, indicating that these <br /> found that overexpressing a KD version of Ark1p inhibits                 genes are functionally related. Ark1p was also capable of phos- <br /> cell division, indicating that Ark1p plays an additional role            phorylating Bir1p in vitro, which suggests that these proteins <br /> in regulating cytokinesis. Ark1p thus functions at multiple              are capable of interacting at least transiently. Ark1p is unlikely <br /> points in the cell cycle and may serve to coordinate chromo-             to be the only crucial factor downstream of Bir1p, however, <br /> some segregation with cytokinesis.                                       because high levels of Ark1p fail to rescue the temperature- <br />   Ark1p interacts, functionally and/or physically, with                  sensitive defects of bir1&#226;&#8364;&#8220; 46 at 34&#194;&#176;C. Ark1p phosphorylates the <br /> Bir1p and Pic1p. Purified Ark1p phosphorylates N-terminal                N-terminal region of Bir1p, which contains two BIR domains. <br /> Bir1p fragments in vitro, and overexpressed Ark1p allevi-                The single BIR motif of human survivin mediates homo-dimer- <br /> ates the slow-growth phenotype of the bir1&#226;&#8364;&#8220; 46 temperature-              ization (Chantalat et al., 2000; Muchmore et al., 2000; Verdecia et <br /> sensitive strain. Pic1p was identified as an Ark1-K147R in-              al., 2000), and so Ark1p could serve to regulate Bir1p dimer- <br /> teracting protein, and overexpression of its C-terminal                  ization. Ark1p might also be required to trigger the movements <br /> Ark1p-binding domain leads to cytokinesis defects identical              of Bir1p during mitosis. The transfer of survivin from chromo- <br /> to those produced by Ark1-K147R. Ark1-K147R truncation                   somes to the centromere, midzone, and cell cortex seems to <br /> mutants that fail to bind Pic1p also fail to act as dominant-            depend on aurora activity, because these migrations are inhib- <br /> negatives, indicating that Ark1p-Pic1p complexes are re-                 ited by INCENP1&#226;&#8364;&#8220; 405, which lacks the aurora-binding IN box <br /> quired for the completion of cytokinesis in fission yeast.               (Wheatley et al., 2001). Future studies will be aimed at deter- <br />  <br />  <br /> 1140                                                                                                           Molecular Biology of the Cell <br />                                                                                                  The Fission Yeast Aurora-related Kinase <br />  <br />  <br /> mining whether Ark1p phosphorylates Bir1p in vivo and what            ra-A associates with Cdc20 in HeLa cells (Farruggio et al., <br /> the functional significance of such modifications might be.           1999) and appears to be degraded in an APC-dependent <br />    Human aurora-B was recently shown to bind, but not phos-           manner (Honda et al., 2000; Walter et al., 2000). Although <br /> phorylate, survivin (Wheatley et al., 2001). Conversely, we           aurora-A is not stabilized by mutations in putative destruc- <br /> found that Ark1p is able to phosphorylate Bir1p in vitro but          tion boxes, the role of the KEN box has not been investi- <br /> failed to detect Ark1p-Bir1p interactions in our two-hybrid           gated. <br /> screens or in direct pairwise tests (our unpublished results). It        Our studies indicate that Ark1p levels do not need to be <br /> is possible that another, as yet unidentified, protein might be       tightly regulated. Overexpression of Ark1p does not signifi- <br /> required to bridge, physically and/or functionally, between           cantly affect cell growth (Figure 6A), and we failed to detect cell <br /> Ark1p and Bir1p in fission yeast. Survivin orthologs are quite        cycle&#226;&#8364;&#8220; dependent changes in Ark1p levels (Figure 4C). Al- <br /> divergent, and although human survivin can partially substi- <br />                                                                       though we cannot rule out the possibility that specific sub- <br /> tute for C. elegans BIR-1 (Speliotes et al., 2000), it is unable to <br />                                                                       populations of Ark1p are targeted for destruction, the KEN <br /> rescue budding yeast deleted for BIR1 (Li et al., 2000) or fission <br /> yeast deleted for bir1&#226;&#171;&#185; (Huang et al., in preparation), indicating    sequence does not appear to function as a degron. This may be <br /> that these proteins function in slightly different manners. Like      simply because the Ark1p sequence does not conform closely <br /> budding yeast Bir1p, fission yeast Bir1p is much larger than          enough to the KEN box consensus (Burton and Solomon, 2001). <br /> survivin and, outside of its N-terminal BIR domains, shows               In the course of performing a structure-function analysis of <br /> only limited homology to it. The C-terminal extension in bud-         the Ark1-K147R mutant, we identified mutant 88 &#226;&#8364;&#8220;384-KD as <br /> ding yeast Bir1p has been shown to mediate association with           the minimal <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> capable of exerting dominant-negative activ- <br /> the spindle apparatus in anaphase (Uren et al., 1999) and also        ity. Residues 88 &#226;&#8364;&#8220;116 must also be appended to the functional <br /> with the kinetochore protein Ndc10p (Yoon and Carbon, 1999).          kinase domain to rescue ark1&#226;&#171;&#185;-disrupted haploids (our unpub- <br /> Ndc10p homologues have not been identified in mammals, so             lished results). Although these residues are not absolutely re- <br /> it clearly is possible that distinct Bir1 intermolecular interac-     quired for Pic1p binding, our results indicate that they do play <br /> tions have evolved in various organisms. It will be important to      an essential role and perhaps mediate interactions with other <br /> determine whether the yeast Bir1 proteins also dimerize and           Ark1p partners and/or substrates. Surprisingly, KD mutants <br /> exactly which proteins bind to their various domains in vivo.         with additional N-terminal sequence (residues 48 &#226;&#8364;&#8220; 87 or 72&#226;&#8364;&#8220; 87) <br />                                                                       did not behave as dominant-negatives. This raised the possi- <br /> Ark1p/Aurora and Pic1p/INCENP                                         bility that residues 72&#226;&#8364;&#8220; 87 function as an autoinhibitory do- <br />                                                                       main, and indeed, mutants 48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD failed <br /> We identified Pic1p in an unbiased yeast two-hybrid screen <br />                                                                       to bind Pic1p in direct two-hybrid tests, whereas mutant 88 &#226;&#8364;&#8220; <br /> for Ark1-K147R interacting proteins. C-terminal fragments <br />                                                                       384-KD bound quite strongly. Mutant 24 &#226;&#8364;&#8220;384-KD also bound <br /> of Pic1p that include the highly conserved IN box (Adams et <br /> al., 2000) were sufficient for Ark1p binding. The correspond-         Pic1p, indicating that residues 24 &#226;&#8364;&#8220; 47 may function to relieve <br /> ing region in murine INCENP is sufficient to bind human               the inhibitory effect of residues 72&#226;&#8364;&#8220; 87. Thus, whereas <br /> aurora kinases from HeLa cell extracts (Kaitna et al., 2000),         24 &#226;&#8364;&#8220;384-KD may represent a fully regulatable Ark1 protein, <br /> and a C-terminal piece of C. elegans ICP-1 could also bind to         48 &#226;&#8364;&#8220;384-KD and 72&#226;&#8364;&#8220;384-KD may be constitutively inhibited, i.e., <br /> AIR-2, albeit weakly. We have further defined the Ark1p-              unable to bind Pic1p. <br /> Pic1p interacting domains, demonstrating that Pic1p resi-                A recent study of the Xenopus laevis aurora-A kinase revealed <br /> dues 925&#226;&#8364;&#8220;972 are sufficient for Ark1p binding and most                that its N-terminal extension plays a role in localizing it to <br /> likely make contact with the kinase domain (Figures 5 and             centrosomes (Giet and Prigent, 2001). This domain behaves as <br /> 6). Overexpressing Ark1p-binding fragment 925-1018 led to             a dominant-negative mutant in Xenopus egg extracts, acting to <br /> cytokinesis defects that were indistinguishable from those            inhibit bipolar spindle formation and to destabilize assembled <br /> seen with Ark1-K147R, whereas overexpressing a fragment               spindles. In our studies, overproducing the Ark1p N-terminal <br /> that fails to bind Ark1p had no effect. Furthermore, the              domain had no discernible effect on cell growth (Figures 3 and <br /> ability of truncated Ark1-K147R mutants to inhibit cytoki-            6), indicating that it probably does not interfere with wild-type <br /> nesis required Pic1p binding. These data strongly indicate            Ark1p function. <br /> that wild-type Ark1p-Pic1p complexes are required for the <br /> completion of cell division. <br />  <br />                                                                       <br /> What Is the Role of the Ark1 N-Terminus? <br /> Our data indicate that INCENPs probably bind to the highly           <br />                                                                       <br /> conserved kinase domains of aurora family members. Out- <br />                                                                        <br /> side of the kinase domain, aurora family members exhibit <br />                                                                        <br /> only limited homology. Their N-terminal extensions range             <br /> from 7 to 162 amino acids (for review, see Giet and Prigent,         <br /> 1999), and several possess a sequence that resembles the              <br /> recently described KEN box (Pfleger and Kirschner, 2000).             <br /> The KEN box was originally identified in the APC activator            <br /> and substrate Cdc20 (Pfleger and Kirschner, 2000) and acts            <br /> as a degron that is bound by APC-Hct1/Cdh1 or APC-                    <br /> Cdc20 complexes (Burton and Solomon, 2001; Hilioti et al.,          <br /> 2001; Pfleger et al., 2001; Schwab et al., 2001). Human auro-        <br /> </body></html>