<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />               Cdc1 and the Vacuole Coordinately Regulate Mn21 Homeostasis <br />                            in the Yeast Saccharomyces cerevisiae <br />  <br />   <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The yeast CDC1 gene encodes an essential protein that has been implicated in the regulation of cytosolic <br />               [Mn2 1]. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> identify factors that impinge upon Cdc1 or the Cdc1-dependent process, we isolated second- <br />               site suppressors of the conditional cdc1-1(Ts) growth defect. Recessive suppressors define 15 COS (CdcOne <br />               Suppressor) genes. Seven of the fifteen COS genes are required for biogenesis of the vacuole, an organelle <br />               known to sequester intracellular Mn21 . An eighth gene, COS16, encodes a vacuolar membrane protein <br />               that seems to be involved in Mn2 1 homeostasis. These results suggest mutations that block vacuolar Mn2 1 <br />               sequestration compensate for defects in Cdc1 function. Interestingly, Cdc1 is dispensable in a cos16D <br />               deletion strain, and a cdc1D cos16D double mutant exhibits robust growth on medium supplemented with <br />               Mn2 1. Thus, the single, essential function of Cdc1 is to regulate intracellular, probably cytosolic, Mn21. <br />  <br />  <br />  <br /> C     ELLS of the yeast Saccharomyces cerevisiae divide by <br />        budding. Bud development is a complex process <br /> that appears to require the metal ion Mn21. Early studies <br />                                                                            glycosylation has been shown to affect bud growth. Mu- <br />                                                                            tants of och1D lack a Mn21-dependent Golgi glycosyl- <br />                                                                            transferase and exhibit a conditional defect in bud <br /> with ion chelators and Ca21 ionophores (Iida et al. 1990)                  growth, but Och1 is not essential under standard growth <br /> suggested that Ca21 was essential for cell growth and                      conditions (Nagasu et al. 1992). Thus, it is not clear <br /> proliferation, but a subsequent report suggested Mn21,                     if Och1, or even Golgi Mn21 , is related to the Mn21 <br /> not Ca21, was the limiting ion (Youatt and McKinnon                        requirement identified by the depletion studies. <br /> 1993). Yeast cells depleted of both Mn21 and Ca21 ex-                         Several proteins have been implicated in intracellular <br /> hibit a defect in bud growth, and Mn21 is 500&#226;&#8364;&#8220;1000-fold                    Mn21 homeostasis. Mn21 uptake across the plasma mem- <br /> more effective than Ca21 in reversing the growth defect,                   brane appears to be mediated by Smf1, an integral <br /> suggesting that Mn21 is the physiologically important                      plasma membrane protein (Supek et al. 1996). A second <br /> ion (Loukin and Kung 1995).                                                protein, Smf2, shares significant homology with Smf1 <br />    Yeast Mn21 is present in several subcellular compart-                   and might also be involved in Mn21 uptake (Eide and <br /> ments, any one of which might represent, or influence,                     Guerinot 1997). Of the intracellular organelles, only <br /> the Mn21 pool essential for bud growth. The cytosol                        the Golgi has been characterized in any detail with re- <br /> contains Mn21 as well as such Mn21 -dependent enzymes                      spect to Mn21 flux. Two structurally unrelated proteins, <br /> as pyruvate carboxylase, glutamine synthetase, and argi-                   Pmr1 (a Ca21-ATPase homologue) and Ccc1, seem to <br /> nase (Wedler 1994). Mn21 is also present in the Golgi,                     play a role in the transport of Mn21 <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the cytosol <br /> where it activates glycosyltransferases that are involved                  into the Golgi (Lapinskas et al. 1995; Lapinskas et al. <br /> in the processing of secreted proteins (Wedler 1994).                      1996). A third protein, Atx2, has been implicated in <br /> Mitochondrial Mn21 is required by enzymes of the citric                    the release of Golgi Mn21 into the cytosol (Lin and <br /> acid cycle (Wedler 1994) as well as proteases involved                     Culotta 1996). Although two of these proteins (Pmr1 <br /> in mitochondrial protein import (Supek et al. 1996).                       and Smf1) are essential for growth under Mn21 -limiting <br /> Finally, Mn21 is found in the yeast vacuole (Okorokov                      conditions, the phenotypes of Mn21-depleted pmr1 or <br /> et al. 1977), an acidic, membrane-bound organelle that                     smf1 strains have not been reported. Thus, it is not clear <br /> has been implicated in Mn21 detoxification. Of these                       if the activity of either protein affects the Mn21 pool <br /> known Mn21-dependent enzymes and processes, only                           that is required for bud growth. <br />                                                                               One factor that does affect the growth-related Mn21 <br />                                                                            pool is the essential gene CDC1. Conditional cdc1(Ts) <br />   Corresponding author: Stephen Garrett, Department of Microbiology        mutants were originally identified on the basis of a de- <br /> and Molecular Genetics, UMDNJ-New Jersey Medical Center, 185               fect in bud growth (Hartwell et al. 1970), and several <br /> South Orange Ave., University Heights, Newark, NJ 07103-2714. <br /> E-mail: garretst@umdnj.edu <br />                                                                            lines of evidence implicate Cdc1 in the regulation of <br />   1 <br />     Present address: Department of Biochemistry, University of Connecti-   cytosolic Mn21. First, Mn21 supplement partially sup- <br /> cut Health Center, Farmington, CT 06030.                                   presses the cdc1(Ts) growth defect (Loukin and Kung <br />  <br /> Genetics 148: 1787&#226;&#8364;&#8220;1798 (April, 1998) <br />  1788                                                M. Paidhungat and S. Garrett <br />  <br /> 1995; Paidhungat and Garrett 1998), suggesting that                 lated by patching independent colonies of FY11 and FY12 <br /> Mn21 is limiting in cdc1 mutants. Second, defects in                onto YEPD agar plates, which were then incubated at either <br />                                                                     308 or 368 for 2 days. Temperature-resistant revertants arose <br /> CDC1 cause cells to become sensitive to depletion of                as papillae from the patches; a single papilla was picked from <br /> cytosolic Mn21 (Supek et al. 1996; Paidhungat and Gar-              each patch to ensure that revertants were independent. At <br /> rett 1998). Finally, CDC1 overexpression makes certain              238, two revertants gave rise to small, temperature-resistant <br /> chelator-sensitive mutants more tolerant to Mn21 deple-             colonies as well as large, temperature-sensitive colonies. <br /> tion (Paidhungat and Garrett 1998). Thus, we pro-                   Crosses showed that the small colonies were disomic for chro- <br />                                                                     mosome IV, whereas the large colonies contained a single <br /> pose that the bud-growth defect of cdc1(Ts) mutants                 copy of chromosome IV. <br /> results from depletion of cytosolic Mn21, which implies,               Cloning COS4, COS5, and COS15: COS4, COS5, and COS15 <br /> in turn, that cytosolic Mn21 is critical for bud growth             were cloned from a YCp50-based yeast genomic library (Rose <br /> (Paidhungat and Garrett 1998). However, neither                     et al. 1987) by their ability to complement the temperature- <br /> the mechanism by which Cdc1 regulates cytosolic Mn21                sensitive growth defects of cos4-8 cdc1-1 [YEp13-CDC1], cos5- <br />                                                                     17 cdc1-1 [YEp13-CDC1], and cos15-118 cdc1-1 [YEp13-CDC1] <br /> nor the Mn21-dependent processes are known.                         strains, respectively. Four COS4 plasmids, pFB77&#226;&#8364;&#8220;pFB80, con- <br />   To identify cellular components that affect the essen-            tained distinct, but overlapping, genomic regions that also <br /> tial Cdc1 function, we isolated suppressors of the cdc1-            reversed the temperature-resistant growth of a cos4-8 cdc1- <br /> 1(Ts) temperature-sensitive growth defect. Several re-              1(Ts) strain. Plasmid pFB87 was generated by subcloning the <br /> cessive suppressors define genes previously implicated              4.3-kb HindIII-BglII genomic fragment from pFB78 into the <br />                                                                     HindIII and BamHI sites of the low-copy URA3 vector pRS316 <br /> in biogenesis of the vacuole, an organelle known to                 (Sikorski and Hieter 1989). This fragment complemented <br /> sequester divalent cations such as Mn21. In addition, one           all of the cos4 mutant phenotypes and was physically linked <br /> suppressor gene, COS16, encodes a vacuolar membrane                 to the PEP3 region (Riles et al. 1993). The PEP3 gene was <br /> protein that appears to be involved in vacuolar Mn21                disrupted by inserting the HIS3 marker into a unique BamHI <br /> sequestration. These results suggest mutations that ei-             site in pFB78 to generate plasmid pFB107. The pep3::HIS3 <br />                                                                     disruption was excised from pFB107 using EcoRI. <br /> ther directly or indirectly block vacuolar Mn21 sequestra-             Seven (pFB94&#226;&#8364;&#8220;pFB100) of eleven COS5 plasmids recovered <br /> tion compensate for defects in Cdc1 function. Our re-               contained distinct, but overlapping, genomic regions. Plasmid <br /> sults also show that Cdc1 is dispensable in cells lacking           pFB109 was constructed by inserting the 4.6-kb BglII-BamHI <br /> Cos16, suggesting that Cdc1 function is not required in             fragment from pFB99 into the BamHI site of pRS316. This <br /> cells that exhibit a defect in vacuolar Mn21 sequestra-             plasmid complemented all of the cos5 mutant phenotypes. <br />                                                                     Plasmid pFB162 was used to mark the COS5 locus and was <br /> tion. Thus, it seems likely that the essential function of          constructed by inserting the 1.9-kb EcoRI fragment from <br /> Cdc1 is to maintain cytosolic [Mn21] above a minimum                pFB109 into a derivative (pFB119) of the integrating vector <br /> threshold that is required for growth.                              pRS306, which lacked the SpeI and XbaI sites in the multiple <br />                                                                     cloning sequence (MCS). Plasmid pFB162 was linearized at <br />                                                                     the unique XbaI site in the insert and integrated by homology- <br />                MATERIALS AND METHODS                                directed recombination into the yeast genome. <br />                                                                        Two identical COS15 plasmids, pFB135 and pFB136, con- <br />    Media: Standard yeast media were prepared as previously          tained the PEP5 gene, as determined by comparing its restric- <br /> described (Paidhungat and Garrett 1998). LiCl and MgCl2             tion map to that of a YCp50-PEP5 plasmid obtained from E. <br /> were added to rich yeast medium (YEPD) after autoclaving.           Jones (Carnegie Mellon University, Pittsburgh). <br /> Medium containing CaCl2 was made by autoclaving YEPD me-               Cloning and disruption of COS8: The COS8 gene was origi- <br /> dium, buffering to pH 5.5 with 50 mm succinate/NaOH and             nally cloned as a high-copy suppressor of the cdc1-1(Ts) growth <br /> then adding CaCl2 to the indicated concentration.                   defect. Plasmid pFB57 was isolated from a high-copy (pRS202) <br />    Strains and plasmids: Revertants were derived from strains       yeast genomic library (C. Connelly and P. Hieter, unpub- <br /> FY11 [MATa ade1 trp1 leu2 his3 ura3 cdc1-1(Ts)] and FY12            lished results) by its ability to suppress the temperature-sensi- <br /> [MATa ade8 trp1 leu2 his3 ura3 cdc1-1(Ts)]. Strain FY 70 (FY11      tive growth defect of strain FY11 (cdc1-1) at 308. A 1.8-kb <br /> CDC1) was derived from FY11 by reversion and contains the           HindIII genomic fragment from pFB57 was subcloned into <br /> wild-type CDC1 allele. Strain FY 71 (MATa ade8 trp1 leu2 his3       the HindIII site of pRS202 to generate plasmid pFB67, which <br /> ura3 CDC1) has been described (Paidmungat and Garrett               suppressed the cdc1-1(Ts) temperature-sensitive growth de- <br /> 1998) and was used to determine suppressor linkage to CDC1.         fect. Interestingly, plasmid pFB67 complemented the salt sen- <br /> Diploid strain FY1 (MATa/MATa cdc1D::HIS3/CDC1 ade2/                sitivity of the cos8 mutants. To determine if pFB57 contained <br /> ADE2 trp1/TRP1 ura3/ura3 leu2/leu2 his3/his3 lys2/lys2) and         COS8, the 1.8-kb HindIII fragment was subcloned into the <br /> a cos16D derivative (FY1 cos16D::LEU2/COS16) were derived           low-copy vector pRS316. The resulting plasmid, pFB161, com- <br /> from strain Y1029 (Garrett and Broach 1989) by transfor-            plemented all of the cos8 mutant phenotypes. The 1.8-kb Hin- <br /> mation. Bacterial strains MC1066 and DH5a were used for             dIII fragment was physically mapped to a region of chromo- <br /> plasmid manipulations and have been described (Casadaban            some XVI (Riles et al. 1993 and references therein) that <br /> et al. 1983; Woodcock et al. 1989).                                 contained a single, complete open reading frame, VPS4/ <br />    Plasmid YEp13-CDC1 (pFB28) has been described (Paid-             END13. <br /> hungat and Garrett 1998). Plasmid YIp5-CDC1 (pFB56)                    The 1.8-kb HindIII COS8 fragment was inserted into pUC8 <br /> was constructed by inserting the 3.5-kb HindIII CDC1 fragment       (NEB catalogue) to generate plasmid pFB217. Two different <br /> into the HindIII site of YIp5 (NEB catalogue; New England           derivatives of pFB217 were used to disrupt the chromosomal <br /> Biolabs, Beverly, MA). The resulting plasmid was integrated         COS8 gene. In the first construct, the URA3 marker was in- <br /> at the CDC1 locus by linearizing with XbaI before yeast transfor-   serted into a unique MscI site in COS8 to generate plasmid <br /> mation.                                                             pFB277. The cos8::URA3 disruption was liberated from plasmid <br />    Genetic manipulations: The cdc1(Ts) revertants were iso-         pFB277 with KpnI, and transplaced into the yeast genome. To <br />                                                       Mn21 Homeostasis in Yeast                                                1789 <br />  <br /> confirm the cos8::URA3 disruption from pFB277 represented           CDCFDA was used instead of CFDA. CPY secretion was de- <br /> the complete loss of Cos8 function, a cos8D::URA3 deletion          tected by a colony hybridization assay (Roberts et al. 1991). <br /> was constructed from pFB217 by replacing the EcoRV-SpeI re-            Isolation of vacuoles and Cos16 localization: Crude fraction- <br /> gion of COS8 with URA3 to generate plasmid pFB275. The              ation of cell extracts was performed as described (Cowles et <br /> cos8D::URA3 disruption was liberated with HindIII and trans-        al. 1994), except that the 35S-labeling step was omitted. Vacu- <br /> placed into the yeast genome. Although transformation fre-          oles were purified as described (Cardenas et al. 1995; Ohsumi <br /> quencies using the second construct were reduced by the             and Anraku 1981), with the exception that the SW40Ti rotor <br /> limited flanking homology, transformants derived from either        was used for ultracentrifugation (rpms were adjusted to gener- <br /> construct exhibited identical phenotypes.                           ate the required g force). Protein content of extracts was <br />    Cloning COS9 : The recessive, cold-sensitive (188) growth        measured by the Bradford assay (Bio-Rad, Richmond, CA) <br /> defect of the cos9-26(Cs) revertant was exploited to select COS9    with BSA standards. For antigen detection and quantitation, <br /> clones from a low-copy yeast genomic library (Rose et al. 1987).    extracts were diluted into sample buffer, heated at 658 for 10 <br /> Two plasmids, pFB188 and pFB190, containing overlapping             min, separated on 10% SDS-PAGE, and blotted to nitrocellu- <br /> regions complemented the conditional growth defect of the           lose (Shleicher and Schuell, Keene, NH). The Cos16-LacZ <br /> cos9-26(Cs) strain and reversed the temperature-resistant           fusion protein was detected with a-b-Gal antiserum (Cappel <br /> growth of a cdc1-1(Ts) cos9-26(Cs) double mutant at 308. A 1.0-     Research Products). CPY and Vph1 were detected with mono- <br /> kb EcoRI-XbaI complementing fragment from plasmid pFB188            clonal antibodies from Molecular Probes (Eugene, OR). Kar2 <br /> was subcloned into pRS316 to generate plasmid pFB220. This          and Pma1 were detected using antisera obtained from M. D. <br /> region was physically linked to the ARF1 region on chromo-          Rose (Princeton University, Princeton, NJ) and M. Cardenas <br /> some IVL (Riles et al. 1993 and references therein). A frame-       (Duke University Medical Center, Durham, NC), respectively. <br /> shift <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> was introduced into the ARF1 gene by digesting       HRP-conjugated secondary antibodies (Sigma Chemical, St. <br /> pFB220 with BglII and filling in the staggered ends to generate     Louis; Promega, Madison, WI) were used for detection with <br /> plasmid pFB254. The arf1D::HIS3 construct was obtained from         the ECL system (Amersham, Arlington Heights, IL). <br /> Dr. R. Kahn (Emory University, Atlanta).                               Mn21 estimation: The cells were prepared as previously de- <br />    Cloning COS16: The COS16 gene was cloned from a low-             scribed (Lapinskas et al. 1995), and concentrated by centrifu- <br /> copy yeast genomic library (Rose et al. 1987) by screening for      gation to 20&#226;&#8364;&#8220;80 A600 units/ml. The cell slurry (25 ml, 1&#226;&#8364;&#8220;2 A600 <br /> temperature-sensitive transformants of a cdc1-1(Ts) cos16-57        units) was digested with 50% nitric acid for 2 hr at 958 (until <br /> mutant. Two plasmids (pFB163 and pFB164) with overlapping           solution cleared) in an acid-rinsed Eppendorf tube, cooled <br /> inserts were recovered. Plasmids pFB168 and pFB169 were             to room temperature, and diluted to 1 ml with deionized <br /> derived from the low-copy LEU2 vector pRS315 (Sikorski and          water. Atomic absorption spectroscopy was performed with a <br /> Hieter 1989) and contained 2.7-kb PstI-BamHI and 3.2-kb             L&#226;&#8364;&#8482;vov platform furnace in a Z3030 (Perkin-Elmer, Norwalk, <br /> HindIII-PstI fragments, respectively, from pFB163. Plasmid          CT) instrument with 25 mg Mg(NO 3)2 (Sigma Chemical, ACS <br /> pFB179 was generated by inserting a complementing 2.1-kb            grade) matrix modifier. The furnace temperature program <br /> Bgl II-SpeI (the SpeI site was from the MCS) fragment from          was: Dry 1408 (10 sec ramp, 60 sec hold); Char 14008 (8 sec <br /> pFB168 between the BamHI and XbaI sites of pRS316. Physical         ramp, 25 sec hold); Atomize and reading 22008 (0 sec ramp, <br /> analysis of the 2.1-kb fragment (Riles et al. 1993 and references   5 sec hold); Clean 26508 (1 sec ramp, 4 sec hold); Cool 208 <br /> therein) showed it contained a single, complete open reading        (1 sec ramp, 30 sec hold). Argon gas flow was maintained at <br /> frame, YCR44c. A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> was introduced into            300 ml/min, except during atomization (0 ml/min). Each <br /> YCR44c by linearizing pFB179 at the unique XbaI site within         digested sample was analyzed twice and averaged. The results <br /> YCR44c, filling in the staggered ends, and religating to gener-     of two independent experiments are shown. The Mn21 content <br /> ate plasmid pFB182.                                                 in each sample was normalized to the A600. <br />    The cos16D::HIS3 disruption was generated in several steps. <br /> First, the 2.7-kb PstI-BamHI insert from pFB168 was inserted into <br /> the XbaI and XhoI sites of pRS306, using the SpeI and SalI sites <br /> of the pFB168 MCS. This left unique XhoI and XbaI sites in the                                RESULTS <br /> insert of the resulting plasmid pFB211. The 0.6-kb XhoI-XbaI <br />                                                                        Suppressors of the cdc1-1(Ts) temperature-sensitive <br /> fragment within the COS16 coding region of pFB211 was re- <br /> placed with a XhoI-XbaI HIS3 fragment to generate plasmid           growth defect: Suppressors of the cdc1-1(Ts) growth de- <br /> pFB218. The cos16D::HIS3 allele was transplaced into the yeast      fect were isolated by selecting spontaneous, tempera- <br /> genome after liberating it from plasmid pFB218 with EcoRI. The      ture-resistant revertants of MATa cdc1-1(Ts) and MATa <br /> cos16D::LEU2 allele was constructed by replacing the XhoI-SnaBI     cdc1-1(Ts) strains (see materials and methods). Fifty- <br /> region of YCR44c in plasmid pFB211 with a SalI-HpaI LEU2 <br />                                                                     four independent revertants were isolated at 308, and <br /> fragment to generate plasmid pFB371. The cos16D::LEU2 allele <br /> was liberated from pFB371 with BamHI and Bgl II.                    two revertants were recovered at 368. None of the re- <br />    The COS16-lacZ fusion was generated by inserting the MfeI        vertants isolated at 308 grew at 368, suggesting that the <br /> fragment carrying the COS16 open reading frame into the             growth defect at 308 was less severe than that at 368 <br /> EcoRI site of YEp356R (Myers et al. 1986) to generate plasmid       (Figure 1). Three of the 54 revertants isolated at 308, <br /> pFB325. This fusion lacked the carboxy-terminal residue of the      and both revertants isolated at 368, contained dominant <br /> Cos16 protein. An integrating version (pFB333) of the COS16- <br /> lacZ fusion plasmid was constructed by inserting the AatII          suppressors. Temperature-resistant growth segregated <br /> COS16-lacZ fusion fragment from pFB325 in place of the ho-          2:2 in all cases, indicating that each revertant carried a <br /> mologous AatII fragment of YIp356R (Myers et al. 1986).             single nuclear suppressor mutation. <br /> Plasmid pFB333 was integrated at the URA3 locus after lineariz-        Intragenic suppressors were identified by crossing <br /> ing with StuI.                                                      each cdc1-1(Ts) cos (cdcone suppressor) strain to a wild- <br />    Vacuolar staining and carboxypeptidase Y (CPY) secretion: <br /> Log-phase cells were stained with the vital vacuolar dye            type CDC1 COS strain, and examining tetrads for tem- <br /> CDCFDA (Molecular Probes, Eugene, OR), using a procedure            perature-sensitive recombinants [cdc1-1(Ts) COS]. Sup- <br /> described previously (Manolson et al. 1992), except that            pressors in both revertants isolated at 368, as well as one <br />  1790                                                  M. Paidhungat and S. Garrett <br />  <br />                                                                                                  TABLE 1 <br />                                                                                Recessive suppressors of cdc1-1(Ts) define 15 <br />                                                                                          complementation groups <br />  <br />                                                                        Locus                       Mutant allele(s) recovered <br />                                                                        cos1                 cos1-1, cos1-124 <br />                                                                        cos2                 cos2-2, cos2-6, cos2-13, cos2-16, cos2-19, <br />                                                                                                cos2-22, cos2-27 <br />                                                                        cos3                 cos3-3, cos3-109, cos3-115, cos3-127, <br />                                                                                                cos3-129 <br />                                                                        pep3 (cos4)          pep3-8, pep3-112, pep3-122 <br />                                                                        vph4 (cos5)          vph4-17 <br />                                                                        cos6                 cos6-18 <br />                                                                        cos7                 cos7-20, cos7-31 <br />                                                                        vps4 (cos8)          vps4-21, vps4-23, vps4-130 <br />                                                                        arf1 (cos9)          arf1-26 (Cs), arf1-32, arf1-132 <br />                                                                        cos10                cos10-28 <br />                                                                        cos11                cos11-29, cos11-113 <br />                                                                        cos13                cos13-101, cos13-125 <br />                                                                        cos14                cos14-102, cos14-103, cos14-116, cos14-120, <br />                                                                                                cos14-131 pep5 (cos15) pep5-118 <br />                                                                        cos16                cos16-57, cos16-128 <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;Suppressors of the cdc1-1(Ts) growth defect. <br /> The parent cdc1-1(Ts) strain and seven spontaneous revertants          cos isolates for growth at 308. By this analysis, 40 of the <br /> were streaked onto YEPD agar and incubated at 238, 308, and <br /> 368. Strains were: C DC1 (FY 70 5 FY 11 C DC1); cos2 (FY 11 <br />                                                                        48 suppressors were assigned to 15 different comple- <br /> cos2-2); cos11 (FY 11 cos11-29); cos4 (FY 11 cos4-8); cos16 (FY11      mentation groups (Table 1). <br /> cos16-57); cos3 (FY 11 cos3-3); cos1 (FY 11 cos1-1); cdc1-1 (FY 11).      Several cdc1-1(Ts) cos double mutants were sensitive <br />                                                                        to salts at 238. To examine the salt sensitivity in more <br />                                                                        detail, we transformed a representative mutant from <br /> recessive revertant isolated at 308, failed to recombine               each complementation group with a YIp5-CDC1 plas- <br /> with the cdc1-1(Ts) lesion (.30 tetrads examined). The                 mid, and tested growth on medium containing 200 mm <br /> cdc1-1(Ts) allele resulted from a single C to T transition             CaCl2, 200 mm MgCl2, and 100 mm LiCl. Seven CDC1 <br /> [Pro351 (CCT) to Leu351 (CTT)]. The CDC1 gene from                     cos mutants (cos4, cos5, cos15, cos6, cos11, cos8, and cos9) <br /> one of the revertants (FY70) isolated at 368 contained                 exhibited varying degrees of sensitivity to all of the salts <br /> the wild-type CCT sequence at codon 351; the recessive                 tested (Figure 2 and data not shown; cos6 and cos11 <br /> intragenic suppressor, cdc1-130, retained the original                 mutants were less sensitive than cos4 mutants to all salts, <br /> cdc1-1(Ts) lesion and a second C to T transition [Thr231               and their sensitivity to 200 mm MgCl2 is not readily <br /> (ACA) to Ile231 (ATA)].                                                apparent in the figure). Interestingly, three cos mutants <br />   Aberrant segregation of the ade8 marker suggested                    (cos1, cos3, and cos14) were sensitive only to CaCl2 and <br /> two recessive revertants were disomic for chromosome                   LiCl, two cos mutants (cos13 and cos16) were specifically <br /> IV. Because CDC1 is located on chromosome IV, it                       sensitive to MgCl2 (Figure 2 and data not shown), and <br /> seemed likely that cdc1-1(Ts) duplication was sufficient               three cos mutants (cos2, cos7, and cos10) did not exhibit <br /> to allow growth at 308. Consistent with this idea, a cdc1-             measurable sensitivity to any of the salts tested (Figure <br /> 1(Ts) haploid strain carrying an extra copy of the cdc1-               2 and data not shown). In all cases, salt sensitivity and <br /> 1(Ts) allele on an integrating plasmid grew at 308 (data               cdc1-1(Ts) suppression were tightly linked (no recombi- <br /> not shown). By contrast, a diploid strain containing four              nation in .10 tetrads), implying a single mutation was <br /> copies of the cdc1-1(Ts) allele failed to grow at 308 (data            responsible for both phenotypes. Finally, sensitivity to <br /> not shown). Thus, diploid cells need disproportionately                specific cations probably reflects differences in the cellu- <br /> more Cdc1 activity than haploid cells. This may explain                lar functions of particular COS genes because all of <br /> why one of the intragenic suppressors (cdc1-130) and                   the mutants within each of the latter complementation <br /> the chromosome IV duplications were recessive.                         groups (cos1, cos3, cos14, cos13, cos16, cos2, cos7, and <br />   Recessive suppressors define 15 genes that fall into                 cos10) exhibited sensitivity to the same range of cations <br /> four phenotypic groups: Forty-eight of the 54 suppres-                 as the representative isolate (Figure 2 and data not <br /> sors isolated at 308 were recessive, and unlinked to CDC1.             shown). Accordingly, the CDC1 cos mutants were divided <br /> These were placed into complementation groups by                       into four broad phenotypic groups, I to IV (Table 2). <br /> testing diploids from pairwise matings of the cdc1(Ts)                 Mutations in three group III genes, COS4, COS5, and <br />                                                     Mn21 Homeostasis in Yeast                                              1791 <br />  <br />  <br />  <br />  <br />                                                                                                      Figure 2.&#226;&#8364;&#8221;Salt tolerance <br />                                                                                                   of the cos mutants. The indi- <br />                                                                                                   cated strains were trans- <br />                                                                                                   formed with the YIp5-C DC1 <br />                                                                                                   plasmid, and then streaked <br />                                                                                                   onto YEPD agar, YEPD pH <br />                                                                                                   5.5 agar containing 200 mm <br />                                                                                                   CaCl2, YEPD agar containing <br />                                                                                                   200 mm MgCl2, or YEPD agar <br />                                                                                                   containing 100 mm LiCl, <br />                                                                                                   and incubated at 308 for 3&#226;&#8364;&#8220;5 <br />                                                                                                   days. Strains were: cos2 (FY11 <br />                                                                                                   cos2-2); cos11 (FY11 cos11- <br />                                                                                                   29); cos6 (FY11 cos6-18); cos4 <br />                                                                                                   (FY11 cos4-8); cos16 (FY11 <br />                                                                                                   cos16-57); cos3 (FY11 cos3-3); <br />                                                                                                   cos1 (FY11 cos1-1); 1 (FY11). <br />  <br />  <br />  <br />  <br /> COS15, also blocked accumulation of red pigment in an             vacuolar protein sorting (vps/pep) mutants (Banta et <br /> ade1 background and conferred a temperature-sensitive             al. 1988). The group IIIA cos mutants also displayed <br /> (368) growth defect in a CDC1 strain (data not shown).            several additional phenotypes exhibited by a subset of <br /> Thus, group III genes were subclassified as group IIIA            the vps/pep mutants, including sensitivity to 500 mm <br /> (COS4, COS5, and COS15), and group IIIB (COS6, COS8,              NaCl, 4 mm ZnCl2, and 8 mm MnCl2, as well as failure <br /> COS9, and COS11).                                                 to sporulate as homozygous diploids (data not shown). <br />   Group IIIA COS genes are identical to class C VPS genes:        To determine if group IIIA COS genes were related to <br /> The ade1 pigment-accumulation defect, temperature                 known VPS genes, we isolated the COS4, COS5, and <br /> sensitivity, and salt sensitivity of the group IIIA cos mu-       COS15 genes from a yeast genomic library by comple- <br /> tants resembled phenotypes exhibited by a subset of               menting the temperature-sensitive growth defect of <br />                                                                   CDC1 cos4, CDC1 cos5, and CDC1 cos15 strains, respec- <br />                                                                   tively (see materials and methods). We also examined <br />                            TABLE 2 <br />                                                                   the ability of cos4, cos5, and cos15 mutants to complement <br />             Phenotypic grouping of COS genes                      the phenotypes of known vps/pep mutants. These studies <br />                                                                   showed (Table 1) that COS4 is identical to VPS18/PEP3 <br />               Complementation                                     (Preston et al. 1991; Robinson et al. 1991), COS5 is <br /> Group           groups/Loci                Sensitivity to: a <br />                                                                   identical to VPS16/VPH4 (Horazdovsky and Emr <br /> I           cos, cos3, cos14 b         CaCl2, LiCl                1993; Preston et al. 1992), and COS15 is identical to <br /> II          cos13, cos16               MgCl2                      VPS11/PEP5 (Woolford et al. 1990). Finally, these as- <br /> III         Group IIIA: pep3           CaCl2, LiCl, MgCl2,        signments were confirmed by showing that mutations <br />                (cos4), vps16 (cos5),     MnCl2, ZnCl2             inactivating VPS18 (pep3::LEU2), VPS16 (vph4-5) or <br />                pep5 (Cos15) <br />                                                                   VPS11 (pep5D::URA3), as well as a fourth class C VPS <br />             Group IIIB: vps4           CaCl2, LiCl, MgCl2, <br />                (cos8), cos6, cos11,      MnCl2, ZnCl2;            gene VPS33 (pep14-5) (Banta et al. 1990) suppressed <br />                arf1 (cos9)               sensitivity less         the growth defect of a cdc1-1(Ts) strain at 308 (data not <br />                                          severe than              shown). Thus, mutations in all four class C VPS genes <br />                                          group IIIA               suppressed the cdc1-1(Ts) growth defect at 308. <br /> IV          cos2, cos7, cos10          None                          Group IIIB cos mutants mislocalize the vacuolar pro- <br />   a <br />     Sensitivity to: 200 mm CaCl2; 200 mm MgCl2; 100 mm LiCl;      tein CPY: Group IIIB cos mutants shared several pheno- <br /> 4 mm ZnCl2; 8 mm MnCl2.                                           types with the class C vps mutants, including general <br />   b <br />     All strains were CDC1.                                        salt sensitivity and a sporulation defect, with the class C <br />  1792                                            M. Paidhungat and S. Garrett <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;CPY secretion. Patches of the indicated strains were replica plated to YEPD agar, overlaid with nitrocellulose <br /> membrane, and incubated at 238. After 12 hr, membranes were rinsed in water and probed with a-CPY antibodies. All strains <br /> in rows A&#226;&#8364;&#8220;C carried a YIp5-CDC1 plasmid. Strains were, from left to right, row A: FY11 cos16-57, FY11 cos3-3, FY11 cos1-1, and <br /> FY11; row B: FY11 cos2-13, FY11 cos11-29, FY11 cos6-18, and FY11 pep3-8; row C: FY11 arf1-32, FY11 pep3-8, FY11 vps4-21, FY11; <br /> row D: FY70 [pRS202-COS4], FY70 [pRS202], FY11 [pRS202-COS4], FY11 [pRS202]. <br />  <br />  <br />  <br />  <br /> vps mutants (although cos9 mutants did not exhibit a             cdc1-1(Ts) mutant at 308 (data not shown). Thus, loss <br /> sporulation defect). To determine if group IIIB COS              of class D Vps function suppressed the cdc1(Ts) growth <br /> genes were also involved in vacuolar protein sorting, we         defect. COS6 and COS11 were distinct from PEP7 and <br /> examined localization of the vacuolar protein CPY.               might define other class D VPS genes. <br /> Wild-type strains efficiently target CPY to the vacuole.            COS8 is identical to VPS4/END13, a class E VPS gene: <br /> Mutants with defects in the vacuolar protein sorting             cos8 cells exhibited a single, prominent vacuolar struc- <br /> pathway, by contrast, secrete significant amounts of CPY         ture surrounded by one or more small, CDCFDA-stain- <br /> (Raymond et al. 1992). Similar to the class C vps18 (cos4-       ing vesicles (Figure 4). Although this vacuolar morphol- <br /> 8) mutant, all of the group IIIB cos mutants (cos6, cos8,        ogy resembled that of wild-type cells (Figure 4), class A <br /> cos9, and cos11 mutants) secreted significantly more CPY         and class E vps mutants also contain normal-looking <br /> than the wild-type strain (Figure 3, B and C). The sorting       vacuoles (Raymond et al. 1992). Indeed, molecular char- <br /> defect was less apparent, but nevertheless obvious, in           acterization (see materials and methods) showed that <br /> the cos9 mutant (Figure 3C). Thus, group IIIB COS                COS8 was identical to VPS4/END13, a class E VPS gene <br /> genes are necessary for efficient vacuolar protein sort-         implicated in vacuolar protein sorting (Munn and Riez- <br /> ing. By contrast, cos mutants from groups II and IV did          man 1994). Furthermore, a vps4D::URA3 disruption sup- <br /> not exhibit significant CPY secretion (Figure 3), and            pressed the cdc1-1(Ts) growth defect at 308 (data not <br /> group I cos mutants displayed a slight defect in CPY             shown), indicating that loss of class E Vps function sup- <br /> secretion (Figure 3A) that became more severe in older           pressed the cdc1-1(Ts) growth defect. <br /> colonies.                                                           Because VPS4 was also identified as a high-copy sup- <br />    Mutations in class D VPS genes suppress the cdc1-1(Ts)        pressor of the cdc1-1(Ts) growth defect (see materials <br /> growth defect: To determine if group IIIB cos mutations          and methods), we determined if Vps4 overproduction <br /> affected vacuolar biogenesis, we examined vacuolar               blocked vacuolar protein sorting. As shown in Figure 3D, <br /> morphology using the vital stain CDCFDA. In contrast             an increase in VPS4 dosage caused CDC1 and cdc1-1(Ts) <br /> to the multilobed vacuolar structure of wild-type cells          cells to secrete more CPY, consistent with the idea that <br /> (Figure 4), cos6 and cos11 mutant cells displayed a single       Vps4 overproduction suppressed the cdc1-1(Ts) growth <br /> vacuolar lobe (Figure 4 and data not shown) that was             defect by impeding vacuolar protein sorting. <br /> similar to the abnormal vacuolar morphology of class D              Loss of Arf1 function suppresses the cdc1-1(Ts) growth <br /> vps mutants (Raymond et al. 1992). Like class D vps mu-          defect: The last group IIIB mutant, cos9, displayed numer- <br /> tants, cos6 and cos11 cells failed to form vacuolar segrega-     ous (.15) CDCFDA-staining vesicles, indicative of vacuo- <br /> tion structures and exhibited a defect in vacuolar inheri-       lar fragmentation (Figure 4; this <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> was clearer <br /> tance (Figure 4 and data not shown). Finally, disruption         under Nomarski optics because vesicles outside the focal <br /> of the class D VPS gene, VPS19/PEP7/VAC1 (Weisman                plane interfered with visualization of the CDCFDA fluo- <br /> and Wickner 1992), alleviated the growth defect of a             rescence). Interestingly, cos9 mutants also exhibited abnor- <br />                                                       Mn21 Homeostasis in Yeast                                              1793 <br />  <br />                                                                     major role in protein turnover. Several vacuolar hy- <br />                                                                     drolases, including proteinase B and CPY, are activated <br />                                                                     by the vacuolar Pep4 proteinase (proteinase A). As a <br />                                                                     result, loss of Pep4 function results in a 90&#226;&#8364;&#8220;95% reduc- <br />                                                                     tion in vacuolar hydrolase activity ( Jones et al. 1982). <br />                                                                     Nevertheless, a cdc1-1(Ts) pep4D::LEU2 strain failed to <br />                                                                     grow at 308 (data not shown). Thus, neither the loss <br />                                                                     of Pep4 nor the accompanying reduction in vacuolar <br />                                                                     hydrolase activity could account for cdc1(Ts) suppres- <br />                                                                     sion by defects in vacuolar biogenesis. <br />                                                                        Group II gene COS16: The group II cos mutants exhib- <br />                                                                     ited a subset of the group III cos mutant phenotypes, <br />                                                                     including Mg21 sensitivity (Figure 2), failure to sporu- <br />                                                                     late, and a slight growth defect at 368 (data not shown). <br />                                                                     However, group II cos mutants did not secrete CPY (Fig- <br />                                                                     ure 3). We cloned the COS16 gene from a yeast genomic <br />                                                                     library by complementation of the temperature-resis- <br />                                                                     tant growth of the cdc1-1(Ts) cos16-57 mutant. The com- <br />                                                                     plementing fragment was mapped to a region of chro- <br />                                                                     mosome IIIR (data not shown), which contained a single <br />                                                                     1071-bp open reading frame, YCR44c. A frameshift mu- <br />                                                                     tation introduced into the YCR44c coding region abol- <br />                                                                     ished the ability of the 2.1-kb fragment to complement <br />                                                                     the cos16-57 mutation (data not shown). In addition, a <br />                                                                     YCR44c disruption suppressed the cdc1(Ts) temperature- <br />                                                                     sensitive growth defect (data not shown) and conferred <br />                                                                     Mg21 sensitivity and temperature-sensitive growth (at 368) <br />                                                                     to a CDC1 strain (data not shown). Finally, all of the <br />                                                                     spontaneous cos16 alleles were tightly linked (,2.5 cM <br />    Figure 4.&#226;&#8364;&#8221;Vacuolar morphology of cos mutants visualized          with at least 20 tetrads) to the MAT locus, as predicted <br /> by CDCFDA. Exponentially growing cells were stained with the        by the physical proximity of YCR44c to MAT. Thus, COS16 <br /> vital dye CDCFDA and visualized by differential interference        is identical to the open reading frame YCR44c. <br /> contrast (DIC) or fluorescence (CDCFDA) microscopy. Strains: <br /> COS1 (FY 11 [YIp5-C DC1]); cos4 (vps18) (FY 11 cos4-8 [YIp5- <br />                                                                        Subcellular localization of Cos16: The predicted pri- <br /> C DC1]); cos6 (FY11 cos6-18 [YIp5-CDC1]); cos8 (vps4) (FY11 cos8-   mary structure of the Cos16 protein did not exhibit sig- <br /> 21 [YIp5-CDC1]); cos9 (arf1) (FY11 cos9-32 [YIp5-CDC1]).            nificant homology to proteins in the available databases. <br />                                                                     Nevertheless, hydropathy analysis (Kyte and Doolit- <br />                                                                     tle 1982) showed that Cos16 contained 8 putative mem- <br /> malities in cell size and shape (Figure 4), suggesting Cos9         brane-spanning helices, suggesting that it was an inte- <br /> might affect several cellular processes. We cloned COS9             gral membrane protein. To examine Cos16 localization, <br /> and found it was identical to the ARF1 gene (see materi-            we constructed a hybrid gene between COS16 and lacZ. <br /> als and methods), which encodes a GTPase implicated                 A single copy of the COS16-lacZ fusion was integrated <br /> in transport between the endoplasmic veticulum (ER) and             at the URA3 locus in a cos16D::HIS3 mutant. Most (.80%) <br /> Golgi compartments (Stearns et al. 1990). As expected,              of the Cos16-LacZ fusion protein was detected in the par- <br /> the arf1D::HIS3 disruption allowed a cdc1-1(Ts) mutant to           ticulate fraction (100,000 g for 1 hr), and could be solubil- <br /> grow at 308 (data not shown). Although ARF1 has not                 ized with Triton X-100, but not 1 m NaCl or 2 m urea <br /> been previously described as a VPS gene, the arf1D::HIS3            (data not shown). Thus, the Cos16-LacZ fusion protein is <br /> mutant secreted CPY, displayed abnormal vacuolar mor-               probably an integral membrane protein. <br /> phology, and exhibited decreased tolerance to salt. Be-                The subcellular localization of the Cos16-LacZ fusion <br /> cause Arf1 function is required for normal vacuolar bio-            protein was initially examined by crude fractionation. <br /> genesis, the cos9 mutations probably suppress the cdc1-             Cell lysates were fractionated by sequential centrifuga- <br /> 1(Ts) growth defect by debilitating vacuolar function.              tion (Cowles et al. 1994), and the fusion protein in <br />   Proteinase A deficiency does not suppress the cdc1-1(Ts)          each fraction was detected by Western blotting. Greater <br /> growth defect: Because the group III cos mutations                  than 90% of the Cos16-LacZ fusion protein cofrac- <br /> blocked formation of a normal vacuole, the cdc1(Ts)                 tionated with the vacuolar membrane marker, Vph1, as <br /> growth defect was probably alleviated by the absence of             well as the ER lumenal marker, Kar2, in the P13 fraction <br /> a normal vacuole, rather than a defect at a specific stage          (data not shown). To determine if COS16 encoded a <br /> of vacuolar protein sorting. The yeast vacuole plays a              vacuolar membrane protein, vacuoles were purified by <br />  1794                                             M. Paidhungat and S. Garrett <br />  <br />                                                                  uolar localization of the Cos16-LacZ fusion protein <br />                                                                  probably reflects the subcellular location of the native <br />                                                                  Cos16 protein. <br />                                                                     Cos16 and Mn21 homeostasis: Because the vacuole <br />                                                                  sequesters Mn21 from the cytosol (Okorokov et al. 1977), <br />                                                                  we examined if Cos16 was also involved in Mn21 homeo- <br />                                                                  stasis. The cos16D::HIS3 deletion had a modest effect, <br />                                                                  similar to that of the vps4 mutation, on the Mn21 sensitiv- <br />                                                                  ity of an otherwise wild-type strain (data not shown). Muta- <br />                                                                  tions in the Golgi Mn21 transporter gene, PMR1, increase <br />                                                                  sensitivity to Mn21 as the result of Mn21 accumulation <br />                                                                  in the cytosol (Lapinskas et al. 1995). The cos16D::HIS3 <br />                                                                  deletion exacerbated this sensitivity (Figure 6A). Thus, <br />                                                                  deletion of COS16 altered tolerance to Mn21. <br />                                                                     We also examined the effect of COS16 on the chelator <br />                                                                  sensitivities of several Mn21-homeostasis mutants. Strains <br />                                                                  lacking Pmr1 function are sensitive to EGTA, and this <br />                                                                  sensitivity can be ameliorated by increasing Mn21 influx <br />                                                                  across the plasma membrane (Paidhungat and Gar- <br />                                                                  rett 1998). As shown in Figure 6B, the cos16D::HIS3 <br />                                                                  deletion allowed a pmr1D::LEU2 strain to grow on me- <br />                                                                  dium containing 0.5 mm EGTA. Mutations in SMF1 also <br />                                                                  cause cells to become sensitive to EGTA, in this case as <br />                                                                  the result of the loss of a high-affinity Mn21 uptake <br />                                                                  system. This EGTA sensitivity is alleviated by Mn21 sup- <br />                                                                  plement (Supek et al. 1996), as well as overproduction <br />                                                                  of Cdc1 (Paidhungat and Garrett 1998). Again, only <br />   Figure 5.&#226;&#8364;&#8221;Subcellular localization of Cos16-b-gal fusion       the cos16D::HIS3 smf1D::URA3 double mutant tolerated <br /> protein. Protein from whole-cell lysates (Total) or purified <br />                                                                  4 mm EGTA in the medium (Figure 6C). Finally, the <br /> vacuoles (Vacuole) were separated by SDS-PAGE and trans- <br /> ferred to nitrocellulose membrane. The b-gal lane contains       EGTA sensitivity (1.5 mm) of the cdc1-1(Ts) strain was <br /> purified b-galactosidase. Vacuoles were purified as previously   alleviated by the cos16D::HIS3 deletion (Figure 6D). <br /> described (Cardenas and Heitman 1995). The membrane              These findings implicate Cos16 in Mn21 homeostasis <br /> was probed with antibodies against bacterial b-gal protein, as   and suggest Cos16 is involved in the sequestration of <br /> well as the yeast proteins CPY, Kar2, and Pma1. <br />                                                                  Mn21 into the vacuole. The last result is also consistent <br />                                                                  with the notion that the cos16 mutations alleviate the <br />                                                                  cdc1(Ts) conditional growth defect through their effect <br /> flotation on a Ficoll gradient (Cardenas and Heitman             on Mn21 homeostasis. <br /> 1995), and analyzed for presence of the Cos16-LacZ                  Mn21 content of cdc1 and cos16 mutants: Several genes <br /> fusion protein. The intact Cos16-LacZ fusion was en-             that affect Mn21 homeostasis, including PMR1 and ATX2, <br /> riched in the vacuolar extract to an extent that was             alter intracellular Mn21 content (Lapinskas et al. 1995; <br /> comparable to the vacuolar lumenal marker, CPY (Fig-             Lin and Culotta 1996). By contrast, intracellular Mn21 <br /> ure 5). By contrast, the ER protein, Kar2, and the               levels are unaffected by overexpression of CCC1, a high- <br /> plasma-membrane protein, Pma1, were not significantly            copy suppressor of the pmr1 Mn 21 defect (Lapinskas <br /> enriched in the vacuolar fraction (Figure 5). Similar            et al. 1996). Because our results suggested that Cdc1 and <br /> studies using vacuolar vesicles purified by a different          Cos16 affected the intracellular distribution of Mn21, we <br /> method (Ohsumi and Anraku 1981) yielded identical                examined if mutations in either CDC1 or COS16 altered <br /> results (data not shown). These results suggest that             intracellular Mn21 levels as judged by atomic absorption <br /> Cos16 is an integral membrane protein of the vacuole.            spectroscopy. The Mn21 content of cdc1-1(Ts) (38.9 6 <br />    Misfolded membrane proteins default to the vacuole            8.3 pmols/A600) and cos16D::HIS3 (41.4 6 12.3 pmols/ <br /> (Stack and Emr 1993). However, that seems an unlikely            A600) mutants were similar to the Mn21 content of the <br /> explanation for the vacuolar localization of the Cos16-          isogenic wild-type (41.2 6 7.0 pmols/A600) strain, sug- <br /> LacZ fusion protein, because single-copy COS16-lacZ              gesting that depletion of neither Cdc1 nor Cos16 had <br /> complemented all of the cos16D phenotypes (data not              a significant effect on whole-cell Mn21 content. Cellular <br /> shown). Moreover, a different epitope-tagged COS16               Mn21 levels were also unaffected by CDC1 overexpres- <br /> fusion, in which the myc epitope was fused to the C              sion (data not shown). Thus, both Cdc1 and Cos16 seem <br /> terminus of Cos16, was functional, and exhibited a simi-         to affect the intracellular distribution of Mn21 without <br /> lar localization <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (data not shown). Thus, the vac-        changing the overall cellular content. <br />                                                     Mn21 Homeostasis in Yeast                                                 1795 <br />  <br />  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;COS16 and Mn21 homeostasis. (A) cos16D exacerbates the Mn21 sensitivity of a pmr1D mutant. The indicated strains <br /> were streaked onto YEPD pH 5.5 agar, supplemented with 0 mm, 2 mm, or 4 mm MnCl2, and incubated at 308 for 3&#226;&#8364;&#8220;5 days. <br /> Strains were: cos16D pmr1D (FY 70 cos16D::HIS3 pmr1D::LEU2); COS16 pmr1D (FY70 pmr1D::HIS3), cos16D PMR1 (FY70 cos16D::HIS3); <br /> COS16 PMR1 (FY70). (B) cos16D suppresses the EGTA sensitivity of a pmr1D mutant. The strains of panel A were streaked onto <br /> YEPD agar, supplemented with 0 mm or 0.5 mm EGTA, and incubated at 308 for 3&#226;&#8364;&#8220;5 days. (C) cos16D suppresses the EGTA <br /> sensitivity of a smf1D mutant. 10-fold dilutions of exponentially growing cultures of the indicated strains were spotted onto YEPD <br /> agar or YEPD agar containing 4 mm EGTA, and incubated at 308 for 3 days. Strains were: 1 (FY70), cos16D (FY70 cos16D::HIS3), <br /> smf1D (FY70 smf1D::URA3), and cos16D smf1D (FY 70 cos16D::HIS3 smf1D::URA3). (D) cos16D suppresses the EGTA sensitivity of <br /> a cdc1-1(Ts) mutant. The indicated strains were streaked onto YEPD agar or YEPD agar containing 1.5 mm EGTA and incubated <br /> at 308 for 3 days. Strains were: cdc1 cos16D (FY11 cos16D::HIS3); cdc1 (FY11); CDC1 (FY70); cos16D (FY70 cos16D::HIS3). <br />  <br />  <br />  <br />    Cdc1 is dispensable in cos16D mutants: Because Cos16             .90% of the expected cdc1D::HIS3 cos16D::LEU2 prog- <br /> seemed to be involved in vacuolar Mn21 homeostasis, we              eny formed healthy colonies when supplemented with <br /> determined if a cdc1D cos16D double mutant could grow               2 mm or 4 mm Mn21 (Figure 7). Thus, Mn21 augments <br /> in the presence of exogenous Mn21. Isogenic diploids                the ability of the cos16D deletion to bypass Cdc1 function. <br /> (cdc1D::HIS3/CDC1 COS16/COS16 and cdc1D::HIS3/ <br /> CDC1 cos16D::LEU2/COS16) were sporulated and dis- <br /> sected onto YEPD medium agar with, or without, Mn21                                        DISCUSSION <br /> supplement. Two viable His2 (CDC11) segregants were <br /> recovered from each tetrad of the cdc1D::HIS3/CDC1                     Previous studies implicated Cdc1 in the regulation of <br /> COS16/COS16 diploid (Figure 7), and the 2:0 segrega-                intracellular Mn21 (Loukin and Kung 1995; Paidhun- <br /> tion was not affected by Mn21 (data not shown). By con-             gat and Garrett 1998). That hypothesis was based on <br /> trast, tetrads from the cdc1D::HIS3/CDC1 cos16D::LEU2/              several observations, including: (1) Mn21 supplement <br /> COS16 diploid yielded 60% of the expected cdc1D::HIS3               partially alleviated the conditional cdc1(Ts) growth de- <br /> cos16D::LEU2 segregants (Figure 7). Thus, Cdc1 is not               fect; and (2) CDC1 overexpression ameliorated the EGTA <br /> essential in strains lacking Cos16 function. Moreover,              sensitivity of two, unrelated Mn21-homeostasis mutants. <br />  1796                                              M. Paidhungat and S. Garrett <br />  <br />                                                                   from the block in vacuolar Mn21 sequestration because <br />                                                                   the same vacuolar biogenesis mutations also relieve the <br />                                                                   EGTA-sensitive phenotype of the cdc1(Ts) mutants (data <br />                                                                   not shown). These results eliminate the vacuole as the <br />                                                                   essential, Cdc1-dependent Mn21 store and suggest that <br />                                                                   the vacuolar biogenesis mutations suppress the cdc1(Ts) <br />                                                                   growth defect by attenuating the depletion of cytosolic <br />                                                                   Mn21. <br />                                                                      Lesions in vacuole biogenesis (VPS) genes affect many <br />                                                                   vacuolar functions, including proteolysis, Ca 21 accumu- <br />                                                                   lation, proton uptake and Mn21 sequestration. However, <br />                                                                   the conditional cdc1(Ts) growth defect is suppressed <br />                                                                   only by mutations that affect the last of these processes. <br />                                                                   In particular, the cdc1(Ts) growth defect is suppressed <br />                                                                   by the inactivation of a protein, Cos16, that appears <br />                                                                   to be involved in Mn21 sequestration into the vacuole. <br />    Figure 7.&#226;&#8364;&#8221;Cdc1 is dispensable in a cos16 mutant. Heterozy- <br />                                                                   Several lines of evidence support this assertion. First, <br /> gous cdc1D::HIS3/C DC1 COS16/COS16 (FY 1) or cdc1D::HIS3/         Cos16 is an integral membrane protein of the vacuole, <br /> C DC1 cos16D::LEU2/COS16 (FY 1 cos16D::LEU2) diploids were        as judged by the subcellular localization pattern of a <br /> sporulated and dissected onto rich yeast medium with, or with-    functional Cos16-LacZ fusion protein (Figure 5). This <br /> out, Mn21 supplement (see materials and methods). Tiny            observation is consistent with the presence of eight puta- <br /> (or small) colonies were cdc1D::HIS3 cos16D::LEU2 as deter- <br />                                                                   tive transmembrane domains within the predicted cod- <br /> mined by their growth on minimal medium agar lacking histi- <br /> dine and leucine. (a, b, c, and d refer to individual spores of   ing region. Thus, Cos16 is appropriately positioned to <br /> a tedrad.)                                                        affect transport between the cytosol and vacuole. Sec- <br />                                                                   ond, a cos16D deletion suppresses the chelator sensitivity <br />                                                                   of a strain (smf1D) compromised for Mn21 uptake into <br />                                                                   the cytosol (Figure 6C), suggesting that loss of Cos16 <br /> We show here that, in certain genetic backgrounds, Mn21           function compensates for the low influx into the cytosol. <br /> supplement completely bypasses the essential Cdc1 re-             cos16 mutations also relieve the chelator sensitivity of a <br /> quirement. Indeed, a cdc1D cos16D double mutant exhibits          mutant (pmr1D) that can be efficiently suppressed by <br /> robust growth when the medium is supplemented with                genetic manipulations that raise cytosolic Mn21 (Figure <br /> Mn21 (Figure 7). Because COS16 is also involved in intra-         6B; Paidhungat and Garrett 1998). Finally, Cos16 <br /> cellular Mn21 homeostasis, these results are best explained       inactivation exacerbates the Mn21 sensitivity of both <br /> by a model in which Mn21 regulation is the single, essential      wild-type (data not shown) and pmr1D mutant strains <br /> function of Cdc1. These results also rule out the possibility     (Figure 6A), presumably by aggravating the accumula- <br /> that the Mn21 dependence of cdc1 strains reflects the fact        tion of cytosolic Mn21. Thus, Cos16 appears to antago- <br /> that Cdc1 is a Mn21-dependent enzyme (Supek et al. 1996;          nize the cdc1(Ts) growth defect by sequestering cytosolic <br /> Paidhungat and Garrett 1998).                                     Mn21 into the vacuole. <br />    Although the mechanism by which Cdc1 affects intra-               If COS16 and CDC1 are involved in the homeostasis of <br /> cellular Mn21 levels is not clear, the evidence favors a          intracellular Mn21, why do intracellular Mn21 levels not <br /> model in which Cdc1 functions to maintain cytosolic               vary with changes in Cos16 or Cdc1 function? A previous <br /> Mn21. Mn21 depletion from the Golgi elicits a protein             report (Okorokov et al. 1977) suggested that most intra- <br /> glycosylation defect not observed in the cdc1(Ts) mu-             cellular Mn21 is found in the vacuole. According to that <br /> tants, and the cdc1(Ts) growth defect is exacerbated,             report, a defect in cytosolic Mn21 retention, as predicted <br /> not alleviated, by alterations that increase Mn21 flux            for cdc1(Ts) mutants, might not affect total intracellular <br /> from the cytosol to the Golgi (Paidhungat and Gar-                Mn21 content. However, it is harder to reconcile the fact <br /> rett 1998). Thus, we suggested that the cytosol, rather           that the cos16D deletion did not alter the level of intracellu- <br /> than the Golgi apparatus, was the more likely site of             lar Mn21. We can think of several possible explanations <br /> cdc1(Ts) Mn21 depletion (Paidhungat and Garrett                   for this apparent paradox. First, Cos16 might either be <br /> 1998). Another major intracellular Mn21 store is the              involved in the regulation of other ions, or affect an organ- <br /> vacuole (Okorokov et al. 1977). Although the vacuole              elle that might copurify with the vacuole. Although we <br /> is known to play a role in Mn21 detoxification, nothing           have not formally ruled out these possibilities, they do not <br /> is known about the cellular function(s) of the vacuolar           easily account for the observation that cos16 mutations <br /> Mn21 pool. However, mutations that inhibit vacuolar               affect the growth of Mn21 homeostasis mutants. Alterna- <br /> biogenesis (and function) at several distinct steps sup-          tively, Cos16 inactivation might block cytosolic to vacuole <br /> press, rather than exacerbate, the cdc1(Ts) growth de-            Mn21 transfer without having a measurable effect on vacu- <br /> fect. Suppression of conditional growth probably results          olar Mn21 content. <br />  <br />  <br />    Finally, along with COS16 and the vacuolar biogenesis                 <br />                                                                            <br /> genes characterized in this study, we identified several                    <br /> other suppressors of the cdc1(Ts) growth defect. One                              <br /> suppressor, cos13, falls within the same phenotypic and                   <br />                                                                      <br /> epistasis group as cos16 and might, therefore, identify                         <br /> a component of a complex or pathway involved in vacuo-                      <br /> lar Mn21 homeostasis. By contrast, the remaining reces-            <br /> sive suppressors (cos1, cos3, cos14, cos2, cos7, and cos10)         <br /> confer phenotypes that are not obviously related to                          <br /> Mn21 regulation. Thus, the genes corresponding to                <br /> these suppressors might specify receptors of the Cdc1            <br /> (and Mn21?)-dependent growth process. Future studies                  <br /> of these genes might help elucidate the role of Cdc1                          <br /> in Mn21 homeostasis and growth.                                             <br /> </body></html>