<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                         Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br />  <br />  <br /> Synthetic lethal interactions conditional poly(A) polymerase <br /> alleles identify LCP5, gene involved 18S rRNA maturation <br /> T. Wiederkehr, R. F. Pretot L. Minvielle-Sebastia <br />  <br /> RNA 1998 4: 1357-1372 <br />  <br />  <br />  <br />       References <br />                         Article cited in: <br />                         http://www.rnajournal.org/cgi/content/<span id='am-1' about='protege:abstract' typeof='owl:Thing'>abstract</span>/4/11/1357#otherarticles <br />     Email alerting      Receive free email alerts new articles cite article - sign box the <br />           service       right corner article click here <br />  <br />  <br />  <br />              Notes <br />  <br />  <br />  <br />  <br /> <span id='am-9' about='protege:TO' typeof='owl:Thing'>To</span> subscribe RNA to: <br /> http://www.rnajournal.org/subscriptions/ <br />  <br />  <br />  <br /> &#194;&#169; 1998 RNA Society <br />                                    Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> RNA (1998), 4:1357&#226;&#8364;&#8220;1372+ Cambridge University Press+ Printed USA+ <br /> Copyright &#194;&#169; 1998 RNA Society+ <br />  <br />  <br />  <br />  <br /> Synthetic lethal interactions conditional <br /> poly(A) polymerase alleles identify LCP5, <br /> a gene involved 18S rRNA maturation <br />  <br />  <br /> THOMAS WIEDERKEHR, REN&#195;&#8240; F. PR&#195;&#8240;T&#195;&#8221;T, LIONEL MINVIELLE-SEBASTIA <br /> Department Cell Biology, Biozentrum, University Basel, Klingelbergstrasse 70, CH-4056, Basel, Switzerland <br />  <br />  <br />  <br />  <br /> ABSTRACT <br /> To identify new genes involved 39-end formation mRNAs Saccharomyces cerevisiae, carried screen <br /> for synthetic lethal mutants conditional poly(A) polymerase allele, pap1-7. Five independent temperature- <br /> sensitive mutations called lcp1 lcp5 (for lethal conditional pap1 allele) isolated. Here, the <br /> characterization essential gene LCP5 codes protein calculated molecular mass 40.8 kD. <br /> Unexpectedly, mutations LCP5 caused defects pre-ribosomal RNA (pre-rRNA) processing, whereas <br /> mRNA 39-end formation vitro comparable wild-type. Early cleavage steps (denoted A0 A2 ) lead the <br /> production mature 18S rRNA impaired. In vivo depletion Lcp5p inhibited pre-rRNA processing. As a <br /> consequence, mutant depleted cells showed decreased levels polysomes compared wild-type cells. Indirect <br /> immunofluorescence indicated predominant localization Lcp5p nucleolus. In addition, antibodies directed <br /> against Lcp5p specifically immunoprecipitated yeast U3 snoRNA snR17, suggesting protein directly <br /> involved pre-rRNA processing. <br /> Keywords: ribosome biogenesis; translation; U3 snoRNP <br />  <br />  <br />  <br /> INTRODUCTION                                                           &amp; Sonenberg, 1997), direct link 39 and <br /> The poly(A) tail 39 end eukaryotic                     59 ends mRNA likely occur+ Consis- <br /> messenger RNAs (mRNAs) essential modifica-                       tent observations, discrimination against <br /> tion occurs posttranscriptional level (for                 poly(A)-deficient mRNAs observed trans- <br /> reviews, Keller, 1995; Manley &amp; Takagaki, 1996;                    lation synthesis ribosomal subunits im- <br /> Wahle &amp; Keller, 1996; Colgan &amp; Manley, 1997; Keller                    paired (Proweller &amp; Butler, 1997)+ <br /> &amp; Minvielle-Sebastia, 1997)+ Besides probable in-                    Synthetic lethality (Huffacker et al+, 1987; Guarente, <br /> volvement nucleocytoplasmic transport RNA                       1993) shown efficient genetic method <br /> localization, poly(A) tail plays important roles                analysis multisubunit complexes involved in <br /> mRNA stability translation (Gallie, 1991; Gallie                splicing, rRNA processing, nucleopore assembly <br /> &amp; Tanguay, 1994; Caponigro &amp; Parker, 1995; re-                     (Frank et al+, 1992; Venema &amp; Tollervey, 1996; Doye &amp; <br /> views, Sachs &amp; Wahle, 1993; Jacobson &amp; Peltz,                      Hurt, 1997)+ In order new factors potentially in- <br /> 1996; Sachs et al+, 1997)+ The function poly(A)                 volved mRNA 39-end formation, screened <br /> tail translation mediated Pab1p (Sachs &amp; Da-                  mutations confer synthetic lethality poly(A) <br /> vis, 1989; Tarun &amp; Sachs, 1995)+ Synthetic lethal in-                  polymerase temperature-sensitive mutant, pap1-7+ We <br /> teraction coimmunoprecipitation studies showed                     restricted analysis mutations conferred <br /> that eukaryotic translation initiation factor eIF4G                temperature-sensitive <span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> mu- <br /> binds Pab1p binding required stim-                 tants likely display deficient 39-end processing <br /> ulate translation polyadenylated mRNAs (Tarun &amp;                     activity own+ We isolated synthetic <br /> Sachs, 1996; Tarun et al+, 1997)+ Because eIF4G                   lethal, temperature-sensitive lcp mutations (for lethal <br /> interacts cap-binding protein eIF4E (Haghighat                conditional pap1 allele)+ Here report clon- <br />                                                                        ing characterization LCP5+ Surprisingly, this <br />                                                                        <span id='am-167' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-168' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> impaired processing ribosomal RNA <br />   Reprint requests to: Lionel Minvielle-Sebastia, Department Cell <br /> Biology, Biozentrum, University Basel, Klingelbergstrasse 70, CH-   precursors, showed normal pre-mRNA 39-end pro- <br /> 4056, Basel, Switzerland; e-mail: minvielle@ubaclu+unibas+ch+          cessing activity+ <br />                                                                    1357 <br />                               Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1358                                                                                                      T. Wiederkehr et al. <br />  <br />   The maturation rRNA involves complex series of <br /> modifications, notably pseudouridinylation, methylation, <br /> and exo- endonucleolytic cleavage events <br /> that lead formation 18S, 5+8S, 25S rRNAs+ <br /> 18S maturation involves cleavage sites the <br /> primary transcript, denoted A0 A2 + The pre-rRNA is <br /> processed sites A1 A2 , 59 39 mature <br /> 18S rRNA, respectively+ Cleavage site A2 separates <br /> the pathways maturation small large ribo- <br /> somal subunit rRNAs+ <br />   In lcp5-1 cells early cleavage steps A0 A2 <br /> were affected+ The mutant showed reduced <br /> amounts polysomes strong increase free <br /> 60S ribosomal subunits decreased 40S levels+ <br /> In addition, depletion protein vivo inhibited <br /> pre-rRNA processing led depletion 40S <br /> subunits, concomitant 60S accumulation, de- <br /> creased amounts polysomes+ Consistent its <br /> role pre-rRNA processing, Lcp5p <br /> protein located nucleolus associated <br /> with snR17, yeast homologue snoRNA U3+ <br />  <br />  <br /> RESULTS <br />  <br /> Synthetic lethal screen identifies gene <br /> with unknown function <br /> To identify components pre-mRNA 39-end <br /> processing machinery, screen synthetic lethal <br /> mutations poly(A) polymerase temperature- <br /> sensitive allele pap1-7 carried (Fig+ 1A)+ Mu- <br /> tant strains did grow absence the <br /> wild-type PAP1 gene identified plates contain-      FIGURE 1. Schematic representation synthetic lethal screen <br /> ing 5-fluoroorotic acid [5-FOA; Figs+ 1B,C]+ We ob-         pap1-7+ A: The starting strain, containing PAP1 disruption, <br /> tained 26 temperature-sensitive mutants+ For        wild-type PAP1 allele pap1-7 allele+ B: Genotype after <br />                                                             UV mutagenesis presence wild-type PAP1 gene+ C: <br /> them, tetrad analysis demonstrated genetic linkage        Resulting genotype counterselection wild-type PAP1 <br /> between temperature- 5-FOA-sensitive pheno-         gene 5-FOA plates+ 1: interaction supporting cell viability+ 2: <br /> types+ These mutations, termed lcp1 lcp5,        aberrant interaction fails support cell viability+ LCP : wild-type <br />                                                             allele UV mutagenesis, lcp 2 : mutated allele UV muta- <br /> linked plasmid-borne pap1-7 allele define        genesis+ LEU2, ADE2, URA3 : selectable marker genes+ <br /> five complementation groups (results shown)+ <br />    lcp mutations isolated outcrosses <br /> with W303 wild-type strain (see Table 1)+ Extracts <br /> were prepared mutant strains           resistant+ The gene previously characterized <br /> tested pre-mRNA 39-end processing vitro accord-      systematic Ty insertional analysis chromosome <br /> ing standard procedures (Minvielle-Sebastia et al+,      V+ Ty insertion YER127w resulted severe growth <br /> 1994)+ We extracts mutants         defects rich medium (Smith et al+, 1996)+ The pro- <br /> processed 32 P-labeled pre-mRNA substrate ex-       tein interact Ngg1p a <br /> tent comparable wild-type extract, ele-        two-hybrid screen (Martens et al+, 1996)+ Ngg1p acts <br /> vated temperature (Fig+ 2A; data shown)+ These          transcriptional coactivator/repressor in- <br /> results suggested mutated genes cod-      volved acetylation nucleosomal histones+ How- <br /> ing subunits essential factors involved mRNA      ever, best knowledge function been <br /> 39-end formation+                                           assigned YER127w far+ <br />    Below, characterization LCP5+ Wild-     LCP5 single copy gene+ It codes protein of <br /> type LCP5 cloned complementation              357 amino acids calculated molecular <br /> temperature-sensitive phenotype corresponding        mass 40+8 kD+ The predicted polypeptide high <br /> mutant+ Open reading frame YER127w (Mewes et al+,           content charged residues (33% D 1 E 1 K 1 R)+ <br /> 1997) sufficient render cells temperature-       Computational analysis detected putative nuclear lo- <br />                                   Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                          1359 <br />  <br />        TABLE 1+ Strains used study+ <br />  <br />        Strain         Relevant Genotype                                       Source <br />  <br />        BMA41          Mata/a, ade2-1/ade2-1, leu2-3,112/leu2-3,112, ura3-     A+ Baudin-Baillieu (Baudin-Baillieu et al+, 1997) <br />                       1/ura3-1, trp1D/trp1D, his3-11,15/his3-11,15, can1- <br />                       100/can1-100 <br />        JL17-3C        Mata, ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15,   J+ Lingner (Lingner, 1992) <br />                       can1-100, pap1::LEU2, pHCp50 ( CEN4 URA3 PAP1 ) <br />        JL17-3A        JL17-3C, Mata                                    J+ Lingner (Lingner, 1992) <br />        W303           ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15, can1-   R+ Rothstein (Columbia University, New York) <br />                       100 <br />        YTW1           ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15, can1-   work <br />                       100, lcp1-1 <br />        YTW4           ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15, can1-   work <br />                       100, lcp4-1 <br />        YTW5           ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15, can1-   work <br />                       100, lcp5-1 <br />        YTW6           Mata/a ade2-1/ade2-1, leu2-3,112/leu2-3,112, ura3-1/    work <br />                       ura3-1, trp1D/trp1D, his3-11,15/his3-11,15, can1-100/ <br />                       can1-100, LCP5/lcp5 ::TRP1 <br />        YTW7           Mata, ade2-1, leu2-3,112, ura3-1, trp1D, his3-11,15,    work <br />                       can1-100, lcp5::TRP1, pFLH-lcp5-1 ( CEN4 HIS3 <br />                       lcp5-1 ) <br />        YTW10          Mata, ade2-1, leu2-3,112, ura3-1, trp1D, his3-11,15,    work <br />                       can1-100, lcp5::TRP1, pFLU-LCP5 ( CEN4 URA3 <br />                       LCP5 ) <br />        YTW11          ade2-1, leu2-3,112, ura3-1, trp1D, his3-11,15, can1-    work <br />                       100, lcp5::TRP1, pGUR1-LCP5 ( CEN4 ADE2 Ubi-R- <br />                       LCP5 ) <br />        YTW12          Mata/a ade2-1/ade2-1, leu2-3,112/leu2-3,112, ura3-1/    work <br />                       ura3-1, trp1D/trp1D, his3-11,15/his3-11,15, can1-100/ <br />                       can1-100, lcp5::TRP1/lcp5::TRP1, pHH2-LCP5 <br />                       ( CEN4 ADE2 HA-His12-LCP5 ) <br />        YDK2-7A        Mata, ade2-1, leu2-3,112, ura3-1, trp1-1, his3-11,15,   P+ Linder (Kressler et al+, 1997) <br />                       fal1::HIS3MX6, pRS416 ( FAL1, URA3 ) <br />        Nop1-ProtA     Mata, ade2, leu2, lys1, ura3, nop1::URA3 pUN100-        D+ Tollervey (Jansen et al+, 1993) <br />                       ProtA-Nop1p <br />        Gar1-ProtA     Mata, ade2, his3, lys2, trp1, ura3,                     M+ Caizergues-Ferrer (Ganot et al+, 1997) <br />                       URA3::GAL10::gar1 pMCGZZ1-ProtA-Gar1p <br />  <br />  <br />  <br />  <br /> calization signal (Dingwall &amp; Laskey, 1991) po-               gene essential vegetative growth, URA3 - <br /> tential coiled coil region (Fig+ 2B)+ The amino acid                marked centromeric plasmid pFLU-LCP5 (see Materi- <br /> sequence protein shows similarities putative            als Methods) bearing wild-type LCP5 gene was <br /> orthologue (accession No+ SWISS-PROT: Q09713)                    transformed diploid YTW6 carrying hetero- <br /> Schizosaccharomyces pombe (26+3% identity),                  zygous disruption LCP5+ Meiosis led viable <br /> a CCAAT enhancer-binding-like protein Caenorhab-                 spores rich medium+ Two spores could <br /> ditis elegans (Wilson et al+, 1994)+                                grow SC lacking tryptophan, SC supple- <br />                                                                     mented 5-FOA+ Taken results dem- <br />                                                                     onstrate LCP5 essential cell viability+ <br /> LCP5 essential cell viability <br /> To determine LCP5 essential gene, a <br />                                                                     The lcp5-1 mutation leads 18S <br /> linear fragment 90% open reading frame <br />                                                                     rRNA underaccumulation <br /> (ORF) replaced TRP1 marker trans- <br /> formed diploid BMA41 (see Materials Meth-              Because extracts lcp5-1 cells showed wild-type <br /> ods)+ Integration correct locus verified              mRNA 39-end formation activity vitro, wanted to <br /> Southern blot analysis transformants+ Tetrad              test mutant cells decreased steady <br /> dissection revealed spores                state mRNA levels temperature shift (Minvielle- <br /> formed colonies rich medium (results shown)+                 Sebastia et al+, 1991)+ Total RNAs isolated <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> None viable spores grow minimal me-                 lcp5-1 mutant cells isogenic wild-type <br /> dium (SC) lacking tryptophan+ To demonstrate               sister spore+ Remarkably, RNAs prepared the <br />                                      Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1360                                                                                                         T. Wiederkehr et al. <br />  <br />                                                                          2 h, 18S rRNA content did decrease further <br />                                                                          (Fig+ 2C, compare lanes 5 6 lane 4)+ The de- <br />                                                                          crease 18S rRNA likely responsible en- <br />                                                                          hanced generation time lcp5-1 mutant cells <br />                                                                          (Table 2)+ This observation suggested LCP5 could <br />                                                                          involved rRNA metabolism+ <br />  <br />                                                                          LCP5 required pre-rRNA processing <br />                                                                          To verify mutant allele cloned gene was <br />                                                                          responsible observed phenotype, gene was <br />                                                                          isolated gap repair sequence deter- <br />                                                                          mined+ The mutant Lcp5-1p exhibited G197E muta- <br />                                                                          tion stop codon position 209+ The mutant allele <br />                                                                          introduced strain bearing lcp5 <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>null allele</span>, <br />                                                                          rescued pFLU-LCP5 (YTW10)+ The wild-type allele <br />                                                                          counterselected medium containing 5-FOA+ The <br />                                                                          resulting strain, YTW7, temperature sensitive and <br />                                                                          exhibited decrease steady state 18S rRNA <br />                                                                          level comparable mutant isolated screen <br />                                                                          (results shown)+ This experiment confirmed <br />                                                                          actually isolated wild-type allele LCP5 from <br />                                                                          genomic library complementation lcp5-1 <br />                                                                          mutant, suppressor+ In addition, ubiquitin- <br />                                                                          Lcp5p fusion protein cloned downstream re- <br />                                                                          pressible PGAL10/CYC1 hybrid promoter (Jenny et al+, <br />                                                                          1996)+ In construct, ubiquitin moiety precedes an <br />                                                                          arginine, haemagglutinin tag, histidine tag the <br />                                                                          ORF LCP5+ In vivo, ubiquitin cleaved off, ex- <br />                                                                          posing arginine N-terminus polypep- <br />                                                                          tide (hence called R-Lcp5p)+ According N-end <br />                                                                          rule, confers instability polypeptide (Var- <br />                                                                          shavsky, 1996)+ The construct transformed into <br />                                                                          YTW10 cells, containing lcp5::TRP1 deletion res- <br />                                                                          cued pFLU-LCP5+ After URA3 counterselection on <br />                                                                          5-FOA plates, strain expressing R-Lcp5p (YTW11) <br />                                                                          grew plates containing rich medium 2% galac- <br />                                                                          tose, plates containing 5% glucose (data not <br />                                                                          shown)+ In liquid cultures, shift glucose dramati- <br /> FIGURE 2. lcp5-1 mutant affected pre-mRNA 39-end pro- <br /> cessing shows underaccumulation 18S rRNA+ A: Cleavage             cally slowed cell growth 10&#226;&#8364;&#8220;12 h of <br /> and polyadenylation CYC1 precursor RNA vitro+ The 39-end       incubation+ RNAs extracted glucose-shifted cells <br /> processing activity wild-type (WT; lanes 2, 5), mutant    showed vivo depletion Lcp5p decreased the <br /> extracts lcp5-1 (lanes 3, 6) lcp4-1 (lanes 4, 7) assayed <br /> under standard conditions (see Materials Methods)+ The reac-         18S rRNA level extent similar lcp5-1 <br /> tions proceeded 70 min 30 8C (lanes 1&#226;&#8364;&#8220; 4) 31+5 8C (lanes 5&#226;&#8364;&#8220;   mutant cells (results shown)+ <br /> 7)+ The positions CYC1 precursor, 59 cleavage fragment,          Several cleavage steps required generate ma- <br /> and polyadenylated species indicated+ B: Primary structure of <br /> Lcp5p+ The predicted coiled coil region boxed+ Predicted, bipartite   ture 18S rRNA (see Fig+ 3A)+ The primary transcript <br /> nuclear localization signals (NLSs) underlined C: Electrophoretic    (35S) cleaved site A0 vitro homologue <br /> separation total RNA, isolated wild-type ( LCP5, lanes 1 3)   bacterial RNase III, Rnt1p (Elela et al+, 1996)+ The <br /> and mutant lcp5-1 cells (lanes 4 6)+ The cells grown room <br /> temperature shifted 37 8C 0, 60, 120 min (time indi-      pre-rRNA processed 59 end mature 18S <br /> cated top)+ The position mature 18S 25S rRNA species       rRNA 39 end internal transcribed spacer <br /> is indicated+                                                            1 (ITS1)+ Cleavage site A3 carried vitro <br />                                                                          MRP RNase (Lygerou et al+, 1996)+ Recent evi- <br />                                                                          dence suggests processing steps A0 A3 occur in <br /> temperature-sensitive strain grown room tempera-                      common complex vivo (Allmang et al+, 1996)+ It has <br /> ture (RT) showed substantial reduction 18S rRNA                     previously shown inhibition cleavage at <br /> compared wild-type control (Fig+ 2C, compare                      sites A0 A2 results accumulation 23S <br /> lanes 1 4)+ After temperature shift 37 8C                species (reviewed Venema &amp; Tollervey, 1995)+ Sim- <br />                                     Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                                  1361 <br />  <br />        TABLE 2+ In vivo phenotypes strains+ <br />  <br />                                                                                       %40S subunits               %40S subunits <br />                              Generation time           Generation time             wild-type strain             wild-type strain <br />        Strain                 (YPAD) min                (SC) min                       (RT)                         (37 8C) <br />  <br />        YTW10                       156                         219                          100 b                         100 b <br />        LCP5 <br />        YTW7                        381                         700                            23                             17 <br />        lcp5-1 <br />        YTW11                       n+d+                        360                            82                           n+d+ <br />        R-Lcp5p <br />        (GAL) <br />        YTW11 c                      &#226;&#8364;&#8221;                           &#226;&#8364;&#8221;                             10                           n+d+ <br />        R-Lcp5p <br />        (GLC) <br />  <br />           Generation times 40S small ribosomal subunit content strains used study+ For analysis ribosomal subunits <br />        37 8C, lcp5-1 cells grown RT, harvested, cultured overnight nonpermissive temperature+ <br />            a <br />              Doubling times determined 23 8C+ <br />            b <br />              The 40S subunits wild-type strain set 100%+ <br />            c <br />              R-Lcp5p expressing cells cultured overnight YPAD, containing 5% glucose+ Ratios determined calcu- <br />        lating areas subunit peaks &#226;&#8364;&#339;run off&#226;&#8364;? ribosomal subunit analysis described Materials Methods+ <br />           n+d+: determined+ <br />  <br />  <br />  <br /> ilarly, deletion snR10 leads production                  wild-type, depleted strains, indicating pro- <br /> 21S species (Tollervey, 1987; Morrissey &amp; Tollervey,                    cessing site A3 influenced Lcp5p (Fig+ 3B, <br /> 1993)+ To determine 18S rRNA production                     panel D)+ <br /> affected lcp5 mutants, RNAs isolated                         To determine decrease steady <br /> strains bearing null allele lcp5 rescued                 state 18S rRNA primary conse- <br /> the wild-type allele LCP5, temperature-sensitive                 quence LCP5 inactivation, RNAs isolated after <br /> allele lcp5-1 isolated gap repair,                 short glucose shift R-Lcp5p expressing cells <br /> YTW11 strain expressing R-Lcp5p cultured 8 h                     subjected Northern blot analysis probe <br /> glucose+ We analyzed pre-rRNA processing inter-                         A2 /A3 (Fig+ 4A)+ After 35 min, 21S 23S species <br /> mediates Northern blotting+ An oligonucleotide com-                  accumulated RNAs culture grown glu- <br /> plementary sequence upstream cleavage                       cose (Fig+ 4A, compare lanes 1 3)+ A decrease of <br /> site A0 (oligo 59A0 ) used probe (Fig+ 3B,                     27SA2 intermediate observed well+ Fig- <br /> panel A)+ In depleted (designated Glc)                 ure 4B shows reduction Lcp5p protein con- <br /> temperature-sensitive strains ( lcp5-1 ) aberrant RNA                centration correlated pre-rRNA processing <br /> species comparable size aforementioned 23S                    defects, assessed Western blot analysis+ These <br /> molecule observed+ Concomitantly, 35S pre-                      results showed depletion Lcp5p mutations <br /> rRNA precursor accumulated+ The stronger accumula-                      LCP5 lead predominant processing site A3 , <br /> tion 23S 35S species depleted R-Lcp5p                 skipping early cleavage steps A0 A2 + The <br /> expressing strain severe phenotype                    fast effects pre-rRNA processing glucose shift <br /> of strain compared temperature-sensitive                    suggested Lcp5p directly involved pre-rRNA <br /> mutant+ We probed membrane                        maturation+ <br /> oligonucleotide complementary sequences between <br /> cleavage sites D A2 (oligo D/A2 )+ The 20S pre- <br />                                                                         Efficient A0 A2 cleavages <br /> rRNA processing intermediate strongly reduced in <br />                                                                         dependent Lcp5p <br /> the temperature-sensitive mutant, strain <br /> depleted Lcp5p (Fig+ 3B, panel B)+ In addition, an-                 Cleavage individual sites analyzed primer <br /> other aberrant RNA comparable size 21S spe-                   extension+ RNAs isolated strains express- <br /> cies detected+ When blot hybridized                    ing R-Lcp5p following growth YP-Galactose (Gal) <br /> with oligonucleotide complementary sequences                      YP-Glucose (Glc)+ Figure 5A shows primer ex- <br /> between cleavage sites A2 A3 (oligo A2 /A3 ),                tension carried oligo E/C2 + Equal amounts of <br /> found 27SA2 intermediate severely re-                      extension products cleavage sites B1L , B1S and <br /> duced depletion Lcp5p temperature-                    A3 obtained (compare Fig+ 5A, lanes 1&#226;&#8364;&#8220;3 with <br /> sensitive mutant lcp5-1 (Fig+ 3B, panel C)+ Finally,                 lanes 4&#226;&#8364;&#8220;6)+ In contrast, RNAs terminating A2 were <br /> oligonucleotide complementary sequences                      strongly reduced depletion Lcp5p+ Longer ex- <br /> cleavage sites E C2 (oligo E/C2 ) detected compa-                   tension products visible lanes demonstrat- <br /> rable amounts 27S pre-rRNAs mutant,                       ing reduction artifactual <br />                                    Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1362                                                                                                                  T. Wiederkehr et al. <br />  <br />  <br />  <br />  <br />        FIGURE 3. Aberrant pre-rRNA processing lcp5 mutants+ A: Simplified overview 35S pre-rRNA processing yeast+ Pro- <br />        cessing intermediates designated respective sedimentation coefficient+ Processing sites indicated+ .: exo- <br />        nucleolytic digestion indicated site (adapted Venema &amp; Tollervey (1995))+ B: Northern blots RNA extracted <br />        R-Lcp5p expressing cells shifted 5% glucose 8 h (Glc), wild-type cells grown 5% glucose 24 8C ( LCP5 ) and <br />        lcp5-1 mutant cells grown 5% glucose RT ( lcp5-1 )+ The labeled oligonucleotide used probe blot were: Panel A: <br />        oligonucleotide 59 A0 ; Panel B: oligonucleotide D/A2; Panel C: oligonucleotide A2 /A3 ; Panel D: oligonucleotide E/C2 + The po- <br />        sitions sequences recognized probes indicated top+ Processing intermediates products indicated+ <br />  <br />  <br />  <br />  <br /> inhibition reverse transcriptase+ An oligonucle-                  mulation products extending transcription start <br /> otide hybridizing immediately upstream A1 cleav-                  site (compare Fig+ 5B, lanes 1&#226;&#8364;&#8220;3 lanes 4&#226;&#8364;&#8220;6)+ <br /> age site (Elela et al+, 1996) used primer                     These results obtained Northern <br /> extension shown Figure 5B+ We observed re-                          blot analyses demonstrated cleavage effi- <br /> duced A0-cleaved RNAs, accu-                        ciency sites A0 A2 reduced Lcp5p is <br />                                          Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                         1363 <br />  <br />                                                                              genesis+ The content ribosomal subunits mutant <br />                                                                              wild-type cells compared (see Materials and <br />                                                                              Methods)+ As predicted, 40S subunits <br />                                                                              relative 60S subunits reduced 23% lcp5-1 <br />                                                                              cells (Table 2)+ To test imbalance would <br />                                                                              lead defect translation, lcp5-1 mutant wild- <br />                                                                              type strains assayed growth rich medium <br />                                                                              containing translation inhibitors paromomycin, neo- <br />                                                                              mycin cycloheximide+ Growth mutant cells was <br />                                                                              inhibited neomycin concentration 1 mg/ml, a <br />                                                                              paromomycin concentration 1 mg/ml, cy- <br />                                                                              cloheximide concentration 0+04 mg/ml, the <br />                                                                              wild-type strain grow (T+ Wiederkehr, unpubl+ <br />                                                                              data)+ This suggested translation affected the <br />                                                                              lcp5-1 mutant strain+ <br />                                                                                 The reduction 40S subunit concentration sug- <br />                                                                              gested defect polysome formation+ The polysome <br />                                                                              profiles mutant wild-type cells determined <br />                                                                              low-salt buffer (Fig+ 6)+ In mutant, polysome peaks <br />                                                                              strongly reduced+ Because depletion of <br />                                                                              40S ribosomal subunits, large peak free 60S sub- <br />                                                                              units overlapped monosome peak (compare <br />                                                                              Fig+ 6A 6B)+ To demonstrate depletion 40S <br />                                                                              ribosomal subunits direct consequence mu- <br />                                                                              tations lcp5, R-Lcp5p expressing cells grown <br />                                                                              YP-Galactose transferred YP-Glucose for <br />                                                                              hours+ The polysome profile Lcp5p-depleted <br />                                                                              cells showed decrease 40S ribosomal subunits, as <br />                                                                              consequence increase free 60S ribosomal sub- <br />                                                                              units decrease polysomes (compare Fig+ 6C <br />                                                                              6D)+ The strength observed phenotype was <br />                                                                              good agreement levels 18S rRNA as <br />                                                                              judged ethidium&#226;&#8364;&#8220;bromide staining denaturing aga- <br />                                                                              rose gels (results shown)+ These results support <br />                                                                              conclusion defective pre-rRNA process- <br />                                                                              ing, assembly 40S ribosomal subunits transla- <br />                                                                              tion affected lcp5-1 mutations+ <br />  <br />  <br /> FIGURE 4. Pre-rRNA processing defects following vivo depletion <br />                                                                              Lcp5p predominantly located <br /> of Lcp5p+ A: YTW11 cells expressing R-Lcp5p grown rich me-           nucleolus <br /> dium containing galactose transferred rich medium con- <br /> taining galactose (Gal; lanes 1 2) glucose (Glc; lanes 3       The subcellular localization Lcp5p assessed <br /> and 4) 35 70 min+ Northern blot hybridization performed          indirect immunofluorescence wild-type cells+ A <br /> with probe A2 /A3 + Arrowheads indicate processing intermediates and <br /> products+ B: YTW11 cells expressing R-Lcp5p grown galac- <br />                                                                              strong signal observed nucleus (Fig+ 7A; <br /> tose shifted glucose+ At indicated times (in min glu-   Fig+ 7D phase contrast), weak <br /> cose shift; top), total protein extracted heating sample buffer,   background cytoplasmic staining+ When same <br /> separated SDS-PAGE, Lcp5p visualized Western blot <br /> analysis+ Affinity-purified anti-Lcp5p-antibody used 1:5,000 <br />                                                                              cells stained DAPI (Fig+ 7B), clear <br /> dilution+ Lane M: molecular weight markers kDa+                           signal Lcp5p antibodies confined part <br />                                                                              nucleus DAPI staining intense <br />                                                                              (see Fig+ 7C overlay Figs+ 7A 7B), sug- <br /> inactivated, processing downstream sites                          gesting Lcp5p antibodies decorated nuc- <br /> remains largely unaffected+                                                  leolus+ To confirm observation, diploid strain <br />                                                                              expressing HA-tagged Lcp5p prepared immu- <br />                                                                              nofluorescence, described Materials Meth- <br /> Translation affected lcp5 mutants <br />                                                                              ods+ As control nucleolar staining, monoclonal <br /> The reduction 18S rRNA caused lcp5-1 mutations                         antibody (mAb) A66 directed Nop1p, yeast <br /> suggested impairment 40S ribosomal subunit bio-                        homologue fibrillarin (Schimmang et al+, 1989), <br />                                   Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1364                                                                                                               T. Wiederkehr et al. <br />  <br />  <br />  <br />  <br />         FIGURE 5. Lcp5p depletion affects pre-rRNA processing sites A0 A2 + A: R-Lcp5p expressing cells grown on <br />         YP-Galactose transferred YP-Galactose (Gal, lanes 1 3) YP-Glucose (Glc, lanes 4 6) indicated <br />         times (in hours, top)+ RNA extracted subjected primer extension oligonucleotide E/C2 + Extensions sites <br />         B1L , B1S , A2 A3 indicated+ B: As A oligonucleotide 59A1 used detect A0-cleaved species+ <br />         Extension transcription start (11) extension site A0 indicated+ The products separated on <br />         denaturing 6% polyacrylamide/8+3 M urea gels+ Labeled Hpa II-restricted pBR322 fragments used size markers <br />         (lane M, nucleotides)+ <br />  <br />  <br />  <br /> was used time affinity-purified anti-                    combination antibodies bona fide <br /> bodies directed Lcp5p+ Confocal microscopy                      nucleoplasmic protein, poly(A) polymerase (Pap1p)+ <br /> showed signals Nop1p Lcp5p over-                        Whereas antibodies Pap1p uniformly stained <br /> lapped (Fig+ 7E,F; Fig+ 7G overlay) dem-                 nucleus, antibodies directed HA <br /> onstrates Lcp5p resides nucleolus+ In addition,             epitope localized Lcp5p restricted, crescent- <br /> antibodies directed HA epitope used                    shaped region nucleus (Fig+ 7I; Fig+ 7J an <br />                                       Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                      1365 <br />  <br />                                                                         7)+ The interaction Lcp5p U3 likely be <br />                                                                         specific significant amounts U3 be <br />                                                                         recovered extensive washes 300 mM <br />                                                                         NaCl (Fig+ 8, lane 6)+ In conditions, un- <br />                                                                         related protein A-tagged fusion (Nsp1p) did precip- <br />                                                                         itate RNA (data shown)+ <br />  <br />  <br />                                                                         fal1 lcp5-1 mutant alleles <br />                                                                         synthetic lethal <br />                                                                         Recently, Kressler et al+ (1997) shown FAL1 <br />                                                                         required 18S rRNA processing+ We crossed de- <br />                                                                         letion mutant fal1, rescued wild-type allele on <br />                                                                         centromeric plasmid (containing URA3 gene) with <br />                                                                         strain harboring lcp5 deletion, rescued plas- <br />                                                                         mid borne mutant lcp5-1+ After meiosis, isolated a <br />                                                                         strain contained chromosomal disruptions res- <br />                                                                         cued wild-type FAL1 gene mutant allele <br />                                                                         lcp5-1+ The cells transformed centromeric plas- <br />                                                                         mids containing fal1-1 fal1-9 mutant alleles <br />                                                                         (Kressler et al+, 1997; D+ Kressler, J+ la Cruz, M+ Rojo, <br />                                                                         &amp; P+ Linder, pers+ comm+)+ Independent transformants <br />                                                                         streaked minimal medium supplemented with <br />                                                                         5-FOA medium lacking drug (Fig+ 9)+ None of <br />                                                                         double mutants fal1-1 lcp5-1 fal1-9 lcp5-1 could <br /> FIGURE 6. Aberrant polysome formation lcp5 mutants+ Polysome <br /> profiles recorded deletion strains rescued LCP5 (A) lcp5-1   grow presence drug, growth <br /> (B) grown permissive temperature+ Polysome profiles R-Lcp5p     rates mutant fal1 strains not <br /> expressing cells grown YP-Galactose transferred        significantly reduced (results shown)+ This synthetic <br /> YP-Galactose (C) YP-Glucose (D) hours+ The positions <br /> of 40S 60S ribosomal subunit peaks, position      lethality provides additional evidence Lcp5p par- <br /> the 80S monosome peak indicated+                                    ticipates processing pre-rRNAs+ <br />                                                                            Cumulatively, experiments described above <br />                                                                         strongly support conclusion Lcp5p new <br />                                                                         essential component nucleolar pre-rRNA pro- <br /> overlay)+ The absence nucleoplasmic cytoplas-                    cessing machinery+ <br /> mic staining antibodies directed HA <br /> epitope suggested signal obtained affinity- <br /> purified polyclonal antibodies cross-                   DISCUSSION <br /> reactivity unrelated polypeptide+ These results                 We identified new protein (Lcp5p) involved in <br /> showed Lcp5p predominantly located                       pre-rRNA processing+ The LCP5 gene found <br /> nucleolus+                                                              screen synthetic lethal mutations the <br />                                                                         temperature-sensitive allele poly(A) polymerase, <br />                                                                         pap1-7+ Several experiments showed Lcp5p not <br /> Lcp5p associated U3 snoRNA <br />                                                                         essential pre-mRNA 39-end processing+ First, ex- <br /> In addition protein factors, formation 18S rRNA                   tracts prepared lcp5-1 mutant active in <br /> requires snoRNAs snR10, snR17 (U3), snR30                           vitro cleavage polyadenylation+ Second, im- <br /> snR128 (U14)+ To determine Lcp5p associ-                     munodepletion wild-type extracts antibodies di- <br /> ated snoRNAs, carried immunoprecipita-                      rected Lcp5p did lead inactivation the <br /> tion antibodies directed Lcp5p extracted               39-end processing activity (results shown)+ In addi- <br /> potentially associated RNAs (see Materials Meth-                    tion, temperature shift lcp5-1 cells 37 8C 2 h <br /> ods)+ As positive controls, extracts strains express-              did cause strong reduction steady state <br /> ing protein A-tagged fusions Gar1p Nop1p                    level ACT1 mRNA (results shown)+ In contrast, <br /> prepared+ The precipitates control polypep-                   temperature-sensitive mutations genes coding for <br /> tides contained snR10, U3, snR30 U14 (Fig+ 8,                       39-end processing components Rna14p, Rna15p <br /> lanes 2,3)+ Interestingly, antibodies directed                  Fip1p (Minvielle-Sebastia et al+, 1991; Minvielle- <br /> Lcp5p precipitated exclusively U3,                      Sebastia et al+, 1994; Preker et al+, 1995) did allow <br /> the tested snoRNAs precipitated pre-immune                     accumulation ACT1 mRNA shifting mu- <br /> serum (Fig+ 8, compare lanes 4 6 lanes 5                   tant cells nonpermissive temperature+ Finally, <br />                                  Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1366                                                                                                              T. Wiederkehr et al. <br />  <br />  <br />  <br />  <br />        FIGURE 7. Nucleolar localization Lcp5p indirect immunofluorescence+ Lcp5p localized wild-type strain <br />        BMA41 (A&#226;&#8364;&#8220;D) confocal microscopy strain YTW12 expressing HA-histidine-tagged fusion Lcp5p (E&#226;&#8364;&#8220;J)+ <br />        A: Cells decorated affinity-purified antibodies directed Lcp5p+ B: DAPI staining (see Materials and <br />        Methods)+ C: Overlay DAPI staining (red) Lcp5p staining (green)+ Red color assigned DAPI signal+ D: <br />        Nomarski phase contrast+ E,H: Affinity-purified antibodies directed Lcp5p Pap1p, respectively+ F,I: Monoclonal <br />        antibodies directed Nop1p HA epitope, respectively+ G: Overlay Nop1p (red) Lcp5p (green) staining+ <br />        J: overlay Pap1p (green) Lcp5p (red) staining+ The outline cells scale bar (5 mm, E H) depicted <br />        confocal panels+ <br />                                       Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                                1367 <br />  <br />  <br />  <br />  <br /> FIGURE 8. Antibodies directed Lcp5p precipitate U3 snoRNA+       FIGURE 9. Synthetic lethality fal1 lcp5 mutant alleles+ Strains <br /> Immunoprecipitation performed protein A-tagged fusions       deleted fal1 lcp5 rescued wild-type FAL1 gene <br /> Gar1p (lane 2), Nop1p (lane 3), antibodies directed     mutant alleles lcp5-1, fal1-1 fal1-9 (on centromeric plas- <br /> Lcp5p (lanes 4, 6 8) cognate preserum (lanes 5, 7 9)      mids) streaked SC 5-FOA-containing SC-plates+ Plates <br /> at indicated salt concentration (top)+ Lane 10: Mock precipitation   incubated 24 8C+ <br /> without antibody+ Precipitated RNAs subjected Northern hy- <br /> bridization oligonucleotides complementary snR10, snR17, <br /> snR30 snR128+ <br />                                                                          depletion protein, aberrant process- <br />                                                                          ing intermediates accumulated con- <br /> Lcp5p polypeptides                     trol levels half hour glucose repression+ <br /> copurify essential 39-end processing factors                       To test Lcp5p involved methyl- <br /> (Kessler et al+, 1996; Kessler et al+, 1997; Minvielle-                  ation pre-rRNA indirectly affect pro- <br /> Sebastia et al+, 1997; Preker et al+, 1997; Zhao et al+,                 cessing, performed pulse-chase labeling with <br />                                                                          3 <br /> 1997)+                                                                     H-methionine+ R-Lcp5p expressing cells shifted <br />   Instead, results showed LCP5 gene prod-                   hours YP-Glucose pulse-chase <br /> uct required ribosomal RNA maturation+ This con-                  labeling+ Northern-blot analyses showed accumulation <br /> clusion supported observation rRNA                       labeled 35S precursor lower abundance of <br /> isolated strain expressing R-Lcp5p                    18S rRNA suggesting aberrant processing was <br /> control repressible promoter accumulated aberrant                   overall methylation defect (results not <br /> processing species shift glucose compared                  shown)+ <br /> a control grown galactose+ Upon glucose shift, pre-                      The apparent similarity Lcp5p CCAAT <br /> rRNA processing rapidly affected leading                       enhancer-binding-like protein identification of <br /> inhibition cleavage processing sites A0 A2 ,                 ORF two-hybrid screen Ngg1p have <br /> indicated accumulation 23S 21S aberrant                    suggested role transcriptional activation+ To the <br /> processing species+ Similar effects observed                   best knowlegde, experimental data <br /> by depletion snoRNP components required                      hint involvement Lcp5p process+ How- <br /> for 18S rRNA maturation (Li et al+, 1990; Hughes &amp;                       ever, vivo depletion Lcp5p inacti- <br /> Ares, 1991; Morrissey &amp; Tollervey, 1993; Beltrame &amp;                      vation temperature-sensitive mutation led the <br /> Tollervey, 1995; Venema &amp; Tollervey, 1996)+ An over-                     accumulation unprocessed rRNA processing inter- <br /> night glucose shift R-Lcp5p-expressing cells re-                      mediates+ In addition, shown FAL1 and <br /> sulted depletion 18S rRNA comparable                      LCP5 interact genetically, suggesting primary involve- <br /> level observed temperature-sensitive lcp5-1                     ment Lcp5p pre-rRNA processing+ <br /> strain (results shown)+ Furthermore, inhibition                     As expected defective 18S rRNA production, <br /> of pre-rRNA processing direct effect                    mutant lcp5-1 showed reduced amounts 40S <br />                                 Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1368                                                                                                   T. Wiederkehr et al. <br />  <br /> ribosomal subunits hypersensitivity aminoglyco-         fects, like mRNA stability specific transcripts, con- <br /> sidic antibiotics (neomycin paromomycin)            tribute synthetic lethality+ <br /> cycloheximide+ Hypersensitivity paromomycin                We demonstrated Lcp5p present the <br /> been reported mutants NSR1, putative yeast          nucleolus Nop1p Lcp5p antibodies dec- <br /> nucleolin homologue (Lee et al+, 1992) FAL1             orated crescent-shaped regions previously <br /> (Kressler et al+, 1997)+ As result, gene products       shown correspond yeast nucleoli (Guthrie &amp; Fink, <br /> were shown involved 18S rRNA maturation+              1991)+ Furthermore, detection Pap1p nucleus <br /> Aminoglycosidic antibiotics inhibit early steps trans-      weaker regions Lcp5p signal was <br /> lation (Eustice &amp; Wilhelm, 1984b), act suppressors          located+ <br /> of nonsense codons (Palmer et al+, 1979; Singh et al+,            We shown snR17 tightly associated <br /> 1979) cause misreading translation (Eustice &amp;           Lcp5p snoRNAs <br /> Wilhelm, 1984a)+ Cycloheximide binds 60S ribo-          snR10, snR128, snR30, precipitated the <br /> somal subunit inhibits initiation elonga-         experiment+ Upon temperature shift mu- <br /> tion translation (Hampsey, 1997)+ The observed              tant lcp5-1 depletion protein the <br /> hypersensitivity consistent defective transla-       R-Lcp5p expressing strain, decrease steady- <br /> tion initiation lcp5 mutants consequence          state levels snR17 snR128 detected (re- <br /> reduced 40S ribosomal subunits+ Polysome             sults shown)+ Sof1p protein in <br /> profile analyses confirmed translation impaired        Saccharomyces cerevisiae associated with <br /> in lcp5-1 mutants+ Almost ribosomal subunits           U3 snoRNA (Jansen et al+, 1993)+ Recently, another <br /> were present subpolysomal fractions+ The low num-           component U3 snoRNP yeast, Mpp10p, been <br /> ber free 40S ribosomal subunits deficient 18S        identified (Dunbar et al+, 1997)+ Like Lcp5p, immuno- <br /> rRNA formation probably prevents efficient polysome            precipitation Mpp10p Sof1p precipitates snR17+ <br /> formation (see Table 2)+ Therefore, antibiotics lead         Further experiments underway examine <br /> to aberrant translation products inhibit initiation      Lcp5p directly associated com- <br /> translation enhance mutant phenotype              ponents U3 snoRNP+ Our results reported here <br /> <span id='am-4' about='Thesaurus:Tolerable_Level' typeof='owl:Thing'>tolerable level</span>+ It likely enhanced turnover    demonstrate Lcp5p novel essential protein <br /> R-Lcp5p vivo, cells cultured YP-          required rRNA maturation, suggest all <br /> Galactose, led increase free 60S ribosomal            factors involved complex processing reac- <br /> subunits 80S monosomes compared wild-               tion uncovered yet+ <br /> type control (compare Fig+ 6A 6C)+ This agree- <br /> ment Northern-blot analyses R-Lcp5p expressing <br />                                                                MATERIALS AND METHODS <br /> cells (compare Fig+ 4A, lanes 1 2, Fig+ 3B, panel <br /> C, lane LCP5 )+ The 18S rRNA correlates <br /> well strength observed phenotypes (re-         Yeast strains, media, genetic methods <br /> sults shown)+                                              The S. cerevisiae strains used study listed in <br />    Biochemical genetic analyses shown            Table 1+ Media genetic methods described else- <br /> poly(A)-bound Pab1p binds initiation factor eIF4G       (Guthrie &amp; Fink, 1991)+ Yeast cells transformed <br /> (Tarun &amp; Sachs, 1996; Tarun et al+, 1997) turn        treatment lithium acetate polyethylene glycol <br /> recognizes 59 cap structure cap binding            (Gietz et al+, 1992)+ <br /> protein eIF4E (Haghighat &amp; Sonenberg, 1997)+ Re- <br /> cently, shown mRNAs shorter         Synthetic lethal screen <br /> poly(A) tracts shifted monosomes poly- <br /> some profiles mutant pap1-1 cells ribosomal            The strain JL17-3C transformed pApap1-7 (Minvielle- <br /> subunits depleted time (Proweller &amp;            Sebastia et al+, 1994)+ About 7 3 10 4 transformants were <br /> Butler, 1997)+ Similarly, severe depletion       spread 42 SC plates, lacking adenine, uracil leucine+ <br />                                                                UV irradiation carried handheld germicidal <br /> 40S small ribosomal subunit pre-rRNA pro- <br />                                                                light 254 nm 80 s distance 50 cm+ Viable colonies <br /> cessing defects, observed lcp5 mutants, could <br />                                                                (1 3 10 4 ) replica-plated SC supplemented 5-FOA+ <br /> lead inefficient translation eventually cell death   About 0+5% cells able grow absence <br /> in lcp5-1 pap1-7 double mutant+ Other experiments          pHCp50, containing PAP1 (see Table 1)+ The mutants were <br /> also suggested synthetic lethality pap1           restreaked tested temperature-sensitive and <br /> and lcp5-1 mutants allele dependent+ We          5-FOA-sensitive phenotypes+ The temperature-sensitive mu- <br /> the reduction poly(A)-tail <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span>        tation lcp5-1 isolated outcrosses W303+ <br /> pap1 mutant alleles correlated hypersensitivity to <br /> translation inhibitors, mutant polysome profiles, and <br />                                                                Cloning LCP5 disruption <br /> synthetic lethality lcp5-1 temperature (T+ <br /> Wiederkehr, P+J+ Preker, &amp; L+ Minvielle-Sebastia,           Strain YTW5, containing temperature-sensitive allele lcp5-1 <br /> prep+)+ However, exclude additional de-         transformed genomic library constructed the <br />                                    Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                    1369 <br />  <br /> centromeric plasmid pFL39 (a gift Fran&#195;&#167;ois Lacroute)+ Sev-      trotransformation Escherichia coli according estab- <br /> eral thousand transformants replica-plated rich me-      lished procedures (Guthrie &amp; Fink, 1991)+ <br /> dium 37 8C+ One temperature-resistant colony picked, <br /> restreaked, plasmid isolated, genomic insert <br /> sequenced ends+ We used tFASTA algorithm (Pear-        RNA isolation, blotting, primer extension <br /> son &amp; Lipman, 1988) obtain complete sequence infor- <br />                                                                    RNAs extracted strains grown YPAD OD600 <br /> mation complementing genomic region (accession No+ <br />                                                                    ;1+ For temperature shift, equal volume prewarmed <br /> L11119; EMBL)+ The genomic region analyzed com- <br />                                                                    (51 8C) medium added culture incubated at <br /> plementation cloning restriction fragments pFL39 and <br />                                                                    37 8C+ Aliquots corresponding 10 OD600 removed+ For <br /> transformation lcp5-1 mutants+ As consequence frag- <br />                                                                    galactose-glucose shift, cells grown YP-Galactose <br /> ment containing YER127w detectable ORF- <br />                                                                    OD600 0+07&#226;&#8364;&#8220;0+5, culture centrifuged, resus- <br /> coding gene identified+ For disruption, genomic fragment <br />                                                                    pended YP, split parts, sugar concentration <br /> containing LCP5 cloned pUC19+ A TRP1 marker <br />                                                                    adjusted 5% glucose 2% galactose+ Aliquots <br /> cassette introduced N-terminal Nsi I a <br />                                                                    corresponding 3+5&#226;&#8364;&#8220;10 OD600 withdrawn times <br /> C-terminal Bbs I site+ The deletion construct cut from <br />                                                                    indicated added equal volume crushed ice+ After <br /> pUC19 restriction flanking sites polylinker and <br />                                                                    centrifugation 4 8C, cells washed 1 ml cold <br /> directly used transform BMA41 cells+ <br />                                                                    water cell pellets frozen dry ice+ RNA iso- <br />                                                                    lated hot acidic phenol (Collart &amp; Oliviero, 1994)+ Equal <br /> Synthetic lethality fal1 alleles lcp5-1                     amounts total RNA (5 mg) separated 1+2% <br />                                                                    formaldehyde/agarose gels used treatment <br /> Strain YDK2-7A crossed strain YTW7, diploids            primer extensions+ Northern blotting, hybridization, and <br /> were sporulated tetrads dissected+ A spore con-           primer extension according published proce- <br /> taining chromosomal disruptions fal1::HIS3MX6              dures (Venema &amp; Tollervey, 1996; Kressler et al+, 1997)+ The <br /> lcp5::TRP1 transformed mutant alleles fal1-1          sequences primers are: Oligonucleotide 59 A0 : GGT <br /> and fal1-9 centromeric plasmids (Kressler et al+, 1997)+        CTCTCTGCTGCCGG, oligonucleotide A2 /A3 : TGTTACCTC <br /> Transformants streaked SC- 5-FOA-containing             TGGGCCC, Oligonucleotide D/A2 : CGGTTTTAATTGTCCTA, <br /> SC-plates+                                                         Oligonucleotide E/C2 : GGCCAGCAATTTCAAGTTA, Oligo- <br />                                                                    nucleotide 59A1 : ACTATCTTAAAACAAGCAACAAGCAG <br />                                                                    (Elela et al+, 1996)+ For probing snoRNAD following <br /> Plasmids                                                           oligonucleotides used (Kressler et al+, 1997): Oligonu- <br />                                                                    cleotide anti-snR10: CCTTGCAACGGTCCTCATCCGGG, oli- <br /> Plasmids constructed standard procedures (Sam- <br />                                                                    gonucleotide anti-snR17: TTCGGTTTCTCACTCTGGGGTAC, <br /> brook et al+, 1989)+ pFL39-LCP5 derived com- <br />                                                                    oligonucleotide anti-snR30: GAAGCGCCATCTAGATG, oligo- <br /> plementing plasmid Pst I/Bam H I digestion religation+ <br />                                                                    nucleotide anti-snR128: GGAACCAGTCTTTCATCACC+ <br /> pFLU-LCP5 derived pFL39-LCP5 exchanging <br /> the TRP1 marker URA3 marker+ pGM-LCP5 con- <br /> structed cutting pGM10 (Martin &amp; Keller, 1996) Nde I/      Polysome analysis <br /> Bam H I inserting Nde I/Bam H I genomic fragment <br /> containing C-terminus Lcp5p 39 untranslated se-         Polysomes prepared according Kressler et al+ (1997)+ <br /> quences+ The Nde I site used insert N-terminus       To 200-ml cultures yeast cells (OD600 5 0+4&#226;&#8364;&#8220;1) YPAD, <br /> Lcp5p PCR primers 59-LCP5: GGGAATTC                    100 mg/ml cycloheximide added, mixed, cells <br /> CATATGTCTGAACTTAATGCATTATTA/ 39-LCP5: CTACAG                       kept ice 5 min+ The cells harvested, washed <br /> TCTCCTTTGAGCTCTATCCCAGGC overnight digestion                   lysis buffer (10 mM Tris-HCl [pH 7+5 (24 8C)], 100 mM NaCl, <br /> with Nde I+ The resulting plasmid cut Nsi I/Bam H I       30 mM MgCl2 , 100 mg/ml cycloheximide, 200 mg/ml hep- <br /> and Nsi I/Afl III fragment pFL39-LCP5 covering       arin), centrifuged, resuspended 1 ml lysis buffer+ After <br /> the open reading frame inserted+ For construction       10-s centrifugation, cells resuspended 3 vol- <br /> pGUR1-LCP5, PCR fragment containing flanking Not I sites         umes lysis buffer, 1 volume glass beads added, and <br /> (Primer 59 rGAL-LCP5: AAGGAAAAAAGCGGCCGCGAG                        cells broken vortexing times 30 s+ Be- <br /> AGGGAGTCACCATCACCATCACCATAT/ 39 rGAL-LCP5: T                       tween vortexing, cells kept ice 30 s+ The <br /> TCCTTTTTTGCGGCCGCCTACAGTCTCCTTTGAGC)                           lysate centrifuged 10 min 4 8C, 1/10 volume 80% <br /> amplified pGM-LCP5, inserted Not I site       glycerol added, extract frozen liquid ni- <br /> pGUR1 (Jenny et al+, 1996)+ pHH2-LCP5 constructed           trogen+ 0+2&#226;&#8364;&#8220;0+4 ml (OD260 5 25&#226;&#8364;&#8220; 40) loaded 12 ml <br /> inserting Not I cassette pGUR1-LCP5 containing            15+7&#226;&#8364;&#8220;54% (w/v) sucrose (Bio-Rad) gradients in <br /> histidine tag ORF Lcp5p Not I restricted pHH2      50 mM Tris-acetate [pH 7+5 (24 8C)], 50 mM NH4Cl, 12 mM <br /> (centromeric vector containing HA- histidine tag; L+      MgCl2 , 1 mM DTT, 100 mg/ml heparin+ The gradients <br /> Minvielle-Sebastia, unpubl+ vectors)+ The resulting construct      spun 39,000 rpm (204,000 3 g ) 165 min a <br /> contains HA- consecutive histidine tags+ For gap        TST41+14 rotor 4 8C+ The gradients unloaded the <br /> repair complementing plasmid cut Pst I            OD254 continuously monitored+ For determination ri- <br /> religated+ The resulting vector linearized Bam H I/       bosomal subunit monosome peaks elution profile, <br /> Apa I digestion transformed lcp5-1 mutant cells+          fractions collected RNA extracted ana- <br /> Transformants temperature sensitive plasmid           lyzed 18S 25S rRNA contents+ For analysis ribo- <br /> was recovered yeast plasmid DNA preparation elec-           somal subunits, cells grown above, sodium azide <br />                                  Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> 1370                                                                                                     T. Wiederkehr et al. <br />  <br /> was added culture final concentration 1 mM,    purified polyclonal antibodies directed Lcp5p (dilution <br /> cells shaken 15 min harvested extracted       1:100), mouse monoclonal antibody 12CA5 directed <br /> above+ Lysis buffer consisted 50 mM Tris-HCl [pH 7+4           HA epitope (Boehringer Mannheim; dilution 1:10) <br /> (24 8C)], 50 mM NaCl, 1 mM DTT+ The extracts             1 h moist chamber+ The coverslips washed as <br /> loaded 7&#226;&#8364;&#8220;50% (w/v) sucrose gradients lysis        overlayed 200 ml FITC-conjugated anti- <br /> buffer+ The gradients centrifuged 210 min 39,000      bodies directed rabbit IgG Texas-Red-conjugated <br /> rpm TST41+14 rotor 4 8C+ The gradients unloaded      antibodies (Vector) directed mouse IgG dilution <br /> and OD280 continuously monitored+                         1:250 1 h+ The coverslips washed above, DAPI <br />                                                                   added concentration 0+5 mg/ml, incubated for <br />                                                                   5 min+ The coverslips washed twice above+ The cells <br /> Expression recombinant protein                             mounted mowiol, followed incubation 10 min at <br /> production antibodies                                          50 8C+ <br /> pGM-LCP5 used transform BL21(DE3)LysS cells <br /> (Studier, 1991)+ The histidine-tagged protein purified     Microscopy <br /> Ni 21 -nitrilotriacetic-acid-agarose (Qiagen) columns accord- <br /> ing manufacturers&#226;&#8364;&#8482; instructions denaturing purifica-   FITC, Texas Red, DAPI viewed standard fluo- <br /> tion proteins+ A rabbit injected 50&#226;&#8364;&#8220;100 mg         rescence microscopy FITC, TRITC, UV filter, <br /> gel-purified protein 1 ml volume containing 50% adjuvant     respectively, Nikon Microphot-FXA+ FITC Texas Red <br /> (Specol)+ Affinity purification adsorption anti-   viewed confocal microscopy Noran Odyssey, <br /> serum nitrocellulose pieces (Schleicher &amp; Schuell) blotted     equipped argon/krypton laser, excitation wave- <br /> with recombinant Lcp5p protein, subsequent elution acidic      lengths 488 nm 568+2 nm+ <br /> buffer neutralization (Harlow &amp; Lane, 1988)+ <br />                                                                   Immunoprecipitation <br /> Pre-mRNA 39-end processing activity                               Total yeast extracts prepared glass bead lysis as <br />                                                                   described (Mitchell et al+, 1996) strain YTW12, ex- <br /> Preparation cell extracts vitro 39-end processing <br />                                                                   pressing HA-histidine-tagged Lcp5p+ Lysis buffer con- <br /> assays described (Minvielle-Sebastia et al+, <br />                                                                   sisted 20 mM Tris pH 8+0 [24 8C], 150 mM KCl, 5 mM <br /> 1994)+ <br />                                                                   MgCl2 , 0+01% Triton X-100, 10 mM vanadyl-ribonucleoside <br />                                                                   complex (GIBCO), 2 mM phenylmethylsulfonylfluoride <br />                                                                   (Serva), 1 mM DTT+ The lysate cleared centri- <br /> Computational analysis <br />                                                                   fugation 10 min 15,000 rpm+ For immunoprecipitation, <br /> For prediction coiled coil regions nuclear localiza-   50 ml immune preimmune serum coupled to <br /> tion signals used programs COILS (Lupas et al+, 1991)      80-ml-packed protein A sepharose (Pharmacia) Ipp500 <br /> and PSORT (Nakai &amp; Kanehisa, 1992), respectively+ Both        [10 mM Tris pH 8+0 (24 8C), 500 mM NaCl, 0+1% NaN3 , <br /> available menu TOOLS ExPASY Server           0+1% NP40]+ After overnight incubation 4 8C, pellets <br /> the Geneva University Hospital University Geneva       washed Ipp150 200 ml yeast extract were <br /> (internet access: http://expasy+hcuge+ch/www/expasy-top+          added equal volume Ipp150+ The mixture was <br /> html)+                                                            revolved 2 h 4&#226;&#8364;&#8220;10 8C+ After centrifugation super- <br />                                                                   natant discarded pellet washed times <br />                                                                   20 min 4 8C Ipp buffer indicated salt con- <br /> Immunofluorescence                                                centration+ Proteins digested Proteinase K 50 8C <br />                                                                   RNAs recovered PCI (phenol:chloroform: <br /> The strains grown OD600 ;1+ To 25 ml             isoamylalcohol, 25:24:1) extractions+ The RNA ethanol <br /> culture, 2+5 ml 37% formaldehyde added incu-          precipitated, loaded denaturing 8% polyacrylamide gel, <br /> bation continued 1 h+ The cells ice    transferred Hybond-N 1 filters (Amersham), sub- <br /> to 90 min, harvested resuspended 100 mM Tris/HCl           jected Northern hybridization specific oligonucleo- <br /> (pH 9+5, 24 8C), 10 mM DTT+ The fixated cells shaken         tides (see RNA isolation, blotting primer extension)+ <br /> for 8 min centrifuged+ The pellets resuspended        For immunoprecipitation strains expressing protein <br /> 1 ml buffer A [50 mM KPi (pH 7+4, 24 8C), 1+2 M Sorbitol],     A-tagged fusion proteins, 20 ml IgG Sepharose (Sigma) <br /> washed, resuspended buffer containing             equilibrated Ipp150, 100 ml extract and <br /> 0+3 mg/ml Zymolyase 100T (Seigakaku Co+)+ Following incu-         300 ml Ipp150 added processed above+ <br /> bation 20 min 30 8C, cells centrifuged, resus- <br /> pended buffer A centrifuged twice (once 24 8C, once <br /> at 4 8C)+ The pellet resuspended buffer A          <span id='am-2' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>Genbank accession number</span> <br /> suspension added poly-L-lysine (Sigma)-treated cov- <br /> erslips allowed dry minutes+ All subsequent       LPC5 assigned Genbank accession no+ 603366+ <br /> steps carried Buffer B (1 3 PBS, 1% BSA, 0+5% <br /> Tween-20) 24 8C+ The coverslips incubated             ACKNOWLEDGMENTS <br /> 20 min, washed times 5 min, incubated upside <br /> down 30 ml buffer B containing monoclonal mouse       We thank F+ Lacroute contributing yeast genomic li- <br /> antibody A66 directed Nop1p (dilution 1:30), affinity     brary+ We thank W+ Keller support, S+ Barabino, <br />                                       Downloaded www.rnajournal.org February 14, 2006 <br />  <br />  <br /> LCP5 required pre-rRNA processing                                                                                                1371 <br />  <br /> W+ Keller, P+ Linder, M+ Ohnacker helpful suggestions           Guthrie C, Fink GR+ 1991+ Guide Yeast Genetics Molecular <br /> and reading manuscript+ We thank P+J+ Preker, P+ Linder,            Biology+ San Diego, California: Academic Press+ <br />                                                                         Haghighat A, Sonenberg N+ 1997+ eIF4G dramatically enhances the <br /> J+ la Cruz, D+ Kressler, M+ Beltrame, D+ Tollervey                   binding eIF4E mRNA 59-cap structure+ J Biol Chem <br /> M+ Caizergues-Ferrer strains, materials advice,                 272 :21677&#226;&#8364;&#8220;21680+ <br /> E+ Hurt monoclonal antibody A66+ This work sup-             Hampsey M+ 1997+ A review phenotypes Saccharomyces cere- <br /> ported grants W+ Keller Kantons Basel,                visiae+ Yeast 13 :1099&#226;&#8364;&#8220;1133+ <br /> Swiss National Science Foundation, European Union               Harlow E, Lane D+ 1988+ Antibodies: A laboratory manual+ Cold Spring <br />                                                                             Harbor, New York: Cold Spring Harbor Laboratory Press+ <br /> via &#226;&#8364;&#339;Bundesamt f&#195;&#188;r Bildung und Wissenschaft&#226;&#8364;? (Bern)+                Huffacker TC, Hoyt MA, Botstein D+ 1987+ Genetic analysis the <br /> R+F+P+ supported grant T+ B&#195;&#188;rglin Swiss                yeast cytoskeleton+ Annu Rev Genet 21:259&#226;&#8364;&#8220;284+ <br /> National Science Foundation (No+ 31-050839)+                            Hughes JMX, Ares M+ 1991+ Depletion U3 small nucleolar RNA <br />                                                                             inhibits cleavage 59 external transcribed spacer yeast <br />                                                                             pre-ribosomal RNA impairs formation 18S ribosomal RNA+ <br /> Received June 8, 1998; returned revision June 30,                       EMBO J 10 :4231&#226;&#8364;&#8220; 4239+ <br /> 1998; revised manuscript returned July 27, 1998                         Jacobson A, Peltz SW+ 1996+ Interrelationships pathways of <br />                                                                             mRNA decay translation eukaryotic cells+ Annu Rev Bio- <br />                                                                             chem 65 :693&#226;&#8364;&#8220;739+ <br />                                                                         Jansen R, Tollervey D, Hurt EC+ 1993+ A U3 snoRNP protein with <br /> REFERENCES                                                                  homology splicing factor PRP4 Gb domains required for <br />                                                                             ribosomal RNA processing+ EMBO J 12 :2549&#226;&#8364;&#8220;2558+ <br /> Allmang C, Henry Y, Morrissey JP, Wood H, Petfalski E, Tollervey D+     Jenny A, Minvielle-Sebastia L, Preker PJ, Keller W+ 1996+ Sequence <br />    1996+ Processing yeast pre-rRNA sites A2 A3             similarity 73-kilodalton protein mammalian CPSF <br />    linked+ RNA 2 :63&#226;&#8364;&#8220;73+                                                    subunit yeast polyadenylation factor I+ Science 274 :1514&#226;&#8364;&#8220; <br /> Baudin-Baillieu A, Guillemet E, Cullin C, Lacroute F+ 1997+ Construc-       1517+ <br />    tion yeast strain deleted TRP1 promoter coding      Keller W+ 1995+ No end messenger RNA 39 processing! Cell <br />    region enhances efficiency polymerase chain              81:829&#226;&#8364;&#8220;832+ <br />    reaction-disruption method+ Yeast 13 :353&#226;&#8364;&#8220;356+                       Keller W, Minvielle-Sebastia L+ 1997+ A comparison mammalian <br /> Beltrame M, Tollervey D+ 1995+ <span id='am-245' about='xsp:base' typeof='owl:Thing'>Base</span> pairing U3              yeast pre-mRNA 39-end processing+ Curr Opin Cell Biol 9:329&#226;&#8364;&#8220; <br />    pre-ribosomal RNA required 18S rRNA synthesis+ EMBO J             336+ <br />    14 :4350&#226;&#8364;&#8220; 4356+                                                      Kessler MM, Henry MF, Shen E, Zhao J, Gross S, Silver PA, Moore <br /> Caponigro G, Parker R+ 1995+ Multiple functions poly(A)-            CL+ 1997+ Hrp1, sequence-specific RNA-binding protein that <br />    binding protein mRNA decapping deadenylation yeast+            shuttles nucleus cytoplasm, required for <br />    Genes &amp; Dev 9 :2421&#226;&#8364;&#8220;2432+                                                mRNA 39-end formation yeast+ Genes &amp; Dev 11:2545&#226;&#8364;&#8220;2556+ <br /> Colgan DF, Manley JL+ 1997+ Mechanism regulation mRNA            Kessler MM, Zhao J, Moore CL+ 1996+ Purification Saccharo- <br />    polyadenylation+ Genes &amp; Dev 11:2755&#226;&#8364;&#8220;2766+                               myces cerevisiae cleavage/polyadenylation factor I+ J Biol Chem <br /> Collart MA, Oliviero S+ 1994+ Preparation Yeast RNA+ In: Ausubel         271:27167&#226;&#8364;&#8220;27175+ <br />    FM, Brent R, Kingston RE, Moore DD, Seidman JG, Smith JA,            Kressler D, la Cruz J, Rojo M, Linder P+ 1997+ Fal1p essential <br />    Struhl K, eds+ Current Protocols Molecular Biology. New York:         DEAD-box protein involved 40S-ribosomal-subunit biogenesis <br />    Current Protocols+ pp 13+12+1&#226;&#8364;&#8220;13+12+2+                                   Saccharomyces cerevisiae. Mol Cell Biol 17 :7283&#226;&#8364;&#8220;7294+ <br /> Dingwall C, Laskey RA+ 1991+ Nuclear targeting sequences&#226;&#8364;&#8221;a con-         Lee W-C, Zabetakis D, M&#195;&#169;l&#195;&#168;se T+ 1992+ NSR1 required pre- <br />    sensus? Trends Biochem Sci 16 :478&#226;&#8364;&#8220; 481+                                 rRNA processing proper maintenance steady-state <br /> Doye V, Hurt E+ 1997+ From nucleoporins nuclear pore complexes+          levels ribosomal subunits+ Mol Cell Biol 12 :3865&#226;&#8364;&#8220;3871+ <br />    Curr Opin Cell Biol 9 :401&#226;&#8364;&#8220; 411+                                     Li HV, Zagorski J, Fournier MJ+ 1990+ Depletion U14 small nuclear <br /> Dunbar DA, Wormsley S, Agentis TM, Baserga SJ+ 1997+ Mpp10p,              RNA (snR128) disrupts production 18S rRNA Saccharomy- <br />    U3 small nucleolar ribonucleoprotein component required pre-         ces cerevisiae+ Mol Cell Biol 10 :1145&#226;&#8364;&#8220;1152+ <br />    18S rRNA processing yeast+ Mol Cell Biol 17 :5803&#226;&#8364;&#8220;5812+           Lingner J+ 1992+ Biochemical genetic analysis poly(A) poly- <br /> Elela SA, Igel H, Ares M+ 1996+ RNase III cleaves eukaryotic pre-           merase Saccharomyces cerevisiae+ Doctoral thesis+ Univer- <br />    ribosomal RNA U3 snoRNP-dependent site+ Cell 85 :115&#226;&#8364;&#8220;               sity Basel, Basel, Switzerland+ <br />    124+                                                                 Lupas A, Van Dyke M, Stock J+ 1991+ Predicting coiled coils from <br /> Eustice DC, Wilhelm JM+ 1984a+ Fidelity eucaryotic codon&#226;&#8364;&#8220;            protein sequences+ Science 252 :1162&#226;&#8364;&#8220;1164+ <br />    anticodon interaction: Interference aminoglycoside antibiotics+   Lygerou Z, Allmang C, Tollervey D, S&#195;&#169;raphin B+ 1996+ Accurate pro- <br />    Biochemistry 23 :1462&#226;&#8364;&#8220;1467+                                              cessing eukaryotic precursor ribosomal RNA ribonuclease <br /> Eustice DC, Wilhelm JM+ 1984b+ Mechanisms action aminogly-            MRP vitro+ Science 272 :268&#226;&#8364;&#8220;270+ <br />    coside antibiotics eucaryotic protein synthesis+ Antimicrob       Manley JL, Takagaki Y+ 1996+ The end message&#226;&#8364;&#8221;another link <br />    Agents Chemother 26 :53&#226;&#8364;&#8220; 60+                                             yeast mammals+ Science 274 :1481&#226;&#8364;&#8220;1482+ <br /> Frank D, Patterson B, Guthrie C+ 1992+ Synthetic lethal mutations       Martens JA, Genereaux J, Saleh A, Brandl CJ+ 1996+ Transcriptional <br />    suggest interactions U5 small nuclear RNA               activation yeast PDR1p inhibited association with <br />    proteins required second step splicing+ Mol Cell Biol         NGG1p/ADA3p+ J Biol Chem 271:15884&#226;&#8364;&#8220;15890+ <br />    12 :5197&#226;&#8364; <br /> </body></html>