<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2001 Genetics Society America <br />  <br />  <br />  <br /> Genetic Analysis Endocytosis Caenorhabditis elegans: Coelomocyte Uptake <br />                              Defective Mutants <br />  <br />                                                 Hanna Fares*,&#226;&#8364;&#160; Iva Greenwald* <br />  *Department Biochemistry Molecular Biophysics, Howard Hughes Medical Institute, Columbia University College Physicians and <br />  Surgeons, New York, New York 10032 &#226;&#8364;&#160;Department Molecular Cellular Biology, University Arizona, Tucson, Arizona 85721 <br />                                                         Manuscript received May 1, 2001 <br />                                                       Accepted publication May 25, 2001 <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                The coelomocytes Caenorhabditis elegans scavenger cells continuously nonspecifically <br />              endocytose fluid pseudocoelom (body cavity). Green fluorescent protein (GFP) secreted into <br />              pseudocoelom body wall muscle cells endocytosed degraded coelomocytes. We show <br />              toxin-mediated ablation coelomocytes results viable animals fail endocytose pseudocoelo- <br />              mic GFP, indicating endocytosis coelomocytes essential growth <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> C. elegans <br />              normal laboratory conditions. We examined known viable endocytosis mutants, performed <br />              RNAi known endocytosis genes, c oelomocyte uptake defective (Cup) phenotypes. We also <br />              screened new genes involved endocytosis isolating viable mutants Cup defects; screen <br />              identified 14 different genes, multiple alleles. A variety Cup terminal phenotypes were <br />              observed, consistent defects various steps endocytic pathway. Available molecular information <br />              indicates Cup mutant screen identified novel components endocytosis machinery that <br />              conserved mammals Saccharomyces cerevisiae, organism large- <br />              scale genetic screens endocytosis mutants performed. <br />  <br />  <br />  <br /> E    NDOCYTOSIS basic function eukaryotic cells <br />       leads internalization fluid the <br /> extracellular medium, <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake, recy- <br />                                                                            recycled surface. Many different protein compo- <br />                                                                            nents vesicles identified bio- <br />                                                                            chemical methods using mammalian cell culture systems. <br /> cling membrane components. In multicellular organ-                        Genetic analysis important tool for <br /> isms, endocytosis adapted specialized                     dissecting endocytosis. The extensive informa- <br /> cells specific functions, including downregula-                    tive screens yeast Saccharomyces <br /> tion signaling, synaptic vesicle recycling, antigen                 cerevisiae (Riezman 1985; Chvatchko et al. 1986; Munn <br /> presentation. There various routes endocy-                    Riezman 1994; Wendland et al. 1996; Zheng et al. <br /> tosis accomplished, involving different cellular struc-                 1998). For example, numerous insights machin- <br /> tures: clathrin-coated pits, non-clathrin-coated pits, ca-                 ery involved trafficking molecules the <br /> veolae, macropinosomes (Watts Marsh 1992;                          Golgi compartment, endosomes, vacuole (yeast <br /> Robinson et al. 1996; Mukherjee et al. 1997). The par-                     lysosome) emerged investigations (Lem- <br /> ticular mechanism used regulate pathways de-                      mon Traub 2000). In particular, studies have <br /> pends ligand internalized cell type                   crucial elucidating protein complexes involved <br /> examined. The multiplicity options adds complexities                    retrograde transport molecules Golgi <br /> that suggest endocytosis studied using                      compartment. <br /> several different approaches experimental systems.                       Multicellular organisms utilize endocytosis many <br />   Many steps endocytosis determined                    contexts unicellular organisms, raising <br /> through studies mammalian tissue culture cells                     possibility exhibit differences regulating <br /> are technically suited kind analysis (Watts                    endocytosis. One difference yeast cells <br /> and Marsh 1992; Robinson et al. 1996; Mukherjee et                         mammalian cells, example, adaptor <br /> al. 1997). In brief, vesicle containing internalized                 complexes needed clathrin function for <br /> medium buds membrane. The internalized                            endocytosis yeast (Huang et al. 1999; Yeung et al. <br /> substances sorted traverse series                    1999). Although screens mutants defective endo- <br /> membranous compartments, called endosomes,                          cytosis carried mammalian cells (Rob- <br /> either reaching lysosomes degraded getting                    bins et al. 1983; Krieger 1986; Colbaugh et al. 1988; <br />                                                                            Hobbie et al. 1994) Dictyostelium discoideum (Bacon <br />                                                                            et al. 1994), concerted effort screen for <br />   Corresponding author: Hanna Fares, University Arizona, Depart-        endocytosis mutants multicellular organism has <br /> ment Molecular Cellular Biology, Life Sciences South Bldg., <br /> Rm. 531, 1007 E. Lowell St., Tucson, AZ 85721.                             study receptor-mediated endocytosis of <br /> E-mail: fares@email.arizona.edu                                            yolk Caenorhabditis elegans oocytes (Grant Hirsh <br /> Genetics 159: 133&#226;&#8364;&#8220;145 (September 2001) <br />  134                                                  H. Fares I. Greenwald <br />  <br /> 1999). C. elegans particularly suited large-scale ge-         Diptheria toxin A fragment bearing K51E substitution <br /> netic screens, genes conserved                 replacing GFP plasmid pcc1. The Diptheria toxin A frag- <br />                                                                     ment bearing K51E substitution 10% activity of <br /> worms humans yeast (Lander et                  wild-type protein (DeLange et al. 1979). <br /> al. 2001; Venter et al. 2001). Indeed, molecular                   Transgenes strains: Extrachromosomal arrays gen- <br /> analysis C. elegans endocytosis mutants far dem-          erated coinjecting plasmid DNA 1&#226;&#8364;&#8220;50 &#226;?&#174;g/ml with <br /> onstrated novel components involved endocyto-               marker plasmids pRF4 [rol-6(su1006)] (Mello et al. 1991) <br /> sis unique multicellular organisms identified             pMH86 [dpy-20(&#226;&#171;&#185;)] (Han Sternberg 1991) the <br />                                                                     germline integrated irradiation using standard <br /> using genetic analysis C. elegans (see discussion).              methods (Mello Fire 1995). <br />   Here, report approach studying endocytic                    arIs37[pmyo-3::ssGFP] (Fares Greenwald 2001) was <br /> pathways C. elegans, using assay based uptake          integrating extrachromosomal array made <br /> of fluid-phase markers pseudocoelom coe-                injecting DNA mix containing pJF25 [pmyo-3::ssGFP] (62.5 <br /> lomocytes. We coelomocyte uptake                  &#226;?&#174;g/ml) &#226;&#171;&#185; pMH86 (20 &#226;?&#174;g/ml) &#226;&#171;&#185; pBluescript SK&#226;&#171;&#185; (100 &#226;?&#174;g/ <br />                                                                     ml; Stratagene, La Jolla, CA) dpy-20(e1282) (Hosono et <br /> (Cup) assay different fluid-phase markers               al. 1982) background. The isolated integrant backcrossed <br /> endocytosed coelomocytes. We behavior               times dpy-20(e1282) prior use. We checked 107 <br /> of known endocytosis mutants RNA-mediated inter-                Unc F1 progeny segregating dpy-20(e1282); arIs37[pmyo- <br /> ference (RNAi) known endocytosis genes using                3::ssGFP]/unc-54(e190) hermaphrodites: Dpy, sug- <br />                                                                     gesting arIs37[pmyo-3::ssGFP] maps close unc-54(e190) <br /> Cup assay results screen viable cup                chromosome I. However, eventually able con- <br /> mutants. This work establishes Cup assay power-            struct unc-54(e190) arIs37[pmyo-3::ssGFP] strain. <br /> ful genetic screen endocytosis mutants multicel-              arIs36[phsp::ssGFP] (Fares Greenwald 2001) made <br /> lular organism.                                                     integrating extrachromosomal array by <br />                                                                     injecting DNA mix containing pJF43 [phsp::ssGFP] (20 &#226;?&#174;g/ <br />                                                                     ml) &#226;&#171;&#185; pMH86 (25 &#226;?&#174;g/ml) &#226;&#171;&#185; pBluescript SK&#226;&#171;&#185; (100 &#226;?&#174;g/ml) <br />                                                                     dpy-20(e1282) background. The integrant backcrossed <br />                MATERIALS AND METHODS                                twice dpy-20(e1282) prior use. <br />                                                                        arEx218[pcc1::DT-AK51E] injecting DNA mix <br />    C. elegans methods strains: General methods the <br />                                                                     containing pJF142 [pcc1::DT-AK51E] (10 &#226;?&#174;g/ml) &#226;&#171;&#185; pRF4 (100 <br /> handling maintenance C. elegans described pre- <br />                                                                     &#226;?&#174;g/ml) dpy-20(e1282); arIs37[pmyo-3::ssGFP] hermaphro- <br /> viously (Brenner 1974). The wild-type parent strains <br />                                                                     dites. <br /> was C. elegans var. Bristol (Brenner 1974), sequence- <br />                                                                        All genetic markers used article found <br /> tagged site (STS) mapping used var. Bergerac strain DP13 <br />                                                                     referenced C. elegans II book Riddle et al. (1997), <br /> (Williams et al. 1992). Standard methods used ge-          eat-20(nc4) (Shibata et al. 2000), nrf-4(sa528), nrf- <br /> netic analysis (Brenner 1974), DNA microinjection (Fire             5(sa513), nrf-6(sa525) (Choy Thomas 1999), spg- <br /> 1986; Mello et al. 1991), double-stranded RNA synthesis         (e2335) (L. Avery, personal communication). <br /> and microinjection (Fire et al. 1998).                                 Molecular analysis: Standard methods used the <br />    Plasmid construction: pJF25 [pmyo-3::ssGFP] (Fares           manipulation recombinant DNA (Sambrook et al. 1989). <br /> Greenwald 2001) DNA encoding 79 amino acids           All enzymes New England Biolabs (Beverly, MA), <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> SEL-1 (Grant Greenwald 1997), including signal           unless indicated. PCR using Expand <br /> sequence, fused green fluorescent protein (GFP; Chalfie          Long Template PCR (Boehringer Mannheim, Mann- <br /> et al. 1994), S65T substitution, control     heim, Germany) according manufacturer&#226;&#8364;&#8482;s instructions. <br /> of myo-3 promoter.                                                 Mutant screen mapping: Adult dpy-20(e1282); arIs37 <br />    pJF43 [phsp::ssGFP] (Fares Greenwald 2001) DNA           [pmyo-3::ssGFP] hermaphrodites mutagenized using ethyl <br /> encoding SEL-1 (Grant Greenwald 1997) signal                methanesulfonate (EMS) previously described (Brenner <br /> sequence-GFP fusion control hsp16-41 ( Jones       1974). Five mutagenized parents placed single nem- <br /> et al. 1986) heat-shock promoter.                                   atode growth media (NGM; Brenner 1974) plate had <br />    <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> make construct expressing Diphtheria toxin A            previously seeded Escherichia coli OP50 (Brenner <br /> fragment (catalytic) specifically coelomocytes, used plas-    1974). The F2 hermaphrodites plate screened <br /> mid pcc1 (P. Sengupta, personal communication),           defect endocytosis coelomocytes (evidenced a <br /> made subcloning 3993-bp fragment, known              change GFP pseudocoelom and/or <br /> the upstream region unc-122 (P. Loria O. Hobert,             coelomocytes) using modified dissection microscope equipped <br /> personal communication), plasmid pPD95.77 (A. Fire,            fluorescence optics (Zeis, Thornwood, NY). Mutant F2 <br /> personal communication). pcc1 GFP coding            worms picked individual fresh NGM plates (with <br /> sequence unc-122 5&#226;&#172;&#732; sequences upstream unc-54 3&#226;&#172;&#732;-          OP50). <br /> untranslated sequences downstream. When transformed               We mutagenized 720 P0 hermaphrodites checked the <br /> worms, GFP exclusively seen coelomocytes, albeit       progeny &#226;&#181;&#8216;18,000 F1 hermaphrodites. We picked 175 <br /> in worms, coelomocytes fluorescent        adult F2 mutant hermaphrodites (not independent isolates) of <br /> or weakly fluorescent. In addition, occasional        97 did viable progeny. To ensure independent <br /> weak expression seen certain neurons.                    isolation mutants, chose mutant each <br />    We used polymerase chain reaction (PCR) amplify           plate, cases mutant phenotypes <br /> a 0.6-kb fragment plasmid pDT-AK51E (DeLange et al.            distinguishably different; cases, mutants <br /> 1979) using primers J197 (CACACAGGATCCAAAATGG                   picked plate mutations mapped to <br /> GCGCTGATGATGTTGTTGATTC; BamHI site underlined)                   different locations (see below). We saw F2 hermaph- <br /> and J198 (CACACAGAATTCTCATTAACGATTTCCTGCACA                         rodites strong accumulations GFP pseudo- <br /> GGC; EcoRI site underlined). The amplified PCR fragment          coelom arrested earlier larval stages; these <br /> was restriction digested BamHI EcoRI subcloned         counted picked separate plates. Finally, <br /> into sites pcc1. The resulting plasmid, pJF142,     kept 55 viable mutant hermaphrodites analysis. <br />                                                      C. elegans Endocytosis Mutants                                                 135 <br />  <br />    We backcrossed 55 mutant hermaphrodites            combinants unc-34(e566) unc-60(e723). Two of <br /> times dpy-20(e1282); arIs37[pmyo-3::ssGFP] males. We          recombinants picked cup-8(ar466). cup-8(ar466) <br /> mapped backcrossed mutations region genome            complemented nrf-5(sa513). <br /> using STS mapping, previously described (Williams et al.              cup-9 V: We isolated 22 recombinants sma-1(e30) unc- <br /> 1992). Individual cup mutations tested complementa-           39(e257)/cup-9(ar467); arIs37[pmyo-3::ssGFP] hermaphrodites. <br /> tion analysis Cup <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> cup rme           Of 14 Sma non-Unc recombinants, 6 picked cup-9 <br /> mutations mapped region             (ar467). Of 8 Unc non-Sma recombinants, 5 picked up <br /> other mutations (see below) region. We               cup-9(ar467). cup-9(ar467) complemented nrf-5(sa513) and <br /> determined 55 mutations defined 14 genes. Three              dyf-4(m158). <br /> of these, rme-1, rme-6, rme-8, previously identified         cup-10 V: We isolated 47 Dpy non-Lag recombinants from <br /> mutants defective yolk uptake oocytes (Grant Hirsh          lag-2(q420) dpy-11(e224)/cup-10(ar479); arIs37[pmyo-3::ssGFP] <br /> 1999). We named rest cup-1 cup-11 (cup             hermaphrodites: 4 picked cup-10(ar479). We tested whether <br /> c oelomocyte uptake defect); mapped using          cup-10(ar479) complement deletions region (D. <br /> conventional markers.                                                 Baillie, personal communication). cup-10(ar479) did not <br />    cup-1 I: We isolated 48 recombinants unc-101(m1) glp-         complement deficiencies sDf26, sDf28, sDf42, sDf50. <br /> 4(bn2) arIs37[pmyo-3::ssGFP]/cup-1(ar478) arIs37[pmyo-3::ssGFP]       cup-10(ar479) did complement sDf31, sDf32, sDf45, sDf46. <br /> hermaphrodites. Of 48 Unc non-Glp recombinants, 31                cup-10(ar479) complemented nrf-5(sa513). <br /> picked cup-1(ar478). cup-1(ar478) complemented nrf-4(sa528).          cup-11 X: We isolated 15 Unc non-Mup recombinants from <br />    cup-2 I: We isolated 55 recombinants dpy-24(s71) unc-         mup-2(e2346) unc-6(e78)/cup-11(ar472); arIs37[pmyo-3::ssGFP] <br /> 101(m1) arIs37[pmyo-3::ssGFP]/cup-2(ar506) arIs37[pmyo-3::ssGFP]      hermaphrodites: 9 picked cup-11(ar472). cup-11(ar472) <br /> hermaphrodites. None 23 Dpy non-Unc recombinants               complemented dyn-1(ky51ts). <br /> picked cup-2(ar506). All 32 Unc non-Dpy recombi-               Phenotypic analysis mutants: To check defects yolk <br /> nants picked cup-2(ar506). We isolated 36 recombi-            uptake oocytes, backcrossed mutant strains and <br /> nants unc-13(e51) dpy-24(s71) arIs37[pmyo-3::ssGFP]/cup-         unmutagenized parent strain grown NGM <br /> 2(ar506) arIs37[pmyo-3::ssGFP] hermaphrodites. None            (&#226;&#171;&#185;OP50) plates 20&#226;&#172;&#353; gravid adult hermaphrodites were <br /> 36 Dpy non-Unc recombinants picked cup-2(ar506). Thus,             checked Nomarski microscopy accumulation of <br /> cup-2(ar506) maps close dpy-24(s71). cup-2(ar506) comple-          lipid droplets pseudocoelom. This test be <br /> mented nrf-4(sa528) dyf-5(mn400).                                 sensitive score mutants mild defect <br />    cup-3 II: We isolated 38 recombinants dpy-10(e128) unc-       oocyte uptake. <br /> 4(e120)/cup-3(ar496); arIs37[pmyo-3::ssGFP] hermaphrodites.              To check defects uptake substances intestinal <br /> Of 15 Dpy non-Unc recombinants, 5 picked cup-                  cells lumen gut, placed L4 the <br /> 3(ar496). Of 23 Unc non-Dpy recombinants, 10 picked            backcrossed mutant strains wild-type parent indi- <br /> cup-3(ar496). cup-3(ar496) complemented nrf-6(sa525) dyf-         vidual NGM (&#226;&#171;&#185;OP50; cholesterol) plates 20&#226;&#172;&#353;; NGM <br /> 13(mn396).                                                            plates did added cholesterol contained <br />    cup-4 III: We isolated 10 recombinants sma-3(e491) unc-       rhodamine-conjugated dextran (Mr 11,000) added 0.1 mg/ <br /> 36(e251)/cup-4(ar494); arIs37[pmyo-3::ssGFP] hermaphrodites.          ml. Adult F1 hermaphrodites visually checked fluores- <br /> Of 6 Sma non-Unc recombinants, 4 picked cup-4(ar494).          cence microscopy uptake rhodamine-conjugated <br /> Of 4 Unc non-Sma recombinants, 1 picked cup-4(ar494).          dextran intestinal cells. The mutants did show <br />    cup-5 III: We isolated 18 recombinants sma-3(e491) unc-       defect retested regular NGM plates (&#226;&#171;&#185;OP50) that <br /> 36(e251)/cup-5(ar465); arIs37[pmyo-3::ssGFP] hermaphrodites.          lacking cholesterol rhodamine- <br /> Of 10 Sma non-Unc recombinants, 5 picked cup-5                 conjugated dextran. <br /> (ar465). Of 8 Unc non-Sma recombinants, 5 picked                  Injections pseudocoelom: Substances injected <br /> cup-5(ar465).                                                         pseudocoelom 7&#226;&#8364;&#8220;10 adult dpy-20(e1282); arIs37 <br />    cup-6 III: We isolated 23 recombinants unc-25(e156)           [pmyo-3:ssGFP] hermaphrodites kept 20&#226;&#172;&#353; NGM <br /> bli-5(e518)/cup-6(ar513); arIs37[pmyo-3::ssGFP] hermaphro-            (&#226;&#171;&#185;OP50) plates. The injected worms checked at <br /> dites. Of 20 Unc non-Bli recombinants, 19 picked cup-6         various times 24 hr. <br /> (ar513). None 3 Bli non-Unc recombinants picked                Chemicals: All chemicals used research pur- <br /> cup-6(ar513). We isolated 71 recombinants dpy-18            chased Sigma (St. Louis), Lucifer Yellow, <br /> (e364) unc-64(e246)/cup-6(ar513); arIs37[pmyo-3::ssGFP] her-          Bodipy-conjugated cholesterol, Transferrin Red, which <br /> maphrodites. Of 36 Dpy non-Unc recombinants, 35 picked            purchased Molecular Probes (Eugene, OR). <br /> up cup-6(ar513). None 35 Unc non-Dpy recombinants                 Heat-shock experiment: Hermaphrodites grown 20&#226;&#172;&#353; were <br /> picked cup-6(ar513). Thus, cup-6 maps extreme         synchronized collecting eggs bleaching and <br /> right arm chromosome III, single recombinant            allowing hatch NGM plates. The starved L1 larvae <br /> from mapping cross, confirm exact location         NGM (&#226;&#171;&#185;OP50) plates incubated 20&#226;&#172;&#353; for <br /> relative markers used. cup-6(ar513) complemented dyf-          2 days worms L4 stage. The L4 <br /> 2(m160).                                                              larvae heat-shocked 33&#226;&#172;&#353; 30 min being <br />    cup-7 IV: We isolated 15 recombinants lin-1(e1275)            returned 20&#226;&#172;&#353; checked different times. <br /> unc-33(e204)/cup-7(ar492); arIs37[pmyo-3::ssGFP] hermaphro-              Microscopy: Worms mounted 2% agarose pad in <br /> dites. Of 15 Lin non-Unc recombinants, 6 picked cup-7          3.5 &#226;?&#174;l 1% formaldehyde. Compound microscope images <br /> (ar492). cup-7(ar492) complemented unc-17(e245), dpy-13(e184),        taken Zeiss (Thornwood, NY) Axioplan. <br /> dpy-9(e2), ced-2(e1752), unc-33(e204), cha-1(p1152), eat-7(ad450), <br /> eat-10(ad606), dyf-3(m185). <br />    cup-8 V: We isolated recombinants unc-34(e566)                                     RESULTS <br /> unc-60(e677) dpy-11(e224)/cup-8(ar466); arIs37[pmyo-3::ssGFP]           Coelomocytes endocytose fluid pseudo- <br /> hermaphrodites. All Unc-34 non-Unc-60 recombinants <br /> picked cup-8(ar466). We picked 252 F1 Unc-34 her-             coelom: There coelomocytes pseudo- <br /> maphrodites unc-34(e566) cup-8(ar466)/unc-60(e723);              coelom adult hermaphrodites; present at <br /> arIs37[pmyo-3::ssGFP] hermaphrodites. Four re-          hatching, generated larval stage <br />  136                                             H. Fares I. Greenwald <br />  <br /> (Chitwood Chitwood 1974). The coelomocytes                 20&#226;&#172;&#353; (Figure 2). We that, worms are <br /> are membrane-bound compartments different           returned 20&#226;&#172;&#353;, observe GFP ap- <br /> sizes visualized light microscope (Fig-     pearing pseudocoelom, getting successively <br /> ure 1D). By electron microscopy, membranes              bright pseudocoelom obvious in <br /> the coelomocytes appear multiple invaginations         vesicles coelomocytes, finally disappearing <br /> indicating actively internalizing fluid     altogether (Figure 2). This suggests GFP is <br /> the pseudocoelom (White 1988; D. Hall, personal                taken gets degraded, rendered otherwise <br /> communication). Indeed, GFP secreted pseu-            nonfluorescent, coelomocytes. <br /> docoelom tissues taken predominantly              Toxin-mediated ablation coelomocytes: Coelomo- <br /> by coelomocytes (Fitzgerald Greenwald 1995;                cytes difficult kill laser ablation: apparently <br /> Grant Greenwald 1997; Fares Greenwald                  coelomocytes reported (via personal communication <br /> 2001; A. Fire, personal communication). We                White 1988) eliminated laser abla- <br /> transgenic worms GFP secreted             tion actually recovered retained function <br /> pseudocoelom attached signal sequence          (J. Kimble, personal communication). To assess im- <br /> and expressed body wall muscles. The GFP endocy-         portance coelomocyte function, used coelomocyte- <br /> tosed subsequently degraded, primarily coe-         specific promoter (P. Sengupta, personal communication; <br /> lomocytes. Such arIs37[pmyo-3::ssGFP] worms (materi-           P. Loria O. Hobert, personal communication) to <br /> als methods) display diffuse fluorescence           express variant diphtheria toxin, DT-AK51E (DeLange <br /> pseudocoelom bright fluorescent vesicles coe-           et al. 1979). We verified absence coelomocytes <br /> lomocytes (Figure 1, A&#226;&#8364;&#8220;D).                                     examining anatomy dpy-20; arIs37[pmyo- <br />    To assay specificity uptake      3::ssGFP]; arEx218[pcc1::DT-AK51E] hermaphrodites and <br /> coelomocytes, injected various substances          fate secreted GFP (materials methods). <br /> pseudocoelom. Many different substances proved           Most hermaphrodites lacked coelomocytes and <br /> endocytosed efficiently coelomocytes injected          accumulated GFP pseudocoelom (Figure 1G), <br /> into pseudocoelom: india ink (White 1988; J. Kim-          confirming GFP pseudocoelom removed <br /> ble, personal communication), rhodamine-dextran (Mr            primarily coelomocytes. We did observe effect <br /> 40,000; C. Winter D. Hirsh, personal commu-                viability motility strain easily <br /> nication), fluorescein isothiocyanate (FITC)-BSA (Mr           able maintain extragenic array genera- <br /> 71,000), FITC-lipopolysaccharide (LPS) Salmonella         tions, suggesting strong selection against <br /> typhimurium, FITC-dextran (Mr 35,600), Transferrin Red         it. We conclude coelomocyte function, par- <br /> (Mr 80,500), FITC (Mr 389.4), rhodamine (Mr 536.1)         ticular endocytosis coelomocytes, necessary for <br /> (Figure 1, E F; data shown).                           viability fertility worms normal laboratory <br />    We substances efficiently           conditions. <br /> endocytosed [FITC-LPS E. coli Lucifer Yellow             Involvement genes known function membrane <br /> (Mr 521.56)] substances [Bodipy-choles-          trafficking endocytosis coelomocytes: We tested <br /> terol FITC-conjugated dextran (Mr 2,000,000)]         proteins thought function mem- <br /> appeared completely excluded. Three             brane trafficking systems, directly indirectly, <br /> substances, FITC-LPS (E. coli), Bodipy-cholesterol,        play major role endocytosis coelomocytes. We <br /> Lucifer Yellow, available uptake        reduced eliminated activity genes the <br /> they appear adhere yolk granules (Bossinger          arIs37[pmyo-3::ssGFP] worms, using character- <br /> Schierenberg 1996; Matyash et al. 2001),             ized mutant alleles use RNA-mediated <br /> rarely taken coelomocytes (D. Hall, personal         interference (materials methods). We assayed <br /> communication). However, failure FITC-conju-            effect coelomocytes qualitatively examining <br /> gated dextran (Mr 2,000,000) taken coelomo-        level GFP pseudocoelom number <br /> cytes suggests coelomocytes nonspecifically en-       size vesicles containing GFP coelomocytes. <br /> docytose soluble material certain size           Conventional mutations tested: Because arIs37 <br /> pseudocoelom, FITC-dextran Mr 35,600 effi-            [pmyo-3::ssGFP] hermaphrodites reliably score <br /> ciently endocytosed.                                           older larvae adults endocytosis coelomo- <br />    Fate GFP endocytosed coelomocytes: To deter-          cytes, used mutant alleles did result in <br /> mine fate GFP secreted pseudocoelom,           zygotic lethality. We tested effect coelomocyte <br /> we did vivo pulse chase using strain carrying        endocytosis genes known affect <br /> transgene expresses ssGFP heat-shock pro-          membrane trafficking (snt-1, unc-26, unc-11, unc-13, <br /> moter (phsp::ssGFP). GFP produced normal         unc-64, unc-101, dyn-1, pod-1, unc-104, ced-2, ced-5, nrf-5, <br /> growth temperature 20&#226;&#172;&#353;, brief heat shock           nrf-6) influence actin cytoskel- <br /> 33&#226;&#172;&#353; results finite pulse GFP secreted         eton (unc-60, unc-115, cyk-1, mig-2, unc-73; Table 1), as <br /> pseudocoelom. The fate GFP fol-             various miscellaneous mutations used strain <br /> lowed function time returning worms          constructions (Table 1 legend). None mutations <br />                                                C. elegans Endocytosis Mutants                                            137 <br />  <br />  <br />  <br />  <br />                                                                                   Figure 1.&#226;&#8364;&#8221;Coelomocyte uptake sub- <br />                                                                                stances introduced pseudocoelom. <br />                                                                                Schematic drawing (A), epifluorescence (B <br />                                                                                C), Nomarski (D) micrographs a <br />                                                                                dpy-20(e1282); arIs37[pmyo-3::ssGFP] hermaph- <br />                                                                                rodite. (E F) Epifluorescence micrographs <br />                                                                                coelomocyte N2 hermaphrodite <br />                                                                                injected FITC-conjugated BSA rhoda- <br />                                                                                mine-conjugated dextran (Mr 40,000), respec- <br />                                                                                tively. (G) Epifluorescence micrograph a <br />                                                                                dpy-20(e1282); arIs37[pmyo-3::ssGFP]; arEx218 <br />                                                                                [pcc1::DT-AK51E] hermaphrodite the <br />                                                                                coelomocytes ablated expression <br />                                                                                Diphtheria toxin. B G low magnifi- <br />                                                                                cation images worms (anterior to <br />                                                                                left). Note presence GFP in <br />                                                                                pseudocoelom coelomocytes <br />                                                                                (five seen) B accumulation of <br />                                                                                GFP pseudocoelom G. (C D) <br />                                                                                Higher magnification fluorescence No- <br />                                                                                marski images coelomocyte, re- <br />                                                                                spectively. Arrows indicate coelomocytes <br />                                                                                B vesicles filled GFP C D. <br />                                                                                The shading represents GFP A. B, C, and <br />                                                                                D published (Fares and <br />                                                                                Greenwald 2001). Bar, 3.7 &#226;?&#174;m C. <br />  <br />  <br />  <br />  <br /> tested, dyn-1(ky51ts) (see below), resulted       (ky51ts); arIs37[pmyo-3::ssGFP] worms shifted <br /> a noticeable Cup phenotype. Where null alleles             25&#226;&#172;&#353; soon start displaying Cup phenotype, character- <br /> used, conclude activity gene tested      ized accumulation GFP pseudocoelom and <br /> is required coelomocyte endocytosis. Where hy-          greatly reduced number vesicles coelomocytes <br /> pomorphic alleles used, remains possible       (Figure 3). <br /> lack Cup phenotype reflects residual gene activity.           RNAi: We tested genes (Table 2) encode <br />    dyn-1(ky51ts) temperature-sensitive <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span>         proteins homologous, domains homolo- <br /> dynamin, large GTPase regulates early step         gous, proteins known function membrane traf- <br /> endocytosis involved cellular processes         ficking. For rme-1 cup-5, RNAi effective and <br /> (Clark et al. 1997). At permissive temperature, 20&#226;&#172;&#353;,        caused phenotype essentially identical that <br /> dpy-20(e1282); dyn-1(ky51ts); arIs37[pmyo-3::ssGFP] worms       seen null mutations genes (see below). vps- <br /> appear indistinguishable dpy-20(e1282); arIs37             34, sec-5, sec-8 showed strong accumula- <br /> [pmyo-3::ssGFP] worms. However, dpy-20(e1282); dyn-1            tion GFP pseudocoelom F1 worms, while <br />  138                                                 H. Fares I. Greenwald <br />  <br />  <br />  <br />  <br />                                                                                                         Figure 2.&#226;&#8364;&#8221;Behavior of <br />                                                                                                       ssGFP expression and <br />                                                                                                       secretion heat shock dpy- <br />                                                                                                       20(e1282); arIs36[phsp::ssGFP] <br />                                                                                                       hermaphrodites. Worms were <br />                                                                                                       grown 20&#226;&#172;&#353; (no heat shock), <br />                                                                                                       heat-shocked 33&#226;&#172;&#353; 30 min, <br />                                                                                                       20&#226;&#172;&#353; indi- <br />                                                                                                       cated times shown the <br />                                                                                                       micrographs. The row of <br />                                                                                                       micrographs higher mag- <br />                                                                                                       nification bot- <br />                                                                                                       tom row. Arrows indicate coe- <br />                                                                                                       lomocytes. <br />  <br />  <br />  <br />  <br /> the accumulation GFP worms injected vps-54                  When Cup RNAi phenotypes observed, was <br /> and clh-15 double-stranded RNA (dsRNA)                      F1 progeny parents injected <br /> weaker, noticeably higher mock-injected                  double-stranded RNA. The injection method ap- <br /> worms.                                                               peared work best genes tested: we <br />  <br />  <br />                                                              TABLE 1 <br />                                      Alleles tested effects coelomocyte endocytosis <br />  <br />        Genes (alleles)a            Type allele             Protein encoded                          Reference <br />        snt-1(n2665)                Null                       Synaptotagmin                 Nonet et al. (1993) <br />        unc-26(e205)                Null                       Synaptojanin                  Harris et al. (2000) <br />        unc-11(e47)                 Null                       AP180                         Nonet et al. (1999) <br />        unc-13(e51)b                Hypomorph                  C1/C2 domains                 Richmond et al. (1999) <br />        unc-64(e246)                Hypomorph                  Syntaxin                      Saifee et al. (1998) <br />        unc-101(m1)                 Null                       &#226;?&#174;1 (AP1)                      Lee et al. (1994) <br />        dyn-1(ky51ts)               Hypomorph                  Dynamin                       Clark et al. (1997) <br />        pod-1(ye11)c                Null                       Coronin                       Rappleye et al. (1999) <br />        unc-104(e1265)              Hypomorph                  Kinesin HC                    Otsuka et al. (1991) <br />        ced-2(e1752)                Hypomorph                  CrkII                         Reddien Horvitz (2000) <br />        ced-5(n1812)                Null                       DOCK180                       Wu Horvitz (1998) <br />        nrf-5(sa513)                Unknown                    Novel                         Choy Thomas (1999) <br />        nrf-6(sa525)                Null                       Novel                         Choy Thomas (1999) <br />        unc-60(e677)d               Hypomorph                  Actin binding                 McKim et al. (1994) <br />        unc-115(e2225)              Hypomorph                  Actin binding                 Lundquist et al. (1998) <br />        cyk-1(or36)e                Hypomorph                  Formin                        Swan et al. (1998) <br />        mig-2(mn28)                 Null                       Rho                           Zipkin et al. (1997) <br />        unc-73(e936)                Hypomorph                  Trio/kalirin                  Steven et al. (1998) <br />          All worms checked 20&#226;&#172;&#353;, temperature-sensitive mutants checked 25&#226;&#172;&#353;. Many <br />        worms dpy-20(e1282). <br />          a <br />            Other mutations tested include bli-5(e518), ced-9(n1950), dpy-5(e61), dpy-7(e88), dpy-9(e12), dpy-10(e128), dpy- <br />        11(e224), dpy-13(e184), dpy-14(e188), dpy-17(e164), dpy-18(e364), dpy-24(s71), eat-1(e2343), eat-20(nc4), emb-29(g52), <br />        emb-29(b262), glp-4(bn2ts), him-5(e1490), lag-2(q420ts), lin-1(n304), lin-1(e1275), lin-10(e1439), mab-18(bx23), rol- <br />        6(e187), sma-1(e30), sma-3(e491), spg-(e2335), sqt-3(sc63), unc-4(e120), unc-6(e78), unc-6(n102), unc-17(e245), unc- <br />        18(e81), unc-25(e156), unc-29(e1072), unc-30(e191), unc-31(e169), unc-32(e189), unc-33(e204), unc-34(e566), unc- <br />        36(e251), unc-39(e257), unc-42(e270), unc-54(e190), unc-69(e587), unc-76(e911), unc-78(e1217), unc-97(su110). <br />          b <br />            unc-13(e1091) checked. <br />          c <br />            unc-32(e189) pod-1(ye11); arIs37[pmyo-3::ssGFP] worms isolated checked unc-32(e189) pod- <br />        1(ye11)/qC1; arIs37[pmyo-3::ssGFP] parents. <br />          d <br />            unc-60(e723) unc-60(m35) checked. <br />          e <br />            cyk-1(or36) unc-36(e251); arIs37[pmyo-3::ssGFP] worms isolated checked cyk-1(or36) unc- <br />        36(e251)/qC1; arIs37[pmyo-3::ssGFP] parents. <br />                                                     C. elegans Endocytosis Mutants                                           139 <br />  <br />                                                                      [pmyo-3::ssGFP] worms EMS identified 55 viable <br />                                                                      mutants displayed differences endocytosis coe- <br />                                                                      lomocytes compared parent strain. Of muta- <br />                                                                      tions, 53 recessive define 14 complementation <br />                                                                      groups (Table 3; materials methods). We found <br />                                                                      3 complementation groups correspond to <br />                                                                      rme-1 rme-6 rme-8, genes defined screen for <br />                                                                      receptor-mediated endocytosis yolk (Grant and <br />                                                                      Hirsh 1999). The dominant mutations identified <br />                                                                      proved dominant-negative alleles rme-1 (Grant <br />                                                                      et al. 2001). On basis mapping data, inter se com- <br />                                                                      plementation tests, complementation tests with <br />                                                                      selected genes region (see materials <br />                                                                      methods), remainder 11 complementa- <br />                                                                      tion groups appear define new genes, have <br />                                                                      named cup. <br />                                                                         The coelomocyte phenotype cup mutants: In all <br />                                                                      mutants isolated, cup-5(ar465), gross phe- <br />                                                                      notype observed accumulation GFP the <br />                                                                      pseudocoelom, suggesting decrease in, absence <br />                                                                      of, endocytosis coelomocytes. cup-5(ar465) worms had <br />                                                                      opposite phenotype: reduced GFP pseudo- <br />                                                                      coelom greatly intensified GFP accumulation in <br />                                                                      coelomocytes, suggesting enhanced uptake de- <br />                                                                      creased degradation GFP coelomocytes (Fares <br />                                                                      Greenwald 2001). <br />                                                                         We examined closely phenotype coe- <br />                                                                      lomocytes mutant worms grown 20&#226;&#172;&#353; (Figure 4). <br />                                                                      Interestingly, coelomocytes exhibit range <br />                                                                      phenotypes, suggesting endocytic pathway <br />                                                                      perturbed steps and/or ways. We <br />                                                                      examples Figure 4, including no <br />                                                                      uptake GFP coelomocytes cup-10(ar479) worms <br />                                                                      (Figure 4B), accumulation small peripheral GFP- <br />                                                                      filled compartments cup-1(ar478) worms (Figure 4C), <br />    Figure 3.&#226;&#8364;&#8221;Effect dyn-1(ky51ts) coelomocyte endocyto-        accumulation vesicles GFP <br /> sis. All images dpy-20(e1282); dyn-1(ky51ts); arIs37[pmyo-    (indistinguishable wild-type profile level <br /> 3::ssGFP] hermaphrodites grown 20&#226;&#172;&#353; (A&#226;&#8364;&#8220;C) shifted           analysis) cup-11(ar472) worms (Figure 4D), ap- <br /> 8 hr 25&#226;&#172;&#353; (D&#226;&#8364;&#8220;F). (B C; E F) Higher magnification <br /> fluorescence Nomarski images coelomocytes,           pearance large vacuoles little GFP cup-8(ar466) <br /> respectively. Arrows indicate coelomocytes (not visible D         worms (Figure 4E), accumulation GFP in <br /> due decreased uptake GFP). Note presence            large vacuoles cup-5(ar465) worms (Figure 4F). <br /> fewer smaller vesicles coelomocytes shown E            Additional phenotypes: We checked mutants <br /> compared shown B.                                        phenotypes indicate defects endocyto- <br />                                                                      sis cell types (Table 3). On basis our <br />                                                                      limited analysis, seven complementation groups <br /> tried soaking worms double-stranded RNA (Tabara                   deficient endocytosis coelomocytes, and <br /> et al. 1998), saw little effect (data shown).           seven affected endocytosis tissues. <br /> Feeding worms bacteria expressing double-stranded                       Uncoordinated phenotype: An Unc phenotype be <br /> RNA (Timmons Fire 1998) similar effect                  indicative defects synaptic vesicle recycling (for <br /> RNA injection, efficient (data shown).              example, Nonet et al. 1993, 1999; Harris et al. 2000). <br /> We use RNAi assay genes               The cup mutants display Unc defect are <br /> (e.g., clathrin heavy chain, RAB-5, RAB-11)             alleles cup-7, slowly have <br /> known involved endocytosis resulted                slight tendency curl. All mutants have <br /> in lethality F1 progeny injected worms and/            normal movement. <br /> or sterility injected worms.                                  Oocyte uptake yolk: Accumulation yolk pseu- <br />    Screen new mutants defective endocytosis                docoelom visualized bright birefringence us- <br /> coelomocytes: We mutagenized dpy-20(e1282); arIs37                   ing Nomarski microscopy (Kimble Sharrock <br />  140                                              H. Fares I. Greenwald <br />  <br />                                                           TABLE 2 <br />                                      <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> test coelomocyte endocytosis <br />  <br />                                                                       Other <br />         Clone           ORF                  Protein                phenotypes                 Reference <br />  <br />                                              A. Cup phenotype    seen <br />         yk271a1     W06H8.1        RME-1 (EH domain)              Gut vacuoles      Grant et al. (2001) <br />         yk561c9     R13A5.1        CUP-5 (mucolipin-1)                  &#226;&#8364;&#8221;           Fares Greenwald (2001) <br />         yk2e6       B0025.1        VPS-34 (PI-3 kinase)           Slow growth       Stack Emr (1994) <br />         yk557a2     C07H4.2        CLH-15 (chloride channel)            &#226;&#8364;&#8221;           Wang et al. (2000) <br />         yk3e11      T21C9.2        VPS-54                               &#226;&#8364;&#8221;           Conboy Cyert (2000) <br />         yk38b5      Y106G6H.7      SEC-8 (exocyst)                Gut vacuoles,     TerBush et al. (1996) <br />                                                                     slow growth <br />         yk52b9      T23G7.4        SEC-5 (exocyst)                Gut vacuoles      TerBush et al. (1996) <br />                                            B. Cup phenotype seen <br />         yk66d7      H22K11.1       ASP-3 (aspartic protease)         &#226;&#8364;&#8221;              Tcherepanova et al. (2000) <br />         yk98g7      C03B1.12       LMP-1 (LAMP)                      &#226;&#8364;&#8221;              Kostich et al. (2000) <br />         yk397e6     F58G6.1        Amphiphysin                       &#226;&#8364;&#8221;              Lichte et al. (1992) <br />         yk326a2     F45E1.17       Syndapin                          &#226;&#8364;&#8221;              Qualmann et al. (1999) <br />         yk261h11    Y55D9A.1       SEC-7, PHa domain                 &#226;&#8364;&#8221;              Achstetter et al. (1988) <br />         yk392e11    K06H7.4        GRP-1/ SEC-7, PHa domain          &#226;&#8364;&#8221;              Achstetter et al. (1988) <br />         yk33e1      M02B7.5        SEC-7 domain                      &#226;&#8364;&#8221;              Achstetter et al. (1988) <br />         yk61c3      T13F2.8        CAV-1 (caveolin)                  &#226;&#8364;&#8221;              Tang et al. (1997) <br />         yk411g3     F36F2.4        Syntaxin 12/7                     &#226;&#8364;&#8221;              Jantsch-Plunger Glotzer <br />                                                                                       (1999) <br />         PCRb        C01A2.1        Synaptobrevin-like                   &#226;&#8364;&#8221;           Baumert et al. (1989) <br />         PCRb        F23H12.1       Synaptobrevin                        &#226;&#8364;&#8221;           Baumert et al. (1989) <br />         yk481g10    T04C10.2       Epsin                                &#226;&#8364;&#8221;           Chen et al. (1998) <br />         yk21f1      C04F6.4        DAip1 (WD40 repeats)          Uncoordinated      Konzok et al. (1999) <br />                                                                    movement <br />         yk228a8     W03C9.3        RAB-7                                &#226;&#8364;&#8221;           Chavrier et al. (1990) <br />         yk74d9      W09G10.4       ADAPTIN (AP3)                        &#226;&#8364;&#8221;           Simpson et al. (1997) <br />         yk507b6     T05B11.3       Clathrin light chain                 &#226;&#8364;&#8221;           Creutz Harrison (1984) <br />         yk63d5      M01D7.2        SCAMP                                &#226;&#8364;&#8221;           Singleton et al. (1997) <br />         yk445h11    C01F6.1        Copine                               &#226;&#8364;&#8221;           Creutz et al. (1998) <br />         yk71c12     T07A9.10       SEC-1                                &#226;&#8364;&#8221;           Aalto et al. (1991) <br />         yk99b4      R11B5.1        Drainin                              &#226;&#8364;&#8221;           Becker et al. (1999) <br />         yk5h12      ZK370.3        Hip1R                                &#226;&#8364;&#8221;           Seki et al. (1998) <br />         yk304g6     R01H10.3c      Coronin                              &#226;&#8364;&#8221;           Maniak et al. (1995) <br />         yk81h9      F25H2.5        Nm23                          Sterile F1         Steeg et al. (1988) <br />         PCRb        F55A3.6        Nm23                                 &#226;&#8364;&#8221;           Steeg et al. (1988) <br />           dsRNA injected dpy-20(e1282); arIs37[pmyo-3::ssGFP] worms resulting F1 checked at <br />         20&#226;&#172;&#353;. ORF, open reading frame. <br />           a <br />             PH, pleckstrin homology. <br />           b <br />             PCR signifies cDNA clone available ORF, amplified portion ORF <br />         using primers complementary ends sequences T3 T7 polymerase added the <br />         primers&#226;&#8364;&#8482; ends, respectively. Double-stranded RNA directly amplified fragment. <br />           c <br />             R01H10.3 distinct POD-1, second coronin C. elegans. dsRNA yk304g6 injected <br />         unc-32(e189) pod-1(ye11)/qC1; arIs37[pmyo-3::ssGFP] worms homozygous pod-1(ye11) F1 checked, <br />         similarly effect endocytosis coelomocytes. <br />  <br />  <br /> 1983). Using assay, hermaphrodites bearing cup-             added growth medium continued <br /> 1(ar478), cup-3(ar496), cup-6(ar513), cup-7(ar492) ac-        growth C. elegans culture (Brenner 1974) that <br /> cumulated yolk granules pseudocoelom                 required endocytic pathways (Hoekstra <br /> than wild-type parent strains.                               van IJzendoorn 2000). In presence supple- <br />   Intestinal uptake: Rhodamine-dextran (Mr 40,000)               mented cholesterol, rme-8(ar486) defective for <br /> present growth medium endocytosed intesti-          uptake rhodamine-dextran intestinal cells. How- <br /> nal cells. We examined cup mutants intestinal            ever, absence supplemented cholesterol, both <br /> accumulation rhodamine-dextran presence             cup-1(ar478) rme-8(ar486) showed severe defect in <br /> absence cholesterol, membrane component                rhodamine-dextran uptake intestinal cells. The defect <br />                                                   C. elegans Endocytosis Mutants                                            141 <br />  <br />                            TABLE 3                                 cup-1 rme-8 mutants, endocytosis sufficiently im- <br />                   Coelomocyte uptake mutants                       paired rhodamine-dextran accumulation be <br />                                                                    seen. <br />                                                    Other <br />            No.          Map position           endocytic <br /> Gene       alleles          (interval)a          phenotypesb                            DISCUSSION <br /> cup-1         2         I (unc-101 glp-4)      Yolk, Int           We shown coelomocytes continuously <br /> cup-2         1         I (close dpy-24)          &#226;&#8364;&#8221;             efficiently endocytose fluid pseudocoelom <br /> rme-8         1         I (unc-13 lin-10)c     Yolk, Int <br /> cup-3         5         II (dpy-10 unc-4)      Yolk             essential survival animal. We <br /> cup-4         1         III (sma-3 unc-36)        &#226;&#8364;&#8221;             used ability coelomocytes endocytose fluid- <br /> cup-5         1         III (sma-3 unc-36)        &#226;&#8364;&#8221;             phase marker GFP pseudocoelom basis <br /> cup-6         2         III (ND)                  Yolk             identifying mutants defective endocytosis. A re- <br /> cup-7         2         IV (lin-1 unc-33)      Unc, Yolk        verse genetic approach, using RNA-mediated interfer- <br /> cup-8         3         V (unc-34 unc-60)         &#226;&#8364;&#8221;             ence, established certain known genes required <br /> cup-9        15         V (sma-1 unc-39)          &#226;&#8364;&#8221; <br />                                                                    coelomocyte endocytosis blocking endocy- <br /> cup-10        1         V (lag-2 let-478)         &#226;&#8364;&#8221; <br /> rme-1         5         V (unc-46 dpy-11)d     Yolk             tosis leads Cup phenotype. A forward genetic screen <br /> cup-11        9         X (mup-2 unc-6)           &#226;&#8364;&#8221;             mutants identified 14 cup genes, 11 appear <br /> rme-6         7         X (close unc-1)e       Yolk             new. Here, discuss learned about <br />                                                                    process endocytosis coelomocytes our <br />    ND, determined. <br />    a <br />      See materials methods mapping information.            genetic analysis evaluate potential cup <br />    b <br />      Other phenotypes checked uncoordinated move-             mutant screen identifying new endocytosis genes. <br /> ment (Unc), yolk accumulation pseudocoelom (Yolk),             We soluble compounds injected into <br /> and endocytosis intestinal cells (Int). See results de-     pseudocoelom taken coelomocytes. <br /> tails.                                                             The compounds did endocytosed be <br />    c <br />      Zhang et al. (2001). <br />    d <br />      Grant et al. (2001).                                          divided categories. One category comprises <br />    e <br />      B. Grant D. Hirsh (personal communication).               compounds appeared associate yolk gran- <br />                                                                    ules, appear excluded coelomocytes <br />                                                                    (D. Hall, personal communication) size, <br /> appeared fall anterior-posterior gradient,           hydrophobicity, other, unknown, property. The <br /> cells posterior half showing uptake            category single member, FITC-dextran <br /> the rhodamine-dextran. This gradient reflect             (Mr 2,000,000), did appear associate with <br /> concentration ingested substance, pro-            yolk granules; smaller dextran polymer was <br /> gressively decrease passage lu-             efficiently endocytosed coelomocytes, size may <br /> men. Thus, nonmutant worms, endocytosis efficient            important factor determining ability a <br /> enough posterior cells scavenge detect-          compound taken up. One reason size be <br /> able rhodamine-dextran lumen,                  important electron-dense glycocalyx <br />  <br />  <br />  <br />  <br />                                                                                                   Figure 4.&#226;&#8364;&#8221;Phenotype of <br />                                                                                                coelomocytes various <br />                                                                                                cup mutant hermaphrodites. <br />                                                                                                Shown fluorescence (1) <br />                                                                                                corresponding Nomarski <br />                                                                                                micrographs (2) wild-type <br />                                                                                                worms (A) worms carrying <br />                                                                                                following mutations: (B) <br />                                                                                                cup-10(ar479), (C) cup-1(ar478), <br />                                                                                                (D) cup-11(ar472), (E) cup-8 <br />                                                                                                (ar466), (F) cup-5(ar465). <br />                                                                                                All worms dpy-20 <br />                                                                                                (e1282); arIs37[pmyo-3::ssGFP]. <br />                                                                                                F published <br />                                                                                                (Fares Greenwald 2001). <br />  142                                              H. Fares I. Greenwald <br />  <br /> or basement membrane surrounding coelomocyte               membrane trafficking, did involved in <br /> (White 1988; D. Hall, personal communication),            endocytosis coelomocytes. For example, null alleles <br /> may effect act sieve. Analysis uptake         snt-1 Synaptotagmin, unc-26 Synaptojanin, unc-11 <br /> electron microscopic level help resolve issue.         AP180, unc-101 AP1 perturb membrane trafficking <br />    The large number compounds efficiently          neurons tissues C. elegans not <br /> endocytosed raises question coelomo-             cause Cup phenotype. While possible some <br /> cytes material pseudocoelom con-            genes tested expressed coelomo- <br /> stitutive pinocytosis and/or scavenger receptors             cytes, possible expressed coe- <br /> displaying broad spectrum ligand recognition               lomocytes affect pathway used coelomo- <br /> (Pearson 1996). The results genetic analysis C.       cytes internalize fluid. Furthermore, the <br /> elegans far favor possibility receptor-mediated     genes tested relatives C. elegans genome that <br /> mechanism.                                                      afford functional redundancy. <br />    First, reduction elimination dynamin func-            We screened identified mutations involved <br /> tion led endocytosis defect coelomocytes. Dy-          uptake GFP coelomocytes identified <br /> namin known function maturation and/or             alleles 11 new cup genes 3 previously identified <br /> scission clathrin pit (Sever et al. 2000; Hill et al.    rme genes. In initial screen, selected homozy- <br /> 2001) endocytosis caveolae (Henley et            gous viable fertile Cup mutants. It likely, there- <br /> al. 1998). Clathrin itself strongly expressed        fore, identified genes function in <br /> coelomocytes (Y. Zhang D. Hirsh, personal com-              endocytosis regulate essential processes, like <br /> munication) numerous coated pits vesicles           cell-cell signaling embryonic development. In- <br /> found surface cells (White 1988). This          deed, course screen, saw some <br /> suggests clathrin-mediated endocytosis          indication category endocytosis <br /> major pathway coelomocytes. An alternative explana-          mutants associated zygotic maternal- <br /> tion coelomocytes C. elegans, dynamin         effect lethality. For cup genes, identified <br /> might used pathways endocytosis        single alleles, achieved saturation <br /> do involve receptors.                                       screen. Some alleles recovered may <br />    Second, rme-1 identified genetic screen       prove non-null alleles essential genes. <br /> cup mutants. RME-1 shown involved                The cup screen described rme screen <br /> membrane (and membrane protein) recycling                described B. Grant D. Hirsh (1999) both <br /> plasma membrane (Grant et al. 2001; Lin et al. 2001).           powerful screens endocytosis mutants far <br /> In rme-1 <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>null mutants</span>, Cup defect visible        yielded largely complementary sets genes. We <br /> older worms, case scavenger receptors        tested available cup genes yolk accumula- <br /> must depleted plasma membrane              tion, B. Grant D. Hirsh (personal communi- <br /> Cup defect seen. Alternatively, Cup defect rme-1      cation) tested rme genes Cup <br /> mutant worms general depletion             phenotype. Although screen con- <br /> bulk membrane surface func-             ducted saturation, data far suggest roughly <br /> tion RME-1 cells recycling.              one-third one-half genes affect yolk up- <br />    Some genes encoding proteins character-           oocytes pseudocoelomic GFP uptake by <br /> ized roles vesicle trafficking identified       coelomocytes. <br /> playing roles coelomocyte endocytosis virtue            Both Rme Cup screens identified <br /> Cup phenotype RNAi experiments. vps-34 encodes               genes play important roles endocytosis animal <br /> phosphoinositide 3-kinase (PI-3 kinase), protein         cells appear homologs yeast. As <br /> regulates multiple steps endocytosis (Backer 2000).          mentioned above, rme-1, identified both <br /> sec-5 sec-8 encode homologs yeast Sec5p          Rme Cup screens, appears involved in <br /> Sec8p, members exocyst complex. S. cerevisiae            membrane (and membrane protein) recycling the <br /> sec mutants, including sec5 sec8, accumulate          plasma membrane (Grant et al. 2001; Lin et al. 2001). <br /> proteins secretory vesicles, defective endocy-    rme-8, identified Rme Cup screens, <br /> tosis (Riezman 1985). Vps54p S. cerevisaie involved       appears involved endocytic vesicle trafficking <br /> in vacuolar trafficking (Conboy Cyert 2000). clh-           (Zhang et al. 2001). Finally, cup-5, identified <br /> 15 encodes chloride channel human cells            solely Cup screen, encodes C. elegans homolog <br /> involved receptor-mediated endocytosis (Wang et al.          human mucolipin-1 (Fares Greenwald 2001). <br /> 2000); reduction endocytosis clh-15(RNAi)             Loss human mucolipin-1 underlies Mucolipidosis <br /> worms severe genes affect           Type IV, lysosomal storage disease results severe <br /> endocytosis coelomocytes, possibly            developmental neuropathology (Bargal et al. 2000; <br /> existence multiple genes encode chloride chan-          Bassi et al. 2000; Sun et al. 2000). As report elsewhere <br /> nels C. elegans (Nehrke et al. 2000).                        (Fares Greenwald 2001), analysis revealed <br />    Many genes, known function           striking parallels cellular <br />                                                           C. elegans Endocytosis Mutants                                                       143 <br />  <br /> phenotypes endocytosis defects cup-5 mutant C.                      Achstetter, T., A. Franzusoff, C. Field R. Schekman, 1988 <br />                                                                                SEC7 encodes unusual, high molecular weight protein re- <br /> elegans cells patients Mucolipidosis Type                        quired membrane traffic yeast Golgi apparatus. J. <br /> IV. We expect cup genes                          Biol. Chem. 263: 11711&#226;&#8364;&#8220;11717. <br /> also prove components endocytosis machin-                     Backer, J. M., 2000 Phosphoinositide 3-kinases regulation <br />                                                                                vesicular trafficking. Mol. Cell. Biol. Res. Commun. 3: 193&#226;&#8364;&#8220;204. <br /> ery unique animal cells.                                       Bacon, R. A., C. J. Cohen, D. A. Lewin I. Mellman, 1994 Dictyos- <br />    Finally, function coelomocytes                        telium discoideum mutants temperature-sensitive defects in <br /> C. elegans? In invertebrates, coelomocytes/hemo-                          endocytosis. J. Cell Biol. 127: 387&#226;&#8364;&#8220;399. <br />                                                                            Bargal, R., N. Avidan, E. Ben-Asher, Z. Olender, M. Zeigler et <br /> cytes mediate cellular immunity organism                            al., 2000 Identification gene causing Mucolipidosis Type <br /> (Pancer et al. 1999). We evidence                          IV. Nat. Genet. 26: 118&#226;&#8364;&#8220;123. <br /> the coelomocytes C. elegans provide potent defense                    Bassi, M. T., M. Manzoni, E. Monti, M. T. Pizzo, A. Ballabio et al., <br />                                                                                2000 Cloning gene encoding novel integral membrane <br /> against bacterial infection; bacterial cells                     protein, mucolipidin&#226;&#8364;&#8221;and identification major <br /> (OP50) injected pseudocoelom multiply                             founder mutations causing Mucolipidosis Type IV. Am. J. Hum. <br /> and kill worm (H. Fares I. Greenwald, unpub-                           Genet. 67: 1110&#226;&#8364;&#8220;1120. <br />                                                                            Baumert, M., P. R. Maycox, F. Navone, P. De Camilli R. Jahn, <br /> lished observations). This function coelomocytes                           1989 Synaptobrevin: integral membrane protein 18,000 <br /> may better explored comparing susceptibility                         daltons present small synaptic vesicles rat brain. EMBO J. <br /> of wild type vs. worms lacking coelomocytes infection;                      8: 379&#226;&#8364;&#8220;384. <br />                                                                            Becker, M., M. Matzner G. Gerisch, 1999 Drainin required <br /> to date, however, microorganism reported                           membrane fusion contractile vacuole Dictyostelium <br /> invade pseudocoelom worms fungus Drech-                          prototype protein family represented man. <br /> meria coniospora, extends hyphae pseu-                      EMBO J. 18: 3305&#226;&#8364;&#8220;3316. <br />                                                                            Bossinger, O., E. Schierenberg, 1996 The use fluorescent <br /> docoelom (Jansson 1994). However,                            marker dyes studying intercellular communication nema- <br /> C. elegans, coelomocyte functions like liver.                       tode embryos. Int. J. Dev. Biol. 40: 431&#226;&#8364;&#8220;439. <br /> The pseudocoelomic fluid serves circulatory sys-                    Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics <br />                                                                                77: 71&#226;&#8364;&#8220;94. <br /> tem nutrients taken up, ingested, degraded,                   Chalfie, M., Y. Tu, G. Euskirchen, W. W. Ward D. C. Prasher, <br /> and secreted pseudocoelom intestinal                           1994 Green fluorescent protein marker gene expres- <br /> cells. It possible coelomocytes reprocess                     sion. Science 263: 802&#226;&#8364;&#8220;805. <br />                                                                            Chavrier, P., R. G. Parton, H. P. Hauri, K. Simons M. Zerial, <br /> fluid detoxify harmful ingested substances. In                        1990 Localization low molecular weight GTP binding pro- <br /> that event, likely absence coelomocytes                      teins exocytic endocytic compartments. Cell 62: 317&#226;&#8364;&#8220;329. <br /> would harmful worms grown laboratory                       Chen, H., S. Fre, V. I. Slepnev, M. R. Capua, K. Takei et al., 1998 <br />                                                                                Epsin EH-domain-binding protein implicated clathrin- <br /> conditions. It possible test comparing                       mediated endocytosis. Nature 394: 793&#226;&#8364;&#8220;797. <br /> the susceptibility wild type vs. worms lacking coelomo-                 Chitwood, B. G., M. B. Chitwood, 1974 Introduction Nematol- <br /> cytes toxic material introduced growth me-                         ogy. University Park Press, Baltimore. <br />                                                                            Choy, R. K., J. H. Thomas, 1999 Fluoxetine-resistant mutants <br /> dium secreted bacterial pathogens (Tan Ausu-                         C. elegans define novel family transmembrane proteins. <br /> bel 2000). With availability worms lack                            Mol. Cell 4: 143&#226;&#8364;&#8220;152. <br /> coelomocytes worms defective endocyto-                     Chvatchko, Y., I. Howald H. Riezman, 1986 Two yeast mu- <br />                                                                                tants defective endocytosis defective pheromone re- <br /> sis coelomocytes, better position                        sponse. Cell 46: 355&#226;&#8364;&#8220;364. <br /> determine biological function cells C.                     Clark, S. G., D. L. Shurland, E. M. Meyerowitz, C. I. Bargmann <br /> elegans.                                                                       A. M. van der Bliek, 1997 A dynamin GTPase mutation <br />                                                                                causes rapid reversible temperature-inducible locomotion <br />    We grateful David Hirsh inspiring coelomo-       defect C. elegans. Proc. Natl. Acad. Sci. USA 94: 10438&#226;&#8364;&#8220;10443. <br /> cytes, Barth Grant Yinhua Zhang helpful reagents discus-       Colbaugh, P. A., C. Y. Kao, S. P. Shia, M. Stookey R. K. Draper, <br /> sions project, Piali Sengupta coelomocyte-              1988 Three new complementation groups temperature-sensi- <br /> specific promoter Paula Loria Oliver Hobert additional             tive Chinese hamster ovary cell mutants defective endocytic <br /> information it, David Hall communicating unpublished                 pathway. Somat. Cell Mol. Genet. 14: 499&#226;&#8364;&#8220;507. <br />                                                                            Conboy, M. J., M. S. Cyert, 2000 Luv1p/Rki1p/Tcs3p/Vps54p, <br /> observations, John Collier Diphtheria toxin clones. We <br />                                                                                yeast protein localizes late Golgi early endosome, <br /> also thank Alan Coulson Yuji Kohara providing cosmids,                 required normal vacuolar morphology. Mol. Biol. Cell 11: <br /> strains, cDNA clones, Richard Ruiz Ilya Temkin excellent           2429&#226;&#8364;&#8220;2443. <br /> technical assistance, Barth Grant, Oliver Hobert, Sophie Jarriault     Creutz, C. E., J. R. Harrison, 1984 Clathrin light chains and <br /> for critical reading manuscript, past present members           secretory vesicle binding proteins distinct. Nature 308: 208&#226;&#8364;&#8220; <br /> of Greenwald, Hirsh, Hobert laboratories helpful discus-           210. <br /> sions. Some strains provided Caenorhabditis Genetics           Creutz, C. E., J. L. Tomsig, S. L. Snyder, M. C. Gautier, F. Skouri <br /> Center, funded National Institutes Health National          et al., 1998 The copines, novel class C2 domain-containing, <br /> Center Research Resources (NCRR). I.G. Investigator           calcium-dependent, phospholipid-binding proteins conserved <br /> Howard Hughes Medical Institute. H.F. Postdoctoral Associate             Paramecium humans. J. Biol. Chem. 273: 1393&#226;&#8364;&#8220;1402. <br />                                                                            DeLange, R. J., L. C. Williams, R. E. Drazin R. J. Collier, 1979 <br /> of HHMI supported award Metropoli- <br />                                                                                The amino acid sequence fragment A, enzymically active <br /> tan Life Foundation I.G.                                                    fragment diphtheria toxin. III. The chymotryptic peptides, the <br />                                                                                peptides derived cleavage tryptophan residues, the <br />                                                                                complete sequence protein. J. Biol. Chem. 254: 5838&#226;&#8364;&#8220;5842. <br />                       LITERATURE CITED                                     Fares, H., I. Greenwald, 2001 Regulation endocytosis by <br />                                                                                CUP-5, Caenorhabditis elegans mucolipin-1 homologue. Nat. <br /> Aalto, M. K., L. Ruohonen, K. Hosono S. Keranen, 1991 Clon-                Genet. 28: 64&#226;&#8364;&#8220;68. <br />     ing sequencing yeast Saccharomyces cerevisiae SEC1 gene     Fire, A., 1986 Integrative transformation Caenorhabditis elegans. <br />     localized chromosome IV. Yeast 7: 643&#226;&#8364;&#8220;650.                              EMBO J. 5: 2673&#226;&#8364;&#8220;2680. <br />  144                                                           H. Fares I. Greenwald <br /> Fire, A., S. Xu, M. K. Montgomery, S. A. Kostas, S. E. Driver et                  phiphysin, novel protein associated synaptic vesicles. EMBO <br />     al., 1998 Potent specific genetic interference double-                 J. 11: 2521&#226;&#8364;&#8220;2530. <br />     stranded RNA Caenorhabditis elegans. Nature 391: 806&#226;&#8364;&#8220;811.              Lin, S. X., B. Grant, D. Hirsh F. R. Maxfield, 2001 Rme-1 <br /> Fitzgerald, K., I. Greenwald, 1995 Interchangeability                      regulates distribution function endocytic recycling <br />     Caenorhabditis elegans DSL proteins intrinsic signalling activity         compartment mammalian cells. Nat. Cell Biol. 3: 567&#226;&#8364;&#8220;572. <br />     extracellular domains vivo. Development 121: 4275&#226;&#8364;&#8220;            Lundquist, E. A., R. K. Herman, J. E. Shaw C. I. Bargmann, <br />     4282.                                                                         1998 UNC-115, conserved protein predicted LIM and <br /> Grant, B., I. Greenwald, 1997 Structure, function, expres-                actin-binding domains, mediates axon guidance C. elegans. <br />     sion SEL-1, negative regulator LIN-12 GLP-1 C.                 Neuron 21: 385&#226;&#8364;&#8220;392. <br />     elegans. Development 124: 637&#226;&#8364;&#8220;644.                                        Maniak, M., R. Rauchenberger, R. Albrecht, J. Murphy G. <br /> Grant, B., D. Hirsh, 1999 Receptor-mediated endocytosis                    Gerisch, 1995 Coronin involved phagocytosis: dynamics of <br />     Caenorhabditis elegans oocyte. Mol. Biol. Cell 10: 4311&#226;&#8364;&#8220;4326.             particle-induced relocalization visualized green fluorescent <br /> Grant, B., Y. Zhang, M.-C. Paupard, S. X. Lin, D. H. Hall et al.,                 protein tag. Cell 83: 915&#226;&#8364;&#8220;924. <br />     2001 Evidence RME-1, conserved C. elegans EH domain                Matyash, V., C. Geier, A. Henske, B. Gregan, S. Mukherjee et al., <br />     protein, functions endocytic recycling. Nat. Cell Biol. 3: 573&#226;&#8364;&#8220;            2001 Distribution transport cholesterol Caenorhabditis <br />     579.                                                                          elegans. Mol. Biol. Cell 12: 1725&#226;&#8364;&#8220;1736. <br /> Han, M., P. W. Sternberg, 1991 Analysis dominant-negative              McKim, K. S., C. Matheson, M. A. Marra, M. F. Wakarchuk D. L. <br />     mutations Caenorhabditis elegans let-60 ras gene. Genes Dev.           Baillie, 1994 The Caenorhabditis elegans unc-60 gene encodes <br />     5: 2188&#226;&#8364;&#8220;2198.                                                                 proteins homologous family actin-binding proteins. Mol. <br /> Harris, T. W., E. Hartwieg, H. R. Horvitz E. M. Jorgensen,                    Gen. Genet. 242: 346&#226;&#8364;&#8220;357. <br />     2000 Mutations synaptojanin disrupt synaptic vesicle recy-             Mello, C., A. Fire, 1995 DNA transformation. Methods Cell <br />     cling. J. Cell Biol. 150: 589&#226;&#8364;&#8220;600.                                            Biol. 48: 451&#226;&#8364;&#8220;482. <br /> Henley, J. R., E. W. Krueger, B. J. Oswald M. A. McNiven,                 Mello, C. C., J. M. Kramer, D. Stinchcomb V. Ambros, 1991 <br />     1998 Dynamin-mediated internalization caveolae. J. Cell Biol.              Efficient gene transfer C.elegans : extrachromosomal mainte- <br />     141: 85&#226;&#8364;&#8220;99.                                                                   nance integration transforming sequences. EMBO J. 10: <br /> Hill, E., J. vand der Kaay, C. P. Downes E. Smythe, 2001 The                  3959&#226;&#8364;&#8220;3970. <br />     role dynamin binding partners coated pit invagi-            Mukherjee, S., R. N. Ghosh F. R. Maxfield, 1997 Endocytosis. <br />     nation scission. J. Cell Biol. 152: 309&#226;&#8364;&#8220;323.                              Physiol. Rev. 77: 759&#226;&#8364;&#8220;803. <br /> Hobbie, L., A. S. Fisher, S. Lee, A. Flint M. Krieger, 1994               Munn, A. L., H. Riezman, 1994 Endocytosis required the <br />     Isolation classes conditional lethal Chinese hamster              growth vacuolar H(&#226;&#171;&#185;)-ATPase-defective yeast: identification <br />     ovary cell mutants temperature-dependent defects low                  new END genes. J. Cell Biol. 127: 373&#226;&#8364;&#8220;386. <br />     density lipoprotein receptor stability intracellular membrane         Nehrke, K., T. Begenisich, J. Pilato J. E. Melvin, 2000 Into <br />     transport. J. Biol. Chem. 269: 20958&#226;&#8364;&#8220;20970.                                   ion channel transporter function. Caenorhabditis elegans ClC- <br /> Hoekstra, D., S. C. van IJzendoorn, 2000 Lipid trafficking                    type chloride channels: novel variants functional expression. <br />     sorting: cholesterol filling gaps. Curr. Opin. Cell Biol.          Am. J. Physiol. Cell Physiol. 279: C2052&#226;&#8364;&#8220;C2066. <br />     12: 496&#226;&#8364;&#8220;502.                                                              Nonet, M. L., K. Grundahl, B. J. Meyer J. B. Rand, 1993 Synap- <br /> Hosono, R., K. Hirahara, S. Kuno T. Kurihara, 1982 Mutants                    tic function impaired eliminated C. elegans mutants <br />     Caenorhabditis elegans dumpy rounded head pheno-                  lacking synaptotagmin. Cell 73: 1291&#226;&#8364;&#8220;1305. <br />     type. J. Exp. Zool. 235: 409&#226;&#8364;&#8220;421.                                         Nonet, M. L., A. M. Holgado, F. Brewer, C. J. Serpe, B. A. Norbeck <br /> Huang, K. M., K. D&#226;&#8364;&#8482;Hondt, H. Riezman S. K. Lemmon, 1999                       et al., 1999 UNC-11, Caenorhabditis elegans AP180 homologue, <br />     Clathrin functions absence heterotetrameric adaptors                regulates size protein composition synaptic vesicles. <br />     AP180-related proteins yeast. EMBO J. 18: 3897&#226;&#8364;&#8220;3908.                   Mol. Biol. Cell 10: 2343&#226;&#8364;&#8220;2360. <br /> Jansson, H.-B., 1994 Adhesion conidia Drechmeria coniospora             Otsuka, A. J., A. Jeyaprakash, J. Garcia-Anoveros, L. Z. Tang, G. <br />     Caenorhabditis elegans wild type mutants. J. Nematol. 26:              Fisk et al., 1991 The C. elegans unc-104 gene encodes putative <br />     430&#226;&#8364;&#8220;435.                                                                      kinesin heavy chain-like protein. Neuron 6: 113&#226;&#8364;&#8220;122. <br /> Jantsch-Plunger, V., M. Glotzer, 1999 Depletion syntaxins              Pancer, Z., J. P. Rast E. H. Davidson, 1999 Origins immu- <br />     early Caenorhabditis elegans embryo reveals role mem-            nity: transcription factors homologues effector genes the <br />     brane fusion events cytokinesis. Curr. Biol. 9: 738&#226;&#8364;&#8220;745.                   vertebrate immune expressed sea urchin coelomocytes. <br /> Jones, D., R. H. Russnak, R. J. Kay E. P. Candido, 1986 Struc-                Immunogenetics 49: 773&#226;&#8364;&#8220;786. <br />     ture, expression, evolution heat shock gene locus             Pearson, A. M., 1996 Scavenger receptors innate immunity. Curr. <br />     Caenorhabditis elegans flanked repetitive elements. J. Biol.       Opin. Immunol. 8: 20&#226;&#8364;&#8220;28. <br />     Chem. 261: 12006&#226;&#8364;&#8220;12015.                                                   Qualmann, B., J. Roos, P. J. DiGregorio R. B. Kelly, 1999 <br /> Kimble, J., W. J. Sharrock, 1983 Tissue-specific synthesis                 Syndapin I, synaptic dynamin-binding protein associates <br />     yolk proteins Caenorhabditis elegans. Dev. Biol. 96: 189&#226;&#8364;&#8220;196.              neural Wiskott-Aldrich syndrome protein. Mol. Biol. Cell <br /> Konzok, A., I. Weber, E. Simmeth, U. Hacker, M. Maniak et al., 1999               10: 501&#226;&#8364;&#8220;513. <br />     DAip1, Dictyostelium homologue yeast actin-interacting           Rappleye, C. A., A. R. Paredez, C. W. Smith, K. L. McDonald and <br />     protein 1, involved endocytosis, cytokinesis, motility. J.          R. V. Aroian, 1999 The coronin-like protein POD-1 required <br />     Cell Biol. 146: 453&#226;&#8364;&#8220;464.                                                      anterior-posterior axis formation cellular architecture in <br /> Kostich, M., A. Fire D. M. Fambrough, 2000 Identification                 nematode Caenorhabditis elegans. Genes Dev. 13: 2838&#226;&#8364;&#8220;2851. <br />     molecular-genetic characterization LAMP/CD68-like protein            Reddien, P. W., H. R. Horvitz, 2000 CED-2/CrkII CED- <br />     Caenorhabditis elegans. J. Cell Sci. 113: 2595&#226;&#8364;&#8220;2606.                     10/Rac control phagocytosis cell migration Caenorhabditis <br /> Krieger, M., 1986 Isolation somatic cell mutants defects               elegans. Nat. Cell Biol. 2: 131&#226;&#8364;&#8220;136. <br />     endocytosis low-density lipoprotein. Methods Enzymol.              Richmond, J. E., W. S. Davis E. M. Jorgensen, 1999 UNC-13 <br />     129: 227&#226;&#8364;&#8220;237.                                                                 required synaptic vesicle fusion C. elegans. Nat. Neurosci. <br /> Lander, E. S., L. M. Linton, B. Birren, C. Nusbaum, M. C. Zody et                 2: 959&#226;&#8364;&#8220;964. <br />     al., 2001 Initial sequencing analysis human genome.            Riddle, D. L., T. Blumenthal, B. J. Meyer J. R. Priess, 1997 <br />     Nature 409: 860&#226;&#8364;&#8220;921.                                                          C. elegans II. Cold Spring Harbor Laboratory Press, Plainview, NY. <br /> Lee, J., G. D. Jongeward P. W. Sternberg, 1994 unc-101,                 Riezman, H., 1985 Endocytosis yeast: yeast secretory <br />     gene required aspects Caenorhabditis elegans develop-             mutants defective endocytosis. Cell 40: 1001&#226;&#8364;&#8220;1009. <br />     ment behavior, encodes clathrin-associated protein. Genes           Robbins, A. R., S. S. Peng J. L. Marshall, 1983 Mutant Chinese <br />     Dev. 8: 60&#226;&#8364;&#8220;73.                                                                hamster ovary cells pleiotropically defective receptor-mediated <br /> Lemmon, S. K., L. M. Traub, 2000 Sorting endosomal                     endocytosis. J. Cell Biol. 96: 1064&#226;&#8364;&#8220;1071. <br />     yeast animal cells. Curr. Opin. Cell Biol. 12: 457&#226;&#8364;&#8220;         Robinson, M. S., C. Watts M. Zerial, 1996 Membrane dynam- <br />     466.                                                                          ics endocytosis. Cell 84: 13&#226;&#8364;&#8220;21. <br /> Lichte, B., R. W. Veh, H. E. Meyer M. W. Kilimann, 1992 Am-               Saifee, O., L. Wei M. L. Nonet, 1998 The Caenorhabditis elegans <br />                                                            C. elegans Endocytosis Mutants                                                      145 <br />     unc-64 locus encodes syntaxin interacts genetically             rhabditis elegans. Implications molecular evolution mam- <br />     synaptobrevin. Mol. Biol. Cell 9: 1235&#226;&#8364;&#8220;1252.                                malian caveolin genes. J. Biol. Chem. 272: 2437&#226;&#8364;&#8220;2445. <br /> Sambrook, J., E. F. Fritsch T. Maniatis, 1989 Molecular Clon-           Tcherepanova, I., L. Bhattacharyya, C. S. Rubin J. H. Freed- <br />     ing: A Laboratory Manual. Cold Spring Harbor Laboratory Press,              man, 2000 Aspartic proteases nematode Caenorhabditis <br />     Cold Spring Harbor, NY.                                                     elegans. Structural organization developmental cell-spe- <br /> Seki, N., M. Muramatsu, S. Sugano, Y. Suzuki, A. Nakagawara et                  cific expression asp-1. J. Biol. Chem. 275: 26359&#226;&#8364;&#8220;26369. <br />     al., 1998 Cloning, expression analysis, chromosomal local-          TerBush, D. R., T. Maurice, D. Roth P. Novick, 1996 The <br />     ization HIP1R, isolog huntingtin interacting protein               exocyst multiprotein complex required exocytosis Sac- <br />     (HIP1). J. Hum. Genet. 43: 268&#226;&#8364;&#8220;271.                                         charomyces cerevisiae. EMBO J. 15: 6483&#226;&#8364;&#8220;6494. <br /> Sever, S., H. Damke S. L. Schmid, 2000 Dynamin: GTP controls            Timmons, L., A. Fire, 1998 Specific interference ingested <br />     formation constricted coated pits, rate limiting step            dsRNA. Nature 395: 854. <br />     clathrin-mediated endocytosis. J. Cell Biol. 150: 1137&#226;&#8364;&#8220;1148.         Venter, J. C., M. D. Adams, E. W. Myers, P. W. Li, R. J. Mural et <br /> Shibata, Y., T. Fujii, J. A. Dent, H. Fujisawa S. Takagi, 2000              al., 2001 The sequence human genome. Science 291: <br />     EAT-20, novel transmembrane protein EGF motifs, re-               1304&#226;&#8364;&#8220;1351. <br />     quired efficient feeding Caenorhabditis elegans. Genetics 154:   Wang, S. S., O. Devuyst, P. J. Courtoy, X. T. Wang, H. Wang et <br />     635&#226;&#8364;&#8220;646.                                                                    al., 2000 Mice lacking renal chloride channel, CLC-5, a <br /> Simpson, F., A. A. Peden, L. Christopoulou M. S. Robinson,                  model Dent&#226;&#8364;&#8482;s disease, nephrolithiasis disorder associated <br />     1997 Characterization adaptor-related protein complex,               defective receptor-mediated endocytosis. Hum. Mol. Genet. <br />     AP-3. J. Cell Biol. 137: 835&#226;&#8364;&#8220;845.                                           9: 2937&#226;&#8364;&#8220;2945. <br /> Singleton, D. R., T. T. Wu J. D. Castle, 1997 Three mamma-              Watts, C., M. Marsh, 1992 Endocytosis: What goes and <br />     lian SCAMPs (secretory carrier membrane proteins) highly                how? J. Cell Sci. 103: 1&#226;&#8364;&#8220;8. <br />     related products distinct genes having similar subcellular distri-   Wendland, B., J. M. McCaffery, Q. Xiao S. D. Emr, 1996 A <br />     butions. J. Cell Sci. 110: 2099&#226;&#8364;&#8220;2107.                                       novel fluorescence-activated cell sorter-based screen yeast en- <br /> Stack, J. H., S. D. Emr, 1994 Vps34p required yeast vacuolar            docytosis mutants identifies yeast homologue mammalian <br />     protein sorting multiple specificity kinase exhibits         eps15. J. Cell Biol. 135: 1485&#226;&#8364;&#8220;1500. <br />     protein kinase phosphatidylinositol-specific PI 3-kinase activi-    White, J., 1988 The anato <br /> </body></html>