<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, July 2008, p. 4849&#226;&#8364;&#8220;4858                                                                                 Vol. 190, No. 14 <br /> 0021-9193/08/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00185-08 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />             NrdI Essentiality Class Ib Ribonucleotide Reduction in <br />                               Streptococcus pyogenes&#228;&#338;&#164;&#226;&#8364;&#160; <br />      Ignasi Roca,1 Eduard Torrents,1,2,3 Margareta Sahlin,2 Isidre Gibert,1 Britt-Marie Sjo <br />                                                                                               &#194;&#168;berg2* <br />      Grup Gene`tica Molecular Bacteriana, Institut Biotecnologia Biomedicina, Departament Gene`tica Microbiologia, <br />              Universitat Auto`noma Barcelona, Bellaterra, 08913 Barcelona, Spain1; Department Molecular Biology and <br />               Functional Genomics, Arrhenius Laboratories, Stockholm University, SE-10691 Stockholm, Sweden2; and <br />                      Institute Bioengineering Catalonia (IBEC), Scientific Park Barcelona, Edifici He`lix, <br />                                            Baldiri Reixac 15-21, ES-08028 Barcelona, Spain3 <br />                                               Received 6 February 2008/Accepted 14 May 2008 <br />  <br />              The Streptococcus pyogenes genome harbors clusters class Ib ribonucleotide reductase genes, nrdHEF and <br />           nrdF*I*E*, second stand-alone nrdI gene, designated nrdI2. We clusters expressed <br />           simultaneously independent operons. The NrdEF enzyme functionally active vitro, NrdE*F* <br />           enzyme not. The NrdF* protein lacks highly conserved iron-liganding chains form <br />           dinuclear iron site tyrosyl radical. In vivo, hand, operons functional heterologous <br />           complementation Escherichia coli. The nrdF*I*E* operon requires presence nrdI* gene, nrdHEF <br />           operon gained activity cotranscription heterologous nrdI gene Streptococcus pneumoniae, while <br />           nrdI* nrdI2 S. pyogenes rendered active. Our results highlight essential role flavodoxin <br />           NrdI protein vivo, suggest needed reduce met-NrdF, enabling spontaneous <br />           reformation tyrosyl radical. The NrdI* flavodoxin play direct role ribonucleotide reduction by <br />           NrdF*I*E* system. We discuss possibility nrdF*I*E* operon horizontally transferred <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />           S. pyogenes Mycoplasma spp. <br />  <br />  <br />    Ribonucleotide reductases (RNRs) group enzymes                     function remains unknown (18). Typically, class Ib nrd genes <br /> that carry metabolic pathway providing living                 arranged nrdHIEF operon, examples of <br /> cells optimal amounts necessary buildings blocks                   bacteria bearing class Ib genes slightly different patterns <br /> for DNA replication DNA repair (29). Three classes                      described (43). <br /> RNRs (I, II, III) characterized date accord-                     The genome gram-positive bacterium Streptococcus <br /> ing different mechanisms radical generation, allo-                pyogenes (group A Streptococcus) harbors different clus- <br /> steric regulation, oxygen dependence quater-                 ters genes involved ribonucleotide reduction (10). One <br /> nary structural differences, common                  cluster contains genes anaerobic class III RNR and <br /> reaction mechanism based radical chemistry cata-                    cognate activase (nrdDG), surprisingly, two <br /> lytic core structure (29).                                                     clusters contains genes form unique class Ib operons <br />    Class I RNRs (which subdivided class Ia                (nrdHEF nrdF*I*E*). The S. pyogenes genome con- <br /> class Ib) composed homodimeric proteins, R1 (&#226;?&#163;2)                    tains second stand-alone copy nrdI gene (designated <br /> and R2 (&#226;?&#164;2). The larger &#226;?&#163; subunit contains catalytic                   nrdI2 work) chromosome. Two features <br /> allosteric sites, smaller &#226;?&#164; subunit harbors stable free            make class Ib operons S. pyogenes unique. First, the <br /> radical tyrosine residue diiron center bridged an <br />                                                                                operons arranged way deviates common <br /> oxo (O2&#226;&#171;&#186;) ion. Class Ib shares class Ia subunit com- <br />                                                                                nrdHIEF operon structure, second, residues involved <br /> position conserved residues, including tyrosyl <br />                                                                                iron binding conserved &#226;?&#164; subunit the <br /> radical iron ligands &#226;?&#164; subunit active-site <br />                                                                                nrdF*I*E* operon. <br /> cysteines &#226;?&#163; subunit, class Ib &#226;?&#163; subunits lack <br />                                                                                   Although common multiple RNR classes within <br /> approximately 50 N-terminal amino acids constitute the <br />                                                                                single microorganism (41), just cases potential re- <br /> allosteric-activity site. The &#226;?&#163; &#226;?&#164; subunits encoded the <br />                                                                                dundancy, i.e., duplication nrd genes belonging the <br /> nrdA nrdB genes class Ia nrdE nrdF genes <br /> for class Ib. Class Ib operons contain additional nrd genes:               class, characterized. Saccharomyces cerevisiae <br /> nrdH, encodes electron donor glutaredoxin-like pro-                   class Ia redundancy (Rnrp1 Rnrp4), Mycobacte- <br /> tein (19), nrdI, encodes small flavoprotein                  rium tuberculosis second class Ib-like &#226;?&#164; subunit gene (46, <br />                                                                                48; http://rnrdb.molbio.su.se additional candi- <br />                                                                                dates). S. pyogenes bacteria (all belonging the <br />    * Corresponding author. Mailing address: Department Molecular            Firmicutes) distinct class Ib RNR gene clusters. <br /> Biology &amp; Functional Genomics, Stockholm University, SE-10691                  A redundancy RNR genes great importance <br /> Stockholm, Sweden. Phone: 46-8-164150. Fax: 46-8-166488. E-mail:               different environmental conditions. In study, we <br /> britt-marie.sjoberg@molbio.su.se. <br />                                                                                analyzed vivo vitro functionality of <br />    &#226;&#8364;&#160; Supplemental material article http://jb <br /> .asm.org/.                                                                     S. pyogenes class Ib nrd genes attempt provide an <br />    &#228;&#338;&#164; <br />      Published ahead print 23 May 2008.                                  explanation atypical redundancy. <br />  <br />                                                                         4849 <br />  4850       ROCA ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />                         MATERIALS AND METHODS                                          Protein purifications. Frozen pellets disintegrated X-press (Biox AB) <br />                                                                                     extracted 50 mM Tris-HCl (pH 8), 50 mM KCl, 5 mM dithiothreitol <br />    Bacterial strains, plasmids, growth conditions. Escherichia coli strains <br />                                                                                     (DTT), 1 mM phenylmethylsulfonyl fluoride (PMSF) (buffer L), which <br /> were routinely grown Luria-Bertani (LB) broth 15, 30, 37&#194;&#176;C. Streptococ- <br />                                                                                     nucleic acids precipitated dropwise addition streptomycin sulfate a <br /> cus pyogenes Rosenbach M1 (ATCC 700294, SF370) grown Todd-Hewitt <br />                                                                                     final concentration 1%. The precipitate removed centrifugation, and <br /> broth (Oxoid) supplemented 0.2% yeast extract (THY) 37&#194;&#176;C in <br />                                                                                     solid ammonium sulfate added supernatant 45% saturation for <br /> Columbia sheep blood agar plates (BioMe&#194;&#180;rieux) needed. Streptococcus <br />                                                                                     NrdE NrdE* 60% saturation NrdF NrdF*. The precipitate <br /> pneumoniae (Klein) Chester R6 (ATCC BAA-255) grown Columbia <br />                                                                                     recovered centrifugation. <br /> sheep blood agar plates (BioMe&#194;&#180;rieux) 37&#194;&#176;C. Antibiotics chromogenic <br />                                                                                        The NrdF NrdF* pellets dissolved buffer L desalted dialysis <br /> substrates, required, included culture media agar plates     2 liters buffer L 4 5 h. The dialyzed solution diluted with <br /> at following concentrations E. coli: 100 &#226;?&#174;g/ml carbenicillin, 100 &#226;?&#174;g/ml     buffer L 5-mg/ml protein concentration loaded DEAE-Sepharose <br /> kanamycin, 34 &#226;?&#174;g/ml chloramphenicol, 30 &#226;?&#174;g/ml 5-bromo-4-chloro-3-indo-          column (GE Healthcare) coupled fast-performance liquid chromatography <br /> lyl-&#226;?&#164;-D-galactopyranoside (X-Gal) (Sigma).                                          instrument (Biologic DuoFlow Systems, Bio-Rad). After washing step 140 <br />    The E. coli strain IG101 [nrdA(Ts) nrdB1 thyA thr leu thi deoC and/or deoD       ml 0.15 M KCl buffer L, elution continued 0.15 0.6 M KCl <br /> tonA lacY supE44 gyrA nrdH::Spc] constructed introducing nrdH::Spc       linear gradient buffer L total volume 300 ml. Fractions checked by <br /> allele (IG100 [this laboratory]) KK450 (31) P1 transduction.                sodium dodecyl sulfate-polyacrylamide gel electrophoresis containing NrdF <br />    Sequence analysis DNA manipulations. Genomic DNA S. pyogenes            NrdF* protein pooled concentrated ultrafiltration Centriplus <br /> was extracted described previously (11). Electroporation S. pyogenes      YM-30 instrument (Millipore) buffer L stored &#226;&#171;&#186;80&#194;&#176;C further <br /> performed described previously (25), general DNA manipulations         purification. The NrdF proteins eluted 0.3 0.36 M KCl. The concen- <br /> were standard procedures (34). DNA sequence determination               trated enzyme solution loaded Superdex 75 10/300 GL column (GE <br /> carried using BigDye Terminator kit ABI Prism 310 sequencer          Healthcare) previously equilibrated buffer L. Proteins eluted with <br /> (Applied Biosystems) according manufacturer&#226;&#8364;&#8482;s specifications. Images      buffer constant flow rate 0.4 ml/min. Fractions containing NrdF <br /> gels captured Gel Doc 2000 station Quantity One software        NrdF* pooled, concentrated, stored described above. <br /> package (Bio-Rad Laboratories).                                                        For purification NrdE NrdE* proteins, ammonium sulfate- <br />    All nrd sequences RNRdb, RNR database (http://rnrdb.molbio         precipitated pellets dissolved 50 mM Tris-HCl (pH 8), 0.75 M ammo- <br /> .su.se). The Clustal W 1.8 (40) GeneDoc 2.6.001 (28) programs used      nium sulfate, 2.5 mM DTT, 1 mM PMSF loaded HiLoad 16/10 <br /> perform global alignments.                                                          phenyl-Sepharose high-performance column (GE Healthcare) equilibrated with <br />    RNA isolation. S. pyogenes grown 10 ml THY A600          buffer. Elution performed 0.75 0 M ammonium sulfate <br /> approximately 0.4. Ten-milliliter cultures centrifuged resuspended      linear gradient 50 mM Tris-HCl (pH 8), 2.5 mM DTT, 1 mM PMSF. NrdE <br /> 200 &#226;?&#174;l diethyl pyrocarbonate (Sigma)-treated distilled water quick-frozen    NrdE* eluted 0.44 0.35 M 0.03 0 M ammonium sulfate, <br /> in dry ice bath. Pellets thawed ice added 2 ml FastPrep Blue       respectively. The enzyme-containing fractions pooled concentrated in <br /> tubes containing zirconite/silica beads, 500 &#226;?&#174;l CPRS-Blue, 500 &#226;?&#174;l acid    Centriplus YM-50 (Millipore) adsorbed 2 ml MonoQ anion- <br /> phenol, described manufacturer (Bio101, La Jolla, CA). Bacteria      exchange column (GE Healthcare) equilibrated buffer L. Proteins were <br /> lysed FastPrep FP120 instrument (Bio101, La Jolla, CA) setting 6      eluted 20 ml 0.05 0.5 M KCl linear gradient buffer L flow rate <br /> 11 s immediately placed ice disruption. Samples incubated      0.1 ml/min. NrdE eluted 0.21 0.28 M KCl NrdE* between <br /> 65&#194;&#176;C 10 min centrifuged 10,000 &#226;&#171;&#187; g 5 min 4&#194;&#176;C. The upper           0.19 0.27 M KCl. Fractions 0.4 ml pooled concentrated as <br /> aqueous phase recovered extracted equal volume acid phenol       described stored &#226;&#171;&#186;80&#194;&#176;C. <br /> heated 65&#194;&#176;C. The phases separated centrifugation described above,        Analytical methods enzyme activity assays. Iron reconstitution assays <br /> and extraction repeated acid phenol-chloroform (1:1, vol/vol)      performed anaerobically described reference 17, iron content was <br /> chloroform-isoamyl alcohol (24:1, vol/vol). The recovered aqueous phase         determined described reference 33. Protein concentration determined <br /> treated 50 U DNase I (Ambion, Inc.) 30 min 37&#194;&#176;C, 0.2        Bradford method (6). <br /> volume inactivating reagent (Ambion, Inc.) added eliminate en-           RNR activity assayed 50-&#226;?&#174;l mixtures containing 50 mM Tris-HCl (pH <br /> zyme. RNA concentrations determined measuring A260. Samples             8), 0.2 mM dATP effector, 20 mM magnesium acetate, 30 mM DTT. <br /> were immediately stored &#226;&#171;&#186;80&#194;&#176;C. Reverse transcription-PCR (RT-PCR)            The reaction started addition [3H]CDP (specific activity, 10,000 to <br />                                                                                     20,000 cpm/nmol) final concentration 1 mM. Assay mixtures incu- <br /> performed described reference 43, identical aliquots processed in <br />                                                                                     bated 10 min 29&#194;&#176;C, reaction stopped addition 0.5 ml <br /> parallel addition RT order ensure residual chromosomal <br />                                                                                     1 M ice-cold perchloric acid. The dCDP formed determined by <br /> DNA used template PCR amplification. <br />                                                                                     standard method described reference 39. One <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> enzyme activity <br />    Cloning expression. The nrdF, nrdE, nrdF*, nrdE* genes am- <br />                                                                                     corresponds 1 nmol dCDP formed minute incubation. Km, Vmax, and <br /> plified PCR using primer pairs containing NdeI (upper primers) BamHI <br />                                                                                     KL (ligand binding constant) values obtained direct curve fitting as <br /> (lower primers) restriction sites S. pyogenes genomic DNA template (see <br />                                                                                     described previously (3, 7) Kaleida-Graph software (Synergy Soft- <br /> Table S1 supplemental material). The resulting fragments purified <br />                                                                                     ware). <br /> and digested NdeI BamHI. The digested nrdF, nrdE, nrdE* frag- <br />                                                                                        UV&#226;&#8364;&#8220;visual-light (UV-vis) absorption spectroscopy decay tyrosyl <br /> ments inserted pET22b(&#226;&#171;&#185;) (Novagen) cut enzymes to <br />                                                                                     radical site. Scanning spectra registered Perkin Elmer Lambda2 <br /> give p22-FSpyo, p22-ESpyo, p22-E*Spyo, respectively. The digested nrdF* <br />                                                                                     spectrophotometer 25&#194;&#176;C, 20 &#226;?&#174;M concentration monomer 50 <br /> fragment inserted pET24a(&#226;&#171;&#185;) (Novagen) cut enzymes         mM Tris-HCl, pH 7.6, used. The radical decay nrdF protein was <br /> give p24-F*Spyo. The sequences of above-named constructs            studied addition hydroxyurea (HU) final concentrations ranging <br /> verified sequencing.                                                             3.5 60 mM. Spectra typically recorded 300 750 nm in <br />    E. coli BL21(DE3) (Novagen) cells transformed expression         cycles 150 s, scanning rate 240 nm/min. The endpoint spectrum, when <br /> plasmids (p22-FSpyo, p22-ESpyo, p22-E*Spyo) grown overnight LB           radical absorption 407 nm decayed, subtracted start spec- <br /> medium containing carbenicillin 30 37&#194;&#176;C. This culture used        trum radical concentration using extinction coefficient 3.25 mM&#226;&#171;&#186;1 <br /> inoculate 5-liter flasks containing 2 liters LB medium concen-     cm&#226;&#171;&#186;1 (30). Alternatively, radical concentration determined calculat- <br /> tration antibiotic noted (1:100). Cells grown 30&#194;&#176;C   ing dropline 402 417 nm using &#206;&#181;407 2.11 mM&#226;&#171;&#186;1 cm&#226;&#171;&#186;1 <br /> vigorous shaking A600 &#226;&#172;&#402;0.5 0.6 subsequently induced 0.5 mM     equation {A407 &#226;&#171;&#186; [A402 &#226;&#171;&#186; (5/15) &#228;&#161;&#160; (A402 &#226;&#171;&#186; A417)]}/&#206;&#181;. Kinetic traces 407 <br /> IPTG (isopropyl-&#226;?&#164;-D-thiogalactopyranoside; final concentration) 4 hours.        nm run determine decay rate; incubations E. coli HU <br />    BL21(DE3) transformed plasmid p24-F*Spyo grown overnight LB          run 410 nm controls. Spectra run kinetic <br /> medium containing kanamycin 30&#194;&#176;C. This culture used inoculate        trace measure degree completeness reaction subtractions as <br /> 5-liter flasks containing 2 liters LB medium concentration      described above. <br /> antibiotic noted (1:100). Cells grown 30&#194;&#176;C vigorous         EPR spectroscopy. Electron paramagnetic resonance (EPR) measurements at <br /> shaking. When reached A600 &#226;&#172;&#402;0.5 0.6, cultures chilled     X-band performed &#226;&#177;&#8211;90 K Bruker ESP300 spectrometer using <br /> 15&#194;&#176;C induced 0.3 mM IPTG (final concentration)           nitrogen flow &#226;&#172;?60 K Bruker ESP500 using Oxford helium <br /> grown 15 17 h 15&#194;&#176;C. After induction, cells harvested centrifu-    flow cryostat. Radical concentrations determined, unsaturated con- <br /> gation pellets quickly frozen stored &#226;&#171;&#186;80&#194;&#176;C.                          ditions, using E. coli NrdB sample known tyrosyl radical concentrations <br />  VOL. 190, 2008                                                                             CLASS Ib RNR REDUNDANCY IN S. PYOGENES                     4851 <br />  <br />  <br />  <br />  <br />   FIG. 1. (A) Schematic organization S. pyogenes class Ib nrd genes. Arrows indicate positions numbers primers used for <br /> RT-PCR amplification, black lines labeled A, B, C, D positions lengths expected transcripts. (B) Linkage RT-PCR <br /> of nrdHE (lane A), nrdEF (lane B), nrdFIE (lane C) junctions nrdI2 gene (lane D). The linkage strategy outlined panel A. <br /> MWM, molecular weight markers. <br />  <br />  <br />  <br /> as standard. The error limits radical quantitations &#226;&#171;&#190;10%.           spans 4 kbp. The nrdI* gene encodes unknown protein <br /> Reconstituted samples EPR measurements frozen liquid               18 kDa 162 amino acids 75% similar E. coli <br /> nitrogen 6 min addition 2 Fe ions/polypeptide frozen the <br /> addition 4 Fe ions/monomer followed desalting NAP-5 column to <br />                                                                                         NrdI (19). S. pyogenes nrdE* encodes RNR large subunit <br /> remove unbound iron.                                                                    essential cysteines active site, C-terminal <br />    Heterologous complementation assay. Class Ib nrd genes amplified             cysteine pair, conserved C-terminal tyrosines (re- <br /> PCR using primers pairs containing ribosomal binding site sequence E.            quired transfer radical small subunit) as <br /> coli SacI (upper primers) PstI (lower primers) restriction sites (see <br /> Table S1 supplemental material). The resulting fragments purified <br />                                                                                         residues form allosteric-specificity site (see <br /> and digested SacI PstI inserted pBAD18Km pBAD33Cm                  Table S2 supplemental material). Finally, nrdF* gene <br /> (14) generate final constructs. p18-FI*Spyo, p18-IE*Spyo, p18-               encodes RNR small subunit lacks important <br /> FE*Spyo derived p18-FIE*Spyo enzymatic digestion.                          residues proper function enzyme. Compared to <br />    In heterologous complementation assay, constructs transformed into <br />                                                                                         E. coli NrdF protein, S. pyogenes NrdF* turned have <br /> the E. coli strain IG101 grown overnight 30&#194;&#176;C LB medium containing <br /> 50 &#226;?&#174;g/ml thymine plus corresponding antibiotics. The following day, serial          amino acid substitutions important residues <br /> dilutions liquid cultures plated LB agar plates containing         involved iron binding residues <br /> corresponding antibiotics plus 50 &#226;?&#174;g/ml thymine 0.3% L-arabinose          involved radical stability. Instead Glu115 E. coli NrdB, <br /> 0.3% D-glucose incubated 30 42&#194;&#176;C 24 48 h. Attempts <br />                                                                                         S. pyogenes NrdF* Val116, instead Glu204 Pro176, <br /> to obtain S. pyogenes chromosomal mutants insertional recombinant <br /> inactivation class Ib nrd genes unsuccessful, initially attrib-   instead Glu238 Lys210, instead Phe208 has <br /> uted essentiality genes. However, used muta-           Leu180. All residues important radical stability and <br /> genic strategies obtain chromosomal mutant previously described             transfer conserved (see Table S3 supplemental ma- <br /> nonessential covR gene (9), mutant colonies recovered extremely low <br />                                                                                         terial). <br /> frequencies (1 &#226;&#171;&#187; 10&#226;&#171;&#186;8); therefore, essentiality S. pyogenes class Ib nrd <br /> genes assessed direct complementation.                                     A second stand-alone nrdI gene (nrdI2) annotated else- <br />                                                                                         genome (10). S. pyogenes nrdI2 encodes protein <br />                                                                                         160 amino acids 17.7 kDa 36% similar nrdI*. <br />                                    RESULTS                                                 The redundant class Ib operons expressed simulta- <br />    S. pyogenes encodes redundant class Ib gene clusters.                            neously. To investigate class Ib RNR regions were <br /> The complete S. pyogenes Rosenbach M1 genome (10) con-                                  simultaneously transcribed assess operonic organi- <br /> tains seven genes putatively encoding class Ib RNR-related                              zation cluster, total RNA strain SF370 ex- <br /> proteins different regions genome (Fig. 1A). All                        tracted early exponential phase used perform non- <br /> 10 hitherto-completed S. pyogenes genomes encode                               quantitative RT-PCR experiments. Primer pairs selected <br /> seven class Ib genes (see RNRdb http://rnrdb.molbio                              bracket intergenic junctions nrdHE, nrdEF, and <br /> .su.se).                                                                                nrdF*I*E* (Fig. 1A). Any PCR fragment generated these <br />    The class Ib region spans 4.5 kbp comprises                           oligonucleotides derived polycistronic tran- <br /> the nrdHEF cluster, codes putative glutaredoxin-                            script. <br /> like protein (NrdH) large (NrdE) small                                  An amplicon spanning nrdHE junction detected by <br /> (NrdF) subunits holoenzyme. Both NrdE NrdF                               using primer 2 generate cDNA primers 1 2 (see <br /> proteins contained conserved residues necessary                             Table S1 supplemental material) PCR (Fig. 1B, <br /> ribonucleotide reduction (see Table S2 S3 supple-                            lane A). Similarly, PCR product bracketing junction <br /> mental material).                                                                       nrdE nrdF detected primers 3 <br />    The second Ib region comprises nrdF*I*E* cluster                             4 (Fig. 1B, lane B). Using primer 6 generate reversed <br />  4852     ROCA ET AL.                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. (A) Line a, UV-vis spectra S. pyogenes NrdF protein. The arrow indicates tyrosyl radical. Dotted line b, HU-treated sample. <br /> Line a-b, subtraction HU-treated spectrum purified NrdF spectrum. (B) UV-vis spectra S. pyogenes NrdF* protein upon <br /> purification (dashed line) addition 2 Fe ions/monomer. <br />  <br />  <br />  <br /> transcribed cDNA template primer pair 5-6 amplify              registered EPR spectrum NrdF protein (Fig. 3), and <br /> this template, able detect amplicon span-              tyrosyl radical proved typical class Ib radical similar <br /> ning nrdF*I*E* intergenic junction (Fig. 1B, lane C). An              NrdF proteins S. enterica serovar Ty- <br /> RT-PCR second nrdI2 gene carried                phimurium, Corynebacterium ammoniagenes, M. tubercu- <br /> primers 7 8, corresponding transcript readily             losis (17, 21, 24). At 95 K, radical saturated 1 order of <br /> detected (Fig. 1B, lane D). The nrdI2 product sequenced               magnitude power does E. coli NrdB protein (4.7 <br /> to ensure correct amplification.                                      mW compared 54.0 mW [data shown]) similarly to <br />    Control experiments conducted concurrently set               S. enterica serovar Typhimurium, C. ammoniagenes, M. <br /> of RT-PCR assays confirmed quality samples                 tuberculosis NrdF proteins (3.7 mW, 1.3 mW, 1.3 mW, <br /> guaranteed specificity component RT-PCR.               respectively) (17, 44). The S. pyogenes NrdF radical signal had <br /> (i) Our inability amplify desired product RT              hyperfine structure 10 20 K 95 K that <br /> omitted validated compulsory requirement enzyme               previously observed class Ib radicals mentioned above, <br /> for initiation cDNA synthesis ensured re-              contrast E. coli Ia signal, considerably broad- <br /> sidual DNA contaminated starting RNR preparations; (ii)               ened 95 K compared structure 10 20 K (32). The <br /> the expected amplicons detected genomic DNA                     radical saturation behavior reflects environment, case <br /> acted template, demonstrating fidelity PCR              plausibly coupling iron, geometry. The difference <br /> primers (data shown). These results indicated presence            E. coli Ia radical indicate S. pyogenes <br /> of polycistronic transcript cluster genes        NrdF radical weaker coupling iron site, perhaps <br /> their simultaneous transcription standard laboratory <br /> growth conditions. <br />    Spectroscopic characterization NrdF NrdF*. Once it <br /> was demonstrated class Ib nrd genes S. pyogenes were <br /> actively transcribed, decided check were <br /> capable forming active holoenzyme. The nrdE nrdF <br /> genes nrdHEF nrdF*I*E* operons were <br /> cloned, overproduced, purified described Materials <br /> and Methods. After different purification steps, purified <br /> proteins showed expected theoretical molecular mass. The <br /> purity samples cases 90%. Attempts to <br /> overproduce purify NrdI* NrdI2 proteins were <br /> unsuccessful overproduced proteins continu- <br /> ously recovered insoluble fraction bacterial ex- <br /> tract, regardless incubation temperature IPTG <br /> concentration used (not shown). <br />    Purified NrdF protein showed light absorption spectrum <br /> typical diiron-tyrosyl radical RNR. Charge transfer bands <br /> of &#226;?&#174;-oxobridged diferric center observed 325 and <br /> 370 nm, weaker bands 500 600 nm (Fig. 2A).                   FIG. 3. First-derivative EPR spectra NrdF NrdF*, indi- <br /> The tyrosyl radical showed sharp peak 407 nm. We                cated figure. <br />  VOL. 190, 2008                                                            CLASS Ib RNR REDUNDANCY IN S. PYOGENES                     4853 <br />  <br />      TABLE 1. Iron radical contents S. pyogenes NrdF        spectra (Fig. 2B, dashed line) EPR signal (Fig. 3); i.e., <br />            NrdF* iron reconstitutiona                 diiron-tyrosyl radical site. As expected, analysis the <br />                            No. Fe ions/            No. tyrosyl   purified NrdF* iron content showed 0.05 Fe ion/monomer <br /> Protein <br />                              monomer                radicals/monomer   (Table 1). The addition 2 Fe ions/monomer gave spectrum <br /> NrdF                            1.2                       0.5          radical absorption characteristics of <br /> NrdFR                           2.1                       0.5          diiron site observed NrdF protein absorp- <br /> NrdF*                           0.05                      ND           tion 325- 450-nm region, similar been <br /> NrdFR*                          0.47                      ND           observed iron ligand mutants E. coli NrdB (Fig. 2B, <br />  a <br />      R, iron reconstituted; ND, detected.                      dotted line) (&#206;&#181;350 &#226;&#172;&#8225; 1,800 M&#226;&#171;&#186;1 cm&#226;&#171;&#186;1) (1, 45). Iron analysis of <br />                                                                        reconstituted desalted samples gave values &#226;&#172;&#402;0.47 Fe <br />                                                                        ion/monomer. Presumably, altered residues iron <br /> with water molecule radical closest iron,        binding center protein impair formation a <br /> as class Ib tyrosyl radicals. Addition radical        dinuclear iron center tyrosyl radical. <br /> scavenger HU reduce tyrosyl radical gave spectrum              The NrdEF enzyme enzymatically active, the <br /> shown Fig. 2A (dotted line), shown diiron      NrdE*F* enzyme not. When assayed enzymatic activity, <br /> site remained essentially unchanged. Subtraction end-           proteins activity own. When <br /> point spectrum gives typical tyrosyl radical spectrum (Fig.          mixed assayed together, NrdE NrdF proteins <br /> 2A, lowest spectrum) sharp peak 407 nm. Using             purified nrdHEF active showed pro- <br /> extinction coefficient (&#206;&#181;) 3.25 mM&#226;&#171;&#186;1 cm&#226;&#171;&#186;1 (30) sub-          portional dCDP formation, protein excess and <br /> tracted spectra using dropline specified Materials        increasing amounts partner. Specific activities, the <br /> and Methods, calculated, best preparation, radical        presence dATP positive allosteric effector, 169 <br /> content 0.51 monomer, good agreement 0.57           nmol/min/mg NrdF protein 45 nmol/min/mg the <br /> radical monomer determined EPR. Analysis                 NrdE protein. Activity dependent addition mag- <br /> NrdF iron content showed approximately 1.2 Fe ions/monomer             nesium DTT optimal pH 8.0 required CDP as <br /> (Table 1). It uncommon RNRs purify subop-             substrate, Km 0.34 mM (Fig. 4A). CDP reduction <br /> timal iron content and, consequently, small radical        strongly stimulated dATP effector lesser <br /> when overexpressed E. coli. Hence, added fer-           extent ATP. Optimal activity dATP obtained at <br /> rous ions protein investigate increase       nucleotide concentrations low 0.01 mM, signifi- <br /> yield diiron-radical site. Addition ferrous ions, how-       inhibition seen 2 mM dATP. When ATP <br /> ever, did increase radical yield NrdF radical-       used, concentrations 2 mM to <br /> containing samples. To assess stability tyrosyl radi-       achieve optimal activity. <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> Fig. 4B, apparent KL values of <br /> cal, monitored reactivity radical scavenger          0.0011 mM dATP approximately 0.3 mM ATP can <br /> HU. S. pyogenes NrdF reacted considerably lower rate (k1          calculated. No significant gain specific activity de- <br /> [first order rate constant] 0.12 min&#226;&#171;&#186;1 60 mM HU)          tected iron-reconstituted NrdF protein, agreement <br /> E. coli NrdB (k1 0.50 min&#226;&#171;&#186;1 15 mM HU) 25&#194;&#176;C; i.e.,           data provided EPR UV-vis spectroscopy. <br /> it order magnitude sensitive E. coli      The NrdE* NrdF* proteins activity as- <br /> enzyme reduction HU.                                             sayed together, NrdF* combined cata- <br />     On hand, purified NrdF*, obtained           lytically competent NrdE protein, agrees lack of <br /> the apoform, showed prominent features light absorption          tyrosyl radical NrdF*. One might, hand, <br />  <br />  <br />  <br />  <br />   FIG. 4. (A) Comparison CDP CTP substrates NrdF/NrdE reductase. Assay mixtures standard conditions <br /> with 1.2 &#226;?&#174;M NrdE 5 &#226;?&#174;M NrdF (E, CDP; f, CTP). (B) Effect ATP dATP CDP reduction. Incubation 1.2 &#226;?&#174;M NrdE and <br /> 5 &#226;?&#174;M NrdF, replacing standard concentration 0.2 mM dATP indicated concentrations ATP (F) dATP (f). <br />  4854     ROCA ET AL.                                                                                                                     J. BACTERIOL. <br />  <br />  <br />                                                                         formed p18-FIE*Spyo gave 1.56 &#226;&#171;&#187; 108 CFU/ml 24 to <br />                                                                         48 h 42&#194;&#176;C plates supplemented arabinose, not <br />                                                                         glucose, indicating nrdF*I*E* cluster capable <br />                                                                         complementing temperature-sensitive <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> (Table <br />                                                                         2). The results obtained constructs the <br />                                                                         low-copy-number plasmid pBAD33 instead pBAD18 (data <br />                                                                         shown). Neither pBAD18 pBAD33 able to <br />                                                                         support growth IG101 strain restrictive tem- <br />                                                                         perature. <br />                                                                            Our step determine genes the <br />                                                                         nrdF*I*E* operon required observed vivo <br />                                                                         complementation. A set pBAD18/pBAD33 vectors were <br />                                                                         constructed gene nrdF*I*E* operon <br />                                                                         evaluated pairwise combinations other <br />                                                                         genes. No complementation detected the <br />                                                                         nrdF*I*E* genes missing (Table 2), suggesting that <br />   FIG. 5. Assay competitive inhibition NrdE         entire operon needed complementation. On the <br /> NrdF. Incubations carried standard conditions (i) with <br /> increasing concentrations NrdF presence 0.6 &#226;?&#174;M NrdE (F)    hand, complementation restored missing <br /> and 0.6 &#226;?&#174;M NrdE 2 &#226;?&#174;M NrdF* (f) (ii) increasing    gene provided trans. <br /> concentrations NrdE presence 0.6 &#226;?&#174;M NrdF (v) 0.6          These results stress essential role nrdI* gene in <br /> &#226;?&#174;M NrdF 2 &#226;?&#174;M NrdE* (&#197;&#8217;).                                vivo activity. In view results, S. pyogenes nrdI* <br />                                                                         nrdI2 genes cloned pBAD33 cotransformed with <br />                                                                         nrdHEF operon IG101 strain. However, as <br /> expect activity combining NrdE* NrdF                  shown Table 2, S. pyogenes nrdI* nrdI2 could <br /> proteins, NrdE* catalytically im-           complement S. pyogenes nrdHEF overcome tempera- <br /> portant residues NrdF harbors tyrosyl radical              ture-sensitive phenotype. We checked nrdI <br /> catalytically competent. However, mixing NrdE*               gene closely related bacterium Streptococcus pneu- <br /> NrdF did result enzyme activity. One explanation         moniae restore complementation combined S. <br /> the lack activity case NrdE* NrdF         pyogenes nrdHEF. S. pneumoniae harbors nrdHEF operon <br /> form holoenzyme complex. To test this, performed com-              isolated nrdI2 gene lacks nrdFIE operon <br /> petition assays detect protein interactions subunits         (16). Plasmids p18-HEFSpn p33-I2Spn generated by <br /> from different systems. Fixed concentrations NrdF            cloning nrdHEF nrdI2 genes S. pneumoniae into <br /> NrdE assayed increasing amounts                  pBAD vectors. When assayed alone, p18-HEFSpn did not <br /> NrdE* NrdF*. As shown Fig. 5, concentrations             complement IG101 42&#194;&#176;C. Transformants p18- <br /> 2 &#226;?&#174;M NrdF* NrdE* inhibitory effect NrdF/NrdE               HEFSpn p33-I2Spn, however, restored growth the <br /> holoenzyme activity, suggesting subunits              IG101 strain 42&#194;&#176;C arabinose-LB plates (but glu- <br /> systems cross-react.                                             cose), showing S. pneumoniae nrdHEF operon can <br />    Complementation assays nrdF*I*E* functional             complement supplemented S. pneumoniae nrdI2. <br /> in vivo nrdI essential complementation. We         When combined S. pneumoniae nrdHEF operon with <br /> decided compare vitro results nrdHEF and <br /> nrdF*I*E* systems results complementation assays <br /> of vivo functionality operon. To so, used <br /> the E. coli IG101 strain, nrdA(Ts)-nrdB1/nrdH::Spc          TABLE 2. CFU/ml plating efficiencies pBAD constructs in <br />                                                                                   heterologous complementation assay <br /> double mutant unable reduce ribonucleotides aerobically at <br /> 42&#194;&#176;C unless functional RNR provided complement                                              Mean no. CFU/ <br />                                                                                                                                        Mean plating <br />                                                                              pBAD construct            ml &#226;&#171;&#190; SD 42&#194;&#176;C with <br /> temperature-sensitive phenotype. The entire S. pyogenes                                                                              efficiency &#226;&#171;&#190; SDa <br />                                                                                                           0.3 % arabinose <br /> nrdHEF nrdF*I*E* sequences cloned separately into <br />                                                                         p18-HIEFSal                   1.56 &#226;&#171;&#187; 108 &#226;&#171;&#190; 9.60 &#226;&#171;&#187; 105        0.214 &#226;&#171;&#190; 1.31 &#226;&#171;&#187; 10&#226;&#171;&#186;3 <br /> the pBAD18 vector control E. coli arabinose            p18-HEFSpyo                            &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> promoter (PBAD), on/off switch promoter repressed glu-            p18-FIE*Spyo                  1.56 &#226;&#171;&#187; 108 &#226;&#171;&#190; 8.10 &#226;&#171;&#187; 105        0.214 &#226;&#171;&#190; 1.11 &#226;&#171;&#187; 10&#226;&#171;&#186;3 <br />                                                                         p18-FI*Spyo                            &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> cose induced arabinose, generating plasmids p18-                 p18-FE*Spyo                            &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> HEFSpyo p18-FIE*Spyo, transformed                   p18-IE*Spyo                            &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> into E. coli IG101 strain. Complementation deter-               p18-FI*Spyo/p33-E*Spyo        1.56 &#226;&#171;&#187; 10 &#226;&#171;&#190; 5.69 &#226;&#171;&#187; 105 <br />                                                                                                                8 <br />                                                                                                                                      0.214 &#226;&#171;&#190; 7.78 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br />                                                                         p18-FE*Spyo/p33-I*Spyo        1.56 &#226;&#171;&#187; 10 &#226;&#171;&#190; 4.24 &#226;&#171;&#187; 105 <br />                                                                                                                8 <br />                                                                                                                                      0.213 &#226;&#171;&#190; 5.80 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br /> mined plating serial dilutions liquid cultures LB agar         p18-IE*Spyo/p33-F*Spyo        1.54 &#226;&#171;&#187; 108 &#226;&#171;&#190; 7.78 &#226;&#171;&#187; 105        0.211 &#226;&#171;&#190; 1.06 &#226;&#171;&#187; 10&#226;&#171;&#186;3 <br /> plates supplemented 0.3% L-arabinose 0.3%                p18-HEFSpyo/p33-I2Spyo                 &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br />                                                                         p18-HEFSpyo/p33-I*Spyo                 &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> D-glucose 30 42&#194;&#176;C. As independent positive               p18-HEFSpn/p33-I2Spn          1.55 &#226;&#171;&#187; 108 &#226;&#171;&#190; 9.74 &#226;&#171;&#187; 105        0.212 &#226;&#171;&#190; 1.33 &#226;&#171;&#187; 10 &#226;&#171;&#186;3 <br /> control, used Salmonella enterica serovar Typhimurium            p18-HEFSpn/p33-I2Spyo                  &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br />                                                                         p18-HEFSpn/p33-I*Spyo                  &#226;&#172;?10                     &#226;&#172;?1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> nrdHIEF sequence cloned pBAD18 vector (plasmid                 p18-HEFSpyo/p33-I2Spn         9.66 &#226;&#171;&#187; 10 &#226;&#171;&#190; 3.21 &#226;&#171;&#187; 103 <br />                                                                                                                4 <br />                                                                                                                                   0.00013 &#226;&#171;&#190; 4.39 &#226;&#171;&#187; 10&#226;&#171;&#186;6 <br /> p18-HIEFSal). Unexpectedly, IG101 strain transformed                   a <br />                                                                              Plating efficiencies determined fraction CFU/ml restric- <br /> with p18-HEFSpyo able grow 42&#194;&#176;C pres-              tive conditions compared number CFU/ml nonrestrictive condi- <br /> ence arabinose glucose, IG101 trans-               tions. Values based results independent experiments. <br />  VOL. 190, 2008                                                           CLASS Ib RNR REDUNDANCY IN S. PYOGENES                   4855 <br />  <br />  <br /> either S. pyogenes nrdI* nrdI2 gene, however,          nrdHEF operon required coexpression nrdI gene, this <br /> unable detect complementation restrictive tempera-          case S. pneumoniae. To surprise, addition of <br /> ture, suggesting S. pyogenes NrdI NrdI2          nrdI* nrdI2 S. pyogenes did render nrdHEF <br /> successfully interact S. pneumoniae system. In con-          functional vivo. Moreover, vivo complementation by <br /> trast, growth 42&#194;&#176;C detected combined            nrdF*I*E* achieved genes simul- <br /> the S. pyogenes nrdHEF operon S. pneumoniae nrdI2            taneously present, indicating compulsory requirement <br /> gene, plating efficiency low (Table 2). These        nrdI* gene product. In similar way, S. pneumoniae <br /> results S. pyogenes nrdHEF operon function-         nrdHEF operon able complement coex- <br /> ality vivo, given supplemented functional   pressed nrdI gene. These results S. <br /> nrdI gene.                                                            pyogenes nrdI* functional vivo suggest S. pyogenes <br />                                                                       nrdI2 not. <br />                                                                          An nrdI gene class Ib-containing <br />                          DISCUSSION <br />                                                                       microorganism (http://rnrdb.molbio.su.se/), preliminary <br />    Genomes display multiple RNRs             assays laboratory Salmonella enterica serovar <br /> every group bacteria (41). The concurrence aerobic           Typhimurium class Ib RNR indicate gene required <br /> enzyme (class I) anaerobic (class III) facultative      vivo class Ib ribonucleotide reduction microor- <br /> microorganisms reflects metabolic need. In species        ganism (I. Sala, unpublished results). The Bacillus subtilis nrdI <br /> combinations RNRs used envi-            gene shown essential (22, 36). Protein <br /> ronmental conditions (such classes I II, Ia Ib, II      comparison S. pyogenes NrdI* streptococcal <br /> and III), proposed enzymes          NrdI2 showed distant relatives (36% similarity). <br /> support normal cell growth support              Sequence comparison streptococcal NrdI2 pro- <br /> DNA repair growth recovery (4, 5, 8, 26, 37, 42). The             teins showed different groupings distin- <br /> genome S. pyogenes genomes contain            guished according small differences primary se- <br /> redundant RNR operons&#226;&#8364;&#8221;here, different class Ib operons.           quences (Fig. 6A). The group comprised NrdI2 <br /> Moreover, genes             proteins S. pneumoniae, Streptococcus mutans, Strep- <br /> operons (instead nrdHIEF genes classical           tococcus suis, NrdI-like protein their <br /> class Ib systems). The S. pyogenes nrdHEF cluster lacks nrdI       genomes, second group included NrdI2 proteins <br /> gene, nrdF*I*E* cluster lacks nrdH gene. A second          S. pyogenes, Streptococcus agalactiae, Streptococcus <br /> nrdI gene (nrdI2) S. pyogenes chro-         equi, posses nrdF*I*E* cluster. As shown Fig. <br /> mosome. Both Ib gene clusters transcribed simulta-         6A, clustering sequences separate groups is <br /> neously independent polycistronic operons. In addi-          mainly 9-bp (3 amino acids) indel approximately <br /> tion, Western blotting rabbit-raised nrdF antibodies          10 substituted residues differing groups. Com- <br /> demonstrated presence nrdF nrdF* products             parison B. subtilis NrdI streptococcal NrdI2 proteins <br /> within microorganism (data shown), suggesting           indicates differences affect residues <br /> both enzyme systems expressed simultaneously.                     inferred flavin mononucleotide (FMN) binding (see below), <br />    Only redundant class Ib RNRs S. pyogenes         regions NrdI2 fully conserved all <br /> is active vitro. The NrdF subunit S. pyogenes             streptococcal polypeptide chains. It remarkable the <br /> nrdHEF cluster classical class Ib protein dinuclear       streptococcal species bear nrdI2 gene the <br /> iron center stable tyrosyl radical (30). It catalytically   indel common redundant class Ib operon. <br /> competent assayed vitro presence corre-            Interestingly, nrdFIE gene cluster high levels of <br /> sponding NrdE subunit. Specific activities       similarity streptococcal nrdF*I*E* operon (87%, 57%, <br /> within range described class Ib systems M.       76% similarity NrdF*, NrdI*, NrdE*, respectively) <br /> tuberculosis (48) C. ammoniagenes (11). The nrdHEF-en-            Mycoplasma species. In addition, the <br /> coded enzyme displayed dATP insensitivity,            substitution iron binding residues described for <br /> common feature class Ib enzymes &#226;?&#163; subunit lacks        S. pyogenes NrdF* conserved Mycoplasma NrdFs <br /> 50 60 N-terminal residues class Ia enzymes consti-        (Fig. 6B). Mycoplasma organisms intracellular parasites <br /> tute ATP/dATP allosteric-activity site (29). Collectively         simplest prokaryotes capable self-replication <br /> these results nrdHEF behaves func-          (23). Although Mycoplasma organisms import deoxyribo- <br /> tional class Ib enzyme vitro.                                      nucleotides host, nrdFIE genes encode only <br />    The NrdF* subunit nrdF*I*E* cluster lacks           putative RNR. One possible explanation striking sim- <br /> of conserved chains normally ligate iron        ilarities nrdFIE operons Mycoplasma S. <br /> center. It low iron content lacks tyrosyl radical    pyogenes operon horizontally trans- <br /> well enzyme activity vitro mixtures NrdE*           ferred S. pyogenes compensate loss vivo <br /> subunit. Yet, NrdE* retain catalytically important       functionality streptococcal nrdHEF system, the <br /> residues. Components class Ib cluster cross-          lack functional NrdI2 protein. Further analyses needed <br /> react components cluster.                         clarify suggestion, codon usage current <br />    In vivo heterologous complementation highlights essen-         class Ib nrd genes S. pyogenes agreement that <br /> tiality nrdI. Our vivo results showed completely different    calculated rest S. pyogenes genome (2, 10, 38), <br /> picture. Both S. pyogenes gene clusters proved functional       phagic signatures mobile elements flanking <br /> in heterologous complementation assays                streptococcal nrdF*I*E* genes. Interestingly, results <br />  4856      ROCA ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. (A) Clustal W alignment deduced amino acid fragments nrdI gene. Ssan, Streptococcus sanguis; Spne, Streptococcus <br /> pneumoniae; Ssui, Streptococcus suis; Smut, Streptococcus mutans; Spyo, Streptococcus pyogenes; Sequ, Streptococcus equi; Saga, Streptococcus <br /> agalactiae; Bsub, Bacillus subtilis. Shading sequence names indicates common clustering. Those residues unique group highlighted <br /> in black. Conserved residues streptococci highlighted light gray, FMN binding residues B. subtilis shown dark <br /> gray. (B) Clustal W alignment deduced amino acid fragments nrdF gene. Mpne, Mycoplasma pneumoniae; Styp, Salmonella enterica <br /> serovar Typhimurium. Conserved residues highlighted light gray, conserved radical tyrosyl residue conserved iron binding <br /> residues highlighted dark gray. Substituted residues S. pyogenes, S. equi, S. agalactiae, Mycoplasma pulmonis, Mycoplasma genitalium, and <br /> M. pneumoniae highlighted black. All sequences retrieved RNRdb http://rnrdb.molbio.su.se. <br />  <br />  <br />  <br /> also suggest nrdH dispensable nrdF*I*E* function.               role NrdI flavodoxin reduce iron cluster class <br /> This surprising nrdH gene lacking Myco-          Ib NrdF, allowing reactivation essential <br /> plasma B. subtilis, contain class Ib enzyme.         tyrosyl radical. The iron center Bacillus anthracis NrdF (44) <br /> In E. coli, shown NrdEF reduced                  preliminary experiments efficiently reduced by <br /> glutaredoxin-1 (Grx1) absence NrdH (13), B.                incubation B. anthracis NrdI (M. Sahlin, E. Torrents, and <br /> subtilis, essential thioredoxin-1 (TrxA) proposed           B.-M. Sjo &#194;&#168;berg, unpublished data). Class Ib operons thus <br /> the electron donor class Ib RNR (15). We grx             carry specific electron donors, NrdH, provides re- <br /> homologues S. pyogenes Mycoplasma, contain              ducing power &#226;?&#163; subunit, NrdI, supplies elec- <br /> trx homologue, supply role nrdH.                         trons &#226;?&#164; subunit. <br />    NrdI member flavodoxin family. The three-di- <br />                                                                                Jordan et al. (19) showed activity S. enterica <br /> mensional structure NrdI protein B. subtilis is <br />                                                                             serovar Typhimurium NrdEF enzyme slightly stimu- <br /> available (Protein Data Bank accession no. 1RLJ) pro- <br />                                                                             lated addition partially purified recombinant NrdI <br /> vides evidence NrdI contains flavodoxin-like fold, and <br /> thus, NrdIs classified flavodoxin NrdI family (Pfam              protein, classical NrdEF enzymes analyzed date are <br /> accession no. PF07972). Flavodoxins small electron trans-               active vitro addition NrdI protein (17, 20, <br /> fer proteins utilize noncovalently bound FMN                  21, 44, 48). Likewise, NrdI essential vitro <br /> redox-active component (35). In E. coli, flavodoxin                     activity S. pyogenes NrdEF enzyme. In sharp contrast to <br /> NADPH:flavodoxin oxidoreductase essential class III                 this, S. pyogenes NrdE*F* mixture vitro enzyme <br /> RNR activation (27). Likewise, flavin reductase Fre (12)                activity, likely relating lack iron <br /> and YfaE ferredoxin (47) reactivate non-radical-                  binding chains NrdF*. The obvious vivo activity of <br /> containing met-&#226;?&#164;2 E. coli class Ia RNR. We suggest              S. pyogenes nrdF*I*E* operon suggests NrdI* play <br />  VOL. 190, 2008                                                                              CLASS Ib RNR REDUNDANCY IN S. PYOGENES                                    4857 <br />  <br />  <br /> a direct role ribonucleotide reduction activat-                             Bellido, G. Zhao, C. A. Zook, R. H. Baltz, S. R. Jaskunas, P. R. Rosteck, Jr., <br />                                                                                             P. L. Skatrud, J. I. Glass. 2001. Genome bacterium Streptococcus <br /> ing reduced met-&#226;?&#164;2 protein.                                                               pneumoniae strain R6. J. Bacteriol. 183:5709&#226;&#8364;&#8220;5717. <br />                                                                                       17.   Huque, Y., F. Fieschi, E. Torrents, I. Gibert, R. Eliasson, P. Reichard, M. <br />                           ACKNOWLEDGMENTS                                                   Sahlin, B.-M. Sjo &#194;&#168;berg. 2000. The active form R2F protein class <br />                                                                                             Ib ribonucleotide reductase Corynebacterium ammoniagenes difer- <br />   This work supported grants BFU2004-03383 (Ministerio                            ric protein. J. Biol. Chem. 275:25365&#226;&#8364;&#8220;25371. <br /> Educacio&#194;&#180;n y Ciencia, Spain), 2005SGR-00956 (Generalitat Catalu-                   18.   Jordan, A., E. Aragall, I. Gibert, J. Barbe&#194;&#180;. 1996. Promoter identification <br /> nya, Spain), 621-2004-2864 (Swedish Science Research Council).                          expression analysis Salmonella typhimurium Escherichia coli <br /> I.G. I.R. recipients grant Fundacio <br />                                                           &#194;&#180;n Maria                          nrdEF operons encoding class I ribonucleotide reductases present <br /> Francisca Roviralta. E.T. supported Ramo&#194;&#180;n y Cajal pro-                        bacteria. Mol. Microbiol. 19:777&#226;&#8364;&#8220;790. <br />                                                                                       19.   Jordan, A., F. &#195;&#8230;slund, E. Pontis, P. Reichard, A. Holmgren. 1997. <br /> gram Jeansson Foundations. <br />                                                                                             Characterization Escherichia coli NrdH. A glutaredoxin-like protein with <br />                                                                                             thioredoxin-like activity profile. J. Biol. Chem. 272:18044&#226;&#8364;&#8220;18050. <br />                                  REFERENCES                                           20.   Jordan, A., E. Pontis, F. &#195;&#8230;slund, U. Hellman, I. Gibert, P. Reichard. <br />  1. Andersson, M. E., M. Ho    &#194;&#168;gbom, A. Rinaldo-Matthis, W. Blodig, Y. Liang,              1996. The ribonucleotide reductase Lactococcus lactis. Character- <br />     B. O. Persson, B.-M. Sjo  &#194;&#168;berg, X. D. Su, P. Nordlund. 2004. Structural            ization NrdEF enzyme new electron transport protein. J. Biol. <br />     mutational studies carboxylate cluster iron-free ribonucleotide           Chem. 271:8779&#226;&#8364;&#8220;8785. <br />     reductase R2. Biochemistry 43:7966&#226;&#8364;&#8220;7972.                                          21.   Jordan, A., E. Pontis, M. Atta, M. Krook, I. Gibert, J. Barbe&#194;&#180;, P. <br />  2. Beres, S. B., G. L. Sylva, K. D. Barbian, B. Lei, J. S. Hoff, N. D. Mammarella,         Reichard. 1994. A second class I ribonucleotide reductase Enterobacteri- <br />     M. Y. Liu, J. C. Smoot, S. F. Porcella, L. D. Parkins, D. S. Campbell, T. M.            aceae: characterization Salmonella typhimurium enzyme. Proc. Natl. <br />     Smith, J. K. McCormick, D. Y. Leung, P. M. Schlievert, J. M. Musser.                Acad. Sci. USA 91:12892&#226;&#8364;&#8220;12896. <br />     2002. Genome sequence serotype M3 strain group A Streptococcus:           22.   Kobayashi, K., S. D. Ehrlich, A. Albertini, G. Amati, K. K. Andersen, M. <br />     phage-encoded toxins, high-virulence phenotype, clone emergence.                Arnaud, K. Asai, S. Ashikaga, S. Aymerich, P. Bessieres, F. Boland, S. C. <br />     Proc. Natl. Acad. Sci. USA 99:10078&#226;&#8364;&#8220;10083.                                              Brignell, S. Bron, K. Bunai, J. Chapuis, L. C. Christiansen, A. Danchin, M. <br />  3. Birgander, P. L., S. Bug, A. Kasrayan, S.-L. Dahlroth, M. Westman, E.                   Debarbouille, E. Dervyn, E. Deuerling, K. Devine, S. K. Devine, O. Dreesen, <br />     Gordon, B.-M. Sjo   &#194;&#168;berg. 2005. Nucleotide-dependent formation cat-             J. Errington, S. Fillinger, S. J. Foster, Y. Fujita, A. Galizzi, R. Gardan, C. <br />     alytically competent dimers engineered monomeric ribonucleotide re-                Eschevins, T. Fukushima, K. Haga, C. R. Harwood, M. Hecker, D. Hosoya, <br />     ductase protein R1. J. Biol. Chem. 280:14997&#226;&#8364;&#8220;15003.                                     M. F. Hullo, H. Kakeshita, D. Karamata, Y. Kasahara, F. Kawamura, K. <br />  4. Borovok, I., B. Gorovitz, M. Yanku, R. Schreiber, B. Gust, K. Chater, Y.                Koga, P. Koski, R. Kuwana, D. Imamura, M. Ishimaru, S. Ishikawa, I. Ishio, <br />     Aharonowitz, G. Cohen. 2004. Alternative oxygen-dependent oxy-                  D. Le Coq, A. Masson, C. Mauel, R. Meima, R. P. Mellado, A. Moir, S. <br />     gen-independent ribonucleotide reductases Streptomyces: cross-regulation             Moriya, E. Nagakawa, H. Nanamiya, S. Nakai, P. Nygaard, M. Ogura, T. <br />     physiological role response oxygen limitation. Mol. Microbiol.                Ohanan, M. O&#226;&#8364;&#8482;Reilly, M. O&#226;&#8364;&#8482;Rourke, Z. Pragai, H. M. Pooley, G. Rapoport, <br />     54:1022&#226;&#8364;&#8220;1035.                                                                           J. P. Rawlins, L. A. Rivas, C. Rivolta, A. Sadaie, Y. Sadaie, M. Sarvas, T. <br />  5. Borovok, I., R. Kreisberg-Zakarin, M. Yanko, R. Schreiber, M. Myslovati, F.             Sato, H. H. Saxild, E. Scanlan, W. Schumann, J. F. Seegers, J. Sekiguchi, A. <br />     &#195;&#8230;slund, A. Holmgren, G. Cohen, Y. Aharonowitz. 2002. Streptomyces                   Sekowska, S. J. Seror, M. Simon, P. Stragier, R. Studer, H. Takamatsu, T. <br />     spp. contain class Ia class II ribonucleotide reductases: expression anal-          Tanaka, M. Takeuchi, H. B. Thomaides, V. Vagner, J. M. van Dijl, K. <br />     ysis genes vegetative growth. Microbiology 148:391&#226;&#8364;&#8220;404.                       Watabe, A. Wipat, H. Yamamoto, M. Yamamoto, Y. Yamamoto, K. Yamane, <br />  6. Bradford, M. M. 1976. A rapid sensitive method quantitation              K. Yata, K. Yoshida, H. Yoshikawa, U. Zuber, N. Ogasawara. 2003. <br />     microgram quantities protein utilizing principle protein-dye bind-            Essential Bacillus subtilis genes. Proc. Natl. Acad. Sci. USA 100:4678&#226;&#8364;&#8220;4683. <br />     ing. Anal. Biochem. 72:248&#226;&#8364;&#8220;254.                                                   23.   Koonin, E. V. 2000. How genes make cell: minimal-gene-set <br />  7. Climent, I., B.-M. Sjo&#194;&#168;berg, C. Y. Huang. 1992. Site-directed mutagenesis           concept. Annu. Rev. Genomics Hum. Genet. 1:99&#226;&#8364;&#8220;116. <br />     deletion carboxyl terminus Escherichia coli ribonucleotide          24.   Liu, A., S. Po <br />                                                                                                          &#194;&#168;tsch, A. Davydov, A. L. Barra, H. Rubin, A. Gra  &#194;&#168;slund. 1998. <br />     reductase protein R2. Effects catalytic activity subunit interaction.            The tyrosyl free radical recombinant ribonucleotide reductase My- <br />     Biochemistry 31:4801&#226;&#8364;&#8220;4807.                                                              cobacterium tuberculosis located rigid hydrophobic pocket. Biochem- <br />  8. Dawes, S. S., D. F. Warner, L. Tsenova, J. Timm, J. D. McKinney, G. Kaplan,             istry 37:16369&#226;&#8364;&#8220;16377. <br />     H. Rubin, V. Mizrahi. 2003. Ribonucleotide reduction Mycobacterium         25.   McLaughlin, R. E., J. J. Ferretti. 1995. Electrotransformation strep- <br />     tuberculosis: function expression genes encoding class Ib class II           tococci. Methods Mol. Biol. 47:185&#226;&#8364;&#8220;193. <br />     ribonucleotide reductases. Infect. Immun. 71:6124&#226;&#8364;&#8220;6131.                           26.   Monje-Casas, F., J. Jurado, M. J. Prieto-A  &#194;&#180; lamo, A. Holmgren, C. Pueyo. <br />  9. Federle, M. J., K. S. McIver, J. R. Scott. 1999. A response regulator          2001. Expression analysis nrdHIEF operon Escherichia coli. <br />     represses transcription virulence operons group A strepto-            Conditions trigger transcript level vivo. J. Biol. Chem. 276:18031&#226;&#8364;&#8220; <br />     coccus. J. Bacteriol. 181:3649&#226;&#8364;&#8220;3657.                                                    18037. <br /> 10. Ferretti, J. J., W. M. McShan, D. Ajdic, D. J. Savic, G. Savic, K. Lyon, C. <br />                                                                                       27.   Mulliez, E., D. Padovani, M. Atta, C. Alcouffe, M. Fontecave. 2001. <br />     Primeaux, S. Sezate, A. N. Suvorov, S. Kenton, H. S. Lai, S. P. Lin, Y. Qian, <br />                                                                                             Activation class III ribonucleotide reductase flavodoxin: protein <br />     H. G. Jia, F. Z. Najar, Q. Ren, H. Zhu, L. Song, J. White, X. Yuan, S. W. <br />                                                                                             radical-driven electron transfer iron-sulfur center. Biochemistry 40: <br />     Clifton, B. A. Roe, R. McLaughlin. 2001. Complete genome sequence of <br />                                                                                             3730&#226;&#8364;&#8220;3736. <br />     M1 strain Streptococcus pyogenes. Proc. Natl. Acad. Sci. USA 98:4658&#226;&#8364;&#8220; <br />                                                                                       28.   Nicholas, K. B., H. B. J. Nicholas. 1997. GeneDoc: tool editing and <br />     4663. <br />                                                                                             annotating multiple sequence alignments v. 2.6.001. K. B. Nicholas and <br /> 11. Fieschi, F., E. Torrents, L. Toulokhonova, A. Jordan, U. Hellman, J. Barbe&#194;&#180;, <br />                                                                                             H. B. J. Nicholas (http://www.nrbsc.org/gfx/genedoc/index.html). <br />     I. Gibert, M. Karlsson, B.-M. Sjo  &#194;&#168;berg. 1998. The manganese-containing <br />     ribonucleotide reductase Corynebacterium ammoniagenes class Ib            29.   Nordlund, P., P. Reichard. 2006. Ribonucleotide reductases. Annu. Rev. <br />     enzyme. J. Biol. Chem. 273:4329&#226;&#8364;&#8220;4337.                                                   Biochem. 75:681&#226;&#8364;&#8220;706. <br /> 12. Fontecave, M., R. Eliasson, P. Reichard. 1989. Enzymatic regulation        30.   Petersson, L., A. Gra &#194;&#168;slund, A. Ehrenberg, B.-M. Sjo   &#194;&#168;berg, P. Reichard. <br />     radical content small subunit Escherichia coli ribonucleotide             1980. The iron center ribonucleotide reductase Escherichia coli. <br />     reductase involving reduction redox centers. J. Biol. Chem. 264:9164&#226;&#8364;&#8220;            J. Biol. Chem. 255:6706&#226;&#8364;&#8220;6712. <br />     9170.                                                                             31.   Platz, A., B.-M. Sjo  &#194;&#168;berg. 1980. Construction characterization of <br /> 13. Gon, S., M. J. Faulkner, J. Beckwith. 2006. In vivo requirement                 hybrid plasmids containing Escherichia coli nrd region. J. Bacteriol. <br />     glutaredoxins thioredoxins reduction ribonucleotide reduc-            143:561&#226;&#8364;&#8220;568. <br />     tases Escherichia coli. Antioxid. Redox Signal. 8:735&#226;&#8364;&#8220;742.                     32.   Sahlin, M., L. Petersson, A. Gra &#194;&#168;slund, A. Ehrenberg, B.-M. Sjo  &#194;&#168;berg, L. <br /> 14. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu-               Thelander. 1987. Magnetic interaction tyrosyl free radical and <br />     lation, modulation, high-level expression vectors containing ar-             antiferromagnetically coupled iron center ribonucleotide reductase. <br />     abinose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130.                                     Biochemistry 26:5541&#226;&#8364;&#8220;5548. <br /> 15. Ha &#194;&#168;rtig, E., A. Hartmann, M. Scha  &#194;&#168;tzle, A. M. Albertini, D. Jahn. 2006.    33.   Sahlin, M., B.-M. Sjo &#194;&#168;berg, G. Backes, T. Loehr, J. Sanders-Loehr. 1990. <br />     The Bacillus subtilis nrdEF genes, encoding class Ib ribonucleotide reduc-            Activation iron-containing B2 protein ribonucleotide reductase by <br />     tase, essential aerobic anaerobic growth. Appl. Environ. Micro-             hydrogen peroxide. Biochem. Biophys. Res. Commun. 167:813&#226;&#8364;&#8220;818. <br />     biol. 72:5260&#226;&#8364;&#8220;5265.                                                               34.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br /> 16. Hoskins, J., W. E. Alborn, Jr., J. Arnold, L. C. Blaszczak, S. Burgett, B. S.           laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Plain- <br />     DeHoff, S. T. Estrem, L. Fritz, D. J. Fu, W. Fuller, C. Geringer, R. Gilmour,           view, NY. <br />     J. S. Glass, H. Khoja, A. R. Kraft, R. E. Lagace, D. J. LeBlanc, L. N. Lee,       35.   Sancho, J. 2006. Flavodoxins: sequence, folding, binding, function be- <br />     E. J. Lefkowitz, J. Lu, P. Matsushima, S. M. McAhren, M. McHenney, K.                   yond. Cell. Mol. Life Sci. 63:855&#226;&#8364;&#8220;864. <br />     McLeaster, C. W. Mundy, T. I. Nicas, F. H. Norris, M. O&#226;&#8364;&#8482;Gara, R. B. Peery,        36.   Scotti, C., A. Valbuzzi, M. Perego, A. Galizzi, A. M. Albertini. 1996. The <br />     G. T. Robertson, P. Rockey, P. M. Sun, M. E. Winkler, Y. Yang, M. Young-                Bacillus subtilis genes ribonucleotide reductase similar genes <br />  4858         ROCA ET AL.                                                                                                                                   J. BACTERIOL. <br />  <br />       second class I NrdE/NrdF enzymes Enterobacteriaceae. Microbi-              sion transcriptional analysis aerobic enzymes. J. Biol. Chem. <br />       ology 142:2995&#226;&#8364;&#8220;3004.                                                                  280:16571&#226;&#8364;&#8220;16578. <br /> 37.   Smalley, D., E. R. Rocha, C. J. Smith. 2002. Aerobic-type ribonucleotide    43.   Torrents, E., I. Roca, I. Gibert. 2003. Corynebacterium ammoniagenes <br />       reductase anaerobe Bacteroides fragilis. J. Bacteriol. 184:895&#226;&#8364;&#8220;903.            class Ib ribonucleotide reductase: transcriptional regulation atypical <br /> 38.   Smoot, J. C., K. D. Barbian, J. J. Van Gompel, L. M. Smoot, M. S. Chaussee,           genomic organization nrd cluster. Microbiology 149:1011&#226;&#8364;&#8220;1020. <br />       G. L. Sylva, D. E. Sturdevant, S. M. Ricklefs, S. F. Porcella, L. D. Parkins,   44.   Torrents, E., M. Sahlin, D. Biglino, A. Gra   &#194;&#168;slund, B.-M. Sjo&#194;&#168;berg. 2005. <br />       S. B. Beres, D. S. Campbell, T. M. Smith, Q. Zhang, V. Kapur, J. A. Daly,             Efficient growth inhibition Bacillus anthracis knocking ribonu- <br />       L. G. Veasy, J. M. Musser. 2002. Genome sequence comparative                  cleotide reductase tyrosyl radical. Proc. Natl. Acad. Sci. USA 102:17946&#226;&#8364;&#8220; <br />       microarray analysis serotype M18 group A Streptococcus strains associated          17951. <br />       acute rheumatic fever outbreaks. Proc. Natl. Acad. Sci. USA 99:4668&#226;&#8364;&#8220;       45.   Voegtli, W. C., M. Sommerhalter, L. Saleh, J. Baldwin, J. M. Bollinger, Jr., and <br />       4673.                                                                                 A. C. Rosenzweig. 200 <br /> </body></html>