<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Aug. 1995, p. 4167&#226;&#8364;&#8220;4174                                                                                    Vol. 15, No. 8 <br /> 0270-7306/95/$04.0010 <br /> Copyright 1995, American Society Microbiology <br />  <br />  <br />  <br />                Yeast mRNA Cap Methyltransferase Is 50-Kilodalton <br />                       Protein Encoded Essential Gene <br />                              XIANGDONG MAO,1 BEATE SCHWER,2                        AND   STEWART SHUMAN1* <br />                 Molecular Biology Program, Sloan-Kettering Institute, New York, New York 10021,1 Department <br />                      Biochemistry, Robert Wood Johnson Medical School, Piscataway, New Jersey 088542 <br />                             Received 15 March 1995/Returned modification 25 April 1995/Accepted 2 May 1995 <br />  <br />             RNA (guanine-7-)methyltransferase, enzyme responsible methylating 5 cap structure eukary- <br />          otic mRNA, isolated extracts Saccharomyces cerevisiae. The yeast enzyme catalyzed methyl group <br />          transfer S-adenosyl-L-methionine guanosine <span id='am-81' about='xsp:base' typeof='owl:Thing'>base</span> capped, unmethylated poly(A). Cap methyl- <br />          ation stimulated low concentrations salt inhibited S-adenosyl-L-homocysteine, pre- <br />          sumptive product reaction, S-adenosyl-D-homocysteine. The methyltransferase sedimented in <br />          glycerol gradient single discrete component 3.2S. A likely candidate gene encoding yeast cap <br />          methyltransferase singled phylogenetic grounds. The ABD1 gene, located yeast chromosome II, <br />          encodes 436-amino-acid (50-kDa) polypeptide displays regional similarity catalytic domain the <br />          vaccinia virus cap methyltransferase. That ABD1 gene product RNA (guanine-7-)methyltrans- <br />          ferase established expressing ABD1 protein bacteria, purifying protein homogeneity, and <br />          characterizing cap methyltransferase activity intrinsic recombinant ABD1. The physical biochemical <br />          properties recombinant ABD1 methyltransferase indistinguishable cap methyltrans- <br />          ferase isolated partially purified whole-cell yeast extracts. Our finding ABD1 gene required <br />          yeast growth provides genetic evidence cap methyltransferase (and, inference, cap <br />          methyl group) plays essential role cellular function vivo. <br />  <br />  <br />    Eukaryotic mRNAs contain 59-terminal cap structure con-                sisting polypeptides 80 52 kDa (21). RNA <br /> sisting 7-methylguanosine linked 59 end tran-              triphosphatase activity intrinsic 80-kDa subunit (21), <br /> script 59-59 triphosphate bridge (3). Capping occurs             52-kDa subunit guanylyltransferase (21, 53). <br /> series enzymatic reactions 59 triphosphate            The CEG1 gene encoding 457-amino-acid guanylyltrans- <br /> terminus primary transcript cleaved diphos-              ferase subunit essential cell viability (40). The finding <br /> phate-terminated RNA RNA triphosphatase, capped                     guanylyltransferase activity CEG1 essential pro- <br /> with GMP RNA guanylyltransferase, finally methylated                 vided genetic evidence capping (and, inference, <br /> at N-7 position guanine RNA (guanine-7-)methyl-                   cap) essential vivo (11, 39). <br /> transferase:                                                                   Studies cap methylation focused, part, <br />                                                                             vaccinia virus capping enzyme (6, 15, 16, 26, 28). Al- <br />  pppN(pN)n 3 ppN(pN)n 1 Pi                                                  RNA (guanine-7-)methyltransferase iso- <br />  ppN(pN)n 1 pppG 7 G(59)pppN(pN)n 1 PPi                                     lated rat liver (29) partially purified HeLa cells <br />                                                                             (9), case cellular cap methyltransferase been <br />  G(59)pppN(pN)n 1 AdoMet 3 m7G(59)pppN(pN)n 1 AdoHcy                        purified homogeneity, identification <br />                                                                             cellular gene encoding cap methyltransferase. The <br /> where AdoMet S-adenosylmethionine AdoHcy S-                       7-methyl group guanosine essential RNA <br /> adenosylhomocysteine. This synthetic pathway defined                    transactions cap dependence demonstrated <br /> originally studies vaccinia virus reovirus mRNA               vitro (e.g., translation, pre-mRNA splicing) (1, 8, 23, 32, 33). <br /> synthesis (13, 31). Cellular transcripts capped             Hence, understanding role cap cellular <br /> series reactions, enzymes catalyze steps             RNA metabolism require biochemical genetic analysis <br /> been isolated number cellular sources (30). A dis-                RNA-methylating enzyme(s). In paper, report <br /> tinctive feature cellular enzymes vis-a`-vis well-char-          partial purification RNA (guanine-7-)methyltransferase <br /> acterized vaccinia virus capping enzyme lack tight                extracts S. cerevisiae identification gene <br /> physical association guanylyltransferase                    encoding enzyme. Yeast cap methyltransferase 436- <br /> methyltransferase activities. Whereas capping methyl-               amino-acid polypeptide regional sequence similarity <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ation functions vaccinia virus protein disso-              cap methyltransferases encoded DNA viruses. The cap <br /> ciable conventional purification (50),             methyltransferase gene essential yeast cell growth. <br /> that activities readily separate chromatogra- <br /> phy cell extracts (29). <br />                                                                                                MATERIALS AND METHODS <br />    RNA guanylyltransferase purified HeLa cells, <br /> rat liver, calf thymus, mouse myeloma, brine shrimp, wheat                    Methyltransferase assay. RNA (guanine-7)methyltransferase assayed by <br /> germ, Saccharomyces cerevisiae (30, 43). The enzyme                conversion 32P-cap-labeled poly(A), GpppA(pA)n, methylated capped <br />                                                                             poly(A), m7GpppA(pA)n, presence unlabeled AdoMet (41). Assay <br /> S. cerevisiae purified bifunctional complex con-              mixtures (10 ml) contained 50 mM Tris HCl, (pH 7.5), 5 mM dithiothreitol, <br />                                                                             AdoMet, 32P-cap-labeled acceptor RNA, enzyme. After incubation 378C <br />                                                                             5 min, reaction mixtures heated 958C 3 min adjusted <br />                                                                             50 mM sodium acetate (pH 5.5). Samples incubated 5 mg of <br />   * Corresponding author.                                                   nuclease P1 30 60 min 378C. The digests spotted polyeth- <br />  <br />                                                                      4167 <br />  4168       MAO ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> yleneimine-cellulose thin-layer chromatography (TLC) plates devel- <br /> oped 0.35 M (NH4)2SO4. Labeled products detected autoradiog- <br /> raphy. The extent methylation cap [as m7GpppA/(m7GpppA 1 GpppA)] <br /> was determined excising labeled species counting liquid <br /> scintillation fluid scanning TLC plate FUJIX BAS1000 phos- <br /> phorimager. <br />    Partial purification cap methyltransferase yeast whole-cell extracts. <br /> Extract prepared described previously (25) mid-logarithmic culture <br /> of wild-type S. cerevisiae SS328 (51) grown yeast-peptone-dextrose (YPD) <br /> medium 308C. The protein concentration extract (in 20 mM N-2- <br /> hydroxyethylpiperazine-N9-2-ethanesulfonic acid [HEPES-KOH; pH 7.9], 0.2 <br /> mM EDTA, 50 mM KCl, 0.5 mM dithiothreitol, plus 20% glycerol) 12.7 <br /> mg/ml. (Protein concentration determined using Bio-Rad dye reagent <br /> with bovine serum albumin standard.) All operations purification <br /> were performed 48C. A 1-ml aliquot extract diluted fivefold buffer <br /> A (50 mM Tris HCl [pH 8.0], 1 mM EDTA, 2 mM dithiothreitol, 0.1% Triton <br /> X-100, 10% glycerol) applied 2-ml column phosphocellulose had <br /> been equilibrated buffer A. After washed buffer A, column          FIG. 1. Phosphocellulose chromatography yeast cap methyltransferase. <br /> was eluted stepwise (6 ml step) buffer A containing 0.1, 0.2, 0.5, 1   Extract preparation phosphocellulose chromatography described Ma- <br /> M NaCl. Methyltransferase activity recovered primarily 0.5 M NaCl        terials Methods. Methyltransferase assay mixtures contained 9 fmol 32P- <br /> fraction, contained 0.5 mg protein. An aliquot (0.2 ml) fraction   cap-labeled poly(A), 50 mM AdoMet, 1 0.2 ml whole-cell extract, <br /> was analyzed glycerol gradient sedimentation. The rest 0.5 M phos-        1 ml phosphocellulose flowthrough fraction (F.T.), 1 ml fractions <br /> phocellulose preparation dialyzed buffer B (25 mM Tris HCl [pH          collected serially step elution 0.1, 0.2, 0.5, 1 M NaCl. A control <br /> 8.0], 0.5 mM EDTA, 1 mM dithiothreitol, 0.05% Triton X-100, 5% glycerol)        reaction mixture contained enzyme (2E). An autoradiogram thin-layer <br /> then applied 1-ml column DEAE-cellulose equilibrated buffer B.           chromatogram shown. The origin (Ori) positions methylated and <br /> Methyltransferase activity recovered DEAE flowthrough fraction;     unmethylated caps indicated left. <br /> material applied 1-ml column heparin-agarose been <br /> equilibrated buffer B. The column eluted stepwise 0.1, 0.2, 0.5, and <br /> 1 M NaCl buffer A (4 ml step). Methyltransferase activity recovered <br /> predominantly 0.5 M NaCl eluate (30 mg protein). The heparin-agarose        Plasmid shuffle. A derivative YBS4 lost URA3 gene recom- <br /> fraction dialyzed buffer B. All fractions stored 2808C      bination hisG repeats selected growth 5-fluoroorotic acid <br /> thawed ice prior use. Methyltransferase activity stable multiple       (5-FOA). Into strain (YBS5) introduced p360-ABD, derivative of <br /> rounds freezing thawing.                                                     vector pSE-360 (CEN URA3) contains 1.84-kbp ABD1 insert starting 530 <br />    Expression yeast cap methyltransferase bacteria. A DNA fragment            bp upstream translation start codon ABD1 protein ending at <br /> containing ABD1 gene isolated PCR amplification total genomic         ABD1 translation stop codon. After sporulation YBS5/p360-ABD, Ura1 <br /> DNA S. cerevisiae YPH274 oligonucleotides corresponding 59         haploid progeny tested growth 5-FOA. We obtained strain <br /> end (59-GGGGAAGTCATATGTCAACCAAACCAG) 39 end (39-                         YBS6, deletion chromosomal ABD1 locus viability <br /> CGCATTTCGGGTTGACTCCTAGGCCC) ABD1 gene (7). These oli-                        contingent maintenance p360-ABD. A second plasmid, p358-ABD <br /> gonucleotides introduced NdeI restriction site start codon BamHI    (derived vector pSE-358 [CEN TRP1] containing 1.84-kbp <br /> cleavage site just 39 stop codon. The amplified DNA fragment digested    ABD1 insert p360-ABD), introduced YBS6. A control transformation <br /> with NdeI BamHI ligated T7-based expression vector            performed pSE-358 vector. Trp1 Ura1 transformants selected <br /> pET16b (Novagen) generate plasmid pET-His-ABD1. The sequence              subsequently streaked medium containing 5-FOA (47). <br /> ABD1 insert pET-His-ABD1 (determined dideoxy sequencing) iden- <br /> tical ABD1 sequence deposited GenBank (7). pET-His-ABD1 was <br /> transformed Escherichia coli BL21(DE3). Expression plasmid-en-                                            RESULTS <br /> coded protein induced addition 0.4 mM isopropyl-b-D-thiogalactopy- <br /> ranoside (IPTG) exponentially growing 200-ml culture BL21(DE3)/pET-           Partial purification cap methyltransferase yeast cell <br /> His-ABD1 Luria-Bertani medium (containing 0.1 mg ampicillin ml)        extracts. RNA (guanine-7-)methyltransferase activity was <br /> 378C. Cells harvested centrifugation 4 h postinduction stored at <br /> 2808C. <br />                                                                                     demonstrated yeast whole-cell extracts sensitive and <br />    Purification recombinant cap methyltransferase. All procedures per-      specific assay eschews detection transmethylation reac- <br /> formed 48C. Cell lysis achieved treatment thawed, resuspended          tions unrelated capping. This assay measures conversion <br /> bacteria lysozyme Triton X-100, using modified lysis buffer containing     32P-cap-labeled poly(A) methylated cap-labeled poly(A) <br /> 50 mM Tris HCl (pH 7.5), 0.15 M NaCl, 10% sucrose (26). Insoluble <br /> material removed centrifugation 18,000 rpm 30 min Sorvall <br />                                                                                     presence unlabeled AdoMet. The reaction products <br /> SS34 rotor. The supernatant mixed 2 ml Ni-nitrilotriacetic acid-        digested cap dinucleotides nuclease P1 then <br /> agarose resin (Qiagen) 1 h. The slurry poured column        analyzed polyethyleneimine-cellulose TLC, resolved <br /> washed lysis buffer. The column eluted stepwise IMAC buffer (20       GpppA cap methylated cap m7GpppA. Incubation <br /> mM Tris HCl [pH 7.9], 0.5 M NaCl, 1 mM phenylmethylsulfonyl fluoride, 10% <br /> glycerol) containing 25, 45, 200, 500 mM imidazole. The polypeptide com- <br />                                                                                     yeast extract (a whole-cell 100,000 3 g supernatant fraction) <br /> position column fractions monitored sodium dodecyl sulfate-           cap-labeled substrate AdoMet resulted sub- <br /> polyacrylamide gel electrophoresis (SDS-PAGE). The His-ABD1 polypeptide             stantial conversion GpppA ends negatively charged <br /> eluted 200 mM imidazole. This fraction dialyzed buffer B    species comigrated m7GpppA (Fig. 1). The extent of <br /> applied 2-ml column phosphocellulose equilibrated with <br /> buffer B. The column developed stepwise buffer A containing 0.05, 0.1, <br />                                                                                     methylation proportional extract added <br /> 0.2, 0.5, 1 M NaCl. His-ABD1 recovered 0.5 M NaCl fraction (0.62     (Fig. 1). Partial purification activity achieved by <br /> mg protein). An aliquot (62 mg) phosphocellulose preparation          phosphocellulose column chromatography. The methyltrans- <br /> applied 4.8-ml 15 30% glycerol gradient containing 0.5 M NaCl buffer     ferase activity adsorbed quantitatively phosphocellulose <br /> A. The gradient centrifuged 50,000 rpm Beckman SW50 rotor 15 <br /> h 48C. Fractions (0.14 ml) collected the tube. Marker <br />                                                                                     low ionic strength recovered step elution, <br /> proteins&#226;&#8364;&#8221;catalase, bovine serum albumin, cytochrome c&#226;&#8364;&#8221;were sedimented           predominantly 0.5 M NaCl fraction (Fig. 1). The 0.5 M <br /> in parallel gradients.                                                              phosphocellulose preparation contained approximately 4% of <br />    Disruption ABD1 gene. The ABD1 gene disrupted insertion       protein applied column. The specific methyltrans- <br /> hisG-URA3-hisG cassette described previously (2, 36). The disruption was <br /> performed diploid strain, YBS3 (MATa/MATa ura3/ura3 lys2/lys2 his3/his3 <br />                                                                                     ferase activity preparation increased 120-fold <br /> trp1/trp1 LEU2/leu2). Integrative recombination chromosomal locus elim-      relative extract. The apparent increase total methyl- <br /> inated ABD1 coding sequence amino acid positions 1 432 replace-      transferase activity probably reflected removal interfer- <br /> ment URA3 cassette. Correct insertion ABD1 locus con-         ing activities phosphocellulose chromatography. <br /> firmed Southern blotting Ura1 transformants (strain YBS4). After <br /> sporulation dissection 20 YBS4 tetrads, 32 viable haploids recov- <br />                                                                                        The native size yeast methyltransferase deter- <br /> ered, Ura2. Failure recover Ura1 haploids indicated       mined sedimentation phosphocellulose preparation <br /> deletion ABD1 gene lethal.                                               glycerol gradient. Given prior estimates the <br />  VOL. 15, 1995                                                                                  YEAST mRNA CAP METHYLTRANSFERASE                                 4169 <br />  <br />  <br />  <br />  <br />    FIG. 2. Characterization partially purified yeast cap methyltransferase. (A) Glycerol gradient sedimentation. An aliquot (0.2 ml) 0.5 M NaCl phospho- <br /> cellulose fraction yeast cap methyltransferase applied 4.8-ml 15 30% glycerol gradient containing 0.5 M NaCl buffer A. The gradient centrifuged <br /> at 50,000 rpm Beckman SW50 rotor 12 h 48C. Fractions (0.17 ml) collected the tube. Methyltransferase reaction mixtures contained <br /> 18 fmol cap-labeled poly(A), 50 mM AdoMet, 1 ml 1:4 dilution indicated gradient fractions. The extent cap methylation (in femtomoles) plotted <br /> as function fraction number. Vertical arrows indicate positions marker proteins sedimented parallel gradient (BSA, bovine serum albumin; <br /> cyt C, cytochrome c). (B) AdoMet dependence cap methylation. Reaction mixtures contained 10 fmol cap-labeled poly(A), 50 mM NaCl, 0.5 ml dialyzed <br /> heparin-agarose fraction, AdoMet indicated. (C) Effect NaCl. Reaction mixtures contained 10 fmol cap-labeled poly(A), 50 mM AdoMet, 1 ml of <br /> heparin-agarose enzyme, NaCl indicated. (D) Stereospecific inhibition yeast cap methyltransferase AdoHcy. Reaction mixtures contained 23 fmol of <br /> cap-labeled poly(A), 10 mM AdoMet, 50 mM NaCl, 1 ml heparin-agarose fraction, D- L-AdoHcy concentrations indicated. <br />  <br />  <br />  <br /> native size mammalian cap methyltransferase varied                           part, identify gene encoding enzyme reverse <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> 56 kDa (9) 130 kDa (29), performed analysis                          genetics. While effort way, identified on <br /> under conditions resolve proteins large 350                         phylogenetic grounds candidate gene scanning avail- <br /> kDa. A single peak yeast methyltransferase activity                           able sequence database yeast proteins related cata- <br /> recovered; sedimentation coefficient 3.2S estimated                         lytic domain vaccinia virus RNA (guanine-7-)methyl- <br /> relative marker proteins centrifuged parallel gradient                       transferase. Although genes encoding subunits of <br /> (Fig. 2A).                                                                           vaccinia virus capping enzyme known some <br />    Additional chromatography trials performed di-                         time, recently methyltransferase guanylyl- <br /> alysis phosphocellulose fraction. We yeast                         transferase activities assigned distinct autonomous do- <br /> cap methyltransferase did adsorb DEAE-cellulose                           mains (6, 15, 16, 26, 42, 46). For example, guanylyltrans- <br /> was retained heparin-agarose step eluted 0.5 M NaCl                        ferase localized N-terminal 60-kDa segment the <br /> (results shown). Initial characterization partially                       844-amino-acid (97-kDa) D1 subunit vaccinia virus capping <br /> purified enzyme performed heparin-agarose prep-                         enzyme (15, 46). Several sequence motifs present vac- <br /> aration dialyzed buffer lacking salt. For-                     cinia virus guanylyltransferase domain conserved the <br /> mation m7GpppA RNA ends depended absolutely added                              order spacing guanylyltrans- <br /> AdoMet. Activity increased increasing AdoMet concen-                            ferases encoded S. cerevisiae Schizosaccharomyces <br /> tration 1 10 mM plateaued (Fig. 2B).                          pombe (45). We presumed phylogenetic conservation <br /> Cap methylation inhibited concentration-dependent                           apply cap-methylating enzymes. <br /> fashion L-AdoHcy, product methyl transfer reaction                          The catalytic center vaccinia virus methyltransferase is <br /> (Fig. 2D). D-AdoHcy far potent inhibitor (Fig. 2D).                      contained 305-amino-acid C-terminal segment the <br /> Cap methylation stimulated severalfold low concentra-                         844-amino-acid D1 subunit (16, 26). A search GenBank with <br /> tions salt inhibited progressively NaCl added                       region vaccinia virus enzyme readily identified the <br /> excess 100 mM (Fig. 2C).                                                          known homologs encoded Shope fibroma virus (49) and <br />    A candidate yeast cap methyltransferase gene. Our intent                       African swine fever virus (35). The highest score to <br /> purifying cap methyltransferase yeast extracts was,                      predicted product S. cerevisiae ABD1 gene, is <br />  4170       MAO ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Sequence similarity yeast ABD1 methyltransferase domain viral capping enzymes. Amino acid sequences C-terminal portions <br /> of capping enzyme large subunits vaccinia virus (Vac), Shope fibroma virus (SFV), African swine fever virus (ASF) aligned 205-amino-acid region <br /> of polypeptide encoded ABD1 gene S. cerevisiae (Sc). Residues yeast protein conserved viral enzymes highlighted <br /> in shaded boxes. The bracketed region corresponds presumed AdoMet-binding motif numerous AdoMet-dependent methyltransferases. The residues <br /> indicated asterisks defined mutational analysis important catalytic activity vaccinia virus cap methyltransferase. <br />  <br />  <br />  <br />  <br /> located chromosome II (10). Although computer-based                            ABD1 protein (results shown). Given bacterial RNAs <br /> alignment relatively weak (blast score 5 58), re-                          capped bacterial extracts contain endoge- <br /> markable presence ABD1 (a 436-amino-acid                                   nous RNA (guanine-7-)methyltransferase detectable our <br /> polypeptide) sequence motifs conserved                           standard assay (27), concluded methyltransferase <br /> known viral methyltransferases, including putative AdoMet-                          activity detected phosphocellulose chromatography was <br /> binding motif, VLELGCGKG (boldface type indicates con-                                intrinsic His-ABD1. <br /> served residues) (Fig. 3) (19). The apparent conservation                             The phosphocellulose preparation His-ABD1 centri- <br /> sequence S. cerevisiae protein vaccinia                           fuged 15 30% glycerol gradient. Cap methyltrans- <br /> virus, Shope fibroma virus, African swine fever virus cap-                        ferase activity sedimented single peak coincident the <br /> ping enzymes enhanced manually aligning proteins,                          sedimentation profile His-ABD1 polypeptide (Fig. 5). <br /> as shown Fig. 3. The region sequence similarity spans                         At stage, preparation homogeneous respect <br /> 205-amino-acid segment ABD1 residues 168 372.                              His-ABD1. The apparent sedimentation coefficient of <br /> Most striking conservation ABD1 sequence                             3.2S (relative markers analyzed parallel) suggested that <br /> IHY (denoted asterisks Fig. 3) shown                            recombinant yeast methyltransferase monomer. We <br /> essential catalysis vaccinia virus cap methyltrans-                        emphasize sedimentation behavior recombinant <br /> ferase constitutes enzyme active site (26,                      ABD1 protein (Fig. 5) identical cap meth- <br /> 27). The retention functionally important residues suggested                       yltransferase isolated S. cerevisiae (Fig. 2A). <br /> to ABD1 encode yeast (guanine-7-)methyl-                               Characterization ABD1 cap methyltransferase activ- <br /> transferase.                                                                          ity. The properties recombinant methyltransferase were <br />    ABD1 encodes yeast cap methyltransferase. To test hy- <br /> pothesis, expressed ABD1 protein bacteria. An in- <br /> ducible T7 RNA polymerase-based vector short <br /> histidine-rich amino-terminal leader segment fused the <br /> ABD1 polypeptide constructed. The expression plasmid <br /> was introduced E. coli BL21(DE3), strain contains <br /> the T7 RNA polymerase gene control lacUV5 <br /> promoter. A 52-kDa polypeptide corresponding His-ABD1 <br /> was detectable SDS-PAGE soluble extracts IPTG- <br /> induced bacteria (results shown). The His tag allowed for <br /> rapid enrichment ABD1 basis affinity tag <br /> for immobilized nickel. The bacterial lysate applied to <br /> Ni-agarose, adsorbed material step-eluted in- <br /> creasing concentrations imidazole. SDS-PAGE analysis re- <br /> vealed prominent 52-kDa Coomassie blue-stained species in <br /> the 200 mM imidazole eluate (Fig. 4). This polypeptide not <br /> recovered lysates IPTG-induced BL21(DE3) carrying <br /> either pET vector pET plasmids expressing other <br /> His-tagged gene products subjected Ni-aga-                              FIG. 4. Ni-agarose chromatography His-ABD1. A soluble lysate from <br />                                                                                       bacteria induced express His-ABD1 adsorbed Ni-agarose. Aliquots (20 <br /> rose chromatography procedure (results shown).                                    ml) flowthrough fraction (FT) fractions collected serially during <br />    His-ABD1 purified adsorption phospho-                            step elution 25, 45, 200, 500 mM imidazole denatured SDS <br /> cellulose stepwise elution salt. The 52-kDa polypep-                         analyzed electrophoresis 10% polyacrylamide gel containing <br /> tide recovered 0.5 M NaCl fraction (Fig. 5). Several                       0.1% SDS. Polypeptides visualized staining Coomassie blue dye. A <br />                                                                                       photograph stained gel shown. The positions sizes (in kilodaltons) <br /> low-molecular-weight contaminant polypeptides elimi-                             coelectrophoresed marker proteins indicated left. The His-ABD1 <br /> nated step. Cap methyltransferase activity eluted                         polypeptide, eluted 200 mM imidazole, denoted arrow the <br /> from phosphocellulose column coincident His-                             right. <br />  VOL. 15, 1995                                                                                  YEAST mRNA CAP METHYLTRANSFERASE                                 4171 <br />  <br />  <br />  <br />  <br />                                                                                          FIG. 6. Characterization recombinant yeast cap methyltransferase. (A) <br />                                                                                       AdoMet dependence. Reaction mixtures contained 23 fmol cap-labeled <br />                                                                                       poly(A), 40 mM NaCl, 9 fmol His-ABD1 (glycerol fraction), AdoMet as <br />                                                                                       indicated. (B) Inhibition AdoMet analogs. Reaction mixtures contained 23 <br />                                                                                       fmol cap-labeled poly(A), 10 mM AdoMet, 40 mM NaCl, 3.6 fmol His- <br />                                                                                       ABD1, L-AdoHcy, D-AdoHcy, L-AdoEt indicated. <br />  <br />  <br />  <br />                                                                                       digestion alkaline phosphatase, indicating block- <br />                                                                                       ing guanosine moiety present 59 end. Treatment <br />    FIG. 5. Glycerol gradient sedimentation His-ABD1. The phosphocellu- <br /> lose His-ABD1 preparation sedimented glycerol gradient described in <br />                                                                                       reaction product nuclease P1 liberated species <br /> Materials Methods. (A) Methyltransferase reaction mixtures contained 18           comigrated m7GpppA (as standard methyl- <br /> fmol cap-labeled poly(A), 50 mM AdoMet, 1 ml 1:200 dilution        transferase assay); product P1 digestion remained resis- <br /> indicated fractions. Vertical arrows indicate positions marker proteins   tant alkaline phosphatase (Fig. 7). When reaction prod- <br /> were sedimented parallel gradient. (B) Aliquots (30 ml) phosphocel- <br /> lulose fraction indicated glycerol fractions 1% SDS and <br />                                                                                       uct treated nucleotide pyrophosphatase (NPPase) <br /> analyzed SDS-PAGE. A photograph Coomassie blue-stained gel               nuclease P1, different species liberated; this <br /> shown. The positions sizes (in kilodaltons) coelectrophoresed marker           species comigrated m7GMP completely suscepti- <br /> proteins indicated left.                                                   ble alkaline phosphatase (which quantitatively released 32Pi <br />                                                                                       m7GMP) (Fig. 7). Formation m7GMP NPPase <br />                                                                                       digestion depended prior reaction His-ABD1. Diges- <br /> determined peak glycerol gradient fraction His-                           tion input cap-labeled poly(A) substrate NPPase <br /> ABD1. Methylation capped poly(A) varied input en-                             yielded [32P]GMP, converted 32Pi alkaline <br /> zyme quantitative saturation. His-ABD1 formed 3.7 <br /> mol methylated capped ends mol protein a <br /> 5-min reaction. The extent methylation did increase with <br /> longer times incubation (up 20 min); thus, standard <br /> assay measured yield product reaction rate. <br /> The methyltransferase active broad pH range (6.0 <br /> to 9.5) Tris buffer. MgCl2 strongly inhibited activity a <br /> concentration-dependent fashion; methylation reduced by <br /> an order magnitude 1 mM magnesium. (Similar inhibi- <br /> tion magnesium noted vaccinia virus cap meth- <br /> yltransferase [16, 27].) The stimulatory effect salt His- <br /> ABD1 methyltransferase (optimal 40 mM NaCl) was <br /> virtually identical observed methyltransferase <br /> activity isolated yeast extract (data shown). <br />    The extent methylation varied AdoMet concentra- <br /> tion (Fig. 6A). An apparent K0.5 2 mM AdoMet calcu- <br /> lated double-reciprocal plot data. L-AdoHcy was <br /> a potent inhibitor, D-AdoHcy far effective <br /> (Fig. 6B). S-Adenosyl-L-ethionine (L-AdoEt), analog of <br /> AdoMet, inhibited methyltransferase activity                           FIG. 7. Analysis methyltransferase reaction product. A methyltrans- <br /> potent L-AdoHcy (Fig. 6B). (AdoEt shown previously                           ferase reaction mixture (70 ml) containing 119 fmol cap-labeled poly(A), 50 <br />                                                                                       mM AdoMet, 12.6 pmol His-ABD1 (glycerol fraction) incubated at <br /> to inhibit cap methylation reovirus mRNA synthesis                             378C 5 min heated 958C 6 min. Aliquots (22 ml) were <br /> [12].)                                                                                distributed new tubes treated follows. Two samples were <br />    Identification methyltransferase reaction product.                          adjusted 50 mM sodium acetate (pH 5.5). One tubes was <br /> Additional characterization reaction product per-                          incubated 60 min 378C addition, incu- <br />                                                                                       bated 60 min 378C 5 mg nuclease P1 (Boehringer). The sample <br /> formed different analytical reagents. In ex-                       adjusted 5 mM MgCl2 incubated 60 min 378C 7.4 mg of <br /> periments, RNA methylation reaction performed                              NPPase Crotalus adamanteus (type II; Sigma). Aliquots (10 ml) each <br /> saturating levels His-ABD1. The product analyzed di-                           sample distributed new tubes, adjusted 50 mM Tris HCl (pH 8.0), and <br /> rectly (i.e., nuclease digestion) TLC polyethyl-                        incubated 30 min 378C (lanes 1) (lanes 2) addition 1 <br />                                                                                       U alkaline phosphatase (Alk-Pase; calf intestine; Boehringer). The <br /> eneimine cellulose plates developed 0.2 M ammonium                               samples analyzed TLC polyethyleneimine-cellulose plates developed <br /> sulfate. All 32P-labeled material remained origin, i.e.,                   0.2 M ammonium sulfate. An autoradiogram TLC plate shown. The <br /> in polynucleotide form (Fig. 7). The product refractory                        positions GMP, m7GMP, GpppA, m7GpppA, Pi indicated. <br />  4172     MAO ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br /> phosphatase (results shown). The His-ABD1 reaction               (a positive-strand alphavirus), <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> nsP1 <br /> product identical respects methylated capped           protein (the virus-encoded cap methyltransferase) results in <br /> poly(A) synthesized vaccinia virus capping enzyme         acquisition ability grow conditions methio- <br /> analyzed parallel (27). These results confirmed ABD1         deprivation correlated biochemically en- <br /> catalyzed methylation N-7 position cap             hanced affinity mutant methyltransferase AdoMet <br /> guanosine.                                                           (37). <br />    ABD1 essential gene. One copy ABD1 gene            S. cerevisiae obvious choice genetic stud- <br /> Ura2 diploid yeast deleted replaced URA3 gene         ies capping role cellular RNA metabolism. Mu- <br /> flanked direct repeats hisG sequence (2). Inability         tational analysis CEG1 gene encoding RNA guanylyl- <br /> recover Ura1 haploids sporulation indicated ABD1          transferase established cap formation essential for <br /> was essential cell growth. A Ura2 diploid strain lost       yeast cell growth (11, 39, 40, 45). The present study reports the <br /> the URA3 gene recombination hisG repeats              identification cellular gene encoding cap methyl- <br /> was selected growth 5-FOA. Into strain (YBS5)          transferase. The ABD1 gene singled likely candi- <br /> introduced centromeric plasmid marked URA3 con-           date gene encoding yeast cap methyltransferase the <br /> taining ABD1 gene control natural pro-          basis sequence similarity catalytic domain the <br /> moter. After sporulation, selected Ura1 haploid prog-         vaccinia virus cap methyltransferase. That ABD1 gene <br /> eny grow 5-FOA. We obtained strain         product RNA (guanine-7-)methyltransferase es- <br /> YBS6, deletion chromosomal ABD1 locus             tablished expressing ABD1 protein bacteria, purifying <br /> and viability contingent maintenance             protein homogeneity, characterizing cap meth- <br /> extrachromosomal ABD1 allele. That inability grow          yltransferase activity intrinsic recombinant ABD1. Most im- <br /> 5-FOA reflected requirement ABD1 demonstrated           portant, physical biochemical properties recombi- <br /> the plasmid shuffle technique (47). A centromeric plasmid            nant ABD1 methyltransferase indistinguishable (by all <br /> marked TRP1 containing ABD1 introduced             criteria tested) cap methyltransferase iso- <br /> YBS6 cells. The Trp1 Ura1 transformants plated          lated partially purified whole-cell yeast extracts. We <br /> medium containing 5-FOA select retention           infer results ABD1 enzyme responsible <br /> ABD1 gene URA3 plasmid. Whereas cells containing              cap methylation yeast cells. <br /> the TRP1 plasmid ABD1 gene incapable                The sequence 436-amino-acid ABD1 protein well <br /> growth 5-FOA, cells bearing ABD1/TRP1 plasmid grew            conserved respect methyltransferase catalytic do- <br /> readily (results shown). Thus, yeast cap methyltrans-        main vaccinia virus capping enzyme homologous <br /> ferase encoded essential gene.                              regions capping enzymes Shope fibroma virus (49) <br />                                                                      African swine fever virus (35). The vaccinia virus cap <br />                          DISCUSSION                                  methyltransferase domain heterodimeric protein consisting <br />                                                                      C-terminal region D1 subunit (the catalytic site) <br />    Although studies suggest cap plays role        complexed 33-kDa D12 subunit (6, 16, 26). The D12 <br /> RNA processing (8, 20, 23, 24, 33, 34), transport (14, 48),      subunit, methyltransferase activity itself, stim- <br /> translation (1, 3, 32), definitive genetic       ulates methyltransferase activity D1 protein a <br /> tests cap function vivo. The direct approach       factor 50 100 (16, 26). We unable obvious <br /> problem entails manipulation genes control cap           sequence similarity ABD1 stimulatory D12 <br /> formation. Identifying genes obvious prerequisite.       subunit vaccinia virus methyltransferase. The specific <br />    Viral genes encoding capping methylating activities       activity recombinant His-ABD1 methylation capped <br /> have known time, viral systems,         poly(A) slightly lower observed the <br /> been powerful defining biochemistry capping,          heterodimeric vaccinia virus methyltransferase domain (also <br /> not useful genetic analysis. Vaccinia virus, DNA       His tagged) assay procedures (26); thus, ABD1 <br /> virus, genetically tractable principle, multifunc-     suffice cap methylation yeast cells ne- <br /> tional nature vaccinia virus capping enzyme,            cessity stimulatory factor. Given native size the <br /> serves transcription factor RNA-processing        cap methyltransferase yeast extract identical that <br /> enzyme (44, 52), makes difficult correlate vivo muta-       recombinant ABD1, consider likely yeast <br /> tional effects (i.e., temperature-sensitive replication)      methyltransferase ABD1 monomer. The existence an- <br /> specific biochemical defect (4). Many RNA viruses encode             cillary factors regulate ABD1 methyltransferase <br /> their capping methylating enzymes, viruses         way excluded, factors, present, ap- <br /> (especially negative-strand viruses) easily engi-        pear basis sedimentation analysis tightly <br /> neered genetically. Nevertheless, RNA viruses provided          associated methyltransferase purification from <br /> insights role cap vivo. For example, evidence     yeast cells. The size ABD1 methyltransferase (50.3 kDa) <br /> that vesicular stomatitis virus (a negative-strand rhabdovirus)      similar native size 56 kDa estimated gel filtration <br /> encodes cap-methylating enzyme(s) stems             sucrose gradient sedimentation cap methyltrans- <br /> characterization mutant virus synthesizes mRNA        ferase HeLa cells (9). <br /> capped unmethylated 59 ends (18). Such RNAs stable              The finding ABD1 gene required yeast growth <br /> but untranslatable nonpermissive infection           genetic evidence cap methyltransferase (and, <br /> vesicular stomatitis virus mutant strain. Replication ve-     inference, cap methyl group) plays essential role in <br /> sicular stomatitis virus mutant rescued coinfection       cellular function. It intriguing, then, ABD1 identi- <br /> vaccinia virus, allows methylation vesicular sto-       fied initially virtue mutational synergism BEM1, a <br /> matitis virus transcripts trans vaccinia virus-encoded     yeast gene involved bud emergence (7). The BEM1 protein <br /> methyltransferase (17). This elegant study provides          includes SH3 domains, facilitate protein-pro- <br /> the best evidence date cap methyl group required      tein interactions (5). It suggested BEM1 plays a <br /> for translation vivo. Genetic evidence essential role   role positioning assembling proteins sites bud <br /> cap methylation emerged studies Sindbis virus       formation; indeed, BEM1 binds directly CDC24 protein, <br />  VOL. 15, 1995                                                                                   YEAST mRNA CAP METHYLTRANSFERASE                                  4173 <br />  <br />  <br /> which guanine nucleotide exchange factor activity                                  subunit functions, subcellular localization. J. Biol. Chem. 262:1989&#226;&#8364;&#8220; <br /> which required bud formation (54). Bud emergence                                1995. <br />                                                                                      22.   Kadowaki, T., D. Goldfarb, L. M. Spitz, A. M. Tartakoff, M. Ohno. 1993. <br /> bud site selection regulated multiple gene products,                                Regulation RNA processing transport nuclear guanine nucle- <br /> many resemble components GTPase signaling                                      otide release protein members Ras superfamily. EMBO J. 12: <br /> pathways. How ABD1 fits picture clear us,                          2929&#226;&#8364;&#8220;2937. <br /> nor immediately obvious cap methyltransferase                        23.   Konarska, M. M., R. A. Padgett, P. A. Sharp. 1984. Recognition cap <br />                                                                                            structure splicing vitro mRNA precursors. Cell 38:731&#226;&#8364;&#8220;736. <br /> be relevant GTPase regulation cell polarity. Although                       24.   Krainer, A. R., T. Maniatis, B. Ruskin, M. R. Green. 1984. Normal and <br /> cannot excluded ABD1 functions                                mutant beta-globin pre-mRNAs faithfully efficiently spliced vitro. <br /> its activity cap methylation, note investigators                          Cell 36:993&#226;&#8364;&#8220;1005. <br /> have established connection yeast RNA processing,                          25.   Lin, R. J., A. J. Newman, S. Cheng, J. Abelson. 1985. Yeast mRNA <br />                                                                                            splicing vitro. J. Biol. Chem. 260:14780&#226;&#8364;&#8220;14792. <br /> RNA transport, genes encode putative guanine nu-                            26.   Mao, X., S. Shuman. 1994. Intrinsic RNA (guanine-7-) methyltrans- <br /> cleotide-signaling proteins (22, 38). More extensive genetic                               ferase activity vaccinia virus capping enzyme D1 subunit stimulated <br /> and biochemical studies ABD1 needed understand                               D12 subunit: identification amino acid residues D1 protein <br /> intersection superficially unrelated cellular processes.                          required subunit association methyl group transfer. J. Biol. Chem. <br />                                                                                            269:24472&#226;&#8364;&#8220;24479. <br />                                    REFERENCES                                        27.   Mao, X., S. Shuman. Unpublished results. <br />  1. Adams, B. L., M. Morgan, S. Muthukrishnan, S. M. Hecht, A. J.                28.   Martin, S. A., B. Moss. 1976. mRNA guanylyltransferase mRNA <br />     Shatkin. 1978. The effect cap analogs reovirus mRNA binding wheat             (guanine-7-) methyltransferase vaccinia virions; donor acceptor <br />     germ ribosomes. J. Biol. Chem. 253:2589&#226;&#8364;&#8220;2595.                                          substrate specificities. J. Biol. Chem. 251:7313&#226;&#8364;&#8220;7321. <br />  2. Alani, E., L. Cao, N. Kleckner. 1987. A method gene disruption      29.   Mizumoto, K., F. Lipmann. 1979. Transmethylation transguanyly- <br />     allows repeated use URA3 selection construction multiply dis-             lation 59-RNA capping isolated rat liver. Proc. Natl. Acad. <br />     rupted yeast strains. Genetics 116:541&#226;&#8364;&#8220;545.                                            Sci. USA 76:4961&#226;&#8364;&#8220;4965. <br />  3. Banerjee, A. K. 1980. 59-terminal cap structure eukaryotic messenger          30.   Mizumoto, K., Y. Kaziro. 1987. Messenger RNA capping enzyme from <br />     ribonucleic acids. Microbiol. Rev. 44:175&#226;&#8364;&#8220;205.                                         eukaryotic cells. Prog. Nucleic Acid Res. Mol. Biol. 34:1&#226;&#8364;&#8220;28. <br />  4. Carpenter, M. S., A. M. DeLange. 1991. A temperature-sensitive lesion        31.   Moss, B., A. Gershowitz, C. Wei, R. Boone. 1976. Formation the <br />     small subunit vaccinia virus-encoded mRNA capping enzyme                 guanylylated methylated 59-terminus vaccinia virus mRNA. Virology <br />     causes defect viral telomere resolution. J. Virol. 65:4042&#226;&#8364;&#8220;4050.                  72:341&#226;&#8364;&#8220;351. <br />  5. Chenevert, J., K. Corrado, A. Bender, J. Pringle, I. Herskowitz. 1992. A     32.   Muthukrishnan, S., G. W. Both, Y. Furuichi, A. J. Shatkin. 1975. <br />     yeast gene BEM1 necessary cell polarization product contains             59-terminal 7-methylguanosine eukaryotic mRNA required transla- <br />     SH3 domains. Nature (London) 356:77&#226;&#8364;&#8220;79.                                                tion. Nature (London) 255:33&#226;&#8364;&#8220;37. <br />  6. Cong, P., S. Shuman. 1992. Methyltransferase subunit association         33.   Ohno, M., H. Sakamoto, Y. Shimura. 1987. Preferential excision the <br />     domains vaccinia virus mRNA capping enzyme. J. Biol. Chem. 267:16424&#226;&#8364;&#8220;               59 proximal intron mRNA precursors introns mediated by <br />     16429.                                                                                 cap structure. Proc. Natl. Acad. Sci. USA 84:5187&#226;&#8364;&#8220;5191. <br />  7. Corrado, K., J. R. Pringle. 1993. Molecular analysis ABD1 gene        34.   Patzelt, E., E. Thalman, K. Harmuth, D. Blaas, E. Keuchler. 1987. <br />     Saccharomyces cerevisiae, gene displays mutational synergism            Assembly pre-mRNA splicing complex cap dependent. Nucleic Acids <br />     bud formation gene BEM1. Accession no. L12000. GenBank.                            Res. 15:1387&#226;&#8364;&#8220;1399. <br />  8. Edery, I., N. Sonenberg. 1985. Cap-dependent RNA splicing HeLa          35.   Pena, L., R. Yanez, Y. Revilla, E. Vinuela, M. L. Salas. 1992. African <br />     nuclear extract. Proc. Natl. Acad. Sci. USA 82:7590&#226;&#8364;&#8220;7594.                              swine fever virus guanylyltransferase. Virology 193:319&#226;&#8364;&#8220;328. <br />  9. Ensinger, M., B. Moss. 1976. Modification 59 terminus mRNA         36.   Rothstein, R. 1991. Targeting, disruption, replacement, allele rescue: <br />     RNA (guanine-7-)methyltransferase HeLa cells. J. Biol. Chem.                integrative DNA transformation yeast. Methods Enzymol. 194:281&#226;&#8364;&#8220;301. <br />     251:5283&#226;&#8364;&#8220;5291.                                                                   37.   Scheidel, L. M., R. K. Durbin, V. Stollar. 1989. SVLM21, Sindbis virus <br /> 10. Feldmann, H., et al. 1994. Complete DNA sequence yeast chromosome II.               mutant resistant methionine deprivation, encodes altered methyltrans- <br />     EMBO J. 13:5795&#226;&#8364;&#8220;5809.                                                                  ferase. Virology 173:408&#226;&#8364;&#8220;414. <br /> 11. Fresco, L. D., S. Buratowski. 1994. Active site mRNA capping          38.   Schlenstedt, G., C. Saavedra, J. D. Loeb, C. N. Cole, P. A. Silver. 1995. <br />     enzyme guanylyltransferase Saccharomyces cerevisiae: similarity            The GTP-bound form yeast Ran/TC4 homologue blocks nuclear <br />     nucleotidyl attachment motif DNA RNA ligases. Proc. Natl. Acad.                 protein import appearance poly(A)1 RNA cytoplasm. Proc. <br />     Sci. USA 91:6624&#226;&#8364;&#8220;6628.                                                                 Natl. Acad. Sci. USA 92:225&#226;&#8364;&#8220;229. <br /> 12. Furuichi, Y., M. A. Morgan, A. J. Shatkin. 1979. Synthesis transla-      39.   Schwer, B., S. Shuman. 1994. Mutational analysis yeast mRNA <br />     tion mRNA containing 59-terminal 7-ethylguanosine cap. J. Biol. Chem.               capping enzyme. Proc. Natl. Acad. Sci. USA 91:4328&#226;&#8364;&#8220;4332. <br />     254:6732&#226;&#8364;&#8220;6738.                                                                   40.   Shibagaki, Y., N. Itoh, H. Yamada, S. Hagata, K. Mizumoto. 1992. <br /> 13. Furuichi, Y., S. Muthukrishnan, J. Tomasz, A. J. Shatkin. 1976. Mech-              mRNA capping enzyme: isolation characterization gene encoding <br />     anism formation reovirus mRNA 5-terminal blocked methylated                  mRNA guanylyltransferase subunit Saccharomyces cerevisiae. J. Biol. <br />     sequence m7GpppGmpC. J. Biol. Chem. 251:5043&#226;&#8364;&#8220;5053.                                     Chem. 267:9521&#226;&#8364;&#8220;9528. <br /> 14. Hamm, J., I. Mattaj. 1990. Monomethylated cap structures facilitate          41.   Shuman, S. 1989. Functional domains vaccinia virus mRNA capping <br />     RNA export nucleus. Cell 63:109&#226;&#8364;&#8220;118.                                          enzyme: analysis limited tryptic digestion. J. Biol. Chem. 264:9690&#226;&#8364;&#8220; <br /> 15. Higman, M. A., N. Bourgeois, E. G. Niles. 1992. The vaccinia virus                 9695. <br />     mRNA (guanine-N7-) methyltransferase requires subunits               42.   Shuman, S. 1990. Catalytic activity vaccinia mRNA capping enzyme sub- <br />     mRNA capping enzyme activity. J. Biol. Chem. 267:16430&#226;&#8364;&#8220;16437.                      units coexpressed Escherichia coli. J. Biol. Chem. 265:11960&#226;&#8364;&#8220;11966. <br /> 16. Higman, M. A., L. A. Christen, E. G. Niles. 1994. The mRNA (guanine-         43.   Shuman, S. 1995. Capping enzyme eukaryotic mRNA synthesis. Prog. <br />     7-) methyltransferase domain vaccinia virus mRNA capping enzyme:                Nucleic Acid Res. Mol. Biol. 50:101&#226;&#8364;&#8220;129. <br />     expression Escherichia coli structural kinetic comparison      44.   Shuman, S., S. S. Broyles, B. Moss. 1987. Purification character- <br />     intact capping enzyme. J. Biol. Chem. 269:14974&#226;&#8364;&#8220;14981.                                 ization transcription termination factor vaccinia virions. J. Biol. <br /> 17. Horikami, S. M., F. De Ferra, S. A. Moyer. 1984. Characterization           Chem. 262:12372&#226;&#8364;&#8220;12380. <br />     infections permissive nonpermissive cells host range mutants        45.   Shuman, S., Y. Liu, B. Schwer. 1994. Covalent catalysis nucleotidyl <br />     vesicular stomatitis virus defective RNA methylation. Virology 138:1&#226;&#8364;&#8220;               transfer reactions: essential motifs S. cerevisiae RNA capping enzyme <br />     15.                                                                                    conserved Schizosaccharomyces pombe viral capping enzymes <br /> 18. Horikami, S. M., S. A. Moyer. 1982. Host range mutants vesicular                polynucleotide ligases. Proc. Natl. Acad. Sci. USA 91:12046&#226;&#8364;&#8220; <br />    <br /> </body></html>