<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Feb. 1996, p. 669&#226;&#8364;&#8220;676                                                                              Vol. 16, No. 2 <br /> 0270-7306/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />                        Paf1p, RNA Polymerase II-Associated Factor <br />                          Saccharomyces cerevisiae, May Have both <br />                          Positive Negative Roles Transcription <br />                     XIAOMING SHI,1 ANN FINKELSTEIN,2 ALEXANDER J. WOLF,1 PAUL A. WADE,1 <br />                               ZACHARY F. BURTON,2 AND JUDITH A. JAEHNING1* <br />                           Department Biochemistry, Biophysics Genetics, University Colorado Health <br />                              Sciences Center, Denver, Colorado 80262,1 Department Biochemistry, <br />                                       Michigan State University, East Lansing, Michigan 488242 <br />                          Received 22 June 1995/Returned modification 25 July 1995/Accepted 21 November 1995 <br />  <br />              Regulated transcription initiation requires, addition RNA polymerase II general transcription <br />           factors, accessory factors termed mediators adapters. We used affinity chromatography identify a <br />           collection factors associate Saccharomyces cerevisiae RNA polymerase II (P. A. Wade, W. Werel, <br />           R. C. Fentzke, N. E. Thompson, J. F. Leykam, R. R. Burgess, J. A. Jaehning, Z. F. Burton, submitted for <br />           publication). Here report identification characterization gene encoding factors, PAF1 <br />           (for RNA polymerase-associated factor 1). PAF1 encodes novel, highly charged protein 445 amino acids. <br />           Disruption PAF1 S. cerevisiae leads pleiotropic phenotypic traits, including slow growth, temperature <br />           sensitivity, abnormal cell morphology. Consistent possible role transcription, Paf1p localized <br />           nucleus. By comparing abundances yeast transcripts isogenic wild-type paf1 mutant <br />           strains, identified genes expression affected PAF1. In particular, disruption PAF1 <br />           decreases induction galactose-regulated genes three- fivefold. In contrast, transcript level of <br />           MAK16, essential gene involved cell cycle regulation, greatly increased paf1 mutant strain. Paf1p <br />           required positive negative regulation subsets yeast genes. Like Paf1p, the <br />           GAL11 gene product associated RNA polymerase II required regulated expression of <br />           yeast genes including controlled galactose. We gal11 paf1 double mutant has <br />           severe growth defect single mutants, indicating proteins may <br />           function parallel pathways communicate signals regulatory factors RNA polymerase II. <br />  <br />  <br />    Promoter-specific transcription initiation protein-cod-                unidentified polypeptides (32, 33). The SRB genes ini- <br /> ing genes eukaryotic systems requires concerted action                  tially identified virtue ability suppress cold- <br /> of complex array factors, including general factors                   sensitive phenotypes conferred truncation mutations the <br /> (TFIIA, -B, -E, -F, -H TBP) RNA polymerase II                      RNA polymerase II CTD (40). All SRB genes essential <br /> (for reviews, references 11 72). These factors                     normal growth yeast cells, suggesting importance in <br /> sufficient basal transcription vitro. Transcription initia-             transcriptional regulation (33). Previous biochemical ge- <br /> tion subject regulation promoter-specific activators             netic evidence implicates GAL11 SUG1 transcrip- <br /> and repressors (57). In vitro studies using reconstituted sys-                 tional coactivators (17, 24, 39, 46, 53, 54). A class of <br /> tems revealed additional class factors, coacti-                accessory factors represented yeast SWI-SNF com- <br /> vators adapters, required mediate communication                      plex, affects transcription counteracting repressive <br /> between promoter-specific regulatory factors general                   effects imposed chromatin structure (41, 67). <br /> transcription machinery (8, 32, 33).                                              <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> identify additional factors interact RNA poly- <br />    Transcriptional coactivators divided sub-               merase II, used monoclonal antibody directed the <br /> classes, probably corresponding different pathways com-                  RNA polymerase II CTD immobilize enzyme asso- <br /> munication regulatory factors general ma-                      ciated factors transcriptionally active yeast whole-cell <br /> chinery. The TFIID complex, mediates signals                        extract (60). The collection factors eluted immo- <br /> some activators, consists TBP TBP-associated factors                    bilized RNA polymerase included TFIIB, TFIIS, three <br /> (8). Individual TBP-associated factors shown                      subunits TFIIF factors CTD- <br /> make direct contacts different classes activators (8). A <br />                                                                                interacting mediator complex, including Srb proteins, <br /> second subclass coactivator Saccharomyces cerevisiae is <br />                                                                                Gal11p, Sug1p (32, 33, 60). We identified additional <br /> the mediator complex associated C-terminal <br />                                                                                factors interesting connections regulation gene <br /> domain (CTD) largest subunit RNA polymerase II <br />                                                                                expression (48, 60). Here report analysis these <br /> (32, 33). Independently isolated different laboratories, <br /> the mediator consists products SRB genes (sup-                   factors, Paf1p (RNA polymerase-associated factor 1). Using <br /> pressor RNA polymerase B), Gal11p, Sug1p, additional                    peptide sequence derived RNA polymerase- <br />                                                                                associated proteins, cloned PAF1 gene found <br />                                                                                disruption leads pleiotropic phenotypic traits includ- <br />    * Corresponding author. Mailing address: Department Biochem-             ing slow growth, temperature sensitivity, enlarged cells. <br /> istry, Biophysics Genetics, University Colorado Health Sciences         Consistent role transcription, localized <br /> Center, Box B121, 4200 East Ninth Ave., Denver, CO 80262. Phone:               Paf1p yeast cell nucleus. On basis analysis of <br /> (303) 270-3004. Fax: (303) 270-3326.                                           expression yeast genes PAF1 mutant cells, we <br />  <br />                                                                          669 <br />  670       SHI ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br /> postulate Paf1p act cofactor important tran-                        2708C. The pellets resuspended solution (48C) containing 18% Ficoll, <br /> scriptional activation repression diverse promoters.                         50 mM Tris-acetate (pH 7.9), 0.1 M potassium acetate, 1 mM PMSF. Nuclei <br />                                                                                       isolated centrifuging suspension 3,000 3 g 5 min. The super- <br /> The combination mutations PAF1 GAL11 leads                               natants decanted recentrifuged uniform white color. <br /> severe growth defects consistent possibility                      Nuclei recovered supernatant centrifugation 25,000 3 g for <br /> factors involved duplication essential communication                        1 h. The nuclear pellet resuspended TNE (10 mM Tris-HCl [pH 7.9], 1 <br /> functions regulatory factors RNA polymerase II.                           mM EDTA, 0.1 M NaCl) 0.3 ml g cells. To suspension, 4 M <br />                                                                                       ammonium sulfate (pH 7.9) added, stirring, final concentration of <br />                                                                                       0.9 M. The suspension stirred additional 30 min 48C. Insoluble <br />                     MATERIALS AND METHODS                                             protein removed centrifugation 35,000 rpm SW-40 rotor (Beck- <br />                                                                                       man) 48C. Supernatant protein (soluble nuclear fraction) precipitated by <br />    Strains media. The S. cerevisiae strains used study YJJ453        addition dry ammonium sulfate (0.35 g ml), collected centrifuga- <br /> (MATa/MATa leu2D1/leu2D1 his3D200/his3D200 ura3-52/ura3-52), YJJ564                   tion, resuspended minimal volume TNE, frozen, stored 2708C. <br /> (MATa leu2D1 his3D200 ura3-52 gal11D::LEU2), YJJ662 (MATa leu2D1                      Protein concentrations measured using reagents Bio-Rad. For <br /> his3D200 ura3-52), YJJ664 (MATa leu2D1 ura3-52 his3D200 paf1D::HIS3),                 Western blot (immunoblot) analysis, 300 mg protein spheroplasts, 200 mg <br /> YJJ699 (MATa leu2D1 gal11D::LEU2 ura3-52 his3D200 paf1D::HIS3), YJJ575                protein cytoplasmic fraction, 50 mg protein nuclear <br /> (MATa leu2D1 his3D200 ura3-52), YJJ577 (MATa leu2D1 his3D200 ura3-52              fraction separated sodium dodecyl sulfate-polyacrylamide gel electro- <br /> paf1D::HIS3). These strains derived homozygous diploid              phoresis (SDS-PAGE) (10% polyacrylamide) (22). Affinity-purified anti-Paf1p <br /> YJJ453 (58) isogenic. Yeast strains grown YPD            antibody used detect immobilized Paf1p (22). <br /> synthetic medium prepared standard methods (21).                                      Genomic DNA analysis. Yeast strain YJJ453 grown YPD medium to <br />    Isolation PAF1 gene. Isolation Paf1p sequencing peptides     mid-log phase, DNA isolated 1.5 ml glass bead method (16). <br /> done described previously (60). Two peptide sequences obtained: QEY-          Genomic DNA digested overnight appropriate restriction enzymes, <br /> IAPI AADTPETS. A BLAST search (1) indicated peptides               electrophoresed 1% agarose gel, transferred nitrocellulose membrane, <br /> were perfect match open reading frame chromosome II. The following         cross-linked UV light, probed PAF1 PCR product prepared by <br /> PCR primers designed amplify PAF1 coding sequence yeast          random priming (47). <br /> genomic DNA: 59ATCGCGGATCCATGTCCAAAAAACAGGAATATATTG                                      Yeast mRNA analysis. Total yeast RNA isolated described previously <br /> C39 59ATCGCGGATCCCTATTCTTCTTGTAAAGTTTCC39. Both prim-                             (16). For galactose induction experiment, cells grown synthetic <br /> ers flanked BamHI site (underlined) extra bases facilitate cloning.   glucose dropout medium mid-log phase, pelleted, resuspended syn- <br />    Deletion disruption PAF1. The one-step gene replacement method (44)         thetic galactose dropout medium (2% galactose). RNA samples quanti- <br /> was used disrupt PAF1. A novel PCR-based strategy used make               tated measuring optical density 260 nm, equal amounts RNA were <br /> disruption construct entire coding sequence PAF1 deleted. First,          run 1% agarose-formaldehyde gels, RNA blots prepared the <br /> flanking sequences PAF1 amplified PCR. The 59-flanking sequence         standard methods (47). All probes prepared random priming. The <br /> PAF1 amplified primers P1 (59CTTAGCACAACTGAATTCGA                        probes SalI fragment pJJ58 (to detect GAL7 GAL10 <br /> AAGG39) P2 (59ATCGCGGATCCGTGACCCTTTATACTATTATGAG                                  transcripts [2]), EcoRI-BamHI fragment pJJ157 (58) (to detect ACT1 <br /> C39). The 39-flanking sequence PAF1 amplified primers P3 (59CAT-          transcripts), PCR-amplified FUN38 pBluescript KS1 subclone 159 (6) by <br /> ACCCGCTCGAGGAATATTGAGAAGTGAAGGGAGATTG39) P4 (59AT                                 using universal reverse sequencing primers, PCR-amplified MAK16 <br /> ACGAATGATGTTAATGGAGACTCCAGGATTGTCGACT39). BamHI                                   pBluescript KS1 subclone 162 (6) using reverse universal <br /> XhoI restriction sites added 59 ends P2 P3 (underlined),           sequencing primers. RNA blots quantitated PhosphorImager analysis. <br /> respectively. The PCR products gel purified, cut BamHI XhoI,           b-Galactosidase assays. Reporter plasmids used b-galactosidase assays <br /> ligated XhoI-BamHI fragment HIS3 Sc2676 (52). The ligated              previously described (19, 20). pLGSD5 contains 365-bp GAL10 up- <br /> products amplified PCR primers P1 P4, expected PCR           stream activating sequence (UAS) fragment; pSV14 GAL4 17-mer <br /> product (3 kb) used transform diploid yeast cells (YJJ453) (21). The           binding site. These 2mm plasmids indicated UAS elements inserted 59 <br /> diploid cells sporulated, tetrads dissected (21). Deletion       CYC1 TATA region transcription initiation site. Plasmids trans- <br /> entire coding sequence PAF1 confirmed Southern blot analysis (47).          formed strains YJJ662 YJJ664 standard methods (21). Growth <br />    Preparation anti-Paf1p antibody. Paf1p produced recombinant            induction transformants described Northern (RNA) <br /> protein PCR amplification PAF1 coding sequence total yeast             blot analyses uracil omitted selection plasmids. The <br /> DNA using 59TCCATATGTCCAAAAAACAGGAATATA39 59GCCG                               induction galactose allowed proceed 5 h. Extract preparation and <br /> GATCCTTATTCTTCTTGTAAAGTTTCC39 primers. The underlined se-                          b-galactosidase assays performed described previously (38). <br /> quences NdeI BamHI sites included subsequent cloning steps;          Construction gal11 paf1 double mutant. Strain YJJ564 crossed to <br /> PCR product cleaved restriction enzymes, gel purified, ligated     YJJ577, diploids selected plates lacking uracil leucine. The <br /> to pET11a (Novagen) cut enzymes (47). The resulting plasmid         diploid sporulated, tetrads dissected standard methods (21). <br /> used transform Escherichia coli BL21(DE3), Paf1p produced                 In vitro transcription. The preparation yeast whole-cell transcription ex- <br /> partially purified transformed strain previously described (62). Two      tracts transcription reaction conditions described previously (69). <br /> hundred micrograms insoluble suspension Paf1p mixed              G-less cassette transcription templates lacking (pJJ460) containing UASHSE <br /> equal volume Titer Max Adjuvant (Vaxcel Inc.) used raise antibodies         (pJJ468) used (69). Each reaction mixture contained 5 mg WCE protein <br /> in rabbits (22). The resulting antibody affinity purified using Paf1p puri-    ml 10 mg template ml. Transcription reactions per- <br /> fied chromatography Mono Q (Pharmacia HR5/5) (63). Purified Paf1p           formed 26, 33, 378C. Labeled transcripts isolated, resolved, visual- <br /> coupled Affi-Gel 10 (Bio-Rad) according manufacturer&#226;&#8364;&#8482;s instructions.        ized, quantitated described previously (69). <br /> The anti-Paf1p antiserum applied column, washed solution <br /> containing 20 mM HEPES (N-2-hydroxylethylpiperazine-N9-2-ethanesulfonic <br /> acid) (pH 7.9), 150 mM NaCl, 1 mM EDTA eluted 0.1 M glycine                                                RESULTS <br /> (pH 2.5). One-milliliter fractions collected tubes containing 200 ml of <br /> 1.0 M HEPES, pH 7.9. Peak fractions determined spectrophotometrically,              Cloning sequence analysis yeast PAF1 gene. We <br /> pooled, dialyzed column wash buffer. <br />    Subcellular localization Paf1p. Yeast strain YJJ575 grown YPD <br />                                                                                       isolated RNA polymerase-associated proteins using a <br /> medium (21) 308C A600 1. Cells harvested centrifugation,          monoclonal antibody directed CTD capture as- <br /> washed water, resuspended 0.1 M Tris-SO4 (pH 9.3) containing 10           sociated factors transcriptionally active whole-cell ex- <br /> mM dithiothreitol (0.1 0.3 g cells ml buffer), gently shaken      tract (60). Two peptide sequences obtained the <br /> room temperature 20 min. Cells collected, washed spheroplast- <br /> ing buffer (1.2 M sorbitol, 20 mM potassium phosphate; pH 7.4), resus- <br />                                                                                       RNA polymerase-associated protein apparent molec- <br /> pended spheroplasting buffer 0.1 g cells ml. Yeast lytic enzyme (ICN     ular mass 65 kDa: QEYIAPI AADTPETS (60). The <br /> Biochemicals) added (7.5 mg enzyme g cells), resulting          genes encoding peptides exactly match open read- <br /> suspension shaken gently 308C 80 90% yeast cells          ing frame identified yeast genome sequencing project <br /> spheroplasts. The progress spheroplasting reaction monitored by <br /> diluting aliquots suspension 9 volumes water checking lysis <br />                                                                                       (Fig. 1). This sequence encodes novel gene resides on <br /> by light microscopy. The resulting spheroplasts washed times            right arm chromosome II known RIF1 and <br /> spheroplasting buffer resuspended MIB buffer (0.6 M mannitol, 20 mM            SNF5 genes. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> sequence PAF1 (polymerase- <br /> HEPES [pH 7.4], 1 mM phenylmethylsulfonyl fluoride [PMSF]) 0.5 g cells          associated factor) gene, predict protein 445 amino <br /> per ml. Cells subsequently lysed precooled Dounce homogenizer with <br /> 20 strokes using B pestle diluted twofold MIB buffer. The <br />                                                                                       acids calculated molecular mass 49 kDa (Fig. 1), <br /> resulting suspension centrifuged 5 min 3,000 3 g (48C). Super-        lacking significant homology sequence current <br /> natants spin cytoplasmic fraction frozen stored     database. The protein sequence encoded PAF1 several <br />  VOL. 16, 1996                                                            Paf1p, A YEAST RNA POLYMERASE II-ASSOCIATED FACTOR                                     671 <br />  <br />  <br />  <br />  <br />                                                                                         FIG. 2. Paf1p nuclear protein. Cellular fractionation performed to <br />                                                                                      obtain cytoplasmic nuclear fractions described Materials Methods. <br />                                                                                      Protein samples (300 mg protein spheroplasts, 200 mg protein <br />                                                                                      cytoplasmic fraction, 50 mg protein nuclear fraction) <br />                                                                                      subjected SDS-PAGE (10% polyacrylamide) analyzed poly- <br />                                                                                      clonal antibodies Paf1p, Sup35p, TBP, described Materials and <br />    FIG. 1. Deduced amino acid sequence Paf1p. The PAF1 gene encoded            Methods. <br /> by open reading frame YBR2016 right arm chromosome II (25). The <br /> sequenced peptides used identify gene (QEYIAPI AADTPETS) are <br /> found positions 5 11 419 426. All charged amino acids (negative [D <br /> and E] positive [R K]) underlined. The putative PEST sequence            letion confirmed genomic DNA analysis (data not <br /> (amino acids 25 49) potential bipartite nuclear localization motif (amino   shown), diploid sporulated, tetrads dissected <br /> acids 119 141) underlined twice. <br />                                                                                      (21). At 308C, intact tetrads exhibited <span id='am-315' about='xsp:pattern' typeof='owl:Thing'>pattern</span> large <br />                                                                                      His2 small His1 colonies (Fig. 3B). Genomic DNA <br />                                                                                      analysis confirmed deletion PAF1 His1 spores not <br /> interesting features. In particular, Paf1p highly charged; 40%                    His2 spores (Fig. 3C). Complete rescue paf1 null <br /> of amino acids charged residues (D, E, R, K),                     mutant centromere plasmid containing PAF1 gene <br /> net charge pH 7 223. The carboxyl terminus protein                      demonstrated observed phenotypic traits result from <br /> is extremely acidic, glutamic aspartic acid represent-                      loss Paf1p function effects dis- <br /> ing 40 80 residues. This high charge density explain                      rupting flanking genes (data shown). Consistent the <br /> discrepancy calculated apparent molecular                            small size paf1 colonies, doubling time paf1 <br /> masses (49 versus 65 kDa). Paf1p contains potential bipartite                      mutant strain increased 3 4 h 2-h generation <br /> nuclear localization signal (Fig. 1, amino acids 119 141) (13)                    time isogenic wild-type strain (Fig. 4A). Disruption of <br /> and potential casein kinase phosphorylation sites (14).                      PAF1 resulted temperature-sensitive <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> 36 <br /> The amino terminus Paf1p contains potential PEST se-                            388C (Fig. 4B). Although paf1 cells cold sensi- <br /> quence (Fig. 1, amino acids 25 49) (43), signifies                    tive, grow slowly 14 258C cells the <br /> rapid turnover proteins vivo (43). Analysis yeast RNA                       isogenic PAF1 strain. The paf1 mutant cells abnormal <br /> and genomic DNA indicated PAF1 transcribed                            morphology; cells enlarged (two- threefold larger <br /> single RNA species 1.6 kb single-copy gene (data                       wild-type cells) rounded (Fig. 4C). <br /> shown).                                                                                 The expression levels yeast genes affected by <br />    Paf1p nuclear protein. We produced recombinant                             paf1 <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>mutation</span>. If Paf1p important transcription of <br /> form Paf1p, material used elicit antibodies.                      yeast genes, transcript abundance temperature- <br /> Affinity-purified antibodies used probe fractions                        sensitive null mutation altered subsets genes at <br /> from RNA polymerase II affinity chromatography (60). We                          permissive restrictive temperatures. We have <br /> confirmed Paf1p present anti-CTD eluate                          screened 20 yeast transcription units several <br /> not control column fractions (data shown). In addition,                       transcripts begin fulfill prediction. We analyzed <br /> this antibody failed detect cross-reacting material                        transcription patterns shift nonpermissive tem- <br /> correct size whole-cell extract paf1D strain (data                       perature. Cells grown permissive temperature, <br /> not shown). To determine intracellular location Paf1p,                        308C, shifted 388C, samples isolated <br /> we fractionated lysed spheroplasts cytoplasmic nu-                          RNA analysis. The paf1 cells 1 2 further <br /> clear fractions resolved proteins gel electrophoresis.                    doublings 388C, growth observed 5 h <br /> Using anti-Paf1p antibody probe immobilized pro-                          shift nonpermissive temperature. Most tran- <br /> teins, Paf1p detected nu-                         scripts monitored 5 h shift, including from <br /> clear fraction (Fig. 2). For controls cell fractionation,                    ACT1, EGD1, DED1, CDC73, SPT15, unaffected by <br /> we TBP, general transcription factor (72),                          paf1 mutation (data shown). We did observe two- to <br /> found exclusively nuclear fraction, Sup35p, known                       threefold-lower abundance transcript EGD2 <br /> to associated ribosomes (12), detected                       gene (49) paf1 strain 1 h shift the <br /> cytoplasmic fraction. These results indicate Paf1p                         nonpermissive temperature (data shown). Consistent with <br /> nuclear protein, consistent identification RNA                        results, differential display analysis (35), abundance of <br /> polymerase II-associated protein presence nuclear                       approximately 95% transcripts altered the <br /> localization signal.                                                                 paf1 mutant, abundance remaining 5% the <br />    Pleiotropic phenotypic traits paf1 deletion-disruption                     transcripts increased decreased significantly (48). <br /> mutant. To determine possible functions Paf1p, dis-                               The paf1 mutation impairs induction galactose-induc- <br /> rupted PAF1 gene homozygous diploid yeast strain,                           ible genes. The paf1 mutant strain exhibited slow growth all <br /> deleting entire coding sequence replacing                          carbon sources tested, including glycerol galactose. Since <br /> HIS3 marker (Fig. 3A; Materials Methods). The de-                            slow growth carbon sources observed for <br />  672      SHI ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Disruption PAF1 S. cerevisiae. (A) Schematic representation PCR-based disruption strategy. The 59- 39-flanking sequences PAF1 were <br /> amplified yeast genomic DNA PCR (Materials Methods). Restriction sites BamHI XhoI, absent flanking sequences, added primers <br /> P2 P3 facilitate ligation HIS3 BamHI-XhoI fragment (52). The resulting ligation product (PAF1 59-flanking sequence&#226;&#8364;&#8220;HIS3&#226;&#8364;&#8220;PAF1 39-flanking sequence) <br /> was amplified PCR primers P1 P4. The amplified fragment used transform diploid yeast strain YJJ453. (B) Tetrad analysis paf1 <br /> deletion-disruption mutant. The heterozygous paf1/PAF1 disruption strain sporulated, tetrads dissected. Six dissected asci shown. (C) Confirmation <br /> of paf1 disruption. Genomic DNA prepared spores tetrad, digested PstI, resolved 1% agarose gel. The hybridization probe <br /> was random-primed PCR product 39-flanking sequence PAF1, shown panel A. <br />  <br />  <br />  <br />  <br /> strains mutations genes important induction                      genes affected paf1 mutant; example, the <br /> galactose-regulated genes (29), analyzed effect                       4.8-kb transcript FUN38 gene, shown Fig. 5C. <br /> paf1 mutation galactose induction. Wild-type isogenic                        However, RNA level MAK16 gene dramati- <br /> paf1 mutant strains grown mid-log phase glu-                         cally higher paf1 mutant (five- eightfold) the <br /> cose-containing synthetic medium shifted galactose-                          wild-type strain (Fig. 5C). MAK16 essential gene S. <br /> containing medium, samples collected RNA anal-                         cerevisiae (66), amino acid sequence encoded by <br /> ysis (Fig. 5A). Although GAL7 GAL10 mRNAs                               MAK16 similar casein kinases (6). Although the <br /> induced paf1 mutant, transcript levels signif-                    exact function MAK16 known, gene probably <br /> icantly lower (three- fivefold) wild-type                      involved cell cycle control, temperature-sensitive <br /> strain (Fig. 5A). To confirm result alter-                     mutant MAK16 causes cells arrest G1 phase the <br /> ations transcription, introduced reporter plasmids con-                       restrictive temperature (65). In addition, mutations MAK16 <br /> taining UAS galactose-regulated promoter (UASGAL),                       result loss M1 double-stranded RNA virus-like par- <br /> into wild-type paf1 mutant strains. As shown Fig. 5B,                    ticles S. cerevisiae (66). <br /> b-galactosidase activity UASGAL-lacZ fusion re-                           Paf1p required transcription model promoter <br /> duced five- sixfold paf1 mutant, comparable                        vitro. To test possible involvement Paf1p transcrip- <br /> reduction seen Northern blot analysis. Although mutations                        tion vitro, whole-cell transcription extracts prepared in <br /> in genes affect induction galactose-regulated genes                     parallel wild-type isogenic paf1 mutant strains. As <br /> (29), phenotype conferred paf1 mutation                         shown Fig. 6, basal transcription CYC1 core pro- <br /> secondary effect slow growth. An isogenic cdc73 mutant                           moter (lacking UAS [UAS2]) G-less cassette template, <br /> strain, slowly growing temperature sensitive (42, 48),                     activated transcription similar template containing a <br /> has defect galactose induction (48). The reduced galac-                       heat shock transcription factor-responsive UASHSE, the <br /> tose induction does appear to changes                     ratios transcription templates wild-type and <br /> expression GAL4, paf1 mutation obvious                          mutant extracts essentially indistinguishable three <br /> effect level GAL4 mRNA (68). These results strongly                       temperatures tested. The lack obvious defect consistent <br /> suggest alteration UASGAL-driven expression seen                        results vivo experiments demonstrating that <br /> in paf1 strain occurs transcriptional level                      mutation Paf1p affects expression small subset of <br /> Gal4p general transcription apparatus.                                      promoters; model templates used experiment may <br />    Paf1p negatively regulates MAK16 gene expression. To iden-                       appropriate reveal Paf1p function. <br /> tify additional transcriptional defects paf1 mutant,                         Synthetic phenotype gal11 paf1 double mutant. <br /> surveyed transcripts 17 genes 42-kb segment                            Gal11p, component RNA polymerase II holoenzyme <br /> chromosome I (4). The RNA levels                            (33), postulated play role communicating the <br />  VOL. 16, 1996                                              Paf1p, A YEAST RNA POLYMERASE II-ASSOCIATED FACTOR                                    673 <br />  <br />  <br />  <br />  <br />                                                                          FIG. 4. Pleiotropic phenotypes paf1 mutant. (A) Growth curves the <br />                                                                       wild-type strain paf1 deletion-disruption strain. Wild-type (YJJ575) and <br />                                                                       paf1 mutant (YJJ576) strains grown YPD medium 308C. The growth <br />                                                                       curves shown plotted log scale. (B) Temperature-sensitive phenotype of <br />                                                                       paf1 mutant. Wild-type (YJJ575) paf1 mutant (YJJ576) strains were <br />                                                                       grown YPD plates 388C. (C) Altered cell morphology caused paf1 <br />                                                                       mutation. The wild-type (YJJ575) paf1 mutant (YJJ576) strains grown <br />                                                                       YPD medium mid-log phase 308C photographed same <br />                                                                       magnification. Most mutant cells enlarged rounded. <br />  <br />  <br />  <br />  <br /> RNA polymerase upstream activators repressors (3,             II-associated factor, nuclear localization Paf1p, the <br /> 24, 39, 46, 53). The phenotype conferred gal11 mutation       pleiotropic phenotype conferred <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>null allele</span> PAF1, and <br /> similar paf1 resulting three- fivefold reduction   identification positive negative effects several <br /> in expression galactose-regulated genes (53)            apparently unrelated genes support hypothesis Paf1p <br /> increased decreased transcript abundance                general transcription factor mediator subset of <br /> yeast genes (17, 39). To determine RNA poly-             yeast genes. Paf1p joins growing list yeast transcriptional <br /> merase II-associated factors functioned pathway        regulators positive negative roles gene tran- <br /> in nonoverlapping pathways, constructed gal11 paf1 dou-          scription. These regulators include Rap1p (Grf1p Tuf1p [5, <br /> ble mutant. If Gal11p Paf1p function pathway,         26, 50, 51]), Mcm1p (Prtf Grm [5, 31]), Gal11p (Spt13p [17, <br /> the phenotype double mutant severe             53]), Sin4p (Tsf1p [9, 10, 27, 28]), Sin3p (Rpd1p Gam2p [59, <br /> than phenotype single mutant. If, however,              63, 71]), Spt16p (Cdc68p [37]), Reb1p (Grf2p [7, 64]), and <br /> Gal11p Paf1p function distinct pathways,        Rgr1p (27). A feature common factors promoter- <br /> partially functionally redundant, expect se-          specific function, negative effects certain promoters <br /> vere phenotype double mutant. We constructed           positive effects others. Two general mechanisms have <br /> gal11 <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>deletion mutation</span> (gal11D::LEU2) strain isogenic        invoked explain functions genes. One <br /> the used disruption paf1 observed that,             involves interaction chromatin; mechanism been <br /> previously seen yeast strains (53), mutation       suggested Sin4p (27, 28) Spt16p (37, 45, 70). The other <br /> impaired galactose induction three- fivefold displayed       mechanism envisions direct interaction basal tran- <br /> modest growth defect YPD medium (data shown). We               scriptional machinery. In regard, Gal11p encodes com- <br /> used strain create GAL11/gal11 PAF1/paf1 heterozy-          ponent RNA polymerase II holoenzyme (32). The iso- <br /> gous diploid sporulated, 15 tetrads          lation Paf1p RNA polymerase-associated factor would <br /> were dissected described Materials Methods. All             link closely second proposed mechanism, al- <br /> tetrads showed 2 His1:2 His2 2 Leu1:2 Leu2 segre-             reason assume mechanisms are <br /> gation pattern. However, double gal11D::LEU2 paf1D::HIS3          mutually exclusive. <br /> mutants grew slowly 308C (doubling time .8 h)            A striking feature PAF1 coding sequence highly <br /> than single mutants (Fig. 7). Like paf1 mutant,     charged acidic nature. Much negative charge is <br /> the double mutant inviable elevated temperatures. The          localized carboxyl terminus. High charge content and <br /> severe growth defect double mutant indicates Paf1p        acidity common features transcription factors (30). <br /> and Gal11p function distinct pathways,     For example, products SPT genes, includ- <br /> which required normal yeast cell growth.                       ing Spt5p (56), Spt6p (55), Spt7p (18), Spt8p (15), Spt16p <br />                                                                       (Cdc68p) (45) encode acidic proteins. The proteins <br />                          DISCUSSION                                   encoded SPT6, -8, -16 genes extremely <br />                                                                       acidic N C termini, feature shared Paf1p. The products <br />   In work identified novel factor, Paf1p,        SPT genes, affect transcription subset of <br /> appears involved expression subset yeast         yeast promoters, PAF1 product define new <br /> genes. The identification protein RNA polymerase         family promoter-specific transcription factors. <br />  674       SHI ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 6. Paf1p required transcription model template vitro. <br />                                                                                     Preparation whole-cell transcription extracts vitro transcription of <br />                                                                                     G-less cassette templates described Materials Methods. Transcription <br />                                                                                     reactions performed 26, 33, 378C indicated. The white bars <br />                                                                                     indicate transcripts basal template lacking UAS (UAS2), black <br />                                                                                     bars transcripts template containing UASHSE activated by <br />                                                                                     heat shock transcription factor transcription extract (69). At the <br />                                                                                     figure ratios incorporation templates containing <br />                                                                                     UASHSE templates lacking UAS (UAS2) calculated scanning the <br />                                                                                     indicated bands autoradiogram densitometer. <br />  <br />  <br />  <br />                                                                                     possible connection MAK16 remain investi- <br />                                                                                     gated. The requirement functional Paf1p high tempera- <br />                                                                                     tures decreased expression uni- <br />                                                                                     dentified essential gene, seen EGD2 transcript, to <br />                                                                                     altered ratios expression multiple genes products <br />                                                                                     dependent precise stoichiometries correct function. <br />                                                                                        The Srb protein-containing mediator complex S. cer- <br />                                                                                     evisiae shown mediate transcriptional response by <br />                                                                                     activators vitro vivo (32, 33). Recently, two <br />                                                                                     genes encoding components mediator, SRB10 and <br />                                                                                     SRB11, cloned (36). Srb10p Srb11p involved <br />                                                                                     transcriptional activation (34, 36) tran- <br />    FIG. 5. The paf1 mutation causes altered transcription vivo. (A) The paf1 <br /> mutation reduces induction GAL gene transcripts. Wild-type (YJJ662) and <br />                                                                                     scriptional repression a-specific genes cells <br /> paf1 mutant (YJJ664) strains grown synthetic medium containing glu-         (61) yeast genes response glucose (34). <br /> cose, pelleted, resuspended synthetic medium containing 2% galactose. At     Another component mediator, Gal11p, both <br /> the indicated times, aliquots cells frozen 2708C RNA pre-        positive negative regulatory roles transcription (17, 46). <br /> pared fractionated described Materials Methods. After hybridiza- <br /> tion GAL7-GAL10 probe, blot stripped rehybridized with <br />                                                                                     The isolation Paf1p defines fourth RNA polymerase II- <br /> an ACT1 probe internal control. (B) The paf1 mutation reduces induction       associated factor positive negative effects on <br /> of UASGAL reporter construct. The b-galactosidase assays performed        transcription. Paf1p unlikely component me- <br /> described Materials Methods. For indicated reporter plasmid,         diator complex basis strategy used isola- <br /> least independent clones assayed, experiment repeated at <br /> least times. Mean values presented. Standard errors typically less <br />                                                                                     tion. Immobilization largest RNA polymerase II subunit <br /> than 25% reported value. Units activity obtained fol-       antibody directed CTD shown <br /> lowing formula: (optical density 420 nm 3 1,000)/(milliliters assayed 3 time     exclude interaction mediator RNA <br /> [in minutes] 3 protein concentration [in milligrams milliliter]). (C) The       polymerase II (23, 32). The initial fraction RNA poly- <br /> MAK16 transcript elevated paf1 mutant. Wild-type (YJJ575) paf1 <br /> mutant (YJJ576) strains grown YPD medium, RNA prepared <br />                                                                                     merase-associated factors included Paf1p did con- <br /> and analyzed outlined panels A B. The probes prepared         tain mediator factors (60). In addition, using tagged <br /> by random priming PCR-amplified MAK16 DNA FUN38 DNA (4) (gen- <br /> erous gifts D. Kaback). <br />  <br />  <br />  <br />  <br />    The pleiotropic phenotypic traits paf1 mutant suggest <br /> that Paf1p global transcriptional regulator. This consis- <br /> tent identification functionally unrelated <br /> targets Paf1p, including GAL genes, EGD2, MAK16. <br /> MAK16 (maintenance killer) originally identified one <br /> of genes necessary maintaining L-A double- <br /> stranded RNA virus killer toxin-encoding satellite, M1 <br /> in S. cerevisiae (66). MAK16 essential viability; temper- <br /> ature-sensitive mutant causes G1 cell cycle arrest large <br /> unbudded cells terminal phenotype (65, 66). The termi- <br /> nal morphology paf1 mutant cells arrested 388C                              FIG. 7. Synthetic growth defect gal11 paf1 double mutant. Isogenic <br /> showed high percentage large, unbudded cells (data                         wild-type (YJJ575), paf1 (YJJ576), gal11 (YJJ564), gal11 paf1 (YJJ699) <br /> shown), functional significance observation                    strains streaked YPD plate grown 308C 3 days. <br />  VOL. 16, 1996                                                           Paf1p, A YEAST RNA POLYMERASE II-ASSOCIATED FACTOR                                        675 <br />  <br />  <br /> version Paf1p, identified complexes containing Paf1p                         MAK16-SPO7 region. J. Bacteriol. 176:1872&#226;&#8364;&#8220;1880. <br /> and RNA polymerase II contain Srb proteins                           5. Bender, A., G. F. J. Sprague. 1987. MAT a1 protein, yeast transcription <br />                                                                                         activator, binds synergistically second protein set cell-type- <br /> (49). Paf1p potentially interacts directly indirectly                      specific genes. Cell 50:681&#226;&#8364;&#8220;691. <br /> with region(s) RNA polymerase II CTD.                            6. Bussey, H., D. B. Kaback, W.-W. Zhong, D. T. Vo, M. W. Clark, N. Fortin, <br />    It increasingly clear gene-specific factors (activators                      J. Hall, B. F. F. Ouellette, T. Keng, A. B. Barton, Y. Su, C. J. Davies, and <br /> or repressors) utilize multiple pathways communicate                             R. K. Storms. 1995. The nucleotide sequence chromosome I Sac- <br />                                                                                         charomyces cerevisiae. Proc. Natl. Acad. Sci. USA 92:3809&#226;&#8364;&#8220;3813. <br /> with basal transcription apparatus. In fact, artificial                      7. Chasman, D. I., N. F. Lue, A. R. Buchman, J. W. LaPointe, Y. Lorch, and <br /> creation interaction DNA-binding factor                             R. D. Kornberg. 1990. A yeast protein influences chromatin struc- <br /> Gal11p mediator complex sufficient trigger activa-                         ture UASg functions powerful auxiliary gene activator. Genes <br /> tion transcription (3). For naturally occurring activators                       Dev. 4:503&#226;&#8364;&#8220;514. <br />                                                                                      8. Chen, J.-L., L. D. Attardi, C. P. Verrijzer, K. Yokomori, R. Tjian. 1994. <br /> repressors, different pathways connecting factors                        Assembly recombinant TFIID reveals differential coactivator require- <br /> transcription apparatus totally independent function-                         ments distinct transcriptional activators. Cell 79:93&#226;&#8364;&#8220;105. <br /> ally redundant. A possible interpretation severe pheno-                       9. Chen, S., R. W. West, Jr., S. L. Johnson, H. Gans, B. Kruger, J. Ma. <br /> type paf1 gal11 double mutant reflects                            1993. TSF3, global regulatory protein silences transcription yeast <br />                                                                                         GAL genes, mediates repression a2 repressor identical to <br /> consequence destruction different partially com-                          SIN4. Mol. Cell. Biol. 13:831&#226;&#8364;&#8220;840. <br /> pensatory pathways communication RNA polymerase                             10. Chen, S. C., R. W. West, Jr., J. Ma, S. L. Johnson, H. Gans, G. <br /> II. Regulatory proteins like Gal4p require Paf1p                           Woldehawariat. 1993. TSF1 TSF6, required silencing Saccharomy- <br /> Gal11p levels activation repression. In ab-                          ces cerevisiae GAL genes, global regulatory genes. Genetics 134:701&#226;&#8364;&#8220;716. <br />                                                                                     11. Conaway, R. C., J. W. Conaway. 1993. General initiation factors for <br /> sence RNA polymerase-associated factor, regulation                            RNA polymerase II. Annu. Rev. Biochem. 62:161&#226;&#8364;&#8220;190. <br /> is reduced occurs; factors missing, cell                    12. Didichenko, S. A., M. D. Ter-Avanesyan, V. N. Smirnov. 1991. Ribo- <br /> growth severely impaired.                                                            some-bound EF-1a-like protein yeast Saccharomyces cerevisiae. Eur. J. <br />    Many yeast transcription factors originally identified                          Biochem. 198:705&#226;&#8364;&#8220;711. <br />                                                                                     13. Earnshaw, W. C. 1987. Anionic regions nuclear proteins. J. Cell Biol. <br /> genetically. Paf1p RNA polymerase II-associated                             105:1479&#226;&#8364;&#8220;1482. <br /> factor, Cdc73p (42), identified biochemical screen                     14. Edelman, A. M., D. K. Blumenthal, E. G. Krebs. 1987. Protein serine/ <br /> factors associated RNA polymerase II (48, 60). We                             threonine kinases. Annu. Rev. Biochem. 56:567&#226;&#8364;&#8220;613. <br /> found similarities differences altered patterns                       15. Eisenmann, D. M., C. Chapon, S. M. Roberts, C. Dollard, F. Winston. <br />                                                                                         1994. The Saccharomyces cerevisiae SPT8 gene encodes acidic protein <br /> transcription isogenic strains bearing disruptions                         functionally related SPT3 TATA-binding protein. Genetics <br /> PAF1 CDC73 (48). Both mutations lead increased                                 137:647&#226;&#8364;&#8220;657. <br /> abundance MAK16 DBP2 transcripts (Fig. 5C [48]);                             16. Elder, R. T., E. Y. Loh, R. W. Davis. 1983. RNA yeast trans- <br /> however, unlike gal11 paf1 defects, cdc73 mutation                          posable element Ty1 ends direct repeats, structure similar <br />                                                                                         retrovirus RNA. Proc. Natl. Acad. Sci. USA 80:2432&#226;&#8364;&#8220;2436. <br /> has effect galactose induction. In contrast severe                     17. Fassler, J. S., F. Winston. 1989. The Saccharomyces cerevisiae SPT13/ <br /> phenotype conferred gal11 paf1 double mutation,                              GAL11 gene positive negative regulatory roles transcription. <br /> phenotype conferred cdc73 paf1 mutation resembles                             Mol. Cell. Biol. 9:5602&#226;&#8364;&#8220;5609. <br /> conferred paf1 single mutation (48). We                      18. Gansheroff, L. J., C. Dollard, P. Tan, F. Winston. 1995. The Saccharo- <br />                                                                                         myces cerevisiae SPT7 gene encodes acidic protein important for <br /> that phenotype conferred cdc73 gal11 double mutation                           transcription vivo. Genetics 139:523&#226;&#8364;&#8220;536. <br /> is similar conferred cdc73 single mutation (48).                     19. Giniger, E., S. M. Varnum, M. Ptashne. 1985. Specific DNA binding of <br /> These facts consistent possibility Paf1p                          GAL4, positive regulatory protein yeast. Cell 40:767&#226;&#8364;&#8220;774. <br /> Cdc73p distinct functions single novel pathway                         20. Guarente, L., R. R. Yocum, P. Gifford. 1982. A GAL10-CYC1 hybrid <br />                                                                                         yeast promoter identifies GAL4 regulatory region upstream site. <br /> communication RNA polymerase II. In support                                Proc. Natl. Acad. Sci. USA 79:7410&#226;&#8364;&#8220;7414. <br /> idea, complexes contain RNA poly-                               21. Guthrie, C., G. R. Fink (ed.). 1991. Guide yeast genetics molec- <br /> merase II Paf1p Cdc73p (48). Further charac-                               ular biology. Methods Enzymol. 194:3&#226;&#8364;&#8220;37. <br /> terization PAF1, CDC73, genes encoding                           22. Harlow, E., D. Lane. 1988. Antibodies: laboratory manual. Cold Spring <br />                                                                                         Harbor Laboratory, Cold Spring Harbor, N.Y. <br /> polymerase-associated factors allow resolve role                   23. Hengartner, C. J., C. M. Thompson, J. Zhang, D. M. Chao, S.-M. Liao, A. J. <br /> in eukaryotic transcriptional regulation.                                               Koleske, S. Okamura, R. A. Young. 1995. Association activator <br />                                                                                         RNA polymerase II holoenzyme. Genes Dev. 9:897&#226;&#8364;&#8220;910. <br />                                                                                     24. Himmelfarb, H. J., J. Pearlberg, D. H. Last, M. Ptashne. 1990. GAL11P: <br />                          ACKNOWLEDGMENTS                                                yeast mutation potentiates effect weak GAL4-derived activators. <br />                                                                                         Cell 63:1299&#226;&#8364;&#8220;1309. <br />    We thank M. Woontner critical reading manuscript, <br />                                                                                     25. Holmstrom, K., T. Brandt, T. Kallesoe. 1994. The sequence 32420 <br /> K. Gulyas assistance yeast genetic techniques, M. Ter-                     bp segment located right arm chromosome II Saccharomyces <br /> Avanesyan supplying anti-Sup35p antibodies. We grateful S.                   cerevisiae. Yeast 10:s47&#226;&#8364;&#8220;s62. <br /> Fields suggestion analyze transcripts chromosome                26. Huet, J., A. Sentenac. 1987. TUF, yeast DNA-binding factor specific <br /> I, D. Kaback supplying appropriate clones, F. Winston                 UASrpg upstream activating sequences: identification protein and <br /> supplying lacZ reporter plasmids, T. Fukasawa supplying                    DNA-binding domain. Proc. Natl. Acad. Sci. USA 84:3648&#226;&#8364;&#8220;3652. <br /> GAL11 disruption construct.                                                         27. Jiang, Y. W., P. R. Dohrmann, D. J. Stillman. 1995. Genetic physical <br />    This work supported National Institutes Health grant RO1                   interaction yeast RGR1 SIN4 chromatin organization and <br /> GM38101 J.A.J. Z.F.B. supported Michigan State University                     transcriptional regulation. Genetics 140:47&#226;&#8364;&#8220;54. <br />                                                                                     28. Jiang, Y. W., D. J. Stillman. 1992. Involvement SIN4 global <br /> and Michigan State University Agricultural Experiment Station. <br />                                                                                         transcriptional regulator chromatin structure Saccharomyces cerevi- <br />                                                                                         siae. Mol. Cell. Biol. 12:4503&#226;&#8364;&#8220;4514. <br />                                 REFERENCES                                          29. Johnston, M., M. Carlson. 1992. Regulation carbon phosphate <br />  1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990.           utilization, p. 193&#226;&#8364;&#8220;281. In E. W. Jones, J. R. Pringle, J. R. Broach (ed.), <br />     Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410.                       The molecular cellular biology yeast Saccharomyces: gene expres- <br />  2. Baker, S. M., P. G. Okkema, J. A. Jaehning. 1984. Expression             sion. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br />     Saccharomyces cerevisiae GAL7 gene autonomous plasmids. Mol. Cell.           30. Karlin, S. 1993. Unusual charge configurations transcription factors the <br />     Biol. 4:2062&#226;&#8364;&#8220;2071.                                                                  basic RNA polymerase II initiation complex. Proc. Natl. Acad. Sci. USA <br />  3. Barberis, A., J. Pearlberg, N. Simkovich, S. Farrell, P. Reinagel, C. Bamdad,       90:5593&#226;&#8364;&#8220;5597. <br />     G. Sigal, M. Ptashne. 1995. Contact component polymerase      31. Keleher, C. A., C. Goutte, A. D. Johnson. 1988. The yeast cell-type- <br />     II holoenzyme suffices gene activation. Cell 81:359&#226;&#8364;&#8220;368.                        specific repressor alpha 2 acts cooperatively non-cell-type-specific <br />  4. Barton, A. B., D. B. Kaback. 1994. Molecular cloning chromosome I            protein. Cell 53:927&#226;&#8364;&#8220;936. <br />     DNA Saccharomyces cerevisiae: analysis genes FUN38-          32. Kim, Y.-J., S. Bjorklund, Y. Li, M. H. Sayre, R. D. Kornberg. 1994. A <br />  676         SHI ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />       multiprotein mediator transcriptional activation interaction            yeast pet56-his3-ded1 gene region. Nucleic Acids Res. 13:8587&#226;&#8364;&#8220;8601. <br />       C-terminal domain RNA polymerase II. Cell 77:599&#226;&#8364;&#8220;608.                        53. Suzuki, Y., Y. Nogi, A. Abe, T. Fukasawa. 1988. GAL11 protein, an <br /> 33.   Koleske, A. J., R. A. Young. 1994. An RNA polymerase II holoenzyme                  auxiliary transcription activator genes encoding galactose-metabolizing <br />       responsive activators. Nature (London) 368:466&#226;&#8364;&#8220;469.                                  enzymes Saccharomyces cerevisiae. Mol. Cell. Biol. 8:4991&#226;&#8364;&#8220;4999. <br /> 34.   Kuchin, S., P. Yeghiayan, M. Carlson. 1995. Cyclin-dependent protein            54. Swaffield, J. C., J. F. Bromberg, S. A. Johnston. 1992. Alterations a <br />       kinase cyclin homologs SSN3 SSN8 contribute transcriptional                  yeast protein resembling HIV Tat-binding protein relieve requirement an <br />       control yeast. Proc. Natl. Acad. Sci. USA 92:4006&#226;&#8364;&#8220;4010.                              acidic activation domain GAL4. Nature (London) 357:698&#226;&#8364;&#8220;700. <br /> 35.   Liang, P., A. B. Pardee. 1992. Differential display eukaryotic messen-       55. Swanson, M. S., M. Carlson, F. Winston. 1990. SPT6, essential gene <br />       ger RNA means polymerase chain reaction. Science 257:967&#226;&#8364;&#8220;971.                 affects transcription Saccharomyces cerevisiae, encodes nuclear pro- <br /> 36.   Liao, S.-M., J. Zhang, D. A. Jeffery, A. J. Koleske, C. M. Thompson, D. M.              tein extremely acidic amino terminus. Mol. Cell. Biol. 10:4935&#226;&#8364;&#8220;4941. <br />       Chao, M. Viljoen, H. J. J. van Vuuren, R. A. Young. 1995. A kinase-             56. Swanson, M. S., E. A. Malone, F. Winston. 1991. SPT5, essential gene <br />       cyclin pair RNA polymerase II holoenzyme. Nature (London) 374:                   important normal transcription Saccharomyces cerevisiae, encodes an <br />       193&#226;&#8364;&#8220;196.                                                                                acidic nuclear protein carboxy-terminal repeat. Mol. Cell. Biol. 11: <br /> 37.   Malone, E. A., C. D. Clark, A. Chiang, F. Winston. 1991. Mutations               3009&#226;&#8364;&#8220;3019. <br />       SPT16/CDC68 suppress cis- trans-acting mutations affect promoter           57. Tjian, R., T. Maniatis. 1994. Transcriptional activation: complex puz- <br />       function Saccharomyces cerevisiae. Mol. Cell. Biol. 11:5710&#226;&#8364;&#8220;5717.                    zle easy pieces. Cell 77:5&#226;&#8364;&#8220;8. <br /> 38.   Miller, J. H. 1972. Experiments molecular genetics. Cold Spring Harbor           58. Ulery, T. L., S.-H. Jang, J. A. Jaehning. 1994. Glucose repression of <br />       Laboratory Press, Cold Spring Harbor, N.Y.                                              yeast mitochondrial transcription: kinetics derepression role nu- <br /> 39.   Nishizawa, M., Y. Suzuki, Y. Nogi, K. Matsumoto, T. Fukasawa. 1990.                 clear genes. Mol. Cell. Biol. 14:1160&#226;&#8364;&#8220;1170. <br />       Yeast Gal11 protein mediates transcriptional activation signal           59. Vidal, M., R. Strich, R. E. Esposito, R. F. Gaber. 1991. RPD1 (SIN3/ <br />       different transacting factors, Gal4 general regulatory factor I/repressor/          UME4) required maximal activation repression diverse yeast <br />       activator site binding protein 1/translation upstream factor. Proc. Natl. Acad.         genes. Mol. Cell. Biol. 11:6306&#226;&#8364;&#8220;6316. <br />       Sci. USA 87:5373&#226;&#8364;&#8220;5377.                                                              60. Wade, P. A., W. Werel, R. C. Fentzke, N. E. Thompson, J. F. Leykam, R. R. <br /> 40.   Nonet, M. L., R. A. Young. 1989. Intragenic extragenic suppressors              Burgess, J. A. Jaehning, Z. F. Burton. A novel collection accessory <br />       mutations heptapeptide repeat domain Saccharomyces cerevisiae              factors associated yeast RNA polymerase II. Submitted publication. <br />       RNA polymerase II. Genetics 123:715&#226;&#8364;&#8220;724.                                            61. Wahi, M., A. D. Johnson. 1995. Identification genes required a2 <br /> 41.   Peterson, C. L., J. W. Tamkun. 1995. The SWI-SNF complex: chro-                   repression Saccharomyces cerevisiae. Genetics 140:79&#226;&#8364;&#8220;90. <br />       matin remodeling machine? Trends Biochem. Sci. 20:143&#226;&#8364;&#8220;146.                          62. Wang, B. Q., C. F. Kostrub, A. Finkelstein, Z. F. Burton. 1993. Produc- <br /> 42.   Reed, S. I., J. Ferguson, K. Y. Jahng. 1988. Isolation characteriza-            tion human RAP30 RAP74 bacterial cells. Protein Expr. Purif. <br />       tion genes encoding yeast mating pheromone signaling elements:                   4:207&#226;&#8364;&#8220;214. <br />       CDC72 CDC73. Cold Spring Harbor Symp. Quant. Biol. 53:621&#226;&#8364;&#8220;627. <br />                                                                                           63. Wang, H., I. Clark, P. R. Nic <br /> </body></html>