<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Comparative Functional Genomics <br /> Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br /> Published online Wiley InterScience (www.interscience.wiley.com). DOI: 10.1002/cfg.411 <br />                         Research Article <br />  <br />                         Comparative genomic assessment novel <br />                         broad-spectrum targets antibacterial <br />                         drugs <br /> Thomas A. White1 Douglas B. Kell2 * <br /> 1 Department   Biology, University York, Box 373, Heslington, York YO10 5YW, UK <br /> 2 Department   Chemistry, UMIST, Faraday Building, Sackville St, PO Box 88, Manchester M60 1QD, UK <br />  <br />  <br /> *Correspondence <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>:                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Douglas B. Kell, Department of <br /> Chemistry, UMIST, Faraday            Single multiple resistance antibacterial drugs currently use spreading, <br /> Building, Sackville Street, PO Box   act small number molecular targets; finding novel <br /> 88, Manchester, M60 1QD, UK.         targets anti-infectives great importance. All protein sequences <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> E-mail: dbk@umist.ac.uk              pathogens (Staphylococcus aureus, Mycobacterium tuberculosis Escherichia <br />                                      coli O157:H7 EDL993) assessed comparative genomics methods their <br />                                      suitability antibacterial targets according number criteria, including the <br />                                      essentiality protein, level sequence conservation, distribution in <br />                                      pathogens, bacteria eukaryotes (especially humans). Each protein scored and <br />                                      ranked based weighted variants criteria order prioritize proteins as <br />                                      potential novel broad-spectrum targets antibacterial drugs. A number proteins <br />                                      proved score highly species robust variations scoring <br />                                      used. Sensitivity analysis indicated quantitative contribution metric <br />                                      overall score. After analysis targets, tRNA methyltransferase <br />                                      (trmD) translation initiation factor IF-1 (infA) emerged potential novel <br />                                      antimicrobial targets worthy investigation. The scoring strategy used <br />                                      value areas post-genomic drug discovery. Copyright &#239;&#8250;&#8482; 2004 <br />   Received: 24 November 2003         John Wiley &amp; Sons, Ltd. <br />   Revised: 24 March 2004 <br />                                      Keywords: genomics; antibacterial; antimicrobial; pathogen; virulence; compara- <br />   Accepted: 1 April 2004 <br />                                      tive genomics; antibiotics; bioinformatics <br />  <br />  <br />  <br />  <br /> Introduction                                                         gene transfer mechanisms allow resistance <br />                                                                      passed different bacterial strains <br /> Within decades introduction peni-                      species (Davies, 1994; Heinemann, 1999). <br /> cillin, majority existing classes                      Antibacterial resistance developed steadily as <br /> antibacterial drugs discovered system-                   new agents introduced, past <br /> atic screening natural product libraries. Remark-                 10&#226;&#8364;&#8220;15 years shown dramatic increase the <br /> ably, new chemical classes active antibacte-                   occurrence resistant populations microbes in <br /> rial drugs successfully introduced                community hospital environments (Stru- <br /> 30 years (Hancock Knowles, 1998). Table 1                        elens, 1998). <br /> shows restricted set modes action                     Measures chemical modification exist- <br /> the major antibacterial drugs currently use.                      ing antibacterial drugs development of <br />    Microorganisms shown                      inhibitors resistance genes signif- <br /> be extremely versatile overcoming effects                     icant impact antibacterial therapy short <br /> of antibacterial drugs. Bacteria developed                      term. However, obvious new drug tar- <br /> a variety resistance mechanisms lateral                       gets need use antibacterial <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd. <br />  Assessment novel antibacterial targets                                                                                    305 <br />  <br />  <br />  <br /> Table 1. Mode action principal established antibacterial drugs <br /> Drug/class                        Function inhibited                          Molecular target <br />  <br /> &#206;&#178;-Lactams                         Peptidoglycan synthesis                     Transpeptidases carboxpeptidases <br /> Bacitracin                        Peptidoglycan synthesis                     Undecaprenyl pyrophosphate <br /> D-Cycloserine                     Peptidoglycan synthesis                     D-alanine racemase D-alanyl-D-alanine synthetase <br /> Fosfomycin                        Peptidoglycan synthesis                     UDP-N-acetylglucosamine enolpyruvyl transferase <br /> Glycopeptides                     Peptidoglycan synthesis                     Cell wall peptidoglycan <br /> Quinolones                        DNA replication/transcription               Gyrase topoisomerase IV <br /> Rifamycins                        Transcription                               RNA polymerase <br /> Aminoglycosides                   Protein synthesis                           30S ribosomal subunit <br /> Chloramphenicol                   Protein synthesis                           50S ribosomal subunit <br /> Fusidic adid                      Protein synthesis                           Elongation factor G <br /> Macrolides                        Protein synthesis                           50S ribosomal subunit <br /> Oxazolidinones                    Protein synthesis                           50S ribosomal subunit <br /> Streptogramins                    Protein synthesis                           50S ribosomal subunit <br /> Tetracyclines                     Protein synthesis                           30S ribosomal subunit <br /> Mupirocin                         Charging isoleucyl tRNA                  Isoleucyl tRNA synthetase <br /> Sulphonamides                     Folate synthesis                            Dihydropteroate synthetase <br /> Trimethoprim                      Folate synthesis                            Dihydrofolate reductase <br />  <br /> After Chopra et al. (2002). <br />  <br />  <br /> drugs continue successfully (Schmid, 1998).                 It shown data-driven strate- <br /> To end, genomic approaches providing                   gies used identify novel drug targets <br /> new strategy revealing new molecular targets                   (Spaltmann et al., 1999). A number metrics are <br /> that giving rise novel antibacterial agents                chosen properties potential <br /> (Allsop Illingworth, 2002; Dougherty et al.,                  drug target, essentiality specificity. <br /> 2002; Haney et al., 2002; Isaacson, 2002; Ji, 2002;               Each potential target genome is <br /> McDevitt Rosenberg, 2001), new                       scored properties. These scores be <br /> agents unlikely face current problems                  weighted differently add emphasis <br /> of established mechanisms resistance (McDevitt                 particular property. This scoring can <br /> and Rosenberg, 2001). In anti-infective research,                 tuned targets been <br /> the inevitable selection resistant strains means              identified score highly, showing scoring <br /> that drugs multiple targets preferred                 capable identifying useful targets. Pre- <br /> (e.g. multiple penicillin-binding proteins multi-              viously unidentified genes score highly, <br /> ple forms two-component systems; Stephenson                    prioritized potential drug targets <br />                                                                   study. The top-scoring gene study <br /> and Hoch, 2002; Stephenson Hoch, 2004). In <br />                                                                   carried Spaltmann et al. (1999) antifun- <br /> other pharmaceutical areas encouraging that <br />                                                                   gal targets &#206;&#177;,&#206;&#177;-trehalose-phosphate synthase, <br /> the rational utility traditional targets              gene suggested <br /> confirmed systematic <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>knock-out</span> studies (Zam-                   potential drug target. This shows post- <br /> browicz Sands, 2003).                                         genomic research offer terms of <br />    With release data numerous sequenc-                novel target identification (Allsop Illingworth, <br /> ing projects, number potential drug targets                2002; Buysse, 2001; Dougherty et al., 2002; Glass <br /> has increased massively. However,                et al., 2002; Haney et al., 2002; Isaacson, 2002; <br /> molecules drug targets (Hopkins                   Ji, 2002; Knowles King, 1998; McDevitt and <br /> Groom, 2002), big challenge select                  Rosenberg, 2001; Payne et al., 2001a, 2001b; Will- <br /> the targets relevant given situation (Ter-             ins et al., 2002). <br /> stappen Reggiani, 2001).                                         In present study number criteria were <br />    Machine learning methods seek devise new                    chosen characterize proteins targets. <br /> ideas hypotheses unstruc-                   These suggested extensive literature <br /> tured data (Gillies, 1996; Kell Oliver, 2004;                 subject (see e.g. Alksne, 2002; Allsop and <br /> Mitchell, 1997; Mjolsness DeCoste, 2001).                     Illingworth, 2002; McDevitt Rosenberg, 2001; <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                   Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  306                                                                             T. A. White D. B. Kell <br />  <br />  <br />  <br /> Projan, 2002; Spaltmann et al., 1999; Terstappen        hit look-up table listed classifi- <br /> and Reggiani, 2001). A list criteria used   cation organism (http://ca.expasy.org/cgi- <br /> is given Methods section.                        bin/speclist). A list bacteria treated pathoge- <br />                                                         nic study given Table 2. Bacteria may <br />                                                         act pathogens, depending cir- <br /> Methods                                                 cumstances host, list given here <br />                                                         covers broad range pathogens perhaps <br /> Data collection motives                             completely comprehensive. <br />                                                            The presence homologues mice con- <br /> Data collected pathogenic bacterial     sidered important allow targets <br /> species, Staphylococcus aureus, Escherichia coli        present higher organisms further <br /> O157:H7 EDL993 Mycobacterium tuberculo-             down-weighted, the <br /> sis. These species chosen represent        line target&#226;&#8364;&#8482;s absence mice make animal <br /> a broad cross-section bacterial types. Targets       trials effective. Lactobacillus spp. con- <br /> which prove score species        sidered beneficial probiotic bacteria, using <br /> will probably good targets broad spec-      metric able prioritize targets which <br /> trum pathogens.                                      diminish unwanted side-effects new drug. <br />    The entire set sequences proteins encoded         The scores BLAST hits pathogens <br /> by S. aureus, E. coli O157:H7 EDL993 M.             parsed conserved <br /> tuberculosis downloaded NCBI              particular gene pathogens. Obvi- <br /> website (http://www.ncbi.nlm.nih.gov/PMGifs/            ously protein well-conserved many <br /> Genomes/micr.html). Each protein char-         pathogens make better target broad- <br /> acterized number criteria      spectrum antibacterial drugs. A high degree of <br /> be used prioritize suitable proteins     conservation mean mutations the <br /> potential antibacterial targets.                        protein tolerated, resistance is <br />    A Perl program carried char-         likely emerge. The numbers identical <br /> acterization automatically (see Figure 1         residues pathogenic hit compared the <br /> overview). Each protein parsed gene     query sequence summed divided by <br /> index (gi) number protein. If       number hits pathogens. This number <br /> function protein known, function     normalized dividing <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> the <br /> had assigned protein basis        query sequence, ratio conservation for <br /> sequence homology, noted.                 protein pathogens. <br />    Each protein submitted BLAST              The query protein submitted BLAST <br /> (Altschul et al., 1990, 1997) search (BLASTp,           separately human genome (protein <br /> using default parameters &#226;&#8364;&#732;expectation     sequences)      (ftp://ftp.ncbi.nih.gov/genomes/H <br /> value&#226;&#8364;&#8482; 0.01) local copy SwissProt   sapiens/protein/) number hits was <br /> database (ftp://ftp.ebi.ac.uk/pub/). The SwissProt      recorded. The closest hit human protein <br /> database used curated,           recorded, ratio similarity given <br /> well annotated, non-redundant, entries        number positive residue matches (matches <br /> are easily parseable consistent format.      amino acids identical similar <br /> There exist large number associated files     biochemical properties) divided length of <br /> and websites use SwissProt-style codes (for       query sequence. The number positives was <br /> species gene/protein names). Using SwissProt        chosen err caution. Any <br /> therefore allows resources integrated       drug designed particular bacterial protein <br /> easily program, making efficient          act just human protein, even <br /> automation possible.                                    certain key residues identical. Similar- <br />    The results BLAST search parsed         ity residues activity. This <br /> to homologues protein             metric included potential targets which <br /> existed bacteria, pathogenic bacteria, eukary-       similar human proteins not <br /> otes, mice Lactobacillus. This          heavily penalized. Even human homo- <br /> comparing SwissProt species ID <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>         logue does exist, possible (e.g. using <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                       Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                 307 <br />  <br />  <br />  <br />                         Read genome                              For query protein <br />                         file split                              carry restricted <br />                         individual protein                               BLAST search against <br />                         sequences (FASTA                                 essential genes Bacilus <br />                         format)                                          subtilis, Escherichia coli K12, <br />                                                                          Mycobacterium tuberculosis <br />                                                                          Staphylococcus aureus <br />                                                                          (expectation value 0.01) <br />                         Take protein <br />                         sequence turn and <br />                         parse gene <br />                         name, gi number and <br />                         function                             For query protein <br />                         assigned                               carry restricted <br />                         protein                                            BLAST search against <br />                                                                            virulence genes in <br />                                                                            Bacillus anthracis, <br />                                                                            Escherichia coli O157:H7 <br />                                                                            EDL993, Mycobacterium <br />                                                                            tuberculosis, Neisseria <br />                         BLAST protein                                 Meningitidis and <br />                         SwissProt                              Staphylococcus aureus <br />                         database (expectation                              (expectation value 0.01) <br />                         value 0.01) <br />  <br />  <br />  <br />  <br />                                                                           For query protein <br />                         Parse BLAST <br />                                                                           carry BLAST <br />                         output each <br />                                                                           search SwissProt (with <br />                         protein find <br />                                                                           higher expectation value <br />                         taxonomic distribution <br />                                                                           1&#195;&#8212;10-10) parse output <br />                         protein its <br />                                                                           hits are <br />                         conservation in <br />                                                                           known antibacterial targets <br />                         pathogens <br />                                                                           structure <br />                                                                           Protein Data Bank <br />  <br />  <br />  <br />                          BLAST protein <br />                          protein <br />                          sequences                                   Output information for <br />                          human (expectation                               protein file. This <br />                          value 0.01). Find                                used score <br />                          number human                                  rank proteins <br />                          homologues                                   according potential as <br />                          similarity closest                            novel broad-spectrum <br />                          human hit.                                       drug targets <br />  <br />  <br /> Figure 1. Flow chart illustrating process data collection <br />  <br />  <br /> structure&#226;&#8364;&#8220;activity relationship studies) design               target structural similarity thought <br /> drug targets bacterial version               safer report close homologues. <br /> protein.                                                             After running BLAST algorithm, output <br />    The query gene submitted                      parsed query gene was <br /> the BLAST program homologues                         homologous known antibacterial target. This <br /> are known antibacterial targets structures                comparing SwissProt gene ID <br /> have deciphered. This time &#226;&#8364;&#732;expectation                    list SwissProt IDs (from ExPASy <br /> value&#226;&#8364;&#8482; 1 &#195;&#8212; 10&#226;&#710;&#8217;10 used, infer suitability              website: http://ca.expasy.org/enzyme/) proteins <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                     Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  308                                                                                             T. A. White D. B. Kell <br />  <br />  <br />  <br /> Table 2. List bacteria treated pathogenic study <br /> Acinetobacter calcoaceticus                Klebsiella pneumoniae                     Shigella dysenteriae <br /> Bacillus anthracis                         Legionella pneumophila                    Shigella flexneri <br /> Bacillus cereus                            Leptospira interrogans                    Staphylococcus aureus <br /> Bordetella pertussis                       Listeria monocytogenes                    Staphylococcus aureus strain Mu50/ATCC 700 699 <br /> Borrelia burgdorferi                       Moraxella catarrhalis                     Staphylococcus aureus strain MW2 <br /> Brucella abortus                           Moraxella lacunata                        Staphylococcus aureus strain N315 <br /> Brucella melitensis                        Mycobacterium leprae                      Staphylococcus capitis <br /> Brucella suis                              Mycobacterium tuberculosis                Staphylococcus epidermidis <br /> Campylobacter jejuni                       Mycoplasma fermentans                     Staphylococcus saprophyticus <br /> Chlamydia muridarum                        Mycoplasma genitalium                     Streptococcus agalactiae <br /> Chlamydia pneumoniae                       Mycoplasma hominis                        Streptococcus agalactiae serotype III <br /> Chlamydia trachomatis                      Mycoplasma penetrans                      Streptococcus agalactiae serotype V <br /> Clostridium botulinum                      Mycoplasma pneumoniae                     Streptococcus mutans <br /> Clostridium perfringens                    Neisseria gonorrhoeae                     Streptococcus pneumoniae <br /> Clostridium tetani                         Neisseria meningitidis                    Streptococcus pyogenes <br /> Corynebacterium diphtheriae                Neisseria meningitidis serogroup A        Streptococcus pyogenes serotype M18 <br /> Enterococcus faecalis                      Neisseria meningitidis serogroup B        Streptococcus pyogenes serotype M3 <br /> Enterococcus faecium                       Neisseria meningitidis serogroup C        Streptococcus pyogenes serotype M5 <br /> Escherichia coli O111:H&#226;&#710;&#8217;                   Pasteurella multocida                     Treponema pallidum <br /> Escherichia coli O127:H6                   Propionibacterium acnes                   Tropheryma whipplei <br /> Escherichia coli O157:H7                   Proteus mirabilis                         Ureaplasma urealyticum <br /> Escherichia coli O6                        Providencia rettgeri                      Vibrio cholerae <br /> Flavobacterium meningosepticum             Providencia stuartii                      Vibrio parahaemolyticus <br /> Francisella tularensis                     Pseudomonas aeruginosa                    Vibrio vulnificus <br /> Fusobacterium nucleatum                    Rickettsia conorii                        Wolinella recta <br /> Haemophilus ducreyi                        Rickettsia prowazekii                     Wolinella succinogenes <br /> Haemophilus influenzae                     Salmonella cholerae-suis                  Xanthomonas maltophilia <br /> Haemophilus parainfluenzae                 Salmonella enteritidis                    Yersinia pestis <br /> Helicobacter pylori                        Salmonella typhi <br /> Helicobacter pylori J99                    Salmonella typhimurium <br />  <br />  <br />  <br /> that known antibacterial targets (Chittum                      homology protein known structure likely <br /> Champney, 1995; Egebjerg et al., 1989; Kornder,                        similar structure (although not <br /> 2002; Lin et al., 1997; Neu Gootz, 1996;                           true) favoured potential <br /> Schnappinger Hillen, 1996). Of course,                     novel drug target. <br /> current drug targets perfect examples; indeed,                       Each protein submitted several <br /> many drugs target toxic                        restricted BLAST searches selected <br /> humans resistance begun emerge                           bacterial genomes. The BLAST searches were <br /> many cases. Nevertheless, treatments utilize                     restricted gi number; specifically gi numbers <br /> these targets shown effective                       genes essential involved in <br /> disease control, novel targets possessing                       virulence. These genomes chosen listed in <br /> similar characteristics known targets                        Table 3. <br /> useful.                                                                  These genomes selected cover a <br />   The SwissProt species protein ID codes                        wide range bacterial types, because <br /> each hit BLAST results compared                            characterized few <br /> to look-up table (ftp://beta.rcsb.org/pub/pdb/                       species work carried to <br /> uniformity/derived data/)                      great extent. For species this <br /> homologues query gene entry                       kind work done, genomics methods <br /> PDB database (ftp://ftp.ncbi.nih.gov/genomes/H                         allow predict essentiality involve- <br /> sapiens/protein/). A protein known struc-                       ment virulence. Proteins significant hits <br /> ture attractive point view fur-                 essential genes genes involved viru- <br /> ther research, structure-based drug design                   lence likely characteristics <br /> carried straightaway. A protein sequence                      score highly potential <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                       Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                309 <br />  <br />  <br />  <br /> Table 3. List genomes used restricted BLAST                   importance, so-called control coefficients <br /> searches essential genes genes involved                (http://dbk.ch.umist.ac.uk/mca home.htm; Fell, <br /> virulence <br />                                                                      1996; Heinrich Schuster, 1996; Kell West- <br /> Essential genes                                                      erhoff, 1986), enzyme controls the <br />   Genomes            Bacillus subtilis (Kobayashi et al., 2003)      flux metabolic pathway, fact be <br />                      Escherichia coli K12                            used relative importance variable <br />                      (http://www.shigen.nig.ac.jp/ecoli/pec/ <br />                      About.html) <br />                                                                      contributes total. The equation giving <br />                      Mycobacterium tuberculosis (Sassetti et al.,    sensitivity overall metric A individual met- <br />                      2003)                                           ric vi given (equation 1) <br />                      Staphylococcus aureus (Forsyth et al., 2002) <br /> Virulence genes                                                                            &#226;&#710;&#8218;A vi  &#226;&#710;&#8218; ln A <br />    Genomes           Bacillus anthracis (Hoffmaster Koehler,                   CiA =      &#194;&#183; =                         (1) <br />                      1999; Koehler, 2002)                                                  &#226;&#710;&#8218;vi A  &#226;&#710;&#8218; ln vi <br />                      Escherichia coli O157:H7 EDL993 (Brunder <br />                      et al., 2001; Sharma Dean-Nystrom,          Here discretized sensitivity analysis was <br />                      2003; Stuber et al., 2003; Wang et al., 2002)   target taking score each <br />                      Mycobacterium tuberculosis (Triccas         metric target, finding 1% score, divid- <br />                      Gicquel, 2000) <br />                      Neisseria meningitidis (Sun et al., 2000) <br />                                                                      ing number total score multiply- <br />                      Staphylococcus aureus (Dunman et al., 2001)     ing 100. When metrics, <br />                                                                      &#226;&#8364;&#732;contributions&#226;&#8364;&#8482; sum 1. Thus, sensitivity <br />                                                                      analysis asks, &#226;&#8364;&#732;By altering score vari- <br /> drug targets. The &#226;&#8364;&#732;model&#226;&#8364;&#8482; genomes                      able 1%, percentage change this <br /> the gene essential, likely                   induce total score?&#226;&#8364;&#8482;. These sensitivity analy- <br /> it gene essential query               ses clearly variables were <br /> species, greater potential target                  exerting too little influence the <br /> for broad-spectrum antibacterial drug.                             total score weights opti- <br />    Having assigned gene query genome                     mized accordingly. This novel approach proved <br /> values number characteristics, values                 useful carefully modifying scoring <br /> could weighted, summed ranked                         systems. <br /> produce list high-priority potential targets.                      Using different weighting schemes allowed <br /> This ranking approach used instead                          analysis robust particular high-ranking <br /> machine learning-based approach, &#226;&#8364;&#732;training                    target weighting scheme. Clearly, a <br /> set&#226;&#8364;&#8482; known antibacterial targets small                target scores highly having favourable <br /> not necessarily optimal (see Introduction). While                    characteristics highly weighted metric less <br /> the ranking approach subjective, does                     good ranks highly number <br /> allow targets prioritized score better                   different scoring systems. <br /> according metrics currently known                           For different scoring systems <br /> targets.                                                             (Table 4) used S. aureus, E. coli O157:H7 <br />                                                                      EDL993 M. tuberculosis 20 ranking <br />                                                                      targets recorded. These 20 lists could <br /> Assigning weights robustness target <br />                                                                      checked whether <br /> prioritization <br />                                                                      robust targets emerged. The 20 lists were <br /> A number different weighting schemes                         cross-checked targets were <br /> tried weighting scheme refined                  robust species (see Table 5). This &#226;&#8364;&#732;vot- <br /> to reflect relative importance various met-               ing&#226;&#8364;&#8482; method approach seen combining the <br /> rics. After weighting scheme run raw                    output weak learners, known to <br /> data, scores metric summed                     effective approach data mining (Bauer <br /> and total scores targets ranked.                     Kohavi, 1999; Dietterich, 2000; Hastie et al., <br /> The refinement weightings                         2001). <br /> carrying sensitivity analysis metric                       The scoring designed give <br /> scores ranking targets. Sensitivity                  influence metrics felt to <br /> analysis normally used biology                    important influence those <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                    Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />                                             Table 4. The different scoring systems used test influence scoring ranking targets <br />                                            Metric                                                                                                         Scoring                                                       310 <br />  <br />                                                                                                                      1                             2                        3                     4                   5 <br />  <br />                                            Copy number                                               20/No. copies gene       100/No. copies gene       Same 1             Same 1             Same 1 <br />                                            Distribution homologues                                (50/No. homologues       (100/No. homologues       Same 1             Same 1             Same 1 <br />                                                                                                      bacteria) &#195;&#8212; No.             bacteria) &#195;&#8212; No. of <br />                                                                                                      homologues pathogens        homologues pathogens <br />                                            Species distribution                                      [(500/89) &#195;&#8212; No. distinct    [(100/89) &#195;&#8212; No. distinct     Same 1             Same 1             Same 1 <br />                                                                                                      pathogens                 pathogens with <br />                                                                                                      homologues] &#226;&#710;&#8217; (No.          homologues] &#226;&#710;&#8217; (No. of <br />  <br />  <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd. <br />                                                                                                      distinct bacteria         distinct bacteria with <br />                                                                                                      homologues &#226;&#710;&#8217; No. distinct   homologues &#226;&#710;&#8217; No. distinct <br />                                                                                                      pathogens)                     pathogens) <br />                                            Conservation pathogens                                 Conservation ratio &#195;&#8212;200        Conservation ratio &#195;&#8212;100         Same 1             Same 1             Conservation <br />                                                                                                                                                                                                                 ratio &#195;&#8212;400 <br />                                            Eukaryotic homologues                                     50/(No. eukaryotic          100/(No. eukaryotic          Same 1             Same 1             Same 1 <br />                                                                                                      homologues + 1)                homologues + 1) <br />                                            Human homologues                                          50/(No. human               100/(No. human               Same 1             Same 1             Same 1 <br />                                                                                                      homologues + 1)                homologues + 1) <br />                                            Proximity closest human                                100 &#226;&#710;&#8217; (proximity ratio         100 &#226;&#710;&#8217; (proximity ratio          Same 1             Same 1             Same 1 <br />                                            homologue                                                 &#195;&#8212;100)                          &#195;&#8212;100) <br />                                            Mouse homologues                                          10 &#226;&#710;&#8217; (No. mouse             100 &#226;&#710;&#8217; (No. mouse             Same 1             Same 1             Same 1 <br />                                                                                                      homologues + 1)                homologues + 1) <br />                                            Lactobacillus homologues                                  10 absent, 0 present     100 absent, 0 present     Same 1             Same 1             Same 1 <br />                                            Function known                                            50 yes, 0             100 yes, 0             Same 1             Same 1             Same 1 <br />                                            Homologous known target                                20 yes, 0             100 yes, 0             Same 1             Same 1             Same 1 <br />                                            Homologous essential      Bacillus subtilis            50 yes, 0             100 yes, 0             Same 1             150 yes, 0   Same 1 <br />                                            gene in:                     Escherichia coli K12         50 yes, 0             100 yes, 0             Same 1             150 yes, 0   Same 1 <br />                                                                         Mycobacterium tuberculosis   20 yes, 0             100 yes, 0             Same 1             100 yes, 0   Same 1 <br />                                                                         Staphylococcus aureus        20 yes, 0             100 yes, 0             Same 1             100 yes, 0   Same 1 <br />                                            Homologous virulence      Bacillus anthracis           10 yes, 0             100 yes, 0             50 yes, 0    Same 1             Same 1 <br />                                            gene in:                     Escherichia coli K12         20 yes, 0             100 yes, 0             100 yes, 0   Same 1             Same 1 <br />                                                                         Mycobacterium tuberculosis   20 yes, 0             100 yes, 0             100 yes, 0   Same 1             Same 1 <br />                                                                         Neisseria meningitidis       20 yes, 0             100 yes, 0             100 yes, 0   Same 1             Same 1 <br />                                                                         Staphylococcus aureus        20 yes, 0             100 yes, 0             100 yes, 0   Same 1             Same 1 <br />                                            Homologous PDB entry                                   10 yes, 0             100 yes, 0             Same 1             Same 1             Same 1 <br />  <br />  <br />  <br />  <br />    Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />                                                                                                                                                                                                                                T. A. White D. B. Kell <br />  Assessment novel antibacterial targets                                                                                                 311 <br />  <br />  <br />  <br /> Table 5. The overall ranking targets <br /> Rank            Gene name/description                                                                    Robustness               Total score <br />  <br />  1              tRNA methyltransferase (trmD)                                                                  15                     13 391 <br />  2              UDP-N-Acetylmuramate-L-alanine ligase (murC)                                                   15                     13 229 <br />  3              UDP-N-acetylglucosamine 1-carboxyvinyl transferase (murA)&#226;&#710;&#8212;                                     13                     13 059 <br />  4              Translation initiation factor IF-1 (infA)                                                      14                     13 019 <br />  5              DNA polymerase III, &#206;&#177; chain (dnaE)                                                             13                     12 992 <br />  6              30S ribosomal protein S4 (rpsD)&#226;&#710;&#8212;                                                               11                     12 779 <br />  7              UDP-N-acetylmuramoylalanine-D-glutamate ligase (murD)                                          11                     12 766 <br />  8              50S ribosomal protein L10 (rplJ)                                                               11                     12 755 <br />  9              Chromosomal replication initiator protein (dnaA)                                               10                     12 716 <br /> 10              UDP-N-acetylmuramoylalanyl-D-glutamate-2,6-diaminopimelate ligase (murE)                        9                     12 573 <br />  <br /> These targets rank highly species used rank 20s of scoring systems used. Robustness times <br /> the gene ranks 20 different scoring systems species used, giving maximum robustness score 15. <br /> Total score sum scores target scoring systems species used. The maximum possible total score 24 120. <br /> &#226;&#710;&#8212; Indicates gene known target antibacterial drug (murA targeted fosfomycin rpsD target tetracyclines). <br />  <br />  <br />  <br /> felt important. Homology essential                         searched conserved motifs identified <br /> genes M. tuberculosis S. aureus, homol-                        BLAST, used more <br /> ogy virulence genes Bacillus anthracis                         distantly related homologues query gene. <br /> weighted lower homology essential vir-                        This approach able identify human <br /> ulence genes organisms. This                         sequences which, closely related in <br /> reflect quality data organisms,                      terms sequence homology, sim- <br /> as different methods used lists essential                     ilar terms structure biochemical proper- <br /> and virulence genes complete.                              ties query gene. Multiple sequence align- <br />    Under second scoring metrics                       ments phylogenetic trees created using <br /> weighted equally, maximum score                         ClustalX (Thompson et al., 1997) Mega2.1 <br /> metric another. For                          (Kumar et al., 2001). This deter- <br /> other scoring systems metrics                           distinct genes pathogens <br /> were weighted system. However,                         homologues non-pathogens and <br /> in scoring homology virulence                         eukaryotes, just &#226;&#8364;&#732;active sites&#226;&#8364;&#8482; <br /> genes given greater influence, fourth                          genes conserved differ- <br /> homology essential genes given greater                             ent pathogenic species. The available literature was <br /> weight, fifth level conservation                     searched gain insights sug- <br /> the target pathogens given importance.                        gested targets. <br />  <br /> Further investigation high-scoring targets                             Results discussion <br /> Having narrowed number potential <br /> drug targets using methods outlined above,                            Scores <br /> the highest-scoring targets investi-                        According scoring systems used, major- <br /> gated greater detail. The genes                         ity genes S. aureus, E. coli O157:H7 EDL993 <br /> subjected BLAST search Swis-                             M. tuberculosis make poor antibac- <br /> sProt database determine pathogens                            terial targets (see Figures 2&#226;&#8364;&#8220;4). In bacteria <br /> they present. The databases Genbank, EMBL                            high-scoring genes. The <br /> and DDBJ searched ENTREZ                                    highest ranking fairly robust <br /> (http://www.ncbi.nlm.nih.gov/entrez/query.fcgi)                           tend rank 20, regardless which <br /> to copy query gene                           scoring used (see Table 5). <br /> existed specific pathogen, case                        It apparent targets are <br /> missed searching SwissProt database.                          perfect way. To obtain perfect score <br /> PROSITE (http://us.expasy.org/prosite/)                          present metrics target present <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  312                                                                                  T. A. White D. B. Kell <br />  <br />  <br />  <br /> in copy, present pathogens            terms properties antibacterial drug tar- <br /> in non-pathogens, eukaryotes humans. It           possess. Hence, scoring sys- <br /> be perfectly conserved pathogens. Its            tem rates important properties &#226;&#8364;&#732;species <br /> function known, homologous           distribution&#226;&#8364;&#8482;, &#226;&#8364;&#732;conservation pathogens&#226;&#8364;&#8482;, &#226;&#8364;&#732;similar- <br /> to known target, homologous essential              ity human&#226;&#8364;&#8482; &#226;&#8364;&#732;homology essential genes&#226;&#8364;&#8482;. <br /> virulence genes model genomes used               A target does perform one <br /> and structure known. Here            criteria probably make good <br /> highest-ranking targets achieve 50%            drug target. The emphasis accorded prop- <br /> the perfect score.                                          erties means targets present in <br />   This discouraging, means            wide range pathogens, conserved, <br /> there little possibility development          similar targets humans, not <br /> a &#226;&#8364;&#732;magic bullet&#226;&#8364;&#8482; drug highly effective,             essential able score highly thus <br /> specifically targets pathogens, easy             prioritized. The metric &#226;&#8364;&#732;species distri- <br /> develop immune problems emerg-             bution&#226;&#8364;&#8482; weighted target receive the <br /> ing resistance. However, likely            maximum score present bacteria <br /> prospect.                                                   treated pathogenic study non- <br />   The unusual peaks distribution graphs              pathogens. It unlikely maximum would <br /> are genes unknown function that,             awarded target, property <br /> submitted BLAST expectation value             given high weighting compensate for <br /> 0.01, did return hits. The peaks             fact. The useful properties target may <br /> E. coli O157:H7 EDL993 target scores occur              possess are, sense, bonuses scored to <br /> the reason, peak higher         reflect this. A target does necessarily need to <br /> score genes return hits have <br />                                                             (directly) involved virulence order a <br /> been assigned sort function, presumably <br />                                                             drug neutralize infection. However, involve- <br /> by methods. <br />                                                             ment virulence bring benefits using a <br />                                                             target, target absent most <br /> Scoring systems sensitivity analysis                    non-pathogens absent humans. The <br /> The scoring used designed               existence homologues humans does mat- <br /> reflect thought important             ter se; rather, similarity (or lack) of <br />  <br />                 800 <br />  <br />                 700 <br />  <br />                 600 <br />  <br />                 500 <br />     Frequency <br />  <br />  <br />  <br />  <br />                 400 <br />  <br />                 300 <br />  <br />                 200 <br />  <br />                 100 <br />  <br />                  0 <br />                       0 19 <br />                       20 39 <br />                       40 59 <br />                       60 79 <br />                       80 99 <br />                       100 119 <br />                       120 139 <br />                       140 159 <br />                       160 179 <br />                       180 199 <br />                       200 219 <br />                       220 239 <br />                       240 259 <br />                       260 279 <br />                       280 299 <br />                       300 319 <br />                       320 339 <br />                       340 359 <br />                       360 379 <br />                       380 399 <br />                       400 419 <br />                       420 439 <br />                       440 459 <br />                       460 479 <br />                       480 499 <br />                       500 519 <br />                       520 539 <br />                       540 559 <br />                       560 579 <br />                       580 599 <br />                       600 619 <br />                       620 639 <br />                       640 659 <br />                       660 679 <br />                       680 699 <br />                       700 719 <br />  <br />  <br />  <br />  <br />                                                        Score between <br />  <br /> Figure 2. Frequency distribution scores potential targets Staphylococcus aureus, based scoring <br /> system used <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                            Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                            313 <br />  <br />  <br />  <br />                 1200 <br />  <br />                 1000 <br />  <br />                 800 <br />     Frequency <br />  <br />  <br />  <br />  <br />                 600 <br />  <br />                 400 <br />  <br />                 200 <br />  <br />                   0 <br />                        0 19 <br />                        20 39 <br />                        40 59 <br />                        60 79 <br />                        80 99 <br />                        100 119 <br />                        120 139 <br />                        140 159 <br />                        160 179 <br />                        180 199 <br />                        200 219 <br />                        220 239 <br />                        240 259 <br />                        260 279 <br />                        280 299 <br />                        300 319 <br />                        320 339 <br />                        340 359 <br />                        360 379 <br />                        380 399 <br />                        400 419 <br />                        420 439 <br />                        440 459 <br />                        460 479 <br />                        480 499 <br />                        500 519 <br />                        520 539 <br />                        540 559 <br />                        560 579 <br />                        580 599 <br />                        600 619 <br />                        620 639 <br />                        640 659 <br />                        660 679 <br />                        680 699 <br />                        700 719 <br />                                                          Score between <br />  <br /> Figure 3. Frequency distribution scores potential targets Escherichia coli O157:H7 EDL993, based first <br /> scoring used <br />  <br />  <br />                 1200 <br />  <br />                 1000 <br />  <br />                  800 <br />     Frequency <br />  <br />  <br />  <br />  <br />                  600 <br />  <br />                  400 <br />  <br />                  200 <br />  <br />                    0 <br />                        0 19 <br />                        20 39 <br />                        40 59 <br />                        60 79 <br />                        80 99 <br />                        100 119 <br />                        120 139 <br />                        140 159 <br />                        160 179 <br />                        180 199 <br />                        200 219 <br />                        220 239 <br />                        240 259 <br />                        260 279 <br />                        280 299 <br />                        300 319 <br />                        320 339 <br />                        340 359 <br />                        360 379 <br />                        380 399 <br />                        400 419 <br />                        420 439 <br />                        440 459 <br />                        460 479 <br />                        480 499 <br />                        500 519 <br />                        520 539 <br />                        540 559 <br />                        560 579 <br />                        580 599 <br />                        600 619 <br />                        620 639 <br />                        640 659 <br />                        660 679 <br />                        680 699 <br />  <br />  <br />  <br />  <br />                                                           Score between <br />  <br /> Figure 4. Frequency distribution scores potential targets Mycobacterium tuberculosis, based scoring <br /> system used <br />  <br />  <br /> the target human homologue impor-               bacterial version protein. In similar way <br /> tant. Again, reflected scoring system,         &#226;&#8364;&#732;known function&#226;&#8364;&#8482; &#226;&#8364;&#732;entry PDB&#226;&#8364;&#8482; crucial <br /> with number human homologues                properties potential target possess. They <br /> important proximity. Of course lack               simply imply known <br /> any human homologues bring benefits,               targets used jumping- <br /> such reduced need QSAR studies             point investigation. &#226;&#8364;&#732;Copy number&#226;&#8364;&#8482; <br /> lead compounds selectively target              potentially important as, protein exists <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  314                                                                                                                 T. A. White D. B. Kell <br />  <br />  <br />  <br /> in copies drug targets copy,                                          perform vagaries one <br /> it possible non-targeted copy                                           scoring system. <br /> over function target, rendering                                             We used sensitivity analyses five <br /> drug useless. However, likely drugs                                         ranking targets S. aureus dif- <br /> can disrupt function copy disrupt                                        ferent scoring systems (Figures 5&#226;&#8364;&#8220;9) also <br /> function both. Therefore, copy number given                                            sensitivity analyses lowest-ranking <br /> a low/intermediate weighting scoring sys-                                             (Figure 10) middle-ranking (Figure 11) targets <br /> tem. (Multiple paralogues make develop-                                             S. aureus scoring used. <br /> ment resistance likely.) The properties                                         It seen cases homology viru- <br /> &#226;&#8364;&#732;eukaryotic homologues&#226;&#8364;&#8482; &#226;&#8364;&#732;mouse homologues&#226;&#8364;&#8482;                                               lence genes does make contribution <br /> are crucial, make process                                           total score top-ranking targets. This <br /> drug development easier. A target homo-                                              generally because, unlike essential genes, viru- <br /> logues animal models useful                                             lence genes occur limited spectrum of <br /> it comes carrying animal trials. The lack                                          pathogens. As virulence genes tend or <br /> eukaryotic homologues allow use                                              specific mode pathogenicity, they <br /> the target developing drugs livestock. The                                            occur broad spectrum organ- <br /> property &#226;&#8364;&#732;homology known target&#226;&#8364;&#8482; weighted                                              isms consequently perform poorly metric <br /> fairly low scoring system. Homology                                             &#226;&#8364;&#732;distribution bacteria&#226;&#8364;&#8482; &#226;&#8364;&#732;distribution homo- <br /> to known target imply                                              logues&#226;&#8364;&#8482;. Similarly, proteins homol- <br /> biochemical properties target                                              ogy virulence genes pathogen, often <br /> relevant drug design process. However, tar-                                           hits virulence genes other <br /> gets currently use necessarily best                                           pathogens (e.g. Figure 5). <br /> available, problems toxicity resistance                                             The top-ranking targets scoring system <br /> indicate. This study aimed finding novel                                          similar sensitivity pro- <br /> targets, property treated                                          files, generally homologous essen- <br /> as important. The scoring sys-                                              tial genes species used, having no <br /> tems used variants scoring system,                                         homologues humans well-conserved <br /> with exception second, simply                                               wide range pathogens. Where target pro- <br /> treats metrics equally important. The                                           files differ extent gene&#226;&#8364;&#8482;s dis- <br /> function scoring systems perturb                                          tribution pathogens, extent con- <br /> top-ranking targets, weed targets                                         servation (although much), a <br />  <br />                                                 0.3 <br />  <br />  <br />                                                0.25 <br />                  Contribution total score <br />  <br />  <br />  <br />  <br />                                                 0.2                                                                             1st <br />                                                                                                                                 2nd <br />                                                0.15                                                                             3rd <br />                                                                                                                                 4th <br />                                                 0.1                                                                             5th <br />  <br />                                                0.05 <br />  <br />  <br />                                                  0 <br />                                                       1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                           Metric <br />  <br /> Figure 5. The contribution metric total score ranking targets Staphylococcus aureus, based <br /> on scoring used. Numbers refer metrics described Table 6. trmD ranking (1st) target under <br /> this scoring system <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                           Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                                        315 <br />  <br />  <br />  <br /> structure known, gene                                               Figure 7 shows profiles middle-ranking <br /> homologous known target,                                             targets vary. These targets score several <br /> the target homologous virulence genes.                                             criteria poorly others, number <br /> From Figure 6 the profiles                                              human homologues homology essential <br /> of lowest-ranking targets metric &#226;&#8364;&#732;similar-                                           genes. <br /> ity human homologue&#226;&#8364;&#8482; relatively more <br /> important. This simply targets <br />                                                                                              High-ranking targets <br /> score poorly metrics. <br /> Many lower-ranking targets hypo-                                                A number genes rank consistently <br /> thetical unknown proteins return                                             highly scoring systems used, but <br /> many (if any) hits submitted BLAST,                                                  appear 20 targets the <br /> so poorly characterized score accordingly.                                           genomes used study (see Table 5). Of these, <br />  <br />                                                0.08 <br />  <br />                                                0.07 <br />                  Contribution total score <br />  <br />  <br />  <br />  <br />                                                0.06 <br />                                                                                                                                   1st <br />                                                0.05 <br />                                                                                                                                   2nd <br />                                                0.04                                                                               3rd <br />                                                                                                                                   4th <br />                                                0.03 <br />                                                                                                                                   5th <br />                                                0.02 <br />  <br />                                                0.01 <br />  <br />                                                  0 <br />                                                       1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                           Metric <br />  <br /> Figure 6. The contribution metric total score ranking targets Staphylococcus aureus, based <br /> on second scoring used. Numbers refer metrics described Table 6 <br />  <br />                                                 0.3 <br />  <br />                                                0.25 <br />                  Contribution total score <br />  <br />  <br />  <br />  <br />                                                 0.2                                                                               1st <br />                                                                                                                                   2nd <br />                                                0.15                                                                               3rd <br />                                                                                                                                   4th <br />                                                 0.1 <br />                                                                                                                                   5th <br />  <br />                                                0.05 <br />  <br />  <br />                                                  0 <br />                                                       1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                           Metric <br />  <br /> Figure 7. The contribution metric total score ranking targets Staphylococcus aureus, based <br /> on scoring used. Numbers refer metrics described Table 6. trmD ranks fifth this <br /> scoring system <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                            Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  316                                                                                                                     T. A. White D. B. Kell <br />  <br />  <br />  <br />                                                    0.2 <br />                                                   0.18 <br />                     Contribution total score <br />  <br />  <br />                                                   0.16 <br />                                                   0.14                                                                                1st <br />                                                   0.12                                                                                2nd <br />                                                    0.1                                                                                3rd <br />                                                   0.08                                                                                4th <br />                                                   0.06                                                                                5th <br />  <br />                                                   0.04 <br />                                                   0.02 <br />                                                     0 <br />                                                          1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                             Metric <br />  <br /> Figure 8. The contribution metric total score ranking targets Staphylococcus aureus, based <br /> on fourth scoring used. Numbers refer metrics described Table 6. trmD ranking target under <br /> this scoring system, IF-1 (infA) ranks fifth <br />  <br />  <br />                                                   0.6 <br />                Contribution total score <br />  <br />  <br />  <br />  <br />                                                   0.5 <br />  <br />                                                   0.4                                                                                 1st <br />                                                                                                                                       2nd <br />                                                   0.3                                                                                 3rd <br />                                                                                                                                       4th <br />                                                   0.2                                                                                 5th <br />  <br />                                                   0.1 <br />  <br />                                                     0 <br />                                                          1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                             Metric <br />  <br /> Figure 9. The contribution metric total score ranking targets Staphylococcus aureus, based <br /> on fifth scoring used. Numbers refer metrics described Table 6. IF-1 ranking target this <br /> scoring system, trmD ranks second <br />  <br /> UDP-N-acetylglucosamine 1-carboxyvinyltransfer-                                                 2001; Tanner et al., 1996). The ribosome of <br /> ase (murA) known target fosfomycin,                                                 course currently heavily targeted antibacterial <br /> the 30S ribosomal subunit protein S4 (rpsD)                                                  drugs suggestions work riboso- <br /> a known target tetracyclines. Of others,                                                 mal proteins previously (Knowles <br /> dnaE, murC, murD murE previ-                                                      King, 1998). The existence known <br /> ously suggested potential drug targets (Bouhss                                               previously suggested targets overall top <br /> et al., 1997; El Zoeiby et al., 2003; Inoue et al.,                                             10 ranking sense validates study, it <br /> 2001; Marmor et al., 2001; Projan, 2002; Tanner                                                 indicates method target prioritization is <br /> et al., 1996), work carried murC, murD                                               able identify useful targets. There follows <br /> and murE revealed effective inhibitors                                             brief discussion potential the <br /> proteins (El Zoeiby et al., 2003; Marmor et al.,                                                novel targets suggested. <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                               Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                                                           317 <br />  <br />  <br />  <br />                                                                  0.7 <br />  <br />                                   Contribution total score    0.6 <br />  <br />                                                                  0.5 <br />                                                                                                                                           5th lowest <br />                                                                  0.4                                                                      4th lowest <br />                                                                                                                                           3rd lowest <br />                                                                  0.3                                                                      2nd lowest <br />                                                                                                                                           lowest <br />                                                                  0.2 <br />  <br />                                                                  0.1 <br />  <br />                                                                       0 <br />                                                                               1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                                    Metric <br />  <br /> Figure 10. The contribution metric total score lowest ranking targets Staphylococcus aureus <br /> based scoring used. Numbers refer metrics described Table 6 <br />  <br />                                                                 0.3 <br />  <br />  <br />                                                            0.25 <br />                    Contribution total score <br />  <br />  <br />  <br />  <br />                                                                 0.2                                                                                    1 <br />                                                                                                                                                        2 <br />                                                            0.15                                                                                        3 <br />                                                                                                                                                        4 <br />                                                                 0.1                                                                                    5 <br />  <br />                                                            0.05 <br />  <br />  <br />                                                                  0 <br />                                                                           1    2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 <br />                                                                                                               Metric <br />  <br /> Figure 11. The contribution metric total score mid-ranking targets Staphylococcus aureus based on <br /> the scoring used. Numbers refer metrics described Table 6 <br />  <br />  <br /> tRNA methyltransferase (trmD)                                                                                       The structure enzyme deter- <br />                                                                                                                  mined available Protein Data <br />                                                                                                                  Bank (http://www.rcsb.org/pdb/) (Accession Nos <br /> tRNA methyltransferase (trmD) catalyses trans-                                                               1UAJ, 1UAK, 1UAL 1UAM) (Ahn et al., <br /> fer methyl group S-adenosyl-L-methio-                                                                  2003). The active site regions enzyme, which <br /> nine (AdoMet) G37 subset bacterial                                                                binds AdoMet tRNA, known are <br /> tRNA species, G residue 36th                                                                 illustrated Figure 12. It seen these <br /> position (Ahn et al., 2003). It essential                                                             active site regions highly conserved. This is <br /> maintenance correct reading frame                                                                  encouraging point view designing <br /> translation. As enzyme probably bet-                                                                  broad-spectrum drug target enzyme and <br /> ter target requiring inhibition                                                                terms reduced potential resistant <br /> protein&#226;&#8364;&#8220;protein interactions, note                                                              mutants emerging. <br /> progress finding inhibitors                                                                TrmD has, knowledge, rec- <br /> made (Oneyama et al., 2002; Paulmurugan et al.,                                                                  ommended antibacterial drug target in <br /> 2004).                                                                                                           scientific literature, &#226;&#8364;&#8221; with <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                                               Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  318                                                                                              T. A. White D. B. Kell <br />  <br />  <br />  <br /> Table 6. Key metrics used                                  bacteria. As seen, trmD present many <br /> Number     Description metric <br />                                                                   pathogens, Gram-positive Gram-negative, <br />                                                                   indicating potential very <br />  1         Copy number                                            good broad-spectrum antibacterial drug target. The <br />  2         Distribution homologues (i.e.              enzyme appears function dimer (Ahn et al., <br />            homologues non-pathogens vs. pathogens?) <br />                                                                   2003), probably seek target the <br />  3         Distribution pathogens (i.e. distinct <br />            pathogens gene present in,      active site. <br />            discrete non-pathogens?)                                 It important know pathogens <br />  4         Conservation pathogens                              trmD absent. Table 7 shows the <br />  5         Number homologues eukaryotes                     pathogens (as defined study) known <br />  6         Number human homologues <br />                                                                   possess trmD. It important note, how- <br />  7         Similarity human homologue <br />  8         Number mouse homologues                             ever, small number strains <br />  9         Lactobacillus plantarum homologues <br /> 10         Function known? <br /> 11         Homology known target                               Table 7. List pathogenic bacteria (as defined this <br /> 12         Homology essential gene Bacillus subtilis        study) known possess homologues of <br /> 13         Homology essential gene Escherichia coli K12     trmD gene expectation value used <br /> 14         Homology essential gene Mycobacterium <br />            tuberculosis                                           Bacillus cereus                 Neisseria gonorrhoeae <br /> 15         Homology essential gene Staphylococcus aureus    Bordetella pertussis            Neisseria meningitidis serogroup C <br /> 16         Homology virulence gene Bacillus anthracis       Brucella abortus                Propionibacterium acnes <br /> 17         Homology virulence gene Escherichia coli         Clostridium botulinum           Proteus mirabilis <br />            O157:H7 EDL993                                         Clostridium tetani&#226;&#710;&#8212;             Providencia rettgeri <br /> 18         Homology virulence gene Mycobacterium            Corynebacterium diptheriae      Providencia stuartii <br />            tuberculosis                                           Enterococcus faecium            Salmonella cholerae-suis <br /> 19         Homology virulence gene Neisseria meningitidis   Escherichia coli 0111:H&#226;&#710;&#8217;        Salmonella enteritidis <br /> 20         Homology virulence gene Staphylococcus aureus    Escherichia coli O127:H6        Shigella dysenteriae <br /> 21         Homology PDB entry                                  Flavobacterium                  Staphylococcus capitis <br />                                                                   meningosepticum                 Staphylococcus epidermidis&#226;&#710;&#8212; <br />                                                                   Francisella tularensis          Staphylococcus saprophyticus <br />                                                                   Haemophilus parainfluenzae      Streptococcus agalactiae <br /> a large batch bacterial proteins &#226;&#8364;&#8221;                   Klebsiella pneumoniae           Streptococcus agalactiae serotype III&#226;&#710;&#8212; <br /> been patented basis experiments                      Legionella pneumophila          Streptococcus pyogenes serotype M5&#226;&#710;&#8212; <br /> S. aureus connection use drug tar-               Moraxella catarrhalis           Vibrio parahaemolyticus&#226;&#710;&#8212; <br /> get (United States Patent Trademark Office                    Moraxella lacunata              Wolinella recta <br />                                                                   Mycoplasma fermentans           Wolinella succinogenes <br /> http://www.uspto.gov/, Patent No. 6 187 541).                     Mycoplasma hominis              Xanthomonas maltophila <br />    Figure 13 shows neighbour-joining tree of <br /> trmD sequences pathogenic non-pathogenic                   &#226;&#710;&#8212;   Species/strains genomes entirely sequenced. <br />  <br />  <br />  <br /> Figure 12. (A) Alignment trmD sequences pathogens. trmD sequences S. aureus, E. coli O157:H7 EDL993 <br /> and M. tuberculosis used BLAST queries (non-redundant) hits searches combined. <br /> BLAST run expectation value 0.01. A sequence Acinetobacter calcoaceticus removed, was <br /> considerably shorter others. Active site regions highlighted black boxes labels function of <br /> the active site Ahn et al. (2003). Sequences aligned using ClustalX (Thompson et al., 1997). Names given <br /> as SwissProt style ID codes: TRMD MYCPN trmD Mycoplasma pneumoniae; MYCGE, Mycoplasma genitalium; <br /> UREUR, Ureaplasma urealyticum; BORBU, Borrelia burgdorferi; TREPA, Treponema pallidum; LISMO, Listeria monocytogenes; <br /> BACAN, Bacillus anthracis; ENTFA, Enterococcus faecalis; STRPY, Streptococcus pyogenes; STRPN, Streptococcus pneumoniae; <br /> STRA5, Streptococcus agalactiae serotype V; STRMU, Streptococcus mutans; STAAN, Staphylococcus aureus N315; CLOPE, <br /> Clostridium perfringens; MYCTU, Mycobacterium tuberculosis; MYCLE, Mycobacterium leprae; TROWH, Tropheryma whipplei; <br /> CHLTR, Chlamydia trachomatis; CHLMU, Chlamydia muridarum; CHLPN, Chlamydia pneumoniae; ECO57, Escherichia coli <br /> O157:H7 EDL993; SHIFL, Shigella flexneri; SALTY, Salmonella typhimurium; YERPE, Yersinia pestis; VIBVU, Vibrio vulnificus; <br /> VIBCH, Vibrio cholerae; PASMU, Pasteurella multocida; HAEIN, Haemophilus influenzae; HAEDU, Haemophilus ducreyi; PSAEA, <br /> Pseudomonas aeruginosa; NEIMA, Neisseria meningitidis; RICPR, Rickettsia prowazekii; RICCN, Rickettsia conorii; BRUME, <br /> Brucella melitensis; CAMJE, Campylobacter jejuni; HELPY, Helicobacter pylori. (B) Alignment selected trmD sequences from <br /> pathogenic bacteria human homologue (marked *). It seen human homologue aligns the <br /> bacterial proteins C-terminal end. See Figure 12 information <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                        Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                             319 <br />  <br />  <br />  <br />                   A <br />  <br />  <br />  <br />  <br />                                    Recognition tRNA general structure         AdoMet binding pocket <br />  <br />  <br />  <br />  <br />                                                AdoMet binding pocket       Lid <br />  <br />  <br />  <br />  <br />                           Recognition tRNA general structure <br />  <br />  <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                 Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  320                                        T. A. White D. B. Kell <br />  <br />  <br />  <br />              B <br />  <br />  <br />  <br />  <br /> Figure 12. Continued <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.   Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                                   321 <br />  <br />  <br />  <br />                                                            55       TRMD Streptococcus pyogenes (serotype M3) <br />                                                           99        TRMD Streptococcus pyogenes <br />                                                         60 <br />                                                                     TRMD Streptococcus pyogenes (serotype M18) <br />                                                         99           TRMD Streptococcus pneumoniae <br />                                                      99              TRMD Streptococcus agalactiae (serotype V) <br />                                                          53          TRMD Streptococcus mutans <br />                                                                  TRMD Lactococcus lactis (subsp. lactis) <br />                                                      92          TRMD Bacillus subtilis <br />                                                    74              TRMD Bacillus anthracis <br />                                                                TRMD Bacillus halodurans <br />                                                79                     TRMD Enterococcus faecalis <br />                                                                      TRMD Listeria monocytogenes <br />                                                              99 TRMD Listeria innocua <br />                                                                     TRMD Staphylococcus aureus (strain N315) <br />                                                             99      TRMD Staphylococcus aureus (strain Mu50) <br />                                                                     TRMD Staphylococcus aureus (strain MW2) <br />                                                                 TRMD Thermoanaerobacter teng congensis <br />                                                                     TRMD Clostridium perfringens <br />                                                     99             TRMD Clostridium acetobutylicum <br />                                                   85                     TRMD Borrelia burgdorferi <br />                                                                             TRMD Treponema pallidum <br />                                                                          TRMD Aquifex aeolicus <br />                                                                      TRMD Fusobacterium nucleatum (subsp. nucleatum) <br />                                                                     TRMD Thermotoga maritima <br />                                                                    99 TRMD Ureaplasma parvum <br />                                                                            TRMD Ureaplasma urealyticum <br />                                                                        TRMD Mycoplasma pulmonis <br />                                                                           TRMD Mycoplasma gallisepticum <br />                                                    89                       TRMD Mycoplasma pneumoniae <br />                                                                99            TRMD Mycoplasma genitalium <br />                                                               99        TRMD Chlamydia trachomatis <br />                                                        99                TRMD Chlamydia muridarum <br />                                                                        TRMD Chlamydia pneumoniae <br />                                                         99          TRMD Buchnera aphidicola (subsp. Schizaphis graminum) <br />                                                    98                TRMD Buchnera aphidicola (subsp. Acyrthosiphon pisum) <br />                                                                      TRMD Buchnera aphidicola (subsp. Baizongia pistaciae) <br />                                                                99      TRMD Neisseria meningitidis (serogroup B) <br />                                                74                      TRMD Neisseria meningitidis (serogroup A) <br />                                                                   99 TRMD Xylella fastidiosa <br />                                                 54         99          TRMD Xylella fastidiosa (strain Temecula1) <br />                                                                       TRMD Xanthomonas campestris (pv. campestris) <br />                                                                99 TRMD Xanthomonas axonopodis (pv. citri) <br />                                                                      TRMD Ralstonia solanacearum <br />                                                                        TRMD Pseudomonas aeruginosa <br />                                                            75         TRMD Pasteurella multocida <br />                                                           87         TRMD Haemophilus influenzae <br />                                                     74 <br />                                                                       TRMD Haemophilus ducreyi <br />                                                             99        TRMD Vibrio vulnificus <br />                                                         99            TRMD Vibrio cholerae <br />                                                               93 TRMD Serratia marcescens <br />                                                          66           TRMD Yersinia pestis <br />                                                                99 TRMD Salmonella typhi <br />                                                             99 <br />                                                                       TRMD Salmonella typhimurium <br />                                                             70       TRMD Shigella flexneri <br />                                                                      TRMD Escherichia coli O157:H7 <br />                                                              99 TRMD Escherichia coli <br />                                                                      TRMD Escherichia coli O6 <br />                                            66                 99         TRMD Mycobacterium tuberculosis <br />                                                      98                   TRMD Mycobacterium leprae <br />                                                 85                     TRMD   Corynebacterium glutamicum <br />                                                        58               TRMD Streptomyces coelicolor <br />                                                                           TRMD Tropheryma whipplei <br />                                                                       TRMD Deinococcus radiodurans <br />                                                                   99        TRMD Rickettsia prowazekii <br />                                                                              TRMD Rickettsia conorii <br />                                                68                         TRMD Caulobacter crescentus <br />                                                     99                   TRMD  Brucella melitensis <br />                                                          99         TRMD Rhizobium loti <br />                                                            80         TRMD Agrobacterium tumefaciens (strain C58) <br />                                                              93        TRMD Rhizobium meliloti <br />                                                                     TRMD Chlorobium tepidum <br />                                                                  TRMD Anabaena sp. (strain PCC 7120) <br />                                                     99           TRMD Synechocystis sp. (strain PCC 6803) <br />                                                                   TRMD Campylobacter jejuni <br />                                             99                      TRMD Helicobacter pylori <br />                                                            99       TRMD Helicobacter pylori J99 <br />                                                                       DCA3 Brassica juncea <br />                                                                   99      DCA1 Arabidopsis thaliana <br />  <br />                                   0.2 <br />  <br />  <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                                       Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  322                                                                                      T. A. White D. B. Kell <br />  <br />  <br />  <br /> Figure 13. Neighbour-joining tree showing distribution trmD pathogenic non-pathogenic bacteria. Sequences <br /> were aligned ClustalX (Thompson et al., 1997) (trmD Acinetobacter calcoaceticus removed, considerably <br /> shorter trmD sequences). The tree created using Mega 2.1 using default parameters (Kumar et al., <br /> 2001). Branches leading Gram-negative bacteria coloured blue, leading Gram-positive bacteria red, <br /> and leading Cyanobacteria dark green. The tree rooted using DCA3 Brassica juncea DCA1 from <br /> Arabidopsis thaliana outgroup (branches highlighted purple). trmD sequences bacteria treated pathogenic <br /> by study marked red diamond. Numbers branches bootstrap support groupings based 100 <br /> replicates (values &lt;50 shown). Scale bar shows number substitutions site <br />  <br />  <br /> have extensively sequenced (those marked                 drug designed attack target a <br /> *). Thus, copy gene exist                     broad spectrum pathogens. A number residues <br /> these species/strains despite absence            identical sequences, indicat- <br /> databases. As seen, small             ing high selection pressure <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span> at <br /> number species/strains entirely            positions. This encouraging point <br /> sequenced possess copy                  view drug resistance. Any resistant strains <br /> trmD.                                                         arising mutation residues could <br />    While version trmD humans,                severely attenuated compared wild-type <br /> the human version protein covers                  form. <br /> covers C-terminal end bacterial protein,              It observed IF-1 contains repeated <br /> as illustrated Figure 12B. This allow                sequence motif (S1-RM) in <br /> selective drug developed targets              ribosomal protein S1 (whose function enhance <br /> bacterial human version protein.           translational initiation Gram-negative bacteria) <br />    Another issue importance development             (Gribskov, 1992). Thus, drug designed tar- <br /> of new drug ease assay development (All-              motif attack different essential <br /> sop, 1998). If copy protein exists yeast,           gene products time, be <br /> then vitro assay development fairly straight-           highly advantageous point view of <br /> forward haploinsufficient <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span> (hp)-based            drug resistance. This motif appears involved <br /> strategies (Giaever et al., 1999). However, trmD           RNA binding. However, motif also <br /> homologue exists yeast, alternative strategy         eukaryotic translation initiation factor &#206;&#177;- <br /> must used here. Other assays course             chains (http://us.expasy.org/prosite/) several <br /> necessary cellular functional assays analy-           copies exist humans. Figure 14 shows <br /> sis vivo.                                                  sequences IF-1 pathogens aligned against <br />                                                               eIF-1&#206;&#177; sequences humans. <br />                                                               A number positions alignment are <br /> Translation initiation factor IF-1 (infA) <br />                                                               highly conserved humans pathogens, <br /> Translation initiation factor IF-1 scores         skill required develop a <br /> criteria homology virulence genes,              drug targets pathogens toxic <br /> homology known targets number                   humans. This highlights problem using <br /> homologues eukaryotes. IF-1 con-              overall sequence similarity determine close <br /> served pathogens, contributes signifi-            functional biochemical relatives gene <br /> cantly high score.                                     product. <br />    The precise function initiation factor IF-1             Nevertheless, exist cases success- <br /> unknown. However, known num-             ful drugs target proteins also <br /> ber factors essential establishment         present humans, antifungal strobil- <br /> correct reading frame translation (Dahlquist           urins target cytochrome bc1 (Weber et al., <br /> and Puglisi, 2000). It determinant           1990). It seen Figure 14 that <br /> of translation accuracy. IF-1 essential cell           number positions protein <br /> viability cells deficient IF-1 exhibit             pathogenic sequences one <br /> polysomes (Cummings Hershey, 1994).                       residue (or biochemically similar residues) but <br />    IF-1 conserved                 human sequences possess biochemically <br /> species/strains present,            different residue. Therefore, plenty of <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                             323 <br />  <br />  <br />  <br />  <br /> Figure 14. Alignment IF1 proteins pathogens human eIF-1&#206;&#177; sequences (bottom sequences, marked *). <br /> Some identical sequences close relatives species/strains shown omitted ease presentation. The alignment <br /> was created using ClustalX (Thompson et al., 1997). As seen, number positions highly conserved both <br /> human pathogen sequences. Black arrows mark positions majority pathogens share similar <br /> residues human sequences possess different residue. IF1 CHLTR IF-1 Chlamydia trachomatis; <br /> CHLPN, Chlamydia pneumoniae; MYCPE, Mycoplasma penetrans; UREUR, Ureaplasma urealyticum; MYCPN, Mycoplasma <br /> pneumoniae; MYCGE, Mycoplasma genitalium; CAMJE, Campylobacter jejuni; HELPY, Helicobacter pylori; NEIMA, Neisseria <br /> meningitidis serogroup A; RICPR, Rickettsia prowazekii; RICCN, Rickettsia conorii; BRUME, Brucella melitensis; ECO57 for <br /> Escherichia coli O157:H7 EDL993; YERPE, Yersinia pestis; VIBCH, Vibrio cholerae; HAEIN, Haemophilus influenzae; PASMU, <br /> Pasteurella multocida; PSEAE, Pseudomonas aeruginosa; STAAN, Staphylococcus aureus N315; BACAN, Bacillus anthracis; <br /> LISMO, Listeria monocytogenes; ENTFA, Enterococcus faecalis; STRPY, Streptococcus pyogenes; STRA5, Streptococcus agalactiae <br /> serotype V; CLOPE, Clostridium perfringens; MYCTU, Mycobacterium tuberculosis; TROWH, Tropheryma whipplei; LEPIN, <br /> Leptospira interrogans; TREPA, Treponema pallidum; BORBU, Borrelia burgdorferi; HUMAN, Homo sapiens <br />  <br /> potential development drug tar-               There number pathogens IF- <br /> gets pathogens interfering human              1 (see Table 8). Again, this <br /> form protein.                                           does necessarily mean pathogens not <br />   The structure IF-1 determined                    possess copy, cases whole-genome <br /> and available Protein Data Bank                    sequencing carried out. As be <br /> (http://www.rcsb.org/pdb/)         (Accession     Nos          seen, small number species <br /> 1HRO 1AH9). IF-1 known function                   entirely sequenced which <br /> a monomer.                                                     IF-1 absent. <br />   Figure 15 shows phylogenetic tree IF- <br /> 1 homologues. It seen the <br /> eukaryotic (chloroplast) sequences              Conclusions <br /> bacteria split fairly well. Sequences from <br /> humans pathogenic bacteria split easily.            This study used simple rational collec- <br /> The sequences pathogens non-pathogens                 tion criteria rate bacterial gene <br /> cannot split, whole-sequence level,         products potential broad-spectrum antibacterial <br /> so prove impossible develop drug                   drug targets. We assessed proteins from <br /> which targets IF-1 pathogens. This              S. aureus, E. coli O157:H7 EDL993 M. tuber- <br /> necessarily overly concerned with,             culosis criteria distribution, essentiality <br /> however, curing disease important               involvement virulence. All proteins from <br /> than preserving commensal flora.                           organisms ranked order of <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                 Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  324                                                                                                T. A. White D. B. Kell <br />  <br />  <br />  <br /> suitability potential drug target. It                                          75 IF1C Amborella trichopoda <br />                                                                                                     IF1C Cabomba caroliniana <br /> shown small proportion                                                      IF1C Cornus mas <br />                                                                                                         IF1C Epifagus virginiana <br /> gene products genomes make                                           57 66 IF1C Leucophyllum frutescens <br />                                                                                                   IF1C Hedera helix <br />                                                                                         77 IF1C Garrya elliptica <br /> useful drug targets, rank highly                                               IF1C Brexia madagascariensis <br />                                                                                                 IF1C Zea mays <br /> so fairly independently scoring used.                               68 98 IF1C Oryza sativa <br />                                                                                          61 IF1C Triticum aestivum <br /> From rankings                                                 IF1C Nicotiana tabacum <br />                                                                                                IF1C Arabidopsis thaliana <br /> do number proteins rank highly                                                IF1C Pinus thunbergii <br />                                                                                                                                  IF1Y Homo sapiens <br />                                                                                                                                  IF1X Homo sapiens <br /> or different scoring systems used,                                                                          99 <br />                                                                                                                              71 IF1H Homo sapiens <br />                                                                                           IF1C Nephroselmis olivacea <br /> they rank highly genomes used.                                       IF1C Psilotum nudum <br />                                                                                      IF1C Marchantia polymorpha <br /> These targets described fur-                                     IF1C Chaetosphaeridium globosum <br />                                                                                       IF1C Spirogyra maxima <br /> ther left suggestions                                    IF1C Mesostigma viride <br />                                                                                62 IF1 Synechocystis sp. (strain PCC 6803) <br /> in-depth analysis.                                                              81 IF1 Synechococcus elongatus <br />                                                                                 64 IF1 Anabaena sp. (strain PCC 7120) <br />                                                                                             IF1C Chlorella vulgaris <br />                                                                                    IF1 Aquifex aeolicus <br />                                                                              56      IF1 Thermoanaerobacter tengcongensis <br /> Table 8. List pathogenic bacteria (as defined                                 IF1 Thermotoga maritima <br />                                                                                       IF1 Bifidobacterium longum <br /> study) possess homologues infA                              IF1 Streptomyces coelicolor <br /> gene expectation value used                                                       IF1 Tropheryma whipplei <br />                                                                              55 IF1 Corynebacterium efficiens <br />                                                                                 73 <br />                                                                               67 IF1 Corynebacterium glutamicum <br />  <br />                                Neisseria meningitidis serogroup B&#226;&#710;&#8212; <br />                                                                                61         IF1    Mycobacterium tuberculosis <br /> Acinetobacter calcoaceticus                                                     95 IF1 Mycobacterium leprae <br />                                                                                              IF1 Fusobacterium nucleatum (subsp. nucleatum) <br /> Bacillus cereus                Neisseria meningitidis serogroup C                    IF1 Chlorobium tepidum <br /> Bordetella pertussis           Propionibacterium acnes                         75 <br />                                                                                        99 IF1 Ureaplasma parvum <br />                                                                                                    IF1 Ureaplasma urealyticum <br /> Brucella abortus               Proteus mirabilis                                                      IF1 Mycoplasma penetrans <br />                                                                                                 IF1 Mycoplasma gallisepticum <br /> Clostridium botulinum          Providencia rettgeri                              63                      IF1 Mycoplasma pneumoniae <br />                                                                                                             IF1 Mycoplasma genitalium <br /> Clostridium tetani&#226;&#710;&#8212; <br />                                                                                                98 <br />                                Providencia stuartii                                    99 IF1 Helicobacter pylori <br />                                                                                  89               IF1 Helicobacter pylori J99 <br /> Corynebacterium diptheriae     Salmonella cholerae-suis                                         IF1 Campylobacter jejuni <br />                                                                                        IF1 1Ralstonia solanacearum <br /> Enterococcus faecium           Salmonella enteritidis                          91              IF1 Neisseria meningitidis (serogroup A) <br /> Escherichia coli O111:H&#226;&#710;&#8217;       Shigella dysenteriae                           98 <br />                                                                                      IF1 Leptospira interrogans <br />                                                                                         IF1 Clostridium perfringens <br /> Escherichia coli O127:H6       Staphylococcus capitis                               IF1 Clostridium acetobutylicum <br />                                                                                   IF1 Bacillus halodurans <br /> Flavobacterium                 Staphylococcus saprophyticus                              IF1 Staphylococcus aureus (strain N315) <br />                                                                                 67 IF1 Staphylococcus aureus (strain Mu50) <br />                                                                                94 <br /> meningosepticum                Streptococcus agalactiae                                  IF1 Staphylococcus aureus (strain MW2) <br />  <br />                                Streptococcus agalactiae serotype III&#226;&#710;&#8212; <br />                                                                                           IF1 Staphylococcus epidermidis <br /> Francisella tularensis                                                                 IF1 Bacillus anthracis <br /> Haemophilus parainfluenzae     Streptococcus pyogenes serotype M18&#226;&#710;&#8212;                IF1 Oceanobacillus iheyensis <br />                                                                              57 IF1 Bacillus subtilis <br />                                                                               54         IF1 Listeria monocytogenes <br /> Klebsiella pneumoniae          Streptococcus pyogenes serotype M5                        IF1 Enterococcus faecalis <br /> Legionella pneumophila         Vibrio parahaemolyticus&#226;&#710;&#8212;                                IF1 Lactococcus lactis (subsp. lactis) <br />                                                                                               IF1 Streptococcus mutans <br />                                Vibrio vulnificus&#226;&#710;&#8212; <br />                                                                                 79 <br /> Moraxella catarrhalis                                                             89 <br />                                                                                                IF1 Streptococcus pneumoniae <br />                                                                                               IF1 Streptococcus pyogenes (serotype M3) <br />                                                                                   57 <br /> Moraxella lacunata             Wolinella recta                                    59 <br />                                                                                               IF1 Streptococcus pyogenes <br />                                                                                               IF1 Streptococcus agalactiae (serotype V) <br /> Mycoplasma fermentans          Wolinella succinogenes                                      IF1 Escherichia coli O157:H7 <br />                                                                                      IF1 Escherichia coli <br /> Mycoplasma hominis             Xanthomonas maltophila                           53        IF1 Salmonella typhi <br />                                                                                           IF1 Shigella flexneri <br /> Neisseria gonorrhoeae                                                                     IF1 Escherichia coli O6 <br />                                                                                           IF1 Salmonella typhimurium <br />                                                                                              IF1 Yersinia pestis <br /> &#226;&#710;&#8212; Species/strains   genomes entirely sequenced.                              IF1 Vibrio cholerae <br />                                                                                                IF1 Haemophilus ducreyi <br />                                                                                              IF1 Haemophilus influenzae <br />                                                                                  62            IF1 Pasteurella multocida <br />                                                                                       IF1 Shewanella oneidensis <br />                                                                               69 IF1 Wigglesworthia glossinidia brevipalpis <br /> Figure 15. Neighbour-joining tree showing distribution                             IF1 Buchnera aphidicola (subsp. Schizaphis graminum) <br /> of infA pathogenic non-pathogenic bacteria                                     IF1 Pseudomonas aeruginosa <br />                                                                                               97 IF1 Chlamydia trachomatis <br /> eukaryotes. Sequences aligned ClustalX (Thompson                                  99           IF1 Chlamydia muridarum <br />                                                                                                        IF1 Chlamydia pneumoniae <br /> et al., 1997). The tree created using Mega 2.1                                            IF12 Ralstonia solanacearum <br />                                                                                   86 IF1 Xanthomonas axonopodis (pv. citri) <br /> using default parameters (Kumar et al., 2001). Branches                          91 IF1 Xanthomonas campestris(pv. campestris) <br />                                                                                              IF1 Xylella fastidiosa <br /> leading Gram-negative bacteria coloured blue,                            99 IF1 Xylella fastidiosa (strain Temecula1) <br />                                                                               55 99              IF1 Rickettsia prowazekii <br /> leading Gram-positive bacteria red, leading                                          IF1 Rickettsia conorii <br /> to Cyanobacteria dark green, leading                           74          IF1    Caulobacter crescentus <br />                                                                                            IF1 Rhizobium loti <br />                                                                                   90 <br /> eukaryotes light green. The tree rooted using RT12                                        IF1 Brucella suis <br />                                                                                                 IF1 Brucella melitensis <br /> from Petunia hybrida, Brassica napus Raphanus sativus                                    IF1 Rhizobium meliloti <br />                                                                                       65 IF1 Agrobacterium tumefaciens (strain C58) <br /> as outgroup (branches highlighted purple). infA                                 IF1 Borrelia burgdorferi <br />                                                                                            IF1 Treponema pallidum <br /> sequences bacteria treated pathogenic study                        IF1 Deinococcus radiodurans <br />                                                                                    IF1 Mycoplasma pulmonis <br /> are marked red diamond. Human sequences                        69                  IF1 Mycoplasma sp. <br /> marked green diamond. Numbers branches                                                 RT12 Petunia hybrida <br />                                                                                                  99 RT12 Brassica napus <br /> bootstrap support groupings based 100 replicates                                           93 RT12 Raphanus sativus <br />  <br /> (values &lt;50 shown). Scale bar shows number                   0.2 <br />  <br /> substitutions site <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                         Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                             325 <br />  <br />  <br />  <br />    A number known previously suggested                        Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. <br /> targets figure prominently overall                      Basic local alignment search tool. J Mol Biol 215: 403&#226;&#8364;&#8220;410. <br />                                                                      Altschul SF, Madden TL, Schaffer AA. 1997. Gapped BLAST <br /> ranking. This shows methods used this <br />                                                                        PSI-BLAST: new generation protein database search <br /> study successfully identify targets                     programs. Nucleic Acid Res 25: 3389&#226;&#8364;&#8220;3402. <br /> the potential useful effective treatment                Bauer E, Kohavi R. 1999. An empirical comparison voting <br /> of disease.                                                            classification algorithms: bagging, boosting variants. <br />    Through study, number proteins                       Machine Learning 36: 105&#226;&#8364;&#8220;139. <br /> been suggested entirely novel drug targets. How-                  Bouhss A, Mengin-Lecreulx D, Blanot D, van Heijenoort J, <br /> ever, proof post-genomics methods                         Parquet C. 1997. Invariant amino acids Mur peptide <br />                                                                        synthetases bacterial peptidoglycan synthesis modi- <br /> will successful development novel                       fication site-directed mutagenesis UDP&#226;&#8364;&#8220;MurNAc:L- <br /> drug. Indeed, scoring metrics attempt                      alanine ligase Escherichia coli . Biochemistry 36: <br /> to consider number &#226;&#8364;&#732;post-target-identification&#226;&#8364;&#8482;                   11 556&#226;&#8364;&#8220;11 563. <br /> criteria &#226;&#8364;&#732;ease assay&#226;&#8364;&#8482;. Indeed, &#226;&#8364;&#732;To date                Brunder W, Khan AS, Hacker J, Karch H. 2001. Novel type of <br /> antibacterial compounds identified target-based                     fimbriae encoded large plasmid sorbitol-fermenting <br /> screening advanced clinical testing,                    enterohemorrhagic Escherichia coli O157:H(&#226;&#8364;&#8220;). Infect Immun <br />                                                                        69: 4447&#226;&#8364;&#8220;4457. <br /> less used clinically treat bacterial infec- <br />                                                                      Buysse JM. 2001. The role genomics antibacterial target <br /> tions&#226;&#8364;&#8482; (Projan, 2002). However, argue                      discovery. Curr Med Chem 8: 1713&#226;&#8364;&#8220;1726. <br /> two candidates identified (trmD infA),                   Chittum HS, Champney WS. 1995. Erythromycin inhibits the <br /> as methods exploited herein, provide promising                     assembly large ribosomal subunit growing Escherichia <br /> strategies identification novel antimi-                     coli cells. Curr Microbiol 30: 273&#226;&#8364;&#8220;279. <br /> crobial targets, analysed iteratively                   Chopra I, Hesse L, O&#226;&#8364;&#8482;Neill AJ. 2002. Exploiting current under- <br /> via hypothesis driven methods (Kell Oliver,                        standing antibiotic action discovery new drugs. J Appl <br />                                                                        Microbiol 92(suppl): 4S&#226;&#8364;&#8220;15S. <br /> 2004). Whilst, course, having good target                    Coello Coello CA, van Veldhuizen DA, Lamont GB. 2002. <br /> hardly having good drug inhibits                    Evolutionary Algorithms Solving Multi-objective Problems. <br /> it, developments sensitivity-based scoring                      Kluwer Academic: Dordrecht. <br /> approach used virtual screening,                    Cummings HS, Hershey JW. 1994. Translation initiation factor <br /> in scoring targets &#226;&#8364;&#732;druggability&#226;&#8364;&#8482; (Hopkins                  IF1 essential cell viability Escherichia coli . J Bacteriol <br /> and Groom, 2002; Zambrowicz Sands, 2003)                           176: 198&#226;&#8364;&#8220;205. <br />                                                                      Dahlquist KD, Puglisi JD. 2000. Interaction translation <br /> and compounds &#226;&#8364;&#732;drug-like&#226;&#8364;&#8482; qualities beyond <br />                                                                        initiation factor IF1 E. coli ribosomal A site. J Mol <br /> the &#226;&#8364;&#732;rule 5&#226;&#8364;&#8482; (Lipinski et al., 2001). In sense,                Biol 299: 1&#226;&#8364;&#8220;15. <br /> multi-objective optimization methods (e.g.                  Dasgupta P, Chakrabarti PP, DeSarkar SC. 1999. Multiobjective <br /> Coello Coello et al., 2002; Dasgupta et al., 1999;                     Heuristic Search. Vieweg: Braunschweig. <br /> Deb, 2001; Knowles et al., 2001; Zitzler, 1999)                      Davies J. 1994. Inactivation antibiotics dissemination <br /> might used advantage.                                       resistance genes. Science 264: 375&#226;&#8364;&#8220;382. <br />                                                                      Deb K. 2001. Multi-objective Optimization Using Evolutionary <br />                                                                        Algorithms. Wiley: Chichester. <br /> Acknowledgements                                                     Dietterich TG. 2000. An experimental comparison three <br /> DBK thanks BBSRC, EPSRC, NERC                              methods constructing ensembles decision trees: bagging, <br />                                                                        boosting, randomization. Machine Learning 40: 139&#226;&#8364;&#8220;157. <br /> RSC financial support. <br />                                                                      Dougherty TJ, Barrett JF, Pucci MJ. 2002. Microbial genomics <br />                                                                        novel antibiotic discovery: new technology search for <br />                                                                        new drugs. Curr Pharm Des 8: 1119&#226;&#8364;&#8220;1135. <br /> References <br />                                                                      Dunman PM, Murphy E, Haney S, et al. 2001. Transcription <br />                                                                        profiling-based identification Staphylococcus aureus genes <br /> Ahn HJ, Kim HW, Yoon HJ, et al. 2003. Crystal structure of <br />   tRNA(m1G37)methyltransferase: insights tRNA recogni-            regulated agr and/or sarA loci. J Bacteriol 183: <br />   tion. EMBO J 22: 2593&#226;&#8364;&#8220;2603.                                          7341&#226;&#8364;&#8220;7353. <br /> Alksne LE. 2002. Virulence target antimicrobial             Egebjerg J, Douthwaite S, Garrett RA. 1989. Antibiotic interac- <br />   chemotherapy. Expert Opin Investig Drugs 11: 1149&#226;&#8364;&#8220;1159.              tions GTPase-associated centre Escherichia coli <br /> Allsop A, Illingworth R. 2002. The impact genomics related      23S rRNA. EMBO J 8: 607&#226;&#8364;&#8220;611. <br />   technologies search new antibiotics. J Appl Microbiol   El Zoeiby A, Sanschagrin F, Levesque RC. 2003. Structure and <br />   92: 7&#226;&#8364;&#8220;12.                                                            function Mur enzymes: development novel inhibitors. <br /> Allsop AE. 1998. New antibiotic discovery, novel screens, novel        Mol Microbiol 47: 1&#226;&#8364;&#8220;12. <br />   targets impact microbial genomics. Curr Opin Microbiol      Fell DA. 1996. Understanding Control Metabolism. Portland <br />   1: 530&#226;&#8364;&#8220;534.                                                          Press: London. <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                           Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  326                                                                                                  T. A. White D. B. Kell <br />  <br />  <br />  <br /> Forsyth RA, Haselbeck RJ, Ohlsen KL, et al. 2002. A genome-           Kumar S, Tamura K, Jakobsen IB, Nei M. 2001. MEGA2: molec- <br />    wide strategy identification essential genes           ular evolutionary genetics analysis software. Bioinformatics 17: <br />    Staphylococcus aureus. Mol Microbiol 43: 1387&#226;&#8364;&#8220;1400.                  1244&#226;&#8364;&#8220;1245. <br /> Giaever G, Shoemaker DD, Jones TW, et al. 1999. Genomic               Lin AH, Murray RW, Vidmar TJ, Marotti KR. 1997. The <br />    profiling drug sensitivities induced haploinsufficiency.      oxazolidinone eperezolid binds 50S ribosomal subunit <br />    Nature Genet 21: 278&#226;&#8364;&#8220;283.                                            competes binding chloramphenicol lincomycin. <br /> Gillies D. 1996. Artificial Intelligence Scientific Method.         Antimicrob Agents Chemother 41: 2127&#226;&#8364;&#8220;2131. <br />    Oxford University Press: Oxford.                                   Lipinski CA, Lombardo F, Dominy BW, Feeney PJ. 2001. <br /> Glass JI, Belanger AE, Robertson GT. 2002. Streptococcus                Experimental computational approaches estimate <br />    pneumoniae genomics platform broad-spectrum                 solubility permeability drug discovery development <br />    antibiotic discovery. Curr Opin Microbiol 5: 338&#226;&#8364;&#8220;342.                settings. Adv Drug Deliv Rev 46: 3&#226;&#8364;&#8220;26. <br /> Gribskov M. 1992. Translational initiation factors IF-1 eIF-      Marmor S, Petersen CP, Reck F, et al. 2001. Biochemical <br />    2&#206;&#177;share RNA-binding motif prokaryotic ribosomal              characterization phosphinate inhibitor Escherichia coli <br />    protein S1 polynucleotide phosphorylase. Gene 119: <br />                                                                         MurC. Biochemistry 40: 12 207&#226;&#8364;&#8220;12 214. <br />    107&#226;&#8364;&#8220;111. <br />                                                                       McDevitt D, Rosenberg M. 2001. Exploiting genomics discover <br /> Hancock REW, Knowles D. 1998. Are approaching end of <br />                                                                         new antibiotics. Trends Microbiol 9: 611&#226;&#8364;&#8220;617. <br />    antibiotic era? Curr Opin Microbiol 1: 493&#226;&#8364;&#8220;494. <br />                                                                       Mitchell TM. 1997. Machine Learning. McGraw Hill: London. <br /> Haney SA, Alksne LE, Dunman PM, Murphy E, Projan SJ. <br />                                                                       Mjolsness E, DeCoste D. 2001. Machine learning science: state <br />    2002. Genomics anti-infective drug discovery &#226;&#8364;&#8221; getting to <br />    endgame. Curr Pharm Des 8: 1099&#226;&#8364;&#8220;1118.                                art future prospects. Science 293: 2051&#226;&#8364;&#8220;2055. <br /> Hastie T, Tibshirani R, Friedman J. 2001. The Elements             Neu HC, Gootz TD. 1996. Antimicrobial chemotherapy. In <br />    Statistical Learning: Data Mining, Inference Prediction.         Medical Microbiology, Baron S (ed.). University Texas <br />    Springer-Verlag: Berlin.                                             Medical Branch. <br /> Heinemann JA. 1999. How antibiotics cause antibiotic resistance.      Oneyama C, Nakano H, Sharma SV. 2002. UCS15A, novel <br />    Drug Discov Today 4: 72&#226;&#8364;&#8220;79.                                          small molecule, SH3 domain-mediated protein&#226;&#8364;&#8220;protein interac- <br /> Heinrich R, Schuster S. 1996. The Regulation Cellular Systems.       tion blocking drug. Oncogene 21: 2037&#226;&#8364;&#8220;2050. <br />    Chapman &amp; Hall: London.                                            Paulmurugan R, Massoud TF, Huang J, Gambhir SS. 2004. <br /> Hoffmaster AR, Koehler TM. 1999. Control virulence gene              Molecular imaging drug-modulated protein&#226;&#8364;&#8220;protein interac- <br />    expression Bacillus anthracis. J Appl Microbiol 87: 279&#226;&#8364;&#8220;281.      tions living subjects. Cancer Res 64: 2113&#226;&#8364;&#8220;2119. <br /> Hopkins AL, Groom CR. 2002. The druggable genome. Nat Rev             Payne DJ, Holmes DJ, Rosenberg M. 2001a. Delivering novel <br />    Drug Discov 1: 727&#226;&#8364;&#8220;730.                                              targets antibiotics genomics. Curr Opin Investig Drugs <br /> Inoue R, Kaito C, Tanabe M, et al. 2001. Genetic identification         2: 1028&#226;&#8364;&#8220;1034. <br />    distinct DNA polymerases, DnaE PolC,           Payne DJ, Warren PV, Holmes DJ, Ji Y, Lonsdale JT. 2001b. <br />    essential chromosomal DNA replication Staphylococcus          Bacterial fatty-acid biosynthesis: genomics-driven target for <br />    aureus. Mol Genet Genom 266: 564&#226;&#8364;&#8220;571.                                antibacterial drug discovery. Drug Discov Today 6: 537&#226;&#8364;&#8220;544. <br /> Isaacson RE. 2002. Genomics prospects discovery       Projan SJ. 2002. New (and new) antibacterial tar- <br />    new targets antibacterial antifungal agents. Curr         gets &#226;&#8364;&#8221; novel drugs come? Curr <br />    Pharm Des 8: 1091&#226;&#8364;&#8220;1098.                                              Opin Pharmacol 2: 513&#226;&#8364;&#8220;522. <br /> Ji Y. 2002. The role genomics discovery novel targets    Sassetti CM, Boyd DH, Rubin EJ. 2003. Genes required for <br />    antibiotic therapy. Pharmacogenomics 3: 315&#226;&#8364;&#8220;323.                 mycobacterial growth defined high density mutagenesis. Mol <br /> Kell DB, Oliver SG. 2004. Here evidence,             Microbiol 48: 77&#226;&#8364;&#8220;84. <br />    hypothesis? The complementary roles inductive           Schmid MB. 1998. Novel approaches discovery of <br />    hypothesis-driven science post-genomic era. Bioessays         antimicrobial agents. Curr Opin Chem Biol 2: 529&#226;&#8364;&#8220;534. <br />    26: 99&#226;&#8364;&#8220;105. <br />                                                                       Schnappinger D, Hillen W. 1996. Tetracyclines: antibiotic action, <br /> Kell DB, Westerhoff HV. 1986. Metabolic control theory: role <br />                                                                         uptake, resistance mechanisms. Arch Microbiol 165: <br />    microbiology biotechnology. FEMS Microbiol Rev 39: <br />                                                                         359&#226;&#8364;&#8220;369. <br />    305&#226;&#8364;&#8220;320. <br />                                                                       Sharma VK, Dean-Nystrom EA. 2003. Detection enterohemor- <br /> Knowles DJ, King F. 1998. The impact bacterial genomics on <br />                                                                         rhagic Escherichia coli O157:H7 using multiplex real-time <br />    antibacterial discovery. Adv Exp Med Biol 456: 183&#226;&#8364;&#8220;195. <br /> Knowles JD, Watson RA, Corne DW. 2001. Reducing local                   PCR assay genes encoding intimin Shiga toxins. Vet <br />    optima single-objective problems multi-objectivization. In     Microbiol 93: 247&#226;&#8364;&#8220;260. <br />    Proceedings 1st International Conference Evolutionary    Spaltmann F, Blunck M, Ziegelbauer K. 1999. Computer-aided <br />    Multi-criterion Optimization (EMO&#226;&#8364;&#8482;01), E. Zitzler, et al. (eds).     target selection-prioritizing targets antifungal drug discovery. <br />    Springer: Berlin; 269&#226;&#8364;&#8220;283.                                           Drug Discov Today 4: 17&#226;&#8364;&#8220;26. <br /> Kobayashi K, Ehrlich SD, Albertini A, et al. 2003. Essential          Stephenson K, Hoch JA. 2002. Two-component phosphorelay <br />    Bacillus subtilis genes. Proc Natl Acad Sci USA 100:                 signal-transduction systems therapeutic targets. Curr Opin <br />    4678&#226;&#8364;&#8220;4683.                                                           Pharmacol 2: 507&#226;&#8364;&#8220;512. <br /> Koehler TM. 2002. Bacillus anthracis genetics virulence gene      Stephenson K, Hoch JA. 2004. Developing inhibitors selec- <br />    regulation. Curr Top Microbiol Immunol 271: 143&#226;&#8364;&#8220;164.                 tively target two-component phosphorelay signal transduc- <br /> Kornder JD. 2002. Streptomycin revisited: molecular action       tion systems pathogenic microorganisms. Curr Med Chem 11: <br />    microbial cell. Med Hypoth 58: 34&#226;&#8364;&#8220;46.                                765&#226;&#8364;&#8220;773. <br />  <br />  <br /> Copyright &#239;&#163;&#169; 2004 John Wiley &amp; Sons, Ltd.                                                            Comp Funct Genom 2004; 5: 304&#226;&#8364;&#8220;327. <br />  Assessment novel antibacterial targets                                                                                            327 <br />  <br />  <br /> Struelens MJ. 1998. The epidemiology antimicrobial resistance        Wang G, Clark CG, Rodgers FG. 2002. Detection Escherichia <br />    hospital acquired infections: problems possible solutions.      coli genes encoding major virulence factors, genes <br />    Br Med J 317: 652&#226;&#8364;&#8220;654.                                                 defining O157:H7 serotype, components type <br /> Stuber K, Frey J, Burnens AP, Kuhnert P. 2003. Detection type          2 Shiga toxin family multiplex PCR. J Clin Microbiol 40: <br />    III secretion genes general indicator bacterial virulence.     3613&#226;&#8364;&#8220;3619. <br />    Mol Cell Probes 17: 25&#226;&#8364;&#8220;32.                                           Weber W, Anke T, Bross M, Steglich W. 1990. Strobilurin D <br /> Sun YH, Bakshi S, Chalmers R, Tang CM. 2000. Functional                   strobilurin F: new cytostatic antifungal (E)-&#206;&#178;- <br />    genomics Neisseria meningitidis pathogenesis. Nature Med            methoxyacrylate antibiotics Cyphellopsis anomala (1). <br />    6: 1269&#226;&#8364;&#8220;1273.                                                          Planta Med 56: 446&#226;&#8364;&#8220;450. <br /> Tanner ME, Vaganay S, van Heijenoort J, Blanot D. 1996.                 Willins DA, Kessler M, Walker SS, Reyes GR, Cottarel G. 2002. <br />    Phosphinate inhibitors D-glutamic acid-adding enzyme            Genomics strategies antifungal drug discovery &#226;&#8364;&#8221; from <br />    peptidoglycan biosynthesis. J Org Chem 61: 1756&#226;&#8364;&#8220;1760.               gene discovery compound screening. Curr Pharm Des 8: <br /> Terst <br /> </body></html>