<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br /> Characterization of mammalian RAD51 double <br /> strand break repair using non-lethal dominant- <br /> negative forms <br />  <br />  <br />  <br />                                                                  homologous recombination, initiated by a DNA strand <br />    invasion step. In yeast, conservative recombination is thus <br />    dependent on ScRAD51. In contrast, SSA is RAD51 <br />  <br />  <br />                                                                  independent (Ivanov et al., 1996). Thus, in mammalian <br />                                    cells, the comprehension of DSB repair regulation pre- <br />                                                                  supposes the precise determination of the different path- <br /> In contrast <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> yeast RAD51, mammalian mRAD51 is                  ways involved. <br /> an essential gene. Its role in double strand break                  The search for RAD51 orthologues has led to the <br /> (DSB) repair and its consequences on cell viability              identi&#194;&#174;cation of a growing number of mammalian RAD51 <br /> remain to be characterized precisely. Here, we used a            homologues (for a review see Thacker, 1999). This <br /> hamster cell line carrying tandem repeat sequences               suggests either redundancy of functions or the splitting <br /> with an I-SceI cleavage site. We characterized con-              up of ScRAD51&apos;s role on several mammalian genes, or an <br /> servative recombination after I-SceI cleavage as gene            evolutionary divergence of function for some of the <br /> conversion or intrachromatid crossing over associated            homologues. In this context, the mammalian mRAD51 <br /> with random reintegration of the excised reciprocal              presents both similarities to and differences <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> product. We identi&#194;&#174;ed two dominant-negative RAD51                ScRAD51, and its exact role in DSB repair remains to be <br /> forms that speci&#194;&#174;cally inhibit conservative recombin-            established. Human Rad51 is able to promote DNA strand <br /> ation: the yeast ScRAD51 or the yeast&#194;&#177;mouse chimera              exchange in vitro, but less ef&#194;&#174;ciently and with some <br /> SMRAD51. In contrast, the mouse MmRAD51 stimu-                   biochemical differences compared with the yeast protein <br /> lates conservative recombination. None of these                  (Baumann et al., 1996; Gupta et al., 1997; Benson et al., <br /> RAD51 forms affects non-conservative recombination               1998). In vivo, chromosome breakages and a decrease in <br /> or global DSB healing. Consistently, although resist-            sister chromatid exchanges have also been reported prior <br /> ance to g-rays remains unaffected, MmRAD51                       to cell death in rad51&#194;&#177;/&#194;&#177; depleted DT40 cells, a chicken <br /> stimulates whereas ScRAD51 or SMRAD51 prevents                   immortalized B-cell line harbouring a particularly high <br /> radiation-induced recombination. This suggests that              frequency of spontaneous gene conversion (Sonoda et al., <br /> mRAD51 does not signi&#194;&#174;cantly affect the global DSB               1998, 1999). Moreover, mouse Rad51 protein was shown <br /> repair ef&#194;&#174;ciency but controls the classes of recombin-           to associate with mouse meiotic chromosomes and to <br /> ation events. Finally, both ScRAD51 and SMRAD51                  relocalize to nuclear foci in cells treated with genotoxic <br /> drastically inhibit spontaneous recombination but not            agents (Haaf et al., 1995; Barlow et al., 1997). However, <br /> cell proliferation, showing that RAD51-dependent                 contrasting with the situation in yeast and bacteria, <br /> spontaneous and DSB-induced conservative recombin-               mRAD51 has been described to be involved in cell <br /> ation can be impaired signi&#194;&#174;cantly without affecting             proliferation and is an essential gene in vertebrates since <br /> cell viability.                                                  <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>null mutants</span> are not viable (Tsuzuki et al., 1996; Sonoda <br /> Keywords: double strand break repair/homologous                  et al., 1998). Moreover, mRAD51 participates, via BRCA2, <br /> recombination/mammalian cells/RAD51/trans-species                in the regulation of p53 transactivation activity <br /> dominant-negative allele                                         (Marmorstein et al., 1998). Since the transactivation <br />                                                                  activity of p53 is involved in cell cycle control but is not <br />                                                                  correlated with the role of p53 in recombination <br />                                                                  (Dudenhoffer et al., 1999; Saintigny et al., 1999), this <br /> Introduction                                                     result may suggest the participation of mRAD51 in the cell <br /> A DNA double strand break (DSB) is a lesion that can be          cycle checkpoint controlled by p53 transactivation <br /> produced by genotoxic agents such as ionizing radiation or       activity. Thus, chromosomal aberrations observed in <br /> that can occur during physiological processes such as yeast      rad51&#194;&#177;/&#194;&#177; depleted cells could also be explained by cell <br /> meiosis (Sun et al., 1989). Repair of DSBs is essential to       cycle defects. <br /> ensure genome integrity and cell viability, but may also            Several hypotheses could account for these apparent <br /> result in genome rearrangements. Two major processes             paradoxes. The &#194;&#174;rst hypothesis is that mRAD51 controls <br /> can repair DSBs: non-homologous end-joining (NHEJ) or            one DSB repair pathway essential for cell viability, as <br /> a homology-directed (HD) process that takes advantage of         proposed for the chicken DT40 lines (Sonoda et al., 1998, <br /> a homologous sequence to repair the DSB (Liang et al.,           1999; Morrison et al., 1999). The second hypothesis is that <br /> 1998; for reviews see Baumann and West, 1998; Kanaar             mRAD51 possesses two distinct roles: a non-essential role <br /> and Hoeijmakers, 1998). Actually an HD process refers to         controlling DSB repair and another essential for cell <br /> two independent pathways: the non-conservative single            viability. A third hypothesis is that mRAD51 is involved <br /> strand annealing (SSA) process that occurs between direct        in an essential function but not in homologous <br />  <br /> 3090                                                                              &#195;&#163; European Molecular Biology Organization <br />                                                                               Mammalian RAD51 double-strand break repair <br />  <br />  <br /> recombination, while other homologues of RAD51 would           acting with endogenous mRAD51, leading to the poisoning <br /> be required for homologous recombination. This latter          of the mRAD51 nucleoprotein &#194;&#174;lament via the extra <br /> hypothesis is based on the facts that a growing number of      N-terminal part of the yeast protein. Another possibility <br /> RAD51 homologues are being reported and that some of           could be that ScRAD51 would lead to the titration of the <br /> them, such as XRCC2 and XRCC3, are involved in HD              endogenous recombination complex. To check these <br /> DSB repair (Johnson et al., 1999; Pierce et al., 1999).        hypotheses, we also expressed the chimera SMRad51 <br />    To determine whether mRAD51 actually acts on DSB            composed of the 55 N-terminal amino acids from ScRad51 <br /> repair in mammalian cells and to characterize its putative     fused to the entire MmRad51, and DNtScRad51 corres- <br /> role precisely, we used the CHO cell line DRA10 carrying       ponding to ScRad51 with its 55 N-terminal amino acids <br /> a recombinant substrate with a unique site for the rare        deleted (see Figure 1A). <br /> cutting endonuclease I-SceI (Liang et al., 1998). We show        These different RAD51 forms are expressed in the <br /> that overexpression of mouse MmRAD51 does not sub-             parental CHO-DRA10 line (Liang et al., 1998), and the <br /> stantially stimulate the overall DSB repair but speci&#194;&#174;cally    names of the derivative cell lines are listed in Figure 1A. <br /> stimulates the conservative recombination DSB repair           Figure 1B shows the expression of the endogenous and <br /> pathway. More importantly, we also identi&#194;&#174;ed forms of          exogenous RAD51s, veri&#194;&#174;ed by western blotting. <br /> RAD51 (the yeast ScRAD51 or the yeast&#194;&#177;mouse chimera            Expression of DNtScRAD51 has been veri&#194;&#174;ed by RT&#194;&#177; <br /> SMRAD51) that speci&#194;&#174;cally inhibit DSB-induced conser-          PCR (data not shown). <br /> vative recombination as well as spontaneous recombina- <br /> tion. None of the RAD51 forms used here affect SSA. <br /> These results indicate that mRAD51 participates in DSB <br /> repair, does not modify the global ef&#194;&#174;ciency of DSB repair     Characterization of the DSB-induced <br /> but acts on the balance controlling the different classes of   recombination events <br /> events: conservative versus non-conservative recombin-         The CHO-DRA10 cell line and its derivatives carry a <br /> ation. Finally, our results show that it is possible to        unique recombination substrate with the cleavage site for <br /> decrease the ef&#194;&#174;ciency of RAD51 DSB repair and RAD51           the rare-cutting endonuclease I-SceI (see Figure 2). <br /> spontaneous recombination substantially without affecting      Transient expression of I-SceI ef&#194;&#174;ciently produces a <br /> cell viability and proliferation.                              DSB in the recombination substrate (Liang et al., 1998). <br />                                                                   Single-drug resistance to G418 (NeoR) monitors re- <br />                                                                combinant clones arising by an HD recombination <br />                                                                process, both conservative and non-conservative (SSA) <br />                                                                events (Figure 2). Double resistance to both G418 (NeoR) <br /> Results <br />                                                                and hygromycin (HygR) monitors only conservative events <br /> Strategy and cell lines used                                   due to gene conversion not associated with crossing over <br /> In order to de&#194;&#174;ne the DSB repair pathway controlled by         or associated with crossing over, but with a faithful <br /> mRAD51 precisely, we measured the impact of the                reintegration of the excised HygR sequence (Figure 2B). <br /> overexpression of different forms of RAD51 on recombin-        NeoR&#194;&#177;Hygs monitors SSA events plus crossing over with <br /> ation induced either by an acute DSB targeted to the           loss of the pop out circle (POC) (Figure 2A). <br /> recombination substrate or by ionizing radiation. It has          To demonstrate the occurrence of the recombination <br /> been shown that the overexpression of mRAD51 in                products predicted in Figure 2, we analysed the molecular <br /> mammalian cells results in an increase in spontaneous          structure of the recombination locus from NeoR&#194;&#177;HygR <br /> recombination between intrachromosomal repeat                  clones by Southern blotting (Figure 3A). We analysed 11 <br /> sequences (Vispe&#195;&#8218; et al., 1998; Arnaudeau et al., 1999;        NeoR&#194;&#177;HygR recombinant clones, seven from the control <br /> Huang et al., 1999). However, it has been argued that          line Cm3 and four from the MmRAD51-overexpressing <br /> overexpression of RAD51 interferes with DNA metabol-           line Rm2 (Figure 3). With the S2neo sequence probe <br /> ism and results in DNA lesions that could induce               (probe A), nine clones show the gene conversion 4 kb band <br /> recombination, as has been pointed out (Huang et al.,          and two recombinant clones show the 1.1 kb deletion event <br /> 1999). Importantly, the role of mRAD51 in DSB repair and       band (Figure 3B), despite the fact that these two clones <br /> the putative pathways involved have not been addressed in      exhibit a hygromycin-resistant <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. We thus probed <br /> mammalian cells.                                               the same &#194;&#174;lters with the hygromycin resistance gene <br />    In addition, in an attempt to identify a dominant-          sequence (probe B). The structure of the nine gene <br /> negative RAD51 form, we expressed ScRAD51 from                 conversion events previously identi&#194;&#174;ed is con&#194;&#174;rmed, since <br /> Saccharomyces cerevisiae. Trans-species dominant-              they show the 4 kb restriction band with this new probe <br /> negative effects have been described between RAD52             (Figure 3C). In the two deleted clones, however, the <br /> from Kluyveromyces lactis and S.cerevisiae (Milne and          presence of the hygromycin resistance gene sequence is <br /> Weaver, 1993). More speci&#194;&#174;cally, the active species for        revealed but at a different location (Figure 3C), suggesting <br /> recombination is the RecA/Rad51 nucleoprotein &#194;&#174;lament          a random integration after a crossing over event. These <br /> (Radding, 1991). Alignment of mouse MmRad51 and                results show that NeoR&#194;&#177;HygR clones actually score gene <br /> ScRad51 shows that the yeast protein is longer than that of    conversion and some crossing over events, both initiated <br /> the mouse, with a block of 55 additional amino acids at the    by strand invasion (Szostak et al., 1983). In addition, these <br /> N-terminal extremity (Figure 1A). Furthermore, ScRAD51         results demonstrate the existence of the predicted DNA <br /> interacts with the human RAD51 and XRCC3 (Schild et al.,       fragment excised by an intrachromatid crossing over (see <br /> 2000). Our hypothesis was that ScRAD51 may act as a            Figure 2B), followed by its random integration into the <br /> dominant-negative allele in mammalian cells by inter-          genome. <br />  <br />                                                                                                                       3091 <br />  S.Lambert and B.S.Lopez <br />  <br />  <br />  <br />  <br /> Fig. 1. Overexpression of the different RAD51 genes in hamster cells. (A) Structure of the RAD51 genes used. The boxes correspond to the <br /> homologous region between the different RAD51 genes: grey, mouse MmRAD51; white, yeast ScRAD51. ScRAD51 is longer at the N-terminal part. <br /> The white box corresponds to the region homologous to MmRAD51 and the hatched box to the extra sequence. The numbers correspond to the <br /> positions of the amino acids. The numbers in black correspond to the yeast amino acids and the numbers in grey correspond to the mouse amino <br /> acids. The chimeric protein SMRad51 was constructed with the N-terminal part of ScRad51 (hatched box) fused to the entire MmRad51 (white box). <br /> DNtScRad51 corresponds to ScRad51 with 55 N-terminal amino acids deleted. These proteins are expressed in the CHO-DRA10 line, and the <br /> corresponding derivative lines are listed on the right. (B). Overexpression of exogenous RAD51. Protein extracts were obtained from stable <br /> transfectants. The expression of MmRAD51 and SMRAD51 was veri&#194;&#174;ed by western blotting using an antibody raised against the human Rad51 protein <br /> and normalized with an anti-actin antibody. Rm2 and 4 correspond to two independent stable transfectants overexpressing MmRAD51. SMRad51-1 <br /> and SMRad51-2 correspond to two independent stable transfectants overexpressing the fusion SMRAD51. Cm3 is a control clone corresponding to <br /> the CHO-DRA10 line transfected with the empty pCDNA3.1puro plasmid. Expression of ScRAD51 is visualized by western blotting using an anti- <br /> ScRad51 antibody. The &#194;&#174;rst lane corresponds to transient expression (Tr) of ScRAD51 (used as control). The second lane corresponds to extracts <br /> from mock-transfected cells (CHO-DRA10). The third and fourth lanes correspond to two independent clones with stable expression of ScRAD51. <br /> MmRad51 is Rad51 from Mus musculus; ScRad51 is Rad51 from S.cerevisiae; CgRad51 is Rad51 from Cricetellus griseus; and SMrad51 is a <br /> chimeric fusion protein MmRad51&#194;&#177;ScRad51. <br />  <br />  <br />  <br /> RAD51 speci&#194;&#174;cally affects conservative                                    contrast, the frequency of NeoR&#194;&#177;HygR colonies is <br /> recombination events in mammalian cells                                   decreased 48 times by expression of ScRAD51. The <br /> We measured here the impact of the overexpression of the                  chimera SMRAD51 also exhibits a pronounced dominant- <br /> different RAD51 forms (Figure 1) on recombination                         negative effect since inhibition of the occurrence of <br /> induced by an I-SceI DSB, targeted to the recombination                   NeoR&#194;&#177;HygR colonies is decreased 42- to 127-fold <br /> substrate.                                                                compared with the control line Cm3 (Table I). <br />    We &#194;&#174;rst veri&#194;&#174;ed that the transfection ef&#194;&#174;ciency of the                    The percentage of conservative events (double-resistant <br /> I-SceI expression vector was equivalent in the control lines              NeoR&#194;&#177;HygR colonies) relative to all the recombinant <br /> and the overexpressing RAD51 cell lines (data not shown).                 colonies (NeoR alone) was examined (Figure 4). Since <br /> We also measured the same 100- to 1000-fold increase in                   this value is normalized to the frequency of NeoR colonies <br /> the frequency of G418-resistant colonies (total recombin-                 (representing the whole recombinant population), the <br /> ant clones) following transient transfection of the I-SceI                calculation is based on an internal standard and is <br /> expression vector (Liang et al., 1998). As compared with                  independent of transfection and cleavage ef&#194;&#174;ciencies. In <br /> the control cell line (Cm3), increases in the frequency of                the Cm3 control line, the frequency of conservative events <br /> G418-resistant clones by factors of 1.8&#194;&#177;2.6 are found in                  (NeoR&#194;&#177;HygR colonies) is 27%, and thus the non-conserva- <br /> lines expressing MmRAD51 and decreases in the fre-                        tive events represent 73% of the total recombinant <br /> quency of G418-resistant clones by factors of 1.7&#194;&#177;2.4 in                  colonies. Comparable ratios are obtained with the parental <br /> lines expressing ScRAD51 or SMRAD51 (Table I).                            CHO-DRA10 line (Liang et al., 1998). In the lines <br />    The impact of overexpression of the different RAD51                    overexpressing the MmRAD51 gene, the relative propor- <br /> forms is much stronger on the frequency of NeoR&#194;&#177;HygR                      tion of NeoR&#194;&#177;HygR colonies rises to 65&#194;&#177;72% (Figure 4). In <br /> recombinants. Indeed, overexpression of MmRAD51                           contrast, the percentage of NeoR&#194;&#177;HygR colonies falls to <br /> stimulates the frequency of NeoR&#194;&#177;HygR colonies by 4&#194;&#177;5                     2% in the line expressing ScRAD51, and to 1.6 or 0.9% in <br /> times compared with the control line Cm3 (Table I). In                    lines expressing SMRAD51 (Figure 4). The expression of <br />  <br /> 3092 <br />                                                                                            Mammalian RAD51 double-strand break repair <br />  <br />  <br />  <br />  <br /> Fig. 2. Strategy used to measure DSB-induced recombination events. <br /> After the formation of a DSB induced by I-SceI, two processes can <br /> compete. (A) The non-conservative SSA that produces NeoR and <br /> hygromycin-sensitive recombinants. (B) Homologous recombination <br /> events leading to gene conversion either with or without crossing over. <br /> The products of gene conversion events are resistant to both G418 and <br /> hygromycin (NeoR&#194;&#177;HygR). Intrachromatid crossing over leads to the <br /> formation of POCs. If the POCs are eliminated, the products are only <br /> NeoR; if the POC is reintegrated into the genome, it could result in a <br /> NeoR&#194;&#177;HygR colony. Thus, double resistance (NeoR&#194;&#177;HygR) scores only <br /> conservative recombination events. Not shown are unequal sister <br /> chromatid exchange events in which the HygR segregates from the <br /> NeoR gene. <br />                                                                           Fig. 3. (A) Restriction <span id='am-159' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of recombinant clones induced by an <br />                                                                           I-SceI-generated DSB. Probe A corresponds to the S2neo promoter <br /> the deleted DNtScRAD51 has no effect on the distribution                  sequence and probe B to part of the hygromycin-resistant gene, located <br />                                                                           between the two neo cassettes. The corresponding expected XhoI&#194;&#177; <br /> of the different events, compared with the control lines                  HindIII sizes (with both probes) and the corresponding resistance <br /> (Figure 4). These results demonstrate a dominant-negative                 phenotype are indicated. POC: pop out circle (see Figure 2). <br /> effect of ScRAD51 and of the chimera SMRAD51 with                         (B) Southern blot analysis with probe A. The sizes of the bands are <br /> respect to the homologous recombination initiated by                      indicated on the sides of the panels. Left panel: seven independent <br />                                                                           NeoR&#194;&#177;HygR clones (numbered on the top of the panel) from the <br /> strand invasion. In addition, the dominant-negative effect                control cell line Cm3. Right panel: four independent NeoR&#194;&#177;HygR <br /> requires the 55 N-terminal amino acids of the ScRAD51                     clones (numbered on the top of the panel) from the Rm2 cell line, <br /> product.                                                                  overexpressing MmRAD51. (C) The same &#194;&#174;lters as in (B) but <br />    The present data agree completely with the view that                   hybridized to probe B, characteristic of the intervening sequence. <br /> DSBs can be repaired by NHEJ, by a conservative strand <br /> invasion mechanism or by SSA. Indeed, expression of <br /> MmRAD51, ScRAD51 or SMRAD51 only very moderately                          different RAD51 forms on radiation resistance and on <br /> affects the total frequency of recombinants, indicating that              radiation-induced recombination. <br /> the ratio of NHEJ to HD repair is not modi&#194;&#174;ed signi&#194;&#174;-                        None of the different RAD51 forms modify the radiation <br /> cantly. In addition, overexpression of MmRAD51 speci&#194;&#174;-                    resistance of the recipient cell lines in the dose range tested <br /> cally increases the occurrence of conservative events,                    (Figure 5A). These results are consistent with our previous <br /> whereas overexpression of ScRAD51 or of the chimera                       I-SceI experiments showing that mRAD51 does not affect <br /> SMRAD51 prevents conservative events. Thus, the                           the ef&#194;&#174;ciency of global DSB healing but controls the <br /> different RAD51 forms do not affect the SSA process                       classes of recombination events. Indeed, radiation-induced <br /> itself, but modify the ratio of conservative events to SSA.               recombination is strongly dependent on the type of <br />                                                                           exogenous RAD51 expressed: MmRAD51 expression <br />                                                                           stimulates recombination frequency 10- to 20-fold after a <br /> RAD51 controls radiation-induced recombination                            dose of 6 Gy, whilst SMRAD51 as well as ScRAD51 <br /> but not radiation resistance                                              completely abolish the induction of radiation-induced <br /> In the above experiments, RAD51 does not substantially                    recombination (Figure 5B). These results show that in an <br /> affect the total ef&#194;&#174;ciency of DSB healing but rather the                  asynchronous cell population of mammalian cells, radi- <br /> ratio of the different recombination classes. Since DSB is                ation resistance is a RAD51-independent process whereas <br /> one of the main lethal lesions induced by ionizing                        radiation-induced recombination is a RAD51-dependent <br /> radiation, we measured the impact of expression of                        process. <br />  <br />                                                                                                                                          3093 <br />  S.Lambert and B.S.Lopez <br />  <br />  <br /> Table I. Effect of different Rad51 proteins on DSB-induced recombination <br /> Cell line                 Mean NeoR frequency                 Stimulation factorb              Mean NeoR&#194;&#177;HygR                     Stimulation factorb <br />                           (3 10&#194;&#177;3)a                                                            frequency (3 10&#194;&#177;3)a <br />  <br /> Cm3                       1.19 6 0.2                                                           0.38 6 0.1 <br /> Rm2                       3.09 6 0.7                          2.6                              2    6 0.4                           5.3 <br /> Rm4                       2.1 6 0.9                           1.8                              1.59 6 0.9                           4.2 <br />                                                               Inhibition factorb                                                  Inhibition factorb <br /> ScRad51-3                 0.64 6 0.4                          1.8                              0.008 6 0.005                       47.5 <br /> SMRad51-1                 0.71 6 0.1                          1.7                              0.009 6 0.007                       42.2 <br /> SMRad51-2                 0.5 6 0.2                           2.4                              0.003 6 0.002                      126.7 <br /> aMean   values from three independent experiments. <br /> bCompared   with the Cm3 cell line control. <br />  <br />  <br />  <br />  <br /> Fig. 4. Effect of the overexpression of MmRAD51, ScRAD51 or SMRAD51 on the frequency of conservative recombination events. Control refers <br /> to the parental line transfected with empty expression vector (Cm3). MmRAD51 refers to two independent transfectants overexpressing MmRAD51; <br /> ScRAD51 refers to one transfectant expressing yeast ScRAD51; SMRad51-1 and SMRad51-2 refer to two independent clones expressing the chimera <br /> SMRAD51; ScRad51DNt-1 and ScRad51DNt-23 refer to two independent clones expressing the deleted DNtScRAD51. Values correspond to the <br /> percentage of double-resistant (NeoR&#194;&#177;HygR) colonies (black boxes) and single-resistant (NeoR) colonies (grey boxes). The percentage of conservative <br /> recombination events is calculated from the ratio of the number of NeoR&#194;&#177;HygR clones to the NeoR clone frequency. The percentages are indicated on <br /> the top of the corresponding histograms. <br />  <br />  <br />  <br />  <br /> Spontaneous recombination is affected by                                         Overexpression of mRAD51 has previously been <br /> ScRAD51 and SMRAD51                                                           reported to increase spontaneous recombination (Vispe&#195;&#8218; <br /> RAD51 is an essential gene in non-irradiated mammalian                        et al., 1998; Arnaudeau et al., 1999; Huang et al., 1999). <br /> cells. It has been proposed that spontaneous recombination                    However, none of these studies used a &#194;&#175;uctuation analysis <br /> controlled by mRAD51 would be essential for cell viability                    to measure the process. In order to measure spontaneous <br /> by repairing spontaneous damage occurring during repli-                       recombination in an exponentially growing population, we <br /> cation; this would also be different from yeast. However,                     performed &#194;&#175;uctuation analysis using the Luria and <br /> the above results show that RAD51 recombinational DSB                         Delbruck assay (Luria and Delbruck, 1943), adapted by <br /> repair can be decreased substantially without modifying                       Capizzi and Jameson (1973). As expected, we con&#194;&#174;rmed <br /> the cell viability after radiation. Taking into account our                   that MmRAD51 stimulates spontaneous recombination 3- <br /> results, the proposed hypothesis should thus consider that                    to 4-fold per cell per generation (Figure 6A). <br /> spontaneous and DSB-induced recombination are two                                In contrast to MmRAD51, ScRAD51 and the chimera <br /> separable mechanisms involving mRAD51. We thus tested                         SMRAD51 strongly decrease the spontaneous recombin- <br /> whether spontaneous recombination can also be inhibited                       ation frequency in mammalian cell lines, in an interspecies <br /> by ScRAD51 and SMRAD51, as is the case for DSB-                               dominant-negative manner (Figure 6B). The recombin- <br /> induced recombination.                                                        ation inhibition is so pronounced that the recombination <br />  <br /> 3094 <br />                                                                                                  Mammalian RAD51 double-strand break repair <br />  <br />  <br />  <br />  <br /> Fig. 5. Effect of RAD51 on radiation resistance (A) and on radiation-induced recombination (B). Cells were irradiated at the doses indicated. Controls <br /> correspond to the parental CHO-DRA10 line and to Cm3, which is CHO-DRA10 transfected with the empty expression vector. Rm2 and Rm4 are <br /> two independent clones overexpressing MmRAD51. ScRad51-3 and ScRad51-6 are two independent clones expressing ScRAD51. Radiation-induced <br /> recombination (B): the values correspond to the number of NeoR in 106 surviving irradiated cells, following subtraction of the number of NeoR in <br /> 106 non-irradiated cells. <br />  <br />  <br />  <br /> rate cannot be calculated by the Luria and Delbruck test. <br /> Indeed, most of the cultures contain no recombinant <br /> colonies. Consequently, ScRAD51 as well as SMRAD51 <br /> also inhibit spontaneous recombination in growing cells, <br /> in a dominant-negative manner. <br />    Finally, despite the fact that spontaneous recombination <br /> is almost totally defective, neither the generation time, the <br /> plating ef&#194;&#174;ciency nor the cell cycle measured by &#194;&#175;ow <br /> cytometry (data not shown) are modi&#194;&#174;ed in the different <br /> lines expressing ScRAD51 or SMRAD51, showing that cell <br /> viability and proliferation are unaffected in these lines. <br />  <br /> Discussion <br /> The different lines devised here derive from the same <br /> parental line. Thus, recombination frequencies are calcu- <br /> lated for one copy of substrate located at the same locus in <br /> each cell line. Using these lines, we have determined the <br /> precise pathway involving mRAD51 for DSB repair in                             Fig. 6. Effect of the overexpression of the different RAD51 <br /> mammalian cells: it acts speci&#194;&#174;cally on conservative                           forms on spontaneous recombination. (A) MmRAD51 stimulates <br /> recombination and does not affect non-conservative SSA.                        the spontaneous recombination rate measured by &#194;&#175;uctuation analysis <br /> This phenotype is similar to that of ScRAD51 in yeast,                         using the Luria and Delbruck test (Luria and Delbruck, 1943). The <br />                                                                                names of the different lines are indicated under the histograms. <br /> despite the numerous differences between the yeast and                         DRA10, parental line; Cm3 (control line), DRA10 transfected <br /> mammalian RAD51 protein products. However, an import-                          with empty expression vector; Rm2 and Rm4 correspond to two <br /> ant difference from the yeast model is that in mammalian                       independent clones overexpressing MmRAD51. The number of <br /> cells, mRAD51 DSB repair can be decreased substantially                        independent cultures for each line is reported under the histograms. <br /> without signi&#194;&#174;cant effects on the global DSB repair                            (B) ScRAD51 and SMRAD51 inhibit the frequency of spontaneous <br />                                                                                recombination. A recombination rate cannot be calculated because <br /> ef&#194;&#174;ciency. Indeed, in yeast, alteration of the RAD51                           most of the cultures do not contain any recombinant. The frequency <br /> pathway leads to a drastic decrease in the global DSB                          of recombination reported here corresponds to the sum of the <br /> repair ef&#194;&#174;ciency. In mammalian cells, a balanced regula-                       recombinants from all the independent cultures in relation to the total <br /> tion maintains the total level of DSB repair ef&#194;&#174;ciency, but                    number of cells summed from all the independent cultures. DRA10 <br />                                                                                is the parental line; ScRad51-3 and ScRad51-6 correspond to two <br /> modi&#194;&#174;es the class of recombination events.                                     independent clones expressing ScRAD51; SMRad51-1, SMRad51-2 and <br />    Southern blot analysis of DSB-induced NeoR&#194;&#177;HygR                             SMRad51-24 refer to three independent clones expressing the chimera <br /> clones shows the existence of both gene conversion and                         SMRAD51. <br />  <br />  <br />                                                                                                                                                 3095 <br />  S.Lambert and B.S.Lopez <br />  <br />  <br /> crossing over events, the former being detected more <br /> frequently in the system used here. The restriction patterns <br /> of some clones are compatible with an intrachromatid <br /> crossing over followed by a random reintegration of the <br /> excised DNA. Although the putative circular nature of the <br /> intermediate excised DNA remains to be established, the <br /> present data demonstrate the genome rearrangement <br /> predicted by the intrachromatid crossing over model (see <br /> Figure 2). Such processes have important biological <br /> consequences with regard to genome stability/variability. <br /> For example, POCs created by intrachromatid crossing <br /> over in rDNA repeats are involved in cell ageing in yeast <br /> (Sinclair and Guarente, 1997). In addition, random inte- <br /> gration of an excised fragment generated by a DSB- <br /> induced intrachromatid crossing over can inactivate a <br />                                                                 Fig. 7. Role of RAD51 in DSB repair in mammalian cells. On a DSB, <br /> recipient integration locus. Taking into account the high       Ku and RAD52 compete to process the DSB. Ku orientates DSB repair <br /> number of homologous repeat sequences dispersed                 towards NHEJ. Conversely, RAD52 orientates repair to an HD process <br /> throughout the genome, the process described here could         that can be either SSA or homologous recombination. In the absence <br /> correspond to an actual mutagenic mechanism, particu-           of RAD51, SSA takes place. This is the most frequent reaction in the <br /> larly after treatments generating DSBs, such as ionizing        parental lines. If mRAD51 is present, it can bind to RAD52 and channel <br />                                                                 the reaction to homologous recombination initiated by strand invasion. <br /> radiation. <br />    When inducing one DSB, targeted to one repeat of <br /> the duplication, the overexpression of MmRAD51 only <br /> slightly stimulates the frequency of total recombinant          an NHEJ process (Van Dyck et al., 1999). Our results <br /> clones (NeoR). However, the ratio of NeoR&#194;&#177;HygR to NeoR          suggest that RAD51 does not substantially affect the <br /> clones is increased several-fold. These results demonstrate     channelling of the NHEJ versus the HD repair process, but <br /> that MmRAD51 speci&#194;&#174;cally increases conservative re-             that inside the HD pathway (the RAD52 pathway), RAD51 <br /> combination events and not SSA events (most of the NeoR         may favour the strand invasion over the SSA recombin- <br /> clones). In yeast, ScRAD51 acts in a complex comprising         ation pathway. This observation is consistent with the fact <br /> the components of the RAD52 epistasis group (Milne and          that mRAD51 does not modify radiation sensitivity since <br /> Weaver, 1993; Hays et al., 1995; Johnson and Symington,         the global ef&#194;&#174;ciency of DSB healing remains essentially <br /> 1995; Rattray and Symington, 1995). In mammalian cells,         unaffected. Moreover, the fact that the status of RAD51 <br /> homologues of most of these genes have been described.          affects the extent of radiation-induced recombination is <br /> In addition, mRAD51 interacts with another set of proteins      compatible with the hypothesis of a channelling role for <br /> without homologues in yeast, such as the tumour suppres-        mRAD51 towards the conservative recombination path- <br /> sor and cell cycle control proteins p53, BRCA1 and              way. The simplest explanation would be that RAD51 <br /> BRCA2 (Sturzbecher et al., 1996; Buchhop et al., 1997;          constitutes a limiting factor for the homologous recombin- <br /> Mizuta et al., 1997; Scully et al., 1997; Marmorstein et al.,   ation pathway. RAD52 would bind the DSB and would <br /> 1998). In line with this, p53 is involved in homologous         predominantly channel repair to SSA, the most ef&#194;&#174;cient <br /> recombination independently of its role in G1/S transition      HD process in these experiments (Liang et al., 1998; this <br /> and of its transactivation activity (Dudenhoffer et al.,        study). Alternatively, RAD52 could load mRAD51 on the <br /> 1999; Saintigny et al., 1999), a result compatible with a       broken DNA, resulting in the channelling of DSB repair <br /> role for p53 in the homologous recombination process. In        towards a conservative homologous recombination pro- <br /> addition, BRCA1 has been shown to be involved in DSB            cess (Figure 7). With such a hypothesis, increasing the <br /> repair in mammalian cells (Moynahan et al., 1999). Taken        intracellular amount of mRAD51 should favour the <br /> together, these data suggest that mRAD51 acts in one (or        channelling of the repair to strand invasion conservative <br /> several) highly elaborated complex(es). This raises the         events, as is actually the case in our experiments. <br /> question as to how overexpression of only one component            Expression of ScRAD51 speci&#194;&#174;cally inhibits conserva- <br /> of such (a) complex(es) can stimulate the whole recom-          tive events in mammalian cells. The 55 N-terminal amino <br /> bination process. The present results suggest that RAD51 is     acids of the protein appear to be essential for this speci&#194;&#174;c <br /> a limiting factor for homologous recombination in mam-          dominant-negative effect. Indeed, ScRad51 lacking these <br /> malian cells. Recombination stimulation by overexpres-          N-terminal amino acids (DNtScRad51) loses its dominant- <br /> sion of RAD52, another component of the complex, has            negative effect. In contrast, the fusion of these 55 amino <br /> also been described in mammalian cells (Park, 1995).            acids to mouse MmRad51 (SMRad51) confers a speci&#194;&#174;c <br /> However, it is not known whether both conservative              dominant-negative effect to the chimera. Our hypothesis <br /> recombination and SSA events are stimulated since               was that ScRAD51 molecules would be incorporated into <br /> RAD52 also promotes SSA (Fishman-Lobell et al., 1992;           the mRAD51 single-stranded DNA nucleoprotein &#194;&#174;lament, <br /> Mortensen et al., 1996; Van Dyck et al., 1999). Thus, it        generally considered as the active intermediate in the <br /> cannot be excluded that this observation in mammalian           homologous recombination process (Radding, 1989); <br /> cells refers to stimulation of SSA events. Consistent with      indeed, ScRAD51 has been shown to interact with <br /> this hypothesis, it has been suggested that Rad52 and Ku        human RAD51 (Schild et al., 2000). However, our <br /> proteins compete for binding to DSBs, leading to the            hypothesis was also that the divergences between the <br /> channelling of the DSB repair to an HD DSB repair or to         two kinds of RAD51 molecules would then poison the <br />  <br /> 3096 <br />                                                                               Mammalian RAD51 double-strand break repair <br />  <br />  <br /> nucleoprotein &#194;&#174;lament and inhibit the RAD51-dependent         intermediates would not accumulate and on the other hand <br /> recombination. Consistent with this hypothesis is the fact    DSBs would be repaired by another pathway such as <br /> that the chimera SMRAD51 speci&#194;&#174;cally inhibits recombin-       NHEJ. In this hypothesis, cells should not be radiation <br /> ation initiated by a strand invasion, whereas the wild-type   sensitive. We do no have direct evidence that SMRAD51 <br /> MmRAD51 stimulates it. An alternative explanation would       acts at initiation or at a later step of strand invasion. <br /> be that the remaining homologies between the protein          However, our results are compatible with the channelling <br /> products of mRAD51 and ScRAD51 are suf&#194;&#174;cient to               towards SSA or NHEJ for DSB repair. In addition, our <br /> promote interactions with some of mRAD51&apos;s partners, but      results and the third hypothesis are fully consistent with <br /> that the divergence would impair the formation of a           the fact that no RAD51 mutant alleles have been isolated in <br /> functional recombination complex. ScRAD51 could thus          the screen for radiation-sensitive mutants in CHO cell <br /> titrate the components of the recombination complex. Our      lines. <br /> results do not argue in favour of this second hypothesis.        RAD51 is an essential gene in non-irradiated mamma- <br /> Indeed, the N-terminal part of ScRad51 is essential to        lian cells. It has been proposed that spontaneous <br /> poison mammalian homologous recombination, but this           recombination controlled by mRAD51 would be essential <br /> part of the protein is absent in its mammalian orthologue;    for cell viability by repairing spontaneous damage <br /> in contrast, the rest of the yeast protein (amino acids 55&#194;&#177;   occurring during replication. Remarkably, the present <br /> 400) is unable to titrate the mammalian complex, despite      results show that ef&#194;&#174;cient mRAD51 recombination repair <br /> its strong sequence homologies to mRad51 that should          of DSBs induced by I-SceI or ionizing radiation is not <br /> favour interactions with endogenous mRad51&apos;s partners.        required for cell viability. This could suggest that DSB- <br /> In addition, interactions at least with Rad52 have been       induced recombination and cell viability are two separable <br /> reported to be species speci&#194;&#174;c (Shen et al., 1996).           functions of mRAD51. It can be argued that the essential <br /> Nevertheless, from the genetic point of view, ScRAD51         role of mRAD51 is spontaneous recombination. This does <br /> as well as SMRAD51 are dominant-negative alleles              not agree with our results showing that both DSB-induced <br /> speci&#194;&#174;c for homologous recombination in mammalian             recombination and spontaneous recombination are stimu- <br /> cells; they constitute useful and universal tools to devise   lated by MmRAD51 and both are inhibited by ScRAD51 or <br /> cell lines speci&#194;&#174;cally defective in conservative recombin-    SMRAD51. However, since the endogenous wild-type <br /> ation without affecting cell proliferation and viability.     RAD51 is still present in our cell lines, we cannot exclude <br />    XRCC2 and XRCC3 proteins show homologies to                the possibility that low or undetectable levels of <br /> Rad51 and they interact with each other and with Rad51        recombination would be suf&#194;&#174;cient to ensure its putative <br /> (Schild et al., 2000). Cell lines mutant for XRCC2 or         role in cell proliferation and viability. Nevertheless, the <br /> XRCC3 are de&#194;&#174;cient in recombinational DSB repair              ef&#194;&#174;ciency of the mRAD51 recombination pathway, <br /> (Johnson et al., 1999; Pierce et al., 1999). Moreover,        involved in spontaneous as well as DSB-induced <br /> these XRCC2 or XRCC3 mutant lines are sensitive to            conservative recombination, can be decreased substan- <br /> ionizing radiation, correlating the radiation resistance to   tially without affecting cell proliferation and viability. <br /> the ef&#194;&#174;ciency of recombination (Jones et al., 1987; Liu       However, since deletion of the RAD51 gene is lethal in <br /> et al., 1998). In contrast, this correlation does not exist   mammalian cells, this could suggest that mRAD51 has <br /> with our lines expressing the SMRAD51 (or ScRAD51)            other role(s), essential to cell viability. Its interactions with <br /> dominant-negative form. Several hypotheses could              p53, BRCA1 and BRCA2 (Sturzbecher et al., 1996; <br /> account for these apparent differences. First, our lines      Buchhop et al., 1997; Mizuta et al., 1997; Scully et al., <br /> contain an endogenous wild-type RAD51, and one can            1997; Marmorstein et al., 1998) and its participation in the <br /> suggest that low or undetectable levels of homologous         regulation of p53 transactivation activity (Marmorstein <br /> recombination would be suf&#194;&#174;cient to maintain radiation        et al., 1998) suggest a potential role in the cell cycle <br /> resistance; however, radiation-induced recombination is       checkpoint (beside potential roles in DNA repair), which <br /> strongly decreased. Secondly, one of the other paralogues     could be a good candidate for this new putative essential <br /> can substitute for RAD51 for radiation resistance but not     function of RAD51. Indeed, no good structural homo- <br /> for radiation-induced recombination; they would be            logues of the mRAD51 interactors p53, BRCA1 and <br /> unable to substitute for XRCC2 or XRCC3. These two            BRCA2 have been identi&#194;&#174;ed in yeast; in addition, both <br /> hypotheses uncouple radiation resistance and homologous       null BRCA&#194;&#177;/&#194;&#177; mice and null RAD51&#194;&#177;/&#194;&#177; mice show embry- <br /> recombination ef&#194;&#174;ciency. A third hypothesis reconciles all    onic lethality that can be partially rescued by inactivation <br /> the different results. This hypothesis proposes that RAD51    of the p53 gene (Lim and Hasty, 1996; Hakem et al., <br /> initiates the recombination process and that its partners,    1997). Another possible essential role could be the <br /> such as XRCC2 or XRCC3, facilitate the completion of the      protective effect of RAD51 against apoptosis (Huang <br /> process by maturing and resolving the recombination           et al., 1999). A connection between these two roles of <br /> intermediates generated by RAD51. Non-processed inter-        RAD51 cannot be excluded. <br /> mediates would result in cell toxicity. We show here that        A growing amount of evidence connects mRAD51 to <br /> radiation-induced recombination is a RAD51-dependent          tumorigenesis: mRAD51 interacts with tumour suppressor <br /> pathway; thus, radiation stimulates the initiation of         gene products and, in addition, mutations or loss of <br /> recombination by RAD51. The absence of XRCC2 or               heterozygosity have been reported in some cancers for <br /> XRCC3 would lead to the accumulation of non-processed         chromosomal loci containing BRCA1, BRCA2 and <br /> intermediates and would result in radiation sensitivity of    RAD51B or mRAD51&apos;s partners RAD52, RAD54 and <br /> the XRCC2 or XRCC3 mutant cell lines. If the initiation of    RAD54B (Gonzalez et al., 1999; Hiramoto et al., 1999; <br /> homologous recombination is impaired by the dominant-         Matsuda et al., 1999; Schoenmakers et al., 1999). Our <br /> negative SMRAD51, on the one hand toxic recombination         results show that cell viability and proliferation remain <br />  <br />                                                                                                                         3097 <br />  S.Lambert and B.S.Lopez <br />  <br />  <br /> unaffected in the absence of an ef&#194;&#174;cient mRAD51-                              Recombination frequency after g-radiation. Cells were irradiated in <br />                                                                               PBS, using a 60Co irradiator (2.5 Gy/min) at the dose indicated. After <br /> dependent spontaneous and DSB-induced recombination <br />                                                                               irradiation, the cells were incubated in DMEM at 37&#194;&#176;C for 24 h. The cells <br /> process, but that mRAD51 participates in DSB repair by                        were then trypsinized, counted and divided into two fractions. The &#194;&#174;rst <br /> controlling the types of recombination events, i.e.                           fraction was used to calculate the viability by measuring the plating <br /> conservative recombination. This function should confer                       ef&#194;&#174;ciency. The second fraction was plated under G418 selection to <br /> an important role on mRAD51 in the maintenance of                             measure the recombination frequency. <br />                                                                                  Fluctuation analysis for spontaneous recombination was performed as <br /> genetic stability of vegetative cells beside an essential                     previously described (Liskay et al., 1984). For each line analysed, several <br /> function in cell proliferation.                                               independent cultures were plated and cultured to con&#194;&#175;uence. Cells were <br />                                                                               then trypsinized, counted and one fraction was used for plating ef&#194;&#174;ciency <br />                                                                               estimation. The remaining cells were plated under G418 selection. The <br />                                                                               resulting number of TK+ or Neo+ clones allowed us to calculate the <br /> Materials and methods                                                         recombination frequency. The rate of recombination per cell per <br />                                                                               generation was calculated by using the &#194;&#175;uctuation tests of Luria and <br /> DNA manipulations                                                             Delbruck (Luria and Delbruck, 1943; Capizzi and Jameson, 1973). <br /> All DNA manipulations were performed as described (Sambrook et al., <br /> 1989). <br />                                                                              <br /> Cells and plasmids <br /> CHO-DRA10 cells (Liang et al., 1998) and their derivative lines were         <br /> cultured at 37&#194;&#176;C with 5% CO2 in Dulbecco&apos;s modi&#194;&#174;ed Eagle&apos;s medium            <br /> (DMEM) supplemented with 10% fetal bovine serum. NeoR clones were          <br /> selected in 500 mg/ml G418, and hygromycin-resistant clones were           <br /> selected in 500 mg/ml hygromycin. Transfections were performed using         <br /> Fugene 6 (Boehringer, Mannheim). The SMRAD51 cDNA was con-                 <br /> structed as follows: the N-terminal part of ScRAD51 was ampli&#194;&#174;ed by            <br /> PCR using a 5&#194;&#162; primer containing an EcoRI restriction site and the             <br /> beginning of the yeast gene (ccggaattcATGtctcaagaacaacat), and a 3&#194;&#162; <br /> primer containing the sequence corresponding to the last amino acids of <br /> the N-terminal part of ScRAD51 and the beginning of MmRAD51 cDNA              <br /> up to the AvaII site (and without the &#194;&#174;rst ATG). The sequence of this 3&#194;&#162; <br /> oligonucleotide is: gctgtggaccaaaactttcttcttccactgaagtatctgcacttgcttcaa-      <br /> gctgcatttgcatagcgccgttggt ggcctcaatatc. The rest of the MmRAD51                <br /> sequence was ampli&#194;&#174;ed using a 5&#194;&#162; primer at the AvaII site (ttttgg-             <br /> tccacagcctatttcaccgc) and a 3&#194;&#162; primer at the end of the MmRAD51 cDNA   <br /> and containing a XhoI site (cggccgctcgagggagtccagtctttggcatcgccc).         <br /> Double digests of the PCR products with EcoRI&#194;&#177;AvaII and AvaII&#194;&#177;XhoI          <br /> were co-cloned in pBluescript vector digested by EcoRI&#194;&#177;XhoI.                  <br /> MmRAD51, ScRAD51 and the fusion SMRAD51 cDNAs were cloned in                 <br /> pcDNA3.1puro plasmid. This plasmid was constructed by replacing the          <br /> 1.7 kb PvuII fragment of pcDNA3.1Zeo (In Vitrogen) by the 1.4 kb        <br /> PvuII&#194;&#177;BamHI fragment from pPuro plasmid (Clontech). We used the          <br /> I-SceI (HA-tagged) expression vector described by Liang et al. (1998).   <br /> The deleted DNtScRAD51 was constructed as follows: the cDNA coding          <br /> for amino acids 55&#194;&#177;400 was ampli&#194;&#174;ed by PCR using an intact ScRAD51             <br /> cDNA as matrix, a 5&#194;&#162; primer (cggccgctcgagccaccatgtccggcgatggtggcgga)   <br /> and a 3&#194;&#162; primer (atcaccaaatacctactcgtcttc). The amplimer was digested         <br /> with XhoI and cloned in pBluescript vector. After veri&#194;&#174;cation of the          <br /> sequence, an XhoI&#194;&#177;EcoRV fragment was cloned in PcDNA3.1puro.                    <br />                                                                               <br /> Western blot analysis                                                           <br /> All extract preparation steps were performed at 4&#194;&#176;C. After washing with          <br /> phosphate-buffered saline (PBS), cells were suspended in lysis buffer A       <br /> [25 mM Tris pH 7.5, 5 mM EDTA, 600 mM NaCl, 1 mM dithiothreitol                <br /> (DTT), 0.1% NP-40, 5 mg/ml leupeptin, 2 mM pepstatin, 1 mM                    <br /> phenylmethylsulfonyl &#194;&#175;uoride (PMSF), 10% glycerol] and incubated              <br /> for 40 min on ice. Extracts were centrifuged for 30 min at 15 000 g, the        <br /> supernatant was retrieved and the protein concentration was determined          <br /> using a Bio-Rad Protein Assay (Bio-Rad). Forty micrograms per well of         <br /> the boiled samples were loaded on a 10% polyacrylamide gel in the             <br /> presence of SDS. After migration, the proteins were electrotransferred         <br /> onto a nitrocellulose membrane and probed with speci&#194;&#174;c antibodies: anti-   <br /> human Rad51 and anti-actin (Sigma). Standard procedures were used for        <br /> the electrophoresis, transfer and western blotting. Antibodies were      <br /> visualized using the ECL detection kit (Amersham).                         <br />                                                                        <br /> Recombination measurements                                                     <br /> Recombination after induction of a DSB. A total of 3 3 105 cells (for the        <br /> Rm2, Rm4, DRA10 and Cm3 lines) and up to 1.8 3 106 cells (for the ScR      <br /> and SM lines) were plated and transfected with 2 and 12 mg, respectively,  <br /> of an expression vector for the I-SceI endonuclease (pCMV I-SceI). At   <br /> 24 h post-transfection, G418 or G418/hygromycin selection was added.       <br /> The NeoR and NeoR&#194;&#177;HygR colony frequencies are expressed in relation to  <br /> the total number of cells plated. The relative percentage of conservative   <br /> recombination events is calculated as the ratio of the frequency of double-     <br /> resistant NeoR&#194;&#177;HygR clones to the frequency of Neo clones.                     <br />  <br /> </body></html>