<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Candida albicans pescadillo homolog is required <br /> for normal hypha-<span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>-yeast morphogenesis and <br /> yeast proliferation <br />  <br /> A single species, Candida albicans, causes half of all invasive fungal             We propose that the filament-to-yeast switch is a critical part <br /> infections in humans. Unlike other fungal pathogens, this organism              of the life cycle of C. albicans, because yeast cells fulfill specific <br /> switches between growth as budding yeast and as pseudohyphal                    roles during colonization and infection. Yeast possess suitable <br /> and hyphal filaments in host organs and in vitro. Both cell types               dimensions and physical properties to provide the fungus facile <br /> play a role in invasive disease: while hyphal and pseudohyphal                  access to the host&#226;&#8364;&#8482;s bloodstream. While cultured endothelial cells <br /> filaments penetrate host cells and tissues, yeast cells are likely to           <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> human umbilical cord veins phagocytose hyphae (12), <br /> facilitate dissemination through the bloodstream and establish-                 other types of endothelial cells phagocytose yeast. Endocytosis <br /> ment of distant foci of infection. Many regulators of the yeast-to-             of yeast by endothelial cells was observed in cells of the human <br /> hypha switch have emerged from intensive investigations of this                 blood brain barrier (13), endothelium of porcine whole blood <br /> morphogenetic step, but the hypha-to-yeast switch remains poorly                vessel strips (14), and bovine aortic endothelial cells (15). In <br /> understood. Using a forward genetic approach, a novel putative                  addition to their function in bloodborne dispersal during invasive <br /> regulator involved in the hypha-to-yeast switch was identified, the             disease, C. albicans yeast cells have been found to play distinct <br /> C. albicans pescadillo homolog, PES1. In eukaryotes from yeast to               roles from hyphal cells in the interaction with the host immune <br /> human, pescadillo homologs are involved in cell cycle control and               system. Exposure of human monocytes to C. albicans yeast <br /> ribosome biogenesis, and are essential. We find a pescadillo                    blocks their development into dendritic cells, a critical class of <br /> homolog to act in fungal morphogenesis, specifically in lateral                 antigen-presenting cells (16). The ability to manipulate specific <br /> yeast growth on filamentous cells. We also find essentiality of PES1            features of the host immune system by its different cell types is <br /> in C. albicans to be dependent on cell type, because hyphal cells,              likely to contribute to the success of C. albicans in maintaining <br /> but not yeast cells, tolerate its loss. PES1 is therefore critical for          its commensal status and in invasive disease. <br /> completion of the C. albicans life cycle, in which the fungus                      So far, two genes are known to affect lateral yeast growth: <br /> switches between filamentous and yeast growth. Consistent with                  mutants in PDE2 have decreased, and those in CAP1 have <br /> these in vitro findings, PES1 is required for C. albicans virulence in          increased lateral yeast growth (9, 17). Given the dearth of studies <br /> an in vivo insect model of infection.                                           of the hypha-to-yeast switch, our goal was to identify novel <br />                                                                                 regulators of this switch. This search identified a hyperfilamen- <br /> filament &#229;&#8230;&#169; virulence &#229;&#8230;&#169; essential gene &#229;&#8230;&#169; cell cycle <br />                                                                                 tous mutant with a transposon insertion in the promoter of the <br />                                                                                 C. albicans pescadillo homolog, PES1. <br />                                                                                    Pescadillo was originally identified in a screen for genes <br /> I  n humans, Candida albicans causes more invasive disease than <br />    any other fungal species. Mortality from C. albicans blood- <br /> stream infections is &#226;&#172;&#8225;49% (1). In addition to its role as a <br />                                                                                 required for development of zebrafish embryos (18). In organ- <br />                                                                                 isms from yeast to human, the highly conserved homologs of this <br />                                                                                 gene participate in cell cycle control, DNA replication, and <br /> pathogen, C. albicans colonizes the mucous membranes of                         ribosome biogenesis (19, 20). In all organisms studied to date, <br /> 30&#226;&#8364;&#8220;60% of humans (2). How has C. albicans evolved as the most                   pescadillo homologs are essential (19, 20). However, the bio- <br /> successful human fungal commensal and pathogen? This fun-                       chemical activity of pescadillo homologs in proliferation, cell <br /> gus&#226;&#8364;&#8482;s dramatic and frequent switch between growth as ovoid                      cycle control, and ribosome biogenesis is unknown. <br /> yeast and as filamentous pseudohyphae and hyphae has long                          Here, we report that the C. albicans pescadillo homolog is <br /> been implicated in its success as a pathogen (3). Work from many                required for normal growth of cells induced as yeast, and for <br /> laboratories has elucidated regulatory mechanisms for the switch                normal production of lateral yeast on filaments. While a het- <br /> from yeast to filamentous growth, and the role of hyphal growth                 erozygote retaining an intact allele of the C. albicans pescadillo <br /> in virulence (3). Little is known about the reverse switch, from <br /> filamentous to yeast growth, although mutants defective in the <br /> hypha-to-yeast switch show reduced virulence in each case in                    Author contributions: L.E.C. and J.R.K. designed research; J.S., L.E.C., A.M.G., L.C., and J.R.K. <br />                                                                                 performed research; L.E.C. contributed new reagents/analytic tools; J.S., L.E.C., A.M.G., and <br /> which this trait was examined (4&#226;&#8364;&#8220;10).                                           J.R.K. analyzed data; and J.R.K. wrote the paper. <br />    The switch from yeast to hyphal growth is induced by specific <br />                                                                                 The authors declare no conflict of interest. <br /> conditions, such as low cell density, alkaline pH, high tem- <br />                                                                                 This article is a PNAS Direct Submission. <br /> perature, a poor carbon source, and serum. In yeast-inducing <br />                                                                                 1Present address: Department of Oncology, Novartis Institutes for Biomedical Research, 250 <br /> conditions, yeast do not produce hyphal cells. In contrast,                     Massachusetts Avenue, Cambridge, MA 02139. <br /> hyphal and pseudohyphal filaments constitutively produce                        2Present address: PhD Program, Biological and Biomedical Sciences, Harvard Medical <br /> yeast cells on their subapical segments, while apical segments                  School, Boston, MA 02120. <br /> continue to produce filamentous cells. We call these cells                      3To whom correspondence should be addressed. E-mail: julia.koehler@childrens. <br /> lateral yeast (11). Lateral yeast growth presumably occurs in                   harvard.edu. <br /> vivo and in vitro, because filamentous forms are practically                    This article contains supporting information online at www.pnas.org/cgi/content/full/ <br /> never seen without yeast forms in deep organs of individuals                    0809147105/DCSupplemental. <br /> with invasive candidiasis.                                                      &#194;&#169; 2008 by The National Academy of Sciences of the USA <br />  <br />                                                                                              <br />  <br />  <br />  <br />                                                                                                     <br /> Fig. 1. Lateral yeast are produced under conditions that induce hyphal tip <br /> growth and hyphal branching. Yeast cells of SC5314 (PES1/PES1) were inocu- <br /> lated into Spider medium, permitted to settle on a fibronectin-coated cover- <br /> slip, and incubated at 37 &#194;&#176;C for 6.5 h. Cells of the same culture on the same <br /> coverslip were photographed. White arrow, original yeast from which hyphal <br /> outgrowth occurred. Black arrow, hyphal branch or tip. Black star, lateral      Fig. 2. Hyphae-inducing conditions permit growth when expression of PES1 <br /> yeast. (Scale bar, 10 &#226;&#65533;&#174;m.)                                                      is repressed from tetO. Strains were grown overnight in YPD without doxy- <br />                                                                                 cycline at 30 &#194;&#176;C and diluted to an initial density of OD600 0.2. Fivefold dilutions <br />                                                                                 were spotted onto YPD (A), YPD with 20 &#226;&#65533;&#174;g/ml doxycycline (B), Spider (C), <br /> homolog is viable (21), a previous large-scale study found this                 Spider with 20 &#226;&#65533;&#174;g/ml doxycycline (D), 20% calf serum 2% glucose (E), and 20% <br /> gene to be essential in C. albicans (22). In contrast, we find that             calf serum 2% glucose with 20 &#226;&#65533;&#174;g/ml doxycycline (F). Plates were incubated at <br /> C. albicans cells tolerate loss of the pescadillo homolog under                 37 &#194;&#176;C for 4 days. (1) JKC915 (HIS1/his1::FRT-tetR PES1/PES1). (2) JKC956 <br />                                                                                 (HIS1/his1::FRT-tetR PES1/pes1::FRT). (3) JKC962 (HIS1/his1::FRT-tetR <br /> hyphae-inducing, but not under yeast-inducing conditions. De- <br />                                                                                 PES1/pes1::FRT). (4) JKC1137 (HIS1/his1::FRT-tetR pes1::FRT/FRT-tetO-PES1). (5) <br /> lineating the pathways in which PES1 functions in the hypha-                    JKC1143 (HIS1/his1::FRT-tetR pes1::FRT/FRT-tetO-PES1). <br /> to-yeast switch will begin to elucidate this step of the C. albicans <br /> life cycle. <br />                                                                                 number of eukaryotic pescadillo orthologs, including S. cerevisiae <br /> Results                                                                         YPH1. To examine whether the C. albicans pescadillo homolog <br /> Lateral Yeast Are Produced in Conditions That Induce Hyphal Tip                 can functionally substitute for loss of the essential gene YPH1, its <br /> Growth and Hyphal Branching. In all media tested, lateral yeast                 ability to rescue growth of S. cerevisiae yph1 mutants was tested <br /> were produced from subapical cells of both pseudohyphal and                     in two experiments. First, in a S. cerevisae yph1 deletion mutant <br /> hyphal filaments, whose apical cells were producing new fila-                   in the &#226;&#338;&#186;1278b genetic background, PES1 was able to substitute <br /> mentous cells. The same cultures contained filaments producing                  for YPH1 when expressed in either low- or high-copy number. <br /> hyphal branches (Fig. 1 and data not shown). Because in liquid                  Supporting information (SI) Fig. S1, published on the PNAS <br /> cultures, microenvironments of apical and subapical filamentous                 Web site, shows complementation of yph1&#226;&#338;&#172; by PES1 in low-copy <br /> cells are expected to be similar, we conclude that lateral yeast                number. Second, expression of PES1 from a high-copy plasmid <br /> growth occurs in hyphae-inducing conditions.                                    permitted growth of S. cerevisiae yph1&#226;&#338;&#172; temperature-sensitive <br />                                                                                 (ts) mutants in the W303 genetic background (19) at the <br /> A Hyperfilamentous Transposon Mutant in the C. albicans pescadillo <br />                                                                                 restrictive temperature (not shown). C. albicans PES1 can com- <br /> Homolog, PES1. We sought to identify novel regulators of lateral <br />                                                                                 plement loss of S. cerevisiae YPH1 in two genetic backgrounds, <br /> yeast growth. A previous forward genetic approach to filamen- <br />                                                                                 establishing that it is an ortholog of YPH1. <br /> tous growth successfully used haploinsufficiency phenotypes of <br /> C. albicans heterozygous mutants to identify regulators of this <br />                                                                                 PES1 Is Essential in C. albicans in Yeast-Inducing Conditions. The <br /> process (23). From a new heterozygous mariner transposon <br />                                                                                 effect of loss of Pes1 function was examined by constructing <br /> mutant collection, we isolated a hyperfilamentous mutant with <br /> a transposon insertion 47 nt upstream of the putative transla-                  three conditional alleles of PES1. Given the essentiality of <br /> tional start site of ORF 19.4093. This ORF encodes a homolog                    pescadillo homologs in other organisms, we did not attempt to <br /> of a protein conserved from yeast to human, originally called                   construct a homozygous PES1 deletion strain. Two different <br /> pescadillo when it was cloned in zebrafish (18), and YPH1 (yeast                repressible promoters were used to control expression of PES1 <br /> pescadillo homolog) or NOP7 in Saccharomyces cerevisiae (24,                    in heterozygotes that contain a deletion of one allele and a <br />                                                                                                                                                                       MICROBIOLOGY <br />  <br />  <br />  <br /> 25). We named this gene CaPES1 and abbreviate it as PES1 here.                  replacement of the native PES1 promoter of the remaining allele <br /> We created de novo mutations in PES1 in the prototrophic wild                   with the repressible MAL2 or tetO promoters (pMAL2 and tetO) <br /> type, SC5314 (26), by deleting PES1 on one chromosome from                      (27, 28). Transcription from pMAL2 is repressed by glucose in <br /> the putative start to the putative stop codons, and replacing the               the growth medium, partially repressed by mannitol, and in- <br /> second allele with a conditional allele as described below. The                 duced by maltose. In our system, transcription from tetO is <br /> strains carrying targeted mutations in PES1 did not recapitulate                repressed by doxycycline or tetracycline and induced in the <br /> the <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of the transposon mutant, which grew exclusively                  absence of these drugs. When PES1 transcription was repressed <br /> as pseudohyphal and hyphal filaments under all conditions                       from either promoter on rich yeast-inducing medium, YPD, <br /> tested. Instead, they showed specific defects in yeast growth                   there was minimal growth of pes1&#226;&#338;&#172;/pMAL2-PES1 or pes1&#226;&#338;&#172;/tetO- <br /> when PES1 was depleted, as described below.                                     PES1 strains (Fig. 2 and Fig. S2). <br />                                                                                   To test the effect of an abrupt perturbation of Pes1 function, <br /> PES1 Is an Ortholog of the S. cerevisiae pescadillo Homolog YPH1. The           we constructed heterozygotes in which the only allele of PES1 <br /> Pes1 predicted protein sequence is highly homologous to a large                 encodes a ts protein Pes1W416R (Fig. S3). A homologous amino <br />  <br /> Shen et al.                                                                                       PNAS &#229;&#8230;&#169; December 30, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 52 &#229;&#8230;&#169; 20919 <br />  Fig. 3. PES1 is expressed in both yeast and hyphal cells, and control of PES1 <br /> expression from the tetO promoter is independent of yeast- or hyphae-inducing <br /> conditions. Strains were grown overnight in YPD at 30 &#194;&#176;C, diluted to an initial <br /> density of OD600 0.2, and incubated in four conditions: YPD at 30 &#194;&#176;C (samples 1-6 <br /> and 13-18), YPD with 10% calf serum at 37 &#194;&#176;C (samples 7-12 and 19-24), YPD with <br /> 100 &#226;&#65533;&#174;g/ml doxycycline at 30 &#194;&#176;C (samples 25-30), and YPD with 10% calf serum and <br /> 100 &#226;&#65533;&#174;g/ml doxycycline at 37 &#194;&#176;C (samples 31-36). Cells were harvested at the <br /> indicated time points, and RNA preparations were blotted and probed with PES1. <br /> GSP1 was used as a loading control probe. (1-12) SC5314 (PES1/PES1). (13-36) <br /> JKC1137 (HIS1/his1::FRT-tetR pes1::FRT/FRT-tetO-PES1). <br />  <br />  <br />  <br /> acid substitution confers temperature sensitivity on S. cerevisiae <br /> mutants in YPH1 (25). Mutants with the genotype pes1&#226;&#338;&#172;/ <br /> PES1W416R had a growth defect on YPD at 24 &#194;&#176;C and did not <br /> grow at 37 &#194;&#176;C (Fig. S3). On rich yeast-inducing medium, func- <br /> tional Pes1 is required for growth. <br />  <br /> Loss of Pes1 Is Not Lethal in Hyphae. The effect of loss of Pes1 <br /> function on <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>survival</span> and growth of yeast or hyphal cells was                        Fig. 4. Lateral yeast growth is defective during Pes1 inactivation. Wild-type <br /> examined in mutants containing three different conditional                          and pes1&#226;&#338;&#172;/PES1W416R strains were grown for 9 days at 24 &#194;&#176;C on YP 2% galac- <br /> PES1 alleles. On yeast-inducing medium ts Pes1W416R mutants                         tose (A), and on RPMI1640, buffered to pH 7.5 with 165 mM Mops (B). (1, 3, and <br />                                                                                     5) SC5314 (PES1/PES1). (2 and 4) JKC1155 (pes1::FRT/FRT-PES1W416R). (6) <br /> were not viable after 3 days at the restrictive temperature, 37 &#194;&#176;C. <br />                                                                                     JKC1160 (pes1::FRT/FRT-PES1W416R). (1 and 2) Tips of young filaments. (3&#226;&#8364;&#8220;6) <br /> In contrast, on hyphae-inducing media, these mutants were able                      Filaments emanating from colony rim. Arrows, clusters of lateral yeast. (Scale <br /> to survive for 3 days at the restrictive temperature and then                       bar, 100 &#226;&#65533;&#174;m.) <br /> resume growth at the semipermissive temperature, 24 &#194;&#176;C (Fig. <br /> S3). Like the Pes1W416R ts mutants, mutants expressing the <br /> repressible promoter alleles of PES1 showed a differential                          lateral yeast growth (Fig. S4). Pes1W416R ts mutants were also <br /> requirement for PES1 in yeast- vs. hyphae-inducing conditions.                      defective in lateral yeast growth at the semipermissive temper- <br /> Cells in which PES1 expression was repressed from either                            ature (Fig. 4). Loss of Pes1 function therefore results in defective <br /> pMAL2 or tetO grew under hyphae-inducing conditions (Fig. 2                         lateral yeast growth. <br /> and Fig. S2), indicating that in hyphal cells, loss of PES1 is not <br /> lethal.                                                                             Loss of PES1 Does Not Lead to Constitutive Polarized Growth. For- <br />    To confirm that PES1 message is depleted during promoter                         mation of the ovoid shape of a budding yeast cell requires that <br /> repression of pes1&#226;&#338;&#172;/tetO-PES1 strains in both yeast- and hyphae-                    the polarized growth direction of an emerging bud switches to <br /> inducing conditions, levels of PES1 message were examined by                        isotropic expansion around the entire cell circumference as the <br /> Northern blot. Cells were grown under yeast- and hyphae-                            growing bud prepares for mitosis (30). We tested whether <br /> inducing conditions, in medium with and without doxycycline to                      defective lateral yeast growth of PES1 mutants is the result of an <br /> repress transcription from tetO. pes1&#226;&#338;&#172;/tetO-PES1 cells grown in                     inability to appropriately switch to isotropic growth, i.e., whether <br /> medium without doxycycline strongly overexpressed PES1 com-                         these mutants remain trapped in a polarized growth mode. <br /> pared to wild-type cells, irrespective of morphogenetic condi- <br />                                                                                     Terminal phenotypes of Pes1W416R ts mutants were examined in <br /> tions. Conversely, PES1 expression was strongly repressed by <br />                                                                                     hyphae- and yeast-inducing conditions. Mutant yeast cells inoc- <br /> doxycycline in both conditions, although a minimal amount of <br />                                                                                     ulated into hyphae-inducing conditions at the restrictive tem- <br /> PES1 message was still detectable (Fig. 3). PES1 expression in <br /> these strains is therefore controlled effectively from the tetO                     perature produced germ tubes indistinguishable from those of <br /> promoter, independently of morphogenesis-inducing conditions.                       wild type, which arrested as short hyphae (Fig. S5). When <br />                                                                                     Pes1W416R ts yeast cells were inoculated into yeast-inducing <br /> PES1 Is Expressed in Yeast and Hyphal Cells. We examined whether                    conditions at the restrictive temperature, they arrested in the <br /> PES1 is differentially expressed in yeast- vs. hyphae-inducing                      yeast form (Fig. S5). <br /> conditions. We used conditions previously described in large-                          The terminal phenotype of pes1&#226;&#338;&#172;/pMAL2-PES1 yeast during <br /> scale microarrray analyses of the yeast-to-hypha switch (29). The                   promoter repression was followed for 3 days. Mother cells <br /> wild type was found to express PES1 at comparable levels in both                    divided several times with each daughter receiving a nucleus, <br /> yeast- and hyphae-inducing conditions (Fig. 3).                                     eventually forming clusters of attached large yeast. After 3 days <br />                                                                                     of PES1 depletion, the nuclei appeared fragmented (Fig. S5). <br /> Loss of PES1 Perturbs Lateral Yeast Growth. The effect of loss of                   Thus, both abrupt inactivation of Pes1 and depletion of PES1 <br /> Pes1 function on morphogenesis was examined. pes1&#226;&#338;&#172;/pMAL2-                           from a repressible promoter did not interfere with appropriate <br /> PES1 strains growing on media containing mannitol, which                            switching of yeast cells from polarized to isotropic growth. <br /> partially represses PES1 expression, had considerably reduced                       Mutants in PES1 are not trapped in a polarized growth state. <br />  <br /> 20920 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0809147105                                                                                                   Shen et al. <br />  Fig. 5. PES1 overexpression leads to decreased hyphal growth and increased <br /> lateral yeast growth. Strains were streaked on Spider medium without doxy- <br /> cycline and incubated at 37 &#194;&#176;C for 4 days. (1) JKC915 (HIS1/his1::FRT-tetR <br /> PES1/PES1). (2) JKC962 (HIS1/his1::FRT-tetR PES1/pes1::FRT). (3) JKC1143     Fig. 6. PES1 depletion leads to loss of virulence in a Galleria mellonella <br /> (HIS1/his1::FRT-tetR pes1::FRT/FRT-tetO-PES1). (Scale bar, 100 &#226;&#65533;&#174;m.)          model. C. albicans pes1&#226;&#338;&#172;/pMAL2-PES1 mutant and wild-type strains were <br />                                                                              grown overnight at 30 &#194;&#176;C in YPD to repress PES1 expression in the mutant, or <br />                                                                              in YP 2% maltose (YPM) to induce PES1 expression in the mutant, and <br /> Quorum Sensing Is Not Perturbed by Pes1 Inactivation. To test                resuspended in PBS to an inoculum of 106 cells. Sixteen larvae per condition <br /> whether PES1 mutants&#226;&#8364;&#8482; defect in lateral yeast growth is the result           were injected and kept at 37 &#194;&#176;C. (1) Larvae injected with PBS only. (2) Larvae <br />                                                                              injected with JKC681 (pes1::FRT/FRT-pMAL2-PES1) pregrown in YPM. (3) Lar- <br /> of an inability to respond to quorum-sensing molecules that <br />                                                                              vae injected with JKC681 (pes1::FRT/FRT-pMAL2-PES1) pregrown in YPD. (4) <br /> stimulate a switch from hyphal to yeast growth and could be                  Larvae injected with SC5314 (PES1/PES1) pregrown in YPM. (5) Larvae injected <br /> secreted by subapical filamentous cells, we examined the re-                 with SC5314 (PES1/PES1) pregrown in YPD. The P value for difference in <br /> sponse of Pes1W416R ts mutants to two compounds that activate                survival curves between larvae injected with JKC681 pregrown in YPD and <br /> the C. albicans quorum-sensing response: farnesol and dodeca-                with JKC681 pregrown in YPM is 0.0001 (Mantel-Cox test). The P value for <br /> nol (31). At the restrictive temperature for Pes1W416R ts cells,             difference in survival curves between larvae injected with SC5314 pregrown in <br /> 37 &#194;&#176;C, wild-type yeast switched to filamentous growth at a                   YPD and with SC5314 pregrown in YPM is 0.547. <br /> significant rate when diluted into rich medium without quorum- <br /> signaling compounds (29). Filamentous growth was suppressed <br /> by addition of dodecanol or farnesol to the medium, with no                  by any growth defect. PES1-depleted cells have the least growth <br /> observable difference between Pes1W416R ts cells and the wild                defect when they are exposed to a combination of strong <br /> type (Fig. S5). Defective lateral yeast growth in PES1 mutants is            hyphae-inducing signals (Fig. 2 and Fig. S2). Larvae were held <br /> not the result of an inability to respond to quorum-signaling                at the hyphae-inducing temperature of 37 &#194;&#176;C, but we do not know <br /> molecules.                                                                   whether other hyphae-inducing signals prevail in the larva. <br />                                                                              Virulence attenuation during loss of PES1 is nevertheless con- <br /> PES1 Overexpression Results in Decreased Filamentous Growth and              sistent with a pathogenic role of Pes1-dependent yeast growth. <br /> Increased Lateral Yeast Growth. To investigate whether PES1                     Pescadillo homologs of yeast, mouse, and human act in pro- <br /> overexpression has the opposite effect from PES1 depletion,                  liferation control and ribosome biogenesis (19, 20). It is not clear <br /> pes1&#226;&#338;&#172;/tetO-PES1 mutants were grown on medium without doxy-                   whether these distinct functions are based in distinct biochemical <br /> cycline. In these conditions, PES1 is strongly overexpressed (Fig.           activities. PES1 is cotranscribed with genes involved in rRNA <br /> 3). On all solid media tested without doxycycline, these strains             processing and ribosome biogenesis (33) (Candida Genome <br /> showed decreased hyphal growth, and the hyphae that were                     Database), suggesting that it may participate in these functions <br /> formed showed overabundant lateral yeast (Fig. 5). In liquid rich            in C. albicans as well. One possibility is that growth arrest during <br /> medium, strains overexpressing PES1 did not have a significantly             PES1 depletion is caused by disruption of rRNA processing, <br /> altered growth rate compared to the wild type (Fig. S2). Over-               leading to activation of a &#226;&#8364;&#732;&#226;&#8364;&#732;nucleolar stress&#226;&#8364;&#8482;&#226;&#8364;&#8482; cell cycle checkpoint <br /> expression of PES1 increases, while its depletion decreases                  (34), or to depletion of ribosomes and slowing of translation. If <br /> lateral yeast growth.                                                        this is the case, tolerance of hyphae for PES1 depletion might <br />                                                                              indicate that hyphal cells need less ribosomes to grow than yeast <br /> PES1 Is Required for Virulence in the Galleria mellonella Larva Model.       cells. Alternatively, Pes1 may have separate roles in proliferation <br /> An insect model was used to examine the effect of Pes1                       control and rRNA processing. <br /> perturbation on C. albicans virulence (32). When injected at                    Cell cycle control in S. cerevisiae is linked to determination of <br /> inocula between 5 &#226;&#171;&#187; 105 and 106 cells, wild-type C. albicans                 cell shape (30). To form an ovoid yeast cell, new material is <br /> SC5314 rapidly kill G. mellonella larvae held at 37 &#194;&#176;C. pes1&#226;&#338;&#172;/               secreted toward the tip of a young bud from S phase until the <br /> pMAL2-PES1 cells, preincubated in glucose to deplete PES1,                   G2/M transition, in a polarized or apical direction. At that point, <br /> were significantly reduced in virulence in this system (Fig. 6).             growth is redirected to distribute new material isotropically <br /> Consistent results were obtained with the pes1&#226;&#338;&#172;/tetO-PES1 and                around the entire cell circumference. Delay of mitosis, by <br /> pes1&#226;&#338;&#172;/PES1W416R mutants (data not shown). PES1 is therefore                  prolonging the duration of polarized growth, leads to the elon- <br /> required for C. albicans virulence in this insect model.                     gated cells that form a pseudohyphal filament, and this effect is <br />                                                                                                                                                               MICROBIOLOGY <br />  <br />  <br />  <br />                                                                              enhanced by mutations in cell cycle regulators required for <br /> Discussion                                                                   progression through S and for the G2/M transition (30). The link <br /> In C. albicans, the pescadillo homolog PES1 is essential in yeast,           between cell cycle control and morphogenesis is more complex <br /> whereas hyphal cells tolerate PES1 depletion (Fig. 2 and Fig. S2).           in C. albicans because perturbation of many C. albicans cell cycle <br /> The pes1 yeast growth defect has implications for host&#226;&#8364;&#8220;pathogen              regulators, involved at all phases of the cell cycle, results in <br /> interactions because C. albicans uses its yeast form during its              constitutive polarized growth (35). In C. albicans, the gene <br /> commensal state and during dissemination and deep organ                      encoding G1 cyclin Cln3 is specifically required for growth in <br /> proliferation in invasive disease. Wild-type C. albicans constitu-           yeast-inducing conditions (36, 37), similarly to PES1. But unlike <br /> tively produce lateral yeast on filaments in conditions that induce          PES1 depletion, depletion of CLN3 in yeast-inducing conditions <br /> hyphal tip growth (Fig. 1), and this process is disturbed when               leads to growth arrest and then outgrowth of a hyphal or <br /> functional Pes1 is lacking (Fig. 4 and Fig. S4). In an insect model,         pseudohyphal filament (36, 37). In contrast, the terminal phe- <br /> virulence of cells deficient in Pes1 was attenuated. A caveat here           notype of pes1 mutants consists of growth arrest of yeast cells <br /> is that attenuation of pes1 mutant cells is likely to be potentiated         (Fig. S5). Consequently, inability to switch from polarized to <br />  <br /> Shen et al.                                                                                   PNAS &#229;&#8230;&#169; December 30, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 52 &#229;&#8230;&#169; 20921 <br />  isotropic growth cannot be the reason for the growth defect of                     Mutagenesis occurred by integration of a fragment of genomic DNA <br /> PES1-depleted yeast cells. In S. cerevisiae, the pescadillo homolog                carrying a transposon, into the C. albicans chromosome by homologous <br /> Yph1 is required for entry into the cell cycle from G0, for                        recombination, thereby integrating the transposon into the transfor- <br />                                                                                    mant&#226;&#8364;&#8482;s chromosome. A total of 17,000 mutants was arrayed in 96-well <br /> progression from G1 to S, and for progression from G2 to M <br />                                                                                    dishes; of these, 14,000 were viable in standard conditions. The <br /> (25). It remains to be determined whether Pes1 is required in one                  PES1/pes1::tn transposon mutant was isolated as a hyperfilamentous clone <br /> or in multiple phases of the C. albicans cell cycle and how this                   directly from the transformation plate (YPD &#226;&#171;&#185; nourseothricin). <br /> activity differs between yeast and hyphal cells. <br />    In S. cerevisiae, cell cycle entry and ribosome biogenesis are                  Plasmid Construction. Plasmids constructed for this study are described in SI <br /> tightly coordinated at the transcriptional level in response to                    Text on the PNAS Web site. Previously published plasmids used in this study are <br /> <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability (38, 39). Du and Stillman found evidence                     shown in Table S1, and primers are shown in Table S2, both of which are <br /> that in S. cerevisiae, the pescadillo homolog Yph1 participates in                 published as SI Text. <br /> nutritional signaling (25). Pes1 could participate in a signaling <br /> network that relays information about the cell&#226;&#8364;&#8482;s nutritional status                Microscopy of Lateral Yeast Growth in Liquid Medium. Wild-type strain SC5314 <br /> to both the cell cycle machinery and the ribosomal biogenesis                      was grown in YPD to saturation overnight at 24 &#194;&#176;C, diluted to an OD600 of <br />                                                                                    0.005 in Spider medium, and inoculated into wells of 6 well plates containing <br /> machinery simultaneously to coordinate these two processes. <br />                                                                                    fibronectin-coated coverslips. After 6.5 h incubation at 37 &#194;&#176;C, cultures were <br /> High levels of PES1 expression in fresh rich media, and decrease                   fixed with formaldehyde and examined by brightfield microscopy. <br /> of PES1 levels over time (Fig. 3), may be consistent with this idea. <br /> The finding that overexpression of PES1 has the opposite                           Growth Assays. For cell dilutions spotted onto solid media, cells were grown <br /> morphogenetic effect as its depletion (Fig. 5) may support a                       overnight in YPD at 24 &#194;&#176;C for experiments using Pes1W416R-containing mu- <br /> regulatory role for Pes1. If Pes1 is linked to a nutritional                       tants, and at 30 &#194;&#176;C for all other experiments. Cells were diluted to an OD600 of <br /> signaling network, the cAMP-dependent protein kinase (PKA)                         0.2 in PBS and further diluted in fivefold steps in PBS. Cell suspensions were <br /> pathway is a candidate for functional interactions with Pes1. The                  spotted onto appropriate media using a replicator with prongs calibrated to <br /> only previously described C. albicans mutants with altered lateral                 deliver 1.5 &#226;&#65533;&#174;l (V&amp;P Scientific, VP407). For growth curves in liquid media, all <br /> yeast growth, in PDE2 and CAP1, have abnormal intracellular                        strains were grown to saturation overnight in YPD with 0.01 &#226;&#65533;&#174;g/ml of doxy- <br />                                                                                    cycline at 24 &#194;&#176;C. They were washed once in the cognate growth medium (YPD, <br /> cAMP levels (9, 17). In addition, a phosphatase linked to the <br />                                                                                    Spider, or 20% serum and 2% glucose in water) without or with 20 &#226;&#65533;&#174;g/ml of <br /> PKA pathway, Yvh1, interacts physically with the pescadillo                        doxycycline and diluted to an OD600 of 0.05. One hundred microliters of each <br /> homolog in S. cerevisiae and in Plasmodium falciparum (40, 41).                    cell suspension was inoculated into wells of flat bottom 96-well dishes, and <br /> Signaling through the PKA pathway is required for ribosome                         cultures were incubated at the cognate temperature without shaking in a <br /> biogenesis (38), consistent with a model whereby Pes1, in                          Spectramax250 plate reader. OD600 readings were obtained every 30 min and <br /> interaction with the PKA pathway, responds to availability of                      imported into Excel at the end of the experiment. Means and standard <br /> nutrients to promote rRNA processing and cell proliferation.                       deviations were calculated and graphed in Excel. <br /> Whether C. albicans hyphal cells require less PES1 because <br /> hyphal growth is less dependent on such a nutritional signal                       Northern Blot. Cells of the wild-type strain SC5314, and a pes1/tetO-PES1 strain, <br />                                                                                    JKC1137, were grown to saturation (OD600 &#226;&#172;&#381;12) in YPD at 30 &#194;&#176;C overnight. <br /> remains to be determined. <br />                                                                                    They were diluted to an initial density of OD600 0.2 into 12 ml medium per <br />    In summary, Pes1 is a putative regulator implicated both in the <br />                                                                                    sample. The pes1/tetO-PES1 strain was incubated at 200 rpm in four condi- <br /> filament-to-yeast switch and in proliferation of yeast, two pro-                   tions: YPD at 30 &#194;&#176;C, YPD with 10% calf serum at 37 &#194;&#176;C , YPD with 100 mg/ml <br /> cesses that are critical for the vegetative life cycle of this principal           doxycycline at 30 &#194;&#176;C and YPD with 10% calf serum and 100 mg/ml doxycycline <br /> fungal pathogen of humans.                                                         at 37 &#194;&#176;C . The wild type was incubated in the first two conditions. Cells were <br />                                                                                    harvested at six time points: 0.5, 1, 3, 6, 9, and 24 h. Total RNA was extracted <br /> Materials and Methods                                                              using TRIzol (Invitrogen) according to the manufacturer&#226;&#8364;&#8482;s instructions. Total <br /> Strains and Culture Conditions. The C. albicans and S. cerevisiae strains used     RNA was blotted to a nylon membrane (Hybond XL) and probed with the <br /> in this study are described in Table S1. All C. albicans strains were generated    insert of pJK945, which was labeled with [32]ATP by random priming. As the <br /> in the SC5314 genetic background using the CaNAT1 selectable marker, as            loading control probe, a portion of the GSP1 ORF was amplified with primers <br /> described in ref. 42. Each of the three conditional alleles of PES1 was            871 and 998. GSP1 encodes a small G protein whose expression level is not <br /> generated from two independent heterozygotes. PES1 deletion and re-                affected during morphogenesis (D. Kadosh, personal communication). <br /> placement of the PES1 promoter with pMAL2 and tetO, and introduction <br /> of the Pes1W416R ts-encoding <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> were confirmed by Southern blot.             Quorum-Sensing Response Assay. Strains were grown in YPD overnight at <br /> The genotype of S. cerevisiae strains was confirmed by PCR spanning the            24 &#194;&#176;C, and diluted to an OD600 of 0.025 into 250 &#226;&#65533;&#174;l of each medium: YPD with <br /> upstream and downstream homologous recombination junctions of trans-               10% calf serum, YPD with 0.5% DMSO, and YPD with 200 &#226;&#65533;&#174;M dodecanol or <br /> forming constructs. For routine growth of yeast strains, YPD (1% yeast             farnesol, and incubated in 24-well dishes at 37 &#194;&#176;C without shaking. <br /> extract/2% peptone/2% glucose) was used. Spider medium was made as in <br /> ref. 43. Solid serum medium was made with 10% or 20% calf serum                    G. mellonella Killing Assay. Injection of C. albicans in the G. mellonella killing <br /> (Gibco/Invitrogen cat. no.16170 &#226;&#8364;&#8220; 078), water, and 2% agar; 2% glucose or          assay was performed essentially as described (46). In brief, larvae in the final <br /> maltose were added as indicated.                                                   instar larval stage were obtained from Canadian Feeders. Inocula of C. albi- <br />    The heterozygous transposon mutant collection will be described else-           cans for the injections were prepared by growing 50 ml YPD cultures overnight <br /> where in detail. Briefly, a C. albicans genomic library, consisting of 4.9 &#226;&#171;&#187; 104   at 30 &#194;&#176;C. Cells were pelleted by centrifugation at 3,000 rpm for 10 min <br /> independent clones (44), was mutagenized by transposon insertion in vitro. A       followed by three washes in PBS. Cells were resuspended in 1 ml of PBS and cell <br /> hyperactive C9 mariner transposase was purified as a fusion with maltose           densities were determined by hemacytometer count and additionally con- <br /> binding protein, and transposition conditions were optimized according to          firmed by OD600 comparison for PES1-depleted cultures in which yeast cells <br /> considerations described in ref. 45. The transposon was adapted to contain the     formed aggregates of unseparated yeast. Final dilutions for injection of 106 <br /> CaNAT1 marker, which confers resistance to the aminoglycoside nourseothri-         cells were prepared in PBS. Sixteen larvae (330 &#226;&#171;&#190; 25 mg) were used per group <br /> cin on C. albicans and on Escherichia coli (42). A transposon-mutagenized          and 5-&#226;&#65533;&#174;l injections were performed via the last left proleg. Larvae were <br /> genomic C. albicans plasmid library consisting of 1.26 &#226;&#171;&#187; 105 clones was            incubated in Petri dishes at 37 &#194;&#176;C in the dark and the number of dead larvae <br /> generated. In our system, each clone represents an independent insertion           was scored daily. Kill curves were plotted and estimation of differences in <br /> event because in vitro mutagenesis was followed by electroporation of prod-        survival by log-rank (Mantel-Cox) test was analyzed by the Kaplan-Meier <br /> ucts of transposition into E. coli and selection of E. coli transformants on       method using GraphPad Prism statistical software. <br /> nourseothricin (clonNAT, Werner BioAgents). <br />    C. albicans wild-type strain SC5314 was transposon mutagenized by              <br /> transformation with inserts excised from the in vitro mutagenized genomic        <br /> library and selection on rich medium (YPD) containing nourseothricin.         <br />  <br />  <br />  <br />  logistical aid. The Harvard Digestive Diseases Imaging Core and Jessica Wag-                    script. This work was funded by a Charles A. Janeway Child Health Research <br /> ner are gratefully acknowledged. Thanks to Eric Rubin, Chris Sassetti, Joachim                  Center Award and by NIH R21 AI064715 to J.R.K. L.E.C. is supported by a Career <br /> Morschha&#194;&#168;user, John Woolford, Todd Milne, Phil Hieter and Bill Fonzi for                        Award in the Biomedical Sciences from the Burroughs Wellcome Fund, by a <br /> plasmids and strains. We are grateful to Simon Dove, Paula Watnick, Haoping                     Canada Research Chair in Microbial Genomics and Infectious Disease, and by <br /> Liu, Joyce Fingeroth, and Mike Lorenz for valuable comments on the manu-                        Canadian Institutes of Health Research Grant MOP-86452. <br />  <br /> </body></html>