<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> A novel plant gene essential for meiosis is related <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> the human CtIP and the yeast COM1/SAE2 gene <br />  <br />                                           replicated homologous chromosomes segregate, while during <br />                                                  meiosis II, sister chromatids are separated in an equational- <br />                                                like division. During prophase of meiosis I, reciprocal recom- <br />                                                bination between homologous chromosomes occurs before <br />                                                  they are segregated at meiosis I. The process of homologous <br />                                                                                         recombination (HR) contributes to the genetic diversity of the <br />  <br />                                                                                         meiotic products and is a prerequisite for the correct segrega- <br />  <br />                           tion of homologous chromosomes (Zickler and Kleckner, <br />  <br />  <br />                                                                                         1999; Page and Hawley, 2003). <br />                      Meiotic recombination is initiated by DNA double-strand <br />                     breaks (DSBs), generated by Spo11, a homologue of the <br />                                                                                         archaebacterial topoisomerase subunit Top6A (Bergerat <br /> Obligatory homologous recombination (HR) is required                                    et al, 1997; Keeney et al, 1997). Arabidopsis thaliana has <br /> for chiasma formation and chromosome segregation in                                     three SPO11-like genes, while all other analysed species <br /> meiosis I. Meiotic HR is initiated by DNA double-strand                                 possess only one such gene (Hartung and Puchta, 2000, <br /> breaks (DSBs), generated by Spo11, a homologue of the                                   2001). Arabidopsis Atspo11-1 (Grelon et al, 2001) and <br /> archaebacterial topoisomerase subunit Top6A. In                                         Atspo11-2 mutants (Stacey et al, 2006) exhibit defects in <br /> Saccharomyces cerevisiae, Rad50, Mre11 and Com1/Sae2                                    pairing and recombination during meiosis, while Atspo11-3 <br /> are essential to process an intermediate of the cleavage                                is not involved in meiosis (Yin et al, 2002). In budding yeast, <br /> reaction consisting of Spo11 covalently linked to the 50                                the help of at least nine proteins (Smith and Nicolas, 1998; <br /> termini of DNA. While Rad50 and Mre11 also confer                                       Keeney, 2001; Prieler et al, 2005) is required for Spo11- <br /> genome stability to vegetative cells and are well conserved                             mediated catalytic cleavage. Further processing of DSBs <br /> in evolution, Com1/Sae2 was believed to be fungal-speci-                                requires some of the same and several additional proteins, <br /> fic. Here, we identify COM1/SAE2 homologues in all eu-                                  among them Com1/Sae2 (Keeney and Kleckner, 1995; McKee <br /> karyotic kingdoms. Arabidopsis thaliana Com1/Sae2                                       and Kleckner, 1997; Prinz et al, 1997; Cartagena-Lirola et al, <br /> mutants are sterile, accumulate AtSPO11-1 during meiotic                                2006). Spo11 is found covalently linked to the 50 termini of <br /> prophase and fail to form AtRAd51 foci despite the pre-                                 DNA in an intermediate of the DNA cleavage reaction. The <br /> sence of unrepaired DSBs. Furthermore, DNA fragmenta-                                   release of Spo11 <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> DNA is mediated by ssDNA nick <br /> tion in AtCom1 is suppressed by eliminating AtSPO11-1. In                               formation next to the DSB site, thereby liberating the Spo11 <br /> addition, AtCOM1 is specifically required for mitomycin C                               protein attached to a few nucleotides. In budding yeast, this <br /> resistance. Interestingly, we identified CtIP, an essential                             release requires Rad50, Mre11 and Com1/Sae2. Mutations in <br /> protein interacting with the DNA repair machinery, as the                               these genes cause accumulation of Spo11 on DNA ends <br /> mammalian homologue of Com1/Sae2, with important                                        (Keeney and Kleckner, 1995; Neale et al, 2005; Prieler et al, <br /> implications for the molecular role of CtIP.                                            2005). Following DNA cleavage and Spo11 removal, a 50 -30 <br /> The EMBO Journal (2007) 26, 5061&#226;&#8364;&#8220;5070. doi:10.1038/                                     exonucleolytic activity, mediated by a nuclease not yet iden- <br /> sj.emboj.7601913; Published online 15 November 2007                                     tified, generates 30 single-stranded DNA tails. These tails are <br /> Subject Categories: genome stability &amp; dynamics                                         believed to serve as probes for identifying homologous <br /> Keywords: Arabidopsis; CtIP; DNA repair; meiosis; SAE2                                  partner chromosomes, and to initiate D-loop formation and <br />                                                                                         single-end invasion followed by second-end capture (Paques <br />                                                                                         and Haber, 1999; Hunter and Kleckner, 2001; Neale and <br />                                                                                         Keeney, 2006). <br />                                                                                             Homologues of RAD50, MRE11 and their complex partner <br /> Introduction <br />                                                                                         XRS2/NBS1 have been identified in higher eukaryotes, while <br /> The process of meiosis includes two successive nuclear                                  homologues of Com1/Sae2 have remained elusive. Yeast two- <br /> divisions that follow a single round of DNA replication.                                hybrid assays have demonstrated that the Arabidopsis <br /> During the reductional division, called meiosis I, pairs of                             proteins AtRAD50 and AtMRE11 physically interact and that <br />                                                                                         the rice OsMRE11 protein interacts with OsNBS1 (Bhatt et al, <br /> *Corresponding author. Department of Chromosome Biology, Max F                          2001; Daoudal-Cotterell et al, 2002; Akutsu et al, 2007). <br /> Perutz Laboratories, University of Vienna, Dr Bohr-Gasse 1, Vienna <br />                                                                                         A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> in AtRAD50 leads to meiotic defects, sterility <br /> A-1030, Austria. Tel.: &#195;&#190; 43 1 4277 56240; Fax: &#195;&#190; 43 1 4277 9562; <br /> E-mail: peter.schloegelhofer@univie.ac.at                                               and sensitivity to methyl-methano-sulphonate (Gallego et al, <br /> 5 <br />   These authors contributed equally to this work                                        2001; Bleuyard et al, 2004). In vertebrates, MRE11 is an <br /> 6 <br />   Present address: Department of Botany, Federal University of                          essential gene with roles in both somatic and meiotic cells <br /> Pernambuco, 50670-420, Recife, Brazil <br />                                                                                         (D&#226;&#8364;&#8482;Amours and Jackson, 2002), while in A. thaliana, MRE11 <br /> Received: 18 June 2007; accepted: 17 October 2007; published online:                    is not essential. Depending on the MRE11 mutant allele, <br /> 15 November 2007                                                                        plants are either sterile due to perturbed meiosis and exhibit <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                       The EMBO Journal   VOL 26 | NO 24 | 2007 5061 <br />        Analysis of Arabidopsis COM1/SAE2 gene <br />       C Uanschou et al <br />  <br />  <br />  <br />       severe developmental abnormalities or they only display            revealed conservation of its C-terminal half (conservation <br />       enhanced sensitivity to genotoxic agents (Bundock and              plot (Rice et al, 2000)), corresponding to amino acids <br />       Hooykaas, 2002; Puizina et al, 2004). In contrast to yeast,        190&#226;&#8364;&#8220;345 of the yeast Com1/Sae2 protein. Consequently, <br />       MRE11 and RAD50 are not essential for the formation of             reciprocal PSI-BLAST searches (against the NCBI non-redun- <br />       meiotic DSBs in A. thaliana, but only in the ensuing proces-       dant database (nr), version 12/2006; low-complexity filtered, <br />       sing and repair (Bleuyard et al, 2004; Puizina et al, 2004).       inclusion cut-off 0.001) (Altschul et al, 1997; Marchler-Bauer <br />          Yeast&#226;&#8364;&#8482;s COM1/SAE2 was detected independently in three           et al, 2002) were carried out with the conserved region of all <br />       genetic screens. Two screens were designed to isolate meiotic      fungal relatives (Supplementary Figure S1A). This procedure <br />       mutants, defective after the initiation of Spo11-induced DSBs,     yielded a coherent set of related sequences in a wide variety <br />       but before resolution of recombination intermediates (McKee        of eukaryotes including the Arabidopsis NP_850683 <br />       and Kleckner, 1997; Prinz et al, 1997). The phenotypes of the      (At3g52115) (Supplementary Figure S1B) and the metazoan <br />       isolated com1/sae2-null mutations are similar to those con-        CtIP protein (Homo sapiens CtIP, Rbbp8; Fusco et al, 1998), <br />       ferred by the previously identified non-null mutations of          shown in Figure 1A aligned by their common C-terminal <br />       RAD50 (rad50S) and MRE11 (mre11S) (Alani et al, 1990;              homology. The Arabidopsis and the human proteins share <br />       Keeney et al, 1997; Nairz and Klein, 1997; Tsubouchi and           54% identity and 45% similarity in their conserved C-termini <br />       Ogawa, 1998). These mutations completely block resection           and their close relation are further illustrated by cluster <br />       and turnover of meiosis-specific DSBs. The third screen            analysis (Figure 1B; Frickey and Lupas, 2004). Arabidopsis <br />       aimed to identify mutants with low fidelity DSB repair during      At3g52115 was picked up in a screen as being upregulated <br />       vegetative growth (Rattray et al, 2001). In mitotic cells,         after ionising irradiation and was called AtGR1 (A. thaliana <br />       Com1/Sae2 (together with proteins of the Mre11 complex)            gamma response gene 1) (Deveaux et al, 2000). Garcia et al <br />       is essential for the repair of hairpin-capped DSBs, for holding    (2003) demonstrated that ATM kinase is required for tran- <br />       DNA ends in close proximity after DSB formation and for            scriptional induction of AtGR1. However, neither AtGR1&#226;&#8364;&#8482;s <br />       preventing chromosome rearrangements (Lobachev et al,              relation to COM1/SAE2, nor its importance for meiosis was <br />       2002; Clerici et al, 2005). This explains how Com1/Sae2            noticed. We suggest calling the gene AtGR1/COM1 and here- <br />       could confer resistance to genotoxic treatments (Birrell           after refer to it as AtCOM1. The CtIP protein was also not <br />       et al, 2002; Deng et al, 2005). Furthermore, detailed analysis     recognised as a homologue of COM1/SAE2 despite a more <br />       has shown that Com1/Sae2 is phosphorylated by the protein          detailed characterisation (see Discussion). <br />       kinases Mec1 and Tel1 (yeast homologues of ATR and ATM,               To underline the functional significance of the conserved <br />       respectively) and that it interferes with the DNA replication      COM1/SAE2 C-terminus, yeast Dcom1 strains (Prinz et al, <br />       and damage checkpoints during mitosis and meiosis (Baroni          1997) where transformed with plasmids either carrying wild- <br />       et al, 2004; Cartagena-Lirola et al, 2006; Clerici et al, 2006).   type or mutagenised versions of the COM1/SAE2 gene, with <br />       Com1/Sae2 is among the first proteins to be found at somatic       single, double or triple amino-acid exchanges of the most <br />       DSB sites, after Mre11 and Tel1, but before Rad52. Com1/           conserved residues, respectively. Whereas the cells which <br />       Sae2 and Mre11 do not rely on each other with respect to their     obtained a wild-type COM1/SAE2 gene showed 100% spore <br />       initial localisation to DSBs, but mutually depend on each          viability (as opposed to the untransformed Dcom1 strain which <br />       other for their timely disappearance from DSBs (Lisby et al,       has 0.06% spore viability), the cells which obtained a muta- <br />       2004; Clerici et al, 2006).                                        genised version showed strongly reduced spore viability: <br />          The analysis of the A. thaliana AtCOM1 gene represents          (1) R262,264A: 3.4%; (2) S267A: 10.4%; (3) P268A: 0.28%; <br />       the first characterisation of COM1/SAE2 in a higher eukar-         (4) GF270AA: 0.03%; (5) FPS276AAA: 0.7% (Figure 1A). <br />       yote. Our data demonstrate that a COM1/SAE2 homologue <br />       exists in various higher eukaryotes and that it is essential for   Molecular characterisation of Atcom1 <br />       female and male meiosis in A. thaliana. We show that               To study the function of the AtCOM1 protein in plants, we <br />       AtCOM1 acts downstream of AtSPO11-1 and upstream of                obtained two T-DNA insertion mutant lines from the Salk <br />       AtDMC1 during meiosis. We provide evidence that AtCOM1             Institute Genomic Analysis Laboratory T-DNA collection <br />       is needed for regular turnover of AtSPO11-1 and processing of      (Alonso et al, 2003) harbouring alleles Atcom1-1 and <br />       meiotic DSBs. Furthermore, we show that AtCOM1 is essen-           Atcom1-2, respectively. We confirmed the position of the <br />       tial for a specific DNA repair process in somatic cells.           Atcom1-1 and Atcom1-2 T-DNA insertions within the 2nd <br />       Importantly, our findings correlate yeast&#226;&#8364;&#8482;s Com1/Sae2 protein      exon of the AtCOM1 gene by PCR, sequenced the insertion <br />       with the mammalian DNA repair-related protein CtIP, whose          borders and determined the structure of Atcom1-1 <br />       significance in meiosis has not yet been elucidated.               by Southern analysis (Supplementary Figure S2A&#226;&#8364;&#8220;C). <br />                                                                          Expression analysis by reverse transcription revealed the <br />                                                                          presence of AtCOM1 transcripts in the all wild-type tissues <br />       Results                                                            examined (buds, leaves, seedlings), and a strong induction in <br />       Identification of COM1/SAE2 homologues in higher                   irradiated tissues. Only a truncated mRNA, corresponding to <br />       eukaryotes                                                         the region upstream of the T-DNA insertion sites, was de- <br />       To identify homologues of the yeast COM1/SAE2 gene, we             tected, in the mutant plants (Supplementary Figure S2D). <br />       first collected a set of fungal Com1/Sae2 homologues by            As both the Atcom1-1 and the Atcom1-2 coding regions are <br />       applying PSI-BLAST and reciprocal proteome BLAST searches          interrupted by a stop codon (Supplementary Figure S2B), the <br />       (with low-complexity filtering, E-value cut-off 0.001)             mutant proteins are predicted to consist of the N-terminal 456 <br />       (Altschul et al, 1997) starting with the full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>               and 479 amino acids, respectively, both lacking the conserved <br />       Saccharomyces cerevisiae Com1/Sae2 protein sequence.               C-terminus (the full-length protein has a predicted length of <br />       Alignment of 20 different fungal Com1/Sae2 homologues              588 amino acids). <br />  <br /> 5062 The EMBO Journal VOL 26 | NO 24 | 2007                                                &amp; 2007 European Molecular Biology Organization <br />                                                                                                                  Analysis of Arabidopsis COM1/SAE2 gene <br />                                                                                                                                         C Uanschou et al <br />  <br />  <br />  <br />  <br /> Figure 1 The Com1/Sae2 motif is conserved among eukaryotes. (A) Alignment of putative homologues of the S. cerevisiae Com1/Sae2 protein. <br /> The alignment (muscle (Edgar, 2004); clustal-like colouring) shows the C-terminal protein region of best conservation. Residue conservation <br /> above selected thresholds (50, 70, 90%) is indicated below the alignment. Relative frequency was scored for individual amino acids as well as <br /> for groups of related amino acids: alcoholic (denoted by &#226;&#8364;&#732;o&#226;&#8364;&#8482; and including the amino acids S, T), aliphatic (&#226;&#8364;&#732;l&#226;&#8364;&#8482; for I, L, V), aromatic (&#226;&#8364;&#732;a&#226;&#8364;&#8482; for F, H, <br /> W, Y), positive (&#226;&#8364;&#732; &#195;&#190; &#226;&#8364;&#8482; for H, K, R), negative (&#226;&#8364;&#732; &#226;&#8364;&#732; for D, E). Numbers and asterisks above the alignment refer to amino-acid exchanges introduced <br /> into the S. cerevisiae protein sequence. (B) Cluster analysis of COM1/SAE2 homologues. The analysis was performed with CLANS (Frickey and <br /> Lupas, 2004). Individual proteins are shown as vertices and are connected by lines reflecting pairwise sequence similarities from all-against-all <br /> BLAST searches. Attractive forces increasing with the sequence similarity between proteins lead to clustering of sequence related proteins. The <br /> strength of line shading increases with increasing sequence similarity. Abbreviations: (Ag) Ashbya gossypii, (An) Aspergillus nidulans, (At) <br /> Arabidopsis thaliana, (Cc) Coprinopsis cinerea, (Cg) Candida glabrata, (Ch) Cryptosporidium hominis, (Cn) Cryptococcus neoformans, (Cp) <br /> Cryptosporidium parvum, (Dd) Dictyostelium discoideum, (Dg) Drosophila grimshawi, (Dm) Drosophila melanogaster, (Dp) Drosophila <br /> pseudoobscura, (Dr) Danio rerio, (Gz) Gibberella zeae, (Hs) Homo sapiens, (Kl) Kluyveromyces lactis, (Mm) Mus musculus, (Nc) Neurospora <br /> crassa, (Os) Oryza sativa, (Pc) Phanerochaete chrysosporium, (Pt) Populus trichocarpa, (Sc) Saccharomyces cerevisiae, (Tc) Trichoderma reesei, <br /> (Xl) Xenopus laevis, (Yl) Yarrowia lipolytica, (Zm) Zea mays. <br />  <br />  <br /> Atcom1 mutant plants are sterile due to chromosome                            Atcom1-1 and Atcom1-2 mutant plants are completely devoid <br /> fragmentation during meiosis                                                  of seeds (Figure 2A). This and all the other observed pheno- <br /> Plants homozygous for the Atcom1 mutant alleles (Atcom1-1,                    types were completely reversed by introduction of a genomic <br /> Atcom1-2) do not show growth aberrations during vegetative                    wild-type copy of the gene (Figure 2A). To further character- <br /> development. The mutants germinate and develop at the                         ise the fertility defect, male and female gametophyte devel- <br /> same time and rate as wild-type plants. Rosette leaves are                    opment was monitored. Atcom1 mutant plants develop <br /> normal in size, shape and number, and bolting is not delayed.                 anthers that do not contain viable pollen (Figure 2A) and <br /> Inflorescences look normal, but none of the siliques of                       only aberrant embryo sacs (Supplementary Figure S3A), <br /> Atcom1 mutant plants elongate, whereas shoots of wild-                        implying that the sterility <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> of Atcom1 mutant plants <br /> type plants have long siliques. The short siliques of                         is based on gametogenesis defects. This conclusion is sup- <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                          The EMBO Journal      VOL 26 | NO 24 | 2007 5063 <br />        Analysis of Arabidopsis COM1/SAE2 gene <br />       C Uanschou et al <br />  <br />  <br />  <br />       ported by the fact that Atcom1-1 plants (n &#194;&#188; 8 buds) could not          to diakinesis and metaphase I (Figure 3O, P). In anaphase I <br />       be fertilised with wild-type pollen. Prior to gametogenesis,            DNA fragments, typically telomeric fragments are left behind <br />       haploid cells have to be formed during the process of meiosis.          at the metaphase plate in Atcom1-1 plants (Figure 3Q&#226;&#8364;&#8220;S), <br />       A key step during meiosis is synapsis and recombination                 indicating that meiotic DSBs remain unrepaired in the mu- <br />       between homologous chromosomes. Cytological analysis                    tant. Figure 3R also shows a DNA bridge connecting two <br />       of male meiosis fails to detect synapsed chromosomes in                 centromeres, which segregate to different poles. FISH analy- <br />       Atcom1-1 plants during stages corresponding to zygotene and             sis with a probe directed to the 180 bp repeat region of <br />       pachytene (Figure 3B, C, M, N). To identify homologous                  centromeres shows that some of the bridges observed in <br />       chromosomes, chromosome I (at an interstitial region) and               Atcom1-1 mutants consist of centromeric DNA, indicating <br />       chromosome II (at a subtelomeric region) were simulta-                  that bridges could either originate from separation of sister <br />       neously labelled by fluorescent in situ hybridisation (FISH).           chromatids, persisting DNA catenation or from a nonhomo- <br />       Indeed, no close pairing of either homologous signal pair was           logous repair mechanism (data not shown). DNA fragments <br />       observed in the mutant (Figure 3M, N), but close pairing was            and bridges are never observed in wild-type cells during <br />       frequently seen in wild-type zygotene stages and always                 anaphase or telophase I (Figure 3G, H). In addition to <br />       observed in wild-type pachytene stages (Figure 3B, C). This             fragmentation, massive chromosome missegregation is re- <br />       implies that no stable interactions between homologous                  vealed by the often asymmetric distribution of nuclear mate- <br />       chromosomes can be formed in homozygous Atcom1-1.                       rial in the first meiotic division in Atcom1-1 cells (Figure 3). <br />       While five well-defined bivalents emerge from condensation              These are expected consequences of the earlier homologue <br />       during diakinesis (Figure 3E) and metaphase I in wild type              pairing and DSB repair defects. Anaphase II figures were also <br />       (Figure 3F), chromosomes are always connected and en-                   observed, and all showed a high incidence of fragmentation <br />       tangled in the Atcom1-1 mutant during stages corresponding              and missegregation, usually culminating in the formation of <br />                                                                               more than four, poorly condensed, unequal masses of chro- <br />                                                                               matin at telophase II (Figure 3U). Equivalent defects were <br />           A                                                                   observed during female meiosis in Atcom1-1 mutants <br />                                                                               (Supplementary Figure S3B). We conclude that the sterility <br />                                                                               of Atcom1-1 mutants is caused by aberrant meiosis and <br />                                                                               defective repair of meiotic DSBs. <br />  <br />                                                                               Chromosome fragmentation observed in Atcom1-1 <br />                                                                               mutants depends on SPO11-1 but not on the <br />                                                                               RecA-related DMC1 protein <br />                                                                               If fragmentation and chromosome bridges in Atcom1-1 meio- <br />                                                                               sis were solely caused by the inability to process meiotic <br />                                                                               DSBs, reduction of DSB formation by mutating the nuclease, <br />                                                                               AtSPO11-1 (Grelon et al, 2001), should strongly curb such <br />           wt                    Atcom1-1                                      aberrations (the Atspo11-1-1 mutation does not completely <br />                                                                               eliminate DSBs, either due to residual activity or due to its <br />                                                                               paralog AtSPO11-2 (Stacey et al, 2006)). Indeed, when we <br />                                                       Atcom1-1/               generated the homozygous Atcom1-1 Atspo11-1-1 double <br />                                                       gAtCOM1                 mutant, DNA fragmentation was undetectable in most cells <br />                                                                               (Figure 4), showing that the inability to repair meiotic DSBs is <br />                                     40 &#194;&#181;M MMC                                 responsible for Atcom1-1 chromosome aberrations. <br />           B <br />                                                                                   Cells repair meiotic DSBs by using the homologue rather <br />                                                                               than the sister chromatid as a template, a phenomenon called <br />                                                                               interhomologue bias (Zickler and Kleckner, 1999). We, there- <br />                                                                               fore, asked whether relaxing interhomologue bias might <br />                                                                               permit repair in Atcom1-1 meiocytes. DMC1, a meiosis-spe- <br />                                                                               cific RecA recombinase, is specifically required for interho- <br />                                                                               mologue interactions (Schwacha and Kleckner, 1997) and <br />           wt                                   Atcom1-1                       Atdmc1 mutants are thought to repair all DSBs by using <br />                                                                               the sister chromatid as a template (Couteau et al, 1999). <br />       Figure 2 Phenotypes of the Atcom1-1 mutant. (A) Atcom1-1 mu- <br />       tants look like wild-type plants, but do not develop mature siliques.   However, chromosome fragmentation persists in the <br />       The left panel shows a stem with mature siliques of a wild-type (wt)    Atcom1 Atdmc1 double mutant meiosis (Figure 4), suggesting <br />       plant. The middle panel shows the stem of an Atcom1-1 mutant            that Atcom1 affects both intersister and interhomologue <br />       plant of the same age, which failed to develop mature siliques. The     recombination alike. This result is expected, if AtCOM1 is <br />       right panel shows an Atcom1-1 mutant plant containing a fertility- <br />       restoring genomic copy of AtCOM1. Inlays: anthers of wild-type          not specific to the interhomologue repair pathway, but <br />       (wt) and Atcom1-1 plants stained as described (Alexander, 1969).        obligatory for meiotic repair as Com1/Sae2 is in yeast. <br />       The purple-stained cytoplasm indicates viable pollen grains; green <br />       indicates empty pollen. No regular sized and viable purple pollen is    AtCOM1 is essential for regular turnover of AtSPO11-1 <br />       present in Atcom1 anthers. (B) Seedlings grown on media plates <br />       containing 40 mM MMC. Wild-type (wt) plants develop true leaves.        and normal processing of DSBs <br />       In contrast, Atcom1-1 mutant plants only expand their cotyledons        A very early step in meiotic DSB repair after removal of <br />       but do not develop true leaves.                                         Spo11 and formation of 30 single-stranded ends is coating <br />  <br />  <br />  <br />  <br />  <br /> Figure 3 Meiosis in pollen mother cells of wild-type (A&#226;&#8364;&#8220;K) and Atcom1-1 mutant (L&#226;&#8364;&#8220;U) plants. Wild-type: (A) leptotene; (B) zygotene; (C) <br /> pachytene; (D) diplotene; (E) diakinesis; (F) metaphase I; (G) anaphase I; (H) prophase II; (I) metaphase II; (J) anaphase II; (K) telophase II. <br /> Meiosis in the severely disrupted Atcom1-1 mutant plants: (L) leptotene; (M) zygotene-like stage; (N) pachytene-like stage, without normal <br /> chromosome pairing; (O) diakinesis-like stage; (P) metaphase I-like stage with entangled chromosomes; (Q, R) progression of anaphase I with <br /> fragmentation of chromosomes; (S) prophase II-like stage; (T) anaphase II-like stage; (U) telophase II-like stage. Green and red arrows highlight <br /> FISH signals corresponding to an arm region of chromosome I (BAC F1N21, green) and a subtelomeric region of chromosome II (BAC F11L15, <br /> red), respectively (panels B, C, H, M&#226;&#8364;&#8220;Q, S and U). White arrows highlight chromosome bridges seen during Atcom1-1 meiosis (R, U). Meiotic <br /> progression in pollen mother cells was followed after chromosomes were stained with DAPI. Scale bar: 10 mm. <br />  <br />  <br /> of the single strand by the recombinase Rad51 for subsequent                 Klein, 1997; Tsubouchi and Ogawa, 1998) and Com1/Sae2 <br /> strand invasion and D-loop formation (Li et al, 2004;                        (McKee and Kleckner, 1997; Prinz et al, 1997) in S. cerevisiae. <br /> Shinohara and Shinohara, 2004). Filament formation can be                    In such a situation, Spo11 foci were shown to accumulate in <br /> observed cytologically, as the formation of Rad51 foci                       mutant nuclei by visualising an epitope-tagged Spo11 (Prieler <br /> upon indirect immunofluorescence (IF) staining. While                        et al, 2005). We tried to address this question in A. thaliana <br /> large numbers of Rad51 foci appear transiently in wild-type                  using an AtSPO11-1-specific antibody (Sanchez-Moran et al, <br /> prophase cells, only very few (comparable to those found in                  2007). While AtSPO11-1 was virtually undetectable in meio- <br /> Atspo11-1-1 and Atmre11-3) are seen in Atcom1-1 cells                        cytes of wild-type cells (99.7% of prophase I cells show no <br /> (Figure 5; Supplementary Figure 4). One interpretation                       AtSPO11-1 foci, n &#194;&#188; 440), the same antibody detected <br /> of this result could be that meiotic DSBs are reduced or                     AtSPO11-1 in 98% of Atcom1-1 meiocytes (n &#194;&#188; 50), some of <br /> absent. We therefore performed IF for gH2AX (Sanchez-                        which showed striking AtSPO11-1 hyperaccumulation (30/50 <br /> Moran et al, 2007), a phospho form of a histone H2A variant                  cell showed very intensive staining, 19/50 showed staining <br /> representing a specific, local and fast response to DSBs                     with lower intensity) (Figure 5; Supplementary Figure 5A). <br /> (Rogakou et al, 1998; Fernandez-Capetillo et al, 2003;                       Interestingly, we find a similar hyperaccumulation of <br /> Friesner et al, 2005; Supplementary Figure 5B). This control                 AtSPO11-1 in meiocytes of Atmre11-3 and Atrad50 mutants <br /> confirmed the presence of high levels of DSBs in Atcom1-1                    (Supplementary Figure 5A). We summarise that available <br /> cells (and in Atmre11-3 and AtRad50 mutant cells), similar to                evidence places the defect of Atcom1-1 before AtRAD51 <br /> the staining observed in early stages in wild-type meiocytes                 filament formation and suggests a problem with AtSPO11-1 <br /> and consistent with the observation of fragmentation in                      removal. Thus, our study suggests that yeast Com1/Sae2 and <br /> anaphase I (Figure 3Q&#226;&#8364;&#8220;S). Thus, we infer that Atcom1-1                       Arabidopsis AtCOM1 are required for equivalent mechanisms <br /> mutants generate DSBs, but do not form Rad51 filaments                       during meiotic DSB repair. <br /> efficiently. <br />    A critical early step in repair is the removal of Spo11, which            AtCOM1 is required for resistance to mitomycin C <br /> remains covalently linked after it has cleaved a step known to               Aside from meiosis, the S. cerevisiae COM1/SAE2 gene is <br /> depend on Rad50 (; Alani et al, 1990), Mre11 (Nairz and                      required for genomic stability and for resistance to DNA <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                       The EMBO Journal        VOL 26 | NO 24 | 2007 5065 <br />        Analysis of Arabidopsis COM1/SAE2 gene <br />       C Uanschou et al <br />  <br />  <br />  <br />       Figure 4 AtCOM1 acts downstream of AtSPO11-1 and upstream of AtDMC1. Comparison of meiotic progression in the Atspo11-1-1 mutant <br />       (upper panel) and in the Atcom1-1Atspo11-1-1 double mutant (second panel from top). In both, zygotene-like stages are followed by <br />       progressive condensation and formation of univalents, which subsequently segregate at random, forming polyads at the end of meiosis II. <br />       Comparison of meiotic progression in the Atdmc1 mutant (third panel from top) and in the Atcom1-1Atdmc1 double mutant (lowest panel). <br />       Whereas Atdmc1 mutants form univalents, which segregate at random to give rise to polyads at the end of meiosis II, the Atcom1-1Atdmc1 <br />       double mutant resembles Atcom1-1, with fragmented chromosomes. Chromosomes are stained with DAPI. Scale bar: 10 mm. <br />  <br />  <br />       Figure 5 Atcom1-1 cells fail to form AtRAD51 foci, but accumulate AtSPO11-1. (A) Atcom1-1 cells fail to form AtRAD51 foci. <br />       Immunofluorescent staining of spreads of meiotic cells with antibodies directed against the axial element protein Asy1 (green) and against <br />       AtRad51 (red). Whereas in wild type (wt) numerous AtRAD51 foci are observed in zygotene, representing loci of meiotic DNA repair and <br />       recombination, only very few foci are seen in Atspo11-1-1, Atcom1-1 and Atmre11-3 mutants. Scale bar: 10 mm. (B) AtSPO11-1 is enriched in <br />       prophase I of Atcom1-1 mutant meiosis. Immunofluorescent staining of spreads of cells in prophase I stages of wild type (wt) and Atcom1-1 <br />       meiocytes with an antibody directed against AtSPO11-1 (red). DNA was counterstained with DAPI (blue). Scale bar: 10 mm. <br />  <br />  <br />  <br />  <br />       cross-linking agents, including mitomycin C (MMC) in vege-                type, true leaves do not develop in the mutant, while expan- <br />       tative cells (Lobachev et al, 2002; Wu et al, 2004). Figure 2B            sion of cotyledons, which does not require cell division, is <br />       shows wild-type and Atcom1-1 seedlings, which were germi-                 unaffected. The strong MMC sensitivity was 100% penetrant <br />       nated on media containing 40 mM MMC. In contrast to wild                  for homozygous Atcom1-1 mutants, whereas heterozygous or <br />  <br /> 5066 The EMBO Journal VOL 26 | NO 24 | 2007                                                           &amp; 2007 European Molecular Biology Organization <br />                                                                                                   Analysis of Arabidopsis COM1/SAE2 gene <br />                                                                                                                          C Uanschou et al <br />  <br />  <br />  <br /> wild-type plants were resistant. However, no difference in        plants because of AtSPO11-1 not being removed from DNA <br /> sensitivity to a number of other genotoxic agents was found       ends at DSB sites, thereby blocking processing of DNA ends. <br /> in a series of experiments described in the Supplementary         Future experiments are needed to resolve the question as to <br /> data. In addition, in yeast, diploid com1/sae2D mutants were      whether the second, meiotically active Arabidopsis SPO11 <br /> only very weakly sensitive to X-ray or a radiomimetic alkylat-    protein, AtSPO11-2, behaves similarly to AtSPO11-1 in an <br /> ing agent in our hands (McKee and Kleckner, 1997; Prinz           Atcom1 mutant background. <br /> et al, 1997). Work in recent years has implicated Com1/Sae2 <br /> in the processing of very selective lesions such as hairpin       Somatic function of the AtCOM1 protein <br /> capped DNA ends (data not shown; Rattray et al, 2001;             Expression data for the AtCOM1 gene, presented by Deveaux <br /> Lobachev et al, 2002) or interstrand cross-links (Wu et al,       et al (2000) and confirmed by this study, showed strong <br /> 2004). The selective sensitivity of vegetative cells of both      accumulation of the AtCOM1 transcript upon exposure to IR <br /> Atcom1-1 and yeast com1/sae2D mutants to MMC is further           in somatic tissues. Furthermore, data presented by Garcia et al <br /> evidence for the conservation of their roles in evolution and,    (2003) show that transcription of AtCOM1 is strictly dependent <br /> furthermore, may reflect a specificity of AtCOM1 for the          on ATM, a conserved protein kinase known to mediate signal- <br /> repair of only a subset of DNA lesions.                           ling of DNA lesions (Shiloh, 1998). Thus, to unravel a potential <br />                                                                   somatic function of the AtCOM1 protein, the Atcom1-1 mutant <br />                                                                   was tested for resistance to various genotoxic treatments. <br /> Discussion                                                        Under the conditions tested, only the presence of MMC led to <br /> This study describes the first identification and characterisa-   strong inhibition of mutant seedling growth. In contrast, all <br /> tion of a Com1/Sae2 homologue in a higher eukaryote, the          other treatments including IR, did not affect the growth of <br /> AtCOM1 gene of A. thaliana. This study provides evidence          homozygous Atcom1-1 mutants compared to heterozygous or <br /> that both female and male meiosis are affected by the Atcom1      wild-type plants. This may reflect a specificity of AtCOM1&#226;&#8364;&#8482;s <br /> mutation, which thus leads to sterility. Atcom1 mutants do not    involvement in the repair of a subset of DNA lesions. It is <br /> form any viable microspores or macrospores, as judged from        conceivable that, upon DNA damage, ATM-mediated signalling <br /> our analysis of gametogenesis. Furthermore, Atcom1 mutants        leads to broad transcriptional induction of DNA repair-related <br /> do not form seeds, even when pollinated with wild-type            genes, and that it depends on the actual type of DNA damage, <br /> pollen. Analysis of DAPI-stained chromosomes reveals that         which of the corresponding gene products is effectively needed <br /> during progression of both male and female meiosis, severe        for repair. In somatic tissues, AtCOM1 seems to be needed for <br /> chromosome fragmentation occurs. Chromosome pairing,              the repair of interstrand DNA cross-links, the dominant effect of <br /> visualised by DAPI and FISH analysis, never occurred in           MMC, but not for other types of DNA damage, which are <br /> Atcom1 mutants. Chromosome fragmentation and failure of           predominantly formed by IR, HU, cisplatin, bleomycin or <br /> pairing indicate that meiotic DSBs are produced, but down-        aphidicolin. The selective sensitivity of vegetative cells of <br /> stream events such as DNA end processing or heteroduplex          both Atcom1-1 and yeast com1/sae2D mutants to DNA cross- <br /> formation are impaired in Atcom1 mutants. Epistasis analysis      linking agents provides further evidence for the conservation of <br /> clearly shows that the DNA fragmentation observed in Atcom1       their roles in evolution. <br /> mutant plants depends on the activity of the AtSPO11-1 <br /> protein. This result places Atcom1 downstream of Atspo11-1.       COM1/SAE2 homologues are present in higher <br /> In contrast, Atcom1-1 Atdmc1 double mutants show severe           eukaryotes <br /> chromosome fragmentation and, therefore, resemble the             As demonstrated in Figure 1B, AtCOM1 is much more closely <br /> Atcom1-1 mutant rather than the Atdmc1 mutant. This result        related to the vertebrate COM1/SAE2 homologues than to the <br /> implies that meiotic DSBs are formed in Atdmc1 mutants, and       fungal branch. The mammalian relative of AtCOM1 and, <br /> that their repair depends on AtCOM1. This result places           thus, also of yeast Com1/Sae2, is CtIP (Fusco et al, 1998; <br /> AtCOM1 upstream of the RecA-related DMC1 protein.                 Schaeper et al, 1998), which has been strongly implicated in <br />    The mechanistic role of Com1/Sae2 is still unknown.            DNA repair. Initially, CtIP was identified as a protein that <br /> Nevertheless, it has been established that Com1/Sae2 is           binds to Rb, BRCA1 and CtBP1 (Fusco et al, 1998; Schaeper <br /> needed for the removal of Spo11 from DNA during meiosis           et al, 1998; Meloni et al, 1999; Yu and Baer, 2000). CtIP&#226;&#8364;&#8482;s <br /> in yeast. As an intermediate of the DNA cleavage reaction,        interaction with the transcriptional regulators CtBP, Rb and <br /> Spo11 is covalently bound to the 50 end phosphate of the DNA      LMO4 involves it in the regulaton of cell-cycle and DNA <br /> at the DSB site. Removal of the Spo11 protein occurs by           repair-related proteins (Yu et al, 1998; Sum et al, 2002; Liu <br /> nicking the DNA next to the cleavage site, thereby releasing      and Lee, 2006). CtIP is most highly expressed in thymus and <br /> the catalytic protein attached to a few nucleotides (Neale        testis (Wong et al, 1998), where DNA DSBs are naturally <br /> et al, 2005). If Com1/Sae2 is not present, this reaction does     generated. Yu et al (2006) show that CtIP is ubiquitinated in a <br /> not occur and subsequent steps in meiotic progression are         BRCA1/BARD1-dependent manner following DNA damage <br /> blocked. The present study reveals that the plant protein         and that this modification leads to its association with <br /> AtCOM1 is involved at the same stage during meiosis.              chromatin, not to its degradation. Upon DNA damage, CtIP <br /> Moreover, evidence is presented that Atcom1-1 mutant plants       associates with BRCA1/BARD1, Rad50, Mre11 and NBS1 <br /> accumulate the AtSpo11-1 protein during prophase I of meio-       (Greenberg et al, 2006). Furthermore, CtIP&#226;&#8364;&#8482;s interaction <br /> sis. Furthermore, the strand invasion mediator AtRAD51,           with BRCA1 seems to be essential for the activation of a <br /> which forms numerous foci on meiotic chromosomes and is           DNA damage checkpoint in G2 (Yu and Chen, 2004; <br /> visible from leptotene to pachytene in wild-type meiocytes (Li    Greenberg et al, 2006), possibly mediated by CtIP&#226;&#8364;&#8482;s phos- <br /> et al, 2004), cannot be detected in Atcom-1-1 mutants. These      phorylation by ATM, a process that depends on BRCA1 (Foray <br /> results imply that processing of DSBs is impaired in mutant       et al, 2003). Homozygous Ctip&#226;&#8364;&#8220;/&#226;&#8364;&#8220;-<span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice die during <br />  <br /> &amp; 2007 European Molecular Biology Organization                                           The EMBO Journal      VOL 26 | NO 24 | 2007 5067 <br />        Analysis of Arabidopsis COM1/SAE2 gene <br />       C Uanschou et al <br />  <br />  <br />  <br />       embryonic stage 4 and heterozygous Ctip &#195;&#190; /&#226;&#8364;&#8220; mice are prone            dark photoperiod. Seeds of the SALK_061706 (Atcom1-1) and <br />                                                                              SALK_122386 (Atcom1-2) lines were obtained from the Nottingham <br />       to developing tumours (Chen et al, 2005). While all previous <br />                                                                              Arabidopsis Stock Center (Nottingham, UK). Mutants and double <br />       data are consistent with a role of CtIP in DSB repair similar to       mutants were identified by PCR (see Supplementary data). <br />       Com1/Sae2, these observations have not provided a clue                 Transformation of plants heterozygous for the Atcom1-1 mutation <br />       regarding the molecular role of CtIP. Both CtIP and Com1/              with pBIB-Hyg/gAtCOM1, utilising Agrobacterium tumefaciens <br />                                                                              strain GV3101, was performed as described previously (Bechtold <br />       Sae2 are phosphorylated by ATM/Tel1 kinases in response to <br />                                                                              et al, 1993). <br />       DNA damage (Foray et al, 2003; Baroni et al, 2004), cooperate <br />       with Mre11 and Rad50 (Lobachev et al, 2002; Lisby et al, <br />                                                                              Vector construction <br />       2004; Greenberg et al, 2006) and play a role in DNA damage             For construction of the complementing pBIB-Hyg/gAtCOM1, a <br />       checkpoint signalling (Yu and Chen, 2004; Clerici et al, 2006).        4811 bp genomic DNA fragment derived from BAC clone ATF4F15 <br />       CtIP, however, also interacts physically with BRCA1, Rb and            (ABRC, Columbus, Ohio) by digestion with BsmI and subsequent T4 <br />       CtBP1, which are not part of the yeast DNA repair repertoire.          polymerase treatment, was cloned into the SmaI restrictions site of <br />                                                                              the plant binary vector pBIB-Hyg (Becker, 1990) and subsequently <br />       Thus, CtIP and possibly AtCOM1 may have acquired acces-                used for transformation. <br />       sory functions in addition to those conserved with yeast <br />       Com1/Sae2. So far, a detailed analysis of CtIP function has <br />                                                                              Analysis of meiotic chromosomes <br />       been complicated by the embryonic lethality caused by                  DAPI staining of male meiotic chromosomes was performed as <br />       homozygous CtIP deficiency in mice. However, heterozygous              described previously (Ross et al, 1996). Female meiotic chromo- <br />       CtIP leads to increased incidence of cancer (Chen et al, 2005).        somes were observed as described previously (Motamayor et al, <br />       Our findings lead to the prediction that CtIP is required for          2000), except that pictures were taken with a Zeiss Axioplan 2 <br />                                                                              microscope, including an LSM 510 Laser module. Data processing <br />       meiotic DSB repair and for resistance to certain genotoxic             was performed with Zeiss LSM 510 Meta and Helicon Focus <br />       agents (e.g. MMC). CtIP may function by modulating the                 software. FISH and fluorescent immunolocalisation analysis of <br />       nuclease activity of Mre11 and may be an important factor in           ASY1 and AtRAD51 was performed as described previously <br />       processing hairpin capped DNA ends and DNA ends blocked                (Kerzendorfer et al, 2006), except that both the anti-RAD51 <br />                                                                              antibody (rat) and the anti-ASY1 antibody (rabbit) were used in <br />       by covalently bound proteins (such as in topoisomerase II in           1:500 dilutions. The polyclonal rabbit antibody against AtSPO11-1 <br />       the presence of topo II poisons). Interestingly, deletion of           (a gift from F.C.H Franklin) was diluted 1:100 and a polyclonal <br />       RAD50 is embryonic-lethal (Luo et al, 1999) and a hypo-                rabbit anti-phospho-H2AX (Ser 139) (Upstate) antibody was diluted <br />       morphic mutation of MRE11 severely compromises embryo                  1:100. Immunolocalisation analysis of AtSPO11-1 and gH2AX <br />                                                                              antibodies was carried out on squashed prophase I meiocytes from <br />       viability (Theunissen et al, 2003) in mammals, just as dele-           Atcom1-1 mutant plants as described previously (Page et al, 1998; <br />       tion of CtIP (Chen et al, 2005), but all three genes are not           Sanchez-Moran et al, 2007). <br />       essential in yeast and in plants. This is consistent with <br />       evolutionary conservation of a potential regulatory role of            Yeast spore viability assay <br />       Com1/Sae2 for the Mre11 complex. The function of yeast                 Yeast strain SK1 (with the following relevant genotype: MATa/ <br />       Com1/Sae2 may overlap with that of flap endonuclease Fen1,             alpha, ura3/ura3, com1::KanMX4/com1::KanMX4), was trans- <br />       because they display synthetic lethality, a relation that              formed with plasmid pRS316, including the COM1/SAE2 coding <br />                                                                              sequence (with additional 682 bp upstream and 251 bp downstream <br />       might also be conserved between mammalian Fen1 and                     regions of the COM1/SAE2 coding sequence) between the SacII and <br />       CtIP. Mammalian Fen1 suppresses tumour progression                     HpaI restriction sites. Indicated point mutations were introduced by <br />       (Kucherlapati et al, 2002). The phenotype of Atcom1-1 pro-             standard techniques and verified by sequencing. Yeast spore <br />       vides information which links yeast Com1/Sae2 and mam-                 viability was determined as described by Prinz and co-workers <br />                                                                              (Prinz et al, 1997). <br />       malian CtIP. Based on the detailed knowledge on Com1/Sae2, <br />       we suggest CtIP to be required for genome stability as a <br />                                                                          <br />       tumour suppressor, for fertility through meiotic DNA repair <br />                                                                        <br />       and possibly for V(D)J recombination, because in yeast                 <br />       Com1/Sae2 is instrumental together with Mre11 to process <br />       hairpins (Rattray et al, 2001; Lobachev et al, 2002 our own <br />       observations, manuscript in preparation). Com1/Sae2 and                <br />       CtIP have been studied separately, but the knowledge of             <br />       their relationship should strongly stimulate both hitherto              <br />       separated fields.                                                  <br />       Materials and methods                                               <br />       Plant growth conditions and plant transformation                        <br />       All plants were germinated on soil after 2 days of stratification in    <br />       water at 41C. Plants were grown at 221C with a 16:8 h light:            <br />  <br /> </body></html>