<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />                            The Box H/ACA snoRNP assembly factor Shq1p is a chaperone <br />                            protein homologous <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> Hsp90 co-chaperones that binds to the <br />                            Cbf5p enzyme <br />  <br />                             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Box H/ACA small nucleolar ribonucleoproteins (snoRNPs) are responsible for the formation of <br />                                  pseudouridine in a variety of RNAs, and are essential for ribosome biogenesis, modification of <br />                                  spliceosomal RNAs and telomerase stability. The mature snoRNP has been reconstituted in vitro and <br />                                  is composed of a single RNA and four proteins. However, snoRNP biogenesis in vivo requires <br />                                  multiple factors to coordinate a complex and poorly understood assembly and maturation process. <br />                                  Among the factors required for snoRNP biogenesis in yeast is Shq1p, an essential protein necessary <br />                                  for the snoRNA stable expression. We have found that Shq1p consists of two independent domains <br />                                  that contain Casein Kinase 1 phosphorylation sites. We also demonstrate that Shq1p binds the <br />                                  pseudourydilating enzyme Cbf5p through the C-terminal domain in synergy with the N-terminal <br />                                  domain. The NMR solution structure of the N-terminal domain has striking homology to the &#226;&#8364;&#732;Chord <br />                                  and Sgt1&#226;&#8364;&#8482; (CS) domain of known Hsp90 co-chaperones, yet Shq1p does not interact with the yeast <br />                                  Hsp90 homologue in vitro. Surprisingly, Shq1p has stand-alone chaperone activity in vitro. This <br />                                  activity is harbored by the C-terminal domain but it is increased by the presence of the N-terminal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  domain. These results provide the first evidence of a specific biochemical activity for Shq1p and a <br />                                  direct link to the H/ACA snoRNP. <br />  <br />  <br />                            Keywords <br />                                  H/ACA snoRNP biogenesis; Shq1; CS domain; chaperone <br />  <br />  <br />  <br />  <br />                            &#194;&#167;To whom correspondence should be addressed: tel 206.543.7113; fax 206 685 8665; email varani@chem.washington.edu. <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers <br />                            we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting <br />                            proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could <br />                            affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Godin et al.                                                                                            Page 2 <br />  <br />  <br />                               Introduction <br />                                           Box H/ACA small nucleolar (sno)RNAs are a class of non-coding RNA that play a fundamental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           role in ribosome biogenesis, spliceosome assembly, and telomere maintenance (for review see <br />                                           1 and 2). In mammals, each of the ~100 H/ACA snoRNAs combines with four highly conserved <br />                                           protein components to form enzymatically active small nucleolar ribonucleoproteins <br />                                           (snoRNPs): the Cbf5p catalytic subunit (dyskerin in humans or NAP57 in rodents), Nop10p, <br />                                           Nhp2p (L7Ae in archaea), and Gar1p. These enzymes primarily function in the nucleolus to <br />                                           catalyze site-specific pseudouridylation of pre-ribosomal (pre-r)RNA3, 4, but a fraction modify <br />                                           small nuclear (sn)RNPs when directed to Cajal bodies by Cajal body specific (sca)RNAs5, 6. <br />                                           Some H/ACA snoRNPs do not have pseudouridylase activity, including yeast snR30 (E1/U17 <br />                                           in vertebrates) that participates in pre-rRNA cleaveage7, 8 and the telomerase H/ACA scaRNA <br />                                           motif9 that is essential in vivo for proper RNA processing, activity, and nucleolar <br />                                           localization10, 11, 12. <br />  <br />                                           The macromolecular architecture and the enzymatic role of each protein have been studied <br />                                           with in vitro reconstituted archaeal small (s)RNPs, consisting of a single hairpin small (s)RNA <br />                                           and the four evolutionarily conserved protein homologues (for review see 13). However, in <br />                                           eukaryotes, H/ACA snoRNP biogenesis pathway is surprisingly complex requiring several <br />                                           more proteins than are present in the mature functional holoenzyme (for review see 14). <br />                                           Assembly of the mature proteins follows a coordinated stepwise process requiring the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           formation and disassembly of intermediate structures within specific sub-nuclear <br />                                           compartments15, 16. Stable expression of H/ACA snoRNAs requires the co-transcriptional <br />                                           formation of an inactive RNP by the association of Cbf5p, Nop10p, Nhp2p, and the H/ACA <br />                                           assembly factor Naf1p17, 18. Gar1 is not found at the transcription site with the other snoRNP <br />                                           proteins but is concentrated in the nucleolus19 and within Cajal bodies20; this protein likely <br />                                           associates with the RNP late in the maturation process. Cbf5p mediates the interaction of Naf1p <br />                                           with the other snoRNP proteins during snoRNA transcription15, 17, 18, yet Naf1p does not <br />                                           localize with the other snoRNP proteins to the nucleolus or Cajal bodies. Since the specificity <br />                                           of Naf1p for the H/ACA snoRNP arises <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> its Gar1-like domain21, which makes it interact <br />                                           with Cbf5p in a manner that is competitive with Gar1, the snoRNP is presumed to undergo a <br />                                           remodeling event to introduce Gar1 and form mature functional snoRNP particles. <br />  <br />                                           A second eukaryotic H/ACA snoRNP assembly factor, Shq1, was found to associate with <br />                                           Naf1p in proteomic, in vitro pull-down, and co-localization studies22. Genetic depletion of <br />                                           Shq1p results in ribosomal RNA processing defects due to the loss of stable accumulation of <br />                                           H/ACA snoRNAs. Shq1p was also found to associate with in vitro translated Nhp2p22 and <br />                                           flag-tagged Cbf523, providing an additional and direct link to H/ACA snoRNPs biogenesis. <br />                                           Since levels of Nhp2p protein are unaffected in Shq1p depleted cells, Shq1p probably functions <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           directly in the maturation of H/ACA snoRNPs. <br />  <br />                                           Sequence analysis indicates that Shq1p has two structural domains. We show that these two <br />                                           domains do not interact with each other, yet the interaction between Shq1p and Cbf5, that we <br />                                           report here, requires both domains. We report the solution structure of its N-terminal domain <br />                                           which shows that it adopts a &#226;&#8364;&#732;Chord and Sgt1&#226;&#8364;&#8482; (CS) fold related to the Hsp90 co-chaperones <br />                                           Sgt1 and Sba1p/p23 proteins but, as recently reported, this domain does not interact directly <br />                                           with the yeast Hsp90 homologue24. We further show that the C-terminal domain does not <br />                                           rescue binding of Shq1p to Hsp90. However, we report that the C-terminal domain has stand- <br />                                           alone chaperone-like activity that is enhanced in the complete protein. We also observe that <br />                                           both domains of Shq1p are substrates for Casein Kinase 1 (CK1) phosphorylation. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 3 <br />  <br />  <br />                               Results <br />                               Shq1p has two structurally independent domains <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Yeast Shq1p consists of 507 residues with very high sequence conservation from yeast to <br />                                           humans (Figure 1). Comparison of Shq1p with sequences from known domains or motifs using <br />                                           the interpro database25 suggests that Shq1p contains two domains, a CS domain within the first <br />                                           100 residues and a C-terminal sequence unique to Shq1. To verify this assumption, purified <br />                                           wild type Shq1p was digested three different ways with proteinase K, trypsin, or chymotrypsin. <br />                                           Analysis by SDS-PAGE consistently showed the presence of a small ~12kDa product and a <br />                                           larger fragment that ranged in size from ~30&#226;&#8364;&#8220;45kDa. Gel filtration revealed that the two <br />                                           domains eluted as independent entities at the appropriate molecular weight identified from <br />                                           SDS-PAGE (data not shown), suggesting that the two domains do not interact with each other <br />                                           strongly or self-aggregate. <br />  <br />                                           The N-terminal construct Shq1Np (Figure 1B) identified from partial proteolysis ranges from <br />                                           residue I2 to N132 (~15kDa). It gave uniform peak shapes and dispersion in a 15N-HSQC <br />                                           spectrum (Figure 2A), indicative of a well-folded protein. A 15N-HSQC of the full <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> Shq1p <br />                                           contained all resonances of the Shq1Np fragment with chemical shifts consistent with the <br />                                           individually purified N-terminal domain (Figure 2B), supporting the conclusion that the N- <br />                                           terminal domain does not associate with the larger C-terminal domain. Altered chemical shifts <br />                                           were observed for S27, G30, T64, K80, D89&#226;&#8364;&#8220;L92, A98, and the amines of Q10 and N75. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Shq1p interacts with Cbf5 <br />                                           To establish if and how Shq1p associates with H/ACA protein components during snoRNP <br />                                           biogenesis, we analyzed the interaction of each of its domains with snoRNP components. <br />                                           Earlier studies showed that Shq1p is essential for snoRNA accumulation and it was initially <br />                                           reported to interact with Naf1p and Nhp2p22. However, we observed no interaction with the <br />                                           Gar1-domain of Naf1p by pull-down assays or even by NMR (data not shown), confirming <br />                                           the results obtained using Tap-tag pull-downs of Naf1p which failed to retrieve Shq1p18. We <br />                                           found instead that full-length Shq1p was pulled down efficiently by refolded His-tagged <br />                                           Cbf5&#206;&#8221;p (Figure 2C, lane 2) confirming previous observations that Shq1p co-purifies with over- <br />                                           expressed flag-tagged Cbf523 and consistent with recent studies of mammalian SHQ126. In <br />                                           addition, Shq1Cp appears to be weakly associated with Cbf5&#206;&#8221;p, but Shq1Np does not interact <br />                                           with Cbf5p at all (Figure 2C, lane 3 and 4). These results support a role for both domains of <br />                                           Shq1p in Cbf5p recognition, but reveal a prominent role for the C-terminal domain. <br />  <br />                               The Shq1p N-terminus is structurally homologous to Hsp90 co-chaperones <br />                                           The structure of the putative CS domain of Shq1Np (PDB 2K8V) was determined by NMR <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           using methods described below and calculated with CYANA 2.1 based on 4521 NOEs selected <br />                                           from 15N-edited NOESY, 13C-edited NOESY, and 1H,1H-NOESY, as well as additional <br />                                           dihedral angle and hydrogen bond constraints generated from the NMR data as described. In <br />                                           the early rounds of CYANA, each assignment was validated by a network NOEs arising from <br />                                           neighboring residues and residues that satisfy the tertiary fold. The NOE assignments were <br />                                           used to derive a total of 2621 meaningful upper distance limits (~20 constraints/residue) in the <br />                                           last round of refinement and produced 20 structures with the lowest violations, the best overall <br />                                           statistics (Table 1) and excellent structural convergence (Figure 3A). The high quality is <br />                                           reflected in the tight convergence of the 20 structures with the respective average backbone <br />                                           and heavy atom r.m.s.d. to the mean of 0.25&#195;&#8230; and 0.71&#195;&#8230; over the structured region and the <br />                                           very favorable Ramachandran statistics. The exceptional structural statistics and strong <br />                                           validation of the predicted CS domain fold (discussed below) indicated that the CYANA <br />                                           solution was correct and additional refinement using Residual Dipolar Coupling was not <br />                                           necessary. <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                               Page 4 <br />  <br />  <br />                                           Shq1Np contains 7 &#206;&#178;-strands and a small &#206;&#177;-helix kinked by a single proline residue that packs <br />                                           into a compact two-layer &#206;&#178;-sandwich (Figure 3B), as in the recently-reported crystal structure <br />                                           of the same domain24. This topology is consistent with known CS domain proteins Sgt1, Sba1p/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           p23, and Siah-interacting protein (SIP) where the larger &#206;&#178;-sheet contains 4 antiparallel strands <br />                                           &#206;&#178;1, &#206;&#178;2, &#206;&#178;6, and &#206;&#178;7 and the smaller has 3 anti-parallel strands &#206;&#178;3, &#206;&#178;4, and &#206;&#178;5. The two sheets <br />                                           pack against each other through a vast network of hydrophobic side chains. The hydrophobic <br />                                           core accommodates 7 aromatic residues, all of which are very highly conserved and tightly <br />                                           packed against the abundant aliphatic side chains that anchor the &#206;&#178;-strands within each &#206;&#178;-sheet <br />                                           and across the sandwich interior. <br />  <br />                                           The NMR structure superposes very well with the crystal structure (3EUD, 1.26 &#195;&#8230; r.m.s.d), <br />                                           with only small differences in the conformation of the &#206;&#178;2&#226;&#8364;&#8220;&#206;&#178;3 loop which is involved in crystal <br />                                           packing24. In solution, residues 100&#226;&#8364;&#8220;134 are disordered, confirming that the structural core of <br />                                           the N-terminal domain encompasses residues 1&#226;&#8364;&#8220;100, corresponding to the crystallized <br />                                           fragment. A search on the DALI server27 retrieved the expected structural homologues of <br />                                           Shq1p within the CS domain family proteins Sgt1 (1RL1, 7.8 Z-score, 1.9&#195;&#8230; r.m.s.d), p23 (yeast <br />                                           Sba1) (1EJF, 7.8 Z-score, 2.2&#195;&#8230; r.m.s.d.), and Siah-interacting protein (1X5M, 7.6 Z-score, <br />                                           2.6&#195;&#8230; r.m.s.d.). More distant similarities were found with several small heat shock protein-like <br />                                           (sHsp-like) chaperones with the predominant &#206;&#177;-crystallin fold Hsp26 (2H53, 6.8 Z-score, 1.9&#195;&#8230; <br />                                           r.m.s.d), Hsp16.9B (1GME, 6.8 Z-score, 2.2&#195;&#8230; r.m.s.d), and M. jannaschii sHsp (1SHS, 7.1 Z- <br />                                           score, 2.2&#195;&#8230; r.m.s.d). In comparison to CS domain proteins, Hsp20 chaperones have an extended <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           &#206;&#178;4&#226;&#8364;&#8220;&#206;&#178;5 loop that facilitates dimerization by domain swapping28; in the CS domain proteins this <br />                                           loop is considerably shorter; the shortest so far is found in Shq1Np. <br />  <br />                                           The edges of the sandwich are sealed on three of the four sides forming a robust and compact <br />                                           globular structure. Hydrogen bonded amides remained observable in 2H2O for at least four <br />                                           months and at temperatures of up to 55&#194;&#176;C. Several residues located within loops between &#206;&#178;- <br />                                           strands 1&#226;&#8364;&#8220;2, 3&#226;&#8364;&#8220;4, 4&#226;&#8364;&#8220;5, and from strand &#206;&#178;7 to the &#206;&#177;-helix showed protection from exchange as <br />                                           well, suggesting that the loops are rigid and highly ordered. The lack of hydrogen bond <br />                                           restraints along strands &#206;&#178;3 and &#206;&#178;6 suggests instead that this region is solvent exposed and <br />                                           corresponds to a noticeable crevice that is not observed in either Sba1p/p23 or Sgt1. This pocket <br />                                           is unique to Shq1Np and forms as a result of the lack of hydrophobic packing between the <br />                                           shorter internal Ser and Ala side chains of &#206;&#178;6 and the larger hydrophobic residues of &#206;&#178;3. <br />                                           Analysis of the crystal packing of the crystal structure of Shq1p shows that this pocket is always <br />                                           involved in protein-protein interactions24. Of notice is also Lys80, a strictly conserved residue <br />                                           which assumes an unusual position for a charged residue, being deeply buried within the <br />                                           hydrophobic core. <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Mutation</span> of this residue has been shown to be lead to severe growth defects <br />                                           in yeast24. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Shq1p does not interact directly with Hsp90 <br />                                           The CS domains of both Sba1p/p23 and Sgt1 interact with the ubiquitous macromolecular <br />                                           chaperone Hsp9029, 30. Both proteins share the CS domain fold and have been shown in x-ray <br />                                           crystallographic structures to interact with Hsp90 in very distinct ways31, 32, 33. The Hsp90 <br />                                           binding interface of both Sba1p/p23 and Sgt1 overlap with the regions displaying the most <br />                                           highly conserved surface exposed residues found along different faces of each CS domain. The <br />                                           Sba1p/p23-Hsp90 interface includes several residues from the unique C-terminal &#206;&#178;8 strand and <br />                                           wraps around the &#206;&#178;5-&#206;&#178;8-&#206;&#178;1 edge of Sba1p/p23 to the surface exposed residues found in the <br />                                           &#206;&#178;2&#226;&#8364;&#8220;&#206;&#178;3 loop and &#206;&#178;6&#226;&#8364;&#8220;&#206;&#178;7 loop along the lower face of the large 4 stranded &#206;&#178;-sheet (Figure 4A) <br />                                           31. This protein interacts with the amino terminus of ATP-bound Hsp90 constraining features <br />                                           important for ATP turnover and, as a result, regulates the rate of ATP hydrolysis and stabilizes <br />                                           this bound state for chaperone activity. The Sgt1 CS domain recently crystallized with Hsp90 <br />                                           shows a distinct mode of interaction with the Hsp90 N-terminus. Sgt1 utilizes highly conserved <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 5 <br />  <br />  <br />                                           residues found on the first 3 strands of the large &#206;&#178;-sheet belonging to its CS domain (Figure <br />                                           4B) but does not interact with features of Hsp90 important for ATP hydrolysis32, 33. Unlike <br />                                           Sba1p/p23, Sgt1 does not require ATP to interact with Hsp9034, but appears to behave instead <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           as an adaptor that links Hsp90, through its CS domain, to other client proteins35, 36, 37. <br />  <br />                                           The structural homology of Shq1p with CS domain proteins obviously suggests that Shq1p is <br />                                           also an Hsp90 co-chaperone. Highly conserved residues on the Shq1Np surface appear along <br />                                           the &#206;&#178;5-&#206;&#177;1-&#206;&#178;1 face and within the &#206;&#178;2&#226;&#8364;&#8220;&#206;&#178;3 loop that corresponds to the Sba1p/p23 Hps90-interface <br />                                           (Figure 4A and B). When comparing the surface electrostatic potentials, we observe that <br />                                           Shq1Np also shares a similar electrostatic distribution with the Hsp90 binding face of Sba1p/ <br />                                           p23 (Figure 4A), suggesting that Shq1Np could interact with Hsp90 in a similar way to what <br />                                           has been seen in the Sba1p/p23-Hsp90 complex. However, surface exposed residues along the <br />                                           Sba1/p23 face important for Hsp90 recognition are not found with the same Shq1Np surface. <br />  <br />                                           In order to establish whether the CS domain of Shq1p is capable of interacting with Hsp90 in <br />                                           vitro, we expressed and purified full length Hsp82p (the yeast Hsp90 homologue). As recently <br />                                           reported24, 26, we observed no interaction between Hsp82p and the N-terminal Shq1p domain <br />                                           by NMR, pull-down assay or gel filtration, both in the presence or absence of ATP or ATP <br />                                           analogs. Reasoning that perhaps binding surfaces on either protein could be masked in the full <br />                                           proteins and only exposed by some unknown regulatory event, we also independently <br />                                           expressed the N-terminal, middle, and C-terminal domains of Hsp82p. We then probed for the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           interaction with Shq1p of each of these three constructs independently, but also failed to <br />                                           observe any interaction (data not shown). Since we have shown that Shq1p interacts with Cbf5p <br />                                           prominently through the C-terminal domain, and that the N-terminal domain does not bind <br />                                           Cbf5p by itself, we also probed the interaction of Hsp82p with full-length Shq1p and the <br />                                           separate Shq1p N- and C-terminal domains. As can be seen in Figure 4C, no interaction could <br />                                           be observe between Hsp82p and the complete full length Shq1p or even with the separate <br />                                           Shq1Np or Shq1Cp domain by pull-down assays or gel filtration, both in the presence or <br />                                           absence of ATP or ATP analogs. These results conclusively demonstrate that neither the N- or <br />                                           C-terminal domains of Shq1p are capable of binding Hsp82p in vitro. <br />  <br />                               Shq1p is phosphorylated by Casein Kinase I <br />                                           While testing the interaction between Shq1Np and Hsp90 by NMR, we observed small <br />                                           chemical shift changes of Shq1Np residues in the presence of ATP (data not shown). These <br />                                           changes are not indicative of specific ATP binding since the same changes were observed with <br />                                           GTP and were not observed in the absence of Mg2+ counter ions. Additionally we could not <br />                                           identify any ATP binding when using electrospray ionization mass spectrometry or radioactive <br />                                           binding assays (data not shown). However, these chemical shift changes drew our attention to <br />                                           two deep pockets located on the &#206;&#178;3&#226;&#8364;&#8220;&#206;&#178;6 face and the pocket generated by Leu44 extending from <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           the &#206;&#178;4&#226;&#8364;&#8220;&#206;&#178;5 loop to Phe26 (Figure 5A). These two pockets are not found in either Sba1p/p23 or <br />                                           Sgt1 that have longer &#206;&#178;4&#226;&#8364;&#8220;&#206;&#178;5 loops and where the &#206;&#178;3&#226;&#8364;&#8220;&#206;&#178;6 face is filled by aromatic residues on <br />                                           both &#206;&#178;-strands. Interestingly, residual electron density in the pocket located on the &#206;&#178;3&#226;&#8364;&#8220;&#206;&#178;6 face <br />                                           in one of the molecules of the asymmetric <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> of the crystal structure of Shq1p24 reveals an <br />                                           unidentified bound small molecule. <br />  <br />                                           The pockets contain three Serines, Ser27, Ser45 and Ser63; interestingly, the Ser63 hydroxyl <br />                                           group is within ~3&#195;&#8230; of the Asp59 acetate, suggestive of a catalytically active site that may <br />                                           facilitate abstraction of the hydrogen from the Serine hydroxyl. We hypothesized that the <br />                                           serines within these pockets are substrates for post-translation phosphorylation but did not <br />                                           observe autophosphorylation upon incubation of Shq1Np or Shq1FLp with 32P-[&#206;&#179;]ATP (data <br />                                           not shown). Consistent with this suggestion, KinasePhos38 predicts that Ser63, found within <br />                                           the consensus sequence (D/E/SpXXS) for Casein Kinase 1 (CK1) phosphorylation, is a <br />                                           potential phosphorylation site. CK1 plays an important role in many diverse cellular activities <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 6 <br />  <br />  <br />                                           by phosphorylating regulatory proteins that participate in membrane transport, cell division, <br />                                           DNA repair, circadian rhythms, and nuclear localization (for review see 39). Using the <br />                                           CK1&#206;&#181;or&#206;&#180; isoform of this kinase, we observed transfer of the 32P-[&#206;&#179;]ATP &#206;&#179;-phosphate group to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           the R3 region of APC (adenomatis polyposis coli), a protein that facilitates nuclear export of <br />                                           &#206;&#178;-catenin and is hyperphosphorylated by CK140, as a positive control for kinase activity. We <br />                                           observed that CK1 phosphorylates each construct of Shq1p indicating that Shq1p is indeed a <br />                                           CK1 substrate (Figure 5B). The CK1 phosphorylation site within the Shq1p C-terminal domain <br />                                           is predicted by KinasePhos to reside at Ser462. Rna15 from yeast, a protein involved in <br />                                           processing RNAPII transcription products41, was not predicted to be a CK1 substrate by <br />                                           KinasePhos nor is it known as phosphoprotein. As expected, Rna15p was not phosphorylated <br />                                           in a negative control experiment, demonstrating that Shq1Np is a bona fide substrate of CK1. <br />  <br />                                           Since CK1 is known to be a promiscuous enzyme42 in vitro, its specificity for Ser63 was tested <br />                                           by generating a Ser63Ala <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> of the Shq1Np construct. However, phosphorylation <br />                                           of Shq1Np-S63A by CK1 remained unchanged; we reasoned that the specific activity of CK1 <br />                                           on Ser63 may be masked by non-specific phosphorylation of residues in the C-terminal tail of <br />                                           the protein. Consistent with this hypothesis, truncation of both Shq1Np constructs to residue <br />                                           Gly101 (Shq1Np-101wt and Shq1Np-101S63A) greatly reduced the extent of phosphorylation <br />                                           by CK1 and showed a significant decrease in the phosphorylation of the Ser63Ala mutant as <br />                                           well (Figure 5C). Residual phosphorylation of Shq1Np-101S63A is likely due to additional <br />                                           non-specific activity of CK1. Kinases that were not predicted by KinasePhos, Protein Kinase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           A (PKA) and Casein Kinase II (CK2), were incubated with Shq1Np-101 overnight and were <br />                                           unable to induce significant phosphorylation (Figure 5D). These results suggest that Shq1p is <br />                                           a specific target for CK1 directed phosphorylation and that these post-translation modifications <br />                                           may regulate its activity or protein-protein interactions. <br />  <br />                               Shq1p has general chaperone activity <br />                                           Surprised by the lack of interaction between Shq1Np and Hsp90, we performed a classical <br />                                           assay to test general chaperone activity in an attempt to assign a molecular function to Shq1p. <br />                                           The assay involves monitoring thermal aggregation of citrate synthase by light scattering43. <br />                                           The addition of a general molecular chaperone, such as p23, has been shown to prevent <br />                                           aggregation of citrate synthase and thus to reduce light scattering44, 45. <br />  <br />                                           Thermal aggregation of citrate synthase was observed during incubation at 45&#194;&#176;C, as expected <br />                                           (Figure 6). A concentration-dependent decrease in light scattering was observed and full length <br />                                           Shq1p was very efficient at suppressing citrate synthase aggregation at 1:1 molar ratio (Figure <br />                                           6A). Citrate synthase aggregation was also strongly inhibited by the presence of Shq1Cp (over <br />                                           50% of light scattering reduction at a ratio of 1:1), whereas Shq1Np had no effect (Figure 6B). <br />                                           A mix of Shq1Np and Shq1Cp had the same activity as Shq1Cp alone, showing that the N- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           terminal domain of Shq1p increases the chaperone activity only when it is present in the same <br />                                           polypeptide chain. In a negative control experiment, an equimolar ratio of BSA did not affect <br />                                           the thermal aggregation behaviour of citrate synthase (Figure 6A). In further control <br />                                           experiments, we verified that Shq1FL, Shq1Np, and Shq1Cp were not susceptible to <br />                                           aggregation under these conditions and that the addition of ATP did not have a significant <br />                                           impact on activity (data not shown). Thus, Shq1p has a stand-alone general chaperone activity <br />                                           provided by its C-terminal domain but reinforced by its N-terminal CS domain. <br />  <br />                               Discussion <br />                                           Stable accumulation of mature H/ACA snoRNAs in eukaryotes requires co-transcriptional <br />                                           association with protein factors that protect the snoRNA from degradation and promote the <br />                                           step-wise assembly of the mature RNP15, 17, 18. This assembly process is assisted by at least <br />                                           two proteins that appear to be specific for this class of snoRNPs, Naf1p and Shq1p22. A direct <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                               Page 7 <br />  <br />  <br />                                           role for Naf1p recruitment to Cbf5p during snoRNA transcription and exchange with Gar1p <br />                                           at later stages of the maturation process was demonstrated15, 17, 18. Here we show that Shq1p <br />                                           is composed of two structurally independent domains that contain phosphorylation sites, harbor <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           protein chaperone activity and bind to Cbf5p directly. Consistent with this result, while <br />                                           preparing this manuscript, mammalian SHQ1 was reported to interact with NAP57 specifically <br />                                           through its C-terminal domain and prior to the association of NAF1 and other H/ACA snoRNP <br />                                           proteins49. Although these new findings contradict the initial yeast two-hybrid and genetic <br />                                           depletion experiments that suggested a direct association with Naf1p, they are in agreement <br />                                           with our biochemical analysis. We confirm and extend recent results showing that Shq1p does <br />                                           not interact directly with Hsp82p24, 26, the yeast Hsp90 homologue, although the N-terminus <br />                                           of Shq1p has strong sequence and structural homology to the CS domains of Hsp90 co- <br />                                           chaperones. Our solution structure of Shq1N is consistent with the x-ray crystal structure24 <br />                                           and other CS domain proteins; we have further expanded this analysis by including deuterium <br />                                           exchange experiments, relaxation measurements, and radioactive phosphorylation assays to <br />                                           propose a functional role for Shq1p. <br />  <br />                                           Genetic depletion and co-immuno precipitation from mammalian cell extracts46, as well as cell <br />                                           viability studies and snoRNA microarrays in yeast47, have shown that the chaperone Hsp90 <br />                                           and the R2TP complex are generally important for the biogenesis of RNPs containing L7Ae <br />                                           homologs, These RNPs include the H/ACA snoRNPs and a second related class responsible <br />                                           for 2&#226;&#8364;&#178;O-methylation known as the C/D snoRNPs. A direct association between Cbf5p and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Hsp90 has also been observed in vivo using TAP tag pull-downs48, 49, supporting a role for <br />                                           the chaperone activity of Hsp90 in Box H/ACA biogenesis. Consistent with a role for Shq1p <br />                                           in Hsp82p recruitment, the integrity of the Sba1p/p23-Hsp90-like interface of Shq1p was <br />                                           recently shown to be important in vitro and in vivo for protein stability and snoRNA <br />                                           processing24; yet, we and others24 have independently failed to observe any direct interaction <br />                                           between this domain of Shq1p and Hsp82p in vitro. Reasoning that the Hsp82p-Shq1p <br />                                           interaction may be auto-inhibited in the full-length proteins, we also tested multiple domain <br />                                           constructs of each protein, but still failed to observe any interaction even at the concentrations <br />                                           required for NMR. Shq1Np is structurally distinct from Sgt1 and Sba1p/p23 because of the <br />                                           presence of a C-terminal &#206;&#177;-helix tightly bound to strands &#206;&#178;1 and &#206;&#178;5, the shorter &#206;&#178;4&#226;&#8364;&#8220;&#206;&#178;5 loop <br />                                           that forms a small surface pocket as well as the proposed Ser63 phosphorylation pocket. <br />                                           Interestingly, these two regions contained residues whose chemical shifts are altered when in <br />                                           the full length protein. Thus, it remains formally possible that Shq1p interacts with Hsp82p <br />                                           only when it is in a defined conformation or when bound to another protein cofactor yet to be <br />                                           discovered. <br />  <br />                                           Although the presence of ATP did not reverse the negative results of our in vitro binding studies, <br />                                           as was hoped, it nonetheless brought attention to two deep surface pockets containing potential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           phosphorylation substrates. The deep pocket between strands &#206;&#178;3 and &#206;&#178;6 contains a canonical <br />                                           Casein kinase 1 (CK1) recognition motif suggesting that Ser63 is a phosphorylation target39, <br />                                           40. This loop is also the most solvent-exposed region within the structure, suggesting that it <br />                                           may be easily accessible for kinase modification. Indeed we observed that CK1 readily <br />                                           phosphorylates the N- as well as C-terminal domain of Shq1p. The region of Shq1p around <br />                                           the Ser63 pocket is not well conserved, but serines elsewhere in the sequence might be targeted <br />                                           for phosphorylation in other species (Figure 1B) or this phosphorylation event may be specific <br />                                           for S. cerevisiae. Nevertheless, our observation of in vitro phosphorylation raises the question <br />                                           of whether Shq1p is phosphorylated in vivo. Preliminary studies show that in vivo Shq1p is <br />                                           phosphorylated (data not shown). However, since we have shown both the N- and C-terminal <br />                                           domains are phosphorylated by CK1 in vitro further work is underway to characterize which <br />                                           residues are phorphorylated in vivo and the impact of their phosphorylation on H/ACA snoRNP <br />                                           assembly in vivo. <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 8 <br />  <br />  <br />                                           Surprisingly, we found that Shq1p has stand-alone chaperone activity contained mainly by the <br />                                           Shq1p C-terminal domain but enhanced by the N-terminal domain. Enhancement of the <br />                                           chaperone activity requires both domains to be present in the same polypeptide. Similarly, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Shq1p binds to Cbf5p primarily through its C-terminal domain, but the interaction is most <br />                                           clearly seen with the full length protein. Thus, Shq1p appears to identify potential client <br />                                           proteins and act as a chaperone on them primarily using the C-terminal domain, yet the two <br />                                           domains act in synergy to provide optimal activity. These properties were also observed with <br />                                           p23, where its flexible 50 amino acid C-terminal tail is necessary for identifying Hsp90 client- <br />                                           proteins and for inherent p23 chaperone activity45. Thus, Shq1p may act as a snoRNP-specific <br />                                           chaperone independently of Hsp90, or it may also assist the chaperone activity of Hsp90 by <br />                                           identifying Cbf5p as a client protein. The identification of Cbf5p as a potential Shq1p target <br />                                           is significant because all proteins and the RNA components of H/ACA particles contact Cbf5p, <br />                                           placing Shq1p at the heart of the complex biogenesis of this class of snoRNPs. <br />  <br />                               Conclusions <br />                                           Our studies demonstrate that Shq1p is capable of Cbf5p recognition and that it has chaperone <br />                                           activity independent of its interaction with the yeast Hsp90 chaperone, suggesting that it may <br />                                           participate in chaperone-assisted maintenance or assembly of the H/ACA snoRNP. Since <br />                                           Shq1p does not interact efficiently with mature H/ACA particles, the interaction with Cbf5p <br />                                           occurs transiently during RNP assembly. The potential phosphorylation sites in both the N- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           and C-terminal domains suggest that the activity of Shq1p may be regulated by <br />                                           phosphorylation/dephosporylation to promote a specific chaperone activity, protein <br />                                           interaction, or subcellular localization. For example, Shq1p might be involved in the initial <br />                                           folding of Cbf5p or in the remodeling of H/ACA snoRNP that is required for maturation or <br />                                           subnuclear trafficking, and in stabilizing and destabilizing transient complexes during the <br />                                           maturation of the particle. In vivo experiments to dissect the function of Shq1p have been <br />                                           focused on the N-terminal domain and have been inconclusive. Our results clearly demonstrate <br />                                           that the C-terminal domain of Shq1p has identifiable biochemical activity. Its analysis will be <br />                                           necessary to understand when and how this protein functions as a chaperone or remodeling <br />                                           factor during H/ACA snoRNP biogenesis. <br />  <br />                               Materials and Methods <br />                               Plasmids <br />                                           Full length Shq1p (Shq1FL), the N-terminal domain consisting of residues Ile2-Asn134 <br />                                           (Shq1Np), and the C-terminal domain (Shq1C) from Ile123-Gln507 were amplified by PCR <br />                                           from single yeast colonies of the CRY1 strain and cloned into pET32 (Novagen) using ligation <br />                                           independent cloning (LIC) techniques. All pET32-Shq1p constructs contain an N-terminal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           thioredoxin fusion protein and a 6xHis tag followed by a TEV protease cleavage site. Cbf5&#206;&#8221;p <br />                                           was PCR amplified from yeast as previously described50 and cloned into pET28 with an N- <br />                                           terminal 6xHis tag. Hsp82p (yeast Hsp90) was also PCR amplified from yeast and cloned in <br />                                           pET30 by LIC techniques. Mutations were introduced using standard site directed mutagenesis. <br />  <br />                               Expression, purification, and proteolysis <br />                                           Expression and purification of Cbf5&#206;&#8221;p and Hsp82p constructs was executed following <br />                                           previously reported protocols31, 50. Shq1p constructs were transformed into the E. coli Rosetta <br />                                           expression strain (Novagen) and grown at 37&#194;&#176;C, induced with 0.2mM IPTG at 0.6 OD600, and <br />                                           expressed for 10 hours at 30&#194;&#176;C. For NMR, isotopic labeling was achieved by growth in M9 <br />                                           minimal media containing [15N] NH4Cl and [13C] glucose. Cell cultures were harvested by <br />                                           centrifugation at 6,000rpm 4&#194;&#176;C for 10 minutes and lysed by sonication in buffer A (400mM <br />                                           KCl, 50mM potassium phosphate (pH 7.0), 4mM &#206;&#178;ME). The cleared lysate was applied to a <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 9 <br />  <br />  <br />                                           HisTrap nickel affinity column (Amersham) equilibrated in Buffer A and eluted with a linear <br />                                           gradient to Buffer B (Buffer A with 400mM Imidazole). Purified protein was then dialyzed <br />                                           into buffer C (200mM KCl, 50mM potassium phosphate (pH 7.0), 4mM &#206;&#178;-mercaptoethanol) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           and digested with TEV protease containing a C-terminal 5xArg tag. In order to isolate the <br />                                           desired protein from the free 6xHis-tag and TEV protease, the digestion mixture was passed <br />                                           over another His-trap column equilibrated in buffer C before a final round of purification over <br />                                           a SD75 gel filtration column (Amersham) equilibrated in buffer C. For NMR studies, the <br />                                           proteins were dialyzed into 50mM KCl, 50mM potassium phosphate (pH 7.0), 4mM DTT. <br />  <br />                                           The individual domains of Shq1p were identified after digesting the full-length protein with <br />                                           proteinase K, chymotrypsin, or trypsin according to the manufacturer&#226;&#8364;&#8482;s directions. The <br />                                           sequence identity of the C-terminal domain was verified by N-terminal protein sequencing by <br />                                           the Protein and Nucleic Acid facility at the Medical College of Wisconsin. <br />  <br />                               NMR Spectroscopy and Resonance Assignment <br />                                           All NMR spectra collected for structure determination were performed at a protein <br />                                           concentration of 1.0mM and were recorded at 298K (unless otherwise stated). Triple resonance <br />                                           and HCCH-TOCSY51 were recorded on a Bruker Avance 500 MHz spectrometer. 3D 15N- <br />                                           edited NOESY and 3D 13C-edited NOESY52 were recorded at the Environmental Molecular <br />                                           Sciences Laboratory (EMSL) at the Pacific Northwest National Laboratory (PNNL) on a <br />                                           Varian INOVA 800 MHz and a Varian INOVA 600 MHz with cryoprobe, respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Backbone resonance assignments of H, N, C&#206;&#177;, C&#226;&#8364;&#178;, and side-chain C&#206;&#178;were done using Triple <br />                                           resonance HNCA53, 54, HNCO54, 55, CBCA(CO)NH56, 57, HNCACB, HBHACONH, and HN <br />                                           (CO)CA. The remaining side-chain resonance assignments were obtained from the analysis <br />                                           of 15N-edited NOESY, 13C-edited NOESY, and HCCH-TOCSY spectra. Aromatic ring <br />                                           protons and protected amides were identified from 2D 1H,1H-NOESY, 1H,1H-TOCSY <br />                                           and 15N-HSQC, respectively, collected from a sample exchanged into a 100% 2H2O NMR <br />                                           buffer. Spectral data were processed using NMRPipe58 and analyzed using Sparky59. <br />  <br />                               Structure determination <br />                                           Structure calculations were based on NOE cross-peaks identified from 3D 15N- and 13C-edited <br />                                           NOESYs and from a 2D 1H, 1H-NOESY recorded in 100% 2H2O. CYANA 2.160, 61 was <br />                                           implemented for the semi-automated assignment of NOE cross-peaks based upon the chemical <br />                                           shifts generated from backbone and side-chain resonance assignments. Structure calculations <br />                                           were performed with the CALC algorithm implementing torsion angle dynamics and <br />                                           undergoing 8 cycles of iterative structure calculations. After the first rounds of calculations, <br />                                           the resulting structures and NOESY assignments were reviewed to remove artifact peaks, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           correct input backbone and side-chain assignments, to incorporate additional NOEs to improve <br />                                           regions of poor structural quality and to correct misidentified NOEs to be consistent with <br />                                           generated structural models. In order to satisfy the criteria for convergence defined by <br />                                           Herrmann et al.62, additional constraints from torsion angle (&#207;&#8224;/&#207;&#8226;) values derived by <br />                                           TALOS63 and hydrogen bonds were included in later rounds of refinement. Identification of <br />                                           the secondary structure elements was supported by the Chemical Shift Index (CSI) <br />                                           analysis64. The final CYANA 2.1 refinement cycle generated 20 conformers with the lowest <br />                                           target function, violations, and RMSD values of all calculations (Table 1). The final structures <br />                                           were validated using PROCHECK-NMR65 (Table 1). Structural homologues were found <br />                                           through the DALI server66 and figures were generated with MOLMOL67. The structure and <br />                                           experimental data were deposited with the PDB and BMRB. <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                                              Page 10 <br />  <br />  <br />                               ATP phosphorylation <br />                                           Shq1p constructs were assayed for phosphorylation by incubating ~3ug of each protein with <br />                                           0.5ug human Casein Kinase 1&#206;&#180;or Casein Kinase 1&#206;&#181;(generously donated by Dr David <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Kimelman) in buffer D supplemented with 200uM ATP and 25nM 32P-[&#206;&#179;]ATP (3000Ci/mmol) <br />                                           at 30&#194;&#176;C for 1hr. For control kinase reactions, the same conditions were used for Protein Kinase <br />                                           A and Casein Kinase II. The reactions were stopped by addition of SDS loading dye, separated <br />                                           with 12.5% SDS-PAGE gel, dried and exposed overnight. The gels were imaged by <br />                                           autoradiography and analyzed using ImageQuant Molecular Dynamics software. <br />  <br />                               Citrate Synthase Aggregation Assay <br />                                           Porcine heart citrate synthase (Roche Applied Science) at 0.4 &#206;&#188;M final concentration was <br />                                           incubated with different amounts of Shq1FLp in 40 mM HEPES pH 7.5, 20 mM KOH, 50 mM <br />                                           KCl, 10 mM (NH4)2SO4, and 2 mM CH3COOK. Shq1Np and/or Shq1Cp or bovine serum <br />                                           albumin (BSA, New England Biolabs) were used instead of Shq1FLp in some assays. Samples <br />                                           (100 &#206;&#188;L) were transferred to the thermostated cell of a Varian Eclipse fluorescence <br />                                           spectrophotometer and held at 45&#194;&#176;C. Right angle light scattering was monitored for 15 min <br />                                           with the following settings: &#206;&#187;EX/EM = 500 nm, slitEX/EM = 2.5 nm. Control experiments were <br />                                           performed with each protein alone under the same conditions. <br />  <br />                               Cbf5p and Hsp90 interaction assay <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Refolded Cbf5&#206;&#8221;(3.2nmol) was incubated with 40&#206;&#188;L Ni-agarose beads (Qiagen) in buffer PD <br />                                           (200mM KCl, 20mM Tris-HCl (pH 8.0), 2mM &#206;&#178;ME, 0.2% Triton) at 4&#194;&#176;C. After 1 hour, <br />                                           Cbf5&#206;&#8221;p was pulled down, washed 3 times with 200&#206;&#188;L of buffer PD, and resuspended in PD <br />                                           buffer containing 2&#226;&#8364;&#8220;3 times molar excess of Shq1FL, Shq1Np, or Shq1Cp and incubated at 4&#194;&#176; <br />                                           C for an additional hour. The mixture was pulled down and washed with buffer PD as previously <br />                                           stated and eluted with 400mM imidazole. The eluate was analyzed by SDS-PAGE and <br />                                           Coomassie Brilliant Blue staining. The pull-down assay with Hsp82p, the yeast Hsp90 <br />                                           homologue, was conducted in the same manner as just described for the Cbf5p interaction. <br />                                           Before incubation with Ni-agarose beads, full length Hsp82p or the different Hsp90 constructs <br />                                           were activated by a 30 min pre-warming step at 30&#194;&#176;C both in the presence or absence of 10- <br />                                           fold molar excess of ATP or ATP analogs. Sba1p, the yeast p23 homologue, was used as a <br />                                           positive control. <br />  <br />                               Accession Numbers <br />                                           Coordinates and structure factors have been deposited in the Protein Data Bank with accession <br />                                           number 2K8Q <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />  <br />                                           Figure 1. Conservation of Shq1p in eukaryotes <br />                                           A) Cartoon representation of the Shq1p domain organization. B) Sequence alignment of the <br />                                           Shq1p N-terminus (Shq1Np) from yeast to humans showing the secondary structural elements <br />                                           as identified by NMR. Identical residues are indicated in blue; the darkest blue represents 100% <br />                                           conservation. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                         Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 2. Assessment of Shq1p domain structure by 15N-HSQC <br />                                           A) 15N-labeled HSQC of Shq1Np 50mM KCl, 50mM potassium phosphate (pH 7.0), and 4mM <br />                                           DTT. B) 15N-HSQC of Shq1FLp in 200mM KCl, 50mM potassium phosphate (pH 7.0), and <br />                                           4mM DTT overlayed with the 15N-HSQC of Shq1Np shown in part A. Both spectra were <br />                                           recorded at 298K on a Bruker Avance 500MHz spectrometer. C) Shq1p interacts with Cbf5p <br />                                           in vitro. Refolded Cbf5&#206;&#8221;p bound to Ni-agarose beads (lane 1) was treated with 3-fold molar <br />                                           excess of each Shq1p construct. SDS-PAGE analysis of each elution shows that full length <br />                                           Shq1p (lane 2) associates with Cbf5&#206;&#8221;p while Shq1Cp (lane 3) is only weakly bound and <br />                                           Shq1Np (lane 4) is not apparently binding to Cbf5&#206;&#8221;p under these conditions. The far left well <br />                                           shows the molecular weight marker and the asterisk marks a Cbf5&#206;&#8221;p degradation product. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                         Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 3. Solution structure of the Shq1p CS domain <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           A) Backbone trace of 20 structures calculated with CYANA-2.1 having the lowest target <br />                                           function and violations. The structures agree to within ~0.20A r.m.s.d. of the mean, when <br />                                           calculated over the structured regions. B) Cartoon representation showing each face of the <br />                                           Shq1Np CS domain &#206;&#178;-sandwich. C) Structural topology of Shq1Np. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                             Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 4. Comparison of Shq1Np with Hsp90 co-chaperones <br />                                           Shq1Np (bottom views) is compared with the Hsp90 binding interface of A) S. cereviseae <br />                                           Sba1p (2CG9) and B) A. thaliana Sgt1 (2JK1). Sba1p and Sgt1 use distinct modes of Hsp90 <br />                                           recognition and each are shown along their Hsp90 binding interfaces. In each panel, Shq1Np <br />                                           is shown in the orientation that corresponds with either the Hsp90 binding interface of Sba1p <br />                                           in A) or that of Sgt1 in B). In the left panel, each protein is shown as a ribbon representation. <br />                                           Shq1Np shares the same &#206;&#178;-sheet organization with both co-chaperones, but contains a C- <br />                                           terminal helix where Sgt1 lacks structure beyond the &#206;&#178;-sandwich and Sba1 instead has a &#206;&#178;- <br />                                           strand that interacts with the N-terminal region of the 4-stranded &#206;&#178;-sheet. In the middle panel, <br />                                           residues found at the Hsp90 interface of Sba1p and Sgt1 are shown in red and compared with <br />                                           highly conserved surface exposed residues of Shq1Np, which are colored according to the <br />                                           sequence alignment shown in Figure 1. In the right panel, the electrostatic surface potential <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                           Page 19 <br />  <br />  <br />                                           corresponding to the Hsp90 binding interface of Sba1p in A) and Sgt1 in B). The middle and <br />                                           right panel show that conserved residues on Shq1Np correspond with the interaction <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                                           observed for the Sba1p-Hsp90 interaction interface and that these two proteins are more similar <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           to each other, with regards to electrostatic surface potential, than Shq1Np and Sgt1. C) Shq1p <br />                                           does not interact directly with Hsp82p, the yeast Hsp90 homologue, in vitro. Hsp82p bound <br />                                           to Ni-agarose beads in the presence of 10-fold molar excess of ATP (lane 1) was incubated <br />                                           with a 3-fold molar excess of each Shq1p construct. SDS-PAGE analysis of each elution shows <br />                                           that full length Shq1p (lane 2), Shq1Cp (lane 3) and Shq1Np (lane 4) do not bind to Hsp82p <br />                                           under these conditions. The middle well contains molecular weight marker. Lanes 5 to 8 show <br />                                           the migration profiles of each protein involved in the pull-down assay. The same results were <br />                                           obtained with Hsp82p without ATP or with ATP analogs (data not shown). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                            Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 5. Shq1p is a phosphorylated protein <br />                                           A) Shq1Np has two serine-containing pockets; one located in a region of negative electrostatic <br />                                           potential between strands &#206;&#178;3 and &#206;&#178;6 (top) and a second the between the &#206;&#178;2&#226;&#8364;&#8220;&#206;&#178;3 and &#206;&#178;4&#226;&#8364;&#8220;&#206;&#178;5 loops <br />                                           (bottom). The upper boxed region shows a closer look at residues within a CK1 consensus <br />                                           sequence specifically S63 (in green) that is within 3&#195;&#8230; of D59 (in yellow). B) Incubation of full <br />                                           length Shq1p and its N- and C-terminal domains with CK1&#206;&#181;in the presence of 32P-[&#206;&#179;]ATP <br />                                           shows that &#206;&#179;-phosphate is transferred to all Shq1p constructs. The R3 region of APC is a known <br />                                           target of CK1 and was used as a positive control while Rna15p, a protein associated with 3&#226;&#8364;&#178; <br />                                           end processing of RNAP II transcripts, was predicted to not contain a CK1 phosphorylation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           and showed no CK1 phosphorylation. C) Phosphorylation of Shq1Np, the G101 truncation <br />                                           mutation of Shq1Np, and the G101 truncation of the Shq1Np S63A mutation with CK1&#206;&#180; at <br />                                           30&#194;&#176;C for 30 minutes. D) Phosphorylation of Shq1Np-G101 with kinases PKA and CK2 at 30&#194;&#176; <br />                                           C for 16hrs. <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                             Godin et al.                                                                                         Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 6. Shq1p has chaperone activity <br />                                           (A) Aggregation of citrate synthase (CS) was followed by light scattering at 45&#194;&#176;C without <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           additives and in the presence of an equimolar ratio (based on monomers) of BSA, or with <br />                                           increasing molar ratios of full length Shq1. (B) Aggregation of citrate synthase at 45&#194;&#176;C was <br />                                           measured by light scattering in the presence of equimolar ratios (based on monomers) of full <br />                                           length Shq1p, Shq1Np and/or Shq1Cp, or in the absence of additional proteins. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />                                 Godin et al.                                                                                                     Page 22 <br />  <br />                                                                                                 Table 1 <br />                                                                      Experimental restraints and structural statistics <br />  <br />                                                                                                                                   Shq1N <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Structural Constraints <br />  <br />                            Distance constraintsa <br />                            &#226;&#8364;&#402;Total meaningful NOE restraints                                                                        2469 <br />                            &#226;&#8364;&#402;&#226;&#8364;&#402;Short range (|I&#226;&#8364;&#8220;j| = 1)                                                                               1326 <br />                            &#226;&#8364;&#402;&#226;&#8364;&#402;Medium-range (|I&#226;&#8364;&#8220;j| &lt; 4)                                                                              304 <br />                            &#226;&#8364;&#402;&#226;&#8364;&#402;Long-range (|I&#226;&#8364;&#8220;j| &gt; 5)                                                                                839 <br />                            Hydrogen bond restraints                                                                                 96 <br />                            Saltbridge restraints                                                                                    2 <br />  <br />                            Dihedral angle restraints (&#207;&#8226;,&#207;&#710;)b                                                                         54 <br />  <br />  <br />                            Total Number of Constraints                                                                             2621 <br />                            &#226;&#8364;&#402;&#226;&#8364;&#402;Constraints per residue                                                                              ~19.8 <br />  <br />                            Residual Constraint Violationsc <br />                            Cyana Target Function (&#195;&#8230;2)                                                                         1.57 +/&#226;&#710;&#8217; 0.0057 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#226;&#8364;&#402;Distance violations (&gt;0.2 &#195;&#8230;)                                                                         0 (0.18) <br />                            &#226;&#8364;&#402;Torsion angles (&gt;5.0 &#195;&#8230;)                                                                              0 (4.31) <br />  <br />                            Average RMSD to mean (&#195;&#8230;)d <br />                            &#226;&#8364;&#402;Heavy atom                                                                                         0.71 +/&#226;&#710;&#8217; 0.04 <br />                            &#226;&#8364;&#402;Backbone                                                                                           0.25 +/&#226;&#710;&#8217; 0.05 <br />  <br />                            Ramachandran Statistics (%)e <br />                            Favored                                                                                                 78.4 <br />                            Additionally allowed                                                                                    20.5 <br />                            Generously allowed                                                                                       0.2 <br />                            Disallowed                                                                                               0.9 <br />  <br />  <br />                            a <br />                             Final 20/100 structures selected based on lowest CYANA target function <br />                            b <br />                             Dihedral angle restraints derived from TALOS <br />                            c <br />                             Maximum violation shown in parenthesis <br />                            d <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Average RMSD to mean calculated over regions of consistent structure. <br />                            e <br />                             Ramachandran analysis was performed by PROCHECK NMR. <br />  <br />  <br />  <br />  <br />                                                               J Mol Biol. Author manuscript; available in PMC 2010 July 10. <br />   <br /> </body></html>