<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Systems Biology                                                                                                                      BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> How identify essential genes molecular networks? <br /> Gabriel del Rio*1, Dirk Kosch&#195;&#188;tzki2,3 Gerardo Coello1 <br />  <br /> Address: 1Department Biochemistry Structural Biology, Universidad Nacional Aut&#195;&#179;noma M&#195;&#169;xico, Instituto Fisiolog&#195;&#173;a Celular, Circuito <br /> Exterior s/n ciudad Universitaria, 04510 M&#195;&#169;xico D.F., M&#195;&#169;xico, 2Department Molecular Genetics, Leibniz Institute Plant Genetics Crop <br /> Plant Research (IPK), Corrensstr. 3, 06466 Gatersleben, Germany 3Department Computer Electrical Engineering, Furtwangen <br /> University Applied Sciences, Robert-Gerwig-Platz 1, 78120 Furtwangen, Germany <br /> Email: Gabriel del Rio* - gdelrio@ifc.unam.mx; Dirk Kosch&#195;&#188;tzki - dirk.Koschuetzki@hs-furtwangen.de; Gerardo Coello - gcoello@ifc.unam.mx <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 13 October 2009                                                     Received: 26 February 2009 <br />                                                                                Accepted: 13 October 2009 <br /> BMC Systems Biology 2009, 3:102   doi:10.1186/1752-0509-3-102 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1752-0509/3/102 <br /> &#194;&#169; 2009 del Rio et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: The prediction essential genes molecular networks way test the <br />                  understanding essentiality context known network. However, the <br />                  current knowledge molecular network structures incomplete yet, consequently the <br />                  strategies aimed predict essential genes prone uncertain predictions. We propose that <br />                  simultaneously evaluating different network structures different algorithms representing gene <br />                  essentiality (centrality measures) identify essential genes networks reliable fashion. <br />                  Results: By simultaneously analyzing 16 different centrality measures 18 different <br />                  reconstructed metabolic networks Saccharomyces cerevisiae, single centrality <br />                  measure identifies essential genes networks statistically significant way; however, <br />                  combination 2 centrality measures achieves reliable prediction all <br />                  essential genes. No improvement achieved prediction essential genes 3 or <br />                  4 centrality measures combined. <br />                  Conclusion: The method reported describes reliable procedure predict essential genes <br />                  molecular networks. Our results essential genes predicted by <br />                  combining centrality measures, revealing complex nature function essential genes. <br />  <br />  <br />  <br />  <br /> Background                                                                     feature biological system. This ability rests the <br /> Modelling molecular mechanisms present living                           assumed relationship biological observed fea- <br /> organisms represents active area research biology.                    ture (commonly referred <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>) molecular <br /> Such modelling appealing scientists applied                      structure cell. We refer relationship, as <br /> and basic research areas, represent way                      structure-function relationship paradigm. This paradigm <br /> test basic understanding cells organized.                       stated follows: <br /> The models potential accelerate dis- <br /> covery new drugs (e.g., antibiotics) guide engi-                  S1. A given cellular phenotype related molecular <br /> neering new organisms better producing                          structure cell. <br /> desired compounds (e.g., vitamins). Different approaches <br /> are available build molecular networks, case                     Additionally, molecular structure cell repre- <br /> these models need able reproduce observed                          sented molecular network; is, set molecules <br />  <br />  <br />                                                                                                                                        Page 1 12 <br />                                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                            http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> and molecule-molecule interactions (e.g., genetic rela-        used known genetic relationships yeast infer <br /> tionships, protein-protein interactions).                      role genes [4]; however, reported genetic <br />                                                                relationships used particular process per- <br /> According S1, molecules interactions      formed yeast (e.g., sexual reproduction use dif- <br /> be known order model phenotypes. The advances         ferent set genes and/or genetic relationships cell <br /> DNA manipulation techniques allowed scientists         division). Hitherto, best knowledge, the <br /> systematically identify genes different organ-    available experimental data theoretical approaches <br /> isms consequently, genes set mol-      completed mapping genetic rela- <br /> ecules model phenotypes. It important note       tionships cellular phenotype. <br /> although genes represent different <br /> types molecules biological systems (e.g., proteins,      On hand, diverse nature essential genes <br /> ions, carbohydrates, lipids), genetic relationships     [5] explain limited success previous attempts <br /> other hand represent different molecular interactions          predict [6-8]. Hence, current scenario <br /> (e.g., protein-protein interactions, protein-DNA interac-      uncertainty completeness the <br /> tions). Mapping genetic relationships advantage        network likely algorithms fail identify <br /> that observed relatively simple experiments       true nature essentiality. In order evaluate sig- <br /> such gene deletion experiments gene expression           nificance algorithm used predict essential genes, <br /> studies. For instance, using culture media           necessary vary network structure the <br /> Arginine grow different gene deletion mutant strains        algorithm predict essentiality. In way, compar- <br /> may used identify genes related biosynthesis      ing significance prediction algorithm differ- <br /> of Arginine; alternatively, extracting comparing       ent network structures possible most <br /> mRNA strain cultured Arginine          significant result tested networks. <br /> may used identify up-regulated gene transcripts in <br /> the presence Arginine, indicating potential func-     Here, combined analysis two <br /> tional relationship. Thus, having access set      aspects reliably predict essential genes net- <br /> genes relationships (otherwise, having access     works follows (see Figure 1). First, according S1, a <br /> a genetic network) ultimately render model            phenotype represented molecular network <br /> describe phenotype.                                          n genes; formally, network G defined sets, <br />                                                                set genes V set relations R, that: <br /> The goal present work computa- <br /> tional procedure achieve reliable prediction         V(G) = {v1, v2, ... vn} n total number of <br /> essential genes genetic networks. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> achieve         genes network V(G) empty. <br /> that, need access set molecular networks and <br /> algorithms reproducing features essentiality. The      R(G) &#239;&#402;? V(G) &#195;&#8212; V(G) R(G) empty. <br /> need different sets networks justified cur- <br /> rent incompleteness biological networks.                    Now define set critical genes associated phe- <br />                                                                notype c; thus: <br /> In previous studies effective reconstruction tran- <br /> scriptional regulatory networks (TRN) achieved                                  c &#226;&#352;&#8218; V(G) <br /> [1,2]. Wagner instance [1], described algorithm <br /> where parsimonious TRN built based exper-        Consequently, postulate that: <br /> imental data source n2 steps, n number <br /> of genes. A parsimonious network uses lowest number        P1. There mathematical operation G to <br /> of genetic relationships connect pair genes [1].   rank V(G) way obtain c. <br /> However, algorithm does attempt establish the <br /> completeness network, instead depends         It expected G incomplete, ability iden- <br /> available data. Alternatively, group Palsson [2]        tify c limited well. In condition, best <br /> described procedure TRN reconstruction well. To       approach predict c identify G allows for <br /> validate completeness TRN, predicted cell           complete prediction set c. This constitutes the <br /> growth rates metabolic enzymes included        limiting condition P1. <br /> TRN compared observed ones [2]. Although this <br /> is powerful tool, argue based      In present work, develop method based on <br /> chemicals genetic networks based genes;        postulate P1, is, method achieve <br /> we argued above, information genes read-       reliable prediction essential genes genetic net- <br /> ily available complete [3]. Other reports        works. We propose mathematical operation <br />  <br />  <br />                                                                                                                    Page 2 12 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br />  <br /> Figure 1prediction essential genes <br /> Reliable <br /> Reliable prediction essential genes. Prediction essential genes depends quality network the <br /> efficacy prediction algorithm reproduce features essentiality. To account factors, method <br /> uses set n networks modelling biological process, m different centrality measures applied to identify <br /> both network(s) largest predictable set essential genes centrality(ies) effective identify the <br /> essential genes. To determine reliability predictions necessary access set known essential <br /> genes. <br />  <br />  <br />  <br />  <br /> described P1 calculation network centralities. A        genes KEGG database EC classifica- <br /> centrality defined, measure computes rela-       tion. In KEGG2, genes EC number dif- <br /> tive importance network elements (genes) net-         ferent substrate forced accept substrates found <br /> work. Therefore, given centrality measure satisfies        given EC number. All programs used rebuild <br /> postulate P1, include basic features           analyze genetic metabolic networks coded in <br /> determine essentiality gene network.          Java programming language. <br />  <br /> Methods                                                           Directionality networks <br /> Databases programs                                   Enzymatic reactions involved metabolism be <br /> The genetic metabolic networks S. cerevisiae described         reversible irreversible. We refer reac- <br /> in work based KEGG PATHWAY database              tion type. However, genetic relationships reversi- <br /> [9] hand-curated model iND750 [10]. Genetic               ble irreversible consequence experimental <br /> metabolic networks built connecting genes             way detected (gene deletions gene transcrip- <br /> annotated enzymes metabolites. That is,            tion profiles). To account different ways rep- <br /> genes coding metabolic enzymes linked          resent relationships genes, built genetic <br /> share metabolite product reac-        metabolic networks assuming annotated reversibility <br /> tions substrate other. All metabolites       reaction (e.g., KEGG-derived networks the <br /> considered cases indicated (see below).        &quot;type&quot; word, e.g., KEGGtype network) assuming the <br /> Additionally, depurated version KEGG built           genetic relationships reversible (KEGG-derived net- <br /> named KEGG2. This version obtained eliminating             works &quot;type&quot; word, e.g., KEGG KEGGpath) <br />  <br />  <br />                                                                                                                        Page 3 12 <br />                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                             http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> (see Table 1). In summary, KEGG-derived networks                         lism present anaerobic conditions [12]. In <br /> named &quot;type&quot; keyword include reversible                          order account insulation metabolic path- <br /> genetic relationships (i.e., undirected networks).                           ways, built group genetic metabolic networks by <br />                                                                              linking genes pathway; additionally, we <br /> Pathway insulation compartamentalization                                 different pathway genes present these <br /> For decades metabolism organized path-                         different pathways (e.g., KEGGpath network). Alterna- <br /> ways recently observed genes path-                   tively, build metabolic networks linking genes <br /> ways co-regulated [11]. It known                 disregarding pathway (e.g., KEGG network) (see Table <br /> the genes pathways expressed. For                     1). In summary, KEGG-derived networks labelled with <br /> instance, enzymes involved aerobic primary metabo-                        &quot;path&quot; keyword include genetic relationships <br />  <br /> Table 1: Genetic metabolic networks used study <br />  <br />  Network                                      Over-linked metabolites                                   Vertices:Edges          Overlap (%) <br />  <br />  KEGG                              H2O, ATP, ADP, NAD+, NADH, NADP+, NADPH, Oxygen                              636:10038                33.85 <br />  <br />  KEGGtype                                                       Ibid                                             629:6590                26.81 <br />  <br />  KEGGpath                                                       Ibid                                             634:7752                31.57 <br />  <br />  KEGGtypepath                                                   ibid                                             621:5223                25.11 <br />  <br />  KEGG2                                                          Ibid                                            609:10518                34.24 <br />  <br />  KEGG2type                                                      Ibid                                             602:7099                27.44 <br />  <br />  KEGG2path                                                      Ibid                                             608:8130                31.89 <br />  <br />  KEGG2typepath                                                  Ibid                                             595:5691                25.65 <br />  <br />  iND750_0                                                    H2O, H+                                             990:8427                19.19 <br />  <br />  iND750_1                                                  H2O, H+, Pi                                           976:6995                18.27 <br />  <br />  iND750_2                                               H2O, H+, Pi, ATP                                         976:6278                17.85 <br />  <br />  iND750_3                                            H2O, H+, Pi, ATP, Glu-L                                     974:5742                17.60 <br />  <br />  iND750_4                                         H2O, H+, Pi, ATP, Glu-L, ADP                                   969:5186                17.18 <br />  <br />  iND750_0nh                                                  H2O, H+                                             634:4761                19.19 <br />  <br />  iND750_1nh                                                H2O, H+, Pi                                           619:3963                18.27 <br />  <br />  iND750_2nh                                             H2O, H+, Pi, ATP                                         618:3387                17.85 <br />  <br />  iND750_3nh                                          H2O, H+, Pi, ATP, Glu-L                                     617:3122                17.60 <br />  <br />  iND750_4nh                                       H2O, H+, Pi, ATP, Glu-L, ADP                                   613:2720                17.18 <br />  <br />  The names KEGG iND750-derived networks indicated &quot;Network name&quot; column. Note networks end <br />  &quot;nh&quot;, indicating hypothetical reactions included networks. The column labeled &quot;Over-linked metabolites&quot; indicates <br />  metabolites removed reconstructing network: H2O: water; H+: proton; Pi: inorganic phosphorus; ATP: adenosine <br />  triphosphate; Glu-L: L-glutamate ADP: adenosine diphosphate, NADP+/NADPH: Nicotinamide adenine dinucleotide phosphate, NAD/NADH: <br />  Nicotinamide adenine dinucleotide. The number genes (values bold) gene-to-gene relationships (values italics) graph are <br />  indicated separated colon Vertices:Edges column. The column labeled &quot;Overlap (%)&quot; indicates percentage known gene-to-gene <br />  relationships (network links) reconstructed network reported probabilistic functional network yeast genes [4] included <br />  metabolic genes. According procedure build networks, KEGG, KEGGpath, KEGG2 KEGG2path networks undirected. <br />  <br />  <br />                                                                                                                                       Page 4 12 <br />                                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> restricted annotated pathways, share      clustering coefficient, degree (in out) sphere <br /> metabolite.                                                     degree (the number reachable vertices distance <br />                                                                 2) [16-18]. An inverse network derived a <br /> Finally, known, inside eukaryotic cells   directed network switching nodes edge. <br /> are barriers proteins small molecules sorted     Note centrality value equal zero, the <br /> compartments [10]. Such insulation taken               inverse value set zero study. Eccentricity, <br /> account iND750 derived networks. Specifically,        average distance, closeness, shortest-path betweenness, <br /> did link genes different cellular compartments.          Katz index, PageRank, radiality integration global <br /> Additionally, groups networks derived          centrality measures local sense <br /> the hand-curated iND750 network. In group,        measure depends immediate connec- <br /> the reactions contained original model consid-      tions gene. Thus, gene assigned centrality <br /> ered (e.g., iND750_0, Table 1), second,        value list genes sorted descendent fash- <br /> hypothetic reactions eliminated (e.g., iND750_0nh).        ion. From ordered list, took possible fraction <br /> For groups, different sets over-linked         list determine correspondence the <br /> metabolites eliminated network (see              known lethal genes coding metabolic enzymes. <br /> Table 1). <br />                                                                 Additionally performing analysis based single <br /> Elimination genetic relationships emerging highly       centrality, combined individual centrality in <br /> connected metabolites                                           groups 2, 3 4 centralities generated combined <br /> Including highly connected metabolites (e.g., ATP            centrality score gene network, CS(v), <br /> water) reconstructed metabolite networks induce          according following formula: <br /> small average path <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 3 [13]. By eliminating highly <br />  <br />                                                                           &#226;&#710;&#8216; <br /> connected metabolites, distinct chemical metabolic                       m <br />                                                                 CS( v ) = (          ((MAX _ C &#226;&#710;&#8217; C ( v )) /(MAX _ C &#226;&#710;&#8217; MIN _ C ) 2 )1 / 2 <br /> networks obtained [14]. All recon-                         =1 <br />  <br /> structed networks treated bipartite networks, com-      Where MAX_Ci MIN_Ci define maximum and <br /> posed enzymes metabolites, effect         minimum score obtained ith-centrality given net- <br /> prediction essential genes evaluated.          work, respectively, Ci(v) refers centrality given <br /> Here, evaluated effect eliminating highly con-      gene m summation refers centrality com- <br /> nected metabolites prediction essential           binations evaluated; instance, m = 2 groups 2 <br /> genes using genetic networks. All KEGG-derived net-         centralities, m = 3 groups 3 centralities m = 4 <br /> works eliminated following over-linked metabolites:         groups 4 centralities. CS(v) estimates far from <br /> water, ATP, ADP, NAD+, NADH, NADP+, NADPH                   largest observed centrality measures centralities <br /> Oxygen.                                                         genes analysed. Thus, lower combined score <br />                                                                 is, higher individual centrality measures are. <br /> Network properties <br /> Two sets determined genetic networks,           Assessing reliability centrality predictions <br /> the genes network (vertices) connections           The Yeast Genome Consortium [19] provides list of <br /> between genes (edges). From sets, eccen-        genes deletion yeast genome prevent <br /> tricity radius determined; eccentricity         yeast growth YPD media. Those lethal phenotypes <br /> defined largest shortest path given gene,    arising deletion genes involved metabolism are <br /> while radius defined smallest eccentricity        considered critical metabolism function, inacti- <br /> value genes. Thus, path connecting   vation primary metabolism lead cell death. This <br /> genes, radius network equal infinitum        included total 246 genes: iND750 networks this <br /> the network connected. This measure used        work 107 115 essential metabolic genes the <br /> to determine presence giant strong component         KEGG KEGG2 networks number varies 127 <br /> in models [15].                                             131. The iND750 networks KEGG networks <br />                                                                 share 93 essential genes. <br /> Centrality measures <br /> We used 16 different centrality measures (see Table 2       The genes network ranked according each <br /> a mathematical description each): eccentricity, 1/eccen-     16 centrality values (the 4 non directed networks <br /> tricity, closeness, average distance, shortest-path between-    used 13 measurements). Therefore, created 276 gene <br /> ness, Katz index (for network analysis        lists compare list known essential genes <br /> corresponding inverse network), PageRank (for net-          single centrality values used. When pairs of <br /> work analysis corresponding inverse net-          centrality used score gene, 2,736 compari- <br /> work), radiality, integration, clustering coefficient, 1/ <br />  <br />  <br />                                                                                                                                 Page 5 12 <br />                                                                                                         (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                              http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> Table 2: Centrality measures used study <br />  <br />  Centrality                                          Formula                                                    Description <br />  <br />  In degree                                          Cu = kin(u)                                     Number connections node u <br />  <br />  Out degree                                        Cu = kout(u)                                   Number connections node u <br />  <br />  Sphere degree                Cu = kout(u) + &#239;&#402;&#165;w&#239;&#402;&#8225;u kout(w); w neighbor u.         Number nodes 1 2 connections node u <br />  <br />  <br />                                                               |e| <br />  Clustering coefficient                    Cu = ktot(u)(ktot(u) &#226;&#710;&#8217;1)                    The fraction connections neighbors node u <br />  <br />  <br />  <br />                                                    ktot( u)( ktot( u) &#226;&#710;&#8217;1) <br />  1/Clustering coefficient                  Cu =             |e| <br />  <br />  <br />  Eccentricity                              Cu = max{dist(u, w): w &#239;&#402;&#381; V}                  The distance node u distant node in <br />                                                                                                                net. <br />  <br />  1/Eccentricity                           Cu = 1/max{dist(u, w): w &#239;&#402;&#381; V} <br />  <br />  <br />                                                     &#226;&#710;&#8216; w&#226;&#710;&#710;V dist( u , w ) <br />  Average distance                            Cu =        n &#226;&#710;&#8217;1 <br />                                                                                          Average distance node u rest nodes net <br />  <br />  <br />  <br />  Closeness                                   Cu =        n &#226;&#710;&#8217;1                                            Inverse average distance <br />                                                     &#226;&#710;&#8216; w&#226;&#710;&#710;V dist( u , w ) <br />  <br />  <br />                                                          &#226;&#710;&#382; <br />  Katz                                      c katz = &#226;&#710;&#8216; =1 &#239;?&#161; ( A T ) a1                A node larger ckatz value paths reach it. <br />  <br />  <br />  <br />                                                           &#226;&#710;&#382; <br />  KatzR                                      c katz = &#226;&#710;&#8216; =1 &#239;?&#161; ( A) a1                  A node larger ckatz value paths leave the <br />                                                                                                                     node. <br />  <br />  <br />  PageRank                                    cPR = d P cPR + ((1 - d) 1)               The centrality node depends incoming connections <br />                                                                                             relative connectivity connections <br />  <br />  PageRankR                                  cPR = d PT cPR + ((1 - d) 1)                     The centrality node depends outcoming <br />                                                                                              connections relative connectivity these <br />                                                                                                                  connections <br />  <br />  <br />                                               &#226;&#710;&#8216; t&#226;&#710;&#710;V (diamG +1&#226;&#710;&#8217; dist( t , u)) <br />  Integration                           Cu =              n &#226;&#710;&#8217;1 <br />                                                                                            The easiness reaching node u node <br />  <br />  <br />  <br />                                               &#226;&#710;&#8216; t&#226;&#710;&#710;V (diamG +1&#226;&#710;&#8217; dist( u,t )) <br />  Radiality                             Cu =              n &#226;&#710;&#8217;1 <br />                                                                                               The easiness reaching node node u <br />  <br />  <br />  <br />                                                                &#239;?&#179; G ( s , t|u) <br />  SP-betweenness                             Cu =       &#226;&#710;&#8216; <br />                                                    s &#226;&#8240;&#160; u &#226;&#8240;&#160; t&#226;&#710;&#710;V <br />                                                                 &#239;?&#179; G (s , t ) <br />                                                                                           The fraction shortest paths inside network, which <br />                                                                                                                utilize node u <br />  <br />  <br />  In table, kin(u), kout(u) ktot(u) refer incoming, outgoing total number edges node u. diamG refers diameter the <br />  graph dist(u, v) stands distance nodes u v. In clustering coefficient, |e| stands observed paths the <br />  neighbours node. In Katz A adjacency matrix &#239;?&#161; damping factor. In PageRank d damping factor P transition matrix. In the <br />  formula shortest path (SP) betweenness &#239;?&#179;G denotes number shortest path s t. For detailed description these <br />  centralities, read [17]. <br />  <br />  <br />  <br />                                                                                                                                         Page 6 12 <br />                                                                                                                   (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                               http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> sons performed, 13,200 groups 3 centrali-        Therefore, using different methods analyze alterna- <br /> ties, 48,300 groups 4.                                tive representations network, identify <br />                                                                  algorithm(s) reproduce(s) best <br /> The quality comparisons assessed               feature essential genes presented networks <br /> Receiver-Operator Characteristic curve (ROC curve),              and, hand network(s) largest <br /> where sensitivity vs. false discovery rate (FDR)     identifiable number essential genes. Here refer to <br /> each possible cut-off ranked gene list plotted. The      Rich Essential Metabolic Genetic network, or <br /> area ROC curve (AUC) calculated using          REMG network. <br /> empirical method [20]. The AUC estimate how <br /> good classifier differentiate essential      18 different networks built following procedures <br /> non-essential genes. This variant Mann Whitney       described (see Methods). These networks be <br /> U statistic. An effective classifier generate AUC sig-   grouped according data source used build them: <br /> nificantly greater 0.50 (the expected AUC value       KEGG iND750 (see Table 1). These data sets con- <br /> random classifier), calculated confidence       stitute common sources metabolism recon- <br /> intervals (CI) AUC scores using formula CI = &#194;&#177;       struction S. cerevisiae. Note reconstructing a <br /> z * SE(AUC), z z-value given significance      model metabolism, assumptions made <br /> level (0.1, 0.05, 0.01 case) SE(AUC)           regarding edge set present model. In study <br /> standard error AUC [20]. In addition, calculated       consider common assumptions, <br /> the minimum error (minE) ROC curve follows,          including reversibility relationships, insulation <br /> minE = (MIN [(1-sensitivity)2+(FDR)2]1/2). This tracks       organelle function, elimination highly <br /> point curve closer perfect prediction. The   connected metabolites (see Methods). Thus, genetic <br /> prediction accuracy ROC curve calculated         networks used study similar (see Table 1) and <br /> [TP+TN]/[TP+TN+FP+FN] [7]. Sensitivity defined TP/         constitute sampling common reconstructing strat- <br /> (TP+FN), TP = true positives (genes truly predicted        egies build models metabolism. <br /> as essential) FN = false negatives (missed essential <br /> genes). FDR defined FP/(TN+FP), TN = true            All networks analyzed disconnected (i.e., <br /> negatives (genes truly predicted non-critical) FP =       presented gene not <br /> false positives (missed non-essential genes). Note          path connect rest genes) presented <br /> Specificity = 1 - FDR. <br />  <br /> Estimating completeness genetic networks <br /> We estimated fraction identical edges each <br /> reconstructed network work probabilistic <br /> functional network yeast genes [4]; percentage of <br /> identical links estimated dividing value the <br /> number edges present reconstructed network <br /> (See Table 1, column &quot;Overlap&quot;). <br />  <br /> To estimate completeness networks, missing <br /> relationships non-predicted essential genes the <br /> KEGG2path network determined grouping all <br /> essential genes Gene Ontology biological process <br /> ID. Then, missing local relationships unpredicted <br /> essential genes groups determined. <br />                                                                  Figure <br />                                                                  The <br />                                                                  trality <br />                                                                        efficacy <br />                                                                          measures <br />                                                                            2 predict essential metabolic genes cen- <br /> Using criteria, estimated completeness       The efficacy predict essential metabolic genes <br /> the reconstructed networks; however, kept        centrality measures. The maximum Area Under the <br />                                                                  ROC Curve (max AUC) obtained Mann-Whitney <br /> mind precise approach estimate <br />                                                                  test (see Methods) applied 18 different metabolic net- <br /> the completeness genetic networks, results           works used study (see Table 1) shown plot <br /> may used approximation problem.                 black squares. Each square represent max AUC <br />                                                                  obtained groups 1, 2, 3 4 centrality measures used <br /> Results                                                          differentiate essential non-essential genes. The verti- <br /> The reliable identification essential genes biologi-     cal lines crossing square represent confidence inter- <br /> cal network depends network structure            val 99%. <br /> the algorithm used identify essentiality genes. <br />  <br />  <br />                                                                                                                      Page 7 12 <br />                                                                                                (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                               http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> a giant component (data shown), previously                 REMGNetSuppTable3?modo=2&amp;table=suppTable3B&amp;or <br /> reported metabolic networks [15].                       derBy=C&amp;orderMode=DESC                           http:// <br />                                                                   bis.ifc.unam.mx:8090/REMGNets/ <br /> Prediction essential genes single centrality measure      SuppTable3?modo=2&amp;table=suppTable3C&amp;orderBy=AU <br /> Using Mann Whitney U test normalized                   C&amp;orderMode=DESC). Particularly, combined cen- <br /> number possible pairings (see Methods), evaluated           tralities closeness-&quot;1/clustering coefficient&quot;, excentricity- <br /> the efficacy 16 different centrality measures differen-     &quot;1/clustering coefficient&quot;, 1/excentricity-&quot;1/clustering <br /> tiate essential non-essential genes 18 genetic net-       coefficient&quot; radiality-&quot;1/clustering coefficient&quot; ren- <br /> works. We selected best network-centrality pairs           dered significant predictions iND750 KEGG <br /> those i) AUC score 0.50 and, ii) AUC             networks. <br /> score confidence interval 0.50 99% of <br /> significance (see Methods). From analysis,                Completeness REMG networks <br /> detected KEGG2path-Closeness pair satisfied         The combination 2 centralities did not <br /> these criteria (see Additional File 1 Figure 2).          improve correct prediction metabolic essential <br /> However, 276 tests performed (14 &#195;&#8212; 16 + 4 &#195;&#8212; 13),          genes (see Figure 2). In case, possible some <br /> almost 3 network-centrality pairs expected chance         essential genes detected because <br /> to positive level significance. Thus, results   REMG networks include relevant genetic <br /> indicate single centrality measures identify         relations metabolism, reasons. Indeed, <br /> critical genes statistical significance.                     observed networks did include the <br />                                                                   known genetic relationships metabolic genes: only <br /> Prediction essential genes combining centrality             17% 34% known metabolic genetic relationships <br /> measures                                                          (see Methods) included given network used in <br /> We noticed genes predicted individual centrality          study (see Table 1). <br /> measures did overlap (to access specific sets of <br /> overlapping genes http://bis.ifc.unam.mx:8090/REM             Furthermore, identify relevant missing genetic rela- <br /> GNets/SupplementaryData/SuppTable2.html); therefore,              tionships REMG network (KEGG2path network, as <br /> we hypothesized combining predictions dif-          detected combining 1 2, 3 4 centrality measures; <br /> ferent centralities improve reliability achieved                     http://bis.ifc.unam.mx:8090/REMGNets/ <br /> the individual centralities. To test hypothesis, com-      SuppTable3?modo=2&amp;table=suppTable3A&amp;orderBy=AU <br /> bined scores 2, 3 4 centrality measures sin-      C&amp;orderMode=DESC,              http://bis.ifc.unam.mx:8090/ <br /> gle Combined Score, CS(v), assigned new                  REMGNetSuppTable3?modo=2&amp;table=suppTable3B&amp;or <br /> centrality measure gene (see Methods). That is,          derBy=C&amp;orderMode=DESC                           http:// <br /> instead assigning 16 different centrality measures          bis.ifc.unam.mx:8090/REMGNets/ <br /> every gene given network, number combined             SuppTable3?modo=2&amp;table=suppTable3C&amp;orderBy=AU <br /> centrality measures n!/(n-i)!*i! (n: number indi-          C&amp;orderMode=DESC respectively), aimed the <br /> vidual centrality measures applied given network; i:         unpredicted essential metabolic genes. It shown <br /> number combined centralities, i.e., = 2, 3 4). That       essential genes tend Essential Complex <br /> is, number combined centralities 2,562,               Biological Modules [21]; case, missing local <br /> 13,244 48,300 groups 2, 3 4 individual         genetic relationships unpredicted essential genes may <br /> centrality measures, respectively. For group com-         modules. As proof concept, <br /> bined centralities, efficiency differentiating essen-      essential genes predicted local (Clustering <br /> tial non-essential genes estimated using               coefficient) global (Closeness centrality) centrality <br /> variant Mann Whitney U test (see Methods). In         measures using minimum error criteria case. <br /> case too, best network-&quot;combined centrality meas-             This identifies 99 essential genes 131 total known <br /> ures&quot; pairs selected based AUC score &gt; 0.5             metabolic        essential       genes        (see   http:// <br /> where confidence interval 99% &gt; 0.5. Figure 2          bis.ifc.unam.mx:8090/REMGNets/SupplementaryData/ <br /> shows maximum AUC scores obtained groups 2,            SuppTable2.html). The missing 32 essential genes be <br /> 3 4 centralities. The numbers positive hits observed       reduced 18 including essential genes predicted <br /> (884, 5,023 20,250) clearly expected            local centrality measure (YBR153W, YDR236C, <br /> number positive hits chance (26, 132 483            YDR050C, YKL192C, YNR016C, YBR196C, YDL055C, <br /> the groups 2, 3 4 combined centrality measure-             YBL076C, YPR033C, YPL160W, YFL022C, YDR023W, <br /> ments, respectively), 99% confidence level (see http://        YGR185C, YHR019C, YGR094W, YDR341C, YBR121C, <br /> bis.ifc.unam.mx:8090/REMGNets/                                    YDR037W). These genes classified using April&apos;s <br /> SuppTable3?modo=2&amp;table=suppTable3A&amp;orderBy=AU                    2008 Gene Ontology classification [22] 6 <br /> C&amp;orderMode=DESC,             http://bis.ifc.unam.mx:8090/        (YER003C, YKL152C, YCR012W, YKL182W, YPL231W, <br />  <br />  <br />                                                                                                                       Page 8 12 <br />                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                                  http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> YBR256W) 18 unpredicted essential meta-                            centrality measures. Our results indicate closeness <br /> bolic genes shared classification predicted                       centrality applied KEGG2path network identify the <br /> essential genes (see Additional File 2). According                       largest fraction essential metabolic genes (see Addi- <br /> reported probabilistic functional gene network yeast                         tional File 1). As stated, result agreement with <br /> [4], 6 genes engaged 119 different metabolic                       previous reports showing global centrality measures <br /> genetic relations, 26 present                    (e.g., betweeness, closeness, synthetic lethality) other <br /> KEGG2path network (data shown). Thus, REMG                              chemical-based approaches [23-25] efficient identifi- <br /> network KEGG2path does missing genetic relation-                           ers essential genes [26-32]. However, overall <br /> ships.                                                                          number predictions study taken account, <br />                                                                                 prediction significant: closeness have <br /> Discussion                                                                      observed 4 times reliable predictor in <br /> Our work describes procedure identify essential genes                      18 similar networks. Hence, closeness highly <br /> from genetic networks (see Figure 1). We claim                      depends structure KEGG2path network to <br /> current scenario knowledge molecular                           achieve correct identification essential genes. We <br /> network structures complete yet, necessary                      conclude single centrality measure tested <br /> test mathematical operation reproduce essenti-                       predict essential genes statistically significant way. <br /> ality genes single model network. To achieve <br /> this, report procedure simultaneously varies                          The discrepancy results previous reports may <br /> both network structure mathematical opera-                          explained different statistical analysis per- <br /> tion reproduces gene essentiality. Considering                         formed: previous studies prediction essential <br /> infinite number possible network structures given                      genes achieved particular centrality cut-off value <br /> set genes, reasonable approach use knowledge-                        (e.g., 20% genes largest centrality values) and <br /> based networks reconstructed different criteria.                           cut-off value statistical parameter reported <br /> Hence, used 2 knowledge-based networks sharing                             (i.e., sensitivity, specificity, accuracy; Table 3 a <br /> common set genes genetic relationships (i.e.,                            summary reports). This assumption may <br /> KEGG iND750) use variations accounting                              induce errors parameters reported Table <br /> some common assumptions metabolic                            dependent chosen cut-off value: large cut- <br /> network reconstruction (see Methods Table 1).                               values, sensitivity tend present large values and <br />                                                                                 specificity small ones, trend reversed <br /> To differentiate essential non-essential genes 18                       small cut-off values. Our method considered possi- <br /> different metabolic genetic networks, total 204,233                        ble cut-off values did observe statistical <br /> centrality calculations performed using 16 different <br />  <br /> Table 3: Comparison statistical parameters used estimate efficacy identify essential metabolic genes yeast <br />  <br />  Model         Method         Essentials/Non-essentials                Sensitivity        Specificity        Error      Accuracy          Reference <br />  <br />  iFF708        FBA            23/90                                    0.13               1                  0.87       82%               22 <br />  <br />  iFF708        FBA            91/508                                   0.31               0.95               0.69       85%               6 <br />  <br />  iFF708        FBA            79-146/562-629                           0.40-0.53          90-96%             NR         NR                33 <br />  <br />  iND750        FBA            118/NR                                   0.31               NR                 NR         NR                10 <br />  <br />  iLL672        FBA            NR/NR                                    0.68-0.80          96-98%             NR         NR                33 <br />  <br />  iFF708        MOMA           46/302                                   0.60               0.92               0.41       88%               34 <br />  <br />  iLL672        MOMA           NR/NR                                    0.73-0.80%         NR                 NR         NR                33 <br />  <br />  iND750        SA             100/NR                                   0.14               NR                 NR         NR                7 <br />  <br />  Previous studies efficiency identify essential metabolic genes yeast, report different statistical parameters confidence (Sensitivity, <br />  Specificity, Error Accuracy columns), commonly use single network (Model column) single method (Method column) identify <br />  essential genes. The data obtained works reported &quot;Reference&quot; column. FBA: Flux-Balance Analysis; MOMA: Method of <br />  Minimization Metabolic Adjustment; SA: Synthetic accessibility; NR: value reported cited reference. <br />  <br />  <br />                                                                                                                                               Page 9 12 <br />                                                                                                                      (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                             http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> significance predictions gene essentiality      These results indicate best way represent gene <br /> any individual centrality measure.                             essentiality combining single features estimated <br />                                                                individual centrality measures. This turn, reflects the <br /> Because this, valid ask current net-   complex nature essential genes [5]. Specifically, genes <br /> work structure metabolism efficiently related     essential establish different relationships <br /> gene essentiality? Recent studies shows incom-        (local and/or global) genes organism: <br /> plete knowledge gene function metabolism          genes local contacts pathways essential <br /> explain incorrect gene essentiality predictions achieved    cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>survival</span> (e.g., glycolysis); alternatively, essential <br /> centrality factors responsible    genes distant relationships genes other <br /> essential role genes metabolism [8,21,33,34]. How-       non-essential pathways essential maintain <br /> ever, argue ways measure centrality net-   cellular homeostasis instance. <br /> works improve correct prediction essential <br /> genes. Our argument based observation dif-      It important emphasize method <br /> ferent essential genes predicted different centrality   improved limitations previous works reliably <br /> measures (see http://bis.ifc.unam.mx:8090/REMGNets/            identify essential genes, does correctly predict the <br /> SupplementaryData/SuppTable2.html). Thus, combining            known essential metabolic genes. This expected, the <br /> centrality measures improve significance          essentiality genes comes regulatory dynam- <br /> these predictions.                                             ics quantitative aspect function. Although <br />                                                                aspects captured static net- <br /> Indeed, combining 2 centrality measures improved           works analyzed work centrality measures <br /> overall significant predictions (see Figure 2). The com-       tested here, possible missing essential genes <br /> bined predictions specifically achieved best          detected adding missing genetic relations <br /> results included local global centralities (see       metabolic networks, any. To determine are <br /> http://bis.ifc.unam.mx:8090/REMGNets/                          missing genetic relationships REMG networks, we <br /> SuppTable3?modo=2&amp;table=suppTable3A&amp;orderBy=AU                 performed analyses: i) identification known <br /> C&amp;orderMode=DESC,             http://bis.ifc.unam.mx:8090/     metabolic genetic relationships (see Methods) our <br /> REMGNetSuppTable3?modo=2&amp;table=suppTable3B&amp;or                  reconstructed networks ii) identification the <br /> derBy=C&amp;orderMode=DESC                           http://   missing local relationships non-predicted essential <br /> bis.ifc.unam.mx:8090/REMGNets/                                 genes (see Methods). <br /> SuppTable3?modo=2&amp;table=suppTable3C&amp;orderBy=AU <br /> C&amp;orderMode=DESC). Particularly, top-ranked com-           Our results known metabolic genetic <br /> bined centralities (AUC &gt; 0.60) successfully identified        relationships missing networks. In case, <br /> essential genes KEGG iND750 networks:          improvements prediction essential genes <br /> observed number predictions (4 more) 4         achieved adding missing genetic <br /> combined centralities expected number        relationships; missing relations be <br /> random prediction 18 different networks tested (p      added? <br /> = 0.01). These combined centralities composed the <br /> local centrality 1/&quot;clustering coefficient&quot; global     We achieved particular REMG <br /> centralities closeness, radiality excentricity. Small 1/   network (KEGG2path) extracting unpredicted <br /> &quot;clustering coefficient&quot; values identify genes highly     essential genes Essential Complex Biological Mod- <br /> connected neighbouring genes, like edge        ules [21]. We observed 6 18 unpredicted <br /> networks leaf node (last node network)        essential metabolic genes share classification <br /> only 1 single neighbour connected; hence,      truly predicted ones (see Additional File 2) may <br /> genes large CS(v) values tend end      local interactions essential complexes <br /> of metabolic pathways critical enzymatic            belong to. For instance, gene YER003C an <br /> reactions located. On hand, closeness, excen-    unpredicted essential gene shares biological <br /> tricity radiality estimate near gene       process classification (i.e., GDP-mannose biosynthetic <br /> the others; thus, genes high values centrali-    process) essential genes (see Additional File 2); the <br /> ties tend hubs network communication             location YER003C end metabolic network <br /> where essential proteins located too. Thus, com-      connects pathways, detected by <br /> bining centrality measures identify genes       approach. Interestingly, 2 genes constitute the <br /> are hubs metabolism, genes located end met-      GDP-mannose biosynthetic process S. cerevisiae were <br /> abolic pathways both.                                       KEGG dataset, KEGG did include the <br />                                                                GDP-mannose biosynthetic pathway, REMG net- <br />                                                                work did include relationship. In summary, our <br />  <br />  <br />                                                                                                                    Page 10 12 <br />                                                                                               (page number citation purposes) <br />  BMC Systems Biology 2009, 3:102                                                            http://www.biomedcentral.com/1752-0509/3/102 <br />  <br />  <br />  <br /> REMG network does missing relations. Adding                      Acknowledgements <br /> of improved prediction essential                     We like thank Aurora Labastida performing cal- <br /> genes using method.                                                    culations reported work suggestion use statistical test <br />                                                                            reported study. Dr. Boris Thibert providing <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> to <br />                                                                            calculate centralities used work; acknowledge the <br /> As noted above, essential role genes deter- <br />                                                                            technical support received Dra. Maria Teresa Lara Ortiz the <br /> mined factors included network                     Information Technology core Instituto Fisiolog&#195;&#173;a Celular-UNAM; <br /> structures, variations media composi-                   like acknowledge fruitful encouraging discussions on <br /> tion and/or kinetics. Therefore, method comple-                    subject Dra. Alicia Gonzalez Manjarrez. Finally, greatly appre- <br /> ment analysis guide                              ciate reviewers BMC Systems Biology journal helped us <br /> experimentalists direct experiments complete                     improve description work. <br /> understanding structure-function relationship <br /> present metabolism.                                                     This work supported grants Universidad <br />                                                                            Nacional Aut&#195;&#179;noma Mexico (Macroproyecto UNAM: Tecnolog&#195;&#173;as para <br />                                                                            la Universidad la Informaci&#195;&#179;n y la Computaci&#195;&#179;n PAPIIT IN217708) <br /> Conclusion                                                                 grant SEP-CoNaCyT 82308, GDR, Ger- <br /> In summary, effective procedure identify                 man Ministry Education Research (BMBF) grant 0312706A to <br /> essential genes genetic networks. Our method use-                  DKO. <br /> ful current scenario uncertainty about <br /> the completeness network consequently,                   References <br /> correct representation gene essentiality. We                  1.    Wagner A: How reconstruct large genetic network from <br />                                                                                  n gene perturbations fewer n2 easy steps. Bioinformatics <br /> essentiality represented combination centrality                       2001, 17:1183-1197. <br /> measures, revealing complex nature function                  2.    Barret CL, Palsson BO: Iterative reconstruction transcrip- <br /> essential genes. It expected improvements                        tional regulatory networks: algorithmic approach. PLoS <br />                                                                                  Comp Biol 2006, 2:e52. <br /> in essential gene predictions achieved adding                    3.    Ideker T, Thorsson V, Ranish JA, Christmas R, Buhler J, Eng JK, Bum- <br /> missing genetic relationships metabolic networks.                           garner R, Goodlett DR, Aebersold R, Hood L: Integrated genomic <br />                                                                                  proteomic analyses systematically perturbed meta- <br />                                                                                  bolic network. Science 2001, 292:929-934. <br /> Authors&apos; contributions                                                     4.    Lee I, Li Z, Marcotte EM: An Improved, Bias-Reduced Probabil- <br /> GDR conceived original idea, developed                           istic Functional Gene Network Baker&apos;s Yeast, Saccharo- <br />                                                                                  myces cerevisiae. PLoS ONE 2007, 2:e988. <br /> codes generate networks reported work,                      5.    Gerdes S, Edwards R, Kubal M, Fonstein M, Stevens R, Osterman A: <br /> developed <span id='am-240' about='Thesaurus:code' typeof='owl:Thing'>code</span> calculate combined centralities                        Essential genes metabolic maps. Curr Opin Biotechnol 2006, <br /> reported work mainly wrote manuscript.                          17:448-56. <br />                                                                            6.    F&#195;&#182;rster J, Famili I, Palsson BO, Nielsen J: Large-scale evaluation of <br /> DK implemented codes calculate cen-                           silico gene deletions Saccharomyces cerevisiae. OMICS <br /> trality measures reported work; discussed                        2003, 7:193-202. <br />                                                                            7.    Wunderlich Z, Mirny LA: Using topology metabolic net- <br /> and wrote parts manuscript. GC performed                             works predict viability mutant strains. Biophys J 2006, <br /> classification genes based GO database gen-                        91:2304-2311. <br /> erated supplementary Figure 1. All authors read                        8.    Becker SA, Palsson BO: Three factors underlying incorrect in <br />                                                                                  silico predictions essential metabolic genes. BMC Syst Biol <br /> approved final manuscript.                                                   2008, 2:14. <br />                                                                            9.    Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, <br />                                                                                  Kawashima S, Katayama T, Araki M, Hirakawa M: From genomics <br /> Additional material                                                              chemical genomics: new developments KEGG. Nucleic <br />                                                                                  Acids Res 2006, 34:D354-D357. <br />                                                                            10.   Duarte NC, Herrg&#195;&#165;rd MJ, Palsson B&#195;&#732;: Reconstruction valida- <br />  Additional file 1                                                               tion Saccharomyces cerevisiae iND750, fully compart- <br />  Table S1. AUC values obtained local global centralities         mentalized genome-scale metabolic model. Genome Res 2004, <br />  genetic metabolic network.                                                      14:1298-1309. <br />                                                                            11.   Segal E, Shapira M, Regev A, Pe&apos;er D, Botstein D, Koller D, Friedman <br />  Click file                                                             N: Module networks: identifying regulatory modules and <br />  [http://www.biomedcentral.com/content/supplementary/1752-                       condition-specific regulators gene expression <br />  0509-3-102-S1.HTML]                                                             data. Nat Genet 2003, 34:167-177. <br />                                                                            12.   Kuh <br /> </body></html>