<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>298&#226;&#8364;&#8220;308 Nucleic Acids Research, 2009, Vol. 37, No. 1                                                        Published online 28 November 2008 <br /> doi:10.1093/nar/gkn925 <br />  <br />  <br /> Rex1p deficiency leads accumulation of <br /> precursor initiator tRNAMet polyadenylation <br /> of substrate RNAs Saccharomyces cerevisiae <br /> Sarah G. Ozanick1, Xuying Wang1, Michael Costanzo2, Renee L. Brost2, <br /> Charles Boone2 James T. Anderson1,* <br /> 1 <br />  Department Biological Sciences, Marquette University, Milwaukee, WI, USA 2Banting Best <br /> Department Medical Research Department Molecular Genetics, Donnelly CCBR, University <br /> of Toronto, Toronto, Canada <br />  <br /> Received August 5, 2008; Revised October 1, 2008; Accepted November 3, 2008 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                           While 50 leader removal, carried RNase P, a <br />                                                                                 conserved process diverse organisms, 30 end pro- <br /> A synthetic genetic array used identify lethal                           cessing complex varied <br /> and slow-growth phenotypes produced                                      process, di&#239;&#172;&#8364;erences organisms, but <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> TRM6, encodes tRNA modifica-                                di&#239;&#172;&#8364;erent tRNAs single organism <br /> tion enzyme subunit, combined dele-                                (3,4). In Saccharomyces cerevisiae, endonucleases <br /> tion non-essential gene Saccharomyces                                 exonucleases process 30 ends tRNA <br /> cerevisiae. We deletion REX1                                  vitro (5). A model tRNA 30 end <br /> gene resulted slow-growth <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                                 processing proposed Lhp1 protein <br /> trm6-504 strain. Previously, REX1 shown                               binds 30 end precursor tRNA promotes pro- <br /> involved processing 3&#226;&#8364;&#8482; ends 5S rRNA                               cessing endonucleolytic cleavage, absence <br /> the dimeric tRNAArg-tRNAAsp. In study,                             Lhp1p, precursors instead processed exonu- <br /> discovered requirement Rex1p processing                                cleases (6). However, tRNA precursors, no <br />                                                                                 change 30 end processing seen strains lacking <br /> the 3&#226;&#8364;&#8482; end tRNAiMet precursors pre- <br />                                                                                 Lhp1p (6,7). This observation, fact LHP1 is <br /> cursor tRNAiMet accumulates trm6-504 rex1D                                 essential gene, suggests tRNA 30 end processing in <br /> strain. Loss Rex1p results polyadenylation                             yeast occur multiple pathways (8). <br /> its substrates, including tRNAiMet, suggesting                                The nucleases involved tRNA 30 processing in <br /> defects 3&#226;&#8364;&#8482; end processing activate                                   S. cerevisiae generally unknown, including endo- <br /> nuclear surveillance pathway. Finally, purified                                 nuclease cleaves tRNAs bound Lhp1, been <br /> Rex1p displays Mg2+-dependent ribonuclease                                      identi&#239;&#172;?ed fully characterized. Several exo- and <br /> activity vitro, enzyme inactivated                                endonucleases process 30 ends tRNAs vitro <br /> by mutation highly conserved amino acids.                                puri&#239;&#172;?ed yeast, corresponding <br />                                                                                 genes identi&#239;&#172;?ed, role enzymes <br />                                                                                 tRNA processing vivo unknown (5). tRNase Z, an <br />                                                                                 endonuclease domains life that <br /> INTRODUCTION <br />                                                                                 removes 30 trailer tRNAs, identi&#239;&#172;?ed <br /> The production mature non-coding RNA,                                yeast (TRZ1) essential viability <br /> tRNA rRNA, does result transcription                               (9,10). While vitro endonuclease activity this <br /> alone, requires number post-transcriptional                       enzyme observed using human precursor <br /> processing modi&#239;&#172;?cation reactions. For example,                              tRNAArg substrate (11), function yeast Trz1p <br /> tRNAs transcribed precursors extra                             vivo known. Lastly, ribonuclease encoded the <br /> nucleotides 50 30 ends&#226;&#8364;&#8221;and                            REX1 gene shown trim 30 ends some <br /> introns well&#226;&#8364;&#8221;that need removed (1). Further-                           intron-containing tRNAs tRNAArg produced <br /> more, tRNAs acquire abundant nucleoside modi&#239;&#172;?cations,                           dimeric transcript encoding tRNAAsp (12,13). <br /> including <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> ribose methylations, base isomeriza-                            Previously, REX1 named RNA82 described as <br /> tions base deaminations, introduced post-                          nuclease involved processing 5S rRNA (14). In an <br /> transcriptionally numerous enzymes (2).                                      rna82.1 mutant strain, 5S rRNA an <br />  <br /> *<span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed. Tel: +1 414 288 1481; Fax: +1 414 288 7357; Email: james.anderson@mu.edu <br />  <br /> &#195;&#376; 2008 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />                                                                        Nucleic Acids Research, 2009, Vol. 37, No. 1 299 <br />  <br />  <br /> extended 30 end, 13 extra nucleotides detected     Plasmid construction <br /> in pulse-labeling experiments, addi- <br />                                                               IMT4 previously cloned HindIII site of <br /> tional nucleotides steady-state RNA samples          pRS316 HindIII fragment isolated plasmid C- <br /> (14). In addition, dimeric tRNAArg-tRNAAsp tran-          50 (24), giving pJA108. Mutagenesis 30 end IMT4 <br /> script fully processed extract rna82.1     carried using Quik Change site-directed mutagen- <br /> cells, 30 end tRNAArg retain nucleo-   esis (Stratagene) pJA108 template mutagenic <br /> tides normally removed extract wild-type         primers JA546 JA547. The correct sequence the <br /> strain (15). Rex1p appears process 30 ends     mutant, p467, con&#239;&#172;?rmed DNA sequencing. A <br /> some tRNAs contain introns,                      yeast shuttle vector containing REX1 control <br /> tRNATyr(GUA) tRNALys(UUU) (13). With avail-           galactose-inducible promoter, pAV101, kindly <br /> ability protein sequence data, Rex1p classi&#239;&#172;?ed    provided Dr. Ambro van Hoof. Alanine substitutions <br /> as member DEDD superfamily exonucleases           residues D229 E231 introduced using Quik <br /> (16), activity demonstrated        Change site-directed mutagenesis (Stratagene), pri- <br /> using puri&#239;&#172;?ed Rex1 protein.                                   mers JA616 JA617 con&#239;&#172;?rmed DNA sequen- <br />    Previously, identi&#239;&#172;?ed characterized Trm6p       cing, giving p532. <br /> Trm61p S. cerevisiae, subunits tRNA            In vitro T7 RNA polymerase transcription constructs <br /> modi&#239;&#172;?cation enzyme methylates A58 T	C loop        created pUC18 (25), mature <br /> of tRNAs (17). In study, syn-       tRNAiMet construct (HG300), obtained a <br /> thetic slow-growth phenotype results REX1             gift Dr. Henri Grosjean. IMT3 approximately <br /> deleted trm6-504 mutant strain. We demonstrate           500-bp upstream sequence 600 bp downstream <br /> that Rex1p role initiator tRNAMet (tRNAiMet)         sequence ampli&#239;&#172;?ed yeast genomic DNA using <br /> 30 end processing tRNAiMet precursors,          primers JA571 JA572 cloned BamHI <br /> particularly extended 30 trailers, accumulate      sites pRS316 (21) BamHI fragment. The correct <br /> in trm6-504 rex1D strain. We loss Rex1p        sequence plasmid (p480) con&#239;&#172;?rmed DNA <br /> results polyadenylation tRNAiMet,        sequencing. For T7 transcription, IMT3 ampli&#239;&#172;?ed <br /> previously identi&#239;&#172;?ed substrates Rex1p&#226;&#8364;&#8221;5S rRNA          p480 using PCR primers JA602, includes <br /> tRNAArg-tRNAAsp. Rex1p puri&#239;&#172;?ed yeast displays            T7 promoter, JA603. The PCR product was <br /> ribonuclease activity mature precursor                digested SalI BamHI ligated pUC18 <br /> tRNAiMet provided substrate mutation           digested SalI BamHI, giving <br /> amino acids conserved DEDD exonucleases elim-           p522. The IMT4 T7 transcription construct created <br /> inates activity.                                         similarly amplifying tRNAiMet sequence HG300 <br />                                                               using PCR primers JA602 JA618. The PCR pro- <br />                                                               duct digested SalI BamHI ligated to <br /> MATERIALS AND METHODS                                         pUC18 digested SalI BamHI, <br />                                                               giving p528. <br /> Yeast strains <br /> REX1 deleted strains Y200 Y190 (18) inser-      SGA analysis <br /> tion kanamycin resistance gene. PCR ampli&#239;&#172;?cation       trm6-504 genetic interactions identi&#239;&#172;?ed using high- <br /> of KanMX4 cassette REX1 &#239;&#172;&#8218;anking sequence             throughput, array-based method known Synthetic <br /> was performed using primers JA396 JA397. Yeast            Genetic Array (26&#226;&#8364;&#8220;28). First, trm6-504 mutation was <br /> strains transformed described (19) PCR       introduced strain Y5563 [C. Boone lab, (29)] using <br /> product allowed grow Yeast extract&#226;&#8364;&#8220;Peptone&#226;&#8364;&#8220;         PCR products; containing trm6-504 with <br /> Dextrose (YPD) liquid media 308C 3 h      sequence 30 end complementary URA3 pro- <br /> plated YPD 200 mg/ml geneticin. Transformants         moter (made using primers JA269 JA270), a <br /> were replica printed YPD geneticin result-    second product encoding URA3MX6 sequence at <br /> ing colonies screened correct insertion KanMX4     30 end complementary sequence downstream of <br /> by yeast colony PCR (20) primer sets JA370           trm6-504. [ampli&#239;&#172;?ed using primers JA271 JA272 <br /> JA398, JA385 JA398 JA399 JA398.                   template p4348 (28)]. Y5563 co-transformed <br /> Deletion REX1 Y190 named Y386                DNA fragments transformants were <br /> Y200 named Y387. Y200, Y190, Y386 Y387                selected Sc URA. The integration URA3 cor- <br /> transformed IMT4 (p108) imt4-3 (p467)            rect location con&#239;&#172;?rmed PCR presence of <br /> strains Y450-453 Y454-457, respectively. Similarly,       trm6-504 veri&#239;&#172;?ed temperature sensitive pheno- <br /> strains carrying plasmid pRS316 (21) strains    type; strain crossed set  5000 <br /> Y468-471. Y200, Y190, Y386 Y387 strains carrying          viable haploid gene deletion strains (29). A series of <br /> the plasmid YCplac33 (22) Y266, Y265, Y439          robotic arraying procedures enabled selection haploid <br /> and Y402, respectively. For puri&#239;&#172;?cation wild-type (Wt)     double mutant meiotic progeny, subsequently <br /> and mutant Rex1p, Y386 transformed pAV101,           examined defects colony growth 268C 308C <br /> to create Y438, p532, create Y495. Strain H2457        (29). Approximately 200 candidate synthetic lethal slow <br /> with high-copy plasmids carrying IMT1-4 genes         growth interactions identi&#239;&#172;?ed high-throughput <br /> been described (23).                                          screen. <br />  300 Nucleic Acids Research, 2009, Vol. 37, No. 1 <br />  <br />  <br /> RNA isolation                                                   run sample, accounting lower <br /> Total RNA yeast puri&#239;&#172;?ed using                      total RNA used oligo (dT) cellulose puri&#239;&#172;?- <br /> MasterPure Yeast RNA Puri&#239;&#172;?cation Kit (Epicentre).               cation sample. <br /> When using MasterPure Kit, cultures grown to <br /> OD600 = 0.6   0.8 cells pelleted 1.5&#226;&#8364;&#8220;3 ml         Purification Rex1p <br /> of culture. <br />                                                                 A 25 ml culture Y438 Y495 grown 308C syn- <br /> Northern blot analysis                                          thetic complete medium lacking uracil (Sc URA) used <br />                                                                 inoculate 1 l culture Sc URA lacking glucose and <br /> Total RNA separated 8% 10% polyacrylamide             supplemented 1% galactose 2% sucrose (ScGal/ <br /> gels containing 8 M urea electrophoresis carried        Suc URA). Cells grown 308C, harvested at <br /> out 1   TBE 450 V. Gels containing radioactive sam-       OD600 = 0.8   1, pellets frozen dry ice and <br /> ples, used activity assays, dried prior   stored  808C. To purify Rex1p, cell pellet was <br /> to visualization using phosphorimaging (GE Healthcare).         thawed ice resuspended breaking bu&#239;&#172;&#8364;er <br /> Gels &#239;&#172;?rst &#239;&#172;?xed 15 min 15% methanol, 5%            [20 mM HEPES, pH 7, 150 mM NaCl, 1  complete pro- <br /> acetic acid solution, dried 728C 55 min      tease inhibitor cocktail (Roche)]. Cells disrupted <br /> a slab gel dryer (Thermo Scienti&#239;&#172;?c). For northern blotting,     using French press (Thermo IEC) pressure 20 <br /> gels soaked 10 min 0.5   TBE samples <br />                                                                 000 psi lysate subjected centrifugation 3000g, <br /> transferred Biodyne B nylon membrane (Pall Corpora- <br />                                                                 48C, 10 min. This clari&#239;&#172;?ed extract subjected <br /> tion) using semi dry electroblotter (Owl Separation <br />                                                                 centrifugation 70 000g, 48C, 30 min, clar- <br /> Systems) 400 mA 60 min. Following transfer, the <br />                                                                 i&#239;&#172;?ed supernatant added anti-FLAG M2 agarose <br /> RNA cross-linked membrane using UV <br />                                                                 (Sigma) pre-equilibrated wash bu&#239;&#172;&#8364;er (20 mM HEPES, <br /> stratalinker (Stratagene). After probed, blots <br />                                                                 pH 7.0, 150 mM NaCl). After incubation gentle <br /> were stripped 800 ml boiling hot 1% SDS, <br />                                                                 mixing 48C 2 h, resin collected centrifu- <br /> and tested residual signal exposure a <br /> phosphorimager screen overnight prior probed           gation 1000   g, 48C, 5 min supernatant <br /> again.                                                          discarded. The resin washed 10 ml wash bu&#239;&#172;&#8364;er, <br />                                                                 collected again, supernatant discarded. The resin <br />                                                                 transferred 10 ml column washed with <br /> Purification poly (A)+ RNA <br />                                                                 10 ml wash bu&#239;&#172;&#8364;er. Rex1p eluted 1 ml breaking <br /> Yeast strains grown O.D.600 0.5&#226;&#8364;&#8220;0.6            bu&#239;&#172;&#8364;er containing 150 ng/ml FLAG peptide (Sigma), and <br /> 250 ml YPD 308C. Cells harvested centrifuga-         collected &#239;&#172;?ve fractions. The fractions analyzed <br /> tion 6000   g 5 min 48C, washed                  SDS-PAGE followed Coomassie staining or <br /> 1   phosphate bu&#239;&#172;&#8364;ered saline frozen   808C.              Western blotting anti-FLAG M2 antibody <br /> Puri&#239;&#172;?cation total poly (A) + RNA car-           (Stratagene). <br /> ried described (18) using 50 mg oligo (dT) cellu- <br /> lose (New England Biolabs). Poly (A) + RNA eluted <br /> with 2 ml elution bu&#239;&#172;&#8364;er collected 0.5 ml fractions       Rex1p vitro assays <br /> The OD260 fraction determined fractions <br /> having greatest amounts RNA pooled.                 tRNA substrates transcribed vitro using T7 RNA <br /> Samples extracted AE (50 mM sodium               polymerase. DNA templates linearized restriction <br /> acetate, pH 5.0 10 mM EDTA) bu&#239;&#172;&#8364;er-saturated                 digest using BstNI HG300 DraI p522 and <br /> phenol aqueous layer extracted chloro-       p528. The vitro transcription reaction (100 ml) consisted <br /> form: isoamylalcohol (24:1). RNA precipitated          1  T7 RNA polymerase bu&#239;&#172;&#8364;er (New England Biolabs), <br /> 1/10 volume 3 M sodium acetate (pH 5.3) 3 volumes           1.25 mM rNTPs, 10 mM DTT, 20&#226;&#8364;&#8220;40U RNasin ribonu- <br /> of ethanol  208C overnight. After precipitation, poly        clease inhibitor (Promega),  400 ng DNA template, and <br /> (A) + RNA collected centrifugation 14 000 rpm         250U T7 RNA polymerase (New England Biolabs). After <br /> for 15 min 48C. Pellets washed 70% ethanol         incubation 378C 3 h, 1U RQ1 DNase added and <br /> and allowed air dry. The poly (A) + RNA resus-           samples incubated 378C 20 min. RNA extracted <br /> pended 15 ml nuclease free water. For northern blot-      AE bu&#239;&#172;&#8364;er-saturated phenol, pH 5, ethanol precipi- <br /> ting, equal amounts poly (A) + RNA loaded               tated, resuspended nuclease free water. After <br /> based di&#239;&#172;&#8364;erent inputs total RNA used              separation 10% denaturing polyacrylamide gel, <br /> oligo (dT) selection. The total RNA concentration            tRNA visualized methylene blue staining gel <br /> each sample determined UV spectroscopy               puri&#239;&#172;?ed. tRNAs dephosphorylated 50 end <br /> the sample highest total RNA             labeled using 32P-ATP. Scintillation counting used to <br /> set 100%. All samples calculated per-        determine concentration labeled tRNA. Activity <br /> centage sample poly                   assays carried 20 mM HEPES, pH 7, 5 mM <br /> (A) + RNA loaded gel increased based            MgCl2 100 mM NaCl 308C. Reaction products <br /> the di&#239;&#172;&#8364;erence total RNA input. For example,       separated 10% denaturing polyacrylamide <br /> of samples 65% total RNA            gel visualized using phosphorimaging or <br /> sample set 100%, 35% poly (A) + RNA            autoradiography. <br />                                                                          Nucleic Acids Research, 2009, Vol. 37, No. 1 301 <br />  <br />  <br /> RESULTS <br /> trm6-504 rex1D exhibit synthetic genetic interaction <br /> Trm6p Trm61p comprise m1A58 tRNA methly- <br /> transferase S. cerevisiae, previously shown <br /> that trm6-504 mutant, lacks m1A58 all <br /> tRNA, reduced levels tRNAiMet grown at <br /> 308C (23). Synthetic genetic array (26) analysis con- <br /> ducted identify synthetic lethal (SGA) synthetic <br /> slow-growth interactions trm6-504 nonessen-      Figure 1. A trm6-504 rex1D double mutant strain displays synthetic <br /> tial yeast genes. Genes encoding proteins known      slow-growth phenotype. Wt (Y200), trm6-504 (Y190), trm6-504 rex1D <br /> to involved tRNA modi&#239;&#172;?cation processing       (Y386) rex1D (Y387) yeast strains streaked SD+His+Ura <br />                                                              grown room temperature 3 days. <br /> detected array, including tRNA modi&#239;&#172;?cation <br /> enzymes act tRNAiMet. In addition, genes encod- <br /> ing proteins unknown function, proteins   Table 1. Oligonucleotides used northern analysis <br /> known functions established roles tRNAiMet <br /> production, identi&#239;&#172;?ed (Anderson,J.T., unpublished       Name       Target                Sequence (50 !30 ) <br /> data). One genes known function RNH70/         JA 11      tRNAiMet              TCGGTTTCGATCCGAGGACATC <br /> REX1, encodes nuclease shown                                          AGGGTTATGA <br /> be involved rRNA tRNA maturation.                     JA   48    50 IMT3               ACGGCGCTTAACTTTTATG <br />    Because REX1 known process tRNAiMet,         JA   66    50 IMT4               GGCGCTTAGCCAACTTG <br />                                                              JA   72    50 IMT2               GGCGCTGCTAAATCATGAG <br /> were intrigued SGA &#239;&#172;?ndings. According          JA   99    5S rRNA               TCGCGTATGGTCACCCACTACA <br /> SGA analysis, trm6-504 rex1D double-mutant strain          JA   555   tRNAVal(CAC)          GAAGGCAACGTGATAGCCGC <br /> exhibited severe slow-growth phenotype; however,        JA   557   dimeric pre-tRNAArg   AGAAACAAAGCACTCACGAT <br /> did observe consistent slow-growth phenotype a <br /> trm6-504 rex1D strain sporulation tetrad dissec- <br /> tion strain generated array. Therefore, we <br /> deleted REX1 trm6-504 strain (Y190) stu- <br />                                                              trm6-504 trm6-504 rex1D strains, noticed <br /> died extensively (18). A trm6-504 strain grows slowly at <br />                                                              subset tRNAiMet precursors accumulated to <br /> 308C, unable additional growth <br />                                                              greater levels trm6-504 rex1D lane Wt, <br /> defect loss REX1 trm6-504 rex1D <br />                                                              trm6-504, rex1D lanes (Figure 2A). The blot was <br /> double-mutant strain grown rich medium <br />                                                              probed 5S rRNA loading control small dif- <br /> (YPD) 308C. However, trm6-504 rex1D <br /> double-mutant cells grown synthetic dextrose minimal      ferences observed samples. Strains which <br /> medium supplemented histidine uracil                REX1 deleted slightly longer 5S rRNA than <br /> (SD+His+Ura), conditions resemble          Wt strain (Figure 2A), agreement been <br /> SGA screen, 308C room temperature, consistently        reported previously (12,14). <br /> displayed modest growth defect compared             <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> previous work, know forms of <br /> trm6-504 strain (Figure 1).                                  precursor tRNAiMet migrate di&#239;&#172;&#8364;erent positions on <br />                                                              denaturing 8% polyacrylamide gel di&#239;&#172;&#8364;erences in <br />                                                              <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> 30 trailer (Figure 2B) (23). The slower- <br /> Initiator tRNAMet processing altered                      migrating species consists tRNAiMet precursors <br /> in trm6-504 rex1D double mutant strain                     encoded IMT2 IMT3, 6&#226;&#8364;&#8220;7 nucleotides <br /> Although trm6-504 strain grows slowly 308C (23)     end mature tRNAiMet sequence a <br /> exhibits reduced levels mature tRNAiMet, raising          RNA Polymerase III (Pol III) termination signal, stretch <br /> the level initiator tRNAMet introducing high-        &#239;&#172;?ve thymines (30) (Figure 2B C), creating a <br /> copy (hc) plasmid carrying IMT4, encodes               long trailer (LT). tRNAiMet precursors encoded IMT1 <br /> tRNAiMet, restores robust growth (23). Because SGA       IMT4 migrate quickly, having nucleo- <br /> and experiments revealed trm6-504 rex1D         tide, short trailer (ST), predicted Pol III ter- <br /> double mutant grew slower trm6-504 mutant,          mination signal (Figure 2B C). As seen Northern <br /> we wanted determine mature              analysis, slower-migrating species more <br /> tRNAiMet trm6-504 rex1D strain trm6-      intense trm6-504 rex1D double mutant strain <br /> 504 strain. Total RNA extracted Wt (Y200),          (Figure 2A), suggesting tRNAiMet precursors encoded <br /> trm6-504 (Y190), trm6-504 rex1D (Y386) rex1D             IMT2 IMT3 accumulate, encoded by <br /> (Y387) strains grown 308C separated electro-       IMT1 IMT4 not. To test hypothesis, we <br /> phoresis using denaturing 8% polyacrylamide gel.           probed Northern blot total RNA Wt <br /> Northern analysis performed using probe (JA11,         (Y200), trm6-504 (Y190), trm6-504 rex1D (Y386) and <br /> Table 1) detects precursor tRNAiMet, con-    rex1D (Y387) strains speci&#239;&#172;?cally IMT2 IMT3 <br /> tains 50 leader 30 trailer sequence, mature          using probes hybridize unique 50 leader <br /> tRNAiMet (Figure 2A). While did observe            sequences (Figure 2D). Both IMT2- IMT3-encoded <br /> obvious di&#239;&#172;&#8364;erence mature tRNAiMet           precursors (LT) abundant trm6-504 rex1D <br />  302 Nucleic Acids Research, 2009, Vol. 37, No. 1 <br />  <br />  <br />                                                                            strain Wt, trm6-504, rex1D strain (Figure 2D). <br />                                                                            In addition, appeared small increase the <br />                                                                            IMT2- IMT3-encoded precursor (LT) <br />                                                                            tRNAiMet rex1D strain compared Wt <br />                                                                            strain, suggesting Rex1p involved processing <br />                                                                            tRNAiMet lacks m1A58. Levels of <br />                                                                            IMT3-encoded precursor tRNAiMet elevated in <br />                                                                            trm6-504 strain, lesser degree seen in <br />                                                                            trm6-504 rex1D strain (Figure 2D). Finally, IMT4- <br />                                                                            encoded precursor did accumulate trm6-504 <br />                                                                            rex1D strain (data shown). These data suggest that <br />                                                                            Rex1p required e&#239;&#172;&#402;ciently process tRNAiMet precur- <br />                                                                            sors longer 30 trailing sequences (LT) as <br />                                                                            crucial processing precursors short 30 trailing <br />                                                                            sequences (ST). <br />                                                                               Because IMT2 IMT3 tRNAiMet precursors accu- <br />                                                                            mulate trm6-504 rex1D double mutant strain, rea- <br />                                                                            soned precursor tRNAiMet encoded IMT1 or <br />                                                                            IMT4 accumulate 30 trailer sequence <br />                                                                            lengthened. Mutagenesis carried plasmid <br />                                                                            carrying IMT4 (pJA108), short 30 trailer (ST) <br />                                                                            IMT4 replaced longer trailer (LT) of <br />                                                                            IMT3. Therefore, precursor tRNAiMet transcribed <br />                                                                            imt4-3 construct (p467) 50 leader of <br />                                                                            IMT4 30 trailer IMT3 preceding &#239;&#172;?rst Pol <br />                                                                            III termination sequence (Figure 2C). To ensure this <br />                                                                            mutant tRNA functional, imt4-3 plasmid was <br />                                                                            tested ability promote growth trm6-504 <br />                                                                            strain 308C, temperature strain a <br />                                                                            slow-growth phenotype (23). While trm6-504 strain <br />                                                                            carrying hcIMT4 showed vigorous growth, strain carry- <br />                                                                            ing vector grew slowly (Figure 3A). The trm6- <br />                                                                            504 strain carrying imt4-3 plasmid showed robust <br />                                                                            growth (Figure 3A), suggesting functional tRNAiMet <br />                                                                            produced imt4-3 plasmid. <br />                                                                               Total RNA prepared Wt (Y200), trm6-504 <br />                                                                            (Y190), trm6-504 rex1D (Y386) rex1D (Y387) strains <br />                                                                            containing plasmid plasmids IMT4 imt4- <br />                                                                            3. Northern analysis performed using probes that <br />                                                                            hybridize 5&#226;&#8364;&#8482; leader IMT4 5S rRNA, <br />                                                                            serves loading control. All strains carry- <br />                                                                            ing plasmid approximately same <br />                                                                            IMT4-encoded precursor tRNAiMet, did all <br />                                                                            strains carrying IMT4 plasmid (Figure 3B). <br /> Figure 2. A trm6-504 rex1D strain accumulates tRNAiMet encoded          However, strains expressing imt4-3-encoded <br /> IMT2 IMT3. (A) Total RNA (5 mg) isolated Wt (Y200), trm6-         precursor tRNAiMet, increase precursor <br /> 504 (Y190), trm6-504 rex1D (Y386) rex1D (Y387) strains sepa-       trm6-504 strain greater increase in <br /> rated electrophoresis using 8% denaturing polyacrylamide gel. <br /> Northern analysis performed radiolabeled oligonucleotides         trm6-504 rex1D strain (Figure 3B). This result indi- <br /> that hybridize tRNAiMet (JA11) 5S rRNA (JA99). Results          cates length 30 trailer sequence in&#239;&#172;&#8218;uences <br /> visualized autoradiography. (B) Northern analysis total RNA          precursor tRNAiMet in&#239;&#172;&#8218;uenced Rex1p <br /> (10 mg) isolated trm6-504 strain (H2457) carrying        processing. <br /> four IMT genes high-copy plasmid. The blot probed for <br /> tRNAiMet (JA11) results visualized autoradiography. <br />                                                                               We note trm6-504 rex1D strain carrying high- <br /> (C) An alignment tRNAsiMet encoded IMT genes            copy IMT4 plasmid shows accumulation precursor <br /> from S. cerevisiae. A portion sequence mature          species longer IMT4-encoded precursor <br /> tRNA shown black background. The 30 trailer sequences          tRNAiMet. It likely precursor tRNAiMet cre- <br /> shown black text, putative RNA Polymerase III termination      ated Pol III fails stop transcribing &#239;&#172;?rst <br /> sites underlined. A downstream terminator IMT4 underlined a <br /> dashed line. The imt4-3 mutant alteration length 30   termination sequence instead reads a <br /> trailer preceding &#239;&#172;?rst transcriptional terminator. (D) Northern ana-   second downstream terminator (Figure 2C). In fact, the <br /> lysis total RNA (20 mg) performed described (A), probing     sequences CAAAAC CATATC, similar to <br /> oligonucleotides complementary 50 leader IMT2- IMT3-          sequence follows &#239;&#172;?rst IMT4 terminator <br /> encoded tRNAiMet (JA72 JA48, respectively). <br />                                                                            (Figure 2C) shown allow transcriptional <br />                                                                                      Nucleic Acids Research, 2009, Vol. 37, No. 1 303 <br />  <br />  <br />  <br />  <br /> Figure 3. Changes 30 trailer length in&#239;&#172;&#8218;uence tRNAiMet accumulation trm6-504 rex1D strain. (A) Wt (Y200) trm6-504 (Y190) strains <br /> carrying vector (YCplac33), hcIMT4, hcimt4-3 grown 308C 2 days SC URA. (B) Total RNA (20 mg) isolated Wt (Y200), trm6- <br /> 504 (Y190), trm6-504 rex1D (Y386) rex1D (Y387) strains carrying vector (YCplac33), hcIMT4 (pJA108) hcimt4-3 (p467) subjected <br /> to Northern analysis. The blot probed radiolabeled oligonucleotide complementary 50 leader IMT4 (JA66), results <br /> visualized using phosphorimaging. The blot stripped, probed oligonucleotide hybridizes 5S rRNA (JA99), analyzed by <br /> phosphorimaging. <br />  <br /> read-through yeast RNA Pol III vitro (31). As                  poly(A) + tRNAiMet present trm6-504 strain, <br /> species longer trailer, consistent            Wt strain, consistent published data <br /> other results precursor tRNAiMet accumulates                 (Figure 4A) (18). Poly(A) + tRNAiMet detected <br /> the trm6-504 rex1D strain.                                                rex1D strain, &#239;&#172;?nding suggests ine&#239;&#172;&#402;cient 30 end <br />                                                                           processing tRNAiMet activates nuclear tRNA sur- <br /> Increased polyadenylation tRNAiMet occurs                           veillance pathway. In trm6-504 rex1D strain, greater <br /> a trm6-504 rex1D double mutant strain                                     poly(A) + tRNAiMet evident, approxi- <br />                                                                           mately times trm6-504 rex1D <br /> Previously, hypomethylated tRNAiMet                     strain (Figure 4A B). This &#239;&#172;?nding implies tRNAiMet <br /> from trm6-504 strain activates nuclear tRNA surveil-                  lacks m1A58 unprocessed 30 end espe- <br /> lance pathway results polyadenylation                         cially susceptible polyadenylation, and, presumably, <br /> and degradation (18). We wondered activation                    degradation. Furthermore, polyadenylation another <br /> surveillance pathway changed trm6-504 rex1D                      m1A58-containing tRNA&#226;&#8364;&#8221;tRNAVal(CAC)&#226;&#8364;&#8221;occurs in <br /> strain. As polyadenylated tRNAiMet detected                     absence Rex1p (Figure 4C). tRNAVal(CAC) precur- <br /> in total RNA samples, isolated total RNA                      sors detected total RNA samples accumulated in <br /> poly(A) + RNA Wt (Y200), trm6-504 (Y190), trm6-                      strains lacking Rex1p (Figure 4C), note that <br /> 504 rex1D (Y386) rex1D (Y387) strains. Northern                       tRNAVal(CAC) encoded genes Pol <br /> analysis performed using probe detects                    III terminator located 4 12 nucleotides downstream <br /> mature precursor tRNAiMet. In samples total                        end mature tRNAVal(CAC) sequence. <br /> RNA, precursor tRNAiMet accumulated trm6-504 <br /> rex1D strain (Figure 4A), lesser extent a <br /> trm6-504 rex1D strain, observed previously                          Loss Rex1p results polyadenylation 5S <br />                                                                           rRNA tRNAArg <br /> (Figure 2A), substantial di&#239;&#172;&#8364;erences (1.5-fold at <br /> most) seen detecting loading control                          Prior discovery nuclear RNA surveillance <br /> (data shown). In poly(A) + RNA samples,                               pathway, 5S rRNA rna82.1 (rex1) mutant strain <br />  304 Nucleic Acids Research, 2009, Vol. 37, No. 1 <br />  <br />  <br />                                                                            Northern analysis poly(A) + RNA performed a <br />                                                                            probe 5S rRNA showed poly(A) + 5S rRNA pre- <br />                                                                            sent rex1D trm6-504 rex1D strains, Wt <br />                                                                            trm6-504 strain (Figure 5A). There obvious <br />                                                                            di&#239;&#172;&#8364;erence poly(A) + 5S RNA in <br />                                                                            trm6-504 rex1D strain rex1D strain. Therefore, <br />                                                                            unlike seen tRNAiMet, polyadenylation 5S <br />                                                                            RNA result loss Rex1p does be <br />                                                                            in&#239;&#172;&#8218;uenced presence absence trm6-504. This <br />                                                                            result expected, 5S RNA Trm6p substrate, <br />                                                                            expected exhibit altered structure <br />                                                                            absence m1A. <br />                                                                               The tRNAArg-tRNAAsp dimeric transcript processed <br />                                                                            series endonucleolytic exonucleolytic steps to <br />                                                                            produce mature tRNAs (32). While endonucleolytic <br />                                                                            cleavages separate tRNAs gen- <br />                                                                            erate mature 50 ends, exonucleolytic processing by <br />                                                                            Rex1p believed produce mature 30 end of <br />                                                                            tRNAArg, possibly tRNAAsp (12,15,32). To look at <br />                                                                            possible polyadenylation tRNAArg-tRNAAsp, con- <br />                                                                            ducted Northern analysis probe hybridizes to <br />                                                                            30 end tRNAArg 10 nt linker region between <br />                                                                            tRNAs. In total RNA samples, observed <br />                                                                            greater amounts tRNAArg fully processed <br />                                                                            trm6-504 rex1D rex1D strains (Figure 5B). This <br />                                                                            result consistent previous studies showed a <br />                                                                            rex1D strain contains 30 extended tRNAArg precursors <br />                                                                            encoded dimeric tRNAArg-tRNAAsp transcript <br />                                                                            (12). 5S rRNA detected loading control no <br />                                                                            signi&#239;&#172;?cant variations (1.3-fold most) seen between <br />                                                                            total RNA samples (data shown). The dimeric <br />                                                                            transcript detected total RNA samples, but <br />                                                                            abundant trm6-504 strain; instead, much <br />                                                                            larger RNAs, appearing smear near the <br />                                                                            membrane, detected. While explain this <br />                                                                            observation present, hope understand result <br />                                                                            future studies RNA processing trm6-504 <br />                                                                            strain. From poly(A) + samples, that <br />                                                                            polyadenylation tRNAArg occurring trm6- <br />                                                                            504 rex1D rex1D strains, approximately the <br />                                                                            extent (Figure 5B). In addition, note some <br />                                                                            poly(A) + RNA detected large the <br />                                                                            dimeric tRNAArg-tRNAAsp transcript. We conclude that <br />                                                                            tRNAArg possibly dimeric tRNAArg-tRNAAsp <br />                                                                            transcript e&#239;&#172;&#402;ciently processed 30 ends <br />                                                                            Rex1p absent polyadenylated. This <br /> Figure 4. An increase level polyadenylated tRNAiMet seen in <br /> a trm6-504 rex1D strain. (A) Total RNA (5 mg) poly(A) + RNA            contrast polyadenylation tRNAiMet seen a <br /> (3&#226;&#8364;&#8220;4 mg, normalized total RNA yield, &#226;&#8364;&#732;Methods&#226;&#8364;&#8482; section)     trm6-504 mutant believe altered tRNAiMet <br /> separated 6% denaturing polyacrylamide gel. Northern analysis         structure, absence m1A58, activates the <br /> was performed radiolabeled oligonucleotide complementary         RNA surveillance pathway. While tRNAArg m1A58 <br /> tRNAiMet (JA11). (B) A bar graph relative amounts of <br /> poly(A) + tRNAiMet Wt, trm6-504, trm6-504 rex1D rex1D <br />                                                                            Wt strain, tRNAArg polyadeny- <br /> strains, seen (A). Quantitation performed using ImageQuant       lated lacking modi&#239;&#172;?cation, trm6-504 <br /> TL software (GE Healthcare). (C) The blot (A) stripped          strain (Figure 5B). These data indicate ine&#239;&#172;&#402;ciently <br /> probed tRNAVal(CAC) (JA555). Results visualized auto-          processed precursor tRNAs activate RNA surveil- <br /> radiography blots. The mature tRNA poly(A) + lanes         lance pathway. <br /> represents tRNA eliminated oligo-(dT) selection. <br />  <br /> had poly(A) tail 20 A&#226;&#8364;&#8482;s                         Rex1p exhibits tRNA 3&#226;&#8364;&#8482; end processing activity <br /> in length (14). This led test 5S rRNA,                   A 60 kDa exonuclease capable removing 30 trailer <br /> the known processing substrate Rex1p, tRNAArg-                    tRNA substrate previously puri&#239;&#172;?ed from <br /> tRNAAsp, polyadenylated rex1D strain.                        S. cerevisiae extracts (5); given Rex1p predicted <br />                                                                                 Nucleic Acids Research, 2009, Vol. 37, No. 1 305 <br />  <br />  <br />                                                                        Western blotting anti-FLAG antibodies (data <br />                                                                        shown). In order con&#239;&#172;?dent observed <br />                                                                        exonuclease activity belonged Rex1p con- <br />                                                                        taminant, designed puri&#239;&#172;?ed D229A/E231A <br />                                                                        mutant Rex1p. This mutant alanine substituted the <br />                                                                        highly conserved DE residues ExoI motif of <br />                                                                        DEDD exoribonucleases (Figure 6A). For RNase T from <br />                                                                        Escherichia coli, DEDD exonuclease, has <br />                                                                        shown alanine substitution these <br />                                                                        positions inactivates enzyme (Figure 6A) (33). <br />                                                                           To assay activity Rex1p, tRNAiMet sub- <br />                                                                        strates prepared vitro using T7 RNA polymerase&#226;&#8364;&#8221; <br />                                                                        &#226;&#8364;&#732;mature&#226;&#8364;&#8482; tRNAiMet ends CCA, &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; <br />                                                                        tRNAiMet seven nucleotides trailer of <br />                                                                        IMT3 stretch &#239;&#172;?ve uridines, &#226;&#8364;&#732;IMT4&#226;&#8364;&#8482; <br />                                                                        tRNAiMet, ends nucleotide trailer of <br />                                                                        IMT4 stretch &#239;&#172;?ve uridines. tRNAiMet substrates <br />                                                                        labeled 32P 50 end incubated with <br />                                                                        puri&#239;&#172;?ed Wt mutant Rex1p. Reaction products were <br />                                                                        separated denaturing polyacrylamide gel visua- <br />                                                                        lized exposure dried gel phosphorimager <br />                                                                        screen. All substrates shortened 30 <br />                                                                        ends presence Wt Rex1p, unchanged <br />                                                                        incubated bu&#239;&#172;&#8364;er presence the <br />                                                                        mutant Rex1p (Figure 6C). This result illustrates the <br />                                                                        nuclease activity Rex1p, demonstrates this <br />                                                                        activity lost highly conserved amino acids <br />                                                                        mutated. All substrates converted pro- <br />                                                                        ducts similar length, meaning nucleotides <br />                                                                        removed &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; substrate the <br />                                                                        &#226;&#8364;&#732;IMT4&#226;&#8364;&#8482; substrate, &#226;&#8364;&#732;IMT4&#226;&#8364;&#8482; substrate <br />                                                                        &#226;&#8364;&#732;mature&#226;&#8364;&#8482; substrate. We note &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; sub- <br />                                                                        strate fully digested 10 min incubation <br />                                                                        period. Despite gel puri&#239;&#172;?cation, substrate consisted of <br />                                                                        bands, slower-migrating species consis- <br />                                                                        tently resistant trimming Rex1p. <br />                                                                           To visualize exonuclease activity Rex1p, mon- <br />                                                                        itored product formation time. The &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; substrate <br />                                                                        incubated Wt Rex1p period 0.5, 1, 3 or <br />                                                                        7 min. In addition, control reactions consisting the <br />                                                                        &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; substrate added MgCl2 &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; <br /> Figure 5. Loss Rex1p results polyadenylation 5S rRNA      &#226;&#8364;&#732;mature&#226;&#8364;&#8482; substrates Rex1p, carried out <br /> tRNAArg dimeric tRNAArg-tRNAAsp transcript. (A) Total and <br /> poly (A) + RNA analyzed described Figure 4. The blot was <br />                                                                        7 min. Reaction products separated denatur- <br /> probed 5S rRNA (JA99) results visualized autoradio-     ing 10% polyacrylamide gel visualized phosphor- <br /> graphy. (B) Northern analysis performed using labeled oligonu-   imaging. Exonucleolytic processing &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; <br /> cleotide (JA557) hybridizes 30 end tRNAArg      substrate Rex1p observed, illustrated the <br /> linker sequence tRNAs. Phosphorimaging used        intermediates seen (Figure 6D). Processing <br /> to analyze results. The presence mature forms RNAs the <br /> poly(A) + lanes represent RNAs eliminated oligo-(dT)        observed control reaction lacking Mg2+ <br /> selection.                                                             (Figure 6D), demonstrating Mg2+ required for <br />                                                                        Rex1p activity, unexpected other <br />                                                                        DEDD family members require divalent cations (16). <br />                                                                           In addition vitro studies, investigated the <br /> to 63 kDa, likely exonuclease Rex1p.            e&#239;&#172;&#8364;ects Wt mutant Rex1p expression vivo. Total <br /> Rex1p, Rex2p, Rex3p Rex4p, possesses                    RNA isolated trm6-504 rex1D strains containing <br /> four acidic amino acids, spread Exo motifs,                 vector, REX1, rex1 mutant, grown either <br /> which highly conserved DEDD family                    synthetic complete medium lacking uracil (Sc URA), <br /> exoribonucleases (Figure 6A) (16). We used galactose-                does induce REX1 expression, Sc URA lack- <br /> inducible promoter drive over-expression FLAG-                   ing glucose supplemented galactose sucrose <br /> tagged Rex1p trm6-504 rex1D strain, puri&#239;&#172;?ed                   (ScGal/Suc URA), induces REX1 expression. <br /> Rex1p using a&#239;&#172;&#402;nity chromatography (Figure 6B).                         Northern analysis performed using probe detects <br /> Puri&#239;&#172;?ed protein con&#239;&#172;?rmed Rex1p based size                    precursor mature tRNAiMet showed that <br />  306 Nucleic Acids Research, 2009, Vol. 37, No. 1 <br />  <br />  <br />                                                                          induction Wt Rex1p prevented accumulation of <br />                                                                          tRNAiMet precursors observed previously, induc- <br />                                                                          tion mutant Rex1p did prevent accumulation of <br />                                                                          precursor tRNAiMet (Figure 6E). A loading control did <br />                                                                          substantial di&#239;&#172;&#8364;erences (1.5-fold most) between <br />                                                                          samples (data shown). We conclude exoribo- <br />                                                                          nuclease activity Rex1p responsible processing <br />                                                                          hypomethylated tRNAiMet precursors vivo. <br />  <br />  <br />                                                                          DISCUSSION <br />                                                                          We shown tRNAiMet precursors accumulate a <br />                                                                          trm6-504 rex1D double-mutant strain&#226;&#8364;&#8221;in particular those <br />                                                                          encoded IMT2 IMT3. Compared IMT1 and <br />                                                                          IMT4, IMT2 IMT3 genes extended 30 <br />                                                                          trailer sequence preceding Pol III termination signal. <br />                                                                          We increasing length 30 trailer pre- <br />                                                                          ceding terminator IMT4 seven nucleo- <br />                                                                          tides resulted accumulation trm6-504 rex1D <br />                                                                          strain. One explanation results Rex1p <br />                                                                          speci&#239;&#172;?cally process tRNAiMet precursors long <br />                                                                          30 trailers, short 30 trailers. <br />                                                                          Alternatively, absence Rex1p, ribonu- <br />                                                                          cleases cell capable e&#239;&#172;&#402;ciently pro- <br />                                                                          cessing tRNAiMet precursors short trailers. In <br />                                                                          support interpretation, trm6-504 <br />                                                                          rex1D strain continued accumulate tRNAiMet precur- <br />                                                                          sors despite over-expression LHP1 (data shown). <br />                                                                          As increased levels Lhp1p proposed pro- <br />                                                                          mote endonucleolytic processing (6), does not <br />                                                                          appear endonuclease compensate for <br />                                                                          loss Rex1p function. In E. coli, redundancy <br />                                                                          substrate preference observed exonu- <br />                                                                          cleases process 30 ends tRNAs. While E. coli <br />                                                                          strain lacking &#239;&#172;?ve exonucleases&#226;&#8364;&#8221;RNases II, D, T, BN and <br />                                                                          PH&#226;&#8364;&#8221;is viable, growth restored of <br />                                                                          enzymes introduced, indicating exonu- <br />                                                                          cleases overlapping abilities (34). In addition, each <br /> Figure 6. Rex1p displays tRNA 30 end processing activity vivo     exonuclease able process precursor tRNA spe- <br /> in vitro. (A) A diagram Rex1p showing conserved amino acids       cies e&#239;&#172;&#402;ciently, poorly, suggesting that <br /> found Exo motifs. An alignment Rex1, 2, 3 4, and <br /> E. coli RNase T protein sequence shown underneath illustrate   di&#239;&#172;&#8364;erent groups tRNAs e&#239;&#172;&#8364;ectively processed <br /> conservation D   E sequence ExoI domain. (B) Wt      di&#239;&#172;&#8364;erent nucleases (4). <br /> and mutant FLAG-tagged Rex1p puri&#239;&#172;?ed yeast (strains               Currently, precursor tRNAs known poly- <br /> Y438 Y495, respectively) using a&#239;&#172;&#402;nity chromatography. Puri&#239;&#172;?ed        adenlyated degraded nuclear surveillance path- <br /> protein subjected SDS-PAGE visualized Coomassie <br /> staining. The positions molecular weight standards (Broad Range       way hypomodi&#239;&#172;?ed tRNAiMet S. cerevisiae a <br /> Protein Marker, New England Bioloabs) indicated. (C) Gel puri&#239;&#172;?ed     mutant tRNASer Schizosaccharomyces pombe <br /> 32 <br />   P- 50 end labeled tRNAsiMet (   10 pM) incubated bu&#239;&#172;&#8364;er         (18,35,36). We shown tRNAiMet and <br /> alone Wt mutant Rex1p ( 7.5 nM) 308C 10 min.           tRNAVal(CAC) polyadenylated rex1D strain, sug- <br /> Reaction products separated 10% denaturing polyacrylamide <br /> gel visualized autoradiography. The positions RNAs of <br />                                                                          gesting tRNAs unprocessed 30 ends sub- <br /> known length (Decade Marker, Ambion) indicated (in nucleotides).     strates nuclear surveillance machinery. The &#239;&#172;?nding <br /> (D) Wt Rex1p ( 7.5 nM) incubated labeled &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; tRNAiMet        tRNAiMet lacking m1A58 containing 30 exten- <br /> ( 20 pM) known periods time 0.5&#226;&#8364;&#8220;7 min, indicated.         sion apparently susceptible polyadenylation <br /> Control reactions &#226;&#8364;&#732;IMT3&#226;&#8364;&#8482; tRNAiMet lacking enzyme          suggests features in&#239;&#172;&#8218;uence e&#239;&#172;&#402;ciency of <br /> Mg2+ shown, control reaction lacking enzyme that <br /> contained &#226;&#8364;&#732;mature&#226;&#8364;&#8482; tRNAiMet. After separation denaturing        recognition nuclear surveillance machinery. In <br /> 10% polyacrylamide gel, results visualized using phosphorima-       addition, 5S rRNA dimeric tRNAArg-tRNAAsp <br /> ging. (E) Northern analysis total RNA (10 mg) isolated         transcript polyadenylated strains lacking Rex1p, <br /> trm6-504 rex1D strain carrying vector (pRS316), plasmid    suggesting RNAs subject degradation by <br /> with galactose-inducible Wt REX1 (pAV101) mutant rex1 (p532), <br /> grown non-inducing inducing conditions. The blot probed     nuclear surveillance pathway 30 ends are <br /> with radiolabeled oligonucleotide complementary tRNAiMet (JA11)     e&#239;&#172;&#402;ciently processed. This phenomenon be <br /> and results visualized using autoradiography.                        restricted 30 end processing defects, the <br />                                                                        Nucleic Acids Research, 2009, Vol. 37, No. 1 307 <br />  <br />  <br /> polyadenylated species observed retains 50 leader    degradation, shown RNase T (38). It <br /> sequence. This indicate loss m1A           noted tRNA substrates used this <br /> tRNAiMet perturbs 50 end processing RNaseP,            assay transcribed vitro lack modi&#239;&#172;?cations, <br /> that activates surveillance pathway. Since    including m1A58. The ability Rex1p process these <br /> is widely accepted RNaseP cleavage initiates proces-     unmodi&#239;&#172;?ed tRNAs vitro consistent observa- <br /> sing precursor tRNAs (32,37), explain       tion precursor tRNAiMet accumulates a <br /> increased levels polyA + tRNAiMet seen strains      rex1D strain tRNAiMet lacks m1A58. <br /> in 50 30 end processing slowed. We             We shown Rex1p Mg2+-dependent <br /> conclude RNAs unprocessed 30 ends           3 !50 exonuclease activity vitro, results are <br />                                                                 0 <br /> be added current list substrates nuclear    consistent previous &#239;&#172;?ndings 60 kDa protein pur- <br /> surveillance pathway.                                         i&#239;&#172;?ed yeast (5). In yeast, mature 30 ends of <br />    The role Rex1p RNA processing similar     tRNAiMet (prior CCA addition), tRNAsArg (from the <br /> of non-orthologous enzyme RNase T E. coli,           dimeric transcript, prior CCA addition) 5S rRNA <br /> which 30 &#226;&#8364;&#8220;50 exonuclease member        consist double-stranded stem 30 overhang of <br /> DEDD exonuclease superfamily (16,38). RNase T              single nucleotide. This indicates Rex1p func- <br /> required maturation 5S rRNA, and, like rex1D         tion binding single-stranded 30 trailer sequences and <br /> strain, E. coli strain lacking RNase T contains 5S         trimming RNA nucleotide double- <br /> rRNA extra nucleotides 30 end (39). RNase         stranded stem. Further studies identify products <br /> T involved tRNA maturation,           generated Rex1p vitro needed determine <br /> found trim 30 ends tRNA precursors (40). The        true. In addition, currently do <br /> crystal structures DEDD exonucleases, RNase        fully understand in&#239;&#172;&#8218;uence m1A58 modi&#239;&#172;?ca- <br /> T RNase D, reveal highly conserved acidic <br />                                                               tion 30 end processing. Rex1p appears able to <br /> amino acids (DEDD) coordinate divalent metal ions <br />                                                               process hypomethylated precursor tRNAiMet some <br /> involved catalysis (41,42). Consistent &#239;&#172;?nd- <br />                                                               degree, accumulation trm6-504 strain <br /> ings, observed Rex1p activity dependent on <br />                                                               trm6-504 rex1D strain. We propose the <br /> the presence Mg2+ alanine substitutions <br />                                                               absence m1A58 Rex1p activity, processing <br /> of conserved acidic amino acids (DE) inactivated the <br /> enzyme. The speci&#239;&#172;?city Rex1p RNase T 5S       precursor tRNAsiMet long 30 trailers carried <br /> rRNA tRNAs suggests features common          ine&#239;&#172;&#402;ciently nucleases, resulting the <br /> both substrates recognized enzymes.      accumulation polyadenylation tRNAsiMet have <br /> In yeast, 30 end precursor tRNAiMet 5S        observed. However, m1A58 present, processing by <br /> rRNA consists double-stranded stem followed           nucleases appears e&#239;&#172;&#8364;ective, the <br /> several unpaired nucleotides. For RNAs single        absence Rex1p, observe slight increase in <br /> stranded 30 end expected poly(U) tail      IMT2- IMT3-encoded precursor tRNAsiMet a <br /> due transcription RNA Pol III termination           rex1D strain. A thorough characterization of <br /> signal, detected poly(U) tails IMT3-       nucleases known involved identi&#239;&#172;?cation addi- <br /> encoded precursor tRNAiMet accumulates trm6-        tional nucleases involved tRNA 30 end processing may <br /> 504 rex1D strain (data shown). Interestingly,      allow better understand relationship between <br /> dimeric tRNAArg-tRNAAsp transcript, linker region         tRNA modi&#239;&#172;?cation 30 end maturation. <br /> between tRNAs U rich. Therefore, a <br /> double-stranded stem single-stranded U-rich <br /> sequence important Rex1p recognize its <br /> substrate, future studies needed explore      ACKNOWLEDGEMENTS <br /> possibility. While Rex1p recently shown             We thank Drs Ambro van Hoof, Michelle Steiger and <br /> involved 30 end processing intron-contain-     Richard Maraia critical reading manuscript. <br /> ing tRNAs (13), tRNAiMet tRNAVal con- <br /> tain introns, indicating presence intron not <br /> required recognition Rex1p. <br />    While results vivo experiments suggested <br /> Rex1p process tRNAiMet precursors long 30       FUNDING <br /> trailers, Rex1p processed 30 ends tRNAsiMet       National Institute General Medical Sciences <br /> both long short trailers vitro assay.           (GM066791 J.T.A., GM069949 J.T.A.); National <br /> Exonuclease activity observed                 Institutes Health Bethesda, Maryland; Genome <br /> tRNAiMet ending CCA used substrate. The           Canada Ontario Genomics Institute, <br /> absence nucleases proteins involved           Toronto, Ontario (2004-OGI-3-01 C.B.); Canadian <br /> tRNA processing vitro assay allow Rex1p         Institutes Health Research Ottawa, Ontario, (GSP- <br /> to remove nucleotides tRNAiMet substrates           41567 C.B.). Funding open access charge: <br /> would normally processed e&#239;&#172;&#402;ciently           National Institute General Medical Sciences, National <br /> enzymes vivo. Furthermore, tRNAiMet substrate          Institutes Health Bethesda, Maryland (GM069949). <br /> ending CCA aminoacylated <br /> in vivo, presumably block exonucleolytic          Con&#239;&#172;&#8218;ict statement. None declared. <br />  308 Nucleic Acids Research, 2009, Vol. 37, No. 1 <br />  <br />  <br /> REFERENCES                                                                   23. Anderson,J., Phan,L., Cuesta,R., Carlson,B.A., Pak,M., Asano,K., <br />                                                                                  Bjork,G.R., Tamame,M. Hinnebusch,A.G. (1998) <br />  1. Wolin,S.L. Matera,A.G. (1999) The trials travels tRNA.            The essential Gcd10p-Gcd14p nuclear complex required for <br />     Genes Dev., 13, 1&#226;&#8364;&#8220;10.                                                        1-methyladenosine modi&#239;&#172;?cation maturation initiator <br />  2. Bjo&#194;&#168;rk,G.R. (1995) In So&#194;&#168;ll,D., RajBhandary, U. L. (ed.), tRNA:              methionyl-tRNA. Genes Dev., 12, 3650&#226;&#8364;&#8220;3662. <br />     Structure Biosynthesis Function. ASM Press, Washington, D.C.         24. Dever,T.E., Yang,W., A&#203;&#353;stro&#194;&#168;m,S., Bystro&#194;&#168;m,A.S. and <br />     pp. 165&#226;&#8364;&#8220;206.                                                                 Hinnebusch,A.G. (1995) Modulation tRNAiMet, eIF-2 and <br />  3. Schu&#194;&#168;rer,H., Schi&#239;&#172;&#8364;er,S., Marchfelder,A. Mo&#194;&#168;rl,M. (2001) This Is          eIF-2B expression shows GCN4 translation inversely coupled <br />     End: Processing, Editing, Repair tRNA 30 -Terminus.           level eIF-2.GTP.Met-tRNAiMet ternary complexes. <br />     Biol. Chem., 382, 1147&#226;&#8364;&#8220;1156.                                                 Mol. Cell Biol., 15, 6351&#226;&#8364;&#8220;6363. <br />  4. Li,Z. Deutscher,M.P. (1996) Maturation pathways E. coli          25. Yanisch-Perron,C., Vieira,J. Messing,J. (1985) Improved M13 <br />     tRNA precursors: random multienzyme process vivo. Cell,                 phage cloning vectors host strains: nucleotide sequences of <br />     86, 503&#226;&#8364;&#8220;512.                                                                 M13mp18 pUC19 vectors. Gene, 33, 103&#226;&#8364;&#8220;119. <br />  5. Papadimitriou,A. Gross,H.J. (1996) Pre-tRNA 30 -processing           26. Palacios Moreno,J.M., Kildsgaard,J., Siwanowicz,I., <br />     Saccharomyces cerevisiae. Puri&#239;&#172;?cation characterization             Mortensen,K.K. Sperling-Petersen,H.U. (1998) Binding of <br />     exo- endoribonucleases. Eur. J. Biochem., 242, 747&#226;&#8364;&#8220;759.                  Escherichia coli initiation factor IF2 30S ribosomal subunits: a <br />  6. Yoo,C.J. Wolin,S.L. (1997) The yeast La protein required              functional role N-terminus factor. Biochem. Biophys. <br />     30 endonucleolytic cleavage matures tRNA precursors.            Res. Commun., 252, 465&#226;&#8364;&#8220;471. <br />     Cell, 89, 393&#226;&#8364;&#8220;402.                                                       27. Tong,A.H., Lesage,G., Bader,G.D., Ding,H., Xu,H., Xin,X., <br />  7. Kufel,J. Tollervey,D. (2003) 30 -processing yeast tRNATrp             Young,J., Berriz,G.F., Brost,R.L., Chang,M. et al. (2004) Global <br />     precedes 50 -processing. RNA, 9, 202&#226;&#8364;&#8220;208.                                    mapping yeast genetic interaction network. Science, <br />  8. Yoo,C.J. Wolin,S.L. (1994) La proteins Drosophila mela-             303, 808&#226;&#8364;&#8220;813. <br />     nogaster Saccharomyces cerevisiae: yeast homolog La         28. Tong,A.H.Y., Evangelista,M., Parsons,A.B., Xu,H., Bader,G.D., <br />     autoantigen dispensable growth. Mol. Cell Biol., 14, 5412&#226;&#8364;&#8220;5424.       Page,N., Robinson,M., Raghibizadeh,S., Hogue,C.W.V., Bussey,H. <br />  9. Chen,Y., Beck,A., Davenport,C., Chen,Y., Shattuck,D.                     et al. (2001) Systematic genetic analysis ordered arrays yeast <br />     Tavtigian,S.V. (2005) Characterization TRZ1, yeast homolog              deletion mutants. Science, 294, 2364&#226;&#8364;&#8220;2368. <br />     human candidate prostate cancer susceptibility gene ELAC2         29. Tong,A.H. Boone,C. (2006) Synthetic Genetic Array <br />     encoding tRNase Z. BMC Mol. Biol., 6, 12.                                    Analysis Saccharomyces cerevisiae. Methods Mol. Biol., <br /> 10. Vogel,A., Schilling,O., Spath,B. Marchfelder,A. (2005) The               313, 171&#226;&#8364;&#8220;192. <br />     tRNase Z family proteins: physiological functions, substrate          30. Allison,D.S. Hall,B.D. (1985) E&#239;&#172;&#8364;ects alterations 30 <br />     speci&#239;&#172;?city structural properties. Biol. Chem., 386, 1253&#226;&#8364;&#8220;1264.           &#239;&#172;&#8218;anking sequence vivo vitro expression yeast <br /> 11. Takaku,H., Minagawa,A., Takagi,M. Nashimoto,M. (2003) <br />                                                                                  SUP-o tRNATyr gene. EMBO J., 4, 2657&#226;&#8364;&#8220;2664. <br />     A candidate prostate cancer susceptibility gene encodes tRNA 30 <br />                                                                              31. Braglia,P., Percudani,R. Dicci,G. (2005) Sequence Context <br />     processing endoribonuclease. Nucleic Acids Res., 31, 2272&#226;&#8364;&#8220;2278. <br />                                                                                  E&#239;&#172;&#8364;ects Oligo(dT) Terminiation Signal Recognition by <br /> 12. van Hoof,A., Lennertz,P. Parker,R. (2000) Three conserved <br />                                                                                  Saccharomyces cerevisiae RNA Polymerase III. J. Biol. Chem., <br />     members RNase D family unique overlapping <br />                                                                                  280, 19551&#226;&#8364;&#8220;19562. <br />     functions processing 5S, 5.8S, U4, U5, RNase MRP and <br />                                                                              32. Engelke,D.R., Gegenheimer,P. Abelson,J. (1985) <br />     RNase P RNAs yeast. EMBO J., 19, 1357&#226;&#8364;&#8220;1365. <br />                                                                                  Nucleolytic processing tRNAArg-tRNAAsp dimeric precursor <br /> 13. Copela,L.A., Fernandez,C.F., Sherrer,R.L. Wolin,S.L. (2008) <br />                                                      <br /> </body></html>