<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2010 Genetics Society America <br /> DOI: 10.1534/genetics.110.114462 <br />  <br />  <br />  <br />          IscR Regulates RNase LS Activity Repressing rnlA Transcription <br />  <br />                       Yuichi Otsuka,* Kumiko Miki,* Mitsunori Koga,* Natsu Katayama,* <br />                        Wakako Morimoto,* Yasuhiro Takahashi&#226;&#8364;&#160; Tetsuro Yonesaki*,1 <br />           *Department Biological Sciences, Graduate School Science, Osaka University, Osaka 560-0043, Japan &#226;&#8364;&#160;Division <br />                 Life Science, Graduate School Science Engineering, Saitama University, Saitama 338-8570, Japan <br />                                                       Manuscript received January 19, 2010 <br />                                                       Accepted publication April 23, 2010 <br />  <br />  <br />                                                                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The Escherichia coli endoribonuclease LS originally identified potential antagonist of <br />               bacteriophage T4. When T4 dmd gene defective, RNase LS cleaves T4 mRNAs antagonizes <br />               T4 reproduction. This RNase plays important role RNA metabolisms E. coli. rnlA essential <br />               gene RNase LS activity, transcriptional regulation gene remains elucidated. An <br />               Fe-S cluster protein, IscR, acts transcription factor controls expression genes are <br />               necessary Fe-S cluster biogenesis. Here, report overexpression IscR suppressed RNase LS <br />               activity, causing loss antagonist activity phage T4. This suppressive effect did require the <br />               ligation Fe-S cluster IscR. b-Galactosidase reporter assays showed transcription rnlA <br />               promoter increased iscR-deleted cells compared wild-type cells, gel-mobility shift assays revealed <br />               specific binding IscR rnlA promoter region. RT&#226;&#8364;&#8220;PCR analysis demonstrated endogenous <br />               rnlA mRNA reduced overexpression IscR increased deletion iscR. <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> results, <br />               conclude IscR negatively regulates transcription rnlA represses RNase LS activity. <br />  <br />  <br />  <br />  <br /> A    N mRNA degradation activity Escherichia coli is <br />       rigidly controlled various cellular condi- <br /> tions. The degradation E. coli mRNA usually <br />                                                                             Crp (Iwamoto et al. 2008). rnlA essential RNase <br />                                                                             LS activity encodes endonuclease activity (Otsuka <br />                                                                             Yonesaki 2005; Otsuka et al. 2007). Recently, we <br /> initiated endonucleolytic cleavage (Kushner 2002)                        rnlB (formerly yfjO), located <br /> and major endonuclease, RNase E, triggers deg-                        downstream rnlA, role regulation of <br /> radation mRNAs (Bernstein et al. 2004). E. coli                     RNase LS activity (M. Koga, T. Yonesaki, un- <br /> has additional endonucleases RNase I, III,                          published results). rnlA rnlB reasonably as- <br /> G, P, Z cleave mRNAs,                           sumed form operon, transcriptional <br /> spectrum action limited (Bardwell et al. 1989;                        regulation genes assessed. <br /> Cannistraro Kennell 1991; Alifano et al. 1994;                             We isolated E. coli mutants defective RNase LS <br /> Umitsuki et al. 2001; Perwez Kushner 2006). We                          activity T4 dmd mutant able grow <br /> discovered endonuclese, RNase LS,                              (Otsuka et al. 2003). Two (std-2 -5) have <br /> potential antagonist T4 phage. When T4 dmd                             <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span>(s) rnlA. Interestingly, deletion iscR, <br /> mutant infects E. coli cells, RNase LS activity remarkably                  encodes transcription factor repress some <br /> increases early middle genes T4 phage                      genes depending cellular Fe-S level (Schwartz <br /> expressed (KAI et al. 1998; Ueno Yonesaki 2001)                         et al. 2001), completely eliminated ability other <br /> and causes rapid degradation mRNAs late                          (std-1, -3, -4) mutation(s) support the <br /> stages, leading growth defect T4 phage (Kai et al.                  growth T4 dmd mutant, indicating requirement of <br /> 1996). In vitro analysis confirmed dmd encodes                     iscR effects. Although relationship between <br /> an inhibitor RNase LS (Otsuka et al. 2007). In                           mutations iscR uncertain, iscR be <br /> addition T4 mRNA decay, RNase LS plays role                       key players regulation RNase LS <br /> in metabolism E. coli mRNAs 23S rRNA                        activity. <br /> (Otsuka Yonesaki 2005). Especially, RNase LS                               The isc operon, consisting iscRSUA-hscBA-fdx, <br /> degrades adenylate cyclase (cyaA) mRNA, resulting                        encodes proteins essential Fe-S cluster formation <br /> in accumulation cAMP transcription factor                      (Tokumoto Takahashi 2001). IscR originally <br />                                                                             discovered role negative control this <br />                                                                             operon (Schwartz et al. 2001). Transcriptome analysis <br />   Supporting information available online http://www.genetics.org/    identified 40 genes 20 predicted operons, were <br /> cgi/content/full/genetics.110.114462/DC1. <br />   1 <br />                                                                             regulated IscR aerobic anaerobic con- <br />    Corresponding author: Department Biological Sciences, Graduate <br /> School Science, Osaka University, 1-1 Machikaneyama-cho, Toyonaka,       ditions (Giel et al. 2006). IscR itself Fe-S cluster <br /> Osaka 560-0043, Japan. E-mail: yonesaki@bio.sci.osaka-u.ac.jp               protein Fe-S cluster IscR necessary for <br />  <br /> Genetics 185: 823&#226;&#8364;&#8220;830 ( July 2010) <br />  824                                                        Y. Otsuka et al. <br />  <br /> transcriptional repression isc operon (Schwartz                acids RnlA fused b-galactosidase constructed a <br /> et al. 2001). Recently, Nesbit et al. (2009) reported            similar manner. pRSrnlB-long constructed follows. A <br />                                                                       DNA fragment amplified PCR W3110 DNA a <br /> the Fe-S cluster dispensable IscR regulation of <br />                                                                       template primers 59-GGAATTCCATATCTCACTG <br /> some target promoters. Whether Fe-S                  GAGCCA 59-CGGATCCCCTGAGACGTTAATTCCGG. <br /> cluster necessary IscR-mediated transcriptional                The resulting fragment containing 854 bp upstream rnlB <br /> regulation appears depend type IscR-                     N-terminal 11 amino acids RnlB digested with <br /> binding sequence (Giel et al. 2006; Nesbit et al. 2009).              EcoRI BamHI ligated pRS552. <br />                                                                          Burst size assay: MH1 cells different <br />    In study, overexpression IscR <br />                                                                       plasmids grown 5 3 108 cells/ml LB medium and <br /> suppressed RNase LS activity alleviated growth                infected wild-type T4 dmd mutant multiplicity of <br /> defect T4 dmd mutant. Ligation Fe-S cluster               0.1 30&#194;&#176;. At 8 min, cells diluted 10 4-fold fresh <br /> with IscR required suppression RNase LS                LB medium incubated 70 min. After cells were <br /> activity. In addition, characterized transcription            lysed chloroform, total number progeny phages <br />                                                                       counted plating. The burst size ratio the <br /> the promoters rnlA rnlB, IscR <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                       number progeny number input phage and <br /> negatively regulate transcription rnlA promoter               infection wild-type cells wild-type T4 gives burst size <br /> but rnlB promoter.                                        80&#226;&#8364;&#8220;110. The relative burst size expressed percentage <br />                                                                       wild-type T4 wild-type cells harboring plasmid a <br />                                                                       vector plasmid. Each value mean three <br />                                                                       independent experiments standard deviation. <br />               MATERIALS AND METHODS <br />                                                                          RNA purification, Northern blot, primer extension, and <br />    Phages bacterial strains: Wild-type bacteriophage T4        RT&#226;&#8364;&#8220;PCR analysis: Isolation total RNA northern blot <br /> T4D. The amSF16 mutant contains amber mutation              analysis carried described previously (Kai et al. <br /> dmd gene (Kai et al. 1996; Ueno Yonesaki 2001). T4 GT7            1996). For primer extension analysis identify transcription <br /> phage used transduction (Wilson et al. 1979). The             start sites rnlA rnlB, used primers 59-GGAATTCC <br /> E. coli strains MH1 (sup0 hsdR DlacX74 rpsL) MG1655          TGGCATAGGACCAATTG rnlA 59-CGGATCCCCTGA <br /> used wild types. Strain MH1 DiscRTkan constructed           GACGTTAATTCCGG rnlB, respectively. For RT&#226;&#8364;&#8220;PCR anal- <br /> GT7-mediated transduction kanamycin-resistance marker            ysis, reverse transcription performed incubation 42&#194;&#176; <br /> from strain JW2515 (Baba et al. 2006) provided National            45 min 2 mg total RNA, 100 units ReverTra Ace <br /> BioResource Project (NIG, Japan). E. coli strain MH1 DrnlAB           reverse transcriptase (TOYOBO), 1 mm dNTPs, 5 pmol <br /> has deletion rnlA rnlB (M. Koga T. Yonesaki,             gene-specific primers 20 ml reverse transcription <br /> unpublished results). MG1655 Disc deletion isc           buffer. Primers, rnlA-dw N134 (59-CTGGCATAGGACCAATT), <br /> operon iscR iscX (formerly ORF3) (Tokumoto                ompA-dw (59-TTGGATTTAGTGTCTGCACG), lpp dw (59- <br /> Takahashi 2001).                                                      TTATCTTGCGGTATTTAGTAG), b2531-up (59-TTTATCG <br />    Plasmids: To clone iscR, DNA fragment amplified           CCGCCCTGGCAGCC) used reverse transcription of <br /> PCR using primers 59-GAGATCCACGCCCTGCTTCTTA                       rnlA, ompA, lpp, iscR, respectively. PCR amplification was <br /> CATTC 59-CGGATCCTGATCAGTACCTTCAAGTTC,                         performed 0.2 mm dNTPs, 2 ml reverse transcription <br /> DNA E. coli W3110 strain template. The amplified            mixture rnlA, 1 ml ompA lpp, 1.5 ml iscR, <br /> fragment digested NruI EcoRI blunted. The            10 pmol sense antisense primers, 0.3 units of <br /> resulting fragment (1.3 kb) containing ORF iscR,      KOD Dash (TOYOBO) 25 ml PCR buffer. A thermal cycle <br /> promoter, Shine-Dalgarno sequence ligated            95&#194;&#176; 30 s, 51&#194;&#176; 15 s, 72&#194;&#176; 30 s repeated 20 <br /> pBluescript II KS1 previously digested EcoRV construct        times rnlA, 16 times lpp, 12 times ompA. A thermal <br /> pBS477NE, iscR oriented opposite direction           cycle 95&#194;&#176; 15 s, 60&#194;&#176; 15 s, 72&#194;&#176; 30 s repeated <br /> as lac promoter. The 531-bp DNA fragment containing           20 times iscR. Primers PCR 59-ATGTTTCTATGG <br /> Shine-Dalgarno sequence iscR ORF obtained              GATCCAGG rnlA-dw N134 rnlA, 59-ATGAAAAAGACAG <br /> digestion pBS477NE EcoRI SmaI cloned             CTATCGC ompA-dw ompA, 59-TTATCTTGCGGTATT <br /> pRKNSE (Nakamura et al. 1999) construct pRKSmE,              TAGTAG lpp-dw lpp, 59-GTCTGCGTAAAAATGGTCTG <br /> iscR oriented direction lac promoter.          b2531-up iscR, respectively. The products each <br /> pRKC92AR, pRKC98AR, pRKC104AR encoded alanine-                    reaction separated 5% polyacrylamide gel. <br /> substituted IscR Cys92, Cys98, Cys104, respectively.              Western blot analysis: Proteins separated 15% <br /> These mutants generated PCR using mutagenic <br /> oligonucleotides pRKSmE template.                            polyacrylamide gel containing SDS electroblotted onto <br />    To construct pQ-ORF2-95 expression His-tagged IscR,         Immuno-Blot PVDF membrane (Bio-Rad). The membranes <br /> a DNA fragment containing iscR PCR amplified                 probed rabbit antibody IscR succes- <br /> MG1655 DNA template primers 59-GGATCCAGA                 sively horseradish peroxidase-conjugated donkey anti-rabbit <br /> CTGACATCTAAAGGGCG 59-GTCGACTATTAAGCGCG                            IgG (GE Biosciences). The blot developed Immobilon <br /> TAACTTAACGTC, digested BamHI SalI, ligated               Western Chemiluminescent HRP Substrate (Millipore). <br /> into pQE30 (Qiagen).                                                     b-Galactosidase assay: Twenty microliters bacterial cul- <br />    A DNA fragment containing N-terminal 39 amino acids            ture mixed 80 ml permeabilization buffer (100 mm <br /> of RnlA 67 bp upstream rnlA amplified PCR          Na2HPO4, 20 mm KCl, 2 mm MgSO4, 0.8 mg/ml CTAB, <br /> W3110 DNA template. The antisense downstream primer              0.4 mg/ml sodium deoxycholate, 5.4 ml/ml b-mercaptoe- <br /> was 59-GGAATTCCTGGCATAGGACCAATTG sense                        thanol) lyse cells, 600 ml substrate solution (60 mm <br /> upstream primer 59-GGAATTCCATACACTTCAG. The                       Na2HPO4, 40 mm NaH2PO4, 1 mg/ml ONPG, 2.7 ml/ml <br /> amplified fragment digested EcoRI ligated           b-mercaptoethanol) added incubated room tem- <br /> pRS552 (Simons et al. 1987) construct rnlA&#226;&#8364;&#8220;lacZ fusion          perature 5&#226;&#8364;&#8220;15 min. After stopping reaction adding <br /> gene, generating pRSrnlA67. pPN&#226;&#8364;&#8220;lacZ containing 200 bp                700 ml 1m Na2CO3, absorbance measured OD420nm. <br /> upstream initiation codon N-terminal 50 amino          The Miller <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> calculated using following formula: <br />                                               Repression rnlA Transcription IscR                                     825 <br />  <br /> 1 Miller <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> &#194;&#188; 1,000A420/(bacterial density A600)(ml        accumulated cells harboring pBS477NE cells <br /> bacterial culture)(min reaction time).                          harboring pBSSK  (Figure 1C). <br />    Purification IscR: MH1 cells harboring pQ-ORF2-95 <br />                                                                       We examined effect overexpressed IscR on <br /> were grown 37&#194;&#176; density 3 3 108 cells/ml 100 ml of <br /> LB containing 50 mg/ml ampicillin. After addition            progeny production measuring number progeny <br /> 0.2 mm IPTG, cells incubated 2 hr harvested           infected cell (burst size). A T4 dmd mutant grew very <br /> by centrifugation. All following procedures per-          poorly MH1 cells, burst size 2.3% that <br /> formed 4&#194;&#176;. Cells suspended 5 ml lysis           wild-type phages (Table 1). The burst size T4 dmd <br /> buffer (50 mm NaH2PO4, 300 mm NaCl, 10 mm imidazole, 10% <br />                                                                    mutant MH1 cells harboring pBS477NE re- <br /> (v/v) glycerol, 1 mm PMSF, pH 8.0) disrupted an <br /> ultrasonic generator, UD-201 (Tomy Seiko). After centrifuga-       producibly higher (21%) MH1 cells. Thus, <br /> tion 30,000 3 g 30 min, cleared lysate loaded       suppression overexpression IscR remarkable, <br /> onto Ni-beads column (Ni-NTA Superflow, Qiagen) accord-          partial, std-1, -3, std-4 mutations <br /> ing manufacturer&#226;&#8364;&#8482;s instruction. The fraction containing     (Otsuka et al. 2003). The burst size wild-type <br /> His-tagged IscR eluted 250 mm imidazole and <br />                                                                    phage MH1 cells harboring pBS477NE reduced <br /> dialyzed TEGX buffer [20 mm Tris Cl (pH 8.0), <br /> 0.5 mm EDTA (pH 8.0), 10% (v/v) glycerol, 0.2% Triton-X             86%, compared parental MH1 cells. <br /> 100] containing 0.1 m NaCl. After sample loaded          Because deprivation intracellular iron known to <br /> an Affi-Gel Blue column (Bio-Rad), column washed           induce expression IscR, measured burst <br /> with TEGX buffer containing 0.3 m NaCl His-tagged IscR         size T4 dmd mutant MH1 cells presence of <br /> was eluted TEGX buffer containing 1 m NaCl. The <br />                                                                    0.25 mm 2, 29-dipyridyl, membrane-permeant ferrous <br /> purified protein precipitated ammonium sulfate, <br /> dialyzed TEGX buffer containing 0.1 m NaCl 50%         iron chelator causes deprivation intracellular <br /> (v/v) glycerol, stored  20&#194;&#176;.                                iron (supporting information, Table S1). Treatment <br />    Gel mobility shift assay: A DNA probe (222 bp) containing       reagent partially restored growth T4 <br /> the rnlA promoters prepared PCR pBRMpNO                dmd mutant, reduced growth ability of <br /> carrying yfjM rnlB template primers 59- <br />                                                                    wild-type T4. This result suggested increase of <br /> [32P]CTGGCATAGGACCAATT 59-GGAATTCCGGAGTG <br /> CCCCCTATTC. A DNA probe (448 bp) containing rnlB               intracellular IscR counteracts RNase LS activity. <br /> promoter prepared PCR pBRMpNO template               Next, investigated effect IscR gene <br /> and primers 59-[32P] CGGATCCCCTGAGACGTTAATTC                   expression T4 dmd mutant. Gene expression at <br /> CGG 59-CCGCATCTCCACGTGACTGC. For binding                  stage T4 infection examined pulse <br /> assay, 4 3 103 cpm gel-purified DNA probe the <br />                                                                    labeling newly synthesized proteins (Figure 1B). Con- <br /> indicated His-tagged IscR added the <br /> binding reaction mixture (15 ml) consisting 20 mm Tris-HCl      sistent previous reports, protein synthesis T4 <br /> (pH 8.0), 50 mm KCl, 4 mm MgCl2, 0.5 mm EDTA, 0.02 mg/ml           dmd mutant late times (23 40 min) impaired <br /> BSA, 5 mm DTT, 10% (v/v) glycerol, 200 ng poly-             MH1 cells carrying pBSSK . In contrast, protein <br /> [dI-dC] (Roche). After incubating room temperature          synthesis MH1 cells harboring pBS477NE infected <br /> 20 min, half reaction mixture loaded 8% <br />                                                                    dmd mutant high MH1 cells <br /> polyacrylamide gel 0.25 3 TBE electrophoresed 4&#194;&#176; <br /> for 2&#226;&#8364;&#8220;2.5 hr 150 V. The gel analyzed Bio-Image           infected wild-type T4. <br /> Analyzer (Fuji BAS-1800).                                             Suppression growth defect T4 dmd mutant <br />                                                                    accompany stabilization late-gene <br />                                                                    mRNAs. To confirm expectation, steady-state <br />                                                                    level T4 late-gene soc mRNA examined by <br />                           RESULTS <br />                                                                    Northern blot, degradation exten- <br />    Overexpression IscR partially restores growth            sively characterized (Kai Yonesaki 2002). As shown <br /> of T4 dmd mutant: To investigate effect iscR           Figure 1D, soc mRNA late stages (25 <br /> the growth T4 dmd mutant, wild-type T4 dmd               35 min) T4 dmd infection MH1 cells a <br /> mutant plated measure efficiency plating             vector hardly detected rapid <br /> (Figure 1A). A T4 dmd mutant grew normally MH1                  degradation. However, T4 dmd mutant infected <br /> DrnlAB cells lacking RNase LS, efficiency               MH1 cells pBS477NE, soc mRNA present as <br /> plating decreased ,10 4 parental MH1 cells                abundantly MH1 cells infected wild-type T4. <br /> MH1 DiscR cells harboring vector plasmid, pBluescript            Our primer extension analysis confirmed loss of <br /> II SK  (pBSSK ). Deletion iscR effect             RNase LS-mediated specific cleavages MH1 cells with <br /> growth wild-type T4. Introduction iscR cloned           pBS477NE (Figure S1). All results clearly <br /> plasmid, pBS477NE, allowed T4 dmd mutants form                  indicated overexpression IscR decreased the <br /> plaques MH1 MH1 DiscR cells efficiency            RNase LS activity allow growth T4 dmd <br /> plating nearly 1, MH1 DrnlAB cells. Because           mutant. <br /> high-copy plasmid-encoded iscR genome-                    Ligation Fe-S cluster IscR required <br /> encoded iscR restored growth T4 dmd mutant,               inhibition RNase LS activity: We examined <br /> this effect caused overproduction IscR.             ligation IscR Fe-S cluster necessary <br /> Indeed, overproduction IscR clearly demon-                  suppression growth defect T4 dmd <br /> strated Western blot; 10-fold IscR                mutant. Cys92, Cys98, Cys104 predicted Fe-S <br />  826                                                       Y. Otsuka et al. <br />  <br />                                                                                                    Figure 1.&#226;&#8364;&#8221;Effect IscR on <br />                                                                                                 RNase LS activity. (A) A solution <br />                                                                                                 containing wild-type T4 dmd <br />                                                                                                 mutant phages serially di- <br />                                                                                                 luted 10-fold steps, 2-ml <br />                                                                                                 samples containing number <br />                                                                                                 phages indicated left <br />                                                                                                 spotted plate seeded <br />                                                                                                 MH1 derivative cells <br />                                                                                                 carrying plasmid shown be- <br />                                                                                                 low figure. Photographs were <br />                                                                                                 taken overnight incubation <br />                                                                                                 30&#194;&#176;. (B) MH1 cells with <br />                                                                                                 pBSSK  pBS477NE in- <br />                                                                                                 fected wild-type T4 a <br />                                                                                                 dmd mutant newly synthe- <br />                                                                                                 sized proteins pulse labeled <br />                                                                                                 3 min times indicated <br />                                                                                                 figure, described <br />                                                                                                 previously (Ueno Yonesaki <br />                                                                                                 2001). Labeled proteins an <br />                                                                                                 equal portion infected cells <br />                                                                                                 separated 12.5% SDS&#226;&#8364;&#8220; <br />                                                                                                 PAGE analyzed Bio-Image <br />                                                                                                 Analyzer (Fuji BAS-1800). The <br />                                                                                                 numbers right margin <br />                                                                                                 representative late genes <br />                                                                                                 encoding proteins. The 23* <br />                                                                                                 indicates processed product en- <br />                                                                                                 coded gene 23. (C) MH1 cells <br />                                                                                                 pBSSK  pBS477NE were <br />                                                                                                 grown 5 3 108 cells/ml LB <br /> containing 50 mg/ml ampicillin total proteins equal portion cells analyzed Western blot antibody <br /> against IscR. (D) MH1 cells pBSSK  pBS477NE grown LB 30&#194;&#176; mid-log phase infected wild-type T4 or <br /> a dmd mutant total RNAs extracted 25 35 min infection. Two micrograms RNA analyzed by <br /> Northern blot 32P-labeled DNA probe soc described previously (Kai Yonesaki 2002). <br />  <br />  <br /> cluster ligands IscR indeed, IscR-C92A                isc operon did alleviate growth defect T4 <br /> alanine substitution did contain Fe-S cluster,           dmd mutant (Table S2). This result suggested that <br /> it purified anaerobic condition (Nesbit et al.             components iscS iscX involved the <br /> 2009). Cells harboring plasmid expressing IscR                inhibition RNase LS activity. <br /> individual alanine substitutions amino acids                   Transcription start sites rnlA rnlB were <br /> were examined supporting growth T4 dmd                  identified primer extension analysis: We tested <br /> mutant (Table 2). pRKSmE expressing wild-type IscR                   IscR directly inhibit RNase LS activity <br /> raised burst size T4 dmd mutant 1.1            vitro using S30 extract MH1 cells, IscR could <br /> plasmid 28.3 plasmid. Each substitutions            inhibit RNA cleavages RNase LS (Figure S2). <br /> (pRKC92AR, pRKC98AR, pRKC104AR) resulted                     Next, investigated possibility IscR <br /> burst size similar wild-type IscR. In addition, soc      regulates RNase LS activity transcriptional control <br /> mRNA infection MH1 cells harboring mutant <br /> plasmids T4 dmd mutant accumulated as <br /> abundantly cells harboring pRKSmE (Figure 2).                                      TABLE 1 <br /> These results demonstrated ligation IscR with <br />                                                                      Effect iscR cloned multi-copy plasmid growth <br /> Fe-S cluster effect IscR-mediated <br />                                                                                          T4 dmd mutant <br /> repression RNase LS activity. <br />    Genes downstream iscR isc operon                                                      Relative burst size <br /> involved inhibition RNase LS activity: iscR part <br />                                                                      Plasmid                 Wild type                dmd mutant <br /> of isc operon negatively regulates own <br /> expression (Schwartz et al. 2001). Repression down-               pBSSK                     100                      2.3 6 0.1 <br /> stream genes operon IscR respon-                 pBS477NE                 86 6 3                    21 6 2 <br /> sible inhibition RNase LS activity. To investigate              The burst size T4 wild-type phage dmd mutant was <br /> this possibility, measured progeny production T4             measured MH1 cells indicated plasmids accord- <br /> dmd mutant MG1655 MG1655 Disc. The deletion                   ing method described materials methods. <br />                                             Repression rnlA Transcription IscR                                         827 <br />  <br />                          TABLE 2 <br />    Effect alanine substitutions IscR growth <br />                     T4 dmd mutant <br />  <br />                                  Relative burst size <br /> Plasmid                 Wild type               dmd mutant <br /> None                       100                    1.0   6   0.5 <br /> pRKSmE                    96 6   5                 27   6   4 <br /> pRKC92A                  110 6   7                 19   6   3 <br /> pRKC98A                   79 6   4                 26   6   3 <br /> pRKC104A                  86 6   5                 24   6   5 <br />   The burst size T4 wild-type phage dmd mutant were <br /> measured MH1 cells plasmid (none) the <br /> indicated plasmids according method described ma- <br /> terials methods. <br />  <br />  <br /> of rnlA and/or rnlB. To identify promoters experimen-               Figure 2.&#226;&#8364;&#8221;Effect Fe-S cluster IscR RNase LS <br /> tally, mapped 59 ends rnlA rnlB transcripts.        activity. MH1 cells pRKNSE vector, pRKSmE, <br /> Unfortunately, detect endogenous rnlA                pRKC92AR, pRKC98AR, pRKC104AR grown LB me- <br />                                                                   dium 30&#194;&#176; mid-log phase infected T4 dmd <br /> and rnlB transcripts primer extension Northern <br />                                                                   mutant, total RNAs extracted indicated times after <br /> blot analysis expression weak.             infection, soc mRNA analyzed Northern blot as <br /> To circumvent problem, constructed                    described previously (Kai Yonesaki 2002). <br /> plasmid derivatives pBR322: pPN-lacZ containing <br /> 200 bp upstream translation start site rnlA and <br /> 150 bp encoding N-terminal 50 amino acids RnlA,               IscR represses transcription rnlA promoter but <br /> and pRSrnlB-long containing 854 bp upstream                rnlB promoter: To investigate IscR <br /> translation start site rnlB 33 bp encoding             regulates transcription rnlA rnlB pro- <br /> N-terminal 11 amino acids RnlB. MH1 cells trans-               moters, examined effect iscR deletion on <br /> formed plasmids incubated               expression b-galactosidase pRSrnlA67, from <br /> until log phase total RNA extracted.                 expression rnlA&#226;&#8364;&#8220;lacZ translational fusion <br /> Primer extension analysis identified 59 termini              gene depended 67 bp sequence upstream rnlA, <br /> specific rnlA transcripts 65, 46, 32, 24 bp                pRSrnlB-long (Figure 4). First, plasmid containing <br /> upstream initiation codon (Figure 3A). In                  promoter lacZ resulted low level b-galac- <br /> addition strong signal site, site          tosidase activity (background value 75). On other <br /> solely matches transcription site promoter             hand, b-galactosidase activity expressed rnlA <br /> predicted GENETYX program (Figure 7A).                     rnlB promoters significantly higher back- <br /> Therefore, rnlA mRNA mainly transcribed                  ground, indicating inserted DNAs provided <br /> from 24 bp upstream rnlA initiation codon. This            promoter activity. Second, deletion iscR led three- <br /> idea confirmed b-galactosidase reporter assays          fourfold increase b-galactosidase activity driven by <br /> the section. On hand,              rnlA promoter cell growth stage. In contrast, <br /> sites correspond unexpected reverse transcrip-           deletion iscR effect activity driven by <br /> tase stop sites RNA processing sites transcripts            rnlB promoter. Third, b-galactosidase activity driven <br /> originating weak plasmid promoter. Two 59                  rnlA promoter increased cell growth both <br /> termini specific rnlB transcripts detected           wild-type iscR-deleted cells, activity driven by <br /> 123 124 bp upstream initiation codon                   rnlB promoter nearly constant. Interestingly, <br /> (Figure 3B), rnlA ORF. Two starts            b-galactosidase activity driven rnlB promoter was <br /> depending single promoter observed              considerably higher driven rnlA pro- <br /> E. coli (for example, refer Otsuka et al. 2003).               moter cell growth stage; activity was <br /> Although promoter corresponding sites                  eightfold the absence iscR. <br /> was predicted GENETYX, program,                    This result indicates transcription activity the <br /> BPROM        (http://linux1.softberry.com/berry.phtml?            rnlA promoter weaker rnlB <br /> topic&#194;&#188;bprom&amp;group&#194;&#188;programs&amp;subgroup&#194;&#188;gfindb),                      promoter, RnlB synthesized more <br /> identified promoter matching sites tran-               vigorously RnlA wild-type cells. Putting all <br /> scription start sites (Figure 7B). Indeed, gene expres-           together, conclude rnlA rnlB promoters <br /> sion driven sequence containing region              regulated independently IscR represses <br /> confirmed section.                                    transcription rnlA promoter (refer Figure 7A). <br />  828                                                      Y. Otsuka et al. <br />  <br />                                                                     50 nm IscR, 200 nm IscR did significantly <br />                                                                     shift rnlB DNA probe. <br />                                                                        IscR represses transcription chromosomal rnlA: <br />                                                                     Finally, confirmed IscR represses transcription <br />                                                                     rnlA chromosome. The endogenous <br />                                                                     rnlA mRNA MH1 cells harboring pBSSK  or <br />                                                                     pBS477NE measured RT&#226;&#8364;&#8220;PCR (Figure 6A). <br />                                                                     Simultaneously, lpp ompA mRNAs measured <br />                                                                     loading controls iscR mRNA measured to <br />                                                                     confirm overexpression. As expected, MH1 cells <br />                                                                     harboring pBS477NE overexpressed iscR mRNA by <br />                                                                     approximately threefold (bottom panel) instead <br />                                                                     underexpressed rnlA mRNA approximately twofold <br />                                                                     (lane 4), compared MH1 cells pBSSK  (lane 3). <br />                                                                     On hand, endogenous rnlA <br />                                                                     mRNA MH1 DiscR cells approximately twofold <br />                                                                     MH1 cells (Figure 6B). Finally, we <br />                                                                     observed MH1 cells oxidative stress over- <br />                                                                     produced iscR mRNA approximately threefold and <br />                                                                     simultaneously underexpressed rnlA mRNA  1.5- <br />                                                                     fold (Figure 6C). <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;Identification rnlA rnlB transcription                               DISCUSSION <br /> start sites. (A) Total RNAs extracted MH1 cells with <br />                                                                        In study, demonstrated overexpression of <br /> pPN&#226;&#8364;&#8220;lacZ grown LB medium 30&#194;&#176; OD600 <br /> reached 0.7. Ten micrograms RNA used primer ex-          IscR repressed RNase LS activity allowed growth <br /> tension analysis described materials methods. (B)         T4 dmd mutant (Figure 1). The b-galactosidase <br /> Total RNA extracted MH1 cells harboring pRSrnlB-           reporter assay Figure 4 strongly suggested IscR <br /> long (rightmost lane) OD600 reached 0.65           represses transcription rnlA promoter. Further- <br /> analyzed using primer extension analysis. A set sequence <br />                                                                     more, His-tagged IscR bound DNA containing the <br /> ladders rnlA rnlB obtained dideoxy sequencing <br /> method primer run parallel fig-        rnlA promoter (Figure 5). Finally, rnlA mRNA was <br /> ure. Arrows indicate bands detected primer extension.            reduced overexpression IscR increased by <br />                                                                     deletion iscR vivo (Figure 6). Taken together, we <br />                                                                     conclude IscR represses transcription rnlA via <br />    IscR bind rnlA promoter: Since                  binding promoter region reduces <br /> results suggested IscR specifically binds rnlA          RNase LS activity. Analysis using IscR mutants contain- <br /> promoter, performed gel mobility shift assays               ing amino acid substitution predicted Fe-S <br /> different concentrations purified His-tagged IscR            ligands demonstrated effect reduction of <br /> 32 <br />   P-labeled DNA probes. The DNA probe rnlA                      RNase LS activity growth T4 dmd mutant <br /> contained region  99 1117, DNA                    (Figure 2 Table 2). Therefore, IscR appears able to <br /> probe rnlB  410 133. As shown Figure 5,              repress rnlA transcription regardless presence of <br /> a shifted rnlA DNA probe apparent presence               Fe-S cluster. <br />  <br />  <br />                                                                                           Figure 4.&#226;&#8364;&#8221;b-Galactosidase reporter <br />                                                                                         assay rnlA rnlB promoter. MH1 <br />                                                                                         MH1 DiscR cells pRSrnlA67 <br />                                                                                         (A) pRSrnlB-long (B) grown <br />                                                                                         LB medium 30&#194;&#176; b-galactosi- <br />                                                                                         dase activities measured de- <br />                                                                                         scribed materials methods, <br />                                                                                         OD600 reached value indi- <br />                                                                                         cated figure. The b-galactosidase <br />                                                                                         activity shown Miller units. Each <br />                                                                                         value given subtracting back- <br />                                                                                         ground value 75, obtained <br />                                                                                         cells harboring pRS552, shows <br />                                                                                         mean 6 SD inde- <br />                                                                                         pendent experiments. <br />                                            Repression rnlA Transcription IscR                                        829 <br />  <br />  <br />  <br />  <br />    Figure 5.&#226;&#8364;&#8221;Binding His-tagged IscR rnlA promoter <br /> region. His-tagged IscR increasing concentrations 0 <br /> to 200 nm incubated room temperature 20 min with <br /> a 32P-labeled DNA probe (4 3 103 cpm) containing rnlA <br /> promoters rnlB promoter. The mixtures separated <br /> through 8% polyacrylamide gel 0.25 3 TBE 4&#194;&#176;. <br /> F C indicate free probe DNA&#226;&#8364;&#8220;protein complex, <br /> respectively. <br />                                                                   Figure 6.&#226;&#8364;&#8221;Negative correlation levels rnlA <br />                                                                 mRNA iscR mRNA vivo. (A) MH1 cells (lane 1), MH1 <br />    There types IscR-binding sites: Type 1,           DrnlAB cells (lane 2), MH1 cells pBSSK  (lane 3) or <br /> consisting 25-bp ATASYYGACTRWWWYAGTCRR                   pBS477NE (lane 4) grown mid-log growth phase LB <br /> STAT (R, purine; Y, pyrimidine; S, G C; W, A T)       medium total RNAs extracted. RT&#226;&#8364;&#8220;PCR analyses for <br />                                                                 rnlA, ompA, lpp, iscR mRNAs performed described <br /> and type 2, consisting 26-bp AWARCCCYTSNG                materials methods. Various amounts pBRMpNO <br /> TTTGMNGKKKTKWA (N, any; K, T G; M, A C)               carrying sequence yfjM rnlB used template to <br /> (Giel et al. 2006). Because IscR regulation promoters        demonstrate semiquantitative profile PCR conditions; <br /> containing type 2 sites appears require ligation     lane 5, 2 pg; lane 6, 4 pg; lane 7, 8 pg; lane 8, 16 pg. <br /> of Fe-S cluster, searched sequence         An asterisk indicates band nonspecific product. Lane 9 <br />                                                                 shows size markers. (B) MH1 MH1 DiscR cells grown <br /> rnlA promoter sequence homologous type 2 sites         mid-log growth phase LB medium total RNAs were <br /> and sequence 65% (17/26 bp) homology            analyzed rnlA iscR mRNAs RT&#226;&#8364;&#8220;PCR described <br /> the type 2 consensus (Figure 7A), partly             above. (C) MH1 cells grown mid-log growth phase <br /> matches &#226;&#8364;&#732;&#226;&#8364;&#732;AxxxCCxxAxxxxxxxTAxGGxxxT&#226;&#8364;&#8482;&#226;&#8364;&#8482; suggested                 M9C medium treated 1 mm H2O2 <br /> for IscR-binding motif (Nesbit et al. 2009). These              10 min. Total RNAs analyzed RT&#226;&#8364;&#8220;PCR rnlA <br />                                                                 iscR mRNAs described above. <br /> features suggest IscR binds sequence for <br /> transcriptional repression. <br />    In b-galactosidase reporter assays,         increases mid late log phase, suggesting its <br /> interesting features regulation rnlA rnlB         dependence cell growth stage. Because increase <br /> expression (Figure 4). First, genes regu-          does require IscR, transcription factors may <br /> lated independently. We identified promoter specific            function regulation. Because transcription pro- <br /> for rnlB ORF rnlA. Because             moted s38 gradually activated starting late log <br /> transcription terminator-like sequence downstream            phase (Gruber Gross 2003), examined the <br /> rnlB terminator upstream rnlB promoter,           effect s38 transcriptional activation rnlA. <br /> rnlB transcribed different modes,        However, activation observed strain <br /> polycistronic mRNA transcribed rnlA pro-               lacking rpoS encoding s38 (K. Miki T. Yonesaki, <br /> moter monocistronic mRNA tran-                 unpublished results). Nonaka et al. (2006) reported <br /> scribed promoter. Second, transcription            rnlA promoter regulated s32, controls <br /> activity rnlB promoter higher            heat-shock response promoter located <br /> from rnlA promoter. High rnlB expression           downstream rnlA promoter. However, cells were <br /> be important cell viability unpublished         cultivated low temperature (30&#194;&#176;) 59 end <br /> data suggest RnlA absence RnlB induces           corresponding s32-regulated promoter not <br /> cell death (M. Koga T. Yonesaki, unpublished                detected primer extension analysis study. In <br /> results). Finally, transcription rnlA promoter             connection, RNase LS activity appears weak at <br />  830                                                               Y. Otsuka et al. <br />  <br />                                                                                   tion using DNA microarrays. Proc. Natl. Acad. Sci. USA 97: 6640&#226;&#8364;&#8220; <br />                                                                                   6645. <br />                                                                              Cannistraro, V. J., D. Kennell, 1991 RNase I*, form of <br />                                                                                   RNase I, mRNA degradation Escherichia coli. J. Bacteriol. <br />                                                                                   173: 4653&#226;&#8364;&#8220;4659. <br />                                                                              Giel, J. L., D. Rodionov, M. Liu, F. R. Blattner P. J. Kiley, <br />                                                                                   2006 IscR-dependent gene expression links iron&#226;&#8364;&#8220;sulphur clus- <br />                                                                                   ter assembly control O2-regulated genes Escherichia <br />                                                                                   coli. Mol. Microbiol. 60: 1058&#226;&#8364;&#8220;1075. <br />                                                                              Gruber, T. M., C. A. Gross, 2003 Multiple sigma subunits and <br />                                                                                   partitioning bacterial transcription space. Annu. Rev. <br />                                                                                   Microbiol. 57: 441&#226;&#8364;&#8220;466. <br />                                                                              Iwamoto, A., S. Lemire T. Yonesaki, 2008 Post-transcriptional <br />    Figure 7.&#226;&#8364;&#8221;Nucleotide sequence rnlA rnlB pro-                        control Crp-cAMP RNase LS Escherichia coli. Mol. Micro- <br /> moter region. (A) The  35  10 sequences correspond-                           biol. 70: 1570&#226;&#8364;&#8220;1578. <br /> ing rnlA promoter underlined. &#226;&#8364;&#732;&#226;&#8364;&#732;IC&#226;&#8364;&#8482;&#226;&#8364;&#8482; asterisks                 Kai, T., T. Yonesaki, 2002 Multiple mechanisms degrada- <br /> show initiation codon transcription start site, re-                   tion bacteriophage T4 soc mRNA. Genetics 160: 5&#226;&#8364;&#8220;12. <br /> spectively. The predicted IscR-binding type 2 site boxed.                 Kai, T., H. E. Selick T. Yonesaki, 1996 Destabilization bacterio- <br /> The numbers sequence nucleotide positions                          phage T4 mRNAs mutation gene 61.5. Genetics 144: 7&#226;&#8364;&#8220;14. <br /> from nucleotide initiation codon. (B) The  35               Kai, T., H. Ueno T. Yonesaki, 1998 Involvement bac- <br /> and  10 sequences corresponding rnlB promoter                           teriophage T4 genes blockade protein synthesis and <br /> underlined.                                                                       mRNA destabilization mutation gene 61.5. Virology <br />                                                                                   248: 148&#226;&#8364;&#8220;155. <br />                                                                              Kushner, S. R., 2002 mRNA decay Escherichia coli comes age. <br />                                                                                   J. Bacteriol. 184: 4658&#226;&#8364;&#8220;4665. <br /> 42&#194;&#176; comparison 30&#194;&#176; (Kai et al. 1996)                   Nakamura, M., K. Saeki Y. Takahashi, 1999 Hyperproduction <br /> the effect s32 transcription rnlA remains                                recombinant ferredoxins escherichia coli coexpression <br /> puzzling.                                                                         ORF1&#226;&#8364;&#8220;ORF2-iscS-iscU-iscA-hscB-hscA-fdx-ORF3 gene clus- <br />                                                                                   ter. J. Biochem. 126: 917&#226;&#8364;&#8220;926. <br />    Because expression IscR induced oxida-                        Nesbit, A. D., J. L. Giel, J. C. Rose P. J. Kiley, 2009 Sequence- <br /> tive stress iron deprivation, RNase LS activity                          specific binding subset IscR-regulated promoters does not <br /> be repressed conditions. At present,                            require IscR Fe&#226;&#8364;&#8220;S cluster ligation. J. Mol. Biol. 387: 28&#226;&#8364;&#8220;41. <br />                                                                              Nonaka, G., M. Blankschien, C. Herman, C. A. Gross V. A. <br /> unknown repression RNase LS activity                                Rhodius, 2006 Regulon promoter analysis E.coli <br /> necessary adaptation cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span>                             heat-shock factor, s32, reveals multifaceted cellular response <br /> stresses. Our previous data showed RNase LS plays                          heat stress. Genes Dev. 20: 1776&#226;&#8364;&#8220;1789. <br />                                                                              Otsuka, Y., T. Yonesaki, 2005 A novel endoribonuclease, <br /> role metabolism mRNAs 23S                                   RNase LS, Escherichia coli. Genetics 169: 13&#226;&#8364;&#8220;20. <br /> rRNA normal growth condition (Otsuka                               Otsuka, Y., H. Ueno T. Yonesaki, 2003 Escherichia coli endori- <br /> Yonesaki 2005; Iwamoto et al. 2008). It                                  bonucleases involved cleavege bacteriophage T4 mRNAs. <br />                                                                                   J. Bacteriol. 185: 983&#226;&#8364;&#8220;990. <br /> important cells adjusting stresses in-                       Otsuka, Y., M. Koga, A. Iwamoto T. Yonesaki, 2007 A role of <br /> activate RNase LS activity stabilize RNase LS target                          RnlA RNase LS activity Escherichia coli. Genes Genet. <br /> mRNAs rRNA.                                                                    Syst. 82: 291&#226;&#8364;&#8220;299. <br />                                                                              Outten, F. W., O. Djaman G. Storz, 2004 A suf operon <br />   We cordially thank John W. Drake National Institute                   requirement Fe-S cluster assembly iron starvation in <br /> Environmental Health Sciences invaluable help manu-                  Escherichia coli. Mol. Microbiol. 52: 861&#226;&#8364;&#8220;872. <br /> script. We thank staff Radioisotope Research Center            Perwez, T., S. R. Kushner, 2006 RNase Z Escherichia coli plays <br /> Toyonaka, Osaka University facilitating research,          significant role mRNA decay. Mol. Microbiol. 60: 723&#226;&#8364;&#8220;737. <br /> our experiments using radioisotopes carried center.          Schwartz, C. J., J. L. Giel, T. Patschkowski, C. Luther, F. J. Ruzicka <br /> This work supported grant program Grants-               et al., 2001 IscR, Fe&#226;&#8364;&#8220;S cluster-containing transcription factor, <br /> in-Aid Scientific Research Ministry Education, Culture,           represses expression Escherichia coli genes encoding Fe&#226;&#8364;&#8220;S cluster <br />                                                                                   assembly proteins. Proc. Natl. Acad. Sci. USA 98: 14895&#226;&#8364;&#8220;14900. <br /> Sports, Science Technology Japan. <br />                                                                              Simons, R. W., F. Houman N. Kleckner, 1987 Improved single <br />                                                                                   multicopy lac-based cloning vectors protein operon <br />                                                                                   fusions. Gene 53: 85&#226;&#8364;&#8220;96. <br />                       LITERATURE CITED                                       T <br /> </body></html>