<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Developmental Biology                                                                                                                BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> An unexpectedly high degree specialization widespread <br /> involvement sterol metabolism C. elegans putative <br /> aminophospholipid translocases <br /> Nicholas N Lyssenko*1,3,4, Yana Miteva1,5, Simon Gilroy2,6, Wendy Hanna- <br /> Rose1 Robert A Schlegel1 <br />  <br /> Address: 1Department Biochemistry Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, 16802, USA, <br /> 2Department Biology, The Pennsylvania State University, University Park, Pennsylvania, 16802, USA, 3The Huck Institutes Life Sciences <br />  <br /> at Pennsylvania State University, University Park, Pennsylvania, 16802, USA, 4Department Cell Biology, NC10, Lerner Research Institute, <br /> 9500 Euclid Avenue, Cleveland Clinic Foundation, Cleveland, Ohio, 44195, USA, 5Department Physiology, University Pennsylvania, <br /> Philadelphia, Pennsylvania, 19104, USA 6Department Botany, University Wisconsin, B117 Birge Hall, Madison, Wisconsin, 53706, USA <br /> Email: Nicholas N Lyssenko* - lyssenn@ccf.org; Yana Miteva - miteva@mail.med.upenn.edu; Simon Gilroy - sgilroy@wisc.edu; Wendy Hanna- <br /> Rose - wxh21@psu.edu; Robert A Schlegel - ur3@psu.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 2 October 2008                                                     Received: 30 July 2008 <br />                                                                               Accepted: 2 October 2008 <br /> BMC Developmental Biology 2008, 8:96   doi:10.1186/1471-213X-8-96 <br /> This article available <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-213X/8/96 <br /> &#194;&#169; 2008 Lyssenko et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />         <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />         Background: P-type ATPases subfamily IV exclusively eukaryotic transmembrane proteins been <br />         proposed directly translocate aminophospholipids phosphatidylserine phosphatidylethanolamine from <br />         exofacial cytofacial monolayer plasma membrane. Eukaryotic genomes contain genes <br />         encoding members subfamily. At present unclear genes kind organism <br />         individual roles genes perform organism development. <br />         Results: We systematically investigated expression developmental function six, tat-1 6, <br />         subfamily IV P-type ATPase genes encoded Caenorhabditis elegans genome. tat-5 ubiquitously- <br />         expressed essential gene group. tat-6 poorly-transcribed recent duplicate tat-5. tat-2 4 exhibit <br />         tissue-specific developmentally-regulated expression patterns. Strong expression tat-2 tat-4 occurs in <br />         intestine certain cells alimentary system. The expressed uterus, during <br />         spermatogenesis fully-formed spermatheca. tat-2 expressed pharyngeal gland cells, the <br />         excretory cells developing vulva. The expression <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span> tat-3 completely <br />         different tat-2 tat-4. tat-3 expression detectable steroidogenic tissues: hypodermis <br />         XXX cells, cells pharynx (except gland), various tissues reproductive <br />         (except uterus spermatheca) seam cells. Deletion tat-1 4 individually interferes little <br />         regular progression organism growth development normal conditions. <br />         However, tat-2 4 essential reproductive growth sterol starvation. <br />         Conclusion: tat-5 likely encodes housekeeping protein performs proposed aminophospholipid <br />         translocase function routinely. Although individually dispensable, tat-1 4 partly <br />         redundant. Expression patterns sterol deprivation hypersensitivity deletion <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> tat-2 4 <br />         suggest genes carry subtle metabolic functions, fine-tuning sterol metabolism digestive <br />         steroidogenic tissues. These findings uncover unexpectedly high degree specialization widespread <br />         involvement sterol metabolism genes encoding putative aminophospholipid translocases. <br />  <br />  <br />  <br />                                                                                                                                        Page 1 17 <br />                                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Background                                                       IV P-type ATPases work concert efficiently <br /> Subfamily IV P-type ATPase superfamily group         sequester aminophospholipids cytofacial <br /> of exclusively eukaryotic large multipass transmembrane          leaflet [17]. When extrapolated multicellular organ- <br /> proteins appear function inward &#226;&#8364;&#8220; exo-       isms, loss aminophospholipid transmem- <br /> facial cytofacial monolayer &#226;&#8364;&#8220; translocases         brane asymmetry PS appearance cell surface <br /> aminophospholipids phosphatidylserine (PS) phos-             fatal [18], yeast paradigm predicts each <br /> phatidylethanolamine (PE) choline lipid phos-         somatic cell express individually nones- <br /> phatidylcholine (PC) [1,2]. PS, PE PC             sential ubiquitous P-type ATPases subfamily IV: <br /> unexpected substrates proteins &quot;clas-      Golgi apparatus plasma mem- <br /> sical&quot; P-type ATPases subfamilies I, II III pump          brane. <br /> metal cations protons [3]. Two lines evidence sug- <br /> gest members subfamily IV translocate amino-             We systematically investigated six, tat-1 through <br /> phospholipids. First, biochemical investigations            6, P-type ATPase subfamily IV genes expressed mul- <br /> determined heterologously expressed purified            ticellular organism Caenorhabditis elegans that <br /> subfamily IV P-type ATPases progress catalytic       expression patterns deletion phenotypes these <br /> cycle hydrolyze ATP presence specifically PS       genes inconsistent predictions yeast <br /> and PE [4,5]. And second, genetic studies consist-          paradigm. This does mean C. elegans P-type <br /> ently deletion genes encoding subfamily IV         ATPases subfamily IV translocate PS PE. In <br /> members increases exposure cell surface endog-         fact, recent report shows loss tat-1 leads the <br /> enous PS PE, normally concealed,            appearance PS surface germline certain <br /> and diminishes translocation cytofacial leaflet        somatic cells [6]. Rather, findings suggest individ- <br /> exogenously-introduced labeled PS PE,              ually nonessential subfamily IV members, tat-1 4, <br /> normally quickly internalized [6-10]. Evidence PC       accomplish bulk aminophospholipid <br /> transported substrate extensive, consisting      internalization group and, instead, are <br /> observations labeled PC internalization Saccharomy-        specialized particular tissues, genes <br /> ces cerevisiae [10]. There skepticism sub-    subtly regulate sterol metabolism by, perhaps, adjusting <br /> family IV P-type ATPases directly translocate          transbilayer lipid distribution. The housekeeping amino- <br /> phospholipids (as opposed pumping sub-             phospholipid translocase encoded tat-5, a <br /> stance concentration gradient drives phospholipid          homolog NEO1. <br /> flip actual translocase) [11] directly <br /> responsible internalization PS, PE PC (as             Results <br /> opposed facilitating vesicular traffic required proper    C. elegans animals express subfamily IV P-type <br /> working actual translocase) [12]. However,        ATPases <br /> an alternative candidate transported substrate         The C. elegans genome encodes predicted members of <br /> ATPases strong evidence altered vesicular traffic     P-type ATPase subfamily IV. The genes named <br /> the cause PS, PE PC exposure cell surface       transbilayer amphipath transporter (tat) 1 6 (Addi- <br /> subfamily IV P-type ATPase mutants far emerged.         tional file 1). The detected splice isoforms tat-1 dif- <br />                                                                  fer respect final exons (Figure 1A). Each <br /> Eukaryotic genomes contain genes encoding P-type            isoform distinct stop codon predicted gen- <br /> ATPases subfamily IV (14 mice humans) [13].            erate product divergent C terminus (Additional <br /> This brings questions: eukaryotes require          file 2). The detected splice isoforms tat-2 differ with <br /> so putative aminophospholipid translocases,            respect penultimate exons are <br /> are individual functions genes,             predicted generate products sequence <br /> should genes divided subgroups order         variability N C termini (Figure 1B). Only <br /> study them? Investigations single-cell fungus S. cer-     slightly different isoforms tat-3 identified <br /> evisiae offer answers. The S. cerevisiae genome             (Figure 1C). The product longer isoform contains a <br /> includes subfamily IV P-type ATPase genes. One           extra C-terminal amino acids, absent the <br /> these, NEO1, essential [14]. The remaining &#226;&#8364;&#8220; DRS2,       shorter version. The tat-4 locus includes open reading <br /> DNF1, DNF2 DNF3 &#226;&#8364;&#8220; individually dispensable               frames (ORFs), tat-4 T24H7.6, appear form <br /> but comprise essential subgroup [15].                operon (Figure 1D). T24H7.6, tat-4, cDNA <br /> Although Drs2p resides predominantly Golgi appa-          amplified using splice leader 2 (SL2) primer. In <br /> ratus [16] Dnf1p Dnf2p reside predominantly           C. elegans, trans-splicing SL2 usually indicates a <br /> the plasma membrane [10], deleted          gene occupies subordinate position operon [19]. <br /> the highest levels PS PE exposure cell surface     The tat-4 stop codon resides exon least <br /> [10,12]. These findings imply number subfamily         weak polyadenylation signals (Additional file 2). <br />  <br />  <br />                                                                                                                       Page 2 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                              http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />                                                                     clones [20] low cDNA amplification yield (data not <br />                                                                     shown), suggests gene recent poorly- <br />                                                                     expressed duplicate strongly expressed tat-5. <br />                                                                     For reasons, tat-6 characterized detail. <br />  <br />                                                                     P-type ATPases subfamily IV customarily divided <br />                                                                     classes [13]. A phylogenetic analysis these <br />                                                                     ATPases expressed S. cerevisiae, C. elegans humans <br />                                                                     reveals substantial evolutionary dissimilarity between <br />                                                                     class 2 classes (Figure 2). After early split <br />                                                                     branch leading class 2 branch lead- <br />                                                                     ing rest classes, class 2 genes dupli- <br />                                                                     cated significantly. Thus, yeast express single gene this <br />                                                                     class (NEO1), C. elegans humans express two <br />                                                                     class 2 genes (tat-5 tat-6 nematode). The <br />                                                                     branch subfamily, contrast, quickly <br />                                                                     undergone multiplication diversification. This differ- <br />                                                                     ence extent evolutionary expansion indicate <br />                                                                     strictly preserved class 2 proteins perform some <br /> Figure <br /> Detected 1 transcripts tat-1 4                           essential function conserved evolution all <br /> Detected transcripts tat-1 4. While tat-1 (A)            eukaryotes, frequently duplicating ATPases in <br /> and tat-2 (B) consist generally shorter exons undergo        classes rapidly evolve new roles. <br /> significant alternative splicing, tat-3 (C) tat-4 (D) include <br /> somewhat longer exons encode essentially ver-          tat-5 housekeeping gene <br /> sion product.                                                Two tat-5 expression cassettes constructed (Figure <br />                                                                     3A). In tat-5b::nls::gfp cassette, region operon <br />                                                                     promoter drives transcription sequence encoding a <br /> Bicistronic tat-4 T24H7.6 messages detected           nuclear localization signal (NLS)-tagged green fluorescent <br /> terminate sole T24H7.6 polyadenilation signal              protein (NLS-GFP). In tat-5a/c::nls::gfp, NLS-GFP under <br /> and likely arise tat-4 polyadenylation sig-          control fragment spanning second tat-5 <br /> nals fail induce poly(A) tail addition. The choice            intron. The cassettes introduced nematode <br /> polyadenylation site affects translation tat-4       genome particle bombardment. Two transgenic lines &#226;&#8364;&#8220; <br /> T24H7.6.                                                            integrated (lcIs481.4) extrachromosomal <br />                                                                     (lcEx481.2) &#226;&#8364;&#8220; carrying tat-5b::nls::gfp transgenes iso- <br /> Three alternative transcripts tat-5 detected. tat-5b        lated (Figure 3A). In lines GFP fluorescence ema- <br /> begins sequence short exons                   nates broadly inspected tissues all <br /> located close proximity upstream ORF                developmental stages, early embryos the <br /> undergoes trans-splicing SL2 (Figure 3A). tat-5a tat-        germline (Figure 3B&#226;&#8364;&#8220;F). One integrated line (lcIs461.3) <br /> 5c identical other, start         carrying tat-5a/c::nls::gfp transgenes identified. GFP <br /> exon located 3 kilobases downstream             signal detected lcIs461.3 embryos her- <br /> two tat-5b exons spliced SL1 SL2. In addi-           maphrodites. These findings suggest operon pro- <br /> tion trans-splicing SL2, subordinate cistrons           moter controls tat-5 transcription. The ubiquitous <br /> operon usually reside close previous ORF [19].          pattern tat-5 expression revealed using reporter is <br /> By criteria, tat-5b subordinate cistron        fully supported situ staining data nematode <br /> operon. The status tat-5a/c certain. The long            expression pattern database (NEXTDB [20]). Thus, tat-5 is <br /> sequence end previous ORF start           ubiquitously expressed gene. <br /> tat-5a/c transcripts conceivably hold (in <br /> addition operon promoter) cis-acting regulatory              N2 (wild-type) animals fed tat-5 double-stranded RNA <br /> element drives transcription isoforms.            (dsRNA) order suppress TAT-5 <span id='am-398' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-399' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br />                                                                     (RNAi) bore dead embryos showing signs extensive <br /> tat-6 73% identical tat-5. A tat-6 deletion mutant          necrosis (Figure 3G). Some eggs uterus tat- <br /> (ok1984, large middle portion protein product              5(RNAi) hermaphrodites disintegrate <br /> removed; C. elegans Gene <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>Knockout</span> Consortium)               (data shown). Similar tat-5(RNAi) phenotypes (her- <br /> is reportedly viable. Comparatively weak expression tat-         maphrodite sterility embryonic lethality) been <br /> 6, evidenced paucity publicly available cDNA          detected systematic RNAi screens genes sup- <br />  <br />  <br />                                                                                                                          Page 3 17 <br />                                                                                                    (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> A <br /> Figure <br />   phylogenetic <br />         2      tree S. cerevisiae, C. elegans human P-type ATPases subfamily IV <br /> A phylogenetic tree S. cerevisiae, C. elegans human P-type ATPases subfamily IV. The tree assembled <br /> using ClustalW2 [48]. The outgroup Drosophila melanogaster calcium-transporting P-type ATPase. <br />  <br />  <br /> pression causes clear morphological developmental           tat-2 reporter clearly detectable 2-fold stage <br /> abnormalities [21,22]. The deletion mutants tat-5      embryos sets pharyngeal cells, developing <br /> (tm1823, tm1772 tm1741) reportedly                 pharyngeal-intestinal valve set cells poste- <br /> homozygous lethal (see National Bioresource Project         rior (Figure 4B). By larval (L1) stage, GFP fluores- <br /> the Nematode, Tokyo, Japan). Being ubiquitously                 cence appears intestine (Figure 4C). L4 and <br /> expressed essential <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span>, tat-5 charac-     adult animals exhibit reporter signals unidentified cells <br /> teristics housekeeping gene.                               pharyngeal procorpus, gland cells located the <br />                                                                 posterior bulb pharynx, pharyngeal-intestinal <br /> tat-2 4 exhibit developmentally-regulated tissue-       valve, rectal gland cells, intestine cells the <br /> specific expression patterns                                    excretory (Figure 4D 4E, data shown). <br /> Two integrated extrachromosomal transgenic            tat-2 reporter signals seen L4 larvae pri- <br /> lines carrying tat-2::nls::gfp expression cassettes   mary vulval lineage vulE vulF cells proxi- <br /> by particle bombardment (Figure 4A). With exception         mal gonad (Figure 4F 4G). The vulval fluorescence <br /> of instances ectopic transcription extrachro-   vanishes moderately strong uterine signal appears <br /> mosomal lines, generally exhibit nearly identical      uterine-vulval connection complete adults <br /> patterns reporter expression. Curiously, GFP fluores-        (data shown). The gonadal signal, emanating from <br /> cence tat-2::nls::gfp transgenic nematodes emanates      spermatids, migrates spermatheca time <br /> not nucleus, expected NLS-         ovulation (Figure 4G 4H). <br /> tagged reporter, plasma membrane <br /> region (Figure 4B&#226;&#8364;&#8220;H). Apparently, short TAT-2-coding        Three integrated extrachromosomal transgenic <br /> fragment retained tat-2 expression cassette      lines carrying tat-3::nls::gfp cassettes derived (Figure <br /> &quot;overpowers&quot; NLS directs chimeric reporter          5A). Four (all integrated extrachro- <br /> peptide plasma membrane compartment.                     mosomal) investigated exhibit <br />                                                                 nearly identical expression patterns. tat-3 reporter signal <br />                                                                 appears embryos developing pharynx (data <br />  <br />  <br />                                                                                                                     Page 4 17 <br />                                                                                               (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-5 a3housekeeping gene <br /> tat-5 housekeeping gene. The 5&apos; end tat-5 locus structure tat-5 expression cassettes (A). Expres- <br /> sion tat-5b::nls::gfp lcIs481.4 line embryos (B C) head region adult (D), lcEx481.2 isolate at <br /> the head-intestine junction (E) developing somatic gonad (F). Necrotic death tat-5(RNAi) embryos (G). Abbrevia- <br /> tions: DTC &#226;&#8364;&#8220; distal tip cell; &#226;&#8364;&#8220; intestinal nucleus; sg &#226;&#8364;&#8220; somatic gonad. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 5 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-2 reporter <br />          4       expression pattern <br /> tat-2 reporter expression pattern. The 5&apos; end tat-2 locus, locations deleted regions tm1634 tm1773 <br /> mutants (bars) structure tat-2 expression cassette (A). Expression tat-2 reporter lcIs982.4.2 line in <br /> embryos (B), intestine L2 larvae (C) gland cells pharynx, pharyngeal-intestinal valve, excretory <br /> gland cell intestine young adult (D). GFP fluorescence lcEx982.2.4.5 line excretory pore excretory <br /> canal cells (E). Reporter signal outlining vulF cells early lcIs982.4.2 L4 larvae (the frame series shows over- <br /> lay pseudocolored UV image bright light image, enlarged re-sampling using Photoshop; dashes extend <br /> along contact surface tissues) (F). tat-2 reporter expression spermatogenesis (G) adult sper- <br /> matheca (H) lcIs982.4.2 animals. Abbreviations: &#226;&#8364;&#8220; excretory gland cell; epc &#226;&#8364;&#8220; excretory pore cell; g &#226;&#8364;&#8220; pharyngeal gland cell; <br /> ga &#226;&#8364;&#8220; gonad arm; ph &#226;&#8364;&#8220; pharynx; pv &#226;&#8364;&#8220; pharyngeal-intestinal valve; oo &#226;&#8364;&#8220; oocyte; r &#226;&#8364;&#8220; rectum; sp &#226;&#8364;&#8220; spermatheca. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 6 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-3  reporter <br />           5      expression pattern <br /> tat-3 reporter expression pattern. The 5&apos; end tat-3 locus, location deleted region tm1275 mutants (bar) <br /> and structure tat-3 expression cassette (A). tat-3 reporter expression pharyngeal-intestinal valve muscle and <br /> marginal, gland, cells pharynx lcIs471.12 line (B). GFP fluorescence muscle buccal epithelial cells <br /> of pharynx procorpus XXX cells lcIs471.4B line (C). tat-3 reporter signals rectum tail region <br /> (D), seam cells hypodermis (E), DTC (F) AC (G) lcIs471.4B animals. GFP staining vulva late <br /> L4 stage lcIs881.3.2 larvae (H). Reporter expression adult lcIs471.4B vulva (I). Cell labels: AC &#226;&#8364;&#8220; anchor cell; DTC &#226;&#8364;&#8220; dis- <br /> tal tip cell; e1 2 &#226;&#8364;&#8220; buccal epithelial cells; g1 &#226;&#8364;&#8220; gland cell; mc &#226;&#8364;&#8220; marginal cell; pm2, 5, 6 7 &#226;&#8364;&#8220; pharyngeal muscle cells; vul &#226;&#8364;&#8220; <br /> vulval cells; utse &#226;&#8364;&#8220; uterine seam cell; XXX &#226;&#8364;&#8220; XXX cells. Abbreviations: hn &#226;&#8364;&#8220; hypodermal nucleus; pv &#226;&#8364;&#8220; pharyngeal-intestinal <br /> valve; sn &#226;&#8364;&#8220; seam cell nucleus. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 7 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                            http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> not shown). In fully formed alimentary system,            germ line apoptotic cells annexin V-GFP, peptide <br /> strong GFP fluorescence observed muscle, mar-            binds specifically PS, altered tat-1(RNAi) her- <br /> ginal buccal epithelial cells pharynx, pha-         maphrodites [23], suggesting RNAi tat-1 did <br /> ryngeal-intestinal valve and, lesser intensity,           suppress target. <br /> rectal epithelial cells (Figure 5B&#226;&#8364;&#8220;D). Seam cells display <br /> very strong fluorescence soon lineage           While RNAi studies conducted, deletion <br /> established embryonic development (Figure 5E).              mutants tat-2 4 available. tat- <br /> In adults, moderate weak fluorescence arise            2(tm1773) (frame shift, splicing acceptor deleted), tat- <br /> from XXX cells, unidentified cells head            3(tm1275) (frame shift) tat-4(tm1801) (a basal pro- <br /> and tail regions hypodermis (Figure 5C 5E).            moter region large portion coding sequence <br /> In reproductive system, tat-3 reporter expression              deleted) likely null (Figure 4A, Figure 5A Fig- <br /> begins distal tip cells (DTC) L1 anchor       ure 6A). tat-2(tm1634) lacks N-terminal exon encoding <br /> cell (AC) early L3 (Figure 5F 5G). GFP signal later      44 amino acids present isoforms, be <br /> visible dividing progeny vulval precursor cells      partly functional (Figure 4A). tat-2(tm1773), tat- <br /> (VPCs). In late L4, anchor cell fuses uterine         2(tm1634), tat-3(tm1275) tat-4(tm1801) single and <br /> seam cell (utse), does express reporter (Fig-        tat-4(tm1801); tat-3(tm1275) double mutants via- <br /> ure 5H). The vulval cells continue exhibiting moderate             ble. <br /> fluorescence adulthood (Figure 5H 5I). <br />                                                                    <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> determine deletion tat-2 4 exerts a <br /> Four integrated extrachromosomal tat-4::nls::gfp           negative effect nematode growth reproduction, <br /> transgenic lines derived (Figure 6A). Five           synchronized mutant N2 larvae followed <br /> lines (2 integrated 3 extrachromosomal) investi-          developmental stages, viable progeny of <br /> gated detail. Notable tat-4 reporter expression begins       adult hermaphrodites counted. Wild-type, tat- <br /> 2&#226;&#8364;&#8220;3 fold embryos developing pharyngeal-intestinal           3(tm1275), tat-4(tm1801) tat-4(tm1801); tat- <br /> valve unidentified cells posterior (Figure 6B). In      3(tm1275) animals essentially indistinguishable <br /> the fully formed alimentary system, GFP fluorescence               timing progression <br /> emanates strongly pharyngeal-intestinal valve,            developmental stages number via- <br /> rectal gland cells intestine (Figure 6C 6D). tat-      ble progeny produced hermaphrodites (Figure 7). tat- <br /> 4 reporter expressed uterus, sperma-         2(tm1634) mutants passed development slightly <br /> togenesis proximal gonad spermatheca             slower un-mutated larvae. This evident the <br /> of previously ovulated adults (Figure 6E&#226;&#8364;&#8220;6G).                      lower higher number tat-2(tm1634) offspring pro- <br />                                                                    duced during, respectively, the sampling <br /> The available NEXTDB [20] situ staining images             period, comparison numbers N2 progeny. <br /> corroborate expression tat-3 pharynx vulva,          Around 20% (n = 10) tat-2(tm1773) hermaphrodites <br /> of tat-4 spermatheca, intestine, pharyngeal-intesti-        notably fewer progeny rest animals the <br /> nal valve uterus tat-2 intestine. tat-1          genotype. This reflected large standard devi- <br /> expression pattern obtained             ation value mutant. However, pair-wise statistical <br /> near perfect inverted repeats located 5&apos; end         analysis (ANOVA) shows total number viable <br /> tat-1 locus destabilized expression cassette vectors (data         hatchlings tat-2(tm1634) tat-2(tm1773) <br /> not shown). However, images NEXTDB                  mutants significantly different correspond- <br /> tat-1 expressed tissue-specifically. Thus, tat-2, tat-3,   ing number N2 animals (P = 0.22 P = 0.30, respec- <br /> tat-4 and, likely, tat-1 appear expressed devel-      tively). Overall, deletion tat-1, tat-2, tat-3 tat-4 <br /> opmentally regulated tissues-specific patterns (Table 1).          individually tat-3 tat-4 does to <br />                                                                    impair nematode development reproduction sig- <br /> tat-1 4 nonessential regular growth              nificant degree regular growth conditions. <br /> conditions <br /> TAT-2 4 expression patterns pro-           tat-2 tat-4 mutant animals hypersensitive sterol <br /> teins present critical tissues key periods        deprivation <br /> the nematode development. To determine cur-                C. elegans sterol heterotroph uptakes various <br /> tailing expression tat-1, tat-2, tat-3 tat-4 lead      exogenous sterols converts compounds 7- <br /> to gross morphological developmental abnormali-                dehydrocholesterol [24-26]. The metabolite pre- <br /> ties, N2 animals fed dsRNA genes.            cursor dafachronic acids, hormone promotes <br /> tat-1(RNAi), tat-2(RNAi), tat-3(RNAi) tat-4(RNAi)              reproductive growth [27]. Sterol uptake conversion to <br /> animals did exhibit notable developmental mor-            7-dehydrocholesterol occurs intestine [28-30], <br /> phological defect (data shown). However, staining           dafachronic acids synthesized primarily the <br />  <br />  <br />                                                                                                                       Page 8 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                            http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-4 reporter <br />          6      expression pattern <br /> tat-4 reporter expression pattern. The 5&apos; end tat-4 locus, location deleted region tm1801 mutants (bar) <br /> and structure tat-4 expression cassette (A). Reporter expression pharyngeal-intestinal valve, intestine rectal <br /> gland cells 3-fold embryos (B) advanced stage larvae (C D) lcIs911.30 line. GFP fluorescence uterus <br /> (E), spermatogenesis (F) spermatheca previously ovulated hermaphrodites (G) lcIs911.30 animals. <br /> Abbreviations: pv &#226;&#8364;&#8220; pharyngeal-intestinal valve; r &#226;&#8364;&#8220; rectum; rgc &#226;&#8364;&#8220; rectal gland cells; sp &#226;&#8364;&#8220; spermatheca. <br />  <br />  <br /> hypodermis [31]. If subfamily IV P-type ATPases TAT-2             The solid support medium routine nematode growth <br /> through 4 facilitated step sterol           contains sterols substances used prepara- <br /> transport pathway &#226;&#8364;&#8220; site exogenous sterol             tion supplemented cholesterol final <br /> uptake site 7-dehydrocholesterol conversion          concentration 5000 ng/ml [32]. The combined amount <br /> dafachronic acids, tat-2 4 mutants             sterol medium sufficient opti- <br /> exhibit sterol deprivation hypersensitivity evident            mal nematode growth: N2 animals grow just well <br /> decreased reproductive growth.                                    plates enriched cholesterol 1000 ng/ml [30]. <br />                                                                   To determine tat-2 4 mutants hyper- <br />  <br />  <br />                                                                                                                        Page 9 17 <br />                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                        http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> Table 1: Expression tat-2, tat-3, tat-4 tat-5 C. elegans <br /> tissues. <br />  <br />  Cells tissues                            tat-2   tat-3   tat-4   tat-5 <br />  <br />  All somatic tissues                                                  +++ <br />  Germline                                       -       -       -      - <br />  <br />  Alimentary system <br />  Pharyngeal gland cells                       +++       -      - <br />  Pharyngeal muscle cells                       -      +++      - <br />  Pharyngeal marginal cells                     -      +++      - <br />  Buccal epithelium                             -      +++      - <br />  Pharyngeal-intestinal valve                  +++     +++     +++ <br />  Intestine                                    +++       -     +++ <br />  Rectal gland cells                           +++       -     +++ <br />  Rectal epithelial cells                       -       ++      - <br />  <br />  Reproductive system <br />  DTCs                                           -     +++      - <br />  AC                                             -     +++      - <br />  Developing vulva: vulE vulF cells   +++     +++      - <br />  Developing vulva: VPC progeny                  -     +++      -              Figure   7 using <br />                                                                               Reproduction <br />                                                                               growth <br />                                                                               performed <br />                                                                               tions) conditions  mutant <br />                                                                                                   (n = 10 <br />                                                                                               ANOVA;   to <br />                                                                                                           error <br />                                                                                                         N2               regular <br />                                                                                                                 bars statistical <br />                                                                                                              12 nematodes standard <br />                                                                                                                 animals;             analysis <br />                                                                                                                                      devia- <br />  Adult vulva                                    -      ++      -              Reproduction mutant N2 nematodes under <br />  Uterus                                        ++       -     +++             regular growth conditions (n = 10 12 animals; statisti- <br />  Spermatogenesis                              +++       -     +++             cal analysis performed using ANOVA; error bars stand- <br />  Adult spermatheca                            +++       -     +++             ard deviations). <br />  Excretory                             +++       -       - <br />  <br />  Epithelial                                                            <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> [29]. The second generation lacked sterol reserves <br />  Seam cells                                     -     +++       -             exhibited notable effects sterol deprivation. <br />  Hypodermis                                     -      ++       - <br />                                                                               Growth N2 animals test plates proportional <br />  Other <br />  Head tail region cells/XXX cells           -     ++        -             cholesterol medium (Figure 8). By <br />                                                                               fifth day, N2 populations 1000 ng/ml 100 ng/ <br />  Expression pattern tat-1 determined method            ml cholesterol plates cleared bacterial food began <br />  choice, tat-6 excluded gene appears          starving. This indicates generation animals <br />  recent poorly expressed duplicate tat-5.                                  produced plenty viable progeny second <br />                                                                               generation grew quickly significant mortality. N2 <br /> sensitive cholesterol deprivation, test plates spe-                   populations 10 ng/ml cholesterol plates cleared food <br /> cially prepared eliminate exogenous sources sterol                  day 7. By time, N2 1 ng/ml cholesterol plates <br /> and supplemented cholesterol final con-                      contained plenty food fewer smaller <br /> centrations 1000 ng/ml, 100 ng/ml, 10 ng/ml 1 ng/                       second-generation animals. <br /> ml. OP50 strain Escherichia coli cultures grown a <br /> synthetic defined medium sterol, supple-                         tat-3(tm1275) populations grew test plates the <br /> mented cholesterol concentrations                     pace wild-type populations. In contrast, tat- <br /> plates. Test plates spotted bacterial culture                  2(tm1634) tat-4(tm1801) populations exhibited a <br /> the cholesterol concentration. The resultant food                        dramatic retardation growth (Figure 8). <br /> lawns test plates identical size. Eggs                  Food tat-2(tm1634) tat-4(tm1801) 1000 ng/ml <br /> from gravid hermaphrodites grown regular full-sterol                       cholesterol plates cleared day 7: days later <br /> plates collected using alkaline hypochlorite                         cholesterol concentration N2 tat- <br /> method, hatched overnight no-sterol plates,                       3(tm1275) plates. Significantly, 100 ng/ml lower <br /> the hatchlings transferred test plates, 30 plate                  cholesterol concentration plates, growth second <br /> (on day one). The generation larvae test plates                  generation tat-2(tm1634) tat-4(tm1801) larvae pro- <br /> grew maturity reproduced animals                         gressed minimally. This evident slight, any, changes <br /> contained reserve sterol deposited oocytes                        size tat-2(tm1634) tat-4(tm1801) popula- <br /> mothers maintained regular high supply                          tions individual animals day 5 7 (Figure 8). <br />                                                                               tat-2(tm1634) tat-4(tm1801) mutants did appear <br />  <br />  <br />                                                                                                                                   Page 10 17 <br />                                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figuredeprivation <br /> Sterol   8           sensitivity mutant N2 nematodes <br /> Sterol deprivation sensitivity mutant N2 nematodes. Notations: arrows indicate edge food lawn on <br /> plates bacterial food; arrow heads point generation animals; stars identify plates began starving. <br /> The dark lines photographs numbers written underside plates. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 11 17 <br />                                                                                                     (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                        http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> to suffer particular developmental morphologi-       sented indicate tat-5 encodes sole <br /> cal defect respond sterol limitation dra-   ubiquitously expressed essential subfamily IV P-type <br /> matically N2 tat-3 animals. Furthermore,          ATPase. tat-5(RNAi) phenotype dramatic: sterility and <br /> developmental delay exhibited tat-2(tm1634) animals         embryonic lethality; furthermore, tissues tat-5(RNAi) <br /> on sterol-limited test plates severe         embryos exhibit extensive necrotic cell death. These obser- <br /> than mild lag development mutants showed          vations suggest TAT-5 performs critical house- <br /> on regular plates. tat-2(tm1773) populations exhib-       keeping function. Together yeast Neo1p [14], <br /> ited severe growth retardation sterol-limited test plates   essential, TAT-5 segregates class 2, the <br /> (data shown). Since unlikely independ-    ancient class P-type ATPases subfamily IV. <br /> ently derived deletion alleles related genes      Thus, class 2 genes encode highly specialized <br /> contain secondary mutations cause identical phe-       subfamily IV P-type ATPases execute critically <br /> notype, mutagenesis specifically tat-2 tat-4      important housekeeping function. <br /> to induce hypersensitivity sterol deprivation. <br />                                                                Individual deletion remaining C. elegans subfamily <br /> tat-4(tm1801); tat-3(tm1275) double mutants           IV P-type ATPase genes, tat-1 4, (excluding tat-6, <br /> sensitive sterol availability tat-4(tm1801) single     recent poorly expressed duplication tat-5) does not <br /> mutants                                                        grossly impair C. elegans growth development. This <br /> The data suggest tat-3 major player     consistent high degree redundancy <br /> sterol metabolism; however, gene expressed       genes. However, analysis expression <br /> hypodermis, major steroidogenic tissue [28],      involvement sterol metabolism exhibited tat-2 <br /> pharynx, potential sterol storage organ [32]. To test        4 suggests different explanation. While tat-4 is <br /> whether tat-3 contributes marginally sterol metabo-         expressed tissues tat-2, tat-3 expression <br /> lism, tat-3(tm1275) mutants crossed tat-             occurs completely different tissues those <br /> 4(tm1801) mutants derive double mutant         expressing tat-2 tat-4 tat-2 (Table 1). <br /> genes. tat-3 tat-4 expression patterns overlap     Furthermore, tat-4; tat-3 double mutants just viable <br /> the pharyngeal-intestinal valve (Table 1), terms     tat-4 tat-3 single mutants wild-type animals. <br /> development reproduction regular conditions,         These observations indicate closely related <br /> tat-4(tm1801); tat-3(tm1275) animals indistinguisha-       (both class 5), tat-3 tat-4 redundant. <br /> ble N2 nematodes (Figure 7). On sterol-deprivation        Although closely related tat-2 (class 1b) tat-4 <br /> test plates, double mutants genes performed         expressed tissues, deletion one <br /> worse tat-4(tm1801) nematodes (Figure 9). tat-            confers hypersensitivity sterol deprivation same <br /> 4(tm1801) populations noticeably larger tat-         extent, suggests partly <br /> 4(tm1801); tat-3(tm1275) populations day 7 1000          redundant. tat-2 gene expressed in <br /> ng/ml cholesterol concentration plates. By day 8        excretory system. Curiously, tat-3 expression <br /> same plates, tat-4(tm1801) animals cleared food,     occurs VPC progeny vulval development and <br /> there plenty bacteria consume dou-    continues adult vulva, tat-2 expression <br /> ble mutants. Growth minimal 100 ng/ml               takes place specifically vulE vulF cells during <br /> lower cholesterol concentration plates tat-           vulval development only. This suggests genes <br /> 4(tm1801); tat-3(tm1275) tat-4(tm1801) animals             perform different roles development vulva. <br /> during 9-day observation period. Thus, dele-      Expression pattern tat-1 (class 1a) be <br /> tion tat-3 exacerbates sterol-deprivation hypersensitiv-    obtained method choice (transcriptional <br /> ity tat-4 mutants.                                          reporters stably introduced genome using particle <br />                                                                bombardment). However, data presented Darland- <br /> Discussion                                                     Ransom et al. [6] indicate gene expressed the <br /> Specialization redundancy C. elegans             gonad body-wall muscle cells &#226;&#8364;&#8220; tissues tat-2 <br /> putative aminophospholipid translocases                        4 expressed. Overall, lack gross <br />                                                                growth development impairment tat-1 4 <br /> Eukaryotic genomes contain genes encoding sub-            mutants does stem high level redun- <br /> family IV P-type ATPases [13]. The need abun-      dancy genes. <br /> dance genes organism individual <br /> functions presently poorly understood. Closely related     While individually dispensable S. cerevisiae subfamily <br /> genes partly redundant          IV P-type ATPase genes (DRS2, DNF1, DNF2 DNF3) <br /> functionally highly specialized. The C. elegans genome         essential group [17], individually dispensable <br /> includes six, tat-1 6, genes protein prod-       C. elegans subfamily IV P-type ATPase genes be <br /> ucts P-type ATPases subfamily IV. The data pre-         dispensable group. tat-1 dispen- <br />  <br />  <br />                                                                                                                   Page 12 17 <br />                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> mutantssensitivity <br /> Higher <br /> Figure  9          sterol limitation tat-4(tm1801); tat-3(tm1275) double mutants comparison tat-4(tm1801) single <br /> Higher sensitivity sterol limitation tat-4(tm1801); tat-3(tm1275) double mutants comparison tat- <br /> 4(tm1801) single mutants. The dark lines photographs numbers written underside plates. <br />  <br />  <br />  <br />  <br />                                                                                                                      Page 13 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                          http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> sable subfamily IV P-type ATPase expressed body-          fine-tune aspects sterol processing sterol- <br /> wall muscle cells [6]. tat-1 null animals lose extra       metabolizing tissues. tat-2 homolog mamma- <br /> muscle cells, does translate notable         lian Atp8b1/ATP8B1, encodes subfamily IV P-type <br /> growth defect [6]. This suggests class    ATPase localizes apical membrane hepato- <br /> 1 5 subfamily IV P-type ATPases nonessential         cytes, enterocytes polarized cells mam- <br /> routine growth development C. elegans cells.              malian digestive [35,36]. Future investigations <br />                                                                  determine TAT-2 localizes api- <br /> Investigations S. cerevisiae suggest individually dis-   cal membrane TAT-4 resides same <br /> pensable subfamily IV P-type ATPases redundant,              membrane TAT-2. tat-3 expressed hypodermis <br /> especially respect internalization PS PE          XXX cells, steroidogenic tissues 7- <br /> [17]. However, does case indi-       dehydrocholesterol converted dafachromic acids <br /> vidually dispensable C. elegans subfamily IV P-type              [28]. Thus, likely product gene medi- <br /> ATPases. Considering C. elegans germline                ates, partly, turnover 7-dehydrocholesterol after <br /> somatic cells normally conceal PS [23] loss tat-     release metabolite intestine. Future <br /> 1 leads PS appearance surface germline          research test hypothesis, determine <br /> certain somatic cells [6], contradiction         membrane compartment TAT-3 resides. <br /> yeast nematode paradigms does imply             Overall, lack notable phenotype tat-1(RNAi) <br /> subfamily IV P-type ATPases organism        tat-2 4 <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>null mutants</span> growth under <br /> translocate aminophospholipids. Rather, considering         normal conditions inability tat-2 tat-4 <br /> class 2 subfamily IV members execute bio-           mutants grow sterol limiting conditions indicate <br /> chemical functions subfamily IV members              critical function dispensable subfamily IV <br /> classes [14] loss asymmetric PS PE distri-       P-type ATPases fine tune metabolism sterol <br /> bution plasma membrane incompatible           and, perhaps, lipids. <br /> survival [33], findings suggest that, nematode, <br /> TAT-5 carries bulk aminophospholipids inter-          Recent reports indicate yeast DRS2 also <br /> nalization, TAT-1 4 adjust cross-bilayer           involved sterol (ergosterol) metabolism [37-39]. <br /> lipid distribution particular tissues.                        DRS2&#206;&#8221; mutants exhibit reduced ability uptake exoge- <br />                                                                  nous sterol. Atp8b1/ATP8B1 mediates sterol (cholesterol) <br /> C. elegans subfamily IV P-type ATPases fine-tune sterol          metabolism mammals [40,41]. Mutants Atp8b1 <br /> metabolism                                                       secrete cholesterol bile [40]. And homozygous <br /> C. elegans sterol heterotroph employs steroid-       individuals compromised ATP8B1 exhibit reduced <br /> mediated developmental program. The nematode takes               levels high-density lipoprotein (HDL)-cholesterol and <br /> up exogenous sterols intestine converts          greatly increased levels oxidized cholesterol [41]. <br /> them 7-dehydrocholesterol [28]. The metabolite         Together previous findings yeast mam- <br /> is used synthesis reproductive development-pro-       mals, involvement tat-2 4 sterol metabolism <br /> moting hormone, dafachronic acids, hypodermis             C. elegans suggests role performed sub- <br /> [27,28]. In unfavorable growth conditions, production         family IV P-type ATPases process conserved <br /> the hormone stops nematode larvae embark              eukaryotes far greater currently recog- <br /> dauer (dispersal) developmental pathway. Complete                nized. <br /> sterol deprivation mimics cessation hormone syn- <br /> thesis favorable growth environment [34].        Conclusion <br /> At sterol supplies optimal, nematode development           The presented study shows individually dispensable <br /> and reproduction proportional sterol concentration        C. elegans putative aminophospholipid translocase genes <br /> in medium [30]. As result, C. elegans reproductive         exhibit high degree specialization. This finding con- <br /> growth sensitive indicator sterol level       tradicts expectation derived investigations in <br /> hypodermis lipid taken intestine        yeast genes broadly redundant and <br /> and traveled animal body.                       answers question genes of <br />                                                                  kind organism. Our study partly answers the <br /> Our data tat-2 tat-4 required growth       question individual roles performed putative ami- <br /> during sterol starvation deletion tat-3 exacer-      nophospholipid translocases uncovering widespread <br /> bates sterol hypersensitivity tat-4 mutants. From         involvement members group sterol metabo- <br /> late stages embryonic development rest         lism. <br /> the nematode life cycle, tat-2 tat-4 strongly <br /> expressed intestine cells alimentary <br /> system; likely protein products genes <br />  <br />  <br />                                                                                                                     Page 14 17 <br />                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Methods                                                         Chromosomal fragments contain promoter regions <br /> C. elegans culture genetics                                 tat-2 5 cloned YAC Y17G9 and <br /> The following strains mutants C. elegans used:      cosmids T24H7, W09D10 F36H2 pPD95.69, a <br /> var. Bristol strain N2 (wild type), unc-119(-), tat-            promoter-less NLS-GFP expression vector Fire <br /> 2(tm1634) IV, tat-2(1773) IV, tat-3(tm1275) III, tat-           vector kit, using standard cloning methods (see Addi- <br /> 4(tm1801) II tat-4(tm1801) II; tat-3(tm1275) III. The       tional file 4). DNA expression vector co-precip- <br /> 4 tat mutants isolated var. Bristol strain N2       itated DNA vector carrying unc-119(+) <br /> background National Bioresource Project          (pDP#MM016B [44]) gold particles described by <br /> Experimental Animal Nematode [42]. The mutants             Tucker et al. [45], coated particles bom- <br /> out-crossed 2 8 times N2 animals. Except            barded gonad unc-119(-) nematodes as <br /> noted, C. elegans animals maintained standard           described Praitis et al. [46] modifications <br /> nematode growth medium plates spotted E. coli              (see Additional file 4). After bombardment, nematodes <br /> OP50 20&#194;&#176;C [32].                                              washed plate transferred 2 3 fresh <br />                                                                 plates. Following incubation period, stably transgenic <br /> Gene transcription analysis                                     lines (not exhibiting Unc phenotype) selected <br /> Mixed-stage N2 animals grown enriched peptone           plates: hermaphrodite plate <br /> plates. Populations approaching starvation col-            non-Unc animals. Integrated lines identified by <br /> lected, combined, re-suspended Trizol&#194;&#174; (Invitrogen,          animals homozygous trangene be <br /> Carlsbad, CA), freeze-thawed times treated         derived. <br /> 20 strokes A pestle 40 ml Dounce homogenizer. <br /> Total RNA isolated recommended Tri-          RNA interference <br /> zol manufacturer. mRNA purified total RNA          500 900 bp-long 5&apos; fragments tat-1 5 cDNA <br /> with Poly(A)Pure kit (Ambion, Austin, TX). cDNA           clones sub-cloned pPD129.36 (inverted-T7-pro- <br /> synthesized using dT(18) adaptor-dT(18)       moter vector Fire vector kit). dsRNA production <br /> primers (see Additional file 4). For 5&apos; RACE, splice          nematode feeding conducted described in <br /> leader-specific primer gene-specific primer (GSP)         Kamath et al. [47] (see Additional file 4). <br /> were employed; 3&apos; RACE, GSP adaptor- <br /> dT(18) primer used. Products cDNA amplification         Growth reproduction regular conditions <br /> were resolved aragose gels, purified cloned       Gravid N2 mutant hermaphrodites allowed to <br /> TA helper vector. A number clones distinct          lay eggs fresh plates 2 h. The animals then <br /> band agarose gels sequenced (see Additional         removed, eggs left hatch overnight. The next <br /> file 4). The following ESTs kindly provided Dr. Kohara       day, 10&#226;&#8364;&#8220;12 larvae genotype transferred indi- <br /> were sequenced: yk1228h06, yk34c11, yk1496c06,                  vidual plates: larva plate. 94 118 h later, the <br /> yk209d5, yk36b8, yk214f10 yk126b10.                         adult nematodes transferred new plates. The <br />                                                                 old plates incubated overnight permit viable <br /> Gene expression analysis                                        eggs hatch, chilled ice halt nematode movement, <br /> Nucleus-targeted transcriptional reporters used         larvae counted. 0&#226;&#8364;&#8220;94 h, 94&#226;&#8364;&#8220;118 h and <br /> visualize tat-2 5 expression. This approach         118&#226;&#8364;&#8220;154 h sampling periods one, three, <br /> chosen tats long genes; tag desta-     respectively. <br /> bilizes P-type ATPases subfamily IV ([14] Lyssenko, <br /> unpublished observations); yeast mammals, mem-           Sterol deprivation assay <br /> bers subfamily exit Golgi apparatus        A mixture agarose (8 g/l) ether-extracted peptone <br /> heterodimer proteins Cdc50p/Lam3p group             water autoclaved 30 min, cooled 65&#194;&#176;C a <br /> [7,12], over-expression tats lead accumula-       water bath enriched desired concentrations <br /> tion un-paired peptide Golgi apparatus            5 mg/ml cholesterol ethanol. An OP50 strain E. <br /> potentially induction unfolded protein response,         coli culture grown minimal medium (20 mM <br /> which trigger apoptosis reporter-             NH4Cl, 0.2% w/v D-glucose, 2 mg/ml uracil M9 buffer) <br /> expressing cells; finally, concentrating reporters   overnight 37&#194;&#176;C. The culture divided 4 aliq- <br /> nucleus facilitates cell identification [43]. Note    uots, centrifuged remove supernatant, <br /> when long portions (long <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> 65             re-suspended volume peptone solution <br /> amino acids) tat-2 tat-4 translated sequences       (0.5% w/v peptone M9 buffer) enriched cho- <br /> included expression cassettes, cassettes did     lesterol 4 final concentrations plates. <br /> assemble extrachromosomal arrays did inte-         The suspensions pipetted carefully cor- <br /> grate genome (Additional file 3).                      responding concentration plates produce uniform <br />                                                                 spots. Nematodes synchronized L1 hatching <br />  <br />  <br />                                                                                                                   Page 15 17 <br />                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> eggs no-sterol plates. L1 larvae transferred                 4.    Ding J, Wu Z, Crider BP, Ma Y, Li X, Slaughter C, Gong L, Xie XS: <br />                                                                                    Identification functional expression isoforms of <br /> spotted plates, exactly 30 animals plate. 3 replica                            ATPase II, putative aminophospholipid translocase. <br /> plates seeded cholesterol concentration gen-                          Effect isoform variation ATPase activity phos- <br /> otype. The populations allowed grow 20&#194;&#176;C                            pholipid specificity. J Biol Chem 2000, 275:23378-23386. <br />                                                                              5.    Paterson JK, Renkema K, Burden L, Halleck MS, Schlegel RA, William- <br /> 1.5&#226;&#8364;&#8220;2 weeks periodic observations.                                            son P, Daleke DL: Lipid specific activation murine P4- <br />                                                                                    ATPase Atp8a1 (ATPase II). Biochemistry 2006, 45:5367-5376. <br />                                                                              6.    Darland-Ransom M, Wang X, Sun CL, Mapes J, Gengyo-Ando K, <br /> Authors&apos; contributions                                                             Mitani S, Xue D: Role C. elegans TAT-1 protein maintain- <br /> NNL conceived project, designed experiments, per-                              ing plasma membrane phosphatidylserine asymmetry. Sci- <br /> formed experiments, analyzed data wrote manu-                              ence 2008, 320:528-531. <br />                                                                              7.    Paulusma CC, Folmer DE, Ho-Mok KS, Waart DR, Hilarius PM, <br /> script. YN performed experiments analyzed data. SG,                            Verhoeven AJ, Oude Elferink RP: ATP8B1 requires accessory <br /> WH RAS contributed experimental design, ana-                                protein endoplasmic reticulum exit plasma mem- <br /> lyzed data edited manuscript. All authors read                         brane lipid flippase activity. Hepatology 2008, 47(1):268-78. <br />                                                                              8.    Iwamoto K, Kobayashi S, Fukuda R, Umeda M, Kobayashi T, Ohta A: <br /> approved final manuscript.                                                     Local exposure phosphatidylethanolamine yeast <br />                                                                                    plasma membrane implicated cell polarity. Genes Cells <br />                                                                                    2004, 9:891-903. <br /> Additional material                                                          9.    Wang L, Beserra C, Garbers DL: A novel aminophospholipid <br />                                                                                    transporter exclusively expressed spermatozoa is <br />                                                                                    required membrane lipid asymmetry normal fertili- <br />      Additional file 1                                                             zation. Dev Biol 2004, 267:203-215. <br />      WormBase IDs given names C. elegans ORFs encoding P-     10.   Pomorski T, Lombardi R, Riezman H, Devaux PF, van Meer G, <br />                                                                                    Holthuis JC: Drs2p-related P-type ATPases Dnf1p Dnf2p <br />      type ATPases subfamily IV. <br />                                                                                    required phospholipid translocation yeast <br />      Click file                                                           plasma membrane serve role endocytosis. Mol Biol <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Cell 2003, 14:1240-1254. <br />      213X-8-96-S1.doc]                                                       11.   Kuhlbrandt W: Biology, structure mechanism P-type <br />                                                                                    ATPases. Nat Rev Mol Cell Biol 2004, 5:282-295. <br />                                                                              12.   Chen S, Wang J, Muthusamy BP, Liu K, Zare S, Andersen RJ, Graham <br />      Additional file 2                                                             TR: Roles Drs2p-Cdc50p complex protein trans- <br />      Peculiarities tat-1 tat-4 transcription.                               port phosphatidylserine asymmetry yeast plasma <br />      Click file                                                           membrane. Traffic 2006, 7:1503-1417. <br />      [http://www.biomedcentral.com/content/supplementary/1471-               13.   Halleck MS, Schlegel RA, Williamson PL: Reanalysis ATP11B, a <br />      213X-8-96-S2.doc]                                                             type IV P-type ATPase. J Biol Chem 2002, 277:9736-9740. <br />                                                                              14.   Wicky S, Schwarz H, Singer-Kr&#195;&#188;ger B: Molecular interactions of <br />                                                                                    yeast Neo1p, essential member Drs2 family ami- <br />      Additional file 3                                                             nophospholipid translocases, role membrane traf- <br />      Fusions GFP long N-terminal portions TAT-2 TAT-4         ficking endomembrane system. Mol Cell Biol 2004, <br />      expressed apparent toxicity.                                  24:7402-7418. <br />                                                                              15.   Hua Z, Fatheddin P, Graham TR: An essential subfamily Drs2p- <br />      Click file                                                           related P-type ATPases required protein trafficking <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Golgi complex endosomal/vacuolar sys- <br />      213X-8-96-S3.doc]                                                             tem. Mol Biol Cell 2002, 13:3162-3177. <br />                                                                              16.   Natarajan P, Wang J, Hua Z, Graham TR: Drs2p-coupled amino- <br />                                                                                    phospholipid translocase activity yeast Golgi membranes <br />      Additional file 4                                                             relationship vivo function. Proc Natl Acad Sci USA 2004, <br />      Supplementary materials methods.                                          101:10614-10619. <br />      Click file                                                     17.   Graham TR: Flippases vesicle-mediated protein transport. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Trends Cell Biol 2004, 14:670-677. <br />      213X-8-96-S4.doc]                                                       18.   Schlegel RA, Williamson P: Phosphatidylserine, death knell. <br />                                                                                    Cell Death Differ 2001, 8:551-563. <br />                                                                              19.   Blumenthal T: Trans-splicing polycistronic transcription in <br />                                                                                    Caenorhabditis elegans operons. Trends Genet 1995, 11:132-136. <br />                                                                              20.   Shin-i T, Kohara Y: NEXTDB: expression pattern map <br />                                                                                    database C. elegans. Genome Informatics 1999, 10:213-214. <br /> Acknowledgements                                                             21.   Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann <br /> We like express gratitude Dr. Graham Thomas sharing            M, Ahringer J: Functional genomic analysis C. elegans chro- <br /> equipment, Dr. Peggy Halleck providing technical expertise Katie           mosome I systemic RNA interference. Nature 2000, <br />                                                                                    408:325-330. <br /> Estes helping identify gene expression patterns. NNL supported <br />                                                                              22.   Maeda I, Kohara Y, Tamamoto M, Sugimoto A: Large-scale analysis <br /> by consecutive pre-doctoral fellowships American Heart                gene function Caenorhabditis elegans high-through- <br /> Association.                                                                       RNAi. Curr Biol 2001, 11:171-176. <br />                                                                              23.   Z&#195;&#188;llig S, Neukomm LJ, Jovanovic M, Charette SJ, Lyssenko NN, Hal- <br />                                                                                    leck MS, Reutelingsperger CP, Schlegel RA, Hengartner MO: Amino- <br /> References                                                                         phospholipid            translocase       TAT-1         promotes <br /> 1.      Lenoir G, Williamson P, Holthuis JCM: On origin lipid               phosphatidylserine exposure C. elegans apoptosis. <br />         asymmetry: flip ion transport. Curr Opin Chem Biol             Curr Biol 2007, 17:994-999. <br />         2007, 11(6):654-61.                                                  24.   Chitwood DJ: Biochemistry function nematode ster- <br /> 2.      Tang X, Halleck MS, Schlegel RA, Williamson P: A subfamily P-           oids. Crit Rev Biochem Mol Biol 1999, 34:273-284. <br />         type ATPases aminophospholipid transporting activity.           25.   Kurzchalia TV, Ward S: Why worms need cholesterol? Nat <br />         Science 1996, 272:1495-1497.                                               Cell Biol 2003, 5:684-688. <br /> 3.      Axelsen KB, Palmgren MG: Evolution substrate specificities of <br />         P-type ATPase superfamily. J Mol Evol 1998, 46:84-101. <br />  <br />  <br />  <br />                                                                                                                                       Page 16 17 <br />                                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> 26.   Merris M, Kraeft J, Tint GS, Lenard J: Long-term effects sterol    44.   Maduro M, Pilgrim D: Identification cloning unc-119, a <br />       depletion C. elegans: sterol content synchronized wild-               gene expressed Caenorhabditis elegans nervous sys- <br />       type mutant populations. <br /> </body></html>