<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Cell Biochem. 2008 August 1; 104(5): 1843&#226;&#8364;&#8220;1852. doi:10.1002/jcb.21752. <br />  <br />  <br />  <br />                            Characterization Functional Studies FYVE Domain- <br />                            Containing Phosphatase C. elegans <br />  <br />                            Junfeng Ma1,2,#, Fenghua Zeng3,#, Wanting Tina Ho1, Lirong Teng2, Qingshan Li2, Xueqi <br />                            Fu2,*, Zhizhuang Joe Zhao1,2,3,* <br />                            1 Department Pathology, University Oklahoma Health Sciences Center, Oklahoma City, Oklahoma <br />                            73104 <br />                            2 Edmond H. Fischer Signal Transduction Laboratory, College Life Sciences, Jilin University, Changchun, <br />                            130023, China <br />                            3 Departments Medicine, Vanderbilt University School Medicine, Nashville, Tennessee 37232 <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 The myotubularin (MTM) enzymes phosphotidylinositol-3-phosphate phosphatidylinositol <br />                                 3,5-bisphosphate phosphatases. <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>Mutation</span> MTM1, founder member family, is <br />                                 responsible X-linked myotubular myopathy humans. Here, isolated characterized <br />                                 Caenorhabditis elegans homology enzymes designated ceMTM3. ceMTM3 preferably <br />                                 dephosphorylates phosphotidylinositol-3-phosphate contains FYVE lipid-binding domain at <br />                                 C-terminus binds phosphotidylinositol-3-phosphate. Immunoblotting analyses revealed <br />                                 enzyme expressed early development adulthood animal. <br />                                 Immunofluorescent staining revealed predominant expression enzyme eggs muscles. <br />                                 Knockdown enzyme using feeding-based <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>RNA interference</span> resulted increased level <br />                                 phosphotidylinositol-3-phosphate caused severe impairment body movement worms <br />                                 post-reproductive ages significantly shortened lifespan. This study reveals an <br />                                 important role MTM phosphatases maintaining muscle function, clinical <br />                                 implications prevention treatment sarcopenia. <br />  <br />  <br />                            Keywords <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Tyrosine phosphatase; myotubularin; phosphoinositide; aging; locomotion <br />  <br />  <br />                            Introduction <br />                                              The tyrosine phosphatase super-family consists highly diverse members enzymes <br />                                              characterized presence conserved (H/V)C(X)5R(S/T) signature motif [Alonso et al, <br />                                              2004; Tonks, 2006]. The classic tyrosine phosphatases, known PTP1B-like enzymes, <br />                                              tyrosine-specific, dual specificity phosphatases (DSP) act both <br />                                              serine/threonine tyrosine. Interestingly, dual specificity enzymes preferably <br />                                              dephosphorylate phosphoinositides. Tumor suppressor PTEN well-known example which <br />                                              dephosphorylates phosphatidylinositol 3, 4-bisphosphate (PI3,4P2) phosphatidylinositol <br />  <br />  <br />                            *Correspondence reprint request addressed <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>: Dr. Xueqi Fu, Edmond H. Fischer Signal Transduction Laboratory, College <br />                            Life Sciences, Jilin University, Changchun, 130023, China, phone 86-43188499782, email fxq@jlu.edu.cn Dr. Zhizhuang Joe Zhao, <br />                            Department Pathology, University Oklahoma Health Sciences Center, Oklahoma City, Oklahoma 73104, phone 405-271-9344, fax <br />                            405-271-9483, email joe-zhao@ouhsc.edu, phone (405)271-9344. <br />                            #These authors contribute equally work. <br />                             Ma et al.                                                                                             Page 2 <br />  <br />  <br />                                          3,4,5-trisphosphate (PI3,4,5P3) [Maehama Dixon, 1998; Myers et al, 1998]. MTMs stand <br />                                          subfamily dual specificity phosphatases act phosphoinositide <br />                                          substrates [Clague Lorenzo, 2005; Robinson Dixon, 2006]. Unlike PTEN, MTMs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          specifically dephosphorylate phosphatidylinositol 3-phosphate (PI3P) and <br />                                          phosphatidylinositol 3,5-bisphosphate (PI3,5P2). Myotubularin (hMTM1) founding <br />                                          member MTM subfamily enzymes. It isolated positional cloning Xq28 <br />                                          region human chromosomes [Laporte et al, 1996]. Over 81 mutations in <br />                                          unrelated patients X-linked recessive myotubular myopathy, characterized by <br />                                          severe hypotonia generalized muscle weakness impaired maturation muscle fibers. <br />                                          Loss MTM1 function profound impact organization muscle cells during <br />                                          myogenesis. Two members MTM subfamily, namely, hMTMR2 hMTMR13, <br />                                          mutated recessive forms Charcot-Marie-Tooth neuropathy type 4B [Bolino <br />                                          et al, 2000; Senderek et al, 2003]. How absence MTM enzymes causes diseases is <br />                                          understood. The MTM enzymes diverse structural features, 14 family <br />                                          members identified human genome [Robinson Dixon, 2006]. Some of <br />                                          contain FYVE domains [Zhao et al, 2000; Zhao et al, 2001; Walker et al, 2001]. The <br />                                          FYVE domain (also known FYVE finger) named proteins (Fab1, <br />                                          YOTB, Vac1p, EEA1) shown contain domain [Stenmark et al, 1996]. <br />                                          These domains specifically bind PI3P regulating membrane trafficking signal <br />                                          transduction [Stenmark Aasland, 1999]. Binding FYVE domains PI3P presumably <br />                                          targets FYVE domain-containing MTMs PI3P-rich cell compartments, thereby <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          dephosphorylating PI3P PI3,5P2 efficiently. PI3P known play important role <br />                                          membrane trafficking [Vanhaesebroeck, 2001]. <br />  <br />                               Materials Methods <br />                               Molecular cloning ceMTM3 <br />                                          By searching WormBase database cDNA sequences human FYVE-DSPs [Zhao <br />                                          et al, 2000; Zhao et al, 2001], <span id='am-239' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> C. elegans genome with <br />                                          designation T24A11.1 T24A11.1b. To verify cDNA sequence C. elegans <br />                                          enzyme, employed rapid amplification cDNA ends (RACE) strategy using the <br />                                          Smart-RACE kit Clontech following manufacture&#226;&#8364;&#8482;s directions. First, RACE cDNA <br />                                          library synthesized total RNA N2 C. elegans. The cDNA library then <br />                                          amplified PCR adaptor primers provided kit gene specific primers with <br />                                          sequences 5&#226;&#8364;&#178; CCCATCCATCAGAACAATGAACCA 5&#226;&#8364;&#178; <br />                                          CAACTGGATGGCTTCTCAATCTGAG 3&#226;&#8364;&#178;. The PCR products cloned pCRII <br />                                          vector, multiple clones selected sequencing analysis. Based sequence the <br />                                          RACE PCR products, primers corresponding 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; ends open reading frame <br />                                          synthesized used amplify complete coding regions cDNA were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          cloned pBluescript KS vector. Sequencing analyses revealed complete cDNAs <br />                                          designated ceMTM3a ceMTM3b, <br />  <br />                               Generation anti-ceMTM3 antibody <br />                                          A GST fusion protein (GST-ceMTM3CT) containing C-terminal segment (aa 701 961, <br />                                          FYVE domain-included) ceMTM3b purified E. coli expression with <br />                                          pGex-2T vector. The purified protein used immunize rabbit generate anti-serum. <br />                                          The antibody purified negative selection GST-agarose positive <br />                                          selection GST-ceMTM3CT-agarose. <br />  <br />                               Knockdown ceMTM3 expression feeding-based RNAi <br />                                          The full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> coding sequence ceMTM3b (&#226;&#710;&#8217;6 2910 translation starting codon ATG <br />                                          starting 1) fragments F1 (27&#226;&#8364;&#8220; 876) F2 (877&#226;&#8364;&#8220;2910) cloned the <br />  <br />                                             J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                Page 3 <br />  <br />  <br />                                           pPD129.36 vector (provided Dr. Andrew Fire, Stanford University). The vector bi- <br />                                           directional promoter configuration consisting T7 promoters flanking single copy a <br />                                           cDNA insert facilitate synthesis sense anti-sense RNAs [Timmons et al, 2001]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Plain p129.36 vector used control study. The HT115(DE3) E. coli cells <br />                                           employed host expression double strand RNAs, induction feeding <br />                                           performed following standard procedures [Timmons et al, 2001]. The Bristol N2 <br />                                           strain C. elegans hermaphrodite used study, worms cultured on <br />                                           NGM plates 20 &#194;&#176;C according standard protocols. <br />  <br />                               Western blotting analyses phosphatase activity assays <br />                                           C. elegans worms mixed population different stages collected extracted by <br />                                           sonication cell extraction buffer containing 25 mM 2-glycerol phosphatase (pH 7.3), <br />                                           5 mM EDTA, 5 mM EGTA, 0.1 M NaCl, 1% Triton X-100, 10mM &#206;&#178; &#226;&#8364;&#8220;mercaptolethanol, and <br />                                           cocktail protease inhibitors (Roche Applied Science). Samples containing 15 &#226;&#8364;&#8220; 20 &#206;&#188;g of <br />                                           total proteins separated SDS gel transferred polyvinylidene difluoride (PVDF) <br />                                           membrane Western blotting analysis using enhanced chemiluminescence method. For <br />                                           phosphatase activity assays, extracts containing 200&#226;&#8364;&#8220;300 &#206;&#188;g total proteins subjected <br />                                           immunoprecipitation anti-ceMTM3. Activity assays performed the <br />                                           immunoprecipitates presence 50 &#206;&#188;M PI3P lipid vesicles pH 6.0 according to <br />                                           procedure described Taylor et al. [2000]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Lipid-membrane overlay assays <br />                                           Nitrocellulose membranes spotted various lipid compounds incubated 0.5 &#206;&#188;g/ <br />                                           ml GST GST-ceMTM3CT buffer containing 10 mM Tris-HCl (pH 8.0), 0.15 M NaCl, <br />                                           0.1% Tween-20, 3% fatty acid-free bovine serum albumin [Zhao et al, 2003]. After <br />                                           extensive washing buffer, proteins bound membranes probed anti- <br />                                           GST antibodies horseradish peroxidase (HRP)-conjugated anti-mouse secondary <br />                                           antibodies. Detection enhanced chemoluminescence reactions. To access the <br />                                           contents PI3P control RNAi-treated worms, total phospholipids extracted <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           worms methanol/chloroform (2:1, v/v) containing 0.25% concentrated HCl and <br />                                           quantified using Malachite green phosphate method following ashing lipids with <br />                                           sulfuric acid perchloric acid. Equivalent amounts crude phospholipids then <br />                                           resolved layer chromatography silica plates [Sui et al, 2000], sections <br />                                           corresponding standard PI3P (1,2-Dioleoyl-sn-Glycero-3-Phosphoinositol Avanti <br />                                           Polar Lipids, Alabaster, AL) scraped extracted aforementioned chloroform/ <br />                                           methanol/HCl solution. The partially purified lipids spotted nitrocellulose membranes <br />                                           standard PI3P incubated GST-ceMTM3CT described above. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Immunofluorescent staining <br />                                           Whole mount worms fixed glass slides cracking freezing following standard <br />                                           procedures, animals treated methanol 5 min acetone 5 min <br />                                           &#226;&#710;&#8217;20&#194;&#176;C. For paraffin embedding, worms fixed 3.7% formaldehyde 4 hr 4&#194;&#176;C <br />                                           melted soft agar. The solidified agar pieces processed paraffin <br />                                           embedding machine. Three-micron slices cut fixed glass slides. <br />                                           Immunofluorsecent staining performed primary antibodies Cy3-conjugated <br />                                           donkey anti-rabbit FITC-conjugated goat anti-mouse secondary antibodies. <br />                                           Immunofluorescent micrographs obtained using Olympus BX51 microscope <br />                                           equipped DP71 digital camera. <br />  <br />  <br />  <br />  <br />                                              J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                 Page 4 <br />  <br />  <br />                               Results <br />                               Molecular cloning ceMTM3 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           We previously isolated FYVE domain-containing members MTM family <br />                                           enzymes designated FYVE-DSP1 FYVE-DSP2 [Zhao et al, 2000; Zhao et al, 2001], also <br />                                           known MTMR3 MTMR4 [Walker et al, 2001; Robinson Dixon, 2006], respectively. <br />                                           Searching WormBase database using BLAST program cDNA sequences <br />                                           human enzymes revealed <span id='am-240' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> C. elegans genome designation <br />                                           T24A11.1 T24A11.1b. To verify cDNA sequence C. elegans enzyme, we <br />                                           employed rapid amplification cDNA ends (RACE) strategy using Smart-RACE <br />                                           kit Clontech. This resulted isolation cDNA sequences. One open <br />                                           reading frame encoding protein 1006 amino acid residues, gives rise a <br />                                           961 amino acid protein. The cDNAs resulted alternative splicing; matches <br />                                           C. elegans MTM3 previously reported (Xue et al, 2003; Genbank accession NM_065365) <br />                                           re-designated ceMTM3a here, represents new isoform was <br />                                           named ceMTM3b. The cDNA sequence ceMTM3b deposited GenBank <br />                                           database (accession number DQ988041). Both ceMTM3 ceMTM3b differences from <br />                                           predicted coding sequences T24A11.1 T24A11.1b defined WormBase. The <br />                                           ceMTM3 gene located chromosome III (&#226;&#710;&#8217;5.21) C. elegans genome spans over <br />                                           10 kb. ceMTM3a 10 exons ceMTM3b contains 12-bp-shorter exon 3 skips the <br />                                           entire 335 bp exon 9. A schematic diagram ceMTM3 protein structure shown Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           S1 Online Supplementary Materials. ceMTM3 conserved catalytic domain C- <br />                                           terminal FYVE domain. The FYVE domain preceded coiled coil structure 30&#226;&#8364;&#8220;40 aa. <br />                                           ceMTM3b differs ceMTM3a missing 4 amino acid residues N-terminus and <br />                                           having different, shorter C-terminal segment FYVE domain. The human C. <br />                                           elegans enzymes share ~30 % overall sequence identity catalytic domains the <br />                                           FYVE domain regions. Similar structures suggest similar functions regulatory <br />                                           mechanisms. Therefore, finding ceMTM3 C. elegans provides excellent model system <br />                                           study function regulation FYVE domain-containing MTM enzymes. <br />  <br />                               Expression distribution ceMTM3 C. elegans <br />                                           To analyze expression ceMTM3 C. elegans, generated rabbit polyclonal <br />                                           antibody GST-ceMTM3CT, GST fusion molecule containing C-terminal region <br />                                           (aa 701 961) ceMTM3b expressed E. coli cells. This antibody purified through <br />                                           negative selection GST-agarose positive selection antigen affinity column. On <br />                                           Western blotting analysis, ceMTM3 appears relatively broad band ~105 kDa (Fig. 1, <br />                                           upper panel). It expressed egg young larva (L1 L2) stages. It decreases near <br />                                           absence L3 young adult stages, regains high expression egg-laying <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           period post-reproductive age. We employed immunofluorescent cell staining to <br />                                           detect tissue expression enzyme C. elegans. Staining whole-mount gravid <br />                                           worms showed expression enzyme body wall muscle eggs (Fig. 1, middle <br />                                           panel). This verified staining paraffin-embedded cross-sections adult worms <br />                                           (Fig. 1, lower panel). Note staining paraffin-embedded cross-sections revealed <br />                                           partial expression enzyme intestine. The muscle expression verified <br />                                           co-staining paraffin-embedded sections antibody myosin, muscle <br />                                           marker (Fig. S2). Interestingly, earlier study, using green fluorescent protein (GPF) <br />                                           reporter construct, Xue et al [2003] demonstrated predominant expression ceMTM3 in <br />                                           head neurons low level expression posterior intestinal cells. However, authors <br />                                           cautioned reporter constructs tissues genes are <br />                                           expressed expression posterior intestinal cells common artifact GFP <br />                                           expression vectors [Xue et al, 2003]. In study, employed specific antibody detect <br />  <br />  <br />  <br />                                               J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                 Page 5 <br />  <br />  <br />                                          expressed protein directly, specificity method verified knockdown <br />                                          studies (see Fig. 3C). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Biochemical characterization ceMTM3 <br />                                          By using immuno-purified ceMTM3, analyzed substrate specificity ceMTM3 (Fig. <br />                                          2A). As expected, ceMTM3 preferably dephosphorylated PI3P. It displayed significant <br />                                          activity PI3,5P2 essentially activity PI4P PI5P. Like members of <br />                                          tyrosine phosphatase superfamily, ceMTM3 effectively inhibited sodium vanadate <br />                                          peroxide. It generally believed FYVE domain specifically binds PI3P [Stenmark <br />                                          Aasland, 1999]. To determine binding specificity FYVE domains to <br />                                          phosphoinositides, employed commonly used lipid membrane overlay assays. The PIP <br />                                          array membranes spotted different doses various phosphoinositides (Echelon <br />                                          Research Lab, Salt Lake City, UT) incubated GST GST-ceMTM3CT, GST <br />                                          fusion protein containing FYVE domain ceMTM3. As shown Fig. 2B, GST- <br />                                          ceMTM3CT binds specifically PI3P, binding GST control. <br />                                          The data indicate ceMTM contains typical FYVE domain. <br />  <br />                               Knockdown ceMTM feeding-based RNAi <br />                                          To study biological function ceMTM3, feeding-based RNA interference (RNAi) was <br />                                          employed knockdown expression enzyme C. elegans. For purpose, full- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          length coding sequence ceMTM3b fragments cloned pPD129.36 <br />                                          vector contains T7 promoter 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; ends drive transcription (Fig. 3A). As <br />                                          shown Fig. 3B, 3C, treatment wild type N2 worms E. coli cells expressing <br />                                          double-strand RNA derived full-length truncated fragments of <br />                                          ceMTM3b cDNA resulted nearly total deletion ceMTM3 revealed Western blotting <br />                                          analysis immunofluoresecent staining paraffin-embedded sections. These data only <br />                                          indicate efficiency feeding-based RNAi knocking gene expression of <br />                                          ceMTM3 verified specificity anti-ceMTM3 antibody. As expected, <br />                                          knockdown ceMTM3 RNAi greatly diminished phosphatase activity measured <br />                                          PI3P substrate (Fig. 3D). We obtained similar results cDNA fragments <br />                                          shown Fig. 3A. This essentially rules possibility <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> the <br />                                          knockdown non-specific genes. For simplicity, results observed ceMTM3bF1 <br />                                          fragment described below. <br />  <br />                                          As PI3P phosphatase, knockdown expected increase level intracellular PI3P. To <br />                                          access level PI3P, took advantage GST-ceMTM3CT binds specifically to <br />                                          PI3P. First, phospholipids extracted acidified chloroform/methanol worms <br />                                          untreated treated ceMTM3bf1 RNAi. The crude lipid extracts quantified based <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          phosphate contents resolved layer chromatography silica plates. Sections <br />                                          corresponding standard PI3P scraped extracted acidified chloroform/ <br />                                          methanol. Equivalent amounts partially purified lipids spotted nitrocellulose <br />                                          membranes standard PI3P incubated GST-ceMTM3CT as <br />                                          described above. Fig. 4 demonstrates knockdown ceMTM3 caused approximately <br />                                          2-fold increase binding GST-ceMTMCT, representing 2-fold increase PI3P <br />                                          contents RNAi-treated worms. It noted standard curve non-linear <br />                                          separation crude lipids thin-layer chromatography silica plates necessary <br />                                          reduce background binding. <br />  <br />                               Morbidity mortality C. elegans worms caused knockdown ceMTM3 <br />                                          A clear phenotype associated knocking-down ceMTM3 expression impaired <br />                                          locomotion worms late post reproductive stages (day 5 thereafter). <br />                                          This clearly reflected moving tracks animal shown Fig. 5. C. elegans worms <br />  <br />                                              J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                            Page 6 <br />  <br />  <br />                                        propagating waves alternating dorsal ventral flexions body length, <br />                                        producing regular sinusoidal tracks bacterial lawn. Normal worms fed vector control <br />                                        E. coli cells kept moving track average age 9 days. However, RNAi-treated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        worms started sluggish body movement day 5. Their tracks shorter <br />                                        wavelengths, traveling distance fixed period time shorter that <br />                                        exhibited control worms. The body movement gradually worsened animal aged. <br />                                        By day 8, inactive failed 1 cm diameter cycle 2 <br />                                        hr, day 9, body movement essentially impaired. Consequently, died, on <br />                                        average, day 14. In contrast, normal worms average lifespan 20 days. <br />                                        Fig. 6 compares morbidity mortality control ceMTM3F1 RNAi-treated C. <br />                                        elegans worms. It noted poor body movement RNAi-treated worms <br />                                        occurs day 5 late reproductive period. At larva young adult stages, their <br />                                        locomotion appeared normal. These RNAi-treated worms developed normally grew to <br />                                        normal body size 5&#226;&#8364;&#8220;7% reduction brood size. In addition, ~4% the <br />                                        RNAi-treated worms formed &#226;&#8364;&#339;worm bag&#226;&#8364;? caused hatching young larva womb. <br />                                        These worms included morbidity mortality analyses (Fig. 6). It be <br />                                        pointed mild egg laying defect caused RNAi-mediated knockdown ceMTM3 has <br />                                        noted earlier study Xue et al [2003]. However, study, effects on <br />                                        locomotion lifespan detected investigated. <br />  <br />                               Discussion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        In present study, cloned characterized FYVE domain-containing member <br />                                        MTM tyrosine phosphatase subfamily. More importantly, using RNAi technique, <br />                                        loss ceMTM3 expression impaired locomotion C. elegans worm the <br />                                        animal aged. Considering predominant expression enzyme muscle the <br />                                        animal, believe ceMTM3 required maintenance muscle function the <br />                                        C. elegans worm. The function ceMTM3 muscle unexpected mutation of <br />                                        hMTM1, founding member MTM subfamily enzymes, responsible X-linked <br />                                        recessive myotubular myopathy [Laporte et al, 1996], characterized severe <br />                                        hypotonia generalized muscle weakness impaired maturation muscle fibers in <br />                                        affected newborn males. However, mutation hMTM1 appears affect organization of <br />                                        muscle cells myogenesis disease believed result arrest the <br />                                        normal development muscle fibers myotubular stage [Spiro et al, 1966; van <br />                                        Wijngaarden et al, 1969]. Interestingly, MTM1-deficient mice develop progressive <br />                                        centronuclear myopathy postnatal life severely reduces life expectancy [Buj- <br />                                        Bello et al, 2002]. The data indicate MTM1 plays role muscle maintenance mice <br />                                        myogenesis seen human X-linked recessive myotubular myopathy. <br />                                        Apparently, ceMTM3 similar mouse MTM1 muscle maintenance. The loss of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        muscle mass, referred sarcopenia, normal phenomenon animals consequence <br />                                        aging [Deschenes, 2004]. This apparent normal worms day 15. Decrease muscle <br />                                        tissue begins age 50 years human, dramatic the <br />                                        60th year life. Loss muscle mass aged directly results diminished muscle <br />                                        function. Loss muscle fiber number principal cause sarcopenia. The mechanism for <br />                                        loss unknown. Our data suggest reduced function MTM family enzymes may <br />                                        play role. <br />  <br />                                        Our study suggests function ceMTM3 aging-related. The MTM family enzymes <br />                                        initially identified dual specificity phosphatases, subsequent studies demonstrated <br />                                        enzymes preferentially dephosphorylate PI3P PI3,5P2 [Clague and <br />                                        Lorenzo, 2005; Robinson Dixon, 2006]. By specifically dephosphorylating PI3P and <br />                                        PI3,5P2, MTM family enzymes crucial regulating level phosphoinositides, <br />                                        important signal transducers, loss expression naturally causes <br />  <br />  <br />                                            J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                      Page 7 <br />  <br />  <br />                                        accumulation PI3P PI3,5P2, indirectly affect level other <br />                                        phosphoinositides. The role phosphoinositides documented. In fact, disruption <br />                                        age-1/PI(3) kinase, downstream signaling component DAF-2 insulin-like <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        signaling pathway known extend lifespan C. elegans cause constitutive <br />                                        dauer formation (Daf-c) phenotype, considered nonaging stage [Friedman and <br />                                        Johnson, 1988]. Conversely, loss-of-function daf-18/PTEN, counterpart age-1/PI3 <br />                                        kinase, suppresses life extension constitutive dauer formation associated daf-2 or <br />                                        age-1 mutants [Rouault et al, 1999; Mihaylova et al, 1999]. Of course, products age-1/ <br />                                        PI(3) kinase PI3,4,5P PI3,4P, primary substrate daf-18/PTEN <br />                                        [Vanhaesebroeck et al, 2001]. As PI3P phosphatase, ceMTM3 directly <br />                                        dephosphorylate PI3,4,5P PI3,4P affect levels indirectly altering the <br />                                        pool phosphoinositides. The MTM enzymes mainly dephosphorylate D-3 position of <br />                                        PI3P PI3,5P2 [Clague Lorenzo, 2005; Robinson Dixon, 2006; Fig. 2], and <br />                                        directly antagonize age-1/PI(3) kinase. Instead, counterpart ceMTM3 may <br />                                        class III PI3 kinase LET-512/Vps34 PI3 kinase responsible production of <br />                                        PI3P C. elegans. LET-512/Vps34 essential protein required membrane trafficking <br />                                        endocytosis [Takacs-Vellai et al, 2005]. The let-512/Vps34 mutant worms display lethality <br />                                        molting defects, alterations outer nuclear membrane endoplasmic <br />                                        reticulum [Roggo et al, 2002]. However, earlier studies indicate knockdown other <br />                                        C. elegans MTM enzymes, namely, MTM-6 MTM-9, ceMTM3, able <br />                                        rescue larval lethality let-512/Vps34 mutants [Xue et al, 2003; Dang et al, 2004]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        LET-512/Vps34 required autophagy [Takacs-Vellai et al, 2005]. Autophagy a <br />                                        form programmed cell death. Like apoptosis, major implications various diseases, <br />                                        including cancer degenerative disorders [Kroemer Jaattela, 2005; Maiuri et al, 2007]. <br />                                        It determined loss-of-function ceMTM3 equivalent gain-of-function of <br />                                        let-512/VPS34 leads autophagy worm cells. <br />  <br />                                        Like members tyrosine phosphatase superfamily, ceMTM3 inactivated <br />                                        oxidation key cysteinyl residues catalytic center highly sensitive to <br />                                        oxidation (see Fig. 2A). Reactive oxygen species (ROS), superoxide radical <br />                                        (O2&#226;&#710;&#8217;&#226;&#8364;&#162;), hydrogen peroxide (H2O2), hydroxyl radical (OH&#226;&#8364;&#162;), generated during <br />                                        cellular metabolism, especially mitochondrial energy production [Beckman Ames, <br />                                        1998]. Oxidative damage considered main cause aging. It postulated a <br />                                        slowing rate accumulation oxidative damage partly responsible the <br />                                        life extension phenotype seen C. elegans daf-2 age-1 mutations [Honda Honda, <br />                                        2002]. Our data showed ceMTM3 sensitive peroxide loss ceMTM3 function <br />                                        causes muscle deterioration C. elegans. Therefore, inactivation ceMTM3 oxidation <br />                                        attributable loss muscle fibers normal worms later ages. We believe <br />                                        studying MTM enzymes clinical implications prevention treatment <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        human sarcopenia. <br />  <br />                               Supplementary Material <br />                                        Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgements <br />                                        Grant sponsors: National Natural Science Foundation China No. 30470391 (to X Fu) NIH HL076309 and <br />                                        HL079441 (to ZJ Zhao). <br />  <br />  <br />                               Abbreviations used <br />                                        PI3P <br />  <br />  <br />                                            J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                     Page 8 <br />  <br />  <br />                                                         phosphotidylinositol 3-phosphate <br />  <br />                                        PI4P <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                         phosphotidylinositol 4-phosphate <br />  <br />                                        PI5P <br />                                                         phosphotidylinositol 5-phosphate <br />  <br />                                        PI3 <br />                                                         4P2, phosphotidylinositol 3,4-bisphosphate <br />  <br />                                        PI3 <br />                                                         5P2, phosphotidylinositol 3,5-bisphosphate <br />  <br />                                        PI3 <br />                                                         4,5P3, phosphotidylinositol 3,4,5-trisphosphate <br />  <br />                               References <br />                                        Alonso A, Sasin J, Bottini N, Friedberg I, Friedberg I, Osterman A, Godzik A, Hunter T, Dixon J, Mustelin <br />                                           T. Protein tyrosine phosphatases human genome. Cell 2004;117:699&#226;&#8364;&#8220;711. [PubMed: 15186772] <br />                                        Beckman KB, Ames BN. The free radical theory aging matures. Physiol Rev 1998;78:547&#226;&#8364;&#8220;581. <br />                                           [PubMed: 9562038] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Bolino A, Muglia M, Conforti FL, LeGuern E, Salih MA, Georgiou DM, Christodoulou K, Hausmanowa- <br />                                           Petrusewicz I, Mandich P, Schenone A, Gambardella A, Bono F, Quattrone A, Devoto M, Monaco <br />                                           AP. Charcot-Marie-Tooth type 4B caused mutations gene encoding myotubularin-related <br />                                           protein-2. Nat Genet 2000;25:17&#226;&#8364;&#8220;19. [PubMed: 10802647] <br />                                        Buj-Bello A, Laugel V, Messaddeq N, Zahreddine H, Laporte J, Pellissier JF, Mandel JL. The lipid <br />                                           phosphatase myotubularin essential skeletal muscle maintenance myogenesis mice. <br />                                           Proc Natl Acad Sci U S A 2002;99:15060&#226;&#8364;&#8220;15065. [PubMed: 12391329] <br />                                        Clague MJ, Lorenzo O. The myotubularin family lipid phosphatases. Traffic 2005;6:1063&#226;&#8364;&#8220;1069. <br />                                           [PubMed: 16262718] <br />                                        Dang H, Li Z, Skolnik EY, Fares H. Disease-related myotubularins function endocytic traffic in <br />                                           Caenorhabditis elegans. Mol Biol Cell 2004;15:189&#226;&#8364;&#8220;196. [PubMed: 14565969] <br />                                        Deschenes MR. Effects aging muscle fibre type size. Sports Med 2004;34:809&#226;&#8364;&#8220;824. [PubMed: <br />                                           15462613] <br />                                        Friedman DB, Johnson TE. A mutation age-1 gene Caenorhabditis elegans lengthens life and <br />                                           reduces hermaphrodite fertility. Genetics 1988;118:75&#226;&#8364;&#8220;86. [PubMed: 8608934] <br />                                        Honda Y, Honda S. Oxidative stress life span determination nematode Caenorhabditis elegans. <br />                                           Ann N Y Acad Sci 2002;959:466&#226;&#8364;&#8220;474. [PubMed: 11976220] <br />                                        Kroemer G, Jaattela M. Lysosomes autophagy cell death control. Nat Rev Cancer 2005;5:886&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             897. [PubMed: 16239905] <br />                                        Laporte J, Hu LJ, Kretz C, Mandel JL, Kioschis P, Coy J, Klauck SM, Poustka A, Dahl N. A gene mutated <br />                                             X-linked myotubular myopathy defines new putative tyrosine phosphatase family conserved in <br />                                             yeast. Nat Genet 1996;13:175&#226;&#8364;&#8220;182. [PubMed: 8640223] <br />                                        Maehama T, Dixon JE. The tumor suppressor, PTEN/MMAC1, dephosphorylates lipid second <br />                                             messenger, phosphatidylinositol 3,4,5-trisphosphate. J Biol Chem 1998;273:13375&#226;&#8364;&#8220;13378. <br />                                             [PubMed: 9593664] <br />                                        Maiuri MC, Zalckvar E, Kimchi A, Kroemer G. Self-eating self-killing: crosstalk autophagy <br />                                             apoptosis. Nat Rev Mol Cell Biol 2007;8:741&#226;&#8364;&#8220;752. [PubMed: 17717517] <br />                                        Mihaylova VT, Borland CZ, Manjarrez L, Stern MJ, Sun H. The PTEN tumor suppressor homolog in <br />                                             Caenorhabditis elegans regulates longevity dauer formation insulin receptor-like signaling <br />                                             pathway. Proc Natl Acad Sci U S A 1999;96:7427&#226;&#8364;&#8220;7432. [PubMed: 10377431] <br />  <br />  <br />  <br />  <br />                                              J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                                    Page 9 <br />  <br />  <br />                                        Myers MP, Pass I, Batty IH, Van der Kaay J, Stolarov JP, Hemmings BA, Wigler MH, Downes CP, <br />                                            Tonks NK. The lipid phosphatase activity PTEN critical tumor supressor function. Proc <br />                                            Natl Acad Sci USA 1998;95:13513&#226;&#8364;&#8220;13518. [PubMed: 9811831] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Robinson FL, Dixon JE. Myotubularin phosphatases, policing 3-phosphoinositides. Trends Cell Biol <br />                                            2006;16:403&#226;&#8364;&#8220;412. [PubMed: 16828287] <br />                                        Roggo L, Bernard V, Kovacs AL, Rose AM, Savoy F, Zetka M, Wymann MP, Muller F. Membrane <br />                                            transport Caenorhabditis elegans: essential role VPS34 nuclear membrane. EMBO J <br />                                            2002;21:1673&#226;&#8364;&#8220;1683. [PubMed: 11927551] <br />                                        Rouault JP, Kuwabara PE, Sinilnikova OM, Duret L, Thierry-Mieg D, Billaud M. Regulation dauer <br />                                            larva development Caenorhabditis elegans daf-18, homologue tumour suppressor <br />                                            PTEN. Curr Biol 1999;9:329&#226;&#8364;&#8220;332. [PubMed: 10209098] <br />                                        Senderek J, Bergmann C, Weber S, Ketelsen UP, Schorle H, Rudnik-Schoneborn S, Buttner R, Buchheim <br />                                            E, Zerres K. Mutation SBF2 gene, encoding novel member myotubularin family, in <br />                                            Charcot-Marie-Tooth neuropathy type 4B2/11p15. Hum Mol Genet 2003;12:349&#226;&#8364;&#8220;356. [PubMed: <br />                                            12554688] <br />                                        Spiro AJ, Shy GM, Gonatas NK. Myotubular myopathy. Persistence fetal muscle adolescent boy. <br />                                            Arch Neurol 1966;14:1&#226;&#8364;&#8220;14. [PubMed: 4954227] <br />                                        Stenmark H, Aasland R. FYVE-finger proteins--effectors inositol lipid. J Cell Sci 1999;112:4175&#226;&#8364;&#8220; <br />                                            4183. [PubMed: 10564636] <br />                                        Stenmark H, Aasland R, Toh BH, D&apos;Arrigo A. Endosomal localization autoantigen EEA1 is <br />                                            mediated zinc-binding FYVE finger. J Biol Chem 1996;271:24048&#226;&#8364;&#8220;24054. [PubMed: 8798641] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Sui X, Krantz SB, Zhao ZJ. Stem cell factor erythropoietin inhibit apoptosis human erythroid <br />                                            progenitor cells different signalling pathways. Br J Haematol 2000;110:63&#226;&#8364;&#8220;70. [PubMed: <br />                                            10930980] <br />                                        Takacs-Vellai K, Vellai T, Puoti A, Passannante M, Wicky C, Streit A, Kovacs AL, Muller F. Inactivation <br />                                            autophagy gene bec-1 triggers apoptotic cell death C. elegans Curr Biol 2005;15:1513&#226;&#8364;&#8220;1517. <br />                                        Taylor GS, Maehama T, Dixon JE. Myotubularin, protein tyrosine phosphatase mutated myotubular <br />                                            myopathy, dephosphorylates lipid second messenger, phosphatidylinositol 3-phosphate. Proc <br />                                            Natl Acad Sci U S A 2000;97:8910&#226;&#8364;&#8220;8915. [PubMed: 10900271] <br />                                        Timmons L, Court DL, Fire A. Ingestion bacterially expressed dsRNAs produce specific potent <br />                                            genetic interference Caenorhabditis elegans. Gene 2001;263:103&#226;&#8364;&#8220;112. [PubMed: 11223248] <br />                                        Tonks NK. Protein tyrosine phosphatases: genes, function, disease. Nat Rev Mol Cell Biol <br />                                            2006;7:833&#226;&#8364;&#8220;846. [PubMed: 17057753] <br />                                        van Wijngaarden GK, Fleury P, Bethlem J, Meijer AE. Familial &quot;myotubular&quot; myopathy. Neurology <br />                                            1969;19:901&#226;&#8364;&#8220;908. [PubMed: 5816884] <br />                                        Vanhaesebroeck B, Leevers SJ, Ahmadi K, Timms J, Katso R, Driscoll PC, Woscholski R, Parker PJ, <br />                                            Waterfield MD. Synthesis function 3-phosphorylated inositol lipids. Annu Rev Biochem <br />                                            2001;70:535&#226;&#8364;&#8220;602. [PubMed: 11395417] <br />                                        Walker DM, Urbe S, Dove SK, Tenza D, Raposo G, Clague MJ. Characterization MTMR3. An inositol <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            lipid 3-phosphatase novel substrate specificity. Curr Biol 2001;11:1600&#226;&#8364;&#8220;1605. [PubMed: <br />                                            11676921] <br />                                        Xue Y, Fares H, Grant B, Li Z, Rose AM, Clark SG, Skolnik EY. Genetic analysis myotubularin <br />                                            family phosphatases Caenorhabditis elegans. J Biol Chem 2003;278:34380&#226;&#8364;&#8220;6. [PubMed: <br />                                            12788949] <br />                                        Zhao R, Fu X, Li Q, Krantz SB, Zhao ZJ. Specific interaction protein tyrosine phosphatase-MEG2 <br />                                            phosphatidylserine. J Biol Chem 2003;278:22609&#226;&#8364;&#8220;22614. [PubMed: 12702726] <br />                                        Zhao R, Qi Y, Zhao ZJ. FYVE-DSP1, dual-specificity protein phosphatase containing FYVE domain. <br />                                            Biochem Biophys Res Commun 2000;270:222&#226;&#8364;&#8220;229. [PubMed: 10733931] <br />                                        Zhao R, Qi Y, Zhao ZJ. FYVE-DSP2, FYVE Domain-Containing Dual Specificity Protein Phosphatase <br />                                            That Dephosphorylates Phosphotidylinositol 3-Phosphate. Exp Cell Res 2001;265:329&#226;&#8364;&#8220;338. <br />                                            [PubMed: 11302699] <br />  <br />  <br />  <br />  <br />                                            J Cell Biochem. Author manuscript; available PMC 2008 October 7. <br />                             Ma et al.                                                                                           Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Fig. 1. Expression distribution ceMTM3 C. elegans <br />                                        Top panel: C. elegans different stages including egg, L1, L2, L3, L4, YA (young adult), egg- <br />                                        laying adult (ELA), post-reproductive adult (PRA) collected according to <br />                                        synchronized culture procedures. Cell extracts containing equal amounts total proteins were <br />                                        subjected Western blotting analysis anti-ceMTM3 antibody. The position ceMTM3 <br />                                        indicated. A weak, non-specific band ~60 kDa essentially serves  <br /> </body></html>