<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                       Dyskerin Ablation in Mouse Liver Inhibits rRNA <br />                                Processing and Cell Division <br />  <br />              Dyskerin is a component of small nucleolar ribonucleoprotein complexes and acts as a pseudouridine <br />           synthase <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> modify newly synthesized ribosomal, spliceosomal, and possibly other RNAs. It is encoded by the <br />           DKC1 gene, the gene mutated in X-linked dyskeratosis congenita, and is also part of the telomerase complex. <br />           The yeast ortholog, Cbf5, is an essential protein, but in mammals the effect of dyskerin ablation at the cellular <br />           level is not known. Here we show that mouse hepatocytes can survive after induction of a Dkc1 deletion. In the <br />           absence of dyskerin, rRNA processing is inhibited with the accumulation of large precursors, and fibrillarin <br />           does not accumulate in nucleoli. A low rate of apoptosis is induced in the hepatocytes, which show an induction <br />           of the p53-dependent cell cycle checkpoint pathway. Signs of liver damage including an increase in serum <br />           alanine aminotransferase activity and a disordered structure at the histological and macroscopic levels are <br />           observed. In response to carbon tetrachloride administration, when wild-type hepatocytes mount a rapid <br />           proliferative response, those without dyskerin do not divide. We conclude that hepatocytes can survive without <br />           dyskerin but that the role of dyskerin in RNA modification is essential for cellular proliferation. <br />  <br />  <br />    Modification of specific bases of newly synthesized ribo-                   &#226;&#8364;&#339;scaRNPs&#226;&#8364;&#65533; are characterized by the presence in the scaRNA <br /> somal and spliceosomal RNAs is an early posttranscriptional                    of a conserved Sca motif (43) and function in the modification <br /> event and is carried out by specialized nucleolar complexes, the               of snRNAs (9), the small RNAs essential for pre-mRNA splic- <br /> snoRNPs. There are 2 classes of snoRNPs, H/ACA RNPs,                           ing. In vertebrates the 3&#226;&#172;&#732; domain of telomerase RNA, TERC, <br /> which pseudouridylate specific uridines, and C/D RNPs, which                   is an H/ACA RNA (36), and this RNA, which provides the <br /> catalyze the methylation of specific residues. In both cases the               template for synthesis of telomere repeats, is found in an RNP <br /> complexes consist of a guide RNA and 4 proteins (1, 12, 55). In                complex with the same 4 proteins found in other snoRNPs as <br /> the case of H/ACA RNPs the guide RNA is an H/ACA                               well as telomerase reverse transcriptase and other proteins (13, <br /> snoRNA which uses <span id='am-394' about='xsp:base' typeof='owl:Thing'>base</span> pairing to guide the complex to                         37, 48, 49). TERC RNA contains a Sca motif and is found in <br /> specific uridines in the nascent target RNA (14). The 4 pro-                   Cajal bodies (25). Finally, some H/ACA RNPs have no known <br /> teins are dyskerin (20, 35), NOP10 (21), NHP2 (21, 52), and                    function, their integral RNA having no known target. In one <br /> GAR1 (16); dyskerin is the pseudouridine synthase (29). RNA                    case of such an orphan RNA its expression appears to be <br /> modification is an ancient process, and the snoRNP proteins                    restricted to the brain (8), opening up the possibility that <br /> are highly conserved. Despite this, the role of RNA modifica-                  H/ACA RNAs can effect tissue-specific gene expression. <br /> tions is not clearly understood. Pseudouridines are frequently                    In Drosophila the dyskerin ortholog, minifly, is essential for <br /> found in regions of rRNAs that are functionally important, and                 viability (15). Minifly mutations that retain some function lead <br /> they are thought to increase the stability of folded domains.                  to small size, developmental delay, and reduced fertility <br /> Though depletion of individual snoRNAs does not seem to                        through defects in pseudouridylation and rRNA processing. In <br /> affect growth in yeast, the yeast ortholog of dyskerin, Cbf5, is               mice complete <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>knockout</span> of the gene encoding dyskerin, Dkc1, <br /> essential (26). Interestingly, mutations that render Cbf5 unable               is embryonic lethal (19). In humans hypomorphic mutations in <br /> to form pseudouridine permit cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> but lead to a severe                the DKC1 gene on the X chromosome cause X-linked dysker- <br /> reduction in ribosome synthesis (56). <br />                                                                                atosis congenita (DC) (20, 28), an inherited bone marrow <br />    H/ACA snoRNPs have other cellular functions as well as <br />                                                                                failure syndrome characterized by the presence of hypo- and <br /> pseudouridylation of rRNA (34). Some are involved in cleav- <br />                                                                                hyperpigmentation, nail dystrophy, and mucosal leukoplakia <br /> age of rRNA precursors, and a class of H/ACA RNPs is local- <br />                                                                                (50). Interestingly most of the pathological effects of the dys- <br /> ized in the Cajal body rather than the nucleolus (9). These <br />                                                                                kerin mutations in DC patients seem to be due to problems <br />                                                                                with telomere maintenance, since other forms of DC are <br />    * Corresponding author. Mailing address: Dept. Internal Medicine,           caused by mutations in other components of telomerase (51) <br /> Div. Hematology, Washington University School of Medicine, 660 S.              and biochemical investigations show a decrease in levels of <br /> Euclid Ave., Box 8125, St. Louis, MO 63110. Phone: (314) 362-8814.             telomerase RNA, TERC, and telomerase activity but no de- <br /> Fax: (314) 362-8826. E-mail: pmason@im.wustl.edu. <br />    &#226;&#8364;&#161; Supplemental material for this article may be found at http://mcb <br />                                                                                tectable change in rRNA processing or snoRNA metabolism <br /> .asm.org/.                                                                     (53, 54). Mice, or mouse embryonic stem (ES) cells, containing <br />    &#228;&#338;&#164; <br />      Published ahead of print on 16 November 2009.                             dyskerin amino acid changes that cause DC in humans so far <br />  <br />                                                                          413 <br />  414       GE ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br /> show variable effects, some having decreased Terc RNA and                           described previously (19). RNA was isolated using Trizol (Invitrogen, Carls- <br /> telomerase and mild defects in ribosome biogenesis (18, 38).                        bad, CA). <br />                                                                                        Nuclear protein extraction was performed using NE-PER nuclear and cyto- <br /> So far none of these mice has shown the pathological features                       plasmic extraction reagents (Pierce, Rockford, IL). Twenty micrograms of pro- <br /> of DC, probably due to the fact that laboratory mice have very                      tein was resolved by SDS-polyacrylamide gel electrophoresis (PAGE) analysis <br /> long telomeres. In the complete absence of telomerase, mice                         and transferred to a Hybond-ECL nitrocellulose membrane (GE Healthcare, <br /> show no striking defects in the first generation but they pass                      Piscataway, NJ). Antibodies were detected by ECL plus Western blotting detec- <br />                                                                                     tion reagents (GE Healthcare) and visualized by autoradiography. Primary an- <br /> shortened telomeres to their offspring and generation 3 and                         tibodies used in this study were polyclonal antiserum to 2 dyskerin peptides, <br /> later generations show some features similar to those seen in                       DCSNPLKREIGDYIR (aa 74 to 88) and GLLDKHGKPTDNTPAC (aa 382 to <br /> DC (3, 32). Recently we described a mouse line produced by                          396 &#226;&#171;&#185; C) (38); a polyclonal antiserum against the dyskerin peptide DRKPLQE <br /> gene targeting which contained a copy of a human <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>mutation</span>                           DDVAEIQHA (aa 18 to 33); anti-TATA box binding protein (1:2,000; Abcam, <br />                                                                                     Cambridge, MA); anti-p53 (1:1,000; Abcam); anti-p21 (1:200; Abcam); and an- <br /> consisting of a 3&#226;&#172;&#732; truncation of the Dkc1 gene, encoding a <br />                                                                                     tifibrillarin (1:1,000; Abcam). The secondary antibody was either horseradish <br /> protein with the last 21 amino acids (aa) missing (17). These                       peroxidase-conjugated goat anti-mouse antibody (1:10,000; Abcam) or horserad- <br /> mice showed the slower growth of proliferative tissues charac-                      ish peroxidase-conjugated goat anti-rabbit antibody (1:10,000; Abcam). <br /> teristic of DC. The slow growth was dependent on telomerase,                           Pre-rRNA analysis. Pre-rRNA analysis by Northern blotting was as described <br /> independent of telomere <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, and associated with the accu-                       previously (44) using synthetic oligonucleotide DNA probes. RNA (10 &#226;&#65533;&#174;g) was <br />                                                                                     electrophoresed in 1% (wt/vol) agarose-formaldehyde gels and blotted to Hybond- <br /> mulation of DNA damage.                                                             N&#226;&#171;&#185; membranes (GE Healthcare) overnight. Oligonucleotides (10 pmol) were <br />    In mammalian cells, though dyskerin is essential for embry-                      end labeled with [&#226;&#65533;&#165;-32P]ATP and T4 polynucleotide kinase (Fisher, Pittsburgh, <br /> onic development, it is not known if it is essential for cell                       PA). The temperature of hybridization and washing was determined by the <br /> survival or for cell division. In this paper we used an inducible                   melting temperature (Tm) of each probe. The probes used were ITS1, 5&#226;&#172;&#732;-CTC <br />                                                                                     TCACCTCACTCCAGACACCTCGCTCCA-3&#226;&#172;&#732;; ITS2, 5&#226;&#172;&#732;-ACCCACCGCAGC <br /> knockout strategy to delete the Dkc1 gene in adult mice and                         GGGTGACGCGATTGATCG-3&#226;&#172;&#732;; and 28S, 5&#226;&#172;&#732;-CACCTTTTCTGGGGTCTGA <br /> found that while most tissues were reconstituted <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> cells that                    T-3&#226;&#172;&#732;. <br /> had escaped the deletion the liver survived for weeks with the                         Sucrose gradient centrifugation. Sucrose gradient centrifugation for polysome <br /> majority of its cells containing the deleted gene. This gave us                     analysis was performed as previously described (24). One gram of liver tissue, 200 <br />                                                                                     &#226;&#65533;&#174;l hypotonic buffer (1.5 mM KCl, 2.5 mM MgCl2, and 5.0 mM Tris-Cl, pH 7.4), <br /> the opportunity of studying mammalian cells that did not con- <br />                                                                                     and 200 &#226;&#65533;&#174;l hypotonic lysis buffer (1.5 mM KCl, 2.5 mM MgCl2, and 5.0 mM <br /> tain dyskerin. These cells, despite their long-term survival,                       Tris-Cl, pH 7.4; 2% sodium deoxycholate, 2% Triton X-100, and 2.5 mM dithio- <br /> were defective in rRNA synthesis, showed signs of a cell cycle                      threitol [DTT]) were gently homogenized using a Dounce homogenizer. The <br /> checkpoint response and the induction of apoptosis, and were                        lysates were centrifuged at 8,000 &#226;&#171;&#187; g for 10 min at 4&#194;&#176;C. The supernatant (&#226;&#172;&#402;800 <br />                                                                                     &#226;&#65533;&#174;l) was supplemented with 4 &#226;&#65533;&#174;l RNase inhibitor (Invitrogen) and 80 &#226;&#65533;&#174;l heparin <br /> defective in cell division following liver injury. We conclude <br />                                                                                     and stored at &#226;&#171;&#186;80&#194;&#176;C. Linear 10% to 45% sucrose gradients (80 mM NaCl, 5 mM <br /> that dyskerin is not required for <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> but is essential for                MgCl2, 20 mM Tris-Cl, pH 7.4, and 1 mM DTT) were formed using a Gradient <br /> cell division.                                                                      Master (BioComp). Gradients were centrifuged at 38,000 rpm for 3 h at 4&#194;&#176;C and <br />                                                                                     analyzed with an ISCO fractionator (ISCO). <br />                                                                                        Histology and immunohistochemistry. Liver sections from the age-matched <br />                         MATERIALS AND METHODS                                       knockout mice and control littermates were analyzed by hematoxylin and eosin <br />                                                                                     (HE) staining to count mitotic bodies characteristic of cells in M phase of the cell <br />    Generation of Dkc1 conditional KO mice. The generation of the Dkc1lox allele <br />                                                                                     cycle. Indirect immunohistochemistry for BrdU was used to identify proliferating <br /> in C57BL/6 mice has been previously described (19). Mice carrying the Alb-Cre       cells in S phase. Terminal deoxynucleotidyltransferase-mediated dUTP nick-end <br /> transgene (42) and mice carrying the Mx1-Cre transgene were obtained from the       labeling (TUNEL) was used to detect apoptotic nuclei. <br /> Jackson Laboratory (Bar Harbor, ME). Homozygous floxed dyskerin mice were              Double staining of dyskerin and fibrillarin or dyskerin and BrdU was used to <br /> bred to Alb-Cre mice, and male mice (9 weeks old unless specified) which carry      determine their expression and/or localization. Briefly, 4-&#226;&#65533;&#174;m-thick paraffin sec- <br /> a floxed dyskerin allele and Alb-Cre were selected for study (Dkc1loxTg:AlbCre,     tions were passed through xylene and graded alcohol and rinsed in phosphate- <br /> referred to as knockout [KO] mice). For all experiments and analysis, mice with     buffered saline (PBS). Slides were then treated with Diva Decloaking solution <br /> the genotype Dkc1lox or Dkc1&#226;&#171;&#185;Tg:AlbCre were used as controls and are referred       (BioCare Medical, Walnut Creek, CA) in a pressure cooker for 3 min. Slides <br /> to as wild type (WT) throughout. Similarly, male Dkc1loxTg:Mx1Cre mice were         were rinsed with PBS three times. Slides were then incubated with primary rabbit <br /> selected for study (KO) along with their Dkc1lox or Dkc1&#226;&#171;&#185;Tg:Mx1Cre littermates      anti-dyskerin antibody (1:200) and mouse antifibrillarin (1:1,000) or with anti- <br /> (WT). For induction of the Mx1-Cre transgene, mice (5 weeks old) were injected      dyskerin and rat anti-BrdU (1:400; Accurate Chemical &amp; Scientific Corp, West- <br /> with poly(I:C) (250 &#226;&#65533;&#174;g intraperitoneally [i.p.]) every other day for a total of 3   bury, NY) overnight at 4&#194;&#176;C. After washes, the sections were incubated in the <br /> injections. Mice were sacrificed 4 weeks after the last injection.                  secondary anti-rabbit Cy3-conjugated antibody (1:500; Jackson ImmunoResearch, <br />    All experiments on mice were performed under the strict guidelines of the        West Grove, PA) and anti-mouse fluorescein isothiocyanate (FITC)-conjugated <br /> National Institutes of Health and the Institutional Animal Use and Care Com-        antibody (Jackson ImmunoResearch) or with anti-rabbit Cy3 and anti-rat FITC- <br /> mittee at Washington University. Age- and sex-matched conditional knockout          conjugated antibody (1:200; Jackson ImmunoResearch) for 1 h at room temper- <br /> mice and control littermates were weighed and sacrificed (n &#226;&#171;&#189; 3 to 5). Livers       ature. Sudan black autofluorescence inhibitor (Chemicon International, Te- <br /> were isolated, and the wet weights were recorded to calculate the liver weight-     mecula, CA) was used to quench the background fluorescence. Slides were rinsed <br /> to-body weight ratio. Livers were also used for DNA and RNA isolation, protein      with distilled water (dH2O) and counterstained with DAPI (4&#226;&#172;&#732;,6-diamidino-2- <br /> extraction, histology, and immunohistochemistry.                                    phenylindole). Slides were viewed under an Axioskop 40 (Zeiss) upright <br />    CCl4 treatment. Nine-week-old male Dkc1loxTg:AlbCre mice and controls were       research microscope, and digital images were obtained with a Nikon Coolpix <br /> subjected to carbon tetrachloride (CCl4) (Sigma-Aldrich, St. Louis, MO) treat-      camera. <br /> ment (2 &#226;&#65533;&#174;g/g of body weight; i.p.) to induce liver damage. Equal numbers of KO         The positive cells were counted in 5 low-power fields (200&#226;&#171;&#187;) in 3 sections for <br /> and WT mice were sacrificed by CO2 asphyxiation at different times after CCl4       HE, BrdU, and TUNEL staining or a higher-power field (1,000&#226;&#171;&#187;; up to 300 cells) <br /> treatment. Mice were injected i.p. with 2.5 mg/mouse bromodeoxyuridine (BrdU;       from 3 different knockout or control livers for double staining. <br /> Sigma-Aldrich, St. Louis, MO) 1 h before sacrifice. The regenerating livers were       mRNA processing and quantitative reverse transcription-PCR (qRT-PCR). <br /> harvested and used for DNA isolation, total RNA isolation, protein extraction,      The levels of pre-mRNA and mRNA of glyceraldehyde-3-phosphate dehydro- <br /> and paraffin embedding.                                                             genase (GADPH) and nucleolin were measured by RT-PCR. 28S was used as a <br />    DNA, RNA, and protein analysis. Liver tissues from the WT and KO mice            control. The SYBR green master mix (Applied Biosystems, Foster City, CA) was <br /> were used for isolating and purifying DNA and RNA. DNA was isolated by the          used to follow RNA amplification in the real-time reaction. Reverse transcrip- <br /> standard sodium dodecyl sulfate (SDS)-proteinase K method. Southern blotting        tion and PCRs were performed according to the manufacturer&#226;&#8364;&#8482;s recommenda- <br /> to distinguish between the Dkc1lox and the Dkc1&#226;&#338;&#172;DC1 alleles was performed as        tions for the SuperScript III first-strand synthesis system for RT-PCR (Invitro- <br />  VOL. 30, 2010                                                                                   DYSKERIN ABLATION IN MOUSE LIVER                415 <br />  <br />  <br />  <br />  <br />    FIG. 1. Deletion of the Dkc1 gene in various tissues of Dkc1loxTg:Mx1Cre transgenic mice at different times. Groups of 4 mice (6 weeks old) <br /> were injected with poly(I:C) (i.p.; 250 &#226;&#65533;&#174;g) 3 times every other day. Genomic DNA was prepared from the indicated organs, and Cre-mediated <br /> deletion of the Dkc1 gene was detected by Southern blotting. (A) Diagram of Dkc1 gene and the targeted alleles. The Dkc1lox allele has a lox <br /> element in intron 11 and an IRES Neo selection cassette followed by a second lox element in the 3&#226;&#172;&#732; UTR. In the presence of the Cre recombinase, <br /> the lox elements recombine, removing exons 12 to 15, producing the deleted allele Dkc1&#226;&#338;&#172;DC1. (B) Southern blotting for the proportions of Dkc1lox <br /> and Dkc1&#226;&#338;&#172;DC1 in various tissues collected 4 weeks after poly(I:C) injection. BM, bone marrow. (C) Quantification of the Dkc1 recombination <br /> efficiency in various tissues of Dkc1loxTg:Mx1Cre mice over time. The surviving cells in liver are more than 80% recombined at the Dkc1 locus, while <br /> those in thymus and bone marrow are only 0 to &#226;&#172;&#402;20% recombined. (D) Cells in various tissues of female Dkc1lox/&#226;&#171;&#185; mice consist of 50% deleted <br /> and 50% WT alleles after poly(I:C) treatment, showing that the induction of the deletion is efficient. In this experiment the mice were Dkc1DC7 <br /> with differently placed lox sites (19). CER, cerebellum. (E) Western blot showing the decrease in the amount of dyskerin in the liver of 2 mice, <br /> one with a 60% reduction and one with a 90% reduction in the amount of dyskerin. TBP, TATA box binding protein. <br />  <br />  <br />  <br /> gen). The primers were specifically designed with Primer Express 3.0 software        Dkc1lox allele has a lox element in intron 11 and an internal <br /> (Applied Biosystems). For nucleolin, the forward primer for spliced mRNA was         ribosome entry site (IRES)-Neo selection cassette followed by <br /> 5&#226;&#172;&#732;-CATGTGACTAAGTCCTTCATGACTGA-3&#226;&#172;&#732;, the forward primer specific <br /> for unspliced pre-mRNA was 5&#226;&#172;&#732;-GGATGGGAAAAGTAAAGGGATTG-3&#226;&#172;&#732;,                            a second lox element in the 3&#226;&#172;&#732; untranslated region (UTR) (Fig. <br /> and the reverse primer was 5&#226;&#172;&#732;-GCCTTTGACCTTTCTCTCCAGTAT-3&#226;&#172;&#732;. For                       1A). In the presence of the Cre recombinase, the lox elements <br /> GAPDH, the forward primer for spliced mRNA was 5&#226;&#172;&#732;-TTGTGGAAGGGCT                      recombine, removing exons 12 to 15, producing the deleted <br /> CATGACCACAGTCCA-3&#226;&#172;&#732;, the forward primer for unspliced pre-mRNA was <br />                                                                                      allele Dkc1&#226;&#338;&#172;DC1. In experiments with mouse embryonic fibro- <br /> 5&#226;&#172;&#732;-CTCACTCATTGCCCCCGTGTTTTCTA-3&#226;&#172;&#732;, and the reverse primer was 5&#226;&#172;&#732;- <br /> TCAGATCCACGACGGACACATTGGGGG-3&#226;&#172;&#732;. For U2, the forward primer                           blast (MEF) cells (not shown), we do not detect any truncated <br /> was 5&#226;&#172;&#732;-TTTGGCTAAGATCAAGTGTAGTATCTGTT-3&#226;&#172;&#732; and the reverse                              protein corresponding to the expected product from the de- <br /> primer was 5&#226;&#172;&#732;-CCAACTCCTACTTCCAAAAATCCA-3&#226;&#172;&#732;. For 28S, the forward                      leted allele, and so the Dkc1&#226;&#338;&#172;DC1 allele is essentially a null <br /> primer was 5&#226;&#172;&#732;-GGTAGCCAAATGCCTCGTCAT-3&#226;&#172;&#732; and the reverse primer was <br /> 5&#226;&#172;&#732;-CCCTTGGCTGTGGTTTCG-3&#226;&#172;&#732;. Fluorescence was measured using an Ap- <br />                                                                                      allele. Having found that the Dkc1&#226;&#338;&#172;DC1 allele is embryonic <br /> plied Biosystems 7500 Fast Real-Time PCR system (Applied Biosystems).                lethal (19), we were interested in whether adult tissues could <br />    Statistical analysis. An unpaired two-tailed Student t test was used to compare   survive in the absence of dyskerin. We thus bred male <br /> hepatocellular BrdU incorporation, TUNEL staining, mitotic body index, liver <br />                                                                                      Dkc1loxTg:Mx1Cre mice, taking advantage of the Mx1 interfer- <br /> weight-to-body weight ratio, alanine aminotransferase (ALT) activity, and DNA <br /> recombination efficiency with significance set at 0.05.                              on-inducible promoter, which can be conveniently induced by <br />                                                                                      double-stranded RNA in the form of poly(I:C). After Cre <br />                                                                                      induction mice were sacrificed at 2 days, 1 week, 1 month, and <br />                                   RESULTS <br />                                                                                      3 months, and DNA was extracted from different tissues and <br />   Dkc1 knockout mouse models. We previously described the                            examined by Southern blotting for the proportions of Dkc1lox <br /> construction of the Dkc1lox allele by gene targeting (19). The                       and Dkc1&#226;&#338;&#172;DC1 alleles. The results (Fig. 1B and C) show that in <br />  416     GE ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. Loss of dyskerin and fibrillarin in hepatocyte nucleoli in Dkc1loxTg:Mx1Cre transgenic mice by immunohistochemistry. (A) Dyskerin <br /> localizes to the nucleolus and colocalizes with the C/D snoRNP protein fibrillarin. The liver tissue was paraffin embedded and incubated with rabbit <br /> antidyskerin and mouse antifibrillarin antibody. The nucleoli were counterstained with DAPI. (B) Western blotting showed that the amount of <br /> nuclear fibrillarin protein in the KO mice was not changed. TATA box binding protein (TBP) was used as a loading control. <br />  <br />  <br /> most tissues the proportion of surviving cells containing the               and one with a 90% reduction in the amount of dyskerin. The <br /> deletion is about 40% (heart and kidney) or close to zero (bone             antibody used in this experiment (38) was raised against a <br /> marrow, thymus, spleen, and lung). In the liver, however, the               mixture of 2 peptides, one of which is deleted as part of the <br /> proportion of surviving cells is much higher, greater than 80%              Dkc1&#226;&#338;&#172;DC1 deletion. It is thus possible that a truncated protein <br /> in some animals; it reaches a maximum after 2 days, remains                 is present in the livers after deletion that is not detected by this <br /> high for 4 weeks, and then slowly falls afterwards. These results           antibody. With a second antibody made against an N-terminal <br /> almost certainly arise from competitive outgrowth of the few                peptide, however, no truncated peptide is detected (see Fig. S1 <br /> cells that escape the Dkc1 deletion populating most of the                  in the supplemental material), indicating that if this peptide is <br /> adult organs and tissues, as has been seen in experiments                   made, it is very unstable. <br /> involving the deletion of essential genes in adult mice (46). In               The loss of dyskerin from the liver in these mice was con- <br /> experiments in which female Dkc1lox/&#226;&#171;&#185; animals were injected,                firmed by immunohistochemistry. In WT hepatocytes dyskerin <br /> most tissues consisted of 50% deleted and 50% WT alleles,                   colocalizes with fibrillarin as dense spots at the nucleoli, but in <br /> indicating that the efficiency of deletion was close to 100% and            livers from the knockout mice dyskerin is not detectable in <br /> the surviving cells were presumably the ones with the WT allele             many of the hepatocytes (Fig. 2A). Interestingly, in many hepa- <br /> on the active X chromosome (Fig. 1D). Thus, we conclude that                tocytes lacking dyskerin, fibrillarin is not localized in dense <br /> hematopoietic tissues with high rates of cellular proliferation             spots as before. Fibrillarin is a component of C/D box snoRNAs <br /> cannot survive without dyskerin, while tissues with a lower                 which participate in methylation of nascent rRNA molecules <br /> proliferative rate, such as the uninjured adult liver, can tolerate         during the same stages of rRNA biogenesis in which H/ACA <br /> high levels of Dkc1 deletion.                                               snoRNAs act (7). To ask if the lack of detectable fibrillarin is <br />    Deletion of Dkc1 in mouse liver. We wanted to investigate in             due to destabilization of fibrillarin caused in some way by lack <br /> more detail the induction of the Dkc1 deletion in the liver in              of dyskerin, we looked at fibrillarin by Western blotting (Fig. <br /> the Dkc1loxTg:Mx1Cre mice after poly(I:C) injection. The pro-               2B). There was no difference between WT and knockout livers, <br /> portion of cells with the deletion reached 90% after 2 days and             suggesting that in the absence of dyskerin C/D snoRNPs are <br /> remained at 90% until 1 month after injection before gradually              not maintained in nucleoli. <br /> decreasing (Fig. 1C). We therefore chose to use livers 1 month                 To confirm our result that mice can tolerate lack of dyskerin <br /> after injection for our studies. The maximum extent of deletion             in liver and to use a system independent of interferon induc- <br /> was variable as measured either by Southern blotting or by                  tion, we next decided to induce the Dkc1 deletion by using the <br /> Western blotting. Figure 1E shows the decrease in the amount                Cre recombinase under a liver-specific promoter, i.e., the <br /> of dyskerin in the livers of 2 mice, one with a 60% reduction               mouse albumin promoter, AlbCre (41). We studied the appear- <br />  VOL. 30, 2010                                                                               DYSKERIN ABLATION IN MOUSE LIVER                 417 <br />  <br />  <br />  <br />  <br />    FIG. 3. Deletion of Dkc1 gene in liver of Dkc1loxTg:AlbCre trans- <br /> genic mice at different times. (A) Dkc1 recombination efficiency in <br /> mice of different ages determined by Southern blot analysis. Liver <br /> DNA was isolated from 5- to &#226;&#172;&#402;16-week-old mice and subjected to <br /> Southern blot analysis as shown in Fig. 1; the percentage of Dkc1&#226;&#338;&#172;DC1 <br /> alleles reached a peak of &#226;&#172;&#402;70% at 6 to 12 weeks. (B) Western blotting <br /> showed that the dyskerin protein in liver is significantly decreased in <br /> the KO mice. TBP is TATA box binding protein used as a loading <br /> control. <br />  <br />  <br />  <br /> ance of the Dkc1 deletion in male Dkc1loxTg:AlbCre mice. <br /> Southern blot analysis showed a peak of 70% of liver cells <br /> containing the deletion at 6 weeks with this level being main- <br /> tained until 12 weeks and then declining (Fig. 3A). Because <br /> AlbCre mice express Cre only in hepatocytes and possibly in <br /> bile duct and hepatocytes make up only about 70% of liver <br /> cells, these calculations underestimate the amount of deletion. <br /> A similar decrease in dyskerin was detected at the protein level <br /> (Fig. 3B). Immunofluorescence with dyskerin and fibrillarin <br /> antibodies was similar to that observed with Dkc1loxTg:Mx1Cre               FIG. 4. Dyskerin deletion activates a p53-dependent cell cycle ar- <br /> mice.                                                                     rest pathway in Dkc1loxTg:AlbCre mice. (A) HE staining of liver section <br />                                                                           showing the fibrotic regions and the appearance of small proliferating <br />    Dkc1loxTg:AlbCre mice show signs of chronic liver damage               cells in the portal areas in KO mice (200&#226;&#171;&#187;). (B) Detection of apoptotic <br /> and impaired rRNA processing. The histology of the livers                 hepatocytes in KO mice by TUNEL staining (200&#226;&#171;&#187;; P &#226;&#172;&#65533; 0.05). The <br /> depleted of dyskerin by Mx1-Cre induction appeared normal                 arrows show positive nuclei. (C) Increased hepatocyte proliferation in <br /> (data not shown). However, in the livers of Dkc1loxTg:AlbCre              KO mice by BrdU staining (200&#226;&#171;&#187;; P &#226;&#172;&#65533; 0.05). (D) Western blot show- <br />                                                                           ing the elevation of p53 and p21 protein in the livers of KO mice. <br /> mice at 9 weeks of age marked abnormalities were observed.                &#226;&#65533;&#164;-Actin was used as a loading control. <br /> These were not present at 6 weeks (data not shown), indicating <br /> that they may develop several weeks after Dkc1 deletion. For <br />                                                                           formed analysis of hepatocellular BrdU incorporation, which <br /> example, these livers exhibited fibrotic regions and small pro- <br />                                                                           revealed that 1.3% of hepatocytes in KO mice but less than <br /> liferating cells in periportal areas (Fig. 4A). In addition the <br />                                                                           0.5% in WT mice incorporated BrdU, reflecting increased <br /> liver weight and body weight of the KO mice were significantly <br />                                                                           cellular turnover in KO livers (Fig. 4C). <br /> reduced compared with WT mice (Table 1). To further assess <br /> liver damage in these mice, we measured serum ALT activity <br /> and bilirubin. Comparable and normal levels of bilirubin were                    TABLE 1. Body weight and liver weight in 9-week-old WT <br /> observed in the WT and KO mice. However, the KO mice                                   (Dkc1lox) and KO (Dkc1loxTg:AlbCre) mice <br /> showed a 3.5-fold increase in serum ALT, indicating the pres- <br />                                                                                   Variable                   WT                        KO <br /> ence of liver damage. We also used TUNEL staining to mea- <br /> sure the extent of apoptosis in the liver tissue and found that           Liver wt (g)                  1.310 &#226;&#171;&#190; 0.045            1.178 &#226;&#171;&#190; 0.051a <br /> 0.2 to 0.3% of hepatocytes in the knockout livers stained pos-            Body wt (g)                  28.503 &#226;&#171;&#190; 0.991           24.620 &#226;&#171;&#190; 0.629a <br />                                                                           Liver/body wt ratio               0.046                    0.049 <br /> itive while positive-staining hepatocytes in WT age-matched <br /> animals were virtually undetectable (Fig. 4B). Finally, we per-             a <br />                                                                                 P &#226;&#172;&#65533; 0.05. <br />  418     GE ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Dyskerin ablation disrupts rRNA processing in the Dkc1loxTg:AlbCre mice. (A) Scheme of pre-rRNA processing in mouse liver (5). The <br /> probes at ITS1 and ITS2 were used to detect the precursors of 18S and 5.8S, respectively. (B) Northern blot of RNA from livers of male mice of <br /> different genotypes. With the ITS1 probe a decrease in the amount of the 21S and accumulation of the 30S, which are the precursors of the mature <br /> 18S RNA, are observed in the KO mice. With the ITS2 probe a decrease in the precursors of 41S, 17S, and 12S was observed. (C) Effect of dyskerin <br /> deletion on polysomes. Polysomes from liver extracts were sedimented through 10 to 45% sucrose gradients, showing a decrease in the amount <br /> of 80S ribosomes but no significant change in the polysome profile. The 40S and 60S subunits were present in wild-type and KO livers in very small <br /> amounts. <br />  <br />  <br />  <br />    Because impaired ribosome biogenesis, a possible effect of              processing in mouse liver. To investigate this, hepatic rRNA <br /> dyskerin ablation, is known to promote p53-dependent cell                  from Dkc1loxTg:AlbCre mice and their wild-type littermates of <br /> cycle arrest and apoptosis (33, 40), we next quantified the                different ages was analyzed by Northern blotting. The results <br /> expression of p53 and the p53 target p21 in KO and WT livers.              show that dyskerin deficiency results in changes in the accu- <br /> Figure 4D shows increased levels of both proteins in KO liver,             mulation of rRNA precursors including decreased 21S, 41S, <br /> indicating that the p53 cell cycle arrest pathway is activated in          17S, and 12S RNA. Of the products of the ITS1 cleavage, the <br /> hepatocytes lacking dyskerin.                                              30S species increases while the 32S species is not altered (Fig. <br />    Together, our observations that lack of dyskerin leads to               5A and B; see also Fig. S2 in the supplemental material). <br /> mislocalization of fibrillarin (Fig. 2A) and induces a p53-de-             Finally we analyzed the polysome profile of WT and KO livers, <br /> pendent cell cycle arrest (Fig. 4D) suggested that ribosome                which showed a decrease in the relative amount of 80S ribo- <br /> biogenesis may be impaired in the hepatocytes lacking dys-                 somes in the KO livers (Fig. 5C). <br /> kerin. Although the effect of dyskerin ablation on ribosome                   Dkc1loxTg:AlbCre mice exhibit an impaired hepatocellular <br /> biogenesis in mammalian cells has not been investigated, Cbf5-             proliferative response to acute liver damage. The data de- <br /> deficient yeasts exhibit a complete block in rRNA synthesis (6).           scribed above, showing that hepatocytes can survive for months <br /> Furthermore, yeast Cbf5 mutants that fail to pseudouridylate               after Dkc1 deletion, suggest that dyskerin is not essential for <br /> RNA display a growth defect and impaired ribosome produc-                  cell survival (Fig. 1B and D). However, the modestly increased <br /> tion but do not show abnormal accumulation of rRNA pro-                    liver damage seen in these mice raises the question of whether <br /> cessing intermediates (56). Based on these observations, we                the slow loss of Dkc1-deleted liver cells over time is the result <br /> were interested in the effects of dyskerin ablation on rRNA                of a relative proliferative advantage of wild-type hepatocytes <br />  VOL. 30, 2010                                                                         DYSKERIN ABLATION IN MOUSE LIVER                     419 <br />  <br />  <br />  <br />  <br />   FIG. 6. Hepatocyte proliferation before and after CCl4 treatment in Dkc1loxTg:AlbCre mice. (A) Similar perivenous damage in the livers of WT <br /> and KO mice after CCl4 treatment by HE staining (200&#226;&#171;&#187;). (B) Impaired hepatocyte proliferation in the liver of KO mice 72 h after CCl4 treatment <br /> by BrdU staining (200&#226;&#171;&#187;). (C) Quantification of BrdU-positive hepatocytes after CCl4 treatment at different time points. At time zero, BrdU <br /> incorporation was seen in 1.3% of hepatocytes in KO mice but in less than 0.5% of hepatocytes in WT mice. The peak of the response in WT liver <br /> was seen at 72 h with 24.7% hepatocytes positive, but only 6.5% hepatocytes were positive in KO livers (P &#226;&#172;&#65533; 0.05 at time zero, 48 h, and 72 h). <br /> (D) Decreased mitotic index in the KO hepatocytes due to the poor proliferative response. <br />  <br />  <br />  <br /> over Dkc1-null cells during the hepatic regenerative response             DNA from the KO livers before and 48 h after the CCl4 <br /> to such injury. To answer this question, we examined the de-              treatment shows that the proportion of the Dkc1lox allele ex- <br /> gree of injury and hepatic regenerative response to CCl4 in               pressing WT dyskerin goes from 30% before to 70% after (Fig. <br /> Dkc1loxTg:AlbCre and Dkc1lox mice. Both KO and WT mice                    7A), which is consistent with this possibility. Immunohisto- <br /> showed comparable perivenous damage (Fig. 6A), indicating                 chemical analysis showed that 48 h after CCl4 treatment all the <br /> that dyskerin deletion did not affect the susceptibility of liver         BrdU-positive hepatocytes in Dkc1loxTg:AlbCre mice were pos- <br /> cells to CCl4 damage. In contrast, analysis of hepatocellular             itive for dyskerin expression (Fig. 7B and C). These data pro- <br /> BrdU incorporation showed significantly (P &#226;&#172;&#65533; 0.05) increased              vide convincing evidence that dyskerin is essential for hepato- <br /> hepatocellular proliferation at 48 and 72 h after CCl4 injection          cellular proliferation. <br /> in WT compared with KO mice (Fig. 6B and C). The reduced                     mRNA processing in the absence of dyskerin. We have ob- <br /> proliferative index in KO cells was further demonstrated by the           served that dyskerin can be ablated from a large proportion of <br /> significantly diminished mitotic index 72 h after CCl4 treat-             mouse hepatocytes without any overt effect on the health of the <br /> ment in these animals (Fig. 6D). Interestingly, the proportion            animal. As well as being involved in ribosome biogenesis and <br /> of cells that are BrdU positive at 48 h to those in mitosis at 72 h       telomere maintenance, dyskerin is involved indirectly in <br /> is the same in WT and KO livers, implying that there is no                mRNA splicing, since snRNAs are modified by pseudouridy- <br /> block at the G2-M transition in KO mouse hepatocytes. Figure              lation guided by H/ACA scaRNAs. Inhibition of mRNA splic- <br /> 6B also illustrates the presence of a population of nonparen-             ing may be expected to cause a more severe <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> than we <br /> chymal periportal liver cells that are labeled with BrdU. These           observed. For this reason we wanted to examine mRNA splic- <br /> cells are much more abundant in KO than WT livers. Although               ing in WT and KO livers (31). We thus chose 2 genes ex- <br /> we have not yet identified them, we suspect that they are                 pressed in liver, GAPDH and nucleolin, and measured the <br /> infiltrating immunological cells. These data show that hepato-            steady-state levels of unspliced and spliced transcripts in <br /> cellular proliferation is impaired in Dkc1loxTg:AlbCre mice.              WT and KO animals, reasoning that the ratio between them <br />    Dyskerin is necessary for cell division. The observation that          would reflect the efficiency of splicing. The calculated ratio <br /> hepatocellular proliferation is reduced in Dkc1loxTg:AlbCre               between spliced and unspliced transcripts did not show any <br /> versus control mice and that some hepatocytes in Dkc1loxTg:               significant change for either gene, implying that dyskerin <br /> AlbCre mice escape the Dkc1 deletion raises the possibility that          ablation did not affect splicing efficiency (Fig. 8A). This <br /> the small number of proliferating hepatocytes in the KO livers            result was not expected since spliceosomal RNAs, including <br /> were those that escaped deletion. Southern blotting of liver              U2, contain a high proportion of modified nucleotides and <br />  420     GE ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                               FIG. 8. Dkc1 deletion does not affect mRNA processing. (A) Lev- <br />                                                                            els of nucleolin and GAPDH pre-mRNA and mRNA in KO and WT <br />                                                                            liver. Dkc1 deletion has no effect on the RNA levels or the pre-mRNA/ <br />                                                                            mRNA ratio. 28S RNA was used as an internal control. (B) Measure- <br />                                                                            ment of U2 RNA levels in KO and WT liver by qRT-PCR. <br />  <br />  <br />  <br />                                                                            These data indicate that there is no gross defect in mRNA <br />                                                                            processing in the absence of dyskerin. <br />  <br />  <br />                                                                                                      DISCUSSION <br />                                                                               Most of our knowledge about dyskerin and snoRNPs and <br />                                                                            their role in modification of rRNA and snRNA comes from <br />                                                                            genetic studies of yeast and structural studies of archaebacte- <br />                                                                            ria. Modification using guide RNAs is a highly conserved pro- <br />                                                                            cess, and the proteins associated with H/ACA RNA and C/D <br />                                                                            RNAs are also highly conserved (22). In humans mutations in <br />                                                                            DKC1 cause X-linked dyskeratosis congenita (20), and there is <br />                                                                            good evidence that the major effect of these pathogenic dys- <br />                                                                            kerin mutations is to decrease telomerase activity, causing in- <br />                                                                            creased telomere shortening and exhaustion of stem cells in <br />                                                                            highly proliferating tissues (50). It is possible, however, that <br />                                                                            defects in snoRNP function also contribute to X-linked DC <br />                                                                            (38, 45). In mice a Dkc1 knockout is embryonic lethal (19), <br />                                                                            while other mutations cause variable effects on telomerase <br />                                                                            activity and rRNA biogenesis (17, 18, 38, 45). Here we have <br />                                                                            used an inducible deletion strategy to knock out Dkc1 in adult <br />                                                                            mice. We found that while most tissues could not survive with- <br />    FIG. 7. Dyskerin is necessary for cell division. (A) Southern blot of <br /> liver DNA from the Dkc1loxTg:AlbCre mice before and 48 h after the         out dyskerin and became populated by cells that had escaped <br /> CCl4 treatment. The proportion of the WT Dkc1lox allele in the KO          the deletion, the liver could survive with a large proportion of <br /> liver goes from 30% before to 70% after (P &#226;&#172;&#65533; 0.001). (B) Immuno-           its Dkc1 genes deleted for several months. In fact the rate of <br /> fluorescent detection of the presence of BrdU and dyskerin in the          turnover of hepatocytes in the livers of knockout animals was <br /> hepatocytes of Dkc1loxTg:AlbCre mice 48 h after treatment with CCl4. <br /> Paraffin-embedded liver sections were double stained with antidys-         higher than that in wild-type animals. This was most likely due <br /> kerin and anti-BrdU antibodies, and the nucleoli were counterstained       to the induction of apoptosis in a small percentage of cells and <br /> with DAPI. Hepatocellular BrdU incorporation was correlated with           their replacement by the division of hepatocytes that had, we <br /> dyskerin in the dyskerin-null mice. (C) Quantification of dyskerin- and    presume, escaped the deletion. Deletion in solid tissues using <br /> BrdU-positive hepatocytes. Three hundred hepatocytes were counted, <br />                                                                            the Mx1-Cre system is never complete, but since we achieved <br /> and all those that had incorporated BrdU were dyskerin positive. <br />                                                                            the maximum possible efficiency of deletion of about 50% in <br />                                                                            Dkc1lox/&#226;&#171;&#185; females, we assume that the efficiency of deletion in <br />                                                                            our mice was high. <br /> in the case of U2 it has been shown in vitro that pseudouri-                  Ribosome synthesis is inhibited in the absence of dyskerin. <br /> dines are required for efficient splicing (10). We thus exam-              One of our primary objectives was to determine to what extent <br /> ined the steady-state level of U2 RNA in WT and KO livers                  lack of dyskerin affects rRNA processing and ribosome bio- <br /> by qRT-PCR and found no significant difference (Fig. 8B).                  genesis. Our experiments suggest that in the absence of dys- <br />  VOL. 30, 2010                                                                      DYSKERIN ABLATION IN MOUSE LIVER                              421 <br />  <br />  <br /> kerin ribosome biogenesis cannot proceed. Firstly, in total liver   damage induced by CCl4. Moreover, regeneration was accom- <br /> RNA probed for rRNA precursors by Northern blotting we see          panied by an increase in the relative amount of wild-type DNA <br /> a marked reduction in more mature processed rRNA precur-            compared with DNA containing the Dkc1 deletion, indicating <br /> sors and also an accumulation of the 30S precursor. In              that much of the regeneration had involved division of cells <br /> Dkc1loxTg:AlbCre mice the dyskerin gene is deleted in hepato-       that did not contain a deleted gene. Compelling evidence that <br /> cytes, which comprise about 70% of the liver cells, and as seen     absence of dyskerin is incompatible with cell division is our <br /> in the immunofluorescence experiments, some hepatocytes es-         observation that when Alb-Cre mice were treated with CCl4 <br /> cape the deletion. Therefore, a decrease in the amount of an        and allowed to recover from the resulting liver damage very <br /> RNA precursor likely means that it is absent in cells with the      few cells in the knockout livers were replicating their DNA as <br /> deletion. This assumption is supported by the differences in the    assessed by BrdU incorporation. Remarkably the few cells that <br /> rRNA processing <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span> between Alb-Cre and Mx1-Cre exper-          did incorporate BrdU were all among the minority of cells in <br /> iments. With the Dkc1loxTg:Mx1Cre mice after poly(I:C) treat-       the livers that stained positive with the dyskerin antibody. <br /> ment, the dyskerin deletion extends to nonhepatocytes, and in       Presumably these were cells that had escaped the deletion. <br /> these experiments, we see a more extreme change in the pat-         Whether the block in the cell cycle is caused by nucleolar <br /> tern of rRNA processing (data not shown). The precise nature        disruption activating the p53 checkpoint response (39, 41) or is <br /> and sequence of processing events that take place during the        due to an unknown mechanism that couples cell division and <br /> maturation of the primary 47S transcript to the mature 18S,         ribosome biogenesis (2, 41) remains to be determined. <br /> 5.8S, and 28S rRNAs in mouse are not completely established            Conclusions. Using conditional deletion of the mouse Dkc1 <br /> (27, 47). However, two alternative early events are cleavage in     gene in hepatocytes, we have produced a model in which we <br /> the 5&#226;&#172;&#732; external transcribed spacer (ETS) to give a 41S precur-      can study mammalian cells without any dyskerin. This model <br /> sor or cleavage within ITS1 to give a 30S and a 32S RNA. Very       should prove useful in delineating the role of dyskerin and <br /> little 41S RNA is detected in KO livers, suggesting that dys-       H/ACA snoRNAs in ribosome biogenesis, telomere mainte- <br /> kerin is required for the early cleavage event in the 5&#226;&#172;&#732; ETS        nance, and mRNA expression. Furthermore, the influence of <br /> (27). The 30S and 32S precursors in normal mouse cells are          ribosome biogenesis on cell cycle progression may be amena- <br /> further processed, 30S being processed via a 21S intermediate       ble to experimentation using this model. The experiments pre- <br /> to mature 18S RNA and 32S being processed via 17S and 12S           sented here show that lack of dyskerin blocks rRNA processing <br /> intermediates to mature 5.8S RNA. The probes that we used           after cleavage of the primary transcript in ITS1 and leads to <br /> (Fig. 5) hybridize to the 30S, 32S, 17S, 12S, and 21S species and   disruption of ribosome biogenesis. In the absence of dyskerin, <br /> show that while the early products of ITS1 cleavage accumu-         hepatocytes are not able to divide. <br /> late, the amount of 12S, 17S, and 21S intermediates is reduced <br /> in KO compared with WT livers. Together these data suggest                                     <br /> that in the absence of dyskerin the initial ITS1 cleavage event <br />                                                                       <br /> takes place but the 30S and 32S precursors are not processed        <br /> further, indicating that further processing requires pseudouri-      <br /> dylation of the nascent transcripts and/or an H/ACA RNA-            <br /> dependent cleavage event. Pseudouridylation of rRNA is              <br /> thought to take place on the early transcripts, prior to cleavage    <br /> in ITS1 (4, 11). Our results are compatible with recent studies     <br /> showing that while most cleavage events in rRNA processing, <br /> including those in the 5&#226;&#172;&#732; ETS, require the participation of                                          <br /> snoRNPs, a cleavage site within ITS1 requires, at least in yeast,     <br /> only the action of the catalytic RNA RNase MRP (23, 30).              <br />    More compelling evidence that ribosome biogenesis is              <br /> halted in cells without dyskerin is our observation that in these        <br /> cells immunofluorescence with a fibrillarin antibody does not        <br /> show a brightly staining nucleolus as seen in wild-type cells.         <br /> Fibrillarin is the active methylase present in C/D snoRNPs,          <br /> and in ribosome biogenesis it is active on the same molecules        <br /> as is dyskerin. These molecules will not become pseudouridy-         <br /> lated in the absence of dyskerin, and our results suggest that       <br /> lack of pseudouridylation affects the action of C/D snoRNAs.       <br /> The mechanism underlying the apparent linkage between                   <br /> these 2 processes will be the subject of further experiments&#226;&#8364;&#8221;         <br /> it will be interesting to know if lack of methylation inhibits       <br /> pseudouridylation. Moreover, the factors that maintain snoRNPs       <br /> in the nucleolus may be amenable to analysis in this system.         <br />    Dyskerin is essential for cell division. The second major             <br /> finding from our experiments is the absence of any cell division         <br /> in cells with no dyskerin. The dyskerin-depleted livers that we         <br /> studied showed a much-reduced proliferative response to liver       <br /> </body></html>