<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Developmental Control Lepidopteran Pest <br /> Spodoptera exigua Ingestion Bacteria Expressing <br /> dsRNA Non-Midgut Gene <br /> Honggang Tian1, Han Peng1, Qiong Yao1, Hongxin Chen1, Qi Xie2, Bin Tang1, Wenqing Zhang1* <br /> 1 State Key Laboratory Biocontrol, School Life Sciences, Sun Yat-sen University, Guangzhou, China, 2 Institute Genetics Developmental Biology, Chinese <br /> Academy Sciences, Beijing, China <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) induced double stranded RNA (dsRNA) used widely study gene <br />      function insects. Recently, reported gene knockdown insects induced successfully <br />      feeding dsRNA. However, unknown phenotypic silencing genes expressed the <br />      midgut occurs ingestion insect dsRNA. <br />  <br />      Principal Findings: Using chitin synthase gene A (SeCHSA) target gene, expressed cuticle tracheae <br />      lepidopteran pest Spodoptera exigua, showed growth development S. exigua larvae fed Escherichia <br />      coli expressing dsRNA SeCHSA disturbed, resulting lethality. In 4th 5th larval instars, prepupae, pupae, <br />      mean <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> rates insects fed dsRNA-containing diet 88.64%, 74.24%, 68.43% 62.63% respectively. The <br />      survival rates 5th instar larvae, prepupae pupae stages significantly lower controls, and <br />      significant lethality differences dsSeCHSA treatment dsControl ddH2O control 4th <br />      instar larvae. The effects ingesting bacterially expressed dsRNA transcription target gene, tissue structure, and <br />      survival rates insects dose-dependent. <br />  <br />      Conclusions: Our results suggest SeCHSA dsRNA useful means insect pest control. <br />  <br />   Citation: Tian H, Peng H, Yao Q, Chen H, Xie Q, et al. (2009) Developmental Control Lepidopteran Pest Spodoptera exigua Ingestion Bacteria Expressing <br />   dsRNA Non-Midgut Gene. PLoS ONE 4(7): e6225. doi:10.1371/journal.pone.0006225 <br />   Editor: Immo A. Hansen, New Mexico State University, United States America <br />   Received March 9, 2009; Accepted June 15, 2009; Published July 13, 2009 <br />   Copyright: &#195;&#376; 2009 Tian et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Basic Research Program China (grant IBN-2006CB102001) National Natural Science Foundation China <br />   (grant IBN-30471143). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lsszwq@mail.sysu.edu.cn <br />  <br />  <br />  <br />  <br /> Introduction                                                                          transgene-encoded ingestible dsRNA plant day be <br />                                                                                       used insect pest control. <br />    RNA interference (RNAi) emerged powerful tool                            In 2006, global area use biotech crops continued <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> the rapid analysis gene function variety organisms. RNAi                   climb tenth consecutive year sustained double-digit <br /> also shows great promise use biotechnology. Relevant                           growth rate 13%, reaching 102 million hectares. However, none <br /> applications include capacity avoid unwanted transgene                         transgenes employed large-scale commercialized <br /> silencing genetically engineered lines exploitation                     transgenic plants insect origins. Recent results that <br /> various types silencing inactivate undesirable genes [1,2].                     RNAi induced insects ingestion plant-expressed hairpin <br /> Direct microinjection commonly used procedure                         RNA offers promise insect pest control [2,7,8]. However, <br /> delivery double-stranded RNA (dsRNA) organisms.                               identification suitable gene transgenic RNAi plants is <br /> However, scientists variety fields exploring                        important; gene insecticidal <br /> more simple convenient means dsRNA delivery, including                         effects target pests, safe pests&#226;&#8364;&#8482; <br /> soaking [3], oral feeding [4], transgenic plant expression [5].                   natural enemies human beings. The achievement these <br /> Of delivery methods, transgenic plants expressing viral                         goals requires method large scale screening pest gene <br /> dsRNAs, including Arabidopsis thaliana [6] sugar beet Beta                    pools. Fortunately, RNAi dsRNA ingestion provides a <br /> vulgaris [5], used control plant pathogens. The                          good method discovery appropriate gene(s). The RNAi <br /> application RNAi insect pest control lagged                      feeding protocol advantages microinjection; most <br /> application plant pathogen control following years unsuc-                       notably convenient laborious [9]. In insects, <br /> cessful attempts [2]. However, 2007, groups major                         Tuner colleagues reported RNAi triggered <br /> progress exploitation transgenic plants engineered                       oral delivery dsRNAs gut gene EposCXE1, an <br /> express insect dsRNAs entomological research field control                    antennae gene EposPBP1, using microvolume syringe deliver <br /> of insect pests [7,8]. These exciting results encourage believe                 dsRNA mouthparts lepidopteran Epiphyas postvittana <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                                    July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                               RNAi Feeding Insect <br />  <br />  <br /> [10], later, feeding dsRNA cricket Gryllus           To characterize developmental expression <span id='am-317' about='xsp:pattern' typeof='owl:Thing'>pattern</span> sid-1 <br /> bimaculatus [11], termite Reticulitermes flavipes [12],          like gene S. exigua, sid-1 mRNA levels detected using RT- <br /> trigger RNAi response. In particular, gene silencing R. falvipes        PCR method developmental stages, including larvae 1st <br /> had lethal effects. However, lepidopteran insects,       5th instars, prepupae, pupae adults. The developmental <br /> unknown RNAi response insecticide effects                 expression pattern S. exigua sid-1 showed gene was <br /> be produced, especially non-midgut gene voluntary               present life stages. However, prepupae pupae <br /> ingestion dsRNA.                                                           developmental stages, sid-1 mRNA exhibited relatively lower <br />     dsRNAs ingestion protocols synthesized using                expression compared larvae adult life stages <br /> RNAi kit produced bacteria. Escherichia coli-mediated delivery         (Figure 1B). In order verify tissue distribution sid-1 <br /> of dsRNA reported Caenorhabditis elegans 1998 [13]           gene S. exigua, transcript levels sid-1 cuticle, trachea, <br /> planarians 2003 [14]. In comparison producing dsRNA                midgut fat body 1 day 5th instar larvae detected, and <br /> vitro kit, bacterially expressed dsRNA cheaply          results indicated transcript levels sid-1 similar <br /> more easily used large scale gene function analysis [15].                  tissues (Figure 1C). <br /> Therefore, used ingestion bacterially expressed dsRNA to <br /> study non-midgut gene lepidopteran pest, Spodoptera exigua.            Ingestion bacterially expressed dsRNA reduces SeCHSA <br />     Insect chitin synthases key enzymes cuticle, trachea,         mRNA level <br /> midgut development [16,17]. They encoded classes                   Since important gene sid-1 systemic RNAi proven to <br /> genes: CHSA CHSB. CHSA genes specifically expressed                   present S. exigua, constructed vector expressing partial <br /> in ectodermal cells, including epidermal tracheal cells,            sequence chitin synthase SeCHSA (dsSeCHSA) whether <br /> CHSB genes restricted gut epithelial cells produce            ingestion dsRNA SeCHSA gene, expressed in <br /> peritrophic matrix midgut [18]. Since chitin mainly           midgut, reduce target mRNA expression. To verify <br /> in arthropods fungi vertebrates plants [19],               result specific RNAi SeCHSA gene, <br /> chitin synthesis ideal target insect growth                   dsSeCHSA (635 bp, Figure 2A) aligned SeCHSB; 19 bp <br /> regulators [16,18]. Chitin synthase gene A S. exigua (SeCHSA)          consecutive identical sequences (Figure 2B). <br /> is expressed midgut, previous research using               dsSeCHSA inserted plasmid L4440 NcoI <br /> injection SeCHSA dsRNA S. exigua larvae demonstrated                  HindIII sites (Figure 3A), SeCHSA dsRNA produced <br /> disruption larval development [20]. Thus, used chitin           bacteria strain HT115(DE3) containing recombinant plasmid <br /> synthase gene A study.                                                L4440-SeA3 (Figure 3B). At time, used non-related <br />     The SID-1 (systemic RNA interference defective) gene             gene, D. melanogaster white gene, control dsRNA expression <br /> suggested act dsRNA signal channel systemic RNAi C.             (Figure 3C D). <br /> elegans [21]. To date, SID-1 genes                 To confirm bacterially expressed dsRNA introduced <br /> animal genomes exception dipteran genomes;               larval gut ingestion, experiment was <br /> be reason feeding Drosophila yeast-expressed dsRNA failed             performed S. exigua larvae fed artificial diet coated with <br /> as means inducing RNAi [22]. In present study,              bacteria colored FD&amp;C Blue. The colored bacteria <br /> characterized partial sequence SID-1 like gene S. exigua         easily detectable inside larvae (Figure 4). After day of <br /> order confirm vital systemic RNAi molecule present            feeding diet containing colored bacteria, blue diet was <br /> in species. Subsequently, fed bacterially expressed SeCHSA            observed midgut larvae (Figure 4A); blue color <br /> dsRNA S. exigua larvae seven days later observed                 accumulated midgut days later (Figure 4C). These <br /> RNAi response; larvae completely remove old                   results demonstrate bacteria ingested larvae and <br /> cuticles, exhibiting lethal phenotypes insecticide effects.               accumulated midgut. <br /> Further examination lethal <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> showed treated                  Previous reports showed ingestion high dose dsRNA <br /> larvae exhibited decreased endocuticle double-cuticles. Our               induce RNAi effect. Therefore, order select suitable <br /> results suggest RNAi effectively inhibit gene expression               concentration RNAi response, designed gradient <br /> lepidopteran insects insects fed dsRNA non-                  concentration dsRNA. After expressing dsRNA, bacteria cells <br /> midgut genes. Ingestion bacterially expressed dsRNA                 collected diluted different volumes ddH2O. The <br /> applicable large scale optimal gene screening ingestible              dsRNA concentrations designated high dose (2506), <br /> RNAi plants insect pest control.                                           medium dose (506), low dose (106) based dilution <br />                                                                               factors. Larvae fed bacteria 3, 5, 7 9 days, and <br /> Results                                                                       RT-PCR performed detect transcript SeCHSA. To <br />                                                                               avoid possible disturbance dsRNA delivered larval <br /> Characterization expression pattern analysis sid-1                     body, primers designed detecting SeA1 fragment, but <br /> like gene S. exigua                                                        dsSeCHSA fragment used dsRNA expression <br />    Using degenerate primers designed basis conserved                (Figure 2A). The RT-PCR results showed SeCHSA mRNA <br /> domain insect deduced SID-1 amino acid sequences, partial                substantially decreased days 7 9 larvae that <br /> sequence (231 bp) obtained S. exigua RT-PCR,                ingested high dose dsSeCHSA, compared negative controls <br /> longer 939 bp sequence isolated using 39-RACE                    (dsControl, L4440 ddH2O) (Figure 5A B). However, larvae <br /> specific primers 231 bp PCR fragment. Multiple alignment               fed medium dose dsSeCHSA showed slight decrease in <br /> and phylogenetic relationship analysis deduced amino acids             SeCHSA transcript levels, change SeCHSA transcript was <br /> the SID-1 conserved domain insects nematode C. elegans                 larvae fed low dose dsSeCHSA (Figure 5A B). <br /> SID-1 indicated 939 bp fragment (GenBank accession                   These results suggested sufficient concentration of <br /> no. FJ619650) belongs sid-1 like gene (Figure 1A S1).                dsRNA induce target gene knockdown. In addition, expression <br /> The phylogenetic tree showed S. exigua SID-1 like 1             midgut specific chitin synthase gene (SeCHSB) S. exigua larvae <br /> closest relationship lepidopteran Bombyx mori SID-1 like 2               affected feeding dsSeCHSA different <br /> deduced proteins (Figure 1A).                                                 concentrations (Figure 5A B), indicating observed <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                                        RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 1. Phylogenetic relationship analysis expression pattern sid-1 Spodoptera exigua. (A) The phylogenetic analysis of <br /> deduced SID-1 amino acids insects nematode C. elegans conducted using MEGA 4.0, nodes distance bootstrap values .50% were <br /> indicated, branches blue color represents closest group S. exigua SID-1 like 1. (B) Developmental expression patterns sid-1 <br /> like 1 gene S. exigua 1st instar larvae adult. (C) Tissue distribution sid-1 like 1 gene S. exigua 5th instar larvae. The housekeeping gene <br /> b-actin used internal control. <br /> doi:10.1371/journal.pone.0006225.g001 <br />  <br />  <br /> RNAi gene specific. We did detect variation SeCHSA                 dsSeCHSA. The results showed transcript levels of <br /> mRNA levels ingestion dsRNA 3 5 days,                  SeCHSA decreased cuticle trachea compared controls <br /> high dose dsSeCHSA (Figure S2). These results suggested                  (Figure 6). The target gene knockdown cuticle trachea <br /> RNAi response induced ingestion dsRNA requires                           suggested systemic RNAi induced ingestion of <br /> accumulation dsRNA larvae.                                              dsRNA non-midgut gene S. exigua. <br />    Because SeCHSA expressed cuticle and <br /> trachea [17], observation specific transcript levels                 RNAi feeding control larval development <br /> decreasing ingestion high dose dsSeCHSA S. exigua,                  After observing specific gene silencing following ingestion of <br /> further examination performed analyze SeCHSA mRNA                      bacterially expressed dsSeCHSA, interested determine <br /> levels cuticle trachea 9 days feeding high dose                 inhibition SeCHSA mRNA expression leads lethal <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             3                                   July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                               RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 2. Schematic diagram nucleotide sequence SeCHSA dsRNA expression. (A) Schematic diagram SeCHSA mRNA. The <br /> dsSeCHSA region (green box) dsRNA expression, SeA1 region (blue box) RT-PCR detection, red box represents putative <br /> catalytic domain SeCHSA. (B) Alignment nucleotide sequences SeCHSA SeCHSB region dsRNA generation. <br /> doi:10.1371/journal.pone.0006225.g002 <br />  <br />  <br /> phenotype S. exigua larvae. Larvae fed diet containing              control larvae. Thus, abnormal phenotype appears a <br /> bacterially expressed dsRNA SeCHSA day 1 1st instar,         result RNAi. The phenotypes observed following ingestion the <br /> continuously fed 3rd day 5th instar, using different       SeCHSA dsRNA similar caused benzoylphenyl ureas, <br /> dsRNA concentrations. Prior 3rd instar, interference                insect growth regulators [23]. This suggests the <br /> phenotype observed. After continuous feeding                     mechanisms action dsRNA ureas similar. <br /> dsSeCHSA-containing diet days, observed                  In 4th 5th instars larvae, prepupae, pupae, the <br /> 25.7% individuals fed high dose (2506) dsSeCHSA exhibited               average survival rates insects fed high dose dsSeCHSA- <br /> an obvious &#226;&#8364;&#732;&#226;&#8364;&#732;half-ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype early 4th instars         containing diet 88.64%, 74.24%, 68.43%, 62.63%, <br /> the 4th 5th instars, especially larval molting process         respectively (Figure 8A). The survival rates 5th instar larvae, <br /> (Figure 7A B). Approximately 10% &#226;&#8364;&#732;&#226;&#8364;&#732;half-ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; larvae         prepupae pupae stages significantly lower (p,0.05, <br /> developed &#226;&#8364;&#732;&#226;&#8364;&#732;double heads&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype (Figure 7B D). These               Duncan&#226;&#8364;&#8482;s test) controls, significant lethality <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;half ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype larvae usually remove         differences dsSeCHSA treatment and <br /> their old cuticle, failed complete ecdysis, died         dsControl ddH2O control 4th instar larvae. In addition, <br /> two days. On contrary, nearly larvae fed medium          80% dead larvae high dose dsSeCHSA <br /> low doses dsSeCHSA, control larvae, molted normally             feeding group exhibited &#226;&#8364;&#732;&#226;&#8364;&#732;half-ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype. However, no <br /> into stage. This RNAi-mediated malformed phenotype                   significant differences developmental duration of <br /> correlates decreased mRNA levels SeCHSA larvae             insects different developmental stages ingestion the <br /> fed diets containing bacteria-expressed dsSeCHSA (Figure 5).               high dose dsSeCHSA (p.0.05, ANOVA) (Figure 8B). In larvae <br /> However, abnormal phenotype observed larvae                    fed medium (506) low doses (106) dsSeCHSA control <br /> prepupae, prepupae pupae, pupae adult,                    larvae, obvious differences survival rate or <br /> continuous ingestion bacterially expressed dsSeCHSA (data              development duration (data shown). <br /> shown). In addition, nearly remaining living 5th instar larvae, <br /> prepupae pupae developed normally, showing significant                 Knockdown SeCHSA expression disrupts larval cuticle <br /> differences controls. During stage prepupae pupae,           structure <br /> small number larvae fed dsRNA-containing diet exhibited                   Our previous study showed SeCHSA expressed the <br /> variable malformed pupae (Figure S3); observed               cuticle, trachea, midgut (possibly contamination <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         4                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                          RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 3. Schematic diagram recombinant plasmids dsRNA expression production dsRNAs. (A) Schematic diagram of <br /> L4440-SeA3 SeCHSA dsRNA expression. (B) Total RNA extracted bacteria HT115(DE3) containing plasmid L4440-SeA3 blank <br /> plasmid L4440 IPTG induction. The position dsRNA produced indicated arrowhead. (C) Schematic diagram L4440-DmWhite for <br /> DmWhite control dsRNA expression. (D) Total RNA extracted bacteria. The position dsRNA produced indicated arrowhead. <br /> doi:10.1371/journal.pone.0006225.g003 <br />  <br /> trachea) Northern blot analysis [17]. Abnormal larvae injected        fed dsSeCHSA-containing diet. In abnormal <br /> with siRNA dsRNA SeCHSA gene contain disordered                larvae (high dose dsSeCHSA feeding group), integral structure <br /> chitin layers cuticle [20]. Thus, set determine         cuticle disordered, particularly endocuticle, a <br /> whether ingestion dsSeCHSA affects structure larval            double exocuticle present (Figure 9A); cuticles controls <br /> cuticle, trachea midgut peritrophic matrix abnormal larvae.        well-structured (Figure 9B, C D). Clear peritrophic <br />    Hematoxylin eosin (H &amp; E) staining used investigate        matrix midgut observed abnormal control <br /> the histological structure abnormal 4th instar day larvae         larvae (Figure 9I, J, K L), structural differences were <br />  <br />  <br />       PLoS ONE | www.plosone.org                                     5                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                                RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 4. Ingestion accumulation bacteria midgut Spodoptera exigua larvae. (A) The midgut (slightly blue) 4th instar <br /> larvae fed diet containing FD&amp;C Blue colored bacteria 1 day. (B) The midgut larvae fed normal artificial diet 1 day. (C) A <br /> section larval midgut larvae fed diet containing FD&amp;C Blue colored bacteria 3 days. (D) A section larval midgut from <br /> larvae fed normal artificial diet 3 days. Scale bars: 1 mm A B; 2.5 mm C D. <br /> doi:10.1371/journal.pone.0006225.g004 <br />  <br />  <br />  <br />  <br /> Figure 5. Effects ingesting bacterially expressed dsRNA transcription SeCHSA SeCHSB. On day 7 (A) day 9 (B) post- <br /> feeding, total RNA extracted individual larvae feeding diet containing bacterially expressed SeCHSA dsRNA (dsSeCHSA) SeCHSA <br /> and SeCHSB transcripts detected using RT-PCR. RNA larvae fed diet containing control bacteria DmWhite (dsControl), L4440 <br /> or ddH2O served controls. Lanes 1, 4 7 represent larvae fed high bacteria concentration (2506), lanes 2, 5 8 represent results <br /> from larvae fed medium bacteria concentration (506), lanes 3, 6, 9 represent larvae fed low bacteria concentration (106). The <br /> housekeeping gene b-actin used reference. <br /> doi:10.1371/journal.pone.0006225.g005 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                                  RNAi Feeding Insect <br />  <br />  <br />                                                                                differences cuticle peritrophic matrix of <br />                                                                                midgut trachea compared controls (Figure S4); <br />                                                                                similar results observed larvae ingestion low <br />                                                                                dose dsSeCHSA (Figure S5). These results suggest only <br />                                                                                sufficient SeCHSA transcriptional level decreasing effect on <br />                                                                                normal histological structure S. exigua. <br />  <br />                                                                                Discussion <br />                                                                                Ingestion bacterially expressed SeCHSA dsRNA induces <br />                                                                                lethal phenotypes S. exigua larvae <br />                                                                                   In study, demonstrated ingestion bacterially <br />                                                                                expressed dsRNA specifically inhibit transcription a <br />                                                                                targeted gene lepidopteran insect S. exigua. Knockdown of <br />                                                                                target gene SeCHSA suppresses transcription levels, <br /> Figure 6. Effects ingesting bacterially expressed dsRNA                  affects larval growth development, leading lethal <br /> SeCHSA mRNA cuticle trachea. Total RNA extracted                phenotypes. A previous study lepidopteran Spodoptera litura <br /> from cuticle trachea tissues respectively feeding           using ingestion dsRNA did induce RNAi response [24]. <br /> bacterially expressed dsRNA (the high bacteria concentration, 2506) for <br /> 9 days, ddH2O bacteria containing DmWhite dsRNA (dsControl)             Moreover, oral delivery dsRNA larvae lepidopteran <br /> blank plasmid L4440 served controls. SeCHSA transcript detected         Epiphya postvittana did cause mortality, spite reduction in <br /> using RT-PCR, S. exigua b-actin gene used reference.                  transcript levels target gene [10]. Similarly, the <br /> doi:10.1371/journal.pone.0006225.g006                                          hemipteran Rhodnius prolixus, ingestion dsRNA (13 mg larva) <br />                                                                                salivary nitrophorin 2 gene reduced target gene <br /> found trachea (Figure 9E, F, G H). These results prove              expression significant changes salivary gland color <br /> that SeCHSA affects chitin synthesis cuticle. In addition,         [25]. More recently, food intake larvae orthopteran <br /> result supports previous suggestion apparent                  Gryllus bimaculatus fed dsRNA (6 mg larva) a <br /> SeCHSA expression midgut contamination                 nervous neuropeptide gene SK stimulated [11]. Also, <br /> the tracheae surrounding midgut [17]. However,                 ingestion artificial diet containing dsRNA midgut gene V- <br /> slight SeCHSA mRNA knockdown medium dose                          ATPase A larvae coleopteran insect western corn <br /> dsSeCHSA feeding larvae (Figure 5), didn&#226;&#8364;&#8482;t significant             rootworm (WCR) Diabrotica virgifera virgifera resulted larval <br />  <br />  <br />  <br />  <br /> Figure 7. Phenotypes Spodoptera exigua ingestion bacterially expressed dsRNA (high bacteria concentration, 2506). (A) <br /> The &#226;&#8364;&#732;&#226;&#8364;&#732;half-ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotypes 4th instar (left) 5th instar (right) larvae feeding bacterially expressed SeCHSA dsRNA. (B) A larva with <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;half-ecdysis&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype &#226;&#8364;&#732;&#226;&#8364;&#732;double heads&#226;&#8364;&#8482;&#226;&#8364;&#8482;. (C) The normal phenotype 4th instar larva fed control bacteria dsControl, L4440 ddH2O. <br /> (D) The enlarging &#226;&#8364;&#732;&#226;&#8364;&#732;double-heads&#226;&#8364;&#8482;&#226;&#8364;&#8482; (B). (E) The enlarging normal head (C). Scale bars represent 2 mm A, B C, 0.4 mm D E. <br /> doi:10.1371/journal.pone.0006225.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                                 July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                            RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 8. Effects ingested dsSeCHSA Spodoptera exigua survival developmental duration. (A) Survival rates insects at <br /> different developmental stages ingestion dsSeCHSA (the high bacteria concentration, 2506). Different letters developmental <br /> stage indicate significant difference survival rates (p,0.05, Duncan&#226;&#8364;&#8482;s test). (B) Developmental duration insects ingestion of <br /> dsSeCHSA. No significant difference ANOVA (p.0.05). All error bars represent standard deviation (n = 3). <br /> doi:10.1371/journal.pone.0006225.g008 <br />  <br /> stunting mortality; remarkably, transgenic corn plants                 plant expressed dsRNA specific cytochrome P450 gene <br /> engineered express WCR dsRNAs exhibited significant                   (CYP6AE14), transcript levels gene reduced in <br /> decreasing WCR feeding damage growing chamber assay              midgut larval growth retarded [8]. However, <br /> [7]. At time, research group reported that,          unclear insecticide phenotypic silencing can <br /> a lepidopteran cotton bollworm Helicoverpa armigera larvae fed        observed genes expressed midgut through <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      8                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                             RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> Figure 9. Effects ingested dsSeCHSA different tissues Spodoptera exigua larvae. Cuticle, trachea midgut peritrophic matrix <br /> were dissected abnormal larvae 9 days feeding diet containing bacterially-expressed SeCHSA dsRNA (dsSeCHSA, high <br /> bacteria concentration, 2506), larvae stage fed bacterially-expressed DmWhite dsRNA (dsControl), control L4440 bacteria <br /> or ddH2O. All tissues stained Hemotoxylin eosin (H &amp; E). The filled open red triangles A, B, C D represent exocuticle and <br /> endocuticle, respectively; red arrows I, J, K L represent peritrophic matrix midgut. Scale bar represents 50 mm. <br /> doi:10.1371/journal.pone.0006225.g009 <br />  <br />  <br />  <br /> ingestible dsRNA. We showed ingestion dsRNA gene                  We showed larval-larval ecdysis disrupted S. <br /> (SeCHSA) expressed midgut caused lethal            exigua larvae fed diet containing bacterially expressed SeCHSA <br /> larval phenotype.                                                            dsRNA. Abnormal larvae ingested dsSeCHSA failed to <br />    Our results suggest dsRNA introduced midgut              remove old cuticle completely 10% could <br /> larvae transported insect cells tissues;             remove head capsule (Figure 7). These phenotypes are <br /> process occur intercellular dsRNA transport                  similar observed insects treated acylureas [23]. As <br /> protein SID-1 [2,10]. SID-1 identified C. elegans,          insect growth regulators, acylureas selectively disrupt chitin <br /> been proven required spreading RNA interfering                     synthesis insects fungi, played a <br /> information tissues, usually leads systemic RNAi              significant role IPM (Integrated Pest Management) more <br /> [21]. To determine SID-1 exists S. exigua, cloned            30 years [16,18]. Bacteria expressing dsRNA used in <br /> partial sequence sid-1 like gene beet armyworm,            insect pest control near future, crude extracts of <br /> shares high identity (70%) B. mori sid-1 like 2 gene (GenBank         bacterially expressed dsRNA effectively protect plants virus <br /> accession no. AB327185) C. elegans sid-1 gene (46.8%)                    infections [26]. <br /> (GenBank accession no. NM_071971); consistent                      Closer examination cuticle abnormal S. exigua <br /> with phylogenetic relationship analysis SID-1 insects          larvae using H &amp; E staining revealed exo- endocuticle <br /> C. elegans (Figure 1A). The expression pattern analysis sid-1 gene        abnormal larvae disordered (Figure 9A). This result <br /> suggested expressed developmental stages                  suggests SeCHSA essential larval-larval growth and <br /> (Figure 1B), exhibited similar expression levels            development, observation partly agreement with <br /> the tissues cuticle, trachea, midgut fat body S. exigua       phenotypes observed injection dsRNA TcCHS1 the <br /> (Figure 1C). Existence systemic RNAi vital gene transcript           red flour beetle Tribolium castaneum [27]. However, did find <br /> in larval developmental stages tissues S. exigua         significant changes phenotype larval-pupal and <br /> account systemic RNAi effect insect ingestion       pupal-adult stages ingestion bacteria-expressed dsSeCHSA <br /> dsRNA. To knowledge, report ingestion          S. exigua (data shown). This observation differed from <br /> bacterially expressed non-midgut gene dsRNA cause lethal                 results T. castaneum, T. castaneum larvae injected with <br /> larval phenotypes lepidopteran insect.                                    dsRNA penultimate larval instar failed pupate, larvae <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                               July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                               RNAi Feeding Insect <br />  <br />  <br /> injected instars did complete pupal development,               combine efforts entomological research biotech- <br /> although larval-pupal molting initiated [27]. One possible                 nology industry [2]. In order approach successful, <br /> reason distinct changes phenotypes larval-pupal             suitable insect genes transgenic plants discovered. In <br /> development observed study dsRNA                   addition, necessary transgenic plants expressing insect <br /> ingested instar larval gut reached levels           dsRNA safe effective. Gene screening an <br /> high cause changes phenotype. Another possible                    important step process. <br /> explanation RNAi lepidopteran insects                    Chitin widespread natural amino polysaccharide. <br /> effective coleopteran beetle T. castaneum,                  Since chitin synthesis crucial growth development in <br /> dsRNA-mediated responses easily observed                   insects, chitin plants vertebrates, chitin <br /> injection little 5 pg dsRNA [28]. The dsRNA                 synthesis deemed ideal target novel insecticides <br /> injected lepidopteran insects various studies ranged               1970s [16,18]. Our results feeding S. exigua <br /> from 0.1 mg 6 mg larva [10,29,30] 20 mg pupa              dsSeCHSA potential act insecticide, and <br /> in silkworm B. mori [31].                                                  suggest chitin synthase SeCHSA ideal target gene pest <br />     Although transcription SeCHSA trachea down-              control. Already, transgenic plants expressing SeCHSA dsRNA <br /> regulated ingestion dsSeCHSA (Figure 6), structural                produced lab, safety testing transgenic <br /> difference trachea (Figure 9E, F, G H). One               plants vertebrates higher animals currently underway. <br /> possible reason trachea used cross section <br /> of main trachea tube originated spiracle, hardly              Materials Methods <br /> observing tracheole larvae using histological method, <br /> little knockdown SeCHSA transcript couldn&#226;&#8364;&#8482;t result                   Insect culture <br /> abnormal trachea structure. In addition,                    Larvae beet armyworm, S. exigua, obtained from <br /> amount tracheole abnormal larvae treated                        Nanjing Agricultural University reared laboratory <br /> dsSeCHSA normal larvae certain                      25uC62uC 75%65% relative humidity 14 hr-light/ <br /> degree. It suggests tracheole affected ingestion             10 hr-dark photoperiod using artificial diet [37]. <br /> dsRNA, consistent previous report tiny <br /> trachea defect observed using dsRNA injection method               Cloning sid-1 like gene fragment expression <br /> same gene [20].                                                                pattern analysis <br />                                                                                   Total RNA extracted 5th instar larvae S. exigua <br /> Delivery dsRNA feeding convenient reliable                      using Trizol reagent (Invitrogen) cDNA synthesized using <br /> RNAi method, especially gene screening                                     AMV reverse transcriptase XL (Takara) following <br />    Several methods delivery dsRNA siRNA                reaction conditions: 37uC 10 min, 42uC 1 h, 99uC for <br /> used recent years knock specific gene expression. Direct            5 min, 5uC 5 min. In order clone S. exigua sid-1 gene, <br /> microinjection remains popular method simple                  degenerate primers based conserved amino acids <br /> protocol effectiveness knocking expression target               FNHVFSN (forward primer) NFQFDT (reverse primer) were <br /> genes, including homothorax gene evenskipped gene                  used fragment sid-1 gene. The primers used as <br /> in cricket G. bimaculatus [32,33], chitin synthesis gene        follows: SID1F1 (59- TTY AAY CAY GTN TTY WSN AA-39, <br /> midgut mosquito Aedes aegypti [34], BgEcR genes            forward primer), SID1R1 (59-NTG RTC RAA YTG RAA <br /> hemimetabolous species Blattella germanica [35,36], bursicon             RTT- 39, reverse primer). PCR amplification primers <br /> gene silkworm B. mori [31]. However, difficult            SID1F1 SID1R1 performed following: 94uC for <br /> achieve RNAi injection small insects like aphids early              5 min, 5 cycles 94uC 40 s, 58uC 40 s 72uC 40 s, <br /> instar larvae nymphs, injection result high                    25 cycles 94uC 40 s, 55uC 40 s 72uC 40 s, and <br /> mortality. In addition, usually requires special techniques             extension step 72uC 10 min end. The <br /> equipment.                                                                     amplified fragment subcloned pMD-18T vector <br />    Delivery dsRNA feeding advantages                    (Takara) sequenced. To longer cDNA sequence, we <br /> microinjection feeding labor-intensive                performed 39-RACE using gene specific primers sid-1 and <br /> expensive extremely convenient performing RNAi                  anchor primers supplied SMART RACE cDNA Amplifica- <br /> a large number genes individuals [9]. Ingestion dsRNA                tion Kit (Clontech). For 39-RACE, gene specific primers that <br /> has proven confer lethal phenotypes silencing midgut-               designed based partial amplified fragment mentioned <br /> specific genes. Recent studies shown transgenic plants               above, forward primer nested primer were: SID1-GSP2 <br /> expressing insect dsRNA exploited control insect pests               (59-CTG GGA TAT GTG CTA CTA GGC CTG C-39), SID1- <br /> [2,7,8]. However, identification ideal genes inhibition              NGSP2 (59-GCT GGG TAT ACC ACA GCA TTT TGG CC- <br /> transgenic plants needed. Our results suggest delivery              39), respectively. <br /> dsRNA feeding convenient reliable method                          The methods total RNA extraction cDNA synthesis <br /> screening large number essential insect genes, reported             described above. For analysis sid-1 mRNA <br /> beetles termites [7,12].                                                   different developmental stages, total RNA extracted from <br />                                                                                1st 5th instars larvae, prepupae, pupae adult. For analysis of <br /> SeCHSA good target gene RNAi application pest                      sid-1 mRNA different tissues, RNA extracted the <br /> control                                                                        cuticle, trachea, midgut fat body 1 day 5th instar larvae. <br />    Insect-resistant transgenic plants commercial use              The primers used sid-1 expression pattern analysis were <br /> for 10 years. Theoretically, better use genes          forward primer SID1-GSP2 described reverse <br /> originate insects transgenes,                primer SID1-GSP1.1 (59- CAG GCA GGT GGC GAG GTG <br /> realized. Damage caused insect pests reduced plants              GAG GAC C-39) described, pair primers spanned a <br /> express insect dsRNA [7]. This presents opportunity                   nucleotide fragment 402 bp. Amplification reactions comprised <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         10                               July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                           RNAi Feeding Insect <br />  <br />  <br /> 30 cycles 94uC 401 s, 55uC 40 s, 72uC 40 s,           An unrelated gene D. melanogaster, white gene (DmWhite) <br /> a final extension step 72uC 10 min.                                 (GenBank accession no. X51749), selected use control <br />    The deduced amino acids S. exigua SID-1, insect SID-1          dsRNA. The recombinant plasmid DmWhite dsRNA expression <br /> and nematode C. elegans SID-1, deduced amino acids                  protocol dsSeCHSA. The PCR primers <br /> conserved domain aligned ClustalW. Phylogenetic                    used amplify fragment (688 bp) DmWhite follows: <br /> analysis conducted neighbour-joining method using              forward primer DmWhiteF (59-GG CTCGAG ATG GGC TAC <br /> MEGA version 4.0 [38]. Bootstrap values assessed 1000            CGG CGC CCA GGA AAC ATT-39, spanning nucleotides 322&#226;&#8364;&#8220; <br /> replicates. The deduced amino acid sequences insect SID-1           342; underlined letters indicate XhoI site) reverse primer <br /> C. elegans SID-1 obtained GenBank. GenBank accession             DmWhiteR (59-GG GCGGCCGC TAG GAA AAG AAG TCG <br /> numbers deduced sequences used following: S. exigua          ACG GCT TCG C-39, spanning nucleotides 1009&#226;&#8364;&#8220;985; under- <br /> SID-1 like 1, ACM47363; B. mori SID-1 like 1, BAF95805; B.                 lined letters indicate NtoI site). <br /> mori SID-1 like 2, BAF95807; B. mori SID-1 like 3, BAF95806; T. <br /> castaneum SID-1 related A, NP_001099012; T. castaneum SID-1                Feeding bioassays <br /> related B, NP_001103253; T. castaneum SID-1 related C,                        A randomized block design used feeding <br /> NP_001099128; A. mellifera SID-1 like, XP_395167; C.                   experiment. The artificial diet pellets cut sizes, the <br /> elegans SID-1, AAL78657.                                                   larger 20 mm610 mm62 mm 0.9 g in <br />                                                                            weight S. exigua neonates, 1st 2nd instars larvae, a <br /> Vector construction dsRNA preparation                                  smaller 10 mm610 mm62 mm 0.45 g in <br />    In order construct plasmid expresses dsRNA                    weight 3rd&#226;&#8364;&#8220;5th instars larvae. Each piece diet overlaid <br /> corresponding S. exigua chitin synthase A (SeCHSA, GenBank              50 ml suspension following: bacterial culture <br /> accession no. DQ062153), 635 bp fragment (dsSeCHSA)                  containing bacteria expressing dsRNA SeCHSA, bacterial <br /> amplified RT-PCR using total RNA template. The                     culture containing bacteria expressing control dsRNA for <br /> forward primer SeA3F (59-TCG CCATGG TTG TAT TCT                        DmWhite, plasmid L4440, 50 ml ddH2O. For experiments <br /> TCT TCG CCT TG-39), spans nucleotides 4156&#226;&#8364;&#8220;4175,                     employing different concentrations (2506, 506, 106) feeding <br /> and reverse primer SeA3R (59-GAC AAGCTT GTC                        protocols described above. Prior 3rd instar, <br />                                                                            S. exigua neonates larvae reared pieces diet <br /> GCC TAT GGT GGT TTC GT-39), spans nucleotides <br />                                                                            controlled chamber groups conditions as <br /> 4790&#226;&#8364;&#8220;4771. The underlined portions sequence NcoI and <br />                                                                            described above. Ten day 3rd instar larvae were <br /> HindIII restriction sites, respectively. Amplification reactions <br />                                                                            selected weight (about 0.0025 g 5 mm body <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>) from <br /> comprised 30 cycles 94uC 40 s, 58uC 40 s, 72uC <br />                                                                            experimental group control group was <br /> for 40 s, final extension step 72uC 8 min. PCR <br />                                                                            subsequently reared individually piece diet each <br /> products confirmed separation 1% agarose gels and <br />                                                                            condition. Each group replicated times. All diets were <br /> visualized ethidium bromide staining. The PCR product was <br />                                                                            replaced daily. Insect molting, survival, abnormal phenotype <br /> then cloned plasmid L4440 [4] NcoI                observed recorded daily individual insect until <br /> HindIII sites. The L4440 plasmid, provided Andrew             death. Five normally-developed larvae randomly selected the <br /> Fire (Stanford University, CA, USA) obtained Addgene              experimental control groups weighed daily using <br /> Inc. (Cambridge, MA, USA), T7 promoters inverted                electronic scale (0.0001 g) (Mettler Toledo, AB104-S, Switzer- <br /> orientation flanking multiple cloning site [13]. The resulting         land) 3rd instar day 5th instar day two. Data on <br /> recombinant vector L4440-SeA3 introduced competent                survival rate (arcsine square root transformed), weight and <br /> HT115(DE3) cells [4,9] lacking RNase III. The HT115(DE3)                   developmental duration analyzed one-way analysis of <br /> bacterium following genotype: F-, mcrA, mcrB, IN (rrnD-            variance (ANOVA). When treatment effects detected, <br /> rrnE)1, rnc14::Tn10 (DE3 lysogen: lavUV5 promoter -T7                      Duncan&#226;&#8364;&#8482;s test used determine significant <br /> polymerase); RNase III gene disrupted Tn10                     differences exist survival rates. <br /> transposon carrying tetracycline-resistance marker [9]. This                To confirm bacteria ingested larvae, FD&amp;C Blue <br /> bacterium induced express T7 polymerase                   (Lab Dr. Ehrenstorfer, Augsburg, Germany) used color the <br /> presence isopropyl b-D-thiogalactoside (IPTG). The                      bacteria overlaid surface insect artificial <br /> HT115(DE3) provided Lisa Timmons (University                     diet. Fourth instar day larvae allowed feed the <br /> Kansas, KS, USA) &amp; Andrew Fire obtained CGC                       colored diet. One day feeding diet containing FD&amp;C <br /> (Caenorhabditis Genetics Center, Minneapolis, MN, USA).                    Blue, larvae dissected midguts examined <br />    To produce dsRNA, single colonies HT115(DE3) bacteria                Carl Zeiss Axiostar plus microscope (Go&#194;&#168;ttingen, Germany). <br /> containing dsSeCHSA cloned L4440 grown 14 h               To determine colored bacteria accumulated the <br /> shaking LB 100 mg/ml ampicillin plus 12.5 mg/ml                    midgut, midguts dissected observation. All samples <br /> tetracycline 37uC. The culture diluted 100-fold 100 ml           photographed using Sony DSC-F717 digital camera. <br /> of 26YT medium allowed grow OD595 = 0.4. Synthesis <br /> of T7 polymerase induced addition IPTG 0.4 mM                 RT-PCR analysis confirm gene silencing <br /> and bacteria incubated shaking additional 4 h            To monitor transcriptional levels SeCHSA larvae after <br /> at 37uC. The expressed dsRNA extracted described                    feeding bacteria express dsRNA, total RNA was <br /> previously [4,26]. The length dsRNA confirmed                extracted experimental control larvae using Trizol <br /> electrophoresis 1% agarose gel. To prepare bacterial cells         reagent (Invitrogen) according manufacturer&#226;&#8364;&#8482;s instructions. <br /> express dsRNA, procedures described used.                   Each RNA sample (0.5&#226;&#8364;&#8220;1 mg) used cDNA synthesis a <br /> Bacterial cells collected 100 ml IPTG-induced culture            40 ml reaction mixture using following reaction conditions: <br /> by centrifugation 10,000 g 2 min, resuspended 0.4 ml             37uC 10 min, 42uC 1 h, 99uC 5 min, 5uC for <br /> (2506), 2 ml (506) 10 ml (106) sterile water, used         5 min. For PCR reactions, 2 ml S. exigua cDNA used a <br /> S. exigua feeding bioassays.                                               template 50 ml reactions using SeCHSA specific primers. The <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      11                               July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                             RNAi Feeding Insect <br />  <br />  <br /> forward primer SeA1F (59-TAC CCT CTA TGT ACT TGC                       post-feeding. Total RNA extracted individual larvae <br /> TT-39, spanning nucleotides 3280&#226;&#8364;&#8220;3299) reverse primer              feeding diet containing bacteria-expressed dsRNA of <br /> was SeA1R (59-AAA GAT AAT TCG GCG GGA CT-39,                               dsSeCHSA SeCHSA SeCHSB transcripts detected <br /> spanning nucleotides 3871&#226;&#8364;&#8220;3852). The PCR amplification condi-              using RT-PCR. RNA larvae fed diet containing the <br /> tions described above. In order determine              control bacteria DmWhite (dsControl), L4440 with <br /> whether RNAi SeCHSA gene affects expression S.               ddH2O served controls. The lane 1, 4 7 represent the <br /> exigua chitin synthase gene B, pair primers SeCHSB                high bacteria concentration (2506) feeding larvae,the lane 2, 5 <br /> (GenBank accession no. EU622827) designed follows:                 8 represent middle bacteria concentration (506) feeding <br /> forward primer SeCHSB1F (59-TTA CGT CAC CAT TCC CAG                        larvae, lane 3, 6, 9 represent low bacteria <br /> C-39, spanning nucleotides 3114&#226;&#8364;&#8220;3132), reverse primer                  concentration (106) feeding larvae. The house keeping gene b- <br /> SeCHSB1R (59-AAG TTA GTC TCT GCC GTC G-39,                                 actin used reference. <br /> spanning nucleotides 3656&#226;&#8364;&#8220;3638). The PCR amplification                Found at: doi:10.1371/journal.pone.0006225.s002 (0.25 MB TIF) <br /> primers SeCHSB1F SeCHSB1R performed follows: <br /> 94uC 5 min, 30 cycles 94uC 40 s, 55uC 40 s              Figure S3 The variable malformed pupae ingestion high <br /> 72uC 40 s; 72uC 10 min end. As control, S.        dose dsRNA Spodoptera exigua. The frenquency these <br /> exigua b-actin (GenBank accession no. AY507963) transcript             phenotypes nearly dsRNA feeding and <br /> amplified PCR using cDNA template actin-                   controls. A, B C variable phenotypes pupae that <br /> specific primers, Se-actinF (59-GGT TGG TAT GGG TCA GAA                    appeared group. D shows normal pupae. The scale bar <br /> GGA-39, spanning nucleotides 200&#226;&#8364;&#8220;220) Se-actinR (59-GCG                represents 2 mm. <br /> GTG GTG GTG AAA GAG TA-39, spanning nucleotides 676&#226;&#8364;&#8220;                       Found at: doi:10.1371/journal.pone.0006225.s003 (0.56 MB TIF) <br /> 657). The PCR reaction conditions                   Figure S4     The Effects ingestion bacterially-expressed <br /> described previously [17]. The RT-PCR products analyzed               dsRNA (the medium bacteria concentration, 506) cuticle, <br /> by 1% agarose gel electrophoresis gel stained             trachea midgut peritrophic matrix Spodoptera exigua <br /> ethidium bromide. To analyze SeCHSA transcriptional            larvae. Cuticle, trachea midgut peritrophic matrix were <br /> levels cuticle trachea ingestion high dose             dissected abnormal larvae 9 days feeding diet <br /> dsSeCHSA expressing bacteria days, total RNA                  containing bacterially-expressed SeCHSA dsRNA,or the <br /> extracted corresponding tissues methods                   larvae stage fed bacterially-expressed DmWhite <br /> described above. The primers PCR program used                dsRNA (Control), control L4440 bacteria ddH2O. All tissues <br /> to analysis SeCHSA expression levels using RT-PCR described             stained Hemotoxylin eosin (H &amp; E). The filled and <br /> above.                                                                     open red triangles A, B, C D represent exocuticle and <br />                                                                            endocuticle, respectively; red arrows I, J, K L represent <br /> Microscopy histological examination                                    peritrophic matrix midgut. Scale bar represents 50 mm. <br />    To determine impact gene silencing development            Found at: doi:10.1371/journal.pone.0006225.s004 (1.20 MB TIF) <br /> of S. exigua cuticle, trachea midgut peritrophic matrix, five <br /> abnormal larvae (4th instar) selected days feeding         Figure S5     The Effects ingestion bacterially-expressed <br /> on dsRNA-containing diet, number larvae               dsRNA (the low bacteria concentration, 106) cuticle, <br /> randomly selected control groups. These larvae               trachea midgut peritrophic matrix Spodoptera exigua <br /> fixed 4 h Carnoy (6 ethanol: 1 acetic acid: 3 chloroform),        larvae. Cuticle, trachea midgut peritrophic matrix were <br /> dehydrated ethanol series xylene, embedded              dissected abnormal larvae 9 days feeding diet <br /> in paraffin. The larvae sectioned 7 mm using Baffalo             containing bacterially-expressed SeCHSA dsRNA,or the <br /> microtome (American Optical Company, New York, USA)                    larvae stage fed bacterially-expressed DmWhite <br /> stained hematoxylin eosin (H &amp; E). All samples               dsRNA (Control), control L4440 bacteria ddH2O. All tissues <br /> examined Olympus research inverted microscope               stained Hemotoxylin eosin (H &amp; E). The filled and <br /> IX71 connected color digital camera Olympus DP71. Digital             open red triangles A, B, C D represent exocuticle and <br /> images captured using Image Pro Plus (Media Cybernetics,              endocuticle, respectively; red arrows I, J, K L represent <br /> Silver Spring, MD) processed using Microsoft Office Picture            peritrophic matrix midgut. Scale bar represents 50 mm. <br /> Manager (Microsoft Corporation, USA) Adobe Photoshop CS                Found at: doi:10.1371/journal.pone.0006225.s005 (1.19 MB TIF) <br /> to enhance contrast brightness. <br />                                                                            Acknowledgments <br /> Supporting Information <br />                                                                            We grateful Addgene Inc. (Cambridge, MA, USA) kindly <br /> Figure S1 Multiple alignment Spodoptera exigua SID-1                supplying plasmid L4440; Caenorhabditis Genetics Center (CGC, <br /> other insects nematode Caenorhabditis elegans SID-1 translat-           Minneapolis, MN, USA) friendly providing bacteria strain <br /> ed amino acid sequences. S. exigua SID-1 like 1, GenBank                   HT115(DE3), funded NIH National Center Research <br />                                                                            Resources (NCCR); Prof. Shuanglin Dong (Nanjing Agricultural Univer- <br /> accession no. ACM47363; B. mori SID-1 like 1, BAF95805; B. <br />                                                                            sity) help insect culture; Dr. Xiaoqiang Yu (University of <br /> mori SID-1 like 2, BAF95807; B. mori SID-1 like 3, BAF95806; T.            Missouri-Kansas City, MO, USA.) Dr. Sheng Li (Institute Plant <br /> castaneum SID-1 related A, NP_001099012; T. castaneum SID-1                Physiology Ecology, Chinese Academy Sciences) critical <br /> related B, NP_001103253; T. castaneum SID-1 related C,                     reading manuscript. <br /> NP_001099128; A. mellifera SID-1 like, XP_395167; C. <br /> elegans SID-1, AAL78657.                                                   Author Contributions <br /> Found at: doi:10.1371/journal.pone.0006225.s001 (1.21 MB TIF) <br />                                                                            Conceived designed experiments: HT QX WZ. Performed the <br /> Figure S2 Effects ingestion bacteria-expressed dsRNA              experiments: HT HP QY HC. Analyzed data: HT QY HC BT. Wrote <br /> transcription SeCHSA SeCHSB day 3 (A) day 5 (B)              paper: HT WZ. <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      12                                July 2009 | Volume 4 | Issue 7 | e6225 <br />                                                                                                                                                            RNAi Feeding Insect <br />  <br />  <br />  <br />  <br /> References <br />  1. Kooter JM, Matzke MA, Meyer P (1999) Listening silent genes: transgene              21. Winston WM, Molodowitch C, Hunter CP (2002) Systemic RNAi C. elegans <br />     silencing, gene regulation pathogen control. Trends Plant Sci 4: 340&#226;&#8364;&#8220;347.                  requires putative transmembrane protein SID-1. Science 295: 2456&#226;&#8364;&#8220;2459. <br />  2. Gordon KHJ, Waterhouse PM (2007) RNAi insect-proof plants. Nat Biotech                 22. Gura T (2000) A silence speaks volumes. Nature 404: 804&#226;&#8364;&#8220;808. <br />     25: 1231&#226;&#8364;&#8220;1232.                                                                             23. Arthur Retnakaran, James EWright (1987) Control insect pests with <br />  3. Orii H, Mochii M, Watanabe K (2003) A simple &#226;&#8364;&#732;&#226;&#8364;&#732;soaking method&#226;&#8364;&#8482;&#226;&#8364;&#8482; RNA                        benzoylphenyl ureas. In: James E. Wright, ed. Chitin Benzoylphenyl <br />     interference planarian Dugesia japonica. Dev Genes Evol 213: 138&#226;&#8364;&#8220;141.                   Ureas. Dordrecht: Kluwer Academic Publishers. pp 205&#226;&#8364;&#8220;282. <br />  4. Timmons L, Court D, Fire A (2001) Ingestion bacterially expressed dsRNAs                24. Rajagopal R, Sivakumar S, Agrawal N, Malhotra P, Bhatnagar RK (2002) <br />     produce specific potent genetic interference Caenorhabditis elegans.                Silencing midgut Aminopeptidase N Spodoptera litura double-stranded <br />     Gene 263: 103&#226;&#8364;&#8220;112.                                                                             RNA establishes role Bacillus thuringiensis toxin receptor. J Biol Chem 277: <br />  5. Britt-Louise L, Eugene IS, Jan B, Elisabeth W, Petra vR, et al. (2006) dsRNA-                  46849&#226;&#8364;&#8220;46851. <br />     mediated resistance Beet Necrotic Yellow Vein Virus infections sugar beet            25. Araujo RN, Santos A, Pinto FS, Gontijo NF, Lehane MJ, et al. (2006) RNA <br />     (Beta vulgaris L. ssp. vulgaris). Mol Breeding 18: 313&#226;&#8364;&#8220;325.                                    interference salivary gland nitrophorin 2 triatomine bug Rhodnius <br />  6. Niu QW, Lin SS, Reyes JL, Chen KC, Wu HW, et al. (2006) Expression                          prolixus (Hemiptera: Reduviidae) dsRNA ingestion injection. Insect <br />     artificial microRNAs transgenic Arabidopsis thaliana confers virus resistance.              Biochem Mol Biol 36: 683&#226;&#8364;&#8220;693. <br />     Nat Biotech 24: 1420&#226;&#8364;&#8220;1428.                                                                 26. Tenllado F, Martinez-Garcia B, Vargas M, az-Ruiz J (2003) Crude extracts of <br />  7. Baum JA, Bogaert T, Clinton W, Heck GR, Feldmann P, et al. (2007) Control                   bacterially expressed dsRNA used protect plants virus <br />     coleopteran insect pests RNA interference. Nat Biotech 25: 1322&#226;&#8364;&#8220;1326.                  infections. BMC Biotech 3: 3. <br />  8. Mao YB, Cai WJ, Wang JW, Hong GJ, Tao XY, et al. (2007) Silencing cotton                 27. Arakane Y, Muthukrishnan S, Kramer KJ, Specht CA, Tomoyasu Y, et al. <br />     bollworm P450 monooxygenase gene plant-mediated RNAi impairs larval                         (2005) The Tribolium chitin synthase genes TcCHS1 TcCHS2 specialized <br />     tolerance gossypol. Nat Biotech 25: 1307&#226;&#8364;&#8220;1313.                                <br /> </body></html>