<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Dec. 2007, p. 8575&#226;&#8364;&#8220;8583                                                                               Vol. 189, No. 23 <br /> 0021-9193/07/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00653-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />      Reassessment In Vivo Functions DNA Polymerase I and <br />                   RNase H Bacterial Cell Growth&#228;&#338;&#164;&#226;&#8364;&#160; <br />  Sanae Fukushima,1 Mitsuhiro Itaya,2 Hiroaki Kato,1 Naotake Ogasawara,3 Hirofumi Yoshikawa1* <br />  Department Bioscience, Tokyo University Agriculture, Sakuragaoka 1-1-1, Setagaya-ku, Tokyo 156-8502, Japan1; Institute for <br />         Advanced Biosciences, Keio University, 403-1, Nipponkoku, Daihoji, Tsuruoka, Yamagata 997-0017, Japan2; and <br />             Department Bioinformatics Genomics, Graduate School Information Science, Nara Institute of <br />                            Science Technology, Takayama 8916-5, Ikoma, Nara 630-0101, Japan3 <br />                                             Received 26 April 2007/Accepted 18 September 2007 <br />  <br />              A major factor removing RNA primers processing Okazaki fragments DNA polymerase <br />           I (Pol I). Pol I thought remove RNA primers resulting gaps simultaneously. RNase H, <br />           encoded rnh genes, factor removing RNA primers, disagreement respect <br />           essentiality polA rnh genes. In previous study, looked synthetic lethality of <br />           paralogs Bacillus subtilis detected essential doublet paralogs, including polA ypcP pair. YpcP <br />           consists 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain. In current study, confirmed polA genes of <br />           Escherichia coli B. subtilis completely deleted. We 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity <br />           encoded polA ypcP xni required growth B. subtilis E. coli. Also, 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; <br />           exonuclease activity Pol I indispensable cyanobacterium Synechococcus elongatus. These results <br />           suggest 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity essential organisms. Our success constructing B. subtilis <br />           strain lacked RNase H genes indicates enzymatic activity dispensable, wild <br />           type. Increasing 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity partially compensated defective <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> RNase <br />           H-deficient mutant, suggesting cooperative functions enzyme systems. Our search distri- <br />           bution 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain 250 bacterial genomes resulted finding eubacteria, <br />           archaea, possess domain. <br />  <br />  <br />    Structural characterization biochemical studies sev-                 determination RNA primer <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 9 12 nucleotides <br /> eral prokaryotic DNA polymerase I (Pol I, PolA) estab-                      (20). Later progress revealed enzymes variety <br /> lished organization functional domains: N-ter-                  prokaryotic eukaryotic organisms classified <br /> minal domain associated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity,                   major families, type 1 type 2 RNase H (31). E. coli <br /> central domain mediates proofreading 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonu-                  RNase HI, encoded rnhA (17), belongs type 1 en- <br /> clease activity, C-terminal domain responsible                   zymes; E. coli RNase HII, encoded rnhB (14), Bacillus <br /> polymerase activity (33). In combination activities                   subtilis RNase HII (rnhB) HIII (rnhC) (30) categorized <br /> derived terminal domains, so-called nick trans-                  type 2 enzymes. Biochemical characterization several <br /> lation activity, Pol I thought act simultaneously               enzymes suggested type 1 type 2 RNase H func- <br /> removing RNA primers filling resulting gaps. Evi-                   tionally related. <br /> dently, Escherichia coli polA1 mutant (7), lacks poly-                  Previous studies disagreed regarding essentiality both <br /> merase activity 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity, able                polA rnh genes. The polA(Ex1) mutant described <br /> grow, accumulated Okazaki fragments, prob-                    exhibited temperature-sensitive growth, indicating re- <br /> ably inability gaps (32). In addition,                  quirement 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease catalytic domain Pol I. <br /> polA(Ex1) mutant, lacks 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity                    Moreover, polymerase 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain <br /> 43&#194;&#176;C, accumulated Okazaki fragments grow                    required growth rich media (16). On hand, <br /> at high temperatures (23).                                                     E. coli genome analysis, complete <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>knockout</span> strain <br />    RNase H specifically cleaves RNA strand RNA/DNA                      polA grew normally LB medium obtained, re- <br /> hybrids plays role removing RNA primers Okazaki                    vealing dispensability polA (28). However, have <br /> fragments (29), process ribonucleoti-                 descriptions molecular mechanism(s) involved <br /> des DNA-RNA junction sites. In fact, isolation                 processing RNA primers polA mutants, although <br /> a double mutant polA rnh (which encodes RNase H)                        studies involving work polA rnh double mutant <br /> made possible detect RNA primers contributed                  described support initiation replication <br />                                                                                plasmids chromosomes. Together, rnhA and <br />                                                                                rnhB genes dispensable E. coli presence polA; <br />    * Corresponding author. Mailing address: Department Bioscience,          hand, rnhB rnhC genes B. subtilis <br /> Tokyo University Agriculture, Sakuragaoka 1-1-1 Setagaya-ku, Tokyo          manifested synthetic lethality (15). <br /> 156-8502, Japan. Phone: 81-3-5477-2758. Fax: 81-3-5477-2668. E-mail:              A number microbial-genome-sequencing projects re- <br /> hiyoshik@nodai.ac.jp. <br />    &#226;&#8364;&#160; Supplemental material article http://jb <br />                                                                                vealed bacterial genomes consist half singlet <br /> .asm.org/.                                                                     genes half paralogous genes (24). Our comprehensive <br />    &#228;&#338;&#164; <br />      Published ahead print 28 September 2007.                            study, looked synthetic lethality paralogs in <br />  <br />                                                                         8575 <br />  8576       FUKUSHIMA ET AL.                                                                                                                             J. BACTERIOL. <br />  <br />  <br /> B. subtilis, detected essential functions doublet para-                    PCR-generated fragment containing antibiotic gene marker (primers -For <br /> logs (Y. Shiwa, S. Fukushima, H. Yoshikawa, unpublished                           -Rev). In case E. coli, strain KM22 used host dis- <br />                                                                                       ruptants obtained homologous recombination provided &#226;?&#173; red func- <br /> data); polA ypcP pair. The YpcP protein                         tion (27) PCR-generated fragments. Fragments containing antibiotic resis- <br /> categorized 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease (http://bacillus.genome.jp/);                      tance gene markers amplified pair 5&#226;&#172;&#732; 3&#226;&#172;&#732; primers 40 <br /> we considered attempt resolve discrepancy                          bases upstream downstream target gene attached (see Table S1 <br /> that polA dispensable 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity                       supplemental material). The PCR products directly used trans- <br />                                                                                       form KM22. The templates antibiotic markers B. subtilis Tn554 <br /> is essential function maturation Okazaki frag- <br />                                                                                       Staphylococcus aureus spectinomycin (spc) resistance; pC194 S. aureus <br /> ments. The essentiality 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease function B.                       chloramphenicol resistance (cat); pMUTIN-T3CC (K. Asai, unpublished <br /> subtilis suggested Duigou et al. (9), reported                        data), derivative pMUTIN4 (42) erythromycin (erm) resistance; and <br /> less efficient transformation &#226;&#338;&#172;polA strain partially                      pBEST309 (13) Enterococcus faecalis tetracycline (tet) resistance. The <br /> deleted ypcP gene pMUTIN disruptant DNA. In de-                                templates markers E. coli R100 plasmid tetracycline (tet) resis- <br />                                                                                       tance synthetic plasmid pHSG399 chloramphenicol (cat) resistance. <br /> tailed analysis reported here, used complete disruptants of <br />                                                                                       Plasmid pUC4K (Pharmacia) used confer kanamycin (kan) resistance on <br /> entire genes domains genes inducible gene                             Synechococcus. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> generate desired PCR construct, used recombinant <br /> expression system.                                                                    PCR method (12), secondary-stage PCR products generated <br />    In addition rnh genes described above,                        three-piece primary PCR fragments. All constructs gene disruptions <br /> two genes <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> RNase H B. subtilis                               verified diagnostic PCR assay. The PCR primers used listed in <br />                                                                                       Table S1 supplemental material. Pyrobest DNA polymerase ExTaq <br /> genome. One these, ypdQ, similar sequence type 1                          DNA polymerase (TaKaRa, Shiga, Japan) used PCR. <br /> enzyme genes, reported lack critical                        Construction strains inducible expression system. The B. subtilis <br /> amino acids active-site residues purified                         strain harboring ypcP control IPTG-inducible promoter was <br /> protein exhibits RNase H activity (30). To surprise,                        constructed follows. A 321-bp region containing ribosomal binding se- <br />                                                                                       quences 5&#226;&#172;&#732; terminus ypcP amplified PCR primer pair <br /> additional RNase H gene, ypeP, B. subtilis <br />                                                                                       (see Table S1 supplemental material) and, digestion HindIII and <br /> genome reassignment open reading frame (S. Ish-                             BamHI, cloned HindIII/BamHI sites pMUTIN-T3CC. The resulting <br /> ikawa, personal communication). In current study,                        plasmid used transform B. subtilis strain 168 erythromycin resistance to <br /> confirmed polA genes E. coli B. subtilis                         obtain strain NBS5014. Consequently, integrant, ypcP placed under <br /> could completely deleted. Then, demonstrated                             control IPTG-inducible spac promoter, promoter activity of <br />                                                                                       ypcP itself monitored &#226;?&#164;-galactosidase activity (41). Alternatively, <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity indispensable B. subtilis                    B. subtilis strain harboring polA, N-terminal region polA, ypcP under <br /> E. coli cyanobacteria, Synechococcus                        control IPTG-inducible promoter Phyper-spank (derived pDR111a) <br /> elongatus strain PCC 7942, suggesting essential function                           amyE locus &#226;&#338;&#172;polA mutant, constructed. The fragments con- <br /> common organisms. Finally, constructed B. subtilis                        taining full-length polA, 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA, ypcP were <br />                                                                                       amplified PCR primer pairs cloned pDG148 and, after <br /> strain depleted RNase H genes. Our findings indicate <br />                                                                                       digestion XhoI SalI HindIII, cloned SalI/HindIII sites of <br /> that RNase H enzymatic activity dispensable,                       pDR111a (gift M. Fujita, University Houston, Houston, TX) (5, 10). The <br /> the wild type. Based findings, discuss                      resulting plasmids used transform B. subtilis strain NBS182 obtain <br /> cooperation 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease RNase H functions                           strain NBS247-249. <br /> in vivo.                                                                                 SOS induction assay. As indicator SOS induction, expression yneA <br />                                                                                       analyzed transcriptionally fused lacZ reporter assay. In control <br />                                                                                       experiments, strain YNEAd grown optical density 600 nm (OD600) <br />                                                                                       0.35 LB medium 37&#194;&#176;C treated 50 ng/ml <br />                         MATERIALS AND METHODS <br />                                                                                       mitomycin C 10 min. The mutagen-treated cells harvested resus- <br />    Bacterial strains, plasmids, genetic technique. The bacterial strains used     pended fresh medium, culture continued. At indicated times, <br /> in study listed Table 1. B. subtilis strains maintained Schaef-   aliquots withdrawn &#226;?&#164;-galactosidase activity measured. Simulta- <br /> fer&#226;&#8364;&#8482;s sporulation medium (37) transformed chromosome plasmid              neously, NBS218 grown absence mitomycin C; culture <br /> DNA according method Anagnostopoulos Spizizen (1). Selection,           reached OD600 withdrawn YNEAd, aliquots assayed for <br /> when required, erythromycin (1 &#226;?&#174;g/ml); kanamycin, chloram-             &#226;?&#164;-galactosidase activity. <br /> phenicol, neomycin (5 &#226;?&#174;g/ml); tetracycline (15 &#226;?&#174;g/ml); spectino-            Bioinformatics. All genome sequences obtained GenomeNet <br /> mycin (100 &#226;?&#174;g/ml). When necessary, genes overexpressed cloning            (http://www.genome.ad.jp/); domain structures predicted SMART <br /> them multicopy plasmid pDG148 (40) derivative, pAN18,             program (http://smart.embl-heidelberg.de/; 25, 38). The hidden Markov model <br /> contains multiple cloning sites pUC18 HindIII site pDG148         (HMM) profiles 3_5_exonuclease (PF01612) DNA_pol_A (PF00476) <br /> induced 1 mM IPTG (isopropyl-&#226;?&#164;-D-thiogalactopyranoside). The lacZ ex-            obtained Pfam (http://www.sanger.ac.uk/Software/Pfam/). We built a <br /> pression experiment performed growing B. subtilis cells 2&#226;&#171;&#187; SG medium        custom profile HMM 5_3_exonuclease based multiple sequence align- <br /> (2) 37&#194;&#176;C, aliquots culture subjected &#226;?&#164;-galactosidase activity   ment 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain generated SMART program <br /> assay previously described (2).                                                    (SMART accession no. SM00475). Motif searches genomes per- <br />    E. coli K-12 strain KM22, derived AB1157 (27), grown LB                formed using HMMER program; e-value cutoff 0.01. <br /> medium supplemented 25 &#226;?&#174;g/ml kanamycin, necessary, 20 &#226;?&#174;g/ml <br /> chloramphenicol 25 &#226;?&#174;g/ml tetracycline added. Transformation carried <br /> out described previously (27). <br />    The cyanobacterium S. elongatus PCC 7942 grown BG-11 (6) supple- <br />                                                                                                                         RESULTS <br /> mented, necessary, 10 &#226;?&#174;g/ml kanamycin. Transformation Synecho- <br />                                                                                         The 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain polA ypcP es- <br /> coccus according method Porter (34). We designed primer <br /> sequences based information contained database closely          sential B. subtilis. In genome B. subtilis, ypcP was <br /> related strain S. elongatus PCC 6301 (http://cyano.genome.jp/).                       paralog polA; initial attempts obtain a <br />    Construction strains carrying various deletion mutations. Gene disruptant       double-knockout mutant genes failed. Therefore, <br /> strains B. subtilis S. elongatus PCC 7942 constructed homologous      constructed polA null mutant strain NBS5015, which <br /> recombination PCR-generated fragments. The primary PCR-generated frag- <br /> ments contained 600 1,000 bp upstream (using primers &#226;&#171;&#186;11 and <br />                                                                                       ypcP placed IPTG-inducible promoter, shown <br /> &#226;&#171;&#186;12) (see Table S1 supplemental material) downstream (primers &#226;&#171;&#186;23          Fig. 1A. NBS5015 grow medium containing <br /> and &#226;&#171;&#186;24) sequences target gene, overlap end        IPTG; Fig. 1B shows growth defect IPTG depletion. <br />  VOL. 189, 2007                                                        REASSESSMENT OF Pol I AND RNase H FUNCTIONS                 8577 <br />  <br />  <br />                                 TABLE 1. Genotype bacterial strains plasmids used study <br />         Strain plasmid                                            Genotype                                          Source <br />  <br /> B. subtilis strains <br />   168                                 trpC2                                                                    Laboratory stock <br />   NBS5004                             trpC2   polA::spc                                                        This study <br />   NBS182                              trpC2   polA::erm                                                        This study, <br />   POLAd                               trpC2   pMUTIN-polA erm                                                  22 <br />   NBS5014                             trpC2   pMUTIN-T3CC-ypcP erm                                             This study <br />   NBS5015                             trpC2   polA::spc pMUTIN-T3CC-ypcP erm                                   This study, NBS50143 <br />                                                                                                                  NBS5004 <br />   NBS4070                             trpC2   ypcP::spc                                                        This study <br />   NBS183                              trpC2   polAC::cat                                                       This study <br />   NBS184                              trpC2   polAN::spc                                                       This study <br />   NBS185                              trpC2   polAC::cat ypcP::spc                                             This study, NBS1833 <br />                                                                                                                  NBS4070 <br />   NBS202                              trpC2 rnhB::spc                                                          This study <br />   BEST218 1A1plus                     rnhB21::neo                                                              15 <br />   NBS203                              trpC2 rnhB21::neo                                                        This study, BEST218 <br />                                                                                                                   1A1 plus3168 <br />   NBS204                              trpC2   rnhC::cat                                                        This study <br />   NBS205                              trpC2   ypdQ::spc                                                        This study <br />   NBS206                              trpC2   ypeP::tet                                                        This study <br />   NBS207                              trpC2   rnhB::spc rnhC::cat                                              This study, NBS2043 <br />                                                                                                                   NBS202 <br />   NBS208                              trpC2 rnhB::spc ypeP::tet                                                This study, NBS2063 <br />                                                                                                                   NBS202 <br />   NBS209                              trpC2 rnhC::cat ypeP::tet                                                This study, NBS2043 <br />                                                                                                                   NBS206 <br />   NBS210                              trpC2 rnhB::spc rnhC::cat ypeP::tet                                      This study, NBS2043 <br />                                                                                                                   NBS208 <br />   NBS216                              trpC2 rnhB21::neo rnhC::cat ypeP::tet ypdQ::spc                          This study, NBS2043 <br />                                                                                                                   NBS224 <br />   NBS217                              trpC2 rnhB21::neo rnhC::cat ypeP::tet ypdQ::spc polA::erm                This study, NBS1823 <br />                                                                                                                   NBS216 <br />   NBS218                              trpC2 rnhB::spc rnhC::cat ypeP::tet pMUTIN-yneA erm                      This study, YNEAd3 <br />                                                                                                                   NBS210 <br />   NBS223                              trpC2 rnhB21::neo ypdQ::spc                                              This study, NBS2053 <br />                                                                                                                   NBS203 <br />   NBS224                              trpC2 rnhB21::neo ypeP::tet ypdQ::spc                                    This study, NBS2063 <br />                                                                                                                   NBS223 <br />   YNEAd                               trpC2 pMUTIN-yneA erm                                                    22 <br />   NBS247                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-polA spc::&#226;&#172;&#732;amyE                           This study, pDR111a- <br />                                                                                                                   polA3NBS182 <br />   NBS248                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-polA-N spc::&#226;&#172;&#732;amyE                         This study, pDR111a- <br />                                                                                                                  polAN3NBS182 <br />   NBS249                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-ypcP spc::&#226;&#172;&#732;amyE                           This study, pDR111a- <br />                                                                                                                  ypcP3NBS182 <br />  <br /> E. coli strains <br />   KM22                                argE3 his-4 leuB6 proA2 thr-1 ara-14 galK2 lacY1 mtl-1 xyl-5 thi-1       27 <br />                                         rpsL31 tsx-33 supE44 &#226;&#338;&#172;(recC ptr recB recD)::Plac-bet exo kan <br />   NBS198                              polA::cat                                                                This study <br />   NBS200                              xni::tet                                                                 This study <br />  <br /> S. elongatus PCC 7942 strains <br />    NBC100                             Wild type                                                                Laboratory stock <br />    NBC318                             &#226;&#338;&#172;polAC::kan                                                              This study <br />  <br />  <br /> Plasmids <br />   pDG148                              bla   kan                                                                40 <br />   pAN18                               bla   kan                                                                This study <br />   pMUTIN-T3CC                         bla   erm                                                                Kei Asai <br />   pDR111a                             bla   spc                                                                Masaya Fujita <br />  <br />  <br />  <br />   Although ypcP identified polA paralog, encoded          terminal domains PolA (designated &#226;&#338;&#172;polAC) &#226;&#338;&#172;ypcP <br /> small protein corresponded 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease           background. We succeeded constructing strain NBS185. Our <br /> domain PolA; enzymatic activity appeared re-              results indicated ypcP 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> quired viability. To verify this, tried delete C-         domain PolA sufficed cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>survival</span>. <br />  8578       FUKUSHIMA ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />        TABLE 2. Test sensitivity B. subtilis strains                   treatment (data shown). This contrast E. coli polA <br />                           mitomycin Ca                                              (43). It noted global mobility shift assay and <br />   Strain                   Relevant genotype                  Surviving fraction    DNA microarray analysis identified LexA binding site the <br />                                                                                     upstream region B. subtilis polA (3). <br /> 168                            Wild type                             0.54 <br /> NBS5004                        &#226;&#338;&#172;polA                             2.2 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br />                                                                                        xni E. coli counterpart ypcP. The E. coli ortholog of <br /> NBS4070                        &#226;&#338;&#172;ypcP                                 0.27           ypcP xni; reported encode exonuclease <br />   a <br />                                                                                     (http://genolist.pasteur.fr/Colibri/). Based results with <br />     Cells grown exponentially (optical density 600 nm, 0.4) 37&#194;&#176;C LB <br /> medium supplemented appropriate antibiotics. Mitomycin C sensitivity       B. subtilis, tested synthetic lethality polA and <br /> tested plating serial dilutions culture LB LB supplemented       xni applied E. coli. First, constructed polA xni <br /> with 50 ng/ml mitomycin C.                                                          disruptant E. coli KM22 independently. Then, trans- <br />                                                                                     formed xni disruptant &#226;&#338;&#172;polA chromosomal DNA to <br />                                                                                     obtain double-knockout strain; however, effort failed. <br />    Characterization polA ypcP mutants ex-                          The phenotype polA disruptant constructed was <br /> pression profiles. The E. coli polA null mutant manifests tem-                      similar reported Joyce Grindley (16) that <br /> perature-sensitive growth (16). We analyzed growth pro-                         grew LB medium rendered competent <br /> files various B. subtilis polA mutants combination                   slow growth minimal medium. No distinct <br /> ypcP mutant. As shown Fig. 2B, entire polA disruptant                        transformant obtained, control experiments in <br /> (&#226;&#338;&#172;polA) exhibited temperature-sensitive growth 56.5&#194;&#176;C                       transformation wild-type strain using DNA <br /> in B. subtilis. On hand, &#226;&#338;&#172;ypcP mutant able                     yielded transformants. For example, <br /> grow temperature, wild type. Interestingly,                      used fragment containing polA::cat cassette to <br /> the polA mutant (&#226;&#338;&#172;polAN) lacked N-terminal                            transform KM22, obtained approximately 100 transfor- <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain temperature sensitive (Fig.                       mants 1 &#226;?&#174;g DNA. On hand, used the <br /> 2B, NBS184); true mutant lacking Klenow                       DNA fragment transform NBS200 (xni::tet), few <br /> fragment (&#226;&#338;&#172;polAC; NBS183). Moreover, growth phenotype                           colonies appeared confirmed disrupted both <br /> of strain &#226;&#338;&#172;polAC &#226;&#338;&#172;ypcP                        polA xni genes. However, recipient cells yielded a <br /> that wild type. Together, results demonstrated                    normal number (&#226;&#172;&#381;100) transformants genetic <br /> the 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA necessary cell                         markers. Therefore, postulated transformants <br /> growth high temperatures.                                                        suppressors concluded polA xni exhibited <br />    The ypcP gene encodes 296 amino acid residues                          synthetic lethality. Our results indicated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> 70% amino acid sequence homology 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease                         activities encoded polA xni essential in <br /> domain Pol I. As described above, lack polA 5&#226;&#172;&#732;-                           E. coli. <br /> terminal region gave rise temperature-sensitive cell growth,                        The N-terminal domain cyanobacterial polA indispens- <br /> while &#226;&#338;&#172;ypcP did not, suggesting natures                       able. The genome cyanobacterium S. elongatus PCC <br /> enzymes different. As shown Fig. 3, expression                          7942 does encode YpcP ortholog. Therefore, exam- <br /> profiles genes similar. Note expression                    ined essentiality polA (Synpcc7942_0194). As described <br /> level ypcP-lacZ &#226;&#338;&#172;polA background slightly higher                      Materials Methods, attempted construct en- <br /> than wild type; reflect compensation the <br /> lack 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity. Consequently, con- <br /> structed polA- ypcP-inducible strain amyE locus. <br /> As shown Fig. 2C, &#226;&#338;&#172;polA background (NBS182), <br /> strains harboring polA N-terminal fragment suppressed the <br /> temperature sensitivity (Fig. 2C, c d), did control strains <br /> carrying full-length polA (Fig. 2C, b). On hand, <br /> ypcP-expressing strains (Fig. 2C, e f) unable sup- <br /> press phenotype &#226;&#338;&#172;polA mutant completely. Note <br /> that ypcP-expressing strain partially suppressed tem- <br /> perature sensitivity temperature (Fig. 2C, e f). <br /> These results confirmed temperature sensitivity was <br /> due lack 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA and <br /> that enzymes differed slightly nature. As described <br /> below, multicopy ypcP, polA, partially suppressed the <br /> RNase H deficiency; thus, enzymes share func- <br /> tions, natures differ somewhat. <br />    Another function Pol I repair enzyme SOS                         FIG. 1. Growth defect polA ypcP double mutant. (A) Scheme <br /> response. Presumably, function derived Klenow                      showing gene structures polA ypcP loci NBS5015. <br /> fragment Pol I YpcP involved. As indicated                         (B) Growth profiles strain NBS5015. NBS5015 streaked LB <br /> Table 2, &#226;&#338;&#172;polA strain showed profound sensitivity                        plates containing 0.01 mM IPTG incubated overnight 37&#194;&#176;C. LB <br />                                                                                     medium (open circles) (closed circles) 1 mM IPTG <br /> DNA-damaging agent mitomycin C; reaction &#226;&#338;&#172;ypcP                           inoculated diluted aliquots overnight culture (starting <br /> strain comparable wild type. In addition,                    OD600 &#226;&#171;&#189; 0.008) incubated 37&#194;&#176;C. Growth monitored OD <br /> expression of genes induced mitomycin C                       measurements. <br />  VOL. 189, 2007                                                          REASSESSMENT OF Pol I AND RNase H FUNCTIONS                     8579 <br />  <br />  <br />  <br />  <br />    FIG. 2. Growth various polA ypcP mutants LB plates 37&#194;&#176;C (A) 56.5&#194;&#176;C (B). Relevant genotypes shown: &#226;&#338;&#172;polA, polA deletion <br /> strain NBS5004; &#226;&#338;&#172;ypcP, ypcP deletion strain NBS4070; &#226;&#338;&#172;polAC, polA Klenow fragment deletion strain NBS183; &#226;&#338;&#172;polAN, polA 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> domain deletion strain NBS184; &#226;&#338;&#172;ypcP, &#226;&#338;&#172;polAC, Klenow fragment ypcP deletion strain NBS185. (C) Alternatively, polA deletion strain <br /> NBS182 derivatives examined LB plates containing 1 mM IPTG 56.5&#194;&#176;C. Two independent transformants (each) NBS247 <br /> (amyE::polA) (a b), NBS248 (amyE::polAN) (c d), NBS249 (amyE::ypcP) (e f) shown. <br />  <br />  <br />  <br /> tire polA <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>gene knockout</span> strain 7942polA series primer           primers, process ribonucleotides at <br /> and C-terminal Klenow domain disruptant using                    DNA-RNA junction. Four genes encoding RNase H the <br /> 7942polAc series primer (see Table S1 supplemental                B. subtilis genome, i.e., rnhB, rnhC, ypdQ, ypeP, been <br /> material). Since cyanobacteria contain multiple copies            identified. YpeP, similar RNase HI, newly <br /> chromosome, usually possible obtain disruptant colonies         identified; possessed known conserved residues involved <br /> for essential gene. However, successive patch-         catalysis E. coli RnhA, i.e., D10, E48, D70, H124, and <br /> ings, nonessential genes completely replaced. Using        D134 (Fig. 4). Each RNase H gene disruptant exhibited a <br /> this criterion, C-terminal disruptant stably maintained,         normal growth phenotype, rnhC ypeP mutants <br /> and PCR analysis confirmed complete replacement ge-               formed small colonies, complete medium. It has <br /> nome mutant allele. The resulting strain named              reported rnhB rnhC genes confer syn- <br /> NBC318. On hand, entire polA gene disruptant               thetic lethality cells B. subtilis, unlike E. <br /> could maintained stably, PCR analysis colo-          coli, RNase H indispensable (15). However, obtained B. <br /> nies revealed wild-type allele polA retained (data       subtilis strain lacked rnhB rnhC, the <br /> not shown). These results demonstrated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucle-          colony size small cell morphology was <br /> ase domain Synechococcus Pol I essential growth.               filamentous. Moreover, disruptants RNase <br />    RNase H function dispensable B. subtilis. RNase H,              H genes able grow 37&#194;&#176;C (Table 3). Mutants lacking <br /> encoded rnh genes, factor removing RNA              rnhB rnhC showed temperature-sensitive growth at <br />                                                                          56.5&#194;&#176;C, strain NBS210, additionally lacked ypeP, <br />                                                                          exhibited cold-sensitive growth 22&#194;&#176;C. The growth rates LB <br />                                                                          medium high- low-temperature-sensitive mutants at <br />                                                                          37&#194;&#176;C half wild type (data shown). <br />                                                                             The filamentous phenotype RNase H-defective strains is <br />                                                                          SOS response. When triple mutant NBS210 was <br />                                                                          grown LB medium 37&#194;&#176;C, cells mid-log <br />                                                                          phase exhibited filamentous cell morphology (Fig. 5, pDG148). <br />                                                                          This filamentous phenotype suppressed of <br />                                                                          depleted genes, rnhB, rnhC, ypeP, propagated a <br />                                                                          multicopy plasmid. On hand, overexpression ypdQ <br />                                                                          effect morphology NBS210 (Fig. 5). <br />                                                                             RNase H deficiency thought result accumulation <br />                                                                          Okazaki fragments induce SOS response. To <br />                                                                          examine possibility, performed lacZ reporter assay of <br />                                                                          yneA. The expression yneA control LexA and <br />    FIG. 3. Expression profiles &#226;?&#164;-galactosidase polA-lacZ     induced SOS response functions overlap those <br /> ypcP-lacZ fusions. Each strain carrying lacZ fusions grown 2&#226;&#171;&#187;     SulA E. coli (18). As shown Fig. 6A, yneA-lacZ is <br /> SG medium, &#226;?&#164;-galactosidase activity assayed. The time         constitutively expressed NBS218, NBS210 derivative car- <br /> transition exponential growth stationary phase designated    rying yneA-lacZ construct. The level expression re- <br /> time zero. Assays carried times yielded similar <br /> patterns; representative data shown. Closed circles, POLAd (polA- <br />                                                                          markably higher wild type, cellular <br /> lacZ); closed squares, NBS5014 (ypcP-lacZ); open squares, NBS5015        SOS response induced mitomycin C. Moreover, the <br /> (ypcP-lacZ &#226;&#338;&#172;polA background).                                     yneA disruptant NBS210 (NBS218) ceased exhibit the <br />  8580       FUKUSHIMA ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Multiple alignments type 1 RNase H sequences. The B. subtilis RNase HI homolog YpdQ novel RNase HI candidate YpeP <br /> were aligned E. coli RNase HI (eRnhA) Halobacterium RNase H (hRnh). The asterisks indicate conserved amino acid residues with <br /> more 65% identity. <br />  <br />  <br />  <br /> filamentous phenotype, consequently, impairment              This findings E. coli cyanobacteria suggest that <br /> cell division exhibited RNase H-defective strain             5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity Pol I activity <br /> suppressed (Fig. 6B).                                                   encoded ypcP xni required bacterial growth. <br />    Partial complementation RNase H 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease                 With assistance RNase H, excision RNA prim- <br /> activity. It suggested RNase H Pol I             ers vitro promoted (11). RNase H genes been <br /> function synergistically processing RNA primers           reported essential B. subtilis (15). However, we <br /> of Okazaki fragments (35). Hence, deficiency RNase H             succeeded disrupting candidate genes RNase H, we <br /> might compensated enhanced 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease ac-              conclude RNase H necessary cell viability. How- <br /> tivity. As shown Fig. 5 (polAFL ypcP), filamentous           ever, growth rates strains bearing RNase H <br /> phenotype strain NBS210 obviously suppressed                gene deletions, including rnhB rnhC, low, suggesting <br /> the polA ypcP gene overexpressed. A similar result           RNase H involved processing RNA primers. <br /> obtained gene encoding N-terminal domain               Based colony morphologies individual disruptants, <br /> Pol I (Fig. 5, polAN). These results indicate filamen-         rnhC appears prominent RNase <br /> tous morphology defective RNase H activity               H genes; rnhB rnhC double mutant exhibited temperature- <br /> that overexpression of 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domains          sensitive growth filamentous cell morphology. Moreover, <br /> could complement rnh mutations.                                     phenotypes apparently gene <br />                                                                         disruptants, suggesting major functions RNase H <br />                                 DISCUSSION                              attributable genes. The overexpression the <br />                                                                         newly identified ypeP suppressed filamentation rnhB <br />   Our discovery synthetic lethality pair doublet         rnhC ypeP mutant, ypdQ did not. Although (21) <br /> paralogs, polA ypcP, confirmed previous results (9)             claimed YpeP lacks RNase H activity based struc- <br /> more clearly demonstrated polymerase 5&#226;&#172;&#732;-3&#226;&#172;&#732; exo-            tural requirements, revised assignment open reading <br /> nuclease activity essential function B. subtilis cells.       frames careful alignment revealed conservation ac- <br />                                                                         tive amino acids (Fig. 4). These results suggest YpeP has <br />  <br />  TABLE 3. Cold-sensitive temperature-sensitive phenotypes of <br />                various gene disruption strains <br />                                                       Growth ata: <br />   Strain                Relevant genotype <br />                                                22&#194;&#176;C     37&#194;&#176;C   56.5&#194;&#176;C <br />  <br /> 168           Wild type                         &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS5004       &#226;&#338;&#172;polA                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#186; <br /> NBS4070       &#226;&#338;&#172;ypcP                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS202        &#226;&#338;&#172;rnhB                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS204        &#226;&#338;&#172;rnhC                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS205        &#226;&#338;&#172;ypdQ                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS206        &#226;&#338;&#172;ypeP                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS208        &#226;&#338;&#172;rnhB &#226;&#338;&#172;ypeP                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS209        &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185;     FIG. 5. Effects overexpression various genes filamen- <br /> NBS207        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#186;   tous phenotype B. subtilis triple-RNase H-deficient strain. NBS210 <br /> NBS210        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP                 &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   harboring pDG148 derivatives grown 37&#194;&#176;C LB medium <br /> NBS216        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP &#226;&#338;&#172;ypdQ           &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   containing 5 &#226;?&#174;g/ml kanamycin 1 mM IPTG. Cell morphologies at <br /> NBS217        &#226;&#338;&#172;polA &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP &#226;&#338;&#172;ypdQ     &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   mid-log phase observed microscope. Genes cloned in <br />                                                                         pDG148 shown. FL N indicate full-length N-terminal <br />  a <br />      &#226;&#171;&#185;, growth; &#226;&#171;&#186;, growth.                                           domains. Bar, 10 &#226;?&#174;m. <br />  VOL. 189, 2007                                                            REASSESSMENT OF Pol I AND RNase H FUNCTIONS                     8581 <br />  <br />  <br />                                                                            mutant, accumulate Okazaki fragments fail survive (32). <br />                                                                            The relationship SOS induction temperature sen- <br />                                                                            sitivity remains unknown; however, results B. subtilis <br />                                                                            indicate inefficient processing RNA primers the <br />                                                                            lagging strand results recognition DNA damage leads <br />                                                                            SOS induction temperature sensitivity. <br />                                                                               In Haemophilus influenzae, <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>mutation</span> RNase HI in- <br />                                                                            creases mutation rates tetranucleotide repeats re- <br />                                                                            sults phase variation delayed mutagenic Okazaki <br />    FIG. 6. SOS induction assay. (A) Kinetics yneA-lacZ induction.       fragment processing; additional deletion Klenow <br /> Aliquots YNEAd culture treated (closed squares) with-       domain Pol I gives rise higher phase variation rates <br /> out (open squares) mitomycin C withdrawn 20 (lane 1), 40 (lane <br /> 2), 60 (lane 3) min drug treatment assayed &#226;?&#164;-galac-      (4). These results support hypothesis enzyme <br /> tosidase activity. Portions alternatively grown NBS218 culture,    systems cooperate processing Okazaki fragments. <br /> triple-RNase H-deficient strain, sampled (open circles) de-           The dispensability RNase H function attributed to <br /> scribed Materials Methods assayed &#226;?&#164;-galactosidase           complementation 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activities Pol I <br /> activity. (B) Effects yneA deletion B. subtilis cell morphology. <br />                                                                            YpcP. The filamentous phenotype induced deficient <br /> Cells exponentially growing cultures examined a <br /> microscope. (a) 168 (wild type). (b) Triple-RNase H-deficient strain       RNase H partially suppressed overexpression of <br /> NBS210. (c) Triple-RNase H- yneA-deficient strain NBS218. Bar,         polA, N-terminal polA, ypcP. These results sug- <br /> 10 &#226;?&#174;m.                                                                     gest enhanced 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity suppressed the <br />                                                                            RNase H deficiency, therefore, enzymes appear to <br />                                                                            cooperate. <br /> some RNase H activity YpdQ does support                      Our results indicated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity <br /> earlier findings ypdQ did function B. subtilis (15, 30).       indispensable E. coli, cyanobacterial <br />    Among genes related processing RNA                    strain S. elongatus PCC 7942. It reported the <br /> primers Okazaki fragments, disrupted,                 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity Pol I appears essential two <br /> ypcP confer viability cells. The expression pro-            bacteria, H. influenzae Streptococcus pneumoniae, <br /> files ypcP polA similar. The strain lacking                lack ypcP ortholog (4, 8). Hence, enzymatic <br /> N-terminal domain Pol I (&#226;&#338;&#172;polAN) temperature sensi-                 activity essential eubacterial cells. Our <br /> tive, propagation ypcP gene amyE locus             finding B. subtilis ypcP E. coli ortholog, xni, led to <br /> not suppress temperature sensitivity polA mutant                look distribution types 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases <br /> (Fig. 2C), suggesting difference natures             bacteria. E. coli xni identified designated <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases. It noted partial suppres-           exo (36), encoded enzyme previously reported to <br /> sion observed Fig. 2C gene dosage effect              mainly 3&#226;&#172;&#732;-5&#226;&#172;&#732; single-stranded DNA exonuclease activity <br /> of presence copies ypcP gene genome.              5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity (39). Although charac- <br /> Hence, activity YpcP inactive high tem-              teristic nature RNA/DNA hybrids analyzed, we <br /> perature, vitro experiments demonstrated               posit process substrate manner as <br /> possesses 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity (J. Sekiguchi, personal               YpcP, classified genes based structural <br /> communication). Moreover, RNase H remove                   similarities. We categorized genes possessing 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucle- <br /> RNA primers completely (30), temperature-sensitive                 ase domain types, i.e., polA type, ypcP type, a <br /> phenotype &#226;&#338;&#172;polA strain elucidated. Al-                  type, consisting 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain a <br /> though RNase H disruptant identified temper-               polymerase domain 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease domain is <br /> ature sensitive regardless Pol I activity, strongly suspect          homologous polA (e-value &#226;&#172;? 0.01). A total 250 <br /> that RNase H function, addition Pol I, required           bacterial genomes far sequenced classified six <br /> efficient processing Okazaki fragments (35) high-                 classes, shown Fig. 7 (see Fig. S1 supplemental <br /> temperature conditions.                                                    material). Both B. subtilis E. coli belong class 4, al- <br />    The expression profiles yneA-lacZ multiple-gene               3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease activity B. subtilis known to <br /> disruptants, including rnhB rnhC, clearly demonstrated                 absent (9). Archaea (class 6) exceptional because, in- <br /> that SOS response taken place. In E. coli, SOS                 stead 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases, like eukaryotes, flap <br /> response leads sulA expression inhibits Z-ring forma-               endonucleases. On hand, eubacteria least <br /> tion (26). In B. subtilis, detailed mechanism different,        type 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease. About 90% eubacterial <br /> the enhanced activity YneA indirectly inhibits Z-ring for-              genomes contain polA-type genes; 37% carry ypcP-type genes. <br /> mation (18, 19). Hence, filamentous cell morphology                Only genomes type 3 genes (class 3). On other <br /> be consequence depleted Z-ring formation attributable              hand, 20 genomes ypcP-type genes (class 2) exclusively, <br /> the activation LexA repressor accumulated un-                indicating 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease polymer- <br /> processed Okazaki fragments. The basic regulatory mechanism                ase domain essential. <br /> of SOS response controlled complex circuitry in-                  Therefore, used informatics approach look the <br /> volving RecA LexA proteins. When chromosomal DNA                    distribution repair-related proteins PolA, i.e., <br /> damaged replication inhibited, RecA activated              PolB IMS domain, includes Pol IV Pol V <br /> inducing signals LexA repressor degraded. It                (see Fig. S1 supplemental material). Using analysis, <br /> been reported polA mutant, RNase H                    archaea possess DNA polymerase II only <br />  8582       FUKUSHIMA ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                        3. Au, N., E. Kuester-Schoeck, V. Mandava, L. E. Bothwell, S. P. Canny, K. <br />                                                                                           Chachu, S. A. Colavito, S. N. Fuller, E. S. Groban, L. A. Hensley, T. C. <br />                                                                                           O&#226;&#8364;&#8482;Brien, A. Shah, J. T. Tierney, L. L. Tomm, T. M. O&#226;&#8364;&#8482;Gara, A. I. Goranov, <br />                                                                                           A. D. Grossman, C. M. Lovett. 2005. Genetic composition Bacillus <br />                                                                                           subtilis SOS system. J. Bacteriol. 187:7655&#226;&#8364;&#8220;7666. <br />                                                                                        4. Bayliss, C. D., W. A. Sweetman, E. R. Moxon. 2005. Destabilization of <br />                                                                                           tetranucleotide repeats Haemophilus influenzae mutants lacking RnaseHI <br />                                                                                           Klenow domain PolI. Nucleic Acids Res. 33:400&#226;&#8364;&#8220;408. <br />                                                                                        5. Britton, R. A., P. Eichenberger, J. E. Gonzalez-Pastor, P. Fawcett, R. <br />                                                                                           Monson, R. Losick, A. D. Grossman. 2002. Genome-wide analysis the <br />                                                                                           stationary-phase sigma factor (sigma-H) regulon Bacillus subtilis. J. Bac- <br />                                                                                           teriol. 184:4881&#226;&#8364;&#8220;4890. <br />                                                                                        6. Castenholz, R. W. 1988. Culturing methods cyanobacteria. Methods <br />                                                                                           Enzymol. 167:68&#226;&#8364;&#8220;93. <br />                                                                                        7. De Lucia, P., J. Cairns. 1969. Isolation E. coli strain a <br />                                                                                           mutation affecting DNA polymerase. Nature 224:1164&#226;&#8364;&#8220;1166. <br />                                                                                        8. Diaz, A., S. A. Lacks, P. Lopez. 1992. The 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonuclease activity <br />                                                                                           DNA polymerase I essential Streptococcus pneumoniae. Mol. Mi- <br />    FIG. 7. Venn diagram bacterial species according type                     crobiol. 6:3009&#226;&#8364;&#8220;3019. <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease gene. A total 250 bacterial genomes (228 eubac-                   9. Duigou, S., S. D. Ehrlich, P. Noirot, M. F. Noirot-Gros. 2005. DNA <br /> teria 22 archaea) sequenced date categorized                         polymerase I acts translesion synthesis mediated Y-polymerases in <br /> classes based gene structure encoding 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease. Each                      Bacillus subtilis. Mol. Microbiol. 57:678&#226;&#8364;&#8220;690. <br />                                                                                       10. Fujita, M., R. Losick. 2005. Evidence entry sporulation in <br /> circle corresponds types 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease: PolA type, YpcP                   Bacillus subtilis governed gradual increase level activity of <br /> type, (5&#226;&#172;&#732;-3&#226;&#172;&#732; EXO-DNA_POL) type (see text). Class 1                        master regulator Spo0A. Genes Dev. 19:2236&#226;&#8364;&#8220;2244. <br /> consisted genomes containing PolA type. Class 2 consisted                 11. Funnell, B. E., T. A. Baker, A. Kornberg. 1986. Complete enzymatic <br /> of genomes containing YpcP type. Class 3 consisted ge-                        replication plasmids containing origin Escherichia coli chromo- <br /> nomes containing type. Class 4 consisted genomes                        some. J. Biol. Chem. 261:5616&#226;&#8364;&#8220;5624. <br /> containing PolA type YpcP type. Class 5 consisted                      12. Higuchi, R. 1989. Using PCR engineer DNA, p. 61&#226;&#8364;&#8220;70. In H. A. Erlich <br /> genomes containing YpcP type type. Class 6 consisted                    (ed.), PCR technology. Stockton Press, New York, NY. <br /> of genomes containing 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease genes.                                     13. Itaya, M. 1992. Construction novel tetracycline resistance gene cassette <br />                                                                                           useful marker Bacillus subtilis chromosome. Biosci. Biotechnol. <br />                                                                                           Biochem. 56:685&#226;&#8364;&#8220;686. <br />                                                                                       14. Itaya, M. 1990. Isolation characterization second RNase H (RNase <br /> seven species, simultaneously belong class 2 Fig. 7,                          HII) Escherichia coli K-12 encoded rnhB gene. Proc. Natl. Acad. <br />                                                                                           Sci. USA 87:8587&#226;&#8364;&#8220;8591. <br /> possess repair polymerase (see Fig. S1 supplemental                         15. Itaya, M., A. Omori, S. Kanaya, R. J. Crouch, T. Tanaka, K. Kondo. <br /> material). These species belong genera, Buchnera,                                 1999. Isolation RNase H genes essential growth Bacillus <br /> Wigglesworthia, Phytoplasma, Tropheryma, sym-                             subtilis 168. J. Bacteriol. 181:2118&#226;&#8364;&#8220;2123. <br />                                                                                       16. Joyce, C. M., N. D. Grindley. 1984. Method determining a <br /> biotic infectious bacteria. Although known                               gene Escherichia coli essential: application polA gene. J. Bacteriol. <br /> whether bacteria sensitive DNA-damaging agents,                              158:636&#226;&#8364;&#8220;643. <br />                                                                                       17. Kanaya, S., R. J. Crouch. 1983. DNA sequence gene coding for <br /> such mitomycin C, require DNA repair func-                                Escherichia coli ribonuclease H. J. Biol. Chem. 258:1276&#226;&#8364;&#8220;1281. <br /> tions environments inhabit.                                       18. Kawai, Y., S. Moriya, N. Ogasawara. 2003. Identification protein, <br />    As noted above, archaea eubacteria developed distinct                              YneA, responsible cell division suppression SOS response in <br />                                                                                           Bacillus subtilis. Mol. Microbiol. 47:1113&#226;&#8364;&#8220;1122. <br /> types processing enzymes, i.e., flap endonuclease 5&#226;&#172;&#732;-3&#226;&#172;&#732;                        19. Kawai, Y., N. Ogasawara. 2006. Bacillus subtilis EzrA FtsL syner- <br /> exonuclease, respectively. There weak homologies                                gistically regulate FtsZ ring dynamics cell division. Microbiology <br /> the domains polymerase I, hyperthermophilic                              152:1129&#226;&#8364;&#8220;1141. <br />                                                                                       20. Kitani, T., K. Yoda, T. Ogawa, T. Okazaki. 1985. Evidence discon- <br /> bacterium Aquifex aeolicus, belongs class 2, diverged                            tinuous DNA replication Escherichia coli primed approximately 10 to <br /> close root phylogenetic tree. Since Aquifex                            12 residues RNA starting purine. J. Mol. Biol. 184:45&#226;&#8364;&#8220;52. <br /> possesses gene corresponding separate Klenow frag-                             21. Knizewski, L., K. Ginalski. 2005. Bacillus subtilis YkuK protein dis- <br />                                                                                           tantly related RNase H. FEMS Microbiol. Lett. 251:341&#226;&#8364;&#8220;346. <br /> ment, domains, i.e., 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease, evolved                      22. Kobayashi, K., S. D. Ehrlich, A. Albertini, G. Amati, K. K. Andersen, M. <br /> first, duplication combination <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>                              Arnaud, K. Asai, S. Ashikaga, S. Aymerich, P. Bessieres, F. Boland, S. C. <br />                                                                                           Brignell, S. Bron, K. Bunai, J. Chapuis, L. C. Christiansen, A. Danchin, M. <br /> yielded variety related enzyme patterns different                              Debarbouille, E. Dervyn, E. Deuerling, K. Devine, S. K. Devine, O. Dreesen, <br /> organisms.                                                                                J. Errington, S. Fillinger, S. J. Foster, Y. Fujita, A. Galizzi, R. Gardan, C. <br />                                                                                           Eschevins, T. Fukushima, K. Haga, C. R. Harwood, M. Hecker, D. Hosoya, <br />                                                                                           M. F. Hullo, H. Kakeshita, D. Karamata, Y. Kasahara, F. Kawamura, K. <br />                           ACKNOWLEDGMENTS                                                 Koga, P. Koski, R. Kuwana, D. Imamura, M. Ishimaru, S. Ishikawa, I. Ishio, <br />    We grateful Naoto Ohtani Keio University helpful                         D. Le Coq, A. Masson, C. Mauel, R. Meima, R. P. Mellado, A. Moir, S. <br />                                                   <br /> </body></html>