<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Transcriptional profiling endogenous germ layer <br /> precursor cells identifies dusp4 essential <br /> gene zebrafish endoderm specification <br /> Jamie L. Brown*, Mirit Snir*, Houtan Noushmehr*&#226;&#8364;&#160;, Martha Kirby&#226;&#8364;&#161;, Sung-Kook Hong&#194;&#167;, Abdel G. Elkahloun&#194;&#182;, <br /> and Benjamin Feldman*&#229;&#8218;&#168; <br /> *Medical Genetics Branch, &#226;&#8364;&#161;Genetics Molecular Biology Branch, &#194;&#182;Genome Technology Branch, National Human Genome Research Institute; <br /> and &#194;&#167;Program Genomics Differentiation, Eunice Kennedy Shriver National Institute Child Health Human Development, National <br /> Institutes Health, Bethesda, MD 20892 <br />  <br /> Communicated Francis S. Collins, National Institutes Health, Bethesda, MD, June 10, 2008 (received review February 21, 2008) <br />  <br /> A major goal developmental biologists define behav-                  A reproducible map embryonic cell fates zebrafish is <br /> iors molecular contents differentiating cells. We                   established &#226;&#172;&#8225;5 h post-fertilization (hpf), late blastula stage <br /> devised strategy isolating cells diverse embryonic                   (6). At time, embryonic cell layers arranged <br /> regions stages zebrafish, using computer-guided laser                dome surrounds hemisphere single large yolk <br /> photoconversion injected Kaede protein flow cytometry.                   cell (Fig. 1A). The endoderm precursor field lies the <br /> This strategy enabled perform genome-wide transcriptome                 lowest tiers dome (Fig. 1 A B) (7). The mesoderm <br /> comparison germ layer precursor cells. Mesendoderm ec-                   precursor field overlaps endoderm precursor field and <br /> toderm precursors cells isolated method differentiated                  extends higher tiers (6). The cells comprise over- <br /> appropriately transplantation assays. Microarray analysis                 lapping fields collectively termed mesendoderm precursors. <br /> these cells reidentified known genes efficiently                 Above sixth tier, staggered transition meso- <br /> previously reported strategies relied artificial mesend-                derm fates ectoderm fates, ectoderm precursors con- <br /> oderm activation inhibition. We identified large set               tinuing embryo (6). <br /> uncharacterized mesendoderm-enriched genes ecto-                        Traditional methods isolating early mesendoderm and <br /> derm-enriched genes. Loss-of-function studies revealed              ectoderm cells technically challenging (excising tops of <br /> these genes, MAP kinase inhibitor dusp4, essential early             embryos eyebrow hair explantation individual cells <br /> development. Embryos injected antisense morpholino oligo-                  micropipet) studies published <br /> nucleotides targeted Dusp4 displayed necrosis head tissues. <br />                                                                                 examined fate changes undergo transplanted <br /> Marker analysis late gastrulation revealed specific loss of <br />                                                                                 host embryos (8, 9) explanted cultured (10). <br />                                                                                 Transgenic zebrafish lines mesoderm-, endoderm-, ec- <br /> sox17, endoderm markers, analysis later <br />                                                                                 toderm-restricted fluorescent protein expression re- <br /> stages revealed loss foregut pancreatic endoderm. This <br />                                                                                 ported (11, 12), late-blastula stage, are <br /> specific loss sox17 establishes new class endoderm speci- <br />                                                                                 bright FACS-based purification (J.L.B. B.F., <br /> fication defect. <br />                                                                                 unpublished observations). <br />                                                                                    <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> understanding germ-layer differentiation, <br /> microarray &#229;&#8230;&#169; zgc:55423 &#229;&#8230;&#169; gastrulation &#229;&#8230;&#169; mkp2 &#229;&#8230;&#169; sox17 <br />                                                                                 developed FACS-assisted microdissection photola- <br />                                                                                 beled (FAM-P) cells, strategy isolating zebrafish embryonic <br />  <br /> T   establishment germ layers coordinated <br />     rearrangement gastrulation core developmental <br /> process (1). In organisms ranging humans flatworms, <br />                                                                                 cells. FAM-P uses recombinant photoconvertible Kaede protein, <br />                                                                                 overcomes spatiotemporal limitations transgeni- <br />                                                                                 cally expressed fluorescent proteins injected mRNA and <br /> three primary germ layers arise: mesoderm forms deep                        allows quantitative isolation early embryonic zebrafish <br /> tissues blood muscle, endoderm produces viscera                 cells user-defined regions interest. We isolated endog- <br /> like digestive tract liver, ectoderm forms                  enous mesendoderm ectoderm precursors FAM-P and <br /> epidermis central nervous system. Cell lineage tracing                      transplanted host embryos, contributed to <br /> studies mapped precursor cells fated form                  similar range tissues mesendoderm ectoderm pre- <br /> these germ layers, key commonalities fate                  cursors traditionally explanted micropipet. DNA <br /> maps different vertebrates (2). Germ layer precursor cells               microarray analysis FAM-P-isolated mesendoderm ecto- <br /> typically distributed extended fields, making quantitative <br /> isolation manual microsurgery particularly challenging. <br />                                                                                 Author contributions: J.L.B., M.S., M.K., A.G.E., B.F. designed research; J.L.B., M.S., M.K., <br />    Fluorescent proteins opened door new microdis-                   S.-K.H., A.G.E. performed research; J.L.B., M.S., H.N., S.-K.H., B.F. analyzed data; <br /> section approaches developmental biology. Placing different                  B.F. wrote paper. <br /> fluorescent proteins different promoters, distinct colors                The authors declare conflict interest. <br /> have expressed cell types comprising nascent skin                Data deposition: The data reported paper deposited Gene <br /> late gastrula endoderm developing mice (3, 4). In                      Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE8654). <br /> studies, fluorescently labeled cells newborn                     &#226;&#8364;&#160;Presentaddress: Eli Edythe Broad Center Regenerative Medicine Stem Cell <br /> embryonic mice dissociated, separated FACS,                          Research, Keck School Medicine, University Southern California, Los Angeles, CA <br />                                                                                  90089. <br /> mRNA harvested identify tissue-specific genes mi- <br />                                                                                 &#229;&#8218;&#168;To <br />                                                                                                                                                                                     DEVELOPMENTAL <br />  <br />  <br />  <br />  <br /> croarray analysis. More recently, photoconvertible fluorescent                      correspondence addressed at: Vertebrate Embryology Section, Med- <br />                                                                                 ical Genetics Branch, National Human Genome Research Institute, National Institutes of <br /> proteins Kaede (5) available, allowing <br />                                                                                                                                                                                        BIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                 Health, Building 35 Room 1B205, 35 Convent Drive, MSC 3717, Bethesda, MD 20892-3717. <br /> researchers interactively convert expressing cells                  E-mail: bfeldman@mail.nih.gov. <br /> color another. Several promising uses photoconvertible                    This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> proteins embryology described; however,                  0805589105/DCSupplemental. <br /> not used facilitate tissue-specific microdissection.                &#194;&#169; 2008 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805589105                                                    PNAS &#229;&#8230;&#169; August 26, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 12337&#226;&#8364;&#8220;12342 <br />  Fig. 1. Use Kaede protein <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> embryonic cells trace lineages. (A) Schematic late blastula stage (40% epiboly) embryo. The embryonic layer <br /> (EL) color coded according germ layer precursor distribution, dark red representing endoderm precursor-enriched, red representing mesoderm <br /> precursors, green representing ectoderm precursors. Also indicated yolk cell (YC, tan), enveloping layer (EVL, purple), yolk syncytial layer <br /> (YSL, light blue). For clarity, yolk cell represented separated overlying tissues. (B) Fate map late-blastula stage embryo Kimmel et al. <br /> (6) using color scheme A including precursor domains indicated outcomes H. (C) Coomassie-blue stained polyacrylamide gel, showing <br /> a 34-kDa monomer bacterially expressed Kaede column purification. (D) Four-cell-stage (1 hpf) embryo shortly injection 3.5 ng purified <br /> Kaede protein, lateral view. (E) 40% epiboly-stage (5 hpf) embryo just photolabeling margin, animal pole view. (F) Same embryo E, just <br /> after photolabeling margin, animal pole view. (G) Close-up Kaede protein-injected embryo, directly photolabeling margin, lateral <br /> view. Only optical sections near surface shown. Brackets numbers indicate lowest cellular tiers. (H) Same embryo G, at <br /> pharyngula stage (24 hpf), lateral view, anterior left. Representative tissues indicated H relative pixel values (% red total red &#226;&#171;&#185; green) match <br /> expected outcomes follows: 1 (22%), 2 (25%), 3 (52%), 4 (52%), 5 (77%), 6 (78%). <br />  <br />  <br />  <br /> derm precursors revealed continue express molecular                          generated recombinant Kaede protein (Fig. 1C). Injection of <br /> markers specific regions origin. Microarray analysis                     purified Kaede protein nontoxic brightly labeled em- <br /> also identified dozens new mesendoderm precursor-enriched                         bryos time injection (Fig. 1D) late <br /> genes. Morpholino knockdown genes, dusp4, pro-                          blastula stage (Fig. 1 E&#226;&#8364;&#8220;G) persisted 24 hpf (Fig. 1H), <br /> duced loss-of-function <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, characterized down-                       time signal gradually faded (data shown), <br /> regulation single endoderm marker gastrulation,                          presumably cellular catabolism. <br /> sox17, endoderm markers expressed nor-                              To specifically label mesendoderm precursors, microin- <br /> mally, loss foregut pancreatic endoderm later                        jected Kaede protein fertilized eggs late blastula <br /> stages. Our studies illustrate just ways FAM-P                      stage embryos mounted scanning confocal <br /> can applied experimental embryology. Given spatial                         microscope, embryo nearest lens (Fig. <br /> and temporal flexibility FAM-P offers, useful                      1E). Orientation multiple specimens facilitated mount- <br /> for experiments zebrafish possibly                        ing embryos, transparent chorions intact, arrayed conical <br /> model organisms.                                                                     molds (13, 14). A circular swath defined comprising the <br />                                                                                      outermost, lowest, tiers cells, mes- <br /> Results                                                                              endoderm precursors photoconverted green red by <br /> Differential Labeling Germ Layer Precursors Kaede Pro-                   computer-controlled scanning near-UV laser (Fig. 1F). <br /> tein. The mesendoderm precursor field zebrafish late blastula                     Using method, label mesendoderm precursors of <br /> stage (40&#226;&#8364;&#8220;50% epiboly, &#226;&#172;&#8225;5 hpf) embryos consists narrow                          8&#226;&#8364;&#8220;12 embryos single experiment. <br /> band cells encircle delicate yolk cell (Fig. 1 A B).                      Traditional fate mapping requires lineage tracing single <br /> Because efficient isolation cells using traditional                   cells dozens embryos. Labeling entire margin visualized <br /> microsurgical tools untenable, set develop method                     key features fate map (Fig. 1B) single embryo. Image <br /> for fluorescently labeling cells isolating                   analysis day 2 shows tissues normally derived lower tiers <br /> FACS. For purpose, selected photoconvertible                             red pixels, tissues normally higher tiers with <br /> fluorescent reagent Kaede, stony coral protein fluoresces                     green pixels, tissues normally derived near the <br /> green (518 nm) full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> form red (582&#226;&#8364;&#8220;627 nm)                    margin/nonmargin boundary similar numbers red and <br /> photocleavage near-UV light (405 nm) (5). We discovered                           green pixels, presumably representing mixture red, green, <br /> that nascent fluorescent protein maturation slow relative                      half-labeled yellow cells (Fig. 1H). This analysis demon- <br /> zebrafish development, late blastula-stage fluores-                        strates correctly target mesendoderm precur- <br /> cence injected kaede mRNA [see supporting information                           sor field indicates Kaede protein labeling does alter <br /> (SI) Fig. S1] zygotically expressed transgenic fluorescent                       cellular destinies. We similarly used Kaede protein label <br /> proteins (J.L.B. B.F., unpublished observations)                        trace outcomes embryonic regions (Fig. S1). Such <br /> bright FACS. To circumvent difficulty,                            lineage tracing experiments facilitated ability cells <br />  <br /> 12338 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805589105                                                                                                      Brown et al. <br />  labeled Kaede protein hold color time. When <br /> kaede mRNA injected, photolabeled lineages signif- <br /> icantly yellowed newly synthesized green Kaede <br /> protein (see Fig. S1). <br />  <br /> Isolation Mesendoderm Ectoderm Precursors. To quantita- <br /> tively purify mesendoderm ectoderm precursors, disso- <br /> ciated labeled embryos single cell suspensions sorted <br /> cells FACS (Fig. S2). To minimize losses labeled cells <br /> during processing (e.g., pipetting, centrifugation, FACS <br /> steps), mixed unlabeled cells uninjected <br /> sibling embryos point trypsinization. Forward side <br /> scatter values single embryonic cells determined, and <br /> events gates sorted relative red <br /> and green fluorescence. Using parameters, purify <br /> homogenous populations green red cells, recovery <br /> rate input embryonic cells close 75% 99% viability <br /> (data shown). <br />    A critical test FAM-P&#226;&#8364;&#8482;s utility determine whether <br /> sorted cells retain developmental potentials biases. It <br /> has previously shown ectoderm precursors trans- <br /> planted animal pole reliably incorporated host <br /> ectoderm (8, 9). We true FAM-P-isolated <br /> ectoderm precursors, contributed ectoderm fates, such <br /> as retinal neurons forebrain cells (Fig. S2 Table S1). It <br /> was previously shown early-stage mesendoderm precursors <br /> (4.7&#226;&#8364;&#8220;5 hpf) readily reprogrammed ectoderm (8, 9), but <br /> shortly [beyond 50% epiboly (5.3 hpf)] com- <br /> mitted mesendoderm fates (8). Our FAM-P purified mesen- <br /> doderm cells come intermediate stage [40&#226;&#8364;&#8220;50% epiboly <br /> (5.0&#226;&#8364;&#8220;5.3 hpf)] consistent this, contributed to <br /> ectoderm- mesendoderm-derived tissues, did marginal <br /> cells stage traditionally transplanted directly <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> embryo (Fig. S2 Table S1). We <br /> conclude FAM-P-purified mesendoderm ectoderm <br /> precursors maintain endogenous commitment status and <br /> developmental potential. <br />                                                                       Fig. 2. Transcriptome analysis mesendoderm neurectoderm precur- <br />                                                                       sor cells. (A B) Volcano plots showing 31,356 (A) 25,837 (B) oligos each. <br /> Identification Previously Characterized Uncharacterized Germ <br />                                                                       Each oligo mapped y axis according relative enrichment in <br /> Layer-Specific Genes. To study transcriptomes         direct comparison source tissues (average normalized Log2 the <br /> mesendoderm ectoderm precursors, harvested am-             hybridization signal ratio) x axis according significance <br /> plified RNA purified precursor cells cohybridized          (negative Log2 scale uncorrected P value). (A) FAM-P isolated mesendoderm <br /> them oligo chips microarray analysis (Fig. 2A). As         precursor cells (&#226;&#172;&#381; 0 Y axis) vs. cognate FAM-P isolated ectoderm precursor <br /> control photolabeling effects, prepared           cells (&#226;&#172;? 0 Y axis). (B) Kaede-injected photoconverted embryo cells (&#226;&#172;&#381; 0 <br /> cohybridized amplified RNA probes green red control          Y axis) vs. cells Kaede-injected embryos photoconverted <br /> cells, yielding substantially enriched genes (Fig. 2B). Thus,      (&#226;&#172;? 0 Y axis). (C D) Bar graphs showing representation selected <br /> the expression profile embryonic cells substantially        molecular functions (C) signaling pathways (D) 60 most- <br />                                                                       enriched genes (Table S2). Mes, mesendoderm precursor enriched; Ect, ecto- <br /> altered photolabeling se. We identified 188 full-length <br />                                                                       derm precursor enriched; Con, red control cell enriched; E, Enzyme; LAR, <br /> cDNAs significantly enriched (P &#226;&#172;? 0.05)              ligand (agonist), antagonistic ligand receptor; TF, transcription factor; N, <br /> mesendoderm precursor pool 106 similarly enriched cDNAs           Nodal pathway; W, Wnt pathway; F, FGF pathway; RA, retinoic acid pathway. <br /> in ectoderm precursor pool.                                       Genes acting multiple pathways arbitrarily assigned pathway <br />    We compiled cohorts 60 unique genes           following priority: RA &#226;&#172;&#381; F &#226;&#172;&#381; W &#226;&#172;&#381; N. <br /> comparison minimum annotated information was <br /> available (Table S2) examined over-represented <br /> gene ontology terms (Fig. 2C Table S2). A striking percent-       mesendoderm precursors (15) known ectoderm- <br /> age (45%) mesendoderm precursor genes encode tran-             specific genes. We 38.3% mesendoderm <br /> scription factors (Fig. 2C). The frequency transcription factors   precursor cohort genes fall four <br /> on entire microarray, contrast, 10%, similar    pathways, ectoderm control cohort incidences were <br /> frequency seen control cohort (Fig. 2C). We noted      8.3% 1.7%, respectively (Fig. 2D). Although identified <br /> an excess ligands, agonists, receptors paucity        far fewer highly enriched ectoderm genes, slightly increased <br /> enzymes mesendoderm precursors compared                false discovery filter 5.5% reveals substantial enrichment of <br /> the control cohort (Fig. 2C). An elevated number transcrip-        characterized neurectoderm genes, sox3 (16), <br />                                                                                                                                                           DEVELOPMENTAL <br />  <br />  <br />  <br />  <br /> tion factors (32%) seen ectoderm precursors            sox 19a (17), lhx5 (18) otx2 (19), nonneural <br /> (Fig. 2C), molecular class           ectoderm gene foxi1 (20) (Fig. 2C Table S2). <br />                                                                                                                                                              BIOLOGY <br />  <br />  <br />  <br />  <br /> underrepresented.                                                        A number mesendoderm-enriched genes identified <br />    We scored mesendoderm, ectoderm, control              reported expression late blastula stage. We <br /> cohorts membership signaling pathways&#226;&#8364;&#8221;Nodal,              characterized 21 genes mesendoderm precursor <br /> WNT, FGF, retinoic acid&#226;&#8364;&#8221;which known active              pool cloning performing mount situ hybridiza- <br />  <br /> Brown et al.                                                                               PNAS &#229;&#8230;&#169; August 26, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 12339 <br />                                                                                   expression mesoderm endoderm markers Dusp4 <br />                                                                                  morphants. We observed drastic reduction expression of <br />                                                                                  sox17 gastrula-stage endoderm cells specialized <br />                                                                                  group cells termed dorsal forerunners (Fig. 3 L M) <br />                                                                                  (22). Intriguingly, stage, endoderm markers as <br />                                                                                  foxa2 (23) (Fig. 3 N O) expressed normally Dusp4 <br />                                                                                  morphants. These phenotypes specific able to <br />                                                                                  phenocopy morphological molecular defects a <br />                                                                                  second, nonoverlapping MO Dusp4 (data shown) <br />                                                                                  rescue coinjection dusp4 mRNA (Fig. <br />                                                                                  3K). Analysis differentiated endoderm probes for <br />                                                                                  foxa2 (24) insulin (24) transcripts revealed loss foregut <br />                                                                                  (Fig. 3 P Q) pancreatic (Fig. 3 R S) endoderm. <br />  <br />                                                                                  Discussion <br />                                                                                  This paper introduces method viably isolating embryonic <br />                                                                                  cells user-defined regions stages interest. Injected <br />                                                                                  Kaede protein brightly label cells earlier exoge- <br />                                                                                  nous mRNA, problem novo synthesis over <br />                                                                                  course lineage tracing experiments. Caged fluorophores <br />                                                                                  suitable alternatives Kaede protein; however, these <br />                                                                                  reagents commercially unavailable years. <br />                                                                                  Another possible alternative recombinant photoconvertible <br />                                                                                  EosFP protein, used achieve early cell labeling <br />                                                                                  frog embryos (25). <br />                                                                                     For studies described here, microdissected embryos <br />                                                                                  1 h labeling; however, labeled cells allowed to <br />                                                                                  differentiate isolation. For instance, easily <br />                                                                                  isolated early-labeled mesendoderm precursors mid- and <br /> Fig. 3. Expression function new mesendoderm genes. (A&#226;&#8364;&#8220;H) Whole            late-gastrula stages (data shown). Thus, diverse range of <br /> mount situ hybridizations (WISH) late-blastula-stage (5 hpf) embryos,      embryonic cell types labeled purified FAM-P. One <br /> validating mesendoderm-specific expression mesendoderm pre-         topological limitation noted. The cells labeled were <br /> cursor-enriched genes, indicated. Animal pole views shown, C&#226;&#8364;&#8220;F        conveniently arranged shallow half dome. Differential <br /> have dorsal sides right (black arrows), confirmed double      labeling deep superficial cells feasible the <br /> stains chordin gene (data shown). (I&#226;&#8364;&#8220;K) Embryos injected     single-photon apparatus used achievable with <br /> control morpholino (Ctrl MO), Dusp4 morpholino (Dusp4 MO), rescuing <br />                                                                                  multiphoton systems, reported photoconversion <br /> combination Dusp4MO dusp4 mRNA, shown pharyngula stage <br /> (28 hpf). (L&#226;&#8364;&#8220;O) WISH late-gastrula-stage (80% epiboly) control Dusp4 <br />                                                                                  KikGR (26). <br /> MO-injected embryos, visualizing expression endoderm markers sox17           To knowledge, microarray analysis such <br /> (L M, dorsal views) foxa2 (N O, lateral views). Black arrowheads     study endogenous germ layer precursors zebrafish; how- <br /> indicate sample endoderm cells, open arrows point stained dorsal          ever, papers reported transcriptomes of <br /> forerunner cells. (P&#226;&#8364;&#8220;S) WISH 18 somite-stage control (P), pharyngula-stage    zebrafish embryos forced over- underproduce mesend- <br /> (26 hpf) Dusp4 MO-injected (Q), pharyngula-stage (30 hpf) control        oderm alterations Nodal signaling (27&#226;&#8364;&#8220;29). Each these <br /> morphant (R S) embryos, visualizing foxa2 (P Q) insulin (R S).   studies yielded important complementary data, is <br /> Embryos P Q stage matched time matched account        useful perform detailed comparison assess endoge- <br /> for developmental delays morphants. Black arrows P Q point to <br />                                                                                  nous approach. Our method best identifying non-Nodal <br /> posterior endoderm, including foregut region. Black arrows magnifica- <br /> tions R S highlight pancreatic stains. <br />                                                                                  mesendoderm pathway members (i.e., FGF, WNT, retinoic <br />                                                                                  acid), 12 60 genes categories, <br />                                                                                  compared 9 98 (28) 6 75 (27) studies <br /> tions late blastula-stage embryos. Ten genes showed                  presented nonvalidated hits (Table S2). Our approach <br /> margin-specific staining late blastula embryos, validating              successful identifying Nodal pathway genes. The <br /> enrichment mesendoderm precursors (Fig. 3 A&#226;&#8364;&#8220;H                          highest number validated Nodal-regulated genes emerging <br /> data shown). We validated ectoderm-specific ex-                     studies 34 curated list 72 (29). Our 188 <br /> pression previously uncharacterized ectoderm precur-                  mesendoderm genes included 26 genes list have <br /> sor genes (data shown).                                                      dozens more, Fig. 3, pending validation (Table <br />    We used antisense morpholino oligonucleotides (MOs)                        S3). Thus, screening endogenous cells yielded overall greater <br />                                                                                  diversity pathway-specific genes. Our proposed explanation <br /> block translation validated mesendoderm precur- <br />                                                                                  difference hyperactivation Nodal signaling <br /> sor genes associate distinct and <br />                                                                                  extinguishes expression Nodal non-Nodal pathway <br /> reproducible phenotype: zgc:55423, 86% homology                        genes uniquely expressed lower endogenous Nodal <br /> (Clustal alignment) human dual-specificity phosphatase 4                      signaling levels. <br /> (DUSP4, alias: MKP2) accordingly termed                           Many mesendoderm precursor-enriched genes iden- <br /> dusp4. Dusps attenuate mitogen-activated protein kinase                          tified previously shown expressed mesend- <br /> (MAPK) signaling de-phosphorylation key residues,                     oderm precursors, half tested showed margin- <br /> with human DUSP4 reported target MAPKs ERK                            enriched expression situ, including dusp4. The loss of <br /> JNK (21). At pharyngula stage, Dusp4-depleted embryos                        expression sox17, foxa2, observed Dusp4 <br /> displayed developmental delays small necrotic heads,                     morphants gastrulation unusual define new <br /> suggesting essential role Dusp4 anterior development                   class endoderm defect; reduction sox17 been <br /> (Fig. 3J). Given dusp4&#226;&#8364;&#8482;s expression margin, investigated               seen zebrafish sqt, cyc, oep, bon, fau, cas mutants, foxa2 <br />  <br /> 12340 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805589105                                                                                     Brown et al. <br />  has seen coordinately down-regulated exam-                                   media cell viability assays transplants. Methods transplantation <br /> ined (30). We report loss foregut pancreatic                                    experiments legend Fig. S2. <br /> endoderm Dusp4 morphants. This outcome consistent with <br /> sox17 decreases, mouse <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>knockout</span> Sox17                                        RNA Isolation, Amplification, Cy Labeling. Cells sorted TRIzol, <br />                                                                                               vortexed, frozen dry ice, stored &#226;&#171;&#186;80&#194;&#176;C, extracted according the <br /> reported lack gut endoderm (31). <br />                                                                                               manufacturer&#226;&#8364;&#8482;s protocol slight modifications. Approximately 300 ng 1 <br />    In summary, ability microdissect wide range early                              &#226;?&#174;g RNA linearly amplified using Amino Allyl MessageAmp II aRNA <br /> cells zebrafish opens numerous experimental possibilities                               Amplification kit (Ambion) yields ranging 12 30 &#226;?&#174;g aRNA. <br /> for developmental biologists. Future studies include,                               aRNA samples split labeled, half Cy3 mono NHS ester half <br /> instance, comparisons dorsal vs. ventral younger vs. older                              Cy5 mono NHS ester (CyDyes GE Healthcare; post-labeling reagents <br /> embryonic cell populations. We shown FAM-P-                                         MessageAmp II kit). Labeled aRNAs fragmented incubation <br /> purified zebrafish cells used transplantation assays                            94&#194;&#176;C 15 min fragmentation buffer (Affymetrix). Paired probes were <br /> microarray studies, applications envi-                                  prepared independent experiments microdissecting mesendoderm <br /> sioned, direct transcriptome sequencing, proteome anal-                               ectoderm precursors independent experiments using control <br /> ysis, establishment new embryonic cell lines.                                       red control green cells prepared follows. A number embryos were <br />                                                                                               injected Kaede photoconverted red toto, equivalent <br /> Materials Methods                                                                         number Kaede-injected embryos left green-fluorescent state. <br />                                                                                               These red-labeled green-labeled embryos pooled together <br /> Microinjection Kaede Photoconversion. Embryos obtained WT <br />                                                                                               unlabeled stage-matched embryos, disaggregated, entire cell <br /> adult zebrafish (AB, WIK, EK) natural mating. pKaede-S1 vector (MBL <br />                                                                                               population sorted isolate red green control cells. <br /> International) subcloned pRSET B (Invitrogen), expressed E. coli, <br /> and purified using Ni-NTA agarose affinity, sizing, hydroxyapatite col- <br />                                                                                               Microarray Hybridization, Transcriptomes Analysis, Validation. In-house <br /> umns (expression purification ProteinOne Inc.). The protein stored <br />                                                                                               microarray chips used (13). Four biological replicates mesendoderm <br /> in 5 mM Hepes (pH 7) 0.2 M KCl buffer &#226;&#171;&#186;80&#194;&#176;C. For kaede dusp4 <br />                                                                                               ectoderm precursor probes cohybridized, dye swap, <br /> mRNA, NotI-linearized Kaede-pCS2&#226;&#171;&#185; (32) dusp4-pCS2 used tem- <br />                                                                                               total hybridizations. Similarly, biological replicates of <br /> plate mMessage mMachine synthesis using SP6 RNA polymerase (Ambion). <br />                                                                                               red-cell green-cell probes dye swap cohybridized total <br /> Kaede protein injections 3.5 ng/embryo mRNA injections 300 pg/ <br />                                                                                               additional hybridizations. Hybridizations performed overnight at <br /> embryo carried 1&#226;&#8364;&#8220;2 cell stage embryos subsequently <br />                                                                                               45&#194;&#176;C Maui Mixer FL hybridization chambers (BioMicro Systems). Post- <br /> incubated 28&#194;&#176;C. Embryos stage mounted their <br />                                                                                               hybridization processing scanning previously described (13). Data <br /> chorions agarose array conical imprints (13, 14). By using laser <br />                                                                                               points quality values &#226;&#172;?0.85 eliminated, dataset was <br /> scanning microscope (Carl Zeiss LSM 510 META), Kaede photoconverted <br />                                                                                               normalized median shift following log2 transformation using the <br /> in regions interest, margin, 405-nm laser imaged via <br />                                                                                               Avadis 4.0 software program. P values average log ratios deter- <br /> 488-nm 543-nm excitation green red Kaede, respectively. Emitted <br />                                                                                               mined oligo array described ref. 13 predicted genes, <br /> light respectively filtered 505&#226;&#8364;&#8220;530-nm BP 560-nm LP filters. <br />                                                                                               ESTs, duplicates disregarded. Representation molecular catego- <br /> For loss-of-function studies, 1&#226;&#8364;&#8220;3 ng antisense splicing MO (Gene Tools), <br />                                                                                               ries enriched genes mesendoderm precursor vs. ectoderm <br /> targeting zygotic dusp4 translation (5&#226;&#172;&#732;-TAGCCCCCTGTAAAAGTGGAAAAGG- <br />                                                                                               precursor comparisons red vs. green control-cell comparisons was <br /> 3&#226;&#172;&#732;), injected cell-stage embryos. A similar, nonoverlapping MO <br />                                                                                               preliminarily assessed GeneSifter software (VizX Labs), looking ap- <br /> (5&#226;&#172;&#732;-TGCTGCTTGTGTATTTACCTGGTCG-3&#226;&#172;&#732;) yielded indistinguishable results. <br />                                                                                               proximately 100 genes available RefSeq IDs. This assessment was <br /> For control embryos, 5 ng standard control MO (5&#226;&#172;&#732;-TAGTTAAGC- <br />                                                                                               followed hand-curated analysis 60 most-enriched genes each <br /> CTAGCTCTCATAAACT-3&#226;&#172;&#732;) injected. For rescue, 50-pg full-length dusp4 <br />                                                                                               category (Table S2). <br /> mRNA coinjected. <br />                                                                                                  For validation studies, primers designed amplify coding se- <br />                                                                                               quence 5&#226;&#172;&#732; 3&#226;&#172;&#732; UTRs genes selected study cloned by <br /> Embryonic Cell Sorting Transplantation. Embryos dechorionated with <br />                                                                                               PCR TOPO TA vectors (Invitrogen). DIG-labeled RNA probes pre- <br /> Pronase (P8811, Sigma) transferred siliconized glass vials. Uninjected <br />                                                                                               pared using PCR-generated DNA templates DIG RNA labeling kit with <br /> embryos added samples &#226;&#172;?20 labeled embryos bring total <br />                                                                                               SP6 T7 polymerase (Roche), situ hybridizations carried <br /> number 20 &#226;&#8364;&#8220;30. For dissociation, embryos incubated 6 min 300 <br />                                                                                               essentially described ref. 33, post-hybridization washes per- <br /> &#226;?&#174;l trypsin solution (27) triturated transfer pipette midin- <br />                                                                                               formed automated liquid exchanger (Biolane HTI, Ho&#194;&#168;lle &amp; Hu&#194;&#168;ttner). <br /> cubation. After dissociation trypsin, 30 &#226;?&#174;l heat-inactivated FBS (HI-FBS) <br />                                                                                                  All zebrafish work approved National Human Genome Research <br /> (Gibco) added vial flooded L-15 Medium (Sigma) <br />                                                                                               Institute&#226;&#8364;&#8482;s Animal Care Use Committee. <br /> containing 0.1% BSA, 0.8 mM CaCl2, 2 mM L-glutamine, 100 U/ml penicillin, <br /> and 0.1 mg/ml streptomycin. The cells centrifuged 100 &#226;&#171;&#187; g 4 min <br /> at 4&#194;&#176;C &#226;&#172;&#8225;700 &#226;?&#174;l media removed. The cells transferred                   ACKNOWLEDGMENTS. We thank Darryl Leja artwork expert assis- <br />                                                                                               tance graphics; Alexandra Joyner, Paul Liu, David Bodine, Igor Dawid, <br /> to Titertube Micro Tube (Bio-Rad) inside 5-ml polystyrene tube, 70 <br />                                                                                               Mary LaMarca helpful comments manuscript; Erich Roessler and <br /> &#226;?&#174;l HI-FBS added. The cells transported ice flow cytometer                 Stacie Anderson valuable discussions; Atsushi Miyawaki sharing <br /> (FACSDiVa Vantage, BD Biosciences) sorted based                      Kaede-pCS2&#226;&#171;&#185; vector. This research supported Intramural <br /> forward scatter vs. scatter green vs. red fluorescence using                    Research Program National Human Genome Research Institute, Na- <br /> 100-&#226;?&#174;m tip 8 &#226;&#8364;&#8220;10 psi TRIzol (Invitrogen) RNA isolation cell             tional Institutes Health. <br />  <br />  <br />  1. Haeckel E (1874) Die gastraea-theorie: Die phylogenetische classification des thier-       9. Ho RK, Kimmel CB (1993) Commitment cell fate early zebrafish embryo. <br />     reichs und die homologie der keimbla&#194;&#168;tter. Jenaische Zeitschrift fu&#194;&#168;r Naturwissen-            Science 261:109 &#226;&#8364;&#8220;111. <br />     schaften 8:1&#226;&#8364;&#8220;55.                                                                          10. Sagerstrom CG, Gammill LS, Veale R, Sive H (2005) Specification enveloping <br />  2. Hatada Y, Stern CD (1994) A fate map epiblast early chick embryo.               layer lack autoneuralization zebrafish embryonic explants. Dev Dyn <br />     Development 120:2879 &#226;&#8364;&#8220;2889.                                                                   232:85&#226;&#8364;&#8220;97. <br />  3. Rendl M, Lewis L, Fuchs E (2005) Molecular dissection mesenchymal-epithelial           11. Ellingsen S, et al. (2005) Large-scale enhancer detection zebrafish genome. <br />     interactions hair follicle. PLoS Biol 3:e331.                                          Development 132:3799 &#226;&#8364;&#8220;3811. <br />  4. Sherwood RI, et al. (2007) Prospective isolation global gene expression analysis   12. Fan X, et al. (2007) Nodal signals mediate interactions extra-embryonic <br />     definitive visceral endoderm. Dev Biol 304:541&#226;&#8364;&#8220;555.                                       embryonic tissues zebrafish. Dev Biol 310:363&#226;&#8364;&#8220;378. <br />  5. Ando R, Hama H, Yamamoto-Hino M, Mizuno H, Miyawaki A (2002) An optical marker            13. Pei W, et al. (2007) An early requirement maternal FoxH1 zebrafish gastru- <br />     based UV-induced green-to-red photoconversion fluorescent protein. Proc           lation. Dev Biol 310:10 &#226;&#8364;&#8220;22. <br />                                                    <br /> </body></html>