<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Met30p, a Yeast Transcriptional Inhibitor That Responds <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                 S-Adenosylmethionine, Is an Essential Protein <br />                             with WD40 Repeats <br />  <br />              A specific repression mechanism regulates the biosynthesis of sulfur amino acids in Saccharomyces cerevisiae. <br />           When the intracellular S-adenosylmethionine (AdoMet) concentration increases, transcription of the sulfur <br />           genes is repressed. Using a specific reporter system, we have isolated mutations impairing the AdoMet- <br />           mediated transcriptional regulation of the sulfur network. These mutations identified a new gene, MET30, and <br />           were shown to also affect the regulation of the methyl cycle. The MET30 gene was isolated and sequenced. <br />           Sequence analysis reveals that Met30p contains five copies of the WD40 motif within its carboxy-terminal part, <br />           like the yeast transcriptional repressors Hir1p and Tup1p. We identified one target of Met30p as Met4p, a <br />           transcriptional activator regulating the sulfate assimilation pathway. By the two-hybrid method, we showed <br />           that Met30p interacts with Met4p and identified a region of Met4p involved in this interaction. Further <br />           analysis reveals that expression of Met30p is essential for cell viability. <br />  <br />  <br />    Transcriptional regulation of gene expression can be achieved                It is not yet known whether the transcriptional activation <br /> through either activation or repression mechanisms. While the                of sulfur genes involves additional factors. Furthermore, the <br /> processes by which transcription factors increase the rate of                molecular events responsible for the AdoMet-induced repres- <br /> transcription have been extensively investigated, only a few                 sion of the sulfur genes are not clearly established. Functional <br /> systems of repression have been studied. The sulfur network of               dissection of Met4p has shown that one part of the AdoMet- <br /> the yeast Saccharomyces cerevisiae constitutes one metabolic                 mediated repression occurs through the inhibition of the <br /> pathway that is negatively regulated at the level of transcrip-              transcription activation function of Met4p (27). A second com- <br /> tion. This pathway involves a set of enzymes catalyzing the bio-             ponent of AdoMet-mediated repression involves a proximal <br /> synthesis of cysteine, methionine, and S-adenosylmethionine                  sequence element that has been mapped on the MET25 pro- <br /> (AdoMet). The sulfur network comprises more than 20 un-                      moter (37). <br /> linked genes, the expression of which is regulated by AdoMet:                   To gain new insights into the mechanisms permitting the <br /> in response to an increase in the intracellular AdoMet concen-               regulation of sulfur metabolism in S. cerevisiae, we have devel- <br /> tration, transcription of the sulfur genes is turned off (37, 39).           oped a genetic screen aimed at isolating mutations in genes <br />    To date, two regulatory factors are known to be involved in               encoding factors involved in transcriptional regulation of the <br /> the transcriptional control of this metabolic network. The first             sulfur genes. Our screen yielded mutations in two genes. In this <br /> one, Met4p, belongs to the basic leucine zipper protein (bZIP)               report, we describe one of them, MET30, that encodes a pro- <br /> family and functions as a transcriptional activator (38). Its                tein containing five WD40 motifs. The Met30 protein (Met30p) <br /> activity was shown to be sensitive to the level of intracellular             was shown to function in the AdoMet-mediated inhibition of <br /> AdoMet (27). The second one, Cbf1p (also called Cp1p and                     the transcription function of Met4p. Moreover, we have found <br /> Cpf1p), belongs to the basic helix loop helix protein family,                that the role of Met30p extends to other parts of the cellular <br /> which recognizes the DNA motif RCACGTG (where R is a                         metabolism and that Met30p is essential for cell viability. <br /> purine) (1, 7, 30). This motif is present in either one or two <br /> copies in the promoter region of almost all genes involved in                                   MATERIALS AND METHODS <br /> sulfur metabolism (28). This motif also constitutes the centro- <br /> mere-determining element 1 (CDE1) of all the centromeres in                     Strains, media, and microbiological techniques. Escherichia coli JM103 was <br /> S. cerevisiae (19). Accordingly, inactivation of the gene CBF1               used as the host for plasmid maintenance. The yeast strains used in this work are <br /> has two phenotypic consequences: methionine auxotrophy and                   listed in Table 1. Strain C161 was constructed by integrating the MET25-xylE <br />                                                                              gene fusion at the LEU2 locus of strain W303-1B. Strain W303-1B was trans- <br /> a 9- to 25-fold increase in the rate of mitotic chromosome loss              formed by plasmid pYiX25 linearized at the unique KpnI site located in the <br /> (1, 7). trans as well as cis experiments have shown that Cbf1p               LEU2 gene, and leucine prototophic transformants were selected. Correct inte- <br /> is indeed required for the expression of several genes of sulfur             gration was assessed in strain C161 by Southern blotting. <br /> metabolism (28, 33, 38). However, DNA-bound Cbf1p does                          Standard yeast media were prepared as described by Cherest and Surdin- <br />                                                                              Kerjan (9). S. cerevisiae was transformed after lithium chloride treatment as <br /> not appear to act as a transcriptional activator; it lacks tran-             described by Gietz et al. (16). Genetic crosses, sporulation, dissection, and <br /> scription activation function but rather appears to aid local                scoring of markers were done as described by Sherman et al. (35). <br /> reconfiguration of the chromatin structure in the transcription                 Yeast strains expressing the Pseudomonas putida catechol oxidase were re- <br /> initiation regions (33, 38).                                                 vealed by using a 0.25 M catechol aqueous solution sprayed on colonies grown <br />                                                                              for 48 h on plates. The yellow color develops within 10 min. <br />                                                                                 Mutagenesis and genetic analysis. Strain W303-1B was transformed with <br />                                                                              plasmid pYrX25 (bearing a MET25-xylE fusion gene [22]). Mutants were ob- <br />   * Corresponding author. Phone: 1-69-82-34-64. Fax: 1-69-82-31-29.          tained by subjecting the transformed strain to UV mutagenesis. A thousand <br /> Electronic mail address: kerjan@cgmvax.cgm.cnrs-gif.fr.                      viable cells per petri dish were plated on minimal synthetic medium containing <br />  <br />                                                                       6526 <br />  VOL. 15, 1995                                                                              YEAST TRANSCRIPTIONAL INHIBITOR Met30                              6527 <br />  <br />  <br />                        TABLE 1. Yeast strains used                                   tains four lexA operators at position 2167 (18). We used oligonucleotides to <br />                                                                                      amplify a fragment of pM30-1 that was cloned into plasmid pEG202 (17) di- <br />                                                                   Reference or       gested by BamHI, yielding pLexM30-4. Amplifications were performed with Pfu <br />    Strain                   Relevant genotype <br />                                                                      source          DNA polymerase (Stratagene, La Jolla, Calif.). The amplified fragment corre- <br />                                                                                      sponds to the Met30p region 4 to 640. Plasmids JK1621 and CK30 were gener- <br /> C119          MATa his3 ura3 leu2::SAM2-lacZ::LEU2              41                   ously provided by A. D. Johnson. <br /> C121          MATa his3 ura3 leu2::SAM1-lacZ::LEU2              41                      Enzymatic assays. Yeast cells were grown to 107/ml in appropriate minimal <br /> C161          MATa his3 leu2 ade2 trp1 ura3 leu2::              22                   synthetic medium. Acellular extracts were made as described by Thomas et al. <br />                MET25-xylE::LEU2                                                      (38) in the presence of 1 mM phenylmethylsulfonyl fluoride and 1 mM EDTA. <br /> C170          MATa his3 leu2 ade2 trp1 met4::TRP1               27                   After centrifugation for 15 min at 10,000 3 g, the crude extract was used for <br />                                                                                      enzymatic assays. ATP sulfurylase, 39-phosphoadenosyl 59-phosphosulfate (PAPS) <br />                ura3::lexAop-lacZ::URA3 <br />                                                                                      reductase, and homocysteine synthase were assayed as described by Thomas et al. <br /> C172          MATa his3 leu2 ade2 trp1 met4::TRP1               This study           (38). b-Galactosidase activity was assayed as described by Kuras and Thomas <br />                MET30-1 ura3::lexAop-lacZ::URA3                                       (27). Protein concentrations were estimated by the method described by Lowry <br /> CC572         MATa/MATa his3/his3 leu2/leu2 ura3/ura3           22                   et al. (29). <br />                ade2/ade2 trp1/trp1                                                      Nucleotide sequence accession number. The MET30 gene sequence has been <br /> CD126         MATa/MATa his3/his3 leu2/leu2 ura3/ura3           This study           assigned EMBL/<span id='am-3' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> L26505. <br />                ade2/ade2 trp1/trp1 MET30/met30::URA3 <br /> CC682-24      MATa his3 ura3 leu2::SAM1-lacZ::LEU2              This study <br />                MET30-1                                                                                               RESULTS <br /> CC687-1A      MATa his3 ura3 leu2::SAM2-lacZ::LEU2              This study <br />                MET30-1                                                                  As previously reported (22), we have developed a specific <br /> CI2-11D       MATa trp1 ura3 leu2::MET25-xylE::LEU2             This study           reporter system allowing the genetic analysis of the transcrip- <br /> CM100         MATa/MATa ade2/ADE2 his3/HIS3 trp1/               This study           tional mechanisms that regulate the sulfur network. In this <br />                trp1 ura3/ura3 leu2::MET25-xylE::LEU2/                                system, heterologous expression of the catechol oxidase <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                leu2::MET25-xylE::LEU2                                                P. putida (encoded by the xylE gene [47]) is provided under the <br /> CM100-1A      MATa ade2 his3 MET30-1 trp1 ura3                  This study           control of the MET25 promoter. Yeast colonies expressing the <br />                leu2::MET25-xylE::LEU2                                                P. putida catechol oxidase turn yellow in a few minutes when <br /> CM100-1B      MATa trp1 ura3 leu2::MET25-xylE::LEU2             This study <br />                                                                                      sprayed with catechol. In growth conditions which lead to re- <br /> CM100-1C      MATa his3 trp1 ura3 leu2::MET25-xylE::            This study <br />                LEU2 <br />                                                                                      pression of the sulfur network, i.e., addition of 1 mM L-methi- <br /> CM100-1D      MATa ade2 MET30-1 trp1 ura3 leu2::                This study           onine to the medium, the colonies remain white after catechol <br />                MET25-xylE::LEU2                                                      spraying. A search for mutations that give rise to yellow colo- <br /> X2180-1A      MATa                                              Yeast Genetic        nies even in the presence of repressing amounts of methionine <br />                                                                   Stock Center       should thus identify genes involved in AdoMet-induced repres- <br /> W303-1A       MATa ade2 his3 leu2 trp1 ura3                     R. Rothstein         sion. Indeed, it was previously established that the repressive <br />                                                                                      effect of added methionine is actually mediated through an <br />                                                                                      increase in the intracellular AdoMet concentration as a result <br />                                                                                      of the efficient transformation of methionine into AdoMet by <br /> all auxotrophic requirements of the strain and 1 mM L-methionine. After 3 days       the two AdoMet synthases. <br /> at 288C, the plates were sprayed with a catechol solution. Yellow colonies were <br /> picked and streaked for single colonies on selection medium. From 45,000 <br />                                                                                         Strain W303-1B was transformed with a plasmid bearing the <br /> colonies, 14 mutants exhibiting yellow or faint yellow coloration in the catechol    MET25-xylE gene fusion and subjected to UV mutagenesis. <br /> test were selected for further study. They were cured of the plasmid by growth for   Mutants that were yellow when sprayed with catechol after <br /> 30 generations on a nonselective medium. They were then retransformed with           growth in presence of 1 mM L-methionine were isolated (see <br /> plasmid pYrX25 and tested once again by the catechol test. Of the primary <br /> isolates, three exhibited clearly a yellow <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> in the catechol test after <br />                                                                                      Materials and Methods). Here we report the analysis of one of <br /> growth in repressive conditions (1 mM L-methionine). The MET25-xylE fusion           these mutants, the yellow phenotype of which was shown to <br /> gene was integrated at the LEU2 locus of the three mutants, and the resulting        result from a <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> in a single gene that we called MET30 <br /> strains were mated to strain CI2-11D. In the three cases, tetrad analyses of the     (see Materials and Methods). <br /> progeny of heterozygotes displayed a 21:22 segregation <span id='am-243' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of the yellow <br /> color. This indicates that a mutation affecting a single gene was causing the <br />                                                                                         MET30 mutations impair AdoMet-mediated repression of <br /> phenotype of each mutant. Diploids containing two different mutations were           the sulfur network. To confirm that a mutation in MET30 <br /> made, and tetrad analysis of the progeny showed that two of the three mutants        impaired the repression of the sulfur network by AdoMet, we <br /> studied belonged to the same complementation group. This group defined a new         first performed enzymatic analyses with the four spores of a <br /> gene that was named MET30. <br />    Recombinant DNA methods. Plasmids pEmblYe23, -25, and -31 (2) were used <br />                                                                                      tetrad isolated after sporulation of diploid CM100 (MET30-1/ <br /> as shuttle vectors between S. cerevisiae and E. coli. The S. cerevisiae genomic      MET30). As described below, MET30-1 is a dominant allele, <br /> library used for the cloning of the gene MET30 was constructed by inserting the      indicated by capital letters. In the two MET30-1 spores, the <br /> product of a partial Sau3A digest of chromosomal DNA from the wild-type strain       activity of homocysteine synthase (encoded by MET25 [24]), <br /> X2180-1A in the BamHI site of pEMBLYe23. Northern (RNA) blotting was <br /> performed as described by Thomas (42), with total cellular RNA extracted from <br />                                                                                      measured after growth in the presence of a repressing amount <br /> yeast cells as described by Hoffman and Winston (21). Probes were made radio-        of methionine, was three- to fourfold higher than that mea- <br /> active by the random priming method described by Hodgson and Fisk (20).              sured in the two MET30 spores grown in the same conditions <br />    For the sequencing of MET30, systematic deletion subclones were generated         (Table 2). The MET30-1 mutation also affects the AdoMet- <br /> by the method of Thomas and Surdin-Kerjan (40). Single-stranded phage DNA <br /> prepared from these deletions were sequenced with the Pharmacia T7 sequenc- <br />                                                                                      mediated repression of other genes from the sulfate assimila- <br /> ing kit. Sequence analyses and comparisons were performed by using the Bisance       tion pathway. As shown in Table 2, in MET30-1 spores, both <br /> service (12). To disrupt MET30, the BamHI-BamHI fragment of pM30-1 was               ATP sulfurylase and PAPS reductase activities were increased <br /> first cloned into plasmid pUC19. The resulting plasmid was then digested with        5- to 10-fold after growth in the presence of a repressing <br /> EcoRV, dephosphorylated, and ligated with a blunt-ended BglII-BglII fragment <br /> containing the URA3 gene. The resulting plasmid was then digested with BamHI, <br />                                                                                      amount of methionine compared with the activities measured <br /> and the product of the digestion was used to transform a wild-type diploid strain.   in MET30 spores. <br />    The pGADM4Di plasmids were constructed by transferring the EcoRI-BamHI               We performed a Northern analysis to confirm that the dif- <br /> fragments from the corresponding plasmid pLexM4Di (27) into plasmid pGAD424          ferences in the level of homocysteine synthase activity are due <br /> (3). For the two-hybrid analysis, plasmids pLexM30-4 and pGadM4Di were co- <br /> transformed into strain C170, which contains a GAL1-lexAop-lacZ gene fusion <br />                                                                                      to modifications in MET25 gene transcription. The repression <br /> integrated at the ura3 locus. In the GAL1-lexAop-lacZ fusion reporter, the GAL       of MET25 was monitored by shifting wild-type and MET30-1 <br /> upstream activation sequence (UASGAL) has been deleted, and instead it con-          cells growing in a medium lacking methionine to one contain- <br />  6528           THOMAS ET AL.                                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />         TABLE 2. Effect of the MET30-1 mutation on the sulfate                            TABLE 3. Effect of the MET30-2 mutation on the AdoMet- <br />                         assimilation pathwaya                                                    mediated regulation of SAM1 and SAM2a <br />                                              Sp act (nmol/min/mg of protein)                                                                b-Galactosidase sp act <br />                  Relevant     Growth                                                                                            Growth <br />                                                                                          Strain        Relevant genotype                      (nmol/min/mg of <br />       Strain <br />                  genotype    conditions   Homocysteine    ATP        PAPS                                                     conditionsb <br />                                                                                                                                                   protein) <br />                                             synthase   sulfurylase reductase <br />                                                                                       C121           SAM1-lacZ                   NR                   685 <br /> CM100-1A MET30-1                NR              410            168         0.26                                                  R                     72 <br />                                 R               208             40         0.10       CC682-24       SAM1-lacZ MET30-2           NR                   930 <br /> CM100-1B MET30                  NR              254            150         0.18                                                  R                    550 <br />                                 R                62            ,5         ,0.03       C119           SAM2-lacZ                   NR                 2,620 <br /> CM100-1C MET30                  NR              220            144         0.13                                                  R                    510 <br />                                 R                68            ,5         ,0.03       CC687-1A       SAM2-lacZ MET30-2           NR                 2,120 <br /> CM100-1D MET30-1                NR              340            170         0.21                                                  R                  1,440 <br />                                 R               183             63         0.16 <br />                                                                                          a <br />                                                                                            b-Galactosidase activity was expressed from either a SAM1-lacZ or SAM2- <br />   a <br />     The specific activities of enzymes involved in the sulfate assimilation pathway   lacZ fusion gene integrated in the chromosome of MET30 and MET30-2 mutant <br /> were assayed in wild-type and MET30-1 spores grown in either nonrepressive            strains. Data are the averages of two independent experiments (deviation was <br /> (NR) or repressive (R, 1 mM L-methionine) growth conditions. Data are the             less than 10%). <br /> averages of two independent experiments (deviation was less than 10%).                   b <br />                                                                                            NR, nonrepressive; R, repressive (1 mM L-methionine). <br />  <br />  <br />  <br /> ing repressing amounts of methionine. Total RNA was then                              a wild-type strain. The results of this experiment (Table 3) <br /> extracted at different times after the shift and analyzed with a                      prove that Met30p indeed participates in the AdoMet-medi- <br /> MET25-specific probe. As shown in Fig. 1, in wild-type cells,                         ated regulation of these two genes. Accordingly, the resistance <br /> addition of repressing amounts of methionine to the medium                            to S-adenosylethionine was shown to be achieved through a <br /> results in a rapid repression of transcription: MET25 mRNAs                           large accumulation of intracellular AdoMet (data not shown). <br /> are four times less abundant 10 min after the addition of                                In addition, assays of homocysteine synthase activity in <br /> methionine and are virtually undetectable 20 min after the                            MET30/MET30 and MET30/MET30-1 diploid cells showed <br /> shift. By contrast, in MET30-1 cells, MET25 mRNA levels                               that the MET30-1 allele is dominant (Table 4) (the same result <br /> decrease more slowly, and about 20% of MET25 mRNAs re-                                was obtained for the MET30-2 allele [not shown]). <br /> main 40 min after the addition of methionine to the growth                               Cloning of MET30. We first tried to clone the wild-type <br /> medium. The MET30-1 mutation thus prevents the complete                               allele of the MET30 gene by using the MET25-xylE gene re- <br /> repression of MET25 transcription by AdoMet.                                          porter system. We transformed strain CM100-1A (MET30-1 <br />    Next, genetic analyses showed that the MET30-1 mutation                            MET25-xylE::LEU2) with a multicopy-plasmid-based genomic <br /> was allelic to a mutation isolated in our laboratory several                          library and selected colonies that were white when sprayed <br /> years ago and known to allow growth in presence of 0.1 mM                             with catechol on a medium containing 1 mM L-methionine. For <br /> S-adenosylethionine, a potent toxic analog of AdoMet (this                            unknown reasons, only false-positive clones were obtained. We <br /> allele is thereafter called MET30-2). This result led us to ex-                       thus determined whether the MET30-1 mutation might result <br /> amine whether Met30p might control the regulation of the                              in a particular phenotype that could lead to a positive screen- <br /> genes SAM1 and SAM2, which encode the two AdoMet syn-                                 ing procedure. Selenate is a well-known toxic analog of sulfate <br /> thases in yeast cells. For this purpose, we measured the activity                     that inhibits the growth of S. cerevisiae at concentrations higher <br /> of two reporter genes, SAM1-lacZ and SAM2-lacZ, which were                            than 1 mM (5). A wild-type yeast strain grown in the presence <br /> integrated in the chromosome of either a MET30-1 mutant or                            of high concentrations of organic sulfur compounds is resistant <br />                                                                                       to selenate because of the repression of the sulfate transport <br />                                                                                       system by AdoMet. A MET30 mutant was expected to be <br />                                                                                       sensitive to selenate even in the presence of organic sulfur <br />                                                                                       compounds because, in these conditions, it expresses the sul- <br />                                                                                       fate transport system. Plate assays revealed that wild-type but <br />                                                                                       not MET30-1 strains were indeed capable of growth in the <br />                                                                                       presence of both 5 mM selenate and 1 mM L-methionine. <br />                                                                                          Strain CM100-1A was transformed with a pEmblYe23-based <br />                                                                                       genomic library, and about 30,000 Ura1 transformants were <br />                                                                                       selected on 25 plates. These transformants were recovered by <br />  <br />  <br />    FIG. 1. Transcriptional repression of a sulfur gene in a MET30-1 mutant.                       TABLE 4. Dominance of the MET30-1 mutationa <br /> The repression kinetics of the transcription of gene MET25 was monitored by <br /> shifting a wild-type and a MET30-1 mutant strain from a medium without                                                                      Homocysteine synthase <br />                                                                                                                              Growth <br /> methionine to one containing a repressing amount of methionine. The strains            Strain       Relevant genotype                        sp act (nmol/min/mg <br />                                                                                                                            conditionsb <br /> used were W303-1A (MET30) and CM100-1A (MET30-1). Strains were grown in                                                                           of protein) <br /> B medium (9) in the presence of 0.1 mM sulfate as the sulfur source. When the <br /> cell concentration reached about 107/ml, L-methionine was added to the medium         CC572         MET30/MET30               NR                    345 <br /> at a final concentration of 1 mM. Samples were then withdrawn at different                                                    R                      56 <br /> times, and total RNAs were extracted. For each time point, 10 mg of total RNA         CM100         MET30/MET30-1             NR                    370 <br /> was electrophoresed on a 1% agarose gel and transferred to a nylon membrane.                                                  R                     180 <br /> The transferred RNAs were hybridized to the radioactive XbaI-EcoRV fragment <br />                                                                                         a <br /> from the MET25 gene (24). The half-life of MET25 mRNA was previously shown                Data are the averages of two independent experiments (deviation was less <br /> to be about 8 min (24). The actin probe was used as a control for the amounts         than 10%). <br />                                                                                         b <br /> of RNA loaded.                                                                            NR, nonrepressive; R, repressive (1 mM L-methionine). <br />  VOL. 15, 1995                                                                       YEAST TRANSCRIPTIONAL INHIBITOR Met30                    6529 <br />  <br />  <br />                                                                                two classes of proteins. The first one includes the b subunits of <br />                                                                                the heterotrimeric G complexes; these subunits are made up <br />                                                                                almost entirely of WD40 repeats (31). The second class in- <br />                                                                                cludes proteins which contain between four and eight repeats <br />                                                                                of the WD40 motif together with N-terminal or C-terminal <br />                                                                                extensions. These proteins function in numerous cellular pro- <br />                                                                                cesses, such as vesicular traffic, regulation of cytoskeletal as- <br />                                                                                sembly, RNA processing, and regulation of gene expression <br />                                                                                (reviewed by Neer et al. [32]). The function of WD40 repeats <br />                                                                                is not exactly known, although it has been suggested that they <br />                                                                                mediate protein-protein interactions. Interestingly, it seems <br />                                                                                that transcription factors bearing WD40 repeats, such as <br />    FIG. 2. Physical map of the MET30 region. The fragments subcloned in <br />                                                                                Tup1p, Cop1p, and Hir1p, are more specifically involved in <br /> plasmid pEmblYe23 and their ability to complement the mutation of strain       repression mechanisms (11, 36, 45). One of the most studied <br /> CM100-1A (ura3 MET30-1) are shown. The ability of transformants plated on B    among these factors is the protein Tup1 of S. cerevisiae. Tup1p <br /> medium containing 5 mM selenate plus 1 mM L-methionine to grow is indicated;   forms a complex with Ssn6p, a protein that contains 10 repeats <br /> 6, only small colonies that could not grow further appeared. <br />                                                                                of the tetratricopeptide (TPR) motif (45, 46). The complex <br />                                                                                Ssn6-Tup1 is known to be a general repressor involved in the <br />                                                                                repression of several sets of genes, such as a-cell-specific genes, <br /> washing the plates, and the resulting cell suspensions were                    haploid genes, and glucose-repressible genes (14, 34, 43). Nei- <br /> plated onto B medium containing 5 mM selenate and 1 mM                         ther Tup1p nor Ssn6p possesses a DNA-binding motif. <br /> L-methionine (B medium is a synthetic sulfur-free medium                          DNA-bound Met30p does not repress transcription. Models <br /> [9]). Cells that could grow in these conditions were derived                   explaining how a general repressor could recognize a wide <br /> from two plates of the first selection step. Plasmid DNA was                   variety of promoters invoke multiple protein interactions: the <br /> isolated from eight colonies and used to retransform strain                    complex Ssn6-Tup1 locates the genes that it represses by rec- <br /> CM100-1A. All eight DNA preparations led to transformants                      ognizing specific DNA-bound proteins present at each pro- <br /> that were resistant to selenate on B medium containing 1 mM                    moter. Several lines of evidence support this model. Especially, <br /> L-methionine. These results were confirmed by using the cat-                   it was shown that both the Tup1p and Ssn6p proteins can <br /> echol test. All transformants were white when sprayed with                     repress the transcription of a functional promoter when they <br /> catechol after growth on a medium containing repressing                        are bound upstream through a LexA DNA-binding domain <br /> amounts of methionine. The eight clones were found to harbor                   (23, 44). Since Met30p does not appear to be a DNA-binding <br /> identical plasmids with a 2.9-kbp insert. Subcloning experi-                   protein, it seemed plausible that Met30p could function in a <br /> ments showed that no smaller complementing fragment could                      similar way in mediating the AdoMet-mediated repression of <br /> be easily obtained (Fig. 2).                                                   the sulfur network. We thus tested whether Met30p could act <br />    To determine whether the insert directed integration to the                 as a repressor when brought to DNA at a promoter. We con- <br /> MET30 region of the yeast genome, strain C161 (ura3 MET30                      structed a fusion between LexA and the intact Met30 protein <br /> MET25-xylE::LEU2) was transformed with a plasmid lacking                       and tested it with two CYC1-lacZ gene fusions. One gene <br /> an autonomous yeast replication sequence but bearing the                       fusion (plasmid JK1621) contains four lexA operators up- <br /> URA3 gene and the complementing insert. Integration was                        stream of the CYC1 UAS; the other (plasmid CK30) contains <br /> directed by cleaving the plasmid within the insert with BamHI.                 one lexA operator positioned between the UAS and the TATA <br /> One Ura1 transformant was crossed to strain CM100-1D (ura3                     box of the CYC1 promoter (23). b-Galactosidase assays of <br /> MET30-1 MET25-xylE::LEU2), the diploid was sporulated, and                     cotransformed cells (Table 5) grown in either nonrepressive or <br /> the tetrads were dissected. In all cases (20 tetrads), two spores              repressive conditions demonstrate that the LexA-Met30 fusion <br /> were yellow and two spores were white when sprayed with                        protein does not significantly affect the expression of the re- <br /> catechol after growth in the presence of repressing amounts of                 porter genes. Additional results, using the interference assay <br /> methionine. Furthermore, all of the segregants that were white                 developed by Brent and Ptashne (4), show that the LexA- <br /> after catechol spraying were uracil prototrophs. These results                 Met30 fusion protein is indeed capable of binding efficiently to <br /> indicate that the cloned insert directed integration to the                    lexA operators (not shown). These results thus indicate that the <br /> MET30 locus and therefore contained the MET30 gene.                            Met30 protein functions in repression of transcription differ- <br />    Five WD40 repeats in the C-terminal part of the MET30-                      ently than Tup1p. <br /> encoded product. The complementing insert of pM30-1 was                           AdoMet-mediated inhibition of the transcription activation <br /> entirely sequenced on both strands, and the result is shown in                 function of Met4p is Met30p dependent. Met4p, a member of <br /> Fig. 3. The DNA sequence contains a single long open reading                   the basic leucine zipper factor family, is a positive regulator of <br /> frame with the potential to encode a protein of 640 amino acid                 the sulfur network in S. cerevisiae. We have recently demon- <br /> residues. The predicted protein has a molecular mass of 72,800                 strated that the transcription activation function of Met4p is <br /> and a calculated isoelectric point of 7.8. Inspection of the                   specifically inhibited when the intracellular AdoMet level in- <br /> Met30p sequence does not reveal the presence of any canon-                     creases (27). Accordingly, when cells expressing a LexA-Met4 <br /> ical DNA-binding motif. However, the C-terminal part of                        fusion protein and containing a lacZ gene placed downstream <br /> Met30p contains five repeats of a structural motif of about 40                 of lexA operators are grown in the presence of a repressing <br /> amino acid residues that resemble a peptide motif first identi-                amount of methionine, a fourfold decrease in b-galactosidase <br /> fied in the b subunit of transducin, a G protein from photore-                 activity compared with that under nonrepressive growth con- <br /> ceptor rod cells, and called the b-transducin or WD40 motif                    ditions is measured (27). The regulation of the transcriptional <br /> (15). As shown in Fig. 4, the consensus sequence derived from                  activation function of Met4p was shown to be mediated by two <br /> the alignment of the repeats of Met30p is strongly similar to                  distinct domains within the protein, called the inhibitory region <br /> that derived for this repeating <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> by Dalrymple et al. (10).                 and the auxiliary domain (27). To determine whether the reg- <br /> The WD40 family is confined to eucaryotes and is composed of                   ulation of Met4p activity depends on the integrity of Met30p, <br />  6530      THOMAS ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Nucleotide and deduced amino acid sequences of the MET30 region. The nucleotide sequence is numbered from nucleotide 1 of the presumed initiation <br /> codon of MET30. <br />  <br />  <br />  <br />  <br /> various LexA-Met4p derivatives, each lacking one of the func-                   rity of Met30p. However, it must be noted that the MET30-1 <br /> tional domains identified within Met4p, were expressed from                     mutation also lowered the activity of the fusion proteins in <br /> plasmids introduced into MET30-1 cells. These cells moreover                    nonrepressive growth conditions. <br /> contained a lacZ reporter gene placed downstream of six lexA                       In vivo interactions between Met4p and Met30p. The results <br /> operators in the chromosome. In contrast to what we measured                    presented above are consistent with the possibility that Met30p <br /> in a wild-type strain, in MET30-1 cells, all of the LexA-Met4p                  interacts directly with Met4p. To test this hypothesis, we car- <br /> derivatives activate the reporter gene in repressive growth con-                ried out two-hybrid studies (13). We introduced various deriv- <br /> ditions as much as in nonrepressive growth conditions (Table                    atives of Met4p fused to the activation domain of Gal4p (Gad) <br /> 6). We therefore conclude that the AdoMet-mediated inhibi-                      into strains expressing the LexA-Met30 fusion protein. The <br /> tion of the transcription function of Met4p requires the integ-                 combination of LexA-Met30 fusion protein and the Gad-Met4 <br />  VOL. 15, 1995                                                                              YEAST TRANSCRIPTIONAL INHIBITOR Met30                             6531 <br />  <br />  <br />                                                                                          TABLE 6. Dependence of AdoMet-mediated inhibition of the <br />                                                                                                 transcription function of Met4p on Met30pa <br />                                                                                                                                    b-Galactosidase sp act <br />                                                                                                     Met4p                        (nmol/min/mg of protein) <br />                                                                                     Expressed                    Growth <br />                                                                                                     domain <br />                                                                                      protein                   conditionsb             Ratio,               Ratio, <br />                                                                                                     deleted                  MET30              MET30-1 <br />                                                                                                                                        NR/R                 NR/R <br />  <br />                                                                                    LexM4-1        None            NR         3,250               1,225 <br />                                                                                                                   R            800      4.06     1,160       1.05 <br />                                                                                    LexM4D14 Inhibitory            NR         3,150               1,200 <br />                                                                                                                   R          2,850      1.14     1,170       1.02 <br />                                                                                    LexM4D37 Auxiliary             NR           820                 380 <br />                                                                                                                   R            780      1.05       318       1.2 <br />                                                                                      a <br />                                                                                         Plasmids based on pLexM4Di and expressing various LexA-Met4p deriva- <br />                                                                                    tives were transformed into strain C170 (met4::TRP1 MET30 ura3::lexAop-lacZ:: <br />    FIG. 4. Met30p contains WD40 repeats. Amino acid residues are numbered <br />                                                                                    URA3) and C172 (met4::TRP1 MET30-1 ura3::lexAop-lacZ::URA3). Data are the <br /> on the left and on the right. Residues conserved within the Met30 protein are in <br />                                                                                    averages of three assays performed with independent transformants (standard <br /> boldface. Conservation of a given amino acid on at least three occasions yields <br />                                                                                    deviation was less than 10%). <br /> the Met30p consensus sequence shown below the alignment. The reported con-            b <br />                                                                                         NR, nonrepressive (0.05 mM L-methionine), R, repressive (1 mM L-methi- <br /> sensus for the WD40 motif is from Dalrymple et al. (10). A schematic represen- <br />                                                                                    onine). <br /> tation of the WD40 motifs (solid boxes) within Met30p is shown at the top. <br />  <br />  <br />  <br /> derivative containing the entire <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> of Met4p confers 13-                       terials and Methods). This construction was used to transform <br /> fold-higher expression of the reporter gene than either protein                    the diploid strain CC572. Stable Ura1 transformants were <br /> alone (Table 7). This result suggests that Met4p and Met30p                        selected. DNA blot analysis of genomic DNA isolated from <br /> indeed interact in vivo. Moreover, the use of several Gad-Met4                     one of the transformed diploid strains (strain CD126) is shown <br /> fusion proteins allowed us to identify the region of Met4p that                    in Fig. 5. A 1.8-kb wild-type MET30 fragment is detected as <br /> associates with Met30p. Indeed, the two-hybrid interactions                        well as the two smaller fragments that are expected from cleav- <br /> are significantly lowered by deletion of the inhibitory region of                  age at the StuI site within the URA3 sequence. As shown in Fig. <br /> Met4p. Conversely, the Gad-Met4 derivatives lacking either                         5, sporulation of diploid CD126, heterozygous for met30:: <br /> the activation domain or the auxiliary domain of Met4p appear                      URA3, gave rise to tetrads containing no more than two viable <br /> to interact with the LexA-Met30 fusion protein as well as the                      spores that were invariably auxotrophs for uracil. The absence <br /> original Gad-Met4 hybrid (Table 7). These results suggest that                     of uracil prototroph spores suggested that MET30 inactivation <br /> the inhibitory region of Met4p is necessary for the formation of                   is lethal. In order to confirm this result, we transformed diploid <br /> the Met4-Met30 complex. In addition, it must be noted that                         CD126 (MET30/met30::URA3) with a multicopy plasmid bear- <br /> interactions between Met4p and Met30p cannot be easily de-                         ing the wild-type allele of MET30 and the gene LEU2 (pM30-5). <br /> tected by the two-hybrid method when the cells are grown in                        The transformed diploid was sporulated. As expected, tetrads <br /> the presence of a repressing amount of methionine. Indeed,                         containing four, three, and two viable spores were obtained. <br /> when fused to Met4p, the Gal4p activation domain is repressed                      Scoring of the nutritional requirements of these spores showed <br /> by the Met4p regulatory regions when the intracellular                             that all of the uracil prototroph spores obtained were also <br /> AdoMet level is high (27) (not shown).                                             leucine prototrophs. These cells (met30::URA3/pM30-5) were <br />    MET30 essential for cell viability. To determine whether the                    shown to be incapable of losing plasmid pM30-5 under nonse- <br /> role of Met30p extends to other areas of metabolism, we con-                       lective growth conditions: they were grown in complete me- <br /> structed a strain that did not express a functional Met30 pro-                     dium for 50 generations and plated. The 150 resulting colonies <br /> tein. The coding sequence of MET30 from codon 264 to codon                         were tested and shown to be, in all cases, both uracil and <br /> 448 was replaced with the selectable marker URA3 (see Ma-                          leucine prototrophs. In contrast, the same experiment per- <br />                                                                                    formed with uracil auxotroph spores (MET30/pM30-5) leads to <br />  <br />          TABLE 5. Function of LexA-Met30 fusion proteina <br />    Expressed         Reporter         Growth           b-Galactosidase sp act                  TABLE 7. Interactions of Met4p and Met30p in <br />     protein          plasmid        conditionsb      (nmol/min/mg of protein)                            the two-hybrid systema <br /> None                  JK1621            NR                     1,930                                                                 b-Galactosidase <br />                                         R                      1,650                                                   Met4p <br />                                                                                                                                       sp act (nmol/        Fold <br /> LexA-Met30p           JK1621            NR                     2,300                           Plasmids                domain <br />                                                                                                                                        min/mg of         activation <br />                                                                                                                        deleted <br />                                         R                      2,310                                                                     protein) <br /> None                  CK30              NR                     1,595 <br />                                         R                      1,300               pLexM30-4 1 pRS315               None                1.5 6 0.5 <br /> LexA-Met30p           CK30              NR                     1,100               pRS313 1 pGadM4-1                None                1.3 6 0.5 <br />                                         R                        950               pLexM30-4 1 pGadM4-1             None               20.1 6 1.0          13.4 <br />                                                                                    pLexM30-4 1 pGadM4D12            Activation         17.0 6 1.0          11.3 <br />   a <br />     Wild-type strain W303-1A was transformed either with a plasmid (pLexM30-4)     pLexM30-4 1 pGadM4D37            Inhibitory          5.5 6 0.7           3.6 <br /> expressing the fusion protein LexA-Met30 or with vector plasmid pRS313, to-        pLexM30-4 1 pGadM4D30            Auxiliary          19.5 6 2.1          13.0 <br /> gether with one plasmid containing the lexAop-CYC1-lacZ reporter gene. Plas- <br />                                                                                       a <br /> mid JK1621 contains four lexA operators upstream of the CYC1 UAS; plasmid               Strain C170 (met4::TRP1 ura3::lexAop-lacZ::URA3) was cotransformed with <br /> CK30 contains one lexA operator between the UAS and the TATA box of CYC1           plasmid pLexM30-4 and different plasmids expressing various Gad-Met4p deriv- <br /> (23). Data are the averages of two independent experiments performed with          atives. The cells were grown in nonrepressive growth conditions (0.05 mM L- <br /> three independent transformants (standard deviation was less than 10%).            methionine). Data are the averages of four assays performed with independent <br />   b <br />     NR, nonrepressive; R, repressive (1 mM L-methionine).                          transformants. <br />  6532        THOMAS ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. MET30 is essential for cell viability. (A) Physical map of the met30::URA3 region. (B) Southern blot analysis of MET30 gene disruption. Lane 1, <br /> BamHI-StuI-NheI digest of genomic DNA extracted from strain W303-1A; lane 2, BamHI-StuI-NheI digest of DNA extracted from strain CD126. The filter was probed <br /> with the BamHI-NheI fragment of MET30. The arrows indicate the two fragments expected from StuI cleavage within the URA3 gene. (C) Tetrads obtained from the <br /> sporulation of diploid CD126 (met30::URA3/MET30). <br />  <br />  <br />  <br />  <br /> 85% leucine auxotrophs. Taken together, these results prove                                                   DISCUSSION <br /> that MET30 is indeed an essential gene. <br />                                                                                     We report the identification of a new gene of S. cerevisiae, <br />    Since the two MET30 alleles isolated were shown to be <br />                                                                                  MET30, the product of which is involved in negative regulation <br /> dominant for the regulatory effect, it seemed important to <br />                                                                                  of the sulfur network. This gene was isolated via a specific <br /> determine whether the null met30 allele, which is recessive for <br />                                                                                  genetic screen that uses a fusion between the reporter gene <br /> lethality, is also recessive for the regulatory effect. We thus <br />                                                                                  xylE from P. putida and the promoter region of MET25, one <br /> assayed homocysteine synthase activity in the CD126 diploid <br />                                                                                  gene governing the sulfate assimilation pathway. Enzymatic <br /> strain (MET30/met30::URA3). The results show that, in strain <br />                                                                                  assays as well as Northern experiments confirm that mutations <br /> CD126 as in the parental wild-type strain, homocysteine syn- <br />                                                                                  in MET30 impair the transcriptional regulation of the sulfur <br /> thase activity is ninefold lower after growth in the presence of <br />                                                                                  network. <br /> a repressing amount of methionine (Table 8). The met30 null <br />                                                                                     MET30 was cloned, and sequence analysis revealed that its <br /> allele is thus recessive for the regulatory effect. <br />                                                                                  encoded product belongs to an expanding family of proteins <br />    In addition, we transformed the CD126 diploid strain with <br />                                                                                  that all comprise several repeats of a 40-amino-acid segment, <br /> plasmid pLexM30-4. A transformed diploid was sporulated. <br />                                                                                  called the WD40 motif. The Met30 protein contains five well- <br /> Tetrads containing four or three viable spores were obtained, <br />                                                                                  conserved repeats as well as one degenerate repeat of the <br /> and all the uracil prototroph spores were shown to be histidine <br />                                                                                  WD40 motif. The WD40 consensus sequence derived from the <br /> prototrophs. This indicates that the pLexM30-4 plasmid is ca- <br />                                                                                  alignment of the Met30p repeats is extremely similar to that <br /> pable of complementing the met30 <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>null allele</span>, demonstrating <br />                                                                                  described by Dalrymple et al. (10). The WD40 proteins have <br /> that the LexA-Met30 protein is active. <br />                                                                                  been found in eucaryotes but not in procaryotes. They are <br />                                                                                  involved in numerous cell processes, such as cell cycle regula- <br />                                                                                  tion, transcription regulation, transmembrane signaling, and <br />                                                                                  RNA splicing (32). All the members of the WD40 family seem <br />           TABLE 8. Recessive nature of met30::URA3 mutationa <br />                                                                                  to have a regulatory function, and none is known to be an <br />                                                      Homocysteine synthase       enzyme. Within this family, Met30p shows the strongest simi- <br />                                         Growth <br />  Strain        Relevant genotype                      sp act (nmol/min/mg        larity to the recently described protein Scon2 from Neurospora <br />                                       conditionsb <br />                                                            of protein)           crassa (26). Scon2p actually appears to be the functional equiv- <br /> CC572        MET30/MET30                  NR                  345                alent of Met30p in N. crassa, since this protein was identified as <br />                                           R                    56                a negative regulator of the sulfur network in this fungus (6). <br /> CD126        MET30/met30::URA3            NR                  350                However, nothing is known about how Scon2p functions (26). <br />                                           R                    40                   The present identification of Met30p as a negative regulator <br />   a <br />     Data are the averages of two independent experiments (deviation was less <br />                                                                                  of sulfur metabolism in S. cerevisiae reinforces the pattern <br /> than 10%).                                                                       suggesting that WD40 repeats may be found specifically in <br />   b <br />     NR, nonrepressive; R, repressive (1 mM L-methionine).                        transcriptional factors that are involved in repression mecha- <br />  VOL. 15, 1995                                                            YEAST TRANSCRIPTIONAL INHIBITOR Met30                                 6533 <br />  <br />  <br /> nisms. For instance, before Met30p, the two yeast transcrip-       is thus conceivable that through Met30p, AdoMet controls the <br /> tional factors known to include WD40 repeats were Hir1p, a         expression of a large number of genes, the global derepression <br /> negative regulator of histone gene expression, and Tup1p,          of which is incompatible with cell viability. <br /> which is involved in the repression of different sets of genes        In this regard, it is noteworthy that in yeast cells, the three <br /> (36, 45). Like Met30p, Hir1p and Tup1p do not appear to be         transcriptional factors that belong to the WD40 protein family, <br /> DNA-binding proteins. To explain how such molecules could          Hir1p, Met30p, and Tup1p, are all negative regulators without <br /> repress transcription, it has been proposed that they are re-      a DNA-binding domain and that mutations in them were all <br /> cruited to the target promoters by interacting with specific       found to produce pleiotropic effects on transcription (34, 36, <br /> DNA-bound factors. Such a model was later confirmed by             43; this study). This suggests that the ability to bind multiple <br /> several studies on the function of Tup1p (23, 25, 44). In accord   targets may be the raison d&#226;&#8364;&#8482;e&#203;&#8224;tre of the WD40 repeats within <br /> with such a model, Tup1p has the ability to repress transcrip-     transcriptional factors. <br /> tion when brought to a promoter as a fusion with a DNA- <br /> binding domain (44). In contrast to Tup1p, we show here that                                  <br /> Met30p did not repress transcription when it was tethered to a <br /> promoter through a fusion with the DNA-binding domain of             <br /> LexA.                                                               <br />    During this work, we have identified one target of Met30p as     <br /> Met4p, a transcriptional activator regulating the sulfate assim-    <br /> ilation pathway. Met4p was shown to respond to the level of        <br /> intracellular AdoMet: an increase in the intracellular AdoMet <br /> level inhibits the transcription activation function of Met4p                                        <br /> (27). A functional analysis of Met4p has demonstrated that this      <br /> activator contains, in addition to its bZIP, three domains: one         <br /> activation domain, located in the NH2-terminal part of the          <br /> protein; an inhibitory region, which mediates the AdoMet in-         <br /> hibition of the activation domain; and an auxiliary domain,             <br /> which counteracts the function of the inhibitory region when         <br /> the intracellular AdoMet level is low (27). Using LexA-Met4            <br /> fusion proteins, we show here that MET30 mutations prevent             <br /> the AdoMet-mediated inhibition of the Met4p activation func-         <br /> tion. Since a model explaining the intramolecular regulation of         <br />                                                                        <br /> Met4p predicted the interaction of the Met4p inhibitory region      <br /> with a regulatory protein (27), these results led us to look for       <br /> physical interactions between Met4p and Met30p. Using vari-          <br /> ous Gal4-Met4 derivatives, we demonstrate here by the two-             <br />                                                                         <br /> hybrid method that Met4p and Met30p interact in vivo. More-         <br /> over, we show that the inhibitory region of Met4p is involved         <br /> in this interaction. As the two-hybrid experiments were per-           <br /> formed in nonrepressive growth conditions, the results suggest      <br />                                                                        <br /> that Met4p and Met30p may form a complex even when the                 <br /> intracellular AdoMet level is low.                                     <br />    From these results, we propose that Met30p functions as a        <br /> transcriptional inhibitor which interacts with the Met4p inhib-        <br />                                                                         <br /> itory region to repress the activation domain when the intra-      <br /> cellular AdoMet level is high. Further studies are required to          <br /> define more accurately how Met30p functions and to under-              <br /> stand how the AdoMet signal is transduced to the Met4-Met30         <br /> couple. It would be also of importance to determine why, in            <br /> contrast to the met30 null mutation, the isolated alleles of       <br /> MET30 are dominant for the regulatory effect. In contrast to          <br /> that of Met4p, the function of Met30p is not restricted to the         <br /> sulfate assimilation pathway but also concerns the methyl cycle        <br /> and the AdoMet-mediated regulation of MET19, which en-             <br /> codes the first enzyme of the pentose phosphate pathway (data           <br /> not shown). This implies that Met30p has the ability to interact    <br /> with transcriptional factors other than Met4p. Such multiple            <br /> interactions would be possible because of the presence of               <br /> WD40 repeats within Met30p. Indeed, each WD40 repeat is            <br /> theoretically capable of interacting with at least one other            <br /> protein, as demonstrated by the recent study of the interactions   <br /> between Tup1p and homeodomain protein a2 (25). The lethal-             <br /> ity associated with MET30 gene disruption is consistent with      <br /> the involvement of Met30p in the regulation of numerous sets           <br /> of genes. Perhaps this lethality originates from the central       <br /> metabolic position filled by AdoMet, which is second only to            <br /> ATP in the number of reactions in which it participates (8). It        DNA. Cell 42:913&#226;&#8364;&#8220;921. <br /> </body></html>