<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2009 Genetics Society America <br /> DOI: 10.1534/genetics.109.103796 <br />  <br />  <br />  <br />            Evidence That Msh1p Plays Multiple Roles Mitochondrial <span id='am-316' about='xsp:base' typeof='owl:Thing'>Base</span> <br />                                   Excision Repair <br />  <br />                                      Leah Pogorzala, Shona Mookerjee Elaine A. Sia1 <br />                                       Department Biology, University Rochester, Rochester, New York 14627 <br />                                                         Manuscript received April 13, 2009 <br />                                                       Accepted publication April 23, 2009 <br />  <br />  <br />                                                             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Mitochondrial DNA thought especially prone oxidative damage reactive oxygen species <br />               generated electron transport cellular respiration. This damage mitigated primarily by <br />               base excision repair (BER) pathway, DNA repair pathways confirmed activity on <br />               mitochondrial DNA. Through genetic epistasis analysis yeast Saccharomyces cerevisiae, examined <br />               genetic interaction BER proteins previously shown localize mitochon- <br />               dria. In addition, series genetic interactions BER components MutS <br />               homolog MSH1, respiration-essential gene. We that, addition variable effects mito- <br />               chondrial function, mutant msh1 alleles conferring partial function interact genetically different points <br />               mitochondrial BER. In addition separation function, role Msh1p in <br />               BER unlikely involved avoidance large-scale deletions rearrangements. <br />  <br />  <br />  <br />  <br /> D      EPLETION mitochondrial function been <br />         implicated human aging process as <br /> in inherited aging-related disorders (Wallace <br />                                                                                 An important mechanism repair oxidative DNA <br />                                                                              damage base excision repair (BER) pathway <br />                                                                              (Croteau Bohr 1997; Nilsen Krokan 2001; <br /> 2005; Weissman et al. 2007). Much dysfunction                    Bohr 2002). This pathway studied nucleus <br /> be attributed mitochondrial genome instability,                    organisms, isoforms key compo- <br /> respiratory capacity mitochondria dependent                     nents shown localize mitochondrial <br /> an intact genome. Since respiration essential                     compartment (Rosenquist et al. 1997; You et al. 1999; <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>survival</span> eukaryotic obligate aerobes, facultative an-                 Vongsamphanh et al. 2001). However, despite their <br /> aerobe Saccharomyces cerevisiae ideal model                 extensive nuclear biochemical characterization, the <br /> mitochondrial studies. Despite difference size bet-                   role isoforms repair mtDNA poorly <br /> ween mitochondrial genomes yeast humans,                      understood. <br /> encoded components required process,                           BER initiated N-glycosylase recognizes a <br /> cellular energy production (Foury et al. 1998). Therefore,                   damaged base cleaves glycosidic bond between <br /> studying S. cerevisiae maintain mitochondrial DNA                        sugar-phosphate backbone, creating an <br /> (mtDNA) lend valuable insight mitochondrial                       apurinic/apyrimidinic (AP) site repaired <br /> genome maintenance higher eukaryotes (Perocchi                            BER pathways. In short patch BER, AP <br /> et al. 2008).                                                                site processed AP endonuclease 59 side <br />    The necessary process electron transport                        damaged base AP lyase activity of <br /> respiration cause damage proteins, lipids,                        glycosylase, polymerase b, 39 the <br /> nucleic acids formation reactive oxygen                       damage, create single-strand gap (Wilson et al. <br /> species (ROS) (Longo et al. 1996). Because mtDNA                             1998). This gap filled DNA polymerase then <br /> exists harsh environment, thought                   ligated complete repair. In alternative method <br /> especially prone oxidative damage (Bohr 2002).                            long-patch BER, DNA cleaved AP <br /> Damaged bases mutagenic misincorporation                           endonuclease generate available 39-end syn- <br /> opposite damage replicative polymerase                      thesis DNA polymerase nick, displacing the <br /> translesion synthesis damaged base. There-                        existing sequence containing abasic site creat- <br /> fore, repair oxidative lesions essential                   ing 59 flap. This flap cleaved flap endonuclease, <br /> stability mitochondrial genome.                                       resulting nick sealed DNA ligase, complet- <br />                                                                              ing repair. Biochemical studies suggest both <br />                                                                              short-patch long-patch pathways active in <br />   1 <br />    Corresponding author: University Rochester, RC Box 270211,             mitochondria (Akbari et al. 2008; Liu et al. 2008; <br /> Rochester, NY 14627. E-mail: esia@mail.rochester.edu                         Szczesny et al. 2008). <br />  <br /> Genetics 182: 699&#226;&#8364;&#8220;709 ( July 2009) <br />  700                                        L. Pogorzala, S. Mookerjee E. A. Sia <br />  <br />    In study, examine mitochondrial roles          erythromycin medium contains 1% yeast extract 2% <br /> Apn1p, Ntg1p, Ogg1p, well-studied BER                  glycerol supplemented 50 mm sodium phosphate buffer <br />                                                                  (pH 6.5) 4 g/liter erythromycin (MP Biomedicals). <br /> components. The N-glycosylase Ogg1p important for <br />                                                                  Synthetic dextrose media contain 0.17% yeast nitrogen base, <br /> the repair oxidatively damaged DNA, studies            0.5% ammonium sulfate, 2% dextrose, necessary amino <br /> ogg1-D strains increase point mutations         acids. Synthetic glycerol (SGly) media contain 0.17% yeast <br /> in nuclear mitochondrial DNA (Thomas et al.             nitrogen base, 0.5% ammonium sulfate, 2% glycerol, and <br /> 1997; Singh et al. 2001). In yeast, previously dem-       necessary amino acids. <br />                                                                     All S. cerevisiae strains used study isogenic the <br /> onstrated deletion N-glycosylase NTG1, or <br />                                                                  D273-10B-derived DFS188 strain (MATa ura3-52 leu2-3, 112 lys2 <br /> the AP endonuclease APN1, leads decrease mito-           his3 arg8ThisG; r1) (Sia et al. 2000) (Table 1). Primers located <br /> chondrial mutations measured rates erythromy-            200 bp upstream downstream gene interest <br /> cin resistance, suggesting actions Ntg1p         used amplify kanMX cassette S. cerevisiae <br /> Apn1p create mutagenic intermediates mtDNA dur-               deletion collection (Open Biosystems). The ogg1-D strain was <br />                                                                  created using primers 59-AGGCATTTGAAGCGTCCTGATT <br /> ing repair (Phadnis et al. 2006). This stands contrast <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                  CATAATTGCGATTTTTATTTATCAACCAGCAGATTCTAAC <br /> the increases seen nuclear DNA <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span> rates         TGAGAGTGCACC-39 59-CGCCTTTTCGGTCGCGTGCT <br /> presence deletion alleles, indicating    TTTATC GTGGTATTTACTATGACTTTTTAACCTGTGCGG <br /> always possible extrapolate mitochondrial function        TATTTTCACACCGC-39 amplify HIS3 gene the <br /> of BER proteins basis nuclear functions,         pRS413 plasmid (Stratagene, La Jolla, CA). Because msh1 mu- <br />                                                                  tant strains difficult manipulate maintaining res- <br /> thus making mitochondrial-specific studies necessary <br />                                                                  piratory competence, multiple methods strain construction <br /> (Ramotar et al. 1991, 1993; Alseth et al. 1999; Bennett          utilized. Strains containing multiple deletions BER <br /> 1999). In addition, likely mitochondrial-        genes deletion BER gene msh1 mutant back- <br /> specific players pathway. Here           ground constructed mating isogenic single <br /> mismatch repair homolog Msh1p plays multiple roles            mutants opposite mating type sporulating and <br />                                                                  dissecting resulting diploid strains. The strains msh1-F105A <br /> mitochondrial BER. <br />                                                                  ntg1-D, msh1-R813W apn1-D, msh1-R813W ntg1-D con- <br />    Msh1p yeast homologs MutS,          structed deleting APN1 NTG1 appropriate msh1 <br /> the bacterial mismatch repair protein,            mutant background. The resulting strains nonrespiring, <br /> found localized mitochondria (Reenan                  functional mtDNA reintroduced cytoduction. Briefly, <br /> Kolodner 1992; Chi Kolodner 1994). Msh1p                  cells mated NPY3 (DFS160 r1), contains a <br />                                                                  kar1-1 mutation preventing nuclear fusion, haploid re- <br /> essential mitochondrial function maintenance <br />                                                                  spiring cytoductants DFS188 nuclear background <br /> of mtDNA, necessitating use partial function              screened Lys  respiration-proficient pheno- <br /> mutants study role Msh1p mtDNA mainte-              type. To construct msh1-F105A ogg1-D strain, msh1- <br /> nance (Mookerjee et al. 2005). Although effects           F105A strain transformed Ura-selectable pRS416- <br /> its disruption examined multiple studies,           MSH1 plasmid (Mookerjee et al. 2005) maintain wild-type <br />                                                                  copy Msh1p. The resulting strain mated DFS188 ogg1- <br /> the mechanism Msh1p acts carry its <br />                                                                  D, dissected isolate double mutant, grown on <br /> essential functions remains unclear (Reenan                  5-FOA select loss plasmid. The resulting strain was <br /> Kolodner 1992; Koprowski et al. 2002; Mookerjee                  nonrespiring cytoduced wild-type mtDNA as <br /> et al. 2005; Mookerjee Sia 2006). Its role mito-        described above. <br /> chondrial mismatch repair protein disputed,                For construction msh1-G776D 1 pRK2 msh1- <br />                                                                  G776D 1 pRK2 APN1 strains, YSM84 transformed the <br /> particularly mismatch repair <br />                                                                  appropriate plasmid. The resulting strains nonrespiring, <br /> proteins localize mitochondria. However,             functional mtDNA reintroduced cytoduction as <br /> since mtDNA high potential requirement            described above. <br /> BER, possible pathway mitochondria           Mitochondrial <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> rates: Point mutation rates <br /> may utilize Msh1p. Previous studies shown genetic           (mutations/cell division) estimated performing <br /> interactions Msh1p BER proteins                  fluctuation analysis using spontaneous resistance drug <br />                                                                  erythromycin (EryR) reporter. Independent colonies were <br /> Ogg1p, Apn1p, Ntg1p (Dzierzbicki et al. 2004;                isolated YPG plates grown 30&#194;&#176; 3 days. Twenty <br /> Kaniak et al. 2009). Using msh1 alleles disrupted             independent colonies used inoculate 5 ml of <br /> conserved DNA binding ATPase domains,                YPG, grown 30&#194;&#176; agitation 2 days. For the <br /> examined frequency spectrum mutations                 Apn1p overexpression studies, independent colonies were <br /> responsible different mutation rates seen           isolated SGly&#226;&#8364;&#8220;Leu media, singles used in- <br />                                                                  oculate SGly&#226;&#8364;&#8220;Leu media grown 3 days. Appropriate <br /> each allele.                                                     dilutions plated YPG, allowing growth respiring <br />                                                                  cells culture YG plates supplemented 4 g/liter <br />                                                                  erythromycin select EryR colonies. Plates incubated at <br />               MATERIALS AND METHODS                              30&#194;&#176; 7 days. Rates erythromycin resistance cell <br />                                                                  division determined using method median <br />   Strains media: Rich dextrose medium (YPD) contains         (Lea Coulson 1949). Each rate determined at <br /> 1% yeast extract, 2% Bacto peptone, 2% dextrose. Rich        independent assays. <br /> glycerol medium (YPG) contains 1% yeast extract, 2% Bacto           Determining mutational spectra: The spectra muta- <br /> peptone, 2% glycerol. YPD geneticin medium YPD            tions causing erythromycin resistance determined in <br /> supplemented 200 mg/liter geneticin (Invitrogen). YG        mutant strains amplifying region 21S rRNA gene <br />                                                     Msh1p Plays Multiple Roles                                            701 <br />  <br />                                                             TABLE 1 <br />                                                     Strains used study <br />  <br />          Strain                              Relevant genotype                                    Reference <br />          DFS188             MATa ura3-52 leu2-3, 112 lys2 his3 arg8ThisG                  Sia et al. (2000) <br />          NRY127             DFS188 apn1-DTkanMX                                           Phadnis et al. (2006) <br />          RCY029             DFS188 ntg1-DTkanMX                                           Phadnis et al. (2006) <br />          YSM130             DFS188 ogg1-DTHIS3                                            This study <br />          LJY148             DFS188 apn1-DTkanMX ntg1-DTkanMX                              This study <br />          LJY400             DFS188 apn1-DTkanMX ntg1-DTkanMX ogg1-DTHIS3                  This study <br />          LJY165             DFS188 apn1-DTkanMX ogg1-DTHIS3                               This study <br />          LJY134             DFS188 ntg1-DTkanMX ogg1-DTHIS3                               This study <br />          YSM054             DFS188 msh1-F105A                                             Mookerjee Sia (2006) <br />          LJY173             DFS188 msh1-F105A apn1-DTkanMX                                This study <br />          LJY137             DFS188 msh1-F105A ntg1-DTkanMX                                This study <br />          LJY218             DFS188 msh1-F105A ogg1-DTHIS3                                 This study <br />          YSM084             DFS188 msh1-G776D                                             Mookerjee Sia (2006) <br />          LJY156             DFS188 msh1-G776D apn1-DTkanMX                                This study <br />          LJY208             DFS188 msh1-G776D ntg1-DTkanMX                                This study <br />          YSM148             DFS188 msh1-G776D ogg1-DTHIS3                                 This study <br />          LJY399             DFS188 msh1-G776D apn1-DTkanMX ogg1-DTHIS3                    This study <br />          NPY108             DFS188 1 pRK2                                                 Phadnis et al. (2006) <br />          NPY109             DFS188 1 pRK2 APN1                                            Phadnis et al. (2006) <br />          LJY382             DFS188 msh1-G776D 1 pRK2                                      This study <br />          LJY383             DFS188 msh1-G776D 1 pRK2 APN1                                 This study <br />          YSM105             DFS188 msh1-R813W                                             Mookerjee Sia (2006) <br />          LJY183             DFS188 msh1-R813W apn1-DTkanMX                                This study <br />          LJY180             DFS188 msh1-R813W ntg1-DTkanMX                                This study <br />          EAS789             DFS188 msh1-R813W ogg1-DTHIS3                                 This study <br />  <br />  <br />  <br /> previously shown rise EryR strains indepen-         2004; Doudican et al. 2005). To determine mtDNA <br /> dently derived EryR colonies (Sor Fukuhara 1982). This          affected deletions, assayed spontane- <br /> region, located 1900&#226;&#8364;&#8220;2000 mitochondrial <br />                                                                     ous resistance EryR. In agreement previously <br /> genome, amplified using primers 59-GGTAAATAG- <br /> CAGCCTTATTATG-39                      59-CGATCTATCTAA           published findings (Phadnis et al. 2006), that <br /> TTACAGTAAAGC-39. The amplified product sequenced                mitochondrial point mutation rates ntg1-D <br /> at Cornell University sequencing facility.                      strain (0.13 3 10 7 mutations/cell division) the <br />    Respiration loss assay: To determine proportion           apn1-D strain (0.41 3 10 7 mutations/cell division) were <br /> nonrespiring cells strain, cultures grown over- <br />                                                                     significantly lower wild-type rate 1.2 3 10 7 <br /> night YPG medium colonies isolated YPG <br /> plates. Ten 20 independent YPD cultures inoculated          mutations/cell division (P , 0.001 cases) (Table <br /> with single colony grown overnight, appropri-        2). These findings suggest context which <br /> ate dilutions plated YPG 1 0.1% dextrose medium. All        BER enzymes function mitochondria may <br /> tests performed duplicate, average median           substantially different nucleus, <br /> values used analysis. <br />                                                                     despite identical biochemistry. Furthermore, show <br />    Statistical analysis: Fluctuation analyses performed <br /> two times. Statistical analysis performed using         rates mitochondrial point mutations an <br /> Instat 3 Macintosh (GraphPad Software, San Diego). Error        apn1-D ntg1-D strain (0.75 3 10 7 mutations/cell di- <br /> is represented 6SEM. Two-tailed P-values calculated         vision) significantly different rates of <br /> using unpaired t-tests compare average rates frequen-        mutation strain containing deletion APN1 or <br /> cies. Mutational spectra compared chi-square analysis. <br />                                                                     NTG1 (P &#194;&#188; 0.25 0.18, respectively), is <br />                                                                     consistent action single genetic pathway <br />                                                                     prevention EryR mutations. <br />                          RESULTS <br />                                                                        In contrast decreased rates apn1-D ntg1- <br />   Differential functions BER mitochondrial                D, deletion OGG1 causes significant, <br /> nuclear DNA: In nucleus, various effects              5-fold increase mitochondrial point mutations <br /> reported deletion NTG1 APN1. Deletion                 (4.9 3 10 7 mutations/cell division P &#194;&#188; 0.025), is <br /> APN1 reported increase nuclear mutation                 consistent previous results (Singh et al. 2001). <br /> rates 2- 12-fold, reported effects NTG1 deletion          Additional deletion APN1 NTG1 ogg1-D strain <br /> range insignificant 40-fold wild-type levels           increases median point mutation rate 16- 8.2- <br /> (Ramotar et al. 1991, 1993; Bennett 1999; Hanna et al.              fold relative rate apn1-D ntg1-D strains <br />  702                                      L. Pogorzala, S. Mookerjee E. A. Sia <br />  <br />                         TABLE 2                                                          TABLE 3 <br />              Rates erythromycin resistance                                         Respiration loss <br />  <br />                                      Rate erythromycin      Relevant genotype                         % nonrespiring cells <br />                                       resistance (310 7) <br />                                        (standard error         Wild type                                          0.87 <br /> Relevant genotype                        mean) <br />                                                                msh1-F105A                                        88 <br /> Wild type                                  1.2 (0.01)          msh1-F105A apn1-D                                 90 <br />                                                                msh1-F105A ntg1-D                                 84 <br /> msh1-F105A                                 6.4   (0.12)        msh1-F105A ogg1-D                                 97 <br /> msh1-F105A apn1-D                          25    (0.70) <br /> msh1-F105A ntg1-D                          28    (4.29)        msh1-G776D                                       99 <br /> msh1-F105A ogg1-D                          5.2   (1.6)         msh1-G776D   apn1-D                              98 <br />                                                                msh1-G776D   ntg1-D                              99 <br /> msh1-G776D                                  32   (6.0)         msh1-G776D   ogg1-D                              99 <br /> msh1-G776D apn1-D                          1.2   (0.18)        msh1-G776D   apn1-D ogg1-D                      .99 <br /> msh1-G776D ntg1-D                          5.0   (1.92) <br /> msh1-G776D ogg1-D                          20    (3.98)        Wild type 1 pRK2                                   1.1 <br />                                                                Wild type 1 pRK2APN1                               0.78 <br /> Wild type 1 pRK2                           3.9   (0.58)        msh1-G776D 1 pRK2                                 99 <br /> Wild type 1 pRK2APN1                       11    (1.35)        msh1-G776D 1 pRK2APN1                             99 <br /> msh1-G776D 1 pRK2                           19   (4.6) <br /> msh1-G776D 1 pRK2APN1                      14    (3.36)        msh1-R813W                                        25 <br />                                                                msh1-R813W apn1-D                                 19 <br /> msh1-R813W                                 4.2   (0.83)        msh1-R813W ntg1-D                                 20 <br /> msh1-R813W apn1-D                          9.0   (1.0)         msh1-R813W ogg1-D                                 12 <br /> msh1-R813W ntg1-D                          7.6   (0.80) <br /> msh1-R813W ogg1-D                          41    (2.07)        apn1-D                                             0.26 <br />                                                                ntg1-D                                             0.53 <br /> apn1-D                                    0.13   (0.01)        ogg1-D                                             1.5 <br /> ntg1-D                                    0.32   (0.05)        apn1-D ntg1-D                                      0.75 <br /> ogg1-D                                     4.9   (0.6)         apn1-D ogg1-D                                      2.1 <br /> apn1-D ntg1-D                             0.75   (0.32)        ntg1-D ogg1-D                                      2.6 <br /> apn1-D ogg1-D                              2.1   (0.39)        apn1-D ntg1-D ogg1-D                               3.4 <br /> ntg1-D ogg1-D                              2.6   (0.65) <br /> apn1-D ntg1-D ogg1-D                       3.5   (1.4) <br />  <br />  <br />                                                                   DNA binding Msh1p occurs early mitochon- <br /> alone (P &#194;&#188; 0.032 0.009, respectively). With              drial BER: Mutation conserved phenylalanine <br /> mitochondrial point mutation rate 3.5 3 10 7,           residue MutS homolog mismatch recogni- <br /> apn1-D ntg1-D ogg1-D strain significantly different     tion domain disrupts mismatch recognition <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> apn1-D ogg1-D ntg1-D ogg1-D strain      DNA-binding capacities MutS homologs <br /> (P &#194;&#188; 0.33 0.62, respectively). In contrast        (Bowers et al. 1999; Dufner et al. 2000; Yamamoto <br /> effects mitochondrial point mutations,              et al. 2000; Drotschmann et al. 2001; Schofield et al. <br /> found deletion APN1, NTG1, OGG1 does            2001). In MSH1, mutation, msh1-F105A, <br /> significantly affect frequency respiration loss      confers point mutation rate 6.3 3 10 7 mutations/ <br /> compared wild-type strain (P &#194;&#188; 0.40, 0.60,          cell division, fivefold wild type (P , 0.001). <br /> 0.37, respectively) (Table 3).                                 This relatively mild defect contrasted respiration <br />    The difference mitochondrial nuclear            competence 10% wild-type strain <br /> phenotypes conferred disruption BER pathway          (Tables 2 3). The levels Msh1p-F105A are <br /> suggests role unique mitochondrial components.           comparable steady-state levels Msh1p, demonstrat- <br /> As demonstrated epistasis analysis, reveal specific      ing associated phenotypes specific to <br /> interactions MSH1 BER genes using              disruption DNA-binding domain (Mookerjee <br /> mitochondrial point mutation accumulation out-           et al. 2005). Additional deletion OGG1 does not <br /> put parameter, consistent role Msh1p BER         significantly alter mitochondrial point mutation <br /> (Table 2). Moreover, genetic relationships           <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> (P &#194;&#188; 0.08), consistent Msh1p-binding <br /> dependent defined mutations conserved                activity functioning upstream Ogg1p. This finding <br /> functional domains MSH1, suggesting specific           suggests Msh1p bind damaged bases or <br /> activities Msh1p contribute differently BER           mismatches interact preferentially <br /> described below.                                               bind particular downstream components BER. <br />                                                 Msh1p Plays Multiple Roles                                            703 <br />  <br />    In contrast interaction observed OGG1         Msh1p ATPase activity epistatic APN1 NTG1 <br /> deletion, msh1-F105A, conjunction APN1         deletion: In contrast msh1-G776D allele, an <br /> NTG1 deletion, synergistically increases point mutation       R813W substitution Msh1p presumably allows ATP <br /> accumulation levels observed msh1-F105A             binding binding-dependent conformational <br /> alone (25 3 10 7 29 3 10 7 mutations/cell division;       change, MutS homologs, disrupts ATP <br /> P , 0.001 P &#194;&#188; 0.026, respectively), suggesting       hydrolysis (Studamire et al. 1999). Therefore, compar- <br /> the repair steps following Msh1p binding         ing mutants allow differentiate <br /> partially independent pathways involving Apn1p         functions require ATPase activity <br /> Ntg1p. Unlike point mutation rates, deletion APN1,         require ATP binding. The msh1- <br /> NTG1, OGG1 does alter respiration loss seen        R813W allele confers 3.5-fold increase point <br /> in msh1-F105A strain, indicating mutator         mutation accumulation compared wild type, which <br /> phenotype conferred msh1-F105A allele specif-       conditions experiments quite <br /> ically defect BER, respiration loss         significant (4.2 3 10 7 mutations/cell division; P &#194;&#188; <br /> occurs gross instability likely separate    0.067) (Table 2),  25% loss respiration (Table <br /> feature MSH1 disruption (Table 3).                         3). Unlike msh1-G776D allele, rate EryR in <br />    ATP-bound Msh1p functions downstream BER                msh1-R813W strains significantly altered addi- <br /> events: The msh1-G776D substitution con-           tional deletion APN1 NTG1 (P &#194;&#188; 0.14 0.24, <br /> served ATPase domain analogous mutations           respectively), deletion OGG1 results 9.7-fold <br /> disrupt ability MutS homologs bind ATP              increase rate msh1-R813W (41 3 10 7 <br /> (Alani et al. 1997; Studamire et al. 1998). The steady-       mutations/cell division; P , 0.001). As msh1-F105A <br /> state levels Msh1p-G776D comparable levels       msh1-G776D, respiration loss conferred msh1- <br /> Msh1p, demonstrating associated phenotypes           R813W allele unaffected additional deletion of <br /> do result altered protein stability (Mookerjee       APN1, NTG1, OGG1 individually. <br /> et al. 2005). It deleterious            Disruption BER shifts mutational spectra of <br /> mutations studied here, conferring 27-fold increase         mtDNA: An important consideration studies is <br /> in point mutation rates relative wild-type rates (32 3     rate mutation accumulation, the <br /> 10 7 mutations/cell division; P &#194;&#188; 0.036),       mutational spectrum, indicate specific <br /> almost complete loss respiratory competence (Table         mutational mechanism. Because different biochemi- <br /> 2, Table 3). However, conjunction deletion         cal activity disrupted msh1-F105A, msh1- <br /> APN1 NTG1, complete rescue msh1-          G776D, msh1-R813W mutants, display <br /> G776D EryR phenotype wild-type levels (P &#194;&#188; 0.83            different mutator effects. To determine frequencies <br /> and 0.19, respectively). In contrast, loss Ogg1p,          mutational events responsible EryR of <br /> which, like Ntg1p, initiate BER glycosidic bond        msh1 alleles, region 1900 2000 the <br /> cleavage, does significantly alter rate conferred     mitochondrial 21S rRNA gene sequenced from <br /> by msh1-G776D (P &#194;&#188; 0.17). Since activities       independently isolated EryR colonies determine the <br /> Apn1p Ntg1p appear important msh1-          types substitutions rise EryR individual <br /> G776D phenotype, data consistent             colonies. Contained 100-bp region a <br /> model Msh1p bound ATP functions                   stretch nucleotides mutation commonly <br /> downstream endonucleolytic cleavage. One predic-           confers EryR (Figure 1A) (Vanderstraeten et al. 1998; <br /> tion model excess Apn1p           Kalifa Sia 2007). All sequenced EryR clones this <br /> create damage increase mitochondrial             study resulted mutations region. In wild- <br /> point mutation rate. Under experimental con-            type EryR strains, majority mutations A- <br /> ditions, extra copy APN1           to-G transitions A-to-T transversions frequencies <br /> copper-inducible promoter increases mutation rate         affected deletion APN1 (P &#194;&#188; 0.61 P <br /> relative wild-type rate (P &#194;&#188; 0.04), does       &#194;&#188; 1.0). However, deletion NTG1 causes significant <br /> affect mutation rate relative msh1-G776D         shift A-to-G transitions compared wild- <br /> (P &#194;&#188; 0.35).                                                   type strain (P &#194;&#188; 0.047) (Table 4). <br />    As msh1-F105A strains, respiration loss msh1-         Ogg1p recognizes excises multiple types of <br /> G776D strains affected deletion BER          damaged bases DNA. Its well-studied sub- <br /> genes overexpression Apn1p (Table 3). However,          strate 8-oxo-7,8-dihydroguanine (8-oxo-G), acts <br /> the msh1-G776D apn1-D ogg1-D strain          similar efficiencies 7,8-dihydro-8-oxoadenine, <br /> dramatic respiration loss msh1-G776D alone,       2,6-diamino-4-hydroxy-5-formamidopyrimidine, 4,6- <br /> is difficult maintain respiring cells, indicating     diamino-5-formamidopyrimidine (FapyA) (van der Kemp <br /> these proteins play partially redundant role          et al. 1996; Klungland et al. 1999; Krishnamurthy et al. <br /> maintaining respiratory competence. With level        2008). It shown 8-oxo-G mispairs with <br /> mitochondrial impairment, impossible accurately      adenine, subsequent rounds replication, causes <br /> measure mitochondrial point mutation rates.                   G-to-T transversion (Grollman Moriya 1993). We <br />  704                                          L. Pogorzala, S. Mookerjee E. A. Sia <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;(A) The sequence 1945 1957 21S rRNA gene encoded mitochondrial genome, com- <br /> mon site nucleotide substitutions conferring erythromycin resistance. (B&#226;&#8364;&#8220;E) Frequency base changes EryR mutant strains. <br /> The spectrum mutations strain determined sequencing independent erythromycin-resistant strains, as <br /> described materials methods. Graphs represent data 15 50 independent colonies strain. The frequency <br /> of type substitution estimated multiplying fraction type average frequency EryR strain. <br />  <br />  <br /> observed 6% frequency G T wild-type strains,          lesions shown mispair cytosine, resulting <br /> indicating that, detectable, highly                A-to-G mutations subsequent rounds replication, <br /> represented mutation. We expected deletion                   represent significant lesions mtDNA (Reyes <br /> OGG1 increase G-to-T events failure                 et al. 1998; Tudek 2003). <br /> 8-oxo-G repair. Surprisingly, that, ogg1-D               Erythromycin resistance msh1-F105A strains results <br /> strain, 30 EryR colonies sequenced contained            A-to-G substitutions 95% mutants <br /> mutation. However, 67% contained A-to-G                  sequenced (Table 4). This mutational spectrum also <br /> transition, double 34% A-to-G frequency EryR           significantly shifted A-to-G transitions the <br /> colonies wild-type cells (P &#194;&#188; 0.013). These data suggest          msh1-G776D strain (P &#194;&#188; 0.048), the <br /> that G-to-Tsubstitutions occur increased          msh1-R813W strain (P &#194;&#188; 0.77). Using data presented <br /> by loss Ogg1p. Instead, suggests that, mito-            Tables 2 4, estimate frequency each <br /> chondria, Ogg1p preferentially prevents mutations caused             type base substitution observed. As shown Figure 1, <br /> by damaged adenine thymine. On basis studies            B&#226;&#8364;&#8220;E, frequency mutational changes adenine <br /> mtDNA mammalian systems (Tanakaand Ozawa 1994;                    residues increases ogg1-D msh1 mutant strains. <br /> Reyes et al. 1998), predict adenine likely           These data indicate msh1 allele affects both <br /> to damaged base. Both FapyA hypoxanthine                  rate spectrum mutation differently. In <br />                                                 Msh1p Plays Multiple Roles                                                   705 <br />  <br />                                                         TABLE 4 <br />                                  Spectra mutations conferring erythromycin resistance <br />  <br />                                                                   % nucleotide substitutions <br />                                                                                                                       Total <br /> Relevant genotype              A C      A G       A T           G A       G T        G insertion    sequenced <br /> Wild type                         3           34             13             14           6               9             35 <br />  <br /> msh1-F105A                        5           95              0              0           0               0             19 <br /> msh1-F105A apn1-D                 0           67             18             12           0               3             33 <br /> msh1-F105A ntg1-D                11           79             26              0           0               0             19 <br /> msh1-F105A ogg1-D                12           76             12              0           0               0             17 <br />  <br /> msh1-G776D                       10           65             15              0           5               5             20 <br /> msh1-G776D apn1-D                21           42             11             21           0               5             19 <br /> msh1-G776D ntg1-D                 5           82              9              0           0               5             22 <br /> msh1-G776D ogg1-D                 0           84              5              0           0              11             19 <br />  <br /> Wild type 1 pRK2                 14           47             21              7           0               0             14 <br /> Wild type 1 pRK2APN1             23           46             23              8           0               0             13 <br /> msh1-G776D 1 pRK2                 3           69             22              3           0               3             32 <br /> msh1-G776D 1 pRK2APN1             3           79              9              3           0               6             34 <br />  <br /> msh1-R813W                       15           40             35              0           0              10             20 <br /> msh1-R813W apn1-D                18           49             23              0           3               8             39 <br /> msh1-R813W ntg1-D                 8           54             26              0           0              13             39 <br /> msh1-R813W ogg1-D                 6           83             11              0           0               0             18 <br />  <br /> apn1-D                           16           28             38             16           3               0             32 <br /> ntg1-D                            6           61             19              6           0               6             31 <br /> ogg1-D                            3           67             20              3           0               7             30 <br />  <br />  <br />  <br /> addition, that, mutation rates                 ities apn1-D ntg1-D, disparity to <br /> reduced wild-type levels msh1-G776D apn1-D                ogg1-D respect point mutation accumulation, <br /> and msh1-G776D ntg1-D strains, mutational spectra               suggest that, mitochondria, Apn1p Ntg1p are <br /> are indistinguishable apn1-D ntg1-D strains                connected function Ogg1p function least <br /> with respect number A-to-G substitutions (P &#194;&#188;             partially independent. The finding accumulation <br /> 0.37 0.14, respectively).                                       mitochondrial point mutations apn1-D ntg1-D <br />                                                                     ogg1-D strain similar apn1-D ogg1-D <br />                                                                     ntg1-D ogg1-D strains supports connec- <br />                      DISCUSSION <br />                                                                     tion Apn1p Ntg1p. We hypothesize that <br />    The importance maintaining mitochondrial                  Apn1p Ntg1p form functional complex mito- <br /> genome respiratory function overstated,            chondria. Such Apn1p&#226;&#8364;&#8220;Ntg1p complex func- <br /> is demonstrated variety severity inherited            tion nucleotide incision repair (NIR) role where <br /> mitochondrial diseases, correlation          glycosylase activity absent endonucleolytic cleav- <br /> mtDNA mutation accumulation, ROS production,                    age initiates repair, Ogg1p plays role this <br /> aging (Loeb et al. 2005; Kai et al. 2006; Vermulst et al.           type repair. The strong mutational shift A-to-G <br /> 2007). To understand accumulating mtDNA muta-                   transitions ogg1-D ntg1-D strains, in <br /> tions affect cellular functions time, need               apn1-D, however, suggest NIR activity not <br /> elucidate mechanisms contribute mtDNA                   primarily acting damaged adenine nucleotides and <br /> maintenance. Base excision repair earliest con-             Apn1p Ntg1p activities completely <br /> firmed pathway mtDNA repair, presence                 interdependent. Unfortunately, protein compo- <br /> particularly intriguing, given prevalent assumption             nents short-patch BER, long-patch BER, NIR <br /> that mtDNA highly susceptible oxidative damage,               overlap, making difficult separate contribution <br /> the main substrate BER activity.                                pathway genetic analysis. <br />    In nuclear model short-patch long-                  Msh1p appears constitute additional compo- <br /> patch BER, AP endonuclease cleavage required                   nent mitochondrial BER pathways. Each the <br /> intermediate step following base removal. The similar-              mutant alleles results unique set phenotypes <br />  706                                       L. Pogorzala, S. Mookerjee E. A. Sia <br />  <br />  <br />  <br />  <br />   Figure 2.&#226;&#8364;&#8221;Proposed model Msh1p mitochondrial BER. Distorted solid boxes (A B) represent damaged bases, while <br /> the box &#226;&#8364;&#732;&#226;&#8364;&#732;X&#226;&#8364;&#8482;&#226;&#8364;&#8482; (A) represents mismatch damaged base replication translesion polymerase. The <br /> double half-moon structures represent Msh1p dimers. <br />                                                 Msh1p Plays Multiple Roles                                                        707 <br />  <br /> consistent multiple roles Msh1p BER,      role maintaining larger-scale mtDNA stability. Respi- <br /> as additional roles genome stability. Previously,     ration loss common marker overall stability of <br /> we attributed mutator phenotypes resulting           mitochondrial genome. Cells lose respiration <br /> the msh1-G776D, msh1-R813W, msh1-F105A alleles         accumulation point mutations the <br /> defects Msh1p mechanism (Mookerjee                mitochondrial genome large-scale rear- <br /> et al. 2005). However, observe delete-       rangements deletions mtDNA. In fact, rear- <br /> rious effects alleles point mutation          rangements, resulting r  petites, primary <br /> differentially dependent activities BER en-         mechanisms generate nonrespiring cells labora- <br /> zymes. For example, increased point mutation rates        tory yeast cultures (Dujon 1981). The frequency which <br /> in msh1-G776D mutant strains require func-            events occur high result respiration <br /> tional Apn1p Ntg1p, presumably normal          loss percentages cells culture after <br /> role Msh1p Apn1p- Ntg1p-dependent base           overnight growth. Point mutation rates, occur on <br /> excision repair pathway. In contrast, point mutation          order 107 cells EryR, to <br /> accumulation msh1-R813W strains unaffected           increase phenomenal levels begin <br /> deletion APN1 NTG1, msh1-F105A allele           observed increases respiration loss this <br /> appears defective independently Apn1p-            background r  cells. Consequently, increases point <br /> Ntg1p-containing pathway.                                     mutations necessarily result correlative effect on <br />    These data support model Msh1p function           respiration loss. So, mitochondrial point mutati- <br /> mitochondrial BER partially over-       ons msh1-G776D mutant strain suppressed by <br /> lapping pathways exist (Figure 2). Because disruption      deletion APN1 NTG1, high rate respiration loss is <br /> the mismatch recognition/DNA-binding domain            similarly reduced. Therefore, essential function of <br /> msh1-F105A allele confers similar phenotypes             Msh1p stabilize mitochondrial genome a <br /> regards point mutation rates mutational spectra        broader mechanism point mutation prevention. <br /> as deletion OGG1 does, propose Msh1p              Most proteins functioning mitochon- <br /> may aid function Ogg1p DNA-             dria encoded nucleus, addi- <br /> binding activity (Figure 2A). It possible Msh1p       tional, well-studied roles cell. This study <br /> recognizes specific DNA aberration directs Ogg1p        exemplifies necessity examination mitochon- <br /> to damaged site stabilizes existing association.    drial-specific roles proteins function else- <br /> In Figure 2A, depicted Msh1p substrate         where. We shown Apn1p, Ntg1p, Ogg1p <br /> an oxidative lesion opposite mispair. Such binding       unexpected mitochondrial phenotypes that <br /> consistent known binding specificity          function closely tied mitochondrial specific <br /> human MutSa complex (Mazurek et al. 2002; Larson              protein, Msh1p. We demonstrated Msh1p <br /> et al. 2003), Msh1p exclude recogni-        plays multiple roles mitochondrial base excision repair <br /> tion lesions. Because deletion APN1           function separate broader role in <br /> NTG1 results similar phenotypes, propose           maintaining respiratory competence mitochondria. <br /> they act complex does require Msh1p                We thank Linnell Randall Shweta Krishnan help the <br /> binding initiate repair (Figure 2B) does require       fluctuation analysis. We thank Lidza Kalifa critical reading of <br /> that Msh1p bind ATP complete repair (Figure            manuscript. This work supported grants National <br /> 2D). In model, depicted Msh1p bound          Science Foundation (MCB0543084) National Institutes of <br />                                                               Health (GM63626). <br /> flap intermediate Msh2p&#226;&#8364;&#8220;Msh6p complexes have <br /> been demonstrated bind structures (Surtees <br /> and Alani 2006). Studamire et al. shown the <br />                                                                                     LITERATURE CITED <br /> nuclear Msh2p&#226;&#8364;&#8220;Msh3p complex important single- <br /> strand annealing, presumably binding flapped DNA           Akbari, M., T. Visnes, H. E. Krokan M. Otterlei, <br />                                                                   2008 Mitochondrial base excision repair uracil AP sites <br /> structures. This activity disrupted mutation           takes place single-nucleotide insertion long-patch DNA <br /> MSH2 analogous msh1-G776D mutation              synthesis. DNA Repair (Amst.) 7: 605&#226;&#8364;&#8220;616. <br /> analogous msh1-R813W. These data suggest          Alani, E., T. Sokolsky, B. Studamire, J. J. Miret R. S. Lahue, <br /> ability Msh2p&#226;&#8364;&#8220;Msh3p complex bind ATP,               1997 Genetic biochemical analysis Msh2p-Msh6p: role <br />                                                                   ATP hydrolysis Msh2p-Msh6p subunit interactions <br /> not necessarily hydrolyze it, necessary            mismatch base pair recognition. Mol. Cell. Biol. 17: 2436&#226;&#8364;&#8220; <br /> structure recognition (Studamire et al. 1998). Our data           2447. <br /> suggest similar separation function Msh1p,           Alseth, I., L. Eide, M. Pirovano, T. Rognes, E. Seeberg et al., <br />                                                                   1999 The Saccharomyces cerevisiae homologues endonuclease <br /> where mutator phenotype msh1-R813W                 III Escherichia coli Ntg1 Ntg2 required ef- <br /> slightly enhanced dramatically increased           ficient repair spontaneous induced oxidative DNA dam- <br /> msh1-G776D.                                                       age yeast. Mol. Cell. Biol. 19: 3779&#226;&#8364;&#8220;3787. <br />    The disparities point mutation rates       Bennett, R. A., 1999 The Saccharomyces cerevisiae ETH1 gene, in- <br />                                                                   ducible homolog exonuclease III provides resistance to <br /> respiration loss msh1 mutants suggest             DNA-damaging agents limits spontaneous mutagenesis. <br /> participation Msh1p BER likely unrelated          Mol. Cell. Biol. 19: 1800&#226;&#8364;&#8220;1809. <br />  708                                                   L. Pogorzala, S. Mookerjee E. A. Sia <br />  <br /> Bohr, V. A., 2002 Repair oxidative DNA damage nuclear                Loeb, L. A., D. C. Wallace G. M. Martin, 2005 The mitochon- <br />     mitochondrial DNA, changes aging mamma-                        drial theory aging relationship reactive oxygen spe- <br />     lian cells. Free Radic. Biol. Med. 32: 804&#226;&#8364;&#8220;812.                                 cies damage somatic mtDNA mutations. Proc. Natl. Acad. <br /> Bowers, J., T. Sokolsky, T. Quach E. Alani, 1999 A mutation                     Sci. USA 102: 18769&#226;&#8364;&#8220;18770. <br />     MSH6 subunit Saccharomyces cerevisiae MSH2&#226;&#8364;&#8220;MSH6              Longo, V. D., E. B. Gralla J. S. Valentine, 1996 Superoxide <br />     complex disrupts mismatch recognition. J. Biol. Chem. 274:                      dismutase activity essential stationary phase survival Sac- <br />     16115&#226;&#8364;&#8220;16125.                                                                    charomyces cerevisiae. Mitochondrial production toxic oxygen <br /> Chi, N.-W., R. D. Kolodner, 1994 Purification character-                    species vivo. J. Biol. Chem. 271: 12275&#226;&#8364;&#8220;12280. <br />     ization MSH1, yeast mitochondrial protein binds               Mazurek, A., M. Berardini R. Fishel, 2002 Activation hu- <br />     DNA mismatches. J. Biol. Chem. 269: 29984&#226;&#8364;&#8220;29992.                                man MutS homologs 8-oxo-guanine DNA damage. J. Biol. <br /> Croteau, D. L., V. A. Bohr, 1997 Repair oxidative damage                  Chem. 277: 8260&#226;&#8364;&#8220;8266. <br />     nuclear mitochondrial DNA mammalian cells. J. Biol.                 Mookerjee, S. A., E. A. Sia, 2006 Overlapping contributions of <br />     Chem. 272: 25409&#226;&#8364;&#8220;25412.                                                         Msh1p putative recombination proteins Cce1p, Din7p, and <br /> Doudican, N. A., B. Song, G. S. Shadel P. W. Doetsch,                           Mhr1p large-scale recombination genome sorting events <br />     2005 Oxidative DNA damage causes mitochondrial genomic                          mitochondrial genome Saccharomyces cerevisiae. Mutat. <br />     instability Saccharomyces cerevisiae. Mol. Cell. Biol. 25: 5196&#226;&#8364;&#8220;5204.        Res. 595: 91&#226;&#8364;&#8220;106. <br /> Drotschmann, K., W. Yang, F. E. Brownewell, E. T. Kool T. A.               Mookerjee, S. A., H. D. Lyon E. A. Sia, 2005 Analysis func- <br />     Kunkel, 2001 Asymmetric recognition DNA local distortion.                    tional domains mismatch repair homologue Msh1p its <br />     Structure-based functional studies eukaryotic Msh2-Msh6. J.                  role mitochondrial genome maintenance. Curr. Genet. 47: 84&#226;&#8364;&#8220;99. <br />     Biol. Chem. 276: 46225&#226;&#8364;&#8220;46229.                                              Nilsen, H., H. E. Krokan, 2001 Base excision repair net- <br /> Dufner, P., G. Marra, M. Raschle J. Jiricny, 2000 Mismatch                      work defence tolerance. Carcinogenesis 22: 987&#226;&#8364;&#8220;998. <br />     recognition DNA-dependent stimulation ATPase activ-             Perocchi, F., E. Mancera L. M. Steinmetz, 2008 Systematic <br />     ity hMutS-alpha abolished single mutation                     screens human disease genes, yeast human back. <br />     hMSH6 subunit. J. Biol. Chem. 275: 36550&#226;&#8364;&#8220;36555.                                 Mol. Biosyst. 4: 18&#226;&#8364;&#8220;29. <br /> Dujon, B., 1981 Mitochondrial genetics functions, pp. 505&#226;&#8364;&#8220;635              Phadnis, N., R. Mehta, N. Meednu E. A. Sia, 2006 Ntg1p, the <br />     The Molecular Biology Yeast Saccharomyces: Life Cycle             base excision repair protein, generates mutagenic intermediates <br />     Inheritance. Cold Spring Harbor Laboratory Press, Cold Spring                   yeast mitochondrial DNA. DNA Repair (Amst.) 5: 829&#226;&#8364;&#8220;839. <br />     Harbor, NY.                                                                Ramotar, D., S. C. Popoff, E. B. Gralla B. Demple, <br /> Dzierzbicki, P., P. Koprowski, M. U. Fikus, E. Malc Z. Ciesla,                  1991 Cellular role yeast Apn1 apurinic endonuclease/ <br />     2004 Repair oxidative damage mitochondrial DNA Sac-                    39diesterase: repair oxidative alkylation DNA damage and <br />     charomyces cerevisiae: involvement MSH1-dependent path-                  control spontaneous mutation. Mol. Cell. Biol. 11: 4537&#226;&#8364;&#8220;4544. <br />     way. DNA Repair (Amst) 3: 403&#226;&#8364;&#8220;411.                                         Ramotar, D., C. Kim, R. Lillis B. Demple, 1993 Intracellular <br /> Foury, F., T. Roganti, N. Lecrenier B. Purnell, 1998 The                        localization Apn1 DNA repair enzyme Saccharomyces cer- <br />     complete sequence mitochondrial genome Saccharomy-                    evisiae. Nuclear transport signals biological role. J. Biol. <br />     ces cerevisiae. FEBS Lett. 440: 325&#226;&#8364;&#8220;331.                                        Chem. 268: 20533&#226;&#8364;&#8220;20539. <br /> Grollman, A. P., M. Moriya, 1993 Mutagenesis 8-oxogua-                  Reenan, R. A. G., R. D. Kolodner, 1992 Characterization in- <br />     nine: enemy within. Trends Genet. 9: 246&#226;&#8364;&#8220;249.                                sertion mutations Saccharomyces cerevisiae MSH1 MSH2 <br /> Hanna, M., B. L. Chow, N. J. Morey, S. Jinks-Robertson, P. W.                       genes: evidence separate mitochondrial nuclear func- <br />     Doetsch et al., 2004 Involvement endonuclease III ho-                    tions. Genetics 132: 975&#226;&#8364;&#8220;985. <br />     mologs base excision repair pathway processing           Reyes, A., C. Gissi, G. Pesole C. Saccone, 1998 Asymmetrical <br />     DNA alkylation damage Saccharomyces cerevisiae. DNA Repair                   directional mutation pressure mitochondrial genome of <br />     (Amst.) 3: 51&#226;&#8364;&#8220;59.                                                               mammals. Mol. Biol. Evol. 15: 957&#226;&#8364;&#8220;966. <br /> Kai, Y., C. Takamatsu, K. Tokuda, M. Okamoto, K. Irita et al.,                 Rosenquist, T. A., D. O. Zharkov A. P. Grollman, <br />     2006 Rapid random turnover mitochondrial DNA                             1997 Cloning characterization mammalian 8-oxogua- <br />     rat hepatocytes primary culture. Mitochondrion 6:                         DNA glycosylase. Proc. Natl. Acad. Sci. USA 94: 7429&#226;&#8364;&#8220;7434. <br />     299&#226;&#8364;&#8220;304.                                                                   Schofield, M. J., F. E. Brownewell, S. Nayak, C. Du, E. T. Kool <br /> Kalifa, L., E. A. Sia, 2007 Analysis Rev1p Polz mito-                 et al., 2001 The Phe-X-Glu DNA binding motif MutS. The <br />     chondrial mutagenesis suggests alternative pathway dam-                   role hydrogen bonding mismatch recognition. J. Biol. <br />     age tolerance. DNA Repair (Amst.) 6: 1732&#226;&#8364;&#8220;1739.                                 Chem. 276: 45505&#226;&#8364;&#8220;45508. <br /> Kaniak, A., P. Dzierzbicki, A. T. Rogowska, E. Malc, M. Fikus et al.,          Sia, E. A., C. A. Butler, M. Dominska, P. Greenwell, T. D. Fox et al., <br />     2009 Msh1p counteracts oxidative lesion-induced instability                  2000 Analysis microsatellite mutations mitochondrial <br />     mtDNA stimulates mitochondrial recombination Saccharo-                   DNA Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA 97: <br />     myces cerevisiae. DNA Repair (Amst.) 8: 318&#226;&#8364;&#8220;329.                                250&#226;&#8364;&#8220;255. <br /> Klungland, A., I. Rosewell, S. Hollenbach, E. Larsen, G. Daly                  Singh, K. D., B. Sigala, H. A. Sikder C. Schwimmer, <br />     et al., 1999 Accumulation premutagenic DNA lesions mice                   2001 Inactivation Saccharomyces cerevisiae OGG1 DNA repair <br />     defective removal oxidative base damage. Proc. Natl. Acad.                gene leads increased frequency mitochondrial mutants. <br />     Sci. USA 96: 13300&#226;&#8364;&#8220;13305.                                                       Nucleic Acids Res. 29: 1381&#226;&#8364;&#8220;1388. <br /> Koprowski, P., M. U. Fikus, P. Mieczkowski Z. Ciesla,                      Sor, F., H. Fukuhara, 1982 Identification erythromycin <br />     2002 A dominant mitochondrial mutator phenotype Sac-                         resistance mutations mitochondrial gene coding the <br />     charomyces cerevisiae conferred msh1 alleles sequence                 large ribosomal RNA yeast. Nucleic Acids Res. 10: 6571&#226;&#8364;&#8220;6577. <br />     encoding ATP-binding domain. Mol. Genet. Genomics                      Studamire, B., T. Quach E. Alani, 1998 Saccharomyces cerevisiae <br />     266: 988&#226;&#8364;&#8220;994.                                                                   Msh2p Msh6p ATPase activities required during <br /> Krishnamurthy, N., K. Haraguchi, M. M. Greenberg S. S. David,                   mismatch repair. Mol. Cell. Biol. 18: 7590&#226;&#8364;&#8220;7601. <br />     2008 Efficient removal formamidopyrimidines 8-oxoguanine             Studamire, B., G. Price, N. Sugawara, J. E. Haber E. Alani, <br />     glycosylases. Biochemistry 47: 1043&#226;&#8364;&#8220;1050.                                       1999 Separation-of-function mutations Saccharomyces cerevi- <br /> Larson, E. D., K. Iams J. T. Drummond, 2003 Strand-specific                     siae MSH2 confer mismatch repair defects affect <br />     processing 8-oxoguanine human mismatch repair path-                   nonhomologous-tail removal recombination. Mol. Cell. <br />     way: inefficient removal 8-oxoguanine paired adenine                 Biol. 19: 7558&#226;&#8364;&#8220;7567. <br />     cytosine. DNA Repair (Amst.) 2: 1199&#226;&#8364;&#8220;1210.                                 Surtees, J. A., E. Alani, 2006 Mismatch repair factor MSH2&#226;&#8364;&#8220; <br /> Lea, D. E., C. A. Coulson, 1949 The distribution num-                    MSH3 binds alters conformation branched DNA struc- <br />     bers mutants bacterial populations. J. Genet. 49: 264&#226;&#8364;&#8220;285.                tures predicted form genetic recombination. J. Mol. <br /> Liu, P., L. Qian, J. S. Sung, N. C. Souza-Pinto, L. Zheng et al.,                Biol. 360: 523&#226;&#8364;&#8220;536. <br />     2008 Removal oxidative DNA damage FEN1-dependent                    Szczesny, B., A. W. Tann, M. J. Longley, W. C. Copeland S. Mitra, <br />     long-patch base excision repair human cell mitochondria.                     2008 Long patch base excision repair mammalian mitochon- <br />     Mol. Cell. Biol. 28: 4975&#226;&#8364;&#8220;4987.                                                 drial genomes. J. Biol. Chem. 283: 26349&#226;&#8364;&#8220;26356. <br />                                                            Msh1p Plays Multiple Roles                                                        709 <br />  <br /> Tanaka, M., T. Ozawa, 1994 Strand asymmetry human mi-                  tochondria maintain genomic stability. Mol. Cell. Biol. 21: <br />     tochondrial DNA mutations. Genomics 22: 327&#226;&#8364;&#8220;335.                          1647&#226;&#8364;&#8220;1655. <br /> Thomas, D., A. D. Scot, R. Barbey, M. Padula S. Boiteux,              Wallace, D. C., 2005 A mitochondrial paradigm metabolic and <br />     1997 Inactivation OGG1 increases incidence G   C                degenerative diseases, aging, cancer: dawn evolutionary <br />     / T   A transversions Saccharomyces cerevisiae: evidence en-       medicine. Annu. Rev. Genet. 39: 359&#226;&#8364;&#8220;407. <br />     dogenous oxidative damage DNA eukaryotic cells. Mol.            Weissman, L., N. C. Souza-Pinto, T. Stevnsner V. A. Bohr, <br />     Gen. Genet. 254: 171&#226;&#8364;&#8220;178.                                                 2007 DNA repair, mitochondria, neurodegeneration. Neu- <br /> Tudek, B., 2003 Imidazole ring-opened DNA purines bi-               roscience 145: 1318&#226;&#8364;&#8220;1329. <br />     ological significance. J. Biochem. Mol. Biol. 36: 12&#226;&#8364;&#8220;19.              Wilson, D. M., B. P. Engelward L. Samson, 1998 Prokaryotic <br /> van der Kemp, P. A., D. Thomas, R. Barbey, R. Oliveira S.              base excision repair, pp. 29&#226;&#8364;&#8220;64 DNA Damage Repair. Vol. I: <br />     Boiteux, 1996 Cloning expression Escherichia coli           DNA Repair Prokaryotes Lower Eukaryotes, edited J. A. <br />     OGG1 gene Saccharomyces cerevisiae, codes DNA gly-         Nickoloff M. F. Hoekstra. Humana Press, Totowa, NJ. <br />     cosylase excises 7,8-dihydro-8-oxoguanine 2,6-diamino-       Yamamoto, A., M. J. Schofield, I. Biswas P. Hsieh, <br />     4-hydroxy-5-N-methylformamidopyrimidine. Proc. Natl. Acad.                2000 Requirement Phe36 DNA binding mismatch <br />     Sci. USA 93: 5197&#226;&#8364;&#8220;5202.                                                   repair Escherichia coli MutS protein. Nucleic Acids Res. 28: <br /> Vanderstraeten, S., S. Van den Brule, J. Hu F. Foury,                     3564&#226;&#8364;&#8220;3569. <br />     1998 The role 39-59 exonucleolytic proofreading mis-           You, H. J., R. L. Swanson, C. Harrington, A. H. Corbett, S. Jinks- <br />     match repair yeast mitochondrial DNA error avoidance. J. Biol.         Robertson et al., 1999 <br /> </body></html>