<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                                         JCB: ARTICLE <br />  <br />  <br />  <br />  <br />                               Sphingomyelin synthase-related protein SMSr <br />                               controls ceramide homeostasis ER <br />                               Ana M. Vacaru,1 Fikadu G. Tafesse,1 Philipp Ternes,1 Vangelis Kondylis,3 Martin Hermansson,4 Jos F.H.M. Brouwers,2 <br />                               Pentti Somerharju,4 Catherine Rabouille,3 Joost C.M. Holthuis1 <br />                               1 <br />                                 Membrane Enzymology, Bijvoet Center Institute Biomembranes, 2Biochemistry Cell Biology, Faculty Veterinary Medicine Institute Biomembranes, <br />                                Utrecht University, 3584 CH Utrecht, Netherlands <br />                               3 <br />                                 The Cell Microscopy Center, Department Cell Biology Institute Biomembranes, University Medical Center Utrecht, 3584 CX Utrecht, Netherlands <br />                               4 <br />                                 Medical Biochemistry, Institute Biomedicine, University Helsinki, Helsinki 00014, Finland <br />  <br />  <br />  <br />  <br />                               C <br />                                       eramides central intermediates sphingo-                                   produces trace amounts CPE, i.e., 300-fold less <br />                                       lipid metabolism critical functions cell                                 SMS1-derived SM. Nevertheless, blocking cat- <br />                                       organization <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span>. They synthesized                                  alytic activity causes substantial rise ER ceramide <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               cytosolic surface endoplasmic reticulum                                    levels structural collapse early secretory <br />                               (ER) transported ceramide transfer protein                                     pathway. We <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> caused <br />                               Golgi conversion sphingomyelin (SM) SM                                     depletion CPE consequence cer- <br />                               synthase SMS1. In study, report identification                               amide accumulation ER. Our results establish <br />                               SMS1-related (SMSr) enzyme, catalyses                                        SMSr key regulator ceramide homeostasis that <br />                               synthesis SM analogue ceramide phospho-                                       operate sensor converter of <br />                               ethanolamine (CPE) ER lumen. Strikingly, SMSr                                     ceramides ER. <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               Sphingolipids vital components cellular membranes                              rheostat, appears critical normal cell growth lifespan <br />                               organisms ranging mammals yeast. Besides providing                             (Mandala et al., 1998; Kobayashi Nagiec, 2003). Moreover, <br />                               structural framework plasma membrane (PM) organization,                              sphingolipids form gradients secretory pathway that <br />                               sphingolipids dynamic regulators wide range cellular                         affect protein sorting hydrophobic matching of <br />                               processes. Sphingoid long-chain bases (LCBs), ceramides,                             membrane spans (Bretscher Munro, 1993; Holthuis et al., <br />                               intermediates sphingolipid metabolism act signaling                          2001; Patterson et al., 2008). <br />                               molecules regulation cell growth, death, migration,                               Given impact cell organization survival, the <br />                               membrane trafficking (Spiegel Milstien, 2003; Hannun                             local concentration sphingolipids metabolic inter- <br />                               Obeid, 2008). Notably, sensitive balance phosphory-                            mediates tightly controlled. Although mechanisms <br />                               lated LCBs ceramides, referred LCBP/ceramide                               regulate membrane sterol concentrations established <br />                                                                                                                        (Goldstein et al., 2006), little known mechanisms <br />                                                                                                                        controlling sphingolipid homeostasis. As ceramides constitute <br />                               A.M. Vacaru, F.G. Tafesse, P. Ternes, V. Kondylis contributed equally <br />                               paper.                                                                           backbone sphingolipids directly participate cel- <br />                               Correspondence Philipp Ternes: pternes@uni-goettingen.de; Joost C.M.               lular life death decisions (Morales et al., 2007), controlling <br />                               Holthuis: j.c.holthuis@uu.nl                                                             local concentration critical. Ceramides generated <br />                               P. Ternes&#226;&#8364;&#8482; present address Albrecht-von-Haller Institut f&#195;&#188;r Plantzenwissenschaften, <br />                               37077 G&#195;&#182;ttingen, Germany <br />                                                                                                                        breakdown sphingomyelin (SM) SMases <br />                               Abbreviations used paper: CERT, ceramide transfer protein; CPE, cer&#194;&#173; <br />                                                                                                                        PM (Andrieu-Abadie Levade, 2002) synthesized <br />                               amide phosphoethanolamine; CPES, CPE synthase; ds GFP, dsRNA-targeting GFP;              novo N-acylation LCBs cytosolic surface the <br />                               ds SMSr, dsRNA-targeting SMSr; dSMSr, Drosophila SMSr; dsRNA, double- <br />                               stranded RNA; gav, average g; GlcCer, glucosylceramide; HB, homogenization <br />                                                                                                                        ER (Mandon et al., 1992; Hirschberg et al., 1993). The latter <br />                               buffer; hSMSr, human SMSr; LA, lamin A; LCB, long-chain <span id='am-398' about='xsp:base' typeof='owl:Thing'>base</span>; MS, mass <br />                               spectrometry; PC, phosphatidylcholine; PE, phosphatidylethanolamine; PEMT, <br />                                                                                                                        &#194;&#169; 2009 Vacaru et al.&#226;&#8364;&#402; This article distributed terms Attribution&#226;&#8364;&#8220; <br />                               PE N-methyltransferase; PM, plasma membrane; PNS, postnuclear supernatant;               Noncommercial&#226;&#8364;&#8220;Share Alike&#226;&#8364;&#8220;No Mirror Sites license months publica- <br />                               SAM, sterile &#238;?&#161; motif; si hSMSr, siRNA-targeting hSMSr; si LA, siRNA-targeting            tion date (see http://www.jcb.org/misc/terms.shtml). After months available a <br />                               LA; SM, sphingomyelin; SMS, SM synthase; SMSr, SMS related; tER, transi-                 Creative Commons License (Attribution&#226;&#8364;&#8220;Noncommercial&#226;&#8364;&#8220;Share Alike 3.0 Unported license, <br />                               tional ER.                                                                               described http://creativecommons.org/licenses/by-nc-sa/3.0/). <br />  <br />  <br />  <br />  <br />                               The Rockefeller University Press&#226;&#8364;&#402;&#226;&#8364;&#352;$30.00 <br />                               J. Cell Biol. Vol. 185 No. 6&#226;&#8364;&#402; 1013&#226;&#8364;&#8220;1027 <br />                               www.jcb.org/cgi/doi/10.1083/jcb.200903152                                                                                                                                       JCB      1013 <br />             reaction controlled components TORC2 (target rapa-        mixtures showed presence NBD-labeled product a <br />            mycin complex 2) presumably phosphorylation            retention factor value distinct NBD&#226;&#8364;&#8220;ceramide phos- <br />            ceramide synthase (Aronova et al., 2008).                             phoinositol NBD-SM. The product missing reactions <br />                    In mammalian cells, bulk newly synthesized ce-         performed control (empty vector) cells (Fig. 1 C). Liquid chro- <br />            ramides converted SM lumen Golgi (Tafesse         matography (LC) mass spectrometry (MS)/MS analysis identified <br />            et al., 2006). Efficient delivery ER-derived ceramides      product NBD-CPE (unpublished data), suggesting that <br />            site SM production requires cytosolic ceramide transfer          SMSr proteins synthesize CPE. <br />            protein (CERT; Hanada et al., 2003). Besides START domain                  <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> determine SMSr proteins recognize natural <br />            binds ceramide, CERT contains FFAT motif inter-           ceramides substrates CPE production, yeast strain was <br />            action ER membrane pleckstrin homology domain          used endogenous enzymes ceramide production <br />            targeting Golgi. CERT phosphorylated serine repeat        replaced mouse ceramide synthase CerS5 (Cerantola <br />            motif, down-regulates ceramide transport function.          et al., 2007). As ceramides produced strain structur- <br />            Loss SM cholesterol cells causes dephosphoryla-           ally resemble animal cells, expected to <br />            tion motif, resulting CERT activation (Kumagai et al.,      suitable substrates CPE biosynthesis. LC/MS/MS analy- <br />            2007). Thus, local concentration ceramides cells         ses revealed presence molecular species CPE <br />            controlled intricate network lipid-metabolizing enzymes,     lipid extracts hSMSr- dSMSr-expressing strains <br />            transfer proteins, kinases, phosphatases.                         (Fig. 1 D). The extracts devoid SM. No CPE detect- <br />                    SM production mediated phosphatidylcholine (PC)/      able control cells. Together, results demonstrate that <br />            ceramide cholinephosphotransferase SM synthase (SMS).              SMSr proteins function CPESs. <br />            Mammalian cells contain SMS isoforms, SMS1 Golgi <br />            SMS2 PM (Huitema et al., 2004). In addition, mam-      SMSr represents major CPES activity in <br />            malian genome encodes SMS-related (SMSr) protein           mammalian cells <br />            unknown function homologues nematodes insects like        To investigate SMSr corresponds CPES activity <br />            Drosophila melanogaster (Fig. 1 A). Drosophila does synthe-       previously described mammalian cells (Malgat et al., 1986, <br />            size SM produces SM analogue ceramide phosphoethanol-         1987), lysates HeLa cells hSMSr overexpressed <br />            amine (CPE) major membrane constituent (Rao et al., 2007).       depleted RNAi analyzed CPES activity. Incubation <br />            CPE production occurs mammals catalyzed           control cell lysates NBD-Cer yielded fluorescent <br />            phosphatidylethanolamine (PE)/ceramide ethanolamine phos-             products, corresponding NBD-glucosylceramide (GlcCer), <br />            photransferase CPE synthase (CPES; Malgat et al., 1986, 1987).     NBD-SM, NBD-CPE (Fig. 2 A, left lane). Although forma- <br />            The identity responsible enzyme known. Production       tion NBD-CPE stimulated addition PE, addition of <br />            SM CPE involves similar reaction chemistry (Malgat           PC CDP-ethanolamine effect, suggesting PE <br />            et al., 1986). Because SMS1, SMS2, SMSr structurally re-      headgroup donor CPES reaction (Fig. 2 B; Fig. S1; <br />            lated share highly conserved sequence motifs puta-       Fig. 4 E). Heterologous expression V5-tagged hSMSr <br />            tive active site residues (Fig. 1 B; Huitema et al., 2004; Tafesse    led sixfold increase CPES activity, leaving SMS activity <br />            et al., 2006), SMSr prime candidate elusive CPES.        unaffected (Fig. 2, A C). Conversely, CPES activity dropped <br />                    In study, SMSr displays CPES activity       &gt;50% depletion hSMSr RNAi (Fig. 2 D). This shows <br />            and, contrary SMS1 2, localizes ER. However,         hSMSr represents major CPES activity HeLa cells. <br />            SMSr produces trace amounts CPE                   To determine contribution SMSr CPE biosynthesis <br />            bulk production CPE insect cells mediated different     vivo, endogenous CPE production levels monitored by <br />            enzyme. Unexpectedly, blocking SMSr activity causes marked          metabolic labeling various cell lines. Labeling human colon car- <br />            increase ceramide levels ER. This accompanied       cinoma Caco-2, liver carcinoma Hep-G2, HeLa cells 48 h <br />            fragmentation ER exit sites structural collapse       [14C]ethanolamine yielded trace amounts radiolabeled <br />            Golgi. These morphological aberrations caused            CPE (Fig. 2 F). The true HeLa CHO-K1 cells <br />            lack CPE consequence ceramide accumu-          labeled [3H]sphingosine (Fig. 2 G). In contrast, labeling <br />            lation ER. We propose SMSr CPES dual            Drosophila S2 cells [14C]ethanolamine showed CPE <br />            activity ceramide sensor control ceramide homeostasis        production level cells higher (Fig. 2 F, right). <br />            ER process critical integrity   Indeed, MS analyses revealed S2 cells contain sub- <br />            early secretory pathway.                                          stantial CPE (15.3 &#194;&#177; 1.1 mol% total phospholipid; <br />                                                                                  n = 3), CPE levels HeLa CHO-K1 cells exceedingly <br />            Results                                                               low (&#238;?&#186;0.03 mol%, i.e., 300-fold lower SM; Fig. 5 A). <br />  <br />            SMSr proteins display CPES activity                                   SMSr produces trace amounts CPE <br />            To test SMSr proteins catalyze CPE production, human          Although mammalian cells contain readily detectable CPES <br />            SMSr (hSMSr) Drosophila SMSr (dSMSr) expressed               activity (Fig. 2 A), CPE content low. One explana- <br />            budding yeast, organism lacking endogenous CPES activity.       tion discrepancy SMSr resides com- <br />            SMSr-expressing cells lysed incubated fluores-          partment easily reached newly synthesized <br />            cent C6-NBD&#226;&#8364;&#8220;ceramide (NBD-Cer). TLC analysis reaction          ceramide. To test this, subcellular distribution hSMSr-V5 <br />  <br />  <br />     1014   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  Figure 1.&#226;&#8364;&#402; SMSr proteins display CPES activity. (A) Phylogenetic tree SMS family members Homo sapiens (h), Xenopus tropicalis (x), Fugu rubripes <br /> (f), Drosophila (d), Apis mellifera (a), C. elegans (c). The tree constructed using ClustalX. (B) SMSr proteins share common domain structure with <br /> vertebrate SMS1, includes transmembrane helices, active site consisting conserved His (H) Asp (D) residues, N-terminal SAM <br /> domain. (C) TLC separation reaction products formed NBD-Cer incubated lysates yeast strains expressing hSMSr dSMSr trans- <br /> fected vector (control). White line indicates intervening lanes spliced out. (D) Yeast strains expressing hSMSr dSMSr produce <br /> CPE vivo. Different molecular species CPE detected electrospray LC/MS/MS (neutral loss 141) alkaline hydrolysis glycerolipids. <br /> No CPE detected lipid extracts control (untransfected) yeast. <br />  <br />  <br />                                                                                 3 <br /> was analyzed HeLa cells immunofluorescence microscopy.                     H-CPE, indicating low CPE content mammalian cells <br /> hSMSr-V5 gave reticular nuclear envelope staining <span id='am-399' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                 unlikely caused scavenging ceramides site of <br /> and colocalized ER marker protein disulfide isomerase                  CPE production. <br /> (Fig. 2 E Fig. S2 A). The ER localization hSMSr does                       Another explanation low CPE content the <br /> explain mammalian cells contain trace amounts                       presence inhibitory factor environment ER. <br /> CPE, substrates CPE synthesis, PE, ceramide                  We removal N-terminal sterile &#238;?&#161; motif (SAM) <br /> made ER.                                                                 domain hSMSr caused redistribution ER the <br />        We tested substrates limited access                Golgi (Fig. 2 E Fig. S2 B) did affect enzymatic <br /> the enzyme&#226;&#8364;&#8482;s active site. SMSr SMS1 share common                          activity (see Fig. 4 A). However, metabolic labeling HeLa <br /> transmembrane domain organization active site facing                   cells expressing hSMSr&#238;?&#8222;SAM showed significant increase <br /> the exoplasmic leaflet (Fig. 1 B; Huitema et al., 2004). SMSr-                  CPE production levels compared cells expressing the <br /> catalyzed CPE production occur ER                        full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> protein (Fig. 2 F). Thus, ER does to <br /> lumen. As ceramide synthesized cytosolic surface               impose inhibitory environment SMSr enzymatic activity. <br /> ER (Mandon et al., 1992; Hirschberg et al., 1993),                        Last, investigated CPE readily converted to <br /> scavenged CERT able flip site CPE                  product. Although SM production mammalian cells <br /> production. To test this, used CHO-K1&#226;&#8364;&#8220;derived LY-A mutant                    mainly occurs SMS1-mediated headgroup transfer <br /> cells, defective CERT-mediated ceramide transport                  PC ceramide (Tafesse et al., 2006), alternative path- <br /> (Hanada et al., 2003). As shown Fig. 2 G, [3H]sphingosine                    way involves methylation CPE (Muehlenberg et al., 1972). <br /> labeling CHO-K1 LY-A cells yielded similar levels                     To investigate relative contribution pathway <br />  <br />  <br />                                                                 SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                                   1015 <br />             Figure 2.&#226;&#8364;&#402; Mammalian cells synthesize trace amounts CPE despite ER residency SMSr. (A) TLC analysis reaction products formed when <br />            lysates control (untransfected [untransf]) hSMSr-V5&#226;&#8364;&#8220;expressing HeLa cells incubated NBD-Cer. (B) TLC analysis reaction products formed <br />            NBD-Cer incubated lysates HeLa cells expressing hSMSr-V5 presence absence externally added PE PC. (C) Expression <br />            hSMSr-V5 stimulates CPES activity HeLa cells. CPES SMS activity levels determined TLC analysis reaction products formed cell <br />            lysates incubated NBD-Cer expressed relative control cells. (D) CPES SMS activity levels determined lysates HeLa cells <br />            treated 7 d siRNA-targeting LA (si LA) siRNA-targeting hSMSr (si hSMSr) nonsilencing siRNA (si NS) C expressed relative to <br />            si LA&#226;&#8364;&#8220;treated cells. (E) Confocal sections HeLa cells transfected hSMSr-V5 SAM-deficient truncation mutant, hSMSr&#238;?&#8222;SAM-V5, and <br />            immunolabeled V5. Note removal SAM causes hSMSr-V5 redistribute ER Golgi. Bars, 5 &#194;&#181;m. (F) Human Caco-2, Hep-G2, HeLa, <br />            hSMSr&#238;?&#8222;SAM-V5-expressing HeLa, Drosophila S2 cells labeled [14C]ethanolamine 48 h subjected lipid extraction, TLC analysis, <br />            autoradiography. In extracts, glycerolipids deacylated mild alkaline hydrolysis (hydr; +). (G) Chinese hamster CHO-K1, CHO-K1&#226;&#8364;&#8220; <br />            derived LY-A (CERT mutant), LY-A/human CERT cells (mutant expressing human CERT) labeled [3H]sphingosine 24 h subjected lipid <br />            extraction, TLC analysis, autoradiography. Error bars indicate SD; n = 3. <br />  <br />  <br />  <br />            SM production, HeLa cells metabolically labeled                        rate CPE HeLa cells similar SM <br />            D4-ethanolamine D9-choline simultaneously. MS/MS                      (unpublished data). Thus, low CPE content mammalian <br />            analysis showed D4/D9 ratio SM 24-h labeling                 cells unlikely result CPE short-lived meta- <br />            &#238;?&#186;1:100, indicating headgroup transfer PC                  bolic intermediate. <br />            major pathway SM biosynthesis. Moreover, D4-labeled                            Together, results suggest SMSr proteins in- <br />            SM pool likely originated headgroup transfer                      trinsically unable produce bulk amounts CPE. This con- <br />            D4-labeled PC, pool reduced depletion human                  sistent finding CPE levels SMSr-expressing yeast <br />            SMS1 (unpublished data). Finally, pulse-chase experiments                      strains invariably low (&#226;&#8240;&#164;0.5 mol% total phosphosphingo- <br />            [14C]ethanolamine [14C]choline showed turn-                  lipid). As bulk amounts CPE present Drosophila, <br />  <br />  <br />     1016   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  Figure 3.&#226;&#8364;&#402; dSMSr resides ER insect cells. (A) Confocal sections Drosophila S2 cells double transfected native dSMSr ER marker <br /> PEMT-GFP immunolabeled dSMSr. (B) Confocal sections S2 cells transfected untagged dSMSr double labeled dSMSr and <br /> cis-Golgi marker dGMAP. (A B) Arrows indicate nuclear envelope staining. Bars, 5 &#194;&#181;m. (C) Localization dSMSr immunoelectron microscopy is <br /> shown. Ultrathin cryosections S2 cells transfected dSMSr double labeled dSMSr (15-nm gold) dSec23 (tER site marker; 10-nm gold). <br /> G, Golgi stack; N, Nucleus; E, endosome. Bar, 500 nm. (D) Quantification immunogold labeling S2 cells cotransfected dSMSr PEMT-GFP. <br /> Low-expressing cells &lt;40 gold particles cell section; high-expressing cells &#226;&#8240;&#165;40 gold particles cell section. <br />  <br />  <br /> we addressed contribution dSMSr CPE production                 GFP-tagged version ER marker PE N-methyltrans- <br /> in Drosophila S2 cells.                                                       ferase (PEMT; PEMT-GFP) Golgi microtubule- <br />                                                                               associated protein dGMAP (Fig. 3, A B). The ER residency <br /> Bulk production CPE insect cells                                     dSMSr confirmed quantitative immunoelectron micros- <br /> independent SMSr                                                           copy (Fig. 3, C D). As observed hSMSr, removal its <br /> We analyzed subcellular distribution dSMSr S2 cells           SAM domain caused dSMSr redistribute ER the <br /> by using antibody specifically cross reacted dSMSr               Golgi (Fig. S2, C&#226;&#8364;&#8220;E), consistent fundamental role <br /> on immunoblots S2 cell extracts (see Fig. 4 C). As endoge-             SAM ER retention SMSr proteins. <br /> nous protein hardly detectable immunofluorescence                            We investigated contribution dSMSr CPE <br /> microscopy, S2 cells transfected Cu+2-inducible dSMSr             production S2 cells. Incubation S2 cell lysates NBD- <br /> construct, expression protein enhanced fivefold (as            Cer led formation NBD-CPE (Fig. 4 A). NBD-CPE forma- <br /> indicated immunoblotting). dSMSr colocalized extensively                   tion increased fivefold overexpression dSMSr. <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>Mutation</span> <br />  <br />  <br />                                                                SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                                1017 <br />             Figure 4.&#226;&#8364;&#402; Bulk production CPE insect cells independent SMSr. (A) TLC analysis reaction products formed NBD-Cer incubated with <br />            lysates Drosophila S2 cells expressing dSMSr-V5, active site mutants dSMSr-V5 (H401A D405E), hSMSr-V5, active site mutant hSMSr-V5 <br />            (D348E), SAM-deficient hSMSr-V5 (hSMSr&#238;?&#8222;SAM-V5). (B) S2 cells treated ds GFP dSMSr (ds dSMSr #1 #2) 7 d metabolically labeled <br />            NBD-Cer (intact cells) lysed incubated NBD-Cer 2 h (cell lysates). CPES activity levels determined TLC analysis expressed <br />            relative mock-treated cells. Error bars indicate SD; n = 3. (C) Immunoblots S2 cells treated dsRNA B stained dSMSr dGol- <br />            gin245. Note anti-dSMSr antibody cross reacts weakly dSMSr-unrelated protein (asterisk). (D) S2 cells treated dsRNA B were <br />            metabolically labeled [14C]ethanolamine ([14C]eth) 2 h subjected TLC analysis autoradiography. (E) TLC analysis products formed <br />            S2 (top) HeLa cell lysates (bottom) incubated 50 &#194;&#181;M NBD-Cer presence absence 500 &#194;&#181;M CDP-ethanolamine (CDP-ethanol), <br />            5 mM MgCl2, 10 mM MnCl2. (F) TLC analysis products formed lysates dsRNA-treated S2 cells incubated NBD-Cer MnCl2 pres- <br />            ence absence CDP-ethanolamine E. <br />  <br />  <br />            active site residues His401 Asp405 abolished dSMSr-                      did enhance NBD-CPE production HeLa cell lysates <br />            mediated CPES activity (Fig. 4 A). Depletion endogenous                     (Fig. 4 E, bottom). <br />            dSMSr resulted 60&#226;&#8364;&#8220;70% drop cell lysate&#226;&#8364;&#8220;associated CPES                       In sum, insect cells contain distinct CPESs: PE/ceram- <br />            activity (Fig. 4 B, left). Surprisingly, significant reduction           ide ethanolamine phosphotransferase corresponding dSMSr and <br />            CPE production levels intact dSMSr-depleted                     dSMSr-unrelated CDP-ethanolamine/ceramide ethanolamine <br />            S2 cells metabolically labeled NBD-Cer (Fig. 4 B,                    phosphotransferase likely responsible bulk production of <br />            right) [14C]ethanolamine (Fig. 4 D), RNAi treat-               CPE. Importantly, findings confirm SMSr proteins in <br />            ment reduced dSMSr protein levels &#238;?&#186;90% (Fig. 4 C). These                    general lack ability produce bulk amounts CPE. <br />            results suggest S2 cells able produce CPE a <br />            second, dSMSr-independent enzyme.                                              Blocking SMSr catalytic activity causes <br />                  One reason second enzyme readily detect-                 ceramide accumulation ER <br />            able cell lysates requires soluble substrate        Although SMSr produces trace amounts CPE, SMSr- <br />            continuously regenerated living cells. CDP-ethanolamine               depleted HeLa cells showed fourfold increase ceramide <br />            attractive candidate substrate given role head-           levels (<span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> 0.31 &#194;&#177; 0.01 1.29 &#194;&#177; 0.15 mol%; n = 3). This was <br />            group donor PE biosynthesis (Vance, 2008). Indeed, CPE                      accompanied threefold increase GlcCer levels (from <br />            production S2 cell lysates increased dramatically addi-               0.32 &#194;&#177; 0.03 0.86 &#194;&#177; 0.09 mol%; n = 3; Fig. 5 A). Depletion of <br />            tion CDP-ethanolamine provided Mn2+ ions pres-                    SMS1 caused similar rise ceramide GlcCer levels. <br />            ent (Fig. 4 E, top). The CDP-ethanolamine&#226;&#8364;&#8220;dependent CPES                       In case, however, fully explained block <br />            activity unaffected dSMSr depletion (Fig. 4 F)                      SM production (Tafesse et al., 2007). In contrast, amount <br />            appears unique insect cells, addition CDP-ethanolamine               ceramide GlcCer accumulating SMSr-depleted HeLa <br />  <br />  <br />     1018   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  cells 50-fold higher ascribed block CPE          COPII subunit dSec23) maintained, structures <br /> production (Fig. 5 A). The rise ceramides affected major         fragmented, hazy appearance (Fig. 6 A; Fig. S4, A&#226;&#8364;&#8220;I). <br /> species (Fig. S3 A Table S1), detected             This effect quantitative, half depleted <br /> [14C]serine labeling, accompanied increase         cells displayed fragmented Golgi (Fig. 7 C). However, global <br /> sphingosine levels (Fig. S3 C). Moreover, occurred           architecture ER unaffected depletion. <br /> SMSr-depleted hamster CHO-K1 Drosophila S2 cells, al-                      At ultrastructural level, Golgi morphology dSMSr- <br /> though case lesser extent (Fig. S3 B).               depleted S2 cells profoundly affected. Instead showing <br />        As SMSr resides ER, ceramide accumulation          typical organization stacked cisternae (Fig. 6, B D), <br /> SMSr-depleted cells likely originates organelle. This         Golgi converted short tubular structures the <br /> was confirmed subcellular fractionation, showed           majority cells (Fig. 6, C, E, F). These tubular clusters <br /> the ER SMSr-depleted HeLa cells contains nearly 300%            devoid KDEL-containing proteins posi- <br /> ceramide ER lamin A (LA)&#226;&#8364;&#8220;depleted control cells             tive dSec23 (Fig. 6, E F), suggesting represent <br /> (Fig. 5, B C). To verify S2 cells, created      mixture Golgi tER membranes. <br /> ER-resident form hSMS1 analyzed ability reduce                   hSMSr-depleted HeLa cells showed similar pheno- <br /> ceramide levels dSMSr-depleted S2 cells. Consistent role      type. The Golgi ribbon completely fragmented than <br /> of SMSr-derived SAM ER retention, hSMS1 swap mutant               half cells indicated immunostaining markers <br /> carrying SAM-containing cytosolic tail hSMSr, hSMS1Nr,           medial (MannII-RFP) trans-Golgi compartments (TGN46; <br /> localized primarily ER (Fig. 5, D E). hSMS1Nr                Fig. 6 G; Fig. 7 C; Fig. S4, J&#226;&#8364;&#8220;L). These results strengthen <br /> retained SMS activity, expression S2 cells sup-           functional similarity hSMSr dSMSr. Moreover, <br /> ported production SM (Fig. 5 F). Expression ER resident           requirement SMSr Golgi integrity appears specific be- <br /> hSMS1Nr Golgi-associated hSMS1 prevented ac-                cause Golgi ribbon unperturbed hSMS1-depleted <br /> cumulation [14C]serine-labeled ceramides dSMSr-depleted           HeLa cells (Fig. 6 G). <br /> S2 cells (Fig. 5 G). Together, results indicate SMSr acts           We tested fragmentation early secretory <br /> as negative regulator ceramide levels ER.                   pathway result result inhibition anterograde <br />        To investigate SMSr requires catalytic activity   transport. To end, monitored transport efficiency the <br /> prevent ceramide accumulation, dSMSr-depleted S2 cells             transmembrane protein Delta ER PM S2 cells <br /> transfected wild type enzyme-dead version hSMSr.          (Kondylis Rabouille, 2003). Brefeldin A treatment blocked <br /> Expression wild-type hSMSr effectively reduced ceramide          PM delivery Delta caused accumulation ER. <br /> pool dSMSr-depleted S2 cells (Fig. 5 H). However, mutation        In contrast, PM delivery Delta significantly impaired in <br /> active site residue Asp348 Glu sufficient abrogate       dSMSr-depleted S2 cells, suggesting fragmented tER&#226;&#8364;&#8220; <br /> effect. Transfection hSMSr-depleted HeLa cells siRNA-           Golgi units cells fully competent anterograde trans- <br /> resistant hSMSr hSMSrD348E yielded results (Fig. 5 H).      port (Fig. S5 A). Similar results obtained hSMSr-depleted <br /> Thus, SMSr produces trace amounts CPE,              HeLa cells using well-established VSV-045G transport assay <br /> this appears essential control ceramide pool ER.          (Fig. S5, C D). Moreover, SMSr depletion major impact <br />                                                                         protein glycosylation (Fig. S5 E). Together, findings indi- <br /> Loss SMSr perturbs organization                                  cate SMSr proteins play critical role acquisition of <br /> of early secretory pathway                                          Golgi stack morphology affecting functional integrity <br /> Recent work revealed CERT&#238;&#8364;&#178;/&#238;&#8364;&#178; mouse embryos accumulate              organelle. This adds evidence notion trans- <br /> ceramide ER display chronic state ER stress char-       port cargo to, through, Golgi complex does re- <br /> acterized vesiculation engorgement ER up-             quire fully developed organelle proceed conditions <br /> regulation downstream components unfolded protein             cisternal stacks severely disorganized (Kondylis <br /> response pathway (Wang et al., 2009). Even SMSr-                 Rabouille, 2003; Zolov Lupashin, 2005). <br /> depleted S2 HeLa cells substantial rise ER cer- <br /> amide levels, did display morphological (ER         Golgi fragmentation SMSr-depleted <br /> dilation vesiculation) biochemical (XBP-1 splicing, induc-       cells caused ceramide accumulation <br /> tion CHOP/GADD153, up-regulation ER chaperones)               ER <br /> features ER stress (unpublished data). Instead,        To test catalytic activity SMSr required prevent <br /> the organization ER exit sites Golgi complex             Golgi fragmentation, monitored rescue phenotype by <br /> greatly perturbed.                                                      transfecting dSMSr-depleted S2 cells wild-type enzyme- <br />        Golgi stacks S2 cells form large Golgi ribbon        dead hSMSr. Expression wild-type hSMSr completely <br /> capping nucleus mammalian cells. Instead, remain         restored Golgi integrity. However, mutation active site residue <br /> discrete cytoplasm close proximity tran-     Asp348 Glu sufficient abolish rescuing effect (Fig. 7, <br /> sitional ER (tER) sites, forming 17&#226;&#8364;&#8220;25 tER&#226;&#8364;&#8220;Golgi units (Kondylis        A C). These results completely parallel obtained with <br /> and Rabouille, 2003; Kondylis et al., 2007). In dSMSr-depleted          hSMSr-depleted HeLa cells transfection siRNA-resistant <br /> S2 cells, tER&#226;&#8364;&#8220;Golgi units smaller                hSMSr hSMSrD348E (Fig. 7, B C). <br /> numerous mock-depleted cells. Though spatial relation-             One possible interpretation results struc- <br /> ship Golgi (marked d120kd) tER sites (marked         tural integrity early secretory pathway depends trace <br />  <br />  <br />                                                            SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                          1019 <br />             Figure 5.&#226;&#8364;&#402; SMSr-depleted cells accumulate ceramides ER. (A) HeLa cells treated si LA, si hSMSr, siRNA-targeting hSMS1 (si hSMS1) 3 d <br />            subjected lipid extraction. Lipid classes quantified MS/MS analyses expressed mole percent total lipid analyzed. Data shown <br />            representative independent experiments. (B) HeLa cells expressing hSMS1-V5 subjected subcellular fractionation, yielding PNS <br />            fractions enriched mitochondria (mito)/Golgi, PM/early endosomes (EE), ER. Equal amounts protein fraction analyzed by <br />            immunoblotting. Note ER fraction virtually free organelles minor contamination PM (&lt;10%). (C) MS/MS analysis <br />            ceramide PE levels ER fractions prepared HeLa cells treated si LA si hSMSr 3 d. (D) HeLa cells transfected hSMS1-V5 or <br />            hSMS1 swap mutant carrying N-terminal tail hSMSr, hSMS1Nr-V5, double labeled V5 cis-Golgi marker GM130. (E) S2 cells <br />            cotransfected hSMS1-V5 hSMS1Nr-V5 ER marker PEMT-GFP labeled V5. The white box marks cell captured different field. <br />            Note hSMS1Nr-V5 localizes ER nuclear envelope (D E, arrows). (F) S2 cells transfected hSMS1-V5 hSMS1Nr-V5 labeled <br />            [14C]choline 5 h subjected lipid extraction, TLC, autoradiography. (G) dSMSr-depleted S2 cells transfected hSMS1-V5 or <br />            hSMS1Nr-V5 labeled 5 h [14C]serine. Levels radiolabeled ceramides determined TLC autoradiography expressed a <br />            percent mock-treated cells. (H) SMSr-depleted cells (S2/HeLa) transfected hSMSr-V5 hSMSrD348E-V5 labeled 5 h [14C]serine. <br />            Levels radiolabeled ceramides determined G. SMSr depletions ds dSMSr #2 (S2) si hSMSr #2 (HeLa). Transfection HeLa cells <br />            siRNA-resistant (res) hSMSr constructs. Untransf, untransfected. Error bars: (G) SD, n = 3; (C H) range, n = 2. <br />  <br />  <br />  <br />  <br />     1020   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  Figure 6.&#226;&#8364;&#402; SMSr depletion disrupts tER&#226;&#8364;&#8220;Golgi units insect mammalian cells. (A) S2 cells treated 7 d ds GFP, ds dSMSr #1, ds dSMSr <br /> #2 double labeled dSec23 (tER sites) d120kd (Golgi stacks). Confocal projections merge presented. The single channels pre- <br /> sented Fig. S5 A. Bar, 5 &#194;&#181;m. (B&#226;&#8364;&#8220;F) S2 cells treated 5 d ds GFP (B D) dSMSr #2 (C, E, F) fixed processed immunoelectron <br /> microscopy. Ultrathin cryosections visualized (B C) double labeling dSec23 (15-nm gold) KDEL (10-nm gold; D&#226;&#8364;&#8220;F). Note <br /> that Golgi stacks dSMSr-depleted cells converted cluster short tubular profiles (arrowheads) cytoplasmic regions normally occupied by <br /> tER&#226;&#8364;&#8220;Golgi units (between brackets). These short tubules positively labeled Sec23, suggesting comprise membrane compartments. <br /> G, Golgi stack. (G) Mannosidase II-RFP&#226;&#8364;&#8220;expressing HeLa cells treated si LA, si hSMSr #2, hSMS1 3 d labeled TGN46 DNA <br /> (DAPI). Confocal projections merge presented. The single channels presented Fig. S5 (J&#226;&#8364;&#8220;L). Bars: (A G) 5 &#194;&#181;m; (B&#226;&#8364;&#8220;F) 200 nm. <br />  <br />  <br />  <br />  <br />                                                                 SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                                   1021 <br />             Figure 7.&#226;&#8364;&#402; Structural integrity tER&#226;&#8364;&#8220;Golgi units requires enzyme-active form SMSr. (A) S2 cells treated ds SMSr #2 transfected with <br />            hSMSr-V5 hSMSrD348E-V5. After 7 d, cells double labeled Golgi marker dGMAP V5 mark transfected cells. Confocal projections are <br />            presented. Note cells expressing hSMSr (arrows) typical wild-type Golgi organization 17&#226;&#8364;&#8220;25 large spots. In contrast, nontransfected <br />            cells expressing enzyme-dead hSMSr (arrowheads) typically numerous smaller Golgi spots. (B) Mann II-RFP&#226;&#8364;&#8220;expressing HeLa <br />            cells treated si hSMSr #2 transfected siRNA-resistant (res) hSMSr-V5 hSMSrD348E-V5. After 3 d, cells labeled V5 mark <br />            transfected cells. Confocal projections presented. Note cells expressing hSMSr-V5 contain intact Golgi (arrows). In contrast, cells expressing <br />            enzyme-dead hSMSr fragmented Golgi (arrowheads). (C) Quantitation rescue Golgi fragmentation phenotype SMSr-depleted <br />            cells (HeLa/S2) transfection wild-type enzyme-dead SMSr. SMSr-depleted cells compared control cells (si LA ds GFP treated), and <br />  <br />  <br />     1022   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  amounts CPE produced SMSr. Alternatively, enzyme-                  mechanism. However, SM readily dissociates from <br /> dead mutant SMSr unable prevent ceramide accumulation                 SMS1, propose CPE remains bound active site of <br /> in ER (Fig. 5 H), fragmentation early secretory pathway             SMSr, blocking round catalysis. This the <br /> in SMSr-depleted cells caused rise ceramide                 result factors. First, SMSr higher affinity <br /> levels ER. To distinguish possibilities,                  CPE PE. Second, dissociation SM SMS1 <br /> ER resident SMS hSMS1Nr expressed SMSr-depleted S2                   facilitated preferential interaction cholesterol. <br /> HeLa cells analyzed ability rescue Golgi fragmenta-             Contrary SM, CPE lacks interaction (Terova et al., 2005). <br /> tion. Expression hSMS1Nr partially restored Golgi integrity              Thus, presence cholesterol, SM production be <br /> both cell types (Fig. 7 D), indicating acquisition Golgi               thermodynamically favorable CPE production. <br /> stack morphology strictly dependent production CPE.                      The notion SMSr unable produce significant <br /> As expression hSMS1Nr effective preventing ceramide                  amounts CPE substantiated finding bulk <br /> accumulation SMSr-depleted cells (Fig. 5 G), asked               production CPE insects mediated different enzyme. <br /> blocking ceramide production stimulating ceramide export                    This second CPES uses CDP-ethanolamine instead PE as <br /> from ER suppress Golgi fragmentation. Treatment                 headgroup donor, analogous ethanolamine phospho&#194;&#173; <br /> with ceramide synthase inhibitor fumonisin B1 overexpression                transferases Kennedy pathway (Vance, 2008). This implies <br /> of CERT partially reproducibly restored Golgi integrity                 that, contrary SM synthesis mammals, bulk production of <br /> both SMSr-depleted HeLa S2 cells (Fig. 7 D). This indicates                CPE occurs cytosolic leaflet membrane. As insects <br /> that structural collapse early secretory pathway                 require CERT efficient CPE production (Rao et al., 2007), <br /> SMSr-depleted cells result ceramide accumulation                  second enzyme likely resides Golgi. <br /> the ER. Thus, results support primary role SMSr <br /> in controlling ceramide homeostasis ER.                                 SMSr qualifies ceramide sensor <br />                                                                                ER <br />  <br /> Discussion                                                                     Although SMSr poor CPES, enzyme a <br />                                                                                major impact ceramide levels ER. Indeed, SMSr- <br /> SMSr proteins define unconventional                                         depleted mammalian cells display rise ceramide levels is <br /> class sphingolipid synthases                                                order magnitude high explained reduced <br /> In study, SMSr, conserved member                 ceramide consumption CPE biosynthesis. SMSr-depleted in- <br /> the multigenic SMS family, catalyzes production SM ana-                 sect cells SMSr-<span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>null mutants</span> Caenorhabditis elegans also <br /> logue CPE. SMSr likely accounts CPES activity described                marked increase ceramide levels (this study; unpub- <br /> two decades ago rat brain liver microsomes (Malgat et al.,              lished data). Strikingly, single <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>point mutation</span> active site <br /> 1986, 1987). Common features enzymes include                     SMSr proved sufficient induce ceramide accumulation. This <br /> the use PE headgroup donor, ER residency, mem-               shows SMSr catalytically active ER cer- <br /> brane topology active site facing ER lumen.                amide levels low, enzyme lacks ability consume <br /> Although mammalian cells contain readily detectable CPES                     significant amounts ceramide. Together, results establish <br /> activity, CPE content low (i.e., 300-fold lower             SMSr novel regulator ceramide homeostasis ER. <br /> SM). This enigma caused limited supply sub-                        How does SMSr control ceramide levels ER? We <br /> strates enzyme&#226;&#8364;&#8482;s active site CPE short-lived             anticipate normal conditions, small amounts of <br /> metabolic intermediate. Instead, SMSr intrinsically               ceramide reach enzyme&#226;&#8364;&#8482;s active site. This cer&#194;&#173; <br /> lacks ability synthesize significant amounts CPE. This            amides synthesized cytosolic surface ER con- <br /> radically different closely related SMS1, catalyses                 tinuously removed CERT (Fig. 8 A). Consequently, a <br /> bulk production SM (Tafesse et al., 2007).                                  minor fraction SMSr enzymes trapped CPE-bound <br />        What makes SMSr different SMS1? SMS1 uti-                       form. Yet rate ceramide production exceeds ceramide <br /> lizes lipid phosphate phosphatase&#226;&#8364;&#8220;type reaction mechanism,                   removal CERT, pool CPE-bound enzymes expand. <br /> which involves single lipid-binding site proceeds trans-             It possible CPE-bound SMSr adopts conformation that <br /> fer phosphocholine PC conserved histidine                  relays signal inhibit ceramide biosynthesis and/or stimulate <br /> enzyme&#226;&#8364;&#8482;s active site (Huitema et al., 2004; Tafesse et al., 2006).             ceramide degradation (Fig. 8 B). Alternatively, SMSr-derived <br /> When DAG replaced ceramide, phosphocholine head-                     CPE directly influence activity ceramide-metabolizing <br /> group transferred ceramide form SM. The              enzymes ER. Either feedback mechanism ensure <br /> released active site allow round catalysis.             maintenance ceramide homeostasis ER. Distinguishing <br /> Mutational analysis putative active site residues indicates            possibilities require reconstitution experi- <br /> SMSr-mediated CPE production involves similar reaction                  ments purified enzymes. <br />  <br /> &#226;&#8364;&#8218; <br />  <br /> SMSr-depleted cells transfected wild-type enzyme-dead SMSr compared untransfected (untransf) SMSr-depleted cells. (D) Quantification <br /> of rescue Golgi fragmentation phenotype SMSr-depleted cells (Hela/S2) transfection hSMS1Nr-V5 CERT-Flag treatment <br /> with 50 &#194;&#181;g/ml fumonisin B1 (FB1) 2 d (HeLa) 5 h (S2) fixation. SMSr-depleted cells expressing hSMS1Nr-V5 CERT-Flag compared <br /> with untransfected cells, mock-treated cells compared FB1-treated cells. Errors bars indicate SD, n = 3. *, P &lt; 0.001 two-tailed unpaired <br /> Student&#226;&#8364;&#8482;s t test. Bars, 5 &#194;&#181;m. <br />  <br />  <br />                                                                SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                                 1023 <br />             Figure 8.&#226;&#8364;&#402; Model SMSr control ceramide homeostasis ER. (A) Normally, small amounts newly synthesized ceramides reach <br />            active site SMSr ER lumen result efficient removal cytosolic surface CERT. (B) When CERT can&#226;&#8364;&#8482;t cope rate of <br />            ceramide production, ceramides reach active site SMSr, resulting production CPE. Because high affinity enzyme&#226;&#8364;&#8482;s <br />            active site, CPE remains bound SMSr, blocking round catalysis. CPE-trapped SMSr adopts conformation relays signal inhibiting <br />            ceramide production stimulating ceramide breakdown ER, restoring ceramide homeostasis. Sph, sphingoid base. <br />  <br />  <br />                   Collectively, data suggest SMSr CPES                grow large produce COPII vesicles. A rise ER <br />            dual role ceramide sensor control ceramide homeostasis                 ceramides perturb local physical environment required to <br />            ER. As active site faces ER lumen, SMSr                  stabilize patches tER proteins. Alternatively, ceramides may <br />            sense ceramides present exoplasmic leaflet, i.e., molecules              affect tER site morphology altering membrane turnover of <br />            escaped CERT-mediated extraction cytosolic surface                  COPII components analogous situation sterol-depleted <br />            flipped lumenal side. Notably, arrangement implies              cells (Runz et al., 2006). Moreover, ceramides serve early medi- <br />            cooperativity SMSr CERT keeping ER ceramide                      ators apoptosis, event accompanied fragmentation the <br />            levels low. Thus, SMSr serves biological role different               Golgi complex. Apoptosis-induced Golgi fragmentation requires <br />            closely related SMS1 analogous ensemble                      caspase 3&#226;&#8364;&#8220;mediated cleavage golgins, family coiled-coil <br />            membrane proteins monitor sterol concentrations ER                  proteins involved maintenance Golgi stack morphology (Lane <br />            (Goldstein et al., 2006). In respect, SMS enzyme family                et al., 2002). Interestingly, SMSr-depleted HeLa cells display hall- <br />            displays interesting parallel <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> transporter family.          marks apoptosis, including activation caspase 3 (unpublished <br />            Besides conventional transporters, family harbors               data). Whether blocking caspase activation prevents Golgi frag- <br />            transceptors play dual role <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> sensor             mentation SMSr-deficient cells remains established. <br />            conformational change transport process trigger                          A tight regulation ceramide homeostasis ER likely <br />            downstream signaling pathways (Holsbeeks et al., 2004).                         functional implications sustaining structural <br />                                                                                            integrity early secretory pathway. Down-regulation CERT <br />            Functional implications SMSr-controlled                                      sensitizes cancer cells paclitaxel, suggesting ER ceramides <br />            ceramide homeostasis                                                            integral taxane-induced cell death (Swanton et al., 2007). <br />            Concurrent accumulation ceramide ER, blocking                Moreover, CERT&#238;&#8364;&#178;/&#238;&#8364;&#178; mouse embryos accumulate ceramide the <br />            SMSr activity causes structural collapse early secretory               ER display chronic state ER stress (Wang et al., 2009). <br />            pathway. The disruption tER&#226;&#8364;&#8220;Golgi organization completely                    As components unfolded protein response up-regulated <br />            parallels rise ER ceramide levels. Indeed, condition            animal development (Reimold et al., 2000; Iwakoshi et al., <br />            normalizes ceramide concentration ER restore                2007), defective transport ceramide potentiate ER <br />            Golgi stack morphology; i.e., presence enzymatically active              stress embryo (Wang et al., 2009). Whether loss SMSr <br />            SMSr, consumption ceramide ER resident SMS, stimula-                  sensitizes cells ER stress&#226;&#8364;&#8220;inducing conditions currently <br />            tion ER export ceramide overexpression CERT,                   investigation. <br />            block ceramide production ceramide synthase inhibitors.                           The ER highly efficient lipid distribution is <br />            Conversely, blocking SM production accumulate ceramides                   intimately associated organelles, including mitochon- <br />            Golgi does perturb morphology.                                      dria (Levine Rabouille, 2005). Ceramides impli- <br />                   How excess ceramides ER affect Golgi orga-              cated mitochondria-induced apoptosis reach these <br />            nization? As biogenesis Golgi tightly linked           organelles contact sites ER (Siskind et al., 2008; <br />            tER sites (Bevis et al., 2002), disruption tER sites likely con-           Stiban et al., 2008; Wang et al., 2009). The present identification <br />            tributes structural collapse Golgi. tER sites thought         SMSr key regulator ceramide homeostasis ER <br />            arise patches ER membrane proteins weak,                   provides novel opportunities explore role ceramides in <br />            cooperative affinity (Bevis et al., 2002; Heinzer               mitochondria-induced cell death modulating taxane <br />            et al., 2008). These patches capture additional tER proteins              effects tumors. <br />  <br />  <br />     1024   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  Materials methods                                                            CTGGAATTTCTTGC-3&#238;&#8364;&#185;; hSMS1, 5&#238;&#8364;&#185;-AACTACACTCCCAGTACCTGG-3&#238;&#8364;&#185;. <br />                                                                                  For rescue experiments, cells transfected siRNA-resistant hSMSr <br /> Chemicals                                                                        constructs 8 h start siRNA treatment using Effectene (QIAGEN). <br /> NBD-Cer obtained Invitrogen, NBD-SM, POPE (2-oleoyl-1-palmitoyl-        Constructs siRNA resistant introducing silent point muta- <br /> sn-glycerol-3-phosphoethanolamine), POPC (2-oleoyl-1-palmitoyl-sn-           tions siRNA target sequence. <br /> glycerol-3-phosphocholine) obtained Avanti Polar Lipids, Inc.                  Drosophila S2 cells grown Schneider&#226;&#8364;&#8482;s insect medium with <br /> NBD-CPE provided P. Devaux (Institut Biologie Physico-chimique,        10% FBS. Treatment double-stranded RNA (dsRNA) performed <br /> Paris, France). 2-[14C]ethan-1-ol-2-amine hydrochloride 3-[14C]L-serine      described previously (Clemens et al., 2000). On day 1, 106 cells were <br /> were obtained GE Healthcare, methyl-[14C]choline chloride           plated 35-mm dish incubated 30 &#194;&#181;g dsRNA 1 ml serum- <br /> [3H]sphingosine obtained MP Biomedicals. D4-ethanolamine               free medium 1 h RT followed addition 2 ml complete medium. <br /> and D9-choline obtained Cambridge Isotope Laboratories.                Treatment dsRNA repeated day 4. For rescuing experiments, <br /> All lipids chemicals obtained Sigma-Aldrich.                 cells transfected SMS/pMT/V5-HisB constructs day 5 using <br />                                                                                  Effectene. Expression recombinant SMS protein induced addi- <br />                                                                                  tion 1 mM CuSO4 3 h followed 2-h chase presence of <br /> Antibodies <br />                                                                                  150 &#194;&#181;g/ml cycloheximide. dsRNA synthesized vitro transcription <br /> The following antibodies used: rabbit polyclonal anti-V5 mouse <br />                                                                                  PCR products flanked T7 RNA polymerase-binding sites (5&#238;&#8364;&#185;-TTAATAC- <br /> monoclonal anti-Flag (Sigma-Aldrich), mouse monoclonal anti-V5 (Invitro- <br />                                                                                  GACTCACTATAGGGAGA-3&#238;&#8364;&#185;) using MEGASCRIPT T7 transcription <br /> gen), mouse monoclonal anti-GM130 (BD), mouse monoclonal anti-KDEL, <br />                                                                                  kit (Applied Biosystems). dsRNA-targeting dSMSr synthesized from <br /> rabbit polyclonal anti-calnexin anti-GRP75 (Santa Cruz Biotechnology, <br />                                                                                  &#238;?&#186;800-bp PCR products generated dSMSr cDNA using <br /> Inc.), mouse monoclonal anti-d120kd (EMD), rabbit polyclonal anti-dGOL- <br />                                                                                  following primer sets: 5&#238;&#8364;&#185;-T7-atgtgcgacggtgaaattg-3&#238;&#8364;&#185;/5&#238;&#8364;&#185;-T7- <br /> GIN245 (provided S. Munro, Medical Research Council Laboratory of <br />                                                                                  gagccggttatttcgcacat-3&#238;&#8364;&#185; (dsRNA-targeting SMSr [ds SMSr] #1) <br /> Molecular Biology, Cambridge, England, UK), rabbit polyclonal anti-Na+/ <br />                                                                                  5&#238;&#8364;&#185;-T7-ttggtcctactgaggcgattc-3&#238;&#8364;&#185;/5&#238;&#8364;&#185;-T7-cgactggtgag- <br /> K+-ATPase (C356-M09; provided J. Koenderink, Radboud University <br />                                                                                  gactaaagaaagc-3&#238;&#8364;&#185; (ds SMSr #2). dsRNA-targeting GFP (ds GFP) <br /> Nijmegen, Nijmegen, Netherlands), rabbit polyclonal antiprotein disulfide <br />                                                                                  synthesized &#238;?&#186;800-bp PCR product, generated <br /> isomerase (provided I. Braakman, Utrecht University, Utrecht, Nether- <br />                                                                                  pMT/EGFP plasmid using primer set 5&#238;&#8364;&#185;-T7-CTAGGCGGCCG- <br /> lands), rabbit polyclonal anti-GFP (Invitrogen), mouse monoclonal anti- <br />                                                                                  CAAAAAATG-3&#238;&#8364;&#185;/5&#238;&#8364;&#185;-T7-CTGTTCGAAGGCGCGCCTTGT-3&#238;&#8364;&#185;. <br /> EEA1 (BD), rabbit polyclonal anti-dGMAP (Kondylis et al., 2007), rabbit <br /> polyclonal anti-dSec23p, mouse monoclonal C594.9B (Kondylis and <br /> Rabouille, 2003). Horseradish peroxidase&#226;&#8364;&#8220;conjugated secondary anti-              In vitro enzyme assays <br /> bodies obtained PerBio, antibodies conjugated FITC          100 ODs yeast cells lysed bead bashing 10 ml ice-cold reac- <br /> and Texas red Alexa dyes obtained Jackson Immunoresearch            tion buffer (0.3 M sucrose, 15 mM KCl, 5 mM NaCl, 1 mM EDTA, 20 mM <br /> Laboratories Invitrogen, respectively.                                       Hepes-KOH, pH 7.0) containing freshly added protease inhibitors. HeLa <br />         To raise antibody dSMSr, N terminus dSMSr (resi-       S2 cells lysed ice-cold reaction buffer passing 20 times <br /> dues 1&#226;&#8364;&#8220;199) expressed glutathione S-transferase fusion Esche-          26-G 0.75 needle. 200 &#194;&#181;l postnuclear supernatant (PNS; 700 g <br /> richia coli purified affinity chromatography glutathione agarose       10 min 4&#194;&#176;C) combined 200 &#194;&#181;l reaction buffer containing <br /> (Sigma-Aldrich). The purified fusion protein used PickCell Laborato-      0.002% Triton X-100, 40 nmol POPE and/or POPC, 50 &#194;&#181;M NBD-Cer <br /> ries immunization mice, yielding monoclonal anti-dSMSr anti-          (Avanti Polar Lipids, Inc.), incubated 37&#194;&#176;C (HeLa/yeast) 27&#194;&#176;C <br /> body B5-G7.                                                                      (S2 cells) 2 h. Reactions stopped adding 1 ml MeOH 0.5 ml <br />                                                                                  CHCl3, lipids extracted according Bligh Dyer (1959). <br /> DNA constructs                                                                   The lower phase evaporated N2 reaction products ana- <br /> hSMSr, hSMS1, dSMSr cDNAs cloned yeast expression vec-             lyzed TLC, developed acetone CHCl3/ <br /> tor pYES2.1/V5-His-TOPO mammalian expression vector pcDNA3.1/                MeOH/25% NH4OH (50/25/6 [vol/vol/vol]; Figs. 1 2) CHCl3/ <br /> V5-His-TOPO (Invitrogen) described previously (Huitema et al., 2004).         acetone/MeOH/acetic acid/H2O (50/20/10/10/5 [vol/vol/vol/vol/ <br /> For expression studies Drosophila S2 cells, dSMSr, hSMSr, hSMS1           vol]; Fig. 4). Fluorescent lipids visualized image analysis sys- <br /> cDNAs PCR amplified ligated copper-inducible pMT/              tem (STORM 860; GE Healthcare) quantified Quantity One soft- <br /> V5-His B vector (Invitrogen). Mutation active site residues dSMSr (H401A   ware (Bio-Rad Laboratories). <br /> and D405E) hSMSr (D348E) performed using megaprimer PCR <br /> method (Orr-Weaver et al., 1983). SAM-deficient mutants obtained            Lipid MS <br /> by truncation 186 N-terminal residues dSMSr 68      100 ODs yeast cells washed water, treated trichloroacetic <br /> N-terminal residues hSMSr. An ER-resident hSMS1 swap mutant, hSMS1Nr,         acid inactivate lipases, finally boiled 10 min lipid extrac- <br /> was swapping cytosolic N terminus hSMS1 (residues 1&#226;&#8364;&#8220;133)          tion bead bashing CHCl3/MeOH/H2O (10/10/3 [vol/vol/vol]) ac- <br /> with hSMSr (residues 1&#226;&#8364;&#8220;150). The cDNA human PEMT pur-             cording Folch et al. (1957). HeLa S2 lipid extracts prepared <br /> chased RZPD (clone IRATp970D1078D6) cloned pMT/EGFP.               according Folch et al. (1957), omitting salt. The lower phase was <br /> Human CERT&#226;&#8364;&#8220;Flag (Fugmann et al., 2007) Drosophila CERT&#226;&#8364;&#8220;Flag ex-              evaporated N2. Glycerolipids removed mild alkaline hydro- <br /> pression constructs (Rao et al., 2007) provided M. Olayioye (Uni-        lysis 0.5 M sodium methoxide MeOH 1 h RT. The lipids were <br /> versity Stuttgart, Stuttgart, Germany) J. Acharya (National Cancer        analyzed triple quadrupole mass spectrometer (PE Sciex API-365; <br /> Institute Frederick, Frederick, MD), respectively.                            Applied Biosystems). Positive ions generated turbo ion spray <br />                                                                                  ionization source operating +5.5 kV ionization potential. N2 used <br /> Yeast culture                                                                    drying gas 350&#194;&#176;C. The declustering potential (cone voltage) set <br /> Yeast strain 4&#238;?&#8222;.Lass5 (MATa ade2-101ochre his3-&#238;?&#8222;200 leu2-&#238;?&#8222;1 lys2-801amber        50 V, collision energy 36 V, entrance potential &#238;&#8364;&#178;10 V, and <br /> trp1-&#238;?&#8222;63 ura3-52 lag1::TRP1 lac1::LEU2 ydc1::natMX ypc1::kanMX4                  focus potential 160 V. The lipid molecular species classes were <br /> p413MET25:Lass5; Cerantola et al., 2007) transfected hSMSr           quantified described previously Tafesse et al. (2007). <br /> dSMSr pYES2.1/V5-His-TOPO grown selective synthetic medium <br /> containing 2% (wt/vol) galactose 50 mg/l myo-inositol.                       Metabolic labeling <br />                                                                                  S2 cells (2&#226;&#8364;&#8220;5 &#195;&#8212; 106) labeled 0.5 ml complete Schneider&#226;&#8364;&#8482;s insect <br /> Animal cell culture RNAi                                                     medium 10 nmol NBD-Cer, 1 &#194;&#181;Ci [14C]ethanolamine, 1 &#194;&#181;Ci [14C]choline, <br /> CHO-K1 LY-A cells grown F-12 medium 10% FCS (Invitro-           1 &#194;&#181;Ci [14C]serine 27&#194;&#176;C indicated time. Mammalian cells (10-cm <br /> gen), LY-A/human CERT cells grown F-12 medium contain-           dish; 50% confluency) labeled complete medium 1 &#194;&#181;Ci <br /> ing 0.8 mg/ml G-418 (Invitrogen). HepG2 cells grown DME             [14C]ethanolamine 1 &#194;&#181;Ci [3H]sphingosine indicated time. Lipids <br /> 10% FBS (Cambrex) CaCo-2 cells DME-GlutamaxI 10% FCS.                extracted CHCl3/MeOH/10 mM acetic acid (1/4.4/0.2 [vol/ <br /> HeLa cells grown DME 10% FCS. Stable transfections             vol/vol]) processed according Bligh Dyer (1959). Half the <br /> hSMS1-V5, hSMSr-V5, mouse &#238;?&#161;-mannosidase II-RFP pcDNA3.1                extract subjected mild alkaline hydrolysis. Radiolabeled lipids were <br /> treatment siRNA (QIAGEN) performed described previously             analyzed TLC CHCl3/MeOH/25% NH4OH (50/25/6 [vol/vol/vol]; <br /> (Tafesse et al., 2007). siRNA target sequences were: nonsense, 5&#238;&#8364;&#185;-AA&#194;&#173;T&#194;&#173;TC&#194;&#173;       [14C]ethanolamine/choline), CHCl3/MeOH/2 M NH4OH (40/10/1 <br /> TCCGAACGTGTCACGT-3&#238;&#8364;&#185;; LA, 5&#238;&#8364;&#185;-AACTGGACTTCCAGAAGAACA-3&#238;&#8364;&#185;;                            [vol/vol/vol]; [14C]serine), CHCl3/MeOH/0.2% CaCl2 (60/40/9 [vol/ <br /> hSMSr #1, 5&#238;&#8364;&#185;-AATCTTCTTCATCTTGGCTGC-3&#238;&#8364;&#185;; hSMSr #2, 5&#238;&#8364;&#185;-CA&#194;&#173;AGA&#194;&#173;A&#194;&#173;G&#194;&#173;                  vol/vol]; [3H]sphingosine), detected exposure imaging screens (BAS-MS; <br />  <br />  <br />                                                                    SMSr CONTROLS CERAMIDE HOMEOSTASIS IN THE ER &#226;&#8364;&#162; Vacaru et al.                                   1025 <br />             FujiFilm), scanned Personal Molecular Imager (Bio-Rad Laboratories),      transport PM impaired SMSr-depleted cells. Table S1 shows <br />            quantified Quantity One software.                                     quantitative analysis ceramide species control, hSMSr-depleted, <br />                                                                                           hSMS1-depleted HeLa cells. Online supplemental material available <br />            Isolation ER membranes                                                      http://www.jcb.org/cgi/content/full/jcb.200903152/DC1. <br />            HeLa cells washed twice harvested scraping ice-cold <br />            homogenization buffer (HB; 10 mM Hepes, pH 7.8, 1 mM EGTA, 25 mM               We thank Kenneth Hanada, Sean Munro, Andreas Conzelmann, Philippe <br />            KCl, 1 mM protease inhibitor cocktail, 1 mM PMSF) supplemented        Devaux, Jairaj Acharya, Monilola Olayioye gifts reagents cell <br />            0.25 M sucrose. Cells pelleted centrifugation 600 g gently      lines, Tarja Grundstr&#195;&#182;m, Elleke Bosma, Martin Harterink technical <br />            disrupted dounce homogenizer 11 ml HB 4&#194;&#176;C. The homogenate           assistance, colleagues department Membrane Enzymology <br />            centrifuged twice 500 average g (gav; 5 min 4&#194;&#176;C) generate       stimulating discussions critical reading manuscript. <br />            PNS. The PNS centrifuged 10,000 gav (10 min 4&#194;&#176;C) pellet mito-              A.M. Vacaru supported European Union Framework V Program <br />            chondria. The postmitochondrial supernatant loaded 1 ml 68%         (the Flippase Project), P. Ternes supported Marie Curie Intra-European <br />            (wt/wt) sucrose cushion (prepared HB) centrifuged 100,000 gav.       Fellowship European Molecular Biology Organization Long-Term Fellow- <br />            (1 h 4&#194;&#176;C) collect crude microsomes. The microsomes resuspended      ship, V. Kondylis supported Horizon Programme National Regie- <br />            1 ml HB loaded 25&#226;&#8364;&#8220;60% (wt/wt) sucrose step gradient         Orgaan Genomics (grant 050-71-029), J.C.M. Holthuis supported <br />            generated HB using following steps: 2 ml 25%, 2 ml 30%,        grants Dutch Organization Sciences (NWO-CW) Utrecht <br />            2 ml 37%, 2 ml 39%, 2 ml 42%, 1 ml 60%. After centrifugation            University High Potential Program. <br />            100,000 gav (18 h 4&#194;&#176;C) rotor (SW41 Ti; Beckman Coulter), 20 &#195;&#8212; <br />            0.6 ml fractions collected top. PM ER membranes peaked       Submitted: 30 March 2009 <br />            fractions 7&#226;&#8364;&#8220;9 17 and18, respectively. Equal amounts protein     Accepted: 18 May 2009 <br />            PNS, crude mitochondrial membrane pellet, collected PM <br />            ER membrane peaks subjected immunoblot analysis. This re- <br />            vealed collected ER membranes virtually devoid other <br />            organelles minor contamination PM (&lt;10%).                    References <br />            Microscopy image analysis                                                  Andrieu-Abadie, N., T. Levade. 2002. Sphingomyelin hydrolysis during <br />            Cells fixed 4% paraformaldehyde/PBS, processed immuno-                    apoptosis. Biochim. Biophys. Acta. 1585:126&#226;&#8364;&#8220;134. <br />            fluorescence described previously S2 (Kondylis Rabouille, 2003)     Aronova, S., K. Wedaman, P.A. Aronov, K. Fontes, K. Ramos, B.D. Hammock, <br />            HeLa cells (Tafesse et al., 2007), mounted Vectashield medium              T. Powers. 2008. Regulation ceramide biosynthesis TOR com- <br />            containing DAPI (Vector Laboratories). Images captured RT using             plex 2. Cell Metab. 7:148&#226;&#8364;&#8220;158. <br />            confocal microscope (LSM 510 Meta; Carl Zeiss, Inc.) 63&#195;&#8212; 1.40           Bevis, B.J., A.T. Hammond, C.A. Reinke, B.S. Glick. 2002. De novo forma- <br />            NA Plan Apo oil objective (Carl Zeiss, Inc.) confocal microscope            tion transitional ER sites Golgi structures Pichia pastoris. Nat. <br />            (D-eclipse C1; Nikon) 60&#195;&#8212; 1.40 NA Plan Apo oil objective (Nikon).                Cell Biol. 4:750&#226;&#8364;&#8220;756. <br />            The fluorochromes used DAPI, &#238;?&#172;ex = 360 nm &#238;?&#172;em= 460 nm; FITC/          Bligh, E.G., W.J. Dyer. 1959. A rapid method total lipid extraction and <br />            Alexa Fluor 488, &#238;?&#172;ex = 488 nm &#238;?&#172;em = 515 nm; Texas red/Alexa Fluor                 purification. Can. J. Biochem. Physiol. 37:911&#226;&#8364;&#8220;917. <br />            568, &#238;?&#172;ex = 568 nm &#238;?&#172;em = 585 nm. Projections obtained col-          Bretscher, M.S., S. Munro. 1993. Cholesterol Golgi apparatus. <br />            lecting series 0.4-&#194;&#181;m sections acquired combined using               Science. 261:1280&#226;&#8364;&#8220;1281. <br />            LSM software (Carl Zeiss, Inc.) EZ-C1 software (Nikon). Images         Cerantola, V., C. Vionnet, O.F. Aebischer, T. Jenny, J. Knudsen, A. <br />            processed using Photoshop software (version 7.0.1; Adobe).                    Conzelmann. 2007. Yeast sphingolipids need contain long <br />                     The tER&#226;&#8364;&#8220;Golgi fragmentation phenotype S2 cells assessed            chain fatty acids. Biochem. J. 401:205&#226;&#8364;&#8220;216. <br />            described previously Kondylis Rabouille (2003). In brief, cells     Clemens, J.C., C.A. Worby, N. Simonson-Leff, M. Muda, T. Maehama, B.A. <br />            projection assigned categories: cells             Hemmings, J.E. Dixon. 2000. Use double-stranded RNA inter- <br />            wild-type Golgi morphology, corresponds 17&#226;&#8364;&#8220;25 large,                  ference Drosophila cell lines dissect signal transduction pathways. <br />            dGMAP, d120kd-labeled spots, cells fragmented Golgi mor-                Proc. Natl. Acad. Sci. USA. 97:6499&#226;&#8364;&#8220;6503. <br />            phology, corresponding numerous smaller labeled spots. Results         Folch, J., M. Lees, G.H. Sloane Stanley. 1957. A simple method isola- <br />            expressed percentage cells displaying fragmented Golgi mor-                 tion purification total lipides animal tissues. J. Biol. Chem. <br />            phology. The averages derived independent experiments,                226:497&#226;&#8364;&#8220;509. <br />            analyzing 400 (or &#226;&#8240;&#165;75 transfected) cells condition. The      Fugmann, T., A. Hausser, P. Schoffler, S. Schmid, K. Pfizenmaier, M.A. <br />            statistical significance data obtained assessed two-tailed un-          Olayioye. 2007. Regulation secretory transport protein kinase <br />            paired Student&#226;&#8364;&#8482;s t tests. P &#226;&#8240;&#164; 0.001 considered significant                 D&#226;&#8364;&#8220;mediated phosphorylation ceramide transfer protein. J. Cell Biol. <br />            marked asterisk, P &gt; 0.001 indicated (Fig. 7, C D).              178:15&#226;&#8364;&#8220;22. <br />                     S2 cells fixed processed immunoelectron microscopy       Goldstein, J.L., R.A. DeBose-Boyd, M.S. Brown. 2006. Protein sensors for <br />            described previously Kondylis Rabouille (2003). Ultrathin cryo-             membrane sterols. Cell. 124:35&#226;&#8364;&#8220;46. <br />            sections examined using electron microscope (1200 EX; JEOL).           Hanada, K., K. Kumagai, S. Yasuda, Y. Miura, M. Kawano, M. Fukasawa, and <br />            All antibodies used immunolabeling described Online supple-                M. Nishijima. 2003. Molecular machinery non-vesicular trafficking of <br />            mental material.                                                                      ceramide. Nature. 426:803&#226;&#8364;&#8220;809. <br />                                                                                           Hannun, Y.A., L.M. Obeid. 2008. Principles bioactive lipid signalling: les- <br />            Delta VSV-045G transport assays                                                   sons sphingolipids. Nat. Rev. Mol. Cell Biol. 9:139&#226;&#8364;&#8220;150. <br />            The Delta transport assay performed using cell line Delta-WTNd-        Heinzer, S., S. Worz, C. Kalla, K. Rohr, M. Weiss. 2008. A model the <br />            eMYC described previously Kondylis Rabouille (2003). In brief,              self-organization exit sites endoplasmic reticulum. J. Cell Sci. <br />            S2 cells treated different dsRNA indicated time. The                121:55&#226;&#8364;&#8220;64. <br />            Delta protein induced 1 h 1 mM CuSO4 chased            Hirschberg, K., J. Rodger, A.H. Futerman. 1993. The long-chain sphingoid <br />            absence CuSO4 different time points. Subsequently, cells                  base sphingolipids acylated cytosolic surface endoplasmic <br />            fixed, immunostained, Delta arrival PM monitored immuno&#194;&#173;                reticulum rat liver. Biochem. J. 290:751&#226;&#8364;&#8220;757. <br />            fluorescence microscopy. For VSV-045G transport assay, HeLa cells         Holsbeeks, I., O. Lagatie, A. Van Nuland, A. Van Velde, J.M. Thevelein. <br />            treated different siRNA constructs 7 d, transfected VSV-                2004. The eukaryotic plasma membrane <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing device. <br />            045G-GFP, kept 39.5&#194;&#176;C 16 h, chased 32&#194;&#176;C different                   Trends Biochem. Sci. 29:556&#226;&#8364;&#8220;564. <br />            time points indicated.                                                      Holthuis, J.C., T. Pomorski, R.J. Raggers, H. Sprong, G. van Meer. 2001. <br />                                                                                                  The organizing potential sphingolipids intracellular membrane <br />            Online supplemental material                                                          transport. Physiol. Rev. 81:1689&#226;&#8364;&#8220;1723. <br />            Fig. S1 shows effect externally added phospholipid CPE produc-       Huitema, K., J. van den Dikkenberg, J.F. Brouwers, J.C. Holthuis. 2004. <br />            tion levels lysates normal hSMSr-overexpressing HeLa cells. Fig.            Identification family animal sphingomyelin synthases. EMBO <br />            S2 shows loss SAM domain causes redistribution SMSr             J. 23:33&#226;&#8364;&#8220;44. <br />            ER Golgi. Fig. S3 shows SMSr-depleted cells accumulate     Iwakoshi, N.N., M. Pypaert, L.H. Glimcher. 2007. The transcription factor <br />            major species ceramide sphingosine. Fig. S4 shows single chan-          XBP-1 essential development survival dendritic cells. <br />            nels images presented Fig. 6. Fig. S5 shows anterograde protein            J. Exp. Med. 204:2267&#226;&#8364;&#8220;2275. <br />  <br />  <br />     1026   JCB &#226;&#8364;&#162; VOLUME 185 &#226;&#8364;&#162; NUMBER 6 &#226;&#8364;&#162; 2009 <br />  Kobayashi, S.D., M.M. Nagiec. 2003. Ceramide/long-chain base phosphate            Vance, J.E. 2008. Phosphatidylserine phosphatidylethanolamine mamma- <br />        rheostat Saccharomyces cerevisiae: regulation ceramide synthesis                lian cells: metabolically related aminophospholipids. J. Lipid Res. <br />        Elo3p Cka2p. Eukaryot. Cell. 2:284&#226;&#8364;&#8220;294.                                        49:1377&#226;&#8364;&#8220;1387. <br /> Kondylis, V., C. Rabouille. 2003. A novel role dp115 organization      Wang, X., R.P. Rao, T. Kosakowska-Cholody, M.A. Masood, E. Southon, H. <br />        tER sites Drosophila. J. Cell Biol. 162:185&#226;&#8364;&#8220;198.                                Zhang, C. Berthet, K. Nagashim, T.K. Veenstra, L. Tessarollo, et al. 2009. <br /> Kondylis, V., H.E. van Nispen tot Pannerden, B. Herpers, F. Friggi-Grelin, C.            Mitochondrial degeneration apoptosis primary cause of <br />        Rabouille. 2007. The Golgi comprises paired stack separated              embryonic lethality ceramide transfer protein mutant mice. J. Cell Biol. <br />        G2 modulation actin cytoskeleton Abi Scar/WAVE.                 184:143&#226;&#8364;&#8220;158. <br />        Dev. Cell. 12:901&#226;&#8364;&#8220;915.                                                         Zolov, S.N., V.V. Lupashin. 2005. Cog3p depletion blocks vesicle-mediated <br /> Kumagai, K., M. Kawano, F. Shinkai-Ouchi, M. Nishijima, K. Hanada. 2007.                 Golgi retrograde trafficking HeLa cells. J. Cell Biol. 168:747&#226;&#8364;&#8220;759. <br />        Interorganelle trafficking ceramide regulated phosphorylation- <br />        dependent cooperativity PH START domains CERT. <br />        J. Biol. Chem. 282:17758&#226;&#8364;&#8220;17766. <br /> Lane, J.D., J. Lucock, J. Pryde, F.A. Barr, P.G. Woodman, V.J. Allan, and <br />        M. Lowe. 2002. Caspase-mediated cleavage stacking protein <br />        GRASP65 required Golgi fragmentation apoptosis. J. Cell <br />        Biol. 156:495&#226;&#8364;&#8220;509. <br /> Levine, T., C. Rabouille. 2005. Endoplasmic reticulum: continuous network <br />        compartmentalized extrinsic cues. Curr. Opin. Cell Biol. 17:362&#226;&#8364;&#8220;368. <br /> Malgat, M., A. Maurice, J. Baraud. 1986. Sphingomyelin ceramide- <br />        phosphoethanolamine synthesis microsomes plasma membranes <br />        rat liver brain. J. Lipid Res. 27:251&#226;&#8364;&#8220;260. <br /> Malgat, M., A. Maurice, J. Baraud. 1987. Sidedness ceramide-phospho- <br />        ethanolamine synthesis rat liver brain microsomal membranes. <br />        J. Lipid Res. 28:138&#226;&#8364;&#8220;143. <br /> Mandala, S.M., R. Thornton, Z. Tu, M.B. Kurtz, J. Nickels, J. Broach, R. <br />        Menzeleev, S. Spiegel. 1998. Sphingoid base 1-phosphate phospha- <br />        tase: key regulator sphingolipid metabolism stress response. <br />        Proc. Natl. Acad. Sci. USA. 95:150&#226;&#8364;&#8220;155. <br /> Mandon, E.C., I. Ehses, J. Rother, G. van Echten, K. Sandhoff. 1992. <br />        Subcellular localization membrane topology serine palmitoyltrans- <br />        ferase, 3-dehydrosphinganine reductase, sphinganine N-acyltransferase <br />        mouse liver. J. Biol. Chem. 267:11144&#226;&#8364;&#8220;11148. <br /> Morales, A., H. Lee, F.M. Goni, R. Kolesnick, J.C. Fernandez-Checa. 2007. <br />        Sphingolipids cell death. Apoptosis. 12:923&#226;&#8364;&#8220;939. <br /> Muehlenberg, B.A., M. Sribney, M.K. Duffe. 1972. Occurrence bio- <br />        synthesis ceramide phosphorylethanolamine chicken rat liver. <br />        Can. J. Biochem. 50:166&#226;&#8364;&#8220;173. <br /> Orr-Weaver, T.L., J.W. Szostak, R.J. Rothstein. 1983. Genetic applications of <br />        yeast transforma <br /> </body></html>