<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RESEARCH ARTICLE <br />  <br />  <br />  <br />  <br /> The Ubiquitous Conserved Glycopeptidase Gcp Prevents <br /> Accumulation Toxic Glycated Proteins <br /> Chen Katz, Ifat Cohen-Or, Uri Gophna, Eliora Z. Ron <br /> Department Molecular Microbiology Biotechnology, Life Sciences, Tel Aviv University, Tel Aviv, Israel <br /> U.G. E.Z.R. contributed equally article. <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> Amadori-modified proteins (AMPs) products nonenzymatic glycation formed reaction reducing sug- <br /> ars primary amine-containing amino acids develop advanced glycated end products (AGEs), highly stable toxic <br /> compounds. AGEs known participate age-related human diseases, including cardiovascular, neurological, and <br /> liver diseases. The metabolism glycated proteins understood, mechanisms reduce accumula- <br /> tion known far. Here, Escherichia coli conserved glycopeptidase (Gcp, called Kae1), is <br /> encoded nearly sequenced genome domains life, prevents accumulation Amadori products and <br /> AGEs. Using mutants, Gcp depletion results accumulation AMPs eventually leads accumulation of <br /> AGEs. We demonstrate Gcp binds glycated proteins, including pyruvate dehydrogenase, previously shown a <br /> glycation-prone enzyme. Our experiments severe <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> Gcp depletion relieved condi- <br /> tions low intracellular glycation. As glycated proteins ubiquitous, involvement Gcp metabolism AMPs and <br /> AGEs likely conserved evolution, suggesting universal involvement Gcp cellular aging explaining <br /> the essentiality Gcp organisms. <br /> IMPORTANCE Glycated proteins (Amadori-modified proteins [AMPs] advanced glycated end products [AGEs]) known <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> participate age-related diseases. Their existence fast-growing organisms considered unlikely, formation <br /> was assumed slow. Yet, recent evidence demonstrated existence bacteria, data suggest bacterial mecha- <br /> nism reduced accumulation. We identify Escherichia coli protein, Gcp, carries function. Gcp con- <br /> served domains life essential organisms. Although annotated chaperon protease, no <br /> experimental data support function. Our findings compatible annotation open studies bac- <br /> terial metabolism glycated proteins. Furthermore, data bacterial systems instrumental under- <br /> standing metabolism glycated proteins, including toxicity human health disease. <br />  <br />    Received 19 July 2010 Accepted 26 July 2010 Published 24 August 2010 <br />    Citation Katz, C., I. Cohen-Or, U. Gophna, E. Z. Ron. 2010. The ubiquitous conserved glycopeptidase Gcp prevents accumulation toxic glycated proteins. mBio 1(3): <br />    e00195-10. doi:10.1128/mBio.00195-10. <br />    Editor Rino Rappuoli, Novartis Vaccines Diagnostics <br />    Copyright &#194;&#169; 2010 Katz et al. This open-access article distributed terms Creative Commons Attribution-Noncommercial-Share Alike 3.0 Unported License, <br />    permits unrestricted noncommercial use, distribution, reproduction medium, provided original author source credited. <br />    Address correspondence Eliora Z. Ron, eliora@post.tau.ac.il. <br />  <br />  <br />  <br />  <br /> O     -sialoglycoprotein endopeptidase (OSGEP), called Gcp <br />       (glycopeptidase), tops list &#226;&#8364;&#339;conserved hypothetical&#226;&#8364;? <br /> proteins, considered priority targets experi- <br />                                                                                             ATP-dependent proteases chaperon activity (5), no <br />                                                                                             function demonstrated Gcp homolog. In Saccha- <br />                                                                                             romyces cerevisiae, Gcp homolog Kae1 of <br /> mental studies Galperin Koonin (1). This ubiquitous en-                              complex involved telomere uncapping elongation (6, 7), <br /> zyme highly conserved proteins encoded                           Gcp homolog hyperthermophilic archaeon Pyrococ- <br /> every sequenced genome domains life,                               cus abyssi exhibits apurinic endonuclease activity vitro (8). <br /> exception highly degraded, organelle-like bacterial genomes                          These data suggest Gcp involved DNA metabolism, also <br /> of Carsonella ruddii Sulcia muelleri (2). Gcp described                       supported finding Gcp-depleted Escherichia coli cells <br /> as secreted protein Mannheimia haemolytica (formerly known                             modified nucleoids Gcp-deficient yeast mitochon- <br /> as Pasteurella haemolytica), isolated bovine pneumonia,                                dria lose mitochondrial DNA (9&#226;&#8364;&#8220;11). These studies not <br /> where exhibits glycopeptidase activity specificity O-sialic                           provide direct evidence molecular function GCP but <br /> acid-containing glycoproteins (3). However, ubiquity Gcp                             indicate function affects broad range physiological <br /> homologs explained function, sialic acids                          activities. <br /> highly uncommon organisms vertebrates (4).                                       Gcp represents evolutionary puzzle: highly <br />     There publications dealing                             similar conserved proteins (2) diverged func- <br /> potential role Gcp homologs. These homologs                                tion domains life, divergence se- <br /> have HSP70 actin-like folding predicted                            quence? Alternatively, possible conserved roles <br />  <br />  <br /> July/August 2010 Volume 1 Issue 3 e00195-10                                                                                                                       mbio.asm.org 1 <br />  Katz et al. <br />  <br />  <br />  <br />  <br /> FIG 1 Construction growth phenotype Gcp-depleted strain. (A) Schematic representation Gcp-depleted strain. The chromosomal ygjD gene, <br /> coding Gcp, deleted complemented low-copy-number plasmid arabinose (Ara)-inducible ygjD allele. (B) Growth Gcp-depleted <br /> strain presence 0.2% arabinose 0.2% glucose activation repression ygjD expression (respectively). Growth monitored measuring <br /> absorbance 600 nm. (C) Scanning electron micrographs wild-type Gcp-depleted cells. <br />  <br />  <br />  <br /> of Gcp identified Gcp or-               complex, previously shown susceptible car- <br /> thologs multiple physiological roles.                                    bonylation E. coli (22). Moreover, correlation be- <br />     We assumed Gcp functions glycoprotein-related                   tween internal level glycation inhibitory effect of <br /> chaperon protease, suggested bioinformatics, structural                 Gcp depletion. <br /> analysis (5), enzymatic glycopeptidase activity (3). Therefore,               These findings indicate direct involvement Gcp me- <br /> we looked sugar-related pathway highly conserved               tabolism glycated proteins support predicted function <br /> in evolution important physiology cells. One               Gcp glycoprotein-related chaperon protease. On basis <br /> such conserved role handling toxic glycation products               findings, present model compatible the <br /> that accumulate organisms harmful living cells.             complex Gcp-related phenotypes suggest conserved role for <br /> On basis assumption, examined possible role             Gcp activity. <br /> Gcp metabolism glycated proteins (Amadori products <br /> and advanced glycated end products [AGEs]).                                   RESULTS <br />     Amadori-modified proteins (AMPs) products non- <br /> enzymatic glycation formed reaction reducing sugars                 Construction physiology Gcp-depleted E. coli. In order to <br /> with primary amine-containing amino acids proteins. These                  investigate possible function Gcp metabolism gly- <br /> glycated proteins formed multistep chemical reaction                cated proteins E. coli, constructed strain Gcp <br /> known Amadori rearrangement develop                     levels reduced. In strain, chromosomal ygjD gene, <br /> into irreversible, highly stable compounds known AGEs (12&#226;&#8364;&#8220;                 coding Gcp, deleted complemented presence of <br /> 15).                                                                          low-copy-number plasmid carrying ygjD coding sequence <br />     AGEs participate pathophysiology human dis-             regulation arabinose-dependent pBAD promoter <br /> eases, including cardiovascular, neurological, liver diseases             (Fig. 1A) (see Materials Methods). The bacteria grew nor- <br /> (16&#226;&#8364;&#8220;19). As formation rate AGEs depends carbohy-                mally presence 0.2% arabinose presence of <br /> drate oxygen concentrations, conditions chronic hypergly-              glucose, repressed expression Gcp. When bacte- <br /> cemia oxidative stress, diabetes, result increased             rial cultures diluted glucose-containing medium, the <br /> accumulation AGEs consequent toxic effects (20).                       growth rate gradually declined, growth dramatically re- <br />     The formation AGEs shown slow process, taking              duced time period equivalent generations <br /> weeks months. On basis view, assumed               (Fig. 1B). The viability bacteria Gcp depletion was <br /> only long-lived proteins potential risk developing           monitored plating arabinose-containing LB plates. No re- <br /> modifications nonenzymatic glycation bacterial pro-               duction viability observed, demonstrating growth <br /> teins unlikely, relatively short generation time        inhibition Gcp-depleted cells restored activation of <br /> bacteria. However, 2001, Mironova et al. provided ev-            ygjD expression. Moreover, increase tur- <br /> idence nonenzymatic glycation presence AGEs                 bidity, increase cell number, suggesting Gcp <br /> E. coli, rapidly growing organism (21).                                     depletion results smaller bacteria. Indeed, using electron mi- <br />     Here, provide genetic biochemical evidence func-            croscopy, Gcp-depleted cells smaller <br /> tion Gcp metabolizing glycated proteins (Amadori                 spherical, resembling phenotype stationary-phase bac- <br /> products AGEs). We Gcp depletion                     teria (Fig. 1C). It noted previous reports dem- <br /> E. coli mutant, growth rate dramatically reduced cells accu-           onstrated Gcp depletion results altered cell morphology <br /> mulate highly active AMPs subsequently AGEs. We                   (11). Handford et al. observed Gcp depletion unusual <br /> show immunoprecipitation Gcp primarily binds glycated                 morphology, varied appearance consisting en- <br /> proteins characterize candidate substrates, including             larged cells. The fact saw small cells explained <br /> the elongation factor Tu pyruvate dehydrogenase (PDH)                 use different E. coli strains (MC4100 versus MG1655), <br />  <br />  <br /> 2             mbio.asm.org                                                                                  July/August 2010 Volume 1 Issue 3 e00195-10 <br />                                                                                                                Conserved Glycopeptidase Reduces Bacterial AGEing <br />  <br />  <br />  <br />  <br /> FIG 2 Accumulation glycated proteins AGEs following Gcp depletion. Lysates extracted wild-type Gcp-depleted cells further <br /> separated proteins low-molecular-mass compound fractions, described Materials Methods. (A) Proteins subjected SDS-PAGE, <br /> followed staining Coomassie brilliant blue total proteins (c) diol-specific silver stain glycated proteins (s). (B) Quantification glycated <br /> proteins use periodate-based colorimetric assay. (C) AGE-specific fluorescence (relative; excitation 370 nm emission 440 nm) following Gcp <br /> depletion total lysates fractions purified proteins low-molecular-weight (MW) compounds. The results represent independent <br /> experiments. <br />  <br />  <br />  <br />  <br /> but clear specific mechanism involved cell morphol-               increase accumulation AGEs relative levels ly- <br /> ogy determination affected Gcp-restrictive conditions.                   sates nondepleted cells (Fig. 2C). Lysates separated <br />     Accumulation AMPs AGEs Gcp-depleted cells. In                       before, proteins low-molecular-weight fractions were <br /> order determine levels glycated proteins wild-type               examined individually. Our data presented Fig. 2C demonstrate <br /> Gcp-depleted cells, lysates extracted separated                  protein fraction derived Gcp-depleted cells showed <br /> into soluble proteins low-molecular-weight compounds                       40% higher levels AGEs derived wild-type <br /> use size exclusion chromatography. Proteins separated                  cells, low-molecular-weight fraction, accumula- <br /> SDS-PAGE, gels stained Coomassie bril-                     tion AGEs 2.5-fold higher. These results demonstrate that <br /> liant blue (for total proteins) sugar-specific silver staining           Gcp depletion results accumulation AGEs of <br /> diols (for glycated proteins) (23). The results presented Fig. 2               AGEs accumulate low-molecular-weight compounds. <br /> indicate E. coli proteins showed specific staining pat-                   In order establish origin AGEs accumulated Gcp <br /> tern, demonstrating steady state glycated proteins                   depletion, examined potential separated fractions <br /> these conditions. Reduction Gcp levels did significantly                   wild-type Gcp-depleted cells form AGEs vitro. The <br /> change total protein <span id='am-165' about='xsp:pattern' typeof='owl:Thing'>pattern</span> resulted accumulation                  kinetics AGE formation measured using AGE-specific flu- <br /> of subset glycated proteins (Fig. 2A). Interestingly, accu-              orescence 30-min intervals incubation E. coli <br /> mulation glycated proteins observed mainly high-                    physiological temperature 37&#194;&#176;C. Under conditions, puri- <br /> molecular-mass region (~75 150 kDa). As spectrum pro-                   fied proteins extracted Gcp-depleted cells rapidly devel- <br /> teins size range fairly limited E. coli, possible        oped AGEs accumulation AGEs continued least <br /> these bands represent mobility shift lower-molecular-weight                  2 h (Fig. 3B). On hand, proteins extracted the <br /> proteins caused glycation. In order quantify                   wild-type strain showed reduced potential development of <br /> levels glycated proteins, used periodate-based colorimetric               AGEs, accumulation reached low plateau 30 min of <br /> assay specific quantification Amadori protein products                 incubation (Fig. 3A). A kinetic analysis AGE formation the <br /> (23). The results demonstrate Gcp depletion resulted                low-molecular-weight fractions indicated fractions derived <br /> 2.5-fold accumulation Amadori products purified pro-                    wild-type strains Gcp-depleted strains dem- <br /> tein fraction (Fig. 2B).                                                          onstrated low potential forming AGEs (Fig. 3E F). The <br />     Amadori products relatively unstable intermediates                 formation AGEs AMPs known oxidation depen- <br /> process protein glycation. However, molecules fur-                   dent. Indeed, control experiment potent frac- <br /> ther develop highly stable AGEs. The presence AGEs                    tion (a protein fraction Gcp-depleted cells), addition of <br /> was determined AGE-specific fluorescence, measurement                     2 mM dithiothreitol (DTT) resulted considerable decrease in <br /> of emission 440 nm excitation 370 nm. We observed                      rate AGE formation (Fig. 3C). Moreover, addition of <br /> that lysates extracted Gcp-depleted cells, 60%                50 mM aminoguanidine completely abolished AGE accumula- <br />  <br />  <br /> July/August 2010 Volume 1 Issue 3 e00195-10                                                                                                     mbio.asm.org 3 <br />  Katz et al. <br />  <br />  <br />  <br />                                                                                                          active site produce proteins that <br />                                                                                                          bind substrates lack catalytic ac- <br />                                                                                                          tivity. The Gcp-Trap allele con- <br />                                                                                                          structed changing essential <br />                                                                                                          metal-binding histidine residues po- <br />                                                                                                          sitions 111 115 alanines the <br />                                                                                                          predicted metallopeptidase domain <br />                                                                                                          (Fig. 4A). This modified allele was <br />                                                                                                          cloned pBR322 vector se- <br />                                                                                                          quence coding FLAG tag (27) at <br />                                                                                                          5= end gene. The plasmid car- <br />                                                                                                          rying Gcp-Trap allele intro- <br />                                                                                                          duced strain deleted the <br />                                                                                                          chromosomal Gcp-coding gene, and <br />                                                                                                          culture depleted Gcp. The al- <br />                                                                                                          lele coding Gcp-Trap strain did <br />                                                                                                          compensate depletion the <br />                                                                                                          chromosomal ygjD gene, demonstrat- <br />                                                                                                          ing importance metal- <br />                                                                                                          binding domain Gcp function. <br />                                                                                                          When growth ceased, cells dis- <br />                                                                                                          rupted inactive Gcp protein was <br />                                                                                                          precipitated using anti-FLAG aga- <br />                                                                                                          rose resin, described Materials and <br />                                                                                                          Methods. Total proteins, unbound and <br />                                                                                                          bound fractions, separated using <br />                                                                                                          SDS-PAGE stained either <br />                                                                                                          Coomassie brilliant blue diol-specific <br />                                                                                                          silver stain detection glycated <br />                                                                                                          proteins. The results shown Fig. 4B <br />                                                                                                          demonstrate Gcp-Trap strain <br />                                                                                                          accumulated glycated proteins, similar <br />                                                                                                          accumulation observed the <br />                                                                                                          nontagged Gcp-depleted strain, recon- <br />                                                                                                          firming importance metal- <br />                                                                                                          binding site enzymatic function. <br />                                                                                                          Moreover, proteins be <br />                                                                                                          bound Gcp-Trap predomi- <br /> FIG 3 Kinetic studies AGE formation proteins low-molecular-weight fractions. Samples <br />                                                                                                          nantly glycated polypeptides, their <br /> were kept 37&#194;&#176;C, AGEs monitored 30 min 2 h measurement AGE-specific           pattern fairly similar pattern <br /> fluorescence. The emission spectrum 400 nm 480 nm excitation 370 nm presented.        subset glycated proteins found <br /> Kinetics AGE formation protein fractions (A B) low-molecular-weight fractions (E       accumulate Gcp depletion <br /> and F) wild-type Gcp-depleted (gcp) strains, respectively. Effect 2 mM DTT (C) 100 mM       (compare Fig. 4B Fig. 2A). <br /> aminoguanidine (AG) (D) kinetics AGE formation protein fractions extracted <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Gcp-depleted bacteria.                                                                                       The proteins bound Gcp-Trap <br />                                                                                                          allele separated two-dimensional <br />                                                                                                          (2D) PAGE chromatography, 10 of <br />                                                                                                          most-prominent spots visu- <br /> tion, agent known prevent cross-linking alized gel identified using mass spectrometry. These <br /> essential AGE formation (24) (Fig. 3D).                                      proteins included cell division protein FtsZ, F1-&#226;?&#164; subunit <br />     Search potential Gcp substrates. The findings Gcp ATP synthase AtpD, elongation factor Tu (EF-Tu), <br /> depletion results accumulation AMPs subsequently essential proteins E. coli. Moreover, com- <br /> AGEs suggest direct involvement Gcp metabolism. ponents pyruvate dehydrogenase (PDH) complex, AceE, <br /> In order capture potential Gcp substrates, used coim- AceF, Lpd, Gcp-trapped proteins <br /> munoprecipitation assay. Since potential Gcp substrates accu- (Fig. 4C). These results compatible obtained the <br /> mulate primarily absence (Fig. 2A) enzyme- high-throughput experiments performed Butland et al. (28), <br /> substrate interaction generally rapid captured interaction Gcp EF-Tu AceE. <br /> coimmunoprecipitation, constructed &#226;&#8364;&#339;Gcp-Trap&#226;&#8364;? strain, It noted locations proteins <br /> an established method substrate capturing (25, 26). The differed theoretical location predicted molecular <br /> method, previously used identifying substrates known weight. FtsZ AtpD appear higher molecular weights, <br /> chaperon proteases, based idea mutations Lpd EF-Tu present multiple spots. These abnor- <br />  <br />  <br /> 4             mbio.asm.org                                                                             July/August 2010 Volume 1 Issue 3 e00195-10 <br />                                                                                                           Conserved Glycopeptidase Reduces Bacterial AGEing <br />  <br />  <br />  <br />                                                                                                              suggest Gcp involved pre- <br />                                                                                                              serving glycation-prone PDH com- <br />                                                                                                              plex. <br />                                                                                                                  In order determine re- <br />                                                                                                              duction active PDH physiologically <br />                                                                                                              significant, measured level its <br />                                                                                                              substrate, pyruvate. It previously <br />                                                                                                              shown pyruvate accumulates mu- <br />                                                                                                              tants inactive PDH (32), there- <br />                                                                                                              fore, increase pyruvate levels repre- <br />                                                                                                              sents inability PDH function. <br />                                                                                                              Indeed, depletion Gcp resulted ap- <br />                                                                                                              proximately 2-fold accumulation of <br />                                                                                                              pyruvate, compared level wild- <br />                                                                                                              type cells (Fig. 5D), demonstrating se- <br />                                                                                                              vere physiological effects reduction in <br />                                                                                                              active PDH complexes. <br />                                                                                                                  The severity Gcp depletion <br />                                                                                                              phenotype correlates level of <br />                                                                                                              protein glycation. Our data suggest that <br />                                                                                                              Gcp prevents accumulation glycated <br />                                                                                                              proteins. Therefore, assumed the <br />                                                                                                              level glycation influence sever- <br /> FIG 4 Characterization Gcp substrates use immunoprecipitation. Immunoprecipitation              ity effect Gcp depletion, which <br /> performed use strain containing inactive, metal-binding-deficient Gcp (Gcp-Trap). <br /> Bacteria grown described Fig. 1, Gcp depleted addition glucose. When           results dramatic growth rate reduction. <br /> growth stopped, cells lysed, proteins purified described Fig. 2 subjected    This assumption tested re- <br /> immunoprecipitation using anti-FLAG resin described Materials Methods. (A) Schematic            ducing intracellular levels glycated <br /> representation Gcp-Trap allele. The predicted zinc-binding histidines replaced alanines,      proteins, predicted lessen the <br /> and FLAG tag added 5= end. (B) SDS-PAGE total proteins fractions bound and <br />                                                                                                              effect Gcp depletion. Unfortunately, no <br /> unbound anti-FLAG resin, stained Coomassie brilliant blue total proteins diol- <br /> specific silver stain glycated proteins. (C) Separation Gcp-suspected substrates two-              known extracellular agent de- <br /> dimensional gel electrophoresis gels. Proteins identified mass spectrometry, protein names       crease glycation E. coli currently <br /> are indicated spot.                                                                             available. However, reported by <br />                                                                                                              Dimitrova et al. (33) intracellular gly- <br />                                                                                                              cation levels dramatically reduced <br /> malities suggest proteins underwent posttranslational bacteria grown minimal M9 medium. We therefore <br /> modification, probably involving glycation step.                           compared growth rates Gcp depletion strain rich LB <br />     Some potential substrates captured Gcp-Trap medium minimal M9 medium presence arabinose <br /> previously shown subjected carbonylation. PDH, EF-Tu, (for activation gcp expression) glucose (for repression). <br /> and ATP synthase subunits modified mammalian mi- Cultures diluted optical densities 600 nm (OD600s) of <br /> tochondria (29, 30), PDH EF-Tu 0.005 (106 cells/ml), turbidity monitored 10 h. The <br /> carbonylated E. coli (22).                                                      data presented Fig. 6A demonstrate bacteria were <br />     The finding FtsZ glycated Gcp-depleted cells inter- grown rich LB medium, repression gcp expression resulted in <br /> esting explain morphology changes observed occur dramatic reduction growth rate generations. <br /> under Gcp depletion, FtsZ known participate septation However, bacteria grown M9 minimal medium to <br /> and mediate cell wall biosynthesis interaction comparable cell concentrations, repression Gcp expression did not <br /> peptidoglycan-modifying enzymes (31).                                              affect growth rate time course. Moreover, Gcp <br />     Lower levels active PDH complex Gcp-depleted cells. repression conditions, depleted strains grew increased rate in <br /> The levels active PDH complex investigated <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-inferior minimal medium, leading increased turbidity <br /> compared wild-type Gcp-depleted cells. Active 8 h, contrast growth phenotype observed non- <br /> complexes partially purified using ultracentrifugation. repressing conditions. <br /> Under conditions, stable complexes precipitate,                          These results indicate depletion Gcp reduced effect <br /> to massive molecular weight. The PDH complex vi- growth rate conditions reduce glycation. However, <br /> sualized SDS-PAGE, results indicate possible Gcp synthesis shut minimal <br /> 60% decrease stable PDH complex lysates medium efficiently rich medium. This possibility ruled <br /> from Gcp-depleted cells, compared the experiment represented Fig. 6B, showing the <br /> wild type (Fig. 5A B). The precipitated complexes levels gcp transcripts, measured real-time PCR, actually <br /> similar specific activities, measured monitoring accu- lower minimal medium LB repression con- <br /> mulation NADH presence pyruvate coenzyme ditions. Thus, repression conditions (in presence of <br /> A 60 s (Fig. 5C). These results indicate smaller glucose), gcp transcript level cells grown LB medium was <br /> amount stable PDH complex Gcp-depleted cells reduced 35% normal expression level in <br />  <br />  <br /> July/August 2010 Volume 1 Issue 3 e00195-10                                                                                               mbio.asm.org 5 <br />  Katz et al. <br />  <br />  <br />  <br />                                                                                                     depletion severe rich <br />                                                                                                     medium. These results compatible <br />                                                                                                     hypothesis severe phe- <br />                                                                                                     notype Gcp depletion accu- <br />                                                                                                     mulation glycated proteins. <br />  <br />                                                                                                     DISCUSSION <br />                                                                                                      On basis data presented here, <br />                                                                                                      model (Fig. 7) suggests Gcp is <br />                                                                                                      directly involved AMP metabolism <br />                                                                                                      depletion results accumu- <br />                                                                                                      lation AMPs, serve poten- <br />                                                                                                      tial development AGEs. These <br />                                                                                                      AGEs probably degraded in <br />                                                                                                      cell, accumulate fraction <br />                                                                                                      low-molecular-weight compounds. <br />                                                                                                          The molecular mechanism Gcp <br />                                                                                                      function fully understood. One <br />                                                                                                      possibility Gcp involved Ama- <br />                                                                                                      dori product neutralization protein <br />                                                                                                      degradation. This hypothesis compati- <br />                                                                                                      ble current annotation Gcp as <br />                                                                                                      glycopeptidase, shown for <br />                                                                                                      M. haemolytica (3). Another possibility is <br />                                                                                                      Gcp plays role recycling pro- <br />                                                                                                      teins assisting deglycation using pu- <br />                                                                                                      tative chaperon activity (5). <br />                                                                                                          Several enzymes, known collectively <br />                                                                                                      &#226;&#8364;&#339;amadoriases,&#226;&#8364;? have <br />                                                                                                      deglycating activity, having ability to <br />                                                                                                      reverse nonenzymatic glycation cleav- <br />                                                                                                      ing bond amino acids and <br />                                                                                                      corresponding sugars (34, 35). One such <br />                                                                                                      enzyme, capable deglycating fructosel- <br />                                                                                                      ysine, E. coli (36). These en- <br />                                                                                                      zymes shown act low- <br />                                                                                                      molecular-weight peptides are <br /> FIG 5 Partial purification activity measurements pyruvate dehydrogenase (PDH) complex     unable work proteins. Gcp act <br /> from wild-type Gcp-depleted cells. Equal amounts proteins wild-type Gcp-depleted     introducing glycated proteins ama- <br /> cells centrifuged precipitation PDH complex. (A) SDS-PAGE PDH purification.   doriases allowing access the <br /> The complex components indicated. (B) Relative quantification PDH complex <br />                                                                                                      sugar moiety full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> proteins <br /> from independent experiments. (C) In vitro activity measurement PDH complex from <br /> wild-type Gcp-depleted cells. The graph represents accumulation reaction product      (using predicted chaperon activity) or <br /> NADH measuring specific absorption 339 nm. Specificity validated comparison    cleaving Amadori product way <br /> negative control lacking substrate, pyruvate.                                                    enable better access amadoria- <br />                                                                                                      ses glycated peptides. <br />                                                                                                          The mechanism leads cessa- <br />                                                                                                      tion cell growth Gcp depletion is <br /> wild-type cells, M9 medium, repression reduced gcp <br /> transcript levels 10% normal expression level. In likely reason essentiality Gcp organisms. <br /> view result, showing reduced gcp transcript levels phys- Our data suggest Gcp depletion results accumulation gly- <br /> iological effects Gcp depletion, determined levels AGEs cated protein products cause growth arrest. Support this <br /> the wild type mutant rich minimal media. The assumption obtained finding Gcp depletion greatly <br /> results shown Fig. 6C indicate that, indeed, lower levels reduced effect growth conditions reduce protein gly- <br /> glycation products minimal medium, reported (33). Gcp cation (Fig. 6). Accumulation AMPs AGEs potentially <br /> depletion resulted increase level AGEs minimal toxic, AGEs increase cellular oxidative stress (37). <br /> medium. However, level lower level AGEs However, addition antioxidants lipoic acid, <br /> the wild type grown rich medium, agreement lack ascorbic acid, glutathione growth medium did relieve <br /> phenotypic manifestation Gcp depletion minimal medium.                  Gcp depletion phenotypes. Another possibility glycated <br />     Taken together, results demonstrate proteins left untreated cause metabolic dysfunction. Damaged <br /> minimal medium Gcp depleted bacteria, effect enzymes potentially toxic dominant negative <br />  <br />  <br /> 6             mbio.asm.org                                                                            July/August 2010 Volume 1 Issue 3 e00195-10 <br />                                                                                                           Conserved Glycopeptidase Reduces Bacterial AGEing <br />  <br />  <br />  <br />  <br />                                                                   FIG 6 Gcp depletion growth phenotype rich LB medium minimal M9 medium. <br />                                                                   (A) The Gcp-depleted strain grown presence 0.2% arabinose 0.2% glucose <br />                                                                   activation repression Gcp expression, respectively, rich LB medium and <br />                                                                   minimal M9 medium. The cells diluted arabinose-containing medium into <br />                                                                   fresh medium supplemented arabinose glucose indicated concentration of <br />                                                                   106/ml, growth monitored 10 h measuring absorption 600 nm. (B) <br />                                                                   Relative expression levels Gcp-depleted strain grown different conditions as <br />                                                                   indicated. Relative mRNA levels measured using real-time PCR presented. The wild- <br />                                                                   type expression level represents chromosomal gcp expression levels wild-type cells <br />                                                                   grown LB medium. Wild-type expression levels similar media were <br />                                                                   significantly altered supplementation glucose arabinose. <br />  <br />  <br />  <br />  <br /> effect, competing substrates normal enzyme          The enzymatic mechanism responsible AGE degradation cur- <br /> leading inactivation complexes. This explanation appears prom-        rently unknown. Yet, eliminated possible participation the <br /> ising, potential substrates Gcp identified study   major cytosolic ATP-dependent proteases, triple mutant at <br /> were parts multimeric protein complexes.                                 Lon, ClpX, HslV accumulates low-molecular-weight AGEs <br />     In mammalian cells, toxicity AGEs facilitated        Gcp depleted. However, genetic Gcp is <br /> interaction specific receptors, soluble membrane            completely abolished, rule possibility that <br /> bound. Our bioinformatics search suggests E. coli genome           Gcp involved AGE degradation. <br /> does contain genes coding homologs receptors. It               The specific molecules lead protein glycation bacteria <br /> is possible direct interaction AGEs specific target    currently unknown. Since extracellular concentration re- <br /> proteins involved toxicity. Localization putative         ducing sugars did correlate intracellular glycation levels, <br /> complexes shed light function complicated            postulated intracellular metabolic pathways the <br /> the facts types AGEs availability       source agents (33). One potentially active agent shown to <br /> of antibodies limited.                                                   produced E. coli methylglyoxal, synthesized meth- <br />     The downstream metabolism AGEs in-            ylglyoxal synthase MgsA (38). However, deletion msgA did not <br /> vestigated. We high fraction AGEs present      relieve Gcp depletion phenotypes, demonstrating this <br /> low-molecular-weight fraction. These low-molecular-weight AGEs              agent dominant factor Gcp metabolic function (C. Katz <br /> probably result proteolytic breakdown modified proteins.          E. Z. Ron, unpublished data). <br />  <br />  <br /> July/August 2010 Volume 1 Issue 3 e00195-10                                                                                                mbio.asm.org 7 <br />  Katz et al. <br />  <br />  <br />  <br />                                                                                                              MATERIALS AND METHODS <br />                                                                                                               Bacterial strains plasmids. The strains <br />                                                                                                               plasmids used work described <br />                                                                                                               Table 1. <br />                                                                                                                   Growth conditions. Bacteria grown <br />                                                                                                               standard LB medium (Difco) minimal <br />                                                                                                               M9 medium 37&#194;&#176;C. The medium supple- <br />                                                                                                               mented arabinose glucose 0.2%. <br />                                                                                                               Control strains grown same <br />                                                                                                               conditions Gcp-depleted strains reduce <br />                                                                                                               possible background sugar concen- <br />                                                                                                               trations present growth media. Antibi- <br />                                                                                                               otics added required follow- <br />                                                                                                               ing concentrations: ampicillin (Amp), <br />                                                                                                               100 &#226;?&#174;g/ml; kanamycin (Km), 50 &#226;?&#174;g/ml; <br />                                                                                                               chloramphenicol (Cm), 25 &#226;?&#174;g/ml. <br />                                                                                                                   Genetic manipulations. The chromo- <br />                                                                                                               somal ygjD gene deleted, previously de- <br />                                                                                                               scribed (39), presence comple- <br />                                                                                                               mentation plasmid pACYC:pBAD:ygjD. The <br />                                                                                                               Gcp-Trap allele constructed steps. <br />                                                                                                               First, ygjD open reading frame (ORF) and <br />                                                                                                               promoter amplified using FLAG <br />                                                                                                               epitope tag-containing primers cloned <br />                                                                                                               EcoRI-digested pBR322. Second, con- <br /> FIG 7 Schematic model Amadori-modified proteins AGE metabolism E. coli. The model <br />                                                                                                               served metal-binding domain manipu- <br /> suggests Gcp participates metabolism glycated proteins reducing level Amadori <br /> products, decreasing formation AGEs. Once AGEs formed, broken             lated replacing conserved histidines <br /> smaller molecules.                                                                                            (positions 111 115) alanines use of <br />                                                                                                               PCR-based site-directed mutagenesis. <br />                                                                                                                   Preparation samples determination <br />                                                                                                               Amadori products AGEs. Lysates were <br />     The results presented demonstrate role Gcp pre-                                             extracted using Qproteome bacterial protein <br /> venting accumulation glycated proteins, ubiquitous preparatory kit (Qiagen) according manufacturer&#226;&#8364;&#8482;s recommendation. <br />                                                                                  The lysates separated purified proteins low-molecular- <br /> substrates living cells. Thus, metabolic function likely to <br />                                                                                  weight compounds use GE Healthcare HITRAP desalting columns, <br /> extend domains life tested Ar- equilibrated phosphate-buffered saline (PBS) H O AGE or <br />                                                                                                                                                 2 <br /> chaea Eukarya. These findings open avenue inves- Amadori product analysis, respectively, according manufacturer&#226;&#8364;&#8482;s                  rec- <br /> tigating glycated proteins bacteria, including metabolism, ommendation. These columns possess exclusion limit 5,000 g/mol of <br /> functions, possible toxicity. The ability study glycated pro- globular compound; thus, compounds lower molar masses here <br /> teins easy model organism, E. coli, amenable referred low-molecular-weight compounds. Nucleic acids digested <br /> to genetic manipulation, advance investigations glycation Qproteome extraction procedure present <br /> processes ageing human diseases.                                       low-molecular-weight fraction. Separations performed <br />  <br />  <br />  <br /> TABLE 1 Strains plasmids <br /> Plasmid strain            Feature(s)                                                 Description, genotype, reference(s) <br /> Plasmids <br />   pBAD24                     Plasmid containing arabinose-inducible promoter         44 <br />   pBAD:ygjD                  Ampr; arabinose-dependent ygjD expression                  pBAD24 ygjD ORF cloned EcoRI XbaI digestion <br />   pACYC:pBAD:ygjD            Cmr; low-copy-number plasmid enhanced control       pBAD:ygjD digested BspHI cloned pACYC184 digested <br />                                ygjD expression                                             enzyme <br />     pBR322                                                                              45, 46 <br />     pBR322                   Ampr; constitutive expression FLAG-tagged inactive      5= FLAG-tagged ygjD endogenous promoter cloned an <br />       Gcp-Trap                ygjD immunoprecipitation                                 EcoRI-digested PBR322 plasmid; metal-binding histidine <br />                                                                                            residues positions 111 115 modified alanines <br /> Bacterial strains <br />   K-12 MG1655                Wild type <br />   K-12 MG1655                Wild type carrying plasmid cloned inducible gcp     K-12 MG1655 pACYC:pBAD:ygjD <br />      pACYC:pBAD:ygjD <br />   Gcp depletion              Used Gcp depletion; deletion complemented    K-12 MG1655 &#226;&#338;&#172;ygjD pACYC:pBAD:ygjD <br />      strain                    inducible ygjD gene plasmid <br />   Gcp-Trap strain            Used Gcp immunoprecipitation; Gcp          K-12 MG1655 &#226;&#338;&#172;ygjD pACYC:pBAD:ygjD pBR322 Gcp-Trap <br />                                depletion strain additional plasmid coding for <br />                                peptidase-deficient Gcp protein <br />  <br />  <br />  <br /> 8             mbio.asm.org                                                                                     July/August 2010 Volume 1 Issue 3 e00195-10 <br />                                                                                                               Conserved Glycopeptidase Reduces Bacterial AGEing <br />  <br />  <br />  <br /> and evaluated using Amersham AKTA Prime Plus fast-performance liq-           gene 6000 (Corbett) using standard cycling parameters. Relative gene <br /> uid chromatography (FPLC) system.                                                expression data analysis carried &#226;&#338;&#172;CT method described <br />     Analysis glycated proteins SDS-PAGE gels. Proteins ex-            manufacturer&#226;&#8364;&#8482;s protocol (Corbett). <br /> tracted purified described separated using 10% SDS- <br /> PAGE. The gels stained using Coomassie brilliant blue staining for <br />                                                                                  ACKNOWLEDGMENTS <br /> total protein analysis using diol-specific silver staining described   We thank Dvora Biran Aviram Rasouly productive discussions <br /> reference 23 glycated-protein analysis. The Amadori product colori-          suggestions. We grateful Sagi Huja Doerte Becher (Greif- <br /> metric assay preformed described reference 40.                         swald University) helping proteomic experiments. <br />     Determination AGEs. AGEs quantified using AGE-specific                   C.K. supported fellowship Alfried Krupp von <br /> fluorescence scanning emissions ranging 400 nm 480 nm            Bohlen Halbach Foundation graduate school, &#226;&#8364;&#339;Functional Genomics <br /> excitation 370 nm 37&#194;&#176;C HORIBA scientific FluoroLog-3 spec-          Pathogens.&#226;&#8364;? <br /> trofluorometer. The data presented represent full-range spec-         REFERENCES <br /> trum 440-nm emission peak. <br />                                                                                   1. Galperin, M. Y., E. V. Koonin 2004. &#226;&#8364;&#732;Conserved hypothetical&#226;&#8364;&#8482; <br />     Gcp-Trap coimmunoprecipitation assay. The Gcp-Trap strain was <br />                                                                                      proteins: prioritization targets experimental study. Nucleic Acids <br /> grown overnight presence arabinose diluted 1/200 fresh            Res. 32:5452&#226;&#8364;&#8220;5463. <br /> LB supplemented 0.2% glucose. The culture kept logarithmic            2. Galperin, M. Y. 2008. Social bacteria asocial eukaryotes. Environ. <br /> ODs (0.2 0.8) dilutions growth ceased. Twenty millili-           Microbiol. 10:281&#226;&#8364;&#8220;288. <br /> ters culture OD600 0.4 harvested subjected total           3. Abdullah, K. M., R. Y. Lo, A. Mellors. 1991. Cloning, nucleotide <br /> soluble protein purification described sample preparation. A              sequence, expression Pasteurella haemolytica A1 glycoprotease <br /> desalting stage used buffer replacement, purified proteins               gene. J. Bacteriol. 173:5597&#226;&#8364;&#8220;5603. <br /> were eluted Tris-buffered saline (TBS) buffer, pH 7.4, followed con-        4. Angata, T., A. Varki. 2002. Chemical diversity sialic acids and <br />                                                                                      related alpha-keto acids: evolutionary perspective. Chem. Rev. 102: <br /> centration 0.5 ml using 10K Amicon Ultra (Millipore). The sample was <br />                                                                                      439 &#226;&#8364;&#8220; 469. <br /> incubated overnight 4&#194;&#176;C 50 &#226;?&#174;l anti-FLAG M2-agarose (Sigma)             5. Aravind, L., E. V. Koonin. 1999. Gleaning non-trivial structural, <br /> prewashed TBS. Samples washed times 1 ml TBS buffer              functional evolutionary information proteins iterative data- <br /> and eluted using 200 &#226;?&#174;l TBS supplemented 100 &#226;?&#174;M 3&#226;&#171;&#187; FLAG pep-                    <span id='am-164' about='xsp:base' typeof='owl:Thing'>base</span> searches. J. Mol. Biol. 287:1023&#226;&#8364;&#8220;1040. <br /> tide (Sigma). Total proteins, unbound bound fractions, sub-              6. Mao, D. Y., D. Neculai, M. Downey, S. Orlicky, Y. Z. Haffani, D. F. <br /> jected SDS-PAGE stained Coomassie brilliant blue               Ceccarelli, J. S. Ho, R. K. Szilard, W. Zhang, C. S. Ho, L. Wan, C. Fares, <br /> with diol-specific silver stain.                                                   S. Rumpel, I. Kurinov, C. H. Arrowsmith, D. Durocher, F. Sicheri. <br />     2D analysis protein identification. Fifty micrograms proteins             2008. Atomic structure KEOPS complex: ancient protein kinase- <br />                                                                                      containing molecular machine. Mol. Cell 32:259 &#226;&#8364;&#8220;275. <br /> obtained Gcp-Trap coimmunoprecipitation assay separated using              7. Downey, M., R. Houlsworth, L. Maringele, A. Rollie, M. Brehme, S. <br /> two-dimensional gel electrophoresis 13-cm IPG strips, pH 4 7 (GE             Galicia, S. Guillard, M. Partington, M. K. Zubko, N. J. Krogan, A. Emili, <br /> Healthcare), identified using mass spectrometry, described ref-            J. F. Greenblatt, L. Harrington, D. Lydall, D. Durocher. 2006. A <br /> erence 41.                                                                           genome-wide screen identifies evolutionarily conserved KEOPS com- <br />     Pyruvate dehydrogenase purification activity measurement.                    plex telomere regulator. Cell 124:1155&#226;&#8364;&#8220;1168. <br /> Ten milliliters wild-type Gcp-depleted cultures OD600 0.4         8. Hecker, A., N. Leulliot, D. Gadelle, M. Graille, A. Justome, P. Dorlet, C. <br /> was harvested, pellets resuspended 1 ml buffer (50 mM                Brochier, S. Quevillon-Cheruel, E. Le Cam, H. van Tilbeurgh, P. <br /> K-phosphate, pH 7.5, 5 mM EDTA, 2 mM DTT) disrupted using                    Forterre. 2007. An archaeal orthologue universal protein Kae1 an <br />                                                                                      iron metalloprotein exhibits atypical DNA-binding properties and <br /> sonication. Lysates centrifuged 30 min 40,000 &#226;&#171;&#187; g mem-               apurinic-endonuclease activity vitro. Nucleic Acids Res. 35:6042&#226;&#8364;&#8220; 6051. <br /> brane precipitation, equal protein amounts membrane-free               9. Oberto, J., N. Breuil, A. Hecker, F. Farina, C. Brochier-Armanet, E. <br /> supernatants centrifuged 1.5 h 150,000 &#226;&#171;&#187; g PDH                 Culetto, P. Forterre. 2009. Qri7/OSGEPL, mitochondrial version <br /> precipitation. The PDH complexes analyzed SDS-PAGE de-                    universal Kae1/YgjD protein, essential mitochondrial genome <br /> scribed above. PDH activity measured described reference 42.               maintenance. Nucleic Acids Res. 37:5343&#226;&#8364;&#8220;5352. <br /> Control reactions pyruvate used negative control           10. Hecker, A., M. Graille, E. Madec, D. Gadelle, E. Le Cam, H. van <br /> background assessment.                                                               Tilbergh, P. Forterre. 2009. The universal Kae1 protein the <br />                                                                                      associated Bud32 kinase (PRPK), mysterious protein couple probably <br />     Pyruvate measurements. Pyruvate extraction based hot- <br />                                                                                      essential genome maintenance Archaea Eukarya. Biochem. Soc. <br /> water method described reference 43. Ten-milliliter samples                 Trans. 37:29 &#226;&#8364;&#8220;35. <br /> OD600 0.4 collected centrifuged, pellets washed         11. Handford, J. I., B. Ize, G. Buchanan, G. P. Butland, J. Greenblatt, A. <br /> with distilled water. The pellets resuspended 200 &#226;?&#174;l boiled H2O            Emili, T. Palmer. 2009. Conserved network proteins essential for <br /> and incubated 95&#194;&#176;C Thermomixer (Eppendorf) strong agita-                bacterial viability. J. Bacteriol. 191:4732&#226;&#8364;&#8220; 4749. <br /> tion 5 min, samples centrifuged 30 min &#226;&#171;&#186;10&#194;&#176;C,        12. Baynes, J. W., N. G. Watkins, C. I. Fisher, C. J. Hull, J. S. Patrick, M. U. <br /> supernatants immediately used pyruvate measurement using                  Ahmed, J. A. Dunn, S. R. Thorpe. 1989. The Amadori product on <br /> BioVision pyruvate assay kit according manufacturer&#226;&#8364;&#8482;s fluorescence            protein: structure reactions. Prog. Clin. Biol. Res. 304:43&#226;&#8364;&#8220; 67. <br />                                                                                  13. Horvat, S., A. Jakas. 2004. Peptide amino acid glycation: new <br /> protocol. Fluorescence measured using Synergy HT multidetection <br />                                                                                      insights Maillard reaction. J. Pept. Sci. 10:119 &#226;&#8364;&#8220;137. <br /> reader (excitation, 530 nm; emission, 590 nm).                                   14. Njoroge, F. G., V. M. Monnier. 1989. The chemistry Maillard <br />     RNA extraction real-time PCR experiment. RNA stabilized                  reaction physiological conditions: review. Prog. Clin. Biol. Res. <br /> using RNAprotect reagent (Qiagen), followed extraction using                   304:85&#226;&#8364;&#8220;107. <br /> RNeasy minikit (Qiagen) according manufacturer&#226;&#8364;&#8482;s protocols. Re-           15. Singh, R., A. Barden, T. Mori, L. Beilin. 2001. Advanced glycation <br /> sidual DNA digested using RQ1 RNase-free DNase (Promega),                    end-products: review. Diabetologia 44:129 &#226;&#8364;&#8220;146. <br /> the samples purified using RNeasy cleanup procedure.                    16. Hyogo, H., S. Yamagishi. 2008. Advanced glycation end products <br /> Five nanograms total RNA reverse transcribed using                    (AGEs) involvement liver disease. Curr. Pharm. Des. 14: <br />                                                                                      969 &#226;&#8364;&#8220;972. <br /> random hexamers (Amersham) ImPromII reverse transcriptase (Pro- <br />                                                                                  17. Jandeleit-Dahm, K., M. E. Cooper. 2008. The role AGEs in <br /> mega). Real-time PCRs determination gcp transcript levels                cardiovascular disease. Curr. Pharm. Des. 14:979 &#226;&#8364;&#8220;986. <br /> performed using 500 nM primers GCAAATACCATTCGTGACAA                          18. Takeuchi, M., S. Yamagishi. 2008. Possible involvement advanced <br /> TGCACTTAATCATCAGCGTAT 10-&#226;?&#174;l volume SYBR green                              glycation end-products (AGEs) pathogenesis Alzheimer&#226;&#8364;&#8482;s disease. <br /> PCR master mix (Applied Biosystems). Reactions run Rotor-                  Curr. Pharm. Des. 14:973&#226;&#8364;&#8220;978. <br />  <br />  <br />  <br /> July/August 2010 Volume 1 Issue 3 e00195-10                                                                                                     mbio.asm.org 9 <br />  Katz et al. <br />  <br />  <br />  <br /> 19. Takeuchi, M., S. Yamagishi. 2009. Involvement toxic AGEs                        intracellular metabolite concentrations metabolic flux analysis by <br />     (TAGE) pathogenesis diabetic vascular complications Alz-                 13C-labeling experiments. J. Biotechnol. 122:254 &#226;&#8364;&#8220;266. <br />     heimer&#226;&#8364;&#8482;s disease. J. Alzheimers Dis. 16:845&#226;&#8364;&#8220; 858.                                33.   Dimitrova, R., R. Mironova, I. Ivanov. 2004. Glycation proteins <br /> 20. Goh, S. Y., M. E. Cooper. 2008. Clinical review: role advanced              Escherichia coli: effect <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> broth ingredients glycation. Bio- <br />     glycation end products progression complications diabetes. J.                technol. Biotechnol. Equip. 18:99 &#226;&#8364;&#8220;103. <br />     Clin. Endocrinol. Metab. 93:1143&#226;&#8364;&#8220;1152.                                           34.   Sakai, Y., N. Yoshida, A. Isogai, Y. Tani, N. Kato. 1995. Purification <br /> 21. Mironova, R., T. Niwa, H. Hayashi, R. Dimitrova, I. Ivanov. 2001.                  properties fructosyl lysine oxidase Fusarium oxysporum <br />     Evidence non-enzymatic glycosylation Escherichia coli. Mol. Micro-              S-1F4. Biosci. Biotechnol. Biochem. 59:487&#226;&#8364;&#8220; 491. <br />     biol. 39:1061&#226;&#8364;&#8220;1068.                                                              35.   Takahashi, M., M. Pischetsrieder, V. M. Monnier. 1997. Isolation, <br /> 22. Fredriksson, A., M. Ballesteros, S. Dukan, T. Nystrom. 2005.                       purification, characterization amadoriase isoenzymes (fructosyl <br />     Defense protein carbonylation DnaK/DnaJ proteases                amine-oxygen oxidoreductase EC 1.5.3) Aspergillus sp. J. Biol. <br />     heat shock regulon. J. Bacteriol. 187:4207&#226;&#8364;&#8220; 4213.                                      Chem. 272:3437&#226;&#8364;&#8220;3443. <br /> 23. Dubray, G., G. Bezard. 1982. A highly sensitive periodic acid-silver         36.   Wiame, E., G. Delpierre, F. Collard, E. Van Schaftingen. 2002. <br />     stain 1,2-diol groups glycoproteins polysaccharides poly-                Identification pathway utilization Amadori product <br />     acrylamide gels. Anal. Biochem. 119:325&#226;&#8364;&#8220;329.                                           fructoselysine Escherichia coli. J. Biol. Chem. 277:42523&#226;&#8364;&#8220; 42529. <br /> 24. Lehman, T. D., B. J. Ortwerth. 2001. Inhibitors advanced glyca-           37.   Loske, C., A. Neumann, A. M. Cunningham, K. Nichol, R. Schinzel, P. <br />     tion end product-associated protein cross-linking. Biochim. Biophys.                   Riederer, G. Munch. 1998. Cytotoxicity advanced glycation end- <br />     Acta 1535:110 &#226;&#8364;&#8220;119.                                                                    products mediated oxidative stress. J. Neural Transm. 105: <br /> 25. Kerner, M. J., D. J. Naylor, Y. Ishihama, T. Maier, H. C. Chang, A. P.                 1005&#226;&#8364;&#8220;1015. <br />     Stines, C. Georgopoulos, D. Frishman, M. Hayer-Hartl, M. Mann,               38.   Booth, I. R., G. P. Ferguson, S. Miller, C. Li, B. Gunasekera, S. <br />     F. U. Hartl. 2005. Proteome-wide analysis chaperonin-dependent pro- <br />                                                                                            Kinghorn. 2003. Bacterial production methylglyoxal: <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span> strategy <br />     tein folding Escherichia coli. Cell 122:209 &#226;&#8364;&#8220;220. <br />                                                                                            death misadventure? Biochem. Soc. Trans. 31:1406 &#226;&#8364;&#8220;1408. <br /> 26. Flynn, J. M., S. B. Neher, Y. I. Kim, R. T. Sauer, T. A. Baker. 2003. <br />                                                                                      39.   Datsenko, K. A., B. L. Wanner. 2000. One-step inactivation of <br />     Proteomic discovery cellular substrates ClpXP protease reveals <br />                                                                                            chromosomal genes Escherichia coli K-12 using PCR products. Proc. <br />     classes ClpX-recognition signals. Mol. Cell 11:671&#226;&#8364;&#8220; 683. <br />                                                                                            Natl. Acad. Sci. U. S. A. 97:6640 &#226;&#8364;&#8220; 6645. <br /> 27. Thomas, P. H., S. P. Kathryn, L. P. Virginia, T. L. Randell, J. M. Carl, <br />     P. C. Douglas, L. U. David, P. J. Conlon. 1988. A short polypeptide          40.   Ahmed, N., A. J. Furth. 1991. A microassay protein glycation <br />     marker sequence useful recombinant protein identification puri-                based periodate method. Anal. Biochem. 192:109 &#226;&#8364;&#8220;111. <br />     fication. Nat. Biotechnol. 6:1204 &#226;&#8364;&#8220;1210.                                         41.   Rosen, R., K. Buttner, R. Schmid, M. Hecker, E. Z. Ron. 2001. <br /> 28. Butland, G., J. M. Peregrin-Alvarez, J. Li, W. Yang, X. Yang, V. Cana-                 Stress-induced proteins Agrobacterium tumefaciens. FEMS Microbiol. <br />     dien, A. Starostine, D. Richards, B. Beattie, N. Krogan, M. Davey, J.                  Ecol. 35:277&#226;&#8364;&#8220;285. <br />     Parkinson, J. Greenblatt, A. Emili. 2005. Interaction network con-           42.   Bisswanger, H. 1981. Substrate specificity pyruvate dehydrogenase <br />     taining conserved essential protein complexes Escherichia coli. Na-             complex Escherichia coli. J. Biol. Chem. 256:815&#226;&#8364;&#8220; 822. <br />     ture 433:531&#226;&#8364;&#8220;537.                                                                43.   Hiller, J., E. Franco-Lara, D. W <br /> </body></html>