<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                             JCB: REPORT <br />  <br />  <br />  <br />  <br />                               A genome-wide RNAi screen reveals multiple <br />                               regulators caspase activation <br />                               Caroline H. Yi, Dodzie K. Sogah, Michael Boyce, Alexei Degterev, Dana E. Christofferson, Junying Yuan <br />                               Department Cell Biology, Harvard Medical School, Boston, MA 02115 <br />  <br />  <br />  <br />  <br />                               A <br />                                         poptosis evolutionally conserved cellular                       promoting cell death. Further characterization uncovers <br />                                         suicide mechanism activated re-                        10 genes in&#239;&#172;&#8218;uence caspase activation re- <br />                                         sponse variety stressful stimuli. Increasing                  moval Drosophila inhibitor apoptosis 1. This set <br />                               evidence suggests apoptotic regulation relies                           includes Drosophila initiator caspase Dronc and, sur- <br />                               specialized cell death signaling pathways inte-                        prisingly, metabolic regulators, candidate tumor <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               grates diverse signals additional regulatory circuits,                     suppressor, Charlatan, N-acetyltransferase, ARD1. <br />                               including cellular homeostasis. We present                           Importantly, genes functional con- <br />                               genome-wide <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>RNA interference</span> screen systematically                           servation regulating apoptosis mammalian cells. <br />                               identify regulators apoptosis induced DNA damage                          Our data suggest previously unappreciated fundamental <br />                               Drosophila melanogaster cells. We identify 47 double-                        connection various cellular processes caspase- <br />                               stranded RNA target functionally diverse set                             dependent cell death. <br />                               genes, including known function in <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               Apoptosis evolutionally conserved cell death pathway                      differentiation (Tsuda et al., 2006), ARD1, N-acetyl- <br />                               regulates development tissue homeostasis. Caspases,                  transferase involved cell fate specification (Whiteway and <br />                               family cysteine proteases, play critical role mediating                 Szostak, 1985), regulate caspase activation. Importantly, show <br />                               execution apoptosis. Although CED-3 sole cas-                     certain fly genes functionally conserved modifiers <br />                               pase required programmed cell death Caenorhabditis                       caspase activation mammalian system. Our screen <br />                               elegans, multiple caspases mediate apoptotic cell death flies                implicates Chn ARD1 molecular link cellular <br />                               mammals. In systems, activation upstream                       differentiation apoptosis. <br />                               initiator caspases response proapoptotic signals leads <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                               activation downstream executioner caspases. Although                     Results discussion <br />                               core apoptotic pathway studied extensively, many <br />                               aspects signaling networks control cellular de-                 To determine feasibility RNAi approach identifying <br />                               cision undergo apoptosis remain unknown. Complex bio-                        apoptotic regulators, tested knockdown Dcp-1, <br />                               logical processes dissected successfully using                        downstream effector caspase functionally similar mamma- <br />                               genome-wide RNAi screens Drosophila melanogaster cells.                      lian caspase-3, protects DNA damage&#226;&#8364;&#8220;induced apoptosis <br />                               In study, isolation 10 genes, including                 Drosophila embryonic hemocyte Kc cells (Song et al., 2000). <br />                               apical caspase Dronc, required caspase                    We used topoisomerase II inhibitor, doxorubicin (dox), in- <br />                               activation response DNA damage. Surprisingly, dis-                     duce dose-dependent cell death (Fig. S1 A, available http:// <br />                               covered Charlatan (Chn), regulator neuronal cell                      www.jcb.org/cgi/content/full/jcb.200708090/DC1) be <br />                                                                                                               suppressed z-VAD.fmk treatment (Fig. S1 B). As expected, <br />                                                                                                               dcp-1 RNAi partially protected cells apoptosis induced by <br />                               C.H. Yi D.K. Sogah contributed equally paper.                       dox (Fig. S1 C), consistent previous observa- <br />                               Correspondence Junying Yuan: jyuan@hms.harvard.edu                           tions (Xu et al., 2006). We conclude dox induces caspase- <br />                               Abbreviations used paper: Chn, Charlatan; DIAP1, Drosophila inhibitor <br />                               apoptosis 1; dox, doxorubicin; dsRNA, double-stranded RNA; NRSF, neuron- <br />                                                                                                               dependent cell death Kc cells suppressed a <br />                               restrictive silencer factor; REST, RE1-silencing transcription factor.          specific double-stranded RNA (dsRNA) and, therefore, represents <br />                               The online version article contains supplemental material.              suitable identifying modulators apoptosis. <br />  <br />                               &#194;&#169; The Rockefeller University Press $30.00 <br />                               The Journal Cell Biology, Vol. 179, No. 4, November 19, 2007 619&#226;&#8364;&#8220;626 <br />                               http://www.jcb.org/cgi/doi/10.1083/jcb.200708090 <br />                                                                                                                                                                             JCB   619 <br />        Figure 1. Identi&#239;&#172;?cation con&#239;&#172;?rmation dsRNAs provide protection DNA damage&#226;&#8364;&#8220;induced apoptosis. (A) dsRNAs screened dupli- <br />       cate plates protection dox treatment. The threshold hit set z score 2 plates (above blue dashed line). The plot <br />       represents z score analysis plates enriched genes shown Table I (red diamonds). (B) dsRNAs identi&#239;&#172;?ed primary <br />       screen individually tested, provided statistically signi&#239;&#172;?cant protection dox independent experiments shown <br />       (P &lt; 0.05; t test). The graph represents pool 10 sample plates, error bars represent SD dsRNAs tested triplicate. (C) This graph depicts <br />       functional distribution genes shown B according information provided FlyBase (version FB2006_01). <br />  <br />  <br />  <br />  <br /> 620   JCB &#226;&#8364;&#162; VOLUME 179 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2007 <br />  A diverse set dsRNAs protects &#239;&#172;&#8218;y cells <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> dox-induced apoptosis <br /> To identify dsRNAs inhibit DNA damage&#226;&#8364;&#8220;induced apopto- <br /> sis Kc cells, performed high-throughput screen using an <br /> established genome-wide Drosophila RNAi library targets <br /> 19,470 genes (Boutros et al., 2004). 81 dsRNAs resulted a <br /> z score &gt;2, threshold defining hit pri- <br /> mary screen (Fig. 1 A). To eliminate dsRNAs directly en- <br /> hanced cellular ATP levels, effect dsRNAs ATP levels <br /> was measured rescreen. We verified 62 dsRNAs spe- <br /> cifically protected cells dox-induced apoptosis (Fig. 1 B). <br /> To minimize off-target effects, examined dsRNA <br /> with 19-nucleotide sequence identity off-target <br /> gene (Echeverri et al., 2006; Ma et al., 2006) testing alterna- <br /> tive dsRNAs distinct original targeting sequence <br /> for protection cell death induced dox treatment <br /> (Fig. S2 A, available http://www.jcb.org/cgi/content/full/ <br /> jcb.200708090/DC1) caspase suppression induced by <br /> Drosophila inhibitor apoptosis 1 (diap1) RNAi treatment as <br /> described Fig. 3 (Fig. S2 B). Any dsRNA given gene <br /> failing provide significant protection (P &lt; 0.05) of <br /> these assays eliminated, resulting final set 47 genes <br /> (Fig. 1 B Table S1). <br />        The identification known regulators cell death <br /> validates ability screen uncover genes required for <br /> promoting apoptosis. Silencing Dronc provided maximal <br /> protection dox treatment (Fig. 1 B), consistent <br /> with role main checkpoint apoptosis fly <br /> (Chew et al., 2004; Daish et al., 2004). In addition, knockdown <br /> of ecdysone-induced protein Eip63F-1 provided fourth <br /> strongest protection DNA damage. The increased ex- <br /> pression Eip63F detected premetamorphic salivary          Figure 2. Classi&#239;&#172;?cation genes function upstream executioner <br /> gland Drosophila larvae, immediately ecdysone-           caspases. (A) Kc cells transfected dsRNAs indicated and <br />                                                                        treated dox induce caspase activation. This graph representative <br /> mediated induction massive autophagic cell death (Gorski            independent experiments. RLU, relative light <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. (B) dsRNAs were <br /> et al., 2003; Rusten et al., 2004). Lastly, screen isolated Jra,   tested suppression caspase activity induced dox treatment. <br /> the Drosophila orthologue known proapoptotic mammalian            Selected dsRNAs provided statistically signi&#239;&#172;?cant inhibition caspase <br />                                                                        activity independent experiments shown (*, P &lt; 0.05; <br /> transcriptional factor, c-Jun, mediator DNA damage&#226;&#8364;&#8220;            t test). This graph shows pool sample plates. The error bars represent <br /> induced apoptosis (Verheij et al., 1996).                              SD dsRNAs tested triplicate. <br />        Approximately 85% genes identified RNAi <br /> screen characterized genes known function contain <br /> well-conserved functional domains, regulate wide range         Identi&#239;&#172;?cation genes involved caspase- <br /> of cellular processes, including signaling, metabolism, tran-      dependent cell death <br /> scription, remaining 15% genes known        Next, classified genes specifically involved in <br /> functional domains (Fig. 1 C). Altogether, RNAi screen im-         caspase-dependent cell death. We observed substantial <br /> plicates cell death genes (three), signaling molecules (13), met-      induction caspase activity 8 h dox treatment, preceding <br /> abolic regulators (10), metabolite transport factors (four), genes     detectable cell death (Fig. S1, A D). Any RNAi suppressing <br /> involved ER/Golgi trafficking (four), chromatin/transcription       activity implicates target gene early regulation cas- <br /> regulators (two), RNA-processing factors (two), structural cyto-   pase activation. In addition dcp-1 RNAi, knockdown dronc <br /> skeletal proteins (two), genes unknown function (seven)      jra significantly (P &lt; 0.05) suppressed caspase-3/7&#226;&#8364;&#8220;like <br /> mediating DNA damage&#226;&#8364;&#8220;induced apoptosis (Table S1). Strikingly,         activity presence dox, negative control, <br /> 20% genes directly involved cellular metabolic           RNAi calpain A, calcium-dependent cysteine prote- <br /> processes (Fig. 1 C), supporting earlier proposal cel-     ase, did affect pathway (Fig. 2 A). We expanded this <br /> lular metabolic state critically influences threshold in-      analysis genes identified initial RNAi screen <br /> duction apoptosis (Hammerman et al., 2004). To investigate          discovered 20 dsRNAs suppressed caspase activation <br /> where genes operate apoptotic pathway, con-            induced DNA damage (Table S1). Interestingly, shown in <br /> ducted specific enzymatic epistatic assays fly mam-         Fig. 2 B, 12 genes epistatic diap1, <br /> malian cells.                                                          discussed section. <br />  <br />  <br />                                                                 IDENTIFYING REGULATORS OF CASPASE ACTIVATION &#226;&#8364;&#162; YI ET AL.                                 621 <br />        Figure 3. A set genes epistatic Diap1 criti- <br />       cal caspase activation. (A) Kc cells transfected <br />       dsRNAs indicated followed transfection <br />       RNAi diap1. This graph representative <br />       independent experiments. (B) dsRNAs were <br />       tested protection cell death induced the <br />       loss Diap1. Selected dsRNAs provided statisti- <br />       cally signi&#239;&#172;?cant protection diap1 RNAi at <br />       independent experiments shown. This graph <br />       represents pool sample plates. (C) dsRNAs <br />       tested suppression caspase activation in- <br />       duced loss Diap1. Note 10/12 dsRNAs <br />       showed statistically signi&#239;&#172;?cant suppression caspase <br />       activation compared RNAi diap1 only. <br />       This graph representative independent ex- <br />       periments. The error bars represent SD dsRNAs <br />       tested triplicate. *, P &lt; 0.05; t test. <br />  <br />  <br />  <br />  <br />       Identi&#239;&#172;?cation genes epistatic DIAP1                              induced diap1 RNAi, suggesting genes required <br />       Next, performed diap1 epistatic analysis catego-         apoptosis induced multiple stimuli (Table I). To confirm <br />       rize genes. DIAP1, fly orthologue mammalian             genes necessary activation caspases, we <br />       inhibitors apoptosis proteins, direct inhibitor caspases,   determined dsRNAs suppress spontaneous <br />       deficiency DIAP1 leads rapid caspase activation          caspase activity induced diap1 RNAi. We observed maximal <br />       apoptosis vivo (Wang et al., 1999). Thus, apoptosis induced         induction caspase activity diap1 RNAi 24 h, this <br />       loss DIAP1 presents alternative apoptotic assay in-       effect completely suppressed dsRNA dcp-1 <br />       dependent DNA damage. Silencing genes regulate acti-        (Fig. S1 E). Importantly, ablating 10/12 dsRNAs resulted the <br />       vation core apoptotic machinery provide protection          significant (P &lt; 0.05) suppression caspase activity compared <br />       apoptosis induced DNA damage loss           diap1 RNAi (Fig. 3 C). <br />       DIAP1. RNAi dcp-1 partially suppressed cell death induced             In addition dronc RNAi, dsRNAs targeting chn and <br />       depletion DIAP1 Kc cells (Fig. 3 A). Also, dronc RNAi     dARD1 provided strongest suppression spontaneous cas- <br />       potently protected cells apoptosis induced deficiency       pase activity. Consistent observation RNAi against <br />       DIAP1 reported previously (Meier et al., 2000; Chai et al.,      chn protects DNA damage&#226;&#8364;&#8220;induced cell death, mam- <br />       2003). Altogether, 32 genes confirmed primary          malian orthologue neuron-restrictive silencer factor (NRSF)/ <br />       screen provided significant (P &lt; 0.05) protection cell         RE1-silencing transcription factor (REST) recently identi- <br />       death induced silencing DIAP1 (Table S1 Fig. 3 B).       fied candidate tumor suppressor epithelial cells (Westbrook <br />             Interestingly, 12 dsRNAs suppressed caspase-3/7&#226;&#8364;&#8220;like             et al., 2005). Previous work indicates Chn NRSF/REST <br />       activity dox treatment protected cell death          function transcriptional repressor neuronal-specific genes <br />  <br />  <br /> 622   JCB &#226;&#8364;&#162; VOLUME 179 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2007 <br />  Table I. A summary genes identi&#239;&#172;?ed RNAi screen modulate caspase activation <br /> Gene ID                 Gene                Known mammalian                       Gene ontology                     Functional class <br />                                                     orthologue <br />  <br /> CG8090                Dronc (Nc)                     Caspase                    Apical caspase                          Cell death <br /> CG5410                dMiro                          Miro                       Mitochondrial GTPase                    Signaling <br /> CG11989               dARD1                          hARD1                      Protein acetyltransferase               Signaling <br /> CG14740               NA                             Cs                         Citrate synthase                        Metabolism <br /> CG31674               NA                             GRHPR                      Glyoxylate reductase                    Metabolism <br /> CG12170               NA                             OXSM                       Fatty acid biosynthesis                 Metabolism <br /> CG10725               NA                             NA                         Polysaccharide metabolism               Metabolism <br /> CG9258                nrv1                           Atp1b1                     Na+/K+ATPase                            Transporters <br /> CG8029                NA                             NA                         Proton-coupled ATP synthase             Transporters <br /> CG7438                myo31DF                        Myo1D                      Type 1 myosin                           Transporters <br /> CG11798               Charlatan (chn)                NRSF/REST                  Transcriptional repressor               Transcription <br /> CG6712                NA                             RPF1                       Ribosomal RNA processing                RNA processing <br /> NA, applicable. <br />  <br />  <br />  <br /> (Chong et al., 1995; Schoenherr Anderson, 1995; Tsuda                In cells transfected nontargeting control siRNA, dox treat- <br /> et al., 2006), suggesting cellular differentiation render       ment resulted typical apoptotic morphology, including cell <br /> cells refractory caspase activation apoptosis. Also,           rounding membrane blebbing (Fig. 4 B). In direct contrast, <br /> identified metabolic genes, CG31674, CG14740,                cells transfected siRNAs hARD1 maintained a <br /> CG12170, involved general regulation cas-          normal healthy morphology continued proliferate in <br /> pase activation. Recently, Nutt et al. (2005) demonstrated          presence dox. <br /> NADPH produced pentose phosphate pathway regulates                       To examine protection provided siRNAs <br /> the activation caspase-2 <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-deprived Xenopus laevis          targeting hARD1 plk3 associated suppression of <br /> oocytes. Together results, observations provide           caspase activation, measured caspase activity cells <br /> further evidence intimate link regulation          treated dox. RNAi plk3 provided partial suppres- <br /> metabolism induction apoptosis.                                   sion caspase activity, supporting protection pheno- <br />                                                                          type observed Fig. 4 A. Interestingly, depletion REST <br /> Evolutionary conservation novel                                   resulted suppression caspase activity presence <br /> regulators apoptosis                                                  dox protection cell death not <br /> To explore significance findings, examined         statistically significant (Fig. 4 C). Consistent viability <br /> whether silencing mammalian orthologues fly genes             assay, complete suppression caspase-3/7 activity observed <br /> identified RNAi screen confers protection dox-          cells transfected hARD1 siRNA (Fig. 4 C). These results <br /> induced cell death mammalian cells. We selected set mam-         indicate hARD1 required caspase-dependent cell <br /> malian orthologues believed nonredundant. The list        death induced DNA damage. Furthermore, observed that <br /> includes orthologues dMiro, functions Rho-like         siRNAs targeting hARD1 individually capable of <br /> GTPase; dARD1, functions N-acetyltransferase;                providing robust protection cell death, strongly suggest- <br /> CG12170, functions fatty acid synthase; Chn,        ing siRNAs target hARD1 specifically (Fig. 4 D). <br /> functions transcriptional repressor (RHOT1, hARD1, OXSM,                   Because silencing hARD1 dramatically suppressed <br /> and REST, respectively; FlyBase). In addition, tested Plk3,           activation downstream caspases, examined whether <br /> a mammalian orthologue Polo, dsRNA targeting polo                  activation upstream caspases response dox treatment <br /> potently protected dox treatment (Fig. 1).                       perturbed. Remarkably, hARD1 RNAi inhibited cleav- <br />        We assessed ability siRNAs targeting gene             age caspase-2 -9 cells treated dox, cas- <br /> interest protect DNA damage HeLa cells. As posi-         pase cleavage readily detected control cells (Fig. 4 E). <br /> tive control, cells transfected siRNAs targeting Bax           Thus, propose hARD1 regulates signal transduction <br /> or Bak, central regulators mammalian cell death (Wei              pathway apical apoptotic machinery DNA damage <br /> et al., 2001; Zong et al., 2001). Indeed, silencing Bax Bak        response itself activation upstream caspases. <br /> resulted significant protection (P &lt; 0.05) dox-                      Consistent results caspase-3/7 assay (Fig. 4 C), <br /> induced cell death (Fig. 4 A). We observed plk3 RNAi pro-           silencing hARD1 completely inhibited appearance of <br /> vided partial protection dox treatment, consistent      activated caspase-3 induced dox (Fig. 4 E). We used this <br /> with previous studies implicating Plk3 stress-induced apop-           assay hARD1 complementation experiment demonstrate <br /> tosis (Xie et al., 2001a,b; Bahassi et al., 2002). Interestingly,        proapoptotic role hARD1 response DNA damage. <br /> the knockdown hARD1 dramatically enhanced cell <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span>               We used new siRNA pool targeting 5&#226;&#8364;&#178; untranslated region <br /> in presence dox levels similar Bak. This pro-       hARD1 (5&#226;&#8364;&#178;si), inhibited caspase-3 cleavage induced <br /> tective effect evident morphological level.              dox treatment (Fig. 4 F). Furthermore, observed caspase-3 <br />  <br />  <br />                                                                  IDENTIFYING REGULATORS OF CASPASE ACTIVATION &#226;&#8364;&#162; YI ET AL.                          623 <br />        Figure 4. hARD1 required caspase activation response DNA damage mammalian cells. (A) HeLa cells transfected pool four <br />       siRNAs indicated followed treatment 1.25 &#206;&#188;g/ml dox 20 h. This graph representative independent experiments. (B) HeLa cells <br />       transfected siRNAs indicated treated 1.25 &#206;&#188;g/ml dox followed 12-h incubation. Cells visualized bright-&#239;&#172;?eld micros- <br />       copy, representative images samples tested triplicate shown. (C) HeLa cells transfected siRNAs indicated treated with <br />       5 &#206;&#188;g/ml dox 8 h. This graph representative independent experiments. (D) HeLa cells transfected pool siRNAs individual <br />       siRNAs targeting hARD1 indicated treated 1.25 &#206;&#188;g/ml dox 20 h. This graph representative independent experiments. <br />       (E) HeLa cells transfected siRNAs indicated treated 5 &#206;&#188;g/ml dox 8 h. Note RNAi hARD1 completely inhibited cleav- <br />       age caspase-2, caspase-9, caspase-3 compared nontargeting siRNA (arrows). This experiment representative independent experiments. <br />       (F) HeLa cells transfected siRNAs indicated treated 5 &#206;&#188;g/ml dox 8 h. Asterisks represent nonspeci&#239;&#172;?c bands recognized the <br />       indicated antibody. This experiment representative independent experiments. (A, C, D) *, P &lt; 0.05; t test. Error bars represent SD. <br />  <br />  <br />  <br />  <br /> 624   JCB &#226;&#8364;&#162; VOLUME 179 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2007 <br />  cleavage reconstituted hARD1 knockdown cells. Because                 Materials methods <br /> six siRNAs hARD1 provided strong protection <br /> against DNA damage&#226;&#8364;&#8220;induced apoptosis complementation                 Tissue culture <br />                                                                          Drosophila Kc167 cells maintained Schneider&#226;&#8364;&#8482;s Drosophila medium <br /> of hARD1-sensitized cells caspase activation, conclude             10% FBS presence penicillin streptomycin (Invitrogen). <br /> that functional role ARD1 dox-induced apoptosis            HeLa cells maintained DME 10% newborn calf serum the <br /> evolutionally conserved Drosophila mammals.                      presence penicillin streptomycin (Invitrogen) passaged for <br />                                                                          6 wk. <br />        In contrast results, Arnesen et al. (2006) reported that <br /> hARD1 necessary maintain <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>. One possible ex-           RNAi screening <br /> planation discrepancy attributed inherent dif-    Drosophila Kc cells (105/well 384-well plates) transfected with <br />                                                                          prealiquoted dsRNAs (0.5 &#206;&#188;g/well) followed 48-h incubation al- <br /> ferences siRNAs used study used          low protein turnover. Subsequently, cells treated 5 &#206;&#188;g/ml dox <br /> Arnesen et al. We observed two siRNAs used            (Sigma-Aldrich) additional 48-h incubation. Cell viability deter- <br /> the Arnesen et al. (2006) study resulted decrease cell sur-      mined measuring cellular ATP levels (CellTiter-Glo luminescent cell via- <br />                                                                          bility assay; Promega). Positive hits screen de&#239;&#172;?ned dsRNAs <br /> vival absence stress signal, siRNAs        increased cellular ATP levels units SD the <br /> tested negative effect cell survival (Fig. S2 D).       mean level plate, z score two, duplicate. <br />        In summary, used unbiased RNAi screening platform <br />                                                                          Rescreen <br /> in Drosophila cells identify genes involved promoting DNA          Con&#239;&#172;?rmation hits performed using newly synthesized dsRNAs. <br /> damage&#226;&#8364;&#8220;induced apoptosis. We isolated 47 dsRNAs sup-                cDNA ampli&#239;&#172;?ed &#239;&#172;&#8218;y genomic DNA using primers optimally de- <br /> press cell death induced dox. These genes encode known            signed RNAi (sequences provided N. Ramadan, Drosophila RNAi <br />                                                                          Screening Center, Harvard Medical School, Boston, MA). dsRNA synthesis <br /> apoptotic regulators Dronc, Drosophila orthologue         conducted according manufacturer&#226;&#8364;&#8482;s recommended protocol <br /> the known proapoptotic transcriptional factor c-Jun, ecdy-        (MEGAscript; Ambion). Kc cells (2 &#195;&#8212; 105/well 96-well plates) were <br /> sone-regulated protein, Eip63F-1, validating primary         transfected dsRNAs (1 &#206;&#188;g/well) followed treatment dox. Each <br />                                                                          dsRNA tested triplicate independent experiments. <br /> screen. Furthermore, study implicates large class metabolic     Cell viability determined ratio ATP levels cells treated <br /> genes previously suspected role modu-         given dsRNA presence dox control cells treated with <br /> lating caspase activation apoptosis, genes involved          dsRNA only. Statistically signi&#239;&#172;?cant protection (P &lt; 0.05) dox- <br />                                                                          induced cell death determined comparing cell viability cells <br /> in fatty acid biosynthesis (CG11798), amino acid/carbohydrate            treated RNAi presence dox cell viability cells <br /> metabolism (CG31674), citrate metabolism (CG14740), complex              treated dox alone. Fold protection dox determined the <br /> carbohydrate metabolism (CG10725), ribosome biosynthesis             ratio mean cell viability cells transfected dsRNA the <br />                                                                          presence dox cells treated dox. <br /> (CG6712). These results support earlier proposal (Hammerman <br /> et al., 2004; Nutt et al., 2005) cellular metabolic status      Caspase activation assay <br /> regulates threshold activation apoptosis plays       Kc cells transfected 96-well plates 62 dsRNAs were <br /> a critical role decision cell live die.                con&#239;&#172;?rmed primary screen followed treatment dox 8 h to <br />                                                                          induce caspase activation. Each dsRNA tested triplicate at <br />        Of particular identification ARD1. We pre-     independent experiments. Caspase-3/7&#226;&#8364;&#8220;like activity quanti- <br /> sent evidence RNAi ARD1 provides protection                 &#239;&#172;?ed using luciferin-labeled DEVD peptide substrate (Caspase-Glo 3/7 <br /> against cell death leads suppression caspase activation    assay; Promega). Fold caspase-3/7&#226;&#8364;&#8220;like activity determined the <br />                                                                          ratio mean caspase-3/7&#226;&#8364;&#8220;like activity cells transfected with <br /> induced DNA damage fly cells HeLa cells. Furthermore,          dsRNA presence dox cells treated dox only. <br /> deficiency dARD1 renders fly cells resistant spontane- <br /> ous caspase activity cell death associated loss Diap1.       DIAP1 epistasis studies <br />                                                                          Kc cells transfected 96-well plates individual dsRNAs target- <br /> Importantly, provide substantial evidence hARD1 re-           ing 62 genes con&#239;&#172;?rmed initial screen. After 24-h incubation, <br /> quired caspase activation presence DNA damage           cells transfected 1 &#206;&#188;g diap1 dsRNA additional 48-h <br /> mammalian cells. Cleavage initiator executioner caspases          incubation cell viability quanti&#239;&#172;?ed (as described Rescreen <br />                                                                          section). Fold protection diap1 RNAi determined ratio <br /> are suppressed hARD1 RNAi cells treated dox, suggesting          mean cell viability cells transfected indicated <br /> that hARD1 functions upstream caspase activation,             dsRNA diap1 RNAi mean cell viability cells transfected with <br /> and complementation hARD1 knockdown cells restores                diap1 RNAi only. For caspase activation assay, Kc cells trans- <br />                                                                          fected dsRNA indicated followed transfection dsRNA <br /> caspase-3 cleavage. These data indicate ARD1 necessary           diap1 additional 24-h incubation. <br /> for DNA damage&#226;&#8364;&#8220;induced apoptosis flies mammals. <br />        ARD1 functions complex N-acetyltransferase           Mammalian RNAi studies <br />                                                                          Low-passage HeLa cells transiently transfected (5 &#195;&#8212; 105/well in <br /> catalyze acetylation N&#206;&#177;-terminal residue newly             96-well plates) pool siRNAs unless indicated (50 nM; <br /> synthesized polypeptides implicated regula-          predesigned ON-TARGETplus siRNAs; Dharmacon) using HiPerFect trans- <br /> tion heterochromatin, DNA repair, maintenance              fection reagent (QIAGEN). After 48-h incubation, cells treated with <br />                                                                          dox (concentrations incubation times noted &#239;&#172;?gures). siRNAs <br /> genomic stability yeast (Mullen et al., 1989; Aparicio et al.,        tested triplicate independent experiment. Cell viability and <br /> 1991; Ouspenski et al., 1999; Wilson, 2002; Bilton et al., 2006).        caspase-3/7 activity quanti&#239;&#172;?ed described KC cells. Images were <br /> These studies suggest ARD1 involved regulating            obtained using bright-&#239;&#172;?eld microscopy. <br />                                                                                 For detection caspase cleavage, HeLa cells transfected with <br /> an early step response DNA damage. We anticipate              siRNAs (1.5 &#195;&#8212; 105/well six-well plates) followed treatment dox. <br /> future studies focus determining ARD1 func-              Cells lysed directly SDS sample buffer subjected SDS-PAGE <br /> tions similar processes mammals. The diversity genes            analysis. Caspases detected immunoblotting using following <br />                                                                          antibodies: caspase-9 (R&amp;D Systems), caspase-2 (Alexis), caspase-3 (Cell <br /> identified screen illustrates complex cellular integra-       Signaling Technology), cleaved caspase-3 (Cell Signaling Technology). <br /> tion survival death signals multiple pathways.            hARD1 detected using rabbit polyclonal antibody (provided by <br />  <br />  <br />                                                                   IDENTIFYING REGULATORS OF CASPASE ACTIVATION &#226;&#8364;&#162; YI ET AL.                               625 <br />        M.C. Brahimi-Horn, University Nice, Centre National la Recherche                  Gorski, S.M., S. Chittaranjan, E.D. Pleasance, J.D. Freeman, C.L. Anderson, R.J. <br />       Scienti&#239;&#172;?que, Nice, France).                                                                    Varhol, S.M. Coughlin, S.D. Zuyderduyn, S.J. Jones, M.A. Marra. <br />              For hARD1 reconstitution experiments, HeLa cells transfected                       2003. A SAGE approach discovery genes involved autophagic <br />                                                                                                      cell death. Curr. Biol. 13:358&#226;&#8364;&#8220;363. <br />       pool siRNAs targeting 5&#226;&#8364;&#178; untranslated region hARD1 <br />       (sequences CUGACUGCGCCUUCACGAUUU GCUGACUGCGCC-                                      Hammerman, P.S., C.J. Fox, C.B. Thompson. 2004. Beginnings signal- <br />       UUCACGAUU; Dharmacon) six-well plates followed 24-h incuba-                            transduction pathway bioenergetic control cell survival. Trends <br />                                                                                                      Biochem. Sci. 29:586&#226;&#8364;&#8220;592. <br />       tion. This followed transient transfection C-terminal&#226;&#8364;&#8220;tagged <br />       hARD1-myc/his using TransIT-LT1 transfection reagent (Mirus) addi-               Ma, Y., A. Creanga, L. Lum, P.A. Beachy. 2006. Prevalence off-target <br />       tional 24-h incubation. After dox treatment, cells lysed analyzed                     effects Drosophila RNA interference screens. Nature. 443:359&#226;&#8364;&#8220;363. <br />       Western blotting.                                                                    Meier, P., J. Silke, S.J. Leevers, G.I. Evan. 2000. The Drosophila caspase <br />                                                                                                      DRONC regulated DIAP1. EMBO J. 19:598&#226;&#8364;&#8220;611. <br />       Microscopy                                                                              Mullen, J.R., P.S. Kayne, R.P. Moerschell, S. Tsunasawa, M. Gribskov, M. <br />       Images HeLa cells obtained using microscope (Eclipse TE300;                          Colavito-Shepanski, M. Grunstein, F. Sherman, R. Sternglanz. 1989. <br />       Nikon) 10&#195;&#8212; Ph L objective lens camera (ORCA-ER; Hamamatsu)                        Identification characterization genes mutants N-terminal <br />                                                                                                      acetyltransferase yeast. EMBO J. 8:2067&#226;&#8364;&#8220;2075. <br />       room temperature. Openlab 3.1.7 acquisition software (Improvision) was <br />       used analyze images.                                                             Nutt, L.K., S.S. Margolis, M. Jensen, C.E. Herman, W.G. Dunphy, J.C. Rathmell, <br />                                                                                                      S. Kornbluth. 2005. Metabolic regulation oocyte cell death through <br />                                                                                                      CaMKII-mediated phosphorylation caspase-2. Cell. 123:89&#226;&#8364;&#8220;103. <br />       Online supplemental material <br />       Fig. S1 shows characterization dox-induced cell death controls             Ouspenski, I.I., S.J. Elledge, B.R. Brinkley. 1999. New yeast genes impor- <br />                                                                                                      tant chromosome integrity segregation identified dosage ef- <br />       RNAi screen. Fig. S2 presents semiquantitative analysis dsRNAs vali- <br />                                                                                                      fects genome stability. Nucleic Acids Res. 27:3001&#226;&#8364;&#8220;3008. <br />       date RNAi library. Table S1 provides comprehensive list genes <br />       identi&#239;&#172;?ed primary screen. Online supplemental material available              Rusten, T.E., K. Lindmo, G. Juhasz, M. Sass, P.O. Seglen, A. Brech, H. <br />                                                                                                      Stenmark. 2004. Programmed autophagy Drosophila fat body is <br />       http://www.jcb.org/cgi/content/full/jcb.200708090/DC1.                                      induced ecdysone regulation PI3K pathway. Dev. Cell. <br />                                                                                                      7:179&#226;&#8364;&#8220;192. <br />       We thank Norbert Perrimon members Drosophila RNAi Screening <br />       Center advice. Dr. M.C. Brahimi-Horn provided hARD1 antibody hARD1-             Schoenherr, C.J., D.J. Anderson. 1995. The neuron-restrictive silencer fac- <br />       myc/his expression vector. We grateful Andrew Steele Vanessa Runner                 tor (NRSF): coordinate repressor multiple neuron-specific genes. <br />                                                                                                      Science. 267:1360&#226;&#8364;&#8220;1363. <br />       critical reading manuscript Yuan laboratory comments. <br />                This work supported, part, Innovator&#226;&#8364;&#8482;s Award             Song, Z., B. Guan, A. Bergman, D.W. Nicholson, N.A. Thornberry, E.P. <br />       U.S. Army Breast Cancer Research Program (grant DAMD17-02-1-0403),                           Peterson, H. Steller. 2000. Biochemical genetic interactions be- <br />                                                                                                      tween Drosophila caspases proapoptotic genes rpr, hid, grim. <br />       grant National Institute Aging (R37-012859) J. Yuan, graduate                 Mol. Cell. Biol. 20:2907&#226;&#8364;&#8220;2914. <br />       research fellowship National Science Foundation D.K. Sogah, and <br />       Kirchstein National Research Service Award National Institute             Tsuda, L., M. Kaido, Y.M. Lim, K. Kato, T. Aigaki, S. Hayashi. 2006. <br />                                                                                                      An NRSF/REST-like repressor downstream Ebi/SMRTER/Su(H) regu- <br />       Neurological Disorders Stroke (grant 1F31NS057872-01) C.H. Yi.                          lates eye development Drosophila. EMBO J. 25:3191&#226;&#8364;&#8220;3202. <br />                                                                                               Verheij, M., R. Bose, X.H. Lin, B. Yao, W.D. Jarvis, S. Grant, M.J. Birrer, E. <br />       Submitted: 27 August 2007                                                                      Szabo, L.I. Zon, J.M. Kyriakis, et al. 1996. Requirement ceramide- <br />       Accepted: 15 October 2007                                                                      initiated SAPK/JNK signalling stress-induced apoptosis. Nature. <br />                                                                                                      380:75&#226;&#8364;&#8220;79. <br />                                                                                               Wang, S.L., C.J. Hawkins, S.J. Yoo, H.A. Muller, B.A. Hay. 1999. The <br />       References                                                                                     Drosophila caspase inhibitor DIAP1 essential cell survival is <br />                                                                                                      negatively regulated HID. Cell. 98:453&#226;&#8364;&#8220;463. <br />       Aparicio, O.M., B.L. Billington, D.E. Gottschling. 1991. Modifiers posi-         Wei, M.C., W.X. Zong, E.H. Cheng, T. Lindsten, V. Panoutsakopoulou, A.J. <br />               tion effect shared telomeric silent mating-type loci                Ross, K.A. Roth, G.R. MacGregor, C.B. Thompson, S.J. Korsmeyer. <br />               S. cerevisiae. Cell. 66:1279&#226;&#8364;&#8220;1287.                                                     2001. Proapoptotic BAX BAK: requisite gateway mitochondrial <br />       Arnesen, T., D. Gromyko, F. Pendino, A. Ryningen, J.E. Varhaug, J.R.                       dysfunction death. Science. 292:727&#226;&#8364;&#8220;730. <br />               Lillehaug. 2006. Induction apoptosis human cells RNAi-                 Westbrook, T.F., E.S. Martin, M.R. Schlabach, Y. Leng, A.C. Liang, B. Feng, J.J. <br />               mediated knockdown hARD1 NATH, components protein                        Zhao, T.M. Roberts, G. Mandel, G.J. Hannon, et al. 2005. A genetic screen <br />               N-alpha-acetyltransferase complex. Oncogene. 25:4350&#226;&#8364;&#8220;4360.                             candidate tumor suppressors identifies REST. Cell. 121:837&#226;&#8364;&#8220;848. <br />       Bahassi, El M., C.W. Conn, D.L. Myer, R.F. Hennigan, C.H. McGowan, Y.                   Whiteway, M., J.W. Szostak. 1985. The ARD1 gene yeast functions in <br />               Sanchez, P.J. Stambrook. 2002. Mammalian Polo-like kinase 3                        switch mitotic cell cycle alternative developmental <br />               (Plk3) multifunctional protein involved stress response pathways.              pathways. Cell. 43:483&#226;&#8364;&#8220;492. <br />               Oncogene. 21:6633&#226;&#8364;&#8220;40.                                                           Wilson, T.E. 2002. A genomics-based screen yeast mutants altered <br />       Bilton, R., E. Trottier, J. Pouyssegur, M.C. Brahimi-Horn. 2006. ARDent                    recombination/end-joining repair ratio. Genetics. 162:677&#226;&#8364;&#8220;688. <br />               acetylation deacetylation hypoxia signalling. Trends Cell          Xie, S., Q. Wang, H. Wu, J. Cogswell, L. Lu, M. Jhanwar-Uniyal, W. <br />               Biol. 16:616&#226;&#8364;&#8220;621.                                                                       <br /> </body></html>