<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genetic Basis Growth Adaptation Escherichia coli <br /> after Deletion pgi, Major Metabolic Gene <br /> Pep Charusanti1, Tom M. Conrad1, Eric M. Knight1&#194;&#164;, Karthik Venkataraman1, Nicole L. Fong1, Bin Xie2, <br /> Yuan Gao2,3, Bernhard &#195;&#732;. Palsson1* <br /> 1 Department Bioengineering, University California San Diego, La Jolla, California, United States America, 2 Center Study Biological Complexity, Virginia <br /> Commonwealth University, Richmond, Virginia, United States America, 3 Department Computer Science, Virginia Commonwealth University, Richmond, Virginia, <br /> United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Bacterial <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>survival</span> requires adaptation different environmental perturbations exposure antibiotics, changes in <br />      temperature oxygen levels, DNA damage, alternative <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> sources. During adaptation, bacteria develop <br />      beneficial mutations confer increased fitness new environment. Adaptation loss major non-essential <br />      gene product cripples growth, however, studied whole-genome level. We investigated ability <br />      Escherichia coli K-12 MG1655 overcome loss phosphoglucose isomerase (pgi) adaptively evolving ten <br />      replicates E. coli lacking pgi 50 days glucose M9 minimal medium characterizing endpoint clones through <br />      whole-genome re-sequencing <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> profiling. We 1) growth rates endpoint clones <br />      increased approximately 3-fold 50-day period; 2) mutations arose adaptation, frequently in <br />      NADH/NADPH transhydrogenases udhA pntAB stress-associated sigma factor rpoS; 3) despite similar <br />      growth rates, distinct endpoint phenotypes developed defined different rates acetate formate <br />      secretion. These results demonstrate E. coli adapt loss major metabolic gene product handful <br />      mutations adaptation result multiple, alternative phenotypes. <br />  <br />   Citation: Charusanti P, Conrad TM, Knight EM, Venkataraman K, Fong NL, et al. (2010) Genetic Basis Growth Adaptation Escherichia coli Deletion pgi, <br />   Major Metabolic Gene. PLoS Genet 6(11): e1001186. doi:10.1371/journal.pgen.1001186 <br />   Editor: Josep Casadesu&#194;&#180;s, Universidad Sevilla, Spain <br />   Received May 11, 2010; Accepted September 29, 2010; Published November 4, 2010 <br />   Copyright: &#195;&#376; 2010 Charusanti et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: We acknowledge funding NIH R01GM062791. The funders role study design, data collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: bpalsson@ucsd.edu <br />   &#194;&#164; Current address: Center Systems Biology, University Iceland, Reykjavik, Iceland <br />  <br />  <br /> Introduction                                                                             resistance [12]. Bacteria develop antibiotic resistance through <br />                                                                                          <span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>mutation</span> target enzyme horizontal gene transfer <br />    Recent advances DNA sequencing technology enable                                   (HGT) fit drug-sensitive counterparts, but <br /> bacterial genomes fully sequenced resolution high                           wild-type fitness levels restored acquire <br /> enough differences relative reference sequence.                         additional mutations compensate lower fitness. <br /> These developments make possible study, whole-genome                          Crucially, mechanism does alter structure of <br /> level, genetic basis bacteria adapt different                    different networks bacterium suite genes <br /> perturbations. For example, E. coli adaptively evolved achieve                        remains. In contrast, does change network structure: the <br /> optimal growth glycerol                               organism gains new genes assimilated networks <br /> mutations endpoint clones re-sequenced compared                            controlling transcriptional regulation, metabolism, transcrip- <br /> <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> parental wild-type strain [1]. Allelic replacement introduc-                      tion/translation. <br /> ing discovered mutations parental strain used                         Here address important related question: does a <br /> to phenotypic causality mutation [1]. In                        bacterium adjust complete loss major gene product, not <br /> example, long-term adaptive evolution study E. coli revealed                        just mutations gene? The answer question would <br /> that genomic evolution did decrease time expected.                           provide insight plasticity bacterial genomes and <br /> Instead, genomic evolution remained nearly constant 20,000                          networks emerge proteins encode. We <br /> generations nearly mutations appeared                           selected gene pgi study plays major role central <br /> beneficial [2,3]. Other adaptive evolution resequencing-based                        metabolism transcribing enzyme catalyzes second <br /> studies examined genome level E. coli adapts                          step glycolysis (Figure 1A). In E. coli, loss pgi significantly alters <br /> growth carbon sources glycerol [4&#226;&#8364;&#8220;6],                               structure metabolic network disabling use upper <br /> Myxococcus xanthus transitions cooperative behavior                              glycolysis, situation cripples growth (growth rate ,20% of <br /> cheating [7], different pathogens develop                               wild-type levels glucose minimal media [13]) does kill <br /> antibiotic resistance [8&#226;&#8364;&#8220;11].                                                            organism. E. coli Dpgi mutants remain viable glycolytic <br />    Another topic received intense investigation                      flux rerouted pentose phosphate pathway (PPP) <br /> compensatory mutations, especially regard antibiotic                             [14&#226;&#8364;&#8220;15]; however, introduces redox imbalance problem <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                         1                        November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                      E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />   Author Summary                                                             Adaptive evolution produced multiple, different <br />                                                                              endpoint clones defined rates metabolic <br />   Bacteria constantly adapt different environ-                  by-product secretion <br />   mental challenges, adapt lose <br />                                                                                 All strains evolved similar endpoint growth rates and <br />   key gene product? We addressed question using <br />   Escherichia coli lacking pgi, major metabolic gene                       glucose uptake rates but, interestingly, different metabolic functional <br />   involved sugar utilization, serially passing replicates              states. Specifically, strains stratified three <br />   lacking pgi gene 50 days resequencing                      distinct groups based rates acetate formate secretion: <br />   entire genome endpoint clones isolated                   secrete acetate formate, secrete <br />   replicates. We repeatedly mutations rpoS,                   acetate only, secrete acetate formate <br />   gene active stationary phase                    (Figure 2C). Those secrete acetate rate one- <br />   regulates expression fifty genes                       tenth lower acetate secretion rate unevolved wild- <br />   exponential phase growth, udhA pntAB,                        type E. coli (12.868.87 mmol acetate/gram dry weight/hour) and <br />   genes involved maintaining redox balance cell.                   E. coli strains adapted grow glucose M9 50 days <br />   We multiple distinct endpoint phenotypes;                   (5.6160.24 mmol acetate/gram dry weight/hour) (Figure 2C). <br />   replicates stratified different groups                 The parental Dpgi clone did secrete acetate. <br />   adaptation based rates acetate and <br />   formate secretion. These results support view                 Two mutations detected course <br />   metabolic network E. coli robust adjust loss <br />   major metabolic gene alternative ways.                             adaptation evolved Dpgi strains were <br />                                                                              sequenced <br />                                                                                 The mutations detected 50-day adaptive evolution <br /> since excess NADPH produced which, turn, perturbs                    period sequenced strains summarized Table 2. <br /> significant portion metabolic network [16]. Thus, pgi                 Mutations rpoS common, appearing nine <br /> represents good candidate gene study mechanisms                      endpoints. The rpoS mutation pgi_gluc7 encodes stop codon at <br /> compensation gene loss.                                                   position; rpoS mutations pgi_gluc4, pgi_gluc5 and <br />    To study E. coli overcome limitations imposed            pgi_gluc6 likely result truncated forms protein; rpoS <br /> loss pgi consequent flux imbalances, serially passed           mutation pgi_gluc3 SNP results G279V change in <br />                                                                              protein; rpoS mutation pgi_gluc2 in-frame nine <br /> E. coli Dpgi replicates 50 days glucose minimal media and <br />                                                                              <span id='am-633' about='xsp:base' typeof='owl:Thing'>base</span> pair duplication. <br /> characterized evolved, endpoint clones phenotypic level <br />                                                                                 Mutations soluble transhydrogenase udhA [18] (six <br /> through measurements growth rates rates substrate <br />                                                                              mutations strains) membrane-bound transhydrogenase <br /> uptake secretion. We assessed genotypic changes nine <br />                                                                              subunits pntA pntB (two mutations each) common <br /> of strains whole-genome resequencing using Nimble- <br />                                                                              (Table 2). Loss pgi directly perturbs catalyze the <br /> gen tiling arrays (all strains) Illumina technology (three <br />                                                                              oxidation reduction NAD(P)/NAD(P)H (Figure 1B). <br /> strains) identify possible adaptive mutations arose during <br />                                                                              Interestingly, position, &#226;&#8364;&#8220;64 base pairs (bp) upstream <br /> evolution. <br />                                                                              annotated udhA transcription start site, mutated in <br />                                                                              strains udhA mutations. This mutation the <br /> Results                                                                      developed outside coding region. On other <br /> The growth rates glucose uptake rates 50-day                          hand, pntAB mutations likely result truncated, <br />                                                                              nonfunctional proteins: mutation pgi_gluc2 pntA a <br /> evolved Dpgi mutants increased 3.6- 2.6-fold                          <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> pgi_gluc7 pntA pgi_gluc4 pntB <br /> average, respectively, adaptive evolution                              mutations truncate proteins 510 265 462 154 <br />    All replicates converged similar endpoint phenotypes               amino acids, respectively. The pgi_gluc10 pntB mutation extends <br /> after fifty days serial passage adaptive evolution glucose         protein <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> amino acids: additional alanines are <br /> M9 minimal media assessed changes growth rates                   added region alanines present, <br /> (Figure 2A) glucose uptake rates (Figure 2B). These strains,         resulting consecutive alanine residues. <br /> as strains used study, summarized              Whereas evolved Dpgi replicates frequently developed <br /> Table 1. On average, growth rates strains exhibited          mutations rpoS, udhA, pntAB, mutations be <br /> a 3.6-fold increase starting unevolved Dpgi strain final       detected genes wild-type (pgi+) E. coli <br /> value 0.5060.03 hr21. That growth rates parallel replicates            replicates evolved growth conditions. This finding <br /> converge adaptive evolution observed previously              suggests mutations rpoS, udhA pntAB stemmed <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> and reproducible phenotypic outcome               adaptation loss pgi possible selection <br /> studies [17]. The glucose uptake rates evolved strains               pressures growth medium, phenomenon occurs <br /> exhibited variability; increased 2.6-fold final average         wild-type E. coli cultured minimal media containing <br /> value 4.6860.46 mmol/gram dry weight/hour.                                glycerol lactate sole carbon source [1,4]. <br />    Three replicates wild-type E. coli serially passed           Another noteworthy observation frequency which <br /> glucose M9 fifty days assessed changes growth rate.           genes involved global regulation mutated &#226;&#8364;&#8220; in <br /> The average initial growth rate 0.6960.0069 hr21. The                    strains. Besides rpoS, genes modulate <br /> average growth rate serial passage 0.7960.0092 hr21,               transcription multiple loci developed mutations <br /> which constitutes 1.1-fold increase. A separate study             included rpoA, rpoB, rpoC, cyaA cpxR (Table 2). This result <br /> wild-type E. coli evolved glucose minimal media 44               implies adaptation required global, network-level changes to <br /> days reported slightly lower initial final growth rates            transcriptional regulation metabolism, emerging a <br /> similar value fold increase evolutionary period             general, recurring theme multiple organisms. For example, <br /> [13].                                                                        E. coli strains adapted grow glycerol sole carbon source <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              2                     November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                               E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   3   November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                     E. coli Adaptation pgi Loss <br />  <br />  <br />  <br /> Figure 1. Role pgi, udhA, pntAB cellular metabolism. A. The gene pgi catalyzes isomerization glucose 6-phosphate fructose <br /> 6-phosphate upper glycolysis. Removal gene forces glycolytic flux pentose phosphate pathway, creating redox imbalance due <br /> to excess NADPH production. B. The genes udhA pntAB catalyze interconversion NAD/NADH NADP/NADPH. UdhA soluble protein <br /> whereas PntAB membrane-bound. <br /> doi:10.1371/journal.pgen.1001186.g001 <br />  <br /> often develop causative mutations RNAP [1]; mutation            knock-ins, double knock-ins, triple knock-in from <br /> transcription factor Spt15p confers greater ethanol tolerance            starting unevolved Dpgi strain. Like rpoS, pntA single knock- <br /> yeast [19]; mutation sensor kinase two-                   strain grew slightly slowly unevolved Dpgi strain <br /> component signal transduction enhanced transition                 (Figure 5). This slight growth rate reduction expected <br /> invasive infection virulence mouse model Group A                c197a mutation pntA changes codon triplet that <br /> Streptococcus pathogenesis [8].                                             position serine stop codon, truncating protein from <br />    Lastly, detected large indel: loss 15.4 kbp e14            510 65 amino acids likely rendering non-functional. The <br /> prophage pgi_gluc2. Its deletion initially suggested         udhA mutation, hand, does impact growth rate; <br /> both analysis resequencing data optical map          udhA single knock-in strain grew rate 1.4 times faster than <br /> strain. It subsequently confirmed PCR analysis ends           rpoS pntA single knock-in strains (P,0.001). <br /> of prophage flanking regions (Figure 3). We                  Construction double knock-in strains revealed both <br /> detect large indels resequencing data strain;          positive negative epistasis mutations. There is <br /> however, rule possibility        positive epistasis rpoS udhA mutations. Since the <br /> present, especially transposition mobile elements                rpoS single knock-in strain growth rate similar of <br /> insertion sequences (IS elements), limitations inherent          unevolved Dpgi clone, expect growth rate the <br /> in short-read resequencing technologies employed here. IS           rpoS + udhA double knock-in strain closely mimic the <br /> elements particular shown translocate frequently            single udhA knock-in strain mutations independent. <br /> in E. coli undergoes adaptive evolution wide                Instead, double knock-in growth rate 1.8 1.5 times <br /> variety conditions [20&#226;&#8364;&#8220;23]. If raw sequences                  greater unevolved single udhA knock-in <br /> assemble contigs large sufficiently span mobile          strains, respectively (Figure 5). We conclude data that, <br /> element flanking regions, difficult map                  addition udhA, rpoS mutation causal. In contrast, is <br /> accurately possible new locations genome.                         negative epistasis rpoS pntA mutations. This <br />                                                                             double knock-in strain grew overnight pre-culture LB <br /> Most mutations rpoS likely result nonfunctional                       medium, failed grow washing twice transferring to <br /> proteins confer increased growth rate                         glucose M9 media, days incubation. Lastly, udhA <br />                                                                             pntA mutations apparent epistasis the <br /> present alone <br />                                                                             growth rate double knock-in strain essentially identical <br />    The high frequency rpoS mutations prompted investigate <br />                                                                             udhA single knock-in strain. <br /> whether set mutations impact growth rate <br />                                                                                The growth rate triple knock-in strain only <br /> increases seen evolved Dpgi strains. When rpoS <br />                                                                             0.21 hr21, half evolved pgi_gluc2 strain <br /> mutations introduced chromosome starting <br />                                                                             (Figure 5). This finding implies deletion e14 prophage <br /> unevolved Dpgi clone, knock-in strains displayed <br />                                                                             likely contributes significantly adaptation loss pgi this <br /> an increase growth rate (Figure 4). Surprisingly, growth <br />                                                                             genetic background. <br /> rates slightly lower unevolved clone, indicating <br /> that neutral slightly deleterious (Student&#226;&#8364;&#8482;s t-test, <br /> P,0.01 five). We investigated indirectly           Discussion <br /> rpoS mutants encoded functional proteins flooding single              We investigated E. coli overcomes loss pgi, a <br /> colonies of knock-in strains hydrogen                 challenge forces glycolytic flux pentose phosphate <br /> peroxide. Vigorous bubbling occurs rpoS functional             pathway creates redox imbalance cell. The data <br /> RpoS control katE expression [24]. For comparison,               presented indicate adapted Dpgi mutants accomplish this <br /> performed assay evolved strains, starting               task mutations key genes, particular rpoS the <br /> unevolved Dpgi strain DrpoS mutant obtained Keio             transhydrogenases udhA pntAB, suppress bacterial <br /> collection [25]. The knock-in strains, evolved strains         stress response likely ameliorate redox imbalance, <br /> harboring rpoS mutations and, interestingly, evolved             respectively. Multiple alternative phenotypes arise these <br /> strains did rpoS mutation (pgi_gluc1) exhibited        mutations defined rates acetate formate <br /> reduced bubbling contact hydrogen peroxide (Table                 secretion. <br /> S1). The pgi_gluc1 result indicates genes rpoS              One explanation high frequency rpoS mutations is <br /> can control katE expression. As expected, DrpoS mutant             subordination rpoS-controlled stress response favor of <br /> exhibited reduced bubbling. In contrast, bubbling remained                  rpoD-controlled maximization <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake utilization. <br /> vigorous evolved strains (pgi_gluc8                The rpoS gene encodes sigma factor active stationary <br /> pgi_gluc10) did harbor rpoS mutation. It              phase affects expression fifty genes <br /> appears majority rpoS mutations result non-functional        log phase [26]. It controls general stress response E. coli <br /> proteins significant impact growth           related bacteria [27&#226;&#8364;&#8220;28] does considerable <br /> rates present alone.                                                   expense reduced expression rpoD-controlled housekeeping <br />                                                                             genes [29]. Perhaps importantly, rpoD controls <br /> The mutations pgi_gluc2 display positive                      expression genes involved <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> scavenging <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>- <br /> and negative epistatic interactions                                         limited environments [29]. This trade-off, designated stress <br />   We investigated rpoS, udhA pntA mutations             protection nutritional competence (SPANC) [30&#226;&#8364;&#8220;31], conse- <br /> pgi_gluc2 influence growth rate constructing single        quently creates conflict hunger stress responses <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             4                     November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                               E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   5   November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                                       E. coli Adaptation pgi Loss <br />  <br />  <br />  <br /> Figure 2. Growth rates, glucose uptake rates, acetate secretion rates unevolved evolved strains. A. Growth rates the <br /> starting unevolved Dpgi strain evolved strains adaptive evolution. The growth rate unevolved 50-day evolved wild-type E. coli <br /> K12 MG1655 medium 0.69(0.0069 0.79(0.0092 hr-1. B. Glucose uptake rates unevolved (pgi strain evolved <br /> strains adaptive evolution. Unevolved 50-day evolved wild-type E. coli glucose uptake rate 8.43(0.72 11.6(0.41 mmol/gDW/ <br /> hour, respectively. C. Acetate secretion rates unevolved (pgi strain evolved strains adaptive evolution. Unevolved 50-day <br /> evolved wild-type E. coli acetate secretion rate 12.8(8.87 5.61(0.24 mmol/gDW/hour, respectively. The symbol * indicates strains that <br /> also secrete formate (pgi_gluc1: 0.49(0.13 mmol/gDW/hour; pgi_gluc3: 0.22(0.02 mmol/gDW/hour). All error bars represent standard deviation <br /> from biological replicates. Abbreviations &#226;&#8364;&#8220; gDW: gram dry weight; hr: hour. <br /> doi:10.1371/journal.pgen.1001186.g002 <br />  <br /> in E. coli sigma factors simultaneously bind                           unevolved Dpgi mutants [32], speculate mutation <br /> RNAP. Since growth media Dpgi wild-                            upregulates udhA expression. In contrast, pntAB <br /> type E. coli evolutions contained initial glucose                           mutations likely reduce abolish PntAB function they <br /> concentration (2 g/L), greatly reduced growth rate                            produce peptides severely truncated. Decreasing PntAB <br /> unevolved Dpgi clone compared wild-type E. coli demonstrates                          function prevents redox imbalance condition caused loss of <br /> clearly does convert nutrients biomass                          pgi worsening: 13C-flux measurements PntAB <br /> optimally. This growth rate reduction, imposes stress,                        produces 35&#226;&#8364;&#8220;45% NADPH E. coli standard batch <br /> likely sufficient induce high levels RpoS unevolved Dpgi                    growth glucose [32]. The mutations udhA pntAB detected <br /> strain shift SPANC balance metabolism stress.                            support previous findings [32] transhy- <br /> The rpoS mutations emerged evolved strains,                    drogenases divergent functions despite shared ability to <br /> which reduce functionality protein indicated                            catalyze interconversion NAD/NADH NADP/ <br /> peroxidase assay (Table S1), shift balance stress                         NADPH reversibly. Whereas UdhA plays major role oxidizing <br /> back metabolism allow greater rpoD-controlled                     NADPH NAD, membrane-bound PntAB plays major <br /> nutrient acquisition consequent faster growth.                                       role reoxidizing NADH NADP. The pgi_gluc1 fabZ <br />    There notable mutations rpoS.                      mutation (missense; P167L) noteworthy fabZ involved <br /> As mentioned, &#226;&#8364;&#8220;64 position upstream udhA transcription                        fatty acid biosynthesis, process utilizes NADPH. This <br /> start site mutated strains. Because udhA plays                   raises possibility NADPH overabundance Dpgi mutants <br /> a major role oxidizing NADPH NAD conditions                            drives excessive fatty acids biosynthesis, translating redox <br /> excess NADPH overexpression increases growth rate                         imbalance imbalance fatty acid production. The fabZ <br />  <br />  Table 1. Strains used study. <br />  <br />  <br />  Strain                     Characteristics                                                                                                   Source <br />  <br />  Dpgi                       Starting strain adaptive evolutions                                                                           Fong et al. [45] <br />  pgi_gluc1                  50-day evolved Dpgi strain. Secretes acetate formate                                                          study <br />  pgi_gluc2                  50-day evolved Dpgi strain.                                                                                       study <br />  pgi_gluc3                  50-day evolved Dpgi strain. Secretes acetate formate                                                          study <br />  pgi_gluc4                  50-day evolved Dpgi strain.                                                                                       study <br />  pgi_gluc5                  50-day evolved Dpgi strain. Secretes acetate                                                                      study <br />  pgi_gluc6                  50-day evolved Dpgi strain. Secretes acetate                                                                      study <br />  pgi_gluc7                  50-day evolved Dpgi strain.                                                                                       study <br />  pgi_gluc8                  50-day evolved Dpgi strain.                                                                                       study <br />  pgi_gluc9                  50-day evolved Dpgi strain.                                                                                       study <br />  pgi_gluc10                 50-day evolved Dpgi strain.                                                                                       study <br />  KI2_rpoS                   Dpgi in-frame duplication rpoS 595&#226;&#8364;&#8220;603                                                                 study <br />  KI4_rpoS                   Dpgi 1 base pair deletion rpoS position 841                                                            study <br />  KI5_rpoS                   Dpgi 1 base pair deletion rpoS position 850                                                            study <br />  KI6_rpoS                   Dpgi out-of-frame duplication rpoS 597&#226;&#8364;&#8220;603                                                             study <br />  KI7_rpoS                   Dpgi c829t mutation rpoS                                                                                  study <br />  KI2_udhA                   Dpgi udhA g(-64)a mutation                                                                                   study <br />  KI2_pntA                   Dpgi pntA c197a mutation                                                                                     study <br />  KI2_RU                     Dpgi double knock-in strain rpoS 595&#226;&#8364;&#8220;603 duplication udhA g(-64)a SNP                                    study <br />  KI2_RP                     Dpgi double knock-in strain rpoS 595&#226;&#8364;&#8220;603 duplication pntA c197a SNP                                      study <br />  KI2_UP                     Dpgi double knock-in strain udhA g(-64)a pntA c197a mutations                                            study <br />  KI2_3KI                    Dpgi triple knock-in strain rpoS 595&#226;&#8364;&#8220;603 duplication udhA g(-64)a pntA c197a SNPs                    study <br />  <br />  Knock-in strains constructed introducing mutations identified adaptive evolution starting unevolved Dpgi strain designated &#226;&#8364;&#732;&#226;&#8364;&#732;KI.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />  This designation followed number corresponds evolved replicate knock-in strain based. The identity gene(s) containing the <br />  mutation follows last. Abbreviations &#226;&#8364;&#8220; RU: rpoS + udhA; RP: rpoS + pntA; UP: udhA + pntA; 3KI: triple knock-in. <br />  doi:10.1371/journal.pgen.1001186.t001 <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                         6                      November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                                     E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />  Table 2. Mutations detected clones isolated evolved Dpgi strains isolated 50 days adaptive <br />  evolution. <br />  <br />  <br />              pgi_gluc1     pgi_gluc2            pgi_gluc3      pgi_gluc4             pgi_gluc5       pgi_gluc6         pgi_gluc7        pgi_gluc8      pgi_gluc10 <br />  <br />  rpoA                                           c854t                                                D944..963                                         t805c <br />  rpoB        a3724c <br />  rpoC        c3520a <br />  rpoS                      Dupl. (595&#226;&#8364;&#8220;603)      g836t          D1bp (841)            D1bp (850)      Dupl.             c829t <br />                                                                                                      (597&#226;&#8364;&#8220;603) <br />  udhA        a949g         g(264)a              g(264)t                                                                g(264)a          g(264)a; <br />                                                                                                                                         g58a <br />  pntA                      c197a                                                                                       D680..690 <br />  pntB                                                          D1bp (430)                                                                              Dupl. <br />                                                                                                                                                        (698&#226;&#8364;&#8220;703) <br />  rep         t242g <br />  fabZ        c167t <br />  cpxR                                           g614a          D4bp, 585..588 <br />  yfeH                                                          D1bp (297) <br />  fruK                                                                                c886t <br />  rodA                                                                                                                  c263a <br />  cyaA                                                                                                                                   a1175c <br />  bipA                                                                                                                                   1bp Dupl. <br />                                                                                                                                         (960) <br />  ispU                                                                                                                                                  t659g <br />  Large                     e14 prophage <br />  indel                     deletion <br />  <br />  All sequenced using Nimblegen tiling arrays. Strains pgi_gluc1, pgi_gluc7, pgi_gluc10 sequenced using first-generation Solexa (Illumina) <br />  technology. Strain pgi_gluc9 sequenced. Sanger sequencing used validate reported mutations. Additionally, strains sequenced the <br />  rpoS, udhA, pntA pntB genes entirety using Sanger sequencing, just specific regions indicated Nimblegen Solexa technologies contain a <br />  mutation. The length genes similarly sequenced evolved wild-type (rpoS+) replicates; mutations position. No <br />  genomic positions sequenced evolved wild-type replicates. Abbreviations &#226;&#8364;&#8220; bp: base pair; Dupl: duplication. <br />  doi:10.1371/journal.pgen.1001186.t002 <br />  <br />  <br /> mutation pgi_gluc1 help alleviate imbalance                      TCA cycle E. coli Dpgi mutants. It known that <br /> decreasing flux pathway.                                                wild-type E. coli secretes acetate aerobic growth glucose <br />    The observation pgi_gluc2 triple knock-in strain                   [40&#226;&#8364;&#8220;41], phenomenon known overflow metabolism [42], <br /> not reproduce growth rate evolved strain implies                     acetyl-CoA converts acetate enter TCA <br /> that loss e14 prophage plays role adaptation. This                       cycle. On hand, loss pgi forces flux normally <br /> prophage lies position 1,195,432 bp 1,210,646 bp [33]                   occurring upper glycolysis pentose phosphate <br /> chromosome contains 24 putative ORFs,                         pathway, reducing flux lower glycolysis [32] and <br /> unknown function [34]. Exposure UV radiation consistently                         presumably reducing acetyl-CoA enters the <br /> activates excision element [35&#226;&#8364;&#8220;36], probably                         TCA cycle relative wild-type E. coli. Minimizing by- <br /> induction SOS response [37]. This observation implies                    product secretion overflow metabolism therefore <br /> the SOS response probably induced point                           serve maximize reduced flux flowing through <br /> adaptation loss pgi pgi_gluc2. Deletion e14                        lower glycolysis enters TCA cycle. The regulation of <br /> contribute adaptation loss pgi ways. First, e14                  acetate metabolism, however, complex process [43&#226;&#8364;&#8220;44], and <br /> contains genes presence/absence likely affects                         factors involved. <br /> metabolism, example toxin encoded kil kills cell                      Besides contrasting acetate secretion profiles, hypothesize <br /> in absence repressor, encoded e14                        evolved Dpgi strains utilize glyoxylate shunt their <br /> [34,38]. Second, deletion e14 introduces synonymous                       evolved wild-type (pgi+) counterparts shunt is <br /> two non-synonymous mutations amino acid sequence                         active unevolved E. coli lacking pgi [32]. Routing flux <br /> isocitrate dehydrogenase (icd), key metabolic enzyme [39].                         partly glyoxylate shunt TCA cycle <br /> Although mutations alters specific activity                 avoid exacerbating redox imbalance problem because <br /> protein cell-free experiments [39], exert effect                 NAD(P)H biosynthesis normally arising conversion of <br /> vivo potential impact translation efficiency.                       isocitrate a-ketoglutarate (NADPH) a-ketoglutarate to <br />    Acetate secretion rates zero approximately times                       succinyl-CoA (NADH) occur. <br /> lower evolved Dpgi replicates compared unadapted                          In conclusion, examined robustness E. coli <br /> and glucose-adapted wild-type E. coli (Figure 2C), implicates                  metabolic network genetic basis adaptation loss <br /> a connection acetate metabolism loss pgi. We                          pgi adaptive evolution, resequencing phenotypic <br /> speculate connection stems need maximize flux                    assays. Two mutations detected adaptation and <br />  <br />  <br />          PLoS Genetics | www.plosgenetics.org                                    7                       November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                   E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />  <br /> Figure 3. Loss e14 prophage pgi_gluc2. A. Structure location e14 prophage. It integrated icd gene E. coli <br /> chromosome position 1,195,432 position 1,210,646. B. PCR analysis unevolved (pgi, pgi_gluc2 triple knock-in (KI2_3KI) evolved <br /> pgi_gluc2 strains confirms loss e14 prophage pgi_gluc2. Numbers 1 4 correspond PCR amplification regions indicated the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                            8                    November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                        E. coli Adaptation pgi Loss <br />  <br />  <br />  <br /> top panel. Regions 1 3 span terminal segments integrated prophage adjacent chromosomal DNA. Region 2 spans segment wholly <br /> within prophage. Region 4 spans integration site amplified using left primer Region 1 right primer Region 3. <br /> doi:10.1371/journal.pgen.1001186.g003 <br />  <br />  <br /> were frequently located alternative sigma factor rpoS,            genotype phenotype creation additional knock-in <br /> the soluble transhydrogenase udhA, membrane-bound                     mutants examining effect mutation metabolic <br /> transhydrogenase pntAB. The genetic biochemical data                      flux. Such studies highlight plasticity the <br /> collected paints picture general mechanism                bacterial genome cellular networks delineate <br /> through E. coli adapts loss pgi manages redox             mechanisms bacteria adapt genetic perturba- <br /> imbalance problem occurs 1) favoring rpoD-controlled hunger               tions. <br /> response rpoS-controlled stress response, 2) curtailing (and <br /> possibly eliminating) PntAB function lessen PntAB-catalyzed                Methods <br /> NADPH biosynthesis. We evidence showing that <br /> adaptive evolution lead multiple, alternative phenotypes               Strains <br /> within evolutionary landscape, defined study                    The strains used study summarized Table 1. The <br /> difference byproduct secretion rates. Looking forward, adapta-             starting strain adaptively evolved replicates E. coli <br /> tion loss pgi, loss metabolic genes,            Dpgi strain constructed wild-type E. coli K12 MG1655 <br /> might constitute model investigate link               (ATCC, Manassas, VA) described previously [45]. Whole- <br />  <br />  <br />  <br />  <br /> Figure 4. Growth rates rpoS knock-in strains. The strains constructed introducing rpoS mutations detected after <br /> adaptive evolution starting unevolved Dpgi strain site-directed mutagenesis. A similar knock-in strain containing pgi_gluc3 <br /> rpoS mutation constructed. Growth rate data starting unevolved Dpgi strain shown comparison. Error bars represent the <br /> standard deviation biological replicates. A description strain abbreviations Table 1. Abbreviations &#226;&#8364;&#8220; hr: hour. <br /> doi:10.1371/journal.pgen.1001186.g004 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               9                     November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                E. coli Adaptation pgi Loss <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   10   November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                      E. coli Adaptation pgi Loss <br />  <br />  <br />  <br /> Figure 5. Growth rates glucose uptake rates pgi_gluc2 single, double, triple knock-in strains. A. Growth rates three <br /> single, double triple knock-in strains constructed based mutations appeared pgi_gluc2. The growth rates the <br /> starting unevolved Dpgi strain evolved pgi_gluc2 strain shown comparison. B. Corresponding glucose uptake rates seven <br /> knock-in strains. Data unevolved evolved strains shown comparison. Error bars represent standard deviation <br /> from biological replicates. A description strain abbreviations Table 1. Abbreviations &#226;&#8364;&#8220; gDW: gram dry weight; hr: <br /> hour. <br /> doi:10.1371/journal.pgen.1001186.g005 <br />  <br /> genome resequencing identified base pair changes                based tiling arrays [49] Illumina technology. Nimblegen also <br /> MG1655 Dpgi strain utilized present                provided data analysis capabilities [49] identification of <br /> sequenced MG1655 strain: cRg mutation ybeB (genomic                     possible mutations, Solexa results analyzed using a <br /> position 667965), aRg mutation ylbE_1 (genomic position                software program developed in-house counts total number <br /> 547694) cRa mutation nupC (genomic position 2511373).               base given position compares consensus call <br /> Sanger sequencing used confirm mutations.                 reference E. coli genome. All reported mutations were <br /> Strains pgi_gluc1 pgi_gluc10 generated               confirmed PCR amplification surrounding DNA region <br /> this study adaptive evolution starting E. coli Dpgi           Sanger sequencing. Additionally, used Sanger <br /> strain. Knock-in strains mutations detected adaptive          sequencing sequence full-length rpoS, udhA, pntA, pntB <br /> evolution introduced starting unevolved Dpgi strain            genes resequenced, evolved Dpgi strains three <br /> were created using method Tischer et al. [46]                evolved wild-type E. coli replicates, just regions immediately <br /> following modifications: 1) pKD46, temperature-sensitive                   surrounding mutations reported Nimblegen and/or Illumina. <br /> plasmid carries bacteriophage l red genes (c, b, exo)               For evolved wild-type replicates, genomic positions <br /> under control arabinose-inducible ParaBAD promoter                sequenced full-length genes four. The list <br /> [47], electroporated cells transformed              primers used PCR amplification Sanger <br /> subsequently used Red recombination step, 2) pKD13             sequencing given Table S3. The optical map pgi_gluc2 <br /> [47] used amplify kanamycin resistance gene instead            provided service OpGen, Inc. (Gaithersburg, MD) <br /> pACYC177, 3) pACBSR, contains chloramphenicol-                   using NcoI restriction enzyme. <br /> resistance selection marker encodes I-SceI l red <br /> system control arabinose-dependent                      Growth rate substrate uptake/secretion rate <br /> promoter [48], used second Red recombination step. <br />                                                                              measurements <br /> The primers used construct knock-in strains in <br />                                                                                 Overnight pre-cultures strain grown LB medium <br /> Table S2. <br />                                                                              spun washed twice glucose M9 minimal <br />                                                                              medium, used inoculate 500 mL <br /> Growth media                                                                 Erlenmeyer flasks containing 250 mL glucose M9 triplicate. <br />     The starting Dpgi clone maintained propagated                 These flasks incubated overnight air incubator <br /> Luria-Bertrani (LB) broth (EMD Chemicals, Gibbstown, NJ). It                 maintained 37uC using magnetic stir bars aeration, <br /> was exposed glucose M9 minimal medium prior                   conditions identical used adaptive <br /> study. The composition glucose M9 was: dextrose (2 g/L),              evolutions. The day inoculum replicate was <br /> CaCl2 (100 mM), MgSO4 (200 mM), Na2HPO4 (13.6 g/L),                          transferred new 500 mL Erlenmeyer flask containing <br /> KH2PO4 (6 g/L), NaCl (1 g/L), NH4Cl (2 g/L) trace                        250 mL glucose M9 optical density 600 nm <br /> elements (500 mL). The trace element solution consisted (per <br />                                                                              (OD600) reduced 0.005. These flasks then <br /> liter): FeCl3N6H2O (16.67 g), ZnSO4N7H2O (0.18 g), CuCl2N2H2O <br />                                                                              placed water bath maintained 37uC. Magnetic stir bars <br /> (0.12 g), MnSO4NH2O (0.12 g), CoCl2N6H2O (0.18 g) and <br />                                                                              used aeration. <br /> Na2EDTAN2H2O (22.25 g). <br />                                                                                 Once OD600 reached 0.05 continuing periodically <br />                                                                              thereafter, optical density sample recorded and <br /> Adaptive evolution protocol                                                  culture media collected filtered 0.22 mm <br />    The adaptive evolution protocol used described              membranes. HPLC analysis (Waters, Milford, MA) then <br /> previously [45]. To summarize, Dpgi replicates               carried filtered media using Bio-Rad Aminex HPX- <br /> three wild-type replicates serially passed daily 250 mL              87H ion exclusion column (300 mm67.8 mm) 5 mM H2SO4 <br /> glucose M9 minimal medium 500 mL Erlenmeyer flasks 50                 mobile phase flow rate 0.5 mL/min temperature <br /> days 37uC using magnetic stir bars aeration. The volume               45uC. <br /> transferred adjusted day account changes in <br /> growth rate order maintain cultures prolonged exponential <br />                                                                              Supporting Information <br /> phase growth avoid entry stationary phase. On <br /> Day 50, aliquot replicate streaked LB plates           Table S1 Indirect assessment RpoS activity utilizing the <br /> containing 20 g/L bacteriological agar (Sigma-Aldrich, St. Louis,            peroxidase assay [19]. Colonies functional RpoS exhibit <br /> MO) incubated 37uC 24 hours. A well-isolated colony               vigorous bubbling come contact hydrogen <br /> was selected suspended LB broth containing 25%                   peroxide mechanism based rpoS control katE expression. <br /> glycerol stored long-term 280uC.                                      We defined vigorous bubbling (v) bubble formation occurring <br />                                                                              seconds contact hydrogen peroxide; medium <br /> Whole-genome resequencing                                                    bubbling (m) bubble formation occurring ten <br />   We isolated genomic DNA using Qiagen DNeasy kit                          seconds contact; slight bubbling (s) bubble formation <br /> (QIAGEN, Valencia, CA) single-colony sample                    occurring seconds contact. This assay performed 24 <br /> collected long-term storage. Whole-genome sequencing                 48 hours colonies inoculated LB plates. <br /> then carried using platforms, Nimblegen hybridization-               Abbreviations: repl - replicate. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             11                    November 2010 | Volume 6 | Issue 11 | e1001186 <br />                                                                                                                                                 E. coli Adaptation pgi Loss <br />  <br />  <br /> Found at: doi:10.1371/journal.pgen.1001186.s001 (0.06 MB                                       Found at: doi:10.1371/journal.pgen.1001186.s003 (0.06 MB <br /> DOC)                                                                                           DOC) <br /> Table S2    Primers used introduce mutations detected after <br /> adaptive evolution starting unevolved Dpgi strain                                Acknowledgments <br /> according method Tischer et al [46].                                                 We thank Markus Herrgard Byung-Kwok Cho helpful discussions <br /> Found at: doi:10.1371/journal.pgen.1001186.s002 (0.04 MB                                       Jaqueline Miller, Elisa Abate, Courtney Chew, Howard Li for <br /> DOC)                                                                                           technical assistance. <br /> Table S3 Primers used Sanger sequencing confirm <br /> mutations reported Nimblegen Illumina sequencing                                        Author Contributions <br /> technologies. The rpoS, udhA, pntA, pntB genes sequenced                              Conceived designed experiments: B&#195;&#732;P. Performed experi- <br /> in entirety evolved strains using Sanger                                     ments: PC TMC EMK KV NLF BX YG. Analyzed data: PC TMC <br /> sequencing, just regions immediately surrounding reported                                  EMK KV NLF B&#195;&#732;P. Contributed reagents/materials/analysis tools: BX <br /> mutations. Primers designed purpose dividing                              YG. Wrote paper: PC TMC B&#195;&#732;P. <br /> four genes regions approximately 800&#226;&#8364;&#8220;900 base pairs with <br /> an overlap regions approximately 100 base pairs. <br />  <br /> References <br />  1. Herring CD, Raghunathan A, Honisch C, Patel T, Applebee MK, et al. (2006)                  21. Stoebel DM, Hokamp K, Last MS, Dorman CJ (2009) Compensatory evolution <br />     Comparative genome sequencing Escherichia coli allows observation                        gene regulation response stress Escherichia coli lacking RpoS. PLoS <br />     bacterial evolution laboratory timescale. Nat Genet 38: 1406&#226;&#8364;&#8220;1412.                        Genet 5: e1000671. doi:10.1371/journal.pgen.1000671. <br />  2. Barrick JE, Yu DS, Yoon SH, Jeong H, Oh TK, et al. (2009) Genome evolution                 22. Naas T, Blot M, Fitch WM, Arber W (1994) Insertion sequence-related genetic <br />     adaptation long-term experiment Escherichia coli. Nature 461:                    variation resting Escherichia coli K-12. Genetics 136: 721&#226;&#8364;&#8220;730. <br />     1243&#226;&#8364;&#8220;1247.                                                                                 23. Riehle MM, Bennett AF, Long AD (2001) Genetic architecture thermal <br />  3. Barrick JE, Lenski RE (2009) Genome-wide Mutational Diversity Evolving                   adaptation Escherichia coli. Proc Natl Acad Sci U S A 98: 525&#226;&#8364;&#8220;530. <br />     Population Escherichia coli. Cold Spring Harb Symp Quant Biol.                          24. Zambrano MM, Siegele DA, Almiron M, Tormo A, Kolter R (1993) Microbial <br />  4. Conrad TM, Joyce AR, Applebee MK, Barrett CL, Xie B, et al. (2009) Whole-                      competition: Escherichia coli mutants stationary phase cultures. <br />     genome resequencing Escherichia coli K-12 MG1655 undergoing short-term                      Science 259: 1757&#226;&#8364;&#8220;1760. <br />     laboratory evolution lactate minimal media reveals flexible selection                25. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction <br />     adaptive mutations. Genome Biol 10: R118.                                                      Escherichia coli K-12 in-frame, single-gene <span id='am-477' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-478' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-479' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-487' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-489' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-491' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-507' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-510' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-526' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-531' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-542' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-548' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-551' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-553' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-554' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio <br />  5. Lee DH, Palsson BO. Adaptive Evolution Of Escherichia coli K-12 MG1655                         collection. Mol Syst Biol 2006 2: 2006 0008. <br />     On A Non-Native Carbon Source, L-1,2-Propanediol. Appl Environ Microbiol.                  26. Dong T, Schellhorn HE (2009) Control RpoS global gene expression of <br />  6. Maharjan R, Zhou Z, Ren Y, Li Y, Gaffe J, et al. Genomic identification                   Escherichia coli minimal media. Mol Genet Genomics 281: 19&#226;&#8364;&#8220;33. <br />     <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>novel mutation</span> hfq provides multiple benefits evolving glucose-limited          27. Hengge-Aronis R (2002) Signal transduction regulatory mechanisms <br />     populations Escherichia coli. J Bacteriol.                                                  involved control sigma(S) (RpoS) subunit RNA polymerase. <br />  7. Velicer GJ, Raddatz G, Keller H, Deiss S, Lanz C, et al. (2006) Comprehensive                  Microbiol Mol Biol Rev 66: 373&#226;&#8364;&#8220;395, table contents. <br />     mutation identification evolved bacterial cooperator cheating                28. Hengge-Aronis R (2002) Recent insights general stress response <br />     ancestor. Proc Natl Acad Sci U S A 103: 8107&#226;&#8364;&#8220;8112.                                             regulatory network Escherichia coli. J Mol Microbiol Biotechnol 4: 341&#226;&#8364;&#8220;346. <br />  8. Sumby P, Whitney AR, Graviss EA, DeLeo FR, Musser JM (2006) Genome-                        29. Notley-McRobb L, King T, Ferenci T (2002) rpoS mutations loss general <br />     wide analysis group streptococci reveals mutation modulates                        stress resistance Escherichia coli populations consequence conflict <br />     global phenotype disease specificity. PLoS Pathog 2: e5. doi:10.1371/                      competing stress responses. J Bacteriol 184: 806&#226;&#8364;&#8220;811. <br />     journal.ppat.0020005.                                                                      30. King T, Ishihama A, Kori A, Ferenci T (2004) A regulatory trade-off source <br />  9. Mwangi MM, Wu SW, Zhou Y, Sieradzki K, Lencastre H, et al. (2007)                           strain variation species Escherichia coli. J Bacteriol 186: 5614&#226;&#8364;&#8220;5620. <br />     Tracking vivo evolution multidrug resistance Staphylococcus aureus <br />                                                                                                31. Ferenci T (2005) Maintaining healthy SPANC balance regulatory and <br />     whole-genome sequencing. Proc Natl Acad Sci U S A 104: 9451&#226;&#8364;&#8220;9456. <br />                                                                                                    mutational adaptation. Mol Microbiol 57: 1&#226;&#8364;&#8220;8. <br /> 10. Friedman L, Alder JD, Silverman JA (2006) Genetic changes correlate with <br />                                                                                                32. Sauer U, Canonaco F, Heri S, Perrenoud A, Fischer E (2004) The soluble and <br />     reduced susceptibility daptomycin Staphylococcus aureus. Antimicrob <br />                                                                                                    membrane-bound transhydrogenases UdhA PntAB divergent <br />     Agents Chemother 50: 2137&#226;&#8364;&#8220;2145. <br />                                                                                                    functions NADPH metabolism Escherichia coli. J Biol Chem 279: <br /> 11. Andries K, Verhasselt P, Guillemont J, Gohlmann HW, Neefs JM, et al. (2005) <br />                                                                                                    6613&#226;&#8364;&#8220;6619. <br />     A diarylquinoline drug active ATP synthase Mycobacterium <br />                                                                                                33. Blattner FR, Plunkett G, 3rd, Bloch CA, Perna NT, Burland V, et al. (1997) <br />     tuberculosis. Science 307: 223&#226;&#8364;&#8220;227. <br />                                                                                                    The complete genome sequence Escherichia coli K-12. Science 277: <br /> 12. Andersson DI, Hughes D (2010) Antibiotic resistance cost: possible to <br />                                                                                                    1453&#226;&#8364;&#8220;1462. <br />     reverse resistance? Nat Rev Microbiol 8: 260&#226;&#8364;&#8220;271. <br /> 13. Fong SS, Palsson BO (2004) Metabolic gene-deletion strains Escherichia coli             34. Mehta P, Casjens S, Krishnaswamy S (2004) Analysis lambdoid prophage <br />     evolve computationally predicted growth phenotypes. Nat Genet 36:                           element e14 E. coli K-12 genome. BMC Microbiol 4: 4. <br />     1056&#226;&#8364;&#8220;1058.                                                                                 35. Aertsen A, Van Houdt R, Vanoirbeek K, Michiels CW (2004) An SOS response <br /> 14. Hua Q, Yang C, Baba T, Mori H, Shimizu K (2003) Responses central                       induced high pressure Escherichia coli. J Bacteriol 186: 6133&#226;&#8364;&#8220;6141. <br />     metabolism Escherichia coli phosphoglucose isomerase glucose-6-                  36. Harris DR, Pollock SV, Wood EA, Goiffon RJ, Klingele AJ, et al. (2009) <br />     phosphate dehydrogenase knockouts. J Bacteriol 185: 7053&#226;&#8364;&#8220;7067.                                 Directed evolution ionizing radiation resistance Escherichia coli. J Bacteriol <br /> 15. Canonaco F, Hess TA, Heri S, Wang T, Szyperski T, et al. (2001) Metabolic flux                 191: 5240&#226;&#8364;&#8220;5252. <br />     response phosphoglucose isomerase <span id='am-399' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-400' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-475' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-476' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>knock-out</span> Escherichia coli impact              37. Greener A, Hill CW (1980) Identification novel genetic element in <br />     overexpression soluble transhydrogenase UdhA. FEMS Microbiol Lett                    Escherichia coli K-12. J Bacteriol 144: 312&#226;&#8364;&#8220;321. <br />     204: 247&#226;&#8364;&#8220;252.                                                                              38. Plasterk RH, van Putte P (1985) The invertible P-DNA segment the <br /> 16. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, et al. (2007) A                         chromosome Escherichia coli. Embo J 4: 237&#226;&#8364;&#8220;242. <br />     genome-scale metabolic reconstruction Escherichia coli K-12 MG1655                39. Hill CW, Gray JA, Brody H (1989) Use isocitrate dehydrogenase <br />     accounts 1260 ORFs thermodynamic information. Mol Syst Biol 3: 121.                    structural gene attachment e14 Escherichia coli K-12. J Bacteriol 171: <br /> 17. Fong SS, Joyce AR, Palsson BO (2005) Parallel adaptive evolution cultures                   4083&#226;&#8364;&#8220;4084. <br />     Escherichia coli lead convergent growth phenotypes different gene                  40. Andersen KB, von Meyenburg K (1980) Are growth rates Escherichia coli in <br />     expression states. Genome Res 15: 1365&#226;&#8364;&#8220;1372.                                                   batch cultures limited respiration? J Bacteriol 144: 114&#226;&#8364;&#8220;123. <br /> 18. Boonstra B, French CE, Wainwright I, Bruce NC (1999) The udhA gene                      41. Holms WH, Bennett PM (1971) Regulation isocitrate dehydrogenase activity <br />     Escherichia coli encodes soluble pyridine nucleotide transhydrogenase.                       Escherichia coli adaptation acetate. J Gen Microbiol 65: 57&#226;&#8364;&#8220;68. <br />     J Bacteriol 181: 1030&#226;&#8364;&#8220;1034. <br /> </body></html>