<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mycobacterium tuberculosis Rv3802c Encodes a <br /> Phospholipase/Thioesterase Is Inhibited the <br /> Antimycobacterial Agent Tetrahydrolipstatin <br /> Sarah K. Parker1*, Robert M. Barkley3, John G. Rino1, Michael L. Vasil2 <br /> 1 Department Pediatrics, University Colorado Denver, Aurora, Colorado, United States America, 2 Department Microbiology, University Colorado Denver, <br /> Aurora, Colorado, United States America, 3 Department Pharmacology, University Colorado Denver, Aurora, Colorado, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The cell wall M. tuberculosis central success pathogen. Mycolic acids key components cell wall. The <br />      genes involved joining mero mycolates located cluster, beginning Rv3799c extending least <br />      Rv3804c. The role enzyme encoded genes fairly understood, Rv3802c. Rv3802 is <br />      seven putative cutinases encoded genome M. tuberculosis. In phytopathogens, cutinases hydrolyze the <br />      waxy layer plants, cutin. In strictly mammalian pathogen, M. tuberculosis, likely proteins perform a <br />      different function. Of seven, chose focus Rv3802c location mycolic acid synthesis gene <br />      cluster, putative essentiality, ubiquitous presence actinomycetes, conservation minimal genome of <br />      Mycobacterium leprae. We expressed Rv3802 Escherichia coli purified enzymatically active form. We probed its <br />      activities inhibitors characterizing relevant possible role mycolic acid biosynthesis. In addition its <br />      reported phospholipase A activity, Rv3802 significant thioesterase activity, inhibited tetrahydrolipstatin <br />      (THL). THL described anti-tuberculous compound unknown mechanism, reportedly targets cell wall <br />      synthesis. Taken together, data circumstantially support role Rv3802 mycolic acid synthesis and, cell wall <br />      integral M. tuberculosis pathogenesis, identification novel cell wall enzyme inhibition therapeutic and <br />      diagnostic implications. <br />  <br />   Citation: Parker SK, Barkley RM, Rino JG, Vasil ML (2009) Mycobacterium tuberculosis Rv3802c Encodes Phospholipase/Thioesterase Is Inhibited the <br />   Antimycobacterial Agent Tetrahydrolipstatin. PLoS ONE 4(1): e4281. doi:10.1371/journal.pone.0004281 <br />   Editor: Niyaz Ahmed, University Hyderabad, India <br />   Received November 19, 2008; Accepted January 8, 2009; Published January 26, 2009 <br />   Copyright: &#195;&#376; 2009 Parker et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grant (K08 AI050646) National Institute Allergy Infectious Diseases, grant Potts Foundation, a <br />   grant The Children&#226;&#8364;&#8482;s Hospital Research Institute grant American Lung Foundation Sarah K. Parker, grant (HL062608) the <br />   National Heart, Lung Blood Institute Michael L. Vasil. The contribution Robert M. Barkley supported LIPID MAPS, Large-Scale Collabrative Grant <br />   National Institutes Health (Grant GM-069338). The funders role study design, data collection analysis, decision publish, preparation <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: sarah.parker@ucdenver.edu <br />  <br />  <br />  <br /> Introduction                                                                            trehalose arabinogalactan located gene cluster <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                         Rv3799c Rv3804c (Figure 1). The role enzyme <br />    M. tuberculosis infects world, makes 8                         encoded genes fairly understood, the <br /> million ill year, kills 1.8 million, 450,000                       product Rv3802c, genes account all <br /> children [1]. Though oldest known human pathogens,                           necessary steps pathway [7]. A summary presented in <br /> our ability combat spread disease remains insufficient                       Figure 2. In brief, FadD32 (fatty acid dehydrogenase, Rv3801c) <br /> and global health burden tuberculosis increasing [2]. Key                     AccD4 (Rv3799c), acyl-CoA carboxylases, <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> success tubercule bacillis uniquely complex lipid                  AccA3 AccD5, participate deriving meroacyl-AMP (from <br /> rich cell wall, cell wall synthesis pathways current target                     FAS-II) 2-carboxyl-C26-S-CoA (from FAS-I), respectively. <br /> areas drug development [3&#226;&#8364;&#8220;6]. The cell wall mycobacteria                      These attached thioester bonds phosphopentathienes on <br /> considered bilayer, lipids integral bilayer                    acyl carrier domains Pks13 (polyketide synthase, Rv3800c), a <br /> myolic acids. Just exterior cell membrane lies mycolyl-                      multidomain protein which, Claisen condensation, creates a <br /> arabinogalactan-peptidoglycan complex (mAGP). This complex                              mature mycolic acid. A reduction step needed enzyme <br /> forms stable scaffolding outer component surface-                        far, CmrA, identified role [8]. The deposition <br /> exposed lipids glycolipids, trehalose monomycolate                          export mycolates understood. A mature <br /> (TMM) dimycolate (TDM). The mycolic acids                                  mycolic acid eventually transferred arabinogalactan (to <br /> glycolipids noncovalently intercolated mAGP.                               form mAGP) Ag85 proteins, a,a-trehalose- <br /> Mycolic acid containing lipids essential                           6-phosphate (T6P, leading TMM) unknown mycolyl- <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span> M. tuberculosis, contribute pathogenesis                  transferase [9]. A phospholipid carrier, 6-0-mycolyl-a-D-manno- <br /> variety mechanisms cell signaling host evasion                           pyranosyl-1-monophosphoheptaprenol (Myc-PL), reported be <br /> granuloma formation. The genes involved joining mero                         involved Mycobacterium smegmatis[10]. For TDM formation, a <br /> a mycolate mature mycolic acid transferring                            second mycolic acid added TMM antigen 85 complex <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   1                                 January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                      M. tb Rv3802 Inhibited THL <br />  <br />  <br />  <br />  <br /> Figure 1. Mycolic acid synthesis gene cluster M. tuberculosis H37Rv. Cluster highly conserved actinomycetes. <br /> doi:10.1371/journal.pone.0004281.g001 <br />  <br /> mycolyltransferases, including Ag85A (Rv3804c) [11]. In study            catalytic H 30 amino acids D 12 <br /> we investigated possible contributions Rv3802c gene                described classic motif[16]. This motif places Rv3802 and <br /> product mycolic acid synthesis pathway. Our data                       cutinases a/b-hydrolase fold family, company <br /> demonstrate products encoded Rv3802c non-                 known mycolyltransferases (Ag85 complex), known PLAs and <br /> orthologous homologue MSMEG_1403 possess phospho-                    known thioesterases. Like a/b-hydrolase fold enzymes, the <br /> lipase A (PLA) activity thioesterase activity. These activities           predicted structure oxyanion hole, provides a <br /> are consistent role mycolic acid biosynthesis               positively charged hole stabilize transient oxyanions formed <br /> pathway involves multiple ester thioester bonds. In addition,             acylation deacylation, acyl binding pocket lined <br /> the Rv3802c gene product, distant homologue               hydrophobic residues [17,18]. Rv3802 predicted a <br /> MSMEG_1403 M. smegmatis, inhibited known human                   membrane protein, unclear cytosolic or <br /> fatty acid synthase thioesterase (FASTE) domain inhibitor, THL.               extracytosolic side[19]. It predicted membrane <br /> THL demonstrated decrease mycolic acid synthesis,                 spanning helix located putative secretion signal; the <br /> leading defects mycobacterial cell wall, targets         GRAVY value secretion signal 20.15 and <br /> of antimycobacterial action unknown. Though lines              20.29, respectively, implying moderate solubility, indeed <br /> of evidence circumstantial, taken genetic               experimentally membrane fraction [20,21]. <br /> data, findings support role Rv3802 mycobacterial <br /> mycolic acid synthesis.                                                       Expression purification recombinant proteins M. <br />                                                                               tuberculosis Rv3802, mutants, M. smegmatis <br /> Results                                                                       mc2155 MSMEG_1403 E. coli <br />                                                                                  The genes Rv3802c MSMEG_1403 PCR-amplified <br /> Genome comparisons Rv3802c Locus                                       respective genomes secretion signals and <br />    We analyzed 30-kb chromosomal region surrounding Rv3802c                 placed control IPTG inducible T7 promoter in <br /> in M. tuberculosis H37Rv compared                     expression vector pET23a. The vector derived 66 His tag was <br /> mycobacteria corynebacteria. Rv3802c located mycolic              used lieu engineered streptavidin tag, expression <br /> acid synthesis gene cluster Rv3799c 3804c, encode               recombinant strain verified using anti-streptavidin <br /> AccD4, Pks13, FadD32, Rv3802, Ag85D Ag85A, respectively                   antibody (Figure 4). The product MSMEG_1403 the <br /> (Figure 1). The cluster conserved mycobacteria,                     ortholog Rv3802, homologous protein purified <br /> corynebacteria, nocardia, rhodococci available annotated             M. smegmatis culture supernatant PLA activity [22]. <br /> databases, indicative functionality common actinomycetes. Of            Though focus manuscript, include information <br /> the seven cutinases present M. tuberculosis, Rv3802c       MSMEG_1403 confirms cutinases have <br /> reported essential, conserved M. leprae,         PLA TE activity, novel activities class enzymes. In <br /> which considered minimal genome [12,13]. Essentiality                addition, demonstrates non-orthologous cutinase, <br /> determined Himar-1-based mutagenesis experiments M.                   MSMEG_1403, inhibited THL, significant <br /> tuberculosis Mycobacterium bovis BCG, Rv3802c                  growth M. smegmatis culture inhibited by <br /> insertion/genomic probe ratio 0.08 (cut-off essentiality               THL [23]. Amino acid changes Rv3802 serines <br /> experimentally set 0.2) [12]. However, screen           86, 87 175. Each changed glutamic acid. Ser175 is <br /> and essential nature gene needs confirmed              predicted catalytic serine. Either Ser86 Ser87 is <br /> deletion/complementation studies. The upstream genes                    predicted stabilize oxyanion hole, based publications <br /> encoding AccD4, Pks13 FadD32 proteins similar                    F. solani cutinase Ser42; mutated goal of <br /> insertion/genomic probe ratios (0.02&#226;&#8364;&#8220;0.16) confirmed              finding partially active mutant [24]. Solubility increased <br /> be essential M. tuberculosis M. smegmatis [14,15]. The amino           use sarcosyl membrane disruption. Purity assessed <br /> acid sequence Rv3802 highly similar orthologs            95% SDS polyacrylamide gel stained a <br /> other mycobacteria (76&#226;&#8364;&#8220;100%, Figure 3) 60% similar              technique sensitive 10 ng protein (Figure 4). Notably, <br /> ortholog C. glutamicum; shares 0&#226;&#8364;&#8220;40% similarity           MSMEG_1403 electrophoreses larger gel Rv3802, <br /> other annotated cutinases M. tuberculosis. Rv3802 337 amino           Rv3802 predicted 32.5 kD MSMEG_1403 <br /> acid protein predicted secretion signal altered,            30 kD. Reasons unclear, consistent finding. <br /> recognizable, cutinase motif [16]. The motif contains classic S,          In addition, MSMEG_1403, dimers trimers as <br /> D, H catalytic triad, Rv3802 orthologs probable             monomers visible. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                           January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                           M. tb Rv3802 Inhibited THL <br />  <br />  <br />  <br />  <br /> Figure 2. Approximate schematic proposed mycolic acid synthesis. Fas I enzyme FasII elaborate mero chains for <br /> nascent mycolic acids. Carboxylation AccD4, AccA3 AccD5 activation FadD32 convert respective acyl-S-CoA acyl-S- <br /> AMP forms. The AT (acyltransferase) domain Pks13 attaches tioester bond phosphopentathiene-modified ACP domains, and <br /> facilitates transfer KS (ketosynthase) domain. Via Claisen-type condensation reduction, chains joined form mature mycolic <br /> acid attached thioester Pks13, loss CO2. The mature mycolic acid hydrolyzed Pks13 TE1 domain Pks13, or <br /> an external TE. An external TE2 function unclog Pks13 mis-acylated. The liberated mycolic acid transferred lipid carrier, <br /> such Myc-PL, unkown MT (mycolyltranferase); facilitate transfer plasma membrane. Eventually tranferred by <br /> another MT TMM. TMM used mycolic acid donor acceptors TMM, form TDM, AG, form mAG. Double-boxed enzymes <br /> indicate step involves proteins encoded genes mycolic acid synthesis gene cluster Rv3802c. Question marks represent activities <br /> encoded unknown enzymes possible role Rv3802. Adapted references [7,13,36,44]. <br /> doi:10.1371/journal.pone.0004281.g002 <br />  <br />  <br /> Activity purified proteins M. tuberculosis Rv3802                       event, PLA activity necessary process, we <br /> M. smegmatis mc2155 MSMEG_1403 various                                      investigated PLA activity purified Rv3802 and <br />                                                                                MSMEG_1403. Of note, described PLA activity previously, <br /> substrates <br />                                                                                partially purified native protein purified <br />    In order understand location cell wall pathway,               exogenous protein [22]. To investigate ability Rv3802 and <br /> various substrates activities explored. This included                 MSMEG_1403 hydrolyze host mycobacterial phospholip- <br /> assays esterase, lipase, phospholipase, thioesterase                   ids, hydrolysis 14C-radiolabeled phospholipids assessed by <br /> mycolyltransferase activities.                                                 TLC. Rv3802 MSMEG_1403 active PLA-type <br />    Phospholipids integral host membranes                   fashion (hydrolysis respective lysophospholipid fatty acid) <br /> mycobacterial cell wall, known exogenous                        phosphatidylcholine, phosphatidylethanolamine, phospha- <br /> radiolabeled 14C-phospholipids incorporated cell wall             tidylserine, sphingomyelin (Figure 5). The enzymes were <br /> [25,26]. In theory, fatty acid incorporation          able hydrolyze palmitic acid arachidonic acid <br /> FAS-II pathway, complete catabolism occur. In                  containing phospholipids. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          3                             January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                         M. tb Rv3802 Inhibited THL <br />  <br />  <br />  <br />  <br /> Figure 3. Clustal W alignment Rv3802 orthologs mycobacteria demonstrating high conservation. Cutinase motif boxed <br /> with asterisk putative catalytic site amino acids. <br /> doi:10.1371/journal.pone.0004281.g003 <br />  <br />  <br />  <br />  <br /> Figure 4. Products Rv3802c MSMEG_1403 expression E. coli. Left: polyacrylamide gel visualized Sypro OrangeTM. Right: <br /> immunoblot streptavidin antibody. <br /> doi:10.1371/journal.pone.0004281.g004 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                4                         January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                                   M. tb Rv3802 Inhibited THL <br />  <br />  <br />  <br />  <br /> Figure 5. Hydrolysis various phospholipid substrates F. solani cutinase (FS), Snake venom PLA (SV), MSMEG_1403 and <br /> Rv3802, demonstrating hydrolysis SM. Thin layer chromatography using 14C-labeled DPPC = dipalmitoyl phosphatidylcholine <br /> labeled fatty acids (FA), AAPC = phosphatidylcholine 14C-arachidonic acid sn-2, SM = sphingomyelin 14C-labeled headgroup, <br /> PE = phosphatidylethanolamine 14C-labeled sn-2 FA, PS = phosphatidylserine 14C-labeled headgroup. Inset equivalent aqueous phase of <br /> phosphatidylserine assay. <br /> doi:10.1371/journal.pone.0004281.g005 <br />  <br />    Because proteins annotated cutinases, hydrolysis                 mannose-6-phosphate glycerol-6-phosphate, observed <br /> nitrophenyl butyrate (NPB) investigated surrogate                      acylated product. These acceptors used donor <br /> cutinase activity. This essentially esterase activity                    changed 14C-P-CoA (thioester bond) 14C-DPPC (ester bond), <br /> colorimetric detection nitrophenyl group freed NPB.                     acylation occurred, hydrolysis donors <br /> Both enzymes active esterases, turnover rates                       did occur. 14C-P-CoA 14C-DPPC used donors, <br /> were relatively slow (Table 1).                                                     crude cell wall extract (lipids proteins, ATP added) <br />    Though Rv3802 function hydrolysis only, evaluation                    used acceptor; new acylated products emerged. These <br />                                                                                     14 <br /> the mycolic acid synthesis pathway suggests possible role                       C-P-CoA 14C-DPPC assays repeated addition <br /> transferase (Figure 2). Rv3802 Antigen85A (Rv3804), known                     crude cell wall extract cytosolic extract, the <br /> mycolyltranferase, a/b-hydrolase fold enzymes. The acyl                    hope native donor acceptor present, again, <br /> binding pocket Ag85A functions transfer second mycolic                    transacylation observed. Transacylation assays were <br /> acid TMM TDM. We investigated                                repeated times different amounts donor, acceptor <br /> acyltransferase activity Rv3802 MSMEG_1403. Various                          enzyme, different amounts time 1 72 H. The <br /> acyl donors paired acceptors enzyme assays. First,                  result Rv3802 MSMEG_1403 assays the <br /> investigate transacylation trehalose (similar Ag85) 14C-                      release fatty acid (not shown). <br /> trehalose used acceptor, cell wall lipids crude cell                 As global cell wall assay, M. smegmatis mc2 155 cell wall <br /> wall extract (with lipid protein) used donation;                    radiolabeled 14C-acetate, outer cell wall was <br /> transacylation detected positive control,                       extracted methods. This labeled cell wall assayed with <br /> Ag85C (data shown). T6P currently commercially                           Rv3802 MSMEG_1403 protein, diminution labeled <br /> available, similar assay acceptors                         lipid appearance new lipid similar retention <br />  <br />  <br />  Table 1. Kinetics Rv3802 Rv3802 S87DE mutant. <br />  <br />  <br />  Enzyme Substrate assayed              Km, mM                Vmax Moles[S] min21 mg21              Kcat sec21               Specificity constant M21 sec21 <br />  <br />  Rv3802                                    23.52+/23.78          1.62+/20.0861028                      8.81+/20.3161023         0.37+/20.01 <br />  NPB <br />  Rv3802                                    12.91+/22.53          8.76+/20.5561029                      4.77+/20.2861023         0.37+/20.02 <br />  S87DE <br />  NPB <br />  Rv3802                                    0.017+/20.003         1.35+/20.0461027                      7.33+/20.6261022         37.00+/211.08 <br />  Palmitoyl-S-CoA <br />  Rv3802                                    2.28+/21.06           1.11+/20.2561027                      8.45+/22.5361022         37.00+/211.08 <br />  Decanoyl-S-CoA <br />  <br />  Rv3802 S86DE S175DE inactive assays, snake venom PLA2. NPB substrate represents ester bond, acyl-CoA substrates represent thioester <br />  bonds. <br />  doi:10.1371/journal.pone.0004281.t001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               5                               January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                        M. tb Rv3802 Inhibited THL <br />  <br />  <br />                                                                               CoA) measured thioesterase activity, NPB for <br />                                                                               esterase activity (see Table 1). These kinetic values similar to <br />                                                                               obtained thioesterases. While Rv3802 appears to <br />                                                                               favor thioester bonds ester bonds considering kinetic <br />                                                                               values, difference acyl chain <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> account this, as <br />                                                                               Rv3802 higher affinity hexadecanoyl decanoyl <br />                                                                               thioesters. A comparison similarly acylated thioesters and <br />                                                                               esters interest, limited available <br />                                                                               substrates solubility compounds, nitrophenyl esters <br />                                                                               soluble aqueous solvents CoA thioesters, thus <br />                                                                               end comparing soluble substrate micellar <br />                                                                               substrate, influence results [29]. Rv3802 active <br />                                                                               acetyl-CoA. The Rv3802 mutants S175DE S86DE were <br />                                                                               essentially inactive, S87DE mutant partially active in <br />                                                                               assays (Table 1). Snake venom PLA2 active either <br />                                                                               colorimetric esterase thioesterase assays (data shown). <br />  <br />                                                                               Inhibition M. tuberculosis Rv3802 non-inhibition of <br /> Figure 6. Hydrolysis PIM purified waters column. Dot                  M. smegmatis mc2155 MSMEG_1403 THL <br /> over O represents origin. Lane 1 purified PIM control. Lanes 3 4          THL reported inhibit thioesterase domain human <br /> contain PIM plus Rv3802 MSMEG_1403, respectively.                         FAS, investigated ability THL inhibit M. tuberculosis <br /> doi:10.1371/journal.pone.0004281.g006                                         Rv3802. Inhibition PLA activity demonstrated Figure 7. The <br />                                                                               kinetics inhibition performed using thioesterase assay. <br /> factor (Rf) known 14C-palmitic acid resulted. Mass spectrometry            Complete inhibition seen THL concentration 3.125 mM, a <br /> of apparent substrate revealed phosphatidylinositol                 molar excess 6.7:1 inhibitor enzyme. The apparent Ki and <br /> mannoside 2 (PIM2); purified normal phase                         IC50 THL Rv3802 5 250 nm, respectively, <br /> chromatography silica column, hydrolysis Rv3802                   lower human FASTE, the <br /> and MSMEG_1403 confirmed (Figure 6). Based Rf                      apparent Ki IC50 reported 100 nm 1.3560.34 mM, <br /> solvent, PIM hydrolysis product consistent fatty acid,          respectively [30]. It significantly better 200:1 molar <br /> including tuberculostearic acid[27]. This result surprising           excess needed partial inhibition reported only <br /> given demonstrated enzymes possess PLA                reported mycobacterial target THL, Rv0183[30&#226;&#8364;&#8220;32]. Interest- <br /> activity.                                                                     ingly, THL did inhibit MSMEG_1403 THL concentra- <br />    We investigated thioesterase activity enzymes                 tions 25 mM, 500 M excess. <br /> because possibility Rv3802 orthologs hydrolyze <br /> the thioester bond nascent mycolic acids Pks13                   Discussion <br /> terminal event synthesis or, alternatively, Rv3802c gene <br /> product &#226;&#8364;&#732;&#226;&#8364;&#732;proof reading&#226;&#8364;&#8482;&#226;&#8364;&#8482; thioesterase. This assay uses              In report described heterologous expression, <br /> acyl-CoA thioester substrate, colorimetric detection           purification enzymatic characterization annotated <br /> TNB22 formed DTNB reacts hydrolyzed product                     cutinase, Rv3802, mammalian pathogen. Though the <br /> HS-CoA[28]. Both M. tuberculosis Rv3802 M. smegmatis                      proteins encoded M. tuberculosis cutinases, including Rv3802, <br /> MSMEG_1403 protein active thioesterases (Table 1).                    immunologically characterized, function in <br />    For kinetic analysis Rv3802, time-dependent                     organisms encounter cutin elucidated <br /> hydrolysis palmitoyl CoA (P-CoA) decanoyl CoA (D-                      [33]. We previously attributed mycobacterial PLA activity the <br />  <br />  <br />  <br />  <br /> Figure 7. TLC demonstrating thioesterase activity Rv3802 14C-labeled P-CoA inhibition THL, structure above <br /> (adapted DrugBank, http://www.drugbank.ca/drugs/DB01083). Lanes: 1) 14C-P-CoA alone, 2&#226;&#8364;&#8220;8) serially dilutions decreasing the <br /> amount THL 12.5&#226;&#8364;&#8220;0 mM THL 3.0 mg Rv3802. Origin indicated O, start substrate indicated P-CoA, hydrolysis product <br /> indicated FA (fatty acid). <br /> doi:10.1371/journal.pone.0004281.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                            January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                     M. tb Rv3802 Inhibited THL <br />  <br />  <br /> cutinases, manuscript additionally ascribe thioesterase         required pathway, similarities known <br /> activity. Logical potential uses activities mammalian        mycolyltranferase Ag85 proteins Rv3802 hard ignore. <br /> pathogen include host lipid hydrolysis carbon scavenging,            However, unusual ability hydrolyze thioesters strongly <br /> mycobacterial cell wall construction/remodeling. As               suggests type interaction nascent mycolic acids while <br /> seven annotated cutinases M. tuberculosis, serve        docked thioester bonds Pks13. <br /> variety functions. The function Rv3802c orthologs             Interaction Pks13 form exogenous <br /> suggested location gene cluster mycolic acid                type 1 thioesterase, (TE1) exogenous type 2 thiosterase <br /> synthesis. In addition, importance Rv3802c supported           (TE2) [37]. Type I polyketide synthases (PKSs) modular, <br /> its putative essentiality, ubiquitous presence actinomycetes,        &#226;&#8364;&#732;&#226;&#8364;&#732;assembly line&#226;&#8364;&#8482;&#226;&#8364;&#8482;-like enzymes responsible steps the <br /> and unique conservation minimal genome M. leprae.             biosynthesis polyketides mycolic acids [38]. Pks13 M. <br /> To date, genes surrounding Rv3802c demonstrated           tuberculosis necessary modules produce mature <br /> serve vital functions mycolic acid synthesis. This genetic data          mycolic acid: acyltranferase domain loading transfer <br /> implying role mycolic acid synthesis supported           selected acyl chains phosphopentatheine (PP) binding <br /> Rv3802&#226;&#8364;&#8482;s: (i) membership a/b-hydrolase fold family              motif acyl-S-carrier protein (ACP) domain; ACPs for <br /> known PLAs, mycolyltransferases thioesterases; (ii) possession          thioester binding, PP, meroacyl chain FadD32 <br /> of thioesterase activity; (iii) enzymatic inhibition known         2 carboxyl-C26 chain AccD4, AccA3 AccD5; a <br /> mycolic acid synthesis inhibitor, THL.                                      ketoacyl synthase domain Claisen-type condensation; one <br />    The compound THL sold counter weight loss             TE1 domain chain termination hydrolysis. The reduction <br /> agent AlliTM. It inhibits pancreatic lipase long chain fatty        event form mature mycolic acid catalyzed more <br /> acids absorbed. During activity based screening, THL              protein, date CmrA identified[8]. The <br /> was noted selective inhibitor TE domain human             mature mycolic acid exported transferred the <br /> FAS [30]. FAS modular protein like Pks13. FAS little effect        mycolate containing components mycobacterial cell wall, <br /> on normal cells dietary lipids but, tumor           unknown mycolyltranferases. However, proposed <br /> cells, FAS upregulated contributes angiogenesis;             TE1 C terminal domain Pks13 does liberate <br /> THL currently lead compound development                      mature mycolic acid presence free mycolic acid <br /> anticancer drugs [28,34]. This development focusing creating          unwieldy cell [7]. A solution problem to <br /> more absorbable specific forms[30,35]. The published                    external TE1 mycolyltranferase activity. In theory, <br /> apparent Ki IC50 THL human FASTE                        Rv3802 role, admittedly clear <br /> 100 nm 1.3560.34 mM, respectively[36]. THL                     precedent literature. <br /> demonstrated inhibit M. tuberculosis growth ,30 mg/mL,                   An alternative role protein precedented <br /> decreasing production mycolic acids concomitant                 TE2. TE2s described association Pks NRPS <br /> defects cell wall, target(s) undefined [23].         modules bacteria, fungi plants, serve proofread the <br /> Possible targets THL, including diacylglycerol acyltransfer-         nascent product synthetic module [39]. They specific <br /> ase Rv3130 FAS-I II pathways investigated,             corresponding ACP[40,41]. Without functioning TE2, <br /> none identified target[23]. A monoglyceride lipase,               erroneous acyl chains Pks module clog machinery, <br /> Rv0183, later identified possible target, 200:1 molar        decreasing product production 90% [38]. If Pks <br /> excess inhibitor enzyme needed effect[32]. Based        product secondary metabolite, diminution crucial. <br /> on knowledge Rv3802 a/b-hydrolase fold TE               However, case mycolic acids, diminution could <br /> activity, structure THL, investigated ability          render bacterium non-viable. <br /> THL inhibit recombinant Rv3802 inhibit                   A role TE1 TE2 consistent the <br /> nanomolar concentrations, described. The mechanism                    putative essentiality Rv3802c decrease mycolic acids <br /> inhibition assumed similar mechanism human               observed THL. Significant amino acid similarity between <br /> FASTE inhibition. For human FASTE, THL demonstrated                   TE1s TE2s cutinases search programs, yet <br /> inhibit competitively, palmitic core THL fitting       TE1s, TE2s cutinases a/b-hydrolase fold enzymes <br /> hydrophobic substrate binding pocket hexanoyl tail packing          SDH motifs. In Rv3802, shares little amino <br /> against catalytic histidine, significantly delaying hydrolysis          acid similarity cutinases (0&#226;&#8364;&#8220;40%), spacing SDH is <br /> (Figure 7) [31,34]. Eventually hydrolyzed, inhibition         similar thioesterases cutinases; structure prediction <br /> reversible. The inhibition enzyme gene located        models structural similarity Rv3802 known cutinases <br /> mycolic acid synthesis gene cluster, previously               thioesterases 100% 75% precision, respectively <br /> published observations THL affects cell wall decreases         (Figure 8) [42]. <br /> mycolic acids, supports cause effect relationship THL            Though multiple lines evidence involvement in <br /> and Rv3802. Although demonstrated THL inhibits Rv3802               mycolic acid synthesis, drawback studies data is <br /> in assays, does confirm Rv3802 primary target       indirect; evaluation underway, complexity will <br /> THL culture. It likely THL inhibits enzymes. In        time. The genetic location Rv3802c the <br /> case, knowledge THL inhibits Rv3802 gives               thioesterase activity gene product likely be <br /> framework exploring inhibition Rv3802 THL                  coincidental, inhibition THL supports this <br /> targets M. tuberculosis.                                         hypothesis. However, alternative role interacting host or <br />    Though investigation required pinpoint            mycobacterial lipids Rv3802 mycobacterial <br /> exact function Rv3802, examination mycolic acid                   annotated cutinases possibility. These enzymes clearly possess <br /> synthetic pathway suggests role Pks associated thioesterase          PLA activity, able hydrolyze mycobacterial <br /> or mycolyltransferase. Though demonstrate                 phospholipid PIM. If phospholipid carrier <br /> mycolyltransferase activity, data limited need use         involved mycolic acid synthesis, host derived fatty <br /> surrogate substrates acceptors, remains distinct            acids directly FAS-II pathway, PLA activity may <br /> possibility. There multiple unknown mycolyltransferases                 actually linked mycolic acid synthesis [7,10]. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        7                           January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                       M. tb Rv3802 Inhibited THL <br />  <br />  <br />  <br />  <br /> Figure 8. Structure prediction Rv3802 using Phyre server (http://www.sbg.bio.ic.ac.uk/,phyre/, Bennett-Lovsey et al, 2008). <br /> Model left predicts cutinase 100% precision; modeling right predicts thioesterase 70% precision. Black arrow represents location <br /> of catalytic serine 175. <br /> doi:10.1371/journal.pone.0004281.g008 <br />  <br />  <br /> Conclusions                                                                  Expression purification recombinant proteins M. <br />    Our studies suggest M. tuberculosis Rv3802, previously annotated          tuberculosis Rv3802, mutants, M. smegmatis <br /> as cutinase, involved mycolic acid synthesis. Activity            mc2155 MSMEG_1403 E. coli <br /> either mycolyltransferase Pks associated thioesterase                The constructs pET23a described transformed into <br /> logical roles Rv3802, consistent genetic                electrocompetent Rosetta 2 (DE3)pLysS expression using T7 <br /> location, discovered thioesterase activity, inhibition THL,       promoter. Cultures induced 1 mM isopropyl-B-D- <br /> and previously published cell wall defects observed use          thiogalactopyranoside (IPTG) 3 H 37uC. Following induction, <br /> THL antimycobacterial compound. However, evidence                 cell pellets collected re-suspended 5 mL Strep-tactin lysis <br /> is circumstantial direct evidence needed support           buffer (50 mM NaH2PO4 300 mM NaCl, pH 8.0 1 mg/ <br /> or refute role Rv3802.                                            mL lysozyme 5 mg/mL DNase I) 200 mL original induction <br />                                                                              culture, stored 220uC overnight. Thawed resuspensions with <br /> Materials Methods                                                        2% (w/v) n-lauryl sarcosine mixed $60 min room <br />                                                                              temperature lysate clear large clumps. After centrifuga- <br /> Bacterial strains culture conditions <br />                                                                              tion, target protein purified lysate using strep-tactin <br />   M. smegmatis mc2 155 (ATCC 700084) purchased from <br />                                                                              Superflow resin (IBA, Germany) company protocol. Protein <br /> ATCC. M. smegmatis grown 37uC Luria-Bertani (LB) <br />                                                                              concentration absorbance spectra aliquots were <br /> broth 0.05% Tween80H. Escherichia coli strains DH5a and <br /> Rosetta 2 (DE3)pLysS (Merck) grown 37uC LB broth.                 evaluated 280 nm NanodropTM system. All eluted protein <br /> Antibiotics (Sigma) added media following                     samples, used immediately, stored 270uC. Selected <br /> concentrations: Ampicillan 100 mg/mL, Kanamycin 25&#226;&#8364;&#8220;50 mg/                    fractions analyzed sodium dodecyl sulfate-polyacryl- <br /> mL, Chloramphenicol 34 mg/mL.                                                amide gel electrophoresis stained immunoblotted using <br />                                                                              anti-streptavidin tag antibody (IBA) 1:1500 secondary <br />                                                                              horseradish peroxidase goat anti-mouse (1:20 K), visualized with <br /> Cloning Procedures                                                           chemiluminescence. Fractions estimated 95% pure <br />    Standard PCR strategies Taq DNA polymerase (Invitro- <br />                                                                              using SYPRO orange staining (molecular probes S5692, per <br /> gen) used amplify M. tuberculosis Rv3802c M. <br />                                                                              protocol, sensitive 5 ng protein) phosphoimager detection. <br /> smegmatis M_SMEG 1403 genes. PCR amplifications consisted of <br /> one cycle denaturation (95uC, 10 min), followed 30 cycles of <br /> amplification included denaturation (95uC, 1 min), annealing            Whole Cell Radiolabeling Experiments <br /> (65uC, 1 min), primer extension (72uC, 1 min). The primers                  For radiolabeling M. smegmatis mc2155 cells 14C, <br /> used follows: 59-catatgggatgccctgacctgcactggatcggc-39            M. smegmatis grown LB medium supplemented 0.05% <br /> 59-gagctctcatttttcgaactgcgggtggctccaagcgctgcctaacgtcggagcg-                  TweenH 80 and, log phase, 14C-acetate (ARC 0173B) was <br /> ggctc-39 M. smegmatis MSMEG_1403 gene, 59-catatgggc-                 added 1 mCi/mL cells grown additional H. <br /> gaatcgccgcccagcgcgg-39 59-gagctctcatttttcgaactgcgggtgg-                  Cells pelleted, extracted 30 mL <br /> ctccaagcgctcctatgtttggggtggggcgc-39 M. tuberculosis Rv3802c              CHCl3:CH3OH (1:2, v/v) gram wet cells. Debris was <br /> gene. Primers designed provide PCR amplified fragments               pelleted, extract removed, re-extracted same <br /> containing C-terminal streptavidin tag (bold) flanking NdeI            volume CHCl3:CH3OH (2:1, v/v). Extracts combined and <br /> and SacI restriction sites (underline). Fragments cloned           Folch washed. Organic layer aliquoted dried pre- <br /> PCR2.1 (Invitrogen) colonies subsequent cloning                 weighed vials. For mass spectrometry, parallel experiment was <br /> confirmed sequencing. Correct clones sub cloned               14C-acetate. To isolate PIM species, 400 mg was <br /> pET23a (Novagen). This vector used template make                 resuspended CHCl3 applied silica column (Waters, <br /> three mutants M. tuberculosis Rv3802c QuickChangeH II             WAT023537) eluted CHCl3:CH3OH, percentages (v/ <br /> method (Stratagene, 2005245, manufacturer protocol).                     v) 90:10, 80:20, 70:30, 60:40 50:50. PIMs collected <br /> Primers used 59-ctgatcgggtttgagcagggcgcggtg-39 59-cacc-             60:40 50:50 fractions. <br /> gcgccctgctcaaacccgatcag-39 Rv3802 DS175E, 59-ggaacctgg- <br /> gaggaatcgccgcagcag-39 59-ctgctgcggcgattcctcccaggttcc-39              Mass spectrometry extracted cell wall lipids <br /> Rv3802 DS86E, 59-cctgggagtcagagccgcagcagaaccc-39 59-                    Cell wall lipids isolated above. Lipids applied to <br /> gggttctcgtgcggctctgactcccagg-39 Rv3802 DS87E.                            Silica gel 60A &#203;&#353; plates (Whatman, 4807-425), developed in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        8                            January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                        M. tb Rv3802 Inhibited THL <br />  <br />  <br /> CHCl3:CH3OH:NH4OH (80:20:2, v/v/v). Individual bands                      min. The organic phase plated 60 A       &#203;&#353; silica plates <br /> lipid scraped plate columns Pasteur             (Whatman 4807-425) developed CHCl3:CH3OH:H2O <br /> pipettes fitted 0.2 mm, Type FG Fluoropore                    (v/v/v, 14:6:1) solvent. Lipids visualized phosphoimager. <br /> membrane filters (Millipore). The silica gel rinsed 1 mL <br /> of 2:1 i-C3H7OH/H2O lipid species eluted 1 mL               Thioesterase, esterase, transferase activity of <br /> 2:1 (v/v) CHCl3/CH3OH. Samples evaporated dryness,                   recombinant M. tuberculosis Rv3802, mutants, and <br /> re-dissolved 2 mL solvent 0.5 mL deposited <br />                                                                              M. smegmatis mc2155 MSMEG_1403, without <br /> onto sample stage pre-spotted 2,5- <br /> dihydroxybenzoic acid (DHB) analysis matrix assisted laser            inhibitor THL <br /> desorption ionization (MALDI) mass spectrometry. Mass spectra                   Control enzymes included F. solani cutinase, general esterase <br /> using laser wavelength 337 nm obtained m/z 100                Thermomyces lanuginosus (Sigma L-0777) M. tuberculosis Ag85C <br /> to 2400 using QSTAR XL tandem mass spectrometer (Applied                   (Tuberculosis Research Materials Vaccine Testing, Colorado <br /> Biosystems/MDS Sciex, Thornhill, Ontario, Canada). The mass                  State University). <br /> analyzer configuration instrument, mass-resolving quadru-               Nitrophenyl ester assays. hydrolysis NPB (Sigma, <br /> pole &#226;&#8364;&#8220; collision quadrupole &#226;&#8364;&#8220; orthogonal time-of-flight (Q-q-                N9876) used surrogate cutinase activity. Though <br /> oTOF), allowed collision induced dissociation (CID)            M. smegmatis MSMEG_1403 M. tuberculosis Rv3802 were <br /> spectra obtained. PIM species identified based                 active assay, kinetics Rv3802. For <br /> molecular weight, interpretation CID spectra, comparison              kinetic calculations, 200 mL reaction contained 0.01 mg of <br /> of spectra data literature [43,44].                        enzyme (1.53 mM), path length 0.8 cm extinction <br />                                                                              coefficient 15250 M21 cm21[45]. The NPB 5.69 M stock was <br />                                                                              diluted 50 mM bis-tris/5% glycerol (pH 7.0) use the <br /> Cell-free Assay M. smegmatis crude cell wall and <br />                                                                              assays. The final substrate concentration ranged 1.56 mM&#226;&#8364;&#8220; <br /> purified PIM components                                                      200 mM. Assays 96 plates, triplicate, 37uC <br />    Aliquots 14C-radiolabeled cell wall purified PIM               H, proper controls. There replicate <br /> resuspended 2 mg/mL 50 mM Bis-tris, pH 7, 0.25%                   experiments. Absorbance read plate spectrophotometer <br /> TritonH-X 100. 50 mL added 10&#226;&#8364;&#8220;50 mg enzyme              415 nm various time points. Prizm 5.0 software used to <br /> 50 mL, total reaction volume 100 mL. After 2&#226;&#8364;&#8220;5 H,               determine Km Vmax used determine kcat <br /> reaction quenched 200 mL CHCl3:CH3OH:HCl                         specificity constant esterase activity Rv3802 and <br /> (50:50:0.3, v/v/v), organic phase removed plated                 mutants. Up 10 units snake venom PLA tested and <br /> under nitrogen stream Silica gel 60A&#203;&#353; plates (Whatman, 4807- <br />                                                                              inactive. <br /> 425). Plates developed CHCl3:CH3OH:NH4OH (80:20:2,                      Thioesterase activity assays. hydrolysis P-CoA (Avanti, <br /> v/v/v) visualized phosphoimager (radiolabel) and/or               870716) D-CoA, (Sigma, D5269) used measure <br /> cupric sulfate Dittmer-Lester stains (radiolabeled unla-              thioesterase activity. When thioester bond hydrolyzed, the <br /> beled). For mass spectrometric analysis, parallel experiment           free sulfur CoA attacked 5,59-dithio-bis(2-nitrobenzoic <br /> performed radiolabel lipid species recovered             acid) (DTNB), releases measurable nitrophenyl group, 5- <br /> described above.                                                             thio-2-nitrobenzoate (TNB22). For kinetic calculations, each <br />                                                                              200 mL reaction contained 3 mg enzyme (461 nm), path <br /> Determination activity recombinant proteins M.                         length 0.8 cm extinction coefficient was <br /> tuberculosis Rv3802, mutants, M. smegmatis                           14150 M21 cm21 TNB22 (Sigma). The P-CoA 20 mM stock <br /> mc2155 MSMEG_1403 phospholipid substrates                           diluted 50 mM bis-tris (pH 7.0) use assays. The <br />   Control enzymes. 6 units snake venom PLA2 (Naja                         DTNB 40 mM stock diluted EtOH molar <br /> mossambica mossambica, Sigma P-7778) used reaction              equivalent DTNB acyl-CoA reaction <br /> a PLA activity control. Lyophilized F. solani cutinase (kind gift       (Increasing DTNB relative P-CoA did increase reaction <br /> Unilever Research Development, Vlaardingen, The                          rate, DTNB did react enzyme alone). The final P- <br /> Netherlands) resuspended 3 mg/mL H2O                      CoA concentration ranged 4.875 mM &#226;&#8364;&#8220; 2.5 mM, the <br /> further diluted depending assay.                                          final D-CoA concentration ranged 0.078 5.0 mM. Assays <br />   Phospholipid assays. Unlabeled dipalmitoyl phosphatidyl-                   96 plates, triplicate, 37uC H, with <br /> choline (DPPC) resuspended 23 mM chloroform,                   proper controls. There replicate experiments. <br /> used micellar backbone reactions. Radiolabeled phos-             Absorbance read plate spectrophotometer 415 nm <br /> pholipids follows: phosphatidylcholine L-a-dipalmitoyl               various time points. Prizm 5.0 software used determine <br /> [dipalmitoyl-1-14C] (NEC682, 110 mCi/mM), bovine sphingo-                    Km Vmax used determine kcat and <br /> myelin [choline-methyl-14C] (NEC663, 52 mCi/mM), phos-                   specificity constant thioesterase activity Rv3802. For <br /> phatidylethanolamine, L-a-palmitoyl-2-arachidonyl [arachidonyl-              radiolabel 14C-P-CoA assays, protocol followed 14C- <br /> 1-14C] (NEC-783, 48 mCi/mM), NEN Life Science                       phospholipids assays above. <br /> Products, 3-phosphatidyl-[3-14C]serine 1,2-dioleoyl (CFA757,                THL inhibition assays. concentrations THL (Sigma, <br /> 54 mCi/mM) Amersham Biosciences. Cold DPPC                         04139) selected order determine Ki nature of <br /> 0.1 mL 14C-radiolabeled phospholipid reaction dried              inhibition Rv3802, based pilot experiments: 1.56 mM, <br /> down resuspended 50 mL 50 mM bis-tris pH 7.4                0.78 mM, 0.39 mM, 0.195 mM. 100% inhibition activity <br /> buffer 0.25% TritonH-X 100 reaction make mixed                   seen 3 mM final concentration. Inhibition assays were <br /> micelles. This solution sonicated 30 s63, vortexing                   performed identically thioesterase assays inhibitor <br /> between. 50 mL substrate added 50 mL                  described previously, addition THL and <br /> enzyme solution (10&#226;&#8364;&#8220;50 mg), incubated 37uC H. The               addition P-CoA, assay incubated 37uC for <br /> reaction extracted 100 mL chloroform/methanol/HCl                30 min [39]. The Reaction run 1&#226;&#8364;&#8220;2 H, duplicate, and <br /> (50:50:0.3, v/v/v), vortexed centrifuged 13,000 rpms              replicates performed. Changes Km Vmax <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                             January 2009 | Volume 4 | Issue 1 | e4281 <br />                                                                                                                                                  M. tb Rv3802 Inhibited THL <br />  <br />  <br />                                                                                               14 <br /> (compared non-inhibited Rv3802) determined using Prizm                                 C-DPPC instead 14C-P-CoA. As positive control the <br /> 5.0 software, apparent Ki IC50 values. For inhibition                             mycolyltranferase assays using 14C-trehalose acceptor, <br /> of M. smegmatis MSMEG_1403, 200 mL reaction contained                                    Ag85C used 10&#226;&#8364;&#8220;50 mg assay. <br /> 0.3 mg (0.495 mM). THL concentration increased a <br /> maximum 25 mM (a molar excess compared enzyme                                        Acknowledgments <br /> 505) attempt detect inhibition. <br />    Transferase assays. donors recipients paired                                  Cutinase F. solani kind gift Unilever Research and <br /> various transferase assays assess purified enzymes                             Development, Vlaardingen, The Netherlands. We thank Colorado State <br />                                                                                               University NIH, NIAID Contract NO1 AI75320 entitled <br /> could hydrolyze known substrate transfer acceptor. <br />                                                                                               &#226;&#8364;&#732;&#226;&#8364;&#732;Tuberculosis Research Materials Vaccine Testing&#226;&#8364;&#8482;&#226;&#8364;&#8482; M. tuberculosis <br /> Assays included 10&#226;&#8364;&#8220;50 mg Rv3802 MSMEG_1403                                       H37Rv products. We thank researchers Vasil laboratory at <br /> protein, incubated 1&#226;&#8364;&#8220;72 H. Pairings included: 1) 14C-                            Colorado State University help advice. We thank Andrew <br /> trehalose (0.25 mCi) crude cell wall lipids (50 mg) M.                              Tomaras, Pfizer, help cloning. <br /> tuberculosis; 2) 14C-P-CoA (0.25 mCi) donor mannose-6- <br /> phosphate (1&#226;&#8364;&#8220;10 mM) glycerol-6-phosphate (1 mM) acceptor                                Author Contributions <br /> with crude cell wall extract (proteins lipids unknown <br /> cofactors acceptors, total 0.1 1 mg,                                     Conceived designed experiments: SKP RMB. Performed the <br />                                                                                               experiments: SKP RMB JGR. Analyzed data: SKP RMB JGR. <br /> previously described protocol[27]; 3) addition <br />                                                                                               Contributed reagents/materials/analysis tools: SKP RMB MLV. Wrote <br /> of crude cytosolic extract, 4)                                  paper: SKP. <br /> addition 14C-trehalose (0.25 mCi); 5) with <br />  <br /> References <br />  1. Corbett EL, Watt CJ, Walker N, Maher D, Williams BG, et al. (2003) The                    20. Mattow J, Siejak F, Hagens K, Schmidt F, Koehler C, et al. (2007) An improved <br />     growing burden tuberculosis: global trends interactions HIV                   strategy selective efficient enrichment integral plasma membrane <br />     epidemic. Arch Intern Med 163: 1009&#226;&#8364;&#8220;1021.                                                     proteins mycobacteria. Proteomics 7: 1687&#226;&#8364;&#8220;1701. <br />  2. Organization WH (March 2006) Tuberculosis. Fact sheet 104. Geneva,                        21. Kyte J, Doolittle RF (1982) A simple method displaying hydropathic <br />     Switzerland: World Health Organization.                                                       character protein. J Mol Biol 157: 105&#226;&#8364;&#8220;132. <br />  3. Barry CE 3rd (2001) Interpreting cell wall &#226;&#8364;&#732;virulence factors&#226;&#8364;&#8482; Mycobacterium           22. Parker SK, Curtin KM, Vasil ML (2007) Purification characterization of <br />     tuberculosis. Trends Microbiol 9: 237&#226;&#8364;&#8220;241.                                                    mycobacterial phospholipase A: activity associated mycobacterial <br />  4. Kusner DJ (2005) Mechanisms mycobacterial persistence tuberculosis. Clin                cutinase. J Bacteriol 189: 4153&#226;&#8364;&#8220;4160. <br />     Immunol 114: 239&#226;&#8364;&#8220;247.                                                                     23. Kremer L, Chastellier C, Dobson G, Gibson KJ, Bifani P, et al. (2005) <br />  5. Brennan PJ (2003) Structure, function, biogenesis cell wall                     Identification structural characterization unusual mycobacterial <br />     Mycobacterium tuberculosis. Tuberculosis (Edinb) 83: 91&#226;&#8364;&#8220;97.                                   monomeromycolyl-diacylglycerol. Mol Microbiol 57: 1113&#226;&#8364;&#8220;1126. <br />  6. McKinney JD, Honer zu Bentrup K, Munoz-Elias EJ, Miczak A, Chen B, et al.                 24. Egmond MR, Vlieg J (2000) Fusarium solani pisi cutinase. Biochimie 82: <br />     (2000) Persistence Mycobacterium tuberculosis macrophages mice                      1015&#226;&#8364;&#8220;1021. <br />     requires glyoxylate shunt enzyme isocitrate lyase [see comments]. Nature              25. Wheeler PR, Besra GS, Minnikin DE, Ratledge C (1993) Stimulation mycolic <br />     406: 735&#226;&#8364;&#8220;738.                                                                                 acid biosynthesis incorporation cis-tetracos-5-enoic acid cell-wall <br />  7. Takayama K, Wang C, Besra GS (2005) Pathway synthesis processing                    preparation Mycobacterium smegmatis. Biochim Biophys Acta 1167: <br />     mycolic acids Mycobacterium tuberculosis. Clin Microbiol Rev 18: 81&#226;&#8364;&#8220;101.                   182&#226;&#8364;&#8220;188. <br />  8. Lea-Smith DJ, Pyke JS, Tull D, McConville MJ, Coppel RL, et al. (2007) The                26. Wheeler PR, Ratledge C (1988) Use carbon sources lipid biosynthesis in <br />     reductase catalyzes mycolic motif synthesis required efficient                    Mycobacterium leprae: comparison pathogenic mycobacteria. J Gen <br />     attachment mycolic acids arabinogalactan. J Biol Chem 282: 11000&#226;&#8364;&#8220;11008.                 Microbiol 134: 2111&#226;&#8364;&#8220;2121. <br />                                                                                               27. Jackson M, Crick DC, Brennan PJ (2000) Phosphatidylinositol essential <br />  9. Jackson M, Raynaud C, Laneelle MA, Guilhot C, Laurent-Winter C, et al. <br />                                                                                                   phospholipid mycobacteria. J Biol Chem 275: 30092&#226;&#8364;&#8220;30099. <br />     (1999) Inactivation antigen 85C gene profoundly affects mycolate <br />                                                                                               28. Browne CD, Hindmarsh EJ, Smith JW (2006) Inhibition endothelial cell <br />     content alters permeability Mycobacterium tuberculosis cell <br />                                                                                                   proliferation angiogenesis orlistat, fatty acid synthase inhibitor. Faseb J <br />     envelope. Mol Microbiol 31: 1573&#226;&#8364;&#8220;1587. <br />                                                                                                   20: 2027&#226;&#8364;&#8220;2035. <br /> 10. Besra GS, Sievert T, Lee RE, Slayden RA, Brennan PJ, et al. (1994) <br />                                                                                               29. Deems RA (2000) Interfacial enzyme kinetics phospholipid/water <br />     Identification apparent carrier mycolic acid synthesis. Proc Natl Acad <br />                                                                                                   interface: practical considerations. Anal Biochem 287: 1&#226;&#8364;&#8220;16. <br />     Sci U S A 91: 12735&#226;&#8364;&#8220;12739. <br />                                                                                               30. Kridel SJ, Axelrod F, Rozenkrantz N, Smith JW (2004) Orlistat novel <br /> 11. Belisle JT, Vissa VD, Sievert T, Takayama K, Brennan PJ, et al. (1997) Role of <br />                                                                                                   inhibitor fatty acid synthase antitumor activity. Cancer Res 64: <br />     major antigen Mycobacterium tuberculosis cell wall biogenesis. Science <br />                                                                                                   2070&#226;&#8364;&#8220;2075. <br />     276: 1420&#226;&#8364;&#8220;1422. <br />                                                                                               31. Richardson RD, Ma G, Oyola Y, Zancanella M, Knowles LM, et al. (2008) <br /> 12. Sassetti CM, Boyd DH, Rubin EJ (2003) Genes required mycobacterial                        Synthesis novel beta-lactone inhibitors fatty acid synthase. J Med Chem 51: <br />     growth defined high density mutagenesis. Mol Microbiol 48: 77&#226;&#8364;&#8220;84.                          5285&#226;&#8364;&#8220;5296. <br /> 13. Vissa VD, Brennan PJ (2001) The genome Mycobacterium leprae: minimal                 32. Cotes K, Dhouib R, Douchet I, Chahinian H, Caro A, et al. (2007) <br />     mycobacterial gene set. Genome Biol 2: REVIEWS1023.                                           Characterization exported monoglyceride lipase Mycobacterium <br /> 14. Portevin D, De Sousa-D&#226;&#8364;&#8482;Auria C, Houssin C, Grimaldi C, Chami M, et al.                        tuberculosis possibly involved metabolism host cell membrane lipids. <br />     (2004) A polyketide synthase catalyzes condensation step mycolic acid             Biochem J 408: 417&#226;&#8364;&#8220;427. <br />     biosynthesis mycobacteria related organisms. Proc Natl Acad Sci U S A              33. West NP, Wozniak TM, Valenzuela J, Feng CG, Sher A, et al. (2008) <br />     101: 314&#226;&#8364;&#8220;319.                                                                                 Immunological diversity family cutinase-like proteins Mycobac- <br /> 15. Portevin D, Sousa-D&#226;&#8364;&#8482;Auria C, Montrozier H, Houssin C, Stella A, et al.                     terium tuberculosis. Vaccine 26: 3853&#226;&#8364;&#8220;3859. <br />     (2005) The acyl-AMP ligase FadD32 AccD4-containing acyl-CoA                           34. Pemble CWt, Johnson LC, Kridel SJ, Lowther WT (2007) Crystal structure of <br />     carboxylase required synthesis mycolic acids essential                 thioesterase domain human fatty acid synthase inhibited Orlistat. Nat <br />     mycobacterial growth: identification carboxylation product                         Struct Mol Biol 14: 704&#226;&#8364;&#8220;709. <br />     determination acyl-CoA carboxylase components. J Biol Chem 280:                    35. Richardson RD, Smith JW (2007) Novel antagonists thioesterase domain <br />     8862&#226;&#8364;&#8220;8874.                                                                                    human fatty acid synthase. Mol Cancer Ther 6: 2120&#226;&#8364;&#8220;2126. <br /> 16. Finn RD, Mistry J, Schuster-Bockler B, Griffiths-Jones S, Hollich V, et al. (2006)        36. Zhang W, Richardson RD, Chamni S, Smith JW, Romo D (2008) Beta-lactam <br />     Pfam: clans, web tools services. Nucleic Acids Res 34: D24 <br /> </body></html>