<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Yeast v-SNARE Vti1p Mediates Two Vesicle Transport Pathways <br /> through Interactions t-SNAREs Sed5p Pep12p <br /> Gabriele Fischer von Mollard, Steven F. Nothwehr, Tom H. Stevens <br /> Institute Molecular Biology, University Oregon, Eugene, Oregon 97403-1229 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span>. Membrane traffic eukaryotic cells requires                      forms CPY secreted protein invertase <br /> that specific v-SNAREs transport vesicles interact                        nonpermissive temperature. Overexpression of <br /> with specific t-SNAREs target membranes. We                               yeast cis-Golgi t-SNARE Sed5p suppressed ac- <br /> identified novel Saccharomyces cerevisiae v-SNARE                          cumulation ER form CPY did lead <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (Vti1p) encoded essential gene, VTI1. Vti1p in-                       CPY transport vacuole vti1-11 cells. Over- <br /> teracts prevacuolar t-SNARE Pep12p direct                        expression Sed5p allowed growth absence of <br /> Golgi prevacuolar traffic. vti1-1 mutant cells mis-                       Vti1p. In vitro binding coimmunoprecipitation <br /> sorted secreted soluble vacuolar hydrolase car-                      studies revealed Vti1p interacts directly the <br /> boxypeptidase Y (CPY) rapidly reversibly                            t-SNAREs, Sed5p Pep12p. These data suggest <br /> vti1-1 cells shifted restrictive temperature.                    Vti1p plays role cis-Golgi membrane traffic, <br /> However, overexpression Pep12p suppressed                             essential yeast viability, nonessential <br /> CPY secretion defect exhibited vti1-1 cells 368C.                      role fusion Golgi-derived vesicles pre- <br /> Characterization second vti1 mutant, vti1-11, re-                       vacuolar compartment. Therefore, single v-SNARE <br /> vealed Vti1p plays role membrane traffic                      interact functionally different t-SNAREs <br /> at cis-Golgi stage. vti1-11 mutant cells displayed                       directing membrane traffic yeast. <br /> growth defect accumulated ER early Golgi <br />  <br />  <br />  <br />  <br /> E <br />        ukaryotic cells contain numerous membrane com-                        SNARE) provide specific recognition mem- <br />         partments, maintains distinct protein                  branes (Ferro-Novick Jahn, 1994; Rothman, 1994). <br />         lipid compositions specific tasks. Traffick-                 v- t-SNAREs structural similarities. They con- <br /> ing different membrane compartments occurs                       tain single COOH-terminal transmembrane domain and <br /> transport vesicles. Transport proteins secre-                   regions predicted form coiled coil domains. The <br /> tory pathway starts protein translocation en-                  yeast proteins Sec22p (Sly2p), Bet1p (Sly12p), Bos1p, and <br /> doplasmic reticulum (ER)1. Proteins transported                      possibly Ykt6p identified v-SNAREs ER <br /> the intermediate compartment Golgi apparatus,                   Golgi transport (Newman et al., 1990; Ossig et al., 1991; <br /> can subdivided cis-, medial-, trans-Golgi com-                   Sogaard et al., 1994). Their partner t-SNARE Sed5p, <br /> partments. These Golgi compartments character-                      localized Golgi compartment yeast <br /> ized morphological level mammalian cells (Far-                     (Hardwick Pelham, 1992). In addition, Sed5p interacts <br /> quhar Palade, 1981). The Golgi compartments                           v-SNARE Sft1p, localized dis- <br /> yeast defined glycosylation pro-                     tal Golgi compartment involved retro- <br /> cessing enzymes contain (Franzusoff Schekman,                       grade transport (Banfield et al., 1995). Recently, Ufe1p, a <br /> 1989).                                                                       t-SNARE localized ER involved retrograde <br />   The SNARE hypothesis (S&#195;&#182;llner et al., 1993) proposes                       transport ER, identified (Lewis Pel- <br /> that unique set proteins transport vesicle                       ham, 1996). Mutations inhibiting retrograde traffic the <br /> (v-SNARE) proteins target membrane (t-                            ER cis-Golgi compartment lead rapid <br />                                                                              blockage forward traffic, making difficult distin- <br /> Please address correspondence Tom H. Stevens, Institute Molec-     guish phenotypic analysis protein involved <br /> ular Biology, University Oregon, Eugene, OR 97403-1229. Tel.: (541) <br /> 346-5884; Fax: (541) 346-4854; E-mail: stevens@molbio.uoregon.edu            anterograde retrograde transport. The tight coupling <br />     S.F. Nothwehr&#226;&#8364;&#8482;s present address Division Biological Sciences,      anterograde retrograde membrane traffic suggests <br /> University Missouri, Columbia, MO 65211                                   components needed forward transport be <br />                                                                              recycled (Letourneur et al., 1994; Lewis Pelham, 1996). <br /> 1. Abbreviations used paper: CPY, carboxypeptidase Y; ER, endo-         The yeast late Golgi compartment, corresponds <br /> plasmic reticulum; FOA, fluoroorotic acid.                                   mammalian TGN, functions sorting compartment <br />  <br />  <br />  <br /> &#239;&#163;&#169; The Rockefeller University Press, 0021-9525/97/06/1511/14 $2.00 <br /> The Journal Cell Biology, Volume 137, Number 7, June 30, 1997 1511&#226;&#8364;&#8220;1524   1511 <br />  (Pryer et al., 1992; Nothwehr Stevens, 1994; Stack et al.,             Plasmids Strains <br /> 1995). Over 50 VPS (vacuolar protein sorting) PEP                      The plasmids used study listed Table II. pSN206 con- <br /> (peptidase deficient) genes identified                  structed PCR amplifying VPS10 tail (codons 1413&#226;&#8364;&#8220;1579) with <br /> involved transport soluble vacuolar protease                 EcoRI&#226;&#8364;&#8220;BamHI ends cloning 2m-HIS3 vector pEG202, <br /> carboxypeptidase Y (CPY) Golgi apparatus                       makes LexA fusion proteins control ADH pro- <br />                                                                            moter (Golemis et al., 1996). A 1.1-kb fragment starting SacI site <br /> the vacuole (Jones, 1977; Bankaitis et al., 1986; Rothman <br />                                                                            150 bp upstream VTI1-ATG adding XhoI site amplified <br /> and Stevens, 1986), genes essential               genomic DNA cloned pBluescript KS1. For disruption <br /> cell growth (Stack et al., 1995). CPY binds CPY                     construct pSN258 VTI1 coding region HindIII NdeI <br /> sorting receptor, Vps10p, late Golgi compartment                    sites replaced HIS3 gene. pFvM7 consists VTI1 (codons 1&#226;&#8364;&#8220;194) <br /> (Marcusson et al., 1994), complex directed away                PCR-amplified BamHI&#226;&#8364;&#8220;SmaI ends pGex2T (Smith Johnson, <br />                                                                            1988). A HindIII site introduced upstream VTI1 initiation <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> secretory pathway packaging transport                     codon HindIII&#226;&#8364;&#8220;XhoI fragment cloned GAL1- <br /> vesicles eventually fuse prevacuolar endo-                promoter DNA pRS314 (Sikorski Hieter, 1989) yield pFvM17. <br /> somal compartment (Raymond et al., 1992b; Vida et al.,                     pFvM16 identical pFvM17 presence single HA tag <br /> 1993; Piper et al., 1995; Stack et al., 1995). Whereas CPY              BglII site. The MBP&#226;&#8364;&#8220;VTI1::HA fusion construct derived by <br />                                                                            PCR amplification VTI1 (codons 1&#226;&#8364;&#8220;194) BamHI&#226;&#8364;&#8220;HindIII ends and <br /> released transported vacuole, CPY re- <br />                                                                            cloning pMal-p2 (Maina et al., 1988). pFvM28, pFvM29, pFvM32 <br /> ceptor Vps10p recycles Golgi apparatus (Cereghino                   consist 1.8-kb fragment encoding VTI1 SacI XhoI <br /> et al., 1995; Piper et al., 1995; Cooper Stevens, 1996).               site pRS314, pRS316, YEp352 (Hill et al., 1986). VTI1 (codons 1&#226;&#8364;&#8220;194) <br />    Recently, t-SNARE Pep12p local-                     PCR amplified BamHI&#226;&#8364;&#8220;SmaI ends pQE30 pro- <br /> ized yeast prevacuolar compartment (Becherer et al.,                duce pFvM38. pFvM40 constructed PCR amplification SED5 <br />                                                                            (codons 1&#226;&#8364;&#8220;324; Hardwick Pelham, 1992) EcoRI BamHI ends <br /> 1996). Deletion PEP12 gene leads accumula-                   subcloning fragment pGex2T. pFvM89 contains PCR-ampli- <br /> tion small vesicles, suggesting necessary fu-            fied 1.8-kb fragment encoding SED5 starting 400 bp upstream ATG <br /> sion Golgi-derived transport vesicles prevacu-                 BamHI SacI ends YEp351. PCR mutagenesis performed <br /> olar compartment.                                                          described (Muhlrad et al., 1992) using PvuII-digested pFvM28 tem- <br />                                                                            plate oligonucleotides annealing nucleotides 1823&#226;&#8364;&#8220;1844 2074&#226;&#8364;&#8220; <br />    Here report identification novel yeast <br />                                                                            2053 pRS314. In PCR reaction, 3 mM MgCl2, 0.25 mM MnCl2 and <br /> v-SNARE, Vti1p, interacts Pep12p in-                      25 mM dATP, 250 mM dCTP, dGTP, dTTP, 25 mM dGTP, 250 mM <br /> volved membrane traffic Golgi apparatus                     dATP, dCTP, dTTP used. The PCR fragments cotrans- <br /> the prevacuolar compartment. VTI1                       formed XhoI&#226;&#8364;&#8220;SacI digested pRS314 FvMY6 pFvM29. The trans- <br /> essential gene, essential function related               formants pooled, plasmid pFvM29 removed plating 5-fluo- <br />                                                                            roorotic acid (5-FOA) (Boeke et al., 1984), colonies screened CPY <br /> interaction Vti1p cis-Golgi t-SNARE Sed5p.                      secretion 378C colony overlay (Rothman et al., 1986) growth <br /> Therefore, Vti1p v-SNARE interacts functionally                  228 378C. Mutant plasmids rescued, retransformed, the <br /> with t-SNAREs, Pep12p localized prevacuolar                     plasmid pFvM29 removed allow isolation pFvM92 pFvM93. <br /> compartment Sed5p localized cis-Golgi com-                      pFvM103 consists PCR-amplified fragment SSO2 (codons 1&#226;&#8364;&#8220;269; <br />                                                                            Aalto et al., 1993) BamHI EcoRI ends subcloned pGex2T. <br /> partment. <br />                                                                            pFvM104 constructed subcloning 2.9-kb EcoRI&#226;&#8364;&#8220;SacI fragment <br />                                                                            pRB58 (Carlson Botstein, 1982) encoding SUC2 pRS316. <br />                                                                               To delete VTI1, yeast diploid strain (SEY6210 3 SEY6211) trans- <br /> Materials Methods                                                      formed SacI&#226;&#8364;&#8220;XbaI-digested pSN258 selected His1 colonies. <br />                                                                            A vti1D::HIS3 <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>knockout</span> confirmed PCR, strain trans- <br /> Materials                                                                  formed VTI1 covering plasmid. After sporulation, tetrads dis- <br />                                                                            sected vti1D cells identified (FvMY6, Table I). For hydroxylamine <br /> Reagents used following sources: enzymes DNA manipula- <br />                                                                            mutagenesis, 250 mg pFvM28 purified CsCl-gradient incubated <br /> tion New England Biolabs (Beverly, MA) Boehringher Mannheim <br />                                                                            2 ml 0.4 M hydroxylamine 150 mM KPO4, pH 6.0, 758C 60 <br /> Biochemicals (Indianapolis, IN), secondary antibodies Promega Bio- <br />                                                                            min (Schauer et al., 1985). The DNA cycled E. coli trans- <br /> tech. (Madison, WI), Amersham Corp. (Arlington Heights, IL), Jack- <br />                                                                            formed FvMY6 containing plasmid pFvM29. The transformants were <br /> son ImmunoResearch Labs (West Grove, PA); 35S-Express <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> ECL <br />                                                                            pooled, plasmid pFvM29 removed plating 5-FOA plates, and <br /> solution New England Nuclear (Boston, MA); fixed Staphylococcus <br />                                                                            colonies screened CPY secretion 378C colony overlay (Rothman <br /> aureus cells (IgGsorb) The Enzyme Center (Malden, MA); Oxalyticase <br />                                                                            et al., 1986). A mutant plasmid recovered, retested, vti1-1 DNA <br /> from Enzogenetics (Corvallis, OR); protein G&#226;&#8364;&#8220;Sepharose Pharmacia <br />                                                                            subcloned integration vector pRS306 (Sikorski Hieter, 1989). <br /> Fine Chemicals (Uppsala, Sweden); plasmid pQE30 Ni-NTA <br />                                                                            FvMY7 constructed integration linearized DNA SEY6211 <br /> resin Qiagen (Hilden, Germany). All reagents purchased <br />                                                                            looping wild-type VTI1 5-FOA plates (Boeke et al., 1984). <br /> from Sigma Chemical Co. (St. Louis, MO). <br />    Plasmid manipulations performed Escherichia coli strains <br /> MC1061 XL1Blue using standard media. Yeast strains (Table I)       Two Hybrid Screen <br /> grown rich media (1% yeast extract, 1% peptone, 2% dextrose, YEPD) <br /> or standard minimal medium (SD) appropriate supplements. To in-       The strain CTY10-5D carrying plasmid pSN206 expressing LexA- <br /> duce expression GAL1 promoter, dextrose replaced 2%        VPS10(tail) fusion protein transformed libraries pGAD1, <br /> raffinose 2% galactose.                                                pGAD2, pGAD3 (Chien et al., 1991; Bartel Fields, 1995; Golemis <br />  <br />  <br /> Table I. Yeast Strains Used Study <br /> Strain                                                          Genotype                                                             Reference <br />  <br /> CTY10-5D                       MATa leu2-3,112 his3-D200 trp1-D901 gal4 gal80 URA3::lexAop-LacZ                              (Bartel Fields, 1995) <br /> SEY6210                        MATa leu2-3,112 ura3-52 his3-D200 trp1-D901 lys2-801 suc2-D mel-                              (Robinson et al., 1988) <br /> SEY6211                        MATa leu2-3,112 ura3-52 his3-D200 ade2-101 trp1-D901 suc2-D9 mel-                             (Robinson et al., 1988) <br /> SEY5186                        MATa leu2-3,112 ura3-52 gal2 sec18-1                                                          (Emr et al., 1984) <br /> FvMY6                          MATa leu2-3,112 ura3-52 ade2-101 trp1-D901 suc2-D9 mel- vti1::DHIS3                           This study <br /> FvMY7                          MATa leu2-3,112 ura3-52 his3-D200 ade2-101 trp1-D901 suc2-D9 mel- vti1-1                      This study <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                              1512 <br />  Table II. Plasmids Used Study <br /> Plasmid                                                             Description                                                              Reference <br />  <br /> pSN206                           ADH1-LexA-VPS10(tail) fusion pEG202 (2m-HIS3)                                                   This study <br /> p2H-1                            5 kb genomic DNA VTI1 (codons 106-217) pGAD3                                               This study <br /> pSN258                           vti1D::HIS3 disruption construct pRS305                                                         This study <br /> pFvM7                            E. coli expression vector pGex2T encoding GST-Vtilp (aal-194)                                      This study <br /> pFvM16                           GAL1-VTI1::HA pRS314 (CEN6-TRP1)                                                                This study <br /> pFvM17                           GAL1-VTI1 pRS314 (CEN6-TRP1)                                                                    This study <br /> pFvM20                           E. coli expression vector pMal-p2 encoding MBP-Vtilp::HA (aal-194)                                 This study <br /> pFvM28                           1.8 kb containing VTI1 pRS314 (CEN6-TRP1)                                                       This study <br /> pFvM29                           1.8 kb containing VTI1 pRS316 (CEN6-URA3)                                                       This study <br /> pFvM32                           1.8 kb containing VTI1 YEp352 (2m-URA3)                                                         This study <br /> pFvM38                           E. coli expression vector pQE30 encoding 6His-Vtilp (aal-194)                                      This study <br /> pFvM40                           E. coli expression vector pGEX2T encoding GST-Sed5p (aa1-324)                                      This study <br /> pFvM47                           7.7 kb Chrom XII (nt190000-197700) containing SED5 YEp24 (2m-URA3)                              This study <br /> pFvM89                           SED5 YEp351 (2m-LEU2)                                                                           This study <br /> pFvM92                           vti1-11 pRS314 (CEN6-TRP1)                                                                      This study <br /> pFvM93                           vti1-2 pRS314 (CEN6-TRP1)                                                                       This study <br /> pFvM103                          E. coli expression vector pGex2T encoding GST-Sso2p (aal-269)                                      This study <br /> pFvM104                          2.9 kb EcoRI-SnaBI DNA pRB58 SUC2 pRS316                                              This study <br /> pRCP59                           E. coli expression vector pGex3X encoding GST-Pep12p (aal-263)                                     R.C. Piper*/T.H. Stevens <br /> pRCP110                          PEP12 YEp351 (2m-LEU2)                                                                          R.C. Piper*/T.H. Stevens <br /> *Institute Molecular Biology, University Oregon, Eugene, OR <br />  <br />  <br />  <br /> et al., 1996). About 30,000 colonies screened expression          collected top. Sucrose density measured refractome- <br /> lacZ gene. The rescued library plasmid containing GAL4 activation             ter. Fractions concentrated TCA precipitation directly <br /> domain fused codons 109&#226;&#8364;&#8220;217 Vti1p, p2H-1, did interact         analyzed SDS-PAGE immunoblotting. <br /> LexA protein alone. <br />                                                                                   Immunoprecipitation 35S-labeled Proteins <br /> Production Vti1p Pep12p Antisera <br />                                                                                   The procedure CPY immunoprecipitation described earlier <br /> An antiserum raised rabbits fusion protein containing glu-       (Vater et al., 1992). Briefly, log-phase growing yeast cells labeled for <br /> tathione&#226;&#8364;&#8220;S-transferase amino acids 1&#226;&#8364;&#8220;194 Vti1p purified E.            10 min 35S-Express label (10 ml/0.5 OD cells) followed 30-min <br /> coli. The antiserum affinity purified Affigel 10 (BioRad Labo-        chase (chase initiated addition 500 mg/ml methionine cys- <br /> ratories, Hercules, CA) column covalently bound maltose-binding              teine). Invertase immunoprecipitation secondary immunoprecipita- <br /> protein Vti1p (amino acids 1&#226;&#8364;&#8220;194) fusion protein purified E. coli. The       tion proteins containing a1,6-mannose linkages performed de- <br /> Pep12p antiserum raised rabbits purified fusion protein          scribed (Franzusoff Schekman, 1989). Invertase derepressed by <br /> containing glutathione&#226;&#8364;&#8220;S-transferase amino acids 1&#226;&#8364;&#8220;263 Pep12p.             incubation minimal medium containing 0.1% glucose, 50 mM KPO4, <br />                                                                                   1 mg/ml BSA 45 min 228C 30 min 228C plus 15 min 378C. <br /> Immunofluorescence Microscopy                                                     Pulse labeling 35S-Express label (10 ml/0.5 OD cells) 7 <br />                                                                                   min, aliquot removed (in experiments), remaining sam- <br /> Indirect immunofluorescence microscopy performed SEY6211                 ple chased 30 min. Cells pelleted, medium removed, and <br /> cells previously described (Raymond et al., 1992a). The following anti-        cells spheroplasted. The spheroplasts pelleted lysed <br /> body combination used: 1:50 diluted affinity-purified Vti1p antiserum,        giving rise intracellular fraction. The supernatant combined <br /> 1:100 diluted biotin-conjugated goat anti&#226;&#8364;&#8220;rabbit IgG (H 1 L), 1:100           medium fraction yield extracellular fraction. Immunopre- <br /> diluted rhodamine-conjugated streptavidin.                                        cipitates analyzed SDS-PAGE autoradiography. <br />  <br /> SDS-PAGE Western Blotting                                                     Suppressor Screen <br /> Yeast extracts prepared proteins subjected SDS-PAGE               To identify multicopy suppressors allow growth absence of <br /> and electroblotting described (Laemmli, 1970; Towbin et al., 1979). Af-        Vti1p, strain FvMY6/pFvM16 (vti1D/pCEN-GAL1-VTI1) used. <br /> finity-purified rabbit antisera used detection Vti1p, Pep12p,     FvMY6/pFvM16 cells transformed YEp24 2m library (Carlson <br /> Vps10p (Piper et al., 1995), dipeptidyl aminopeptidase A (DPAP A;             Botstein, 1982) plated SD-Ura plates, conditions turn off <br /> Roberts et al., 1992). Rabbit antiserum used PGK monoclonal           expression VTI1 (due repression GAL1 promoter by <br /> antibodies Vph1p (Molecular Probes Inc., Eugene, OR), Sac1p (May-             dextrose). Colonies grew tested absence Vti1p im- <br /> inger et al., 1995), Dpm1p (Molecular Probes Inc.). An incubation        munoblot analysis inability grow 5-FOA plates determine <br /> HRP-conjugated secondary antibodies followed visualization            suppression plasmid linked. Plasmids recovered, retrans- <br /> ECL reagents.                                                                     formed, colonies, lost pFvM16 plasmid, selected confirm <br />                                                                                   suppression. Portions inserts subcloned YEp351 to <br /> Subcellular Fractionation                                                         determine minimal DNA fragment needed suppression. <br />  <br /> Membrane fractionation differential centrifugation performed as <br /> described (Paravicini et al., 1992). SEY6211 spheroplasts lysed, cen- <br />                                                                                   Native Immunoprecipitation <br /> trifuged 500 g (5 min) remove debris, 13,000 g (10 min) yield         1 OD600nm (z107 cells) wild-type yeast cells spheroplasted, solu- <br /> P13 S13, 200,000 g (20 min) produce P200 S200.                  bilized 700 ml 0.5% Triton X-100 PBS, centrifuged 12,000 g <br />     The S13 fraction separated sucrose gradient centrifuga-        10 min. The supernatant fractions incubated anti-Vti1p anti- <br /> tion (Becherer et al., 1996). The gradient consisted following steps:      serum pre-immune serum 2 h 48C protein G-sepharose <br /> 0.5 ml 60%, 1 ml 37%, 1.5 ml 34%, 2 ml 32%, 2 ml 29%, 1 ml 27%, 1.5 ml            additional 2 h 48C. The sepharose beads washed times <br /> 22%, 1 ml 19% (wt/wt) sucrose 10 mM Hepes-NaOH, pH 7.6,                5 min 0.5% Triton X-100 PBS. The proteins bound beads <br /> was centrifuged 17 h 170,000 g SW41 rotor. 13 fractions         analyzed SDS-PAGE immunoblotting antibodies <br />  <br />  <br />  <br />  <br /> Fischer von Mollard et al. The v-SNARE Vti1p Interacts Sed5p Pep12p      1513 <br />  against Vti1p, Pep12p, Vph1p, Sac1p. To quantify each <br /> protein immunoprecipitated, supernatant fraction cor- <br /> responding 30 10% total protein present immunopre- <br /> cipitations analyzed SDS-PAGE immunoblotting. <br />  <br />  <br /> In vitro Binding Assay <br /> Recombinant fusion proteins purified E. coli extracts. Specifi- <br /> cally, bacteria induced mid-log phase 0.5 mM IPTG 6 h at <br /> 308C express 6-His&#226;&#8364;&#8220;Vti1p (amino acids 1&#226;&#8364;&#8220;194). The bacterial pellet      Figure 1. Amino acid sequence yeast Vti1p. The predicted <br /> resuspended 50 mM NaPO4, pH 8.0, 300 mM NaCl, frozen, lysed          transmembrane domain boxed. The predicted coiled coil do- <br /> sonification. After centrifugation 10 min 12,000 g, supernatant   main underlined (amino acids 159&#226;&#8364;&#8220;188 P 5 0.285 using <br /> was incubated Ni-NTA resin mixing 1 h 48C. The resin was <br />                                                                                paircoil program [Berger et al., 1995]; amino acids 149&#226;&#8364;&#8220;169 <br /> loaded column, washed 50 mM NaPO4, pH 6.0, 300 mM NaCl, <br /> and 10% glycerol, followed washes 20 100 mM imidazole           P 5 0.58 using coil program window 21 [Lu- <br /> the wash buffer. 6-His&#226;&#8364;&#8220;Vti1p eluted 500 mM imidazole         pas et al., 1991]). These sequence data available Gen- <br /> dialyzed PBS. Bacteria induced 0.5 mM IPTG 4 h        Bank accession number AF006074. <br /> 378C express GST, GST-Pep12p, GST-Sed5p, GST-Sso2p. Bacte- <br /> rial pellets resuspended PBS, frozen, lysed sonification. <br /> Debris removed centrifugation 12,000 g 10 min. The super- <br /> natant incubated glutathione&#226;&#8364;&#8220;agarose 15 min. The beads            Sft1p (32% amino acid identity), smaller <br /> were washed times bound protein estimated after <br /> separation SDS-PAGE visualization Coomassie staining. <br />                                                                                proteins. Sft1p yeast v-SNARE involved retrograde <br /> About 1 mg immobilized GST-fusion proteins incubated         transport cis-Golgi compartment (Banfield et al., <br /> 9 mg 6-His&#226;&#8364;&#8220;Vti1p 0.5% Triton X-100 PBS mixing 2 h         1995), synaptobrevin v-SNARE synaptic <br /> 48C. The beads washed times 0.5% Triton X-100 PBS           vesicles. Domains close NH2 terminus Vti1p (amino <br /> and bound protein eluted analyzed SDS-PAGE immuno-              acids 39&#226;&#8364;&#8220;63) synaptobrevin (amino acids 29&#226;&#8364;&#8220;56) are <br /> blotting anti-Vti1p antibodies. <br />                                                                                predicted adopt structure viral fusion pep- <br />                                                                                tides charged amino acids a-helix <br /> Electron Microscopy                                                            bulky hydrophobic amino acids other. The <br /> Electron microscopy experiments performed described (Kaiser            presence domain sort close NH2 termi- <br /> and Schekman, 1990; Piper et al., 1994).                                       nus characteristic v-SNAREs (Dascher et al., 1991; <br />    To quantify number vesicles, 12&#226;&#8364;&#8220;20 cells condition were <br /> photographed. The negatives scanned, vesicle-like dark structures were <br />                                                                                White, 1992; Jahn S&#195;&#188;dhof, 1994). In general, members <br /> counted, cytoplasmic area determined using NIH Image           v-SNARE family share common structural elements <br /> program. The number vesicles <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cytoplasmic area compared         exhibit low sequence identities pairwise combina- <br /> for different conditions. The <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> membrane sheets ad-      tions (18&#226;&#8364;&#8220;35% amino acid identities), Vti1p fits this <br /> jacent cell wall nucleus measured using NIH       <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. <br /> Image program divided cytoplasmic area measure ER <br /> accumulation.                                                                     t-SNAREs display significant amino acid sequence iden- <br />                                                                                tity coiled coil domain adjacent transmem- <br />                                                                                brane domain, regions identity extend <br /> Results                                                                        v-SNAREs Vti1p. Most t-SNAREs contain nine <br />                                                                                invariant amino acid residues region (Pelham, 1993; <br /> Identification VTI1                                                         Lewis Pelham, 1996). The coiled coil domain Vti1p <br /> We interested identifying proteins in-                        does share sequence identity coiled coil do- <br /> volved trafficking CPY sorting receptor Vps10p                       main different t-SNAREs (8% compared Pep12p, <br /> (Marcusson et al., 1994). Since cytosolic tail Vps10p                   13% compared Sed5p, 17% compared syntaxin3 <br /> directs recycling (Cereghino et al., 1995; Cooper                      syntaxin5). Only invariant amino acid <br /> Stevens, 1996), portion protein fused                   residues present Vti1p. Therefore, the <br /> LexA DNA-binding domain create &#226;&#8364;&#339;bait&#226;&#8364;? protein                          criteria, Vti1p belongs v-SNARE family of <br /> screen interacting proteins yeast two-hybrid sys-                   proteins. <br /> tem (Brent Ptashne, 1985; Chien et al., 1991). An acti- <br /> vating plasmid containing VTI1 gene (Vps interact-                     Vti1p Localizes Golgi Prevacuolar Membranes <br /> ing) isolated insert sequenced. The sequence                       A rabbit antiserum generated Vti1p. Anti- <br /> matched open reading frame upstream CIK1 (Page                           Vti1p antibodies identified single 27-kD band protein <br /> and Snyder, 1992), sequenced Ge-                         extracts wild-type yeast cells analyzed SDS- <br /> nome Sequencing Project chromosome XIII desig-                          PAGE immunoblotting. This band intense <br /> nated open reading frame YMR197c. VTI1 predicted                         strain carrying high copy VTI1 plasmid absent in <br /> encode protein 217 amino acids COOH-termi-                         vti1D mutant cells (data shown). Vti1p immunolo- <br /> nal transmembrane domain (Fig. 1).                                             calized indirect immunofluorescence wild-type cells. <br />   The regions adjacent transmembrane domain                         Vti1p antibodies labeled numerous punctate structures <br /> predicted fold amphipathic a-helical coiled coil                    cytosol (Fig. 2 a), adja- <br /> domain. These structural elements characteristic                        cent vacuole, vacuolar ER membranes were <br /> SNARE proteins (Ferro-Novick Jahn, 1994; Rothman,                          stained. <br /> 1994). In pairwise sequence comparisons SNARE pro-                           Differential centrifugation wild-type yeast lysates fol- <br /> teins, Vti1p shares highest sequence similarity                       lowed SDS-PAGE immunoblot analysis (Fig. 2 b) <br /> synaptobrevin (VAMP; 28% amino acid identity)                         revealed Vti1p predominantly localized the <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                                  1514 <br />                                                                                 VTI1 Is Essential Gene Required CPY Sorting <br />                                                                                The VTI1 gene disrupted yeast determine the <br />                                                                                functional role Vti1p. One copies VTI1 a <br />                                                                                diploid strain (SEY6210 3 SEY6211) disrupted re- <br />                                                                                placement HIS3 gene (vti1D::HIS3). Tetrad dis- <br />                                                                                sections did yield viable His1 spores, revealing that <br />                                                                                VTI1 essential gene. His1 spores derived tetrad <br />                                                                                dissection FvMY4 transformation VTI1 <br />                                                                                plasmid viable presence VTI1-con- <br />                                                                                taining plasmid. vti1D::HIS3 cells carrying VTI1&#226;&#8364;&#8220;URA3 <br />                                                                                plasmid grow 5-FOA plasmid loss was <br />                                                                                lethal, confirming VTI1 essential gene. By con- <br />                                                                                trast, VPS PEP genes essential hap- <br />                                                                                loid yeast cell growth (Stack et al., 1995). <br />                                                                                   The phenotypic consequences loss Vti1p were <br />                                                                                investigated placing VTI1 expression control <br />                                                                                GAL1 promoter. vti1D cells carrying GAL1&#226;&#8364;&#8220; <br />                                                                                VTI1 plasmid maintained galactose-containing <br />                                                                                medium. Approximately 20 h shifting cells to <br /> Figure 2. Vti1p localization. (a) Immunofluorescence microscopy                glucose-containing medium (about cell divisions) the <br /> of Vti1p. Anti-Vti1p antibodies reveal punctate staining pattern             cells stopped dividing (Fig. 3 a). Immunoblot analysis re- <br /> distinct vacuoles seen Nomarski optics. (b) Differen- <br /> tial centrifugation total yeast homogenate (H). Vti1p pre- <br /> dominantly 200,000 g pellet (P200) lower <br /> amounts 13,000 g pellet (P13). The syntaxin-related pro- <br /> tein Pep12p similar distribution. (c) Sucrose density centrif- <br /> ugation S13 fraction. Vti1p migrated peaks, co- <br /> fractionating prevacuolar protein Pep12p other <br /> with Golgi protein DPAP A. Fractions collected from <br /> the gradient (fraction 1) analyzed SDS-PAGE <br /> and immunoblotting. <br />  <br />  <br />  <br />  <br /> 200,000 g membrane pellet (P200), contained small <br /> vesicles Golgi membranes, defined enrich- <br /> ment proteins DPAP A Vps10p (Paravicini et al., <br /> 1992; Roberts et al., 1992; Marcusson et al., 1994). Lower <br /> amounts Vti1p present 13,000 g pellet (P13), <br /> which included vacuolar (Vph1p [Manolson et al., 1992]), <br /> ER (Dpm1p [Orlean, 1990]), mitochondrial membranes. <br /> The prevacuolar t-SNARE Pep12p abundant in <br /> the P200 fraction lower amounts P13 fraction, <br /> consistent previous results (Becherer et al., 1996). As <br /> expected, cytosolic protein PGK fractionated the <br /> S200 soluble protein fraction. <br />                                                                                Figure 3. Depletion Vti1p leads growth arrest CPY mis- <br />   To study localization Vti1p detail,                       localization. (a) Growth curves GAL1&#226;&#8364;&#8220;VTI1 GAL1&#226;&#8364;&#8220; <br /> 13,000 g supernatant (S13) separated 19&#226;&#8364;&#8220;37% su-                       VTI1-HA strains glucose-containing medium. Cells both <br /> crose density equilibrium gradient (Becherer et al., 1996).                    strains displayed equivalent growth properties stopped divid- <br /> Fractions collected analyzed                          ing 22 h. Cell density measured optical density 600 <br /> SDS-PAGE immunoblot analysis (Fig. 2 c). Vti1p                         nm. The cultures diluted reached optical den- <br /> found different buoyant densities: fractions 6&#226;&#8364;&#8220;8                     sity cells growing logarithmic phase. Tak- <br /> (at 29.5% sucrose) fraction 12 (36.4% sucrose). As                      ing dilution factor account, extrapolated cell densities <br /> observed previously (Becherer et al., 1996), Golgi mem-                        calculated measured optical densities. (b) CPY <br /> branes (as indicated presence DPAP A) mi-                            sorting GAL1&#226;&#8364;&#8220;VTI1-HA strain grown galactose for <br />                                                                                different amounts time glucose medium. CPY sorting was <br /> grated close the gradient cofraction- <br />                                                                                followed pulse&#226;&#8364;&#8220;chase labeling CPY immunoprecipitation <br /> ated higher density Vti1p peak. By contrast, Pep12p                   cellular extracts (I, intracellular) medium (E, extracel- <br /> was single peak lower density                     lular). For cells cultured galactose, CPY reached vacuole <br /> Vti1p peak. These data indicate smaller portion                      (presence mCPY). After 16 h glucose CPY was <br /> Vti1p localized Golgi apparatus, ma-                     secreted late Golgi precursor form, p2CPY. After longer <br /> jority colocalized Pep12p prevacuolar com-                      time periods glucose, ER early Golgi forms CPY <br /> partment.                                                                      (p1CPY) accumulated cell. <br />  <br />  <br />  <br />  <br /> Fischer von Mollard et al. The v-SNARE Vti1p Interacts Sed5p Pep12p   1515 <br />  vealed VTI1 expression control GAL1           treated VTI1 CEN-based plasmid, mutant iden- <br /> promoter z10-fold higher presence galactose       tified (vti1-1) secreted CPY 378C grew with <br /> than expression VTI1 promoter. Wild-type levels        nearly wild-type doubling time temperature. The <br /> of Vti1p reached z12 h shift glucose medium       vti1-1 <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> integrated genome further <br /> (data shown).                                               analysis. Additional vti1 alleles (vti1-2 vti1-11) were <br />    To determine sorting soluble vacuolar hydro-       generated PCR mutagenesis. vti1-11 grew normally at <br /> lase CPY affected lowered Vti1p levels, fate      208C grew slowly 378C. The doubling time of <br /> newly synthesized CPY monitored GAL1&#226;&#8364;&#8220;VTI1                vti1-11 378C 10 h compared 2.5 h wild- <br /> cells grown galactose medium shift glucose       type yeast cells. The vti1-2 mutant cells slight growth <br /> medium different periods time (Fig. 3 b). Pulse&#226;&#8364;&#8220;         defect 378C. <br /> chase immunoprecipitations CPY GAL1&#226;&#8364;&#8220;VTI1 cells               The effects vti1 mutations CPY sorting were <br /> grown galactose medium grown glucose medium            studied CPY immunoprecipitations pulse&#226;&#8364;&#8220;chase <br /> for 12 h revealed CPY mature,        labeling. CPY correctly transported vacuole at <br /> vacuolar form intracellular fraction, reflecting cor-    228C vti1-1 mutant (Fig. 4, row). If 35S label was <br /> rect sorting delivery CPY vacuole. After          added immediately shift 368C, little CPY <br /> growth glucose medium 16 h, hardly newly syn-        reached vacuole, instead Golgi-modified p2CPY <br /> thesized CPY reached vacuole. Instead, Golgi-           secreted accumulated intracellularly. The <br /> modified precursor form CPY (p2CPY [Stevens et al.,          p1CPY detected small varied some- <br /> 1982]) secreted. These data indicate Vti1p nec-     experiments. In cells labeled at <br /> essary sorting soluble vacuolar hydrolase CPY.       228C preincubation 368C (30 min), majority <br /> This missorting CPY occurred growth defect        CPY reached vacuole (as evidenced presence <br /> became apparent, suggesting indirect ef-     mature CPY; mCPY), indicating vti1-1 temper- <br /> fect. After GAL1&#226;&#8364;&#8220;VTI1 cells grown glucose            ature-sensitive mutation rapidly reversible. This ex- <br /> medium 20 24 h, ER early-Golgi form           tremely fast onset CPY sorting defect fast re- <br /> CPY (p1CPY [Stevens et al., 1982]) accumu-         covery demonstrate Vti1p directly involved CPY <br /> late. These data imply transport Golgi ap-     transport Golgi apparatus vacuole. The <br /> paratus slowed blocked severe depletion          mutant vti1-2 exhibited fast onset CPY secretion <br /> Vti1p. However, remains unclear p1CPY accu-          shift 368C (Fig. 4, middle row), however, re- <br /> mulation caused directly Vti1p depletion in-       covery somewhat slower. The extent CPY secretion <br /> direct effect severe growth defect.                  greater vti1-2 mutant cells vti1-1 mutant <br />                                                                 cells shift 368C; p2CPY accumulated intra- <br />                                                                 cellularly p1CPY did not. <br /> vti1 Mutants Are Defective Two Vesicle Traffic Steps            In vti1-11 mutant, temperature sensitive <br /> The genes identified required vacuolar protein     growth, CPY reached vacuole processed <br /> sorting (VPS) nonessential yeast, VTI1 func-     normally 228C (Fig. 4, row). p1CPY pre- <br /> tion required CPY sorting, gene essential.    dominant form protein vti1-11 cells <br /> This suggests VTI1 second function addition       labeled restrictive temperature (368C). Lower <br /> to vacuolar protein sorting. Temperature-sensitive vti1 mu-     amounts intracellular secreted p2CPY also <br /> tants generated elucidate role Vti1p          present, mature CPY not. After return per- <br /> Golgi vacuolar traffic essential function   missive temperature, CPY sorting vacuole recov- <br /> protein. Using shuttle mutagenesis hydroxylamine-          ered quickly vti1-11 cells, indicating vti1-11 tem- <br />  <br />  <br />                                                                                                   Figure 4. CPY sorting in <br />                                                                                                   vti1-ts mutants. CPY sorting <br />                                                                                                   followed vti1-1, <br />                                                                                                   vti1-2, vti1-11 strains by <br />                                                                                                   pulse&#226;&#8364;&#8220;chase labeling and <br />                                                                                                   CPY immunoprecipitations <br />                                                                                                   cellular extracts (I, in- <br />                                                                                                   tracellular) medium (E, <br />                                                                                                   extracellular). vti1-1 cells <br />                                                                                                   preincubated 30 min <br />                                                                                                   368C vti1-2 vti1-11 <br />                                                                                                   cells 15 min. In vti1-1 <br />                                                                                                   vti1-2 cells CPY <br />                                                                                                   secreted accumulated <br />                                                                                                   intracellularly p2CPY at <br />                                                                                                   restrictive temperature. <br />                                                                                                   In vti1-11 cells, predominantly <br />                                                                                                   ER early Golgi form <br />                                                                                                   p1CPY accumulated intracel- <br />                                                                                                   lularly restrictive tem- <br />                                                                                                   perature. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                   1516 <br />  perature-sensitive mutation rapidly reversible. These                       mon et al., 1981), accumulated core glycosylated ER <br /> experiments indicate vti1-11 cells, CPY accu-                      form invertase restrictive temperature (Fig. 5 c, <br /> mulated ER cis-Golgi compartment                       lanes 1&#226;&#8364;&#8220;4). Invertase glycosylation transport nor- <br /> (as p1CPY) rapid onset shift restrictive                    mal vti1 mutants 228C. In vti1-11 mutant <br /> temperature. CPY reached trans-Golgi compart-                         labeled 368C, ER form slightly larger form of <br /> ment targeted vacuole instead secreted,                     invertase accumulated intracellularly (Fig. 5 a, lane 15). <br /> suggesting Golgi vacuole traffic defective                  Invertase molecules secreted vti1-11 mutant <br /> vti1-11 mutant cells.                                                          cells severely underglycosylated (Fig. 5 a, lane 16). <br />    To characterize mutants, trafficking                   By contrast, nearly invertase secreted vti1-1 cells, <br /> secretory protein invertase studied (Fig. 5). In wild-                     somewhat underglycosylated restrictive <br /> type yeast cells core glycosylated ER form invertase                    temperature (Fig 5 b, lanes 15 16). Glycosylation and <br /> seen chase period received outer chain glycosy-                     secretion invertase nearly normal vti1-2 cells <br /> lation secreted chase 228 (Fig. 5 a,                     nonpermissive temperature (Fig. 5 b, lanes 7 8), <br /> lanes 1&#226;&#8364;&#8220;4) 368C (lanes 5&#226;&#8364;&#8220;9). As expected, sec18-1                       indicating vti1-2 mutant transport the <br /> cells, defective ER Golgi vesicle traffic (Es-                 late-Golgi compartment vacuole blocked. <br />  <br />  <br />  <br />  <br />                                                                                                                  Figure 5. Invertase sorting in <br />                                                                                                                  vti1-ts mutants. (a&#226;&#8364;&#8220;c) Inver- <br />                                                                                                                  tase immunoprecipitations <br />                                                                                                                  intracellular (I) ex- <br />                                                                                                                  tracellular (E) fractions were <br />                                                                                                                  performed directly the <br />                                                                                                                  pulse 30-min chase. <br />                                                                                                                  Wild-type (WT), vti1, and <br />                                                                                                                  sec18-1 cells labeled at <br />                                                                                                                  228C 15-min prein- <br />                                                                                                                  cubation 368C. Invertase <br />                                                                                                                  processed secreted <br />                                                                                                                  correctly vti1-2 strains at <br />                                                                                                                  temperature. In vti1-1 <br />                                                                                                                  cells, partially glycosylated <br />                                                                                                                  invertase secreted at <br />                                                                                                                  368C. In vti1-11 cells, ER <br />                                                                                                                  early Golgi forms in- <br />                                                                                                                  vertase accumulated, and <br />                                                                                                                  partially glycosylated inver- <br />                                                                                                                  tase secreted 368C. (d) <br />                                                                                                                  Invertase immunopre- <br />                                                                                                                  cipitated intracellu- <br />                                                                                                                  lar fraction vti1-11 and <br />                                                                                                                  sec18-1 cells, in- <br />                                                                                                                  cubated 368C chased <br />                                                                                                                  30 min. Secondary immu- <br />                                                                                                                  noprecipitations per- <br />                                                                                                                  formed          antiserum <br />                                                                                                                  invertase (Inv.) or <br />                                                                                                                  a1, 6&#226;&#8364;&#8220;linked mannose <br />                                                                                                                  (1-6). vti1-11 cells accumu- <br />                                                                                                                  lated ER form inver- <br />                                                                                                                  tase, received <br />                                                                                                                  a1,6&#226;&#8364;&#8220;linked mannose, and <br />                                                                                                                  slightly larger cis-Golgi <br />                                                                                                                  form, precipi- <br />                                                                                                                  tated antiserum against <br />                                                                                                                  a1,6&#226;&#8364;&#8220;linked mannose. <br />  <br />  <br />  <br />  <br /> Fischer von Mollard et al. The v-SNARE Vti1p Interacts Sed5p Pep12p   1517 <br />     Glycosylation occurs characteristic order Saccha-            fusion (Jedd et al., 1995). As vti1-11 cells, ypt1-1 mu- <br /> romyces cerevisiae glycosylation enzymes lo-                tant cells slowly secrete underglycosylated invertase (Segev <br /> calized separate Golgi subcompartments (Franzusoff               et al., 1988). These data indicate Vti1p required both <br /> Schekman, 1989; Graham Emr, 1991). In Golgi               membrane traffic cis-Golgi compartment for <br /> compartment, a1-6 mannose residues                      sorting CPY Golgi apparatus vacuole. <br /> added Asn-linked carbohydrate chains. Further a1-6                      Electron microscopy used investigate whether <br /> mannose outer chain elongation follows com-                 vti1 mutant cells accumulated membrane organelles. Wild- <br /> partment. The modification addition a1-2                 type, vti1-1, vti1-11 cells grown 228C, incu- <br /> a1-3 mannose. Secondary immunoprecipitation anti-                  bated 1 h 378C, prepared electron micros- <br /> serum a1,6-mannose linkages used deter-                  copy (Fig. 6). The vti1-1 vti1-11 cells contained about <br /> mine internal invertase accumulated                 times number small vesicles relative iso- <br /> vti1 mutant cells received modification chase            genic wild-type cells. In addition, vti1-11 cells accumulated <br /> at restrictive temperature. The lower molecular weight              ER membranes wild-type cells, vti1-11 cells <br /> fraction invertase accumulated vti1-11 cells (Fig.           contained similar amounts ER membranes 228 <br /> 5 d, lane 1) received a1,6-linked mannose               378C. The accumulation vesicles vti1-1 vti1-11 <br /> similar core glycosylated, ER-localized invertase                mutant cells consistent proposed role Vti1p as <br /> found sec18-1 mutants (Fig. 5 d, lanes 3 4),               v-SNARE vesicle docking membrane fusion. <br /> could immunoprecipitated antiserum against <br /> a1-6 mannose linkages. The slightly larger molecular <br /> weight fraction invertase vti1-11 cells (Fig. 5 d, <br />                                                                         SED5 Overexpression Suppresses vti1D Lethality <br /> lane 2) contained a1,6-linked mannose                 To elucidate essential function VTI1, we <br /> reached cis-Golgi compartment. These data indicate                  screened multicopy suppressors allow yeast cells <br /> that ER Golgi membrane traffic drastically slowed              grow absence Vti1p. A vti1D strain harboring <br /> vti1-11 mutant cells shifted nonpermissive tempera-              GAL1&#226;&#8364;&#8220;VTI1 plasmid transformed multi- <br /> ture. Secretion under-glycosylated invertase occurred          copy yeast library (Carlson Botstein, 1982) trans- <br /> slowly, suggesting vti1-11 allele &#226;&#8364;&#339;leaky&#226;&#8364;?              formants plated glucose plates switch expression <br /> that Golgi apparatus lost ability glycosylate            VTI1. Yeast transformants capable forming colony <br /> invertase normally.                                                     glucose media tested absence Vti1p by <br />    The behavior vti1-11 mutant cells reminiscent               immunoblotting inability lose multicopy <br /> sec22D/sly2D cells nonpermissive temperature (Ossig et al.,          plasmid confirm suppression linked the <br /> 1991) ypt1-1 mutant cells (Segev et al., 1988). Sec22p/             library plasmid. In addition, library plasmid recov- <br /> Sly2p v-SNARE necessary ER cis-Golgi mem-                   ered, retransformed, vti1D strain isolated did <br /> brane traffic. Ypt1p, rab-related small GTPase, in-                contain GAL1&#226;&#8364;&#8220;VTI1 plasmid. A plasmid contain- <br /> volved ER cis-Golgi cis- medial-Golgi vesicle              ing 7.7-kb DNA insert identified supported <br />  <br />  <br />  <br />  <br /> Figure 6. Electron microscopy wild-type vti1 mutant cells. Cells grown 228C transferred 378C 1 h. In vti1-1 cells <br /> (b), small vesicles (arrowheads) accumulated. In vti1-11 cells (c), small vesicles ER membranes (arrow) accumulated. N, nucleus; V, <br /> vacuole. Bar, 500 nm. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                           1518 <br />  growth absence Vti1p. The minimal complement- <br /> ing fragment (2.7 kb) contained gene SED5, en- <br /> codes t-SNARE involved anterograde transport from <br /> the ER early-Golgi compartment retro- <br /> grade transport medial- early-Golgi com- <br /> partment (Hardwick Pelham, 1992; Banfield et al., <br /> 1995). To confirm SED5 vti1D suppressor, a <br /> fragment encoding Sed5p plus 400 bp 59 400 bp 39 of <br /> the coding region amplified PCR cloned a <br /> multicopy yeast vector. The 2m-SED5 plasmid (pFvM89) <br /> was transformed vti1D cells carried CEN-based <br /> URA3&#226;&#8364;&#8220;VTI1 plasmid. The URA3&#226;&#8364;&#8220;VTI1 plasmid re- <br /> moved plating cells 5-FOA plates, resulting vti1D <br /> cells containing 2m-SED5 plasmid able to <br /> grow doubling time 5 h compared doubling                        Figure 7. SED5 overexpression suppresses vti1D lethality. Growth <br /> time 3 h VTI1 wild-type yeast cells. Immunoblot anal-                   GAL1&#226;&#8364;&#8220;VTI1 strain overexpressing Sed5p or <br /> ysis demonstrated cells expressing                         Pep12p plates containing galactose (Gal) glucose (Glc). <br /> Vti1p.                                                                         The GAL1&#226;&#8364;&#8220;VTI1 strain grow glucose ex- <br />    To investigate overexpression prevacu-                       pression VTI1 shut off. Overexpression early Golgi <br /> olar t-SNARE Pep12p suppress lethality                        t-SNARE Sed5p allowed growth absence Vti1p. Over- <br />                                                                                expression prevacuolar-localized t-SNARE Pep12p did not <br /> vti1D mutation, vti1D cells carrying URA3&#226;&#8364;&#8220;VTI1 plas- <br />                                                                                support growth vti1D cells. <br /> mid transformed multicopy PEP12 plasmid <br /> (2m-PEP12). Overexpression Pep12p confirmed by <br /> Western blot analysis (data shown). vti1D cells carry- <br /> ing 2m-PEP12 URA3-VTI1 plasmids failed                                We investigated overexpression either <br /> grow 5-FOA, indicating cells lose                      Pep12p Sed5p suppress membrane traffic de- <br /> the VTI1 plasmid. It determined vti1D                         fect cis-Golgi compartment vti1-11 mutant cells. <br /> cells carrying 2m-PEP12 2m-SED5 plasmid                         Overexpression Sed5p vti1-11 strain reduced the <br /> expressing VTI1 GAL1 promoter grow                              p1CPY accumulated restrictive tempera- <br /> when Vti1p expression repressed glucose medium.                         ture increased secreted p2CPY (Fig. 8 b, <br /> Whereas 2m-SED5 plasmid support growth                            lanes 3 4). Whereas 56% total CPY found <br /> vti1D cells shut GAL1&#226;&#8364;&#8220;VTI1 expression (Glc                        p1CPY 21% secreted p2CPY vti1-11 mu- <br /> medium), multicopy PEP12 plasmid (Fig. 7).                       tant cells restrictive temperature (Fig. 8 b, lanes 1 <br /> These results demonstrate VTI1 SED5 interact                          2), overexpression Sed5p led decrease p1CPY <br /> genetically, providing evidence essential                     accumulation 28% increase secreted p2CPY <br /> function VTI1 involves membrane traffic of <br /> the early-Golgi compartment. <br />  <br /> Overexpression Pep12p Sed5p Restores <br /> CPY Transport <br /> The function Vti1p investigated monitor- <br /> ing effects overexpression t-SNAREs Pep12p <br /> and Sed5p trafficking CPY various vti1-ts mu- <br /> tants (Fig. 8). Whereas vti1-1 mutant cells secreted 44% of <br /> CPY 368C (average experiments; Fig. 8 a, lane 2) <br /> and sorted 6% CPY vacuole, overexpression <br /> of Pep12p led significant decrease CPY secretion <br /> (26%, lane 6) increase intracellular mature CPY <br /> (30%, lane 5). This dramatic increase intracellular CPY <br /> (6&#226;&#8364;&#8220;30%) Pep12p overexpression reflects highly <br /> significant suppression CPY sorting defect vti1-1 <br /> mutant cells. This suppression allele specific because <br /> overexpression Pep12p did suppress sorting de- <br /> fect observed vti1-2 mutant cells (data shown). By <br /> contrast, overexpression Sed5p did effect <br /> on CPY sorting vti1-1 cells (Fig. 8 a, lanes 3 4). <br /> These data indicate vti1-1 mutant cells,                     Figure 8. CPY sorting vti1-1 (a) vti1-11 (b) cells over- <br /> defective transport CPY late-Golgi appara-                      expressing Sed5p Pep12p. Overexpression Pep12p vti1-1 <br /> tus vacuole, overexpression prevacuolar-                     cells partially restored CPY sorting vacuole restrictive <br /> localized t-SNARE Pep12p restored ability vesicles                      temperature (368C). Overexpression Sed5p vti1-11 mutant <br /> to dock fuse prevacuolar compartment.                             cells reduced p1CPY accumulation restrictive temperature. <br />  <br />  <br />  <br />  <br /> Fischer von Mollard et al. The v-SNARE Vti1p Interacts Sed5p Pep12p   1519 <br />  to 44% (average experiments). By contrast, over- <br /> expression Pep12p vti1-11 cells did suppress the <br /> CPY missorting defect accumulation p1CPY (Fig. <br /> 8 b, lanes 5 6). These results indicate vti1-11 mu- <br /> tant cells, blocked traffic early-Golgi <br /> compartment, overexpression early-Golgi t-SNARE <br /> Sed5p (but Pep12p) promoted CPY transport the <br /> ER Golgi compartment effect trans- <br /> port CPY late-Golgi compartment vacuole. <br />  <br /> Vti1p Interacts Cytosolic Domains Pep12p <br /> and Sed5p <br /> To determine Vti1p Pep12p interact directly, <br /> Vti1p immunoprecipitated yeast lysates solubi- <br /> lized 0.5% Triton X-100. The native immunoprecipi- <br /> tation samples analyzed SDS-PAGE immuno- <br /> blotted Vti1p, Pep12p, control membrane <br /> proteins. Approximately 10% total Pep12p co- <br /> precipitated Vti1p (Fig. 9 a). Importantly, preimmune <br /> serum did precipitate Pep12p, anti-Vti1p antibod- <br /> ies did coprecipitate membrane proteins, <br /> Vph1p (a vacuolar membrane protein [Manolson et al., <br /> 1992]) Sac1p (ER Golgi membrane protein [May- <br /> inger et al., 1995]). These data indicate Vti1p and <br /> Pep12p interact specifically. <br />    Because Vti1p identified two-hybrid screen <br /> with cytosolic domain Vps10p &#226;&#8364;&#339;bait,&#226;&#8364;? investi- <br /> gated Vti1p Vps10p interacted physically. <br /> Only low levels CPY receptor, Vps10p (,2% <br /> of total Vps10p), coprecipitated Vti1p na- <br /> tive immunoprecipitation conditions (data shown), <br /> and additional experiments suggest effect be <br /> nonspecific. Specific binding Vps10p Vti1p            Figure 9. Physical interaction Vti1p Sed5p Pep12p. <br /> could detected using chemical crosslinking          (a) Coimmunoprecipitation Pep12p Vti1p. Yeast sphero- <br /> followed immunoprecipitation vitro binding as-           plasts solubilized 0.5% Triton X-100. Vti1p quanti- <br /> says soluble cytosolic domains. This reflect       tatively precipitated Triton extract native immuno- <br /> low degree colocalization Vps10p Vti1p          precipitation anti-Vti1p antibodies, z10% total <br />                                                                   Pep12p coimmunoprecipitated. In contrast, membrane <br /> that proteins interact transiently. Whether <br />                                                                   proteins Vph1p (vacuole) Sac1p (ER Golgi apparatus) <br /> Vps10p Vti1p interact functionally remains unclear.           coimmunoprecipitated, pre-immune serum did not <br />    An vitro binding assay used study        precipitate proteins. For quantitative comparison, <br /> interaction Vti1p different yeast t-SNAREs. The           Triton extracts equivalent 30 10% total protein present <br /> soluble cytosolic domain Vti1p expressed E. coli        immunoprecipitations analyzed lanes 3 4. The <br /> and purified 6-His&#226;&#8364;&#8220;fusion protein. GST-fusion pro-           proteins bound beads starting extracts ana- <br /> teins soluble domains yeast t-SNAREs              lyzed SDS-PAGE immunoblotting antibodies against <br /> Pep12p (prevacuolar compartment [Becherer et al., 1996]),         Vti1p, Pep12p, Vph1p, Sac1p. (b) In vitro binding sol- <br /> Sed5p (cis-Golgi compartment [Hardwick Pelham,                uble domain Vti1p immobilized GST fusion proteins con- <br /> 1992]), Sso2p (plasma membrane [Aalto et al., 1993])          taining soluble domains different yeast t-SNAREs. Vti1p <br />                                                                   interacted GST-Sed5p GST-Pep12p GST- <br /> were expressed E. coli purified. Each GST- <br />                                                                   Sso2p GST alone. Equal amounts immobilized GST fusion <br /> fusion proteins immobilized glutathione agarose            proteins incubated 6-His&#226;&#8364;&#8220;Vti1p. The bound proteins <br /> incubated constant purified 6-His&#226;&#8364;&#8220;           analyzed SDS-PAGE immunoblotting anti- <br /> Vti1p (see Materials Methods). The pellets washed        Vti1p antibodies. <br /> and bound protein eluted, separated SDS-PAGE, <br /> and analyzed immunoblotting bound Vti1p (Fig. <br /> 9 b). High levels Vti1p bound GST&#226;&#8364;&#8220;Sed5p fusion. <br /> Consistent coimmunoprecipitation data, Vti1p also <br />                                                                   Discussion <br /> bound GST&#226;&#8364;&#8220;Pep12p. By contrast, Vti1p did bind           We identified new v-SNARE, Vti1p, functions <br /> GST&#226;&#8364;&#8220;Sso2p GST alone. These data indicate Vti1p         membrane traffic yeast vacuole. More impor- <br /> interacts specifically prevacuolar t-SNARE (Pep12p)        tantly, studies revealed single v-SNARE <br /> and cis-Golgi t-SNARE (Sed5p) t-SNARE            function separate vesicle trafficking steps. The yeast <br /> (Sso2p) involved pathway (secretory vesicle fusion      v-SNARE Vti1p interact t-SNAREs, <br /> the plasma membrane) unaffected vti1 mutant cells.     cis-Golgi compartment (Sed5p) the <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                     1520 <br />  second prevacuolar compartment (Pep12p). The                            invertase observed nonpermissive tempera- <br /> specificity v- t-SNAREs                         ture sly2/sec22D cells (Ossig et al., 1991). Sec22p/Sly2p <br /> necessarily require intimate involvement additional                     v-SNARE involved vesicle transport ER to <br /> proteins (e.g., Rab/Ypt proteins, Sec1 protein family mem-                     early-Golgi compartment. These similarities indicate <br /> bers, and/or proteins) single v-SNARE                      trafficking and/or early-Golgi compart- <br /> control fusion different classes vesicles.                       ment blocked severely slowed vti1-11 mutant <br />                                                                                cells. The mechanisms causing underglycosylation un- <br /> Role Vti1p Golgi Prevacuole Membrane Traffic                          clear. Possible explanations indirect effects due <br />                                                                                failure retrieve glycosylation enzymes TGN <br /> Upon depletion Vti1p, yeast cells secreted vacuolar <br />                                                                                prevacuolar compartment, conditions the <br /> hydrolase CPY, <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>phenotype</span> characteristic vacuolar <br />                                                                                Golgi lumen unfavorable glycosylation en- <br /> protein sorting (vps) mutants (Stack et al., 1995). This re- <br />                                                                                zymes. <br /> sult confirmed isolation temperature- <br />                                                                                   The cytosolic domains Vti1p early-Golgi <br /> sensitive vti1 mutants, vti1-1 vti1-2, displayed CPY <br />                                                                                t-SNARE Sed5p (Hardwick Pelham, 1992) were <br /> secretion rapid onset shift restrictive temper- <br />                                                                                interact in vitro binding assay. The binding <br /> ature rapid recovery return permissive <br />                                                                                data suggest Vti1p Sed5p form SNARE com- <br /> temperature. Therefore, Vti1p directly involved trans- <br />                                                                                plex required membrane transport cis-Golgi <br /> port CPY late Golgi compartment vacuole. <br />                                                                                compartment (Fig. 10). Genetic analysis revealed strong <br />    Recently, Pep12p identified t-SNARE involved <br />                                                                                interaction VTI1 SED5 genes. Strains in <br /> in sorting CPY, protein localized pre- <br />                                                                                VTI1 deleted able grow Sed5p was <br /> vacuolar compartment, suggesting acts docking <br />                                                                                overexpressed, indicating SED5 bypass suppres- <br /> and fusion Golgi-derived transport vesicles (Becherer <br />                                                                                sor lethality vti1D mutation. In addition, <br /> et al., 1996). Therefore, tested Vti1p the <br />                                                                                overexpression Sed5p vti1-11 cells suppressed ac- <br /> v-SNARE partner Pep12p. Overexpression Pep12p <br />                                                                                cumulation p1CPY, indicating excess Sed5p over- <br /> in vti1-1 strain suppressed CPY sorting defect sig- <br />                                                                                came cis-Golgi transport block cells. However, <br /> nificantly restrictive temperature. This suppression <br />                                                                                Sed5p overexpression suppress cis-Golgi <br /> did represent general bypass Vti1p-dependent <br />                                                                                transport block vti1-11 cells, Sed5p overexpression had <br /> step allele specific. Native immunoprecipi- <br />                                                                                effect vacuolar protein sorting defect vti1 <br /> tation vitro binding assay demonstrated physical <br />                                                                                mutant (vti1-1, vti1-11, vti1D/2m-SED5). These data are <br /> interaction Vti1p Pep12p. In addition, Vti1p <br />                                                                                inconsistent Vti1p&#226;&#8364;&#8220;Sed5p vitro binding <br /> localized Golgi apparatus prevacuolar <br />                                                                                data. SED5 able act bypass suppressor a <br /> compartment, consistent role Golgi prevacu- <br />                                                                                VTI1 deletion second protein transport vesicles <br /> olar traffic. vti1-1 cells accumulate small ves- <br />                                                                                interacts Vti1p facilitates formation the <br /> icles restrictive temperature. Taken together, these <br />                                                                                Sed5p&#226;&#8364;&#8220;Vti1p SNARE-complex. This protein allow a <br /> data indicate Vti1p&#226;&#8364;&#8220;Pep12p SNARE complex forma- <br />                                                                                efficient fusion transport vesicles high levels of <br /> tion required consumption Golgi-derived trans- <br />                                                                                Sed5p present target membrane. In fact, has <br /> port vesicles prevacuolar compartment (Fig. 10). <br />                                                                                demonstrated complex v-SNAREs <br />                                                                                Bos1p Sec22p ER-derived transport vesicles nec- <br /> Role Vti1p Early-Golgi Membrane Traffic                                  essary efficient formation SNARE complex with <br /> In contrast VPS genes (Stack et al., 1995), VTI1                  Sed5p ER cis-Golgi transport (Lian et al., 1994; Jiang et <br /> essential yeast gene, suggesting second                     al., 1995). <br /> function addition regulating membrane traffic                        It important note block ER Golgi traffic <br /> vacuole. The mutant vti1-11 grew slowly rapidly                       does necessarily implicate Vti1p exact traffick- <br /> accumulated ER early-Golgi form proCPY                              ing step. It reported recently block retrograde <br /> (p1CPY) shift restrictive temperature. CPY                         traffic Golgi apparatus ER leads rapid <br /> sorting recovered quickly return permissive                       block anterograde traffic ER Golgi ap- <br /> temperature. The core glycosylated ER form                             paratus (Letourneur et al., 1994; Lewis Pelham, 1996). <br /> slightly larger a1,6-linked mannose-containing, cis-Golgi                      In fact differences lag p1CPY <br /> form secreted protein invertase accumulated intra-                      accumulated cells conditionally defective proteins <br /> cellularly vti1-11 strain restrictive tempera-                   involved anterograde traffic (sec18, sec12) retrograde <br /> ture. The fact invertase received initial                    traffic (ufe1, sec20, sec21). It unlikely Vti1p acts to- <br /> a1,6-mannose modification indicates glycosylation                      gether v-SNAREs Sec22p/Sly2p, Bet1p/Sly12p, and <br /> the cis-Golgi compartment normal. The remainder                          Bos1p ER cis-Golgi traffic, Sec22p, Bet1p, <br /> the invertase secreted severely underglycosy-                      Bos1p localized predominantly ER a <br /> lated. This suggests invertase failed receive a1,6-                    combination immunofluorescence subcellular frac- <br /> mannose chain elongation, occurs medial-                          tionation (Newman et al., 1992; Jiang et al., 1995), whereas <br /> Golgi compartment. These defects invertase secretion                        Vti1p detected ER. <br /> and processing reminiscent phenotype                            We propose Vti1p involved retrograde traffic <br /> ypt1-1 mutant (Segev et al., 1988). Ypt1p acts ER                        cis-Golgi compartment yeast. Whereas antero- <br /> early-Golgi early- medial-Golgi vesicle fusion                       grade Golgi traffic studied extensively, known <br /> steps (Jedd et al., 1995). An accumulation ER core gly-                     retrograde transport. Because v-SNAREs bud with <br /> cosylated invertase secretion partially glycosylated                    transport vesicles donor compartment in- <br />  <br />  <br />  <br />  <br /> Fischer von Mollard et al. The v-SNARE Vti1p Interacts Sed5p Pep12p   1521 <br />                                                                                                        Figure 10. Model Vti1p <br />                                                                                                       function vesicle docking <br />                                                                                                       fusion. Yeast Golgi- <br />                                                                                                       derived vesicles contain the <br />                                                                                                       v-SNARE Vti1p the <br />                                                                                                       membrane allowing its <br />                                                                                                       interaction t-SNARE <br />                                                                                                       Pep12p surface the <br />                                                                                                       prevacuolar      compartment <br />                                                                                                       (PVC). This interaction pro- <br />                                                                                                       vides specificity <br />                                                                                                       fusion Golgi-derived <br />                                                                                                       vesicles prevacuole. <br />                                                                                                       In addition, Vti1p interacts <br />                                                                                                       early Golgi t-SNARE <br />                                                                                                       Sed5p. We propose Vti1p <br />                                                                                                       involved retrograde <br />                                                                                                       traffic pre- <br />                                                                                                       vacuolar compartment the <br />                                                                                                       late Golgi compartment back <br />                                                                                                       early Golgi compart- <br />                                                                                                       ment. Other v-SNAREs and <br />                                                                                                       t-SNAREs functioning in <br />                                                                                                       yeast ER, Golgi, plasma <br />                                                                                                       membrane (PM) traffic are <br />                                                                                                       included. E, early Golgi; M, <br />                                                                                                       medial Golgi; L, late Golgi; <br />                                                                                                       PVC, prevacuolar compart- <br />                                                                                                       ment; PM, plasma membrane. <br />  <br />  <br /> corporated target membrane              identifying putative proteins retrieval the <br /> mechanism v-SNAREs components                cis-Golgi compartment required secretion yeast. <br /> the transport machinery recycle donor compart- <br /> ment. Recently, evidence retrograde traffic me- <br /> dial- cis-Golgi compartment yeast reported <br />                                                                    Implications SNARE Hypothesis <br /> (Banfield et al., 1995). It reported cis-        In original form, SNARE hypothesis proposed that <br /> Golgi enzyme Och1p rapidly reaches TGN returns             specific set v-SNAREs transport vesicles a <br /> to cis-Golgi compartment retrograde transport               specific set t-SNAREs target membranes serve as <br /> (Harris Waters, 1996). The localization medial-         identifying markers. Their specific pairwise recognition <br /> Golgi protein Emp47p involves retrograde transport          ensure transport vesicles fuse with <br /> ER (Schr&#195;&#182;der et al., 1995). Recycling late-Golgi pro-           correct target organelle (S&#195;&#182;llner et al., 1993; Ferro-Novick <br /> teins prevacuolar compartment Golgi           Jahn, 1994; Rothman, 1994). Recently extension of <br /> apparatus demonstrated (Cereghino et al.,            original SNARE hypothesis necessary the <br /> 1995; Piper et al., 1995; Cooper Stevens, 1996; Noth-          discovery Sed5p interact the <br /> wehr et al., 1996), unclear Golgi stack serves     v-SNAREs Sec22p/Sly2p, Bet1p/Sly12p, Bos1p that <br /> as acceptor membrane. In examples           required ER early Golgi traffic the <br /> clear retrograde traffic moves com-              v-SNARE Sft1p, involved retrograde traffic <br /> partment compartment, mirroring anterograde flow,            later Golgi early-Golgi compartment (Ban- <br /> whether retrograde transport vesicles different com-          field et al., 1995). Vti1p-containing vesicles repre- <br /> partments common target, e.g., ER cis-           sent kind vesicle fusing early-Golgi com- <br /> Golgi compartment (Rothman Wieland, 1996).                     partment. This mean t-SNARE defines a <br />    Vti1p probably involved retrograde traffic       target organelle interact v-SNAREs in- <br /> the medial- early-Golgi compartment           coming vesicles organelle, instead re- <br /> the v-SNARE Sft1p, phenotypes resulting           stricted specific vesicle fusion event. <br /> vti1 sft1 mutations different. At restrictive tem-        We demonstrate genetic biochemical stud- <br /> perature, sft1-ts mutants accumulated large amounts             ies single v-SNARE interact functionally two <br /> Golgi membranes early-Golgi ER                 different t-SNAREs distinct compartments. A model to <br /> form invertase (Banfield et al., 1995). An attractive pos-      explain data v-SNARE functioning antero- <br /> sibility Vti1p involved retrograde vesicle traffic   grade trafficking target organelle plays critical <br /> from prevacuolar compartment cis-Golgi             role docking fusion retrograde vesicles originat- <br /> compartment. Alternatively, Vti1p control retro-             ing target organelle, v-SNARE itself is <br /> grade traffic late-Golgi cis-Golgi compart-        recycling. Alternatively, Vti1p directs anterograde <br /> ment, possible pathway Och1p retrieval yeast              retrograde traffic late-Golgi compartment <br /> (Harris Waters, 1996). Future efforts directed         mean v-SNARE present different <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 137, 1997                      1522 <br />  vesicle populations budding single donor compart-                               totic fusion machine. EMBO (Eur. Mol. Biol. Organ.) J. 14:224&#226;&#8364;&#8220;231. <br />                                                                                     Esmon, B., P. Novick, R. Schekman. 1981. Compartmentalized assembly of <br /> ment define donor membranes. To interact                                   oligosaccharides exported glycoproteins yeast. Cell. 25:451&#226;&#8364;&#8220;460. <br /> specifically anterograde retrograde                          Farquhar, M.G., G.E. Palade. 1981. The Golgi apparatus (complex)-(1954&#226;&#8364;&#8220; <br /> t-SNARE, v-SNARE able assume                                    1981)-from artifact center stage. J. Cell. Biol. 91:77S&#226;&#8364;&#8220;103S. <br />                                                                                     Ferro-Novick, S., R. Jahn. 1994. Vesicle fusion yeast man. Nature <br /> two different conformations, modified states, form                              (Lond.). 370:191&#226;&#8364;&#8220;193. <br /> complexes additional proteins modify action                           Franzusoff, A., R. Schekman. 1989. Functional compartments yeast <br />                                                                                        Golgi apparatus defined sec7 mutation. EMBO (Eur. Mol. Biol. <br /> of v-SNARE. A protein able fulfill type                           Organ.) J. 8:2695&#226;&#8364;&#8220;2702. <br /> function identified mammalian neurons. The                              Golemis, E., J. Gyuris, R. Brent. 1996. Interaction trap/two-hybrid systems <br /> v-SNARE synaptobrevin forms complex syn-                                    identify interacting proteins. In Current Protocols Molecular Biology. <br />                                                                                        F.M. Ausubel, editor. John Wiley &amp; Sons, New York. 20.1.1&#226;&#8364;&#8220;20.1.28. <br /> aptic vesicle protein synaptophysin, excludes                             Graham, T.R., S.D. Emr. 1991. Compartmental organization Golgi-spe- <br /> t-SNARE syntaxin 1 (Edelmann et al., 1995), suggesting                               cific protein modification vacuolar protein sorting events defined a <br /> role synaptophysin control synaptobrevin activity.                           yeast sec18 (NSF) mutant. J. Cell Biol. 114:207&#226;&#8364;&#8220;218. <br />                                                                                     Hardwick, K.G., H.R. Pelham. 1992. SED5 encodes 39-kD integral mem- <br /> Therefore, likely activity possibly                          brane protein required vesicular transport ER Golgi <br /> specificity v-SNAREs regulated interactions                             complex. J. Cell Biol. 119:513&#226;&#8364;&#8220;521. <br />                                                                                     Harris, S.L., M.G. Waters. 1996. Localization yeast early Golgi manno- <br /> additional proteins single v-SNARE direct                                syltransferase, Och1p, involves retrograde transport. J. Cell Biol. 132:985&#226;&#8364;&#8220;998. <br /> vesicle fusion different classes vesicles.                                Hill, J.E., A.M. Myers, T.J. Koerner, A. Tzagologg. 1986. E. coli shuttle <br />                                                                                        vectors multiple unique restriction sites. Yeast. 2:163&#226;&#8364;&#8220;167. <br /> We thank J. Selker superb electron microscopy work G. Hamilton              Jahn, R., T.C. S&#195;&#188;dhof. 1994. Synaptic vesicles exocytosis. Annu. Rev. <br /> for help two-hybrid screen. We acknowledge R.C. Piper S.R.                Neurosci. 17:219&#226;&#8364;&#8220;246. <br /> Gerrard providing PEP12 plasmids anti-Pep12p antiserum. N.J.                Jedd, G., C. Richardson, R. Litt, N. Segev. 1995. The Ypt1 GTPase essen- <br />                                                                                        tial steps yeast secretory pathway. J. Cell Biol. 131: <br /> Bryant, R.C. Piper, W. Voos, E. Conibear, G. Flynn, G.F. Sprague, Jr.              583&#226;&#8364;&#8220;590. <br /> are thanked stimulating discussions critical reading manu-           Jiang, Y., M. Sacher, B. Singer-Kr&#195;&#188;ger, J.P. Lian, S. Stone, S. Ferro-Novick. <br /> script. We thank M. Snyder supplying CIK1 plasmid, R. Brent                 1995. Factors mediating late stages ER-to-Golgi transport yeast. <br /> and S. Fields two-hybrid plasmids strains, R. Schekman anti-           Cold Spring Harbor Symp. Quant. Biol. LX, 119&#226;&#8364;&#8220;126. <br />                                                                                     Jones, E.W. 1977. Proteinase mutants Saccharomyces cerevisiae. Genetics. 85: <br /> serum a1,6-linked mannose, P. Orlean anti-Dpm1p anti-                  23&#226;&#8364;&#8220;33. <br /> bodies DPM1 plasmids.                                                           Kaiser, C.A., R. Schekman. 1990. Distinct sets SEC genes govern trans- <br />    This work supported grants Deutsche Forschungs-                     port vesicle formation fusion early secretory pathway. Cell. 61: <br /> gemeinschaft, American Cancer Society (S.F. Nothwehr; PF-                     723&#226;&#8364;&#8220;733. <br />                                                                                     Laemmli, U.K. 1970. Cleavage structural proteins assembly the <br /> 3608), National Institutes Health (T.H. Stevens;                       head bacteriophage T4. Nature (Lond.). 227:680&#226;&#8364;&#8220;685. <br /> GM32448).                                                                           Letourneur, F., E.C. Gaynor, S. Hennecke, C. D&#226;&#8364;&#8482;Emolliere, R. Duden, S.D. <br />                                                                                        Emr, H. Riezman, P. Cosson. 1994. Coatomer essential retrieval of <br /> Received publication 20 February 1997 revised form 8 April                  dilysine-tagged proteins endoplasmic reticulum. Cell. 79:1199&#226;&#8364;&#8220;1207. <br /> 1997.                                                                               Lewis, M.J., H.R. Pelham. 1996. SNARE-mediated retrograde traffic from <br />                                                                                        Golgi complex endoplasmic reticulum. Cell. 85:205&#226;&#8364;&#8220;215. <br />                                                                                     Lian, J.P., S. Stone, Y. Jiang, P. Lyons, S. Ferro-Novick. 1994. Ypt1p impli- <br /> References                                                                             cated v-SNARE activation. Nature. 372:698&#226;&#8364;&#8220;701. <br />                                                                                     Lupas, A., M. Van Dyke, J. Stock. 1991. Predicting coiled coils pro- <br /> Aalto, M.K., H. Ronne, S. Keranen. 1993. Yeast syntaxins Sso1p Sso2p           tein sequences. Science (Lond.). 252:1162&#226;&#8364;&#8220;1164. <br />   belong family related membrane proteins function vesicular        Maina, C.V., P.D. Riggs, A.G.D. Grandea, B.E. Slatko. 1988. An Escheri- <br />   transport. EMBO (Eur. Mol. Biol. Organ.) J. 12:4095&#226;&#8364;&#8220;4104.                            chia coli vector express purify foreign proteins fusion sepa- <br /> Banfield, D.K., M.J. Lewis, H.R. Pelham. 1995. A SNARE-like protein re-            ration maltose-binding protein. Gene. 74:365&#226;&#8364;&#8220;373. <br />   quired traffic Golgi complex. Nature (Lond.). 375:806&#226;&#8364;&#8220;809.        Manolson, M.F., D. Proteau, R.A. Preston, A. Stenbit, B.T. Roberts, M.A. <br /> Bankaitis, V.A., L.M. Johnson, S.D. Emr. 1986. Isolation yeast mutants          Hoyt, D. Preuss, J. Mulholland, D. Botstein, E.W. Jones. 1992. The <br />   defective protein targeting vacuole. Proc. Natl. Acad. Sci. USA. 83:       VPH1 gene encodes 95-kDa integral membrane polypeptide required for <br />   9075&#226;&#8364;&#8220;9079.                                                                           vivo assembly activity yeast vacuolar H(1)-ATPase. J. Biol. <br /> Bartel, P.L., S. Fields. 1995. Analyzing protein-protein interactions using        Chem. 267:14294&#226;&#8364;&#8220;14303. <br />   two-hybrid system. Methods Enzymol. 254:241&#226;&#8364;&#8220;263.                                  Marcusson, E.G., B.F. Horazdovsky, J.L. Cereghino, E. Gharakhanian, and <br /> Becherer, K.A., S.E. Rieder, S.D. Emr, E.W. Jones. 1996. Novel syntaxin            S.D. Emr. 1994. The sorting receptor yeast vacuolar carboxypeptidase Y <br />   homolog, Pep12p, required sorting lumenal hydrolases lyso-         encoded VPS10 gene. Cell. 77:579&#226;&#8364;&#8220;586. <br />   some-like vacuole yeast. Mol. Biol. Cell. 7:579&#226;&#8364;&#8220;594.                           Mayinger, P., V.A. Bankaitis, D.I. Meyer. 1995. Sac1p mediates aden- <br /> Berger, B., D.B. Wilson, E. Wolf, T. Tonchev, M. Milla. 1995. Predicting           osine triphosphate transport yeast endoplasmic reticulum re- <br />   coiled coils use pairwise residue correlations. Proc. Natl. Acad. Sci.         quired protein translocation. J. Cell. Biol. 131:1377&#226;&#8364;&#8220;1386. <br />   USA. 92:8259&#226;&#8364;&#8220;8263.                                                                Muhlrad, D., R. Hunter, R. Parker. 1992. A rapid method localized mu- <br /> Boeke, J.D., F. LaCroute, G.R. Fink. 1984. A positive selection mutants        tagenesis yeast genes. Yeast. 8:79&#226;&#8364;&#8220;82. <br />   lacking orotidine-59-phosphate decarboxylase activity yeast: 5-fluoro-         Newman, A.P., J. Shim, S. Ferro-Novick. 1990. BET1, BOS1, SEC22 <br />   orotic acid resistance. Mol. Gen. Genet. 197.                                        members group interacting yeast genes required transport <br /> Brent, R., M. Ptashne. 1985. A eukaryotic transcriptional activator bearing        endoplasmic reticulum Golgi complex. Mol. Cell. Biol. 10: <br />   DNA specificity prokaryotic repressor. Cell. 43:729&#226;&#8364;&#8220;736.                    3405&#226;&#8364;&#8220;3414. <br /> Carlson, M., D. Botstein. 1982. Two differentially regulated mRNAs         Newman, A.P., M.E. Groesch, S. Ferro-Novick. 1992. Bos1p, membrane <br />   different 59 ends encode secreted proteins intracellular forms yeast         protein required ER Golgi transport yeast, co-purifies car- <br />   invertase. Cell. 28:145&#226;&#8364;&#8220;154.                                                         rier vesicles Bet1p ER membrane. EMBO (Eur. Mol. Biol. <br /> Cereghino, J.L., E.G. Marcusson, S.D. Emr. 1995. The cytoplasmic tail do-          Organ.) J. 11:3609&#226;&#8364;&#8220;3617. <br />   main vacuolar protein sorting receptor Vps10p subset VPS          Nothwehr, S.F., T.H. Stevens. 1994. Sorting membrane proteins the <br />   gene products regulate receptor stability, function, localization. Mol.          yeast secretory pathway. J. Biol. Chem. 269:10185&#226;&#8364;&#8220;10188. <br />   Biol. Cell. 6:1089&#226;&#8364;&#8220;1102.                                                          Nothwehr, S.F., N.J. Bryant, T.H. Stevens. 1996. The newly identified yeast <br /> Chien, C.T., P.L. Bartel, R. Sternglanz, S. Fields. 1991. The two-hybrid sys-      GRD genes required retention late-Golgi membrane proteins. <br />   tem: method identify clone genes proteins interact          Mol. Cell. Biol. 16:2700&#226;&#8364;&#8220;2707. <br />   protein interest. Proc. Natl. Acad. Sci. USA. 88:9578&#226;&#8364;&#8220;9582.                    Orlean, P. 1990. Dolichol phosphate mannose synthase required vivo for <br /> Cooper, A.A., T.H. Stevens. 1996. Vps10p cycles late-Golgi             glycosyl phosphatidylinositol membrane anchoring, O mannosylation, N <br />   prevacuolar compartments function sorting receptor             glycosylation protein Saccharomyces cerevisiae. Mol. Cell. Biol. 10: <br />   multiple yeast vacuolar hydrolases. J. Cell. Biol. 133:529&#226;&#8364;&#8220;541.                      5796&#226;&#8364;&#8220;5805. <br /> Dascher, C., R. Ossig, D. Gallwitz, H.D. Schmitt. 1991. Identification      Ossig, R., C. Dascher, H.H. Trepte, H.D. Schmitt, D. Gallwitz. 1991. The <br />   structure yeast genes (SLY) able suppress functional         yeast SLY gene products, suppressors defects essential GTP-bind- <br />   loss YPT1, member RAS <br /> </body></html>