<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />   Neurospora crassa heat shock factor 1 Is an Essential Gene; a Second <br />          Heat Shock Factor-Like Gene, hsf2, Is Required for <br />                       Asexual Spore Formation <br />  <br />  <br />              Appropriate responses of organisms <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> heat stress are essential for their <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span>. In eukaryotes, adaptation <br />           to high temperatures is mediated by heat shock transcription factors (HSFs). HSFs regulate the expression of <br />           heat shock proteins, which function as molecular chaperones assisting in protein folding and stability. In many <br />           model organisms a great deal is known about the products of hsf genes. An important exception is the <br />           filamentous fungus and model eukaryote Neurospora crassa. Here we show that two Neurospora crassa genes <br />           whose protein products share similarity to known HSFs play different biological roles. We report that heat <br />           shock factor 1 (hsf1) is an essential gene and that hsf2 is required for asexual development. Conidiation may be <br />           blocked in the hsf2 <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>knockout</span> (hsf2KO) strain because HSF2 is an integral element of the conidiation pathway <br />           or because it affects the availability of protein chaperones. We report that genes expressed during conidiation, <br />           for example fluffy, conidiation-10, and repressor of conidiation-1 show wild-type levels of expression in a hsf2KO <br />           strain. However, consistent with the lack of macroconidium development, levels of eas are much reduced. <br />           Cultures of the hsf2KO strain along with two other aconidial strains, the fluffy and aconidial-2 strains, took <br />           longer than the wild type to recover <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> heat shock. Altered expression profiles of hsp90 and a putative <br />           hsp90-associated protein in the hsf2KO strain after exposure to heat shock may in part account for its reduced <br />           ability to cope with heat stress. <br />  <br />  <br />    The heat shock response is a protective mechanism against                   mRNA which encode HSF isoforms with different transcrip- <br /> sudden temperature change and other forms of physical and                      tional activities (16). Although the functional significance of <br /> chemical stress. The response is brought into effect by heat                   the HSF isoforms is not yet understood clearly, one possibility <br /> shock proteins (HSPs), molecular chaperones that assist pro-                   is that they activate different subsets of HSP or stress-respon- <br /> tein folding and stability (35). Several highly conserved families             sive genes. Moreover, in species with multiple hsf genes, a <br /> of HSPs, named according to their molecular masses, are ex-                    functional specialization of certain HSFs toward developmen- <br /> pressed in response to heat shock and include HSP110,                          tal roles has occurred. For example, of the four vertebrate <br /> HSP100, HSP90, HSP70, HSP60, and a number of small HSPs.                       HSFs, HSF1 mediates heat shock gene expression, whereas <br /> Some of the family members are expressed in non-heat shock                     HSF2 is reported to be unresponsive to temperature, while its <br /> conditions, but the expression of others is induced only upon                  activity changes throughout differentiation. In addition, alter- <br /> exposure to stress due to the action of heat shock transcription               native splicing of hsf1, hsf2, and hsf4 transcripts generates a <br /> factors (HSFs). HSFs are a functionally conserved group of <br />                                                                                variety of HSF isoforms (13, 18, 45). Since these splice forms <br /> molecules. They were first identified as regulators of the heat <br />                                                                                show tissue-specific expression, it is likely that they also have <br /> shock response (reviewed in reference 28), but it is now evi- <br />                                                                                distinct roles. <br /> dent that certain HSFs also regulate developmental pathways <br />                                                                                   In contrast, no Neurospora HSFs have yet been isolated, <br /> (for a review, see reference 36). <br />    Drosophila melanogaster and Saccharomyces cerevisiae each                   although HSPs are expressed in this organism during heat <br /> have one essential HSF (7, 48). In S. cerevisiae, while the                    shock and cell differentiation (14, 19) suggesting that, as seen <br /> induction of genes encoding chaperones and associated pro-                     for other organisms, the Neurospora HSF(s) is required for <br /> teins is under the control of its HSF, a subset of genes induced               development. Neurospora crassa undergoes both sexual and <br /> by a variety of environmental stresses and encoding metabolic                  asexual developmental cycles. When nitrogen is limiting, fe- <br /> enzymes, components of the ubiquitin-proteosome degrada-                       male sexual development ensues, and when nuclei of opposite <br /> tion pathway, and antioxidant defense proteins are under the                   mating types fuse numerous asci are produced, each containing <br /> control of the transcription factors Msn2, Msn4, Yap1, and                     eight haploid spores (42). In addition, two asexual develop- <br /> Skn7 (5, 17, 6, 25). In Drosophila, temperature-dependent al-                  mental pathways are described: macroconidiation and micro- <br /> ternative splicing of dHSF gives rise to four forms of dHSF                    conidiation. Macroconidial development in Neurospora can be <br />                                                                                induced by desiccation, <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation (47), light (23), <br />                                                                                oxidative stress (29), and signals relayed from the organism&#226;&#8364;&#8482;s <br />    * Corresponding author. Mailing address: Faculty of Life Sciences,          circadian clock (37). Once induced, aerial hyphae are formed <br /> University of Manchester, Oxford Road, Manchester M13 9PT, United <br />                                                                                which branch and eventually form chains of macroconidia, <br /> Kingdom. Phone: 44-161-275-5996. Fax: 44-161-275-5082. E-mail: susan <br /> .k.crosthwaite@manchester.ac.uk.                                               each containing between one and five nuclei (44). In a dis- <br />    &#228;&#338;&#164; <br />      Published ahead of print on 27 June 2008.                                 tinctly different developmental process, microconidia contain- <br />  <br />                                                                         1573 <br />  1574       THOMPSON ET AL.                                                                                                                           EUKARYOT. CELL <br />  <br />  <br /> ing one or rarely two nuclei are formed from smaller branched                           Liquid culture medium for the growth of mycelial mats and shaking cultures <br /> microconidiophores (30).                                                             contained 1&#226;&#171;&#187; Vogel&#226;&#8364;&#8482;s salts, 2% glucose, and 500 ng/ml biotin. One hundred <br />                                                                                      milliliters of liquid culture medium was inoculated with 2.3-cm-diameter discs cut <br />     Several phases in the development of macroconidia have                           from mycelial mats. These cultures were put into LL at 25&#194;&#176;C with shaking at 150 <br /> been recorded (31, 44). The first step is the production of                          rpm. After 24 h, cultures were placed into a shaking water bath at 45&#194;&#176;C. Flasks <br /> branching aerial hyphae. Minor constrictions then develop,                           were removed at regular intervals and their contents were immediately harvested <br /> followed by the production of proconidia via budding from the                        by vacuum filtration onto filter paper and frozen in liquid nitrogen. <br />                                                                                         The experiment examining nitrogen starvation-induced conidial development <br /> terminal cells of each branch. Finally, proconidia are separated <br />                                                                                      was first described in the work of Bailey-Shrode and Ebbole (1). Strains cultured <br /> by major constrictions and mature into conidia, which are                            on Vogel&#226;&#8364;&#8482;s minimal medium slants (2% sucrose) were used to inoculate culture <br /> easily separated and dispersed by wind or water. A number of                         dishes of modified Vogel&#226;&#8364;&#8482;s minimal medium (1.5% sucrose) with 50 mM NH4Cl <br /> mutants affecting this developmental pathway are known. The                          in place of NH4NO3. After 3 days at 30&#194;&#176;C, a cork borer was used to punch discs <br /> fluffyoid (fld) and aconidial-2 (acon-2) mutants are blocked                         from the resulting mycelial mats. These discs were then transferred to 100-ml <br />                                                                                      flasks containing 50 ml of modified Vogel&#226;&#8364;&#8482;s minimal medium and incubated for <br /> before and the fluffy (fl) mutant is blocked after the formation                     22 h at 34&#194;&#176;C in LL plus shaking at 150 rpm. The cultures were then washed in <br /> of minor constrictions (31, 44). The acon-3 mutant is blocked                        sterile distilled water and transferred to flasks containing Vogel&#226;&#8364;&#8482;s minimal me- <br /> after the maturation of minor constrictions. conidial separa-                        dium lacking any nitrogen source. Tissue was harvested before and 1 h, 6 h, 12 h, <br /> tion-1 (csp-1) and csp-2 are required for conidial separation                        and 24 h after transfer. <br />                                                                                         Induction of conidiation by exposure to air was carried out essentially as <br /> (41). Genetic and molecular studies place acon-2 upstream of <br />                                                                                      described in the work of Lee and Ebbole (26), with the following modifications. <br /> fl, with acon-3, csp-1, and csp-2 being downstream (9, 1).                           Disks were cut from mycelial mats and placed into Vogel&#226;&#8364;&#8482;s minimal medium with <br />     We report here that Neurospora hsf1 is an essential gene and                     2% glucose. After 24 h in DD at 25&#194;&#176;C, disks were blotted dry and placed on agar <br /> that a deletion strain can be maintained only as a hetero-                           blocks (Vogel&#226;&#8364;&#8482;s minimal medium, 2% glucose, 2% agar) in open petri dishes <br /> karyon. In strains with hsf2 deleted, no dramatic alteration of                      containing 15 ml of fresh culture medium. <br />                                                                                         RT-PCR. The RevertAid H&#226;&#171;&#186; first-strand cDNA synthesis kit (Fermentas) was <br /> the heat shock response is detected. However, macroconidia-                          used for all reverse transcription-PCR (RT-PCR). Primers are listed in Table 1. <br /> tion in hsf2 is defective; aerial hyphae emerge but no macro-                        For semiquantitative RT-PCR, 1 &#226;&#65533;&#174;g of total RNA was reverse transcribed using <br /> conidia are produced. Consistent with this, differences are seen                     the poly(T) primer supplied with the kit, following the manufacturer&#226;&#8364;&#8482;s instruc- <br /> between the wild-type and hsf2 knockout (hsf2KO) strains in the                      tions. A limiting number of cycles were used to stop the subsequent PCRs during <br />                                                                                      the exponential phase of amplification. After agarose gel electrophoresis, quan- <br /> expression of two conidiation-associated transcripts, fl and eas. <br />                                                                                      tification was performed using Bio-Rad QuantityOne software and data were <br />                                                                                      normalized against actin transcript levels. <br />                                                                                         Construction of the hsf1 knockout strain. A hsf1 deletion cassette was made <br />                         MATERIALS AND METHODS                                        containing the hph gene (confers resistance to hygromycin) flanked by approximately <br />     Bioinformatic analysis. Genes were identified by searching the Broad Institute   1.6 kbp 5&#226;&#172;&#732; and 2 kbp 3&#226;&#172;&#732; of the NCU08512 ORF. One hundred microliters of a mus-51 <br /> Neurospora crassa database. Protein prediction was performed using NCBI              conidial cell suspension (2 &#226;&#171;&#187; 109 conidia/ml) in a 2-mm gap cuvette (Bio-Rad) was <br /> BLAST at http://www.ncbi.nlm.nih.gov/BLAST/ and COILS at http://www.ch               electroporated with 0.5 &#226;&#65533;&#174;g of the knockout construct. A Bio-Rad gene pulser was <br /> .embnet.org/software/COILS_form.html. Sequence alignments were performed             used with the following settings: 12.5 kV/cm, 400 &#226;&#65533;&#8364;, 25 &#226;&#65533;&#174;F. Transformants were <br /> using ClustalW at http://www.ebi.ac.uk/clustalw/.                                    selected on hygromycin plates (200 &#226;&#65533;&#174;g/ml) incubated at 30&#194;&#176;C for 4 days. Transfor- <br />     Strains and growth conditions. mus-51a (FGSC9718), hsf2 a (FGSC 11671),          mants were crossed with 87-3 bd a to obtain homokaryons. <br /> acon-2 a (FGSC3263), fl a (FGSC818), and the hsp90a (NCU01792), hsp30                   RNA extraction and Northern blotting. RNA was extracted using Trizol (In- <br /> (NCU09364), hsp78 (NCU02630), hsp101 (NCU00104), hsp70 (NCU08683), and               vitrogen) following the manufacturer&#226;&#8364;&#8482;s protocol. Equal amounts of RNA were <br /> hsp90 (NCU04142) strains were obtained from the Fungal Genetics Stock Center         run through a 1% agarose formaldehyde gel and blotted onto Hybond N&#226;&#171;&#185; <br /> (http://www.fgsc.net/) and crossed into either 30-7 bd A or 87-3 bd a to introduce   nitrocellulose membrane (GE Healthcare). Blots were probed with randomly <br /> the bd <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. 87-3 bd a was obtained from Christian Heintzen (University of       primed 32P-labeled (Roche labeling kit; GE Healthcare radionucleotide) cDNA <br /> Manchester, United Kingdom). fl,bd was a kind gift from D. Bell-Pedersen             or 32P-labeled RNA (MAXIscript kit; Ambion) and detected using phosphorim- <br /> (Texas A&amp;M University). The hsf1KO strain was made by amplifying 1-kbp               ager screens (Fuji). Blots were prehybridized (NorthernMax; Ambion) and hy- <br /> regions 5&#226;&#172;&#732; and 3&#226;&#172;&#732; of the hsf1 open reading frame (ORF) and fusing these to the       bridized at 42&#194;&#176;C for DNA probes and at 68&#194;&#176;C for RNA probes. Blots were <br /> hygromycin phosphotransferase (hph) gene. mus-51a was transformed with this          washed twice for 10 min in 2&#226;&#171;&#187; SSC (1&#226;&#171;&#187; SSC is 0.15 M NaCl plus 0.015 M sodium <br /> fragment and transformants were selected on hygromycin-containing agar plates        citrate) plus 0.1% sodium dodecyl sulfate (SDS), once for 15 min in 1&#226;&#171;&#187; SSC plus <br /> (8). Strains were maintained on minimal medium (1&#226;&#171;&#187; Vogel&#226;&#8364;&#8482;s salts [10], 2%            0.1% SDS at room temperature, and once for 15 min in 0.1&#226;&#171;&#187; SSC plus 0.1% SDS <br /> sucrose, 500 ng/ml biotin, 1.5% agar). Crossing medium contained 1&#226;&#171;&#187; Wester-          at either 42 or 68&#194;&#176;C. <br /> gaard&#226;&#8364;&#8482;s salts, 2% sucrose, 500 ng/ml biotin, and 1.5% agar (10). Spores were            DNA extraction and Southern blotting. Genomic DNA was extracted from <br /> picked onto slants containing Vogel&#226;&#8364;&#8482;s minimal medium, heat shocked at 60&#194;&#176;C for       homogenized tissue by use of a cetyltrimethylammonium bromide protocol. DNA <br /> 1 h to induce germination, and then incubated at 30&#194;&#176;C. Progeny were screened         was digested with NdeI (hsf2) or NdeI and BamHI (hsf1), separated through 0.8% <br /> for hygromycin resistance on minimal medium containing 200 &#226;&#65533;&#174;g/ml of hygro-           agarose gels, and transferred onto Hybond N&#226;&#171;&#185; nitrocellulose membrane. The DNA <br /> mycin B and/or for rhythmic conidiation on race tubes.                               was cross-linked to membranes with UV light. Blots were hybridized to digoxigenin <br />     Race tube medium contained 1&#226;&#171;&#187; Vogel&#226;&#8364;&#8482;s salts, 0.1% glucose, 0.17% arginine,       (DIG)-labeled (Roche) probes against the 5&#226;&#172;&#732;-flanking region of the knockout <br /> 500 ng/ml biotin, and 1.5% agar. After inoculation with 5- to 10-day-old macro-      cassettes (http://www.dartmouth.edu/&#226;&#172;&#402;neurosporagenome/knockouts_completed <br /> conidia, race tubes were placed in constant light (LL) at 25&#194;&#176;C for 48 h and then     .html). Primers for the amplification of probe DNA are listed in Table 1. Each blot <br /> transferred to constant darkness (DD). Growth fronts were marked daily. Heat         was incubated with prehybridization solution (NorthernMAX; Ambion) at 42&#194;&#176;C for <br /> shocks were given by immersing race tubes in a 50&#194;&#176;C water bath for 1 h after 24 h    1 h before the addition of 300 ng of denatured probe in 4 to 5 ml of fresh hybrid- <br /> in the dark at 25&#194;&#176;C. The race tube cultures were then returned to 25&#194;&#176;C to recover    ization solution and incubation at 42&#194;&#176;C overnight. After hybridization, the blots were <br /> and examined daily for regrowth.                                                     washed according to the manufacturer&#226;&#8364;&#8482;s protocol, with the final washes carried out <br />     Microconidiation was induced according to the protocol of Ebbole and Sachs       at 42&#194;&#176;C. Detection of bound DIG-labeled probe was also carried out according to the <br /> (11). Conidia were harvested by vortexing in 2 ml of sterile distilled water and     manufacturer&#226;&#8364;&#8482;s protocol, and the blots were exposed to X-ray film for 5 to 25 min. <br /> filtered through 5-&#226;&#65533;&#174;m filters (Millipore) into sterile Eppendorf tubes. Micro- <br /> conidia were plated onto minimal (1&#226;&#171;&#187; Vogel&#226;&#8364;&#8482;s salts, 1% sorbose, 0.005% fruc- <br /> tose, 0.005% glucose, 1 M sorbitol) or minimal hygromycin-containing plates                                             RESULTS <br /> (200 &#226;&#65533;&#174;g/ml). Plates were incubated at 30&#194;&#176;C until single colonies were detected (3 <br /> or 4 days later). Colonies were picked onto minimal agar slants, and this was <br />                                                                                        Identification of putative HSFs. Two putative Neurospora <br /> followed by the transfer of conidia onto hygromycin-containing (200 &#226;&#65533;&#174;g/ml)           HSFs encoded by hsf1 (NCU08512) and hsf2 (NCU08480) <br /> slants.                                                                              were identified in a BLAST search of the Neurospora genome. <br />  VOL. 7, 2008                                                                   NEUROSPORA HEAT SHOCK FACTOR-LIKE GENES                      1575 <br />  <br />                                                                TABLE 1. Primers used <br />                                                                                                     Primerb <br />        Gene                    Usea <br />                                                                           Forward                                            Reverse <br />  <br /> hsf1             RT/cloning/cDNA probe                CGGGATCCCGCATGTCTTCTCCTAACC                        CGGGATCCCGCGACAGAAGCAACAGAT <br />                  RNA probe/sqRT-PCR (1; 35)           CGGGATCCCGCATGTCTTCTCCTAACC                        TAATACGACTCACTATAGGGACGTGA <br />                                                                                                            GGGCCTGCTGGAATAAGGTCG <br /> hsf2L            RT/cloning/cDNA probe                TAGGATCCTACAGTGCAGATGGCTACC                        TAGGATCCACATAACGTCTCAAGATGC <br />                  RNA probe/sqRT-PCR (2; 30)           TAGGATCCTACAGTGCAGATGGCTACC                        TAATACGACTCACTATAGGGATCCTC <br />                                                                                                            CATCTGGCGCTGAACCTGC <br /> hsf2S            RT/cloning                           TAGGATCCTACAGTGCAGATGGCTACC                        TAGGATCCCAAGCTAACCAGGTCTCCG <br /> hsf2 5&#226;&#172;&#732; flank    Southern blot DNA probe              GTAGCCTCTCCATACCTACC                               GTAGAGTCCATTCTACCTCG <br /> hsp70            DNA probe                            GTCTCGACAAGAAGGTCG                                 TCTCGGAGGTGGACTTGG <br /> hsp90            DNA probe                            GGATCCTTGTTGCCGACAGAGTTA                           GGATCCCATCAATGGGGTCGACAAGG <br /> NCU01792         DNA probe                            AGCGCTCTTCTGCTACCG                                 CATGGATCTTGGGTATGC <br /> fluffy           RNA probe/sqRT-PCR (1; 30)           TGTGATGGCCAGATGCCATGC                              TAATACGACTCACTATAGGGACGAGT <br />                                                                                                            TATCGCTGCCCTCACG <br /> actin            sqRT-PCR (0.5; 20)                   GGCTGGCCGTGATCTTACCGACTA                           GAGAGCGAGGCGAGAATGGAACC <br /> con-10           sqRT-PCR (1; 26)                     CGTCAACATGGCTGGCACTGGTAACG                         TTGTAATACGACTCACTATAGGGCAA <br />                                                                                                            ATTGTCAAAGCATTCAGTTGC <br /> rco-1            sqRT-PCR (1; 35)                     GTACGAGGATGAGATTGCGCT                              TCTGATGAGCTTGTCCTCGGCTC <br />   a <br />     For semiquantitative RT-PCR (sqRT-PCR), the numbers in parentheses indicate the amounts of cDNA in &#226;&#65533;&#174;l followed by the cycle numbers. <br />   b <br />     Modifications are in lightface, and annealing portions are in boldface. <br />  <br />  <br />  <br />  <br /> We cloned RNAs expressed from these genes and analyzed                         mokaryons, microconidiation was induced in two of the het- <br /> their sequences to determine the details of any introns. hsf1                  erokaryons (11). Only five colonies derived from microconidia <br /> contains one intron and encodes a 787-amino-acid (aa) pro-                     were hygromycin resistant, and once again these were hetero- <br /> tein. Alternative splicing of hsf2 RNA gives rise to at least                  karyons (data not shown). In parallel, two of the transformants <br /> three different transcripts encoding polypeptides of 613 aa, 374               were backcrossed to the bd strain. Sixty-five spores derived <br /> aa, and 173 aa or of 579 aa, 340 aa, and 139 aa depending at                   from these crosses were selected and all 39 that germinated <br /> which of the three possible in-frame ATGs translation is initi-                were inoculated onto hygromycin slants. None of these strains <br /> ated (Fig. 1). HSFs from different species, though binding to <br /> similar DNA consensus sequences, are not highly conserved. <br /> However, all contain a DNA-binding domain at the amino <br /> terminus and an adjacent cluster of hydrophobic amino acids <br /> organized into heptad repeats required for oligomerization, <br /> and they often also contain a heptad repeat close to the car- <br /> boxy terminus (49). In order to become functionally active, <br /> HSF protein monomers form homotrimers through the inter- <br /> action of coiled-coil domains found toward the N terminus. <br /> HSF1 and the large HSF2 isoforms contain putative DNA- <br /> binding and trimerization domains. In addition, a carboxy hep- <br /> tad repeat is present in HSF1. In contrast, the 173-aa isoform <br /> of HSF2 contains only a partial DNA-binding domain (Fig. 1). <br /> Although the presence of these alternatively spliced transcripts <br /> in the five cDNAs we cloned and sequenced suggests that they <br /> are present at a high level relative to what is seen for the fully <br /> spliced transcript, further investigation revealed that in &#226;&#172;&#381;99% <br /> of the transcripts, introns 1 to 3 have been removed (data not <br /> shown). <br />    hsf1 is an essential gene; hsf2 is required for asexual devel- <br /> opment. To knock out hsf1, at least 1.5 kb of DNA 5&#226;&#172;&#732; and 3&#226;&#172;&#732; of <br /> the ORF was ligated to the hygromycin phosphotransferase gene <br /> (Fig. 2A). This construct was electroporated into conidia of a <br /> mus-51::bar strain which lacks an enzyme required for nonho- <br /> mologous DNA joining. Due to the absence of mus-51, tar- <br /> geted transformation is almost 100% efficient (33). DNA from                      FIG. 1. RNA splice forms and protein domain structures of HSF1 <br /> 20 transformants was extracted and recombination events at                     and HSF2 isoforms. For each pair of diagrams the top schematic <br />                                                                                represents RNA and the bottom schematic represents protein. Open <br /> the hsf1 gene were checked by Southern blot analysis (Fig. 2B).                rectangles in mRNAs represent exons. Nucleotide numbers below <br /> All transformants were heterokaryons harboring both nuclei                     transcripts start from the ATGs of the respective ORFs. Amino acid <br /> with hsf1 intact and nuclei with hsf1 deleted. To obtain ho-                   numbers are indicated below proteins. DBD, DNA-binding domain. <br />  1576      THOMPSON ET AL.                                                                                                    EUKARYOT. CELL <br />  <br />  <br />                                                                           used in these experiments. The bd mutation allows conidiation <br />                                                                           to occur in the presence of the high carbon dioxide levels which <br />                                                                           accumulate in the enclosed race tube. In response to a 1-h, <br />                                                                           50&#194;&#176;C heat shock during the third day in DD at 25&#194;&#176;C, growth of <br />                                                                           both bd and hsf2,bd strains stopped. On the second day after <br />                                                                           the heat shock 100% of the bd cultures had resumed growth, <br />                                                                           while only 4 days after the heat shock did 50% of the hsf2,bd <br />                                                                           cultures resume growth (Fig. 4A). We wondered if the delayed <br />                                                                           recovery was correlated with the aconidial state of the hsf2,bd <br />                                                                           strain, and we therefore repeated the experiment, this time <br />                                                                           including the acon-2,bd and fl,bd strains. Of note is that the <br />    FIG. 2. Analysis of mus-51::bar, &#226;&#8364;&#353;hsf1 transformants. (A) Sche-        acon-2,bd strain is a temperature-sensitive strain and no spores <br /> matic diagram showing the location of SacI restriction enzyme sites in    develop at 34&#194;&#176;C or above. Therefore, all of the strains were <br /> the region of hsf1. (Left) Wild-type hsf1 locus; (right) locus with the <br />                                                                           cultured at either 25&#194;&#176;C or 34&#194;&#176;C before and after a 50&#194;&#176;C heat <br /> hsf1 gene deleted and replaced with hph. (B) Southern blot of <br /> mus-51::bar, &#226;&#8364;&#353;hsf1 strains. Genomic DNA digested with SacI and            shock. In both sets of conditions, the aconidial strains took <br /> probed with DIG-labeled DNA homologous to a region 5&#226;&#172;&#732; of the hsf1         longer to recover from heat shock than did the bd strain (Fig. <br /> ORF. Lanes: 1, 7, and 12, DNA ladder; 8, control parental mus-51          4B and C). These data suggest either that the germination of <br /> genomic DNA; 2 to 6, 9 to 11, 13, and 14, DNA from different <br /> transformants. <br />                                                                           conidia that fall to the agar surface after heat shock accounts <br />                                                                           for the faster regrowth of the bd strain and/or that all the <br />                                                                           aconidial strains are inherently temperature sensitive. <br /> were resistant to hygromycin, supporting our conclusion that                 If the latter of these possibilities is true, then these results <br /> hsf1 is an essential gene.                                                implicate HSF2 in the heat shock response. Therefore, we <br />    A hsf2 deletion strain (8) was obtained from the Fungal                followed levels of both hsf1 and hsf2 and of three HSP tran- <br /> Genetics Stock Center (University of Missouri, Kansas City)               scripts in control and heat-shocked cultures. In a bd strain, the <br /> and checked by Southern analysis (data not shown). The veg-               levels of hsfI and hsf2 mRNA declined during the first 45 min <br /> etative hyphal growth of the hsf2 mutant appears normal, as               of a prolonged heat shock. In the hsf2,bd strain, hsf1 transcrip- <br /> does the growth rate, but the deletion strain has a striking              tion also decreased and, as expected, hsf2 RNA was not ex- <br /> <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>; it produces very few macroconidia (Fig. 3). Only               pressed (Fig. 5A). In spite of this, the three HSP transcripts we <br /> after several weeks are a few proconidial chains present, per-            assayed, namely, those for hsp70 (NCU09602) and hsp90 <br /> haps induced by starvation or desiccation.                                (NCU04142) and a transcript that arises from NCU01792 <br />    The heat shock response in the hsf2,bd strain. We first as-            which we have named heat shock protein 90-associated <br /> sayed the phenotypic response of the hsf2 mutant to heat shock            (hsp90a), were upregulated in response to heat shock in both <br /> by monitoring its growth along race tubes after exposure to               strains (Fig. 5B). Interestingly, two hsp70 transcripts can be <br /> high temperature. In order to assess the effect of heat shock on          distinguished after transfer to high temperatures in both the <br /> conidiation as well as on growth a hsf2,bd strain was initially           wild-type and hsf2,bd strains. The lower-molecular-weight <br />  <br />  <br />  <br />  <br />  FIG. 3. The hsf2KO strain is aconidial. Light microscopy showing growth of bd (left) and hsf2KO,bd (right) strains on minimal agar plates (low <br /> magnification) and at higher magnification (inset; magnification, &#226;&#171;&#187;80) at the growth front. Macroconidia (asexual spores) are bright orange. <br />  VOL. 7, 2008                                                             NEUROSPORA HEAT SHOCK FACTOR-LIKE GENES                            1577 <br />  <br />  <br />  <br />  <br />    FIG. 4. Response of wild-type and aconidial strains to heat shock. <br /> Strains inoculated onto race tubes were germinated and grown in LL <br /> at 25&#194;&#176;C (A and B) or 34&#194;&#176;C (C) for 48 h before transfer to DD at 25&#194;&#176;C <br /> (A and B) or 34&#194;&#176;C (C). After 24 h in DD, cultures were exposed to a <br /> 1-h 50&#194;&#176;C heat shock and then transferred back to DD at 25&#194;&#176;C (A and <br /> B) or 34&#194;&#176;C (C). Growth was monitored twice daily over the following <br /> 9 days. The graph shows the percentages of recovery for the following <br /> strains against time (h) after heat shock. (A) 30-7 bd (n &#226;&#171;&#189; 21), 16-31 <br /> bd (n &#226;&#171;&#189; 18), 16-21 hsf2KO,bd (n &#226;&#171;&#189; 17), and 16-6 hsf2KO,bd (n &#226;&#171;&#189; 18). <br /> Numbers preceding the strain names refer to the cross and progeny <br /> identifiers, e.g., 16-31 indicates cross number 16 and progeny number <br /> 31. (B) 16-21 hsf2KO,bd (n &#226;&#171;&#189; 17), 16-6hsf2KO,bd (n &#226;&#171;&#189; 18), acon-2,bd <br /> (n &#226;&#171;&#189; 17), and fl,bd (n &#226;&#171;&#189; 21). (C) 16-31 bd (n &#226;&#171;&#189; 17), 16-21hsf2KO,bd <br /> (n &#226;&#171;&#189; 18), fl,bd (n &#226;&#171;&#189; 15), and acon-2,bd (n &#226;&#171;&#189; 18). The average results <br /> of three independent experiments are plotted. HS, heat shock. <br />  <br />  <br />  <br />  <br /> transcript is most likely another member of the hsp70 family, <br /> ssb1 (NCU02075), which shares 87% identity over 239 nucle- <br /> otides to our 901-nucleotide hsp70 probe. Of note is that hsp90 <br /> transcripts were significantly more abundant 15 min after the <br /> onset of heat shock in the hsf2,bd strain than in the bd strain. <br /> One other significant difference was detected after 45 min of <br /> exposure to heat shock between levels of hsp90a RNA in the bd <br /> and hsf2,bd strains, with the levels being higher in the bd strain. <br />    RNA levels of genes associated with conidiation are affected <br /> in the hsf2 strain. To test where HSF2 acts in the known <br /> conidiation pathway, we investigated the expression of hsf1, <br /> hsf2 (Fig. 6), fl, conidiation-10 (con-10), repressor of conidia-           FIG. 5. Gene expression in wild-type and hsf2,bd knockout strains. <br /> tion-1 (rco-1), and eas (Fig. 7) in the wild-type, fl, acon-2, and       Northern blots of bd and hsf2KO,bd RNA probed for hsf1 and hsf2 <br /> hsf2 strains. We assayed gene expression upon the transfer of            RNA (A) and hsp70, hsp90, and hsp90a RNA (B). Cultures transferred <br /> cultures to medium lacking nitrogen, conditions known to in-             from LL at 25&#194;&#176;C to LL at 45&#194;&#176;C were harvested for RNA extraction <br />                                                                          after 0, 15, 30, 45, and 60 min. There are two sets of data for the hsf1 <br /> duce the conidiation pathway (1). Although there was a gen-              transcript and three for all other transcripts. The y axes represent the <br /> eral increase in hsf1 mRNA as development progressed, no                 relative amounts of each transcript (percentages of the maximum). HS, <br /> significant difference in the levels of either hsf1 or hsf2 RNA          heat shock. <br /> was detected among the four strains. As previously reported, <br /> levels of fl and con-10 RNA in the wild type increase from 6 <br /> hours after the transfer of cultures to medium lacking nitrogen          pressor of con-10, were detected between any of the strains at <br /> (1). Among the aconidial strains, fl transcript levels are higher        the time points sampled. In our study, by far the most striking <br /> in the acon-2 and hsf2 strains at 6 h, whereas con-10 transcripts        difference between the aconidial strains and the wild type is the <br /> show significantly increased levels of expression in the hsf2            greatly reduced levels of eas in the fl, acon-2, and hsf2 strains. <br /> strain (similar to con-10 levels in the wild-type) compared to              Conidiation can be induced by a number of environmental <br /> those in the acon-2 and fl strains at 0, 6, and 12 h. To our             cues, for example, carbon and nitrogen deprivation and light <br /> surprise, no differences in the transcript levels of rco-1, a re-        (47). Synchronous conidiation can also be induced by the ex- <br />  1578      THOMPSON ET AL.                                                                                                     EUKARYOT. CELL <br />  <br />  <br />                                                                                                      DISCUSSION <br />  <br />                                                                               The following three genes encoding proteins with domains <br />                                                                            sharing similarity to known HSFs are present in Neurospora (4): <br />                                                                            hsf1 and hsf2, which are discussed in this paper, and a third gene <br />                                                                            not included in our study, NCU02413, which encodes response <br />                                                                            regulator-2 (rrg-2). rrg-2 contains a truncated HSF DNA-binding <br />                                                                            domain and is involved in Neurospora&#226;&#8364;&#8482;s response to oxidative <br />                                                                            stress (2). HSF1, HSF2a, and HSF2b all contain conserved HSF <br />                                                                            binding domains and coiled-coil regions that in other HSFs are <br />                                                                            required for trimerization (43). HSF binding domains recognize <br />                                                                            and bind to heat shock elements (HSE), consensus sequences <br />                                                                            found in the promoters of hsp genes (28). The basic HSE shows <br />                                                                            similarity across a wide range of organisms and is based around <br />                                                                            GAA repeats, the spacing and orientation of which vary and may <br />                                                                            influence which sites are recognized by different HSFs or differ- <br />                                                                            ently phosphorylated HSFs (20). For Neurospora, HSE are not yet <br />                                                                            well defined, but nGAAn .. nTTCn motifs (where &#226;&#8364;&#339;n&#226;&#8364;&#65533; can be any <br />                                                                            nucleotide) are present in the promoters of hsp30 (38) and hsp70 <br />                                                                            (21). Additionally, a factor in Neurospora protein extracts binds <br />                                                                            specifically to labeled oligonucleotides containing the classic yeast <br />                                                                            HSF binding sequence nTTCnnGAAnnTTCn (32). Band shift <br />                                                                            assays with purified Neurospora HSF(s) should aid in the identi- <br />                                                                            fication of the discriminating consensus binding sites for HSF1 <br />   FIG. 6. Semiquantitative RT-PCR of hsf and hsf2 RNA in wild-             and HSF2. <br /> type, flC-1835, acon-2, and hsf2KO mutants. Cultures were harvested <br /> before and at various times after transfer to medium lacking nitrogen, <br />                                                                               Our finding that various splice forms of hsf2 exist is consis- <br /> conditions that initiate conidiation in the wild-type strain. (A) Repre-   tent with recent publications from other laboratories. For ex- <br /> sentative gels showing RT-PCR products of each transcript. (B and C)       ample, alternative splicing of hsf RNA is developmental stage <br /> Relative amounts of each transcript graphed after normalization            specific in Schistosoma mansoni (22), and in Drosophila differ- <br /> against actin. % max, percentages of the maximum.                          ent splice forms are made depending on the stress to which the <br />                                                                            organism is exposed (16). Moreover, in both Drosophila and <br />                                                                            humans, alternative splicing of hsf RNA produces either an <br />                                                                            activator or a repressor of transcription (45). The repressors <br /> posure of mycelia to air (44). Interestingly, although develop-            retain the DNA-binding and trimerization domains but lack <br /> mentally induced genes are expressed at similar times in cul-              the C-terminal activation domain, similar to Neurospora <br /> tures induced to conidiate by either nitrogen starvation or                HSF2b and HSF2c. Nonetheless, further work is required to <br /> exposure to air, evidence suggests that their sensitivity to other         elucidate the conditions under which the three splice variants <br /> stresses, such as heat shock, may change. For example, it is               of Neurospora hsf2 are expressed and their purpose. Intrigu- <br /> reported that in air-exposed cultures, con-10 is induced by heat           ingly, a possible function for HSF2b and HSF2c arises from a <br /> shock, whereas in liquid cultures it is not (26). However, in our          study reporting that in response to heat shock, proteins bind to <br /> hands, no significant difference in the developmental induction            putative HSE and/or stress elements in the glycogen synthase <br /> of con-10 expression between the wild-type and hsf2 strains                gene of Neurospora and repress transcription (15). Although <br /> during air-induced development was revealed, and in neither                we found only low levels of alternatively spliced hsf transcripts <br /> strain was con-10 induced by heat shock (Fig. 8). con-10 has               and observed that these did not correlate with reduced levels <br /> been shown to be regulated by numerous factors e.g., carbon                of glycogen synthase mRNA (data not shown), this does not <br /> and nitrogen deprivation and light, and it is probable that slight         rule out the possibility that forms of HSF2 protein change with <br /> differences in experimental conditions account for the discrep-            time and in response to heat shock and/or development. <br /> ancy between our results and those of Lee and Ebbole (26).                    Whereas hsf2KO strains are viable but have an aconidial <br />    In an effort to determine whether development in the hsf2               phenotype, we report that hsf1 is an essential gene. No ho- <br /> strain is indirectly affected through the misregulation of HSPs, we        mokaryon hsf1KO transformants were obtained, and no viable <br /> also assayed expression levels of the hsp90 transcript. Again, no          hsf1KO progeny were obtained from crosses of hsf1KO and <br /> significant differences in expression were detected (Fig. 7B). Fur-        hsf1&#226;&#171;&#185; heterokaryons with the wild type. The single S. cerevisiae <br /> thermore, strains with hsp90a (NCU01792), hsp30 (NCU09364),                and Drosophila HSF genes are also essential. In yeast, evidence <br /> hsp78 (NCU02630), and hsp101 (NCU00104) deleted showed no                  suggests that HSF is required for the expression of both the <br /> overt defect in conidiation (data not shown). While we can con-            constitutive and heat-inducible forms of Hsp90 (Hsc82 and <br /> clude that these HSPs are not essential for normal asexual devel-          Hsp82), and in the S. cerevisiae hsf-82 mutant, in which Hsp82 <br /> opment, hsp70 (NCU08683) and hsp90 (NCU04142) are essential                is no longer induced upon heat shock, spindle pole body for- <br /> genes, and thus, as for hsf1, their role in development could not be       mation is affected and completion of the cell cycle at 37&#194;&#176;C is <br /> assessed by simple phenotypic observation of the respective gene           blocked (51). Moreover, indirect effects of Hsp90 misexpres- <br /> knockout strains.                                                          sion on cell wall integrity, for example, occurs via a reduction <br />  VOL. 7, 2008                                                                  NEUROSPORA HEAT SHOCK FACTOR-LIKE GENES                            1579 <br />  <br />  <br />  <br />  <br />   FIG. 7. Comparison of fl, con-10, rco-1, and eas expression in wild-type and conidiation-defective strains of Neurospora. Semiquantitative <br /> RT-PCR of fl, con-10, rco-1, and eas RNA in the wild type and the flC-1835, acon-2, and hsf2KO mutants. Cultures were harvested before and at <br /> various times after transfer to medium lacking nitrogen, conditions that initiate conidiation in the wild-type strain. (A) Representative gels showing <br /> RT-PCR products of each transcript. (B, C, and D) Relative amounts of each transcript graphed after normalization against actin. % max, <br /> percentages of the maximum; wt, wild type. <br />  <br />  <br />  <br /> in Slt2 kinase activity and a consequent lack of transcriptional             (reviewed in reference 34) and the inhibition or overexpression <br /> activation of a subset of cell wall genes (46). The essential                of these chaperones is associated with defects in development <br /> nature of Neurospora hsf1 would be easily explained if future                in Drosophila and in yeast (40, 12, 51, 46). The putative protein <br /> experiments to elucidate its targets reveal a subset of genes                encoded by NCU01792 contains a CS domain, a domain that <br /> that encode constitutive as well as heat shock-induced protein               binds to Hsp90 and which is found in p23, a Hsp90 cochaper- <br /> chaperones.                                                                  one (27). In our experiments, there were no significant differ- <br />    Our results suggest that HSF2 may have a role in regulating               ences between hsp70 transcript levels in the bd and hsf2,bd <br /> the level of HSP transcripts. We looked at the levels of hsp70,              strains. However, immediately following heat shock, hsp90a <br /> hsp90, and hsp90a specifically, because in other organisms the               RNA is significantly lower in the hsf2,bd strain compared to <br /> interaction of chaperones hsp70 and hsp90 with hormone re-                   levels in the wild type, and hsp90 transcript levels are higher <br /> ceptors, kinases, and other signaling molecules is well known                than those seen for the wild type 45 min after exposure to heat <br />  1580     THOMPSON ET AL.                                                                                                                EUKARYOT. CELL <br />  <br />  <br />                                                                          of the hsf2KO strain is that the induction of HSPs that interact <br />                                                                          with components of the conidiation pathway are no longer <br />                                                                          sufficiently abundant for conidiation to proceed. Alternatively, <br />                                                                          HSF2 or a HSF2-induced protein is an integral component of <br />                                                                          the developmental pathway. We were unable in this study to <br />                                                                          distinguish conclusively between these possibilities. On the one <br />                                                                          hand, asexual development is normal in strains with different <br />                                                                          hsp genes deleted, and no significant differences in hsp90 ex- <br />                                                                          pression over the course of development were detected. More- <br />                                                                          over, levels of many of the transcripts we assayed show wild- <br />                                                                          type expression patterns in the hsf2 strain, which would be not <br />    FIG. 8. con-10 and hsp90 expression during development in air-        be expected if protein folding is affected. On the other hand, <br /> exposed cultures. (A) Northern blots of wild-type and hsf2 total RNA     some hsp genes are essential for survival, and their effect on <br /> probed for con-10. Mycelial mats were taken out of liquid cultures and <br /> exposed to air at time zero. Control mats (no heat shock [&#226;&#171;&#186;HS]) were     asexual development could not be easily tested. However, if <br /> harvested at 2-h intervals. Heat-shocked samples (&#226;&#171;&#185;HS) were exposed      HSF2 or a HSF2-induced protein is indeed an integral part of <br /> to 50&#194;&#176;C for 1 hour before being harvested. (B) (Left) Representative     the conidiation program, the gene expression data we have <br /> gel showing semiquantitative RT-PCR products of hsp90 and actin          place HSF2 in a pathway parallel to that of fluffy. In support of <br /> transcripts; (right) quantitation of hsp90 RNA from the same samples <br />                                                                          this conclusion, wild-type fl and con-10 (repressed in a fluffy <br /> (n &#226;&#171;&#189; 3). rel., relative; wt, wild type. <br />                                                                          mutant) transcript levels are seen throughout development in <br />                                                                          the hsf2 strain. Hence, while acon-2 and fluffy can regulate the <br />                                                                          level of both con-10 and eas, the hsf2 strain must regulate levels <br /> shock. Both of these departures from wild-type levels of gene            of eas and other as yet unidentified genes essential for normal <br /> expression could be directly due to the lack of HSF2.                    development via an independent pathway. <br />    Deleting hsf2 had a dramatic effect on asexual development; <br /> aerial hyphae developed but no conidial chains formed. The                                          <br /> following six mutants blocked at different stages of conidial            <br /> development have been placed in a conidiation pathway: the              <br /> fld (linkage group IV [LGIV]), fl (LGII), acon-2 (LGIII), and         <br /> acon-3 (LGIV) mutants and two conidial separation mutants,                 <br /> the csp-1 and csp-2 mutants. In addition, several recently gen-    <br /> erated knockout strains have been recorded as aconidial, in-                                               <br /> cluding strains with mutations in vegetative growth and asexual          <br /> development (the vad-2 [LGI] and vad-4 [LGIII] strains). hsf1              <br /> and hsf2 are both located on LGII, separated by approximately            <br /> 100 kbp, and do not map to any of the previously described                 <br /> aconidial mutants. Bailey-Shrode and Ebbole place acon-2 up-              <br /> stream of fl and acon-3 in the pathway to conidiation (1). In          <br /> this scheme, fl regulates acon-3 and eas, a gene encoding a                <br /> hydrophobin found on the surfaces of macroconidia (3, 24),              <br /> while acon-3 regulates con-6 and con-10. The rco-1 gene prod-            <br /> uct represses the expression of eas, con-6, and con-10 (50, 26,       <br /> 1). We were interested to find out if HSF2 is an integral               <br /> element of the proposed pathway or whether it acts as a facil-       <br /> itator ensuring that the necessary chaperones are present to                 <br /> ensure the correct conformation of pathway components.                <br />    After initiation of development induced by the transfer of             <br /> cultures to medium lacking nitrogen we found, in agreement          <br /> with previous reports, that fl transcript levels in the wild type             <br /> increase after 6 hours (1). fl is a transcriptional activator and a          <br /> major regulator of conidiation in Neurospora (1, 39), whose                 <br /> expression has previously been reported to be low in the acon-2            <br /> strain (1). However, we found that fl was increased in all three            <br /> aconidial strains, data which suggest that ACON2 regulates                 <br /> some posttranscriptional aspect of fl gene expression. In keep-              <br /> ing with the lack of spore formation in the acon-2 and fluffy             <br /> mutants, levels of con-10 are low in these strains. With the                <br /> exception of eas, and the higher levels of fl at 6 h, all of the           <br /> transcripts assayed throughout development showed similar                 <br /> levels of expression in the hsf2 and wild-type strains.                      <br />    One scenario that could account for the aconidial phenotype              <br /> </body></html>