<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.083782 <br />  <br />  <br />  <br />               Identification Drosophila Mutants Altering Defense and <br />                        Endurance Listeria monocytogenes Infection <br />  <br />                              Janelle S. Ayres,* Nancy Freitag&#226;&#8364;&#160; David S. Schneider*,1 <br />                       *Department Microbiology Immunology, Stanford University, Stanford, California 94305 and <br />                          &#226;&#8364;&#160; <br />                           Department Microbiology Immunology, University Illinois, Chicago, Illinois 60612 <br />                                                     Manuscript received October 24, 2007 <br />                                                  Accepted publication December 19, 2007 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 We extended use Drosophila model signaling pathways required <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />              recognition immune signaling fly used identify genes required patho- <br />              genesis host&#226;&#8364;&#8220;pathogen interactions. We performed forward genetic screen identify Drosophila <br />              mutations altering sensitivity intracellular pathogen Listeria monocytogenes. We recovered 18 mutants <br />              increased susceptibility infection, previously shown function Drosophila <br />              immune response. Using secondary screens, divided mutants groups: In group, <br />              mutants reduced endurance infections change bacterial growth. This new fly <br />              immunity <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> commonly studied. In second group, mutants typical defense <br />              defect bacterial growth increased <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>survival</span> decreased. By challenging mutant flies <br />              L. monocytogenes mutants, identified subgroups fly mutants affect specific stages L. <br />              monocytogenes life cycle, exit vacuole, actin-based movement. There overlap our <br />              genes hundreds genes identified Drosophila S2 cells fighting L. monocytogenes infection, <br />              using genomewide RNAi screens vitro. By using whole-animal model screening host survival, <br />              revealed genes involved physiologies different previous screens, which <br />              defects defensive immune signaling. <br />  <br />  <br />  <br />  <br /> I  NTRACELLULAR pathogens responsible a <br />     large group infectious diseases; example, .500 <br /> million people worldwide suffer tuberculosis, <br />                                                                           (Cossart et al. 2003). Entry nonphagocytic cells <br />                                                                           dependent surface proteins called internalins <br />                                                                           (Cossart Lecuit 1998; Lecuit et al. 1999, 2001). <br /> AIDS, malaria year (http://www.who.int). By                      Upon entry, single-membrane vacuole forms around <br /> residing host cell, pathogens protect them-                    bacterium (Cossart Lecuit 1998). Secretion <br /> selves host immune responses drug                           pore-forming cytotoxin, listeriolysin O (LLO), dis- <br /> therapies. The ability enter survive host                 rupts phagosome membrane, freeing bacterium <br /> cell requires close intricate interaction                   cytosol grow divide (Kuhn et al. <br /> pathogen host; manipulating host processes,                        1988; Portnoy et al. 1988; Gedde et al. 2000; O&#226;&#8364;&#8482;Riordan <br /> pathogens prevent immune responses subvert                         Portnoy 2002). Expression actin-nucleating <br /> host processes aid infection.                                       protein ActA facilitates polymerization host actin at <br />    Listeria monocytogenes gram-positive, facultative in-             pole bacterium, resulting directional move- <br /> tracellular bacterium cause listeriosis,                  ment cytosol. Bacteria able spread <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> serious food-borne disease. L. monocytogenes                     neighboring cells escape second double- <br /> widely used model pathogen better understand                      membrane phagosome cytosol new host <br /> the molecular cellular aspects intracellular path-                 cell (Tilney Portnoy 1989; Tilney et al. 1990; <br /> ogenesis mammalian cell-mediated immunity. In                         Domann et al. 1992; Kocks et al. 1992, 1993). <br /> vitro studies using cultured cells model host defined                   The development <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) technol- <br /> the L. monocytogenes intracellular life cycle virulence               ogy provided researchers new approach studying <br /> factors (Cossart Lecuit 1998; Portnoy et al. 1988;                    host factors involved host&#226;&#8364;&#8220;pathogen interactions the <br /> Kreft Vazquez-Boland 2001; Vazquez-Boland                             fly (Ramet et al. 2002; Foley O&#226;&#8364;&#8482;Farrell 2004; Ayres <br /> et al. 2001) demonstrated L. monocytogenes                   Schneider 2006). Two recent reports use <br /> enter professional phagocytes nonphagocytic cells                      RNAi Drosophila S2 cells (embryo-derived phagocytic <br />                                                                           cells Drosophila) identify host factors involved in <br />   1 <br />                                                                           L. monocytogenes infection; screens helped clarify the <br />    Corresponding author: Department Microbiology Immunology, <br /> Stanford University, D333 Fairchild Bldg., 299 Campus Dr., Stanford, CA   cell biology interactions L. monocytogenes with <br /> 94305. E-mail: dschneider@stanford.edu                                    professional phagocytes (Agaisse et al. 2005; Cheng et al. <br />  <br /> Genetics 178: 1807&#226;&#8364;&#8220;1815 (March 2008) <br />  1808                                        J. S. Ayres, N. Freitag D. S. Schneider <br />  <br /> 2005). The assays used vitro screens measured           Imd pathways (Meng et al. 1999; Leulier et al. 2000; <br /> bacterial loads qualitative changes bacterial patterns.       Rutschmann et al. 2000; Wu et al. 2001; Khush et al. <br /> This limited scope screens identification         2002; Gesellchen et al. 2005; Yagi Ip 2005; Kambris <br /> of fly genes regulating bacterial life cycle. Surprisingly,     et al. 2006). In screen monitored fly survival well <br /> these screens did identify genes affecting well-            bacterial growth mutant flies. By focusing phe- <br /> characterized pattern recognition pathways, Toll Imd.           notypes AMP transcription, anticipated <br /> Moreover, vitro systems obviously limited           identify host genes affecting wider array <br /> their ability explore whole-animal physiologies              immunological pathological functions. <br /> interact infecting microbe L. monocyto-                We identified 18 mutants increased sensitivity to <br /> genes enter survive wide variety cell types.        L. monocytogenes infection. None genes were <br />    To overcome limitations, performed for-               described previously immune pathogenesis <br /> ward genetic screen Drosophila identify host            functions. Secondary screens grouped mutants <br /> genes required survive infections. We goals.           phenotypic classes. The flies split classes <br /> First, wanted cast wider net pre-             basis ability control growth L. <br /> viously used measure immunity Drosophila. Much                monocytogenes. We define group having defects in <br /> of know fly immunity deciphered              endurance suspect difficulties control- <br /> using extracellular microbes immunocompromised                   ling pathogenesis mutant flies die faster than <br /> flies. By using L. monocytogenes study able          wild-type flies bacterial load same <br /> to identify host genes important survival intra-          wild-type flies. A second class appears lack de- <br /> cellular infection. These genes specific L.             fense functions. We define group immunocom- <br /> monocytogenes infection, intracellular pathogens              promised shows increased fly death is <br /> general, variety pathogens. By coupling survival         correlated rapid growth L. monocytogenes. The <br /> and bacterial proliferation output phenotypes,            mutants split sensitive wild- <br /> were able identify mutants defense defects          type classes basis response Staphylococ- <br /> as lines pathological defects&#226;&#8364;&#8221;those die              cus aureus Salmonella typhimurium. Secondary screens <br /> the disease faster altered bacterial load. We hy-           used L. monocytogenes mutants allowed de- <br /> pothesize second class mutants able         termine fly mutants acting at <br /> endure stress infection. Second, wanted             vacuole cytoplasmic stages bacteria life <br /> determine genes required survive infection            cycle. Thus, screening survival endpoint <br /> whole flies identified cultured           instead monitoring transcriptional output Toll <br /> cells RNAi screens. By doing this, screen              imd signaling, identified host genes involved in <br /> reveal differences host genes involved inter-         immunity pathogenesis identi- <br /> actions whole-animal physiology microbe,            fied previously. <br /> as host genes involved interactions <br /> between cell type infecting microbe. Third, <br />                                                                                    MATERIALS AND METHODS <br /> we wanted probe host contributions infec- <br /> tion studies involving host&#226;&#8364;&#8220;pathogen inter-               Flies, bacterial strains, media: The Exelixis Drosophila <br /> actions intracellular pathogen focused              melanogaster piggyBac insertion mutant collection used for <br /> identifying microbial factors required infection.               survival screen. To serve wild-type control w1118 <br />                                                                     line used generate isogenic collection used. All <br />    The Drosophila innate immune response                  viable lines available Bloomington Drosophila Stock <br /> arms: First, cellular immune response depends                Center beginning screen obtained (1231 <br /> circulating phagocytic cells engulf clear              lines total). Flies kept standard fly bottles containing <br /> foreign microbes. Second, melanization response                 dextrose medium. L. monocytogenes strains 10403S, DP-L2161 <br /> produces melanin toxic reactive oxygen species               (Dhly), DP-L3078 (DactA) stored 80&#194;&#176; brain&#226;&#8364;&#8220; <br />                                                                     heart infusion (BHI) broth containing 15% glycerol. The L. <br /> wounds sites infection. The branch              monocytogenes mutant strains derived 10403S <br /> humoral immune response, involves produc-                 parent strain generated laboratory Daniel A. <br /> tion antimicrobial peptides (AMP) fat body.               Portnoy (University California, Berkeley, CA). S. typhimu- <br /> These AMPs produced largely control                rium SL1344 stored 80&#194;&#176; LB media containing 15% <br /> the Toll Imd pathways. Both pathways               glycerol. S. aureus clinical isolate Clinical <br />                                                                     Microbiological Laboratory Stanford University was <br /> activated microbial elicitors disruption            stored 80&#194;&#176; BHI media containing 15% glycerol. <br /> pathways immunocompromises fly fly                   Survival screen: For infection adult flies, L. monocytogenes <br /> becomes sensitive normally nonpathogenic bacteria                grown overnight BHI medium 37&#194;&#176; shaking. A <br /> like Escherichia coli (Lemaitre et al. 1995, 1996, 1997;            total 10 adult flies, aged 5&#226;&#8364;&#8220;7 days old, mutant line were <br /> Kopp Medzhitov 1999, 2003; Khush et al. 2001).                  injected 50 nl culture using picospritzer pulled <br />                                                                     glass needle. Flies injected anterior abdomen on <br /> Genetic screens flies primarily concen-               ventrolateral surface  1000 colony-forming units <br /> trated AMP signaling output, strengthening                 (CFUs). Mutant lines tested groups minimum of <br /> our understanding signaling Toll                 20 lines, addition wild-type control. Once infected, flies <br />                                           A Survival-Based Screen Defense Endurance                                       1809 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 1.&#226;&#8364;&#8221;L. monocytogenes monocytogenes infec- <br />                                                                                   tion Drosophila melanogaster. Wild-type flies were <br />                                                                                   injected 103 CFUs wild-type mu- <br />                                                                                   tant L. monocytogenes. Survival growth (growth <br />                                                                                   wild-type bacteria only) monitored over <br />                                                                                   course infection. (A) L. monocytogenes <br />                                                                                   strain 10403S (wild type) infection. Wild-type flies <br />                                                                                   exhibit median time death (MTD)  5 days <br />                                                                                   postinfection. (B) Growth 10403S flies. Bac- <br />                                                                                   teria reach levels .104 CFUs 48 hr postinfec- <br />                                                                                   tion. **P , 0.005 (t-test). (C) Dhly infection in <br />                                                                                   flies. Wild-type flies exhibit MTD  20 days <br />                                                                                   postinfection. (D) DactA infection flies. In- <br />                                                                                   fected flies exhibit MTD  10 days postinfec- <br />                                                                                   tion. Statistical analysis survival curves was <br />                                                                                   using log-rank analysis lines P , <br />                                                                                   0.05 considered statistically significant. <br />  <br />  <br />  <br />  <br /> were transferred vials containing dextrose medium              overnight shaking. Thirty-five 7-day-old male flies per <br /> incubated 29&#194;&#176; number dead flies line       mutant line infected independent replicates for <br /> counted 24 hr postinfection flies dead. The      mutant fly line flies injected  1000 CFUs. <br /> median time death (MTD), time postinfection               The death rate line directly compared wild- <br /> 50% infected flies succumbed infection,       type control analyzed using Graphpad Prism software as <br /> line determined exhibiting extreme             described above. <br /> phenotypes flagged testing. Mutant lines           Verification PiggyBac insertion site: Inverse PCR was <br /> failed exhibit phenotype initial test fur-       determine insertion site PiggyBac (PBac) <br /> ther tested. Candidate lines retested indepen-          element following protocol provided Exelixis. Briefly, <br /> dent experiments eliminate false positives. Thirty-five        Genomic DNA mutant fly line isolated using the <br /> 7-day-old male flies line infected         QIAGEN (Valencia, CA) DNeasy kit. 59- 39-end digestions <br /> death rate mutant flies line compared             using Sau3AI HinPI, respectively, in- <br /> directly death rate wild-type control flies, using          cubated 37&#194;&#176; 3 hr. Ligations using T4 DNA <br /> Graphpad Prism software. Using log-rank analysis, P-value         ligase 4&#194;&#176; overnight. PCR reactions performed using the <br /> was determined. Lines exhibited death rate              primer sequences reaction conditions provided Ex- <br /> P-value ,0.05 retests considered positive          elixis (http://flystocks.bio.indiana.edu). <br /> mutants. <br />    Secondary screens: S. aureus S. typhimurium secondary <br /> screens: For infection adult flies S. aureus, bacteria                 RESULTS AND DISCUSSION <br /> grown overnight BHI medium 37&#194;&#176; shaking. Ap- <br /> proximately 100 CFUs injected fly. For in-                Screening host genes important survival: To <br /> fection S. typhimurium, bacteria grown LB medium         search host genes important surviving an <br /> at 37&#194;&#176;  10,000 CFUs injected fly. For              infection, conducted forward genetic screen a <br /> all infections, 35 7-day-old male flies mutant line were <br /> infected mentioned death rate line          mutant population Drosophila monitored death <br /> was directly compared wild-type flies analyzed      rates following challenge L. monocytogenes. L. <br /> previously mentioned.                                                 monocytogenes establishes lethal infection injec- <br />    Quantification CFUs flies: Infected flies homog-        tion Drosophila (Figure 1A) (Mansfield et al. <br /> enized BHI media supplemented 1% Triton X-100.                2003). Wild-type flies injected 103 CFUs exhibited a <br /> Homogenates serially diluted plated BHI agar <br /> and incubated overnight 37&#194;&#176;. The number CFUs                MTD 4&#226;&#8364;&#8220;5 days postinfection. The predictable death <br /> mutant line compared wild-type flies using             kinetics allowed use survival output pheno- <br /> Graphpad Prism software independent experiments.            type screen. We hypothesized screening flies <br /> Using unpaired t-test, P-value determined. Mutant          ability survive L. monocytogenes infections <br /> lines exhibited P-value ,0.05 retests       allow determine immune mechanisms used <br /> considered significantly different bacterial growth <br /> compared wild-type control.                                    fight microbe, host mechanisms exploited the <br />    L. monocytogenes mutants: For infection adult flies        pathogen, physiologies drive pathogenesis in <br /> both DP-L2161 DP-L3078, bacteria grown 37&#194;&#176;                fly. <br />  1810                                       J. S. Ayres, N. Freitag D. S. Schneider <br />  <br />                           TABLE 1                                                              TABLE 2 <br />                Distribution mutant fly lines                        Screening genes affecting defense endurance <br />  <br /> Mean time                                      No. PBac                      No.      No.        No. of <br /> death (day)                                          lines         Retest           lines      positive     negative      % positive <br />                                                                    no.             tested       lines         lines         lines <br /> 1                                                     85 <br /> 2                                                    145           Initial test     1231         101           &#226;&#8364;&#8221;              8.2 <br /> 3                                                    249           1                 101          35           66            34.6 <br /> 4                                                    510           2                  35          25           10            71.4 <br /> 5                                                    168           3                  25          18            7            72 <br /> 6                                                     58 <br /> 7                                                     16              Over 1200 PBac insertion mutant lines tested their <br />                                                                    ability survive L. monocytogenes infection MTD for <br />    After initial test, MTD mutant Drosophila       line determined. The mutant lines exhibited <br /> line determined. The majority mutant lines exhibit        extreme phenotypes MTDs day 1 day <br /> MTD 4 days postinfection extreme pheno-            7 ( 100 mutants) considered positive hits tested fur- <br /> types observed 1 7 days postinfection. Mutants ex-        ther eliminate false positives. The candidate mutants <br /> hibiting extreme phenotypes flagged              tested independent experiments, death <br /> further testing eliminate false positives.                      curves compared wild-type control, <br />                                                                    log-rank statistical analysis done. Lines exhibited <br />                                                                    death rate P-value ,0.05 retests con- <br />    For initial screen tested 1231 viable mutants            sidered positive mutants. After rounds retesting <br />                                                                    .80% mutants considered false positives and <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> publicly available collection Exelixis PBac            total 18 mutants, increased sensitivity infection, <br /> homozygous transposon insertion lines. We tested              identified positive mutants affect host susceptibil- <br /> homozygous viable lines number represents                 ity L. monocytogenes infection. <br />  8% coverage Drosophila genome. Age-matched <br /> male flies mutant line injected 103            reduce possibility background effects host <br /> CFUs L. monocytogenes survival monitored.               survival. Using computed gene (CG) numbers for <br /> Approximately half mutants tested exhibited               genes determined gene ontology (GO) terms <br /> MTD 4 days postinfection (Table 1). The mutant lines            FlyBase (http://flybase.bio.indiana.edu), which <br /> that exhibited extreme phenotypes MTDs               listed Table 3. <br /> of day 1 day 7 ( 100 mutants) tested              Ubiquitination components: From screen we <br /> eliminate false positives. The candidate mutants               identified mutant lines candidate genes <br /> were tested independent experiments               categorized genes involved ubiquitina- <br /> death curves compared wild-type               tion process. These crossbronx, CG15120, CG2247. <br /> control statistical analysis done. After             This surprising previous studies demonstrated <br /> rounds retesting .80% mutants                     importance ubiquitin-mediated protein degrada- <br /> considered false positives total 18 mutants,          tion regulation Toll Imd pathways. <br /> with increased sensitivity infection, identified        Activation Toll pathway requires ubiquitination <br /> positive mutants affect host susceptibility L.             degradation cactus, inhibitor NF-kb-like <br /> monocytogenes infection (Table 2). For study,               transcription factor DIF. Lys-63 polyubiquitination of <br /> define fly exhibits reproducible, significantly         TRAF6 necessary signaling Imd path- <br /> different faster death rate compared wild-type flies            way (Zhou et al. 2005). Additionally, ubiquitination is <br /> as sensitive fly slower death rate com-             important controlling cytotoxicity LLO after <br /> pared wild type resistant. We injected positive          release L. monocytogenes vacuole during <br /> mutants media confirm increased             infection. <br /> mortality infection dependent (data shown).                   cbx characterized ubiquitin-conjugating E2 <br />    Identification genes: A benefit using Exelixis        enzyme. It studied processes including axon <br /> PBac collection insertion sites trans-          pruning, sperm development, sperm individualiza- <br /> posons mutants collection publicly              tion. In context fly immunity, cbx iden- <br /> available. The Bloomington Drosophila Stock Center                 tified genetic screen possibly role in <br /> (http://flystocks.bio.indiana.edu) verified re-            crystal cell development, type immune cell fly <br /> ported insertion sites. For additional verification,            (Milchanowski et al. 2004). The identification this <br /> identified PBac-element insertion site          gene reveal role crystal cells host de- <br /> positive mutants using iPCR BLAST sequence                  fense bacterial infection. <br /> analysis flanking DNA sequences. Our gene iden-                CG15120 homolog Parkin coregulated gene <br /> tities agree reported Exelixis (Table 3). We         protein, PACRG, involved ubiquitination; <br /> also outcrossed 18 mutants parental strain              moreover, polymorphisms PACRG related <br /> and challenged F4 generation L. monocytogenes             gene, PARK2, linked increased susceptibility to <br />                                           A Survival-Based Screen Defense Endurance                                     1811 <br />  <br />                                                              TABLE 3 <br />                                                Summary positive Drosophila lines <br />  <br /> PBac line          CG no.          Symbol        PBac orientation relative gene         Gene ontology/molecular function <br /> 10036        CG11489a                           Within fifth intron CG11489          ATP binding, receptor signaling, <br />                                                                                           protein serine/threonine kinase activity <br /> 10067        CG10536a              cbx          Within intron cbx              Ubiquitin-conjugating enzyme activity <br /> 10269        CG13564a                           Within 59-UTR CG13564 <br /> 10496        CG14899a                           Within 59-UTR CG14899 <br /> 10618        CG11293a                           400 bp CG11293 <br /> 10675        CG15120/CG16926                    Within 39 end CG15120 <br />                                                   CG16926 <br /> 10743        CG13788a              gr28b        Within intron Gr28b            Taste receptor activity <br /> 10881        CG32706a                           Within 100 bp 39 end CG32706      mRNA binding <br /> 17872        CG4857a,b                          Within intron CG4857 <br /> 18050        CG2247a                            Within intron CG2247 <br /> 18270        CG3527b                            Within 59-UTR CG3527                 rRNA processing, ribosomal small <br />                                                                                           subunit processing <br /> 18398        CG2152a,b             pcmt         Within 200 bp 59 end pcmt         Protein-l-isoaspartate (d-aspartate) <br />                                                                                           O-methyltransferase activity <br /> 18609        CG12487a              bobA         Within coding region BobA            Notch signaling pathway, <br />                                                                                           cell fate specification <br /> 18678        CG2914a,b             ets21C       Within intron Ets21C           DNA binding, transcription factor activity <br /> 19181        CG6128b                            Within coding region CG6128          a-l-fucosidase activity, fucose metabolism <br /> 19207        CG2397b               cyp6a13      Within coding region Cyp6a13         Electron transporter activity, <br />                                                                                           oxidoreductase activity <br /> 19220        CG3056b                            Within 39 end CG3056                 mRNA binding <br /> 19305        CG7408b                            Within 59-UTR CG7408                 N-acetylgalactosamine-4-sulfatase activity <br />    The insertion sites closest gene PBac element mutant line listed. The insertion sites determined by <br /> Exelixis using iPCR verified FlyBase publicly available. <br />    a <br />      We verified insertion sites lines iPCR. The proposed molecular functions genes provided FlyBase also <br /> listed. <br />    b <br />      Genes tested RNAi screens Cheng et al. (2005). <br />  <br />  <br />  <br /> the intracellular pathogen Mycobacterium leprae hu-               pression upregulated Drosophila S2 cells re- <br /> mans (Lorenzetti et al. 2004; Schurr et al. 2006).                   sponse LPS challenge regulation dependent <br /> Polymorphisms PACRG PARK2 linked                    activation JNK signaling Imd pathway <br /> to susceptibility humans pathogens S. typhi            (Boutros et al. 2002; Park et al. 2004). No immune <br /> and S. paratyphi (Ali et al. 2006). The ubiquitin-             function gene demonstrated. <br /> related gene, CG2247 represented line 18050, con-                    gr28b seven-pass transmembrane gustatory re- <br /> tains F-box domain facilitates interactions              ceptor important taste perception. Expression stud- <br /> between proteins polyubiquitination. Our iden-                ies shown protein expressed bitter <br /> tification genes reveal new processes                taste cells sensilla fly taste organs. Iden- <br /> host defense pathogenesis ubiquitin-                    tifying gene involved taste perception context <br /> mediated protein degradation involved.                            immunity pathogenesis somewhat surprising. <br />    RNA binding/processing components: The second                     Future experiments reveal feeding behavior of <br /> main class mutants identified screen                  fly affected, affecting immune response <br /> includes proteins involved various processes in-                  fly. Alternatively, gr28b serving function <br /> volving RNA. CG3527 involved rRNA processing,                  taste perception fly. <br /> specifically processing small ribosomal subunit.                 Phenotypic characterization mutants&#226;&#8364;&#8221;defense <br /> CG32706 CG3056 reported mRNA-                        defects vs. endurance defects: To distinguish of <br /> binding activity. Expression <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> analyses              mutants defects immunity vs. pathogenesis, <br /> not published genes.                                  performed secondary screens. First, mea- <br />    Additional interesting genes recovered: Two addi-                 sured L. monocytogenes growth determine flies <br /> tional genes recovered great             defect preventing growth microbe. <br /> are ets21C gr28b. ets21C DNA-binding protein                Second, measured survival infected S. <br /> with transcription factor activity. Previous studies            aureus determine mutants general <br /> demonstrated ets21C immune regulated: Its ex-                defect preventing gram-positive bacterial growth <br />  1812                                       J. S. Ayres, N. Freitag D. S. Schneider <br />  <br />                                                                TABLE 4 <br />                 Characterization Drosophila mutants&#226;&#8364;&#8221;sensitivity S. aureus S. typhimurium growth <br />                                                     L. monocytogenes <br />  <br />         PBac line               CG no.                Symbol          S. aureus        S. typhimurium         Bacterial load <br />         10067             CG10536                     cbx                1                   0                      1 <br />         10675             CG15120/CG16926                                1                   0                      1 <br />         10881             CG32706                                        1                   0                      1 <br />         17872             CG4857                                         1                   0                      1a <br />         18050             CG2247                                         1                   0                      1 <br />         19207             CG2397                      cyp6a13            1                   0                      1b <br />         19220             CG3056                                         1                   0                      1 <br />         10036             CG11489                                        1                   0                      1 <br />         18609             CG12487                     bobA               1                   0                      1b <br />         18270             CG3527                                         1                   0                      0 <br />         18398             CG2152                      pcmt               1                   0                      0 <br />         19305             CG7408                                         1                   0                      0 <br />         10496             CG14899                                        0                   0                      1 <br />         19181             CG6128                                         0                   0                      1 <br />         18678             CG2914                      ets21C             0                   0                      0 <br />         10269             CG13564                                        0                   0                      0 <br />         10618             CG11293                                        1                   1                      0 <br />         10743             CG13788                     gr28b              1                                          1 <br />            Mutant lines injected 10 CFUs S. aureus 104 CFUs S. typhimurium survival flies <br />                                               2 <br />  <br />         monitored course infection. The growth L. monocytogenes mutant lines determined <br />         described materials methods. 1 indicates increased growth sensitivity compared wild-type <br />         flies, indicates resistance decreased growth compared wild-type flies. 0 indicates change sensitivity <br />         growth compared wild type. Statistical analysis survival curves using log-rank analysis lines <br />         P , 0.05 considered statistically significant. Statistical analysis bacterial growth using t- <br />         test lines P , 0.05 considered statistically significant. <br />            a <br />              Increased growth 24 hr postinfection only. <br />            b <br />              Increased growth 48 hr postinfection only. <br />  <br />  <br /> (Toll mutants sensitive microbes).            These flies able endure infection than <br /> Third, measured survival infected S.                    wild-type flies bacterial levels similar. <br /> typhimurium determine mutants sen-                  Often, studies examining Drosophila immune <br /> sitive gram-negative bacteria (as expected               response gram-positive bacteria involve challenges <br /> for Imd pathway mutants) (Table 4).                                  nonpathogens used specific molecular <br />    To monitor L. monocytogenes growth, flies in-                readouts AMP production assess immune <br /> fected, homogenized 0, 24, 48 hr postinfection,               activity (Michel et al. 2001; Degregorio et al. 2002; <br /> and plated count viable bacteria. Twelve mutants                  Rutschmann et al. 2002; Tauszig-Delamasure et al. <br /> had increased L. monocytogenes growth compared                  2002). Defects signaling resulted inability to <br /> to wild-type flies. We consider nature muta-            mount immune response tested gram-positive <br /> tions immunocompromising fly                    microbes. On basis examination mu- <br /> they allow increased growth L. monocytogenes. An                  tants, clear Drosophila response gram- <br /> alternative explanation mutants              positive bacteria complex previously <br /> be environment pathogen mutant                appreciated. Not mutants sensitive a <br /> fly favorable L. monocytogenes growth             second gram-positive pathogen, S. aureus. The presence <br /> mutation affects physiology outside nor-               mutant lines clearly immunocom- <br /> mally considered immune system. The remaining                 promised respect L. monocytogenes no <br /> mutants exhibited significant difference L. mono-              change S. aureus susceptibility indicates are <br /> cytogenes levels compared wild type. This new              multiple mechanisms pathways used fight gram- <br /> class mutant phenotype sense                positive pathogenic bacteria. <br /> commonly studied field fly immunity. Since                    To determine mutations causing increased <br /> microbial proliferation unchanged group,                  sensitivity L. monocytogenes infection affect <br /> these mutants regarded immunocom-                   infections gram-negative bacteria, challenged <br /> promised; instead, suggest flies               flies S. typhimurium. Only gene, CG11293, shows <br /> altered physiology makes sensitive                 increased sensitivity S. typhimurium. This mutant ex- <br /> the pathology induced L. monocytogenes infection.             hibits increased sensitivity L. monocytogenes S. aureus <br />                                        A Survival-Based Screen Defense Endurance                                   1813 <br />  <br />                           TABLE 5                                 mutants Dhly DactA determine our <br />         Characterization L. monocytogenes mutants           Drosophila mutants involved specific steps the <br />                        Drosophila lines                           bacterial life cycle. We hypothesized fly <br />                                                                   mutations involved specific stages life cycle, <br /> PBac line           CG no.           Symbol     Dhly      DactA   fly susceptibility mutant bacterial infections <br />                                                                   different compared to wild-type bacterial <br /> 18270         CG3527 <br />                                                                   infection. <br /> 10067         CG10536                cbx <br /> 10269         CG13564                                                In 14 mutant fly lines, mutant L. <br /> 10496         CG14899                                             monocytogenes strains acted wild-type flies, <br /> 10618         CG11293                                             killing flies slowly wild-type L. monocyto- <br /> 10675         CG15120/CG16926                                     genes (Table 5) (individual death curves provided in <br /> 18050         CG2247                                              supplemental materials). For 4 fly mutants <br /> 19181         CG6128                                              (CG11489, CG4857, CG32706, gr28b) flies were <br /> 19207         CG2397                 cyp6a13 <br />                                                                   equally sensitive L. monocytogenes mutant <br /> 19220         CG3056 <br /> 19305         CG7408                                              wild-type L. monocytogenes. Two mutants, CG11489 <br /> 18678         CG2914                 ets21C                       CG4857, just sensitive infection DactA <br /> 18609         CG12487                bobA                         infection wild-type bacteria while <br /> 10036         CG11489                                      0      sensitive Dhly mutants. In other <br /> 17872         CG4857                                       0      words, DactA L. monocytogenes de- <br /> 10881         CG32706                             0               fective cell-to-cell spread, mutant fly lines are <br /> 18398         CG2152                 pcmt <br />                                                                   just sensitive infected wild- <br /> 10743         CG13788                gr28b        0        0 <br />                                                                   type bacteria. However, mutant fly lines respond <br />    Fly lines infected Dhly DactA mutant lines     like wild-type flies infection L. monocytogenes <br /> described materials methods. indicates resistance          lacking LLO&#226;&#8364;&#8221;they resistant. On basis of <br /> compared 10403S infection. 0 indicates change sen- <br /> sitivity compared 10403S infection. Statistical analysis    RNAi screen results, Cheng et al. (2005) suggested <br /> survival curves using log-rank analysis lines   mutants like sensitive LLO <br /> P , 0.05 considered statistically significant.               knocked-down genes involved in <br />                                                                   controlling LLO toxicity. This model apply to <br /> infection unaltered L. monocytogenes growth. We sug-          mutants; however, mutants also <br /> gest flies general pathogenesis defect          sensitive S. aureus presumably sensitivity S. <br /> because sensitive different pathogenic          aureus caused perturbed process that <br /> bacteria bacterial load comparable          result sensitivity LLO. Perhaps mutant <br /> that wild-type flies. gr28b exhibits increased resistance      affects general process involved sensing bacteria. <br /> to S. typhimurium infection sensitive            CG32706 sensitive Dhly mutants compara- <br /> S. aureus. This increased resistance unexpected            tively insensitive DactA infection. That is, though <br /> phenotype; aware fly mutations            Dhly strain defective vacuole escape, kills this <br /> that cause resistance gram-negative bacteria               mutant fly line similar kinetics wild-type L. <br /> increased sensitivity gram-positive bacteria.                  monocytogenes. Yet fly line resistant L. mono- <br />    Characterization L. monocytogenes mutants                cytogenes lack ActA defective cell-to-cell <br /> Drosophila mutants: We chose L. monocytogenes              spread. Perhaps flies LLO required L. <br /> pathogen screen life cycle viru-          monocytogenes released cytosol. Similar <br /> lence factors characterized, and, using         phenotypes observed in vitro screens in <br /> bacterial mutants, test changes pathogen-           genes involved later stages vesicular traffick- <br /> esis mutant flies. After taken host          ing knocked (Cheng et al. 2005). <br /> cell, L. monocytogenes secretes LLO lyse phago-               gr28b sensitive Dhly DactA L. monocytogenes. As <br /> some membrane release bacteria                   shown above, gr28b sensitive L. monocytogenes and <br /> cytoplasm. The protein ActA important nucleating           S. aureus, increased resistance S. typhimurium. <br /> host actin pole bacterium. Actin polymer-           Because mutant strains L. monocytogenes not <br /> ization drives motility L. monocytogenes cyto-          attenuated fly line, suggest LLO ActA <br /> plasm infected cell enables bacteria             required bacteria elicit pathogenic <br /> move host cell entering           effects fly line. In words, L. monocytogenes <br /> extracellular space, avoiding immune defenses              lacking factors required normal intracellular patho- <br /> host. We Dhly (LLO deletion) DactA                 genesis attenuated, indicating L. monocytogenes <br /> mutants strongly attenuated wild-type Drosophila,          gr28b mutant extracellular population. <br /> with MTD 18&#226;&#8364;&#8220;20 days 11&#226;&#8364;&#8220;12 days, respectively                Members immunity pathways identified: <br /> (Figure 1, C D). We examined susceptibility                Two known immunity genes present the <br /> our fly mutants infection L. monocytogenes            collection mutant flies tested (imd kenny). <br />  1814                                     J. S. Ayres, N. Freitag D. S. Schneider <br />  <br /> Because previous studies fly immunity primarily          bacteria host genes involved in <br /> focused Toll Imd pathways, anticipated             infection. Attempts overcome limitations <br /> that genes identified                RNAi screens S2 cells infected <br /> screen. These mutants median times death               L. monocytogenes. These screens revealed im- <br /> 2 days, tested            portant host factors required internalization and <br /> outside cutoff median time death 1 day.        growth L. monocytogenes host cells not <br /> As screened relatively small number mutants,       capable identifying immunity- pathogenesis-related <br /> possible did hit members Toll             genes. By screening host survival whole-fly model, <br /> signaling pathway small sample size. In           probe host contributions infection that <br /> addition, Toll pathway mutants meet               extend processes screens <br /> our viability requirement excluded. Re-             uncovered genes discovered the <br /> gardless, screen demonstrates fighting infec-       methods. <br /> tion interactions occur host               We thank Marc Dionne, Linh Pham, Eric Mabery, Madeleine <br /> pathogen complex, processes addition             Moule critical review manuscript. DP-L3078 (DactA) DP- <br /> to pathways regulating AMP transcription involved.       L2161 (Dhly) kindly provided Julie Theriot (Stanford <br />    The importance performing vivo                 University). The S. aureus strain provided Ellen Jo Baron. We <br />                                                                  thank Stanley Falkow, Michael Simon, Julie Theriot, members <br /> in vitro screens: We compared list mutant genes <br />                                                                  Schneider lab helpful discussions. We especially thank <br /> with hundreds genes identified RNAi screens            Adam, Baxter, Buster Ayres support. This work was <br /> monitoring L. monocytogenes growth cultured Dro-              supported National Institutes Health grants RO1 A1053080-01 <br /> sophila cells. Because mutations essential               (D.S.S.) A1055651 (D.S.S.) National Science Founda- <br /> functions <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>             tion graduate research program ( J.S.A.). <br /> fly, anticipated genes in <br /> the fly identified vitro. Conversely, we <br /> also expected genes important                                 LITERATURE CITED <br /> whole animal tissue culture. Nonetheless, we <br />                                                                  Agaisse, H., L. S. Burrack, J. Philips, E. J. Rubin, N. Perrimon et al., <br /> expected significant overlap sets genes                2005 Genome wide RNAi screen host factors required for <br /> isolated screens; instead,                   intracellular bacterial infection. Science 309: 1248&#226;&#8364;&#8220;1251. <br /> overlap. The screen Agaisse et al. (2005)               Ali, S., A. M. Vollaard, S. Widjaja, C. Surjadi, E. van Vosse et al., <br />                                                                       2006 PARK2/PACRG polymorphisms susceptibility ty- <br /> genomewide screen theoretically mutants                phoid paratyphoid fever. Clin. Exp. Immunol. 144: 425&#226;&#8364;&#8220;431. <br /> should tested screen. The dsRNA                Ayres, J. S., D. S. Schneider, 2006 Genomic dissection mi- <br /> library used Cheng et al. (2005) contained                crobial pathogenesis cultured Drosophila cells. Trends Micro- <br />                                                                       biol. 14: 101&#226;&#8364;&#8220;104. <br /> our positive mutants, identified              Boutros, M., H. Agaisse N. Perrimon, 2002 Sequential activa- <br /> from vitro screen (see Table 3). Both               tion signaling pathways innate immune responses in <br /> in vitro screens selected bacterial phenotypes, in-               Drosophila. Dev. Cell 3: 711&#226;&#8364;&#8220;722. <br />                                                                  Cheng, L. W., J. P. Viala, N. Stuurman, U. Wiedermann, R. D. Vale <br /> cluding growth vacuole escape, utilized cell              et al., 2005 Use RNA interference Drosophila S2 cells to <br /> type. The majority genes identified vitro            identify host pathways controlling compartmentalization in- <br /> screens are, surprisingly, involved vesicular traf-            tracellular pathogen. Proc. Natl. Acad. Sci. USA 102: 13646&#226;&#8364;&#8220;13651. <br />                                                                  Cossart, P., M. Lecuit, 1998 Interactions Listeria monocyto- <br /> ficking phagocytosis. By screening host survival              genes mammalian cells entry actin-based move- <br /> in whole-animal model, able identify different           ment: bacterial factors, cellular ligands signaling. EMBO J. <br /> host genes involved variety processes infec-           17: 3797&#226;&#8364;&#8220;3806. <br />                                                                  Cossart, P., J. Pizarro-Cerda M. Lecuit, 2003 Invasion of <br /> tion, including immune response pathogenesis.                 mammalian cells Listeria monocytogenes: functional mimicry <br />    Conclusions: In study demonstrate power                subvert cellular functions. Trends Cell Biol. 13: 23&#226;&#8364;&#8220;31. <br /> of combining genetically tractable organisms,            Degregorio, E., P. T. Spellman, P. Tzou, G. M. Rubin B. <br />                                                                       Lemaitre, 2002 The Toll Imd pathways major reg- <br /> host pathogen, reveal previously unchar-                 ulators immune response Drosophila. EMBO J. 21: <br /> acterized group genes involved immunity                     2568&#226;&#8364;&#8220;2579. <br /> pathogenesis. Previous genetic studies using or-            Domann, E., J. Wehland, M. Rohde, S. Pistor, M. Hartl et al., <br />                                                                       1992 A novel bacterial virulence gene Listeria monocytogenes <br /> ganism separately greatly limited scope genes              required host cell microfilament interaction homology <br /> processes identified. The use Drosophila               proline-rich region vinculin. EMBO J. 11: 1981&#226;&#8364;&#8220;1990. <br /> to study innate immune response infection                 Foley, E., P. H. O&#226;&#8364;&#8482;Farrell, 2004 Functional dissection in- <br />                                                                       nate immune response genome-wide RNAi screen. PLoS <br /> facilitated understanding signaling events                Biol. 2: E203. <br /> lead production AMPs. Previous screens          Gedde, M. M., D. E. Higgins, T. G. Tilney D. A. Portnoy, <br /> fly S2 cells focused identifying genes                 2000 Role listeriolysin O cell-to-cell spread Listeria mono- <br />                                                                       cytogenes. Infect. Immun. 68: 999&#226;&#8364;&#8220;1003. <br /> involved signaling pathways, did           Gesellchen, V., D. Kuttenkeuler, M. Steckel, N. Pelte M. <br /> look process. In vitro studies using L.                   Boutros, 2005 An RNA interference screen identifies Inhibi- <br /> monocytogenes expanded knowledge better                   tor Apoptosis Protein 2 regulator innate immune sig- <br />                                                                       naling Drosophila. EMBO Rep. 6: 979&#226;&#8364;&#8220;984. <br /> understand pathogenic mechanisms used intracellu-             Kambris, Z., S. Brun, I. H. Jang, H. J. Nam, Y. Romeo et al., <br /> lar pathogens. Yet genetic studies focused                 2006 Drosophila immunity: large-scale vivo RNAi screen <br />                                               A Survival-Based Screen Defense Endurance                                                1815 <br />  <br />     identifies serine proteases required Toll activation. Curr.     Michel, T., J. M. Reichart, J. A. Hoffmann J. Royet, 2001 Dro- <br />     Biol. 16: 808&#226;&#8364;&#8220;813.                    <br /> </body></html>