<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Int J Biochem Cell Biol. 2008 ; 40(11): 2560&#226;&#8364;&#8220;2571. doi:10.1016/j.biocel.2008.05.003. <br />  <br />  <br />  <br />                            Expression, essentiality, microtiter plate assay for <br />                            mycobacterial GlmU, bifunctional glucosamine-1-phosphate <br />                            acetyltransferase N-acetylglucosamine-1-phosphate <br />                            uridyltransferase <br />  <br />                            Wenli Zhang1, Victoria C. Jones2, Michael S. Scherman2, Sebabrata Mahapatra2, Dean <br />                            Crick2, Suresh Bhamidi2, Yi Xin3, Michael R. McNeil2*, Yufang Ma1,4* <br />                            1Department Biochemistry Molecular Biology, Dalian Medical University, Dalian 116044, China <br />                            2Department Microbiology, Immunology Pathology, Colorado State University, Fort Collins, <br />                            Colorado 80523, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            3Department Biotechnology, Dalian Medical University, Dalian 116044, China <br />                            4Liaoning Provincial Core Lab Glycobiology Glycoengineering, Dalian 116044, China <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 UDP-N-acetyl-D-glucosamine (UDP-GlcNAc) essential precursor peptidoglycan the <br />                                 rhamnose-GlcNAc linker region mycobacterial cell wall. In Mycobacterium tuberculosis H37Rv <br />                                 genome, Rv1018c shows strong homology GlmU protein involved formation UDP- <br />                                 GlcNAc bacteria. GlmU bifunctional enzyme catalyzes sequential steps in <br />                                 UDP-GlcNAc biosynthesis. Glucosamine-1-phosphate acetyl transferase catalyzes formation of <br />                                 N-acetylglucosamine-1-phosphate, N-acetylglucosamine-1-phosphate uridylyltransferase <br />                                 catalyzes formation UDP-GlcNAc. Since inhibition peptidoglycan synthesis results <br />                                 cell lysis, M. tuberculosis GlmU potential anti-tuberculosis drug target. In study cloned <br />                                 M. tuberculosis Rv1018c (glmU gene) expressed soluble GlmU protein E. coli BL21(DE3). <br />                                 Enzymatic assays showed M. tuberculosis GlmU protein exhibits glucosamine-1-phosphate <br />                                 acetyltransferase N-acetylglucosamine-1-phosphate uridylyltransferase activities. We also <br />                                 investigated effect Mycobacterium smegmatis activity GlmU fully removed or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 reduced genetic approach. The results showed activity GlmU required growth of <br />                                 M. smegmatis bacteria did grow absence active GlmU enzyme. As of <br />                                 functional GlmU enzyme gradually reduced temperature shift experiment, M. <br />                                 smegmatis cells non-viable morphology changed normal rod shape stubby- <br />                                 rounded morphology cases lysed. Finally microtiter plate based assay GlmU <br />                                 activity OD340 read developed. These studies support further <br />                                 development M. tuberculosis GlmU enzyme target new anti-tuberculosis drugs. <br />  <br />  <br />  <br />  <br />                            *Co-corresponding authors: Yufang Ma, Department Biochemistry Molecular Biology, Dalian Medical University, Dalian 116044, <br />                            China. Email: yufang_ma@hotmail.com. Fax: +86-411-86110139. Phone: +86-411-86110307; Michael McNeil, Department of <br />                            Microbiology, Immunology Pathology, Colorado State University, Fort Collins, Colorado 80523, USA. Email: <br />                            mmcneil@colostate.edu. Fax: (970) 491-1815. Phone: (970) 491-1784. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service customers <br />                            providing early version manuscript. The manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. Please note production process errors discovered could <br />                            affect content, legal disclaimers apply journal pertain. <br />                             Zhang et al.                                                                                                  Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     Mycobacterium tuberculosis; Mycobacterium smegmatis; GlmU; glucosamine-1-phosphate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     acetyltransferase; N-acetylglucosamine-1-phosphate uridylyltransferase <br />  <br />  <br />                               1. Introduction <br />                                              The emergence drug resistant strains bacteria problem in <br />                                              controlling infectious disease (Peterson, 2005). This especially true case of <br />                                              tuberculosis. Tuberculosis (TB) remains major threat world health; world-wide <br />                                              statistics million people developing active tuberculosis nearly fourth these <br />                                              dying disease stark reality (Dye et al.1999). (See Global Alliance TB <br />                                              drug Development www.tballiance.org.) TB, public health problem, been <br />                                              complicated lack wide array chemotherapeutics causative agent, <br />                                              Mycobacterium tuberculosis, hence, emergence drug-resistant strains few <br />                                              front-line drugs large problem recent years. Although current first-line anti-TB <br />                                              drug regimens achieve 99% efficacy, reduced drug <br />                                              resistance (O&apos;Brien Nunn, 2001). Multiple drug resistant TB (MDR-TB), bacillus <br />                                              resistant line drugs used fearsome type resistance; now, <br />                                              however, strains resistant second line antibiotics appearing (Gandhi et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              2006; Van Rie Enarson, 2006); dubbed extreme drug resistant TB (XDR- <br />                                              TB). <br />  <br />                                              The mycobacterial cell wall core consists layers (McNeil Brennan, 1991; Lee et al., <br />                                              2006). The highly impermeable outer layer composed mycolic acids lipids <br />                                              containing 70 90 carbons. The inner layer consists peptidoglycan. These layers are <br />                                              covalently tethered connecting polysaccharide arabinogalactan (McNeil Brennan, <br />                                              1991; Lee et al., 2006), attached peptidoglycan disaccharide linker, <br />                                              &#206;&#177;-L-rhamnosyl-(1,3)-&#206;&#177;-D-GlcNAc-(phosphate). Thus, GlcNAc essential component of <br />                                              peptidoglycan (as bacteria) rhamnose-GlcNAc linker cell wall in <br />                                              mycobacteria. <br />  <br />                                              In E.coli (Mengin-Lecreulx Vanheijenoort, 1994; Riley et al., 2006), enzymes <br />                                              catalyze sequential steps biosynthesis UDP-GlcNAc, activated nucleotide <br />                                              sugar GlcNAc. The glutamine fructose-6-phosphate transferase, encoded glmS gene, <br />                                              catalyzes fructose-6-phosphate glutamine glucosamine-6-phosphate glutamate, the <br />                                              phosphoglucosamine mutase, encoded glmM gene, converts glucosamine-6-phosphate <br />                                              (GlcNH2-6-P) glucosamine-1-phosphate (GlcNH2-1-P). The GlmU encoded glmU a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              bifunctional enzyme. The activity GlmU, glucosamine-1-phosphate acetyltransferase, <br />                                              catalyzes formation N-acetylglucosamine-1-phosphate (GlcNAc-1-P) <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                              glucosamine-1-phosphate using acetyl-CoA acetyl donor. The second activity GlmU, <br />                                              N-acetylglucosamine-1-phosphate uridylyltransferase, catalyzes formation UDP- <br />                                              GlcNAc GlcNAc-1-P uridine triphosphate (UTP). The active sites reside the <br />                                              C N terminal regions protein respectively (Olsen Roderick, 2001). Analysis of <br />                                              genome sequence Mycobacterium tuberculosis H37Rv shown protein <br />                                              encoded M. tuberculosis Rv3436c, Rv3441c Rv1018c homologous E. coli GlmS, <br />                                              GlmM, GlmU, respectively. Therefore, propose mycobacteria same <br />                                              biosynthetic pathway UDP-GlcNAc E. coli (Fig. 1). Antibiotics attacking <br />                                              peptidoglycan synthesis, &#206;&#178;-lactams glycopeptides result cell death typically <br />                                              lysis (Holtje, 1995), reason attractive. GlmS GlmM <br />                                              catalyze reactions identical similar eukaryotes. However reaction <br />  <br />  <br />  <br />                                               Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                Page 3 <br />  <br />  <br />                                           catalyzed GlmU unique prokaryotes GlmU important potential <br />                                           drug target. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           In study, expressed M. tuberculosis GlmU protein E. coli BL21(DE3) identified <br />                                           GlmU glucosamine-1-phosphate acetyltransferase N-acetylglucosamine-1- <br />                                           phosphate uridylyltransferase. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> determine effect active GlmU mycobacterial <br />                                           growth, prepared M. smegmatis glmU <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> strain complemented M. <br />                                           tuberculosis glmU rescue plasmid temperature sensitive replication. Finally <br />                                           developed microtiter plate based assay activity. <br />  <br />                               2. Materials methods <br />                               2.1 BACTERIAL STRAINS AND PLASMIDS <br />                                           The characteristics bacterial strains plasmids used study described Table <br />                                           1. E. coli DH5&#206;&#177;, NovaBlue BL21(DE3) cells grown Luria-Bertani (LB) broth or <br />                                           LB agar 37&#194;&#176;C routinely. M. smegmatis mc2155 cells grown LB broth containing <br />                                           0.05% Tween 80 LB agar 37&#194;&#176;C routinely. Sucrose added LB agar final <br />                                           concentration 10% selecting M. smegmatis glmU knockout strain ZW-2. Incubation <br />                                           M. smegmatis ZW-2 strain carrying temperature sensitive rescue plasmid 30&#194;&#176; <br />                                           C (permissive temperature) 42&#194;&#176;C (non-permissive temperature) respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The final concentration antibiotics used study follows: ampicillin (Amp), 100 <br />                                           &#194;&#181;g ml&#226;&#710;&#8217;1; gentamicin (Gen), 5 &#194;&#181;g ml&#226;&#710;&#8217;1 E. coli M. smegmatis; kanamycin (Kan), 50 &#194;&#181;g <br />                                           ml&#226;&#710;&#8217;1 E. coli 25 &#194;&#181;g ml&#226;&#710;&#8217;1 M. smegmatis; streptomycin (Str), 20 &#194;&#181;g ml&#226;&#710;&#8217;1 E. <br />                                           coli 10 &#194;&#181;g ml&#226;&#710;&#8217;1 M. smegmatis. <br />  <br />                               2.2 CLONING AND GENETIC MANIPULATION <br />                                           Cloning procedures performed according standard protocols (Sambrook Russell, <br />                                           2001). M. smegmatis mc2155 genomic DNA prepared previously described (Jackson <br />                                           et al., 2000). The Probe Southern blotting labeled digoxygenin (DIG)-labeled <br />                                           dUTP using DIG High Prime Labeling Detection Starter Kit I (Roche). The SmaI- <br />                                           digested M. smegmatis genomic DNA fragments separated 0.8% agarose gel and <br />                                           transferred Nytran membrane (Whatman Schleicher Schuell) described (Li et al., <br />                                           2006). DNA hybridization detection performed according Roche&#226;&#8364;&#8482;s instructions. <br />  <br />                               2.3 CLONING, EXPRESSION AND PURIFICATION OF M. TUBERCULOSIS GLMU AND E. COLI <br />                               MURA AND MURB <br />                                           The M. tuberculosis glmU gene (Rv1018c) amplified M. tuberculosis H37Rv <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           genomic DNA (supplied Colorado State University NIH contract &#226;&#8364;&#339;Tuberculosis <br />                                           research materials vaccine testing.&#226;&#8364;?) using Mtb glmU1 primer, 5&#226;&#8364;&#8482; <br />                                           CATATGACGTTTCCTGGTGACAC 3&#226;&#8364;&#8482; (underlined sequence NdeI site) Mtb <br />                                           glmU2 primer, 5&#226;&#8364;&#8482; CTCGAGTCACGGTGTCTGATCAGCGTC 3&#226;&#8364;&#8482; (underlined sequence is <br />                                           XhoI site). The amplified PCR product ligated pBluescript II KS(&#226;&#710;&#8217;) (Strategene) to <br />                                           generate plasmid pBS-Mtb glmU (Table 1). Following sequence confirmation Mtb glmU <br />                                           gene ligated NdeI XhoI sites pET-16b (Novagen), resulting pET16b- <br />                                           Mtb glmU (Table 1). pET16b-Mtb glmU transformed E. coli BL21(DE3) expression <br />                                           GlmU protein. <br />  <br />                                           E. coli BL21(DE3) containing pET16b-Mtb glmU grown 2 liters LB broth containing <br />                                           Amp overnight 37&#194;&#176;C. The cells transferred shaker room temperature induced <br />                                           overnight addition isopropyl-D-thiogalactopyranoside (IPTG) final concentration <br />                                           0.1 mM. The cells harvested pellets resuspended 20 ml lysis prep buffer (20 <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 4 <br />  <br />  <br />                                           mM Tris-HCl, pH 8.0, 500 mM NaCl, 20% glycerol) 1 &#194;&#181;M leupeptin, 1 &#194;&#181;M pepstatin A, <br />                                           1 mM phenylmethylsulphonyl fluoride (PMSF). The cells broken using French <br />                                           Pressure method, followed centrifugation 20 000 x g 40 minutes. The supernatant was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           applied 1.5 ml column volume HIS-Select HF affinity gel (Sigma). The column washed <br />                                           40 ml wash buffer (lysis prep buffer 20 mM imidazole, pH 8.0). The protein was <br />                                           eluted 15 ml elute buffer (20 mM Tris-HCl, 500 mM NaCl, 20% glycerol, 200 mM <br />                                           imidazole, pH 7.0) 1 &#194;&#181;M leupeptin, 1 &#194;&#181;M pepstatin A, 1 mM PMSF, 5 <br />                                           ml collected testing. <br />  <br />                                           The purified protein run 12% SDS-PAGE transferred nitrocellulose membrane <br />                                           (Whatman Schleicher Schuell) blotting buffer (20 mM Tris-<span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span>, 150 mM glycine and <br />                                           20% methanol). The membrane blocked 2% BSA TBST buffer (10 mM Tris-HCl, <br />                                           pH 8.0, 150 mM NaCl, 0.05% Tween 20) incubated (anti)-polyhistidine clone HIS-1 <br />                                           antibody (Sigma) 1 6000 dilution followed washing TBST. The membrane was <br />                                           incubated antimouse-IgG-AP conjugate (Sigma) protein band visualized in <br />                                           BCIP/NBT solution. <br />  <br />                                           The gene murA amplified E. coli K12 genomic DNA following two <br />                                           primers incorporating NcoI SmaI restriction enzyme sites (underlined): 5&#226;&#8364;&#8482; <br />                                           AACTCCATGGATAAATTTCGTGTTCAG 3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482; <br />                                           AGACCCGGGTTCGCCTTTCACACGC 3&#226;&#8364;&#8482;. The PCR products digested NcoI <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SmaI, ligated expression vector pTYB4, previously digested with <br />                                           NcoI SmaI. The E. coli murB gene amplified E. coli genomic DNA the <br />                                           following primers incorporating NcoI SmaI restriction enzyme sites (underlined) and <br />                                           cloned pTYB4: GCAGCCATGGACCACTCCTTAAAACC and <br />                                           GTTATCCCGGGTGAAATTGTCTCCAC. pTYB4-Ec murA pTYB4-Ec murB were <br />                                           transformed ER2566 respectively. The expressed E. coli MurA E.coli MurB proteins were <br />                                           purified IMPACT&#226;&#8222;&#162; Protein Purification according New England Biolabs&#226;&#8364;&#8482; <br />                                           instructions. <br />  <br />                               2.4 DETERMINATION OF ACETYL TRANSFERASE AND URIDYLYLTRANSFERASE <br />                               ACTIVITIES OF M. TUBERCULOSIS GLMU PROTEIN <br />                                           The acetyl transferase uridylyltransferase activities M. tuberculosis GlmU protein were <br />                                           determined separately. For acetyl transferase activity, reaction mixture contained 100 <br />                                           &#194;&#181;M acetyl CoA glucosamine-1-phosphate 10 mM Tris buffer (0.1 mM MgCl2 <br />                                           0.01% Triton X 100, pH 8). The transacetylation carried addition 100 <br />                                           ng GlmU (purified) crude M. smegmatis extract reaction incubated 30 <br />                                           minutes room temperature. A control experiment, addition enzyme source, <br />                                           substrates buffer carried out. After incubation, part <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           reaction mixture dried cleaned glass tube mild hydrolysis carried out <br />                                           0.2 mM TFA (Trifluroacetic acid) 80&#194;&#176;C 1 hour. The tubes cooled room <br />                                           temperature dried. This drying repeated twice presence methanol. This was <br />                                           followed acetylation 1:1 deutero acetic anhydride/pyridine, overnight. After drying <br />                                           solvents, acetylated products extracted D2O:CHCl3 (1:1) chloroform <br />                                           layer subjected GC/MS. The acetylated-derivatives dissolved chloroform prior <br />                                           injection DB-5 column initial temperature 50&#194;&#176;C held 1 min. The temperature <br />                                           increased 275&#194;&#176;C rate 5&#194;&#176;C/min. The mass spectral analysis carried positive <br />                                           CI mode collision gas methane. The starting material showed M+H-63 ion <br />                                           3 AMU higher product non-deuterated methyl group product. <br />  <br />                                           To determine uridylyltransferase active, enzyme reaction run reverse <br />                                           direction PPi UDP-GlcNAc GlcNAc-1-P UTP (Mengin-Lecreulx and <br />                                           Vanheijenoort, 1994). The reaction performed 40 &#194;&#181;l total volume containing 25 &#194;&#181;M <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                           Page 5 <br />  <br />  <br />                                           UDP-GlcNAc, 1 mM pyrophosphate (PPi) 42 &#194;&#181;g/ml GlmU protein 50 mM Tris-HCl <br />                                           (pH 8.2) containing 1 mM MgCl2. After incubation 37&#194;&#176;C 30 minutes, reaction was <br />                                           run Beckman HPLC 30 minute gradient starting 100% 75 mM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           KH2PO4 ending 100% 500 mM KH2PO4. <br />  <br />                                           For microtiter plate assay, GlmU activity linked MurA MurB E. coli. <br />                                           The assay run combining 4 &#194;&#181;g purified M. tuberculosis GlmU, 0.6 &#195;&#172;g purified E. coli <br />                                           MurA 80 ng purified E. coli MurA 80 &#194;&#181;l 200 mM Tris containing 10 mM MgCl2, 200 <br />                                           &#194;&#181;M NADPH, 20 mM KCl 96 plate. The reaction started by <br />                                           adding 20 &#194;&#181;l 200 mM Tris, 10 mM MgCl2 containing 1 mM UTP, 1 mM acetyl CoA, 1 mM <br />                                           glucosamine-1-phosphate, 1 mM phosphoenol pyruvate. The Absorbance 340 nm was <br />                                           followed 45 minutes 30&#194;&#176;C Benchmark Plus (Bio-Rad) plate reader data were <br />                                           analyzed using Microsoft Excel. <br />  <br />                               2.5 CONSTRUCTION OF SUICIDE PLASMID AND RESCUE PLASMID <br />                                           The amino acid sequence M. tuberculosis H37Rv GlmU protein <br />                                           (http://genolist.pasteur.fr/TubercuList/) used query BLASTP identify most <br />                                           <span id='am-160' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> M. smegmatis mc2155 genome. The M. smegmatis glmU gene <br />                                           (MSMEG_5426) 75% identical homology Rv1018c. M. smegmatis <br />                                           glmU (1449 bp) upstream region (491 bp) amplified mc2155 genomic DNA <br />                                           using LA Taq DNA polymerase (Takara), glmU1 primer <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (5&#226;&#8364;&#8482;ACTAGTCTCGACACGGACCCGGGAGC 3&#226;&#8364;&#8482;, underlined sequence SpeI site) and <br />                                           glmU2 primer (5&#226;&#8364;&#8482;TGCGGCCGCTCAGCTCTCGTCGCCCAACG3&#226;&#8364;&#8482;, underlined <br />                                           sequence NotI site). The PCR product 1947 bp purified cloned pMD18-T <br />                                           vector (Takara) generate plasmid plasmid pZW-1 (Table 1). The glmU gene disrupted <br />                                           inserting kanamycin resistance cassette (kanR) pUC4K (GE Healthcare), yielding <br />                                           pZW-2 (Table 1). The pZW-2 digested NotI SpeI resulting glmU::kanR <br />                                           fragment (3.1 kb) ligated NotI SpeI sites pPR27-xylE (Li et al., 2006), <br />                                           resulting suicide plasmid pZW-3 (Table 1). The parent plasmid pPR27 (Pelicic et al., <br />                                           1997) shuttle vector containing replication origins E.coli mycobacteria. <br />                                           Specifically, replication origin mycobacteria mutations resulting plasmid <br />                                           replication temperature sensitive mycobacteria. Therefore, pZW-3 replicates at <br />                                           permissive temperature (30&#194;&#176;C), replicate non-permissive temperature (42&#194;&#176;C), <br />                                           resulting integration M. smegmatis genome intact glmU gene and <br />                                           glmU::kanR present. <br />  <br />                                           M. tuberculosis glmU gene amplified M. tuberculosis H37Rv genomic DNA using <br />                                           Mtb glmU1 primer, 5&#226;&#8364;&#8482; CATATGACGTTTCCTGGTGACAC 3&#226;&#8364;&#8482; (underlined sequence is <br />                                           NdeI site)&#226;&#8364;&#8482; Mtb glmU3 primer, 5&#226;&#8364;&#8482;GGATCCTCACGGTGTCTGATCAGCGTC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           3&#226;&#8364;&#8482; (underlined sequence BamHI site). The PCR product M. tuberculosis glmU cloned <br />                                           pMD18-T vector generate plasmid pZW-4 (Table 1). The M. tuberculosis glmU gene <br />                                           ligated NdeI BamHI sites pET23b-Phsp60 (Li et al., 2006), yielding in <br />                                           pZW-5 (Table 1). The Phsp60-Mtb glmU ligated XbaI BamHI sites pCG76 <br />                                           (Guilhot et al., 1994), resulting rescue plasmid pZW-6 (Table 1). The M. tuberculosis <br />                                           glmU gene transcribed promoter heat shock protein 60 M. bovis BCG <br />                                           M. tuberculosis GlmU protein complemented glmU::kanR M. smegmatis ZW-2. The <br />                                           parent plasmid pCG76 temperature-sensitive mycobacterial replication origin as <br />                                           pPR27 replicate 30&#194;&#176;C, effectively lost 42&#194;&#176;C. <br />  <br />                               2.6 SELECTION OF M. SMEGMATIS GLMU KNOCKOUT STRAIN ZW-2 <br />                                           M. smegmatis electrocompetent cells prepared described (Pelicic et al., 1997) and <br />                                           suicide plasmid pZW-3 electroporated competent cells Gene Pulser (Bio- <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 6 <br />  <br />  <br />                                            Rad). Transformants grown LB agar plates containing Kan Gen 30&#194;&#176;C. One <br />                                            colony propagated LB broth containing 0.05% Tween 80, Kan Gen 30&#194;&#176;C, <br />                                            cells plated LB agar plate containing Kan Gen incubated at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            42&#194;&#176;C colonies appeared. M. smegmatis ZW-1 (Table 1) integrated pZW-3 the <br />                                            glmU locus selected using Southern blot. <br />  <br />                                            The pZW-6 rescue plasmid electroporated ZW-1 cells. Transformants grown on <br />                                            LB agar plate containing Kan Str, incubated 30&#194;&#176;C. Transformants then <br />                                            plated LB agar plate containing Kan, Str 10% sucrose. M. smegmatis ZW-2 <br />                                            (glmU <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> strain) (Table 1) confirmed using Southern blot. <br />  <br />                               2.7 MONITORING GROWTH OF M. SMEGMATIS ZW-2 <br />                                            The growth M. smegmatis ZW-2 followed monitoring optical density (OD) at <br />                                            600 nm cultures measuring colony forming units (CFU). M. smegmatis ZW-2 was <br />                                            inoculated 2 tubes containing 5 ml LB broth 0.05% Tween 80 appropriate <br />                                            antibiotics, cells incubated 30&#194;&#176;C 42&#194;&#176;C respectively. M. smegmatis mc2155 <br />                                            M. smegmatis mc2155 pZW-6 used controls. The OD600 detected the <br />                                            interval 24 hours growth curves 30&#194;&#176;C 42&#194;&#176;C obtained. <br />  <br />                                            To measure CFU M. smegmatis ZW-2 cells grown LB broth containing 0.05% Tween <br />                                            80 Kan 30&#194;&#176;C OD600 reached 0.10, cells transferred 42&#194;&#176;C <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            incubator. The OD600 cells grown 42&#194;&#176;C determined interval 24 hours. <br />                                            For CFU determinations (taken time points), dilutions spread LB agar <br />                                            plates containing Str Kan. The plates incubated 30&#194;&#176;C CFU counted. <br />  <br />                               2.8 DETECTION OF M.TUBERCULOSIS GLMU PROTEIN IN M. SMEGMATIS ZW-2 <br />                                            M. smegmatis ZW-2 cells grown 30&#194;&#176;C (OD600 0.10) switched 42&#194;&#176;C incubator. <br />                                            The ZW-2 cells kept growing 30&#194;&#176;C controls. The cells harvested interval <br />                                            24 hours pellets resuspended lysis prep buffer 1 mM PMSF. The cells were <br />                                            broken sonication followed centrifugation 20 000 x g 30 minutes. The supernatants <br />                                            run 10% SDS-PAGE Western blot performed described 2.3. The <br />                                            membrane incubated anti-M. smegmatis GlmU serum mouse, was <br />                                            prepared lab (unpublished data), incubated antimouse-IgG-AP conjugate <br />                                            (Sigma). The protein band visualized BCIP/NBT solution. <br />  <br />                               2.9 MORPHOLOGY OF M. SMEGMATIS ZW-2 AT 42 &#194;&#176;C <br />                                            The effects decreasing GlmU activity morphology ZW-2 examined by <br />                                            scanning electron microscopy (SEM) time points CFUs determined. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            The cells washed times 0.1 M Phosphate buffer (pH 7.4) fixed 2.5% <br />                                            glutaraldehyde 1% OsO4 followed dehydration graded series ethanol (20, <br />                                            40, 60, 70, 80, 90, 100%). The cell pellet resuspended 100% ethanol, applied silicon <br />                                            wafer slide, gold coated thickness 5 nm. The cells examined JSM-6500F <br />                                            scanning electron microscope (at Central Instrumentation Facility, Colorado State <br />                                            University) using accelerating voltage 15 kV. <br />  <br />                               3. Results <br />                               3.1. INITIAL IDENTIFICATION OF GLMU IN MYCOBACTERIA <br />                                            The amino acid sequence encoded M. tuberculosis H37Rv gene Rv1018c expected to <br />                                            encoded GlmU homology bacterial GlmU proteins. This sequence was <br />                                            used query BLASTP identify <span id='am-161' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> M. smegmatis mc2155 <br />                                            genome. The M. smegmatis glmU gene (MSMEG_5426) 75% identical <br />  <br />                                            Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 7 <br />  <br />  <br />                                           homology Rv1018c. The sequence alignment mycobacterial GlmU proteins <br />                                           E. coli GlmU protein shown Fig. 2. To confirm presence GlmU M. <br />                                           smegmatis, conversion GlcNH2-1-P GlcNAc-1-P, N-acetylase activity GlmU <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           shown occur crude M. smegmatis extracts (Fig. 3c) using GC/MS method shown <br />                                           Fig. 3a. <br />  <br />                               3.2 EXPRESSION AND PURIFICATION OF M. TUBERCULOSIS GLMU PROTEIN <br />                                           The M. tuberculosis glmU gene cloned NdeI XhoI sites pET16b yielding <br />                                           N-terminal histidine tag GlmU fusion protein expected molecular weight of <br />                                           54.10 kD. The M. tuberculosis GlmU expressed soluble protein E. coli BL21(DE3) <br />                                           induction 0.1 mM IPTG room temperature purified Ni2+ affinity <br />                                           chromatography. Purified M. tuberculosis GlmU protein determined expected <br />                                           size SDS-PAGE Western blot analysis (Fig. 4). <br />  <br />                               3.3 ACETYL TRANSFERASE AND URIDYLYLTRANSFERASE ACTIVITIES OF M. <br />                               TUBERCULOSIS GLMU PROTEIN <br />                                           The acetyl transferase purified M. tuberculosis GlmU protein demonstrated using the <br />                                           GC/MS assay shown Fig. 3d. The uridylyltransferase activity shown reverse <br />                                           direction UDP-GlcNAc converted UTP GlcNAc-1-P. Thus incubation of <br />                                           UDP-GlcNAc PPi 30 minutes lead complete disappearance UDP-GlcNAc (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           5c) control reaction (Fig. 5b), PPi included reaction mixture, <br />                                           lead reaction. <br />  <br />                               3.4 GENERATION OF M. SMEGMATIS GLMU GENE KNOCKOUT STRAIN ZW-2 <br />                                           M. smegmatis mc2155 transformants pZW-3 selected LB agar plates containing <br />                                           Kan Gen 30&#194;&#176;C colonies turned yellow catechol sprayed plates. <br />                                           The yellow colonies propagated LB broth containing 0.05% Tween 80, Kan Gen <br />                                           30&#194;&#176;C, spread LB agar plates containing Kan Gen 42&#194;&#176;C. Since pZW-3 <br />                                           plasmid unable replicate 42&#194;&#176;C, Kan resistant colonies plates necessarily <br />                                           integrated glmU::kanR glmU locus M. smegmatis mc2155 genome. An M. <br />                                           smegmatis transformant arose single homologous recombination event the <br />                                           glmU locus (ZW-1) confirmed Southern blot. <br />  <br />                                           After rescue plasmid pZW-6 electroporated M. smegmatis ZW-1, generation of <br />                                           M. smegmatis ZW-2 (glmU knockout strain) (can grow LB agar containing Kan 10% <br />                                           sucrose) selected for. Six M. smegmatis ZW-2 strains confirmed Southern blot <br />                                           (Fig. 6). All strains showed expected bands 1.0 kb (faint), 1.59 kb 1.79 kb. The band <br />                                           1.34 kb missing, confirming loss intact glmU gene. The bands 0.65 kb, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           0.80 kb 5.3 kb pZW-6 plasmid. <br />  <br />                               3.5 ESSENTIALITY OF GLMU GENE FOR MYCOBACTERIAL GROWTH <br />                                           In order determine GlmU required mycobacterial growth, growth curves of <br />                                           M. smegmatis ZW-2 30&#194;&#176;C 42&#194;&#176;C obtained (Fig. 7A). The results clearly showed <br />                                           M. smegmatis ZW-2 grew 30&#194;&#176;C, 42&#194;&#176;C pZW-6 to <br />                                           replicate. In contrast, M. smegmatis mc2155 M. smegmatis mc2155 containing pZW-6 <br />                                           grew 30&#194;&#176;C 42&#194;&#176;C. Therefore, results showed M. tuberculosis glmU gene <br />                                           essential mycobacterial growth. <br />  <br />                                           M. smegmatis ZW-2 grown 30&#194;&#176;C 0.10 OD600, culture switched to <br />                                           incubator 42&#194;&#176;C. The CFU OD600 time obtained shown Fig. 7B. Since <br />                                           functional GlmU protein produced M. smegmatis ZW-2 cells 30&#194;&#176;C, cells <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                 Page 8 <br />  <br />  <br />                                           multiplied 24 hours temperature shift (Fig. 7B). Then dramatically, with <br />                                           increased incubation time 42&#194;&#176;C, CFUs dropped GlmU decreased. <br />                                           Western blot data (Fig. 7C) confirmed expected loss GlmU protein, 1 day <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           insufficient GlmU detection western blot seen (Fig 7C lane 3). <br />  <br />                               3.6 MORPHOLOGICAL CHANGES OF M. SMEGMATIS ZW-2 AT 42&#194;&#176;C <br />                                           Since UDP-GlcNAc important precursor peptidoglycan mycobacterial cell wall, <br />                                           expected change morphology possibly lysis bacteria biosynthesis <br />                                           decreased. Therefore, temperature switched 30&#194;&#176;C 42&#194;&#176;C, morphology <br />                                           M. smegmatis ZW-2 grown 42&#194;&#176;C 24, 48, 72, 96 hours observed SEM. <br />                                           The morphology M. smegmatis ZW-2 30&#194;&#176;C just switching 42&#194;&#176;C (Fig. 8A) is <br />                                           indistinguishable M. smegmatis mc2155 wild type (Fig. 8G). With time, M. smegmatis <br />                                           ZW-2 cells 42&#194;&#176;C dramatically shorter rounder (Fig. 8B, C D). Some M. <br />                                           smegmatis ZW-2 cells 42&#194;&#176;C 96 hours showed evidence lysis (Fig. 8E F). These <br />                                           SEM results suggest lack GlmU activity result profound negative effects the <br />                                           bacterial cell morphology. <br />  <br />                               3.7 DEVELOPMENT OF A MICROTITER PLATE ASSAY FOR GLMU <br />                                           Given desirability GlmU drug target designed microtiter plate based assay <br />                                           detect activity. This assay (Fig. 1) relies enzymes MurA MurB convert <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           UDP-GlcNAc UDP-MurNAc concomitant reduction NADPH. Thus E. coli <br />                                           MurA MurB cloned expressed E coli resulting proteins purified as <br />                                           described material methods. As evident Fig. 9, enzymes substrates <br />                                           present NADPH oxidized concomitant decrease OD340. <br />  <br />                               4. Discussion <br />                                           UDP-GlcNAc essential precursor peptidoglycan rhamnose-GlcNAc linker region <br />                                           mycobacterial cell wall. The pathway UDP-GlcNAc biosynthesis significantly <br />                                           different eukaryotes prokaryotes. In eukaryotes, N-acetylation occurs GlcNH2-6-P <br />                                           GlcNH2-1-P. The second reaction, formation UDP-GlcNAc UTP and <br />                                           GlcNAc-1-P, prokaryotes eukaryotes. In eukaryotes, uridylyltransferase <br />                                           activity carried distinct monofunctional enzymes little sequence homology with <br />                                           GlmU. However, tertiary structure active site eukaryotic enzymes responsible for <br />                                           uridylyltranserase activity similar uridylyltransferase active site prokaryotic <br />                                           enzymes (Peneff et al., 2001a; Peneff et al., 2001b). Given similarity, acetylation <br />                                           reaction considered attractive drug target bacteria. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           We shown M. tuberculosis GlmU protein distinct activities; <br />                                           glucosamine-1-phosphate acetyltransferase N-acetylglucosamine-1-phosphate <br />                                           uridylyltransferase. For mycobacteria, GlmU enzyme clearly essential, it <br />                                           knocked M. smegmatis grow presence rescue plasmid containing the <br />                                           M. tuberculosis glmU gene, grow rescue plasmid does replicate. <br />                                           Insertional mutagenesis studies using transposon site hybridization (TraSH) methodology <br />                                           reported M. tuberculosis glmU essential gene (Sassetti et al.2003); now <br />                                           extended determine fate bacteria gene turned (Fig 7 &amp; Fig 8). Our <br />                                           experimental approach somewhat limited GlmU enzyme abruptly turned <br />                                           off, replication rescue plasmid containing functional M. tuberculosis <br />                                           glmU gene stopped. This results decrease GlmU present cell over <br />                                           time. SEM data showed dramatic change morphology including appeared lysis <br />                                           cells latest time point. The cells cleanly broken half happens <br />                                           &#206;&#178;-lactams, cells appear ruptured (Fig. 8E F). Therefore, the <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                        Page 9 <br />  <br />  <br />                                           essentiality GlmU effect loss activity mycobacterial cells strongly <br />                                           support development target new TB drugs. Potentially, new drugs have <br />                                           similar bactericidal properties &#206;&#178;-lactams. The assay developed microtiter plates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           allows inhibitors acetyl transferase uridylyltransferase screened <br />                                           for. <br />  <br />                                           Inhibitors GlmU important terms inhibiting linker region the <br />                                           mycobacterium cell wall. We previously demonstrated effects knocking the <br />                                           rhamnosyl formation enzymes, RmlD (Ma et al., 2002) Rml B RmlC (Li et al., <br />                                           2006). Assays inhibitors RmlC RmlD proceeding NIH Roadmap screening <br />                                           center. This work validates develops techniques appropriate similar screening of <br />                                           GlmU inhibitors. <br />  <br />                               Acknowledgement <br />                                           This work supported funds provided National Basic Research Program China (2006CB504400), <br />                                           National Natural Science Foundation China (30670454) National Institute Health (USA) (AI 33706). <br />  <br />  <br />                               References <br />                                           Dye C, Scheele S, Dolin P, Pathania V, Raviglione MC. WHO Global Surveillance Monitoring <br />                                              Project. Consensus statement. Global burden tuberculosis: estimated incidence, prevalence, and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              mortality country. The Journal American Medical Association 1999;282:677&#226;&#8364;&#8220;686. <br />                                           Gandhi NR, Moll A, Sturm AW, Pawinski R, Govender T, Lalloo U, Zeller K, Andrews J, Friedland G. <br />                                              Extensively drug-resistant tuberculosis cause death patients co-infected tuberculosis <br />                                              HIV rural area South Africa. Lancet 2006;368:1575&#226;&#8364;&#8220;1580. [PubMed: 17084757] <br />                                           Guilhot C, Otal I, Van Rompaey I, Martin C, Gicquel B. Efficient transposition mycobacteria: <br />                                              construction Mycobacterium smegmatis insertional mutant libraries. Journal Bacteriology <br />                                              1994;176:535&#226;&#8364;&#8220;539. [PubMed: 8288551] <br />                                           Holtje JV. From growth autolysis: murein hydrolases Escherichia coli. Archives Microbiology <br />                                              1995;164:243&#226;&#8364;&#8220;254. [PubMed: 7487333] <br />                                           Jackson M, Crick DC, Brennan PJ. Phosphatidylinositol essential phospholipid mycobacteria. <br />                                              Journal Biological Chemistry 2000;275:30092&#226;&#8364;&#8220;30099. [PubMed: 10889206] <br />                                           Kostrewa D, D&apos;Arcy A, Takacs B, Kamber M. Crystal structures Streptococcus pneumoniae N- <br />                                              acetylglucosamine-1- phosphate uridyltransferase, GlmU, apo form 2.33 A resolution in <br />                                              complex UDP-N-acetylglucosamine Mg(2+) 1.96 A resolution. Journal Molecular Biology <br />                                              2001;305:279&#226;&#8364;&#8220;289. <br />                                           Lee A, Wu SW, Scherman MS, Torrelles JB, Chatterjee D, McNeil MR, Khoo KH. Sequencing of <br />                                              oligoarabinosyl units released mycobacterial arabinogalactan endogenous arabinanase: <br />                                              identification distinctive novel structural motifs. Biochemistry 2006;45:15817&#226;&#8364;&#8220;15828. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              [PubMed: 17176104] <br />                                           Li W, Xin Y, McNeil MR, Ma Y. rmlB rmlC genes essential growth mycobacteria. <br />                                              Biochemical Biophysical Research Communications 2006;342:170&#226;&#8364;&#8220;178. [PubMed: 16472764] <br />                                           Ma Y, Pan F, McNeil MR. Determination dTDP-rhamnose formation essential growth of <br />                                              mycobacteria. J.Bacteriol 2002;184:3392&#226;&#8364;&#8220;3395. [PubMed: 12029057] <br />                                           Mengin-Lecreulx D, Vanheijenoort J. Copurification glucosamine-1-phosphate acetyltransferase and <br />                                                N-acetylglucosamine-1-phosphate uridylyltransferase activities Escherichia coli: characterization <br />                                                glmU gene product bifunctional enzyme catalyzing subsequent steps pathway <br />                                                UDP-N-acetylglucosamine synthesis. Journal Bacteriology 1994;176:5788&#226;&#8364;&#8220;5795. [PubMed: <br />                                                8083170] <br />                                           McNeil MR, Brennan PJ. Structure, function, biogenesis cell envelope mycobacteria in <br />                                                relation bacterial physiology, pathogenesis, drug resistance; thoughts possibilities <br />                                                arising recent structural information. Research Microbiology 1991;142:451&#226;&#8364;&#8220;463. [PubMed: <br />                                                1871433] <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                     Page 10 <br />  <br />  <br />                                           O&apos;Brien RJ, Nunn PP. The need new drugs tuberculosis, Obstacles, opportunities, next <br />                                               steps. American Journal Respiratory Critical Care Medicine 2001;163:1055&#226;&#8364;&#8220;1058. [PubMed: <br />                                               11316634] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Olsen LR, Roderick SL. Structure Escherichia coli GlmU pyrophosphorylase acetyltransferase <br />                                               active sites. Biochemistry 2001;40:1913&#226;&#8364;&#8220;1921. [PubMed: 11329257] <br />                                           Olsen LR, Vetting MW, Roderick SL. Structure E. coli bifunctional GlmU acetyltransferase active <br />                                               site substrates products. Protein Science 2007;16:1230&#226;&#8364;&#8220;1235. [PubMed: 17473010] <br />                                           Pelicic V, Jackson M, Reyrat JM, Jacobs WR Jr, Gicquel B, Guilhot C. Efficient allelic exchange and <br />                                               transposon mutagenesis Mycobacterium tuberculosis. Proceedings National Academy of <br />                                               Sciences 1997;94:10955&#226;&#8364;&#8220;10960. <br />                                           Peneff C, Mengin-Lecreulx D, Bourne Y. The crystal structures Apo complexed Saccharomyces <br />                                               cerevisiae GNA1 shed light catalytic mechanism amino-sugar N-acetyltransferase. <br />                                               Journal Biological Chemistry 2001;276:16328&#226;&#8364;&#8220;16334. [PubMed: 11278591] <br />                                           Peneff C, Ferrari P, Charrier V, Taburet Y, Monnier C, Zamboni V, Winter J, Harnois M, Fassy F, Bourne <br />                                               Y. Crystal structures human pyrophosphorylase isoforms complexes UDPGlc(Gal) <br />                                               NAc: role alternatively spliced insert enzyme oligomeric assembly active site <br />                                               architecture. The EMBO Journal 2001;20:6191&#226;&#8364;&#8220;6202. [PubMed: 11707391] <br />                                           Peterson LR. Squeezing antibiotic balloon: impact antimicrobial classes emerging resistance. <br />                                               Clinical Microbiology Infection 2005;11:4&#226;&#8364;&#8220;16. [PubMed: 16138814] <br />                                           Riley M, Abe T, Arnaud MB, Berlyn MK, Blattner FR, Chaudhuri RR, Glasner JD, Horiuchi T, Keseler <br />                                               IM, Kosuge T, Mori H, Perna NT, Plunkett G III, Rudd KE, Serres MH, Thomas GH, Thomson NR, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Wishart D, Wanner BL. Escherichia coli K-12: cooperatively developed annotation snapshot-2005. <br />                                               Nucleic Acids Research 2006;34:1&#226;&#8364;&#8220;9. [PubMed: 16397293] <br />                                           Sambrook, J.; Russell, DG. Molecular Cloning: A Laboratory Manual. 3 rd ed. New York: Cold Spring <br />                                               Harbor Laboratory Press; 2001. <br />                                           Sassetti CM, Boyd DH, Rubin EJ. Genes required mycobacterial growth defined high density <br />                                               mutagenesis. Molecular Microbiology 2003;48:77&#226;&#8364;&#8220;84. [PubMed: 12657046] <br />                                           Van Rie A, Enarson D. XDR tuberculosis: indicator public-health negligence. Lancet <br />                                               2006;368:1554&#226;&#8364;&#8220;1556. [PubMed: 17084741] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                        Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 1. <br />                                           (A) Biosynthetic pathway UDP-GlcNAc formation mycobacteria. GlmU (Rv1018c) a <br />                                           bifunctional enzyme glucosamine-1-phosphate acetyl transferase N- <br />                                           acetylglucosamine-1-phosphate uridylyl transferase activities. (B) The reactions MurA and <br />                                           MurB act UDP-GlcNAc product GlmU used linked assay in <br />                                           loss NADPH monitored. In bacterium, product, UDP-MurNAc, then <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           converted peptidoglycan. <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                             Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 2. <br />                                           Sequence alignment glmU E. coli, M. tuberculosis M. smegmatis. The symbol &#226;&#8364;&#339;.&#226;&#8364;? <br />                                           indicates breaks inserted order obtain best alignment. <br />                                           The sequence underlines indicate uridylyl transferase active site (Kostrewa et al., 2001). <br />                                           The active site acetylation involves amino acids adjacent, interacting beta <br />                                           helices depicted follows: bold letters Acetyl Coenzyme A binding amino <br />                                           acids, bold italics letters amino acids bind glucosamine-1-phosphate <br />                                           (Olsen et al., 2007). The numbering end line sequences refers the <br />                                           amino acid number species&#226;&#8364;&#8482; sequence. Consensus 1 shows homology M. <br />                                           tuberculosis M. smegmatis sequences (second lines) compared while <br />                                           Consensus 2 shows homology sequence organisms (first, second, and <br />                                           lines) compared. The symbols used Consensus 1 Consensus 2 follows: <br />                                           &#226;&#8364;&#339;.&#226;&#8364;? means consensus found; upper case abbreviation amino acid means a <br />                                           perfect consensus present; lower case abbreviation amino acid means imperfect <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           consensus amino acid present sequence. Other symbols <br />                                           indicate closely related amino acids thus: &#226;&#8364;&#339;!&#226;&#8364;? I V found; <br />                                           &#226;&#8364;&#339;$&#226;&#8364;? L M found; &#226;&#8364;&#339;%&#226;&#8364;? F Y found; &#226;&#8364;&#339;#&#226;&#8364;? N, D, Q, or <br />                                           E found. <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                            Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 3. <br />                                           Determination GlmU acetyl transferase activity. (A) The preparation tetra-O- <br />                                           perdeuterioacetyl-N-perdeuterioacetyl derivative starting GlcNH2-1-phosphate and <br />                                           tetra-O-perdeuterioacetyl-N-acetyl derivative product GlcNAc-1-phosphate. D6Ac2O = <br />                                           hexadeuterioacetic anhydride. (B) The M+H-63 ion (63 weight trideuterio acetic acid) <br />                                           detected GC/MS (CI mode) analysis derivatization reactants incubated buffer <br />                                           enzyme (control reaction). (C) The M+H-63 ion detected GC/MS (CI mode) analysis <br />                                           derivatization reactants incubated buffer crude M. smegmatis enzymes. (D) <br />                                           The M+H-63 ion detected GC/MS (CI mode) analysis derivatization reactants <br />                                           incubated buffer M. tuberculosis GlmU expressed E. coli. In C D starting <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                        Page 14 <br />  <br />  <br />                                           material fully converted product. The acetyl transferase activity incubations in <br />                                           200 &#194;&#181;l 10 mM Tris buffer (0.1 mM MgCl2 0.01% Triton X 100), pH 8 containing 100 <br />                                           &#194;&#181;M acetyl CoA glucosamine-1-phosphate. Either 100 ng purified M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tuberculosis GlmU (D), 0.5 &#194;&#181;g crude M. smegmatis extract (C), enzyme (B) was <br />                                           added reactions incubated 30 minutes room temperature derivatization. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 4. <br />                                           Analysis purified M. tuberculosis GlmU protein SDS-PAGE (A) Western blot <br />                                           (B). Lane 1. PageRuler prestained protein ladder (Fermentas, bottom: 170, 130, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           100, 70, 55, 40, 35, 25 kD); Lane 2. purified M. tuberculosis GlmU protein (with histidine tag) <br />                                           (5 &#194;&#181;g) expected molecular weight 54.10 kD. The (anti)-polyhistidine clone HIS-1 <br />                                           antibody 1 6000 dilution used Western blot analysis (B). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                      Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The uridylyl transferase activity GlmU enzyme assayed HPLC. (A) UDP-GlcNAc <br />                                           standard, (B) UDP-GlcNAc GlmU PPi, (C) UDP-GlcNAc GlmU with <br />                                           PPi. The enzyme reactions incubated 37&#194;&#176;C 30 minutes 40 &#194;&#181;l 50 mM Tris-HCl <br />                                           (pH 8.2) containing 1 mM MgCl2, 25 &#194;&#181;M UDP-GlcNAc, 1 mM pyrophosphate (PPi) (in C <br />                                           only), 42 &#194;&#181;g/ml GlmU protein. <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 6. <br />                                           Southern blot analysis M. smegmatis ZW-2 strains. <br />                                           (A) The expected DNA fragments M. smegmatis mc2155. (B) The expected DNA fragments <br />                                           M. smegmatis ZW-2. (C) Confirmation M. smegmatis ZW-2 strains Southern blot. <br />                                           Lanes 1&#226;&#8364;&#8220;6. M. smegmatis ZW-2 strains hybridized DNA bands 1.0 kb, 1.59 kb, <br />                                           1.79 kb; Lane 7. pZW-6 hybridized DNA bands 0.65 kb, 0.8 kb 5.3 kb; Lane <br />                                           8. M. smegmatis mc2155 wild type hybridized DNA bands 1.34 kb 1.79 kb. (D) <br />                                           The map plasmid pCG76-Mtb glmU (pZW-6) rescue plasmid. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                            Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 7. <br />                                           Growth curves M. smegmatis ZW-2 strain. <br />                                           (A) Growth curves M. smegmatis ZW-2 strain 30&#194;&#176;C 42&#194;&#176;C. M. smegmatis ZW-2 at <br />                                           30&#194;&#176;C (&#226;&#8211;&#178;); M. smegmatis ZW-2 42&#194;&#176;C (&#226;&#8211;&#161;). M. smegmatis mc2155 30&#194;&#176;C (&#226;&#8212;?); M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           smegmatis mc2155 42&#194;&#176;C (&#226;&#8212;&#8249;). M. smegmatis mc2155 carrying pZW-6 30&#194;&#176;C (&#226;&#8211;&#160;); M. <br />                                           smegmatis mc2155 carrying pZW-6 42&#194;&#176;C (&#226;&#8211;&#161;). (B) CFU growth curve ZW-2. The <br />                                           cells grown 30&#194;&#176;C 24 hours transferred 42&#194;&#176;C incubator. CFU and <br />                                           OD600 M. smegmatis ZW-2 grown 42&#194;&#176;C determined interval 24 hours. Log <br />                                           CFUs M. smegmatis ZW-2 (&#226;&#8211;&#178;); OD600 M. smegmatis mc2155 ZW-2 42&#194;&#176;C (&#226;&#8212;?). <br />                                           (C) Western blot GlmU showing loss temperature shift. The antibody against <br />                                           M. smegmatis GlmU cross reacts M. tuberculosis GlmU. Lanes are: 1. purified M. <br />                                           tuberculosis GlmU protein (with histidine tag) (54.10 kD); 2. PageRuler&#226;&#8222;&#162; Prestained Protein <br />                                           Ladder (Fermentas), MW protein visualized 55 kD; 3, 5, 7, 9. M. smegmatis ZW-2 <br />                                           grown 42&#194;&#176;C 1, 2, 3, 4 days shifting temperature 30&#194;&#176;C 42&#194;&#176;C; 4, 6, 8, 10. <br />                                           M. smegmatis ZW-2 grown 30&#194;&#176;C 1, 2, 3 4 days. The molecular weight M. <br />                                           tuberculosis GlmU expressed pCG76-Mtb glmU M. smegmatis ZW-2 50.29 kD. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                       Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 8. <br />                                           Scanning electron micrographs M. smegmatis ZW-2 strain. <br />                                           (A) M. smegmatis ZW-2 cells grown 30&#194;&#176;C 24 hours just transfer 42&#194;&#176;C (10 <br />                                           000X); (B&#226;&#8364;&#8220;E) M. smegmatis ZW-2 cells 42&#194;&#176;C 24; 48, 72 96 hours (10 000X); (F) <br />                                           M. smegmatis ZW-2 cells 42&#194;&#176;C 96 hours (the E) higher magnification (30 <br />                                           000X); (G) Wild type M. smegmatis control. Arrows (E) (F) point areas possible <br />                                           lysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                          Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 9. <br />                                           GlmU, MurA MurB 96 plate assay. <br />                                           The enzymatic activity M. tuberculosis GlmU determined 96 plate format <br />                                           utilizing MurA MurB, (both E. coli) NADPH. When enzymes present (&#226;&#8212;&#352;) <br />                                           A340 drops (average slope &#226;&#710;&#8217;0.0054, standard deviation 0.00013) loss NADPH. <br />                                           When substrates omitted reaction (&#206;&#8221;-no acetyl Coenzyme-A (average slope <br />                                           &#226;&#710;&#8217;0.0007, standard deviation 0.00033), &#226;&#8212;&#8249; - glucosamine-1-phosphate (average slope <br />                                           &#226;&#710;&#8217;0.0005, standard deviation 0.00023), &#195;&#8212; - phosphoenol pyruvate (average slope &#226;&#710;&#8217;0.0008, <br />                                           standard deviation 0.00018) &#226;&#8211;&#161; - UTP (average slope &#226;&#710;&#8217;0.0006, standard deviation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           0.00017)) A340 does decrease. Similar lack activity seen (data presented in <br />                                           graph) enzymes left (no GlmU (average slope &#226;&#710;&#8217;0.00094, <br />                                           standard deviation 0.000012), MurA (average slope &#226;&#710;&#8217;0.00043, standard deviation 0.00012) <br />                                           MurB (average slope &#226;&#710;&#8217;0.0006, standard deviation 0.00019). <br />  <br />  <br />  <br />  <br />                                   <br /> </body></html>