<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    FRQ-Interacting RNA Helicase Mediates Negative and Positive Feedback <br />                     in the Neurospora Circadian Clock <br />  <br />                                                                     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The Neurospora circadian oscillator comprises FREQUENCY (FRQ) and its transcription activator, the <br />              White Collar Complex (WCC). Repression of WCC&#226;&#8364;&#8482;s transcriptional activity by FRQ via negative feedback <br />              is indispensable for clock function. An unbiased genetic screen that targeted mutants with defects in negative <br />              feedback regulation yielded a fully viable arrhythmic strain bearing a novel allele of FRQ-interacting RNA <br />              helicase ( frh), an essential gene that encodes a putative exosome component protein. In the allele, frhR806H, <br />              clock function is completely disturbed, while roles of FRQ-interacting RNA helicase (FRH) essential for <br />              viability are left intact. FRHR806H still interacts with FRQ, but interaction between the FRQ&#226;&#8364;&#8220;FRHR806H complex <br />              (FFC) and WCC is severely affected. Phosphorylation of WC-1 is reduced in the mutant leading <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> constantly <br />              elevated WCC activity, which breaks the negative feedback loop. WCC levels are considerably reduced in the <br />              mutant, especially those of WC-1, consistent both with loss of positive feedback (FRQ-dependent WC-1 <br />              stabilization) and with a reduced level of the FRQ-mediated WCC phosphorylation that leads to high WCC <br />              activity accompanied by rapid transcription-associated turnover. FRH overexpression promotes WC-1 <br />              accumulation, confirming that FRH together with FRQ plays a role in WC-1 stabilization. Identification of a <br />              viable allele of frh, displaying virtually complete loss of both negative and positive circadian feedback, <br />              positions FRH as a core component of the central oscillator that is permissive for rhythmicity but appears not <br />              to modulate periodicity. Moreover, the results suggest that there are clock-specific roles for FRH that are <br />              distinct <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the predicted essential exosome-associated functions for the protein. <br />  <br />  <br />  <br />  <br /> C     IRCADIAN clocks are an evolutionarily conserved <br />         means through which metabolic, physiological, and <br /> developmental processes are controlled (Bell-Pedersen <br />                                                                             activators (positive elements), a heterodimer of pro- <br />                                                                             teins interacting via PAS domains (Young and Kay <br />                                                                             2001). The inhibition combined with delayed turnover <br /> et al. 2005; Gatfield and Schibler 2007; Brunner and                        of the negative elements leads to oscillatory transcrip- <br /> Kaldi 2008). A circa 24-hr clock allows organisms to                        tion factor activity. Since those positive elements also <br /> predict day&#226;&#8364;&#8220;night changes thus providing them bene-                         control expression of other genes (named clock-controlled <br /> fits for <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span> (Ouyang et al. 1998; Dodd et al. 2005).                   genes, ccgs), the negative feedback loop gives rise to <br /> Our understanding of the circadian clock has advanced                       rhythmic expression of many ccgs that are involved in <br /> dramatically since clock genes period and frq were cloned                   processes of metabolism, development, stress responses, <br /> in Drosophila and Neurospora, respectively (Young                           etc. <br /> and Kay 2001; Dunlap 2008). Since then, research on                            In the Neurospora clock, components of the FRE- <br /> many model organisms has revealed a conserved aspect                        QUENCY (FRQ)&#226;&#8364;&#8220;White Collar Complex (WCC)-based <br /> of regulation in which negative feedback plays an                           oscillator have been identified by forward genetics, and <br /> essential role in maintaining a self-sustained circadian                    negative feedback was proven by showing that FRQ <br /> clock (Dunlap 1999; Heintzen and Liu 2007; Mackey                           overexpression would stop the clock (McClung et al. <br /> 2007). The circadian oscillator in fungi and animals, for                   1989; Aronson et al. 1994). Subsequent identification <br /> instance, comprises a negative element(s) whose gene                        of the heteromeric WCC as the activator of frq expres- <br /> product(s) inhibits the activity of its own transcriptional                 sion provided the minimal set of elements for the <br />                                                                             feedback loop, and the model of the core oscillator as <br />                                                                             a single negative feedback loop (Crosthwaite et al. <br />    Supporting information is available online at http://www.genetics.org/ <br /> cgi/content/full/genetics.109.111393/DC1.                                   1997). Later work established that FRQ actually binds <br />    1 <br />     Present address: Department of Neuroscience, Howard Hughes Medical      to and inhibits the activity of its own transcription <br /> Institution, University of Pennsylvania, Philadelphia, PA 19104.            activators, the WCC (Cheng et al. 2001a; Denault <br />    2 <br />     Present address: Fonterra Research Centre, Private Bag 11029, Palmer-   et al. 2001; Froehlich et al. 2003), and this negative <br /> ston North 4442, New Zealand. <br />    3 <br />                                                                             feedback gives rise to rhythmic expression of frq and <br />     Corresponding author: 7400 Remsen, Department of Genetics, Dart- <br /> mouth Medical School, Hanover, NH 03755.                                    other ccgs (Correa et al. 2003). In the current view of <br /> E-mail: Jay.C.Dunlap@dartmouth.edu                                          the feedback loop, newly synthesized FRQ enters the <br /> Genetics 184: 351&#226;&#8364;&#8220;361 (February 2010) <br />  352                                                              M. Shi, et al. <br />  <br /> nucleus, where it brings casein kinase (CK I) and CK                       integration via nonhomologous recombination (Ninomiya et al. <br /> II to phosphorylate the WCC and inhibits its activity                      2004; Colot et al. 2006). The transformants were backcrossed to <br />                                                                            WT to obtain homokaryon knock-in strains. <br /> (Schafmeier et al. 2005; He et al. 2006). In the process, <br />                                                                               Media recipes and Neurospora manipulation methods <br /> appreciable levels of the FRQ&#226;&#8364;&#8220;WCC complex are seen in                      are adapted from those available at the Fugal Genetics <br /> the nucleus, and the feedback loop is finally closed as                    Stock Center (FGSC) (http://www.fgsc.net/Neurospora/ <br /> FRQ-mediated phosphorylation of the WCC impedes                            NeurosporaProtocolGuide.htm and http://www.fgsc.net/ <br /> DNA binding, WCC is stabilized (Schafmeier et al.                          methods/stanford.html). The race tube assay has been de- <br /> 2008), and the complex is exported from the nucleus                        scribed elsewhere (Aronson et al. 1994). The race tube <br />                                                                            medium contained 13 Vogel&#226;&#8364;&#8482;s salts, 0.1% glucose, 0.17% <br /> (Hong et al. 2008).                                                        arginine, 50 ng/ml biotin, and 1.5% Bacto-agar. The regular <br />    An important addition to the understanding of the                       liquid culture medium contained 13 Vogel&#226;&#8364;&#8482;s salts, 2% glucose, <br /> cycle came when a putative RNA helicase found to                           0.5% arginine, and 100 ng/ml biotin. The quinic acid (QA) <br /> copurify with FRQ was shown to be essential for the                        induction liquid culture medium contained 13 Vogel&#226;&#8364;&#8482;s salts, <br /> clock (Cheng et al. 2005). The FRQ-interacting RNA                         0.1% glucose, 0.5% arginine, 100 ng/ml biotin, and 0.01 m <br />                                                                            QA. Race tubes were analyzed as previously described (Shi <br /> helicase (FRH), encoded by an essential gene, frh, shows                   et al. 2007). <br /> significant homology to mtr4p, a cofactor of the Sac-                         Genetic screen for negative feedback loss mutants: A liquid <br /> charomyces exosome complex involved in mRNA ex-                            suspension of conidia from strain 197-9 (2 3 107/ml) was <br /> port and degradation (Mitchell and Tollervey 2000).                        treated with 100 mm EMS in 0.067 m potassium phosphate <br /> Recently it was shown that FRH interacts, albeit weakly,                   buffer (pH 7.0) for 5 hr. The treated conidia, which displayed <br />                                                                            an  25% survival rate, were washed three times with buffer <br /> with exosome components and strains with reduced                           and diluted 10-fold in the final wash. Conidial suspension <br /> FRH have reduced frq mRNA stability (Guo et al. 2009).                     ( 107 cells) was plated onto 13 Vogel&#226;&#8364;&#8482;s plus 13 FIGS plates <br />    To further understand the mechanism of negative                         with 400 mg/ml hygromycin. The plates were cultured in <br /> feedback, we utilized a genetic screen aimed at uncov-                     constant dark and in total, 83 surviving colonies were picked <br /> ering components required for feedback. A strain,                          after 7 days of culture. <br />                                                                               DNA and RNA analyses: Neurospora genomic DNA was <br /> Mut10, was isolated, which showed essentially normal                       prepared from liquid cultures using the Puregene DNA <br /> growth and viability but an arrhythmic <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span>.                          isolation kit from Gentra Systems. Neurospora RNA was <br /> Genetic mapping revealed a chromosomal transloca-                          prepared as previously described (Aronson et al. 1994). In <br /> tion in the strain tightly linked to a <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>missense mutation</span>                   Southern analysis, 0.5 mg restriction digested genomic DNA <br /> (R806H) in the frh locus. Those two mutations were                         was resolved on a 1% agarose gel and the digoxigenin (DIG) <br />                                                                            system from Roche was used for probe labeling and detection. <br /> separated and the clock phenotype was shown to be due                      In Northern analysis, 10 mg total RNA was resolved on a 1.2% <br /> to the R806H change in FRH, which leads to a weakened                      agarose gel containing 2% formaldehyde, and the DIG system <br /> interaction between FRQ and the WCC and a defect in                        was used for probing and detection. Allele-specific PCR was <br /> WCC phosphorylation. While confirming involvement                          used to differentiate SNP genotypes of mapping progeny <br /> of FRH in negative feedback, this result strongly sug-                     or frh alleles of mutant and knock-in strains. Allele specific <br />                                                                            DNA oligos were designed with the online SNAPER program <br /> gests that the essential functions of FRH do not con-                      (http://ausubellab.mgh.harvard.edu/). Oligo MS266 (AAGC <br /> stitute important elements of the clock cycle, since the                   CATTGTTATCGCCGTTGA) and MS254 (GAACGATGTTC <br /> frh mutant strain shows normal growth and develop-                         GGCTGTAGGTA) flanked the translocation site and the <br /> ment while having completely lost circadian regulation.                    PCR product was used in Southern analysis for chromosomal <br /> Interestingly, WC-1 and WC-2 are both downregulated                        translocation. <br />                                                                               Antiserum generation and protein analysis: FRQ, WC-1, and <br /> in the mutant strain, suggesting that FRH, like FRQ, <br />                                                                            WC-2 antibodies have been described (Garceau et al. 1997; <br /> participates in the interlocked positive feedback loop                     Lee et al. 2000; Denault et al. 2001). Preparation of FRH <br /> that maintains WCC stability (Lee et al. 2000; Cheng                       antibody was adapted from Cheng et al. (2005). Western <br /> et al. 2001b; Schafmeier et al. 2006, 2008).                               analysis, immunoprecipitation, and phosphatase treatment <br />                                                                            were performed as previously described (Garceau et al. 1997). <br />                                                                               SNP marker identification and SNP mapping: Conidia of a <br />                 MATERIALS AND METHODS                                      Mauriceville isolate (FGSC no. 2225) were cultured at 30&#194;&#176; in <br />                                                                            constant light (LL) for 4 hr. A cDNA library was made from the <br />    Plasmids, strains, media, and Neurospora methods: All                   extracted mRNA using a Stratagene Uni-Zap kit. The cDNA <br /> strains used in this research included in their genetic back-              clones were sequenced and SNP markers were identified by <br /> ground the ras-1bd allele that serves to clarify the clock-controlled      comparing the Mauriceville sequences with the genomic <br /> conidiation <span id='am-473' about='xsp:pattern' typeof='owl:Thing'>pattern</span> in race tube assays (Belden et al. 2007).              sequence of the reference Oak Ridge isolate (Lambreghts <br /> Because ras-1bd does not affect the rhythm but only clarifies its          et al. 2009). SNP6 and SNP72 on LGI were named after the <br /> expression, strains bearing only ras-1bd are referred to here as           contig numbers where they are localized (http://www.broad. <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;wild type&#226;&#8364;&#8482;&#226;&#8364;&#8482; (WT). A construct pYL51A was made by Y. Liu,                 mit.edu/annotation/genome/neurospora/Home.html) and <br /> containing the hph gene with frq 39-UTR under control of the frq           selected for mapping purposes. Additional SNP markers <br /> promoter and the frq ORF under control of the qa-2 promoter.               between those two markers were found by sequencing part <br /> pYL51A was targeted to the his-3 locus and the strain containing           of the Mauriceville genomic DNA in this region. Progeny from a <br /> the reporter gene, 197-9 [ras-1bd, his-3T(frqp-hph-frq39UTR, qa-2p-        cross between WT in Mauriceville and Mut10 in Oak Ridge were <br /> frq)], was subjected to EMS mutagenesis as described below. The            subject to SNP mapping. The rhythmicity phenotype was <br /> knock-in (KI) constructs frhR806H-KI, qa-frh, qa-wc-1, and qa-wc-2         scored using the race tube assay and the SNP genotype was <br /> were transformed into the ras-1bd; mus-52 strain to avoid ectopic          scored by allele-specific PCR. <br />                                         Roles of FRH in the Neurospora Circadian Clock                                        353 <br />  <br />    Sequence analysis of the chromosome translocation: KpnI <br /> digests of Mut10 genomic DNA were self-ligated and subjected <br /> to inverse PCR with DNA oligos near the mapping locus on <br /> LGI. The PCR products were cloned and sequenced to obtain <br /> the region of chromosome translocation. <br />    Forced heterokaryon assay: Mut10 was individually crossed <br /> with inl and pan-2. Progeny were screened for Mut10; inl and <br /> Mut10; pan-2 strains. A forced heterokaryon strain of Mut10 <br /> and WT was obtained by culturing Mut10; inl and pan-2 <br /> together on minimal medium, as well as the reciprocal <br /> heterokaryon of Mut10; pan-2 and inl. <br />    Real-time PCR analyses: cDNA samples were prepared with <br /> the Invitrogen Superscript III RT reaction kit. Real-time PCR <br /> analyses were performed using the power SYBR Green system <br /> from ABI. <br />  <br />  <br />                          RESULTS <br />  <br />    A novel genetic screen for negative feedback <br /> mutants: The concept underlying circadian negative <br /> feedback is that the protein product of the frq locus acts <br /> to turn down its own expression. Therefore, to develop <br /> a screen for mutations affecting negative feedback, <br /> we engineered a strain bearing (1) a selectable marker             Figure 1.&#226;&#8364;&#8221;A genetic screen for mutants defective in circa- <br /> under control of the frq promoter, and (2) a means              dian negative feedback. (A) Scheme of the genetic screen. Ex- <br /> through which FRQ could be overexpressed. In this               pression of the selectable marker, hph, is under control of the <br />                                                                 frq promoter ( frqP); therefore it is repressed by negative feed- <br /> strain, when FRQ is overexpressed, the selectable               back in the dark via FRQ inhibiting the WCC transcriptional <br /> marker should be turned off unless the feedback loop            activity. Mutants with defects in negative feedback were ob- <br /> connecting FRQ to repression is blocked by <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span>.            tained after EMS mutagenesis through selection for resistance <br /> Specifically, we constructed strain 197-9 [ras-1bd, his-        to hygromycin. Overexpression of FRQ by QA induction <br /> 3T(frqp-hph-frq39UTR, qa-2p-frq)], which contains (1)           driven through the qa-2 promoter (qa-2P) was subsequently <br />                                                                 used to confirm that there was a break in the negative feed- <br /> the selectable marker gene hph under control of the             back loop connecting FRQ to the WCC. (B) Race tube assay <br /> frq promoter and thereby activated by the transcription         with hygromycin selection. The mycelial growth front was <br /> factor WCC and (2) the frq ORF under control of the QA          marked with a thin black line each day. The thick solid black <br /> inducible qa-2 promoter (Figure 1A). Expression of hph          lines on race tubes represent light (LL) to dark (DD) transfer. <br /> encodes hygromycin phosphotransferase, which con-               The dots mark growth fronts 1 day after LD transfer. The orig- <br />                                                                 inal strain 197-9 that was subjected to EMS mutagenesis grew <br /> fers cell resistance to hygromycin. Since FRQ represses         normally in light, reflecting light-induced WCC activity that <br /> WCC activity in constant darkness (DD), growth of               activates the frqP. After LD transfer, the growth rate decreased <br /> the strain was inhibited in darkness under hygromycin           dramatically, verifying that the WCC activity was inhibited by <br /> selection. However, since frq is expressed in the light         FRQ. In comparison with 197-9, the growth rate of Mut10 was <br /> regardless of the degree of FRQ overexpression                  faster in DD, suggesting that negative feedback was severely <br />                                                                 affected. Consistent with this, QA-induced FRQ could not re- <br /> (Crosthwaite et al. 1995), the strain was hyg resistant         store the loss of negative feedback in Mut10, confirming that <br /> in the light. After EMS mutagenesis, survivors were             the strain contains a bona fide feedback mutation not at the frq <br /> plated and kept in the dark under hygromycin, which             locus. (C) Race tube assay in free running conditions. WT and <br /> killed most of cells. Out of  107 mutated conidia, 83           Mut10 sibling strains were cultured in light and their clocks <br /> mutants grew on hygromycin, therefore signaling a po-           were assayed after transfer to DD. Mut10 is arrhythmic under <br />                                                                 free running conditions, although its growth rate is compara- <br /> tential loss of negative feedback. Among those mutant           ble to the WT sibling strain. Period <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> is shown from a re- <br /> strains, 21 were arrhythmic, 6 showed period lengths            gression analysis of n &#194;&#188; 6 bands with a standard deviation <br /> longer than WT, and 13 showed shorter period lengths,           (SD). AR, arrhythmic. <br /> and the remaining mutants showed no obvious defects <br /> in clock oscillation, suggesting that they developed            ras-1bd; a). The frq7 allele confers a long period length <br /> alternative means of resistance. These data are consis-         ( 29 hr) and un-10, which is tightly linked to frq, is <br /> tent with the concept that negative feedback is essential       temperature sensitive for growth. So in these crosses, <br /> for the circadian clock and that defects in this regula-        novel alleles of frq arising from the EMS mutagenesis <br /> tion might lead to clock phenotypes.                            and screen would fail to segregate progeny with wild- <br />    Since FRQ is a negative element in the oscillator            type rhythms and virtually all un-10 strains would have a <br /> (Aronson et al. 1994), we assumed that some mutants             29-hr period length. In this way, 6 of the 21 arrhythmic <br /> represented frq alleles. To identify the frq mutants,           mutants were localized to frq, and the recessive nature of <br /> arrhythmic strains were crossed with 732-1 (frq7, un-10;        these mutations was confirmed by a failure to grow on <br />  354                                                     M. Shi, et al. <br />  <br /> plates containing hygromycin and QA. These novel frq <br /> alleles were set aside and attention focused on the <br /> mutations identifying potential new clock genes. <br />    A <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>novel mutation</span> conferring a loss of negative <br /> feedback and an arrhythmic phenotype: Among the <br /> strains corresponding to the remaining 15 arrhythmic <br /> mutants was Mut10. On race tubes with hygromycin <br /> selection, both Mut10 and the control strain (197-9) <br /> grew normally in light, confirming that WCC activity is <br /> not repressed by FRQ in light (Crosthwaite et al. <br /> 1995). After L-D (light to dark) transfer, growth of 197-9 <br /> dropped dramatically, reflecting the repression of WCC <br /> activity by negative feedback. In comparison, the growth <br /> rate of Mut10 slowed but then rebounded, suggesting <br /> that negative feedback is severely affected. In this strain, <br /> overexpression of FRQ by QA could not restore negative <br /> feedback, confirming that Mut10 is not a frq allele but a <br /> bona fide negative feedback mutant (Figure 1B). <br />    The Mut10 strain is profoundly arrhythmic on race <br /> tubes under free running conditions in the dark with <br /> regular medium, suggesting that the loss of negative <br /> feedback indeed disrupts clock function (Figure 1C). <br /> While the conidiation rhythm as seen on race tubes is an <br /> output of the circadian clock, it can mask the underlying <br /> clock rhythm in certain conditions (Shi et al. 2007). To <br /> confirm the circadian phenotype at the molecular level, <br /> the time course of frq mRNA expression was followed in <br /> the mutant strain with Northern analysis. Consistent <br /> with previous reports (Aronson et al. 1994), frq mRNA                Figure 2.&#226;&#8364;&#8221;Loss of negative feedback in Mut10. (A) North- <br /> in the WT strain oscillated in a circadian manner with            ern analysis showing rhythmic frq expression in the molecular <br /> two peaks around time points DD12 and DD36 (Figure                clock. RNA samples collected over 2 days from a WT strain <br /> 2A). In the Mut10 strain, overall frq message level was           (top) were subjected to a Northern analysis probing with <br /> distinctly elevated compared with WT and the frq expres-          frq. A normal circadian frq mRNA oscillation was observed <br />                                                                   with two peaks around time DD12 (12-hr culture in dark) <br /> sion pattern showed a random fluctuation presumably               and DD36, respectively. In Mut10 (bottom), frq mRNA was up- <br /> reflecting the loss of negative feedback regulation               regulated and displayed a random fluctuation representing <br /> (Figure 2A). As expected (Garceau et al. 1997), FRQ               unregulated and noncircadian WCC activity due to the loss <br /> expression and protein phosphorylation oscillated in              of negative feedback. (B) Western analysis of FRQ. Samples <br /> the WT strain (Figure 2B). In comparison, FRQ ex-                 were collected every 4 hr from cultures held in constant dark- <br />                                                                   ness (DD) as in A, and subjected to Western analysis with <br /> pression in Mut10 was constitutively high with all                anti-FRQ antiserum. A WT strain (top) showed a circadian os- <br /> phospho-isoforms present at all time points (Figure               cillation of FRQ protein level and FRQ phosphorylation. In <br /> 2B), as if there were overexpression of FRQ resulting             comparison, Mut10 (bottom), having lost negative feedback, <br /> from abundant frq transcription, but the newly synthe-            showed a constitutively high level of FRQ and the presence of <br /> sized FRQ could not shut down frq transcription                   all phospho-isoforms at all time points. Asterisks mark a non- <br />                                                                   specific band. <br /> effectively due to defects in negative feedback. <br />    Single nucleotide polymorphism mapping of Mut10 <br /> identified a chromosomal translocation and a missense             SNP6 and SNP72 near centromere I (CENI) (Figure <br /> mutation in frh: Initial genetic mapping revealed little          3A). Unexpectedly however, 2 of 10 recombinant <br /> segregation between the arrhythmic phenotype of                   progeny showed recombination with both SNP markers, <br /> Mut10 and two genetic markers on linkage group I                  suggesting either an exceptional frequency of double <br /> (LGI) near the centromere, mat (10.9 cM) and his-3 (7.3           recombination, gene conversion, or the presence of <br /> cM) (Figure 3A). SNP mapping (Dunlap et al. 2007;                 a chromosomal rearrangement tightly linked to the <br /> Lambreghts et al. 2009) using Mut10 (in the Oak Ridge             mutation conferring arrhythmicity. To resolve this, <br /> background) and WT (in the Mauriceville geographical              additional SNP markers were identified by sequencing <br /> variant background of Neurospora) (Metzenberg et al.              Mauriceville genomic DNA in the area between SNP6 <br /> 1985) was used to achieve a finer genetic resolution.             and SNP72 and were scored among those recombinant <br /> Consistent with the initial mapping data, the mutation            progeny. Using these additional markers, Mut10 was <br /> was mapped to a region of  5 MU between markers                   mapped in a 20-kb region; the two &#226;&#8364;&#732;&#226;&#8364;&#732;aberrant&#226;&#8364;&#8482;&#226;&#8364;&#8482; progeny <br />                                           Roles of FRH in the Neurospora Circadian Clock                                    355 <br />  <br />                                                                     also showed recombination with all newly identified <br />                                                                     markers, supporting the idea of a chromosomal trans- <br />                                                                     location tightly associated with arrhythmicity. To con- <br />                                                                     firm this and to localize the breakpoint on LGI, primer <br />                                                                     pairs were chosen in the 20-kb region from WT DNA and <br />                                                                     used for PCR using Mut10 genomic DNA as a template. <br />                                                                     All pairs except for one yielded the expected products, <br />                                                                     and the one that did not yield any product indicated the <br />                                                                     site of a chromosomal translocation breakpoint, later <br />                                                                     confirmed by Southern blot (Figure 3B). <br />                                                                        The translocation junction fragment was cloned <br />                                                                     using self-ligation of mutant genomic DNA digest and <br />                                                                     inverse PCR. Sequencing analysis identified a reciprocal <br />                                                                     chromosomal translocation between LGI L and LGII L <br />                                                                     (Figure 3, B and C; supporting information, Figure S1 <br />                                                                     A). We hypothesized that arrhythmicity could derive <br />                                                                     from effects of the translocation on nearby gene(s), but <br />                                                                     knockouts (Colot et al. 2006) of 10 ORFs on LGI near <br />                                                                     the junction showed no clock phenotype (data not <br />                                                                     shown). However, ORFs on LGII in proximity to the <br />                                                                     breakpoint included frh only about 100 kb away (,5 cM) <br />                                                                     from the junction site, and sequencing of Mut10 DNA <br />                                                                     identified a G-to-A transition leading to an arginine-to- <br />                                                                     histidine missense mutation as FRHR806H (Figure 3D). <br />                                                                     FRH is highly homologous to mtr4p, a component <br />                                                                     of the Saccharomyces exosomal complex. The frh gene <br />                                                                     is essential, and even knockdowns are barely viable <br />                                                                     (Cheng et al. 2005), in contrast to the Mut10 strains, <br />                                                                     which are fully viable and show robust growth. <br />                                                                        Identification of this mutation in frh suggested that <br />                                                                     the translocation, although tightly linked, was irrelevant <br />                                                                     to the clock defect, so SNP markers were used to score <br />    Figure 3.&#226;&#8364;&#8221;Genetic mapping of Mut10 reveals a transloca-          additional progeny of a Mut10 with WT cross to identify <br /> tion with a breakpoint linked to frh. (A) Genetic mapping an-       strains bearing each mutation separately (Figure S1, A <br /> alyzing progeny from a cross between Mut10 and WT showed <br />                                                                     and B). The strain with only the chromosomal trans- <br /> linkage to traditional markers (mat and his-3) and newly identi- <br /> fied SNP markers near the centromere region on linkage group        location between LGI L and LGII L displayed a wild-type <br /> I (LGI). The genetic distance is reported as the number of re-      clock phenotype and a normal growth rate (data not <br /> combinants over the number of total progeny analyzed. SNP           shown). This strain designated as T(IL;IIL)JD10 may <br /> markers were named after their contig numbers in Neurospora         serve as a useful tool for genetic study in Neurospora <br /> genome (release 7) (http://www.broad.mit.edu/annotation/ <br />                                                                     (Perkins 1997). FRH expression appears normal in <br /> genome/neurospora). (B) Chromosomal translocation in <br /> Mut10. A diagram of the chromosomes in WT and Mut10 is              frhR806H (Figure S1 C), suggesting the mutation does not <br /> shown (left). The red solid line and the blue dashed line rep-      affect gene expression level or protein stability, but only <br /> resent chromosomes of LGI and LGII, respectively, from WT           the function of the protein. <br /> (top left) and Mut10 (bottom left). The thick black bar indi-          Codominance of frhR806H with respect to rhythm <br /> cates a fragment amplified by oligos MS266 and MS254 (see <br />                                                                     expression: Heterokaryons (HETs) were generated be- <br /> materials and methods) from WT genomic DNA. The frag- <br /> ment flanks a BglII site (arrowhead) and was used as a probe in     tween Mut10 and WT strains using auxotrophic markers <br /> the Southern analysis (right); other BglII sites are also marked    pan-2 and inl. All heterokaryon strains show weak <br /> by arrows. The breakpoint of the chromosomal translocation is       conidiation rhythms with approximately normal period <br /> within the probe region, therefore the probe hybridizes to DNA      lengths, an intermediate phenotype between Mut10 <br /> from two chromosomes in Mut10, resulting in a different diges- <br />                                                                     and WT (Figure 4A), indicating that frhR806H is codom- <br /> tion pattern from WT as shown in the Southern analysis. See <br /> text and Figure S1 for details. (C) The translocation junction      inant to WT. Given this result, and to confirm that <br /> fragment was cloned by inverse PCR and sequencing of the            no additional mutations in the Mut10 background <br /> junction region identified the exact translocation point be-        contributed to the arrhythmic phenotypes, a pheno- <br /> tween LGI (sequence in red) and LGII (sequence in blue).            copy approach was used. A construct bearing the frhR806H <br /> A thymidine nucleotide was lost as a result of the translocation <br />                                                                     fragment linked to the bar gene (which confers re- <br /> event. (D) Sequencing of the frh locus in Mut10 revealed a G-to-A <br /> mutation leading to an arginine-to-histidine change (R806H).        sistance to ignite) was targeted by transformation to the <br />                                                                     frh locus in WT (Figure S2 A). The transformants were <br />  356                                                            M. Shi, et al. <br />  <br />                                                                          frhR806H while FRQ is overwhelmingly abundant sug- <br />                                                                          gested that the positive loop is disrupted in the mutant <br />                                                                          as well. To evaluate this, we examined the expression of <br />                                                                          the wc genes in more detail. The reduced level of WC-1 <br />                                                                          and WC-2 seen in frhR806H could be due to reduced <br />                                                                          transcript levels and/or reduced protein stability. Tran- <br />                                                                          script levels were followed by RT&#226;&#8364;&#8220;PCR, revealing that <br />                                                                          both wc-1 and wc-2 mRNA levels were mildly reduced in <br />                                                                          frhR806H (Figure 5C). The reduction in wc-2 mRNA levels <br />    Figure 4.&#226;&#8364;&#8221;frhR806H knock-in phenocopies Mut10. (A) Anal-              (approximately two- to threefold) was similar to the <br /> ysis of forced heterokaryons between WT and Mut10. HET1                  change in WC-2 protein and may explain the reduction <br /> (heterokaryon of Mut10; pan-2 and inl) and HET2 (hetero-                 in the WC-2 levels. However, the approximately twofold <br /> karyon of Mut10; inl and pan-2) showed weak rhythmic conidia-            reduction in wc-1 mRNA level did not seem sufficient to <br /> tion on race tubes with a strong conidiation background. The <br />                                                                          account for the six- to eightfold reduction in the WC-1 <br /> phenotype is between wild type and Mut10, suggesting that <br /> Mut10 is codominant with WT. Period is shown by regression               levels. Given this, another possibility is that FRH <br /> analysis of n &#194;&#188; 6&#226;&#8364;&#8220;7 conidial bands on the race tube. (B) Race            influences the stability of the WCC. <br /> tube assay of frh knock-in strains (see Figure S2 for details). The          To assess stability, wc-1 and wc-2 were placed under <br /> frhWT-KI (SM52) strain, bearing a normal copy of frh with bar as a       control of the qa-2 promoter and overexpressed in the <br /> selectable marker, showed a normal circadian rhythm in free <br />                                                                          frhR806H background (Figure S3 A and Figure 6A). Sub- <br /> running conditions. In comparison, two separate frhR806H-KI <br /> strains (SM53 and SM54) showed arrhythmicity, which phe-                 sequent transfer to low QA and high glucose medium <br /> nocopied Mut10 and frhR806H.                                             repressed additional WC synthesis so WC-1 and WC-2 <br />                                                                          stability could be examined. WC-2 protein was stable <br />                                                                          in the frhR806H background (data not shown), consistent <br /> crossed with WT to obtain homokaryotic KI progeny.                       with the possibility that downregulation of WC-2 in the <br /> Genotyping of the frh allele in the knock-in strains was                 mutant was at the transcriptional level. In contrast, WC- <br /> achieved by Southern analysis and allelic-specific PCR                   1 protein was markedly less stable in frhR806H in compar- <br /> (Figure S2, B and C). The control strain in which a WT                   ison with frhWT, suggesting that FRH also regulates WC-1 <br /> sequence copy of frh linked with bar was knocked into                    in a post-translational manner (Figure 6B). Recent data <br /> the locus, frhWT-KI, showed rhythmic conidiation on a                    suggest that FRQ stabilizes WC-1 by promoting its in- <br /> race tube assay. In comparison, frhR806H-KI strains were                 activation via phosphorylation (Schafmeier et al. 2008), <br /> arrhythmic (Figure 4B). These data confirm that the                      and indeed Western analysis shows WC-1 to be more <br /> FRHR806H missense mutation breaks the clock, probably                    hypophosphorylated in the Mut10 strain (Figure 5, A <br /> by effectively competing with WT FRH, without adversely                  and B). Since WC-1 levels are low in the mutant strains, <br /> affecting cell viability or growth. Given the myriad es-                 we induced WC-1 in WT and frhR806H respectively to <br /> sential functions of the Saccharomyces homolog of                        compare phosphorylation (Figure 6C). WC-1 becomes <br /> FRH, including mRNA processing, transport and stabi-                     hyperphosphorylated in WT apparently due to FRQ- <br /> lization, the identification of an allele that specifically              dependent phosphorylation (Schafmeier et al. 2005) <br /> abrogates the clock function without impairing essen-                    whereas WC-1 is hypophosphorylated in frhR806H (Figure <br /> tial functions is noteworthy.                                            6C). Therefore, an interpretation consistent with the <br />    Loss of positive feedback in Mut10 suggests that                      data is that FRHR806H blocks or fails to promote this WC-1 <br /> FRH regulates WC-1 post-translationally: As a part of                    phosphorylation, and the resulting hyperactive and <br /> the circadian negative feedback loop, FRQ inhibits DNA                   hypophosphorylated WC-1 is unstable (Schafmeier <br /> binding by the WCC by mediating WCC phosphoryla-                         et al. 2008). <br /> tion (Froehlich et al. 2003; Schafmeier et al. 2005).                        These results from mutant strains suggest that FRH <br /> As a result of the loss of negative feedback, WC-1 and                   plays its role in positive feedback through effects on WC- <br /> possibly WC-2 were hypophosphorylated in Mut10 and                       1 regulation. To confirm this hypothesis, we constructed <br /> frhR806H strains (Figure 5, A and B and Figure S1 C). This               qa-frh strains (Figure S4, A and B) in which FRH dosage <br /> led to an increase of WCC activity and the dramatic                      is controlled by the inducer QA (Aronson et al. 1994). <br /> increase of frq mRNA and FRQ expression (Figure 2).                      Western analyses showed that FRH can be well induced <br /> Intriguingly however, WC-1 and WC-2 expression levels                    by QA, and induction of FRH increased both FRQ and <br /> were very low in frhR806H, while FRQ levels were extremely               WC-1 levels (Figure 6D). All FRQ protein is bound by <br /> high (Figure 5 and Figure S1 C). This result seems at                    FRH, and FRH stabilizes FRQ (Cheng et al. 2005). FRH <br /> first paradoxical given the other role of FRQ in the post-               stabilizes WC-1 by promoting FRQ-dependent WC-1 <br /> transcriptional positive feedback loop in which, when                    phosphorylation. In summary, these results suggest that <br /> FRQ is induced, WC-1 is upregulated within hours (Lee                    it is most likely the interaction of both FRQ and FRH as <br /> et al. 2000; Cheng et al. 2001b; Schafmeier et al. 2006).                the FFC with the WCC that is essential to promote WC-1 <br /> The failure to induce or maintain WCC expression in                      phosphorylation and stability. In this model FRH, the <br />                                     Roles of FRH in the Neurospora Circadian Clock                                      357 <br />  <br /> constant partner of FRQ, assists FRQ both in repressing     (Cheng et al. 2005) (Figure S1 C), indicating that <br /> WCC activity and in maintaining WC-1 stability, and         rhythmic production of FRH is not a part of the clock <br /> FRH, as a component in the oscillator, has dual roles in    cycle. Given this, the influence of FRH dosage on period <br /> both negative and positive feedback.                        length could be assessed in a straightforward manner <br />    Although the level of FRQ is under circadian regula-     by using qa-frh strains. At low QA concentration, qa-frh <br /> tion, the amount of FRH was invariant through time          yielded low FRH and showed a slow growth rate with <br />                                                             arrhythmicity on race tubes. This is consistent with a <br />                                                             previous study using RNAi to reduce frh expression <br />                                                             (Cheng et al. 2005), and confirms that FRH plays es- <br />                                                             sential roles in development and in the clock. However, <br />                                                             the change from arrhythmicity to rhythmicity is abrupt; <br />                                                             when FRH levels pass a certain threshold, negative feed- <br />                                                             back is recovered and the clock operates with a normal <br />                                                             period (Figure S4 C). This result suggests that FRH func- <br />                                                             tions more like a permissive &#226;&#8364;&#732;&#226;&#8364;&#732;switch&#226;&#8364;&#8482;&#226;&#8364;&#8482; for rhythmicity <br />                                                             than a period &#226;&#8364;&#732;&#226;&#8364;&#732;fine tuner.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                FRHR806H affects interactions among FRQ, FRH, and <br />                                                             the WCC: FRHR806H is associated with reduced WC-1 <br />                                                             phosphorylation, and yet Q-PCR analysis confirmed that <br />                                                             there is no difference in CKI or CKII levels between WT <br />                                                             and frhR806H (data not shown). Given this, two hypotheses <br />                                                             to explain the nature of the defect in FRHR806H are that <br />                                                             the mutation might affect the interaction of FRH with <br />                                                             FRQ or the FRH/FRQ complex with the WCC, in either <br />                                                             case breaking the feedback loop. The first of these is not <br />                                                             the case: Co-immunoprecipitation using a-FRH clearly <br />                                                             showed that FRHR806H still forms a complex with FRQ <br />                                                             (Figure 7A), a finding that is consistent with codomi- <br />                                                             nance of the mutant. In contrast to this, the interaction <br />                                                             between FRQ-FRHR806H and the WCC is disrupted in <br />                                                             Mut10 as indicated by reciprocal co-IP data using a-FRQ <br />                                                             or a-WC-2 in which FRQ no longer associates with WC-2 <br />                                                             in the Mut10 background (Figure 7B). Because a caveat <br />                                                             of the co-IP analysis is that WCC level is low in Mut10, we <br />  <br />  <br />  <br />                                                                Figure 5.&#226;&#8364;&#8221;Loss of positive feedback in Mut10 and frhR806H. <br />                                                             (A) Circadian time course samples of WT and Mut10 were <br />                                                             subjected to Western analysis and blotted with FRQ, WC-1, <br />                                                             and WC-2 antisera. The progeny 248-9 and 353-9 are from <br />                                                             two successive generations of backcrosses between Mut10 <br />                                                             and WT; both strains are arrhythmic. While FRQ levels were <br />                                                             elevated in Mut10, WC-1 and WC-2 levels in such strains were <br />                                                             much lower than in WT, suggesting that the high FRQ level <br />                                                             could not maintain the WCC level, because the positive feed- <br />                                                             back loop of FRQ promoting WCC expression was disrupted <br />                                                             in the mutant (see text for details). Arrowheads point to <br />                                                             bands corresponding to hyperphosphorylated WC-1 or WC-2, <br />                                                             representing inactive WCC (Schafmeier et al. 2005). Both <br />                                                             White Collar proteins were hypophosphorylated in Mut10, <br />                                                             consistent with loss of negative feedback that leads to upregu- <br />                                                             lated WCC activity. (B) frhR806H-KI strains phenocopy Mut10 at <br />                                                             the molecular level, showing defects in negative feedback <br />                                                             leading to high expression of FRQ and defects in positive <br />                                                             feedback leading to reduced expression of the WCC. Arrow- <br />                                                             heads point at hyperphosphorylated WCC. (C) Real-time PCR <br />                                                             analysis of mRNA levels in samples collected over 1 day in <br />                                                             darkness or after 24 hr in light. wc-1 (top) and wc-2 (bottom) <br />                                                             mRNA levels were in general reduced in frhR806H compared to <br />                                                             WT. (Error bar 6 SD, n &#194;&#188; 3 PCR repeats.) <br />  358                                                   M. Shi, et al. <br />  <br /> performed co-IP in strains in which WC-1 and WC-2                                       DISCUSSION <br /> expression was driven by the qa-2 promoter and could <br />                                                                    We have identified a dominant negative allele of frh <br /> be artificially induced (Figure 6A). Here, despite the <br />                                                                 with a discrete loss of the circadian clock-specific role(s) <br /> high level of qa-2-driven gene expression, the mutant <br />                                                                 for the FRH protein. FRH had been previously identi- <br /> strains showed low levels of WCC interacting with FRQ, <br />                                                                 fied as a protein copurifying with FRQ whose knock- <br /> especially WC-1 (Figure 7C) and were arrhythmic on <br />                                                                 down resulted in arrhythmicity (Cheng et al. 2005), and <br /> race tubes (Figure S3 B). Thus, frhR806H appears to be a <br />                                                                 it was therefore proposed as a component of the core <br /> clock-specific allele that affects the interaction of the <br />                                                                 clock. However, because frh appears to be the ortholog <br /> FFC with the WCC, leading to a defect in the negative <br />                                                                 of an essential gene in Saccharomyces, mtr4 (dob1), that <br /> feedback loop. <br />                                                                 encodes a DEAD/DEAH box RNA helicase component <br />    These results, taken in the context of the forced <br />                                                                 of the exosome complex, and the rhythm defects were <br /> heterokaryon data (Figure 4A), also suggest that frq R806H <br />                                                                 exposed only in a narrow range of expression allowing <br /> acts as a dominant negative allele. We know that FRH is <br />                                                                 extremely weak growth and aberrant development, it <br /> much more abundant than FRQ in the cell (Cheng et al. <br />                                                                 was difficult to completely rule out pleiotropic effects <br /> 2005). In heterokaryons of WT and frhR806H, FRHR806H <br />                                                                 as underlying the rhythm defect. Identification of <br /> may compete with FRHWT for FRQWT interaction, par- <br />                                                                 FRHR806H obviates this concern; strains bearing this <br /> tially blocking negative feedback and obscuring rhyth- <br />                                                                 mutant protein are normally robust, displaying essen- <br /> mic conidiation on race tubes. <br />                                                                 tially wild-type growth and development, and yet are <br />                                                                 profoundly arrhythmic. Although other explanations <br />                                                                 are possible, the simplest interpretation of the data is <br />                                                                 that all of the essential functions performed by FRH are <br />                                                                 maintained normally in FRHR806H, and the corollary of <br />                                                                 this is that those essential functions may not be required <br />                                                                 for the clock-specific roles of FRH. <br />  <br />  <br />  <br />  <br />                                                                    Figure 6.&#226;&#8364;&#8221;Reduced WC-1 phosphorylation and stability in <br />                                                                 frhR806H suggest a role for FRH in positive feedback. (A) WCC <br />                                                                 induction in frhR806H. The qa-wc-1; qa-wc-2; frhR806H strain was <br />                                                                 cultured in 2% glucose ( QA condition) and then moved <br />                                                                 to 0.1% glucose with 0.01 m QA (1QA condition) and grown <br />                                                                 for 8 hr. Extracts were made and subjected to Western analysis <br />                                                                 and the result showed that WC-1 and WC-2 could be induced <br />                                                                 by QA. (B) Reduced WC-1 stability in frhR806H. WC-1 and WC-2 <br />                                                                 were induced in WT and frhR806H by QA for 12 hr before QA <br />                                                                 was washed away and CHX was added. Time course samples <br />                                                                 collected after QA release were subjected to Western analysis <br />                                                                 and the change in WC-1 levels as determined by densitometry <br />                                                                 is plotted as a function of time. WC-1 was less stable in frhR806H <br />                                                                 than in WT, suggesting that FRHR806H had a defect in main- <br />                                                                 taining WC-1 stability and FRH has a role in post-translation <br />                                                                 regulation of WC-1. WC-2 was stable in frhR806H. (C) Defects in <br />                                                                 WC-1 phosphorylation in frhR806H. (Left) WCC was induced in <br />                                                                 WT and frhR806H, and samples were cultured in dark and ex- <br />                                                                 tracted at subjective dusk time. WC-1 in frhR806H shows faster <br />                                                                 mobility, consistent with hypophosphorylation. (Right) WC- <br />                                                                 1 is normally phosphorylated under these conditions and <br />                                                                 can be dephosphorylated by l-PPase and this dephosphoryla- <br />                                                                 tion can be inhibited by sodium vanadate. Samples from <br />                                                                 CT15 (DD4) were analyzed: N signifies no addition to the re- <br />                                                                 action mix; PPase, addition of phosphatase; PPase1V, addi- <br />                                                                 tion of both PPase and the PPase-inhibitor vanadate. (D) <br />                                                                 FRH regulates FRQ and WC-1 expression. qa-frhKI strains were <br />                                                                 initially cultured in 2% glucose media for 48 hr (0 QA) and <br />                                                                 then moved into 0.1% glucose media with 0.01 m QA for in- <br />                                                                 duction. Time course samples (6 hr and 24 hr) were collected <br />                                                                 and extracts were subjected to Western analysis. FRH is induc- <br />                                                                 ible by QA in qa-frhKI. Induction of FRH leads to increases in <br />                                                                 FRQ and WC-1 expression. The experiment shown is repre- <br />                                                                 sentative; variability in the kinetics of FRH, FRQ, and WC-1 <br />                                                                 induction was noted, but not in the presence of induction. <br />                                       Roles of FRH in the Neurospora Circadian Clock                                    359 <br />  <br />    In the 1107 amino acid FRH protein, the DEAD/              its synthesis, hypophosphorylated FRQ goes to the <br /> DEAH domain is near the N terminus, a helicase                nucleus (Luo et al. 1998) and interacts with the WCC <br /> domain is in the middle of the protein, and the DSHCT         (Cheng et al. 2001b; Denault et al. 2001), resulting in a <br /> (DOB1/SKY12/helY-like helicase C-terminal) domain             decrease in the ability of the WCC to bind to DNA <br /> is near to the C terminus. The R806H mutation is near         (Froehlich et al. 2003) probably as a result of its <br /> to but not within the C-terminal DSHCT domain, and            increased phosphorylation (Schafmeier et al. 2005; <br /> since all domains essential for RNA helicase function         He et al. 2006). FRQ then ushers the WCC out of the <br /> appear intact in frhR806H which, further, does not affect     nucleus closing the feedback loop (Hong et al. 2008). <br /> the interaction with FRQ, it seems likely that this              The data presented here show that FRH is clearly <br /> mutation is in the region specific for binding with           required for completion of the negative feedback loop <br /> WCC. Consistent with this, salient aspects of circadian       because this cycle is blocked at this first step, the <br /> function previously ascribed to FRQ are compromised           interaction between FRQ and the WCC, with the result <br /> in FRHR806H. The original and principal role of FRQ in        that FRQ expression remains constitutively high. These <br /> the oscillator is to mediate negative feedback: Soon after    data are also wholly consistent with the recent observa- <br />                                                               tion (Schafmeier et al. 2008) that transcriptionally <br />                                                               active WC-1 is unstable and that positive feedback <br />                                                               (Lee et al. 2000) is a delayed result of the FRQ-mediated <br />                                                               WCC phosphorylation that initiates negative feedback. <br />                                                               It is clear that FRH is also required for this, because in <br />                                                               frhR806H, despite the presence of elevated levels of FRQ, <br />                                                               WC-1 and WC-2 fail to accumulate and WC-1 remains <br />                                                               hypophosphorylated (Figure 5B). By promoting the <br />                                                               phosphorylation of WC-1 that reduces its DNA binding <br />                                                               and both inactivates and stabilizes it, FRH supports the <br />                                                               positive feedback loop that allows WCC accumulation <br />                                                               (Lee et al. 2000; Schafmeier et al. 2006). The slight <br />                                                               decline in wc-1 and wc-2 mRNA levels (Figure 5C) may <br />                                                               reflect additional subtle interconnections among the <br />                                                               feedback loops, but because the oscillator is known to be <br />                                                               insensitive to such changes in the levels of WCC <br />                                                               expression (Cheng et al. 2001b), these minor changes <br />                                                               in the frhR806H strain are not likely to underlie the <br />                                                               complete loss of rhythmicity. Rather it is the loss of <br />  <br />  <br />  <br />                                                                  Figure 7.&#226;&#8364;&#8221;The <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> in FRHR806H has no effect <br />                                                               on FRQ and FRH interaction, but abrogates interaction <br />                                                               between FRQ, FRH, and the White Collar Complex. (A) <br />                                                               Co-immunoprecipitation assay. FRQ and FRH form the <br />                                                               FRQ&#226;&#8364;&#8220;FRH complex (FFC) in WT, shown by reciprocal co-IP <br />                                                               using a-FRQ and a-FRH. Similarly, the complex formation <br />                                                               is intact in Mut10 as well, where a-FRH co-immunoprecipi- <br />                                                               tates FRQ from extracts made from Mut10 strains. PI, pre- <br />                                                               immune serum. (B) Reciprocal co-immunoprecipitation <br />                                                               assay using a-WC-2 and a-FRQ shows that the FFC and the <br />                                                               WCC interact in extracts from WT but not Mut10 cultures. <br />                                                               (Top left) IP with a-WC-2 pulls down WC-2 and FRQ. (Bottom <br />                                                               left) WC-1 is detected in the immunoprecipitates from WT ex- <br />                                                               tracts treated with a-FRQ or a-WC-2. In contrast, in extracts <br />                                                               made from Mut10, a-WC-2 did not pull down FRQ (top right), <br />                                                               neither did a-FRQ pull down WC-1 (bottom right). However <br />                                                               in Mut10 extracts, a-WC-2 does pull down WC-1 (bottom <br />                                                               right), indicating that the WCC is intact in Mut10. (C) Co- <br />                                                               immunoprecipitation assay in strains where WCC is induced <br />                                                               shows that interaction between the FRQ/FRH complex and <br />                                                               the WCC is reduced in frhR806H. (Left) In WT, IP with a-WC-1 <br />                                                               or a-WC-2 pulls down FRQ. (Right) In frhR806H, a-WC-1 or <br />                                                               a-WC-2 pulls down some FRQ protein, but the level is reduced <br />                                                               in comparison with WT, arguing the interaction between FFC <br />                                                               and WCC is reduced in mutant. <br />  <br />  <br /> negative feedback, and the data show that FRH is                         <br /> required with FRQ to execute this function.                                      <br />    A novel and potentially very powerful genetic screen                 <br /> for strains with defects in circadian negative feedback                            <br /> has also been presented, and the resolution of the                            <br /> defect in Mut10 presents several lessons. First it under-                     <br /> scores the value of unbiased genetic screens, which here                        <br /> uncovered a very discrete mutation in an essential gene                  <br /> that allows a distinction to be made between the                                  <br /> essential functions of FRH and its role as a part of the                           <br /> FFC in the clock. Second, the co-occurrence of the T(IL;                     <br /> IIL) translocation tightly linked to frhR806H in Mut10 may                   <br /> serve as a cautionary note. Because the strain was fully                      <br /> viable and the rhythm defect was linked to the trans-                          <br /> location that involved LGI, traditional forward genetic                   <br /> mapping of the arrhythmic mutant with respect to                          <br /> widely spaced conventional markers on LGI always                         <br /> failed to correctly localize frh on LGII. Progeny obtained                       <br /> by independent assortment, or through the crossover of                           <br /> a translocation chromosome with centromere II (bear-                     <br /> ing the mutant frh) and the genetically marked normal                        <br /> sequence chromosome bearing LGI, would be lacking                                <br /> all genes distal to the translocation breakpoint on LGII;                 <br /> these would not survive. It was only through the use of                           <br /> SNP markers, 6 years after the strain was isolated, that                    <br /> frhR806H could be identified.                                                 <br />    While identification of FRHR806H allows separation of                       <br /> clock-specific functions of FRH from those functions                    <br /> required for cell viability, we cannot categorically ex-                      <br /> clude the possibility that some of the essential functions                    <br /> of FRH may also impact the clock. For example, the                       <br /> relatively minor downregulation of wc-1 and wc-2 mRNA                        <br /> levels in frhR806H might signal a role for FRH in RNA                       <br /> stability. In any case, continued study of FRH, the FFC,                        <br /> and the circadian FFC&#226;&#8364;&#8220;WCC complex of FRQ, FRH,                                      <br /> WC-1, WC-2, and its associated kinases is likely to be                        <br /> informative.                                                                  <br />   <br />  <br />  <br />  <br />                                                                                                              Shi_Figure S1 <br />  <br />  <br />                          A                  SNP72 M              SNP72 O <br />                                          WT CEN1 M               Mut10 CEN1 O <br />  <br />  <br />  <br />                                          SNP6 M <br />  <br />  <br />  <br />                                          SNP6 O <br />  <br />                                               SNP8 O              SNP8 M <br />                                              frh   R806H <br />                                                                   frhWT <br />                                          Mut10 CEN2              WT CEN2 <br />  <br />  <br />                          B <br />                                                                                    period&#194;&#177;S.D. (hrs) <br />                      WT sibling                                                       22.6&#194;&#177;1.2 <br />                          frh R806H                                                    AR <br />  <br />  <br />  <br />  <br />                          C                frhR806H            WT <br />                                        16 19 23 3 6 10    16 19 23 3 6 10 CT time <br />                                      LL 4 8 12 16 20 24 LL 4 8 12 16 20 24 DD(hrs) <br />                              FRQ <br />  <br />  <br />                           WC-1 <br />  <br />  <br />                           WC-2 <br />  <br />  <br />                               * <br />                              FRH <br />                                * <br />  <br />  <br />                        Loading <br />  <br />  <br />  <br />  <br />    FIGURE S1.&#226;&#8364;&#8221;Genetic and molecular resolution of the mutations found in the Mut10 strain. (A) Shown is an alternative view of <br /> the homology based chromosomal pairing at meiosis in a cross between Mut10 (Oak Ridge) and WT (Mauriceville). Since Mut10 <br /> contains a chromosomal translocation, pairing can only be achieved through formation of a cruciform structure during meiosis. <br /> Progeny generated from a crossover between the frh locus and the translocation site would separate the translocation and the frh <br /> missense mutation. By scoring the genetic markers shown, we obtained progeny bearing only the translocation (containing <br /> markers SNP6O and SNP72O) and progeny bearing frhR806H but not the translocation (containing SNP6M and SNP72M). (B) <br /> Race tube assay shows that frhR806H, like Mut10, is arrhythmic under free running conditions. (C) Western analysis of clock <br /> proteins in frhR806H freed from the chromosomal translocation background showed elevated constitutive FRQ expression, <br /> indicating loss of negative feedback, and reduced WCC expression level, indicating defects in positive feedback. FRH level in <br /> frhR806H was invariant and similar to WT, suggesting that the mutation affects the protein function but not FRH gene expression <br /> or stability. <br />                                                              M. Shi et al.                                                      3 SI <br />  <br />  <br />  <br />  <br />                                                                                                                Shi_Figure S2 <br />  <br />  <br />                                           A <br />  <br />  <br />  <br />  <br />                                           B                   BstEII        NruI <br />  <br />  <br />  <br />  <br />                                                             SM54 <br />                                                             SM53 <br />                                                             SM52 <br />                                                             SM54 <br />                                                             SM53 <br />                                                             SM52 <br />  <br />  <br />                                                              WT <br />                                                             WT <br />                                                         L <br />  <br />                                            8.6kb <br />                                            4.9kb <br />                                            3.6kb <br />  <br />  <br />  <br />                                             1.5kb <br />  <br />                                               1.0kb <br />  <br />  <br />  <br />                                           C         frhWT-KI frhR806H-KI <br />                                              gDNA SM52 SM53 SM54 WT      Mut <br />                                      allele specific W M W M W M W M W M <br />                                      PCR product <br />  <br />  <br />     FIGURE S2.&#226;&#8364;&#8221;Molecular confirmation and characterization of the frhR806H-KI strains examined in Figure 4. (A) Constructs <br /> bearing frhWT or frhR806H flanked by the selectable bar marker at the 3&#226;&#8364;&#8482; end were transformed into a mus-52 (ku80) strain chosen to <br /> foster a high rate of homologous recombination (COLOT et al. 2006; NINOMIYA et al. 2004). The thick bar region represents a <br /> probe used in the Southern analysis below. The double crossovers point to the approximate positions where the recombination <br /> would occur to generate frh knock-in strains. Arrow heads mark the positions of PCR primers used in (C). (N: NruI site, B: BstEII <br /> site) (B) Genomic DNA samples from WT, frhWT-KI and frhR806H-KI were subjected to enzyme digestion and Southern analysis. <br /> Since knock-in strains contained the selectable marker bar, a size shift was expected and also observed on the Southern gel <br /> confirming the strain identity. (C) A G-A missense mutation was found by sequencing in the frhR806H allele. Based on this, allele <br /> specific oligos (see materials and methods) were used in PCR analyses to differentiate frhR806H-KI and frhWT-KI. <br />  4 SI                                                        M. Shi et al. <br />  <br />  <br />  <br />  <br />                                                                                                                   Shi_Figure S3 <br />  <br />  <br />  <br />  <br />                                   A <br />  <br />                                    flanking     5!-UTR    wc ORF            3!-UTR flanking   gDNA <br />  <br />  <br />                                           bar   qa-2 P    wc ORF            3!-UTR      KI PCR fragment <br />  <br />  <br />  <br />  <br />                                   B <br />                                                                                      0.01M QA <br />                               qa-wcc <br />                            qa-wcc; frhR806H <br />  <br />  <br />  <br />    FIGURE S3.&#226;&#8364;&#8221;Restoration of positive feedback in frhR806H. (A) Diagram of the knock-in strategy used to replace the wc-1/wc-2 <br /> loci with QA inducible wc-1/wc-2 constructs. qa-2 P: qa-2 promoter region. Those two constructs were transformed into frhR806H <br /> respectively, and a cross between qa-wc-1; frhR806H and qa-wc-2; frhR806H was made to obtain a qa-wc-1; qa-wc-2; frhR806H strain in <br /> the progeny. (B) Race tube assay of strains overexpressing WC-1 and WC-2 in frhR806H. Restoration of the positive feedback loop <br /> was mimicked through WCC induction by QA. However, the strain was still arrhythmic on race tubes, suggesting that loss of <br /> negative feedback in frhR806H leads to clock malfunction and that restoring the positive feedback loop alone cannot rescue the <br /> clock. <br />                                                                       M. Shi et al.                                                5 SI <br />  <br />  <br />  <br />  <br />                                                                                                                 Shi_Figure S4 <br />  <br />  <br />  <br />  <br />                                A <br />                                    BamHI                     probe    BamHI <br />  <br />                                  flanking           5!-UTR     frh ORF         3!-UTR flanking   gDNA <br />  <br />  <br />                                            bar      qa-2 P     frh ORF          3!-UTR     KI PCR fragment <br />  <br />  <br />                                                    BamHI <br />                                B <br />                                             qa-frhKI    WT <br />  <br />  <br />                                4.5 kb <br />                                3.2 kb <br />  <br />  <br />  <br />  <br />                                 C                  QA                                        Period+S.D.(hrs) <br />                                                                                                    _ <br />                                                     0                                        AR <br />                                                  1X10-6M                                     AR <br />                                                  1X10-5M                                         _ <br />                                                                                              22.1+1.2 <br />                                                  1X10-4M                                     21.7+0.9 <br />                                                                                                  _ <br />                                                  1X10-3M                                         _ <br />                                                                                              21.9+0.5 <br />                                                  1X10-2M                                     22.6+1.2 <br />                                                                                                  _ <br />                              qa-frh <br />                                              1 X 10-6 M                                      AR <br />                                              2 X 10-6 M                                      AR <br />                                              4 X 10-6 M                                      AR <br />                                              6 X 10-6 M                                          _ <br />                                                                                              20.6+1.4 <br />                                              8 X 10-6 M                                      22.0+1.9 <br />                                                                                                  _ <br />                                              1 X 10-5 M                                          _ <br />                                                                                              23.8+1.4 <br />                                              1 X 10-5 M                                          _ <br />                                                                                              21.2+0.5 <br />                                              2 X 10-5 M                                          _ <br />                                                                                              21.7+1.3 <br />                                              4 X 10-5 M                                          _ <br />                                                                                              21.5+1.2 <br />                                              8 X 10-5 M                                          _ <br />                                                                                              21.6+1.1 <br />                                              1 X 10-4 M                                      21.8+0.6 <br />                                                                                                  _ <br />  <br />  <br />  <br />     FIGURE S4.&#226;&#8364;&#8221;Changing FRH dosage has little or no effect on the circadian clock. (A) Diagram of the knock-in strategy used to <br /> replace the frh locus with a QA inducible frh construct. qa-2 P: qa-2 promoter region. Arrowheads point to BamHI sites. (B) <br /> Genomic DNA samples digested with BamHI were subjected to Southern analysis using the probe shown in panel A. DNA bands <br /> in qa-frh strains displayed a size shift due to insertion of bar gene in the locus. (C) The qa-frh strain was grown on race tubes with <br /> different QA concentrations. The strain grew slowly at low QA conditions, consistent with the observation that frh is an essential <br /> gene (Cheng et al. 2005). The rhythm was missing at low QA concentrations, and gradually appeared at QA concentrations <br /> around 10-5 M. This concentration range was examined more closely in the middle and bottom panels. Around 1x10-5 M QA, <br /> the strain showed a clock phenotype intermediate between wild type and arrhythmic, suggesting that changes of the FRH level <br /> might impact the strength of the negative feedback loop. When the rhythm appeared, however, the period length was wild type, <br /> around 22 hours, suggesting that FRH serves as an &#226;&#8364;&#732;on/off switch&#226;&#8364;&#8482; rather than a &#226;&#8364;&#732;period modulator&#226;&#8364;&#8482; in the circadian clock. <br />   <br /> </body></html>