<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genome-wide search for yeast RNase P substrates <br /> reveals role in maturation of intron-encoded <br /> box C/D small nucleolar RNAs <br />  <br /> Ribonuclease P (RNase P) is an essential endonuclease responsible             related <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> bacterial RNase P, termed RNase P and RNase MRP. <br /> for the 5&#226;&#180;&#8226;-end maturation of precursor tRNAs. Bacterial RNase P               RNase P is responsible for processing pre-tRNAs, whereas <br /> also processes precursor 4.5S RNA, tmRNA, 30S preribosomal RNA,               RNase MRP processes pre-rRNA, mitochondrial RNA primers <br /> and several reported protein-coding RNAs. Eukaryotic nuclear                  and is required for the regulated turnover of a cell cycle mRNA <br /> RNase P is far more complex than in the bacterial form, employing             (18&#226;&#8364;&#8220;21). Both the eukaryotic RNase P and RNase MRP <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> multiple essential protein subunits in addition to the catalytic RNA          nuclei are far more complex enzymes than bacterial RNase P <br /> subunit. RNomic studies have shown that RNase P binds other                   (22). Each enzyme still employs a distinct but related RNA <br /> RNAs in addition to tRNAs, but no non-tRNA substrates have                    subunit and contains multiple required protein subunits for <br /> previously been identified. Additional substrates were identified             function in vivo. In yeast the two enzymes have eight identical <br /> by using a multipronged approach in the budding yeast Saccha-                 proteins subunits, with RNase P having one unique protein and <br /> romyces cerevisiae. First, RNase P-dependant changes in RNA                   RNase MRP having two unique proteins (23, 24). Seven of the <br /> abundance were examined on whole-genome microarrays by us-                    nine RNase P proteins are highly positively charged (pI 9.3&#226;&#8364;&#8220; <br /> ing strains containing temperature sensitive (TS) mutations in two            10.0), which could provide multiple substrate RNA binding sites <br /> of the essential RNase P subunits, Pop1p and Rpr1r. Second, RNase             in addition to the ones for pre-tRNAs that would be analogous <br /> P was rapidly affinity-purified, and copurified RNAs were identified          to the bacterial enzyme. This might explain why yeast RNase P <br /> by using a genome-wide microarray. Third, to identify RNAs that               is much more susceptible to inhibition by single-stranded RNAs <br /> do not change abundance when RNase P is depleted but accumu-                  than bacterial RNase P (25). The additional protein components <br /> late as larger precursors, &gt;80 potential small RNA substrates were            might provide the ability to hold other types of RNA in position <br /> probed directly by Northern blot analysis with RNA from the RNase             to occupy the active cleavage site provided by the conserved, <br /> P TS mutants. Numerous potential substrates were identified, of               catalytic RNA subunit. Thus, given the number of non-tRNA <br /> which we characterized the box C/D intron-encoded small nucle-                substrates cleaved by even the bacterial enzymes, it seems <br /> olar RNAs (snoRNAs), because these both copurify with RNase P                 possible that nuclear RNase P has been incorporated into the <br /> and accumulate larger forms in the RNase P temperature-sensitive              processing pathways for a number of different RNAs. Previous <br /> mutants. It was previously known that two pathways existed for                studies of the eukaryotic enzymes have suggested this (26, 27), <br /> excising these snoRNAs, one using the pre-mRNA splicing path and              and there is substantial evidence that the closely related RNase <br /> the other that was independent of splicing. RNase P appears to                MRP participates in regulated turnover of specific mRNAs <br /> participate in the splicing-independent path for the box C/D <br />                                                                               (18, 19). <br /> intron-encoded snoRNAs. <br />                                                                                  To search for physiologically relevant, previously unrecog- <br />                                                                               nized substrates for nuclear RNase P, we used three different <br /> RNA &#229;&#8230;&#169; biogenesis <br />                                                                               approaches in Saccharomyces cerevisiae (Fig. 1). In the first, the <br />                                                                               RNase P holoenzyme was affinity purified, and RNAs that <br />  <br /> R   ibonuclease P (RNase P) is a conserved endoribonuclease <br />     responsible for removing the 5&#226;&#172;&#732; leader sequence from pre- <br /> cursor transfer RNAs (pre-tRNAs) found in bacteria, archaea, <br />                                                                               copurify with the enzyme were identified by using a whole- <br />                                                                               genome microarray. The second and third approaches utilize <br />                                                                               temperature-sensitive (TS) RNase P mutant strains. In the <br /> eukarya (1, 2). In all cases, with the possible exception of some             second approach, multiple TS mutant strains in multiple RNase <br /> organelles, RNase P is composed of both RNA and protein                       P subunits were grown at the restrictive temperature and changes <br /> subunits. Bacterial RNase P is the simplest form of the holoen-               in the abundance of individual RNAs were measured by using a <br /> zyme, with one large RNA subunit and a single small protein                   whole genome microarray. In the third approach, we examined <br /> subunit (1). Although the RNA subunit of bacterial RNase P is                 the processing of possible small RNA substrates in TS mutant <br /> sufficient for catalysis in vitro at high salt concentrations (3), both       strains by Northern blot analysis to detect RNAs of altered size <br /> the RNA and protein subunits are required in vivo. The protein                that accumulate in the absence of RNase P activity, even though <br /> subunit appears to stabilize the catalytically active conformation <br /> of RNase P RNA and assist with substrate binding (4&#226;&#8364;&#8220;7). In <br /> addition to pre-tRNAs, bacterial RNase P is known to process                  Author contributions: D.J.C. and D.R.E. designed research; D.J.C., J.A.P., and S.C.W. per- <br /> several substrates that are proposed to contain tRNA-like                     formed research; D.J.C. and G.B.W. analyzed data; and D.J.C. wrote the paper. <br /> structures: 4.5S RNA, tmRNA, viral RNAs, mRNAs, ribo-                         The authors declare no conflict of interest. <br /> switches, ColE1 replication origin control RNAs, and C4 anti-                 This article is a PNAS Direct Submission. <br /> sense RNA from phages P1 and P7 (8&#226;&#8364;&#8220;16). The presence of                       Data deposition: The data reported in this paper have been deposited in the Gene <br /> the protein subunit in the RNase P holoenzyme increases                       Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE10514). <br /> the substrate versatility of the enzyme over the RNA enzyme                   &#194;&#167;To   whom correspondence should be addressed. E-mail: engelke@umich.edu. <br /> alone (17).                                                                   This article contains supporting information online at www.pnas.org/cgi/content/full/ <br />    The nature of the eukaryotic nuclear RNase P is much more                  0801906105/DCSupplemental. <br /> complex. First, there are two very similar enzymes that are                   &#194;&#169; 2008 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 12218 &#226;&#8364;&#8220;12223 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; August 26, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 34                                                        www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0801906105 <br />     Affinity tagged                  Temperature sensitive                      Results <br />    RNase P strain                   RNase P mutant strain                      Identifying RNAs That Copurify with RNase P. Potential RNase P <br />                                                                                substrates were determined by identifying RNAs that copurify <br />                                                                                with RNase P. RNase P was purified by using either a small RNA <br />    Rapidly isolate                                                             affinity tag (aptamer) in the RNA subunit (Rpr1r) that binds to <br />      RNase P                       Grow strains and shift to                   streptavidin (29, 30) or a TAP tag (31) on the protein subunit <br />                                     restrictive temperature                    that is unique to RNase P, Rpr2p. Untagged wild-type RNase P <br />                                                                                was subjected to the same purification steps to establish a <br />   <span id='am-4' about='rdfs:label' typeof='owl:Thing'>Label</span> co-purified <br />                                                                                background for RNA contaminants in the purification process. <br />        RNAs <br />                                                                                The coisolated RNA was then reverse transcribed into fluores- <br />                          Label RNAs that         Identify processing           cently labeled cDNA. The cDNA was used to probe a microarray <br />                            accumulate                  defects                 containing oligos to the entire yeast genome: ORFs, known <br />                                                     in noncoding               noncoding RNAs, and intergenic regions (32, 33). <br />                                                       RNAs by                     Comparison of results from independent purifications indi- <br />         Whole genome microarray                     northern blot              cated that data from the RNA subunit aptamer tag purification <br />                                                                                (R2 &#226;&#171;&#189; 0.708) were much more consistent than the protein subunit <br /> Fig. 1. Multipronged approach to identify additional RNase P substrates.       (Rpr2p) TAP purification (R2 &#226;&#171;&#189; 0.196). It would be expected <br /> Three complementary approaches were taken to discover previously unrec- <br /> ognized in vivo substrates for yeast RNase P. RNAs that physically associate <br />                                                                                that more transient interactions would be lost during the more <br /> with RNase P were identified by copurification with affinity-tagged holoen-    protracted dual-column purification of the Rpr2p-TAP tag than <br /> zyme. Functional relationships were identified in temperature-sensitive mu-    the single-column RPR1-aptamer tag. Additionally, the RNA <br /> tant strains by examining changes in abundance by microarray or accumula-      aptamer purification involves mild elution from the affinity resin <br /> tion of aberrant-size processing products by Northern blot. Microarray data    with only the addition of a small molecule (biotin). Therefore, we <br /> are summarized in Table S1.                                                    focused on data obtained from the RNA subunit tag purification. <br />                                                                                   Numerous RNAs were detected as copurifying with RNase P. <br />                                                                                The 250 most abundantly copurified RNAs mapped predomi- <br /> they might not change in abundance. Here we report that this                   nantly to ORFs encoding components of the translation ma- <br /> multipronged approach identified numerous potential sub-                       chinery [Table 1; full listing can be found in supporting infor- <br /> strates, and we focus on characterization of a particular class of             mation (SI) Table S1]. The prevalence of these mRNAs in the <br /> RNAs that both copurify with RNase P and accumulate larger                     copurification does not correlate to their abundance in the cell <br /> forms in the RNase P temperature-sensitive mutants. This class                 (R2 &#226;&#171;&#189; 0.263), referenced to the yeast transcriptome (34) (Fig. <br /> is the set of box C/D small nucleolar RNAs (snoRNAs) that are                  S1), consistent with selective association with RNase P. The <br /> encoded in the introns of six pre-mRNA introns. It was previ-                  correlation drops even further (R2 &#226;&#171;&#189; 0.125) when limited to the <br /> ously known that two pathways existed for excising these                       250 most abundantly copurified RNAs. It is interesting to note <br /> snoRNAs, one using the pre-mRNA splicing path and the other                    that tRNAs are not identified in this isolation, possibly because <br /> that was independent of splicing (28). RNase P appears to                      of the transient binding of tRNA substrates and products to <br /> participate in the splicing-independent path.                                  RNase P. However, it is also possible that cDNA probe produc- <br />  <br />  <br />  <br />  <br />                                                                                                                                                            BIOCHEMISTRY <br /> Table 1. Nuclear-encoded RNAs that copurify with RNase P <br /> mRNAs with intron encoded                     ASC1 (3.67), EFB1 (3.01), IMD4 (3.14), RPL7A (4.36), RPL7B (3.18), TEF4 (2.90) <br />   snoRNAs <br /> Ribosomal small subunit mRNAs                 ASC1 (3.67), RPS0A (3.66), RPS0B (3.05), RPS1A (4.04), RPS1B (4.24), RPS2 (3.61), RPS3 (2.76), RPS4A <br />                                                 (4.74), RPS4B (3.36), RPS5 (3.88), RPS6A (3.59), RPS7A (4.62), RPS7B (4.24), RPS8A (3.95), RPS9A <br />                                                 (2.74), RPS10A (3.72), RPS10B (2.71), RPS11A (4.17), RPS11B (3.67), RPS12 (4.90), RPS13 (3.10), <br />                                                 RPS14A (3.75), RPS15 (2.63), RPS16B (3.72), RPS17A (3.87), RPS17B (4.15), RPS18A (3.94), RPS18B <br />                                                 (3.19), RPS19A (4.20), RPS19B (4.18), RPS20 (2.80), RPS21A (2.91), RPS21B (3.44), RPS22B (4.84), <br />                                                 RPS23A (2.58), RPS23B (4.00), RPS24A (3.06), RPS24B (3.28), RPS25A (4.71), RPS25B (4.51), RPS27A <br />                                                 (3.61), RPS27B (2.86), RPS28A (4.33), RPS28B (2.71), RPS29A (3.67), RPS29B (3.84), RPS30A (2.76), <br />                                                 RPS31 (3.23) <br /> Ribosomal large subunit mRNAs                 RPL1A (3.04), RPL1B (4.54), RPP1B (3.01), RPL2B (3.97), RPP2A (4.18), RPP2B (3.24), RPL3 (2.90), RPL4B <br />                                                 (2.98), RPL5 (6.05), RPL6A (2.84), RPL6B (4.48), RPL7A (4.36), RPL7B (3.18), RPL8A (3.34), RPL9A (5.61), <br />                                                 RPL9B (3.60), RPL11A (4.32), RPL11B (4.62), RPL12A (4.47), RPL13A (7.24), RPL13B (2.89), RPL14A <br />                                                 (3.62), RPL14B (3.78), RPL15B (4.94), RPL16B (2.91), RPL17B (3.13), RPL18A (4.73), RPL18B (4.11), <br />                                                 RPL19A (5.01), RPL19B (5.01), RPL20A (3.81), RPL20B (3.54), RPL21B (3.94), RPL22A (2.57), RPL23A <br />                                                 (3.78), RPL23B (3.77), RPL24A (3.05), RPL24B (2.90), RPL26A (3.47), RPL26B (2.67), RPL27A (3.17), <br />                                                 RPL27B (4.45), RPL28 (3.86), RPL29 (3.55), RPL30 (4.65), RPL31A (3.74), RPL33A (3.80), RPL33B (3.55), <br />                                                 RPL34A (5.11), RPL34B (5.72), RPL35B (2.86), RPL36A (2.99), RPL36B (3.57), RPL37B (2.67), RPL38 <br />                                                 (4.56), RPL40A (3.43), RPL40B (3.69), RPL41A (3.57), RPL41B (3.73), RPL42A (3.78), RPL42B (3.39), <br />                                                 RPL43A (2.70) <br /> RNA polymerase I subunit mRNAs                RPB8 (2.70), RPA135 (3.10), RPC40 (3.06), RPO26 (3.66) <br /> Translation Initiation mRNAs                  PAB1 (2.86), TIF1 (2.67), TIF11 (3.20), NIP1 (2.82), TIF3 (3.79) <br /> Translation elongation mRNAs                  EFB1 (3.01), TEF4 (2.90), EFT1 (3.27) <br /> Noncoding RNAs                                RUF5 (3.43) <br />  <br />   Shown is enrichment with aptamer-tagged RPR1 listed in parentheses; bolded genes accumulate an RNase P ts strain. <br /> *Full listing of individual genes can be found in Table S1. <br />  <br />  <br /> Coughlin et al.                                                                                  PNAS &#229;&#8230;&#169; August 26, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 12219 <br />  Table 2. Nuclear-encoded RNAs that accumulate with RNase P <br /> RNAs that accumulate in RPR1 TS strain (from top 250)* <br />   Ribosomal small subunit mRNAs            RPS30A (2.05), RPS17A (1.87), RPS18B (2.05), RPS16A (1.88), RPS10B (1.93), RPS28A (2.19), RPS10A (2.17) <br />   Ribosomal large subunit mRNAs            RPL31A (1.98), RPL41B (2.79), RPL41A (3.05), RPL37B (2.60), RPL34A (2.02), RPL30 (2.01), RPL34B (1.89), <br />                                              RPL38 (3.92), RPL36A (2.17) <br />   RNA polymerase I subunit mRNAs           RPC10 (2.53), RPA34 (1.88), RPC19 (2.00), RPO26 (2.04) <br />   RNA polymerase III subunit mRNAs         RPC10 (2.53), RPC19 (2.00), RPO26 (2.04) <br />   Mitochondrial Ribosome                   MRPL37 (1.86), MRPL49 (2.09), MRP49 (1.82), MRPL3 (2.09), MRPL24 (1.85), MRPL40 (2.16) <br />   Noncoding RNAs                           RUF5 (2.08) <br /> RNAs that accumulate in POP1660D6 TS strain (from top 250)* <br />   Ribosomal large subunit mRNAs            RPL41A (1.98), RLP24 (2.03), RPL38 (1.97) <br />   RNA polymerase I subunit mRNAs           RPC10 (1.93), RPA34 (2.43), RPC19 (1.93) <br />   Noncoding RNAs                           RUF5 (2.92) <br /> RNAs that accumulate in POP1233E11 TS strain (from top 250)* <br />   Ribosomal large subunit mRNAs            RPL41A (2.04), RPL27B (2.23), RPL37B (2.93), RPL40B (2.10), RPL38 (2.34), RPL36A (2.39), RPL33B (2.34) <br />   Ribosomal small subunit mRNAs            RPS28A (6.51), RPS31 (2.12), RPS26A (2.51), RPS20 (2.05), RPS27A (2.08) <br />   Noncoding RNAs                           RUF5 (2.10) <br /> RNAs that accumulate larger forms (possible precursors) in RPR1 TS (identified by northern blot) <br />   Intron-encoded snoRNAs                   snR24, snR18, snR54, snR39, snR38 <br />  <br />   Accumulation values are included in parenthesis following the gene name, and bolded gene products copurify with RNase P RNA affinity tag. <br /> *Full listing of individual genes can be found in Table S1. <br />  <br />  <br />  <br /> tion is inefficient due to difficulties reverse transcribing tRNAs           scripts and intron-encoded snoRNAs (38). There are eight total <br /> because of nucleotide modifications or tertiary structure.                   intron-encoded snoRNAs, six of which are box C/D. RNA from <br />                                                                              all six box C/D snoRNA ORFs were in the top 250 RNAs <br /> Identifying RNAs That Accumulate in Temperature-Sensitive Mutants.           copurifying with RNase P. Of the two box H/ACA snoRNAs, <br /> The next approach to identifying previously unrecognized                     only the snR44/RPS22B locus was in the top 250 copurifying <br /> RNase P substrates was to identify RNAs that change in                       RNAs, whereas snR191/NOG2 was the 319th-ranked RNA. <br /> abundance in the TS RNase P mutant strains. Temperature- <br /> sensitive RNase P mutations were available in two subunits of                Box C/D Intron-Encoded snoRNAs Accumulate Known Processing In- <br /> yeast RNase P: the unique RNA subunit, Rpr1r (35, 36), and the               termediates in an RNase P Temperature-Sensitive Mutant. As part of <br /> largest protein subunit, Pop1p, that is also a component of                  our screen for possible substrates for RNase P, we performed <br /> RNase MRP (37).                                                              Northern blot analysis on RNA from RNase P TS strains to see <br />    The RNAs affected in the temperature-sensitive strains are                whether some processing intermediates of some small RNAs <br /> vastly different between RPR1 TS and the two POP1 TS strains                 might accumulate, even though the overall amount of RNA from <br /> (Table S1). This could be because of the dual role of Pop1p in               that transcription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> did not change significantly. Accumula- <br /> RNases P and MRP or the different time courses of the                        tion of pre-tRNAs in these mutants was previously demonstrated <br /> temperature shift (2 h to see growth inhibition in RPR1 TS                   (36, 37), and we opted to use the TS in the RPR1 RNA subunit <br /> compared with 6 h for POP1 TS strains). However, there is an                 because it, unlike Pop1 protein, is unique to RNase P and not <br /> interesting general preference for the RNAs that coisolate with              found in MRP. Here, 79 additional noncoding RNAs were <br /> the RNase P and accumulate in response to the TS mutations                   examined by Northern blot, representing all classes of small <br /> (Table 2) in that they tend to be components of the translation              nuclear RNAs, small cytoplasmic RNA (SCR1), and box H/ACA <br /> machinery out of proportion to their abundance in the cell (Fig.             and C/D small nucleolar RNAs from independently transcribed, <br /> S1). This carries through when considering only the RNAs that                polycistronic, and intron-encoded genes (Table S2). In most <br /> both coisolate and accumulate in response to TS <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span>. Of                 cases RNAs other than tRNAs did not accumulate larger (or <br /> these RNAs, 16 are mRNAs encoding protein subunits of the                    smaller) forms in the RNase P mutant. A subset of the box C/D <br /> ribosome. The remaining RNAs that both copurify with RNase                   snoRNAs was the notable exception. All six of the intron- <br /> P and accumulate in the mutant strain include mRNAs for two                  encoded box C/D snoRNAs accumulate a processing interme- <br /> translation initiation factors (TIF11, SUI13), a box C/D snoRNA              diate in the RNase P TS mutant that is larger than the mature <br /> binding protein (SNU13), a common subunit in RNA poly-                       snoRNA (Fig. 2). There are also two intron-encoded box H/ACA <br /> merases I, II, and III (RPO26), the CUP1-1/RUF5 locus, and six               snoRNAs, but no aberrant forms of these were observed. <br /> intergenic regions. Signal from three of the intergenic regions                 Probing Northern blots with oligonucleotides spaced at the <br /> neighboring ribosomal protein genes (RPL42B, RPL41A,                         indicated positions (Fig. 2) showed that each accumulated RNA <br /> RPL38) were also identified, although signal from the coding                 is a 5&#226;&#172;&#732;-extended pre-snoRNA that contains the 5&#226;&#172;&#732; exon, the <br /> regions of the genes themselves was not found and no charac-                 intron 5&#226;&#172;&#732; of the snoRNA, and the snoRNA itself. Primer <br /> terized RNA is made from these intergenic regions. We note that              extension confirms the 5&#226;&#172;&#732; end is the expected primary transcript <br /> tRNAs do not accumulate substantially in this microarray anal-               for the pre-mRNA containing the intron (data not shown). The <br /> ysis, but this is not unexpected in that the amount of uncut                 intron sequence 3&#226;&#172;&#732; of the snoRNA and 3&#226;&#172;&#732; exons are absent in the <br /> pre-tRNAs that accumulates before the cells stop growing is                  accumulated RNA, suggesting the RNase P defect causes loss <br /> small compared with the stable population of mature tRNAs.                   of an essential cleavage that initiates processing on the 5&#226;&#172;&#732; side of <br />                                                                              these snoRNAs. Unfortunately, attempts to identify the sites of <br /> Intron-Encoded snoRNAs. In the top 250 copurified RNAs, we                   cleavage in vitro or in vivo failed to identify specific RNase P <br /> obser ved seven intron-encoded small nucleolar RNAs                          cleavage sites (see below). However, the nature of these 5&#226;&#172;&#732;- <br /> (snoRNAs). There are 75 known snoRNAs in the yeast genome.                   extended pre-snoRNAs is compatible with predicted processing <br /> The majority of the snoRNAs in yeast are independently tran-                 intermediates in one maturation pathway for intron-encoded box <br /> scribed, however there are also examples of polycistronic tran-              C/D snoRNAs (28, 39). <br />  <br /> 12220 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0801906105                                                                                     Coughlin et al. <br />                                                                                                                                     Poly-cistronic <br />                                                     Intron-encoded box C/D snoRNAs                                                box C/D snoRNA <br />            wt ts                  wt ts                  wt ts              wt ts                               wt ts                  wt ts <br />      nt                      nt                      nt                         nt                       nt <br />    1,000 <br />                    mRNA                                               mRNA    1,000           mRNA                       mRNA <br />                                            mRNA <br />      750                                                                       750 <br />                                                      750 <br />  <br />     500 <br />                     -*      500                      500 <br />                                                                                400 <br />                             400 <br />                                                                                                          400 <br />     100 <br />                                                                                100 <br />                             300 <br />                    snR24                             100                                      snR39 <br />                                                                      snR54                                                                      snR74 <br />                                                                                                                         snR38 <br />                                            snR18 <br />                             100                                                                                                             snR74 <br />             snR24 ASC1             snR18 EFB1               snR54    IMD4              snR39 RPL7A              snR38 TEF4 <br />            1 2 3 45678            1 2 3 45678              1 2 3     45678       X    1 2 3 45678              1 2 3 45678 <br />  <br /> Fig. 2. Box C/D intron-encoded snoRNAs accumulate processing intermediate in temperature-sensitive RNase P strain. Northern blot analysis of RNA from <br /> wild-type (WT) and temperature-sensitive (TS) RNase P mutants reveals an accumulation of an unusual processing form of the box C/D intron-encoded snoRNAs <br /> in only the RNase P-deficient strain. Because RPL7A and RPL7B are highly homologous, only 7A and its snoRNA are shown. The identity of the accumulated <br /> transcripts is indicated schematically to the right of the panels and was determined by which oligo probes detected them and by the <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> of the primer <br /> extension product to the 5&#226;&#172;&#732; end of the RNA. (Asterisk indicates a breakdown intermediate with an undetermined 5&#226;&#172;&#732; terminus.) Probes 1 and 2 hybridize to the <br /> processing intermediate. Oligo probes 3 and X did not give a signal. Probes 4 &#226;&#8364;&#8220; 8 hybridize to the mRNA. Probe 2 hybridizes to the mature snoRNA. Noncoding <br /> RNAs (79) were examined for altered forms by Northern blot (Table S1). A blot of a snoRNA from a polycistronic transcript is shown for contrast. <br />  <br />  <br />  <br />    Maturation of box C/D snoRNAs can proceed through two                        the presence of the TS RPR1 mutation were unsuccessful (data <br /> pathways in yeast (28, 39): the primary pathway depends on                      not shown), suggesting the combination is synthetically lethal <br /> splicing the pre-mRNA to free a snoRNA-containing lariat, and                   and strengthening the interpretation that RNase P is required in <br /> the minor pathway is independent of the splicing reaction (Fig.                 the alternative pathway. <br /> 3). The primary splicing-dependent pathway requires lineariza-                     Although computer folding of the pre-snoRNAs does not <br /> tion of the intron lariat product by debranching enzyme (Dbr1p),                predict tRNA-like structures, we have previously shown that <br /> allowing subsequent release of the snoRNA by endonucleases                      highly purified, yeast nuclear RNase P binds tightly to single- <br /> and exonucleases. In the minor, the snoRNA is excised from the                  stranded RNAs (25) and cuts efficiently at highly preferred sites <br /> intron of the pre-mRNA directly, leading to the destruction of                  in pre-rRNA that do not have obvious tRNA-like structures (35). <br /> the mRNA. Although this process is thought to involve the Rnt1                  We therefore tested whether the pre-snoRNA alone (without <br /> endonuclease (40), the enzymatic path is not well defined and                   protein) was a highly selective substrate for yeast nuclear RNase <br /> our data suggest that in vivo it also involves cleavage by RNase                P. Purified RNase P cuts T7-transcribed, 5&#226;&#172;&#732;-extended pre- <br />                                                                                 snoRNAs in multiple places in vitro (Fig. S2). As previously <br />  <br />  <br />  <br />  <br />                                                                                                                                                               BIOCHEMISTRY <br /> P upstream of the snoRNA. The unusual 5&#226;&#172;&#732;-extended pre- <br /> snoRNAs that accumulate in the RNase P mutant strain have                       observed for other non-tRNA substrates (35), there was not a <br /> already undergone 3&#226;&#172;&#732; maturation of the snoRNA and require                       clear correlation to specific sequences or structures upstream of <br /> one or more 5&#226;&#172;&#732; cleavages. Multiple attempts to delete DBR1 in                   the mature 5&#226;&#172;&#732; end of the snoRNAs. The major cleavage site in <br />                                                                                 pre-snR14 is at the 5&#226;&#172;&#732; end of the mature snoRNA, with a minor <br />                                                                                 cut site 38 nucleotides upstream in the intron. In pre-snR18, the <br />                                                                                 major cleavage site corresponds to &#226;&#172;&#8225;43 nucleotides upstream of <br />                        RNA                                                      the 5&#226;&#172;&#732; end of the mature snoRNA, which is seven nucleotides <br />                     sno <br />                                           pre-mRNA                              upstream of a proposed stem essential for splicing-independent <br />                                                                                 snoRNA maturation (39, 40). RNase P makes multiple cuts in <br />                                    processing at 5&apos; or 3&apos;                       pre-snR38, the strongest is 58 nucleotides upstream of the <br />   splicing                                end of snoRNA                         mature snoRNA, with additional cuts at the 5&#226;&#172;&#732; end of the mature <br />                                                                                 snoRNA and 17 nucleotides and 21 nucleotides upstream. <br />                                                                                 Because these naked RNA cleavages did not form a detectable <br />                                                                                 <span id='am-85' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, we refrain from suggesting correlations to physiologi- <br />  mature mRNA                                                                    cally relevant sites. <br />                                                                                    Because the in vivo pre-snoRNA substrates are likely to be <br />            +                  RNase P + ?                                       ribonucleoprotein complexes (28, 38&#226;&#8364;&#8220;40), we also tried to cleave <br />                                                                                 accumulated RNPs in soluble extracts of the RNase P TS mutant <br />                      debranching                                                after accumulation of precursor. A large excess of purified <br />                        (Dbr1p)                                                  RNase P was added to such extracts under conditions where the <br />       snoRNA                                                         mature     longer forms were not cleaved by the endogenous temperature- <br />       in lariat                                                     snoRNA      sensitive activity (37&#194;&#176;C). Although the longer snoRNAs de- <br />                                                                                 graded slowly with time (data not shown), no RNase P-specific <br /> Fig. 3. Intron-encoded snoRNA processing pathways. Two distinct process- <br />                                                                                 preferred cleavage locations were identified, including those <br /> ing pathways exist for intron-encoded snoRNAs. The splicing-dependent path- <br /> way produces the mature mRNA and snoRNA after the intron lariat form has <br />                                                                                 shown to be sensitive in the naked RNA. Thus, if correct <br /> been opened by Dbr1p and further processing. The splicing-independent           precursor small nucleolar RNPs (snoRNPs) for RNase P cleav- <br /> pathway produces only the mature snoRNA. The dashed lines indicate the step     age can be reconstituted, it will require more extended biochem- <br /> affected by RNase P.                                                            ical analysis. <br />  <br /> Coughlin et al.                                                                                    PNAS &#229;&#8230;&#169; August 26, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 34 &#229;&#8230;&#169; 12221 <br />  Discussion                                                              ribosomes, the two most abundant RNA components of the <br /> Recent studies by other groups have suggested various substrates        translational machinery. <br /> for eukaryotic RNase P. Although it has been shown that RNase <br /> P can cut the noncoding HRA1 RNA in vitro (27), we see no               Materials and Methods <br /> evidence of an in vivo association or function, because HRA1            Yeast Strains. Affinity purification. In a yeast strain containing a C-terminal <br />                                                                         tandem affinity purification (TAP) tag on RPR2 (Open Biosystems; YSC1178- <br /> neither copurifies with RNase P nor does it accumulate in any <br />                                                                         7501110), chromosomal RPR1 was disrupted with HIS3 and replaced with a <br /> of the three temperature-sensitive mutants. Another recent <br />                                                                         plasmid, pRS315, containing RPR1 with RNA affinity tags for streptavidin and <br /> study depleted one of the protein subunits, Rpp1p, found in both        Sephadex (29, 30). <br /> RNases P and MRP (26). This identified 74 RNAs that accu-               Temperature-sensitive mutations. Previously described strains with defined <br /> mulate with the Rpp1p depletion. Of these 74 noncoding RNAs,            RNase P mutations were used in this study: G207G211 RPR1 (42), R233K POP1, <br /> two copurify with RNase P in our hands, MAN7 and TLN1, and              and R626L/P628K POP1 (37). <br /> another one, TLN20, accumulates in the RPR1 TS mutant strain <br /> (Table S3). Interestingly, MAN7 and TLN1 are both found                 Yeast Growth. Yeast were grown in standard synthetic media containing <br /> antisense from genes encoding ribosomal proteins. Thus, a               dextrose and lacking histidine (SDC-H). For temperature-sensitive (TS) assays, <br /> future direction of interest might be to investigate participation      yeast were grown at 30&#194;&#176;C into log phase (OD600 of 0.6 &#226;&#8364;&#8220; 0.8) and then diluted <br /> of RNase P in processing and turnover of antisense RNAs.                into SDC-H media prewarmed to 37&#194;&#176;C. The strain-specific time period was <br />                                                                         determined from growth curves for wild-type (WT) and TS RNase P strains, <br />    In this study we focused on the intron-encoded snoRNAs, <br />                                                                         using the earliest time after the growth curve of the WT and TS strains <br /> because transcripts encoding them copurify with RNase P. We <br />                                                                         diverged. G207G211 RPR1 was grown at 37&#194;&#176;C for 2 h, and the POP1 TS strains <br /> find that 5&#226;&#172;&#732;-extended pre-snoRNAs for each of the box C/D               were grown at 37&#194;&#176;C for 6 h. <br /> intron-encoded snoRNAs, but not the H/ACA snoRNAs, accu- <br /> mulated in the TS RNase P strains. The link between the                 RNase P Purification. Two yeast strains were subjected to RNase P purification: <br /> intron-encoded box C/D snoRNAs and the other RNAs that                  (i) a control strain that expresses WT RPR1 and WT RPR2 and (ii) a tagged strain <br /> copurify with affinity-purified RNase P is strengthened by the          that expresses TAP-tagged RPR2 and aptamer-tagged RPR1. <br /> identities of the host mRNAs of the intron-encoded snoRNA:                  RNase P was purified by using either a single-column aptamer-streptavidin <br /> seven of eight host mRNAs encoded proteins are involved in              affinity purification or a two-column TAP purification. Sequential TAP then <br /> translation. Although the abundance of the pre-mRNAs from               aptamer-streptavidin purifications were also attempted but did not yield <br /> this pathway did not increase significantly in the RNase P TS           sufficient amounts of RNase P for analysis. Briefly, 8 liters of yeast were grown <br /> mutants, this is not unexpected for a maturation, rather than           in yeast extract/peptone/dextrose media to an OD600 of 0.8 &#226;&#8364;&#8220;1.0. Yeast were <br />                                                                         lysed in lysis buffer [50 mM Hepes, pH 7.5/10% glycerol/0.5 mM EDTA/150 mM <br /> turnover defect. The 5&#226;&#172;&#732;-extended pre-snoRNA is a known pro- <br />                                                                         NaCl/1 mM DTT/0.1% Nonidet P-40/complete, EDTA-free protease inhibitor <br /> cessing intermediate in the splicing-independent, intron-               (Roche Diagnostics Corporation)] with a model 110-Y Microfluidizer (Mi- <br /> encoded snoRNA maturation pathway (Fig. 3). This splicing-              crofluidics), using five passes each through a 200-&#226;&#65533;&#174;m chamber and a 100-&#226;&#65533;&#174;m <br /> independent pathway requires endonucleolytic cuts both 5&#226;&#172;&#732; and           chamber. Cell extract was cleared by centrifugation for 10 min at 17,000 &#226;&#171;&#187; g, <br /> 3&#226;&#172;&#732; of the snoRNA and leads to the destruction of the mRNA               followed by a 100-min spin at 105,000 &#226;&#171;&#187; g. Twenty five microliters of a 50% <br /> (28). The pre-snoRNA that accumulates in the RNase P mutant             slurry of IgG Sepharose was added per 1 ml of cell extract and incubated on a <br /> strain has already been trimmed at the 3&#226;&#172;&#732; end of the snoRNA,            rotating drum at 4&#194;&#176;C for 2 h. The IgG Sepharose was washed with 25 column <br /> but it still contains the full transcript 5&#226;&#172;&#732; to the snoRNA. It is       volumes of lysis buffer. RNase P was eluted overnight with excess tobacco etch <br /> possible that RNase P cuts at the 5&#226;&#172;&#732; end of these snoRNAs in            virus (TEV) protease. The elution was adjusted to 2 mM CaCl2 and bound for <br /> vivo, but it is also possible that it cuts one or more sites upstream   2 h to 25 &#226;&#65533;&#174;l of a 50% slurry of Calmodulin Affinity Resin (CAR) per milliliter of <br />                                                                         starting extract. The CAR was washed five times with five volumes of lysis <br /> of the snoRNA and the 5&#226;&#172;&#732; maturation is subsequently performed <br />                                                                         buffer and 2 mM CaCl2. RNase P was eluted with five volumes of lysis buffer <br /> by other nucleases. This would resemble the case for 5&#226;&#172;&#732; matu-           and 10 mM EGTA. <br /> ration of 5.8S rRNA by RNase MRP cleavage followed by further <br /> trimming.                                                               Microarray Preparation. Associated RNAs were reverse transcribed to cDNAs <br />    In vitro cleavage assays with purified RNase P and pre-              and fluorescently labeled with either Cy3 or Cy5 dyes (32). Labeled cDNAs were <br /> snoRNAs did not produce a consistent model for the nature of            then hybridized to a yeast whole-genome microarray, which contains &#226;&#172;&#381;13,000 <br /> the RNase P cleavage site in the intron-encoded snoRNAs.                features corresponding to both known ORFs and intergenic regions. The <br /> Although this is likely because of a lack of appropriate additional     design is accessible at the Gene Expression Omnibus (GEO) database (ww- <br /> components in the purified system (e.g., snoRNA structure), it          w.ncbi.nlm.nih.gov/geo/) under platform accession no. GPL6424 (43). The <br /> leaves open the possibility that the RNase P effects on the             ratio of Cy3 to Cy5 fluorescence indicates the relative amounts of RNA coming <br /> intronic pre-snoRNAs is indirect. However, direct participation         through the purification of the tagged to untagged yeast strains. The result- <br />                                                                         ing data were analyzed as previously described (32). All data are accessible at <br /> by RNase P is strongly suggested by the combination of (i) the <br />                                                                         the GEO (43) database under provisional series accession no. GSE10514. <br /> physical interaction demonstrated by copurification of snoRNAs             RNAs were detected by microarray analysis as described (32). Briefly, RNAs <br /> with RNase P, (ii) the functional relationship seen by the              were reverse transcribed into cDNA in the presence of aminoallyl-dUTP using <br /> accumulation of known processing intermediates in the RNase             random nonamers as primers. The cDNA was then labeled with either Cy3 or <br /> P mutant strain, and (iii) the previously demonstrated ability of       Cy5 (Amersham Biosciences). Labeled cDNA was then hybridized to a yeast <br /> yeast nuclear RNase P to cleave non-tRNA substrates at high             whole-genome microarray (44). <br /> rates and in a partially sequence-specific fashion (35). The <br /> analysis of the sequences and RNP structures that lead to RNase         Northern Blotting of RNAs. Hot acid phenol (45) was used to extract total RNA <br /> P recognition will presumably be an extended undertaking,               from yeast cells harvested at 30&#194;&#176;C and 37&#194;&#176;C, and concentrations were deter- <br /> especially because the highly complex 10-subunit RNase P RNP            mined by UV absorbance. Ten micrograms of total yeast RNA per lane was <br /> structure could hypothetically recognize a relatively large num-        electrophoresed on denaturing 8% polyacrylamide gels. The RNA was then <br />                                                                         electrotransferred to a Nytran SuperCharge membrane (Schleicher &amp; Schuell <br /> ber of signals. <br />                                                                         Bioscience). <br />    It is especially interesting that one of the intron-encoded             Specific oligodeoxynucleotide probes were designed to the majority of <br /> snoRNA processing pathways is compromised, because this                 yeast small nuclear RNAs (snRNAs) and small nucleolar RNAs (snoRNAs). Probes <br /> suggests the pathway might be nucleolar. Not only is the final          were radiolabeled with [&#226;&#65533;&#165;-32P]ATP. Labeled probes were hybridized and washed <br /> destination of the snoRNPs the nucleolus, but RNase P is also           according to instructions accompanying the Nytran SuperCharge membrane. <br /> found primarily in the nucleolus in yeast (41). Thus, RNase P           Signals on the Northern blots were determined with a PhosphorImager <br /> might provide a link between production of both tRNAs and               (Molecular Dynamics; model no. 445 SI). <br />  <br />  <br /> 12222 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0801906105                                                                                      Coughlin et al. <br />  In Vitro Cleavage Reactions and Primer Extensions. Templates for T7 transcrip-                   kinased with [&#226;&#65533;&#165;-32P]ATP, which was then gel isolated on a 12% polyacrylamide <br /> tion were made from genomic S. cerevisiae DNA by PCR using primers to the                        gel. The labeled primers were hybridized with the treated and untreated <br /> region 100 nucleotides 5&#226;&#172;&#732; of the mRNA (to include any essential 5&#226;&#172;&#732;-                              pre-snoRNA for 1 h at 42&#194;&#176;C and then extended by using SuperScript II reverse <br /> untranslated region that may be structurally significant) and a primer com-                      transcriptase according to manufacturers instructions (Invitrogen). The cDNA <br /> plementary to the 3&#226;&#172;&#732; end of the intron-encoded snoRNA. These templates                           was then electrophoresed onto an 8% polyacrylamide gel along with respec- <br /> were then used for T7 transcription. The in vitro-transcribed RNA was gel                        tive dideoxy sequencing ladders (46). <br /> purified on an 8% polyacrylamide gel. RNase P purified using the tandem <br /> affinity purification (TAP) tag on the unique protein subunit Rpr1p was added               <br /> to the gel-purified 5&#226;&#172;&#732;-extended pre-snoRNA in RNase P assay buffer (10 mM <br />                                                                                                   <br /> Hepes, pH 7.9/100 mM KCl/10 mM MgCl2) and incubated for 15 min at 37&#194;&#176;C.                       <br /> The reaction was then treated with proteinase K, and the RNA was extracted                      <br /> by using phenol-chloroform. Primers complementary to the snoRNA were                       <br /> </body></html>