<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Mol Biol. 2010 March 26; 397(2): 481&#226;&#8364;&#8220;494. doi:10.1016/j.jmb.2010.01.051. <br />  <br />  <br />  <br />                            Crystal structures trypanosomal histidyl-tRNA synthetase <br />                            illuminate differences eukaryotic prokaryotic <br />                            homologs <br />  <br />                            Ethan A Merritta,b,*, Tracy L Arakakia,b, J Robert Gillespiea,c, Eric T Larsona,b, Angela <br />                            Kelleya,c, Natascha Muellera,c, Alberto J Napulia,c, Jessica Kima,b, Li Zhanga,b, Christophe <br />                            L M J Verlindea,b, Erkang Fana,b, Frank Zuckera,b, Frederick S Bucknera,c, Wesley C Van <br />                            Voorhisa,c, Wim G J Hola,b <br />                            Medical Structural Genomics Pathogenic Protozoa http://www.msgpp.org <br />  <br />                            b   Department Biochemistry, University Washington, Seattle, WA 98195 USA <br />                            c   Department Medicine, University Washington, Seattle, WA 98195 USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Crystal structures histidyl-tRNA synthetase eukaryotic parasites Trypanosoma brucei <br />                                  Trypanosoma cruzi provide structural view eukaryotic form enzyme, reveal <br />                                  differences bacterial homologs. Histidyl-tRNA synthetases general contain extra domain <br />                                  inserted conserved motifs 2 3 Class II aminoacyl-tRNA synthetase catalytic core. <br />                                  The current structures dimensional topology domain different in <br />                                  bacterial archaeal/eukaryotic forms enzyme. Comparison apo histidine-bound <br />                                  trypanosomal structures indicates substantial active site rearrangement histidine binding, but <br />                                  relatively little subsequent rearrangement reaction histidine ATP form enzyme&#226;&#8364;&#8482;s <br />                                  reaction product, histidyladenylate. The specific residues involved forming binding pocket <br />                                  adenine moiety differ substantially previously characterized binding site in <br />                                  bacterial structures homologous residues human histidyl-tRNA synthetases. The <br />                                  essentiality single histidyl-tRNA synthetase gene T. brucei shown severe depression <br />                                  parasite growth rate results partial suppression expression <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>RNA interference</span>. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Keywords <br />                                  aminoacyl-tRNA synthetase; protozoa; drug target; tropical disease; RNAi <br />  <br />  <br />                            Introduction <br />                                               Aminoacyl-tRNA synthetases (aaRS) mediate key biological function translating an <br />                                               RNA message corresponding protein sequence. Each specific aminoacyl-tRNA <br />                                               synthetase responsible attaching particular amino acid proper cognate tRNA. This <br />  <br />  <br />                            *Corresponding author: Phone: 206-543-1421 Fax: 206-685-7002, merritt@u.washington.edu. <br />                            Accession Numbers <br />                            Coordinates structure factors deposited Protein Data Bank accession numbers 3HRI, 3HRJ, 3HRK. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service customers <br />                            providing early version manuscript. The manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. Please note production process errors discovered could <br />                            affect content, legal disclaimers apply journal pertain. <br />                             Merritt et al.                                                                                               Page 2 <br />  <br />  <br />                                             proceeds separate reactions, catalyzed active site. First, amino acid <br />                                             adenylated reaction ATP yield aminoacyladenylate + pyrophosphate. In second <br />                                             reaction, adenylate activated amino acid replaced terminal adenosine residue <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             tRNA acceptor stem. In general results charged tRNA uniquely associates <br />                                             anticodon sequence carries anticodon loop amino acid carries its <br />                                             acceptor stem. A exceptional cases require subsequent additional chemical modification <br />                                             attached amino acid1. As expected molecule key role, aminoacyl- <br />                                             tRNA synthetases living organisms. They group classes defined by <br />                                             sequence phylogeny consequent structural homology core domains. The <br />                                             Class I catalytic domain contains Rossmann fold highly conserved sequence motifs. <br />                                             The Class II catalytic domain contains core antiparallel &#206;&#178;-sheet surrounded &#206;&#177;-helices, and <br />                                             uniquely identified conserved sequence motifs. Further subdivision Class II into <br />                                             subclasses a, b, c based nature anticodon binding domain. The aaRS for <br />                                             amino acid lysine belongs consistently Class I Class II forms <br />                                             life. Despite broad conservation core structural functional features, many <br />                                             individual aaRS evolved idiosyncratic structural features, additional inserted domains, <br />                                             new biological functions supplement essential role. This <br />                                             combination essential biological function cross-species variation suggests that <br />                                             individual aaRS useful targets drugs targeting infectious disease [review: Hurdle <br />                                             et al.2]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             We chose trypanosomal Histidyl-tRNA synthetase (HisRS) study broader effort <br />                                             Medical Structural Genomics Pathogenic Protozoa collaborative investigate <br />                                             aminoacyl-tRNA synthetases possible targets drug discovery3. Among target <br />                                             pathogens Trypanosoma brucei, causative agent African sleeping sickness, and <br />                                             Trypanosoma cruzi, causative agent Chagas disease. The chronic forms diseases <br />                                             fatal, existing therapeutics unsatisfactory poor effectiveness toxicity. <br />                                             Many higher eukaryotes, including mammals, separate cytosolic mitochondrial forms <br />                                             aminoacyl-tRNA synthetase. In contrast, genomes lower eukaryotes, <br />                                             including trypanosomatids, contain aaRS sequences cytosolic grouping. As a <br />                                             complication, trypanosomatids evolved separate mitochondrial forms for <br />                                             specific aaRS, apparently gene duplication cytosolic sequence4,5. However, <br />                                             HisRS these; T. brucei T. cruzi genomes contain single HisRS, of <br />                                             typical eukaryotic cytosolic type, used cytosolic mitochondrial <br />                                             translation. <br />  <br />                                             HisRS Class IIa aminoacyl-tRNA synthetase containing conserved domains. The first <br />                                             catalytic domain, core fold shared Class II aaRS. The second is <br />                                             C-terminal domain, &#206;&#177;/&#206;&#178; fold shared Class IIa enzymes HisRS, ThrRS, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             ProRS, eukaryotic/archaeal GlyRS6. This C-terminal domain holds binding <br />                                             site tRNA anticodon loop, anticodon itself primary specificity <br />                                             determinant recognizing tRNAHis. Cognate tRNA recognition instead believed be <br />                                             mediated residues Class II motif 2 loop catalytic domain7. No structure <br />                                             reported HisRS:tRNAHis complex, tRNA binding modeled <br />                                             approximately based structures bacterial threonyl- prolyl-tRNA synthetase:tRNA <br />                                             complexes8,9. The conserved domain unique HisRS, consists 50&#226;&#8364;&#8220;75 residue <br />                                             &#206;&#177;-helical insertion conserved motifs 2 3 catalytic domain. Prokaryotic <br />                                             sequences insertion domain contain highly conserved residues surface <br />                                             near active site proposed interact 3&#226;&#8364;&#178; strand backbone tRNA <br />                                             acceptor stem10. Guth et al.11 insertion domain important the <br />                                             histidine adenylation reaction absence tRNA, fluorescent resonance transfer <br />                                             experiments Escherichia coli HisRS suggest forward rate adenylation is <br />                                             limited concomitant rigid body rotation insertion domain. Furthermore, this <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 3 <br />  <br />  <br />                                             conformational change introduces asymmetry reaction rate monomers <br />                                             making HisRS dimer. No equivalent characterization insertion domain been <br />                                             reported eukaryotic HisRS. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             In addition conserved features, HisRS eukaryotes including humans <br />                                             contains copies short, N-terminal WHEP domain6. WHEP domains named <br />                                             presence non-homologous aaRS (tryptophanyl, histidyl, glutamyl, <br />                                             prolyl), mediate various non-canonical biological activities promoting <br />                                             association multiple aaRS molecules larger assembly12,13. The HisRS WHEP domain <br />                                             implicated major epitope human autoimmune disease14. NMR studies <br />                                             shown single WHEP domain human HisRS forms antiparallel 2-helix <br />                                             bundle [unpublished; PDB accession 1&#195;&#8212;59]. This structure observed individual <br />                                             modules repeated WHEP domain sequences aaRS15. The N-terminal <br />                                             sequence trypanosomal HisRS equivalent <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> human WHEP domain has <br />                                             low sequence similarity. It known N-terminus trypanosomal HisRS <br />                                             mediates secondary biological function. <br />  <br />                                             There mammalian HisRS genes, HARS HARS2 (also known HARSL), which <br />                                             arose inverted gene duplication16,17. The human HisRS gene products 79% sequence <br />                                             identical outside WHEP domain. Although trypanosomal human <br />                                             HisRS sequences belong same, eukaryotic branch HisRS phylogenetic tree18, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             maximum pairwise sequence identity human genes either <br />                                             trypanosomal homolog discussed 30%. The pairwise sequence identity <br />                                             eukaryotic sequences previously studied bacterial HisRS homologs also <br />                                             30%. <br />  <br />                                             Crystal structures bacterial HisRS published, offering insight the <br />                                             specific residues binding interactions involved binding substrates histidine and <br />                                             ATP10,19,20,9, intermediate product histidyladenylate (HAM)10,21. The apo structure <br />                                             archaeal HisRS, Thermococcus acidophilum, determined <br />                                             crystallographically [unpublished; PDB accession 1wu7]. No structure eukaryotic HisRS <br />                                             reported date. <br />  <br />                                             We report crystal structure apo HisRS T. brucei 2.85 &#195;&#8230; resolution, and <br />                                             crystal structures HisRS T. cruzi complex amino acid substrate, histidine, <br />                                             product activation reaction, histidyladenylate, 1.8 &#195;&#8230; 3.05 &#195;&#8230; resolution <br />                                             respectively. These structures eukaryotic HisRS insertion domain in <br />                                             eukaryotic HisRS adopts different three-dimensional structure bacterial insertion <br />                                             domain. They reveal significant differences binding site histidyladenylate when <br />                                             compared previously reported crystal structures bacterial homologs19,10. The residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             contributing differences active site distinguish trypanosomal HisRS <br />                                             human HisRS homologs, suggesting possible design selective <br />                                             inhibitors trypanosomal enzymes. <br />  <br />                                Results Discussion <br />                                Comparison overall structure HisRS organisms <br />                                             The three-dimensional structures T. cruzi T. brucei HisRS homologs similar, <br />                                             reflecting fact 84% identical sequence (Figures 1, 2). The sequence identity <br />                                             trypanosomal HisrRS previously studied bacterial homologs order <br />                                             25% catalytic domain lower elsewhere. The entire trypanosomal HisRS protein <br />                                             visible crystal structures exceptions N-terminus, truncated <br />                                             expressed proteins begin residue 45 (T. cruzi numbering), residues 456&#226;&#8364;&#8220;467 <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 4 <br />  <br />  <br />                                             near C-terminus anticodon binding domain. Residues 158&#226;&#8364;&#8220;164 conserved Class <br />                                             II aaRS motif 2 well-ordered apo structure histidyladenylate complex, but <br />                                             visible higher resolution histidine complex. Conversely, residues 220&#226;&#8364;&#8220;241, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             comprising half insertion domain, poorly ordered apo structure but <br />                                             well-defined ligand-bound structures. The overall folds catalytic anticodon <br />                                             binding domains previously seen well-studied bacterial homologs, there <br />                                             differences residues forming active site discussed separately <br />                                             below. The catalytic anticodon binding domains connected extended stretch of <br />                                             residues, 376&#226;&#8364;&#8220;383, following Class II motif 3. <br />  <br />                                             All HisRS sequences characterized additional domain inserted conserved <br />                                             motifs 2 3 Class II catalytic core. The insertion tends larger eukaryotes than <br />                                             prokaryotes. Sequence conservation eukaryotic, archaeal, prokaryotic insertions <br />                                             low, eukaryotes higher. The trypanosomal human HisRS sequences are <br />                                             roughly 30% identical region (Figure 2). The insertion domain proposed <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                             play role tRNA recognition21,10, specific model developed. The present <br />                                             work reveals eukaryotic prokaryotic insertion domains different three <br />                                             dimensional structures (Figure 3). After superposition structurally similar eukaryotic <br />                                             prokaryotic core domains, evident correspondence position of the <br />                                             secondary structural elements belonging respective insertion domains. Given the <br />                                             divergence sequence structure region, unlikely single model <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             interaction insertion domain tRNA apply prokaryotic and <br />                                             eukaryotic HisRS. <br />  <br />                                             The N-terminus eukaryotic HisRS folds 2-helix bundle (WHEP domain)15,22. <br />                                             Truncation 60 N-terminal residues human HisRS results inactive protein as <br />                                             measured production His-tRNAHis&#226;&#8364;&#8240;23, clear loss the <br />                                             45-residue WHEP domain absence following 15 residues belonging start <br />                                             catalytic domain. Both trypansomal proteins present work N-terminal <br />                                             truncations equivalent removal WHEP domain. The 44-residue N-terminal <br />                                             truncation T. cruzi HisRS studied retains ability activate histidine, was <br />                                             assayed ability charge tRNAHis. Although obviously present crystal <br />                                             structures, interested evidence WHEP-equivalent domain the <br />                                             N-terminus trypanosomal HisRS. There 10% sequence identity the <br />                                             trypanosomal human sequences 46 residues, corresponding helices <br />                                             human WHEP domain (Figure 2). To evaluate likelihood trypanosomal <br />                                             HisRS N-terminus forms helical bundle, examined trypanosomal sequences for <br />                                             helix propensity. Two different secondary structure predictors indicate strong propensity for <br />                                             helix formation residues 10&#226;&#8364;&#8220;21 (Jpred24) 5&#226;&#8364;&#8220;28 (Porter25) predict second <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             helical region pair form WHEP-like 2-helix bundle. Equivalent <br />                                             predictions obtained N-terminal sequence HisRS Leishmania major, <br />                                             related pathogenic protozoan. <br />  <br />                                Conformational changes associated substrate binding <br />                                             The <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> active site Class II aaRS formed central strands conserved <br />                                             &#206;&#178;-sheet. These &#206;&#178;6, &#206;&#178;10, &#206;&#178;11 present structures (Figure 4). The binding surface <br />                                             amino acid ATP substrates largely contributed residues conserved <br />                                             Class II sequence motifs 2 3. In case HisRS, histidine recognition binding is <br />                                             mediated additional motifs, His A His B, shared aaRS21. <br />                                             While Class II aaRS bind ATP coordination divalent metal ion helps to <br />                                             position &#206;&#178;- &#206;&#179;- phosphates acts electrophile adenylation reaction, HisRS <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                 Page 5 <br />  <br />  <br />                                             atypical binding ATP catalyzing adenylation involvement divalent <br />                                             cation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS atypical extent substrate binding proceeds induced-fit <br />                                             mechanism. The catalytic domain undergoes substantial conformational change upon <br />                                             histidine binding (Figure 4). Helix &#206;&#177;3 shifts 10 &#195;&#8230; active site. Helices <br />                                             &#206;&#177;8, &#206;&#177;9, &#206;&#177;10, &#206;&#177;11 insertion domain shift concert approximately 3 &#195;&#8230; towards <br />                                             active site. The HisRS-specific histidine recognition sequence (His A motif), residues 314&#226;&#8364;&#8220; <br />                                             318 T. cruzi enzyme, shifts 3 &#195;&#8230; interact directly histidine. This <br />                                             effect repositioning Arg 314 placed interact &#206;&#177;-phosphate of <br />                                             ATP, probably contributes increased affnity ATP histidine bound (Figure <br />                                             5). As discussed later, attribute positioning &#206;&#178;- &#206;&#179;-phosphates coordination by <br />                                             additional arginine residues, conserved HisRS sequences. <br />                                             Somewhat unexpectedly, portion conserved Class II motif 2 interacts the <br />                                             adenosine ring system, residues 158&#226;&#8364;&#8220;165 T. cruzi sequence, ordered apo <br />                                             structure histidine-bound structure. We observed lack order in <br />                                             different crystal form complex (not detailed here), making likely this <br />                                             true consequence binding histidine artifact crystal lattice packing. <br />  <br />                                             There conformational change histidine substrate-bound enzyme the <br />                                             histidyladenylate product-bound enzyme. Helix &#206;&#177;3 shifts slightly farther active site. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             In addition, motif 2 residues 158&#226;&#8364;&#8220;165 fully ordered interact directly ring <br />                                             adenyl moiety. This constitutes shift approximately 7 &#195;&#8230; position of <br />                                             C&#206;&#177; atoms residues tip loop compared position apo <br />                                             crystal structure. As discussed below, specific residues making overall binding <br />                                             pocket adenine ring significantly different current structures their <br />                                             equivalents bacterial archaeal homologs previously studied. There little <br />                                             shift insertion domain trypanosomal histidine- histidyladenylate- <br />                                             bound structures. Of course does preclude movement insertion domain during <br />                                             course reaction; domain flexibility necessary accommodate the <br />                                             recognition binding ATP subsequent release &#206;&#178;- &#206;&#179;- phosphates after <br />                                             catalytic displacement previously bound histidine. <br />  <br />                                Order asymmetry substrate binding <br />                                             The structural details tRNA binding HisRS directly observed, by <br />                                             analogy binding mode seen Class IIa aaRS, cognate tRNA believed to <br />                                             interact monomer HisRS dimer8,9. The geometry binding that <br />                                             tRNA molecules bind simultaneously, monomer dimer. The two <br />                                             monomers act independently, however. Francklyn coworkers extensively <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             characterized reaction ordering kinetics E. coli HisRS dimer. The E. coli enzyme <br />                                             interacts weakly, all, ATP alone11. In presence tRNA, activation histidine <br />                                             second monomer dependent prior completion aminoacyl transfer tRNA the <br />                                             active site monomer26. Even absence tRNA, rate formation of <br />                                             histidyladenylate different active sites11. If HisRS dimer repurified size <br />                                             exclusion chromatography reaction, recovered complex molar ratio one <br />                                             bound histidyladenylate dimer27. This led model activation histidine <br />                                             ATP site induces rate-limiting conformational change, possibly rigid-body rotation <br />                                             insertion domain, interferes ATP binding and/or reactivity second <br />                                             site11. <br />  <br />                                             The conformational difference apo histidine-bound trypanosomal HisRS <br />                                             crystal structures substantial, provides structural basis sequential binding the <br />                                             substrates, histidine ATP second. Thermal melt data trypanosomal HisRS <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 6 <br />  <br />  <br />                                             homologs agreement order binding solution (Figure 5). In each <br />                                             case addition AMP, ADP, ATP does stabilize HisRS dimer solution, <br />                                             addition histidine shifts TM protein 4 6&#194;&#176; C. In presence of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             histidine, addition AMP ADP causes small significant increase stability. Addition <br />                                             histidine ATP leads substantial stabilization protein corresponding <br />                                             &gt;7&#194;&#176; C increase TM relative apo protein. As enzymes active, probably <br />                                             represents stability complex protein reaction product, <br />                                             histidyladenylate. This ordering &#206;&#8221;TM observed consistently T. cruzi T. <br />                                             brucei protein constructs used current structural analysis, length (no <br />                                             N-terminal truncation) HisRS related trypanosomatid Leishmania naiffi. <br />  <br />                                             However, weak evidence structural asymmetry active sites in <br />                                             trypanosomal HisRS activation histidine ATP. The monomers making <br />                                             crystallographically observed apo His-bound dimers structurally same. <br />                                             Indeed, crystals T. cruzi HisRS:His complex contain single monomer the <br />                                             asymmetric <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>; monomers biological dimer related perfect <br />                                             crystallographic 2-fold symmetry. The crystallographically independent monomers <br />                                             making dimer T. cruzi HisRS:HAM complex identical the <br />                                             precision model; superposition monomers, RMSD 414 paired <br />                                             C&#206;&#177; atoms model 0.17 &#195;&#8230;. This does preclude existence slight differences, but <br />                                             effect atomic geometry active site monomers evident the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             crystal structure. <br />  <br />                                             The dimers seen previously characterized complexes T. thermophilus histidine <br />                                             (PDB accession 1adj) HAM (PDB accession 1ady) symmetric. In these <br />                                             structures, monomers dimer superimpose rmsd 0.50 &#195;&#8230; all <br />                                             C&#206;&#177; atoms insertion domain, superposition insertion domains the <br />                                             paired monomers differ rotation leads displacement distal helices to <br />                                             4 &#195;&#8230;. The insertion domain single archaeal homolog studied crystallographically, T. <br />                                             acidophilum, structurally like current eukaryotic structures of <br />                                             bacterial homologs. Only apo form archaeal structure reported (PDB <br />                                             1wu7), insertion domains monomers dimer differ a <br />                                             rotation leads displacement roughly 4 &#195;&#8230; distal end. <br />  <br />                                             Notwithstanding symmetric geometry monomers T. cruzi HisRS:HAM <br />                                             complex, evidence differential binding histidyladenylate at <br />                                             active sites. Electron density corresponding complete molecule histidyladenylate <br />                                             present sites (Figure 6). However, electron density monomer A greater than <br />                                             monomer B. Furthermore, monomer A density adenine moiety better <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             resolved density histidine, monomer B density histidine moiety <br />                                             stronger AMP moiety. This indicate reaction went completion <br />                                             monomer A, fraction monomer B copies present crystal the <br />                                             histidine remains unactivated. <br />  <br />                                The trypanosomal HisRS active site <br />                                             As noted above, ATP binding site HisRS fully formed a <br />                                             conformational rearrangement induced prior binding histidine. In E. coli HisRS <br />                                             complex histidinol ATP19, ATP phosphates positioned interaction <br />                                             guanidinium groups arginine residues. The &#206;&#177;-phosphate positioned by <br />                                             association bacterial residues Arg 113 Arg 259. The homologous T. cruzi residues <br />                                             Arg 156 Arg 314, analogous interactions &#206;&#177;-phosphate seen in <br />                                             present complex histidyladenylate. Arg 314 believed act electrophile <br />                                             catalysis, substituting divalent metal cation present aaRS. In absence <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                             Page 7 <br />  <br />  <br />                                             ATP-bound structure eukaryotic enzyme, used E. coli complex model <br />                                             interactions &#206;&#178;- &#206;&#179;-phosphates. The &#206;&#178;-phosphate interacts bacterial residue Arg <br />                                             113. The &#206;&#179;-phosphate positioned bacterial residues Arg 121 Arg 311. We infer <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             sequence structure alignment role bacterial Arg 121 played homologous <br />                                             trypanosomal Arg 164. However, arginine similar residue trypanosomal <br />                                             sequence alignment equivalent bacterial Arg 311. The spatial proximity sidechain <br />                                             trypanosomal residue Arg 334 current structures suggests role positioning <br />                                             ATP &#206;&#179;-phosphate prior catalysis played instead residue non-conserved <br />                                             region sequence preceding His B motif (Figure 2). The E. coli human homologs <br />                                             glycine position, equivalent interaction possible. <br />  <br />                                             The hydrogen-bonding network secures sugar moiety histidyladenylate is <br />                                             conserved bacterial trypanosomal complexes. The sidechain strongly <br />                                             conserved Gln 170 bridges ribose O4&#226;&#8364;&#178; histidyl O. The ribose O2&#226;&#8364;&#178; donates hydrogen <br />                                             bond backbone carbonyl oxygen Leu 336 accepts hydrogen bond the <br />                                             backbone N strongly conserved residue Gly 363. The ribose O3&#226;&#8364;&#178; donates hydrogen <br />                                             bond backbone carbonyl oxygen Ala 335. There conserved hydrogen bond <br />                                             terminal O&#206;&#181; strongly conserved residue Glu 158 N6 adenine ring. <br />  <br />                                             However, binding pocket occupied adenine ring current T. cruzi <br />                                             complex histidyladenylate formed intriguingly different set residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously observed ATP histidyladenylate complexes bacterial homologs. <br />                                             The adenine ring seen E. coli T. thermophilus HAM complexes sandwiched <br />                                             Phe 125 Arg 311 (E. coli numbering; Figure 7a). In trypanosomal homologs, <br />                                             residue equivalent Phe 125 His 168 (T. cruzi numbering), histidine ring does <br />                                             perform equivalent role stacking adenine ring (Figure 7b). <br />                                             But sequence structural trypanosomal equivalent bacterial Arg 311. On <br />                                             the binding pocket bacterial Arg 311 extends, trypanosomal <br />                                             enzymes instead pair residues, Cys 365 Val 366, sidechains provide a <br />                                             hydrophobic surface adjacent adenine. The role stacking face the <br />                                             adenine ring instead played sidechain Arg 164, extends the <br />                                             opposite binding pocket (Figure 7). <br />  <br />                                             Unlike structural difference bacterial trypanosomal insertion domains, the <br />                                             differences ATP binding mode demonstrated T. cruzi E. coli HisRS not <br />                                             represent generic difference eukaryotic bacterial forms HisRS active site. <br />                                             The corresponding set key residues eukaryotic HisRS sequences, including <br />                                             particular human homologs HARS HARS2, directly equivalent of <br />                                             E. coli enzyme T. cruzi enzyme (Table 1, Figure 2). That is, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             details ATP coordination deduced studies bacterial HisRS21,10,7 likely be <br />                                             valid HisRS homologs including higher eukaryotes, trypanosomal <br />                                             homologs deviate quite remarkably this. <br />  <br />                                RNA interference HisRS gene T. brucei <br />                                             As trypanosomes possess single gene coding histidyl-tRNA synthetase, be <br />                                             expected gene product essential protein. We able confirm RNAi <br />                                             silencing partial suppression HisRS expression profound effect growth <br />                                             bloodstream-form T. brucei. Targeted knockdown single HisRS gene T. brucei was <br />                                             performed using established RNA interference methods T. brucei 28. Cells induced to <br />                                             express RNAi construct showed profound suppression growth, factor &gt;103 at <br />                                             day 4 beyond, compared uninduced cultures (Figure 8a). Interestingly, degree <br />                                             mRNA knockdown far complete indicated quantitative PCR mRNA <br />                                             (Figure 8b). It likely complete gene knockdown lead more <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 8 <br />  <br />  <br />                                             complete growth arrest possibly death. The profound growth <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> partial <br />                                             knockdown mRNA supports selection HisRS drug target, pharmacological <br />                                             inhibition enzyme complete effectively stop growth possibly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             kill cells. <br />  <br />                                Potential HisRS anti-protozoan drug target <br />                                             Humans HisRS genes. Both homologous trypanosomal HisRS, the <br />                                             overall sequence identity trypanosomal enzymes human homolog is <br />                                             30%. As current structures, lack sequence conservation extends <br />                                             residues making active site. The distinctive differences adenine <br />                                             binding pocket trypanosomal HisRS compared human homologs offer <br />                                             opportunity identification selective inhibitors provide basis anti- <br />                                             trypanosomal drug design. <br />  <br />                                             In order determine opportunity extends pathogens, reexamined <br />                                             HisRS sequences protozoa representative higher eukaryotes. In particular we <br />                                             looked presence cluster residues equivalent T. cruzi [Arg 165, Arg 334, Cys <br />                                             365, Val 366] human E. coli [Phe/Tyr, Gly, Glu, Arg] corresponding <br />                                             positions. We cluster single HisRS closely related trypanosomatid <br />                                             Leishmania major apicoplast-specific HisRS sequences various species the <br />                                             apicomplexan parasites Plasmodium Toxoplasma. However, cytosolic HisRS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             homologs identified apicoplexan genomes instead matched cluster residues <br />                                             human HisRS sequences, did sequences identified Entamoeba, Giardia, <br />                                             Cryptosporidia. It remains open question hypothetical drug specific the <br />                                             T. cruzi-like HisRS homologs apicoplast effective apicomplexans <br />                                             possessing second, cytosolic HisRS. <br />  <br />                                Summary <br />                                             The trypanosomal histidyl-tRNA synthetase structures presented illuminate differences <br />                                             eukaryotic prokaryotic HisRS homologs, highlight differences between <br />                                             trypanosomal human enzymes. The insertion domain histidyl-tRNA synthetase in <br />                                             eukary-otes seen adopt different three-dimensional structure reported for <br />                                             bacterial homologs. As insertion domain bacterial HisRS implicated the <br />                                             mechanisms tRNA binding aminoacylation10,11, leaves open question of <br />                                             equivalent eukaryotic mechanisms differ extent. It be <br />                                             informative structure eukaryotic HisRS:tRNAHis complex, date is <br />                                             lacking. We details substrate binding differ eukaryotic HisRS <br />                                             homologs. The substrate binding pocket accepts adenine ring ATP differs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             significantly character trypanosomal structures reported previously <br />                                             characterized bacterial archaeal structures. The surface binding pocket adjacent to <br />                                             adenine ring formed pair hydrophobic residues trypanosomes, Cys 365 and <br />                                             Val 366, replace arginine sequence previously studied homologs, both <br />                                             bacterial human. The previously assigned functions missing arginine taken over <br />                                             Arg 164, observe stack adenine ring, Arg 311, we <br />                                             infer coordinate ATP &#206;&#179;-phosphate. This peculiarity trypanosomes related <br />                                             protozoa, human HisRS sequences key regions like of <br />                                             bacteria trypanosomes. Whatever evolutionary basis difference, it <br />                                             constitutes possibly exploitable target identification selective anti-protozoan drugs <br />                                             targeting essential enzyme histidyl-tRNA synthetase. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                   Page 9 <br />  <br />  <br />                                Materials Methods <br />                                Choice expression constructs crystallization strategy <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Following standard MSGPP procedure, length truncated variants target T. <br />                                             brucei T. cruzi sequences selected expression. Based multiple sequence <br />                                             alignment HisRS sequence family, priority given truncation boundary <br />                                             extra N-terminal (WHEP) domain conserved HisRS core domains. Both <br />                                             variants proteins cloned E. coli expression vector AVA0421, derived <br />                                             vector pET14b 29. The AVA0421 vector contains N-terminal His-tag followed a <br />                                             protease 3C cleavage site, giving option purifying cleaved uncleaved protein for <br />                                             crystallization trials. In event, expression observed length T. cruzi variant. <br />                                             Both cleaved uncleaved forms length T. brucei variant subject to <br />                                             crystallization trials addition histidine co-crystallant, crystals <br />                                             obtained diffracted better 20 &#195;&#8230; resolution. Crystallization trials the <br />                                             uncleaved truncation variants screened crystallization histidine, and <br />                                             ATP. When crystals diffracting better 3 &#195;&#8230; obtained apo <br />                                             T. brucei histidine-bound T. cruzi proteins, focused optimizing than <br />                                             exploring additional truncation variants tag cleavage. <br />  <br />                                Protein production crystallography <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             T. brucei HisRS&#226;&#8364;&#8221;The sequence corresponding GenBank accession no. AAZ11768.1, <br />                                             coding residues 44&#226;&#8364;&#8220;477 477 residue T. brucei HisRS, PCR amplified from <br />                                             genomic DNA T. brucei strain TREU927 GUTat 10.1, cloned AVA0421 vector, <br />                                             expressed E. coli. Protein purified Ni-NTA affnity column followed XK <br />                                             26/60 Superdex 75 column (Amersham Pharmacia Biotech) using SGPP standard buffer <br />                                             (20mM HEPES, 0.5 M NaCl, 2 mM &#206;&#178;-mercaptoethanol, 5% glycerol, 0.025% NaN3 pH 7.5) <br />                                             30. Purified protein retained 22 residue expression tag. Crystals used data collection <br />                                             grown vapor diffusion sitting drops equilibrated reservoir containing <br />                                             2 M lithium sulfate, 0.1 M HEPES buffer pH 7.3, 1 mM TCEP. The initial crystallization <br />                                             drops consisted 0.2 &#206;&#188;l protein 22 mg/ml standard buffer plus 0.2 &#206;&#188;l reservoir <br />                                             solution. Crystals soaked 20% ethylene glycol prior frozen liquid nitrogen. <br />  <br />                                             Crystallographic diffraction images collected single crystal beamline 9&#226;&#8364;&#8220;2 the <br />                                             Stanford Synchrotron Radiation Lightsource processed using Mosflm31. An initial <br />                                             structural model available previous molecular replacement experiment using lower <br />                                             resolution data collected crystals different crystal form (data shown). This model <br />                                             generated program Phaser using monomer 35% sequence identical <br />                                             HisRS archaeote Thermoplasma acidophilum (PDB 1wu7) probe. However, this <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             relatively poor model did refine major portions later rebuilt using the <br />                                             higher resolution structure T. cruzi His:HisRS complex guide. <br />  <br />                                             T. cruzi HisRS&#226;&#8364;&#8221;The sequence corresponding residues 45&#226;&#8364;&#8220;478 GenBank sequence <br />                                             accession no. EAN89395.1, coding 478 residue T. cruzi HisRS, cloned from <br />                                             genomic DNA strain TCCL Brener expressed E. coli. Protein purified above. <br />                                             Purified protein retained 22 residue expression tag. Crystals grown vapor diffusion <br />                                             sitting drops equilibrated reservoir containing 0.2 M lithium sulfate, 0.1 M <br />                                             BisTris buffer pH 5.5, 26&#226;&#8364;&#8220;28% w/v PEG 3350, 1 mM TCEP, 10 mM histidine. The <br />                                             initial crystallization drops consisted 0.15 &#206;&#188;l protein 24 mg/ml standard buffer plus <br />                                             0.15 &#206;&#188;l reservoir solution. Crystals differing <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cells space groups found <br />                                             grow conditions. Diffraction data 1.8 &#195;&#8230; resolution single crystal <br />                                             His:HisRS complex belonging space group C2 collected SSRL beamline 9&#226;&#8364;&#8220; <br />                                             2. Integration scaling performed Mosflm31 Scala32. An initial molecular <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                              Page 10 <br />  <br />  <br />                                             replacement model generated BALBES33, used multiple fragments T. <br />                                             acidophilum homolog (1wu7) probes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             The histidyladenylate:HisRS complex (HAM:HisRS) generated allowing enzyme <br />                                             catalyze aminoacyl activation reaction <br />  <br />  <br />  <br />  <br />                                             crystals His:HisRS complex. The reaction buffer prepared adding 0.2 &#206;&#188;l 250 <br />                                             mM ATP standard buffer 0.2 &#206;&#188;l 250 mM MgCl2 standard buffer 5 &#206;&#188;l the <br />                                             crystallization reservoir solution neighboring containing slightly higher PEG <br />                                             concentration (30% w/v). Then 2 &#206;&#188;l reaction buffer added directly the <br />                                             crystallization drop containing grown crystals. Most crystals crumbled immediately, <br />                                             survived re-formed. These left re-equilibrate original reservoir <br />                                             solution 85 minutes frozen directly liquid nitrogen. Diffraction data from <br />                                             single crystal collected SSRL beamline 9-2, integrated scaled using <br />                                             HKL200034. The space group HAM:HisRS crystal different those <br />                                             determined unsoaked His:HisRS crystals (Table 2). The treated crystal diffracted to <br />                                             3 &#195;&#8230; resolution, substantial degradation diffraction quality untreated crystals. It <br />                                             unclear crystal represents minority crystal form better tolerated exposure <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             reaction buffer, morphed new crystal lattice reformed rapidly <br />                                             reaction-induced disruption original lattice. <br />  <br />                                             Crystallographic data handling project management used CCP4 program suite32,35. <br />                                             Structural superpositions performed using Coot SSM36. Refinement three <br />                                             structural models proceeded using alternating cycles refinement Refmac537 model <br />                                             building Coot38. Moderate non-crystallographic symmetry (NCS) restraints applied to <br />                                             apo (6 NCS-related chains) HAM:HisRS complex (2 NCS-related chains) models <br />                                             refinement. Model validation performed using Molprobity39. The refined <br />                                             crystallographic model structure included multi-segment model atomic <br />                                             displacements generated TLSMD40. Crystallographic statistics Protein Data Bank <br />                                             accession codes given Table 2. <br />  <br />                                Thermal melt assays <br />                                             Protein stability measured solution function temperature adding hydrophobic <br />                                             dye, Sypro Orange (SigmaAldrich). The dye emits low fluorescence aqueous solution <br />                                             surrounding well-folded protein gives increased fluorescent signal consequence of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             binding hydrophobic patches exposed protein releases bound ligand eventually <br />                                             denatures. In presence high-affnity ligand, protein general resistant <br />                                             thermal denaturation, resulting shift &#206;&#8221;TM inflection point melting <br />                                             curve41. The effect substrate binding trypanosomal histidyl-tRNA synthetases was <br />                                             assayed adding substrate potential ligands 0.5 mg/ml protein standard buffer <br />                                             monitoring fluorescence temperature range 20&#226;&#8364;&#8220;90&#194;&#176; C. The assay carried out <br />                                             96-well trays DNA Engine Opticon 2 RT-PCR machine (BioRad). Melting curve shifts <br />                                             measured T. cruzi T. brucei N-terminally truncated HisRS proteins used for <br />                                             structural studies, length HisRS trypanosomatid, Leishmania <br />                                             naiffi. <br />  <br />                                RNA interference T. brucei HisRS <br />                                             To confirm HisRS gene product essential, T. brucei gene subjected RNAi <br />                                             knockdown. The region gene coding sequence used RNAi selected <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                                Page 11 <br />  <br />  <br />                                               using RNA target selection program RNAit42 ensure significant <br />                                               sequence homology genes genome. The sequence T. brucei <br />                                               genomic DNA used construction RNAi plasmids taken GeneDB <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               accession number Tb927.6.2060. Bases 467 1097 amplified T. brucei 927 <br />                                               genomic DNA using primers 5&#226;&#8364;&#178;-ATGAGGCAATTACTCGTGGG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                               ATAACGCAATCACCAAAGCC-3&#226;&#8364;&#178;. The resulting amplicon ligated TA cloning into <br />                                               vector p2T7TABlue (a gift D. Horn, London School Hygiene Tropical Medicine) <br />                                               28, sequence confirmed nucleotide sequence analysis insert. The construct <br />                                               linearized NotI restriction enzyme (New England Biolabs, Ipswich, MA). T. <br />                                               brucei bloodstream-form parasites (provided G. Cross, Rockefeller University) expressing <br />                                               T7 RNA polymerase Tet repressor single selection marker (SM), G418 <br />                                               resistance, cultured HMI-9 medium 10% heat-inactivated fetal bovine serum and <br />                                               G418 2.5 &#206;&#188;g/ml 37&#194;&#176;C 5% CO2 atmosphere43. Ten micrograms linearized <br />                                               p2T7TABlue containing HisRS RNAi fragment electroporated mid-log-phase T. <br />                                               brucei (2.5 &#195;&#8212; 107 cells) suspended 500 &#206;&#188;l cytomix (120 mM KCl, 0.15 mM CaCl2, 10 <br />                                               mM K2HPO4-KH2PO4, pH 7.6, 25 mM HEPES, 2 mM disodium EDTA, 5 mM MgCl2). <br />                                               Electroporation 4-mm gap cuvettes 1.6 kV 24 &#206;&#169; resistance, described <br />                                               previously44. The cells resuspended HMI-9 medium supplemented 2.5 &#206;&#188;g/ml <br />                                               hygromycin 2.5 &#206;&#188;g/ml G418. Stable individual clones selected 5 7 days after <br />                                               transfection. Selected cultures diluted 1 &#195;&#8212; 105 cells/ml induced express double- <br />                                               stranded RNA addition 1 &#206;&#188;g/ml tetracycline. Parasite growth measured period <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               8 days following induction RNAi cultures passed daily 1:2 1:20 dilution as <br />                                               needed, monitored ATPLite luminescence ATP detection assay (catalog <br />                                               no. 6016941; Perkin-Elmer, Waltham, MA). For quantification gene knockdown, cDNA <br />                                               prepared messenger RNA collected 0h, 24h, 48h RNAi induction with <br />                                               tetracycline. mRNA signal knock-down analyzed quantitative PCR cDNA using <br />                                               primers 5&#226;&#8364;&#178;-TCAATTAAGGCGTCTGATAGCG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                               TGTCGTGCAAGTGATGGTGTC-3&#226;&#8364;&#178;, amplified separate region gene that <br />                                               used RNAi construct. The amplified products 23 PCR cycles were <br />                                               quantified densitometry (normalized &#206;&#178;-tubulin). <br />  <br />                                Acknowledgments <br />                                               This work funded NIAID award P01AI067921 (Medical Structural Genomics Pathogenic Protozoa). We <br />                                               thank Angela Criswell Rigaku Americas Corporation collecting X-ray data used initial characterization of <br />                                               crystals T. cruzi HisRS. Portions research carried Stanford Synchrotron Radiation <br />                                               Lightsource, national user facility operated Stanford University behalf U.S. Department Energy, <br />                                               Office Basic Energy Sciences. <br />  <br />  <br />                                Abbreviations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS          histidyl-tRNA synthetase <br />                                             HAM            histidyl-adenosinemonophosphate (histidyladenylate) <br />                                             aaRS           aminoacyl-tRNA synthetase <br />  <br />                                References <br />                                               1. Sheppard K, Yuan J, Hohn MJ, Jester B, Devine KM, Soll D. From amino acid another: tRNA- <br />                                                  dependent amino acid biosynthesis. Nucleic Acids Res 2008;36:1813&#226;&#8364;&#8220;25. [PubMed: 18252769] <br />                                               2. Hurdle JG, O&#226;&#8364;&#8482;Neill AJ, Chopra I. Prospects aminoacyl-tRNA synthetase inhibitors new <br />                                                  antimicrobial agents. Antimicrob Agents Chemother 2005;49:4821&#226;&#8364;&#8220;33. [PubMed: 16304142] <br />                                               3. Fan E, Baker D, Gelb MH, Buckner FS, Van Voorhis WC, Phizicky E, Dumont M, Mehlin C, Grayhack <br />                                                  EJ, Sullivan M, Verlinde CL, DeTitta G, Meldrum D, Merritt EA, Earnest TN, Soltis M, Zucker F, <br />  <br />  <br />                                                      J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                        Page 12 <br />  <br />  <br />                                                Myler P, Schoenfeld L, Kim D, Worthey EA, LaCount D, Vignali M, Li J, Mondal S, Massey A, <br />                                                Carroll B, Gulde S, Luft JR, DeSoto L, Holl M, Caruthers JM, Bosch J, Robien MA, Arakaki T, Holmes <br />                                                MA, LeTrong I, Hol WG. Structural genomics pathogenic protozoa: An overview. Methods in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Molecular Biology 2008;426:497&#226;&#8364;&#8220;513. [PubMed: 18542886] <br />                                             4. Charriere F, O&#226;&#8364;&#8482;Donoghue P, Helgadottir S, Marechal-Drouard L, Cristodero M, Horn EK, Soll D, <br />                                                Schneider A. Dual Targeting tRNAAsp Requires Two Different Aspartyl-tRNA Synthetases in <br />                                                Trypanosoma brucei. J Biol Chem 2009;284:16210&#226;&#8364;&#8220;7. [PubMed: 19386587] <br />                                             5. Charriere F, Helgadottir S, Horn EK, Soll D, Schneider A. Dual targeting single tRNA(Trp) requires <br />                                                different tryptophanyl-tRNA synthetases Trypanosoma brucei. Proc Natl Acad Sci U S A <br />                                                2006;103:6847&#226;&#8364;&#8220;52. [PubMed: 16636268] <br />                                             6. Wolf YI, Aravind L, Grishin NV, Koonin EV. Evolution aminoacyl-tRNA synthetases&#226;&#8364;&#8220;analysis of <br />                                                unique domain architectures phylogenetic trees reveals complex history horizontal gene <br />                                                transfer events. Genome Res 1999;9:689&#226;&#8364;&#8220;710. [PubMed: 10447505] <br />                                             7. Hawko SA, Francklyn CS. Covariation specificity-determining structural motif aminoacyl- <br />                                                tRNA synthetase tRNA identity element. Biochemistry 2001;40:1930&#226;&#8364;&#8220;6. [PubMed: 11329259] <br />                                             8. Sankaranarayanan R, Dock-Bregeon AC, Romby P, Caillet J, Springer M, Rees B, Ehresmann C, <br />                                                Ehresmann B, Moras D. The structure threonyl-tRNA synthetase-tRNA(Thr) complex enlightens <br />                                                repressor activity reveals essential zinc ion active site. Cell 1999;97:371&#226;&#8364;&#8220;81. [PubMed: <br />                                                10319817] <br />                                             9. Yaremchuk A, Tukalo M, Grotli M, Cusack S. A succession substrate induced conformational <br />                                                changes ensures amino acid specificity Thermus thermophilus prolyl-tRNA synthetase: <br />                                                comparison histidyl-tRNA synthetase. J Mol Biol 2001;309:989&#226;&#8364;&#8220;1002. [PubMed: 11399074] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             10. Aberg A, Yaremchuk A, Tukalo M, Rasmussen B, Cusack S. Crystal structure analysis the <br />                                                  activation histidine Thermus thermophilus histidyl-tRNA synthetase. Biochemistry <br />                                                  1997;36:3084&#226;&#8364;&#8220;94. [PubMed: 9115984] <br />                                             11. Guth E, Farris M, Bovee M, Francklyn CS. Asymmetric Amino Acid Activation Class II Histidyl- <br />                                                  tRNA Synthetase Escherichia coli. J Biol Chem 2009;284:20753&#226;&#8364;&#8220;62. [PubMed: 19487703] <br />                                             12. Rho SB, Lee KH, Kim JW, Shiba K, Jo YJ, Kim S. Interaction human tRNA synthetases <br />                                                  involves repeated sequence elements. Proc Natl Acad Sci U S A 1996;93:10128&#226;&#8364;&#8220;33. [PubMed: <br />                                                  8816763] <br />                                             13. Jia J, Arif A, Ray PS, Fox PL. WHEP domains direct noncanonical function glutamyl-Prolyl tRNA <br />                                                  synthetase translational control gene expression. Mol Cell 2008;29:679&#226;&#8364;&#8220;90. [PubMed: <br />                                                  18374644] <br />                                             14. Mathews MB, Bernstein RM. Myositis autoantibody inhibits histidyl-tRNA synthetase: model for <br />                                                  autoimmunity. Nature 1983;304:177&#226;&#8364;&#8220;9. [PubMed: 6866113] <br />                                             15. Cahuzac B, Berthonneau E, Birlirakis N, Guittet E, Mirande M. A recurrent RNA-binding domain is <br />                                                  appended eukaryotic aminoacyl-tRNA synthetases. EMBO J 2000;19:445&#226;&#8364;&#8220;52. [PubMed: <br />                                                  10654942] <br />                                             16. O&#226;&#8364;&#8482;Hanlon TP, Raben N, Miller FW. A novel gene oriented head-to-head configuration the <br />                                                  human histidyl-tRNA synthetase (HRS) gene encodes mRNA predicts polypeptide <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  homologous HRS. Biochem Biophys Res Commun 1995;210:556&#226;&#8364;&#8220;66. [PubMed: 7755634] <br />                                             17. O&#226;&#8364;&#8482;Hanlon TP, Miller FW. Genomic organization, transcriptional mapping, evolutionary <br />                                                  implications human bi-directional histidyl-tRNA synthetase locus (HARS/HARSL). Biochem <br />                                                  Biophys Res Commun 2002;294:609&#226;&#8364;&#8220;14. [PubMed: 12056811] <br />                                             18. Brindefalk B, Viklund J, Larsson D, Thollesson M, Andersson SGE. Origin evolution the <br />                                                  mitochondrial aminoacyl-tRNA synthetases. Mol Biol Evol 2007;24:743&#226;&#8364;&#8220;56. [PubMed: 17182897] <br />                                             19. Arnez JG, Augustine JG, Moras D, Francklyn CS. The step aminoacylation atomic level <br />                                                  histidyl-tRNA synthetase. Proc Natl Acad Sci U S A 1997;94:7144&#226;&#8364;&#8220;9. [PubMed: 9207058] <br />                                             20. Qiu X, Janson CA, Blackburn MN, Chhohan IK, Hibbs M, Abdel-Meguid SS. Cooperative structural <br />                                                  dynamics novel fidelity mechanism histidyl-tRNA synthetases. Biochemistry <br />                                                  1999;38:12296&#226;&#8364;&#8220;304. [PubMed: 10493797] <br />                                             21. Arnez JG, Harris DC, Mitschler A, Rees B, Francklyn CS, Moras D. Crystal structure histidyl- <br />                                                  tRNA synthetase Escherichia coli complexed histidyl-adenylate. EMBO J 1995;14:4143&#226;&#8364;&#8220; <br />                                                  55. [PubMed: 7556055] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                       Page 13 <br />  <br />  <br />                                             22. Shiba K. Intron positions delineate evolutionary path pervasively appended peptide five <br />                                                 human aminoacyl-tRNA synthetases. J Mol Evol 2002;55:727&#226;&#8364;&#8220;33. [PubMed: 12486531] <br />                                             23. Raben N, Nichols R, Dohlman J, McPhie P, Sridhar V, Hyde C, Le3 R, Plotz P. A motif human <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 histidyl-tRNA synthetase shared aminoacyl-tRNA synthetases coiled- <br />                                                 coil essential enzymatic activity contains major autoantigenic epitope. J Biol Chem <br />                                                 1994;269:24277&#226;&#8364;&#8220;83. [PubMed: 7523371] <br />                                             24. Cole C, Barber JD, Barton GJ. The jpred 3 secondary structure prediction server. Nucleic acids <br />                                                 research 2008;36:W197&#226;&#8364;&#8220;201. [PubMed: 18463136] <br />                                             25. Pollastri G, McLysaght A. Porter: new, accurate server protein secondary structure prediction. <br />                                                 Bioinformatics 2005;21:1719&#226;&#8364;&#8220;20. [PubMed: 15585524] <br />                                             26. Guth EC, Francklyn CS. Kinetic discrimination tRNA identity conserved motif 2 loop a <br />                                                 class II aminoacyl-tRNA synthetase. Mol Cell 2007;25:531&#226;&#8364;&#8220;42. [PubMed: 17317626] <br />                                             27. Guth E, Connolly SH, Bovee M, Francklyn CS. A substrate-assisted concerted mechanism for <br />                                                 aminoacylation class II aminoacyl-tRNA synthetase. Biochemistry 2005;44:3785&#226;&#8364;&#8220;94. [PubMed: <br />                                                 15751955] <br />                                             28. Alibu VP, Storm L, Haile S, Clayton C, Horn D. A doubly inducible RNA interference <br />                                                 rapid RNAi plasmid construction Trypanosoma brucei. Mol Biochem Parasitol 2005;139:75&#226;&#8364;&#8220; <br />                                                 82. [PubMed: 15610821] <br />                                             29. Alexandrov A, Vignali M, LaCount DJ, Quartley E, deVries C, Rosa DD, Babulski J, Mitchell SF, <br />                                                 Schoenfeld LW, Fields S, Hol WG, Dumont ME, Phizicky EM, Grayhack EJ. A facile method for <br />                                                 high-throughput co-expression protein pairs. Mol Cell Proteomics 2004;3:934&#226;&#8364;&#8220;938. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 15240823] <br />                                             30. Mehlin C, Boni E, Buckner FS, Engel L, Feist T, Gelb M, Haji L, Kim D, Liu C, Mueller N, Myler <br />                                                 PJ, Reddy JT, Sampson JN, Subramanian E, Van Voorhis WC, Worthey E, Zucker F, Hol WGJ. <br />                                                 Heterologous expression proteins plasmodium falciparum: results 1000 genes. <br />                                                 Molecular Biochemical Parasitology 2006;148:144&#226;&#8364;&#8220;160. [PubMed: 16644028] <br />                                             31. Leslie AGW. Recent changes mosflm package processing film image plate data. Joint <br />                                                 CCP4 ESF-EACBM Newsletters Protein Crystallography 1992;26:x. <br />                                             32. Collaborative Computational Project No. 4. The ccp4 suite: programs protein crystallography. <br />                                                 Acta Crystallographica Section D 1994;50:760&#226;&#8364;&#8220;763. <br />                                             33. Long F, Vagin AA, Young P, Murshudov GN. BALBES: molecular-replacement pipeline. Acta <br />                                                 Crystallographica Section D 2008;64:125&#226;&#8364;&#8220;132. <br />                                             34. Otwinowski, Z.; Minor, W. Processing X-ray diffraction data collected oscillation mode. In: <br />                                                 Carter, CW., Jr; Sweet, RM., editors. Methods Enzymol. Part A. Vol. 276. Academic Press; New <br />                                                 York: 1997. p. 307-326.Macromolecular Crystallography <br />                                             35. Potterton L, McNicholas S, Krissinel E, Gruber J, Cowtan K, Emsley P, Murshudov GN, Cohen S, <br />                                                 Perrakis A, Noble M. Developments ccp4 molecular-graphics project. Acta Crystallographica <br />                                                 Section D 2004;60:2288&#226;&#8364;&#8220;94. <br />                                             36. Krissinel E, Henrick K. Secondary-structure matching (SSM), new tool fast protein structure <br />                                                 alignment dimensions. Acta Crystallographica Section D 2004;60:2256&#226;&#8364;&#8220;2268. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             37. Murshudov GN, Vagin AA, Dodson EJ. Refinement macromolecular structures maximum- <br />                                                 likelihood method. Acta Crystallographica Section D 1997;53:240&#226;&#8364;&#8220;255. <br />                                             38. Emsley P, Cowtan K. Coot: model-building tools molecular graphics. Acta Crystallographica <br />                                                 Section D 2004;60:2126&#226;&#8364;&#8220;32. <br />                                             39. Lovell S, Davis I, Arendall WB III, deBakker P, Word J, Prisant M, Richardson J, Richardson D. <br />                                                 Structure validation C&#206;&#177; geometry: &#207;&#8224;, &#207;&#710; C&#206;&#178; deviation. Proteins: Structure, Function, and <br />                                                 Genetics 2003;50:437&#226;&#8364;&#8220;450. <br />                                             40. Painter J, Merritt EA. Optimal description protein structure terms multiple groups undergoing <br />                                                 tls motion. Acta Crystallographica Section D 2006;62:439&#226;&#8364;&#8220;450. <br />                                             41. Lo MC, Aulabaugh A, Jin G, Cowling R, Bard J, Malamas M, Ellestad G. Evaluation fluorescence- <br />                                                 based thermal shift assays hit identification drug discovery. Anal Biochem 2004;332:153&#226;&#8364;&#8220;9. <br />                                                 [PubMed: 15301960] <br />                                             42. Redmond S, Vadivelu J, Field MC. RNAit: automated web-based tool selection RNAi <br />                                                 targets Trypanosoma brucei. Mol Biochem Parasitol 2003;128:115&#226;&#8364;&#8220;8. [PubMed: 12706807] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                   Page 14 <br />  <br />  <br />                                             43. Hirumi H, Hirumi K. Continuous cultivation Trypanosoma brucei blood stream forms medium <br />                                                 containing low concentration serum protein feeder cell layers. J Parasitol 1989;75:985&#226;&#8364;&#8220; <br />                                                 9. [PubMed: 2614608] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             44. Wirtz E, Leal S, Ochatt C, Cross GA. A tightly regulated inducible expression conditional <br />                                                 gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol Biochem Parasitol <br />                                                 1999;99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br />                                             45. Eriani G, Delarue M, Poch O, Ganglo3 J, Moras D. Partition tRNA synthetases classes <br />                                                 based mutually exclusive sets sequence motifs. Nature 1990;347:203&#226;&#8364;&#8220;6. [PubMed: 2203971] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. The dimer T. cruzi HisRS bound histidyladenylate <br />                                             Monomer A shown dark brown (catalytic domain), yellow (anticodon binding domain), <br />                                             light brown (insertion domain). Monomer B similarly shown blue. The active site of <br />                                             monomer occupied histidyladenylate, product half-reaction catalyzed <br />                                             enzyme. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                              Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. Sequence alignment HisRS T. brucei T. cruzi human cytosolic (HARS) <br />                                             mitochondrial (HARS2) homologs, E. coli homolog <br />                                             The conserved characteristic Class II aaRS catalytic domain motifs defined Eriani <br />                                             et al.45 indicated sequence, conserved histidine recognition motifs <br />                                             specific HisRS. Secondary structure elements observed T. cruzi His:HisRS structure <br />                                             shown sequences. The &#206;&#177;-helices N-terminal WHEP domain seen in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             NMR studies human eukaryotic homologs indicated human HARS2 <br />                                             sequence. This domain included expression constructs trypanosomal <br />                                             proteins studied here. The region E. coli sequence highlighted orange indicates a <br />                                             portion insertion domain structurally equivalent bacterial archaeal/ <br />                                             eukaryotic structures. Residues interacting directly histidyladenylate T. cruzi <br />                                             structure indicated blue letters sequence: H/P/S/A signify interaction the <br />                                             histidyl/phosphate/sugar/adenine moieties respectively. The red P residue 334 indicates <br />                                             inferred interaction &#206;&#179;-phosphate ATP. Points relevant sequence difference <br />                                             trypanosomal human sequences highlighted cyan. These residues <br />                                             contribute difference shape hydrophobic character binding surface in <br />                                             active site. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 3. The insertion domain eukaryotic bacterial HisRS structures <br />                                             All HisRS contain additional domain inserted motifs 2 3 conserved <br />                                             Class II catalytic domains. However, insertion domains eukaryotic bacterial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS homologs similar sequence structure. The figures generated by <br />                                             superimposing T. cruzi (3hrk) T. thermophilus (1ady) dimers separating their <br />                                             respective insertion domains pure translation horizontal. The grey helix lower <br />                                             left structure structurally equivalent element following insertion domain. <br />                                             (a) The T. cruzi insertion domain, residues 210&#226;&#8364;&#8220;285, followed helix &#206;&#177; 13 conserved <br />                                             catalytic domain. (b) The T. thermophilus insertion domain, residues 164&#226;&#8364;&#8220;235, followed by <br />                                             homologous catalytic domain helix. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 4. Superposition apo HisRS histidyladenylate:HisRS complex <br />                                             The protein undergoes substantial conformational change binding histidine. The apo <br />                                             form shown light colors; histidyladenylate:HisRS complex shown dark colors. <br />                                             In superposition apo form T. brucei histidine-bound form from <br />                                             T. cruzi; however, sequence identity trypanosomal HisRS 84% believe <br />                                             highlighted conformational change attributed histidine binding than <br />                                             sequence differences. The apo form little affnity ATP. The rearrangement residues <br />                                             314&#226;&#8364;&#8220;317, His A motif, bind histidine acts reposition Arg314 oriented <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             interact &#206;&#177;-phosphate ATP. Other structural elements undergo substantial <br />                                             conformational shift. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 5. Effect substrate binding thermal melt curves <br />                                             (a) Fluorescence function temperature. Each curve normalized maximum <br />                                             peak height. Two replicate curves shown experimental condition. (b) Shift in <br />                                             TM, defined change inflection point melting curve presence ligand, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             relative inflection point melting curve protein ligand. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                    Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                             Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 6. Difference electron density active sites T. cruzi HAM:HisRS complex <br />                                             (a) Site A. Difference density contoured 2.4&#207;&#402; (grey cages) 4.5&#207;&#402; (green cages) a <br />                                             sigma-A weighted omit map. (b) Site B. Difference density contoured 2.0&#207;&#402; 3.6&#207;&#402; a <br />                                             sigma-A weighted omit map. The view angle slightly different (a) order more <br />                                             clearly better density histidyl moiety. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                    Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                         Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 7. Adenine binding pocket <br />                                             (a) The binding mode histidyladenylate seen previously structures T. <br />                                             thermophilus (1ady) E. coli HisRS (1kmm, shown). One face adenine ring is <br />                                             stacked sidechain Arg 311. (b) The binding mode histidyladenylate seen <br />                                             structure T. cruzi HisRS (3hrk). The dot surface surrounds hydrophobic residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cys 365 Val 366, occur sequence position equivalent bacterial residue <br />                                             Arg 311. In trypanosomal HisRS, adenine ring stacked instead sidechain <br />                                             Arg 164, extends opposite face binding pocket. Residue 334, visible <br />                                             left figure, far away interact directly adenine ring, is <br />                                             hypothesized coordinate &#206;&#179;-phosphate ATP. <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 8. RNAi silencing T. brucei HisRS <br />                                             (a) Growth bloodstream T. brucei cells culture 1 day intervals following RNAi <br />                                             induction. RNA interference HisRS induced tetracycline (TCN +) compared <br />                                             cells RNAi induction (TCN &#226;&#710;&#8217;). The growth rate compared &#226;&#8364;&#732;single <br />                                             marker&#226;&#8364;&#8482; (SM) strain T. brucei transfected RNAi expression vector. <br />                                             Cumulative cell densities shown log-scale product cell number the <br />                                             total dilution. Another clone observed nearly identical phenotype (data not <br />                                             shown). (b) Knockdown T. brucei HisRS mRNA 0, 24, 48 hours induction of <br />                                             RNAi tetracycline. PCR products 23 cycles imaged staining agarose gel <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                        Page 25 <br />  <br />  <br />                                             ethidium bromide. Even loading demonstrated PCR &#206;&#178;-tubulin gene. The <br />                                             intensity bands 24h 48h 42% 71% 0h control, respectively, as <br />                                             measured densitometry. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                                                 Page 26 <br />  <br />  <br />  <br />  <br />                                                                                                  Table 1 <br />                            Residues involved ATP binding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                T. cruzi (3HRK)                             E. coli (1KMN)       human HARS(2) <br />  <br />                             H-bond adenine N6               Glu 158                                     Glu 114              Asp(Glu) 159 <br />  <br />                             sandwiches Adenine ring            His 168                                     Phe 125              Phe 171 <br />                                                                Arg 164 (not structurally homologous)       Arg 311              Arg 388 <br />  <br />                             coordinates &#206;&#177; phosphate            Arg 156                                     Arg 113              Arg 157 <br />                                                                Arg 314                                     Arg 259              Arg 326 <br />  <br />                             coordinates &#206;&#178; phosphate            Arg 156 (inferred)                          Arg 113              Arg 157 <br />  <br />                             coordinates &#206;&#179; phosphate            Arg 164 (inferred)                          Arg 121              Arg 167 <br />                                                                Arg 334 (not structurally homologous)       Arg 311              Arg 388 <br />  <br />                             provides hydrophobic surface       Cys 365 Val 366                             equivalent        equivalent <br />  <br />                            The residues tabulated T. cruzi observed HisRS:HAM complex, coordination &#206;&#178;- &#206;&#179;-phosphates inferred. <br />                            The residue numbering bacterial HisRS E. coli sequence. No structure available human HisRS, set residues shown, <br />                            human HARS HARS2 sequences align exactly E. coli sequence (Figure 2). Residue numbering human HARS. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                            J Mol Biol. Author manuscript <br /> </body></html>