<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Biochemistry                                                                                                                         BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> The FF domains yeast U1 snRNP protein Prp40 mediate <br /> interactions Luc7 Snu71 <br /> Claudia Ester1 Peter Uetz*1,2 <br />  <br /> Address: 1Forschungszentrum Karlsruhe, Institute Toxicology Genetics, P. O. Box 3640, D-76021 Karlsruhe, Germany 2J Craig Venter <br /> Institute (JCVI), 9704 Medical Center Drive, Rockville, MD 20850, USA <br /> Email: Claudia Ester - claudia.ester@itg.fzk.de; Peter Uetz* - uetz@jcvi.org <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 11 November 2008                                                   Received: 4 June 2008 <br />                                                                               Accepted: 11 November 2008 <br /> BMC Biochemistry 2008, 9:29   doi:10.1186/1471-2091-9-29 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2091/9/29 <br /> &#194;&#169; 2008 Ester Uetz; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: The FF domain conserved eukaryotes usually acts adaptor <br />                  module RNA metabolism transcription. Saccharomyces cerevisiae encodes FF domain <br />                  proteins, Prp40, component U1 snRNP, Ypr152c, protein unknown function. The <br />                  structure Prp40, relationship proteins U1 snRNP, precise function <br />                  remain little understood. <br />                  Results: Here investigated essentiality interaction properties FF domains <br />                  yeast Prp40. We C-terminal FF domains Prp40 dispensable. Deletion <br />                  additional FF domains lethal. The FF domain Prp40 binds U1 protein Luc7 yeast <br />                  two-hybrid GST pulldown experiments. FF domains 2 3 bind Snu71, known U1 <br />                  protein. Peptide array screens identified binding sites FF1-2 Snu71 (NDVHY) FF1 <br />                  Luc7 (&#207;&#8226;[FHL] &#195;&#8212; [KR] &#195;&#8212; [GHL] &#207;&#8226; hydrophobic amino acid). <br />                  Conclusion: Prp40, Luc7, Snu71 appear form subcomplex yeast U1snRNP. <br />                  Our data suggests N-terminal FF domains critical interactions. Crystallization <br />                  Prp40, Luc7, Snu71 failed far co-crystallization pairs tri-complex <br />                  facilitate crystallographic functional analysis. <br />  <br />  <br />  <br />  <br /> Background                                                                     In yeast essential U1snRNP component Prp40 plays <br /> Spliceosome assembly yeast occurs stepwise man-                        important role bringing 5&apos; 3&apos; splice sites into <br /> ner U1, U2, U4/U6 U5 snRNPs binding sequen-                           spatial proximity intron spliced of <br /> tially pre-mRNA other. The defined                       pre-mRNA [1]. Ito et al[11] Prp40 interact <br /> step building commitment complex yeast                        Snu71 &quot;core&quot; yeast two-hybrid (Y2H) <br /> or E complex metazoans U1snRNP binds                          data. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> knowledge, direct interaction between <br /> initially 5&apos; splice site. The metazoan U1snRNP con-                     Prp40 Luc7 reported proteins <br /> tains proteins U1-A, U1-70K U1-C. The yeast                            complex multiple times (e.g. <br /> U1snRNP contains homologs proteins,                               [12,13]). <br /> Mud1, Snp1 Yhc1, seven additional pro- <br /> teins [1-10].                                                                  Prp40 modular protein consisting pair WW <br />                                                                                domains followed series FF domains (Fig. 1). <br />  <br />  <br />                                                                                                                                        Page 1 11 <br />                                                                                                                  (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                         http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br />  <br />              A                               Preys:        Snu71                   Luc7 <br />  <br />                                                       wt    I        II       wt     I         II <br />  <br />                                              FF1 <br />  <br />                                              FF2 <br />  <br />                                              FF3 <br />                         Prp40 domain baits <br />  <br />  <br />  <br />  <br />                                              FF4 <br />  <br />                                              FF1-2 <br />  <br />                                              FF2-3 <br />  <br />                                              FF3-4 <br />  <br />                                              FF1-3 <br />  <br />  <br />  <br />              B                                              Yeast U1snRNP proteins <br />                                                                                                                            II <br />  <br />                                                              Snu71                                                        PWI <br />                                                              Mud1                                             x <br />                                                                                                                   I <br />                                                              Prp39                       FF1-3 <br />                                                              Prp42 <br />                                                              Prp40        W    W   F1    F2         F3        F4 <br />                                                              Snp1                  FF1 <br />                                                              Snu56 <br />                                                              Nam8 <br />                                                              Yhc1                         II <br />                                                                                                x <br />                                                              Luc7 <br />                                                                                I <br />  <br />  <br /> FigureFF1 domains 1&#226;&#8364;&#8220;3 bind Snu71 I FF domain 1 binds Luc7 II <br /> Prp40 <br /> Prp40 FF domains 1&#226;&#8364;&#8220;3 bind Snu71 I FF domain 1 binds Luc7 II. Binding sites FF domains Luc7 and <br /> Snu71 roughly mapped Yeast-2-hybrid assays. A, Two-hybrid assay testing binding Prp40 FF domain constructs <br /> (FF1, FF2, FF3; FF4, FF1-2, FF2-3, FF3-4, FF1-3) full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> clones fragments Snu71 Luc7 (for domains frag- <br /> ments panel B). Each encircled colony indicates two-hybrid interaction (weak interactions hatched circles). B, Sche- <br /> matic overview Prp40 FF domain interactions (arrows) U1snRNP proteins Snu71 Luc7 context the <br /> U1snRNP illustrated human U1snRNP (modified [9]). Note structural model human U1snRNP <br /> does include proteins studied here. The dashed line extending FF1 domain indicates length <br /> Snu71 interacts FF2-3 Snu71-I interacts domains FF1-3. <br />  <br />  <br />  <br />  <br />                                                                                                                                 Page 2 11 <br />                                                                                                          (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br /> FF domains described 1999 Bedford &amp;          Saccharomyces cerevisiae Gal4-activation domain <br /> Leder domains 60 amino acids including        fusions (preys [25]). These screens resulted two <br /> conserved Phenylalanines (F) domain       interaction partners, Snu71 Luc7, known <br /> named [14]. Typically FF domains occur tandem arrays.      components U1 snRNP [12,13,26]. No interaction <br /> For example, 78 metazoan FF domain proteins         partners identified full-length Ypr152c <br /> listed SMART database [15], 13 2 3 FF          protein, FF domain bait. <br /> domains, 4 6. The structure of <br /> three FF domains solved recently: FF1            Once Prp40 identified interacting bait, cloned <br /> domain human protein HYPA/FBP11, FF1               FF domains combinations thereof tested these <br /> domain yeast protein Prp40, single FF          baits fragments Snu71 Luc7 preys (see <br /> domain Urn1/Ypr152c [16-18]. FF domain proteins            methods domain definitions). These experiments <br /> can classified families: p190 Rho GTPase-       showed FF1 domain Prp40 binds C-termi- <br /> related proteins [19] WW/FF family mem-         nal fragment Luc7 (Luc7 II, Fig. 1). Similarly, C-termi- <br /> bers contain WW domains followed sev-          nal fragment Snu71 (Snu71 I) binds Prp40 FF <br /> eral FF domains. Three proteins family      domains 1&#226;&#8364;&#8220;3. Interestingly, length Snu71 not <br /> known recognize phosphorylated C-terminal              Snu71-I interacts FF2-3 domain fragment (Fig. <br /> domain (CTD) RNA polymerase II FF            1). <br /> domains, human transcription elongation fac- <br /> tor CA150, human splicing factor FBP11 yeast      We screened isolated domains combinations <br /> splicing factor Prp40 [16,20-22]. Furthermore, FF             thereof baits genome-wide prey array. How- <br /> domains CA150 use multiple independent             ever, new proteins way: FF1-4 inter- <br /> binding sites bind cooperatively proteins   acted Snu71 Luc7, FF1-3 FF2-3 <br /> such transcription splicing associated factor      interacted Snu71 Luc7 screens <br /> Tat-SF1 [23]. The FF1 domain Prp40 known inter-      (data shown). <br /> act Clf1 (Crooked neck-like factor), essential and <br /> well conserved multifunctional protein [24]. The role      These results encouraged revisit protein topology <br /> the second yeast FF protein, Urn1/Ypr152c, con-         U1 snRNP. We tested U1snRNP associated pro- <br /> tains WW domain FF domain, splic-        teins systematic pairwise yeast two-hybrid tests using <br /> ing factor [18].                                          full-length bait prey constructs the <br />                                                               previously detected interactions Prp40-Snu71 Prp40- <br /> Here investigated role FF domains     Luc7. <br /> binding specificity. More specifically, study aims to <br /> complement data order define consensus          To verify yeast two-hybrid experiments inde- <br /> binding site FF domains, sites known    pendent method performed GST pulldowns. Different <br /> other domains SH3 domain (whose consensus         FF domains expressed GST fusion proteins E. <br /> binding site &quot;proline-rich&quot;). We addressed problem    coli purified glutathione sepharose beads. Surpris- <br /> by combination genome-wide yeast two-hybrid              ingly, protein constructs containing FF3 domain con- <br /> screens, in-vitro binding assays, peptide arrays,         sistently failed expressed significant amounts. We <br /> genetic experiments.                                          conclude FF3 domain renders FF3, FF2-3, FF1- <br />                                                               3 FF1-4 constructs insoluble unstable. <br /> We FF domains Prp40         Nevertheless, Snu71 Luc7 translated 35S- <br /> are essential explain behaviour interac-    labeled vitro incubated remaining GST-FF <br /> tions components yeast U1 snRNP.        fusion proteins. These experiments clearly showed that <br /> The binding site significantly differs previ-   Luc7 binds specifically FF1 possibly FF2 <br /> ously published binding sites FF domains          domains Prp40 binding FF2 signfi- <br /> implicates FF domains interaction modules       cantly weaker FF1 (Fig. 2). Unfortunately, the <br /> a wide range specificities, stark contrast     interaction Prp40-FF1-3 Snu71 be <br /> domains SH3 PDZ domains.                           confirmed way involved inaccessible FF3 <br />                                                               domain. <br /> Results <br /> Prp40 interacts Luc7 Snu71 FF domains        FF domains bind specific peptides Luc7 Snu71 <br /> To investigate interaction properties yeast FF         The two-hybrid mapping experiments did provide any <br /> domains, screened Prp40 Urn1/Ypr152c          information precise binding sites Prp40 <br /> well isolated FF domain baits genome       FF domains Luc7 Snu71. In order map the <br /> wide yeast two-hybrid arrays containing ORFs       binding sites Prp40 FF domains synthesized <br />  <br />  <br />                                                                                                                   Page 3 11 <br />                                                                                             (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                 http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br />  <br />                                                                  GST-Prp40-                                        GST <br />  <br />                        L      I-Luc7               FF1 FF2 FF4 FF1-2 FF3-4 <br />  <br />   66 kDa <br />   45 kDa <br />  <br />   31 kDa <br />  <br />  <br />  <br />  <br />    Luc7 <br />  <br />  <br /> Figure <br /> The FF12domain Prp40 binds Luc7 vitro <br /> The FF1 domain Prp40 binds Luc7 vitro. Various GST-Prp40FF domain fusion proteins (FF1, FF2; FF4, FF1-2, <br /> FF3-4) tested Luc7 binding vitro. Top panel shows Coomassie blue-stained GST fusion proteins, row <br /> shows autoradiography bound Luc7 protein. Luc7 bound construct containing FF1-2 domains FF1 <br /> domain alone, barely FF2 alone. I, input; L, ladder (molecular mass marker). <br />  <br />  <br /> overlapping 15-mer peptides Luc7 Snu71 nitro-        ing motif generalized &#207;&#8226;[FHL] &#195;&#8212; [KR] &#195;&#8212; [GHL] <br /> cellulose membranes probed GST-FF                &#207;&#8226; hydophobic amino acid (Fig. 3E). <br /> domain fusions (Fig. 3). The peptides used covered         Interestingly, Luc7 binding site showed similarity <br /> regions Luc7 (amino acids 93&#226;&#8364;&#8220;261) Snu71 (amino                   interaction   region     (peptide    B1: <br /> acids 329&#226;&#8364;&#8220;536) shown interact Y2H assays       H418LANDVHYDHHRSFK432) Snu71 was <br /> (Fig. 1). These screens identified peptide C-termi-   obtained approach (Fig. 3A). A consistent <br /> nal half Luc7, D214RRLADHFLGKIHLG228 (A24),            interaction region (N421DVHY) narrowed four <br /> appeared primary interactor FF1 domain        overlapping peptides detected Snu71 II was <br /> (Figs. 3B&#226;&#8364;&#8220;D). Although results shown Figures 3A&#226;&#8364;&#8220;B       synthesised CAPE-membrane 20-mer overlapping <br /> used fusion FF1-2 explicit goal project    peptides 3 amino acid shifts probed GST- <br /> to identify targets single FF domains. Hence follow-    FF1-2. <br /> ing experiments (Figs. 3C&#226;&#8364;&#8220;D) carried FF1 as <br /> this domain appeared contribute binding (Fig.       Only FF1 FF2 domains essential yeast Prp40 <br /> 2). The A24 peptide characterized systematic            Although two-hybrid mapping experiments indicated <br /> alanine scans (Fig. 3C) substitution analysis (Fig. 3D)    FF domains Prp40 impor- <br /> to define critical amino acids interaction    tant ones, physiological role remained unclear. To <br /> FF1 domain Prp40. This analysis clearly showed          determine physiological role FF domains in <br /> importance FLGKIH motif binding &#226;&#8364;&#8220; Gly-         Prp40 deleted FF domains starting from <br /> cine-223 Histidine-226 important           C-terminus vivo (Fig. 4). In mutant strain the <br /> binding. Based substitution analysis interact-     deleted FF domain replaced proteinA/kanMX6 <br />  <br />  <br />                                                                                                                   Page 4 11 <br />                                                                                             (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                                                 http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br />  <br />                                         Snu71-I peptides probed GST-FF1-2 <br />                                A <br />                                         1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 <br />  <br />                                    A <br />                                    B <br />                                    C                                                                               controls <br />  <br />  <br />  <br />  <br />                                B        Luc7-II peptides probed GST-FF1-2 <br />                                         1   2   3   4   5   6   7   8   9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 <br />  <br />                                    A <br />                                    B <br />  <br />  <br />  <br />                                C        Luc7 peptides, probed GST-FF1 <br />                                                                                             E                   Luc7-peptide sequence <br />                                wt D R R L A D H F L G K I H L G                                                 F L G K I H L G <br />                                                                                                             Y     Y     Y           Y <br />                                                                                                             W   W W W W W W         W   W <br />                                                                                                             V           V           V <br />                                D       Luc7 peptide sequence (           = wildtype)                        T     T T   T T         T T <br />                                    D R R L A D H F L G K I H L G                                            S     S S   S S         S S <br />                                Y                                                                            R     R R R R R         R R <br />                                W                                                                            Q     Q Q   Q           Q Q <br />                                V                                                                            P     P P   P P           P <br />                                T <br />                                                                                               amino acids <br />  <br />  <br />                                                                                                             N     N N   N N         N N <br />                                S <br />                                                                                                             M     M M   M M         M M <br />                                R <br />                                                                                                             L   L L       L         L L <br />                                Q <br />                                P                                                                            K         K <br />                                N                                                                            I   I       I           I <br />                  amino acids <br />  <br />  <br />  <br />  <br />                                M                                                                            H     H H     H         H H <br />                                L                                                                            G     G G G G G         G G <br />                                K                                                                            F   F F                 F F <br />                                I                                                                            E <br />                                H                                                                            D                           D <br />                                G                                                                            C <br />                                F <br />                                                                                                             A     A           A A A A <br />                                E <br />                                D                                                                                  F             G <br />                                C                                                                                1 H x K x H x x <br />                                                                                                                 _ <br />                                A                                                                                  L     R       L <br />  <br /> Figure <br /> FF  domains <br />          3 Prp40 bind specific peptides Snu71 Luc7 <br /> FF domains Prp40 bind specific peptides Snu71 Luc7. A, Peptides Snu71-I (aa 329&#226;&#8364;&#8220;536) Luc7- <br /> II (aa 93&#226;&#8364;&#8220;261) synthesised 15 mers 3 aa shifts &#206;&#178;-alanine membrane. A control peptide (QRALAKDLIVPRRP, <br /> position C17) binds &#206;&#177;-GST antibody. Incubation GST-FF1-2 showed strong signal peptide B1 A24 (panel <br /> B). B, Peptide scan A Luc7 peptides probed GST-FF1-2. Control membranes probed GST were <br /> negative (data shown). C, Alanine-scan peptide B (position A24, wildtype sequence: DRRLADHFLG- <br /> KIHLG). Letters mark amino acids replaced alanine. GST-FF1 served probe. The important amino <br /> acid interaction shown bold. D, Substitution analysis Luc7 peptide sequence C, probed GST-FF1. <br /> Each residue sequence substituted 20 naturally occurring L-amino acids. All spots circles represent the <br /> wild-type amino acids. All spots single substitution analogs, rows defining sequence position substi- <br /> tuted columns defining amino acid replaces wild-type residue. E, The Luc7 motif bound FF1 domain of <br /> Prp40 D. &#207;&#8226; stands hydrophobic amino acids, green indicates hydrophobic, yellow indicates polar, blue indicates <br /> basic orange indicates acidic amino acids. White letters black background represent wildtype amino acids the <br /> sequence. <br />  <br />  <br />  <br />                                                                                                                                                      Page 5 11 <br />                                                                                                                                (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                   http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br /> cassette selection deletion mutants easy protein     We showed FF1 domain Prp40 responsible <br /> detection (Fig. 4A). We unable isolate deletion       binding Luc7 region FF1-3 binds to <br /> of FF domain 4 assumed essential. How-        length Snu71 C-terminal fragment <br /> ever, deletion FF domains 3&#226;&#8364;&#8220;4 (FF&#206;&#8221;3-4ProA)           Snu71-I. The fragment containing FF2-3 binds full- <br /> viable albeit showed clear growth defect com-         length Snu71 Snu71-I. This suggests bind- <br /> pared control constructs (FF1-4ProA, i.e. wildtype       ing cooperative regions outside Snu71-I <br /> Prp40 Protein A fused C-terminus            present additional binding sites. However, single <br /> wildtype control strain PJ69-4&#206;&#177;) (Fig. 4B). Deletions FF     binding site indicates strong site <br /> domains 2&#226;&#8364;&#8220;4 1&#226;&#8364;&#8220;4 appear lethal         N421DVHY motif. In case, results con- <br /> not isolate mutants. These results confirmed ini-      firm different FF domains clearly different bind- <br /> tial two-hybrid findings indicated FF1 FF2       ing specificities FF1-2 region business <br /> are critical Prp40 FF3 FF4             end Prp40 [17]. <br /> increase activity and/or stability. <br />                                                                 The ability FF domains bind non-phosphorylated <br /> FF&#206;&#8221;3&#226;&#8364;&#8220;4 deletions inhibit splicing DBP2            peptides refutes suggestion FF domain an <br /> ECM33                                                           exclusively phospho-peptide binding domain. <br /> Because Prp40 well-established component U1 <br /> snRNP, reasoned <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> probably        The FF domain binding sites identified study <br /> a defect splicing. To investigate question, tested   Luc 7 Snu71 share obvious similarity <br /> splicing efficiency selected Saccharomyces cerevisiae    previously identified binding motifs (Table 1). Sev- <br /> intron-containing genes, DBP2 ECM33,              eral studies showed FF domains bind phospho- <br /> commonly used splicing reporters. However, RT-PCR            peptides, usually phospho-CTD RNA polymerase II <br /> assays RNAs genes revealed splicing      (Table 1). Similarly, FF domains CA150 were <br /> defect FF&#206;&#8221;3&#226;&#8364;&#8220;4 mutant (data shown).                   shown bind acidic peptides. However, this <br />                                                                 true Prp40 FF domains 2&#226;&#8364;&#8220;3 FF1 domain of <br /> Discussion                                                      protein appears prefer basic residues. This finding <br /> Although composition the yeast U1 snRNP          agreement acidic nature Prp40-FF1 (bind- <br /> been known decade [12,13] precise interactions        ing basic peptides) basic FF2 FF3 domains <br /> among components atomic structure remain          bind acidic Snu71 peptide. Unfortu- <br /> elusive. Equally puzzling fact human U1         nately little information derive reliable <br /> snRNP commonly assumed consist 10 proteins             consensus binding sites. Similarly, available 3D structures <br /> (U1-A, U1-70K U1-C plus seven Lsm proteins (in              FF domains bound cognant ligands not <br /> addition snRNAs) [9,27] yeast counterpart      sufficient derive rules allow predict binding <br /> contains 18 proteins [12,13]. In species U1-A,       sites generally. Clearly, structural work is <br /> U1-70K U1-C (Mud1, Snp1, Yhc1 yeast) form              required understand binding mode FF domains. <br /> complex associated heptameric Sm protein <br /> ring complex [9]. Except Mud1-Snp1-Yhc1 core            Murphy et al. [31] investigated functional role of <br /> complex Lsm ring little information available        motifs Prp40, including putative RNA-binding <br /> about structure yeast U1 snRNP. Most complex         domains &quot;region 1&quot; &quot;region 2&quot; which <br /> purification studies large complexes 10 51 pro-      overlap FF domains (Figure 4C). While <br /> teins individual U1 components used baits           deletion region 1 lethal, deletion region 2 resulted <br /> [10,28]. Here suggest Prp40, Luc7, Snu71            slow growth phenotype. However, Murphy et al. were <br /> form subcomplex U1 snRNP. Surprisingly,            able RNA-binding &quot;domains&quot; and <br /> we did interaction U1snRNP        likely similarity RNA-binding <br /> when systematically tested pairwise interactions         domains spurious. The fact overlap signifi- <br /> among known 10 subunits, suggesting limitations        cantly FF domains supports notion. Never- <br /> of yeast two-hybrid requirement           theless, deletion region 2 partially deletes FF2 <br /> RNA prevented detection interactions. Many          domain. It possible truncated FF2 domains <br /> the U1snRNP associated proteins contain RNA binding             residual binding activity shows a <br /> domains known bind RNA directly [29,30].             &quot;slow growth&quot; defect. We tested this <br /> Interestingly, did known Prp40 inter-       partial deletion FF2 binds Snu71 given the <br /> actor, Clf1 [24], initial genome-wide yeast two-         non-essential role Snu71, binding abso- <br /> hybrid screen. However, subsequent verification          lutely required U1 function. <br /> pertinent array position revealed Clf1 ORF was <br /> missing prey array. <br />  <br />  <br />                                                                                                                      Page 6 11 <br />                                                                                                (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                         http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br />  <br />                A            construct                    Prp40FF deletion                            phenotype <br />  <br />  <br />                             FF1-4 ProA    WW       WW    FF1   FF2          FF3         FF4   ProA         vital <br />  <br />  <br />                             1 FF4 ProA    WW       WW    FF1   FF2          FF3   ProA                    lethal ? <br />  <br />  <br />  <br />                         1 FF3-4 ProA <br />                                           WW       WW    FF1   FF2   ProA                            Slow growth <br />  <br />  <br />                                            WW      WW    FF1   ProA                                       lethal ? <br />                         1 FF2-4 ProA <br />  <br />  <br />  <br />  <br />                B                                   growth defect <br />                                                                                               wt control PJ69-42 <br />                                                                                               FF1-4ProA <br />  <br />  <br />  <br />                                                                                               1 FF3-4 ProA <br />                    OD 600 <br />  <br />  <br />  <br />  <br />                                                     time (min) <br />  <br />                C <br />                                                 132-188 201-257 ~280 ~345 348-359 355-413            493-552 <br />  <br />  <br />                                W W                 F1     F2                       F3                 F4 <br />                                                                  NES1 NES2 ARM <br />                                                     Region 1 Region 2 <br />                                                     175-218 235-281 <br />                                Mutant phenotype:    1:lethal 1:slow growth 36&#194;&#176;ts   wt     33&#194;&#176; ts <br />  <br />  <br /> Figure <br /> Only  Prp40 <br />         4 FF domains 1&#226;&#8364;&#8220;2 essential vivo <br /> Only Prp40 FF domains 1&#226;&#8364;&#8220;2 essential vivo. A, A strain Protein A fused full-length Prp40 (FF1-4ProA) <br /> shows signficant difference growth wild-type. FF domain deletions C-terminus insertion Protein A <br /> into respective FF domain locus resulted viable mutant lacks FF domains 3&#226;&#8364;&#8220;4. However, replacements FF <br /> domains 4 3&#226;&#8364;&#8220;4 lethal. B, The mutant lacking FF domains 3&#226;&#8364;&#8220;4 viable shows a <br /> growth defect compared wild-type controls. C, Summary domains functional motifs Prp40. Numbers are <br /> amino acid sequence positions. Regions 1 2 similar RNA-binding domains. These motifs, NES 1 and <br /> 2 (nuclear export signals) arginine-rich motif (ARM) deleted [&#206;&#8221;] mutated Murphy et al. [31]. The phe- <br /> notypes mutations shown (ts = temperature-sensitive). Note predicted coiled-coil region indicated <br /> as bar FF2 FF3 A shown known function. <br />  <br />  <br />  <br />  <br />                                                                                                                             Page 7 11 <br />                                                                                                       (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                                http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br /> Table 1: Binding sites FF domains <br />  <br />  FF protein        FF domain          Target protein           Binding site                                                        Ref. <br />  <br />  Prp40             FF1                Clf1                     GSTNIDILDLEELREYQRRKRTEYEGYLKRNRLD                                  [17] <br />  Prp40             WW+FF1-2           RNAPII-CTD               YpSPTpSPS                                                           [22]*** <br />  Prp40             FF1                Luc7                     &#207;&#8224;[FHL] &#195;&#8212; [KR] &#195;&#8212; [GHL] = FLGKIHLG                                    This study <br />  Prp40             FF2-3              Snu71                    NDVHY                                                               This study <br />  CA150             FF1,2,3**          Tat-SF1                  (D/E)2/5-F/W/Y-(D/E)2/5 *                                           [23,21] <br />  CA150             FF5                RNAP-CTD                 YpSPTpSPS                                                           [20] <br />  FBP11             FF1                RNAP-CTD                 YpSPTpSPS                                                           [16] <br />  Rho-GAP           FF1-4              TFII-I                   (N-terminal 90 amino acids)                                         [15] <br />  <br />  * i.e., residues preceding following aromatic residue negatively charged glutamic acid asparagine. <br />  ** FF1, FF2, FF3 bound similar affinity. <br />  *** interaction required WW FF domains. Neither able bind CTD itself (pS indicates phospho-Serine). <br />  <br /> The role Snu71 remains elusive too. Newo et al. [32]                   dispensable, convey considerable growth disadvan- <br /> have analyzed U1 snRNP Schizosaccharomyces pombe                   tage absent. We conclude assist with <br /> and homolog Snu71 homologs                        spliceosome assembly activity. This reflected by <br /> Prp40 Luc7 clearly present. However, spec-                  evolutionary conservation 4 or, 5, FF <br /> ulate Usp107p functional homolog                         domains homologous proteins. We suggest our <br /> Snu71, given similar size presence PWI                   observations help characterize U1 snRNP <br /> domain Usp107p.                                                        structurally suggest previous crystallization <br />                                                                           efforts failed Luc7 Snu71 expressed <br /> While Prp40 does bind conserved sequence                      individually crystallization attempts included only <br /> Snu71, binding site Luc7 highly conserved. As                 individual proteins. We speculate Luc7 be <br /> more rigorous confirmation specificity Prp40                crystallized Prp40 fragments thereof. Fur- <br /> and FF domains, interesting compare                  ther studies required derive general rules physi- <br /> the peptide-binding specificities available domains                ological FF domain functions activities. <br /> with cognate peptides comparable conditions. <br /> Similarly, vivo relevance interactions                      Methods <br /> described elucidated detailed struc-                2.1. Plasmids Strains <br /> tural analysis <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> interaction epitopes                pOBD2 pOAD [36] low-copy plasmids used to <br /> vivo.                                                                     express fusions Gal4-Gal4 activation domain (AD) <br />                                                                           (preys) Gal4 DNA binding domain (DBD) (baits). <br /> For detailed understanding U1 protein function, crys- <br /> tal structures individual proteins or, preferably,             Construction Gal4DBD-ORF fusions performed <br /> whole complex, required. The structure                 means PCR recombination [37] described in <br /> tell FF domains Prp40 bind 3-dimen-                  [25]. Transformation performed using lithium <br /> sional epitopes linear motifs. Since membrane-                     acetate procedure [38]. Bait constructs transformed <br /> bound peptides used study folded                 yeast strain PJ69-4&#206;&#177; [36] prey constructs into <br /> in native protein, produce artifactual results.              PJ69-4a [39]. <br />  <br /> This study provide new insight                       For Y2H mapping experiments FF domain constructs from <br /> important role Prp40 mediator transcrip-                  Prp40 baits Snu71 prey produced <br /> tion splicing. While Prp40 consistently                      using following primers (forward primers shown as <br /> shown component U1 snRNP, precise                      codons): ForwardFF1: A TTC CAG CTG ACC ACC ATG <br /> role splicing remains unclear. Similarly, mechanis-                AGA AGG ACT AAA GAA GAA, ReverseFF1: GA TCC CCG <br /> tic details involvement transcription require addi-             GGA ATT GCC ATG TGT TTC ATT GTG TTC CT, <br /> tional data. Several publications indicate direct                       ForwardFF2: A TTC CAG CTG ACC ACC ATG AAG GAA <br /> connection specific steps transcription                    CAC AAT GAA ACA, ReverseFF2: GAT CCC CGG GAA TTG <br /> mRNA-processing eukaryotes, i. e. co-transcriptional                   CCA TGG ATT CTT TCT GAG TGT CG, ForwardFF3: A TTC <br /> mRNA-processing [33-35].                                                  CAG CTG ACC ACC ATG AAT TAT ACC AGA GAC CGT, <br />                                                                           ReverseFF3: ATC CCC GGG AAT TGC CAT GAC GTC TGT <br /> Conclusion                                                                TGG GCT ATT G, ForwardFF4: A TTC CAG CTG ACC ACC <br /> Our results FF domains 1 2 critical                 ATG CAA AAT GAG CGT AGG ATA, ReverseFF4: GAT CCC <br /> Prp40 function. However, FF domains 3 4                     CGG GAA TTG CCA TGC GCT TTC GGC AGT CGG, <br />  <br />  <br />                                                                                                                                    Page 8 11 <br />                                                                                                              (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                  http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br /> ForwardSnu71II: AA TTC CAG CTG ACC ACC ATG TCC                  pled reticulocyte presence [35S] methio- <br /> GAG AGA AGC GCG GCA GAG, ReverseSnu71II: GAT                    (Promega, Madison, WI). PCR primers as <br /> CCC CGG GAA TTG CCA TGC TCT GCC GCG CTT CTC                     follows: Luc7p Forward: GGATCCTAATACGACTCAC- <br /> TCG GA, ForwardSnu71I: AA TTC CAG CTG ACC ACC                   TATAG GGAAACAGCCACCATGTCAACTATGTCAACGC <br /> ATG GCC AAA GGG AGC GCC AAT ACA, ReverseSnu71I:                 CT, Luc7p Reverse: CTA CAC AAA GCG TCT TCC GGG; <br /> GAT CCC CGG GAA TTG CCA TGT GTA TTG GCG CTC                     Snu71p Forward: ATCCTAATACGACTCACTATAG GGAA <br /> CCT TTG GC. PCR reactions using primers resulted          ACAGCCACCATGAGGGAT ATTGTATTTGTA, Snu71p <br /> in following Gal4-DBD and/or GST fusion constructs          Reverse: TCA GGT CCC CAA GCG AAA TTC. <br /> (all based wildtype sequences http://www.yeast <br /> genome.org): amino acids 1-583 (Prp40wt), 1-75                  GST-FF domain fusion proteins GST cou- <br /> (Prp40WW1-2); 129&#226;&#8364;&#8220;560 (Prp40FF1-4); 129&#226;&#8364;&#8220;428                     pled glutathione-Sepharose beads (Amersham Pharma- <br /> (Prp40FF1-3);     129&#226;&#8364;&#8220;264    (Prp40FF1-2);    196&#226;&#8364;&#8220;428           cia Biotech) incubated 4 &#206;&#188;l vitro-translated <br /> (Prp40FF2-3);     351&#226;&#8364;&#8220;560    (Prp40FF3-4);    129&#226;&#8364;&#8220;201           proteins pulldown buffer (40 mM Hepes pH 8.0; 2.5 <br /> (Prp40FF1); 196&#226;&#8364;&#8220;264 (Prp40FF2); 351&#226;&#8364;&#8220;428 (Prp40FF3);             mM MgCl2; 0,1 mM EDTA; 1 mM DTT; 1 mM PMSF; 0,2% <br /> 487&#226;&#8364;&#8220;560 (Prp40FF4); 1&#226;&#8364;&#8220;620 (Snu71 wt); 530&#226;&#8364;&#8220;620                   Triton-X-100; 100 mM NaCl) 2 h 4&#194;&#176;C rota- <br /> (Snu71I); 329&#226;&#8364;&#8220;536 (Snu71 II).                                   tion. Beads bound proteins washed times <br />                                                                 (for 10 min rotation 4&#194;&#176;C) pulldown buffer <br /> Luc7 constructs created using existing restriction sites   proteins harvested SDS-sample buffer, separated by <br /> within open reading frame (Luc7I: StuI, DNA                 SDS-PAGE, analyzed autoradiography. <br /> position429; Luc7II, EcoRI DNA position 274). <br />                                                                 2.5. Peptide SPOT synthesis <br /> Constructs expressing Gal4DBD- Gal4AD-ORF               Cellulose membrane-bound peptides prepared <br /> fusions verified DNA sequencing.                        according standard SPOT synthesis protocols [41,42] <br />                                                                 using automated Spot synthesizer (MultiPep, Intavis <br /> 2.2. Two-Hybrid Screens, Retests Mapping                    AG Bioanalytical Instruments, K&#195;&#182;ln, Germany); Fmoc <br /> Experiments                                                     derivatives amino-acids peptide synthesis were <br /> An array containing ~6,000 Saccharomyces cer-       obtained Novabiochem. The generated peptide <br /> evisiae ORFs expressed Gal4AD fusions used            arrays synthesized amino-PEG membranes <br /> screen interacting proteins. Haploid transformants          (Intavis AG Bioanalytical Instruments, K&#195;&#182;ln, Germany) or <br /> expressing full-length Gal4DBD-ORF fusion pro-         &#206;&#178;-alanine membranes. &#206;&#178;-alanine membranes pro- <br /> tein Gal4DBD-FF domain construct fusion protein            duced using hardened low ash whatman 50 paper incu- <br /> were mated array [36]. The resulting diploids       bated night Fmoc-&#206;&#178;-alanine. After Fmoc- <br /> pinned Biomek 2000 Laboratory Automation                 deprotection membranes spotted Fmoc-&#206;&#178;- <br /> Workstation (Beckman-Coulter, Fullerton, CA)               alanine-Opfp (Bachem AG, Switzerland) spacer. <br /> selective media described in [36]. Positive prey <br /> clones first-round screen re-arrayed       2.6. SPOT Membrane Probing <br /> tested single tests reproducibility. Deletion con-       After activation membranes methanol the <br /> structs mapping yeast two-hybrid tests             membrane-bound peptide arrays blocked 3 h in <br /> obtained PCR digestion compatible          blocking buffer (2% milk powder 5% sucrose Tris- <br /> restriction enzymes (see section 2.1). These constructs         buffered saline (TBS), pH 8.0) incubated over- <br /> were tested preys existing Prp40 FF bait       night 4&#194;&#176;C 10 &#206;&#188;g/ml purified GST fusion protein or <br /> constructs using standard Y2H protocols used       GST control protein blocking buffer. After washing <br /> for genome-wide screens.                                    times TBS membranes probed with <br />                                                                 anti-GST antibody (G1160; Sigma-Aldrich, M&#195;&#188;nchen, <br /> 2.3. Protein Expression                                         Germany) blocking buffer 0.2% Tween 3 h. <br /> GST GST-Prp40 constructs expressed                  The membrane washed times TBS then <br /> Escherichia coli BL21 purified using glutathione            incubated 1.5 h horseradish-peroxidase-coupled <br /> Sepharose 4B beads (Amersham Pharmacia, Uppsala,                anti-mouse mAb (Sigma-Aldrich; M&#195;&#188;nchen, Germany) in <br /> Sweden) described [40].                                   blocking buffer 0.2% Tween followed washing <br />                                                                 times TBS. Analysis quantification pep- <br /> 2.4. GST pull assays                                       tide-bound GST fusion proteins carried using <br /> Modified primers Luc7 Snu71, containing T7            ECL (Amersham Biosciences, Freiburg, Germany). <br /> promotor eukaryotic translation initiation site      QRALAKDLIVPRRP known bind GST used as <br /> used generate PCR products use TNT&#226;&#8222;&#162;-cou-        positive control. <br />  <br />  <br />  <br />                                                                                                                    Page 9 11 <br />                                                                                              (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                                     http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br /> 2.7. In vivo FF-domain Deletion                                                References <br /> In order delete FF domain vivo PCR products                     1.    Abovich N, Rosbash M: Cross-intron bridging interactions in <br />                                                                                      yeast commitment complex conserved mammals. <br /> created containing protein A/kanMX6 cassette                              Cell 1997, 89(3):403-412. <br /> pYM8-plasmid [43] recombined                               2.    K&#195;&#164;ufer NF, Potashkin J: Analysis splicing machinery fis- <br /> Prp40 locus replaced endogenous FF                                    sion yeast: comparison budding yeast mammals. <br />                                                                                      Nucleic Acids Res 2000, 28(16):3003-3010. <br /> domains shown Fig. 4A. PCR-constructs trans-                        3.    Gornemann J, Kotovic KM, Hujer K, Neugebauer KM: Cotranscrip- <br /> formed PJ69-4&#206;&#177; strain plated geniticin                               tional spliceosome assembly occurs stepwise fashion and <br /> plates [200 mg/l] selection. Genomic DNA pre-                                requires cap binding complex. Mol Cell 2005, 19(1):53-63. <br />                                                                                4.    Seraphin B, Rosbash M: Mutational analysis interactions <br /> pared geniticin positive colonies correct                               U1 small nuclear RNA pre-mRNA yeast. <br /> deletion verified PCR. PCR-products recombina-                                Gene 1989, 82(1):145-151. <br />                                                                                5.    Seraphin B, Rosbash M: Identification functional U1 snRNA- <br /> tion prepared using following primers:                                      pre-mRNA complexes committed spliceosome assembly <br /> Prp40ProA: Forward, GCG TCA AAA AAG AGG CAT TTA                                      splicing. Cell 1989, 59(2):349-358. <br /> ACT CCG GCT GTG GAA TTG GAC TAT CGT ACG CTG                                    6.    Seraphin B, Rosbash M: The yeast branchpoint sequence not <br />                                                                                      required formation stable U1 snRNA-pre-mRNA <br /> CAG GTC GAC, Reverse, ATA ATT TAT ATA ATG ATT AAC                                    complex recognized absence U2 snRNA. <br /> AAG ATA GAG GTC GAC ACG TCA GAA ATC GAT GAA                                          Embo J 1991, 10(5):1209-1216. <br /> TTC GAG CTC G; &#206;&#8221;FF4ProA: Forward, AGA AAC GAA                                  7.    Michaud S, Reed R: A functional association 5&apos; and <br />                                                                                      3&apos; splice site established earliest prespliceosome <br /> AAG ATA CAA CAG AAA CTC CAA AAT GAG CGT AGG                                          complex (E) mammals. Genes Dev 1993, 7(6):1008-1020. <br /> ATA CGT ACG CTG CAG GTC GAC Reverse, ATA ATT TAT                               8.    Michaud S, Reed R: An ATP-independent complex commits <br />                                                                                      pre-mRNA mammalian spliceosome assembly path- <br /> ATA ATG ATT AAC AAG ATA GAG GTC GAC ACG TCA                                          way. Genes Dev 1991, 5(12B):2534-2546. <br /> GAA ATC GAT GAA TTC GAG CTC G; &#206;&#8221;FF3-4ProA: For-                                9.    Stark H, Dube P, L&#195;&#188;hrmann R, K&#195;&#164;stner B: Arrangement RNA <br /> ward, CTT CAA AAC AAA CTA AAT GAG CTC CGA CTG                                        proteins spliceosomal U1 small nuclear ribonucle- <br />                                                                                      oprotein particle. Nature 2001, 409(6819):539-542. <br /> CGC AAT TAT ACC AGA CGT ACG CTG CAG GTC GAC;                                   10.   Stevens SW, Ryan DE, Ge HY, Moore RE, Young MK, Lee TD, Abel- <br /> Reverse, ATA ATT TAT ATA ATG ATT AAC AAG ATA GAG                                     son J: Composition functional characterization the <br />                                                                                      yeast spliceosomal penta-snRNP. Mol Cell 2002, 9(1):31-44. <br /> GTC GAC ACG TCA GAA ATC GAT GAA TTC GAG CTC G;                                 11.   Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, Sakaki Y: A compre- <br /> &#206;&#8221;FF2-4ProA: Forward, CTT TCC AAT AGA TCA GCC GAT                                     hensive two-hybrid analysis explore yeast protein <br /> CAA CTT CTT AAG GAA CAC AAT GAA CGT ACG CTG                                          interactome. Proc Natl Acad Sci USA 2001, 98(8):4569-4574. <br />                                                                                12.   Neubauer G, Gottschalk A, Fabrizio P, Seraphin B, L&#195;&#188;hrmann R, Mann <br /> CAG GTC GAC, Reverse, ATA ATT TAT ATA ATG ATT AAC                                    M: Identification proteins yeast U1 small nuclear <br /> AAG ATA GAG GTC GAC ACG TCA GAA ATC GAT GAA                                          ribonucleoprotein complex mass spectrometry. Proc Natl <br /> TTC GAG CTC G. Control primers verification                               Acad Sci USA 1997, 94(2):385-390. <br />                                                                                13.   Gottschalk A, Tang J, Puig O, Salgado J, Neubauer G, Colot HV, Mann <br /> successful recombination: Forward, GGA CGA ACT ATA                                   M, Seraphin B, Rosbash M, L&#195;&#188;hrmann R, et al.: A comprehensive <br /> AAC GAG (Prp40 specific bp 322&#226;&#8364;&#8220;339); Reverse, GTC GAC                                biochemical genetic analysis yeast U1 snRNP <br />                                                                                      reveals novel proteins. Rna 1998, 4(4):374-393. <br /> CTG CAG CGT ACG (pYM8 specific S3R primer [43].                                14.   Bedford MT, Leder P: The FF domain: novel motif often <br />                                                                                      accompanies WW domains.                   Trends Biochem Sci 1999, <br /> To determine growth rate FF deletion strains cell                         24(7):264-265. <br />                                                                                15.   Letunic I, Copley RR, Pils B, Pinkert S, Schultz J, Bork P: SMART 5: <br /> densities (OD600) normalized measured                                  domains context genomes networks. Nucleic <br /> hourly 540 min.                                                                 Acids Res 2006:D257-260. <br />                                                                                16.   Allen M, Friedler A, Schon O, Bycroft M: The structure FF <br />                                                                                      domain human HYPA/FBP11.                       J Mol Biol 2002, <br /> Authors&apos; contributions                                                               323(3):411-416. <br /> PU conceived study. PU CE performed yeast                         17.   Gasch A, Wiesner S, Martin-Malpartida P, Ramirez-Espain X, Ruiz L, <br />                                                                                      Macias MJ: The structure Prp40 FF1 domain interac- <br /> two-hybrid screens. CE mapped interaction domains, car-                              tion crn-TPR1 motif Clf1 gives new insight into <br /> ried pull-downs, SPOT-synthesis mapping                              binding mode FF domains.                    J Biol Chem 2006, <br /> experiments. CE vivo FF domain dele-                                281(1):356-364. <br />                                                                                18.   Bonet R, Ramirez-Espain X, Macias MJ: Solution structure the <br /> tions characterized phenotype. CE PU wrote                             yeast URN1 splicing factor FF domain: Comparative analysis <br /> the paper. All authors read approved final manu-                             charge distributions FF domain structures-FFs and <br />                                                                                      SURPs, domains similar fold. Proteins 2008, <br /> script.                                                                              73(4):1001-9. <br />                                                                                19.   Jiang W, Sordella R, Chen GC, Hakre S, Roy AL, Settleman J: An FF <br /> Acknowledgements                                                                     domain-dependent protein interaction mediates signaling <br />                                                                                      pathway growth factor-induced gene expression. Mol Cell <br /> This project initiated laboratory Stan Fields, University           2005, 17(1):23-35. <br /> Washington, Seattle. We like thank Simone Reber help         20.   Carty SM, Goldstrohm AC, Sune C, Garcia-Blanco MA, Greenleaf AL: <br /> cloning FF domains, Christiane Landgraf peptide spotting, Nils Johns-         Protein-interaction modules organize nuclear function: <br /> son providing vectors Tanja Kuhn technical assistance. This              FF domains CA150 bind phosphoCTD RNA <br />                                                                                      polymerase II. Proc Natl Acad Sci USA 2000, 97(16):9015-9020. <br /> work funded J Craig Venter Institute DFG grant Ue 50/2       21.   Goldstrohm AC, Albrecht TR, Sune C, Bedford MT, Garcia-Blanco <br /> PU.                                                                                  MA: The transcription elongation factor CA150 interacts <br />                                                                                      RNA polymerase II pre-mRNA splicing factor <br />                                                                                      SF1. Mol Cell Biol 2001, 21(22):7617-7628. <br />  <br />  <br />  <br />  <br />                                                                                                                                         Page 10 11 <br />                                                                                                                    (page number citation purposes) <br />  BMC Biochemistry 2008, 9:29                                                                    http://www.biomedcentral.com/1471-2091/9/29 <br />  <br />  <br />  <br /> 22.   Morris DP, Greenleaf AL: The splicing factor, Prp40, binds the <br />       phosphorylated carboxyl-terminal domain RNA polymer- <br />       ase II. J Biol Chem 2000, 275(51):39935-39943. <br /> 23.   Smith MJ, Kulkarni S, Pawson T: FF domains CA150 bind tran- <br />       scription splicing factors multiple weak interac- <br />       tions. Mol Cell Biol 2004, 24(21):9274-9285. <br /> 24.   Chung S, McLean MR, Rymond BC: Yeast ortholog Dro- <br />       sophila crooked neck protein promotes spliceosome assem- <br />       bly stable U4/U6.U5 snRNP addition. Rna 1999, <br />       5(8):1042-1054. <br /> 25.   Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, Knight JR, Lock- <br />       shon D, Narayan V, Srinivasan M, Pochart P, et al.: A comprehen- <br />       sive     analysis         protein-protein       interactions   in <br />       Saccharomyces cerevisiae. Nature 2000, 403(6770):623-627. <br /> 26.   Fortes P, Bilbao-Cortes D, Fornerod M, Rigaut G, Raymond W, <br />       Seraphin B, Mattaj IW: Luc7p, novel yeast U1 snRNP protein <br />       role 5&apos; splice site recognition. Genes Dev 1999, <br />       13(18):2425-2438. <br /> 27.   Hochleitner EO, K&#195;&#164;stner B, Fr&#195;&#182;hlich T, Schmidt A, L&#195;&#188;hrmann R, <br />       Arnold G, Lottspeich F: Protein stoichiometry multiprotein <br />       complex, human spliceosomal U1 small nuclear ribonu- <br />       cleoprotein: absolute quantification using isotope-coded <br />       tags mass spectrometry.                    J Biol Chem 2005, <br />       280(4):2536-2542. <br /> 28.   Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, <br />       Schultz J, Rick JM, Michon AM, Cruciat CM, et al.: Functional organ- <br />       ization yeast proteome systematic analysis pro- <br />       tein complexes. Nature 2002, 415(6868):141-147. <br /> 29.   Puig O, Gottschalk A, Fabrizio P, Seraphin B: Interaction U1 <br />       snRNP nonconserved intronic sequences affects 5&apos; <br />       splice site selection. Genes Dev 1999, 13(5):569-580. <br /> 30.   Zhang D, Rosbash M: Identification proteins cross- <br />       link pre-mRNA yeast commitment complex. Genes <br />       Dev 1999, 13(5):581-592. <br /> 31.   Murphy MW, Olson BL, Siliciano PG: The yeast splicing factor <br />       Prp40p contains functional leucine-rich nuclear export sig- <br />       nals essential splicing. Genetics 2004, 166(1):53-65. <br /> 32.   Newo AN, L&#195;&#188;tzelberger M, Bottner CA, Wehland J, Wissing J, J&#195;&#164;nsch <br />       L, K&#195;&#164;ufer NF: Proteomic analysis U1 snRNP of <br />       Schizosaccharomyces pombe reveals essential organ- <br />       ism-specific proteins. Nucleic Acids Res 2007, 35(5):1391-1401. <br /> 33.   Ares M Jr, Proudfoot NJ: The spanish connection: transcription <br />       mRNA processing closer.                      Cell 2005, <br />       120(2):163-166. <br /> 34.   Lin KT, Lu RM, Tarn WY: The WW domain-containing proteins <br />       interact early spliceosome participate pre- <br />       mRNA splicing vivo. Mol Cell Biol 2004, 24(20):9176-9185. <br /> 35.   Proudfoot NJ, Furger A, Dye MJ: Integrating mRNA processing <br />       transcription. Cell 2002, 108(4):501-512. <br /> 36.   Cagney G, Uetz P, Fields S: High-throughput screening pro- <br />       tein-protein interactions using two-hybrid assay. Methods <br />       Enzymol 2000, 328:3-14. <br /> 37.   Ma H, Kunes S, Schatz PJ, Botstein D: Plasmid construction by <br />       homologous recombination yeast. Gene 1987, 58(2&#226;&#8364;&#8220; <br />       3):201-216. <br /> 38.   Ito H, Fukuda Y, Murata K, Kimura A: Transformation intact <br />       yeast cells treated alkali cations. J Bacteriol 1983, <br />       153(1):163-168. <br /> 39.   James P, Halladay J, Craig EA: Genomic libraries host strain <br />       designed highly efficient two-hybrid selection yeast. <br />       Genetics 1996, 144(4):1425-1436.                                                Publish Bio Med Central every <br /> 40.   Ron D, Dressler H: pGSTag &#226;&#8364;&#8220; versatile bacterial expression <br />       plasmid enzymatic labeling recombinant proteins. Bio-                   scientist read work free charge <br />       techniques 1992, 13(6):866-869.                                            &quot;BioMed Central significant development for <br /> 41.   Frank R: Spot-Synthesis &#226;&#8364;&#8220; Easy Technique Position-              disseminating results biomedical researc h lifetime.&quot; <br />       ally Addressable, Parallel Chemical Synthesis Mem- <br />       brane Support. Tetrahedron 1992, 48(42):9217-9232.                              Sir Paul Nurse, Cancer Research UK <br /> 42.   Frank R: The SPOT-synthesis technique. Synthetic peptide                      Your research papers be: <br />       arrays membrane supports &#226;&#8364;&#8220; principles applications. <br />       J Immunol Methods 2002, 267(1):13-26.                                           available free charge entire biomedical community <br /> 43.   Knop M, Siegers K, Pereira G, Zachariae W, Winsor B, Nasmyth K,                 peer reviewed published immediately acceptance <br />       Schiebel E: Epitope tagging yeast genes using PCR-based <br />       strategy: tags improved practical routines. Yeast                      cited PubMed archived PubMed Central <br />       1999, 15(10B):963-972.                                                          &#226;&#8364;&#8221; copyright <br />  <br />                                                                              Submit manuscript here:                                BioMedcentral <br />                                                                              http://www.biomedcentral.com/info/publishing_adv.asp <br />  <br />  <br />  <br />  <br />                                                                                                                                             Page 11 11 <br />                                                                                                               <br /> </body></html>