<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>IRE1a Disruption Causes Histological Abnormality of <br /> Exocrine Tissues, Increase of Blood Glucose Level, and <br /> Decrease of Serum Immunoglobulin Level <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Accumulation of unfolded proteins in the endoplasmic reticulum (ER) causes ER stress. As a cellular adaptive response <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> ER <br />      stress, unfolded protein response (UPR) activates molecules for the quality control of ER proteins. One enzyme that plays an <br />      important role in UPR is Inositol Requiring Enzyme-1 (IRE1), which is highly conserved <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to humans. In particular, <br />      mammalian IRE1a activates X-box-binding protein 1 (XBP1) by unconventional splicing of XBP1 mRNA during ER stress. <br />      From analysis of <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>knockout</span> mice, both IRE1a and XBP1 have been shown to be essential for development and that XBP1 is <br />      necessary for the secretory machinery of exocrine glands, plasma cell differentiation, and hepatic lipogenesis. However, the <br />      essentiality of IRE1a in specific organs and tissues remains incompletely understood. Here, we analyzed the <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br />      IRE1a conditional knockout mice and found that IRE1a-deficient mice exhibit mild hypoinsulinemia, hyperglycemia, and a <br />      low-weight trend. Moreover, IRE1a disruption causes histological abnormality of the pancreatic acinar and salivary serous <br />      tissues and decrease of serum level of immunoglobulin produced in the plasma cells, but not dysfunction of liver. <br />      Comparison of this report with previous reports regarding XBP1 conditional knockout mice might provide some clues for <br />      the discovery of the novel functions of IRE1a and XBP1. (196 words) <br />  <br />  <br />  <br /> Introduction                                                                           ER-located type I transmembrane protein with a kinase domain <br />                                                                                        and RNase domain in the cytosolic region. When exposed to ER <br />    Since the majority of secretory proteins, such as antibodies,                       stress, via trans-autophosphorylation and activation of its RNase <br /> digestive enzymes, and hormones, are synthesized in the cytoplasm                      domain, IRE1 induces unconventional splicing of an mRNA <br /> and are cotranslationally translocated into the lumen of the                           encoding a specific transcription factor responsible for UPR <br /> endoplasmic reticulum (ER) through a narrow channel called                             activation [4&#226;&#8364;&#8220;8]. IRE1 is highly conserved from yeast to humans, <br /> translocon on the ER membrane, they are initially located in the                       and two IRE1 paralogues have been reported in mammals: IRE1a <br /> ER as unfolded and unmodified nascent polypeptides. These                              and IRE1b [9&#226;&#8364;&#8220;11]. Specifically, IRE1a induces the unconven- <br /> proteins then undergo meticulous folding by molecular chaper-                          tional splicing of XBP1 mRNA under ER stress condition [12]. <br /> ones, correct disulfide bond formation by protein disulfide                            The spliced XBP1 mRNA is then translated into a functional <br /> isomerases, and proper oligosaccharide modification by the                             transcription factor to induce UPR. Besides IRE1, two ER-located <br /> oligosaccharyltransferase complex, sugar trimming enzymes, and                         transmembrane proteins, PERK and ATF6, play important roles <br /> calnexin/calreticulin cycle in the ER [1,2]. Therefore, when cells                     in mammalian UPR [13,14]. On sensing ER stress, PERK induces <br /> produce these proteins in large amounts, the ER is thought to be                       the phosphorylation of eIF2a and the translational activation of <br /> liable to become overloaded for the maturation of these proteins.                      ATF4 [15]. On the other hand, under ER stress condition, ATF6 <br /> Accumulation of unfolded proteins in the ER also causes ER                             is cleaved by Site-1 and Site-2 proteases, and its cytoplasmic <br /> stress. To adaptively respond to ER stress, the cell induces the                       domain is translocated to the nucleus [16,17]. Both ATF4 and the <br /> transcriptional activation of molecules for the maturation of                          cleaved ATF6 work as transcription factors in UPR induction, as <br /> proteins in the ER. This response is called unfolded protein                           well as XBP1 which is activated by IRE1a. <br /> response (UPR) [3]. Thus, UPR is an important cellular response                           As described above, IRE1a directly catalyzes the cleavage of <br /> for the mass production of functional secretory proteins from                          XBP1 mRNA in the splicing reaction under ER stress condition <br /> unfolded proteins in cells which produce them in large amounts.                        [12]. To our knowledge, this reaction is exclusively dependent on <br />    To date, several molecules have been reported to play                               IRE1a activity and is not detected in IRE1a-deficient cells [18]. <br /> important roles in UPR. One of these molecules, IRE1, is an                            This implies that IRE1a and XBP1 function on the same signal <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  1                            September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                        Phenotype of IRE1a CKO Mice <br />  <br />  <br /> transduction pathway in ER stress response. Also, IRE1a knockout            bromophenol blue), and the lysate was heated to 98uC for 10 min. <br /> (KO) mice and XBP1 KO mice commonly have embryonic                          SDS-PAGE (10% gel) was used to resolve the proteins in the <br /> lethality and that both IRE1a and XBP1 play an essential role in            lysate. After electrophoresis, the proteins were electrotransferred <br /> mammalian development [19&#226;&#8364;&#8220;21]. However, although embryonic                  onto polyvinylidene fluoride microporous membranes. Immuno- <br /> lethality of XBP1 KO mice is rescued with an XBP1 transgene                 detection of amylase and glyceraldehyde 3-phosphate dehydroge- <br /> specifically expressed in the liver [22], that of IRE1a KO mice is          nase (GAPDH) was performed using rabbit polyclonal anti- <br /> rescued with endogenous IRE1a specifically expressed in the                 amylase antibody (Sigma) and mouse monoclonal anti-GAPDH <br /> extra-embryonic tissues and not in the liver [18]. This suggests that       antibody (Cell signaling), respectively. <br /> not only a known IRE1a-dependent XBP1 function but also an <br /> XBP1-independent IRE1a function(s) may exists in extra-embry-               Analysis of B cell differentiation <br /> onic tissues and that an IRE1a-independent XBP1 function(s) may                Serum levels of IgM and IgG1 were measured by ELISA <br /> exists in the fetal liver. Thus, a comparison analysis of                   (Bethyl Laboratories Inc.) according to the manufacturer&#226;&#8364;&#8482;s <br /> conventional and conditional KO mice in terms of IRE1a and                  instruction. Flow cytometry analysis was performed with FACS <br /> XBP1 may further provide some clues for the discovery of                    vantage SE (Becton Dickinson) and FlowJo ver. 6.3.2 software. For <br /> additional tissue-specific functions of each molecule. Analysis of          the analysis, single-cell suspensions from the spleen were prepared <br /> XBP1 conditional KO mice, including XBP1 KO mice rescued                    according to the protocol of Beckman Coulter Japan. Splenocytes <br /> with an XBP1 transgene specifically expressed in the liver,                 were then stained with PE-conjugated IgD antibody (Becton <br /> previously demonstrated that XBP1 is required for the secretory             Dickinson), FITC-conjugated IgM antibody (eBioscience), and <br /> machinery of exocrine glands, plasma cell differentiation, and              APC-conjugated B220 antibody (Becton Dickinson). <br /> hepatic lipogenesis [22&#226;&#8364;&#8220;24]. However, it remains unclear whether <br /> IRE1a plays an essential function for these biological phenomena.           Measurement of transaminase activity in serum <br /> To elucidate this, we analyzed the phenotype of IRE1a conditional              Both aspartate aminotransferase (AST) and alanine amino- <br /> KO mice in this study.                                                      transferase (ALT) activities in serum were measured using the <br />                                                                             transaminase assay kit (Wako). <br /> Methods <br /> IRE1a conditional KO mice                                                   Quantitative PCR analysis <br />                                                                                Quantitative PCR analysis of each transcript was performed using <br />   As previously described, we generated viable IRE1a conditional <br />                                                                             TaqMan probe and 7900HT (Applied Biosystems) in accordance with <br /> KO mice (Mox2+/Cre; IRE1aDNeo/DR) and control mice (Mox2+/+; <br />                                                                             the manufacturer&#226;&#8364;&#8482;s instructions with GAPDH transcript as an internal <br /> IRE1aDNeo/DR) by breeding Mox2+/Cre; IRE1a+/DR mice with <br />                                                                             control. Results are expressed as mean 6 S.E.M from triplicate <br /> Mox2+/+; IRE1aDNeo/DNeo mice [18]. IRE1a conditional KO mice <br />                                                                             experiments using RNA isolated from three independent tissues. Each <br /> and control mice were born at near-Mendelian ratios. All mice <br />                                                                             probe/primer set, namely, Mm00469005_m1, Mm00499536_m1, <br /> used in the experiment were maintained on a mixed (C57BL/6 x <br />                                                                             Mm00772290_m1, and Mm01204691_m1 (all from Applied Biosys- <br /> 129/SvE) background. Experimental protocols involving animals <br />                                                                             tems) was used for the quantification of IRE1b, diacylglycerol O- <br /> were approved by Animal Studies Committees at RIKEN (the <br />                                                                             acyltranserase 2 (Dgat2), stearoyl-coenzyme A desaturase 1 (Scd1), and <br /> permit number; H22-1-105) and NAIST (the permit number; <br />                                                                             acetyl-coenzyme A carboxylase 2 (Acc2) transcripts, respectively. <br /> 1011). <br />                                                                             IRE1a transcripts were quantified using the forward primer: 59-ggt cca <br />                                                                             atc gta cgg cag tt-39, the reverse primer: 59-tct ctc aca gag cca cct ttg <br /> Measurement of blood glucose and insulin                                    tag-39, and probe: 59-FAM-cct gca gac aga tct-MGB-39. Total XBP1 <br />    Blood glucose level was measured using a portable glucose                transcripts were quantified using the forward primer: 59-gaa tgg aca cgc <br /> measuring device (Arkray). Insulin level was determined by                  tgg atc ct-39, the reverse primer: 59-gcc acc agc ctt act cca ctc-39, and <br /> enzyme linked immunosorbent assay (ELISA) using mouse insulin               probe: 59-FAM-cct ctg gaa cct cg-MGB-39. Spliced XBP1 transcripts <br /> as a standard (Shibayagi). Glucose tolerance tests were performed           were quantified using the forward primer: 59-gaa tgg aca cgc tgg atc <br /> on 20-week-old IRE1a conditional KO and control mice that had               ct-39, the reverse primer: 59-cag agt cca tgg gaa gat gtt ct-39, and probe: <br /> been fasted for 16 hours. Mice were orally administered with                59-FAM-cac ctg ctg cgg act-MGB-39. <br /> 2 mg/g body weight glucose. Blood glucose level and serum <br /> insulin level were measured at indicated intervals. <br />                                                                             Results <br /> Histological analysis                                                       IRE1a-deficient mice exhibit mild hypoinsulinemia, <br />    Each tissue was fixed in 10% formalin and then embedded in               hyperglycemia, and low-weight trend <br /> paraffin. Paraffin blocks were sliced into 5-mm-thick sections                  Although IRE1a conventional KO mice show embryonic lethality <br /> and stained with hematoxylin and eosin for general histopath-               as described earlier, embryo proper-restricted IRE1a conditional KO <br /> ological analysis. Immunohistochemical analysis was performed               mice, which specifically express IRE1a in the extra-embryonic <br /> using 6-mm-thick paraffin sections. Immunoreactivity of insulin             tissues, can avoid embryonic lethality [18,20,21]. We initially <br /> and glucagons was detected using guinea pig polyclonal anti-                measured body weight and blood glucose level from age 4 weeks to <br /> insulin and rabbit polyclonal anti-glucagon antibodies (both                age 16 weeks to examine the growth process and health condition of <br /> from Dako), respectively. Immunoreaction with secondary                     embryo proper-restricted IRE1a conditional KO mice (hereinafter <br /> antibodies and chromogenic reaction with 3,39-diaminobenzi-                 IRE1a CKO mice). IRE1a CKO mice exhibited mild hyperglycemia <br /> dine tetrahydrochloride were performed according to standard                and a low-weight trend compared with control mice under free <br /> procedures.                                                                 feeding conditions (Figure 1A), although there was no difference in <br />                                                                             body <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>, amount of feed and water intake per day between IRE1a <br /> Western blot analysis                                                       CKO and control mice (Table S1). To investigate the blood glucose <br />   The tissues were lysed in SDS sample buffer (50 mM Tris-HCl               regulatory ability of IRE1a CKO mice, we performed oral glucose <br /> pH 6.8, 2% SDS, 50 mM DTT, 10% glycerol, and 1 mg/ml                        tolerance test (OGTT). The measurement of blood insulin level <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                         September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                           Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison of body weight, blood glucose level, and plasma insulin level. (A) Weekly changes in body weight (left) and blood <br /> glucose level (right) under an ad libitum feeding condition between 4 and 16 weeks after birth. The solid line with squares and the broken line with <br /> circles refer to the control and IRE1a CKO mice data, respectively. The upper figures and lower figures refer to the data for male and female mice, <br /> respectively. The plots indicate mean and error bars denote S.E.M (n = 7&#226;&#8364;&#8220;10). Statistical significance of differences between control and IRE1a CKO <br /> mice was determined by Student&#226;&#8364;&#8482;s t-test (*p,0.05, **p,0.01). (B) Changes in blood glucose level (left) and blood insulin level (right) after oral <br /> administration of glucose. The solid line with squares and the broken line with circles refer to the control and IRE1a CKO mice data, respectively. The <br /> upper figures and lower figures refer to the data for male and female mice, respectively. Measurement was performed at 20 weeks old. The plots <br /> indicate mean and error bars denote S.E.M (n = 7&#226;&#8364;&#8220;10). Statistical significance of differences between control and IRE1a CKO mice was determined by <br /> Student&#226;&#8364;&#8482;s t-test (*p,0.05, **p,0.01). <br /> doi:10.1371/journal.pone.0013052.g001 <br />  <br />  <br /> during OGTT revealed that insulin level at its peak was decreased by             (Figure 3A and 3B and Figure S1). HE-stained sections revealed that <br /> 30,50% in IRE1a CKO mice compared with control mice. On the                      the salivary gland serous acini of the IRE1a CKO mice were half the <br /> other hand, the measurement of blood glucose level during OGTT                   size compared with those of the control mice; however, the size of <br /> showed that IRE1a CKO mice have a significantly higher blood                     the salivary gland mucous acini of the IRE1a CKO mice was similar <br /> glucose level at its peak than control mice and that IRE1a CKO mice              to that of the control mice (Figure 3C and 3D and Figure S2 and <br /> required a longer time for blood glucose level to recover than control           S3). On the other hand, the pancreatic acinar tissue of the IRE1a <br /> mice (Figure 1B). Moreover, we measured the level of insulin                     CKO mice showed local spongy lesion with loss of acinar cells <br /> extracted from the pancreas and islets of IRE1a CKO mice by ELISA                (Figure 3E and Figure S4), which started occurring four weeks after <br /> in which the amount of total proteins was used as an internal                    birth. We suspected that this pathological change was caused by <br /> standard. In addition, we performed immunohistochemical analysis                 apoptosis; however, TUNEL analysis showed no difference between <br /> of insulin and glucagons in the tissue sections of the pancreas of               the IRE1a CKO mice and the control mice (Table S3). Next, we <br /> IRE1a CKO mice. The level of insulin extracted from the pancreas                 examined the effects of IRE1aon the production of digestive <br /> and islets of IRE1a CKO mice was quite similar to that of control                enzymes in the pancreas and salivary glands. Amylase is one of the <br /> mice (Table S2). The signals of insulin and glucagons in the pancreas            key digestive enzymes produced both in the pancreatic acinar and <br /> of IRE1a CKO mice were normally detected (Figure 2A). Number                     salivary gland serous cells. Western blot analysis performed using <br /> and size of the islets and ratio of insulin+ cells to glucagons+ cells per       the tissue lysate and saliva revealed no difference in the amount of <br /> islet were also nearly equal between IRE1a CKO and control mice                  amylase expression and secretion between the IRE1a CKO mice <br /> (Figure 2B). Thus, there is no histological difference in the islets             and the control mice (Figure 3F). <br /> between IRE1a CKO and control mice. <br />                                                                                  IRE1a is necessary for the differentiation of B cells into <br /> IRE1a disruption causes histological abnormality of the                          plasma cells <br /> pancreatic acinar and salivary gland serous tissues                                 Previous studies have demonstrated that IRE1a was activated <br />    As described above, the embryonic lethality of XBP1 KO mice                   during the differentiation of B cells into plasma cells and that <br /> could be rescued by the expression of a liver-specific XBP1                      XBP1 played an essential role during the same process [21,23,25]. <br /> transgene; however, the mice die a few days after birth. This death              To investigate the roles of IRE1a in the differentiation of B cells in <br /> is thought to be caused by the dysfunction of digestion-associated               mice, serum IgM and IgG1 as well as the populations of IgM+, <br /> exocrine glands, such as the pancreas and salivary glands [22].                  IgD+ and B220+ cells in splenocytes were analyzed in the IRE1a <br /> Therefore, we investigated the size and histological phenotype of the            CKO mice. Serum IgM and IgG1 serve as markers of plasma cell <br /> pancreas and salivary glands of IRE1a CKO mice. We found that                    differentiation, and IgM, IgD and B220 serve as markers of <br /> the size of the salivary glands of the IRE1a CKO mice was normal,                mature B cells. ELISA revealed that serum levels of IgM and IgG1 <br /> but their pancreas was 30,50% smaller than that of control mice                  were reduced by half in the IRE1a CKO mice compared with the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            3                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                     Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 2. Histological analysis of the islets in IRE1a CKO mice and control mice. (A) The upper, middle, and lower panels show HE-stained <br /> sections, immunostained images using anti-insulin antibodies, and immunostained images using anti-glucagon antibodies, respectively. The three <br /> images for the control or IRE1a CKO mice are derived from the serial sections (scale bar: 50 mm). (B) aThe number of islets counted per 1 cm2 of <br /> pancreas section is shown as mean 6 S.E.M (n = 10 pancreas sections). bThe diameter (mm) of islet is shown as mean 6 S.E.M (n = 50). cThe ratio of <br /> insulin+ cells to glucagons+ cells per islet is shown as mean 6 S.E.M (n = 10). All the data were obtained from 20 weeks old female mice. <br /> doi:10.1371/journal.pone.0013052.g002 <br />  <br /> control mice. Lipopolysaccharide (LPS) stimulation increases the            of immunoglobulin produced in the plasma cells, but does not <br /> concentration of these immunoglobulins two- to three-fold;                  have any effect on the differentiation into mature B cells. <br /> however, as in the case of no LPS stimulation, the levels of IgM <br /> and IgG1 were approximately half in the IRE1a CKO mice                      Hepatic function of IRE1a-deficient mice is normal <br /> compared with the control mice (Figure 4A). Flow cytometry                     Hepatic abnormality has been reported in the embryo of both <br /> analysis of splenocytes showed that the populations of positive or          IRE1a KO mice and XBP1 KO mice [19,21]. Moreover, XBP1 <br /> negative cells for IgM, IgD and B220 markers were similar                   has been shown as necessary for lipid metabolism in the <br /> between the IRE1a CKO mice and the control mice (Figure 4B).                hepatocytes of mature mice [24]. On the other hand, since <br /> This suggests that IRE1a disruption causes decrease of serum level          histological analysis of the embryonic liver of IRE1a CKO mice <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       4                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                    Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   5   September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                        Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br /> Figure 3. Effect of the deletion of IRE1a on the pancreatic exocrine gland and salivary glands. (A) Comparison of raw weight (g) of <br /> organs between the IRE1a CKO mice and the control mice (20 weeks old). Data are expressed as mean 6 S.E.M (n = 7&#226;&#8364;&#8220;10). (B) Pancreas collected from <br /> the mice. The pancreas come with the spleen and part of the small intestine (scale bar: 1 cm). (C) HE-stained sections of the serous tissue of the <br /> salivary glands (scale bar: 200 mm). The longest diameter of serous acini is shown as mean 6 S.E.M (n = 50). Statistical significance of differences <br /> between control and IRE1a CKO mice was determined by Student&#226;&#8364;&#8482;s t-test (p,0.005). (D) HE-stained sections of the mucous tissue of the salivary <br /> glands (scale bar: 100 mm). The longest diameter of mucous acini is shown as mean 6 S.E.M (n = 50). (E) HE-stained sections of the pancreatic acinar <br /> tissue (scale bar: 200 mm). (F) Western blot analysis of amylase in the pancreas, salivary glands and saliva. For each genotype, samples derived from <br /> three mice were used. Each saliva sample used was equal in volume. GAPDH was used as an internal standard. All the data in B-F were obtained from <br /> 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g003 <br />  <br /> previously failed to find abnormality [18], we performed                     abnormality of insulin secretion. Insulin is prevented from entering <br /> phenotype analysis of the liver of IRE1a CKO mature mice by                  its secretory pathway by the quality control of ER proteins if it <br /> measurement of serum AST and ALT, observation of HE-stained                  does not receive proper modifications (e.g., folding and disulfide <br /> sections, and analysis of the expression of lipid metabolism-related         bond formation) during its biosynthesis and secretion. It is possible <br /> genes. However, no clear differences were found between the                  that IRE1a is indirectly involved with the maturation process of <br /> IRE1a CKO mice and the control mice (Figure 5).                              insulin in the ER by inducing the expression of chaperones, <br />                                                                              disulfide isomerase and other enzymes. On the other hand, based <br /> Functional IRE1a mRNA is significantly disrupted in IRE1a                    on in vitro analysis of the effect of IRE1a on insulin, suppression of <br /> CKO mice                                                                     IRE1a has been reported to cause low insulin biosynthesis at the <br />    Quantitative PCR analysis of IRE1a and XBP1 in various                    translation level but has little impact on insulin secretion [28]. The <br /> tissues was performed to determine the degree of functional                  downregulation of IRE1a has also been reported to suppress <br /> disruption of IRE1a mRNA in the IRE1a CKO mice. In all the                   insulin mRNA degradation that occurs under a chronic high- <br /> tissues tested, PCR signals of IRE1a were detected in the control            glucose environment [29]. However, we have not yet performed <br /> mice as previously reported [18], but weakly or not detected in the          experiments to confirm these findings, because it is not easy to <br /> IRE1a CKO mice (Figure 6A). Although PCR signals of total                    prepare many samples and mice for detailed experiments on <br /> XBP1 were similarly detected in both the control and IRE1a CKO               insulin biosynthesis; in theory, the IRE1a CKO mice that we used <br /> mice (Figure 6B), those of spliced XBP1 were slightly or obviously           here have only a 25% chance of being born. In the future, we <br /> detected in the control mice, but little in the IRE1a CKO mice               should aim to analyze the functions of IRE1a in association with <br /> (Figure 6C and 6D). In the case that mice were injected with                 insulin biosynthesis in the b cells of the islets by generating mice in <br /> tunicamycin, an ER streesor, the spliced XBP1 level of IRE1a                 which IRE1a is disrupted specifically in b cells. It was previously <br /> CKO mice were hardly detected in all the tested tissues, but that of         reported that loss of XBP1 had no effect on the development of the <br /> control mice strongly increased in the liver and kidney and slightly         islets [22] and that XBP1+/2 mice tended to have high blood <br /> increased in other tissues (Figure 6C and 6D). In addition,                  glucose levels [30]. However, it remains unclear how IRE1a and <br /> although IRE1b is specifically expressed in the gastrointestinal             XBP1 function in terms of insulin biosynthesis/secretion and the <br /> tract [26], IRE1b expression was examined in the same tissues by             control of blood glucose level. In other words, it is uncertain <br /> quantitative PCR. As expected, PCR signals of IRE1b were                     whether IRE1a and XBP1 function on the same pathway or <br /> predominantly detected in the gastrointestinal tract in both the             independently. To answer these questions, we need to generate <br /> control and IRE1a CKO mice, but slightly in the other tissues                mice expressing the active (spliced) form of XBP1. We believe that <br /> (Figure 6E). These results confirm that functional IRE1a mRNA                valuable information on the functional relationship between <br /> and spliced XBP1 mRNA are significantly disrupted in the IRE1a               IRE1a and XBP1 can be obtained by analysis of insulin <br /> CKO mice and suggest that IRE1b may not complement the role                  biosynthesis/secretion and blood glucose level in the offspring <br /> of IRE1a in the IRE1a CKO mice. On the other hand, IRE1a                     produced by crossing IRE1a KO mice and XBP1 KO mice to the <br /> mRNA and spliced XBP1 mRNA were slightly detected in some                    mice expressing the spliced XBP1. <br /> tissues of the IRE1a CKO mice. Thus, IRE1a is not completely                    Size reduction of acinar tissue and partial deletion of acinar cells <br /> disrupted in the IRE1a CKO mice, which may be the reason why                 in the pancreas and size reduction of serous acini in the salivary <br /> the phenotype of IRE1a CKO mice is not clear-cut.                            gland were observed in the IRE1a CKO mice. The pancreatic <br />                                                                              acinar tissue and salivary gland serous tissue are both representative <br /> Discussion                                                                   organs for the secretion of digestive enzymes and produce a large <br />                                                                              amount of proteins such as amylase. As with insulin, amylase is <br />    ER stress is associated with various diseases [27], and the in vivo       prevented from entering the secretory pathway by quality control of <br /> functions of ER stress-related molecules are presently receiving             ER proteins if it does not receive proper modifications in the ER <br /> attention. With regard to IRE1a the functional analysis of this              (e.g., folding and disulfide bond formation) during its biosynthesis <br /> molecule in adult KO mice has been lagging because the KO mice               and secretion. IRE1a is activated in the pancreatic acinar tissue and <br /> suffer embryonic lethality; however, we could show here using the            salivary gland even under physiological conditions and excessive <br /> conditional KO method that IRE1a-deficient mice exhibit mild                 expression of amylase causes ER stress and IRE1a activation <br /> hypoinsulinemia, hyperglycemia, and a low-weight trend, and that             [31,32]. Based on these previous findings, it is possible that IRE1a is <br /> IRE1a disruption causes histological abnormality of the pancreatic           involved in homeostasis in the ER and the development and/or <br /> acinar and salivary serous tissues and decrease of serum level of            <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>survival</span> of exocrine cells in the pancreatic acinar tissue and salivary <br /> immunoglobulin produced in the plasma cells.                                 gland by constantly inducing the expression of chaperones, disulfide <br />    In the present study, the level of insulin protein extracted from         isomerase and other enzymes. As described above, a developmental <br /> the pancreas and islets was similar between the IRE1a CKO mice               disorder of the pancreatic acinar tissue and salivary gland is also <br /> and the control mice. Presumably, insulin expression level is                observed in XBP1 CKO mice [22]. Therefore, signal transduction <br /> normal in IRE1a-deficient b cells. Therefore, the lower level of             from IRE1a to XBP1 is important for the development and/or <br /> blood insulin in the IRE1a CKO mice may be due to an                         maintenance of the pancreatic acinar tissue and salivary gland. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        6                        September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                       Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 4. Effect of the deletion of IRE1a on B cells. (A) Serum immunoglobulin levels measured by ELISA. Columns indicate mean and error bars <br /> denote S.E.M (n = 5). All the data were obtained from 8 weeks old female mice. (B) Flow cytometry analysis of splenocytes using IgM, IgD and B220 as <br /> markers. The values at the four corners of each chart indicate the proportions of positive or negative cells for each marker. Reproducible data from <br /> three separate experiments are shown. All the data were obtained from 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g004 <br />  <br /> However, unlike in IRE1a CKO mice, phenotypes are more severe                indigestion of food, and death shortly after birth [22]. This may <br /> in XBP1 CKO mice as exemplified by observation of apoptotic                  indicate the importance of IRE1a-independent XBP1 functions for <br /> signals in the pancreatic acinar tissue, reduced production of               the development and/or maintenance of the pancreatic acinar <br /> digestive enzymes in the pancreatic acinar tissue and salivary gland,        tissue and salivary gland. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                       Phenotype of IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 5. Effect of the deletion of IRE1a on the liver. (A) HE-stained sections of the liver tissue (scale bar: 50 mm). (B) Comparison of the serum <br /> AST and ALT levels between the IRE1a CKO mice and the control mice. Data are expressed as mean 6 S.E.M (n = 5). (C) Quantitative PCR analysis of <br /> lipid synthesis genes in the liver of mice fed normal or high-fructose feed. Columns indicate mean and error bars denote S.E.M (n = 3). All the data <br /> were obtained from 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g005 <br />  <br />    Like hormones (e.g., insulin and glucagons) and digestive                 histological abnormalities in the embryonic liver of IRE1a CKO <br /> enzymes, antibodies are also secretory proteins and are prevented            mice has been described [18]. Also, XBP1 has been shown to be <br /> from entering their secretory pathway by quality control of ER               essential for lipid metabolism in the hepatic cells of mature mice <br /> proteins if they do not receive proper modifications in the ER (e.g.,        [24]. In the present study, hepatic abnormalities were not found <br /> folding and disulfide bond formation) during their biosynthesis and          on analysis of IRE1a CKO mice. This may suggest the existence of <br /> secretion. Therefore, IRE1a is believed to play an important role            IRE1a-independent XBP1 functions. If XBP1 exerts functions <br /> in plasma cells in terms of the production and quality of                    depending on IRE1a, splicing of XBP1 mRNA by IRE1a would <br /> antibodies. In the present study, the serum levels of IgM and                be necessary. Also, splicing of XBP1 mRNA is blocked by <br /> IgG1 in the IRE1a CKO mice were half of those in the control                 inactivation of IRE1a [18]. Given these observations, the <br /> mice, and this trend was observed regardless of the presence of              functions of IRE1a-independent XBP1 may be brought about <br /> LPS stimulation. Previous studies also revealed IRE1a activation             by unspliced XBP1. <br /> during the differentiation of B cells into plasma cells and that                Phenotype analysis of IRE1a CKO mice in this study provided <br /> XBP1 played an essential role in the differentiation process                 important information regarding the physiological functions of <br /> [21,23,25]. Based on these findings, IRE1a may contribute to the             IRE1a. Comparison of our results with the phenotypes of XBP1 <br /> normal production of antibodies by activating XBP1 on known                  KO mice previously reported indicates the possibilities that IRE1a <br /> UPR pathways, thereby inducing the expression of chaperones,                 has functions independent of XBP1 and XBP1 has functions <br /> disulfide isomerase and other enzymes in the ER.                             independent of IRE1a. This study therefore provided findings that <br />    Hepatic abnormalities in the embryos of IRE1a KO and XBP1                 are important for revealing new roles played by IRE1a and XBP1 <br /> KO mice have been reported [19,21], while the absence of                     in vivo. <br />  <br />  <br />  <br /> Figure 6. Quantitative PCR analysis of IRE1a, IRE1b, and XBP1. (A) IRE1a. (B) Total XBP1. (C) Spliced XBP1. (D) Ratio of spliced XBP1 to total <br /> XBP1. (E) IRE1b. Tun 2 or + indicates i.p. injection with saline or tunicamycin (500 ng/g body weight) 16 before tissue collection. Li; liver, K; kidney, <br /> Sp; spleen, H; heart, Lu; lung, B; brain, Sa; salivary gland, St; stomach, In; intestine, Co; colon, Pa; pancreas, Is; islet. All the data were obtained from 20 <br /> weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g006 <br />  <br />    In this regard, however it is also possible that the discrepancies                       Found at: doi:10.1371/journal.pone.0013052.s004 (0.62 MB TIF) <br /> between the present data and those from Glimcher&#226;&#8364;&#8482;s [22,24] and <br />                                                                                             Figure S2 Fig. 3C is enlarged for the presentation of fine <br /> Kaufman&#226;&#8364;&#8482;s [21] groups may depends on difference among each <br />                                                                                             histological structures. Two distinctive serous acini in each panel <br /> experimental condition. For example, in conditional KO mice <br />                                                                                             are surrounded by a dashed line for ease of comparing the size of <br /> with the exogenous target gene [21,22], the exogenous target gene <br />                                                                                             acini between control and IRE1a CKO mice. <br /> may be different from the endogenous target gene in expression <br /> level and have any unexpected dominant effects, unlikely in                                 Found at: doi:10.1371/journal.pone.0013052.s005 (1.06 MB TIF) <br /> conditional KO mice with flox/Cre system. Further, difference in                            Figure S3 Fig. 3D is enlarged for the presentation of fine <br /> mouse genetic background may raise distinct compensatory                                    histological structures. Two distinctive mucous acini in each panel <br /> mechanism by other UPR pathways between IRE1a and XBP1                                      are surrounded by a dashed line for ease of comparing the size of <br /> CKO mice even if these mice are commonly generated with flox/                               acini between control and IRE1a CKO mice. <br /> Cre system. Therefore we need to carefully analyze the                                      Found at: doi:10.1371/journal.pone.0013052.s006 (1.03 MB TIF) <br /> phenotypes of IRE1a and XBP1 disruption under an identical <br /> experimental condition in the future.                                                       Figure S4      Fig. 3E is enlarged for the presentation of fine <br />                                                                                             histological structures. Ac; acinar tissue, Is; islet. The local spongy <br />                                                                                             lesion (unstained area) with loss of acinar cells is impressive in <br /> Supporting Information <br />                                                                                             pancreatic tissue of IRE1a CKO compared with that of control <br /> Table S1 Body length, feed intake, and water intake.                                        mice. <br /> Found at: doi:10.1371/journal.pone.0013052.s001 (0.03 MB                                    Found at: doi:10.1371/journal.pone.0013052.s007 (0.95 MB TIF) <br /> DOC) <br /> Table S2 Insulin content in the islet and the pancreas.                            <br /> Found at: doi:10.1371/journal.pone.0013052.s002 (0.03 MB                                  <br /> DOC)                                                                                <br /> Table S3 Total number of TUNEL positive cells in all acinar <br /> cells in a tissue section.                                                          <br /> Found at: doi:10.1371/journal.pone.0013052.s003 (0.03 MB                                    <br /> DOC)                                                                                        <br /> Figure S1 Fig. 3B is enlarged for the presentation of fine                                 <br /> histological structures. Sp; spleen, In; intestine, Pa; pancreas, <br /> which is surrounded by a dashed line. <br /> </body></html>