<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Sept. 1988, p. 3967-3977                                                                           Vol. 170, No. 9 <br /> 0021-9193/88/093967-11$02.00/0 <br /> Copyright C 1988, American Society Microbiology <br />  <br />   Gene Organization Region Containing New Gene Involved <br />             Chromosome Partition Escherichia coli <br />                        JUN-ICHI KATO,1t YUKINOBU NISHIMURA,2 MASAO YAMADA,2 <br />                                   HIDEHO SUZUKI,&apos;* AND YUKINORI HIROTA2&#194;&#167; <br />       Laboratory Genetics, Department Biology, Faculty Science, University Tokyo, Hongo, Tokyo 113,1 <br />                          National Institute Genetics, Mishima, Shizuoka-ken 4JJ,2 Japan <br />                                              Received 14 March 1988/Accepted 11 June 1988 <br />             A new <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span>, parC, causing abnormal chromosome segregation identified thermosensitive <br />          mutants Escherichia coli. The thermosensitive growth mutants corrected pLC4-14 the <br />          Clarke-Carbon collection. This plasmid carries putative gene suppress ceUl division defect due <br />          ftsl (pbpB) termed sufi (sui). The nearness parC metC confirmed, and <br />          cotransduction frequency parC 59% metC 20% gkc. The parC-sufl region analyzed <br />          subcloning chromosome region pLC4-14. The parC sufi gene products electrophoret- <br />          ically identified proteins 75 55 kilodaltons (kDa), respectively. The allelism parC+ pLC4-14 <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />          parC1215 confirmed cloning parCi215. The sufi gene appeared dispensable cell viability, and <br />          overproduction product caused suppression ftsI. An essential gene coding 25-kDa protein was <br />          parC sufi gene. These genes transcribed direction may <br />          organized operon, parC proximal internal promoters distal <br />          genes. The localization gene products examined maxicells. The sufi protein synthesized a <br />          precursor chased mature form. The major mature form the <br />          soluble fraction. The 25-kDa protein exclusively membrane fraction. The parC protein <br />          associated membrane fraction presence Mg2` soluble fraction when <br />          Mg2+ sequestered EDTA. <br />  <br />    A goal cell division segregational transmission                replicon; cis-acting factor DNA segment presumably <br /> genome daughter cells. To prevent production chromo-                   forming complex trans-acting gene products and <br /> someless cells, process cell division Escherichia                  providing site membrane attachment (23, 30). The <br /> coli, organisms, placed elaborate               oriC plasmid stabilized maintenance joined <br /> control couple replication partition chromo-               sop region F plasmid (30). <br /> somes. For example, perturbation chromosome replica-                        Two kinds Escherichia coli mutants defective chro- <br /> tion arrests cell division induction division inhibitor,             mosome partition described parA parB. <br /> the sulA gene product, targeting ftsZ (sulB) gene prod-                  They thermosensitive reveal distinctive morphol- <br /> uct, function essential progression                      ogy centrally congregated nucleoids restrictive <br /> division process (13, 14, 19). Chromosome partition                   temperature (11, 12). The par mutations recently been <br /> be process regulates cell division. This process,               investigated relationship termination of <br /> although barely understood, include aspects:                    DNA replication, parB probably allelic dnaG <br /> molecular separation (resolution topological linkages)                (for primase) located 67 min genetic mnap (29). <br /> topographical segregation replicated chromosomes.                     Analyses mutations expected provide <br /> The aspect postulated involve attach-                 clues mechanism chromosome partition its <br /> ment replicons cell surface structure, growth               control cell division process. <br /> which contributes topographical segregation at-                       We par mutant thermosensitive <br /> tached replicons (15).                                                       mutant stock described previously (33). The thermosensitive <br />    The segregation mechanism replicons recently                  <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> mutant corrected pLC4-14, is <br /> investigated plasmids maintain                   known cover 65-min region E. coli chromosome <br /> or copies chromosome require certain                       map (6). The new par mutation clearly different <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> partition functions stable inheritance cell                parA parB, described present report, and <br /> division cycles. Genetic loci responsible plasmid parti-                 termed parC. The plasmid pLC4-14 corrected <br /> tion identified plasmids shown                   thermosensitive defect ftsI (pbpB 2 min, for <br /> function cis trans (20, 21, 28, 30). One cis-acting             penicillin-binding protein 3 [PBP3]), putative gene <br /> two trans-acting factors encoded sop region F                   suppressftsl mutation called sui (34) and <br /> plasmid involved stable inheritance F                      sufi (37). The term (suf, suppression Fts) <br />                                                                              comes finding similar suppression phenomena <br />   * <br />     Corresponding author.                                                    seeh combinations differentfts mutations <br />   t Present address: Department Bacteriology, National Institute          plasmids carrying distinct chromosome region (27, 37), <br /> of Health, Tokyo 141, Japan.                                                 ftsE (76 min) pLC19-48 (carrying 3-min <br />   t Present address: National Children&apos;s Medical Research Center,            region). <br /> Tokyo, Japan.                                                                   The chromosomal region carried pLC4-14 include <br />   &#194;&#167; Deceased 23 December 1986.                                               cluster genes essential chromosome partition its <br />                                                                       3967 <br />  3968      KATO ET AL.                                                                                                   J. BACTERIOL. <br />  <br />                       TABLE 1. Bacterial strains                        appropriate drug following concentrations: tetracy- <br />                  Relevant         Other markers          Source   cline, 10 ,ug/ml; kanamycin, 40 ,ug/ml; ampicihlin, 20 ,ug/ml; <br />  Strain          markers             derivation             reference   chloramphenicol, 15 p.g/ml. Transductants selected <br />                                                                         LB plates containing tetracycline (2.5 gxg/ml) kanamy- <br /> JE11215 parC1215              From PA3092 (27)                          cin (40 ,ug/ml) both. <br /> JE10572 parCS72               From PA3092                                  Mapping parC. JE11215 (parCl2 15) mated with <br /> PL8-31 glc-l str              F- thr-l leu-6 thi-J      Y. Komeda       P4x8 construct Hfr strains carrying parC. The. resultant <br />           serA25                metG87 hisC3 proA2                      Hfr (parC1215) crossed PL8-31 selected for <br />                                 metK85 mtl xyl galK <br />                                 lac Y supE                              SerA+ Strr. Examination thermosensitivity the <br /> JE6839     metC               F- argG mtl malA      GSRC&apos;           recombinants suggested marker order serA-glc-par <br />                                 gal lac str tsx                         C1215-str. One recombinants (glc parC) used a <br /> JE8515    parC1215            Met+ transductant                      donor P1 transduction parC JE6839 (metC); the <br />                                 JE6839                                  transductants selected MetC+ scored the <br /> JE7611    ftsI                F- thi met pro xyl str    25              ParC Glc phenotypes, parC+ transduced <br />                                 lacZ tonA tsx recA                      JE6839 MetC+ transductants re- <br /> P4x8                          Hfr metBI                 GSRC            ceived parC glc selecting Glc+ scored the <br /> N1126     polA12              F- thi leu pro malA   35              ParC MetC phenotypes. The thermosensitive phenotype <br />                                 thy cysC lacZ ara <br />                                 mtl xyl str spc                         shown ParC- mapping. <br /> TH1219    minB recA           F- rpsL tsr tar           7                  Plasnild construction. To facilitate vitro excision and <br /> C600                          F- thi thr leuB lacY      1               insertion required DNA region, BamHI fragment <br />                                 tonA supE                               containing translation terminator pTR27 Cmr <br /> JM109     recAl (lacP)        A(lac-pro) endAl          38              marker pJL3-4779 inserted BamHI site on <br />                                 gyrA96 thi-l hsdR17                     pUC8 construct pJK270 pJK272, respectively. The <br />                                 supE44 relAl                            terminator pJK270 Cmr marker pJK272 be <br />                                 (F&apos; traD36 proA                         remnoved blunt-end site excising fragment with <br />                                 lacIqZMJ5)                              blunt ends. <br /> EJ812     parC1215            Met+ transductant of <br />                                 C600 met::TnlO                             The mini-F vectors pJK280, pJK282, pJK284 were <br />                                 derivative                              constructed shown Fig. 1. The HpaI fragment of <br /> NK6027 metC162::TnlO          HfrH A(gpt-lac)5 relAl    GSRC            pSC138, contained essential region replication <br />                                 spoT) thi-l                             stable maintenance mini-F plasmid, ligated with <br />   a <br />     GSRC, Supplied Genetic Stock Research Center National <br />                                                                         PvuII fragments carrying lac promoter-operator region <br /> Institute Genetics (Mishima, Japan).                                 Tcr gene Kmr gene constructing pJK280 <br />                                                                         pJK282, respectively. The PvuII fragment pJK280 was <br />                                                                         constructed inserting EcoRI fragment pJK250 into <br /> linkage cell division, motivated analyze            EcoRI site pUC9, pJK282 con- <br /> the gene organization region. The analyses               structed inserting PvuII fragment pYS8 the <br /> parC-sufl gene region disclosed essential gene lo-              HincIl site pUC8. To construct pJK284, mini-F <br /> cated parC sufl genes. The products                  fragment pJK282 excised EcoRI HindIlI <br /> these genes identified, localization gene               nucleases site derived pUC8 ligated the <br /> products investigated. The results suggested               EcoRI-HindIII fragment carrying Apr marker pSC138. <br /> three genes organized operon internal                  The plasmids shown Fig. 2 constructed follows. <br /> promoters parC protein interacts                  The SmaI fragment containing parC gene excised <br /> membrane.                                                               pLC4-14; EcoRI synthetic linker attached the <br />                                                                         ends; fragment inserted EcoRI site on <br />                 MATERUILS AND METHODS                                   pBR322, pACYC184, pJK282, pJK284 (Fig. 1) pro- <br />                                                                         duce pJK800, pJK805 (used construct pJK806 below), <br />    Bacterial strains, plasmids, culture media. Bacterial            pJK816, pJK818, respectively. A translation terminator <br /> strains listed Table 1. Plasmid pTR27 carried               excised SmaI-HincIl fragment pJK270 ligated <br /> BamHI fragment carrying <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span> sequences termi-                 linear fragments pJK800 digested PvuII to <br /> nation signals occur reading frame direction          obtain pJK809. The termninator obtained BamHI frag- <br /> translation (I. N. Maruyama, K. Horikoshi, M. Soma,                 ment pTR27 similarly ligated pJK800 digested <br /> Y. Hirota, unpublished data). pLC4-14 synthetic ColEl              BamHI obtain pJK807. To construct pJK811, the <br /> plasmid Clarke Carbon (4). pYS8 pYS10                   HpaI fragment containing parC gene cloned a <br /> carry kanamycin resistance (Kni) marker                  ScaI site pBR322. To construct pJK815, Cmr gene on <br /> removed PvuII BamHI fragment, respectively,                  SmaI-HinclI fragment pJK272 inserted the <br /> and kindly provided Y. Suzuki. Derivatives                   PvuII site parC gene. pJK806 obtained deleting <br /> mini-F plasmids pSC138 (36) pKO16E (35) plasmid               PstI fragments pJK805. pJK801 constructed the <br /> with BamHI fragment containing chloramphenicol                    procedure used pJK800 construction (see <br /> resistance (Cm&apos;) gene, pJL3-4779, kindly provided               above); pJK800 pJK801 carry EcoRI fragment in <br /> K. Tanimoto. pACYC184 described (3).                           opposite orientations. pJK823 constructed deleting <br />    Bacteria grown routinely LB broth LB plates            small BamHI fragment pJK801. To construct <br /> (22). Thymine added final concentration 60 mg/              pJK813, pJK823 digested exonuclease Bal31 from <br /> liter needed. S-broth contained (per liter): Bacto-                unique BamHI site, plasmid carrying appro- <br /> tryptone, 5 g; yeast extract, 3 g; NaCl, 3.5 g; thymine, 60 mg;         priate deletion selected. pJK803 obtained insert- <br /> glucose, 3.6 g; 20 ml buffered salt solution (16).               ing SmaI fragment containing parC region pLC4- <br /> Transformants selected LB plates containing                  14 SmaI site pUC9. <br />  VOL. 170, 1988                                                           CHROMOSOME PARTITION GENE IN E. COLI                                             3969 <br />  <br />              Hp <br />                                                                                                                                                      parC <br />                                                                                  P PSP                           P   B(2)                S B(1) P            P <br />                                                                       pLC4-14                                                                        . <br />                                                                                                                                                          monow <br />                                                                                   P (E)             E B                         E       (E)                     P <br />                                                                       pJK800                                               = <br />  <br />  <br />                                                                                                     E B Pv                      E       (E) (Pv) Pv P <br />                                                                       pJK809 P(E) <br />                                                                       pJK807 P (E)                  E B                         E       (E) B                P <br />                                                                                                                                               A <br />                                                                                                 Sc E B                     Sc                 Hp           Hp <br />                                                                        pJK81 1                            i. <br />  <br />  <br />  <br />  <br />                                                                                                                                                    Cm <br />                                                                                   P (E)             E B                         E       (E)         3Pv P <br />                                                                        pJK815 <br />                                                                                    P(E)                      E H                E       (E)          P <br />                                                                        pJK806         U <br />                                                                               P(E               EIE                                     (E)                  P <br />                                                                        PJK816 t2E) <br />                                                                                                 E                                   E   (E)                 P <br />                                                                        jJK818     P(E) <br />                                                                                   P (E)                 E                  B E          (E) B               P <br />                                                                        pJK801 L=i <br />                                                                                   P       (E)           E                   B                 B             P <br />                                                                        pJK823 e <br />                                                                                   P (E)                 E             (B)                          (B)      P <br />                                                                        pJK813     t                                  ::S <br />                                                                                   P S                   S            S                    S             P <br />                                                                        pJK803 t=                                                                  _Iuomm <br />  <br />  <br />                                                                         FIG. 2. Restriction map plasmids carrying parC region. <br />                                                                       Solid open bars indicate regions derived chromosome <br />                                                                       plasmids, respectively, region containing parC <br />                                                                       gene indicated. Open squares A indicate insertion sites of <br />                                                                       translation terminators. The ability correct temperature- <br />                                                                       sensitive growth JE11215 (parC) used cloning parC <br />                                                                       region. B, BamHI; E, EcoRI; H, HindIII; Hp, HpaI; P, PstI; Pv, <br />                                                                       PvuII; S, SmaI; Sc, ScaI. Two BamHI sites pLC4-14 shown <br />   FIG. 1. Vectors derived mini-F plasmids. Three mini-F          B(1) B(2) map pLC4-14 convenience. <br /> vectors, pJK280 (Tcr), pJK282 (Kmr), pJK284        (Ap%,   were <br /> constructed pSC138, derivative mini-F. The HpaI frag- <br /> ment shown open bar contains regions essential        pJK855, pJK858, pJK863, pJK864 derived from <br /> replication stable partition mini-F replicon. E, EcoRI; H,   pLC4-14. To construct pJK855, Kmr marker was <br /> HindIII; Hp, HpaI; HII, HincII; Pv, PvuII.                            excised XhoI fragment pKO16E inserted into <br />                                                                       unique SalI site pLC4-14; resultant plasmid was <br />    The plasmids shown Fig. 3 constructed follows.          subjected excision small KpnI fragment sufi <br /> To obtain pJK850, SmaI-HincII terminator fragment             region replacement KpnI fragment containing a <br /> inserted SmaI site BamHI fragment cloned           Cmr marker. The fragment derived the <br /> pMY310 pMY311. To construct pJK853, SalI-                plasmid constructed inserting Cmr marker an <br /> EcoRI fragment containing sufi region pHS132               SmaI-HincII fragment pJK272 SmaI site the <br /> cloned pSC138 joining SalI-EcoRI frag-            sufi region pMY311. To construct pJK858, the <br /> ment containing Apr marker; resultant plasmid              BamHI fragment containing suft region pLC4-14 was <br /> digested EcoRI partially BamHI excise            replaced BamHI fragment containing Cmr marker <br /> EcoRI-BamHI fragment containing sufi Apr                 excised pJL3-4779; resultant plasmid received its <br /> gene, excised fragment ligated EcoRI-            unique SmaI site insertion Kmr marker contained in <br /> BamHI fragment containing F replicon pJK280.                   PvuII fragment pYS8. To construct pJK863, two <br /> pHS132 constructed inserting BamHI sufl frag-              derivatives pLC4-14 constructed: derivative, <br /> ment pLC4-14 -the BamHI site pACYC184. To                  BamHI fragment containing Cmr marker pJL3-4779 <br /> construct pJK877 pJK878, BamHI fragment contain-              inserted BamHI(1) site; other, the <br /> ing sufi region pLC4-14 inserted BamHI            BamHI fragment containing Kmr marker pYS10 was <br /> site pUC8; plasmids received BamHI frag-               inserted BamHI(2) site. Then, SmaI fragment <br /> ment opposite orientations. To construct pJK875                derived carrying Cmr marker was <br /> pJK876, PstI fragment containing sufi region              ligated SmaI fragment obtained partial <br /> similarly inserted PstI site pUC8;                digestion carrying Kmr marker. pJK864 <br /> plasmids received PstI fragment opposite orientations.         constructed inserting Kmr marker pYS10 into <br />  3970     KATO ET AL.                                                                                                       J. BACTERIOL. <br />  <br />                                                 -W1                        sodium dodecyl sulfate-polyacrylamide gel electrophoresis <br />                                        suf I                               (SDS-PAGE) 12% polyacrylamide gels, fluorography <br />              P PSP                    B(2)          SHB(lp p               described previously (16). <br />  PLC4-14 5                                                                    Protein fractionation gene product localization. Maxi- <br /> pMY3I 0                             B                 B    E           B   cells  suspended 300 ,ul methionine assay medium <br />                                                                    2    (Difco) labeled 60 min 37&#194;&#176;C divided two <br /> PMY311 <br />                        B        *tE. B                   B                 parts. During following procedure, 10 mM MgCl2 was <br />                                 tFt                                        included buffer containing 10 mM Tris hydrochloride <br />  pJK850                                          (S ) B E             sB   (pH 8.0) 0.005% 2-mercaptoethanol, part; 10 mM <br />  pJK853 <br />                    E               B                   tB                E EDTA added other. The cells were <br />                                                                            harvested,  washed, resuspended 300 RI1 buffer <br />                          (Ap)                          1 <br />                                     B                    B      B          described above. The cells disrupted times by <br />  pJK878                                                    t===            sonication 100 W 30 s Ohtake 520-2D <br />  pJK877 <br />                             B       B                  1ac                 sonicator. Each 150 ,ul sonicated preparation stored <br />                                 ITc                        pIP             ice analyzed total fraction. The residual 150-,ul <br />  pJK875 <br />                                 P                            _ = pportions centrifuged 42,000 rpm 30 min a <br />                           P     P <br />                                                            lac             Beckman LP42Ti rotor. The supematant reserved the <br />  pJK876                                                                    soluble fraction. The precipitate suspended 150 ,ul of <br />                               I c                                          buffer regarded insoluble (membrane) frac- <br />                                                   Clm <br />   pJK855 <br />              p <br />               - <br />                             Sa&apos;7                                           tion. <br />                                                                               Observation nucleoids. Nucleoids stained either <br />                Km                          Cm                              ethidium bromide (EtBr) (18) 4&apos;,6-diamidino-2- <br />                                                         B           P      phenylindole (DAPI) (26). For EtBr staining, cells washed <br />   pJK858 P IvS                                                             M9 medium suspended solution containing 10 <br />                                    Km                   Cm                 mM Tris hydrochloride (pH 7.4), 5 mM EDTA, 0.05% <br />                                                       Bj7                  2-mercaptoethanol,   0.05% chloramphenicol, 0.8% etha- <br />   pJt863 P                      B\7                                 P <br />                                                                            nol mixed equal volume 0.01% EtBr <br />                                    Cm                    Km                solution. For DAPI staining, cells 1 ml culture were <br />               aP                 Bt7                I                      fixed harvest adding 1 ml 5% glutaraldehyde <br />   pJK864                                                                   containing 10 mM EDTA left 4&#194;&#176;C 30 min. The fixed <br />                                                                 lkb.j      cells collected centrifugation, suspended 1 ml of <br />                                                                            buffer containing 10 mM Tris hydrochloride (pH 7.7), 1 <br />    FIG. 3. Restriction map plasmids carrying tihe sufi region. The      mM EDTA, 0.5 mM spermidine, 1 mM dithiothreitol and <br /> restriction map drawn described le gteInd Fig.2. B,       supplied 1 ml 5% glutaraldehyde. After centrifuga- <br /> BamHI; E, EcoRI; H, HindIII; Hp, HpaI; P, Plsa; Pv,     Pvure; <br /> SmaI; Sa, Sall; Sc, ScaI. Arrows tet lac represent the <br />                                                                         S, tion, cells suspended appropriate of <br /> direction transcription tet lac prom                         buffer combined equal volume of <br />                                                                            DAPI solution (2 ,ug/ml). The cells stained nucleoids <br />                                                                            observed photographed incident fluores- <br /> the BamHI(1) site Cmr marker f pJL3-4779               cence microscope equipped phase-contrast optics <br /> the BamHI(2) site pLC4-14; marker 1was excised                (Olympus AHB-RFL-LB). <br /> BamHI fragment respective plasmid, partial <br /> digestion BamHI applied second insertion.                                            RESULTS <br />    Preparation manipulation DNA. The techniques                    Location new par mutation sufi gene. Among <br /> preparation plasmid DNA, manipulation DNA,                       mutants thermosensitive mutant stock (33), two <br /> transformation described previously (16). To trans-                strains showed temperature-insensitive <br /> form polA(Ts) strain N1126 replacing plasmid (see               growth cross-streaked JA200(pLC4-14). These <br /> Results), procedure modified follows. The trans-                strains, JE11215 JE10572, identified par mutants <br /> formation mixtures left 0&#194;&#176;C 30 min. Heat shock                 morphology nucleoids cells incubated the <br /> was applied 30&#194;&#176;C 2 min, mixtures                       restrictive temperature. Figure 4 shows morphological <br /> preincubated 30&#194;&#176;C 2 h LB broth. The cells                   Par phenotype revealed EtBr-stained nucleoids of <br /> harvested incubated 30&#194;&#176;C ca. 20 h S-broth               JE11215 incubated restrictive temperature. The mu- <br /> containing appropriate drug. The drug-resistant recombi-                tated gene leading Par phenotype distinct from <br /> nants selected LB plates containing NaCl                    parB parB locus distant chromosome <br /> tetracycline kanamycin 42&#194;&#176;C. The acridine orange                     region covered pLC4-14 distinct <br /> method (10) used curing mini-F derivatives.                 parA inasmuch morphological Par phenotype of <br /> Chromosomal DNA prepared procedure parA mutant MFT110 (isolated PA3092 M. Ricard <br /> Cosloy Oishi (5).                                                      Y. Hirota) corrected plasmid equivalent to <br />    Analyses gene products. The preparation labeling              pLC4-14. The new par mutation referred parC. <br /> minicells containing appropriate plasmids immuno-                     The chromosome region carried pLC4-14 covers metC <br /> logical separation described previously (16).                 (6), par mutation mutants be <br /> Minicells prepared stationary-phase cultures.                    located ca. 65 min. E. coli chromosome map. This <br /> Preparation labeling maxicells carried                  confirmed mating transduction experiments for <br /> described Stoker et al. (32) recA strain JM109 parCJ215 JE11215, described Materials Meth- <br /> C600 recA. In vitro synthesis protein performed                ods. The cotransduction frequency parC1215 59% (29 <br /> coupled transcription-translation described previ-               49) metC 20% (5 25) glc. This places parC <br /> ously (25). The labeled gene products analyzed                     65.3 min metC assumed lie 65.0 min. <br />  VOL. 170, 1988 <br />  <br />  <br />  <br />  <br /> A <br />                                                                           CHROMOSOME PARTITION GENE IN E. COLI <br />  <br />                                                                               mW 1 2 3 4 5 6 <br />                                                                               ...6. <br />  <br />  <br />                                                                               69z0&gt;.o <br />                                                                                 4- <br />  <br />  <br />  <br />                                                                               30.0&gt; <br />                                                                                     . <br />  <br />                                                                               46.0 &gt; . <br />                                                                                          ~ ~. .  .. <br />                                                                                                       Awls <br />  <br />                                                                                                         _ <br />                                                                                                                   f: <br />                                                                                             WF~~~~~~~~~~~~~~~~~~~~: <br />                                                                                                                   ..... <br />                                                                                                                           .. <br />                                                                                                                                7 % 9 10 <br />                                                                                                                                 **..;.:!.:!.:I <br />  <br />  <br />                                                                                                                                      -_ <br />                                                                                                                                                  3971 <br />  <br />  <br />  <br />  <br />                                                                                             n :~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~... <br />                                                                                                         WE . ... <br />                                                                               18.4 <br />                                                                               1 2.3 <br />  <br />                                                                           FIG. 5. Identification gene products. Proteins syn- <br />                                                                        thesized vitro 14C-amino acid mixture (lanes 1 6) or <br />                       ----.---.........                                [35S]methionine direction (lanes 1 10) pJK800, (lane <br />                                                                        2) pJK809, (lane 3) pJK811, (lane 5) pMY310, (lane 6) pJK850, (lane <br />                                                                        7) pJK823, (lane 8) pJK813, (lane 9) pJK807. Lane 4 shows mass <br />                                                                        markers. The 25K protein produced pJK800 indicated with <br />                                                                        arrow lane 10. This protein present lanes 1 2 <br />                                                                        absent lanes 5 6, entirely confirmative <br />                                                                        lanes overexposed. <br />  <br />                                                                        region. The SmaI BamHI fragment pLC4-14 <br />                                                                        cloned pBR322 shown Fig. 2 3 obtain <br />                                                                        pJK800, pMY310, pMY311. While pJK800 corrected the <br />                                                                        thermosensitive defect JE11215, pMY310 pMY311 <br />                                                                        suppressed ftsl phenotype. The plasmids <br />                                                                        received BamHI fragment pLC4-14 in <br />                                                                        opposite orientations, suppression FtsI more <br />                                                                        distinct pMY310 pMY311. The parC gene should <br />                                                                        located PstI SmaI sites according to <br />                                                                        restriction map shown Fig. 2, supt gene lies in <br />                                                                        BamHI fragment, probably oriented transcription in <br />                                                                        direction Tc&apos; gene pMY310. The thermo- <br />                                                                        sensitivity parC transductant, JE8515, corrected <br />                                                                        parC region supf region introduced <br />                                                                        plasmid. <br />                                                                           Identification parC product. To identify parC <br />                                                                        product, translation terminator (Materials Methods) <br />                                                                        (17) inserted PvuII sites pJK800 to <br />                                                                        construct pJK809, shown Fig. 2. The PvuII site on <br />                                                                        pJK809 received terminator located the <br />                                                                        region essential expression parC gene because <br />                                                                        pJK809 did correct temperature-sensitive growth <br />                                                                        defect JE11215. The proteins synthesized vitro under <br />                                                                        direction plasmids analyzed PAGE. The <br />                                                                        synthesis protein apparent mass 75 kilodal- <br />                                                                        tons (kDa) detected pJK800 pJK809 (Fig. <br />                                                                        5, lanes 1 2); pJK809 directed instead synthesis a <br />   FIG. 4. Abnormal <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span> chromosome partition parC         protein migrating position 69 kDa (Fig. 5, lane <br /> mutant. Cells grown overnight S-broth 300C diluted 1:100    2). This 69,000-Mr (69K) product observed for <br /> into fresh S-broth. The diluted cells JE11215 (parC) grown     pJK800. The pattern protein products much <br /> for 3 h 40 30&apos;C, diluted 1:100, grown   plasmids. The smaller protein produced <br /> 3 h temperature. The diluted cells JE7611 (ftsI730) <br /> were grown 3 h 40&apos;C. The cells collected low-speed      pJK809 truncated polypeptide 75K <br /> centrifugation treated described Materials Methods.      protein. The 75K protein detected the <br /> (A) JE11215 (parC) grown 40&apos;C; (B) JE11215 (parC) grown          products directed pJK811 (Fig. 2) carrying HpaI <br /> 30&apos;C; (C) JE7611 (ftsI730) grown 420C. Bar, 10 I&#194;&#177;m.                 fragment, chromosomal region smaller carried by <br />                                                                        pJK800 (Fig. 5, lane 3). These results suggest 75K <br />                                                                        protein parC gene product. It ruled a <br />   Plasmid pLC4-14 carries stiff (sui) gene,              polar effect developed pJK809 terminator <br /> suppress thermosensitive defect ftsI (see Intro-            insertion, preventing expression parC gene located <br /> duction). Thus, chromosome region pLC4-14                   region distal gene 75K protein. However, <br /> analyzed parC stiff loci subcloning parts            described later section, polypeptide over- <br />  3972     KATO ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                       I <br />  <br />  <br />  <br />  <br />   FIG. 6. Correction Par phenotype parC+ plasmids. To confirm morphology resulting parC mutation, parC1215 was <br /> cotransduced metC+ C600 Met- advance introduction metC:: TnlO NK6027 obtain EJ812, strain was <br /> transformed plasmids carrying parC region. Growing cells EJ812 plasmid removed 30 42&#194;&#176;C. After <br /> 2 h incubation 42&#194;&#176;C, chloramphenicol added 250 ,g/ml incubation continued 30 min (39). The cultures were <br /> treated DAPI staining described Materials Methods observe morphology. (A) Without plasmid; (B) pJK800 (parC+); (C) <br /> with pJK811 (parC+); (D) pJK809 (parC interrupted translational terminator). See Fig. 3 parC regions plasmids. Note <br /> the prominent production nucleoidless cells A D. Bar, 5 ,um. <br />  <br /> produced increasing transcription DNA                  retaining nucleoids 2-h incubation 42&#194;&#176;C (Fig. 6A). <br /> region coding C-terminal portion 75K protein.              The parC phenotype manifested EJ812 corrected by <br /> To confirm correction ParC morphology                       pJK800 pJK811 (Fig. 6B C) pJK809, <br /> plasmids directed synthesis 75K protein, possible             unable direct synthesis 75K protein owing <br /> interference ParC unknown mutations elimi-                    introduction translation terminator (Fig. 6D). <br /> nated constructing EJ812 parC transductant                   Cloning parC mutant gene. In order confirm that <br /> derivative C600. EJ812 failed grow LB plates 42&#194;&#176;C             parC gene carried pLC4-14 inactivated the <br /> and surprisingly produced considerable fraction nu-                  interruption PvuII site, cloning mutant gene <br /> cleoidless cells manifestation ParC cells                carrying parC1215 undertaken. This accomplished <br />  VOL, 170, 1988                                                          CHROMOSOME PARTITION GENE IN E. COLI                      3973 <br />  <br />  <br />  <br />  <br />                                ..                                                 .. <br />  <br />   FIG. 7. Growth cells transformed plasmids carrying parC(Ts). N1126 (parC:: cat)(pJK818 [Ap]) subjected transfornmation <br /> treatment (A E) plasmid, (B F) pJK282 (Km&apos;), (C G) pJK816 (KMr parC+), (D H) pJK820 [KmW <br /> parC(Ts)] plated LB plates containing kanamycin 30&#194;&#176;C (A-D) 42&#194;&#176;C (E-H). <br /> by application compensating-replacing plasmid prin-           parC(Ts) defect ascribable parC gene this <br /> ciple (17) follows. A strain constructed         plasmid. <br /> parC gene chromosome inactivated pres-               Identfication supf gene product. In studying the <br /> ence mini-F plasmid having parC+ compensate              nature sujp gene detail, effect gene <br /> the loss parC function (a compensating plasmid). Cloning          growth ftsl mutant examined. The <br /> of parC performed using mini-F vector             suppression weak pMY310 weaker <br /> displaces compensating plasmid. N1126 (polA12)               pMY311. The readthrough transcription tet <br /> transformed pJK816 (Kmr parC+), served               promoter contributed sup7 expression in <br /> compensating F-replicon plasmid, pJK815                pMY310, elevated level sufi gene product may <br /> (Cmr Tcr Apr), constructed inserting Cmr gene          crucial suppression, suggested previously (34). <br /> PvuII site parC gene (Fig. 2). The plasmid         The suppression observable indUced <br /> served replacing plasmid replace parC gene        state BamHI fragment inserted the <br /> chromosome inactivated parC gene plasmid             inducible promoter-operator correct orientation when <br /> through genetic recombination. This strain grown        using low-copy-number plasmid. This examined with <br /> 30&#194;&#176;C propagated 42&#194;&#176;C cure population           pJK853 (Fig. 3), replicon mini-F the <br /> the replacing plasmid, replicate             BamHI fragment joined lac promoter-operator <br /> polA(Ts) mutation. In population, cells Cmr           region pJK280 (Fig. 1) orientation the <br /> Tcs Aps. Some acquired Cmr                  transcription lac promoter proceed this <br /> insertion chrorhosomal parC locus exchange            fragment tet promoter BamHI fragment <br /> the plasmid-carried region prior loss plasmid. One         pMY310. The ftsl mutant carrying pJK853 showed ther- <br /> such isolate designated N1126(parC: :cat)(pJK816                 moresistance induction isopropylP-D-thioga- <br /> [Kml). Its structure confirmed following exper-           lactopyranoside (IPTG), extent shown by <br /> iments. N1126(parC::cat)(pJK816 [Kmi]) Kms               strain carrying pMY311. Thus, suppression theftsI <br /> replacing pJK816 pJK818 (Apr), having repli-           mutation sufi gene results increase the <br /> con compensating capacity pJK816. The resultant               suff gene product. <br /> Kms strain transformed pJK282 (Kmi)                    The sufl product identified inserting terminator <br /> pJK816 (Ktn). The transformants developed pJK816             fragment region essential sufj activity and <br /> 42&#194;&#176;C 30&#194;&#176;C, pJK282               analyzing products synthesized vitro. A 55K polypep- <br /> temperature (Fig. 7B, C, F, G). This shows                  tide present products synthesized vitro with <br /> N1126(parC::cat)(pJK818 [Apr]) viable pres-           pMY310 absent synthesized pJK850, <br /> ence compensating plasmid. Thus, insertion             constructed inserting terminator fragment the <br /> present chromosome original com-           SmaI site BamHI fragment cloned pMY310 <br /> pensating locus. This strain used host             (Fig. 5, lanes 5 6). The ftsl mutant carrying pJK850 <br /> transformation plasmids ligation pJK282              remained thermosensitive. The BamHI fragment in- <br /> (Kmr) BamHI fragments JE11215 chromosomal                serted BamHI site pUC8 construct pJK877 and <br /> DNA. Transformants selected KMr 30&#194;&#176;C. These              pJK878 (Fig. 3). The plasmid BamHI fragment <br /> had lost pJK818 (Apr) displacement. The plasmid pJK820            orientation pJK853 respect the <br /> (Km&apos;) isolated candidate containing parCJ215.           lac promotor. A remarkable overprod-uction 55K <br /> When N1126(parC:: cat)(pJK818 [Apr]) transformed                 protein observed in vitro (Fig. 8A, lanes 1 2) and <br /> with pJK820 selected Kmr, colonies Kmr trans-             maxicell protein synthesis (Fig. 8B, lanes 3 6) pJK878 <br /> formants developed normally 30&#194;&#176;C scarcely at           pJK877. These results strongly support the <br /> 42&#194;&#176;C (Fig. 7D H). This indicated DNA region             interpretation 55K protein encoded sufi <br /> containing parC1215 locus cloned plasmid.            gene. The maxicell experiments shown Fig. 8B revealed <br /> From follows ability pLC4-14 correct      extra band migrated slightly faster 55K <br />  3974         KATO ET AL.                                                                                                                  J. BACTERIOL. <br />  <br />             A 1 2        4   B      1   2         3 4 5 6             7 8 <br />                                                                                           TABLE 2. Plating efficiency cells cured pJK818 (Ap) <br />                                                                                                             acridine orange <br />                                                                                                                                  No. ApS colonies/100 <br />                                                                                                                                   colonies examined at <br />                                                                                           Host strain          Expt no.          acridine orange concn: <br />                                                                                                                                0 ,ug/ml          25 Fsg/ml <br />                                                                                        N1126                      I               1                 63 <br />                                                                                                                   II              1                 62 <br />                                                                                        Nl126 Cmr Kms                              0                  0 <br />                                                                                                                   II              0                  0 <br />  <br />  <br />                                                                                        Kms clone 200 Cmr transformants exam- <br />   FIG.      8.   Overproduction        sufi       25K    proteins.   [5S]   ined, percent Cmr transformants <br /> methionine-labeled  proteins synthesized vitro (A)                       Kms parallel experiment pJK855, had <br /> maxicells  (B) direction (lanes A-1, B-5, B-6)                        Cmr Kmr markers, shown Fig. 3. The transforma- <br /> pJK877, (lanes A-2, B-3, B-4) pJK878, (lanes A-3 B-i)                          tion repeated N1126 cells having pJK853 a <br /> pJK876, (lanes A-4, B-29, B-7, B-8) pJK875. The labeling                        compensating plasmid. This plasmid carried same <br /> procedures described Materials Methods. (B) Odd- <br /> numbered lanes proteins synthesized maxicells <br />                                                                                        BamHI fragment lost pJK858 led overpro- <br /> incubated presence glucose, even-numbered lanes <br />                                                                                        duction sufi gene product described above <br /> show labeled induction lac promoter IPTG in <br />                                                                                        section. The result (no Cmr Kms cells were <br /> the maxicells incubated absence glucose. Lanes 7 8                       found) presence compensating plasmid, <br /> show patterns              30-mmn   chase followed 10      suggesting function essential DNA region was <br /> min     labeling. Arrowheads panel A indicate suji product <br />                                                                                     destroyed deletion BamHI fragment not <br /> (top) 25K protein. Arrowheads left lanes B-i, B-3,                   supplied compensating plasmid spanned <br /> and B-7 indicate precursor (top) mature (middle) forms                      interrupted BamHI sites [BamHI(M) and <br /> the suft product 25K protein (bottom), present. The sufi <br />                                                                                        BamHI(2) Fig. 3]. Two replacing plasmids, pJK863 and <br /> product synthesized vitro coffesponds precursor form <br />                                                                                        pJK864, constructed. Both drug resistance <br /> synthesized maxicells judged electrophoretic mobility. <br /> The band indicated 25K protein, band <br />                                                                                        genes; pJK863 Cmr gene BamHI(1) site the <br /> immediately it, agreed relative mobility 25K <br />                                                                                        Kmr gene BamHI(2) site, pJK864 Kmr <br /> protein band shown Fig. 5, upper band                         gene BamHIMl) site Cmr gene BamHI(2) <br /> intensified lane A-3. The cause increase upper-band                  site (Fig. 3). When transformation selection per- <br /> protein lane A-3 unknown. (The conclusion reading                        formed using N1126 cells described above, Km&apos; <br /> direction gene 25K protein consistent                      colonies obtained Cmr transformants iso- <br /> unpublished sequence data.)                                                            lated pJK864 pJK863. The result suggested <br />                                                                                        existence essential gene lost function the <br /> protein     band. The    faster-migrating protein observed                     interruption BamHI(1) site. Since pJK807 (Fig. 2), <br /> in vitro      products. The maxicells synthesizing 55K                      having insertion translational terminator the <br /> protein labeled 10 min chased 30 min. The <br /> intensity faster-migrating band increased 55K <br />                                                                                        BamHI(M)    site, rendered JE11215 (parC) thermoresistant, the <br />                                                                                        gene inactivated interruption BamHI(1) site <br /> protein band decreased chase, demonstrated                             parC gene. <br /> the pattern obtained pJK875 (Fig. 8B, lanes 7 8).                                Isolation Cmr Kms strain carried by <br /> This suggests sufi product undergoes processing.                              using pJK863 replacing plasmid presence a <br />   To examine sufi gene dispensable                                 compensating plasmid, pJK818 (Apr), mini-F <br /> growth, replacing-plasmid method applied isolate                            derivative carries SmaI fragment covering the <br /> a mutant lost sufi gene function (see preceding <br />  <br /> section). Strain N1126 transformed pJK855 (Cmr <br />                                                                                        BamHI(M)    site (Fig. 2). In presence compensating <br />                                                                                        plasmid, Kms clones obtained 100 Cmr <br /> KMr)   sufi region interrupted insertion                           transformants. One Cmr Km&apos; clones investigated <br /> of cat gene (Fig; 3), selected CMr                                  viability scoring plating efficiency curing <br /> restrictive temperature             PoIA       function. SiX Of             CMr    compensating plasmid (pJK818) acridine orange. <br /> transformants           Km&apos;.   When         Cm&apos;     gene       inserted   As shown Table 2, Aps colony obtained the <br /> into parC gene,            CMr Tcs         transformant        obtained     Cmr Km&apos; recombinant, parental strain carrying <br /> by examining similar numbers cells unless compensating                            pJK818 gave rise Aps colonies frequency about <br /> plasmid present. Therefore, isolation Cm&apos;                               60%. The results strongly suggest gene operon <br /> Km&apos; transformants using pJK855 suggests dispens-                                covering BamHI(l) site indispensable cell viability. <br /> ability sufl gene function cell viability,                            The gene product affected interruption the <br /> further analyses required definite conclusion.                               BamHI(1) site investigated SDS-PAGE analyses of <br />    Identification essential gene parC anid sufi.                         vitro products pJK807 pJK800. These <br /> Construction sufi deletion strain attempted                                plasmids contained chromosome region that <br /> establish dispensability sufi gene cell viability.                      carried pJK818 insertion translation <br /> The BamHI fragment containing sufl gene pLC4-14                                 terminator BamHI(l) site pJK807 (Fig. 2). A ca. <br /> was replaced CMr gene, KMr gene                                   25K protein band intensified lane pJK800 <br /> inserted SmnaI site ColEl region pLC4-14                         compared pJK807 (Fig. 5, lanes 9 10, <br /> produce pJK858 (Fig. 3). Cells N1126 transformed                               arrow). It inferred, following considerations, that <br /> with pJK858 plated 420C. Contrary expectation,                            increased band intensity synthesis the <br />  VOL. 170, 1988                                                               CHROMOSOME PARTITION GENE IN E. COLI                      3975 <br />  <br />                                  P PSP                            P B(2)                              S B(1) P                  p <br />                                                               I                                I   I7       a <br />                    pLC4-14 <br />  <br />                                                              S         Hp.1 aB(1)       Pv   Pa PvI                    Hp <br />                                                              a <br />                                                                                     I <br />                                                                                         I <br />                                                                                                       ...               u Fkh <br />                                                                                                                        1kb <br />                                                    &lt;J -          -3z &lt;       _Z                                a <br />  <br />  <br />                                                          suf I             25K               parC <br />   FIG. 9. Possible location genes parC, sufi, 25K protein. The arrows map indicate roughly estimated <br /> location direction transcription parC gene, suff gene, gene 25K protein. The ends sufi gene the <br /> gene 25K protein inferred present work shown dotted lines. Two BamHI sites pLC4-14 described <br /> as B(1) B(2) map pLC4-i4 convenience. B, BamHI; Hp, HpaI; P, PstI; Pv, PvuII; S, SmaI. The symbols map, sufl, <br /> 25K, parC, represent suji gene, gene 25K protein, parC gene, respectively. <br />  <br /> 25K protein encoded gene covers BamHI(1)                   protein parC gene transcribed direction <br /> site protein band seen position                PvuII BamHI(1) site. Considering coding <br /> pJK807 related 25K protein. The protein band                capacity required 75K protein distance 0.6 <br /> at 25K position (background 25K band),                      kb PstI site PvuII sites, the <br /> pJK807, invariably far dense plasmids carry-               parC coding region cover region containing the <br /> ing region cut interrupted BamHI(1), site             PstI site rightward PvuII site. <br /> for pJK811, observed short exposures. The level                    To investigate effect transcription proceed <br /> of background 25K band essentially unaffected                      parC gene expression gene the <br /> whether BamHI(1) site cut (e.g., pJK823) inter-                 25K protein sufi gene, pJK875 pJK876 were <br /> rupted insertion translation terminator (pJK807);               constructed. These plasmids received PstI fragment <br /> deletion BamHI(1) site (e.g., pMY310                 opposite orientations (Fig. 3); pJK875 direction of <br /> and pJK823 pJK813); insertion orientation                  parC transcription matches lac promoter. The <br /> BamHI fragment Tcr gene, expression                    products vitro protein synthesis directed these <br /> the suff gene fragment affected (pMY310                    plasmids analyzed. The 25K protein sufl <br /> pMY311). A similar 25K band present pattern                product synthesized pJK875 higher level than <br /> of pJK811, thinner pMY310, suggest-                synthesized pJK876 (Fig. 8A, lanes 3 4). This <br /> ing essential gene 25K protein inactive             result suggests gene 25K protein, sujp <br /> pJK811. In plasmid BamHI(M) site present                  gene, parC gene ate transcribed same <br /> the chromosome region cut HpaI site (Fig. 2).                   direction. The gene products examined maxi- <br /> Thus, concluded 25K protein identified                   cells. Under IPTG-induced condition, maxicells <br /> pJK800 product essential gene containing                 harboring pJK875 synthesized larger amounts 25K <br /> BamHI(1) site also, probably, HpaI site. (This                     protein sufi product harboring pJK876 <br /> conclusion consistent unpublished sequence                    (Fig. 8B, lanes 1 2). These results suggest operon <br /> data.)                                                                     exists, containing parC gene, gene 25K <br />   Possible operon formation parC, sufi, 25K                     protein, sufp gene order proximal <br /> protein genes. The insertion translation terminator              side. However, gene 25K protein supf <br /> the PvuII site located closer BamHI(j) site (pJK809)                gene significantly expressed DNA fragment <br /> resulted production smaller parC protein, suggesting               connected opposite orientation lac promoter; <br /> that PvuII site located distal end              e.g., pJK876 genes pJK877 the <br /> parC coding region. Since BamHI(1) site located                     sufi gene. Thus, genes own <br /> outside parC gene, parC coding region expected               promoter. Based results section, coding <br /> to end PvuII BamHI(1) sites. Therefore,                regions direction transcription genes <br /> a short deletion extending BamHI(1) site               schematically represented Fig. 9. <br /> PvuII site introduced, effect protein                      Localization gene products cells. The location of <br /> product examined. For purpose, pJK813 con-                    gene products investigated maxicells labeled <br /> structed appropriate digestion exonuclease BaI31                [35S]methionine direction plasmid con- <br /> the BamHI site pJK823 subsequent insertion                     taining corresponding gene(s). The plasmids used were <br /> translation terminator digested site (Fig. 2). The                pJK803 carrying parC gene gene 25K <br /> deletion created BamHI site pJK813 shortened                 protein, pJK878 carrying suf7 gene, pJK878 carrying <br /> BamHI(1)-PvuII region 0.7 kilobases (kb)                       relevant genes. The maxicells lysed by <br /> BamHI(1) site. The vitro protein synthesis directed                  sonication fractionated centrifugation presence <br /> pJK813 showed production protein smaller                       10 mM MgCl2 10 mM EDTA, soluble and <br /> apparent size 75K protein regarded parC                    insoluble fractions analyzed SDS-PAGE pre- <br /> product larger 69K protein resulting                 cipitating proteins trichloroacetic acid (TCA). <br /> terminator insertion PvuII site pJK809 (Fig. 5, lanes               The 25K protein exclusively the <br /> 7 8, 1 2). The result consistent interpre-             insoluble fraction, distribution affected by <br /> tation 69K protein represents truncated parC                    presence absence Mg2+, parC protein <br />  3976         KATO ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  A   1 2 3     4    5   6   B 1   2   3   4   5   6   C 1   2   3   4    5   6   tion localization gene products). The Mg2+- <br />                                                                                  dependent localization sopB protein <br />                                                                                  involved partition mini-F plasmid (K. Nagai, S. <br />                                                                                  Inamoto, S. Hiraga, T. Ogura, G. Tamura, Annu. Meet. <br />                                                                                  Mol. Biol. Soc. Jpn. 1984, abstr. no. 7:B-17). The similarity <br />                                                                                  Mgs+ dependence suggest property common the <br />                         -~W <br />                                                                                  proteins engaged partitioning replicons. Alternatively, <br />                                                                                  suggest involvement parC protein the <br />                                                                                  interaction sopB product mini-F. In case, the <br />                                                                                  parC protein form complex sopB product <br />                                                                                  localization. The basic mechanism of <br />    FIG. 10. Localization parC sufp gene products              partition the mini-F the <br /> 25K protein. [355]methionine-labeled proteins synthesized                chromosome, postulated Jacob (15). <br /> maxicells JM109 direction plasmids pJK803 (A),                      The suff product identified 55K protein found <br /> pJK878 (B), pJK875 (C) fractionated described                      synthesized precursor form. The sufi products <br /> Materials Methods. The fractionated protins precipitated                largely soluble fraction localization <br /> by addition 20 ,ul 50% TCA analyzed SDS-PAGE                     experiments. The sufi gene originally assumed sup- <br /> described Materials Methods. Lanes 1 4 total                 press ftsI overproduction gene products (34). This <br /> fraction, lanes 2 5 soluble fraction, lanes 3 6             confirmed present study. Overproduction was <br /> show insoluble fraction. Lanes 1 3 fractiorns <br /> prepared presence EDTA, lanes 4 6                    required correction temperature-sensitive <br /> prepared presence Mg2&apos;. Arrowheads left lane 1               defect ftsI correction generally <br /> indicate (from top) parC product 25K protein (A),            imperfect. The function sufi product cell not <br /> precursor mature form sufi product (B), precursor             obvious appears dispensable cell viability. <br /> and mature forms sufi product 25K protein (C).                    However, vital function redundant due <br />                                                                                  existence protein equivalent futnction. <br />                                                                                  The mechanism suppression ftsl under- <br /> was detected soluble fraction absence                  Mg2+     stood. The ftsI product PBP3, participates septal <br /> and insoluble fraction presence Mg2+ (Fig.                      murein synthesis (2, 31, 33), active center PBP3 <br /> 10A). The 25K protein probably localized                          exist periplasmic space murein lies. Con- <br /> membrane, parC product appears incor-                        sidering existence precursor form, sufi <br /> porated membrane attachment membrane                         products exist periplasmic space help to <br /> depends         Mg2&quot;. <br />                      The majority sufi products,                     stabilize mutant PBP3, presumably weak inter- <br /> the precursor mature form, present                           actions attained overproduction sufi protein. <br /> soluble fraction irrespective presence absence                         Analyses parC-sufl region elucidated gene <br /> Mg2&apos;.  However, small precursor                         indispensable cell viability. The genes, parC, the <br /> detected insoluble fraction presence Mg2+                       gene 25K protein, sufp, probably constitute an <br /> (Fig. 10B C). The precursor 13-lactamase associated                       operon (or one). However, two <br /> with insoluble materials increased little                       genes appear promoters. It inter- <br /> presence Mg2+ preparations. Although                      esting clarify function 25K protein, is <br /> be suspected mature form sufi protein                         indispensable cell viability, investigate possi- <br /> released periplasm, localization protein                 ble overlapped regulation 25K protein synthesis. The <br /> not obvious present.                                                          isolation thermosensitive mutants defective 25K <br />                                                                                  protein useful step. It facilitated by <br />                               DISCUSSION                                         application compensating-replacing plasmid principle <br />                                                                                  utilized defining essential gene described the <br />   A new mutation affects chromosome partition E.                         present previous report (17). With approach, <br /> coli discovered termed parC, mapped 65.3 min                       mutants 25K protein isolated displacing <br /> P1 transduction. The parC gene product identified                       compensating plasmid mutagenized ones. These <br /> 75K protein SDS-PAGE. Proteins similar size                           mutants investigation. <br /> been seen oriC-membrane complexes (9, 24) F parti- <br /> tion-specific complexes (8).                                                                          ACKNOWLEDGMENTS <br />   The process chromosome partition presumably in-                                We thank Y. Suzuki K. Tanimoto plasmids, T. <br /> volves interaction chromosome cell surface                       Kuroiwa helpful suggestions nucleoid staining methods, <br /> structure, origin complex structure                    Keiko Kato excellent technical assistance. <br /> interaction. The 75K protein F partition-specific                      This work supported grants Ministry Educa- <br /> complex assumed originate background                             tion, Science, Culture Japan. <br /> synthesis minicells. The parC product identical <br /> to protein shared chromosome mini-F                                               LITERATURE CITED <br /> for partitioning. A genetic analysis proteins                     1. Appleyard, R. K. 1954. Segregation new lysogenic types <br /> would effective approach problem, analyses                          growth doubly lysogenic strain derived Esch- <br /> of par mutants help understanding roles                            erichia coli K12. Genetics 39:440-452. <br />                                                                                   2. Botta, G. A., J. T. Park. 1981. Evidence involvement of <br /> proteins contained DNA-membrane complex.                                      penicillin-binding protein 3 murein synthesis septation <br />   The parC product retained mem-                              cell elongation. J. Bacteriol. 145:333-340. <br /> brane fraction presence Mg2&quot; distributed                     3. Chang, A. C. Y., S. N. Cohen. 1978. Construction and <br /> soluble fraction absence Mg2+ (although present                        characterization amplifiable multicopy DNA cloning vehicles <br /> approach does exclude possible effects overproduc-                            derived P1SA cryptic mini plasmid. J. Bacteriol. 134: <br />  VOL. 170, 1988                                                               CHROMOSOME PARTITION GENE IN E. COLI                          3977 <br />  <br />     1141-1156.                                                                NRl. J. Bacteriol. 141:87-99. <br />  4. Clarke, L., J. Carbon. 1975. Biochemical construction         22. Miller, J. H. 1972. Experiments molecular genetics. Cold <br />     selection hybrid plasmids containing specific segments E.           Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     coli genome. Proc. Natl. Acad. Sci. USA 72:4361-4365.                 23. Mori, H., A. Kondo, A. Ohshima, T. Ogura, S. Hiraga. <br />  5. Cosloy, S. D., M. Oishi. 1973. The nature transfor-            1986. Structure function F plasmid genes essential for <br />     mation process Escherichia coli K12. Mol. Gen. Genet. 124:             partitioning. J. Mol. Biol. 192:1-15. <br />     1-10.                                                                 24. Nagai, K., W. Hendrickson, R. Balakrishnan, H. Yamaki, D. <br />  6. Dwivedi, C. M., R. C. Ragin, J. R. Uren. 1982. Cloning,               Boyd, M. Schaechter. 1980. Isolation replication origin <br />     purification, characterization 3-cystathionase Esch-          complex Escherichia coli. Proc. Natl. Acad. Sci. USA 77: <br />     erichia coli. Biochemistry 21:3064-3069.                                  262-266. <br />  7. Harayama, S., G. L. Hazelbauer. 1982. A gene coding         25. Nakamura, M., I. N. Mauayama, M. Soma, J. Kato, H. Suzuki, <br />     periplasmic protein located near locus termination          Y. Hirota. 1983. On process cellular division in <br />     chromosome replication Escherichia coli. J. Bacteriol. 151:            Escherichia coli: nucleotide sequence gene penicillin- <br />     1391-1396.                                                                binding protein 3. Mol. Gen. Genet. 191:1-9. <br />  8. Hayakawa, Y., T. Murotsu, K. Matsubara. 1985. Mini-F              26. Nishibayashi, S., T. Kuroiwa. 1982. Behavior leucoplast <br />     protein binds unique region partition mini-F. J.         nucleoids epidermal cell onion (Allium cepa) bulb. <br />     Bacteriol. 163:349-354.                                                   Protoplasma 110:177-184. <br />  9. Hendrickson, W. G., T. Kusano, H. Yamaki, R. Balakrishnan,            27. Nishimura, Y., Y. Takeda, A. Nishimura, H. Suzuki, M. Inoue, <br />     M. King, J. Murchie, M. Schaechter. 1982. Binding              Y. Hirota. 1977. Synthetic ColEl plasmids carrying genes <br />     origin replication Escherichia coli outer membrane.          cell division Escherichia coli. Plasmid 1:67-77. <br />     Cell 30:915-923.                                                      28. Nordstrom, K., S. Molin, H. Aagard-Hansen. 1980. Parti- <br /> 10. Hirota, Y. 1960. The effect acridine dyes mating type               tioning plasmid Rl Escherichia coli. I. Kinetics loss of <br />     factors Escherichia coli. Proc. Natl. Acad. Sci. USA 46:57-            plasmid derivatives deleted par region. Plasmid 4:215- <br />     64.                                                                       227. <br /> 11. Hirota, Y., A. Ryter, F. Jacob. 1968. Thermosensitive             29. Norris, V., T. Alliotte, A. Jaffe, R. D&apos;Ari. 1986. DNA <br />     mutants E. coli affected processes DNA synthesis             replication termination Escherichia coli parB (a dnaG allele), <br />     cellular division. Cold Spring Harbor Symp. Quant. Biol.              parA, gyrB mutants affected DNA distribution. J. Bac- <br />     33:677-693.                                                               teriol. 168:494-504. <br /> 12. Hirota, Y., M. Ricard, B. Shapiro. 1971. The use               30. Ogura, T., S. Hiraga. 1983. Partition mechanism F <br />     thermosensitive mutants E. coli analysis cell                plasmid: plasmid gene-encoded products cis-acting <br />     division. Biomembranes 2:13-31.                                           region involved partition. Cell 32:351-360. <br /> 13. Huisman, O., R. D&apos;Ari, J. George. 1980. Further character-        31. Spratt, B. G. 1975. Distinct penicillin-binding proteins involved <br />     ization sfiA sfiB mutations Escherichia coli. J. Bacte-         cell division, elongation, shape Escherichia coli <br />     riol. 44:185-191.                                                         K12. Proc. Natl. Acad. Sci. USA 72:2999-3003. <br /> 14. Huisman, O., R. D&apos;Ari, S. Gottesman. 1984. Cell-division          32. Stoker, N. G., J. M. Pratt, I. B. Holland. 1984. In vitro gene <br />     control Escherichia coli: specific induction SOS                expression systems prokaryotes, p. 153-178. In B. D. Hames <br />     function SfiA protein sufficient block septation. Proc. Natl.       S. J. Higgins (ed.), Transcription translation. IRL, <br />     Acad. Sci. USA 81:4490-4494.                                              Oxford. <br /> 15. Jacob, F., S. Brenner, F. Cuzin. 1963. On regulation       33. Suzuki, H., Y. Nishimura, Y. Hirota. 1978. On process <br />     DNA replication bacteria. Cold Spring Harbor Symp. Quant.              cellular division Escherichia coli: series mutants E. <br />     Biol. 28:329-348.                                                         coli altered penicillin-binding proteins. Proc. Natl. Acad. <br /> 16. Kato, J., H. Suzuki, Y. Hirota. 1984. Overlapping              Sci. USA 75:664-668. <br />     coding regions -y components penicillin-binding          34. Takeda, Y., A. Nishimura, Y. Nishimura, M. Yamada, S. <br />     protein lb Escherichia coli. Mol. Gen. Genet. 196:449-457.             Yasuda, H. Suzuki, Y. Hirota. 1981. Synthetic ColEl <br />                                                                               plasmids carrying genes penicillin-binding proteins Esch- <br /> 17. Kato, J., H. Suzuki, Y. Hirota. 1985. Dispensability        erichia coli. Plasmid 6:86-98. <br />     penicillin-binding protein-la -lb involved essential        35. Tanimoto, K., T. Iino. 1984. An essential gene replica- <br />     process cell elongation Escherichia coli. Mol. Gen. Genet.         tion mini-F plasmid origin I. Mol. Gen. Genet. 196: <br />     200:272-277.                                                              59-63. <br /> 18. Kuroiwa, T., S. Nishibayashi, S. Kawano, T. Suzuki. 1981.         36. Timmis, K., F. Cabello, S. Cohen. 1975. Cloning, isolation, <br />     Visualization DNA various phages (T4, X, T7, 429)                characterization replication region complex plasmid <br />     ethidium bromide epifluorescent microscopy. Experimenta 37:               genomes. Proc. Natl. Acad. Sci. USA 72:2242-2246. <br />     969-970.                                                              37. Yamada, M., Y. Hirota. 1981. Isogenic allogenic <br /> 19. Lutkenhaus, J. F. 198 <br /> </body></html>