<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genetic Interactions MAF1 Identify Role Med20 in <br /> Transcriptional Repression Ribosomal Protein Genes <br /> Ian M. Willis1*, Gordon Chua2&#194;&#164;a, Amy H. Tong2&#194;&#164;b, Renee L. Brost2, Timothy R. Hughes2, Charles Boone2, <br /> Robyn D. Moir1 <br /> 1 Department Biochemistry, Albert Einstein College Medicine, Bronx, New York, United States America, 2 Banting Best Department Medical Research, <br /> University Toronto, Toronto, Ontario, Canada <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Transcriptional repression ribosomal components tRNAs coordinately regulated response wide variety of <br />      environmental stresses. Part response involves convergence different nutritional stress signaling pathways <br />      Maf1, protein essential repressing transcription RNA polymerase (pol) III Saccharomyces cerevisiae. Here <br />      identify functions buffering yeast cells unable down-regulate transcription RNA pol III. MAF1 genetic <br />      interactions identified screens non-essential gene-deletions conditionally expressed essential genes reveal highly <br />      interconnected network 64 genes involved ribosome biogenesis, RNA pol II transcription, tRNA modification, ubiquitin- <br />      dependent proteolysis processes. A survey non-essential MAF1 synthetic sick/lethal (SSL) genes identified six <br />      gene-deletions defective transcriptional repression ribosomal protein (RP) genes following rapamycin <br />      treatment. This subset MAF1 SSL genes included MED20 encodes head module subunit RNA pol II Mediator <br />      complex. Genetic interactions MAF1 subunits structural module Mediator investigated <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      examine functional relationship transcriptional regulators. Gene expression profiling identified a <br />      prominent highly selective role Med20 repression RP gene transcription multiple conditions. In <br />      addition, attenuated repression RP genes rapamycin observed strain deleted Mediator tail module <br />      subunit Med16. The data suggest Mediator Maf1 function parallel pathways negatively regulate RP mRNA and <br />      tRNA synthesis. <br />  <br />   Citation: Willis IM, Chua G, Tong AH, Brost RL, Hughes TR, et al. (2008) Genetic Interactions MAF1 Identify Role Med20 Transcriptional Repression of <br />   Ribosomal Protein Genes. PLoS Genet 4(7): e1000112. doi:10.1371/journal.pgen.1000112 <br />   Editor: Michael Snyder, Yale University, United States America <br />   Received January 7, 2008; Accepted May 28, 2008; Published July 4, 2008 <br />   Copyright: &#195;&#376; 2008 Willis et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institutes Health grant GM42728 funds Albert Einstein College Medicine IMW by <br />   grants Canadian Institute Health Research CB TRH. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: willis@aecom.yu.edu <br />   &#194;&#164;a Current address: Institute Biocomplexity Informatics, Department Biological Sciences, University Calgary, Calgary, Alberta, Canada <br />   &#194;&#164;b Current address: Genome Research Center, Li Ka Shing Faculty Medicine, University Hong Kong, Pokfulam, Hong Kong, Special Administrative Region, <br />   People&#226;&#8364;&#8482;s Republic China <br />  <br />  <br />  <br />  <br /> Introduction                                                                          nents structure pathways, targets and <br />                                                                                       mechanisms action. <br />    Nuclear gene transcription proliferating cells dedicated                        Studies RP gene transcription identified several <br /> primarily synthesis ribosomes tRNAs. As illustrated                     regulatory factors including Sfp1, Rap1, Fhl1, Ifh1 Crf1 [5&#226;&#8364;&#8220; <br /> by studies Saccharomyces cerevisiae, doubling cell mass                7,9] references unclear proteins <br /> each cell cycle involves production ,200,000 ribosomes                         communicate general RNA pol II transcription <br /> along 3&#226;&#8364;&#8220;6 million molecules tRNA consumes .80%                            machinery. In contrast complexity, single negative <br /> of nucleotides needed transcription ,100 mi-                      regulatory protein, Maf1, appears serve conduit through <br /> nute interval [1&#226;&#8364;&#8220;3]. This expenditure metabolic energy tightly                  repression signals pass order affect transcription by <br /> regulated diverse signaling pathways sense quality                    RNA pol III [4,10]. The Maf1 protein interacts directly Brf1, <br /> quantity nutrients environmental stresses [1,2]. Under                          subunit initiation factor TFIIIB, RNA pol III <br /> conditions unfavorable cell growth, transcription                     interactions inhibit assembly function TFIIIB- <br /> rDNA tRNA genes RNA pols I III RNA pol II                              DNA complexes vitro [10,11]. The functional importance of <br /> transcription ribosomal protein (RP) genes rapidly                          interactions supported conservation yeast to <br /> coordinately repressed [1,4]. Current evidence suggests                     humans [12]. The essential role Maf1 repression RNA <br /> coordinate response results convergence specific                          pol III transcription demonstrates capacity integrate responses <br /> signaling pathways transcription components                         multiple nutritional stress signaling pathways that <br /> each polymerase [4&#226;&#8364;&#8220;9] references therein. However,                         coordinately regulate ribosome tRNA synthesis [13]. This <br /> substantial gaps understanding remain concerning compo-                        property Maf1 provides unique opportunities examine the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                       1                                July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                       Genetic Interactions MAF1 <br />  <br />  <br />  <br />   Author Summary <br />   The Maf1 protein essential negative regulator of <br />   transcription RNA polymerase III S. cerevisiae and <br />   functions integrate responses diverse nutritional <br />   stress signaling pathways coordinately regulate <br />   ribosome tRNA synthesis. These signaling pathways <br />   well-defined, efforts understand role of <br />   Maf1 process complicated lack of <br />   known functional motifs protein paucity of <br />   direct physical interactions Maf1. To understand the <br />   biological importance down-regulating RNA polymerase <br />   III transcription identify functional relationships <br />   Maf1, employed synthetic genetic array (SGA) <br />   analysis. We genetic neighborhood around <br />   Maf1 highly interconnected enriched small <br />   number functional categories, are <br />   logically linked function Maf1 regulator                 Figure 1. Genetic interactions MAF1 non-essential <br />   RNA polymerase III transcription. We                        gene deletions. Representative viable gene-deletion strains (G418- <br />   deletions subset MAF1 SSL genes, including                        resistant) confirmed random spore analysis having <br />   subunits RNA polymerase II Mediator complex, lead                  fitness defects maf1D arrayed quadruplicate crossed <br />   defects transcriptional repression ribosomal                     clonNat-resistant MAF1 (Y5518) maf1D (Y6338) query strains to <br />   protein (RP) genes. Since Mediator subunits                       compare growth haploid double-drug resistant strains following <br />   efficiently cross-linked RP genes chromatin,                    standard SGA protocol [15]. The final double-drug containing plates <br />   results suggest Mediator interactions                     incubated 30uC. A his3D strain included negative <br />   highly expressed genes fundamentally different                   control (i.e. interaction maf1D strain). <br />                                                                             doi:10.1371/journal.pgen.1000112.g001 <br />   genes. <br />  <br /> mechanisms signal integration, nature upstream                [17], additional 29 SSL interactions validated random <br /> pathways, downstream targets effects                 spore analysis triplicate screens maf1D query strain <br /> transcription machinery.                                                    Tet-promoter array. The entire collection 64 genes <br />    Yeast strains deleted MAF1 viable exhibit wild-type          exhibiting synthetic interactions MAF1 highly enriched for <br /> growth rates 10&#226;&#8364;&#8220;15% nuclear gene transcription            small number functional categories, are <br /> refractory repression [2]. Maf1 does contain motifs           logically linked function Maf1 transcriptional <br /> known function evidence variety sources suggests              regulator RNA pol III genes. Notably, 40% MAF1 SSL genes <br /> that majority Maf1 yeast stably associated            (26/64 genes, p,7.0E-18) involved ribosome biogenesis or <br /> other proteins normal repressing conditions: Co-                   translation (Table S1). Other functional categories are <br /> immunoprecipitation experiments 10&#226;&#8364;&#8220;20% cellular                represented significantly higher frequencies expected by <br /> Maf1 associated RNA pol III ,1% Maf1 associated                 chance include RNA pol II transcription (9 genes, p,5.0E-4), <br /> with Brf1 [10,11]. No significant interactions              tRNA modification (6 genes, p,4.0E-6) ubiquitin-dependent <br /> found affinity purification mass spectrometry protein             proteolysis (5 genes, p,7.9E-3). These data suggest important <br /> complexes yeast genome-wide hybrid screens [14].               functional relationships MAF1 genes these <br /> Given limited physical interactions Maf1, initiated             categories [18]. <br /> study functional relationships using synthetic genetic array            To determine relationships genes MAF1 <br /> (SGA) analysis. The local genetic neighborhood MAF1               genetic interaction network, SSL gene queried the <br /> highly interconnected enriched components                BioGRID database [14] compile list known genetic and <br /> protein complexes involved ribosome biogenesis RNA pol               physical interactions. These interactions superimposed on <br /> II transcription. We genetic interactions MAF1            set MAF1 SSL genes overlap displayed <br /> and subunits RNA pol II Mediator complex, particular              graphically using Osprey software (Figure 2). The resulting <br /> MED20, functionally linked common role repression            interaction network remarkably coherent; 70% (45/64) of <br /> tRNA RP gene transcription, respectively.                               MAF1 SSL genes connected genetic protein-protein <br />                                                                             interactions genes network. The majority of <br /> Results                                                                     interactions (47 gray edges 54 total interactions, <br />                                                                             Figure 2) determined multiple studies affinity <br /> Synthetic Genetic Array Analysis MAF1                                    purification mass spectrometry [14] identify components <br />    A maf1D strain screened triplicate ordered             known macromolecular complexes (the 26S <br /> array ,4700 viable gene-deletion strains relative                proteasome, ssu processome, exosome, pre-ribosomal <br /> growth double mutants scored computer-based                   processing intermediates, cytoplasmic Lsm complex, TFIID <br /> image analysis [15]. Random spore analysis used                 SAGA complexes RNA pol II Mediator complex). The <br /> validate candidate genetic interactions. The initial list MAF1           connectivity complexes suggests relatively small <br /> SSL interactions contained 35 genes (Figure 1 Table S1).                number biological explanations account ability of <br /> Subsequently, analysis extended array ,800                 MAF1 SSL genes buffer cells unable down-regulate <br /> strains containing different essential genes tetracycline (Tet)       RNA pol III transcription (see Discussion). <br /> promoter control [16]. Consistent ,five-fold higher                   Within broad functional category ribosome biogenesis, <br /> interaction density essential genes synthetic genetic networks        defects synthesis large small ribosomal subunits <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              2                            July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                         Genetic Interactions MAF1 <br />  <br />  <br />  <br />  <br /> Figure 2. MAF1 genetic interaction network. Genetic physical interactions BioGRID shown MAF1 SSL genes identified in <br /> screens essential non-essential strain arrays. Nodes colored Bioprocess. Circles identify well-defined protein complexes for <br /> the tRNA modification &amp; export group genes related biochemical function. Two MAF1 SSL genes (FYV5 YGL007W) lacked <br /> interactions BioGRID database shown figure. Genetic interactions shown green protein-protein interactions <br /> determined affinity purification shown gray. <br /> doi:10.1371/journal.pgen.1000112.g002 <br />  <br /> resulting impaired rRNA processing, reduced levels                   module Mediator complex, MAF1 SSL <br /> ribosomal proteins inefficient assembly yield synthetic             genes exhibiting defects repression RP genes. Given that <br /> phenotypes MAF1. Interestingly, genes (TIF6               role head module Mediator Med20 specifically, <br /> and RPL genes) previously shown block                   typically associated transcriptional repression, we <br /> repression rDNA RP gene transcription following                       confirmed effect deleting MED20 RPL3 RPL28 <br /> interruption secretory pathway [19,20]. Similarly,                mRNA levels northern analysis multiple biologically <br /> genetic interaction UTP22 MAF1 (Figure 2) suggests               independent samples (Figure S1C). In experiments, rapa- <br /> a functional relationship transcription processing           mycin-mediated repression med20D strain reduced 2.6&#226;&#8364;&#8220; <br /> of large rRNAs transcription RP RNA pol III               5.0 fold relative wild-type strain. This result led to <br /> genes. These functional associations reflect role Utp22          question subunit 25 subunit Mediator <br /> subunit ssu processome CURI complex                      complex [23] identified having genetic interaction with <br /> [21,22]. Based results, hypothesized MAF1             MAF1 (Figure 2). Estimates false negative rate SGA <br /> SSL genes ribosome biogenesis category, genes           screens [18] potential differences strength the <br /> some functional categories, play role                synthetic <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> suggested Mediator subunit <br /> regulating transcription ribosomal components. Indeed,              deletion strains exhibit fitness defects combination with <br /> survey non-essential MAF1 SSL genes revealed                 deletion MAF1. To examine possibilities, direct random <br /> rapamycin-mediated transcriptional repression RP genes                spore tests performed additional deletion strains <br /> substantially attenuated RPL20B, MRT4, KEM1, BUD20, LSM1                  representing Mediator subunits structural <br /> and MED20 mutant strains (Figure S1A S1B). Relative               modules complex; middle, tail Cdk modules. <br /> untreated wild type mutant controls, northern analysis            Growth haploid meiotic products conducted 30uC <br /> affected strains showed levels RPL3 RPL28 mRNAs              elevated temperatures noted MAF1 SSL <br /> following rapamycin treatment elevated fold               phenotypes frequently stronger conditions. This <br /> over wild type (Figure S1B). Along elevated levels RP            illustrated med20D maf1D strain shows conditional <br /> mRNAs seen cells depleted Utp22 Tif6 [20,22],            synthetic lethality 35uC (Figure 3 Figure S2B). <br /> it appears subset MAF1 SSL genes associated                While tested Mediator subunit deletion strains <br /> defects repression RP gene transcription.                          exhibited fitness defects maf1D 30uC, nine <br />                                                                              deletion strains showed reduced viability and/or slow growth at <br />                                                                              higher temperatures (Figure 3 data shown). Notably, <br /> Multiple Mediator Subunits Exhibit Genetic Interactions                      deletion MED16 (SIN4) conferred conditional synthetic lethality <br /> with MAF1                                                                    37uC. Consistent fact loss MED16 dissociates a <br />    In light preceding observations, especially                set physically interacting tail module subunits (including Med2, <br /> intrigued Med20 (Srb2), non-essential subunit head           Med3, Med15) rest complex [24], similar <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              3                             July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                          Genetic Interactions MAF1 <br />  <br />  <br />                                                                                 prepare Cy5- Cy3-labeled cDNAs. Pairs dye-reversed <br />                                                                                 cDNA samples hybridized spotted arrays yeast <br />                                                                                 ORFs. The resulting data filtered select genes whose <br />                                                                                 expression increased decreased two-fold the <br />                                                                                 pairwise comparisons (med20D/MED20, MED206rapamycin, <br />                                                                                 med20D6rapamycin med20D+rapamycin/MED20+rapamy- <br />                                                                                 cin, Table S2) subjected hierarchical clustering <br />                                                                                 (Figure 4). Several important conclusions emerged these <br />                                                                                 experiments: (i) Deletion MED20 does appreciably affect the <br />                                                                                 global <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene expression normal growth conditions: <br />                                                                                 Only 116 genes affected two-fold cutoff our <br />                                                                                 experiments. Using criteria, fewer genes were <br />                                                                                 affected previously reported comparison unstressed wild- <br />                                                                                 type med20D strains [25] (see Text S1). An analysis the <br />                                                                                 combined datasets shared GO Bioprocess terms indicates that <br />                                                                                 major cellular process ribosome biogenesis assembly, <br />                                                                                 translation, transcription, organization biogenesis the <br />                                                                                 nucleus, membranes cytoskeleton, other <br />                                                                                 processes, largely entirely unaffected deletion of <br />                                                                                 MED20 (Table S3). In particular, expression genes involved <br />                                                                                 synthesis, processing function RNA pol III transcripts <br />                                                                                 affected med20D strain RNA pol III gene <br />                                                                                 transcription effectively repressed rapamycin treatment the <br />                                                                                 absence MED20 (Figure S2A). Thus, function Med20 in <br />                                                                                 RNA pol III transcription discounted explanation for <br />                                                                                 genetic interaction MAF1. (ii) Rapamycin treatment the <br />                                                                                 wild-type strain showed characteristic response the <br /> Figure 3. Genetic interactions MAF1 multiple <br /> Mediator subunits. G418-resistant Mediator subunit deletions in <br />                                                                                 induction repression specific sets genes representing <br /> strain BY4741 crossed clonNat-resistant maf1D query strain            ,20% genome (Figure 4, Text S1, Figure S3). As <br /> (Y6338) genetic interactions assessed random spore                  reported studies ([26] references therein), RP genes <br /> analysis [15]. Growth 35uC (Med20) 37uC (all strains)         genes Ribi regulon involved ribosome biogenesis and <br /> compared haploid selection plates containing G418 G418                related functions strongly repressed rapamycin while <br /> clonNat, selects strains indicated genotypes. In         general amino acid control genes Gcn4-regulated <br /> absence effects strain viability, approximately equal numbers of <br /> haploid colonies expected media. Images haploid               genes strongly induced (Text S1). (iii) Within group of <br /> selection plates containing antibiotics clonNat (which selects       rapamycin-responsive genes, deletion MED20 selectively <br /> for maf1D) omitted clarity growth maf1D             diminished level induction repression (Figure 4). For <br /> single mutant indistinguishable wild-type. Deletion MED5,            example, level activation subset Gcn4-regulated <br /> MED9, MED31 cycC MED12 resulted synthetic growth             genes attenuated significantly: Of 197 genes whose <br /> defects maf1D 37uC random spore assay (data not <br />                                                                                 expression rapamycin treatment 2&#226;&#8364;&#8220;12 fold lower the <br /> shown). <br /> doi:10.1371/journal.pgen.1000112.g003                                           med20D strain wild-type strain, 74 (38%) Gcn4 <br />                                                                                 targets (p = 1E-32). Notably, genes involved amino acid <br />                                                                                 biosynthesis related metabolic processes highly enriched <br /> conditional synthetic phenotype observed deletion of <br />                                                                                 group (25 genes, GOID 6519, p = 7.34E-19, Figure 4C). <br /> MED3. In summary, results extend functional relation- <br />                                                                                 These results consistent requirement Mediator in <br /> ship MAF1 MED20 inferred genetic <br />                                                                                 activation specific Gcn4-regulated genes [24,27] extend <br /> interaction 30uC subunits structural module the <br />                                                                                 requirement larger group Gcn4-target promoters by <br /> Mediator complex. <br />                                                                                 identifying critical role Med20 activation following <br />                                                                                 rapamycin treatment. In addition, 97 138 RP <br /> Positive Negative Roles Med20                                    genes 170 genes expression following rapamycin <br /> Transcriptional Response Rapamycin                                           treatment 2 6-fold higher med20D strain the <br />    The finding multiple Mediator subunits interact genetically             treated wild-type strain (Figure 4B, Table S2). In agreement with <br /> with MAF1 suggests Mediator Maf1 function parallel                  expectations northern blotting specific RP mRNAs <br /> pathways. We considered buffering pathways                     (Figure S1), deletion MED20 compromises repression RP <br /> involve transcriptional response conditions repress                 genes rapamycin. The attenuated repression RP genes the <br /> ribosome tRNA synthesis role Maf1 repressing                absence Med20 highly specific repression genes the <br /> RNA pol III transcription entails integration signals               Ribi regulon, nearly identical transcriptional responses <br /> coordinately regulate processes [4,10,13]. To examine                 different environmental conditions [5,28], was <br /> function Med20 repressing conditions, conducted                     unaffected: Similar numbers Ribi genes down-regulated <br /> microarray experiments wild-type med20D strains                 rapamycin wild-type med20D strains (125 133 <br /> been treated (or not) rapamycin inhibit TOR signaling                   genes, respectively, two-fold cutoff). Moreover, six <br /> (microarray data available National Center                       Ribi genes (statistically equivalent random distribution) were <br /> Biotechnology Information GEO database accession                          170 genes exhibiting two-fold higher <br /> number GSE11397). Messenger RNA representing                        difference expression comparing rapamycin-treated <br /> four conditions (wild-type, med20D, 6rapamycin) used                     med20D wild-type strains. Thus, data indicate unique <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 4                           July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                           Genetic Interactions MAF1 <br />  <br />  <br />  <br />  <br /> Figure 4. Microarray expression profiles wild-type med20D strains treatment rapamycin. Clustergram <br /> comparisons gene expression profiles obtained dye-swap experiments conditions; left right panel, med20D/MED20, <br /> MED206rapamycin, med20D6rapamycin med20D+rapamycin/MED20+rapamycin. RNA samples prepared cells grown 30uC. <br /> Decreased (green) increased (red) expression shown relative wild-type strain untreated control. A The expression 1420 genes <br /> that increased decrease two-fold pair-wise comparisons subjected hierarchical clustering. B The <br /> repression RP genes (96 138 genes) rapamycin specifically attenuated med20D strain. The average level repression RP genes <br /> was two-fold med20D strain versus six-fold wild-type strain. C Rapamycin induction subset Gcn4-regulated genes <br /> is diminished significantly med20D strain. Expression ratios compared 25 Gcn4-regulated genes involved amino acid biosynthesis. <br /> doi:10.1371/journal.pgen.1000112.g004 <br />  <br />  <br />  <br /> and highly selective requirement head module subunit                   revealed similar profiles rapamycin, tunicamycin, CPZ <br /> Mediator repression RP gene transcription rapamycin.               treatments post-diauxic cells (Figure S4). These <br />                                                                                 similarities especially pronounced RP genes (Figure 5), <br />                                                                                 highly enriched genes exhibiting attenuated <br /> Med20 Is Required Efficient Repression RP Genes                          repression med20D strain (p values ranged 1.85E-9 to <br /> under Multiple Conditions                                                       1.7E-128). These data suggest integral role Med20 the <br />    RP genes coordinately down-regulated wide variety                repression RP gene transcription conditions. <br /> of <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-limiting stress conditions [1,28]. Virtually       In contrast, significant contribution Med20 evident the <br /> conditions cause Maf1-dependent repression RNA pol III                  down-regulation RP genes conditions oxidative mild <br /> transcription [4,10,13]. Given essential function Maf1                heat stress (Figure 5). The lack effect RP genes these <br /> conveying signals diverse pathways RNA pol III                  experiments apparently specific deletion MED20 clearly <br /> transcription machinery, interested know                     affected responses (Figure S4). For example, induction of <br /> Med20 serves general condition-specific role repressing RP              heat shock genes increased med20D strain following <br /> gene transcription. Microarray profiles generated pairs            heat stress (11 62 genes two-fold cutoff, p = 2.42E-8, <br /> fluor-reversed experiments wild-type med20D strains                   Table S4). The recruitment Mediator heat shock genes its <br /> were treated tunicamycin, chlorpromazine (CPZ), hydrogen                   requirement gene activation heat stress known [29,30] <br /> peroxide mild heat stress (29&#226;&#8364;&#8220;39uC). In addition, expression                 role Med20 process previously been <br /> profiles strains compared following diauxic shift           described. Similarly, characteristic induction oxidative <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> glucose fermentation respiratory metabolism. All               stress heat shock response genes hydrogen peroxide-treated <br /> conditions repress dramatically transcription RP genes [1,28].           cells increased substantially med20D strain (17 of <br /> Clustergram comparisons 1063 genes expression differed                 260 genes, p = 1.16E-7, Table S4). The contribution Med20 in <br /> two-fold conditions (including rapamycin),            response consistent previous work demonstrating the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 5                            July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                          Genetic Interactions MAF1 <br />  <br />  <br />  <br />  <br /> Figure 5. Analysis RP gene expression Mediator subunit deletion strains different repressing conditions. Clustergram <br /> comparison expression ratios shown 96 RP genes. Changes expression (increased red decreased green) shown relative the <br /> treated wild-type strain repressing conditions (rapamycin, tunicamycin, CPZ, post-diauxic shift, transient heat shock hydrogen peroxide, <br /> see Methods). The effect rapamycin compared Mediator subunit deletion strains. Each deleted subunit represents different structural <br /> module complex (med20 head, med31 middle, med16 tail cycC Cdk module). Except transient heat shock, <br /> all strains grown 30uC. <br /> doi:10.1371/journal.pgen.1000112.g005 <br />  <br />  <br /> importance Cdk module inactivation induction                    pathway signal transduction specific Mediator <br /> oxidative stress response genes [31].                                         subunits [25]. This led examine role subunits the <br />   Expression profiling Mediator subunit deletion strains             middle, tail Cdk modules Mediator repression RP <br /> normal growth conditions revealed epistatic relationships             gene transcription rapamycin. In contrast deletion of <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               6                             July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                         Genetic Interactions MAF1 <br />  <br />  <br /> MED20 head module, deletions MED31 CYCC                 genes range environmental conditions (Figures 4, 5 <br /> middle Cdk modules, respectively, detectable effect            Figure S4). For induction Gcn4-regulated genes by <br /> the repression RP genes 30uC relative wild-type strain          rapamycin, effect deleting MED20 consistent other <br /> (Figure 5, Table S5). Repression RP gene transcription           reports showing reduced recruitment Mediator promoter- <br /> examined northern analysis strain deleted MED13                  bound Gcn4 diminished transcriptional activation Gcn4- <br /> (SRB9). This subunit repressive Cdk module direct target         controlled genes Med20 subunits tail module are <br /> of protein kinase A (PKA) TOR kinase signaling thought             deleted [24,27]. For RP genes, association Mediator by <br /> control ribosome biogenesis antagonizing Ras/PKA              ChIP poor, evidence supporting direct role Mediator in <br /> pathway [32,33]. However, wildtype MED13 deletion                    repression based specificity response fact that <br /> strains showed differences response rapamycin (data           changes gene expression unstressed med20D cells minimal <br /> not shown). These results consistent genetic                    unlikely impact RP gene transcription ([25] see <br /> interaction data synthetic phenotypes MAF1               below). RP Ribi genes nearly identical transcription <br /> Mediator subunits middle Cdk modules                   responses environmental genetic perturbation [5,28] even <br /> apparent 30uC revealed 37uC (Figure 3).                  promoters genes generally contain different cis- <br /> Deletion MED16 (SIN4) tail module showed modest                  acting elements (Rap1 and/or Abf1 sites RP genes, PAC and/or <br /> reduction extent repression RP genes 30uC (1.560.2           RRPE elements Ribi genes). Despite differences, sets <br /> fold relative wild-type 121 RP genes yielding signals          genes regulated Sfp1 response nutrients stress <br /> the repressed gene set, Figure 5, Table S5). This effect consistent       conditions including rapamycin [5]. The fact Ribi genes are <br /> with difference strength synthetic phenotypes           repressed normally rapamycin med20D strains the <br /> the med16D maf1D med20D maf1D strains 30uC.                       repression RP genes attenuated indicates TOR <br /> Considering double mutant strains syntheti-              signaling pathway mediating response impaired and <br /> cally lethal elevated temperatures (Figure 3), findings               suggests differences repression likely independent of <br /> indicate Med16 plays minor role relative Med20                  Sfp1. Molecular genetic, biochemical structural studies indicate <br /> rapamycin repression RP genes normal growth                         deletion MED20 does significantly perturb overall <br /> conditions.                                                                  structure Mediator: The absence Med20 does affect the <br />                                                                              assembly head module subunits stable complex [40] <br /> Discussion                                                                   association head module modules of <br />                                                                              Mediator [23,24,41]. These data crystal structure <br />    The large (.1 MDa) Mediator complex organized                Med8-C-Med18-Med20 submodule EM images suggest <br /> structurally distinct modules, head, middle, tail Cdk                Med20 occupies peripheral position head module and <br /> modules, functions transduce regulatory information              complete complex [40,41]. In support limited structural <br /> DNA&#226;&#8364;&#8220;bound activators repressors general RNA pol II                effects deleting MED20, expression profile unstressed <br /> transcription machinery [23,34,35]. In addition role               med20D cells shows small number genes affected <br /> regulating transcription, studies temperature-sensitive head            (Figure 4, Table S3, [25]). Importantly, annotated functions of <br /> module subunits (e.g. Med17/Srb4) suggested Mediator               small group genes reveal changes transcription or <br /> is essential transcription vivo [36]. This supported        processes indirectly account attenuated <br /> the ability Mediator stimulate basal transcription vitro        repression RP genes. Given data indicating Mediator is <br /> by temperature-sensitivity stimulation extracts         essential RNA pol II transcription [36,37], findings are <br /> srb4-138 mutant strain [37]. Recently, ubiquitous function            consistent direct effect Mediator RP gene transcription <br /> Mediator transcription questioned based chromatin             specific repressing conditions. However, noted above, <br /> immunoprecipitation (ChIP) experiments showing                      Mediator subunits efficiently cross-linked RP genes in <br /> association Mediator RNA pol II actively                    ChIP assays [29,38,39]. We infer nature the <br /> transcribed genes correlated [29]. Indeed, observation            interactions Mediator RP genes fundamentally <br /> that Mediator associates poorly enhancer regions            different genes exhibit robust Mediator ChIP <br /> RP glycolytic genes, account 50%              signals. One possibility function Mediator RP genes <br /> of RNA pol II transcription actively growing cells [1],               require transient association. Alternatively, physical <br /> suggested Mediator required transcrip-             nature interaction Mediator nucleoprotein <br /> tion [29]. Other groups reported Mediator associations             complexes assembled RP genes compatible its <br /> the coding regions highly expressed genes [38,39]. However,               efficient crosslinking. Focusing prominent effect Med20 <br /> Mediator binding ratios RP coding regions low               (Figure 4), explanation protein functions <br /> (e.g. average binding ratio 1.3 determined 28                 independently Mediator complex repression RP <br /> experiments versus 4.3 13 experiments RNA pol II, [38]).            genes. While exclude possibility, does account <br /> Our examination molecular basis synthetic fitness defects         attenuated repression observed tail module subunit <br /> between Maf1 different Mediator subunits revealed                  Med16 deleted (Figure 5 Table S5). Moreover, synthetic <br /> prominent role non-essential head module subunit, Med20,               interactions MAF1 Mediator subunits representing <br /> in repression RP gene transcription different           structural module complex imply function of <br /> conditions. Together similar observations tail module             Mediator, just Med20, underlies functional relationship with <br /> subunit, Med16, results bear directly issue Mediator           Maf1. As discussed below, growing body evidence supports the <br /> involvement RP gene transcription.                                        view relationship involves coordinate regulation of <br />    Studies published date attributed head module              ribosome tRNA synthesis. Given role Maf1 repressing <br /> Mediator largely positive role transcription [25]; negative        RNA pol III transcription, analogous role Mediator RP <br /> regulation head module subunits specific nutritional             gene transcription consistent typical interpretation SSL <br /> environmental conditions reported. We                 interactions, namely, genes function parallel pathways. <br /> Med20 functions positively negatively different subsets          Therefore, suggest Mediator Maf1 function the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              7                             July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                       Genetic Interactions MAF1 <br />  <br />  <br /> downstream end distinct signaling pathways negatively                 previous proposal concerning paradoxical anti-suppressor <br /> regulate RP mRNA tRNA synthesis, respectively.                          phenotype maf1D strains. Loss MAF1 function causes a <br />    Unlike deletion MAF1, quantitatively blocks repression          significant increase cellular level mature tRNA (from <br /> of RNA pol III transcription [4], deletion MED20                    ,10% ,25% total RNA) activity SUP11-o <br /> attenuates repression RP genes. Thus, signaling pathways             nonsense suppressor decreases [47]. This anti-suppressor pheno- <br /> that repress RP genes multiple targets RNA             type suggested result incomplete isopentenylation of <br /> pol II transcription machinery. Besides Mediator,                adenine <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> (A37, adjacent anticodon) is <br /> transcriptional targets involved repression RP genes?         important tRNA decoding efficiency. A recent study of <br /> Previous work identified Crf1 TOR kinase-regulated                 synthetic interactions certain non-essential tRNA mod- <br /> corepressor RP genes [7]. We tested deletion CRF1,            ifying enzymes highlighted function tRNA stability and <br /> either itself combination deletion MED20           cell <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>survival</span> [48]. Our findings demonstrate tRNA modifica- <br /> affect rapamycin-mediated repression RP genes SGA                 tions critical maf1D strain additional loss of <br /> strain background (S288C). Although generated crf1D                  tRNA modifying enzymes results synthetic <br /> strains novo, northern analysis multiple RNA samples did          growth defect (Figure 1). We anticipate analysis the <br /> reveal quantitative differences compared controls (data          genetic interactions MAF1 group enzymes will <br /> not shown). This result consistent findings W303             provide new insights biological function. <br /> strain background [42], indicating corepressor function of <br /> Crf1 RP genes strain-specific. Other observations suggest        Materials Methods <br /> the TFIID complex participate repression RP genes. <br /> TFIID occupancy RP genes high [43] transcription           SGA Methods <br /> RP genes strongly TFIID-dependent [44]. This dependence                     Triplicate SGA screens maf1D query strain (Y6338 Mata <br /> reflects core promoter recognition function                    can1D::MFA1pr-HIS3 lyp1D ura3D0 leu2D0 his3D1 met15D0 <br /> coactivator function TFIID promoters [44,45]. Our               maf1D::natR) performed non-essential gene- <br /> SGA screens identified synthetic fitness defects MAF1           deletion array (,4700 strains) array condition- <br /> five TAFs, (TAF8 TAF11) unique                  ally-expressed essential genes (,800 Tet-promoter strains). Each <br /> TFIID complex [43]. The basis genetic interactions            screen conducted duplicate copies array 768 <br /> be similar MED20. In words, synthetic growth defects               colony plate format described previously [15,17,18]. In Tet- <br /> may result, part, inability repress RNA pol III              promoter array screens, haploid double mutant strains were <br /> transcription coupled attenuated repression RP gene                 scored growth medium doxycycline <br /> transcription. This interpretation consistent identifica-       (10 mg/ml). Visual inspection computer-based analysis of <br /> tion genetic interactions MAF1 genes                  digital images used identify double mutant strains <br /> ribosome biogenesis category (TIF6 RPL genes),            exhibiting fitness (growth) defects [18] relative control set of <br /> functional insufficiencies known block repression             double mutants obtained using strain Y5518 (Mata mfa1D::M- <br /> rDNA RP gene transcription following interruption                FA1pr-HIS3 lyp1D ura3D0 leu2D0 his3D1 met15D0 can1D::natR). <br /> secretory pathway [19,20]. Another link transcriptional control          Candidate synthetic genetic interactions validated random <br /> of ribosome synthesis provided genetic interaction                spore analysis [15,17] 30uC elevated temperatures <br /> between UTP22 MAF1. UTP22 encodes essential                (35&#226;&#8364;&#8220;37uC) enhanced severity synthetic fitness <br /> gene products (the Ifh1 Rrp7) associate          defect cases. The enrichment GO Bioprocess terms in <br /> casein kinase II (CK2) form CURI complex [22]. This                  MAF1 SSL gene set calculated hypergeometric <br /> complex thought coordinate parallel pathways necessary            distribution aid MIPS Functional Catalogue Database. <br /> for ribosome synthesis, namely, transcription processing of <br /> pre-rRNA transcription ribosomal protein genes. The              Construction MAF1 Genetic Interaction Network <br /> presence CK2 complex strengthens proposed               Random spore-validated MAF1 SSL genes queried against <br /> functional association MAF1 ribosome synthesis                  BioGRID Database version 2.0.23 (released Jan 3, 2007) to <br /> based studies CK2 transcriptional response RNA              compile list 4012 interactions involving 1225 genes. <br /> pols I III DNA damage [46]. Finally, nearly            Interactions MAF1 SSL genes (Fyv5, <br /> one-fifth MAF1 SSL genes identified non-essential             YGL007W). The set interactions superimposed onto <br /> gene-deletion array associated defects repression        MAF1 SSL gene set using Osprey software filtered to <br /> RP gene transcription (Figure S1). These observations support           reveal interactions nodes MAF1 genetic interaction <br /> hypothesis genetic interaction MAF1 MED20              network. <br /> is related combination defects repression RNA <br /> pol III RP gene transcription. This interpretation does             Microarray Experiments <br /> exclude possibility changes maf1 med20 double            Strain BY4741 (Mata ura3D0 leu2D0 his3D1 met15D0) and <br /> mutant strain contribute synthetic phenotype. Given          isogenic deletion strains (xxxD:kanR) grown YPD 30uC to <br /> genetic interactions MAF1 subunits Mediator              optical density (A600) ,0.2 addition drugs drug <br /> TFIID complex, identification negative regulatory                  vehicle, unless indicated. Treatments rapamycin <br /> function Med20 RP genes suggests possible relationship             (0.2 mg/ml 1 mg/ml stock solution DMSO, AG <br /> with TFIID process head module Mediator                Scientific) CPZ (250 mM 500 mM stock solution in <br /> contains multipartite TBP-binding site includes direct             water, Sigma) 1 hour [4]. Treatments hydrogen <br /> interaction TBP Med20 [41].                                     peroxide (0.32 mM, Sigma) tunicamycin (2.5 mg/ml a <br />    In addition genes involved ribosome biogenesis                 5 mg/ml stock 75% methanol, Sigma) 30 min. and <br /> transcription, SGA analysis MAF1 revealed significant              90 min. respectively [4,28]. A transient mild heat shock treatment <br /> functional relationship enzymes involved tRNA modifica-             cells growing 29uC achieved centrifugation and <br /> tion (Figure 2, Table S1). This group interactions supports            resuspension pre-warmed, 39uC medium 20 min. [28]. To <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              8                            July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                                                 Genetic Interactions MAF1 <br />  <br />  <br /> compare cells following diauxic shift, early log culture                              Found at: doi:10.1371/journal.pgen.1000112.s006 (0.43 MB PDF) <br /> (OD600 = 0.01) grown 48 hours 30uC then <br />                                                                                              Table S3 Yeast GO bioprocess terms represented merged <br /> harvested. Detailed procedures culturing cells, RNA prepara- <br />                                                                                              med20D versus wild-type datasets. <br /> tion, hybridization, image acquisition data processing for <br />                                                                                              Found at: doi:10.1371/journal.pgen.1000112.s007 (0.04 MB PDF) <br /> microarrays described [49]. Replicates sample <br /> were performed using fluor-reversal strategy [50]. Microarray                              Table S4 Expression ratios (log base 10) comparing med20D <br /> data deposited Gene Expression Omnibus                                      versus wild-type strains different repressing conditions. <br /> Database accession number GSE11397.                                                    Found at: doi:10.1371/journal.pgen.1000112.s008 (0.41 MB PDF) <br />                                                                                              Table S5 Expression ratios ribosomal protein genes compar- <br /> Supporting Information                                                                       ing different Mediator subunit deletions versus wild-type after <br /> Figure S1 Northern analysis RP genes wild-type MAF1                                rapamycin treatment. <br /> SSL strains rapamycin treatment.                                            Found at: doi:10.1371/journal.pgen.1000112.s009 (0.04 MB PDF) <br /> Found at: doi:10.1371/journal.pgen.1000112.s001 (0.15 MB PDF) <br />                                                                                              Text S1 Supporting text. <br /> Figure S2 Transcription tRNALeu gene robustly                                        Found at: doi:10.1371/journal.pgen.1000112.s010 (0.10 MB PDF) <br /> repressed rapamycin med20 strain. <br /> Found at: doi:10.1371/journal.pgen.1000112.s002 (0.07 MB PDF)                                Acknowledgments <br /> Figure S3    Genes induced repressed rapamycin treat-                                 IMW thanks CB, Brenda Andrews members Boone <br /> ment strain S288c.                                                                        laboratory hospitality sabbatical leave. We thank <br /> Found at: doi:10.1371/journal.pgen.1000112.s003 (0.10 MB PDF)                                Huiming Ding Bilal Sheikh help computed-based image <br />                                                                                              analysis Jon Warner Greg Prelich comments the <br /> Figure S4 Clustergram comparison med20D versus wild-type <br />                                                                                              manuscript. <br /> expression ratios different environmental conditions. <br /> Found at: doi:10.1371/journal.pgen.1000112.s004 (0.05 MB PDF) <br />                                                                                              Author Contributions <br /> Table S1 Phenotypes functions MAF1 SSL genes. <br />                                                                                              Conceived designed experiments: IW CB RM. Performed the <br /> Found at: doi:10.1371/journal.pgen.1000112.s005 (0.03 MB PDF)                                experiments: IW GC AT RB RM. Analyzed data: IW GC AT RB RM. <br /> Table S2 Expression ratios (log base 10) comparing med20D                                Contributed reagents/materials/analysis tools: TH. Wrote paper: IW. <br /> wild-type strains rapamycin treatment. <br />  <br /> References <br />  1. Warner J (1999) The economics ribosome biosynthesis yeast. Trends                  17. Davierwala AP, Haynes J, Li Z, Brost RL, Robinson MD, et al. (2005) The <br />     Biochem Sci 24: 437&#226;&#8364;&#8220;440.                                                                     synthetic genetic interaction spectrum essential genes. Nat Genet 37: <br />  2. Willis IM, Desai NA, Upadhya R (2004) Signaling repression transcription               1147&#226;&#8364;&#8220;1152. <br />     RNA polymerase III yeast. Prog Nucl Acid Res Mol Bio 77: 323&#226;&#8364;&#8220;353.                 18. Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al. (2004) Global mapping of <br />  3. Phizicky EM (2005) Have tRNA, travel. Proc Natl Acad Sci U S A 102:                     yeast genetic interaction network. Science 303: 808&#226;&#8364;&#8220;813. <br />     11127&#226;&#8364;&#8220;11128.                                                                             19. Miyoshi K, Tsujii R, Yoshida H, Maki Y, Wada A, et al. (2002) Normal <br />  4. Upadhya R, Lee J, Willis IM (2002) Maf1 Is Essential Mediator Diverse                  assembly 60 S ribosomal subunits required signaling response a <br />     Signals Repress RNA Polymerase III Transcription. Mol Cell 10:                          secretory defect Saccharomyces cerevisiae. J Biol Chem 277: 18334&#226;&#8364;&#8220;18339. <br />     1489&#226;&#8364;&#8220;1494.                                                                               20. Zhao Y, Sohn JH, Warner JR (2003) Autoregulation biosynthesis of <br />  5. Jorgensen P, Rupes I, Sharom JR, Schneper L, Broach JR, et al. (2004) A                      ribosomes. Mol Cell Biol 23: 699&#226;&#8364;&#8220;707. <br />     dynamic transcriptional network communicates growth potential ribosome                21. Dragon F, Gallagher JE, Compagnone-Post PA, Mitchell BM, Porwancher KA, <br />     synthesis critical cell size. Genes Dev 18: 2491&#226;&#8364;&#8220;2505.                                   et al. (2002) A large nucleolar U3 ribonucleoprotein required 18S ribosomal <br />  6. Marion RM, Regev A, Segal E, Barash Y, Koller D, et al. (2004) Sfp1 stress-             RNA biogenesis. Nature 417: 967&#226;&#8364;&#8220;970. <br />     <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensitive regulator ribosomal protein gene expression. Proc Natl         22. Rudra D, Mallick J, Zhao Y, Warner JR (2007) Potential interface between <br />     Acad Sci U S A 101: 14315&#226;&#8364;&#8220;14322.                                                             ribosomal protein production pre-rRNA processing. Mol Cell Biol 27: <br />  7. Martin DE, Soulard A, Hall MN (2004) TOR regulates ribosomal protein gene                    4815&#226;&#8364;&#8220;4824. <br />     expression PKA Forkhead transcription factor FHL1. Cell 119:                 23. Guglielmi B, van Berkum NL, Klapholz B, Bijma T, Boube M, et al. (2004) A <br />     969&#226;&#8364;&#8220;979.                                                                                     high resolution protein interaction map yeast Mediator complex. Nucleic <br />  8. Claypool JA, French SL, Johzuka K, Eliason K, Vu L, et al. (2004) Tor pathway                Acids Res 32: 5379&#226;&#8364;&#8220;5391. <br />     regulates Rrn3p-dependent recruitment yeast RNA polymerase I                   24. Zhang F, Sumibcay L, Hinnebusch AG, Swanson MJ (2004) A triad subunits <br />     promoter does participate alteration number active genes.               Gal11/tail domain Srb mediator vivo target of <br />     Mol Biol Cell 15: 946&#226;&#8364;&#8220;956.                                                                   transcriptional activator Gcn4p. Mol Cell Biol 24: 6871&#226;&#8364;&#8220;6886. <br />  9. Rudra D, Zhao Y, Warner JR (2005) Central role Ifh1p-Fhl1p interaction             25. van Peppel J, Kettelarij N, van Bakel H, Kockelkorn TT, van Leenen D, et al. <br />     synthesis yeast ribosomal proteins. EMBO J 24: 533&#226;&#8364;&#8220;542.                               (2005) Mediator expression profiling epistasis reveals signal transduction <br /> 10. Desai N, Lee J, Upadhya R, Chu Y, Moir RD, et al. (2005) Two Steps Maf1-                  pathway antagonistic submodules highly specific downstream targets. <br />     dependent Repression Transcription RNA Polymerase III. J Biol Chem                     Mol Cell 19: 511&#226;&#8364;&#8220;522. <br />     280: 6455&#226;&#8364;&#8220;6462.                                                                          26. Chen JC, Powers T (2006) Coordinate regulation multiple distinct <br /> 11. Oficjalska-Pham D, Harismendy O, Smagowicz WJ, Gonzalez dP, Boguta M, et                     biosynthetic pathways TOR PKA kinases S. cerevisiae. Curr Genet <br />     al. (2006) General Repression RNA Polymerase III Transcription Is                         49: 281&#226;&#8364;&#8220;293. <br />     Triggered Protein Phosphatase Type 2A-Mediated Dephosphorylation                   27. Qiu H, Hu C, Zhang F, Hwang GJ, Swanson MJ, et al. (2005) Interdependent <br />     Maf1. Mol Cell 22: 623&#226;&#8364;&#8220;632.                                                                  recruitment SAGA Srb mediator transcriptional activator Gcn4p. Mol <br /> 12. Reina JH, Azzouz TN, Hernandez N (2006) Maf1, New Player                            Cell Biol 25: 3461&#226;&#8364;&#8220;3474. <br />     Regulation Human RNA Polymerase III Transcription. PLoS ONE 1: e134.                  28. Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al. (2000) <br /> 13. Willis IM, Moir RD (2007) Integration nutritional stress signaling                    Genomic expression programs response yeast cells environmental <br />     pathways Maf1. Trends Biochem Sci 32: 51&#226;&#8364;&#8220;53.                                              changes. Mol Biol Cell 11: 4241&#226;&#8364;&#8220;4257. <br /> 14. Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz A, et al. (2006)               29. Fan X, Chou DM, Struhl K (2006) Activator-specific recruitment Mediator in <br />     BioGRID: general repository interaction datasets. Nucleic Acids Res 34:                vivo. Nat Struct Mol Biol 13: 117&#226;&#8364;&#8220;120. <br />     D535&#226;&#8364;&#8220;D539.                                                                               30. Singh H, Erkine AM, Kremer SB, Duttweiler HM, Davis DAI, et al. (2006) A <br /> 15. Tong AH, Boone C (2006) Synthetic genetic array analysis Saccharomyces                    functional module yeast mediator governs dynamic range heat- <br />     cerevisiae. Methods Mol Biol 313: 171&#226;&#8364;&#8220;192.                                                   shock gene expression. Genetics 172: 2169&#226;&#8364;&#8220;2184. <br /> 16. Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, et al. (2004)                    31. Krasley E, Cooper KF, Mallory MJ, Dunbrack R, Strich R (2006) Regulation of <br />     Exploration essential gene functions titratable promoter alleles. Cell 118:           oxidative stress response Slt2p-dependent destruction cyclin C in <br />     31&#226;&#8364;&#8220;44.                                                                                       Saccharomyces cerevisiae. Genetics 172: 1477&#226;&#8364;&#8220;1486. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                             9                                  July 2008 | Volume 4 | Issue 7 | e1000112 <br />                                                                                                                                                Genetic Interactions MAF1 <br />  <br />  <br /> 32. Chang YW, Howard SC, Herman PK (2004) The Ras/PKA signaling pathway                     43. Kuras L, Kosa P, Mencia M, Struhl K (2000) TAF-Containing TAF- <br />     directly targets Srb9 protein, component general RNA polymerase II             independent forms transcriptionally active TBP vivo. Science 288: <br />     transcription apparatus. Mol Cell 15: 107&#226;&#8364;&#8220;116.                                              1244&#226;&#8364;&#8220;1248. <br /> 33. Schmelzle T, Beck T, Martin DE, Hall MN (2004) Activation RAS/cyclic             44. Mencia M, Moqtaderi Z, Geisberg JV, Kuras L, Struhl K (2002) Activator- <br />     AMP pathway suppresses TOR deficiency yeast. Mol Cell B <br /> </body></html>