<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>6570&#194;&#177;6577 Nucleic Acids Research, 2003, Vol. 31, No. 22 <br /> DOI: 10.1093/nar/gkg859 <br />  <br />  <br /> Gene essentiality determines chromosome <br /> organisation bacteria <br /> Eduardo P. C. Rocha1,2,* Antoine Danchin1 <br /> 1 <br />  Unite&#195;&#8218; Ge&#195;&#8218;ne&#195;&#8218;tique des Ge&#195;&#8218;nomes Bacte&#195;&#8218;riens, Institut Pasteur, 28, rue du Dr Roux, 75724 Paris Cedex 15, France <br /> and 2Atelier Bioinformatique, Universite&#195;&#8218; Pierre et Marie Curie, 12, Rue Cuvier, 75005 Paris, France <br />  <br /> Received August 1, 2003; Revised Accepted September 25, 2003 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        techniques allowed de&#194;&#174;nition set essential <br />                                                                                 genes bacteria. This achieved through <br /> In Escherichia coli Bacillus subtilis, essen-                               different techniques. Saturation transposon mutagenesis, fol- <br /> tiality, expressivity, drives distribution                           lowed sequencing, allowed de&#194;&#174;nition essential genes <br /> genes replicating strands.                                      Mycoplasma genitalium (1) Haemophilus in&#194;&#175;uenzae <br /> Although essential genes tend coded                                (2). Antisense RNA used speci&#194;&#174;cally silence genes <br /> leading replicating strand, underlying selective                            Staphylococccus aureus (3). These techniques fast and <br /> constraints evolutionary extent                                easy implement, especially bacteria hard <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> &#194;&#174;ndings subject comparative                              cultivate. However, require important resources, tend <br /> studies. Here, extend previous analysis                               over-represent number essential genes of <br /> the genomes low G + C &#194;&#174;rmicutes g-proteo-                                polarity transcription operons insertion biases. For <br /> bacteria, second step sequenced                                 example, studies identi&#194;&#174;ed 658 essential <br /> bacterial genomes. The inference essentiality                             genes S.aureus 478 H.in&#194;&#175;uenzae, precise <br />                                                                                 systematic gene inactivation larger genome free- <br /> homology allows essential genes <br />                                                                                 living Bacillus subtilis resulted identi&#194;&#174;cation only <br /> are frequent leading strand                               277 essential genes (4). In Escherichia coli, equivalent <br /> other genes, compared non-                                       published work unavailable, PEC database the <br /> essential highly expressed genes. Smaller biases                                Japan National Institute Genetics provides extensive <br /> were genomes obligatory intra-                                  information essentiality two-thirds genes. <br /> cellular bacteria, assignment essen-                           This database classi&#194;&#174;es 206 genes essential E.coli, of <br /> tiality homology fast growing free-living                               82% classi&#194;&#174;ed essential B.subtilis. The <br /> bacteria problematic. Cross-comparisons                                 study essentiality provides important information cell <br /> used assess potential errors assignment                            functioning; used develop new antibiotic targets, <br /> essentiality homology revealed that,                                 substantial evolutionary interest. For example, is <br /> cases, variations assignment criteria                               currently subject intense debate essentiality plays <br /> little in&#194;&#175;uence overall results. Essential                               important role constraining evolutionary rate of <br />                                                                                 proteins (5&#194;&#177;8). Also, patterns loss `essential&apos; genes in <br /> genes tend conserved leading <br />                                                                                 process genome degradation important the <br /> strand average genes, consistent with <br />                                                                                 understanding co-evolution bacteria their <br /> selection positioning impose                                 hosts (9). <br /> strong constraint chromosomal rearrangements.                                   The existence larger number genes leading <br /> These results indicate essentiality plays                                strand, relative lagging strand, known two <br /> fundamental role distribution genes                                decades. It &#194;&#174;rst observed rDNA ribosomal <br /> most bacterial genomes.                                                         proteins systematically coded leading strand the <br />                                                                                 E.coli chromosome (10). This speculated result <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                 selection prevent collisions DNA polymerase <br /> INTRODUCTION                                                                    (DNAP) RNA polymerase (RNAP) (11). DNA replication <br /> The concept `essential&apos; gene ambiguous,                      transcription occur simultaneously DNA <br /> genes needed basic supplies indispensable cell                       molecule. DNAP E.coli proceeds 10&#194;&#177;20 times faster than <br /> metabolites essential special circumstances.                          RNAP (12), head-on co-oriented collisions will <br /> However, uses medium enriched presumably                      occur replicating bacteria. However, outcome of <br /> necessary components, essentiality pertains core                  collision different depending poly- <br /> cell&apos;s functions, particular processes involved                    merases co-oriented not, resulting transcription <br /> information transfer compartmentalisation. The avail-                       abortion severe replication slow-down genes <br /> ability complete genome sequences high-throughput                        lagging strand (13,14). Thus, preferential <br />  <br /> *To correspondence addressed Atelier Bioinformatique, Universite&#195;&#8218; Pierre et Marie Curie, 12 rue Cuvier, 75005 Paris, France. <br /> Tel: +33 1 44 27 65 36; Fax: +33 1 44 27 63 12; Email: erocha@abi.snv.jussieu.fr <br />  <br /> Nucleic Acids Research, Vol. 31 No. 22 &#195;&#163; Oxford University Press 2003; rights reserved <br />                                                                             Nucleic Acids Research, 2003, Vol. 31, No. 22       6571 <br />  <br />  <br /> positioning translation-related highly expressed genes           frequent leading strand genes. Thirdly, we <br /> (rDNA ribosomal proteins) leading strand             tested non-highly expressed essential genes biased <br /> been widely deemed result selection lower               highly expressed non-essential genes. Fourthly, tested <br /> transcription abortion higher replication rates. The            conservation essential genes leading strand is <br /> observations genes coding rDNA ribosomal               important genes. To check the <br /> proteins leading strand bacterial genomes        dif&#194;&#174;culties associated assignment essentiality by <br /> seemed con&#194;&#174;rm hypothesis (15,16). As selection        homology biasing results, set tests and <br /> only effective highly expressed genes acts        cross-comparisons available experimental data on <br /> positioning genes leading strand, create     B.subtilis, E.coli, M.genitalium H.in&#194;&#175;uenzae. <br /> gene strand bias proportional expression level in <br /> replicating bacteria. The bias proportional       MATERIALS AND METHODS <br /> the frequency replication. Fast-growing bacteria should <br /> therefore higher strand bias.                                Data <br />    Several lines evidence recently challenged           Sequence data corresponding annotations taken <br /> emphasis model replication slow-down basis        GenBank Genomes (ftp://ftp.ncbi.nih.gov/genomes/ <br /> of gene strand bias. First, gene strand bias strongly depends    Bacteria/). The following genomes low G + C &#194;&#174;rmicutes <br /> the composition DNAP, bacteria containing               analysed: B.subtilis 168, Bacillus halodurans, <br /> different a-subunits exhibiting higher strand biases (17).     Clostridium acetobutylicum ATCC824, Clostridium perfrin- <br /> In bacteria, a-subunit dedicated       gens 13, Lactococcus lactis IL1403, Listeria inocua <br /> to replication DNA strand (18), average 78%        Clip11262, Listeria monocytogenes EGD, Oceanobacillus <br /> of genes coded leading strand,     iheyensis, Staphylococcus aureus N315, Streptococcus aga- <br /> not highly expressed. Secondly, largest gene            lactiae serotype V, Streptococcus pneumoniae TIGR4, <br /> strand biases slow-growing bacteria. Among             Streptococcus pyogenes M1 Thermoanaerobacter teng- <br /> bacteria containing different DNAP a-subunits, slow-        congensis. The following genomes g-proteobacteria were <br /> growing M.genitalium highest biases (80%          analysed: E.coli MG1655, Buchnera Ap, H.in&#194;&#175;uenzae Rd, <br /> genes leading strand), fast-growing              Pasteurella multocida PM70, Pseudomonas aeruginosa <br /> B.subtilis lowest (74%). For genomes           PA01, Pseudomonas putida KT2440, Salmonella enterica <br /> replicated gene coding DNAP a-subunit,        Typhimurium LT2, Shewanella oneidensis MR-1, <br /> the slow-growing Borrelia burgdorferi shows highest bias        Xanthomonas axonopodis, Xanthomonas campestris, Xylella <br /> (65%). Comparing equally sized chromosomes, &#194;&#174;nds           fastidiosa 9a5c Yersinia pestis CO92. Because different <br /> the slow-growing Mycobacterium tuberculosis (59%)           strains species co-linear, one <br /> biased fast-growing E.coli (55%). Thirdly,         strain species analysed. Essentiality data were <br /> recently shown gene strand bias B.subtilis E.coli       taken literature following genomes: B.subtilis <br /> stems `essentiality&apos; genes. Controlling         (4), M.genitalium (1), H.in&#194;&#175;uenzae (2) E.coli (http:// <br /> essentiality revealed signi&#194;&#174;cant role expression levels   www.shigen.nig.ac.jp/ecoli/pec/). Identi&#194;&#174;cation origins of <br /> gene strand bias (19). This suggests anti-oriented             replication leading strands use of <br /> collisions selected problems associated          linear discriminant analysis GC skews, detailed <br /> with existence aborted transcripts, replication          (23). <br /> pausing. It suggests essentiality major deter- <br /> minant chromosome structure. Rearrangements result-          Analysis orthology <br /> ing important inversions leading strands            Orthologues identi&#194;&#174;ed reciprocal best hits. <br /> extremely deleterious (20,21). The strand switch essential       First, BlastP search identi&#194;&#174;ed 10 best hits protein <br /> genes partly responsible observations.             genome. These proteins aligned using a <br />    To substantiate previous observation concerning essen-       global alignment end gaps facing largest <br /> tial genes, limited fast-growing bacteria     sequence ignored. Two proteins regarded as <br /> similar genome sizes, tested essentiality plays similar     orthologues reciprocal best hits least <br /> role bacterial genomes. Putative essentiality          40% similarity amino acid sequence &lt;20% of <br /> identi&#194;&#174;ed homology genomes B.subtilis (for           difference protein <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> (24). Essentiality derived by <br /> low G + C &#194;&#174;rmicutes) E.coli (for g-proteobacteria). As         homology B.subtilis genomes low G + C <br /> working hypothesis, simplifying assump-             &#194;&#174;rmicutes E.coli genomes g-proteobacteria. <br /> tions. First, de&#194;&#174;ned set putative orthologues        In second step, analysed set complete bacterial <br /> genomes, using reciprocal best hits. Secondly, assumed           genomes origin terminus replication can <br /> that orthologues essential genes likely       reliably identi&#194;&#174;ed. In case, essentiality derived by <br /> essential genomes. Thirdly, de&#194;&#174;ned putative         homology B.subtilis subset. The classi&#194;&#174;cations the <br /> highly expressed genes genome codon adaptation         E.coli B.subtilis genes used reference be <br /> index (CAI) values computed ribosomal proteins (22). At        obtained http://wwwabi.snv.jussieu.fr/~erocha/essential/ <br /> this point, tested different hypotheses concerning      Supplementary Material NAR Online. <br /> higher frequency genes leading strand. First, we <br /> tested essential genes frequent leading     Analysis codon usage <br /> strands non-essential genes. Secondly, tested      CAI values computed using EMBOSS package <br /> the non-essential genes, highly expressed genes            (http://www.uk.embnet.org/Software/EMBOSS). The reference <br />  6572   Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br /> values codon usage highly expressed genes (understand-      two-tailed c2 tests 2 3 2 contingency tables the <br /> ing high expression cells growing exponen-       comparisons class orthologous <br /> tially) computed using ribosomal proteins (22). It       essential genes versus remaining orthologues. Because for <br /> recently shown yeast correlation CAI     type tests multiple tests being <br /> (de&#194;&#174;ned using ribosomal proteins) transcriptome data      (one genome), used Bonferroni correction <br /> good, good different transcriptome data sets        taxonomic group. Thirdly, test all <br /> among (under experimental conditions)         sequenced genomes group genes biased than <br /> (25). Another recent work indicated CAI values      another, performed single paired t-test. In case, <br /> robust small changes learning set (26). We         test carried hypothesis no <br /> previously checked transcriptome proteome data,          Bonferroni correction used. <br /> when available, con&#194;&#174;rmed analysis gene strand bias <br /> made CAI values (19). Here, gene regarded as <br /> potentially highly expressed CAI 10% <br /> highest values genome. The use 10% threshold       RESULTS <br /> somewhat arbitrary, re&#194;&#175;ects results classi- <br /> &#194;&#174;cations authors, using CAI factorial          Distribution essential genes replicating <br /> analysis, indicating 5&#194;&#177;10% genes particularly         strands <br /> highly expressed (27&#194;&#177;29). We shown previously           Low G + C &#194;&#174;rmicutes. We explored link between <br /> varying threshold range 5&#194;&#177;15% did change      essentiality gene strand bias genomes closely <br /> results signi&#194;&#174;cantly (19).                                        related low G + C &#194;&#174;rmicutes (excluding distant <br />                                                                   Mycoplasmas). For this, identi&#194;&#174;ed orthologues between <br /> Tests CAI <br />                                                                   B.subtilis genomes, assigning essentiality <br /> Codon usage bias good estimator gene              according B.subtilis classi&#194;&#174;cation. Then performed <br /> expression bacteria (30). Hence, designed set     tests. First, tested essential genes more <br /> tests verify codon usage ribosomal proteins       frequently leading strand non-essential <br /> signi&#194;&#174;cantly biased, codon usage classi&#194;&#174;cation      genes. This case genomes (Table 1). <br /> the genome using set discriminant using       Therefore, essentiality plays important role gene strand <br /> random set genes. For this, 100 random                 bias low G + C &#194;&#174;rmicutes. Secondly, tested the <br /> experiments. In each, randomly took 50 genes (corres-          non-essential genes, highly expressed genes frequent <br /> ponding number ribosomal proteins used           leading strand non-highly expressed genes. This <br /> previous analysis) build reference table codon usage.   test identi&#194;&#174;ed case signi&#194;&#174;cantly higher frequency <br /> Using table, computed CAI values genes    highly expressed genes leading strand <br /> then frequency 50 initial genes 10%          (C.acetobutylicum) (Table 1). Therefore, expression levels <br /> highest CAI. A CAI indicating signi&#194;&#174;cant codon usage bias         contribute signi&#194;&#174;cantly gene strand bias, <br /> should high consistency (i.e. genes initially    essentiality taken account. Thirdly, tested <br /> hypothesised highly expressed classed          essential non-highly expressed genes biased <br /> highly expressed) genome average value smaller         non-essential highly expressed genes. With the <br /> expected chance (to discriminate minority highly        above-mentioned exception C.acetobutylicum, which <br /> expressed genes rest genome). Signi&#194;&#174;cant          difference statistically signi&#194;&#174;cant, test showed <br /> codon usage bias ribosomal proteins met conditions.      generally true. <br /> First, &gt;40% ribosomal proteins classed highly <br /> expressed 10% CAI values. Secondly, frequency      g-Proteobacteria. The analysis g- <br /> of ribosomal proteins classed 10% CAI values      proteobacteria, starting PEC database E.coli <br /> be signi&#194;&#174;cantly higher obtained random        essential genes. This database contains 1700 genes for <br /> experiments genes used build codon index           essentiality character unknown, were <br /> (signi&#194;&#174;cant difference taken &gt;3 SDs away       removed simplicity. The resulting sample contains 2410 <br /> mean random experiments). Thirdly, average CAI      E.coli genes, used identify orthologous genes in <br /> all genes, CAI computed using ribosomal proteins,     g-proteobacteria. Because incomplete classi- <br /> must signi&#194;&#174;cantly lower obtained random       &#194;&#174;cation lack extensive experimental gene <br /> experiments (&gt;3 SDs away).                                        disruption programme E.coli, analysis more <br />                                                                   error prone. Still, results qualitatively similar those <br /> Statistical tests <br />                                                                   observed low G + C &#194;&#174;rmicutes (Table 1). With the <br /> We performed types statistical tests adapted   exception Buchnera, genomes high relative <br /> different situations. To test groups different        frequencies leading strand essential genes. When analysing <br /> frequency leading strand genes genome, carried     non-essential genes gene strand bias function of <br /> two-tailed c2 tests 2 3 2 contingency tables. Strand           expression levels, genomes higher biases among <br /> conservation groups genes genomes               highly expressed genes, genomes lower biases <br /> computed frequency orthologues          highly expressed genes genomes non- <br /> same replicating strand genomes. We tested    signi&#194;&#174;cant differences. Thus, non-essential genes, <br /> value different expected given average           expression does play signi&#194;&#174;cant role gene <br /> strand conservation orthologues. This test consisted    strand bias. Finally, genomes showed a <br />                                                                                               Nucleic Acids Research, 2003, Vol. 31, No. 22                 6573 <br />  <br />  <br /> Table 1. Distribution genes leading strands genomes <br /> Genome                  Genes                                           % Genes lead                                 Statistical tests <br />                         EH           EnH        nEH        nEnH         EH      EnH          nEH        nEnH         Ess        Exp      EvH         SC       CB <br />  <br /> B.subtilis               78          199         119       3710          94       94         72         74           +          0        +                    + <br /> B.halodurans            240           15        1503        402          93       93         81         79           +          0        +           +        + <br /> C.acetobutylicum        218           10         918        251          93       91         91         80           +          +        0           +        + <br /> C.perfringens           208            9         770        196          95       93         88         85           +          0        +           +        + <br /> L.lactis                225           11         698        186          88       89         74         81           +          0        +           +        + <br /> L.inocua                242           16        1069        255          95       94         75         80           +          0        +           +        + <br /> L.monocytogenes         243           18        1035        243          96       93         73         79           +          0        +           +        + <br /> O.iheyensis             248           14        1366        395          96       93         79         76           +          0        +           +        + <br /> S.agalactiae            219           11         665        186          93       92         76         79           +          0        +           +        + <br /> S.aureus                237           13         956        261          95       89         75         75           +          0        +           +        + <br /> S.pneumoniae            214           10         654        153          93       90         72         80           +          0        +           +        + <br /> S.pyogenes              208            9         575        131          92       93         78         80           +          0        +           +        + <br /> T.tengcongensis         212           11         771        166         100       94         90         87           +          0        +           0        0 <br />  <br /> E.coli                  112           94         249       1955          80       71         57         56           +          0        +                    + <br /> Buchnera Ap             127           33         169        116          70       47         73         61           0          0        &#194;&#177;           0        0 <br /> H.in&#194;&#175;uenzae             149           44         389        289          79       67         52         51           +          0        +           +        + <br /> P.multocida             152           44         450        318          67       72         65         54           +          0        +           +        + <br /> P.aeruginosa            150           40         628        437          75       82         51         59           +          &#194;&#177;        +           +        + <br /> P.putida                145           39         615        422          75       78         54         61           +          &#194;&#177;        +           +        + <br /> S.enterica              154           49         967        718          80       70         57         57           +          &#194;&#177;        +           +        + <br /> S.oneidensis            148           40         569        393          81       72         62         57           +          +        +           +        + <br /> X.axonopodis            145           35         474        323          79       77         56         60           +          0        +           +        + <br /> X.campestris            141           34         482        321          79       79         53         59           +          0        +           +        + <br /> X.fastidiosa            139           35         342        216          99       63         98         51           +          +        &#194;&#177;           +        + <br /> Y.pestis                156           46         730        567          80       74         59         53           +          +        +           +        + <br />  <br /> The classi&#194;&#174;cation low G + C &#194;&#174;rmicutes homology B.subtilis g-proteobacteria E.coli. The genes classed according <br /> to expression level essential <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> highly expressed (H) versus non-highly expressed (nH) essential (E) versus non-essential (nE). The <br /> acronyms concatenated classes (e.g. EnH indicates essential non-highly expressed genes). The columns `Statistical tests&apos; (two-tail c2 <br /> tests contingency tables) indicate signi&#194;&#174;cance test P &lt; 0.05, applying Bonferroni correction. The + &#194;&#177; signs indicate rejection the <br /> hypothesis (right- left-side, respectively), 0 indicates rejection. The tests correspond to: E &#194;&#185; nE (Ess), nEH &#194;&#185; nEnH (Exp), EnH &#194;&#185; nEH (EvH), <br /> larger strand conservation essential genes (SC) signi&#194;&#174;cant codon usage bias ribosomal proteins (CB). <br />  <br />  <br />  <br />  <br /> signi&#194;&#174;cantly higher frequency leading strand essential                      (Buchnera, Chlamydia, Spirochetes Rickettsia), also <br /> non-highly expressed genes non-essential highly                           obligatory pathogen (X.fastidiosa). <br /> expressed genes. The exceptions trend non- <br /> free-living Bacteria Buchnera X.fastidiosa, not <br />                                                                                    The positioning essential genes leading strand <br /> showing suf&#194;&#174;cient codon usage bias reliably determine <br />                                                                                    highly conserved <br /> expression levels. <br />                                                                                    These observations suggest essentiality important in <br /> Extension procedure. The assignment essentiality                      low G + C &#194;&#174;rmicutes g-proteobacteria, possibly <br /> homology distant phylogenetic domains prone                        bacterial domains. However, closely related genomes <br /> more important errors, essentiality relative set                   resemble E.coli B.subtilis phylogenetic <br /> of functions lifestyle de&#194;&#174;nition                            inertia, common selective pressure a <br /> orthology increasingly error prone evolutionary                         positioning essential genes leading strand. Some <br /> distances increase. In cases, de&#194;&#174;nition                       close genomes, E.coli Y.pestis (31) or <br /> high expression based CAI problematic. Still,                    S.agalactiae S.pneumoniae (32), severely <br /> made preliminary analysis entire set sequenced                         shuf&#194;&#175;ed speciation. Interestingly, genome rearrange- <br /> bacteria, including 30 fully sequenced genomes                            ments tend symmetrical origin replication <br /> different species. For genome, determined                              change replicating strand gene is <br /> orthologues B.subtilis. These orthologues classed                     coded, disrupt gene order (33,34). Therefore, <br /> as essential non-essential according classi&#194;&#174;cation                  performed test determine strand <br /> B.subtilis. The results qualitatively similar                        conservation stronger essential genes among <br /> presented above. First, essential genes biased                       genes (SC column Table 1). The results indicate that <br /> non-essential genes nearly genomes (Fig. 1). Secondly,                      21 23 genomes, conservation leading strand <br /> among highly expressed genes, essential genes                           character essential genes signi&#194;&#174;cantly important <br /> more biased non-essential ones. Thirdly, highly expres-                       that genes (P &lt; 0.05, applying the <br /> sed non-essential genes biased essential                     Bonferroni correction multiple tests) (Table 1). The two <br /> highly expressed genes. In analysis, observed                        exceptions concern Buchnera T.tengcongensis. In the <br /> some exceptions, concerning intracellular bacteria                          latter, conservation important but, genome <br />  6574     Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 2. Comparison essential genes data set E.coli and <br />                                                                                  B.subtilis. Upper panel: assignment essentiality orthologues (black <br />                                                                                  squares represent essential genes B.subtilis). Lower panel: presence or <br />                                                                                  absence orthologues depending classi&#194;&#174;cation essentiality the <br />                                                                                  genomes (black squares represent genes present B.subtilis absent <br />                                                                                  E.coli). The area blocks proportional relative frequency of <br />                                                                                  category. <br />  <br />  <br />                                                                                  Methodological problems lead erroneous infer- <br />                                                                                  ence orthology gene duplication non- <br />                                                                                  orthologous gene displacement (36). Hence, important <br />                                                                                  test robustness assumptions concerning <br />                                                                                  essentiality. The low G + C &#194;&#174;rmicutes sharing fewer <br />                                                                                  orthologues B.subtilis essential genes S.pyogenes <br />                                                                                  C.acetobutylicum (Table 1). Yet, cases, could <br />                                                                                  identify orthologues 78% B.subtilis essential genes. <br /> Figure 1. Frequency leading strand genes 53 genomes bacteria. For <br />                                                                                  Among g-proteobacteria, genome containing fewer <br /> the bacteria chromosome, pooled               orthologues E.coli essential genes Buchnera. Buchnera <br /> chromosomes. Essentiality assigned homology B.subtilis,           relatively close E.coli, evolutionary history of <br /> only genes presenting orthologous gene B.subtilis considered        extensive genome reduction led genome only <br /> the analysis. (A) Essential versus non-essential genes. (B) Essential versus     approximately 500 genes (37). Although 13% the <br /> non-essential genes putative highly expressed genes. (C) Highly <br /> expressed non-essential genes versus essential non-highly expressed genes.       E.coli non-essential genes orthologue Buchnera, <br /> The diagonal line indicates similar frequency sets. Intracellular   77% essential genes orthologue. These <br /> obligatory pathogens symbionts indicated open circles          results indicate majority essential genes are <br /> other genomes circles. In panels, essential genes     conserved evolutionary distances, very <br /> biased set (P &lt; 0.001, paired t-tests). <br />                                                                                  genes conserved. We turned comparison of <br />                                                                                  distantly related E.coli B.subtilis genomes. <br />  <br /> shows highest overall strand bias (35), difference                Comparison essential genes B.subtilis E.coli. Among <br /> signi&#194;&#174;cant application Bonferroni correction.                       1129 orthologues E.coli B.subtilis, found <br />                                                                                  75% B.subtilis essential genes 26% non- <br /> Tests                                                                            essential genes (Fig. 2). Thus, expected, presence of <br /> The previous analyses strongly suggest majority                   essential genes conserved other <br /> bacteria, essentiality drives gene strand bias. Yet, assigning                   genes large evolutionary distances. Further, 82% the <br /> essentiality homology prone errors. For example,                   E.coli essential genes classed essential B.subtilis <br /> all genomes lack orthologues essential B.subtilis                    (and 7% non-essential genes regarded essential <br /> genes, indicating non-essentiality bacteria.                      B.subtilis) (Fig. 2). Given large evolutionary <br />                                                                                          Nucleic Acids Research, 2003, Vol. 31, No. 22                6575 <br />  <br />  <br />                      Table 2. Cross-comparisons frequency genes leading strand using experimental data <br />                      sets essential genes genomes <br />                                            Experimental data              Homology                        Both <br />                                            Essential   Non-essential      Essential     Non-essential     Essential     Non-essential <br />  <br />                      M.genitalium (Bs)     82%          75%               85%           78%               86%           77% <br />                      H.in&#194;&#175;uenzae (Ec)      55%          55%               70%           51%               73%           53% <br />                      B.subtilis (Ec)       94%          72%               97%           83%               97%           80% <br />                      E.coli (Bs)           76%          56%               75%           56%               84%           56% <br />  <br />                      Three types information essentiality used compute frequency leading strand essential <br />                      genes: (i) using experimental data set genome; (ii) homology data set genome <br />                      (Bs, B.subtilis; Ec, E.coli); (iii) making intersection (i) (ii). For example, M.genitalium shows <br />                      82, 85 86% essential genes leading strand essentiality de&#194;&#174;ned respectively its <br />                      experimental data, homology essential genes B.subtilis, using intersection two <br />                      criteria (essential B.subtilis M.genitalium). <br />  <br />  <br />  <br /> distance bacteria, &#194;&#174;nding                        Table 3. Percentage genes leading strand genomes, <br /> large fraction essential genes conserved                      according essentiality <br /> genomes suggests assigning essentiality homology                    Comparison                             Leading       Leading       Expected <br /> reasonable, free-living bacteria.                                                                    essential     others <br />  <br /> Comparisons experimental data sets. We used                        B.subtilis versus M.genitalium         85%           70%           60% <br /> experimental data sets essential genes B.subtilis, E.coli,               B.subtilis versus E.coli               73%           44%           41% <br />                                                                                E.coli versus H.in&#194;&#175;uenzae              60%           35%           35% <br /> M.genitalium H.in&#194;&#175;uenzae make set cross- <br /> comparisons. In comparison, different                       A gene regarded essential experimentally essential <br /> analyses (Table 2). The &#194;&#174;rst analysis consists simply                       genomes. <br /> computing frequency leading strand essential non- <br /> essential genes genome, essentiality taken from <br /> the experimental data set [e.g. M.genitalium data set <br /> in analysis `M.genitalium (Bs)&apos;]. The second analysis                      leading strand character essential genes. For this, we <br /> consists computing frequency leading strand essential                compared B.subtilis genomes M.genitalium and <br /> genes genome essentiality assigned                          E.coli, H.in&#194;&#175;uenzae. We that <br /> homology genome [e.g. assigned                             genes essential genomes tend co-occur the <br /> homology B.subtilis analysis `M.genitalium (Bs)&apos;].                   leading strand higher frequency other <br /> The analysis consists computing frequency                      genes (Table 3). For example, comparison B.subtilis <br /> leading strand essential genes essentiality assigned                  E.coli, gene order <br /> only gene essential experimental data set              conservation (39), shows 73% essential genes leading <br /> its orthologue essential genome [e.g. gene              strand genomes. The similar analysis applied the <br /> is experimentally essential M.genitalium                   remaining genes indicates 44% genes conserved the <br /> and B.subtilis analysis `M.genitalium (Bs)&apos;]. The                    leading strand, expected frequency, taking into <br /> three analyses consistently results,                        account average gene strand composition genome, <br /> essential genes frequent leading strand                      41%. Between B.subtilis M.genitalium, difference is <br /> independently information used de&#194;&#174;ne essentiality.                   smaller (85 versus 70%), expected value much <br /> Only analysis H.in&#194;&#175;uenzae&apos;s experimental data set                   larger (60%). This conservation positioning essential <br /> indicates signi&#194;&#174;cant difference essential non-                  genes leading strand consistent existence of <br /> essential genes. The experimental analysis H.in&#194;&#175;uenzae                      selective forces displacement essential genes to <br /> identi&#194;&#174;ed 474 essential genes 1272 analysed (i.e. ~70%                   lagging strand. <br /> of genome) (2), twice essential <br /> genes B.subtilis (4), predicted H.in&#194;&#175;uenzae <br /> from computational genome analysis (38). In H.in&#194;&#175;uenzae,                       DISCUSSION <br /> essentiality determined transposon mutagenesis, which <br /> is likely overestimate number essential genes.                       Essentiality drives gene strand bias <br /> Unsurprisingly, use E.coli data set intersection             As discussed above, lines evidence indicate the <br /> between data sets indicates signi&#194;&#174;cant bias (73%                  positioning certain genes leading strand subject to <br /> essential leading strand genes latter). This general              selection. Brewer proposed biases caused <br /> trend: highest biases intersection               selection avoidance frequent head-on collisions <br /> data sets (Table 2), probably case likehood                DNAP RNAP (13). Here, con&#194;&#174;rmed our <br /> of falsely assigning essentiality minimised.                                previous analyses, showing bacteria, essentiality, not <br />                                                                                expressiveness, plays major role gene strand bias (19). <br /> Testing degree gene strand conservation. We used                   This compatible previous model emphasis is <br /> the experimental data sets test conservation                shifted rate collisions polymerases the <br />  6576   Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br /> transcript resulting collisions. Most importantly,     non-essential genes, differences typically high. <br /> is gene product, expression rate,          There are, however, exceptions analysis of <br /> important selective factor. Thus, understanding fate     highly expressed genes versus essential highly <br /> transcript cast light causes gene strand bias. A      expressed genes, concern obligatory parasites/ <br /> possibility that, transcription abortion occurs       symbionts, obligate intracellular (Fig. 1C). One <br /> lower rates co-oriented collisions head-on collisions    propose expression levels important in <br /> (14), truncated transcripts lead truncated, non-    genomes. This unlikely reasons. First, these <br /> functional, peptides. When peptides parts large         bacteria grow slowly expression levels <br /> complexes (e.g. ribosome DNAP), typically       important fast growers. Secondly, we <br /> basis dominant-negative phenotypes. This naturally           used CAI determine high levels expression. Yet, the <br /> most counter-selected essential functions.                    genomes spirochetes (30) Chlamydia (40) exhibit weak <br />    We extended previous observations concerning gene            (although signi&#194;&#174;cant) codon usage biases highly <br /> strand bias directions. First, shown       expressed genes, Buchnera genomes nearly devoid <br /> spite substantial rearrangements E.coli          biases (41). Our tests CAI two <br /> B.subtilis, leading strand positioning essential genes    genomes g-proteobacteria low G + C &#194;&#174;rmicutes <br /> conserved. It conserved average       lack signi&#194;&#174;cant codon usage bias ribosomal proteins: <br /> orthologue. Secondly, results putatively          T.tengcongensis Buchnera. If CAI bad predictor of <br /> identi&#194;&#174;ed essential genes preferably located leading     expression levels genomes, safely <br /> strand analysed genomes low G + C              attribute atypical bias expression levels. Thirdly, gene <br /> &#194;&#174;rmicutes g-proteobacteria. Hence, collision problems       strand bias putative highly expressed non-essential <br /> associated gene strand bias likely general        genes relatively small. Therefore, expression unlikely to <br /> importance bacterial world. Thirdly,       play important role bacteria in <br /> essentiality taken account, expression levels        B.subtilis E.coli. Genome rearrangements change the <br /> signi&#194;&#174;cantly contribute explain gene strand bias        leading strand character gene and, frequent, they <br /> groups. Although essential genes synthesised         lead lower gene strand bias. Although genomes of <br /> high levels, frequency highly expressed genes among <br />                                                                     obligatory intracellular bacteria stable, Buchnera <br /> essential genes higher average (e.g. nearly 30% <br />                                                                     suffered extensive rearrangements associated process <br /> among B.subtilis essential genes). It quite natural that <br />                                                                     genome reduction, lowered gene strand <br /> previous analyses using rDNA ribosomal proteins have <br />                                                                     bias. It possible bacteria, leading strand <br /> revealed expressiveness basis gene strand bias: <br />                                                                     positioning essential genes weaker selection, e.g. <br /> these genes simultaneously essential highly expressed. <br /> Here, essentiality taken account,         drift associated small population <br /> expression plays small, any, role determining           sizes. Many bacteria suffer relaxed selection of <br /> distribution genes replication strands. All        housekeeping functions (9). Also, depend host for <br /> results derived analyses orthology assignment       functions deemed essential. As such, the <br /> of essentiality orthologous relationships. As       assignment essentiality homology error <br /> shown, prone errors. Yet, degree             prone functions, especially metabolic ones. <br /> consistency results phylogenetic          On hand, intracellular bacteria some <br /> groups cross-validations performed suggest          essential genes (e.g. transporters metabolites) that <br /> that errors strongly biasing conclusions.          essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span>, essential in <br />                                                                     B.subtilis. A clearer insight gene strand bias these <br /> Understanding differences gene strand bias intensity             genomes require availability experimental data sets <br /> There signi&#194;&#174;cant differences terms overall gene           essential genes bacteria (note M.genitalium <br /> strand bias bacteria. As discussed (17),        does belong category an <br /> composition DNAP explain differences.          intracellular bacterium). <br /> This effect independent genome size                 In addition suggesting lower biases obligatory <br /> species ecology relate differential stability     intracellular bacteria, data indicate different levels of <br /> the replication fork. Bacillus subtilis higher general   gene strand bias different bacteria. The causes may <br /> gene strand bias (75%) higher essential gene strand bias      different stabilities replication forks, as <br /> (94%) E.coli (55 76%, respectively). Hence,            discussed above, different RNAP processivity, leading to <br /> effect essentiality gene strand bias adds caused   RNAP robust collisions, different ef&#194;&#174;ciency <br /> differences DNAP composition, effects            mechanisms degrading truncated peptides. When an <br /> likely related. For example, speculate         aborted transcript results stalled ribosomes, tmRNA tags the <br /> having DNAP a-subunits leads increased asym-              truncated peptide directs protease degradation <br /> metry outcome collisions DNAP RNAP.           machinery (42,43). tmRNA does direct elimination of <br /> This lead higher frequency leading strand genes       corresponding mRNA likely saturated <br /> in general, essential genes particular.                   replicating bacteria, large number collisions. <br />    Except Buchnera, essential genes           Nevertheless, tmRNA intrinsically ef&#194;&#174;cient, one <br /> more frequent leading strand remaining          expect lower levels gene strand bias bacteria. <br /> genes (Fig. 1A). When plots, putative highly              Experimental work necessary tackle these <br /> expressed genes, distribution essential versus               questions. <br />                                                                                           Nucleic Acids Research, 2003, Vol. 31, No. 22                 6577 <br />  <br />  <br /> SUPPLEMENTARY MATERIAL                                                          20. Louarn,J.M., Bouche,J.P., Legendre,F., Louarn,J. Patte,J. (1985) <br />                                                                                     Characterization properties large inversions E.coli <br /> Supplementary Material available NAR Online.                                  chromosome origin-to-terminus axis. Mol. Gen. Genet., 201, <br />                                                                                     467&#194;&#177;476. <br />                                                                                 21. Segall,A., Mahan,M.J. Roth,J.R. (1988) Rearrangement the <br /> ACKNOWLEDGEMENTS                                                                    bacterial chromosome: forbidden inversions. Science, 241, 1314&#194;&#177;1318. <br />                                                                                 22. Sharp,P.M. Li,W.H. (1987) The codon adaptation index&#195;?a measure <br /> We thank Joe&#195;&#710;l Pothier comments manuscript,                         directional synonymous codon usage bias potential <br /> the scientists disrupted studied genes                       applications. Nucleic Acids Res., 15, 1281&#194;&#177;1295. <br /> B.subtilis genome, particular                             23. Rocha,E.P.C. Danchin,A. (2001) Ongoing evolution strand <br /> enterprise happen S. D. Ehrlich, F. Kunst, N. Ogasawara                         composition bacterial genomes. Mol. Biol. Evol., 18, 1789&#194;&#177;1799. <br />                                                                                 24. Tatusov,R.L. Koonin,E.V. (1997) A genomic perspective protein <br /> H. Yoshikawa.                                                                       families. Science, 278, 631&#194;&#177;637. <br />                                                                                 25. Coghlan,A. Wolfe,K.H. (2000) Relationship codon bias mRNA <br />                                                                                     concentration protein length Saccharomyces cerevisiae. Yeast, 16, <br /> REFERENCES                                                                          1131&#194;&#177;1145. <br />  1. Hutchison,C.A., Peterson,S.N., Gill,S.R., Cline,R.T., White,O.,             26. Jansen,R., Bussemaker,H.J. Gerstein,M. (2003) Revisiting codon <br />     Fraser,C.M., Smith,H.O. Venter,J.C. (1999) Global transposon                adaptation index whole-genome perspective: analyzing the <br />     mutagenesis minimal Mycoplasma genome. Science, 286,                      relationship gene expression codon occurrence yeast <br />     2165&#194;&#177;2169.                                                                      using variety models. Nucleic Acids Res., 31, 2242&#194;&#177;2251. <br />  2. Akerley,B.J., Rubin,E.J., Novick,V.L., Amaya,K., Judson,N.              27. Sharp,P.M. Li,W.-H. (1986) An evolutionary perspective on <br />     Mekalanos,J.J. (2002) A genome-scale analysis identi&#194;&#174;cation              synonymous codon usage unicellular organisms. J. Mol. Evol., 24, <br />     genes required growth survival Haemophilus in&#194;&#175;uenzae.                 28&#194;&#177;38. <br />     Proc. Natl Acad. Sci. USA, 99, 966&#194;&#177;971.                                     28. Andersson,S.G.E. Kurland,C.G. (1990) Codon preferences in <br />  3. Forsyth,R.A., Haselbeck,R.J., Ohlsen,K.L., Yamamoto,R.T., Xu,H.,                free-living microorganisms. Microbiol. Rev., 54, 198&#194;&#177;210. <br />     Trawick,J.D., Wall,D., Wang,L., Brown-Driver,V., Froelich,J.M. et al.       29. Me&#195;&#8218;digue,C., Rouxel,T., Vigier,P., Henaut,A. Danchin,A. (1991) <br />     (2002) A genome-wide strategy identi&#194;&#174;cation essential genes       Evidence horizontal gene transfer E.coli speciation. J. Mol. Biol., <br />     Staphylococcus aureus. Mol. Microbiol., 43, 1387&#194;&#177;1400.                          222, 851&#194;&#177;856. <br />  4. Kobayashi,K., Ehrlich,S.D., Albertini,A., Amati,G., Andersen,K.K.,          30. Perriere,G. Thioulouse,J. (2002) Use misuse correspondence <br />     Arnaud,M., Asai,K., Ashikaga,S., Aymerich,S., Bessieres,P. et al. (2003)        analysis codon usage studies. Nucleic Acids Res., 30, 4548&#194;&#177;4555. <br />     Essential Bacillus subtilis genes. Proc. Natl Acad. Sci. USA, 100,          31. Parkhill,J., Wren,B.W., Thomson,N.R., Titball,R.W., Holden,M.T., <br />     4678&#194;&#177;4683.                                                                      Prentice,M.B., Sebaihia,M., James,K.D., Churcher,C., Mungall,K.L. et al. <br />  5. Jordan,I.K., Rogozin,I.B., Wolf,Y.I. Koonin,E.V. (2002) Essential           (2001) Genome sequence Yersinia pestis, causative agent of <br />     genes evolutionarily conserved nonessential genes          plague. Nature, 413, 523&#194;&#177;527. <br />     bacteria. Genome Res., 12, 962&#194;&#177;968.                                         32. Glaser,P., Rusniok,C., Buchrieser,C., Chevalier,F., Frangeul,L., <br />  6. Pal,C., Papp,B. Hurst,L.D. (2003) Rate evolution gene                Msadek,T., Zouine,M., Couve,E., Lalioui,L., Poyart,C. et al. (2002) <br />     dispensability. Nature, 421, 496&#194;&#177;497.                                           Genome sequence Streptococcus agalactiae, pathogen causing <br />  7. Hirsh,A.E. (2003) Rate evolution gene dispensability&#195;?reply.              invasive neonatal disease. Mol. Microbiol., 45, 1499&#194;&#177;1513. <br />     Nature, 421, 497&#194;&#177;498.                                                       33. Tillier,E.R. Collins,R.A. (2000) Genome rearrangement by <br />  8. Rocha,E.P.C. Danchin,A. (2003) An analysis determinants               replication-directed translocation. Nature Genet., 26, 195&#194;&#177;197. <br />     protein substitution rates bacteria. Mol. Biol. Evol., press.         34. Eisen,J.A., Heidelberg,J.F., White,O. Salzberg,S.L. (2000) Evidence <br />  9. Ochman,H. Moran,N.A. (2001) Genes lost genes found:                 <br /> </body></html>