<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIOXIDANTS &amp; REDOX SIGNALING <br /> Volume 10, Number 11, 2008 <br /> &#194;&#169; Mary Ann Liebert, Inc. <br /> DOI: 10.1089/ars.2008.2120 <br />  <br />  <br />  <br />  <br />                         Forum Original Research Communication <br />  <br />               Role Multifunctional DNA Repair and <br />            Redox Signaling Protein Ape1/Ref-1 Cancer and <br />             Endothelial Cells: Small-Molecule Inhibition the <br />                         Redox Function Ape1 <br />  <br />                      Meihua Luo,1 Sarah Delaplane,2 Aihua Jiang,3 April Reed,1 Ying He,1 <br />                      Melissa Fishel,1 Rodney L. Nyland II,5 Richard F. Borch,5 Xiaoxi Qiao,3 <br />                                  Millie M. Georgiadis,2 Mark R. Kelley1,2,4,6 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  <br /> The DNA <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> excision-repair pathway responsible repair DNA damage caused oxidation/alky- <br /> lation protects cells effects endogenous exogenous agents. Removal damaged base <br /> creates baseless (AP) site. AP endonuclease1 (Ape1) acts site continue BER-pathway repair. Fail- <br /> ure repair baseless sites leads DNA strand breaks cytotoxicity. In addition repair role Ape1, <br /> it functions major redox-signaling factor reduce activate transcription factors AP1, p53, <br /> HIF-1 , control expression genes important cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> cancer promotion and <br /> progression. Thus, Ape1 protein interacts proteins involved DNA repair, growth-signaling path- <br /> ways, pathways involved tumor promotion progression. Although knockdown studies siRNA <br /> have informative studying role Ape1 normal cancer cells, knocking Ape1 does <br /> not reveal individual role redox repair functions Ape1. The identification small-molecule in- <br /> hibitors specific Ape1 functions critical mechanistic studies translational applications. Here dis- <br /> cuss small-molecule inhibition Ape1 redox effect cancer endothelial cells. Antioxid. Re- <br /> dox Signal. 10, 1853&#226;&#8364;&#8220;1867. <br />  <br />  <br /> Introduction                                                     enine. Ogg1, Nth1, NEIL1, NEIL2 recognize remove <br />                                                                  purine pyrimidine oxidative DNA-damaged <br />  <br /> T    HE DNA BASE EXCISION-REPAIR (BER) pathway respon- <br />      sible repair DNA caused oxidative, alkyla- <br /> tion, ionizing radiation protects cells against <br />                                                                  adducts. All glycosylases require pro- <br />                                                                  cessing initiating step BER pathway Ape1 <br />                                                                  (Fig. 1). Ape1 (apurinic/apyrimidinic endonuclease 1) the <br /> the toxic effects endogenous exogenous agents. Re-        major repair protein abasic sites, accounting 95% of <br /> moval incorrect damaged base DNA glycosy-         AP endonuclease activity (77). It cleaves 5  abasic <br /> lase, simple glycosylase methylpurine       site generating normal 3 -hydroxyl group abasic <br /> DNA glycosylase (MPG AAG) complex glycosylase            deoxyribose-5-phosphate, processed subsequent <br /> (Ogg1, Nth1, NEIL1, etc.) composes step BER     enzymes BER pathway. The removal damaged <br /> pathway (Fig. 1). MPG repairs alkylated DNA major cy-        base results AP site that, repaired, result in <br /> totoxic lesion, 3-methyladenine (3-me A) functions      block DNA replication genetic instability (77). DNA <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> cleave major product lipid oxidation; 1,N(6)-ethenoad-   polymerase   recruited abasic site Ape1 can <br />  <br />  <br />   1Department Pediatrics (Section Hematology/Oncology), Herman B Wells Center Pediatric Research, 2Department Bio- <br />  <br /> chemistry &amp; Molecular Biology, 3Department Ophthalmology, 4Department Pharmacology &amp; Toxicology, Indiana University <br /> School Medicine, Indianapolis, Indiana. <br />   5Department Medicinal Chemistry Molecular Pharmacology Cancer Center, Purdue University, West Lafayette, Indiana. <br />  <br />  <br />                                                             1853 <br />  1854                                                                                                             LUO ET AL. <br />  <br />  <br />  <br />  <br /> FIG. 1. Oxidative alkylation DNA damage effectively repaired DNA BER pathway. This sim- <br /> plified drawing DNA BER pathway. For detailed discussion, Fishel et al. (21). (A) Damaged bases (in this <br /> case N3-adenine) removed glycosylases result baseless apurinic (AP) sites. Ape1 acts AP sites, pro- <br /> duces nick allows &#195;&#8240;-polymerase insert correct base DNA ligase connect DNA backbone. (B) In ad- <br /> dition repair function, Ape1 serves redox factor maintaining transcription factors active reduced state. <br /> Some example targets shown. Therefore, protection Ape1 DNA damage twofold: directly through <br /> its DNA-repair function indirectly redox activity enabling transcription factors bind DNA re- <br /> spond DNA damage. Lack Ape1 activity lead cell death. <br />  <br />  <br />  <br />  <br /> use 3 OH substrate removed 5  de- <br /> oxyribose phosphate terminus. DNA ligase I ligates the <br /> remaining break phosphodiester backbone, complet- <br /> ing repair. Other enzymatic repair activities associated <br /> with Ape1 include 3 -repair diesterase phosphatase ac- <br /> tivity (62), important repair DNA damaged <br /> by IR, 3 -5  exonuclease activity, reported play a <br /> role excision deoxyribonucleoside analogues (7&#226;&#8364;&#8220;10). <br />    However, Ape1 major cellular function: <br /> Ape1 functions reduction&#226;&#8364;&#8220;oxidation (redox) factor and <br /> stimulates DNA-binding activity numerous transcrip- <br /> tion factors involved cancer promotion pro- <br /> gression, AP-1 (Fos/Jun), NF- B, PAX, HIF-1 , HLF, <br /> p53, (Fig. 1) (16, 21, 71). The redox function of <br /> Ape1 mammals vertebrates, <br /> as demonstrated lack redox function ze- <br /> brafish Ape1 (zApe1) (Fig. 2). The acquisition redox <br /> function Ape1 proteins discussed recent publica-      FIG. 2. Comparison selected Ape1 family members <br /> tion (25). Each molecularly distinct functional domains   bacteria mammals. The percentage displayed rep- <br /> of Ape1, redox repair, completely independent         resents amino acid similarity respective pro- <br /> their function [i.e., mutations cysteine position 65   tein human Ape1. Only mammals acquired re- <br /> Cys65 (C65) removes redox function does affect       dox function. <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                           1855 <br />  <br /> the repair function (Fig. 3), <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>mutation</span> variety     alkylating oxidative DNA-damaging agents <br /> amino acids required DNA repair activity, His309       (19&#226;&#8364;&#8220;21, 30, 44&#226;&#8364;&#8220;47, 51, 70, 75, 81). Finally, blocking activ- <br /> (H309) (51) (Fig. 3) does affect redox func-    ity Ape1 IR lead increase tumor cell <br /> tion]. Whereas DNA-repair active site Ape1         killing (37, 68, 73). <br /> clearly delineated (26), redox domain region ob-       However, use antisense methods cancer cells re- <br /> vious. It appears required Cys redox        moves repair function Ape1, re- <br /> function Cys65, buried Ape1 protein         dox function protein&#226;&#8364;&#8220;protein interactions mediated <br /> (Fig. 3). This recently confirmed Georgiadis        Ape1. Ape1 thought interact large number of <br /> co-workers (25) mutated Thr (T) zebrafish           repair-related proteins including Ogg1 (66), DNA gly- <br /> Ape1 located position Cys mammalian          cosylases, DNA polymerase   (61), x-ray cross-complement- <br /> Ape1s Cys, subsequent protein gained redox            ing-1 (XRCC1) (4, 53, 61), proliferating cell nuclear antigen <br /> function (25).                                                     (PCNA) (14), Cockayne syndrome B (CSB) (78), flap en- <br />    Interestingly, zebrafish crystal structure quite simi-   donuclease 1 (FEN1) (14, 61, 64). <br /> lar human Ape1, seven Cys            Additional functions ascribed Ape1 also <br /> mammalian Ape1s structurally conserved. The           complicate analysis siRNA. For example, been <br /> are conserved Cys65 Cys138 (25). An evolution-         implicated nucleotide incision repair (NIR) (76), involved <br /> ary analysis Cys residues vertebrates suggests       NK cell&#226;&#8364;&#8220;mediated killing granzyme A (GzmA) <br /> that presence Cys65 distinguishes mammalian            (17, 50), prevents oxidative stress negatively regulating <br /> nonmammalian vertebrate sequences (25).                            Rac1/GTPase activity(60), regulates endothelial NO pro- <br />    The importance Ape1 cell demonstrated       duction vascular tone (38), recent finding that <br /> fact possible generate animal knock-    important studies, Ape1 appears suppress the <br /> out model. Ape1 mouse knockouts embryonic lethal            activation PARP1 [poly (ADP-ribose) polymerase 1] dur- <br /> days E5 E9, viable cell lines established      ing repair process oxidative DNA damage, may <br /> that completely deficient Ape1 (79). Previously,         allow cells avoid cell death (63). Other findings rela- <br /> small number studies using DNA antisense methods           tion Ape1 Bcl2 levels (39) acting neg- <br /> implicated Ape1 cellular resistance variety agents      ative regulator parathyroid hormone gene been <br /> (6, 16, 59, 74). By using antisense Ape1, hypersensitivity      shown (1, 11, 42, 58). Finally, recent data link Ape1 with <br /> human HeLa, rat glioma, human lung carcinoma cells           PTEN (18), major tumor suppressor impli- <br /> alkylating oxidative agents ionizing radiation      cated playing crucial role variety cancers, in- <br /> but UV radiation indicated (16). More recently,          cluding brain tumors (5). <br /> similar enhanced sensitivity osteogenic sarcoma cells          Thus, using antisense RNA similar technology, it <br /> DNA-damaging agents reported (75). Cells lacking          possible determine precisely role en- <br /> Ape1 activity hypersensitive alkylating agents in-     donuclease redox function Ape1 cancer normal <br /> duce formation AP sites (13, 24). Targeted reduction        cells specific inhibitors function indepen- <br /> of Ape1 protein specific anti-sense oligonucleotides ren-       dently. Therefore, long-term goal developing can- <br /> ders mammalian cells hypersensitive methylmethane sul-          cer therapeutic agents, small-molecule inhibitors <br /> fonate (MMS), H2O2, bleomycin, temozolomide (TMZ),             major functions Ape1, present data further <br /> gemcitabine (2, 32, 43, 59, 74). Additionally, blocking re-    characterizing Ape1 redox mechanism action and <br /> pair function Ape1 led enhanced effectiveness         demonstrating role redox function Ape1 can- <br /> DNA-damaging agents temozolomide (TMZ),                cer retinal endothelial cells (RECs) survival and <br />  <br />  <br />  <br /> FIG. 3. Cartoon major amino acids <br /> involved DNA repair redox <br /> function Ape1. (A) Seven cysteines ex- <br /> ist Ape1, Cys65 appears be <br /> involved redox activity. Of Cys <br /> residues, Cys65 unique mammals <br /> (25). The lower left panel (B) shows rel- <br /> ative positions seven Cys residues <br /> on ribbon rendering Ape1. The lower <br /> right panels demonstrate Cys65 is <br /> a buried residue. All atoms Ape1 are <br /> shown van der Waals surface render- <br /> ing (C), C65 green P112 in <br /> red. (D) A molecular surface rendering <br /> colored way same <br /> orientation (C). Cys 65 visible <br /> because surface accessible (i.e., it <br /> is buried). (For interpretation refer- <br /> ences color figure legend, the <br /> reader referred web version this <br /> article www.liebertonline.com/ars). <br />  1856                                                                                                                 LUO ET AL. <br />  <br /> growth using specific small-molecule inhibitor just       (1:9 EtOAc/hexanes), tetramethoxytoluene product was <br /> the redox function Ape1. Inhibition redox function      collected yellow oil (2.27 g, 88%). Rf   0.57 (7:13 <br /> blocks endothelial cell growth blocks growth tu-       EtOAc:hexanes); 1H NMR (CDCl3):   2.43 (s, 3H); 3.74 (s, 3H); <br /> mor cell lines single agent. This effect independent   3.89 (s, 3H); 3.92 (s, 3H); 4.00 (s, 3H); 10.41 (s, 1H). <br /> the Ape1 DNA-repair function.                                        According procedure Ohkawa et al. (57), tetram- <br />                                                                   ethoxytoluene (0.690 g, 3.25 mmol) dissolved CH2Cl2 <br /> Materials Methods                                             (2.0 ml) flame-dried 25-ml round-bottom flask and <br />                                                                   cooled 0&#194;&#176;C.  , -Dichloromethyl methyl ether (0.85 ml, 9.6 <br /> Synthesis E3330, 3-[5-(2,3-dimethoxy-6-methyl-1,4- <br />                                                                   mmol) added 0&#194;&#176;C, followed TiCl4 (1 M in <br /> benzoquinoyl)]-2-nonyl-2-propionic acid <br />                                                                   CH2Cl2, 9.0 ml, 9.0 mmol) 0&#194;&#176;C. The reaction warmed <br />    Reagents used received commercial sources.        slowly room temperature stirred 6 h argon. <br /> THF CH2Cl2 distilled use, acetone         The reaction poured chilled water stirred <br /> dried activated 4-&#195;&#8230; molecular sieves 2 h ar-       10 min. The reaction diluted EtOAc, washed <br /> gon. Flash chromatographic separations performed          brine, dried MgSO4, filtered, condensed. The <br /> using 32- 63- m silica gel; thin-layer chromatography      crude oil purified flash chromatography (3:17 <br /> performed Analtech 250- m GHLF plates 254-nm            EtOAc:hexanes) provide 3 (0.696 g, 89%) yellow oil. <br /> fluorescent indicator. All 1H-NMR spectra obtained        Rf   0.26 (3:17 EtOAc:hexanes); 1H NMR (CDCl3):   2.44 (s, <br /> a Bruker 300-MHz NMR equipped multinuclear (1H,            3H); 3.74 (s, 3H); 3.89 (s, 3H); 3.92 (s, 3H); 4.00 (s, 3H); 10.24 <br /> 13C, 19F, 31P) 5-mm probe. 1H spectra calibrated      (s, 1H). <br /> CHCl3 7.24 ppm. 13C NMR calibrated CHCl3 77.0 <br /> ppm. Elemental analyses performed Purdue Uni-         Ethyl (E)-3-(6-methyl-2,3,4,5-tetramethoxyphenyl)-2- <br /> versity Microanalysis Laboratory Dr. Daniel Lee.               nonylpropenoate (4) <br />                                                                      By following modification procedure Shinkawa <br /> 1,2,3,4-Tetramethoxybenzene (2) <br />                                                                   et al. (55), NaH (0.24 g, 10.0 mmol) added flame- <br />    By following modified procedure Tremblay et al. (72),     dried 50-ml three-neck round-bottom flask connected a <br /> 2,3,4-trimethoxybenzaldehyde 1 (6.08 g, 31.0 mmol) dis-       water-jacketed reflux condenser. The flask purged with <br /> solved MeOH (60 ml), H2SO4 (0.6 ml) 35%           argon, drying tube attached top. THF (15 <br /> aq. H2O2 (4.0 ml,  40 mmol) added room tempera-            ml) added flask, followed triethyl 2-phospho- <br /> ture. The reaction heated reflux 2 h. The so-     noundecanoate (1.90 g, 5.44 mmol) dissolved THF (5 ml) <br /> lution cooled extracted ethyl acetate. The lay-      room temperature. The reaction heated reflux <br /> ers separated, organic layer washed         30 min, aldehyde 3 (1.04 g, 4.32 mmol) dis- <br /> brine, dried MgSO4, filtered. Flash chromatogra-         solved THF (8 ml) added slowly reflux. The re- <br /> phy (1:4 EtOAc:hexanes) gave 2,3,4-trimethoxyphenol (5.56         action stirred 3 h reflux then <br /> g, 98%) light-red oil. Rf   0.29 (1:4 EtOAc:hexanes); 1H     cooled, acidified dilute HCl, diluted ethyl acetate, <br /> NMR (CDCl3):   3.79 (s, 3H); 3.87 (s, 3H); 3.94 (s, 3H); 5.34     washed brine. The organic layer dried <br /> (bs, 1H, OH); 6.57 (q, 2H).                                       (MgSO4), filtered, condensed. The resulting oil pu- <br />    The intermediate phenol (3.85 g, 20.9 mmol) dissolved      rified flash-column chromatography (3:17 EtOAc:hexa- <br /> in dry acetone (90.0 ml), K2CO3 (9.36 g, 67.7 mmol)       nes) provide 4 (1.28 g, 68%) colorless oil. Rf   0.40 <br /> added, followed MeI (9.0 ml, 140 mmol). The reaction           (3:17 EtOAc:hexanes); 1H NMR (CDCl3):   0.84 (t, 3H); 1.08 <br /> mixture heated reflux 26 h, cooled room          &#226;&#8364;&#8220; 1.22 (m, 11H); 1.24 &#226;&#8364;&#8220; 1.35 (m, 3H); 1.33 (t, 3H); 2.03 (s, 3H); <br /> temperature, filtered. The solvent removed          2.12 (m, 2H); 3.69 (s, 3H); 3.78 (s, 3H); 3.88 (s, 3H); 3.93 (s, <br /> reduced pressure, residue taken CH2Cl2          3H); 4.25 (q, 2H); 7.37 (s, 1H). <br /> (50 ml). The suspension filtered, dried MgSO4, fil- <br /> tered again, condensed. The crude product purified        (E)-3-(4,5-dimethoxy-2-methyl-3,6-dioxocyclohexa-1,4- <br /> by flash chromatography (1:4 EtOAc:hexanes) re-         dienyl)-2-nonylpropenoic acid (5) <br /> crystallization Et2O/hexanes provide 2 (3.41 g, 82%) <br />                                                                      Ester 4 (1.57 g, 3.60 mmol) dissolved EtOH (12 ml), <br /> as white needles, mp   86&#226;&#8364;&#8220;87&#194;&#176;C (Lit. 87&#226;&#8364;&#8220;87.5&#194;&#176;C) (3). 1H NMR <br />                                                                   KOH (0.41 g, 7.3 mmol) added. The solution <br /> (CDCl3):   3.80 (s, 6H); 3.88 (s, 6 H); 6.56 (s, 2H). <br />                                                                   heated reflux stirred 30 min. The reaction was <br />                                                                   cooled, acidified HCl, extracted ethyl ac- <br /> 6-Methyl-2,3,4,5-tetramethoxybenzaldehyde (3) <br />                                                                   etate. The organic layer washed brine, dried over <br />    By following modified procedure Hansen et al. (28),       MgSO4, filtered, condensed. The resulting acid then <br /> tetramethoxybenzene 2 (2.40 g, 12.1 mmol) added          used following reaction purification. Alter- <br /> 100-ml flame-dried round-bottom flask THF (50 ml)        natively, product purified flash chromatogra- <br /> was added. The solution cooled 0&#194;&#176;C, n-BuLi (2.5        phy (2:3 Et2O:hexanes 0.5% AcOH) provide free acid <br /> M hexanes, 6.0 ml, 15 mmol) added slowly 0&#194;&#176;C.           (1.43 g, 97%) colorless oil: Rf   0.36 (2:3 Et2O:hexanes <br /> The solution stirred 1 h 0&#194;&#176;C; MeI (1.2 ml, 19.3   0.5% AcOH).1H NMR (CDCl3):   0.84 (t, 3H, J   6.9 Hz); 1.13 <br /> mmol) added slowly, reaction stirred 0&#194;&#176;C       (m, 12H); 1.36 (m, 2H); 2.04 (s, 3H); 2.14 (m, 2H); 3.70 (s, 3H); <br /> 2 h, reaction mixture warmed room temper-          3.79 (s, 3H); 3.89 (s, 3H); 3.93 (s, 3H); 7.54 (s, 1H). <br /> ature stirred additional hour. The reaction acid-         By following modifications procedures Shinkawa <br /> ified dilute HCl, washed twice brine, dried        et al. Flader et al. (23, 56), crude acid obtained ear- <br /> MgSO4, filtered, condensed. After flash chromatography        lier (1.13 g, 2.77 mmol) dissolved ethyl acetate (15.0 <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                             1857 <br />  <br /> ml) room temperature; HNO3 (1.5 ml) AcOH (6               transfected control pcDNA vector plasmid, <br /> drops) added, reaction stirred 4 h. The          pcDNA-wtApe1 redox mutant, pcDNA-C65A, a <br /> mixture diluted EtOAc (20 ml) washed               Renilla luciferase control reporter vector pRL-CMV (Promega <br /> brine, dried MgSO4, filtered, condensed. The red             Corp., Madison, WI) 10:1 ratio using lipofectamine <br /> oil purified flash column chromatogra-             TM 2000. After 24-h transfection, cells lysed, the <br /> phy (2:3 Et2O/hexanes, 0.5% AcOH) recrystallization           Firefly Renilla luciferase activities assayed using <br /> Et2O/hexanes afford 5 (0.443 g, 42%) red solid; mp            Dual Luciferase Reporter Assay System (Promega Corp.) <br /> 56&#226;&#8364;&#8220;57&#194;&#176;C (Lit. 68&#194;&#176;C) (56). Elemental analysis: calc C, 66.65%,         Renilla luciferase activity normalization 96-well <br /> H, 7.99%; C, 66.32%, H, 7.89%. 1H NMR (CDCl3):                  microtiter plate luminometer (Thermolabs Systems, <br /> 0.84 (t, 3H), 1.18 (bs, 14H); 1.39 (bs, 2H); 1.94 (d, 3H); 2.09 (t,   Franklin, MA). All transfection experiments per- <br /> 3H); 3.99 (s, 3H); 4.02 (s, 3H); 7.26 (d, 1H).                        formed triplicate repeated 3 times inde- <br />                                                                       pendent experiments. <br /> Expression purification proteins <br />                                                                          Ape1 transactivation mutation assays. The stable Skov-3X <br />   Proteins purified previously described (25, 29, 36,         ovarian cancer cell line NF- B-Luc gene (luciferase gene <br /> 40, 80).                                                              NF- B&#226;&#8364;&#8220;responsive promoter) established by <br />                                                                       transfecting Skov-3X cells plasmid pNF- B-Luc (Strat- <br /> Ape1 DNA repair assays                                                agene) using lipofectamine TM 2000 (Invitrogen Life <br />   Oligonucleotide gel-based Ape1 endonuclease activity as-            Technologies) screening luciferase-positive colonies <br /> says performed previously described (20, 41). The             using Luciferase Assay Kit (Promega Corp.). The stable <br /> base excision repair (BER) pathway analysis performed             Skov-3X cells NF- B-Luc gene cotransfected <br /> as described (67).                                                    plasmid pcDNA-wtApe1 redox mutant pcDNA- <br />                                                                       C65A Renilla luciferase control reporter vector pRL- <br />                                                                       CMV (Promega Corp.) 1:10 ratio using lipofectamine <br /> Redox assays <br />                                                                       TM 2000. The analysis conducted described earlier <br />    Electrophoretic mobility shift assay (EMSA). EMSAs            E3330 studies. <br /> performed described (15) following modifica- <br /> tions. The 10- g/ l purified Ape1 protein reduced            <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>Cell survival</span>/killing assays <br /> 1.0 mM DTT 370C 10 min diluted PBS buffer to <br /> yield final concentrations 2  g/ l protein 0.2 mM                 3-(4,5-Dimethylthiazol-2-yl)-5-(3-carboxymeyhoxyphenyl)- <br /> DTT. Two  l reduced Ape1 protein added EMSA                 2-(4-sulfophenyl)-2H-tetrazolium (MTT) dye assay cell <br /> reaction buffer [10 mM Tris (pH 7.5), 50 mM NaCl, 1 mM                growth performed previously published (20, 75). <br /> MgCl2, 1 mM EDTA, 5% (vol/vol) glycerol] 6  g nu- <br /> clear extract (treated 0.01mM diamide 10 min)           Murine retinal endothelial cell isolation culture <br /> Hey-C2 cells source AP1 protein total volume              Mouse retinal vascular endothelial cells (RVECs) iso- <br /> of 18  l incubated 30 min. room temperature.               lated described modifications (69). In brief, <br /> One  l poly(dI-dC)   poly(dI-dC) (1 g/ul) (Amersham Bio-              retinal tissues young adult wild-type mice carrying a <br /> sciences, Piscataway, NJ) added 5 min followed             SV40 transgene Charles River Laboratories (Im- <br /> 1  l 5  hexachloro-fluorescein phosphoramidite                 mortoMouse) digested 0.2 mg/ml collagenase <br /> (HEX)-labeled double-stranded oligonucleotide DNA (The                type I (Worthington, Lakewood, NJ) DMEM 37&#194;&#176;C 3 <br /> Midland Certified Reagent Company, Midland, TX) con-                  h. After titration filtration 40- m nylon mem- <br /> taining AP-1 consensus sequence (5 CGCTTGAT-                      brane (Fisher Scientific, Hanover Park, IL), dissociated <br /> GACTCAGCCGGAA-3 ) (0.1 pmol), mixture                     cells incubated sheep anti-rat magnetic beads (Dy- <br /> further incubated 30 min room temperature. The final           nal Biotech, Lake Success, NY) precoated rat anti- <br /> concentration DTT redox reactions 0.02 mM.              mouse PECAM-1 monoclonal antibody (BD Biosciences, San <br /> Samples loaded 5% nondenaturing polyacry-                   Jose, CA) affinity binding 4&#194;&#176;C 45 min. Bead-bound <br /> lamide gel subjected electrophoresis 0.5X TBE               cells separated magnetic force, plated 24-well <br /> buffer (200 V 1 h 4&#194;&#176;C) detected using Hi-           plate precoated 2  g/ml human fibronectin (BD Bio- <br /> tachi FMBio II Fluorescence Imaging System (Hitachi Genetic           sciences, San Jose, CA) grown endothelial growth <br /> Systems, South San Francisco, CA). The HEX fluorophore             media (EBM-2MV, Cambrex). Cells allowed grow <br /> excited solid-state laser 532 nm (Perkin-Elmer)           beads reaching confluence  10 days us- <br /> emits fluorescent light signal 560 nm, mea-        ing assays. <br /> sured using 585-nm filter. <br />                                                                       Cell-proliferation assay <br />   Transactivation assay <br />                                                                          Equal numbers RVECs seeded 2,000 cell/well <br />    E3330 assays. SKOV-3X Hey-C2 ovarian cancer cell               96-well plate. Cells cultured 10 <br /> lines co-transfected NF- B-Luc AP-1-Luc              ng/ml recombinant mouse basic fibroblast growth factor <br /> genes (luciferase gene NF- B AP-1 responsive              (bFGF; R&amp;D Systems, Inc.) supplement various concen- <br /> promoter) using plasmid pNF- B-Luc pAP-1-Luc                    trations E3330 (10, 25, 50, 100  M). Each group consisted <br /> (Stratagene, La Jolla, CA) lipofectamine TM 2000 (Invit-          wells (n   5) including vehicle control group. <br /> rogen Life Technologies, Carlsbad, CA). The cells co-            Two days seeding, total number cells as- <br />  1858                                                                                                              LUO ET AL. <br />  <br /> sayed using CellTiter 96 AQueous One Solution             methyl-1,4-benzoquinoyl)]-2-nonyl-2-propionic acid (E3330) <br /> (Promega Corporation). The proliferation rate calculated     (82). This molecule previously shown block Ape1 <br /> according manufacturer&#226;&#8364;&#8482;s instructions. Proliferations      function studies NF- B (35) hematopoietic <br /> RVECs different groups compared ve-           cells (82) specific Ape1 high binding <br /> hicle control group statistical significance.                affinity specificity (34, 65). To continue studies, <br />                                                                  E3330, commercially available, syn- <br /> Tube-formation assay                                             thesized using new approach, developed and <br />                                                                  described Methods section. This route developed <br />    Matrigel (BD Biosciences, Bedford, MA) used coat <br />                                                                  encountering difficulties previously reported <br /> each (50  l) precooled 96-well plate. RVECs 5,000 <br />                                                                  synthetic route (22, 23, 55). The result synthesis is <br /> cells seeded incubated EBM 1% <br />                                                                  shown Fig. 5. <br /> FBS 10-ng/ml bFGF 37&#194;&#176;C 24 h. The formation of <br />                                                                     By using E3330 series EMSA experiments, dem- <br /> capillary-like structures RVECs Matrigel quan- <br />                                                                  onstrate E3330 blocks redox function Ape1 with <br /> tified counting number closed tube units each <br />                                                                  AP-1 downstream target vitro (Fig. 6A) as <br /> well. The percentage tube formation vehicle con- <br />                                                                  treatment ovarian cancer cells E3330 (Fig. <br /> trol group calculated E3330 (1, 5, 20  M) treated <br />                                                                  6B). Additionally, demonstrate E3330 blocks Ape1 <br /> group (n   3). <br />                                                                  redox activity HIF-1  demonstrating redox in- <br />                                                                  hibition specific downstream target (Fig. 6C). <br /> Results <br />                                                                  Similar results NF- B EMSA studies been <br />    A number studies implicated Cys65 redox        observed (data shown). <br /> function Ape1. However, disagreement exists           Although E3330 blocked redox function Ape1, had <br /> whether Cys Cys involved, just major     effect Ape1 repair endonuclease activity (Fig. 7A), nor <br /> Cys involved redox activity. Recent data using zebrafish    did affect ability proteins involved BER path- <br /> Ape1 (zApe1) model demonstrates acquisition      way perform repair AP sites (Fig. 7B) using the <br /> Cys65 human Ape1 converts redox-inactive zApe1         established BER-pathway assay (67). These studies demon- <br /> redox-active zApe1 (25). To confirm this, mutated         strate specificity E3330 redox, re- <br /> Cys65 human Ape1 Ala (A) performed cellular         pair function Ape1. <br /> transactivation reporter analysis using AP-1 NF- B           As additional approach demonstrate specificity <br /> as downstream targets Ape1 (Fig. 4). When Cys65        E3330 Ape1, used mutant zebrafish (T58C), <br /> mutated Ala, Ape1 loses redox activity     converted redox-inactive Ape1 pro- <br /> cell-based reporter system, addition wild-type        tein redox-active protein (25), reacted E3330. <br /> Ape1 results enhanced redox activity (Fig. 4), regardless     We performed redox EMSA using AP-1 the <br /> of downstream target.                                        target protein. As shown Fig. 8, E3330 blocked ability <br />    Our series experiments involved use small    T58C zebrafish Ape1, acquired redox <br /> molecule redox inhibitor Ape1, 3-[5-(2,3-dimethoxy-6-         function, similar fashion blocks hu- <br />  <br />  <br />  <br />  <br />                                                                           FIG. 4. Effect human Ape1 Cys65 redox <br />                                                                           Ape1 mutant NF-      B AP-1 transactivation <br />                                                                           cell-based reporter assay system. SKOV-3X <br />                                                                           cells transfected AP-1 NF- B&#226;&#8364;&#8220;Luc con- <br />                                                                           struct containing NF- B AP-1&#226;&#8364;&#8220;response pro- <br />                                                                           moter driving expression luciferase <br />                                                                           gene. The cells cotransfected plasmid <br />                                                                           pcDNA&#226;&#8364;&#8220;wild-type hApe1 redox mutant <br />                                                                           (pcDNA-C65A) Renilla luciferase control re- <br />                                                                           porter vector pRL-CMV. After 24-h transfection <br />                                                                           period, cells lysed, Firefly Renilla lu- <br />                                                                           ciferase activities assayed using Renilla lu- <br />                                                                           ciferase activity normalization. All trans- <br />                                                                           fection experiments performed triplicate <br />                                                                           repeated 3 times independent ex- <br />                                                                           periments. Data expressed mean   standard <br />                                                                           error representative experiment, Stu- <br />                                                                           dent&#226;&#8364;&#8482;s t tests performed. *Significant differ- <br />                                                                           ence p   0.05 level comparing Cys65 mu- <br />                                                                           tant wt-Ape1 wt-Ape1 mock or <br />                                                                           pcDNA. <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                      1859 <br />  <br />  <br />  <br />  <br /> FIG. 5. Synthesis 3-[5-(2,3-dimethoxy-6- <br /> methyl-1,4-benzoquinoyl)]-2-nonyl-2-propi- <br /> onic acid (E3330) using novel approach. <br />  <br />  <br />  <br />  <br /> man Ape1 redox activity. The concentration E3330 re-         cancer cell lines, Hey-C2 SKOV-3X, pNF- <br /> quired inhibit 50% redox activity 6.5  M       B-Luc pAP-1-Luc constructs Renilla luciferase re- <br /> wild-type hApe1 5  M T58C zApe. Although            porter vector pRL-CMV control transfection. After <br /> conclusive, data demonstrate E3330 affect-    transfection, treated cells increasing amounts of <br /> ing cysteines Ape1 proteins       E3330. As observed Fig. 9, increasing amounts E3330 <br /> Cys65 human Ape1 primary target E3330      causes analogous decrease AP-1 or <br /> main cysteine involved redox function Ape1. This      NF- B activation, evidenced decrease amount <br /> was confirmed studies mutated       AP-1 NF- B activation luciferase reporter gene. <br /> cysteine human Ape1 individually, Cys65         These results indicate decreased binding proteins <br /> had major effect redox function Ape1 (data    blockage conversion oxidized to <br /> shown). Experiments progress demonstrate un-          reduced state Ape1 E3330 blocking Ape1 re- <br /> equivocally interaction E3330 Ape1 protein       dox function. <br /> the mechanism involved (Georgiadis Kelley, personal            Because E3330 does affect repair functions Ape1, <br /> communication, 2008).                                           shown Fig. 7, tested effect blocking Ape1 re- <br />   Further confirm activity E3330 Ape1 redox     dox function ovarian cancer cell lines. As shown in <br /> inhibitor cell-based assays, co-transfected ovarian   Fig. 10, E3330 causes decrease cell pro- <br />  <br />  <br />  <br />  <br /> FIG. 6. EMSA studies AP-1 <br /> or HIF-1    binding E3330 <br /> treatments. Increasing levels of <br /> E3330 decrease purified c-Jun <br /> binding vitro (A) SKOV- <br /> 3X ovarian cancer cells treated <br /> with E3330 (B). HIF-1  a <br /> downstream target redox <br /> function Ape1, inhibition <br /> of Ape1 redox E3330 blocks <br /> HIF-1  binding (C). (D) A West- <br /> ern blot analysis samples <br /> in (B) using Ape1 AP-1 <br /> antibodies, demonstrating that <br /> the level Ape1 AP-1 the <br /> cell extracts identical, and <br /> the results observed due <br /> to decrease respective <br /> proteins. <br />  1860                                                                                                                 LUO ET AL. <br />  <br />  <br />  <br />  <br /> FIG. 7. Oligonucleotide gel-based endonuclease BER assays. Analysis performed described Methods (20, <br /> 41). (A) One nanogram purified Ape1 protein used increasing E3330, indicated gel <br /> lanes. Ape1 protein extracts added reaction mixture containing 0.2 pmol HEX-labeled double-stranded tetrahy- <br /> drofuran oligonucleotide, 50 mM HEPES, 50 mM KCl, 10 mM MgCl2, 2 mM DTT, 1% BSA, 0.05% Triton X-100 (pH <br /> 7.5). Reactions E3330 incubated 15 min 37&#194;&#176;C. Reactions halted adding 10  l for- <br /> mamide. Ape1 assay products separated 20% polyacrylamide gel containing 7 M urea. The upper band (26-mer) <br /> represents uncleaved AP oligonucleotides, lower band (14-mer) reacted oligonucleotide. (B) BER path- <br /> way vitro analysis using gap-filling assay, described Singhal et al. (67). The gap-filling assay initiated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> a uracil oligonucleotide substrate, removed using uracil-glycosylase. If BER pathway fully func- <br /> tional, 48-mer band observed gel, 21-mer band indicates incomplete BER-pathway activity. E3330 <br /> was added reactions, indicated, effect pathway, indicating effect Ape1-repair function. <br />  <br />  <br /> liferation increasing amounts drug cell lines        simple highly reproducible method quantifying <br /> with IC50 33 37  M, respectively. We observed              cell proliferation used determine effect of <br /> similar results cancer cell lines (data shown).       E3330 proliferation rate RVECs. As shown Fig. <br /> These results indicate redox activity functions          11A, addition E3330 inhibited RVEC growth dose-de- <br /> Ape1 growth survival response cancer cells          pendent manner. Supplement bFGF basal media <br /> is independent DNA repair activity Ape1. These             significantly boosted proliferation rate. However, sim- <br /> findings confirm similar findings observed hemato-           ilar inhibition effect evident various doses of <br /> poietic model embryonic stem cells (82). Further mecha-           E3330 treatments. The difference E3330-treated <br /> nistic studies occurring currently       group corresponding vehicle control same <br /> way, preliminary studies indicate redox-           medium statistically significant (p   0.01). <br /> inhibitory effect necessarily cell killing,        An vitro angiogenesis assay used determine <br /> a block cell proliferation cytostatic effect similar   effect E3330 RVEC formation capillary-like struc- <br /> observed using Ape1 siRNA vivo (19, 20).                       tures Matrigel. The inhibition effect E3330 similar <br />                                                                      proliferation assay lower dose range <br />                                                                      (Fig. 11B). E3330 1  M sufficient elicit 30% re- <br /> E3330 inhibits RVEC proliferation and <br />                                                                      duction tube formation. Quantitative measurement the <br /> angiogenesis vitro <br />                                                                      number tube units revealed difference sta- <br />   Proliferation endothelial cells (ECs) important in-       tistically significant (p   0.05). The tube formation com- <br /> dex angiogenic ability EC vitro. The MTS assay         pletely abolished 5  M higher concentration. <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                                 1861 <br />  <br />                                                                     DNA-repair enzymes, including MGMT, polyADP-ribose <br />                                                                     polymerase (PARP1), ataxia&#226;&#8364;&#8220;telangiectasia mutated kinase <br />                                                                     (ATM kinase), Ape1, DNA PKcs varying degrees of <br />                                                                     success development (12, 21, 48). <br />                                                                        Toward goal developing inhibitors Ape1 po- <br />                                                                     tential cancer therapeutic, developing specific, in- <br />                                                                     dependent inhibitors of major functions Ape1 <br />                                                                     (redox repair), focused blocking Ape1 <br />                                                                     redox function. Although overwhelming evidence supports <br />                                                                     observation Cys65 main cysteine involved in <br />                                                                     redox role human Ape1, confirm this <br />                                                                     premise site-directed mutagenesis Cys65 site, <br />                                                                     leading lack NF- B AP-1 transactivation cell- <br />                                                                     model (Fig. 4) inhibition Ape1 redox us- <br />                                                                     ing small molecule&#226;&#8364;&#8220;specific inhibitor, E3330 (Figs. 6 9). <br />                                                                     Furthermore, demonstrate humanized zebrafish <br />                                                                     Ape1, previously demonstrated gained redox ac- <br />                                                                     tivity (25), inhibited E3330 (Fig. 8). In addition, the <br />                                                                     redox inhibition Ape1 effect repair function <br /> FIG. 8. Redox EMSA showing E3330 inhibition mutant               Ape1, demonstrated lack inhibition sep- <br /> zebrafish Ape1. An increasing E3330 incu-             arate DNA-repair assays (Fig. 7). E3330 able block <br /> bated 2  l purified hApe1(lanes 1&#226;&#8364;&#8220;7) mutant ze-             redox function Ape1 ovarian cancer cell lines <br /> brafish Ape1 (T58C-zApe1) (lanes 8&#226;&#8364;&#8220;13; reduced 1.0 <br />                                                                     using transactivation reporter assay dose-depen- <br /> mM DTT diluted 2  g/ l 0.2 mM DTT in <br /> PBS) EMSA reaction buffer [10 mM Tris (pH 7.5), 50 mM            dent manner (Fig. 9). These data support cellular role of <br /> NaCl, 1 mM MgCl2, 1 mM EDTA, 5% (vol/vol) glycerol]                 Ape1 redox activation downstream targets dem- <br /> with total volume 10  l 30 min, EMSA          onstrate E3330 small molecule block transcription-fac- <br /> assay performed. E3330 inhibited AP1-DNA bind-              tor activation; particularly transcription factors in- <br /> ing stimulated hApe1 T58C-zApe1 ex-              volved cancer cell growth proliferation, as <br /> tent. Lane 1 unreduced hApe1 control; lane 2 control      NF- B, AP-1, HIF-1 . These data line our <br /> DTT carryover, lane 13 unreduced T58C-zApe1 con-             previous findings demonstrating blocking redox, but <br /> trol. Lanes 3&#226;&#8364;&#8220;7 human Ape1 reacted increasing          repair function Ape1 led blocking cell proliferation <br /> amounts E3330. Lanes 8&#226;&#8364;&#8220;12 T58C zebrafish Ape1 re-           (82). Although previous studies demonstrated altering <br /> acted increasing amounts E3330.                             Ape1 levels leads blockage cell growth increased <br />                                                                     cancer cell sensitivity (2, 6, 16, 19, 20, 30, 33, 37, 43&#226;&#8364;&#8220;47, 51, <br />                                                                     59, 68, 70, 73&#226;&#8364;&#8220;75, 81), studies used overexpression of <br />                                                                     Ape1, Ape1 antisense oligonucleotides, Ape1 siRNA: The <br />    These results demonstrate inhibition redox activ- <br />                                                                     quandary approach, valid, of <br /> ity significantly attenuates RVEC proliferation capillary <br />                                                                     approaches changes total cellular content level of <br /> formation vitro. The capillary formation RVECs appears <br />                                                                     Ape1 removes Ape1 functions, just re- <br /> much sensitive redox inhibition. <br />                                                                     pair redox activities. Because Ape multiple functions <br />                                                                     protein&#226;&#8364;&#8220;protein interactions DNA re- <br /> Discussion <br />                                                                     pair signaling proteins, increase decrease Ape1 <br />    It long recognized DNA-damaging agents             protein result prejudiced inexact results. Use of <br /> are useful killing cancer cells mainstay    specific small-molecule inhibitors E3330 im- <br /> cancer treatments (31, 49). However, significant problem          portant delineate true role Ape1 various cancer, <br /> with success DNA-damaging agents upregula-            disease, normal cellular functions. Ultimately, using <br /> tion activation number different pathways con-      Ape1 redox inhibitors Ape1-specific endonuclease re- <br /> fer resistance cancer cells treatment DNA-       pair inhibitors clearer picture multiple ac- <br /> damaging agents. This includes pathways               tivites Ape1. <br /> involved signaling, multidrug resistance, cell-cycle check-         Toward goal developing specific Ape1 redox inhibi- <br /> points, antiangiogenesis, potential approaches        tion study role Ape1 normal cancer cells, we <br /> to treat kill cancer. Much work focused block-      performed single-agent dose&#226;&#8364;&#8220;response studies using E3330 <br /> ing ability cancer cell recognize repair        ovarian cancer cell lines determined E3330 does <br /> damaged DNA results front-line cancer treatments          single-agent cancer cell killing abilities (Fig. 10). This find- <br /> (chemotherapy radiation). One systems includes         ing restricted just ovarian cancer cell lines been <br /> DNA-repair enzymes, Ape1, leading resistance             observed cell lines representing colon, lung, breast, brain, <br /> and failure agent kill cancer cells (16, 21). These   pancreatic, prostate, multiple myeloma cancers (data not <br /> findings suggest possible strategy preventing re-      shown). In contrast, significant growth inhibi- <br /> sistance improving efficacy DNA-targeting agents         tion studies normal cells, hematopoietic em- <br /> would inhibit DNA-repair enzyme responsible           bryonic cells (82), RVEC (Fig. 11) E3330 inhibits RVEC cell <br /> resistance. This potential recognized, evidenced        growth (MTS assay results). Additionally, see <br /> by identification small-molecule inhibitors       cell killing human CD34  progenitor cells (unpublished <br />  FIG. 9. Effect E3330 Ape1 redox transactivation assay. Effect E3330 endogenous cellular Ape1 redox ac- <br /> tivity using functional transactivation assay described Methods Fig. 4. Increasing amounts E3330 were <br /> added Hey-C2 SKOV-3X ovarian cancer cell lines. At 24 h transfection, ratio firefly luciferase activ- <br /> ity Renilla luciferase activity determined measure AP-1 NF- B activity. Data expressed mean   SEM <br /> of independent experiments performed triplicate presented percentage transactivation compared with <br /> the normalized control E3330. <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                             1863 <br />  <br />  <br />  <br />  <br /> FIG. 10. Cell-growth assay using E3330 ovarian cancer cell lines. The MTT growth/survival assay per- <br /> formed described Methods. Increasing amounts E3330 resulted decreasing cell growth decreased cell num- <br /> bers. Data expressed mean   SEM independent experiments performed triplicate. <br />  <br />  <br />  <br /> data). These data novel implicate redox role     tion using E3330 blocks RVEC cell growth, with <br /> of Ape cancer, &#226;&#8364;&#339;normal&#226;&#8364;? cell survival. This impli-      bFGF stimulation (Fig. 11). Additional studies using RVEC <br /> cates redox role independent Ape1 DNA-repair            formation capillary-like structures Matrigel resulted in <br /> function independent addition DNA-damaging              decreased angiogenesis, measured assay, but <br /> agents. This supported recently published data           lower dose range MTS assays (Fig. 11). <br /> ovarian cancer studies xenografts, demonstrating          These results demonstrate inhibition redox activity <br /> knockdown Ape1 results blocking cell growth          significantly attenuates RVEC proliferation capillary for- <br /> proliferation, cell death (19). This time can-   mation vitro, does cause cell death. Furthermore, <br /> cer cell killing reported using small-molecule in-      capillary formation RVECs appears sen- <br /> hibitor Ape1 redox function.                                       sitive redox inhibition Ape1 proliferation. This <br />    One exciting results presented details        time role Ape1 clearly demon- <br /> potential role Ape1 angiogenesis (Fig. 11). As demon-           strated, necessitated use specific small-mole- <br /> strated RVEC studies, inhibition Ape1 redox func-           cule Ape1 redox inhibitor identify novel activity. <br />  <br />  <br />  <br />  <br /> FIG. 11. Inhibition redox activity Ape1 reduces retinal vascular endothelial cell angiogenesis vitro. (A) In the <br /> MTS proliferation assay, RVECs treated bFGF various doses E3330 (10&#226;&#8364;&#8220;100  M) the <br /> culture 2 days. A dose-dependent inhibition RVEC growth evident basal media (EBM) bFGF-sup- <br /> plemented (10 ng/ml) media (EBM   bFGF). The number viable cells E3330-treated groups significantly dif- <br /> ferent (p   0.01) corresponding vehicle control cells medium. (B) In tube-formation assay, RVECs <br /> cultured bFGF medium Matrigel treated various doses E3330 (1&#226;&#8364;&#8220;20  M) 1 day. RVEC formation of <br /> capillary-like structure significantly inhibited lower dose range. The 30% reduction tube units 1  M <br /> E3330 statistically different vehicle control (p   0.05). <br />  1864                                                                                                               LUO ET AL. <br />  <br />    As discussed, Ape1 originally identified pri-      tetrahydrofuran; TMZ, temozolomide; XRCC1, x-ray cross <br /> mary target E3330. E3330 immobilized beads,        complementing 1. <br /> Ape1 identified nuclear extract Jurkat cells <br /> as protein specifically binds E3330. By using sur- <br /> face plasmon resonance, kinetic constant (KD) 1.6 nM        References <br /> was obtained binding E3330 Ape1, suggesting <br /> a specific interaction. It shown E3330           1. Bhakat KK, Izumi T, Yang SH, Hazra TK, Mitra S. Role <br /> blocked ability Ape1 reduce NF- B, interfer-          acetylated human AP-endonuclease (APE1/Ref-1) reg- <br /> ing redox activity Ape1 (27, 35, 52, 65). Through        ulation parathyroid hormone gene. EMBO J 22: <br /> a series N- C-terminal truncation mutants, peptide          6299&#226;&#8364;&#8220;6309, 2003. <br />                                                                   2. Bobola MS, Finn LS, Ellenbogen RG, Geyer JR, Berger MS, <br /> including residues 72-88 Ape1 proposed bind to <br />                                                                      Braga JM, Meade EH, Gross ME, Silber JR. Apurinic/ <br /> E3330, suggesting E3330 interacts specific site <br />                                                                      apyrimidinic endonuclease activity associated re- <br /> on Ape1 (34, 54, 65). However, believe ap- <br />                                                                      sponse radiation chemotherapy medulloblastoma <br /> proach imperfect, truncated Ape1 molecules           primitive neuroectodermal tumors. Clin Cancer Res 11: <br /> are unlikely fold correctly, identification       7405&#226;&#8364;&#8220;7414, 2005. <br /> the binding site suspect. In addition, proposed bind-      3. Brimble MA, McEwan JF, Turner P. Asymmetric Diels- <br /> ing site somewhat puzzling light structure           Alder addition cyclopentadiene chiral naphtho- <br /> Ape1, obvious binding site noted (data              quinones tetrahedron. Asymmetry 9: 1239&#226;&#8364;&#8220;1255, 1998. <br /> shown). These residues (aa 72&#226;&#8364;&#8220;80) form ridge sur-        4. Campalans A, Marsin S, Nakabeppu Y, O&#226;&#8364;&#8482;Connor TR, Boi- <br /> face molecule obvious cavities binding             teux S, Radicella JP. XRCC1 interactions multiple <br /> pockets sufficiently large bind E3330. Currently,        DNA glycosylases: model recruitment base exci- <br /> we pursuing studies delineate binding site re-         sion repair. DNA Repair (Amst) 4: 826&#226;&#8364;&#8220;835, 2005. <br /> gion E3330 Ape1.                                            5. Chakravarti A, Delaney MA, Noll E, Black PM, Loeffler JS, <br />    In conclusion, Ape1 multifunctional protein        Muzikansky A, Dyson NJ. Prognostic pathologic <br /> important DNA-repair redox capabilities. However,             significance quantitative protein expression profiling in <br /> delineate various functions Ape1, small-molecule in-          human gliomas. Clin Cancer Res 7: 2387&#226;&#8364;&#8220;2395, 2001. <br /> hibitors function necessary ultimately con-    6. Chen DS Olkowski ZL. Biological responses human <br /> clude function required normal cancer cell           apurinic endonuclease radiation-induced DNA damage. <br /> function. Additionally, data demonstrate              Ann NY Acad Sci 726: 306&#226;&#8364;&#8220;308, 1994. <br /> new role Ape1 involvement angiogenesis,        7. Chou KM Cheng YC. The exonuclease activity hu- <br /> subsequent inhibition Ape1s redox function abrogates         man apurinic/apyrimidinic endonuclease (APE1): biochem- <br />                                                                      ical properties inhibition natural dinucleotide <br /> role. The use E3330 new analogues currently under <br />                                                                      Gp4G. J Biol Chem 278: 18289&#226;&#8364;&#8220;18296, 2003. <br /> development critical reagents future studies of <br />                                                                   8. Chou KM Cheng YC. An exonucleolytic activity hu- <br /> the cellular functions Ape1 potential therapeu- <br />                                                                      man apurinic/apyrimidinic endonuclease 3  mispaired <br /> tic applications.                                                    DNA. Nature 415: 655&#226;&#8364;&#8220;659, 2002. <br />                                                                   9. Chou KM, Kukhanova M, Cheng YC. A novel action of <br /> Acknowledgments                                                      human apurinic/apyrimidinic endonuclease: excision L- <br />    Financial support work provided               configuration deoxyribonucleoside analogs 3  ter- <br /> National Institutes Health, National Cancer Institute             mini DNA. J Biol Chem 275: 31009&#226;&#8364;&#8220;31015, 2000. <br /> CA94025, CA106298, CA114571, CA121168 M.R.K. ,            10. Chou KM, Lam W, Cheng YC. A novel DNA exonu- <br />                                                                      clease activity human apurinic/apyrimidinic endonucle- <br /> CA 114571 M.M.G, IU Simon Cancer Center Translational <br />                                                                      ase (APE1): implication DNA mismatch repair. Proc Am <br /> initiative pilot funding (ITRAC) M.R.K. M.M.G., and <br />                                                                      Assoc Ca Res 42: 2988, 2001. <br /> the Riley Children&#226;&#8364;&#8482;s Foundation (M.R.K.). R.L.N. sup- <br />                                                                  11. Chung U, Igarashi T, Nishishita T, Iwanari H, Iwamatsu A, <br /> ported training grant 5T32CA009634.                               Suwa A, Mimori T, Hata K, Ebisu S, Ogata E, Fujita T, and <br />                                                                      Okazaki T. The interaction Ku antigen REF1 <br /> Abbreviations                                                        protein mediates negative gene regulation extracellular <br />    AP, apurinic/apyrimidinic; Ape1, AP endonuclease1;                calcium. J Biol Chem 271: 8593&#226;&#8364;&#8220;8598, 1996. <br /> ATM, ataxia&#226;&#8364;&#8220;telangiectasia mutated; BER, base excision re-       12. Damia G D&#226;&#8364;&#8482;Incalci M. Targeting DNA repair a <br />                                                                      promising approach cancer therapy. Eur J Cancer 43: 1791&#226;&#8364;&#8220; <br /> pair; CSB, Cockayne syndrome B; EC, endothelial cell; <br />                                                                      1801, 2007. <br /> EMSA, electrophoretic mobility shift assay; FEN1, flap en- <br />                                                                  13. Demple B Harrison L. Repair oxidative damage to <br /> donuclease 1; GZMA, granzyme A; HEX, 5  hexachloro- <br />                                                                      DNA: enzymology biology. Annu Rev Biochem 63: <br /> fluorescein phosphoramidite; IR, incision repair; ITRAC,             915&#226;&#8364;&#8220;948, 1994. <br /> IU Simon Cancer Center Translational Research Accelera-          14. Dianova II, Bohr VA, Dianov GL. Interaction human <br /> tion Collaboration; MMS, methylmethane sulfonate; NIR,               AP endonuclease 1 flap endonuclease 1 prolifer- <br /> nucleotide incision repair; NK, natural killer; NMR, nu-             ating cell nuclear antigen involved long-patch base exci- <br /> clear magnetic resonance; NO, nitric oxide; PARP1,                   sion repair. Biochemistry 40: 12639&#226;&#8364;&#8220;12644, 2001. <br /> polyADP-ribose polymerase 1; PCNA, proliferating cell            15. DiGiuseppe JA, Hunting DJ, Dresler SL. Aphidicolin- <br /> nuclear antigen; PTEN, phosphatase tensin homo-                  sensitive DNA repair synthesis human fibroblasts dam- <br /> logue; REC, retinal endothelial cell; redox, reduction oxy-          aged bleomycin distinct UV-induced repair. <br /> genation; RVEC, retinal vascular endothelial cell; THF,              Carcinogen Carcinogenesis 11: 1021&#226;&#8364;&#8220;1026, 1990. <br />  REDOX ROLE OF APE1 IN CANCER AND ENDOTHELIAL CELLS                                                                               1865 <br />  <br /> 16. Evans AR, Limp-Foster M, Kelley MR. Going APE         32. Herring CJ, Deans B, Elder RH, Rafferty JA, MacKinnon J, <br />     ref-1. Mutat Res 461: 83&#226;&#8364;&#8220;108, 2000.                                Barzilay G, Hickson ID, Hendry JH, Margison GP. Ex- <br /> 17. Fan Z, Beresford PJ, Zhang D, Xu Z, Novina CD, Yoshida             pression levels DNA repair enzyme HAP1 cor- <br />     A, Pommier Y, Lieberman J. Cleaving oxidative re-          relate radiosensitivities human HAP1-trans- <br />     pair protein Ape1 enhances cell death mediated                  fected rat cell lines. Br J Cancer 80: 940&#226;&#8364;&#8220;945, 1999. <br />     granzyme A. Nat Immunol 4: 145&#226;&#8364;&#8220;153, 2003.                      33. Herring CJ, West CM, Wilks DP, Davidson SE, Hunter RD, <br /> 18. Fantini D, Vacotto C, Deganuto M, Bivi N, Gustincich S,            Berry P, Forster G, MacKinnon J, Rafferty JA, Elder RH, <br />     Marcon G, Quadrioglio FDG, Bhakat K, Mitra S.                  Hendry JH, Margison GP. Levels DNA repair <br />     APE1/Ref-1 regulates PTEN expression mediated Egr-1.            enzyme human apurinic/apyrimidinic endonuclease <br />     Free Radic Res 42: 20&#226;&#8364;&#8220;29, 2008.                                    (APE1, APEX, Ref-1) associated intrinsic ra- <br /> 19. Fishel ML, He Y, Reed AM, Chin-Sinex H, Hutchins GD,               diosensitivity cervical cancers. Br J Cancer 78: 1128&#226;&#8364;&#8220;1133, <br />     Mendonca MS, Kelley MR. Knockdown DNA re-               1998. <br />     pair redox signaling protein Ape1/Ref-1 blocks ovarian     34. Hiramoto M, Shimizu N, Nishi T, Shima D, Aizawa S, <br />     cancer cell tumor growth. DNA Repair (Amst) 7:                 Tanaka H, Hatakeyama M, Kawaguchi H, Handa H. <br />     177&#226;&#8364;&#8220;186, 2008.                                                     High-performance affinity beads identifying anti-NF- <br /> 20. Fishel ML, He Y, Smith ML, Kelley MR. Manipulation             kappa B drug receptors. Methods Enzymol 353: 81&#226;&#8364;&#8220;88, 2002. <br />     base excision repair sensitize ovarian cancer cells   35. Hiramoto M, Shimizu N, Sugimoto K, Tang J, Kawakami Y, <br />     alkylating agent temozolomide. Clin Cancer Res 13: 260&#226;&#8364;&#8220;267,        Ito M, Aizawa S, Tanaka H, Makino I, Handa H. Nu- <br />     2007.                                                              clear targeted suppression NF-kappa B activity the <br /> 21. Fishel ML Kelley MR. The DNA base excision repair              novel quinone derivative E3330. J Immunol 160: 810&#226;&#8364;&#8220;819, <br />     protein Ape1/Ref-1 therapeutic chemopreventive            1998. <br />     target. Mol Aspects Med 28: 375&#226;&#8364;&#8220;395, 2007.                     36. Hsieh MM, Hegde V, Kelley MR, Deutsch WA. Activa- <br /> 22. Flader C, Liu J, Borch RF. Development novel qui-           tion APE/Ref-1 redox activity mediated reactive <br />     phosphorodiamidate prodrugs targeted DT-di-                oxygen species PKC phosphorylation. Nucleic Acids Res <br />     aphorase. J Med Chem 43: 3157&#226;&#8364;&#8220;3167, 2000.                          29: 3116&#226;&#8364;&#8220;3122, 2001. <br /> 23. Flader C, Liu J, Borch RF. Development novel qui-       37. Ide H Kotera M. Human DNA glycosylases involved in <br />     phosphorodiamidate prodrugs targeted DT-di-                repair oxidatively damaged DNA. Biol Pharm Bull 27: <br />     aphorase. J Med Chem 43: 3157&#226;&#8364;&#8220;3167, 2000.                          480&#226;&#8364;&#8220;485, 2004. <br /> 24. Fung H Demple B. A vital role Ape1/Ref1 protein        38. Jeon BH, Gupta G, Park YC, Qi B, Haile A, Khanday FA, Liu <br />     repairing spontaneous DNA damage human cells. Mol            YX, Kim JM, Ozaki M, White AR, Berkowitz DE, Irani <br />     Cell 17: 463&#226;&#8364;&#8220;470, 2005.                                            K. Apurinic/apyrmidinic endonuclease 1 regulates endo- <br /> 25. Georgiadis M, Luo M, Gaur R, Delaplane S, Li X, Kel-           thelial NO production vascular tone. Circ Res 95: <br />     ley M. Evolution redox function mammalian                902&#226;&#8364;&#8220;910, 2004. <br />     apurinic/apyrimidinic. Mutat Res 2008.                         39. Jin Z, May WS, Gao F, Flagg T, Deng X. Bcl2 suppresses <br /> 26. Gorman MA, Morera S, Rothwell DG, la Fortelle E, Mol            DNA repair enhancing c-Myc transcriptional activity. J <br />     CD, Tainer JA, Hickson ID, Freemont PS. The crystal            Biol Chem 281: 14446&#226;&#8364;&#8220;14456, 2006. <br />     structure human DNA repair endonuclease HAP1            40. Kelley MR, Hansen W, Xu Y, Yacoub A, Williams D, and <br />     suggests recognition extra-helical deoxyribose DNA       Deutsch WA. Combining O6-methylguanine DNA methyl- <br />     abasic sites. EMBO J 16: 6548&#226;&#8364;&#8220;6558, 1997.                          transferase (MGMT) AP endonuclease (APE) DNA re- <br /> 27. Goto M, Yamada K, Katayama K, Tanaka I. Inhibitory             pair activities chimeric proteins: use retroviral gene <br />     effect E3330, novel quinone derivative able sup-           therapy chemotherapeutic dose intensification. Proc Am <br />     press tumor necrosis factor-alpha generation, activa-           Assoc Cancer Res 38: 383, 1997. <br />     tion nuclear factor-kappa B. Mol Pharmacol 49: 860&#226;&#8364;&#8220;873,     41. Kreklau EL, Limp-Foster M, Liu N, Xu Y, Kelley MR, and <br />     1996.                                                              Erickson LC. A novel fluorometric oligonucleotide assay to <br /> 28. Hansen CA, Dean AB, Draths KM, Frost JW. Synthesis             measure O(6)-methylguanine DNA methyltransferase, <br />     1,2,3,4-tetrahydroxybenzene D-glucose: exploiting          methylpurine DNA glycosylase, 8-oxoguanine DNA glyco- <br />     myo-inositol precursor aromatic chemicals. J Am            sylase abasic endonuclease activities: DNA repair sta- <br />     Chem Soc 121: 3799&#226;&#8364;&#8220;3800, 1999.                                     tus human breast carcinoma cells overexpressing <br /> 29. Hansen WK, Deutsch WA, Yacoub A, Xu Y, Williams DA,                methylpurine DNA glycosylase. Nucleic Acids Res 29: 2558&#226;&#8364;&#8220; <br />     Kelley MR. Creation fully functional human chi-           2566, 2001. <br />     meric DNA repair protein: combining O6-methylguanine           42. Kuninger DT, Izumi T, Papaconstantinou J, Mitra S. Hu- <br />     DNA methyltransferase (MGMT) AP endonuclease                   man AP-endonuclease 1 hnRNP-L interact a <br />     [APE/redox effector factor 1 (Ref 1)] DNA repair proteins.         nCaRE-like repressor element AP-endonuclease 1 pro- <br />     J Biol Chem 273: 756&#226;&#8364;&#8220;762, 1998.                                    moter. Nucleic Acids Res 30: 823&#226;&#8364;&#8220;829, 2002. <br /> 30. Harrison JF, Rinne ML, Kelley MR, Druzhyna NM, Wilson          43. Lau JP, Weatherdon KL, Skalski V, Hedley DW. Effects <br />     GL, LeDoux SP. Altering DNA base excision repair: use          gemcitabine APE/ref-1 endonuclease activity pan- <br />     nuclear mitochondrial-targeted N-methylpurine               creatic cancer cells, therapeutic potential antisense <br />     DNA glycosylase sensitize astroglia chemotherapeutic         oligonucleotides. Br J Cancer 91: 1166&#226;&#8364;&#8220;1173, 2004. <br />     agents. Glia 55: 1416&#226;&#8364;&#8220;1425, 2007.                              44. Liu L Gerson SL. Therapeutic impact methoxyamine: <br /> 31. Havelka AM, Berndtsson M, Olofsson MH, Shoshan MC,                 blocking repair abasic sites base excision repair <br />     Linder S. Mechanisms action DNA-damaging an-             pathway. Curr Opin Invest Drugs 5: 623&#226;&#8364;&#8220;627, 2004. <br />     ticancer drugs treatment carcinomas: acute apopto-    45. Liu L, Nakatsuru Y, Gerson SL. Base excision repair as <br />     sis &#226;&#8364;&#339;off-target&#226;&#8364;? effect? Mini Rev Med Chem 7: 1035&#226;&#8364;&#8220;1039,        therapeutic target colon cancer. Clin Cancer Res 8: <br />     2007.                                                              2985&#226;&#8364;&#8220;2991, 2002. <br />  1866                                                                                                                     LUO ET AL. <br />  <br /> 46. Liu L, Yan L, Donze JR, Gerson SL. Blockage abasic        63. Peddi SR, Chattopadhyay R, Naidu CV, Izumi T. The <br />     site repair enhances antitumor efficacy 1,3-bis-(2-               human apurinic/apyrimidinic endonuclease-1 suppresses <br />     chloroethyl)-1-nitrosourea colon tumor xenografts. Mol            activation poly(adp-ribose) polymerase-1 induced by <br />     Cancer Ther 2: 1061&#226;&#8364;&#8220;1066, 2003.                                      DNA single strand breaks. Toxicology 224: 44&#226;&#8364;&#8220;55, 2006. <br /> 47. Liu L, Yan L, Mahajan V, Donze JR, Gerson SL. Inhibition     64. Ranalli TA, Tom S, Bambara RA. AP endonuclease 1 co- <br />     AP site repair coupled action topoisomerase II poi-       ordinates flap endonuclease 1 DNA ligase I activity in <br />     son: potential strategy enhance efficacy chemothera-         long patch base excision repair. J Biol Chem 277: 41715&#226;&#8364;&#8220;41724, <br />     peutic alkylating agents. Proc Am Assoc Ca Res 44, 2003.             2002. <br /> 48. Madhusudan S Hickson I. DNA repair inhibition: se-         65. Shimizu N, Sugimoto K, Tang J, Nishi T, Sato I, Hiramoto <br />     lective tumour targeting strategy. Trends Mol Med 11:                M, Aizawa S, Hatakeyama M, Ohba R, Hatori H, Yoshikawa <br />     503&#226;&#8364;&#8220;511, 2005.                                                       T, Suzuki F, Oomori A, Tanaka H, Kawaguchi H, Watanabe <br /> 49. Martin F Solary E. Tumor cell resistance DNA-dam-             H, Handa H. High-performance affinity beads iden- <br />     aging agents: apoptosis neiosis. Curr Med Chem An-           tifying drug receptors. Nat Biotechnol 18: 877&#226;&#8364;&#8220;881, 2000. <br />     ticancer Agents 4: 461&#226;&#8364;&#8220;463, 2004.                                66. Sidorenko VS, Nevinsky GA, Zharkov DO. Mechanism <br /> 50. Martinvalet D, Zhu P, Lieberman J. Granzyme induces            interaction human 8-oxoguanine-DNA glycosy- <br />     caspase-independent mitochondrial damage, required           lase AP endonuclease. DNA Repair 6: 317&#226;&#8364;&#8220;328, 2007. <br />     step apoptosis. Immunity 22: 355&#226;&#8364;&#8220;370, 2005.                  67. Singhal RK, Prasad R, Wilson SH. DNA polymerase beta <br /> 51. McNeill DR Wilson DM 3rd. A dominant-negative form               conducts gap-filling step uracil-initiated base excision <br />     major human abasic endonuclease enhances cellular             repair bovine testis nuclear extract. J Biol Chem 270: <br />     sensitivity laboratory clinical DNA-damaging agents.          949&#226;&#8364;&#8220;957, 1995. <br />     Mol Cancer Res 5: 61&#226;&#8364;&#8220;70, 2007.                                   68. Spitz DR, Azzam EI, Li JJ, Gius D. Metabolic oxida- <br /> 52. Miyamoto K, Nagakawa J, Hishinuma I, Hirota K, Yasuda                tion/reduction reactions cellular responses ionizing <br />     M, Yamanaka T, Katayama K, Yamatsu I. Suppressive                radiation: unifying concept stress response biology. Can- <br />     effects E3330, novel quinone derivative, tumor ne-           cer Metastasis Rev 23: 311&#226;&#8364;&#8220;322, 2004. <br />     crosis factor-alpha generation monocytes macro-         69. Su X, Sorenson CM, Sheibani N. Isolation charac- <br />     phages. Agents Actions 37: 297&#226;&#8364;&#8220;304, 1992.                            terization murine retinal endothelial cells. Mol Vis 9: <br /> 53. Nazarkina ZK, Khodyreva SN, Marsin S, Lavrik OI,                 171&#226;&#8364;&#8220;178, 2003. <br />     Radicella JP. XRCC1 interactions base excision repair       70. Taverna P, Liu L, Hwang HS, Hanson AJ, Kinsella TJ, and <br />     DNA intermediates. DNA Repair (Amst) 6: 254&#226;&#8364;&#8220;264, 2007.               Gerson SL. Methoxyamine potentiates DNA single strand <br /> 54. Nishi T, Shimizu N, Hiramoto M, Sato I, Yamaguchi Y,                 breaks double strand breaks induced temozolomide <br />     Hasegawa M, Aizawa S, Tanaka H, Kataoka K, Watanabe                  colon cancer cells. Mutat Res 485: 2 <br /> </body></html>