<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Cell Biol. 2009 December ; 11(12): 1444. doi:10.1038/ncb1993. <br />  <br />  <br />  <br />                            YAP-dependent induction amphiregulin identifies non-cell- <br />                            autonomous component Hippo pathway <br />  <br />                            Jianmin Zhang1, Jun-Yuan Ji1, Min Yu1,2, Michael Overholtzer3, Gromoslaw A. Smolen1, <br />                            Rebecca Wang1, Joan S. Brugge3, Nicholas J. Dyson1, Daniel A. Haber1,2,* <br />                            1 Massachusetts General Hospital Cancer Center, Harvard Medical School, Charlestown, MA 02129 <br />  <br />                            2   Howard Hughes Medical Institute, Harvard Medical School, Boston MA 02115 <br />                            3   Department Cell Biology, Harvard Medical School, Boston MA 02115 <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  The Hippo signaling pathway regulates cellular proliferation <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>survival</span>, exerting profound <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  effects normal cell fate tumorigenesis1-3. The pivotal effector pathway YAP, a <br />                                  transcriptional co-activator amplified mouse human cancers, promotes epithelial <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                  mesenchymal transition (EMT) malignant transformation4-10. To date, studies YAP target <br />                                  genes focused cell-autonomous mediators; YAP-expressing MCF10A <br />                                  breast epithelial cells enhance proliferation neighboring untransfected cells, implicating non- <br />                                  cell autonomous mechanism. We identify epidermal growth factor receptor (EGFR) ligand, <br />                                  amphiregulin (AREG), transcriptional target YAP, induction contributes YAP- <br />                                  mediated cell proliferation migration, EMT. Knockdown AREG addition an <br />                                  EGFR kinase inhibitor abrogates proliferative effects YAP expression. Suppression the <br />                                  negative YAP regulators LATS1/2 sufficient induce AREG expression, consistent with <br />                                  physiological regulation AREG Hippo pathway. Genetic interaction the <br />                                  Drosophila YAP orthologue Yorkie Egfr signaling components support link these <br />                                  highly conserved signaling pathways. Thus, YAP-dependent secretion AREG implicates <br />                                  activation EGFR signaling important non-cell autonomous effector Hippo pathway, <br />                                  implications regulation physiological malignant cell proliferation. <br />  <br />                                             Normal cells require mitogenic growth signals proliferate, whereas, tumor cells often <br />                                             generate proliferative signals secretion growth factors activation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             growth factor receptors 11. We shown YAP transduced MCF10A cells proliferate <br />                                             3D acinar cultures absence EGF 7. MCF10A immortalized, non transformed <br />                                             human mammary epithelial cells, exhibit dependence growth factors proliferation <br />                                             survival 12, raising possibility YAP itself induces secretion required growth <br />                                             factors cytokines cells. To test hypothesis, performed mixing experiments <br />                                             cells transduced GFP-labeled YAP Red-Cherry tagged vector. MCF10A <br />                                             cells expressing GFP-YAP, Cherry-vector, formed acini 3D cultures absence <br />                                             exogenous EGF. Remarkably, vector transduced cells did produce acini co-cultured in <br />                                             1:1 ratio YAP expressing cells (Fig. 1a). Thus, ectopic expression YAP MCF10A <br />                                             cells appears result secretion factors enable proliferation neighboring <br />                                             untransduced cells. <br />  <br />  <br />  <br />  <br />                            *To correspondence addressed: MGH Cancer Center, Bldg. 149, 13th Street, Charlestown, MA 02129; Tel (617) <br />                            726-7805; Fax: (617) 724-6919; haber@helix.mgh.harvard.edu. <br />                             Zhang et al.                                                                                          Page 2 <br />  <br />  <br />                                           To identify mediators apparent YAP-induced non-cell-autonomous effect, use <br />                                           constitutively active YAP mutant, YAP-S127A. <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>Mutation</span> serine residue targeted <br />                                           phosphorylation negative regulatory kinases LATS1/2 prevents cytoplasmic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           sequestration YAP 14-3-3 proteins 5,10, resulting exclusively nuclear <br />                                           localization (Fig. S1a). Retroviral vectors containing wild type YAP YAP-S127A induced <br />                                           similar levels protein expression (Fig. 1b). Ectopic expression YAP-S127A induced a <br />                                           strong EMT cell migration <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. S1b S1c), robustly promoted EGF- <br />                                           independent acinar growth MCF10A 3D-cultures (Fig. 1b). The enhanced YAP phenotype <br />                                           displayed YAP-S127A led increased effect co-culturing experiments, with <br />                                           YAP-S127A expressing MCF10A cells, dramatically stimulating acini formation by <br />                                           neighboring vector-transfected cells (Fig. 1a). To test YAP-expressing cells <br />                                           release growth-inducing factors, collected conditioned medium 3D cultures and <br />                                           applied parental MCF10A cells 3D culture assays. Conditioned media derived <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           cells transduced wild type YAP (YAP-Wt), greater extent YAP-S127A <br />                                           vector, enabled MCF10A acinus formation absence EGF (Fig. 1c), supporting <br />                                           existence diffusible factor mediating non-cell autonomous effect. <br />  <br />                                           To identify putative factors secreted YAP-expressing MCF10A cells, applied <br />                                           conditioned media Cytokine Antibody Array (RayBiotech&#226;&#8222;&#162;) representing 41 growth <br />                                           factors cytokines 13. When media collected cells grown standard 3D growth <br />                                           conditions (including EGF supplementation, allowing acinus formation parental MCF10A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           cells), significant differences observed vector YAP-S127A transduced <br />                                           cells (Fig. 2a). However, conditioned media collected cultures absence of <br />                                           EGF supplementation revealed proteins highly enriched (3-fold over <br />                                           background) YAP-S127A transduced cells: amphiregulin (AREG), insulin-like growth <br />                                           factor binding protein -6 (IGFBP-6), platelet-derived growth factor -AA (PDGF-AA) and <br />                                           macrophage colony-stimulating factor-receptor (M-CSF-R) (Fig. 2a). To test these <br />                                           proteins specifically induced YAP, byproducts 3D acini formation. We <br />                                           analyzed vector versus YAP-S127A expressing MCF10A cells grown 2D monolayer <br />                                           cultures following withdrawal EGF (12 hrs). We observed AREG dramatically <br />                                           induced YAP-S127A cells determined immunoblotting analysis (Fig. 2b). AREG thus <br />                                           constitutes primary candidate YAP-induced secreted factor involved cellular <br />                                           proliferation. <br />  <br />                                           To test AREG transcriptional target YAP, measured AREG mRNA in <br />                                           YAP transduced cells. A five-fold increase AREG transcript observed YAP- <br />                                           S127A transduced cells cultured absence EGF (Fig. 2c). Binding YAP AREG <br />                                           promoter vivo demonstrated chromatin immunoprecipitation (ChIP) assays. Anti- <br />                                           YAP- immunoprecipited chromatin yielded strong reproducible PCR amplification for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           fragment AREG promoter (Fig. 2d), comparable observed CTGF <br />                                           promoter, known YAP target gene 9,14. <br />  <br />                                           Given identification AREG transcriptional target YAP, undertook series of <br />                                           experiments test functional consequences interaction. We tested whether <br />                                           neutralizing antibodies AREG suppressed YAP-mediated 3D acini formation. Addition <br />                                           1&#206;&#188;g/ml anti-AREG IgG suppressed acini formation YAP-S127A cells 90%, whereas <br />                                           blocking antibodies IGFBP6, PDGF-AA M-CSF-R effect (Fig. 2e). AREG <br />                                           contributes YAP-mediated proliferation assay. To test AREG alone <br />                                           sufficient mediate 3D growth MCF10A cells, added recombinant AREG to <br />                                           cultures parental MCF10A cells. A dose-dependent effect AREG evident, equivalent <br />                                           EGF generating 3D acini (Fig. 3a). <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                            Page 3 <br />  <br />  <br />                                           To determine YAP-induced secretion AREG associated increased ErbB <br />                                           receptor family signaling, applied lysate YAP-S127A transduced cells EGFR <br />                                           Phosphorylation Antibody Array (RayBiotech&#226;&#8222;&#162;). In presence exogenous EGF, 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ErbB phosphorylation sites interrogated array comparable vector and <br />                                           YAP-S127A transduced cells (Fig. 3b). However, absence EGF, cellular lysates from <br />                                           YAP-S127A transduced cells displayed significantly increased phosphorylation classical <br />                                           EGFR target residues, EGFR-Y845, Y1068 Y1148, select residues within <br />                                           ErbB-2, ErbB-3 ErbB-4 (Fig. 3b). Of note, EGFR activation AREG cell types <br />                                           reported involve autocrine loop culminating EGFR-dependent AREG <br />                                           induction 15. To exclude mechanism, measured AREG levels presence or <br />                                           absence erlotinib, specific kinase inhibitor EGFR. Efficient inhibition EGFR <br />                                           signaling effect YAP-mediated AREG induction (Fig. 3c 3d). Thus, AREG <br />                                           induction YAP appears result direct transcriptional mechanism from <br />                                           EGFR-dependent feedback loop. Erlotinib treatment completely abrogated 3D acini <br />                                           formation YAP-S127A transduced cells (Fig. 3e), supporting key role EGFR <br />                                           signaling YAP-mediated proliferation. <br />  <br />                                           Ectopic overexpression YAP mimics effect gene amplification cancer. However, <br />                                           model physiological activation Hippo pathway, use ACHN kidney cancer <br />                                           cells, homozygous deletion upstream negative regulator Salvador, leads to <br />                                           elevated endogenous YAP activity 5,10,16. Knockdown YAP cells led significant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           reduction baseline AREG expression (Fig. 3f). YAP activity inhibited upstream <br />                                           kinases LATS1/2, mediate cytoplasmic retention YAP 5,6,10. To test whether <br />                                           physiological activation YAP abrogation LATS1/2 leads induction AREG, <br />                                           knocked LATS1/2 transcripts MCF10A cells. Dramatic induction AREG <br />                                           expression evident following suppression LATS1/2 cells (Fig. 3h). Taken <br />                                           together, observations demonstrate regulation endogenous AREG expression YAP <br />                                           induction disruption multiple levels Hippo pathway. <br />  <br />                                           To test contribution AREG induction various YAP-mediated phenotypes, made <br />                                           use lentiviral shRNA-mediated knockdown constructs targeting AREG itself. In addition to <br />                                           3D acini formation, YAP-dependent phenotypes MCF10A cells include induction of <br />                                           EMT increased cellular migration. Efficient knockdown baseline YAP-S127A <br />                                           induced AREG expression (both precursor cleaved AREG) accomplished using two <br />                                           independent AREG-targeting constructs (Fig. 4a). YAP-induced phosphorylation the <br />                                           downstream signaling molecules AKT ERK effectively suppressed AREG <br />                                           knockdown, demonstrating disruption autocrine pathway cells (Fig. 4a). As <br />                                           expected, AREG-targeting constructs dramatically inhibited EGF-independent 3D <br />                                           acini formation (Fig. 4b). In addition, suppression AREG dramatically inhibited cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           migration induced YAP-Wt YAP-S127A (Fig. 4c). In contrast, knockdown of <br />                                           AREG effect expression EMT-related markers (Fig. 4d), suggesting AREG <br />                                           contribute distinct YAP-dependent effect. <br />  <br />                                           Connective tissue growth factor (CTGF) recently identified transcriptional target <br />                                           YAP transcription factor TEAD, contributing YAP-mediated EMT 14. <br />                                           To test CTGF plays role YAP-dependent non-cell-autonomous effect studied <br />                                           here, targeted CTGF expression YAP-transduced MCF10A cells using independent <br />                                           lentiviral shRNA constructs. Efficient knockdown CTGF accomplished, no <br />                                           effect YAP-mediated acini formation (Fig. S 2a 2b). Therefore, CTGF AREG may <br />                                           serve complementary effectors YAP, mediating distinct downstream phenotypes. <br />  <br />                                           The complex Hippo pathway shows high degree evolutionary conservation from <br />                                           Drosophila, uncovered mammals. Despite fact components the <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                                 Page 4 <br />  <br />  <br />                                           pathway leading activation mammalian YAP Drosophila yki highly conserved <br />                                           1,2, downstream effectors yki identified date Drosophila screens do <br />                                           appear similarly regulated mammalian cells 5,9. We asked the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           relevant components EGFR pathway interact genetically core factors the <br />                                           Drosophila Hippo pathway. <br />  <br />                                           EGFR signaling Drosophila involves receptor (Egfr) Egfr ligands: Spitz, Keren, <br />                                           Gurken Vein 17. To test genetic interactions, crossed flies carrying mutations <br />                                           EGFR, EGFR ligands, EGFR processing proteins 17 flies harboring mutations in <br />                                           Hippo pathway, including warts (wts)and Hippo (Hpo), negative regulators yki, which <br />                                           over-expressed control eye-specific promoter (GMR). Consistent a <br />                                           previous report 16, GMR-wts eyes normal appearance (Fig. 5b) compared wild- <br />                                           type (Fig 5a). A rough eye phenotype evident GMR-wts combined a <br />                                           heterozygous loss-of-function mutation Egfr itself does phenotype (Fig. <br />                                           5c). A similar synthetic interaction observed GMR-wts combined with <br />                                           loss-of-function alleles Egfr (Table S1), mutant allele vein, an <br />                                           EGFR ligand (Fig. 5d). In contrast, mutant alleles EGFR ligands, spitz, <br />                                           keren,and gurken, produced synergistic effect (Table S1). We carried second set of <br />                                           genetic tests combining GMR-hpo EGFR pathway components. Unlike GMR-wts, the <br />                                           eye-specific overexpression Hippo (hpo) generates small, rough eye phenotype, which <br />                                           regular ommatidial organization disrupted individual ommatidia be <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           distinguished 18 (Fig. 5e). When combined heterozygous mutations Egfr (Fig. 5f) or <br />                                           vein (Fig. 5g), ommatidial organization severely disrupted: eyes glassy <br />                                           appearance fused ommatidia apparent (Fig. 5f, g), suggesting mutant alleles of <br />                                           Egfr vein dominantly enhanced GMR-hpo rough-eye phenotype. In contrast, mutant <br />                                           alleles spitz, keren, gurken hadnosucheffect (Table S1). In set genetic analyses, <br />                                           used previously reported GMR-yki phenotype 19 (Fig. S 3b), causes bulgy, <br />                                           rough eye phenotype (GMR-yki). This yki overgrowth phenotype partially suppressed by <br />                                           heterozygous loss-of-function alleles Egfr (Fig. S 3c). Again, partial suppression this <br />                                           phenotype observed mutant alleles vein, EGFR ligands, <br />                                           i.e., spitz, keren, gurken (Table S1). Taken together, genetic crosses indicate that <br />                                           increased activity Drosophila YAP ortholog yki partially suppressed Egfr <br />                                           pathway mutants, reduced activity yki pathway resulting overexpression of <br />                                           negative regulators wts hpo enhanced loss function Egfr pathway. <br />  <br />                                           To investigate molecular mechanisms Hpo Egfr signaling pathways <br />                                           interact Drosophila, measured phospho-ERK activity immunoblotting wing <br />                                           imaginal discs ectopically expressing constitutively active form human YAPS127A <br />                                           (genotype: w; nubbin-Gal4/UAS-YAPS127A; +). Increased phospho-ERK levels observed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           YAPS127A wing discs (Fig. 5h), consistent activation Egfr signaling. <br />  <br />                                           To determine Egfr ligands induced Drosophila Yki, tested their <br />                                           expression using qRT-PCR eye discs ectopically expressing constitutively active form <br />                                           Yki (YkiS168A, equivalent YAPS127A, Huang et al., 2005) (genotype: yw eyFlp/+; tub&gt;y <br />                                           +&gt;ykiS168A/+). Among Egfr ligands, vein displayed moderate, reproducible increased <br />                                           expression yki activated clones (Fig. 5i). The selective effect Egfr ligand is <br />                                           particularly interesting consistent genetic interactions yki Egfr <br />                                           pathways described above. These genetic studies implicated vein, ligands, <br />                                           modifier Hippo phenotypes (Table S1). Taken together, results suggest <br />                                           interactions Hippo Egfr pathways Drosophila compatible the <br />                                           functional relationship identified mammalian cells. Since genetic interactions often <br />                                           complex, studies necessary determine vein expression directly <br />                                           regulated Yki, altered indirectly. Nevertheless, results suggest selective <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                              Page 5 <br />  <br />  <br />                                           induction Egfr ligand Yki contribute interplay Egfr Hippo <br />                                           pathways Drosophila. Unlike mammalian cells, YAP effectors implicated <br />                                           cell autonomous pathways, secreted ligands linked Drosophila Hippo <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pathway. Yki activation leads activation Wingless, ligand Wnt pathway and <br />                                           Serrate, ligand Notch pathway 20,21. Loss upstream Hippo component fat also <br />                                           results activation Egfr signaling 22. Activation Egfr signaling AREG secretion <br />                                           mammalian cells represent evolutionarily conserved feature Hippo <br />                                           pathway. <br />  <br />                                           In summary, EGFR ligand AREG appears downstream effector Hippo <br />                                           pathway direct target YAP. Although role YAP mammalian systems was <br />                                           established tumorigenesis studies, recently linked regulation of <br />                                           progenitor cell pools organ size liver regeneration 4,5. As such, YAP target genes <br />                                           play important roles physiological malignant proliferation signals. YAP target <br />                                           genes investigated gene expression profiling MCF10A cells mouse <br />                                           5,6,9,14. Interestingly, AREG potential targets listed screens MCF10A <br />                                           cells. YAP mediated transcriptional induction thought involve binding known <br />                                           transcription factors, including p73, Runx2 TEAD family members 1,2,23. Our Chip <br />                                           experiments unable identify binding sites transcription factors AREG <br />                                           promoter (data shown), pointing potential additional transcriptional partners YAP in <br />                                           mediating AREG induction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Like YAP, AREG overexpression implicated normal organ proliferation and <br />                                           malignancy. In addition secretion cancer cell lines 24-26, AREG mediates <br />                                           protection liver injury mouse model 27, breast, key mediator of <br />                                           estrogen-induced pubertal epithelial morphogenesis. Mouse knockouts estrogen receptor <br />                                           alpha, AREG EGFR display similar defects mammary gland development 28. Thus, <br />                                           breast potentially tissues, AREG mediate developmentally regulated <br />                                           proliferation malignant transformation, consistent contribution broad role <br />                                           YAP Hippo pathway regulation cellular homeostasis. The fact YAP <br />                                           mediate key functions secreted growth factor implicates non-cell <br />                                           autonomous mechanism contribute expansion normal malignant <br />                                           progenitors, raising possibility eventual therapeutic applications. <br />  <br />                               METHODS <br />                               Cell Culture Transfections <br />                                           MCF10A cell culture 3D culture performed described 29. ACHN cells were <br />                                           maintained Dulbecco&apos;s Modified Eagle Medium (DMEM) supplemented 10% fetal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           bovine serum, 2 mM L-glutamine, 50 U/ml penicillin/streptomycin incubated 5% <br />                                           CO2 37 &#194;&#176;C. Transwell cell migration assay performed previously described 7. <br />  <br />                                           For knockdown studies, shRNA hairpins targeting human YAP AREG obtained from <br />                                           RNAi Consortium (Broad Institute). Forward oligo sequences listed (in 5&#226;&#8364;&#178;-3&#226;&#8364;&#178; <br />                                           direction): shYAP-A: <br />                                           CCGGCCCAGTTAAATGTTCACCAATCTCGAGATTGGTGAACATTTAACTGGGTTT <br />                                           TTG; shYAP-B: <br />                                           CCGGGCCACCAAGCTAGATAAAGAACTCGAGTTCTTTATCTAGCTTGGTGGCTT <br />                                           TTTG. shAREG-A9: <br />                                           CCGGCACTGCCAAGTCATAGCCATACTCGAGTATGGCTATGACTTGGCAGTGTT <br />                                           TTTG; shAREG- <br />                                           G12:CCGGGAACGAAAGAAACTTCGACAACTCGAGTTGTCGAAGTTTCTTTCGTT <br />                                           CTTTTTG. Control shRNA: <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                        Page 6 <br />  <br />  <br />                                           CCGGCAACAAGATGAAGAGCACCAACTCGAGTTGGTGCTCTTCATCTTGTTGTT <br />                                           TTT. shCTGF-1 shCTGF-2 previously reported 14. Lentivirus packaging, MCF10A <br />                                           cell transduction drug selection performed following standard protocols. Human YAP <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           expression construct described previously 7; YAP-S127A expression construct was <br />                                           synthesized PCR-based mutagenesis. siRNA experiments performed described 9. <br />                                           siRNA duplex (ON-TARGETplus-SMARTpool) targeting human LATS1(L-004632-00), <br />                                           LATS2 (L-003865-00) non-targeting control (D-001810-10) purchased from <br />                                           Dharmacon RNAi Technologies. <br />  <br />                                           For conditioned medium treatment, vector, YAP-Wt YAP-S127A transduced MCF10A cells <br />                                           grown 3D culture absence EGF. Media collected added MCF10A <br />                                           parental cells 3D culture 4 days. <br />  <br />                               Antibodies Molecular Analyses <br />                                           Phospho-AKT (Ser-473), phospho-p44/42 MAP kinase (Thr202/Tyr204), total AKT, ERK, <br />                                           YAP (phospho-S127) antibodies purchased Cell Signaling Technology <br />                                           (Beverly, MA); Phospho-EGFR (Y1068) antibody Biosource (Camarillo, CA); YAP, <br />                                           EGFR antibodies Santa Cruz biotechnology (Santa Cruz, CA); LATS1 LATS2 <br />                                           antibodies Bethyl (Montgomery TX); &#206;&#178;-Tubulin antibody upstate (Lake Placid, <br />                                           NY); Flag (M2) antibody Sigma (St. Louis, MO); hAREG, hIGFBP-6, hPDGF-AA, hM- <br />                                           CSF R control IgG neutralizing antibodies purchased R &amp; D systems <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (Minneapolis, MN). For Western blot analysis, cells washed phosphate-buffered <br />                                           saline collected IP buffer: 20 mM Tris-HCl (pH 8.0), 150 mM NaCl, 20% glycerol, <br />                                           0.5% NP-40, 1x protease inhibitor cocktail (Complete&#226;&#8222;&#162; EDTA-free, Roche). Cell lysate was <br />                                           cleared centrifugation 14,000 rpm 20 min 4 &#194;&#176;C. Lysate loaded 4-15% <br />                                           SDS-PAGE gel (ReadyGel, Bio-Rad) 2X SDS sample buffer. For immunoblotting <br />                                           analysis, proteins transferred Immobilon PVDF (Millipore), detected various <br />                                           antibodies visualized Western Lightning Plus Chemiluminescence Kit (Perkin <br />                                           Elmer). The Human Cytokine Antibody Array Human EGFR Phosphorylation Antibody <br />                                           Array analyses performed according user manual (RayBiotech, Norcross, GA). <br />  <br />                                           For RNA preparation qRT-PCR detection, RNA extracted using RNeasy Mini Kit <br />                                           (Qiagen). cDNA synthesis performed using First-Strand cDNA Synthesis Kit (GE <br />                                           Healthcare) quantitative real-time PCR performed using Power SYBR Green PCR <br />                                           Master Mix (Applied Biosystems). <br />  <br />                                           Sequences qPCR primer pairs (in 5&#226;&#8364;&#178;-3&#226;&#8364;&#178; direction) follows: GAPDH-F: <br />                                           GGTGAAGGTCGGAGTCAACGG; GAPDH-R: GAGGTCAATGAAGGGGTCATTG; <br />                                           YAP-F: CCTTCTTCAAGCCGCCGGAG; YAP-R: CAGTGTCCCAGGAGAAACAGC; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           AREG-F: CGAACCACAAATACCTGGCTA; AREG-R: TCCATTTTTGCCTCCCTTTT; <br />                                           IGFBP6-F: CACTGCCCGCCCACAGGATGTG; IGFBP6-R: <br />                                           CTCGGTAGACCTCAGTCTGGAG; PDGF-AA-F: <br />                                           GGGGTCCATGCCACTAAGCATGTGC; PDGF-AA-R: <br />                                           GGAATCTCGTAAATGACCGTCC; M-CSF1-R-F: CAGGAAGGTGATGTCCATCAGC; <br />                                           M-CSF1-R-R: CCAGCTCTGCAGGCACCAG. All measurements performed in <br />                                           triplicate standardized levels GAPDH. Sequences qPCR primer pairs (in <br />                                           5&#226;&#8364;&#178;-3&#226;&#8364;&#178; direction) flies follows: Rp49-F: ACAGGCCCAAGATCGTGAAGA; <br />                                           Rp49-R: CGCACTCTGTTGTCGATACCCT; Tubulin-F: AGACCTACTGCATCGACAAC; <br />                                           Tubulin-R: GACAAGATGGTTCAGGTCAC; Yki-F: GCGCCTTGCCGCCGGGATG; Yki- <br />                                           R: GCTGGCGATATTGGATTCTG; Grk-F: CTGTTGCGACGCCAAATTG; Grk-R: <br />                                           CCGATTGTCCACCACTAGGAAA; Krn-F: CTGCATTGCCACATTCCCA; Krn-R: <br />                                           CGGTAGGTAAGCGCCGATTAA; Spi-F: AGGAACTGCAGCAGGAATACGA; Spi-R: <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                                              Page 7 <br />  <br />  <br />                                           AGCTTGCGCTCCAGAATGACT; Vn-F: ATCAAGCATGGAAAGAAGCTGC; Vn-R: <br />                                           CGCTTGCGATTGATGGACTT <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Chromatin Immunoprecipitation (ChIP) <br />                                           Assays performed previously described 30. Briefly, MCF10A cells transduced with <br />                                           YAP-S127A retrovirus cross-linked, lysed, sonicated generate DNA fragments <br />                                           average size 0.5 kb. The immunoprecipitation performed using 1 &#206;&#188;g antibody <br />                                           IgG YAP (Santa Cruz, CA), respectively. Sequences ChIP PCR primers (in 5&#226;&#8364;&#178;-3&#226;&#8364;&#178; <br />                                           direction) are: AREG-F: CCAAAAGAATCTTTTGAAATCTTTGGGC; AREG-R: <br />                                           GTGAGGGTATGACCAAGTATGGGAAATAAG. Sequences CTGF PCR primers <br />                                           previously reported 14. <br />  <br />                               Drosophila Genetic Analyses <br />                                           All flies maintained standard cornmeal-agar-molasses medium, w1118 flies were <br />                                           used control. To test genetic modification yki+ over-expression phenotype, we <br />                                           established w&#226;&#710;&#8217;; GMR-Gal4/CyO; UAS-yki+/TM6B stock combining GMR-Gal4 <br />                                           chromosome UAS-yki+. The female virgins (w&#226;&#710;&#8217;;GMR-Gal4/CyO; UAS-yki+/TM6B) <br />                                           crossed males different EGFR signaling mutant alleles (obtained the <br />                                           Bloomington Drosophila Stock Center, Table S1). All crosses cultured 25&#194;&#176;C <br />                                           resulting female progenies balancer chromosomes (with genotypes either <br />                                           w&#226;&#710;&#8217;; GMR-Gal4/mutant; UAS-yki+/+ w&#226;&#710;&#8217;;GMR-Gal4/+; UAS-yki+/mutant) analyzed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           presented Figure S3 Table S1. To test potential interactions hpo wts, we <br />                                           performed similar genetic crosses using GMR-hpo+/CyO GMR-wts+/CyO stocks Figure <br />                                           5 Table S1.. <br />  <br />                                           Immunoblotting dpERK performed dissecting wing imaginal discs third <br />                                           instar larvae control (w1118)or nubbin-Gal4/UAS-YAPS127A. qRT-PCR analysis Egfr <br />                                           ligands carried using eye discs control (w1118)or YkiS168A over-expression <br />                                           larvae (genotype: yw eyFlp/+; tub&gt;w+&gt;ykiSA/+). Following standard protocols both <br />                                           western blot qRT-PCR. Primers qRT-PCR listed previous section. <br />  <br />                               Supplementary Material <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                           We thank Drs Duojia Pan, Max Frolov, G Halder, Zhichun Lai generously sharing Drosophila stocks used in <br />                                           work. We appreciate discussions Drs Julie Wells, Fajun Yang Tianyi Zhang thank Bill Fowle for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           assistance electron microscopy. This work supported National Institutes Health (NIH) grant R01 <br />                                           95281, Doris Duke Foundation Distinguished Clinical Investigator Award, National Foundation Cancer <br />                                           Research grant Howard Hughes Medical Institute (to D.A.H.); NIH grants CA080111 CA089393 the <br />                                           Breast Cancer Research Foundation (to J.S.B.); NIH T32 training grants CA09361-27 (to J.Z.) CA09361 (to M.O.); <br />                                           NIH grants F32 CA117737 (to G.A.S.); NIH grants GM81607 GM053203 Saltonstall Foundation (to <br />                                           N.J.D.); Tosteson postdoctoral Fellowship (to J.Y.J.). <br />  <br />  <br />                               REFERENCES <br />                                           1. Harvey K, Tapon N. The Salvador-Warts-Hippo pathway - emerging tumour-suppressor network. <br />                                              Nat Rev Cancer 2007;7:182&#226;&#8364;&#8220;91. [PubMed: 17318211] <br />                                           2. Pan D. Hippo signaling organ size control. Genes Dev 2007;21:886&#226;&#8364;&#8220;97. [PubMed: 17437995] <br />                                           3. Saucedo LJ, Edgar BA. Filling Hippo pathway. Nat Rev Mol Cell Biol 2007;8:613&#226;&#8364;&#8220;21. <br />                                              [PubMed: 17622252] <br />  <br />  <br />  <br />  <br />                                                  Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                                    Page 8 <br />  <br />  <br />                                           4. Camargo FD, et al. YAP1 Increases Organ Size Expands Undifferentiated Progenitor Cells. Curr <br />                                              Biol 2007;17:2054&#226;&#8364;&#8220;60. [PubMed: 17980593] <br />                                           5. Dong J, et al. Elucidation universal size-control mechanism Drosophila mammals. Cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              2007;130:1120&#226;&#8364;&#8220;33. [PubMed: 17889654] <br />                                           6. Hao Y, Chun A, Cheung K, Rashidi B, Yang X. Tumor suppressor LATS1 negative regulator of <br />                                              oncogene YAP. J Biol Chem 2008;283:5496&#226;&#8364;&#8220;509. [PubMed: 18158288] <br />                                           7. Overholtzer M, et al. Transforming properties YAP, candidate oncogene chromosome 11q22 <br />                                              amplicon. Proc Natl Acad Sci U S A 2006;103:12405&#226;&#8364;&#8220;10. [PubMed: 16894141] <br />                                           8. Zender L, et al. Identification validation oncogenes liver cancer using integrative <br />                                              oncogenomic approach. Cell 2006;125:1253&#226;&#8364;&#8220;67. [PubMed: 16814713] <br />                                           9. Zhang J, Smolen GA, Haber DA. Negative regulation YAP LATS1 underscores evolutionary <br />                                              conservation Drosophila Hippo pathway. Cancer Res 2008;68:2789&#226;&#8364;&#8220;94. [PubMed: 18413746] <br />                                           10. Zhao B, et al. Inactivation YAP oncoprotein Hippo pathway involved cell contact <br />                                               inhibition tissue growth control. Genes Dev 2007;21:2747&#226;&#8364;&#8220;61. [PubMed: 17974916] <br />                                           11. Hanahan D, Weinberg RA. The hallmarks cancer. Cell 2000;100:57&#226;&#8364;&#8220;70. [PubMed: 10647931] <br />                                           12. Soule HD, et al. Isolation characterization spontaneously immortalized human breast <br />                                               epithelial cell line, MCF-10. Cancer Res 1990;50:6075&#226;&#8364;&#8220;86. [PubMed: 1975513] <br />                                           13. McAllister SS, et al. Systemic endocrine instigation indolent tumor growth requires osteopontin. <br />                                               Cell 2008;133:994&#226;&#8364;&#8220;1005. [PubMed: 18555776] <br />                                           14. Zhao B, et al. TEAD mediates YAP-dependent gene induction growth control. Genes Dev. 2008 <br />                                           15. Willmarth NE, Ethier SP. Autocrine juxtacrine effects amphiregulin proliferative, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               invasive, migratory properties normal neoplastic human mammary epithelial cells. J Biol <br />                                               Chem 2006;281:37728&#226;&#8364;&#8220;37. [PubMed: 17035230] <br />                                           16. Tapon N, et al. salvador Promotes cell cycle exit apoptosis Drosophila mutated in <br />                                               human cancer cell lines. Cell 2002;110:467&#226;&#8364;&#8220;78. [PubMed: 12202036] <br />                                           17. Shilo BZ. Regulating dynamics EGF receptor signaling space time. Development <br />                                               2005;132:4017&#226;&#8364;&#8220;27. [PubMed: 16123311] <br />                                           18. Wu S, Huang J, Dong J, Pan D. hippo encodes Ste-20 family protein kinase restricts cell <br />                                               proliferation promotes apoptosis conjunction salvador warts. Cell 2003;114:445&#226;&#8364;&#8220;56. <br />                                               [PubMed: 12941273] <br />                                           19. Huang J, Wu S, Barrera J, Matthews K, Pan D. The Hippo signaling pathway coordinately regulates <br />                                               cell proliferation apoptosis inactivating Yorkie, Drosophila Homolog YAP. Cell <br />                                               2005;122:421&#226;&#8364;&#8220;34. [PubMed: 16096061] <br />                                           20. Cho E, et al. Delineation Fat tumor suppressor pathway. Nat Genet 2006;38:1142&#226;&#8364;&#8220;50. [PubMed: <br />                                               16980976] <br />                                           21. Mao Y, et al. Dachs: unconventional myosin functions downstream Fat regulate growth, <br />                                               affinity gene expression Drosophila. Development 2006;133:2539&#226;&#8364;&#8220;51. [PubMed: 16735478] <br />                                           22. Garoia F, et al. The tumor suppressor gene fat modulates EGFR-mediated proliferation control <br />                                               imaginal tissues Drosophila melanogaster. Mech Dev 2005;122:175&#226;&#8364;&#8220;87. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               15652705] <br />                                           23. Zhao B, Lei QY, Guan KL. The Hippo-YAP pathway: new connections regulation organ <br />                                               size cancer. Curr Opin Cell Biol 2008;20:638&#226;&#8364;&#8220;46. [PubMed: 18955139] <br />                                           24. Berquin IM, Dziubinski ML, Nolan GP, Ethier SP. A functional screen genes inducing epidermal <br />                                               growth factor autonomy human mammary epithelial cells confirms role amphiregulin. <br />                                               Oncogene 2001;20:4019&#226;&#8364;&#8220;28. [PubMed: 11494130] <br />                                           25. Kenny PA, Bissell MJ. Targeting TACE-dependent EGFR ligand shedding breast cancer. J Clin <br />                                               Invest 2007;117:337&#226;&#8364;&#8220;45. [PubMed: 17218988] <br />                                           26. Streicher KL, et al. Activation nuclear factor kappaB/interleukin-1 positive feedback loop by <br />                                               amphiregulin human breast cancer cells. Mol Cancer Res 2007;5:847&#226;&#8364;&#8220;61. [PubMed: 17670913] <br />                                           27. Berasain C, et al. Novel role amphiregulin protection liver injury. J Biol Chem <br />                                               2005;280:19012&#226;&#8364;&#8220;20. [PubMed: 15753092] <br />                                           28. McBryan J, Howlin J, Napoletano S, Martin F. Amphiregulin: role mammary gland development <br />                                               breast cancer. J Mammary Gland Biol Neoplasia 2008;13:159&#226;&#8364;&#8220;69. [PubMed: 18398673] <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                                 Page 9 <br />  <br />  <br />                                           29. Debnath J, Muthuswamy SK, Brugge JS. Morphogenesis oncogenesis MCF-10A mammary <br />                                               epithelial acini grown three-dimensional basement membrane cultures. Methods 2003;30:256&#226;&#8364;&#8220;68. <br />                                               [PubMed: 12798140] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           30. Wells J, Farnham PJ. Characterizing transcription factor binding sites using formaldehyde <br />                                               crosslinking immunoprecipitation. Methods 2002;26:48&#226;&#8364;&#8220;56. [PubMed: 12054904] <br />                                           31. Gabay L, Seger R, Shilo BZ. In situ activation <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Drosophila EGF receptor pathway during <br />                                               development. Science 1997;277:1103&#226;&#8364;&#8220;6. [PubMed: 9262480] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                       Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 1. YAP-induced secreted factor enhances EGF-independent growth MCF10A cells <br />                                           (a) Non-cell-autonomous effect YAP. Vector (tagged Cherry marker) YAP- <br />                                           Wt YAP-S127A (tagged GFP) transduced MCF10A cells cultured Matrigel <br />                                           separately 1:1 mixture 25 days EGF. Representative light and <br />                                           fluorescence images shown. (Scale bars, 100&#206;&#188;m) (b) Both YAP YAP-S127A promote <br />                                           EGF-independent growth MCF10A cells 3D culture. Cells transduced vector, <br />                                           YAP YAP-S127A cultured Matrigel 25 days absence EGF. <br />                                           Representative phase contrast images shown. (Scale bars, 100&#206;&#188;m) Immunoblotting analysis <br />                                           endogenous exogenous YAP, using antibodies detect Flag-tag, YAP or <br />                                           phosphorylated S127 residue mutated YAP-S127A. &#206;&#178;-Tubulin used loading <br />                                           control. (c) YAP-Wt YAP-S127A conditioned media induce EGF-independent growth <br />                                           parental MCF10A cells 3D culture. Equal numbers MCF10A cells plated in <br />                                           Matrigel fed 25 days medium vector, YAP YAP-S127A transduced 3D <br />                                           cultures. Representative phase contrast images shown. Numbers represent total acini in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           culture +/&#226;&#710;&#8217; SD (four 200X fields). (Scale bars, 100&#206;&#188;m) <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2010 June 1. <br />                             Zhang et al.                                                                                          Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 2. Amphiregulin (AREG) mediates YAP-induced EGF-independent growth <br />                                           (a) Secreted growth factor screen. Human Growth Factor Antibody Array analysis <br />                                           (RayBiotech, Norcross, GA) performed using 25th day conditioned medium from <br />                                           vector YAP-S127A transduced cells grown presence (left) absence (right) EGF. <br />                                           The membrane encoded antibodies 41 growth factors receptors, four <br />                                           positive negative controls upper left corner. Four proteins predominantly <br />               <br /> </body></html>