<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A nucleolar protein related ribosomal protein L7 is <br /> required early step large ribosomal <br /> subunit biogenesis <br /> David A. Dunbar*, Franc&#194;&#184;ois Dragon*, Sarah J. Lee, Susan J. Baserga&#226;&#8364;&#160; <br /> Departments Therapeutic Radiology Genetics, P. O. Box 208040, Yale University School Medicine, New Haven, CT 06520-8040 <br />  <br /> Communicated Joan A. Steitz, Yale University, New Haven, CT, September 21, 2000 (received review September 5, 2000) <br />  <br /> The Saccharomyces cerevisiae Rlp7 protein extensive iden-                known major minor pathways (6). In pathways, <br /> tity similarity large ribosomal subunit L7 proteins           initial 5&#226;&#172;&#732; end processing reactions occur concurrently with <br /> shares RNA-binding domain them. Rlp7p ribo-                 3&#226;&#172;&#732; end processing B2 site. In major pathway, which <br /> somal protein; however, encoded essential gene               generates 85% 5.8S 25S rRNA, 27SA2 transcript <br /> therefore perform function essential cell growth. In              cleaved RNase MRP A3 site. The 5&#226;&#172;&#732; end the <br /> this report, Rlp7p nucleolar protein plays            resulting 27SA3 transcript processed exonucleolytically <br /> a critical role processing precursors large ribosomal           5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleases, Rat1p Xrn1p, generate the <br /> subunit RNAs. Pulse&#226;&#8364;&#8220; chase labeling experiments Rlp7p-                  27SBS transcript (6). In minor pathway, generates <br /> depleted cells reveal 5.8SS, 5.8SL, 25S pro-             15% 5.8S 25S rRNA, little known the <br /> duced, indicating major minor processing                   factors involved initial processing step generates the <br /> pathways affected. Analysis processing intermediates               27SBL pre-rRNA 27SA2 precursor. However, is <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            BIOCHEMISTRY <br /> primer extension indicates Rlp7p-depleted cells accumulate              clear 5&#226;&#172;&#732; ends 27SBS 27SBL precursors are <br /> the 27SA3 precursor RNA, normally major substrate               mature 5&#226;&#172;&#732; ends 5.8SS 5.8SL <br /> (85%) used produce 5.8S 25S rRNAs, ratio               rRNAs, respectively. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> point on, processing steps <br /> 27SBL 27SBS precursors changes approximately 1:8 8:1              major minor pathways identical. Endonu- <br /> (depleted cells). Because 27SA3 direct precursor 27SBS,            cleolytic cleavage 27SBS&#229;&#8230;&#190;27SBL precursors C2 site <br /> we conclude Rlp7p specifically required 5&#226;&#180;&#8226; 3&#226;&#180;&#8226;             ITS2 generates 7S precursors 5.8S rRNAs. The <br /> exonucleolytic trimming 27SA3 27SBS precursor.               7S precursor processed 3&#226;&#172;&#732; end series 3&#226;&#172;&#732; 5&#226;&#172;&#732; <br /> As essential processing major minor                exonucleases, includes Rex proteins exo- <br /> pathways, propose Rlp7p act specificity factor              generate 5.8SS 5.8SL (7, 8). The 5&#226;&#172;&#732; end 25S <br /> that binds precursor rRNAs tethers enzymes carry                precursor generated cleavage C2 matured Rat1p <br /> out early 5&#226;&#180;&#8226; 3&#226;&#180;&#8226; exonucleolytic reactions generate            Xrn1p produce mature 25S rRNA (9). <br /> mature rRNAs. Rlp7p required endonucleo-                    <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> date, proteins far implicated 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br /> lytic cleavage internal transcribed spacer 2 separates           exonucleolytic processing rRNA precursors Rat1 and <br /> 5.8S rRNA 25S rRNA.                                                 Xrn1 proteins. Their subcellular localization difficult re- <br />                                                                              solve proposed collaborative role pre-rRNA pro- <br />                                                                              cessing nucleolus. Rat1p, encoded essential gene, is <br /> R    ibosomes large ribonucleoproteins translate mRNA <br />      protein. They ribozymes, possessing solely RNA in <br /> their active site (1). The large small ribosomal subunits <br />                                                                              nuclear, Xrn1p, encoded nonessential gene, is <br />                                                                              cytoplasmic (6, 10, 11). Both participate additional exoribo- <br /> harbor distinct RNA protein components. In yeast                     nucleolytic reactions respective cellular compartments <br /> Saccharomyces cerevisiae, small subunit bears 1 rRNA,               (6, 10&#226;&#8364;&#8220;13). These exoribonucleases act multiple substrates <br /> the 18S, large subunit bears 3 rRNAs, 5S, 5.8S,              require proteins specificity factors to <br /> and 25S. The 18S, 5.8S, 25S rRNAs processed                   direct particular substrates. <br /> polycistronic RNA polymerase I transcript assembled                    One U3 snoRNP components required ribosome <br /> their respective small large subunit proteins cell                biogenesis protein called Imp3p (14). Imp3p member of <br /> nucleolus. Ribosome biogenesis highly regulated                S4 family ribosomal proteins (S9 yeast humans), <br /> cell cycle different growth conditions (2). An insuf-              RNA-binding proteins remarkable versatility (15). <br /> ficient number ribosomes checkpoint cell growth              We reasoned uncharacterized proteins <br /> in size cell cycle progression (3).                        S. cerevisiae extensive identity similarity <br />    Ribosome biogenesis S. cerevisiae begins transcrip-               ribosomal protein components cyto- <br /> tion 35S precursor rRNA (pre-rRNA) transcript                   plasmic ribosomes. By using key features gleaned from <br /> modified nucleotide level processed number                study Imp3p guide, identified protein named <br /> specific sites release mature 18S, 5.8S, 25S rRNAs                Rlp7p Yeast Proteome Database (16). Rlp7p been <br /> (Fig. 1; refs. 4, 5). Processing RNA precursors 18S                previously discovered extensive identity and <br /> rRNA requires number endonucleolytic cleavage reac-                     similarity large ribosomal subunit protein L7, its <br /> tions. The small nucleolar ribonucleoproteins (snoRNPs),                function elucidated (17, 18). Because Imp3p, a <br /> U3, U14, snR10, snR30 play essential role these <br /> reactions, shown possess the <br />                                                                              Abbreviations: pre-rRNA, precursor rRNA; snoRNA, small nucleolar RNA; snoRNP, small <br /> cleaving enzyme. Processing large subunit RNAs, 5.8S                  nucleolar ribonucleo-protein; ITS, internal transcribed spacer; YPD, yeast extract&#229;&#8230;&#190;peptone&#229;&#8230;&#190; <br /> and 25S, requires endonucleolytic cleavage exonu-                   dextrose. <br /> cleolytic processing cleavage products begins              *D.A.D. F.D. contributed equally work. <br /> 27SA2 precursor, results endonucleolytic cleavage                 &#226;&#8364;&#160;To   reprint requests addressed. E-mail: susan.baserga@yale.edu. <br /> at A2 site internal transcribed spacer (ITS) 1 (Fig. 1).              The publication costs article defrayed page charge payment. This <br /> Two forms 5.8S rRNAs (5.8SS 5.8SL) differing slightly                 article marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 U.S.C. <br /> in <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> different processing pathways                     &#194;&#167;1734 solely indicate fact. <br />  <br />  <br />  <br />                                                                                            PNAS &#229;&#8230;&#169; November 21, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13027&#226;&#8364;&#8220;13032 <br />                                                                                        One allele RLP7 gene disrupted YPH260 using <br />                                                                                    PCR-based strategy (14, 19, 20). The oligonucleotides used <br />                                                                                    create disruption cassette synthesized com- <br />                                                                                    plementary RLP7 5&#226;&#172;&#732; 3&#226;&#172;&#732; flanking regions HIS3 <br />                                                                                    gene plasmid pAD11. The sequences oligonucleo- <br />                                                                                    tides are: 5&#226;&#172;&#732;-TAGA A AGA AGAGGCTGCCACAGGGA- <br />                                                                                    AAAAGAATAAGATATACAAGTCACACTCTTGGCCTC- <br />                                                                                    CTCTA (rlp7.1) 5&#226;&#172;&#732;-TTAGATAACTAACAACCTAT- <br />                                                                                    GTACTATACAATTTTAAAATACTCTCTTAATCGTTCA- <br />                                                                                    GAATGACACG (rlp7.2). Confirmation RLP7 gene <br />                                                                                    replaced HIS3 carried using PCR <br />                                                                                    chromosomal DNA oligonucleotides flank the <br />                                                                                    RLP7 locus. The oligonucleotides used confirmation are <br />                                                                                    5&#226;&#172;&#732;-GGTACTTTCGTTATT (rlp7.3) 5&#226;&#172;&#732;-TCTGCTCCT- <br />                                                                                    TCAATG (rlp7.4). <br />                                                                                       To create conditionally expressed RLP7 gene, cloned <br />                                                                                    control GAL1&#226;&#8364;&#8220;10 promoter centromeric- <br />                                                                                    based plasmid pGAD3 (14, 19, 21). The GAL1&#226;&#8364;&#8220;10 promoter is <br />                                                                                    activated presence galactose repressed the <br />                                                                                    presence glucose. The RLP7 gene amplified PCR from <br />                                                                                    yeast chromosomal DNA 5&#226;&#172;&#732; oligonucleotide 5&#226;&#172;&#732;- <br />                                                                                    CCGCGGATCCATGTCTTCAACTCAAGAC (rlp7.5) that <br />                                                                                    contains BamHI restriction site 3&#226;&#172;&#732; oligonucleotide <br />                                                                                    5&#226;&#172;&#732;-GCATAGT T TAGCGGCCGCT TA AT TCA ACT TGG- <br />                                                                                    CTAATAAAG (rlp7.6) contains NotI restriction site. The <br />                                                                                    PCR product digested BamHI NotI ligated into <br />                                                                                    sites plasmid pGAD3 make pGAD3-RLP7. This <br /> Fig. 1. The pre-rRNA processing pathway S. cerevisiae. Yeast pre-rRNA        plasmid transformed heterozygous disrupted strain by <br /> synthesized 35S primary transcript gets cleaved trimmed           lithium acetate transformation. Diploids sporulated, and <br /> several sites produce mature 18S, 5.8S, 25S rRNAs ribosome.      tetrads dissected plates containing galactose. The strains <br /> Pre-rRNA processing results number recognizable pre-rRNA processing        resulting spores streaked 5-fluoroorotic acid <br /> intermediates. The processing steps occur 5&#226;&#172;&#732; portion 35S RNA   identify haploid strain harbors plasmid-borne RLP7 <br /> and lead production precursors small ribosomal subunit RNA;       gene chromosomally disrupted RLP7 gene (22). We call <br /> these processing reactions turn release 27SA2 precursor, will <br />                                                                                    strain pGAL1::RLP7. <br /> produce large ribosomal subunit RNAs. The 27SA2 precursor enter two <br /> different pathways generate mature 5.8S 25S rRNAs. Processing <br /> through major pathway initiated RNase MRP cleavage A3 site,       Indirect Immunofluorescence Microscopy. To obtain HA-tagged <br /> and 27SA3 RNA gets trimmed 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleases;              Rlp7 protein, ORF RLP7 amplified using the <br /> exonucleolytic reaction extends 5&#226;&#172;&#732; end 27SBS precursor (site B1S),   polymerase chase reaction oligonucleotides 5&#226;&#172;&#732;-TC- <br /> which 5&#226;&#172;&#732; end mature 5.8SS rRNA. Processing 27SA2             CCCGGGCATGTCTTCAACTCAAGAC (19.72) 5&#226;&#172;&#732;- <br /> precursor minor pathway initiated 5&#226;&#172;&#732; 3&#226;&#172;&#732;              CCGCCTCGAGTAATTAATTCAACTTGGC (20.20) bear- <br /> exonucleolytic enzyme(s); reaction extends site B1L generates      ing XmaI XhoI sites, respectively, cloned XmaI <br /> 5&#226;&#172;&#732; end 27SBL, mature 5&#226;&#172;&#732; end 5.8SL rRNA. In major   XhoI sites pACT2 (CLONTECH). The BglII DNA <br /> and minor pathways, endonucleolytic cleavages ITS2 generate mature <br />                                                                                    fragment coding HA-Rlp7p subcloned BamHI <br /> 5&#226;&#172;&#732; end 25S rRNA 7S precursors trimmed 3&#226;&#172;&#732; 5&#226;&#172;&#732; <br /> exonucleases produce mature 3&#226;&#172;&#732; end 5.8S rRNAs. This figure been <br />                                                                                    site plasmid p415 ADH (23) generate pFD207. Proper <br /> adapted Kressler et al. (5).                                                  orientation insert checked restriction digestion. <br />                                                                                    Strain pGAL1::RLP7 transformed pFD207 grown <br />                                                                                    presence glucose 5-fluoroorotic acid shuffle the <br /> member family small ribosomal subunit proteins,                        plasmid pFD207 pGAD3-RLP7 (22). Expression HA- <br /> found required processing small subunit pre-                      Rlp7p different transformants checked Western blot- <br /> rRNA, hypothesized Rlp7p might, analogy,                             ting, strain (YFD44) chosen analysis. <br /> required processing RNA precursors large                         Strain YFD44 grown yeast extract&#229;&#8230;&#190;peptone&#229;&#8230;&#190;dextrose <br /> subunit RNAs.                                                                      (YPD) OD600 0.5 10 ml cells processed for <br />   Here report Rlp7p nucleolar protein                          indirect immunofluorescence microscopy, described Mi- <br /> required early step processing precursors                  chael Snyder&#226;&#8364;&#8482;s laboratory (http:&#229;&#8230;&#190;&#229;&#8230;&#190;ygac.med.yale.edu&#229;&#8230;&#190;triples&#229;&#8230;&#190; <br />                                                                                    Immu_protocols.html), glusulase&#229;&#8230;&#190;zymolase treat- <br /> the 5.8S 25S rRNAs major minor processing <br />                                                                                    ment reduced 30 min. Fixed cells incubated <br /> pathways. Because possesses conserved RNA-binding do- <br />                                                                                    overnight 4&#194;&#176;C rabbit anti-HA.11 antibodies (Covance, <br /> main, propose serve specificity factor                 Richmond, CA) diluted 1:250 mouse anti-Nop1p mAb A66 <br /> 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleases involved 5&#226;&#172;&#732; end maturation                    (a kind gift John Aris, University Florida; ref. 24) diluted <br /> the pre-rRNAs. Rlp7p required essential                        1:2,000. Secondary antibodies (Jackson ImmunoResearch) were <br /> endonucleolytic cleavage occurs ITS2 separate 5.8S                  used dilution 1:400 FITC-conjugated anti-rabbit and <br /> from 25S rRNA.                                                                 1:200 tetramethylrhodamine B isothiocyanate-conjugated <br />                                                                                    anti-mouse antibodies incubated 2 h room tempera- <br /> Materials Methods                                                              ture. Nuclear DNA stained incubation 1 &#226;?&#174;g&#229;&#8230;&#190;ml <br /> Disruption RLP7 Expression Rlp7 Protein Conditional               4&#226;&#172;&#732;,6-diamidin-2-phenylindole (Sigma) 5 min followed two <br /> Promoter. Studies performed diploid strain                           washes. Cells mounted Fluoromount-G (Southern <br /> YPH260 haploid derivatives, YPH258 YPH259,                          Biotechnology Associates) containing 2% 1,4-diazobicyclo- <br /> grown previously described (14).                                                (2,2,2)-octane (Sigma). Specimens observed Zeiss <br />  <br /> 13028 &#229;&#8230;&#169; www.pnas.org                                                                                                                      Dunbar et al. <br />  Axiophot 2 microscope using &#226;&#171;&#187;100 oil-immersion objective. <br /> Images acquired Photometrics Quantix charge- <br /> coupled device camera, digitized images treated with <br /> Scanalytics IP LAB (Billerica, MA) software. <br />  <br /> Pulse&#226;&#8364;&#8220;Chase Labeling rRNA. Pulse&#226;&#8364;&#8220;chase labeling L-[meth- <br /> yl-3H]methionine carried pGAL1::RLP7 and <br /> YPH259 strains previously described (14, 19). Twenty thou- <br /> sand cpm sample resolved 1.2% formaldehyde&#226;&#8364;&#8220; <br /> agarose gel. Labeled RNAs transferred zeta-probe <br /> membrane (Bio-Rad), sprayed EN3HANCE (Dupont), and <br /> exposed x-ray film. <br />    For pulse&#226;&#8364;&#8220;chase labeling [5,6-3H] uracil, control strain <br /> was made, YPH259-pGAD3, YPH259 transformed <br /> with pGAD3 plasmid contains URA3. Both <br /> pGAL1::RLP7 YPH259-pGAD3 cells grown an <br /> OD600 approximately 1.0 SGal&#226;&#8364;&#8220;uracil (synthetic medium <br /> with galactose supplements uracil) media and <br /> diluted 1:500 fresh SD&#226;&#8364;&#8220;uracil media (synthetic medium with <br /> dextrose supplements uracil). Approximately 50 <br /> OD600 units pGAL1::RLP7 cells 10 OD600 units of <br /> YPH259-pGAD3 cells resuspended 0.4 ml SD&#226;&#8364;&#8220;uracil <br /> media labeled 25 &#226;?&#174;Ci [5,6-3H] uracil milliliter (47   Fig. 2. Rlp7p localizes nucleolus. Immunofluorescence microscopy was <br /> Ci&#229;&#8230;&#190;mmol) room temperature 2 min. The chase                 performed strain expressing HA-tagged Rlp7p. Rabbit anti-HA <br />  <br />  <br />  <br />  <br />                                                                                                                                                       BIOCHEMISTRY <br /> initiated adding pulsed-labeled cells 4 ml SD medium     antibodies used detect HA-Rlp7p (A) mouse mAb A66 was <br />                                                                       used detect nucleolar protein Nop1p (fibrillarin) (B). Nuclear DNA was <br /> containing 1 mg&#229;&#8230;&#190;ml cold uracil. At 0, 5, 30, 60 min after <br />                                                                       stained 4&#226;&#172;&#732;,6-diamidin-2-phenylindole (C). The images merged (D); <br /> the chase, 1-ml aliquots cells pelleted, resuspended       yellow staining results overlapping green red signals. <br /> cold dH2O, repelleted, quick frozen dry-ice ethanol <br /> bath. Total labeled RNAs isolated hot-phenol <br /> technique (25). Thirty thousand cpm sample resolved        synthesis mature 18S rRNA (14). Analysis Yeast <br /> either 1.2% agarose gel 8% polyacrylamide gel. Labeled        Proteome Database (16) indicated uncharac- <br /> RNAs transferred zeta-probe membrane, sprayed          terized essential ORFs extensive identity (greater than <br /> EN3HANCE (Dupont), exposed x-ray film.                         20%) ribosomal proteins. These proteins low codon bias, <br />                                                                       indicating expressed lower levels than <br /> Mapping 5&#226;&#180;&#8226; Ends Mature pre-rRNAs Primer Extension.   ribosomal proteins. RLP7 ORF extensive <br /> The pGAL1::RLP7 strain grown galactose medium (yeast           homology rRNA-binding protein, ribosomal protein L7 <br /> extract&#229;&#8230;&#190;peptone&#229;&#8230;&#190;galactose) YPGal OD600 1.0               (17, 18). Rlp7p 32% identical yeast ribosomal protein <br /> diluted 1:500 fresh glucose medium (YPD). Cells        L7 29% identical rat ribosomal protein L7. It does not <br /> grown glucose medium different periods time. Cells          cosediment ribosomes substitute ribosomal <br /> were pelleted, total RNA isolated hot-phenol <br />                                                                       protein L7. Both yeast rat L7 proteins conserved <br /> technique (25). The oligonucleotide A2&#229;&#8230;&#190;A3 (5&#226;&#172;&#732;-GGCCAG- <br />                                                                       RNA-binding domain located carboxy terminus (27) that <br /> CAATTTCAAGTTA) used map cleavages A2, A3, <br />                                                                       conserved Rlp7p. We hypothesized Rlp7p may <br /> B 1S , B 1L processing sites. The C 1 oligonucleotide (5&#226;&#172;&#732;- <br />                                                                       represent example transacting nucleolar factor that <br /> TACTAAGGCAATCCGGTTGG) used map 5&#226;&#172;&#732; ends <br />                                                                       &#226;&#8364;&#732;&#226;&#8364;&#732;borrowed&#226;&#8364;&#8482;&#226;&#8364;&#8482; ancient RNA-binding domain ribo- <br /> of 25S rRNA. These oligonucleotides used <br /> previously map 5&#226;&#172;&#732; ends yeast precursor mature           somal protein use ribosome biogenesis nucleolus. <br /> rRNAs (26). Primer extensions carried total RNA <br />                                                                       Rlp7p Is Localized Nucleolus. To determine subcellular <br /> from pGAL1::RLP7 cells grown galactose medium (YPGal) <br /> and glucose medium (for depletion; YPD) compared             localization Rlp7p, yeast strain expressing HA epitope- <br /> total RNA YPH259 cells grown media. For              tagged Rlp7p used indirect immunofluorescence micros- <br /> each sample, 5 &#226;?&#174;g total RNA annealed 15 ng               copy (Fig. 2). The N-terminally tagged Rlp7p (HA-Rlp7p) <br /> 32P-labeled oligonucleotides 10 min 70&#194;&#176;C. Primer exten-        localizes crescent-shaped region nucleus (Fig. 2 A) that <br /> sions carried 20 &#226;?&#174;l buffer containing 50 mM          characteristic yeast nucleolus. Cells stained for <br /> Tris&#228;&#161;&#160;HCl, pH 8.3, 75 mM KCl, 3 mM MgCl2, 10 mM dTT             Nop1p (yeast fibrillarin; refs. 28, 29) nucleolar marker (Fig. <br /> the presence 10 units&#229;&#8230;&#190;microliter Superscript reverse tran-         2B). The merge images green channel (HA- <br /> scriptase (GIBCO&#229;&#8230;&#190;BRL) 1 mM dNTPs 42&#194;&#176;C 1 h. For             Rlp7p) red channel (Nop1p) clearly shows HA-Rlp7p <br /> primer extensions using 5.8S C1 oligonucleotides,         Nop1p colocalize nucleolus, appears yellow <br /> extension products resolved 10% sequencing gel. For         Fig. 2D. Therefore, conclude Rlp7p nucleolar <br /> primer extensions using A2&#229;&#8230;&#190;A3 oligonucleotide, exten-         protein function compartment. <br /> sion products resolved 5% polyacrylamide gel. <br />                                                                       Rlp7p Is Required Processing Precursors 5.8S 25S <br /> Results                                                               rRNAs. To test Rlp7p plays role pre-rRNA process- <br /> Mining S. cerevisiae Genome Proteins Required Ribosome    ing, constructed strain pGAL1::RLP7. This haploid strain <br /> Biogenesis. We previously characterized Imp3p, U3-             bears disrupted chromosomal copy RLP7 maintains a <br /> specific protein member S4 family ribosomal       plasmid RLP7 gene control galactose- <br /> proteins. The S4 ribosomal protein (S9 yeast humans)        inducible&#229;&#8230;&#190;glucose-repressible promoter. Rlp7p expressed <br /> a primary RNA-binding protein. Like depletion U3           strain grown galactose grown <br /> small nucleolar RNA (snoRNA), depletion Imp3p prevents             glucose. We assessed growth characteristics of <br />  <br /> Dunbar et al.                                                                           PNAS &#229;&#8230;&#169; November 21, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13029 <br />  Fig. 3.    Rlp7p depletion prevents synthesis 25S rRNA. The <br /> pGAL1::RLP7 strain depleted Rlp7p 17 20 h (depleted) and <br /> compared isogenic strain, YPH259 (nondepleted). Cells <br /> were pulse labeled 2.5 min L-[methyl-3H] methionine chased <br /> for 3 12 min excess cold methionine. Total RNA extracted, <br /> and 20,000 cpm lane loaded resolved 1.2% formaldehyde&#226;&#8364;&#8220; <br /> agarose gel. Labeled RNAs transferred &#226;?&#168;-probe membrane visu- <br /> alized fluorography. <br />                                                                                Fig. 4. Rlp7p depletion prevents synthesis 5.8SS 5.8SL rRNA. <br />                                                                                The pGAL1::RLP7 cells depleted Rlp7p 17 h (depleted) com- <br /> pGAL1::RLP7 growth glucose begins slow                    pared isogenic strain YPH259-pGAD3 (nondepleted). Cells <br /> 10&#226;&#8364;&#8220;15 h switch galactose (data shown),               pulse labeled 2 min [5,6-3H] uracil chased 5, 30, and <br />                                                                                60 min excess cold uracil. Total RNA extracted, 30,000 cpm <br /> been observed previously depletion essential pro- <br />                                                                                lane resolved 8% polyacrylamide gel. Labeled RNAs were <br /> teins required ribosome biogenesis (14, 19).                               transferred zeta-probe membrane visualized fluorography. <br />    We investigated Rlp7p required processing of <br /> the large subunit rRNAs analysis newly rRNAs via <br /> L-[methyl-3H] methionine pulse&#226;&#8364;&#8220;chase labeling vivo. L-[meth-                long chase, Rlp7p required synthesis (Fig. 4, <br /> yl-3H] methionine labels 2&#226;&#172;&#732;-O-methyl groups ribose                  lanes 4 8). Taken together, analysis pre-rRNA <br /> moieties newly synthesized rRNA. In isogenic YPH259                     processing indicates Rlp7p required processing of <br /> strain, 35S precursor processed rapidly 27S               precursors large subunit RNAs, 5.8S, 25S. Further- <br /> 20S pre-rRNAs mature 25S 18S rRNAs                       more, form 5.8S rRNAs made, Rlp7p <br /> (Fig. 3, lanes 1&#226;&#8364;&#8220;3). In Rlp7p-depleted strain, overall                 required pre-rRNA processing major minor <br /> incorporation <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> methyl groups ribose moieties               pathways. <br /> is reduced, observed previously (14, 19), perhaps <br /> because growth strain greatly slowed. In contrast            Rlp7p Is Required Reaction That Processes 27SA3 Precursor. <br /> nondepleted strain, pre-rRNA processing defects observed                  To pinpoint precise step(s) pre-rRNA processing is <br /> on Rlp7p depletion. The 27S precursor was                     blocked, 27S precursors mapped 5&#226;&#172;&#732; ends by <br /> processed mature 25S rRNA (Fig. 3, lanes 4&#226;&#8364;&#8220;9). The                    primer extension analysis. Using oligonucleotide hybrid- <br /> depletion Rlp7p did significantly affect processing              izes region ITS2, primer extensions performed to <br /> 18S rRNA, 20S 18S rRNAs present time                   map levels 27SA2, 27SA3, 27SBL, 27SBS transcripts <br /> points (Fig. 3A, lanes 4&#226;&#8364;&#8220;9). Small effects pre-18S processing,              Rlp7p depletion. Rlp7p depletion results slightly de- <br /> probably secondary defect large subunit biogenesis,              creased levels 27SA2 precursor, observed <br /> be seen lag 35S 32S conversion. Similar results              previously large ribosomal subunit biogenesis inter- <br /> obtained using [5,6-3H] uracil pulse&#226;&#8364;&#8220;chase labeling (data               rupted attributed negative feedback control of <br /> shown). Therefore, Rlp7p depletion inhibits processing                  pre-18S processing (4). An increased signal A3 cleavage <br /> 27S rRNA precursor mature 25S rRNA suggests                    site apparent, indicating Rlp7p depletion results in <br /> Rlp7p required processing 27S rRNA precursors.                   increased levels 27SA3 precursor (Fig. 5, lanes 4 5). <br />    The processing precursors small rRNAs (5.8S 5S)               A large reduction signal B1S cleavage site was <br /> was assessed analysis newly synthesized rRNA using                    observed, indicating large reduction levels 27SBS <br /> [5,6-3H] uracil pulse&#226;&#8364;&#8220;chase labeling vivo. In non-Rlp7p-                precursor Rlp7p depleted (Fig. 5, lanes 4 5). The <br /> depleted strain (YPH259-pGAD3), 7S pre-rRNA pro-                        ratio 27SBL 27SBS precursors changes approximately <br /> cessed mature 5.8SL 5.8SS rRNAs (Fig. 4, lanes 1&#226;&#8364;&#8220;4).              1:8 (wild-type cells) 8:1 (depleted cells). The oligonucleotide <br /> In Rlp7p-depleted cells, growing                       used primer extension hybridizes 7S precursor. <br /> slowly nondepleted, incorporation [5,6-3H] uracil             On Northern blots, 7S steady-state levels greatly <br /> newly synthesized RNAs slowed. However, Rlp7p                  reduced Rlp7p depletion, ratios 7S long short <br /> depleted, 7S precursor mature 5.8S rRNAs absent                    unchanged (data shown). Because this, likely that <br /> even latest time points (Fig. 4, lanes 5&#226;&#8364;&#8220;8), suggesting            primer extension oligonucleotide reflects levels of <br /> Rlp7p required synthesis mature 5.8SL                   27SB precursors. The increased levels 27SA3 precursor <br /> 5.8SS rRNAs. Because levels newly synthesized 5S                    decreased levels 27SBS precursor, the <br /> tRNA similar Rlp7p-depleted nondepleted cells                 exonucleolytic digestion product 27SA3 precursor, suggest <br />  <br /> 13030 &#229;&#8230;&#169; www.pnas.org                                                                                                                              Dunbar et al. <br />                                                                                      Fig. 6. Model role Rlp7p processing 27SA precursor RNAs. <br />                                                                                     Rlp7p interacts pre-rRNA RNA-binding domain and <br />                                                                                     tethers exonucleases (Exo) carry 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleolytic reactions <br />                                                                                     5&#226;&#172;&#732; end 27SA precursors. Rlp7p interact ITS2 attract <br />                                                                                     endonuclease (Endo) cleaves it, symbolized arrow and <br />                                                                                     question mark. <br />  <br />  <br />  <br />                                                                                     RNA-binding domain (Fig. 6). Rlp7p interact with <br />                                                                                     unidentified 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleases participate this <br />                                                                                     reaction. In addition, results indicate Rlp7p required <br />                                                                                     production long form 5.8S an <br />                                                                                     essential component minor processing pathway (Fig. 1). <br />                                                                                     Although experimentally demonstrated, process- <br />                                                                                     ing minor pathway carried 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br />                                                                                     exonucleases (4). If so, Rlp7p serve specificity factor in <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         BIOCHEMISTRY <br />                                                                                     pathway maturation 5&#226;&#172;&#732; end 5.8SL well. On the <br /> Fig. 5. The 27SA3 pre-rRNA accumulates Rlp7p depletion. RNA was <br />                                                                                     basis results presented report, Rlp7p only <br /> isolated pGAL1::RLP7 strain growth galactose (lane 3) 17 h <br /> (lane 4) 20 h (lane 5) switching growth glucose. RNA was <br />                                                                                     known factor shared major minor pathways at <br /> isolated isogenic strain, YPH259, growth galactose       early processing step large ribosomal subunit biogenesis. <br /> (lane 2) 24 h switch growth glucose (lane 1). Primer extension      Rlp7p required proposed endonucleolytic <br /> with oligonucleotide A2&#229;&#8230;&#190;A3 hybridizes ITS2 used reveal           cleavage ITS2 C2 site candidate <br /> processing sites A2, A3, B1L, B1S. Primer extension oligonucleotide    protein required processing event (Fig. 6). In cell where <br /> C1 hybridizes 25S rRNA used map 5&#226;&#172;&#732; end 25S rRNA.      normal pre-rRNA processing occurring, cleavage ITS2 at <br /> Products primer extension reactions resolved polyacrylamide <br />                                                                                     C2 site results appearance 7S precursors, which <br /> gels DNA sequencing ladder (not shown 25S) exposed x-ray <br /> film.                                                                               direct precursor 5.8S rRNAs. Because 7S RNA <br />                                                                                     detectable analysis newly synthesized rRNA (Fig. <br />                                                                                     4) analysis steady-state rRNA (data shown) in <br /> that Rlp7p depletion prevents exonucleolytic processing                      Rlp7p-depleted cells, Rlp7p essential this <br /> the 27A3 27SBS pre-rRNA (Fig. 1).                                            processing event (Fig. 1). <br />   Primer extensions performed map mature 5&#226;&#172;&#732;                           We distinguish primary role Rlp7p 5&#226;&#172;&#732; <br /> end 25S rRNA. Reduced signal 5&#226;&#172;&#732; end 25S (C1                        3&#226;&#172;&#732; exonucleolytic processing endonucleolytic cleavage in <br /> processing site) observed using oligonucleotide                      ITS2. One possibility disruption 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonu- <br /> hybridizes 25S rRNA, indicating Rlp7p depletion leads                    cleolytic processing 27SA3 leads defect ITS2 cleavage. <br /> diminished amounts mature 25S rRNA (Fig. 5, lanes 4                      However, previous results suggest 5&#226;&#172;&#732; end maturation the <br /> 5). This result similar results obtained pulse&#226;&#8364;&#8220;chase                   5.8S rRNAs required cleavage C2. Studies both <br /> labeling (Fig. 3) Northern blot analysis steady-state                     RNA protein components MRP ribonucleopro- <br /> RNA (data shown).                                                               tein, required cleavage A3 site, suggest that <br />                                                                                     cleavage occur normally ITS2 5&#226;&#172;&#732; end maturation 5.8SS <br /> Discussion <br />                                                                                     blocked (reviewed ref. 4). In cases, extended forms of <br /> The nucleolar protein, Rlp7p, plays unique role processing                     5.8S 7S rRNA detectable Northern blot hybridization. <br /> of precursors large subunit rRNAs, 5.8S 25S.                         After Rlp7p depletion, unable detect 5&#226;&#172;&#732; extended <br /> Analysis newly synthesized rRNA Rlp7p depletion <br />                                                                                     forms 5.8S 7S rRNA Northern blot hybridization (data <br /> indicates Rlp7p required production 5.8S and <br />                                                                                     shown). Rlp7p may, fact, required the <br /> 25S rRNAs 18S rRNA. Primer extension analysis indi- <br />                                                                                     maturation 5&#226;&#172;&#732; ends 5.8S rRNAs crucial C2 <br /> cates Rlp7p depletion leads accumulation the <br /> 27SA3 pre-rRNA subsequent decrease 27SBS                             cleavage event leads release 25S rRNA, <br /> processing product, concomitant reversal 27BSS                     ensuring production RNA components the <br /> 27BL ratio (Fig. 1). The results pulse&#226;&#8364;&#8220;chase analysis                  large ribosomal subunit coordinately regulated. <br /> appearance specific processing intermediates                            In addition role trans-acting factors ribosome <br /> Rlp7p depletion indicate Rlp7p required early                    biogenesis, 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleases Rat1p Xrn1p also <br /> step processing pre-rRNA precursors.                                      implicated maturation small, stable RNAs <br />    Processing 27SA3 27SBS rRNA precursor                       U18 U24 (12, 13). We examined Rlp7p was <br /> major pathway involves 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleolytic processing                         required reactions probing 5&#226;&#172;&#732; extended U18 and <br /> reaction. This reaction matures 5&#226;&#172;&#732; end 5.8SS rRNA                    U24 snoRNAs Northern blots RNA isolated Rlp7p- <br /> involves Rat1p Xrn1p exonucleases. Our results suggest                      depleted cells. No 5&#226;&#172;&#732; extended U18 U24 snoRNAs were <br /> that Rlp7p required 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonucleolytic                        detectable (data shown). Similarly, levels small <br /> processing reaction. As exonucleases promiscuous                      RNAs (U1, U3, U4, U5, U6) affected (data shown). <br /> and solely nucleolar, Rlp7p act specificity factor                Thus, potential cooperation Rlp7p Rat1p&#229;&#8230;&#190; <br /> brings exonucleases 27SA3 precursor proposed                     Xrn1p does extend processing snoRNA precursors. <br />  <br /> Dunbar et al.                                                                                          PNAS &#229;&#8230;&#169; November 21, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13031 <br />     Recently, Rlp7p complexed                             Rlp7p. A protein, Rai1p, nonessential protein that <br /> Nop7p number protein components                                  binds Rat1p and, disrupted, leads accumula- <br /> therefore large processing complex committed                            tion 27SA3 (31). Therefore, Ebp2p, Rpf1p, Rai1p may <br /> large ribosomal subunit biogenesis (P. Harnpicharnchai, E.                              interact Rlp7p carry processing 27SA rRNA <br /> Horsey, J. Jakovljevic, J. Roman, M. Unlu, M. Rout, J.                              precursors. <br /> Woolford, Jr., personal communication). In separate studies,                             Imp3p Rlp7p required processing rRNA <br /> snoRNA trans-acting protein factors involved                            precursors small large ribosomal subunits, respec- <br /> with Rlp7p pre-rRNA processing basis genetic                               tively. Imp3p homology small subunit protein is <br /> disruption functions. Depletion essential snR30                         required pre-18S rRNA processing, Rlp7p has <br /> RNA, member box H&#229;&#8230;&#190;ACA class snoRNAs, leads                               homology large subunit protein required pre-5.8S <br /> accumulation 27SA3 pre-rRNA (6). To investigate                                  pre-25S processing. Both bear conserved RNA-binding <br /> whether Rlp7p component snR30 ribonucleopro-                               domains, shown bind <br /> tein, performed immunoprecipitations HA-tagged Rlp7p                              directly RNA. It tempting speculate is <br /> followed direct labeling resulting associated small                           acting placeholder homologous ribosomal protein on <br /> RNAs. These analyses revealed reproducible Rlp7p-snoRNA <br />                                                                                         nascent pre-rRNA. <br /> association (data shown). Likewise, accumulation 27SA3 <br /> is visualized strain harbors temperature-sensitive <br />                                                                                         We thank John Aris anti-Nop1 monoclonal antibody, Jesus la <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>mutation</span> Ebp2 protein, required large                          Cruz, University Sevilla, Spain, protocol [5,6-3H] uracil <br /> rRNA biogenesis (30). Depletion Rpf1p leads accu-                            labeling, Jim Jamieson use microscope. We grateful <br /> mulation 27SA3 precursor defect biogenesis                       Baserga lab members Karen Wehner Jen Gallagher, provided <br /> 5.8S 25S rRNAs (K. A. Wehner, S.J.L., S.J.B., unpub-                            insightful comments manuscript help preparation the <br /> lished results). In examples (Ebp2 Rpf1p)                          figures. F.D. supported Anna Fuller Fund Molecular <br /> has studied, 27SBL 27SBS precursor ratios                              Oncology Yale Cancer Center. S.J.B. member Yale <br /> unchanged, distinct observed                            Cancer Center. <br />  <br />  <br />  1. Nissen, P., Hansen, J., Ban, N., Moore, P. B. &amp; Steitz, T. A. (2000) Science 289,   17. Mizuta, K., Hashimoto, T. &amp; Otaka, E. (1995) Curr. Genet. 28, 19&#226;&#8364;&#8220;25. <br />     920&#226;&#8364;&#8220;930.                                                                            18. Lalo, D., Mariotte, S. &amp; Thuriaux, P. (1993) Yeast 9, 1085&#226;&#8364;&#8220;1091. <br />  2. Warner, J. R. (1999) Trends Biochem. Sci. 24, 437&#226;&#8364;&#8220;439.                              19. Dunbar, D. A., Wormsley, S., Agentis, T. M. &amp; Baserga, S. J. (1997) Mol. Cell. <br />  3. Volarevic, S., Stewart, M. J., Ledermann, B., Zilberman, F., Terracciano, L.,           Biol. 17, 5803&#226;&#8364;&#8220;5812. <br />     Montini, E., Grompe, M., Kozma, S. C. &amp; Thomas, G. (2000) Science 288,              20. Baudin, A., Ozier-Kalogeropoulos, O., Denouel, A., Lacroute, F. &amp; Cullin, C. <br />     2045&#226;&#8364;&#8220;2047.                                                                              (1993) Nucleic Acids Res. 21, 3329&#226;&#8364;&#8220;3330. <br />  4. Venema, J. &amp; Tollervey, D. (1999) Annu. Rev. Genet. 33, 261&#226;&#8364;&#8220;311.                    21. Sobel, S. G. &amp; Synder, M. (1995) J. Cell Biol. 131, 1775&#226;&#8364;&#8220;1788. <br />  5. Kressler, D., Linder, P. &amp; la Cruz, J. (1999) Mol. Cell. Biol. 19, 7897&#226;&#8364;&#8220;7912.    22. Boeke, J. D., Trueheart, J., Natsoulis, G. &amp; Fink, G. R. (1987) Methods <br />  6. Henry, Y., Wood, H., Morrissey, J. P., Petfalski, E., Kearsey, S. &amp; Tollervey,          Enzymol. 154, 164&#226;&#8364;&#8220;175. <br />     D. (1994) EMBO J. 13, 2452&#226;&#8364;&#8220;2463.                                                    23. Mumberg, D., Muller, R. &amp; Funk, M. (1995) Gene 156, 119&#226;&#8364;&#8220;122. <br />  7. Mitchell, P., Petfalski, E., Shevchenko, A., Mann, M. &amp; Tollervey, D. (1997)        24. Aris, J. P. &amp; Blobel, G. (1988) J. Cell Biol. 107, 17&#226;&#8364;&#8220;31. <br />     Cell 91, 457&#226;&#8364;&#8220;466. <br />                                                                                         25. Ausubel, F., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., Smith, <br /> </body></html>