<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Jan. 2008, p. 240&#226;&#8364;&#8220;250                                                                              Vol. 190, No. 1 <br /> 0021-9193/08/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.01528-07 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       Processing as-48ABC RNA AS-48 Enterocin Production by <br />                            Enterococcus faecalis&#228;&#338;&#164; <br />   Matilde Ferna&#194;&#180;ndez,1 Marina Sa&#194;&#180;nchez-Hidalgo,1 Nieves Garc&#196;&#177;&#194;&#180;a-Quinta&#194;&#180;ns,2 Manuel Mart&#196;&#177;&#194;&#180;nez-Bueno,1 <br />                        Eva Valdivia,1 Paloma Lo <br />                                               &#194;&#180;pez,2 Mercedes Maqueda1* <br />            Departamento Microbiolog&#196;&#177;&#194;&#180;a, Facultad Ciencias, Universidad Granada, 18071 Granada, Spain,1 and <br />                                                   &#194;&#180;gicas, CSIC, Ramiro Maeztu 9, 28040 Madrid, Spain2 <br />                     Centro Investigaciones Biolo <br />                                          Received 21 September 2007/Accepted 22 October 2007 <br />  <br />             Enterocin AS-48 production immunity characters encoded 10 genes (as-48ABCC1DD1EFGH) of <br />          pMB2 plasmid Enterococcus faecalis S-48 strain. Among these, as-48A, encoding AS-48 peptide, <br />          as-48BC genes constitute cluster required AS-48 biogenesis immunity. In study, the <br />          levels expression cluster altered insertion site-directed mutagenesis by <br />          expression coupled trans complementation. Phenotypic studies mutants indicated cotranscription <br />          genes revealed inactivation as-48B prevents production AS-48, thus <br />          confirming essentiality AS-48 biogenesis. These studies supported involvement as-48C in <br />          enterocin immunity. In addition, established intergenic region as-48A as-48B <br />          genes decisive AS-48 expression, 3-bp substitution, disrupt potential 47-nucleotide <br />          complex secondary structure, resulted hypoproducing <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>. Transcriptional analyses E. <br />          faecalis wild-type mutant strains supports possibility as-48ABC genes transcribed the <br />          PA promoter located upstream as-48A. Moreover, analysis bioinformatic predictions RNA folding <br />          indicate as-48ABC mRNA processed secondary structure located as-48A as-48B. <br />          Thus, synthesis AS-48 peptide appears controlled posttranscriptional level uncoupled <br />          as-48BC translation. This mechanism genetic regulation previously described the <br />          regulation bacteriocin expression enterococci. <br />  <br />  <br />    The production antimicrobial peptides (bacteriocins) di-             48BC) located 73 nucleotides (nt) downstream TAA <br /> rected competitive microorganisms widespread                    termination codon as-48A intergenic region includes <br /> among gram-positive bacteria (21). A large variety            long inverted repeat (IR). This repeat previously pro- <br /> secreted lactic acid bacteria, peptides               posed transcriptional terminator attenuator, since <br /> extensively studied potential use natural pre-             Northern hybridization revealed abundant RNA species <br /> servatives food applications (21). Enterocin AS-48                 containing structural gene (TA) minor species des- <br /> broad-spectrum, circular peptide hydrophobic cationic               ignated T1, include as-48ABC as-48BC <br /> nature, ribosomally synthesized posttranslation-              (34). The as-48 cluster contains ABC transporters <br /> ally circularized head-<span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>-tail peptide bond (19). The well-           encoded as-48C1D as-48EFGH genes, are <br /> defined, three-dimensional structure AS-48, recently solved             cotranscribed polycistronic mRNA, as-48D1 <br /> by nuclear magnetic resonance X-ray diffraction tech-                  (12). As-48C1D devoted exporting newly <br /> niques (reviewed reference 30), reference numerous             synthesized bacteriocin provides low levels immunity. <br /> studies molecules structured &#226;?&#163; helices (39, 42). As usual         The function transporter replaced the <br /> for bacteriocin determinants, genes encoding processing, <br />                                                                            second multicomponent ABC transporter, As-48EFGH, which <br /> secretion, immunity functions clustered. In fact, the <br />                                                                            involved self-protection exogenously administered <br /> as-48 gene cluster encoding AS-48 trait located the <br />                                                                            AS-48 works complementary immunity mechanism <br /> 68-kb pMB2 conjugative pheromone response plasmid identi- <br />                                                                            (12). We postulated As-48B, association <br /> fied Enterococcus faecalis S-48 (31) functionally trans- <br />                                                                            ABC transporter As-48C1D, form pore is <br /> ferred E. faecalis JH2-2 (34). The expression AS-48 <br />                                                                            specific secretion AS-48 molecules. The ultimate <br /> and immunity depends coordinated expression the <br /> as-48ABCC1DD1EFGH genes (12, 34). Among these, as-48A                   activity multicomponent proteins, As-48BC1D, be <br /> the structural gene as-48D1, encodes small cationic            removal leader peptide promoting head-to-tail cir- <br /> hydrophobic peptide, provides degree bacteriocin re-               cularization concomitantly export molecule <br /> sistance itself identified immunity            (34). However, mechanisms underlying AS-48 processing <br /> determinant (34). Interestingly, overlapping genes (as-                transport cell surface remain unknown. <br />                                                                               The expression of bacteriocin systems far <br />                                                                            characterized (lantibiotics [reviewed reference 28] non- <br />    * Corresponding author. Mailing address: Departamento Microbio-         lantibiotics [reviewed reference 15]) controlled so- <br /> log&#196;&#177;&#194;&#180;a, Facultad Ciencias, Universidad Granada, Fuentenueva <br />                                                                            called, three-component regulatory system, auto- <br /> s/n, E-18071 Granada, Spain. Phone fax: 34 958 242857. E-mail: <br /> mmaqueda@ugr.es.                                                           regulatory model includes secreted bacteriocin as <br />    &#228;&#338;&#164; <br />      Published ahead print 2 November 2007.                          peptide pheromone signal transduction corre- <br />  <br />                                                                      240 <br />  VOL. 190, 2008                                                                                              ANALYSIS OF THE as-48ABC CLUSTER                            241 <br />  <br />  <br />                                                         TABLE 1. Strains plasmids used study <br />    Strain plasmid                                                              Descriptiona                                                        Reference source <br />  <br /> E. faecalis JH2&#226;&#8364;&#8220;2                           Plasmid-free derivative E. faecalis JH-2                                                              49 <br /> E. coli DH5&#226;?&#163;                                supE44 &#226;&#338;&#172;lacU169(&#226;?&#190;80lacZ&#226;&#338;&#172;M15)hsdR17 recA1 endA1 gyrA96 thi-1 relA1                                        Bethesda Research <br />                                                                                                                                                         Laboratories <br /> E. coli RYC1000                             araD139 &#226;&#338;&#172;lacU169 rpsL &#226;&#338;&#172;rib7 thiA gyrA recA56                                                             7 <br /> pAM401                                      Bifunctional E. coli-E. faecalis cloning vector; Cmr Tcr                                                 17 <br /> pMB2                                        AS-48&#226;&#171;&#185; Imm&#226;&#171;&#185;; 68-kb as-48 cluster (as-48ABCC1DD1EFGH)                                                     32 <br /> pMG36e                                      Lactoccocal cloning expression vector, replicative E. coli; Eryr                                  48 <br /> pGEM-3Zf(&#226;&#171;&#185;)                                 Cloning vector; Ampr                                                                                     Promega <br /> pSL1180                                     Cloning vector; Ampr                                                                                     Pharmacia <br /> pAM401-76                                   AS-48&#226;&#171;&#186;, partially Imm&#226;&#171;&#185;; 1.9-kb SphI-BglII (as-48A promoter (PA)                                      12 <br />                                               as-48A structural gene) 6.6-kb BglII (as-48H) DNA fragments <br />                                               pMB2 cloned pAM401; Cmr <br /> pAM401-81                                   AS-48&#226;&#171;&#185; Imm&#226;&#171;&#185;; complete 25-kb as-48 region (PA as-                                                     12 <br />                                                48ABCC1DD1EFGH) pMB2 cloned pAM401; Cmr <br /> pAM401-52                                   AS-48&#226;&#171;&#185; partially Imm&#226;&#171;&#185;; 7.8-kb SphI-PstI (PA as-48ABCC1DD1EF)                                         34 <br />                                               DNA fragment pMB2 cloned pAM401; Cmr <br /> pAM401-58                                   AS-48&#226;&#171;&#186; Imm&#226;&#171;&#185; 0.16.7 kb as-48 region (carries 3&#226;&#172;&#732; region                                     34 <br />                                               as-48A, intergenic region as-48A as-48B, and <br />                                               as-48BCC1DD1EF); derivative pAM401-52 deletion the <br />                                               1.7-kb SphI-PstI DNA fragment; Cmr Tcr <br /> pAM401-81:: Tn5PA                           Tn5 insert PA (as-48A promoter); Cmr Tcr                                                              This   work <br /> pAM401-81::Tn5B                             Tn5 insert as-48B gene; Cmr Tcr                                                                       This   work <br /> pAM401-81::Tn5C                             Tn5 as-48C gene; Cmr Tcr                                                                              This   work <br /> pMG36-A                                     PA-as-48A cloned pMG36e; Eryr                                                                       This   work <br /> pMG36-BC                                    as-48BC cloned pMG36e control P32 promoter; Eryr                                           This   work <br /> pAM401-76X                                  pAM401-76 containing XhoI restriction site 22 nt downstream                                           This   work <br />                                               as-48A gene; Cmr <br /> pBgD12S                                     1.9-kb SphI-BglII D fragment pMB2 cloned pSL1180; Ampr                                         This work <br /> pAM401-81X                                  pAM401-81 containing XhoI restriction site 22 nt downstream                                       This work <br />                                               as-48A gene; Cmr <br />   a <br />     Cmr, chloramphenicol resistant; Tcr, tetracycline resistant; Ampr, ampicillin resistant; Eryr, erythromycin resistant; AS-48&#226;&#171;&#185;, AS-48 production; AS-48&#226;&#171;&#186;, AS-48 <br /> activity detected; AS-48r, AS-8 resistant; AS-48s, AS-48 sensitive. <br />  <br />  <br />  <br /> sponding two-component regulatory signaling system. We                               used predicting RNA secondary structures folding, respectively. <br /> previously shown genes encoding transduction signals                              Calculation predicted termination efficiency secondary structure <br />                                                                                           located as-48A as-48B performed using algorithm de- <br /> not exist as-48 cluster cloning DNA <br />                                                                                           veloped d&#226;&#8364;&#8482;Aubenton Carafa et al. E. coli (10) optimized De Hoon <br /> region pAM401 plasmid results expression                              et al. (11) low G&#226;&#171;&#185;C content bacteria, d value given <br /> the AS-48 character (production immunity) (12). Here                               terminator calculated following equation: d &#226;&#171;&#189; 7.9(&#226;&#338;&#172;G/nSL) &#226;&#171;&#185; 2.67T &#226;&#171;&#186; <br /> present detailed transcriptional analysis as-48ABC genes                             14.61, &#226;&#338;&#172;G Gibbs free energy stem-loop formation kilocalories <br /> reporting posttranscriptional mechanism regulates                                mole (calculated Mfold program), nSL number nucleotides <br />                                                                                           entire stem-loop structure, T parameter depends the <br /> their expression essential role as-48B plays the <br />                                                                                           number consecutive U residues 14-nt stretch adjacent 3&#226;&#172;&#732; end of <br /> biogenesis AS-48.                                                                      stem-loop exponential decay weight 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br />                                                                                           direction applied. The condition d &#226;&#172;&#381; 0 establishes threshold separates <br />                          MATERIALS AND METHODS                                            transcriptional terminators intracistronic structures shows pos- <br />                                                                                           itive value d taken strong indication &#226;?&#179;-independent terminators. <br />    Bacterial strains plasmids. The strains plasmids used study, <br />                                                                                           This d score sensitivity 96.93% detection predicted <br /> together Tn5 insertion mutants pAM401-81 enterocin determi- <br />                                                                                           terminators E. faecalis V583 entire genome, terminators have <br /> nant, listed Table 1. Enterococcal cells grown aeration at <br />                                                                                           average values &#226;&#338;&#172;G, stem <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, uracil stretch &#226;&#171;&#186;18.2, 11.9, 9.9, <br /> 37&#194;&#176;C brain heart infusion broth (BHI-B; Oxoid). Escherichia coli strains were <br /> grown shaking Luria broth 37&#194;&#176;C. The plasmid-free strain E. faecalis            respectively (11), yield average d value 3.07. An analysis different <br /> JH2-2 (49) used cloning experiments as-48ABC cluster              regions structure I (see Fig. 5A) revealed entire structure (47 nt) has <br /> indicator strain detect enterocin activity. When necessary, indicated anti-        d value &#226;&#171;&#186;11.85, best score corresponds central region (tA), <br /> biotics (Sigma-Aldrich, Madrid, Spain) added follows: ampicillin (Ap),            negative value &#226;&#171;&#186;1.2. tA includes stem 11 nt, followed by <br /> 50 &#226;?&#174;g ml&#226;&#171;&#186;1; chloramphenicol (Cm), 20 &#226;?&#174;g ml&#226;&#171;&#186;1; erythromycin (Ery), 200 &#226;?&#174;g                  UUUUUUAAUU stretch close average E. faecalis terminators, but <br /> ml&#226;&#171;&#186;1 E. coli 6 &#226;?&#174;g ml&#226;&#171;&#186;1 E. faecalis; kanamycin (Km), 30 &#226;?&#174;g ml&#226;&#171;&#186;1.           &#226;&#338;&#172;G &#226;&#171;&#186;7.5, minimize formation efficiency and, conse- <br />    Antimicrobial activity assays. AS-48 production different E. faecalis trans-        quently, attenuation effect transcription RNA polymerase this <br /> formants assayed spotting 2 &#226;?&#174;l liquid overnight culture BHI            host. <br /> agar (1.5%), followed incubation 37&#194;&#176;C 16 h. We overlaid plate             Plasmids DNA manipulations. DNA cloning E. coli transformations <br /> with 6 ml BHI soft agar (0.75%, antibiotics) containing 2%           performed according standard protocols (44). E. faecalis plasmid DNA <br /> inoculum indicator strain incubated 37&#194;&#176;C 12 18 h reading          extracted according method Anderson McKay (4). E. faecalis <br /> the results. To test AS-48 sensitivity, 5-&#226;?&#174;l spots containing different quantities     cells transformed electroporation according method Fiedler <br /> purified AS-48 applied lawns seeded E. faecalis JH2-2 (harboring             Wirth (16). Antibiotic-resistant transformants screened AS-48 <br /> different plasmids) plates examined inhibition halos 18 h          production replica plating overlaid sensitive strain. DNA was <br /> of incubation 37&#194;&#176;C.                                                                    sequenced ABI Prism Dye Terminator cycle sequencing ready reaction <br />    Prediction genetic analysis. Mfold (50) Fold programs (Genetics Com-               kit (Perkin-Elmer, Applied Biosystems). Synthetic oligonucleotides <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> puter Group package; Wisconsin University), based energy minimization,                 Amersham Biosciences Europe GmBH (Cerdanyola, Barcelona, Spain). <br />  242           &#194;&#180; NDEZ ET AL. <br />           FERNA                                                                                                                                      J. BACTERIOL. <br />  <br />    PCR DNA sequencing. All primers used PCRs synthesized              rRNAs analysis agarose gels stained ethidium bromide. The <br /> Amersham Pharmacia Biotech (Uppsala, Sweden) based pub-              total RNA concentrations determined UV spectrophotometry. <br /> lished DNA sequence as-48 locus E. faecalis (EMBL X79542). Total              Semiquantitative RT-PCR. Semiquantitative analysis RNA levels contain- <br /> DNA prepared described (38) used DNA template          ing regions as-48A, as-48AB, as-48BC, 16S genes performed by <br /> PCR amplifications carried Biotools DNA polymerase (B&amp;M Labs,               two-step RT-PCR assays. For RT assays, total RNAs (500 ng) added to <br /> Madrid, Spain) iCycler 170-8720 thermal cycler (Bio-Rad Laboratories,          20-&#226;?&#174;l RT reaction mixtures containing 4 &#226;?&#174;l cDNA synthesis buffer, 5 mM <br /> Madrid, Spain). The presence molecular size PCR products             dithiothreitol, 40 U RNaseOUT (Invitrogen), 1 mM deoxynucleoside triphos- <br /> determined electrophoresis 1% (wt/vol) agarose gels using &#226;?&#173; DNA digested       phate mix, 10 &#226;?&#174;M concentration appropriate gene-specific primer, and <br /> with HindIII and/or 100-bp ladder (Gibco-BRL, Invitrogen, United King-           15 U ThermoScript RT (Invitrogen) incubated 30 min 56&#194;&#176;C <br /> dom) molecular weight standard. All PCR products purified       (as-48A), 60&#194;&#176;C (as-48BC), 61&#194;&#176;C (as-48AB). Reaction mixtures containing <br /> Perfectprep Gel Cleanup kit (Eppendorf AG, Hamburg, Germany) se-                 RNA instead cDNA included analysis confirm absence of <br /> quenced.                                                                             DNA contamination RNA preparations. The primers used R-16S (5&#226;&#172;&#732;-C <br />    Generation transposon insertional mutants. The transposition Tn5       TGCTGCCTCCCGTA-3&#226;&#172;&#732;), R-AsA (5&#226;&#172;&#732;-CTGCTCTTTTTCCTTTTTTCTTAAT <br /> pAM401-81 performed introducing plasmid E. coli RYC1000              TTCTTT-3&#226;&#172;&#732;), R-AsAB (5&#226;&#172;&#732;-CGTGTGTTCTATAACAATTTGATGCTCCTTG <br /> according method Mart&#196;&#177;&#194;&#180;nez-Bueno et al. (34). Selected pAM401-81::Tn5      AACA-3&#226;&#172;&#732;), R-AsBC (5&#226;&#172;&#732;-GGTATCTAATCGCATGTACAACAAAAATTC <br /> mutagenized DNAs transferred E. faecalis JH2-2 electroporation,       CTTTC-3&#226;&#172;&#732;) synthesis 16S, as-48A, as-48AB, as-48BC cDNAs, <br /> the new recombinant strains screened AS-48 production immunity.         respectively. RT terminated incubation 37&#194;&#176;C 20 min presence <br />    Site-directed DNA mutagenesis. An XhoI restriction site intergenic         2 U RNase H (Invitrogen). <br /> region as-48A as-48B genes generated overlap extension           For amplification cDNA, 10% cDNA synthesis reaction <br /> PCR mutagenesis (24). The external oligonucleotides used M13 uni-           mixture, 50 pmol primer, 500 &#226;?&#174;M deoxynucleoside triphosphate <br /> versal (5&#226;&#172;&#732;-GTAAAACGACGGCCAGT-3&#226;&#172;&#732;) reverse (5&#226;&#172;&#732;-AACAGCTATGA                         (dATP, dCTP, dGTP, dTTP), 3.5 mM MgCl2, 20 mM Tris-HCl (pH 8.4), and <br /> CCAT-3&#226;&#172;&#732;) primers. The internal primers, mutXh3 (5&#226;&#172;&#732;-TTTAACAATATGATA                   50 mM KCl used 50-&#226;?&#174;l PCR, performed 2 U Taq <br /> AAAAAACTCGAGATTTTCTAGAGATATTCTGTTTT-3&#226;&#172;&#732;) mutXh4 (5&#226;&#172;&#732;-                              DNA polymerase (Invitrogen). The cycling conditions 1 cycle (96&#194;&#176;C 2 <br /> AAAACAGAATATCTCTAGAAAATCTCGAGTTTTTTTATCATATTGTT                                      min) 15, 20, 25, 30 cycles (94&#194;&#176;C 1 min, 56&#194;&#176;C, 60&#194;&#176;C, 61&#194;&#176;C 1 min <br /> AAA-3&#226;&#172;&#732;) (underlined sequences added restriction sites, bold             as-48A, as-48BC, as-48AB, respectively, 72&#194;&#176;C 1 min). Six microli- <br /> nucleotides point mutations), exhibited mismatches relative     ters RT-PCR product analyzed 1% agarose gel electrophoresis and <br /> wild-type sequence, generating new XhoI site. The template mutagenesis       visualized Quantity One Gel Doc 2000 software (Bio-Rad Laborato- <br /> was pBgD12S construction, pSL1180 derivative bearing D (SphI-BglII)        ries). The primers used PCR amplification F-AsA (5&#226;&#172;&#732;-TTTTGAGGA <br /> fragment pMB2 (Table 1). Amplification conditions described          GGAGTATCATGGTTAAAGAAAA-3&#226;&#172;&#732;) R-AsA synthesis 321-nt <br /> Higuchi (23), resulting 2-kb amplified fragment containing XhoI          amplicon containing 100 codons as-48A detection as-48A <br /> restriction site SphI-BglII digested cloned pAM401-76 (12)           mRNA (TA), F-AsAB (5&#226;&#172;&#732;-TGCAGCAGGAAGAGAGTCAATTAAAGCATA <br /> replacing equivalent SphI-BglII fragment [pAM401-76X]. Finally, ligation      CCT-3&#226;&#172;&#732;) R-AsAB synthesis 420-nt amplicon containing 26 <br /> the frame C fragment (BglII) 6.4 kb pMB2 pAM401-76X,             codons as-48A 88 codons as-48B detection as-48AB <br /> previously digested enzyme, yielded plasmid pAM401-81X (25         mRNA, F-AsBC (5&#226;&#172;&#732;-CGCTTCTAAGATCGTCTCCTAATGGTTTGA-3&#226;&#172;&#732;) and <br /> kb). The proper orientation D, C, B fragments established          R-AsBC synthesis 492-nt amplicon containing 164 codons B <br /> restriction analysis, presence XhoI restriction site confirmed    5 codons C detection as-48BC mRNA, F-16S <br /> by DNA sequencing.                                                                   (5&#226;&#172;&#732;-AGAGTTTGATCCTGGCTCAG-3&#226;&#172;&#732;) R-16S synthesis 368-nt <br />    Complementation plasmid constructions compatibility test. The lactococ-       amplicon containing 5&#226;&#172;&#732; region gene encoding 16S rRNA. <br /> cal expression vector pMG36e (Eryr) chosen construct complementation             Northern blot hybridization. For Northern blot analysis, 5 &#226;?&#174;g total <br /> plasmids. This broad host range plasmid derivative pWV01, repli-       RNAs denatured 30 &#226;?&#174;l MOPS (morpholinepropanesulfonic acid) (40 <br /> cates rolling circle (48), consequently, E. faecalis com-    mM MOPS, 10 mM sodium acetate, 2 mM EDTA) buffer containing 50% <br /> patible pAM401 (Cmr), stable theta-replicating plasmid able           deionized formamide, 17% formaldehyde, 0.4% bromophenol. Then they <br /> accommodate large inserts (such 14.8 kb as-48 cluster)           fractionated 1% agarose-formaldehyde gel run 23 V room <br /> maintaining high segregational stability. To test compatibility, JH2-       temperature 14 h. Staining gel ethidium bromide allowed to <br /> 2[pAM401-81] JH2-2[pAM401] strains electroporated pMG36e               check integrity rRNAs fractionation proper transfer the <br /> plasmid transformants selected plates containing chloramphenicol         membrane blotting. Nucleic acids transferred nylon membrane <br /> and/or erythromycin. Several transformants transformation grown         (HybondN&#226;&#171;&#185;; Amersham Pharmacia) capillary blotting 16 h using 20&#226;&#171;&#187; <br /> in presence absence selective antibiotics gener-   SSC (150 mM NaCl plus 15 mM sodium citrate) buffer. The RNAs fixed to <br /> ations, stability plasmids confirmed electrophoretic DNA      membrane exposing filters UV light 4 min. The DNA probes <br /> plasmid analysis (data shown). On basis experiments, pMG36e         used described above: 321-nt amplicon (containing 100 <br /> was used clone as-48A as-48BC genes independently. To construct           codons as-48A) 492-nt amplicon (containing 164 codons B <br /> pMG36-A, 644-bp EcoRV/EcoRI fragment containing structural gene            5 codons C). The amplicons PCR synthesized described <br /> its PA promoter obtained pAM401-81. This fragment cloned             purified use FavorPrep kit (Favorgen Biotech Corp., Kaohsi- <br /> pMG36e, previously digested SmaI/EcoRI enzymes, delete P32            ung, Taiwan). Then used substrates radioactive synthesis and <br /> promoter. Plasmid pMG36-BC constructed PCR amplification               labeling probe thermocycling reaction mixture using 20 ng template <br /> coding regions as-48BC overlapping genes promoter using          DNA, 25 pmol primer, 1.5 mM MgCl2, 20 mM Tris-HCl (pH 8.4), 50 mM <br /> appropriate primers as-48BXma (5&#226;&#172;&#732;-CGCCCGGGGATGAATCTCTTTGGAA                          KCl, 150 &#226;?&#174;M deoxynucleoside triphosphate (dATP, dGTP, dTTP), <br /> TTCTA-3&#226;&#172;&#732;) as-48AC2 (5&#226;&#172;&#732;-CTGCAGCATGCGGATCCGATATCGTAAT                              50 &#226;?&#174;M dCTP, 150 nM [&#226;?&#163;-32P]dCTP (3,000 Ci/mmol; Amersham). Free <br /> ACCGATGCACTTTTTCAA-3&#226;&#172;&#732;). The PCR product (2,620 bp) digested                      [&#226;?&#163;-32P]dCTP removed filtration MicroSpin G25 column (Amersham <br /> with SmaI/SphI cloned pMG36e vector, previously digested       Pharmacia). Hybridization performed overnight 50&#194;&#176;C ULTRAhyb <br /> same restriction enzymes, downstream P32 promoter (18). Therefore,            buffer (Ambion, Applera Hispania S.A., Madrid, Spain) contains 50% form- <br /> pMG36-BC carries P32&#226;&#8364;&#8220;as-48BC transcriptional fusion. These constructions           amide. For autoradiography, blotted filter exposed Agfa CURIX RP2 <br /> were E. coli DH5&#226;?&#163; transferred E. faecalis JH2-2, harboring       Plus films. The radioactivity present filter bands quantified directly <br /> different plasmids.                                                                  PhosphorImager (Fujifilm, Barcelona, Spain). <br />    RNA isolation. Cultures E. faecalis JH2-2, harboring indicated plasmids       Primer extension analysis DNA sequencing. Primer extension analysis was <br /> (see Fig. 2, 3, 4), grown optical density 600 nm (OD600) 0.4    performed previously described (20). The primer used detect start site <br /> for semiquantitative reverse transcription-PCR (RT-PCR) primer extension         as-48ABC mRNA F-5&#226;&#172;&#732; mRNA (5&#226;&#172;&#732;-TCTTTAACCATGATACTCCTCC- <br /> analysis OD600 values 0.4 0.9 Northern hybridization analysis.      3&#226;&#172;&#732;), complementary transcript nucleotides 46 25. One <br /> Cells harvested centrifugation, total RNAs extracted       picomole primer annealed 3.8 &#226;?&#174;g total RNA JH2- <br /> use Fast RNA Pro Blue kit (Q-Biogen, MP Biomedicals, Illkirch, France),       2[pAM401-81] JH2-2[pAM401-81X] strain. The primer used determine the <br /> following manufacturer&#226;&#8364;&#8482;s instructions. For RT, total RNAs treated      processing site as-48ABC mRNA R-Processing (5&#226;&#172;&#732;-TCTTGATTTGCTC <br /> 5 U RNase-free DNase I (Roche Diagnostics, San Cugat del Valle&#194;&#180;s, Spain)      TTAACTTC-3&#226;&#172;&#732;), complementary mRNA 470 450. One <br /> 1 h 37&#194;&#176;C. The total RNA preparations checked integrity yield     picomole primer annealed 7.5 &#226;?&#174;g 15 &#226;?&#174;g total RNA the <br />  VOL. 190, 2008                                                                                     ANALYSIS OF THE as-48ABC CLUSTER                              243 <br />  <br />  <br />                                                                                  pAM401-81 and, subsequently, insertions different <br />                                                                                  locations selected, namely, nucleotides upstream <br />                                                                                  &#226;&#171;&#186;10 region putative PA promoter as-48A <br />                                                                                  (Tn5PA) as-48B (Tn5B) as-48C <br />                                                                                  (Tn5C) gene (Fig. 1). The new insertion plasmids were <br />                                                                                  transferred independently JH2-2 strain, subse- <br />                                                                                  quently, bacteriocin production immunity the <br />                                                                                  transposition mutants analyzed comparison with <br />                                                                                  JH2-2[pAM401-81], JH2-2[pAM401-52], JH2- <br />                                                                                  2[pAM401-58] (Table 2). <br />                                                                                     The JH2-2[pAM401-81] transformant produces AS-48 and <br />                                                                                  resistance levels enterocin similar the <br />                                                                                  original isolate S-48[pMB2] (12) recombinant bacteria <br />                                                                                  JH2-2[pMB2] (32, 34) (Table 2). Nonproducer phenotypes <br />                                                                                  detected JH2-2[pAM401-81::Tn5PA], JH2-2[pAM401- <br />                                                                                  81::Tn5B], JH2-2[pAM401-58]. By contrast, interrup- <br />                                                                                  tion as-48C pAM401-81::Tn5C decreased levels <br />                                                                                  AS-48 production (hypoproducing phenotype) JH2-2 <br />                                                                                  background. These results supported essential function for <br />                                                                                  as-48A as-48B expression AS-48 <br />                                                                                  character (production immunity). Moreover, results <br />                                                                                  suggested PA transcriptional promoter and, indeed, <br />   FIG. 1. Physical map plasmids carrying as-48 genes. A schematic             primer extension experiments (see below) showed be <br /> representation inserts present plasmids depicted. The <br /> locations Tn5 insertions mutants constructed work <br />                                                                                  case. <br /> are indicated pAM401-81. Putative promoters (PA, P2, P32)                Unexpectedly, insertion mutants displayed the <br /> shown plasmid.                                                          reduced levels resistance AS-48 JH2- <br />                                                                                  2[pAM401-81] did. In addition, levels immunity were <br />                                                                                  higher JH2-2[pAM401-58] (resistance 6 &#226;?&#174;g 5 <br /> JH2-2[pAM401-81] JH2-2[pAM401-81X] strain. The primers previously        &#226;?&#174;l&#226;&#171;&#186;1 versus 1 &#226;?&#174;g 5 &#226;?&#174;l&#226;&#171;&#186;1). Interestingly, JH2-2[pAM401-58] <br /> labeled 5&#226;&#172;&#732; ends using [&#226;?&#165;-32P]ATP T4 polynucleotide kinase.       showed higher sensitivity AS-48 JH2-2[pAM401-52] <br /> Primer extension reactions performed incubating annealing mixture <br />                                                                                  (Table 2) did, despite fact difference between <br /> with 15 U ThermoScript RT (Invitrogen) 50&#194;&#176;C 30 min. The sizes the <br /> reaction products determined using 8% polyacrylamide gel containing   absence functional as-48A gene its <br /> 7 M urea. Bands labeled 32P detected autoradiography Kodak       putative PA promoter (Fig. 1; Table 1). Therefore, results <br /> X-Omat S film. The length extended products inferred use of <br /> an unrelated DNA sequence ladders. The double-stranded plasmid pFS21 (47), <br /> which harbors lactococcal citQRP operon heterologous sequence, unla- <br /> beled primer R-Pcit (5&#226;&#172;&#732;-CGGGTATCAAGTCATGG-3&#226;&#172;&#732;) used se- <br />                                                                                  TABLE 2. AS-48 production resistance E. faecalis JH2&#226;&#8364;&#8220;2 <br /> quencing reactions. The nucleotide sequence lactococcal citQRP operon <br />                                                                                            harboring plasmids used work <br /> has accession number S77101 EMBL data bank. DNA sequencing was <br /> determined dideoxy chain terminator method (45) using T7 poly-                                                            Sensitivity indicated <br />                                                                                                                               AS-48 <br /> merase sequencing kit (Amershan-Pharmacia) labeled [&#226;?&#163;-32P]dCTP.                                                     production zone   amt (&#226;?&#174;g 5 &#226;?&#174;l&#226;&#171;&#186;1) of <br />    Accession numbers nucleotide sequences follows: as-48A, X79542                   Plasmid <br />                                                                                                                           inhibition         AS-48b <br /> (33); as-48ABCC1DD1, Y12234 (34); as-48EFGH fragment B BglII                                                        (mm)a      0.5 1 3 6 9 18 <br /> cloned plasmid pAM401-81, AJ438950 (12). <br />                                                                                  pMB2 (positive control)                      16            R   R   R   R    R    R <br />                                                                                  pAM401 (negative control)                    ND            S   S   S   S    S    S <br />                                 RESULTS                                          pAM401-81                                    16            R   R   R   R    R    R <br />                                                                                  pAM401-81::Tn5PA                             ND            R   R   R   R    S    S <br />    Mutational insertion analysis as-48ABC cluster. E.                     pAM401-81::Tn5B                              ND            R   R   R   R    S    S <br /> faecalis JH2-2 plasmid-free strain sensitive AS-48 ente-                 pAM401-81::Tn5C                               8            R   R   R   R    S    S <br /> rocin, previously constructed, transferred                    pAM401-81X                                   11            R   R   R   R    R    R <br /> it, pAM401-81 recombinant plasmid (12) carrying as-                      pAM401-52                                    11            R   R   R   S    S    S <br /> 48ABCC1DD1EFGH wild-type cluster pMB2 de-                           pAM401-58                                    ND            R   R   S   S    S    S <br />                                                                                  pAM401-81::Tn5PA/pMG36-Ac                     7            R   R   R   R    S    S <br /> rivatives pAM401-52 pAM401-58 (Fig. 1 Table 1).                          pAM401-58/pMG36-Ac                           ND            R   R   S   S    S    S <br /> Both deletion plasmids lack coding genes                   pAM401-81::Tn5B/pMG36-BCc                    11            R   R   R   R    R    R <br /> of ABC transporter As-48EFGH (34). In addition,                                a <br />                                                                                      AS-48 production assayed plates indicator strain E. faecalis <br /> pAM401-58 lacks 5&#226;&#172;&#732; region as-48A structural gene                      JH2&#226;&#8364;&#8220;2&#229;&#8230;&#179;pAM401&#229;&#8230;&#180; using AS-48&#229;&#8230;&#179;pMB2&#229;&#8230;&#180; control (wild-type phenotype). ND, <br /> and PA promoter (Fig. 1) (34). Consequently, plas-                      detectable production. <br />                                                                                    b <br />                                                                                      To test AS-48 sensitivity, 5-&#226;?&#174;l spots containing different amounts purified <br /> mid express structural as-48A gene,                         AS-48 applied lawns seeded standard inoculum E. faecalis <br /> carries 3&#226;&#172;&#732; region as-48A, intergenic region be-                       JH2&#226;&#8364;&#8220;2 harboring different plasmids. The plates examined inhibition <br /> tween as-48A as-48B as-48BCC1DD1EF genes.                            halos 18 h incubation 37&#194;&#176;C. R, resistant; S, sensitive. <br />                                                                                    c <br />                                                                                      Erythromycin present growth strains liquid solid <br /> Here, characterize expression as-48ABC genes,                          media, JH2&#226;&#8364;&#8220;2&#229;&#8230;&#179;pAM401/pMG36e&#229;&#8230;&#180; used indicator AS-48 pro- <br /> different Tn5 insertions randomly introduced                           duction. <br />  244         &#194;&#180; NDEZ ET AL. <br />         FERNA                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Analysis as-48AB cotranscription. (A) Northern blot analysis. RNA isolated E. faecalis S-48[pMB2] JH2-2 carrying <br /> the indicated pAM401-derived plasmid end exponential phase. The upper panel shows autoradiogram blotting. The lines <br /> labeled 16S rRNA 23S rRNA indicate locations rRNA species, known trap smaller RNA species (25). The lower <br /> panel shows 23S rRNA 16S rRNA loaded gel stained ethidium bromide. TA, RNA species containing as-48A gene (see <br /> the legend Fig. 5 details). (B) Semiquantitative RT-PCR detection. RNA isolated E. faecalis JH2-2 carrying pM401-81 wild-type <br /> and mutant plasmids. The specific primers used reaction conditions described Materials Methods. The plasmid carried JH2-2 <br /> strain number cycles sample indicated. <br />  <br />  <br />  <br /> suggested as-48C involved immunity              PA. This differential complementation mu- <br /> this gene cotranscribed as-48AB PA.                          tants did appear to gene dosage effects, the <br />    Complementation analysis genes as-                    transcription as-48A (the expected TA RNA approxi- <br /> 48ABC cluster. To test hypothesis, plasmid trans                mately 400 nt) pMG36-A detected Northern blot <br /> complementation experiments performed indepen-                 hybridization both, lower levels JH2- <br /> dently cloning PA-as-48A transcriptional <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> as-             2[pAM401-81] (Fig. 2A data shown). These results <br /> 48BC cluster, lacks known promoter. Independent                   suggested nonproducer AS-48 phenotype JH2- <br /> cloning as-48B as48C attempted,                  2[pAM401-58/pMG36-A] lack as-48BC expres- <br /> overlapping genes (see Fig. 4) lack appar-           sion, indeed, undetectable pAM401-58 back- <br /> ent RBS as-48C indicated coupled translation                ground RT-PCR (data shown). They results also <br /> genes. For cloning, pMG36e (Eryr) plasmid used (48).              indicated transcriptional read-through Tn5 <br /> This expression vector carries strong constitutive lactococ-          carried pAM401-81::Tn5PA allowed low levels synthesis <br /> cal P32 promoter (18), support expression              As-48B, detectable RT-PCR analysis the <br /> as-48BC, shown compatible E. faecalis            intergenic region as-48A as-48B (Fig. 2B). Con- <br /> pAM401-81 (Cmr), harbors as-48 cluster (see Ma-                 sequently, wild-type situation, as-48B co- <br /> terials Methods details). Two recombinant plasmids,               transcribed as-48A PA involved AS-48 biogen- <br /> pMG36-BC pMG36-A, generated (Fig. 1);                  esis. Supporting hypothesis, JH2-2[pAM401-81::Tn5B/ <br /> carries as-48BC genes control P32,           pMG36-BC], expression pMG36-BC P32 <br /> second lacks promoter harbors as-48A structural              promoter partially restored AS-48-producing capacity <br /> gene coupled natural PA promoter. Subsequently,                (Table 2). <br /> the JH2-2[pAM401-58/pMG36-A], JH2-2[pAM401-81::Tn5PA/                        In addition, presence pMG36-A did increase the <br /> pMG36-A], JH2-2[pAM401-81::Tn5B/pMG36-BC] strains                     immunity levels JH2-2[pAM401-81::Tn5PA] JH2- <br /> were constructed plasmid transfer production              2[pAM401-58], suggesting absence necessary expres- <br /> AS-48 immunity enterocin investigated              sion as-48BC accompanying genes. However, expression <br /> (Table 2).                                                                pMG36-BC JH2-2[pAM401-81::Tn5B/pMG36-BC] <br />    Remarkably, expression as-48A trans                     conferred wild-type levels resistance strain. This total <br /> pMG36-A deletion insertion mutants different               complementation immunity phenotype, coupled the <br /> effects. The introduction pMG36-A JH2-2[pAM401-58]                  overall results demonstrated indeed, low resistance <br /> did result AS-48-proficient producer phenotype,                 AS-48 Tn5B Tn5C insertion mutants due <br /> whereas presence pMG36-A JH2-2[pAM401-81::                      polar effects expression as-48D1EFGH down- <br /> Tn5PA] partially restored AS-48-producing capacity (7 ver-            stream genes involved double-immunity mechanisms <br /> sus 16 mm halo isogenic wild-type strain)                previously described (12, 34). Consequently, as-48C con- <br /> enterococcal cells lost Tn5 insertion            stitute determinant involved process. <br />  VOL. 190, 2008                                                                         ANALYSIS OF THE as-48ABC CLUSTER                245 <br />  <br />  <br />  <br />  <br />   FIG. 3. Transcriptional analysis as-48ABC genes JH2-2[pAM401-81] JH2-2[pAM401-81X] strains. (A) Northern blotting detection <br /> using specific probes derived as-48A gene. RNA extracted indicated OD600. O.D., optical density. (B) Semiquantitative <br /> RT-PCR analysis as-48A (upper panel), as-48AB (middle panel), as-48BC (lower panel) mRNAs indicated strains. The plasmid <br /> carried JH2-2 strain number cycles sample indicated. <br />  <br />  <br />  <br />    Transcriptional analysis as-48ABC region. The ex-            pAM401-81::Tn5PA, presumably driven aminoglyco- <br /> pression as-48ABC gene cluster JH2-2, carrying           phosphotransferase promoter Tn5. The detection of <br /> natural pMB2 plasmid pAM401-based derivatives,              amplicon JH2-2[pAM401-81] fact as-48B <br /> investigated Northern blot analysis, results obtained       as-48C overlap (Fig. 4A) suggested as-48ABC <br /> were corroborated RT-PCR analysis. Total RNA ex-                genes constitute transcriptional <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> driven PA pro- <br /> tracted end exponential growth phase hybrid-         moter. Moreover, results indicated TA a <br /> ized specific probes derived as-48A as-48B           specific stable degradation product unstable T1 <br /> genes (see Materials Methods details). The detection           transcript (TABC) previously detected JH2-2[pAM401-52] as <br /> of RNA hybridization as-48A probe autoradiogram            minor band, migrating position 23S RNA and <br /> and rRNAs ethidium bromide stain depicted Fig. 2A.         encompassing as-48ABC genes (34). <br /> A single prominent band corresponding as-48A                   Analysis intergenic region as-48A as- <br /> transcript (TA) detected RNA preparations                  48BC genes. No obvious promoter sequences observed in <br /> S-48[pMB2] JH2-2 carrying pAM401-81                      73-nt intergenic region as-48A as-48B. More- <br /> pAM401-52 (Fig. 2A), correlating AS-48 producer phe-        over, AS-48 nonproducer phenotype JH2-2[pAM401- <br /> notype. As expected, JH2-2[pAM401-58] JH2-2[pAM401-              58], complemented pMG36-A reduced immunity <br /> 81::Tn5PA] lacking functional PA promoter, TA de-            (Table 2), rules existence promoter as- <br /> tected blotting. In addition, TA revealed (albeit        48BC genes region. However, analysis region with <br /> low levels) RNA samples JH2-2[pAM401-58/pMG36-                 Mfold program revealed existence long IR with <br /> A], expression structural gene driven          potential form complex secondary structure having an <br /> the PA-as-48A transcriptional <span id='am-162' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> carried pMG36-A (Fig.            estimated &#226;&#338;&#172;G free energy &#226;&#171;&#186;22.4 (Fig. 4A 5A). This kind <br /> 2A). Furthermore, faint smeared bands larger TA           structure characteristic substrates endoribonucle- <br /> were detected strains carrying functional PA            ases, suggesting implicated mRNA process- <br /> promoter (Fig. 2A 3A data shown). However,                 ing decisive expression as-48BC genes (1, <br /> when as-48B probe used, clear hybridization bands            13, 26, 31). To test hypothesis, analyzed effect of <br /> could detected blottings obtained; weak          replacing bases IR site-directed mutagen- <br /> marks detected just 23S (2,905 nt) 16S          esis experiment, creating XhoI restriction site 21 nt down- <br /> (1,522 nt) rRNAs (data shown). The overall results suggest         stream as-48A gene (Fig. 4A). For purpose, the <br /> that transcripts carrying as-48B short life span,           pAM401-81X plasmid, exhibiting mismatches relative to <br /> which impaired detection Northern hybridization.                wild-type sequence, constructed overlap extension <br />    Therefore, confirm cotranscription as-48A as-         PCR mutagenesis. This plasmid encode mRNA with <br /> 48B genes, 73-nt intergenic region analyzed RT-           altered secondary structure (Fig. 5B). When plasmid <br /> PCR using specific primers (F-AsAB R-AsAB) (Fig. 4)                transferred JH2-2 strain, transformants pro- <br /> deduced coding regions genes. The expected           duced smaller inhibition zones wild-type <br /> 420-nt amplicon detected strains analyzed             strains (11 versus 16 mm), fully resistant 18 &#226;?&#174;g <br /> (Fig. 2B), JH2-2[pAM401-81], JH2-2[pAM401-81::Tn5PA],                  5 &#226;?&#174;l pure AS-48 added exogenously (Table 2). <br /> and JH2-2[pAM401-81::Tn5PA/pMG36-A]. The existence                     A comparative analysis Northern blotting as-48A ex- <br /> RNA substrate RT-PCR amplification Tn5PA                pression JH2-2[pAM401-81X] isogenic JH2- <br /> mutant low level affected presence                2[pAM401-81] wild-type control exponential phase <br /> pMG36-A, demonstrated transcriptional read-through                growth revealed <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> introduced inter- <br />  246         &#194;&#180; NDEZ ET AL. <br />         FERNA                                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Relevant DNA sequences transcription translation as-48ABC operon determination initiation transcription <br /> and mRNA processing. (A) The PA promoter as-48ABC operon RBSs, initiation termination translational codons, <br /> are underlined. The long IR marked arrows. The bases changed generate XhoI site gray box. Transcription initiation <br /> sites as-48A (inferred results presented panel B) underlined labeled &#226;&#171;&#185;1. The processing sites mRNA (inferred from <br /> results presented panel C corresponding positions 401 402 mRNA) indicated arrowheads. Also indicated the <br /> location, sequence, polarity primers used RT-PCR experiments depicted Fig. 2 (F-AsAB R-AsAB) 3 (F-AsA, F-AsAB, <br /> F-AsBC, R-AsA, R-AsAB, R-AsBC) primer extensions depicted panel B (R-5&#226;&#172;&#732; mRNA) panel C (R-processing). F, forward; <br /> R, reverse. Determination initiation transcription PA (B) mRNA processing (C) primer extension wild-type and <br /> pAM401-81X mutant strains. Autoradiograms gels depicted, detected extended products labeled corresponding <br /> coordinates as-48ABC mRNA. The coordinates inferred length products (95 nt &#226;&#171;&#185;1 panel B 70 69 nt <br /> for 400 401 panel C) estimated use unrelated DNA sequence (A, C, T, G) ladders. <br />  <br />  <br />  <br /> genic region as-48A as-48B important con-                 ting accordance hypoproducer phenotype of <br /> sequences mRNA level. The significant                 mutant strain (Table 2). By contrast, opposite situation <br /> almost undetectable amounts TA present RNA prepara-                 observed as-48AB as-48BC mRNAs, their <br /> tions mutant (Fig. 3A), likely related        levels higher mutant, accordance full <br /> conformational changes provoked mutation                    resistance AS-48 correlating detection of <br /> mRNA level (see below). In fact, degraded mRNA, visual-                 hybridization bands longer as-48A Northern blot anal- <br /> ized faint smeared bands TA position,                ysis substrate RT-PCR amplification. These <br /> observed mutant wild-type strain (Fig.            results strongly indicated intergenic region between <br /> 3A), spite fact rRNAs strains intact          as-48A as-48B plays important role fate the <br /> (Fig. 3A, panel). Furthermore, levels RNA                  unstable as-48ABC mRNA transcribed PA promoter. <br /> species carrying as-48A analyzed RT-PCR (Fig. 3B)                    Therefore, initiation transcription PA (Fig. 4B), <br /> using F-AsA R-AsA primers (Fig. 4A), higher levels             processing mRNA (Fig. 4C), investigated <br /> as-48A mRNA detected wild type                    wild-type pAM401-81X mutant strains primer <br /> mutant, correlating results obtained Northern blot-           extension. The detection 5&#226;&#172;&#732; end mRNA revealed <br />  VOL. 190, 2008                                                                         ANALYSIS OF THE as-48ABC CLUSTER                247 <br />  <br />  <br />  <br />  <br />    FIG. 5. Prediction secondary structures present processing region as-48ABC mRNAs 5&#226;&#172;&#732; end processed species <br /> folding. Secondary structures present TAA translation termination codon as-48A fifth codon as-48B as-48ABC <br /> mRNAs encoded pAM401-81 (A) pAM401-81X (B). Prediction Mfold program. Folding 5&#226;&#172;&#732; RNA species generated <br /> from pAM401-81 (C) pAM401-81X (D) transcripts mRNA processing major cleavage sites detected Fig. 4C indicated here <br /> in panels A B arrowheads. The folding performed Fold program. <br />  <br />  <br />  <br /> in strains, extended products, corresponding tran-         DNA sequencing region revealed rearrange- <br /> script starts adenosine guanidine residue located        ments mutant plasmid. <br /> 38 39 nt upstream start codon as-48A             In addition, use primer located proximal 3&#226;&#172;&#732; <br /> correct position transcription driven PA promoter       end secondary structure revealed wild-type <br /> (Fig. 4A). This promoter (Fig. 4A) canonical TATAAT              strain, exist RNA species (one major minor) <br /> &#226;&#171;&#186;10 region TTGctA (lowercase letters indicate bases              (Fig. 4C) having 5&#226;&#172;&#732; ends adjacent uridine residues <br /> that conform consensus sequence) &#226;&#171;&#186;35 region              located right arm secondary structure (Fig. <br /> that separated 17 nt,              5A). Interestingly, minor RNA species detected in <br /> correct spacing binding vegetative sigma factor      mutant strain, predicted encode mRNA with <br /> bacterial RNA polymerase need activator.             altered secondary structure (Fig. 5B). Moreover, folding of <br /> Moreover, inverted direct repeats characteristic tran-           processed RNA species carrying as-48A wild-type <br /> scriptionally regulated operators present       strain (Fig. 5C) revealed protective secondary structures the <br /> surrounding PA promoter. Consequently, physiolog-            5&#226;&#172;&#732; 3&#226;&#172;&#732; ends present mutant RNA <br /> ical conditions, transcription PA wild type occurs         species (Fig. 5D). These predictions, results <br /> during exponential growth (Fig. 2 4). Unexpectedly, lower         transcriptional analysis (Fig. 3), indicated alteration <br /> amount extended DNA fragments detected               3&#226;&#172;&#732; end processed TA species mutant should <br /> pAM401-81X mutant (Fig. 4B). This result             result unstable good substrate fast <br /> shorter half-life TA mutant wild-type        degradation 3&#226;&#172;&#732; exonucleases accumulating specific <br /> strain, lower levels as-48A mRNA detected           degradation products. <br /> in mutant Northern blot RT-PCR analyses (Fig. 3)               Finally, overall results RT-PCR primer exten- <br />  248         &#194;&#180; NDEZ ET AL. <br />         FERNA                                                                                                               J. BACTERIOL. <br />  <br />  <br /> sion experiments (Fig. 3B 4) indicate mutation         cleolytic cleavage primary TABC transcript region <br /> the secondary structure pAM401-81X causes decrease             as-48A as-48B subsequent resistance of <br /> mRNA processing JH2-2 background. However,                 resulting product 3&#226;&#172;&#732;-5&#226;&#172;&#732; exoribonucleolytic degradation <br /> decrease accompanied increase species longer          (6, 22). Supporting proposed role small secondary <br /> TA TABC (Fig. 3A), indicating degradation            structure, TA encoded pAM401-81X mutant much <br /> the 3&#226;&#172;&#732; end TABC efficient mutant      abundant wild type (Fig. 3) does to <br /> the detected degradation products substrate               possess structure 3&#226;&#172;&#732; end (Fig. 5D). The high level of <br /> for increased RT-PCR amplification as-48BC observed             TA wild type synthesis indepen- <br /> in JH2-2[pAM401-81X] (Fig. 3B).                                        dent as-48A transcript PA, upper of <br />                                                                        structure I (tA) includes 3-nt loop 11-nt stem, fol- <br />                           DISCUSSION                                   lowed sequence UUUUUUAAUU, occurs the <br />                                                                        weak transcriptional terminators act attenuators (36). <br />    In work, present evidence constitutive PA            This resembles model described ltnA1A2 transcript <br /> promoter drives transcription as-48ABC operon              production lacticin 3147 immunity (37). How- <br /> includes as-48A bacteriocin structural gene, as-48B            ever, predicted &#226;?&#179;-independent terminators E. faecalis <br /> gene essential function AS-48 biogenesis,            average d value 3.07 (see Materials Methods for <br /> putative cognate immunity gene (as-48C). Moreover,             details) (11), tA negative d score &#226;&#171;&#186;1.2, which <br /> shown expression as-48ABC controlled             makes unlikely tA acts attenuator. Neverthe- <br /> posttranscriptional level.                                             less, termination transcription occurs low <br />    RT-PCR analysis showed as-48A as-48B,           efficiency location, predicted case of <br /> as-48B as-48C, linked (Fig. 3B). Primer extension         processing, yield stable as-48A transcript pro- <br /> analysis revealed RNA species exists, carrying as-48B          tective secondary structure 3&#226;&#172;&#732; end (data shown). <br /> and having 5&#226;&#172;&#732; end located putative secondary structure           Thus, appears RNA processing main mecha- <br /> (Fig. 4 5A). Northern blotting analysis suggests          nism uncouples transcription translation as-48A <br /> as-48ABC operon mRNA transcript, TABC, rapidly degraded             as-48BC (8). The presence stem-loop structure be- <br /> by endonucleases, leading stable mRNA product, TA,            tween structural gene downstream biosynthetic <br /> a TBC short life span, makes detection         genes feature number regulated bacteriocinogenic <br /> impossible (Fig. 2 3). The detection RNA species con-           systems, mersacidin (2), sakacin A (5), sakacin P (35), <br /> taining overlapping 3&#226;&#172;&#732; end as-48B 5&#226;&#172;&#732; end as-48C          nisin (29), lacticin 3147 (9), notable differences in <br /> by sensitive RT-PCR assay (Fig. 2 3) supports             concentrations transcripts structural gene and <br /> this hypothesis.                                                       genes involved biosynthetic machinery also <br />    The computation secondary structure as-48AB           reported. It established addition to <br /> intergenic region codons as-48B         allowing partial read-through downstream synthetic genes, <br /> Mfold program predicted stem-loop secondary structures             stem-loop structure Pep5 required mRNA <br /> (I II) (Fig. 5A). Structure I includes detected            stability (40). Our results demonstrate mutagenesis of <br /> processing sites. The putative RBS as-48B located         structures reliable way improve read-through <br /> stem structure II, modulate translation          adversely impacting mRNA stability reducing <br /> As-48B. The processing sites adjacent uracils located          bacteriocin production. <br /> in double-stranded region structure I. The ubiquitous                With regard as-48ABC operon translation, as-48A and <br /> endoribonuclease RNase III identified lactic acid bacteria          as-48B preceded putative RBSs (5&#226;&#172;&#732;-tgAGGAGGag-3&#226;&#172;&#732;, <br /> (3) cleaves double-stranded regions located stem-loop            lowercase letters indicate bases conform to <br /> structures (14, 43), possible respon-   consensus sequence, 5&#226;&#172;&#732;-AAAGGGGGGC-3&#226;&#172;&#732;) comple- <br /> sible processing as-48ABC transcript. Interest-         mentary 3&#226;&#172;&#732; end E. faecalis 16S rRNA (3&#226;&#172;&#732;-UUU <br /> ingly, specificity RNase III E. faecalis depend     CCUCCCGAGUG-5&#226;&#172;&#732;) (41) correct spacing from <br /> on mRNA sequence, TABC transcript encoded             AUG translation initiation codons. These RBSs have <br /> the JH2-2[pAM401-81X] mutant processed second                computed &#226;&#338;&#172;G free-energy values &#226;&#171;&#186;12.1 &#226;&#171;&#186;14.2, respec- <br /> uracil residue (Fig. 4 5B). The computation predicted         tively, strong binding 3&#226;&#172;&#732; end enterococcal 16S <br /> this U located double-stranded region mutant       rRNA according rules Schur et al. (46). By contrast, <br /> mRNA, structure III structure I (Fig. 5B).          as-48C does possess sequences correspond com- <br /> The &#226;&#338;&#172;Gs predicted formation secondary struc-            petent RBS overlapping 5&#226;&#172;&#732; end 3&#226;&#172;&#732; end of <br /> ture intergenic regions wild type (&#226;&#171;&#186;22.4)        as-48B suggests coupled translation genes occurs, <br /> mutant (&#226;&#171;&#186;16.6) indicate processing occur            constituting second mechanism posttranscriptional con- <br /> latter frequency, detected (Fig. 4).           trol. This case sakacin Q, translation <br />    Here, reported mRNA processing yields               structural immunity genes coupled (35). <br /> abundant TA wild-type strain. The prediction                It frequently postulated bacteriocin-encoding and <br /> folding TA (Fig. 5C) revealed 3&#226;&#172;&#732; end located       immunity protein-encoding genes cotranscribed <br /> in small secondary structure similar &#226;?&#179;-independent            expressed ensure producer strain killed by <br /> transcriptional terminators, act barrier 3&#226;&#172;&#732;-5&#226;&#172;&#732;       bacteriocin. The results presented support fact <br /> exoribonucleases. Consequently, prominent 0.4-kb TA                transcription as-48BC takes place PA <br /> RNA species result combination endoribonu-             linked as-48A structural gene. The precise function of <br />  VOL. 190, 2008                                                                                            ANALYSIS OF THE as-48ABC CLUSTER                             249 <br />  <br />  <br /> As-48B As-48C unknown,                               18. Gaeng, S., S. Scherer, H. Neve, M. J. Loessner. 2000. Gene cloning and <br />                                                                                            expression secretion Listeria monocytogenes bacteriophage-lytic en- <br /> significant homology CirB (58%) CirC (65%),                                 zymes Lactococcus lactis. Appl. Environ. Microbiol. 66:2951&#226;&#8364;&#8220;2958. <br /> are required production circularin A (27). The rea-                         19. Ga&#194;&#180;lvez, A., E. Valdivia, M. Mart&#196;&#177;&#194;&#180;nez-Bueno, M. Maqueda. 1989. Puri- <br /> son genes as-48ABC cotranscribed                                    fication amino acid composition peptide antibiotic AS-48 produced by <br />                                                                                            Streptococcus (Enterococcus) faecalis subsp. liquefaciens S-48. Antimicrob. <br /> investigated functional analysis gene prod-                           Agents Chemother. 33:437&#226;&#8364;&#8220;441. <br /> ucts cell, according results, As-48B indis-                       20. Garc&#196;&#177;&#194;&#180;a-Quinta  &#194;&#180;ns, N. C. Magni, D. Mendoza, P. Lo      &#194;&#180;pez. 1998. The <br /> pensable AS-48 biogenesis, integral membrane                                 citrate transport Lactococcus lactis subsp. lactis biovar diacetylactis <br />                                                                                            induced acid stress. Appl. Environ. Microbiol. 64:850&#226;&#8364;&#8220;857. <br /> protein As-48C, postulated transmembrane domains                             21. Heng, N. C. K., P. A. Wescombe, J. P. Burton, R. W. Jack, J. R. Tagg. <br /> (34), involved protection functions,                             2007. The diversity bacteriocins produced gram-positive bacteria, p. <br /> the immunity determinants as-48D1 as-48EFGH,                                     45&#226;&#8364;&#8220;92. In M. A. Riley M. A. Chavan (ed.), Bacteriocins&#226;&#8364;&#8221;ecology and <br />                                                                                            evolution. Springer, Heidelberg, Germany. <br /> which located independent operon (12, 34).                                   22. Higgins, C. F., S. W. Peltz, A. Jacobson. 1992. Turnover mRNA in <br />                                                                                            prokaryotes lower eukaryotes. Curr. Opin. Genet. Dev. 2:739&#226;&#8364;&#8220;747. <br />                           ACKNOWLEDGMENTS                                              23. Higuchi, R. 1990. Recombinant PCR, p. 177&#226;&#8364;&#8220;183. In M. A. Innis, D. H. <br />                                                                                            Gelfand, J. J. Sninsky, T. J. White (ed.), PCR protocols: guide to <br />   This study supported Spanish Direccio  &#194;&#180;n General                     methods applications. Academic Press, San Diego, CA. <br /> Investigacio&#194;&#180;n Cient&#196;&#177;&#194;&#180;fica y Te&#194;&#180;cnica (projects BIO98-0908-CO2-01                  24. Ho, S. N., H. D. Hunt, R. M. Horton, J. K. Pullen, L. R. Pease. 1989. <br /> BIO2001-3237). M. Ferna&#194;&#180;ndez M. Sa&#194;&#180;nchez-Hidalgo recipi-                          Site-directed mutagenesis overlap-extension using polymerase chain <br /> ents fellowships Spanish Ministry Education, Culture,                       reaction. Gene 77:51&#226;&#8364;&#8220;59. <br /> and Sports.                                                                            25. Homuth, G., S., Masuda, A. Mogk, Y. Kobayashi, W. Schumann. 1997. <br />                                                                                            The dnaK operon Bacillus subtilis heptacistronic. J. Bacteriol. 179:1153&#226;&#8364;&#8220; <br />   We thank Stephen Elson critical reading manuscript. <br />                                                                                            1164. <br />                                  REFERENCES                                            26. Homuth, G., A. Mogk, W. Schumann. 1999. Post-transcriptional regu- <br />                                                                                            lation Bacillus subtilis dnaK operon. Mol. Microbiol. 32:1183&#226;&#8364;&#8220;1197. <br />  1. Allenby, N. E. E., N. O&#226;&#8364;&#8482;Connor, Z. Pra    &#194;&#180;gai, N. M. Carter, M. Miethke, S.       27. Kemperman, R., M. Jonker, A. Nauta, O. P. Kuipers, J. Kok. 2003. <br />     Engelmann, M. Hecker, A. Wipat, A. C. Ward, C. R. Harwood. 2004.                   Functional analysis gene cluster involved production bacte- <br />     Post-transcriptional regulation Bacillus subtilis pst operon encoding         riocin circularin A Clostridium beijerinckii ATCC 25752. Appl. Environ. <br />     phosphate-specific ABC transporter. Microbiology 150:2619&#226;&#8364;&#8220;2628.                        Microbiol. 69:5839&#226;&#8364;&#8220;5848. <br />  2. Altena, K., A. Guder, C. Cramer, G. Bierbaum. 2000. Biosynthesis        28. Kleerebezem, M. 2004. Quorum sensing controls lantibiotic production; <br />     lantibiotic mersacidin: organization type B lantibiotic gene cluster. Appl.       nisin subtilin autoregulate biosynthesis. Peptides 25:1405&#226;&#8364;&#8220; <br />     Environ. Microbiol. 66:2565&#226;&#8364;&#8220;2571.                                                      1414. <br />  3. Amblar, M., S. C. Viegas, P. Lo &#194;&#180;pez, C. M. Arraiano. 2004. Homologous         29. Kuipers, O. P., M. M. Beerthuyzen, R. J. Siezen, W. M. De Vos. 1993. <br />     heterologous expression RNase III Lactococcus lactis. Biochem.             Characterization nisin gene cluster nisABTCIPR Lactococcus lactis. <br />     Biophys. Res. Commun. 323:884&#226;&#8364;&#8220;890.                                                     Requirement expression nisA nisI genes development of <br />  4. Anderson, D. G., L. L. McKay. 1983. A simple rapid method                  immunity. Eur. J. Biochem. 216:281&#226;&#8364;&#8220;291. <br />     isolating large plasmid DNA lactic streptococci. Appl. Environ. Micro-        30. Maqueda, M., A. Ga     &#194;&#180;lvez, M. Mart&#196;&#177;&#194;&#180;nez-Bueno, M. J. Sa  &#194;&#180;nchez-Barrena, C. <br />     biol. 46:549&#226;&#8364;&#8220;552.                                                                      Gonza  &#194;&#180;lez, A. Albert, M. Rico, E. Valdivia. 2004. Peptide AS-48: proto- <br />  5. Axelsson, L., A. Holck. 1995. The genes involved production              type new class cyclic bacteriocins. Curr. Protein Pept. Sci. 5:399&#226;&#8364;&#8220;416. <br />     immunity sakacin A, bacteriocin Lactobacillus sake Lb706. J. <br />                                                                                        31. Marque&#194;&#180;s, S., J. L. Ramos, K. N. Timmis. 1993. Analysis mRNA <br />     Bacteriol. 177:2125&#226;&#8364;&#8220;2137. <br />                                                                                            structure Pseudomonas putida TOL meta fission pathway operon <br />  6. Belasco, J. C., C. F. Higgins. 1988. Mechanisms mRNA decay in <br />                                                                                            transcription initiation point, xylTE xylFJ regions. <br />     bacteria: perspective. Gene 72:15&#226;&#8364;&#8220;23. <br />                                                                                            Biochim. Biophys. Acta 1215:227&#226;&#8364;&#8220;236. <br />  7. Casadaban, M. 1976. Transposition fusion lac genes selected <br />                                                                                        32. Mart&#196;&#177;&#194;&#180;nez-Bueno, M., A. Ga    &#194;&#180;lvez, E. Valdivia, M. Maqueda. 1990. A <br />     promoters Escherichia coli using bacteriophage lambda mu. J. Mol. <br />                                                                                            transferable plasmid associated AS-48 production Enterococcus fae- <br />     Biol. 104:541&#226;&#8364;&#8220;555. <br />                                                                                            calis. J. Bacteriol. 172:2817&#226;&#8364;&#8220;2818. <br />  8. Condon, C. 2003. RNA processing degradation Bacillus subtilis. Mi- <br />                                                                                        33. Mart&#196;&#177;&#194;&#180;nez-Bueno, M., M. Maqueda, A. Ga     &#194;&#180;lvez, B. Samyn, J. van Beeumen, J. <br />     crobiol. Mol. Biol. Rev. 67:157&#226;&#8364;&#8220;174. <br />                                                                                            Coyette, E. Valdivia. 1994. Determination gene sequence and <br />  9. Cotter, P. D., L. H. Deegan, E. M. Lawton, L. A. Draper, P. M. O&#226;&#8364;&#8482;Connor, C. <br />                                                                                            molecular structure enterococcal peptide antibiotic AS-48. J. Bacte- <br />     Hill, R. P. Ross. 2006. Complete alanine scanning two-component <br />                                                                                            riol. 176:6334&#226;&#8364;&#8220;6339. <br />     lantibiotic lacticin 3147: generating blueprint rational drug design. Mol. <br />     Microbiol. 62:735&#226;&#8364;&#8220;747.                                                             34.  <br /> </body></html>