<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Using logical model predict growth yeast <br /> KE Whelan* RD King <br />  <br /> Address: Department Computer Science, Aberystwyth University, Aberystwyth, Wales, UK <br /> Email: KE Whelan* - knw@aber.ac.uk; RD King - rdk@aber.ac.uk <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 12 February 2008                                                   Received: 11 January 2007 <br />                                                                               Accepted: 12 February 2008 <br /> BMC Bioinformatics 2008, 9:97   doi:10.1186/1471-2105-9-97 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2105/9/97 <br /> &#194;&#169; 2008 Whelan King; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: A logical model known metabolic processes S. cerevisiae constructed <br />                  iFF708, existing Flux Balance Analysis (FBA) model, augmented information from <br />                  KEGG online pathway database. The use predicate logic knowledge representation <br />                  modelling enables explicit representation structure metabolic network, and <br />                  enables logical inference techniques used model identification/improvement. <br />                  Results: Compared FBA model, logical model information additional 263 <br />                  putative genes 247 additional reactions. The correctness model evaluated by <br />                  comparison iND750 (an updated FBA model closely related iFF708) evaluating the <br />                  performance models predicting empirical minimal medium growth data/essential gene <br />                  listings. <br />                  Conclusion: ROC analysis statistical studies revealed use simpler logical <br />                  form larger coverage results significant degradation performance compared <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                  iND750. <br />  <br />  <br />  <br />  <br /> Background                                                                    trophic experiments: Robot Scientist [1,2]. In first <br /> In Silico modelling cellular processes                                     incarnation scope Robot Scientist limited to <br /> This paper describes construction application                    rediscovery knowledge single metabolic path- <br /> logical model known metabolic processes bakers&apos;                         way yeast (the Aromatic Amino Acid (AAA) biosynthe- <br /> yeast (Saccharomyces cerevisiae). The model outlines                      sis pathway), logical model pathway [3] <br /> relationships Open Reading Frames (ORFs),                             acted background knowledge Robot <br /> enzymes reactions comprising metabolic path-                          Scientist able generate hypotheses. The logical <br /> ways yeast, presenting knowledge facts                         model presented here, &quot;aber&quot;, updated and <br /> restricted form First Order Logic (FOL). This represen-                    expanded version limited model (the is <br /> tation allows model behave deductive (or rela-                    derived &quot;Aberystwyth&quot; &#226;&#8364;&#8220; model con- <br /> tional) database, model study                         structed). The model expanded include most <br /> metabolic behaviour.                                                          known yeast metabolism. The model has <br />                                                                               updated explicitly relational data <br /> A focus previous research develop-                        structure; replacing single relation included <br /> ment implementation automated                        enzyme metabolite details separate relations, <br /> design, execution interpretation wet lab auxo- <br />  <br />  <br />                                                                                                                                        Page 1 16 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                  http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> enabling natural versatile knowledge repre-            The important quantitative methods ODE <br /> sentation.                                                        (Ordinary Differential Equations) FBA (Metabolic <br />                                                                   Flux Balance Analysis). ODEs established <br /> To evaluate correctness model used          modelling representation science. In using ODEs to <br /> to predict results empirical growth experiments         model metabolism concentration metabolite <br /> <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>knockout</span> mutants growing defined minimal medium                calculated single ODE encapsulating reac- <br /> [4]. Although model designed able           tions metabolite synthesised consumed, <br /> generate growth predictions defined growth media,          fluxes determining transformations from <br /> model contains essential genes, list          metabolites network. The use ODE models <br /> essential genes generated gene-deletion study         main technique quantitative sciences. There <br /> was used conjunction minimal medium                   number specialised ODE modelling <br /> growth results accurately reflect predictions      packages metabolism, e.g. Gepasi/Copasi [15-17] and <br /> the model. An assessment ability model           e-Cell [18]. Despite this, current application ODEs <br /> correctly predict gene essentiality performed;        modelling large-scale metabolism number seri- <br /> this case predictions growth compared          ous problems: know, likely know <br /> list essential genes alone, reference min-      soon, necessary parameters models <br /> imal medium experimental data. ROC analysis used              demand; numerical analysis problems in <br /> to compare logical model iND750 [5], state-of-          solving ODE metabolic models, resulting quantitative <br /> the-art Flux Balance Analysis (FBA) model [5,6] (iND750           qualitative differences simulators. <br /> reflects naming scheme systems biology models, i <br /> refers silico model, ND reflects creators      Flux Balance Analysis (FBA) [5,6,19-21] currently the <br /> model (Natalie Duarte) 750 represents number           common approach quantitatively modelling <br /> genes included model). Both models               metabolism. Standard FBA assumes steady state model <br /> been compared predicted growth outcomes gen-             cell metabolism; recent developments <br /> erated simple majority class classifier,      FBA [20] extended dynamic flux balance <br /> probability prediction success purely random.            analysis capable modelling cells state <br />                                                                   change. Cell reactions modelled matrices: one <br /> Systems biology modelling biochemical                  corresponding stoichiometry reactions, the <br /> networks                                                          containing fluxes reactions. Typically <br /> Systems Biology [7-10] represents shift syner-        fluxes remain unknown experimental <br /> gistic approach cell modelling, concen-         measurement ranges small number &quot;con- <br /> tration interactions inter-related                 trol&quot; fluxes linear/nonlinear programming used to <br /> components behaviour individual            determine values unknown fluxes, based an <br /> components. Advances mathematics sci-             assumption optimality. FBA models metabolism <br /> ence led development diverse techniques            number organisms exist, e.g. S. cerevisiae [5,6] and <br /> and formalisms allowing silico modelling          E. coli [19,20] recently human metabolic network <br /> cell systems. All models represent varying               [21]. The steady state assumption (relaxed dynamic <br /> degrees abstraction observable phenomena              FBA however) inaccuracies unknown fluxes <br /> they represent, coarse large scale models capture       lead inaccuracies overall simulation [22]. <br /> only essential interactions components          However, increased experimental evidence decrease <br /> e.g. KEGG [11,12] EcoCyc [13], higher fidelity rep-        inaccuracies. <br /> resentations detailed functioning interactions a <br /> smaller set components [14].                                   A number flux based alternatives FBA been <br />                                                                   developed address changes fluxes occur <br /> There main groups modelling techniques used            major environmental perturbations as <br /> to represent metabolic networks:                                  gene knockouts: MOMA (Minimisation Metabolic <br />                                                                   Adjustment) [23] ROOM (Regulatory on/off minimi- <br /> 1) Quantitative methods aim capture changes           sation) [24]. Both make use distance measures deter- <br /> in quantities metabolites enzymes etc. rep-         point flux space closest wild type <br /> resenting cellular processes, e.g. reaction kinetics,   flux distribution, keeping homeostasis hypothe- <br />                                                                   sis. MOMA minimises changes individual flux, <br /> 2) Qualitative methods aim simply model pres-            overall network similar possible the <br /> ence interactions, e.g. reactions modelled chemi-       wild type. In contrast, ROOM minimises number of <br /> cal transformations, representation reaction           significant flux changes, better approximating <br /> kinetics                                                          short alternative pathways allow redundancy met- <br />  <br />  <br />                                                                                                                      Page 2 16 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> abolic networks. In predictive study using lethal/non-        The BIOCHAM [29] dedicated biochemical <br /> lethal knockouts E. coli, ROOM accuracy            reasoning engine uses rule based temporal logic lan- <br /> 85%, comparable standard FBA approach.          guage model query possible behaviours <br /> Elementary mode analysis [25] determines set             given biochemical model, MAPK signal trans- <br /> smallest sub-networks larger metabolic network        duction cascades used example. Random <br /> still allow metabolic steady state reached. Each ele-   <span id='am-6' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> networks [30,31] used model <br /> mentary mode represents alternative organism        106 genes comprising yeast transcriptional net- <br /> may use conditions perturbation. Stelling et al [25]      work. Random Boolean useful modelling systems <br /> use elementary-mode analysis determine lethal/non-           interactions known beforehand. Kauffman <br /> lethal gene deletions central carbon metabolism       et al [31] used process identify networks that <br /> E. coli, lethal mutants set       stable, representing rules biological <br /> elementary modes, non-lethal mutants          relevance regulation gene transcription. In gen- <br /> one elementary mode overall positive growth             eral graph models equivalent propositional <br /> rate. Stelling et al [25] observed 90% correct prediction     logic models adequately represent metabolic <br /> rate 90 knockouts studied. However       reactions nodes arcs. However representation <br /> be large number elementary modes moder-         expressive adequately represent rela- <br /> ately large metabolic network, indicating        tionships ORF/enzymes etc. control reac- <br /> be problems scaling approach metabolism           tions. The increased expressivity FOL required this <br /> networks.                                                       additional complexity. <br />  <br /> Logical Graph (LG) based models [1,11-13]           The use qualitative reasoning (QR) [32] interme- <br /> commonest qualitative representations modelling             diate representation quantitative logical <br /> metabolism. Graph based models used metabolic            models, e.g. King et al used QR model Glycolysis <br /> databases e.g. KEGG [11,12], EcoCyc/MetaCyc [13]. Met-          pathway [33]. Rather presence/absence metabo- <br /> abolic pathways represented explicitly, metabo-        lite cell coarse representation logical/ <br /> lite node graph edges represent             graph models, quantitative concentration in <br /> chemical transformations reactions compris-        ODE models, QR models record qualitative state of <br /> ing pathway. Edges annotated             metabolite, e.g. concentration increas- <br /> enzyme(s) catalyse reactions, turn    ing, decreasing constant. Qualitative differential equa- <br /> are related gene(s) encode enzymes. Lemke       tions [33] used calculate change quantity of <br /> et al [26,27] developed graph-based model         metabolites enzymes, FOL representation <br /> metabolic network E. coli used analyse        metabolite enzyme components system. <br /> how damage absence enzyme causes            A common task QR generate complete envi- <br /> the metabolic network. Metabolic damage defined           sionment system, i.e. possible qualitative <br /> the number metabolites longer pro-            states derived initial starting state. <br /> duced organism. They 9%                This computationally expensive task, King et al <br /> enzymes prevent production 5 metabolites,            27,254 possible states Glycolysis model. <br /> but 50% essential enzymes be <br /> found group.                                            Results <br />                                                                 To test utility &quot;aber&quot; model compared with <br /> Logical models use computationally efficient forms          iND750 FBA model, majority classifier that <br /> of propositional FOL (Prolog) represen-       assigns prediction commonly occurring <br /> tation language [28]. As graph models, reaction net-     classification (in case continued growth the <br /> work represented series metabolite nodes         strains). Although iND750 aber model both <br /> chemical transformation arcs, increased             constructed iFF708 [6], subsequent develop- <br /> expressive power logic allow accurate repre-        ment led divergence number ORFs for <br /> sentations relationships genes,              experimental results Giaever et al [4]: <br /> enzymes gene products used annotations            iND750 makes predictions 681 ORFs, aber <br /> reactions, various cellular compartments. For        model makes predictions 940, 641 predictions <br /> example, enzyme names used annotate             shared models. Evaluation performance of <br /> reaction arcs Lemke et al [6,27], unlike ORF EC          models examined shared genes total <br /> number aber model. Metabolic modelling          number genes model, effect add- <br /> by keeping tally metabolites added cell      ing newly categorised ORFs original set be <br /> each reaction (see section 5 detail).                  examined. <br />  <br />  <br />  <br />                                                                                                                     Page 3 16 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                           http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Two different cases <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>gene knockout</span> sensitivity                  Table 2: Formulas used calculate Model Validation metrics <br /> MMD+ura+hist+leu considered, 1) genes to <br />                                                                       Metric                                      Formula <br /> be significantly sensitive 5 generations 2) genes <br /> found significantly sensitive 5 generations          tpr (true positive rate) <br /> remain significantly sensitive 15 generations. As                                                             TP <br /> iND750, metabolic network starting media                                                             tpr = POS <br /> definitions used aber model altered reflect            fpr (false positive rate) <br /> the additional requirements uracil, histidine leu-                                                               FP <br />                                                                                                                   fpr = NEG <br /> cine. An additional comparison                 rfp (relative frequency positives) <br /> iND750 results preprocessing step used                                                                        POS <br />                                                                                                                  rfp = <br /> identify borderline cases retarded growth sensi-                                                               ( POS+NEG ) <br /> tive/refractory data. This preprocessing step used         C (class ratio skew ratio) <br /> sensitivity results ORF, ignoring refractory                                                        c = NEG <br /> scores ORFs, sensitivity data used                                                       POS <br />                                                                       sk_acc (accuracy) <br /> rather focusing ORFs sensitive <br /> in Giaever experiment sets (The iND750 results                                                                                 ( 1&#226;&#710;&#8217; fpr ) <br />                                                                                                                  sk_acc = tpr + c           1 <br />                                                                                                                                                      +c <br /> include ORFs sensitive sets). The <br />                                                                       sk_prec (precision) <br /> imbalance prediction counts results shortfall <br />                                                                                                                                   tpr <br /> of experiment results iND750 preprocessing step                                                         sk_prec = tpr +c &#226;&#8249;&#8230; fpr <br /> (comparison set C Table 1). In cases experi- <br /> mental results comparison set C available, <br /> results set B used. Definitions 3 sets           rics skewed accuracy skewed precision. Table 2 shows <br /> experimentally derived growth growth outcomes                 formulas metrics formulas for <br /> are given Table 1.                                                metrics representing intermediate steps. In table TP <br />                                                                      represents number true positive predictions, TN the <br /> Although logical model uses MMD+ura+hist+leu              number true negatives, FP FN number of <br /> set starting compounds, comparison essential             true false positives negatives respectively. POS <br /> gene list valid knockouts grow             corresponds total number positive predictions <br /> rich YPD certainly grow simple             NEG, total number negative predictions. <br /> defined medium. Analysis model comparisons <br /> made use performance metrics statistical tests           A McNemar &#207;&#8225;2 test used compare performance <br /> evaluated significance observed difference             aber model, iND750 majority classifier on <br /> model performance. These described below, followed               3 comparison sets. All sets ORF <br /> by discussion gene essentiality findings            sources (aber, iND750 shared) used compar- <br /> results minimal medium growth study                           ison aber model iND750. The shared ORFs and <br />                                                                      aber ORF sources used comparison the <br /> Skewed performance metrics significance tests                    aber model majority classifier. The iND750 ORFs <br /> In analysis use ROC space Flach [34] defines            shared ORFs sources used comparison of <br /> a number metrics use skew ratio (c Table 2)          iND750 majority classifier. The McNemar test <br /> allow bias introduced classes dissimilar size (as          analyses proportions mistakes predictions com- <br /> is case model predictions experimental           mon models. To enable comparison equivalent <br /> results.) These metrics used measure clas-          numbers majority class used obtain predictions <br /> sifier&apos;s performance conditions evaluation             ORFs model (i.e. 40 ORFs from <br /> rules/tree nodes etc. classifiers constructed. The met-       iND750 aber model 299 ORFs in <br />  <br /> Table 1: Comparison set definitions comparing model predictions experimental results using MMD defined growth medium <br />  <br />  Prediction/Experiment Comparison Set Growth Outcome                                              No Growth Outcome <br />  <br />                    A                      NOT essential AND NOT significantly sensitive           essential OR significantly sensitive 5 <br />                                           5 generations experimental repeats        generations experimental repeats <br />                    B                      NOT essential AND NOT significantly sensitive           essential OR significantly sensitive 5 <br />                                           5 15 generations                 15 generations experimental <br />                                           experimental repeats                                    repeats <br />                    C                      NOT essential AND NOT sensitive iND750               essential OR sensitive iND750 preprocessing <br />                                           preprocessing <br />  <br />  <br />  <br />  <br />                                                                                                                                         Page 4 16 <br />                                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                     http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> the aber model iND750). Significance                McNemar tests significant difference per- <br /> the McNemar tests determined p &gt; 0.05                formance models, sets of <br /> direction test result collected (i.e. model         ORFs. This reflection small number essential <br /> outperforms significant difference                 vs non-essential ORFs (18.4 % shared ORFs and <br /> observed). The following equation defines McNemar                aber model essential; 17.5% the <br /> test used, b number predictions             iND750 ORFs essential). Indeed slightly smaller <br /> be correct model 1 incorrect model 2 c              proportion essential ORFs iND750 set may <br /> the converse (incorrect model 2 correct model              account apparent increase accuracy IND750. <br /> 1). A Cumulative binomial test used deter-               The accuracy results aber model iND750 are <br /> mine probability generating predictions            slightly poorer results iFF708, indicating that <br /> aber model chance.                                                expansion models slightly decreased abil- <br />                                                                      ity predict essential genes. <br />  <br />                                                                      Comparing predicted experiment outcomes growth <br />                     c   2 <br />                             = <br />                               ( abs( b &#226;&#710;&#8217; c ) &#226;&#710;&#8217;1 ) 2                  outcomes obtained genome gene-deletion <br />                                       b+c                            study performance iND750 <br />                                                                      Table 5, containing results growth predictions the <br /> Predicting gene essentiality                                         defined medium, shows classifiers sim- <br /> We use ability metabolic network models pre-             ilar values performance statistics indicating similar <br /> dict essentiality gene measure model qual-          performance logical model, iND750 the <br /> ity. Gene essentiality results presented               majority classifier comparison sets; however <br /> iFF708, prime source knowledge aber model             slight decrease performance comparison <br /> and precursor iND750, aber model              sets B C. The McNemar test results (Tables 6, 7 8) <br /> compared. F&#195;&#182;rster et al [35] report predictive accuracy         indicate significant difference pre- <br /> 85% gene essentiality iFF708. A model&apos;s prediction           dictive accuracy aber model iND750 all <br /> can tested comparison empirical &quot;wet&quot;                 ORF sources comparison sets, mod- <br /> experiment growing knockout strain. The growth                els perform significantly better majority classifier <br /> (<span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span>) knockout strain depends gen-            stringent definitions growth/no <br /> otype (what gene(s) removed) environ-              growth. This case majority class its <br /> ment. To use model metabolism predict gene                   largest (82% 83%) predictive model <br /> essentiality, necessary know composi-         performs better. This indicates use compli- <br /> tion growth medium (the environment). This                    cated formalism FBA does improve prediction of <br /> makes use &quot;wet&quot; results using common                 healthy/retarded growth compared logical modelling. How- <br /> growth medium yeast, YPD, problematic                observation tempered fact a <br /> exact composition undefined. This                  fully direct comparison FBA prediction tech- <br /> focused using Giaever et al data [4] growth          nique logical model prediction technique not <br /> defined minimal medium.                                              possible, network corresponding the <br />                                                                      shared ORFs incomplete. The results McNemar <br /> Table 3 shows skewed accuracy skewed precision               tests indicate expanding coverage the <br /> results aber model, iND750 majority clas-            model adding information KEGG signif- <br /> sifier (maj) 3 sets genes described             icantly compromised performance component <br /> Table 4 presents results McNemar tests determine           model originating iFF708. Indeed, been <br /> the significance difference models.              possible increase coverage metabolic network <br /> These results little variation performance,             30% significant degradation per- <br /> with iND750 scoring slightly higher terms accuracy             formance. There significant difference <br /> on ORFs corresponding source set. The                 aber model iND750 comparison <br />  <br /> Table 3: Performance results gene essentiality predictions <br />  <br />  ORF Source                              Aber         Shared     iND750       Shared        Aber          iND750             Shared <br />  <br />  Model                                   Aber         Aber       iND750       iND750         maj             maj               maj <br />  Num ORFs                                940          641          681          641          940             681               641 <br />  Skewed Accuracy                         0.81         0.80         0.82         0.80         0.82            0.83              0.82 <br />  Skewed Precision                        0.83         0.86         0.84         0.85         0.82            0.83              0.82 <br />  <br />  <br />  <br />  <br />                                                                                                                           Page 5 16 <br />                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                               http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Table 4: Results McNemar significance tests gene essentiality <br />  <br />  Comparison                                                          Aber ORFs (940)        iND750 ORFs (681)               Shared ORFs (641) <br />  <br />  Aber Model vs iND750                            result                     1.19                       2.93                          2.64 <br />                                                  significance                                                                   no <br />                                                  direction <br />  Aber Model vs Majority Classifier               result                     0.31                       NA                              0 <br />                                                  significance                                                                      no <br />                                                  direction <br />  iND750 vs Majority Classifier                   result                     NA                         2.56                          2.31 <br />                                                  significance                                                                      no <br />                                                  direction <br />  <br />  <br /> set B &#226;&#8364;&#8220; stringent <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> sensitive growth.                  essential/retarded growth genes). This test resulted a <br /> This partly skewed nature experi-                  probability, P(X &#226;&#8240;&#165; 889)bin = 8.81 &#195;&#8212; 10-32: low prob- <br /> mental results (the skew ratio C varies 0.22                    ability predictions generated chance. <br /> 0.34 result sets ORF source combinations) and <br /> partly generally high false positive rates                   Discussion <br /> (variation 0.55 0.86), iND750                    Comparison logical model iND750 demon- <br /> the aber model make incorrect predictions                          strated significant difference per- <br /> growth.                                                                     formance of models prediction gene <br />                                                                             essentiality. A significant increase performance was <br /> Estimating probability chance occurrence model                    observed aber model iND750 w.r.t the <br /> predictions                                                                 majority classifier predicting growth outcome on <br /> A cumulative binomial test used evaluate prob-                   defined growth medium, stringent <br /> ability predictions logical model                      <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> growth Giaever et al [4]. There also <br /> could chance. This test calculates                    significant difference aber model and <br /> significance deviations binomial distribution                   iND750 defined growth medium study. The gene <br /> of observations falling categories (predictions                 essentiality study indicates models equally <br /> growth growth), assuming random binomial                           poor predicting essential genes, an <br /> model assigns genes essential/retarded growth                   improvement simple majority classifier. As the <br /> non essential/healthy growth categories chance. The                      case minimal medium experimental data, the <br /> probability calculated range r..n, r = 889:                essential gene data highly skewed, reflecting amount <br /> the number predictions growth n total                     redundancy yeast genome. The improvement in <br /> number predictions (growth + growth: 940):                       performance models w.r.t defined medium <br />                                                                             study result precise <span id='am-243' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> medium <br />                                n                                            components (only approximations undefined media <br />                                    &#226;&#381;&#8250;       &#226;&#381;&#382; <br />                               &#226;&#710;&#8216; &#226;&#381;&#339;&#226;&#381;&#339;&#226;&#381;? ( n &#226;&#710;&#8217;i )!&#226;&#8249;&#8230;i! &#226;&#8249;&#8230; p &#226;&#8249;&#8230; q <br />                                          n! <br />           P ( X &#226;&#8240;&#165; r ) bin =                &#226;&#381;&#376;&#226;&#381;&#376; <br />                                                          q &#226;&#710;&#8217;i             possible). The similarity results aber model <br />                       i=r                   &#226;&#381;&#160;                               iND750 indicate additional complexity of <br /> where p = 0.82, proportion ORFs result                       FBA required prediction essential genes or <br /> experimental growth q = 1 - p (the proportion                        determination growth/retarded growth defined <br />  <br /> Table 5: Values metrics used validate models experimental comparisons using defined medium <br />  <br />                                 ORF Source             Aber        Shared      iND750        Shared           Aber      iND750           Shared <br />  <br />                                         Model              Aber      Aber          iND750     iNd750           maj           maj             maj <br />  <br />                                        No ORFs              940      641            681         641            940           681             641 <br />  <br />  Skewed Accuracy                        Exp A               0.79     0.79           0.80        0.80           0.77          0.77            0.76 <br />  Skewed Precision                       Exp A               0.80     0.80           0.83        0.82           0.77          0.77            0.76 <br />  Skewed Accuracy                        Exp B               0.81     0.82           0.80        0.79           0.82          0.83            0.82 <br />  Skewed Precision                       Exp B               0.83     0.84           0.86        0.85           0.82          0.83            0.82 <br />  Skewed Accuracy                        Exp C               0.83     0.85           0.84        0.83           0.82          0.83            0.82 <br />  Skewed Precision                       Exp C               0.83     0.85           0.85        0.84           0.82          0.83            0.82 <br />  <br />  <br />  <br />  <br />                                                                                                                                       Page 6 16 <br />                                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                      http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Table 6: McNemar test results aber model vs iND750 <br />  <br />  Comparison Set                                   Aber ORFs (940)              iND750 ORFs (681)              Shared ORFs (641) <br />  <br />  A                         result                        0.72                        0.60                              0.14 <br />                            significance                                                                           no <br />                            direction <br />  B                         result                        1.19                        2.93                              2.64 <br />                            significance                                                                           no <br />                            direction <br />  C                         result                        0.25                        1.05                              1.09 <br />                            significance                                                                           no <br />                            direction <br />  <br />  <br /> media. However logical model currently restricted             training examples incrementally, most <br /> to binary growth/no growth predictions capa-              informative example (given current hypothesis set) can <br /> bility predict growth rates unlike FBA based techniques.          chosen iteration loop. In scientific dis- <br /> It case horses courses. The logical model pre-           covery training examples correspond outcome of <br /> sented useful addition currently existing           experiments carried automatically lab robot, and <br /> models appropriate mix expressive representa-             active selection training example corresponds to <br /> tion metabolic concepts simple mechanism                choosing experiment likely refute largest <br /> determining predictive outcomes.                                     number hypotheses, given outcome experi- <br />                                                                      ment (growth growth). Figure 1 illustrates how <br /> The high number inaccuracies prediction                 active learning loop applied functional <br /> growth models indicates important                  genomics discovery task. <br /> information remains discovered yeast metab- <br /> olism.                                                               The formation hypotheses outcomes the <br />                                                                      selected experiments performed limited form of <br /> Using logical model background knowledge                  abductive theory completion [3], performed PROGOL, <br /> active learning hypotheses                                        ILP [36] program performs abductive and <br /> A major motivation use logical formalism             inductive machine learning restricted form FOL. In <br /> that enables exploitation techniques              task logical model formed incomplete back- <br /> large research area Inductive Logic Programming (ILP)             ground knowledge theory, final hypothesis is <br /> and related first-order learning methods [36]. This           added &quot;complete&quot; improve theory, therefore <br /> possible formalisms FBA. In Previous              adding current biological knowledge yeast func- <br /> work [1], precursor aber logical model used             tional genomics. The completion theory analo- <br /> as background theory (re)discovery task involving             gous identifying new edge added the <br /> the function ORFs single pathway (AAA Biosyn-              metabolic network, new ORF annotation for <br /> thesis) Yeast. The task implemented active              edge. This described Figure 2. <br /> learning loop, machine learning technique based on <br /> abductive logic programming mirrors hypo-                   The use restricted form FOL allows compo- <br /> thetico-deductive process hypothesis formation sci-            nents scientific discovery task described using <br /> entific discovery. In active learning classifier given set    knowledge representation, complex- <br />  <br /> Table 7: McNemar tests results aber model vs majority classifier <br />  <br />  Comparsion Set                                             Aber ORFs (940)                  Shared ORFs (641) <br />  <br />  A                                 result                   5.02                             8.51 <br />                                    significance             yes                              yes <br />                                    direction                Aber                             Aber <br />  B                                 result                   0.31                             0 <br />                                    significance                                           no <br />                                    direction <br />  C                                 result                   5.94                             11.17 <br />                                    significance             yes                              yes <br />                                    direction                Aber                             Aber <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 7 16 <br />                                                                                                      (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                   http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Table 8: McNemar results iND750 vs majority classifier <br />  <br />  Comparison Set                                                  iND750 ORFs (681)                  Shared ORFs (641) <br />  <br />  A                               result                                 5.01                                  5.69 <br />                                  significance                           yes                                   yes <br />                                  direction                            iND750                                iND750 <br />  B                               result                                 2.56                                  2.64 <br />                                  significance                                                               no <br />                                  direction <br />  C                               result                                34.38                                 134.31 <br />                                  significance                           yes                                   yes <br />                                  direction                            iND750                                iND750 <br />  <br />  <br /> ity entities relationships required accurate        designed enable automated reasoning about <br /> description enzymes, reactions, experiments,                yeast metabolism. Use model predict essenti- <br /> hypotheses expressed FOL equiv-           ality genes knockout studies provides mechanism <br /> alently expressive knowledge representation (e.g descrip-          used validate model respect to <br /> tion logics, OWL). Abduction hypotheses relating                experimental evidence. ROC analysis statistical <br /> ORFs, enzymes reactions requires                 tests carried evaluate predictive ability of <br /> machine learning program FOL it&apos;s representation            model compare performance state of <br /> language, use programs ILP. The Robot            art FBA model (iND750); evaluate effect of <br /> Scientist concept represents close integration         increasing coverage model iFF708. The <br /> of logical inference techniques used scientific discov-        results using logical model FBA <br /> ery automated laboratory perform experi-           based results significant loss performance, <br /> ments. The previous work AAA Biosynthesis                   loss enlarging coverage the <br /> pathway proof principle study. Recently orig-         model include information KEGG; both <br /> inal robot hardware replaced state-of-the-art        iND750 logical model represent improvement <br /> automated laboratory substantially increases              random predictions. <br /> throughput Robot Scientist [37] <br />                                                                    Methods <br /> Conclusion                                                         Sources logical model <br /> This paper presents logic based model metabolic           The Logical model Yeast metabolism con- <br /> processes S. cerevisiae: &quot;aber&quot;. The core aber           structed main sources: <br /> model FBA model iFF708 [6]. Added core are <br /> an additional 263 ORFs, includes additional            1) iFF708 &#226;&#8364;&#8220; Genome Scale Metabolic Network pro- <br /> 247 reactions taken KEGG [11,12]. The aber model              duced Forster et al [6] <br />  <br />  <br />  <br />  <br /> FigureLearning <br /> Active  1       Scientific Discovery <br /> Active Learning Scientific Discovery. <br />  <br />  <br />  <br />                                                                                                                       Page 8 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br />  <br /> Abductive <br /> Figure 2 Inference Graph Completion <br /> Abductive Inference Graph Completion. <br />  <br />  <br /> 2) The Kyoto Encyclopaedia Genes Genomes                 flux distributions shadow price calculations allowed <br /> (KEGG) [11,12]                                                  reaction list corrected (by hand) &#226;&#8364;&#8220; predic- <br />                                                                 tions final model agreement the <br /> iFF708 consists set reactions exists    experimental results. iFF708 validated by <br /> biochemical evidence S. cerevisiae set open         comparing growth list essential genes (see section <br /> reading frames (ORFs) encode gene products        2). <br /> catalyse reactions. The gene products belong to <br /> enzyme classes, defined Enzyme Commission. For-          Components structure logical model <br /> ster et al used online database sources (MIPS [38], KEGG        The logical model iFF708 include ORFs, <br /> [11,12], SGD [39], Expasy [40] etc) literature sources      enzymes reactions yeast metabo- <br /> to construct network. Forster et al [35] estimate 1    lism. An ORF corresponds DNA sequence is <br /> man year research activity required identify       removed replaced create knockout mutant. <br /> reactions corresponding metabolic network           Enzymes types Enzyme Commission number used <br /> single organism. In constructing network considera-         classify gene products corresponding particular <br /> tion given presence, stoichiometry, co-factors,      enzyme function, reactions individual <br /> reversibility localisation reactions S. cerevisiae.   chemical transformation steps that, taken together <br /> The model building phase required itera-         represent yeast metabolism. In original form iFF708 <br /> tions improvements. ORFs reactions iFF708            treats relationships components if <br /> are localised various compartments corresponding        corresponded single relational database table, <br /> (some of) organelles yeast cell. There 2 com-    resulting duplication components. For exam- <br /> partments &#226;&#8364;&#8220; 3 immediate external environment (e.g.       ple, isoenzymes, gene products two <br /> growth medium) thought compartment.            unique ORFs catalyse reaction, reac- <br /> The internal compartments cytosol mito-         tion repeated table times there <br /> chondrion. The resulting reaction list included reactions       isoenzymes. This lead confusion defining <br /> confined cell compartments transport         various unique entities components. Adapting <br /> reactions membranes external envi-           iFF708 logical model enables separation enti- <br /> ronment cytosol cytosol mitochon-           ties components linked database tables, <br /> drion.                                                          natural way represent components <br />                                                                 relationships. The current logical model repre- <br /> Forster et al used flux balance analysis simulate var-   sents step &#226;&#8364;&#8220; reactions placed sep- <br /> ious phenotypic metabolic behaviours. These                 arate table allowing reaction defined uniquely, <br /> included models capability manufacture key pre-          links ORF/enzyme components <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>/cat- <br /> cursor metabolites amino acids, ATP         alyse reaction. However, remains work to <br /> related costs synthesising macromolecules ulti-          extend representation allow addi- <br /> mately biomass precursors. The method           tional complexities multimeric reactions pro- <br /> used validate model: comparing predicted          tein complexes [5]. Table 9 catalogue various <br /> results previous experimental results aerobic       components relations logical model. The <br /> anaerobic chemostat cultivation, investigation       ORF/Enzyme/Reaction relation logical equivalent of <br />  <br />  <br />                                                                                                                     Page 9 16 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                 http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> the single table iFF708. It seen adding       ment {external, cytosol, mitochondrion}. The <br /> information KEGG added additional 688                Orf/Enzyme/Reaction relations corresponding aber <br /> such relationships. The 1,166 ORFs logical model          logical model available Additional file 1 sup- <br /> include 226 &quot;unknown&quot; ORFs, reactions            plementary information paper. The reactions are <br /> there biochemical evidence included             available Additional file 2. <br /> iFF708, ORF/enzyme information com- <br /> plete (this explains total ORF count 940 used          The reaction mechanism <br /> model validation &#226;&#8364;&#8220; section 2).                                   The Reaction Mechanism model used predict out- <br />                                                                  come knockout mutant experiments essentially the <br /> Representation ORFs, enzymes reactions                    mechanism used [3]. To relate model <br /> Two sets relations lie core logical represen-   observable phenotype associate existence of <br /> tation metabolic network:                                 paths graph growth, i.e. (iff) <br />                                                                  (deduce) path input metabolites the <br /> 1. orf_fact(ORF,ECNumber,EnzymeClass,GeneName,                   growth medium set essential metabolites <br />                                                                  (amino acids, nucleic acid precursors, polysaccharide <br />  GeneDescription,ReactionNumber).                                precursors, lipids, etc.) model predicts the <br />                                                                  yeast cell grow indistinguishably wild type <br /> This relation corresponds single mapping            (observable phenotype). The logical modelling approach <br /> one ORF, Enzyme Commission number (also pre-                 simpler FBA, involves concept of <br /> sented relation traversing hierarchy EC          connection (true/false) flux &#226;&#8364;&#8220; removes <br /> numbers); corresponding Gene name; (brief)                 requirement estimate fluxes. <br /> description function gene; reaction <br /> catalysed gene product. The second relation defines       Reactions processed checking metabo- <br /> the reactions logical model:                              lites reaction present cell <br />                                                                  adding metabolites side. Each reac- <br /> 2. reaction(ReactionNumber,Substrate,Direction,Prod-             tion processed reactions exe- <br /> uct).                                                            cuted final complement cell compounds was <br />                                                                  used evaluate growth outcome particular <br /> where ReactionNumber unique number             growth experiment. This basic approach been <br /> reaction identified, Direction &quot;forwards&quot; (-&gt;)   altered incorporate cell compartments used the <br /> &quot;reversible&quot; (&lt;-&gt;) Substrate Product lists        iFF708. KEGG does specify cell compartments, hence <br /> metabolites cell compartment         network components added KEGG assumed <br /> metabolite added to. The stoichiometry        located cytosol. Figure 4 illustrates the <br /> reaction recorded, reaction mecha-          reaction mechanism used predict outcome <br /> nism used predict outcome experiments does         knockout experiments. The inputs model <br /> currently make use information. Figure 3 uses          points are: <br /> simple example Glycolysis illustrate relation- <br /> ships logical model. Each Metabolite stored       1) The metabolites comprising minimal growth <br /> &quot;reactant&quot;:                                                      medium MMD+ura+hist+leu. These minimum set <br />                                                                  compounds required wild yeast continued <br /> reactant(Compartment,Stiochiometry,CompoundID)                   growth &#226;&#8364;&#8220; additional nutritional require- <br />                                                                  ments Uracil, L-Histidine L-Leucine <br /> where CompoundID unique ID KEGG <br /> database &#226;&#8364;&#8220; conversion notation used            2) The ORF(s) knocked form mutant strain <br /> the iFF708 possible (&gt;90% metabolites). Compart- <br />                                                                  3) Any additional nutrients added identify possible <br /> Table 9: Components Logical Model <br />                                                                  reactions removed metabolic network knock- <br />                                                                  ing ORF. <br />  Component                                          Size <br />                                                                  4) Metabolites deemed &quot;ubiquitous&quot; &#226;&#8364;&#8220; deemed be <br />  ORF/Enzyme/Reaction relations                      2303         present cell. These include co-factors as <br />  ORFs                                               1166         NADPH, NADH, ATP, ADP etc. The list ubiquitous <br />  EC Classes                                         541 <br />                                                                  compounds given Table 10. Ubiquitous compounds <br />  Reactions                                          1087 <br />  Metabolites                                        821          present reactions growth simula- <br />                                                                  tion algorithm requires presence cell the <br />  <br />  <br />                                                                                                                    Page 10 16 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                  http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br />  <br /> Figure 3 <br /> Representation Metabolic Information: An Example Glycolysis <br /> Representation Metabolic Information: An Example Glycolysis. A: reactions Glycolysis, B: representations pro- <br /> log. <br />  <br />  <br />  <br /> start guarantee connectivity wild-type meta-       product list added compartment <br /> bolic network. In biological terms compounds           specified. <br /> present yeast cell separates the <br /> &quot;mother&quot; cell.                                                   2) Reversible reactions processed similarly irrevers- <br />                                                                  ible reactions, processed reverse <br /> 5) The reactions ORFs/enzymes defined                  direction i.e. iff reactants products list found <br />                                                                  correct compartments, reactants substrate <br /> Lists corresponding compartments defined         list added specified compartments. <br /> the iFF708 created: lists corresponding the <br /> cytosol mitochondrion compartments initially set         Each reaction processed turn processed most <br /> to contain minimal medium compounds,                 once. Currently logical model simply records pres- <br /> ubiquitous compounds. The task reaction             ence absence metabolite calculation the <br /> mechanism identify reactions longer        quantities produced undertaken. This approach suffi- <br /> proceed uniquely catalysed gene          cient simple continued/arrested growth predic- <br /> products produced ORF(s) removed create            tions currently generated, revised the <br /> knockout strain. These reactions removed            model modified generate quantitative growth predic- <br /> network processing reaction mechanism              tions. The mechanism processes reactions cyclical <br /> proper. The remaining reactions analysed turn          manner long 1 reactions previous <br /> see following conditions allowing reac-          cycle executed. This process ensures possible <br /> tion proceed met:                                   reactions processed metabolites as <br />                                                                  possible added cell compartment. <br /> 1) Irreversible reactions proceed iff (if if) all <br /> reactants substrate present correct cell       After processing reaction mechanism, determina- <br /> compartments. When reaction runs, reactants              tion continued arrested growth compar- <br />                                                                  ing final complement metabolites list of <br />  <br />  <br />                                                                                                                      Page 11 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br />                                                                variant resulted continued growth, i.e. logical <br />                                                                model contain reactions required the <br />                                                                essential compounds added cell, ubiq- <br />                                                                uitous compounds contain co-factors required <br />                                                                reactions. However, including components found <br />                                                                iFF708 sufficient predict growth net- <br />                                                                work did include reactions enable synthesis all <br />                                                                compounds deemed essential. However, is <br />                                                                aber model focussed modelling via- <br />                                                                bility metabolism. This means end-points are <br />                                                                metabolites considered essential life. The end- <br />                                                                points flux balance analysis model metabolites <br />                                                                important cell biomass. The sets end- <br />                                                                point metabolite necessarily same. <br />                                                                For example, clear cell able <br />                                                                synthesise immediate nucleotide precursors RNA <br />                                                                DNA, end-points iFF708 <br />                                                                model. Inclusion knowledge representing additions <br />                                                                yeast biology KEGG database added subse- <br />                                                                quent completion iFF708; addi- <br />                                                                tional reaction, KEGG, documented in <br />                                                                yeast, essential production Palmitate, <br />                                                                allowed graph traversal algorithm reach essential <br />                                                                compounds. (Note &#226;&#8364;&#8220; &quot;palmitate reaction&quot; now <br />                                                                occur yeast, KEGG [11,12]). Construc- <br />                                                                tion model complete reactions neces- <br />                                                                sary simulate continued growth wild variant had <br />                                                                added model. <br /> FigureThe <br /> trophic <br /> Using   Experiments <br />         4 Logical Model <br />                     In S.to <br />                           cerevisiae <br />                             Predict Outcome Auxo- <br /> Using The Logical Model Predict Outcome Auxo-        Description KEGG components used augment IFF708 <br /> trophic Experiments In S. cerevisiae. <br />                                                                Table 12 shows numbers ORFs reactions <br />                                                                belonging KEGG pathways ORF/enzyme/ <br />                                                                reaction relations used augment IFF708 model, as <br />  <br /> compounds deemed essential healthy cell growth:            Table 10: Compounds deemed Ubiquitous S. cerevisiae <br /> these include amino acids, nucleic acids, compounds such <br /> as D-Glucose 1-phosphate essential forma-       Kegg Compound ID               Compound Name <br /> tion polysaccharides, compounds choline that <br />                                                                 C00005                         NADPH <br /> are essential formation membranes, impor- <br />                                                                 C00004                         NADH <br /> tant intermediary compounds pyruvate. The com-          C00016                         FAD <br /> plete list essential compounds given Table 11. The     C00006                         NADP+ <br /> growth status simulated experiment determined         C00003                         NAD+ <br /> as follows:                                                     C00061                         FMN <br />                                                                 C00575                         3&apos;5&apos;-Cyclic AMP <br /> 1) Continued growth predicted iff &#226;&#710;&#8364;(essential com-           C00399                         ubiquinone <br />                                                                 C00137                         myo-inositol <br /> pounds present Cytosol compartment the <br />                                                                 C00194                         coenzyme B12 <br /> model)                                                          C00032                         protoheme <br />                                                                 C00255                         riboflavin <br /> 2) Arrested growth predicted &#226;&#710;&#402;(essential compound         C00346                         phosphoethanolamine <br /> missing Cytosol compartment)                           C00641                         1 2-Diacyglycerol <br />                                                                 C00448                         farnesyl pyrophosphate <br /> Growth wild variant additional nutrients         C00002                         ATP <br />                                                                 C00008                         ADP <br /> can predicted giving knockout value &quot;none&quot; and <br />                                                                 C00342                         Thioredoxin <br /> an list additional nutrients. To create usa-    C00007                         O2 <br /> ble logical model crucial simulation wild <br />  <br />  <br />                                                                                                                    Page 12 16 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                    http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Table 11: Compounds deemed essential healthy growth S. cerevisiae <br />  <br />        Kegg Compound ID                             Compound Name                            Compound type/function <br />  <br />               C00041                                       L-Alanine                                 amino acid <br />               C00037                                        Glycine                                  amino acid <br />               C00079                                  L-Phenylalanine                                amino acid <br />               C00078                                    L-Tryptophan                                 amino acid <br />               C00082                                      L-Tyrosine                                 amino acid <br />               C00407                                     L-Isoleucine                                amino acid <br />               C00073                                    L-Methionine                                 amino acid <br />               C00062                                      L-Arginine                                 amino acid <br />               C00049                                     L-Aspartate                                 amino acid <br />               C00135                                      L-Histidine                                amino acid <br />               C00097                                      L-Cysteine                                 amino acid <br />               C00025                                     L-Glutamate                                 amino acid <br />               C00064                                     L-Glutamine                                 amino acid <br />               C00047                                        L-Lysine                                 amino acid <br />               C00148                                       L-Proline                                 amino acid <br />               C00065                                       L-Serine                                  amino acid <br />               C00188                                     L-Threonine                                 amino acid <br />               C00183                                        L-Valine                                 amino acid <br />               C00152                                    L-Asparagine                                 amino acid <br />               C00123                                      L-Leucine                                  amino acid <br />               C00242                                       Guanine                                      <span id='am-242' about='xsp:base' typeof='owl:Thing'>base</span> <br />               C00106                                         Uracil                                     base <br />               C00147                                        Adenine                                     base <br />               C00262                                   Hypoxanthine                                     base <br />               C00178                                       Thymine                                      base <br />               C00380                                       Cytosine                                     base <br />               C00212                                      Adenosine                                  nucleoside <br />               C00387                                      Guanosine                                  nucleoside <br />               C00294                                        Inosine                                  nucleoside <br />               C00214                                      Thymidine                                  nucleoside <br />               C00299                                        Uridine                                  nucleoside <br />               C00475                                       Cytidine                                  nucleoside <br />               C00020                                          AMP                                    nucleotide <br />               C00144                                         GMP                                     nucleotide <br />               C00130                                          IMP                                    nucleotide <br />               C00105                                          UMP                                    nucleotide <br />               C00055                                          CMP                                    nucleotide <br />               C00002                                          ATP                                  energy transfer <br />               C00044                                          GTP                                  energy transfer <br />               C00075                                          UTP                                  enery transfer <br />               C00005                                       NADPH                                      coenzyme <br />               C00004                                        NADH                                      coenzyme <br />               C00016                                          FAD                                     coenzyme <br />               C00008                                          ADP                                  energy transfer <br />               C00035                                          GDP                                  energy transfer <br />               C00015                                          UDP                                  energy transfer <br />               C00068                                Thiamin diphosphate                               coenzyme <br />               C03028                                Thiamin triphosphate                              coenzyme <br />               C00006                                        NADP+                                     coenzyme <br />               C00003                                         NAD+                                     coenzyme <br />               C00061                                         FMN                                      coenzyme <br />               C00575                                  3&apos; 5&apos;-Cyclic AMP                             energy transfer <br />               C00002                                          ATP                                    nucleic acid <br />               C00044                                          GTP                                    nucleic acid <br />               C00063                                          CTP                                    nucleic acid <br />               C00075                                          UTP                                    nucleic acid <br />               C00131                                         dATP                                    nucleic acid <br />               C00286                                         dGTP                                    nucleic acid <br />               C00458                                         dCTP                                    nucleic acid <br />  <br />  <br />  <br />                                                                                                                      Page 13 16 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                       http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br /> Table 11: Compounds deemed essential healthy growth S. cerevisiae (Continued) <br />               C00459                                         dTTP                                        nucleic acid <br />               C00103                               D-Glucose 1-phosphate                               polysaccharide <br />               C00043                             UDP-N-acetyl-D-glucosamine                            polysaccharide <br />               C00096                                    GDPmannose                                     polysaccharide <br />               C00114                                        choline                                      membrane <br />               C00157                                        lecithin                                     membrane <br />               C00416                                    phosphatidate                                    membrane <br />               C00422                                    triacylglycerol                                  membrane <br />               C01694                                      ergosterol                                     membrane <br />               C00189                                    ethanolamine                                     membrane <br />               C00116                                        glycerol                                     membrane <br />               C00137                                        inositol                                     membrane <br />               C01120                               sphinganine 1-phosphate                               membrane <br />               C00668                             alphaD-glucose 6-phosphate                             intermediate <br />               C00022                                       pyruvate                                     intermediate <br />               C00024                                     acetyl-CoA                                     intermediate <br />               C00356                           3-Hydroxy-3-methyl-glutaryl CoA                          intermediate <br />  <br />  <br /> well reactions relations KEGG             existing datasets. The method intrinsic meth- <br /> have pathway assigned. The reactions relations                ods identification, models gen- <br /> added KEGG inferred translation              erated improved machine learning/statistics <br /> LIGAND component KEGG, reactions corre-                     techniques e.g. equation discovery computational sci- <br /> sponding ORF/Enzyme relationships                    entific discovery. <br /> IFF708 collected. (These 247 reactions dis- <br /> cussed section 4). This augmentation iFF708                It possible use second method obtain some <br /> undertaken reactions iFF708            measurements validity metabolic models yeast <br /> were insufficient synthesise essential com-            comparing predicted outcomes auxotrophy <br /> pounds listed Table 11. There 318 ORFs involved            experiments results genome gene-dele- <br /> the 688 additional ORF/enzyme/reaction relations;                 tion study undertaken Giaever et al [4]. This study <br /> these 55 iFF708. 37 ORFs reactions             involved repeats competitive batch experiments <br /> added KEGG reactions defined                    knockout strains grown agar base <br /> iFF708, i.e. ORFs known EC number                  supplemented number defined unde- <br /> reaction corresponding EC number               fined growth media. Measurements growth were <br /> biological knowledge subsequent completion                 obtained 5 15 generations presented as <br /> IFF708.                                                              fitness comparisons growth average <br />                                                                      strain rich undefined medium (YPD &#226;&#8364;&#8220; Yeast Extract, <br /> The relations corresponding 247 additional reac-              Peptone Dextrose), strains growing slower <br /> tions belong 47 pathways described KEGG,               average defined sensitive growing <br /> the largest proportion relations correspond              faster refractory. Growth data particular strain <br /> reactions included KEGG pathways. Purine             corresponds relative abundance strain-specific <br /> metabolism (33 reactions), Fatty acid biosynthesis (path             DNA tags hybridisation custom high density <br /> 1) (32 reactions) Pyrimidine metabolism (29 reac-                oligonucleotide array, form continuously <br /> tions) largest numbers reaction assignments;             valued relative fitness scores, expressed &quot;units <br /> Purine metabolism (71 ORFs), Pyrimidine metabolism                   deleterious growth&quot; [4]. There 4 sensitive/refractory <br /> (68 ORFs) RNA polymerase (31 ORFs) larg-                measurements strain/growth medium combina- <br /> est numbers ORF assignments. There simple rela-             tion. Giaever et al used sensitivity score cut &gt;20 <br /> tionship numbers ORFs pathway                units deleterious growth identify strains sig- <br /> the number reactions, e.g. 32 new reactions added             nificant deleterious phenotype 5 generations, rising <br /> to Fatty acid biosynthesis (path 1) pathway              &gt;100 15 generations, reflecting observation that <br /> coded single ORF.                                           strains sensitive earlier represent genes <br />                                                                      important cell growth. Giaever et al [4] <br /> Model validation                                                     compiled list essential strains did grow on <br /> Model validation [41] vital stage construction           YPD. Model validation assessed models&apos; per- <br /> of model. This 1) direct experi-           formance predictions gene essentiality on <br /> mentation model determines rel-              comprehensive analysis growth predictions <br /> evant experiments (preferably), 2) comparison             combine essential gene listings results <br />  <br />  <br />                                                                                                                         Page 14 16 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                 http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br /> Table 12: KEGG Pathways total numbers ORFs reactions added aber model KEGG included IFF708 <br />  <br /> Kegg Pathway                                                           Number Reactions               Number ORFs <br />  <br /> Citrate cycle (TCA cycle)                                                        8                                12 <br /> Tryptophan metabolism                                                            5                                 2 <br /> Bile acid biosynthesis                                                          1                                  2 <br /> Fatty acid biosynthesis (path 1)                                                32                                 1 <br /> One carbon pool folate                                                       1                                  1 <br /> Biotin metabolism                                                               5                                  1 <br /> Selenoamino acid metabolism                                                     10                                 3 <br /> Folate biosynthesis                                                             2                                  1 <br /> Purine metabolism                                                               33                                71 <br /> Pentose phosphate pathway                                                        1                                 1 <br /> Nicotinate nicotinamide metabolism                                          2                                  1 <br /> Fatty acid metabolism                                                           4                                  1 <br /> Lysine degradation                                                              7                                  2 <br /> Propanoate metabolism                                                           1                                  1 <br /> Glycine, serine threonine metabolism                                        17                                12 <br /> C5-Branched dibasic acid metabolism                                             6                                 2 <br /> Pyruvate metabolism                                                              8                                 5 <br /> Methionine metabolism                                                           10                                 3 <br /> Alanine aspartate metabolism                                                4                                  4 <br /> Ubiquitin mediated proteolysis                                                   1                                14 <br /> Sulfur metabolism                                                               8                                  1 <br /> Nitrogen metabolism                                                             2                                  2 <br /> Glutamate metabolism                                                             4                                 4 <br /> Aminosugars metabolism                                                          2                                  1 <br /> N-Glycans biosynthesis                                                           8                                14 <br /> Starch sucrose metabolism                                                   3                                 4 <br /> Pentose glucuronate interconversions                                         2                                 1 <br /> Glycerolipid metabolism                                                         10                                 3 <br /> gamma-Hexachlorocyclohexane degradation                                         7                                  7 <br /> Phenylalanine, tyrosine tryptophan biosynthesis                             2                                  5 <br /> No Pathway Annotation                                                           50                               153 <br /> RNA polymerase                                                                   5                                31 <br /> Lysine biosynthesis                                                             5                                 1 <br /> Oxidative phosphorylation                                                       3                                26 <br /> Butanoate metabolism                                                            10                                 5 <br /> Fructose mannose metabolism                                                 1                                  1 <br /> Pyrimidine metabolism                                                           29                                68 <br /> 1,4-Dichlorobenzene degradation                                                 3                                  1 <br /> Phosphatidylinositol signaling                                            1                                10 <br /> DNA polymerase                                                                   5                                18 <br /> Porphyrin chlorophyll metabolism                                             2                                 1 <br /> Arginine proline metabolism                                                 1                                 2 <br /> Cysteine metabolism                                                             11                                 3 <br /> Galactose metabolism                                                            1                                  1 <br /> Pantothenate CoA biosynthesis                                                6                                 2 <br /> Aminoacyl-tRNA biosynthesis                                                     18                                27 <br /> Glycolysis / Gluconeogenesis                                                    12                                 8 <br /> Valine, leucine isoleucine biosynthesis                                     13                                10 <br /> Riboflavin metabolism                                                           5                                  7 <br />  <br />  <br /> from relevant defined medium. This min-          Authors&apos; contributions <br /> imal medium (MMD+ura+hist+leu), contains              RDK initiated logical modelling concept con- <br /> minimum sets compounds required wild type                structed prototype model. KW constructed logical <br /> growth additional nutritional requirements       model carried validation experiments the <br /> uracil, histidine leucine.                                  statistical tests. RDK KW wrote revised manu- <br />                                                                 script. <br />  <br />  <br />  <br />                                                                                                                    Page 15 16 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:97                                                                       http://www.biomedcentral.com/1471-2105/9/97 <br />  <br />  <br />  <br />  <br /> Additional material                                                               18.   Tomita M, Hashimoto K, Takahashi K, Shimuzu TS, Matsuzaki Y, Miy- <br />                                                                                         oshi F, Saito K, Tanida S, Yugi K, Venter JC, Hutchinson JC: E-CELL: <br />                                                                                         software environment whole-cell simulation. Bioinformatics <br />                                                                                         1999, 15:72-84. <br />      Additional file 1                                                            19.   Edwards JS, Palsson BO: Metabolic flux balance analysis the <br />      ORF, Enzyme Reaction Relations. This file contains Prolog facts cor-           silico analysis Escherichia coli K-12 gene deletions. BMC <br />      responding relationships ORFs, Enzyme Commission                    Bioinformatics 2000, 1:1. <br />      Numbers Reactions Logical Model Yeast Metabolism.             20.   Mahadevan R, Edwards JS, Doyle FJ: Dynamic Flux Balance Anal- <br />                                                                                         ysis Diauxic Growth Escherichia coli. Biophysical Journal <br />      Click file <br />                                                                                         2002, 83:1331-1340. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                    21.   Duarte ND, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, <br />      2105-9-97-S1.PL]                                                                   Palsson BO: Global Reconstruction Human Metabolic <br />                                                                                         Network based Genomic Bibliomic Data. Proc Natl <br />      Additional file 2                                                                  Acad Sci USA 2007, 104(6):1777-82. <br />                                                                                   22.   Kauffman KJ, Prakash P, Edwards JS: Advances flux balance <br />      Reactions. This file contains Prolog facts corresponding Substrate          analysis. Current Opinion Biotechnology 2003, 14:491-496. <br />      Product information Reactions Logical Model Yeast Metabo-      23.   Segre D, Vitkuo D, Church GM: Analysis optimality natural <br />      lism.                                                                              perturbed metabolic networks. Proc Natl Acad Sci USA 2002, <br />      Click file                                                                99(23):15112-15117. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                    24.   Shlomi T, Berkman O, Ruppin E: Regulatory on/off minimization <br />                                                                                         metabolic flux changes genetic perturbations. Proc <br />      2105-9-97-S2.PL]                                                                   Natl Acad Sci USA 2005, 102(21):7695-7700. <br />                                                                                   25.   Stelling J, Klamt S, Bettenbrock K, Schuster S, Gilles ED: Metabolic <br />                                                                                         network structure determines key aspects functionality <br />                                                                                         regulation. Nature 2002, 420(6912):190-193. <br />                                                                                   26.   Lemke N, Heredia F, Barcellos CK, Dor Reis AN, Mombach JC: <br /> References                                                                              Essentiality damage metabolic networks. Bioinformatics <br /> 1.      King RD, Whelan KE, Jones FM, Reiser PJK, Bryant CH, Muggleton S,               2004, 20(1):115-119. <br />         Kell DB, Oliver S: Functional Genomic Hypothesis Generation               27.   Lemke N, Heredia F, Barcellos CK, Mombach JC: A method to <br />         Experimentation Robot Scientist. Nature 2004,                          identify essential enzymes metabolism: application to <br />         427:247-252.                                                                    Escherichia Coli. Computational Methods Systems Biology: First <br /> 2.      Bryant CH, Muggleton SH, Oliver SG, Kell DB, Reiser PGK, King RD:               International Workshop 2003:142-148. (Book chapter) <br />         Combining inductive logic programming, active learning,                   28.   Bratko I: Prolog Programming Artificial Intelligence (Addison Wesley) <br />         robotics discover function genes. Electronic Trans-               1986. <br />         actions Artificial Intelligence 2001, 6(12):1-36.                      29.   Fages F, Soliman S, Chabrier-Rivier N: Modelling querying <br /> 3.      Reiser PGK, King RD, Kell DB, Muggleton SH, Bryant CH, Oliver SG:               interaction networks biochemical abstract machine <br />         Developing logical model yeast metabolism. Electronic                      BIOCHAM. Journal Biological Physics Chemistry 2004, 4:64-73. <br />         Transactions Artificial Intelligence 2001, 5:223-244.                  30.   Gershenson C: Classification Random Boolean Networks in <br /> 4.      Giaever G: Functional profiling Saccharomyces cerevisiae                 Artificial Life VIII Standish, Abbas, Bedau. MIT Press; 2002. <br />         genome. Nature 2002, 418:387-391.                                         31.   Kauffman S, Peterson C, Samuelsson B, Troein C: Random Boolean <br /> 5.      Duarte NC, Herrgard MJ, Pallson BO: Reconstruction Valida-                   network models yeast transcriptional network. Proc <br />         tion Saccharomyces cerevisiae iND750, Fully Compart-                       Natl Acad Sci USA 2003, 100(25):14796-14799. <br />         mentalized Genome-Scale Metabolic Model. Genome Res                       32.   Kuipers B: Qualitative Reasoning MIT Press; 1994. <br />         2004, 14:1298-1309.                                                       33.   King RD, Garrett SM, Coghill GM: On use qualitative rea- <br /> 6.      F&#195;&#182;rster J, Famili I, Fu P, Palsson BO: Nielsen J. Genome-Scale                  soning simulate identify metabolic pathways. Bioinfor- <br />         Reconstruction Saccharomyces cerevisiae Metabolic                        matics 2005, 21(9):2017-2026. <br />         Network. Genome Research 2003, 13:244-253.                                34.   Flach PA: The Geometry ROC Space: Understanding <br /> 7.      Kitano H, Systems Biology: Systems Biology: A Brief Overview.                   Machine Learning Metrics ROC Isometrics. Proceed- <br />         Science 2002, 295:1662-1664.                                                    ings ICML03 2003. <br /> 8.      Csete ME, Doyle JC: Reverse Engineering Biological Com-                35.   F&#195;&#182;rster J, Famili I, Fu P, Palsson BO: Large Scale Evaluation in- <br />         plexity. Science 2002, 295:1664-1669.                                           silico gene knockouts Saccharomyces cerevisiae Omics. <br /> 9.      Chong L, Ray LB: Whole-istic Biology. Science 2002, 295:1661.                   2003, 7:193-202. <br /> 10.     Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh C,           36.   Muggleton S, DeRaedt L: Inductive Logic Programming: Theory <br />         Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, Brown                     Methods. Journal Logic Programming 1994, 19,20:629-679. <br />         CT, Livi CB, Lee PY, Revilla R, Rust AG, Zheng JP, Schilstra MJ, Clarke   37.   Robot Scientist           [http://www.aber.ac.uk/compsci/Research/bio/ <br />         PJC, Arnone MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri                 robotsci/] <br />         H: A Genomic Regulatory Network Development. Science                  38.   Mewes HW, Dietmann S, Frishman D, Gregory R, Mannhaupt G, <br />         2002, 295:1669-1678.                                                            Mayer K, Muensterk&#195;&#182;tter M, Ruepp A, Spannagl M, Stuempflen V, Rat- <br /> 11.     Kanehisa M, Goto S: KEGG: Kyoto Encyclopedia Genes                       tei T: MIPS: Analysis annotation genome information <br />         Genomes. Nucleic Acids Res 2000, 28:27-30.                                      2007. Nuc Acids Res 2008:D196-201. <br /> 12.     Kanehisa M: A database post-genome analysis. Trends Genet             39.   Dwight SS, Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, <br />         1997, 13:375-376.                                                               Engel SR, Feierbach B, Fisk DG, Hirschman J, Hong EL, Issel-Tarver L, <br /> 13.     Karp PD, Riley M, Paley SM, Pellegrini-Toole , Krummenacker M:                  Nash RS, Sethuraman A, Starr B, Theesfeld CL, Andrada R, Binkley G, <br />         EcoCyc:Encyclopedia Es <br /> </body></html>