<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 10, 2905&#226;&#8364;&#8220;2918, September 1999 <br />  <br />  <br /> Rad18 Is Required DNA Repair Checkpoint <br /> Responses Fission Yeast <br /> Heather M. Verkade,*&#226;&#8364;&#160; Sarah J. Bugg,* Howard D. Lindsay,&#226;&#8364;&#161; Anthony M. <br /> Carr,&#226;&#8364;&#161; Matthew J. O&#226;&#8364;&#8482;Connell*&#226;&#8364;&#160;&#194;&#167; <br />  <br /> *Trescowthick Research Laboratories, Peter MacCallum Cancer Institute, Melbourne, Victoria 8006, <br /> Australia; &#226;&#8364;&#160;Department Genetics, University Melbourne, Parkville, Victoria 3052, Australia; and <br /> &#226;&#8364;&#161; <br />   Medical Research Council Cell <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>Mutation</span> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Sussex University, Falmer, East Sussex BN1 9RR, <br /> United Kingdom <br />  <br /> Submitted February 8, 1999; Accepted June 14, 1999 <br /> Monitoring Editor: Mitsuhiro Yanagida <br />  <br />  <br />  <br />           <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> survive damage genome, cells respond activating DNA repair and <br />           checkpoint responses. Using genetic screens fission yeast Schizosaccharomyces pombe, we <br />           recently isolated new genes required DNA damage checkpoint control. We one <br />           strains defines new allele previously described rad18 gene, rad18-74. rad18 an <br />           essential gene, absence extrinsic DNA damage. It encodes conserved protein <br />           related structural maintenance chromosomes proteins. Point mutations rad18 lead to <br />           defective DNA repair pathways responding UV-induced lesions and, here, <br />           double-stranded breaks. Furthermore, rad18p required maintain cell cycle arrest the <br />           presence DNA damage, failure leads highly aberrant mitoses. A gene encoding <br />           BRCT-containing protein, brc1, isolated allele-specific high-copy suppressor of <br />           rad18-74. brc1 required mitotic fidelity cellular viability strains rad18 <br />           mutations essential DNA damage responses. Mutations rad18 brc1 are <br />           synthetically lethal topoisomerase II mutant (top2-191), indicating proteins play <br />           role chromatin organization. These studies role chromatin organization the <br />           maintenance activation responses DNA damage. <br />  <br />  <br /> INTRODUCTION                                                           mitosis. The activation p34cdc2/cyclinB rate-limiting <br />                                                                        step entry mitosis. Analysis G2 DNA damage <br /> Strict control fidelity cell cycle events important       checkpoint fission yeast identified genes that <br /> for cells prevent heritable damage changes ploidy.           essential DNA damage checkpoint (Al-Khodairy <br /> The importance control manifest occurrence           Carr, 1992; Enoch et al., 1992; Walworth et al., 1993; <br /> of cell cycle checkpoints. Checkpoint pathways maintain            Al-Khodairy et al., 1994; Saka et al., 1997; Willson et al., 1997). <br /> interdependency cell cycle transitions pause cell cycle         This checkpoint arrests cell cycle maintenance <br /> progression cellular defects detected. Checkpoints            inhibitory Y15 phosphorylation p34cdc2, is <br /> have described monitor DNA damage, completion                achieved chk1p-dependent signaling wee1p and <br /> of DNA replication, spindle integrity, cell mass, delay-           cdc25p (Furnari et al., 1997; O&#226;&#8364;&#8482;Connell et al., 1997; Peng et al., <br /> ing cell cycle progression requirements contin-          1997; Rhind et al., 1997; Sanchez et al., 1997). Chk1p becomes <br /> uation cell cycle met (Nurse, 1975, 1994; Hartwell          phosphorylated DNA damage, dependent the <br /> and Weinert, 1989; Murray, 1995).                                      checkpoint rad genes, function upstream chk1 <br />   Entry mitosis controlled activity              (Walworth Bernards, 1996). Similar signaling pathways <br /> cdk/cyclin complex p34cdc2/cyclinB (Nurse, 1990). This                 appear activated blocks DNA replication where <br /> complex accumulates G2 maintained                  kinase, cds1p, involved transducing <br /> inactive state phosphorylation tyrosine 15              signals cell cycle machinery (Boddy et al., 1998; Lind- <br /> (Y15) p34cdc2 wee1p related kinases. p34cdc2 be-             say et al., 1998; Zeng et al., 1998). <br /> comes active Y15 dephosphorylated cdc25p                   Maintenance genomic integrity requires DNA be <br /> family phosphatases, cell proceeds                held highly structured proteinaceous complexes called <br />                                                                        chromatin. Chromatin undergoes structural alterations co- <br />   &#194;&#167; <br />       Corresponding author. E-mail address: m.oconnell@pmci.unimelb.   incident DNA replication chromosome segrega- <br />       edu.au.                                                          tion, proteins involved maintaining or <br />  <br />  <br /> &#194;&#169; 1999 The American Society Cell Biology                                                                                      2905 <br />  H.M. Verkade et al. <br />  <br />  <br /> altering chromatin structure essential stability        G2 mitosis, addition role sister <br /> integrity genome (for review, Heck, 1997).               chromatid resolution (Uemura et al., 1987; Holm et al., 1989). <br /> Throughout S G2 phases, sister chromatids associate                The SMC1&#226;&#8364;&#8220; 4 proteins central chromatin <br /> tightly lengths. Perturbation sister chromatin       organization. A class proteins structurally <br /> cohesion results chromosome instability chromosome           related SMC1&#226;&#8364;&#8220; 4 defined rad18p fission yeast and <br /> breaks cause inability repair double-stranded         Rhc18p budding yeast (Lehmann et al., 1995). Rad18 an <br /> breaks (Birkenbihl Subramani, 1992; Strunnikov et al.,          essential gene. Its role inferred <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> the <br /> 1993; Guacci et al., 1997; Michaelis et al., 1997; Losada et al.,   point mutant rad18-X, shows sensitivity ionizing <br /> 1998). In Xenopus, 9S 14S protein complexes, cohesins,          UV-C irradiation, inability repair UV-C-induced <br /> are required generate maintain cohesion (Losada         lesions, chromosomal instability accompanied accu- <br /> et al., 1998). Cohesins contain structural maintenance       mulation mitotic defects (Murray et al., 1994; Lehmann et <br /> chromosomes (SMC) proteins XSMC1p XSMC3p, ho-                   al., 1995). Genetic studies place rad18 epistasis <br /> mologues required sister chromatid cohe-           group rad2 fission yeast RAD51 homologue rhp51 <br /> sion Saccharomyces cerevisiae (SMC1 SMC3) (Michaelis         (Lehmann et al., 1995). Mutations genes <br /> et al., 1997). The 14S cohesin particle contains XRAD21p,      results chromosomal instability (Murray et al., 1994; <br /> which homologues budding yeast (Scc1p/                  Lehmann et al., 1995). However, rad2p required for <br /> Mcd1p) fission yeast (rad21p) (Birkenbihl Subra-            repair UV-C-induced lesions, acts independently <br /> mani, 1995; Guacci et al., 1997; Michaelis et al., 1997; Losada     classical excision repair, rhp51p, analogy to <br /> et al., 1998). In budding yeast Scc1p/Mcd1p required          functional studies RAD51 proteins, required for <br /> establish cohesion S-phase (Uhlmann Nasmyth,                 repair double-stranded breaks. Both pathways be <br /> 1998), dissociates chromatin chromatids segregate,          linked requirement recombination pro- <br /> and degraded (Guacci et al., 1997; Michaelis et al.,        cesses (Lehmann et al., 1995). With structural similarity to <br /> 1997; Ciosk et al., 1998). Mutations fission yeast rad21    SMC proteins, rad18p play role forming chro- <br /> lead phenotypes consistent defective sister        mosomal structures, including required different <br /> chromatid cohesion: chromosome instability, failure dou-         DNA repair mechanisms. <br /> ble-stranded break repair, defects chromosome seg-              Of DNA damage checkpoint genes described fis- <br /> regation (Birkenbihl Subramani, 1995; Tatebayashi et al.,       sion yeast, cut5 essential absence irradiation. <br /> 1998). Another key element required stable inheritence       It possible genes involved aspects genomic <br /> chromosomes budding yeast scc2p which,            integrity checkpoint role reper- <br /> stoichimetric subunit cohesin, required           toire function. Proteins required detection DNA <br /> association cohesin chromosomes, suggesting             damage activation checkpoint repair pathways <br /> that loads cohesin complex chromosomes              described. It likely proteins would <br /> (Toth et al., 1999). Likewise, mutations fission yeast       chromatin associated define essential genes. <br /> homologue mis4 lead defects sister chromatid cohesion         Thus, identify genes essential viability well <br /> and chromosome stability, mis4p appears func-           checkpoint function, specific alleles need be <br /> tion mainly S-phase (Furuya et al., 1998). Strains           isolated, T45M mutation cut5, present all <br /> containing temperature-sensitive mis4-242 allele           cut5 alleles isolated far (Saka et al., 1997; Verkade and <br /> sensitivity UV-C DNA replication inhibitor hy-           O&#226;&#8364;&#8482;Connell, 1998). Further screening approach saturation <br /> droxyurea permissive temperature, G2 check-             genome needed specific mutations in <br /> points intact (Furuya et al., 1998).                            essential genes lead checkpoint deficiencies do <br />    Once paired, sister chromatids intertwined,           alter other, essential, functions. <br /> their subsequent decatenation requires activity topo-           We performed genetic screen fission yeast identify <br /> isomerase II. Failure resolve sister chromatids       new genes involved DNA damage checkpoint (Verk- <br /> segregation results chromosome breakage subse-               ade O&#226;&#8364;&#8482;Connell, 1998). We strain <br /> quent instability genome loss genetic material.       isolated screen, previously termed NA74, defines a <br /> In extreme leads increased rates molecular           novel allele rad18 renamed rad18-74. Unlike <br /> evolution cell death (Holm et al., 1985, 1989; Uemura         previously characterized allele, rad18-X, rad18-74 has <br /> and Yanagida, 1986; Uemura et al., 1987; Downes et al., 1991).      defects DNA damage checkpoint, alleles are <br />    The onset mitosis marked chromatin condensa-            defective DNA repair. Cells deleted rad18 also <br /> tion, alteration chromatin organization compacts         unable arrest G2 presence DNA-damaging <br /> chromosomes many-fold. Two protein complexes chro-             agents. Expression allele mutated putative rad18 <br /> matin condensation activity, condensins, char-        ATP-binding domain failed rescue DNA damage sen- <br /> acterized Xenopus (Kimura Hirano, 1997). These con-          sitivity rad18-X rad18-74, overexpression was <br /> tain SMC proteins XCAP-Ep (SMC2 homologue)                  dominant negative lethal. This indicates essential <br /> XCAP-Cp (SMC4 homologue) (Hirano Mitchison, 1994;               role ATP binding hydrolysis rad18p function. <br /> Hirano et al., 1997), homologues          Rad18 mutants phosphorylation chk1p re- <br /> several species. Loss function budding yeast              sponse DNA damage, event proposed be <br /> XCAP-E homologue (SMC2) fission yeast SMC2               indicative checkpoint initiation. Therefore, rad18p re- <br /> SMC4 homologues (cut14 cut3) causes inability             quired maintain checkpoint-mediated G2 arrest. We also <br /> condense chromatin complete chromosome segrega-              isolated allele-specific high-copy suppressor rad18-74. <br /> tion (Saka et al., 1994b; Strunnikov et al., 1995). Topoisomer-     This gene, brc1, encodes BRCT domain&#226;&#8364;&#8220; containing protein <br /> ase II required chromatin condensation through-         (Koonin et al., 1996; Callebaut Mornon, 1997), deletion of <br />  <br />  <br /> 2906                                                                                                    Molecular Biology Cell <br />                                                                                                                             Fission Yeast rad18 <br />  <br />  <br /> which leads genomic instability mitotic defects                tion, washed twice CSE (20 mM citrate/phosphate, pH 5.6, 40 <br /> not defects DNA repair checkpoint function. Genetic                 mM EDTA, 1.2 M sorbitol), incubated 1 h 37&#194;&#176;C 1 ml <br /> interactions rad18, brc1, genes involved               CSE 1.5 mg Zymolyase 20-T (ICN, Costa Mesa, CA). The cells <br /> chromatin organization suggest role rad18p                   washed CSE resuspended ;6 3 108 cells/ml in <br /> regulation chromatin structure essential                   TSE (10 mM Tris, pH 7.5, 45 mM EDTA, 0.9 M sorbitol) 37&#194;&#176;C. An <br />                                                                           equal volume 1% low gelling temperature agarose TSE (at <br /> genomic integrity responses DNA damage.                            55&#194;&#176;C) added, agarose plugs samples and <br />                                                                           set ice 15 min. The plugs transferred solution 1% <br /> MATERIALS AND METHODS                                                     sarcosyl, 0.5 M EDTA, pH 9.5, 0.5 mg/ml proteinase K incu- <br />                                                                           bated 55&#194;&#176;C 48 h, readdition proteinase K 24 h. <br /> Fission Yeast Methods                                                     The plugs stored 4&#194;&#176;C washed Tris-EDTA before <br />                                                                           loading. Sample plugs run 0.6% gel chromosomal <br /> All strains derivatives 972h2 975h1. Standard procedures       grade agarose (Bio-Rad, Hercules, Ca) 0.53 Tris borate-EDTA for <br /> and media used propagation genetic manipulation              220 h 40 V pulse time 75 min Gene Navigator PFGE <br /> (Moreno et al., 1991). Methods transformation microscopy          apparatus (Pharmacia Biotech, Piscataway, NJ). <br /> have described previously (O&#226;&#8364;&#8482;Connell et al., 1994). The proce- <br /> dure used screen new genes involved DNA damage <br /> checkpoint described previously (Verkade O&#226;&#8364;&#8482;Connell,          Chromosome Loss Assays <br /> 1998). To test temperature sensitivity strains genetic <br />                                                                           Mutant alleles crossed ade6-704 background then <br /> analyses, strains streaked YES placed 25, 30, or <br />                                                                           crossed strain containing artificial minichromosome (Ch10), <br /> 36&#194;&#176;C 3&#226;&#8364;&#8220; 4 d. Strains grown 25&#194;&#176;C defined minimal <br />                                                                           contains ade6-704 tRNA suppressor sup3-5 (Niwa et al., <br /> media exponential phase shifted 30 36&#194;&#176;C. Samples were <br />                                                                           1989). The resulting progeny grown 30&#194;&#176;C selective <br /> fixed ethanol 0.25% gluteraldehyde plus 3.7% formaldehyde <br />                                                                           conditions (minimal medium lacking adenine), transferred to <br /> (O&#226;&#8364;&#8482;Connell et al., 1994) stained DAPI microscopy <br />                                                                           nonselective conditions (YES) 29 generations, samples were <br /> (Moreno et al., 1991). Immunolocalization rad18p tagged N <br />                                                                           plated minimal media containing 10 mg/l adenine. A sample <br /> terminus myc expressed medium-strength nmt1 <br />                                                                           taken beginning time course used a <br /> promoter performed using 9E10 Cy3-coupled anti-mouse <br />                                                                           background correction. After 3 d 30&#194;&#176;C number red colonies <br /> immunoglobulin G (Krien et al., 1998). <br />                                                                           scored percentage total number colonies, this <br />                                                                           used determine rate chromosome loss, calculated as <br /> UV Irradiation                                                            percentage colonies auxotrophic adenine divided the <br />                                                                           number generations. <br /> Cells grown exponential phase 30&#194;&#176;C plated YES, <br /> allowed dry, irradiated range doses UV-C using <br /> a Stratalinker (Stratagene, La Jolla, CA). Colonies counted    Cloning NA74 Recombinant Techniques <br /> 3 d 30&#194;&#176;C. To test ability cdc25-22 rescue radiation <br /> sensitivity, strains cdc25-22 background grown 25&#194;&#176;C,         Standard procedures used construction recombinant <br /> plated YES, irradiated above. These plates placed        plasmids (Sambrook et al., 1989). DNA sequencing performed <br /> directly 25&#194;&#176;C incubated 36&#194;&#176;C 4 h placed        double stranded templates processed ABI377 se- <br /> at 25&#194;&#176;C. Colonies counted 4 &#226;&#8364;&#8220;5 d incubation 25&#194;&#176;C.           quencer (Applied Biosystems, Foster City, CA). The NA74 strain (in <br /> Checkpoint analysis performed cells synchronized            ura4-D18 background) transformed genomic library <br /> cdc25-22 block release protocol. Strains cdc25-22 back-          pURSP1 (Barbet et al., 1992). The transformants replica plated <br /> ground grown 25&#194;&#176;C, shifted 36&#194;&#176;C 3 h, irradiated       YES 0.01% methyl methanesulfonate (MMS) select for <br /> 50 J/m2 UV-C described (O&#226;&#8364;&#8482;Connell et al., 1997), resuspended       complementation. DNA extracted complementing <br /> in prewarmed media 25&#194;&#176;C. Cell cycle progression monitored          transformants used electroporate Escherichia coli. The <br /> by DAPI straining ethanol-fixed samples, counting number        plasmids retested complementing activity transforming <br /> cells passing mitosis, assessed cells mitosis, nuclei   NA74 strain assessing complementation. Complementation <br /> undergoing septation. The proportion cells failing arrest       assessed sensitivity range doses UV-C irradiation <br /> estimated area curve spanning time           (as described above). Three deletion versions suppressing clone <br /> control cells entered mitosis.                                            constructed removing internal 1.4-kb HindIII <br />                                                                           fragment 3.8-kb HindIII fragment extending end the <br />                                                                           clone religating 2.7-kb PmeI&#226;&#8364;&#8220;SmaI fragment, end filling <br /> Ionizing Radiation                                                        Klenow religating. An ATP-binding domain mutation <br /> Exponentially growing cells 30&#194;&#176;C irradiated Varian         (G129A,K130R) constructed vitro mutagenesis (Kunkel et <br /> (Palo Alto, CA) linear accelerator. The focus set 100 cm        al., 1987) rad18 cDNA following oligonucleotide: <br /> the flasks, irradiated 6-MeV electron beam        59-GGTCATAATGGTAGCGCTAGAAGTGCTATACTTACAGGA- <br /> a dose rate ;22 Gy/min. Samples taken plated             39. <span id='am-552' about='xsp:base' typeof='owl:Thing'>Base</span> substitutions underlined create Eco47III site. <br /> YES, colonies counted 3 d 30&#194;&#176;C. Synchronous             Mutant (referred rad18-DN) wild-type rad18 then <br /> cultures obtained using lactose synchronization exponen-          cloned full-strength nmt1 promoter (pREP1), comple- <br /> tially growing cells (Mitchison Carter, 1975). Ionizing radiation     mentation assessed presence thiamine. <br /> of synchronous cultures delivered using Gammacell (Gamma <br /> Elektric, Stenl&#195;&#184;se, Denmark) 1000 137Cs source 12 Gy/min. Cell <br /> cycle progression monitored DAPI staining ethanol-fixed <br />                                                                           Spore Germination rad18::ura4 <br /> samples, counting number cells passing mitosis.                    Construction rad18 deletion strain described previ- <br />                                                                           ously (Lehmann et al., 1995). Sporulating colonies wild-type <br />                                                                           strain genotype ura4-D18/ura4-D18 leu1-32/leu1-32 ade6- <br /> Analysis Double-stranded Break Repair                                  M210/ade6-M216 his3-D1/his3-D1 h1/h2 rad18 deleted <br /> Double-stranded breaks induced exponentially growing              strain genotype rad181/rad18::ura4 ura4-D18/ura4-D18 leu1-32/ <br /> cells treating cultures 450 Gy using Varian linear        leu1-32 ade6-704/ade6-704 h1/h90 grown YE exponential <br /> accelerator described above. The culture placed 30&#194;&#176;C       phase starved minimal media lacking ammonium at <br /> to recover. Fifteen-milliliter samples harvested centrifuga-      30&#194;&#176;C induce sporulation. This cell mix digested overnight in <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                   2907 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                        0.05% b-glucuronidase washed water times; spores <br />                                                                        harvested layering mix 25% glycerol minimal <br />                                                                        media lacking ammonium, collected centrifugation 1500 3 g <br />                                                                        15 min, washed water. Spores (6 3 106) inocu- <br />                                                                        lated minimal media lacking uracil rad18::ura4 spore <br />                                                                        culture including uracil wild-type spore culture. These <br />                                                                        allowed germinate 30&#194;&#176;C polar growth observed <br />                                                                        marker germination (10 &#226;&#8364;&#8220;12 h) incubated 0.01% <br />                                                                        MMS 0.31 mg/ml 4-nitroquinoline-N-oxide (4-NQ) no <br />                                                                        treatment 6 h. Cells fixed ethanol DAPI <br />                                                                        stained counting mitoses fixed glutaraldehyde and <br />                                                                        formaldehyde described photography. <br />  <br />  <br />  <br />                                                                        Construction brc1::ura4 Strain <br />                                                                        For construction <span id='am-474' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-475' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-476' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-477' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-490' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-498' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-516' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-522' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-523' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-524' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-529' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-536' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-540' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-541' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-551' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> plasmid, 7.5-kb fragment was <br />                                                                        subcloned pBluescript KS1 HindIII site had <br />                                                                        ablated digestion, end filling, ligation. <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> this, the <br />                                                                        1.4-kb HindIII fragment, spans amino acid 48 amino <br />                                                                        acid 490 brc1 coding open reading frame excised and <br />                                                                        replaced 1.8-kb ura4 fragment. A diploid strain con- <br />                                                                        structed genotype: ura4-D18/ura4-D18 leu1&#226;&#8364;&#8220;32/leu1&#226;&#8364;&#8220;32 ade6- <br />                                                                        M210/ade6-M216 h1/h2. The brc1 deletion plasmid used to <br />                                                                        transform strain uracil prototrophy. Transformants were <br />                                                                        propagated continuous log-phase 4 d allow integration and <br />                                                                        replated selective conditions. Colonies screened by <br />                                                                        Southern blotting replacement event. Several appropriate <br />                                                                        strains derived, chosen analysis. This <br />                                                                        strain induced sporulate plating MEA medium for <br />                                                                        2 d. Tetrads 20 azygotic asci dissected, cases all <br />                                                                        spores viable gave 2:2 segregation ura1 <br />                                                                        phenotype. brc1::ura4 strains back-crossed wild type to <br />                                                                        exclude possibility suppressor mutations. <br />  <br />  <br />  <br />                                                                        Phosphorylation chk1p <br />                                                                        All strains background chk1::HA (Walworth and <br />                                                                        Bernards, 1996). Cultures irradiated UV-C ionizing <br />                                                                        radiation described above, samples harvested and <br />                                                                        frozen. The frozen cells disrupted using bead beater with <br />                                                                        glass beads extracted lysis buffer (50 mM Tris, pH 7.5, 150 <br />                                                                        mM NaCl, 50 mM NaF, 2 mM EDTA, 1 mM DTT, 80 mM b-glyc- <br />                                                                        erophosphate, 0.1% NP40, 0.1 mM Na-orthovanadate, 1 mM PMSF, <br />                                                                        plus protease inhibitors). The extract cleared centrifugation <br />                                                                        13,000 3 g 15 min, supernatant boiled SDS <br />                                                                        sample buffer. Samples analyzed SDS-PAGE Western <br />                                                                        blotting using 12CA5 antibody. The band intensity quanti- <br />                                                                        fied Molecular Dynamics (Sunnyvale, CA) densitometer using <br />                                                                        ImageQuant software. <br />  <br />  <br />  <br />  <br />                                                                        Figure 1 (cont). fluorescence microscopy. (F) Wild-type cells <br /> Figure 1. NA74 defective response DNA damage. (A)         (closed symbols) NA74 (open symbols) synchronized by <br /> UV-C <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>survival</span> curve wild type (M) NA74 (F). (B) Ionizing       lactose gradient centrifugation irradiated 100 Gy (F <br /> radiation survival curve wild type (M) NA74 (F). (C) DAPI-     E) 500 Gy (&#197;&#8217; &#226;&#8364;&#353;) mock irradiated (f M). Cultures <br /> stained cells NA74 synchronized cdc25-22         incubated 30&#194;&#176;C, ethanol-fixed samples taken for <br /> block release irradiated 50 J/m2 UV-C. Bar, 10 mm. (D)    DAPI staining. The percentages cells passed mitosis <br /> DAPI-stained cells asynchronous NA74 irradiated 450 Gy        assessed sample fluorescence microscopy. (G) <br /> ionizing radiation. Bar, 10 mm. (E) Wild-type cells (f M)      cdc25-22 (f M), cdc25-22 NA74 (F E), cdc25-22 rad3-136 <br /> NA74 (F E) synchronized cdc25-22 block release         (&#197;&#8217; &#226;&#8364;&#353;) grown 25&#194;&#176;C, plated YES medium, irradi- <br /> and irradiated 50 J/m2 UV-C (closed symbols) mock         ated indicated dose UV-C. Plates placed at <br /> irradiated (open symbols). Cultures incubated 25&#194;&#176;C,        25&#194;&#176;C (open symbols) 36&#194;&#176;C 4 h shifted 25&#194;&#176;C <br /> ethanol-fixed samples taken DAPI staining. The percent-       (closed symbols). Surviving colonies, expressed percentage of <br /> ages cells passed mitosis assessed sample   unirradiated controls, counted 4 d. <br />  <br />  <br /> 2908                                                                                                          Molecular Biology Cell <br />                                                                                                                         Fission Yeast rad18 <br />  <br />  <br />                                                                       RESULTS <br />                                                                       The NA74 Mutation Causes Altered Response to <br />                                                                       DNA Damage G2 <br />                                                                       The NA74 mutation defines gene recently identi- <br />                                                                       fied screen novel G2 checkpoint genes fission <br />                                                                       yeast (Verkade O&#226;&#8364;&#8482;Connell, 1998). This strain MMS <br />                                                                       sensitive enters lethal mitoses presence this <br />                                                                       DNA-damaging agent. To characterize mutation <br />                                                                       performed phenotypic analysis effect the <br />                                                                       response DNA damage. Survival curves UV-C <br />                                                                       ionizing radiation showed NA74 hypersensitive to <br />                                                                       DNA damaging agents (Figure 1, A B). DAPI <br />                                                                       staining irradiated cells confirmed pro- <br />                                                                       gressing aberrantly mitosis. In cells, DNA <br />                                                                       stretched division plane, cytokinesis had <br />                                                                       progressed absence completed chromosome segre- <br />                                                                       gation (Figure 1, C D). <br />                                                                          Because defective mitoses DNA damage consistent <br />                                                                       failure G2 DNA damage checkpoint, as- <br />                                                                       sessed cell cycle progression kinetics synchronous cul- <br />                                                                       tures irradiation. In wild-type controls, normal check- <br />                                                                       point-induced delay seen doses irradiation that <br />                                                                       minimal effects <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>cell survival</span>. The response NA74 <br />                                                                       cells was, however, complex. In UV-C-irradiated cells, <br />                                                                       ;30% cells culture failed arrest, entering <br />                                                                       mitosis kinetics similar unirradiated controls. The <br />                                                                       remainder progressed mitosis kinetics similar <br />                                                                       wild-type cells (Figure 1E). Consequently, cultures of <br />                                                                       NA74 cells showed doubling cell number time <br />                                                                       course experiment. However, unlike wild-type cells, <br />                                                                       mitoses irradiated NA74 cells highly aber- <br />                                                                       rant, failed chromosome segregation (Figure 1C), sug- <br />                                                                       gesting fully resolved lesions caused UV-C <br />                                                                       irradiation reinitiating mitosis. This response not <br />                                                                       lack synchrony (as assessed unirradiated <br />                                                                       controls) incomplete arrest cdc25-22 (see Figure 2C). <br />                                                                       These data consistent checkpoint defect <br />                                                                       delay maintained sufficient time to <br />                                                                       ensure mitotic fidelity. In g-irradiated cells, NA74 <br />                                                                       strain showed dose-dependent delay mitosis identical to <br />                                                                       wild-type strain (Figure 1F). This despite fact <br />                                                                       NA74 cells irradiated 500 Gy showed con- <br />                                                                       siderable reduction viability (2%) compared wild- <br />                                                                       type controls (48%), failed faithfully complete mitosis <br />                                                                       (Figure 1D). Because physiological role check- <br />                                                                       point prevent mitosis presence DNA damage, <br />                                                                       data consistent defective cell cycle <br />                                                                       response DNA damage. <br />  <br />  <br />  <br />                                                                       Figure 2 (cont). indicated times DAPI staining. The percent- <br /> Figure 2. Rad18 mutants defective DNA repair. (A) The          ages cells passed mitosis assessed each <br /> indicated strains grown midlogarithmic phase         sample fluorescence microscopy. (C) The indicated strains the <br /> irradiated 450 Gy ionizing radiation. Samples taken   background cdc25-22 arrested late G2 incubation at <br /> the indicated times (hours) processed pulse field gel elec-   36&#194;&#176;C 3 h. Cells irradiated 100 J/m2 UV-C (closed <br /> trophoresis. Unirradiated controls (U) included used      symbols) mock irradiated (open symbols; time 5 0) were <br /> standards positions chromosomes. (B) Wild-type   maintained 36&#194;&#176;C 2 h irradiation shifted 25&#194;&#176;C. <br /> cells (f M) rad18-X (F E) synchronized            Samples assayed DAPI staining, normal mitoses (f <br /> cdc25-22 block release irradiated 50 J/m2 UV-C      M) aberrant mitoses (F E) percentage total cells <br /> (closed symbols) mock irradiated (open symbols). Cultures     counted samples 100 &#226;&#8364;&#8220;150 cells time point. <br /> incubated 25&#194;&#176;C, ethanol-fixed samples taken        Data represent mean samples; SE # 5%. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                2909 <br />  H.M. Verkade et al. <br />  <br />  <br />    If defective G2 DNA damage checkpoint sole           defects rad18-X (Lehmann et al., 1995) (Figure 2C our <br /> cause radiation sensitivity NA74, imposed        unpublished results). Both rad18 strains prevented <br /> cell cycle delay irradiation partially       entering mitosis arrest relieved. These <br /> rescue radiation-induced cell death, de-           data indicate rad18 mutants require p34cdc2 activa- <br /> scribed checkpoint mutants (Al-Khodairy              tion entry aberrant mitoses, nuclear <br /> Carr, 1992). However, rescue observed             abnormalities result passage mitosis rather <br /> NA74 delayed G2 UV-C irradiation imposi-           interphase event. However, following mitoses in <br /> tion cdc25-22 arrest (Figure 1G), rescue          rad18-X using protocol highly aberrant those <br /> clearly evident checkpoint-defective rad3-136 control.     seen irradiated rad18-74 cells po- <br /> Hence, checkpoint defects account ra-         stirradiation arrest (Figures 1E 2C). This to <br /> diation sensitivity NA74, consistent       reported enhancement rad18-X defects 36&#194;&#176;C (Leh- <br /> an additional defect DNA repair.                                mann et al., 1995) suggests failure maintain a <br />                                                                    checkpoint arrest general feature rad18 alleles. Neither <br /> NA74 Is Allele rad18                                         rad18 allele defective DNA replication checkpoint <br /> We aimed clone NA74 gene complementation          assessed hydroxyurea-induced cell cycle arrest im- <br /> MMS sensitivity isolated seven complementing plas-             position temperature-sensitive S-phase mutants. How- <br /> mids, fell classes. One set, defined       ever, like checkpoint-proficient mis4-242 strain, rad18-74 <br /> plasmids, included rad18 gene. NA74 subsequently           rad18-X hydroxyurea sensitive (our unpub- <br /> shown allele rad18 linkage analysis          lished results). <br /> hitherto referred rad18-74. Sequencing rad18-74              Because rad18 essential gene, rad18-74 and <br /> showed single base substitution (G3 A) leading             rad18-X alleles completely null rad18p function. <br /> amino acid substitution (A151T), located 21 resi-         To determine checkpoint status rad18D cells, as- <br /> dues C-terminal putative ATP-binding domain. The            sessed ability wild-type spores spores deleted for <br /> remaining plasmids shown restriction              rad18 (rad18::ura4) maintain checkpoint arrest DNA <br /> hybridization analyses possess overlapping inserts          damage induced MMS UV-mimetic 4-NQ. After <br /> represented high-copy suppressor completely rescued         germination, radiomimetic drugs added cul- <br /> the radiation sensitivity rad18-74. This suppressor, brc1,   tures, followed mitotic figures 6 h. This <br /> discussed below.                                                   continuous exposure drugs circumvents asyn- <br />    Rad18p previously shown required             chrony spore germination precludes spore germina- <br /> repair UV-C-induced lesions (Lehmann et al., 1995). Be-         tion experiments using UV-C g-irradiation. In pres- <br /> cause rad18-74 previously isolated allele,            ence drugs, wild-type spores maintained G2 <br /> rad18-X, sensitive MMS ionizing radiation,         arrest, assessed fluorescence-activated cell-sorting <br /> we assayed ability repair double-stranded DNA             analysis duration experiment (Figure 3 our <br /> breaks using pulse field gel electrophoresis (Figure 2A).          unpublished results). Conversely, rad18::ura4 spores germi- <br /> Wild-type cells efficiently rejoined chromosomal frag-         nated entered defective mitoses presence both <br /> ments 4 h irradiation 450 Gy, little         drugs. After 6 h, aberrant mitoses represented ;30% of <br /> repair evident rad18 strain 10 h       germinated spores, normal mitoses ob- <br /> irradiation. Thus, rad18p required repair double-        served (Figure 3). After 3 h, ;20% aberrant mitoses <br /> stranded DNA breaks, rad18-74 enters mitosis         characterized stretching DNA the <br /> this dose ionizing radiation completing repair.         division plane, remainder resembling classical <br /> These data indicate g-irradiated rad18-74 cells      &#226;&#8364;&#339;cut&#226;&#8364;? phenotype. By 6 h, cytokinesis nuclear mate- <br /> showed checkpoint kinetics similar wild-type cells, they <br />                                                                    rial occurred mitotic cells. In absence MMS <br /> cycle G2 mitosis absence successfully <br />                                                                    4-NQ, ;10% rad18::ura4 spores displayed cut or <br /> completing DNA repair, explains appearance of <br />                                                                    &#226;&#8364;&#339;stretched&#226;&#8364;? phenotype, accounting 75% mitotic fig- <br /> mitotic defects cells. Conversely, rad18-X largely <br /> checkpoint proficient response ionizing radiation (Al-       ures. These data indicate rad18p required success- <br /> Khodairy Carr, 1992). Because rad18 strain          ful passage mitosis absence DNA-dam- <br /> capable repairing DNA damage induced ionizing ra-            aging agents. Germination rad18::ura4 spores 11 mM <br /> diation, difference cell cycle responses dramatic. We    hydroxyurea caused prolonged cell cycle arrest (our unpub- <br /> assayed UV-C checkpoint rad18-X cdc25-22 block         lished results), indicating replication checkpoint is <br /> (36&#194;&#176;C) release (25&#194;&#176;C) synchronization that,          intact, cells capable undergoing interphase <br /> unlike rad18-74, small proportion rad18-X cells          arrest. We compared kinetics aberrant mitoses these <br /> entered mitosis, subsequently little increase     cultures chk1::ura4 strain, completely <br /> cell number (Figure 2B). When strains defective nucleo-         lacks G2 DNA damage control (Walworth et al., 1993). These <br /> tide excision repair (rad13-A, rad15-P, rad16-U [Murray et      cells accumulated similar levels aberrant mitoses caused <br /> al., 1992; Carr et al., 1993, 1994]) irradiated UV-C,     checkpoint failure presence MMS 4-NQ <br /> the cells elongate fail enter mitosis, demonstrating    compared rad18::ura4, cut <br /> effect wild-type checkpoint responding unrepairable        phenotype. The appearance aberrant mitoses was <br /> DNA damage (Al-Khodairy Carr, 1992). Like rad18-74,            delayed compared rad18::ura4 strain. The <br /> the UV-C sensitivity rad18-X rescued               earlier appearance rad18::ura4 inherent <br /> cdc25-22 G2 arrest imposed 2 h irradiation syn-       requirement rad18p mitosis and/or partially <br /> chronized G2 cells, consistent DNA repair        affected different germination kinetics strains. <br />  <br />  <br /> 2910                                                                                                  Molecular Biology Cell <br />                                                                                                                           Fission Yeast rad18 <br />  <br />  <br />  <br />  <br />                                                                          Figure 4. Rad18p required phosphorylation chk1p <br />                                                                          DNA damage. The indicated strains, background of <br />                                                                          hemagglutinin-tagged chk1p, irradiated 100 J/m2, sam- <br />                                                                          ples taken indicated times (minutes). Western blot <br />                                                                          analysis using 12CA5 antibody used detect forms of <br />                                                                          hemagglutinin-tagged chk1p. Time 0 5 unirradiated. Essentially <br />                                                                          identical results obtained cultures irradiated 250 Gy <br />                                                                          ionizing radiation. <br />  <br />  <br />  <br />                                                                          Nevertheless, data indicate rad18p essential to <br />                                                                          prevent entry mitosis presence DNA damage. <br />  <br />                                                                          Rad18-X rad18-74 Show Normal DNA <br />                                                                          Damage-induced chk1p Phosphorylation <br />                                                                          The chk1p protein kinase final element signaling <br />                                                                          cascade activated G2 DNA damage checkpoint. DNA <br />                                                                          damage induces phosphorylation-dependent mobility <br />                                                                          shift chk1p, rad checkpoint gene-dependent manner <br />                                                                          (Walworth Bernards, 1996). This phosphorylation pro- <br />                                                                          motes binding 14-3-3 proteins used surrogate <br />                                                                          marker checkpoint activation (Chen et al., 1999). We used <br />                                                                          marker investigate defective DNA dam- <br />                                                                          age checkpoint responses rad18 mutant strains to <br />                                                                          failed checkpoint activation checkpoint maintenance. We <br />                                                                          assayed ability rad18-X rad18-74 cultures pro- <br />                                                                          mote chk1p phosphorylation response ionizing or <br />                                                                          UV-C irradiation. Both strains showed wild-type chk1p <br />                                                                          phosphorylation kinetics response DNA damage, <br />                                                                          phosphorylated forms appearing 15 min, and <br />                                                                          maintained, albeit slightly lower levels rad18 <br />                                                                          mutants, duration time course (Figure 4). <br />                                                                          Phosphorylated chk1p continued maintained the <br />                                                                          wild-type cells recovered checkpoint arrest <br />                                                                          entered mitosis, suggesting chk1p phosphorylation <br />                                                                          marker checkpoint release. These data consis- <br />                                                                          tent model rad18p required checkpoint <br />                                                                          maintenance initiation. It possible to <br />                                                                          assay chk1p phosphorylation rad18::ura4 background, <br /> Figure 3. Rad18p essential cell cycle arrest DNA dam- <br /> age. (A) Spores rad181/rad18::ura4 germinated the <br /> appearance polar growth (10 h) 30&#194;&#176;C medium lacking uracil <br /> to select rad18::ura4 cells, rad181 spores control. MMS   Figure 3 (cont.) (rad181/rad18::ura4), chk1::ura4 (chk11/ <br /> or 4-NQ added, cultures continued incubation          chk1::ura4) strains left untreated (M), MMS (l) or <br /> further 3 h. Samples taken, fixed, stained DAPI. Note      4-NQ (F) added time 0. Samples taken hourly, <br /> the presence cells septa bisecting rad18::ura4 nuclei     fixed, stained DAPI, mean percent cells with <br /> presence 4-NQ MMS. Bar, 10 mm. (B) Kinetics               aberrant mitoses (either cut DNA stretched division <br /> aberrant mitoses spore germination experiments. Spores           plane; text) recorded samples 100 &#226;&#8364;&#8220;150 cells. <br /> germinated wild-type (ura41/ura4-D18), rad18::ura4          SE # 5%. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                 2911 <br />  H.M. Verkade et al. <br />  <br />  <br />  <br />  <br /> Figure 5. Rad18p nuclear protein essential ATP-binding <br /> domain. (A) Rad18p localizes chromatin compartment the <br /> nucleus. Cells expressing myc-tagged rad18p stained 9E10 <br /> anti-myc antibody, visualized Cy3, counterstained with <br /> DAPI. (B) rad18-DN dominant negative overexpressed. Wild- <br /> type colonies grown presence (1T) absence (2T) of <br /> thiamine 3 d 30&#194;&#176;C. Plasmids: (1) pREP1 vector; (2) pREP1 <br /> wild-type rad18; (3) pREP1 rad18-DN. (C) rad18-DN comple- <br /> ment rad18-X. rad18-X cells grown presence (1MMS) or <br /> absence (2MMS) 0.0075% MMS thiamine-containing media <br /> (which represses lethal expression) 3 d 30&#194;&#176;C. Plasmids: (1) pREP1 <br /> empty vector; (2) pREP1 wild-type rad18; (3) pREP1 rad18-DN. <br /> rad18-DN complement rad18-74 (our unpublished results).             Figure 6. Isolation allele-specific high-copy suppressor of <br />                                                                            rad18-74. (A) Partial restriction map rad18-74-suppressing plas- <br />                                                                            mid. All isolates contained open reading frames 19G10.6 and <br />                                                                            19G10.7 (indicated solid bars; introns 19G10.7 [brc1] indi- <br /> because presence equal number ungerminated                    cated hatched bars). P, PmeI; H, HindIII. Rescue rad18-74 was <br /> (rad181) spores makes data impossible interpret.                    assessed UV-C irradiation survival curves. (B) rad18-74 and <br /> These spores G1, time cells competent                rad18-X transformed vector (M), C19G10.7 (brc1) <br /> for chk1p phosphorylation (Martinho et al., 1998),              (&#226;&#8364;&#353;), rad18 (F). Transformants grown selective media, and <br /> observed absence phoshorylation attributed                  UV-C survival curves determined percent survival com- <br /> cell cycle status defect signaling chk1p.              pared unirradiated controls. <br />  <br />  <br />  <br /> Rad18p Is Nuclear Protein Essential                              activity SMC2/SMC4 heterodimer (Kimura and <br /> ATP-binding Domain                                                         Hirano, 1997). <br /> The structural similarity rad18p SMC <br /> proteins suggests chromatin organization role rad18p.                An Allele-specific High-Copy Suppressor rad18 <br /> Immunostaining cells expressing myc-tagged version                 Encodes BRCT Protein <br /> rad18p showed localization <span id='am-553' about='xsp:pattern' typeof='owl:Thing'>pattern</span> identical DAPI                     In complementation rad18-74 genomic DNA li- <br /> staining, indicating that, like SMC proteins, rad18p              brary, plasmids carrying high-copy suppressor of <br /> found chromatin compartment nucleus (Figure                  rad18-74 isolated addition rad18-containing <br /> 5A) (Strunnikov et al., 1993, 1995; Hirano Mitchison,                  plasmids. Sequence analysis high-copy suppressor <br /> 1994; Saka et al., 1994b; Saitoh et al., 1995). Because ATPase             plasmids showed corresponded locus chromo- <br /> activity demonstrated SMC family members,                     I covered cosmid C19G10 (GenBank accession num- <br /> we mutated ATP-binding domain rad18, construct                   ber Z69909). Two open reading frames present the <br /> the allele rad18-DN. Overexpression rad18-DN wild-                   suppressing plasmids, deletion mapping these <br /> type cells lethal, indicating allele domi-             plasmids showed suppressing activity resided in, and <br /> nant negative effect essential rad18 function cells                dependent on, open reading frame designated <br /> resembling rad18D accumulation mitotic defects                  C19G10.7 (Figure 6A). C19G10.7 encodes protein six <br /> (Figure 5B unpublished results) (Lehmann et al.,                   BRCT domains, termed gene brc1. <br /> 1995). Lower-level expression wild-type rad18 rescued               The suppression rad18-74 brc1 allele specific, it <br /> MMS sensitivity rad18-X rad18-74, expression                 suppressive activity radiation sensitivity of <br /> rad18-DN complement rad18-X rad18-74                   rad18-X (Figure 6B). The suppression radiation sensi- <br /> (Figure 5C). Together, data indicate ATP-                   tivity rad18-74 reflected suppression the <br /> binding domain rad18 essential function, indicating              mitotic defects seen checkpoint failure. These effects are <br /> an essential role ATP hydrolysis, described                unlikely to perturbation cell cycle distribution, <br />  <br />  <br /> 2912                                                                                                             Molecular Biology Cell <br />                                                                                                                  Fission Yeast rad18 <br />  <br />  <br /> because high-level overexpression brc1           logue Aspergillus NIMA kinase, implicated the <br /> nmt1 promoter effect cell cycle progression          control chromatin condensation, activation of <br /> wild-type rad18 mutant cells (our unpublished results).        kinases prematurely induce chromatin con- <br /> Two fission yeast BRCT proteins, encoded cut5        densation independently cell cycle progression <br /> rhp9, effect radiation sensitivity rad18-74      (O&#226;&#8364;&#8482;Connell et al., 1994). When fin1 deleted, cells again <br /> when expressed multicopy plasmid. Therefore, BRCT          similar phenotypes G2 delay mitotic defects. In <br /> domains insufficient suppress rad18-74.              fact, double mutants fin1D top2-191 syn- <br />                                                                   thetic lethality greatly increased frequency of <br /> brc1 Is Required Mitotic Fidelity                             mitotic abnormalities (Krien et al., 1998). This suggests that <br />                                                                   cooperate chromatin condensation. To determine <br /> To determine function brc1p, deleted brc1 re-          rad18p plays role chromatin condensation, we <br /> placed ura4 marker homologous recombina-           examined genetic interactions other <br /> tion diploid cells (Figure 7A). Dissection tetrads          genes involved chromatin organization. The data are <br /> showed 2:2 segregation ura41:ura42, indicating brc1       presented Table 1. <br /> not essential gene. Given allele-specific interaction         Both rad18-74 rad18-X synthetically <br /> with rad18, tested brc1::ura4 cells radiation      lethal top2-191, failing form colonies semiper- <br /> sensitive. Radiation survival curves showed brc1::ura4 cells      missive temperature 30&#194;&#176;C (Figure 8A). Under con- <br /> were sensitive UV-C irradiation, mildly      ditions top2-191 cells delay G2 mitotic defects in <br /> sensitive ionizing radiation (Figure 7, B C), did      ;10% cells able form colonies (Krien et al., <br /> we observe defects DNA damage replication check-            1998). DAPI staining double mutant strains grown the <br /> point kinetics (our unpublished results). However,          permissive temperature 25&#194;&#176;C shifted 30&#194;&#176;C for <br /> phenotypes brc1::ura4 cells indicate brc1p plays role   4 h showed dramatic increase frequency aberrant <br /> in fidelity mitotic chromosome segregation. First, <br />                                                                   mitoses. Most characterized failure com- <br /> assays chromosome stability showed brc1::ura4 cells <br />                                                                   pletely segregate chromosomes cytokinesis (Figure 8, <br /> lost artificial chromosome frequency 200-fold <br />                                                                   C&#226;&#8364;&#8220;I, Table 1). Inactivation topoisomerase II drugs <br /> greater wild-type cell (Figure 7D). Chromosome in- <br />                                                                   mitoxantrone leads G2 arrest double- <br /> stability seen rad18-74 (16-fold) been <br />                                                                   stranded DNA breaks, drugs highly <br /> described rad18-X (74-fold) (Lehmann et al., 1995). This <br />                                                                   toxic strains defective double-stranded break repair in <br /> has described rad18 epistasis group <br />                                                                   budding yeast (Hartwell et al., 1997). As rad18-X;top2-191 <br /> members rad2 rhp51 (Murray et al., 1994; Muris et al., <br />                                                                   rad18-74;top2-191 double mutants proceed aberrant <br /> 1996). Second, brc1::ura4 cells showed consistent although <br />                                                                   mitosis, rad18p required maintain G2 delay caused <br /> variable delay entry mitosis, dividing 19 6 3.8 mm, <br /> compared 13.8 6 0.4 mm wild-type controls (Figure        reduction top2p activity semipermissive <br /> 7E). Fluorescence-activated cell-sorting analysis combined        temperature. An inability repair double-stranded breaks <br /> with mitotic indices showed delay confined       caused top2p inactivation account these <br /> the G2 period cell cycle. Furthermore, ;6% cells        mitotic phenotypes, repair defect of <br /> showed variety chromosome segregation defects result-        rad18-X rad18-74 sole cause syn- <br /> ing classical cut phenotypes, nuclear fragmentation, hy-       thetic lethality. Alternatively, synthetic phenotypes <br /> percondensation, mis-segregation, stretching DNA           suggest top2p rad18p cooperate chromatin <br /> along division axis (Figure 7, F&#226;&#8364;&#8220;J). These phenotypes         organization. We attempted suppress rad18 mutants <br /> resemble observed rad18::ura4 spores (Figure 3A).        overexpressing top2 nmt1 promotor ob- <br /> Together, data consistent defective organiza-      served effect radiation sensitivity mitotic fidelity. <br /> tion chromatin absence brc1p function. Finally,      Overexpression top2 effect wild-type cells (our <br /> no nonparental ditypes recovered crosses          unpublished results). brc1::ura4 synthetically lethal <br /> brc1::ura4 rad18-74 rad18-X. In case,     top2-191 (Figure 8B), increased defects chromo- <br /> presumptive double mutants divided times,            segregation (Figure 8, C&#226;&#8364;&#8220;J, Table 1). <br /> arresting microcolonies elongated and/or         Double mutants constructed rad18-X, rad18-74 <br /> lysed cells, described strains deleted rad18   fin1D showed higher levels mitotic abnormalities <br /> (Lehmann et al., 1995). These data indicated mutations       (7- 10-fold), suggesting cooperation function. To further <br /> in brc1 rad18 result synthetic lethality,   characterize this, investigated functional links the <br /> brc1 essential rad18 function impaired.                condensin SMC proteins encoded cut3 cut14. No <br />                                                                   synthetic phenotypes seen double mutants con- <br />                                                                   structed conditional alleles, cut3-477 and <br /> Interactions Genes Involved Higher-                    cut14-208. This specific alleles, not <br /> Order Chromosome Structure                                        intermediate phenotypes temperatures ranging <br /> The defects chromosome segregation G2 delay            25 34&#194;&#176;C (our unpublished results), we <br /> seen brc1 rad18 mutants reminiscent pheno-         rule functional links time. The functional <br /> types described strains defective chromatin organiza-       interactions top2 fin1 suggest possible role for <br /> tion. The temperature-sensitive allele type II topoisomer-     rad18p chromatin condensation. Double mutants rad18 <br /> ase, top2-191, displays similar phenotypes shifted        alleles rad21-45 strain, analogy homol- <br /> semipermissive temperature (Krien et al., 1998). Top2p ap-        ogous proteins systems likely defective in <br /> pears play roles chromatin organization, includ-       sister chromatid cohesion (Guacci et al., 1997; Michaelis et al., <br /> ing chromatin condensation. Fin1p, fission yeast homo-        1997; Losada et al., 1998), showed synthetic interactions. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                       2913 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                                 Similarly, genetic interactions observed mis4- <br />                                                                                 242, defective sister chromatid cohesion <br />                                                                                 (Furuya et al., 1998). Furthermore, double mutants between <br />                                                                                 rad18 alleles rad21-45 mis4-242 showed en- <br />                                                                                 hanced radiation sensitivity compared parent, <br />                                                                                 indication separable function (our un- <br />                                                                                 published results). Therefore, genetic interactions we <br />                                                                                 observed consistent role rad18p chromatin <br />                                                                                 organization, independent sister chromatid co- <br />                                                                                 hesion. <br />  <br />  <br />                                                                                 DISCUSSION <br />                                                                                 Cells respond DNA damage G2 delaying mitotic <br />                                                                                 onset maintenance p34cdc2 Y15 phosphorylation <br />                                                                                 (O&#226;&#8364;&#8482;Connell et al., 1997; Rhind et al., 1997). Studies the <br />                                                                                 molecular events controlling checkpoint response are <br />                                                                                 beginning unravel biochemical interactions dif- <br />                                                                                 ferent checkpoint proteins checkpoint proteins <br />                                                                                 core cell cycle regulators (Walworth Bernards, 1996; <br />                                                                                 Furnari et al., 1997; O&#226;&#8364;&#8482;Connell et al., 1997; Peng et al., 1997; <br />                                                                                 Rhind et al., 1997; Saka et al., 1997; Sanchez et al., 1997; <br />                                                                                 Kostrub et al., 1998; Kumagai et al., 1998; Chen et al., 1999; <br />                                                                                 Furnari et al., 1999; Kumagai Dunphy, 1999; Lopez- <br />                                                                                 Girona et al., 1999). Classically, DNA damage checkpoint <br />                                                                                 genes defined mutations abolish the <br />                                                                                 checkpoint response phenotypic consequence in <br />                                                                                 absence DNA damage. However, cases, role <br />                                                                                 checkpoint control repertoire func- <br />                                                                                 tion essential gene, case cut5p (Saka and <br />                                                                                 Yanagida, 1993; Saka et al., 1994a; Verkade O&#226;&#8364;&#8482;Connell, <br />                                                                                 1998). Extensive genetic screens needed identify <br />                                                                                 specific checkpoint-defective alleles essential genes to <br />                                                                                 fully define molecules involved response DNA <br />                                                                                 damage. <br />                                                                                    rad18 example essential gene required <br />                                                                                 checkpoint control. Although precise essential func- <br />                                                                                 tion rad18p known, structural similarity the <br />                                                                                 SMC family proteins, localization chromatin, and <br />                                                                                 phenotypes rad18 mutants support role chromatin <br />                                                                                 organization. Deletion rad18 lethal associated <br />                                                                                 mitotic failure. Analysis <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>point mutation</span> rad18, <br />                                                                                 rad18-X, previously shown rad18p required for <br />                                                                                 repair UV-induced lesions (Lehmann et al., 1995), we <br />                                                                                 showed rad18p essential repair of <br />                                                                                 double-stranded DNA breaks. Furthermore, also <br />                                                                                 shown rad18p required maintain checkpoint <br />                                                                                 arrest DNA damage, failed checkpoint <br /> Figure 7. Mitotic defects chromosome instability strain deleted for <br />                                                                                 response leads highly aberrant mitoses subsequent <br /> brc1. (A) Strategy creating <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>null allele</span> brc1. See MATERIALS AND        cell death. We described isolation analysis <br /> METHODS details. (B) UV-C survival curve wild-type (M)              checkpoint-defective allele rad18, rad18-74. In re- <br /> brc1::ura4 (F). (C) Ionizing radiation survival curve wild-type (M)     sponse UV-C ionizing radiation, strains carrying <br /> brc1::ura4 (F). (D) Rates loss artificial chromosome 10 generation.   rad18-74 allele failed maintain checkpoint arrest. <br /> Data expressed percentage total colonies corrected      Moreover, rad18-X cells showed defective checkpoint <br /> loss generation zero. Values wild-type, 0.005; rad18-74, 0.080;          responses held 36&#194;&#176;C irradiation. Importantly, <br /> brc1::ura4, 0.997; rad18-X, 0.368. (E) DAPI phase-contrast image         cells deleted rad18 checkpoint deficiencies compara- <br /> exponentially growing brc1::ura4 cells. Note variability cell <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>     ble chk1 deleted strain. <br /> mis-segregated chromosomes (top left). Bar, 20 mm. (F&#226;&#8364;&#8220;J) Examples of <br /> mitotic abnormalities brc1::ura4. Bars, 10 mm. (F) Nuclear fragmentation <br />                                                                                    The response DNA damage considered a <br /> and mis-segregation. (G) Stretching chromosomes division axis          multistage process involving damage detection, initiation of <br /> and bisection septum. (H) Bisection nucleus septum hyper-          cell cycle arrest DNA repair pathways, maintenance of <br /> condensation. (I) Hypercondensation mis-segregation. (J) Fragmenta-         arrest repair, subsequent recovery cell cycle <br /> tion (above) incomplete segregation chromosomes (below left).            reentry. To assess checkpoint defect lies rad18 <br />  <br />  <br /> 2914                                                                                                               Molecular Biology Cell <br />                                                                                                                           Fission Yeast rad18 <br />  <br />  <br />  <br /> Table 1. Genetic interactions rad18, brc1, top2, fin1, rad21, mis4 <br />  <br />                     rad18-74          rad18-X               brc1D                fin1D          top2-191         rad21-45          mis4-242 <br />  <br /> Wild type              2                6                    6                      4             11                 7                11 <br /> rad18-74               &#226;&#8364;&#8220;              Allelic          Synthetic lethal            27             52                12                 6 <br /> rad18-X                                 &#226;&#8364;&#8220;              Synthetic lethal            41             52                10                12 <br /> brc1D                                                        &#226;&#8364;&#8220;                      8             40                 6                16 <br />  <br /> Numbers represent percent aberrant mitoses double mutants. All strains containing top2-191 mis4-242 grown midlogarithmic <br /> phase 30&#194;&#176;C fixation, DAPI staining, observation fluorescence microscopy. top2-191- mis4-242-containing strains were <br /> grown midlogarithmic phase 25&#194;&#176;C shifted 30&#194;&#176;C 4 h fixation, DAPI staining, observation fluorescence <br /> microscopy. Data represents mean samples 100 cells. <br />  <br />  <br /> mutants, assayed phosphorylation chk1p protein                  brc1p multicopy plasmid. This suppression allele <br /> kinase marker checkpoint activation (Walworth                 specific rad18-74 general feature in- <br /> Bernards, 1996). In time course experiments, rad18-74                creased expression BRCT proteins. BRCT domains show <br /> and rad18-X showed wild-type kinetics chk1p phosphor-                  homology C-terminus breast ovarian cancer <br /> ylation. This observation, kinetics mi-              susceptibility gene BRCA1 (itself G2 checkpoint protein; <br /> totic entry irradiated synchronous cultures, consistent             Xu et al., 1999) variety proteins in- <br /> with model rad18p essential mainte-                   volved DNA repair, DNA replication, checkpoint <br /> nance checkpoint arrest, initiation. This         control (Callebaut Mornon, 1997). In instances <br /> in contrast rad3p protein, required initiate           shown act protein interaction domains <br /> checkpoint arrest maintain (Martinho et al.,                (Saka et al., 1997; Grawunder et al., 1998; Taylor et al., 1998). <br /> 1998). These analyses DNA damage check-                     One possible interpretation suppression rad18-74 is <br /> point genetically separable initiation mainte-                rad18p brc1p physically interact in- <br /> nance events, required survival                  teraction defective rad18-74 mutant. We attempted <br /> DNA damage. Because chk1p phosphorylation persists                     coimmunoprecipitate epitope-tagged derivatives of <br /> wild-type cells reenter cell cycle, rad18              rad18p brc1p expressed wild-type irradiated cells <br /> mutant cells maintain chk1p phosphorylation despite                  saw evidence physical association despite efficient <br /> defective checkpoint maintenance, chk1p itself                immunoprecipitation protein. Furthermore, saw <br /> specific checkpoint initiation.                                       evidence interaction yeast two-hybrid analysis or <br />    In addition role checkpoint maintenance, rad18p              vitro&#226;&#8364;&#8220;transcribed translated proteins (our <br /> is required DNA repair. These defects                 unpublished results). Therefore likely brc1 acts <br /> unrelated present model explaining response                  downstream rad18 bypass suppressor rad18-74, and <br /> rad18 mutants DNA damage. With similarity                   suppression represents subset rad18p <br /> SMC family DNA-binding chromatin organization                      functions. Deletion brc1 results mitotic defects and <br /> proteins, rad18p involved establishing main-                 chromosome instability, phenotypes the <br /> taining chromosomal structures necessary DNA                 primary defect. These phenotypes seen in <br /> repair. The presence structures chromosomes               rad18 mutants. Furthermore, deletion brc1 synthetically <br /> then acts origin signal maintain checkpoint              lethal rad18-X rad18-74, indicating brc1p is <br /> arrest. In scenario, defective rad18p function would,             required viability rad18 function compromised. <br /> we observed, result failure complete repair                These cells not, however, hypersensitive UV-C or <br /> maintain arrest. These structures presumably essential                ionizing radiation, brc1p essential DNA <br /> for normal chromatin organization given essential na-                 repair checkpoint responses, seen rad18p. How- <br /> ture rad18 mitotic defects occur absence.          ever, presence rad18-X rad18-74, brc1 essential, <br />    This phenomenon related checkpoint                  suggesting brc1 plays downstream role, able <br /> monitors completion S-phase particular role              compensate subfunctional rad18. <br /> DNA polymerase (pol1/swi7) response (D&#226;&#8364;&#8482;Urso et al.,                With presence BRCT domains, likely that <br /> 1995). When pol1p present, functional, S-                 brc1p facilitates protein&#226;&#8364;&#8220;protein interactions, inter- <br /> phase progression stalled. This results checkpoint acti-            acting BRCT domain protein. By analogy the <br /> vation prevents entry mitosis. When pol1p de-                 cohesin condensin complexes, rad18p of <br /> leted, cells fail recognize incomplete DNA synthesis               larger complexes (Jessberger et al., 1998). brc1p <br /> enter catastrophic mitoses. Thus, presence pol1p                function adaptor molecule pro- <br /> the stalled replication forks appears signal incomplete                teins, complex involved rad18p-dependent <br /> S-phase activates checkpoint. Similarly, checkpoint               process required mitosis. We attempted localize brc1p <br /> pathways monitor presence proteins involved                 indirect immunofluorescence specific local- <br /> chromatin organization, influence timing                 ization cells despite detecting brc1p Western blots. It <br /> entry mitosis serve alert cell                possible epitopes obscured vivo. We <br /> positioned cell cycle.                                             attempted localize GFP-brc1p fusions no <br />    The radiation sensitivity checkpoint defects                    localization fluorescence cells expressing fusion <br /> rad18-74 completely suppressed expression                  protein nmt1 promoter. Multiprotein complexes <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                2915 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                            tion proteins mitotic fidelity. Mutations in <br />                                                                            rad18, brc1 fin1 synthetically lethal tempera- <br />                                                                            ture-sensitive topoisomerase II semipermissive tempera- <br />                                                                            ture. This consistent models proposing multiple <br />                                                                            roles topoisomerase II chromatin organization (Heck, <br />                                                                            1997). Although rad18p play role top2p-dependent <br />                                                                            decatenation chromatin, defects explain <br />                                                                            rad18 phenotypes. Rad18-74 rad18-X mutants, <br />                                                                            unlike top2-191, minor mitotic defects the <br />                                                                            absence irradiation. Furthermore, mutants de- <br />                                                                            fective maintaining checkpoint arrest irradiated <br />                                                                            late G2 (cdc25-22-arrested cells), time bulk of <br />                                                                            decatenation likely completed. Thus, defective <br />                                                                            mitoses presumably result failed repair check- <br />                                                                            point maintenance defect topoisomerase II <br />                                                                            activity. Indeed, overexpression top2 rescue the <br />                                                                            radiation sensitivity mitotic abnormalities rad18-X <br />                                                                            rad18-74. Brc1-deleted cells delay entry mitosis. A <br />                                                                            combination G2 delay mitotic defects seen in <br />                                                                            strains carrying mutations rad18 (Lehmann et al., 1995) or <br />                                                                            fin1 (Krien et al., 1998) temperature-sensitive top2 <br />                                                                            mutants semipermissive temperature (Krien et al., 1998). <br />                                                                            Such delay consistent model defects in <br />                                                                            chromatin organization monitored mitosis and <br />                                                                            lead activation checkpoint influences timing <br />                                                                            mitotic onset. <br />                                                                               It increasingly evident regulation of <br />                                                                            higher-order chromatin structure vital maintenance <br />                                                                            genomic integrity. SMC1&#226;&#8364;&#8220; 4 proteins central reg- <br />                                                                            ulating chromosome dynamics, functioning key com- <br />                                                                            ponents chromatin condensation sister chro- <br />                                                                            matid cohesion. The phenotypes genetic interactions <br />                                                                            rad18-74, rad18-X, rad18::ura4 mutants provide <br />                                                                            link chromatin organization, DNA repair, and <br />                                                                            control mitotic entry. The conserved structure, lo- <br />                                                                            calization, requirement chromosome stability sug- <br />                                                                            gest rad18p carries related functions SMC1&#226;&#8364;&#8220; 4. <br />                                                                            Genetic interactions rad18 top2 fin1 suggest a <br />                                                                            role chromatin condensation, extensive analyses of <br />                                                                            rad18p needed determine precise function. De- <br />                                                                            fects DNA repair checkpoint maintenance in <br />                                                                            absence rad18p function consistent role <br /> Figure 8. Interactions top2, rad18 brc1. (A) top2-191         rad18p forming essential structures re- <br /> synthetically lethal rad18-74 rad18-X. Strains plated        quired DNA repair recognized proteins <br /> on YES incubated indicated temperatures 4 d <br /> (25&#194;&#176;C) 3 d (30 36&#194;&#176;C). Strains: (1) wild type; (2) rad18-74; (3)     signal checkpoint maintenance. In recent years, ex- <br /> rad18-74 top2-191; (4) rad18-X top2-191; (5) rad18-X; 6, top2-191. (B)     tensive progress understanding check- <br /> top2-191 synthetically lethal brc1::ura4. Strains plated      point initiation, analysis rad18p un- <br /> on YES treated above. Strains: (1) wild type; (2) brc1::ura4; (3)   cover arrest maintained ensure completion <br /> brc1::ura4 top2-191; (4) top2-191. (C&#226;&#8364;&#8220;J) DAPI phase-contrast im-       DNA repair progression mitosis. <br /> ages cells grown midlogarithmic phase 25&#194;&#176;C then <br /> shifted 30&#194;&#176;C 4 h. Strains: (C) wild type; (D) rad18-X; (E) <br /> rad18-74; (F) top2-191; (G) brc1::ura4; (H) top2-191 rad18-X; (I) top2- <br /> 191 rad18-74; (J) top2-191 brc1::ura4. Bar, 10 mm.                         ACKNOWLEDGMENTS <br />                                                                            We thank Dr. Jim Hagekyriakou assistance ionizing irra- <br />                                                                            diations, Dr. Mitsuhiro Yanagida (Kyoto University, Kyoto, Japan) <br /> may impede fluorescence GFP, protein                  provision strains plasmids, Drs Doris Germain, <br /> diffusely localized cell.                                   Michael Hynes, members Cell Cycle Control Laboratory <br />   We investigated functional links brc1 genes                  critical reading manuscript. This work supported by <br /> implicated chromatin organization constructing dou-                  National Health Medical Research Council grant 970580 and <br /> ble mutants combinations lead                    Australian Research Council grant A09804303 M.J.O. H.M.V. a <br /> enhancement phenotype, indicative coopera-                  recipient Australian Postgraduate Award. <br />  <br />  <br /> 2916                                                                                                           Molecular Biology Cell <br />                                                                                                                                        Fission Yeast rad18 <br />  <br />  <br /> REFERENCES                                                                      Hartwell, L., Weinert, T. (1989). Checkpoints: controls ensure the <br />                                                                                 order cell cycle events. Science 246, 629&#226;&#8364;&#8220;634. <br /> Al-Khodairy, F., Carr, A.M. (1992). DNA repair mutants defining G2 <br /> checkpoint pathways Schizosaccharomyces pombe. EMBO J. 11, 1343&#226;&#8364;&#8220;1350.        Hartwell, L.H., Szankasi, P., Roberts, C.J., Murray, A.W., Friend, S.H. <br />                                                                                 (1997). Integrating genetic approaches discovery anticancer <br /> Al-Khodairy, F., Fotou, E., Sheldrick, K.S., Griffiths, D.J.F., Lehman, A.R.,   drugs. Science 278, 1064&#226;&#8364;&#8220;1068. <br /> and Carr, A.M. (1994). Identification characterization new elements <br /> involved checkpoint feedback controls fission yeast. Mol. Biol.       Heck, M.M.S. (1997). Condensins, cohesins chromosome architecture: <br /> Cell 5, 147&#226;&#8364;&#8220;160.                                                                make brake mitotic chromosome. Cell 91, 5&#226;&#8364;&#8220;8. <br />  <br /> Barbet, N., Muriel, W.J., Carr, A.M. (1992). Versatile shuttle vectors      Hirano, T., Kobayashi, R., Hirano, M. (1997). Condensins, chromo- <br /> and genomic libraries use Schizosaccharomyces pombe. Gene 114,         condensation protein complexes containing XCAP-C, XCAP-E and <br /> 59&#226;&#8364;&#8220;66.                                                                          Xenopus homolog Drosophila Barren protein. Cell 89, 511&#226;&#8364;&#8220;521. <br /> Birkenbihl, R.P., Subramani, S. (1992). Cloning characterization     Hirano, T., Mitchison, T.J. (1994). A heterodimeric coiled-coil protein <br /> rad21 essential gene Schizosaccharomyces pombe involved DNA            required mitotic chromosome condensation vitro. Cell 79, 449&#226;&#8364;&#8220;458. <br /> double-strand-break repair. Nucleic Acids Res. 20, 6605&#226;&#8364;&#8220;6611.                   Holm, C., Goto, T., Wang, J., Bostein, D. (1985). DNA topoisomerase <br /> Birkenbihl, R.P., Subramani, S. (1995). The rad21 gene product           II required time mitosis yeast. Cell 41, 553&#226;&#8364;&#8220;563. <br /> Schizosaccharomyces pombe nuclear, cell cycle-regulated phosphopro-        Holm, C., Stearns, T., Botstein, D. (1989). DNA topoisomerase II must <br /> tein. J. Biol. Chem. 270, 7703&#226;&#8364;&#8220;7711.                                            act mitosis prevent nondisjunction chromosome breakage. Mol. <br /> Boddy, M.N., Furnari, B., Mondesert, O., Russell, P. (1998). Replication    Cell. Biol. 9, 159&#226;&#8364;&#8220;168. <br /> checkpoint enforced kinases Cds1 Chk1. Science 280, 909&#226;&#8364;&#8220;912.             Jessberger, R., Frei, C., Gasser, S.M. (1998). Chromosome dynamics: <br /> Callebaut, I., Mornon, J.P. (1997). From BRCA1 RAP1: wide-             SMC protein family. Curr. Opin. Genet. Dev. 8, 254&#226;&#8364;&#8220;259. <br /> spread BRCT module closely associated DNA repair. FEBS Lett. 400, <br />                                                                                 Kimura, K., Hirano, T. (1997). ATP-dependent positive supercoiling of <br /> 25&#226;&#8364;&#8220;30. <br />                                                                                 DNA 13S condensin: biochemical implication chromosome con- <br /> Carr, A.M., Schmidt, H., Kirchhoff, S., Muriel, W.J., Sheldrick, K.S., Grif-    densation. Cell 90, 625&#226;&#8364;&#8220;634. <br /> fiths, D.J., Basmacioglu, C.N., Subramani, S., Clegg, M., Nasim, A. <br />                                                                                 Koonin, E., Altschul, S., Bork, P. (1996). Functional motifs. Nat. Genet. <br /> (1994). The rad16 gene Schizosaccharomyces pombe: homolog the <br />                                                                                 13, 266&#226;&#8364;&#8220;267. <br /> RAD1 gene Saccharomyces cerevisiae. Mol. Cell. Biol. 14, 2029&#226;&#8364;&#8220;2040. <br />                                                                                 Kostrub, C.F., Knudsen, K., Subramani, S., Enoch, T. (1998). Hus1p, a <br /> Carr, A.M., Sheldrick, K.S., Murray, J.M., al-Harithy, R., Watts, F.Z., and <br />                                                                                 conserved fission yeast checkpoint protein, interacts Rad1p is <br /> Lehmann, A.R. (1993). Evolutionary conservation excision repair in <br />                                                                                 phosphorylated response DNA damage. EMBO J. 17, 2055&#226;&#8364;&#8220;2066. <br /> Schizosaccharomyces pombe: evidence family sequences related the <br /> Saccharomyces cerevisiae RAD2 gene. Nucleic Acids Res. 21, 1345&#226;&#8364;&#8220;1349.           Krien, M.J.E., Bugg, S.J., Palatsides, M., Ausoline, G., Morimyo, M., and <br /> Chen, L., Liu, T.H., Walworth, N.C. (1999). Association Chk1        O&#226;&#8364;&#8482;Connell, M.J. (1998). A NIMA homologue promotes chromatin conden- <br /> 14-3-3 proteins stimulated DNA damage. Genes &amp; Dev. 13, 675&#226;&#8364;&#8220;685.          sation fission yeast. J. Cell Sci. 111, 967&#226;&#8364;&#8220;976. <br />  <br /> Ciosk, R., Zachariae, W., Michaelis, C., Shevchenko, A., Mann, M.,          Kumagai, A., Dunphy, W.G. (1999). Binding 14-3-3 proteins and <br /> Nasmyth, K. (1998). An ESP1/PDS1 complex regulates loss sister chro-         nuclear export control intracellular localization mitotic inducer <br /> matid cohesion metaphase anaphase transition yeast. Cell 93,       cdc25. Genes &amp; Dev. 13, 1067&#226;&#8364;&#8220;1072. <br /> 1067&#226;&#8364;&#8220;1076.                                                                      Kumagai, A., Yakowec, P.S., Dunphy, W.G. (1998). 14-3-3 proteins act <br /> D&#226;&#8364;&#8482;Urso, G., Grallert, B., Nurse, P. (1995). DNA polymerase alpha,         negative regulators mitotic inducer Cdc25 Xenopus egg ex- <br /> component replication initiation complex, essential check-    tracts. Mol. Biol. Cell 9, 345&#226;&#8364;&#8220;354. <br /> point coupling S phase mitosis fission yeast. J. Cell Sci. 108, 3109&#226;&#8364;&#8220;     Kunkel, T.A., Roberts, J.D., Zakour, R.A. (1987). Rapid efficient <br /> 3018.                                                                           site-specific mutagenesis phenotypic selection. Methods Enzy- <br /> Downes, C.S., Mullinger, A.M., Johnson, R.T. (1991). Inhibitors          mol. 154, 367&#226;&#8364;&#8220;382. <br /> DNA topoisomerase II prevent chromatid separation mammalian cells            Lehmann, A.R., Walicka, M., Grittiths, D.J.F., Murray, J.M., Watts, F.Z., <br /> but prevent exit mitosis. Proc. Natl. Acad. Sci. USA 88,            McCready, S., Carr, A.M. (1995). The rad18 gene Schizosaccharomy- <br /> 8895&#226;&#8364;&#8220;8899.                                                                      ces pombe defines new subgroup SMC superfamily involved in <br /> Enoch, T., Carr, A., Nurse, P. (1992). Fission yeast genes involved      DNA repair. Mol. Cell. Biol. 15, 7067&#226;&#8364;&#8220;7080. <br /> coupling mitosis completion DNA replication. Genes &amp; Dev. 6,              Lindsay, H.D., Griffiths, D.J.F., Edwards, R.J., Christensen, P.U., Murray, <br /> 2035&#226;&#8364;&#8220;2046.                                                                      J.M., Osman, F., Walworth, N., Carr, A.M. (1998). S-phase-specific <br /> Furnari, B., Blasina, A., Boddy, M.N., McGowan, C.H., Russell, P.           activation Cds1 kinase defines subpathway checkpoint re- <br /> (1999). Cdc25 inhibited vivo vitro checkpoint kinases Cds1         sponse Schizosaccharomyces pombe. Genes &amp; Dev. 12, 382&#226;&#8364;&#8220;395. <br /> and Chk1. Mol. Biol. Cell 10, 833&#226;&#8364;&#8220;845.                                          Lopez-Girona, A., Furnari, B., Mondesert, O., Russell, P. (1999). Nu- <br /> Furnari, B., Rhind, N., Russell, P. (1997). Cdc25 mitotic inducer tar-      clear localization Cdc25 regulated DNA damage 14-3-3 <br /> geted chk1 DNA damage checkpoint kinase. Science 277, 1495&#226;&#8364;&#8220;1497.             protein. Nature 397, 172&#226;&#8364;&#8220;175. <br /> Furuya, K., Takahashi, K., Yanagida, M. (1998). Faithful anaphase        Losada, A., Hirano, M., Hirano, T. (1998). Identification Xenopus <br /> ensured Mis4, sister chromatid cohesion molecule required S phase       SMC protein complexes required sister chromatid cohesion. Genes &amp; <br /> and destroyed G1 phase. Genes &amp; Dev. 12, 3408&#226;&#8364;&#8220;3018.                      Dev. 12, 1986&#226;&#8364;&#8220;1997. <br /> Grawunder, U., Zimmer, D., Leiber, M.R. (1998). DNA ligase IV binds         Martinho, R.G., Lindsay, H., Flaggs, G., DeMaggio, A.J., Hoekstra, M.F., <br /> to XRCC4 motif located BRCT do-            Carr, A.M., Bently, N.J. (1998). Analysis rad3 chk1 protein <br /> mains. Curr. Biol. 8, 873&#226;&#8364;&#8220;876.                                                  kinases defines different checkpoint responses. EMBO J. 17, 7239&#226;&#8364;&#8220;7249. <br /> Guacci, V., Koshland, D., Strunnikov, A. (1997). A direct link      Michaelis, C., Ciosk, R., Nasmyth, K. (1997). Cohesins: chromosomal <br /> sister chromatid cohesion chromosome condensation revealed                  proteins prevent premature separation sister chromatids. Cell 91, <br /> through analysis MCD1 S. cerevisiae. Cell 91, 47&#226;&#8364;&#8220;57.                  35&#226;&#8364;&#8220;45. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                                2917 <br />  H.M. Verkade et al. <br />  <br />  <br /> Mitchison, J.M., Carter, B.L. (1975). Cell cycle analysis. Methods Cell        Saka, Y., Yanagida, M. (1993). Fission yeast cut51, required S phase <br /> Biol. 11, 201&#226;&#8364;&#8220;219.                                                                 onset M phase restraint, identical radiation-damage repair <br />                                                                                    gene rad41. Cell 74, 383&#226;&#8364;&#8220;393. <br /> Moreno, S., Klar, A., Nurse, P. (1991). Molecular genetic analysis of <br /> fission yeast Schizosaccharomyces pombe. Methods Enzymol. 194, 795&#226;&#8364;&#8220;823.            Sambrook, J., Fritsch, E., Maniatis, T. (1989). Molecular Cloning: a <br />                                                                                    Laboratory Manual, 2nd ed., Cold Spring Harbor, NY: Cold Spring Har- <br /> Muris, D.F.R., Vreeken, K., Carr, A.M., Murray, J.M., Smit, C., Lohman, <br />                                                                                    bor Laboratory Press. <br /> P.H.M., Pastink, A. (1996). Isolation Schizosaccharomyces pombe <br /> RAD54 homologue, rhp54, gene involved repair radiation                 Sanchez, Y., Wong, C., Thoma, R.S., Richman, R., Wu, Z., Piwnica-Worms, <br /> damage replication fidelity. J. Cell Sci. 109, 73&#226;&#8364;&#8220;81.                          H., Elledge, S.J. (1997). Conservation Chk1 checkpoint pathway <br />                                                                                    mammals: linkage DNA damage Cdk regulation Cdc25. <br /> Murray, A. (1995). The genetics cell cycle checkpoints. Curr. Opin. <br />                                                                                    Science 277, 1497&#226;&#8364;&#8220;1501. <br /> Genet. Dev. 5, 5&#226;&#8364;&#8220;11. <br />                                         <br /> </body></html>