<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL    OF   BACTERIOLOGY, OCt. 1993, p. 6562-6570                                                                           Vol. 175, No. 20 <br /> 0021-9193/93/206562-09$02.0o/() <br /> Copyright &#194;&#169; 1993, American Society Microbiology <br />  <br />  <br />           A cdc-Like Autolytic Saccharomyces cerevisiae Mutant Altered <br />              Budding Site Selection Is Complemented SP012, <br />                                Sporulation Gene <br />                 G. MOLERO, M. YUSTE-ROJAS, A. MONTESI, A. VAZQUEZ, C. NOMBELA, AND M. SANCHEZ* <br />                  Departamento Microbiologia II, Facultad Farmacia, Universidad Complutense, 28040-Madrid, Spain <br />                                                  Received 10 May 1993/Accepted 6 August 1993 <br />  <br />               LYT1 essential gene growth morphogenesis Saccharomyces cerevisiae. A detailed <br />            characterization mutants carrying lytJ-J allele showed <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> recessive pleiotropic, <br />            affecting mitotic meiotic functions. At nonpermissive temperature 37&#194;&#176;C, lyti haploid strains <br />            budded distal position (instead axial one, wild-type haploid strains) underwent autolysis <br />            buds size mother cells. These mitotic alterations cell stability budding <br />            topology dependent growth protein synthesis. Autolysis prevented inhibiting DNA synthesis <br />            (with hydroxyurea) blocking assembly microtubules (with benomyl), suggesting loss cell <br />            viability occur fixed mitotic cycle stage DNA synthesis mitotic spindle assembly. On the <br />            hand, lyti-Jllyti-l diploids failed sporulate 24 37&#194;&#176;C. Taking account these <br />            characteristics, lyti mutant considered cdc-like mutant. By genetic transformation an <br />            appropriate lyti strain genomic library, ligated multicopy vector YEpl3, isolated gene capable <br />            complementing mitotic alterations meiotic defect. This sporulation-specific gene SP012, <br />            expressed control locus ML4T meiosis expressed mitotic cycle (V. <br />            Parkes L. H. Johnston, Nucleic Acids Res. 20:5617-5623, 1992). A significant level SP012 mRNA can <br />            detected gene inserted multicopy plasmid. <br />  <br />    The isolation fungal mutants displaying thermosensitive              defect cell division cycle impairment <br /> autolytic <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> general strategy identify genes                  sporulation process. Studies carried S. cerevisiae <br /> controlling cell wall integrity cell viability. Mutants          Schizosaccharomyces pombe cdc mutants affected in <br /> type described Saccharomyces cerevisiae (6),                  mitotic meiotic cycles indicated primary defect is <br /> Candida albicans (25), Schizosaccharomyces pombe (29),                   related DNA synthesis, nuclear division, metabolism of <br /> Aspergillus nidulans (3). The characteristic shared                 phosphorylated nucleotides (22, 23, 37). <br /> mutant strains loss viability grown nonpermissive                 In paper characterization S. cerevisiae <br /> temperatures, leading release intracellular con-               lytl-], autolytic mutant, complemented osmotic <br /> tent. The wide variety autolytic mutants isolated             stabilization sorbitol (10), lyses grown 37&#194;&#176;C <br /> reflects complexity functions involved cell wall                   bud reaches size mother cell (21) and <br /> integrity cell viability. However, insight                 bears single pleiotropic recessive mutation affecting mitotic <br /> defect corresponding mutant obtained ana-                   meiotic functions. We cloned gene comple- <br /> lyzing complementation phenotypic defect osmotic                   menting mitotic defects, turned the <br /> stabilization sorbitol, precisely determined               sporulation gene SPO12. <br /> by means flow cytometry (10). Some mutants osmotic <br /> remedial, suggesting phenotypic defect pri- <br /> marily located cell wall (25, 36). Others minimally                               MATERIALS AND METHODS <br /> complemented complemented osmotic <br /> stabilizer, indication primary alteration probably                  Strains media. The S. cerevisiac strains used their <br /> does concern cell wall functions, eventually                characteristics listed Table 1. All yeast genetic proce- <br /> cells release intracellular content.                                   dures media described previously (32). The strains <br />    One aspect autolytic mutants                     Escherichia coli used cloning procedures DH5OS (11), <br /> examined relationship               JM109 (38), HB101 (4). <br /> the cell cycle. In S. cerevisiae autolytic mutants, cell lysis             YED (1% yeast extract 2% glucose), broth agar, was <br /> occurs particular stage cell cycle,            complete medium used growing yeast strains. Minimal <br /> lysed cells correspond cell cycle stage. On                 medium (MM), broth agar, used yeast strains contained <br /> hand, cell division cycle (cdc) mutants display loss            0.67% (wt/vol) Difco yeast nitrogen <span id='am-243' about='xsp:base' typeof='owl:Thing'>base</span> amino acids <br /> viability short incubation nonpermissive temper-              2% (wt/vol) glucose supplemented various <br /> ature, cells reach execution point, indicating             growth factors (amino acids nucleic acid bases) when <br /> they considered autolytic mutants,                  required, usually concentration 20 mg liter - 1, for <br /> detailed below. The number cdc mutants described amounts                  leucine (at 30 mg liter- 1). Sporulation induced 24&#194;&#176;C on <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> 70 S. cerevisiae (37), displaying               agar plates. The cells incubated presporulation <br />                                                                              agar containing 1% (wt/vol) yeast extract, 3% (wt/vol) beef <br />                                                                              extract, 5% glucose transferred sporula- <br />                                                                              tion agar containing 0.98% (wt/vol) potassium acetate, 0.25% <br />   * <br />       Corresponding author.                                                  (wt/vol) yeast extract, 0.1% (wt/vol) glucose. To prepare <br />                                                                       6562 <br />  VOL. 175, 1993                                                           GENE SP012 COMPLEMENTS cdc-LIKE YEAST MUTANTS                      6563 <br />  <br />                                                        TABLE 1. S. cerevisiae strains used work <br />         Strain                                                    Genotype                                                   Source <br /> S288Ca                     AL4Tao mal gal2                                                                     M. 0. Halvorson, Brandeis <br />                                                                                                                  University, Mass. <br /> 373                        MATa ade2                                                                           A. Jimenez, C.B.M., Madrid, Spain <br /> TD28                       MATa ura3A52 inol-131 canr                                                          Yeast Genetics Stock Center <br /> DBY-746                    MATcatrpl-28a leu2-3-112ura3A52 his3A]                                              Yeast Genetics Stock Center <br /> L2                         MATot lyt]-]                                                                        E. Cabib, National Institutes of <br />                                                                                                                  Health, Bethesda, Md. <br /> L2C24d                     MATa lytl-l ura3A52                                                                 This work <br /> L2C34d                     MATa lytl-l leu2-3,112 his4A34                                                      This work <br /> L2C28c                     MATa lyt]-] ura3A52                                                                 This work <br /> D (C34dxlb)                MATa/a lytl-lilytl-l                                                                This work <br /> DU (C24dxC28c)             MA Ta/ot lyt]-l/lytl-1 ura3A52/ura3A52                                              This work <br /> RH 207-lb                  AL4Ta cdc4-1 ade2 his3 leu2 trpl ura3                                               C. Kuhne Biozentrum, Basel, <br />                                                                                                                   Switzerland <br /> RH 209-4a                  MATa cdc13 ade2 his4 leu2 bar]                                                      C. Kuhne <br /> F-393                      MATa cdc14 his4A34 mal2 suc                                                         Yeast Genetics Stock Center <br /> RH 210-3c                  MATa cdc15 ade2-1 his4A34 trpl ura3 leu2 barl                                       C. Kuhne <br /> RH 204-ic                  MATa cdc23 ade2 his4 leu2 bar]                                                      C. Kuhne <br /> K 1414                     MATa cdc28-4 ura3 leu2 ade2-1 GAL&apos;                                                  C. Kuhne <br /> L119-7d                    MA4Tax dbf2-1 ura3A52 trp]/2 adel                                                   L. H. Johnston, National Institute <br />                                                                                                                   Medical Research, London, <br />                                                                                                                   United Kingdom <br /> 23ot113/EG123              mata]l/MATa trpl/trpl ura3/ura3 leu2/leu2 his4/his4                                 F. del Rey, Faculty Biology, <br />                                                                                                                   Salamanca, Spain <br /> 23a182/80                  mato2/MATa trpl/trpl ura3/URA3 leu2lleu2 his4/HIS4 LYSJ/lysl                         F. del Rey <br /> 23a7-12/al131-20           matA/MATot trp]/TRPI ura3/ura3 leu2ILEU2 his4/HIS4 ADE2/ade2 LEU/lieul               F. del Rey <br /> 23a20/a131-20              matal/MATTo trpl/TRPI ura3/ura3 leu2/LEU2 his4/HIS4 ADE2/ade2 LEUJ/leul              F. del Rey <br />  a <br />      Used wild-type strain generation L2. <br />  <br />  <br />  <br /> yeast protoplasts regeneration medium, MM broth sup-                         lum grown 24&#194;&#176;C YED medium. It used to <br /> plemented 1 M sorbitol, 0.4% YEPD broth, 3% agar.                       inoculate flasks containing YED medium, were <br /> For growing bacteria, Luria-Bertani (LB) medium containing                       incubated 24 37&#194;&#176;C, respectively. Samples taken <br /> 1% Bacto-tryptone, 0.5% yeast extract, 1% NaCl SOB                       2 h, percentage cells stained propidium <br /> medium, consisting 2% Bacto-tryptone, 0.5% yeast extract,                     iodide determined. <br /> 10 mM NaCl, 2.5 mM KCl, 10 mM MgCl2, 10 mM MgSO4,                               Budding <span id='am-245' about='xsp:pattern' typeof='owl:Thing'>pattern</span> analysis. A statistic analysis budding <br /> were used. Ampicillin supplemented, required,                      pattern cultures cells grown 24 37&#194;&#176;C, in <br /> concentration 100 ,ug/ml.                                                     YED medium MM (when plasmid selection re- <br />   Assessment lysis cell viability. The study lysis                  quired). Cells collected logarithmic phase of <br /> the cells nonpermissive temperature essential                      growth order count samples observation of <br /> part work. Colonies autolytic cells identified                budded cells phase-contrast microscope, their <br /> screening liberation intracellular enzyme                  axial distal budding pattern. <br /> alkaline phosphatase. This carried method                           Vectors genomic library. The shuttle vectors YEp13 (5) <br /> described previously (6), consisting overlaying plates                    YEp352 (13), carry genes LEU2 URA3, <br /> with soft agar layer containing p-nitrophenylphosphate                         respectively, selection markers, yeast multicopy <br /> (PNP) appropriate pH. Lysis manifested                          vectors used. YCp5O, URA3, low-copy- <br /> formation yellow halo hydrolysis                          number yeast vector used. A yeast genomic library (kindly <br /> phosphate substrate. Alternatively, used method                       supplied F. del Rey, Salamanca, Spain) based YEp13 <br /> based hydrolysis different substrate, 5-bro-                  vector (20) used transform yeast strain L2C34d. <br /> mo-4-chloro-3-indolylphosphate (BCIP), incorpo-                           Plasmids pEMBL18 pEMBL19 (7) used clon- <br /> rated agar concentration 40 ,ug/ml. When                        ing E. coli. Plasmids Bluescript KS SK (Stratagene, <br /> autolysis occurred, colonies progressively stained blue,                     Cultek) used sequencing. Standard methods were <br /> whereas remained white cells did lyse. The                       used plasmid preparation (14, 18). <br /> advantage method avoided overlaying                         Yeast genetic transformation. Genetic transformation S. <br /> plates.                                                                          cerevisiae spheroplast procedure according to <br />    These procedures allowed screen colonies                     method Beggs (2) intact cells according the <br /> autolytic cells. However, order analyze phenomenon                    method Ito et al. (15). <br /> more precisely, quantifying lysed cells cultures, used                     Cloning sequencing. Restriction analysis cloning in <br /> the method developed laboratory (10), based                     E. coli carried described previously (18). Restric- <br /> on staining lysed nonviable cells propidium iodide (PI)                 tion endonucleases T4 DNA ligase obtained <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> and analyzing population flow cytometry FACStar                      Boehringer Mannheim, Bethesda Research Laboratories, and <br /> PLUS flow cytometer equipped argon ion laser tuned                       New England Biolabs. Selected cloned fragments in- <br /> to 488 nm (300 mW).                                                              serted Bluescript plasmids amplified, strands <br />    cdc mutant lysis analysis. For cdc mutant, preinocu-                   separated alkaline denaturation sequenced the <br />  6564     MOLERO ET AL.                                                                                                      J. BACTERIOL. <br />  <br /> dideoxy sequencing technique Sanger et al. (31), using          blocked autolysis (Fig. 2), expected process <br /> the sequencing kit supplied Pharmacia LKB Biotechnology            dependent active growth cells. Our conclu- <br /> and appropriate oligonucleotide primers.                              sion based cycloheximide effect on <br />   RNA analysis. For Northern (RNA) blot analysis used          observation incubation mutant cells under <br /> check expression mitotic cycle, cells             conditions active growth detectable (media in <br /> incubated 24&#194;&#176;C initial logarithmic phase growth       carbon source nonfermentable carbohy- <br /> and 4 h 37&#194;&#176;C. Total RNA extracted, following          drate) did permit expression Lytl phenotype <br /> the method described previously (8). For kind             (data shown). Meiotic functions altered in <br /> analysis sporulation conditions, cultures grown        homozygous diploids lytl-l/lytl-1; incubation strains <br /> for 10 h sporulation media. Total RNA extracted             D(C34dxC21b) DU(C24dxC28c) presporulation and <br /> described Percival-Smith Segall (26).                          sporulation media did lead formation asci 24 or <br />   The RNA transferred Pall BiodyneA nylon mem-                 37&#194;&#176;C (data shown). <br /> branes hybridized probe labelled nick translation,         In order establish relationship lytl-l and <br /> as described previously (30), use Amersham            mutations affect cell cycle similarly (lysis the <br /> nick translation kit.                                                 later stages mitotic cycle), decided check lysis at <br />    PCR analysis gene disruptions. Disruption SP012          permissive nonpermissive temperatures cdc <br /> gene different strains demonstrated amplification        mutants. The results shown Table 2. As seen, <br /> the disrupted gene polymerase chain reaction (PCR).            cdc mutants display lytic phenotype (permeability to <br /> The oligonucleotides used, include beginning            PI) incubated 37&#194;&#176;C. <br /> the end SP012 open reading frame (ORF),                      We constructed heterozygous diploids cdc <br /> 5&apos;-CCCTCGAGATGTCCAACAAGGC-3&apos; 5&apos;-CCGTCG                            mutants execution points close lytl-1, as <br /> AC&apos;FTAGTAGAAATACTCC-3&apos; (the codons                  case cdc14, cdcl5, cdc23 mutations. The <br /> the open reading frame underlined).                               diploids 1ytl-1/cdcl4 1ytl-1/cdc23 grow 37&#194;&#176;C, <br />   The conditions used 10 min 99&#194;&#176;C denature             indicating lytl mutation different these. The <br /> DNA destroy nucleases, followed cycles 30       diploids 1ytl-lIcdclS showed significant levels PI staining <br /> s 94&#194;&#176;C, 1 min 72&#194;&#176;C, 5 min 45&#194;&#176;C. Twenty-five              8 h incubation 37&#194;&#176;C, 22 h incubation, the <br /> additional cycles (for total 30) hybridization    percentage cells stained PI 30%. lytl-l mutant <br /> temperature 55&#194;&#176;C. The result amplification              strains transformed CDC1S gene, could <br /> checked agarose gel analysis.                                      observe slight complementation autolytic phenotype and <br />                                                                       complementation inability sporulate homozy- <br />                             RESULTS                                   gous lytl-l/lytl-l diploids. These experiments close <br />                                                                       relationship genes. <br />    Phenotypic characterization S. cerevisiae lyti. S. cerevisiae      Another approach determine point cell cycle at <br /> L2 (kindly supplied E. Cabib, National Institutes Health        lytl-l mutation expressed use inhibitors <br /> Bethesda, Md.) autolytic mutant strain used           capable arresting progression mitotic cycle at <br /> investigation. Cells strain grow normally 24&#194;&#176;C,        defined points. Hydroxyurea (HU) inhibits DNA synthesis <br /> they lyse transferred nonpermissive temperature           (33). S. cerevisiae cells grown presence inhibitor <br /> of 37&#194;&#176;C; time lysis bud mother cell        typical doublet phenotype single nucleus. Cells of <br /> similar sizes. Strain L2 crossed wild-type strain 373,       strain L2C24d clearly protected lysis grown at <br /> and lytic segregants obtained cross            37&#194;&#176;C presence 0.5 M HU (Fig. 3); small fraction <br /> crossed wild-type strains. The characterization         PI-stained cells similar observed cultures of <br /> diploids analysis tetrads crosses gave          wild type, 100% mutant cells lysed in <br /> straightforward results: heterozygous diploids displayed       absence HU. Similar results obtained benomyl <br /> a wild phenotype, indicating dealing              (MCB), known interfere assembly of <br /> recessive mutation. The lytic phenotype analyzed          microtubules, leading phenotype doublets cells with <br /> successive meioses, consistently segregated 2:2,          short spindle body (28). Again, strain L2C24d 37&#194;&#176;C the <br /> lytic-nonlytic fashion. We conclude mutant phenotype         presence 0.5 mM MCB did lysis (Fig. 3). In both <br /> was determined mutation affecting single Mendelian             cases (HU MBC), cells washed 8 h of <br /> gene named LYTI (21).                                                 incubation 37&#194;&#176;C placed media HU MBC <br />    The comparison S. cerevisiae strain carrying lytl-l      incubated 37&#194;&#176;C. In cases, cells recovered the <br /> trait wild-type strain clearly showed autolytic     Lytl phenotype, certain delay. Consequently, loss of <br /> mutation affected growth cell viability 37&#194;&#176;C (Fig.        cell viability lysis lyti cells grown nonpermissive <br /> 1). Strain L2C34d (Iytl-J) (Table 1) grew normally 24&#194;&#176;C;           temperature occur fixed mitotic cycle stage takes <br /> almost 100% cells remained viable did        place DNA synthesis mitotic spindle assembly, stages <br /> stain PI determined flow cytometry (Fig. 1).               blocked, respectively, HU MCB. In respect lytl <br /> However, cultures 37&#194;&#176;C did double optical              mutant considered cdc-like mutant. <br /> density, cells lost viability               The phenotypic alterations determined lytl mutation <br /> stained PI. Phase-contrast microscopy revealed         restricted autolysis meiotic deficiencies <br /> of cells appeared dark, nonrefringent, buds       affected important aspect morphogenesis as <br /> size mother cell (Fig. 1). The results obtained      budding topology. Bud emergence wild haploid strains takes <br /> when lytl-l strain grown 37&#194;&#176;C presence        place axial position, contrary diploid strains in <br /> osmotic stabilizer sorbitol (0.9 M; data shown).          bud emerges distal previous daughter cell (9, <br /> These results confirm lethality lytl mutations          12, 35). As shown Table 3, statistical analysis budding <br /> nonpermissive temperature suggest lysis occurs        cultures haploid strain L2C24d (lyt1-1) revealed that <br /> last stages cell division cycle. Inhibition protein         bud emergence axial 240C, distal than <br /> synthesis 10 ,ug cycloheximide ml consistently            80% cell population 37&#194;&#176;C. The temperature effect on <br />  VOL. 175, 1993                                                 GENE SP012 COMPLEMENTS cdc-LIKE YEAST MUTANTS                           6565 <br />  <br />  <br />    A)                                                                         B) <br />                                                            1100 <br />          - <br />  <br />  <br />  <br />          0 <br />          0 <br />          0 <br />  <br />  <br />  <br />  <br />          0                       Hours <br />  <br />          0 <br />  <br />  <br />  <br />  <br />          co <br />          0 <br />  <br />  <br />  <br />  <br />                                  Hours <br />   FIG. 1. Viability growth wild-type autolytic strains S. cerevisiae <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> medium (YED) 24 37&#194;&#176;C. Growth was <br /> measured optical density (OD) determinations 600 nm; %PI(-), percentage viable (non-PI-stained) cells strain S288C (upper <br /> panels) L2C34d (lower panels). (A) Growth 24&#194;&#176;C (--E--) 37&#194;&#176;C (-*-) percentage non-PI-stained cells 24&#194;&#176;C (--O--) <br /> and 37&#194;&#176;C (-A-); (B) phase-contrast microscopy cells grown 37&#194;&#176;C. <br />  <br />  <br /> the mutant clearly contrast observations on <br /> the haploid wild-type strain budded axially regardless of <br /> the temperature. No change respect wild type was <br /> detected budding pattern homozygous lytl-l/lytl-l <br /> diploids 24 37&#194;&#176;C. <br />    A gene capable complementing phenotypic                  - 0, <br /> effects lyti mutation. In order clone gene         0 <br /> complement alterations determined lytl-l mutation, <br /> we constructed strain L2C34d (MA Ta lytl-l leu2-3, 112 <br /> his4A34), adequate genetic transformation genomic <br /> library ligated multicopy vector YEp13. Approximately              &#194;&#176;   r%   c <br />  <br /> 15,000 transformants obtained screened their <br /> ability grow 37&#194;&#176;C. This led selection one <br /> transformant grew nonpermissive temperature <br /> on plates MM. Further studies confirm complemen- <br /> tation carried growing cells liquid medium,                                      Hours <br /> showing 10% transformant cells lysed          FIG. 2. Effect cycloheximide (10 pLg/ml) viability growth <br /> after 22 h growth 37&#194;&#176;C. Transformed cells displayed            autolytic mutant L2C34d <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> medium 37&#194;&#176;C. Growth <br /> normal appearance phase-contrast microscopy (Fig. 4). The           measured optical density (OD) determinations 600 nm; <br /> selected transformant carried plasmid 5.5-kb insert       %PI( -), percentage viable (non-PI-stained) cells. Growth (Qx) and <br /> S. cerevisiae DNA, mapped subcloned               percentage non-PI-stained cells (O) absence cyclohex- <br /> smallest fragment capable complementing lytic pheno-            imide growth (0) percentage non-PI-stained cells (V) in <br /> type. Subcloning YEp13 (LEU2 selection           presence cycloheximide shown. <br />  6566            MOLERO ET AL.                                                                                                                                     J. BACTERIOL. <br />  <br />                     TABLE 2. Results lysis permissive                         TABLE 3. Statistical analysis budding pattern S. cerevisiae <br />                           nonpermissive temperatures                                                   haploid strains 24 37&#194;&#176;C <br /> Incubation temp              % cells stained PI incubation for:                                                            Budding pattern at: <br />   strain mutant              4h         6h          8h                                        Strain <br />                                                                   10 h         12 h                                                  24&#194;&#176;C                              37&#194;&#176;C <br />        240C                                                                           S288C (wild type)                        Axial (&gt;95%)                   Axial (&gt;97%) <br />          S288C                  0           0           0.2        0.5          0.8   L2C24d (lytl-1)                          Axial (100%)                   Distal (&gt;80%) <br />          cdc4                   0.1         0.8         0.9        0.5          2     L2C24d (lytl-l SPOJ2&quot;)                   Axial (100%)                   Axial (100%) <br />          cdc13                  0.1         0.7         7          3.9          4 <br />          cdc14                  0.1         0.5         0          0.1          0.3     Numbers parentheses indicate percentages cells indicated <br />          cdclS                  2.5         1.3         1.1        0.7          3     budding patterns. <br />                                                                                         b Transformed multicopy plasmid carrying SP012 gene. <br />          cdc23                  2           1.6         1.8        1.8          2 <br />          cdc28                  1.1         1.6         2.2        3            4 <br />        370C                                                                           marker), complementation capacity veri- <br />          S288C                  0.5        0.3         2           2.3          0.8   fied L2C34d, YEp352 (URA3 selection marker), <br />          cdc4                   4         20          44          68           88     with, case, L2C24d L2C28c (Table 1) check <br />          cdc13                 17         30          50          55           81 <br />          cdc14                 12         62          79          87           99     complementation autolytic phenotype. <br />          cdclS                  3         31          60          68           87        Several plasmid constructions tested, clear <br />          cdc23                 17         30          60          74           85     results. A subfragment size 1.74 kb (Fig. 5) retained <br />          cdc28                 16         35          49          69           87     capacity originally cloned fragment restore growth <br />                                                                                       37&#194;&#176;C transformed strains. The 1.7-kb subfragment was <br />                                                                                       sequenced analyzed reveal carried open <br />                                                                                       reading frames opposite orientation. Furthermore, checking <br />  <br />  <br />   A)                                                                                  B)                                                                               100 <br />  <br />  <br />          0r- <br />                 3                                                                           0 <br />          to <br />          CD                                                                                to <br />         &quot; <br />                 2                                                                          I-I <br />  <br />  <br />  <br />  <br />         0                                                                                  0D <br />  <br />  <br />                                       Hours                                                                                Hours <br />  <br />  <br />   C)                                                                     100 <br />                                                                                       D)           2                                                                   100 <br />                                                                                                                                 -A   - <br />                                                                                                                                          -   - <br />                                                                                                                                                  -A   -   -   -   I&amp; <br />  <br />  <br />                                                                                                                                                                        80 <br />          09 <br />                 3-                                                                                                                                                            - <br />  <br />                                                                                             0                                                                          60 <br />         CO  &quot; <br />                                                                                             CD     1 <br />                 2 <br />                                                                                                                                                                        40 M <br />                                                                                                                                                  m <br />         0                                                                                   o                                        -   - <br />                                                                                                                                               -5 <br />                                                                                                  0,5~ <br />                                                                                                                      oll <br />                                                                                                                            v                                           20 <br />  <br />                                                                                                    nA <br />                                                                                                        0         2              4                6                8 <br />                                       Hours                                                                                    Hours <br />    FIG. 3. Effect HU (A C) MCB (B D) viability growth <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> medium (YED) wild-type autolytic strains <br /> of S. cerevisiae 37&#194;&#176;C. Growth measured optical density (OD) determinations 600 nm presence (-) absence (El) the <br /> corresponding inhibitor. %PI(-), percentage viable (non-PI-stained) cells presence (A) absence (O) corresponding <br /> inhibitor. (A B) Strain S288C; (C D) strain L2C34d. <br />  VOL. 175, 1993                                                                            GENE SP012 COMPLEMENTS cdc-LIKE YEAST MUTANTS                                             6567 <br />  <br />  <br />     A)                                                                                                    B) <br />                                                                                       DO <br />                                                                                     I I <br />  <br />                     6&apos; &lt;=~~~~~~~~~~~~~I <br />                                                                                       0 oo4 <br />      0 <br />                                                                                             04 <br />             4                                         ,/                          6 <br />                                                                                         o <br />                                                                                       0 <br />      0                                                                                .oK <br />             2 2&apos;n <br />                  2 <br />             %.F <br />                 0      5           10    15         20        25             30 <br />                                         Hours <br />   FIG. 4. Viability growth autolytic strain L2C34d transformed SP012 MM 24 37&#194;&#176;C. Growth measured optical <br /> density (OD) determinations 600 nm. %PI(-), percentage viable (non-PI-stained) cells. (A) Growth 24&#194;&#176;C (S) 37&#194;&#176;C (0) the <br /> percentage non-PI-stained cells 24&#194;&#176;C (O) 37&#194;&#176;C (A); (B) phase-contrast microscopy cells grown 37&#194;&#176;C. <br />  <br />  <br /> the Sequence Data Bank (EMBL <span id='am-241' about='Thesaurus:Swiss_Prot' typeof='owl:Thing'>SWISS PROT</span>) led                                              copies, complemented alterations determined lytl mu- <br /> the somewhat unexpected finding open reading                                        tation led loss viability lysis cells grown at <br /> frames corresponded genes SP012 SP016, in-                                          37&#194;&#176;C. Complementation achieved using the <br /> volved sporulation process,                                        low-copy-number plasmid (YCp5O) carrying gene SP012, <br /> cloned (17). Our sequencing data (not shown) did differ                                      showing requirement multiple copies cloned gene <br /> from data reported. We constructed plasmids                                         complementation, suggesting gene fact <br /> disruptions SP012 SP016, insertion URA3                                    suppressor lytic phenotype. Northern blot analysis, <br /> in adequate restriction site (Fig. 5), order test                                 performed cells beginning logarithmic growth <br /> gene responsible complementation autolytic                                    phase, revealed expression SP012 minimal, at <br /> phenotype. The results clearly demonstrated plasmid                                     24 37&#194;&#176;C, wild-type S288C untransformed <br /> with disrupted SP016 gene retained complementation                                       mutant strain L2C34d. On hand, significant tran- <br /> capacity lost disrupting SP012. These observa-                                     scription gene occurred transformants with <br /> tions enabled conclude gene SP012, multiple                                    multiple copies gene, consistent complementa- <br />                                                                                                  tion effect, limited transcription detected the <br />                                                                                                  transformants SP012 included low-copy-number plas- <br />                                                                                                  mids, expected lack complementation <br />                                                                                                  conditions (Fig. 6). Parkes Johnston (24) <br />          1.74kb                                                    0.50 kb                       concluded SP012 gene expressed some <br />                      ECON                                                                        stages mitotic cycle, confirming results. <br />                             PVUN         HkVW <br />       HMM Hib                                     CNW Nei i <br />        [.                    1. <br />                              A <br />                                    Be ftoll <br />                                                          WO 12                                                                       1        2   3 <br />  <br />                                                                                                                                                        -    0.8 <br />                                                 m                                                                              S.&quot;                          0.73 <br />                             URA 3 <br />  <br />                      Eoo <br />                             PuUN         Hhdlll <br />                                                     Il      rv <br />                                   SPO le 1                         ~~~~12 <br />  <br />                                                                                                       FIG.   6.   Autoradiography          hybridization analysis (Northern blot) <br />                                                                                                      probe including SPOJ2 ORF 20 jig total                  RNA the <br />  <br />                                                          URA 3                                   following S. cerevisiae strains incubated                   370C:   L2C34d   (lane 1), <br />                                                                                                  L2C24d transformed SPOI2 gene cloned                           centromeric <br />   FIG. 5.   Scheme arrangement         SP0I2              SP0J6   genes        plasmid      (YCp5O)       (lane 2), L2C34d transformed with <br />                                                                                                                                        SPO&lt;2 <br />  <br /> in genome S. cerevisiae disruption genes                               cloned            multicopy plasmid (YEp13) (lane 3).               The    sizes   (in <br /> URA3 insertion plasmid constructions.                                                         kilobases)         bands shown right. <br />  6568        MOLERO ET AL.                                                                                                        J. BACTERIOL. <br />  <br />                        1 2 3 4                                               The SP012 gene recently described (24) a <br />                                                                           suppressor dbf2 mutation. The Dbf2 phenotype appar- <br />                                                                           ently similar Lytl phenotype. To discern it <br />                                                                           mutation, constructed heterozygous diploids <br />                                                                           dbf2/lytJ-1, able grow 37&#194;&#176;C lysis. We <br />                                                                           conclude different mutations. <br />                                            - 0.8 <br />                                            - 0.73                                                   DISCUSSION <br />                                                                               Autolytic mutants identify genes essential cell <br />                                                                            growth viability. Therefore, characterization these <br />                                                                            mutants contribute characterization corre- <br />                                                                            sponding essential cellular functions. One genes is <br />                                                                           LYT], needed growth viability yeast cell at <br />   FIG. 7.   Autoradiography hybridization analysis (Northern blot)      37&#194;&#176;C. The evidence reported clearly shows lyti <br /> of probe including SP012 ORF 20 ,ug total RNA S.          mutation characterized work pleiotropic reces- <br /> cerevisiae strains incubated sporulation media different <br /> mutations 1MT genes. Lanes: 1, 23ao1 13/EG123 (matotl/MATa); 2, <br />                                                                            sive, affecting mitotic meiotic processes. <br /> 23ot182/80 (mata2/MATa); 3, 23a7-12/a131-20 (mata2/MATot); 4,                 The fact lysis lytl-l cells prevented the <br /> 23a20/ox131-20 (matal/MA To). As observed, SP012 mRNA           presence osmotic stabilizer sorbitol suggests the <br /> (0.73 0.8 kb) detected lanes genes MATal    formation defective cell wall, corresponding <br /> and MA Ta2 mutated, conditions locus AAT              release intracellular content, direct effect the <br /> remains functional. The sizes (in kilobases) bands shown     mutation consequence expressed pleiotropically <br /> right.                                                                     major genetic alteration. <br />                                                                               Active growth protein synthesis required the <br />                                                                            expression alteration lytl-l mutants, the <br />                                                                            autolytic mitotic effects occurred cells reached a <br />    Figure 7 shows expression SP012 gene                certain stage DNA synthesis microtubule assembly. <br /> cultures grown sporulation conditions (see Materials                Thus, LYT1 gene involved completion of <br /> and Methods) controlled locus MAT,                mitotic cell cycle crucial cell viability. <br /> SP012 detected gene MATal AL4Toa2                           So, dealing cdc mutant, is <br /> mutated.                                                                  commonly accepted cdc mutants stop growing not <br />    SP012 multiple copies restored growth                  suffer autolysis. We surprised observe cdc <br /> prevented lysis 37&#194;&#176;C restored axial budding               mutants studied stained PI high proportion after <br /> pattern transformed haploid lytl strain (Table 3).                 hours incubation 37&#194;&#176;C (Table 2). <br /> However, complementation sporulation deficiency                     In order check relationship lytl-l muta- <br /> was observed transformed homozygous lytl-l/lytl-l diploids.            tion cdc mutations execution points close <br /> In order confirm SP012 different gene                lyt1-1, case cdc14, cdcl5, cdc23, <br /> LYTJ, disrupted SP012 URA3 insertion strains TD28                constructed herozygous diploids. The diploids constructed <br /> and DBY746. These disrupted haploid strains wild-type               cdc14 cdc23 mutations able grow 37&#194;&#176;C, <br /> phenotype temperatures, 24 37&#194;&#176;C, regard               indicating lytl-l mutation different these. In <br /> lysis mitotic cycle. On hand, disrupted          case cdcl5, heterozygous diploids 1ytl-1!cdcl5 able <br /> allele spol2::URA3 determined apomictic sporulation ho-                grow 37&#194;&#176;C suffer certain autolysis. On hand, <br /> mozygous diploids (data shown), confirmation                 transformation autolytic strains CDC1S gene <br /> results described previously (17), certifies               renders slight complementation autolytic phenotype <br /> disruption SP012 gene.                                             complete complementation meiotic impairment of <br />   Heterozygous diploids 1ytl-1/spol2::URA3 constructed               lytl-l mutant strains. So, different genes, <br /> and able sporulate. Tetrad analysis revealed                close relationship LYT1 and <br /> different kinds asci: (i) 2 asci ascospores          CDC15 gene products. <br /> (two wild type Lytl) viable, (ii) 12 asci              The effects lytl mutation bud emergence were <br /> only ascospores (two wild type Lytl)            remarkable. It known selection site bud <br /> viable, (iii) 5 asci ascospores (two wild           emergence random S. cerevisiae. Buds emitted <br /> type) viable.                                                        axial position haploid cells distal position <br />    These results indicated SP012 suppressor,               diploids (9, 12, 35). A large number mutants displaying <br /> the existence nonviable spores interpreted meaning              altered budding patterns described previously (16). <br /> that cosegregation lytl-l spol2::URA3                From data, set genes directly involved bud site <br /> determined inviability corresponding spore. Disruption             selection identified. The products these <br /> of SP012 gene lytl-l strains L2C28c L2C24d                 genes closely related proteins involved GDP-GTP <br /> (Table 1) led isolation viable disruptants, indicating          exchange. Moreover, GTP levels play crucial role the <br /> that coincidence mutations makes ascospore                    regulation meiotic mitotic processes (16). Our results <br /> germination inviable vegetative growth 240C. The               clearly LYT1 function crucial bud site <br /> disruption confirmed methods: (i) crossing                 selection completion mitotic and <br /> 1ytl-1/spol2::URA3 strain spol2::URA3 strain (the                 meiotic cycle. Its relationship genes controlling <br /> diploids, sporulated, reproduced apomictic pheno-                budding topology remains established. Other cdc mu- <br /> type) (ii) PCR amplification spol2::URA3 construction              tants, namely, cdc24, cdc42, cdc43 (1, 34), share lytl <br /> from genomic DNA disrupted strains.                            alteration budding pattern. But cdc mutants <br />  VOL. 175, 1993                                                GENE SP012 COMPLEMENTS cdc-LIKE YEAST MUTANTS                             6569 <br />  <br /> or bud mutants described far               gene modification, currently addressing question <br /> phenotypic traits lytl (16, 37).                                 relevant domains SP012 biological activity in <br />    The observations homozygous diploids lytl-li           context. <br /> lytl-l revealed effect mutation, namely, the <br /> inability sporulate, alteration nonconditional                          ACKNOWLEDGMENTS <br /> constitutive expressed 24 37&#194;&#176;C. In view       This work supported grants B1089/0387-C02-01 the <br /> of mutagen used, ethyl methane sulfonate,           Comisi6n Interministerial Ciencia y Tecnologia (CYCIT), Spain, <br /> postulated <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> occurred          BIOT-CT90-0165 (BRIDGE) Commission Euro- <br /> gene LYT1 mutant allele lytl-l lost functionality       pean Community, awarded C.N. G.M. M.Y.-R. were <br /> more drastically regard role meiosis         recipients fellowships Ministerio Educaci6n y Ciencia. <br /> mitosis. The LYT1 gene product functional               We thank P. A. Sansegundo F. del Rey help one <br /> domains, mitotic meiotic functions, affected          experiment Mr. Alberto Alvarez help flow cytometric <br /> differently consequence mutation. Similar phenom-       studies. <br /> ena described genes RAD6 (27) CDC25 <br /> (19). Or, alternatively, single domain carry                                    REFERENCES <br /> functions different requirements functionality. It    1. Adams, A. E. M., D. I. Johnson, R. M. Longnecker, B. F. Sloat, and <br /> also postulated function carried LYTI            J. R. Pringle. 1990. CDC42 CDC43, additional genes <br />                                                                         involved budding establishment cell polarity the <br /> gene product essential mitotic cycle 37&#194;&#176;C.            yeast Saccharomyces cerevisiae. J. Cell Biol. 111:131-142. <br />    The possibility raised pleiotropic pheno-       2. Beggs, J. D. 1978. Transformation yeast replicating hybrid <br /> type Lytl different mutations. However,          plasmid. Nature (London) 275:104-109. <br /> given multiple crosses lytl-l wild-type strains      3. Borgia, P. B., C. L. Dodge. 1992. Characterization Aspergil- <br /> and fact cdc mutants carrying single              lus nidulans mutants deficient cell wall chitin glucan. J. <br /> mutation mitotic meiotic cycles affected,           Bacteriol. 174:377-383. <br /> reasonably assume mutant carries single pleiotropic       4. Boyer, H. W., D. Roulland-Dussoix. 1969. A complementation <br /> mutation responsible phenotypic defects              analysis restriction modification DNA Escherichia <br /> different mutations gene.                                   coli. J. Mol. Biol. 41:459-472. <br />                                                                      5. Broach, J. R., J. N. Stathern, J. B. Hicks. 1979. Transforma- <br />     The search gene capable complementing phe-             tion yeast: development hybrid cloning vector isolation <br /> notype lytl-l mutants led isolation suppressor,          CANI gene. Gene 8:121-133. <br /> gene SP012, structural gene LYTI. The                6. Cabib, E., A. Duran. 1975. Simple sensitive procedure for <br /> explanation LYT1 having isolated apparent,          screening yeast mutants lyse nonpermissive temperatures. <br /> since number transformants screened considered            J. Bacteriol. 124:1604-1606. <br /> sufficient isolate gene. One explanation toxicity    7. Dente, L., G. Cesareni, R. Cortese. 1983. pEMBL: new <br /> of LYT1 product multiple copies. Therefore,               family single stranded plasmids. Nucleic Acids Res. 11:1645- <br /> currently attempting clone LYT1 gene screening            1655. <br /> transformants obtained centromeric library. In case,      8. Elder, R. T., E. Y. Loh, R. W. Davis. 1983. RNA yeast <br />                                                                         transposable element Tyl ends direct repeats, a <br /> another remarkable finding work capacity          structure similar retrovirus RNA. Proc. Natl. Acad. Sci. USA <br /> gene SP012, known involvement meiotic               80:2432-2433. <br /> functions, complement phenotypic alterations lytl-l        9. Freifelder, D. 1960. Bud formation Saccharomyces cerevisiae. J. <br /> mutants affect mitotic functions. Multiple copies SP012         Bacteriol. 80:567-568. <br /> are needed complementation mitotic defects,         10. Fuente, J. la, A. Alvarez, C. Nombela, M. Sanchez. 1992. <br /> namely, loss bud emergence site selection viability. The         Flow cytometric analysis Saccharomyces cerevisiae autolytic <br /> observed increase mitotic transcription SP012                mutants protoplasts. Yeast 8:39-45. <br /> carried plasmid consistent dose-effect depen-      11. Hanahan, D. 1985. Techniques transformation E. coli, p. <br />                                                                          120. In D. M. Glover (ed.), DNA cloning, vol. I. IRL Press, <br /> dence suppression mutant phenotype (Fig. 6).              Oxford. <br /> According Malavasic Elder (17), SP012 expressed           12. Herskowitz, I. 1989. A regulatory hiera <br /> </body></html>