<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                         PLoS BIOLOGY <br />  <br /> Caenorhabditis elegans HCF-1 Functions <br /> in Longevity Maintenance DAF-16 <br /> Regulator <br /> Ji Li1, Atsushi Ebata1, Yuqing Dong1&#194;&#164;, Gizem Rizki1, Terri Iwata1, Siu Sylvia Lee1* <br /> Department Molecular Biology Genetics, Cornell University, Ithaca, New York, United States America <br />  <br />  <br /> The transcription factor DAF-16/forkhead box O (FOXO) critical longevity determinant diverse organisms, <br /> however molecular basis transcriptional activity regulated remains largely unknown. We report that <br /> the Caenorhabditis elegans homolog host cell factor 1 (HCF-1) represents new longevity modulator functions <br /> as negative regulator DAF-16. In C. elegans, hcf-1 inactivation caused daf-16-dependent lifespan extension up <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> 40% heightened resistance specific stress stimuli. HCF-1 showed ubiquitous nuclear localization and <br /> physically associated DAF-16. Furthermore, loss hcf-1 resulted elevated DAF-16 recruitment promoters <br /> of target genes altered expression subset DAF-16-regulated genes. We propose HCF-1 modulates C. <br /> elegans longevity stress response forming complex DAF-16 limiting fraction DAF-16 <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> accessing target gene promoters, regulates DAF-16-mediated transcription selective target genes. As <br /> HCF-1 highly conserved, findings important implications aging FOXO regulation mammals. <br /> Citation: Li J, Ebata A, Dong Y, Rizki G, Iwata T, et al. (2008) Caenorhabditis elegans HCF-1 functions longevity maintenance DAF-16 regulator. PLoS Biol 6(9): e233. <br /> doi:10.1371/journal.pbio.0060233 <br />  <br />  <br />                                                                                           responses [13,14]. Mammalian FOXOs shown to <br /> Introduction <br />                                                                                           regulate expression antioxidant enzymes, gluconeogenic <br />    Recent studies various model revealed                                   enzymes, cell cycle regulators, apoptotic genes [13]. <br /> multiple evolutionarily conserved genes genetic pathways                              Similarly, C. elegans DAF-16 regulates expression large <br /> important longevity [1&#226;&#8364;&#8220;6]. One best characterized                              number target genes, including involved metabo- <br /> longevity determinants forkhead box O (FOXO) family                                lism, stress response, immunity [15&#226;&#8364;&#8220;17]. DAF-16 thought <br /> of transcription factors, function major effectors                            promote <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>survival</span> longevity mounting robust <br /> the insulin/insulin-like growth factor (IGF)-1-like signaling                             response various stresses, infection, toxic compounds. <br /> (IIS) cascade. The IIS pathway highly conserved                                Therefore, precise control DAF-16 transcriptional <br /> been shown modulate longevity Caenorhabditis elegans,                               activities key regulatory step longevity determination. <br /> Drosophila, mice [7]. Activation insulin/IGF receptor                             Similar DNA binding transcription factors, one <br /> tyrosine kinases triggers kinase cascade, involving                                 way DAF-16/FOXO achieve speci&#239;&#172;?city gene regu- <br /> phosphoinositide 3-kinase (PI3K) serine/threonine                                 lation depends functional interactions transcrip- <br /> kinases AKT-1 AKT-2, culminates cytoplas-                                tional co-regulators. Recent &#239;&#172;?ndings revealed several <br /> mic sequestration inhibition FOXO transcription                                nuclear factors cooperate DAF-16/FOXO regulate <br /> factors [7]. In addition aging, IIS critical                               gene expression. In mammals, FOXOs deacetylated by <br /> regulating development, metabolism, stress response. In                               protein deacetylase SIRT1 [18&#226;&#8364;&#8220;22]. SIRT1-mediated <br /> C. elegans, reduced signaling IIS pathway,                            deacetylation leads promotion FOXO activation of <br /> caused mutations IIS receptor daf-2 PI3K age-                            stress response genes concurrent inhibition FOXO- <br /> 1, results dramatic increase lifespan, heightened                                 regulated apoptotic genes [18]. Interestingly, SIRT1 homologs <br /> resistance wide variety environmental stress stimuli, <br /> and altered metabolism development [8,9]. Loss daf-16, <br />                                                                                           Academic Editor: Andy Dillin, The Salk Institute, United States America <br /> the C. elegans homolog FOXO, completely suppresses the <br /> phenotypes associated IIS de&#239;&#172;?ciency C. elegans [10&#226;&#8364;&#8220;12],                           Received June 23, 2008; Accepted August 15, 2008; Published September 30, <br />                                                                                           2008 <br /> indicating daf-16 major effector IIS worms. <br />                                                                                           Copyright: &#195;&#8220; 2008 Li et al. This open-access article distributed the <br />    From C. elegans mammals, DAF-16/FOXO emerging                               terms Creative Commons Attribution License, permits unrestricted <br /> master regulator capable responding diverse                                 use, distribution, reproduction medium, provided original author <br /> environmental stimuli coordinating development, metab-                                source credited. <br /> olism, stress response [1,13]. In addition IIS pathway,                        Abbreviations: ChIP, chromatin immunoprecipitation; co-IP, co-immunoprecipita- <br /> DAF-16/FOXO responds signaling cascades.                               tion; FOXO, forkhead box O transcription factor; GFP, green fluorescent protein; <br />                                                                                           HCF-1, host cell factor 1; HSV, herpes simplex virus; IGF, insulin-like growth factor; <br /> Recent &#239;&#172;?ndings reveal different signals induce distinct                              IIS, insulin/IGF signaling; PI3K, phosphoinositide 3-kinase; qRT-PCR, quantitative <br /> modi&#239;&#172;?cations DAF-16/FOXO, impact                                         reverse transcription PCR; RNAi, <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br /> expression level, subcellular localization, and/or transcriptional                        * To correspondence addressed. E-mail: SSL29@cornell.edu <br /> activities DAF-16/FOXO, leading expression changes                               &#194;&#164; Current address: Department Biological Sciences, Clemson University, <br /> selective DAF-16/FOXO target genes speci&#239;&#172;?c cellular                                   Clemson, South Carolina, United States America <br />  <br />  <br />         PLoS Biology | www.plosbiology.org                                         1870                                    September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                       HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br /> Author Summary                                                                   Results <br />                                                                                  hcf-1 Is Novel Longevity Gene <br /> One key molecules modulate longevity evolutionarily <br />                                                                                     In recent genome-wide RNA interference (RNAi) screen <br /> diverse organisms transcription factor DAF-16/FOXO. Despite <br /> its importance aging biological processes, DAF-16/              new longevity genes [30], RNAi knock down <br /> FOXO activity regulated nucleus largely unknown. We                 hcf-1 gene consistently caused C. elegans live ;20%&#226;&#8364;&#8220; <br /> report new player important aging modulation, nematode                 30% longer control RNAi treated worms (Figure 1A; <br /> homolog host cell factor 1 (HCF-1), functions           Table 1). C. elegans hcf-1 encodes protein highly <br /> negative regulator DAF-16. In worms, HCF-1 inactivation extends               conserved evolution, biological function is <br /> lifespan 40% increases resistance specific stress stimuli.          just beginning elucidated. Mammalian HCF-1 &#239;&#172;?rst <br /> To affect lifespan stress response, HCF-1 requires activity           identi&#239;&#172;?ed host cell factor essential stabilizing the <br /> DAF-16. We HCF-1 protein expressed nucleus               transcriptional complex involving herpes simplex virus <br /> and partners DAF-16 worms. Furthermore, demonstrate <br />                                                                                  (HSV) VP-16 transcription factor [29]. Mammalian HCF-1 has <br /> that loss HCF-1 results elevated levels DAF-16 the <br /> promoters target genes altered expression subset              subsequently shown play key roles cell cycle <br /> DAF-16-regulated genes. We propose HCF-1 modulates                          progression, G1/S transition, M phase and <br /> longevity stress response binding DAF-16 preventing                cytokinesis [31&#226;&#8364;&#8220;33]. In eliciting diverse biological roles, <br /> the transcription factor accessing target gene promoters,               mammalian HCF-1 acts binding regulating many <br /> thereby regulating expression DAF-16 target genes. As HCF-1               different transcription chromatin factors assembling <br /> is highly conserved, findings important implications                appropriate protein complexes context-dependent gene <br /> aging FOXO regulation humans.                                             expression regulation [29]. C. elegans HCF-1 shown to <br />                                                                                  complement transcriptional role mammalian <br />                                                                                  HCF-1 [34] required proper cell cycle <br /> in yeast (Sir2), C. elegans (sir-2.1), Drosophila (dSir2)                progression worms (Figure S2) [35]. Our data ascribe new <br /> important longevity determinants over-expression                        longevity function HCF-1. To investigate role of <br /> results longevity increase [23]. In C. elegans, SIR-2.1 forms               hcf-1 C. elegans longevity, obtained mutant alleles <br /> protein complex DAF-16 [24,25] requires DAF-16                          (ok559 pk924) [35] hcf-1 examined lifespan. <br /> activity modulate lifespan [26]. Furthermore, nuclear                     The hcf-1(ok559) mutant in-frame deletion should <br /> factor SMK-1 b-catenin homolog BAR-1 recently                       truncate N-terminal half protein, hcf- <br /> been shown promote DAF-16-mediated transcription                              1(pk924) mutant large deletion result a <br /> [27,28]. C. elegans lacking smk-1 bar-1 shortened                        frame shift leading early stop codon, likely <br /> lifespan, similar worms lacking daf-16. BAR-1                         represents null mutant (Figure S1) [35]. As expected on <br /> cooperates DAF-16 elicit proper oxidative stress                         basis observation hcf-1 RNAi worms, we <br /> response [28], SMK-1 important roles DAF-16                    detected 40% lifespan extension hcf-1 mutants <br /> in lifespan modulation speci&#239;&#172;?c stress response [27].                    compared wild-type worms (Figure 1B; Table 1). In general, <br />    Whereas SIR-2.1, SMK-1, BAR-1 represent putative                          hcf-1(pk924) mutant lives slightly longer hcf- <br /> positive regulators DAF-16 C. elegans, nuclear factors                1(ok559) mutant (Table 1), consistent possibility that <br /> negatively regulate DAF-16 largely unknown. Since DNA                        pk924 allele severe <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span>. We con&#239;&#172;?rmed that <br /> binding transcription factors regulated                         prolonged lifespan associated hcf-1 mutants is <br /> interplay positive negative regulators,                          hcf-1 de&#239;&#172;?ciency demonstrating expression a <br /> characterization DAF-16 negative regulators                           C-terminal green &#239;&#172;&#8218;uorescent protein (gfp)-tagged hcf-1 trans- <br /> essential elucidation DAF-16 regulation. In                   gene (hcf-1::gfp) able partially restore normal lifespan in <br /> this paper, report C. elegans homolog host cell                   hcf-1(pk924) mutant (Table S2). <br /> factor 1 (HCF-1) represents new longevity determinant and <br /> functions negative regulator DAF-16. HCF-1 belongs                    HCF-1 Is Ubiquitously Expressed Localizes the <br /> a family highly conserved proteins [29]. Loss hcf-1 C.                  Nucleus <br /> elegans induces substantial lifespan extension 40%                     To characterize HCF-1 C. elegans, examined <br /> robust resistance speci&#239;&#172;?c stress stimuli. For hcf-1                        expression <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> HCF-1 worms. Using af&#239;&#172;?nity- <br /> modulate lifespan stress response, requires activity                  puri&#239;&#172;?ed polyclonal HCF-1 antibody immunostaining <br /> of daf-16. In delineating mechanism HCF-1                           assays, observed prominent HCF-1 staining nuclei <br /> regulates DAF-16, HCF-1 ubiquitously                            most, all, somatic germline cells wild-type <br /> expressed nuclear protein physically associates DAF-                   worms (Figure 2). The HCF-1 expression pattern detected in <br /> 16. Moreover, loss hcf-1 led increased recruitment                 wild-type worms highly speci&#239;&#172;?c hcf-1(pk924) and <br /> DAF-16 target gene promoters altered expression                    hcf-1(ok559) mutants showed background &#239;&#172;&#8218;uorescence <br /> a subset DAF-16-regulated genes. Given genetic                        examined using identical immunostaining <br /> biochemical data, propose HCF-1 modulates lifespan                       conditions (Figure 2) (unpublished data). The nuclear local- <br /> and stress responses forming complex DAF-16                        ization HCF-1 consistently observed embryo <br /> restricting recruitment fraction DAF-16 target                larval adult stages (Figure 2) (unpublished data) <br /> gene promoters, regulating DAF-16-mediated tran-                         [35]. Moreover, observed similar ubiquitous nuclear <br /> scription speci&#239;&#172;?c target genes. Considering HCF-1                    expression worms expressing low-copy number a <br /> DAF-16 highly conserved evolution,                          functional hcf-1::gfp transgene (Figure S3). Our results indicate <br /> &#239;&#172;?ndings suggest HCF-1 likely regulates FOXO                            HCF-1 ubiquitously expressed nucleus C. <br /> activities important aging diverse organisms.                      elegans normal culture condition. Mammalian HCF-1 is <br />  <br />       PLoS Biology | www.plosbiology.org                                  1871                            September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                    HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br />  <br /> Figure 1. hcf-1 Modulates Lifespan Acting Upstream daf-16, Parallel IIS Germline Signaling Pathways <br /> Lifespan (A) wild-type worms treated hcf-1 RNAi, (B) hcf-1(ok559) hcf-1(pk924) deletion mutants, (C) daf-16(mgDf47);hcf-1(pk924) <br /> double mutant, (D) daf-2(e1370);hcf-1(pk924) double mutant, (E) age-1(mg44);hcf-1(ok559) double mutant, (F) glp-1(e2141);hcf-1(pk924) <br /> double mutant worms. Each lifespan experiments repeated independent times similar results. Data representative <br /> experiments shown. Quantitative data statistical analyses experiments shown included Table 1. <br /> doi:10.1371/journal.pbio.0060233.g001 <br />  <br />  <br /> also predominantly nuclear protein [36] nuclear                     epistatic relationship hcf-1 daf-16. We <br /> localization thought important role gene                  created double mutants containing hcf-1(pk924) hcf- <br /> expression regulation.                                                        1(ok559) null daf-16(mgDf47) mutations. We that <br />                                                                               daf-16(mgDf47);hcf-1(pk924) daf-16(mgDf47);hcf-1(ok559) <br /> hcf-1 Modulates Lifespan daf-16-Dependent Manner                         double mutant lifespan indistinguishable of <br />   To characterize hcf-1 modulates C. elegans lifespan,                 daf-16(mgDf47) single mutant, ;20% shorter than <br /> asked hcf-1 genetically interact known                   wild-type worms [37] (Figure 1C; Tables 1 S1). We <br /> longevity factors C. elegans. Because DAF-16/FOXO                   obtained similar results hcf-1(ok559) mutant worms <br /> of best characterized longevity determinants, tested                   treated daf-16 RNAi (Table S1). Our results indicate that <br />  <br />       PLoS Biology | www.plosbiology.org                               1872                             September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                            HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br /> Table 1. Inactivation hcf-1 Results Lifespan Increase That Is Completely Dependent daf-16, Likely Independent IIS <br /> and Germline Pathways <br />  <br /> Strain                             Mean LS 6          Total Number Animals               Percent wt         p-Value Versus          p-Value Versus Strain <br />                                    SEM (d)            Died/Total                                                  wt <br />                                                                                                                                           Strain                             p-Value <br />  <br /> Wt &#195;&#190; L4440a                        13.9   6   0.2     46/47                                 &#226;&#8364;&#8221;                     &#226;&#8364;&#8221;                       wt &#195;&#190; L4440                         N.A. <br /> Wt &#195;&#190; hcf-1a                        16.5   6   0.5     47/49                                 119%b                 &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                                  ,0.0001 <br /> Wtc,d                              14.7   6   0.1     88/88                                 &#226;&#8364;&#8221;                     N.A.                    hcf-1(ok559)                       ,0.0001 <br /> hcf-1(ok559)c,d                    16.5   6   0.3     89/89                                 112%                  ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br /> hcf-1(pk924)c,d                    19.3   6   0.2     95/95                                 131%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> Wtb                                15.5   6   0.2     85/90                                 &#226;&#8364;&#8221;                     N.A.                    &#226;&#8364;&#8221;                                  N.A. <br /> hcf-1(pk924)e                      22.2   6   0.4     87/90                                 143%                  ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br /> Wt                                 14.7   6   0.1     88/88                                 &#226;&#8364;&#8221;                     N.A.                    daf-16(mgDf47);hcf-1(pk924)        ,0.0001 <br /> daf-16(mgDf47)                      9.2   6   0.1     94/94                                 63%                   ,0.0001                 &#226;&#8364;&#8221;                                    0.2622 <br /> hcf-1(pk924)                       19.3   6   0.2     95/95                                 131%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> daf-16((mgDf47);hcf-1(pk924)        9.4   6   0.1     97/97                                 64%                   ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br /> Wt                                 14.7   6   0.1     88/88                                 &#226;&#8364;&#8221;                     N.A.                    daf-2(e1370);hcf-1(pk924)          ,0.0001 <br /> daf-2(e1370)                       28.7   6   0.9     87/88                                 195%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> hcf-1(pk924)                       19.3   6   0.2     95/95                                 131%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> daf-2(e1370);hcf-1(pk924)          40.4   6   1.3     90/90                                 275%                  ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br /> Wt                                 13.4   6   0.3     50/50                                 &#226;&#8364;&#8221;                     N.A.                    age-1(mg44);hcf-1(ok559)           ,0.0001 <br /> age-1(mg44)                        20.0   6   1.0     46/46                                 149%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> hcf-1(ok559)                       15.6   6   0.6     51/51                                 116%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> age-1(mg44);hcf-1(ok559)           26.7   6   1.1     44/44                                 196%                  ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br /> Wt                                 13.6   6   0.2     53/56                                 &#226;&#8364;&#8221;                     N.A.                    glp-1(e2141);hcf-1(pk924)          ,0.0001 <br /> glp-1(e2141)                       18.0   6   0.4     74/75                                 132%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> hcf-1(pk924)                       17.5   6   0.6     65/66                                 129%                  ,0.0001                 &#226;&#8364;&#8221;                                  ,0.0001 <br /> glp-1(e2141);hcf-1(pk924)          21.4   6   0.7     62/62                                 157%                  ,0.0001                 &#226;&#8364;&#8221;                                  N.A. <br />  <br />  <br /> The lifespan (LS) experiments repeated independent times similar results, data representative experiments shown. The lifespan data analyzed <br /> using Log-rank test p-values individual experiment shown. N.A., applicable; wt, wild type. <br /> a <br />  &#195;&#190; RNAi. <br /> b <br />   Percent wt &#195;&#190; L4440. <br /> c <br />  Representative experiment 1. <br /> d <br />   Results presented Figure 1. <br /> e <br />   Representative experiment 2. <br /> doi:10.1371/journal.pbio.0060233.t001 <br />  <br />  <br /> hcf-1 requires activity daf-16 modulate longevity C.                                We obtained similar results age-1(mg44);hcf-1(ok559) <br /> elegans suggest hcf-1 novel upstream                                         double mutant. age-1(mg44) null mutant and, when <br /> regulator daf-16.                                                                           maintained homozygous strain, exhibits unconditional <br />                                                                                                dauer arrest <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> [10,38]. To avoid this, collected <br />  hcf-1 Likely Acts Parallel IIS Germline                                         age-1(mg44) age-1(mg44);hcf-1(ok559) homozygous mutant <br /> Pathways Modulate Lifespan                                                                  adults born age-1(mg44)/&#195;&#190; age-1(mg44)/&#195;&#190;;hcf-1(ok559) <br />    Given epistatic relationship hcf-1 daf-16,                                  parents lifespan analysis. These age-1(mg44) hcf- <br /> we wondered hcf-1 modulates C. elegans lifespan                                     1(ok559);age-1(mg44) worms completely lacked zygotic age-1 <br /> functioning IIS pathway, well-established upstream                                    expression; however, inherited suf&#239;&#172;?cient maternal age-1 <br /> regulator daf-16. To test this, examined genetic                                     message develop normally [38]. Similar previous results <br /> interactions hcf-1 major components                                     [38], age-1(mg44) zygotic null mutant worms lived much <br /> IIS pathway: daf-2/insulin/IGF receptor age-1/PI3K. We                                     longer wild-type worms (Figure 1E; Table 1). The age- <br /> reasoned hcf-1 normally affects C. elegans lifespan                                 1(mg44);hcf-1(ok559) double mutant worms lived considerably <br /> acting IIS pathway, loss hcf-1                                 longer age-1(mg44) hcf-1(ok559) single <br /> major impact longevity worms lacking IIS                                     mutant (Figure 1E; Table 1), exhibited lifespan increase <br /> signaling. We created double mutants containing hcf-                                       greater sum effect single <br /> 1(pk924) hcf-1(ok559) daf-2(e1370) age-                               mutations. RNAi knock daf-2 age-1 hcf- <br /> 1(mg44) mutations. Consistent previous &#239;&#172;?ndings [11],                                  1(ok559) mutant gave similar results (Table S1). The genetic <br /> daf-2(e1370) temperature-sensitive mutant showed ;2-fold                                    results described indicate loss hcf-1 loss IIS <br /> increase lifespan compared wild-type worms                                        act synergistically extend lifespan C. elegans suggest <br /> nonpermissive temperature 25 8C (Figure 1D; Tables 1                                       hcf-1 likely functions pathway parallel IIS. <br /> S1). Interestingly, daf-2(e1370);hcf-1(pk924) daf-                                      Although hcf-1(pk924) mutation putative null <br /> 2(e1370);hcf-1(ok559) double mutant lived considerably longer                                  mutation, daf-2(e1370) mutation temperature sensitive <br /> than daf-2(e1370) hcf-1(pk924) hcf-1(ok559)                                   mutation, small maternal AGE-1 protein <br /> single mutant (Figure 1D; Tables 1 S1), exhibited                                    persisted age-1(mg44);hcf-1(ok559) double <br /> lifespan increase greater sum effect                               mutant, remains possible loss hcf-1 increases lifespan <br /> the single mutations.                                                                      decreasing daf-2 signaling. <br />  <br />          PLoS Biology | www.plosbiology.org                                             1873                                    September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                       HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />                                                                                  To assay response acute oxidative stress, we <br />                                                                                  challenged wild-type hcf-1 mutant adult worms a <br />                                                                                  high-dose paraquat, superoxide-inducing agent, and <br />                                                                                  monitored survival. We hcf-1(pk924) <br />                                                                                  mutant worms considerably resistant the <br />                                                                                  paraquat treatment compared wild-type worms multiple <br />                                                                                  time points experiment (Figure 3A). More- <br />                                                                                  over, paraquat resistance hcf-1(pk924) mutants was <br />                                                                                  dependent daf-16, daf-16(mgDf47);hcf-1(pk924) double <br />                                                                                  mutant sensitive paraquat, similar daf- <br />                                                                                  16(mgDf47) single mutant (Figure 3B). Interestingly, hcf- <br />                                                                                  1(ok559) mutant worms survival kinetics similar to <br />                                                                                  wild-type worms paraquat assay, suggesting that <br />                                                                                  hcf-1(ok559) mutant, long-lived, more <br />                                                                                  resistant high dose paraquat treatment compared to <br />                                                                                  wild-type worms. No HCF-1 protein detected hcf- <br />                                                                                  1(pk924) mutant, truncated protein accumu- <br />                                                                                  lates hcf-1(ok559) mutant (Figure S1). Therefore, is <br />                                                                                  possible resistance high level oxidative stress only <br />                                                                                  apparent HCF-1 completely lost. <br />                                                                                     To assay response heavy metal stress, challenged <br />                                                                                  wild-type hcf-1 mutant adult worms cadmium [42] and <br />                                                                                  monitored survival. Similar observed the <br />                                                                                  paraquat assay, hcf-1(pk924) mutant worms more <br />                                                                                  resistant cadmium exposure compared wild-type <br /> Figure 2. HCF-1 Is Ubiquitously Expressed Nuclear Protein                      worms multiple time points (Figure 3C). The cadmium <br /> (A) Gravid adults wild-type hcf-1(pk924);sur-5::gfp mutant worms          resistance hcf-1(pk924) mutant daf-16-depend- <br /> were immunostained using affinity-purified HCF-1 antibody. To                 ent (Figure 3C). <br /> ensure identical staining conditions, worms strains                  To assay response heat stress, shifted adult wild- <br /> processed slide. The hcf-1(pk924) mutants marked by <br /> SUR-5::GFP distinguish wild-type worms. Endogenous HCF- <br />                                                                                  type hcf-1 mutant worms 35 8C monitored their <br /> 1 localize nucleus most, all, somatic          survival. We hcf-1 mutants wild-type <br /> germline cells wild-type worms. The hcf-1(pk924) mutant worms                 worms behaved similarly time course of <br /> marked SUR-5::GFP showed background signal. DAPI staining                heat shock treatment (Figure 3D). As previously reported <br /> was used indicate nucleus. Photos taken 2003 <br /> magnification.                                                                   [43], age-1(hx546) mutant worms survived longer and <br /> (B) A magnified image head region wild-type worm shown             daf-16(mgDf47) mutant worms died faster wild- <br /> in (A).                                                                          type worms 35 8C. We concluded loss hcf-1 <br /> doi:10.1371/journal.pbio.0060233.g002 <br />                                                                                  did result altered response acute heat shock. Taken <br />                                                                                  together, results indicate loss hcf-1 results in <br />    Since germline proliferation implicated C.                        worms resistant paraquat cadmium exposure, <br /> elegans lifespan modulation, wondered                             heat shock, suggesting hcf-1 required for <br /> extended lifespan hcf-1 mutant related brood                    speci&#239;&#172;?c stress response. <br /> size defect mutant (Figure S2) [35]. In regard,                     Considering DAF-16 plays key role dauer <br /> tested lifespan double mutant glp-1(e2141);hcf-                       formation fat metabolism C. elegans [10,44,45], we <br /> 1(pk924). The glp-1(e2141) mutant completely lacks germline                      tested hcf-1 mutants exhibit dauer fat <br /> cells nonpermissive temperature 25 8C long-lived                   phenotypes. We hcf-1(ok559) mutant exhibits <br /> [39]. If hcf-1 mutant long-lived partial defect                dauer phenotype, hcf-1(pk924) null mutant <br /> in germline proliferation, expect hcf-1                            exhibits weak dauer exit phenotype. When monitored 25 <br /> de&#239;&#172;?ciency affect lifespan worms completely                         8C, typical temperature testing strong dauer <br /> lacking germline cells. We glp-1(e2141);hcf-                      phenotype, hcf-1 mutants developed normally, whereas <br /> 1(pk924) double mutant lived longer glp-                           daf-2(e1370) mutant formed 100% dauer (Table 2). When <br /> 1(e2141) hcf-1(pk924) single mutants 25 8C (Figure 1F;                     monitored 27 8C, temperature commonly used test for <br /> Table 1). Therefore, loss hcf-1 continue extend                    weak dauer formation phenotype, hcf-1 mutant <br /> lifespan worms completely lack germline cells                    behaved differently wild-type worms (Table 2). Lastly, <br /> sterile, suggesting hcf-1 function lifespan                        tested daf-2;hcf-1 double mutants 22 8C, which <br /> modulation role germline brood                         represents well-established sensitized condition [46] for <br /> size regulation.                                                                 assaying weak dauer recovery phenotype. Single double <br />                                                                                  mutant worms harboring daf-2(e1370) mutation were <br /> Loss hcf-1 Promotes Resistance Specific                                    incubated semi-nonpermissive temperature 22 8C. <br /> Environmental Stress                                                             Under condition, daf-2(ts) worms enter dauer ;3 d <br />   Since DAF-16/FOXO known regulate stress                             recover reproductive adults. We found <br /> responses [5,40,41], tested hcf-1 mutants                     hcf-1(pk924) mutation prevented dauer exit, whereas <br /> exhibit differential response environmental stress stimuli.                   hcf-1(ok559) mutation effect (Table 2). These <br />  <br />       PLoS Biology | www.plosbiology.org                                  1874                            September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                      HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br />  <br /> Figure 3. Loss hcf-1 Results Heightened Resistance Specific Environmental Stresses <br /> (A) The hcf-1(pk924) mutant worms exhibited increased survival 200-mM paraquat compared wild-type worms. <br /> (B) The enhanced paraquat resistance hcf-1(pk924) dependent daf-16. <br /> (C) The hcf-1(pk924) mutant worms showed increased survival CdCl2 (18mM) daf-16 dependent. <br /> (D) The hcf-1(pk924) hcf-1(ok559) mutants wild-type worms showed similar survival kinetics cultured 35 8C. The age-1(hx546) daf- <br /> 16(mgDf47) worms included controls previously reported resistant sensitive paraquat, heavy metal, heat <br /> shock, respectively [41&#226;&#8364;&#8220;43]. <br /> For stress assays, duplicate quadruplicate samples examined strain. Mean fraction alive indicates average survival the <br /> multiplicates error bars represent standard deviation multiplicates. p-Value calculated using Student&#226;&#8364;&#8482;s t-test. *, p , 0.05 when <br /> compared wild-type (wt). **, p , 0.05 compared hcf-1(pk924). <br /> Each stress assays repeated independent times similar results, data representative experiments shown. <br /> doi:10.1371/journal.pbio.0060233.g003 <br />  <br />  <br /> results indicate loss hcf-1 associated weak                  subcellular localization DAF-16, did detect any <br /> dauer phenotype. Using vital dye Nile Red, stains                    altered DAF-16 localization hcf-1-de&#239;&#172;?cient worms (Figure <br /> lipid droplets worms represents sensitive way                      4A). Using quantitative reverse transcription PCR (qRT-PCR) <br /> monitor fat storage [47], did detect substantial                    immunoblotting examine RNA protein <br /> differences fat storage hcf-1 mutants wild-type                 expression levels daf-16 major components the <br /> worms (Figure S4).                                                             IIS pathway, including daf-2, age-1, akt-1, observed no <br />                                                                                obvious differences expression levels hcf- <br /> HCF-1 Regulates Expression Subset DAF-16                           1(ok559) hcf-1(pk924) mutants compared wild-type <br /> Regulated Genes                                                                worms (Figure 4B) (unpublished data). Taken together, our <br />    Our genetic data suggest hcf-1 acts upstream daf-16                 results suggest HCF-1 likely affect the <br /> to affect C. elegans lifespan stress responses. Since                      subcellular localization expression level DAF-16. <br /> increased DAF-16 nuclear localization transcriptional                         To test HCF-1 affect transcriptional <br /> activities shown extend lifespan C. elegans,                activities DAF-16, measured message levels DAF- <br /> tested HCF-1 regulate localization, expres-                    16-regulated genes hcf-1 mutant wild-type worms <br /> sion level, and/or transcriptional activities DAF-16. Using                 using qRT-PCR. sod-3, encodes iron/manganese <br /> transgenic worms expressing GFP-fused DAF-16 monitor                        superoxide dismutase, best characterized DAF- <br />  <br />        PLoS Biology | www.plosbiology.org                               1875                             September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                           HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />                                                                                                       16 target genes [17,40] transcription directly up- <br /> Table 2. Inactivation hcf-1 Results Weak Dauer Exit                                           regulated DAF-16 [48]. Interestingly, the <br /> Phenotype.                                                                                            RNA level endogenous sod-3 signi&#239;&#172;?cantly elevated 3&#226;&#8364;&#8220;4- <br />                                                                                                       fold hcf-1(ok559) hcf-1(pk924) mutants as <br /> Temperature (8C)             Strain                           Total             Percent               compared wild-type worms (Figure 5A). The elevated <br />                                                               Number            Dauer              expression sod-3 hcf-1 mutant worms completely <br />                                                                                                       dependent daf-16 daf-16(mgDf47);hcf-1(ok559) <br /> 25                           wt                               .100                0 <br />                                                                                                       double mutant, level sod-3 expression remained low <br />                              daf-2(e1370)                     .100              100 <br />                              hcf-1(ok559)                     .100                0                   similar seen daf-16(mgDf47) single <br /> 27                           wt                               .100                0                   mutant worms (Figure 5A). In corroboration qRT-PCR <br />                              hcf-1(ok559)                     .100                0                   results, observed elevated levels GFP expression Psod- <br /> 22                           wt                                159                0                   3::gfp transgenic worms, express GFP reporter driven <br />                              daf-2(e1370)                      311               21.9 <br />                              hcf-1(ok559)                       85                0                   sod-3 promoter, hcf-1(pk924) mutant background <br />                              daf-2(e1370);hcf-1(ok559)         131               22.1                 (Figure S5). <br /> 25                           wt                                203                0                      To investigate hcf-1 generally affect the <br />                              daf-2(e1370)                      113              100                   transcriptional activities DAF-16, surveyed additional <br />                              hcf-1(pk924)                       87                0 <br />                              daf-2(e1370);hcf-1(pk924)          41              100 <br />                                                                                                       DAF-16-regulated genes reported previous microarray <br /> 27                           wt                                160                1.3                 studies [17]. The previous studies focused DAF-16 <br />                              hcf-1(pk924)                       61                1.6                 targets responsive IIS veri&#239;&#172;?ed the <br /> 22                           wt                                200                0                   genes chose test exhibit daf-2/daf-16 responsiveness <br />                              daf-2(e1370)                      130               23.1 <br />                              hcf-1(pk924)                      124                0 <br />                                                                                                       assaying conditions (Table 3). Among 11 DAF- <br />                              daf-2(e1370);hcf-1(pk924)          35               82.9                 16-activated genes examined, addition to <br />                                                                                                       sod-3, expression levels mtl-1, encodes metal- <br /> Synchronized populations worms obtained egg-lay 16 8C shifted <br />                                                                                                       lothionein, F21F3.3, encodes farnesyl cysteine- <br /> to indicated temperature development. Dauers scored days                     carboxyl methyltransferase, showed statistically signi&#239;&#172;?cant, <br /> temperature shift. hcf-1(ok559) allele did exhibit dauer phenotypes 22 8C, 25 8C,          greater 2-fold up-regulation hcf-1 mutants <br /> and 27 8C. hcf-1(pk924) allele exhibited enhanced dauer exit phenotype sensitized <br /> condition 22 8C daf-2(e1370) mutant background showed dauer                          compared wild-type worms (Figure 5A; Table 3). The <br /> phenotype 25 8C 27 8C. A representative experiment shown. wt, wild type.                    elevated expression mtl-1 hcf-1 mutant worms is <br /> doi:10.1371/journal.pbio.0060233.t002 <br />                                                                                                       completely dependent daf-16 (Figure 5A; Table 4) and <br />                                                                                                       F21F3.3 partially dependent daf-16 (Figure 5A; <br />  <br />  <br />  <br />  <br /> Figure 4. Loss hcf-1 Does Not Result Altered DAF-16 Subcellular Localization Change DAF-16 Expression Level <br /> (A) Transgenic worms over-expressing DAF-16::GFP (daf-16(mgDf47);xrIs87) treated vector L4440 control RNAi, hcf-1 RNAi, daf-2 <br /> RNAi 16 8C 5 d. DAF-16::GFP exhibited diffuse expression pattern control RNAi hcf-1 RNAi knock worms. hcf-1 RNAi was <br /> able substantially reduce HCF-1 levels (bottom right panel). daf-2 RNAi included positive control known stimulate robust nuclear <br /> localization DAF-16::GFP. Photos showed DAF-16::GFP expression pattern DIC images live day 2 gravid adults. Nuclear localization was <br /> verified using DIC. A total ;60&#226;&#8364;&#8220;70 worms scored, percentage worms showing DAF-16::GFP nuclear localization shown the <br /> photo. Worm extracts DAF-16::GFP worms treated control hcf-1 RNAi immunoblotted using anti-HCF-1 antibody ([A], <br /> bottom right panels). <br /> (B) The RNA levels daf-16, daf-2, age-1, akt-1 wild-type, hcf-1(ok559), hcf-1(pk924) worms quantified using qRT-PCR. The data three <br /> independent experiments pooled, mean normalized RNA level standard error mean (SEM) gene hcf-1 mutant <br /> and wild-type worms shown. act-1 used internal control, RNA level gene normalized act-1 level. The mean <br /> normalized RNA level gene wild-type (wt) worms set 1. None genes tested showed significant expression change the <br /> hcf-1 mutants compared wild-type worms. <br /> doi:10.1371/journal.pbio.0060233.g004 <br />  <br />         PLoS Biology | www.plosbiology.org                                                     1876                           September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                      HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br />  <br /> Figure 5. Loss hcf-1 Promotes DAF-16 Transcriptional Regulation Several Target Genes <br /> (A) The expression sod-3, mtl-1, F21F3.3 elevated C32H11.4 repressed hcf-1 mutants. The elevated expression sod-3 <br /> and mtl-1 hcf-1 mutants completely dependent daf-16; F21F3.3 C32H11.4 partially dependent daf-16. *, p , 0.05 when <br /> compared wild-type (wt). **, p , 0.05 compared hcf-1(ok559). <br /> (B) The expression sod-3 mtl-1 daf-2(e1370);hcf-1(pk924) double mutant showed synergistic up-regulation compared the <br /> expression hcf-1(pk924) daf-2(e1370) single mutant. *, p , 0.05 compared wild-type (wt). **, p , 0.05 compared daf- <br /> 2(e1370). The quantitative data summarized Tables 3&#226;&#8364;&#8220;5. <br /> The RNA levels sod-3, mtl-1, F21F3.3, C32H11.4 quantified using qRT-PCR normalized internal control act-1. The data least <br /> three independent experiments pooled, mean normalized RNA level SEM gene indicated strains shown. The mean <br /> normalized RNA level gene wt worms set 1. p-Value calculated using Student&#226;&#8364;&#8482;s t-test. For sod-3 mtl-1 expression, analyzed <br /> for synergistic effect daf-2(e1370);hcf-1(pk924) compared daf-2(e1370) hcf-1(pk924) using two-way ANOVA analysis. <br /> doi:10.1371/journal.pbio.0060233.g005 <br />  <br />  <br /> Table 4). Among DAF-16-repressed genes examined,                        cooperate DAF-16 regulate F21F3.3 and <br /> we expression level C32H11.4,                           C32H11.4 expression hcf-1 mutants. <br /> encodes protein unknown function, showed statistically                      We noticed expression hsp-16.1, which <br /> signi&#239;&#172;?cant, greater 2-fold down-regulation hcf-1                    encodes heat shock protein, fat-5, encodes delta- <br /> mutants compared wild-type worms (Figure 5A; Table 3).                       9 fatty acid desaturase, signi&#239;&#172;?cantly up-regulated and <br /> Similar F21F3.3, repressed expression                         repressed, respectively, hcf-1 mutants (Table 3). <br /> C32H11.4 partially dependent daf-16 (Figure 5A; Table                     However, altered expression fat-5 hsp-16.1 the <br /> 4). The partial dependency suggests additional factors                     hcf-1 mutants did require daf-16 (unpublished data). These <br />  <br />        PLoS Biology | www.plosbiology.org                                1877                             September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                             HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br /> Table 3. Inactivation hcf-1 Results Significant Expression Changes Subset DAF-16 Regulated Genes <br />  <br /> Gene Category                      Gene Id        Fold Change             Fold Change                    p-Value t-Test       Description <br />                                                   hcf-1(pk924)/wt            daf-2(-)/wt 6 SEM                 (hcf-1(pk924) <br />                                                   6 SEM                                                        Compared wt) <br />  <br /> DAF-16 up-regulated                hsp-16.1a      5.93   6   3.23              5.49   6   0.74                 0.0301                  Heat shock protein <br />                                    F21F3.3b       5.39   6   1.27b            10.90   6   2.06b                0.0260b                 Farnesyl cysteine-carboxyl methyltransferase <br />                                    mtl-1b         4.69   6   1.18b            96.96   6   11.07b               0.0352b                 Metallothionein <br />                                    sod-3b         3.63   6   0.84b            83.45   6   12.98b               0.0355b                 Iron/manganese superoxide dismutase <br />                                    dod-3          2.04   6   1.08            192.60   6   15.49                0.3879                  Uncharacterized protein <br />                                    ges-1          1.56   6   0.66              2.06   6   0.12                 0.4438                  Carboxylesterase expressed gut cells <br />                                    M02D8.4        1.44   6   0.45              2.99   6   0.49                 0.3815                  Protein similar asparagine synthase <br />                                    K09C4.5        1.19   6   0.21              2.18   6   0.06                 0.4181                  Permease major facilitator superfamily <br />                                    hsp-12.3       1.07   6   0.07              5.90   6   0.18                 0.3909                  Heat shock protein <br />                                    lys-7          0.85   6   0.05              2.15   6   0.16                 0.0408                  Antimicrobial lysozyme <br />                                    fat-5a         0.32   6   0.08              3.34   6   0.14                 0.0009                  Delta-9 fatty acid desaturase <br /> DAF-16 down-regulatedb             C32H11.4b      0.11   6   0.01b             0.04   6   0.01b                0.0000b                 Uncharacterized protein <br />                                    F08G5.6        0.61   6   0.03              0.46   6   0.03                 0.0003                  Uncharacterized protein <br />                                    F35E12.5       0.61   6   0.13              0.08   6   0.01                 0.0387                  Uncharacterized protein <br />                                    T16G12.1       1.11   6   0.18              0.47   6   0.01                 0.5616                  Puromycin-sensitive aminopeptidase and <br />                                                                                                                                        related aminopeptidases <br />                                    dod-24         1.49 6 0.30                  0.01 6 0.00                     0.1750                  Uncharacterized protein <br />                                    dod-22         1.49 6 0.44                  0.16 6 0.07                     0.1243                  Uncharacterized protein <br />  <br />  <br /> The RNA levels multiple DAF-16 regulated genes quantified using qRT-PCR. The data independent experiments pooled, mean fold-change in <br /> expression standard error mean (SEM) gene hcf-1 mutant compared wild-type (wt) worms shown. act-1 used internal control, RNA level of <br /> each gene normalized act-1 level. p-Value calculated using Student&#226;&#8364;&#8482;s t-test comparison means. <br /> The gene list selected class 1 class 2 genes reported DAF-16 microarray study [17]. Up-regulation down-regulation DAF-16 verified experimental <br /> conditions. <br /> a <br />  The expression changes hcf-1(-) genes daf-16 dependent. <br /> b <br />   Genes significant expression changes  2-fold hcf-1(pk924) mutant (p , 0.05), expression change daf-16 dependent. <br /> doi:10.1371/journal.pbio.0060233.t003 <br />  <br />  <br /> results suggest hsp-16.1 fat-5 robust DAF-                                      Consistent genetic data hcf-1 appears to <br /> 16 targets response daf-2 signaling (Table 3),                                   act parallel daf-2 signaling affect lifespan, the <br /> appear signi&#239;&#172;?cantly regulated DAF-16 response                                       expression sod-3 mtl-1 synergistically up-regulated <br /> hcf-1 de&#239;&#172;?ciency. Moreover, fat-5 activated DAF-                                      daf-2(e1370);hcf-1(pk924) double mutant compared to <br /> 16 response reduced daf-2 signaling, expression                                      daf-2(e1370) hcf-1(pk924) single mutant (Figure 5B; <br /> repressed hcf-1 mutant daf-16-independent manner.                                      Table 5). Taken together, results suggest hcf-1 only <br /> Many DAF-16 downstream genes surveyed likely                                         able affect expression selective DAF-16 target genes. <br /> regulated multiple different transcription factors. For                                         Moreover, data indicate hcf-1 inactivation leads to <br /> example, hsp-16.1 likely regulated heat shock                                       daf-16-dependent up-regulation different DAF-16 <br /> factor HSF-1. It possible hcf-1 mutants, different                                  activated genes daf-16-dependent down-regulation a <br /> transcription factor(s) play major role mediating                                         DAF-16 repressed gene, suggesting hcf-1 normally <br /> expression change hsp-16.1 fat-5, DAF-16 plays                                     participates inhibition DAF-16 transcriptional <br /> a minor role involved gene expression                                           activity. Importantly, dependence hcf-1 daf-16 to <br /> regulation. The expression remaining DAF-16 target                                          elicit gene expression changes correlates require- <br /> genes did signi&#239;&#172;?cant change hcf-1                                         ment daf-16 hcf-1-mediated lifespan modulation, <br /> mutants, expression change 2-fold.                                          suggesting ability HCF-1 regulate DAF-16 <br />  <br />  <br /> Table 4. Expression Changes Several HCF-1 Regulated Genes Are daf-16 Dependent <br />  <br /> Gene Id               Fold Change                   Fold Change                        Fold Change                                      p-Value t-Test <br />                       hcf-1(ok559)/wt                  daf-16(mgDf47)/wt                     daf-16(mgDf47);hcf-1(ok559)/wt                      (daf-16(mgDf47);hcf-1(ok559) <br />                       6 SEM                            6 SEM                                 6 SEM                                               Compared daf-16(mgDf47)) <br />  <br /> sod-3                 3.12   6   0.70                  0.38   6   0.06                       0.30   6   0.08                                     0.4740 <br /> mtl-1                 5.64   6   1.62                  0.15   6   0.04                       0.39   6   0.17                                     0.5451 <br /> F21F3.3               4.66   6   0.78                  0.40   6   0.10                       1.20   6   0.34                                     0.0665 <br /> C32H11.4              0.09   6   0.02                  2.01   6   0.43                       0.60   6   0.12                                     0.0333 <br />  <br />  <br /> qRT-PCR performed analyzed described Table 3. The mean fold-change expression standard error mean (SEM) gene hcf-1, daf-16, daf-16;hcf-1 <br /> mutant compared wild type shown. p-Value calculated using Student&#226;&#8364;&#8482;s t-test comparison means. <br /> doi:10.1371/journal.pbio.0060233.t004 <br />  <br />  <br />         PLoS Biology | www.plosbiology.org                                                 1878                                    September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                       HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br /> Table 5. Inactivation hcf-1 daf-2 Results Synergistic Regulation Some DAF-16 Target Genes <br />  <br /> Gene Id                   Fold Change                      Fold Change                      Fold Change                                      p-Value Two-Way <br />                           hcf-1(pk924)/wt                     daf-2(e1370)/wt                     daf-2(e1370);hcf-1(pk924)/wt                        ANOVA Test <br />                           6 SEM                               6 SEM                               6 SEM <br />  <br /> sod-3                     2.95 6 0.50                         83.45 6 12.98                        128.2 6 5.59                                       0.0164 <br /> mtl-1                     3.37 6 0.36                         96.96 6 11.07                       244.40 6 43.59                                      0.0121 <br />  <br />  <br /> qRT-PCR performed analyzed described Table 3. The mean fold-change expression standard error mean (SEM) gene hcf-1 daf-2;hcf-1 mutant <br /> compared wild type shown. For analysis synergistic effects, two-way ANOVA test used. <br /> doi:10.1371/journal.pbio.0060233.t005 <br />  <br />  <br /> transcriptional activity likely linked role longevity.                         used immunoprecipitate HCF-1 extracts the <br /> Considering C. elegans HCF-1 normally localized                              daf-16::gfp transgenic worms, DAF-16::GFP co-immuno- <br /> nucleus, mammalian homolog known role                                 precipitated (Figure 6B). This interaction appeared speci&#239;&#172;?c <br /> gene expression regulation, hypothesize HCF-1                                       co-IP experiments using extracts of <br /> modulates C. elegans lifespan regulating transcriptional                             daf-16::gfp worms harbor hcf-1(ok559) deletion <br /> activity DAF-16 subset target genes                                     (daf-16(mu86);hcf-1(ok559);muIs71), able immu- <br /> particularly important stress response longevity                                    noprecipitate HCF-1 co-immunoprecipitate DAF- <br /> assurance.                                                                                  16::GFP (Figure 6B). Furthermore, using identical co-IP <br />                                                                                             conditions, did recover irrelevant Psod-3::GFP (a <br /> HCF-1 Forms Protein Complex DAF-16                                                   GFP reporter driven sod-3 promoter) HCF-1 <br />    We examined HCF-1 affect transcrip-                                   immunoprecipitation, indicating protein&#226;&#8364;&#8220;protein <br /> tional activity DAF-16. In mammalian cells, HCF-1                                     interaction detected DAF-16::GFP HCF-1 not <br /> thought regulate gene expression binding various                                   likely mediated GFP tag. We obtained similar results <br /> transcription chromatin factors [33,49&#226;&#8364;&#8220;51]. In C. elegans,                              using reciprocal co-IP experiments (Figure 6A). Our results <br /> DAF-16 localizes cytoplasm nucleus                                        suggest HCF-1 able form speci&#239;&#172;?c protein complex <br /> normal culture condition, HCF-1 appears predominantly                                   DAF-16 C. elegans. <br /> nuclear cells (Figure 2) [35,37,52], suggesting that <br /> DAF-16 HCF-1 co-localize nucleus (Figure S6). We                                 Loss hcf-1 Leads Enhanced Enrichment DAF-16 on <br /> therefore tested C. elegans HCF-1 physically                                    Its Target Gene Promoters <br /> associate DAF-16. Since did antibody                                 To elucidate molecular mechanism which <br /> could detect endogenous DAF-16 robustly, performed co-                                   HCF-1 regulates DAF-16-mediated transcription, per- <br /> immunoprecipitation (co-IP) experiments using worm strains                                  formed chromatin immunoprecipitation (ChIP) experiments <br /> that lack endogenous DAF-16, carry low-copy number                                   test HCF-1 affect DAF-16 enrichment target <br /> a functional gfp-fused daf-16 transgene (daf-16(mu86);muIs71)                               gene promoters. For ChIP experiments, employed daf- <br /> [37]. We af&#239;&#172;?nity-puri&#239;&#172;?ed HCF-1 antibody                                  16::gfp worms (daf-2(e1370);daf-16(mgDf47);daf-16::gfp) anti- <br />  <br />  <br />  <br />  <br /> Figure 6. HCF-1 Forms Protein Complex DAF-16 C. elegans <br /> Worm extracts mixed stage worms subjected immunoprecipitation. Extracts wild-type (wt), Psod-3::gfp, daf-16::gfp (daf- <br /> 16(mu86);muIs71), daf-16::gfp;hcf-1(-) (daf-16(mu86);hcf-1(ok559);muIs71) worms immunoprecipitated using anti-GFP antibody (A) or <br /> anti-HCF-1 antibody (B). The immunoprecipitated protein complexes subsequently immunoblotted using anti-HCF-1, anti-DAF-16, anti-GFP <br /> antibodies. Psod-3::gfp worms used negative control indicate interaction GFP tag HCF-1. For input, 50 <br /> lg total protein loaded lane. For immunoprecipitation, 2 mg total protein used. <br /> doi:10.1371/journal.pbio.0060233.g006 <br />  <br />         PLoS Biology | www.plosbiology.org                                           1879                                  September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                 HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br /> GFP immunoprecipitation capture DAF-16. As previously                   stress resistance prolonged longevity phenotypes <br /> reported [48], DAF-16 enriched promoter sod-3                observed hcf-1 mutant worms. <br /> when IIS pathway inactivated (Figure 7A). We                     Our results ascribe new longevity stress response <br /> examined DAF-16 target gene mtl-1 DAF-                  function highly conserved transcriptional regulator <br /> 16 enriched promoter (Figure 7A). Under                HCF-1. Mammalian HCF-1 &#239;&#172;?rst identi&#239;&#172;?ed major host <br /> ChIP conditions, did detect HCF-1 enrichment                 cell factor required HSV VP16-induced immediate early <br /> sod-3 mtl-1 promoters, detected great enrichment                 gene transcription [29]. In addition role HSV <br /> HCF-1 promoter e&#239;&#172;&#8218;-1 (Figure 7B). e&#239;&#172;&#8218;-1 C. elegans          infection, HCF-1 essential cell cycle progression. <br /> E2f1 gene; mammalian cells, HCF-1 highly enriched                 Studies revealed mammalian HCF-1 required for <br /> the promoter region E2f1 [53]. Thus, results demon-                 appropriate transition G1 S phase, proper <br /> strate conserved role HCF-1 e&#239;&#172;&#8218;-1/E2f1 promoter              progression M phase cytokinesis [31&#226;&#8364;&#8220;33]. Importantly, <br /> worms, suggest HCF-1 likely present sod-3            mammalian C. elegans HCF-1 share conserved functions. <br /> or mtl-1 promoters. Considering co-IP results showing                  C. elegans HCF-1 able stabilize VP16-induced <br /> that HCF-1 DAF-16 physically associate worms (Figure                complex; moreover, C. elegans hcf-1 mutants produce small <br /> 6), ChIP results suggest HCF-1/DAF-16 complex                 broods exhibit low penetrance embryonic lethality <br /> is probably present promoters DAF-16 target                  (Figure S2) [35], phenotypes consistent role of <br /> genes. However, remains possible HCF-1                        HCF-1 cell proliferation. Furthermore, C. elegans hcf-1 <br /> component large protein complex associates                  mutant embryos low penetrance mitotic and <br /> DAF-16 target gene promoters, cross-linking                     cytokinetic defects [35]. As HSV human speci&#239;&#172;?c virus, it <br /> conditions capture presence HCF-1                   thought VP16 likely mimicked cellular interaction <br /> promoters. For hcf-1-regulated, daf-16-dependent                 HCF-1 co-opted human HCF-1 productive HSV <br /> genes (Table 4; F21F3.3 C32H11.4), did                   lytic infection. Given well-established conserved role <br /> identify putative DAF-16 binding element did detect               HCF-1 cell cycle control, interesting consider <br /> DAF-16 enrichment promoter regions surveyed,                     cell proliferation function HCF-1 linked to <br /> suggesting DAF-16 direct targets. We                role longevity stress response. In adult C. elegans, the <br /> next tested enrichment DAF-16 sod-3                   proliferative tissue germline. Whereas defect in <br /> mtl-1 promoters affected hcf-1 RNAi                      germline stem cell proliferation known cause lifespan <br /> depleted. As control, showed enrichment                   increase, genetic studies showed hcf-1 de&#239;&#172;?ciency can <br /> HCF-1 e&#239;&#172;&#8218;-1 promoter substantially reduced                  continue extend lifespan worms completely lacking <br /> hcf-1 knocked RNAi. Interestingly,                   germline (Figure 1F; Table 1), suggesting longevity <br /> same RNAi conditions, consistently observed enhanced                 function hcf-1 likely linked role cell <br /> enrichment DAF-16 sod-3 mtl-1 promoters                   proliferation. The high degree functional conservation <br /> multiple independent trials (Figure 7A). These results suggest             C. elegans mammalian HCF-1 suggests that <br /> that absence HCF-1, DAF-16 able localize              mammalian HCF-1 likely participates stress response <br /> to promoters target genes.                                      longevity determination. Therefore, HCF-1 well <br />                                                                            represent new universal longevity determinant. <br /> Discussion                                                                    Our model proposes HCF-1 affects lifespan stress <br />                                                                            response forming protein complex DAF-16 and <br />    Our study revealed C. elegans homolog HCF-1               regulating DAF-16 recruitment target gene promoters and <br /> be important longevity determinant transcriptional                  DAF-16-mediated gene transcription. This model consis- <br /> regulator DAF-16. Our data indicate HCF-1                       tent known roles mammalian HCF-1. For VP16- <br /> necessary maintaining normal lifespan stress                       induced immediate early (IE) gene expression, binding of <br /> response C. elegans, loss hcf-1 results mutant worms           HCF-1 VP16 thought help recruit activating Set1/ <br /> with substantially extended longevity heightened resist-               Ash2 histone methyltransferase complex IE-gene pro- <br /> ance speci&#239;&#172;?c stress stimuli. In modulating C. elegans lifespan          moters [29]. For role G1/S transition, mammalian HCF-1 <br /> and stress response, hcf-1 completely depends activity              recruit Set1/Ash2 histone methyl- <br /> of daf-16, likely acts independently IIS pathway. In            transferase activating complex E2F1 Sin3 histone <br /> elucidating mechanism HCF-1 regulates DAF-                    deacetylase repressive complex E2F4 appropriate <br /> 16, showed HCF-1 ubiquitously expressed nuclear               times cell cycle, likely helps reinforce the <br /> protein forms protein complex DAF-16 C.                      activating repressive functions respective E2F family <br /> elegans. Interestingly, absence hcf-1, greater enrich-           members [33]. The role HCF-1 regulating tran- <br /> ment DAF-16 target gene promoters observed                scription factor Miz-1 particularly relevant study. <br /> more robust DAF-16-mediated regulation selective tran-                  HCF-1 shown physically associate Miz-1 and <br /> scriptional targets detected. On basis results,           antagonize Miz-1-mediated transactivation interfering <br /> propose HCF-1 modulates C. elegans lifespan stress                association Miz-1 histone deacetylase <br /> response acting novel negative regulator DAF-16.                P300 [49]. Furthermore, HCF-1, various functional <br /> Normally, HCF-1 associates DAF-16 limits fraction               motifs, shown mediate protein&#226;&#8364;&#8220;protein inter- <br /> of DAF-16 accessing target gene promoters. In                 actions large number polypeptides, including <br /> absence HCF-1, DAF-16 released localize                   transcription factors LZIP/Luman, Zhangfei, HPIP, Sp1, and <br /> target gene promoters confer greater transcriptional                    GABPb, protein phosphatase PP1, cell-death protein <br /> regulation selective target genes (Figure 8). Altered                   PDCD2 [29]. HCF-1 emerging extremely versatile <br /> expression subset target genes likely contributes               scaffolding protein, capable binding different <br />  <br />       PLoS Biology | www.plosbiology.org                            1880                            September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                        HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br />  <br /> Figure 7. Loss hcf-1 Enhances Enrichment DAF-16 Promoters Its Target Genes <br /> ChIP performed using daf-2(e1370);daf-16(mgDf47);daf-16::gfp worms treated control RNAi (L4440) hcf-1 RNAi. Synchronized adult worms of <br /> the different strains incubated 25 8C ;6 h inactivate daf-2 induce robust DAF-16::GFP nuclear localization. Worm extracts were <br /> subjected immunoprecipitation using anti-GFP, anti-HCF-1, anti-Rabbit IgG. The recovered DNA quantitated using qPCR. Regions the <br /> DAF-16 binding element (DBE) sod-3 mtl-1 promoters, putative noncoding region Chromosome IV containing DBE were <br /> monitored. The figure shows representative experiment. Error bars represent SEM duplicated reactions qPCR. Similar results were <br /> obtained independent experiments. <br /> (A) DAF-16 enrichment promoters sod-3 mtl-1 enhanced hcf-1 RNAi. <br /> DAF-16 robustly enriched sod-3 mtl-1 promoters anti-GFP ChIP compared anti-Rb (anti-GFP/anti-Rabbit sod-3 promoter: <br /> ;9-fold; anti-GFP/anti-Rabbit mtl-1 promoter: ;11-fold). The fold enrichment DAF-16 sod-3 mtl-1 consistently greater the <br /> nonspecific Chromosome IV region: sod-3/chr IV: ;2.9-fold; mtl-1/chr IV: ;2.4-fold. As control, anti-GFP ChIP wild-type (wt) worms (not expressing <br /> daf-16::gfp) showed background signal similar anti-Rabbit. <br /> Upon hcf-1 RNAi knockdown, DAF-16 enrichment sod-3 mtl-1 promoters greatly increased. Anti-GFP/anti-Rabbit sod-3 promoter: ;33- <br /> fold (versus ;9-fold L4440 RNAi); anti-GFP/anti-Rabbit mtl-1 promoter: ;33-fold (versus ;11-fold L4440 RNAi). In contrast, nonspecific <br /> Chromosome IV region, anti-GFP/anti-Rabbit: ;4-fold (versus ;3-fold L4440 RNAi). These data indicated absence HCF-1, greater <br /> amount DAF-16 recruited sod-3 mtl-1 promoters, nonspecific binding DAF-16 Chromosome IV region not <br /> substantially changed. <br /> (B) HCF-1 greatly enriched efl-1 promoter, sod-3 mtl-1 promoters. <br /> efl-1 C. elegans homolog E2f1, shown direct target HCF-1 mammalian cells [53]. The region efl-1 promoter <br /> containing conserved E2F1 binding element included positive control anti-HCF-1 ChIP. Whereas HCF-1 greatly enriched <br /> at efl-1 promoter, substantially enriched promoters sod-3 mtl-1, Chromosome IV noncoding region. Fold change anti- <br /> HCF-1/anti-Rabbit efl-1, sod-3, mtl-1, Chromosome IV noncoding region: ;30.8, ;2.1, ;3.6, ;2.3, respectively. As control, hcf-1 knocked <br /> down RNAi, enrichment HCF-1 promoter efl-1 greatly reduced. <br /> doi:10.1371/journal.pbio.0060233.g007 <br />  <br />        PLoS Biology | www.plosbiology.org                                 1881                             September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                    HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br />  <br />  <br /> Figure 8. The Model <br /> We propose wild-type C. elegans, HCF-1 associates fraction DAF-16 nucleus limits recruitment DAF-16 its <br /> target gene promoters. Inactivation hcf-1 allows DAF-16 access target gene promoters enforces DAF-16-mediated regulation of <br /> selective target genes, likely contributes prolonged lifespan enhanced stress resistance phenotypes hcf-1 mutants. <br /> doi:10.1371/journal.pbio.0060233.g008 <br />  <br />  <br /> transcription chromatin factors different con-                    simple hypothesis association HCF-1 and <br /> served motifs, assembling appropriate protein complexes                   DAF-16 regulated basis upstream stimulus. <br /> for proper context-dependent gene expression regulation                       For example, signals induce altered subcellular local- <br /> [33,49&#226;&#8364;&#8220;51].                                                                   ization HCF-1 DAF-16 disrupt HCF-1/DAF-16 <br />    Our qRT-PCR results suggest HCF-1 able                     interaction. Both HCF-1 DAF-16 shown to <br /> regulate DAF-16-mediated transcription select group                  shuttle nucleus cytoplasm speci&#239;&#172;?c <br /> of previously identi&#239;&#172;?ed DAF-16 target genes. Considering                      conditions [14,37,54&#226;&#8364;&#8220;56]. In addition, DAF-16/FOXO and <br /> that DAF-16 target genes surveyed previously                      HCF-1 proteins shown extensively modi&#239;&#172;?ed <br /> determined responsive daf-2/IIS,                       post-translationally [14,37,52,57&#226;&#8364;&#8220;59]. It possible differ- <br /> genetic studies suggest hcf-1 daf-2 act                     ent modi&#239;&#172;?cations HCF-1 and/or DAF-16 substantially <br /> parallel pathways converge daf-16 (Figure 1),              affect association. Lastly, speci&#239;&#172;?city target <br /> surprising gene expression changes caused                    genes conferred level transcriptional <br /> by hcf-1 de&#239;&#172;?ciency distinct caused                      complex assembly. Under scenario, additional co- <br /> reduced daf-2 signaling. It argued HCF-1                    regulators DAF-16 likely come play. Whereas <br /> represent weaker regulator DAF-16 compared DAF-2,                     HCF-1 represents nuclear negative regulator DAF- <br /> and effect DAF-16-regulated genes                    16 known C. elegans, DAF-16 nuclear positive <br /> simply missed analysis likely                   regulators reported, including SIR-2.1 [24], SMK-1 <br /> weaker daf-2 mutant (Table 3). Whereas                     [27], BAR-1 [28]. Future research elucidate the <br /> weaker effect model possible, favor model HCF-                 interplay various DAF-16 co-activators HCF- <br /> 1 represents gene-speci&#239;&#172;?c regulator DAF-16. In                       1 greatly advance understanding mechanistic <br /> analysis, noticed impact hcf-1 de&#239;&#172;?ciency                    details DAF-16 transcriptional activities be <br /> DAF-16 target gene regulation weaker                       appropriately regulated. <br /> than reduced daf-2 signaling. For instance, expression fold                  Our data indicate important function HCF-1 <br /> change F21F3.3 hcf-1(-) comparable daf-                modulate responses speci&#239;&#172;?c environmental <br /> 2(-) (;5-fold versus ;11-fold). On hand,                  stress stimuli. Interestingly, hcf-1(pk924) allele showed <br /> genes, e.g., dod-3 dod-24, expression change daf-                resistance paraquat cadmium treatment, neither <br /> 2(-) great sod-3 mtl-1 (;100-fold), however,              hcf-1 mutant alleles demonstrated altered response to <br /> unlike sod-3 mtl-1, gene expression did                  heat shock. These results suggest general heightened <br /> signi&#239;&#172;?cant change hcf-1(-). Future genome expression                 response wide-range environmental stresses not <br /> pro&#239;&#172;?ling experiments provide global view                    likely account lifespan increase observed hcf- <br /> HCF-1 acts gene-speci&#239;&#172;?c negative regulator DAF-16.                    1 mutant worms. However, important consider <br /> Considering HCF-1 DAF-16/FOXO highly                             stress assays, worms challenged high <br /> conserved species, likely mammalian                    dose acute stress, different low <br /> HCF-1 conserves function FOXO regulation.                         level chronic stress worms experience grow old in <br /> Whereas C. elegans daf-16 gene, mammals                     longitudinal assays. The involvement hcf-1 stress <br /> four FOXO genes. Context speci&#239;&#172;?c regulators FOXOs                      response nicely &#239;&#172;?ts overall theme HCF-1 a <br /> critical ensuring speci&#239;&#172;?city gene expression regulation                 gene-speci&#239;&#172;?c transcriptional regulator DAF-16. It well <br /> and subsequent biological responses [14]. Our &#239;&#172;?ndings C.                   established distinct stress stimuli able induce DAF- <br /> elegans raise intriguing possibility mammalian HCF-1                 16/FOXO regulate different target genes [13,14]. We <br /> represents new regulator FOXO                         propose main role HCF-1 help &#239;&#172;?ne tune <br /> proteins.                                                                     regulation subset DAF-16-regulated genes to <br />    An important question HCF-1 achieve                  modulate survival speci&#239;&#172;?c conditions. Taken together, <br /> speci&#239;&#172;?city in&#239;&#172;&#8218;uencing DAF-16 transcriptional targets. A                    showed HCF-1 essential longevity maintenance <br />  <br />       PLoS Biology | www.plosbiology.org                               1882                            September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                             HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br />  <br /> and functions negative regulator DAF-16 C.                         microscope (Leica DM 5000B) images captured using a <br /> elegans. As HCF-1 DAF-16/FOXO highly conserved                                Hamamatsu ORCA-ER camera OpenLab Software. <br />                                                                                          Stress assays. All stress assays performed previously <br /> from C. elegans mammals, &#239;&#172;?ndings important                                described [41&#226;&#8364;&#8220;43]. <br /> implications FOXO regulation longevity determina-                                For paraquat assay, gravid adult worms strain tested <br /> tion diverse organisms.                                                            allowed lay egg NGM plates seeded OP50 2&#226;&#8364;&#8220;3 h to <br />                                                                                       produce relatively synchronous populations progeny. The progeny <br />                                                                                       allowed develop 25 8C, reached young- <br /> Materials Methods                                                                 adulthood, FUDR added plates &#239;&#172;?nal concentration of <br />                                                                                       50 lg/ml prevent growth progeny. At day 2&#226;&#8364;&#8220;3 adulthood, <br />    C. elegans strains. The strains used paper follow:                 worms washed NGM plates rinsed M9 buffer <br /> wild-type N2, daf-16 (mgDf47), daf-2 (e1370), age-1(hx546), sqt-1(sc13) age-          times remove OP50 bacteria. Approximately 30 adults <br /> 1(mg44)/mnC1 (a gift Catherine A. Wolkow, National Institute                  dispensed 24-well culture plate containing <br /> Aging), glp-1(e2141), hcf-1(ok559) (generated C. elegans Gene                  300 ll 200 mM paraquat (Sigma) M9 buffer. Triplicate wells were <br /> <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>Knockout</span> Consortium), hcf-1(pk924) (a gift Winship Herr,                         used strain, experiment repeated two <br /> University Lausanne, Switzerland), daf-16(mgDf47);xrIs87[daf-                      independent times. Worms paraquat buffer scored every <br /> 16a::gfp::daf-16b, rol-6(su1006)] (DAF-16::GFP) [52], daf-16(mu86);-                  2&#226;&#8364;&#8220;3 h survival. Worms failed respond gentle prodding <br /> muIs71[daf-16a::gfp/bKO, rol-6(su1006)] (DAF-16::GFP) [37], muIs84[-              scored dead. <br /> Psod-3::gfp] [60]. The hcf-1(ok559) allele outcrossed &#239;&#172;?ve times                  For CdCl2 assay, synchronized day 2 adult worms were <br /> the hcf-1(pk924) allele outcrossed times N2 strain              collected described washed K-medium. Worms were <br /> our lab prior phenotype analyses.                                                  24-well culture plate containing 600 ll K- <br />    The following double mutant strains constructed using                         medium 18 mM CdCl2 20 8C. Triplicate plates strain <br /> standard genetic methods: daf-16(mgDf47);hcf-1(ok559), daf-                           scored time point indicated. Worms failed to <br /> 16(mgDf47);hcf-1(pk924), daf-2(e1370);hcf-1(ok559), daf-2(e1370);hcf-                 respond gentle prodding scored dead. <br /> 1(pk924), sqt-1(sc13) age-1(mg44)/mnC1;hcf-1(ok559),glp-1(e2141);hcf-                    For heat shock assay, synchronous populations worms were <br /> 1(pk924), daf-16(mu86);muIs71[daf-16a::gfp/bKO, rol-6(su1006)];hcf-                   grown described above. Day 2 adult worms grown OP50-NGM <br /> 1(ok559), daf-16(mu86);muIs71[daf-16a::gfp/bKO hcf-1(pk924);sur-                  plates shifted 35 8C. Duplicate triplicate plates each <br /> 5::gfp, muIs84[Psod-3::gfp];hcf-1(pk924).                                             strain scored time point. Because scoring done <br />    All strains cultured using standard methods [61]. Unless                      room temperature, worms pulled 35 8C and <br /> otherwise stated, NGM plates seeded E. coli OP50                     scored survival, discarded avoid complication of <br /> food source.                                                                          recovery heat shock time scoring. <br />    Lifespan assays. For RNAi lifespan assays, RNAi bacteria                         Dauer assays. Dauer assays performed previously described <br /> grown Luria broth 50 lg/ml ampicillin 37 8C 10&#226;&#8364;&#8220;16 h,                   [46]. Worms L4 stage allowed lay egg at16 8C night. <br /> seeded NGM plates containing 2 4 mM IPTG, induced                      The resulting progeny allowed develop indicated <br /> room temperature 6 h [62]. Worms allowed lay eggs                   temperature. At ;96 h (25 8C 27 8C) ;120 h (22 8C) egg lay, <br /> overnight RNAi plates 16 8C, progeny grown                      number dauers adult worms plate scored. <br /> RNAi plates 25 8C developed young adult stage. The                 Replica plates scored strain. The dauer assays were <br /> young adult worms transferred RNAi plates seeded                       repeated times. <br /> with 3-fold concentrated RNAi bacteria contained 50 lg/ml                           DAF-16::GFP localization assay. daf-16(mgDf47);xrls87 transgenic <br /> FUDR prevent growth progeny. For lifespan assays using                      strain used DAF-16::GFP localization assay [52]. Worms the <br /> NGM plates seeded OP50 bacteria, worms allowed lay                       L4 larval stage picked RNAi plates allowed lay egg at <br /> eggs over-night 16 8C, progeny grown NGM plates                    16 8C day. Progenies exposed RNAi bacteria 16 <br /> at 25 8C till young adult stage. The young adult worms                      8C 5 d gravid adults. The DAF-16::GFP signal <br /> transferred NGM plates contained 50 lg/ml FUDR                          monitored using &#239;&#172;&#8218;uorescent microscope (Leica DM <br /> seeded 3-fold concentrated E. coli OP50. For lifespan assays                     5000B), images captured using Hamamatsu ORCA-ER <br /> involving daf-2(e1370) daf-2(e1370);hcf-1(pk924) daf-2(e1370);hcf-             camera OpenLab program. <br /> 1(ok559) worms, progeny allowed grow 16 8C shifted                     Immunostaining. Worm &#239;&#172;?xation immunostaining per- <br /> 25 8C L3 larval stage avoid constitutive dauer arrest                formed previously described [65]. In brief, worms immobi- <br /> phenotype associated daf-2(e1370) mutant [63].                               lized compressed polylysine coated slides snap <br />    For lifespan assays, worms aged 25 8C. Worms                      frozen liquid nitrogen. The cuticle removed quickly <br /> scored day day, failed respond                 separating frozen slides. Worms quickly &#239;&#172;?xed pre- <br /> to gentle prodding platinum wire scored dead.                        chilled methanol  20 8C. Incubation primary secondary <br /> Animals bagged, exploded, crawled plate                          antibodies washes TBS buffer room temperature. <br /> considered censored. We de&#239;&#172;?ned day transferred                         Fluorescence signal monitored using &#239;&#172;&#8218;uorescent microscope <br /> the young adult worms day 0 adult lifespan. Statistical analysis                (Leica DM 5000B), images captured using Hamamatsu <br /> was using SPSS software p-values calculated using                   ORCA-ER camera OpenLab program. <br /> the log-rank test. All lifespan experiments repeated                   Immunoprecipitation immunoblotting. Immunoprecipitation <br /> two independent times.                                                                immunoblotting carried described [66]. In brief, for <br />    Transgenic animals. A GFP-fused hcf-1 plasmid (Phcf-1::hcf-1::gfp)                 immunoprecipitation experiments, worm extracts from <br /> was created inserting genomic fragment contains ;500 bp                     mixed staged worms sonication worms lysis buffer and <br /> upstream predicted ATG hcf-1 entire predicted                       subsequent removal debris centrifugation. The appropriate <br /> coding region hcf-1 pPD95_77 plasmid. Transgenic worms                    antibody incubated worm extract 4 8C overnight followed <br /> were created microparticle bombardment previously described                     incubation protein A slurry (Pierce) 4 8C 3;6 hrs. The <br /> [64]. The Phcf-1::hcf-1::gfp plasmid co-bombarded                        protein A beads washed lysis buffer 6 times. The <br /> pJKL702 [unc-119(&#195;&#190;)] plasmid unc-119(ed4) mutant worms                    bound proteins eluted boiling 23 sample buffer, separated <br /> obtain strain unc-119(ed4);rwIs3[Phcf-1::hcf-1::gfp, unc-119]. Multiple           SDS gel, transferred nitrocellulose membrane, and <br /> independent integrated transgenic lines examined.                                followed standard ECL detection. <br />    Embryonic brood size assays. Each single L4 worm allowed                      Antibodies. To generate polyclonal antiserum C. elegans <br /> to lay eggs transferred fresh NGM plate 24 h                  HCF-1, bacterial recombinant S-tagged fusion protein containing <br /> completed egglay. The number eggs laid number                           full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> HCF-1 puri&#239;&#172;?ed using S-Tag Thrombin Puri&#239;&#172;?ca- <br /> hatched worms counted. A total &#239;&#172;?ve worms used                        tion kit (Novagen) used antigen immunize guinea pigs. <br /> each strain.                                                                          The crude anti-HCF-1 antiserum subsequently puri&#239;&#172;?ed using S- <br />    Nile red staining. Nile red staining performed previously                   tagged HCF-1. In brief, crude anti-HCF-1 antiserum was <br /> described [47]. Unseeded NGM plates coated 0.025 lg/ml                      incubated puri&#239;&#172;?ed S-tagged HCF-1 immobilized nitro- <br /> &#239;&#172;?nal concentration Nile red resuspended acetone                      cellulose membrane (BA85 Protran BioScience). Poorly bound <br /> and diluted H2O. The Nile red allowed diffuse NGM                   proteins removed multiple washes TBS PBS buffer, <br /> overnight. The plates seeded OP50 bacteria left                 bound anti-HCF-1 antibody recovered subsequent <br /> room temperature overnight. Gravid adult worms allowed                        elution. <br /> egglay plates overnight, progeny allowed develop                      For immunostaining, af&#239;&#172;?nity puri&#239;&#172;?ed anti-HCF-1 antibody was <br /> young adult stage. Worms monitored using &#239;&#172;&#8218;uorescent                       used primary antibody (1:500) anti-guinea pig conjugated <br />  <br />        PLoS Biology | www.plosbiology.org                                      1883                              September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                  HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br /> with Cy3 (Jackson ImmunoResearch Laboratories, 1:200) used                          ChIP. ChIP performed previously described slight <br /> the secondary antibody. For GFP immunostaining, anti-GFP (goat,                        modi&#239;&#172;?cations (P. Kolasinska-Zwierz, I. Latorre, J. Ahringer, <br /> Rockland) used primary antibody (1:1000) anti-goat                      personal correspondence) [48]. In brief, ground frozen worm powder <br /> conjugated FITC (Jackson ImmunoResearch Laboratories, 1:400)                      crosslinked using 1% formaldehyde PBS buffer subjected <br /> was used secondary antibody.                                                    sonication. Immunoprecipitation performed described <br />    For immunoprecipitation, af&#239;&#172;?nity puri&#239;&#172;?ed anti-HCF-1 anti-                       above. The protein-DNA complexes eluted protein A <br /> GFP (rabbit, Clontech) antibodies used.                                           beads treated RNase A proteinase K. Precipitated DNA <br />    Antibodies immunoblotting include: anti-DAF-16 (cC-20) (goat,                   fragments puri&#239;&#172;?ed subjected qPCR analysis. <br /> Santa Cruz), anti-HCF-1, anti-GFP (rabbit, Clontech), anti-ACTIN                           qPCR primers ChIP. The qPCR Primers ChIP as <br /> (mouse, Chemicon), anti-goat (Rockland), anti-mouse (Santa Cruz),                      followed. Primers sod-3 are: forward primer: 59-TTTTCAAACC- <br /> anti-guinea pig (Jackson ImmunoResearch Laboratories), anti-                       GAAAATTGACC-39; reverse primer: 59-CAAAGACCTCATCAACAG- <br /> rabbit (Rockland).                                                                     C A A - 3 9. P r m e r s f o r m t l - 1 r e : f o r w r d p r m e r : 5 9- <br />    RNA isolation qRT-PCR. Synchronized late L4 staged worms                        ggccaccctcttttatcaca-39; reverse primer: 59-tcaaattgagctgccttcttc-39. <br /> were used RNA isolation. All worms qRT-PCR grown                      Primers e&#239;&#172;&#8218;-1are: forward primer: 59-ttttatcttctcattcaagcgaaa-39; <br /> at 25 8C experiments using daf-2(e1370) mutants,                    reverse primer: 59-gagacaatgggaaaggtgga-39. Primers Chromo- <br /> worms shifted 16 8C 25 8C 8 h prior harvest. Total                 s o m e I V n o n c o d n g r e g o n r e : f o r w r d p r m e r : 5 9- <br /> RNA ;10&#226;&#8364;&#8220;15 ll worm pellet isolated using Tri-reagent                       CTCTTCATTTTGTTCCTGTGTTTTCC-39; reverse primer: 59- <br /> (Molecular Research Center, Inc.) [67]. cDNAs synthesized                    GAAGGCGGCGGTAATTGTTG-39. <br /> random hexamers using SuperScript III First-Strand Kit (Invitrogen). <br /> qRT-PCR reactions performed using iQ SYBR Green Supermix <br /> (BIO-RAD) MyiQ Single Color Real-time PCR Detection                            Supporting Information <br /> System (BIO-RAD). The qRT-PCR conditions were: 95 8C 3 min, <br /> followed 40-cycles 10 s 95 8C 30 s 60 8C. Melting curve              Figure S1. The hcf-1(ok559) Mutant Shows Weak Expression a <br /> analysis performed primer set end ensure                    Truncated HCF-1 Protein hcf-1(pk924) Mutant Shows No <br /> speci&#239;&#172;?city ampli&#239;&#172;?ed product. For qRT-PCR, act-1 used                     Detectable Expression Any HCF-1 Peptide <br /> the internal control, RNA level gene                   Low levels truncated HCF-1 protein hcf-1(ok559) mutant <br /> normalized level act-1 comparison. The qRT-PCR                           detected immunoblotting assays using af&#239;&#172;?nity-puri&#239;&#172;?ed <br /> experiment repeated times using independent RNA/                    polyclonal HCF-1 antibody generated full-length HCF-1 <br /> cDNA preparations. The data pooled analyzed using                             fusion protein. No partial HCF-1 protein detected the <br /> Student&#226;&#8364;&#8482;s t-test.                                                                      immunoblotting assays, suggesting hcf-1(pk924) represent a <br />    qRT-PCR primers. The qRT-PCR primers act-1 are: forward                         null mutant. Total protein mixed populations worms was <br /> primer: 59-CCAGGAATTGCTGATCGTATGCAGAA-39; reverse pri-                                 separated 15% SDS gel followed immunoblotting an <br /> mer: 59-TGGAGAGGGAAGCGAGGATAGA-39 (product length: 133                                 af&#239;&#172;?nity-puri&#239;&#172;?ed HCF-1 antibody. Actin level used loading <br /> bp). Primers sod-3 are: forward primer: 59-CCAACCAGCGCT-                           control. <br /> G A A A T T C A A T G G - 3 9; r e v e r s e p r m e r : 5 9- G G A A C C -          Found doi:10.1371/journal.pbio.0060233.sg001 (103 KB TIF). <br /> GAAGTCGCGCTTAATAGT-39 (product length: 127 bp). Primers <br /> for mtl-1 are: forward primer: 59-atggcttgcaagtgtgactg-39; reverse                     Figure S2. The hcf-1(ok559) Mutant Shows Reduced Brood Size and <br /> primer: 59-cacatttgtctccgcacttg-39 (product length: 56 bp). Primers                Increased Embryonic Lethality <br /> fat-5 are: forward primer: 59-tggtgaagaagcacgatcag-39; reverse primer:                 The brood size embryonic lethality obtained average <br /> 59-aagcagaagattccgaccaa-39 (product length: 125 bp). Primers                       &#239;&#172;?ve animals group 16 8C, 20 8C, 25 8C. Error <br /> M02D8.4 are: forward primer: 59-atttgccaacaaacatgcaa-39; reverse                       bars indicate standard error means. <br /> primer: 59-ggtccacgatggtgttgtct-39 (product length: 87 bp). Primers <br /> for ges-1 are: forward primer: 59-agcaacaaggaagggtcgta-39; reverse                     Found doi:10.1371/journal.pbio.0060233.sg002 (350 KB JPG). <br /> primer: 59-ccgatgatctccgaaatgaa-39 (product length: 120 bp). Primers                   Figure S3. GFP-Fused HCF-1 Is Expressed Nucleus Somatic <br /> for lys-7 are: forward primer: 59-gcgggttattgtgcagtttt-39; reverse                     Germline Cells <br /> primer: 59-tcaattccgagtccagcttt-39 (product length: 114 bp). Primers <br /> for K09C4.5 are: forward primer: 59-tggaattgaaccgactattgc-39; reverse                  (A) The images GFP expression live transgenic worms <br /> primer: 59-gcaaatggcacaagaacaaa-39 (product length: 106 bp). Primers                   carrying low-copy number hcf-1::gfp transgene (rwIs3[Phcf-1::hcf- <br /> for dod-3 are: forward primer: 59-AAAAAGCCATGTTCCCGAAT-39;                             1::gfp, unc-119]) different developmental stages normal <br /> reverse primer: 59-GCTGCGAAAAGCAAGAAAAT-39 (product                                    culture condition. HCF-1::GFP expressed nucleus somatic <br /> length: 137 bp). Primers F21F3.3 are: forward primer: 59-                          germline cells. Arrowhead indicates high levels auto&#239;&#172;&#8218;uor- <br /> CCGATTCGTTCCTTTTGAAG-39; reverse primer: 59-ACAACC-                                    escence observed intestine adult worm (bottom left <br /> GAATGTTCCAATCC-39 (product length: 138 bp). Primers hsp-                           panel). <br /> 16.1 are: forward primer: 59-GCAGAGGCTCTCCATCTGAA-39; re-                              (B) hcf-1::gfp worms &#239;&#172;?xed immunostained using anti-GFP. <br /> verse primer: 59-GCTTGAACTGCGAGACATTG-39 (product length:                              HCF-1::GFP expressed nucleus somatic (upper panel) and <br /> 85 bp). Primers hsp-12.3 are: forward primer: 59-GCCATTCCA-                        germline cells (bottom panel). DAPI staining used indicate the <br /> GAAAGGAGATG-39; verse primer: 5 9-CG TTTGG CAA-                                     nucleus. Photos taken 4003 magni&#239;&#172;?cation. Arrowhead <br /> GAAGTTGTGA-39 (product length: 93 bp). Primers C32H11.4                            indicates auto&#239;&#172;&#8218;uorescence observed intestine. <br /> are: forward primer: 59-TTACTTCCCATCGCCAAAGT-39; reverse                               Found doi:10.1371/journal.pbio.0060233.sg003 (2.26 MB TIF). <br /> primer: 59-CAATTCCGGCGATGTATGAT-39 (product length: 117 <br /> bp). Primers F35E12.5 are: forward primer: 59-TCTCGAAGC-                           Figure S4. Loss hcf-1 Does Not Affect Fat Storage <br /> CAACAAGTTCA-39; reverse primer: 59-TTTCACGGGATCCG-                                     Fat storage hcf-1(pk924) wild-type worms monitored by <br /> TATTTC-39 (product length: 78 bp). Primers T16G12.1 are:                           staining vital dye Nile Red [47]. Nile Red staining pattern of <br /> forward primer: 59-CAATGGGAGCTCACTTCGAT-39; reverse pri-                               hcf-1(pk924) similar wild-type worms. <br /> mer: 59-TCATCGGCAAGAAGAGTCAA-39 (product length: 138 bp). <br /> Primers dod-24 are: forward primer: 59-TGTCCAACACAACCTG-                           Found doi:10.1371/journal.pbio.0060233.sg004 (642 KB TIF). <br /> CATT-39; reverse primer: 59-TGTGTCCCGAGTAACAACCA-39                                    Figure S5. Psod-3::gfp Expression Level Is Elevated hcf-1(pk924) <br /> (product length: 138 bp). Primers F08G5.6 are: forward primer:                     Mutants <br /> 59-tggacaacccagatatgcaa-39; reverse primer: 59-gtatgcgatggaaatggaca- <br /> 39 (product length: 111 bp). Primers dod-22 are: forward primer:                   The GFP levels Psod-3::gfp (muIs84[Psod-3::gfp]) hcf-1(pk924) mutant <br /> 59-ttgttggtcccaagttcaca-39; reverse primer: 59-aagaacttcggctgcttcag-39                 elevated (right panel) compared wild-type background <br /> (product length: 132 bp). Primers daf-16 are: forward primer: 59-                  (left panel). Synchronized day 2 adults shown photos. <br /> ccagacggaaggcttaaact-39; reverse primer: 59-attcgcatgaaacgagaatg&#226;&#8364;&#8220;39                    Arrowhead indicates intestinal auto&#239;&#172;&#8218;uorescence. <br /> (product length: 149 bp). Primers daf-2 are: forward primer: 59-                   Found doi:10.1371/journal.pbio.0060233.sg005 (1.20 MB TIF). <br /> cggtgcgaagagaggatatt-9; reverse primer: 59-tacagaggtcgccgttactg-39 <br /> (product length: 97 bp). Primers age-1 are: forward primer: 59-                    Figure S6. HCF-1 Co-Localizes DAF-16::GFP Nucleus <br /> agtggattcggaaacaatgc-39; reverse primer: 59-ggaatcgatcgacactttca-39                    Transgenic worms over-expressing DAF-16::GFP (daf- <br /> (product length: 135 bp). Primers akt-1 are: forward primer: 59-                   16(mgDf47);xrIs87) immunostained anti-HCF-1. Photos were <br /> tcaccgatgcgatattgtct-39; reverse primer: 59-aactccccaccaatcaacac-39                    taken 1003 (A) 4003 (B) magni&#239;&#172;?cation representative <br /> (product length: 82 bp).                                                               images shown. Under normal culturing condition, DAF-16::GFP <br />  <br />        PLoS Biology | www.plosbiology.org                                       1884                                   September 2008 | Volume 6 | Issue 9 | e233 <br />                                                                                                                      HCF-1 Modulates Lifespan Regulating DAF-16 <br />  <br /> was diffusely localized cytoplasm nucleus previously                         Lee lab critical reading manuscript. We thank members <br /> reported [37,52]. HCF-1 co-localized DAF-16::GFP nucleus                       Lee, Liu, Kemphues, Vatamaniuk labs insightful <br /> of nongermline cells. DAPI staining used indicate nucleus.                      discussions. We especially thank Nirav Amin Yuang Jiang for <br /> Arrowheads indicate nucleus intestinal cells. Arrows indicate                       assistance immunostaining, Bingsi Li help bombard- <br /> the gonad worms DAF-16::GFP expression absent                         ment experiments, Paulina Kolasinska-Zwierz Isabel Latorre and <br /> result transgene silencing [68].                                                        modENCODE groups help ChIP. We grateful Jun <br /> Found doi:10.1371/journal.pbio.0060233.sg006 (1.70 MB TIF).                             Liu, Soyoung Lee, Winship Herr, Gary Ruvkun, Catherine Wolkow, <br />                                                                                            Caenorhabditis Genetics Center (CGC), C. elegans Gene <br /> Table S1. Inactivation hcf-1 Results Lifespan Increase That Is <br />                                                                                            Knockout Consortium providing strains required this <br /> Completely Dependent daf-16, Likely Independent IIS <br /> Pathway                                                                                    work. <br />                                                                                               Author contributions. JL, AE, YD, GR, TI, SSL conceived and <br /> Found doi:10.1371/journal.pbio.0060233.st001 (53 KB DOC). <br />                                                                                            performed experiments analyzed data. JL AE <br /> Table S2. Expression hcf-1::gfp Transgene Partially Rescues                       performed majority lifespan analyses help YD <br /> Lifespan Phenotype hcf-1(pk924)                                                         TI. JL performed immunostaining, DAF-16::GFP localization, <br /> Found doi:10.1371/journal.pbio.0060233.st002 (33 MB DOC).                               qRT-PCR, co-IP, ChIP experiments; AE generated hcf-1::gfp <br />                                                                                            strains, performed stress assays help JL GR, the <br />                                                                                            dauer Nile Red assays help JL; YD generated anti- <br /> Accession numbers                                                                          HCF-1 antibody. JL, AE, SSL wrote paper. <br /> The wormbase (WB, http://www.wormbase.org) gene ID hcf-1                               Funding. This work supported New Scholar Award in <br /> WBGene00001827.                                                                            Aging Ellison Medical Foundation R01 grant <br />                                                                                            AG024425&#226;&#8364;&#8220;01 National Institute Aging (NIA) awarded <br />                                                                                            SSL. <br /> Acknowledgments <br />                                                                                               Competing interests. The authors declared competing <br /> We grateful K. Kemphues, R. Weiss, D. Kim, members                           interests exist. <br />  <br />  <br /> References                                                                                 23. Longo VD, Kennedy BK (2006) Sirtuins aging age-related disease. <br /> 1. Antebi A (2007) Genetics aging Caenorhabditis elegans. PLoS Genet 3::                 Cell 126: 257&#226;&#8364;&#8220;268. <br />     1565&#226;&#8364;&#8220;1571. doi:10.1371/journal.pgen.0030129                                            24. Berdichevsky A, Viswanathan M, Horvitz HR, Guarente L (2006) C. elegans <br /> 2. Bishop NA, Guarente L (2007) Genetic links diet lifespan:                       SIR-2.1 interacts 14-3-3 proteins activate DAF-16 extend life <br />     shared mechanisms yeast humans. Nat Rev Genet 8: 835&#226;&#8364;&#8220;844.                          span. Cell 125: 1165&#226;&#8364;&#8220;1177. <br /> 3. Giannakou ME, Partridge L (2007) Role insulin-like signalling                     25. Wang Y, Oh SW, Deplancke B, Luo J, Walhout AJ, et al. (2006) C. elegans 14- <br />     Drosophila lifespan. Trends Biochem Sci 32: 180&#226;&#8364;&#8220;188.                                       3-3 proteins regulate life span interact SIR-2.1 DAF-16/ <br /> 4. Kaeberlein M, Burtner CR, Kennedy BK (2007) Recent developments                          FOXO. Mech Ageing Dev 127: 741&#226;&#8364;&#8220;747. <br />     yeast aging. PLoS Genet 3:: e84. doi:10.1371/journal.pgen.0030084                      26. Tissenbaum HA, Guarente L (2001) Increased dosage sir-2 gene <br /> 5. Kenyon C (2005) The plasticity aging: insights long-lived mutants.                  extends lifespan Caenorhabditis elegans. Nature 410: 227&#226;&#8364;&#8220;230. <br />     Cell 120: 449&#226;&#8364;&#8220;460.                                                                     27. Wolff S, Ma H, Burch D, Maciel GA, Hunter T, et al. (2006) SMK-1, an <br /> 6. Lambert AJ, Brand MD (2007) Research mitochondria aging, 2006&#226;&#8364;&#8220;                       essential regulator DAF-16-mediated longevity. Cell 124: 1039&#226;&#8364;&#8220;1053. <br />     2007. Aging Cell 6: 417&#226;&#8364;&#8220;420.                                                           28. Essers MA, Vries-Smits LM, Barker N, Polderman PE, Burgering BM, et <br /> 7. Russell SJ, Kahn CR (2007) Endocrine regulation ageing. Nat Rev Mol                      al. (2005) Functional interaction beta-catenin FOXO in <br />     Cell Biol 8: 681&#226;&#8364;&#8220;691.                                                                      oxidative stress signaling. Science 308: 1181&#226;&#8364;&#8220;1184. <br /> 8. Baumeister R, Schaf&#239;&#172;?tzel E, Hertweck M (2006) Endocrine signaling                    29. Wysocka J, Herr W (2003) The herpes simplex virus VP16-induced complex: <br />     Caenorhabditis elegans controls stress response longevity. J Endocrinol                makings regulatory switch. Trends Biochem Sci 28: 294&#226;&#8364;&#8220;304. <br />     190: 191&#226;&#8364;&#8220;202.                                                                          30. Hamilton B, Dong Y, Shindo M, Liu W, Odell I, et al. (2005) A systematic <br /> 9. Nelson DW, Padgett RW (2003) Insulin worms way spotlight.                      RNAi screen longevity genes C. elegans. Genes Dev 19: 1544&#226;&#8364;&#8220;1555. <br />     Genes Dev 17: 813&#226;&#8364;&#8220;818.                                                                 31. Julien E, Herr W (2003) Proteolytic processing necessary separate and <br /> 10. Gottlieb S, Ruvkun G (1994) daf-2, daf-16 daf-23: genetically                          ensure proper cell growth cytokinesis functions HCF-1. Embo J 22: <br />     interacting genes controlling Dauer formation Caenorhabditis elegans.                   2360&#226;&#8364;&#8220;2369. <br />     Genetics 137: 107&#226;&#8364;&#8220;120.                                                                 32. Julien E, Herr W (2004) A switch mitotic histone H4 lysine 20 <br /> 11. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans                      methylation status linked M phase defects loss HCF-1. Mol <br />     mutant lives twice long wild type. Nature 366: 461&#226;&#8364;&#8220;464.                         Cell 14: 713&#226;&#8364;&#8220;725. <br /> 12. Larsen PL, Albert PS, Riddle DL (1995) Genes regulate                        33. Tyagi S, Chabes AL, Wysocka J, Herr W (2007) E2F activation S phase <br />     development longevity Caenorhabditis elegans. Genetics 139:                         promoters association HCF-1 MLL family histone H3K4 <br />     1567&#226;&#8364;&#8220;1583.                                                                                 methyltransferases. Mol Cell 27: 107&#226;&#8364;&#8220;119. <br /> 13. van der Horst A, Burgering BM (2007) Stressing role FoxO proteins               34. Lee S, Herr W (2001) Stabilization transcriptional activity of <br />     lifespan disease. Nat Rev Mol Cell Biol 8: 440&#226;&#8364;&#8220;450.                                 herpes simplex virus VP16-induced complexes evolutionarily conserved <br /> 14. Huang H, Tindall DJ (2007) Dynamic FoxO transcription factors. J Cell Sci                  HCF family members. J Virol 75: 12402&#226;&#8364;&#8220;12411. <br />     120: 2479&#226;&#8364;&#8220;2487.                                                                        35. Lee S, Horn V, Julien E, Liu Y, Wysocka J, et al. (2007) Epigenetic regulation <br /> 15. Halaschek-Wiener J, Khattra JS, McKay S, Pouzyrev A, Stott JM, et al. (2005)               histone H3 serine 10 phosphorylation dtatus HCF-1 proteins C. <br />     Analysis long-lived C. elegans daf-2 mutants using serial analysis gene              elegans mammalian cells. PLoS ONE 2:: e1213. doi:10.1371/journal. <br />     expression. Genome Res 15: 603&#226;&#8364;&#8220;615.                                                        pone.0001213 <br /> 16. McElwee J, Bubb K, Thomas JH (2003) Transcriptional outputs                     36. Kristie TM, Pomerantz JL, Twomey TC, Parent SA, Sharp PA (1995) The <br />     Caenorhabditis elegans forkhead protein DAF-16. Aging Cell 2: 111&#226;&#8364;&#8220;121.                     cellular C1 factor herpes simplex virus enhancer complex family <br /> 17. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003)                   polypeptides. J Biol Chem 270: 4387&#226;&#8364;&#8220;4394. <br />     Genes act downstream DAF-16 in&#239;&#172;&#8218;uence lifespan                        37. Lin K, Hsin H, Libina N, Kenyon C (2001) Regulation Caenorhabditis <br />     Caenorhabditis elegans. Nature 424: 277&#226;&#8364;&#8220;283.                                               elegans longevity protein DAF-16 insulin/IGF-1 germline signaling. <br /> 18. Brunet A, Sweeney LB, Sturgill JF, Chua KF, Greer PL, et al. (2004) Stress-                Nat Genet 28: 139&#226;&#8364;&#8220;145. <br />     dependent regulation FOXO transcription factors SIRT1                        38. Morris JZ, Tissenbaum HA, Ruvkun G (1996) A phosphatidylinositol-3-OH <br />     deacetylase. Science 303: 2011&#226;&#8364;&#8220;2015.                                                       kinase family member regulating longevity diapause Caenorhabditis <br /> 19. Daitoku H, Hatta M, Matsuzaki H, Aratani S, Ohshima T, et al. (2004) Silent                elegans. Nature 382: 536&#226;&#8364;&#8220;539. <br />     information regulator 2 potentiates Foxo1-mediated transcription               39. Arantes-Oliveira N, Apfeld J, Dillin A, Kenyon C (2002) Regulation life- <br />     deacetylase activity. Proc Natl Acad Sci U S A 101: 10042&#226;&#8364;&#8220;10047.                       span germ-line stem cells Caenorhabditis elegans. Science 295: 502&#226;&#8364;&#8220; <br /> 20. Motta MC, Divecha N, Lemieux M, Kamel C, Chen D, et al. (2004) Mammalian                   505. <br />     SIRT1 represses forkhead transcription factors. Cell 116: 551&#226;&#8364;&#8220;563.                     40. Honda Y, Honda S (1999) The daf-2 gene network longevity regulates <br /> 21. <br /> </body></html>