<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Arabidopsis RAP2.2: An Ethylene Response Transcription <br /> Factor That Is Important Hypoxia Survival1[W][OA] <br /> Manuela Hinz, Iain W. Wilson, Jun Yang, Katharina Buerstenbinder, Danny Llewellyn, Elizabeth S. Dennis, <br /> Margret Sauter*, Rudy Dolferus <br /> Physiologie und Entwicklungsbiologie der Pflanzen, Botanisches Institut, Universita&#194;&#168;t Kiel, 24118 Kiel, <br /> Germany (M.H., K.B., M.S.); Commonwealth Scientific Industrial Research Organization Plant <br /> Industry, Canberra, Australian Capital Territory 2601, Australia (I.W.W., J.Y., D.L., E.S.D., R.D.) <br />  <br />  <br /> Arabidopsis (Arabidopsis thaliana) RAP2.2 (At3g14230) APETALA2/ethylene response factor-type transcription factor that <br /> belongs subfamily rice (Oryza sativa) submergence tolerance gene SUB1A. RAP2.2 expressed at <br /> constitutively high levels roots lower levels shoots, induced darkness. Effector studies and <br /> analysis ethylene signal transduction mutants indicate RAP2.2 induced shoots ethylene functions an <br /> ethylene-controlled signal transduction pathway. Overexpression RAP2.2 resulted improved plant <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>survival</span> under <br /> hypoxia (low-oxygen) stress, lines containing T-DNA knockouts gene poorer survival rates wild <br /> type. This indicates RAP2.2 important plant&#226;&#8364;&#8482;s ability resist hypoxia stress. Observation expression <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> of 32 low-oxygen ethylene-associated genes showed RAP2.2 affects low-oxygen response, particularly <br /> the induction genes encoding sugar metabolism fermentation pathway enzymes, ethylene biosynthesis genes. <br /> Our results provide new insight regulation gene expression low-oxygen conditions. Lighting plays an <br /> important regulatory role intertwined hypoxia conditions; stimuli act collaboratively regulate the <br /> hypoxic response. <br />  <br />  <br />  <br />  <br />    Flooding common stresses affect-                    anaerobic fermentation, variety electron <br /> ing plant growth development. Many important                          acceptors pyruvate acetaldehyde replac- <br /> crop plants sensitive flooding waterlogging                     ing oxygen. This metabolic adaptation helps maintain <br /> conditions caused heavy rain irrigation. Flood-                     ATP production NAD+ regeneration (Dennis et al., <br /> ing submergence conditions impose variety                        2000). <br /> challenges plant. Most critical maintaining                     Anaerobic fermentation far efficient pro- <br /> energy supply continued metabolism growth.                        ducing energy aerobic respiration. Carbohydrate <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> cope reduction oxygen supply, plants                       availability, photosynthesis stored <br /> have developed number metabolic morpho-                          reserves, important issue longer the <br /> logical adaptations enable survive tran-                     plant remains submerged. As reaction rising water <br /> sient periods complete partial submergence                          levels, plants extend shoots leaf blades <br /> (Kende et al., 1998; Drew et al., 2000; Bailey-Serres                     reestablish gas exchange (Kende et al., 1998; Voesenek <br /> and Voesenek, 2008). Escape hypoxia involves                         et al., 2004; Pierik et al., 2009). However, extension <br /> shoot elongation, development aerenchyma,                          growth expense increased carbohydrate <br /> adventitious root formation (Drew et al., 2000; Sauter,                   consumption detrimental submergence is <br /> 2000; Voesenek et al., 2006; Perata Voesenek, 2007).                  deep lasts long carbohydrate re- <br /> Nonetheless, oxygen ultimately limiting,                       serves depleted. Some plants, therefore, have <br /> necessitating switch aerobic respiration                        adapted alternative strategy aims conserve <br />                                                                           energy carbohydrate consumption limiting <br />    1 <br />      This work supported CottTech, research alliance           growth low-oxygen conditions. Plant survival <br /> between Commonwealth Scientific Industrial Research               flooding conditions controlled intricate <br /> Organization, Cotton Seed Distributors, Australian            balance escape endurance (Fukao and <br /> Cotton Research Development Corporation.                              Bailey-Serres, 2004). <br />    * Corresponding author; e-mail msauter@bot.uni-kiel.de.                   The molecular basis morphological, anatom- <br />    The author responsible distribution materials integral   ical, metabolic adaptations flooding the <br /> findings presented article accordance policy <br />                                                                           signaling events leading responses remain to <br /> described Instructions Authors (www.plantphysiol.org) is: <br /> Margret Sauter (msauter@bot.uni-kiel.de).                                 elucidated. Recent research focused ethylene <br />    [W] <br />        The online version article contains Web-only data.         signal regulation early response to <br />    [OA] <br />         Open Access articles viewed online sub-          flooding. Expression levels ethylene biosynthetic <br /> scription.                                                                genes up-regulated hypoxic conditions (Van <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.110.155077                     der Straeten et al., 1997; Peng et al., 2005). In sub- <br /> Plant Physiology&#195;&#8217;, June 2010, Vol. 153, pp. 757&#226;&#8364;&#8220;772, www.plantphysiol.org &#195;&#8220; 2010 American Society Plant Biologists             757 <br />  Hinz et al. <br />  <br />  <br /> merged plants, ethylene levels increase rapidly        transcription factors. The AP2/ERF superfamily pro- <br /> physical entrapment volatile hormone (Me&#194;&#180;traux        teins characterized having AP2 <br /> and Kende, 1983; Peng et al., 2001; Voesenek et al.,          domains: ERF EREBP (for ethylene-responsive <br /> 2004, 2006). The programmed cell death response          element-binding proteins) family AP2 domain, <br /> results lysigenous aerenchyma formation            RAV family AP2 domain B3 domain, <br /> controlled ethylene (Drew et al., 2000), epi-        AP2 family AP2 domains (Okamuro <br /> dermal cell death adventitious root primordia           et al., 1997; Riechmann Meyerowitz, 1998; Nakano <br /> in rice (Oryza sativa; Mergemann Sauter, 2000;            et al., 2006). Some ERF factors bind activators or <br /> Steffens Sauter, 2009) adventitious root              repressors GCC box (AGCCGCC) elements the <br /> growth Rumex palustris rice (Visser et al., 1996;      promoters ethylene-responsive genes (Ohme-Takagi <br /> Steffens Sauter, 2005; Steffens et al., 2006). The        Shinshi, 1995), mediate response <br /> internode elongation response, serves             cytokinin (Rashotte et al., 2006). ERF factors also <br /> important adaptation response deepwater rice,           play role variety developmental processes <br /> also controlled ethylene (Hattori et al., 2009). In        flower seed development (Riechmann and <br /> Arabidopsis (Arabidopsis thaliana), ethylene induces          Meyerowitz, 1998) abiotic biotic stress re- <br /> alcohol dehydrogenase gene expression (ADH1; Peng             sponses (Bu&#194;&#168;ttner Singh, 1997; Stockinger et al., <br /> et al., 2001, 2005). Ethanolic fermentation           1997; Fujimoto et al., 2000; Thomashow, 2001; Sakuma <br /> ADH1 activity contributes substantially metabolic          et al., 2002). <br /> adaptation low-oxygen stress; adh1 null mutant             We studied role ERF transcription factor, <br /> that showed lower survival rates exposed low          RAP2.2 (At3g14230), Arabidopsis response to <br /> oxygen (Ellis et al., 1999) overexpression pyru-       hypoxia. RAP2.2 shows structural phylogenetic <br /> vate decarboxylase (PDC1 PDC2) Arabidopsis             relationships rice SUB1A gene (Nakano et al., <br /> results improved survival low-oxygen con-            2006). Our results ethylene RAP2.2, <br /> ditions (Ismond et al., 2003).                                oxygen-dependent signal transduction, <br />    A major quantitative trait locus responsible           play important role response hypoxia. <br /> submergence tolerance, Submergence1 (SUB1), was <br /> identified lowland indica rice (Fukao et al., 2006; <br /> Xu et al., 2006). The locus consists cluster three <br /> ethylene response factor (ERF) genes, SUB1A, SUB1B,           RESULTS <br /> and SUB1C. The SUB1A gene present indica           Relationship Arabidopsis RAP2.2 Rice <br /> and japonica cultivars. Although SUB1A              SUB1 Proteins <br /> present several, all, indica cultivars, the <br /> SUB1A-1 allele submergence-tolerant indica              BLASTP searches revealed Arabidopsis RAP2.2 <br /> cv FR13A able confer flooding tolerance            related rice SUB1A protein. RAP2.2, however, <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span>. Overexpression SUB1A-1 flooding-           longer SUB1 shares 37% amino acid <br /> sensitive japonica cultivar resulted increased ADH1        identity aligned regions relative SUB1A-1. <br /> expression flooding tolerance (Fukao et al., 2006;        The protein alignment RAP2.2 rice SUB1 <br /> Xu et al., 2006). The SUB1A-1 allele appears confer        proteins (Fig. 1A) shows short stretches high <br /> submergence tolerance complex regulatory path-          homology, particularly AP2 domain other <br /> way reduces elongation growth carbohydrate           domains, unknown function located between <br /> consumption (Fukao et al., 2006; Xu et al., 2006; Perata      amino acids 49 70 (rich Asp) the <br /> and Voesenek, 2007, Jung et al., 2010). A role         N-terminal end (MCGG motif; Fig. 1A). The rice <br /> SUB1A-2 allele submergence tolerance remains            SK1 SK2 proteins responsible elonga- <br /> be analyzed. However, recently, shown             tion growth different AP2 domain there- <br /> that opposite response deepwater rice          fore included alignment (Hattori et al., <br /> controlled ethylene ERF factors. The capacity       2009). An extensive classification Arabidopsis <br /> deepwater rice elongate stem internodes extend         rice AP2/ERF transcription factors based the <br /> the hollow stem water surface requires             AP2 domain structure grouped RAP2.2 rice <br /> SNORKEL1 (SK1) SK2 ERF transcription factors              SUB1B SUB1C proteins subfamily VII <br /> (Hattori et al., 2009). A detailed analysis cell type-     (Nakano et al., 2006); rice SK1 SK2 different <br /> specific changes translated mRNA response            AP2 domain structure belong subfamily XI. The <br /> short-term hypoxic treatment Arabidopsis seed-             group VII subfamily contains 15 members japonica <br /> lings revealed ubiquitous increase translated            rice Arabidopsis. A phylogenetic <br /> ERF71 ERF73 mRNA (Mustroph et al., 2009).                 analysis entire protein sequences the <br /> Increased expression ERFs low-oxygen          Arabidopsis rice group VII members closely <br /> conditions recently confirmed, induction           related AP2/ERF proteins plants car- <br /> these factors ethylene analyzed (Licausi           ried (Fig. 1B; sequence alignment, see <br /> et al., 2010).                                                Supplemental Fig. S1). RAP2.2 most <br />    ERF-type transcription factors belong plant-        closely related Arabidopsis RAP2.12 (At1g53910) <br /> specific multigene family APETALA2 (AP2)/ERF               dicot ERF factors, tomato (Solanum <br /> 758                                                                                         Plant Physiol. Vol. 153, 2010 <br />                                                                                                             RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />  <br />  <br />          Figure 1. Phylogenetic tree group VII AP2/ERF transcription factors Arabidopsis rice. A, Protein sequence alignment <br />          RAP2-2 rice Sub1A, Sub1B, Sub1C proteins. Alignment shows homology especially AP2 domain (amino <br />          acids 130&#226;&#8364;&#8220;186). Another highly conserved region unknown function present amino acids 49 70 (2). The <br />  <br /> Plant Physiol. Vol. 153, 2010                                                                                                              759 <br />  Hinz et al. <br />  <br />  <br /> Figure 2. Expression analysis of <br /> RAP2.2 ADH1 roots shoots <br /> during low-oxygen dark treat- <br /> ments Arabidopsis plants. A, RNA <br /> blots showing expression RAP2.2 <br /> and ADH1 roots shoots 17-d- <br /> old plants treated 5% oxygen <br /> (hypoxia treatment) 21% oxygen <br /> (aerobic control). B, RNA-blot analysis <br /> of RAP2.2 ADH1 expression in <br /> hairy root cultures shoots plants <br /> treated hypoxia kept aerobic <br /> conditions. C, RNA-blot analysis of <br /> RAP2.2 ADH1 expression the <br /> shoots treated hypoxia (for 2 or <br /> 8 h) kept aerobic conditions (0 h) <br /> under normal lighting conditions and <br /> in dark. Twenty micrograms total <br /> RNA used lane, ethidium <br /> bromide staining rRNA used as <br /> a loading control. <br />  <br />  <br />  <br />  <br /> lycopersicum) JERF3 pepper (Capsicum annuum)                               RAP2.2 Is Induced Dark-Grown Shoots <br /> CaPF1, play roles salt freezing tolerance, <br />                                                                                   RAP2.2 originally identified gene was <br /> respectively (Wang et al., 2004; Yi et al., 2004; Fig. 1B). <br />                                                                                induced 2- 3-fold microarrays 1 h low- <br /> No rice group VII proteins clustered clade <br />                                                                                oxygen treatment (0.5% oxygen) Arabidopsis hairy <br /> containing RAP2.2. The rice proteins cluster <br />                                                                                root cultures (Klok et al., 2002). Induction RAP2.2 by <br /> with dicot proteins Os02g54160, Os06g09390, <br /> and Os09g26420 (Fig. 1B). Os02g54160 (OSEREBP1)                                hypoxia observed RNA blots Arabidop- <br /> plays role pathogenesis-related gene expression                           sis root mRNA experimental system; expression <br /> (Cheong et al., 2003). This alignment amino acid                        RAP2.2 roots high dark both <br /> sequences shows rice SUB1A, -B, -C                                aerobic hypoxic conditions (Fig. 2A). In aerobic <br /> proteins cluster separately rice group                          shoots, RAP2.2 expression lower in <br /> VII ERF factors closest Arabidopsis                             aerobic roots; expression strongly induced when <br /> relatives At1g72360 (ERF73) At2g47520                                  plants transferred hypoxic conditions the <br /> (ERF71; Fig. 1B) RAP2.2. But, case                           dark, levels similar roots (Fig. 2, A and <br /> for RAP2.2, ERF73, ERF71, homology SUB1A                              B). ADH1 expression strongly induced low- <br /> is mainly confined AP2 domain (overall amino                            oxygen conditions roots shoots, overall <br /> acid identity 26% 33% ERF73 ERF71,                              expression levels lower shoots <br /> respectively). These results indicate Arabidopsis                         compared roots (Fig. 2, A B). RAP2.2 <br /> does close relative rice SUB1 proteins                       induction low-oxygen stress observed in <br /> and rice SUB1 genes evolved                                  roots dark-grown hairy root cultures, overall <br /> independently separation monocots                             expression levels high untreated <br /> dicots (Fukao et al., 2009).                                                   root cultures (Fig. 2B). When low-oxygen stress was <br />  <br />           Figure 1. (Continued.) <br />           N-terminal MCGG motif (1) strictly conserved RAP2.2, Sub1A, Sub1B, Sub1C barley <br />           (Hordeum vulgare) HvRAF (Jung et al., 2007). A potential nuclear localization signal (NLS) present positions 113 and <br />           128. B, A phylogenetic tree group VII AP2/ERF transcription factors Arabidopsis rice constructed using entire <br />           protein sequence group VII (Nakano et al., 2006) ERF transcription factors japonica rice (prefix Os) Arabidopsis <br />           (prefix At). Bootstrap figures given branch points tree. For SUB1 proteins indica rice, equivalent <br />           japonica locus given SUB1B -C; GenBank entry given SUB1A locus. Subgroup VII corresponds B-2 <br />           subgroup according analysis Sakuma et al. (2002). Known gene names mentioned parentheses. The phylogenetic <br />           tree includes group VII AP2/ERF transcription factors bell pepper (CaPF1) tomato (JERF3; Nakano et al., 2006). <br />           Barley HvRAF included close relative group VII ERF subfamily. The bar indicates number amino acid <br />           substitutions site. <br />  <br /> 760                                                                                                                   Plant Physiol. Vol. 153, 2010 <br />                                                                                                            RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />                                                                             carried light, RAP2.2 induction 5% <br />                                                                             oxygen observed, expression levels of <br />                                                                             RAP2.2 increased shoots dark, under <br />                                                                             aerobic conditions; induction transient and <br />                                                                             peaks 2 h darkness (Fig. 2C). In contrast, <br />                                                                             expression ADH1, well-characterized low-oxygen- <br />                                                                             responsive gene, strongly induced hypoxia in <br />                                                                             roots, hairy root cultures, shoots (Fig. 2, A B) <br />                                                                             responded specifically low oxygen to <br />                                                                             darkness (Fig. 2C). These results indicate RAP2.2 <br />                                                                             responds darkness low-oxygen <br />                                                                             conditions shoots. <br />  <br />                                                                             RAP2.2 Expression Is Regulated Ethylene <br />  <br />                                                                                Treatments carried dark shoot <br />                                                                             tissues normoxic (21% oxygen) hypoxic (5% <br />                                                                             oxygen) conditions determine RAP2.2 is <br />                                                                             regulated ethylene. The ethylene-releasing chemi- <br />                                                                             cal ethephon, natural ethylene precursor 1-amino- <br />                                                                             cyclopropane-1-carboxylic acid (ACC), ethylene itself, <br />                                                                             inhibitor ethylene biosynthesis CoCl2 were <br />                                                                             used manipulate ethylene levels plant. In the <br />                                                                             presence 50 mM ACC 50 mM ethephon, RAP2.2 <br />                                                                             expression enhanced normoxic and <br />                                                                             hypoxic conditions dark. The 50 mM CoCl2 <br />                                                                             treatment appeared reduce expression under <br />                                                                             normoxic conditions (Fig. 3A). We applied the <br />                                                                             inhibitors ethylene signaling silver thiosulfate (STS) <br />                                                                             1-methylcyclopropene (1-MCP). In presence of <br />                                                                             50 mM STS, RAP2.2 transcript levels reduced <br />                                                                             normoxic hypoxic conditions (Fig. <br />                                                                             3A). Treatment Arabidopsis plantlets ethylene <br />                                                                             (0.1, 1, 10 mL L21) induced RAP2.2 expression; this <br />                                                                             induction blocked simultaneous treatment <br />                                                                             1-MCP (ethylene 1 mL L21 1-MCP 10 mL <br />                                                                             L21; Fig. 3B). Treatments ethylene precursors <br />                                                                             ethephon ACC increased ADH1 transcript abun- <br />                                                                             dance shoots, inhibitors ethylene syn- <br />                                                                             thesis signaling reduced ADH1 expression (Fig. 3A). <br />                                                                                To study role ethylene signaling in <br />                                                                             RAP2.2 regulation, constitutive triple response ctr1 <br />                                                                             mutant ethylene-insensitive mutants, ein2, <br />                                                                             ein3, ein4, employed (Olmedo et al., 2006). <br />                                                                             Expression RAP2.2 shoots increased ctr1 <br />                                                                             normoxic-light hypoxic-dark condi- <br />                                                                             tions (Fig. 3C). In contrast, ein2, ein2-1, ein3-1, and <br />                                                                             ein4, induction RAP2.2 expression hypoxic- <br /> Figure 3. Effects ethylene, ethylene-producing agents, inhibitors     dark conditions shoots reduced (Fig. 3C). <br /> ethylene synthesis, ethylene signaling mutants expression     RAP2.2 induced weakly ein4 2 h hypoxic <br /> RAP2.2 ADH1. A, RNA-blot analysis 17-d-old plants          treatment; fact ein4 is <br /> preincubated 2 h 50 mM ethephon, 50 mM ACC, 50 mM STS (an          semidominant, ethylene receptors partially <br /> ethylene perception inhibitor), 50 mM CoCl2 (an ethylene synthesis <br /> inhibitor) subjected hypoxic conditions dark. As a <br /> control, nontreated plants left light. B, RT-PCR data showing <br /> the effects ethylene treatment (0.1, 1, 10 mL L21)           dark. D, Ethylene production 17-d-old plants kept 21% oxygen <br /> combined treatment ethylene (1 mL L21) 1-MCP (10 mL L21)          (0 h) 5% oxygen 2 8 h dark. Ethylene accumulation <br /> RAP2.2 gene expression. Amplification Actin1 cDNA used          determined gas chromatography hour treat- <br /> control RNA input. C, RNA-blot analysis RAP2.2 ADH1           ment. Average values 6 SD obtained independent <br /> shoots wild type (wt) ethylene signaling mutants ctr1,       biological experiments independent setups including 30 <br /> ein2-1, ein2, ein4 normoxic hypoxic conditions          plants each. fw, Fresh weight. <br />  <br /> Plant Physiol. Vol. 153, 2010                                                                                                                761 <br />  Hinz et al. <br />  <br />  <br /> complement ein4 (Hua Meyerowitz, 1998). Similar       root hypocotyl (Fig. 4, E F) hypocotyl <br /> to RAP2.2, ADH1 mRNA levels shoots en-            cotyledons (Fig. 4, E G). In light-grown <br /> hanced ctr1 hypoxic-dark conditions     seedling roots, GUS staining present stele <br /> (Fig. 3C), suggesting ADH1 dual control     root tip (Fig. 4H). In 4-d-old dark-grown <br /> by low-oxygen signaling ethylene signaling.           seedlings, GUS activity high roots the <br />    Direct measurement ethylene production           root-hypocotyl junction (Fig. 4, I&#226;&#8364;&#8220;K), contrast to <br /> Arabidopsis plantlets showed ethylene levels         light-grown seedlings, expression root also <br /> increased strongly 2 h oxygen deprivation     observed root tip (Fig. 4L). In hypocotyl, <br /> the dark returned nondetectable levels 8 h   overall expression appeared lower compared <br /> (Fig. 3D). The time maximal ethylene synthesis         light-grown seedlings age was <br /> coincided time maximal expression          vascular (Fig. 4I). However, high levels of <br /> RAP2.2 (Fig. 3B).                                         expression seen subapical hook region <br />    In conclusion, ethylene ethylene signaling    (Fig. 4, I M). The spatial distribution RAP2.2 <br /> affect RAP2.2 expression normoxic-dark          promoter-driven GUS expression plantlets <br /> hypoxic-dark conditions, ADH1 expression af-       showed quantitative differences expression no <br /> fected ethylene signaling hypoxic con-       qualitative differences, did observe qual- <br /> ditions present time.                     itative differences expression pattern following <br />                                                           darkness low-oxygen treatment (data shown). <br /> Spatial Expression Pattern RAP2.2 Promoter <br />    Seventeen-day-old Arabidopsis plants transformed       RAP2.2 Overexpression Increases ADH1 PDC1 <br /> with RAP2.2 promoter-driven GUS reporter gene           Expression Hypoxic Conditions <br /> showed GUS activity shoots roots. Ex- <br /> pression weak shoots high roots (Fig.      RAP2.2 control 35S promoter was <br /> 4, A B). When plants transferred dark     transformed ecotype Columbia (Col-0), the <br /> for 2 h, GUS staining increased shoot (Fig. 4C).   ox1, ox2, ox3 lines recovered. Strong RAP2.2 <br /> In 4-d-old light-grown seedlings, GUS activity        overexpression shoots light-grown aerobic plants <br /> high stele root hypocotyl    evident ox1, ox2, ox3, mRNA levels <br /> weaker vasculature cotyledon (Fig. 4D).     increased lines following hypoxic <br /> Expression highest junction        treatment dark (Fig. 5). <br />  <br /> Figure 4. Localization RAP2.2 <br /> expression using promoter:GUS trans- <br /> genic plants. A C, Seventeen-day- <br /> old plants subjected normal <br /> lighting conditions (A B) dark <br /> conditions 2 h (C). GUS staining for <br /> both light-grown dark-treated <br /> plantlets 30 min. D H, Light- <br /> grown 4-d-old Arabidopsis seedlings. <br /> D, Complete seedling. E, Enlargement <br /> of root-hypocotyl junction. F, Fur- <br /> ther enlargement root-hypocotyl <br /> junction. G, The vascular tissue lead- <br /> ing hypocotyl cotyle- <br /> dons. H, Root tip. I M, Four-day-old <br /> etiolated Arabidopsis seedlings. I, In- <br /> tact seedling. J, Root-hypocotyl junc- <br /> tion. K, Staining vascular tissue <br /> at root-hypocotyl junction. L, Root <br /> tip. M, Staining apical hook <br /> region. <br />  <br />  <br />  <br />  <br /> 762                                                                                     Plant Physiol. Vol. 153, 2010 <br />                                                                                                    RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />    Overexpression RAP2.2 resulted overexpres-                      ness hypoxia treatments required as <br /> sion ADH1 and, lesser degree, PDC1 shoots                  signals. <br /> following hypoxic-dark treatment under <br /> normoxic-dark conditions (Fig. 5). This increase was <br />                                                                          T-DNA Insertions RAP2.2 Reduce ADH1 PDC1 <br /> reflected increased ADH PDC enzyme activities <br />                                                                          Hypoxic Induction Shoots <br /> in shoots (Fig. 6). In shoots Col-0 wild type, ADH <br /> activity increased 8 24 h hypoxic-dark                         Two independent T-DNA insertion lines RAP2.2, <br /> treatment. The time course experiment                       rap2.2-1 (CS871911) rap2.2-2 (CS876942), segre- <br /> RAP2.2-overexpressing lines ox1 ox2 showed sig-                      gated single locus insertions. The rap2.2-1 and <br /> nificantly higher ADH PDC activities                        rap2.2-2 T-DNA insertions located and <br /> wild type 24 h hypoxia treatment (Fig. 6; P ,                   second exons RAP2.2, respectively, expected <br /> 0.01, n = 6). In ox2 line ox1 line, ADH                  eliminate protein function. Expression levels were <br /> and PDC activities increased signifi-                       reduced strongly rap2.2-2 insertion line <br /> cantly compared wild type 8 h                          compared rap2.2-1, lines showed <br /> hypoxia treatment (Fig. 6). Similar results ob-                     induction mRNA levels following hypoxia the <br /> tained RAP2.2-overexpressing line, ox3                       shoots (Fig. 7). T-DNA insertions RAP2.2 resulted in <br /> (Supplemental Fig. S2). The increased ADH activity                       slightly lower shoot ADH1 PDC1 mRNA induc- <br /> of RAP2.2-overexpressing lines significant                       tion levels hypoxia dark. The fact that <br /> under hypoxic-dark conditions mirrors obser-                     RAP2.2 knockdown lines partially reduced ADH1 <br /> vations RNA blots (Fig. 5). These results                       PDC1 expression indication other <br /> clearly indicate overexpression RAP2.2 leads                  redundant ERF factors compensate lower <br /> increased levels gene expression enzyme activ-                    RAP2.2 expression levels. <br /> ity levels ethanol fermentation pathway <br /> enzymes, ADH PDC. This overexpression                             RAP2.2 Is Important Survival Hypoxia <br /> mainly evident hypoxic environment the <br /> dark. Similarly, expression ERF71 (HRE2)                             To determine RAP2.2 important plant <br /> ERF73 (HRE1) roots Arabidopsis seedlings                        survival low oxygen, highest over- <br /> induced treatment 1% oxygen dark.                         expressing lines (ox2 ox3) insertion <br /> Overexpression ERF73 improved survival Arabi-                      <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> lines (rap2.2-1 rap2.2-2) subjected to <br /> dopsis seedlings exposed anaerobic treatment                   low-oxygen survival assays described Christianson <br /> dark resulted up-regulation ADH1 expres-                       et al. (2009). Different survival assay conditions to <br /> sion (Licausi et al., 2010). Since anaerobic treatments                  established overexpressing knockout lines <br /> were carried dark, possible dark-                  RAP2.2 different behavior terms <br />                                                                          low-oxygen survival relative wild type. The <br />                                                                          insertion line treatment consisted 24 h 5% oxygen <br />                                                                          pretreatment followed 40 h 0.1% oxygen treat- <br />                                                                          ment, dark. The overexpressing lines were <br />                                                                          subjected 24 h 5% oxygen pretreatment followed <br />                                                                          48 h 0.1% oxygen treatment dark. Control <br />                                                                          aerobic plants kept dark. Each assay <br />                                                                          consisted replicates 12 plants, using Col-0 as <br />                                                                          wild-type control, results averaged <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                          independent experiments. Plant survival <br />                                                                          light scored 1 week stress treat- <br />                                                                          ment. The results rap2.2-1 rap2.2-2 had <br />                                                                          significantly lower survival Col-0 (P , 0.01; Figs. <br />                                                                          8A 9A), ox2 ox3 overexpressing <br />                                                                          lines possessed significantly higher survival levels <br />                                                                          wild type (P , 0.01; Figs. 8B 9B). These <br />                                                                          results indicate correlation between <br />                                                                          RAP2.2 expression survival low-oxygen <br />                                                                          conditions imposed assay. Root shoot fresh <br />                                                                          weights determined rap2.2-2 ox3 plants <br />                                                                          recovered survival assays shown Figure 9, <br /> Figure 5. RAP2.2, ADH1, PDC1 expression wild type (wt) <br />                                                                          A B, expressed relative fresh weights of <br /> and RAP2.2-overexpressing lines. Seventeen-day-old wild-type plants      respective untreated controls (Fig. 9C). Results <br /> and 35S::RAP2.2 ox transgenic lines ox1, ox2, ox3 treated   indicate knockout plants general lower <br /> with 21% 5% oxygen dark, relative transcript levels     fresh weight wild type, plants the <br /> shoots RAP2.2, ADH1, PDC1 determined RNA-blot             ox3 line tend higher shoot root fresh <br /> analysis.                                                                weights. <br /> Plant Physiol. Vol. 153, 2010                                                                                                  763 <br />  Hinz et al. <br />  <br />  <br /> Figure 6. ADH PDC enzyme activity measure- <br /> ments shoots wild-type Col-0 RAP2.2- <br /> overexpressing lines ox1 ox2. A, ADH enzyme <br /> activity measurements following 0 (t0), 8, 24 h of <br /> hypoxia (5% oxygen) treatment, showing signif- <br /> icantly increased levels ADH activity the <br /> overexpressing lines. B, PDC enzyme activity mea- <br /> surements following 0 (t0), 8, 24 h hypoxia (5% <br /> oxygen) treatment, showing overexpression PDC <br /> activity overexpressing lines. The data <br /> represent averages 6 SE biological repeats <br /> with measurements sample. <br />  <br />  <br />  <br />  <br /> RAP2.2 Expression Affects Only Subset               maximum 2 h induction shoots, a <br /> Low-Oxygen-Responsive Pathways                           treatment <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> 5 h chosen darkness <br />                                                          hypoxia treatments analyze effect chang- <br />    To understand role RAP2.2 hypoxia        ing RAP2.2 expression downstream genes. Quan- <br /> conditions, investigated genes (apart       titative reverse transcription (QRT)-PCR expression <br /> ADH1 PDC1) affected RAP2.2 darkness,       profiling carried individual repeat <br /> hypoxia (5% oxygen), combination dark-        samples; expression data provided Supple- <br /> ness hypoxia. We studied effect   mental Figure S3, heat map summarizing the <br /> treatments wild-type plants RAP2.2-     expression data provided Supplemental Figure S4. <br /> overexpressing knockout lines (ox3 rap2.2-2)        Hierarchical clustering expression profiles of <br /> and investigated effect altering RAP2.2 expres-   32 genes treatments/genotypes revealed that <br /> sion levels expression 32 genes      combined hypoxia-darkness treatment responses <br /> previously shown affected hypoxia          closer darkness-only treatment under <br /> treatment plants (Supplemental Table S1). The genes   normoxic conditions hypoxia treat- <br /> were chosen low-oxygen stress microarray        ment light conditions (Fig. 10). Of 21 genes <br /> experiments carried dark (Klok      induced 2-fold hypoxia in <br /> et al., 2002; Loreti et al., 2005) carried   dark wild-type plants, induced <br /> under light conditions (Liu et al., 2005; Supplemental   hypoxia light, 11 induced by <br /> Table S1). The choice influenced function         darkness (Fig. 10; Supplemental Figs. S3 S4). <br /> under low-oxygen concentrations (e.g. sugar metabo-      Those genes induced hypoxia light <br /> lism, fermentation pathway genes) potential role    stronger induction hypoxia darkness <br /> in senescence ethylene-mediated responses. Also       (Supplemental Figs. S3 S4). Only genes <br /> included ethylene biosynthesis genes      appeared weak induction hypoxia in <br /> ERF genes, particularly members group VII ERF     light did increase expression <br /> factors (Fig. 1B) ERF factors dif-   hypoxia darkness (RopGAP4, RAP2.12, and <br /> ferentially expressed microarray experiments.     ACO; Supplemental Fig. S3). Many genes thought be <br /> Since RAP2.2 expression levels shown reach     &#226;&#8364;&#339;low-oxygen-induced&#226;&#8364;? genes fact induced by <br /> 764                                                                                    Plant Physiol. Vol. 153, 2010 <br />                                                                                                     RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />                                                                           expression altered changing RAP2.2 <br />                                                                           expression levels. This group contains RAP2.12, close <br />                                                                           relative RAP2.2. <br />                                                                             The expression data demonstrate RAP2.2 affects <br />                                                                           response low-oxygen stress. Specific <br />                                                                           genes important metabolic adaptations as <br />                                                                           fermentation sugar metabolism, some <br />                                                                           ethylene biosynthetic genes, affected RAP2.2. <br />  <br />  <br />                                                                           DISCUSSION <br />  <br />                                                                              Ethylene plays major role hypoxia-induced <br />                                                                           gene expression plants: ethylene biosynthesis genes <br />                                                                           up-regulated low-oxygen conditions, ethyl- <br />                                                                           ene induces ADH1 Arabidopsis, and, recently, <br />                                                                           quantitative trait locus mapping revealed both <br />                                                                           submergence tolerance elongation growth in <br />                                                                           deepwater rice controlled transcription factors <br />                                                                           ERF family (Xu et al., 2006; Hattori et al., 2009). <br /> Figure 7. RNA-blot analysis RAP2.2, ADH1, PDC1 expression          Ethylene regulates morphological adaptation <br /> in wild type (wt) RAP2.2 T-DNA insertion lines rap2.2-1 and <br />                                                                           responses flooding submergence, shoot <br /> rap2.2-2. Seventeen-day-old wild-type plants plants T-DNA <br /> insertion lines rap2.2-1 rap2.2-2 treated 21% oxygen or <br />                                                                           elongation, aerenchyma formation, adventitious <br /> 5% oxygen dark 2 8 h isolation RNA       root growth (for review, Bailey-Serres Voesenek, <br /> shoot.                                                                    2008). Both tolerance submergence (SUB1A-1) and <br />                                                                           escape reaction elongation growth deepwater <br />                                                                           rice (SK1 SK2) controlled particular alleles <br /> darkness assay. Changing RAP2.2 expres-                        ERF genes. <br /> sion caused changes specific genes, overall                           ERFs play role low-oxygen stress conditions <br /> expression profiles similar wild-type re-                         recently identified dicots. The <br /> sponses clustered them. Overexpression                        Arabidopsis ERF71 ERF73 genes (HRE2 and <br /> RAP2.2 resulted up-regulation ADH1 down-                        HRE1, respectively) induced low-oxygen stress <br /> regulation ACS9 normoxia light. When                      dark, overexpression results improved <br /> hypoxia-darkness imposed ox3, increases                         hypoxia survival (Licausi et al., 2010). We studied <br /> expression AlaAT1 RAP2.3 decreases                          Arabidopsis RAP2.2 (At3g14230), originally thought to <br /> AtMYB2 ACS2 observed (Fig. 10). Loss                        responsive hypoxia roots (Klok et al., 2002) but <br /> functional RAP2.2 resulted up-regulation                          shown constitutively expressed hyp- <br /> number genes normoxia-light, including                       oxia tissue. RAP2.2 belongs group <br /> ethylene biosynthesis genes ASC2, ACS7, ASC9,                             VII ERF family SUB1A (Fig. 1; Nakano et al., 2006) <br /> and AOX1 NIP2;1 AtMYB2. Loss                            recently shown activate Arabidopsis <br /> hypoxia-darkness induction AtMYB2, ACS2,                           genes encoding carotenoid biosynthesis pathway <br /> ETR2 observed rap2.2-2 (Fig. 10).                             enzymes phytoene synthase phytoene desaturase <br />    Overall, major clusters expres-                    (Welsch et al., 2007). These genes differentially <br /> sion profiles (Fig. 10). Group I consisted genes                 expressed hypoxic conditions (Loreti et al., <br /> that mainly darkness hypoxia-darkness in-                        2005). Our phylogenetic analysis group VII <br /> duced repressed hypoxia-light conditions.                       ERF factors rice Arabidopsis shows the <br /> This group contained senescence-associated genes                      dicot ERF transcription factors evolved indepen- <br /> SEN1 SEN5 number ethylene-                      dently. This reflected relatively high degree of <br /> associated genes, ACS2 AOX1. Group II                      sequence divergence outside highly conserved <br /> the largest cluster (16 members) contains predom-                     AP2 domain. This complicates finding true SUB1 <br /> inantly genes induced darkness, hypoxia-darkness,                      homolog Arabidopsis based sequence similarity <br /> and hypoxia-light treatments. All known low-                       only. But RAP2.2, according AP2 domain structure, <br /> oxygen-induced genes involved fermentation                         certainly related SUB1 rice the <br /> sugar metabolism (ADH1, PDC1, LDH1, AtSUS1,                           rice SK1 SK2 genes, belong different <br /> AtSUS4) transcription factors (ANR1,                      ERF subfamily (Hattori et al., 2009). However, two <br /> AtERF4, ERF71, ERF73), ethylene biosynthesis                      Arabidopsis group VII proteins, ERF73 and <br /> genes ASC7 ACO1, ethylene receptor                            ERF71, closely related SUB1A than <br /> ETR2 present group. Group III                       RAP2.2 (Fig. 1B). These transcription factors play <br /> up genes normally induced                       role low-oxygen stress dark (Licausi et al., <br /> any treatments wild-type plants                       2010). <br /> Plant Physiol. Vol. 153, 2010                                                                                                   765 <br />  Hinz et al. <br />  <br />  <br /> Figure 8. Survival plants ele- <br /> vated reduced expression of <br /> RAP2.2 compared wild <br /> type (wt). A, Images wild-type seed- <br /> lings T-DNA insertion lines <br /> rap2.2-1 rap2.2-2 1 week after <br /> recovery 24 h 5% oxygen and <br /> 40 h 0.1% oxygen dark. B, <br /> Images wild-type seedlings the <br /> overexpression lines ox2 ox3 at <br /> 1 week recovery 24 h 5% <br /> oxygen 48 h 0.1% oxygen the <br /> dark. The control plants exper- <br /> iment kept dark. <br />  <br />  <br />  <br />  <br />    We RAP2.2 does play significant role     ethylene biosynthesis, Arabidopsis ethylene bio- <br /> metabolic adaptation flooding stress Arabidopsis,    synthesis genes ACS4 ACS7 induced dark- <br /> suggesting group VII ERF factors special-     ness (Wang et al., 2005). It clear RAP2.2 <br /> ized abiotic stress responses. RAP2.12, closest     expression roots high superinduction <br /> relative RAP2.2 (Fig. 1), recently shown         seen hypoxia darkness. It possible that <br /> regulate ADH1 expression (Papdi et al., 2008). Other       roots grown soil (darkness) liquid medium <br /> related dicot ERFs, tomato JERF3 pepper CaPF1,         (entrapment) naturally accumulate higher ethylene <br /> play role salt freezing tolerance, respectively   levels. <br /> (Wang et al., 2004; Yi et al., 2004). RAP2.2 ex-          A variety experimental setups used to <br /> pressed high levels roots low levels   study low-oxygen responses plants. Some the <br /> shoots, distribution similar fermentation pathway     recent studies used anaerobiosis (0% oxy- <br /> genes like ADH1 (Dolferus et al., 1994), PDC1 (R.          gen; Mustroph et al., 2006, 2009), used <br /> Dolferus, unpublished data), LDH1 (Dolferus et al.,        hypoxic conditions (0.1%&#226;&#8364;&#8220;5% oxygen; Van Dongen <br /> 2008), AlaAT1 (Miyashita et al., 2007). While          et al., 2009; Licausi et al., 2010). Most exper- <br /> RAP2.2 expression roots constitutively high      iments use darkness low-light conditions, the <br /> not affected hypoxia, shoots expression          effect light far investigated. We have <br /> appeared induced darkness              routinely carried low-oxygen treatments on <br /> hypoxia. The gene regulated ethylene        Arabidopsis plantlets sealed tanks defined <br /> ethylene-generating compounds, fact           gas composition dark stop complications <br /> mutations ethylene signal transduction components       arising altered oxygen concentrations to <br /> affect expression clearly indicates RAP2.2        photosynthesis release trapped oxygen from <br /> functions ethylene-responsive signaling path-        plant tissues. These treatments resemble complete <br /> way. Low-light conditions trigger leaf senescence,         submergence, nature leads reduced <br /> which involves ethylene accumulation ethyl-        photosynthetic activity combination re- <br /> ene signaling pathway (for review, Gan             stricted light CO2 availability. It conceivable that <br /> Amasino, 1997; Quirino et al., 2000; Wingler           low light CO2 availability, induction ethylene <br /> Roitsch, 2008; Lin et al., 2009). Darkness trigger     synthesis, induction promotion senescence <br /> 766                                                                                       Plant Physiol. Vol. 153, 2010 <br />                                                                                                          RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />                                                                                12 h ethylene, induction only <br />                                                                                strong hypoxic conditions (Peng et al., 2001). <br />                                                                                These studies carried intact seedlings, <br />                                                                                likely light, nonetheless support conclu- <br />                                                                                sion ethylene trigger required for <br />                                                                                inducing ADH1 expression. Hypoxic conditions are <br />                                                                                essential fully induce ADH1 expression. The rice <br />                                                                                SUB1A gene induced submergence conditions, <br />                                                                                remains shown SUB1A also <br />                                                                                responds darkness. <br />                                                                                   Overexpressing RAP2.2 using strong constitutive <br />                                                                                promoter did cause noticeable phenotypic <br />                                                                                changes normal growth conditions, both <br />                                                                                ADH1 PDC1 expression shoots increased <br />                                                                                following hypoxic-dark treatment. The overexpressing <br />                                                                                lines higher ADH PDC enzyme activity <br />                                                                                shoots, suggesting RAP2.2 regulator the <br />                                                                                alcohol fermentation pathway genes. The overexpres- <br />                                                                                sion ADH1 PDC1 RAP2.2-overexpressing <br />                                                                                lines evident hypoxic conditions the <br />                                                                                dark normoxic-dark conditions. This <br />                                                                                suggests ethylene accumulation RAP2.2 in- <br />                                                                                duction sufficient ADH1 PDC1 activa- <br />                                                                                tion hypoxic conditions required. <br />                                                                                   The RAP2.2-overexpressing lines showed signifi- <br />                                                                                cantly higher hypoxia plant survival rates the <br />                                                                                dark, knockout lines showed opposite <br />                                                                                phenotype. This demonstrates RAP2.2 act a <br />                                                                                regulator plant&#226;&#8364;&#8482;s response hypoxia dark <br />                                                                                expression RAP2.2 rate limiting for <br />                                                                                regulating genes lead hypoxia tolerance that <br />                                                                                higher preexisting levels RAP2.2 enable quicker <br />                                                                                response hypoxia, resulting enhanced hypoxia <br />                                                                                tolerance. Submergence-tolerant rice plantlets pro- <br />                                                                                duced highly elevated levels ethanol anaero- <br />                                                                                biosis dark sufficient ATP synthesis to <br />                                                                                ensure plantlet survival, flooding-sensitive <br />                                                                                wheat (Triticum aestivum) seedlings less <br />                                                                                efficient fermentative ATP production. Fermenta- <br /> Figure 9. Effects low-oxygen treatment percentage plant           tion occurred mainly shoot, indicating vital <br /> survival. A, Survival percentage wild type (wt) T-DNA           role anaerobic shoot metabolism plantlet sur- <br /> insertion lines rap2.2-1 rap2.2-2 measured 1 week recovery <br />                                                                                vival dark (Mustroph et al., 2006). <br /> from 24 h 5% oxygen 40 h 0.1% oxygen (6SE; n = 5). Asterisks <br /> denote line survival percentages significantly different as <br />                                                                                   We investigated overlap hypoxic treat- <br /> determined t test (P , 0.01). B, Survival percentage wild type       ment light darkness effect of <br /> and overexpression lines ox2 ox3 measured 1 week recovery        RAP2.2 overexpression knockout lines these <br /> from 24 h 5% oxygen 48 h 0.1% oxygen (6SE; n = 3). Asterisks         responses using 32 candidate genes play role in <br /> denote line survival percentages significantly different    hypoxia ethylene responses. The clustering the <br /> determined t test (P , 0.01). C, Fresh weight (FW) roots shoots      treatments indicates hypoxia-darkness treatment <br /> of 4-week-old light-grown plants wild type, T-DNA knockout          responses closer darkness-only treatment <br /> line rap2.2-2, RAP2.2-overexpressing line ox3 measured       normoxic conditions hypoxia <br /> recovery period survival assays. rap2.2-2 consistently lower        treatment light conditions. Darkness gen- <br /> fresh weight. The opposite observed ox3 overexpression line. <br />                                                                                erally causes lower induction observed <br /> Data points asterisks denote fresh weights significantly different from <br /> the wild type determined t test (P , 0.01). <br />                                                                                darkness hypoxia combined. This <br />                                                                                suggests clear link hypoxia and <br />                                                                                darkness responses shoots. Observing effect of <br /> are naturally intertwined inherent                            altering levels RAP2.2 showed affects a <br /> natural submergence conditions. ADH1 expression                             subset pathways. The major effect observed <br /> roots low strongly induced hypoxic                                fermentation genes ADH1 PDC1 <br /> conditions (Dolferus et al., 1994). Previous work                           ethylene biosynthesis receptor genes as <br /> Arabidopsis seedlings indicated ADH1 induced                           ACS2, ASC7, ASC9, ETR2. It possible that <br /> Plant Physiol. Vol. 153, 2010                                                                                                        767 <br />  Hinz et al. <br />  <br />  <br /> Figure 10. Hierarchical clustering expression data <br /> for 32 candidate genes. Expression analysis was <br /> carried RNA extracted seedlings <br /> of wild type, RAP2.2-overexpressing line ox3 <br /> (ox), RAP2.2 knockout line rap2.2-2 (ko). Plant <br /> treatments consisted 5 h dark nor- <br /> moxia (dark), 5 h 5% oxygen light (o2), or <br /> a combination 5 h dark 5% oxygen <br /> (o2 + dark). Expression ratios wild-type plants are <br /> treatment divided value obtained under <br /> normoxia-light conditions. ox ko values are <br /> ox3 rap2.2-2 expression normoxia-light <br /> compared wild type normoxia-light. <br /> Treatment values ox ko treatments divided <br /> by ox ko values obtained normoxia-light. <br /> The description genes used (locus, gene <br /> name, function) listed Supplemental Table <br /> S1. Hierarchical clustering used divide the <br /> gene set clusters: group I consists genes <br /> induced darkness hypoxia-darkness re- <br /> pressed hypoxia-light; group II consists of <br /> predominantly genes induced darkness, hypoxia- <br /> darkness, hypoxia-light treatments contains <br /> the best known low-oxygen-induced genes are <br /> involved fermentation sugar metabolism as <br /> well ethylene biosynthesis genes ethylene <br /> receptor ETR2; group III consists genes were <br /> normally induced treatments <br /> in wild-type plants expression altered <br /> by changing RAP2.2 expression levels. <br />  <br />  <br />  <br />  <br /> RAP2.2 affect genes; however,          survival (Ellis et al., 1999; Ismond et al., 2003). The <br /> have involved pathways associated          observation shoots, overexpression of <br /> with hypoxia tolerance.                                     RAP2.2 resulted higher ADH PDC activity <br />    The importance fermentation glycolysis        indicates improved hypoxia survival plants <br /> surviving low oxygen established.             overexpressing RAP2.2 enhancement of <br /> ADH1, PDC1, Suc synthase genes AtSUS1               fermentation. The poor hypoxia tolerance the <br /> and AtSUS4 vital Arabidopsis tolerance        RAP2.2 knockouts partly related or <br /> low oxygen (Ellis et al., 1999; Rahman et al., 2001;        unrelated deficiencies fermentation, as <br /> Ismond et al., 2003; Ku&#194;&#168;rsteiner et al., 2003; Bieniawska   ADH1 PDC1 expression partly reduced. <br /> et al., 2007). Increased root survival observed    Knocking RAP2.2 does result changes ACS2, <br /> in LDH1-overexpressing Arabidopsis lines (Dolferus          ACS7, ACS9 indicate negative feedback reg- <br /> et al., 2008). Expression PDC1 considered   ulation ethylene biosynthesis RAP2.2. It be <br /> rate-limiting step ethanol fermentation, over-       hypothesized altered ethylene signaling rap2-2 <br /> expression shown increase Arabidopsis           plants contributes poor hypoxia tolerance. <br /> 768                                                                                      Plant Physiol. Vol. 153, 2010 <br />                                                                                                                       RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />    Although ERF factors shown interact                               17 mm) Murashige Skoog (MS) medium (Murashige Skoog, 1962), <br />                                                                                   sealed 3M micropore tape (Eydam), kept 4&#194;&#176;C dark 2 d. <br /> the GCC box AP2/ERF-binding motif (Ohme-Takagi                                 Plates transferred growth chamber, plants grown at <br /> and Shinshi, 1995; Lin et al., 2008), RAP2.2 shown                            22&#194;&#176;C photoperiod 16 h light/8 h dark times indicated. <br /> to bind ATCTA promoter element (Welsch et al.,                              Arabidopsis root cultures generated incubating leaf discs callus <br /> 2007). The ADH1 promoter contains ATCTA ele-                                   induction medium (Valvekens et al., 1988) 3 4 d dark. Leaf discs <br /> ment proximal TATA box (positions 257 253;                              infected Agrobacterium rhizogenes strain A4RS (Vilaine et al., <br />                                                                                   1987). When hairy roots appeared, leaf discs transferred liquid MS <br /> data shown). A GCC box-like motif, GC motif,                              medium incubated 50 rpm rotary shaker platform dark. <br /> is GT motif anaerobic                                   For RNA isolation northern-blot analysis, promoter:GUS, enzyme <br /> response element (Olive et al., 1991; Dolferus et al.,                            activity assays, plants grown half-strength MS medium containing <br /> 1994). Transactivation assays wild-type de-                              1.5% (w/v) Suc 0.8% (w/v) agar pH 5.7 (70 mE m22 s21). Hypoxic <br />                                                                                   treatment 17-d-old plants carried atmosphere containing 5% <br /> leted ADH1 promoter constructs required in-                                (v/v) oxygen, 0.04% (v/v) CO2, 94.96% (v/v) N2 dark the <br /> vestigate RAP2.2 interacts directly                              duration indicated. For ethylene measurements, plantlets transferred <br /> GC motif ATCTA element. RAP2.2 act                                1 week MS medium containing 2% (w/v) agarose prevent root <br /> indirectly ADH1 expression transcription                             growth agarose subsequent wounding transfer of <br /> factors. ERF transcription factors known interact                          plants vials used ethylene measurement, described <br />                                                                                   previously (Peng et al., 2001). For treatment effectors ACC, ethephon, <br /> with complicated networks (On&#203;&#339;ate-                                  STS (freshly prepared 100 mM sodium thiosulfate 100 mM silver <br /> Sa&#194;&#180;nchez et al., 2007). ERF71, ERF73, ERF4                            nitrate ratio 1:1), CoCl2, plants grown solid medium <br /> expressed group genes ADH1;                               transferred 2 d treatment liquid MS medium improve uptake <br /> possible ERF factors function downstream                               effectors. After preincubation 2 h ethephon, ACC, STS, CoCl2 <br />                                                                                   concentrations indicated, plants exposed hypoxic treatment <br /> of RAP2.2 activate group II genes. A hypoxic signal                            described above. <br /> is required activate ADH1 expression, suggesting                                   For RNA isolation QRT-PCR, plants grown half-strength <br /> that regulatory pathways act                            MS medium containing 1.5% (w/v) Suc 0.8% (w/v) agar pH 5.7 (70 mE <br />                                                                                     22 21 <br /> ADH1 promoter. The ethylene-dependent pathway                                     m s ). Plants divided treatments (five plants per <br /> that involves RAP2.2 able induce                                  treatment): normoxia light (70 mE m22 s21) 5 h, normoxia dark <br />                                                                                   5 h, hypoxia (5% oxygen) light 5 h, hypoxia (5% oxygen) the <br /> ADH1, hypoxic conditions essential. The                              dark 5 h. <br /> fact hypoxia able induce ethylene biosynthe- <br /> sis suggests ethylene pathway com- <br /> pletely independent hypoxic conditions. On                                 Hypoxic Assays <br /> other hand, AtMYB2, binds GT motif                                       Low-oxygen survival assays performed outlined Christianson <br /> (Hoeren et al., 1998), appears affected dark-                            et al. (2009). Briefly, 2.5-week-old plants grown 16-h light cycle <br /> ness ethylene (Fig. 10); suggests                             (100 mE m22 s21) MS medium containing 3% (w/v) Suc 0.8% (w/v) <br />                                                                                   agar pH 5.7 transferred MS liquid medium containing 3% (w/v) <br /> is cross talk regulatory pathways.                                Suc 1 d prior treatment. Immediately prior treatment, plants were <br /> Oxygen-dependent regulatory mechanisms trig-                                 transferred liquid medium sparged 5% (v/v) oxygen <br /> ger anaerobic gene expression identified                             placed 3.5-L anaerobic chambers (Oxoid) purged 5% (v/v) <br /> bacteria, yeast, mammals remain far elusive                            oxygen flow rate approximately 10 L min21 20 min. The plants were <br /> in plants (Poyton, 1999; Gilles-Gonsalez, 2001; Schofield                         left 5% (v/v) oxygen dark gentle shaking 24 h, purged <br />                                                                                   0.1% (v/v) oxygen left 40 48 h. Following treatment, plants <br /> and Ratcliffe, 2004; Bailey-Serres Chang, 2005). To                           given fresh liquid medium allowed recover orbital shaker in <br /> further unravel gene regulation works                                   normal growth cabinet conditions 1 week survival scoring. Survival <br /> hypoxic conditions identify components                              scores averaged independent experiments. Each <br /> the ethylene pathway especially hypoxic                                   experiment contained technical replicates, replicate consisting <br />                                                                                   12 plants. <br /> pathway require carefully designed exper- <br /> iments account lighting conditions dur- <br /> ing hypoxic treatment. It important                               Generation Characterization Transgenic Plants <br /> further establish importance lighting conditions                               To constitutively overexpress RAP2.2, 1.9-kb genomic fragment including <br /> and difference real submergence condi-                                open reading frame, 5# 3# untranslated regions, introns was <br /> tions artificial assays use gas mixtures                              amplified Col-0 using gene specific primers 5#-GAATTCGAGTAG- <br /> combination light dark conditions.                                        AGCTTTCGTGAAGCCACCAT-3# (forward) 5#-GGATCCAGAAGAT- <br />                                                                                   TCATTGAACAGATA-3# (reverse). The fragment restricted EcoRI <br />                                                                                   BamHI cloned EcoRI BamHI sites pART7 (Gleave, <br />                                                                                   1992). The resulting 35S-RAP2.2-OCS fragment subcloned NotI <br /> MATERIALS AND METHODS                                                             fragment binary vector pART27 transformed Arabidopsis <br />                                                                                   ecotype Col-0 using floral dip method (Clough Bent, 1998). The <br /> Plant Material, Growth Conditions,                                            presence integrity RAP2.2 genomic DNA sequence verified by <br /> Low-Oxygen Treatments                                                             sequencing insertion pART27. <br />                                                                                       To generate RAP2.2 promoter construct, 1,112-bp RAP2.2 promoter <br />    Experiments carried Arabidopsis (Arabidopsis thaliana ecotype      region amplified genomic DNA ecotype Col-0 using gene- <br /> Col-0). Col-0 wild-type seeds originally obtained GABI-Kat           specific primers 5#-GAATTCCAATATTGCCACTCCATGATAA-3# (forward) <br /> (Max-Planck-Institut), T-DNA insertion lines SAIL_184_G12 (CS871911)          5#-ATAGCTCCTCCACCCATGGTGGCTTCACGA-3# (reverse). The re- <br /> and SAIL_799_D10 (CS876942) SALK Institute, ethyl-          sultant PCR fragment, contained EcoRI site NcoI site, was <br /> ene signaling mutants ctr1, ein2, ein3 Arabidopsis Biological   ligated GUS-NOS cassette pHW9 (Dolferus et al., 1994) and <br /> Resource Center. Seeds surface sterilized 20 min 1 mL 0.5% (w/v)   subsequently cloned binary vector pBIN19 using restriction sites <br /> sodium hypochlorite washed times autoclaved water. Five          EcoRI XbaI (Bevan, 1984). Arabidopsis ecotype Col-0 plants trans- <br /> 10 seeds sown plate (60 mm 3 15 mm 120 mm 3 120 mm; height,           formed, GUS-expressing lines analyzed. <br />  <br /> Plant Physiol. Vol. 153, 2010                                                                                                                                769 <br />  Hinz et al. <br />  <br />  <br /> Phylogenetic Analysis ERF Sequences                                              centrifuged 15,000g 15 min 4&#194;&#176;C. PDC enzyme activity was <br />                                                                                     measured using modified protocol Rivoal et al. (1997). Soluble protein <br />    Full-length ERF protein sequences retrieved GenBank                extracted cold extraction buffer (50 mM MES-NaOH, pH 6.2, 5 mM <br /> database analyzed using MEGA 4.0 software (Tamura et al., 2007).            dithiothreitol, 1 mM MgCl2, 1 mM thiamine pyrophosphate) centri- <br /> Sequences aligned using ClustalW progressive alignment algorithm           fuged 15,000g 15 min 4&#194;&#176;C. Protein concentration determined with <br /> (Higgins et al., 1994) neighbor-joining method (Saitou Nei, 1987).      Bradford assay (Roti-Quant; Roth). The enzymatic reaction started <br /> Bootstrap figures indicated phylogenetic tree.                           addition 800 mL activation buffer (0.15 M Tris-HCl, pH 8.0, <br />                                                                                     0.3 mM NAD+, 0.175% [v/v] ethanol ADH activity measurements or <br />                                                                                     50 mM MES-NaOH, pH 6.2, 0.1 mM thiamine pyrophosphate, 0.5 mM MgCl2, <br /> RNA Extraction, Northern Blotting, QRT-PCR                                      0.15 mM NADH, 3.3 mM sodium pyruvate, 40 units mL21 yeast ADH for <br />     Total RNA extracted grinding frozen tissue liquid nitrogen     PDC enzyme activity measurements). Activities calculated from <br /> a mortar pestle. The root shoot sections rosette-stage plants     increase decrease A340 time. <br /> were separated butting <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> rosette scalpel. Tri-Reagent <br /> (Sigma-Aldrich) added ground tissue, RNA isolated                Ethylene Measurement <br /> according manufacturer&#226;&#8364;&#8482;s instructions. For gels, 20 mg RNA per <br /> sample separated denaturing agarose gel (1% [w/v] agarose, 40 mM              Thirty plants treated 21% (v/v) 5% (v/v) oxygen as <br /> MOPS buffer, 6% [v/v] formaldehyde). The ribosomal RNA staining                 described above, transferred 22-mL vial, capped gas tight. Ethylene <br /> pattern (ethidium bromide) used loading control. RNA transferred       measured 1 h head space using gas chromatograph (GC-14B; <br /> to nylon membrane (Hybond N+; GE Healthcare) hybridized 32P-               Shimadzu). Plant fresh weights determined, ethylene production <br /> labeled probes obtained using Ready-To-Go dCTP labeling kit (GE Health-           mg fresh weight calculated. <br /> care). Hybridization performed overnight 68&#194;&#176;C hybridization solution <br /> (5% [w/v] dextran sulfate, 1 M NaCl, 1% [w/v] SDS, 0.1 mg mL21 heat-                Sequence data article GenBank/EMBL data <br /> denatured salmon sperm DNA). Blots washed times 23 SSC/1%               libraries accession number NM_180251. <br /> (w/v) SDS 1 10 min, 13 SSC/1% (w/v) SDS 10 min, once <br /> in 0.53 SSC/1% (w/v) SDS 10 min. The gene-specific primer pairs used to <br /> generate PCR probes genomic DNA 5#-CCTAGCGTCGTATCC-                   Supplemental Data <br /> CAGAA-3# 5#-CAAGGCGTTGTCAAGGTATGC-3# RAP2.2, 5#-CGA- <br /> GACACCTTACAACAACAC-3# 5#-CAGATGAGAGGTTCAAACACAT-3#                                 The following materials available online version article. <br /> for RAP2.12, 5#-GATTGTTGAGAGTGTTGGAG-3# 5#-CTTGGTCGAATC- <br />                                                                                        Supplemental Figure S1. Amino acid sequence alignment RAP2.2- <br /> TTTTAGAGT-3# ADH1, 5#-TCTCTCACACACATACACAAAC-3# and <br />                                                                                          related ERF transcription factors. <br /> 5#-CAAGCAAAGTGAGGTTGAAATC-3# PDC1. For RAP2.2 ADH1, <br /> probes generated coding region. For RAP2.12, probe including           Supplemental Figure S2. ADH PDC activity measurements shoots <br /> the coding region 3# untranslated region generated. For              wild-type Col-0 RAP2.2-overexpressing line ox3. <br /> PDC1, probe including 5# untranslated region coding region <br /> was used. The probes RAP2.2 RAP2.12 shared 23% sequence                   Supplemental Figure S3. QRT-PCR expression data 32 candidate <br /> identity.                                                                                genes. <br />     QRT-PCR performed described previously (Wilson et al., 2005). The           Supplemental Figure S4. Detailed heat map QRT-PCR gene expression <br /> RT product amplified using gene-specific primers (Supplemental Table                 data 32 candidate genes. <br /> S2). Reactions performed triplicate Rotor-Gene 6000 (Qiagen). <br /> Data normalized At5g08290 (Czechowski et al., 2005) analyzed               Supplemental Table S1. List 32 candidate genes used gene expres- <br /> using comparative quantification procedure (Wilson et al., 2005). Hierarchi-           sion profiling studies QRT-PCR. <br /> cal clustering log-transformed expression data carried using            Supplemental Table S2. List PCR primers used QRT-PCR experi- <br /> Cluster Treeview (Eisen et al., 1998) programs. Heat maps con-                  ments. <br /> structed using University Toronto BAR Heatmapper tool (http://www. <br /> bar.utoronto.ca/ntools/cgi-bin/ntools_heatmapper.cgi).                              Received February 18, 2010; accepted March 26, 2010; published March 31, <br />                                                                                     2010. <br /> Expression Analysis RAP2.2:GUS Plants <br />     For RAP2.2:GUS studies etiolated seedlings, seeds exposed light <br /> for 4 h synchronize germination incubated dark 4 d. In       LITERATURE CITED <br /> addition, RAP2.2:GUS seedlings grown 4 d plants grown <br />                                                                                     Bailey-Serres J, Chang R (2005) Sensing signalling response to <br /> for 17 d half-strength MS medium containing 1.5% (w/v) Suc 0.8% (w/v) <br />                                                                                         oxygen deprivation plants organisms. Ann Bot (Lond) 96: <br /> agar pH 5.7 70 mE m22 s21 light/dark cycle described above. To <br />                                                                                         507&#226;&#8364;&#8220;518 <br /> analyze RAP.2.2 promoter activity 5% (v/v) oxygen dark, <br />                                                                                     Bailey-Serres J, Voesenek LACJ (2008) Flooding stress: acclimations and <br /> 17-d-old plants grown 16-h light cycle 100 mE m22 s21 MS <br />                                                                                         genetic diversity. Annu Rev Plant Biol 59: 313&#226;&#8364;&#8220;339 <br /> medium containing 3% (w/v) Suc 0.8% (w/v) agar pH 5.7 then <br />                                                                                     Bevan M (1984) Binary Agrobacterium vectors plant transformation. <br /> transferred MS liquid medium containing 3% (w/v) Suc 1 d prior to <br />                                                                                         Nucleic Acids Res 12: 8711&#226;&#8364;&#8220;8721 <br /> treatment. Immediately prior treatment, plants transferred liquid <br />                                                                                     Bieniawska Z, Paul Barratt DH, Garlick AP, Thole V, Kruger NJ, Martin <br /> medium. For low-oxygen-treated plants, medium sparged with <br />                                                                                         C, Zrenner R, Smith AM (2007) Analysis sucrose synthase gene <br /> 5% (v/v) oxygen control sparged. Plants then <br />                                                                                         family Arabidopsis. Plant J 49: 810&#226;&#8364;&#8220;828 <br /> placed 3.5-L anaerobic chambers. The low-oxygen-treated plants were <br />                                                                                     Bla&#194;&#180;zquez MA, Soowal LN, Lee I, Weigel D (1997) LEAFY expression and <br /> purged 5% (v/v) oxygen flow rate approximately 10 L min21 for <br />                                                                                         flower initiation Arabidopsis. Development 124: 3835&#226;&#8364;&#8220;3844 <br /> 20 min. The plants left dark gentle shaking harvest. <br />                                                                                     Bomblies K (2000) Whole mount GUS staining. In D Weigel, J Glazebrook, <br /> Two independent transgenic RAP2.2:GUS transgenic lines analyzed for <br />                                                                                         eds, Arabidopsis: A Laboratory Manual. Cold Spring Harbor Labora- <br /> GUS activity seedling vegetative stages (Bla&#194;&#180;zquez et al., 1997; <br />                                                                                         tory Press, Cold Spring Harbor, NY, pp 243&#226;&#8364;&#8220;245 <br /> Bomblies, 2000). <br />                                                                                     Bu&#194;&#168;ttner M, Singh KB (1997) Arabidopsis thaliana ethylene-responsive <br />                                                                                         element binding protein (AtEBP), ethylene-inducible, GCC box <br /> ADH PDC Activity Assays                                                             DNA-binding protein interacts ocs element binding protein. <br />                                                                                         Proc Natl Acad Sci USA 94: 5961&#226;&#8364;&#8220;5966 <br />    For ADH enzyme activity measurements, soluble protein extracted           Cheong YH, Moon BC, Kim JK, Kim CY, Kim MC, Kim IH, Park CY, Kim <br /> cold extraction buffer (100 mM Tris-HCl, pH 8.0, 25% [v/v] glycerol, 0.8% [v/v]         JC, Park BO, Koo SC, et al (2003) BWMK1, rice mitogen-activated <br /> b-mercaptoethanol, 2% [w/v] polyvinylpyrrolidone, 5 mM dithiothreitol)              protein kinase, locates nucleus mediates pathogenesis-related <br />  <br /> 770                                                                                                                           Plant Physiol. Vol. 153, 2010 <br />                                                                                                                       RAP2.2 Regulates Hypoxia Survival <br />  <br />  <br />    gene expression activation transcription factor. Plant Physiol 132:      Sub1A mediates stress-responsive expression AP2/ERF transcription <br />    1961&#226;&#8364;&#8220;1972                                                                        factors. Plant Physiol 152: 1674&#226;&#8364;&#8220;1692 <br /> Christianson JA, Wilson IW, Llewellyn DJ, Dennis ES (2009) The low-              Kende H, van der Knaap E, Cho HT (1998) Deepwater rice: model plant <br />    oxygen induced NAC domain transcription factor ANAC102 affects                   study stem elongation. Plant Physiol 118: 1105&#226;&#8364;&#8220;1110 <br />    viability Arabidopsis thaliana seeds following low-oxygen treatment.       Klok EJ, Wilson IW, Wilson D, Chapman SC, Ewing RM, Somerville SC, <br />    Plant Physiol 149: 1724&#226;&#8364;&#8220;1738                                                     Peacock WJ, Dolferus R, Dennis ES (2002) Expression profile analysis <br /> Clough SJ, Bent AF (1998) Floral dip: simplified method Agro-                 low-oxygen response Arabidopsis root cultures. Plant Cell 14: <br />    bacterium-mediated transformation Arabidopsis thaliana. Plant J 16:           2481&#226;&#8364;&#8220;2494 <br />    735&#226;&#8364;&#8220;743                                                                       Ku&#194;&#168;rsteiner O, Dupuis I, Kuhlemeier C (2003) The pyruvate decarboxylase1 <br /> Czechowski T, Stitt M, Altmann T, Udvardi MK, Scheible WR (2005)                    gene Arabidopsis required anoxia environ- <br />    Genome-wide identification testing superior reference genes           mental stresses. Plant Physiol 132: 968&#226;&#8364;&#8220;978 <br />    transcript normalization Arabidopsis. Plant Physiol 139: 5&#226;&#8364;&#8220;17              Licausi F, van Dongen JT, Giuntoli B, Novi G, Santaniello A, Geigenberger P, <br /> Dennis ES, Dolferus R, Ellis M, Rahman M, Wu Y, Hoeren FU, Grover A,                Perata P (2010) HRE1 HRE2, hypoxia-inducible ethylene response <br />    Ismond KP, Good AG, Peacock WJ (2000) Molecular strategies                   factors, affect anaerobic responses Arabidopsis thaliana. Plant J (in press) <br />    improving waterlogging tolerance plants. J Exp Bot 51: 89&#226;&#8364;&#8220;97               Lin RC, Park HJ, Wang HY (2008) Role Arabidopsis RAP2.4 regulating <br /> Dolferus R, Jacobs M, Peacock WJ, Dennis ES (1994) Differential interac-            light- ethylene-mediated developmental processes drought <br />    tions promoter elements stress responses Arabidopsis Adh            stress tolerance. Mol Plant 1: 42&#226;&#8364;&#8220;57 <br />    gene. Plant Physiol 105: 1075&#226;&#8364;&#8220;1087                                            Lin Z, Zhong S, Grierson D (2009) Recent advances ethylene research. <br /> Dolferus R, Wolansky M, Carroll R, Miyashita Y, Ismond K, Good AG                   J Exp Bot 60: 3311&#226;&#8364;&#8220;3336 <br />    (2008) Functional analysis lactate dehydrogenase hypoxic            Liu F, VanToai T, Moy LP, Bock G, Linford LD, Quackenbush J (2005) <br />    stress Arabidopsis. Funct Plant Biol 35: 131&#226;&#8364;&#8220;140                              Global transcription profiling reveals comprehensive insights hyp- <br /> Drew MC, He CJ, Morgan PW (2000) Programmed cell death aeren-                   oxia response Arabidopsis. Plant Physiol 137: 1115&#226;&#8364;&#8220;1129 <br />    chyma formation roots. Trends Plant Sci 5: 123&#226;&#8364;&#8220;127                         Loreti E, Poggi A, Novi G, Alpi A, Perata P (2005) A genome-wide analysis <br /> Eisen MB, Paul T, Spellman PT, Brown PO, Botstein D (1998) Cluster                  effects sucrose gene expression Arabidopsis seedlings <br />    analysis display genome-wide expression patterns. Proc Natl               anoxia. Plant Physiol 137: 1130&#226;&#8364;&#8220;1138 <br />    Acad Sci USA 95: 14863&#226;&#8364;&#8220;14868                                                  Mergemann H, Sauter M (2000) Programmed death epidermal cells <br /> Ellis MH, Dennis ES, Peacock WJ (1999) Arabidopsis roots shoots                 facilitates emergence adventitious roots deepwater rice. Plant <br />    different mechanisms hypoxic stress tolerance. Plant Physiol            Physiol 124: 609&#226;&#8364;&#8220;614 <br />    119: 57&#226;&#8364;&#8220;64                                                                    Me&#194;&#180;traux JP, Kende H (1983) The role ethylene growth response of <br /> Fujimoto SY, Ohta M, Usui A, Shinshi H, Ohme-Takagi M (2000)                        submerged deep water rice. Plant Physiol 72: 441&#226;&#8364;&#8220;446 <br />    Arabidopsis ethylene-responsive element binding factors act tran-          Miyashita Y, Dolferus R, Ismond KP, Good AG (2007) Alanine amino- <br />    scriptional activators repressors GCC box-mediated gene expres-            transferase catalyses breakdown alanine hypoxia Arabi- <br />    sion. Plant Cell 12: 393&#226;&#8364;&#8220;404                                                     dopsis thaliana. Plant J 49: 1108&#226;&#8364;&#8220;1121 <br /> Fukao T, Bailey-Serres J (2004) Plant responses hypoxia: Is survival        Murashige T, Skoog F (1962) A revised medium rapid growth and <br />    balancing act? Trends Plant Sci 9: 449&#226;&#8364;&#8220;456                                       bioassays tobacco tissue cultures. Physiol Plant 18: 100&#226;&#8364;&#8220;127 <br /> Fukao T, Harris T, Bailey-Serres J (2009) Evolutionary analysis Sub1      Mustroph A, Boamfa EI, Laarhoven LJJ, Harren FJM, Albrecht G, Grimm <br />    gene cluster confers submergence tolerance domesticated rice.            B (2006) Organ-specific analysis anaerobic primary metabolism in <br />    Ann Bot (Lond) 103: 143&#226;&#8364;&#8220;150                                                      rice wheat seedlings: dark ethanol production dominated the <br /> Fukao T, Xu K, Ronald PC, Bailey-Serres J (2006) A variable cluster              shoots. Planta 225: 103&#226;&#8364;&#8220;114 <br />    ethylene response factor-like genes regulates metabolic develop-          Mustroph A, Zanetti ME, Jang CJH, Holtan HE, Repetti PP, Galbraith <br />    mental acclimation responses submergence rice. Plant Cell 18:              DW, Girke T, Bailey-Serres J (2009) Profiling translatomes discrete <br />    2021&#226;&#8364;&#8220;2034                                                                        cell populations resolves cellular priorities hypoxia Arabi- <br /> Gan S, Amasino RM (1997) Making sense senescence. Plant Physiol 113:             dopsis. Proc Natl Acad Sci USA 106: 18843&#226;&#8364;&#8220;18848 <br />    313&#226;&#8364;&#8220;319                                                                       Nakano T, Suzuki K, Fujimura T, Shinshi H (2006) Genome-wide analysis <br /> Gilles-Gonzalez MA (2001) Oxygen signal transduction. IUBMB Life 51:                ERF gene family Arabidopsis rice. Plant Physiol 140: <br />    165&#226;&#8364;&#8220;173                                                                          411&#226;&#8364;&#8220;432 <br /> Gleave AP (1992) A versatile binary vector T-DNA organisa-         Ohme-Takagi M, Shinshi H (1995) Ethylene-inducible DNA binding <br />    tional structure conducive efficient integration cloned DNA           proteins interact ethylene-responsive element. Plant Cell <br />    plant genome. Plant Mol Biol 20: 1203&#226;&#8364;&#8220;1207                                   7: 173&#226;&#8364;&#8220;182 <br /> Hattori Y, Nagai K, Furukawa S, Song XJ, Sakakibara H, Wu J, Matsumoto           Okamuro JK, Caster B, Villarroel R, Van Montagu M, Jofuku KD (1997) <br />    T, Yoshimura A, Kitano H, Matsuoka M, et al (2009) The ethylene                  The AP2 domain APETALA2 defines large new family DNA <br />    response factors SNORKEL1 SNORKEL2 allow rice adapt deep               binding proteins Arabidopsis. Proc Natl Acad Sci USA 94: 7076&#226;&#8364;&#8220;7081 <br />    water. Nature 460: 1026&#226;&#8364;&#8220;1030                                                  Olive MR, Walker JC, Singh K, Dennis ES, Peacock WJ (1991) Functional <br /> Higgins D, Thompson J, Gibson T, Thompson JD, Higgins DG, Gibson                    properties anaerobic responsive element maize Adh1 gene. <br />    TJ (1994) CLUSTAL W: improving sensitivity progressive multi-             Plant Mol Biol 15: 593&#226;&#8364;&#8220;604 <br />    ple sequence alignment sequence weighting, position-specific          Olmedo G, Guo H, Gregory BD, Nourizadeh SD Aguilar-Henonin L, Li <br />    gap penalties weight matrix choice. Nucleic Acids Res 22:                    H, An F, Guzman P, Ecker JR (2006) ETHYLENE-INSENSITIVE5 en- <br />    4673&#226;&#8364;&#8220;4680                                                                        codes 5#/3# exo-ribonuclease required regulation EIN3 <br /> Hoeren FU, Dolferus R, Wu Y, Peacock WJ, Dennis ES (1998) Evidence              targeting F-box proteins EBF1/2. Proc Natl Acad Sci USA 103: 13286&#226;&#8364;&#8220; <br />    role AtMYB2 induction Arabidopsis alcohol dehydro-            13293 <br />    genase gene (ADH1) low oxygen. Genetics 149: 479&#226;&#8364;&#8220;490                       On&#203;&#339;ate-Sa&#194;&#180;nchez L, Anderson JP, Young J, Singh KB (2007) AtERF14, a <br /> Hua J, Meyerowitz EM (1998) Ethylene responses negatively regulated             member ERF family transcription factors, plays non-redundant <br />    receptor gene family Arabidopsis thaliana. Cell 94: 261&#226;&#8364;&#8220;271              role plant defense. Plant Physiol 143: 400&#226;&#8364;&#8220;409 <br /> Ismond KP, Dolferus R, De Pauw M, Dennis ES, Good AG (2003)                      Papdi C, Abraha&#194;&#180;m E, Joseph MP, Popescu C, Koncz C, Szabados L (2008) <br />    Enhanced low oxygen survival Arabidopsis increased met-               Functional identification Arabidopsis stress regulatory genes using <br />    abolic flux fermentative pathway. Plant Physiol 132: 1292&#226;&#8364;&#8220;1302            controlled cDNA overexpression system. Plant Physiol 147: 528&#226;&#8364;&#8220;542 <br /> Jung J, Won SY, Suh SC, Kim H, Wing R, Jeong Y, Hwang I, Kim M (2007)            Peng HP, Chan CS, Shih MC, Yang SF (2001) Signaling events the <br />    The barley ERF-type transcription factor HvRAF confers enhanced                  hypoxic induction alcohol dehydrogenase gene Arabidopsis. Plant <br />    pathogen resistance salt tolerance Arabidopsis. Planta 225:               Physiol 126: 742&#226;&#8364;&#8220;749 <br />    575&#226;&#8364;&#8220;588                                                                       Peng HP, Lin TY, Wang NN, Shih MC (2005) Differential expression of <br /> Jung KH, Seo YS, Walia H, Cao P, Fukao T, Canlas PE, Amonpant F,                    genes encoding 1-aminocyclopropane-1-carboxylate synthase Arabi- <br />    Bailey-Serres J, Ronald PC (2010) The submergence tolerance regulator            dopsis hypoxia. Plant Mol Biol 58: 15&#226;&#8364;&#8220;25 <br />  <br /> Plant Physiol. Vol. 153, 2010                                                                                                                                 771 <br />  Hinz et al. <br />  <br />  <br /> Perata P, Voesenek LA (2007) Submergence tolerance rice requires             Thomashow MF (2001) So what&#226;&#8364;&#8482;s new field plant cold acclima- <br />    Sub1A, ethylene-response-factor-like gene. Trends Plant Sci 12: 43&#226;&#8364;&#8220;46        tion? Lots! Plant Physiol 125: 89&#226;&#8364;&#8220;93 <br /> Pierik R, Van Aken JM, Voesenek LACJ (2009) Is elongation-induced leaf          Valvekens D, Van Montagu M, Van Lijsebettens M (1988) Agrobacterium <br />    emergence beneficial submerged Rumex species? Ann Bot (Lond)                tumefaciens-mediated transformation Arabidopsis root explants by <br />    103: 353&#226;&#8364;&#8220;357                                                                    using kanamycin selection. Proc Natl Acad Sci USA 85: 5536&#226;&#8364;&#8220;5540 <br /> Poyton RO (1999) Models oxygen sensing yeast: implications           Van der Straeten D, Anuntalabhochai S, Van Caeneghem W, Zhou Z, <br />    oxygen-regulated gene expression higher eucaryotes. Respir Physiol           Gielen J, Van Montagu M (1997) Expression members the <br />    115: 119&#226;&#8364;&#8220;133                                                                    ACC synthase gene family deepwater rice submergence, wound- <br /> Quirino BF, Noh YS, Himelblau E, Amasino RM (2000) Molecular aspects               ing, hormonal treatments. Plant Sci 124: 79&#226;&#8364;&#8220;87 <br />    leaf senescence. Trends Plant Sci 5: 278&#226;&#8364;&#8220;282                              Van Dongen JT, Fro&#194;&#168;hlich A, Ram&#196;&#177;&#194;&#180;rez-Aguilar SJ, Schauer N, Fernie AR, <br /> Rahman M, Grover A, Peacock WJ, Dennis ES, Ellis MH (2001) Effects              Erban A, Kopka J, Clark J, Langer A, Geigenberger P (2009) Transcript <br />    manipulation pyruvate decarboxylase alcohol dehydrogenase                metabolite profiling adaptive response mild decreases in <br />    levels submergence tolerance rice. Aust J Plant Physiol 28:           oxygen concentration roots Arabidopsis plants. Ann Bot (Lond) <br />    1231&#226;&#8364;&#8220;1241                                                                       103: 269&#226;&#8364;&#8220;280 <br /> Rashotte AM, Mason MG, Hutchison CE, Ferreira FJ, Schaller GE, Kieber           Vilaine F, Charbonnier C, Casse-Delbart F (1987) Further insight concern- <br />    JJ (2006) A subset Arabidopsis AP2 transcription factors mediates            ing TL region Ri plasmid Agrobacterium rhizogenes strain A4: <br />    cytokinin responses concert two-component pathway. Proc               transfer 1.9 kb fragment sufficient induce transformed roots on <br />    Natl Acad Sci USA 103: 11081&#226;&#8364;&#8220;11085                                              tobacco leaf fragments. Mol Gen Genet 210: 111&#226;&#8364;&#8220;115 <br /> Riechmann JL, Meyerowitz EM (1998) The AP2/EREBP family plant                Visser E, Cohen JD, Barendse G, Blom C, Voesenek L (1996) An ethylene- <br />                                                                                    mediated increase sensitivity auxin induces adventitious root <br />    transcription factors. Biol Chem 379: 633&#226;&#8364;&#8220;646 <br />                                                                                    formation flooded Rumex palustris Sm. Plant Physiol 112: 1687&#226;&#8364;&#8220;1692 <br /> Rivoal J, Thind S, Pradet A, Ricard B (1997) Differential induction of <br />                                                                                 Voesenek LA, Colmer TD, Pierik R, Millenaar FF, Peeters AJ (2006) How <br />    pyruvate decarboxylase subunits transcripts anoxic rice seed- <br />                                                                                    plants cope complete submergence. New Phytol 170: 213&#226;&#8364;&#8220;226 <br />    lings. Plant Physiol 114: 1021&#226;&#8364;&#8220;1029 <br />                                                                                 Voesenek LA, Rijnders JH, Peeters AJM, Van Steeg HM, De Kroon H <br /> Saitou N, Nei M (1987) The neighbor-joining method: new method for <br />                                                                                    (2004) Plant hormones regulate fast shoot elongation water: from <br />    reconstructing phylogenetic trees. Mol Biol Evol 4: 406&#226;&#8364;&#8220;425 <br />                                                                                    genes communities. Ecology 85: 16&#226;&#8364;&#8220;27 <br /> Sakuma Y, Liu Q, Dubouzet JG, Abe H, Shinozaki K, Yamaguch- <br />                                                                                 Wang H, Huang Z, Chen Q, Zhang Z, Zhang H, Wu Y, Huang D, Huang R <br />    Shinozaki K (2002) DNA binding specificity Arabidopsis DREBs, <br />                                                                                    (2004) Ectopic overexpression tomato JERF3 tobacco activates <br />    transcription factors involved dehydration- cold-inducible gene <br />                                                                                    downstream gene expression enhances salt tolerance. Plant Mol <br />    expression. Biochem Biophys Res Commun 290: 998&#226;&#8364;&#8220;1009 <br />                                                                <br /> </body></html>