<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                          JCB: ARTICLE <br />  <br />  <br />  <br />  <br />                               Impairment starvation-induced constitutive <br />                               autophagy Atg7-deficient mice <br />                               Masaaki Komatsu,1,3 Satoshi Waguri,2 Takashi Ueno,3 Junichi Iwata,3 Shigeo Murata,1 Isei Tanida,3 Junji Ezaki,3 <br />                               Noboru Mizushima,4 Yoshinori Ohsumi,5 Yasuo Uchiyama,2 Eiki Kominami,3 Keiji Tanaka,1 Tomoki Chiba1 <br />                               1 <br />                                 Department     Molecular Oncology, Tokyo Metropolitan Institute Medical Science, Bunkyo-ku, Tokyo 113-8613, Japan <br />                               2 <br />                                 Department     Cell Biology Neurosciences, Osaka University Graduate School Medicine, Osaka 565-0871, Japan <br />                               3 <br />                                 Department     Biochemistry, Juntendo University School Medicine, Bunkyo-ku, Tokyo 113-8421, Japan <br />                               4 <br />                                 Department     Bioregulation Metabolism, Tokyo Metropolitan Institute Medical Science, Bunkyo-ku, Tokyo 113-8613, Japan <br />                               5 <br />                                 Department     Cell Biology, National Institute Basic Biology, Okazaki 444-8585, Japan <br />  <br />  <br />  <br />  <br />                               A <br />                                        utophagy membrane-trafficking mechanism                                     autophagosome formation, amino acid supply neo- <br />                                        delivers cytoplasmic constituents ly-                              nates, starvation-induced bulk degradation pro- <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                                        sosome/vacuole bulk protein degradation.                                     teins organelles mice. Furthermore, Atg7 deficiency <br />                               This mechanism involved preservation nutrients                               led multiple cellular abnormalities, appearance <br />                               starvation condition normal turnover                                 concentric membranous structure deformed mito- <br />                               cytoplasmic component. Aberrant autophagy                                     chondria, accumulation ubiquitin-positive aggre- <br />                               reported neurodegenerative disorders, hepatitis,                               gates. Our results indicate important role autophagy <br />                               myopathies. Here, generated conditional knock-                                     starvation response quality control proteins <br />                               mice Atg7, essential gene autophagy                                      organelles quiescent cells. <br />                               yeast. Atg7 essential ATG conjugation systems <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               There major protein degradation pathways eukaryotic                            membrane. The isolation membrane initially enwraps cytoplas- <br />                               cells: proteasome lysosome. The proteasome                               mic constituents organelles, edges fuse <br />                               self-compartmentalized protease complex catalytic activi-                            forming double membrane structure called <br />                               ties inside central proteinaceous chamber (Baumeister et al.,                         autophagosome. Finally, outer membrane autophago- <br />                               1998). It plays crucial roles selective degradation                        fuses lysosome/vacuole sequestered <br />                               short-lived regulatory proteins abnormal proteins                           cytoplasmic components degraded lysosomal/vacuolar <br />                               eliminated cells (Goldberg, 2003). In contrast,                        hydrolases, inner membrane autopha- <br />                               lysosome vesicle contains hydrolases,                            gosomes (Mizushima et al., 2002). <br />                               separated cytosol limiting membrane. In                                      In mammals, autophagy considered necessary the <br />                               lysosomal pathway, degradation plasma membrane pro-                               turnover cellular components, particularly response <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                               teins extracellular proteins mediated endocytosis,                              starvation glucagons (Mortimore Poso, 1987). Yeast de- <br />                               degradation cytoplasmic components achieved                                 ficient autophagy rapidly die nutrition-poor conditions <br />                               pathways: macroautophagy, microautophagy,                                 (Tsukada Ohsumi, 1993), suggesting important roles in <br />                               chaperone-mediated autophagy (Seglen Bohley, 1992;                                preservation <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> supply. Indeed, autophagy necessary <br />                               Dunn, 1994; Klionsky Emr, 2000; Massey et al., 2004).                                 <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span> early neonatal starvation period mice (Kuma <br />                                      Macroautophagy (hereafter referred autophagy)                         et al., 2004). Furthermore, autophagy plays role cellular re- <br />                               main route sequestration cytoplasm lysosome.                          modeling differentiation development multicel- <br />                               The initial step autophagy elongation isolation                              lular organisms, fly, worm, slime mold (Levine and <br />                                                                                                                         Klionsky, 2004), cellular defense invading strepto- <br />                                                                                                                         coccus (Nakagawa et al., 2004). Plants deficient autophagy <br />                               Correspondence Tomoki Chiba: tchiba@rinshoken.or.jp                                    accelerated senescence (Hanaoka et al., 2002). In humans, <br />                               Abbreviations used paper: MEF, mouse embryonic fibroblast; pIpC, poly- <br />                               inosinic acid&#226;&#8364;&#8220;polycytidylic acid; PNS, postnuclear supernatant; SDH, succinate <br />                                                                                                                         autophagy implicated pathological conditions <br />                               dehydrogenase.                                                                            (Shintani Klionsky, 2004); e.g., low levels autophagy <br />                               The online version article includes supplemental material.                        described malignant tumors (Liang et al., 1999). <br />  <br />                               &#194;&#169; The Rockefeller University Press $8.00 <br />                               The Journal Cell Biology, Vol. 169, No. 3, May 9, 2005 425&#226;&#8364;&#8220;434 <br />                               http://www.jcb.org/cgi/doi/10.1083/jcb.200412022 <br />                                                                                                                                                                                        JCB   425 <br />        In contrast, elevated levels autophagosome formation         noticeable pathological phenotypes. Fig. 1 B shows <br />       reported human pathologies neurodegenerative        Southern blots mice indicated genotypes. Immuno- <br />       diseases, myopathies, liver injury (Mizushima et al., 2002;      blot analysis revealed presence Atg7 protein Atg7F/F <br />       Perlmutter, 2002), autophagy implicated execution      mouse embryonic fibroblasts (MEFs; Fig. 1 C), suggesting that <br />       cell death (Xue et al., 1999; Bursch, 2001). However,         Atg7 efficiently expressed Atg7Flox allele. <br />       high level autophagosome formation does necessarily re- <br />       flect enhanced protein degradation formation          The <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> Atg7 -deficient mice <br />       autophagosomes increased Danon cardiomyopathy,           To examine Atg7-deficient phenotype, bred Atg7F/F <br />       characterized defective lysosomal degradation (Nishino et      mice line transgenic mice express Cre recom- <br />       al., 2000; Tanaka et al., 2000). Thus, clear in-   binase control Zp3 promoter oocyte (Le- <br />       creased levels autophagosome formation reflect activa-        wandoski et al., 1997). The heterozygous mice (referred as <br />       tion defective protein degradation.                               Atg7 / ) obtained female Atg7F/ :Zp3 mice. Atg7 /  <br />              Although autophagy extensively studied, little       mice born healthy fertile noticeable <br />       known molecular mechanism recent dis-        pathological phenotypes 1 yr. The Atg7 /  mice, obtained <br />       covery ATG genes budding yeast (Tsukada Ohsumi,            breeding Atg7 /  mice, born Mendelian frequency <br />       1993). Of ATG genes, seven uniquely compose             ( / :  / :  /    21: 38: 19). The results PCR genotyp- <br />       ubiquitin-like conjugation systems: ATG12 ATG8 conju-            ing shown Fig. 2 A. Neither Atg7 mRNA protein was <br />       gation systems (Mizushima et al., 1998; Ichimura et al., 2000;       detected homozygous mice (Fig. 2, B C). We also <br />       Ohsumi, 2001). The ubiquitin-like protein Atg12p covalently          tested loss Atg7 activity examining ATG conjuga- <br />       attaches Atg5p reaction similar ubiquitination. In   tion systems neonate liver. A 56-kD protein, equivalent <br />       process, Atg12p activated E1-like enzyme, Atg7p             Atg5&#226;&#8364;&#8220;Atg12 conjugate, detected Atg5 antibody in <br />       (Tanida et al., 1999), transferred E2-like enzyme,         control Atg7 /  Atg7 /  liver (Fig. 2 C). In contrast, <br />       Atg10p (Shintani et al., 1999), finally conjugates       free Atg5 30 kD, faintly observed Atg7 /  <br />       Atg5p. Atg8p, ubiquitin-like protein, unique        liver, increased Atg7 /  liver (Fig. 2 C). Mammalian Atg8p <br />       ubiquitin-like molecules, conjugates phosphati-       homologue LC3 forms (i.e., LC3-I LC3-II; Kabeya <br />       dyl-ethanolamine (Ichimura et al., 2000). Atg8p activated      et al., 2000). It generally accepted LC3-I free ma- <br />       Atg7p, common Atg12 conjugation system,          ture form LC3-II lipidated form, analogy to <br />       transferred Atg3p, E2-like enzyme (Ichimura et al.,         yeast Atg8p (Ichimura et al., 2000; Kabeya et al., 2000). Both <br />       2000). In mammals, exist Atg8 homologues        forms detected Atg7 /  liver LC3-I <br />       activated Atg7 (Tanida et al., 2001), GATE-16,    form detected increased Atg7 /  liver (Fig. 2 C). <br />       GABARAP, LC3 (Ohsumi, 2001), localize            When crossed GFP-LC3 transgenic mice (Mizushima et <br />       autophagosome (Kabeya et al., 2000, 2004).                           al., 2004), punctate structures representing autophagosomes <br />              Here, generated conditional <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>knockout</span> mice Atg7          detected Atg7 /  Atg7 /  heart (Fig. 2, D <br />       analyzed roles autophagy neonates adult liver.     E). These results indicate Atg7 essential ATG <br />       Autophagosome formation starvation-induced degradation           conjugation systems autophagosome formation mice. <br />       proteins organelles impaired Atg7-deficient mice              Although homozygous mice normal birth (Fig. <br />       adult livers. We important role autophagy      2 F) apparent developmental defect histological <br />       constitutive turnover cytoplasmic components,          analyses (Fig. S1, available http://www.jcb.org/cgi/content/ <br />       loss resulted accumulation abnormal organelles ubiq-       full/jcb.200412022/DC1), mean body weight Atg7 /  <br />       uitinated proteins. Our results suggest autophagy impor-     mice (0.983   0.0763 g,   SD, n   9) significantly lower <br />       tant clearance ubiquitin-positive aggregates.                 wild-type heterozygote mice (1.20   0.116 g, <br />                                                                            n   29; P   0.01), Atg7 /  mice died 1 d birth <br />       Results                                                              (n   19). We considered Atg7 /  mice survive in <br />                                                                            utero virtue nutrients supplied placenta <br />       Generation Atg7 conditional knockout                              survive supply terminates birth, as <br />       mice                                                                 recently reported (Kuma et al., 2004). We tested survival <br />       To investigate physiological roles autophagy mam-          time Atg7 /  neonates starvation condition cae- <br />       mals, generated Atg7 conditional knockout mice. Mouse             sarean delivery. Wild-type heterozygous mice died 21.7   <br />       Atg7 gene encoded 17 exons span 216-kb long ge-           3.3 h birth, Atg7 /  mice died 13   2.0 h <br />       nomic DNA. The active site cysteine residue essential acti-      (P   0.01; Fig. 2 G). To test cause ear- <br />       vation substrates encoded exon 14 targeting     lier death correlates lower <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> supply, measured <br />       vector designed conditionally disrupt exon Cre-        amino acid concentrations plasma 10 h caesarean de- <br />       loxP technology. The targeted exon 14 modified        livery. Essential branched-chain amino acid concentrations <br />       express Atg7 presence neo-resistant gene        sera Atg7 /  mice lower wild-type <br />       cassette intron 14 (Fig. 1 A). Mice homozygous            mice (essential amino acids: 1.536   0.087 vs. 1.291   0.166 <br />       Atg7Flox allele (referred Atg7F/F mice), ex-        mmol/L, P   0.05; branched-chain amino acids: 0.375   0.038 <br />       pected express intact Atg7, born healthy fertile         vs. 0.268   0.015 mmol/L, P   0.01, respectively). The same <br />  <br />  <br /> 426   JCB &#226;&#8364;&#162; VOLUME 169 &#226;&#8364;&#162; NUMBER 3 &#226;&#8364;&#162; 2005 <br />                                                                                Figure 1. Generation Atg7F/F mice. (A) Schematic repre- <br />                                                                               sentation targeting vector targeted allele of <br />                                                                               Atg7 gene. The coding exons numbered accordance with <br />                                                                               initiation site exon 1 depicted black boxes. <br />                                                                               Green red boxes indicate Atg7 cDNA fragment (aa <br />                                                                               1786&#226;&#8364;&#8220;2097) Atg7 cDNA fragment (aa 1669&#226;&#8364;&#8220;1698) <br />                                                                               stop codon, respectively. The open triangles denote loxP <br />                                                                               sequence. A probe Southern blot analysis shown a <br />                                                                               gray ellipse. Arrows indicate positions PCR primers. The <br />                                                                               asterisk denotes essential cysteine residue exon 14. <br />                                                                               EcoRV, EcoRV sites; neo, neomycin-resistant gene cassette; <br />                                                                               DT-A, diphtheria toxin gene. (B) Southern blot analysis ge- <br />                                                                               nomic DNAs extracted mice tails. Wild-type Flox <br />                                                                               alleles detected 17.5- 7.5-kb bands, respectively. <br />                                                                               (C) Immunoblot Atg7 MEFs. The lysates MEFs indi- <br />                                                                               cated genotypes immunoblotted Atg7 actin. <br />  <br />  <br />  <br />  <br /> results obtained using MEF cells Atg7 /  mice        Intraperitoneal injections pIpC resulted effective recombi- <br /> (unpublished data). Together, results indicate Atg7   nation Atg7Flox allele liver spleen (Fig. S2, <br /> crucial recycling amino acids cells survival   available http://www.jcb.org/cgi/content/full/jcb.200412022/ <br /> newborn mice starvation condition.                            DC1; depicted). No Atg7 transcript, protein, activ- <br />                                                                     ity detected, similar Atg7 /  mice (Fig. S2). Next, we <br /> Starvation response adult mice liver                             tested autophagosome formation fasting condition. 1-d <br /> To delete Atg7 gene adult mice, bred Atg7F/F mice     fasting resulted induction typical autophagosomes con- <br /> with Mx1-Cre transgenic mice express Cre recombi-          trol Atg7F/ :Mx1 mice (Fig. 3, A&#226;&#8364;&#8220;D I). In contrast, such <br /> nase response interferon   chemical inducer, poly-     induction autophagosome formation noted liver <br /> inosinic acid&#226;&#8364;&#8220;polycytidylic acid (pIpC). The Mx1-Cre trans-         fasted Atg7F/F:Mx1 mice (Fig. 3, E, F, I). Although <br /> genic mice excise Flox allele completely liver       autophagosome-like structures occasionally observed <br /> spleen partially kidney heart (Kuhn et al., 1995).   fed fasted mutant mice livers (Fig. 3, G H), they <br />  <br />  <br />  <br />                                                                                              Figure 2. The phenotypes Atg7-deficient <br />                                                                                              mice. (A) PCR analysis genomic DNA ex- <br />                                                                                              tracted wild-type, Atg7 / , Atg7 /  <br />                                                                                              mice tail. The amplified fragments derived <br />                                                                                              wild mutant alleles indicated. <br />                                                                                              (B) Expression Atg7 transcript. Atg7 tran- <br />                                                                                              script detected RT-PCR analysis. The <br />                                                                                              region amplified exons 12 and <br />                                                                                              13. G3PDH cDNA amplified inter- <br />                                                                                              nal control. (C) ATG conjugation systems in <br />                                                                                              Atg7 /  mice liver. The liver homogenate was <br />                                                                                              centrifuged 800 g 10 min post- <br />                                                                                              nuclear supernatant (PNS) immunoblotted <br />                                                                                              antibodies Atg7, Atg5, LC3, and <br />                                                                                              actin loading control. The panel <br />                                                                                              Atg5 blotting long exposure top <br />                                                                                              panel detect free Atg5. Data shown are <br />                                                                                              representative separate experiments. <br />                                                                                              (D E) Deficiency autophagosome for- <br />                                                                                              mation Atg7 /  heart. Atg7 /  (D) and <br />                                                                                              Atg7 /  (E) mice expressing GFP-LC3 were <br />                                                                                              obtained caesarean delivery analyzed <br />                                                                                              fluorescence microscopy. Representative <br />                                                                                              results obtained neonatal heart at <br />                                                                                              3 h caesarean delivery. (F) Morphology <br />                                                                                              Atg7 /  Atg7 /  mice. (G) Kaplan- <br />                                                                                              Meier curves survival newborn mice. <br />                                                                                              Control Atg7 /  mice delivered by <br />                                                                                              caesarean section, survival fol- <br />                                                                                              lowed 26 h. <br />  <br />  <br />                                                              CONDITIONAL DISRUPTION OF ATG7 IN MICE &#226;&#8364;&#162; KOMATSU ET AL.                            427 <br />        Figure 3. Impaired autophagosome formation <br />       Atg7-deficient liver. (A&#226;&#8364;&#8220;H) Electron micro- <br />       graphs liver Atg7F/ :Mx1 (A&#226;&#8364;&#8220;D) and <br />       Atg7F/F:Mx1 (E&#226;&#8364;&#8220;H) mice fed ad libitum (A and <br />       E) fasted 1 d (B F). (C D) Early <br />       stages autophagic vacuoles observed B <br />       highlighted. (E F) Autophagosome <br />       induced mutant hepatocytes upon <br />       fasting. Insets higher magnification <br />       views glycogen granules. (G H) Occa- <br />       sionally observed autophagosome-like struc- <br />       tures mutant hepatocytes. Bars: (A, B, E, <br />       F) 5  m; (C, D, G, H) 0.5  m. (I) <br />       Number autophagosomes hepatocyte <br />       (n   20) genotype counted and <br />       averages shown. (J&#226;&#8364;&#8220;M) Immunofluo- <br />       rescent analysis LC3 primary cultured <br />       hepatocytes. Hepatocytes isolated <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />       Atg7F/ :Mx1 (J K) Atg7F/F:Mx1 mice <br />       (L M) cultured Williams&#226;&#8364;&#8482; E (J L) <br />       Hanks&#226;&#8364;&#8482; solution (K M). Inset highlights <br />       cup-like structure LC3 observed K. <br />       (N) Immunoblot analysis Atg8 homologues <br />       liver. Atg7F/ :Mx1 (lanes 1 2) and <br />       Atg7F/F:Mx1 mice (lanes 3 4) fed <br />       ad libitum (lanes 1 3) fasted 1 d <br />       (lanes 2 4), PNS fractions of <br />       liver analyzed immunoblotting with <br />       anti-LC3, GABARAP, GATE-16, actin anti- <br />       bodies. Asterisk denotes nonspecific band. <br />       Data shown representative sepa- <br />       rate experiments. <br />  <br />  <br />  <br />  <br />       tended smaller observed fasted control               Next, examined fasting response LC3 other <br />       liver hardly contained large cytoplasmic organelles (com-      homologues, GABARAP GATE-16, immunoblotting <br />       pare Fig. 3, C D). The number autophagosomes           (Fig. 3 N). LC3 known up-regulated recruited to <br />       hepatocyte counted mean values shown           autophagosome starvation degraded lyso- <br />       Fig. 3 I. The mutant hepatocytes lacked typical glycogen        somes (Kabeya et al., 2000). Fasting slightly increased the <br />       area, contrast fed hepatocytes (Fig. 3, A E); how-   modification LC3 heterozygous liver. In mutant liver, <br />       ever, well-developed glycogen granules (  granules) ob-       modification LC3 noted LC3-I increased re- <br />       served numerous smooth endoplasmic reticula (Fig.          sponse fasting. These results suggest LC3 up-regulated, <br />       3 E, inset). Immunofluorescent analysis revealed          modification degradation impaired mutant <br />       presence cup-shaped ringlike structures repre-         mice. The level GABARAP did change fasting in <br />       senting autophagosomes control hepatocytes (Fig. 3, J       mutant liver, decreased heterozygous <br />       K). Although LC3-positive dots observed        liver, suggesting GABARAP up-regulated its <br />       mutant hepatocytes, induced response       degradation modification impaired mutant liver. Al- <br />       starvation did form cup-shaped ringlike struc-         GATE-16 hardly detected heterozygous liver <br />       tures (Fig. 3, L M).                                           fed fasting conditions, clearly detected <br />  <br />  <br /> 428   JCB &#226;&#8364;&#162; VOLUME 169 &#226;&#8364;&#162; NUMBER 3 &#226;&#8364;&#162; 2005 <br />  and increased fasting mutant liver. These results <br /> suggest GATE-16 constitutively degraded at <br /> fed condition heterozygous mice up-regulated re- <br /> sponse fasting defective Atg7. The levels LC3 <br /> homologues elevated fed condition mutant <br /> liver, suggesting marked stabilization autophagy-defi- <br /> cient condition. However, possibility transcrip- <br /> tions up-regulated basal level Atg7 deficiency can- <br /> not excluded. We sought determine localizations in <br /> the cells. However, antibodies molecules not <br /> applicable immunofluorescent analyses, localiza- <br /> tions remain clarified. In conclusion, Atg8 homo- <br /> logues respond fasting, different manner, and <br /> their levels affected absence Atg7. <br />  <br /> Atg7 indispensable fasting-induced <br /> degradation cytosolic proteins and <br /> organelles mouse liver <br /> Given autophagosome formation impaired Atg7-defi- <br /> cient liver, examined effects bulk degradation <br /> of proteins organelles fasting condition. After 1-d <br /> fasting control Atg7F/ :Mx1 mutant Atg7F/F:Mx1 mice, <br /> the liver dissected total protein per <br /> whole liver measured. The total liver proteins <br /> decreased  66% 1-d fasting control liver (Fig. 4 A). <br /> In contrast, fasting did significantly decrease of <br /> total liver proteins mutant liver. Moreover, of <br />                                                                      Figure 4. Fasting response Atg7-deficient liver. (A B) Livers from <br /> total proteins mutant liver 1.5-fold control.     Atg7F/ :Mx1 Atg7F/F:Mx1 mice fed ad libitum (Fed) fasted 1 d <br /> These results indicate decrease total proteins de-    (Fast) 20 d pIpC injection dissected, total <br />                                                                      protein (A) SDH activity (B) liver measured. Data are <br /> pendent Atg7 autophagosome formation. <br />                                                                      mean   SD values mice group; *, P   0.01. (C) Cytochrome <br />        We examined fasting causes deg-       c levels cytosolic mitochondria/lysosomal fractions liver <br /> radation cellular organelles mitochondria liv-     20 d injection. Equal PNS fractions centrifuged at <br />                                                                      8,000 g 10 min pellets used mitochondrial/lyso- <br /> ers mice. To quantify the mitochondria, we <br />                                                                      somal fraction (ML). The supernatants centrifuged 100,000 g <br /> first measured activity mitochondrial enzyme succinate        1 h supernatant used cytosolic fraction (C). Actin <br /> dehydrogenase (SDH) total liver extracts. In control liv-     blotted control. Data shown representative separate ex- <br />                                                                      periments. (D) Turnover long-lived protein. Hepatocytes Atg7F/ : <br /> ers, fasting associated significant decrease SDH <br />                                                                      Mx1 Atg7F/F:Mx1 mice isolated labeled [14C]leucine <br /> activity, reduction proportional decrease       24 h, degradation long-lived protein deprived nonde- <br /> in total protein (Fig. 4 B). In contrast, fasting      prived condition measured. Monomethylamine (MA) and/or E64d <br />                                                                      pepstatin (E/P) epoxomicin (epoxo) added indicated. Data <br /> was associated change SDH activity mu- <br />                                                                      mean   SD triplicate experiments. <br /> tant livers. Similar total protein, basal SDH activity in <br /> mutant liver significantly higher control. The effect <br /> of fasting the mitochondria assessed       beled [14C]leucine 24 h chased 2 h, release <br /> by immunoblot analysis mitochondrial protein cytochrome c         TCA-soluble [14C]leucine measured 4 h. In control <br /> (Fig. 4 C). When equal amounts proteins loaded,          hepatocytes, <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation significantly induced protein <br /> level cytochrome c equivalent genotypes         degradation, degradation suppressed addi- <br /> either fed fasting conditions, suggesting ratio mi-   tion lysosomal inhibitors monomethylamine and <br /> tochondria versus total protein altered fasting    E64d pepstatin (Fig. 4 D). The induced degradation was <br /> genotypes. Considering total protein amounts de-            observed presence proteasome inhibitor epoxomi- <br /> creased fasting control liver (Fig. 4 A), results    cin, suggesting protein degradation mediated the <br /> suggest mitochondria cytoplasmic proteins pro-      lysosomal pathway proteasome (Fig. 4 D). In <br /> portionally degraded fasting heterozygous mice. How-         mutant hepatocytes, degradation long-lived protein was <br /> ever, degradation impaired Atg7-deficient liver           induced <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation (Fig. 4 D), indicating that <br /> because levels proteins mitochondria un-         autophagy main route lysosomal degradation under <br /> changed kept higher level.                                  starvation condition. Consistent results, amino acid <br />        Next, investigated effect autophagy deficiency      concentrations starved mutant hepatocytes lower than <br /> on protein turnover. To quantify turnover long-lived pro-     control hepatocytes (unpublished data). Intriguingly, al- <br /> tein, control mutant hepatocytes la-         lysosomal inhibitors inhibited protein degradation even <br />  <br />  <br />                                                               CONDITIONAL DISRUPTION OF ATG7 IN MICE &#226;&#8364;&#162; KOMATSU ET AL.                              429 <br />        Figure 5. Atg7 deficiency liver causes hepatomegaly <br />       hepatic cell swelling. (A) The gross anatomical views of <br />       representative mice indicated day pIpC injection. <br />       (B&#226;&#8364;&#8220;E) Histology representative livers Atg7 deficiency. <br />       Hematoxylin eosin staining Atg7F/ :Mx1 (B C) <br />       Atg7F/F:Mx1 (D E) liver 90 d pIpC injection. <br />  <br />  <br />  <br />  <br />       nondeprived condition control hepatocytes, inhi-    sera (Fig. S4, available http://www.jcb.org/cgi/content/full/ <br />       bition significant mutant hepatocytes (Fig. 4 D),   jcb.200412022/DC1). <br />       indicating significant amounts proteins constitu-             Although hepatocytes alive mutant <br />       tively degraded lysosome autophagic pathway. To-        liver, ultrastructural analysis revealed appearance aber- <br />       gether, results suggest autophagy plays significant   rant concentric membranous structures (Fig. 6, A B), <br />       role turnover long-lived protein.                            observed early 20 d pIpC-injected <br />                                                                          liver (not depicted). These structures surrounded various cyto- <br />       Loss Atg7 liver leads                                 plasmic constituents mitochondria, lipid droplets, and <br />       hepatomegaly accumulation                                   vesicular structures (Fig. 6 A). Their membranous elements <br />       abnormal organelles hepatic cells                               continuous rough ER (Fig. 6 B, arrowheads), and <br />       We chased phenotypes mutant mice      corresponding structures positive calreticulin, an <br />       90 d pIpC injection. Gross anatomy revealed severe en-       ER protein marker (not depicted), indicating struc- <br />       largement liver, filling abdominal cavity    tures originated rough ER. Accumulation peroxi- <br />       (Fig. 5 A). Other major organs normal histologically          somes (Fig. 6 C) deformed mitochondria (Fig. 6, C D) <br />       (Fig. S3, available http://www.jcb.org/cgi/content/full/        observed mutant liver. These results suggest the <br />       jcb.200412022/DC1). The mean liver weights control          important role autophagy turnover organelles, its <br />       mutant mice 90 d pIpC injection 1.39   0.24      defect results accumulation abnormal organelles. <br />       6.10   2.06 g, respectively (n   5 each). Histological analy- <br />       sis revealed disorganized hepatic lobules cell swelling     Formation ubiquitin-positive inclusions <br />       mutant liver (Fig. 5, D E). No hepatocellular prolifer-    Atg7 -deficient hepatocytes <br />       ation regeneration detected (unpublished data). Vacu-       Autophagy implicated organelle turnover <br />       olated hepatic cells occasionally observed          elimination protein aggregates (Kopito, 2000). <br />       associated hepatic cell death, consis-          Protein aggregates ubiquitinated. In step, we <br />       tent leakage alkaline phosphatase, aspartate ami-      immunostained liver ubiquitin antibody examine <br />       notransferase, alanine aminotransferase mutant mice     presence aggregates. Several ubiquitin-positive par- <br />  <br />  <br /> 430   JCB &#226;&#8364;&#162; VOLUME 169 &#226;&#8364;&#162; NUMBER 3 &#226;&#8364;&#162; 2005 <br />                                                                                         Figure 6. Electron micrographs hepatic cells with <br />                                                                                        Atg7 deficiency. (A B) Note presence of <br />                                                                                        concentric membranous structures mutant cells <br />                                                                                        (arrowheads). Higher magnification view (B) shows <br />                                                                                        membranous elements continuous the <br />                                                                                        rough ER (arrowheads). (C D) The mutant cells <br />                                                                                        contained high number peroxisomes (arrows) <br />                                                                                        deformed mitochondria (asterisks). Insets show <br />                                                                                        higher magnification views. Nuc, nucleus. Bars: (A) <br />                                                                                        5  m; (B) 0.5  m; (C D) 1  m. <br />  <br />  <br />  <br />  <br /> ticles various sizes detected Atg7F/F:Mx1      mation (Fig. 2, Fig. 3, Fig. S2). Immunofluorescent analy- <br /> in Atg7F/ :Mx1 hepatic cells 10 90 d pIpC in-       ses revealed LC3-positive dots appeared did form <br /> jection (Fig. 7, A B; Fig. S5, available http://           cup-shaped ringlike structures Atg7F/F:Mx1 livers (Fig. 3). <br /> www.jcb.org/cgi/content/full/jcb.200412022/DC1). The immu-            The LC3-I form usually present S100 fraction the <br /> noblots liver lysates revealed accumulation high-       LC3-II form P100 fraction (Kabeya et al., 2000). In the <br /> molecular mass polyubiquitinated proteins mutant liver         mutant liver, LC3-I present S100 P100 frac- <br /> (Fig. 7 G Fig. S5), suggesting ubiquitin particles       tions (unpublished data), suggesting LC3-positive dots <br /> are aggregates polyubiquitinated proteins. To deter-       mutant hepatocytes LC3-I form. These <br /> mine localization ubiquitin-positive dots, analysis im-     results suggest LC3 recruited dot structures <br /> munoelectron micrographs performed. Numerous particles            independent modification (Fig. 3). In mammals, LC3 <br /> of colloidal gold, indicative ubiquitin, detected lipid    homologues, GABARAP GATE-16, <br /> dropletlike structures, membranous structures, amorphous          share common biochemical characteristics (Tanida et <br /> substances cytoplasm (Fig. 7, C&#226;&#8364;&#8220;F). Such signals          al., 2001) localize autophagosome response fasting <br /> not observed wild-type liver (unpublished data).               (Kabeya et al., 2004). Indeed, modification levels of <br />        The accumulation ubiquitin-positive inclusions       molecules fasting condition affected the <br /> cytoplasm prompted examine effect autophagy de-          mutant liver (Fig. 3 N). However, LC3 homologues have <br /> ficiency proteasome function. Immunoblots proteasome            identified different biological pathway have <br /> subunits (p112/Rpn2, Mss1/Rpt1,  5) showed             diverse functions (Ohsumi, 2001). Thus, functions <br /> relative amounts affected mutant liver (Fig. 7 G).    localizations affected Atg7-deficient cells remains <br /> Furthermore, activities proteasome, measured            clarified. <br /> Suc-LLVY-MCA substrate, comparable                      Although Atg7 /  mice born Mendelian ratio, and <br /> wild-type mutant livers (unpublished data). These results         major organs normal histologically (Fig. S1), <br /> indicate accumulation ubiquitin-positive aggregates         reduced body weight died 1 d birth. <br /> autophagy-deficient hepatocytes despite apparently normal         Atg7 /  mice lower amino acid level died earlier com- <br /> proteasome function.                                                  pared wild type nonsuckling condition caesar- <br />                                                                       ean delivery (Fig. 2 G), suggesting Atg7 important for <br /> Discussion                                                            survival early neonatal starvation period, similar to <br />                                                                       recently reported Atg5 /  mice phenotypes (Kuma et al., 2004). <br /> Autophagy bulk protein degradation pathway, con-        However, suckling Atg7 /  mice died 1 d <br /> served eukaryotes, essential survival unicellular or-   birth (unpublished data), cause death be <br /> ganisms nutrient-poor condition cellular remodel-       low level amino acids. The reason re- <br /> ing multicellular organisms (Mizushima et al., 2002; Levine        duced body size unclear related placental <br /> and Klionsky, 2004). In present study, generated condi-        function inefficient reutilization nutrients during <br /> tional knockout mice Atg7 gene, essential gene         embryogenesis. It note lower level autophagy oc- <br /> for autophagy budding yeast, analyzed roles mice.       curs embryogenesis (Mizushima et al., 2004) when <br />       In mammals, Atg7 essential ATG12 con-            nutrients supplied placenta. Furthermore, Atg7 <br /> jugation, LC3 modification systems, autophagosome for-            null mice possess ubiquitin-positive inclusions some <br />  <br />  <br />                                                                CONDITIONAL DISRUPTION OF ATG7 IN MICE &#226;&#8364;&#162; KOMATSU ET AL.                        431 <br />        organs time birth (unpublished data). This phenotype <br />       related earlier death mutant. Further analysis <br />       Atg7 /  mice required unravel roles autophagy, <br />       analysis currently way breeding Atg7F/F <br />       mice Cre-transgenic mice. <br />              Starvation-induced autophagosomes appeared sequester <br />       cytoplasm randomly (Fig. 3). Consistent notion, the <br />       mitochondria decreased proportion reduction <br />       total protein (Fig. 4, A&#226;&#8364;&#8220;C). These results <br />       suggest mitochondria degraded nonselectively under <br />       fasting condition. In Atg7-deficient liver, autophagosome <br />       formation noted degradation proteins or- <br />       ganelles fasting condition largely impaired. These re- <br />       sults suggest rapid reduction proteins organelles <br />       fasting dependent Atg7 autophagosome formation. <br />              Although autophagy induced starvation, this <br />       pathway place feeding condition basal level. <br />       This constitutive pathway important turnover or- <br />       ganelles cytoplasmic proteins. Indeed, degradation of <br />       long-lived protein inhibited mutant hepatocytes irrespec- <br />       tive nutrient deprivation (Fig. 4 D), multiple abnormali- <br />       ties organelles (e.g., presence concentric membranous <br />       structure accumulation deformed mitochondria) ob- <br />       served Atg7-deficient hepatocytes (Fig. 6). Unexpectedly, the <br />       morphologically abnormal mitochondria appear retain their <br />       function, judged normal membrane potentials the <br />       absence cytochrome c leakage cytosol (unpublished <br />       data). In contrast starvation-induced autophagy, or <br />       constitutive autophagy eliminates abnormal excess or- <br />       ganelles degree selectivity remains clarified.        Figure 7. Accumulation ubiquitin-positive aggregates Atg7-deficient <br />              Beclin 1, human homologue ATG6/VPS30 essential          liver. (A B) Immunofluorescent detection ubiquitin Atg7F/ : <br />                                                                           Mx1 (A) Atg7F/F:Mx1 (B) liver. (C&#226;&#8364;&#8220;F) Immunoelectron micrographs of <br />       autophagy yeast, recently identified tumor sup-     ubiquitin representative mutant liver. The high-magnification view <br />       pressor gene, autophagy implicated regula-      shows ubiquitin particles near lipid dropletlike structure (D&#226;&#8364;&#8220;F). Bars, <br />       tion cellular proliferation (Liang et al., 1999). Indeed, het-   0.5  m. (G) Immunoblot analysis liver. PNS fractions liver at <br />                                                                           90 d injection immunoblotted indicated antibodies. <br />       erozygous disruption mouse Beclin 1 led enhanced              Data shown representative separate experiments. <br />       tumorigenesis (Qu et al., 2003; Yue et al., 2003). Atg7 defi- <br />       ciency led hepatomegaly (Fig. 5 A), suggesting cell pro- <br />       liferation malignant transformation induced      tinated aggregates difficult unfold, proteasomes <br />       Atg7-deficient cells. However, tumorigenesis en-        need unfold substrate degradation (Baumeister <br />       hanced cell proliferation detected tested BrdU incor-     et al., 1998), likely elimination ubiquitin-positive <br />       poration 90 d pIpC injection mutant liver com-      aggregates cells largely dependent autophagic <br />       pared control mice (unpublished data). The hepatomegaly        process. Protein ubiquitination occur protein ag- <br />       observed mutant mice likely increased cellu-      gregation. In case, propose possibility pro- <br />       lar volume cell number, supported      tein ubiquitination serve signal autophagic pro- <br />       swollen appearance hepatocytes (Fig. 5, D E).            cess addition proteasomes pathway. In context, it <br />              In Atg7-deficient liver, detected numerous ubiquitin-     worth noting sperm mitochondria known ubiq- <br />       positive particles indicative protein aggregates (Fig. 7     uitinated degradation fertilization (Sutovsky et <br />       Fig. S5). It reported proteasome inhibition leads     al., 1999). It established ubiquitin regulates <br />       aggregate formation. Conversely, formation protein        proteasomal degradation, lysosomal degrada- <br />       aggregates inhibits proteasome (Bence et al., 2001), result-    tion. Thus, conceivable ubiquitin regulate <br />       ing malignant cycle aggregate formation protea-         autophagic pathway. <br />       inhibition. In mutant liver, failure proteasome            A growing number disease-associated proteins have <br />       postulated; however, impairment proteasome func-          accumulate aggresome, including huntingtin, <br />       tion, terms expression peptidase activities,       parkin,  -synuclein, peripheral myelin protein 22 (Notterpek <br />       noted (Fig. 7 G depicted). Our results suggest     et al., 1999; Ciechanover Brundin, 2003). The aggregation <br />       ubiquitinated proteins eventually aggregate pres-       proteins thought involved pathogenesis of <br />       ence functional proteasomes. Considering ubiqui-       Huntington&#226;&#8364;&#8482;s disease, Parkinson&#226;&#8364;&#8482;s disease, peripheral neu- <br />  <br />  <br /> 432   JCB &#226;&#8364;&#162; VOLUME 169 &#226;&#8364;&#162; NUMBER 3 &#226;&#8364;&#162; 2005 <br />  ropathies, respectively. Enhanced autophagosome formation                 presence absence protease inhibitors (5 mM monomethylamine, 10 <br />                                                                               g/ml E64d pepstatin, 5  M epoxomicin). After 4 h, aliquots of <br /> prevalent diseases (Mizushima et al., 2002),                medium taken one-tenth volume 100% trichloroacetic <br /> and autophagy considered caspase-indepen-                 acid added aliquot. The mixtures centrifuged 12,000 g <br /> dent cell death pathway (Xue et al., 1999; Bursch, 2001). Our                5 min, acid-soluble radioactivity determined using liquid <br />                                                                              scintillation counter. At end experiment, cultures were <br /> Atg7 mutant mice useful examining role                  washed twice PBS, 1 ml cold trichloroacetic acid added <br /> autophagy cell death pathway cellular defense                 fix cell proteins. The fixed cell monolayers washed trichlo- <br /> mechanism pathogenesis diseases.                             roacetic acid dissolved 1 ml 1 N NaOH 37 <br /> C. Radioactivity <br />                                                                              aliquot 1 N NaOH determined liquid scintillation counting. <br />                                                                              The percentage protein degradation calculated according pub- <br /> Materials methods                                                        lished procedures (Gronostajski Pardee, 1984). <br />  <br /> Generation Atg7F/F mice                                                   Histological examination <br /> The targeting vector constructed insertion loxP sequences        Tissues dissected, fixed 4% PFA, paraffin embedded, sec- <br /> within introns 13 14 Atg7 gene. Exon 14 fused cDNA           tioned. Sections stained Meyer&#226;&#8364;&#8482;s hematoxylin eosin. For im- <br /> fragment encoded exons 15, 16, 17 (aa 1786&#226;&#8364;&#8220;2097) polyA            munohistochemical analysis, sections blocked 5% normal goat se- <br /> signal sequence added stop codon. Neo resistant gene cas-      rum PBS containing 0.2% Triton X-100, incubated with <br /> sette (mc1-neo-pA) ligated polyA signal sequence fol-         antiubiquitin antibody (1B3; MBL International Corporation) Alexa <br /> lowed second loxP sequence, splicing acceptor site, exon 14       488&#226;&#8364;&#8220;labeled secondary antibody (Molecular Probes). Apoptotic cells <br /> with stop codon preceding active site. We electroporated targeting   detected TUNEL assay using Apoptag kit (Intergen Company) as <br /> vector mouse TT2 ES cells, selected 200  g/ml G418 (GIBCO          described previously (Tateishi et al., 2001). For GFP-LC3 observations, tis- <br /> BRL), screened homologous recombinants PCR South-        sues fixed 4% PFA, cryosections imaged a <br /> ern blot analyses. PCR primers follows: 5	-TGGCTGCTACTTCTG-          conventional fluorescence microscope. For LC3 staining, hepatocytes <br /> CAATGATGT-3	, 5	-GAAGGGACTGGCTGCTATTGGGCGAAGTGC-3	,                          fixed stained anti-LC3 antibody described previously <br /> and 5	-TTAGCACAGGGAACAGCGCTCATGG-3	. Southern blot analysis                  (Kabeya et al., 2000). All fluorescence images obtained using a <br /> was performed digestion genomic DNA EcoRV hybridiza-          fluorescence microscope (model Q550FV; Leica) equipped cooled <br /> tion probe shown Fig. 1 A. Genotyping mice PCR         charge-coupled device camera (model CTR MIC; Leica). Pictures were <br /> performed using following primers: 5	-TGGCTGCTACTTCTG-               taken using Qfluoro software (Leica). <br /> CAATGATGT-3	 5	-CAGGACAGAGACCATCAGCTCCAC-3	. Prog- <br /> eny containing Atg7Flox allele bred Zp3-Cre Mx1-Cre        EM immunoelectron microscopy <br /> transgenic mice produce Atg7 /  Atg7F/F:Mx1 mice, respectively.       Livers fixed cardiac perfusion using 0.1 M phosphate buffer con- <br /> With regard Atg7F/F:Mx1 mice, Cre expression liver induced     taining 2% PFA 2% glutaraldehyde conventional EM. They were <br /> by i.p. injection pIpC (Sigma-Aldrich). 300  l pIpC solution (1 mg/ml     post-fixed 1% OsO4, embedded Epon812, sectioned. Immu- <br /> in water) injected times 48-h intervals. Mice housed    noelectron microscopy performed cryothin sections described <br /> specific pathogen-free facilities, experimental protocol ap-     previously (Waguri et al., 1995). In brief, livers frozen phosphate <br /> proved Ethics Review Committee Animal Experimentation      buffer 2.3 M sucrose 20% polyvinyl pyrrhoridon. Ultrathin sec- <br /> Tokyo Metropolitan Institute Medical Science.                             tions mounted Formvar carbon-coated nickel grids, blocked with <br />                                                                              1% BSA PBS, incubated antiubiquitin antibody (1B3) col- <br /> RT-PCR analysis                                                              loidal gold conjugated secondary antibody. <br /> cDNA synthesized 5  g DNase I&#226;&#8364;&#8220;treated total RNA using the <br /> SuperScript First-Strand Synthesis System (GIBCO BRL) oligo (dT)12-18    Other procedures <br /> primers. Specific primers gene follows: 5	-ATGCCAG-         MEFs prepared described previously (Murata et al., 2001). Pri- <br /> GACACCCTGTGAACTTC-3	 5	-ACATCATTGCAGAAGTAGCAG-                           mary hepatocytes prepared described previously (Ueno et al., <br /> CCA-3	 Atg7, 5	-GAGCTGAACGGGAAGCTCAC-3	 5	-ACC-                  1990). Cell starvation conducted incubating cells Hanks&#226;&#8364;&#8482; <br /> ACCCTGTTGCTGTAGC-3	 G3PDH.                                               balanced solution separate washes. The SDH activity as- <br />                                                                              sayed described previously (Ueno et al., 1990). <br /> Immunoblot analysis <br /> The fractions immunoblotted described previously (Komatsu et al.,    On line supplemental material <br /> 2001). The antibodies Atg7 (Tanida et al., 1999) Atg5 (Mizushima     Fig. S1 shows histological analyses tissues Atg7 /  and <br /> et al., 2001) described previously. The antibodies ubiquitin        Atg7 /  mice 1 d birth. Fig. S2 shows loss Atg7 protein and <br /> (DakoCytomation) actin (MAB1501R; Chemicon International, Inc.)     activity Atg7F/F:Mx1 mouse liver. Fig. S3 shows histological analy- <br /> purchased. The antibodies LC3, GABARAP, GATE-16             ses tissues Atg7F/ :Mx1 Atg7F/F:Mx1 mice. Fig. S4 shows the <br /> raised rabbits using specific peptides antigens. The antibod-    cell death autophagy-deficient liver. Fig. S5 shows accumulation of <br /> ies p112, Mss1,  5 provided K.B. Hendil (August          ubiquitin-positive inclusions early stage autophagy deficiency. Fur- <br /> Krogh Institute, University Copenhagen, Copenhagen, Denmark).             ther comments data legends. Online supple- <br />                                                                              mental material available http://www.jcb.org/cgi/content/full/ <br /> Caesarean delivery measurement amino acids                            jcb.200412022/DC1. <br /> Newborns delivered caesarean section 19.0 d postcoitus <br />                                                                              We thank T. Kaneko, T. Kouno, K. Tatsumi technical assistance. <br /> and placed humidified, thermostat-controlled chamber (30 <br /> C). <br />                                                                              We thank F. Kaji help EM study; A. Kuma technical guid- <br /> Plasma fixed 3% sulphosalicylic acid. Amino acids super- <br />                                                                              ance caesarean delivery; K. Tateishi H. Uozaki discussion liver <br /> natant plasma samples measured amino acid analyzer <br />                                                                              pathology; T. Fujimura K. Murayama amino acid measurements. <br /> (L8500A; Hitachi). <br />                                                                                     This work supported Grants-in-Aid Ministry of <br /> Protein degradation assay                                                    Education, Culture, Sports, Science Technology Japan. <br /> The assay performed essentially described previously (Grono- <br /> stajski Pardee, 1984). In brief, hepatocytes plated 5   104      Submitted: 3 December 2004 <br /> cells/well collagen-coated 24-well plates cultured Williams&#226;&#8364;&#8482; E     Accepted: 22 March 2005 <br /> medium 10% FCS (Williams&#226;&#8364;&#8482; E/10% FCS) 24 h. Cells incu- <br /> bated Williams&#226;&#8364;&#8482; E/10% FCS containing 0.5  Ci/ml [14C]leucine for <br /> 24 h <span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span> long-lived proteins. Cells washed Williams&#226;&#8364;&#8482;          References <br /> E/10% FCS containing 2 mM unlabeled leucine incubated the <br /> medium 2 h allow degradation short-lived proteins minimize     Baumeister, W., J. Walz, F. Zuhl, E. Seemuller. 1998. The proteasome: par- <br /> the incorporation labeled leucine, released proteolysis             adigm self-compartmentalizing protease. Cell. 92:367&#226;&#8364;&#8220;380. <br /> into protein. The cells washed PBS incubated 37 <br /> C      Bence, N.F., R.M. Sampat, R.R. Kopito. 2001. Impairment ubiquitin- <br /> with Krebs-Ringer bicarbonate medium Williams&#226;&#8364;&#8482; E/10% FCS                 proteasome protein aggregation. Science. 292:1552&#226;&#8364;&#8220;1555. <br />  <br />  <br />                                                                      CONDITIONAL DISRUPTION OF ATG7 IN MICE &#226;&#8364;&#162; KOMATSU ET AL.                                  433 <br />        Bursch, W. 2001. The autophagosomal-lysosomal compartment programmed               Notterpek, L., M.C. Ryan, A.R. Tobler, E.M. Shooter. 1999. PMP22 accumu- <br />              cell death. Cell Death Differ. 8:569&#226;&#8364;&#8220;581.                                             lation aggresomes: implications CMT1A pathology. Neurobiol. Dis. <br />       Ciechanover, A., P. Brundin. 2003. The ubiquitin proteasome                    6:450&#226;&#8364;&#8220;460. <br />              neurodegenerative diseases: chicken, egg.          Ohsumi, Y. 2001. Molecular dissection autophagy: ubiquitin-like systems. <br />              Neuron. 40:427&#226;&#8364;&#8220;446.                                                                   Nat. Rev. Mol. Cell Biol. 2:211&#226;&#8364;&#8220;216. <br />       Dunn, W.A., Jr. 1994. Autophagy related mechanisms lysosome-medi-              Perlmutter, D.H. 2002. Liver injury alpha1-antitrypsin deficiency: aggre- <br />              ated protein degradation. Trends Cell Biol. 4:139&#226;&#8364;&#8220;143.                                gated protein induces mitochondrial injury. J. Clin. Invest. 110:1579&#226;&#8364;&#8220;1583. <br />       Goldberg, A.L. 2003. Protein degradation protection misfolded          Qu, X., J. Yu, G. Bhagat, N. Furuya, H. Hibshoosh, A. Troxel, J. Rosen, E.L. Es- <br />              damaged proteins. Nature. 426:895&#226;&#8364;&#8220;899.                                                kelinen, N. Mizushima, Y. Ohsumi, et al. 2003. Promotion tumorigen- <br />       Gronostajski, R.M., A.B. Pardee. 1984. Protein degradation 3T3 cells              esis heterozygous disruption beclin 1 autophagy gene. J. Clin. <br />              tumorigenic transformed 3T3 cells. J. Cell. Physiol. 119:127&#226;&#8364;&#8220;132.                     Invest. 112:1809&#226;&#8364;&#8220;1820. <br />       Hanaoka, H., T. Noda, Y. Shirano, T. Kato, H. Hayashi, D. Shibata, S. Tabata,         Seglen, P.O., P. Bohley. 1992. Autophagy vacuolar protein degra- <br />              Y. Ohsumi. 2002. Leaf senescence starvation-induced chlorosis                 dation mechanisms. Experientia. 48:158&#226;&#8364;&#8220;172. <br />              accelerated disruption Arabidopsis autophagy gene.            Shintani, T., D.J. Klionsky. 2004. Autophagy health disease: double- <br />              Plant Physiol. 129:1181&#226;&#8364;&#8220;1193.                                                         edged sword. Science. 306:990&#226;&#8364;&#8220;995. <br />       Ichimura, Y., T. Kirisako, T. Takao, Y. Satomi, Y. Shimonishi, N. Ishihara, N.        Shintani, T., N. Mizushima, Y. Ogawa, A. Matsuura, T. Noda, Y. Ohsumi. <br />              Mizushima, I. Tanida, E. Kominami, M. Ohsumi, et al. 2000. A ubiquitin-               1999. Apg10p, novel protein-conjugating enzyme essential autoph- <br />              like mediates protein lipidation. Nature. 408:488&#226;&#8364;&#8220;492.                         agy yeast. EMBO J. 18:5234&#226;&#8364;&#8220;5241. <br />       Kabeya, Y., N. Mizushima, T. Ueno, A. Yamamoto, T. Kirisako, T. Noda, E.              Sutovsky, P., R.D. Moreno, J. Ramalho-Santos, T. Dominko, C. Simerly, and <br />              Kominami, Y. Ohsumi, T. Yoshimori. 2000. LC3, mammalian ho-                     G. Schatten. 1999. Ubiquitin tag sperm mitochondria. Nature. <br />              mologue yeast Apg8p, localized autophagosome membranes af-                   402:371&#226;&#8364;&#8220;372. <br />              ter processing. EMBO J. 19:5720&#226;&#8364;&#8220;5728.                                          Tanaka, Y., G. Guhde, A. Suter, E.L. Eskelinen, D. Hartmann, R. Lullmann- <br />       Kabeya, Y., N. Mizushima, A. Yamamoto, S. Oshitani-Okamoto, Y. Ohsumi,                       Rauch, P.M. Janssen, J. Blanz, K. von Figura, P. Saftig. 2000. Accu- <br />              T. Yoshimori. 2004. LC3, GABARAP GATE16 localize au-                       mulation autophagic vacuoles cardiomyopathy LAMP-2-defi- <br />              tophagosomal membrane depending form-II formation. J. Cell Sci.                    cient mice. Nature. 406:902&#226;&#8364;&#8220;906. <br />              117:2805&#226;&#8364;&#8220;2812.                                                                 Tanida, I., N. Mizushima, M. Kiyooka, M. Ohsumi, T. Ueno, Y. Ohsumi, E. <br />       Klionsky, D.J., S.D. Emr. 2000. Autophagy regulated pathway cellu-               Kominami. 1999. Apg7p/Cvt2p: novel protein-activating enzyme es- <br />              lar degradation. Science. 290:1717&#226;&#8364;&#8220;1721.                                              sential autophagy. Mol. Biol. Cell. 10:1367&#226;&#8364;&#8220;1379. <br />       Komatsu, M., I. Tanida, T. Ueno, M. Ohsumi, Y. Ohsumi, E. Kominami.               Tanida, I., E. Tanida-Miyake, T. Ueno, E. Kominami. 2001. The human ho- <br />              2001. The C-terminal region Apg7p/Cvt2p required ho-                     molog Saccharomyces cerevisiae Apg7p protein-activating en- <br />              modimerization essential E1 activity E1-E2 complex                 zyme multiple substrates including human Apg12p, GATE-16, <br />              formation. J. Biol. Chem. 276:9846&#226;&#8364;&#8220;9854.                                              GABARAP, MAP-LC3. J. Biol. Chem. 276:1701&#226;&#8364;&#8220;1706. <br />       Kopito, R.R. 2000. Aggresomes, inclusion bodies protein aggregation.              Tateishi, K., M. Omat <br /> </body></html>