<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Biochem Parasitol. 2007 December ; 156(2): 246&#226;&#8364;&#8220;254. doi:10.1016/j.molbiopara.2007.09.001. <br />  <br />  <br />  <br />                            Evidence capping enzyme specificity the <br />                            trypanosome spliced leader RNA <br />  <br />                            Jia-peng Ruana, Shuiyuan Shenb, Elisabetta Ullub,c, Christian Tschudia,b,* <br />                            Department Epidemiology &amp; Public Health, Yale University Medical School, 295 Congress Avenue, New <br />  <br />                            Haven, CT 06536-0812, USA <br />                            bDepartment Internal Medicine, Yale University Medical School, 295 Congress Avenue, New Haven, CT <br />                            06536-0812, USA <br />                            cDepartment Cell Biology, Yale University Medical School, 295 Congress Avenue, New Haven, CT <br />                            06536-0812, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  Capping pre-mRNA 5&#226;&#8364;&#178; end addition monomethylated guanosine cap (m7G) essential <br />                                  earliest modification biogenesis mRNA. The reaction catalyzed enzymes: <br />                                  triphosphatase, guanylyltransferase, (guanine N-7) methyltransferase. Whereas modification <br />                                  occurs co-transcriptionally eukaryotic organisms, trypanosomatid protozoa mRNAs acquire <br />                                  m7G cap trans-splicing, entails transfer capped spliced leader (SL) the <br />                                  SL RNA mRNA. Intriguingly, genomes trypanosomatid protozoa sequenced date <br />                                  possess distinct proteins signature motifs guanylyltransferases: TbCGM1 the <br />                                  previously characterized TbCE1. Here provide biochemical evidence TbCgm1 capping <br />                                  enzyme. Whereas RNAi-induced downregulation TbCe1 phenotypic consequences, we <br />                                  TbCGM1 essential trypanosome viability required SL RNA capping. <br />                                  Furthermore, consistent co-transcriptional addition m7G cap, chromatin <br />                                  immunoprecipitation revealed recruitment TbCgm1 SL RNA genes. <br />  <br />  <br />                            Keywords <br />                                  mRNA capping; guanylyltransferase; cap 4 modification; trans-splicing; SL RNA; RNAi <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            1. Introduction <br />                                               In eukaryotic cells analyzed far, including Trypanosoma brucei, addition m7G cap <br />                                               represents earliest event modification 5&#226;&#8364;&#178; end RNA polymerase II (pol II) <br />                                               transcripts [1&#226;&#8364;&#8220;3]. Capping occurs co-transcriptionally RNA achieved chain <br />                                               <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 30 nucleotides (nt). Formation m7GpppN cap entails enzymatic <br />                                               reactions [4]. The 5&#226;&#8364;&#178; triphosphate end hydrolyzed RNA 5&#226;&#8364;&#178; triphosphatase 5&#226;&#8364;&#178; <br />                                               diphosphate, capped GMP RNA guanylyltransferase. Finally, the <br />                                               guanosine moiety GpppN cap methylated RNA (guanine-N7) methyltransferase. <br />                                               Each cap-forming activities essential budding yeast [5&#226;&#8364;&#8220;7]. <br />  <br />  <br />                            *Corresponding author: Tel.: +1-203-7857332; Fax +1-203-7857329; E-mail address: E-mail: christian.tschudi@yale.edu. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service customers <br />                            providing early version manuscript. The manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. Please note production process errors discovered could <br />                            affect content, legal disclaimers apply journal pertain. <br />                             Ruan et al.                                                                                               Page 2 <br />  <br />  <br />                                          Even eukaryotes depend enzymatic reactions m7G capping, the <br />                                          physical organization capping apparatus quite diverse different organisms [4,8,9]. <br />                                          A two-component capping enzyme consisting bifunctional triphosphatase- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          guanylyltransferase polypeptide separate methyltransferase polypeptide present in <br />                                          metazoans plants. In contrast, fungi microsporidia Encephalitozoon cuniculi have <br />                                          component separate triphosphatase, guanylyltransferase and <br />                                          methyltransferase polypeptides. The protozoa Plasmodium falciparum [10] Giardia <br />                                          lamblia [11] yeast-like machinery. Extensive mutational analysis crystal structures <br />                                          revealed RNA guanylyltransferases related DNA RNA ligases [4,8,9] are <br />                                          structurally mechanistically conserved eukaryotic species viruses. <br />                                          Guanylyltransferases catalyze nucleotidyl transfer covalent enzyme-(lysyl-N)- <br />                                          GMP intermediate defined set conserved signature motifs [12]. On the <br />                                          hand, RNA triphosphatases fall mechanistically structurally distinct <br />                                          families. Triphosphatases fungi, microsporidia, protozoa, including T. brucei, and <br />                                          Chlorella virus belong family metal-dependant NTP phosphohydrolases defined by <br />                                          unique active site tunnel architecture [13,14], metazoan plant triphosphatases <br />                                          cysteine phosphatase superfamily [15,16]. The crystal structure (guanine- <br />                                          N7) methyltransferase E. cuniculi highlighted known biochemical specificity of <br />                                          enzyme [17] completed catalogue protein structures enzymes required <br />                                          m7GpppN cap formation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          In eukaryotic organisms m7G capping restricted nascent pol II transcripts. In contrast, <br />                                          studies trypanosomatid protozoa highlighted peculiar aspect mode cap <br />                                          formation, modification present pol II-transcribed spliced leader <br />                                          (SL) RNA, specific subset pol III transcripts, U1, U2, and <br />                                          U4 snRNAs U3 snoRNA. Since abundant pol III transcripts (tRNAs, 5S rRNA <br />                                          7SL RNA) capped, raises numerous questions specificity and <br />                                          regulation capping mechanism. Furthermore, mRNA cap addition occurs post- <br />                                          transcriptional trans-splicing, entails transfer 39-nt long SL sequence, <br />                                          including cap structure, SL RNA 5&#226;&#8364;&#178; end mRNAs [18&#226;&#8364;&#8220;20]. The SL <br />                                          RNA cap unique trypanosomatids contains hypermodified cap 4 structure, is <br />                                          derived methylation seven sites nucleotides SL RNA <br />                                          resulting structure: m7guanosine-ppp-N6,N6,2&#226;&#8364;&#178;-O-trimethyladenosine-p-2&#226;&#8364;&#178;-O- <br />                                          methyladenosine-p-2&#226;&#8364;&#178;-O-methylcytosine-p-N3,2&#226;&#8364;&#178;-O-methyluridine [21]. Recently, 2&#226;&#8364;&#178;-O- <br />                                          methyltransferases required cap 4 biogenesis identified shown be <br />                                          intriguingly similar vaccinia virus VP39 [22&#226;&#8364;&#8220;25]. In addition, cap 4 structure T. <br />                                          brucei specifically recognized unusual nuclear cap binding complex contains three <br />                                          novel essential subunits present trypanosomatids [26]. Taken these <br />                                          observations underscore uniqueness capping apparatus trypanosomatid protozoa. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Our initial characterization enzymatic machinery involved m7G capping in <br />                                          trypanosomatids identified polypeptide Crithidia fasciculata T. brucei (Ce1) with <br />                                          structural biochemical properties characteristic GTP:RNA guanylyltransferases [27]. <br />                                          Whereas carboxy-terminal half Ce1 displays signature motifs characteristic of <br />                                          guanylyltransferases, amino-terminal half contains domain unknown function no <br />                                          resemblance domain associated capping enzymes. In course these <br />                                          studies, noted second polypeptide C. fasciculata T. brucei extracts approximately <br />                                          116 kDa, capable forming covalent protein-GMP complex, indicative capping <br />                                          enzyme. However, characterization polypeptide pursued further. More <br />                                          recently, RNA triphosphatase (TbCet1) characterized T. brucei, structurally <br />                                          mechanistically related fungal metal-dependent phosphohydrolases [28], and <br />                                          recombinant T. brucei Cmt1 shown (guanine-N7) methyltransferase [29]. In <br />                                          addition, data-<span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span> mining identified second candidate T. brucei capping enzyme, TbCgm1, <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                            Page 3 <br />  <br />  <br />                                           predicted N-terminal guanylyltransferase domain C-terminal methyltransferase <br />                                           domain [29]. While manuscript preparation, Ho laboratory reported that <br />                                           TbCgm1 caps SL RNA [30]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Here, used <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>-induced regulation, biochemical properties and <br />                                           chromatin immunoprecipitation examine vivo function T. brucei capping <br />                                           enzymes TbCe1 TbCgm1 interface unusual capping pol II <br />                                           pol III transcripts. <br />  <br />                               2. Materials methods <br />                               2.1. Plasmids, transfections RNAi <br />                                           PCR products encompassing nt 1 - 894 N-terminal domain TbCE1, nt 180 - 1,180 <br />                                           guanylyltransferase domain TbCGM1 inserted sense antisense <br />                                           orientation separated stuffer fragment plasmid pLEW100 [31] transfected into <br />                                           procyclic T. brucei strain 29.13.6 [32], harbors integrated genes T7 RNA <br />                                           polymerase tetracycline repressor. The cells selected phleomycin cloned <br />                                           96-well plates. Clonal cell lines induced adding tetracycline concentration of <br />                                           10 &#206;&#188;g/ml. A PCR-based approach [33] used introduce BB2 epitope tag N- <br />                                           terminus TbCe1 TbCgm1 replace second allele blasticidin (BSR) <br />                                           resistant gene. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> produce recombinant guanylyltransferase E. coli, nt 1- 2,355 TbCGM1, corresponding <br />                                           amino acids 1 - 785, cloned NotI XhoI restriction sites T7 <br />                                           RNA polymerase-based expression vector pET-44a (Novagen). The lysine residue motif I <br />                                           changed alanine two-step PCR procedure <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> verified DNA <br />                                           sequencing. <br />  <br />                               2.2. RNA analysis <br />                                           RNA extraction Northern blot analysis performed described [34]. In Figure 3B, <br />                                           4B 4C, SL RNA primer extended oligonucleotide Y-21, complementary to <br />                                           nt 40 60, Figure 4D SLIN22, complementary nt 110 131, used. U2, U3 and <br />                                           U6 snRNAs primed oligonucleotide U2B-17, complementary nt 46 62, U3-PE, <br />                                           complementary nt 96 123, U6-22, complementary nt 19 40, respectively. <br />  <br />                                           Total RNA subjected immunoprecipitations rabbit polyclonal antibodies against <br />                                           m7G cap (a kind gift Dr. F. Richards) mouse monoclonal antibodies the <br />                                           2,2,7-trimethylguanosine cap (Calbiochem). RNA samples analyzed primer extension. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Uninduced TbCGM1 RNAi-induced procyclic T. brucei cells permeabilized with <br />                                           lysolecithin (L-&#206;&#177;-lysophosphatidylcholine, palmitoyl, Sigma) RNA synthesized for <br />                                           15 minutes presence [&#206;&#177;-32P]GTP [&#206;&#177;-32P]ATP described [3,35,36]. Total RNA, <br />                                           extracted TRIZOL reagent, separated 6% polyacrylamide - 7M urea gels, SL <br />                                           RNA excised eluted water. Tobacco acid pyrophosphatase (TAP) cleavage and <br />                                           nuclease P1 digestion essentially described [3] digestion products were <br />                                           chromatographed Avicel cellulose layer plates, developed isobutyric acid/ <br />                                           concentrated NH4OH/H2O, 66:1:33 (v/v/v). <br />  <br />                               2.3. Other procedures <br />                                           GMP binding [27] ChIP previously described [37]. <br />  <br />  <br />  <br />  <br />                                             Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                                Page 4 <br />  <br />  <br />                               3. Results <br />                               3.1. The guanylyltransferase domain TbCgm1 binds GMP <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            During search T. brucei databases additional components capping machinery, <br />                                            [29] predicted protein, TbCgm1, 116,820 Da (GeneDB accession # <br />                                            Tb927.7.2080) convincing homologies RNA (guanine-N7) methyltransferases the <br />                                            C-terminal half protein. Further domain motif searches revealed N-terminal <br />                                            half characteristics guanylyltransferase domain peptide motifs are <br />                                            absolutely conserved superfamily capping enzymes DNA ligases constitute <br />                                            active site GMP binding nucleotidyl transfer [4,29]. Northern blot, 5&#226;&#8364;&#178; end RACE <br />                                            sequencing cDNA clones established authenticity gene T. brucei <br />                                            genome determined structure mRNA 5&#226;&#8364;&#178; end (data shown). <br />  <br />                                            The identification second putative guanylyltransferase domain eukaryotic organism <br />                                            unique. However, finding rationalized earlier observation T. brucei <br />                                            cell extracts polypeptides 67 116 kDa capable forming covalent <br />                                            GMP-enzyme complex characteristic capping enzymes [27]. Whereas 67 kDa- <br />                                            polypeptide (TbCe1) shown functional guanylyltransferase domain, larger <br />                                            polypeptide pursued further. In view predicted molecular weight 117 kDa and <br />                                            presence putative N-terminal guanylyltransferase domain, TbCgm1 likely <br />                                            candidate second GMP-binding enzyme. To test TbCGM1 encoded protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            bind GMP covalent fashion, initially expressed full-length TbCgm1 in <br />                                            Escherichia coli different expression vectors, did result the <br />                                            production sufficient quantities recombinant protein soluble form perform <br />                                            enzymatic tests. Thus, generated construct containing 785 amino acids TbCgm1 <br />                                            fused N-terminus E. coli NusA protein. A 142 kDa protein corresponding the <br />                                            fusion protein detectable SDS-PAGE soluble extracts IPTG-induced bacteria <br />                                            (upper panel Fig. 1, lane 2). The recombinant protein formed SDS-stable nucleotide- <br />                                            protein adduct (lower panel Fig. 1, lane 2), evident sample expressing <br />                                            NusA protein (lane 1). Importantly, protein lost ability bind GMP, the <br />                                            predicted catalytic lysine residue (amino acid 127) motif I mutated alanine (lane 3), <br />                                            providing evidence TbCgm1 functional guanylyltransferase domain. <br />  <br />                               3.2. TbCgm1 encoded essential gene <br />                                            To begin investigate vivo function T. brucei polypeptides containing <br />                                            guanylyltransferase domains, used RNA interference (RNAi) downregulate the <br />                                            respective mRNAs. To this, nt 1 - 894 TbCE1 nt 180 - 1,180 TbCGM1 were <br />                                            inserted sense antisense orientation separated spacer fragment plasmid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pLEW100 [38]. This cassette, driven tetracycline (tet) inducible ribosomal promoter, was <br />                                            stably integrated non-transcribed rDNA spacer region clonal cell lines were <br />                                            established. Upon RNAi induction tet, production double-stranded RNA and <br />                                            degradation target mRNA verified Northern blot (data shown). In addition, <br />                                            extracts prepared CE1-RNAi cell lines induced 24 hr GMP-enzyme complex <br />                                            formation reduced undetectable levels (Fig. 2A), CGM1-RNAi cell lines the <br />                                            Cgm1 protein severely decreased, albeit detectable 24 hr (Fig. 2B). Importantly, <br />                                            results Figures 2A B demonstrated specificity downregulation Ce1 or <br />                                            Cgm1 corresponding RNAi construct. Next, monitored cell growth period of <br />                                            4 days (Fig. 2C). Compared wild-type uninduced cells, downregulation TbCE1 had <br />                                            noticeable effect cell viability. In fact, maintaining induction 10 days did <br />                                            reveal detectable change growth characteristics. In contrast, monitoring cell growth <br />                                            TbCGM1 RNAi induction revealed growth rate declined day 2 day <br />                                            3 eventually cells died (Fig. 2C), indicating TbCGM1 essential viability. <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                              Page 5 <br />  <br />  <br />                               3.3. Depletion TbCE1 TbCGM1 does affect capping U2 snRNA U3 snoRNA <br />                                           Similar eukaryotes, trypanosomatid U1, U2 U4 snRNAs U3 snoRNA <br />                                           contain 2,2,7-trimethylguanosine (TMG) cap [39&#226;&#8364;&#8220;41]. Little known sequence of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           events leading TMG cap trypanosomatids, presumed RNA molecules <br />                                           initially modified m7G cap [42,43]. Thus, set experiments, asked <br />                                           TbCE1 TbCGM1 downregulation effect cap formation U2 snRNA <br />                                           U3 snoRNA. TbCE1-RNAi cells induced tet 8 days, RNA prepared <br />                                           assayed immunoprecipitation anti-m7G anti-TMG antibodies followed by <br />                                           primer extension analysis (Fig. 3A). As expected, U2 U3 RNAs affinity selected <br />                                           anti-TMG antibodies RNA prepared uninduced cells (Fig. 3A, lane 5), whereas <br />                                           RNAs detectable anti-m7G immunoprecipitate (lane 3). A similar result <br />                                           obtained RNA isolated TbCE1-RNAi cells induced 8 days (lanes 8 10). <br />                                           Furthermore, detectable difference uninduced TbCGM1-RNAi cells and <br />                                           cells induced 2.5 days (Fig. 4A, compare lanes 2 5 lanes 7 10), indicating that <br />                                           cap formation U2 snRNA U3 snoRNA appeared normal conditions where <br />                                           TbCe1 TbCgm1 protein amounts limiting. <br />  <br />                               3.4. TbCgm1 involved SL RNA capping <br />                                           We investigated RNAi-induced downregulation TbCE1 TbCGM1 affected <br />                                           cap formation pol II-transcribed SL RNA. RNA prepared TbCE1-RNAi cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           induced 2.5 8 days assayed primer extension analysis SL RNA (Fig. 3B). <br />                                           As shown previously [22,44] evident Figure 3B (lane 2), primer extension analysis at <br />                                           low deoxyribonucleotide triphosphate concentrations gives rise characteristic primer <br />                                           extension stop 5&#226;&#8364;&#178; end SL RNA indicative fully-methylated cap 4 structure <br />                                           (indicated fm SL). In contrast, hypomethylated SL RNA (hm SL) prepared cells grown <br />                                           presence methylation inhibitor sinefungin, gives rise series bands <br />                                           characteristic SL RNA partially methylated cap 4 (Fig. 3B, lane 1). TbCE1 <br />                                           downregulation 8 days did affect accumulation methylation status SL RNA, <br />                                           assayed primer extension analysis (Fig. 3B, compare lanes 2 4). In contrast, following <br />                                           2 2.5 days induction TbCGM1 RNAi, accumulation SL RNA reduced <br />                                           76% 36%, respectively, compared uninduced cells (Fig. 4B). In addition, clear <br />                                           defect cap 4 formation evident fully-modified SL RNA reduced 34% 13% <br />                                           day 2 2.5, respectively, concomitant appearance longer extension products <br />                                           indicative SL RNA carrying hypomethylated cap 4 (lanes 4 5). <br />  <br />                                           To address apparent effect TbCGM1 downregulation SL RNA cap formation in <br />                                           detail, used immunoprecipitation anti-m7G antibodies (Fig. 4C). Whereas <br />                                           majority fully-modified SL RNA affinity selected RNA isolated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           uninduced cells (lane 3), 2.5 days induction hypomethylated SL RNA was <br />                                           predominantly supernatant (compare lanes 5 6). In contrast, small amount <br />                                           fully-modified SL RNA present 2.5 days induction immunoprecipitated with <br />                                           anti-m7G antibodies (lane 6). Thus, epitope recognized antibody present in <br />                                           considerable proportion steady-state SL RNA accumulating downregulation of <br />                                           Cgm1, suggesting defect m7G cap. <br />  <br />                                           Next, turned analysis newly-synthesized RNA permeabilized T. brucei cells. <br />                                           Uninduced TbCGM1-RNAi cells induced 1 2.5 days permeabilized with <br />                                           lysolecithin incubated 15 min transcription cocktail containing [&#206;&#177;-32P]GTP [35]. <br />                                           Using number cells permeabilization procedure, obvious <br />                                           difference total RNA synthesis induced cells, compared uninduced cells (data <br />                                           shown). The SL RNA gel purified digested tobacco acid pyrophosphatase <br />                                           (TAP), released cap nucleotide SL RNA hydrolyzing pyrophosphate <br />  <br />  <br />                                             Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                               Page 6 <br />  <br />  <br />                                           bond. The digestion products separated layer chromatography (TLC) analyzed <br />                                           phosphorimaging (Fig. 5A). This treatment released 1.5% radioactive <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> SL <br />                                           RNA uninduced cells form m7G (lane 1). This value did change 1 day <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           TbCGM1 depletion (lane 2, 1.5%), reduced 0.8% following 2.5 days induction <br />                                           (lane 3). In uninduced (lane 1) 1 day-induced cells (lane 2) 0.8% radioactive label <br />                                           released TAP form GMP, indicative cap methyl group the <br />                                           N7 position guanosine. This value increased 1.4% sample isolated cells <br />                                           induced 2.5 days (lane 3). In addition, new spot (1.5% radioactive label) <br />                                           did correspond available standards (labeled X1 Fig. 5A). Even two- <br />                                           dimensional TLC analysis (data shown) failed clues possible nature the <br />                                           X1 spot. However, data agreement immunoprecipitation results <br />                                           anti-m7G antibodies indicated CGM1 silencing increase <br />                                           proportion SL RNA bearing unmethylated cap structure. <br />  <br />                                           To define consequences TbCgm1 depletion SL RNA cap, permeabilized <br />                                           cells labeled [&#206;&#177;-32P]ATP order monitor status adenosine position <br />                                           1, i.e. initiating nucleotide, SL RNA. The gel purified SL RNA digested with <br />                                           nuclease P1 (Fig. 5B) generates 5&#226;&#8364;&#178;nucleotide monophosphates, does cleave <br />                                           pyrophosphate bonds, i.e. cleave triphosphate bridge linking m7G the <br />                                           adenosine position 1. As expected, sample prepared uninduced cells majority <br />                                           label (96%) comigrated AMP (lane 1). The prominent spot (3%, designated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           X2) did correspond available standards, decreased induction <br />                                           period 50% abundant 2.5 days. P1 digestion SL RNA labeled with <br />                                           [&#206;&#177;-32P]GTP generated identical spot (data shown) speculate this <br />                                           represents dinucleotide derived cap structure. In addition, 0.7% radioactive <br />                                           label associated ATP, likely originates 5&#226;&#8364;&#178; end primary SL <br />                                           transcripts. Following 1 2.5 days TbCgm1 depletion level ATP did change <br />                                           significantly. A fourth spot identified ADP. Whereas SL RNA uninduced cells <br />                                           0.2% radioactivity associated ADP, downregulation TbCGM1 1 2.5 <br />                                           days resulted gradual increase ADP reaching 0.5% 2.5 days. The likely <br />                                           explanation result TbCgm1 depletion gave rise increase diphosphate- <br />                                           terminated RNA, i.e. capping transcripts blocked triphosphatase step, <br />                                           removed terminal phosphate primary transcript. Thus, taken these <br />                                           observations provided supporting evidence TbCgm1 required capping SL <br />                                           RNA. <br />  <br />                               3.5. Trans-splicing utilization SL RNA reduced TbCgm1-depleted cells <br />                                           Since m7G cap determinants utilization SL RNA trans- <br />                                           splicing, examined trans-splicing activity TbCgm1-depleted cells (Fig. 4D). In RNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           derived uninduced cells (lane 2), primer extension analysis produced cDNA <br />                                           corresponding full-length SL RNA, primer extension stop mapping 5&#226;&#8364;&#178; end <br />                                           SL intron, diagnostic Y-structures (SL RNA intron-pre-mRNA <br />                                           branched intermediates formed step trans-splicing). Y-structures are <br />                                           detectable cells treated sinefungin (lane 1), drug blocks trans-splicing in <br />                                           trypanosomes [45]. By analysis TbCGM1 silencing reduced the <br />                                           accumulation Y-structures 35% 2.5 days induction (Fig. 4D, lane 5). <br />  <br />                               3.6. Recruitment TbCgm1 SL RNA genes <br />                                           To provide independent evidence TbCgm1 involved maturation SL RNA, <br />                                           monitored vivo association polypeptide SL RNA genes using <br />                                           chromatin immunoprecipitation (ChIP). To end, generated T. brucei cell line <br />                                           expressing solely epitope-tagged version TbCgm1. We deleted allele homologous <br />  <br />  <br />                                             Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                                Page 7 <br />  <br />  <br />                                          recombination PCR-generated cassette encoding blasticidin resistance gene then <br />                                          introduced N-terminal BB2 epitope second allele original chromosomal locus. <br />                                          This approach ensured epitope-tagged version protein functional and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          expressed levels comparable endogenous protein. In addition, generated cell line <br />                                          solely expressing BB2 epitope-tagged TbCe1. We previously shown ChIP pol II <br />                                          transcription factor TRF4 recruited SL RNA genes [37]. Using same <br />                                          protocol, TbCgm1 specifically associated SL RNA genes (Fig. 6, lane 6), <br />                                          upstream control region (lane 3). In contrast, did detect recruitment of <br />                                          TbCe1 SL RNA genes (lane 12). This result corroborated RNAi permeable cell <br />                                          experiments supported conclusion TbCgm1 plays role capping the <br />                                          SL RNA. <br />  <br />                               4. Discussion <br />                                          Eukaryotes rely enzymatic reactions formation m7GpppN cap, namely <br />                                          sequential action RNA 5&#226;&#8364;&#178; triphosphatase, RNA guanylyltransferase RNA (guanine- <br />                                          N7) methyltransferase. Although physical organization capping apparatus quite <br />                                          distinct different taxa [4,8,9], enzymatic activity assigned single polypeptide <br />                                          domain. In contrast, distinct novel genetic organization emerging in <br />                                          trypanosomatids, presence guanylyltransferase domains: (TbCe1) fused <br />                                          N-terminal domain unknown function [27] second (TbCgm1) C-terminal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (guanine-N7) methyltransferase [29,30]. In addition, separate RNA triphosphatase, <br />                                          mechanistically related fungal enzymes [28,46], separate (guanine-N7) <br />                                          methyltransferase [29]. Although polypeptides characterized <br />                                          biochemically shown involved m7G capping, challenge remains assign in <br />                                          vivo functions. Here taken step goal analyzed functional <br />                                          consequences TbCe1 TbCgm1 downregulation. Surprisingly, RNAi downregulation of <br />                                          Ce1 phenotypic consequences, precluding investigations route. <br />                                          Nevertheless, interpret result mean TbCe1 needed capping in <br />                                          trypanosomes. Since RNAi does generate &#226;&#8364;&#339;<span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>knockout</span>&#226;&#8364;?, downregulates mRNA <br />                                          levels, likely explanation residual TbCe1 protein sufficient <br />                                          cell growth. Indeed, far able generate &#226;&#8364;&#339;classical&#226;&#8364;? TbCE1 knockout <br />                                          strain, alleles gene replaced antibiotic resistance marker, <br />                                          indicating gene likely essential. Nevertheless, vivo function TbCe1 <br />                                          remains determined. <br />  <br />                                          As shown presence diagnostic motifs [29] mutagenesis analysis, TbCgm1 <br />                                          hallmarks activity RNA guanylyltransferase. In contrast negative <br />                                          results obtained RNAi downregulation CE1, silencing CGM1 severe <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          consequences cell viability SL RNA capping: 2 3 days RNAi <br />                                          induction cells stopped growing demonstrating Cgm1 function essential cell <br />                                          viability. This result similar observed yeast S. cerevisiae, it <br />                                          originally shown Ceg1, gene encoding capping enzyme, essential [7]. During <br />                                          time interval cell growth begins slow down, observed anomalies with <br />                                          regard SL RNA. First, steady-state level SL RNA decreased substantially. This <br />                                          effect instability SL RNA molecules unmethylated cap <br />                                          and/or result inhibition trans-splicing, likely cause havoc in <br />                                          cells. Second, accumulation molecules bearing hypomodified cap 4 structure. <br />                                          The explanation finding likely lies requirement m7G cap the <br />                                          biogenesis cap 4 structure. Indeed, vaccinia virus steps biosynthesis the <br />                                          cap occur unique sequence, i.e. modification nucleotide adjacent cap <br />                                          requires m7G cap place [47]. Third, CGM1 silencing proportion SL <br />                                          RNA bearing methylated cap substantially decreased, assessed antibody <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                                               Page 8 <br />  <br />  <br />                                          immunoprecipitation steady-state SL RNA TAP digestion newly&#226;&#8364;&#8220;synthesized SL <br />                                          RNA. Lastly, P1 digestion TLC analysis showed accumulation newly- <br />                                          synthesized SL RNA molecules ADP terminus. In summary, results showed that <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Cgm1 depletion results defects addition GMP methylation N7 <br />                                          position guanine. On basis analyses, propose guanylyltransferase <br />                                          domain TbCgm1 responsible adding GMP diphosphate terminated SL RNA. <br />                                          Activity assays recombinant protein corroborated proposed function domain. <br />                                          Furthermore, observation relative m7G cap decreased, consequence <br />                                          TbCgm1 depletion, concomitant increase unmethylated GMP cap, provides <br />                                          evidence polypeptide required methylation step capping reaction. <br />                                          So far, able produce putative C-terminal (guanine-N7) <br />                                          methyltransferase domain recombinant form test proposed enzymatic activity. While <br />                                          manuscript preparation, Ho laboratory reported recombinant TbCgm1 a <br />                                          bifunctional capping enzyme capable transferring GMP SL RNA covalent enzyme- <br />                                          GMP intermediate [30]. In addition, shown TbCgm1 methylates guanine N-7 <br />                                          position GpppN-terminated RNA. Taken together, results agreement with <br />                                          TbCgm1 responsible capping SL RNA. <br />  <br />                                          We previously shown SL RNA transcripts short 30 nucleotides capped, <br />                                          consistent co-transcriptional mechanism m7G capping addition [3]. This model is <br />                                          strengthened ChIP experiments presented here, revealed vivo <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          association TbCgm1 SL RNA genes. What remains explored the <br />                                          capping enzyme recruited transcriptional machinery. Since SL RNA genes are <br />                                          transcribed pol II [48], mechanism likely similar described yeast and <br />                                          mammals, physical interaction capping enzyme CTD the <br />                                          largest subunit pol II [49,50]. <br />  <br />                                          The presence guanylyltransferase domains trypanosomatid protozoa unexpected <br />                                          far, best knowledge, reported eukaryotic organism. <br />                                          It important bear mind capping organisms occurs pol II <br />                                          transcripts, SL RNA, specific subset pol III transcripts. One <br />                                          attractive possibility TbCe1 involved capping subset pol III transcripts. How <br />                                          TbCe1 directed U1, U2, U3 U4 snRNAs, abundant <br />                                          pol III transcripts, present matter speculation. Since pol III common these <br />                                          genes, possibilities consider specificity achieved gene- <br />                                          specific transcription factor(s), RNA determinants combination mechanisms. <br />  <br />                               Acknowledgments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          We thank Andrei Alexandrov help discussion TLC analysis. This study received support National <br />                                          Institutes Health Grant AI43594 C.T. <br />  <br />  <br />                               References <br />                                          1. Rasmussen EB, Lis JT. In vivo transcriptional pausing cap formation Drosophila heat <br />                                             shock genes. Proc Natl Acad Sci U S A 1993;90:7923&#226;&#8364;&#8220;7. [PubMed: 8367444] <br />                                          2. Hagler J, Shuman S. A freeze-frame view eukaryotic transcription elongation capping <br />                                             nascent mRNA. Science 1992;255:983&#226;&#8364;&#8220;6. [PubMed: 1546295] <br />                                          3. Mair G, Ullu E, Tschudi C. Cotranscriptional cap 4 formation Trypanosoma brucei spliced <br />                                             leader RNA. J Biol Chem 2000;275:28994&#226;&#8364;&#8220;9. [PubMed: 10880518] <br />                                          4. Shuman S. Structure, mechanism, evolution mRNA capping apparatus. Prog Nucleic Acid <br />                                             Res Mol Biol 2001;66:1&#226;&#8364;&#8220;40. [PubMed: 11051760] <br />  <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                                    Page 9 <br />  <br />  <br />                                          5. Shibagaki Y, Itoh N, Yamada H, Nagata S, Mizumoto K. mRNA capping enzyme. Isolation and <br />                                             characterization gene encoding mRNA guanylytransferase subunit Saccharomyces <br />                                             cerevisiae. J Biol Chem 1992;267:9521&#226;&#8364;&#8220;8. [PubMed: 1315757] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          6. Mao X, Schwer B, Shuman S. Yeast mRNA cap methyltransferase 50-kilodalton protein encoded <br />                                             essential gene. Mol Cell Biol 1995;15:4167&#226;&#8364;&#8220;74. [PubMed: 7623811] <br />                                          7. Tsukamoto T, Shibagaki Y, Imajoh-Ohmi S, Murakoshi T, Suzuki M, Nakamura A, Gotoh H, <br />                                             Mizumoto K. Isolation characterization yeast mRNA capping enzyme beta subunit gene <br />                                             encoding RNA 5&#226;&#8364;&#178;-triphosphatase, essential cell viability. Biochem Biophys Res Commun <br />                                             1997;239:116&#226;&#8364;&#8220;22. [PubMed: 9345280] <br />                                          8. Shuman S. What messenger RNA capping tells eukaryotic evolution. Nat Rev Mol Cell Biol <br />                                             2002;3:619&#226;&#8364;&#8220;25. [PubMed: 12154373] <br />                                          9. Shuman S, Lima CD. The polynucleotide ligase RNA capping enzyme superfamily covalent <br />                                             nucleotidyltransferases. Curr Opin Struct Biol 2004;14:757&#226;&#8364;&#8220;64. [PubMed: 15582400] <br />                                          10. Ho CK, Shuman S. A yeast-like mRNA capping apparatus Plasmodium falciparum. Proc Natl <br />                                              Acad Sci U S A 2001;98:3050&#226;&#8364;&#8220;5. [PubMed: 11248030] <br />                                          11. Hausmann S, Altura MA, Witmer M, Singer SM, Elmendorf HG, Shuman S. Yeast-like mRNA <br />                                              capping apparatus Giardia lamblia. J Biol Chem 2005;280:12077&#226;&#8364;&#8220;86. [PubMed: 15556935] <br />                                          12. Hakansson K, Doherty AJ, Shuman S, Wigley DB. X-ray crystallography reveals large <br />                                              conformational change guanyl transfer mRNA capping enzymes. Cell 1997;89:545&#226;&#8364;&#8220;53. <br />                                              [PubMed: 9160746] <br />                                          13. Lima CD, Wang LK, Shuman S. Structure mechanism yeast RNA triphosphatase: essential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              component mRNA capping apparatus. Cell 1999;99:533&#226;&#8364;&#8220;43. [PubMed: 10589681] <br />                                          14. Gong C, Shuman S. Chlorella virus RNA triphosphatase: mutational analysis mechanism of <br />                                              inhibition tripolyphosphate. J Biol Chem 2002;13:13. <br />                                          15. Takagi T, Moore CR, Diehn F, Buratowski S. An RNA 5&#226;&#8364;&#178;-triphosphatase related protein tyrosine <br />                                              phosphatases. Cell 1997;89:867&#226;&#8364;&#8220;73. [PubMed: 9200605] <br />                                          16. Changela A, Ho CK, Martins A, Shuman S, Mondragon A. Structure mechanism RNA <br />                                              triphosphatase component mammalian mRNA capping enzyme. EMBO J 2001;20:2575&#226;&#8364;&#8220;86. <br />                                              [PubMed: 11350947] <br />                                          17. Fabrega C, Hausmann S, Shen V, Shuman S, Lima CD. Structure mechanism mRNA cap <br />                                              (guanine-N7) methyltransferase. Mol Cell 2004;13:77&#226;&#8364;&#8220;89. [PubMed: 14731396] <br />                                          18. Murphy WJ, Watkins KP, Agabian N. Identification novel Y branch structure intermediate <br />                                              trypanosome mRNA processing: evidence trans splicing. Cell 1986;47:517&#226;&#8364;&#8220;25. [PubMed: <br />                                              3779835] <br />                                          19. Sutton RE, Boothroyd JC. Evidence trans splicing trypanosomes. Cell 1986;47:527&#226;&#8364;&#8220;35. <br />                                              [PubMed: 3022935] <br />                                          20. Laird PW, Zomerdijk JC, Korte D, Borst P. In vivo labelling intermediates discontinuous <br />                                              synthesis mRNAs Trypanosoma brucei. EMBO J 1987;6:1055&#226;&#8364;&#8220;62. [PubMed: 3595556] <br />                                          21. Bangs JD, Crain PF, Hashizume T, McCloskey JA, Boothroyd JC. Mass spectrometry mRNA cap <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              4 trypanosomatids reveals novel nucleosides. J Biol Chem 1992;267:9805&#226;&#8364;&#8220;15. [PubMed: <br />                                              1349605] <br />                                          22. Arhin GK, Li H, Ullu E, Tschudi C. A protein related vaccinia virus cap-specific <br />                                              methyltransferase VP39 involved cap 4 modification Trypanosoma brucei. RNA 2006;12:53&#226;&#8364;&#8220; <br />                                              62. [PubMed: 16301606] <br />                                          23. Arhin GK, Ullu E, Tschudi C. 2&#226;&#8364;&#178;-O-methylation position 2 trypanosome spliced leader cap <br />                                              4 mediated 48 kDa protein related vaccinia virus VP39. Mol Biochem Parasitol <br />                                              2006;147:137&#226;&#8364;&#8220;9. [PubMed: 16516986] <br />                                          24. Zamudio JR, Mittra B, Zeiner GM, Feder M, Bujnicki JM, Sturm NR, Campbell DA. Complete cap <br />                                              4 formation required viability Trypanosoma brucei. Eukaryot Cell 2006;5:905&#226;&#8364;&#8220;15. <br />                                              [PubMed: 16757738] <br />                                          25. Hall MP, Ho CK. Functional characterization 48 kDa Trypanosoma brucei cap 2 RNA <br />                                              methyltransferase. Nucleic Acids Res 2006;34:5594&#226;&#8364;&#8220;602. [PubMed: 17028101] <br />                                          26. Li H, Tschudi C. Novel essential subunits 300-kilodalton nuclear cap binding complex of <br />                                              Trypanosoma brucei. Mol Cell Biol 2005;25:2216&#226;&#8364;&#8220;26. [PubMed: 15743819] <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                                   Page 10 <br />  <br />  <br />                                          27. Silva E, Ullu E, Kobayashi R, Tschudi C. Trypanosome capping enzymes display novel two-domain <br />                                              structure. Mol Cell Biol 1998;18:4612&#226;&#8364;&#8220;9. [PubMed: 9671471] <br />                                          28. Ho CK, Shuman S. Trypanosoma brucei RNA triphosphatase. Antiprotozoal drug target guide <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              eukaryotic phylogeny. J Biol Chem 2001;276:46182&#226;&#8364;&#8220;6. [PubMed: 11553645] <br />                                          29. Hall MP, Ho CK. Characterization Trypanosoma brucei RNA cap (guanine N-7) <br />                                              methyltransferase. RNA 2006;12:488&#226;&#8364;&#8220;97. [PubMed: 16431985] <br />                                          30. Takagi Y, Sindkar S, Ekonomidis D, Hall MP, Ho CK. Trypanosoma brucei encodes bifunctional <br />                                              capping enzyme essential cap 4 formation spliced leader RNA. J Biol Chem <br />                                              2007;282:15995&#226;&#8364;&#8220;6005. [PubMed: 17416901] <br />                                          31. Djikeng, A.; Shen, S.; Tschudi, C.; Ullu, E. Analysis gene function Trypanosoma brucei using <br />                                              RNA interference. In: Melville, S., editor. Parasite Genomics Protocols. Vol. 270. Humana Press <br />                                              Inc; 2004. p. 287-97. <br />                                          32. Wirtz E, Leal S, Ochatt C, Cross GA. A tightly regulated inducible expression conditional <br />                                              gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol Biochem Parasitol <br />                                              1999;99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br />                                          33. Arhin GK, Shen S, Ullu E, Tschudi C. A PCR-based method gene deletion protein tagging <br />                                              Trypanosoma brucei. Methods Mol Biol 2004;270:277&#226;&#8364;&#8220;86. [PubMed: 15153634] <br />                                          34. Djikeng A, Shi H, Tschudi C, Ullu E. RNA interference Trypanosoma brucei: cloning small <br />                                              interfering RNAs provides evidence retroposon-derived 24&#226;&#8364;&#8220;26-nucleotide RNAs. RNA <br />                                              2001;7:1522&#226;&#8364;&#8220;30. [PubMed: 11720282] <br />                                          35. Ullu E, Tschudi C. Permeable trypanosome cells model transcription trans- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              splicing. Nucleic Acids Res 1990;18:3319&#226;&#8364;&#8220;26. [PubMed: 2356121] <br />                                          36. Ullu E, Tschudi C. Trans splicing trypanosomes requires methylation 5&#226;&#8364;&#178; end spliced <br />                                              leader RNA. Proc Natl Acad Sci U S A 1991;88:10074&#226;&#8364;&#8220;8. [PubMed: 1719544] <br />                                          37. Ruan JP, Arhin GK, Ullu E, Tschudi C. Functional characterization Trypanosoma brucei TATA- <br />                                              binding protein-related factor points universal regulator transcription trypanosomes. Mol <br />                                              Cell Biol 2004;24:9610&#226;&#8364;&#8220;8. [PubMed: 15485927] <br />                                          38. Shi H, Djikeng A, Mark T, Wirtz E, Tschudi C, Ullu E. Genetic interference Trypanosoma <br />                                              brucei heritable inducible double-stranded RNA. RNA 2000;6:1069&#226;&#8364;&#8220;76. [PubMed: <br />                                              10917601] <br />                                          39. Djikeng A, Ferreira L, D&#226;&#8364;&#8482;Angelo M, Dolezal P, Lamb T, Murta S, Triggs V, Ulbert S, Villarino A, <br />                                              Renzi SUE, Tschudi C. Characterization candidate Trypanosoma brucei U1 snRNA gene. Mol <br />                                              Biochem Parasitol 2001;113:91&#226;&#8364;&#8220;7. <br />                                          40. Tschudi C, Richards FF, Ullu E. The U2 RNA analogue Trypanosoma brucei gambiense: <br />                                              implications splicing mechanism trypanosomes. Nucleic Acids Res 1986;14:8893&#226;&#8364;&#8220;903. <br />                                              [PubMed: 3786140] <br />                                          41. Mottram J, Perry KL, Lizardi PM, Luhrmann R, Agabian N, Nelson RG. Isolation sequence of <br />                                              small nuclear U RNA genes Trypanosoma brucei subsp. brucei: identification U2, U4, <br />                                              U6 RNA analogs. Mol Cell Biol 1989;9:1212&#226;&#8364;&#8220;23. [PubMed: 2725495] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          42. Gunzl A, Bindereif A, Ullu E, Tschudi C. Determinants cap trimethylation U2 small nuclear <br />                                              RNA conserved Trypanosoma brucei higher eukaryotic organisms. Nucleic <br />                                              Acids Res 2000;28:3702&#226;&#8364;&#8220;9. [PubMed: 11000261] <br />                                          43. Tschudi C, Ullu E. Unconventional rules small nuclear RNA transcription cap modification <br />                                              trypanosomatids. Gene Expr 2002;10:3&#226;&#8364;&#8220;16. [PubMed: 11868986] <br />                                          44. Mandelboim M, Estrano CL, Tschudi C, Ullu E, Michaeli S. On role exon intron sequences <br />                                              trans-splicing utilization cap 4 modification trypanosomatid Leptomonas collosoma SL <br />                                              RNA. J Biol Chem 2002;277:35210&#226;&#8364;&#8220;8. [PubMed: 12121975] <br />                                          45. McNally KP, Agabian N. Trypanosoma brucei spliced-leader RNA methylations required for <br />                                              trans splicing vivo. Mol Cell Biol 1992;12:4844&#226;&#8364;&#8220;51. [PubMed: 1406666] <br />                                          46. Gong C, Martins A, Shuman S. Structure-function analysis Trypanosoma brucei RNA <br />                                              triphosphatase evidence two-metal mechanism. J Biol Chem 2003;278:50843&#226;&#8364;&#8220;52. <br />                                              [PubMed: 14525979] <br />                                          47. Barbosa E, Moss B. mRNA(nucleoside-2&#226;&#8364;&#178;-)-methyltransferase vaccinia virus. Characteristics <br />                                              substrate specificity. J Biol Chem 1978;253:7698&#226;&#8364;&#8220;702. [PubMed: 701282] <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                               Page 11 <br />  <br />  <br />                                          48. Gilinger G, Bellofatto V. Trypanosome spliced leader RNA genes contain identified RNA <br />                                              polymerase II gene promoter organisms. Nucleic Acids Res 2001;29:1556&#226;&#8364;&#8220;64. [PubMed: <br />                                              11266558] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          49. Cho EJ, Takagi T, Moore CR, Buratowski S. mRNA capping enzyme recruited transcription <br />                                              complex phosphorylation RNA polymerase II carboxy-terminal domain. Genes Dev <br />                                              1997;11:3319&#226;&#8364;&#8220;26. [PubMed: 9407025] <br />                                          50. McCracken S, Fong N, Rosonina E, Yankulov K, Brothers G, Siderovski D, Hessel A, Foster S, <br />                                              Program AE, Shuman S, Bentley DL. 5&#226;&#8364;&#178;-Capping enzymes targeted pre-mRNA binding to <br />                                              phosphorylated carboxy-terminal domain RNA polymerase II. Genes Dev 1997;11:3306&#226;&#8364;&#8220;18. <br />                                              [PubMed: 9407024] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                       Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Fig. 1. <br />                                          The guanylyltransferase domain TbCgm1 binds GMP. Soluble protein lysates bacteria <br />                                          expressing NusA protein (pET-44, lane 1), amino acids 1 - 785 TbCgm1 fused the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          NusA protein (wt, lane 2), fusion protein indicated mutation (K127A, lane 3) <br />                                          fractionated SDS-PAGE stained Coomassie blue dye (upper panel) tested <br />                                          GMP binding (lower panel, autoradiograph shown). The sizes (in kilodaltons) of <br />                                          marker proteins shown right position NusA-TbCgm1 fusion proteins <br />                                          indicated arrow. <br />  <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                         Page 13 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Fig. 2. <br />                                          RNAi-induced downregulation TbCE1 TbCGM1. (A) Two independent clonal cell lines <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          expressing TbCE1 dsRNA grown absence (lanes 1 3) presence (lanes 2 and <br />                                          4) 24 hr total cell extracts assayed enzyme-GMP complex formation. (B) Same <br />                                          assay panel (A), TbCGM1 downregulated (lanes 2 4). (C) Cell <br />                                          densities measured 24, 48, 72, 90 96 hr induction RNAi tetracycline in <br />                                          clonal cell line expressing double-stranded RNA TbCE1 TbCGM1. Since the <br />                                          growth curves uninduced cells essentially identical wild-type cells (wt), are <br />                                          represented one. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                            Page 14 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Fig. 3. <br />                                          TbCE1 downregulation RNAi. (A) Total RNA isolated uninduced (lanes 1&#226;&#8364;&#8220;5) or <br />                                          TbCE1-RNAi cells induced 8 days (lanes 6&#226;&#8364;&#8220;10) immunoprecipitated anti-m7G <br />                                          anti-TMG antibodies input (I), supernatant (S) pellet (P) fractions assayed <br />                                          primer extension U2 snRNA (U2) U3 snoRNA (U3). (B) The TbCE1 RNAi cell <br />                                          line induced tet indicated number days (lanes 2&#226;&#8364;&#8220;4) total RNA was <br />                                          assayed primer extension SL intron-specific primer. The position fully-modified <br />                                          (fm SL) hypomodified (hm SL) SL RNA indicated. RNA isolated sinefungin- <br />                                          treated cells served control hypomodified SL RNA (lane 1). A U6 snRNA-specific <br />                                          primer included reactions control RNA amounts quality. <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Fig. 4. <br />                                          TbCGM1 depletion affects SL RNA capping. (A) Total RNA isolated uninduced (lanes <br />                                          1&#226;&#8364;&#8220;5) TbGM1-RNAi cells induced 2.5 days (lanes 6&#226;&#8364;&#8220;10) immunoprecipitated with <br />                                          anti-m7G anti-TMG antibodies input (I), supernatant (S) pellet (P) fractions <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          assayed primer extension U2 snRNA (U2) U3 snoRNA (U3). (B) The <br />                                          TbCGM1 RNAi cell line induced tet indicated number days (lanes 2&#226;&#8364;&#8220;5) and <br />                                          total RNA assayed primer extension SL intron-specific primer. The position <br />                                          fully-modified (fm SL) hypomodified (hm SL) SL RNA indicated. RNA isolated <br />                                          sinefungin-treated cells served control hypomodified SL RNA (lane 1). A U6 <br />                                          snRNA-specific primer included reactions control RNA amounts quality. <br />                                          (C) Total RNA isolated uninduced cells (lanes 1&#226;&#8364;&#8220;3) cells depleted TbCgm1 <br />                                          2.5 days (lanes 4&#226;&#8364;&#8220;6) immunoprecipitated anti-m7G antibodies input (I), <br />                                          supernatant (S) pellet (P) fractions assayed primer extension SL intron- <br />                                          specific primer. A U6-snRNA-specific primer included reactions control for <br />                                          immunoprecipitation specificity (U6). (D) Primer extension total RNA an <br />                                          oligonucleotide complementary nt 110 131 SL RNA. The position Y-structure <br />                                          intermediate (Y) full-length SL RNA (SL) indicated. RNA isolated sinefungin- <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                            Page 16 <br />  <br />  <br />                                          treated cells used control trans-splicing inhibition (lane 1). An oligonucleotide <br />                                          complementary U2 snRNA included loading quality control (U2). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Mol Biochem Parasitol. Author manuscript; available PMC 2009 July 6. <br />                             Ruan et al.                                                                                          Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Fig. 5. <br />                                          Newly-synthesized RNA labeled [&#206;&#177;-32P]GTP [&#206;&#177;-32P]ATP prepared in <br />                                          permeabilized cells, spliced leader RNA gel purified analyzed tobacco acid <br />                                          pyrophosphatase P1 digestion. In panels, SL RNA isolated uninduced <br />                                          cells (lane 1), cells depleted TbCGM1 1 day (lane 2) 2.5 days (lane 3). (A) <br />                                          The cap nucleotide visualized digesting [&#206;&#177;-32P]GTP-labeled SL RNA tobacco <br />                                          acid pyrophosphatase TLC separation. The positions nonradioactive m7Gp Gp <br />                                          markers (visualized UV light) indicated. (B) [&#206;&#177;-32P]ATP-label <br /> </body></html>