<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Bai et al. BMC Developmental Biology 2010, 10:7 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Disruption zebrafish cyclin G-associated kinase <br /> (GAK) function impairs expression Notch- <br /> dependent genes neurogenesis and <br /> causes defects neuronal development <br /> Ting Bai&#226;&#8364;&#160;, Jamie L Seebald&#226;&#8364;&#160;, Kyu-Eui Kim, Hong-Mei Ding, Daniel P Szeto*, Henry C Chang* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: The J-domain-containing protein auxilin, critical regulator clathrin-mediated transport, been <br />   implicated Drosophila Notch signaling. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> ask role auxilin conserved auxilin has <br />   additional roles development, investigated functions auxilin orthologs zebrafish. <br />   Results: Like mammals, zebrafish distinct auxilin-like molecules, auxilin cyclin G-associated kinase <br />   (GAK), differing domain structures expression patterns. Both zebrafish auxilin GAK functionally <br />   substitute Drosophila auxilin, suggesting overlapping molecular functions. Still, not <br />   completely redundant, morpholino-mediated knockdown ubiquitously expressed GAK increase <br />   specification neuronal cells, known Notch-dependent process, decrease expression Her4, a <br />   Notch target gene. Furthermore, inhibition GAK function caused elevated level apoptosis neural tissues, <br />   resulting severe degeneration neural structures. <br />   Conclusion: In support notion endocytosis plays important roles Notch signaling, inhibition of <br />   zebrafish GAK function affects embryonic neuronal cell specification Her4 expression. In addition, analysis <br />   suggests zebrafish GAK functions development neural tissues: early Notch- <br />   dependent role neuronal patterning late role maintaining <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>survival</span> neural cells. <br />  <br />  <br /> Background                                                                           unrelated E3 ubiquitin ligases, neuralized (neur) and <br /> The conserved Notch pathway participates diverse                                  mind bomb (dMib), append ubiquitin DSL (Delta, <br /> aspects animal development, implicated                               Serrate, Lag2) ligands [6-13]. Epsin/lqf (liquid facets) <br /> in human diseases cancers [1-3]. Notch encodes                                 recruits ubiquitinated DSL ligands cla- <br /> transmembrane receptor, which, ligand binding,                                  thrin-coated vesicles (CCVs) [14-18]. The scission of <br /> undergoes proteolytic processing releases intra-                              ligand-containing CCVs plasma mem- <br /> cellular fragment capable acting transcription                               brane critical Notch activation, disruption <br /> co-regulator. As Notch ligands (also trans-                             dynamin function causes Notch-like defect <br /> membrane proteins) widely expressed, activities                            [4,19,20]. <br /> need tightly regulated. One important regula-                               Another relevant factor Drosophila Notch ligand <br /> tion appears ligand internalization, plays                             endocytosis J-domain protein auxilin [21,22]. First <br /> critical role activating Notch receptors [4,5].                                   identified mammals, auxilin known cooperate <br />   Notch ligand internalization utilizes ubiquitin-                                Hsc70 mediating disassembly clathrin tris- <br /> dependent endocytic pathway, structurally                                     kelia coat proteins newly formed CCVs in <br />                                                                                      vitro [23]. The mammalian genomes contain dis- <br /> * Correspondence: szetod@purdue.edu; hcchang@purdue.edu                              tinct auxilin-related genes: auxilin GAK, differing in <br /> &#226;&#8364;&#160; Contributed equally                                                                presence Ark (actin-related kinase) family <br /> Department Biological Sciences, Purdue University, 915 W. State St., West <br /> Lafayette, Indiana 47907-2054, USA                                                   kinase domain tissue distributions [23-25]. <br />  <br />                                         &#194;&#169; 2010 Bai et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                         Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                         medium, provided original work properly cited. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                              Page 2 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> GAK contains Ark domain N-terminus          suitable purpose importance of <br /> is ubiquitously expressed. In contrast, auxilin lacks     Notch ligand endocytosis demonstrated during <br /> kinase domain expression appears neuro-         embryonic neural patterning [9]. We zebra- <br /> nal. However, expression auxilin non-neuronal       fish, like mammals unlike Drosophila, contains both <br /> cells recently demonstrated [26]. Besides            auxilin GAK. Zebrafish auxilin GAK inter- <br /> uncoating, functions endocytosis         changeable abilities substitute dAux dur- <br /> have recently suggested auxilin family proteins,     ing Drosophila Notch signaling, suggesting they <br /> including facilitating clathrin exchange coated-pit    share cellular functions. However, differ- <br /> formation [27,28], participating pit constriction [29]     ent expression patterns development, suggesting <br /> and preventing precipitous assembly clathrin cages         paralogs completely redundant. <br /> [26,30]. Furthermore, GAK implicated cla-         Morpholino-mediated knockdown GAK function dur- <br /> thrin-mediated trafficking trans Golgi network       ing embryogenesis caused increase formation <br /> (TGN) [25,31].                                                neuronal cells decrease expression a <br />    While unclear aforementioned cellu-     Notch target gene, supporting hypothesis the <br /> lar functions relevant physiological con-      role auxilin family proteins Notch signaling con- <br /> ditions, mutations sole Drosophila auxilin             served. Furthermore, showed embryos deficient <br /> ortholog (dAux) clearly disrupt Notch-dependent       GAK function higher level programmed cell <br /> processes [21,22]. One process patterning      death neural tissues, suggesting GAK required <br /> neural tissues, cells destined         survival neuronal cells. <br /> neurons send Notch-mediated signal prevent neigh- <br /> boring cells adopting neuronal fate. In absence      Results <br /> of lateral inhibition, supernumerary neuronal cells      The zebrafish genome contains GAK auxilin <br /> are generated, forming so-called neurogenic pheno-        To examine roles auxilin-related genes during <br /> type [2]. Consistent notion dAux partici-       vertebrate development, sought identify aux- <br /> pates Notch signaling, excessive neurons seen      ilin GAK zebrafish genome. Database search <br /> both embryonic CNS larval eye discs            revealed that, like mammals, zebrafish genome con- <br /> dAux mutants [21,32]. Mosaic analysis showed         tains GAK (zGAK, XP_001919224) auxilin <br /> function dAux Notch signaling required        (zAux, XP_001336673) orthologs, located chromo- <br /> the signal-sending cells, suggesting role    23 6 respectively. This presence dis- <br /> ligand internalization [32,33].                               tinct auxilin-related orthologs appears feature <br />    Given high degree conservation Notch         shared vertebrates, chicken (Gallus gal- <br /> pathway, reasonable expect inhibition       lus, XP_424873 XP_422527) pufferfish (Tetrao- <br /> auxilin function animal systems cause          don nigroviridis, CAG08624 CAG11595) genomes <br /> Notch-like defects. However, multiple distinct endocy-     contain GAK auxilin. In contrast, arthropods <br /> tic pathways exist higher metazoans [34], possible   Drosophila (melanogaster virilis, <br /> that different endocytic pathways functionally sub-       XP_002058717), honeybee (Apis mellifera, XP_396906) <br /> stitute Notch ligand endocytosis.          flour beetle (Tribolium castaneum, XP_967193) <br /> Inhibition auxilin function <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> C.      GAK. C. elegans auxilin- <br /> elegans shown disrupt receptor-mediated            related gene, lacks kinase PTEN <br /> uptake yolk proteins, Notch-related defects         homologous region [35]. <br /> were reported [35]. RNAi-mediated reduction GAK              The zGAK locus encodes polypeptide 1278 amino <br /> function mammalian cells appeared deregulate            acids and, like GAK orthologs, contains N-term- <br /> EGF signaling promote tumorigenesis [36]. Tissue-         inal Ark kinase, PTEN domain, CBM region C- <br /> specific inactivation mouse GAK embryonic           terminal J domain (Figure 1A). Alignment protein <br /> development caused severe degenerations brain, liver,      sequences shows zGAK 65% identical human <br /> and skin [37], unclear         GAK overall, sequence conservations concen- <br /> defects disruption Notch function.           trated kinase, PTEN J-domain. The kinase, <br /> Thus, clear auxilin family proteins       PTEN J-domain zGAK 82.4%, 84.7%, and <br /> are important animal viability, func-       78.8% identical corresponding domains human <br /> tion Notch ligand endocytosis evolutionarily con-       GAK respectively. In contrast, CBM domains of <br /> served requires additional investigations.                    zGAK human GAK divergent, sharing only <br />    We used zebrafish assess roles      31.7% amino acid sequence identity. <br /> auxilin-dependent endocytosis animal development.            The zAux protein, hand, 873 amino <br /> Besides versatile model organism, zebrafish        acid long polypeptide, contains PTEN region, a <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                  Page 3 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 1 The genomic organizations protein sequences zebrafish GAK auxilin. (A) The zGAK locus contains 28 exons and <br />  spans 78 kb genomic DNA. The corresponding cDNA contains 3834 bp ORF, encoding polypeptide 1278 amino acids Ark <br />  kinase domain (blue), PTEN-homologous region (orange), clathrin-binding motif (CBM), J-domain (green). Two morpholinos, GAK-MO1 <br />  GAK-MO2, (red boxes) designed disrupt splicing zGAK mRNA blocking splice acceptors exon3 exon19, <br />  respectively. The amino acids deleted GAK-MO1 injection boxed red. The premature stop codon (after Lys679) generated GAK-MO2 <br />  injection indicated red line. (B) The zAux locus smaller, contains 17 exons spans 40 kb genomic DNA. The zAux <br />  protein, encoded 2619 bp ORF, contains PTEN-related region (orange), CBM, J-domain (green). (C) A schematic drawing comparing <br />  domain composition zebrafish auxilin-related genes. The kinase, PTEN, J-domains represented blue, orange, green <br />  boxes, respectively. <br />  <br />  <br />  <br />  <br /> CBM J-domain, lacks N-terminal kinase                      zGAK, similar human mouse auxilin <br /> (Figure 1B). The protein sequence zAux 59.3%                       proteins (Figure 2A), suggesting divergence of <br /> identical overall human auxilin. Again, con-                  GAK auxilin occurred prior divergence of <br /> servations higher PTEN J-domain,                   fish mammals vertebrate evolution. <br /> these domains share 67.5% 94.5% amino acid iden-                        In addition similarities protein sequences, the <br /> tity respectively. These domains zAux highly                 intron/exon organizations auxilin family genes <br /> related PTEN J-domain zGAK (50.8%                      highly conserved. First, human, mouse, zeb- <br /> and 89.4% protein sequence identity respectively), sug-                  rafish GAK loci contain identical number (27) of <br /> gesting zAux zGAK derived com-                       introns. The zAux locus appears lost intron <br /> mon ancestral gene. However, zAux similar                    CBM evolution, contains 16 introns <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                       Page 4 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 2 zGAK zAux arose gene duplication. (A) A phylogenetic analysis amino acid sequences auxilins GAKs without <br />  kinase domains. The alignment performed using Geneious software (Biomatters). (B) Alignment intron positions PTEN region <br />  auxilin GAK orthologs. The residues shared shaded yellow residues conserved mammals vertebrates are <br />  shaded orange. The presence intron indicated solid line, phase indicated number inside triangle. ^ indicates <br />  intron conserved honeybee Drosophila, * indicates intron conserved fly honeybee. Am: <br />  Apis mellifera, Dm: Drosophila melanogaster, Gg: Gallus gallus, Hs: Homo sapiens, Mm: Mus musculus, Dr: Danio rerio. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                 Page 5 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> while human mouse auxilins 17. Further-                    zGAK transfected cells, depending expression <br /> more, conserved domains, positions                   levels. In cells expressing low levels GFP-zGAK or <br /> phases (0, 1, 2 codons) introns               GFP-zAux (marked asterisks), GFP signals were <br /> precisely aligned. For example, PTEN homologous                     cytosolic slightly enriched near perinuc- <br /> regions zebrafish, mouse, human GAK aux-                     lear regions. These perinuclear zGAK- zAux-posi- <br /> ilin loci contain 8 introns (Figure 2B). These introns,             tive structures showed overlaps clathrin, most <br /> although different nucleotide lengths, located                likely representing TGN. <br /> the positions coding region identical                     In cells expressing high levels GFP-zGAK and <br /> phases. This high degree conservation intron/exon                 GFP-zAux, cytosolic staining enrichments <br /> organization strongly bolsters notion GAK                  near perinuclear regions seen. How- <br /> auxilin derived common ancestral gene                       ever, cells contained large GFP-positive <br /> through gene duplication.                                               aggregates (indicated solid arrows arrowheads) <br /> Overexpression zGAK zAux causes clathrin                          intensely clathrin-positive. In untransfected <br /> aggregations                                                            cells, clathrin staining vesicular appearance <br /> To ask zebrafish auxilin family proteins similar                cytosol, appeared depleted <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> cellular functions respective mammalian coun-                  structures aggregate-containing cells. <br /> terparts, determined subcellular localizations                We noticed zGAK capable caus- <br /> zGAK zAux. Both zGAK zAux tagged                           ing aggregates zAux, intense GFP-positive <br /> with GFP N-termini, placed control                  clathrin-containing structures readily seen in <br /> a CMV promoter pCS2, transiently expressed                    cells expressing milder levels GFP-zGAK. It has <br /> HeLa cells. These GFP-tagged fusion proteins func-                  shown endogenous GAK predominantly <br /> tional, ectopic expression GFP-zGAK                     cytosolic shows elevated associations TGN <br /> GFP-zAux Drosophila restore neurogenic                     [24,25,31], over-expression mammalian auxilin <br /> defect lethality caused dAux mutations (see                  family proteins HeLa cells cause formation <br /> below). To reveal clathrin-positive structures, cells             clathrin-containing &#226;&#8364;&#339;granules&#226;&#8364;? [24,38]. Thus, our <br /> were stained mouse monoclonal antibody                      results showed over-expressed zGAK zAux <br /> against clathrin heavy chain.                                           localized similarly cells. In addition, <br />   As shown Figure 3, qualitatively different GFP pat-                localizations similar respec- <br /> terns seen pCS2-GFP-zAux pCS2-GFP-                          tive mammalian homologs. <br />  <br />  <br />  <br />  <br />  Figure 3 The subcellular distributions zebrafish auxilin family proteins. Spinning disc confocal micrographs HeLa cells transfected <br />  (A) pCS2-GFP-zGAK (B) pCS2-GFP-zAux. The cells stained clathrin heavy chain (red). The low GFP-expressing cells are <br />  indicated asterisks. The large GFP- clathrin-positive aggregates indicated arrows. Scale Bar, 10 &#206;&#188;m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                           Page 6 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> zGAK zAux functionally interchangeable                            neuronal differentiation Drosophila eye discs for <br /> rescuing dAux defects                                                        animal survival. <br /> To determine intrinsic functional dif-                       In wild-type eye discs, regular arrays photoreceptor <br /> ferences zGAK zAux, compared                            clusters revealed aElav antibody, which <br /> abilities rescuing extra photoreceptor defect                     labels nuclei neuronal cells [39]. In contrast, <br /> the lethality caused dAux mutations. N-terminally                         excessive disorganized Elav-positive cells seen <br /> GFP-tagged zGAK zAux placed UAS                               dAux F956X mutant tissues (Figure 4A) [33]. This <br /> control, transgenic flies carrying constructs                      defect extra Elav-positive cells rescued an <br /> were generated. We reasoned zGAK zAux                            Act5C-GAL4-driven expression full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> dAux <br /> function similarly clathrin-mediated transport,                       (Figure 4B) [33]. Similar dAux, UAS-GFP-zGAK <br /> both able supplant dAux function                         UAS-GFP-zAux, driven Act5C-GAL4, could <br />  <br />  <br />  <br />  <br />  Figure 4 Both zGAK zAux substitute Drosophila auxilin. Fluorescent micrographs Drosophila eye imaginal discs stained with <br />  aElav antibody, labels nuclei neuronal cells (blue). The expressions zebrafish Drosophila auxilin genes shown in <br />  green. Homozygous dAuxF956* mutant tissues indicated absence membrane-associated mRFP (red). Regions containing <br />  excessive Elav-positive cells indicated arrows. All flies carry Act5C-GAL4, UAS-FLP second chromosome, relevant <br />  genotypes include: (A) FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP, (B) UAS-dAux-GFP; FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP, (C) UAS- <br />  GFP-zGAK; FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP, (D) UAS-GFP-zAux; FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP, (E) UAS-GFP-zGAK&#206;&#8221;J; <br />  FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP, (F) UAS-GFP-zAux&#206;&#8221;J; FRT5-5Z3515, dAuxF956*/FRT5-5Z3515, GMR-src-mRFP. Scale Bar, 50 &#206;&#188;m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                               Page 7 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> restore normal Elav staining <span id='am-475' about='xsp:pattern' typeof='owl:Thing'>pattern</span> dAuxF956X         zGAK mRNA (Figure 6A). A control antisense mor- <br /> mutant clones (Figure 4C &amp;4D). We previously             pholino oligonucleotide (GAK-MO1C), modi- <br /> shown lethality dAux mutants res-        fied sequence GAK-MO1 nucleotide <br /> cued ubiquitous expression full-length dAux           changes, included. Embryos injected 8.0ng <br /> [33]. Similarly, Act5C-GAL4-driven GFP-zGAK               GAK-MO1 one-cell stage (hereafter referred as <br /> GFP-zAux restore viability dAux             GAK-MO1 morphants) consistently smaller eyes, <br /> mutant flies. These results suggest zGAK zAux        enlarged hindbrain, thinner yolk extension, weak <br /> can functionally substitute dAux.                    circulation 36 48 hpf (85%, n = 100, below). In <br />    Deletion J-domain Drosophila auxilin        contrast, injection higher of <br /> known render non-functional [32,33]. To ask           GAK-MO1C caused developmental defects in <br /> whether J-domain essential zAux           embryos (100%, n = 100). To ensure splicing on <br /> zGAK rescue dAux, UAS-GFP-zGAK DelJ UAS-               zGAK mRNA disrupted, RNAs isolated GAK- <br /> GFP-zAuxDelJ generated. Unlike full-length         MO1 morphants 6 hpf analyzed RT-PCR. As <br /> counter parts, J-deletions failed rescue extra   shown Figure 6A, injection GAK-MO1 resulted <br /> Elav-positive cell <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Figure 4E &amp;4F)          transcript lacking exon 3, generating in-frame <br /> lethality, suggesting that, like dAux, J-domain cri-   deletion kinase domain (Figure 1A). <br /> tical functions auxilin family proteins.       Morphologically, development GAK-MO1 mor- <br /> zGAK zAux differentially expressed             phants appeared normal segmentation stage. <br /> embryonic development                                         At 10-somite stage, GAK-MO1 morphants displayed <br /> Although known mammalian GAK auxilin           cell degeneration eyes, noticeably <br /> are expressed different tissues [24], temporal       opaque (data shown). At 14-somite stage, degen- <br /> and spatial expressions development           erating cells persisted eyes, began appear the <br /> been fully investigated. To better understand             hypothalamic thalamic regions brain the <br /> requirement zGAK zAux, examined               hindbrain (Figure 7E 7F), continued increase <br /> expression various stages zebrafish development         remaining duration somitogenesis. In contrast, <br /> using whole-mount situ hybridization. Zygotic expres-      regions uninjected embryos were <br /> sions zGAK zAux, maternally            transparent (Figure 7A 7B). In cases, the <br /> expressed (data shown), detected diverse tis-     degeneration spread developing neural tube <br /> sues, prominently associated neural        GAK-MO1 morphants. It noteworthy cell <br /> tissues. Between 8- 15-somite stages, zGAK             degeneration observed neuroectodermal tis- <br /> broadly expressed dorsal region embryo,         sues, non-neural tissues noto- <br /> and expression zAux appeared restricted       chord somites. From 24 28 hpf, thinner yolk <br /> the bilateral cell clusters dorsal         extension weak blood circulation observed in <br /> embryo (Figure 5A-F). As control, hybridization using       GAK-MO1 morphants, compared uninjected <br /> sense probes did yield detectable signals (data       embryos stages (Figure 7C, D, G, 7H). <br /> shown). At 19-somite stage, zGAK expressed          At 36 hpf, hindbrain tectal ventricles of <br /> broadly entire embryo, zAux predo-         GAK-MO1 morphants enlarged (Figure 7I-L), <br /> minantly hindbrain neurons, spinal chord neu-        embryos retained ability to <br /> rons, otic vesicles (Figure 5G 5H). At 24-hour        respond touch (data shown). <br /> post fertilization (hpf), zGAK enriched brains,          To ensure phenotypes caused the <br /> eyes, otic vesicles, vasculature, zAux expression     disruption zGAK, tested binding specificity of <br /> was detected hindbrain neurons, spinal chord neu-          GAK-MO1. The complimentary target sequence of <br /> rons, otic vesicles posterior otic capsules (Figure 5I-   GAK-MO1 placed mCherry reporter <br /> N). The fact expression patterns zGAK         gene [40], resulting fusion (GAK-MO1-mCherry) <br /> zAux did coincide completely suggests           cloned pCS2 (Figure 6C). Similarly, compli- <br /> have overlapping distinct functions                mentary sequence GAK-MO1C placed front <br /> development.                                                  mCherry cloned pCS2 control. Using <br /> Inhibiting zGAK function causes neural-specific cell          constructs, GAK-MO1-mCherry GAK-MO1C- <br /> degeneration                                                  mCherry RNAs transcribed vitro injected <br /> Because structural conservation GAK genes       GAK-MO1 GAK-MO1C morpholi- <br /> from Drosophila human, decided focus              nos one-cell stage embryos. Injected embryos 6 <br /> zGAK function embryonic development. To inhi-          hpf analyzed protein expression of <br /> bit zGAK, designed antisense morpholino oligo-          mCherry. As shown Figure 6B, GAK-MO1, not <br /> nucleotide (GAK-MO1) disrupt normal splicing           GAK-MO1C, capable blocking translation of <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                              Page 8 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 5 The expression patterns GAK auxilin zebrafish embryonic development. (A, D) Lateral, (B, E) dorsal, (C, F) <br />  anterior views wild-type embryos 15-somite stage. (A-C) zGAK expressed broadly hindbrain (B), forebrain eyes (C). (D-F) <br />  zAux expressed neural tissues described main text. (G, H) Dorsal views 19-somite stage embryos. (G) zGAK still <br />  ubiquitously expressed (H) zAux remains specific bilateral stripes neural cells. (I, K, L, N) Lateral (J, M) dorsal views 24 hpf <br />  embryos. (I-K) zGAK seen brain, vasculature otic vesicles. (L-N) zAux remains concentrated bilateral stripes neural cells. Panels K <br />  N close-up views posterior regions embryos shown (I) (L), respectively. In images, anterior left, and <br />  lateral views, dorsal up. ey, eye; is, intersomitic vessel; lg, lateral line ganglion; nt, neural tube; ot, otocyte; ov, otic vesicle; sag, <br />  statoacoustic ganglion; Scn, Spinal cord neuron; tg, trigeminal ganglion; v, vasculature; vd, ventral diencephalon. Scale Bar, 100 &#206;&#188;m. <br />  <br />  <br />  <br />  <br /> GAK-MO1-mCherry. Conversely, GAK-MO1                                    GAK gene (Figure 1A). RT-PCR analysis showed that <br /> effect expression GAK-MO1C-mCherry.                                  injection GAK-MO2 caused retention intron <br /> These results demonstrate high degree binding spe-                        19, resulting premature stop codon PTEN <br /> cificity efficacy GAK-MO1 intended target.                       domain zGAK transcripts (Figure 1A Addi- <br />   To phenotypes GAK-MO1                               tional file 1: Figure S1A). Embryos injected 6.0ng <br /> morphants off-target effects, second                         GAK-MO2 exhibited phenotypes those <br /> morpholino antisense oligonucleotide (GAK-MO2)                             observed GAK-MO1 morphants (90%, n= 85; Addi- <br /> designed block splicing different region                        tional file 1: Figure S1B C). Taken together, these <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                           Page 9 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 6 GAK-MO1 specifically disrupts zGAK splicing. (A) RT-PCR analysis total RNA extracted uninjected GAK-MO1 morphant <br />  embryos. From normally spliced zGAK mRNA, reaction primers complementary exon 2 4 (2/4) yield band 178 bp, and <br />  reaction primers complementary exon 18 20 (18/20) yield band 275 bp. While 275 bp band unaffected by <br />  GAK-MO1 injection, level 178 bp band reduced new band 118 bp appeared GAK morphants. Sequence analysis this <br />  118 bp band showed exon 2 spliced exon 4, resulting in-frame deletion kinase domain. (B) The target sequences of <br />  GAK-MO1 GAK-MO1C placed mCherry cloned pCS2. In vitro transcribed mRNAs co-injected morpholino <br />  one-cell stage embryos. Bright field fluorescent images injected embryos 6 hpf shown. Scale Bar, 200 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />  Figure 7 Cell degeneration phenotype GAK morphant embryos development. (A, E) Lateral (B, F) dorsal views (A, B) <br />  uninjected (E, F) GAK morphant embryos 14-somite stage. Visible degeneration eyes (E) forebrain regions (F) GAK <br />  morphants indicated white arrowheads. (C, G) Lateral (D, H) dorsal views (C, D) uninjected (G, H) GAK morphant embryos 24 <br />  hpf. At stage, increased cell degeneration brain (arrowheads) thinner yolk extension GAK morphants apparent. (I-L) <br />  Lateral views (I, J) uninjected (K, L) GAK morphant embryos 36 hpf. In lateral views, anterior left dorsal up. ey, <br />  eye; Hb, hindbrain; nt, neural tube; no, notochord; ov, otic vesicles; ye, yolk extension. Scale Bar, 100 &#206;&#188;m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                         Page 10 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> results strongly suggest developmental defects                      GAK morphants seen neural tissues, not <br /> of GAK-MO1 GAK-MO2 morphants resulted                               non-neural tissues. Thus, appears good <br /> the specific inhibition zGAK function.                                    correlation onset extent phe- <br /> Disrupting zGAK function causes excessive neural-specific                    notypes, suggesting increase pro- <br /> programmed cell death                                                        grammed cell death basis cell degeneration <br /> To define extent understand cause                        GAK morphants. <br /> of cell degeneration GAK morphants, per-                           Reduction zGAK disrupts development specific <br /> formed situ TUNEL assay detect apoptotic cells.                        brain regions <br /> At 14-somite stage, slight increase apoptotic                       To explore role zGAK brain patterning, GAK <br /> cell death observed brain regions GAK                          morphants stained krox20, fibroblast growth <br /> morphants, compared control embryos (Figure                        factor 8 (fgf8) sonic hedgehog (shh), expres- <br /> 8A-D). At 24 hpf, GAK morphants displayed signifi-                         sions delineate various regions brain. Krox20, a <br /> cantly higher level cell death brain,                   zinc finger-containing transcription factor, expressed <br /> in neural tube (Figure 8E-H). Similar afore-                      specifically hindbrain [41]. At 12-somite <br /> mentioned cell degeneration, excessive apoptosis                          stage, pattern level krox20 expression <br />  <br />  <br />  <br />  <br />  Figure 8 Programmed cell death wild-type GAK morphant embryos development. (A, C) Lateral (B, D) dorsal views of <br />  TUNEL-stained (A, B) wild-type (C, D) GAK morphant embryos 14-somite stage. At stage, control embryos display no <br />  detectable apoptosis, GAK morphants low level programmed cell death (indicated black arrowheads). (E, G) Lateral (F, <br />  H) dorsal views TUNEL-stained (E, F) wild-type (G, H) GAK morphants 24 hpf. While control animal exhibits low level apoptosis <br />  anterior brain posterior body regions (arrowheads), GAK morphants exhibit high level apoptotic cell death brain, as <br />  neural tube (arrowheads). However, compared control wild-type embryo, obvious increase apoptosis observed the <br />  posterior region GAK morphants (indicated arrows). In lateral views, anterior left dorsal up. ey, eye; Hb, hindbrain; <br />  MHB, mid-hindbrain boundary; nt, neural tube; s, somites; tb, tailbud. Scale Bar, 100 m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                               Page 11 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> appeared normal GAK morphants (Figure 9A                Notch pathway reduced stage. Similar <br /> 9B). At 24 hpf, pattern krox20 expression GAK        HuC, decrease Her4 expression still <br /> morphants remained similar seen wild type,          observed 10-somite stage GAK morphant embryos <br /> although stripes krox20 rhombomeres          (Additional file 2: Figure S2E-H). At 24 hpf, Her4 <br /> 3 5 appeared closer (Figure 9C 9D). This          expression GAK morphants control embryos was <br /> suggests that, overall patterning brain       significantly different (Figure 10I 10J), suggest- <br /> regions significantly affected, rostral hindbrain   ing GAK required Notch signaling, <br /> is reduced size absence zGAK function.            GAK auxilin redundant Notch signaling <br />    Fgf8, expressed forebrain, MHB (midbrain-        stage. Nevertheless, alterations expres- <br /> hindbrain boundary) mesodermal derivatives           sion HuC Her4 8- 10-somite stages <br /> that include somites tailbud, shown reg-       showed reduction GAK function impair <br /> ulate forebrain development establish segmental        Notch activity zebrafish. <br /> identity hindbrain [42]. At 12-somite stage, <br /> the expression fgf8 GAK morphants appeared            Discussion <br /> be normal (data shown). In contrast, GAK mor-              Zebrafish, like mammals invertebrates, two <br /> phants 18-somite stage exhibited higher levels       distinct auxilin-related genes. Like mammalian <br /> fgf8 expression MHB, optic stalks forebrain,        counterparts, zebrafish GAK auxilin differ the <br /> suggestive requirement GAK repressing fgf8         presence N-terminal kinase respective <br /> expression regions (Figure 9E-H). No alteration       expression patterns. zGAK kinase domain is <br /> in fgf8 expression somites tailbud regions      ubiquitously expressed embryonic development. <br /> observed (Figure 9E 9F), consistent notion        In contrast, expression zAux, ortholog with- <br /> that zGAK function required predominantly neural         kinase domain, appears predominantly the <br /> tissues, non-neural tissues. At 18-somite       neuronal cells. There differences the <br /> stage, shh mRNA detected ventral midbrain,          protein sequence level. For instance, zGAK, <br /> hypothalamus, telencephalon notochord [43]. In             zAux, contains sequence FGDL the <br /> contrast fgf8, shh expression appeared normal        amino acid position 950, matches perfectly the <br /> GAK morphants stage (Figure 9I 9J).                consensus &#207;&#710;G[PDE][&#207;&#710;LM] (&#207;&#710; aromatic residue). <br /> Reduction zGAK impairs Notch signaling                      This motif, conserved mammalian GAKs, been <br /> To ask zGAK, like dAux, role Notch signal-         shown mediate interactions AP1 adaptor in <br /> ing, examined expression HuC Her4                Golgi-lysosomal trafficking [31]. Taken together, these <br /> mRNAs GAK morphants. HuC, zebrafish homo-               structural differences GAK auxilin suggest <br /> log Elav, earliest markers neuronal       molecular functions diverged during <br /> cells [44]. In 8-somite stage wild-type embryos, HuC           evolution. It notable inhibiting zGAK function <br /> transcripts detected bilateral stripes neuronal      causes increase apoptotic cell death neu- <br /> cells anterior-posterior axis [45]. In GAK mor-      roectodermal tissues, GAK auxilin both <br /> phants similar stage, pattern expres-       expressed. This inability zAux compensate the <br /> sion similar, HuC-positive cells seen            zGAK knockdown argue GAK function <br /> (Figure 10A 10B), suggesting additional cells         unique auxilin. However, formally <br /> had adopted neuronal fate. This increase HuC stain-         exclude possibility neural cells a <br /> ing persisted GAK morphant embryos 10-somite             higher demand functions auxilin-related <br /> stage (Additional file 2: Figure S2A-D). Interestingly,     genes. In scenario, functions genes <br /> 24 hpf, GAK morphants appeared fewer HuC-              needed prevent onset apoptosis; <br /> positive cells forebrain midbrain regions (Fig-     therefore, inhibiting GAK sufficient to <br /> ure 10C-F). However, important note          induce extra neural cell deaths. <br /> reduction HuC-positive cells stage coincides          Despite structural differences GAK and <br /> with onset elevated cell death neural tis-       auxilin, clear paralogs over- <br /> sues (see above).                                              lapping molecular functions. Both GAK auxilin are <br /> Her4, homolog Drosophila E(spl) (Enhancer split)       required receptor-mediated endocytosis HeLa <br /> gene, known target Notch pathway parti-        cells, indicating act process [26]. <br /> cipates specification neuronal cells [46]. In 8-         Consistent this, showed subcellular <br /> somite stage wild-type embryos, Her4 expressed           localizations zGAK zAux HeLa cells simi- <br /> bilateral stripes cells [46]. Compared wild type,    lar, over-expression HeLa cells could <br /> level Her4 expression GAK morphants appeared             form clathrin-containing aggregates. More importantly, <br /> lower (Figure 10G 10H), suggesting output         showed over-expression zGAK or <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                         Page 12 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 9 Expression patterns regional brain markers wild-type GAK morphant embryos. (A-D) krox20 expression (A, C) <br />  uninjected (B, D) GAK morphant embryos (A, B) 12- 14-somite stage (C, D) 24 hpf. At 12- 14-somite stage, krox20 expression in <br />  rhombomeres 3 (R3) 5 (R5) control injected embryos appears comparable. However, 24 hpf, spacing R3 and <br />  R5 dramatically reduced GAK morphants (white arrowhead). (E, F) Lateral (G, H) dorsal views fgf8 expression (E, G) wild-type (F, <br />  H) GAK morphant embryos 18- 20-somite stage. The domains fgf8 expression forebrain mid-hindbrain boundary (MHB) are <br />  up-regulated GAK morphants (black arrows). (I, J) shh expression (I) wild-type (J) GAK morphants 18-somite stage. In lateral <br />  views, anterior left dorsal up. d, ventral diencephalon; f, floor plate; os, optic stalk; s, somite; t, telencephalon; dd, dorsal <br />  diencephalon; tb, tailbud. Scale Bar, 100 &#206;&#188;m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                           Page 13 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />  Figure 10 Expression patterns HuC Her4 wild-type GAK morphant embryos. (A, B) Close-up dorsal views HuC expression <br />  (A) wild-type (B) GAK morphant embryos 8-somite stage. Micrographs lower magnification shown insets. At this <br />  stage, cells appeared express HuC GAK morphant embryos, suggesting presence neural progenitor cells. (C-F) Lateral <br />  views HuC expression brain regions (C, E) wild-type (D, F) GAK morphant embryos 24 28 hpf. A comparison (C) (D) <br />  shows reduction HuC-positive cells forebrain midbrain GAK morphants. Similarly, comparison (E) (F) shows the <br />  disorganization reduction HuC-positive cells hindbrain GAK morphants. (G-J) Lateral views Her4 expression patterns (G, I) <br />  wild-type (H, J) GAK morphants (G, H) 8-somite stage (I, J) 24 28 hpf. At 8-somite stage, Her4 expression GAK morphant <br />  embryos significantly reduced, compared wild type. In contrast, expression Her4 brain wild-type GAK <br />  morphant embryos 24 28 hpf comparable. In panels, anterior left, lateral views, dorsal up. Fb, forebrain; <br />  Hb, hindbrain; MHB, mid-hindbrain boundary; Mb, midbrain; mbp, midbrain basal plate; R, rhombomeres; tb, tailbud; te, telencephalon; vd, <br />  ventral diencephalon. Scale Bar, 100 &#206;&#188;m. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                             Page 14 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> zAux Drosophila completely restore neuro-       results neural-specific cell degeneration. Using <br /> genic defects caused dAux, ability rescue     TUNEL staining, showed cell degeneration is <br /> dAux absolutely requires respective J-domains.         caused increased programmed cell death, suggesting <br /> Together, results suggest zGAK zAux       GAK role preventing apoptosis neural <br /> at partially redundant.                                cells. Thus, HuC-positive cells pre- <br />   Unlike zGAK dAux, zAux does N-             sent zGAK-deficient embryos 8- 10-somite <br /> terminal Ark kinase. Nevertheless, over-expression        stages, fewer HuC-positive cells expected at <br /> zAux Drosophila completely restore defects      later stages cell death. Indeed, pre- <br /> caused strong dAux allele. This entirely sur-    cisely observed, fewer HuC-positive cells were <br /> prising previous shown over-expression       seen GAK morphants 24 hpf. Interestingly, mind- <br /> of dAux &#206;&#8221;K , dAux kinase domain deleted,          bomb mutant embryos, Notch ligand endocytosis <br /> could rescue dAux phenotype [33]. In fact, over-         impaired, cell degeneration phenotype observed <br /> expression fragment consisting CBM J         (J.S., unpublished data). This, observation <br /> domains appears sufficient restore function     Her4 expression significantly reduced 24 <br /> of dAux Notch [32,33]. However, kinase domain         hpf, suggests role GAK maintaining neural <br /> does role GAK&#226;&#8364;&#8482;s function Notch kinase        <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>cell survival</span> Notch-independent. Taken together, <br /> domain-specific disruptions, point mutations       results suggest GAK distinct <br /> [33] morpholino-induced mis-splicing (MO1,           functions development neural tissues: an <br /> study), produce Notch-like phenotypes. Still, does        early role patterning neuronal cells later <br /> appear that, expressed high level, kinase      role maintaining survival neuronal cells. <br /> activity required functions auxilin        Furthermore, human GAK recently implicated <br /> family proteins.                                             susceptibility gene familial Parkinson disease [49], <br />   In Drosophila, auxilin shown participate    neurodegenerative phenotype observed GAK <br /> Notch signaling facilitating ligand internalization       morphants certainly supports conclusion. <br /> [21,22]. Given mammals vertebrates            It noteworthy phenotypes GAK mor- <br /> auxilin-related genes, known either, neither,   phants bear strong resemblance &#226;&#8364;&#339;space- <br /> or function Notch pathway. The similarities      head&#226;&#8364;? class zebrafish mutants [50]. These mutants, <br /> in domain structures expression patterns suggest         isolated large-scale screen, characterized by <br /> that GAK likely role Notch. How-        defects including cell degeneration eye the <br /> ever, inactivation GAK affects formation     brain regions, thinner yolk tube, weak blood circula- <br /> multiple tissues mouse [37], unclear       tion [50]. As genes responsible these <br /> these defects caused disrupted Notch signaling.      mutants determined, phenotypic simi- <br /> Here showed that, GAK morphants, number            larities suggest zGAK correspond of <br /> of HuC-positive cells appeared increased, defect analo-    them. If case, provide important <br /> gous neurogenic phenotype. Moreover,               clues understand functions genes in <br /> showed that, GAK morphants, expression             maintaining neuronal cell survival. <br /> Her4, known Notch target gene, reduced. Thus, <br /> while known zGAK participates       Conclusion <br /> ligand internalization, defects HuC Her4        Zebrafish, like mammals invertebrates, two <br /> expressions suggest GAK function required       distinct auxilin-related genes, auxilin GAK. These <br /> for Notch signaling zebrafish. These results provide      genes share molecular functions, not <br /> the evidence requirement GAK-           completely redundant, differentially expressed <br /> dependent endocytic pathway Notch signaling        development. Inhibition GAK function appears <br /> evolutionarily conserved. A requirement mind-         impair Notch signaling embryonic neural pat- <br /> bomb E3 ligase epsin Notch              terning. This, fact auxilin been <br /> demonstrated flies [7,11,12,15,16], fish [9], mouse   implicated Drosophila Notch signaling, suggests that <br /> [8,47,48], which, analysis zebrafish       Notch pathway regulated similar set endo- <br /> GAK, suggests Notch ligand internalization          cytic factors. In addition, showed inhibition of <br /> rely set endocytic genes.                     GAK function increases apoptosis neural tissues, sug- <br />   It demonstrated conditional removal     gesting GAK role promoting maintaining <br /> GAK function mouse brain development causes         survival neural cells. As GAK recently implicated <br /> significant loss neural tissues [37],        familial Parkinson disease [49], results pro- <br /> mechanism known. Likewise, depletion           vide useful model understanding cause <br /> GAK function zebrafish embryonic development          neurodegenerative disease. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                           Page 15 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br /> Methods                                                       For RT-PCR analysis, total RNAs extracted from <br /> Embryos morpholino oligoneucleotides injections         embryos using RNeasy Mini Kit (Qiagen) RT- <br /> All animal procedures reviewed approved     PCR reactions performed using OneStep RT-PCR <br /> Purdue Animal Care Use Committee (PUCAC #06-            Kit (Qiagen). Primers GTATGAGGCCCAGGATTTAG- <br /> 111-09). Adult fish embryos raised main-       GAAG GTCAGACTCTTCTTTACTGATGGAC <br /> tained 28.5&#194;&#176;C water. Embryos obtained     used examine splicing exon 3. Primers <br /> by natural spawning adult AB strain zebrafish. zGAK-     GTGCCCAGAAATGCCTCCACTGTC GCATAA- <br /> specific antisense morpholino oligonucleotides (GAK-        CAGGCTGTCGAACCAGGC used examine the <br /> MO1, GAK-MO2, GAK-MO1C, GAK-MO2C)                  splicing exon 19. <br /> purchased Gene Tools (Philomath, Oregon). At       HeLa cell manipulation microscopy <br /> one-cell stage, embryo injected approxi-      HeLa cells maintained DMEM, supplemented <br /> mately 1 nl volume morpholino using Picospritzer III   10% Fetal Bovine Serum standard conditions. <br /> (Parker Hannifin). Embryos collected appro-     To express GFP-zGAK GFP-zAux, 70% confluent <br /> priate stages [51] fixed 4% paraformaldehyde      HeLa cells 10cm dish transfected a <br /> phosphate-buffered saline (PBS) overnight 4&#194;&#176;C. Fixed     0.5ml cocktail, contained 10 &#206;&#188;g plasmid DNA, 25 <br /> embryos dechorionated, washed 3 times PBS,        &#206;&#188;l Fugene HD (Roche) serum-free DMEM. Cells <br /> and stored methanol -20&#194;&#176;C.                            harvested processed immunostaining 48 <br /> Drosophila genetics                                         hours transfection. Mouse aChc (Affinity BioRea- <br /> Flies raised 25&#194;&#176;C standard food. Mutant          gents) used 1:500, fluorescently-conjugated <br /> clones dAux F956X , strong allele nonsense      secondary antibodies (Molecular Probes) used at <br /> <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> deleting J-domain, generated pre-      1:100. All images collected using Olympus BX61 <br /> viously described [33]. For lethality assay, trans-   equipped Spinning Disc Confocal <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> pro- <br /> heterozygous combination dAuxF956X/dAuxL78H          cessed Photoshop (Adobe). <br /> used avoid potential unrelated lethal mutations   In situ hybridization TUNEL staining <br /> background. The dAuxL78H allele contains missense         Whole-mount situ hybridization performed using <br /> mutation kinase domain [33]. Immunostaining       digoxigenin-labeled antisense RNA probes visua- <br /> eye imaginal discs performed previously              lized using anti-digoxigenin Fab fragment conjugated <br /> described [52]. Mouse aElav 9F8A9 (DSHB, Iowa)          alkaline phosphatase (Roche) previously <br /> used 1:100.                                              described [54]. Riboprobes DNA tem- <br /> Molecular cloning                                           plates, linearized transcribed either <br /> A 4475 bps-long cDNA containing entire zGAK             SP6 T7 RNA polymerases. Embryos processed <br /> ORF pCCM114 obtained OpenBiosystems             hybridized previously described [54]. <br /> (ID 2504096). This particular clone muta-          Whole-mount situ TUNEL (terminal deoxynucleotide <br /> tions, including missense mutations Arg303, Tyr480,      transferase-mediated dUTP nick-end labeling) per- <br /> Asp614, <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> Gly1047. All      formed using AP (alkaline phosphatase) In Situ Cell <br /> repaired using QuickChange site-directed mutagen-       Death Detection Kit (Roche) previously described [55]. <br /> esis kit (Stratagene) resulting cDNA verified <br /> by sequencing. For zAux, Exelixis EST (ID 3410313,        Additional file 1: The phenotypes GAK-MO2 morphants are <br /> OpenBiosystems) containing partial zAux ORF (miss-         similar GAK-MO1 morphants. (A) RT-PCR analysis the <br />                                                              total RNA embryos injected GAK-MO2. Using RNA <br /> ing N-terminal 696 bps) pSPORT1 obtained.         prepared uninjected embryos, 2/4 reaction (using primers <br /> A full-length cDNA clone constructed using RT-           complementary exon 2 4) yielded band 178 bp, 18/ <br /> PCR products zebrafish embryonic mRNA               20 reaction yielded band 275 bp slightly smaller non-specific <br />                                                              band. On hand, using RNA prepared GAK-MO2 <br /> standard cloning techniques, verified sequencing.     morphants, 178 bp band unaffected. However, 275 bp band <br />   To generate pCS2-GFP-zGAK pCS2-GFP-zAux,               absent (the non-specific band unaffected) new band of <br /> the entire zGAK zAux ORFs fused frame         359 bp appeared. Sequence analysis 359 bp band revealed that <br />                                                              injection GAK-MO2 caused retention intron 18, resulting in <br /> the C-terminus EGFP, resulting fusions       frame-shift truncation PTEN region (immediately Lys679). (B, <br /> cloned pCS2 EcoRI-XhoI fragments. To make            C) Lateral views uninjected GAK-MO2 morphants 36 hpf. In (B), <br /> zGAK &#206;&#8221;J zAux&#206;&#8221;J, codons His 1206 zGAK          anterior left dorsal up, (C), anterior and <br />                                                              dorsal right. ey, eye; Hb, hindbrain; ov, otic vesicles; ye, yolk <br /> His801 zAux changed stops site-directed      extension. Scale Bar, 100 &#206;&#188;m. <br /> mutagenesis. To express zGAK zAux Drosophila,         Click file <br /> GFP-tagged full-length J-domain-deleted cDNAs             [ http://www.biomedcentral.com/content/supplementary/1471-213X-10-7- <br />                                                              S1.EPS ] <br /> were cloned pUAST [53]. <br />  Bai et al. BMC Developmental Biology 2010, 10:7                                                                                                     Page 16 17 <br /> http://www.biomedcentral.com/1471-213X/10/7 <br />  <br />  <br />  <br />  <br />                                                                                     10. Lai EC, Deblandre GA, Kintner C, Rubin GM: Drosophila neuralized a <br />  Additional file 2: Expression patterns HuC Her4 wild-type                    ubiquitin ligase promotes internalization degradation of <br />  GAK morphant embryos 10-somite stage. (A, B) Lateral                        delta. Dev Cell 2001, 1:783-94. <br />  views HuC expression (A) wild-type (B) GAK morphant embryos              11. Lai EC, Roegiers F, Qin X, Jan YN, Rubin GM: The ubiquitin ligase <br />  10-somite stage. Similar 8-somite stage, cells                      Drosophila Mind bomb promotes Notch signaling regulating the <br />  appeared express HuC GAK morphant embryos 10-somite                       localization activity Serrate Delta. Development 2005, <br />  stage, suggesting presence neuronal cells. (C-D) Close-up                  132:2319-32. <br />  views HuC expression brain regions (indicated brackets            12. Le Borgne R, Remaud S, Hamel S, Schweisguth F: Two distinct E3 ubiquitin <br />  A&amp;B) (C) wild-type (D) GAK morphant embryos. (E, F) Lateral                  ligases complementary functions regulation delta and <br />  views Her4 expression patterns (E) wild-type (F) GAK morphants               serrate signaling Drosophila. PLoS Biol 2005, 3:e96. <br />  10-somite stage. At stage, Her4 expression GAK morphant             13. Pavlopoulos E, Pitsouli C, Klueg KM, Muskavitch MA, Moschonas NK, <br />  embryos appeared reduced, compared wild type. (G-H) Close-                   Delidakis C: neuralized Encodes peripheral membrane protein involved <br />  views Her4 expression brain regions (indicated                     delta signaling endocytosis. Dev Cell 2001, 1:807-16. <br />  brackets E&amp;F) (G) wild-type (H) GAK morphant embryos. In             14. Overstreet E, Chen X, Wendland B, Fischer JA: Either Drosophila <br />  panels, anterior left, lateral views, dorsal up.       epsin protein, divided ENTH domain, functions endocytosis <br />  Fb, forebrain; Hb, hindbrain; Mb, midbrain; mes, mesencephalon; te,                    delta developing eye. Curr Biol 2003, 13:854-60. <br />  telencephalon; Tg, Trigeminal ganglion; tb, tailbud. Scale Bar, 100 &#206;&#188;m.            15. Overstreet E, Fitch E, Fischer JA: Fat facets Liquid facets promote <br />  Click file                                                                    Delta endocytosis Delta signaling signaling cells. Development <br />  [ http://www.biomedcentral.com/content/supplementary/1471-213X-10-7-                   2004, 131:5355-66. <br />  S2.EPS ]                                                                           16. Wang W, Struhl G: Drosophila Epsin mediates select endocytic pathway <br />                                                                                         DSL ligands enter activate Notch. Development 2004, <br />                                                                                         131:5367-80. <br />                                                                                     17. Polo S, Sigismund S, Faretta M, Guidi M, Capua MR, Bossi G, Chen H, De <br /> Acknowledgements                                                                        Camilli P, Di Fiore PP: A single motif responsible ubiquitin recognition <br /> We thank Dr. Ajay Chitnis providing reagents. We grateful Dr.                monoubiquitination endocytic proteins. Nature 2002, 416:451-5. <br /> Donna Fekete critical reading manuscript. This work                 18. Shih SC, Katzmann DJ, Schnell JD, Sutanto M, Emr SD, Hicke L: Epsins and <br /> supported American Heart Association Scientist Development Grant                 Vps27p/Hrs contain ubiquitin-binding domains function receptor <br /> American Cancer Society Research Scholar Grant H.C. T.B. partially                endocytosis. Nat Cell Biol 2002, 4:389-93. <br /> funded Purdue Research Foundation fellowship.                                  19. Poodry CA: shibire, neurogenic mutant Drosophila. Dev Biol 1990, <br />                                                                                         138:464-72. <br /> Authors&#226;&#8364;&#8482; contributions                                                              20. Seugnet L, Simpson P, Haenlin M: Requirement dynamin Notch <br /> T.B. isolated analyzed zebrafish auxilin GAK cDNA, determined               signaling Drosophila neurogenesis. Dev Biol 1997, 192:585-98. <br /> subcellular localization, performed Drosophila rescue experiments. J.       21. Hagedorn EJ, Bayraktar JL, Kandachar VR, Bai T, Englert DM, Chang HC: <br /> L.S. performed phenotypic analysis zebrafish GAK morphants. K.K.             Drosophila melanogaster auxilin regulates internalization Delta to <br /> assisted RNA situ hybridization determine expression                control activity Notch signaling pathway. J Cell Biol 2006, 173:443- <br /> zebrafish auxilin GAK. H.D. assisted molecular cloning zebrafish          52. <br /> auxilin GAK. H.C.C. D.P.S. responsible                 22. Eun SH, Lea K, Overstreet E, Stevens S, Lee JH, Fischer JA: Identification of <br /> experimental design manuscript preparation. All authors                genes interact Drosophila liquid facets. Genetics 2007, 175:1163- <br /> read approved final manuscript.                                                 74. <br />                                                                                     23. Ungewickell E, Ungewickell H, Holstein SE, Lindner R, Prasad K, Barouch W, <br /> Received: 25 June 2009                                                                  Martin B, Greene LE, Eisenberg E: Role auxilin uncoating clathrin- <br /> Accepted: 18 January 2010 Published: 18 January 2010                                    coated vesicles. Nature 1995, 378:632-5. <br />                                                                                     24. Umeda A, Meyerholz A, Ungewickell E: Identification universal <br /> References                                                                              cofactor (auxilin 2) clathrin coat dissociation. Eur J Cell Biol 2000, <br /> 1. Artavanis-Tsakonas S, Matsuno K, Fortini ME: Notch signaling. Science 1995,          79:336-42. <br />     268:225-32.                                                                     25. Zhang CX, Engqvist-Goldstein AE, Carreno S, Owen DJ, Smythe E, <br /> 2. Lai EC: Notch signaling: control cell communication cell fate.                Drubin DG: Multiple roles cyclin G-associated kinase clathrin- <br />     Development 2004, 131:965-73.                                                       mediated sorting events. Traffic 2005, 6:1103-13. <br /> 3. Greenwald I: LIN-12/Notch signaling: lessons worms flies. Genes         26. Hirst J, Sahlender DA, Li S, Lubben NB, Borner GH, Robinson MS: Auxilin <br />     Dev 1998, 12:1751-62.                                                               depletion causes self-assembly clathrin membraneless cages in <br /> 4. Parks AL, Klueg KM, Stout JR, Muskavitch MA: Ligand endocytosis drives               vivo. Traffic 2008, 9:1354-71. <br />     receptor dissociation activation Notch pathway. Development          27. Wu X, Zhao X, Puertollano R, Bonifacino JS, Eisenberg E, Greene LE: <br />     2000, 127:1373-85.                                                                  Adaptor clathrin exchange plasma membrane trans- <br /> 5. Nichols JT, Miyamoto A, Olsen SL, D&#226;&#8364;&#8482;Souza B, Yao C, Weinmaster G: DSL                Golgi network. Mol Biol Cell 2003, 14:516-28. <br />     ligand endocytosis physically dissociates Notch1 heterodimers            28. Wu X, Zhao X, Baylor L, Kaushal S, Eisenberg E, Greene LE: Clathrin <br />     activating proteolysis occur. J Cell Biol 2007, 176:445-58.                     exchange clathrin-mediated endocytosis. J Cell Biol 2001, 155:291- <br /> 6. Yeh E, Dermer M, Commisso C, Zhou L, McGlade CJ, Boulianne GL:                       300. <br />     Neuralized functions E3 ubiquitin ligase Drosophila                29. Newmyer SL, Christensen A, Sever S: Auxilin-dynamin interactions link the <br />     development. Curr Biol 2001, 11:1675-9.                                             uncoating ATPase chaperone machinery vesicle formation. Dev Cell <br /> 7. Wang W, Struhl G: Distinct roles Mind bomb, Neuralized Epsin              2003, 4:929-40. <br />     mediating DSL endocytosis signaling Drosophila. Development              30. Jiang R, Gao B, Prasad K, Greene LE, Eisenberg E: Hsc70 chaperones <br />     2005, 132:2883-94.                                                                  clathrin primes interact vesicle membranes. J Biol Chem <br /> 8. Koo BK, Lim HS, Song R, Yoon MJ, Yoon KJ, Moon JS, Kim YW, Kwon MC,                  2000, 275:8439-47. <br />     Yoo KW, Kong MP, et al: Mind bomb 1 essential generating                 31. Kametaka S, Moriyama K, Burgos PV, Eisenberg E, Greene LE, Mattera R, <br />     functional  <br /> </body></html>