<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />       Structural basis for the recruitment of ERCC1-XPF <br />       <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> nucleotide excision repair complexes by XPA <br />  <br />  <br />                                                   Introduction <br />                                                     The repair of chemical insults to DNA caused by UV light and <br />                                                              other mutagens is essential for coping successfully with the <br />                                                    intrinsic reactivity of DNA and preserving genetic informa- <br />                                                                   tion. Inherited diseases resulting in the failure to correct <br />       <br />        Department of Biological Chemistry and Molecular Pharmacology,                         spontaneous or environmentally induced damage are typi- <br />       Harvard Medical School, Boston, MA, USA, 2Department of                                 cally associated with genomic instability and a predisposition <br />       Pharmacological Sciences, Stony Brook University, Stony Brook, NY,                      to various cancers (Friedberg et al, 2005). Contrarily, DNA <br />       USA, 3Department of Chemistry, Stony Brook University, Stony Brook, <br />       NY, USA and 4Department of Biochemistry and Molecular Biophysics,                       repair is undesirable when DNA-damaging agents are used <br />       Washington University School of Medicine, St Louis, MO, USA                             for chemotherapy of cancer and other diseases. In these <br />                                                                                               settings, the ability to modulate the DNA repair activities of <br />       The nucleotide excision repair (NER) pathway corrects                                   cells targeted for destruction is a desirable goal (Ding et al, <br />       DNA damage caused by sunlight, environmental mutagens                                   2006). The nucleotide excision repair (NER) pathway is <br />       and certain antitumor agents. This multistep DNA repair                                 essential for normal genomic maintenance, removing bulky <br />       reaction operates by the sequential assembly of protein                                 chemical adducts <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> DNA that are otherwise mutagenic or <br />       factors at sites of DNA damage. The efficient recognition of                            would pose lethal blocks to replication. <br />       DNA damage and its repair are orchestrated by specific                                     NER involves over 30 proteins that recognize damaged <br />       protein&#226;&#8364;&#8220;protein and protein&#226;&#8364;&#8220;DNA interactions within NER                                 sites in DNA and excise an oligonucleotide containing the <br />       complexes. We have investigated an essential protein&#226;&#8364;&#8220;                                   damage (de Laat et al, 1999; Gillet and Scha&#194;&#168;rer, 2006). <br />       protein interaction of the NER pathway, the binding of                                  Following excision of the damaged DNA segment, the result- <br />       the XPA protein to the ERCC1 subunit of the repair                                      ing gap is filled by templated DNA synthesis and ligase seals <br />       endonuclease ERCC1-XPF. The structure of ERCC1 in com-                                  the nick to complete the repair. Cell biological and biochem- <br />       plex with an XPA peptide shows that only a small region                                 ical studies have shown that NER operates by the sequential <br />       of XPA interacts with ERCC1 to form a stable complex                                    assembly of protein factors at the sites of DNA damage, rather <br />       exhibiting submicromolar binding affinity. However, this                                than through the action of a preformed repairosome <br />       XPA peptide is a potent inhibitor of NER activity in a cell-                            (Houtsmuller et al, 1999; Volker et al, 2001). The recruitment <br />       free assay, blocking the excision of a cisplatin adduct from                            of NER factors into protein&#226;&#8364;&#8220;DNA ensembles is guided at each <br />       DNA. The structure of the peptide inhibitor bound to its                                step by numerous protein&#226;&#8364;&#8220;protein interactions (Araujo et al, <br />       target site reveals a binding interface that is amenable to                             2001), imparting specificity to the recognition and verification <br />       the development of small molecule peptidomimetics that                                  of damaged sites. Damage recognition in NER culminates in <br />       could be used to modulate NER repair activities                                         the incision of DNA 50 and 30 to the lesion site by ERCC1-XPF <br />       in vivo.                                                                                and XPG, respectively, to release a 24&#226;&#8364;&#8220;32 nucleotide segment <br />       The EMBO Journal (2007) 26, 4768&#226;&#8364;&#8220;4776. doi:10.1038/                                     containing the damage (de Laat et al, 1999; Gillet and <br />       sj.emboj.7601894; Published online 18 October 2007                                      Scha&#194;&#168;rer, 2006). <br />       Subject Categories: genome stability &amp; dynamics; structural                                For the NER pathway, DNA cleavage by ERCC1-XPF re- <br />       biology                                                                                 quires physical interaction with XPA, a scaffold protein that <br />       Keywords: DNA repair; ERCC1; NMR; nucleotide excision                                   interacts with DNA and several repair proteins, including <br />       repair; XPA                                                                             RPA, TFIIH and the ERCC1 subunit of ERCC1-XPF (Li et al, <br />                                                                                               1994, 1995; Park and Sancar, 1994; Saijo et al, 1996). <br />                                                                                               Although XPA was originally described as a DNA damage- <br />                                                                                               specific sensor or verification protein, recent work suggests <br />                                                                                               that XPA instead recognizes the DNA structural intermediates <br />                                                                                               arising during processing by NER (Jones and Wood, 1993; <br />                                                                                               Camenisch et al, 2006). XPA recruits ERCC1-XPF to NER <br />                                                                                               complexes (Volker et al, 2001), positioning the XPF nuclease <br />       *Corresponding author. Department of Biochemistry and Molecular                         domain at the 50 side of the damage site (Enzlin and Scha&#194;&#168;rer, <br />       Biophysics, Washington University, Washington University School of                      2002). ERCC1-XPF has other roles in DNA metabolism out- <br />       Medicine, 660 S Euclid, Campus Box 8231, St Louis, MO 63110, USA. <br />       Tel.: &#195;&#190; 1 314 747 8893; Fax: &#195;&#190; 1 314 632 4432; <br />                                                                                               side of NER, notably in interstrand crosslink repair and <br />       E-mail: tome@biochem.wustl.edu                                                          homologous recombination (Hoy et al, 1985; Niedernhofer <br />       5 <br />        Present address: Department of Medicinal Chemistry, University of                      et al, 2001). The importance of these additional, NER-inde- <br />       Michigan, Ann Arbor, MI 48109, USA                                                      pendent functions of ERCC1-XPF is underscored by the <br />       6 <br />        These authors contributed equally to this work <br />                                                                                               pronounced sensitivity to crosslinking agents caused by <br />       Received: 19 May 2007; accepted: 25 September 2007; published                           mutations of ERCC1 or XPF in mice and humans (McWhir <br />       online: 18 October 2007                                                                 et al, 1993; Weeda et al, 1997; Niedernhofer et al, 2006). <br />  <br /> 4768 The EMBO Journal VOL 26 | NO 22 | 2007                                                                           &amp; 2007 European Molecular Biology Organization <br />                                                                                                                       Recruitment of ERCC1-XPF to NER complexes by XPA <br />                                                                                                                                                       OV Tsodikov et al <br />  <br />  <br />  <br /> However, the exact biochemical role(s) of ERCC1-XPF in                                          and Scha&#194;&#168;rer, 2002; Nishino et al, 2003; Newman et al, 2005), <br /> crosslink repair remain to be discovered.                                                       except that ERCC1&#226;&#8364;&#8482;s groove lacks the catalytic residues of a <br />    Li et al (1994, 1995) identified residues 59&#226;&#8364;&#8220;114 in XPA as                                   nuclease active site and is instead populated with basic and <br /> the site of interaction with ERCC1, and showed that deletion                                    aromatic residues (Gaillard and Wood, 2001). We previously <br /> of three conserved glycines (Gly72, Gly73, Gly74) abrogates                                     showed that the central domain of ERCC1 binds to single- <br /> the XPA&#226;&#8364;&#8220;ERCC1 interaction as well as the ability of the XPA                                     stranded DNA in vitro (Tsodikov et al, 2005), and proposed <br /> protein to confer UV resistance to XP-A cells. Furthermore,                                     the V-shaped groove as the DNA-binding site. <br /> the expression of a truncated protein comprising residues                                          These observations prompted us to investigate the struc- <br /> 59&#226;&#8364;&#8220;114 of XPA renders cells sensitive to UV light and cisplatin                                 tural and functional basis for the interaction of XPA and <br /> (Rosenberg et al, 2001), suggesting that this region is suffi-                                  ERCC1, and its role in recruiting the XPF-ERCC1 endonu- <br /> cient to disrupt the interaction of the native XPA protein with                                 clease to sites of NER. Here, we describe the structure of a <br /> ERCC1-XPF. Conversely, it can be inferred from several pre-                                     short peptide motif from XPA in complex with the central <br /> vious studies that residues 92&#226;&#8364;&#8220;119 of ERCC1 are necessary for                                   domain of ERCC1. We show that this XPA peptide specifically <br /> the interaction with XPA (Li et al, 1994; Sijbers et al, 1996;                                  inhibits the NER reaction in vitro, creating an opportunity <br /> Gaillard and Wood, 2001).                                                                       for structure-based design of NER inhibitors targeting this <br />    Following these seminal studies, understanding of the                                        protein&#226;&#8364;&#8220;protein interaction. Point mutations in the correspond- <br /> biochemical and structural basis for XPA&#226;&#8364;&#8482;s interaction with                                     ing region in XPA abolish NER activity in vitro, underscoring <br /> ERCC1 has not advanced, although more is known about the                                        the importance of the XPA&#226;&#8364;&#8220;ERCC1 interactions for NER. In <br /> individual proteins. XPA contains a well-defined central do-                                    addition to providing insights into how protein&#226;&#8364;&#8220;protein inter- <br /> main (residues 98&#226;&#8364;&#8220;219; Figure 1A), although the remainder of                                    actions mediate progression through the NER pathways, our <br /> the protein including the ERCC1 interaction domain appears                                      studies will provide a blueprint to develop small molecules <br /> to be poorly structured (Buchko et al, 1998, 2001; Ikegami                                      that intercept the interaction between XPA and ERCC1. Such <br /> et al, 1998; Iakoucheva et al, 2001). Residues 92&#226;&#8364;&#8220;119 of                                        molecules should be valuable for studying the biochemical <br /> ERCC1 fall within the central domain of ERCC1 (Tsodikov                                         functions of ERCC1-XPF in NER and other repair pathways <br /> et al, 2005) that is structurally homologous to the nuclease                                    including DNA interstrand crosslink repair and homologous <br /> domain of the archaeal XPF-like proteins Aeropyrum pernix                                       recombination by selectively inhibiting NER. <br /> Mus81/XPF (Newman et al, 2005) and Pyrococcus furiosus <br /> Hef (Nishino et al, 2003). A V-shaped groove on the surface of <br /> ERCC1 corresponds to the nuclease active site of XPF (Enzlin <br />                                                                                                 Results <br />                                                                                                 Induced fit of the XPA peptide upon interaction with <br />                                                                                                 ERCC1 <br />                                                                                                 Previous reports have suggested that the ERCC1-interacting <br />   A                1         58 67   77 98   129    138             219   226         273       region of XPA (Figure 1A) is unfolded in solution, based on <br />              XPA     Dispensable ERCC1 Zn/RPA                 DNA             TFIIH             NMR studies and its sensitivity to proteolytic cleavage <br />                                                                                                 (Buchko et al, 2001; Iakoucheva et al, 2001). To investigate <br />   B          100                                                                                the structure of the XPA ligand, we collected HSQC NMR <br />                                                                                                 spectra of an 15N-labeled XPA59 93 peptide alone and in <br />              105                                                                                complex with unlabeled central domain of ERCC1 (the <br />                                                                                                 ERCC192 214 protein; Figure 1B). In the absence of ERCC1, <br />              110                                                                                the resonance signals for XPA cluster in a narrow range of <br />                                                                                                 chemical shifts (Figure 1B, inset) that is characteristic of an <br />    N (ppm) <br />  <br />  <br />  <br />  <br />              115 <br />                                                                                                 unstructured polypeptide with poor spectral dispersion. In <br />                                                    100                                          the complex with ERCC1, a subset of XPA backbone amides <br />   15 <br />  <br />  <br />  <br />  <br />                                                    105 <br />                                                                                                 become well dispersed and the peaks are broader. These <br />              120                                   110 <br />                                                    115 <br />                                                                                                 changes are indicative of a well-structured region within <br />                                                    120                                          the bound XPA peptide. Only a few resonance peaks are <br />              125                                   125                                          markedly perturbed when XPA59 93 is bound to ERCC1, and <br />                                                    130 <br />                                                          10    9    8     7     6      5 <br />                                                                                                 among these, three glycine residues (assigned as Gly72, <br />              130                                                                                Gly73 and Gly74) are strongly perturbed in the complex. In <br />                 10           9           8                7               6                 5 <br />                                              1H   (ppm)                                         order to overcome the peak broadening observed in NMR <br />                                                                                                 spectra of the XPA59 93 peptide at concentrations above <br /> Figure 1 XPA domain organization and structure of the ERCC1- <br /> binding peptide. (A) The ERCC1-binding region of XPA (residues                                  0.1 mM, we sought to identify shorter XPA peptide ligands <br /> 67&#226;&#8364;&#8220;77) is located between the central domain (Zn2 &#195;&#190; -binding and                                for ERCC1. A minimal, 14-residue XPA67 80 peptide (de- <br /> DNA-binding subdomains; residues 98&#226;&#8364;&#8220;219) and an N-terminal                                      scribed below) was identified by examining a series of over- <br /> region (residues 1&#226;&#8364;&#8220;58) that is dispensable for functional comple-                               lapping XPA fragments from the region previously shown to <br /> mentation of NER in whole-cell extracts from XP-A mutant cells <br /> (Miyamoto et al, 1992) and a TFIIH-binding region (Park et al, <br />                                                                                                 interact with ERCC1 (Li et al, 1994). <br /> 1995). (B) 15N HSQC spectrum of 15N-labeled XPA59 93 in complex <br /> with unlabeled ERCC1, and in the unbound state (inset). The                                     The structure of XPA in complex with ERCC1 <br /> spectrum of the unbound XPA59 93 (inset) is characteristic of an                                A synthetic XPA67 80 peptide with amino-acid sequence <br /> unfolded peptide. The appearance of new well-dispersed NMR <br /> peaks in the XPA spectrum upon addition of ERCC192 214 (shown                                   KIIDTGGGFILEEE forms a stable complex with ERCC196 214 <br /> in the larger spectrum) indicates that a portion of the XPA peptide                             that can be purified by gel filtration chromatography. Like <br /> adopts a defined conformation in complex with ERCC1.                                            full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> XPA protein, the XPA59 93 and the XPA67 80 <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                        The EMBO Journal       VOL 26 | NO 22 | 2007 4769 <br />        Recruitment of ERCC1-XPF to NER complexes by XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       peptides behave similarly and efficiently co-purify with ERCC1,                  complex with unlabeled XPA67 80 (red, Figure 3). However, <br />       suggesting that XPA67 80 contains all significant binding                        complexes of ERCC196 214 with either XPA67 80 or XPA59 93 <br />       determinants. We confirmed that XPA and ERCC1 form a                             were identical, suggesting that the shorter XPA peptide makes <br />       stoichiometric 1:1 complex by estimating the amount of                           all of the significant binding contacts. The 15N HSQC spec- <br />       each subunit in the purified complex using an Edman de-                          trum of the ERCC1&#226;&#8364;&#8220;XPA complex is consistent with a slow- <br />       gradation reaction, and by analytical centrifugation of the                      exchange regime, implying a dissociation equilibrium con- <br />       complex. Equilibrium sedimentation data for the complex                          stant below 1 mM for the complex. The XPA-binding site on <br />       (Supplementary Figure 1C) were best fit to the expected                          the ERCC1 central domain was identified using the backbone <br />       masses for a 1:1 complex of XPA59 93 and ERCC192 214                             assignments for ERCC196 214 alone and in complex with XPA <br />       (Mw &#194;&#188; (19.471.2) kDa)       and      unbound        ERCC192 214                  (see Materials and methods). A comparison of the 15N HSQC <br />       (Mw &#194;&#188; (15.071.0) kDa). We confirmed that ERCC196 214                             spectra for ERCC1 in the presence and absence of XPA reveals <br />       binds stoichiometrically to the XPA67 80 peptide with a KD                       that, with only one exception, the most prominent changes <br />       of 0.78 mM (Supplementary Figure 2). A structure of the                          in chemical shifts involve a cluster of residues within a <br />       XPA67 80&#226;&#8364;&#8220;ERCC196 214 complex (Figure 2A) was determined                          V-shaped groove of the ERCC1 central domain (Figures 2B <br />       by a combination of NMR-derived distance restraints and X-                       and 3). <br />                                           &#203;&#353; resolution (Table 1 in the <br />       ray diffraction data extending to 4 A                                               The bound XPA peptide fits snugly into the V-shaped <br />       Supplementary data and Materials and methods) as described                       groove of ERCC1 (Figure 2) that we previously speculated <br />       below.                                                                           could be a binding site for single-stranded DNA (Tsodikov <br />                                                                                        et al, 2005). Three consecutive glycines (Gly72, Gly 73, <br />       Identification of the ERCC1-binding site in complex                              Gly74) of the XPA peptide insert into the groove, making a <br />       with XPA                                                                         U-turn with close steric complementary to the binding site. <br />       The binding site for XPA on the surface of ERCC1 (Figure 2B)                     These are the same three conserved glycines previously <br />       was identified using two-dimensional HSQC experiments.                           reported to be essential for the interaction of XPA with <br />       The spectrum of unliganded 15N-labeled ERCC192 214 (blue,                        ERCC1 and required for the functional complementation <br />       Figure 3) showed significant differences from that of the                        of XP-A cells (Li et al, 1994, 1995). A total of 1039 A  &#203;&#353; 2 of <br />  <br />  <br />  <br />  <br />  <br />  <br />  <br />       Figure 2 Structure of the XPA&#226;&#8364;&#8220;ERCC1 complex. (A) The XPA67 80 peptide (orange) is bound to a V-shaped groove of the central domain of <br />       ERCC196 214 (green). An orthogonal view of the bound XPA peptide (left side) is shown in comparison to the peptide in complex with ERCC1 <br />       (right-hand side). (B) The XPA-binding site on the surface of ERCC1 (colored red) was identified by resonance perturbations larger than 0.2 ppm <br />       that are indicative of direct interactions with XPA. <br />  <br />  <br /> 4770 The EMBO Journal VOL 26 | NO 22 | 2007                                                                      &amp; 2007 European Molecular Biology Organization <br />                                                                                                                                    Recruitment of ERCC1-XPF to NER complexes by XPA <br />                                                                                                                                                                    OV Tsodikov et al <br />  <br /> Combined average chemical shift <br />  <br />  <br />  <br /> Figure 3 XPA67 80 binds in a shallow groove of ERCC1. (A) A                                           Figure 4 The XPA67 80 peptide is an effective inhibitor of <br /> comparison of the two-dimensional HSQC spectra for 15N-labeled                                        NER activity. (A) XPA67 80 inhibits the in vitro NER reaction, <br />                                                                                                                                      F75A <br /> ERCC192 214 in the presence and absence of an unlabeled XPA67 80                                      whereas the mutant XPA67 80            peptide has no effect. HeLa <br /> peptide. The 15N HSQC spectra reveal significant chemical shift                                       cell extracts were incubated with a plasmid containing a 1,3- <br /> changes for some ERCC1 residues in the absence (blue) or presence                                     intrastrand cisplatin adduct in the presence of increasing concen- <br />                                                                                                                                           F75A <br /> (red) of unlabeled XPA67 80. (B) Combined average chemical shift                                      trations of either XPA67 80 or XPA67 80  (lane 1, no XPA; lanes 2 and 7, <br /> perturbations are calculated as Dwave &#194;&#188; (((Dw1H)2 &#195;&#190; (Dw15N/5)2)/2)1/2                                 46 nM XPA peptide; lanes 3 and 8, 460 nM; lanes 4 and 9, 4.6 mM; <br /> for each backbone amide of ERCC1 and shown as a histogram.                                            lanes 5 and 10, 46 mM; lanes 6 and 11, 92 mM). Products were <br />                                                                                                       visualized by a fill-in reaction following annealing to an oligonu- <br />                                                                                                       cleotide complementary to the excision product with a 4-nt <br />                                                                                                       overhang (Shivji et al, 1999). The marker DNA ladder is labeled <br />                                                                                                                                                       F75A <br />                                                                                                       LMW DNA ladder. (B) XPA67 80 and XPA67 80             do not affect the <br />                                                                                                       intrinsic nuclease activity of ERCC1-XPF. The stem12-loop22 sub- <br /> accessible surface area from XPA peptide is buried in the                                             strate (6.6 nM) was incubated with different concentrations of <br /> complex with ERCC1, accounting for 61% of the solvent                                                 ERCC1-XPF (lanes 2, 4 and 6: 6.7 nM ERCC1-XPF; lanes 3, 5 <br />                                                                                                       and 7: 26.8 nM) and 0.4 mM MnCl2 in the presence of no peptide <br /> accessible surface area of XPA residues 67&#226;&#8364;&#8220;77, which are in                                                                                                               F75A <br />                                                                                                       (lanes 1&#226;&#8364;&#8220;3), 92 mM XPA67 80 (lanes 4 and 5), and 92 mM XPA67 80 <br /> close proximity to the binding site. The XPA ligand derives                                           (lanes 6 and 7). The DNA substrate and the cleavage products are <br /> many interactions from the core sequence motif (shown in                                              indicated. <br /> boldface; KIIDTGGGFILEEE) of the XPA67 80 peptide. The <br /> side chains of Phe75, Leu77 and Thr71 of XPA are clustered <br /> together at the mouth of the V-shaped groove (Figure 2A)                                              cannot be directly observed from our NMR experiments nor <br /> where Phe75 stacks against Asn110 of ERCC1, and the Ile76                                                                                                 &#203;&#353; ) X-ray <br />                                                                                                       can they be reliably confirmed by low-resolution (4 A <br /> side chain packs against the aliphatic portion of ERCC1 side                                          diffraction data (Table 1 in the Supplementary data). Based <br /> chains Arg144 and Leu148. The binding groove in ERCC1 is                                              on the proximity of atoms modeled in the complex, we infer <br /> capped by XPA Leu77.                                                                                  that the carbonyl oxygen of Gly74 may bond with the main <br />    The glycine-rich loop of XPA67 80 extends far into the                                             chain amide of Ser142 from ERCC1. The orientations of the <br /> groove of ERCC1 where main chain atoms of these XPA                                                   Tyr145 and Tyr152 side chains from ERCC1 would permit <br /> residues stack against the side chains of Tyr145 and Tyr152                                           their hydroxyl groups to make hydrogen-bonding interactions <br /> from ERCC1 (Figure 2A). The main chain amides of these                                                with the backbone carbonyls of Thr71 and Gly73, respec- <br /> glycines could participate in hydrogen-bonding interactions                                           tively. The side chain of XPA Asp70 could participate in <br /> with the ERCC1-binding site, although these interactions                                              electrostatic interactions with the side chain His149 of <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                                     The EMBO Journal            VOL 26 | NO 22 | 2007 4771 <br />        Recruitment of ERCC1-XPF to NER complexes by XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       ERCC1. It is notable that a solvent-exposed salt bridge            activities to that of the wild-type XPA protein. The ability of <br />       between the side chains of Asp129 and Arg156 of ERCC1              the XPA proteins to mediate NER activity was tested by <br />       (PDB <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> 2A1I; Tsodikov et al, 2005) becomes almost               incubating a plasmid containing a 1,3-cisplatin intrastrand <br />       completely buried when XPA is bound.                               crosslink with a cell-free extract generated from XPA-deficient <br />          Phe75 of XPA is completely buried within the ERCC1-             cells supplemented with purified wild-type or mutant XPA <br />       binding site (Figure 2A). We tested whether an alanine             protein (Shivji et al, 1999). Addition of wild-type XPA protein <br />       substitution at this position interferes with binding to 15N-      to this mixture resulted in robust NER activity, as evidenced <br />       labeled ERCC1 by measuring chemical shifts in the 15N HSQC         by formation of the characteristic excision products of 24&#226;&#8364;&#8220;32 <br />       spectra in the presence of the mutant peptide designated           nts in length (Figure 5A, lanes 1&#226;&#8364;&#8220;2). By contrast, no NER <br />            F75A <br />       XPA67 80  . Addition of the mutant peptide failed to perturb       activity was observed following addition of the F75A or <br />       the chemical shifts of ERCC1 seen upon addition of wild-type       G73D/G74D mutants, while the G73D single deletion mutant <br />       XPA67 80 (data not shown), indicating that the mutant pep-         displayed marginal NER activity. <br />       tide does not bind to ERCC1. The TGGGFI-binding motif of              To test if these XPA mutations only affected binding to <br />       the XPA ligand and the corresponding residues of the ERCC1-        ERCC1, we also compared the DNA-binding activities of wild- <br />       binding site are strictly conserved in higher eukaryotes. In       type and mutant XPA proteins. We investigated the binding of <br />       lower eukaryotes, the corresponding sequences of both pro-         wild-type and mutant XPA to a DNA three-way junction, <br />       teins have diverged from this consensus, perhaps indicating        representing a high-affinity target for XPA in band-shift assays <br />       the coevolution of these two proteins and their functions.         (Missura et al, 2001). The wild-type, F75A, G73D and G73D/ <br />                                                                          G74D XPA proteins all bound with similar affinity to a three- <br />       The XPA peptide inhibits NER in mammalian cell                     way junction (Figure 5B), indicating that the mutant proteins <br />       extracts                                                           are fully proficient in DNA binding and unlikely to be <br />       The direct interaction of XPA67 80 peptide with the ERCC1-         misfolded or otherwise inactive. These results show that <br />       binding pocket raised the possibility that this peptide might      single point mutations in XPA can result in a defect in NER <br />       specifically interfere with the recruitment of the ERCC1-XPF <br />       nuclease into the NER reaction pathway. We investigated the <br />                                                 F75A <br />       effect of XPA67 80 and the mutant XPA67 80       peptide on the <br />                                                                          A                                               G73&#226;&#710;&#8224; <br />       dual incision of a DNA lesion during NER in cell-free extracts. <br />                                                                                             WT       F75A      G73&#226;&#710;&#8224;      G74&#226;&#710;&#8224; <br />       A plasmid containing a single site-specific 1,3-cisplatin in- <br />       trastrand crosslink was incubated with HeLa cell-free extract <br />       in the presence of increasing concentrations of XPA peptide <br />       (Shivji et al, 1999). In the absence of XPA peptide, the <br />       characteristic NER excision products of 28&#226;&#8364;&#8220;33 nucleotides <br />       containing the lesion were evident (Figure 4A, lane 1). <br />       Increasing concentrations of XPA67 80 interfered with exci- <br />       sion of the oligonucleotide, and complete inhibition was                    25 <br />       achieved at a concentration of XPA peptide in the low <br />       micromolar range (Figure 4A, lanes 2&#226;&#8364;&#8220;6). In contrast, the                         1        2   3    4    5     6    7     8 <br />                         F75A <br />       addition of XPA67 80   did not affect NER activity at concentra- <br />       tions up to 92 mM, the maximum concentration tested                B                                                          &#226;&#710;&#8224;G73 <br />       (Figure 4A, lanes 7&#226;&#8364;&#8220;11).                                                         WT            F75A             &#226;&#710;&#8224;G73          &#226;&#710;&#8224;G74 <br />                                                                               &#226;&#710;&#8217; <br />          The XPA peptide might inhibit NER activity in vitro by <br />       directly interfering with the endonuclease activity of ERCC1- <br />       XPF, instead of blocking its interaction with XPA. To account       x <br />                                                                           d <br />       for the former possibility, we tested the effects of XPA <br />       peptides on the DNA incision reaction catalyzed by purified         d <br />       ERCC1-XPF using stem-loop DNA substrate (de Laat et al, <br />       1998). ERCC1-XPF efficiently cleaves on the 50 side of the loop         1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 181920 21 <br />       and the XPA peptide has no effect on this activity (Figure 4B)     Figure 5 <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>Mutation</span> of the ERCC1-binding epitope of XPA abolishes <br />       even at a concentration (92 mM) that completely abolishes          NER but not DNA-binding activity. (A) XP-A (XP2OS) cell extracts <br />       NER activity (Figure 4A). We conclude that the inhibitory          were incubated with a plasmid containing a 1,3-intrastrand cisplatin <br />       effect of XPA67 80 on the NER reaction results from disrupting     adduct in the presence of wild-type XPA (XPA-WT) or mutant XPA <br />                                                                          proteins (XPA-F75A, XPA-G73D or XPA-G73D/G74D). The reaction <br />       the interaction of ERCC1 with XPA, an essential protein&#226;&#8364;&#8220;           products were visualized by a fill-in reaction after annealing the <br />       protein interaction for the dual incision of DNA by the NER        excision product to an complementary oligonucleotide with a 4-nt <br />       pathway.                                                           overhang (Shivji et al, 1999). Different XPA concentrations of <br />                                                                          200 nM (lanes 1, 3, 5 and 7) and 800 nM (lanes 2, 4, 6 and 8) <br />                                                                          were tested. The position of a 25 mer of the LMW DNA ladder is <br />       Mutations in the ERCC1-binding epitope of XPA abolish              indicated. (B) A 50 -labeled DNA three-way junction (1 nM) was <br />       NER                                                                incubated with wild-type and mutant XPA proteins for 30 min at <br />       The specificity of inhibition of NER by XPA67 80 suggested         room temperature, then the XPA-bound (xd) and free DNA (d) <br />       that mutations of single residues such as F75 might diminish       oligonucleotides were separated on an 8% native polyacrylamide <br />                                                                          gel. The reaction products generated with different concentrations <br />       the NER activity of the XPA protein. We generated mutant           of XPA are shown: 0 (lane 1), 4 nM (lanes 2, 7, 13, 17), 10 nM (lanes <br />       XPA proteins containing an F75A mutation and DG73 single           3, 8, 13, 18), 25 nM (lanes 4, 9, 14, 19), 60 nM (lanes 5, 10, 15, 20) <br />       and DG73/DG74 double deletion, and compared their                  and 150 nM (lanes 6, 11, 16, 21). <br />  <br />  <br /> 4772 The EMBO Journal VOL 26 | NO 22 | 2007                                                      &amp; 2007 European Molecular Biology Organization <br />                                                                                          Recruitment of ERCC1-XPF to NER complexes by XPA <br />                                                                                                                          OV Tsodikov et al <br />  <br />  <br />  <br /> activity by weakening the interaction between ERCC1 and           the associated entropic penalty for binding. This peptide&#226;&#8364;&#8220; <br /> XPA. Due to the highly cooperative nature of NER (Moggs           protein interaction is sufficient to block NER activity in cell- <br /> et al, 1996), other NER functions and interactions may            free extracts (Figure 4A), raising the possibility that peptido- <br /> be disrupted as a result of blocking the recruitment of           mimetic ligands could be developed to specifically block NER <br /> XPF&#226;&#8364;&#8220;ERCC1.                                                        activity in vivo. Although the competing XPA peptide pre- <br />                                                                   vents the double incision of lesioned DNA during NER, the <br /> XPA competes with single-stranded DNA for binding                 peptide does not interfere with the cleavage of a model DNA <br /> to ERCC1                                                          substrate by purified ERCC1&#226;&#8364;&#8220;XPF (Figure 4B). These results <br /> Because XPA binds in the groove on ERCC1 (Figure 2) that          show that the XPA peptide does not block the nuclease <br /> was previously implicated in DNA-binding activity (Tsodikov       activity of XPF-ERCC1 and is instead likely to interfere with <br /> et al, 2005), we directly tested whether or not XPA competes      the recruitment of the nuclease into the NER protein complex. <br /> with single-stranded DNA for binding to ERCC1. DNA-bind-          Mutations in the ERCC1-binding domain of XPA similarly <br /> ing activity was measured by monitoring fluorescence aniso-       abolish NER activity without affecting the intrinsic DNA- <br /> tropy, using single-stranded 40-mer oligonucleotide labeled       binding activity of XPA. Intriguingly, the central domain of <br /> on the 50 end with 6-carboxyfluorescein. The XPA67 80 pep-        ERCC1 binds to single-stranded DNA in vitro (Tsodikov et al, <br /> tide does not detectably bind to DNA (not shown), although it     2005) and this DNA-binding activity is blocked by XPA <br /> does compete with DNA for binding to ERCC1                        (Supplementary Figure 3). Although these competing activ- <br /> (Supplementary Figure 3). This result confirms that the           ities at first appear contradictory, our cleavage assays em- <br /> DNA-binding site on ERCC1&#226;&#8364;&#8482;s central domain overlaps with          ployed full-length ERCC1 protein in complex with XPF, <br /> the XPA-binding site. The EC50 for binding of XPA67 80 is in      whereas the in vitro binding studies used the central domain <br /> the micromolar range, but quenching of the fluorescent probe      only of ERCC1. Since ERCC1-XPF has multiple DNA-binding <br /> by high concentrations of XPA precluded an accurate mea-          sites (Newman et al, 2005; Tripsianes et al, 2005; Tsodikov <br /> surement of the binding constant. We previously reported an       et al, 2005; Nishino et al, 2005a, b), it is likely that some of <br /> equilibrium-binding constant of 1.5 mM for DNA binding to         the other DNA-binding surfaces of the XPF&#226;&#8364;&#8220;ERCC1 hetero- <br /> the central domain of ERCC1 (Tsodikov et al, 2005). By fitting    dimer can compensate for the interference by the XPA <br /> the XPA competition titration data to a competitive binding       peptide. With respect to the overall NER reaction, it is <br /> model (Equation 9 in the Supplementary data), we obtain the       conceivable that DNA and the XPA protein alternatively <br /> estimated binding constant of Kd &#194;&#188; (5407280) nM for the           bind to the same site on ERCC1 during different steps of the <br /> XPA&#226;&#8364;&#8220;ERCC1 complex. This result agrees well with the affinity      repair process. In this regard, a molecular handoff of ERCC1 <br /> determined directly for this interaction (Supplementary           from XPA to one strand of the unwound DNA substrate could <br /> Figure 2). Thus, XPA binds to the central domain of ERCC1         be envisioned as one of multiple, individually weak interac- <br /> with approximately three-fold higher affinity than single-        tions that drive the progression through the NER pathway in <br /> stranded DNA.                                                     a concerted fashion (Stauffer and Chazin, 2004; Gillet and <br />                                                                   Scha&#194;&#168;rer, 2006). <br />                                                                       Homologs of the XPA and ERCC1 proteins are found only <br /> Discussion                                                        in eukaryotic organisms, despite the presence of homodi- <br /> The removal of bulky and helix-distorting DNA lesions by the      meric XPF-like endonucleases in the Archaea (Nishino et al, <br /> NER pathway requires the coordinated assembly of a large          2003). The ERCC1 residues constituting the XPA-binding site <br /> multiprotein complex (Houtsmuller et al, 1999; Volker et al,      are poorly conserved in Saccharomyces cerivisiae (Rad10 <br /> 2001) that exposes the damaged DNA strand and excises an          protein) and S. pombe (Swi10 protein), and the XPA homo- <br /> oligonucleotide containing the lesion (Gillet and Scha&#194;&#168;rer,       logs (S. saccharomyces Rad14; S. pombe Rhp14) are highly <br /> 2006). We have investigated one of the essential protein&#226;&#8364;&#8220;         divergent from mammalian XPA. Indeed, a different inter- <br /> protein interactions in this pathway. The specific interaction    action site has recently been reported for Rad14 and Rad10/ <br /> of XPA with ERCC1 is responsible for recruitment of the           Rad1, the respective yeast homologs of XPA and ERCC1&#226;&#8364;&#8220;XPF <br /> ERCC1-XPF nuclease to the DNA repair complex (Li et al,           (Guzder et al, 2006). These observations suggest that XPA <br /> 1994). Our structural studies have defined the XPA ligand as a    and ERCC1 may have coevolved to interact specifically with <br /> TGGGFI sequence motif that inserts into a pocket of the           each other in higher eukaryotes, perhaps in response to the <br /> central domain of ERCC1 (Figure 2). It was previously             added complexity and distinct functional organization of the <br /> shown that deletion of the GGG triplet within this motif          eukaryotic NER pathway. Correspondingly, a BLAST search <br /> abolishes the interaction of XPA with ERCC1 (Li et al,            does not identify the TGGGFI motif of the XPA ligand in any <br /> 1995). These glycines insert deep into the ERCC1-binding          other mammalian protein. <br /> site and are likely to make hydrogen-bonding interactions             In conclusion, we have established that only a short <br /> using main chain atoms (Figure 2). The binding site of ERCC1      peptide segment of XPA is sufficient to form a stable and <br /> is mainly a nonpolar surface that is punctuated by several        specific 1:1 complex with ERCC1. The interactions of three <br /> large aromatic side chains (Phe145, Phe152) and a buried salt     consecutive glycines (Gly72, Gly73, Gly74) and several flank- <br /> bridge between Arg156 and Asp129. We show that single             ing residues of XPA complement a V-shaped, hydrophobic <br /> point mutations in XPA (F75A or G73D) effectively abolish         groove in the central domain of ERCC1. This protein&#226;&#8364;&#8220;protein <br /> NER activity in vitro, underscoring the high specificity of the   interaction is essential for NER activity, and the XPA peptide <br /> binding interaction between ERCC1 and XPA.                        is an effective inhibitor of NER activity in a cell-free reaction. <br />    The XPA peptide ligand is unstructured in solution             This work paves the way for development of specific NER <br /> (Figure 1B). It is therefore remarkable that a short peptide      inhibitors targeting the surface of ERCC1 involved in XPA <br /> segment binds to ERCC1 with submicromolar affinity, given         binding. ERCC1 has served as a molecular marker for clinical <br />  <br /> &amp; 2007 European Molecular Biology Organization                                           The EMBO Journal       VOL 26 | NO 22 | 2007 4773 <br />        Recruitment of ERCC1-XPF to NER complexes by XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       resistance to cisplatin-based chemotherapy (Reed, 2006),                    Structural information for the ERCC1&#226;&#8364;&#8220;XPA complex was obtained <br />                                                                                with a differentially labeled sample in which ERCC1 was 15N- <br />       raising the possibility of using ERCC1 antagonists as sensitiz- <br />                                                                                labeled and perdeuterated and the synthetic XPA fragment was <br />       ing agents for tumors resistant to this and other DNA-dama-              unlabeled (D,N-ERCC1/U-XPA) (Walters et al, 1997, 2001). The <br />       ging agents in the treatment of cancer.                                  assignment of the XPA peptide in this sample was performed <br />                                                                                using homonuclear 2D NOESY and 2D TOCSY experiments acquired <br />                                                                                in both H2O and D2O buffers. The total of 92 intramolecular <br />       Materials and methods                                                    distance constraints for the XPA peptide were derived from the 2D <br />                                                                                NOESY experiment acquired in H2O with 100 ms mixing time. <br />       Peptide and DNA                                                          Intermolecular distance restraints were derived from a 15N- <br />       The XPA67 80 peptide corresponding to residues 67&#226;&#8364;&#8220;80 of the XPA          dispersed NOE-HSQC experiment acquired on the D,15N-ERCC1/ <br />                                             F75A <br />       protein and the mutant peptide XPA67 80    were synthesized by solid     U-XPA sample using 200 ms mixing time. A total of 23 intermole- <br />       phase methods, then HPLC-purified by the Molecular Biology Core          cular distance restraints between the amide protons of ERCC1 and <br />       Facility at Tufts University (Boston, MA). A 40-mer DNA oligomer         the protons of XPA were derived from this experiment. The structure <br />       50 -CCGGTGGCCAGCGCTCGGCGT20&#226;&#8364;&#8220;30 with a 50 6FAM <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> (Inte-               of the ERCC1&#226;&#8364;&#8220;XPA complex was calculated using simulated <br />       grated DNA Technologies) was gel-purified by conventional                annealing procedure in XPLOR-NIH (Schwieters et al, 2003). The <br />       techniques.                                                              total energy term used in the calculation incorporated all of the <br />                                                                                NMR-derived distance restraints as well as the 4 A&#203;&#353; X-ray data. Ten <br />                                                                                lowest energy structures out of 100 calculated were deposited in <br />       Protein expression and purification <br />                                                                                the PDB with accession <span id='am-86' about='Thesaurus:code' typeof='owl:Thing'>code</span> 2JNW. The solvent accessible <br />       The central domain of ERCC1 (constructs ERCC192 214 or <br />                                                                                surface areas were calculated for the lowest energy structure using <br />       ERCC196 214) with an N-terminal His6 tag was expressed and <br />       purified as previously described (Tsodikov et al, 2005). Fragments       Surface Racer 4.0 (Tsodikov et al, 2002) with the solvent probe <br />                                                                                              &#203;&#353;. <br />                                                                                radius of 1.4 A <br />       of the XPA protein (XPA1 273, XPA59 273, XPA59 219, XPA59 93) were <br />       cloned into pET19bpps, in which an N-terminal (His)10 tag is <br />       separated from the XPA sequences by a Prescission protease               Competitive binding equilibrium titrations <br />       cleavage site (gift of Dr Tapan Biswas), between NdeI and XhoI           Fluorescence anisotropy measurements were performed as pre- <br />       sites. Full-length XPA protein was expressed in bacteria from            viously reported (Tsodikov et al, 2005). The equilibrium titrations <br />       pET15b-XPA and purified by Ni2 &#195;&#190; -NTA, gel filtration, and heparin       and their analysis are described in detail in the supplementary data. <br />       chromatography. Protein expression and purification are described <br />       in detail in the Supplementary data. <br />                                                                                Construction and expression of mutant XPA proteins <br />       Analytical ultracentrifugation                                           Site-directed mutagenesis using the QuikChange kit (Stratagene) <br />       Sedimentation equilibrium experiments with ERCC192 214 and the           introduced point mutations in the expression vector pET15b-XPA <br />       complex ERCC192 214&#226;&#8364;&#8220;XPA59 93 were performed using Beckman                was performed. pET15b-XPA served as template and oligonucleo- <br />       XLA Analytical Centrifuge. In both cases, proteins were at               tide primers used to generate the mutations contained the desired <br />       concentrations of 0.3&#226;&#8364;&#8220;0.5 mg/ml in NMR Buffer (20 mM Tris buffer         mutation, and a marker restriction site for selection. The following <br />       pH 7.2, 50 mM NaCl, 2 mM b-mercaptoethanol and 0.1 mM EDTA).             primers were used (restriction site are underlined and indicated, <br />       Sedimentation equilibrium data were analyzed as described in the         modified nucleotides are shown in italics): <br />       Supplementary data.                                                         XPA-F75A: GACACAGGAGGAGGCGCCATTCTAGAAGAGGAAGAAG <br />                                                                                (XbaI) <br />       Crystallization of XPA&#226;&#8364;&#8220;ERCC1 complex, data collection and                   XPA-DG74:      GACACAGGAGGATTCATTCTAGAAGAGGAAGAAG <br />       analysis                                                                 (XbaI) <br />       Protein crystallization and X-ray data collection are described in the      XPA-DG73/74: GATAATTGACACAGGATTCATTCTAGAAGAGGAA <br />       Supplementary data. A complete and redundant X-ray data set              GAAG (XbaI). <br />       was collected and processed using HKL2000 (Otwinowski and                   Positive clones were fully sequenced to rule out the introduction <br />       Minor, 1997). The crystals belong to space group I4132 with one          of additional mutations. Mutant XPA proteins were expressed in <br />       ERCC1&#226;&#8364;&#8220;XPA complex in the asymmetric <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. The structure was              Escherichia coli BL21(DE3)pLyS cells and purified by chromato- <br />       determined by molecular replacement (MR) methods using the               graphy on nickel-NTA, gel filtration and heparin columns. <br />       program PHASER (McCoy et al, 2005) and the crystallographic <br />       model of the ERCC1 central domain (Tsodikov et al, 2005; PDB code <br />       2A1I) in which the residues C-terminal to residue 214 were deleted.      Nuclease assay <br />       A difference (Fo Fc) electron density map calculated with phases         ERCC1-XPF was purified and nuclease assays using a stem-loop <br />       from the MR solution revealed the bound XPA peptide. The XPA             substrate were carried out as described previously (Enzlin and <br />       peptide was built into the difference density using distance             Scha&#194;&#168;rer, 2002) (see Supplementary data for details). <br />       restraint information from NMR experiments and the structure of <br />       the complex was then refined as described below, with strong <br />                                                                                DNA-binding assays <br />       geometric restraints imposed on the ERCC1 subunit due to the <br />                                                                                The three-way junction DNA substrate described previously <br />       low-resolution diffraction data and the absence of intramolecular <br />                                                                                (substrate 7 in Table 1 of Hohl et al, 2003) was 50 -32P-end labeled <br />       distance information for ERCC1. All experimental XPA&#226;&#8364;&#8220;ERCC1 <br />                                                                                and incubated at 1 nM concentration with various amounts of <br />       distance restraints were accommodated without violations <br />                                                                                XPA in EMSA buffer (25 mM HepesKOH pH 8.0, 30 mM KCl, 10% <br />       using the structure of unbound ERCC1, suggesting that ERCC1 <br />                                                                                glycerol, 1 mM DTT, 1 mM EDTA, 0.1 mg/ml BSA) at a reaction <br />       does not undergo significant conformational changes upon binding <br />                                                                                volume of 15 ml. After equilibration at room temperature for 30 min, <br />       to XPA. <br />                                                                                the samples were loaded on a 5% (37.5:1) native polyacrylamide <br />                                                                                gel containing 0.5   TBE and electrophoresed at 90 V for 2 h. <br />       NMR experiments and determination of the structure                       Gels were dried and the radioactive bands visualized by autoradio- <br />       of XPA&#226;&#8364;&#8220;ERCC1 complex                                                     graphy. <br />       All NMR data were acquired in the NMR Buffer described above. <br />       The protein concentrations were 0.25 mM for free ERCC196 214 or <br />       ERCC192 214 (which behaved similarly in all experiments), 0.25 mM        In vitro NER assay <br />       for ERCC192 214 in complex with a synthetic XPA67 80 peptide and         This assay was performed using an established protocol (Shivji <br />       0.1 mM for ERCC192 214 complex with XPA59 93 fragment. Higher            et al, 1999), which is described in detail in the Supplementary data. <br />       protein concentrations resulted in line broadening and lower quality <br />       NMR spectra. Backbone assignments of the free ERCC192 214 and <br />       ERCC1&#226;&#8364;&#8220;XPA67 80 complex were performed using a standard set                <br />       of triple-resonance experiments: HNCA/HN(CO)CA, HN(CA)CB/                <br />       HN(COCA)CB and HNCO/HN(CA)CO.                                           <br />  <br /> </body></html>