<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Plaimas et al. BMC Systems Biology 2010, 4:56 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />     METHODOLOGY ARTICLE                                                                                                                               Open Access <br />  <br /> Identifying essential genes bacterial metabolic <br /> Methodology article <br />  <br /> networks machine learning methods <br /> Kitiporn Plaimas1,2, Roland Eils*1,2 Rainer K&#195;&#182;nig1,2 <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Background: Identifying essential genes bacteria supports identify potential drug targets understanding <br />     minimal requirements synthetic cell. However, experimentally assaying essentiality coding genes is <br />     resource intensive feasible bacterial organisms, particular infective. <br />     Results: We developed machine learning technique identify essential genes using experimental data of <br />     genome-wide <span id='am-168' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-169' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>knock-out</span> screens bacterial organism infer essential genes related bacterial <br />     organism. We used broad variety topological features, sequence characteristics co-expression properties <br />     potentially associated essentiality, flux deviations, centrality, codon frequencies sequences, co- <br />     regulation phyletic retention. An organism-wise cross-validation bacterial species yielded reliable results with <br />     good accuracies (area receiver-operator-curve 75% - 81%). Finally, applied drug target <br />     predictions Salmonella typhimurium. We compared predictions viability experimental knock-outs S. <br />     typhimurium identified 35 enzymes, highly relevant considered potential drug targets. <br />     Specifically, detected promising drug targets non-mevalonate pathway. <br />     Conclusions: Using elaborated features characterizing network topology, sequence information microarray data <br />     enables predict essential genes bacterial reference organism related query organism any <br />     knowledge essentiality genes query organism. In general, method beneficial inferring <br />     drug targets experimental data genome-wide <span id='am-246' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-247' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>knockout</span> screens available investigated <br />     organism. <br />  <br />  <br /> Background                                                                                  fully targeted antibiotics inhibiting essential processes <br /> By <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, essential proteins cellular organism                                bacterial genomes [1]. <br /> necessary live replicate, attrac-                                    Several computational techniques developed <br /> tive targets antimicrobial treatments. However, exper-                                  identify essential genes silico. Flux balance analyses <br /> imentally assaying essentiality coding genes                                (FBA) widely used assess essentiality genes [2]. <br /> error prone high throughput. Additionally,                                     However, FBA approaches need clear definitions nutri- <br /> experimental screens resource intensive feasi-                                  tion availability biomass production specifi- <br /> ble organisms, typically, gene knock-                                 cally given environmental conditions [3]. Descriptors for <br /> out strain needs constructed. Besides this, patho-                                    enzymes metabolic network graph <br /> genic bacterial organisms hazardous cultivate                                    theoretical approaches used identify drug <br /> therefore need higher laboratorial efforts. The metabo-                                     targets micro-organisms. Concepts choke-points <br /> lism cell substantial maintaining life                                      load-points successfully applied estimate the <br /> growth, metabolic enzymes success-                                      essentiality enzyme [4-6]. The term &apos;damage&apos; was <br />                                                                                             used assess enzymes serve drug targets <br />                                                                                             inhibition influences substantial number of <br /> * Correspondence: r.eils@dkfz.de                                                            downstream reactions products [7]. In previous <br /> 1Department Bioinformatics Functional Genomics, Institute                         study, examined ability network obtain <br /> Pharmacy Molecular Biotechnology, Bioquant, University Heidelberg, Im                products knocked-out reaction educts <br /> Neuenheimer Feld 267, 69120 Heidelberg, Germany <br /> Full list author information available end article                      alternative pathways used method detect <br />                                             &#194;&#169; 2010 Plaimas et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />             BioMed Central Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                             medium, provided original work properly cited. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                     Page 2 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> potential drug targets P. falciparum [6]. Various            screen, defined 35 enzymes drug targets S. <br /> descriptors centrality node network           typhimurium, 23 described previ- <br /> been successfully applied supported detecting essen-         ously drug targets micro-organisms. We sug- <br /> tial proteins protein-protein interaction networks [8-        gest remaining potential new drug <br /> 12]. It shown proteins central        targets organism studied (S. typhimurium). <br /> position evolve slowly likely be <br /> essential <span id='am-12' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span> [9]. Also sequence features like          Results discussion <br /> codon usage, GC-content localization signals            Predicting essential genes organism-wise cross <br /> used predicting essential genes. They success-          validation <br /> fully applied inferring essential genes S. cerevi-      The general workflow depicted Figure 1. We recon- <br /> siae studied yeast strain S. mikatae [13].           structed metabolic networks investigated <br /> Additionally, phyletic retention          organisms bipartite graph consisting alternating <br /> valuable predictive feature gene essentiality E. coli     reactions compounds. Two reactions linked by <br /> and S. cerevisiae [14,15]. Although protein-protein inter-       compound compound product reac- <br /> action networks provide global view cellular sig-       tion substrate other. The machine learning <br /> naling, interested identifying drug            trained validated large set fea- <br /> targets pathogens inferred properties mal-            tures. Local topology based features used qualita- <br /> functional metabolism having knocked enzy-          tively possible flux deviations. Choke load <br /> matic function.                                                  points defined damage used the <br />    In previous study, developed applied inte-        qualitative flux load stream effects the <br /> grative machine learning method combined              knocked reaction. Centrality features calcu- <br /> topology based methods validate experimental               lated additionally estimate load reactions. <br /> knock-out screen Escherichia coli [16]. We used           We considered existence homologous genes the <br /> the basic concepts strategy enable predicting         corresponding knocked genes expected <br /> essential genes organism experimental         function. Co-regulated genes con- <br /> training data available. For defining essentiality   sidered estimate existence possible analogous <br /> gene, integrated sequence characteristics         genes. Phyletic retention calculated observing <br /> codon usages, <span id='am-8' about='xsp:length' typeof='owl:Thing'>length</span> sequences phyletic reten-        phylogenetic conservation gene was <br /> tion. Furthermore, incorporated centrality-           knocked out. Codon usages calculated gene <br /> descriptors node network. We used experimen-          used features. All features listed Table 1 and <br /> tal datasets comprehensive genome wide knock-out              described in methods. We started pre- <br /> screens Escherichia coli [17,18] Pseudomonas              dicting essential genes E. coli. For this, trained <br /> aeruginosa [19,20] train machines large vari-      classifiers (machines) experimental data two <br /> ety attributes including topology characteristics          genome-wide knock-out screens Pseudomonas aerugi- <br /> mentioned above, developments evaluating pos-             nosa (datasets paeJ paeL experimental studies <br /> sible flux deviations [6], genomic transcriptomic        Jacobs co-workers [19] Liberati co-work- <br /> information. <span id='am-11' about='protege:TO' typeof='owl:Thing'>To</span> develop classification          ers [20], respectively). These datasets taken our <br /> readily applicable predicting essential genes new       gold standard defining true positives true negatives <br /> query organism, needs make accurate pre-           essential genes metabolism training <br /> dictions organism trained.            organism (P. aeruginosa). We trained (100) classi- <br /> Therefore, performed cross-validation            fiers essential genes equal ran- <br /> organisms E. coli P. aeruginosa, i.e. trained     domly selected non-essential genes (stratification the <br /> E. coli validated P. aeruginosa (and vice versa)     training data). The trained machines applied to <br /> obtain quality performance approach.          predict essential genes query organism (E. coli). <br /> We applied trained validated classifiers         The output machines summed used a <br /> the pathogenic bacterium Salmonella typhimurium. We              voting score represented propensity gene to <br /> compared results literature experimen-          lethal cell. In turn, scheme was <br /> tal data large knock-out study S. typhimurium           applied predict essential genes P. aeruginosa with <br /> [21]. Furthermore, analyzed predictions gene         classifiers trained datasets <br /> set enrichment tests metabolic pathways identi-          E. coli (ecoB Baba co-workers&apos; study [17] <br /> fied proteins entire non-mevalonate pathway         ecoG Gerdes co-workers&apos; study [18], <br /> relevant targeting drugs. Its reactions showed          respectively). This organism-wise cross-validation was <br /> typically topological characteristics essential reactions.    applied estimate performance classifiers. We <br /> Using prediction results experimental knock-         compared datasets genome. 79 essen- <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                     Page 3 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 1: List features. <br />  Short form           Explanation <br />  <br />  <br />  Topology features <br />                       a) Deviation <br />  RUP                  Reachable/Unreachable Products (RUP): equals products produced blocking reaction, <br />                       zero <br />  PUP                  Percentage Unreachable Products (PUP): percentage products produced blocking the <br />                       reaction <br />  ND                   Number Deviations (ND) <br />  APL                  Average Path Length (APL): average path length deviations <br />  LSP                  Length Shortest Path (LSP): length shortest path deviations <br />                       b) Local topology <br />  NS                   Number Substrates (NS) <br />  NP                   Number Products (NP) <br />  NNR                  Number Neighboring Reactions (NNR) <br />  NNNR                 Number Neighbors Neighboring Reactions (NNNR) <br />  CCV                  Clustering Coefficient Value (CCV): clustering coefficient reaction <br />  DIR                  Directionality reaction (DIR) <br />                       c) Choke points load scores <br />  CP                   Choke Point (CP): reaction choke point (Rahman et al, 2006) <br />  LS                   Load Score (LS): load score reaction (Rahman et al, 2006) <br />                       d) Damage <br />  NDR                  Number Damaged Reactions (NDR) (Lemke et al, 2004) <br />  NDC                  Number Damaged Compounds (NDC) (Lemke et al, 2004) <br />  NDRD                 Number Damaged Reactions having Deviations (NDRD): number damaged reactions other <br />                       alternative paths reached blocking reaction <br />  NDCD                 Number Damaged Compounds having Deviations (NDCD): number damaged compounds other <br />                       alternative paths reached blocking reaction <br />  NDCR                 Number Damaged Choke point Reactions (NDCR) <br />  NDCC                 Number Damaged Choke point Compounds (NDCC) <br />  NDCRD                Number Damaged Choke point Reactions having Deviations (NDCRD): number damaged choke point <br />                       reactions alternative paths reached blocking reaction <br />  NDCCD                Number Damaged Choke point Compounds having Deviations (NDCCD): number damaged choke point <br />                       compounds alternative paths reached blocking reaction <br />                       e) Centrality <br />  BW                   Betweenness centrality <br />  CN                   Closeness centrality <br />  EC                   Eccentricity centrality <br />  EV                   Eigenvector centrality <br />  Genomic transcriptomic features <br />                       f) Homologs <br />  NAR                  Number Associated Reactions (NAR): number reactions <span id='am-480' about='xsp:base' typeof='owl:Thing'>base</span> knocked-out gene <br />  Hn                   Homology different expectation values: number homologous genes e-value cutoff 10-30,10-20,10-10,10-7,10- <br />                       5,10-3 (H30, H20, H10, H7, H5, H3) <br />  <br />                       g) Gene expression <br />  NGSE                 Number Genes having Similar Expression (NGSE): number genes similar expression (correlation <br />                       coefficient &gt;0.8) <br />  MCC                  Maximum Correlation Coefficients (MCC): maximum value correlation coefficients neighboring genes <br />                       h) Phyletic retention <br />  PR                   Phyletic Retention (PR): number orthologs prokaryotes <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                          Page 4 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 1: List features. (Continued) <br />                        i) Codon usage <br />  Nc                    Number codons <br />  N3s                   Base composition silent sites (T3s, C3s, A3s, G3s) <br />  glt                   The frequency amino acids glutamine (exemplarily) <br />  <br />  <br /> tial genes common ecoB ecoG, 92 com-                          predicting essential genes P. aeruginosa. In turn, <br /> mon paeL paeJ. One machines                                yielded precision 61% (accuracy: 87%, sensitivity: <br /> trained different training-sets knock-out                        27%, validating ecoB) 65% (accuracy: 80%, sensi- <br /> screen. Votes training sets organism                          tivity: 18%, validating ecoG) E. coli. We yielded <br /> were summed defined stringency. A high                              best classifier results using features, com- <br /> number votes essentiality led high specificity,                      parison classification performance using <br /> while lower numbers led higher sensitivity. The result-                     individual sets features (see Additional file 1: <br /> ing receiver operator curves (ROC) classifiers                      SupplementS1 details). <br /> shown Figure 2A predicting P. aeruginosa Fig- <br /> ure 2B predicting E. coli. For predicting essential                        Examining features <br /> genes P. aeruginosa yielded area                           We wanted obtain estimate correlations of <br /> curve (AUC) 0.80 0.79 compared                              features essentiality gene. Therefore, we <br /> experimental datasets paeL paeJ, respectively. In                          calculated Pearson&apos;s correlation coefficients essen- <br /> turn, E. coli yielded AUC 0.81 0.75                      tiality class gene (1 = essential, 0 = non-essential) <br /> compared ecoB ecoG, respectively. We wanted                          corresponding feature values. Figure 3 gives an <br /> obtain reliable list potential drug targets. For this,                    overview features (see Additional file 2: <br /> predictions essential genes needed low number                         SupplementS2 correlation coefficients features). <br /> false positives. Hence, set high stringency calcu-                    In following, major results cor- <br /> lated precision (true predictions predictions                   relation study. <br /> for essentiality) high selection criterion (more than <br />                                                                                Topology features <br /> 195 200 votes). We yielded precision 67% <br />                                                                                The efficiency flux deviations estimated fea- <br /> (accuracy: 87%, sensitivity: 7%, validating paeL) and <br />                                                                                tures RUP PUP gave estimate products <br /> 100% (accuracy: 80%, sensitivity: 3%, validating paeJ) <br />  <br />  <br />  <br />                                                            	 <br />   <br />        	  <br />        <br />   <br />                   <br />       <br />                        	 <br />     <br />   <br />    <br /> 	                                                <br />              <br /> 	                                                   <br />                                                  	   <br />     <br />               <br />                                                     <br />                 <br />                                                     <br />           <br />                                                                             <br />                                                            	     <br />                                                                     <br />                                                          <br />                                                 	    <br />       <br />  <br />                                                              	  <br />         <br />          <br />                                     	 <br />                                                                        	 <br /> 	  <br />         <br />                                                     	 <br /> 	 	  <br />  	     <br />  <br />                                                                                                                     <br />   	 	  <br />                                                             	 <br /> 	  	  	 <br />  	       <br />     	  <br />                                               <br />                                                                	    	 	 <br />     <br />  <br />  <br />                                                                                                                           <br />       	  <br />                                                                                                              <br />       	 <br />  Figure 1 The workflow. Essentiality data organism taken experimental genome wide knock-out screen. It used train the <br />  machine learning using features based topology metabolic network genomic transcriptomic data. The trained clas- <br />  sifier applied organism (query organism) essentiality gene metabolism predicted. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                               Page 5 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                                                                <br />  <br />  <br />  <br />  <br />                                                                                                                 <br />      <br />  <br />  <br />  <br />  <br />                                                                                                 	 <br /> 	     <br />       	 <br /> 	 	 <br />    <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                	 <br />                                               <br />                                                                                                                                                    	 <br />    <br />                                                                                                                                                    	 <br />      <br />                                                                   <br />                                                                                                                                                                 <br />                                                                                                                                     	 <br /> 	      <br />    <br />        	 	 	 <br />    <br />  <br />  <br />  <br />  <br />                                                                                                                 <br />                                                                 <br />  <br />  <br />  <br />                                                                                                 	 <br /> 	     <br />       	 <br /> 	 	 <br />    <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                      <br />                                                                 <br />                                                                   <br />                                                                                                                                                   	 <br />    <br />   <br />                                                                                                                                                   	 <br />      <br />                                                                  <br />                                                                                                                                                                 <br />                                                                                                                                     	 <br /> 	      <br />    <br />        	 	 	 <br />    <br />  <br />  <br />  Figure 2 ROC curves prediction performances. (A) 100 Support Vector Machines trained datasets ecoB ecoG, respectively, <br />  queried using datasets P. aeruginosa (union datasets paeL paeJ). The number machines predicting essentiality <br />  summed (voting score). Results varying thresholds voting score compared experimental results paeL paeJ yield- <br />  ing ROC curves (area curve: 0.80 0.79, respectively). (B) Similar (A) machines trained datasets P. <br />  aeruginosa queried datasets E. coli resulting ROC curves AUC = 0.81 0.75 datasets ecoB ecoG, respectively. <br />  <br />  <br /> of knocked-out reaction produced                         covered alternative pathways. The number of <br /> the reaction (RUP) large percentage non-                       substrates products reactions knocked <br /> producible products (PUP) was. RUP <span id='am-9' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> fea-                         gene (NS, NP) positively correlated gene <br /> ture observe mutant produce products                      essentiality (P = 4.3E-06 P = 0.0172, respectively) <br /> of knocked-out reaction. RUP = 1 set down-                     showing essential enzymes metabolize differ- <br /> stream products produced mutant                         ent compounds. Interestingly, number neighboring <br /> RUP set zero downstream product                        reactions (NNR) number neighbors neigh- <br /> could produced. The number reachable prod-                          boring reactions (NNNR) showed weak negative corre- <br /> ucts (RUP) highly negative correlated per-                        lation essentiality (P = 0.14 P = 0.091, respectively). <br /> centage unreachable products (PUP) highly positive                         This reasonable reaction high number of <br /> correlated essentiality gene (P = 1.2E-10 P                   neighboring reactions metabolites as <br /> = 2.4E-09, respectively) shown Figure 3A. If (in                    products produced alternative enzymes. <br /> silico) mutant produce down-                            The clustering coefficients (CCV) showed ten- <br /> stream products, RUP zero percentage                          dency (negatively correlated, P = 0.018) pointing to <br /> of unreachable products increased comparison                        advantageous alternative pathways. <br /> the situation products produced.                          We estimated feasibility possible flux deviations <br /> The higher percentage unreachable products                      set features describing alternative pathways. The <br /> mutant, products knocked enzyme                           number alternative pathways (ND), average path <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                 Page 6 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                                                                  	 <br />    <br /> 	    <br />  <br /> 	  <br />           	    <br />      <br />                 	     <br />  <br />     	   <br />                                       <br />                                        <br />                                                      <br />  <br />  <br />  <br />  <br />                                                                                                                                         <br />                                                                                                                                         <br />                                                                                                                                         <br />                                                                                                                                       	 <br />  <br />  <br />  <br />  <br />                                                              <br />                                                              <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />                                                               <br />                                                              <br />  <br />  <br />                                                              <br />                                                              <br />  <br />                                                            	  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                             <br />  <br />  <br />  <br />                                                               <br />                                                              <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                           <br />    <br />                                                              <br />                                                              <br />  <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                               <br />  <br />  <br />  <br />  <br />                                                  	  <br />    	  <br />            	           	          <br />  <br />     	     <br />       <br />                                      <br />               	     <br />  <br />     	   <br />                                      <br />                                      <br />                                                     <br />  <br />  <br />  <br />  <br />                                                                                                                                           <br />                                                                                                                                           <br />                                                                                                                                           <br />                                                                                                                                         	 <br />                                                               <br />                                                                <br />                                                                <br />                                                               <br />                                                                <br />                                                            !   <br />  <br />  <br />                                                                <br />                                                                <br />                                                              <br />                                                               <br />                                                               <br />                                                               <br />  <br />                                                                 <br />                                                              <br />                                                              <br />                                                              <br />                                                            	 <br />                                                             <br />  <br />                                                              <br />                                                             <br />  <br />  <br />  <br />  <br />                                                               <br />                                                               <br />                                                                 <br />                                                                <br />                                                             <br />  <br />  <br />  <br />  <br />                                                            <br />  <br />                                                              <br />                                                              <br />                                                              <br />                                                              <br />                                                             <br />                                                              <br />                                                                <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />  Figure 3 Correlation features essentiality. The feature-values gene correlated essentiality gene (1 = essential, <br />  0 = non-essential). (A) shows correlation coefficients topology features, (B) genomic transcriptomic features. High values indi- <br />  cate feature positively correlated essentiality (see Additional file 2: SupplementS2 correlation coefficients). These values were <br />  obtained gold standards (ecoB, ecoG E. coli paeJ, paeL P. aeruginosa). <br />  <br />  <br /> length deviations (APL) length                           alternative pathways. As expected, nega- <br /> shortest alternative path (LSP) described feasibility                   tively correlated essentiality (P = 0.15, P = 3.4E-04 and <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                      Page 7 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> P = 0.0063, respectively), i.e. knocked enzymes           redundant position genetic <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span>. <br /> which alternative pathways existed likely            Matching mRNA tRNA codon nucleotides less <br /> cause lethal <span id='am-90' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span> knocked out. Choke-points             robust position, translational errors are <br /> (CP) uniquely consumed produced compounds               likely occur position. However, <br /> the metabolism showed positive correlation             essential genes need stable protected in <br /> essentiality (P = 2.8E-04) choke-points diffi-       sequence. Thymine genetic <span id='am-482' about='Thesaurus:code' typeof='owl:Thing'>code</span> cope for <br /> cult replaced rest metabolism. Load-          shown thymine protected DNA and <br /> scores (LS) estimate reaction           improved efficiency DNA replication [22]. Con- <br /> involved metabolic processes. They posi-             served genes likely essential [23] thy- <br /> tively correlated essentiality (P = 9.4E-04). Between-         3rd codon position facilitates stable genetic <br /> ness centrality (BW) eccentricity (EC) strongly          inheritance off-springs cellular replicates. Inter- <br /> positive correlated essentiality (P = 1.3E-14 7.6E-        estingly, observed larger difference T3s E. coli <br /> 08, respectively) showing enzymes higher              compared P. aeruginosa. It large <br /> influence vitality placed center network.     average G C content codon position is <br /> Closeness centrality (CN) showed positive correla-         common genes P. aeruginosa [24]. These results <br /> tion (P = 0.0020). Interestingly, eigenvector centrality      low T content codon position we <br /> (EV) showed negative correlation (P = 0.0013).                  observed explain larger difference T3s for <br /> Betweenness, closeness eccentricity centrality            essential non-essential genes E. coli compared P. <br /> global centrality measures considering network          aeruginosa (see Additional file 3: SupplementS3 for <br /> while eigenvector centrality measure local           histograms). <br /> centrality computed neighbors. Note             Even decision trees performed inferior com- <br /> typically node high value eigenvector central-        pared Support Vector Machines (data shown), we <br /> ity hub (node high connectivity) hubs        used decision trees alternative approach deter- <br /> connected it. Hence, flux deviations               discriminating features gene essential- <br /> likely local hubs hubs vicinity mak-       ity. Decision trees trained datasets ecoB, <br /> ing node replaceable global central nodes             ecoG, paeL paeJ. It known decision of <br /> seemed generally substantial maintaining            decision tree performed best discriminating <br /> metabolic flow network. Therefore eigenvector          feature [25]. Therefore, counted occurrence our <br /> centrality network topology          features position run. Out 400 runs, <br /> sense clustering coefficient, specifically respect      got T3s (137&#195;&#8212;), BW (81&#195;&#8212;), Nc (38&#195;&#8212;), H3 (21&#195;&#8212;), H5 <br /> to likelihood alternative pathways.                        (15&#195;&#8212;), val (12&#195;&#8212;), H7 (11&#195;&#8212;), NGSE (11&#195;&#8212;), PUP (10&#195;&#8212;) the <br />                                                                   decision (features 10 counts not <br /> Genomic transcriptomic features                               shown). This showed feature T3s sub- <br /> As expected, number homologous genes (H30,                 stantially supported classification. <br /> H20, H10, H7, H5, H3) showed negative correlation to <br /> essentiality (P = 3.2E-04, 6.3E-04, 1.4E-06, 4.7E-09, 1.1E-       Identifying drug targets S. typhimurium <br /> 10, 1.5E-09, respectively). Interestingly, E-value cut-off     We applied trained machines datasets <br /> of 10-5 (H5) worked best showing non-perfectly          (ecoB, ecoG, paeL, paeJ) predict essential genes for <br /> matching sequences functions                 S. typhimurium obtained votes hundred <br /> knocked gene. The number genes having similar              machines gene S. typhimurium essential. <br /> expression (NGSE) exhibited negative correlation           To obtain reasonable threshold number votes <br /> to essentiality (P = 1.7E-04) co-             predicting gene essential, compared num- <br /> expression genes analogous function. For fea-         ber essentially predicted genes numbers the <br /> ture phyletic retention (PR), number prokaryotes           training sets E. coli P. aeruginosa. For E. coli 104 <br /> having orthologs knocked gene showed posi-           147 genes essential corresponding data- <br /> tive correlation essentiality (P = 2.1E-16) supporting         sets ecoB ecoG, respectively, P. aeruginosa, 92 <br /> the findings previous study conserved genes          150 (corresponding datasets paeL paeJ, <br /> evolution hint essentiality [14].                       respectively). Therefore, set threshold 350 votes <br />   We analyzed codon usage gene related           (of 400 machines) classify gene essential S. <br /> these essentiality gene. We genes        typhimurium obtained comparable 128 <br /> with high number nucleotide thymine         predicted essential genes. The complete list genes <br /> position codons likely essential       predicted essential given supplement <br /> cell viability (feature T3s Figure 3, Additional file 3:   (see Additional file 4: SupplementS4). We compared <br /> SupplementS3 histograms). The codon position            results experimental data Knuth co- <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                   Page 8 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> workers performed large knock-out study S.            * pyrH: The gene uridylate kinase essential in <br /> typhimurium [21]. They detected 6% open reading        Mycobacterium tuberculosis [32]. <br /> frames essential including 53 essential genes          ** uppS: Undecaprenyl pyrophosphate synthase (UPPS) <br /> coding enzymes metabolism. For remaining           novel antibacterial target Streptococcus pneumo- <br /> open reading frames genome didn&apos;t make        niae [33]. <br /> prediction, including 711 genes enzymes metabo-          ** accA: Acetyl-CoA carboxylase drug target for <br /> lism. We compared list essential genes Knuth        anti-obesity antibiotic drugs [34,35]. <br /> and co-workers predictions 27         * hemH: Ferrochelatase essential multiplication <br /> predicted genes list Knuth co-workers yield-    intracellular survival Brucella abortus [36]. <br /> ing precision 21%, accuracy 83% sensitivity      * kdsB: Analogs 3-deoxy-D-manno-octulosonate <br /> of 51%. It note experimental screen Knuth   (KDO) designed inhibit CTP:CMP-KDO cytidy- <br /> and co-workers comprehensive; authors             lyltransferase (CMP-KDO synthetase) [37]. It poten- <br /> stated article genes pre-     tial target Haemophilus influenzae [38] E.coli [39]. <br /> dicted essential, couldn&apos;t conclude          * fabD: Acyl carrier protein S-malonyltransferase a <br /> genes definitely non-essential. Therefore, novel      potential target Mycobacterium bovis BCG [38]. <br /> predictions suit potential new targets       ** fabG: 3-ketoacyl-(acyl-carrier-protein) reductase a <br /> investigations. As conservative robust estimate      well-known drug target E. coli, B. subtilis, S. aureus <br /> essential genes S. typhimurium, defined corre-     [40]. <br /> sponding enzymes genes experimentally             * pssA: Phosphatidylserine synthetase required for <br /> determined (by Knuth co-workers) recog-          motility chemotaxis E. coli [41]. Furthermore, <br /> nized classifiers. We searched litera-     mutants Escherichia coli K12 defective in <br /> ture drug treatments enzymes       phosphatidylserine synthetase, isolated tempera- <br /> micro-organisms. The results listed following.     ture-sensitive, conditional lethals [42]. <br /> Two asterisks (**) mark gene enzyme              * kdtA: 3-deoxy-D-manno-octulosonic-acid trans- <br /> found clear evidence drug target micro-         ferase enzyme kdtA. In E. coli, essential for <br /> organism. One asterisk (*) set gene         cell growth accounts conditional lethality associ- <br /> found reasonable evidence enzyme serve        ated mutations KDO biosynthesis [43]. <br /> drug target micro-organism. Hence, enzymes           ** dfp: Pantothenate kinase uptream phosphopan- <br /> one asterisk serve finding new drug targets       tothenoylcysteine decarboxylase biosynthesis of <br /> enzymes asterisks transferring drug targets      pantothenate CoA. It well-known target anti- <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> bacterial diseases disease studied (S.   microbial drugs E. coli Mycobacterium tuber- <br /> typhimurium). Table 2 gives overview results.       culosis [44]. <br /> We compared open reading frames predicted            * rep: ATP-dependent DNA helicase Rep. Its deletion <br /> genes human transcripts did detect sig-      lethal B. subtilis [45] Stapphylo- <br /> nificant homologs (using BLAST [26] ENSEMBL               coccus aureus [46]. <br /> cDNA transcripts [27]). E-values best hits given     * rpoB: DNA-directed RNA polymerase promising <br /> in supplement (see Additional file 5: SupplementS5).      target discovery new antimicrobial agents <br /> Here brief summary literature evidences      E. coli [47]. <br /> found: <br />    ** murE: UDP-N-acetylmuramoylalanyl-D-glutamate-           The non-mevalonate pathway fatty acid biosynthesis <br /> 2,6-diaminopimelate ligase essential enzyme       highly enriched essential genes S. typhimurium <br /> well-known target bacterial cell walls Staphylo-   We performed gene set enrichment tests (Fisher&apos;s exact <br /> coccus aureus [28].                                           tests) pathways KEGG [48] sig- <br />    * murG: N-acetylglucosaminyl transferase potential    nificant enrichment essential genes non- <br /> antibiotic targeting biosynthesis bacterial peptido-   mevalonate pathway (P = 9.2E-06) fatty acid <br /> glycans. However, difficult design inhibitors    biosynthesis pathway (P = 3.8E-04). Most genes in <br /> this enzyme. Identifying inhibitors current          pathways essential (8 9 genes the <br /> research [29,30].                                             non-mevalonate pathway 8 12 genes the <br />    ** murC: UDP-N-acetylmuramate-L-alanine ligase cat-        fatty acid biosynthesis pathway. The non-mevalonate <br /> alyzes essential step pathway synthesizing      pathway (Figure 4) produces isopentenyl diphosphate <br /> peptidoglycan precursors. Recently, new inhibitors     (IPP) dimethylallyl pyrophosphate (DMAPP) that <br /> MurC enzyme successfully tested Escheri-        serve basis production sterols, dolichols, <br /> chia coli, Proteus mirabilis Klebsiella pneumoniae        ubiquinone components macromole- <br /> [31].                                                         cules prenyl groups proteins [49]. The pathway <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                           Page 9 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 2: Predicted essential genes potential drug targets. <br />  <br />   ORF              Gene Symbol         EC                 Enzyme                                                       Evidence <br />  <br />  <br />   a) Intersection predictions experimental screen <br />   STM0123          murE                6.3.2.13           UDP-N-acetylmuramoylalanyl-D-glutamate-2,6-                     ** <br />                                                           diaminopimelate ligase <br />   STM0128          murG                2.4.1.227          N-acetylglucosaminyl transferase                                * <br />   STM0129          murC                6.3.2.8            UDP-N-acetylmuramate-L-alanine ligase                           ** <br />   STM0154          lpdA                1.8.1.4            Dihydrolipoamide dehydrogenase <br />   STM0218          pyrH                2.7.4.22           Uridylate kinase                                                * <br />   STM0221          uppS                2.5.1.31           Undecaprenyl pyrophosphate synthase                             ** <br />   STM0222          cdsA                2.7.7.41           CDP-diglyceride synthase <br />   STM0228          lpxA                2.3.1.129          UDP-N-acetylglucosamine acyltransferase <br />   STM0232          accA                6.4.1.2            Acetyl-CoA carboxylase                                          ** <br />   STM0489          hemH                4.99.1.1           Ferrochelatase                                                  * <br />   STM0535          lpxH                                   UDP-2,3-diacylglucosamine hydrolase <br />   STM0542          folD                1.5.1.5, 3.5.4.9   Bifunctional 5,10-methylene-tetrahydrofolate dehydrogenase <br />   STM0988          kdsB                2.7.7.38           CTP:CMP-KDO cytidylyltransferase                                * <br />   STM1194          fabD                2.3.1.39           Acyl carrier protein S-malonyltransferase                       * <br />   STM1195          fabG                1.1.1.100          3-ketoacyl-(acyl-carrier-protein) reductase                     ** <br />   STM1200          tmk                 2.7.4.9            Thymidylate kinase <br />   STM1700          fabI                1.3.1.10           Enoyl-(acyl carrier protein) reductase <br />   STM2483          dapE                3.5.1.18           Succinyl-diaminopimelate desuccinylase <br />   STM2652          pssA                2.7.8.8            Phosphatidylserine synthase                                     * <br />   STM3090          metK                2.5.1.6 <br />   STM3415          rpoA                2.7.7.6            DNA-directed RNA polymerase subunit alpha <br />   STM3724          kdtA                                   3-deoxy-D-manno-octulosonic-acid transferase                    * <br />   STM3730          dfp                 4.1.1.36           Pantothenate kinase                                             ** <br />   STM3912          rep                 3.6.1.-            ATP-dependent DNA helicase Rep                                  * <br />   STM3978          yigC                                   3-octaprenyl-4-hydroxybenzoate decarboxylase <br />   STM4153          rpoB                2.7.7.6            DNA-directed RNA polymerase subunit beta                        * <br />   STM4154          rpoC                2.7.7.6            DNA-directed RNA polymerase subunit beta&apos; <br />   b) Predictions non-mevalonate pathway <br />   STM0049          ispH, lytB          1.17.1.2           4-hydroxy-3-methylbut-2-enyl diphosphate reductase              * <br />   STM0220          dxr                 1.1.1.267          1-deoxy-D-xylulose 5-phosphate reductoisomerase                 * <br />   STM0422          dxs                 2.2.1.7            1-deoxy-D-xylulose-5-phosphate synthase                         ** <br />   STM0423          ispA                2.5.1.10           geranyltranstransferase                                         * <br />   STM1779          ispE, ipk           2.7.1.149          4-diphosphocytidyl-2-C-methyl-D-erythritol kinase               ** <br />   STM2523          ispG, gcpE          1.17.7.1           4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase            * <br />   STM2929          ispF                4.6.1.12           2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase           * <br />   STM2930          ispD                2.7.7.60           2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase        * <br />   ** clear evidence drug target <br />   * reasonable evidence serve good drug target <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                       Page 10 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                 	 <br />       <br />                                              	 <br />       <br />    <br />                            	 <br />       <br />                     <br />                                                        <br />    <br />     <br />                                                                                                           <br />  <br />                                               <br />                                                                                       	    <br />  <br />                                                                                                               	   <br />   <br />                                                     <br />                                                <br />   <br />   <br />     <br />                                                                                        	  &quot; <br />                                                                                                        <br />           <br />          <br />         <br />   <br />    <br />       <br />                                                                                 <br />   <br />   <br />     <br />                   <br />                                                                                       	   <br /> ! <br />                                                                                                                     <br />   <br />   <br />     <br />                                                                                                         <br />           <br />       <br />  <br />                 <br />                                                                        	 <br />    <br />                                                                                                      <br />   <br />   <br />                   <br />                                                                                                       <br />   <br />   <br />           <br />       <br />                                 <br />                                                                                                        <br />       <br />                                                                                       	   <br />  <br />             <br />                                                                                             <br />           <br />       <br />                                                                                                               <br />   <br />   <br />     <br />                                                                                       	   <br /> # <br />                                                                                                                 <br />      <br />           <br />          <br />                                                                                                  !	            <br />   <br />   <br />     <br />                                                                                       	 <br />   <br />                                 <br />  <br />  <br />                                                                        	 	                        <br />             <br />   <br />   <br />     <br />                          	 <br />     <br />     <br />     <br />  <br />                                                                                         	 <br />                  <br />    	      <br />    <br />   <br />   <br />                                           	 <br />    <br />        <br />  <br />  <br />  Figure 4 The non-mevalonate pathway. The non-mevalonate pathway produces isopentenyl diphosphate (IPP). It alternative pathway bac- <br />  teria does exist human host uses mevalonate pathway produce IPP. The non-melavonate pathway highly enriched <br />  genes predicted essential. Reactions given EC number gene symbol genes corresponding en- <br />  zymes. <br />  <br />  <br /> for non-mevalonate biosynthesis considered                        irreversibility reactions the <br /> previously attractive targets novel antibiotics                     direction overall flux. This linear pathway <br /> against bacteria [50,51] including S. typhimurium [52,53].                 starts 1-deoxy-D-xylulose-5-phosphate-synthase (EC <br /> Figure 4 shows non-mevalonate pathway                          2.2.1.7) corresponding gene dxs identi- <br /> essential enzymes S. typhimurium. Note                        fied essential experimental knock-out <br /> arrows figure represent information                    study Knuth co-workers [21]. The six <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                   Page 11 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> enzymes downstream predicted essential            separated internal nodes external nodes to <br /> our method. The enzyme pathway            reduce computational complexity [54]). <br /> was geranyltranstransferase (EC 2.5.1.10). It catalyzes         Our method served estimate genes their <br /> reaction produce farnesyl-diphosphate. Recently, Cor-        corresponding enzymes bacterial organisms E. <br /> nish co-workers performed elaborated mutagene-           coli, P. aeruginosa S. typhimurium. Inferring essenti- <br /> sis study non-melavonate pathway S.                   ality information organism organism <br /> typhimurium genes essential (ispD,         facilitated investigating closely related organ- <br /> ispE, ispF, ispG, ispH) [52]. We propose         ism did S. typhimurium closely <br /> enzymes pathway promising potential drug            related E. coli. To apply method micro- <br /> targets S. typhimurium. We searched literature       organisms, metabolism need well <br /> and indicated findings asterisks          described, method need adaptation less <br /> described above. For genes clear evidences         studied organisms special metabolic capa- <br /> (two asterisks) genes reasonable evidences          bilities. For applying method eukaryotic genomes, <br /> (one asterisks) code drug targets (see Additional file   compartments cell reaction occurs <br /> 5: SupplementS5).                                               need considered. It challenging to <br />                                                                 apply method inferring multiple drug targets <br /> Conclusions                                                     using experimental double knock-out screens e.g. of <br /> We established machine learning approach pre-            synthetic lethal project eSGA [55]. For this, attri- <br /> dicts essentiality genes organism         butes related single players (like e.g. sequence <br /> experimental knock-out data available. The classifiers       features) relevance specifically net- <br /> were trained essentiality information genes         work features related synergistic <br /> one organism (e.g. E. coli) employed predict        knock-out effects. The challenge future remains to <br /> essential genes organism (e.g. P. aeruginosa).     integrate topological descriptive approaches with <br /> These predictions did depend essentiality infor-         genetic information systematically explore network <br /> mation query organism predictions          effects enzyme treatments combinations thereof. <br /> were made, solely features calculated <br /> from metabolic network genomic transcrip-           Methods <br /> tomic information query organism. Such data           Network reconstruction <br /> abundantly available pathogenic bacteria. We           The metabolic networks E. coli, P. aeruginosa, S. <br /> applied method predict essential genes Salmo-        typhimurium reconstructed using database of <br /> nella typhimurium query organism         KEGG [48]. Unspecific compounds water, ATP, <br /> proposed 35 potential drug targets. 27 targets resulted         etc. discarded. Additionally, main com- <br /> from intersection predictions            pounds reactions annotated KGML files of <br /> experimental study [21] 8 targets non-               KEGG used. Except centrality features and <br /> mevalonate pathway statistical              clustering coefficients, topology features calcu- <br /> enrichment analysis. The non-mevalonate pathway              lated representation network bipartite <br /> essential algae, plants eubacteria including     graph consisting metabolites reactions alternat- <br /> pathogenic bacteria. Enzymes pathway          ing nodes. For calculating centrality features the <br /> described potential targets development        clustering coefficients, represented metabolic net- <br /> novel antibiotics herbicidal agents [50,52]. For S.         work undirected graph, known reaction- <br /> typhimurium, inferred machine learn-        pair network. It consisted reactions nodes and <br /> ing approach. We discovered interesting correlations         metabolites edges connecting reactions the <br /> our features essentiality gene. Various features    graph. Two reactions connected metabolite if <br /> describing network topology served machine           metabolite product reactions a <br /> select reactions showed possible pathways flux      substrate reaction. Edges discarded <br /> deviations, e.g. linear non-mevalonate pathway.       network loops one <br /> An intelligent combination features seen        edge different nodes. Reactions were <br /> as alternative approach established methods        mapped enzymes enzymes mapped corre- <br /> flux balance analyses (FBA) elementary flux modes           sponding genes using association tables KEGG. <br /> (EFM) detailed growth <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> information            Genes corresponded death-end reactions the <br /> lacking (which needed FBA, [3] in-         network included datasets training <br /> depth refinement metabolic network considered         validation. If gene corresponded one <br /> to labor intensive (in EFM enzymes need        reaction, mean value reaction features was <br />                                                                 taken. For Boolean features (RUP, DIR, CP, below) <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                      Page 12 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> we used Boolean OR-operation, i.e. gene feature         defined knocked reaction sub- <br /> set one reaction feature equaled one.       stantial producing downstream metabolites if <br />                                                                   products produced pathways. <br /> The gold standards                                                The investigation tested knocked reaction <br /> To train validate predictions, used published          performed following algorithm: <br /> datasets genome wide experimental knock-out                     i. All metabolites acting input nodes (substrates) and <br /> screens. All performed LB rich medium. Two                output nodes (products) knocked reaction <br /> datasets E. coli P. aeruginosa. Addi-          selected. The set substrates S defined input <br /> tionally used data knock-out study S. typh-          nodes set products P defined output nodes. <br /> imurium performed LB rich medium.               To broader list available substrates integrated <br /> All datasets taken NMPDR database [56].             substrates S. We included substrates the <br /> For E. coli, used KEIO collection Baba co-          upstream reactions products downstream <br /> workers [17] denoted &apos;ecoB&apos;. It consisted          reactions sets S P, respectively. Substrates of <br /> 104 essential 641 non-essential genes meta-           reactions substrates S sub- <br /> bolic network. The dataset E. coli              strate included S. Further, substrates reac- <br /> Gerdes co-workers [18] denoted &apos;ecoG&apos;.            tions metabolite P substrate were <br /> It consisted 147 essential genes 533 non-essential         included S. <br /> genes network. For P. aeruginosa used data            ii. Reactions selected used available <br /> of Liberati et al. [20] denoted &apos;paeL&apos;. It consisted 92     compounds substrates. <br /> essential genes 615 non-essential genes net-             iii. These selected reactions products were <br /> work. The dataset P. aeruginosa taken          incorporated list discovered reactions and <br /> the study Jacobs et al. [19]. We denoted &apos;paeJ&apos;. It   products. The products set newly available <br /> consisted 150 essential genes 579 non-essential            metabolites network. <br /> genes. The experimental dataset S. typhimurium               iv. Steps ii iii repeated reac- <br /> from Knuth co-workers [21] based insertion-            tions identified. <br /> duplication mutagenesis (IDM). Small, randomly gener-                v. The output nodes produced were <br /> ated genomic fragments cloned conditionally           counted (reachable products P). <br /> replicating vector, resulting library single S.           After finishing process, used number of <br /> typhimurium clones grown permissive condi-              defined output nodes produced the <br /> tions. Upon switching non-permissive temperature,              mutated network features, i.e. quality feature <br /> discrimination lethal non-lethal insertions           defining products produced (RUP, reach- <br /> following homologous recombination allowed trap-              able/unreachable products), percentage prod- <br /> ping genes essential functions. With method,         ucts produced (PUP, percentage of <br /> genes detected indispensable growth.           unreachable products). We run breadth first <br /> However, non-essential genes determined.             search network estimate possible deviations. <br /> For metabolism, 53 genes essential           Starting S, breadth search explored net- <br /> and remaining 711 essentiality           work finding direct products knocked out <br /> determined method [21].                                   reaction. When algorithm visited products, it <br />                                                                   stored corresponding pathway continued its <br /> Defining features <br />                                                                   search alternative paths network <br /> Features obtained network topology proper- <br />                                                                   entirely explored maximal path length 10 <br /> ties genomic transcriptomic information. Table 1 <br />                                                                   reactions reached. The organism many <br /> shows overview features abbreviations. <br />                                                                   pathways produce products making system <br /> Topology based features                                           robust. Thus, counted number possible <br /> a) Deviation features                                             alternative paths yielding feature ND (ND, number of <br /> For following features, used metabolic net-            deviations). We took average path length (APL, aver- <br /> work representation bipartite graph consisting        age path length) shortest path length (LSP, length <br /> of different alternating nodes, i.e. metabolites          shortest path) deviations features clas- <br /> reactions. As reported recently [6,16], implemented          sifier. The deviation features used alternative <br /> breadth algorithm investigate network         pathways produce products knocked reac- <br /> single reaction blocked. We defined reaction             tion substrates S. In metabolic network, these <br /> essential survival basically mutated network         substrates consumed reactions yield- <br /> could yield products reaction                 ing products etc. Therefore, kept track alter- <br /> upstream substrates reaction. Hence, features were <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                         Page 13 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> native paths network potential           reactions [10]. The betweenness centrality Cb(v) a <br /> organism survive reaction blocked.               reaction v given by <br /> b) Local topology <br /> The number substrates products knocked                                             d ij (v) <br /> out reaction counted (NS: number substrates; <br /> NP: number products). We defined features the <br />                                                                       C b ( v) =      &#226;&#710;&#8216; <br />                                                                                    &#226;&#8240;&#160; j &#226;&#8240;&#160; v&#226;&#710;&#710;V <br />                                                                                                   d ij <br />                                                                                                             ,                                  (1) <br />  <br /> number Neighboring reactions (NNR) number <br /> of Neighbors Neighboring reactions (NNNR). We cal- <br />                                                                  dij number shortest paths reac- <br /> culated clustering coefficient (CCV) described in <br /> [57,58] knocked reaction estimate local      tion reaction j, dij(v) number shortest <br /> density network. The reaction direction (DIR,           paths j pass reaction v. The sum is <br /> directionality reactions) taken KEGG set       composed pairs (i, j) reactions network. <br /> as reversible information available.           Closeness centrality approximates edges are <br /> c) Choke-points load-scores                                required access reaction given reac- <br /> A reaction uniquely consumes produces certain        tion [10]. It defined inverse average length <br /> metabolite metabolic network considered            shortest paths reactions. The close- <br /> choke point. Such reaction shows high potential          ness centrality Cc(v) reaction v given by <br /> essentiality [4,5]. We used feature (CP, choke <br /> points). According concept [4], load scores                                    n &#226;&#710;&#8217;1 <br /> defined hot spots network based ratio             C c ( v) = <br />                                                                                          &#226;&#710;&#8216; d vi                                                (2) <br /> the number k-shortest paths passing reac-                              &#226;&#8240;&#160; v ,i&#226;&#710;&#710;V <br /> tion, number nearest neighbor links attached <br /> to it. This ratio compared average load value in <br />                                                                  n number reactions network. <br /> the network. <br /> d) Damage <br />                                                                Eccentricity longest distance given reac- <br />                                                                tion reaction [61]. The eccentricity Ce(v) for <br /> The damage defined determine potentially <br /> effected metabolites reactions downstream           vertex v given by <br /> knocked reaction. We used <span id='am-7' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> damaged <br /> compounds reactions Lemke co-workers                                           n &#226;&#710;&#8217;1 <br />                                                                       C e ( v) =                   .                                           (3) <br /> [7] yielding features NDR (NDR, number damaged                                  max (d vi ) <br /> reactions) NDC (number damaged compounds).                                   &#226;&#8240;&#160; v ,i&#226;&#710;&#710;V <br /> As half reactions annotated revers- <br /> ible, compounds reactions                Eigenvector centrality based assumption that <br /> inferred damage actually just back-traced alter-    utility reaction determined utility the <br /> native pathways. Therefore, calculated number        neighboring reactions [62]. It scores reaction higher it <br /> damaged compounds reactions network               connected high-scoring reactions. It defined the <br /> which alternative pathways discarded yielding         principal eigenvector adjacency matrix net- <br /> NDRD (number damaged reactions devia-               work. Let xi denote score reaction i. Let Aij the <br /> tions) NDCD (number damaged compounds with-             adjacency matrix network. Thus Aij = 1 is <br /> out deviations). In addition, calculated number of <br />                                                                edge reactions j, Aij = 0 otherwise. <br /> damaged choke points (NDCR, number damaged <br /> choke point reactions; NDCC, number damaged choke           For reaction i, centrality score proportional the <br /> point compounds; NDCRD, number damaged choke                average centralities i&apos;s network neighbors: <br /> point reactions deviations; NDCCD, number of <br />                                                                                                                         n <br /> damaged choke point compounds deviations). <br />                                                                                        &#226;&#710;&#8216;                                &#226;&#710;&#8216;A x , <br />                                                                              1                                      1 <br /> e) Centrality                                                         xi =                          xj          =              ij j            (4) <br />                                                                              l                                      l <br /> For features, used network-representation                          j&#226;&#710;&#710;Neighbor (i)                       j =1 <br />  <br /> a reaction-pair network. We computed centrality fea- <br /> tures using R package &apos;igraph&apos; [59] consisting         Neighbor(i) set neighboring reactions <br /> betweenness centrality (BC), closeness centrality (CN),        reaction i, n total number reactions &#206;&#187; a <br /> eccentricity (EC) [10], eigenvector centrality (EV)        constant. This leads directly well-known eigenvec- <br /> [60]. Let G(V, E) simple undirected graph n ver-     tor equation, Ax = &#206;&#187;x. Normally, different eigen- <br /> tices (reactions). Betweenness measures frequency       values &#206;&#187; eigenvector solution exists. <br /> a reaction (node) shortest path pairs   According Perron-Frobenius theorem the <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                          Page 14 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> eigenvector largest eigenvalue eigenvector            T3s, C3s, A3s, G3s thymine, cytosine, adenine and <br /> centrality [60].                                                    guanine, respectively. Additionally, number codons <br />                                                                     coding encoded amino acids (phe, ser, tyr, cys, leu, <br /> Genomic transcriptomic features                                 trp, pro, his, arg, gln, ile, met, thr, asn, lys, val, ala, asp, <br /> f) Homologs                                                         glu, gly) counted. All codon counts normalized <br /> We calculated number homologous genes                   division total number codons (Nc). Nc was <br /> might taken function knocked               used feature. <br /> gene. Homologous genes searched using BLAST <br /> [26] open reading frames respective              The machine learning system <br /> organism (E. coli, P. aeruginosa, S. typhimurium). We               We used Support Vector Machines R package <br /> used different E-value cutoffs, i.e. 10-3, 10-5, 10-7, 10-10, 10-   &apos;e1071&apos; classify essential non-essential genes of <br /> 20, 10-30 obtain features H3, H5, H7, H10, H20           metabolism http://www.r-project.org. A radial basis <br /> and H30, respectively. Sequences open reading                function used kernel. Parameter optimization <br /> frames taken NCBI database (http://                   performed training data regularization <br /> www.ncbi.nlm.nih.gov/, E. coli: [GenBank:NC_000913],                term kernel width. The regularization term <br /> P. aeruginosa: [GenBank:NC_002516], S. typhimu-                 defined costs false classifications opti- <br /> rium: [GenBank:NC_003197]).                                         mized using values 2n n = -4, -2, 0, 2, 4. The same <br /> g) Gene expression                                                  range taken kernel width. This optimization <br /> We collected gene expression data investi-            realized training grid search combi- <br /> gated organisms public resources. The datasets                 nations parameters. The sizes classes <br /> were selected respect unspecific regu-          differed considerably data sets (essential genes: 8 - <br /> lation, i.e. treatments affecting small band         15%, non-essential genes: 85 - 92%). For broad spec- <br /> a broad range metabolic pathways. For E. coli, used           trum different sensitivities specificities, applied <br /> gene expression data study regulation           voting scheme. We trained 100 Support Vector <br /> during oxygen deprivation investigated [63], P.             Machines (SVMs) essential genes equal <br /> aeruginosa study observing response agma-             randomly selected non-essential genes. With <br /> tine putrescine treatment [64] study              this, stratified training data. For classification <br /> quorum-sensing response environmental conditions                 query gene, output machines summed <br /> [65]. For S. typhimurium used data cells treated              used voting score gene essential <br /> with <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation different time points [66]          cell. <br /> data study captured regulatory response <br /> in environment host [67]. The data nor-              Defining discriminating features decision <br /> malized variance stabilization normalization [68].               trees <br /> Genes similar functionality pathway                The decision decision trees applies best dis- <br /> often co-regulation [69]. Therefore, maximum               criminating feature [25]. Therefore, used decision <br /> correlation coefficient (MCC) neighboring reac-              trees alternative approach defining dis- <br /> tions knocked reaction used feature.            criminating features. We applied method decision <br /> Additionally, calculated number reactions            trees using R package &apos;rpart&apos; http://www.r-project.org <br /> similar gene expression (NGSE, correlation coefficient &gt;            classify essential non-essential genes metab- <br /> 0.8) used features estimate co-regulated        olism. Gini impurity [70] used splitting data. <br /> analogous genes.                                                    The minimum number observations exist <br /> h) Phyletic retention                                               node 20 maximum depth 30. We <br /> We selected 177 prokaryotic organisms (except E. coli, P.           stratified training data manner for <br /> aeruginosa, S. typhimurium) described                     training Support Vector Machines. One hundred <br /> Gustafson et al. [14] counted number            decision trees generated gold standard <br /> of organisms having open reading frame                  (ecoB, ecoG, paeL paeJ). To discriminat- <br /> homologous sequence knocked gene. This               ing features, run decision tree <br /> was performed E. coli, P. aeruginosa, S. typhimu-          selected. <br /> rium using bi-directional best BLAST hits (E-value cutoff <br />                                                                     Performance measures statistics <br /> of 0.1). <br />                                                                     For assessing performance classifier, predic- <br /> i) Codon usage <br />                                                                     tions compared gold standard exper- <br /> Codons counted investigated gene its <br />                                                                     imental screens. A prediction true positive (tp, <br /> coding region. We counted base compositions silent <br />                                                                     prediction: essential, gold standard: essential), false posi- <br /> sites (third position codons) yielding features <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                                                                                                                                                                                                                              Page 15 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> tive (fp, prediction: essential, gold standard: non-essen-                                                                                                                                                                                                                               5.    Yeh I, Hanekamp T, Tsoka S, Karp PD, Altman RB: Computational analysis <br />                                                                                                                                                                                                                                                                                                Plasmodium falciparum metabolism: organizing genomic <br /> tial), true negative (tn, prediction: non-essential, gold                                                                                                                                                                                                                                      information facilitate drug discovery. Genome Res 2004, 14:917-924. <br /> standard: non-essential) false negative (fn, prediction:                                                                                                                                                                                                                              6.    Fatumo S, Plaimas K, Mallm JP, Schramm G, Adebiyi E, Oswald M, Eils R, <br /> non-essential, gold standard: essential). We calculated                                                                                                                                                                                                                                    Konig R: Estimating novel potential drug targets Plasmodium <br />                                                                                                                                                                                                                                                                                                falciparum analysing metabolic network knock-out strains in <br /> standard measures accuracy = (tp+tn)/(tp+tn+fp+fn),                                                                                                                                                                                                                                            silico. Infect Genet Evol 2009, 9:351-358. <br /> sensitivity = tp/(tp+fn), specificity = tn/(tn+fp). A                                                                                                                                                                                                                                7.    Lemke N, Heredia F, Barcellos CK, Dos Reis AN, Mombach JC: Essentiality <br /> receiver operator characteristics (ROC-curve) used                                                                                                                                                                                                                                         damage metabolic networks. Bioinformatics 2004, 20:115-119. <br />                                                                                                                                                                                                                                                                                          8.    Gursoy A, Keskin O, Nussinov R: Topological properties protein <br /> to measure performance classifier                                                                                                                                                                                                                                        interaction networks structural perspective. Biochem Soc Trans <br /> various thresholds. In ROC-curve sensitivity                                                                                                                                                                                                                                        2008, 36:1398-1403. <br /> plotted 1 - specificity area                                                                                                                                                                                                                                   9.    Hahn MW, Kern AD: Comparative genomics centrality and <br />                                                                                                                                                                                                                                                                                                essentiality eukaryotic protein-interaction networks. Mol Biol <br /> curve (AUC) yields performance estimate                                                                                                                                                                                                                                           Evol 2005, 22:803-806. <br /> entire range thresholds. P-values Pearson&apos;s correla-                                                                                                                                                                                                                              10.   Estrada E: Virtual identification essential proteins protein <br /> tion coefficients features calculated                                                                                                                                                                                                                                          interaction network yeast. Proteomics 2006, 6:35-40. <br />                                                                                                                                                                                                                                                                                          11.   Przulj N, Wigle DA, Jurisica I: Functional topology network protein <br /> described [71].                                                                                                                                                                                                                                                                             interactions. Bioinformatics 2004, 20:340-348. <br />                                                                                                                                                                                                                                                                                          12.   Acencio ML, Lemke N: Towards prediction essential genes by <br /> Additional material                                                                                                                                                                                                                                                                            integration network topology, cellular localization biological <br />                                                                                                                                                                                                                                                                                                process information. BMC Bioinformatics 2009, 10:290. <br />                                                                                                                                                                                                                                                                                          13.   Seringhaus M, Paccanaro A, Borneman A, Snyder M, Gerstein M: <br />       Additional file 1 ROC curves essential gene predictions                                                                                                                                                                                                                     Predicting essential genes fungal genomes. Genome Res 2006, <br />       subsets features.                                                                                                                                                                                                                                                                     16:1126-1135. <br />       Additional file 2 Correlation coefficients features gene                                                                                                                                                                                                                 14.   Gustafson AM, Snitkin ES, Parker SC, DeLisi C, Kasif S: Towards the <br />       essentiality.                                                                                                                                                                                                                                                                            identification essential genes using targeted genome sequencing <br />       Additional file 3 Histograms frequency T3s essential                                                                                                                                                                                                                       comparative analysis. BMC Genomics 2006, 7:265. <br />       genes non-essential genes.                                                                                                                                                                                                                                                     15.   Hwang YC, Lin CC, Chang JY, Mori H, Juan HF, Huang HC: Predicting <br />       Additional file 4 All examined metabolic genes S. typhimurium                                                                                                                                                                                                                         essential genes based network sequence analysis. Mol Biosyst <br />       computational predictions experimental results                                                                                                                                                                                                                       2009, 5:1672-8. <br />       Knuth et al. (2004).                                                                                                                                                                                                                                                               16.   Plaimas K, Mallm JP, Oswald M, Svara F, Sourjik V, Eils R, Konig R: Machine <br />       Additional file 5 Predicted essential genes potential drug targets                                                                                                                                                                                                                   learning based analyses metabolic networks supports high- <br />       S. typhimurium literature evidences.                                                                                                                                                                                                                                       throughput knockout screens. BMC Syst Biol 2008, 2:67. <br />                                                                                                                                                                                                                                                                                          17.   Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br />                                                                                                                                                                                                                                                                                                Tomita M, Wanner BL, Mori H: Construction Escherichia coli K-12 in- <br /> Authors&apos; contributions                                                                                                                                                                                                                                                                         frame, single-<span id='am-324' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-325' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: Keio collection. Mol Syst Biol <br /> KP RK general concept design study. KP carried                                                                                                                                                                                                                   2006, 2:2006 0008. <br /> the data analysis. KP RK drafted manuscript. All authors read                                                                                                                                                                                                                18.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD, <br /> approved final manuscript.                                                                                                                                                                                                                                                                 Somera AL, Kyrpides NC, Anderson I, Gelfand MS, et al.: Experimental <br />                                                                                                                                                                                                                                                                                                determination level analysis essential genes in <br /> Acknowledgements                                                                                                                                                                                                                                                                               Escherichia coli MG1655. J Bacteriol 2003, 185:5673-5684. <br /> This work funded BMBF-FORSYS consortium Viroquant                                                                                                                                                                                                                         19.   Jacobs MA, Alwood A, Thaipisuttikul I, Spencer D, Haugen E, Ernst S, Will O, <br /> [#0313923]; Nationales Genom-Forschungs-Netz (NGFN+) neuro-                                                                                                                                                                                                                        Kaul R, Raymond C, Levy R, et al.: Comprehensive transposon mutant <br /> blastoma project ENGINE, Commission Higher Education (CHE)                                                                                                                                                                                                                       library Pseudomonas aeruginosa. Proc Natl Acad Sci USA 2003, <br /> Thailand.                                                                                                                                                                                                                                                                                      100:14339-14344. <br />                                                                                                                                                                                                                                                                                          20.   Liberati NT, Urbach JM, Miyata S, Lee DG, Drenkard E, Wu G, Villanueva J, <br /> Author Details                                                                                                                                                                                                                                                                                 Wei T, Ausubel FM: An ordered, nonredundant library Pseudomonas <br /> 1Department Bioinformatics Functional Genomics, Institute                                                                                                                                                                                                                            aeruginosa strain PA14 transposon insertion mutants. Proc Natl Acad <br /> Pharmacy Molecular Biotechnology, Bioquant, University Heidelberg, Im                                                                                                                                                                                                                   Sci USA 2006, 103:2833-2838. <br /> Neuenheimer Feld 267, 69120 Heidelberg, Germany 2B080 Division                                                                                                                                                                                                                    21.   Knuth K, Niesalla H, Hueck CJ, Fuchs TM: Large-scale identification of <br /> Theoretical Bioinformatics, German Cancer Research Center (DKFZ), Im                                                                                                                                                                                                                           essential Salmonella genes trapping lethal insertions. Mol Microbiol <br /> Neuenheimer Feld 280, 69120 Heidelberg, Germany                                                                                                                                                                                                                                                2004, 51:1729-1744. <br />                                                                                                                                                                                                                                                                                          22.   Leon PE: Inhibition ribozymes deoxyribonucleotides the <br /> Received: 17 December 2009 Accepted: 3 May 2010                                                                                                                                                                                                                                                origin DNA. J Mol Evol 1998, 47:122-126. <br /> Published: 3 May 2010                                                                                                                                                                                                                                                                    23.   Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau- <br /> &#194;&#169; 2010 <br /> This <br /> BMC  Open <br />      article <br />       Systems <br />          Plaimas   Access <br />              Biology <br />                 available <br />                    et al; 2010, <br />                              article <br />                           licensee <br />                             from:    BioMed Central <br />                                      distributed <br />                                   4:56 <br />                                     http://www.biomedcentral.com/1752-0509/4/56 <br />                                                  under <br />                                                     Ltd. <br />                                                        terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />                                                                                                                                                                                                                                                                                                Danila A, Anderson K, Andre B, et al.: Functional profiling the <br /> References                                                                                                                                                                                                                                                                                     Saccharomyces cerevisiae genome. Nature 2002, 418:387-391. <br /> 1. Hopkins AL, Groom CR: The druggable genome. Nat Rev Drug Discov                                                                                                                                                                                                                       24.   Grocock RJ, Sharp PM: Synonymous codon usage Pseudomonas <br />     2002, 1:727-730.                                                                                                                                                                                                                                                                           aeruginosa PA01. Gene 2002, 289:131-139. <br /> 2. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD,                                                                                                                                                                                                                       25.   Witten IH, Frank E: Data Mining, Practical Machine Learning Tools and <br />     Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale metabolic                                                                                                                                                                                                                       Techniques San Francisco: Morgan Kaufmann; 2005. <br />     reconstruction Escherichia coli K-12 MG1655 accounts 1260                                                                                                                                                                                                               26.   Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lipman <br />     ORFs thermodynamic information. Mol Syst Biol 2007, 3:121.                                                                                                                                                                                                                             DJ: Gapped BLAST PSI-BLAST: new generation protein <br /> 3. Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objective                                                                                                                                                                                                                           database search programs. Nucleic Acids Res 1997, 25:3389-3402. <br />     functions predicting intracellular fluxes Escherichia coli. Mol Syst                                                                                                                                                                                                          27.   Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M, Chen Y, Clarke L, <br />     Biol 2007, 3:119.                                                                                                                                                                                                                                                                          Coates G, Cunningham F, Cutts T, et al.: Ensembl 2007. Nucleic Acids Res <br /> 4. Rahman SA, Schomburg D: Observing local global properties                                                                                                                                                                                                                            2007, 35:D610-617. <br />     metabolic pathways: &apos;load points&apos; &apos;choke points&apos; metabolic                                                                                                                                                                                                                28.   Bratkovic T, Lunder M, Urleb U, Strukelj B: Peptide inhibitors MurD and <br />     networks. Bioinformatics 2006, 22:1767-1774.                                                                                                                                                                                                                                               MurE, essential enzymes bacterial cell wall biosynthesis. J Basic <br />                                                                                                                                                                                                                                                                                                Microbiol 2008, 48:202-206. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                           Page 16 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br /> 29. Kotnik M, Anderluh PS, Prezelj A: Development novel inhibitors                    xylulose-5-phosphate reductoisomerase (DXR) enzyme. Curr Pharm <br />     targeting intracellular steps peptidoglycan biosynthesis. Curr Pharm              Des 2007, 13:1161-1177. <br />     Des 2007, 13:2283-2309.                                                        52.   Cornish RM, Roth JR, Poulter CD: Lethal mutations isoprenoid <br /> 30. Helm JS, Hu Y, Chen L, Gross B, Walker S: Identification active-site              pathway Salmonella enterica. J Bacteriol 2006, 188:1444-1450. <br />     inhibitors MurG using generalizable, high-throughput                      53.   Testa CA, Cornish RM, Poulter CD: The sorbitol phosphotransferase <br />     glycosyltransferase screen. J Am Chem Soc 2003, 125:11168-11169.                     responsible transport 2-C-methyl-D-erythritol into <br /> 31. Zawadzke LE, Norcia M, Desbonnet CR, Wang H, Freeman-Cook K,                         Salmonella enterica serovar typhimurium. J Bacteriol 2004, <br />     Doughert <br /> </body></html>