<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF VIROLOGY, OCt. 1993, p. 5922-5931                                                                            Vol. 67, No. 10 <br /> 0022-538X/93/105922-10$02.00/0 <br /> Copyright X 1993, American Society Microbiology <br />  <br />  <br />              Two Regions Herpes Simplex Virus Type 1 UL42 <br />                Protein Are Required Its Functional Interaction <br />                         Viral DNA Polymerase <br />                        STEVEN J. MONAHAN,&apos; TAMAR F. BARLAM,2 CLYDE S. CRUMPACKER,2 <br />                                          AND DEBORAH S. PARRIS&quot;* <br />                   Department Medical Microbiology Immunology Comprehensive Cancer Center, <br />                      The Ohio State University, Columbus, Ohio 43210,1 Charles A. Dana Research <br />                           Institute Harvard-Thorndike Laboratory Beth Israel Hospital and <br />                             Department Medicine, Division Infectious Disease, Beth Israel <br />                            Hospital Harvard Medical School, Boston, Massachusetts 022152 <br />                                              Received 24 March 1993/Accepted 12 July 1993 <br />  <br />               Two essential gene products herpes simplex virus type 1, viral DNA polymerase (pol) UL42, its <br />            accessory protein, physically functionally interact form core viral DNA replication complex. <br />            Understanding essential interaction provide basis develop novel anti-herpesvirus <br />            agents. We previously shown coexpressed in vitro transcription-translation system, UL42 <br />            stimulates pol activity (M. L. Gallo, D. I. Dorsky, C. S. Crumpacker, D. S. Parris, J. Virol. 63:5023-5029, <br />            1989). By analyzing various insertion, deletion, frameshift mutations UL42 system, the <br />            C-terminal 149 amino acids dispensable ability protein stimulate pol activity. In addition, <br />            distinct internal regions UL42 required pol stimulation. Regions I II were <br />            defined lie amino acid residues 129 163 residues 202 337, respectively. When <br />            physical association examined antibody UL42, pol coimmunoprecipitate same <br />            level expressed UL42 mutant protein lacking region I wild-type UL42. Thus, mere <br />            physical association insufficient stimulation pol activity. Deletion region H reduced eliminated <br />            coimmunoprecipitation pol. Interestingly, antibody pol specific residues 1216 1224 <br />            coimmunoprecipitated UL42 proteins synthesized baculovirus expression not <br />            rabbit reticulocyte lysates. These results indicate (i) portion region recognized the <br />            pol antiserum accessible pol-UL42 heterodimer (ii) immunoprecipitation results products <br />            different expression systems vary. Thus, distinct regions UL42 essential for <br />            functional interaction pol. Moreover, results point UL42 region I function physical <br />            association pol. <br />  <br />    Herpes simplex virus type 1 (HSV-1), replicates              DNA-protein interactions (32), interactions are <br /> the nuclei eukaryotic cells, provides unique model                  likely involved eukaryotic DNA replication. Indeed, <br /> system eukaryotic DNA replication                protein-protein DNA-protein interactions involving the <br /> proteins required genome replication virus encoded               seven genes required HSV-1 on-dependent replication <br /> (4), sequence entire 152-kbp genome known                  documented. The product UL9 sequence- <br /> (34), abundance viral mutants (46) lends            specific DNA-binding protein binds HSV-1 on <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> powerful genetic analysis. Although HSV-1 DNA replica-                sequences (16, 37) possesses helicase activity important <br /> tion reconstituted vitro, transient transfec-            initiation (1, 17). Three gene products (UL5, UL8, <br /> tion mammalian cells plasmids containing HSV-1                UL52) form stable protein-protein complex which <br /> origin replication (on) various HSV-1 genes                   possesses helicase primase activities (3, 10), all <br /> revealed seven viral genes necessary sufficient             products coimmunoprecipitated HSV-1- <br /> to amplify on-containing plasmids (53). Analysis host-                infected cells specific antibody UL5 protein (55). <br /> range temperature-sensitive mutants defects                  UL5 UL52 sufficient helicase-primase enzymatic <br /> each genes confirmed seven gene                  activities (3, 14), UL8 involved stabilizing <br /> products essential viral DNA replication host cells           interaction primers DNA template (48). <br /> (for review, reference 46). However, likely             The remaining proteins-the DNA polymerase (pol, <br /> other viral and/or cellular proteins required infected            product UL30 [27, 42]), 65-kDa double-stranded DNA- <br /> cells nucleic acid metabolism cleavage            binding protein (65KDBP, product UL42 [33, 41]), the <br /> packaging <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> genomes (5, 44).                                single-stranded major DNA-binding protein (ICP8, product <br />    The proteins involved DNA replication well-charac-              UL29 [8, 52])-also reported physically <br /> terized prokaryotic systems, Eschenchia coli onC <br /> plasmids T4 T7 bacteriophages, form multipro-                interact (19, 50, 51). Results laboratory as <br /> tein complexes held protein-protein and/or                   demonstrated copurification UL42 <br />                                                                          pol standard biochemical chromatography by <br />                                                                          immunoaffinity chromatography employing monoclonal an- <br />                                                                          tibodies (MAbs) ULA2 (9, 19, 50). In cases, ICP8 <br />   * <br />       Corresponding author.                                              copurified pol UL42 (19, 50). Although HSV-1 <br />                                                                   5922 <br />  VOL. 67, 1993                                                     FUNCTIONAL INTERACrION OF HSV-1 PROTEINS                  5923 <br />  <br /> pol expressed heterologous systems possesses catalytic         in-frame insertion mutations HpaI NdeI sites, <br /> activity (15, 18, 19, 24-26, 31), UL42 stimulates pol activity    respectively. <br /> four- sevenfold (18), increases processivity pol        Deletion mutations ULA2 gene by <br /> primed single-stranded DNA templates (24, 26),            removing internal restriction fragments. The d129-163 mu- <br /> been defined pol accessory protein (18, 24, 26). ICP8    tation created digestion pLBN 19A NdeI and <br /> also reported stimulate pol activity twofold (36, 45),    AatII, removal single-stranded overhangs Si <br /> and group reported required UL42        nuclease, religation. The d141-206 <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> con- <br /> increase processivity pol (26). Moreover, pro-       structed digestion pLBN 19A NdeI HpaI, <br /> teins colocalize replication compartments          removal single-stranded overhangs Si nuclease, and <br /> productively infected cells (2, 11, 22, 23, 43).                  religation. The <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>deletion mutation</span> d37-282 (formerly desig- <br />    It shown previously carboxy-terminal         nated APst) described previously (18). To create <br /> (C-terminal) 227 amino acids pol involved           d202-337 deletion mutation, BamHI-to-EcoRI frag- <br /> sufficient complex formation UL42 basis        ment polylinker pLBN 19A removed to <br /> coimmunoprecipitation vitro transcription-translation       delete SstI site vector, 398-bp SstI <br /> products pol-specific antibody (13). Two groups         fragment UL42 gene removed form pLB <br /> attempted recently delineate C-terminal pol        ASst. Since removal SstI fragment placed C-ter- <br /> domain involved physical association UL42.       minal region frame, BamHI linker introduced at <br /> Although Tenney et al. (49) shown 19           unique SstI site create in-frame d202-337 muta- <br /> amino acids pol required functional physical    tion. <br /> association UL42, C-terminal 19 40 residues              Nested deletions UILA2 open reading frame <br /> stabilize physical association UL42,      created unique restriction sites digestion with <br /> residues required pol activity stimulated   exonuclease III (United States Biochemical, Cleveland, <br /> by UL42. By contrast, Digard et al. (12) failed observe        Ohio), removal single-stranded ends mung bean <br /> coimmunoprecipitation truncated pol polypeptides lack-         nuclease, ligation T4 DNA ligase. Deletion d140 <br /> ing C-terminal 19 residues UL42 produced           generated NdeI site pLBN 19A. The <br /> vitro translation. Thus, exact region pol         sequences constructs regions altered sub- <br /> required complex formation UL42 remains unclear.         cloned confirmed dideoxy chain termination <br /> Moreover, domains UL42 required functionally            method Sequenase 2.0 according instructions of <br /> interact pol defined. To determine         manufacturer (United States Biochemical). <br /> domains UL42 required functional physical            In vitro transcription translation. Linearized plasmids <br /> association pol, exploited fact cotrans-    transcribed T7 RNA polymerase (Promega Bio- <br /> lation products UL42 pol transcripts rabbit reticu-     tec, Madison, Wis.) 1 h 40&#194;&#176;C. Transcripts were <br /> locyte lysates possess sevenfold pol activity     translated rabbit reticulocyte lysates (Promega Biotec) for <br /> translation products pol transcripts (18). In        1 h 30&#194;&#176;C final concentration 5 20 ng RNA per <br /> present study, localized distinct regions          ,l. Cotranslation reaction mixtures contained approximately <br /> UL42 required stimulation pol activity. Our results        equal amounts UL42 pol RNA, judged gels <br /> demonstrate domains particularly sensi-      containing serial twofold dilutions RNA species. For <br /> tive disruption mutation required functional     experiments, L-[35S]methionine (specific activity, <br /> but physical interaction pol.                            1,000 1,300 Ci/mmol; Amersham, Arlington Heights, Ill.) <br />                                                                   added translation reaction mixtures final concen- <br />                 MATERIALS AND METHODS                             tration 1 ,uCi/,ul. Labelled translation products were <br />                                                                   analyzed electrophoresis denaturing, polyacryl- <br />    Plasmids. The HSV-1 pol UL42 genes subcloned          amide gradient (10 20%) gels, gels impreg- <br /> into phagemid vectors sense orientation downstream         nated En3Hance (DuPont, Boston, Mass.), dried, and <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> T7 RNA polymerase promoter. Plasmid pGEM2-               visualized fluorography (33). <br /> 702, encodes full-length pol, amino-terminal           pol assays. HSV-1 pol activity measured the <br /> deletion mutants, pT7-7.1 pT7-7.2, encode        incorporation [3H]dTTP (43 82 Ci/mmol; ICN or <br /> the 67 27 amino acids, respectively,          Amersham) trichloroacetic acid-insoluble radioactivity <br /> described previously (15). Plasmid pLBN 19A, en-            described previously (15, 18). To assay vitro translation <br /> codes full-length UL42 (18), used wild-type UL42       products, reactions containing 2 10 p,g maximally <br /> construct, mutant forms UL42 derived it.         activated calf thymus DNA template initiated with <br /> All plasmids routinely propagated E. coli JM109        25 pl translation reaction mixtures. The relative stability <br /> HB101 purified use Quiagen Maxi-Prep col-           UL42 mutant protein assayed incubation of <br /> umns (Quiagen Inc., Chatworth, Calif.) according           labelled translation mixtures pol assay conditions <br /> instructions manufacturer.                                 (with omission bovine serum albumin [BSA], deoxy- <br />    Construction UL42 gene mutations. C-terminal deletions      nucleoside triphosphates, template) 37&#194;&#176;C 30 min, <br /> of UL42 expressed runoff transcripts produced           followed sodium dodecyl sulfate-polyacrylamide gel elec- <br /> from wild-type UL42 plasmid linearized      trophoresis (SDS-PAGE) analysis. <br /> UL42 gene mutagenized insertion BamHI linkers            Preparation proteins expressed recombinant baculo- <br /> (8-, 10-, 12-mer; New England Biolabs, Beverly, Mass.)        viruses. Recombinant baculoviruses (Autographica califor- <br /> to introduce frameshift (fs) mutations. Mutant constructs         nica) express HSV-1 UL42 HSV-1 <br /> N459(fs), N248(fs), N206(fs), N140(fs) encode polypep-        pol gene polyhedrin promoter kind gifts of <br /> tides containing wild-type UL42 amino acids amino        Mark Challberg (National Institutes Health, Bethesda, <br /> terminus (N terminus) residue indicated         Md.) Robert Lehman (Stanford University, Stanford, <br /> frameshifts created NruI, NotI, HpaI, NdeI             Calif.), respectively. The wild-type strain baculovirus and <br /> restriction sites, respectively. Plasmids i206 i140 contain   Sf9 insect cell line used propagate the <br />  5924    MONAHAN ET AL.                                                                                                             J. VIROL. <br />  <br /> baculovirus strains kindly provided Fred Hink             TABLE 1. pol activities amino-terminal deletion mutants <br /> (Ohio State University, Columbus). Sf9 cells routinely                UL30 coexpressed vitro UL42&apos; <br /> propagated Grace medium (GIBCO-BRL, Grand Island, <br /> N.Y.) supplemented 10% lactose hydrolysate (Difco               UL30         Polypeptide         Pol activity (units)&apos;        Stimulation <br />                                                                    construct      composition        -UL42           +UL42           indexc <br /> Laboratories, Detroit, Mich), 10% tissue culture yeastolate <br /> (Difco Laboratories), 100 U penicillin ml, 100 ,ug     pGEM2-702           1-1235       116 &#194;&#177; 62        1080 &#194;&#177; 674           9.3 <br /> streptomycin sulfate ml, 10% fetal bovine serum          pT7-7.2            28-1235       418 &#194;&#177; 121       2416 &#194;&#177; 277           5.8 <br /> (Flow Laboratories, McLean, Va.) 28&#194;&#176;C infected       pT7-7.1            68-1235       311 &#194;&#177; 109       2251 &#194;&#177; 185           7.2 <br /> with baculovirus input multiplicity 5 PFU cell.       UL30 RNA transcribed linearized plasmid translated or <br /> To prepare proteins immunoprecipitation, L-[35S]methio-      UL42 RNA rabbit reticulocyte lysates 60 min 30&#194;&#176;C. <br /> nine added 12 h postinfection final concentration      b Units pol activity defined femtomoles [3H]dTTP <br /> of 50 ,uCi/ml, high salt nuclear extracts prepared      incorporated activated calf thymus DNA 37&#194;&#176;C 30 min. Activities are <br /> essentially described Gallo et al. (18) cells         expressed means + standard deviations. pol assays performed as <br /> harvested 36 48 h postinfection, 10 strokes    triplicates independent experiments pGEM2-702 pT7-7.2 and <br />                                                                  experiments pT7-7.1. <br /> of Dounce homogenization used disrupt cyto-            c (Mean pol activity + UL42)/(mean pol activity - UL42). <br /> plasmic membranes Sf9 cells. <br />   Antibodies coimmunoprecipitation. MAb 6898, specific <br /> for UL42, described previously (41) kindly <br /> provided mouse ascites fluid Howard Marsden (Uni-          pared activity translation products pol RNA <br /> versity Glasgow, Glasgow, Scotland). The epitope recog-       (18). Although UL42 mutant RNA encoding a <br /> nized MAb 6898 shown immunodominant            product lacking 28 C-terminal residues resulted wild- <br /> and maps amino acids 363 369 (35). In addition,          type levels pol stimulation, failure large amino- <br /> polyclonal antibody residues region UL42         terminal (N-terminal) C-terminal deletion mutants of <br /> was raised rabbits. A 40-amino-acid peptide synthe-       UL42 stimulate pol activity study (18) suggested <br /> sized contained UL42 residues 360 377 linked     useful delineating the <br /> amino terminus promiscuous T-cell epitope mea-         domains UL42 required functional interaction with <br /> sles virus (47) 4-residue linker consisting Leu-Gly-    pol. Nevertheless, sensitivity screen <br /> Pro-Ser. The linked B- T-cell epitopes capable       large numbers UL42 mutations hindered low <br /> eliciting antibody response rabbits          variable activities translation products wild- <br /> coupled protein carrier, resulting antibody 834. A       type pol RNA. <br /> synthetic peptide corresponding residues 1216 1224         It suggested pol production regulated <br /> the HSV-1 pol coupled BSA (Sigma, St. Louis, Mo.)         HSV-infected cells posttranscriptionally atten- <br /> and injected rabbits produce pol-specific anti-      uation mechanism resulting secondary-structure forma- <br /> serum 771. Both peptides synthesized Peptide         tion N-terminal end pol RNA (54). Because sec- <br /> and Protein Engineering Laboratory Ohio State Univer-         ondary-structure formation affect ability of <br /> sity (Columbus). The specificity antipeptide serum       RNA translated vitro, tested N-terminal <br /> was confirmed immunoprecipitation presence         deletion mutants pol abilities stimulated by <br /> absence cognate peptide.                                  wild-type UL42 vitro transcription-translation <br />    The pol UL42 transcripts (0.5 3 ,ug RNA)       (Table 1). Transcripts mutant pol con- <br /> translated 70 ,u rabbit reticulocyte lysate described   structs previously reported translated with <br /> above. After removal 7-pI aliquot, 3 volumes B2          greater efficiency wild-type, translation <br /> buffer (10 mM Tris-HCl [pH 8.0], 10% glycerol) added.       products possessed pol activity (15). The results the <br /> To solution, equal volume 2x Zweig&apos;s buffer (0.2      present study demonstrate wild-type pol construct <br /> M Tris-HCl [pH 8.0], 20% glycerol, 1.0% Nonidet P-40,            previously used, pGEM2 702, encodes the <br /> 1.0% sodium deoxycholate, 0.4 mM phenylmethylsulfonyl            full-length (1,235-amino-acid) protein, yielded barely <br /> fluoride) added. Samples precleared incubation       detectable pol activity alone, particular <br /> at room temperature 0.1 volume 50% (wt/vol)          assay activity stimulated 9.3-fold RNA <br /> slurry protein A-Sepharose CL-4B (Sigma) 1 x Zweig&apos;s       cotranslated wild-type ULA2 RNA (Table 1). Both <br /> buffer containing 0.02% sodium azide. Supernatants          mutant pol transcripts encode proteins the <br /> split 1.5-ml tubes gently mixed 20 25 pI    27 (pT7-7.2) 67. (pT7-7.1) amino acids deleted <br /> of antiserum overnight 4&#194;&#176;C. Immune complexes             yielded higher pol activities wild-type trans- <br /> recovered addition 0.2 volume protein          lated alone, confirming earlier report (15). Furthermore, <br /> A-Sepharose 1 h 4&#194;&#176;C. The Sepharose beads             pol activities pT7-7.1 pT7-7.2 transcripts were <br /> washed times buffer containing 0.1 M Tris-HCl (pH      stimulated 7.2- 5.8-fold, respectively, cotranslation <br /> 8.0), 0.5 M NaCl, 1.0% 2-mercaptoethanol. Bound              UL42 transcripts (Table 1). We observed less <br /> protein eluted boiled 3 min 25 RI         variability experiment experiment pol activities <br /> dissociation buffer (0.125 M Tris-HCl [pH 6.8], 2% SDS, 2%       produced transcripts mutated pol genes in <br /> glycerol, 5% 2-mercaptoethanol, 0.004% bromophenol blue)         produced transcripts wild-type gene. On the <br /> and analyzed SDS-PAGE.                                    basis somewhat higher stimulation indices observed <br />                                                                  pT7-7.1 construct, selected func- <br />                           RESULTS                                tional domain mapping studies. <br />                                                                     C-terminal deletions UL42. We determined how <br />   Analysis  pol constructs. In previous report, ob-      C-terminal region UL42 protein was <br /> served four- sevenfold stimulation pol activity     required ability stimulate pol activity. The C-ter- <br /> HSV-1 pol RNA transcribed vitro cotranslated           minal mutant forms UIA2 protein (Fig. 1) were <br /> rabbit reticulocyte lysates wild-type UL42 RNA com-         generated ways: (i) cleavage restriction <br />  VOL. 67, 1993                                                                         FUNCTIONAL INTERACTION OF HSV-1 PROTEINS                    5925 <br />  <br />  <br />                                                                                   V      0.0        E          -    - <br />                                                                                Z       IZZ         C/)        z x <br />                                  pLBN19A         F <br />  <br />  <br />                                                UL42 ORF                                                            ) (488AA) <br />                                                       wt <br />                                                       N459 <br />                                                       N339 <br />                                                       N248 <br />                                                       N206 <br />                                                       N139 <br />   FIG. 1. Generation C-terminal deletion mutants ULA2. Schematic diagram ULA2 gene cloned phagemid vector pTZ <br /> 19U downstream T7 RNA polymerase promoter. Transcripts encoding C-terminal deletions produced (i) linearizing the <br /> open reading frame (ORF) restriction sites indicated produce runoff transcripts (ii) inserting linkers sites introduce <br /> frameshift mutations amino acid (AA) indicated. wt, wild type. <br />  <br />  <br /> enzyme indicated open reading frame                                 translation mixtures containing pol RNA consis- <br /> wild-type gene, followed production runoff transcripts;                         tent competition UL42 RNAs limited translation <br /> (ii) deletion sequences encoding C-terminal end;                         machinery observed previously (18). In fact, we <br /> (iii) insertion linkers various sizes frame                    observed reduction approximately twofold in <br /> indicated sites. Because method produces C-termi-                            pol polypeptide synthesized presence of <br /> nal tails nonrelated sequence, linkers inserted                            wild-type mutant UL42 RNA, compared that <br /> create different out-of-frame mutations. The activities                        observed absence competing RNA (results not <br /> the products pT7-7.1 transcripts translated (pol                         shown). A C-terminal mutant protein generated trunca- <br /> control), wild-type UL42 transcripts (normalized                          tion UL42 gene SmaI site (N339) produced <br /> 100%), ULA2 mutant transcripts                                wild-type levels pol stimulation, did N459 muta- <br /> indicated Table 2. The results indicate UL42 proteins                         tions (Table 2). These results demonstrate C-termi- <br /> containing 139 (140 generated frameshift), 206,                      nal 149 amino acids UL42 dispensable protein&apos;s <br /> or 248 N-terminal residues, regardless method                               ability stimulate pol activity. <br /> which generated, failed stimulate pol activity                              The failure mutant UL42 proteins stimulate pol <br /> above observed absence mutant UIA2 protein.                            activity instability mutant <br /> In fact, pol activities approximately twofold                        proteins. Therefore, relative stability mutant <br /> presence mutant proteins absence                          protein measured expression rabbit reticulocyte <br /> UL42 transcript. The reduction pol activity 50%                         lysates. The mutant proteins incubated pol <br />                                                                                       reaction conditions (+) compared proteins in <br />                                                                                       reactions held 0&#194;&#176;C absence pol reaction buffer <br />     TABLE 2. pol stimulation C-terminal deletion mutants                           (-). Figure 2 shows profile polypeptides some <br />                           UL42                                                     representative samples analyzed SDS-PAGE. The <br />                                            pol stimulation (%)                        mutant polypeptide species indicated small open <br /> Construct                                                                             circles. The results indicate mutations <br />                            Truncation&apos;                  Frameshift mutationc          generated linearizing DNA template UL42 <br /> Wild type                     lood                               lood                 open reading frame relatively unstable pol reac- <br /> N459                          110                                 81                  tion conditions (compare lane 4 lane 5 lane 10 with <br /> N339                          100                                NDe                  lane 11). The frameshift mutations generating same <br /> N248                             11                                8                  UL42 N-terminal sequences, however, stable (com- <br /> N206                             13                                7                  pare lanes 6 7 lanes 12 13). These results suggest <br /> N139                              7                               11                  UL42 proteins possessing C-terminal truncations of <br />   a <br />     pol stimulation calculated ratio pol activity measured       greater 149 residues unstable stabilized <br /> vitro cotranslation products pol mutant ULA2 RNAs measured             presence polypeptide nonrelated sequence <br /> for cotranslation pol wild-type UL42 RNAs. pol (UL30) RNA                  C terminus. The stable polypeptides produced by <br /> transcribed linearized pT7-7.1 and, translated alone, yielded an <br /> average activity 239 units.                                                        N140(fs), N206(fs), N248(fs) frameshift mutations <br />   b Mutant UIA2 RNAs generated linearizing UIA2 open               yielded stimulation pol activity (Table 2), we <br /> reading frame.                                                                        conclude sequences insufficient functional <br />   c Mutant UL42 RNAs generated frameshift mutations linearized              interaction pol. <br /> downstream ULA2 gene. <br />   d (Activity pol mutant ULA2 translation products/activity pol            The multiple polypeptide species produced wild- <br /> wild-type UL42) x 100.                                                                type mutant transcripts observed previously <br />   &apos; ND, done.                                                                     (13, 15, 18) likely initiation translation <br />  5926      MONAHAN ET AL.                                                                                                              J. VIROL. <br />  <br />                                                                               cleaved DNA N140(fs) mutant NotI site <br />                                                                               (wild-type residue 248) instead XbaI site (Fig. 2B, <br />                                                                               lane 2). Although cleavage NotI site predicted <br />                                                                               affect length product resulting N140(fs), it <br />            *(6 <br />            6A    1slf                       1                - -- <br />                                                                               shortened product produced internal <br />                                                                               initiation methionine codon. Although size of <br />                                                                               predicted frameshifted product did change, was <br />                                                                               loss larger polypeptide species, predicted arise <br />                                                                         Ow.   internal initiation (Fig. 2B, compare lanes 1 2). <br />                                                                               Therefore, likely larger polypeptide species <br />                                                                               observed N140(fs) mutant results internal <br />                                                                               initiation translation residue 158, downstream the <br />                                                                               <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>frameshift mutation</span>. Despite presence substantial <br />                         21       3   4           (6   7      ()               levels wild-type C-terminal peptide translation prod- <br />                                                                               ucts N140(fs) mutant RNA generated template <br />                                      NI 4()Ofi    NlI3)(1)                    cleaved XbaI, observed stimulation pol <br />                     I                                                         activity. Thus, polypeptide corresponding residues <br />                                                                               158 488 UL42 insufficient functional interac- <br />                                                                               tion pol. <br />                                                                                  Internal insertion deletion mutations. As means of <br />                                                                               defining critical domains UL42 required its <br />                                                                   ON,         functional interaction pol, created in-frame <br />                                                                               insertion deletion mutations UL42 tested for <br />                                                                               stability (data shown) abilities stimulate pol <br />                                                                               activity described (Fig. 3). Some deletion <br />                                                                               mutants contain insertions novel amino acids, indi- <br />                                                                               cated, result mutation itself result the <br />                              c                                                insertion additional sequences place out-of-frame <br />                                                                               deletion mutation frame. All mutant forms <br />                                                                               indicated stable pol reaction conditions (results <br />    FIG. 2. (A) The stability C-terminal deletion mutants            shown). We confirmed d37-282 deletion mutant, <br /> UL42. Transcripts encoding representative deletion mutants               used previous study wild-type pol construct <br /> Table 1 translated rabbit reticulocyte lysates presence        (18), failed stimulate activity pT7-7.1 pol <br /> of [35S]methionine frozen immediately (-) incubated             construct (Fig. 3). Several smaller deletion mutants <br /> under polymerase assay conditions (+) prior SDS-PAGE 10 to <br /> 20% gradient polyacrylamide gels. Mutants designated               region UL42 failed stimulate pol <br /> last UL42 amino acid truncation (t), frameshift (fs),               activity, including nonoverlapping mutations, d129-163 <br /> in-frame <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>insertion mutation</span> (i). The position full-length             d202-337. Insertion novel amino acids after <br /> product denoted circle. (B) Internal initiation products           residue 206 (i206) little effect ability protein <br /> UL42 expressed downstream frameshift mutations. Tran-                    stimulate pol activity compared wild-type UL42 <br /> scripts produced pLBN 19A linearized downstream                   protein. However, insertion residues amino <br /> UL42 gene (lane 5) internal NdeI site (lane 3)                 acid 140 (i140) completely abolished pol-stimulating <br /> translated described panel A. Transcripts produced           activity UL42. Because sequence inserted i140 <br /> template encoding frameshift residue 140 linearized          encodes proline residues, pro- <br /> XbaI (X) downstream UL42 gene (lane 1) NotI (N) <br /> site gene downstream frameshift mutation (lane          effects secondary tertiary structures some <br /> 2); transcripts translated analyzed SDS-PAGE                   distance site insertion, created subtle <br /> described panel A. The product denoted produced                mutation site deleting residue 140. Interest- <br /> initiation AUG UL42 gene. wt, wild type.                  ingly, d140 stimulated pol activity, consistently at <br />                                                                               half level observed wild-type UL42. Taken <br />                                                                               together, results indicate separable <br />                                                                               domains required UL42 stimulate pol activity. One <br /> from internal AUG codons. Indeed, apparent molecular                    these, referred region I, maps between <br /> masses consistent hypothesis. We observed                    amino acids 129 163 sensitive pertubations <br /> unexpected species apparent molecular mass 44                      produced insertions deletions region. The <br /> kDa (small arrow) translation products N140(fs)                   domain, denoted region II, maps residues 202 <br /> mutant transcriptional template linearized                   337. <br /> at XbaI site polylinker downstream UL42                        Coimmunoprecipitation pol UL42 proteins. Because <br /> gene (Fig. 2B, lane 1). This species size apparently                    ability UL42 stimulate pol activity result <br /> larger predicted mutant protein,                     multiple attributes functions UL42 protein, we <br /> comigrated product predicted initiated                  investigated second property UL42: ability phys- <br /> from AUG codon wild-type protein. Since                  ically associate pol. Others previously used a <br /> third methionine codon (amino acid 158) located down-                      coimmunoprecipitation assay pol antiserum anti- <br /> stream frameshift mutation N140(fs), internal trans-                serum UL42 determine domains pol required for <br /> lation initiation site yield protein                     physical association UL42 (12, 13, 49). It likely <br /> wild-type sequence amino acids 158 488, lacking                   interfering pol-UL42 complex formation would <br /> the N terminus. To determine product                     abolish UL42&apos;s pol-accessory functions. To assess whether <br /> to initiation translation AUG codon,                     region I II deletions abolished ability UL42 to <br />  VOL. 67, 1993                                                             FUNCTIONAL INTERACITION OF HSV-1 PROTEINS                      5927 <br />  <br />                                               UL42 Mutations                        Pol Activity Stimulation <br />                                                                                          (%/) <br />  <br />                         wt                                                                100                   + <br />  <br />                         i206                           A                                   85 <br />                                                      RGSA <br />                         i140                    A <br />                                                                                            16 <br />                                               PDPV <br />                         d140                     n                                         50                 +l- <br />                         dl 29-163               :                                          17 <br />                         d141-206                                                           14 <br />                                                        RI <br />                         d37-282                        Rl                                   11 <br />  <br />                         d202-337                       I            ,1                     15 <br />                         _    .   _                                LRDPA <br />                         Pol Control                                                        20 <br />  <br />  <br />                                                  I           II <br />    FIG. 3. Stimulation pol activity insertion deletion mutants UL42. Schematic diagram insertion mutants inserting <br /> in-frame linkers convenient restriction sites deletion mutants constructed described Materials Methods. The pol activities of <br /> in vitro translation products produced coexpression RNA encoding UL30 (from pT7-7.1) UL42 mutant reported as <br /> percentages activity produced pol wild-type (wt) UL42 transcripts. UL42 mutants stimulated pol activity greater than <br /> 50% level measured wild-type UL42 considered positive (+). Activities averages experiments. <br />  <br />  <br />  <br />  <br /> stimulate pol activity abolishing physical interaction                 region I region II UL42 mutants (lanes 14 and <br /> pol, utilized coimmunoprecipitation assay detect                       11). Likewise, pol antibody 771 immunoprecipitated its <br /> complex formation. pol UL42 transcripts cotrans-                      140-kDa full-length cognate protein pol- <br /> lated immunoprecipitated antibody 834, directed                    related species (lane 6). Although UL42 antibody pos- <br /> UL42 residues 360 377, antibody 771, directed                    sesses cross-reactivity pol, a <br /> pol residues 1216 1224 (Fig. 4). The specificity                     low level pol adheres nonspecifically protein <br /> UL42 antipeptide serum 834 demonstrated precip-                      A-Sepharose, absence antibody upon <br /> itation 63-kDa product translations wild-type                 extensive preclearing protein A-Sepharose (results not <br /> UL42 RNA (lane 2) 61- 49-kDa products                       shown). This nonspecific adherence accounts low <br />                                                                                level pol polypeptide immunoprecipitated the <br />                                                                                absence UL42 (Fig. 4, lane 5). When transcripts encoding <br />                U        P       P    U    d202-337 P   d129-163 P              pol region I mutant, d129-163, coexpressed and <br />            -  (-PI: <br />                uU   - U (Pi -        (   (P    U P        c IP                 immunoprecipitated antibody UL42, pol coim- <br />    Pol                                      Pol~~~~~~-((                       munoprecipitated (lane 14) levels comparable those <br />                                                                                observed wild-type UL42 (lane 8). Thus, region I does <br /> UL42 3-                                                                        appear required pol-UL42 complex formation. <br />                                                                                   By contrast, observed extremely low level of <br />                mK                                                ~~~~~~~~~~~~~~~- <br />                                                                    AUL42 <br />                                                                                pol polypeptide immunoprecipitated UL42 <br />        1    2 3     4  5 6    7 8 9       100    12   13 &apos;4 15 <br />    FIG. 4. Immunoprecipitation pol-UL42 complex formed               region II mutant protein antibody 834 (Fig. 4, lane 11). <br /> vitro. Translation products produced UL42 RNA (U), pol               Although level pol polypeptide precipitated in <br /> RNA (P), UL42 pol RNA expressed (U/P), UL42                       presence region II mutant similar the <br /> region I mutant pol RNA (d129-163/P), UIA2 region II                  nonspecific level observed, relatively low sensitivity of <br /> mutant pol RNA (d202-337/P). Translation products                    assay precludes definitive determination role of <br /> either electrophoresed immunoprecipitation (-)                 region II complex formation. Nevertheless, regions <br /> immunoprecipitated polyclonal antibody 834 UL42 (aU)                required functional interaction UL42 and <br /> antibody 771 pol (cl). Translation products treated                pol. <br /> antibody (lanes 1, 4, 7, 10, 13) represent one-fifth sample <br /> immunoprecipitated. The linotronix print derived                As seen Fig. 4, UL42 coimmuno- <br /> exposure gels run parallel obtained scanning             precipitated pol-UL42 cotranslation products the <br /> and manipulating Adobe Photoshop 2.01 software (Adobe                     pol antibody 771 directed residues 1216 1224. The <br /> Systems, Inc., Mountain View, Calif.).                                         ability antibodies specific region coprecipitate <br />  5928         MONAHAN ET AL.                                                                                                                                       J. VIROL. <br />  <br />                      EXTRACTS                         6898              771       6898 771:                complex contains 1:1 ratio pol UL42 (9, 24). <br />                  ! <br />  <br />  <br />                  Y <br />                       F <br />  <br />  <br />                      MA..I.:j   z,1 <br />                                             {L1   .                                   CL; r <br />                                                                                                            Both viral proteins essential viral DNA <br />        200   - <br />                                                                                                            replication vivo (6, 27-29, 31, 42, 53). Although pol has <br />         97   - <br />                                                                                                            inherent enzymatic activity (15, 18, 24, 25, 30), UL42 <br />        69                                                                                                  stimulates activity pol (18, 49), acting processivity <br />                      1ffi: <br />                      *  _                                 *                           &quot;        ~~~~-UL42   factor vitro (24, 26) increasing affinity by <br />                   - <br />                  ._                       _ ~~~~~~~~~~~~~~~P <br />        46    - <br />                                                                                                            pol binds DNA template. In heterologous <br />                                                                                                            expression systems, pol-UL42 complex forms is <br />        30 <br />                                                                                                            detectable immunoprecipitation antibodies pol <br />                                                                                                            UL42 (12, 13, 21, 49) (Fig. 4 5). We examined <br />                      2           3    4     5     6   7   8    9   10         12 13    14     15 <br />                                                                                                            physical functional interactions UL30 (pol) <br />   FIG. 5. Coimmunoprecipitation pol-UL42 complex ex-                                                UL42 in vitro expression system. By coexpressing <br /> pressed baculovirus recombinants. Nuclear extracts                                               pol mutant forms UL42 vitro rabbit reticulocyte <br /> prepared Sf9 cells metabolically labelled 12 48 h                                             lysates, identified distinct domains UL42 <br /> postinfection 50 ,uCi [35S]methionine ml, were <br /> mock infected (M) infected wild-type A. californica (W),                                         critical stimulation pol activity. <br /> pol-containing recombinant (P), UL42-containing recombinant                                                 To ensure mutant forms UL42 failed to <br /> (U), recombinants (U/P). Nuclear extracts electro-                                            stimulate pol lacked critical domains simply <br /> phoresed immunoprecipitation (lanes 1 5) immu-                                          degraded rapidly, UL42 mutants incubated under <br /> noprecipitated MAb 6898 UL42 (lanes 6 9 14)                                              polymerase reaction conditions (Fig. 2) assess stability, <br /> polyclonal antibody 771 pol (lanes 10 13 15). Nuclear                                            proteins fold properly suscep- <br /> extracts treated antibody (lanes 1 5) represent one-tenth                                      tible increased proteolysis (40). We the <br /> of sample immunoprecipitated.                                                                          exception C-terminal deletion mutants generated <br />                                                                                                            linearized template, mutant forms UL42 were <br />                                                                                                            stable (Fig. 2). However, deletion <br /> UL42 debate                                                 mutants created linker mutagenesis stable, the <br /> the exact C-terminal residues pol required                                                addition nonsense peptide C-terminal end these <br /> physical association UL42 (12, 49). Some possible                                                     deletion mutants sufficient stability. The fact that <br /> interpretations failure pol antibody copre-                                              deletion insertion mutations UL42 <br /> cipitate UL42 (i) antibodies pol sterically                                                resulted unstable polypeptides suggests muta- <br /> interfered ability pol UL42 associate                                               tions did result large perturbations global folding of <br /> (ii) epitope recognized 771 masked pol                                                protein. <br /> and UL42 physically associated, pol free.                                                 We domains UL42 critical its <br /> Since coimmunoprecipitate small amounts                                                   ability stimulate pol activity located amino <br /> pol UL42 vitro translation products                                                   two-thirds protein. The C-terminal 149 amino acids of <br /> UL42 antibody highly stringent wash conditions (500                                                  UL42 deleted affecting stability its <br /> mM NaCl), appeared small percentage UL42                                                 ability stimulate pol activity (Table 2). By coexpressing in <br /> and pol formed stable complexes rabbit reticulocyte                                                     vitro transcription-translation reactions internal insertion <br /> lysates. Furthermore, pol-specific antibody 771                                                deletion mutants UL42 pol (Fig. 3), found <br /> is avid 834 UL42-specific antibody (unpub-                                                   physically separable regions critical pol <br /> lished results), failure detect coprecipitation UL42                                             stimulation. Region I located amino acids 129 and <br /> with pol antiserum reflected lack                                                      163, region II located amino acids 202 and <br /> sensitivity assay system. Therefore, tested                                                 337. Because relatively large number amino acid <br /> ability pol antibody 771 coprecipitate                                                   residues encompassing region II, exclude the <br /> proteins coexpressed high levels insect cells coinfected                                             possibility contains functional domain. <br /> with pol UL42 baculovirus recombinants (Fig. 5). In                                               The information contained amino-terminal 157 resi- <br /> experiment, MAb 6898, recognizes essentially                                                     dues essential UL42 function, indicated the <br /> same region 834, employed UL42 antibody.                                                     failure initiation products AUG codon to <br /> The specificity antibody confirmed ability                                              stimulate pol activity (Fig. 2; Table 2). <br /> of precipitate protein cells singly infected                                                Small alterations introduced region I of <br /> by recombinant expressing cognate protein despite                                                  UL42 define residues critical the <br /> the presence large amounts labelled proteins                                                functional interaction pol. Deletion residue 140 <br /> the extracts. However, large amounts pol coimmu-                                                   (isoleucine) decreased pol stimulation twofold. Fur- <br /> noprecipitated MAb 6898 (ca-UL42) extracts                                                    thermore, insertion mutation amino acid 140 failed <br /> cells coinfected UL42 pol recombi-                                                   stimulate pol. Thus, region close amino acid <br /> nants. It apparent small UL42                                               140 particularly sensitive perturbations. We cur- <br /> be coimmunoprecipitated pol using 771 serum                                                    rently introducing small deletions amino acid <br /> (a-pol). These results demonstrate residues recog-                                                140 point mutations region define <br /> nized pol antiserum (1216 1224) accessible                                             residues critical functional interaction with <br /> least subpopulation complexes pol UL42                                                     pol. <br /> suggest essential physical                                                       To determine region I II involved the <br /> interaction pol UL42.                                                                              physical association pol ULA2, mutant wild-type <br />                                                                                                            UL42 cotranslated pol vitro immunopre- <br />                                                 DISCUSSION                                                 cipitated antibody specific ULA2 (Fig. 4). pol was <br />                                                                                                            coimmunoprecipitated wild-type UL42 region I <br />   Analogously prokaryotic eukaryotic replica-                                                  mutant, d129-163, indicating region I required for <br /> tion systems, HSV-1 DNA polymerase multiprotein                                                   physical association pol. Although pol detected <br />  VOL. 67, 1993                                                      FUNCTIONAL INTERACTION OF HSV-1 PROTEINS                       5929 <br />  <br /> above background levels coexpressed region              Although region I essential physical interactions <br /> II mutant, nonspecific adherence pol poor            pol, required stimulation pol activity vitro. <br /> sensitivity assay make difficult assign        Disruption functional domains affecting protein- <br /> region II required complex formation. Indeed, low        protein interactions reported component of <br /> level coimmunoprecipitation pol UL42 observed          HSV-1 helicase-primase complex. Single amino acid <br /> cotranslations wild-type UL42 pol           substitutions introduced conserved motifs UL5 dis- <br /> specific vitro translation used. It      rupted function viral replication did affect <br /> noted wash conditions utilized immu-           UL5-UL8-UL52 complex formation (56). It possible that <br /> noprecipitations stringent (containing 500 mM             region I mediates binding UL42 and/or the <br /> NaCl) used investigators (12, 13, 21, 49).      pol-UL42 complex DNA template. Chou-Fasman (7) <br /> However, stringent conditions, ob-              Robson-Garnier (20) algorithms predict residues 130 <br /> served quantitative coimmunoprecipitation            139, 140 144, 145 162 form helix-loop-helix <br /> pol UL42 antibody UL42 extracts cells             motif. A similar motif directly involved DNA <br /> which coexpressed pol UL42 recombinant baculo-             binding prokaryotic regulatory proteins eukary- <br /> viruses (Fig. 5). The differences observed ability    otic transcription factors (39). It possible region <br /> two proteins coimmunoprecipitated different          I required maintain conformational integrity of <br /> expression systems differences             UL42, perturbations result the <br /> antibodies used, observed low-level coprecip-         mutations tested sufficient alter global folding <br /> itation pol UL42 vitro translations MAb        lead increased proteolysis. Because the <br /> 6898 (results shown). It possible differences       ability UL42 bind double-stranded DNA no <br /> observed reflect poor ability proteins expressed        apparent sequence specificity (19, 19a, 33), neces- <br /> the rabbit reticulocyte lysates assume conformation            sary purify mutant UL42 proteins determine <br /> necessary proteins form stable physical            abilities bind DNA altered. <br /> association. It possible Kd pol-UL42         We shown ULA2 multiple, separable do- <br /> heterodimer low level protein expressed      mains required pol-accessory functions vitro. How- <br /> rabbit reticulocyte lysates prevents formation signifi-      ever, necessary define critical residues <br /> cant amounts stable pol-UL42. Nevertheless, associ-          region I region II ULA2. Because many <br /> ation proteins translated vitro sufficient      synthetic peptides corresponding linear arrays residues <br /> enable UL42 stimulate pol activity.                          ULA2 inhibit pol activity abilities <br />    Another possible explanation poor ability      interact DNA abilities inhibit the <br /> pol UL42 coimmunoprecipitated vitro               interaction pol UL42 (38), care exer- <br /> translation products presence truncated poly-         cised defining peptide domains interfere with <br /> peptides produced rabbit reticulocyte lysates result     ability pol UL42 functionally physically <br /> initiation translation internal AUG codons. Such            interact. Thus, important gain complete <br /> products present high levels nuclear extracts      understanding function residues overall <br /> cells infected recombinant baculoviruses. It            process viral DNA replication provide basis from <br /> possible polypeptides sufficient      design peptidomimetic antiviral drugs. <br /> stimulate pol activity, compete extent <br /> with wild-type UL42 protein physical association                                  ACKNOWLEDGMENTS <br /> with pol. Although able observe               We thank Mark Challberg Bob Lehman recombinant <br /> such dominant negative effects competing polypeptides         baculovirus constructs, Howard Marsden Ann Cross MAb <br /> the ability wild-type UL42 stimulate pol activity         6898, Fred Hink advice propagation baculoviruses and <br /> (unpublished results), exclude possibility       insect cells, Pravin Kaumaya advice help designing <br /> these polypeptides destabilize interaction          immunogenic peptides. <br /> wild-type UL42 pol.                                               This work supported grant GM34930 the <br />    Our ability detect coimmunoprecipitation UL42         National Institutes Health (D.S.P.) OSU Comprehensive <br /> pol antiserum residues 1216 1224 nuclear extracts        Cancer Center core grant CA16958 National Cancer <br /> of cells coexpress proteins recombinant              Institute. S.J.M. supported National Research <br /> baculoviruses, vitro translation products,          Service award T32 CA09498-06 National Cancer Institute <br /> indicates caution exercised interpreting          fellowship F32 A108566-01 National Institutes of <br />                                                                     Health. T.F.B. supported National Institute Allergy and <br /> negative immunoprecipitation results experiments in-           Infectious Diseases training grant 5T32 AI07387-03 AIDS re- <br /> volving vitro translation products. Thus, failure       search. <br /> pol polypeptide lacking 19 C-terminal residues copre- <br /> cipitate UL42 translation reactions (12)                                    REFERENCES <br /> success mutant proteins immunoprecipitated             1. Bruckner, R. C., J. J. Crute, M. S. Dodson, I. R. Lehman. <br /> with UL42 expressed Saccharomyces cerevisiae                    1991. The herpes simplex virus 1 origin binding protein: DNA <br /> (49) accounted relatively unstable inter-             helicase. J. Biol. Chem. 266:2669-2674. <br /> action proteins present rabbit reticulocyte       2. Bush, M., D. R. Yager, M. Gao, K. Weisshart, A. I. Marcy, <br /> lysates, protein made, differences                  D. M. Coen, D. M. Knipe. 1991. Correct intranuclear <br /> posttranslational modifications expression systems.               localization herpes simplex virus DNA polymerase requires <br /> Therefore, observed significant levels             viral ICP8 DNA-binding protein. J. Virol. 65:1082-1089. <br />                                                                      3. Chadler, J. M., N. D. Stow. 1990. Herpes simplex virus <br /> of coimmunoprecipitation region II UL42 mutant                   helicase-primase: UL8 protein required DNA- <br /> protein d202-337 cotranslations pol (Fig. 4),           dependent ATPase DNA helicase activities. Nucleic Acids <br /> await construction baculovirus recombinant ex-                 Res. 18:3573-3578. <br /> pressing mutations region definitive coim-          4. Challberg, M. D. 1986. A method identifying viral genes <br /> munoprecipitation analysis.                                             required herpesvirus DNA replication. Proc. Natl. Acad. <br />  5930     MONAHAN ET AL.                                                                                                           J. VIROL. <br />  <br />     Sci. USA 83:9094-9098.                                              24. Gottlieb, J., A. I. Marcy, D. M. Coen, M. D. Challberg. <br />  5. Challberg, M. D., T. J. Kelly. 1989. Animal virus DNA               1990. The herpes simplex virus type 1 UL42 gene product: a <br />     replication. Annu. Rev. Biochem. 58:671-717.                            subunit DNA polymerase functions increase proces- <br />  6. Chartrand, P., N. D. Stow, M. C. Timbury, N. M. Wilkie.             sivity. J. Virol. 64:5976-5987. <br />     1979. Physical mapping paar mutations herpes simplex          25. Haffey, M. L., J. T. Stevens, B. J. Terry, D. I. Dorsky, C. S. <br />     virus type 1 type 2 intertypic marker rescue. J. Virol.          Crumpacker, S. M. Wietstock, W. T. Ruyechan, A. K. Field. <br />     31:265-276.                                                             1988. Expression herpes simplex virus type 1 DNA poly- <br />  7. Chou, P. Y., G. D. Fasman. 1978. Empirical predictions           merase Saccharomyces cerevisiae detection virus- <br />     protein conformation. Annu. Rev. Biochem. 47:251-276.                   specific enzyme activity cell-free lysates. J. Virol. 62:4493- <br />  8. Conley, A. J., D. M. Knipe, P. C. Jones, B. Roizman. 1981.          4498. <br />     Molecular genetics herpes simplex virus. VII. Characteriza-      26. Hernandez, T. R., I. R. Lehman. 1990. Functional interac- <br />     tion temperature-sensitive mutant produced vitro             tion herpes simplex-1 DNA polymerase UL42 <br />     mutagenesis defective DNA synthesis accumulation             protein. J. Biol. Chem. 265:11227-11232. <br />     y polypeptides. J. Virol. 37:191-206.                            27. Jofre, J. T., P. A. Schaffer, D. S. Parris. 1977. Genetics of <br />  9. Crute, J. J., I. R. Lehman. 1989. Herpes simplex-1 DNA              resistance phosphonoacetic acid strain KOS herpes <br />     polymerase. Identification intrinsic 5&apos;-3&apos; exonuclease       simplex virus type 1. J. Virol. 23:833-836. <br />     ribonuclease H activity. J. Biol. Chem. 264:19266-19270.            28. Johnson, P. A., M. G. Best, T. Friedmann, D. S. Parris. <br /> 10. Crute, J. J., T. Tsurumi, L. Zhu, S. K. Weller, P. D. Olivo,            1991. Isolation herpes simplex virus type 1 mutant deleted <br />     M. D. Challberg, E. S. Mocarski, I. R. Lehman. 1989.                essential UL42 gene characterization null <br />     Herpes simplex virus 1 helicase-primase: complex             <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span>. J. Virol. 65:700-710. <br />     herpes-encoded gene products. Proc. Natl. Acad. Sci. USA            29. Marchetti, M. M., C. A. Smith, P. A. Schaffer. 1988. A <br />     86:2186-2189.                                                           temperature-sensitive mutation herpes simplex virus type 1 <br /> 11. Bruyn Kops, A., D. M. Knipe. 1988. Formation DNA              gene required viral DNA synthesis maps coordinates <br />     replication structures herpes virus-infected cells requires        0.609 0.614 UL. J. Virol. 62:715-721. <br />     viral DNA binding protein. Cell 55:857-868.                         30. Marcy, A. I., P. D. Olivo, M. D. Challberg, D. M. Coen. <br /> 12. Digard, P., C. S. Chow, L. Pirrit, D. M. Coen. 1993.                1990. Enzymatic activities overexpressed herpes simplex <br />     Functional analysis herpes simplex virus UL42 protein. J.        virus DNA polymerase purified recombinant baculovirus- <br />     Virol. 67:1159-1168.                                                    infected insect cells. Nucleic Acids Res. 18:1207-1215. <br /> 13. Digard, P., D. M. Coen. 1990. A novel functional domain      31. Marcy, A. I., D. R. Yager, D. M. Coen. 1990. Isolation and <br />     alpha-like DNA polymerase. The binding site herpes            characterization herpes simplex virus mutants containing <br />     simplex virus polymerase viral UL42 protein. J. Biol.           engineered mutations DNA polymerase locus. J. Virol. <br />     Chem. 265:17393-17396.                                                  64:2208-2216. <br /> 14. Dodson, M. S., I. R. Lehman. 1991. Association DNA           32. Marians, K. J. 1992. Prokaryotic DNA replication. Annu. Rev. <br />     helicase primase activities subassembly herpes        Biochem. 61:673-719. <br />     simplex virus 1 helicase-primase composed UL5            33. Marsden, H. S., M. E. M. Campbell, L. Haarr, M. C. Frame, <br />     UL52 gene products. Proc. Natl. Acad. Sci. USA 88:1105-1109.            D. S. Parris, M. Murphy, R. G. Hope, M. T. Muller, C. M. <br /> 15. Dorsky, D. I., C. S. Crumpacker. 1988. Expression                Preston. 1987. The 65,000-Mr DNA-binding virion trans- <br />     herpes simplex virus type 1 DNA polymerase gene vitro             inducing proteins herpes simplex virus type 1. J. Virol. <br />     translation effects gene deletions activity. J. Virol.        61:2428-2437. <br />     62:3224-3232.                                                       34. McGeoch, D. J., M. A. Dalrymple, A. J. Davison, A. Dolan, <br /> 16. Elias, P., M. E. O&apos;Donnell, E. S. Mocarski, I. R. Lehman.           M. C. Frame, D. McNab, L. J. Perry, J. E. Scott, P. Taylor. <br />     1986. A DNA binding protein specific origin replication       1988. The complete DNA sequence long unique region in <br />     herpes simplex virus type 1. Proc. Natl. Acad. Sci. USA              genome herpes simplex virus type 1. J. Gen. Virol. <br />     83:6322-6326.                                                           69:1531-1574. <br /> 17. Fierer, D. S., M. D. Challberg. 1992. Purification          35. Murphy, M., P. Schenk, H. M. Lankinen, A. M. Cross, P. <br />     characterization UL9, herpes simplex virus type 1 origin-        Taylor, A. Owsianka, R. G. Hope, H. Ludwig, H. S. <br />     binding protein. J. Virol. 66:3986-3995.                                Marsden. 1989. Mapping epitopes 65K DNA-binding <br /> 18. Gallo, M. L., D. I. Dorsky, C. S. Crumpacker, D. S. Parris.         protein herpes simplex virus type 1. J. Gen. Virol. 70:2357- <br />     1989. The essential 65-kilodalton DNA-binding protein her-           2364. <br />     pes simplex virus stimulates virus-encoded DNA poly-            36. O&apos;Donnell, M. E., P. Elias, B. E. Funnell, I. R. Lehman. <br />     merase. J. Virol. 63:5023-5029.                                         1987. Interaction DNA polymerase single- <br /> 19. Gallo, M. L., D. H. Jackwood, M. Murphy, H. S. Marsden,             stranded DNA-binding protein (infected cell protein 8) herpes <br />     D. S. Parris. 1988. Purification herpes simplex virus type       simplex virus 1. J. Biol. Chem. 262:4260-4266. <br />     1 65-kilodalton DNA-binding protein: properties protein      37. Olivo, P. D., N. J. Nelson, M. D. Challberg. 1988. Herpes <br />     evidence association virus-encoded DNA              simplex virus DNA replication: UL9 gene encodes origin <br />     polymerase. J. Virol. 62:2874-2883.                                     binding protein. Proc. Natl. Acad. Sci. USA 85:5414-5418. <br /> 19a.Gallo, M. L., D. S. Parris. Unpublished observations.           38. Owsianka, A. M., G. Hart, M. Murphy, J. Gottlieb, R. Boehme, <br /> 20. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis         M. Challberg, H. S. Marsden. 1993. Inhibition herpes <br />     accuracy implications simple methods predicting          simplex virus type 1 DNA polymerase activity peptides from <br />     secondary structure globular proteins. J. Mol. Biol.             UL42 accessory protein largely nonspecific. J. Virol. <br />     120:97-120.                                                             67:258-264. <br /> 21. Gibbs, J. S., K. Weisshart, P. Digard, A. deBruyn Kops, D. M.       39. Pabo, C. O., R. T. Sauer. 1992. Transcription factors: <br />     Knipe, D. M. Coen. 1991. Polymerization activity              structural families principles DNA recognition. Annu. <br />     alpha-like DNA polymerase requires conserved 3&apos;-5&apos; exonu-             Rev. Biochem. 61:1053-1095. <br />     clease active site. Mol. Cell. Biol. 11:4786-4795.                  40. Pakula, A. A., R. T. Sauer. 1989. Genetic analysis protein <br /> 22. Goodrich, L. D., F. J. Rixon, D. S. Parris. 1989. Kinetics       stability function. Annu. Rev. Biochem. 23:289-310. <br />     expression gene encoding 65-kilodalton DNA-binding       41. Parris, D. S., A. Cross, L. Haarr, A. Orr, M. C. Frame, M. <br />     protein herpes simplex virus type 1. J. Virol. 63:137-147.           Murphy, D. J. McGeoch, H. S. Marsden. 1988. Identifica- <br /> 23. Goodrich, L. D., P. A. Schaffer, D. I. Dorsky, C. S. Crumpacker,        tion gene encoding 65-kilodalton DNA-binding protein <br />     D. S. Parris. 1990. Localization herpes simplex virus        herpes simplex virus type 1. J. Virol. 62:818-825. <br />     type 1 65-kilodalton DNA-binding protein DNA polymerase         42. Purifoy, D. J. M., R. B. Lewis, K. L. Powell. 1977. <br />     presence absence viral DNA synthesis. J. Virol.           Identification herpes sim <br /> </body></html>