<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />  <br />                            Functional links between Drosophila Nipped-B and cohesin in <br />                            somatic and meiotic cells <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Drosophila Nipped-B is an essential protein that has multiple functions. It facilitates expression of <br />                                 homeobox genes and is also required for sister chromatid cohesion. Nipped-B is conserved <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 yeast <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> man, and its orthologs also play roles in deoxyribonucleic acid repair and meiosis. <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>Mutation</span> <br />                                 of the human ortholog, Nipped-B-Like (NIPBL), causes Cornelia de Lange syndrome (CdLS), <br />                                 associated with multiple developmental defects. The Nipped-B protein family is required for the <br />                                 cohesin complex that mediates sister chromatid cohesion to bind to chromosomes. A key question, <br />                                 therefore, is whether the Nipped-B family regulates gene expression, meiosis, and development by <br />                                 controlling cohesin. To gain insights into Nipped-B&apos;s functions, we compared the effects of several <br />                                 Nipped-B mutations on gene expression, sister chromatid cohesion, and meiosis. We also examined <br />                                 association of Nipped-B and cohesin with somatic and meiotic chromosomes by immunostaining. <br />                                 Missense Nipped-B alleles affecting the same HEAT repeat motifs as CdLS-causing NIPBL <br />                                 mutations have intermediate effects on both gene expression and mitotic chromatid cohesion, linking <br />                                 these two functions and the role of NIPBL in human development. Nipped-B colocalizes extensively <br />                                 with cohesin on chromosomes in both somatic and meiotic cells and is present in soluble complexes <br />  <br />  <br />                            Correspondence to: Dale Dorsett. <br />                            F. Uhlmann <br />                            Gause, Webber, and Misulovin provided equal contributions. <br />                             Gause et al.                                                                                                  Page 2 <br />  <br />  <br />                                     with cohesin subunits in nuclear extracts. In meiosis, Nipped-B also colocalizes with the <br />                                     synaptonemal complex and contributes to maintenance of meiotic chromosome cores. These results <br />                                     support the idea that direct regulation of cohesin function underlies the diverse functions of Nipped- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     B and its orthologs. <br />  <br />  <br />                               Introduction <br />                                               Drosophila Nipped-B was discovered in a genetic screen for factors that facilitate long-range <br />                                               transcriptional activation of the cut and Ultrabithorax (Ubx) homeobox genes by enhancers <br />                                               positioned some 80 and 50 kbp away from the gene promoters (Rollins et al. 1999, 2004). <br />                                               Nipped-B is essential, and homozygous Nipped-B mutants die as larvae. Heterozygous Nipped- <br />                                               B <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>null mutants</span>, which show only a 25 to 30% reduction in Nipped-B expression, are viable but <br />                                               have reduced cut and Ubx expression during imaginal disk development. <br />  <br />                                               Nipped-B encodes a member of a highly conserved protein family, which includes Scc2 and <br />                                               Mis4 of S. cerevisiae and S. pombe, discovered in screens for factors that control sister <br />                                               chromatid cohesion (Furuya et al. 1998; Michaelis et al. 1997), C. cinereus Rad9 identified in <br />                                               a screen for factors required for deoxyribonucleic acid (DNA) repair and meiosis (Valentine <br />                                               et al. 1995), and human Nipped-B-Like (NIPBL/delangin), mutated in Cornelia de Lange <br />                                               syndrome (CdLS; Krantz et al. 2004; Tonkin et al. 2004). Nipped-B family proteins contain <br />                                               seven HEAT repeats implicated in protein&#226;&#8364;&#8220;protein interactions (Neuwald and Hirano 2000), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               and NIPBL missense mutations in all seven cause CdLS (Gillis et al. 2004; Miyake et al. <br />                                               2005; Deardorff and Krantz, personal communication). <br />  <br />                                               Homozygous Drosophila Nipped-B mutants display defects in sister chromatid cohesion before <br />                                               they die, and depletion of vertebrate Nipped-B homologs in vitro or in cultured cells cause <br />                                               cohesion defects, showing that all are functional orthologs of Scc2 and Mis4 (Gillespie and <br />                                               Hirano 2004; Rollins et al. 2004; Seitan et al. 2006; Takahashi et al. 2004; Watrin et al. <br />                                               2006). Scc2 interacts with the Scc4 protein, which is also required for sister chromatid cohesion <br />                                               (Ciosk et al. 2000). Weakly conserved Scc4 homologs in S. pombe, C. elegans, Drosophila, <br />                                               and vertebrates interact with Nipped-B and its orthologs and function in sister chromatid <br />                                               cohesion and development (Bernard et al. 2006; Seitan et al. 2006; Watrin et al. 2006). <br />  <br />                                               The Nipped-B family is required for the cohesin protein complex that mediates sister chromatid <br />                                               cohesion to bind to chromosomes, which explains their role in cohesion (Arumugam et al. <br />                                               2003; Ciosk et al. 2000; Gillespie and Hirano 2004; Seitan et al. 2006; Takahashi et al. 2004; <br />                                               Tomonaga et al. 2000; Watrin et al. 2006). Cohesin consists of a heterodimer of the Smc1 and <br />                                               Smc3 structural maintenance of chromosome proteins, and two other proteins, the &#206;&#177;-kleisin <br />                                               Rad21 (Mcd1/Scc1) and Stromalin (SA/Scc3; reviewed in Hirano 2006; Nasmyth and Haering <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               2005). The Rad21 termini interact with the head domains of Smc1 and Smc3, forming a ring- <br />                                               like structure. Many organisms contain meiosis-specific forms of some cohesin subunits, <br />                                               including Rec8 (Rad21 homolog) in yeast, C. elegans, and mammals and Smc1&#206;&#178; and Stag3 <br />                                               (SA homolog) in mammals (reviewed in Revenkova and Jessberger 2006). The situation <br />                                               regarding meiosis-specific cohesin subunits is less clear in Drosophila, which lacks an obvious <br />                                               Rec8 homolog. <br />  <br />                                               Cohesin is required but not sufficient for sister chromatid cohesion. Its ring-like structure is <br />                                               the basis of the current cohesion models, in which cohesin encircles two sister chromatids or <br />                                               cohesin rings bound to two sisters interlock with each other (Hirano 2006; Huang et al. 2005; <br />                                               Losada 2007; Nasmyth and Haering 2005). In some cases, a cohesin ring that encircles one <br />                                               sister may interact with proteins bound to the other sister to establish cohesion (Chang et al. <br />                                               2005). During interphase, cohesin binds numerous sites along chromosome arms and more <br />                                               densely to pericentric heterochromatin, where it is critical for cohesion at metaphase. <br />  <br />                                                   Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                            Page 3 <br />  <br />  <br />                                           Like the Nipped-B family, cohesin also has roles besides sister chromatid cohesion in meiosis, <br />                                           DNA repair, gene expression, and human development (reviewed in Dorsett 2007; Nasmyth <br />                                           and Haering 2005; Hagstrom and Meyer 2003; Hirano 2006; Revenkova and Jessberger <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           2006). Thus, a central question is whether regulation of cohesin also explains the functions of <br />                                           the Nipped-B family proteins in gene expression and meiosis. In this paper, we determine the <br />                                           effects of several Nipped-B mutations on gene expression, sister chromatid cohesion, and <br />                                           meiosis and the localization of Nipped-B and cohesin on somatic and meiotic chromosomes. <br />                                           Combined, the findings link Nipped-B&apos;s diverse roles to the regulation of cohesin activity. <br />  <br />                               Materials and methods <br />                               Sequencing Nipped-B alleles <br />                                           Total ribonucleic acid (RNA) was isolated from wild-type and homozygous Nipped-B mutant <br />                                           second instar larvae using Trizol (Invitrogen) and reverse transcribed using SuperScript III <br />                                           (Invitrogen) and random hexamer primers. Overlapping segments of Nipped-B complementary <br />                                           DNAs (cDNAs) approximately 800 bp in <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> were amplified by polymerase chain reaction <br />                                           (PCR) and sequenced directly using the amplification primers (Retrogen). Sequence assembly <br />                                           and <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation analysis</span> was performed using CodonCode Aligner software (CodonCode). For <br />                                           the N-terminal region, which showed significant alternative splicing, the PCR products were <br />                                           cloned into plasmid vectors, and several were sequenced. Primer sequences are available upon <br />                                           request. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Nipped-B, Pds5, and Rad21 antibodies <br />                                           A His6-Nipped-B protein fusion containing Nipped-B residues 1 to 409 (GenBank AF114160) <br />                                           was expressed in E. coli using the pMCSG-7 vector (Stols et al. 2002) and purified under <br />                                           denaturing conditions. Insoluble purified protein was washed and suspended in phosphate- <br />                                           buffered saline (PBS) and used to immunize a guinea pig and a rabbit (Pocono Rabbit Farm <br />                                           and Laboratory, Canadensis, PA). A His6-Pds5 protein containing Pds5 residues encoded by <br />                                           exons 6, 7, and 8 of the Pds5 (CG17509) messenger RNA (mRNA) was prepared in the same <br />                                           manner and used to immunize a rabbit, and a His6-Rad21 fusion containing Rad21 residues 1&#226;&#8364;&#8220; <br />                                           350 was used to immunize both a rabbit and a guinea pig. <br />  <br />                                           Antibody specificities were confirmed by immunostaining and Western blots. In Western blots <br />                                           of cultured cell extracts, both Rad21 antisera recognized the same protein slightly larger than <br />                                           the predicted size for Rad21 as previously reported (Vass et al. 2003). The Rad21 protein was <br />                                           coprecipitated by SA, Smc1, and Nipped-B antisera (see &#226;&#8364;&#339;Results&#226;&#8364;&#65533;) and was reduced in Rad21 <br />                                           <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi)-treated cells. In some extracts, two major Rad21 bands were <br />                                           observed. All preimmune sera showed only low-level background immunostaining, and the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Nipped-B, Pds5, and Rad21 antisera show staining that colocalizes with SA and Smc1 on <br />                                           polytene chromosomes (see &#226;&#8364;&#339;Results&#226;&#8364;&#65533;). Homozygous pds5e3 mutants live to the third instar, <br />                                           and the maternal mRNA is virtually absent by the second instar stage (Dorsett et al. 2005). The <br />                                           salivary gland chromosomes from homozygous pds5e3 null mutants show staining for cohesin <br />                                           (Dorsett et al. 2005), but as expected, they show only background staining with the Pds5 <br />                                           antisera. By Western blots of whole-cell and nuclear extracts of cultured cells, whole-cell <br />                                           extracts of imaginal disks, adult ovaries, and embryos, the guinea pig Nipped-B antibody <br />                                           recognizes one major band close to the expected size (237 kDa) that is reduced in levels in <br />                                           embryos from heterozygous mutant mothers and cultured cells treated with anti-Nipped-B <br />                                           RNAi (Supplementary Fig. 1). The rabbit Nipped-B antisera precipitates all Nipped-B from <br />                                           nuclear extracts, as shown by guinea pig Nipped-B antibody Western blots of the precipitates <br />                                           and loss of the Nipped-B protein in the postimmunoprecipitation supernatant (Supplementary <br />                                           Fig. 1). <br />  <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                             Page 4 <br />  <br />  <br />                               Western blots <br />                                           Samples were separated by sodium dodecyl sulfate&#226;&#8364;&#8220;polyacrylamide gel electrophoresis (SDS- <br />                                           PAGE) using 6% gels and electroblotted to Immobilon membrane (Millipore). Nipped-B was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           detected using a 1:2,000 dilution of the guinea pig Nipped-B antiserum, followed by 1:7,500 <br />                                           dilution of donkey horse radish peroxidase-anti-guinea pig secondary antibody (Jackson <br />                                           Laboratories). Similar primary antibody dilutions of 1:2,000 to 1:5,000 were used for SA, <br />                                           Smc1, Rad21, and Pds5. The SA and Smc1 antisera have been described previously (Dorsett <br />                                           et al. 2005). Proteins were detected by chemiluminescence (Western Lightening, Perkin Elmer) <br />                                           and autoradiography. For embryo Western blots, 100 0- to 7-h-old embryos were crushed in <br />                                           20 &#206;&#188;l of PBS (pH 7.0) containing 0.2% Nonidet P40 (NP-40), 0.5 mM dithiothreitol (DTT), <br />                                           and protease inhibitors (1 mM benzamidine, 1 &#206;&#188;g per ml pepstatin A, 1 &#206;&#188;g per ml antipain, 0.5 <br />                                           mM phenymethylsulfonyl fluoride [PMSF]). Seven microliters of 4&#195;&#8212; SDS-PAGE loading dye <br />                                           was added to each sample, and the mixture was heated at 100&#194;&#176;C for 10 min. Extracts of ovaries <br />                                           and imaginal disks were made using the same buffer. Whole-cell extracts of cultured cells were <br />                                           made using radioimmunoprecipitation assay (RIPA) buffer, and nuclear extracts were made <br />                                           as described below. <br />  <br />                               Effects of Nipped-B mutations on ctK and su(Hw)e2 bx34e mutant phenotypes <br />                                           Males with wild-type or balanced mutant alleles of Nipped-B were crossed to ctK females at <br />                                           25 and 27&#194;&#176;C, and the wing margin nicks in the male progeny heterozygous for the Nipped-B <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           allele being tested were quantified as previously described (Gause et al. 2001; Rollins et al. <br />                                           2004). Nipped-B*/+; su(Hw)e2 bx34e flies were generated by crosses, and the bithorax <br />                                           phenotypes were quantified as previously described (Morcillo et al. 1996). Bonferroni&#226;&#8364;&#8220;Dunn <br />                                           post-hoc analysis of variance tests were performed on <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> data using Statview software <br />                                           (SAS Institute). <br />  <br />                               Neuroblast squashes <br />                                           Precocious sister chromatid separation was scored in second instar larval neuroblasts from <br />                                           Oregon R wild-type and various Nipped-B mutants heteroallelic with Nipped-B02047 using <br />                                           colchicine and hypotonic treatments as previously described (Rollins et al. 2004). Data from <br />                                           the mutant groups were compared to each other and Oregon R wild-type using Fisher&apos;s exact <br />                                           test (Fisher 1922; http://www.matforsk.no/ola/fisher.htm). <br />  <br />                               Immunostaining <br />                                           Salivary gland polytene chromosomes were prepared and immunostained as previously <br />                                           described (Dorsett et al. 2005) using a 1:100 or 1:200 dilution of primary antiserum and a 1:200 <br />                                           dilution of secondary antibodies that do not show cross-species reactivity (Jackson <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Laboratories). The SA and Smc1 antisera used for staining polytene chromosomes were <br />                                           described previously (Dorsett et al. 2005). <br />  <br />                                           Intact ovaries whole-mount preparations were fixed and stained as described previously <br />                                           (Webber et al. 2004). Z-stacks were collected and deconvolved using the Improvision Volocity <br />                                           software. Meiotic chromosome spreads were prepared and immunostained as described <br />                                           previously (Khetani and Bickel 2007). For all experiments with meiotic cells, a mixture of <br />                                           guinea pig affinity-purified antibodies raised against Smc1 and Smc3 peptides (Khetani and <br />                                           Bickel 2007) was utilized to visualize cohesin. <br />  <br />                               Meiotic chromosome segregation <br />                                           Nipped-B407/CyO or Nipped-B10E/+ females were crossed to attached X&#203;&#8222;Y, v f B males and <br />                                           scored for Bar+ female and Bar male exceptional progeny. Transmission of the J21A <br />                                           minichromosome was measured as previously described (Murphy and Karpen 1995; Dobie et <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                              Page 5 <br />  <br />  <br />                                           al. 2001). X&#203;&#8222;Y; ry506 males were crossed to Nipped-B*/+; ry506; J21A, ry+ females, and the <br />                                           transmission frequency was calculated by dividing the number of ry+ female progeny by the <br />                                           total number of female progeny. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Immunoprecipitation of cohesin and Nipped-B from the nuclear extract <br />                                           Nuclear extract was prepared from 100 ml of Kc cells grown to a density of approximately 5 <br />                                           &#195;&#8212; 106 cells per ml. Cells were collected by centrifugation at 300 &#195;&#8212; g for 5 min at 4&#194;&#176;C, suspended <br />                                           in 15 ml of phosphate buffered saline (PBS, pH 7.0), and collected by centrifugation at 300 &#195;&#8212; <br />                                           g for 5 min at 4&#194;&#176;C. The washed cells were suspended in 15 ml ice-cold hypotonic buffer (10 <br />                                           mM hydroxyethyl piperazineethanesulfonic acid (HEPES)&#226;&#8364;&#8220;KOH pH 7.9, 50 mM KCl, 1.5 mM <br />                                           DTT, 1 mM benzamidine, 1 &#206;&#188;g per ml pepstatin A, 1 &#206;&#188;g per ml antipain, 0.5 mM PMSF) and <br />                                           collected by centrifugation at 300 &#195;&#8212; g for 5 min at 4&#194;&#176;C. The cell pellet was suspended in 5 vol <br />                                           of hypotonic buffer, incubated on ice for 10 min, and homogenized with 30 strokes in a ground <br />                                           glass homogenizer. Nuclei were collected by centrifugation at 2,000 &#195;&#8212; g for 5 min at 4&#194;&#176;C. The <br />                                           nuclei were suspended in 5 vol of hypotonic buffer and collected by centrifugation at 2,000 &#195;&#8212; <br />                                           g for 5 min at 4&#194;&#176;C. The nuclei were suspended in 2 vol of DNaseI digestion buffer (50 mM <br />                                           HEPES&#226;&#8364;&#8220;KOH pH 7.9, 100 mM KCl, 2.5 mM MgCl2, 1 mM MnCl2, 1 mM DTT, 0.2% [v/v] NP-40, <br />  <br />                                           1 mM benzamidine, 1 &#206;&#188;g per ml pepstatin A, 1 &#206;&#188;g per ml antipain, 0.5 mM PMSF) and incubated <br />                                           on ice for 2 min. The suspended nuclei were treated with 20 &#206;&#188;g of DNaseI (Sigma D-08876, <br />                                           10 Kunitz units) at 25&#194;&#176;C for 15 min. Insoluble material was removed by centrifugation at <br />                                           100,000 &#195;&#8212; g for 60 min at 4&#194;&#176;C, and after adjusting to 10% (v/v) glycerol, the supernatant was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           flash frozen in dry ice-ethanol and stored at &#226;&#710;&#8217;80&#194;&#176;C. <br />  <br />                                           For immunoprecipitation, 200 &#206;&#188;g of nuclear extract was precleared by incubation with 20 &#206;&#188;l <br />                                           of protein-A agarose beads (prewashed in DNaseI digestion buffer) for 1 h at 4&#194;&#176;C, and beads <br />                                           were removed by centrifugation. The precleared extract was incubated with 10 &#206;&#188;l of the <br />                                           indicated preimmune or immune serum and incubated for 4 hr at 4&#194;&#176;C with gentle mixing. <br />                                           Immune complexes were collected by incubating with 20 &#206;&#188;l of prewashed protein A-agarose <br />                                           beads for 1 h at 4&#194;&#176;C and then collecting the beads by centrifugation for 15 s in an Eppendorf <br />                                           centrifuge. The beads were washed five times with 0.5 ml of DNaseI digestion buffer containing <br />                                           0.4% NP-40 and then twice with DNaseI digestion buffer. The beads were boiled in SDS- <br />                                           PAGE dye for 5 min, and the samples were separated by SDS-PAGE in 6% gels. Western blots <br />                                           were prepared and probed for cohesin subunits, Nipped-B, and Pds5 as described above. <br />  <br />                               Results <br />                               Identification of Nipped-B mutations <br />                                           We reasoned that if some of Nipped-B&apos;s functions do not involve cohesin, then it might be <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           possible to find Nipped-B separation-of-function mutations. For example, a mutation could <br />                                           cause recessive cohesion defects but not have dominant effects on gene expression. To <br />                                           investigate this possibility, we characterized several Nipped-B mutations that were induced in <br />                                           an isogenic cn bw chromosome by ethylmethane sulfonate (EMS) mutagenesis (Myster et al. <br />                                           2004). To identify the mutations and distinguish them from polymorphisms, we sequenced <br />                                           Nipped-B transcripts from an isogenic cn bw stock kept in our laboratory, several EMS-induced <br />                                           mutant alleles, and compared them to the previously reported Nipped-B cDNA sequence <br />                                           (Rollins et al. 1999). <br />  <br />                                           The sequences revealed substantial amino acid coding variation in wild-type Nipped-B alleles, <br />                                           including differences in the number of residues (Supplementary Fig. 2). In addition to the wild- <br />                                           type polymorphisms, we found significant alternative splicing at the N terminus, some of which <br />                                           alters the encoded protein (Supplementary Figs. 3 and 4). <br />  <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                                Page 6 <br />  <br />  <br />                                           We also identified sequence changes unique to each of nine EMS-induced Nipped-B mutant <br />                                           alleles (Fig. 1). Three are missense mutations (Nipped-BNC39, Nipped-BNC71; Nipped- <br />                                           BNC78), two of which (NC71, NC78) occur in HEAT repeats. Each <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>missense mutation</span> affects <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           a residue conserved in human NIPBL, and all are close to conserved residues altered by CdLS- <br />                                           causing missense NIPBL mutations (Fig. 2; Gillis et al. 2004;Miyake et al. 2005;Schoumans <br />                                           et al. 2007). Two other Nipped-B mutations, Nipped-BNC7 and Nipped-BNC59, truncate the <br />                                           encoded protein in the C-terminal region after the HEAT repeats, and four, Nipped-BNC6, <br />                                           Nipped-BNC23, Nipped-BNC41, and Nipped-BNC77, truncate the protein in the N-terminal region <br />                                           before the HEAT repeats. All the mutations are single-nucleotide changes, except Nipped- <br />                                           BNC77, which is a short duplication that creates a stop codon and a new splice donor site. All <br />                                           the mutations affect residues encoded by all splicing variants and thus are unlikely to produce <br />                                           any wild-type Nipped-B isoforms. <br />  <br />                                           All the EMS-induced mutations, including the three missense alleles, are recessive lethal at <br />                                           the second to third instar molt, the same as null alleles (Rollins et al. 1999). We tried to detect <br />                                           the truncated proteins by Western blot using antibodies generated against the N-terminal region <br />                                           of the protein in early embryos, which show the highest levels of Nipped-B protein because of <br />                                           maternal loading (Rollins et al. 1999). Only very minor amounts of Nipped-B protein migrating <br />                                           at the predicted smaller sizes, which could potentially be the truncated proteins, were detected <br />                                           in embryos from Nipped-BNC7/+ and Nipped-BNC59/+ mothers (Supplementary Fig. 1). The <br />                                           full-length maternal protein is reduced in level, and we deduce, therefore, that the truncated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           forms of the protein are unstable. Because the truncation alleles are virtually equivalent to null <br />                                           alleles, it cannot be determined if the C-terminal residues are critical for specific Nipped-B <br />                                           functions. <br />  <br />                               Dominant effects of Nipped-B mutations on cut and Ultrabithorax phenotypes <br />                                           To determine if any of the EMS-induced mutations separate the gene expression and cohesion <br />                                           functions of Nipped-B, we measured their effects on cut and Ubx phenotypes and on sister <br />                                           chromatid cohesion. Nipped-B loss-of-function alleles generated by &#206;&#179;-rays, including Nipped- <br />                                           B407, Nipped-B292.1, and T(2;3)Nipped-B359.1, dominantly enhance the mutant phenotypes of <br />                                           some cut gene alleles and reduce expression of a wild-type cut gene (Rollins et al. 1999, <br />                                           2004). Similarly, Nipped-B loss-of-function alleles dominantly increase the severity of the <br />                                           bx34e allele of Ubx when it is partially suppressed by the su(Hw)e2 mutation (Rollins et al. <br />                                           1999). Homozygous Nipped-B mutations and heteroallelic combinations with the Nipped- <br />                                           B02047 P insertion allele cause sister chromatid cohesion defects before death (Rollins et al. <br />                                           2004). <br />  <br />                                           To measure effects of the EMS-induced Nipped-B mutations on cut expression, we used the <br />                                           wing-nicking phenotype caused by the ctK mutation. The ctK allele is an insertion of a gypsy <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           transposon that has fewer potential Su(Hw) insulator protein-binding sites. It causes partial <br />                                           loss of the adult wing margin by partially blocking the remote wing margin enhancer that drives <br />                                           cut expression during wing margin development. Decreases in ctK expression further reduce <br />                                           the amount of wing margin as quantified by counting the number of nicks or gaps in the bristle <br />                                           row along the adult wing margin (Dorsett et al. 2005; Gause et al. 2001; Rollins et al. 2004). <br />                                           Null Nipped-B alleles and in vivo Nipped-B RNAi increase the wing nicking displayed by <br />                                           ctK (Rollins et al. 2004). <br />  <br />                                           All the EMS-induced Nipped-B alleles dominantly increased the ctK wing nicking (Fig. 3). The <br />                                           two missense mutations affecting HEAT repeats, Nipped-BNC71 and Nipped-BNC78, exhibited <br />                                           a significantly weaker effect than the other alleles, while the other missense allele, Nipped- <br />                                           BNC39, had effects similar to the truncations. One of the C-terminal truncation alleles, Nipped- <br />                                           BNC59, reproducibly had stronger effects on the ctK phenotype than the others, but we cannot <br />                                           rule out the possibility that there are other mutations on this chromosome that enhance the <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                               Page 7 <br />  <br />  <br />                                           effect. It is also possible that small amounts of the truncated protein are present during wing <br />                                           margin development, and that this truncated protein might be antimorphic. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           All the EMS-induced Nipped-B alleles dominantly enhanced the bithorax phenotype of su <br />                                           (Hw)e2 bx34e, similar to the effects of null Nipped-B alleles (Rollins et al. 1999). Although the <br />                                           Nipped-BNC71 and Nipped-BNC78 missense mutations had the smallest effects, they were only <br />                                           slightly less than those caused by the other mutant alleles, suggesting that Ubx expression is <br />                                           more sensitive to reduced Nipped-B activity than is cut (Fig. 3). When the alleles are broken <br />                                           into three categories, missense, truncations after the HEAT repeats (C region truncations), and <br />                                           truncations before the HEAT repeats (N region truncations), the missense mutants as a group <br />                                           induce a significantly weaker bithorax phenotype than either of the truncation classes (5.0 vs <br />                                           5.4 and 5.5). Thus, as with cut, the missense mutants as a class have weaker effects on Ubx <br />                                           expression than do the truncation classes. <br />  <br />                               Recessive effects of Nipped-B mutations on sister chromatid cohesion <br />                                           We compared the effects of the Nipped-B alleles on sister chromatid cohesion by examining <br />                                           neuroblast metaphase nuclei for precocious sister chromatid separation (PSCS). To avoid <br />                                           effects of other potential recessive mutations, we tested heteroallelic combinations of each <br />                                           allele with the Nipped-B02047 P element insertion (Rollins et al. 1999, 2004). <br />  <br />                                           Table 1 shows the number of metaphases examined and the number of examples of PSCS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           detected for each mutant, and Fig. 4 shows some examples of the PSCS phenotype. Because <br />                                           the Nipped-B alleles are recessive lethal before the molt to third instar and the mitotic index in <br />                                           the mutants is low, it is difficult to obtain enough metaphases to accurately measure the <br />                                           frequency of PSCS for each individual allele. To make statistical comparisons, therefore, the <br />                                           mutants were categorized into missense, C-terminal region truncation, and N-terminal region <br />                                           truncation classes as described above. All classes increase the frequency of PSCS compared <br />                                           to the wild-type control. The missense alleles, Nipped-BNC39, Nipped-BNC71, and Nipped- <br />                                           BNC78, individually and as a class gave a significantly lower frequency of PSCS (&#226;&#710;&#188;30%) than <br />                                           did the truncation classes, which were similar to each other (59 and 62%) and the frequency <br />                                           reported for Nipped-B null alleles (61%, Rollins et al. 2004). Using Fisher&apos;s exact test, which <br />                                           is applicable when there are low sample numbers (Fisher 1922), the missense mutant class is <br />                                           significantly different from wild-type and both truncation classes, while the two truncation <br />                                           classes are not different from each other (Table 1). Thus, the missense mutants have weaker <br />                                           effects on both gene expression and sister chromatid cohesion than do truncation alleles, and <br />                                           as expected, both classes of truncation alleles have effects on gene expression and sister <br />                                           chromatid cohesion similar to those of null alleles. We conclude, therefore, that none of the <br />                                           nine EMS-generated alleles, including the three missense alleles, separate the gene expression <br />                                           and sister chromatid cohesion functions of Nipped-B. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Nipped-B, cohesin, and Pds5 colocalize on salivary polytene chromosomes <br />                                           If Nipped-B&apos;s roles in sister chromatid cohesion, gene expression, and meiosis all involve <br />                                           regulating cohesin function, we would expect Nipped-B and cohesin to colocalize on <br />                                           chromosomes. To test this, we double immunostained salivary gland polytene for Nipped-B <br />                                           and cohesin. In all the many nuclei examined, Nipped-B staining was completely coincident <br />                                           with staining for the Smc1 (Fig. 5) and SA cohesin subunits (not shown). There was a general <br />                                           correlation in the intensity of staining&#226;&#8364;&#8221;Regions that stained strongly for Nipped-B also stained <br />                                           strongly for cohesin. Similar to the results for Nipped-B, we found that the Pds5 protein, which <br />                                           is required for establishment and/or maintenance of cohesion in multiple organisms (Dorsett <br />                                           et al. 2005;Hartman et al. 2000;Losada et al. 2005;Panizza et al. 2000;Tanaka et al. 2001), also <br />                                           colocalizes with cohesin on polytene chromosomes (Fig. 5). Preimmune serum did not show <br />                                           staining, and the secondary antibodies used for the coimmunostaining experiments do not show <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                             Page 8 <br />  <br />  <br />                                           species cross-reactivity. Homozygous pds5e3 null mutant polytene chromosomes, which stain <br />                                           for cohesin (Dorsett et al. 2005), do not stain for Pds5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           It has been proposed that in addition to regulation of cohesin, interactions between the NIPBL <br />                                           ortholog of Nipped-B with heterochromatin protein HP1 might be functionally important for <br />                                           Nipped-B&apos;s human ortholog (Lechner et al. 2005). In S. pombe, interactions between the Swi6 <br />                                           HP1 ortholog and cohesin are important for sister chromatid cohesion (Bernard et al. 2001; <br />                                           Nonaka et al. 2002). In contrast to the colocalization of Nipped-B with cohesin, however, <br />                                           double-immunostaining experiments indicate that Nipped-B does not colocalize with HP1 on <br />                                           chromosomes (Fig. 5). The brightest regions of HP1 staining, which include the chromocenter <br />                                           and region 31B on the left arm of chromosome 2, stain weakly if at all for Nipped-B. A few <br />                                           bright Nipped-B bands stain weakly for HP1, but most do not. These results indicate that <br />                                           significant HP1&#226;&#8364;&#8220;Nipped-B interactions do not occur on chromosomes. We cannot exclude the <br />                                           possibility that interactions with HP1 mask the epitopes recognized by the Nipped-B <br />                                           antibodies, but if this is the case, we deduce that such sites would lack cohesin, given the <br />                                           complete colocalization of Nipped-B and cohesin. <br />  <br />                                           It is possible that HP1-Nipped-B interactions occur only when the proteins are not associated <br />                                           with chromosomes. Nevertheless, the phenotypes of HP1 mutants indicate that if such <br />                                           interactions do occur, they are not critical for Nipped-B and cohesin function. It has been <br />                                           reported that polytene chromosomes appear relatively normal in HP1 [Su(var)2&#226;&#8364;&#8220;5] mutants <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           and cohesion defects do not occur in any of the several tissues examined (Fanti et al. 1998). <br />                                           Polytene chromosomes that lack cohesin, on the other hand, show a significant change in <br />                                           morphology (Dorsett et al. 2005). <br />  <br />                               Nipped-B colocalizes with cohesin and the synaptonemal complex during female meiosis <br />                                           There is evidence in yeast and vertebrates that meiosis-specific forms of cohesin are involved <br />                                           in formation of the synaptonemal complex (SC; reviewed in Revenkova and Jessberger <br />                                           2006), and the mushroom Rad9 adherin mutant starts but fails to complete SC assembly (Seitz <br />                                           et al. 1996; Cummings et al. 2002). Rad9 mutants also show defective cohesion. Combined, <br />                                           these observations suggest that Nipped-B might also facilitate SC formation and/or <br />                                           maintenance by regulating cohesin function. <br />  <br />                                           We immunostained Drosophila oocyte chromosome spreads with anti-Nipped-B, and observed <br />                                           that it colocalizes extensively with the Smc1 and Smc3 cohesin subunits in a thread-like <span id='am-473' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                                           along the arms of meiotic chromosomes (Fig. 6). It also colocalizes with the Ord protein, which <br />                                           is required for meiotic cohesion in both sexes and SC maintenance in oocytes (Bickel et al. <br />                                           1997;Khetani and Bickel 2007;Webber et al. 2004), and with C(3)G, a transverse filament <br />                                           protein that resides within the central element of the SC (Anderson et al. 2005). These data <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           argue that Nipped-B may play a direct role in regulating meiotic cohesion as well as SC <br />                                           structure and function. <br />  <br />                                           Although Nipped-B exhibits robust localization along the arms of meiotic chromosomes, <br />                                           centromeric Nipped-B staining was surprisingly weak or nonexistent (Fig. 6). In contrast to <br />                                           Nipped-B, bright staining for the cohesion proteins Ord and Smc1/Smc3 is visible in <br />                                           centromeric heterochromatin, which is identified by bright diamidinophenylindole staining and <br />                                           immunostaining for the CID centromere-specific histone (Fig. 6; Khetani and Bickel <br />                                           2007;Webber et al. 2004). These data suggest that Nipped-B may not associate strongly with <br />                                           meiotic centromeres and are consistent with the lack of Nipped-B signal at the centromeres of <br />                                           polytene chromosomes. However, we cannot rule out the possibility that the epitopes <br />                                           recognized by Nipped-B antibodies are masked within pericentric heterochromatin. <br />  <br />  <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                                Page 9 <br />  <br />  <br />                               Nipped-B mutations cause early chromosome core dissolution <br />                                           Nipped-B homozygotes die before meiosis can be observed, but Nipped-B loss-of-function <br />                                           mutations have a dominant effect on meiotic chromosome morphology. Female meiosis starts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           at the tip of each ovariole of the ovary in the germarium, which contains four developmental <br />                                           regions (1, 2A, 2B, 3). Germline mitotic divisions take place in region 1, and prophase I initiates <br />                                           in region 2A, followed by formation of the SC. Even after the oocyte exits the germarium, the <br />                                           SC is normally maintained throughout pachytene until stage 6 of oogenesis. <br />  <br />                                           In 108 out of the 112 ovarioles examined from Nipped-B10E heterozygotes, the thread-like C <br />                                           (3)G and Smc1/3 staining starts to fragment in germarial region 3 or stage 2 oocytes (Fig. 7). <br />                                           Fragmentation does not occur this early in the wild type. Nipped-B10E is a loss-of-function <br />                                           allele generated by excision of the Nipped-B02047 P element insertion, and similar defects were <br />                                           also observed for females heterozygous for the &#206;&#179;-ray-induced Nipped-B407, Nipped-B292.1, and <br />                                           T(2;3)Nipped-B359.1 alleles. <br />  <br />                                           Although fragmentation of cohesin chromosome cores is severe in stage 2 Nipped-B mutant <br />                                           oocytes, Smc1 and Smc3 proteins remain associated with meiotic centromeres (Figs. 7 and 8). <br />                                           The bright Smc1/3 spots in the Nipped-B mutant stage 2 oocytes in Fig. 7 also stain for the <br />                                           centromere-specific histone, CID (Fig. 8). Combined with the lack of Nipped-B staining at <br />                                           centromeres, retention of normal Smc1 and Smc3 staining at centromeres in Nipped-B mutants <br />                                           suggest that Nipped-B may not modulate the behavior of cohesin within pericentric <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           heterochromatin. <br />  <br />                               Early SC breakdown in Nipped-B mutants does not affect meiotic chromosome segregation <br />                                           In homozygous ord mutants, Smc1 and Smc3 fail to accumulate at centromeres, and the Smc1 <br />                                           and Smc3 threadlike staining along chromosome arms is already disrupted in germarium region <br />                                           2B, which is earlier than in the heterozygous Nipped-B mutants (Khetani and Bickel 2007). <br />                                           The ord mutants also show elevated levels of meiotic nondisjunction (Bickel et al. 1997). We <br />                                           tested if the changes in SC structure caused by heterozygous Nipped-B mutants affect <br />                                           chromosome segregation and found no significant meiotic nondisjunction (Table 2; 0 of 1,303 <br />                                           total progeny from Nipped-B mutant females, 1 of 1,342 progeny from Nipped-B mutant <br />                                           males). <br />  <br />                                           A more sensitive test also failed to detect effects on meiotic segregation in heterozygous <br />                                           Nipped-B mutants. Maternal transmission of the J21A minichromosome with a weak <br />                                           centromere is roughly half that expected for a normal chromosome (Murphy and Karpen <br />                                           1995). This reduced transmission is very sensitive to changes in the dosage of genes involved <br />                                           in chromosome segregation (Dobie et al. 2001). For example, heterozygous wings-apart-like <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (wapl) mutations, which decrease mitotic chromatid separation by reducing prophase removal <br />                                           of cohesin from chromosome arms, decrease J21A transmission in female meiosis (Dobie et <br />                                           al. 2001; Gandhi et al. 2006; Kueng et al. 2006; Verni et al. 2000). Females heterozygous for <br />                                           the parental chromosome in which the Nipped-B407 and Nipped-B292.1 mutations were induced <br />                                           (Rollins et al. 1999) transmitted J21A at a frequency of 0.24 (348 of 1,441; Table 2). J21A was <br />                                           transmitted at a frequency of 0.29 (384 of 1,315) by females heterozygous for Nipped-B407, <br />                                           and a frequency of 0.33 (493 of 1,479) by Nipped-B292.1/+ females (Table 2). The control and <br />                                           Nipped-B mutant J21A transmission frequencies all fall within the normal range (0.22 to 0.37) <br />                                           and are close to the wild-type average of 0.27 (Dobie et al. 2001). Thus, centromeric cohesion <br />                                           does not appear to be altered in heterozygous Nipped-B mutant females, and the early SC <br />                                           dissolution does not alter chromosome segregation during meiosis I. <br />  <br />  <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                             Page 10 <br />  <br />  <br />                               Nipped-B is in complexes with the Rad21 and SA cohesin subunits <br />                                           The colocalization of Nipped-B with cohesin along polytene chromosomes and meiotic <br />                                           chromosome cores and the effects of Nipped-B mutations on mitotic sister chromatid cohesion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           and meiotic chromosome core structure suggest that Nipped-B is directly involved in regulating <br />                                           association of cohesin with chromosomes and its organization on chromosomes. To determine <br />                                           if Nipped-B forms complexes with cohesin, we conducted immunoprecipitation experiments <br />                                           with nuclear extracts of Kc-cultured cells. We prepared extracts using moderate ionic strength <br />                                           and DNaseI digestion to release chromosome-bound cohesin and maximize the probability of <br />                                           detecting Nipped-B&#226;&#8364;&#8220;cohesin complexes. <br />  <br />                                           As expected, immunoprecipitation with anti-Smc1 brings down high levels of the SA and <br />                                           Rad21 cohesin subunits, and precipitation with anti-SA coprecipitates high amounts of Smc1 <br />                                           and Rad21 (Fig. 9). In contrast, none of the cohesin subunit precipitations brought down <br />                                           detectable amounts of Nipped-B. Precipitation of Nipped-B, however, brought down small but <br />                                           detectable amounts of SA and Rad21 but not Smc1 (Fig. 9). Because the amounts were small, <br />                                           we repeated the experiment nine times with different nuclear extracts and a different cell line <br />                                           (MLDmD8) and always detected SA and Rad21 but not Smc1. <br />  <br />                                           A Rad21&#226;&#8364;&#8220;SA subcomplex is present in Drosophila cultured cells (Vass et al. 2003), and the <br />                                           immunoprecipitation results thus suggest that there are small amounts of complexes that <br />                                           contain Nipped-B and the Rad21&#226;&#8364;&#8220;SA subcomplex in nuclear extracts. These complexes likely <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           do not contain Smc1, although we cannot rule out the possibility that the anti-Nipped-B <br />                                           antibodies displace the Smc1&#226;&#8364;&#8220;Smc3 heterodimer from the Rad21&#226;&#8364;&#8220;SA subcomplex. Even so, <br />                                           these results, combined with the colocalization with cohesin on somatic and meiotic <br />                                           chromosomes, support the idea that Nipped-B is directly involved in regulating cohesin <br />                                           function. The apparent lack of stable association between Nipped-B and the complete cohesin <br />                                           complex suggests that such interactions are either less stable and more transient than those with <br />                                           the Rad21&#226;&#8364;&#8220;SA subcomplex or that they may be stabilized by DNA. <br />  <br />                               Discussion <br />                                           In this study, we examined the functions of Drosophila Nipped-B in gene expression, mitotic <br />                                           sister chromatid cohesion, and meiosis. The results support the idea that Nipped-B&apos;s diverse <br />                                           functions in somatic and meiotic cells all involve regulation of cohesin function. <br />  <br />                               Nipped-B HEAT repeat mutations link Nipped-B&apos;s roles in gene expression and sister <br />                               chromatid cohesion <br />                                           Relative to null alleles, the missense Nipped-BNC71 and Nipped-BNC78 mutations that alter <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           HEAT repeats have weaker effects on both gene expression and sister chromatid cohesion. <br />                                           This agrees with prior studies suggesting that Nipped-B&apos;s role in gene expression is to regulate <br />                                           cohesin chromosome binding. The prior studies showed that Nipped-B facilitates activation of <br />                                           the cut gene by a distant transcriptional enhancer and that cohesin has an opposite effect to <br />                                           Nipped-B on cut expression (Rollins et al. 1999; Dorsett et al. 2005; Rollins et al. 2004). They <br />                                           also showed that cohesin binds to cut in multiple cell types and that a unique pds5 mutation <br />                                           that reduces binding of cohesin to chromosomes increases cut expression (Dorsett et al. <br />                                           2005). Combined, these findings and the genetic linkage between Nipped-B&apos;s roles in gene <br />                                           expression and sister chromatid cohesion reported here support the hypothesis that Nipped-B <br />                                           facilitates cut expression by alleviating negative effects of cohesin (Dorsett 2004, 2007). This <br />                                           model suggests that cohesin binding to cut inhibits expression and by maintaining a dynamic <br />                                           cohesin chromosome-binding equilibrium, Nipped-B facilitates cohesin removal or relocation. <br />  <br />  <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                              Page 11 <br />  <br />  <br />                                           Based on this model, we posit that the Nipped-BNC71 and Nipped-BNC78 missense mutations <br />                                           in the HEAT repeats are hypomorphic and reduce but do not abolish the ability of Nipped-B <br />                                           to control cohesin binding to chromosomes. In this scenario, the homozygous HEAT repeat <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           mutants show milder cohesion defects than null alleles because loading of cohesin onto <br />                                           chromosomes is only partially reduced, and the heterozygous mutants have weaker effects on <br />                                           gene expression because they only slightly alter the cohesin chromosome-binding equilibrium. <br />  <br />                                           The Nipped-BNC71 and Nipped-BNC78 missense mutations provide the first evidence that some <br />                                           of the HEAT repeats are important for gene expression and sister chromatid cohesion, and they <br />                                           also link Nipped-B&apos;s role in gene expression to the function of NIPBL and cohesin in human <br />                                           development. There are CdLS-causing missense mutations in all seven HEAT repeats of <br />                                           NIPBL, and Nipped-BNC71 and Nipped-BNC78 affect HEAT repeats 2 and 3, respectively <br />                                           (Gillis et al. 2004; Miyake et al. 2005; Deardorff and Krantz, personal communication). Indeed, <br />                                           Nipped-BNC71 and Nipped-BNC78 both affect conserved residues adjacent or very close to <br />                                           conserved residues affected by CdLS-causing mutations, and thus the CdLS mutations are <br />                                           likely to have dominant effects on gene expression similar to Nipped-BNC71 and Nipped- <br />                                           BNC78. The structural similarity between Nipped-BNC71 and Nipped-BNC78 and the NIPBL <br />                                           mutations provides a link to cohesin in human development because like the NIPBL HEAT <br />                                           repeat mutations, missense mutations in the Smc1 and Smc3 cohesin subunits also cause CdLS <br />                                           (Deardorff et al. 2007; Musio et al. 2006). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Does Nipped-B directly regulate cohesin function? <br />                                           The functional connections between cohesin and the Nipped-B protein family in sister <br />                                           chromatid cohesion, gene expression, and development that have been revealed genetically <br />                                           likely reflect direct regulation of cohesin function by Nipped-B. This is supported by the finding <br />                                           that Nipped-B colocalizes with cohesin on polytene and meiotic chromosomes and the presence <br />                                           of soluble complexes containing Nipped-B and the Rad21 and SA cohesin subunits in cell <br />                                           nuclear extracts. We do not know why soluble complexes containing Nipped-B and whole <br />                                           cohesin were not detected, but it is possible that the antibodies used for immunoprecipitation <br />                                           disrupt these interactions, that the epitopes are masked in such complexes, that such interactions <br />                                           are transitory, or that they are stable only when Nipped-B and cohesin are both bound to DNA. <br />  <br />                                           In S. cerevisiae, cohesin does not colocalize with the Scc2 ortholog of Nipped-B on <br />                                           chromosomes (Lengronne et al. 2004). Nevertheless, purification of yeast Scc2&#226;&#8364;&#8220;Scc4 complex <br />                                           brings along small amounts of cohesin subunits, and thus it has also been proposed that the <br />                                           Scc2&#226;&#8364;&#8220;Scc4 complex also directly regulates binding of cohesin to chromosomes (Arumugam <br />                                           et al. 2003). It is suggested that cohesin loads at Scc2-binding sites and translocates away <br />                                           (Lengronne et al. 2004). In contrast, chromatin immunoprecipitation experiments using <br />                                           cultured Drosophila cells reveals that Nipped-B colocalizes with cohesin in the entire <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           nonrepetitive genome, confirming that their colocalization is not unique to polytene and <br />                                           meiotic chromosomes (Misulovin et al., submitted for publication). Thus, the separate <br />                                           localization of the Nipped-B/Scc2 cohesin loader and cohesin may be unique to yeast <br />                                           chromosomes. <br />  <br />                               Does Nipped-B regulate cohesin function at centromeres? <br />                                           Our results raise the possibility that Nipped-B is not critical for the function of cohesin at <br />                                           centromeres. Cohesin binds densely to pericentric heterochromatin, and cohesin at centromeres <br />                                           is protected in both mitotic and meiotic cells by a member of the Shugoshin/Mei-S332 protein <br />                                           family (Kitajima et al. 2004; McGuinness et al. 2005; Salic et al. 2004; Tang et al. 1998, <br />                                           2004). Although Nipped-B completely colocalizes with Smc1 and Smc3 along meiotic <br />                                           chromosome cores, we did not see Nipped-B staining around the centromeres, where cohesin <br />                                           staining is the strongest. We also did also not detect significant staining for Nipped-B at the <br />  <br />  <br />                                              Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                            Page 12 <br />  <br />  <br />                                           centromeres of polytene chromosomes, but this might be explained by the under-replication <br />                                           of pericentric heterochromatin in these cells. Consistent with this idea, we also did not see <br />                                           strong cohesin staining at polytene centromeres. Nipped-B, however, also does not colocalize <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           with the HP1 heterochromatin protein, whose Swi6 ortholog in S. pombe is required for <br />                                           cohesion and interacts with cohesin (Bernard et al. 2001; Nonaka et al. 2002), at positions that <br />                                           are not under-replicated. In cultured Drosophila cells, Rad21 and SA persist at the centromeres <br />                                           until the metaphase&#226;&#8364;&#8220;anaphase transition (Valdeolmillos et al. 2004; Warren et al. 2000). We <br />                                           have immunostained metaphase chromosome spreads from Kc cells and can see Smc1 and SA <br />                                           that colocalizes with the CID centromere-specific histone at centromeres but have not been <br />                                           able to detect Nipped-B (Gause and Dorsett, unpublished). In this case, however, given the <br />                                           relatively weak signals, lack of staining elsewhere in the same nucleus, and the different <br />                                           fixation conditions needed to get good chromosome morphology, it is difficult to rule out the <br />                                           possibility that we cannot detect Nipped-B because of insufficient sensitivity. <br />  <br />                                           Although the lack of pericentric Nipped-B staining in meiotic chromosomes might be caused <br />                                           by epitope masking, other evidence indicates that Nipped-B function is also not critical at <br />                                           meiotic centromeres. In particular, the retention of Smc1 and Smc3 protein at the meiotic <br />                                           chromosome centromeres in heterozygous Nipped-B mutants and the normal meiotic <br />                                           transmission of the J21A minichromosome with a weak centromere indicate that centromeric <br />                                           cohesion is not altered, although the chromosome cores disintegrate early when the dosage of <br />                                           Nipped-B protein is reduced. This contrasts sharply with the chromosome nondisjunction and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           lack of cohesin binding to the meiotic centromeres in ord mutants (Khetani and Bickel 2007). <br />                                           Thus, the evidence suggests that either Nipped-B does not play a critical role in the function <br />                                           of cohesin at meiotic centromeres or that its role at centromeres is not as dosage sensitive as <br />                                           its function along the chromosome cores. Given the lack of Nipped-B staining, the <br />                                           colocalization of Ord and cohesin, and the lack of cohesin at centromeres in ord mutants, it is <br />                                           possible that Ord functionally substitutes for Nipped-B at meiotic centromeres. <br />  <br />                               Roles of Nipped-B in meiosis <br />                                           Our data provide evidence for a functional link between the Nipped-B protein family and <br />                                           cohesin in meiosis. The C. coprinus Rad9 hypomorphic mutant, in addition to having meiotic <br />                                           cohesion defects, fails to complete SC formation, indicating that the Nipped-B family is <br />                                           involved in SC assembly (Cummings et al. 2002; Seitz et al. 1996). The findings that Nipped- <br />                                           B colocalizes with cohesin subunits along meiotic chromosome cores and that cores <br />                                           disassemble prematurely when Nipped-B dosage is reduced indicates that the Nipped-B family <br />                                           is also involved in the maintenance of SC structure. <br />  <br />                                           The Nipped-B family functions in SC assembly and maintenance are both likely to involve <br />                                           cohesin. Cohesin colocalizes with SC proteins and is involved in homolog pairing, SC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           formation, and SC structure in diverse organisms (Bannister et al. 2004; Chan et al. 2003; Eijpe <br />                                           et al. 2000; Khetani and Bickel 2007; Klein et al. 1999; Pasierbek et al. 2001, 2003; Revenkova <br />                                           et al. 2004; Xu et al. 2005). The C(2)M protein, which contains kleisin motifs similar to those <br />                                           in Rad21 and which interacts with Smc3, is needed for SC formation and the coalescence of <br />                                           Smc1 and Smc3 into chromosome cores (Heidmann et al. 2004; Khetani and Bickel 2007; <br />                                           Manheim and McKim 2003). C(2)M, however, is not required for binding of the Smc1 and <br />                                           Smc3 cohesin subunits to either centromeres or chromosome arms (Khetani and Bickel <br />                                           2007). Taken together, these data suggest a model in which Nipped-B and C(2)M collaborate <br />                                           in formation of the cohesin chromosome cores and that Nipped-B and Ord then cooperate to <br />                                           maintain this structure. Thus, the Nipped-B protein family plays a role beyond simply loading <br />                                           cohesin onto chromosomes and is involved directly in regulating the higher-order meiosis- <br />                                           specific organization of cohesin. <br />  <br />                                           Fig. 1. <br />                                           Nipped-B mutations and predicted mutant proteins. The diagrams at the top indicate the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           predicted protein products of the sequenced Nipped-B mutant alleles, and the table at the <br />                                           bottom gives the changes in nucleotide sequence from the parental wild-type cn bw sequence <br />                                           and the associated change in protein coding. Orange boxes show the positions of the HEAT <br />                                           repeats <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                      Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 2. <br />                                           Alignment of Nipped-B missense mutations to CdLS-causing NIPBL mutations. Nipped- <br />                                           BNC71 and Nipped-BNC78 alter conserved residues in HEAT repeats 2 and 3, respectively, and <br />                                           are close to conserved residues altered in NIPBL missense mutants. Nipped-BNC39 affects a <br />                                           conserved residue between HEAT repeats 4 and 5 that is immediately adjacent to a conserved <br />                                           residue altered by a NIPBL missense mutation <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                             Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 3. <br />                                           Effects of Nipped-B mutations on su(Hw)e2 bx34e and ctK mutant phenotypes. The dominant <br />                                           effects of the indicated Nipped-B alleles on the bithorax phenotype displayed by su(Hw)e2 <br />                                           bx34e males and the cut wing phenotype displayed by ctK males were quantified as described <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           in the text. The error bars on the bithorax phenotype data are standard errors. The cut wing <br />                                           phenotype data are presented as box plots with the vertical lines indicating the 10th, 25th, 50th <br />                                           (median), 75th, and 90th percentiles for each genotype. The circles are data points that fall <br />                                           outside the 10th and 90th percentiles. Asterisks indicate phenotypes that differ significantly <br />                                           from the wild-type control (cn bw-a) by the Bonferroni&#226;&#8364;&#8220;Dunn post-hoc test <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                    Page 20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 4. <br />                                           Examples of precocious sister chromatid separation (PSCS) in Nipped-B mutants. The <br />                                           panels show an example wild-type second instar neuroblast metaphase spread and examples <br />                                           of PSCS in Nipped-BNC6/Nipped-B02047 and Nipped-BNC41/Nipped-B02047 second instar <br />                                           metaphase spreads. Arrowheads indicate instances of PSCS. For Table 1, a metaphase was <br />                                           scored as having PSCS if one or more chromosomes show PSCS <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                          Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 5. <br />                                           Immunostaining of salivary gland polytene chromosomes for Nipped-B, Smc1, Pds5, and HP1 <br />                                           proteins. The upper panels show double immunostaining of Oregon R wild-type salivary gland <br />                                           chromosomes for Nipped-B and the Smc1 cohesin subunits. The upper left panel shows a whole <br />                                           nucleus, and the right shows a higher magnification view of the tip of the X chromosome from <br />                                           the same nucleus. The Nipped-B and Smc1 proteins colocalize, and there is a strong correlation <br />                                           in Nipped-B and Smc1 staining intensity. The bottom right panel shows double <br />                                           immunostaining of Oregon R wild-type chromosomes for Pds5 and Smc1, which also <br />                                           colocalize almost completely. The lower left panel shows double immunostaining for Nipped- <br />                                           B and HP1, which do not colocalize. The bright HP1 staining is the heterochromatic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           chromocenter. White scale bars in all panels indicate 5 &#206;&#188;m <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                     Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 6. <br />                                           Immunostaining of female meiotic chromosome spreads for Nipped-B, C(3)G, Smc1/3, Ord, <br />                                           and histone H3 variant, CID. The panels show that Nipped-B colocalizes extensively along <br />                                           the arms with the Smc1 and Smc3 cohesin subunits, the SC component C(3)G and the cohesion <br />                                           protein Ord. Little or no Nipped-B signal is detected at centromeric regions (arrows), as <br />                                           confirmed by staining for the centromere-specific histone H3 variant, CID (bottom panels) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                          Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 7. <br />                                           Dominant effect of the Nipped-B10E mutation on chromosome cores and synaptonemal <br />                                           complex during female meiosis. Meiotic chromosomes at the indicated stages of meiosis and <br />                                           oogenesis were visualized by staining for Smc1/3 and C(3)G protein in wild-type and Nipped- <br />                                           B10E heterozygous mutant ovarioles (whole-mount preparation). Nipped-B10E is a null-like <br />                                           allele generated by P element excision. The developmental stages (germarium regions 2A, 2B, <br />                                           3, and stage 2 of oogenesis) are organized in temporal order from the top down. During wild- <br />                                           type prophase I, the synaptonemal complex first forms in region 2A of the germarium, becomes <br />                                           restricted to the oocyte by region 3, and remains stable until stage 6 of oogenesis. In <br />                                           heterozygous Nipped-B mutants, formation of the chromosome cores in region 2A appears to <br />                                           occur normally, but the cores fragment (lose their linear structure) in germarium region 3 to <br />                                           stage 2 of oogenesis, which is substantially earlier than in the wild type. Despite the early <br />                                           fragmentation, Smc1/3 staining is retained at the centromeres. The bright Smc1/3 spots in the <br />                                           stage 2 oocyte also stain for CID, a centromere-specific histone H3 variant (Fig. 8). Temporal <br />                                           progression of fragmentation defects was quantified in 42 Nipped-B10E/+ ovarioles and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           revealed that more than 50% of region 3 and more than 90% of stage 2 oocytes exhibit extensive <br />                                           fragmentation of chromosome cores. Similar results were obtained with other Nipped-B alleles <br />                                           (see text). The wild type never shows chromosome disintegration at these early stages <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                          Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 8. <br />                                           Colocalization of Smc1/3 and CID in Nipped-B mutant meiotic chromosomes. The upper <br />                                           panels show the staining of the same stage 2 oocyte in Fig. 7 for Smc1/3 and C(3)G in color, <br />                                           and the lower left panel shows the staining for the CID centromere-specific histone. The lower <br />                                           right panel is a merge of the Smc1/3 and CID staining, showing that the bright Smc1/3 spots <br />                                           occur at centromeres <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                             Gause et al.                                                                                               Page 25 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 9. <br />                                           Immunoprecipitation of cohesin subunits and Nipped-B from Kc cell nuclear extracts. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           serum used for precipitation is indicated above each Western blot. For each blot, pre indicates <br />                                           a preimmune serum control precipitation, imm indicates immune serum precipitation, and <br />                                           extract is a control lane containing input nuclear extract. The left panels show that precipitation <br />                                           with anti-SA or anti-Smc1 precipitates significant amounts of other cohesin subunits. We were <br />                                           unable to detect Nipped-B or Pds5 in these precipitates in multiple experiments. The panels <br />                                           on the right show that precipitation of Nipped-B coprecipitates small amounts of Rad21 and <br />                                           SA. In all nine repeated Nipped-B immunoprecipitation experiments with different extract <br />                                           preparations, we detected Rad21 and SA but were unable to detect Smc1. The rabbit anti- <br />                                           Nipped-B in these experiments precipitates all Nipped-B from the extract (Supplementary Fig. <br />                                           1). In some experiments, such as those on the right, we detected multiple forms of Rad21 or <br />                                           SA but do not know if these represent protein modifications or partial proteolysis <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                                                                      NIH-PA Author Manuscript                                               NIH-PA Author Manuscript                                                 NIH-PA Author Manuscript <br />                                                                                                                                                                       Table 1 <br />                                                                     Precocious sister chromatid separation in Nipped-B mutants <br />  <br />                                                                         Nipped-B allelea                   Number of metaphases               Number of PSCSb                Frequency of PSCS               Fisher exact test P valuec vs <br />  <br />                                                                                                                                                                                                              Missense           C-region truncations          N-region truncations <br />                                                                                                                                                                                                                                                                                          Gause et al. <br />  <br />  <br />  <br />  <br />                                                                         Wild type <br />                                                                         &#226;&#8364;&#402;Oregon R                          30                                 1                              0.03                            5.2&#195;&#8212;10&#226;&#710;&#8217;3           1.5&#195;&#8212;10&#226;&#710;&#8217;6                      3.4&#195;&#8212;10&#226;&#710;&#8217;9 <br />                                                                         Missense <br />                                                                         &#226;&#8364;&#402;NC39                              14                                 5                              0.36 <br />                                                                         &#226;&#8364;&#402;NC71                              17                                 4                              0.24 <br />                                                                         &#226;&#8364;&#402;NC78                              12                                 4                              0.33 <br />                                                                         &#226;&#8364;&#402;Total                             43                                 13                             0.30                                               2.5&#195;&#8212;10&#226;&#710;&#8217;2                      6.5&#195;&#8212;10&#226;&#710;&#8217;4 <br />                                                                         C region truncations <br />                                                                         &#226;&#8364;&#402;NC7                               6                                  3                              0.50 <br />                                                                         &#226;&#8364;&#402;NC59                              31                                 19                             0.61 <br />                                                                         &#226;&#8364;&#402;Total                             37                                 22                             0.59                                                                             0.55 <br />                                                                         N region truncations <br />                                                                         &#226;&#8364;&#402;NC6                               34                                 21                             0.62 <br />                                                                         &#226;&#8364;&#402;NC23                              17                                 10                             0.59 <br />                                                                         &#226;&#8364;&#402;NC41                              22                                 14                             0.64 <br />                                                                         &#226;&#8364;&#402;NC77                              8                                  6                              0.75 <br />                                                                         &#226;&#8364;&#402;Total                             81                                 51                             0.63 <br />  <br />                                                                         a <br />                                                                          All alleles were tested as heterozygotes with Nipped-B02047. <br />                                                                         b <br />                                                                          Metaphases with one or more examples of sister chromatid separation <br />                                                                         c <br />                                                                          Fisher&apos;s exact test (Fisher 1922) calculates the probability (P value) that two populations are same using small sample sizes. In this table, we compared the wild-type sample vs the totals for the missense <br />                                                                         mutant, the C-region truncation, and N-region truncation groups and the totals for each of the mutant groups to each other. All the comparisons indicate that the groups are significantly different from <br />                                                                         each other (low P values), except for the comparison of the C and N truncation mutant groups. <br />  <br />  <br />  <br />  <br /> Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />                                                                                                                                                                                                                                                                                          Page 26 <br />                                   Gause et al.                                                                                                                           Page 27 <br />  <br />                                                                                                   Table 2 <br />                                  Meiotic chromosome segregation in heterozygous Nipped-B mutants <br />  <br />                            Parent                                                   Na                       Nondisjunctionb                       J21A transmissionc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#226;&#8482;&#8364; Nipped-B407/+                                          636                      0                                     nad <br />                            &#226;&#8482;&#8364; Nipped-B292.1/+                                        667                      0                                     na <br />                            &#226;&#8482;&#8218; Nipped-B407/+                                          635                      0                                     na <br />                            &#226;&#8482;&#8218; Nipped-B10E/+                                          431                      0.0023                                na <br />                            &#226;&#8482;&#8218; Nipped-B292.1/+                                        150                      0                                     na <br />                            &#226;&#8482;&#8218; Nipped-B359.1/+                                        126                      0                                     na <br />                            &#226;&#8482;&#8364; Nipped-B+/+; ry506; J21A, ry+                          1441                     na                                    0.24&#194;&#177;0.03e <br />                            &#226;&#8482;&#8364; Nipped-B407/+; ry506; J21A, ry+                        1315                     na                                    0.29&#194;&#177;0.03 <br />                            &#226;&#8482;&#8364; Nipped-B292.1/+; ry506; J21A, ry+                      1479                     na                                    0.33&#194;&#177;0.03 <br />  <br />                            a <br />                             Number of progeny scored <br />                            b <br />                             Frequency of exceptional progeny; see text for assay <br />                            c <br />                             Frequency of J21A presence in female progeny; see text for assay. The female parents in the seventh row were heterozygous for the parental chromosome <br />                            (Nipped-B+) used to generate the Nipped-B407 and Nipped-B292.1 mutations (Rollins et al. 1999). <br />                            d <br />                                not applicable <br />                            e <br />                             95% confidence interval <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                            Chromosoma. Author manuscript; available in PMC 2008 February 28. <br />   <br /> </body></html>