<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>TbRGG1, essential protein involved kinetoplastid <br /> RNA metabolism associated novel <br /> multiprotein complex <br />  <br />  <br />                              &#194;&#180; ,2 ASWINI K. PANIGRAHI,2 KENNETH D. STUART,2 JULIUS LUKES&#203;&#8225;1 <br /> HASSAN HASHIMI,1 ALENA ZI&#194;&#180;KOVA <br /> 1 <br />  Biology Centre, Institute Parasitology, Czech Academy Sciences, Faculty Sciences, University South Bohemia, <br /> C&#203;&#8225;eske&#194;&#180; Bude&#203;&#8225;jovice (Budweis), Czech Republic <br /> 2 <br />  Seattle Biomedical Research Institute, Seattle, Washington 98109, USA <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> The uridine insertion/deletion RNA editing kinetoplastid mitochondrial transcripts performed complex machinery <br /> involving number proteins multiple protein complexes. Here effect silencing TbRGG1 gene by <br /> <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>RNA interference</span> RNA editing procyclic stage Trypanosoma brucei. TbRGG1 essential protein cell growth, the <br /> absence results overall decline edited mRNAs, levels never-edited RNAs remain unaltered. <br /> Repression TbRGG1 expression effect 20S editosome MRP1/2 complex. TAP-tag purification TbRGG1 <br /> coisolated novel multiprotein complex, association verified TAP-tag analyses components <br /> of complex. TbRGG1 interaction complex appears mediated RNA. Our results suggest TbRGG1 <br /> protein functions stabilizing edited RNAs editing efficiency associated novel complex role in <br /> mitochondrial RNA metabolism. We provisionally putative mitochondrial RNA-binding complex 1 (put-MRB complex 1). <br /> Keywords: RNA editing; mitochondrion; trypanosome; RNA binding <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                   editosomes present organelle, unique one <br />                                                                                type endonucleases (Panigrahi et al. 2006). <br /> Trypanosoma brucei flagellate protist order <br />                                                                                   During editing, sequence information provided by <br /> Kinetoplastida responsible African sleeping sickness in <br />                                                                                small RNA molecules termed guide (g) RNAs. The role of <br /> humans nagana livestock. As early di- <br />                                                                                mt RNA-binding proteins processing event an <br /> verged eukaryotes, contains array remarkable <br />                                                                                anticipated feature entails RNA/RNA hybridization. In <br /> biological properties present eukary- <br />                                                                                addition component proteins, proteins not <br /> otic cells. A prominent example post-transcriptional <br />                                                                                stably associated editosome involved in <br /> uridine (U) insertion/deletion RNA editing mito- <br />                                                                                process proteins labeled acces- <br /> chondrial (mt) transcripts. Only edited, the <br />                                                                                sory factors (Stuart Panigrahi 2002). There growing <br /> mRNAs translatable, mainly encoding subunits of <br />                                                                                family proteins multiple functions mt RNA <br /> the respiratory complexes. The numerous insertions and <br />                                                                                metabolism. The mt RNA-binding proteins (MRPs) are <br /> deletions Us performed 20S editosome (also <br />                                                                                present MRP1/2 heterotetrameric complex serves <br /> known L-complex). The extensively characterized edito- <br />                                                                                initial stage RNA editing matchmaker by <br /> somes (for recent reviews, Simpson et al. 2004; Lukes&#203;&#8225; <br />                                                                                facilitating hybridization gRNA cognate pre- <br /> et al. 2005; Stuart et al. 2005; Aphasizhev 2007) contain at <br />                                                                                edited mRNA (Aphasizhev et al. 2003a; Schumacher et al. <br /> least 20 proteins, types 20S <br />                                                                                2006). Moreover, pleimorphic <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> caused the <br />                                                                                down-regulation MRP1 and/or MRP2 indicated that <br />                                                                                apart role editing, proteins func- <br />    Reprint requests <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>: Julius Lukes&#203;&#8225;, Biology Centre, Institute Parasi-    tion aspects RNA metabolism (Vondrus&#203;&#8225;kova&#194;&#180; <br /> tology, Branis&#203;&#8225;ovska&#194;&#180; 31, 37005 C&#203;&#8225;eske&#194;&#180; Bude&#203;&#8225;jovice, Czech Republic; e-mail:   et al. 2005). RBP16 RNA-binding protein plays a <br /> jula@paru.cas.cz; fax: 00420-38-5310388. <br />    Article published online ahead print. Article publication date   role RNA editing vivo (Pelletier Read 2003) is <br /> at http://www.rnajournal.org/cgi/doi/10.1261/rna.888808.                       able stimulate insertion editing vitro (Miller et al. <br />  <br /> 970                                    RNA (2008), 14:970&#226;&#8364;&#8220;980. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2008 RNA Society. <br />                                                                                     TbRGG1 component novel complex <br />  <br />  <br />  <br /> 2006). Furthermore, influences stability never- <br /> edited RNAs edited counterparts <br /> (Pelletier Read 2003) subject arginine <br /> methylation regulates functions (Goulah et al. <br /> 2006). Another member heterogeneous family <br /> is REAP-1, protein initially specifically bind to <br /> pre-edited mRNAs, proposed recruit pre- <br /> mRNAs editing complex (Madison-Antenucci et al. <br /> 1998). However, recent study REAP-1 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>knock-out</span> cells <br /> implies role RNA stability editing (Hans <br /> et al. 2007). Although RBP38 identified a <br /> protein binds gRNA&#226;&#8364;&#8220;mRNA duplexes (Sbicego et al. <br /> 2003), function initiation of <br /> kinetoplast DNA replication (Liu et al. 2006). <br />    TbRGG1 protein ranked category of <br /> associated factors RNA editing (Vanhamme et al. 1998). <br /> It contains arginine-glycine-glycine (RGG) tripeptide, <br /> which repeated close proximity and <br /> interspersed aromatic residues, comprising motif <br /> that present number RNA-binding proteins (Burd <br /> and Dreyfuss 1994). The positively charged arginines are <br /> thought electrostatically interact negatively <br /> charged amino acids, facilitating RNA binding. <br /> TbRGG1 T. brucei 85-kDa protein equally <br /> present procyclic bloodstream stages. In glycerol <br /> gradients, TbRGG1 sedimented 35&#226;&#8364;&#8220;40 S fractions that <br /> also exhibited vitro deletion activity, implying both <br /> association macromolecular complex RNA <br /> editing (Vanhamme et al. 1998). The recombinant protein <br /> bound poly(U) high affinity, capacity suggestive of <br /> gRNA binding, molecules 39 oligo(U) tails <br /> (Blum Simpson 1992). Interestingly, UV cross-linking <br /> of radiolabeled gRNAs mitochondrial lysates yielded this <br /> protein (Leegwater et al. 1995). However, evidence for <br /> function TbRGG1 RNA editing ultimately circum- <br /> stantial. Another mitochondrial RGG protein initially <br /> identified mass spectrometric analysis monoclonal      FIGURE 1. RNAi TbRGG1 inhibits cell growth, decreases <br />                                                                  protein mRNA levels. (A) Growth effect. The numbers of <br /> antibody affinity purified editosome protein         noninduced cells (s) induction addition of <br /> called TbRGGm homology Trypano-            1 mg/mL tetracycline (m) plotted logarithmically product <br /> soma cruzi protein dubbed TcRGGm (Panigrahi et al.               cell density total dilution. Growth curves representative <br /> 2003a). Here, present resulting effect             set independent experiments. (B) Effect TbRGG1 <br />                                                                  mRNA. Its levels analyzed blotting 10 mg total RNA <br /> repression TbRGG1 RNA interference (RNAi) T.            extracted 29-13, noninduced (Tet ) induced (Tet+) cells <br /> brucei. Moreover, report RNase-sensitive association      (days 1&#226;&#8364;&#8220;8 post-induction RNAi). The position targeted <br /> with novel complex identification components        mRNA dsRNA synthesized following induction indicated. <br /> by mass spectrometry.                                            As loading control, gel stained ethidium bromide to <br />                                                                  visualize rRNA bands. (C) Effect TbRGG1 protein. Its levels <br />                                                                  analyzed Western blot extracts 29-13, noninduced <br />                                                                  (Tet ) induced (Tet+) cells lysed day 1&#226;&#8364;&#8220;13 48 h. Each <br /> RESULTS                                                          lane loaded protein z5 3 106 cells blots were <br />                                                                  immunodecorated using polyclonal antibody TbRGG1, cyto- <br /> Silencing TbRGG1 inhibits cell growth                         solic glyceraldehyde-3-phosphate dehydrogenase (GAPDH), and <br />                                                                  mitochondrial RNA-binding protein 1 (MRP1). (D) 20 S editosome <br /> An inhibition growth apparent 5 d induction      unchanged. The editosome protein levels analyzed Western <br /> of RNAi silencing TbRGG1 (Fig. 1A). Thereafter,           blots mitochondrial extracts 29-13 noninduced induced <br />                                                                  cells lysed day 1&#226;&#8364;&#8220;13 48 h. Western blots simulta- <br /> induced cells grew approximately times slower       neously probed a-KREPA1, KREPA2, KREL1, KREPA3 <br /> culture tetracycline induction absent,          monoclonal antibodies (top), polyclonal antibodies the <br /> virtually stopped growing day 12. The RNAi knock-             mitochondrial heat shock protein 70 (hsp-70) (bottom). <br />  <br />  <br />                                                                                                         www.rnajournal.org          971 <br />  Hashimi et al. <br />  <br />  <br />  <br /> down cells showed recovery 2 wk RNAi                tification elimination TbRGG1 mRNA, which <br /> induction. The extent TbRGG1 mRNA silencing,           reduced 20% normal level (Fig. 2). <br /> as tightness inducible down-regulation, deter-        The analysis mt mRNAs revealed overall decline of <br /> mined Northern analysis RNA collected par-             levels edited transcripts (Fig. 2). The edited cyto- <br /> ental 29-13, noninduced 1&#226;&#8364;&#8220;8 d post-induction cells            chrome oxidase subunit 3 (co3) NADH dehydrogenase <br /> (Fig. 1B). The target message undetectable 24 h      subunit 7 (ND7) mRNAs decreased 50%, while <br /> induction tetracycline remained 8 d.     decline pronounced ATPase subunit 6 (A6), <br /> In addition, leaky transcription p2T7-177             maxicircle unknown reading frame 2 (MURF2), ribosomal <br /> vector evident, dsRNA appeared             protein S12 (RPS12), co2 mRNAs (about 30%&#226;&#8364;&#8220;40%), <br /> RNAi induction (Fig. 1B). This observation consistent          subtle decrease noted edited cytochrome <br /> with Western analysis, showing level TbRGG1           reductase subunit B (cyB). The decrease edited molecules <br /> protein significantly drops day 5 postinduction            correlated increase respective pre-edited <br /> (Fig. 1C), corresponds nicely appearance        mRNAs, especially co3. There virtually change <br /> growth inhibition (Fig. 1A), down-regulation         levels never-edited ND4 co1 mRNAs. The <br /> persists 2 wk time course. In contrast, signal       abundant mt 9S rRNA remained unaffected, slight <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> GAPDH antibody, used loading              decrease apparent 12S rRNA. <br /> control (Fig. 1C), persisted time course.             To investigate repression TbRGG1 leads to <br /> Western analysis using monoclonal antibodies          changes gRNA population, RNA used <br /> core editosome proteins KREPA1, KREPA2, KREL1,                qPCR assays labeled guanylyltransferase, an <br /> KREPA3 (Fig. 1D) accessory complex protein MRP1           enzyme specifically labels gRNAs, subse- <br /> (Fig. 1C) showed effect abundance proteins         quently visualized denaturing acrylamide gel (Fig. 3). <br /> consequent TbRGG1 silencing.                                   A band corresponding cytosolic RNA labeled <br />    It originally reported ORF TbRGG1           reaction (Aphasizhev et al. 2003b) used a <br /> present steady-state levels dicistronic transcript, <br /> which contains upstream ORF, ex- <br /> pected processed monocistronic RNA (Vanhamme et al. <br /> 1998). Northern analysis performed confirm the <br /> fully processed mRNA upstream ORF af- <br /> fected dsRNA targeting TbRGG1, thus, the <br /> observed growth inhibition TbRGG1 <br /> down-regulation (data shown). <br />  <br /> Reduction edited RNAs upon <br /> TbRGG1 down-regulation <br /> To determine TbRGG1 role RNA editing, <br /> levels mt RNA species compared between <br /> cultures growing 4 d presence absence the <br /> RNAi induction agent tetracycline. This time point was <br /> chosen just observable growth inhibition <br /> (Fig. 1A), TbRGG1 protein virtually elim-          FIGURE 2. RNAi TbRGG1 affects RNA editing. Real-time PCR <br />                                                                   analysis pre-edited, edited, never-edited mRNAs ribosomal <br /> inated (Fig. 1C). Quantitative real-time (q) PCR cDNA          RNAs encoded mt genome. Analysis performed triplicate <br /> from cultures performed primers            cDNAs generated cells grown 4 d presence or <br /> several pre-edited, edited, never-edited RNAs, pre-        absence tetracycline. For target amplicon, relative change <br /> viously described (Carnes et al. 2005). Moreover, newly           RNA abundance induction RNAi silencing TbRGG1 <br />                                                                   determined using cytosolic transcripts b-tubulin (white bar) <br /> designed primers amplifying mt 9S 12S rRNAs,          18S rRNA (gray bar) internal references, transcrip- <br /> as TbRGG1, used. All qPCR reactions performed           tion affected. The relative abundance examined <br /> in triplicate, including determination expres-   transcript synthesis TbRGG1 dsRNA plotted a <br /> sion b-tubulin 18S rRNA, average               logarithmic scale: 1.0 represents wild-type level; levels and <br />                                                                   1.0 mean increase decrease given RNA, respectively. <br /> median standard deviation measured cycle threshold         The following pre-edited (P) edited (E) mRNAs assayed: <br /> (Ct) values 0.1 0.07, respectively (data shown).       ATPase subunit 6 (A6), cytochrome oxidase subunits 2 (co2), 3 <br /> The transcription levels housekeeping genes         (co3), cytochrome reductase subunit b (cyB), maxicircle unknown <br />                                                                   reading frame 2 (MURF2), NADH dehydrogenase subunit 7 (ND7), <br /> unaltered TbRGG1-interference, served              ribosomal protein S12 (RPS12). The following never-edited RNAs <br /> references calculation relative abundances          assayed: 9S RNA, 12S RNA, co1, ND4. The levels the <br /> the examined transcripts. The qPCR allowed quan-            nuclear-encoded TbRGG1 mRNA tested (TbRGG1). <br />  <br />  <br /> 972      RNA, Vol. 14, No. 5 <br />                                                                                             TbRGG1 component novel complex <br />  <br />  <br />  <br />                                                                           lower &#226;&#8364;&#732;&#226;&#8364;&#732;S&#226;&#8364;&#8482;&#226;&#8364;&#8482; value originally reported (Vanhamme et al. <br />                                                                           1998), consistently earlier observation our <br />                                                                           laboratory (Vondrus&#203;&#8225;kova&#194;&#180; et al. 2005). <br />                                                                              The clarified lysate expressed TAP-tagged <br />                                                                           TbRGG1 subjected tandem-affinity purification <br />                                                                           protocol. Western analysis aliquots final four <br />                                                                           TAP eluates showed presence tagged protein (Fig. <br />                                                                           5A), SYPRO Ruby staining SDS-PAGE separated <br />                                                                           sample showed visible protein bands the <br />                                                                           tagged TbRGG1 complex (Fig. 5B). The fraction was <br />                                                                           probed mix monoclonal antibodies against <br />                                                                           core editosome proteins (KREPA1, KREPA2, KREL1, <br />                                                                           KREPA3), reacted protein <br />                                                                           complex containing TbRGG1 (Fig. 5C). Thus, tagged <br />                                                                           TbRGG1 complex does associate editosomes. <br />                                                                              Mass spectrometry analysis tagged TbRGG1 <br />                                                                           complex identified proteins (Table 1; Supplemental <br />                                                                           Tables 1 2), including proteins, Tb927.2.3800 and <br /> FIGURE 3. RNAi TbRGG1 does affect gRNAs. The total                 Tb927.7.2570, known motifs 31% <br /> population minicircle-encoded gRNAs visualized high-          sequence identity 48% similarity 432 amino acids <br /> resolution acrylamide-urea gel labeling 2.5 mg total RNA with <br /> guanylyltransferase [g-32P]GTP. The left right lane contains      them. These proteins identified the <br /> RNA noninduced (Tet ) induced (Tet+) TbRGG1 RNAi                 sample high peptide coverage (data shown), thus, <br /> cells (4 d induction), respectively. The gRNAs, range size    selected reciprocal TAP-tag analyses. Calmodulin <br /> mainly variable lengths 39-oligo(U) tails,        elution fractions Tb927.2.3800 Tb927.7.2570 TAP- <br /> indicated right. The band marked &#226;&#8364;&#732;&#226;&#8364;&#732;*&#226;&#8364;&#8482;&#226;&#8364;&#8482; cytosolic <br /> RNA concurrently labeled enzyme used           tag purifications stained SyproRuby (Fig. 6) and <br /> loading control. The sizes vitro-transcribed RNAs,        analyzed LC-MS/MS, TbRGG1 protein was <br /> were end-labeled separate reactions, indicated left.        detected. <br />                                                                              To assign protein complex, data all <br />                                                                           TAP-tag cell lines compiled. Fourteen proteins <br /> loading control samples induced                      identified probability $0.9 three <br /> noninduced cells. As judged assay, TbRGG1 repression              TAP-tags, unique peptide matches in <br /> does significantly affect steady-state levels gRNAs.               them, assigned component com- <br />                                                                           plex (Table 1). Seven proteins identified <br />                                                                           using above-mentioned conditions, known sub- <br /> TbRGG1 associated multiprotein complex <br />                                                                           units different mt complexes and/or abundant mito- <br /> The sedimentation profile TbRGG1 glycerol                         chondrial nonmitochondrial proteins, operationally <br /> gradient determined Western analysis indicated that <br /> it large protein complex (Vanhamme et al. <br /> 1998). To explore possibility, TAP-tagged TbRGG1 <br /> protein expressed procyclic T. brucei cells the <br /> control tetracycline-inducible promoter. Expression of <br /> the tagged protein monitored glycerol gradients using <br /> the PAP reagent, specifically recognizes Protein A <br /> domain tag. A strong band 105 kDa showed the <br /> distribution protein fractions 3&#226;&#8364;&#8220;13, bulk <br /> being present fractions 5&#226;&#8364;&#8220;9 (Fig. 4). Two additional <br /> bands apparent major uppermost band, <br /> which correspond modification TbRGG1                         FIGURE 4. Sedimentation profile protein complex containing <br /> proteolysis means, observed postulated                 TbRGG1. The cleared lysate hypotonically isolated mitochondria <br />                                                                           cells containing TAP-tagged TbRGG1 grown 2 d the <br /> the original report (Vanhamme et al. 1998). The                      presence 500 ng/mL tetracycline loaded glycerol <br /> gradient probed anti-KREPA3 antibody showed                      gradients fractionated described Materials Methods. <br /> peak 20 S editosome signal fraction 11 (Fig. 4). The                   Fractions immunodecorated PAP reagent, binds <br />                                                                           protein A domain TAP-tag (top), anti-KREPA3 <br /> sedimentation properties tagged TbRGG1 protein                     (bottom) antibody marker 20 S fractions. The lower two <br /> suggest incorporation complex overlaps               bands visualized PAP reagent correspond modification <br /> the 20 S editosome, sedimented somewhat                  TAP-tagged TbRGG1 protein proteolysis means. <br />  <br />  <br />                                                                                                               www.rnajournal.org       973 <br />  Hashimi et al. <br />  <br />  <br />  <br />                                                                           29-13 cell line lysed presence ab- <br />                                                                           sence RNase A, lysates fractionated by <br />                                                                           sedimentation glycerol gradients (Fig. 7A). As evident <br />                                                                           Western analysis using anti-TbRGG1 antibody, <br />                                                                           pronounced shift localization the <br />                                                                           TbRGG1 protein lighter fractions gradient <br />                                                                           RNase treatment, peaking fractions 5 7 <br />                                                                           absent fraction 9. TbRGG1 untreated samples <br />                                                                           peaks fractions 7 11, detected throughout <br />                                                                           gradient. The peak fractionation subunits of <br />                                                                           20 S editosome, KREPA1 KREPA2, unaffected <br />                                                                           RNase treatment (Fig. 7A). <br />                                                                              In light result, initial IgG affinity selection step <br />                                                                           repeated TAP-tagged TbRGG1 presence <br />                                                                           absence RNase A. The eluates resulting TEV <br />                                                                           cleavage, releases protein(s) bound IgG matrix <br />                                                                           TAP-tag, resolved glycerol gradient centri- <br />                                                                           fugation. A shift sedimentation TAP-tagged TbRGG1 <br /> FIGURE 5. Identification proteins associated TAP-tagged <br /> TbRGG1 protein. (A) Eluates E1&#226;&#8364;&#8220;E4 TAP-tag purification           lighter fractions apparent ribonuclease is <br /> using TbRGG1 protein bait visualized Western analysis      present purification step (Fig. 7B). SyproRuby <br /> using anti-his antibody, recognizing his-epitope present   staining calmodulin eluates directly separated on <br /> remaining TAP-tag calmodulin binding           SDS-PAGE gel reveals abundant bands corre- <br /> peptide (TbRGG1-CBP). The eluate enriched TAP- <br /> tagged protein (E2) subjected LC-MS/MS. As glycerol         sponding size Tb927.2.3800 Tb927.7.2570 <br /> gradient, lower bands apparent eluates enriched    proteins disappear presence RNase A (Fig. 7C). <br /> tagged protein, possibly products modification TbRGG1       Furthermore, TbRGG1 peptides detected these <br /> portion fusion. (B) Eluate E2 stained Sypro Ruby.         eluates mass spectroscopy analysis. Reciprocally, TbRGG1 <br /> The predicted position TbRGG1-CBP copurified proteins <br /> Tb927.2.3800 Tb927.7.2570 indicated right &#226;&#8364;&#732;&#226;&#8364;&#732;*&#226;&#8364;&#8482;&#226;&#8364;&#8482;     detected method Tb927.2.3800 TAP <br /> and arrows, respectively. (C) The purified proteins E2    eluates purified presence RNase A, other <br /> fraction subjected simultaneous immunoanalysis using          component proteins complex present, as <br /> anti-KREPA1, KREPA2, KREL1, KREPA3 monoclonal antibodies, <br />                                                                           compared parallel untreated purifications (data not <br /> using whole-cell lysates (WCL) T. brucei positive control. <br />                                                                           shown). We propose novel multiprotein complex <br />                                                                           involved mt RNA metabolism based observa- <br /> regarded contaminants (Supplemental Table 1). We                  tions TbRGG1, protein role, interacts <br /> list numerous proteins identified              RNA-mediated manner, contains <br /> the tagged complexes identified peptide                  proteins RNA-binding processing motifs. <br /> match considered complex <br /> (Supplemental Table 2). <br />                                                                           DISCUSSION <br />    Results detailed sequence analysis compo- <br /> nent proteins public databases presented Table                Since serendipitous discovery 10 yr ago (Vanhamme <br /> 1. Some proteins complex high degree                 et al. 1998), TbRGG1 remained least- <br /> of conservation eukaryotic RNA helicases                     characterized proteins putatively associated RNA editing, <br /> RNA-binding proteins. Moreover, motifs involved RNA&#226;&#8364;&#8220;                   functions RNA editing accessory factors <br /> protein protein&#226;&#8364;&#8220;protein interactions, RRM, GRP,               elucidated, extent. It proposed <br /> and ankyrin repeat polypeptide              TbRGG1 role RNA editing is <br /> sequences. Interestingly, TbRGGm (Tb10.406.0050), pro-                  macromolecular complex based reported <br /> tein previously identified associated affinity              biochemical properties (Vanhamme et al. 1998; Simpson et al. <br /> purified editosome (Panigrahi et al. 2003a),               2004; Lukes&#203;&#8225; et al. 2005; Stuart et al. 2005). Here, test both <br /> the RNA-binding motif containing proteins present                  hypotheses vivo using reverse genetics TAP- <br /> complex. It noteworthy pairs closely related               purification approaches established T. brucei <br /> proteins identified complex (Table 1).                        subsequent discovery protein. <br />                                                                              Using RNAi-based approach, generated cells <br /> Association TbRGG1 multiprotein <br />                                                                           deficient TbRGG1 showed editing all <br /> complex RNA mediated <br />                                                                           analyzed mRNAs down-regulated, varying <br /> To test association TbRGG1 mediated                 degrees, assayed never-edited mRNAs remain <br /> RNA interactions, hypotonically isolated mitochondria                     wild-type levels. A similar phenotype been <br />  <br /> 974       RNA, Vol. 14, No. 5 <br />                                                                                                    TbRGG1 component novel complex <br />  <br />  <br />  <br />  TABLE 1. Mitochondrial proteins associated TAP-tagged TbRGG1 protein, Tb927.2.3800, Tb927.7.2570 protein <br />  procyclic T. brucei <br />  <br />                                                                                                                                     Motif/domain <br />  Protein           GeneDB         TbRGG1       Tb927.2.3800       Tb927.2.2570             Homology (E-value)                   (E-value) <br />  <br />  Tb927.6.2230tag      TbRGG1               O               O                  O           Caenorhabditis hypothetical           &#226;&#8364;&#8221; <br />                                                                                             protein (4e-13) <br />  Tb927.7.2570tag      HP*                  O               O                  O           Tb927.2.38001                         &#226;&#8364;&#8221; <br />  Tb927.2.3800tag      HP*                  O               O                  O           Tb927.7.25701                         &#226;&#8364;&#8221; <br />  Tb927.5.3010         HP                   O               O                  O           Babesia hypothetical protein          Ribosomal S2 protein <br />                                                                                             (9e-09); bacterial putative           signature <br />                                                                                             RNA-binding protein (6e-06) <br />  Tb927.4.1500         HP                   O               O                  O           Oryza ATP-dependent RNA               dsrm (9e-07); <br />                                                                                             helicase (1e-121)                     Helicase C (2e-13) <br />  Tb11.02.5390         HP                   O               O                  O           &#226;&#8364;&#8221;                                     Ankyrin repeat (0.83) <br />  Tb11.01.8620         HP                   O               O                  1           &#226;&#8364;&#8221;                                     &#226;&#8364;&#8221; <br />  Tb10.406.0050        RBP#                 O               O                  O           Caenorhabditis RNA helicase           RRM (6.3e-11) <br />                                                                                             (1e-09) <br />  Tb927.8.8170         HP                   O               O                  1           Tb927.4.41602                         &#226;&#8364;&#8221; <br />  Tb927.4.4160         HP                   O               1                  O           Tb927.8.81702                         &#226;&#8364;&#8221; <br />  Tb927.3.1820         HP                   O               O                  O           &#226;&#8364;&#8221;                                     GRP (0.028) <br />  Tb11.01.7290         Nudix hydr*          O               O                  1           &#226;&#8364;&#8221;                                     NUDIX (4e-05) <br />  Tb927.3.4920         HP                   1               O                  O           Arabidopsis calcium binding           LETM1 (4e-08) <br />                                                                                             mt protein (9e-06) <br />  Tb927.6.2140         HP                   1               O                  1           Bacterial hydratase (6e-05)           2-keto-4 pentenoate <br />                                                                                                                                   hydratase (5e-07) <br />  <br />  (O) Protein identified unique tryptic peptides; (1) protein identified 1 unique tryptic peptide; (*) proteins also <br />  known LtAP1-3 identified L. tarentolae MRP1/2 complex (3); (superscript &#226;&#8364;&#732;&#226;&#8364;&#732;tag&#226;&#8364;&#8482;&#226;&#8364;&#8482;) tagged proteins study; (superscript &#226;&#8364;&#732;&#226;&#8364;&#732;#&#226;&#8364;&#8482;&#226;&#8364;&#8482;) protein also <br />  known TbRGGm (20); (superscript &#226;&#8364;&#732;&#226;&#8364;&#732;1&#226;&#8364;&#8482;&#226;&#8364;&#8482;) proteins 31% sequence identity 48% similarity 432 amino acids them; and <br />  (superscript &#226;&#8364;&#732;&#226;&#8364;&#732;2&#226;&#8364;&#8482;&#226;&#8364;&#8482;) proteins 77% sequence identity 85% similarity 904 amino acids them. <br />  <br />  <br />  <br />  <br /> observed subunits 20 S editosome silenced,                    information proper U-insertion/deletion cog- <br /> which direct role RNA editing (Carnes et al. 2005;                  nate mRNA (Blum Simpson 1992). Because its <br /> Trotter et al. 2005; Salavati et al. 2006). This result note-              aforementioned affinity poly(U), suggested that <br /> worthy compared studies expres-                        TbRGG1 interacts directly gRNAs, function- <br /> sion RNA editing accessory factors disrupted. In                  ing stabilization maturation. As judged the <br /> the procyclic stages, RNAi knock-down of two <br /> subunits MRP1/2 complex decreased levels a <br /> subset edited never-edited mRNAs (Vondrus&#203;&#8225;kova&#194;&#180; <br /> et al. 2005). Gene knock-out MRP1 bloodstages had <br /> a comparable effect (Lambert et al. 1999), silencing of <br /> RBP16 caused reduction edited cyB never-edited <br /> RNAs (Pelletier Read 2003). Such pleimorphic effect <br /> on mt transcripts observed TbRGG1 silenced. <br />    Because vitro poly(U)&#226;&#8364;&#8220;RNA binding activity, the <br /> TbRGG1 protein suggested alternate role <br /> in synthesis and/or processing mt RNAs <br /> (Leegwater et al. 1995; Vanhamme et al. 1998) contain <br /> the post-transcriptionally added 39-oligo(U) extension <br /> (Aphasizhev 2007). This possibility tested as- <br /> saying levels mitoribosomal RNAs parallel <br /> with mRNAs qPCR. In induced cells, the <br /> levels 9S 12S rRNAs remain unchanged and <br /> slightly reduced, respectively, implying TbRGG1 does <br />                                                                               FIGURE 6. Sypro Ruby staining purified Tb927.2.3800 (left) and <br /> not proposed role mt rRNA synthesis. <br />                                                                               Tb927.7.2570 (right) complexes. The predicted position tagged <br />    Another prominent class mt RNAs added                              proteins (Tb927.2.3800-CBP Tb927.7.2570-CBP) indicated on <br /> 39-oligo(U) tails gRNAs, supply essential                   right. <br />  <br />  <br />                                                                                                                        www.rnajournal.org          975 <br />  Hashimi et al. <br />  <br />  <br />  <br />                                                                                                  gRNAs requisite decrypting the <br />                                                                                                  appropriate mRNAs. Although gua- <br />                                                                                                  nylyltransferase labeling mole- <br />                                                                                                  cules shown stability and <br />                                                                                                  processing remain intact repres- <br />                                                                                                  sion TbRGG1, possibility the <br />                                                                                                  protein binds gRNAs order to <br />                                                                                                  facilitate utilization, role pro- <br />                                                                                                  posed RBP16 (Pelletier Read <br />                                                                                                  2003), remains open. However, the <br />                                                                                                  reduction edited co2 mRNA the <br />                                                                                                  TbRGG1 knockdowns quite telling, <br />                                                                                                  gRNA located 39- <br />                                                                                                  UTR, requiring inde- <br />                                                                                                  pendent trans-acting gRNA molecules <br />                                                                                                  (Kim et al. 1994). TbRGG1 binding to <br />                                                                                                  gRNAs, heterogeneous in <br />                                                                                                  sequence, likely involve <br />                                                                                                  conserved features 39 oli- <br />                                                                                                  go(U) tail, absent co2 <br />                                                                                                  cis-gRNA, implied incapacity <br />                                                                                                  act trans (Golden Hajduk <br />                                                                                                  2005). Thus, based observation <br />                                                                                                  co2-edited RNAs affected by <br />                                                                                                  TbRGG1 depletion, gRNA levels <br />                                                                                                  not, conclude protein <br />                                                                                                  does act small RNAs. <br /> FIGURE 7. TbRGG1 association MRB complex 1 appears RNase sensitive. (A) <br />                                                                                                  An explanation consistent with <br /> Hypotonically isolated mitochondria 29-13 cell line lysed absence (left, vivo data TbRGG1 be <br /> RNase-) presence (right, RNase+) 0.1 mg/mL RNase A. The lysates loaded involved stabilization edited <br /> glycerol gradients containing concentration enzyme. Western analysis RNAs. Alternately, role in <br /> on resulting fractions anti-TbRGG1 antibodies, anti-KREPA1 anti- <br /> KREPA2 antibodies markers 20 S fractions. (B) IgG purification TAP-tagged editing efficiency, depletion a <br /> TbRGG1 presence absence RNase A. The eluates resulting TEV cleavage, protein function also <br /> which releases protein(s) bound IgG matrix TAP-tag, resolved glycerol result decrease edited RNAs. <br /> gradient centrifugation. A shift sedimentation TAP-tagged TbRGG1 lighter           It postulated TbRGG1 is <br /> fractions apparent ribonuclease present purification step. The <br /> peak fraction designated black line pooled subjected second affinity associated  protein complex per- <br /> purification step. (C) SyproRuby staining calmodulin eluates purified pooled forming key editing activities, is <br /> glycerol gradient fractions separated 10.5%&#226;&#8364;&#8220;14% SDS-PAGE gel shows abundant well-known 20 S editosome or <br /> bands corresponding size Tb927.2.3800 Tb927.7.2570 proteins disappear L-complex. Yet, detailed analyses the <br /> presence RNase A, indicated arrows. TbRGG1-CBP designated &#226;&#8364;&#732;&#226;&#8364;&#732;*.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                                                  composition editosome(s) T. <br />                                                                                                  brucei failed identify TbRGG1 in <br /> guanylyltransferase labeling assay, significant changes                  (Aphasizhev et al. 2003b; Panigrahi et al. 2003a,b, <br /> are observed steady-state levels gRNAs                   2006). Indeed, RNAi silencing TbRGG1 did affect <br /> processing TbRGG1 silencing.                                           editosome MRP1/2 complex, since <br />    One plausible interpretation results                    subunits incorporated complexes <br /> TbRGG1 role RNA editing, effect                       remained wild-type levels, indication integrity <br /> its silencing limited decrease edited mRNAs                   (Wang et al. 2003; Vondrus&#203;&#8225;kova&#194;&#180; et al. 2005). To charac- <br /> compared assayed transcripts. However,                    terize composition complex predicted interact <br /> most likely does direct role process,               TbRGG1, overexpression TAP-tagged protein <br /> the observed down-regulation occurs corre-                        performed intention identifying any <br /> sponding accumulation pre-edited RNAs, observed                       copurified subunits mass spectrometry. The sedimenta- <br /> when multiround editing disrupted silencing                    tion properties glycerol gradients tagged TbRGG1 <br /> components 20 S editosome (Carnes et al. 2005;                       suggest incorporation complex, lower <br /> Trotter et al. 2005; Salavati et al. 2006). An indirect role             density fractions initially reported (Vanhamme et al. <br /> TbRGG1 process interaction                      1998). The discrepancy described sedimentation <br />  <br /> 976      RNA, Vol. 14, No. 5 <br />                                                                                      TbRGG1 component novel complex <br />  <br />  <br />  <br /> properties TbRGG1 technical differences           hypothesis supported effect RNAi silencing of <br /> in gradient preparation studies, localization    TbRGG1 edited RNAs association with <br /> of vitro editing activity, fractions       proteins containing motifs mediate RNA metabolism <br /> TbRGG1 originally (Vanhamme et al. 1998), higher          and/or protein&#226;&#8364;&#8220;protein interactions. In addition, pro- <br /> density typically observed (Stuart et al. 2005).           vide evidence association TbRGG1 this <br />    The association TbRGG1 proteins                complex mediated RNA. We the <br /> represent separate complex demonstrated               process extensive characterization function and <br /> coisolation novel set proteins TAP-purified           composition enigmatic protein complex. <br /> TbRGG1, detected mass spectrometry. The associa- <br /> tion validated tagging component <br /> proteins complex. Based TAP-tag analyses,         MATERIALS AND METHODS <br /> assigned 14 proteins novel putative complex, which <br /> include Nudix hydrolase, RNA helicase, TbRGGm,               Plasmid construction, cell culture, transfection, <br /> hypothetical proteins identifiable motifs.        RNAi induction, growth curves <br /> A probable TbRGG1 interaction complex RNA            A 543-nt long N-terminal region TbRGG1 gene overlapping <br /> was revealed. This association likely biologi-       RGG motif PCR amplified using primers RG-F (59-GGA <br /> cally relevant, reproducibly detected TbRGG1,          TCCACTACCGAGATCAGCGCAAC) RG-R (59-AAGCTTGC <br /> Tb927.2.3800, Tb927.7.2570 TAP purifications, with-            ATCCATTCTGTAGCCTG) (added BamHI HindIII restriction <br /> out consistently revealing RNA-binding proteins              sites underlined) genomic DNA T. brucei strain 29-13. <br /> such MRPs, RBP16, REAP1, subunits 20 S               The amplified fragment cloned p2T7-177 vector, <br /> editosome ribosomal proteins potentially asso-         electroporated procyclic T. brucei strain 29-13 and <br />                                                                    selected following protocol described (Vondrus&#203;&#8225;kova&#194;&#180; <br /> ciate TAP-tagged proteins nonspecifically to <br />                                                                    et al. 2005). The synthesis dsRNA induced addition <br /> their affinity nucleic acids. However, com-        1 mg/mL tetracycline cell density measured 24 <br /> plex interacts RNase-sensitive manner TbRGG1            h. Cell growth rate determined using Beckman Z2 Coulter <br /> and contains multiple proteins RNA-binding domains            counter period 15 d induction RNAi. <br /> suggestive roles RNA processing, provisionally               To create construct inducible expression C- <br /> name putative mitochondrial RNA-binding complex             terminally TAP-tagged proteins, full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> TbRGG1 gene <br /> 1 (put-MRB complex 1).                                             amplified primers RG-TAP-Fw (59-GGGCACAAGCTT <br />    Interestingly, proteins identified           ATGGTGTGTAGC) RG-TAP-Rv (59-GGCGGAGGATCC <br /> components MRB complex 1 previously en-                GCTGTCTTCCAGCC) (added HindIII BamHI restriction <br /> countered substoichiometric amounts TAP-purifica-            sites underlined) inserted pLew79-MHTAP plasmid <br /> tions MRP1/2 complex Leishmania tarentolae:            (Panigrahi et al. 2003b; Jensen et al. 2006). Similarly, <br />                                                                    Tb927.2.3800 Tb927.7.2570 genes PCR amplified and <br /> Tb927.2.3800 (LtAP-1), Tb927.7.2570 (LtAP-2), Nudix <br />                                                                    cloned pLew79-MHTAP plasmid. The Tb927.2.3800 ORF <br /> hydrolase (LtAP-3) (Aphasizhev et al. 2003a). In contrast,         PCR amplified genomic DNA 29-13 strain using <br /> TAP monoclonal antibody affinity purification            primers Zah1-TAP-Fw2 (59-ATTTCATGCTGCGCGCGCGCC <br /> MRP1/2 complex T. brucei did copurify pro-            TG) Zah1-TAP-Rv (59-AAAGGATCCGTATGCCGAAACGG <br /> teins (Panigrahi et al. 2008; Z&#196;&#177;&#194;&#180;kova&#194;&#180; et al. 2008),   CAGTC). This PCR product used template PCR with <br /> not identified MRP1 MRP2 proteins put-           primers Zah1-TAP-Fw (59-CTGTACTATATTGAAGCTTATGCT <br /> MRP complex 1. Another purification putative                GCG) aforementioned Zah1-TAP-Rv containing the <br /> complex, using slightly different approach yielding          appropriate HindIII BamHI restriction sites cloning, as <br /> an overlapping subunit composition did contain            indicated underlines. The Tb927.7.2570 ORF PCR <br /> any MRP peptides (Panigrahi et al. 2008). This discrepancy         amplified genomic DNA primers Zah2-TAP-Fw (59- <br /> may RNA-based interactions                 TTGACCTCGAGATGCTTCGC) Zah2-TAP-Rv (59-GAGA <br />                                                                    TCCAGATCTCAACTTCGC) (XhoI BglII sites under- <br /> proteins; technical differences manner purifica- <br />                                                                    lined). The NotI-linearized constructs introduced the <br /> tions performed, cells used,         29-13 cells phleomycin-resistant clones selected. <br /> it simply reflects different types associations take <br /> place species.                                          Northern blot analyses <br />    In conclusion, 14 proteins identified using tagged <br />                                                                    Total RNA isolated using Trizol (Sigma) cells collected <br /> TbRGG1, Tb927.2.3800, Tb927.7.2570 be <br />                                                                    regular intervals 10 mg loaded 1% formaldehyde <br /> genuine components novel ribonucleoprotein complex <br />                                                                    agarose gel, blotted, cross-linked following standard proto- <br /> or collection complexes and/or monomers asso-            cols. After prehybridization Church&#226;&#8364;&#8220;Gilbert solution 1 h at <br /> ciate RNA linkers. We termed put-MRB          65&#194;&#176;C, hybridization radiolabeled probe performed <br /> complex 1, features suggest function        overnight solution temperature. <br /> RNA processing activities T. brucei procyclics,             Membranes washed 2x SSC + 0.1% SDS 15 min at <br /> perhaps indirect role RNA editing. This working              room temperature, followed washes 0.2x SSC + 0.1% <br />  <br />                                                                                                        www.rnajournal.org       977 <br />  Hashimi et al. <br />  <br />  <br />  <br /> SDS 20 min 55&#194;&#176;C. The radioactive signals visu-       9S Forward (59-AATGCTATTAGATGGGTGTGGAA-39); <br /> alized using PhosphorImager Storm 860.                            9S Reverse (59-GCTGGCATCCATTTCTGACT-39); <br />                                                                       12S Forward (59-GGGCAAGTCCTACTCTCCTTT-39); <br /> Western blot analyses                                                 12S Reverse (59-TGCCCCAATCAAACATACAA-39); <br />                                                                       TbRGG1 Forward (59-TTTGACGACCCAAGCACTATGTT-39); <br /> Lysates SDS-PAGE loading buffer equivalent 5 3            and <br /> 106 cells/lane separated 10% 4%&#226;&#8364;&#8220;15% SDS-            TbRGG1 Reverse (59-GCCTCCCAGCGGTCCTAT-39). <br /> PAGE gradient gels. Proteins transferred nitrocellulose <br /> membrane electroblotting, blocked overnight 5% (w/v)          The TbRGG1 primer pair complementary dsRNA <br /> nonfat milk solution, probed anti-    region targeting transcript. For primer, 4 mL 1.5 mM <br /> TbRGG1 antiserum (provided L. Vanhamme) polyclonal          stock added 20-mL reaction, addition 10 mL the <br /> rabbit antisera T. brucei MRP1 (Vondrus&#203;&#8225;kova&#194;&#180; et al. 2005),   Power SYBR Green PCR Master Mix (Applied Biosystems) and <br /> GAPDH (provided P.A.M. Michels) hsp70, dilution         2 mL cDNA. In case reference gene qPCR reactions, <br /> 1:5000. Monoclonal antibodies KREPA1 (TbMP81),                cDNA additionally diluted 1:50. The qPCR program as <br /> KREPA2 (TbMP63), KREL1, KREPA3 (TbMP42) used                 follows: 2 min 50&#194;&#176;C, 10 min 95&#194;&#176;C, 40 cycles 15 sec at <br /> as described (Panigrahi et al. 2001b) lysates       95&#194;&#176;C 60 sec 60&#194;&#176;C, fluorescence detected at <br /> hypotonically isolated mitochondria.                                  end step. The correctness amplicon was <br />                                                                       verified final dissociation step electrophoresis the <br /> Small-scale isolation mitochondria                                 qPCR product 2% agarose gel. The relative abundance of <br />                                                                       RNAs samples determined Pfaffl <br /> A pellet 108 cells resuspended 1.5 mL cold hypertonic <br />                                                                       method (Pfaffl 2001). <br /> NET buffer (100 mM NaCl, 100 mM EDTA pH 8.0, 10 mM <br /> Tris-HCl pH 7.9), incubated ice 10 min, spun <br /> down, supernatant discarded. The pellet re-           Guanylyltransferase labeling <br /> suspended cold hypotonic DTE buffer (1 mM Tris-HCl pH <br />                                                                       To visualize gRNAs, 2.5 mg total RNA noninduced and <br /> 7.9, 1 mM EDTA pH 8.0), disrupts plasma mem- <br />                                                                       induced (4 d) cells DNase treated described and <br /> brane, suspension passed 25-gauge needle <br />                                                                       labeled using guanylyltransferase. A 15-mL reaction contained 1.5 <br /> into 170 mL 60% sucrose. The mitochondrial vesicles spun <br />                                                                       mL 10x Capping Reaction Buffer (500 mM Tris pH 8.0, <br /> down cytosolic content supernatant discarded. <br />                                                                       60 mM KCl, 12.5 mM DTT, 12.5 MgCl2, 0.5 mg/mL BSA, 45 mCi <br /> The pelleted mitochondria resuspended 500 mL STM <br />                                                                       [a-32P]GTP, 40 U RNaseOUT Ribonuclease Inhibitor (Invitro- <br /> buffer (250 mM sucrose, 20 mM Tris-HCl pH 7.9, 2 mM <br />                                                                       gen), 5 U guanylyltransferase (Ambion). The reaction was <br /> MgCl2) supplemented 3 mM MgCl2 5 U DNase I left <br />                                                                       incubated 1 h 37&#194;&#176;C stopped addition 0.1% SDS <br /> on ice 1 h. Half mL STE added stop reaction <br />                                                                       10 mM EDTA. The reaction products phenolchloroform <br /> and mitochondria pelleted washed times <br />                                                                       extracted, ethanol precipitated, dissolved 8 mL 60% <br /> in 500 mL STE (250 mM sucrose, 20 mM Tris-HCl pH 7.9, <br />                                                                       formamide. They incubated 80&#194;&#176;C 2 min, 4 mL were <br /> 2 mM EDTA pH 8.0) manner. The mitochondrial <br />                                                                       loaded high-resolution denaturing 12% acrylamide-8 M urea gel. <br /> pellets stored 80&#194;&#176;C use. <br />  <br /> Quantitative real-time PCR                                            Glycerol gradient sedimentation <br /> Ten micrograms total RNA isolated noninduced TbRGG1           For hypotonic purification mitochondria, pellet from <br /> RNAi cells cells following 4 d RNAi induction treated     1010 cells washed 35 mL SBG buffer (100 mM NaCl; <br /> with Turbo DNA free kit (Ambion) following manufac-           20 mM glucose; 20 mM NaHPO4 pH 7.9) dounce <br /> turer&#226;&#8364;&#8482;s protocol. After DNase step, 1 mg RNA run      homogenized 35 mL DTE solution. Next, 5.85 mL of <br /> formaldehyde gel check integrity treated RNA. To        60% sucrose added hypotonically isolated mt vesicles <br /> prepare templates quantitative real time (q) PCR, cDNA        pelleted resuspended 6.8 mL STM buffer, which <br /> synthesized 4.5 mg RNA SuperScript III reverse       supplemented 21 mL 1 M MgCl2 0.1 M <br /> transcriptase (Invitrogen) using random hexamers volume       CaCl2, 61.2 mg DNase. The DNase treatment stopped <br /> 20 mL. A parallel reverse transcription reaction per-        equal volume STE buffer. The enriched mitochondria <br /> formed RNA omitting reverse           lysed 1 mL lysis buffer (10 mM Tris pH 7.2, <br /> transcriptase control genomic DNA contamination. Both        10 mM MgCl2, 100 mM KCl, 1 mM DTT, 1 mg/mL pepstatin, <br /> cDNA reactions diluted 1:10 H2O prior qPCR.                2 mg/mL leupeptin, 1 mM pefabloc) containing 1% Triton X-100, <br />    The qPCR reactions performed using Rotor-Gene 3000        15 min 4&#194;&#176;C. The lysate cleared centrifugation and <br /> Real-Time DNA Detection System thermocycler, data, includ-        loaded 10%&#226;&#8364;&#8220;30% glycerol gradient containing same <br /> ing PCR efficiencies, acquired using Rotor-Gene v6.1         concentration DTT protease inhibitors lysis buffer. <br /> software (Corbett Life Science). The primer pairs used detect      The gradient spun 38,000 rpm SW 40 Ti rotor <br /> mt transcripts described previously (Carnes et al. 2005),     (Beckman) 5 h 4&#194;&#176;C, 500-mL fractions subsequently <br /> are amplifying 18S rRNA b-tubulin cDNAs,         collected top. In experiments testing RNase sensitivity of <br /> used reference genes. Primer pairs detect 9S 12S        TbRGG1 interaction macromolecular complexes, 0.1 mg/mL <br /> mitochondrial rRNAs TbRGG1 transcripts designed          RNase A introduced lysis buffer glycerol <br /> this study follows:                                        gradient, spun using previously mentioned rotor at <br />  <br />  <br /> 978      RNA, Vol. 14, No. 5 <br />                                                                                          TbRGG1 component novel complex <br />  <br />  <br />  <br /> 38,000 rpm 12 h 4&#194;&#176;C 0.1 mg/mL RNase A added          SUPPLEMENTAL DATA <br /> during IgG affinity step, TEV eluates loaded glycerol <br /> gradients spun 38,000 rpm 5 h 4&#194;&#176;C.                      Supplemental material http://www.rnajournal.org. <br />  <br />  <br /> TAP-purification TbRGG1                                            ACKNOWLEDGMENTS <br /> Either cell pellet hypotonically isolated mitochondria   We thank Zden&#203;&#8225;ka C&#203;&#8225;&#196;&#177;&#194;&#180;c&#203;&#8225;ova&#194;&#180; (Biology Centre) help ex- <br /> (scaled protocol described above) 4 3 1010 cells     periments Yuko Ogata (Seattle Biomedical Research Institute) <br /> containing TbRGG1 gene tandem affinity purification        help mass spectrometry analysis. Paul A.M. Michels <br /> (TAP) tag, expression induced 500 ng/mL tetracy-         (De Duve Institute Cellular Pathology, Brussels) Luc <br /> cline, lysed ice 20 min 20 mL IPP150 buffer     Vanhamme (Free University Brussels, Gosselies) kindly provided <br /> (10 mM Tris-HCl pH 8.0, 150 mM NaCl, 0.1% NP40) con-               antibodies. This work supported Grant Agency the <br /> taining 1% Triton X-100 tablets Complete EDTA-free         Czech Republic 204/06/1558, Grant Agency Czech Academy <br /> protease inhibitors (Roche). After centrifugation 9000g 30     Sciences A500960705, Ministry Education the <br /> min, cleared supernatant incubated 200 mL         Czech Republic (LC07032, 2B06129, 6007665801) J.L., the <br /> IgG Sepharose Fast Flow beads (Pharmacia) 2 h 4&#194;&#176;C. After       Ph.D. student grant 524/03/H133 H.H., National Institutes <br /> draining supernatant wash IPP150, beads               Health Grants 5R03TW6445 (to K.D.S. J.L.) AI14102 <br /> incubated 100 U TEV protease (Invitrogen) 1 mL          (to K.S.). <br /> the TEVCB buffer (10 mM Tris-HCl pH 8.0, 150 mM NaCl, <br /> 0.1% NP40, 0.5 mM EDTA, 1 mM DTT) 2 h 16&#194;&#176;C. The TEV            Received October 23, 2007; accepted February 15, 2008. <br /> eluate combined times volume calmodulin <br /> resin (Stratagene) CBB buffer (10 mM Tris-Hcl pH 8.0, <br /> 150 mM NaCl, 0.1% NP40, 10 mM b-mercaptoethanol [ME], 1               REFERENCES <br /> mM Mg acetate, 1 mM imidazole, 2 mM CaCl2), incu- <br />                                                                       Aphasizhev, R. 2007. RNA editing. Mol. Biol. 41: 227&#226;&#8364;&#8220;239. <br /> bated 1 h 4&#194;&#176;C. The tagged complexes eluted 4 3       Aphasizhev, R., Aphasizheva, I., Nelson, R.E., Simpson, L. 2003a. <br /> 250 mL CEB buffer (10 mM Tris-Hcl pH 8.0, 150 mM               A 100-kDa complex RNA-binding proteins mito- <br /> NaCl, 0.1% NP40, 10 mM ME, 1 mM Mg acetate, 1 mM imid-                   chondria Leishmania tarentolae catalyzes RNA annealing <br /> azole, 2 mM EGTA). The protocol used purify                  interacts RNA editing components. RNA 9: 62&#226;&#8364;&#8220; <br /> proteins associated TAP-tagged Tb927.2.3800 Tb927.7.            76. <br /> 2570 proteins corresponding cell lines induced tetracy-       Aphasizhev, R., Aphasizheva, I., Nelson, R.E., Gao, G., Simpson, A.M., <br />                                                                          Kang, X., Falick, A.M., Sbicego, S., Simpson, L. 2003b. <br /> cline. TAP purification Tb927.2.3800 performed parallel in <br />                                                                          Isolation U-insertion/deletion editing complex Leish- <br /> the presence absence 0.1 mg/mL RNase A initial         mania tarentolae mitochondria. EMBO J. 22: 913&#226;&#8364;&#8220;924. <br /> incubation cleared lysate IgG Sepharose beads. The    Blum, B. Simpson, L. 1992. Formation guide RNA messenger- <br /> proteins isolated complexes separated 10.5%&#226;&#8364;&#8220;14%             RNA chimeric molecules vitro, initial step RNA editing, is <br /> SDS-PAGE gel visualized SYPRO Ruby (Invitrogen)                   dependent anchor sequence. Proc. Natl. Acad. Sci. 89: <br /> staining.                                                                11944&#226;&#8364;&#8220;11948. <br />                                                                       Burd, C.G. Dreyfuss, G. 1994. Conserved structures diversity <br />                                                                          functions RNA-binding proteins. Science 265: 615&#226;&#8364;&#8220;621. <br /> Mass spectrometry analysis                                            Carnes, J., Trotter, J.R., Ernst, N.L., Steinberg, A.G., Stuart, K. <br />                                                                          2005. An essential RNase III insertion editing endonuclease in <br /> Proteins analyzed mass spectrometry described                 Trypanosoma brucei. Proc. Natl. Acad. Sci. 102: 16614&#226;&#8364;&#8220;16619. <br /> (Panigrahi et al. 2001a). Briefly, proteins 100 mL TAP      Golden, D.E. Hajduk, S.L. 2005. The 39-untranslated region <br /> eluate precipitated 600 mL acetone, resuspended          cytochrome oxidase II mRNA functions RNA editing of <br /> 8 M urea/1 mM DTT, denatured 50&#194;&#176;C 1 h The sample              African trypanosomes exclusively cis guide RNA. RNA 11: 29&#226;&#8364;&#8220; <br />                                                                          37. <br /> was diluted 1:7 50 mM NH4-HCO3 treated 100 ng <br />                                                                       Goulah, C.C., Pelletier, M., Read, L.K. 2006. Arginine methylation <br /> of mass spectrometry grade trypsin (Promega) 37&#194;&#176;C overnight.          regulates mitochondrial gene expression Trypanosoma brucei <br /> The mixture dried speedvac dissolved 5%                  multiple effector proteins. RNA 12: 1545&#226;&#8364;&#8220;1555. <br /> acetonitrile/0.4% acetic acid. The resulting peptides purified   Hans, J., Hajduk, S.L., Madison-Antenucci, S. 2007. RNA-editing- <br /> using C18 ZipTip (Millipore) following manufacturer&#226;&#8364;&#8482;s              associated protein 1 null mutant reveals link mitochondrial <br /> protocol. The peptides analyzed using LTQ Linear Ion Trap         RNA stability. RNA 13: 881&#226;&#8364;&#8220;889. <br /> Mass Spectrometer. The CID spectra compared T.          Jensen, B.C., Kifer, C.T., Brekken, D.L., Randall, A.C., Wang, Q., <br />                                                                          Drees, B.L., Parsons, M. 2006. Characterization protein <br /> brucei protein database downloaded GeneDB using Turbo-              kinase CK2 Trypanosoma brucei. Mol. Biochem. Parasitol. <br /> Sequest software, protein matches determined using              151: 28&#226;&#8364;&#8220;40. <br /> PeptideProphet ProteinProphet programs (Keller et al. 2002;       Keller, A., Nesvizhskii, A.I., Kolker, E., Aebersold, R. 2002. <br /> Nesvizhskii et al. 2003).                                                Empirical statistical model estimate accuracy peptide <br />                                                                          identifications MS/MS database search. Anal. Chem. <br />                                                                          74: 5383&#226;&#8364;&#8220;5392. <br /> Sequence analysis                                                     Kim, K.S., Teixeira, S.M., Kirchhoff, L.W., Donelson, J.E. 1994. <br />                                                                          Transcription editing cytochrome oxidase II RNAs in <br /> The protein sequences compared NCBI nr database            Trypanosoma cruzi. J. Biol. Chem. 269: 1206&#226;&#8364;&#8220;1211. <br /> using PSI-BLAST motif, domain searches carried           Lambert, L., Muller, U.F., Souza, A.E., Go&#194;&#168;ringer, H.U. 1999. The <br /> out CDD, PFAM, Prosite searches.                                  involvement gRNA-binding protein gBP21 RNA editing-an <br />  <br />  <br />                                                                                                             www.rnajournal.org          979 <br />  Hashimi et al. <br />  <br />  <br />  <br />    vitro vivo analysis. Nucleic Acids Res. 27: 1429&#226;&#8364;&#8220;                  oxidoreductase complex. Mol. Cell. Proteom 7: 534&#226;&#8364;&#8220;545. doi: <br />    1436.                                                                        10.1074/mcp.M700430-MCP200. <br /> Leegwater, P., Speijer, D., Benne, R. 1995. Identification UV        Pelletier, M. Read, L.K. 2003. RBP16 multifunctional gene <br />    cross-linking oligo(U)-binding proteins mitochondria            regulatory protein involved editing stabilization specific <br />    insect trypanosomatid Crithidia fasciculata. Eur. J. Biochem. 227:           mitochondrial mRNAs Trypanosoma brucei. RNA 9: 457&#226;&#8364;&#8220;468. <br />    780&#226;&#8364;&#8220;786.                                                                 Pfaffl, M.W. 2001. A new mathematical model relative quantifi- <br /> Liu, B., Molina, H., Kalume, D., Pandey, A., Griffith, J.D.,                cation real-time RT-PCR. Nucleic Acids Res. 29: e45. doi: <br />    Englund, P.T. 2006. Role p38 replication Trypanosoma                10.1093/nar/29.9.e45. <br />    brucei kinetoplast DNA. Mol. Cell. Biol. 26: 5382&#226;&#8364;&#8220;5393.                  Sbicego, S., Alfonzo, J.D., Estevez, A.M., Rubio, M.A., Kang, X., <br /> Lukes&#203;&#8225;, J., Hashimi, H., Z&#196;&#177;&#194;&#180;kova&#194;&#180;, A. 2005. Unexplained complexity          Turck, C.W., Peris, M., Simpson, L. 2003. RBP38, novel <br />    mitochondrial genome transcriptome kinetoplastid               RNA-binding protein trypanosomatid mitochondria, mod- <br />    flagellates. Curr. Genet. 48: 277&#226;&#8364;&#8220;299.                                       ulates RNA stability. Eukaryot. Cell 2: 560&#226;&#8364;&#8220;568. <br /> Madison-Antenucci, S., Sabatini, R.S., Pollard, V.W., Hajduk, S.L.      Schumacher, M.A., Karamooz, E., Z&#196;&#177;&#194;&#180;kova&#194;&#180;, A., Trant&#196;&#177;&#194;&#180;rek, L., and <br />    1998. Kinetoplastid RNA-editing-associated protein 1 (REAP-1):               Lukes&#203;&#8225;, J. 2006. Crystal structures Trypanosoma brucei MRP1/ <br />    A novel editing complex protein repetitive domains. EMBO J.             MRP2 guide-RNA-binding complex reveals RNA matchmaking <br />    17: 6368&#226;&#8364;&#8220;6376.                                                               mechanism. Cell 126: 701&#226;&#8364;&#8220;711. <br /> Miller, M.M., Halbig, K., Cruz-Reyes, J., Read, L.K. 2006. RBP16        Simpson, L., Aphasizhev, R., Gao, G., Kang, X. 2004. Mitochon- <br />    stimulates trypanosome RNA editing vitro early step          drial proteins complexes Leishmania Trypanosoma <br />    editing reaction. RNA 12: 1292&#226;&#8364;&#8220;1303.                                         involved U-insertion/deletion RNA editing. RNA 10: 159&#226;&#8364;&#8220;170. <br /> Nesvizhskii, A.I., Keller, A., Kolker, E., Aebersold, A. 2003. A        Stuart, K. Panigrahi, A.K. 2002. RNA editing: Complexity and <br />    statistical model identifying proteins tandem mass spec-              complications. Mol. Microbiol. 45: 591&#226;&#8364;&#8220;596. <br />    trometry. Anal. Chem. 75: 4646&#226;&#8364;&#8220;4658.                                     Stuart, K.D., Schnaufer, A., Ernst, N.L., Panigrahi, A.K. 2005. <br /> Panigrahi, A.K., Gygi, S., Ernst, N.L., Igo Jr., R.P., Palazzo, S.S.,           Complex management: RNA editing trypanosomes. Trends <br />    Schnaufer, A., Weston, D., Carmean, N., Salavati, R.,                        Biochem. Sci. 30: 97&#226;&#8364;&#8220;105. <br />    Aebersold, R., et al. 2001a. Association novel proteins,          Trotter, J.R., Ernst, N.L., Carnes, J., Panicucci, B., Stuart, K. 2005. <br />    TbMP52 TbMP48, Trypanosoma brucei RNA editing                   A deletion site editing endonuclease Trypanosoma brucei. Mol. <br />    complex. Mol. Cell. Biol. 21: 380&#226;&#8364;&#8220;389.                                       Cell 20: 403&#226;&#8364;&#8220;412. <br /> Panigrahi, A.K., Schnaufer, A., Carmean, N., Igo Jr., R.P., Gygi, S.P.,     Vanhamme, L., Perez-Morga, D., Marchal, C., Speijer, D., Lambert, L., <br />    Ernst, N.L., Palazzo, S.S., Weston, D.S., Aebersold, R., Salavati, R.,       Geuskens, M., Alexandre, S., Isma&#196;&#177;&#194;&#168;li, N., Go&#194;&#168;ringer, U.H., <br />    et al. 2001b. Four related proteins Trypanosoma brucei RNA            Benne, R., et al. 1998. Trypanosoma brucei TBRGG1, mitochon- <br />    editing complex. Mol. Cell. Biol. 21: 6833&#226;&#8364;&#8220;6840.                             drial oligo(U)-binding protein co-localizes vitro <br /> Panigrahi, A.K., Allen, T.E., Haynes, P.A., Gygi, S.P., Stuart, K.          RNA editing activity. J. Biol. Chem. 273: 21825&#226;&#8364;&#8220;21833. <br />    2003a. Mass spectrometric analysis editosome            Vondrus&#203;&#8225;kova&#194;&#180;, E., van den Burg, J., Z&#196;&#177;&#194;&#180;kova&#194;&#180;, A., Ernst, N.L., Stuart, K., <br />    multiprotein complexes Trypanosoma brucei. J. Am. Soc. Mass               Benne, R., Lukes&#203;&#8225;, J. 2005. RNA interference analyses suggest a <br />    Spectrom. 14: 728&#226;&#8364;&#8220;735.                                                       transcript-specific regulatory role mitochondrial RNA-binding <br /> Panigrahi, A.K., Schnaufer, A., Ernst, N.L., Wang, B., Carmean, N.,             proteins MRP1 MRP2 RNA editing RNA <br />    Salavati, R., Stuart, K. 2003b. Identification novel                  processing Trypanosoma brucei. J. Biol. Chem. 280: 2429&#226;&#8364;&#8220;2438. <br />    components Trypanosoma brucei editosomes. RNA 9: 484&#226;&#8364;&#8220;                 Wang, B.B., Ernst, N.L., Palazzo, S.S., Panigrahi, A.K., Salavati, R., and <br />    492.          <br /> </body></html>