<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization of the binding surface of the <br /> translocated intimin receptor, an essential protein for <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> The translocated intimin receptor (TIR) of enteropathogenic and enterohemorrhagic Escherichia coli <br /> (EPEC and EHEC) is required for EPEC and EHEC infections, which cause widespread illness across <br /> the globe. TIR is translocated via a type-III secretion system into the intestinal epithelial cell membrane, <br /> where it serves as an anchor for E. coli attachment via its binding partner intimin. While many aspects of <br /> EPEC and EHEC infection are now well understood, the importance of the intermolecular contacts made <br /> between intimin and TIR have not been thoroughly investigated. Herein we report site-directed <br /> mutagenesis studies on the intimin-binding domain of EPEC TIR, and how these mutations affect TIR- <br /> intimin association, as analyzed by isothermal titration calorimetry and circular dichroism. These results <br /> show how two factors govern TIR&#226;&#8364;&#8482;s binding <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> intimin: A three-residue TIR hot spot is identified that <br /> largely mediates the interaction, and mutants that alter the b-hairpin structure of TIR severely diminish <br /> binding affinity. In addition, peptides incorporating key TIR residues identified by mutagenesis are <br /> incapable of binding intimin. These results indicate that hot spot residues and structural orientation/ <br /> preorganization are required for EPEC, and likely EHEC, TIR-intimin binding. <br /> Keywords: EPEC; EHEC; intimin; translocated intimin receptor (TIR) <br /> Supplemental material: see www.proteinscience.org <br />  <br />  <br /> Intimin and the translocated intimin receptor (TIR) of                         Knutton et al. 1997). TIR, upon expression, is localized to <br /> enteropathogenic and enterohemorrhagic Escherichia coli                        the E. coli cell membrane and then translocated through <br /> (EPEC and EHEC) play an integral role in attachment to                         the type-III secretion needle complex (Kenny et al. 1997; <br /> intestinal epithelial cells (Marches et al. 2000; Vallance                     Gauthier and Finlay 2003) into the host cell. TIR then <br /> and Finlay 2000). These two proteins are encoded in the                        becomes immobilized in the intestinal epithelium with its <br /> locus of enterocyte effacement and are expressed (TIR) or                      intimin binding domain (IBD) displayed on the intestinal <br /> up-regulated (intimin) when EPEC/EHEC cells come into                          cell surface. Intimin, a transmembrane protein whose <br /> close contact with a host cell (McDaniel et al. 1995;                          extracellular domain (ECD) is displayed on the E. coli <br />                                                                                surface, binds TIR-IBD, intimately attaching the E. coli <br />                                                                                cell to the intestinal epithelium (Kenny et al. 1997). This <br />    Reprint requests to: Benjamin Miller, University of Rochester,              attachment ultimately leads to actin recruitment within <br /> Department of Dermatology, 601 Elmwood Avenue, Box 697, Roches-                the epithelial cell and the formation of attaching and <br /> ter, NY 14642, USA; e-mail: Benjamin_Miller@urmc.rochester.edu;                effacing lesions (Knutton et al. 1987). Lesion formation is <br /> fax: (585) 273-1346. <br />    Article and publication are at http://www.proteinscience.org/cgi/doi/       the precursor of intestinal cell membrane disruption and <br /> 10.1110/ps.073128607.                                                          diarrhea in infected individuals (Knutton et al. 1987, <br />  <br />  <br /> Protein Science (2007), 16:2677&#226;&#8364;&#8220;2683. Published by Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2007 The Protein Society         2677 <br />  Ross and Miller <br />  <br />  <br />  <br /> 1997). In severe cases, infection leads to hemolytic                         strate that short peptides derived <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the intimin- <br /> uremic syndrome, a disease that kills 12% of those                           binding region of TIR-IBD do not have the ability to bind <br /> infected (Gerber et al. 2002; Garg et al. 2003).                             intimin-ECD. These peptides support our hypothesis that <br />    The structure of intimin-ECD has been solved by NMR                       sequential, unstructured mimics alone are not sufficient <br /> (Batchelor et al. 2000) and X-ray crystallography (Luo                       to bind intimin-ECD, and that additional structural re- <br /> et al. 2000). In addition, the cocrystal structure of                        quirements are necessary to modulate this protein&#226;&#8364;&#8220;protein <br /> intimin-ECD bound to TIR-IBD has also been determined                        interaction. Due to the 85% sequence identity between <br /> (Fig. 1; Luo et al. 2000). These structures have had a                       EPEC and EHEC TIR within the b-hairpin region <br /> great impact on our understanding of these two proteins,                     (DeVinney et al. 1999), these results could apply to EHEC <br /> including the elucidation of their binding interface. How-                   TIR as well. <br /> ever, very limited additional work has been conducted <br /> to determine which specific residues of these two pro- <br />                                                                              Results <br /> teins are most responsible for their interaction. To our <br /> knowledge, the only studies along these lines have been                      Alanine point mutants were generated for all nonalanine <br /> reported by the Leong group, who carried out random                          residues in the b-hairpin of TIR-IBD whose side chains <br /> mutagenesis studies on the ECD of intimin (Liu et al.                        were within 5 A  &#203;&#353; of intimin, as determined based on an <br /> 1999, 2002). However, because very few mutants were                          inspection of the Luo et al. (2000) X-ray crystal structure <br /> generated at the binding surface, the vast majority of this                  (Protein Data Bank [PDB] <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1F02). These mutants <br /> protein&#226;&#8364;&#8220;protein interface has, until now, remained                           were then screened for their ability to bind intimin-ECD <br /> unstudied.                                                                   by ITC. The results of this thermodynamic analysis of <br />    In this article, we report the systematic <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> of all                 11 mutants are shown in Tables 1 and 2 (results are the <br /> of the nonalanine residues from the b-hairpin of EPEC                        average 6 SD of at least three independent titrations <br /> TIR-IBD that lie in close proximity to intimin, thereby                      for each TIR-IBD mutant). Figure 2A depicts the location <br /> providing the most complete understanding of the TIR-                        of each of these residues in the b-hairpin of TIR. The <br /> intimin binding interface to date. The ability of each of                    side chains are color-coded according to their respective <br /> these point mutants to bind to intimin-ECD has been                          DDG values for intimin-ECD binding relative to wild- <br /> characterized by isothermal titration calorimetry (ITC)                      type TIR-IBD. <br /> and their secondary structures examined by circular                             Two of the alanine point mutants, V299A and N304A, <br /> dichroism (CD). From this study of 11 point mutants, we                      had no significant effect on TIR-IBD/intimin-ECD bind- <br /> have identified a three-residue TIR-IBD binding hot spot.                    ing (DDG &#194;&#188;  0.5 to +0.5 kcal/mol). Mutation of I301 <br /> Several other mutants implicate key structural require-                      or N306 to alanine had a modest effect (DDG &#194;&#188; +0.5 <br /> ments for TIR-intimin association. Finally, we demon-                        to +1.0 kcal/mol), while D302A caused a more substantial <br />                                                                              change in affinity (DDG &#194;&#188; 1.37 kcal/mol). Four alanine <br />                                                                              mutants (G305, E312, K298, and N300) had a large <br />                                                                              impact on the binding interaction, with DDG values be- <br />                                                                              tween 1.75 and 3.00 kcal/mol. <br />                                                                                 The secondary structure of each of the mutants was <br />                                                                              examined using CD spectroscopy to ensure that the <br />                                                                              changes in binding affinity observed by ITC were due <br />                                                                              to side-chain interactions themselves, and not changes in <br />                                                                              the structure of the protein. As can been seen in Figure <br />                                                                              2B, most of the mutants appear to have similar CD <br />                                                                              spectra to that of wild-type TIR-IBD, with the exception <br />                                                                              of G305A and K298A. These two mutants, which were <br />                                                                              designed to probe the structural importance of TIR-IBD&#226;&#8364;&#8482;s <br />                                                                              b-hairpin, altered the overall structure of TIR-IBD as <br />                                                                              expected. The loss of binding affinity upon mutation of <br />                                                                              G305 or K298 likely reflects the significance of both the <br />                                                                              b-hairpin&#226;&#8364;&#8482;s conformation, and its positioning relative to <br />                                                                              the rest of TIR-IBD. <br />                                                                                 Mutation of asparagine-300 to alanine was found to <br /> Figure 1. (A) Depiction of intimin-ECD (surface) and TIR-IBD (wire) in <br /> their cocrystallized form. (B) Close-up representation of the intimin-ECD/   have the most significant impact on TIR-IBD&#226;&#8364;&#8482;s ability <br /> TIR-IBD interface showing the b-hairpin of TIR-IBD examined in this          to bind to intimin-ECD without altering the global fold <br /> study.                                                                       (DDG &#194;&#188; 2.81 kcal/mol). In an effort to further probe this <br />  <br />  <br /> 2678     Protein Science, vol. 16 <br />                                                                                                                             The binding hot spot of TIR <br />  <br />  <br /> Table 1. Analysis of native and mutant TIR-IBD proteins by ITC <br /> Mutant              Kd (nM)              DG (kcal/mol)           DH (kcal/mol)              DS (cal/mol*K)              TDS (kcal/mol)           N-value <br />       a <br /> Native             577   6   90           8.50   6   0.07        21.80   6   0.74            44.66   6   2.30            13.43   6   0.70      0.75 6 0.12 <br /> E312A           21,586   6   5486         6.39   6   0.15        29.40   6   7.12            77.22   6   24.38           23.01   6   7.27      1.20 6 0.19 <br /> N306A            1,667   6   83           7.89   6   0.03        27.46   6   0.77            65.67   6   2.66            19.57   6   0.79      0.87 6 0.01 <br /> G305A           13,167   6   1303         6.68   6   0.06        36.61   6   4.56           100.44   6   15.39           29.93   6   4.59      0.93 6 0.15 <br /> N304A              390   6   23           8.75   6   0.04        23.08   6   0.42            48.10   6   1.51            14.33   6   0.45      0.93 6 0.02 <br /> D302A            5,760   6   194          7.16   6   0.02        27.19   6   1.19            67.22   6   4.04            20.03   6   1.21      1.04 6 0.05 <br /> I301A            2,189   6   142          7.74   6   0.04        30.56   6   0.73            76.59   6   2.56            22.82   6   0.76      0.98 6 0.03 <br /> N300A           69,881   6   17354        5.69   6   0.14        23.66   6   3.03            60.31   6   10.59           17.97   6   3.16          1.00 <br /> N300D            5,528   6   312          7.18   6   0.03        29.06   6   1.95            73.43   6   6.61            21.88   6   1.97      0.77 6 0.06 <br /> N300Q           20,204   6   878          6.41   6   0.03        19.67   6   0.43            44.50   6   1.53            13.26   6   0.46          1.00 <br /> V299A              291   6   46           8.93   6   0.10        28.73   6   0.68            66.43   6   2.58            19.80   6   0.77      0.94 6 0.02 <br /> K298A           75,114   6   4742         5.64   6   0.04        32.35   6   1.60            89.62   6   5.51            26.71   6   1.64          1.00 <br />  <br /> The thermodynamic parameters governing TIR-IBD/intimin-ECD binding are shown with standard deviations from average values. Each data set represents <br /> dilution controlled experimental results, which were repeated in triplicate. For N300A, N300Q, and K298A, it was necessary to set the N-value at 1.00 in <br /> order for the curve fitting process to converge. <br /> a <br />   Native TIR-IBD/intimin-ECD titrations are the average 6 SD of seven independent titrations (Ross et al. 2007). <br />  <br />  <br />  <br /> interaction, two additional mutations were made. N300D                         asparagine-304 and valine-299, were found to have no <br /> and N300Q both showed a significant decrease in binding                        significant effect on TIR-IBD&#226;&#8364;&#8482;s ability to bind intimin- <br /> free energy (DDG &#194;&#188; 1.32 and 2.09 kcal/mol, respectively)                       ECD. N304 points directly toward bulk solvent, and away <br /> compared with wild-type TIR-IBD, further supporting                            from intimin, in the cocrystal structure of these two <br /> the pivotal role of asparagine-300 in this interaction                         proteins. Accordingly, it was not surprising that mutation <br /> (Tables 1, 2).                                                                 of this residue did not have a pronounced effect on <br />    In order to further examine the importance of confor-                       binding. Valine-299 was slightly more surprising, because <br /> mational constraint on TIR-IBD&#226;&#8364;&#8482;s ability to bind intimin-                      its side chain lies directly on the surface of intimin. How- <br /> ECD, we synthesized and tested three short peptides.                           ever, because it does not fit into a pocket or other surface <br /> These peptides (INV, AINV, and INVK) were designed                             feature, replacing this hydrophobic residue with another <br /> to represent potential minimal binding domains of TIR,                         hydrophobic residue, albeit one with a smaller side chain, <br /> incorporating key residues identified by mutagenesis                           did not reduce binding. <br /> while minimizing total <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> in an effort to reduce the                          Two mutants, N306A and I301A, have only a modestly <br /> entropic penalty for binding. None showed any measur-                          decreased binding free energy (DDG &#194;&#188; +0.5 to +1.0 kcal/mol) <br /> able affinity for intimin-ECD, as determined by ITC (data                      relative to wild-type TIR-IBD. Asparagine-306 is within <br /> not shown).                                                                    hydrogen-bonding distance of lysine-919 of intimin, and <br />  <br />  <br /> Discussion <br /> The binding interface of TIR and intimin is relatively                         Table 2. The effect of mutating a given residue is compared to <br /> small, burying 1335 A  &#203;&#353; 2 of solvent-exposed surface area                     wild-type TIR-IBD/intimin-ECD binding <br /> (Luo et al. 2000; Headd et al. 2007). It is therefore not                                        DDG                DDH              DDS          TDDS <br /> surprising that most of the residues of TIR located near                       Mutant         (kcal/mol)         (kcal/mol)      (cal/mol*K)    (kcal/mol) <br /> the interface have some impact on intimin binding.                             Nativea           0.00               0.00               0.00         0.00 <br /> However, this study implicates three key residues, aspar-                      E312A             2.11               7.60              32.56         9.58 <br /> agine-300, isoleucine-301, and glutamic acid-312, as the                       N306A             0.61               5.66              21.01         6.14 <br /> binding hot spot (Clackson and Wells 1995; Jones and                           G305A             1.82              14.81              55.78        16.50 <br /> Thornton 1997) of this protein&#226;&#8364;&#8220;protein interaction inter-                      N304A             0.25               1.28               3.44         0.90 <br />                                                                                D302A             1.37               5.39              22.56         6.60 <br /> face. Additionally, more conservative mutations of N300                        I301A             0.76               8.76              31.93         9.39 <br /> allowed for the observation that the ability of this                           N300A             2.81               1.86              15.65         4.54 <br /> asparagine to form two intermolecular bonds, and their                         N300D             1.32               7.26              28.77         8.45 <br /> placement, affects binding to intimin-ECD.                                     N300Q             2.09               2.13               0.16         0.17 <br />    Mutagenesis also revealed which residues contribute to                      V299A             0.43               6.93              21.77         6.37 <br />                                                                                K298A             2.86              10.55              44.96        13.28 <br /> the binding interaction to a lesser extent, and some that do <br />                                                                                a <br /> not appear to participate in binding at all. Two residues,                         Native TIR-IBD/intimin-ECD values are from Ross et al. (2007). <br />  <br />  <br />                                                                                                                         www.proteinscience.org      2679 <br />  Ross and Miller <br />  <br />  <br />  <br />                                                                                the removal of the entropic penalty required for freezing <br />                                                                                out I301 side-chain rotation (Harpaz et al. 1994; Stites <br />                                                                                1997; Lee et al. 2000). <br />                                                                                   Mutation of D302 to alanine produces a DDG of <br />                                                                                binding of 1.37 kcal/mol. This was initially somewhat <br />                                                                                puzzling, because D302 does not appear to have any <br />                                                                                direct interactions with intimin. The side-chain oxygen <br />                                                                                does appear to make a cross-strand hydrogen bond to <br />                                                                                the backbone amide of TIR N306. However, mutation of <br />                                                                                this residue to alanine did not alter the CD spectrum of <br />                                                                                TIR-IBD. Aspartic acid cross-strand hydrogen bonds are <br />                                                                                well-documented, stabilizing factors in type-II b-turns <br />                                                                                (Hutchinson and Thornton 1994); thus, mutation of this <br />                                                                                residue likely caused small changes in the orientation or <br />                                                                                conformational dynamics of the b-hairpin (accounting for <br />                                                                                the change in binding free energy), without altering <br />                                                                                global fold of the protein. <br />                                                                                   Four amino acids have a greater than 1.5 kcal/mol <br />                                                                                effect on DG when mutated to alanine. The first, lysine- <br />                                                                                298, participates in both an intramolecular salt bridge <br />                                                                                with TIR aspartic acid-315, and also buries its methylene <br />                                                                                groups on the surface of intimin (Fig. 2A). Mutation of <br />                                                                                K298 to alanine resulted in a DDG of 2.86 kcal/mol. The <br /> Figure 2. (A) TIR-IBD residues studied by alanine scanning mutagenesis         salt bridge made by K298 to D315 is located exactly at <br /> are shown in colors corresponding to their effect on binding free energy <br /> change, DDG (green indicates  0.50 to +0.50 kcal/mol; yellow, +0.50 to <br />                                                                                the boarder of the b-hairpin and a-helix of TIR-IBD. This <br /> +1.0 kcal/mol; orange, +1.0 to +1.75 kcal/mol; red, +1.75 to +3.0 kcal/        bridge may serve to maintain the angle and positioning of <br /> mol). Alanine residues that are part of the b-hairpin but were not mutated     the b-hairpin of TIR-IBD with respect to the rest of the <br /> in this study, are colored gray. (B) Circular dichroism results from each of   protein. In fact, when this bond is broken (by mutation of <br /> the mutants studied. The data are normalized for concentration and are         lysine to alanine), random coil features become a more <br /> buffer subtracted. While the majority of the mutations show the same <br /> global fold as native TIR-IBD, K298A and G305A both show a greater <br />                                                                                predominant component of the secondary structure of <br /> contribution to their overall fold from random coil.                           the protein (Fig. 2B). Additionally, the K298A mutation <br />                                                                                causes a very large decrease in TDDS ( 13.28 kcal/mol), <br />                                                                                consistent with a greater binding-induced loss of con- <br /> the absence of this H-bonding capability could account                         formational freedom for the K298A mutant. Accordingly, <br /> for the 0.61 kcal/mol loss in DG observed in N306A.                            alteration of the structure or alignment of the b-hairpin of <br /> Many investigators have estimated the energetic value of a                     TIR is likely responsible for the majority of this loss in <br /> hydrogen bond in protein association, and it is now                            binding affinity. <br /> generally accepted that a range of 0.6&#226;&#8364;&#8220;1.0 kcal/mol                               The second mutation that had a very pronounced effect <br /> (DG) is a reasonable approximation (Myers and Pace                             on TIR-IBD/intimin-ECD binding was glycine-305. While <br /> 1996; Klostermeier and Millar 2002). I301A has a DDG                           it is unusual to mutate a glycine residue when conducting <br /> of 0.76 kcal/mol. Isoleucine-301 buries its side chain in a                    alanine scanning mutagenesis, we were interested in deter- <br /> pocket on the surface of intimin (Fig. 2A), potentially                        mining whether converting G305 to alanine would disrupt <br /> serving to &#226;&#8364;&#732;&#226;&#8364;&#732;set the register&#226;&#8364;&#8482;&#226;&#8364;&#8482; for the binding interaction via                the type-II b-hairpin, thereby serving as an indicator of <br /> this hydrophobic burial (Chothia and Janin 1975; Tsai                          its structural importance. Indeed, mutation to alanine at <br /> et al. 1997). The low temperature factors for this residue                     position 305 caused a loss of binding affinity (DDG) of 1.82 <br /> (Supplemental Fig. S2) also suggest that I301 is critical                      kcal/mol and also altered the overall fold of the protein <br /> for TIR-intimin interaction. Why, then, does mutating the                      (Fig. 2B). This structural mutation also caused a very large <br /> residue to alanine only result in a DDG of 0.76 kcal/mol?                      entropic penalty for binding (TDDS &#194;&#188;  16.50 kcal/mol). <br /> It is possible that the alanine methyl group is able to                        Taken together, the K298A and the G305A mutants <br /> provide some of the registration function of the isoleucine                    indicate that both the fold and orientation of the b-hairpin <br /> side chain in the native structure. Additionally, we can                       of TIR-IBD are essential for submicromolar binding to <br /> speculate that the less favorable binding entropy for                          intimin-ECD. <br /> I301A binding to intimin is largely due to differences in                         While K298A and G305A have significant effects on <br /> solvent release, although this is partially compensated by                     binding by altering the structure of TIR-IBD, two other <br />  <br />  <br /> 2680      Protein Science, vol. 16 <br />                                                                                                              The binding hot spot of TIR <br />  <br />  <br />  <br /> residues also exhibit large effects on binding without                      loss in affinity displayed by the N300A mutant, but large <br /> disrupting the fold of the protein. Glutamic acid-312                       nevertheless. Second, asparagine-300 was changed to <br /> appears to make a direct salt bridge to lysine-927 of                       an aspartic acid. We anticipated that this would have a <br /> intimin and also forms part of a pocket on the surface                      smaller effect on DG than the N300A mutation, since only <br /> of TIR that is occupied by intimin K927. Thus, it is                        one direct H-bond would be lost. This was the case, <br /> unsurprising that the E312A mutant has a destabilizing                      with a DDG of 1.32 kcal/mol compared with wild-type <br /> effect of 2.11 kcal/mol. It is also noteworthy that E312 is                 TIR-IBD. Interestingly, this change in binding free <br /> the key residue differentiating EPEC TIR-IBD from                           energy more closely resembles what would be expected <br /> EHEC TIR-IBD (DeVinney et al. 1999; Luo et al. 2000).                       if two H-bonds were broken, providing evidence that the <br /> In EHEC TIR-IBD, glutamic acid is replaced by valine at                     H-bond between N300 and intimin N932 may serve to <br /> position 312. It therefore makes sense that our E312A                       orient N932 in such a way that its side-chain oxygen can <br /> mutant roughly matches the affinity of EHEC TIR-IBD for                     H-bond with I301 (Fig. 3). <br /> EPEC intimin (DeVinney et al. 1999).                                           The importance of asparagine-300 is further reinforced <br />    Mutation of asparagine-300 to alanine had the                            when the temperature factors from the cocrystal structure <br /> most destructive effect on TIR-IBD binding affinity                         of intimin-ECD and TIR-IBD are examined (Supplemen- <br /> (DDG &#194;&#188; 2.81 kcal/mol), without affecting the overall                        tal Fig. S2). In comparison with the rest of TIR-IBD, <br /> fold of the protein. A detailed examination of the co-                      N300 atoms have very low B-values (Luo et al. 2000). It <br /> crystal structure revealed that the side chain of aspar-                    appears that asparagine-300 is part of a more complex <br /> agine-300 likely participates in two direct contacts with                   group of interactions between intimin-ECD and TIR-IBD, <br /> intimin-ECD (Fig. 3). The side-chain amino group ap-                        and that it may serve as a central anchoring point for TIR- <br /> pears to form a hydrogen bond with glutamic acid-930,                       IBD. This idea is further supported by a recent review of <br /> and the side-chain oxygen H-bonds to the side-chain                         protein&#226;&#8364;&#8220;protein interactions (Headd et al. 2007), which <br /> amino group of asparagine-932. This hydrogen bond                           found that a common theme in key contact areas is the <br /> to N932 may also orient the side chain of N932 in such                      presence of two to three backbone&#226;&#8364;&#8220;backbone H-bonds, as <br /> a way that its oxygen can accept a hydrogen bond                            may be the case in this region of TIR-IBD and intimin- <br /> from the backbone amide nitrogen of isoleucine-301 of                       ECD (Fig. 3). <br /> TIR-IBD.                                                                       These mutagenesis results can be summed up in an <br />    To further investigate N300, two additional mutations                    overall hypothesis as follows: The key hydrogen-bonding <br /> were made. First, asparagine-300 was mutated to gluta-                      interactions provided by asparagine-300, the lock and key <br /> mine in an effort to determine the importance of the                        type fit of isoleucine-301, and the salt bridge of glutamic <br /> precise placement of this side chain. If unimportant, this                  acid-312 of EPEC TIR serve as a hot spot for intimin- <br /> would argue against asparagine-300 being involved in a                      TIR binding. This is also consistent with the results of <br /> more highly coordinated network of hydrogen bonds                           Liu et al. (1999), who observed that asparagine-932 of <br /> between intimin and TIR. N300Q had a substantial effect                     EHEC intimin (which would interact with N300 of EPEC/ <br /> on binding (DDG &#194;&#188; 2.09 kcal/mol), slightly less than the                    EHEC TIR) is important for TIR-IBD binding. The <br />                                                                             complete lack of affinity demonstrated by synthetic <br />                                                                             peptides INV, AINV, and INVK show that the hot spot <br />                                                                             sequence alone is not enough to confer intimin binding <br />                                                                             ability, consistent with our results for the secondary <br />                                                                             structure-altering mutants K298A and G305A. These <br />                                                                             residues are important for preorganizing TIR-IBD into <br />                                                                             a binding-competent conformation. Glycine-305 and <br />                                                                             lysine-298 allow for the proper type-II b-turn, and its <br />                                                                             positioning, respectively. Due to the very high sequence <br />                                                                             identity between the b-hairpin of EPEC and EHEC TIR <br />                                                                             (85%), it seems likely that these same interactions (with <br />                                                                             the exception of E312 explained earlier) govern the <br />                                                                             binding of both proteins to their cognate receptors. This, <br /> Figure 3. Key bonds between N300 of TIR-IBD and intimin-ECD are             for the first time, provides a detailed explanation of how <br /> shown with black dashed lines. Proposed hydrogen bonds between the          this critical protein&#226;&#8364;&#8220;protein interaction event is controlled <br /> backbone of TIR-IBD and intimin-ECD are shown as gray dashed lines.         in enteropathogenic and enterohemorrhagic E. coli. We <br /> Mutagenesis, CD, and peptidomimetic screening results indicate that while <br /> N300, I301, and E312 are integral residues for TIR-IBD/intimin-ECD          also anticipate that this information will prove useful in <br /> association, structural preorganization of the b-hairpin residues of TIR-   the design of peptidomimetics able to interfere with <br /> IBD is required for binding.                                                intimin-TIR binding. <br />  <br />  <br />                                                                                                           www.proteinscience.org   2681 <br />  Ross and Miller <br />  <br />  <br />  <br /> Materials and Methods                                                experiments, the calorimeter sample cell and injection needle <br />                                                                      were each cleaned with at least 500 mL distilled, deionized <br />                                                                      water, followed by a final cleaning with dialysis buffer. All <br /> Molecular biology                                                    protein samples were degassed with stirring at 25&#194;&#176;C for three <br />                                                                      5-min intervals. Data were fit using the MicroCal Origin 7.0 <br /> Intimin-ECD (amino acids 658&#226;&#8364;&#8220;939) and TIR-IBD (272&#226;&#8364;&#8220;363) <br />                                                                      software on the ITC instrument. Binding curves were analyzed <br /> constructs were kindly provided by Professors Natalie <br />                                                                      using a one-site binding model in Origin (v 7.0; Origin Labs). <br /> Strynadka and Brett Finlay (University of British Columbia) <br />                                                                      Representative isotherms generated for each TIR-IBD mutant <br /> in pET-28a(+) vectors prepared as described previously (Luo <br />                                                                      are available in the Supplemental material (Supplemental <br /> et al. 2000). These constructs were transformed into E. coli <br />                                                                      Fig. S1). Reported binding constants are average values ob- <br /> BL21(DE3)pLysS and BL21(DE3) cell lines, respectively. Cells <br />                                                                      tained from at least three independent measurements. <br /> were grown at 37&#194;&#176;C in Lauria-Bertani media with 25 mg/mL <br /> kanamycin until an optical density (595 nm) of 0.5&#226;&#8364;&#8220;0.8 was <br /> reached. They were then shifted to room temperature, and <br /> isopropyl b-D-1-thiogalactopyranoside (IPTG) was added to a          Circular dichroism <br /> final concentration of 1 mM. Following a 24-h induction period,      CD experiments were conducted using an AVIV 202 spectrom- <br /> cells were pelleted at 3000 rpm in a Beckman GS-6R swinging          eter at 25&#194;&#176;C (AVIV Biomedical). Scans were conducted from <br /> bucket centrifuge (Beckman Coulter). Cell pellets were then          260&#226;&#8364;&#8220;200 nm in 0.5-nm increments with 5-sec sampling time per <br /> resuspended in buffer (20 mM HEPES, 500 mM NaCl, pH 7.5)             0.5 nm, using a 1-mm path length cell (Hellma). Spectra were <br /> and sonicated for 15 min. The suspension was then pelleted at        normalized, truncated at 210 nm (where strong buffer signal <br /> 15,000 rpm for 1 h, filtered through a 0.45-mm filter, and           started to obscure the protein signal), and plotted using Excel <br /> applied to a Pharmacia Hi-trap nickel chelating column using a       (Microsoft Office 2002; Microsoft). <br /> peristaltic pump at a flow rate of 0.5 mL/min. The column was <br /> washed with 20 mM HEPES, 500 mM NaCl, and 50 mM <br /> imidazole (pH 7.5), to remove any nonspecifically bound pro-         Peptide synthesis <br /> tein. Bound protein was eluted in 20 mM HEPES, 500 mM <br /> NaCl, and 300 mM imidazole, pH 7.5, and examined for purity          Unless otherwise stated all chemicals and reagents used in <br /> by 12% SDS-PAGE. Before ITC and CD experiments, all pro-             peptide synthesis were purchased from Sigma-Aldrich and used <br /> tein samples were dialyzed for 24 h with one buffer change           as obtained from the manufacturer. The INV, AINV, and INVK <br /> against 200 volumes of dialysis buffer (50 mM HEPES, 150 mM          peptides were synthesized on para-methyl-benzhydrylamine <br /> NaCl, pH 7.2). Following dialysis, concentrations were deter-        resin (0.4 mmol/g loading) (Advanced ChemTech). Resin was <br /> mined by UV-vis spectrophotometry using the calculated molar         initially swollen in methylene chloride (DCM; Mallinckrodt <br /> extinction coefficients of 37,080 and 1280 (M 1cm 1) for             Baker) and treated with 10% N,N9-diisopropylethylamine <br /> intimin-ECD and TIR-IBD, respectively (Ross et al. 2007).            (DIPEA; Advanced ChemTech) in DCM. Resin was then <br />    TIR-IBD point mutants were prepared using a Stratagene            washed with a sequence of 33 DCM, 33 N,N-dimethylform- <br /> QuikChange mutagenesis kit. Each residue of the TIR-IBD              amide (DMF; Mallinckrodt Baker), 33 methanol (BDH/VWR), <br /> b-hairpin was analyzed to determine if its side chain was within     33 DCM, 13 DMF, nutating for 5 min per wash step. Couplings <br />                                &#203;&#353; ) of intimin. This was determined <br /> interacting distance (limit: 5 A                                     were conducted using 2-(1H-Benzotriazole-1-yl)-1,1,3,3-tetra- <br /> using the previously published cocrystal structure of intimin-       methyluronium hexafluorophosphate (HBTU; 3.0 equivalents <br /> ECD and TIR-IBD (Luo et al. 2000) as visualized with PyMOL           with respect to resin; GenScript), N-a-(9-fluorenylmethyloxy- <br /> (v. 0.99; DeLano Scientific). Primers for the selected mutants       carbonyl; Fmoc)&#226;&#8364;&#8220;protected amino acids (3.0 eq; Advanced <br /> were designed according to kit recommendations, and sequences        ChemTech), and DIPEA (5.0 eq) in DMF for 30 min (3 min <br /> for each are available in the Supplemental material (Supple-         HBTU, Fmoc-amino acid, and DIPEA premixing in DMF). <br /> mental Table S1). The sequences of individual mutants were           Amino acids were double-coupled and then washed as above. <br /> confirmed at the University of Rochester Functional Genomics         The Fmoc protecting group was removed with two cycles of <br /> Center (Rochester, NY). Mutants with the desired sequences           20% piperidine/DMF (30 min each), followed by washing <br /> were transformed into BL21(DE3) E. coli cells, and protein was       as above. This cycle of double-couple, double-deprotect was <br /> prepared, dialyzed (50 mM HEPES, 150 mM NaCl, pH 7.2), and           repeated until the synthesis was completed. After each coupling <br /> analyzed for purity using 12% SDS-PAGE in the same manner            and deprotection, a small (;2 mg) aliquot of resin was heated <br /> as wild-type TIR-IBD. Concentrations were determined using           in ninhydrin stain to determine if free amines were present. In <br /> the same molar extinction coefficient as used for native TIR-        all cases, the desired presence or lack of free amines were <br /> IBD. The buffer used for all experiments discussed herein was        observed. <br /> the same as used for protein dialysis.                                  Following the end of chain elongation, global deprotection <br />                                                                      was undertaken to first remove the Fmoc group, followed by <br />                                                                      washing and removal of the trityl side-chain protecting group <br /> Isothermal titration calorimetry                                     (2% trifluoroacetic acid [TFA], 2% triethylsilane [TES; Oak- <br />                                                                      wood Products], DCM, 2 3 30 min). After a final wash se- <br /> ITC experiments were carried out using a MicroCal VP-ITC             quence, the peptides were liberated from the resin with 5% TES <br /> isothermal titration calorimeter. All experiments were carried       in TFA (2 mL/100 mg resin) for 10 min at 0&#194;&#176;C, followed by <br /> out at 25&#194;&#176;C (unless otherwise stated) with dialysis buffer           addition of trifluoromethanesulfonic acid (TFMSA; 200 mL/100 <br /> (described above) in the reference cell. All titrations were         mg resin). The reaction vessels were allowed to cool, and then <br /> reference subtracted to control for heats of dilution, which in      the resin and solvent mixture were stirred at room temperature <br /> all cases were negligible. All experiments were performed with       for 2 h. Resin and solvent were then filtered through glass wool, <br /> 10 mcal/sec reference power, 290 rpm stir rate, and 60 5-mL          and peptides were precipitated with cold ether (;5 volumes; <br /> injections with 300 sec spacing between injections. Between all      EMD Biosciences). Acid/ether solutions were left at  20&#194;&#176;C <br />  <br /> overnight, and then precipitates were centrifuged at 2000 rpm                    <br /> for 30 min at 4&#194;&#176;C. The acid/ether solution was removed, and                   <br /> pellets were washed with cold ether and centrifuged as before.              <br /> Pellets were washed in this fashion three times (total) and then            <br /> allowed to dry for 6 h before being placed under high vacuum                    <br /> overnight. Peptides were solubilized in protein dialysis buffer, to              <br /> a final concentration of 5 mM. They were then screened for their                 <br /> ability to bind 100 mM intimin-ECD by ITC.                                <br /> </body></html>