<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 3, 971-980, September 1992 <br />  <br />  <br /> Chromosome Loss, Hyperrecombination, Cell <br /> Cycle Arrest Yeast mcml Mutant <br /> Randolph Elble Bik-Kwoon Tye <br /> Section Biochemistry, Molecular Cell Biology, Cornell University, Ithaca, New York 14853 <br /> Submitted April 1, 1992; Accepted June 30, 1992 <br />  <br />  <br />         The original mcml-l mutant identified inability propagate minichromosomes <br />         ARS-specific manner, suggesting defective initiation DNA synthesis <br />         ARSs. This mutant defective expression a-mating-type-specific genes. Further <br />         genetic biochemical studies confirmed Mcml transcription factor mediates <br />         transcriptional regulation number genes, including genes outside mating <br />         type complement, interacting different cofactors. Although MCM1 essential <br />         gene, previously characterized mcml mutants exhibits significant growth defects. <br />         <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> assess roles Mcml essential growth, constructed and <br />         characterized temperature-sensitive conditional mutant mcml, mcml-llOL. This mutant <br />         exhibits temperature-dependent cell-cycle arrest, large, elongated bud single, <br />         undivided nucleus DNA content close 2n. In addition, shows elevated <br />         levels chromosome loss recombination. In spite severity mcml-llOL <br />         <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span>, mutant retains ARS-specific <span id='am-237' about='xsp:pattern' typeof='owl:Thing'>pattern</span> minichromosome instability. <br />         All phenotypes precisely exhibited mutants MCM genes, <br />         MCM2, MCM3, MCM5/CDC46, shown play interacting roles the <br />         early steps DNA replication. <br />  <br /> INTRODUCTION                                                ARSs indirectly, transcription factor reg- <br />                                                             ulates expression replication initiator proteins, or <br /> Mcml versatile transcription factor regulates     directly, replication initiator protein binds to <br /> the expression diverse genes interacting         ARSs. Recently, role replication initiator been <br /> multiple cofactors. It acts positive regulator a-   shown direct binding Mcml multiple sites <br /> specific genes using a1 cofactor (Bender           ARSs (Christ, Chang, Tye, unpublished results). <br /> Sprague, 1987; Passmore et al., 1988, 1989). It acts     Because initiation DNA synthesis commits cell to <br /> a negative regulator a-specific genes binding co-     enter complete cell cycle, Mcml may <br /> operatively a2 (Keleher et al., 1988; Passmore et      play key role coordinating cell division. <br /> al., 1989; Elble Tye, 1991). Recently,         Functional dissection Mcml indicates 80 <br /> shown regulate cell-cycle dependent expression       amino acid DNA binding domain (amino acids 18-97) <br /> of SW15 gene, encodes transcription factor      Mcml sufficient viability, minichromosome <br /> for HO endonuclease, interacting SFF,        maintenance, general transcriptional activation <br /> SW15 factor (Lydall et al., 1991). It involved   (Christ Tye, 1991). Proteins containing homology <br /> the regulation arginine metabolic genes interacting   80 amino acid domain identified in <br /> with ARGR proteins (Dubois Messenguy, 1991).        diverse organisms including yeast, plants, humans. <br /> Additional genes regulation Mcml           Among these, human serum response factor (SRF) <br /> likely identified. It appears combi-     plant homeotic genes, AG DEF, partic- <br /> natorial action multiple cofactors, Mcml play      ularly interesting regulatory role cell <br /> a pivotal role coordinating cell growth yeast.        growth cell division. SRF cell proliferation factor <br />    MCM1 originally identified gene involved     mediates transcription growth-stimulated <br /> the propagation minichromosomes ARS-de-            genes response growth factors serum (Norman <br /> pendent manner. This <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> suggests Mcml           et al., 1988). The AG gene Arabidopsis (Yanofsky et <br /> may involved initiation DNA replication     al., 1990) DEF gene Antirrhinum (Sommer et <br />  <br /> &#194;&#169; 1992 The American Society Cell Biology                                                                     971 <br />  R. Elble B.-K. Tye <br />  <br /> al., 1990) homeotic genes important                   cdc17-2 his7 leu2 trpl ura3), H17C1B1 (MATa cdc17-1 his7 ural). <br /> flower development. Organ development plants re-                   The cdcl 7-2 strain used plasmid stability experiments, whereas <br />                                                                       cdc17-1 allele used ultraviolet light (UV) sensitivity <br /> quires coordination gene expression DNA                    experiment. The tub2 allele used tub2-401, gift T. Huffaker <br /> replication achieve cell differentiation cell divi-            (Cornell University). The mcm3-1 strain R61 described (Gib- <br /> sion.                                                                 son et al., 1990). The rad9 strain 7859-7-4 isogenic wild-type <br />     Although MCM1 essential gene appears                strain 7859-10-2 gift T. Weinert (University Arizona). <br /> play key role regulation gene expression and <br /> DNA replication, currently available mcml                 Determination Cell Doubling Times <br /> mutants exhibits significant growth defects. To learn                 Doubling times determined plotting optical densities of <br /> more essential role Mcml cell,                 cultures growing logarithmically 30&#194;&#176;C yeast extract, peptone, <br /> constructed conditional mutant truncating                    dextrose (YPD) medium. <br /> MCM1 gene amino acid codon 110 (out 286 <br /> total). This mcml -1 OL mutation results extremely              Microscopic Techniques <br /> unstable MCM1 mRNA truncated protein                     Cell staining DNA-specific dye 4,6-diamidino-2-phenylindole <br /> (Elble Tye, 1991). The effect mutation                 (DAPI) carried according published procedures (Williamson <br />                                                                       Fennell, 1975). Cells stained Calcofluor (Sigma, St. <br /> transcriptional regulation mating type-specific genes              Louis, MO) described Pringle et al. (1989). Cells photo- <br /> is dramatic known mcml                      graphed using Zeiss (Thornwood, NY) microscope equipped with <br /> mutant alleles; affects expression a-              lOOX objective. <br /> specific genes, repression acti- <br /> vation a-specific genes (Elble Tye, 1991). In             Flow Cytometry <br /> paper, characterize physiological defects              Cells stained propidium iodide according method of <br /> mcml-IlOL mutant strain depleted Mcml                       Hutter Eipel (1978) analyzed using Ortho Diagnostics <br /> activity. Properties mutant indicate suffers          System 2151 machine (Coulter Electronics, Hialeah, FL). <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> multiple defects characteristic mutants have <br /> lesions DNA replication, cell cycle arrest be-             Chromosome Loss Recombination Assay <br /> fore nuclear division, increased chromosome loss, hy-                 Loss frequencies determined essentially described Gibson <br /> perrecombination, UV sensitivity (Hartwell                    et al. (1990). Single colonies picked plates containing com- <br /> Smith, 1985). In fact, properties mcml-1lOL              plete medium minus uracil, suspended water, spread YPD <br /> mutant remarkably similar exhibited                   plates. The plates incubated 30&#194;&#176;C -25 generations of <br />                                                                       growth. Colonies picked, resuspended water, spread <br /> mutants interacting MCM genes, MCM2 (Sinha                   plates containing 5-fluoroorotic acid (FOA) YPD determine <br /> et al., 1986) MCM3 (Gibson et al., 1990), MCM5/                   fraction cells URA3 marker lost. <br /> CDC46 (Hennessy et al., 1991). This family structur- <br /> ally related MCM genes, like MCMl, required                Mitotic Plasmid Stability Assay <br /> stable maintenance minichromosomes ARS-                      Plasmid stability determined described Christ Tye <br /> specific manner (Yan et al., 1991; Chen et al., press).            (1991). Cells grown 30&#194;&#176;C tub2 strain, which <br />                                                                       grown 20&#194;&#176;C. <br />  <br /> MATERIALS AND METHODS                                                 RESULTS <br /> Strain Constructions                                                  The mcml-llOL Mutation Affects Different ARSs <br /> Construction strains C2-286L C2-11OL strains 14D-      Differently <br /> 286L 14D-1 OL described (Elble Tye, 1991). The       We previously described phenotype the <br /> diploid strains 14D-286L/UH5 14D-11OL/UH5 constructed <br /> by crossing 14D-286L 14D-1 lOL strain UH5-MT (MATa            mcml-l allele (Passmore et al., 1988; Christ Tye, <br /> leu2::URA3 mcmlAXho I/BamHI [MCMI ARSI CEN5 TRP1]) se-            1991). The mcml-l allele ARS-specific mini- <br /> lecting Trp- segregants. Strain UH5-MT contains URA3 gene in-     chromosome-maintenance defect reduced <br /> serted gene replacement LEU2 locus chromosome III.       ability activate a-specific genes. Several observations <br /> This accomplished cleaving plasmid pLURA Hpa I <br /> and EcoRI, transforming, selecting Ura+Leu- transformants. The    suggested mcml-11OL allele represents more <br /> replacement verified Southern analysis. pLURA con-         severe mutation mcml-l . For instance, doubling <br /> structed replacing Cla I/EcoRV fragment YIp351 containing   time mcml-IIOL cells YPD medium 30&#194;&#176;C -4 <br /> LEU2 Cla I/Sma I fragment YIp5 containing URA3. To        h compared 1.5- 2-h doubling time wild- <br /> determine recessiveness mcml-IlOL allele, C2-llOL <br /> strain transformed pMCM1-TRP YIp5-XB286 (Elble     type mcml-l cells. The effects mcml-llOL tran- <br /> and Tye, 1991). pMCM1-TRP constructed inserting 3.3        scriptional activation extreme (Elble and <br /> kb Sph I/BamHI fragment containing MCM1 pSE358.                  Tye, 1991). Thus, expected mcml-llOL allele to <br />                                                                       greater effects minichromosome stability and <br /> Other Strains                                                         sought determine mcml-l1OL mutant <br /> The cdc strains used K72-1 (MATa cdc7-4 ura3), 6433-4C (MATa     maintains ARS-specificity observed mcml-1. To <br /> cdc4-1 leu2-3), F391 (MATa cdc14 leu2 ura3 ade2), 4837-5-2 (MAToa     end, transformed plasmids containing ARSI or <br />  <br /> 972                                                                                                           Molecular Biology Cell <br />                                                                                                                      Yeast mcml Mutant <br />  <br /> ARS120 mcml-llOL isogenic wild-type strains            c <br /> and measured plasmid loss rates. ARSI represen-         0 <br /> tative subset ARSs, including ARS131       I-O <br />                                                                a) <br />                                                                       0.5 -          T                          ARS <br /> telomeric ARSs unstable mcml-l          a)Q                                              * 120 <br /> of ARS-specific mcm mutants (Maine et al.,                  0.4 -                                     *     i <br /> 1984). Similarly, ARS120 represents subset ARSs, <br /> including ARS121, ARS131 C, ARS206,          L.e)   0.3 - <br />                                                                a) <br />                                                                I- <br /> more stable mutants. Figure 1 shows that, in <br /> the mcml-llOL mutant strain, ARSI plasmid                     0.2- <br />                                                                Cu <br />                                                                I- <br /> dramatically stable ARS120 plasmid, ap-          Co <br /> proaching theoretical maximum 50% loss cell                0.1 - <br />                                                                Cl) <br /> division. The specificity mcml defect ARSI       0 <br />                                                                -j     0.0 - <br /> not absolute. Although ARS120 plasmid                             MCM1 mcml          cdc17   cdc4       cdc7   cdc14   tub2 <br /> less affected mcml-11OL mutation                                       mcml <br /> ARSI plasmid, nonetheless 10 times stable                                           St ra n <br /> the mutant wild-type strain. These defects         Figure 1. Mitotic stability minichromosomes containing different <br /> in ARS stability partially complemented plasmid       ARSs wild-type mcml-llOL mutant cells. The minichromosomes <br /> containing wild-type MCMI gene, centromere,          used YCpl20, containing ARSI20, CEN5, LEU2, URA3; and <br /> a TRP1 marker (MCM1/mcml; Figure 1). Incomplete                pCD4, containing ARS1, CEN5, LEU2, URA3. Error bars represent <br />                                                                deviation mean independent assays. The theoretical <br /> complementation instability               maximum 50% loss cell division. MCM1/mcml denotes strain <br /> MCM1-containing plasmid. Derivatives 2,u plas-          mcml-lIOL mutation complemented plasmid <br /> mid affected mcml-I1OL mutation. The           bearing MCM1 gene, centromere, TRPI marker. Loss rate <br /> plasmid YEp24 loss rate 3% cell division          strain based ratio Trp+Ura+ Trp+ cells. <br /> in wild-type strain 15% mutant strain, a <br /> fivefold difference. This instability significant because <br /> the 2,u plasmid shares little common natural           cation-defective cdc mutants point a <br /> chromosomes presence ARS;             specific lesion initiation ARSs. <br /> segregation fidelity apparently based maintenance <br /> of high copy number association mi-           The mcml-11OL Mutation Causes Elevated <br /> crotubules (Som et al., 1988).                                 Frequencies Chromosome Loss Recombination <br />    To assess ARS-specificity minichromo-           A number mutations arrest nuclear division in <br /> some maintenance general property replication-         yeast enhance chromosome loss cells are <br /> defective mutants, measured loss rates          grown semipermissive temperature (Hartwell and <br /> these plasmids cdcl 7 strain, defective DNA poly-      Smith, 1985). However, mutations interfere <br /> merase (Budd Campbell, 1987)           DNA replication repair lead higher levels of <br /> mutant strains defects S phase initiation           recombination addition chromosome loss (Hartwell <br /> chromosome segregation. Figure 1 shows cdc17          Smith, 1985). For example, mutations DNA <br /> mutant severe mcm defect grown 30&#194;&#176;C,             polymerases (cdc17; Budd Campbell, 1987) and <br /> a semipermissive temperature allele,         (cdc2; Sitney et al., 1989) lead increased levels of <br /> ARS1- ARS120-containing plasmids equally              chromosome loss recombination, a <br /> affected. We tested mutants       mutation segregation machinery, tub2, affects only <br /> reported defective maintenance chromosomes         chromosome stability (Huffaker et al., 1988). To deter- <br /> or minichromosomes, namely, cdc4, cdc7, cdcl4,             truncation allele these <br /> tub2 (Hartwell Smith, 1985; Huffaker et al., 1988;         properties, constructed wild-type mutant diploid <br /> Palmer et al., 1990). cdc4 cdc7 thought          strains monitored stability MATa-containing <br /> defective late Gl events leading initiation DNA       copy chromosome III. This chromosome con- <br /> synthesis, cdc14 tub2 defective chro-       tained copy URA3 gene integrated gene- <br /> mosomal segregation (Pringle Hartwell, 1981; Huf-          replacement leu2 locus left arm (Figure <br /> faker et al., 1988). These mutants moderate mcm        2B). This modification allowed use FOA mon- <br /> defects grown semipermissive temperatures              itor loss chromosome ura3 cells can <br /> (30&#194;&#176;C cdc4, cdc7, cdc14; 20&#194;&#176;C tub2)            grow FOA-containing media (Boeke et al., 1984). <br /> showed ARS specificity (Figure 1). Loss rates      However, Ura- cells result recombi- <br /> plasmids mutants ranged 7 13%            nation events centromere leu2::URA3 <br /> per cell division, compared loss rates 1-2%         resulting loss left arm the <br /> wild-type controls. Thus, origin-specific plasmid-             entire chromosome. These events distinguished <br /> maintenance defects general feature repli-        testing Ura- cells ability mate they <br />  <br /> Vol. 3, September 1992                                                                                                               973 <br />  R. Elble B.-K. Tye <br />  <br />        A                                                                  mcml-IlOL allele causes growth defect; doubling <br />                                LOSS            RECOMBINATION <br />                                                                           time YPD 30&#194;&#176;C twice isogenic wild- <br />                     150.                                                  type strain. Furthermore, mutant cells shifted 370C <br />               U)                                                          arrest growth. To determine cells arrested <br />                                                                           particular point cell cycle, cells grown <br />               X     100.                 97                               initially 16&#194;&#176;C, semipermissive temperature for <br />               z <br />                                                                           mutant, shifted 37&#194;&#176;C various times, <br />                                                                           treated DNA-specific stain (DAPI), examined <br />                   50 -                                                  microscopically. At 16&#194;&#176;C, wild-type mutant <br />               w <br />  <br />               LI- <br />                                                                           populations contained unbudded cells normal <br />                                                          23               shape, mutant cells somewhat larger <br />                               2.7                02                       wild type (Figures 3,A B, 4). However, <br />                                                                           4 h 37&#194;&#176;C, 89% mutant cells large <br />                              MCM1       mcml    MCM1    mcml              budded single nucleus near neck, <br />                                                                           wild-type population contained mostly <br />       B <br />                                                                           unbudded Gi cells (Figures 3, C-F, 4). This phe- <br />                                                                           notype indicates mcml truncation mutant is <br />                                                                           blocked nuclear division, consistent defects in <br />                            leu2::URA3           LEU2::MCMI                DNA replication mitosis. This defect recessive. <br />                                                                           Mutant cells wild-type copy MCM1 was <br />                                                                           integrated URA3 delayed nuclear division <br />                                 MATa             MATs                     indistinguishable wild-type cells cell <br />                                                                           cycle distribution morphology. In logarithmic <br />                                                                           culture grown 30&#194;&#176;C, -40% cells unbud- <br /> Figure 2. Frequencies chromosome loss recombination             ded 60% budded, wild-type <br /> wild-type mcml-IIOL mutant strains. (A) Frequencies loss        control. <br /> chromosome III recombination leu2 CEN3               In separate experiment, DNA content the <br /> determined. Each bar represents median 5 independent trials        cells measured flow cytometry. Cells have <br /> for wild-type 10 independent trials mutant cells. (B) Schematic <br /> representation MATa- MATa-containing copies chro-           defects DNA replication arrest DNA contents <br /> mosome III. Sites integration URA3 marker wild-         ln 2n, depending S phase mu- <br /> type mutant Mcml alleles shown. CEN3 represented            tant gene product supposed act tightness <br /> the circles bars.                                              allele; leaky mutants arrest near 2n of <br />                                                                           presence checkpoint DNA integrity the <br /> were cells. Mating indicate MATa,                     G2/M boundary (Weinert Hartwell, 1988; Hartwell <br /> right arm chromosome, lost. In                       Weinert, 1989; Hennessy et al., 1991). Both wild- <br /> contrast, cells retain MATa unable mate                   type mutant cells grown semi-permissive <br /> must lost URA3 recombination event.                             temperature mutant, 16&#194;&#176;C, ln DNA <br />   Wild-type mutant diploid strains grown                      contents, suggesting Gl limiting the <br /> 30&#194;&#176;C YPD medium -25 generations allow                           cell cycle temperature (Figure 5). The DNA peaks <br /> chromosome loss. The cells plated YPD                        mutant cells shifted slightly right, <br /> or FOA-containing medium determine percentage                      possibly accumulation mitochondrial DNA. <br /> of cells lost URA3 marker, Ura- cells                    Mutant cells shifted 37&#194;&#176;C displayed 2n peak with <br /> were  screened ability mate. The frequencies                  large component less-than-2n material dis- <br /> of chromosome loss recombination                       crete ln peak, suggesting presence partially rep- <br /> higher mcml mutant strain wild-type                    licated DNA DNA degradation. <br /> control (Figure 2A). Chromosome loss -40 times                           Examination cells growing logarithmically 30&#194;&#176;C <br /> more frequent recombination -90 times fre-                       revealed mcml-1lOL cells develop multiple, <br /> quent. Elevated chromosome loss restricted                      elongated buds (Figure 6, B D). (Cells bearing the <br /> chromosome III; OFAGE analysis clone cells                   mcml-l allele exhibit large bud morphology at <br /> had lost chromosome III revealed chro-                     37&#194;&#176;C arrest growth develop additional <br /> mosomes lost. The hyper-recombination                       buds.) Most buds appear nucleated; how- <br /> phenotype mcml-llOL consistent defect                     ever, nuclei irregular stretched-out shapes, are <br /> DNA replication mutant strain                       usually near neck, appear con- <br /> segregation defect.                                                       nections nuclei adjacent buds (Figure 6F). To <br /> mcml-11OL Arrests Nuclear Division                                     ascertain elongated structures true buds, <br /> We established previously MCM1 gene es-                       treated cells Calcofluor (Sigma), which <br /> sential yeast cell growth (Passmore et al., 1988). The                stains chitin rings occur neck between <br />  <br /> 974                                                                                                       Molecular Biology Cell <br />                                                                                                                         Yeast mcml Mutant <br />  <br />  <br />  <br />  <br /> Figure 3. Photomicrographs wild-type (A, C, E) mcml-llOL mutant (B, D, F) cells. Cells grown 16&#194;&#176;C (A B), shifted <br /> to 37&#194;&#176;C 4 h (C-F), stained DAPI, photographed. Nomarski optics (A-D) reveals cell morphology, DAPI fluorescence (E and <br /> F) reveals DNA content. Bar, 10 ,um. <br />  <br />  <br />  <br />  <br /> Vol. 3, September 1992                                                                                                                975 <br />   R. Elble B.-K. Tye <br />  <br />                                                                             bud emerged opposite end <br />                                                                             mother cell. The irregular shape cells is <br />                                                                             reminiscent shmoo cells arrested Gl mating <br />                                                                             factor mutations CDC28 Gl cyclin genes <br />                                     ,Q       (7)        (2)         &#194;&#163;       (Reed, 1980; Richardson et al., 1989). We considered <br />                           0i)       ()      (7         (5          (:)      possibility morphology related to <br />                                                                             role Mcml mating type regulator. It was <br />                   MCMl    75        19         5        0           1       conceivable, e.g., derepression a-specific re- <br /> 160C                                                                        ceptor factor cells (Elble Tye, 1991) could <br />                   mcml    56         0        32       7           5        lead autocrine response endogenous factor <br />                                                                             and, consequently, partial arrest. However, same <br />                                                                             aberrant morphology observed mcml-llOL mutant <br />                                                                             cells mating types. Moreover, mcml-IIOL <br />                   MCM1    56        24        12       1          7         cells arrest G2, Gl, DNA content (Figure 5). <br /> 370C                                                                        These morphological aberrations some <br />                   mcml     1         0        56       33         10        unknown Mcml function functions disrupted <br /> rigure &apos;. yuantitation ot arrest morpnologles. rampies ot uLAli- <br />       r:..t                                                    A;;C n ADT   mutant. <br /> stained cells examined Figure 3 analyzed. At 200 cells <br /> were counted sample. Percentages cells unbudded, <br /> small-budded, large-budded indicated. Large-budded cells             Sensitivity UV mcml Mutant <br /> contained single nucleus cell body, undivided nucleus <br /> in bud neck, divided nuclei segregated cell body.          Inhibition DNA replication leads accumulation <br /> Cells considered large-budded bud three-           damaged DNA (Hennessy et al., 1991). The elevated <br /> fourths size mother cell.                                        levels recombination observed mutant are <br />                                                                             consistent possibility. We reasoned DNA <br />                                                                             damage present, mutant cells hy- <br /> mother bud. Figure 6H shows constricted                        persensitive additional DNA damage caused mu- <br /> regions cellular compartments stain heavily,                        tagens ultraviolet light. We mcml- <br /> indicating compartment does represent                           1lOL strain moderately sensitive UV, the <br /> mother cell bud; however, chitin rings                     level rad9 deletion mutant (Figure 7; Schiestl <br /> elongated like rest cell body. The sites bud                  et al., 1989). The UV sensitivity mcml mutant <br /> emergence appear abnormal                         related apparent replication defect to <br /> cells examined. For example, buds Figure                      function controlled Mcml. To help dis- <br /> 6H occurs end row bud scars (arrowhead),                     tinguish possibilities, examined <br />  <br />                   160C                             370C, 4hr <br />  <br />  <br />  <br />                                                                                          MCM1 <br />  <br />  <br /> cc <br /> Lu <br /> m <br /> z <br /> -i <br /> ui <br /> 0 <br />  -1 <br /> ul <br />               I            I   Li <br /> w                                                                                        mcml <br />  <br />  <br />  <br />                                                                                                  Figure 5. DNA content arrested cells. <br />                                                                                                  Wild-type mcml-1 lOL mutant cells were <br />                                                                                                  grown YPD medium 16&#194;&#176;C then <br />                                                                                                  shifted 37&#194;&#176;C 4 h. Aliquots re- <br />                                      RELATIVE DNA CONTENT                                        moved analyzed flow cytometry. <br />  <br /> 976                                                                                                          Molecular Biology Cell <br />                                                                                                                         Yeast mcml Mutant <br />  <br />  <br />  <br />  <br />  E                   F                                        G                        H <br /> Figure 6. Abnormal morphology mcml-1lOL cells growing semipermissive temperatures. Wild-type (A, C, E, G) mcml-lIOL <br /> mutant (B, D, F, H) cells grown 30&#194;&#176;C YPD medium stained DAPI (E F) DAPI plus Calcofluor (G H). <br /> (A-D) Nomarski optics. The arrowheads point chitin rings bud scars. Calcofluor quenches DAPI fluorescence, making difficult to <br /> observe chitin rings DNA simultaneously, especially mutant. Bar, 10 ,um. <br />  <br /> another mcm mutant, mcm3-1, known replication-                  mutation greatly increases levels chromosome <br /> defective mutant, cdc17-1, sensitivity UV. mcm3-               loss recombination, severely destabilizing <br /> 1 previously reported hypersensitive               minichromosomes extreme bias certain <br /> cells grown permissive temperature, 23&#194;&#176;C.                   ARSs, represented ARS1 (Maine et al., 1984). <br /> However, strains moderately                   These just phenotypes expected Mcml in- <br /> hypersensitive shifted restrictive temper-                volved DNA replication, especially level of <br /> ature briefly UV exposure (Figure 7). <br /> This temperature shift little effect <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>survival</span>                  100 <br /> of wild-type strain relative strain grown <br /> continuously 30&#194;&#176;C (Figure 7). Thus appears that <br /> hypersensitivity UV implicit mcm phe-                     10- <br />                                                                                         KI                           wild-type, C2-286L <br /> notype. The moderate nature defects, compared                                                          . <br />                                                                                                                      wild-type, 7859-10-2 [37] <br /> with rad3 mutant, e.g., suggests Mcm proteins              *                                       .      wild-type, 7859-10-2 <br /> may directly involved DNA repair (Haynes                                                                   cdc17, H17C1B1 [37] <br /> and Kunz, 1981). Inspection mcml mutant cells 1 d                                                                 rad9, 7859-7-4 <br /> after irradiation revealed UV treatment greatly ex-                   1 .1-                            -0-      mcm3, R61 [37] <br /> acerbates elongated, irregular bud morphology;                                                              mcml-1 IOL, C2-1 IOL <br /> mcml-l cells, normally wild-type mor- <br /> phology 30&#194;&#176;C, display aberrant phenotype                    .01 <br /> irradiation isogenic wild-type cells not.                                   10       20 <br /> These observations provide additional evidence                              Joules/m 2 <br /> elongated bud phenotype linked DNA damage                    Figure 7. Ultraviolet-light sensitivity wild-type mutant cells. <br /> the mcml mutants.                                                     Dilutions logarithmically growing cultures plated YPD <br />                                                                       exposed ultraviolet germicidal lamp various times. <br /> DISCUSSION                                                            Cells grown 30&#194;&#176;C mcm3 cdcl 7 strains which, <br />                                                                       wild-type control, shifted 37&#194;&#176;C 1.5 h <br /> We shown mutation MCM1 gene                        exposure incubated room temperature <br /> results conditional arrest nuclear division. This             colonies appeared. <br />  <br /> Vol. 3, September 1992                                                                                                                   977 <br />  R. Elble B.-K. Tye <br />  <br /> initiation. The 2n DNA content arrested          different effects. However believe the <br /> cells inconsistent initiation defect.   arrest phenotype results largely inability of <br /> However, Hennessy et al. (1991) demonstrated              mutant cells complete DNA replication. The <br /> that different alleles cdc46, allelic      mutant cells hallmarks replication- <br /> ARS-specific mutant mcm5-1 (Chen, unpublished re-              defective mutants, including elevated levels chro- <br /> sults), arrest DNA contents ln 2n, de-            mosome loss recombination, sensitivity ultravi- <br /> pending tightness particular allele. Other       olet light, appropriate arrest morphology. <br /> workers apparently leaky mutations          However principal characteristic mcml mutant <br /> the genes DNA polymerases lead unex-               ties initiation replication defect ARS <br /> pectedly cell cycle arrests 2n, nearly 2n, DNA      function, phenotype shared mcm mu- <br /> contents (Conrad Newlon, 1983; Hartwell, 1973).            tants, including mcm2 (Sinha et al., 1986), mcm3 (Gibson <br /> Thus, DNA content arrest reliable indicator        et al., 1990), mcm5/cdc46 (Chen et al., press) and <br /> of execution point allele. We expect tighter    abfl mutants (Rhode et al., 1992). Mcm2, Mcm3, <br /> conditional alleles mcml arrest ln DNA           Mcm5 family structurally functionally <br /> content.                                                       related proteins (Hennessy et al., 1991; Yan et al., 1991; <br />    Several transcription factors mammalian cells ap-        Chen et al., press). Their subcellular localization is <br /> pear involved modulating progress Gl          cell-cycle-regulated nuclear late <br /> S phase coordinating cell growth cell division.         M phase Gl/S transition, cytoplasmic at <br /> Some act promoting transcription pro-          phases cell cycle (Yan, unpublished results). <br /> liferation-related genes, e.g., SRF, AP1, E2F (Angel       The properties proteins suggest act <br /> et al., 1987; Norman et al., 1988; Mudryj et al., 1990;        narrow window Gl/S transition <br /> Chittenden et al., 1991); E2F shown           beginning S phase virtue nuclear <br /> interact directly cell cycle regulator Cyclin A       localization effect initiation DNA replication <br /> (Mudryj et al., 1991). Others appear play direct roles      ARSs. In addition similarities phenotypes, <br /> in replication initiation, especially viral origins         Mcml tied mcm mutants genetically. <br /> (DePamphilis, 1988). NF1 promotes SV40 replication             An mcml-l mcm2-1 double mutant forms synthetic <br /> by binding adjacent origin (Jones et al., 1987;         lethal (Yan, unpublished results), implying the <br /> Cheng Kelly, 1989), p53 act negatively         Mcml Mcm2 proteins interact functionally. <br /> on SV40 origin (Bargonetti et al., 1991). The             Does Mcml play direct role activation ARSs <br /> mammalian origin characterized,         direct binding, involved indirectly acti- <br /> DHFR origin, contains potential binding sites      vating expression replication initiation factor? <br /> transcription factors, APi, NFIII (Daily        We believe Mcml act levels. We have <br /> et al., 1990). Mutations abolish function        observed potential Mcml binding sites upstream two <br /> transcription factors available; however,        genes thought involved DNA replication, MCM3 <br /> methods employed demonstrate            RPAl, centered -180 -250, respec- <br /> of factors required Gl S transition.      tively, translation start codons. RPA1 encodes <br /> For example, antibodies Fos Jun proteins           yeast homologue single-stranded-DNA binding <br /> been shown inhibit DNA synthesis cultured cells          protein required SV40 replication vitro (Heyer et <br /> (Kovary Bravo, 1991). Similarly, interference         al., 1990). The possibility Mcml regulate <br /> Myc function antisense oligomers causes cells ar-        RPA1 especially interesting terminal <br /> rest Gl (Heikkila et&apos; al., 1987).                           morphology rpal mutants resembles mcml <br />    In contrast, mcm1 mutation described causes        mutants. On hand, observed <br /> cells arrest later stage cell cycle, nuclear    Mcml binding sites ARSs ARSI found <br /> division. To knowledge, reported         disruption sites reduces ARS function <br /> instance mutation transcription factor causing       (Chang, unpublished data). The difference behavior <br /> a cell cycle arrest nuclear division. Mutations    ARSI ARS120 mcm mutants ex- <br /> essential yeast transcription factors Rapi (Shore      plained differences quality abundance of <br /> and Nasmyth, 1987) Abfl (Rhode et al., 1992)             Mcml binding sites ARSs. We by <br /> not cause arrest specific point cell cycle.        inspection vitro binding experiments that <br /> However, case Abf 1, ARS-           ARS120 contains Mcml binding sites, whereas <br /> binding protein, prior synchronization abfl mutant          ARSI contains fewer sites, lower affinity (Chang and <br /> cells factor arrest cells S phase non-     Christ, unpublished data). In hypomorphic allele such <br /> permissive temperature (Rhode et al., 1992), suggesting        mcml-1 1OL, ARSs ARSI likely to <br /> that essential function performed Abf 1     bind Mcml likely initiate replica- <br /> the G1/S transition DNA replication. Like Abf 1,        tion. Residual initiation events ARSs ARS120 <br /> the fact Mcml transcription factor means        result incomplete replication yeast <br /> the arrest phenotype represent composite              genome. <br />  <br /> 978                                                                                              Molecular Biology Cell <br />                                                                                                                                  Yeast mcml Mutant <br />  <br /> ACKNOWLEDGMENTS                                                            Haynes, R. H., Kunz, B. A. (1981). DNA repair mutagenesis <br />                                                                            yeast. In: The Molecular Biology Yeast Saccharomyces: Life <br /> We thank Tim Huffaker advice, use microscope, critical      Cycle Inheritance, ed. J. N. Strathem, E. W. Jones, J. R. Broach. <br /> reading manuscript. This work supported NIH Grant            Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 371-414. <br /> GM34190. <br />                                                                            Heikkila, R., Schwab, G., Wickstrom, E., Loke, S. L., Pluznik, D., <br />                                                                            Watt, R., Neckers, L. (1987). A c-myc antisense oligodeoxynu- <br /> REFERENCES                                                                 cleotide inhibits entry S phase progress GO Gl. <br /> Angel, P., Imagawa, M., Chui, R., Stein, B., Imbra, R., Rahmsorf, H.,      Nature 328, 445-449. <br /> Jonat, C., Herrlich, P., Karin, M. (1987). Phorbolester-inducible      Hennessy, K. M., Lee, A., Chen, E., Botstein, D. (1991). A group <br /> genes contain common cis element recognized TPA-modulated           interacting yeast DNA replication genes. Genes Dev. 5, 958-969. <br />  trans-acting factor. Cell 49, 729-739. <br />                                                                            Heyer, W., Rao, M., Erdile, L., Kelly, T., Kolodner, R. (1990). An <br />  Bargonetti, J., Friedman, P., Kern, S., Vogelstein, B., Prives, C.    essential Saccharomyces cerevisiae single-stranded DNA binding protein <br /> (1991). Wild-type mutant p53 immunopurified proteins bind          homologous large subunit human RP-A. EMBO J. 9, 2321- <br /> to sequences adjacent SV40 origin replication. Cell 65, 1083-    2329. <br />  1091. <br />                                                                            Huffaker, T. C., Thomas, J. H., Botstein, D. (1988). Diverse effects <br /> Bender, A., Sprague, G. F. (1987). MATal protein, yeast tran-        f,-tubulin mutations microtubule formation function. J. Cell <br /> scription activator, binds synergistically second protein      Biol. 106, 1997-2010. <br /> set ctll-type-specific genes. Cell 50, 681-691. <br />                                                                            Hutter, K. J., Eipel, H. E. (1978). Flow cytometric determinations <br /> Boeke, J. D., LaCroute, F., Fink, G. R. (1984). A positive selection   cellular substances algae, bacteria, molds yeasts. Antonie <br /> for mutants lacking orotidine-5&apos;-phosphate decarboxylase activity       van Leeuwenhoek J. Microbiol. Serol. 44,, 269-282. <br /> yeast: 5-flouro-orotic acid resistance. Mol. Gen. Genet. 197, 345-346. <br />                                                                            Jones, K. A., Kadonaga, J. T., Rosenfeld, P. J., Kelly, T. J., Tjian, <br /> Budd, M., Campbell, J. L. (1987). Temperature-sensitive mutations      R. (1987). A cellular DNA-binding protein activates eukaryotic <br /> in yeast DNA polymerase I gene. Proc. Natl. Acad. Sci. USA 84,         transcription DNA replication. Cell 48, 79-89. <br />  2838-2842. <br />                                                                            Keleher, C., Goutte, C., Johnson, A. (1988). The yeast cell-type- <br /> Chen, Y., Hennessy, K., Botstein, D., Tye, B.K. CDC46/MCM5,            specific repressor a2 acts cooperatively non-cell-type-specific <br /> a yeast protein subcellular localization cell-cycle regulated,    protein. Cell 53, 927-936. <br /> is involved DNA replication ARSs. Proc. Natl. Acad. Sci. (in <br /> press)                                                                     Kovary, K., Bravo, R. (1991). The Jun Fos protein families <br />                                                                            required cell cycle progression fibroblasts. Mol. Cell. <br /> Cheng, L., Kelly, T. J. (1989). Transcriptional activator nuclear      Biol. 11, 4466-4472. <br /> factor I stimu <br /> </body></html>