<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Cohesin promotes the repair of ionizing <br /> radiation-induced DNA double-strand breaks <br /> in replicated chromatin <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        cancer (1,2). DSBs are induced by ionizing radiation, a <br />                                                                                 range of chemotherapeutic drugs and are formed <br /> The cohesin protein complex holds sister                                        endogenously during DNA replication or as initiators of <br /> chromatids together after synthesis until mitosis. It                           programmed genetic rearrangement processes that occur <br /> also contributes <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> post-replicative DNA repair in                              during lymphocyte di&#239;&#172;&#8364;erentiation and meiosis. In order <br /> yeast and higher eukaryotes and accumulates at                                  to repair DSBs, higher eukaryotic cells primarily utilize <br /> sites of laser-induced damage in human cells. Our                               two conceptually di&#239;&#172;&#8364;erent pathways, non-homologous <br /> goal was to determine whether the cohesin subunits                              end-joining and homologous recombination. Non- <br /> SMC1 and Rad21 contribute to DNA double-strand                                  homologous end-joining repairs DSBs with no require- <br /> break repair in X-irradiated human cells in the G2                              ment for sequence homology at the break ends and <br /> phase of the cell cycle. <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>RNA interference</span>-mediated                              operates throughout the mammalian cell cycle. <br /> depletion of SMC1 sensitized HeLa cells to X-rays.                              Homologous recombination, which utilizes an undamaged <br />                                                                                 template of a homologous sequence for faithfully restoring <br /> Repair of radiation-induced DNA double-strand <br />                                                                                 the sequence at the break site, preferentially contributes to <br /> breaks, measured by cH2AX/53BP1 foci analysis,                                  DSB repair in late S/G2 when a sister chromatid is avail- <br /> was slower in SMC1- or Rad21-depleted cells than                                able to serve as template (3&#226;&#8364;&#8220;5). <br /> in controls in G2 but not in G1. Inhibition of the DNA                             During replication, the newly synthesized sister <br /> damage kinase DNA-PK, but not ATM, further                                      chromatids are tied together by the cohesin complex that <br /> inhibited foci loss in cohesin-depleted cells in G2.                            forms a ring around chromatids (6). It consists of Smc1, <br /> SMC1 depletion had no effect on DNA single-                                     Smc3, Scc1/Mcd1/Rad21 and Scc3/SA1/SA2 (7,8). The <br /> strand break repair in either G1 or late S/G2.                                  cohesin complex plays an important role in the &#239;&#172;&#65533;delity <br /> Rad21 and SMC1 were recruited to sites of X-ray-                                of sister chromatid separation and chromosome segrega- <br /> induced DNA damage in G2-phase cells, but not in                                tion during anaphase (9) but is also involved in other <br /> G1, and only when DNA damage was concentrated                                   aspects of chromosome metabolism. Cohesin is believed <br />                                                                                 to facilitate DNA repair by tethering sister chromatids. In <br /> in subnuclear stripes, generated by partially <br />                                                                                 yeast and human cells, proteins needed to load cohesin <br /> shielded ultrasoft X-rays. Our results suggest that                             onto chromosomes and generate cohesion during the <br /> the cohesin complex contributes to cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>survival</span> by                             S phase (Scc2, Eco1, sororin) are also shown to be <br /> promoting the repair of radiation-induced DNA                                   required for repair (10,11). Furthermore, cohesin is <br /> double-strand breaks in G2-phase cells in an ATM-                               recruited to chromatin regions surrounding an <br /> dependent pathway.                                                              enzymatically induced DSB in a gH2AX-dependent <br />                                                                                 manner in Saccharomyces cerevisiae (12,13). Interestingly, <br />                                                                                 recent &#239;&#172;&#65533;ndings suggest that one DSB induced <br /> INTRODUCTION                                                                    enzymatically in one S. cerevisiae chromosome results in <br /> DNA double-strand breaks (DSBs) are a major threat to                           increased sister chromatid cohesion of all chromosomes <br /> the genomic integrity of a cell. They can result in cell death                  (14,15). Cohesin is recruited to regions of laser scissor- <br /> if left un-repaired, or, if incorrectly repaired, can produce                   induced nuclear damage in mammalian cells (16), but <br /> chromosomal aberrations and are thought to induce                               only at very high power settings (17). <br />  <br /> *To whom correspondence should be addressed. Tel: +44-1235-822700; Fax: +44-1235-833891; Email: kai.rothkamm@hpa.org.uk <br />  <br /> &#195;&#376; The Author(s) 2009. Published by Oxford University Press. <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  478 Nucleic Acids Research, 2010, Vol. 38, No. 2 <br />  <br />  <br />    There is evidence that the securin&#226;&#8364;&#8220;separase complex has      Cell culture <br /> a DNA damage repair role in interphase by cleavage of <br />                                                                 HeLa cells were obtained <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> Cancer Research UK <br /> the Rad21 subunit (18,19). It is not yet clear if this occurs   Research Services. Cells were grown in Eagles minimum <br /> after damage to promote repair or whether it happens            essential medium with 10% foetal calf serum, 2 mm <br /> after repair to release the additional loaded cohesin           L-glutamine, 1 mm sodium pyruvate, 0.1x nonessential <br /> subunits.                                                       amino acids, 0.135% sodium bicarbonate, 100 U/ml pen- <br />    A number of studies have presented evidence for the          icillin, 0.1 mg/ml streptomycin with 90%N2, 5%O2, 5% <br /> involvement of cohesin in DSB repair in yeast and verte-        CO2 in a well-humidi&#239;&#172;&#65533;ed incubator at 37  C. <br /> brate cells. However, many of the previous studies were         Exponentially growing cultures with a fraction of  65% <br /> done with enzymatically induced breaks that di&#239;&#172;&#8364;er               G1 and  35% S/G2 cells were used for all experiments. <br /> signi&#239;&#172;&#65533;cantly in their chemical structure from radiation- <br /> induced ones. Also, these approaches tend to monitor            RNA interference <br /> very speci&#239;&#172;&#65533;c repair pathways and events at a speci&#239;&#172;&#65533;c <br /> genomic site. To date, two less selective approaches            SiRNA        transfections were    performed     using <br /> have been used to study the role of cohesin in DNA              DharmaFECT 1 (Dharmacon) and Oligofectamine <br /> repair. The &#239;&#172;&#65533;rst was based on pulsed &#239;&#172;&#65533;eld                       (Invitrogen), according to the suppliers&#226;&#8364;&#8482; instructions. <br /> gel electrophoresis of asynchronous S. pombe cells (20)         SMC1-1 (50 -GCA AUG CCC UUG UCU GUG AUU- <br /> or S. cerevisiae cells that had been chemically arrested in     30 ) and SMC1-3 (50 -GAA CAA AGA CAC AUG AAG <br /> prometaphase (10,12&#226;&#8364;&#8220;14). In the second approach chro-           AUU-30 ) siRNAs (Dharmacon) target SMC1. RAD21 <br /> mosome aberrations were analysed in mitotic vertebrate          (50 -AUA CCU UCU UGC AGA CUG U-30 ) siRNA <br /> cells following chemical synchronization in G1/S and            (MWG Biotech) targets the human RAD21 homologue <br /> gamma-irradiation in the late S phase (21). To avoid            (22). SiCo1 (50 -UAG CGA CUA AAC ACA UCA <br /> any interference from the G2/M checkpoint in an other-          A-30 ), siCo2 (50 -UAA GGC UAU GAA GAG AUA <br /> wise very similar approach, Schmitz et al. (11) treated cells   CUU-30 ) and lacZ (50 -CGU ACG CGG AAU ACU <br /> with ca&#239;&#172;&#8364;eine, a cell cycle checkpoint inhibitor that can        UCG A-30 ) are control siRNAs (Dharmacon). All <br /> block the key DNA damage response kinases ATM,                  siRNAs were used at a &#239;&#172;&#65533;nal concentration of 100 nM <br /> ATR and DNA-PKcs.                                               unless speci&#239;&#172;&#65533;ed otherwise. Cells were assayed 20&#226;&#8364;&#8220;24 h <br />    In contrast to previous studies, we wished to determine,     after transfection. <br /> in de&#239;&#172;&#65533;ned cell cycle stages without the use of chemical <br /> inhibitors, the role of cohesin in the repair of bulk DNA       Drug treatment <br /> damage induced by an environmentally and clinically             The KuDOS Pharmaceuticals agents ATMi/KU55 933 <br /> relevant agent, ionizing radiation. Using a number of           (23) and DNA-PKi/NU7026 (24,25) are potent inhibitors <br /> independent approaches, we have attempted to analyse            of the DNA damage response kinases ataxia telangiectasia <br /> DSB repair in SMC1 or Rad21-depleted human G1- and              mutated (ATM) and DNA-dependent protein kinase <br /> G2-phase cells whilst minimizing interference from check-       (DNA-PK), respectively. Inhibitors were added to the <br /> point functions or chemical agents.                             cells 1 h before irradiation to give a &#239;&#172;&#65533;nal concentration <br />                                                                 of 0.01 mM. Inhibitors were present during and after irra- <br />                                                                 diation. For one set of experiments, cells were incubated in <br /> MATERIALS AND METHODS                                           0.02 mM bromodeoxyuridine (Sigma Aldrich) immedi- <br />                                                                 ately before, during and after irradiation. <br /> Antibodies <br /> Rabbit polyclonal antibodies against SMC1, SA1, SA2             Irradiation <br /> and SMC3 were purchased from Bethyl Laboratories                Conventional X-ray exposures were performed at room <br /> (Universal Biologicals, Cambridge, UK), the mouse anti-         temperature with a 240 kV (constant potential) X-ray set <br /> human Cyclin A (clone 6E6) was from Vector                      (Pantak) at a dose rate of  1 Gy/min. For ultrasoft X-ray <br /> Laboratories (Peterborough, UK) and another rabbit              irradiations, cells were grown in a glass-walled dish with a <br /> polyclonal SMC1 antibody from Autogen Bioclear                  0.9 mm Mylar <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span>, through which the cells were <br /> (ABF021 190; used only in Figure 6C and D). Anti-actin          irradiated. The ultrasoft X-rays were produced using a <br /> antibodies and horseradish peroxidase-conjugated second-        cold cathode discharge tube (26) with an aluminium trans- <br /> ary antibodies were obtained from Sigma (Gillingham,            mission target to produce predominantly Al-K character- <br /> UK). Mouse monoclonal anti-Rad21 and anti-gH2AX                 istic X-rays (1.48 keV) with a discharge of 5 mA at a <br /> (clone JBW301) were from Millipore (Dundee, UK). Rat            potential of 5 kV. Masks for partial shielding were made <br /> anti-bromodeoxyuridine (BrdU) was from Cancer                   from 1 mm thick gold deposited on a 0.5 mm silicon nitride <br /> Research UK. The anti-CENP-F antibody (ab5) and                 membrane. The gold was deposited in 9 mm wide stripes <br /> anti-53BP1 antibody were obtained from Abcam                    separated by 1 mm. The average radiation dose per cell was <br /> (Cambridge, UK). AlexaFluor 488 and 532-conjugated              about 2.5 Gy with the majority of the energy and therefore <br /> secondary antibodies were purchased from Invitrogen             damage being deposited in 1 mm stripes across the cell <br /> (Paisley, UK) and all other secondary antibodies for            separated at 10 mm intervals. Following irradiation, cells <br /> immuno&#239;&#172;&#8218;uorescence microscopy were from Jackson                  were incubated in a well-humidi&#239;&#172;&#65533;ed incubator at 37  C <br /> ImmunoResearch (Stratech, Su&#239;&#172;&#8364;olk, UK).                          with 5% CO2. <br />                                                                            Nucleic Acids Research, 2010, Vol. 38, No. 2 479 <br />  <br />  <br /> Colony assay                                                    washed and incubated in TBST with 2.5% milk and <br />                                                                 primary antibody overnight at 4  C. They were then <br /> Cells were trypsinized, counted and serial dilutions <br />                                                                 washed and incubated with secondary antibodies <br /> in triplicates incubated for colony formation following <br />                                                                 [Qdot655-conjugated goat anti-chicken (Invitrogen) and <br /> X-ray treatment. Flasks were stained with crystal violet <br />                                                                 horseradish peroxidase-conjugated goat anti-rabbit <br /> and only colonies consisting of more than 50 cells were <br />                                                                 (Dako Cytomation)] for 1 h at room temperature, <br /> scored. <br />                                                                 washed several times and incubated with ECL Advance <br /> Micronucleus assay                                              detection reagent (Amersham BioSciences) for 5 min prior <br />                                                                 to scanning. Fluorescence and chemiluminescence imaging <br /> Cells were grown and siRNA-transfected in LabTek four-          was carried out using a Kodak ImageStation 4000MM <br /> well chamber slides. Thirty minutes post-irradiation, cells     system, and quanti&#239;&#172;&#65533;cation was performed using Kodak <br /> were treated with 2 mg/ml cytochalasin B for 4 h at 37  C,      MI software. The level of RNAi knockdown was deter- <br /> washed with phosphate-bu&#239;&#172;&#8364;ered saline and &#239;&#172;&#65533;xed for               mined using the level of actin as a standard. <br /> 10 min in ice-cold methanol. Dried slides were stained <br /> with Giemsa, washed in deionized water and analysed             Comet assay <br /> by bright &#239;&#172;&#65533;eld light microscopy using  400 magni&#239;&#172;&#65533;cation. <br />                                                                 Cells were X-irradiated on ice, incubated for 0, 1 or 2 h at <br /> Immuno&#239;&#172;&#8218;uorescence microscopy                                    37  C, trypsinized and 12 500 cells were embedded in <br />                                                                 agarose on glass slides. Lysis in 1.2 M NaCl, 0.1% <br /> Cells grown in LAB-TEK four-well chamber slides were            N-lauroyl sarcosine, 0.26 M NaOH, 100 mM Na2EDTA, <br /> washed in phosphate-bu&#239;&#172;&#8364;ered saline (PBS) and &#239;&#172;&#65533;xed for           pH &gt; 12.5 (all Sigma, Poole, UK) was performed for <br /> 10 min in either 100% methanol at  20  C or in 3.7%             18&#226;&#8364;&#8220;20 h at 4  C. Slides were rinsed three times in the <br /> formaldehyde at room temperature and permeabilized              0.03 M NaOH/2 mM Na2EDTA electrophoresis bu&#239;&#172;&#8364;er <br /> for 10 min with 0.25% Triton X-100. For ultrasoft X-ray         (pH &gt; 12.5) before starting alkaline electrophoresis at <br /> experiments cells grown in 0.9 mm Mylar glass dishes were       0.6 V/cm for 30 min. After electrophoresis, slides were <br /> treated with 0.1% TritonX-100 in PBS for 2 min at room          placed immediately in cold 70% ethanol followed by <br /> temperature to extract loosely bound proteins and then          90% and 100% ethanol for 10 min each. Slides were air <br /> &#239;&#172;&#65533;xed with formaldehyde as above. Bromodeoxyuridine-             dried, stained with Sybr&#226;&#8364;&#8220;Gold solution diluted 1: 10 000 <br /> treated cells were microwaved at 850 W for 5 min in             and covered with a coverslip using the DABCO antifade <br /> 10 mM citric acid (pH 6.0) to denature DNA. Cells were          mounting medium. The edges were sealed with nail <br /> blocked in PBS with 5% BSA for a minimum of 30 min at           varnish. Comet imaging and analysis was performed <br /> room temperature. All samples were incubated with               using an automated microscope system and analysis <br /> primary antibodies for 1 h at room temperature, washed          software developed by Vojnovic et al. (unpublished <br /> in PBS, 2% BSA for three times 5&#226;&#8364;&#8220;10 min and incubated           data). It calculates automatically various parameters <br /> with &#239;&#172;&#8218;uorophore-conjugated secondary antibodies for 1 h         including the percentage of DNA in the comet tail, tail <br /> at room temperature. Cells were counterstained with 4,6         <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> and tail moment for each cell analysed and gives <br /> diamidino-2-phenylindole (DAPI), washed in PBS and              the staining intensity of each individual cell that allows <br /> mounted. Fluorescence images were captured using a              attribution of the cell cycle phase based on the DNA <br /> Nikon Eclipse TE200 or 90i &#239;&#172;&#8218;uorescence microscope               content. 200&#226;&#8364;&#8220;500 G1-phase cells and 100&#226;&#8364;&#8220;300 S/G2-phase <br /> equipped with cooled charge-coupled device camera and           cells were analysed for each data point. <br /> acquisition software. Identical illumination and camera <br /> settings were used within each data set. For quantitative       Statistical analysis <br /> analysis, intensities of SMC1 in siRNA-treated cells were <br /> determined using the ImageJ software. 53BP1 and gH2AX           Standard errors of the mean were calculated by dividing <br /> foci were counted by eye during the imaging process using       the standard deviation of the mean of more than two <br /> a 100  objective. In at least two independent experiments       experiments by the square root of the number of <br /> for each data point, 40&#226;&#8364;&#8220;100 cells were analysed. Samples        experiments. The signi&#239;&#172;&#65533;cance of any observed e&#239;&#172;&#8364;ects was <br /> were coded to prevent any bias.                                 tested individually for each siRNA using the Student&#226;&#8364;&#8482;s <br />                                                                 t-test. In some cases, di&#239;&#172;&#8364;erences were only signi&#239;&#172;&#65533;cant for <br /> Western blotting                                                pooled analysis of both SMC1-targeting siRNAs. This is <br />                                                                 stated in the text, where applicable. <br /> Cell pellets were lysed in the ice-cold lysis bu&#239;&#172;&#8364;er (1 X TBS, <br /> 1% Triton X-100) containing protease inhibitor cocktail <br /> (Sigma, Gillingham, UK) and then sonicated for 1 min.           RESULTS <br /> Samples were &#239;&#172;&#65533;xed in the Laemmli sample bu&#239;&#172;&#8364;er and <br />                                                                 SMC1 knockdown cells are radiosensitive <br /> heated for 3 min at 100  C. Protein samples were separated <br /> on NuPage Novex 3&#226;&#8364;&#8220;8% Tris-Acetate gels (Invitrogen)             If the phosphorylation of SMC1 after DNA damage is <br /> according to the manufacturer&#226;&#8364;&#8482;s protocol. Proteins were         suppressed, cells show decreased survival, increased chro- <br /> then transferred onto Immobilon FL membranes in a               mosomal aberrations and a defect in the S-phase check- <br /> transfer bu&#239;&#172;&#8364;er for 60 min at 100 V. Membranes were              point (27,28). Here we wanted to investigate whether the <br /> blocked in Tris-bu&#239;&#172;&#8364;ered saline/0.1% Tween20 (TBST)              SMC1 protein itself is contributing to <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>cell survival</span> and <br /> containing 0.5% milk and 1% BSA for 1 h and then                DNA damage repair. To do so we established small <br />  480 Nucleic Acids Research, 2010, Vol. 38, No. 2 <br />  <br />  <br /> interfering RNA (siRNA) to speci&#239;&#172;&#65533;cally target the SMC1            A                     20 nM                         100 nM <br /> subunit of the cohesin complex. SMC1 expression was <br />  <br />  <br />  <br />  <br />                                                                                                                                            untreated <br />                                                                                                                         SMC1-1 <br />  <br />  <br />                                                                                                                                  SMC1-3 <br />                                                                                          SMC1-1 <br />  <br />  <br />                                                                                                      SMC1-3 <br /> depleted to  20% levels in HeLa cells by these two <br />  <br />  <br />  <br />  <br />                                                                                                               siCo1 <br />                                                                                 siCo1 <br /> targeted siRNAs (SMC1-1 and SMC1-3) but was <br /> not reduced in samples treated with control siRNA <br /> (Figure 1A). Depletion of SMC1 was homogenous in the                  SMC1 <br /> cell population (Figure 1B) and radiosensitized HeLa cells            &#206;&#178;-actin <br /> 2-fold at the 37% survival level, as measured by colony-                        112%     31%        41%       126      22%       19%      100% <br /> formation (Figure 1C) but had no signi&#239;&#172;&#65533;cant e&#239;&#172;&#8364;ect on the                                          SMC1 level (% Control) <br /> plating e&#239;&#172;&#402;ciency of un-irradiated cells (43 &#194;&#177; 6% for              B      Untreated           siCo1              SMC1-1               SMC1-3 <br /> SMC1 siRNA versus 44 &#194;&#177; 2% for control siRNA). <br /> These data show that the SMC1 protein is important for <br /> cell survival after X-irradiation. <br />  <br /> SMC1 promotes the repair of radiation-induced DSBs                      100&#194;&#177;28%      110&#194;&#177;22%          37&#194;&#177;20%         32&#194;&#177;19% <br /> in the G2 phase of the cell cycle                                           SMC1 level (% Control &#194;&#177; standard deviation) <br /> To determine whether SMC1 depletion a&#239;&#172;&#8364;ects the repair             C <br /> of ionizing radiation-induced DSBs, RNAi-treated <br /> exponentially growing HeLa cells were X-irradiated, <br /> incubated for repair, &#239;&#172;&#65533;xed and immunostained for the <br /> DSB marker gH2AX (Figure 2A). To distinguish cells in <br /> G1, S and G2 phase, cells were co-immunostained for <br /> CENP-F, a protein whose expression is tightly cell cycle- <br /> regulated (29,30). Radiation-induced gH2AX foci were <br /> scored in CENP-F-negative cells that were classi&#239;&#172;&#65533;ed as <br /> G1-phase cells and in cells with a high CENP-F staining <br /> intensity and interphase-like DAPI staining at the time of <br /> analysis that were classi&#239;&#172;&#65533;ed as late S/G2-phase cells. Cells <br /> with intermediate CENP-F staining were in early to mid-S <br /> phase at the time of analysis and were ignored as their cell <br /> cycle stage at the time of irradiation could have been <br /> either G1 or S. <br />    Depletion of SMC1 by RNAi had no signi&#239;&#172;&#65533;cant e&#239;&#172;&#8364;ect <br /> on the number of foci present in un-irradiated cells; <br /> however, late S/G2-phase cells contained about three <br /> times more foci per cell than G1-phase cells (1.47 &#194;&#177; 0.14      Figure 1. SMC1 depletion radiosensitizes HeLa cells. (A) Depletion of <br /> versus 0.47 &#194;&#177; 0.05), irrespective of the RNAi treatment        SMC1 in HeLa cells using RNA interference. Cells were transfected <br /> (Figure 2B). Also, 30 min after treatment with 1 Gy            with two di&#239;&#172;&#8364;erent siRNAs against SMC1 (SMC1-1 and SMC1-3), <br /> X-rays, foci levels were higher in late S/G2-phase cells       a non-targeting siRNA (siCo1) or untransfected. Protein levels <br /> (36.8 &#194;&#177; 0.8 per cell) than in G1-phase cells (22.5 &#194;&#177; 0.5       were analysed by western blotting. The percentage of SMC1 <br />                                                                levels with respect to the untransfected control is shown. <br /> per cell) but again very similar across the di&#239;&#172;&#8364;erent RNAi      (B) Immuno&#239;&#172;&#8218;uorescence microscopic analysis of SMC1 depletion. <br /> treatments. This di&#239;&#172;&#8364;erence likely re&#239;&#172;&#8218;ects the higher DNA       Details as in (A). (C) Colony formation assay for control and SMC1- <br /> content of late S/G2-phase cells compared to G1-phase          depleted cells. Cells were transfected with two di&#239;&#172;&#8364;erent siRNAs against <br /> cells. At the same radiation dose, a cell with twice the       SMC1 (SMC1-1 and SMC1-3) and two di&#239;&#172;&#8364;erent non-targeting siRNAs <br />                                                                (siCo1 and siCo2). Error bars are standard errors of the mean from <br /> amount of DNA would be expected to encounter twice             three experiments. <br /> the number of DSBs. Importantly, 2 and 4 h following <br /> X-irradiation signi&#239;&#172;&#65533;cantly higher levels of residual <br /> gH2AX foci were present in SMC1-depleted cells than in <br /> controls in the late S/G2 phase (P &lt; 0.005) while no dif-      Cells were immunostained for bromodeoxyuridine, the <br /> ference was observed for G1-phase cells (Figure 2B).           cell cycle-regulated protein Cyclin A and the DSB marker <br /> It should be noted that highly CENP-F-positive cells at        53BP1 (Figure 2C). To exclude any S-phase e&#239;&#172;&#8364;ects, only <br /> the time of analysis could have been at any stage between      bromodeoxyuridine-negative cells were analysed and clas- <br /> the mid-S phase and G2 at the time of irradiation, at least    si&#239;&#172;&#65533;ed into G1 (Cyclin A negative) and G2 (Cyclin A <br /> for the 4 h repair samples.                                    positive, interphase-like DAPI staining) phase cells. <br />    To con&#239;&#172;&#65533;rm this result using a di&#239;&#172;&#8364;erent combination of       Importantly, using this approach, cells could be selectively <br /> DNA damage and cell cycle markers and to determine             analysed that were in G2 at the time of irradiation and <br /> whether SMC1 promotes DSB repair even in non-                  remained there throughout repair incubation. The results <br /> replicating cells irradiated in G2, cells were labelled        (Figure 2D) demonstrate the presence of a signi&#239;&#172;&#65533;cantly <br /> with bromodeoxyuridine 30 min before X-irradiation             higher level of residual 53BP1 foci in SMC1-depleted <br /> and throughout the post-irradiation incubation period.         G2-phase cells 2 (P &lt; 0.03) and 4 h (P &lt; 0.001) after 1 Gy <br />                                                                                                        Nucleic Acids Research, 2010, Vol. 38, No. 2 481 <br />  <br />  <br />                                                                                                                                      Untransfected <br />             A     &#206;&#179;H2AX         CENP-F           Merged            B                                                                 siCo1 <br />                                                                                              G1                                      siCo2 <br />                                                                                       20 <br />  <br />  <br />  <br />  <br />                                                                         Foci / cell <br />                                                                                                                                      SMC1-1 <br />                                                                                                                                      SMC1-3 <br />                                                                                       10 <br />  <br />                                                                                        0 <br />                                                                                                0 Gy       1 Gy, 0.5 h    1 Gy, 2 h     1 Gy, 4 h <br />                                                                                       40 <br />  <br />  <br />  <br />  <br />                                                                       Foci / cell <br />                                                                                       30 <br />                                                                                             Late S/G2 <br />                                                                                       20 <br />             C          53BP1                   Cyclin A <br />                                                                                       10 <br />                                                                                        0 <br />                                                                                                0 Gy      1 Gy, 0.5 h    1 Gy, 2 h      1 Gy, 4 h <br />  <br />                                                                    D <br />  <br />  <br />  <br />  <br />                                                                     Foci / cell <br />                                                                                       20       G1 <br />  <br />                                                                                       10 <br />  <br />                                                                                        0 <br />                                                                                                0 Gy      1 Gy + 0.5 h   1 Gy + 2 h    1 Gy + 4 h <br />                                                                                       40 <br />                                                                                                G2 <br />                                                                     Foci / cell <br />  <br />  <br />                                                                                       30 <br />                                                                                       20 <br />                                                                                       10 <br />                                                                                        0 <br />                        BrdU                    Merged                                          0 Gy      1 Gy + 0.5 h   1 Gy + 2 h    1 Gy + 4 h <br />  <br /> Figure 2. SMC1 depletion impairs the repair of DSBs in G2-phase cells. (A) gH2AX and CENP-F staining in SMC1-depleted cells 2 h after 1 Gy <br /> X-irradiation. (B) Time course for the loss of 1 Gy X-ray-induced gH2AX foci in G1- (CENP-F negative) and late S/G2-phase cells (CENP-F <br /> strongly positive). (C) 53BP1, Cyclin A and BrdU staining in SMC1-depleted cells 2 h after 1 Gy X-irradiation. (D) Time course for the loss of 1 Gy <br /> X-ray-induced 53BP1 foci in G1 (BrdU-negative cyclin A-negative) and G2-phase cells (BrdU-negative cyclin A positive). Error bars are standard <br /> errors of the mean from two to three experiments. <br />  <br />  <br />  <br /> X-irradiation and thus con&#239;&#172;&#65533;rm the above &#239;&#172;&#65533;nding and                                         whether the observed de&#239;&#172;&#65533;cient repair of radiation-induced <br /> show that cohesin contributes to the repair of DSBs that                                   DNA breaks in SMC1-depleted G2-phase cells is <br /> were induced after replication. A similar e&#239;&#172;&#8364;ect was                                        associated with increased chromosome damage following <br /> observed when Rad21 was depleted (Figure 5), indicating                                    irradiation, siRNA-transfected cells were analysed for <br /> that the cohesin complex is required for e&#239;&#172;&#402;cient repair of                                 micronuclei (Figure 3A). The spindle poison cytochalasin <br /> DSBs induced by ionizing radiation in replicated                                           B arrests cells in cytokinesis in a binuclear stage and thus <br /> chromatin.                                                                                 enables the microscopic identi&#239;&#172;&#65533;cation of recent mitotic <br />    While 53BP1 and gH2AX are well established as surro-                                    cells. It was added 30 min after irradiation to allow clear- <br /> gate markers for DSBs (31&#226;&#8364;&#8220;33), they do not directly                                        ance of cells that had been in mitosis at the time of irra- <br /> measure the physical presence of a break. A physical                                       diation. Cells were incubated in cytochalasin B for only 4 h <br /> assay for DSB repair, like pulsed-&#239;&#172;&#65533;eld gel electrophoresis,                                to restrict the analysis to cells that were in late S/G2 at the <br /> would be useful to con&#239;&#172;&#65533;rm that the observed increase in                                    time of irradiation. Accordingly, the fraction of <br /> residual foci levels in G2-phase cohesin knock-down cells                                  binucleated cells was low in un-irradiated cultures and <br /> does indeed re&#239;&#172;&#8218;ect the presence of more residual breaks.                                   dropped further following 1 Gy X-irradiation (Figure <br /> To this end, we attempted a combination of siRNA                                           3B), likely due to radiation-induced activation of the <br /> transfection and double-thymidine block synchronization                                    G2/M checkpoint. Interestingly, SMC1 depletion <br /> followed by irradiation and pulsed-&#239;&#172;&#65533;eld gel electro-                                       resulted in an even lower fraction of binucleated cells in <br /> phoresis. However, we were unable to obtain a su&#239;&#172;&#402;ciently                                   un-irradiated and irradiated cultures (P &lt; 0.003 for pooled <br /> pure G2 cell population. By the time most cells had left the                               analysis of SMC1-1 and SMC1-3), indicating that SMC1 <br /> S phase, a considerable fraction had progressed to G1.                                     is required for e&#239;&#172;&#402;cient cell cycle progression from G2 <br />                                                                                            through to cytokinesis. The mitotic index was higher in <br /> SMC1 depletion a&#239;&#172;&#8364;ects progression through mitosis and                                      unirradiated SMC1-depleted cells than in controls, which <br /> causes elevated chromosome damage following irradiation                                    is consistent with the notion that SMC1-depleted cells <br /> Un-repaired or mis-repaired DSBs are the most important                                    spend more time in mitosis (Figure 3C). Four hours <br /> precursor lesion for chromosome damage. To determine                                       after irradiation with 1 Gy, however, a similar 50&#226;&#8364;&#8220;60% <br />  482 Nucleic Acids Research, 2010, Vol. 38, No. 2 <br />  <br />  <br />   A siRNA                                 24 h                 0.5 h <br />                                                                             Add <br />                                                                                         4h <br />                                                                                              Fix&amp;stain <br />                                                                                                          DSBs arising because of replication failures, for <br />                                                  Irradiation                                             example, when replication encounters nicks (35). To test <br />    transfection                                                        Cytochalasin B          cells <br />                                                                                                          whether depletion of SMC1 results in a general increase in <br />   B                              8                                                            0 Gy       DNA damage levels following irradiation, 4 Gy <br />                                                                                                          X-irradiated SMC1 knock-down cells were analysed with <br />            % binucleated cells <br />  <br />  <br />  <br />  <br />                                                                                               1 Gy <br />                                  6 <br />                                                                                                          the alkaline comet assay that detects both single- and <br />                                                                                                          double-strand breaks but, given the 40 : 1 prevalence of <br />                                  4                                                                       the former over the latter for sparsely ionizing radiation, <br />                                                                                                          mainly measures the induction and repair of single-strand <br />                                  2 <br />                                                                                                          breaks. Automated image acquisition and quantitative <br />                                                                                                          analysis enabled several hundred cells to be analysed per <br />                                  0 <br />                                                                                                          sample, and G1 and late S/G2 populations were &#226;&#8364;&#732;gated&#226;&#8364;&#8482; <br />                                          untreated        siCo2            SMC1-1        SMC1-3          based on the DNA content and analysed separately <br />   C                              5                                                                       (Figure 4A). Similar levels of initial breaks induced by <br />                                                                                                          4 Gy X-rays and a rapid decrease over time with similar <br />      prometa-metaphase <br />  <br />  <br />  <br />  <br />                                  4                                                                       residual levels in control and SMC1-depleted cells were <br />                                                                                                          observed in both G1 (Figure 4B) and late S/G2-phase <br />          % cells in <br />  <br />  <br />  <br />  <br />                                  3 <br />                                                                                                          populations (Figure 4C). <br />                                  2 <br />                                                                                                          E&#239;&#172;&#8364;ect of ATM and DNA-PK inhibition on <br />                                  1                                                                       radiation-induced foci loss in cohesin-depleted cells <br />                                  0                                                                       Following irradiation, SMC1 is phosphorylated on Ser <br />                                          untreated        siCo2            SMC1-1        SMC1-3          966 and Ser 957 by the DNA damage kinase ATM but <br />                                                                                                          not by DNA-PK (27,28). Furthermore, SMC3 is <br />   D                              0.6                                                                     phosphorylated at Ser 1083 by ATM [Bauerschmidt <br />                                  0.5                                                                     et al, unpublished data and (36)]. To test whether the <br />                                                                                                          observed role of SMC1 in the repair of radiation- <br />   binucleated cell <br />   Micronuclei per <br />  <br />  <br />  <br />  <br />                                  0.4 <br />                                                                                                          induced DSBs in replicated chromatin requires ATM or <br />                                  0.3                                                                     DNA-PK, we treated siRNA-transfected cells with the <br />                                  0.2 <br />                                                                                                          ATM inhibitor Ku55 933 (23) or the DNA-PK inhibitor <br />                                                                                                          Nu7026 (24), irradiated cells with 1 Gy X-rays and deter- <br />                                  0.1                                                                     mined residual 53BP1 foci in G1 (CENP-F negative) and <br />                                      0                                                                   late S/G2-phase cells (CENP-F strong positive, interphase <br />                                           untreated        siCo2           SMC1-1        SMC1-3          DAPI staining) by immuno&#239;&#172;&#8218;uorescence microscopy <br />                                                                                                          (Figure 5). In addition, siRNA targeting the cohesin <br /> Figure 3. SMC1 depletion a&#239;&#172;&#8364;ects progression through mitosis and                                          subunit Rad21 was used to test whether the e&#239;&#172;&#8364;ects <br /> enhances micronuclei induction. (A) Schematic representation of the                                      observed when knocking down SMC1 apply to the <br /> experimental protocol. (B) Percentage of binucleated cells, (C) mitotic <br /> index and (D) numbers of micronuclei per binucleated cell obtained for                                   cohesin complex in general. Rad21 expression was e&#239;&#172;&#402;- <br /> di&#239;&#172;&#8364;erent siRNA treatments following 0 or 1 Gy X-irradiation. Error                                       ciently knocked down to very low residual levels <br /> bars are standard deviations from two experiments. The following                                         (Figure 5E) using a previously published siRNA (9,22). <br /> numbers of cells were analysed in total for each data point (from left                                   Like SMC1 knock-down cells, Rad21-depleted cells <br /> to right): 3366, 4290, 3654, 4566, 4877, 5211, 4718 and 5926. <br />                                                                                                          contained higher levels of unresolved foci compared to <br />                                                                                                          control siRNA-treated cells when irradiated in late S/G2 <br /> reduction in the mitotic index was observed in both                                                      (P &lt; 0.001) but not after irradiation in G1 (Figure 5B <br /> controls and SMC1-depleted cells, suggesting that loss of                                                and D). <br /> SMC1 does not compromise the DNA damage-induced                                                             Inhibition of ATM or DNA-PK signi&#239;&#172;&#65533;cantly increased <br /> G2/M checkpoint arrest. SMC1 depletion had no e&#239;&#172;&#8364;ect                                                      the level of residual foci in siCo-treated cells (P &lt; 0.003) as <br /> on spontaneous levels of micronuclei in binucleated cells                                                well as in SMC1- (P &lt; 0.0002) or Rad21-depleted cells <br /> (Figure 3D). Following irradiation, signi&#239;&#172;&#65533;cantly higher                                                  (P &lt; 0.003) following irradiation in G1. In cells irradiated <br /> levels of micronuclei per binucleated cell were observed                                                 in G2, however, ATM inhibition caused higher levels of <br /> in SMC1 knock-down cells than in un-transfected or                                                       residual foci only in control siRNA-treated cells <br /> control siRNA-transfected cells (P &lt; 0.04), indicating                                                   (P &lt; 0.001) but did not have any additional e&#239;&#172;&#8364;ect on <br /> that SMC1 promotes chromosome integrity.                                                                 foci loss in SMC1- or Rad21-depleted cells. These <br />                                                                                                          &#239;&#172;&#65533;ndings indicate that cohesin and ATM may function <br />                                                                                                          together in the same pathway(s) that facilitate the e&#239;&#172;&#402;cient <br /> The formation and repair of DNA single-strand breaks in <br />                                                                                                          repair of radiation-induced DSBs in the G2 phase of the <br /> irradiated cells is not a&#239;&#172;&#8364;ected by SMC1 depletion <br />                                                                                                          cell cycle. Treatment with the DNA-PK inhibitor, on the <br /> SMC1 is required for the ionizing radiation-induced                                                      other hand, resulted in higher residual foci levels in both <br /> S-phase checkpoint (27,28,34) and cohesin has previously                                                 control (P &lt; 0.00001) and SMC1 siRNA-transfected cells <br /> been implicated in sister chromatid-dependent repair of                                                  (P &lt; 0.00001) (Figure 5B and D). These data suggest that <br />                                                                                                                 Nucleic Acids Research, 2010, Vol. 38, No. 2 483 <br />  <br />  <br />  <br /> Figure 4. Comet analysis of DNA strand break induction and repair. <br /> (A) Percentage of DNA in tail versus total DNA signal in a 4 Gy                                               10 <br /> X-irradiated sample. The areas gated for the analysis of G1 and late <br /> S/G2-phase cells are indicated. (B and C) Time course for the percent- <br /> age of damage remaining in G1 and late S/G2-phase cells following <br /> 4 Gy X-irradiation. Error bars are standard deviations from two                                               5 <br /> experiments. <br />  <br />                                                                                                               0 <br /> DNA-PK-independent repair pathways bene&#239;&#172;&#65533;t from the                                                                                                               1 Gy + 4 h, G2 <br /> presence of cohesin in G2-phase cells. <br />                                                                                                                                      E                             lacZ Rad21 <br />  <br />                                                                                                                                              Rad21 <br /> Recruitment of cohesin to radiation-induced DNA double <br /> strand breaks <br />                                                                                                                                              &#206;&#178;-actin <br /> As no recruitment of cohesin subunits to sites of isolated <br /> DSBs induced by conventional X-rays was observed (data <br /> not shown), we used partially shielded soft X-rays (37) to                               Figure 5. Contribution of DNA damage kinases and/or the cohesin <br /> generate microbeam tracks and thereby induce localized                                   factors SMC1 and Rad21 to 53BP1 foci loss in G1 and late S/G2- <br /> DNA damage in 1 mm wide &#226;&#8364;&#732;stripes&#226;&#8364;&#8482; (Figure 6A). After                                     phase cells. 1 Gy X-ray-induced 53BP1 foci were scored in G1- <br />                                                                                          (CENP-F negative) and late S/G2-phase cells (CENP-F strongly <br /> extraction of unbound proteins immuno&#239;&#172;&#8218;uorescence                                         positive) 4 h after 1 Gy X-irradiation. (A and B) Un-irradiated and <br /> microscopy was performed for all components of the                                       irradiated G1-phase cells. (C and D) Un-irradiated and irradiated late <br /> cohesin complex. SMC1, SMC3, SA1 and SA2 did not                                         S/G2-phase cells. Error bars are standard deviations from two <br /> form microscopically visible stripes (Figure 6B).                                        experiments. (E) Depletion of Rad21 using RNA interference. Cells <br />                                                                                          were transfected with siRNAs against Rad21 or non-targeting <br /> However, a possible recruitment of Rad21 to X-ray-                                       siRNA (lacZ). Protein levels were analysed by western blotting. <br /> induced DNA damage was observed in a subset of cells                                     Rad21-depleted cells contained less than 2% of the levels found in <br /> that showed colocalization of Rad21 and 53BP1 stripes.                                   control cells. <br /> Analysis of cells triple-stained for Rad21, CENP-F and <br /> 53BP1 revealed that Rad21 stripes were only present and <br />  484 Nucleic Acids Research, 2010, Vol. 38, No. 2 <br />  <br />  <br />           A              Cells             Mask                          C <br />                                                                           G1 <br />  <br />  <br />                     ultra-soft X-rays <br />  <br />           B                                                              G2 <br />  <br />           53BP1                                                 SA1 <br />                                                                                                 DAPI                            Rad21     Merged <br />                                                                                                          53BP1      CENP-F <br />                                                                                                                                 SMC1 <br />  <br />  <br />                                                                          G1 <br />           53BP1                                                 SMC1 <br />  <br />  <br />  <br />                                                                          G2 <br />           53BP1                                                 Rad21 <br />  <br />  <br />                                                                          D                             G1/early S       mid S           late S/G2 <br />  <br />  <br />           53BP1                                                 SA2      colocalising stripes   100       SMC1 <br />                                                                              % cells with <br />  <br />                                                                                                           Rad21 <br />                                                                                                  50 <br />  <br />           &#206;&#179;H2AX                                                 SMC3                              0 <br />                                                                                                         no/weak        medium            strong <br />                                                                                                                     CENP-F staining <br />  <br />           E                  Rad21                   Tracker                                           DAPI                  Merged <br />  <br />  <br />  <br />  <br /> Figure 6. Recruitment of cohesin to ionizing radiation-induced DNA double strand breaks. (A) Schematic representation of micro-irradiation setup <br /> (left) and bright &#239;&#172;&#65533;eld image of the grid with 1 mm wide gaps/9 mm wide shielding used for generating de&#239;&#172;&#65533;ned patterns of DNA damage within the <br /> nucleus (right). (B) HeLa cells were grown on Mylar foil and irradiated with soft X-rays. After 1 h incubation, cells were pre-extracted with 0.1% <br /> Triton X-100, &#239;&#172;&#65533;xed and stained with the indicated antibodies. Each image is 125 mm wide. (C) As in (B) but cells were furthermore stained with <br /> CENP-F to determine the cell cycle phase. Each image is 15 mm wide. (D) Quantitative analysis of Rad21 and SMC1 stripe formation. Cells were <br /> divided into three groups of no or weak, medium and strong CENP-F staining. Microbeam tracks were predicted using the Rad21 or SMC1 signal, <br /> respectively, and then tested for co-localization with the 53BP1 signal. (E) Immuno&#239;&#172;&#8218;uorescence microscopic analysis of Rad21 depletion. Control <br /> siRNA-transfected cells labelled with CellTrackerTM Green CMFDA (Invitrogen) were mixed with Rad21 siRNA-treated cells and stained for <br /> Rad21. <br />  <br />  <br />  <br /> colocalized with 53BP1 in CENP-F positive, i.e. late S/G2-                        siRNA-treated Tracker-negative cells while Tracker- <br /> phase cells but not in CENP-F-negative G1-phase cells                             positive control siRNA-treated cells showed strong <br /> (Figure 6C and D). This indicates that Rad21 is                                   Rad21 staining (Figure 6E). This result con&#239;&#172;&#65533;rms that the <br /> preferentially recruited to sites of X-ray-induced DNA                            Rad21 &#226;&#8364;&#732;stripes&#226;&#8364;&#8482; shown in panels B and C are speci&#239;&#172;&#65533;c. <br /> damage in late S/G2-phase cells. Similar results were <br /> obtained for SMC1, using a di&#239;&#172;&#8364;erent antibody (Autogen <br /> Bioclear) from the one used in Figure 6B (Bethyl                                  DISCUSSION <br /> Laboratories), albeit with weaker stripe formation than                           This study shows for the &#239;&#172;&#65533;rst time that cohesin promotes <br /> seen for Rad21 (Figure 6C and D). To determine                                    the repair of DSBs induced by a non-lethal radiation dose <br /> whether the immuno&#239;&#172;&#8218;uorescence signal of the Rad21                                 in human G2-phase cells but not in G1. A function for <br /> antibody is speci&#239;&#172;&#65533;c, control siRNA-transfected cells                              cohesin in promoting the repair of ionizing radiation- <br /> labelled with CellTrackerTM Green CMFDA were mixed                                induced DSBs in replicated chromatin provides an attrac- <br /> with Rad21 siRNA-transfected cells and stained for                                tive explanation for the observation in a number of cell <br /> Rad21. No Rad21 staining was observed in Rad21                                    systems and using a variety of assays that the repair of <br />                                                                           Nucleic Acids Research, 2010, Vol. 38, No. 2 485 <br />  <br />  <br /> radiation-induced DSBs, especially in end-joining                  The results presented here suggest that constitutive or <br /> mutants, is enhanced in G2 (3,38&#226;&#8364;&#8220;41). Studies in                DNA damage-induced sister chromatid cohesion, <br /> S. cerevisiae using HO endonuclease and in human cells          facilitated by the cohesin complex, promotes the repair <br /> using I-SceI endonuclease to enzymatically induce a DSB         of ionizing radiation-induced DSBs. Our observation of <br /> at a de&#239;&#172;&#65533;ned site have shown that cohesin is required for        Rad21 recruitment to sub-nuclear regions damaged by <br /> e&#239;&#172;&#402;cient utilization of homologous recombination for             intense X-ray microbeams in G2 but not G1-phase cells <br /> sister chromatid-dependent DNA repair but not                   (Figure 6C and D) supports the notion of an active role of <br /> for intrachromosomal gene conversion (13,42). Whether           the cohesin complex in the DNA damage response that is <br /> this role of cohesin is also important for the repair of        unique to the G2 phase of the cell cycle. It is consistent <br /> ionizing radiation-induced DNA breaks in humans,                with previous &#239;&#172;&#65533;ndings using a high power laser. <br /> where homologous recombination plays a less dominant            Importantly, however, it rules out potential artefacts <br /> role, is not clear. Homologous recombination would              caused by the immense heat produced by dissection <br /> intuitively be regarded as the obvious candidate, and the       lasers that causes denaturation of DNA and proteins <br /> reported S/G2-speci&#239;&#172;&#65533;c radiosensitivity, increased yields of     and leaves microscopically visible burn-marks in cells <br /> radiation-induced chromosome aberrations and DSB                (16). In contrast, the &#226;&#8364;&#732;stripes&#226;&#8364;&#8482; induced here contained the <br /> repair de&#239;&#172;&#65533;ciency of cell lines de&#239;&#172;&#65533;cient in homologous           same types and amounts of damage concentrated in a <br /> recombination (3,4) support this notion. However, a con-        1 mm wide stripe that are induced in a cell during homog- <br /> tribution of homologous recombination to the repair of          enous exposure to 2.5 Gy ultra-soft X-rays. While these <br /> ionizing radiation-induced DSBs was not detected in a           &#239;&#172;&#65533;ndings clearly implicate damage-induced cohesin <br /> number of studies (40,41,43,44). Notably, these studies         recruitment in the radiation response, it is not yet estab- <br /> used pulsed-&#239;&#172;&#65533;eld gel electrophoresis to quantify DSBs           lished whether e&#239;&#172;&#402;cient repair of radiation-induced DSBs <br /> and, because of the limited sensitivity of this technique,      in replicated chromatin does indeed require additional <br /> had to be performed at supra-lethal doses of at least           damage-induced cohesion to improve local pairing of <br /> 10 Gy. The contradictory results regarding the importance       damaged and intact sister chromatid or whether it <br /> of homologous recombination for the repair of ionizing          operates already more e&#239;&#172;&#402;ciently at a constitutive level of <br /> radiation-induced DSBs can potentially be reconciled by         cohesion that is established during the S-phase. It is also <br /> the assumption that homologous recombination cannot             currently not clear why only recruitment of Rad21 and <br /> e&#239;&#172;&#402;ciently process hundreds of DSBs induced by high              SMC1, but not of SMC3 or SA1/2, was observed in this <br /> doses, but contributes signi&#239;&#172;&#65533;cantly to DSB repair at            study. Given that Rad21 and SMC1 recruitment was only <br /> more physiological levels of DNA damage.                        just detectable, despite using highly speci&#239;&#172;&#65533;c antibodies <br />    Figure 1 suggests that SMC1 depletion only moderately        (Figure 6E), co-localization of the other cohesin factors <br /> radiosensitizes HeLa cells. However, two points have to be      may not have been observed because of a lower speci&#239;&#172;&#65533;city <br /> considered in this context. (i) The knock-down e&#239;&#172;&#402;ciency         of the antibodies used. Use of &#239;&#172;&#8218;uorescent protein fusion <br /> of the siRNA transfections performed here was only about        constructs, as employed by Bekker-Jensen et al. (2006) for <br /> 80%. One could therefore assume that considerable levels        recruitment to laser damage, may help resolve this issue. <br /> of SMC1 may have been present in a subset of siRNA-                As SMC1 is involved in the damage-induced intra- <br /> treated cells that would thus have had a survival advan-        S-phase checkpoint and cohesin contributes to the repair <br /> tage and may have determined the shape of the survival          of replication-associated DSBs arising at single-stranded <br /> curve at high doses. To address this point, SMC1                DNA lesions, it was important to consider and, ideally, <br /> intensities of individual control and SMC1-depleted             exclude any S-phase e&#239;&#172;&#8364;ects in order to con&#239;&#172;&#65533;rm a <br /> cells were quanti&#239;&#172;&#65533;ed in immuno&#239;&#172;&#8218;uorescence images                replication-independent function of cohesin in radiation- <br /> (Figure 1B). The standard deviations obtained were very         induced DSB repair. To this end, the alkaline comet assay <br /> similar for all samples, suggesting that there was no large     was used to verify that DNA single-strand break induc- <br /> variation in residual SMC1 protein levels in the knock-         tion and repair was una&#239;&#172;&#8364;ected by SMC1 knock-down <br /> down samples. Overall depletion appeared to be even             (Figure 4). Any de&#239;&#172;&#65533;ciency in DSB repair would remain <br /> less than 80% but quanti&#239;&#172;&#65533;cation of immuno&#239;&#172;&#8218;uorescence            undetected with this assay because DSBs constitute less <br /> images may well have been skewed by background &#239;&#172;&#8218;uo-             than 3% of all breaks induced by X-rays. Also, the <br /> rescence and some non-speci&#239;&#172;&#65533;c binding of the SMC1               &#239;&#172;&#65533;nding of cohesin-dependent enhancement of DSB <br /> antibody. (ii) One would expect that SMC1 depletion             repair was con&#239;&#172;&#65533;rmed using an assay that excludes cells <br /> only radiosensitizes cells in the S and G2 phase of the         in the S-phase during or after irradiation from the <br /> cell cycle. The presence of more than 60% G1-phase              analysis. This was achieved by irradiation and incubation <br /> cells in the exponentially growing cultures used for the        of cells for repair in the presence of bromodeoxyuridine <br /> colony assays may well partly mask the radiosensitizing         and subsequent elimination of all bromodeoxyuridine- <br /> e&#239;&#172;&#8364;ect of SMC1 loss in S/G2. Comparison of colony for-           positive cells from the analysis (Figure 2C and D). <br /> mation in cells irradiated in G1 versus S versus G2 phase       SMC1 has not only been found to be part of cohesin <br /> would not only address this point but would also help           but is also a subunit of recombination complex 1 (RC-1) <br /> dissect SMC1&#226;&#8364;&#8482;s functions in the intra-S-phase checkpoint        (45). Our &#239;&#172;&#65533;nding that Rad21 depletion caused phenotypes <br /> (27,36) and DNA repair in G2. We have tried to use a            similar to the ones seen after SMC1 depletion suggests <br /> &#239;&#172;&#8218;uorescence-activated cell sorter to selectively plate S- and   that the observed e&#239;&#172;&#8364;ects are due to cohesin inactivation. <br /> G2-phase cells for colony formation but failed to obtain           The observation in previous studies that homologous <br /> any statistically sound results.                                recombination-de&#239;&#172;&#65533;cient cells as well as non-homologous <br />  486 Nucleic Acids Research, 2010, Vol. 38, No. 2 <br />  <br />  <br /> end-joining-de&#239;&#172;&#65533;cient cells show enhanced DSB rejoining        <br /> in G2 even following exposure to high radiation doses, <br />                                                         <br /> where homologous recombination would likely be                 <br /> unable to contribute e&#239;&#172;&#402;ciently to DSB repair, may           <br /> indicate that both DNA-PK-dependent end-joining and            <br /> a previously described DNA-PK-independent backup             <br /> end-joining pathway (46) may also bene&#239;&#172;&#65533;t from damage-         <br /> induced cohesion. One could speculate that either end-           <br /> joining process would require the alignment of broken         <br /> DSB ends that may be easier to facilitate if both ends        <br /> are tightly bound by the cohesin complex to the intact         <br /> sister chromatid that would thus act as a sca&#239;&#172;&#8364;old to            <br /> indirectly promote repair. Figure 5 suggests that        <br /> cohesin-dependent enhancement of repair in the G2              <br /> phase occurs with and without DNA-PK inhibition but        <br /> it appears to depend on functional ATM. This &#239;&#172;&#65533;nding            <br /> suggests that cohesin may promote DSB repair via at             <br /> least one additional pathway to DNA-PK-dependent            <br /> end-joining but, given that both ATM and DNA-PK                 <br /> in&#239;&#172;&#8218;uence both homologous recombination and end-              <br /> joining pathways (47), these data do not clarify which    <br /> one or both of the discussed pathways are involved.            <br /> Also, more detailed analysis of DSB repair kinetics          <br /> would be required to better clarify the interaction of          <br /> cohesin with ATM and DNA-PK and its contribution to           <br /> speci&#239;&#172;&#65533;c repair pathways. The ATM and DNA-PK              <br /> inhibitors are highly selective and e&#239;&#172;&#8364;ective at the            <br /> concentrations used (23,24). They are being widely used      <br /> and no non-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects have been reported to date.        <br /> Given that, in addition, Rad21 knock-down was very e&#239;&#172;&#402;-        <br /> cient (Figure 5E) and similar results were obtained for          <br /> Rad21 and SMC1, it is therefore unlikely that the         <br /> observed e&#239;&#172;&#8364;ects are due to combinations of partial              <br /> inhibitions and/or o&#239;&#172;&#8364;-target e&#239;&#172;&#8364;ects.                         <br />    Notably, the moderate size of the e&#239;&#172;&#8364;ect of cohesin          <br /> depletion on DSB repair observed here in G2-phase cells          <br /> is similar to that observed for a number of factors,     <br /> including ATM, Artemis, NBS1, H2AX, 53BP1 and               <br /> others observed in G1-phase cells (48). Our &#239;&#172;&#65533;nding that         <br /> cohesin-dependent enhancement of DSB repair in G2             <br /> requires ATM (Figure 5) indicates that they may act in        <br /> the same pathway in this cell cycle phase. Additional work       <br /> is required, however, to determine whether there is indeed       <br /> a functional link between these observations.                <br /> </body></html>