<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Abnormal neurotransmission in mice lacking synaptic <br /> vesicle protein 2A (SV2A) <br />  <br /> Synaptic vesicle protein 2 (SV2) is a membrane glycoprotein com-              Methods <br /> mon <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> all synaptic and endocrine vesicles. Unlike many proteins              Generation of SV2A-Minus Mice. A portion of the SV2A gene was <br /> involved in synaptic exocytosis, SV2 has no homolog in yeast,                 isolated <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> a mouse 129SV genomic library (Stratagene) by <br /> indicating that it performs a function unique to secretion in higher          screening with a probe encoding bases &#226;&#171;&#186;400 to &#226;&#171;&#185;212 of the rat <br /> eukaryotes. Although the structure and protein interactions of SV2            SV2A cDNA (3). A fragment of &#226;&#172;&#8225;12.5 kb was isolated that <br /> suggest multiple possible functions, its role in synaptic events              contained the exon encoding the translation start site through <br /> remains unknown. To explore the function of SV2 in an in vivo                 most of the first transmembrane domain of the SV2A cDNA. <br /> context, we generated mice that do not express the primary SV2                Mouse SV2A sequence was approximately 95% identical to rat <br /> isoform, SV2A, by using targeted gene disruption. Animals ho-                 in this region. The location of the exon in the genomic fragment <br /> mozygous for the SV2A gene disruption appear normal at birth.                 was identified by restriction enzyme mapping and Southern <br /> However, they fail to grow, experience severe seizures, and die               analysis. A targeting construct was generated in pBlueScript in <br /> within 3 weeks, suggesting multiple neural and endocrine deficits.            which the exon and surrounding DNA were replaced with a gene <br /> Electrophysiological studies of spontaneous inhibitory neurotrans-            encoding neomycin resistance as illustrated in Fig. 1. DNA <br /> mission in the CA3 region of the hippocampus revealed that loss of            encoding thymidine kinase was placed at the end of the short arm <br /> SV2A leads to a reduction in action potential-dependent &#226;&#65533;&#165;-ami-                of the targeting construct to allow for negative selection of <br /> nobutyric acid (GABA)ergic neurotransmission. In contrast, action             homologous recombination by using the antiviral agent Gancy- <br /> potential-independent neurotransmission was normal. Analyses of               clovir (Syntex, Palo Alto, CA). Embryonic stem cells, a gift from <br /> synapse ultrastructure suggest that altered neurotransmission is              Rejean Idzerda and G. Stanley McKnight (Univ. of Washing- <br /> not caused by changes in synapse density or morphology. These                 ton), were transfected with linearized targeting construct by <br /> findings demonstrate that SV2A is an essential protein and impli-             electroporation. Cells were placed under selection in G418 <br /> cate it in the control of exocytosis.                                         (GIBCO&#229;&#8230;&#190;BRL) and Gancyclovir. Resistant colonies were <br />                                                                               screened for homologous recombination by Southern analysis of <br />                                                                               HindIII-digested genomic DNA by using the probe depicted in <br /> T   he release of neurotransmitters is mediated by a secretory <br />     cycle that shares many features with general membrane <br /> trafficking. Work from several disciplines is beginning to identify <br />                                                                               Fig. 1. Four cell lines carrying the SV2A disruption were injected <br />                                                                               into C57BL&#229;&#8230;&#190;6 blastocysts and implanted into pseudopregnant <br />                                                                               females. One of ten chimeric males produced offspring heterozy- <br /> a sequence of protein&#226;&#8364;&#8220;protein and protein&#226;&#8364;&#8220;lipid interactions that             gous for the SV2A gene disruption. These offspring were bred <br /> mediate the formation, targeting, and fusion of transport vesicles            with each other to produce animals homozygous for the muta- <br /> (1). However, neural secretion has several unique features,                   tion. Wild-type offspring of heterozygotes were used to establish <br /> including the mechanism of vesicle filling, the tight coupling of             a colony of genetically matched control animals. All animals <br /> vesicle exocytosis to the influx of calcium, and the strict local-            used in these studies were genotyped by Southern analysis of <br /> ization of vesicle fusion to special &#226;&#8364;&#732;&#226;&#8364;&#732;active&#226;&#8364;&#8482;&#226;&#8364;&#8482; zones. These features         DNA isolated from tail clippings. <br /> are likely to be mediated by proteins specific to neurons and cells <br /> that display regulated secretion.                                             Analyses of Protein Expression. Protein expression was determined <br />    Synaptic vesicle protein 2 (SV2) was one of the first proteins             by immunoblot analysis of 1% Triton X-100 extracts or post- <br /> to be localized to synaptic vesicles (2). Three isoforms have been            nuclear supernatants of whole brain obtained from littermate <br /> identified, referred to as SV2A, SV2B, and SV2C (3&#226;&#8364;&#8220;6). The                    pups of heterozygote crosses. Equal amounts of protein from <br /> absence of SV2 homologs in yeast suggests that it is not a                    wild-type, heterozygous, and homozygous littermates were sep- <br /> component of the basic membrane trafficking mechanism and,                    arated by SDS&#229;&#8230;&#190;PAGE and transferred to nitrocellulose. Proteins <br /> therefore, may mediate a process unique to regulated secretion.               were stained with Ponceau Red to verify equivalent loading <br /> The structure and protein interactions of SV2 suggest multiple                between lanes. Blots were probed for the indicated protein. <br /> possible functions, which include acting as a transporter (3&#226;&#8364;&#8220;5),              Antibody binding was detected by chemiluminescence. <br /> providing a matrix for neurotransmitter concentration and re- <br /> lease (7), acting as a receptor for extracellular matrix proteins             Light Microscopic Analyses. Gross brain morphology was assessed <br /> (8), and regulating the action of other synaptic proteins (9). To             in 10-&#226;&#65533;&#174;m coronal sections from age-matched pups. Mice were <br /> explore the function(s) of SV2 in an in vivo context, we used the <br /> technique of targeted gene disruption to produce animals that <br /> do not express SV2A. Analyses of these animals were guided by                 Abbreviations: SV2, synaptic vesicle protein 2; Pn, postnatal day n; GABA, &#226;&#65533;&#165;-aminobutyric <br /> previous studies of SV2 isoform expression (10). In this paper,               acid; IPSCs, inhibitory postsynaptic currents; mIPSCs, miniature IPSCs; sIPSCs, spontaneous <br />                                                                               IPSCs; CNS, central nervous system; APV, 2-amino-5-phosphonovaleric acid; CNQX, 6-cyano- <br /> we report that SV2A is essential for <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>survival</span> and for normal                  7-nitroquinoxaline-2,3-dione. <br /> nervous system functioning. Electrophysiological and morpho-                  &#226;&#8364;&#160;K.M.C.,   J.M.G., and T.A.J. contributed equally to this work. <br /> logical studies of synapse function and structure suggest that loss           &#194;&#182;To whom   reprint requests should be addressed at: Department of Pharmacology, Univer- <br /> of SV2A leads to a reduction in action potential-dependent                     sity of Washington, Box 357280, Seattle, WA 98195. E-mail: bajjalie@u.washington.edu. <br /> &#226;&#65533;&#165;-aminobutyric acid (GABA)ergic neurotransmission and that                    The publication costs of this article were defrayed in part by page charge payment. This <br /> this decrease is not the result of changes in synapse number or               article must therefore be hereby marked &#226;&#8364;&#339;advertisement&#226;&#8364;&#65533; in accordance with 18 U.S.C. <br /> structure.                                                                    &#194;&#167;1734 solely to indicate this fact. <br />  <br />  <br />  <br /> 15268 &#226;&#8364;&#8220;15273 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; December 21, 1999 &#229;&#8230;&#169; vol. 96 &#229;&#8230;&#169; no. 26 <br />                                                                                     the micrograph. Intersections between a test line and the <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> <br />                                                                                    of synaptic densities directly apposing a presynaptic bouton were <br />                                                                                    counted. The length of postsynaptic densities per synapse (estL&#229;&#8230;&#190; <br />                                                                                    synapse) was estimated by using the formula: estL&#229;&#8230;&#190;synapse &#226;&#171;&#189; <br />                                                                                    (2&#228;&#161;&#160;I&#228;&#161;&#160;a&#229;&#8230;&#190;l)&#229;&#8230;&#190;synapses, where I is the total number of intersections <br />                                                                                    between synaptic densities and test lines, a&#229;&#8230;&#190;l is the area repre- <br />                                                                                    sented by a test line (0.058 &#226;&#65533;&#174;m2&#229;&#8230;&#190;&#226;&#65533;&#174;m, adjusted for magnification), <br />                                                                                    and &#226;&#8364;&#732;&#226;&#8364;&#732;synapses&#226;&#8364;&#8482;&#226;&#8364;&#8482; is the number of synapses included in the sample. <br />                                                                                    Synaptic density thickness was estimated by measuring each <br />                                                                                    density at its widest region. This measurement was converted <br />                                                                                    based on rulers photographed at the same magnification as the <br />                                                                                    tissue. Similarly, synaptic vesicle diameter was approximated by <br />                                                                                    measuring up to 10 vesicles in randomly selected synapses and <br />                                                                                    correcting for magnification. All measures were done with <br />                                                                                    mixed, coded micrographs to control for experimenter bias. <br />  <br />                                                                                    Physiological Analyses. Whole-cell voltage clamp recordings were <br />                                                                                    obtained from CA3 pyramidal neurons in mouse hippocampal <br />                                                                                    slices. Most of the mice were ages postnatal day 9 (P9)&#226;&#8364;&#8220;10 (8 of <br />                                                                                    11 wild type and 6 of 7 knockouts), with the rest being P11&#226;&#8364;&#8220;13. <br />                                                                                    Mice were decapitated, and their brains were quickly removed, <br />                                                                                    placed in ice-cold cutting buffer (220 mM sucrose&#229;&#8230;&#190;3 mM KCl&#229;&#8230;&#190;4 <br />                                                                                    mM MgCl2&#229;&#8230;&#190;1.25 mM NaH2PO4&#229;&#8230;&#190;26 mM NaHCO3&#229;&#8230;&#190;10 mM D- <br />                                                                                    glucose), and equilibrated with 95% O2&#229;&#8230;&#190;5% CO2. Transverse <br />                                                                                    300-&#226;&#65533;&#174;m slices were cut and immediately placed in a holding <br />                                                                                    chamber with oxygenated artificial cerebrospinal fluid buffer <br /> Fig. 1. Targeted disruption of the SV2A gene. (A) Strategy for generating <br />                                                                                    (128 mM NaCl&#229;&#8230;&#190;3 mM KCl&#229;&#8230;&#190;2 mM CaCl2&#229;&#8230;&#190;2 mM MgCl2&#229;&#8230;&#190;1.25 mM <br /> SV2A-minus mice. A portion of the SV2A gene was isolated by screening a            NaH2PO4&#229;&#8230;&#190;26 mM NaHCO3&#229;&#8230;&#190;10 mM D-glucose). Recording pi- <br /> mouse 129SV genomic DNA library with a probe corresponding to the 5&#226;&#172;&#732;-end           pettes were filled with an internal recording solution (130 mM <br /> of the rat SV2A cDNA. The exon containing the translation start site is depicted   CsCl&#229;&#8230;&#190;1 mM EGTA&#229;&#8230;&#190;0.5 mM CaCl2&#229;&#8230;&#190;2 mM MgCl2&#229;&#8230;&#190;2 mM ATP- <br /> as a black box. A targeting construct was generated in which this exon and         Mg&#229;&#8230;&#190;10 mM Hepes&#229;&#8230;&#190;1.5 mM QX-314, pH adjusted to 7.2&#226;&#8364;&#8220;7.4 with <br /> surrounding DNA were replaced with a gene encoding neomycin resistance.            CsOH). The resulting pipet resistance was 4&#226;&#8364;&#8220;6 M&#226;&#65533;&#8364;. Recordings <br /> To allow for negative selection, a gene encoding thymidine kinase was placed       were done at room temperature in a submersion chamber while <br /> at the end of the short arm of the construct. Homologous recombination of          the cell was held at &#226;&#171;&#186;70 mV. Infrared-aided Nomarski differ- <br /> the construct resulted in a disrupted gene that produced a shorter fragment <br />                                                                                    ential interference contrast video microscopy was used to select <br /> when digested with the restriction enzyme HindIII. Cells and animals were <br /> genotyped by Southern analysis of HindIII-digested genomic DNA probed <br />                                                                                    a viable pyramidal cell in the CA3 region. Series resistance was <br /> with the DNA fragment depicted as a gray box above the schematic of the            monitored continuously throughout the recording. Recordings <br /> disrupted gene.H3, HindIII; X1, XbaI; A1, ApaI; S1, SpeI; neo, gene encoding       with series resistance &#226;&#172;&#65533;30 M&#226;&#65533;&#8364; were analyzed. Variation in series <br /> neomycin resistance; TK, gene encoding thymidine kinase; w.t., wild type. (B)      resistance was not compensated and varied, on average, 20%. <br />  <br />  <br />  <br />  <br />                                                                                                                                                         NEUROBIOLOGY <br /> Southern analysis demonstrating disruption of the SV2A gene. Shown is a            We observed no effect of series resistance flux on event fre- <br /> Southern analysis of genomic DNA isolated from two litters produced by mice        quency. All recordings were performed in the presence of 25 &#226;&#65533;&#174;M <br /> heterozygous (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186;) for the SV2A gene disruption. DNA containing the dis-           APV (2-amino-5-phosphonovaleric acid) and 10 &#226;&#65533;&#174;M CNQX <br /> ruption produces a smaller HindIII-digestion fragment. Genotypes are indi-         (6-cyano-7-nitroquinoxaline-2,3-dione). The recording bath also <br /> cated at the tops of the lanes. Homozygous mutants are marked with an <br />                                                                                    included 300 nM tetrodotoxin to block presynaptic action po- <br /> asterisk. wt, wild type. <br />                                                                                    tentials in recordings of miniature inhibitory postsynaptic cur- <br />                                                                                    rents (mIPSCs). Both mIPSCs and spontaneous inhibitory <br /> asphyxiated with CO2; brains were removed, embedded in Tissue                      postsynaptic currents (sIPSCs) were verified as GABAergic, at <br /> Tek OCT compound (Sakura Finetek, Torrance, CA), and kept                          the conclusion of 40% of the recordings, by using the GABA-A <br /> at &#226;&#171;&#186;80&#194;&#176;C until sectioning. Sections were fixed in 4% parafor-                      receptor antagonist, bicuculline methiodide (10 &#226;&#65533;&#174;M; Sigma). In <br /> maldehyde, stained with Toluidine blue, dehydrated, and cov-                       all cases, bicuculline blocked all events. Both sIPSCs and mIPSCs <br /> erslipped.                                                                         were measured by using an Axopatch 200B amplifier (Axon <br />                                                                                    Instruments, Foster City, CA). Recordings were filtered at 1 kHz <br />                                                                                    (80 dB per decade, 4-pole low-pass Bessel filter) and digitally <br /> Electron Microscopic Analyses. Ultrathin (70-nm) sections of the <br />                                                                                    sampled at 5 kHz (Digidata 1200 and pClamp6; Axon Instru- <br /> CA3 region of the hippocampus were obtained by using methods <br />                                                                                    ments). Events were analyzed by using MINI ANALYSIS PROGRAM <br /> similar to those described (11). Synapse density was measured by <br />                                                                                    (Synaptosoft, Leonia, NY), which employs a threshold-based <br /> using the physical dissector method (12) in two sets of five                       event-detection algorithm. Frequencies and amplitudes of sIP- <br /> serially positioned electron micrographs (final magnification                      SCs and mIPSCs were measured in 2-min epochs, taken at least <br /> &#226;&#171;&#187;25,875). Axodendritic synapses were identified by the presence                    4 min after the initiation of recording, by using the software <br /> of at least three vesicles in the presynaptic bouton and by the                    event detector set to identify events above background. mIPSC <br /> presence of a postsynaptic density. Synaptic density was calcu-                    events were visually inspected to ensure no false mIPSCs were <br /> lated as the number of synapses divided by the sample volume.                      included in data sets. The two-tailed Student&#226;&#8364;&#8482;s t test was used to <br /> Synaptic structural measures were made by using series of 4&#226;&#8364;&#8220;12                     assess statistical significance. <br /> electron micrographs per animal, printed at a final magnification <br /> of &#226;&#171;&#187;42,750. The profile area of boutons was measured by using                      Results and Discussion <br /> the technique of systematic point counting (13). The length of                     Targeted Disruption of the SV2A Gene. A portion of the SV2A gene <br /> postsynaptic densities was measured by using a grid-intersection                   containing the translation start site was isolated as described <br /> method. A transparency with a test line system was placed over                     under Methods and used to generate the targeting construct <br />  <br /> Crowder et al.                                                                                    PNAS &#229;&#8230;&#169; December 21, 1999 &#229;&#8230;&#169; vol. 96 &#229;&#8230;&#169; no. 26 &#229;&#8230;&#169; 15269 <br />                                                                                   truncated message expressed at lower levels than the full-length <br />                                                                                  wild-type mRNA. These results, combined with the Western <br />                                                                                  analyses, suggest that no SV2A protein is produced in the <br />                                                                                  knockouts. However, because all available antibodies are di- <br />                                                                                  rected against the amino terminus of SV2A, we cannot rule out <br />                                                                                  the possibility that a truncated SV2A protein is expressed. <br />  <br />                                                                                  SV2A Knockouts Appear Normal at Birth but Fail to Grow, Experience <br />                                                                                  Severe Seizures, and Die by the Third Week of Life. Animals ho- <br />                                                                                  mozygous for the SV2A gene disruption constitute approxi- <br />                                                                                  mately one-quarter of live births resulting from the mating of <br />                                                                                  heterozygous mutants. This indicates that SV2A is not required <br />                                                                                  for embryonic development. Indeed, SV2A knockouts are not <br />                                                                                  readily distinguishable from littermates until approximately P7. <br />                                                                                  However, after P7, the growth rate of knockouts falls signifi- <br />                                                                                  cantly behind that of both wild types and heterozygotes. By P9, <br />                                                                                  homozygotes are approximately one-half the size of littermates. <br />                                                                                  The fact that heterozygous mutants are of normal size indicates <br />                                                                                  that a single copy of SV2A is sufficient to support growth. SV2A <br />                                                                                  homozygous mutants are no more likely to die as neonates than <br />                                                                                  their littermates, however they die between P12 and P23, with an <br />                                                                                  average life span of 16.5 &#226;&#171;&#190; 2.4 days (n &#226;&#171;&#189; 28). Heterozygous <br />                                                                                  mutants have a slightly higher mortality rate in the second <br /> Fig. 2. Disruption of the SV2A gene results in the absence of SV2A protein       through sixth months versus wild types, but in general are viable, <br /> but does not significantly alter the expression of other synaptic proteins.      of normal body weight, and fertile. <br /> Shown are immunoblot analyses of brain protein isolated from wild-type <br />                                                                                     All SV2A homozygous mutants experience severe motor <br /> (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) mice and littermates heterozygous (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186;) and homozygous (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) for <br /> the SV2A <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>mutation</span>. SV2A expression was detected by using an isoform- <br />                                                                                  seizures. The age of onset, P6&#226;&#8364;&#8220;P10, parallels the developmental <br /> specific polyclonal antibody. Mice heterozygous for the mutation express         conversion of GABA from an excitatory to an inhibitory neu- <br /> significantly less SV2A than wild-type littermates, and homozygous mutants       rotransmitter in the central nervous system (CNS) (19, 20). <br /> express nondetectable levels. Western analysis using the anti-SV2 monoclonal     Seizures are generalized in nature, which is indicative of wide- <br /> antibody, which recognizes an epitope present in all known SV2 isoforms,         spread CNS hyperexcitability. Interestingly, heterozygous mu- <br /> revealed that SV2A knockouts (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) express very little total SV2 protein. This   tants also experience seizures. We have observed spontaneous <br /> suggests that SV2A is the primary SV2 isoform in mouse brain. This decrease is   seizures during routine cage changes in 24% (47&#229;&#8230;&#190;197) of the <br /> seen even though the expression of SV2B is increased in these animals.           SV2A heterozygous mutants, as compared with 2.7% (5&#229;&#8230;&#190;182) of <br /> Expression levels were normal for the synaptic proteins synaptotagmin I, <br />                                                                                  wild types. The seizure <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> in heterozygotes is interesting <br /> synaptophysin, and the 39-kDa accessory subunit of the H&#226;&#171;&#185;&#229;&#8230;&#190;ATPase (Ac 39), <br /> and for the t-SNAREs syntaxin and SNAP-25. <br />                                                                                  in light of their viability and normal size. This suggests that CNS <br />                                                                                  functioning is especially sensitive to SV2A expression levels. <br />                                                                                     The effects of the SV2A mutation on growth, CNS excitability, <br /> depicted in Fig. 1. This construct was used to generate mice                     and mortality indicate that SV2A is an essential protein and are <br /> homozygous for the SV2A gene disruption. SV2A mutants were,                      consistent with defects in multiple neuroendocrine systems. For <br /> on average, 50% 129SVJ and 50% C57BL&#229;&#8230;&#190;6. To control for                           example, the growth deficit may result from effects of the SV2A <br /> effects of a mixed genetic background and the possibility of                     mutation on hormone secretion. Likewise, SV2A-<span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>knockout</span> an- <br /> additional mutations that may have occurred in the generation                    imals die at the time when pups become reliant on their own <br /> of disrupted embryonic stem cell lines, wild-type animals used in                insulin production, suggesting that death could be caused by <br /> these studies were descendants of the founding chimeric male.                    aberrant energy metabolism. The presence of multiple secretory <br />    Mice homozygous for the SV2A gene disruption express no                       deficits is also suggested by the observation that the management <br />                                                                                  of seizures with medication does not reverse the growth defect <br /> SV2A protein as detected by immunoblot (Western) analysis <br />                                                                                  nor prevent death in SV2A knockouts (K.M.C. and S.M.B., <br /> (Fig. 2). Total SV2, detected with a monoclonal antibody that <br />                                                                                  unpublished data). <br /> recognizes an epitope present in all known isoforms, was also <br /> significantly reduced in the brain of SV2A knockouts. It <br />                                                                                  Loss of SV2A Does Not Alter Gross Brain Morphology. The brains of <br /> therefore seems that SV2A is the predominant isoform of SV2                      knockouts were smaller than wild types, in proportion with their <br /> in mouse brain. Both heterozygous and homozygous SV2A                            reduced body size. However, examination of Nissl-stained brain <br /> mutants expressed higher levels of SV2B than wild types did,                     section revealed no major differences in the relative size or <br /> suggesting that the expression of these SV2 isoforms is co-                      morphology of major brain structures, including the hippocam- <br /> regulated. However, this increase did not significantly raise                    pus, cortex, and thalamic structures (Fig. 3). Cerebellar mor- <br /> total SV2 levels. Loss of SV2A did not affect the expression of                  phology was also normal (data not shown). This indicates that <br /> the SV2A-binding protein synaptotagmin I, indicating that                        the seizure phenotype of the SV2A mutants is not caused by <br /> synaptotagmin expression and stability are not dependent on                      gross CNS anomalies. <br /> SV2A. There was also no effect on the expression of several <br /> other synaptic proteins, including synaptophysin (14), the                       Inhibitory Neurotransmission Is Reduced in the Hippocampus of SV2A <br /> 39-kDa subunit of the H&#226;&#171;&#185;&#229;&#8230;&#190;ATPase (15) or the target-localized,                    Mutants. Previous analyses of SV2-isoform expression in rat <br /> soluble N-ethylmaleimide-sensitive factor attachment protein                     revealed that SV2A is expressed in all neurons and is the only <br /> receptors (t-SNAREs) syntaxin (16), and SNAP-25 (refs. 17                        known isoform expressed in most GABAergic neurons and in <br /> and 18; Fig. 2).                                                                 hippocampal dentate granule cells (10), glutamatergic neurons <br />    Northern analyses using a probe to the 3&#226;&#172;&#732;-end of the rat SV2A                 that provide major excitatory input to GABAergic interneurons <br /> cDNA revealed that SV2A mutants do not express a full-length                     in the hippocampus (21). These neurons are predicted to be <br /> SV2A mRNA (data not shown). Long exposures revealed a                            devoid of SV2 in SV2A knockouts and, therefore, to be espe- <br />  <br /> 15270 &#229;&#8230;&#169; www.pnas.org                                                                                                                    Crowder et al. <br />  Fig. 3. Gross brain morphology is normal in SV2A-knockout mice. Shown are <br /> Nissl-stained coronal sections of brain from P14 wild-type and SV2A-knockout <br /> mice. The brains of SV2A-knockout mice are smaller, in proportion with their <br /> reduced body size. However, major brain structures, including the cerebral <br /> cortex, hippocampus, and thalamus, appear normal. Cerebellar morphology <br /> was also normal (data not shown). wt, wild type; KO, knockout. <br />  <br />  <br />  <br /> cially affected. If SV2 is required for normal neurotransmission, <br /> its absence in these neurons should result in aberrant inhibitory <br /> neurotransmission, which could underlie the seizure phenotype <br /> of the SV2A mutants. To test this possibility, we examined <br /> GABAergic neurotransmission in a hippocampal slice prepara- <br /> tion. Previous studies have revealed high levels of endogenous                 Fig. 4. GABAergic neurotransmission is reduced in the hippocampus of <br /> GABAergic neurotransmission in the hippocampus (22), most of                   SV2A-knockout animals. sIPSCs were measured in hippocampal CA3 pyramidal <br /> which is action potential-dependent (23). This spontaneous                     neurons. Recordings were made in the presence of the glutamate receptor <br /> GABA release is hypothesized to control hippocampal excit-                     blockers CNQX and APV. (A) Sample traces from wild-type (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) and SV2A- <br /> ability (22, 24). Whole-cell recordings of sIPSCs in CA3 pyra-                 knockout (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) cells. Shown are 4-sec traces of sIPSCs recorded from repre- <br /> midal neurons were made in slices obtained from animals aged                   sentative cells. The downward deflection of the events is the result of outward <br />                                                                                chloride flux in the cells voltage-clamped at &#226;&#171;&#186;70 mV. Currents were filtered at <br /> P9 to P12, with the majority coming from P9&#226;&#8364;&#8220;P10 animals to <br />                                                                                1 kHz and digitized at 5 kHz. There was no digital filtering of the recordings <br /> control for seizure-induced changes in the mutants. All record- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   NEUROBIOLOGY <br />                                                                                to decrease noise in the traces shown. (B) Averaged sIPSC frequencies. Shown <br /> ings were done in the presence of the glutamate receptor                       are histograms representing average sIPSC frequencies. Each was calculated <br /> blockers CNQX and APV to prevent fast, excitatory neurotrans-                  from 30 2-min epochs. Wild-type values were obtained from six cells from six <br /> mission. The frequency of sIPSCs was high in wild-type CA3                     (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) animals. Knockout values were obtained from seven cells from five <br /> neurons, displaying an average of 7.3 Hz, a rate that agrees with              (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) animals. Error bars indicate SEM. Rates were significantly different as <br /> previous reports of sIPSC rates in rat hippocampus (22). In                    determined with the Student&#226;&#8364;&#8482;s t test (P &#226;&#172;&#65533;&#226;&#172;&#65533; 0.01). (C) Frequency histograms of <br /> contrast, sIPSC frequency averaged 2.6 Hz in SV2A knockouts                    sIPSC amplitude. A frequency analysis of event amplitudes binned in 25-pA <br />                                                                                increments is shown. Data were derived from recordings of six cells from six <br /> (Fig. 4), less than half of that found in wild-type neurons. sIPSC             (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) animals and seven cells from four (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) animals. For each cell, a single <br /> amplitude was also significantly reduced in SV2A knockouts as                  2-min epoch of representative frequency was selected. All events detected by <br /> compared with wild types, 23.3 &#226;&#171;&#190; 7.8 pA (SEM) vs. 51.2 &#226;&#171;&#190; 6.8                   the detection program were verified by visual inspection. Note that there <br /> pA, respectively (n &#226;&#171;&#189; 4). This reduced amplitude is consistent                 were many fewer events in (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) cells. The average amplitude in (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) cells <br /> with the decreased frequency of events, the majority of which                  was roughly half that of events in (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) cells, 24.85 pA &#226;&#171;&#190; 7.8 vs. 51.2 pA &#226;&#171;&#190; 6.8 <br /> reflect concomitant transmitter release at multiple sites. The                 for the knockouts and wild type, respectively (n &#226;&#171;&#189; 4 each). <br /> simplest interpretation of these results is that loss of SV2A <br /> reduces GABA secretion. However, because these recordings <br /> measured naturally occurring activity, we cannot distinguish                   quency would indicate that SV2 is a component of the basic <br /> between direct and indirect effects. For example, decreased                    fusion machinery, whereas an increase would suggest that it acts <br /> GABAergic transmission could also be caused by a decrease in                   as a negative regulator of vesicle fusion. In wild-type cells, <br /> excitatory input to GABAergic neurons via metabotropic glu-                    tetrototoxin reduced the frequency of total spontaneous IPSCs <br /> tamate receptors or increased inhibitory input carried by neu-                 by 90%, resulting in an average frequency of 0.52 Hz and <br /> ropeptides. In either case, it is clear that loss of SV2A results in           confirming previous reports that most of the spontaneous in- <br /> abnormal inhibitory neurotransmission in the hippocampus,                      hibitory neurotransmission in the hippocampus is action poten- <br /> which is likely to contribute to the seizure phenotype.                        tial-dependent (22, 24). mIPSC frequency was similar in neurons <br />    To determine whether reduced inhibitory neurotransmission                   from SV2A-knockout animals, averaging 0.50 Hz (Fig. 5). The <br /> in SV2A knockouts reflects alterations in the basic machinery of               lack of a difference in mIPSC frequency suggests that SV2 is not <br /> membrane fusion, we measured the frequency of mIPSCs in CA3                    required for vesicle fusion, an interpretation consistent with its <br /> neurons. Recorded in the presence of the sodium channel                        absence in yeast. mIPSC amplitude, which reflects the number <br /> blocker tetrototoxin, mIPSCs represent action potential-                       of postsynaptic receptors (25), was also measured. We observed <br /> independent transmitter release. A decrease in mIPSC fre-                      similar mIPSC amplitudes in wild-type and SV2A-knockout <br />  <br /> Crowder et al.                                                                                    PNAS &#229;&#8230;&#169; December 21, 1999 &#229;&#8230;&#169; vol. 96 &#229;&#8230;&#169; no. 26 &#229;&#8230;&#169; 15271 <br />  Fig. 5. Action potential-independent neurotransmission is not affected in SV2A-knockout animals. (A) Sample traces from wild-type (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) and SV2A-knockout <br /> (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) cells. Shown are representative 23-sec sweeps of mIPSCs from whole-cell voltage clamp recordings of hippocampal CA3 pyramidal cells held at &#226;&#171;&#186;70 mV. <br /> Recordings were made in the presence of the glutamate receptor blockers APV and CNQX and the sodium channel blocker tetrototoxin. (B) mIPSC frequencies <br /> were similar in cells from wild type and SV2A knockouts. Histograms illustrating the average frequency of mIPSCs recorded from nine (&#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;) cells (six animals) <br /> and nine (&#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186;) cells (six animals). Error bars indicate SEM. Rates were not significantly different as determined with the Student&#226;&#8364;&#8482;s t test (P &#226;&#171;&#189; 0.79). <br />  <br />  <br />  <br /> neurons. In fact, amplitudes were slightly larger in SV2A-mutant <br /> cells (data not shown). These results suggest that loss of SV2A <br /> does not significantly reduce the expression or action of postsyn- <br /> aptic receptors and are consistent with the interpretation that <br /> SV2A is not involved in the regulation of quantal size. <br />  <br /> Loss of SV2A Does Not Alter Synapse Density or Morphology. The <br /> observation that SV2 binds the extracellular matrix protein <br /> laminin (8) suggested that it could participate in the formation <br /> of synapses or the establishment of active zones. To determine <br /> whether the reduced neurotransmission in SV2A knockouts is <br /> caused by changes in synapse density and&#229;&#8230;&#190;or morphology, syn- <br /> apses were analyzed at the electron microscopic level. Two areas <br /> of the CA3 region of the hippocampus were chosen for exami- <br /> nation: the CA3 pyramidal neuron cell-body layer and the <br /> proximal region of CA3 apical dendrites. CA3 cell bodies receive <br /> inhibitory input from GABAergic interneurons via symmetric <br /> synapses (26), whereas proximal dendrites receive mossy fiber <br /> input from dentate granule neurons via asymmetric, presumed <br /> excitatory synapses (27). These synapses express SV2A primarily <br /> or exclusively (10, 28) and, therefore, were predicted to be <br /> especially affected by the SV2A gene disruption. Visual inspec- <br /> tion of micrographs revealed no differences in the density or <br /> morphology of either synapse type (Fig. 6). <br />    Quantitative structural analyses were performed on the more <br /> numerous mossy fiber synapses onto proximal dendrites of CA3 <br /> pyramidal neurons. As listed in Table 1, synapse density was <br /> similar in wild type and SV2A knockouts, indicating that loss of <br /> SV2A does not affect the establishment or pruning of these <br /> synapses. Synapse morphology was also not significantly altered. <br /> We found no difference in the size of synaptic boutons, active <br /> zone length, postsynaptic density thickness, or in the clustering <br /> of vesicles (measured as the number of vesicles per <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> area). It <br /> remains possible that SV2A plays a role in the strengthening of <br /> synapses and that the effects of the mutation on this function are <br /> not apparent in young animals. However, it appears that the <br /> observed phenotype of SV2A knockouts is not attributable to                     Fig. 6. Synapse morphology is normal in SV2A-knockout mice. (A) Shown are <br /> aberrant synapse formation or morphology.                                       representative electron micrographs of symmetric (GABAergic) synapses in the <br />    We also determined whether the loss of SV2A affected                         cell-body region of CA3 hippocampal neurons. These synapses are presumed to <br /> synaptic vesicle formation, stability, or size, by comparing the                be GABAergic based on their location, symmetry of active zones, and irregularly <br />                                                                                 shaped vesicles. Qualitative inspection of these synapses suggested no difference <br /> number of synaptic vesicles and vesicle diameter between wild <br />                                                                                 in density or morphology between wild type and SV2A knockouts. (B) Shown are <br /> type and SV2A knockouts. The average number of synaptic                         representative electron micrographs of asymmetric synapses onto proximal den- <br /> vesicles per synapse did not differ (Table 1). Likewise, vesicle                drites of CA3 hippocampal neurons. Synapses in this region are primarily from <br /> diameters were similar and agreed with previously reported                      dentate granule cells and are largely asymmetric, presumed glutamatergic syn- <br /> measurements (29). This normal vesicle phenotype contrasts                      apses. Similar morphology was observed in wild type and SV2A knockouts. <br /> with that found in rab3A and synapsins I and II knockouts. The                  Quantitative analyses of these synapses are presented in Table 1. <br />  <br />  <br /> 15272 &#229;&#8230;&#169; www.pnas.org                                                                                                                              Crowder et al. <br />  Table 1. Loss of SV2A does not affect synapse number or morphology <br />                            Synapse                   Bouton            Active zone        Active zone            Vesicle              Vesicle                 Vesicle <br />                          density, 106             size, area in          length,           thickness,          clustering,         number, mean              diameter, <br />                         synapses&#229;&#8230;&#190;mm3                  &#226;&#65533;&#174;m2                  nm                 nm              vesicles&#229;&#8230;&#190;&#226;&#65533;&#174;m2          per synapse                 nm <br />  <br /> Wild type                      668              217 &#226;&#171;&#190; 147 (86)       370 &#226;&#171;&#190; 25 (115) 7.5 &#226;&#171;&#190; 22 (115)           2.4 &#226;&#171;&#190; 0.5 (86)      19.1 &#226;&#171;&#190; 18.2 (114)        30.4 &#226;&#171;&#190; 9.4 (155) <br /> SV2A knockout                  620              290 &#226;&#171;&#190; 208 (88)       412 &#226;&#171;&#190; 19 (112) 6.7 &#226;&#171;&#190; 26.3 (103)         2.4 &#226;&#171;&#190; 0.9 (88)      21.5 &#226;&#171;&#190; 17.3 (110)        37.4 &#226;&#171;&#190; 9.4 (190) <br />  <br />    The proximal dendritic region of CA3 hippocampal neurons, which receives mossy fiber input from dentate granule neurons and which has a predominance <br /> of asymmetric, presumed excitatory synapses, was photographed at &#226;&#171;&#187;42,750. Photographs were analyzed for the listed features as described under Methods. <br /> Data were obtained from three wild-type and three SV2A-knockout animals, except for the measure of synapse density, which was obtained from three wild <br /> types and two SV2A knockouts. Numbers in parentheses indicate the total number of synapses or vesicles analyzed. <br />  <br />  <br />  <br /> loss of these synaptic vesicle proteins led to a reduction in                            binding of SV2 and synaptotagmin is regulated by calcium, <br /> synaptic vesicle number (30, 31). In a variation of this effect,                         suggesting that this complex participates in the control of <br /> mutation of the gene encoding the synaptic vesicle protein                               exocytosis. It is interesting to note that targeted disruption of <br /> synaptotagmin, in Drosophila, resulted in a decrease in the                              synaptotagmin I in mice resulted in profoundly reduced evoked <br /> number of vesicles immediately adjacent to the active zone (32).                         transmission without affecting the frequency of action potential- <br /> Although our morphological analyses do not rule out a role for                           independent exocytosis (33), a phenotype analogous to the <br /> SV2A in the recruitment or maintenance of vesicles at active                             SV2A-mutant phenotype. Although the reduced severity of the <br /> zones, they do indicate that SV2A is not needed for vesicle                              SV2A phenotype could be caused by the compensatory action of <br /> formation or stability.                                                                  additional, uncharacterized SV2 isoforms, it is also consistent <br />    The results presented here demonstrate that SV2A is required                          with SV2A acting as a modulator of calcium-stimulated exocy- <br /> for normal neurotransmission. In the absence of SV2A, action                             tosis rather than as an effector. Such a modulatory role could <br /> potential-dependent GABAergic neurotransmission is aberrant.                             occur via interaction with synaptotagmin. Further study, of both <br /> This effect does not appear to be caused by changes in the                               SV2 mutants and SV2 protein interactions, will more clearly <br /> number or structure of synapses. Although we cannot rule out                             resolve the precise molecular action of SV2 in the synapse. <br /> the possibility that the decreased neurotransmission is secondary <br /> to effects at other synapses, the synapses studied were among the <br /> most likely to be affected by the loss of SV2A. An alternate                             We thank Drs. Rejean Idzerda and G. Stanley McKnight for supplying <br />                                                                                          ES cells; Kathy Kafer for blastocyst injections; Drs. Philip Schwartzkroin, <br /> possibility, that the observed phenotype is caused by changes in <br />                                                                                          Jergen Wenzel, and Gregory Terman for helpful discussions; Dr. Roger <br /> the ability of GABAergic neurons to produce or sustain action                            Janz for sharing information on SV2C expression; and Dr. William <br /> potentials, could result from direct or indirect interactions of                         Catterall for reviewing the manuscript. Early portions of this work were <br /> SV2 with presynaptic ion channels, a characteristic that has not                         supported by grants from the Sloan and Whitehall Foundations and an <br /> yet been reported for SV2. The simplest interpretation, however,                         Howard Hughes Medical Institute&#229;&#8230;&#190;University of Washington Pilot Re- <br /> is that SV2A participates in calcium&#226;&#8364;&#8482;s triggering of vesicle fusion.                     search grant to S.B. This work was funded by the National Institutes of <br /> A similar function has been ascribed to synaptotagmin (33), a                            Mental Health Grant R01 MH59842&#226;&#8364;&#8220;01. J.G. was supported by Grants <br /> calcium-binding protein that interacts with SV2 (9). In vitro                            DA07278 and NS33898 from the National Institutes of Health. <br /> </body></html>