<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Evolution Mutational Robustness Yeast <br /> Genome: A Link Essential Genes Meiotic <br /> Recombination Hotspots <br /> Philipp J. Keller, Michael Knop* <br /> European Molecular Biology Laboratory (EMBL), Cell Biology Biophysics <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Heidelberg, Germany <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Deleterious mutations inevitably emerge evolutionary process speculated decisively influence the <br />      structure genome. Meiosis, thought play major role handling mutations population level, <br />      recombines chromosomes non-randomly distributed hot spots meiotic recombination. In genomes, various <br />      types genetic elements distributed patterns currently understood. In particular, important <br />      (essential) genes arranged clusters, explained functional relationship involved <br />      genes. Here simulation essential gene (EG) clustering provides fitness benefit handling <br />      deleterious mutations sexual populations variable levels inbreeding outbreeding. We recessive lethal <br />      mutations enforce selective pressure clustered genome architectures. Our simulations correctly predict (i) the <br />      evolution non-random distributions meiotic crossovers, (ii) genome-wide anti-correlation meiotic crossovers and <br />      EG clustering, (iii) evolution EG enrichment pericentromeric regions (iv) associated absence meiotic <br />      crossovers (cold centromeres). Our results furthermore predict optimal crossover rates yeast chromosomes, match <br />      experimentally determined rates. Using Saccharomyces cerevisiae conditional mutator strain, haploid <br />      lethal phenotypes result predominantly <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> single loci generally impair mating, leads an <br />      accumulation mutational load following meiosis mating. We hypothesize purging deleterious mutations in <br />      essential genes constitutes important factor driving meiotic crossover. Therefore, increased robustness of <br />      populations deleterious mutations, arises clustered genome architectures, provide significant selective <br />      force shaping crossover distribution. Our analysis reveals new aspect evolution genome architectures that <br />      complements insights molecular constraints, interference pericentromeric crossovers with <br />      chromosome segregation. <br />  <br />   Citation: Keller PJ, Knop M (2009) Evolution Mutational Robustness Yeast Genome: A Link Essential Genes Meiotic Recombination Hotspots. PLoS <br />   Genet 5(6): e1000533. doi:10.1371/journal.pgen.1000533 <br />   Editor: Harmit S. Malik, Fred Hutchinson Cancer Research Center, United States America <br />   Received December 11, 2008; Accepted May 22, 2009; Published June 26, 2009 <br />   Copyright: &#195;&#376; 2009 Keller, Knop. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: Funding project occurred entirely internal resources EMBL. PJK supported predoctoral fellowship EMBL International <br />   PhD Programme German National Academic Foundation. The funders role study design, data collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: knop@embl.de <br />  <br />  <br />  <br />  <br /> Introduction                                                                        cause higher mutation rates) studied experimental <br />                                                                                     theoretical work unicellular organisms [6&#226;&#8364;&#8220;8] during <br />    Mating meiosis masterpieces evolutionary                       cancer progression [9,10]. In asexual yeast populations, selective <br /> invention thought meet challenges changing environ-                       advantage mutator alleles demonstrated, serving a <br /> mental conditions need solved mutational inventions.                  prerequisite expanding spectrum mutations typically not <br /> Among hypotheses govern various benefits                       accessible non-mutator genotypes [11]. In study yeast, <br /> mating meiosis [1], main hypotheses stand out: enhanced                     mutator advantage prominent diploid <br /> purging deleterious mutations [2] combination                         haploid cells, explained presumed <br /> beneficial alleles genome [3]. It remains matter                      dominance beneficial mutations [11,12] recessive <br /> discussion, however, advantages constitutes                      nature deleterious mutations yeast [13]. Furthermore, <br /> main reason evolution sexual recombination and,                          accumulation recessive deleterious mutations yeast may <br /> furthermore, continuing prevalence eukaryotic life                      significantly decrease fitness genotype population <br /> forms [4,5].                                                                        growth rates, long diploid nuclear condition maintained <br />    Mutations form DNA lesions caused                        [14]. <br /> environmental factors, e.g. radiation, natural                     S. cerevisiae yeast fungal species seek <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> <br /> byproduct DNA replication. Genotypes exhibit elevated                    maintain diploid state genome, possible; meiosis <br /> mutation rate frequent nature induced studies                  usually happens immediate mating gametes following <br /> on experimental evolution. The complex interplay factors                    germination. Mating occurs closely related spores, <br /> govern adaptive significance &#226;&#8364;&#732;&#226;&#8364;&#732;mutator alleles&#226;&#8364;&#8482;&#226;&#8364;&#8482; (i.e. alleles               products meiosis, spores <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     1                               June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                     Gene Order Evolution Yeast <br />  <br />  <br />  <br />   Author Summary                                                            essential genes (&#226;&#8364;&#732;&#226;&#8364;&#732;housekeeping genes&#226;&#8364;&#8482;&#226;&#8364;&#8482;) reported [28,29]. <br />                                                                             In budding yeast, prominent genome-wide enrichment of <br />   Sexual life cycles constitute costly alternative                     essential genes observed regions cold for <br />   vegetative modes reproduction. Two categories                       meiotic recombination [30]. This finding consistent the <br />   hypotheses seek explain sexual life cycles exist:                  observation slight enrichement essential genes near the <br />   investigating selective advantages                    centromeres [31], known cold meiotic <br />   driven evolution individual parts life cycle               recombination [32,33]. Current models speculate mecha- <br />   rationalizing advantages sexual life cycles                 nisms co-expression reduction gene expression noise as <br />   offer whole, e.g., extant species. Sex                    driving force shaped patterns [29,30,34&#226;&#8364;&#8220;36]. <br />   recombination understood efficient ways                      However, selective advantage scenario been <br />   interact mutations consequences. Mutations                 demonstrated, proof single rearrangements associate with <br />   occur random deleterious or <br />                                                                             advantage sufficient selection provided. <br />   neutral. A prominent hypothesis suggests sex and <br />   recombination advantageous enhance                        An alternative mechanism pool essential genes into <br />   purging deleterious mutations create individ-                 clusters reduced probability disruption frequent <br />   uals lower average deleterious load. Delete-                  crossovers selection based common <br />   rious mutations co-determine parameters                   denominator. This appears essential nature only, as <br />   govern recombination genomes meiosis. Using                      functional correlation essential genes cluster <br />   evolutionary simulation diploid, unicellular                  observed [30]. <br />   sexual populations, recessive lethal muta-                      Meiotic recombination favorable purging deleterious load <br />   tions drive evolution chromosome architectures,                populations. We hypothesize clustering essential genes <br />   essential genes genetically linked                   enhance purging efficiency lethal load sexual <br />   clusters. Evolved architectures exhibit structural properties             populations, non-uniform distributions essential genes and <br />   fitness similar digitized yeast chromosomes                    crossover sites change global genetic linkage relationship all <br />   provide mutational purging capabilities superior                 essential genes (as compared situations uniform random <br />   randomly generated unclustered architectures. Our                   distributions). In context scenario one <br />   study demonstrates importance sexual cycles                 lethal mutation genome, influence segregation <br />   context lethal mutations.                                              mutations meiosis subsequent mating. <br />                                                                                In order address question, computationally studied <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> related cells according population structure [15].                  correlation non-random distributions essential <br /> Outcrossing unrelated strains [16]                 genes meiotic recombination lethal load affecting <br /> closely related species sensu stricto yeast group does occur         essential genes. A Monte-Carlo-simulation breeding diploid <br /> [17]; events appear extremely rare,           yeast populations chromosome architectures, termed S. <br /> important generating new persisting lineages. However,                  digitalis, allowed investigate fitness genome <br /> whether rare events suffice create selective pressure            architecture exposure lethal mutations. We that <br /> towards maintaining sexual cycle doubtable.                            hallmarks yeast chromosome crossing during <br />    Upon inbreeding, high rates homozygotisation occur                 meiosis consistent natural selection imposed recessive <br /> various loci. This reduced loci linked MAT locus,       lethal mutations affecting essential genes (see Figure 1 an <br /> mating type heterozygosity prerequisite mating event.          overview approach results). <br /> A MAT-linkage centromeres frequently observed yeast               Our simulations imply lethal phenotypes frequently <br /> other fungi [18]. In S. cerevisiae, genetic distance        caused single essential gene inactivation. Alternatively, lethal <br /> two loci range 18&#226;&#8364;&#8220;30 cM [19], smaller            phenotypes arise genetic interactions only <br /> than expected physical <span id='am-8' about='xsp:length' typeof='owl:Thing'>length</span> caused region               weakly deleterious mutations. We explored possibilities using <br /> intervening loci cold meiotic recombination           conditional yeast mutator strain analyzed causes the <br /> [20]. In Neurospora tetrasperma MAT linkage centromere            accumulation haploid lethal phenotypes. By determining the <br /> enforced crossover suppression long region                   global effect germination mating, tested the <br /> chromosome, correlates extensive unpaired region              associated load transmitted round diploid <br /> at pachytene [21]. Reasoning arrangements provided              growth. <br /> by population genetic models suggest selective advantage                Our combined results suggest evolutionary history yeast <br /> arising shielding recurrent deleterious load linkage         sex meiosis fulfilled need efficient purging of <br /> the MAT locus [22]. Population genetic modifier models                 mutational load important/essential genes. <br /> investigate alteration inbreeding frequency predict the <br /> evolution high inbreeding rates, particular spores       Results <br /> same tetrad (i.e. automixis), linkage load loci the <br /> MAT locus [23,24]. These MAT linked chromosomal recombina-                  Simulation digital yeast genomes S. digitalis <br /> tion abnormalities believed initiated evolution             We sought assay consequences non-random distribu- <br /> sex chromosomes, continued expansion               tions essential genes meiotic recombination hotspots the <br /> recombination suppressed region recruitment            fitness populations frequent inactivation essential genes <br /> sex-related factors [25,26].                                                lethal mutations. This analysis requires direct comparison of <br />    Non-random distribution meiotic recombination              fitness yeast strains genomic content but <br /> the genomes cold hot regions reported            different arrangements genetic elements. Conducting an <br /> species [27], molecular mechanisms selective         experiment question addressed isolation <br /> forces generate patterns fully understood.         possible consequences human-designed <br />    Similarly, distribution genes chromosomes appears           genomic architectures far trivial. Therefore, developed <br /> to non-random, species significant clustering           simulation populations digital genomes subjected <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              2                           June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                        Gene Order Evolution Yeast <br />  <br />  <br />  <br />  <br /> Figure 1. Conceptual overview methods main results study. We developed simulation (S. digitalis) test the <br /> hypothesis feedback sexual processes (meiosis mating) genome architecture facilitate purging lethal mutations <br /> in populations unicellular organisms (such baker&#226;&#8364;&#8482;s yeast, S. cerevisiae). The implementation simulation based population genetic <br /> concepts results mutation-accumulation experiments Msh2 mutator strain (see main text). Experiments performed with <br /> digitally implemented populations yeast genomes revealed competitive advantage yeast-alike chromosome architectures enabled the <br /> evolution genomes yeast-alike fitness. These investigations provide reasoning hallmarks yeast genome evolution various <br /> parameters population genetics yeasts. <br /> doi:10.1371/journal.pgen.1000533.g001 <br />  <br />  <br /> to digital life cycles mitosis, meiosis mating, modeled               reported exhibit weak impact fitness [37] and <br /> according simple yeast life cycle (Figure 2A). We used             types mutations shielded heterozygous diploids <br /> simulation study relationship genome architecture             [39]. Therefore, decided focus heterozygous lethal <br /> and fitness populations evolution genome            mutations, confer lethal <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> level of <br /> architectures exposure populations essential gene                 haploids homozygotisation mutations diploids <br /> inactivating mutations (Figure 2B).                                          (Supplementary Figure 1A Text S1). <br />    Haploid lethal mutations frequently observed yeast               S. digitalis simulates populations diploid digital organisms with <br /> constitute approximately 40% deleterious mutations [37,38]            chromosome, consists different building blocks: <br /> (see below). The remaining fraction deleterious mutations            genes, intergenic elements, centromeres mating type loci <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              3                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                               Gene Order Evolution Yeast <br />  <br />  <br />                                                                                     (MATa MATa). The digital genes non-essential or <br />                                                                                     essential. Each gene category carries unique <br />                                                                                     identifier. Genes separated intergenic elements (IE). IEs are <br />                                                                                     cold crossing ( = coldspots) hot ( = hotspots). Each <br />                                                                                     feature represented element matrix the <br />                                                                                     population genome, mutations affect elements that <br />                                                                                     represent essential genes (Figure 3A). Typical natural yeast <br />                                                                                     chromosomes contain genes, digital <br />                                                                                     counterparts. Populations finite size typically consist of <br />                                                                                     thousand diploid individuals. <br />                                                                                        Mitosis yields copy original genome. It differs the <br />                                                                                     template genome mutations, introduced random <br />                                                                                     lead inactivation essential genes (Figure 3B). The <br />                                                                                     statistical frequency essential gene mutations diploid <br />                                                                                     genome mitosis given genomic recessive lethal <br />                                                                                     mutation rate R [40] (for example, R = 1 corresponds an <br />                                                                                     average essential gene inactivation diploid genome and <br />                                                                                     mitosis). <br />                                                                                        Genomic rearrangements simulated. They manifest <br />                                                                                     positional swapping genes associated <br />                                                                                     intergenic elements, segmental inversions (Figure 3B). <br />                                                                                     Genomic rearrangements occur mitosis affect both <br />                                                                                     homologous chromosomes. <br />                                                                                        In meiosis, genome duplicates homologous <br />                                                                                     chromosomes undergo meiotic recombination (crossing over). <br />                                                                                     The distribution crossover sites considers meiotic recombination <br />                                                                                     hotspots crossover interference based genetic distance <br />                                                                                     <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> (hotspot distribution). The crossover frequency be <br />                                                                                     adjusted shape factor Erlang distribution used <br />                                                                                     crossover interference (Figure 3C). The meiotic <br />                                                                                     haploid progenitor genomes constitute tetrad. <br />                                                                                        Mutations allowed occur mitosis (see Text S1, section <br />                                                                                     &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Results Discussion&#226;&#8364;&#8482;&#226;&#8364;&#8482; analysis meiotic <br />                                                                                     mutations). This implementation considers single mitotic cycle <br />                                                                                     consecutive meiotic cycles. This mimics situation in <br />                                                                                     consecutive rounds mitoses occur without <br />                                                                                     exponential growth. This applies scenarios high loss <br />                                                                                     individuals occurs (e.g. individuals eaten predators or <br />                                                                                     washed away non-fertile grounds) keeps size local <br />                                                                                     population constant. Under circumstances where <br />                                                                                     deleterious mutations recessive influence fitness <br />                                                                                     cells (as indicated literature [39]), lead the <br />                                                                                     accumulation mutations vegetative period life <br /> Figure 2. Yeast life cycles mutation homeostasis. (A) The key <br /> elements natural yeast life cycles. Life cycles natural yeast <br />                                                                                     cycle. This simplified scenario come close realistic <br /> populations regulated changes environment,               description natural S. cerevisiae consistent absence <br /> permit mitotic growth induce formation spores meiosis. All S.         reports large natural cultures S. cerevisiae (outside human- <br /> cerevisiae isolates nature diploid, indicating              engineered fermentation processes). Moreover, approximation <br /> return diploid lifestyle immediately following germination. This           allows simulate large numbers complete life cycles, which <br /> occur intra-tetrad spore-mating (inbreeding/automixis), mating             inaccessible computational limitations. <br /> of spores different tetrads (outbreeding/amphimixis). Optionally, <br /> mating type switching followed mother-daughter mating                           Upon germination, haploid genomes directly engage in <br /> generate diploid cell lines (see corresponding sections Results              mating haploid genomes (Figure 3D). Mating occur <br /> Discussion). Predominant diploid life cycles             genomes meiosis, called <br /> other species yeasts [91]. (B) Loss individuals            intratetrad mating generally referred automixis <br /> population occurs random decrease fitness caused           inbreeding. Mating haploid genomes different <br /> by mutations. The simulation implements populations cycling <br />                                                                                     tetrads occur, referred amphimixis or <br /> through vegetative sexual stages life cycle (mitosis, meiosis <br /> and mating). The genomes constantly exposed mutations. A                     outbreeding. In article, use terms inbreeding and <br /> homozygotisation recessive lethal mutations (mutations inacti-              outbreeding distinguish principal types of <br /> vate essential genes) leads death individual. Alternatively,           mating partner selection simulation: mating inside and <br /> individuals removed random population                     outside tetrad. Outbreeding events bring <br /> limitations nutritional supply (starvation). In dynamic equilibrium,       closely related genomes together, simply finite size of <br /> in average population growth equals zero, influx new <br />                                                                                     simulated populations. The total fraction inbreeding <br /> recessive lethal mutations equal outflux associated the <br /> death removal individuals. For details fitness impact          matings round mating specified. Mating optionally <br /> mutations, main text Supplementary Figure 1A 1B Text S1.             considers mating types, exist (MATa MATa). The <br /> doi:10.1371/journal.pgen.1000533.g002                                               MAT locus placed chromosome. In this <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     4                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                           Gene Order Evolution Yeast <br />  <br />  <br />  <br />  <br /> Figure 3. Digital genomes life cycle digital populations. (A) Schematic illustration genetic building blocks digital <br /> genomes (top): essential genes (wild type, white squares; mutated, crossed squares); non-essential genes (black squares); intergenic elements <br /> (circles) crossing proficient (recombination hotspots, white circles) silent (crossed circles). Unique identifiers represent essential <br /> genes. In simulation, individual possesses copies single chromosome. Such diploid genome considered viable contains at <br /> least functional copy individual essential gene. The simulation framework implements entire populations individuals diploid <br /> genomes, arranged population matrix (bottom). Individuals different chromosome architectures belong different species. (B) <br /> The life cycle digital populations. See main text details. (C) Crossover interference implemented using Erlang probability density function <br /> with shape factor k = 4 (F. Stahl, personal communication, references [46,92,93]). The function based genetic distance <span id='am-7' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> (hotspot <br /> density). Rescaling Erlang function distance axis allows adjusting crossover frequency chromosome. (D) Breeding occur <br /> between haploid gametes meiosis (intratetrad mating/inbreeding) gametes different meioses (outbreeding). Mating <br /> types optional. See main text details. <br /> doi:10.1371/journal.pgen.1000533.g003 <br />  <br /> case, modeled chromosome considered sex                   mutations&#226;&#8364;&#8482;&#226;&#8364;&#8482;). We decided use simple fitness denominator for <br /> chromosome. Alternatively, simulation employ virtual                 individuals: 1 assigned diploid genomes contain least <br /> second chromosome contains MAT locus                      functional copy essential gene, 0 assigned for <br /> centromere. In intratetrad mating, causes linkage                genomes, copies essential gene are <br /> MAT locus centromeric region investigated                        non-functional. Individuals fitness 0 removed. Hence, <br /> chromosome [41].                                                               criterion underlying loss individual to <br />    Diploid genomes different gene order belong different               mutations homozygotisation mutated essential gene. <br /> species. Individuals different species able mate          This occur different ways: new mutation inactivates <br /> each other. These species represent sub-populations, emerge              second wild type copy mating event brings two <br /> in simulation scenarios genomic rearrangements. Alternatively,            chromosomes contain mutated allele same <br /> different sub-populations specified beginning             position. <br /> simulation, e.g. order compare fitness different genome              Populations limited size according defined <br /> architectures (species) <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> competition assays.                        maximum (the population size cap). Excess individuals are <br />    Fitness individuals assessed diploid stage          removed random round mitotic meiotic <br /> mitotic meiotic cell division. We furthermore assumed                  division. This simulates limited availability nutrients. As result, <br /> mating prevented haploid lethal mutation                     selective pressure introduced potential driving <br /> essential gene (this assumption experimentally tested;                 evolution species ( = different genome architectures) are <br /> below, section &#226;&#8364;&#732;&#226;&#8364;&#732;Mating rescues genomes associated lethal                 better adapted handle lethal mutations. Supplementary Figure <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                5                             June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                           Gene Order Evolution Yeast <br />  <br />  <br /> 1A 1B Text S1 provides overview mechanisms                 mitosis meiosis (Supplementary Figure 2B 2C in <br /> the simulation.                                                                Text S1). <br />    The simulation provides modules different types                         Using survival competition assay, directly compared the <br /> experiments, including mutational robustness benchmarks                     persistence populations random chromosomes of <br /> populations specific genomic architectures, selection advan-              populations yeast chromosome IX different mutation rates <br /> tage assessments isolated populations                    different population sizes. The competition experiments <br /> compete nutrients evolution experiments large in-               revealed clear selective advantage yeast chromosome IX <br /> out-breeding populations constantly undergo genomic                       architecture regions investigated parameter space <br /> rearrangements (see Text S1). Detailed descriptions                     (Figure 4B). A stalemate situation observed low <br /> simulation modules implementation yeast model                   mutation rates R,0.01 extreme inbreeding fractions (i = 0 <br /> chromosomes provided Text S1.                                           = 1) (Figure 4B 4C). We performed control experiment <br />                                                                                demonstrate quantitative outcome survival <br /> S. cerevisiae chromosome architecture provides                                 competition assay unaffected choice, life cycle <br /> enhanced mutational robustness                                                 state mutations simulated (mitosis and/or meiosis) (Figure 2D <br />   Using S. digitalis assessed maximum mutation rate               Text S1). <br /> R populations random distributions genes                              An analysis recently recorded distribution 4,300 single <br /> recombination hotspots resist extinct (the                 crossover events 50 meioses yeast [42] indicated non-random <br /> mutational robustness Rmax). We compared results                  distributions crossovers essential genes entire yeast <br /> mutational robustness obtained S. cerevisiae chromosome                genome. We resulting average level clustering is <br /> IX architecture, deviates significantly random                    2.5s higher level expected random <br /> arrangement essential genes meiotic recombination                       distributions (Table 1 Text S1). Using S. digitalis, obtained <br /> hotspots distribution [30]. This revealed superior mutational                comparative data subjecting digitalized implementations all <br /> robustness yeast chromosome architecture entire                 chromosomes (Text S1) survival competition against <br /> spectrum inbreeding fractions (Supplementary Figure 4A;                     randomly generated chromosomes. The simulation outcome <br /> Supplementary Figure 2A Text S1). We obtained                      attests superior fitness yeast chromosomes <br /> result allowing mutations occur meiosis only,               (Figure 4D). <br />  <br />  <br />  <br />  <br /> Figure 4. Competitive fitness advantage yeast chromosomes presence lethal mutations. (A) Mutational robustness Rmax of <br /> digital populations random chromosomes S. cerevisiae chromosome IX, respectively, different inbreeding fractions. 3,000 experiments <br /> were performed chromosome configuration. A different random architecture composed total number meiotic recombination <br /> hotspots essential genes generated reference experiment. A histogram summarizing results provided Supplementary <br /> Figure 2A Text S1. The digitized chromosome IX architecture based essential gene data obtained www.yeastgenome.org on <br /> data genome-wide distribution meiotic recombination double strand break sites obtained [32]. The representation inset shows <br /> meiotic recombination hotspot (HS) essential gene (EG) densities chromosome IX sliding window analysis (see Text S1). The population <br /> size cap set 200 individuals. (B) Results survival competition experiments S. cerevisiae chromosome IX versus randomly generated <br /> chromosome architectures (n = 10 experiments grid point), entire inbreeding/outbreeding domain high mutation rates Rmax. <br /> The level dominance color-encoded (bright red: chromosome IX wins experiments, bright blue: random architecture wins all <br /> experiments). The population size cap set 2,000 individuals. (C) Results survival competition experiments architectures described in <br /> (B), wide range mutation rates (in logarithmic steps, R 1024 1) different population size caps (150, 1,500 15,000). The <br /> competitive advantage (&#226;&#8364;&#732;&#226;&#8364;&#732;comp. adv.&#226;&#8364;&#8482;&#226;&#8364;&#8482;) indicates percentage chromosome IX wins random architectures. This percentage formed an <br /> average entire inbreeding/outbreeding domain (bars indicate SD). (D) Results survival competition experiments sixteen yeast <br /> chromosomes versus randomly generated chromosome architectures (n = 170 experiments bar; population size cap: 10,000), averaged the <br /> entire inbreeding/outbreeding domain. Mutation rates color-encoded; green: 1022, blue: 1021, red: 1. A competitive advantage 100% indicates <br /> that yeast chromosome architecture outperformed random architectures. A value 50% indicates yeast chromosome <br /> exhibited performance identical random architectures. For simulation details statistical information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis <br /> Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g004 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                6                             June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                          Gene Order Evolution Yeast <br />  <br />  <br />   Taken together, data suggest yeast-like chromosome                  Achiasmate meiosis enhances mutational robustness <br /> architectures contain evolved features consistent selection               digital genomes <br /> imposed lethal mutations.                                                      The tight physical linkage essential genes single, <br />                                                                                large cluster likely configuration natural genomes. <br /> CEN-linked peripheral essential gene clusters provide                      However, achiasmate meiosis (the absence meiotic crossovers) <br /> cumulative benefits                                                            results comparable situation, genetically links together <br />    The fitness advantage structure chromosome IX relative           essential genes chromosome. In following, use <br /> to randomly structured chromosomes result essential gene              word &#226;&#8364;&#732;&#226;&#8364;&#732;achiasmate&#226;&#8364;&#8482;&#226;&#8364;&#8482; denote absence meiotic crossing <br /> clusters centromere-linked peripheral                essential genes present chromosome. <br /> chromosome, cumulatively both. We designed synthetic                   Meiosis crossovers reported species <br /> chromosome architectures discern potential impact              [43] suggested occur hemiascomycete yeast <br /> structural relationships. First, simulations MAT loci,          Saccharomycodes ludwigii [44,45]. Using survival competition <br /> highest average Rmax obtained synthetic chromosomes,               assay achiasmate meiosis exhibits high <br /> which essential genes distributed large clusters                 mutational robustness Rmax (Supplementary Figure 3 Text <br /> (Figure 5A). Moreover, variability Rmax function            S1, genomes essential gene cluster mating <br /> inbreeding ratio larger case chromosomes           types, +MAT) provides particularly strong fitness advantage <br /> essential gene clustering. The number clusters providing best           entire investigated parameter space (considering the <br /> performance depends population size: larger                     inbreeding fraction mutation rate R) MAT <br /> population smaller optimal number clusters pool              present (Figure 6A, panel). <br /> essential genes distributed (Figure 5B). For clustered                 Generally, advantage achiasmate meiosis be <br /> architectures, introduction MAT locus (which constitutes           expected pure outbreeding. The advantage achiasmate <br /> obligatory heterozygosity) clusters increased              meiosis observed simulations pure outbreeding <br /> persistence compared genotypes MAT locus (Figure 5A,              domain explained finite population size, which <br /> inset; Supplementary Figure 3 Text S1). Using survival                  implies individuals related certain degree. <br /> competition assays genomes MAT-linked clusters,                        Without MAT linkage essential gene cluster, the <br /> compared fitness genotypes peripheral essential genes              advantage reduced, significant mutation rates R,1.5 <br /> either clusters random distributions. We obtained fitness           non-extreme inbreeding fractions (0,i,1) (Figure 6A, lower <br /> advantage peripheral clustering wide range inbreeding              panel). Using direct competition, strong advantage of <br /> ratios mutation rates (Figure 5C). Thus, MAT-centromere-                   achiasmate meiosis random chromosomes mutation rates <br /> linked clusters peripheral essential gene clusters provide                 R 1024 1, increases increasing <br /> cumulative fitness benefits.                                                   population size (Figure 6B 6C). In achiasmate meiosis, linkage <br />  <br />  <br />  <br />  <br /> Figure 5. Effect essential gene clustering mutational robustness. (A) Mutational robustness Rmax function inbreeding ratio <br /> (n = 3 inbreeding ratio; error bars indicate SD), clustered model chromosome (seven EG clusters separated regions containing meiotic <br /> recombination hotspots) random chromosomes. Each experiment performed different randomly generated chromosome <br /> containing number elements (400 NEGs, 100 EGs, 166 recombination hotspots). The inset shows average increase Rmax SD <br /> over inbreeding/outbreeding domain response introduction MAT locus. The red blue dots Rmax-axis indicate average <br /> over inbreeding fractions (left: MAT, right: MAT). In clustered model chromosomes, MAT locus placed EG cluster; in <br /> random chromosomes, MAT locus placed centre chromosome. The population size cap set 200 individuals. (B) Rmax <br /> (average entire inbreeding/outbreeding domain, bars indicate SD) function level EG clustering (ranging perfect clustering (1) <br /> with EGs joined single continuous cluster, maximally unclustered architecture (100) pair EGs separated one <br /> meiotic recombination hotspot). The population size cap color-encoded (ranging 50 1,600 individuals). Simulation time conservatively <br /> assigned; increase maximum number generations did change simulation outcome significantly. (C) Results survival <br /> competition experiments model chromosomes MAT-linked cluster clusters chromosomal arm region (CF1+5) versus <br /> model chromosomes MAT-linked cluster random EG distribution arm region (CF1+R), entire inbreeding/outbreeding <br /> domain high mutation rates Rmax (n = 10 experiments grid point). The competitive advantage color-encoded (bright red: CF1+5 won <br /> all competitions, bright blue: CF1+R won competitions). The population size cap set 1,000 individuals. For simulation details statistical <br /> information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g005 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                7                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                               Gene Order Evolution Yeast <br />  <br />  <br />  <br />  <br /> Figure 6. Fitness advantage achiasmate meiosis. (A) Results survival competition experiments random chromosomes versus achiasmate <br /> chromosomes (super-cluster architectures) EGs cluster linked (top panel) linked MAT locus (lower panel). The matrix shows <br /> the color-encoded level dominance entire inbreeding/outbreeding domain high mutation rates Rmax. The population size cap <br /> was set 2,000 individuals. (B) Results survival competition experiments achiasmate chromosomes versus random chromosomes, wide <br /> range mutation rates (in logarithmic steps, R 1024 1) different population sizes (150, 1,500, 15,000). The MAT locus was <br /> placed chromosome. The competitive advantage indicates percentage achiasmate chromosome wins random architectures. <br /> This percentage formed average inbreeding/outbreeding ratios (bars indicate SD inbreeding/outbreeding domain). (C) As for <br /> (B), MAT locus positioned centromere different chromosome genome. (D) Autosomal chromosomes linked the <br /> MAT locus centromeres case intratetrad mating. The cartoon illustrates simple scenario autosomal chromosome sex <br /> chromosome. The autosomal chromosome contains heterozygous mutation (1). The mutation linked centromere; case <br /> for achiasmate meiosis. Also MAT locus linked centromere; case S. cerevisiae achiasmate meiosis. If loci are <br /> linked corresponding centromere, intratetrad mating (from (4) (5)) involve chromatids derived homologous <br /> chromosomes segregated meiosis I (from (2) (3)). This reconstitute heterozygous situation sites genome fulfill this <br /> criterion. In population genetic terminology, type mating called intratetrad mating division restitution. It demonstrated to <br /> occur species, including fern, flies fungi [22] constituted driving force evolution sex chromosomes. For <br /> simulation details statistical information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g006 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                   8                             June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                        Gene Order Evolution Yeast <br />  <br />  <br /> to MAT locus occurs physically (on chromosome                  exclusively based rules govern evolutionary process <br /> where MAT locus located) centromeres (for               genomes context unicellular breeding populations <br /> other chromosomes intratetrad mating). This preserves                  subjected deleterious mutations. <br /> heterozygosity autosomal centromeres (see Figure 6D). The                     In investigation silico evolution clustered <br /> resulting selection advantage provide population-genetic                  genomes, dissected complementary process: the <br /> reasoning secondary loss meiotic crossing                  maintenance essential gene clusters context of <br /> Saccharomycodes ludwigii.                                                     chromosomal rearrangements linkage MAT locus <br />                                                                               gene cluster. We designed initial genome contained all <br />                                                                               essential genes large cluster meiotic recombination <br /> De novo evolution essential gene clustering                             hotspots outside cluster. Scenarios MAT loci <br /> complex chromosomes                                                           considered inbreeding-only domain (full intratetrad <br />    Our experiments demonstrated fitness advantage clustered              mating), favorable successful persistence the <br /> chromosome architectures exposed deleterious mutations.               lineage situation MAT (see Figure 6A). For +MAT <br /> This fitness advantage result cumulative effects arising        scenarios, MAT locus placed outside essential gene <br /> from pre-existing essential gene clusters, does allow        cluster. Populations evolved using different rearrangement <br /> deduce clustered genomes evolve unclustered               rates (r). For rearrangement rates r,1024, simulated architectures <br /> random architectures solely exposure lethal                     MAT loci preserved highly significant levels <br /> mutations. For example, alternative potentially synergistic               essential gene clustering anti-correlated meiotic crossover <br /> mechanisms conceivable (see Discussion). In order constitute           distributions periods 150,000 generations <br /> a driving force, presence deleterious load               (Figure 7A 7B, Video S1). Without MAT, high preservation <br /> cause emergence clusters self-organized manner,                   essential gene clustering observed relatively <br />  <br />  <br />  <br />  <br /> Figure 7. Maintenance novo evolution essential gene clustering. (A) Maintenance EG clustering inbreeding populations. <br /> Initially, EGs meiotic recombination hotspots arranged separate clusters. Chromosomes allowed rearrange positional <br /> swapping genes meiotic recombination hotspots evolution period 150,000 generations. The level essential gene clustering was <br /> determined end experiment using clustering score analysis (as described Text S1). Color scheme: green indicates significant <br /> clustering 150,000 generations. A high clustering score indicates high level essential gene clustering regions low meiotic <br /> recombination hotspot density. (B) Time-course clustering value different regimes (A). The evolution populations 1 2 is <br /> visualized side Video S1. (C) Schematic illustration small model genome essential genes, designed study the <br /> mechanisms involved evolution clustering (see (D)). The clustering scores (red) possible architectures (green) shown the <br /> scheme. Symbols: light box = EG; black box = NEG; circle = meiotic recombination hotspot; circle cross = meiotic recombination coldspot. (D) <br /> Distributions clustering scores inbreeding populations evolved initial genome shown (C). The simulation performed the <br /> presence MAT locus (green line) MAT locus (red line) (n = 1,000 experiments configuration). The distribution of <br /> clustering scores random architectures shown blue serves reference (n = 2?106). In presence MAT locus, clustering evolved <br /> over broad range rearrangement rates (Supplementary Figure 4 Text S1). For simulation details statistical information Text S1, section <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g007 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               9                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                     Gene Order Evolution Yeast <br />  <br />  <br /> narrow range mutation rates (0.6,R,1.4). In presence              rates close &#226;&#8364;&#732;&#226;&#8364;&#732;destructive regime&#226;&#8364;&#8482;&#226;&#8364;&#8482;, emerging <br /> MAT loci, however, clustering maintained                    cluster quickly scrambled. This setting allowed to <br /> significantly broader range mutation rates (R$0.2)              simulate genomic restructuring quantitatively occur over <br /> for higher rearrangement rates (Figure 7A). We               long evolutionary timescales using reasonable amounts of <br /> mating type locus relocated position inside             computation time. However, up-scaling rearrangement rate <br /> cluster stochastic rearrangement (usually 1,000         necessitates up-scaling mutation rate, order arrive at <br /> generations, n = 20 experiments) remained cluster            selective pressure par potential destructive force <br /> afterwards. This experiment demonstrates significant level           introduced random rearrangements. If computation time <br /> of clustering preserved presence lethal mutations.         unlimited, expect qualitatively comparable <br /> Less well-clustered architectures arise presence             outcome lower values R context lower <br /> destructive forces (genomic rearrangements) quenched           rearrangement rates. <br /> selective pressure better-performing clustered archi-              We able qualitatively reproduce results respect <br /> tectures, long rearrangement rate high.                evolution essential gene clustering additional <br />    In step, investigated novo evolution MAT-          series experiments. In experiments, provided the <br /> linked essential gene clusters small model genomes containing            simulation framework unclustered architectures assembled <br /> five essential genes, non-essential genes recombina-          genetic building blocks S. cerevisiae chromosomes VI, <br /> tion hotspots (Figure 7C). We genomes evolved MAT                VII X (Supplementary Figure 5A Text S1 data not <br /> linked essential gene clusters broad range mutation rates         shown). Even large chromosomes chromosome VII- and <br /> and rearrangement rates, consistent fact             X-like sizes, highly significant levels essential gene clustering <br /> inbreeding preserves MAT-linked heterozygosity (Supplementary               reproducibly established. Moreover, similar results were <br /> Figure 7D; Supplementary Figure 4 Text S1). Architectures            obtained using externally linked mating type locus well <br /> with single cluster (2+1+1+1, 3+1+1, 4+1 5 genes)                 using lower rearrangement rates longer evolution <br /> consistently favored multi-cluster architectures (2+2+1 2+3         periods (r = 1025 200,000 generations, Supplementary Figure <br /> genes) (Figure 7C 7D; grey rectangles indicate single-cluster           5A Text S1). <br /> architectures Figure 7D).                                                   Taken together, silico experiments demonstrate that <br />    In order investigate evolution large yeast-like                deleterious mutations inactivating important genes provide a <br /> chromosome architectures switched domains 50%                    sufficient driving force reproducibly evolve chromosome <br /> inbreeding, advantage chromosome-peripheral clus-               architectures resembling natural counterparts respect <br /> tering apparent extreme breeding domains = 0           essential gene distributions, meiotic recombination hotspot <br /> i = 1 (Figure 4 Figure 5C). For series experiments,          distributions MAT-centromere linkage. Therefore, recurrent <br /> implemented species barrier simulation (see             exposure lethal mutations select genome architectures in <br /> section Results, Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Results            order account associated load context sexual <br /> and Discussion&#226;&#8364;&#8482;&#226;&#8364;&#8482;). Thereby, genomic rearrangement leads             cycle. <br /> the formation new species. This scenario mimics reproductive <br /> isolation meiotically incompatible chromosomes. A new                Competitive advantage novo evolved genome <br /> species eventually dominate population                  architectures <br /> extinct depending reproductive success arising                 Any evolutionary process, successful, generate <br /> fitness (dis-)advantage particular genome architecture. We           individuals perform better conditions their <br /> found simple life cycle mitosis, meiosis mating         evolution ancestors. In order determine level of <br /> sufficient reproducibly evolve genomes MAT-linked           success evolutionary simulation, performed survival <br /> as non-random peripheral essential gene distributions (Figure 8A            competition experiments evolved genomes discussed <br /> and Video S2). In order obtain good statistics                   random chromosomes yeast chromosome IX. We <br /> phenomenon, parallelized assay using grid computing,              observed strong fitness advantage +MAT genomes evolved at <br /> which allowed simultaneously evolve unrelated                    R = 1.0 competing random chromosomes yeast <br /> populations (n = 3,000 experiments). On average 15,000                chromosome IX (Figure 8B). As expected presence of <br /> generations, high-R +MAT populations reproducibly evolved                 large MAT-associated essential gene clusters genomes, the <br /> level clustering 2s mean level encountered random           fitness advantage results wide range mutation rates <br /> architectures. The evolution level clustering low-        (R$1023) entire inbreeding/outbreeding domain. The <br /> R +MAT populations 2MAT populations required 2&#226;&#8364;&#8220;3                   genomes +MAT populations evolved R = 0.1 performed <br /> fold longer period. Importantly, high-R +MAT populations            significantly better random architectures, slightly <br /> even small fraction populations eventually arrived        better chromosome IX architecture, exception of <br /> the level clustering natural yeast chromosome IX                  mixed breeding ratios high mutation rates (R = 1.0), which <br /> (emerging 30,000&#226;&#8364;&#8220;70,000 generations, statistics provided          chromosome IX performed better. In statistical average, low-R <br /> in legend Figure 8A). Genomes MAT-linked cluster            +MAT populations exhibited overall performance <br /> dominated high mutation rates (R = 1), genomes              chromosome IX architecture (Figure 8C). <br /> several clusters, associated MAT locus,                  To expand analysis, performed competitions <br /> typically evolved lower mutation rates (R = 0.1).                        chromosome X-like products evolution experiment <br />    We point parameter space explored               shown Supplementary Figure 5A Text S1 random <br /> evolution clustering constitutes compromise enforced                architectures actual S. cerevisiae chromosome X, <br /> limitations computation time. The computation events              using chromosome description derived [42]. The results of <br /> during generation requires approximately seconds CPU             competition experiments qualitatively comparable to <br /> time population 4,000 individuals. In order able           obtained chromosome IX (Supplementary Figure 5B and <br /> perform statistically meaningful number experiments              5C Text S1). The reproduction results context of <br /> different conditions, applied relatively high rearrangement              chromosome X particularly striking, digitalized <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             10                           June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                            Gene Order Evolution Yeast <br />  <br />  <br />  <br />  <br /> Figure 8. Evolution yeast-like essential gene clustering. (A) Evolution chromosome architectures yeast chromosome IX-like size <br /> and genetic content. Beginning completely unclustered chromosomes, populations MAT allowed evolve different <br /> mutation rates R indicated (nred = 830, ngreen = 826, nblue = 839 experiments). Clustering scored measuring size largest <br /> essential gene cluster average size remaining clusters. For genomes containing MAT, largest cluster observed be <br /> linked MAT. The scatter plot shows scores obtained evolution period 100,000 generations. Score distributions different <br /> populations spanned axes (including reference distributions randomly generated populations size). The &#226;&#8364;&#732;&#226;&#8364;&#732;clustering <br /> score&#226;&#8364;&#8482;&#226;&#8364;&#8482; n quantifies level essential gene clustering. It defined sum squared sizes essential gene clusters (maximum-sized groups <br /> of essential genes disrupted hotspot), normalized total number essential genes meiotic recombination hotspots. Random <br /> architectures chromosome IX genetic content score n = 107617 (SD), chromosome IX architecture itself scores n = 224. The <br /> percentages indicate fraction experiments yielding genomes clustering value n 2s average score randomly <br /> generated genomes (n$141). The insets meiotic recombination hotspot essential gene densities obtained sliding window analyses of <br /> four selected genomes (indexed iv). (B) Survival competition evolved genomes shown (A) versus chromosome IX random genome <br /> architectures. The matrices average statistical results evolved genomes green red groups (A). (C) The bars indicate the <br /> total number wins evolved architectures chromosome IX respectively competition experiments shown (B). For simulation <br /> details statistical information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g008 <br />  <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 11                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                            Gene Order Evolution Yeast <br />  <br />  <br /> chromosome X architecture exhibits highest level essential               switching populations load higher lethal mutations <br /> gene clustering sixteen yeast chromosomes (see Table 1                diploid (Supplementary Figure 6 Text S1). In regime, <br /> Text S1) constitutes particularly challenging                   small percentage diploidisation haplo-selfing (2%) <br /> opponent evolution products survival competition.                constituted strong disadvantage. <br />   We conclude that, context reference life cycle,                    To assay impact haplo-selfing fitness different <br /> architectures chromosome IX-like purging evolve                     chromosome configurations, performed analyses. First, <br /> regime characterized lower mutation rate. The high-R                     compared mutational robustness (Rmax) chromosome IX <br /> regime promotes evolution single large gene clusters,                random chromosomes different levels haplo-selfing. A <br /> extreme represents achiasmate meiosis.                                 high haplo-selfing rate 50% leads significant increase the <br />                                                                                 mutational robustness Rmax, random architectures for <br /> Mating type switching influences lethal load                             chromosome IX. At lower rate 10% haplo-selfing, the <br /> populations                                                                     advantage chromosome IX architecture remained, the <br />    So far, investigated correlation lethal                  mutational robustness decreased compared situation <br /> mutations essential genes parameters govern                     haplo-selfing (see Supplementary Figure 2A 2E in <br /> population fitness respect crossing population                 Text S1). We performed competitive advantage experi- <br /> genetics simulations simple life cycles. There            ments chromosome IX vs. random chromosomes 10% haplo- <br /> processes influence lethal load present populations             selfing mutation rates 1023 Rmax. In the <br /> (see Discussion). In particular, mating type switching followed            absence mutagenic pre-load, noticed emergence a <br /> by mating daughter cells (termed &#226;&#8364;&#732;&#226;&#8364;&#732;haplo-selfing&#226;&#8364;&#8482;&#226;&#8364;&#8482;)               phase transition parameter space high mutation rates, <br /> respective mothers prominent feature life cycle S.               indicating region dominated random architectures <br /> cerevisiae (but yeasts, Discussion). Mating type                 (Figure 9). However, extreme scenario chromo- <br /> switching leads homozygous diploid, involves              IX architectures, average, performed better the <br /> haploid genome free lethal mutations. Hence,                      random architectures (57% competition wins chromosome IX <br /> occurring significant rates, haplo-selfing expected              vs. 43% competition wins random architectures). <br /> decrease lethal load populations. However, mutagenic <br /> load population high, small probability           Meiotic crossover rates S. cerevisiae optimal <br /> of generating viable diploids haplo-selfing. Comparing pre-                    Using S. digitalis, determined influence meiotic <br /> loaded chromosomes haplo-selfing revealed                    crossover rate chromosomes fitness ability to <br /> sharp transition competitive advantage favor non-                  purge mutational load. In yeasts, crossover rates vary considerably, <br />  <br />  <br />  <br />  <br /> Figure 9. The effect mating type switching population fitness. Results survival competition experiments S. cerevisiae <br /> chromosome IX versus randomly generated chromosome architectures (n = 10 (top row) 100 (bottom row) experiments grid point), the <br /> presence (10%) absence mating type switching. Different breeding conditions mutation rates (between 1023 Rmax) considered. <br /> The matrixes row outcome high-resolution analysis high-R regime, pronounced differences were <br /> observed. The level dominance color-encoded (bright red: chromosome IX wins experiments, bright blue: random architecture wins all <br /> experiments). The population size cap set 2,000 individuals. For simulation details statistical information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis <br /> Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g009 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                12                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                         Gene Order Evolution Yeast <br />  <br />  <br /> from 0 (achiasmate meiosis) approximately 20&#226;&#8364;&#8220;40 crossovers               revealed increase Rmax increasing crossover rates, <br /> per chromosome S. pombe. The genomic mean S. cerevisiae             reaching saturation levels (95&#226;&#8364;&#8220;99.5%) range 4.5&#226;&#8364;&#8220;8 <br /> chromosomes 5.6 crossovers meiosis (derived                    crossovers. The observed variability Rmax function the <br /> genetic map, www.yeastgenome.org) [46]. This value                         inbreeding fraction minimal 95&#226;&#8364;&#8220;99.5% saturation <br /> confirmed direct assessment crossover frequency                  interval (Figure 10A). A slight dependency chromosome length <br /> distribution [42]. The chromosome-specific number crossovers                apparent (Figure 10A, inset). Direct competition chromo- <br /> scales linearly number genes (R2 = 0.89), range            IX populations subjected natural crossover rate with <br /> 2.5&#226;&#8364;&#8220;9 crossovers individual chromosomes (Supplementary                     chromosome IX populations modified crossover rates demon- <br /> Figure 7 Text S1). We used random genomes different sizes                strated crossing rates higher lower naturally <br /> (250&#226;&#8364;&#8220;1,500 genes) assess mutational robustness Rmax                observed average lead decrease fitness advantage <br /> function crossover frequency inbreeding fraction. This                  (Figure 10B; Supplementary Figure 8 Text S1 plot of <br />  <br />  <br />  <br />  <br /> Figure 10. Natural crossing rates yeast chromosomes optimal. (A) Rmax function number crossovers meiosis <br /> (logarithmic scale) populations random chromosomes 750 genes (20% EGs). Bars indicate SD Rmax entire inbreeding/ <br /> outbreeding domain. Inset: Number crossovers meiosis required reach 95% 99.5% respectively maximum mutational robustness <br /> Rmax, chromosomes different lengths (250, 500, 750, 1,000, 1,500 genes) random configurations. (B) Results survival competition <br /> experiments chromosome IX populations subjected different crossing rates (n = 5 experiments grid point), mutation <br /> rates R Rmax entire inbreeding/outbreeding domain. The color-<span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> indicates winning chromosome architecture. (C) Results of <br /> survival competition experiments yeast chromosome IX random chromosomes different crossing rates (n = 10 experiments per <br /> grid point). Same experimental conditions (B). Supplementary Figure 8 Text S1 shows plots average fitness advantages results (B) <br /> and (C). &#226;&#8364;&#732;&#226;&#8364;&#732;minimum&#226;&#8364;&#8482;&#226;&#8364;&#8482; = crossover chromosome meiosis; &#226;&#8364;&#732;&#226;&#8364;&#732;maximum&#226;&#8364;&#8482;&#226;&#8364;&#8482; = 58 hotspots active meiosis. For simulation details and <br /> statistical information Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;S. digitalis Simulation Settings.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> doi:10.1371/journal.pgen.1000533.g010 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               13                           June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                     Gene Order Evolution Yeast <br />  <br />  <br /> the average performance). Moreover, assessing competition               freely segregating single mutagenic events affecting essential <br /> experiments chromosome IX versus random architectures                function/gene. Importantly, frequency tetrads one <br /> function crossover rate, chromosome IX performed best            dead spore low, indicating low cumulative lethal effects of <br /> regime yeast-like crossing rates (Figure 10C;                    non-lethal mutations recombined haploid genome during <br /> Supplementary Figure 8 Text S1 plot average                  meiosis. Taken together, data suggest losses genomes <br /> performance). This suggests natural rates meiotic                associated random mutagenic events frequently associated <br /> crossovers S. cerevisiae adapted handle deleterious load.          single events leading lethal haploid phenotype. <br />  <br /> Lethal mutations frequently associated single                       Mating rescues genomes associated lethal <br /> loci                                                                         mutations <br />    Haploid lethal phenotypes caused mutational                        In scenario, linked mutation providing advantage is <br /> inactivation essential gene, arise                   lost soon lethal mutation exposed, e.g. by <br /> consequence cumulative effects non-lethal mutations                 haploid growth following meiosis homozygotisation. However, <br /> essential non-essential genes. Dissecting synthetic lethal               genome lethal mutation preserved mating <br /> relationships demonstrated 0.8&#226;&#8364;&#8220;4% chance lethality                 spore containing wild type allele. In order test <br /> two non-essential genes deleted [47]; simple extrapolation             haploid lethal mutations high low chance to <br /> predicts 50% chance <span id='am-322' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-323' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> inactivation            render spores unable mate, investigated rates diploid <br /> approximately 7&#226;&#8364;&#8220;14 non-essential genes, assuming scalability (and            colony formation single dyads using fluorescence activated cell <br /> ignoring possibility non-linear network properties,           sorting (FACS) different mutation accumulation periods <br /> positive epistasis [48]). In case, however, cells           (Supplementary Figure 9 Text S1). The observed frequency of <br /> would died reaching high load,                     diploid colonies did change significantly function of <br /> inactivation 19% essential genes. This                  accumulated mutations, compared wild type. If spores <br /> empirical analysis indicate haploid lethal                   associated lethal mutations frequently unable mate &#226;&#8364;&#8220; <br /> phenotypes caused inactivation single essential gene,            lethal mutation itself co-accumulated <br /> as hypothesized purpose investigation.                     non-lethal mutations &#226;&#8364;&#8220; greater two-fold decrease diploid <br />    In order test hypothesis directly, generated diploid         colonies course experiment been <br /> strain containing MSH2 control weak fully                   expected (Figure 11C). Consistently, very <br /> repressible GalS promoter order perform mutation accumu-               minor fraction viable spores impaired mating that <br /> lation experiments. Deletion mismatch repair gene MSH2                dead spores germinate form microcolonies <br /> leads greatly elevated levels point mutations [49,50].                 (see Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Results Discussion&#226;&#8364;&#8482;&#226;&#8364;&#8482;; <br /> Additionally, reduced sequence specificity homologous recom-             Supplementary Figure 10 Text S1 histogram the <br /> bination observed [51,52], leading increased levels                colony sizes). This result consistent our <br /> recombination similar sequences separate loci (ectopic            observation majority lethal phenotypes caused <br /> recombination). Mismatch repair deficient strains accumulate high            meiosis I non-disjunction, spores lack entire chromo- <br /> levels lethal mutations, associated            somes usually fail germinate. Taken together, results show <br /> an increase gross chromosomal rearrangements [53].                        majority randomly occurring lethal mutations did not <br />    Growth conditional mutator strain glucose-containing           prevent spores mating. <br /> medium led depletion Msh2 cells (Figure 11A and <br /> 11B) accumulation mutant phenotypes reduced               Discussion <br /> viability. In order restrict accumulation mutations to <br /> vegetative growth, shifted cells galactose-containing                 We investigated influence deleterious mutations the <br /> medium induce MSH2 expression prior sporulation                        evolution genome architectures. In breeding unicellular diploid <br /> (Figure 11A), prevents aberrant post-meiotic segrega-             organisms subjected high mutation rates, frequent <br /> tion events ectopic recombination associated MSH2               inactivation important genes selects non-random distribu- <br /> deletion [54]. When grown exclusively galactose-containing                tions important/essential genes meiotic recombination <br /> medium, GalS-MSH2 strain exhibited wild type spore viability             hotspots. In simulated experiments architectures evolve, which <br /> (Figure 11C). Upon mutation accumulation approximately 30&#226;&#8364;&#8220;               bear striking resemblance architectures natural yeast <br /> 36 generations (three growth periods day each), tetrad                chromosomes provide competitive advantage wide <br /> analysis (n = 400) revealed little 3-spore 1-spore viability (8%          range mutation rates. <br /> and 15.5%). The majority tetrads unviable spores <br /> contained viable spores (36%), caused                  Purging deleterious mutations <br /> single locus events leading haploid lethality. Alternatively,           The selective advantage clustered genomes arises from <br /> 2-spore viability resulted meiosis I non-disjunction.          multiple effects cumulatively improve fitness such <br /> We tested option measuring linkage lethal load         genomes. Initially, average mutational load (m) simulated <br /> 2-spore viable asci heterozygous leu2/LEU2 locus,               population subjected mutation rate R increases an <br /> itself centromere-linked (5 cM), lethal load           architecture-dependent equilibrium reached. In equilibrium, <br /> exhibited average partial linkage leu2 (27 cM). Since         influx new stochastic mutations equal outflux <br /> load locus different analyzed tetrad,                     associated loss individuals population (Figure 2B). <br /> approximately 25% cases crossing                     Upon elimination individual, mutations associated its <br /> centromere/leu2 locus prevented (see Text S1, section                     genome removed population (Supplementary <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Results Discussion&#226;&#8364;&#8482;&#226;&#8364;&#8482;), tight               Figure 1A Text S1). Within simulation framework, loss <br /> centromere linkage load consequence                      individuals occurs factors. First, random removal of <br /> homologous non-disjunction meiosis I. Therefore prominent             excess genomes population size cap entirely neutral, <br /> fraction tetrads viable spores likely caused         affects evolved sub-populations same <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              14                          June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                            Gene Order Evolution Yeast <br />  <br />  <br />  <br />  <br /> Figure 11. Mutation accumulation genetic analysis lethality mating. (A) Outline mutation accumulation experiment. <br /> Diploid GalS-MSH2/GalS-MSH2 cells grown presence glucose repress MSH2 expression (MSH2off) growth periods 24 hours <br /> (1/1000 dilution cultures). After growth period, aliquot removed GalS promoter induced spreading cells <br /> on galactose-containing plates 24 hours order prevent accumulation mutations. Upon sporulation, products meiosis were <br /> analyzed viability using tetrad analysis (results given main text) sorting single spores dyads FACS. The ability mating was <br /> analyzed testing mating types sorted dyads order determine formed colonies haploid (one spore unviable, no <br /> mating) diploid (mating). (B) Msh2 levels GalS-MSH2 control cells grown galactose (1), glucose 24 hours (2) and <br /> finally galactose 6 hours (3); detection Western blotting antibodies specific Msh2 (Santa Cruz, Msh2 (yC-15): sc-26230). (C) <br /> Viability spores dyads. For time point 1,536 spores 384 dyads FACS-sorted array patterns YP-Gal plates (140 mm <br /> diameter). Spore viability determined counting number formed colonies. Mating efficiency spores dyads determined by <br /> counting fraction colonies consisted diploid cells. In wild type cells 0 h GalS-MSH2 strain, fraction 53%. The <br /> probability having viable spores opposite mating type dyad calculated time-dependent spore viability the <br /> fraction diploid formation 0 h. These data allowed calculate probability diploid formation (mating), unviable spores unable to <br /> mate (dashed line). <br /> doi:10.1371/journal.pgen.1000533.g011 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                15                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                         Gene Order Evolution Yeast <br />  <br />  <br /> probability. Second, importantly, genomes lost               interacting mechanisms exist conceivable explain <br /> to homozygotisation mutant locus. The probability                   observation. As result, difficult precisely define <br /> homozygotisation new mutation vegetative                     scenario mechanism dominant and <br /> division proportional mutational load present                  different mechanisms interact other. The situation <br /> genome, irrespective genome architecture. A higher                     complicated, usually exceptions each <br /> mutational load population lowers fitness               explanation (e.g. species, certain aspects different): <br /> mitosis increased sensitivity new mutations,            yeast does exhibit centromeric repeats; species, in <br /> the heterozygous load entirely neutral. Therefore,                crossovers occur preferentially centromere-proximal <br /> process outcome improved meiosis loss              regions [59], etc. The fact molecular mechanisms <br /> individuals homozygotisation mating. Since homo-                 provide support different scenarios (crossovers close or <br /> zygotisation single recessive lethal mutation removes              away centromeres) makes difficult deduce the <br /> whole genome population, including associated                    causal connection. <br /> mutations, effect clustering explained generally              Exploring specific mechanism singularly able to <br /> reduction total number individuals lost              provide valid explanation constitutes strategy tackling this <br /> mating. This maximization mutation purging caused                issue. In case S. cerevisiae, evolutionary scenario able to <br /> evolutionary optimization genome context sexual            recapitulate evolution crossover suppression essential <br /> lifestyle. Supplementary Figure 11 Text S1 illustrates simple            gene enrichment pericentromeric regions. Our simulation <br /> scenario.                                                                     provides evidence lethal mutations chromosome <br />     Clustering increases level genetic load                evolution interact. But, course, record true <br /> maintained genome. In order obtain positive overall               historical events enforced constellation. The complete <br /> effect, clustering needs compensate increased mitotic losses,          answer given exploring (derived) genomes all <br /> which simulations random chromosome architectures                       ancestors S. cerevisiae evolution molecular <br /> constitute approximately 20% homozygotisation losses                   mechanisms govern centromeric crossover protection (which <br /> during life cycle. In simulation scenarios,              subject investigation). <br /> large EG cluster evolved (R = 1.0), mitotic losses reach                 While took advantage recent data Mancera et al. [42] <br /> same magnitude meiotic losses (Supplementary Figure 12                  experiments involving digitalization natural yeast <br /> Text S1).                                                                     chromosomes, digital implementation chromosome IX was <br />     In conclusion, advantage clustered genomes arise              derived using hotspot mapping data Gerton et al. [32]. In <br /> from increase reproductive fitness mitosis meiosis.              meantime, additional hotspot data sets published <br /> This results optimized balancing factors                  [60,61]. Using data sets double strand breaks predict the <br /> influence frequency, homozygotisation lethal              distribution crossovers accompanied limitation that <br /> load locus occurs (Supplementary Figure 1B Text S1                     DSB frequencies translate linearly crossover frequen- <br /> Supplementary Figure 11 Text S1). In mitosis, homozygotisa-                cies. In sense Gerton et al. data set (which data <br /> tion lethal mutations dependent new mutations                set available time began analysis) helpful [32], <br /> global load population, meiosis, homozygotisation             provided correct description cold centromeres, as <br /> results mating. This influenced crossover distribution            confirmed yeast genetic map. Two recent studies <br /> and frequencies, breeding behavior genetic linkage                [60,61] detected DSBs near centromeres, which, however, <br /> relationship crossover sites essential gene loci.                 convert crossovers (www.yeastgenome.org, Text <br />     In experiments, applied mutation rates higher         S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Results Discussion&#226;&#8364;&#8482;&#226;&#8364;&#8482; a <br /> average <span id='am-244' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>deleterious mutation</span> rate reported S. cerevisiae             quantitative assessment cM-to-kb relationship pericen- <br /> strains [37,55,56]. However, noted description               tromeric regions). These studies deviate the <br /> simulation (see Results, section &#226;&#8364;&#732;&#226;&#8364;&#732;Simulation digital yeast genomes        Gerton et al. study rest genome (except the <br /> with S. digitalis&#226;&#8364;&#8482;&#226;&#8364;&#8482;), did assume exponential growth          subtelomeric regions, relevant context). <br /> populations, implemented scenario species <br /> compete limited pool nutrients. In scenario                 The mutator scenario genome evolution <br /> implementation round mitosis consecutive                  Natural isolates S. cerevisiae reported exhibit large <br /> meioses mimics scenario mitoses population                  diversity terms mutagenic load, yielding significant fraction <br /> growth. Although properly growth               isolates low extremely low spore viability. Some <br /> population local niche short period time, certainly          proportion load, accumulated during <br /> does recapitulate longer evolutionary periods include               mitotic growth preserved inbreeding [62,63], may <br /> consecutive sexual cycles. It reasonable assume natural            caused mutator phenotypes. Widespread occurrence <br /> yeast populations undergo mitoses meioses. Hence,            natural variation rise mutator phenotypes has <br /> mutation rate compared natural mutation rate,         reported [64,65]; similar type variation be <br /> be divided average number consecutive mitoses. This                 reason formation natural yeast strains highly <br /> number, however, currently known.                                      unstable karyotypes [66]. These caused impaired <br />                                                                               recombinational repair, indicated certain dependency on <br /> Crossing suppression near centromeres                            Rad52 [67]. Interestingly, strains rise meiotic <br /> hotspot&#226;&#8364;&#8220;crossover relation                                                    offspring stable karyotypes. <br />    Crossover suppression near centromeres caused                  Based considerations speculate genomic <br /> the interference pericentromeric crossovers proper meiotic            rearrangements, associated formation new <br /> chromosome segregation [57]. Alternatively, proposed              species lineages, frequently associated high <br /> that cold centromeres caused requirement protect               mutation rates occasionally &#226;&#8364;&#8220; particular the <br /> centromeric repeats [58]. These considerations relate general            periods shape genome &#226;&#8364;&#8220; exert selective pressure by <br /> issue evolutionary biology: frequently, independent             means high levels lethal mutations. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              16                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                         Gene Order Evolution Yeast <br />  <br />  <br />    The yeast evolutionary history governs million                 The occurrence deleterious mutations predicted select <br /> years includes species (at nowadays)                       evolution properties increase robustness, the <br /> predominantly diploid life cycles [19,68,69]. Although average             evolved genomes lower maximum fitness mutation- <br /> frequency meioses unknown (from general perspective               free environment. This compensated higher mean fitness <br /> well individual species), sufficient number              variants present mutant population, known &#226;&#8364;&#732;&#226;&#8364;&#732;survival <br /> sexual cycles genomic rearrangements taken place                 flattest&#226;&#8364;&#8482;&#226;&#8364;&#8482; [77]. Support provided studies viruses and <br /> to allow maintenance selection particular genome               bacteria, studies using digital organisms [78]. An <br /> architectures.                                                                 important body literature summarized Wilke Adami <br />                                                                                (2003) [79]). <br /> Mating type switching loss heterozygosity (LOH)                            In yeast, mechanisms exist account buffering <br />     In yeast, mating highly favored process occurs                   mutational load. One global buffering positive epistasis the <br /> whenever cells opposite mating type meet 2 dense                     fitness reduction combinations non-essential gene deletions <br /> populations level spores tetrad [31,70]. Even            [48]. Essential genes exhibit expression noise non- <br /> considering S. cerevisiae aborts formation           essential genes [80,81]. Batada Hurst [36] proposed that <br /> to spores sporulation (due limited availability             evolution essential gene clustering driven their <br /> nutrients), overall formation spores mating partner              accumulation chromosomal regions low average nucleo- <br /> available tetrad 80%                  occupancy (open chromatin), domains lower <br /> broad range conditions [31]. This circumstance               expression noise coincide domains low <br /> possibility mating spores different tetrads indicates          meiotic recombination [82]. Although idea intriguing, no <br /> mating type switching likely dominant way                   support provided scenario results fitness <br /> diploidisation S. cerevisiae. Thus, appears safe assume          advantage sufficient select relocation essential gene <br /> the relative effects observed level 10% haplo-selfing               cluster. S. cerevisiae grows predominantly diploid. Since the <br /> represents overestimation actual situation,           individual intrinsic noise copy gene uncorrelated <br /> underestimation.                                                               noise copy, lower total noise level present <br />     Mating type switching key inventions lifestyle           diploids compared haploids [83]. Selection low noise for <br /> variation emerged time point genome               essential genes effective diploids in <br /> duplication [68,71]. One wonder reasons                 haploids. Only 9% heterozygous deletion strains of <br /> evolutionary invention place, causes                 essential genes exhibit haplo-insufficiency S. cerevisiae [84], <br /> recurrent secondary loss (mating type switching deficient S.                   indicating low noise particularly important a <br /> cerevisiae constitute 10% strains isolated               diploid situation, copy essential gene inactivated <br /> natural wine fermentation, [72]). Lifestyles evolve            mutation. This minimize noise context an <br /> a preference diploid stages [73], heterozygosity           overall reduced protein level. This result suggest a <br /> maintained. An associated need efficient purging load               significant interaction exist low noise model and <br /> have influenced evolution mating type switching. Occa-                  purging mutations diploids, act synergistically <br /> sionally, mutation rates high meiotic cycles                improving clustering essential genes. <br /> too infrequent, resulting high lethal load able                  Proliferation tumor cells depends inactivation tumor <br /> select mating type switching. Diploidy context               suppressor genes subsequent multiple mutations. Due early- <br /> lethal load enforce evolution high levels                   acquired mutations DNA mismatch repair pathways <br /> intratetrad mating unicellular eukaryotes [23], observed              required genetic stability [10,85&#226;&#8364;&#8220;87], mutator phenotypes have <br /> the non-switching pre-WGD duplication yeast species Saccharomyces              potential accelerating progression cancer develop- <br /> kluyveri [74] Saccharomycodes ludwigii [45,75]. These consider-            ment enhancement variability Darwinian <br /> ations exclude additional roles mating            selection act [9,10,88]. In scenario, evolved robustness <br /> type switching intratetrad mating, suggest lethal            haploid lethal load important factor has <br /> mutations mutagenic load frequently accompany diploid                  implications understanding genetic instability cancer <br /> life cycles unicellular organisms.                                          development. <br />     Additional mechanisms exist influence lethal load in <br /> diploid populations. Loss heterozygosity associated mitosis            Conclusions <br /> acts position-specific manner increased frequency               Purging deleterious load depends tendency to <br /> away centromere. It provide                        accumulate load presence alleles increase the <br /> additional reason essential gene enrichment pericentromeric             rate mutations occur (i.e. mutator alleles). The robustness <br /> regions. However, frequency low (approximately 0.5 10            inherent genome architecture yeast indicates <br /> 10,000 cell divisions young cells 50 500 old mother               high mutation rates genetic load played an <br /> cells [76]) unlikely significantly influence        important role evolution, ability deal with <br /> global lethal load populations.                                         deleterious load co-evolved better suited genome architec- <br />                                                                                tures. Altogether increasing evidence deleterious load <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Survival flattest&#226;&#8364;&#8482;&#226;&#8364;&#8482; improve                        common evolving yeast populations aspects of <br />    In race beneficial mutations exposure new                     cellular physiology evolved allow survival fit &#226;&#8364;&#732;&#226;&#8364;&#732;flat&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> conditions, essential genes effectively reduce genetic drift,         species. <br /> their inactivation caused significant fitness reduction the <br /> diploid organism. Consequently, chance acquiring new                    Methods <br /> mutations immediately accessible original genome <br /> would decrease, e.g. reducing frequency beneficial                      Yeast mutator experiment <br /> mutations accompanied mutation causes                     A diploid strain carrying chromosomal MSH2 control of <br /> loss essential function.                                                 inducible GalS promoter constructed [89] well- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               17                            June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                                            Gene Order Evolution Yeast <br />  <br />  <br /> sporulating SK1 genetic background [90]. The cells                                    size 20 elements). Histograms sliding window analyses are <br /> constantly maintained GalS inducing conditions (YP-                                  shown bottom. Initially, EGs located of <br /> galactose/raffinose (YP-Gal), Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;Supple-                          chromosomes, recombination hotspots form cluster in <br /> mentary Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482;) exhibited wild type spore viability. In                             middle non-essential genes (at end the <br /> order allow random mutations occur, GalS promoter                            chromosome). At high mutation rates, EG clustering is <br /> repressed using glucose cells grown 24, 48                              maintained (left, movie shows period 100,000 generations). At <br /> 72 hours (24 hours&lt;10&#226;&#8364;&#8220;12 generations) using serial dilution. At                            low mutation rates, EG clustering maintained EGs <br /> each time point, aliquot cells (5?107 cells) plated                       distributed random patterns (right). Both simulations <br /> YP-Gal 16 hours. The cells washed off, plated                           used identical values parameters R. <br /> sporulation plate (1% KAc, 0.02% raffinose, 0.02% galactose)                           Found at: doi:10.1371/journal.pgen.1000533.s003 (1.26 MB <br /> incubated 40 hours room temperature. Ascus formation                                MOV) <br /> occurred cases frequencies .99%. After disrupting the <br /> asci, FACS sorting used spot single spores dyads                             Video S2 Evolution EG clustering. Visualization the <br /> large YP-Gal plates (1536 spores 384 dyads plate). Sorted                           evolution clustered genome architectures function time in <br /> spores dyads analyzed colony formation (viability)                            population representative high-R +MAT evolution <br /> and mating type (MATa, MATa mating type = diploid cells)                            experiment shown Figure 8A Main Text. The left panel <br /> using mating type tester strains halo assay (Text S1, section                        shows density difference sliding window plot genomes <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Supplementary Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; Supplementary Figure 9 Text                              (red = essential genes, blue = hotspots, green = equal <br /> S1). The viability spores 400 different tetrads                            densities essential genes hotspots). The panels <br /> analyzed tetrad dissection.                                                             essential gene hotspot densities entire population. <br />                                                                                            The average clustering score population (white, SD red) is <br /> Simulation framework yeast genome evolution&#226;&#8364;&#8221;S.                                          indicated right panel (dashed white line = level of <br /> digitalis                                                                                  clustering random architectures [107617]). The simulation <br />   The source <span id='am-401' about='Thesaurus:code' typeof='owl:Thing'>code</span> simulation S. digitalis provided                            starts maximally unclustered genome architecture. The <br /> Protocol S1. A detailed description simulation included                       inbreeding percentage 50%. Movie playback speed: 6,000 <br /> Text S1, section &#226;&#8364;&#732;&#226;&#8364;&#732;The Computer Simulation S. digitalis;&#226;&#8364;&#8482;&#226;&#8364;&#8482;                          generations second (400 generations frame). <br /> Table 2 Text S1.                                                                        Found at: doi:10.1371/journal.pgen.1000533.s004 (7.17 MB <br />                                                                                            MOV) <br /> Supporting Information <br />                                                                                            Acknowledgments <br /> Text S1 The simulation S. digitalis; supplementary <br /> results discussion; supplementary materials methods;                               We thank Sarah Otto, Lars Juhl Jensen, Michael Hood, Darren Gilmour, <br />                                                                                            unknown reviewers valuable comments constructive <br /> supplementary figures tables; S. digitalis simulation settings. <br />                                                                                            discussions; Nicola Hastings Audun Lier helpful comments the <br /> Found at: doi:10.1371/journal.pgen.1000533.s001 (1.26 MB PDF)                              manuscript. We appreciate generous support Manfred <br /> Protocol S1 S. digitalis simulation. Matlab source code. Version                           Alef, Urban Liebel, Marc Hemberger, Michael Wahlers using the <br /> 1.79 (2005&#226;&#8364;&#8220;2009), Philipp J. Keller, EMBL Heidelberg.                                      clusters EMBL Karlsruhe Institute Technology. <br /> Found at: doi:10.1371/journal.pgen.1000533.s002 (0.05 MB ZIP)                              We thank Andrew Riddell Alexis Perez Gonzalez help flow <br />                                                                                            cytometry. Franklin Stahl kindly acknowledged advice modeling <br /> Video S1 Maintenance EG clustering low high                                   crossover interference yeast Deepankar Pal contributions early <br /> mutation rates. Maintenance essential gene clustering                                experiments. We grateful Ernst Stelzer supporting PJK. Eugenio <br /> evolving inbreeding populations data points                              Mancera, Wolfgang Huber, Lars Steinmetz kindly acknowledged <br /> Figure 7A Main Text. The right panel corresponds data                                sharing data prior publication. <br /> point #1, left corresponds data point #2. <br /> The graphs genomic element densities                           Author Contributions <br /> entire population. Chromosomes vertically aligned. The color-                          Conceived designed experiments: PJK MK. Performed the <br /> coding reports differences density EGs recombination                         experiments: PJK MK. Analyzed data: PJK MK. Contributed <br /> hotspots (sliding window analyses individual genomes, window                            reagents/materials/analysis tools: PJK. Wrote paper: PJK MK. <br />  <br /> References <br /> 1. Kondrashov AS (1993) Classification hypotheses advantage                   10. Nowak MA, Michor F, Komarova NL, Iwasa Y (2004) Evolutionary dynamics of <br />    amphimixis. J Hered 84: 372&#226;&#8364;&#8220;387.                                                            tumor suppressor gene inactivation. Proc Natl Acad Sci USA 101: <br /> 2. Muller HJ (1932) Some Genetic Aspects Sex. American Naturalist 66:                       10635&#226;&#8364;&#8220;10638. <br />    118&#226;&#8364;&#8220;138.                                                                                11. Thompson DA, Desai MM, Murray AW (2006) Ploidy controls success of <br /> 3. Fisher RA (1930) The Genetical Theory Natural Selection. Oxford, UK:                     mutators nature mutations budding yeast evolution. Curr Biol 16: <br />    Oxford University Press.                                                                    1581&#226;&#8364;&#8220;1590. <br /> 4. Zeyl CW, Otto SP (2007) A short history recombination yeast. Trends Ecol          12. Kondrashov AS, Crow JF (1991) Haploidy diploidy: better? Nature <br />    Evol 22: 223&#226;&#8364;&#8220;225.                                                                           351: 314&#226;&#8364;&#8220;315. <br /> 5. Maynard Smith J, Szathmary E (1995) The major transitions eovlution.                 13. Papp B, Pal C, Hurst LD (2003) Dosage sensitivity evolution gene <br />    Oxford, UK: Oxford University Press.                                                        families yeast. Nature 424: 194&#226;&#8364;&#8220;197. <br /> 6. Metzgar D, Wills C (2000) Evidence adaptive evolution mutation rates.        14. Sliwa P, Kluz J, Korona R (2004) Mutational load transition between <br />    Cell 101: 581&#226;&#8364;&#8220;584.                                                                          diploidy haploidy inexperimental populations yeast Saccharomyces <br /> 7. Sniegowski PD, Gerrish PJ, Lenski RE (1997) Evolution high mutation rates                cerevisiae. Genetica 121: 285&#226;&#8364;&#8220;293. <br />    experimental populations E. coli. Nature 387: 703&#226;&#8364;&#8220;705.                            15. Aa E, Townsend JP, Adams RI, Nielsen KM, Taylor JW (2006) Population <br /> 8. Taddei F, Radman M, MaynardSmith J, Toupance B, Gouyon PH, et al. (1997)                    structure gene evolution Saccharomyces cerevisiae. FEMS Yeast Res 6: <br />    Role mutator alleles adaptive evolution. Nature 387: 700&#226;&#8364;&#8220;702.                         702&#226;&#8364;&#8220;715. <br /> 9. Beckman Ra, Loeb La (2006) Efficiency carcinogenesis              16. Ruderfer DM, Pratt SC, Seidel HS, Kruglyak L (2006) Population genomic <br />    mutator mutation. Proc Natl Acad Sci USA 103: 14140&#226;&#8364;&#8220;14145.                                analysis outcrossing recombination yeast. Nature Genet 38: 1077&#226;&#8364;&#8220;1081. <br />  <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                           18                                June 2009 | Volume 5 | Issue 6 | e1000533 <br />                                                                                                                                                 Gene Order Evolution Yeast <br />  <br />  <br /> 17. Wei W, McCusker JH, Hyman RW, Jones T, Ning Y, et al. (2007) Genome                     53. Heck Ja, Gresham D, Botstein D, Alani E (2006) Accumulation recessive <br />     sequencing comparative analysis Saccharomyces cerevisiae strain YJM789.              lethal mutations Saccharomyces cerevisiae mlh1 mismatch repair mutants not <br />     Proc Natl Acad Sci USA 104: 12825&#226;&#8364;&#8220;12830.                                                    associated gross chromosomal rearrangements. Genetics 174: 519&#226;&#8364;&#8220;523. <br /> 18. Zakharov IA (2005) Intratetrad mating genetic evolutionary                  54. Reenan RaG, Kolodner RD (1992) Characterization Insertion Mutations in <br />     consequences. Genetika 41: 508&#226;&#8364;&#8220;519.                                                         Saccharomyces cerevisiae Msh1 Msh2 Genes - Evidence Separate <br /> 19. Knop M (2006) Evolution hemiascomycete yeasts: life styles                Mitochondrial Nuclear Functions. Genetics 132: 975&#226;&#8364;&#8220;985. <br />     importance inbreeding. BioEssays 28: 696&#226;&#8364;&#8220;708.                                        55. Hill Ja, Otto SP (2007) The role pleiotropy maintenance sex yeast. <br /> 20. Baudat F, Nicolas A (1997) Clustering meiotic double-strand breaks yeast              Genetics 175: 1419&#226;&#8364;&#8220;1427. <br />     chromosome III. Proc Natl Acad Sci USA 94: 5213&#226;&#8364;&#8220;5218.                                   56. Joseph SB, Hall DW (2004) Spontaneous mutations diploid Saccharomyces <br /> 21. Gallegos A, Jacobson DJ, Raju NB, Skupski MP, Natvig DO (2000) Suppressed                   cerevisiae: beneficial expected. Genetics 168: 1817&#226;&#8364;&#8220;1825. <br />     recombination pairing anomaly mating-type chromosome                    57. Brar Ga, Hochwagen A, Ee LS, Amon A (2009) The Multiple Roles Cohesin <br />     Neurospora tetrasperma. Genetics 154: 623&#226;&#8364;&#8220;633.                                              Meiotic Chromosome Morphogenesis Pairing. Mol Biol Cell 20: <br /> 22. Hood ME, Antonovics J (2004) Mating meiotic tetrad                       1030&#226;&#8364;&#8220;1047. <br />     maintenance genomic heterozygosity. Genetics 166: 1751&#226;&#8364;&#8220;1759.                         58. Charlesworth B, Langley CH, Stephan W (1986) The Evolution Restricted <br /> 23. Johnson LJ, Antonovics J, Hood ME (2005) The evolution intratetrad mating                Recombination Accumulation Repeated DNA-Sequences. Genetics <br />     rates. Evolution 59: 2525&#226;&#8364;&#8220;2532.                                                             112: 947&#226;&#8364;&#8220;962. <br /> 24. Antonovics J, Abrams JY (2004) Intratetrad mating evolution linkage          59. Guo XM, Allen SK (1996) Complete interference nonrandom distribution <br />     relationships. Evolution 58: 702&#226;&#8364;&#8220;709.                                                       meiotic crossover mollusc, Mulinia lateralis (Say). Biolog Bulletin 191: <br /> 25. Ohno S (1967) Sex chromosomes sex-linked genes. New York, NY:                           145&#226;&#8364;&#8220;148. <br />     Springer-Verlag.                                                                        60. Blitzblau HG, Bell GW, Rodriguez J, Bell SP, Hochwagen A (2007) Mapping of <br /> 26. Rice WR (1987) The Accumulation Sexually Antagonistic Genes                         meiotic single-stranded DNA reveals double-strand-break hotspots near <br />     Selective Agent Promoting Evolution Reduced Recombination                    centromeres telomeres. Curr Biol 17: 2003&#226;&#8364;&#8220;2012. <br />     Primitive Sex-Chromosomes. Evolution 41: 911&#226;&#8364;&#8220;914.                                       61. Buhler C, Borde V, Lichten M (2007) Mapping meiotic single-strand DNA <br /> 27. Petes TD (2001) Meiotic recombination hot spots cold spots. Nature Rev                  reveals new landscape DNA double-strand breaks Saccharomyces cerevisiae. <br />     Genet 2: 360&#226;&#8364;&#8220;369.                                                                           PLoS Biol 5: 2797&#226;&#8364;&#8220;2808. doi:10.1371/journal.pbio.0050324. <br /> 28. Hurst LD, Pal C, Lercher MJ (2004) The evolutionary dynamics eukaryotic              62. Johnston JR, Baccari C, Mortimer RK (2000) Genotypic characterization of <br />     gene order. Nature Rev Genet 5: 299&#226;&#8364;&#8220;310.                                                    strains commercial wine yeasts tetrad analysis. Res Microbiol 151: <br /> 29. Lercher MJ, Urrutia AO, Hurst LD (2002) Clustering housekeeping genes                    583&#226;&#8364;&#8220;590. <br />     provides unified model gene order human genome. Nature Genet 31:            63. Mortimer RK, Romano P, Suzzi G, Polsinelli M (1994) Genome renewal: new <br />     180&#226;&#8364;&#8220;183.                                                                                    phenomenon revealed genetic study 43 strains Saccharomyces cerevisiae <br /> 30. Pal C, Hurst LD (2003) Evidence co-evolution gene order                          derived natural fermentation grape musts. Yeast 10: 1543&#226;&#8364;&#8220;1552. <br />     recombination rate. Nature Genet 33: 392&#226;&#8364;&#8220;395.                                           64. Demogines A, Smith E, Kruglyak L, Alani E (2008) Incompatibilities Involving <br /> 31. Taxis C, Keller P, Kavagiou Z, Jensen LJ, Colombelli J, et al. (2005) Spore                 Yeast Mismatch Repair Genes: A Role Genetic Modifiers Implications <br />     number control breeding Saccharomyces cerevisiae: key role self-             Disease Penetrance Variation Genomic Mutation Rates. PLoS Genet <br />     organizing system. J Cell Biol 171: 627&#226;&#8364;&#8220;640.                                                4: e1000103. doi:10.1371/journal.pgen.1000103. <br /> 32. Gerton JL, DeRisi J, Shroff R, Lichten M, Brown PO, et al. (2000) Inaugural             65. Heck JA, Argueso JL, Gemici Z, Reeves RG, Bernard A, et al. (2006) Negative <br />     article: global mapping meiotic recombination hotspots coldspots              epistasis natural variants Saccharomyces cerevisiae MLH1 PMS1 <br />     yeast Saccharomyces cerevisiae. Proc Natl Acad Sci USA 97: 11383&#226;&#8364;&#8220;11390. <br />                                                                                                 genes results defect mismatch repair. Proc Natl Acad Sci USA 103: <br /> 33. Lambie EJ, Roeder GS (1988) A Yeast Centromere Acts Cis Inhibit                       3256&#226;&#8364;&#8220;3261. <br />     Meiotic Gene Conversion Adjacent Sequences. Cell 52: 863&#226;&#8364;&#8220;873. <br />                                                                                             66. Carro D, Pina B (2001) Genetic analysis karyotype instability natural <br /> 34. Lercher MJ, Hurst LD (2006) Co-expressed yeast genes cluster long range <br />                                                                                                 wine yeast strains. Yeast 18: 1457&#226;&#8364;&#8220;1470. <br />     regularly spaced. J Mol Biol 359: 825&#226;&#8364;&#8220;831. <br />                                                                                             67. Carro D, Bartra E, Pina B (2003) Karyotype rearrangements wine yeast <br /> 35. Poyatos JF, Hurst LD (2006) Is optimal gene order impossible? Trends Genet 22: <br />                                                                                                 strain rad52-dependent rad52-independent mechanisms. Appl Environ <br />     420&#226;&#8364;&#8220;423. <br />                                                                                                 Microbiol 69: 2161&#226;&#8364;&#8220;2165. <br /> 36. Batada NN, Hurst LD (2007) Evolution chromosome organization driven by <br />                                                                                             68. Fabre E, Muller H, Therizols P, Lafontaine I, Dujon B, et al. (2005) <br />     selection reduced gene expression noise. Nature Genet 39: 945&#226;&#8364;&#8220;949. <br />                                                                                                 Comparative genomics hemiascomycete yeasts: Evolution sex, silencing, <br /> 37. Wloch DM, Szafraniec K, Borts RH, Korona R (2001) Direct estimate the <br />                                                                                                 subtelomeres. Mol Biol Evo 22: 856&#226;&#8364;&#8220;873. <br />     mutation rate distribution fitness effects yeast Saccharomyces <br />                                                                                             69. Piskur J, Langkjaer RB (2004) Yeast genome sequencing: power of <br />     cerevisiae. Genetics 159: 441&#226;&#8364;&#8220;452. <br />                                                                                                 comparative genomics. Mol Microbiol 53: 381&#226;&#8364;&#8220;389. <br /> 38. Eyre-Walker A, Keightley PD (2007) The distribution fitness effects new <br />     mutations. Nature Rev Genet 8: 610&#226;&#8364;&#8220;618.                                                 70. Barkai N, Rose MD, Wingreen NS (1998) Protease helps yeast mating <br />                                                                                                 partners. Nature 396: 422&#226;&#8364;&#8220;423. <br /> 39. Szafraniec K, Wloch DM, Sliwa P, Borts RH, Korona R (2003) Small fitness <br />     effects weak genetic interactions deleterious mutations                  71. Wolfe KH, Shields DC (1997) Molecular evidence ancient duplication of <br />     heterozygous loci yeast Saccharomyces cerevisiae. Genet Res 82: 19&#226;&#8364;&#8220;31.               entire yeast genome. Nature 387: 708&#226;&#8364;&#8220;713. <br /> 40. Kondrashov AS (2001) Sex U. Trends Genet 17: 75&#226;&#8364;&#8220;77.                                 72. Mortimer RK (2000) Evolution variation yeast (Saccharomyces) genome. <br /> 41. Zakharov IA (2005) Intratetrad mating genetic evolutionary                      Genome Research 10: 403&#226;&#8364;&#8220;409. <br />     consequences. Russian J Genet 41: 402&#226;&#8364;&#8220;411.                                              73. Bell G (1994) The comparative biology alteration generations. In: <br /> 42. Mancera E, Bourgon R, Brozzi A, Huber W, Steinmetz LM (2008) High-                          Kirkpatrick M, ed. Lectures mathematics life sciences: theories the <br />     resolution mapping meiotic crossovers non-crossovers yeast. Nature                evolution haploid-diploid life cycles: Am Math Soc, Providence, R. I. pp 1&#226;&#8364;&#8220;26. <br />     454: 479&#226;&#8364;&#8220;U471.                                                                          74. Mccullough J, Herskowitz I (1979) Mating Pheromones Saccharomyces kluyveri - <br /> 43. Zickler D, Kleckner N (1999) Meiotic chromosomes: integrating structure                 Pheromone Interactions Saccharomyces kluyveri Saccharomyces cerevisiae. <br />     function. Ann Rev Genet 33: 603&#226;&#8364;&#8220;754.                                                        J Bacteriol 138: 146&#226;&#8364;&#8220;154. <br /> 44. Yamazaki T, Ohara Y, Oshima Y (1976) Rare occurrence tetratype                   75. Winge LO (1939) Saccharomycodes ludwigii Hansen, balanced heterozygote. <br />     tetrads Saccharomycodes ludwigii. J Bacteriol 125: 461&#226;&#8364;&#8220;466.                              C R Trav Lab Carlsberg Ser Physiol 22: 357&#226;&#8364;&#8220;380. <br /> 45. Lindegren CC (1945) Yeast Genetics: Life Cycles, Cytology, Hybridization,           <br /> </body></html>