<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>3374&#226;&#8364;&#8220;3388 Nucleic Acids Research, 2008, Vol. 36, No. 10                                                            Published online 27 April 2008 <br /> doi:10.1093/nar/gkn108 <br />  <br />  <br /> A role Caf1 mRNA deadenylation and <br /> decay trypanosomes human cells <br /> Angela Schwede1, Louise Ellis2, Julia Luther3, Mark Carrington2, <br /> Georg Stoecklin3,4 Christine Clayton1,4,* <br /> 1 <br />  Zentrum fu&#194;&#168;r Molekulare Biologie der Universita&#194;&#168;t Heidelberg (ZMBH), Im Neuenheimer Feld 282, D-69120 <br /> Heidelberg, Germany, 2Department Biochemistry, 80 Tennis Court Rd., Cambridge CB2 1GA, UK, <br /> 3 <br />  German Cancer Research Centre (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany <br /> and 4DKFZ-ZMBH Allianz <br />  <br /> Received October 15, 2007; Revised February 25, 2008; Accepted February 26, 2008 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        Ccr4/Caf1/Not complex (3&#226;&#8364;&#8220;5). Vertebrates have, in <br />                                                                                 addition, DAN/PARN enzyme (6). Evidence far <br /> The eukaryotic Ccr4/Caf1/Not complex involved                                indicates Ccr4/Caf1/Not complex major <br /> in deadenylation mRNAs. The Caf1 Ccr4 sub-                               eukaryotic deadenylase, activities&#226;&#8364;&#8221;PARN <br /> units potentially deadenylating enzyme                                Pan2/Pan3 complex&#226;&#8364;&#8221;play secondary more <br /> activity. We investigate roles Ccr4                             specialized roles (1,7). <br /> Caf1 deadenylation organisms sepa-                                  The central component Ccr4/Caf1/Not complex <br /> rated early eukaryotic evolution: humans                                 Not1, conserved eukaryotic <br /> trypanosomes. In Trypanosoma brucei,                                 evolution serves sca&#239;&#172;&#8364;old attachment of <br /> complex containing CAF1, NOT1, NOT2 NOT5,                                   subunits; essential all <br /> DHH1 possible homologue Caf130;                                     organisms investigated far. Not1p binds Caf1p (8), <br /> homologue Ccr4 found. Trypanosome CAF1                                   turn binds Ccr4p. A leucine-rich domain the <br /> has deadenylation activity, essential cell                           N-terminus Ccr4p required binding Caf1p in <br /> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span>. Depletion trypanosome CAF1 delayed                                 yeast humans conserved Ccr4 <br /> deadenylation degradation constitutively                                 homologues Drosophila Caenorhabditis elegans <br />                                                                                 (9&#226;&#8364;&#8220;11). Other subunits complex Not2, Not5, <br /> expressed mRNAs. Human cells isozymes <br />                                                                                 Not3, Not4, ubiquitin ligase, Caf4p, Caf16p, Caf40p <br /> of Caf1: simultaneous depletion inhibited <br />                                                                                 Caf130p [see Table 1 (12&#226;&#8364;&#8220;14)]; clear <br /> degradation unstable reporter mRNA. In                               subunits associated simultaneously <br /> species, depletion Caf1 homologues inhibited                                 roles unclear (15&#226;&#8364;&#8220;18). In addition, the <br /> deadenylation bulk RNA resulted                                    putative helicase Dhh1p interacts complex both <br /> increase average poly(A) tail <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>.                                        physically genetically (19&#226;&#8364;&#8220;21). <br />                                                                                    A variety observations suggest yeast, Ccr4 has <br />                                                                                 major deadenylation activity, Caf1 serves mainly <br />                                                                                 attach Ccr4 rest complex (22,23). Ccr4 <br /> INTRODUCTION                                                                    (an exonuclease III) 1000 times higher speci&#239;&#172;?c activity <br /> Degradation eukaryotic mRNAs critical cellular                         Caf1, resembles RNase D (24,25). Neither <br /> homoeostasis. The &#239;&#172;?rst, rate-limiting step degradation                       protein essential yeast deletion mutants show <br /> is usually removal poly(A) tail. Subsequently,                       slow growth deadenylation defect (5,25&#226;&#8364;&#8220;27). <br /> mRNA degraded 30 ! 50 direction, probably                         Critically, yeast  caf1 mutants complemented <br /> by exosome. Alternatively, 50 cap removed                         over-expression Ccr4p, expression a <br /> decapping enzymes mRNA degraded                               mutant mouse CAF1 lacking active site (4,24); but <br /> 50 ! 3 direction. RNAs subject rapid, regulated                              human Ccr4, catalytically active the <br /> degradation, bearing AU-rich instability                          yeast homologue, complement yeast  ccr4 <br /> elements, accelerated deadenylation relative                       mutant (3). <br /> stable mRNAs (1,2). Saccharomyces cerevisiae                               Results vertebrates Drosophila, contrast <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> deadenylation complexes, Pan2/Pan3 complex                              yeast, suggest Caf1 important in <br />  <br />  <br /> *To correspondence addressed. Tel: +49 6221 546 876; Fax: +49 6221 545 894; Email: cclayton@zmbh.uni-heidelberg.de <br />  <br /> &#195;&#376; 2008 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />                                                                           Nucleic Acids Research, 2008, Vol. 36, No. 10 3375 <br />  <br />  <br /> Table 1. New plasmids used study <br />  <br /> Plasmid            Description <br />  <br /> pHD1534            DHH1 RNAi                      Part DHH1 ORF ampli&#239;&#172;?ed using forward primer 50 -gag aag atc taa gct tca <br />                                                     cag tca agg act ttg cg-30 reverse primer 50 -cgg aat tcg tcg acc caa tgc ggt gaa ggt <br />                                                     aag t-30 , PCR product cloned p2T7TA blue (39) <br /> pHD1535            LCCR4 RNAi                     Part CCR4-like ORF ampli&#239;&#172;?ed using forward primer 50 -gagaagatctgcatgc- <br />                                                     caaagcaatctgtggaagca-30 reverse primer 50 -cggaattcgtcgacccaaccaccaactcttcgtt-30 , <br />                                                     PCR product cloned p2T7TA blue (39) <br /> pHD1843            CAF1 RNAi                      Part CAF1 ORF ampli&#239;&#172;?ed using forward primer 50 -gaggatcsctctgcctcaca- <br />                                                     gaggatgtg-30 reverse primer 50 -cgggatcccgaggaatcttcacagagcc-30 , PCR product <br />                                                     cloned p2T7TA blue (39) <br /> pHD1844            NOT1 RNAi                      Part NOT1 ORF ampli&#239;&#172;?ed using forward primer 50 -gaggatccgagctggta- <br />                                                     gaagtggacgc-30 reverse primer 50 -cgggatccacgactcgggaccacaatag-30 , PCR product <br />                                                     cloned p2T7TA blue (39) <br /> pHD1847            GST-CAF1                       CAF1 ORF ampli&#239;&#172;?ed using forward primer 50 -gagagaattcatgcagtatggtggcgggac-30 <br />                                                     reverse primer 50 -cgatgaattctcagctatgaccctttaccgc-30 cloned pGEX-  T <br />                                                     (Eco RI digest) <br /> pHD1849            CAF1-TAP tag (C-terminal)      CAF1 ORF ampli&#239;&#172;?ed using forward primer 50 -aagcttatgatgcagtatggtggcggga-30 <br />                                                     reverse primer 50 -gttaacgctatgaccctttaccgcttgtgt-30 cloned pHD 924 <br />                                                     (Hind III, Hpa I digest) (34) <br /> pHD1850            LCCR4- TAP tag (C-terminal)    LCCR4 ORF ampli&#239;&#172;?ed using forward primer 50 -aagcttatggcacggcaccccaacacggc-30 <br />                                                     reverse primer 50 -gttaactttgtacgacaaatcggcaatcaa-30 cloned pHD 1337 <br />                                                     (Hind III, Hpa I digest) (34) <br />  <br />  <br />  <br />  <br /> deadenylation Ccr4. <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi)                 Tsetse &#239;&#172;&#8218;y vector, &#226;&#8364;&#732;procyclic form&#226;&#8364;&#8482; trypomasti- <br /> targeting Caf1, Ccr4, C. elegans resulted       gotes. In bloodstream forms, constitutively expressed <br /> embryonic lethal <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (28,29). Drosophila cells             mRNAs half-lives varying 15 min an <br /> depleted Caf1 showed marked increase overall             hour, mRNAs speci&#239;&#172;?c Tsetse vector <br /> poly(A) length delay deadenylation Hsp70         forms destroyed 5 min; more <br /> mRNA recovery heat shock,                slowly growing procyclic forms, constitutive mRNAs have <br /> Ccr4 deletion mutants viable showed                longer half-lives mRNAs speci&#239;&#172;?c blood- <br /> slight defects deadenylation bulk Hsp70 RNA             stream forms destroyed 5&#226;&#8364;&#8220;10-min half-lives. <br /> (15). The relative roles Caf1 Ccr4 mammalian           The trypanosome exosome play minor role in <br /> deadenylation resolved, essentially            mRNA degradation (34,35). In contrast, putative 50 &#226;&#8364;&#8220;30 <br /> because presence potential homologues.         exonuclease, XRNA, required rapid <br /> Five human genes encode proteins Ccr4-like catalytic        degradation developmentally unstable mRNAs and <br /> domains, two&#226;&#8364;&#8221;Ccr4a Ccr4b&#226;&#8364;&#8221;retain                 operate polyadenylated mRNAs (36). Milone <br /> leucine-rich repeat associate Caf1a (7). Over-          et al. (37) previously demonstrated cap-independent <br /> expression wild-type Ccr4a mammalian cells                 deadenylation activity, inhibited poly(A) <br /> destabilized (normally stable) beta-globin              binding protein, extracts Kinetoplastid <br /> mRNA (7) reporter mRNA containing premature              Leptomonas seymouri deadenylation been <br /> termination codon. In contrast, over-expression             demonstrated vivo [see e.g. (36)]. <br /> catalytically inactive mutant Ccr4a, siRNA treatment             In paper, test roles Caf1 Ccr4 <br /> targeting Ccr4a, slightly delayed deadenylation           mRNA degradation human cells and <br /> mRNA containing premature termination codon,               trypanosomes. <br /> the e&#239;&#172;&#8364;ects subtle (7). There homologues <br /> of Caf1p, called Caf1a Caf1b [(7,14,30)              MATERIALS AND METHODS <br /> Table 1]. Mouse Caf1a processive deadenylase;               Cell culture plasmids <br /> Caf1a /  mice apparently healthy males sterile <br /> (24,31). The roles Ccr4b Caf1b              Procyclic bloodstream form T. brucei stably expressing <br /> examined.                                                        tetracycline repressor (38,39), T7 <br />    So far detailed experiments concerning Ccr4/Caf1/         polymerase expression (39,40), grown, transfected <br /> Not complex performed organisms                clonally selected described ref. (41). Plasmids <br /> single eukaryotic group&#226;&#8364;&#8221;the Opisthokonta (32). The               used study described Table 1. To analyse the <br /> Kinetoplastid protists&#226;&#8364;&#8221;which diverged early            degradation reporter mRNA CAF1 RNAi line <br /> the Opisthokonta&#226;&#8364;&#8221;are unique eukaryotes                  transfected pHD 1034 (42). <br /> relying extremely heavily mRNA degradation the <br /> control gene expression (33). Trypanosoma brucei           CAF1- CCR4-TAP purification <br /> able multiply mammalian bloodstream,                Procyclic cells expressing tet repressor [pHD 1313 (39)] <br /> &#226;&#8364;&#732;bloodstream form&#226;&#8364;&#8482; trypomastigotes, midgut            transfected pHD 918 (tetracycline-inducible <br />  3376 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br /> expression TAP-tag), pHD1849 (tetracycline              washed times 1  PBS incubated three <br /> inducible over-expression TbCAF1-TAP) pHD1850            times 50 mM Tris&#226;&#8364;&#8220;HCl, 10 mM reduced glutathione <br /> (tetracycline inducible over-expression TbLCCR4)            (pH8.0) 10 min room temperature elute fusion <br /> (Table 1). The expression TAP tag alone, TAP-           protein. The purity preparations checked on <br /> tagged TbCAF1 TAP-tagged TbLCCR4 induced                SDS&#226;&#8364;&#8220;page gels stained colloidal Coomassie; single <br /> by adding 1 mg/ml tetracycline cells harvested        dominant band seen faint <br /> 20 h later. For tandem a&#239;&#172;&#402;nity puri&#239;&#172;?cation 6   109 cells        additional bands. Unfortunately, serum rabbit <br /> were used, methodology previously described (43).      immunized puri&#239;&#172;?ed TbCAF1-GST recognized the <br /> The preparation run 10% SDS&#226;&#8364;&#8220;page gel            recombinant protein poor sensitivity, and <br /> and stained SyproRuby (Invitrogen). Bands           failed detect CAF1 trypanosome lysates. A&#239;&#172;&#402;nity <br /> were present TAP-only control identi&#239;&#172;?ed        puri&#239;&#172;?cation did improve sensitivity. <br /> by mass spectrometry. <br />  <br /> Intracellular detection TAP tag                         In vitro deadenylation assay <br />   6 <br /> 10 trypanosomes sedimented, re-suspended,            Ten picomoles RNA oligonucleotide substrate <br /> &#239;&#172;?xed 4% paraformaldehyde (w/v) 1  PBS 18 min,        (see Figure 1B) (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> MWG, Ebersberg) labelled with <br /> sedimented 2 min, re-suspended PBS,           30 U T4 polynucleotide kinase 50 mCi g-32P ATP <br /> allowed settle poly-lysine-coated glass           (GE Healthcare) 378C 30 min. They puri&#239;&#172;?ed <br /> surface 25 min. The cells permeabilized     Sephadex G-25 columns equilibrated 10 mM <br /> 0.2% (v/v) Triton X-100 (in 1  PBS) room temperature        Tris. For assay, substrate (2   105 c.p.m.) incu- <br /> for 20 min. Before staining, slides blocked 0.5%     bated 50 pmol GST-CAF1 10 mM Tris&#226;&#8364;&#8220;HCl and <br /> (w/v) gelatine 20 min. To detect TAP tag, cells   1 mM magnesium acetate. The reaction stopped by <br /> incubated 1:50 000 dilution rabbit a-Protein A       adding 1 volume formamide loading bu&#239;&#172;&#8364;er (95% form- <br /> (Sigma) 1 h 1:500 dilution second        amide, 0.025% xylene cyanol, 0.025% bromophenol <br /> antibody Alexa Fluor 594 goat a- rabbit IgG (Molecular         blue, 18 mM EDTA, 0.025% SDS) appropriate time <br /> probes) 40 min. The kinetoplast nuclear DNA <br />                                                                points. The Decade Marker System (Ambion) used to <br /> were stained 100 ng/ml DAPI/1  PBS for <br />                                                                synthesize 10-bp RNA ladder. The samples run <br /> 10 min. <br />                                                                12% polyacrylamide/urea gels, &#239;&#172;?xed, <br /> RNAi                                                           dried analysed phosphorimager. <br /> For RNAi bloodstream forms, gene-speci&#239;&#172;?c fragments <br /> of  500 nt (44) cloned vector p2T7TA blue        mRNA degradation assays mRNA characterization <br /> (see Table 1). In vector, RNAi fragments are <br /> &#239;&#172;&#8218;anked opposing T7 promoters downstream tet            To analyse mRNA decay trypanosomes, transcription <br /> operators. The resulting plasmids linearized Not     stopped Actinomycin D &#239;&#172;?nal concentration <br /> I transfected bloodstream-form trypanosomes           10 mg/ml. Total RNA isolated using <br /> expressing tet repressor (pHD 1313) T7 RNA             peqGold Trifast (peqLab, Germany) RNeasy mini kit <br /> polymerase (pHD514) (39). Transfectants selected          (QIAGEN) according manufacturer&#226;&#8364;&#8482;s instructions. <br /> in 10 mg/ml hygromycin cloned limiting dilution.           To remove poly(A) tails, 170 pmol oligo d(T) were <br /> RNAi induced adding tetracycline medium          incubated 10&#226;&#8364;&#8220;12 mg total RNA 428C 10 min. <br /> at concentration 1 mg/ml. Samples taken        After 5 U RNaseH (New England Biolabs) and <br /> 18.5&#226;&#8364;&#8220;26 h RNAi induction. For procyclic forms,          RNaseH bu&#239;&#172;&#8364;er added samples incubated <br /> entire CAF1 open reading frame cloned p2T7            40 min 378C. <br /> and transfected trypanosomes containing integrated           Levels mRNAs assessed formaldehyde <br /> plasmids pLEW29 pLEW13 (40). RNAi                agarose denaturing polyacrylamide gel electrophoresis, <br /> induced adding tetracycline medium              followed blotting Nytran membranes (Amersham <br /> concentration 1 mg/ml.                                      Biosciences) hybridization radioactive DNA <br />                                                                (random priming) (Stratagene) anti-sense RNA <br /> Expression GST-CAF1 Escherichia coli                     (MAXIscriptT7/T3, Ambion) probes. Signals mea- <br /> The E. coli strain BL21 transformed pHD1847           sured using phosphorimager. All mRNA signals were <br /> (Table 1). The bacteria grown optical density       normalized relative signal SRP (signal <br /> (600 nm) 2 LB medium 378C. Expression GST-         recognition particle) probe. Northerns stripped by <br /> CAF1 induced 1 mM IPTG bacteria               brief boiling 0.1% SDS; e&#239;&#172;&#402;cacy stripping <br /> were allowed grow 2 h harvesting        assessed phosphorimager residual signals <br /> soni&#239;&#172;?cation 1  PBS. Triton X-100 added        left decay necessary re-use. The half-lives <br /> &#239;&#172;?nal concentration 1%. The sonicate gently mixed        measured plotting relative signal intensities using <br /> for 30 min 48C, cleared centrifugation           Kaleidograph (Synergy software), including the <br /> 12 000g 48C 10 min. The supernatant incubated       segments time course gave exponential <br /> with glutathione&#226;&#8364;&#8220;Sepharose 4B slurry (GE Healthcare)       decay curves (&#239;&#172;?tted linear correlation coe&#239;&#172;&#402;cient <br /> 30 min room temperature column. The slurry         generally exceeding 0.9). <br />                                                                                      Nucleic Acids Research, 2008, Vol. 36, No. 10 3377 <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Diagrammatic representation likely CAF1/NOT complex components trypanosomes, aligned yeast human <br /> homologues. The scale represents amino acid residues. The black bars indicate regions sequence similarity trypanosome and <br /> yeast proteins, percentage identical amino acids shown, using output GeneDB blastp using default settings. (The percentage similar <br /> amino acids 1&#226;&#8364;&#8220;5&#226;&#8364;&#8220;2 times higher.) Grey bars similarly indicate regions sequence similarity human yeast proteins, using the <br /> output Blastp NCBI database default settings. (B) Enzyme activity recombinant TbCAF1. The oligonucleotides shown were <br /> 50 -radiolabelled; samples enzyme incubated 20 min bu&#239;&#172;&#8364;er exclude degradation observed caused by <br /> contaminating RNases (lanes 1 5). Products separated denaturing gel electrophoresis. (C) Proteins associated CAF1-TAP (left), <br /> CCR4L-TAP (right) TAP tag (centre) subjected puri&#239;&#172;?cation IgG calmodulin columns, separated SDS&#226;&#8364;&#8220;PAGE <br /> and stained SyproRuby. The major potential complex components indicated solid arrows; bands analysed and <br /> contained tagged protein only, highly abundant proteins probably contaminants indicated open arrows. Bands the <br /> CAF1 lane numbered right (identities Table 3). HC: hypothetical protein conserved trypanosomatids. (D) Location TAP- <br /> tagged CAF1. Procyclic trypanosomes expressing CAF1-TAP &#239;&#172;?xed permeabilized, TAP tag detected immuno&#239;&#172;&#8218;uorescence. <br /> The kinetoplast (small dot near end parasite) nuclear DNA (larger stain near middle) counter-stained DAPI. <br /> A typical image shown. Cells expressing TAP tag gave &#239;&#172;&#8218;uorescence. (E) Location TAP-tagged LCCR4. Details (D). <br />  3378 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br /> Poly(A) tail assay                                                time, cytoplasmic RNA extracted according to <br />                                                                   ref. 47. Twenty micrograms RNA resolved by <br /> To induce CAF1 RNAi, bloodstream-form trypanosomes <br />                                                                   1.5% agarose/formaldehyde gel electrophoresis as <br /> transfected pHD1843 (Table 1) cultured with <br />                                                                   described previously (48). For Figure 6, b-globin and <br /> 1 mg/ml tetracycline 24 h. Ten micrograms millilitre <br />                                                                   nucleolin mRNA detected digoxigenin-labelled <br /> Actinomycin D added cultures stop <br />                                                                   RNA probes (Roche). Templates generated PCR <br /> transcription samples taken appropriate time <br />                                                                   using primers G1000 G1001 b-globin probe <br /> points. Total RNA isolated using RNeasy mini <br />                                                                   Bex12, primers G83 G1009 nucleolin <br /> kit (QIAGEN) according manufacturer&#226;&#8364;&#8482;s instruc- <br />                                                                   probe. The results quantitated scanning densitom- <br /> tions. Five nanograms total trypanosome <br />                                                                   etry. Oligonucleotides used were: <br /> RNA 1 mg mammalian RNA, 20 U T4 RNA ligase <br />                                                                   G83: 50 -TTACAAAGTCACTCAGGATG-30 ; <br /> (New England BioLabs) 5 mCi pCp (GE Healthcare) <br />                                                                   G1000: 50 -GTGCATCTGTCCAGTG-30 ; <br /> in 50 mM Tris&#226;&#8364;&#8220;HCl, 10 mM MgCl2, 10 mM DTT, 1 mM <br />                                                                   G1001: 50 -GCCGATTTAGGTGACACTATAGAATAC <br /> ATP 10% (v/v) DMSO incubated 378C <br />                                                                   CCTGAAGTTCTC-30 ; <br /> for 1.5 h <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> 30 -ends. After heat inactivation the <br />                                                                   G1009: 50 -GCCGATTTAGGTGACACTATAGAATAC <br /> T4 RNA ligase, labelled RNA incubated 2 mg <br />                                                                   TTAGCGTCTTCG-30 . <br /> RNase A, 5 U T1 (RNase A/T1 mix, Fermentas) and <br />                                                                      For Figure 7, probes 32P-labelled and <br /> 20 mg yeast tRNA (Ambion) 10 mM Tris&#226;&#8364;&#8220;Cl, 2 mM <br />                                                                   hybridization described trypanosome mRNA; <br /> MgCl2, 2 mM DTT, 0.2 mM ATP, 2% (v/v) DMSO, <br />                                                                   signals quantitated using phosphorimager. <br /> 300 mM NaCl 5 mM EDTA 378C 30 min. The <br /> poly(A) tails puri&#239;&#172;?ed phenol/chloroform extrac- <br /> tion ethanol precipitation separated 8%               RESULTS <br /> polyacrylamide/urea gels. Markers used Decade <br /> RNA ladder (Ambion, above), labelled T4                  Trypanosome homologues CCR4/CAF1/NOT <br /> polynucleotide kinase 50 mCi g-32P ATP,                 complex subunits <br /> RNA ladder (Invitrogen), labelled T4 RNA ligase              We searched trypanosome genome database the <br /> and 5 mCi a-32P pCp. Bands detected quanti&#239;&#172;?ed            amino-acid sequences yeast human Ccr4/Caf1/ <br /> in &#239;&#172;?xed dried gels phosphoimager. For              Not complexes, extracted possible homologues and <br /> analysis, poly(A) tails subdivided blocks        con&#239;&#172;?rmed identities reciprocal BLAST analysis <br /> 10 nt, counting bands (Figure 3) using       yeast genome database EMBL database. We <br /> labelled Decade marker (Figure 5). The ratios         possible candidates (reciprocal-blast positive) for <br /> signals &#239;&#172;?rst block blocks             Caf1, Not1, Ccr4, Not2, Not5 Caf40 (Figure 1A and <br /> determined.                                                       Table 2). The matches Caf4 Caf16 restricted <br />                                                                   functional domains, status uncertain. Not3, <br /> Human cell lines siRNA                                        Not4 Caf130 appeared absent. <br /> HTGM5 cells derived human HT1080 &#239;&#172;?bro-                     We &#239;&#172;?rst compared trypanosome CAF1 protein with <br /> sarcoma cells (45). Cells transfected twice             homologues multiple eukaryotic genomes. We found <br /> siRNA duplexes (100 nM) period 96 h using               Caf1 homologues genomes searched, including <br /> Lipofectamine 2000 (Invitrogen). siRNAs designed             Schizosaccharomyces pombe early-branching <br /> using published recommendations (46) purchased                protists Trichomonas (six copies) Giardia. All catalytic <br /> from Ambion. The following target sequences (sense                residues conserved homologues apart that <br /> strand) chosen:                                              S. cerevisiae (see Supplementary Figure 8 the <br /> control siRNA (D0): 50 -GCAUUCACUUGGAUAGUA                        alignment). The &#239;&#172;?rst essential region (starting residue <br /> A-30 ;                                                            188 consensus) sequence DTEF all <br /> Ccr4a siRNA (C4): 50 -CAAGACGGCUGCUGAACU                          homologues S. cerevisiae (STEF). The Asp at <br /> A-30 ;                                                            residue 369 universally conserved. At position 421, the <br /> Ccr4b siRNA (C8): 50 -UGACAGCGCUGCACCTAA                          conserved HQAGSD changed TTTGGQ in <br /> A-30 ;                                                            S. cerevisiae. This supports previous suggestion that <br /> Caf1a siRNA (C5): 50 -UUACGACUUUGGCUACUU                          low deadenylase activity seen S. cerevisiae Caf1 is <br /> A-30 ;                                                            norm (25). <br /> Caf1b siRNA (C6): 50 -GUGCUCAGUUACAGUUAU                             To check enzymatic activity TbCAF1, we <br /> A-30 ;                                                            expressed protein fusion glutathione <br /> Caf1a+b siRNA (C7): 50 -UGAAGAGCUGCAAAAAU                         S-transferase assayed ability digest 5&#226;&#8364;&#8482;-labelled <br /> CU-30                                                             oligonucleotide substrate. TbCAF1 shortened substrate <br /> The e&#239;&#172;&#402;ciency siRNA knockdown determined                 containing 3&#226;&#8364;&#8482;-oligo(A), internal oligo(A) <br /> real-time PCR.                                                    (Figure 1B), con&#239;&#172;?rming deadenylase. <br />                                                                      The best trypanosome match Ccr4 predicted <br />                                                                   protein contained conserved C-terminal exonu- <br /> Quantitation globin-ARE mRNA <br />                                                                   clease domain, lacked leucine-rich repeats <br /> SiRNA-transfected HTGM5 cells treated                   required interaction CAF1 (11) <br /> Actinomycin D (5 mg/ml; Sigma) indicated periods              deadenylase activity (10). In evolutionary trees, <br />                                                                                     Nucleic Acids Research, 2008, Vol. 36, No. 10 3379 <br />  <br />  <br /> the position sequence varied depending                        time) apparent (data shown). After steps of <br /> method used (data shown). It did convincingly                        a&#239;&#172;&#402;nity puri&#239;&#172;?cation, associated proteins identi&#239;&#172;?ed <br /> belong family clearly did branch                         mass spectrometry (Figure 1C). All bands excised from <br /> the con&#239;&#172;?rmed Ccr4 proteins, named LCCR4                             TbLCCR4-TAP puri&#239;&#172;?cation contained predominantly <br /> (Like-Ccr4).                                                                 TbLCCR4 sequence. In addition, identi&#239;&#172;?ed abundant <br />                                                                              proteins routinely contaminate preparations, <br /> Trypanosome CAF1 associated NOT1, NOT2                               tubulin, HSP70 variants, ribosomal proteins and <br /> and NOT5                                                                     glycolytic enzymes, speci&#239;&#172;?c association virtually <br />                                                                              impossible verify biochemically. The proteins <br /> In order &#239;&#172;?nd trypanosomes contain a <br /> Ccr4/Not1/Caf1 complex, expressed TAP-tagged                              Tb08.10J17.920 (annotated PIF1 DNA <br /> versions TbCAF1 TbLCCR4 trypanosomes.                              repair recombination helicase) Tb10.6k15.1430 <br /> Expression tetracycline inducible; 3&#226;&#8364;&#8220;5 days                        (hypothetical protein conserved Kinetoplastids) <br /> tetracycline addition TbCAF1-TAP cells mild                           (Figure 1C, band marked &#226;&#8364;&#732;HC&#226;&#8364;&#8482;). There <br /> growth inhibition (no 50% increase division                     evidence association TbLCCR4 with <br />                                                                              NOT1 CAF1, expected absence the <br /> Table 2. Components yeast CCR4/CAF1/NOT complex,                 leucine-rich repeat. <br /> trypanosome, human Drosophila homologues                                    Proteins co-purifying TbCAF1-TAP gave four <br />                                                                              major bands (Figure 1C). The band (1) 220 kDa <br /> Yeast      Trypanosome Trypanosome        Humana              Drosophila     contained TbNOT1. Judging peptide coverage <br />            protein     GeneDB                                 CG number      (Table 3), major protein band 3 TbNOT5. <br />                        number <br />                                                                              Band 4 contained 61-kDa protein (encoded by <br /> Caf1/Pop2 TbCAF1          927.6.600       CNOT7/hCaf1a        5684           Tb10.6k15.1820), TPR repeat gives a <br />                                           CNOT8/hCaf1b                       poor match (P = 0.6 above) various CNOT10 <br /> Ccr4       None foundb                    CNOT6/hCCR4a        31137 <br />                                           CNOT6L/hCCR4b                      homologues. Band 5 broad contained various <br /> Not1       TbNOT1         10.70.6450      CNOT1               1884           proteins 37&#226;&#8364;&#8220;48 kDa, including TbDHH1, TbNOT2, <br /> Not2       TbNOT2         927.6.850       CNOT2               2138           TbCAF1 protein unknown function encoded by <br /> Not3       &#226;&#8364;&#8220;              &#226;&#8364;&#8220;               CNOT3S <br /> Not4       &#226;&#8364;&#8220;              &#226;&#8364;&#8220;               CNOT4 N/S/L         31716 <br />                                                                              Tb927.8.1960 (Table 3). Band 2 faint and <br /> Not5       TbNOT5         927.3.1920      CNOT3L              8426           contained aminopeptidase contami- <br /> Caf4       unclearc       Unclearc                                           nant; likely contaminants included actin a <br /> Caf16      &#226;&#8364;&#8220;              (927.6.2810)d <br /> Caf40      TbCAF40        927.4.410        CNOT9/hCAF40       14213 <br />                                                                              ubiquitin-like protein identi&#239;&#172;?ed puri&#239;&#172;?ed <br /> Caf130                    ?Tb10.6k15.1820e CNOT10             &#226;&#8364;&#8220;              RNA polymerase II (49). The TbCAF40 protein not <br /> a <br />                                                                              detected, possible TbCAF16. When <br />   Human subunit names preceded &#226;&#8364;&#732;C&#226;&#8364;&#8482; &#226;&#8364;&#732;h&#226;&#8364;&#8482; (14). Ccr4a          immunoprecipitated TbDHH1 speci&#239;&#172;?c antibody <br /> is Acc# NM_015455, Ccr4b, Acc# NM_144571, Caf1a, Acc# <br /> NM_013354, Caf1b, Acc# NM_004779.                                            (50) TbNOT1 major speci&#239;&#172;?cally <br /> b <br />   The work article indicates Tb927.4.2430 functional   associated protein (See Supplementary Figure 9). Addi- <br /> homologue Ccr4. <br /> c <br />                                                                              tional work required con&#239;&#172;?rm association of <br />  Several possible homologues, convincing.                   potential subunits. Since evidence <br /> d <br />   Uncertain. <br /> e <br />  Caf130 molecular weight 130 kDa Tb10.6k15.1820 <br />                                                                              association TbLCCR4, other <br /> protein 35 kDa.                                                      trypanosome CCR4-like protein, components the <br />  <br />  <br /> Table 3. Mass spectrometry proteins associated TAP-tagged CAF1 <br />  <br /> Band     GeneDB number                Protein function (known putative)            Score   Peptides   Coverage   Molecular weight   Migration <br />  <br /> 1        Tb10.70.6450                 NOT1                                            3389    65         33         259                 &gt;220 <br /> 2        Tb927.2.280 (and related)a   Retroposon hotspot protein (several similar      251     7          8          91                90&#226;&#8364;&#8220;100 <br />                                         predicted proteins) <br /> 2        Tb11.02.1070                 Aminopeptidase                                    57     3          3          98                   100 <br /> 3        Tb927.3.1920                 NOT3/5                                           815    19         29          67                    85 <br /> 3        Tb11.02.1600                 Ubiquitin-like                                    92     2          3          67                    85 <br /> 4        Tb10.6k15.1820               TPR repeat protein, similar CNOT10/Caf130     578    11         21          61                    65 <br /> 4        Tb11.02.1860                 Hypothetical conserved, PF02622, domain          105     2          4          52                    65 <br />                                         unknown function <br /> 5        Tb927.1.2340a                Alpha tubulin                                    975    20          5          50                    50 <br /> 5        Tb927.1.2330a                Beta tubulin                                     614    14         28          50                    50 <br /> 6        Tb927.6.600                  CAF1                                             196     7         16          39                    45 <br /> 6        Tb927.6.850                  NOT2                                             172     4          8          37                    45 <br /> 6        Tb10.70.3290                 DHH1                                              52     2          4          46                    45 <br /> 6        Tb09.211.0620                Actin A                                          135     3         11          42                    45 <br /> 6        Tb927.8.1960                 Hypothetical conserved                           714    20         43          48                    45 <br />  <br /> The overall score, number di&#239;&#172;&#8364;erent peptides identi&#239;&#172;?ed, % coverage amino acid sequence shown. <br /> a <br />   Bands 2 5 predominantly consisted proteins frequently contaminants. <br />  3380 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br /> NOT1/CAF1 complex speculated TbCAF1                                                              TbCAF1, TbNOT1 TbDHH1 required <br /> be active deadenylating enzyme complex.                                                             bloodstream trypanosome growth <br />    The intra-cellular locations TbCAF1-TAP                                                                  The roles TbCAF1, TbNOT1, TbDHH1 TbLCCR4 <br /> (Figure 1D) TbLCCR4-TAP (Figure 1E)                                                                   bloodstream trypanosomes probed tetracycline- <br /> analysed immuno&#239;&#172;&#8218;uorescence: appeared                                                             inducible RNAi. Depletion TbCAF1, TbNOT1 or <br /> predominantly&#226;&#8364;&#8221;but exclusively&#226;&#8364;&#8221;in cytoplasm.                                                            TbDHH1 RNA severely inhibited trypanosome growth <br /> Although distribution CAF1 somewhat                                                                 death days 2 3 tetracycline addition <br /> uneven, clear punctate staining indicative association                                                      (Figure 2A). Reduction RNA veri&#239;&#172;?ed northern <br /> with RNA storage degradation granules                                                               blotting (shown CAF1 Figure 2B C, shown <br /> evident; expected since, trypanosomes,                                                      NOT1 DHH1) and, DHH1, western blotting <br /> structures seen immuno&#239;&#172;&#8218;uorescence                                                         immunoprecipitation. Notably, despite lethal <br /> stress [(50,51) unpublished data]. A polyclonal                                                        e&#239;&#172;&#8364;ect, DHH1 protein depleted 50% <br /> antibody CAF1 unfortunately low titre did                                                         (data shown Supplementary Figure 9). TbLCCR4 <br /> not detect protein trypanosome lysates (data                                                        RNAi, contrast, e&#239;&#172;&#8364;ect growth, were <br /> shown).                                                                                                        unable detect mRNA wild-type cells we <br />  <br />  <br />                 A 104                                  CAF1 RNAi                                         NOT1 RNAi                                       DHH1 RNAi <br />                                                                                         104                                             1000 <br />  <br />                                       1000                                             1000 <br />                 cumulative cell no. <br />  <br />  <br />  <br />  <br />                                                                                                                                          100 <br />                                                     6.3&#226;&#8364;&#8220;6.7 h                                         5.9&#226;&#8364;&#8220;6.1 h           NOT1+                               8.6 h <br />                                       100                           CAF1+               100 <br />                                                                                                                                                                             DHH1+ <br />                                                                                                                                             10 <br />                                        10                                                10 <br />                                                                                                                                              1 <br />                                         1                               CAF1              1                   NOT1 <br />                                                                                                                                                       DHH1 <br />                                        0.1                                              0.1                                              0.1 <br />                                              0        1         2             3               0          1        2             3                0       1            2       3 <br />                                                              days                                              days                                          days <br />  <br />  <br />                 B                                  Stage PC BS PC PC                              C          Stage PC BS BS BS BS                    BS BS <br />                                                   Tet (h)     -     -     -       24                      Tet (h)     -     -       -   10 23        -       18 <br />                                                  Cell line    W     W    C        C                     Cell line     W    W        C   C        C   C       C <br />  <br />                                                    CAF1                                                      CAF1                                                   EP <br />  <br />                                                   RRNA                                                    PGKC                                                      HISH4 <br />                                                                                                           PGKB <br />                                                  GPIPLC <br />                                                                                                                                                                     SRP <br />                                                       EP                                                  HISH4 <br />                                                                                                                                                                     RRNA <br />                                                   RRNA                                                       SRP <br />  <br />  <br />                                                   ISG65 <br />  <br />                                                     TUB <br />  <br />                                                   RRNA <br />  <br /> Figure 2. CAF1, DHH1 NOT1 essential bloodstream trypanosome growth. (A) Trypanosomes transfected plasmids designed <br /> for tetracycline-inducible RNAi targeting CAF1, DHH1 NOT1 independent cell lines selected. Growth tetracycline (&#239;&#172;?lled <br /> symbols, continuous lines, CAF1+, NOT1+, DHH1+ cells) illustrated cumulative exponential curve (log scale, initial number divided <br /> by 105) division time indicated. Growth presence tetracycline (added time = 0) shown using open symbols dashed lines; <br /> the depletion CAF1, DHH1 NOT1 indicated downward arrow. The di&#239;&#172;&#8364;erent symbols represent di&#239;&#172;&#8364;erent cell lines each <br /> case. (B) E&#239;&#172;&#8364;ect CAF1 depletion levels various RNAs procyclic trypanosomes. The CAF1 mRNA reduced 10% of <br /> normal RNAi induction. Total glyoxal-treated RNA separated agarose gels, blotted hybridized probes shown left. <br /> PC: procyclic; BS: bloodstream; tet: tetracycline. Cell lines are: W-cells expressing tet repressor T7 polymerase; C-cells inducible CAF1 <br /> RNAi. The experiments performed duplicate. Quantitation using rRNA loading control revealed reproducible e&#239;&#172;&#8364;ects CAF1 <br /> depletion mRNA, apart EP increased 1.2&#226;&#8364;&#8220;3.1-fold. (C) E&#239;&#172;&#8364;ect CAF1 depletion levels various RNAs bloodstream- <br /> form trypanosomes. Details (B) formaldehyde-agarose gels used. <br />                                                                                                               Nucleic Acids Research, 2008, Vol. 36, No. 10 3381 <br />  <br />  <br />  A                                   B                                  Overall poly(A) distribution   Effect TbCAF1 depletion mRNA abundance <br />    Time (Act D)   0     15 60                                   20 <br />                                                                                       No Act D         Figure 2B C e&#239;&#172;&#8364;ects TbCAF1 depletion on <br />            CAF1 +      +   + <br />                                                                                          CAF1+ <br />                                                                                                        abundances various trypanosome mRNAs. First, <br />           200                                                   15                                     looked abundances mRNAs do <br />                                                                                          CAF1 <br />                                                                                                        developmental regulation, encoding Histone H4 <br />            nt                                                   10                                     (HISH4), actin (ACT) Tubulin (TUB) (35,36,52,53). <br />   block                                                                                                The abundances of mRNAs una&#239;&#172;&#8364;ected by <br />    10 <br />  <br />  <br />                                      Radioactivity % total <br />           100                                                    5                                     TbCAF1 depletion (Figure 2B C; ACT later in <br />    9 <br />                                                                                                        Figure 4; note minor variations visible the <br />    8 <br />            80                                                    0                                     blots shown reproducible). In addition, we <br />    7                                                                  1 2 3 4 5 6 7 8 9 10 11 <br />                                                                 20                                     looked mRNAs developmentally regulated: <br />            70                                                                       60 min Act D <br />    6                                                                                                   Cytosolic phosphoglycerate kinase (PGKB) expressed <br />            60 <br />                                                                 15 <br />                                                                                            CAF1+       predominantly procyclic forms (PC) (Figure 2C), while <br />    5                                                                                       CAF1        mRNAs encoding glycosomal phosphoglycerate <br />            50 <br />    4 <br />                                                                                                        kinase (PGKC) (Figure 2C), glycosyl phosphatidylinositol <br />                                                                 10 <br />                                                                                                        phospholipase C (GPIPLC) (Figure 2B), mem- <br />            40 <br />                                                                                                        brane protein ISG65 (Figure 2B) exclusively in <br />    3                                                             5                                     bloodstream forms (BS) (54&#226;&#8364;&#8220;57). By northern blotting, we <br />            30                                                                                          did detect defect developmental regulation of <br />    2 <br />                                                                  0                                     transcripts TbCAF1-depleted cells (Figure 2B and <br />                                                                       1 2 3 4 5 6 7 8 9 10 11 <br />                                                                                block                   C). Results quantitative PCR analysis revealed <br />            20                                                                                          change PGKB mRNA levels (Stewart,M., ZMBH, <br />                                      C                                Radioactivity CAF1 / CAF1+ <br />                                                                                                        data shown). The transcript increased <br />                                                                 2.5 <br />    1                                                                     No Act D                      slightly abundance TbCAF1 depletion that <br />                                                                  2       60 min Act D                  encoding EP procyclin mRNA bloodstream forms <br />            10                                                                                          (Figure 2C). This mRNA exceptional is <br />                                                                 1.5                                    produced RNA polymerase I, transcription <br />                                                                                                        degradation subject developmental regulation; <br />                                                                  1 <br />                                                                                                        possible growth arrest mimic <br />                                                                 0.5                                    &#239;&#172;?rst stages di&#239;&#172;&#8364;erentiation, a&#239;&#172;&#8364;ect <br />                                                                                                        regulation EP expression, mRNAs be <br />                                                                  0                                     discussed here. <br />                                                                        1 2 3 4 5 6 7 8 9 10 11 <br />                  1 2 3 4 5 6                                                  10nt block <br />                                                                                                        TbCAF1is involved deadenylation many <br /> Figure 3. CAF1 depletion inhibits deadenylation bloodstream                                         trypanosome RNAs <br /> trypanosomes. Total RNA puri&#239;&#172;?ed cells inducible <br /> CAF1 RNAi, cultured tetracycline (CAF1+, lanes 1, 3 5)                                     To &#239;&#172;?nd TbCAF1 important poly(A) <br /> or tetracycline (24 h) deplete CAF1 (downward arrow, lanes 2, 4                                shortening trypanosomes, measured overall <br /> and 6). Actinomycin D added times indicated. RNA                                  poly(A) tail distribution total RNA. Brie&#239;&#172;&#8218;y, total RNA <br /> 30 -labelled digested RNases A T1. (A) Phosphorimager <br /> image showing radiolabelled poly(A) tails separated denaturing gel <br />                                                                                                        extracted, labelled 30 end, digested with <br /> electrophoresis. To left positions markers. Further                              RNases A T1, cleave downstream <br /> left indicated blocks 10 residues. Block 1 group                              G, C U. The remaining labelled poly(A) was <br /> 11&#226;&#8364;&#8220;20 As; block 2 21&#226;&#8364;&#8220;30 As, on. (B) Quantitation results                                separated polyacrylamide gel visualized by <br /> from gel shown (A) (no Actinomycin 60-min time point). For                                  phosphorimaging. Results shown Figure 3 [note <br /> each lane, radioactivity block plotted relative <br /> to total radioactivity lane. Black bars: CAF1+; open bars:                                  preparations total cellular RNA, the <br /> CAF1-depleted. (C) Results independent experiments, shown                                  poly(A) tails seen include short tails present on <br /> arithmetic mean standard deviation. We &#239;&#172;?rst generated data like                                    nuclear RNAs destined degradation (58)]. In <br /> that shown (B) experiments. We divided number                                normal cells, poly(A) tails ranged 10 100 nt in <br /> for CAF1-depleted cells number obtained CAF1+ <br /> cells. For block 1, numbers automatically 1. For blocks 2&#226;&#8364;&#8220;11, <br />                                                                                                        length (Figure 3A, lane 2). To quantitate poly(A) tail <br /> numbers &gt;1 indicate CAF1-depleted cells radio-                                      lengths, compared radioactivity tails speci&#239;&#172;?c <br /> activity poly(A) tails normal cells. Here clear                          lengths total radioactive signal, taking blocks of <br /> the di&#239;&#172;&#8364;erences stronger poly(A) tails 70 nt, especially                              10 nt (Figure 3B). Cells depleted CAF1 showed very <br /> after Actinomycin D treatment.                                                                         slight, reproducible decrease proportion of <br />                                                                                                        tails 40 nt corresponding shift longer <br /> could sure RNAi worked (data                                                   tail lengths (Figure 3C). One hour addition of <br /> shown). Similar experiments procyclic trypanosomes                                                  Actinomycin D, cells normal CAF1 level, <br /> gave results: growth TbCAF1 TbDHH1                                                      lower poly(A) tail signal overall signal <br /> RNAi cells ceased 1 day tetracycline addition,                                               concentrated shorter lengths (Figure 3A, lane 6, <br /> numbers started decrease 2 days, growth                                               Figure 3B). In cells CAF1 RNAi (Figure 3A, <br /> of TbLCCR4 RNAi cells una&#239;&#172;&#8364;ected 6 days                                                     lane 3), poly(A) tails clearly longer the <br /> incubation tetracycline (data shown).                                                         normal cells. This di&#239;&#172;&#8364;erence evident 15 min after <br />  3382 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br />    A                       Time (Act D)      0        0 0 0 0 30 30                       30     60    60   60   0                0 0    Actinomycin D treatment pronounced <br />                                      Tet     &#226;&#710;&#8217;        + &#226;&#710;&#8217; + + - +                         +       &#226;&#710;&#8217;    +    +    &#226;&#710;&#8217;                &#226;&#710;&#8217; +    60 min (Figure 3B). As control did similar <br />                                Cell line     W        X C C D C C                         D      C     C    D    W                C C <br />                               RNase H        +        &#226;&#710;&#8217; &#226;&#710;&#8217; &#226;&#710;&#8217; &#226;&#710;&#8217; &#226;&#710;&#8217; &#226;&#710;&#8217;                          &#226;&#710;&#8217;      &#226;&#710;&#8217;     &#226;&#710;&#8217;    &#226;&#710;&#8217;   +                + + <br />                                                                                                                                          experiments DHH1 depletion; increase poly(A) <br />                                                                                                                        0.70 <br />                                                                                                                                          tail lengths seen (data shown). <br />                                      HISH4                                                                             0.65 <br />                                                                                                                                             To quantitate e&#239;&#172;&#8364;ects TbCAF1 depletion, <br />                                                                                                                        0.55              compared poly(A) tail lengths di&#239;&#172;&#8364;erent <br />                                       SRP <br />                                                                                                                        kb 13 14          experiments. First, radioactivity the <br />                                                  1       2    3     4     5    6     7       8       9 10 11 12                          di&#239;&#172;&#8364;erent 10-nt length ranges divided total <br />                                                                                                                                          amount. Next, value CAF1 RNAi cells divided <br />                                                                                                                                          value normal cells. This con&#239;&#172;?rmed that <br />    B             Time (Act D)                0       0       0      0     30 30 30 60 60 60 90 90 90                                0 <br />                                       Tet    &#226;&#710;&#8217;       &#226;&#710;&#8217;       &#226;&#710;&#8217;      +      &#226;&#710;&#8217; &#226;&#710;&#8217; + &#226;&#710;&#8217; &#226;&#710;&#8217; + &#226;&#710;&#8217; &#226;&#710;&#8217; +                                        &#226;&#710;&#8217; <br />                                                                                                                                          60 min addition Actinomycin D, CAF1 RNAi <br />                              Cell line       W W             C      C     W     C        C       W     C    C     W    C C          W    cells 70&#226;&#8364;&#8220;110-nt poly(A) tails normal cells <br />                             RNase H          +       &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217;     &#226;&#710;&#8217;     &#226;&#710;&#8217;        &#226;&#710;&#8217;       &#226;&#710;&#8217;     &#226;&#710;&#8217;    &#226;&#710;&#8217;     &#226;&#710;&#8217;    &#226;&#710;&#8217;      &#226;&#710;&#8217;     +    (Figure 3C, blocks 7&#226;&#8364;&#8220;10, dark bars). These results suggest <br />                                                                                                                                          CAF1 involved deadenylation many <br />                                  PGKC <br />                                                                                                                                          trypanosome mRNAs. <br />                                  PGKB <br />  <br />                                      ACT                                                                                                 TbCAF1 involved constitutive degradation <br />                                                                                                                                          trypanosome mRNAs <br />                                                                                                                                          To assess e&#239;&#172;&#8364;ects CAF1 depletion decay of <br />                                  HISH4                                                                                                   individual mRNAs bloodstream-form trypanosomes, <br />                                                                                                                                          incubated Actinomycin D. Figure 4A shows <br />                                      TUB                                                                                                 results HISH4. Trypanosome transcription poly- <br />                                                                                                                                          cistronic. During processing, 39-nt spliced leader <br />                                      SRP                                                                                                 RNA added 50 end mRNA, position <br />                                              1       2       3      4     5     6        7       8     9    10 11 12 13 14 <br />                                                                                                                                          usually determined polypyrimidine tract. <br />                                                                                                                                          Polyadenylation &#226;&#8364;&#732;upstream&#226;&#8364;&#8482; mRNA follows, a <br />    C                       120                                                      120 <br />                                                                                                                                          reaction coupled splicing downstream <br />                                                                         ACT                                                HISH4         transcript. Polyadenylation occurs speci&#239;&#172;?c single <br />                            100                                                      100 <br />                                                                                                                                          site, various positions 70 140-nt <br />      % RNA remaining <br />  <br />  <br />  <br />  <br />                             80                                                      80                                                   upstream polypyrimidine tract (59,60). From the <br />                             60                                                      60                                                   genome sequence, 8 HISH4 open reading frames <br />                             40                                                      40 <br />                                                                                                                                          organized tandem repeat; &#239;&#172;?rst seven genes <br />                                                                                                       Wild-type                          maximum predicted length mRNA [without the <br />                             20                                                      20                CAF1+ <br />                                                                                                       CAF1                               39-nt spliced leader poly(A) tail] 600 nt. <br />                              0                                                       0                                                   For gene cluster, prediction about <br />                                  0     20        40          60      80       100         0          20     40    60       80      100 <br />                                                               time (min) Actinomycin D                                             500 nt. Deadenylated HISH4 mRNA migrated 6 bands <br />                            120                                                      120 <br />                                                                                                                                          0.55 0.65 kb (Figure 4A, lanes 1 12) and <br />                                                                     PGKC                                                      TUB        polyadenylated mRNA migrated 0.775 and <br />                            100                                                      100 <br />                                                                                                                                          0.575 kb, suggesting maximum poly(A) tail length of <br />          % RNA remaining <br />  <br />  <br />  <br />  <br />                             80                                                       80                                                  100 nt (Figure 4A, lane 3). The choice poly(A) <br />                             60                                                       60                                                  site una&#239;&#172;&#8364;ected CAF1 depletion (Figure 4A, <br />                             40                                                       40 <br />                                                                                                                                          compare lanes 13 14). As previously demonstrated <br />                                                                                                                                          (36), depletion 50 -&gt;30 exonuclease XRNA resulted <br />                             20                                                       20 <br />                                                                                                                                          increase shorter HISH4 mRNA species <br />                              0 <br />                                  0     20        40          60      80       100 <br />                                                                                       0 <br />                                                                                              0       20     40    60       80      100 <br />                                                                                                                                          (Figure 4A, lane 2). Depletion DHH1 e&#239;&#172;&#8364;ect <br />                                                                   time (min) Actinomycin D <br />                                                                                                                                          (Figure 4A, lane 5), CAF1 RNAi resulted clear <br />                                                                                                                                          increase average mRNA size, expected an <br /> Figure 4. CAF1 depletion inhibits degradation ACT, PGKC, TUB                                                                          inhibition deadenylation (Figure 4A, lane 4). Thirty <br /> and HISH4 mRNAs trypanosomes. (A) Bloodstream trypanosomes                                                                            minutes addition Actinomycin D, shortening of <br /> expressing repressor (cell line W) lines inducible RNAi <br /> targeting XRNA (cell line X, lane 2), CAF1 (cell line C) DHH1 (cell                                                                   HISH4 mRNA clearly visible (Figure 4A, lane 6); <br /> line D) grown (&#226;&#8364;&#8220;) (+) tetracycline 24 h                                                                      una&#239;&#172;&#8364;ected depletion DHH1 (lane 8) but <br /> shown. Actinomycin D added times indicated (times                                                                       inhibited CAF1 RNAi (lane 7). This di&#239;&#172;&#8364;erence per- <br /> include centrifugation time). RNA isolated separated                                                                    sisted 60 min Actinomycin D. Notably, deadenyla- <br /> denaturing polyacrylamide gel. For lanes 1, 12, 13 14 mRNA <br /> was treated oligo d(T) RNase H remove poly(A) tails. <br />                                                                                                                                          tion complete (compare lane 11 lane 12); <br /> The histone H4 (HISH4) mRNA signal recognition particle                                                                          results suggested ceased 25&#226;&#8364;&#8220;50 nt of <br /> RNA control (SRP) detected northern blotting. (B) As in <br /> (A), separation agarose gel. RNA detected using various                                                                    loading control, results plotted mean   standard deviation (four <br /> probes, indicated: PGKC: glycosomal phosphoglycerate kinase;                                                                          experiments). Black circles: wild-type cells (two experiments only, <br /> ACT: actin; TUB: alpha tubulin; HISH4 (usually hybridized                                                                        arithmetic mean shown); Black squares: inducible RNAi line <br /> order, intermediate stripping required). (C) Results                                                                   tetracycline (CAF1+); Open squares: inducible RNAi line with <br /> independent experiments [as (B)] quantitated, using SRP                                                                   tetracycline (CAF1 down, downward arrow). <br />                                                                                                                              Nucleic Acids Research, 2008, Vol. 36, No. 10 3383 <br />  <br />  <br /> A        Time (Act D) 0   4       B                                                                                   degradation constitutively expressed trypanosome <br />                CCR4 + + + +                                   25 <br />                                                                                        No Act D      **               mRNAs depends CAF1. The results experiments <br />                CAF1 + + + + <br />                                                               20 <br />                                                                                                                       using reporter mRNA produced RNA polymerase I <br /> block     nt <br />                                                                                *                                      indicated e&#239;&#172;&#8364;ects CAF1 depletion mRNA <br />                                                                                        ** <br />                                                               15 <br />                                                                                                 ** <br />                                                                                                                       degradation linked transcription RNA <br />          300 <br />                                                               10 <br />                                                                                                                       polymerase II (Supplementary Figure 10). <br />  8                                                                     * <br />  <br />  <br />  <br />  <br />                                    Radioactivity % total <br />          200 <br />      7                                                         5                                                  *   Depletion CAF1 inhibits deadenylation human cells <br />  6                                                             0                                                      The clear importance TbCAF1 trypanosomes, <br />                                                                    1       2       3        4   5    6        7   8 <br />          100                                                                                                          combined observations literature, suggested <br />      5    90                                                  25                                         **           Caf1 important mammalian cells. <br />           80                                                                            4h Act D <br />                                                                                                                       We repeated poly(A) tail length measure- <br />  4        70                                                  20 <br />                                                                                *       **                             ments cytoplasmic RNA HTGM5 cells, are <br />           60 <br />                                                               15                                **                    derived human HT1080 &#239;&#172;?brosarcoma cell line <br />      3    50                                                                                                          (45). A typical autoradiogram shown Figure 5A and <br />                                                               10       *                                              quantitations shown Figure 5B C. The overall <br />           40 <br />                                                                5                                                      distributions poly(A) tail lengths compared, <br />  2        30 <br />                                                                                                                       Figure 3 shows total trypanosome RNA and <br />                                                                0 <br />                                                                    1       2       3        4   5    6        7   8   Figure 5 cytoplasmic mammalian RNA. Nevertheless, <br />                                                                                          control                      sample HTGM5 cells substantially <br />           20                                                                             Ccr4a/b                      poly(A) tails 100 nt trypanosomes <br />                                                                                          Caf1a/b                      (compare Figure 5A, lane 1 Figure 3A, lane 1). For <br />      1 <br />                                                                                        * p&lt;.05 <br />                                                                                         vs control                    human RNA, 29% poly(A) tails shorter than <br />           10                                                                           ** p&lt;.01                       40 nt, 5.6% 10&#226;&#8364;&#8220;20-nt range (Figure 5B). <br />                                                                                         vs control <br />                                                                                                                       In recent analysis human poly(A) tail lengths, <br />                                                                                                                       solution hybridization, 25% expressed <br />                                                                                                                       genes appeared &lt;30 nt poly(A) tail a <br />                                                                                                                       quarter transcripts (62). <br />                    1 2 3 4 5 6 <br />                                                                                                                          To deplete human Ccr4 Caf1, targeted two <br />                                                                                                                       paralogues previously identi&#239;&#172;?ed the <br /> Figure 5. CAF1 depletion inhibits deadenylation human cells.                                                       human genome: Ccr4a, Ccr4b, Caf1a Caf1b (Table 2) <br /> HTGM5 cells transfected siRNAs deplete CAF1                                                           (30,63). The mRNA depletion slightly e&#239;&#172;&#402;cient <br /> CCR4 (downward arrow) control siRNAs (+). The e&#239;&#172;&#8364;ect of <br /> the siRNAs targets illustrated Figure 6. Samples were <br />                                                                                                                       Ccr4 (Figure 6A) Caf1 (Figure 6B). Although <br /> taken with, 4 h Actinomycin D treatment. Puri&#239;&#172;?ed                                                    depletion Ccr4a Ccr4b resulted slight <br /> RNA 30 -labelled digested RNases A T1. (A)                                                          increase average poly(A) lengths di&#239;&#172;&#8364;erences were <br /> Phosphorimager image showing radiolabelled poly(A) tails separated                                                 generally statistically signi&#239;&#172;?cant (Figure 5A, lane 2, <br /> denaturing gel electrophoresis. To left positions markers.                                             Figure 5B). Caf1a Caf1b depletion (Figure 5A, <br /> Further left indicated blocks 20&#226;&#8364;&#8220;50 A residues, as <br /> in Figure 5. (B) Quantitation results gel shown (A).                                                   lane 3), contrast, caused shift longer lengths <br /> For individual lane, radioactivity block                                                      particularly marked 100&#226;&#8364;&#8220;150-nt range <br /> was calculated relative total radioactivity lane. The                                                   (Figure 5B, block 6). After 4-h-Actinomycin D treat- <br /> results expressed arithmetic mean   standard deviation. Black                                                  ment, deadenylation occurred control (lane 4) <br /> bars: control siRNA, &#239;&#172;?ve independent experiments; grey bars: <br /> Ccr4a+b-depleted, siRNAs C4+C8, experiments; open bars: <br />                                                                                                                       Ccr4-depleted cells (lane 5), inhibited Caf1 <br /> Caf1a+b-depleted, siRNAs C5+C6, experiments. Upper panel:                                                        RNAi (Figure 5A, lane 6); Caf1-depleted cells, the <br /> steady state, values lanes 1, 2 3 (in order); lower panel:                                               proportion short tails control <br /> after 4 h Actinomycin D, values lanes 4, 5 6. P-values                                              (Figure 5B, blocks 1&#226;&#8364;&#8220;3) 100&#226;&#8364;&#8220;150-nt range <br /> 2-tailed students t-test (unequal variance) probability                                             markedly abundant (Figure 5B, block 6). <br /> the poly(A) tail signals Ccr4- Caf1-depleted cells same <br /> as control.                                                                                              These results con&#239;&#172;?rmed Ccr4 con- <br />                                                                                                                       tribute, previously suggested (7), depletion Caf1 has <br /> poly(A) remained. A poly(A) 27 nt probably su&#239;&#172;&#402;cient                                                             stronger e&#239;&#172;&#8364;ect deadenylation bulk mRNA in <br /> to bind single molecule poly(A) binding protein (61);                                                            human HTGM5 cells. Since Caf1 depletion presumably <br /> further deadenylation presumably leads linear RNA                                                                  results dissociation Ccr4 complex, the <br /> which lacks poly(A) binding protein susceptible                                                             e&#239;&#172;&#8364;ects re&#239;&#172;&#8218;ect failure recruit Ccr4 <br /> rapid degradation.                                                                                                    Caf1 susceptible mRNAs. <br />   Results mRNAs shown Figure 4B, with <br /> quantitation Figure 4C. In normal cells, degradation                                                            Depletion Caf1 inhibits degradation unstable <br /> TUB PGKC mRNAs started delay 30 min;                                                                   mammalian mRNA <br /> decay HISH4 ACT mRNAs faster. Depletion                                                                    In mammalian cells, rapid degradation mRNAs <br /> of CAF1 strongly inhibited degradation                                                                     containing AU-rich element (ARE) initiated by <br /> four RNAs inhibition poly(A) tail shortening                                                                   poly(A) tail shortening (2). We analysed the <br /> was evident. From results, concluded                                                                    e&#239;&#172;&#8364;ects Caf1 depletion human HTGM5 cells that <br />  3384 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br />                        A 200                                                                                                                             B 160 <br />                                                       180                                                                             Ccr4a                                                                                                         Caf1a <br />  <br />  <br />                        relative Ccr4 mRNA level (%) <br />                                                                                                                                                                                         140 <br />  <br />  <br />  <br />  <br />                                                                                                                                                          relative Caf1 mRNA level (%) <br />                                                       160                                                                             Ccr4b <br />                                                                                                                                                                                         120                                                         Caf1b <br />                                                       140 <br />                                                       120                                                                                                                               100 <br />  <br />                                                       100                                                                                                                                80 <br />                                                           80                                                                                                                             60 <br />                                                           60 <br />                                                                                                                                                                                          40 <br />                                                           40 <br />                                                           20                                                                                                                             20 <br />  <br />                                         0                                                                                                                        0 <br />                                                                         control                Ccr4a               Ccr4b Ccr4a+b                                                              control             Caf1a              Caf1b Caf1a+b <br />                                    siRNAs                                (D0)                   (C4)                (C8) (C4+C8)                            siRNAs <br />                                                                                                                                                                                                (D0)                (C5)               (C6)  (C7) <br />  <br />  <br />                                                       C                                                                       &#226;&#8364;&#8220;                                                                                      1 h Act.D <br />                                                                                                          CCR4a+b <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                         CCR4a+b <br />                                                                                                                                                CAF1a+b <br />  <br />                                                                                                                                                                   CAF1a+b <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                CAF1a+b <br />  <br />                                                                                                                                                                                                                                                          CAF1a+b <br />                                                                                                                               CAF1a <br />  <br />                                                                                                                                        CAF1b <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                              CAF1a <br />  <br />                                                                                                                                                                                                                                        CAF1b <br />                                                                                                control <br />  <br />  <br />  <br />                                                                                                                     control <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                               control <br />  <br />  <br />  <br />                                                                                                                                                                                                                    control <br />                                                                siRNA <br />                                                                target <br />  <br />  <br />                                                           globin pA+ <br />                                                            -AREpA&#226;&#8364;&#8220; <br />  <br />  <br />                                                                nucleolin <br />  <br />                                                                                          siRNA D0 C4/8 D0                     C5       C6 C5/6 C7                                             D0 C4/8 D0                     C5 C6 C5/6 C7 <br />                                                                                            lane 1  2    3                      4        5  6    7                                              8  9 10                       11 12 13 14 <br />  <br />  <br />                                                       D                                  350 <br />  <br />                                                                                          300 <br />                                                                globin-ARE pA+ mRNA (%) <br />  <br />  <br />  <br />  <br />                                                                                          250 <br />  <br />                                                                                          200 <br />  <br />                                                                                          150 <br />  <br />                                                                                          100 <br />  <br />                                                                                           50 <br />  <br />                                                                                            0 <br />                                                                                                  1       2          3         4        5       6                7                              8        9         10         11       12       13        14 <br />  <br /> Figure 6. Depletion Caf1a Caf1b inhibits deadenylation ARE-containing b-globin reporter mRNA human cells. (A) HTGM5 cells were <br /> transfected period 5 days siRNAs targeting Ccr4a Ccr4b; (B) Caf1a Caf1b. The knockdown e&#239;&#172;&#402;ciencies measured the <br /> mRNA expression level quantitative PCR. The graphs mean standard deviation assays single experiment; siRNA <br /> transfection measurements repeated (for Ccr4b) times (all siRNAs) similar results. (C) Cytoplasmic RNA <br /> was prepared siRNA-transfected cells prior Actinomycin D treatment. Northern blots hybridized globin probe (upper <br /> panel), nucleolin probe loading control (lower panel). (D) Quantitation poly(A)+ portion globin-ARE mRNA shown (C); mean <br /> and standard deviation (lanes 1, 2, 8 9) (lanes 3&#226;&#8364;&#8220;7, 10&#226;&#8364;&#8220;14) independent experiments. <br />  <br />  <br /> stably express b-globin reporter gene ARE                                                                                                         We tested e&#239;&#172;&#8364;ect knocking Caf1a and <br /> granulocyte-macrophage colony-stimulating factor                                                                                                      Caf1b, combination. This either <br /> 30 UTR (45). Knocking Ccr4a Ccr4b (Figure 6C,                                                                                                       using di&#239;&#172;&#8364;erent, gene-speci&#239;&#172;?c siRNAs, using a <br /> lanes 1and 2) did a&#239;&#172;&#8364;ect expression the globin-                                                                                                    single siRNA (C7) targets conserved region both <br /> ARE reporter mRNA. The poly(A)+ globin-                                                                                                            mRNAs. The total amounts siRNA used <br /> ARE mRNA quanti&#239;&#172;?ed (Figure 6D) result                                                                                                            throughout; knockdown e&#239;&#172;&#8364;ective Caf1a <br /> suggested Ccr4a Ccr4b limiting                                                                                                          (Figure 6B). At steady state, knocking Caf1a no <br /> factors deadenylation ARE-containing                                                                                                           e&#239;&#172;&#8364;ect (Figure 6C D, lane 4). Targeting Caf1b (lane 5), <br /> reporter mRNA.                                                                                                                                               Caf1s (lanes 6 7), gave small increase <br />                                                                                                                  Nucleic Acids Research, 2008, Vol. 36, No. 10 3385 <br />  <br />  <br /> A                                                                                                         TbDHH1 possibly subunits, encoded by <br />     siRNA target      CCR4a+b CAF1a+b CAF1a+b                                                        Tb10.6k15.1820 (a possible equivalent Caf130) and <br />          siRNAs      D0     C4+C8   C5+C6    C7 <br />   Time (h, Act D) 0 1 2 3 0 1 2 3 0 1 2 3 0 1 2 3 <br />                                                                                                           Tb927.8.1960. We know possible <br />                            + <br />                                                                                                           Caf16 Caf40 homologues associated the <br />                    globin A                                                                         A+ <br />                           A&#226;&#710;&#8217;                                                                        A&#226;&#710;&#8217;    complex, so, sub-stoichiometric. <br />                                                                                                              In yeast, mammalian cells Drosophila, Ccr4 <br />                      nucleolin                                                                            subunit integral complex. The only <br />                                      1   2   3   4   5   6   7    8    9 10 11 12 13 14 15 16             possible homologue Ccr4 trypanosomes (and other <br /> B                                                                                                         Kinetoplastids) lacked leucine-rich domain is <br />                    120                                           120 <br />                              Total                                         A+                             required interaction Caf1 deadenylase <br />                                                  Control <br />                    100                           CCR4            100                                      activity, did co-purify complex. There- <br />  % RNA remaining <br />  <br />  <br />  <br />  <br />                                                  CAF1                                                     fore, evidence suggests trypanosomes other <br />                    80                                             80 <br />                                                                                                           Kinetoplastids lack equivalent Ccr4. This absence <br />                    60                                             60                                      indicate ancestral complex lacked Ccr4 <br />                                                                                                           subunit. To investigate possibility looked Ccr4 <br />                    40                                             40 <br />                                                                                                           homologues, de&#239;&#172;?ned predicted proteins including the <br />                    20                                             20                                      leucine-rich repeat addition exonuclease domain, <br />                                                                                                           major eukaryotic groups (32). Yeasts and <br />                     0                                              0 <br />                          0     0.5 1 1.5 2 2.5 3 3.5                   0     0.5 1 1.5 2 2.5 3 3.5        animals, contain Ccr4, belong group, the <br />                              time (h) Actinomycin D                  time (h) Actinomycin D   Opisthokonta; Plantae (higher plants, algae) have <br />                                                                                                           Ccr4. We &#239;&#172;?nd complete genomes the <br /> Figure 7. Depletion Caf1a Caf1b inhibits degradation ARE- <br /> containing b-globin reporter mRNA. Transfection siRNAs                                           Rhizaria. Trypanosomes currently usually grouped in <br /> RNA preparation Figure 5, samples taken 1, 2                                  Excavata, Giardia intestinalis Trichomonas <br /> and 3 h addition Actinomycin D. The experiment twice                                    vaginalis, molecular evidence group lacking <br /> with similar results. (A) Northern blot &#239;&#172;?rst experiment. To                                       (32). Giardia Ccr4 Trichomonas a <br /> quantitate globin mRNA, lower portion globin signal lane <br /> 1 designated poly(A)&#226;&#8364;&#8220;; upper portion, area                                  homologue. We likely Ccr4 Tetrahymena or <br /> as lower one, designated poly(A)+ indicated. Identical areas                                   Plasmodium (Chromalveolata) Dictyostelium and <br /> were used lanes, aligning the                                     Entamoeba (Amoebozoa). From limited dataset, <br /> quantitation box just beneath signal case. (B) Quantitation                               therefore, unlikely Ccr4 component <br /> both experiments. Results control Ccr4a+b knockdown are <br /> shown separately, indicated legend graph; lines are <br />                                                                                                           ancestral Caf1/Not complex. <br /> drawn arithmetic mean. Results siRNAs C5+6 and <br /> C7 (CAF1 RNAi) indistinguishable. These results                                       The roles CAF1 deadenylation <br /> pooled, generate mean standard deviation total four <br /> measurements (2  C5+6 plus 2  C7).                                                                        Deadenylation eukaryotes studied <br />                                                                                                           focusing transcripts relatively unstable, <br /> in poly(A)+ globin-ARE mRNA. The e&#239;&#172;&#8364;ects                                                         contain destabilizing sequences in <br /> pronounced treatment Actinomycin D 1                                                       30 -UTR, open reading frame contains <br /> hour. Knocking Caf1b gave 1.4-fold increase                                                  premature termination codon. In experiments with <br /> in poly(A)+ globin-ARE mRNA (lane 12); despite                                                        mammalian cells, stable transcripts half-lives of <br /> relatively low siRNA e&#239;&#172;&#402;ciency, knocking                                                      hours; degradation far been <br /> both resulted 2- 2.5-fold increase (lanes 13                                             analysed individual basis, using inducible promot- <br /> and 14). In experiment, 1-h Actinomycin D                                                  ers. A transgene mRNA beta-globin 30 -UTR, for <br /> treatment mRNA level decreased control                                                    example, decreased 10% &#239;&#172;?rst 10 h after <br /> cells cells Caf1a+b RNAi.                                                                transcriptional shut-o&#239;&#172;&#8364; (64). In contrast, previous <br />    The results far suggested Caf1a Caf1b                                                  report CAF1 depletion results increase the <br /> redundant, play role degradation globin-                                                  average poly(A) tail length Drosophila cells (15) implies <br /> ARE mRNA. We analysed e&#239;&#172;&#8364;ects Caf1a+b                                                          Drosophila CAF1 plays major role bulk mRNA <br /> RNAi degradation detail. Results shown                                                  deadenylation. <br /> Figure 7. The globin-ARE mRNA decayed half-life                                                       To determine overall contribution Caf1 Ccr4 <br /> of &lt;1 h, Ccr4a+b siRNA treatment little                                                      homologues deadenylation analysed overall <br /> e&#239;&#172;&#8364;ect degradation; contrast, Caf1a+b siRNA                                                          distribution poly(A) tail lengths. In trypanosomes <br /> treatment delayed deadenylation allowed persistence                                                   human cells, reduction CAF1 (Caf1a+b) activity <br /> of 50% mRNA 2&#226;&#8364;&#8220;3 h Actinomycin D                                                              increased average lengths poly(A) tails inhibited <br /> treatment (Figure 6B). This con&#239;&#172;?rmed depletion                                                    deadenylation (Figures 3 5). In human cells, Ccr4 <br /> Caf1a+b inhibited globin-ARE mRNA degradation.                                                            depletion had, best, marginal e&#239;&#172;&#8364;ect poly(A) tail <br />                                                                                                           lengths. The overall poly(A) tail distributions in <br /> DISCUSSION                                                                                                &#239;&#172;?gures compared, experiments <br />                                                                                                           total trypanosome RNA, with <br /> The composition trypanosome CAF1/NOT complex <br />                                                                                                           cytoplasmic human RNA. Nevertheless, notable <br /> Our results indicate trypanosomes complex                                                     longest poly(A) tails shorter in <br /> which contains TbCAF1, TbNOT1, TbNOT2, TbNOT5,                                                            trypanosomes mammals. Perhaps an <br />  3386 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br /> intrinsic property trypanosome polyadenylation         Not1. Our experiments allow tell <br /> complex, favour explanation:           complex Ccr4 activity, Caf1 <br /> steady state, large proportion trypanosome mRNAs         activity, yeast, fully <br /> is undergoing degradation. Even &#226;&#8364;&#732;stable&#226;&#8364;&#8482; mRNAs             functional mammalian deadenylation. Nevertheless, <br /> trypanosomes shorter half-lives            clear e&#239;&#172;&#8364;ects depletion Caf1 homologues <br /> generally seen mammalian cells&#226;&#8364;&#8221;compare example         indicate Caf1a Caf1b redundant but <br /> trypanosome PGKC TUB mammalian cyclophi-             important roles ARE-mediated mRNA decay. <br /> lin RNA&#226;&#8364;&#8221;and polycistronic transcription creates numer-           In vitro decay studies previously shown that <br /> ous unwanted mRNAs rapidly destroyed.               deadenylase, PARN, promote dead- <br />    We examined e&#239;&#172;&#8364;ects Caf1 depletion           enylation ARE-containing RNA (69). Future studies <br /> deadenylation decay individual mRNAs. When          required reveal true vivo, <br /> depleted CAF1 trypanosomes, mRNAs                 PARN cooperate Caf1a/b. The <br /> studied&#226;&#8364;&#8221;PGKC, HISH4, TUB ACT&#226;&#8364;&#8221;were dead-                   analysis nonsense-mediated mRNA decay mouse <br /> enylated slowly stabilized (Figure 4). They     NIH3T3 &#239;&#172;?broblasts suggested reporter mRNA <br /> did not, paradoxically, increase abundance         containing premature termination codon (PTC) is <br /> relative rRNA, stable          initially deadenylated Pan2 length 110 <br /> una&#239;&#172;&#8364;ected RNAi. The simplest explanation       adenosine residues, followed second phase during <br /> inhibition bulk mRNA decay caused feedback inhibi-         Ccr4a/b digests remaining poly(A) tail (7). Since <br /> tion transcription RNA polymerase II. This        contribution Caf1a/b addressed this <br /> result speci&#239;&#172;?c regulatory mechanism; alternatively,    study, clear Caf1a/b participates in <br /> ribonucleotides produced mRNA degradation            deadenylation PTC-containing mRNA. <br /> be signi&#239;&#172;?cant source substrate mRNA synthesis.          The fact RNAi targeting CAF1 T. brucei was <br /> In eukaryotes also, inhibition mRNA degrada-         lethal, clear e&#239;&#172;&#8364;ects deadenylation, shows in <br /> tion does a&#239;&#172;&#8364;ect mRNA abundance (45,65&#226;&#8364;&#8220;68);         trypanosomes deadenylases able provide <br /> for example, yeast lacking Ccr4p, general      essential CAF1 functions. Since experiments gave <br /> increase abundances unstable mRNAs,             similar results organisms entirely di&#239;&#172;&#8364;erent <br /> although mRNAs encoding ribosomal proteins                branches eukaryotic evolution, suggest CAF1 <br /> markedly stabilized steady-state levels little      major deadenylase eukaryotes. <br /> a&#239;&#172;&#8364;ected (67). Thus, sort feedback regulation may <br /> be widespread, mechanism(s) unknown. <br />    Depletion CAF1 trypanosomes e&#239;&#172;&#8364;ect        SUPPLEMENTARY DATA <br /> on abundances highly unstable mRNAs&#226;&#8364;&#8221;              Supplementary Data available NAR Online. <br /> PGKC, GPIPLC ISG65 procyclic forms PGKB <br /> in bloodstream forms. We previously shown that <br /> depletion XRNA does, contrast, cause increase     ACKNOWLEDGEMENTS <br /> the steady-state levels mRNAs (36). Perhaps for <br /> these mRNAs, CAF1-independent, XRNA-dependent             This work supported Deutsche Fors- <br /> mechanism mRNA decay predominates; bulk          chungsgemeinschaft (DFG Cl112/9) (A.S.) Well- <br /> mRNA decay a&#239;&#172;&#8364;ected, transcription con-            come Trust (L.E. M.C.). We thank Ute Leibfried <br /> tinues unabated.                                              (ZMBH) Jochen Kreth (DKFZ) technical assis- <br />    In human cells, depletion Ccr4 homologues did      tance. G.S. supported Young Investigators <br /> not a&#239;&#172;&#8364;ect abundance (Figure 6) degradation             Grant Helmholtz Association German <br /> (Figure 7) unstable mRNA contains ARE.          Research Centres. The page charges paid the <br /> In contrast, simultaneous depletion Caf1b Caf1a        Grundausstattung (State Baden Wu&#194;&#168;rttemberg) the <br /> inhibited degradation ARE mRNA (Figures 6          Clayton laboratory. <br /> 7). We note interpretation results       Con&#239;&#172;&#8218;ict statement. None declared. <br /> straightforward. All experiments subject the <br /> inevitable weakness RNAi: depletion the <br /> target mRNA complete. Thus, residual dead-            REFERENCES <br /> enylase activity CAF1- (or Caf1-)depleted cells   1. Parker,R. Song,H. (2004) The enzymes control of <br /> attributable remaining Caf1, alternative         eukaryotic mRNA turnover. Nat. Struct. Mol. Biol., 11, 121&#226;&#8364;&#8220;127. <br /> deadenylases both. Similarly, absence e&#239;&#172;&#8364;ect   2. Chen,C.Y., Xu,N. Shyu,A.B. (1995) mRNA decay mediated <br /> Ccr4a/b depletion ARE mRNA decay does                     distinct AU-rich elements c-fos granulocyte- <br />                                                                  macrophage colony-stimulating factor transcripts: di&#239;&#172;&#8364;erent <br /> necessarily mean proteins minor role,            deadenylation kinetics uncoupling translation. Mol. Cell. <br /> but merely persistence su&#239;&#172;&#402;cient protein       Biol., 15, 5777&#226;&#8364;&#8220;5788. <br /> to wild-type function. In addition, Ccr4a+b        3. Chen,J., Chiang,Y.-C. Denis,C.L. (2002) CCR4, 30 -50 poly(A) <br /> RNAi, cells retain Caf1&#226;&#8364;&#8220;Not complex. In contrast,        RNA ssDNA exonuclease, catalytic component the <br /> Ccr4 linked Not1 Caf1, depletion Caf1            cytoplasmic deadenylase. EMBO J., 21, 1414&#226;&#8364;&#8220;1426. <br />                                                               4. Tucker,M., Staples,R.R., Valencis-Sanchez,M.A., Muhlrad,D. and <br /> expected result cells Not complex lacking      Parker,R. (2002) Ccr4p catalytic subunit a <br /> Ccr4 Caf1. Even Ccr4 persists cells,           Ccr4p/Pop2p/Notp mRNA deadenylase complex Saccharomyces <br /> recruitment ARE mRNA depend attachment              cerevisiae. EMBO J., 21, 1427&#226;&#8364;&#8220;1436. <br />                                                                                      Nucleic Acids Research, 2008, Vol. 36, No. 10 3387 <br />  <br />  <br />  5. Tucker,M., Valencia-Sanchez,M.A., Staples,R.R., Chen,J.,                 22. Goldstrohm,A.C., Seay,D.J., Hook,B.A. Wickens,M. (2006) <br />     Denis,C.L. Parker,R. (2001) The transcription factor                     PUF protein-mediated deadenylation catalyzed Ccr4p. <br />     associated Ccr4 Caf1 proteins components major                J. Biol. Chem., 282, 109&#226;&#8364;&#8220;114. <br />     cytoplasmic mRNA deadenylase Saccharomyces cerevisiae. Cell,          23. Ohn,T., Chiang,Y.-C., Lee,D., Yao,G., Zhang,C. Denis,C. <br />     104, 377&#226;&#8364;&#8220;386.                                                                (2007) CAF1 plays important role mRNA deadenylation <br />  6. Ko&#194;&#168;rner,C.G., Wormington,M., Muckenthaler,M., Schneider,S.,                  separate contact CCR4. Nucleic Acids Res., 35, <br />     Dehlin,E. Wahle,E. (1998) The deadenylating nuclease (DAN)               3002&#226;&#8364;&#8220;3015. <br />     involved poly(A) tail removal meiotic maturation     24. Viswanathan,P., Ohn,T., Chiang,Y.C., Chen,J. Denis,C.L. <br />     Xenopus oocytes. EMBO J., 17, 5427&#226;&#8364;&#8220;5437.                                     (2004) Mouse CAF1 function processive deadenylase/ <br />  7. Yamashita,A., Chang,T.C., Yamashita,Y., Zhu,W., Zhong,Z.,                    30 -50 -exonuclease vitro yeast deadenylase function of <br />     Chen,C.Y. Shyu,A.B. (2005) Concerted action poly(A)                   CAF1 required mRNA poly(A) removal. J. Biol. Chem., <br />     nucleases decapping enzyme mammalian mRNA turnover.                   279, 23988&#226;&#8364;&#8220;23995. <br />     Nat. Struct. Mol. Biol., 12, 1054&#226;&#8364;&#8220;1063.                                  25. Daugeron,M.C., Mauxion,F. Seraphin,B. (2001) The yeast <br />  8. Bai,Y., Salvadore,C., Chiang,Y.C., Collart,M.A., Liu,H.Y.                POP2 gene encodes nuclease involved mRNA deadenylation. <br />     Denis,C.L. (1999) The CCR4 CAF1 proteins                          Nucleic Acids Res., 29, 2448&#226;&#8364;&#8220;2455. <br />     CCR4-NOT complex physically functionally separated               26. Sakai,A., Chibazakura,Y., Shimizu,Y. Hishinuma,F. (1992) <br />     NOT2, NOT4, NOT5. Mol. Cell. Biol., 19, 6642&#226;&#8364;&#8220;6651.                  Molecular analysis POP2 gene, gene required glucose <br />  9. Draper,M., Liu,H.-Y., Nelsbach,A., Mosley,S. Denis,C. (1994)             repression Saccharomyces cerevisiae. Nucleic Acids Res., 20, <br />     CCR4 glucose-regulated transcription factor leucine-rich          6227&#226;&#8364;&#8220;6233. <br />     repeat binds proteins important placing CCR4          27. Denis,C.L. Malvar,T. (1990) The CCR4 gene from <br />     proper promoter context. Mol. Cell. Biol., 14, 4522&#226;&#8364;&#8220;4531.                    Saccharomyces cerevisiae required both <br /> 10. Clark,L.B., Viswanathan,P., Quigley,G., Chiang,Y.C.,                         nonfermentative spt-mediated gene expression. Genetics, 124, <br />     McMahon,J.S., Yao,G., Chen,J., Nelsbach,A. Denis,C.L.                    283&#226;&#8364;&#8220;291. <br />     (2004) Systematic mutagenesis leucine-rich repeat (LRR)           28. Molin,L. Puisieux,A. (2005) C. elegans homologue Caf1 <br />     domain CCR4 reveals speci&#239;&#172;?c sites binding CAF1               gene, encodes subunit CCR4-NOT complex, is <br />     separate critical role LRR CCR4 deadenylase activity.             essential embryonic larval development meiotic <br />     J. Biol. Chem., 279, 13616&#226;&#8364;&#8220;13623.                                            progression. Gene, 358, 73&#226;&#8364;&#8220;81. <br /> 11. Dupressoir,A., Morel,A.-P., Barbot,W., Loireau,M.-P., Corbo,L.           29. Maeda,I., Kohara,Y., Yamamoto,M. Sugimoto,A. (2001) <br />     Heidmann,T. (2001) Identi&#239;&#172;?cation families yCCR4-              Large-scale analysis gene function Caenorhabditis elegans by <br />     Mg2+-dependent endonuclease-related proteins higher                   high-throughput RNAi. Curr. Biol., 11, 171&#226;&#8364;&#8220;176. <br />     eukaryotes, characterization orthologs yCCR4            30. Albert,T.K., Lemaire,M., van Berkum,N.L., Gentz,R., Collart,M.A. <br />     conserved leucine-rich repeat essential hCAF1/hPOP2 binding.             Timmers,H.T. (2000) Isolation characterization human <br />     BMC Genomics, 2, 9.                                                          orthologs yeast CCR4-NOT complex subunits. Nucleic Acids <br /> 12. Albert,T., Hanzawa,H., Legtenberg,Y.I.A., deRuwe,M.J., van der               Res., 28, 809&#226;&#8364;&#8220;817. <br />     Heuvel,F.A.J., Collart,M.A., Boelens,R. Timmers,M.H.T.               31. Berthet,C., Morera,A.-M., Asensio,M.-J., Chauvin,M.-A., <br />     (2002) Identi&#239;&#172;?cation ubiquitin&#226;&#8364;&#8220;protein ligase subunit        Morel,A.-P., Dijoud,F., Magaud,J.-P., Durand,P. Rouault,J.-P. <br />     CCR4-NOT transcription repressor complex. EMBO J., 21,                       (2004) CCR4-associated factor CAF1 Is essential factor for <br />     355&#226;&#8364;&#8220;364.                                                                     spermatogenesis. Mol. Cell. Biol., 24, 5808&#226;&#8364;&#8220;5820. <br /> 13. Russell,P., Benson,J. Denis,C. (2002) Characterization            32. Parfrey,L., Barbero,E., Lasser,E., Dunthorn,M. and <br />     mutations NOT2 indicates plays important role               Bhattacharya,D.E.A. (2006) Evaluating support for <br />     maintaining integrity CCR4 &#226;&#8364;&#8220; NOT Complex. J. Mol.                 current classi&#239;&#172;?cation eukaryotic diversity. PLoS Genet., <br />     Biol., 322, 27&#226;&#8364;&#8220;39.                                                           2, e220. <br /> 14. Collart,M. Timmers,H. (2004) The eukaryotic Ccr4-Not                 33. Clayton,C. Shapira,M. (2007) Post-transcriptional regulation <br />     complex: regulatory platform Integrating mRNA metabolism                   gene expression trypanosomes leishmanias. Mol. Biochem. <br />     cellular signaling pathways? Prog. Nucleic Acid Res. Mol. Biol.,        Parasitol.,, 156, 93&#226;&#8364;&#8220;101. <br />     77, 289.                                                                 34. Este&#194;&#180;vez,A.M., Lehner,B., Sanderson,C.M., Ruppert,T. and <br /> 15. Temme,C., Zaessinger,S., Meyer,S., Simonelig,M. Wahle,E.                 Clayton,C. (2003) The roles inter-subunit interactions in <br />     (2004) A complex containing CCR4 CAF1 proteins                    exosome stability. J. Biol. Chem., 278, 34943&#226;&#8364;&#8220;34951. <br />     involved mRNA deadenylation Drosophila. EMBO J., 23,               35. Haile,S., Este&#194;&#180;vez,A.M. Clayton,C. (2003) A role the <br />     2862&#226;&#8364;&#8220;2871.                                                                   exosome initiation degradation unstable mRNAs. RNA, <br /> 16. Okazaki,N., Okazaki,K., Watanabe,Y., Kato-Hayashi,M.,                        9, 1491&#226;&#8364;&#8220;1501. <br />     Yamamoto,M. Okayama,H. (1998) Novel factor highly                    36. Li,C.-H., Irmer,H., Gudjonsdottir-Planck,D., Freese,S., Salm,H., <br />     conserved eukaryotes controls sexual development &#239;&#172;?ssion             Haile,S., Este&#194;&#180;vez,A.M. Clayton,C.E. (2006) Roles a <br />     yeast. Mol. Cell. Biol., 18, 887&#226;&#8364;&#8220;895.                                        Trypanosoma brucei 50 -&gt;30 exoribonuclease homologue mRNA <br /> 17. Tsukahara,K., Yamamoto,H. Okayama,H. (1998) An RNA                       degradation. RNA, 12, 2171&#226;&#8364;&#8220;2186. <br />     binding protein negatively controlling di&#239;&#172;&#8364;erentiation &#239;&#172;?ssion yeast.   37. Milone,J., Wilusz,J. Bellofatto,V. (2004) Characterization of <br />     Mol. Cell. Biol., 18, 4488&#226;&#8364;&#8220;4498.                                             deadenylation trypanosome extracts inhibition by <br /> 18. Chen,J., Rappsilber,J., Chiang,Y.C., Russell,P., Mann,M.                 poly(A)-binding protein Pab1p. RNA, 10, 448&#226;&#8364;&#8220;457. <br />     Denis,C.L. (2001) Puri&#239;&#172;?cation characterization 1.0 MDa        38. Biebinger,S., Wirtz,L.E. Clayton,C.E. (1997) Vectors for <br />     CCR4-NOT complex identi&#239;&#172;?es novel components                       inducible over-expression potentially toxic gene products in <br />     complex. J. Mol. Biol., 314, 683&#226;&#8364;&#8220;694.                                        bloodstream procyclic Trypanosoma brucei. Mol. Biochem. <br /> 19. Coller,J.M., Tucker,M., Sheth,U., Valencia-Sanchez,M.A.                  Parasitol., 85, 99&#226;&#8364;&#8220;112. <br />     Parker,R. (2001) The DEAD box helicase, Dhh1p, functions              39. Alibu,V.P., Storm,L., Haile,S., Clayton,C. Horn,D. (2004) A <br />     mRNA decapping interacts decapping                     doubly inducible RNA interference rapid RNAi <br />     deadenylase complexes. RNA, 7, 1717&#226;&#8364;&#8220;1727.                                    plasmid construction Trypanosoma brucei. Mol. Biochem. <br /> 20. Hata,H., Mitsui,H., Liu,H., Bai,Y., Denis,C.L., Shimizu,Y.               Parasitol., 139, 75&#226;&#8364;&#8220;82. <br />     Sakai,A. (1998) Dhh1p, putative RNA helicase, associates          40. Wirtz,E., Hoek,M. Cross,G.A.M. (1998) Regulated processive <br />     general transcription factors Pop2p Ccr4p                       transcription chromatin T7 RNA polymerase Trypanosoma <br />     Saccharomyces cerevisiae. Genetics, 148, 571&#226;&#8364;&#8220;579.                            brucei. Nucleic Acids Res., 26, 4626&#226;&#8364;&#8220;4634. <br /> 21. Maillet,L. Collart,M.A. (2002) Interaction Not1p,          41. Clayton,C.E., Este&#194;&#180;vez,A.M., Hartmann,C., Alibu,V.P., Field,M. <br />     component Ccr4-not complex, global regulator                     Horn,D. (2005) Down-regulating gene expression RNA <br />     transcription, Dhh1p, putative RNA helicase. J. Biol. Chem.,           interference Trypanosoma brucei. Methods Mol. Biol., 309, <br />     277, 2835&#226;&#8364;&#8220;2842.                                                              39&#226;&#8364;&#8220;60. <br />  3388 Nucleic Acids Research, 2008, Vol. 36, No. 10 <br />  <br />  <br /> 42. Quijada,L., Hartmann,C., Guerra-Giraldez,C., Drozdz,M.,               56. Ziegelbauer,K. Overath,P. (1992) Identi&#239;&#172;?cation invariant <br />     Irmer,H. Clayton,C.E. (2002) Expression human                  surface glycoproteins bloodstream stage Trypanosoma <br />     RNA-binding protein HuR Trypanosoma brucei induces                     brucei. J. Biol. Chem., 267, 10791&#226;&#8364;&#8220;10796. <br />     di&#239;&#172;&#8364;erentiation-related changes abundance developmentally-   57. Ziegelbauer,K., Multhaup,G. Overath,P. (1992) Molecular <br />     regulated mRNAs. Nucleic Acids Res., 30, 1&#226;&#8364;&#8220;11.                            characterisation invariant surface glycoproteins speci&#239;&#172;?c for <br /> 43. Este&#194;&#180;vez,A., Kempf,T. Clayton,C.E. (2001) The exosome              bloodstream stage Trypanosoma brucei. J. Biol. Chem., 267, <br />     Trypanosoma brucei. EMBO J., 20, 3831&#226;&#8364;&#8220;3839.                               10797&#226;&#8364;&#8220;10803. <br /> 44. Redmond,S., Vadivelu,J. Field,M.C. (2003) RNAit:               58. Cristodero,M. Clayton,C. (2007) Trypanosome MTR4 in <br />     automated web-based tool selection RNAi                        involved ribosomal RNA processing. Nucleic Acids Res., 35, <br />     targets Trypanosoma brucei. Mol. Biochem. Parasitol., 128,             7023&#226;&#8364;&#8220;30. <br />     115&#226;&#8364;&#8220;118.                                                              59. Liang,X., Haritan,A., Uliel,S. Michaeli,S. (2003) Trans cis <br /> 45. Stoecklin,G., Mayo,T. Anderson,P. (2005) ARE-mRNA                     splicing trypanosomatids: mechanism, factors, regulation. <br />     degradation requires 5&#226;&#8364;&#8482;&#226;&#8364;&#8220;3&#226;&#8364;&#8482; decay pathway. EMBO Rep, 7,                Eukaryot. Cell, 2, 830&#226;&#8364;&#8220;840. <br />     72&#226;&#8364;&#8220;77.                                                                60. Benz,C., Nilsson,D., Andersson,B., Clayton,C. Guilbride,D.L. <br /> 46. Reynolds,A., Leake,D., Boese,Q., Scaringe,S., Marshall,W.             (2005) Messenger RNA processing sites Trypanosoma brucei. <br />     Khvorova,A. (2004) Rational siRNA design RNA interference.            Mol. Biochem. Parasitol., 143, 125&#226;&#8364;&#8220;134. <br />     Nat. Biotechnol., 22, 326&#226;&#8364;&#8220;330.                                        61. Baer,B. Kornberg,R. (1980) Repeating structure cytoplasmic <br /> 47. Gough,N. (1988) Rapid quantitative preparation cytoplasmic         poly(A)-ribonucleoprotein. Proc. Natl Acad. Sci. USA, 77, <br />     RNA small numbers cells. Anal. Biochem., 173, 93&#226;&#8364;&#8220;95.              1890&#226;&#8364;&#8220;1892. <br /> 48. Stoecklin,G., Stubbs,T., Kedersha,N., Wax,S., Rigby,W.F.C.,           62. Meijer,H., Bushell,M., Hill,K., Gant,T., Willis,A., Jones,P. de <br />     Blackwell,T.K. Anderson,P. (2004) MK2-induced                         Moor,C. (2007) A novel method poly(A) fractionation reveals a <br />     tristetraprolin:14-3-3 complexes prevent stress granule association       large population mRNAs short poly(A) tail mammalian <br />     ARE-mRNA decay. EMBO J., 23, 1313&#226;&#8364;&#8220;1324.                               cells. Nucleic Acids Res., 35, e132. <br /> 49. Das,A., Li,H., Liu,T. Bellofatto,V. (2006) Biochemical            63. Wagner,E., Clement,S. Lykke-Andersen,J. (2007) <br />     characterization Trypanosoma brucei RNA polymerase II. Mol.            An unconventional human Ccr4-Caf1 deadenylase complex in <br />     Biochem. Parasitol., 150, 201&#226;&#8364;&#8220;210.                                        nuclear Cajal bodies. Mol. Cell. Biol., 27, 1686. <br /> 50. Hartmann,C., Benz,C., Brems,S., Ellis,L., Luu,V.-D., Stewart,M.,      64. van Dijk,E., Schilders,G. Pruijn,G. (2007) Human cell growth <br />     D&#226;&#8364;&#8482;Orso,I., Busold,C., Fellenberg,K., Frasch,A.C.C. et al. (2007)          requires functional cytoplasmic exosome, involved in <br />     The small trypanosome RNA-binding proteins TbUBP1                     various mRNA decay pathways. RNA, 13, 1027&#226;&#8364;&#8220;1035. <br />     TbUBP2 in&#239;&#172;&#8218;uence expression F box protein mRNAs                  65. Muhlrad,D. Parker,R. (1999) Recognition yeast mRNAs as <br />     bloodstream trypanosomes. Eukaryot. Cell, 6, 1964&#226;&#8364;&#8220;1978.                   &#226;&#8364;&#732;&#226;&#8364;&#732;nonsense containing&#226;&#8364;&#8482;&#226;&#8364;&#8482; leads inhibition mRNA <br /> 51. Cassola,A., De Gaudenzi,J. Frasch,A. (2007) Recruitment               translation mRNA degradation: implications Control of <br />     mRNAs cytoplasmic ribonucleoprotein granules trypano-            mRNA decapping. Mol. Biol. Cell, 11, 3971&#226;&#8364;&#8220;3978. <br />     somes. Mol. Microbiol., 65, 655&#226;&#8364;&#8220;670.                                  66. He,F., Li,X., Spatrick,P., Casillo,R., Dong,S. Jacobson,A. <br /> 52. Hotz,H.-R., Hartmann,C., Huober,K., Hug,M. Clayton,C.E.               (2003) Genome-wide analysis mRNAs regulated the <br />     (1997) Mechanisms developmental regulation Trypanosoma              nonsense-mediated 5&#226;&#8364;&#8482; 3&#226;&#8364;&#8482; mRNA decay pathways yeast. <br />     brucei: polypyrimidine tract 3&#226;&#8364;&#8482;-untranslated region         Mol. Cell, 12, 1439&#226;&#8364;&#8220;1452. <br />     trypanosome surface protein mRNA a&#239;&#172;&#8364;ects RNA abundance             67. Grigull,J., Mnaimneh,S., Pootoolal,J., Robinson,M. Hughes,T. <br />     translation. Nucleic Acids Res., 25, 3017&#226;&#8364;&#8220;3025.                           (2004) Genome-wide analysis mRNA stability using <br /> 53. Hug,M., Carruthers,V., Sherman,D., Hartmann,C., Cross,G.A.M.              transcription inhibitors microarrays reveals posttranscriptional <br />     Clayton,C.E. (1993) A possible role 30 -untranslated          control ribosome biogenesis factors. Mol. Cell. Biol., 24, <br />     region developmental regulation Trypanosoma brucei.                 5534&#226;&#8364;&#8220;5547. <br />      <br /> </body></html>