<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proper Levels Arabidopsis Cohesion Establishment <br /> Factor CTF7 Are Essential Embryo and <br /> Megagametophyte, But Not Endosperm, Development1[OA] <br />  <br /> Ling Jiang2,3, Li Yuan2, Ming Xia4, Christopher A. Makaroff* <br /> Department Chemistry Biochemistry, Miami University, Oxford, Ohio 45056 <br />  <br />  <br /> CTF7 essential gene yeast required formation sister chromatid cohesion. While recent studies have <br /> provided insights sister chromatid cohesion established, known specifically CTF7 facilitates the <br /> formation cohesion, essentially known sister chromatid cohesion established plants. In this <br /> report, isolation characterization CTF7 Arabidopsis (Arabidopsis thaliana). Arabidopsis CTF7 is <br /> similar Saccharomyces cerevisiae CTF7 lacks amino-terminal extension, exhibits acetyltransferase activity, can <br /> complement yeast ctf7 temperature-sensitive <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. CTF7 transcripts plant, highest <br /> levels present buds. Seeds containing T-DNA insertions CTF7 exhibit mitotic defects zygote. Interestingly, the <br /> endosperm developed normally ctf7 seeds, suggesting CTF7 essential mitosis endosperm nuclei. Minor <br /> defects observed female gametophytes ctf71/2 plants, plants overexpress CTF7 exhibited female <br /> gametophyte lethality. Pollen development appeared normal CTF7 <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>knockout</span> overexpression plants. Therefore, <br /> proper levels CTF7 critical female gametophyte embryo development establishment mitotic <br /> cohesion microgametogenesis endosperm development. <br />  <br />  <br />  <br />    The proper formation sister chromatid cohesion                        metaphase-<span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>-anaphase transition opens ring, al- <br /> and subsequent release metaphase-to-ana-                         lowing release sister chromatids (for review, <br /> phase transition essential proper segregation                    Nasmyth Haering, 2009). <br /> of genetic material cell division. It critical                   The establishment sister chromatid cohesion in <br /> the compaction chromosomes bipolar                             yeast involves multistep process begins during <br /> attachment spindle, DNA double-strand break                          telophase, cohesin complexes associate with <br /> repair, regulation gene expression (for                          chromatin process requires Scc2/Scc4 <br /> review, Nasmyth Haering, 2009). Sister chro-                        complex (Ciosk et al., 2000; Gillespie Hirano, 2004) <br /> matid cohesion controlled cohesin complex,                        appears involve binding opening <br /> which consists heterodimer Structural Mainte-                       SMC hinge domain (Gruber et al., 2006; Hirano <br /> nance Chromosome (SMC) proteins, SMC1                                Hirano, 2006). Cohesion established during <br /> SMC3, Sister Chromatid Cohesion (SCC) protein                               S-phase Ctf7/Eco1-dependent process (Skibbens <br /> SCC3, SCC1, a-kleisin protein. Perhaps                           et al., 1999; Toth et al., 1999). Ctf7 interacts a <br /> most widely accepted model cohesin functions                         number replication factors appears facilitate <br /> has referred ring model,                           cohesion formation context passing rep- <br /> cohesin ring encircles replicated sisters, holding                      lication fork (Kenna Skibbens, 2003; Lengronne <br /> them SCC1 cleavage separase                        et al., 2006; Moldovan et al., 2006). An antiestablish- <br />                                                                             ment complex consisting Wings Apart-Like protein <br />   1 <br />      This work supported National Science Foundation <br />                                                                             (WAPL)/Rad61 Pds5 required maintain <br /> (grant no. MCB0718191 C.A.M.).                                           cohesion G2/M stabilizing interaction <br />    2 <br />      These authors contributed equally article.                      cohesin chromosomes (Panizza et al., <br />    3 <br />      Present address: Biotechnology Research Institute, Chinese             2000; Losada et al., 2005; Gandhi et al., 2006). While the <br /> Academy Agricultural Sciences, 12 ZhongGuanCun NanDaJie,                 specific details Ctf7, WAPL/Rad61, Pds5 <br /> Haidian District, Beijing 100081, People&#226;&#8364;&#8482;s Republic China.               function establish maintain <br />    4 <br />      Present address: Division Infectious Diseases, Cincinnati           cohesion need clarified, recent results indi- <br /> Children&#226;&#8364;&#8482;s Hospital Medical Center, 3333 Burnet Avenue, Cincin-             cate Ctf7 acetylates conserved Lys residues in <br /> nati, OH 45229.                                                             SMC3, inhibits antiestablishment function <br />    * Corresponding author; e-mail makaroca@muohio.edu. <br />                                                                             Wpl1-Pds5 complex promotes cohesion <br />    The author responsible distribution materials integral the <br /> findings presented article accordance policy <br />                                                                             establishment (Ben-Shahar et al., 2008; Unal et al., <br /> described Instructions Authors (www.plantphysiol.org) is:        2008; Zhang et al., 2008; Rowland et al., 2009; Sutani <br /> Christopher A. Makaroff (makaroca@muohio.edu).                              et al., 2009). Ctf7 involved postreplicative <br />    [OA] <br />         Open Access articles viewed online subscription.   induction cohesion induced DNA double-strand <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.110.157560                       breaks (Unal et al., 2007). In instance, acetylation <br /> 820           Plant Physiology&#195;&#8217;, October 2010, Vol. 154, pp. 820&#226;&#8364;&#8220;832, www.plantphysiol.org &#195;&#8220; 2010 American Society Plant Biologists <br />                                                                                   Arabidopsis Cohesion Establishment Factor <br />  <br />  <br /> of a-kleisin subunit, Mcd1, Ctf7 required       nificantly lower similarity levels (E values = 0.62 and <br /> cohesion establishment (Heidinger-Pauli et al., 2009).        1.7, respectively) missing highly <br />    The structure CTF7 proteins effects      conserved residues Ctf7/Eco1 proteins <br /> of ctf7 mutations vary considerably orga-             studied date (see below). Therefore, Arabidopsis <br /> nisms (Skibbens et al., 1999; Toth et al., 1999; Tanaka       genome appears contain putative CTF7 gene <br /> et al., 2000; Bellows et al., 2003; Williams et al., 2003;    (At4g31400), called CTF7. A CTF7 cDNA was <br /> Hou Zou, 2005). Saccharomyces cerevisiae ctf7 mu-         isolated compared published genomic <br /> tations cause delay mitosis cells 2C          sequence At4g31400. CTF7 consists exons and <br /> DNA content, separated DNA masses, partially              introns (Fig. 1A). While predicted actual <br /> elongated spindles (Skibbens et al., 1999; Toth et al.,       gene structures similar, differences positions <br /> 1999). The fission yeast eso1-H17 mutant displays           splice sites exons 2, 3, 4. <br /> spindle assembly checkpoint-dependent mitotic delay              Arabidopsis CTF7 capable encoding 345- <br /> and abnormal chromosome segregation (Tanaka et al.,           amino acid protein. Similar S. cerevisiae Eco1 (281 <br /> 2000). Mutation Drosophila DECO affects distri-        amino acids), Arabidopsis CTF7 lacks large N-ter- <br /> bution cohesin proteins, cells progress prema-         minal extension Eco1 <br /> turely anaphase, spindle checkpoint         proteins (Fig. 1B). A PIP box (QxxL/I, QFHL) a <br /> activated (Williams et al., 2003). The small, interfering     C2H2 zinc finger motif present residues 82 86 <br /> RNA-directed depletion ESCO1 ESCO2 HeLa              92 130, respectively, acetyltransferase <br /> cells caused defects sister chromatid cohesion, mis-       domain present amino acids 184 335 <br /> alignment chromosomes spindle, abnor-           (Fig. 1, B C). Arabidopsis CTF7 exhibits between <br /> mal chromosome segregation; however, did               24% 58% similarity CTF7 proteins yeast, <br /> appear affect distribution cohesin proteins         fruit fly, vertebrates, level of <br /> (Hou Zou, 2005). Therefore, possible           similarity observed CTF7 homologs <br /> CTF7 proteins multiple roles organisms            (Fig. 1C). Overall, Arabidopsis CTF7 shows high- <br /> and/or unique properties different organisms.         est sequence similarity human Eco1 (22% identity, <br />    The Arabidopsis (Arabidopsis thaliana) genome con-         58% similarity). <br /> tains orthologs cohesion-associated              CTF7 transcript levels analyzed different <br /> genes identified yeast, studies         tissues using reverse transcription (RT)-PCR. Low but <br /> have shown cohesion machinery generally           detectible levels CTF7 transcripts all <br /> conserved Arabidopsis (Bai et al., 1999; Bhatt et al.,     tissues examined, including buds, roots, leaves, stems, <br /> 1999; Dong et al., 2001; Chelysheva et al., 2005; Lam         young seedlings, buds exhibiting highest <br /> et al., 2005; da Costa-Nunes et al., 2006; Liu            signal mature leaves lowest (Fig. 1E). Expres- <br /> Makaroff, 2006; Jiang et al., 2007). However, essentially     sion profiling data CTF7 consistent these <br /> nothing known sister chromatid cohesion          results CTF7 transcript levels high- <br /> is established plants. Therefore, interested       est bolting plants, buds, young flowers. CTF7 <br /> in determining Arabidopsis contains functional           transcripts elevated E2FaDPa transgenic <br /> ortholog CTF7, so, role plays plant      plants, overexpress S-phase genes (Vlieghe <br /> growth development.                                       et al., 2003). Therefore, CTF7 expression patterns are <br />    In report, present results analysis   consistent predicted role nuclear division. <br /> Arabidopsis CTF7, Cohesion Establishment Factor,             Yeast, fly, human CTF7 proteins exhibit acetyl- <br /> and characterization plants containing muta-           transferase activity (Zou et al., 1999; Ivanov et al., 2002; <br /> tions CTF7. Seeds containing T-DNA insertions           Bellows et al., 2003; Williams et al., 2003). In order to <br /> CTF7 exhibit embryo lethality early development,           determine case Arabidopsis CTF7, <br /> while endosperm develops normally. Minor de-              protein overexpressed Escherichia coli and <br /> fects observed female gametophytes           characterized. Soluble CTF7 produced ex- <br /> pollen germination CTF71/2 plants; however, over-          pressed maltose-binding protein (MBP) fusion in <br /> expression CTF7 blocked female gametophyte de-             presence pGroESL. Acetyltransferase activity <br /> velopment FG1. Therefore, variations CTF7 levels        measured autoacetylation crude E. coli ex- <br /> have different effects different cell types.          tracts purified CTF7 protein. Western blotting <br />                                                               anti-acetyl-Lys antibody matrix-assisted <br />                                                               laser-desorption ionization time flight-mass spec- <br /> RESULTS                                                       trometry demonstrated CTF7 acetylated during <br /> Arabidopsis Contains Single CTF7 Gene That Lacks         vitro autoacetylation assays (Fig. 2A). Therefore, <br /> N-Terminal Extension                                          Arabidopsis CTF7 exhibits acetyltransferase activity in <br />                                                               vitro able undergo autoacetylation. <br />    BLAST searches Arabidopsis protein database            The question Arabidopsis protein <br /> identified gene (At4g31400) high sequence            substitute yeast Eco1 investigated by <br /> similarity human (E value = 3e221) S.       transforming yeast ctf7 temperature-sensitive mu- <br /> cerevisiae (E value = 3e205) Eco1 proteins. Other genes       tant, ctf7-203 (YBS514), Arabidopsis CTF7 <br /> (At3g20350 At5g26190) identified sig-           cDNA. A pAS2-CTF7 construct able restore the <br /> Plant Physiol. Vol. 154, 2010                                                                                         821 <br />  Jiang et al. <br />  <br />  <br />  <br />  <br />           Figure 1. Arabidopsis CTF7 locus protein structure. A, Gene map CTF7. The exon positions T-DNA insertion site are <br />           shown. Primers used study shown map. LB, Left border; RB, right border. B, Schematic representations of <br />           ECO1/CTF7 proteins different organisms. The ECO1/CTF7 domain (gray box), C2H2 zinc finger domain (black box), PIP <br />           box (thin gray line) shown. Accession numbers follows: Arabidopsis CTF7 (EU077499), O. sativa CTF7 (Q7XY81), S. <br />           pombe Eso1 (O42917), S. cerevisiae Eco1 (P43605), Homo sapiens Esco1 (Q5FWF5), Drosophila Eco (Q9VS50), H. sapiens <br />           Esco2 (Q56NI9). aa, Amino acids. C, Alignment ECO1/CTF7 proteins different organisms. The conserved PIP box, C2H2 <br />           zinc finger motif, acetyltransferase domain overlined. Identical similar amino acids shaded black gray, <br />           respectively. D, Phylogenetic tree characterized ECO1/CTF7 proteins. E, CTF7 transcripts different tissues Arabidopsis. <br />           ACTIN8 (ACT8) transcripts used control. <br />  <br />  <br /> ability YBS514 cells grow 33&#194;&#176;C, YBS514                        (ctf7-1) SAIL_1214G06 (ctf7-2) T-DNA insertion <br /> cells containing pAS2 vector (Fig.                        lines. PCR amplification followed DNA sequence <br /> 2B). Therefore, Arabidopsis CTF7 able substitute                       analysis confirmed ctf7-1 ctf7-2 T-DNA <br /> for S. cerevisiae Eco1. Taken together, results                        insertions located intron 4 exon 3, respec- <br /> confirm CTF7 Arabidopsis cohesion estab-                         tively. Both mutations predicted disrupt the <br /> lishment factor.                                                             gene upstream acetyltransferase domain. <br />                                                                                 No homozygous ctf7-1 ctf7-2 plants identi- <br /> Genetic Analysis Arabidopsis ctf7 Plants                                  fied initial screens. Therefore, seeds self-pol- <br />                                                                              linated, heterozygous ctf71/2 plants collected and <br />  The role CTF7 plant growth development                            resown, progeny screened PCR. Fifty- <br /> was investigated characterizing SALK_059500                           percent (223 379) progeny hetero- <br /> 822                                                                                                                Plant Physiol. Vol. 154, 2010 <br />                                                                                                 Arabidopsis Cohesion Establishment Factor <br />  <br />  <br />                                                                             plants obtained, 120 wild-type 91 <br />                                                                             ctf7-1 plants obtained ctf7-11/2 plants were <br />                                                                             used male parent. Similar results obtained <br />                                                                             backcross experiments ctf7-21/2. These results <br />                                                                             indicated mutation transmitted through <br />                                                                             male female gametophytes the <br />                                                                             transmission efficiency reduced male ga- <br />                                                                             metophytes. <br />                                                                                Similar phenotypes observed ctf7-1 and <br />                                                                             ctf7-2 plants analyses. Therefore, the <br />                                                                             results obtained analysis ctf7-1 plants are <br />                                                                             presented, mutant referred generally <br />                                                                             ctf7 on. A light microscope analysis of <br />                                                                             semithin sections stage 11 12 anthers <br />                                                                             (Sanders et al., 1999) failed identify abnormalities in <br />                                                                             pollen development ctf71/2 plants (data shown). <br />                                                                             Pollen ctf71/2 plants appeared uniform size <br />                                                                             normal examined scanning electron <br />                                                                             microscopy (SEM; Fig. 3A). The large majority, not <br />                                                                             all, pollen ctf71/2 plants contained two <br />                                                                             sperm nuclei vegetative nucleus, confirming that <br />                                                                             pollen mitosis disrupted mutation (Fig. <br />                                                                             3B). No significant difference pollen viability was <br />                                                                             observed wild-type ctf71/2 plants (Fig. <br />                                                                             3C). However, pollen ctf71/2 plants did exhibit a <br />                                                                             slightly reduced vitro germination frequency (73% <br />                                                                             wild-type levels; Fig. 3C). <br />                                                                                Further analysis ctf71/2 plants identified no <br />                                                                             obvious morphological alterations, exception <br />                                                                             exhibited reduced seed set (32 6 4 versus <br /> Figure 2. CTF7 Arabidopsis cohesion establishment factor. A, <br />                                                                             43 6 6 silique wild-type plants; n = 50; Fig. <br /> Arabidopsis CTF7 exhibits acetyltransferase activity. Lanes 1 3,        3C). Approximately 25% seeds siliques of <br /> Molecular mass markers; lane 2, Coomassie Brilliant Blue-stained SDS-       ctf71/2 plants cotyledon stage normal in <br /> PAGE gel affinity-purified CTF7-MBP; lane 4, Western blot affinity-   size white; later, seeds shrunken and <br /> purified CTF7-MBP treated anti-acetyl-Lys antibodies. Full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>      orange (Fig. 3D). The presence 25% aborted seeds <br /> CTF7-MBP marked arrowheads. A cleaved version CTF7            inctf71/2 siliques consistent mutation af- <br /> marked arrows, MBP labeled star. B, Arabidopsis          fects embryo development. <br /> CTF7 substitute S. cerevisiae CTF7. Cultures wild-type (WT) <br /> and temperature-sensitive ctf7-203 cells containing pAS2 control <br /> plasmid Arabidopsis CTF7 complementation plasmid grown           CTF7 Mutations Cause Minor Alterations Female <br /> on appropriate selection plates 28&#194;&#176;C 33&#194;&#176;C.                 Gametophyte Development <br />  <br />                                                                                Potential effects ctf7 mutation megagameto- <br /> zygous ctf7-1 plants heterozygous ctf7                         genesis characterized comparing ovules in <br /> T-DNA insertion, 41% plants wild                        wild-type ctf71/2 plants using laser scanning con- <br /> type. No homozygous mutant plants identified.                          focal microscopy. A total 36 pistils various stages of <br /> Likewise homozygous plants containing ctf7-2                         development different ctf71/2 plants were <br /> insertion identified, 54% (143 263)                         analyzed. Female gametophyte development de- <br /> the progeny heterozygous T-DNA inser-                         fined according Christensen et al. (1998). No detect- <br /> tion. These results indicated CTF7 essential                        able abnormalities identified embryo sacs of <br /> and suggested disruption cause embryo                          ctf71/2 plants stages FG0 FG6 (Fig. 4, <br /> lethality. Furthermore, observation self-                          A&#226;&#8364;&#8220;D). However, embryo sac development less <br /> pollinated ctf7-11/2 ctf7-21/2 plants consistently                      uniform siliques ctf71/2 plants of <br /> produced fewer heterozygous progeny expected                           wild-type plants. In contrast three <br /> (combined genetic data lines: 336:276                           developmental stages observed wild-type siliques <br /> [1.2:1] heterozygous:wild type; x2 = 0.03, P . 0.05)                        (Shi et al., 2005), different stages were <br /> also suggested gametophytic effect mutations.                      observed ctf71/2 siliques (Table I). For example, both <br />    To investigate possibility, ctf7-11/2 plants                   fertilized FG5 embryo sacs same <br /> backcrossed wild-type plants male                          silique. This observed wild-type siliques, <br /> and female parents. When ctf7-11/2 plants used                         suggesting ctf7 female gametophytes develop <br /> as female parent, 144 wild-type 146 ctf7-1                          slower wild-type female gametophytes. <br /> Plant Physiol. Vol. 154, 2010                                                                                                       823 <br />  Jiang et al. <br />  <br />  <br />                                                                              effects gametophyte development. While se- <br />                                                                              vere, likely contribute reduced <br />                                                                              transmission mutation observe. <br />  <br />                                                                              Inactivation CTF7 Blocks Embryo Development But <br />                                                                              Has No Significant Effect Endosperm Development <br />                                                                                 Analysis seed development ctf71/2 plants <br />                                                                              identified alterations zygote formation approxi- <br />                                                                              mately 25% seeds. In wild-type Arabidopsis <br />                                                                              plants, single-celled zygote primary endo- <br />                                                                              sperm nucleus normally observed with <br />                                                                              degenerated persistent synergid cells fertil- <br />                                                                              ization egg central cell. The endosperm <br />                                                                              nucleus undergoes series syncytial divisions <br />                                                                              produce endosperm domains: the <br />                                                                              embryo-surrounding region micropylar endosperm <br />                                                                              (MCE), peripheral endosperm (PEN) central <br />                                                                              chamber, chalazal endosperm (CZE; Olsen, <br />                                                                              2004). After endosperm nucleus undergone <br />                                                                              rounds nuclear division, zygote <br />                                                                              elongated divides asymmetrically, gen- <br />                                                                              erating smaller apical cell larger basal cell <br />                                                                              (Ingouff et al., 2005). Subsequent divisions produce an <br />                                                                              embryo composed embryo proper the <br />                                                                              suspensor, composed enlarged basal <br />                                                                              cell file additional cells. The <br />                                                                              embryo proper undergoes successive rounds divi- <br />                                                                              sion, forming two-cell, quadrant, octant (Fig. 5A), <br />                                                                              dermatogen, globular (Fig. 5G), early heart (Fig. 5M), <br />                                                                              heart, torpedo (Fig. 5S), mature embryo turn. <br /> Figure 3. Pollen development ctf71/2 plants. A, SEM micrograph            Fertilization appeared occur normally (Fig. 4, H <br /> an open ctf71/2 stamen. B, DAPI-stained mature pollen grains          I), alterations observed ctf7 zygotes <br /> ctf71/2 plant. C, Histogram comparing seed number, pollen germina-           soon division, number em- <br /> tion, pollen viability wild-type (WT) ctf71/2 plants. Data        bryos arrested subsequently degenerated the <br /> represent averages 20 individuals genotype. D, Wild-     one- two-cell zygote stage (Figs. 4, J&#226;&#8364;&#8220;L, 5D). <br /> type ctf7 seeds: i, ctf7 seeds normal size white; ii,   Some zygotes reached four-celled proembryo <br /> desiccation, ctf7 seeds orange shrunken (bottom row). Bars = <br />                                                                              stage, point suspensor embryo <br /> 20 mm A, 10 mm B, 1 mm D. <br />                                                                              proper exhibited altered division planes (Fig. 5, J and <br />                                                                              P). A small number embryos, exhibited <br />    The developmental abnormality observed                          dramatic alterations division, reached dermato- <br /> at stage FG7, antipodal cells did                         gen early globular stages arresting (Fig. 5V). <br /> degenerate normally ctf7 embryo sacs. Typically,                       Altogether, 23.7% 6 0.6% seeds ctf71/2 siliques <br /> the time egg synergid cells mature,                           (n = 7) exhibited defects early zygote embryo <br /> evidenced opposite positions nuclei                            development, strongly suggesting mutation <br /> vacuoles, antipodal cells degenerated                         causes embryo lethality blocking altering <br /> (Fig. 4A). However, antipodal cells                         nuclear division. <br /> found mature FG7 egg synergid cells                           In contrast, endosperm ctf7 seeds appeared <br /> embryo sacs ctf71/2 siliques. Antipodal cells                        relatively normal development. In wild- <br /> also observed recently fertilized embryo sacs                      type seeds, endosperm contains approximately 26 <br /> embryo sacs containing endosperm                            50 nuclear cytoplasmic domains (NCDs) time <br /> nuclei (Fig. 4, E&#226;&#8364;&#8220;G I). Antipodal cells                       embryo begins divide, octant-stage <br /> observed fertilized embryo sacs wild-type plants                       embryos typically surrounded approximately <br /> (Fig. 4, B&#226;&#8364;&#8220;D). Approximately 50% (45.8% 6 1.4%)                           100 uncellularized NCDs (Fig. 5, B C). The num- <br /> the embryo sacs ctf71/2 siliques exhibited                           bers appearance NCDs normal ctf7 <br /> delayed degeneration antipodal cells, suggesting                          seeds containing arrested zygotes proembryos <br /> that linked ctf7 mutation. This observa-                      (Fig. 5, D&#226;&#8364;&#8220;F). Cellularization MCE begins when <br /> tion, variations timing female                          embryo early globular stage wild-type <br /> gametophyte development reduced pollen                               seeds (Fig. 5G). At time, PEN showed a <br /> germination, indicates ctf7 mutation minor                      gradient cellularization stages (Fig. 5I), endo- <br /> 824                                                                                                        Plant Physiol. Vol. 154, 2010 <br />                                                                                                         Arabidopsis Cohesion Establishment Factor <br />  <br />  <br />  <br />  <br />          Figure 4. Female gametophyte seed development wild-type ctf71/2 siliques. The images represent single 1.5-mm <br />          optical section unless noted. A D, Normal ovules. E L, ctf7 ovules. A, Wild-type female gametophyte stage FG7 <br />          containing synergid cells (SN), egg cell (E), central cell (CC). A projection 1.5-mm optical sections shown. <br />          B, Female gametophyte stage FG8 degenerated synergid cell (DS). A projection 1.5-mm optical sections shown. <br />          C, Fertilized developing seed contains elongated zygote (ZN) endosperm nuclei (EN). A projection 1.5- <br />          mm optical sections shown. D, Developing seed zygote formed suspensor (S) terminal (T) cells. E, First <br />          optical section ovule undegenerated antipodal cells. A degenerated synergid cell, persistent synergid cell, and <br />          egg cell micropylar pole. One antipodal cells (AN) the embryo sac. F, Second optical <br />          section showing antipodal cells the embryo sac. G, Third optical section central cell middle of <br />          embryo sac. H, Fertilized seed containing degenerated persistent synergid cells zygote (Z). The endosperm nucleus <br />          divided nuclei (EN). A projection 3-mm optical sections shown. I, Optical section (3 mm) the <br />          chalazal pole seed H. The nucleus cell membrane antipodal cell nuclei intact. J, ctf7 seed <br />          containing degenerated zygote (DZ) endosperm nuclei. Only endosperm nuclei section as <br />          zygote. K, ctf7 seed containing degenerated zygote 16 endosperm nuclei. Five endosperm nuclei observed. A <br />          projection 1.5-mm optical sections shown. L, Degenerated ctf7 seed containing degenerated zygote degenerated <br />          endosperm (DE) nuclei. A projection 1.5-mm optical sections shown. Bars = 10 mm. <br />  <br />  <br /> sperm nodules chalazal cyst                                completed cellularization endosperm began <br /> observed CZE (Fig. 5H). The endosperm                                signs degeneration (Fig. 5, S&#226;&#8364;&#8220;U). Com- <br /> ctf7 seeds showed similar <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> development                             plete cellularization, followed degeneration the <br /> with obvious abnormalities, em-                             endosperm, observed ctf7 seeds (Fig. 5, <br /> bryo arrested (Fig. 5, J&#226;&#8364;&#8220;L). By late globu-                        W X). <br /> lar/early heart stage, MCE completed                                     Subtle defects NCDs ctf7 seeds, <br /> cellularization PEN started cellulari-                         however, detected, beginning corre- <br /> zation process wild-type seeds (Fig. 5, M&#226;&#8364;&#8220;O). Again,                        sponding wild-type seeds approximately the <br /> cellularization MCE PEN formation                            32-cell embryo stage. In particular, small number of <br /> the chalazal cyst occurred normally ctf7 seeds (Fig.                        NCDs enlarged nucleoli, NCDs contained <br /> 5, Q R). Finally, time wild-type embryo                         multiple nucleoli, NCDs contained multiple <br /> had developed torpedo stage, PEN                                nuclei observed (less 5%; data shown). <br /> Plant Physiol. Vol. 154, 2010                                                                                                                    825 <br />  Jiang et al. <br />  <br />  <br />  <br /> Table I. Female gametophyte development ctf71/2 plants            eight-nucleate embryo sac. The polar nuclei, one <br />    Female gametophyte stages defined according Christensen <br />                                                                      pole, migrated micropylar <br /> et al. (1998).                                                       half developing female gametophyte (FG5; Fig. <br />                                                                      6E) eventually fused form central cell (FG6; <br />    Pistil         No. Female Gametophytes Total Female            Fig. 6F). The antipodal cells degenerated just <br />    No.      FG1 FG2 FG3 FG4 FG5 FG6 FG7 FG8 Gametophytes             fertilization, giving mature female gameto- <br />    1        11   17   17    6                           51           phyte consisting synergid cells, egg cell, <br />    2                  13   17   10    6                 46           central cell (FG7; Fig. 6G). <br />    3                             8   34   4    5        51              In contrast, significant number female <br />    4                                  4   3    8        15           gametophytes, predict overexpress CTF7, <br />                                                                      arrested FG1 (Fig. 6, I J). The arrested female <br />                                                                      gametophytes did exhibit obvious abnormal- <br />                                                                      ities typically failed divide form two- <br /> However, alterations generally rare did               nucleate embryo sac. Occasionally, observed an <br /> not appear significant effect endosperm               arrested female gametophyte FG2 (Fig. 6K); how- <br /> development. Therefore, inactivation CTF7                   ever, quite rare. Obvious abnormalities were <br /> blocks embryo development beginning zygote                    observed arrested FG2 female gameto- <br /> stage, little effect endosperm develop-              phytes, ultimately, arrested female ga- <br /> ment.                                                                metophytes degraded (Fig. 6L). The size embryo <br />                                                                      sac continued enlarge seeds arrested female <br /> Overexpression CTF7 Blocks Early                                  gametophytes, indicating integument growth was <br /> Ovule Development                                                    normal (Fig. 6, I&#226;&#8364;&#8220;L). <br />                                                                         The possibility overexpression CTF7 affects <br />    We tested effect overexpression                    female gametophyte development inves- <br /> Arabidopsis CTF7 plant growth. Transgenic plants                  tigated analyzing transgenic plants express <br /> that overexpress CTF7 35S promoter ap-                      CTF7-yellow fluorescent protein (YFP) 35S <br /> peared normal vegetative growth; however, 10                  promoter. Of 10 transgenic lines analyzed, six <br /> out 13 independent lines exhibited varying degrees                exhibited varying levels reduced fertility (9%&#226;&#8364;&#8220;41% <br /> reduced fertility, ranging 5% 52% aborted                    aborted seeds). Analysis lines showed YFP <br /> seeds. Southern-blot analysis lines (1 18)                expressed 35S promoter female <br /> that exhibited highest levels aborted seeds                   gametophyte ovule development that, <br /> (484:311 568:627 normal:aborted) showed                     similar 35S:CTF7 lines, female gametophytes ar- <br /> they contain single insert. A genetic analysis              rested FG1 FG2 (Fig. 7, A&#226;&#8364;&#8220;H). Therefore, <br /> of lines showed T2 generation segre-                  expression CTF7 35S promoter block <br /> gated approximately 1:1 BASTA resistance (47                     female gametophyte development beginning FG1. <br /> BASTA resistant:53 BASTA sensitive), suggesting that <br /> the 35S:CTF7 construct causes gametophytic lethality.                CTF7 Is Abundant Embryo But Present Very <br />    Further analysis 10 35S:CTF7 reduced fertility             Low Levels Endosperm <br /> lines showed produced normal levels of <br /> viable pollen contained alterations female ga-                   Our observation endosperm development not <br /> metophyte development. Detailed analyses game-                    significantly affected inactivation CTF7 sug- <br /> tophyte development lines 1 18 demonstrated                   gested CTF7 required nuclear division <br /> that pollen development unaffected (data                      endosperm. This raised question whether <br /> shown), 46% (n = 795) ovules                 CTF7 actually present endosperm. To investi- <br /> to abort early development. Specifically, laser scan-             gate question, analyzed CTF7 distribution and <br /> ning confocal microscopy analysis female gameto-                  localization patterns seeds using immunolo- <br /> phyte development lines 1 18 showed                      calization transgenic plants YFP was <br /> approximately half ovules arrested approxi-                fused C terminus CTF7. The CTF7-YFP <br /> mately FG1 (Fig. 6, I J). In ovules exhibiting               fusion protein readily detected embryo, <br /> wild-type development, functional megaspore                      signals endosperm barely detect- <br /> (FG1; Fig. 6A) underwent mitosis rise                   able (Fig. 7, I&#226;&#8364;&#8220;L). While weak fluorescence detected <br /> two-nucleate embryo sac (FG2; Fig. 6B). A central                    MCE, PEN, CZE, signals en- <br /> vacuole formed nuclei (FG3; Fig. 6C),                dosperm weak, uniform, and <br /> and nuclei underwent second division                    background levels. Immunolocaliza- <br /> give rise four-nucleate embryo sac early FG4                 tion experiments using polyclonal antibody full- <br /> (Fig. 6D). At time, division plane                  length Arabidopsis CTF7 revealed strong CTF7 <br /> the chalazal nuclei parallel chalazal-                signal embryo various stages development, <br /> micropylar axis. Then, nuclei migrated               cross-reactivity endosperm typically <br /> so division plane orthogonal               similar observed negative control experi- <br /> to axis. Another round division gave rise               ments preimmune secondary antibody alone <br /> 826                                                                                                Plant Physiol. Vol. 154, 2010 <br />                                                                                                    Arabidopsis Cohesion Establishment Factor <br />  <br />  <br />                                                                            (Fig. 7, M&#226;&#8364;&#8220;P). Strong autofluorescence observed in <br />                                                                            integuments samples. Therefore, CTF7 <br />                                                                            present high levels embryo, present at <br />                                                                            low levels, all, endosperm. Finally, an <br />                                                                            analysis subcellular distribution CTF7 <br />                                                                            showed present nucleus the <br />                                                                            cytoplasm, nuclear signal significantly <br />                                                                            stronger signal detected cytoplasm. A <br />                                                                            function CTF7 outside nucleus been <br />                                                                            described organism. Therefore, significance, <br />                                                                            any, cytoplasmic CTF7 unknown. <br />  <br />  <br />                                                                            DISCUSSION <br />  <br />                                                                               Ctf7 proteins possess highly conserved do- <br />                                                                            mains, including PIP box, C2H2 zinc finger, an <br />                                                                            acetyltransferase domain, present in <br />                                                                            Arabidopsis CTF7. The PIP box required Prolif- <br />                                                                            erating Cell Nuclear Antigen binding (Moldovan et al., <br />                                                                            2006). The C2H2 zinc finger motif appears enhance <br />                                                                            substrate recognition (Onn et al., 2009). The C-terminal <br />                                                                            Ctf7 acetyltransferase domains yeast Ctf7 hu- <br />                                                                            man ESCO1 shown acetylate con- <br />                                                                            served Lys residues SMC3 (Ben-Shahar et al., 2008; <br />                                                                            Unal et al., 2008; Zhang et al., 2008). Furthermore, <br />                                                                            Eco1-dependent acetylation Smc3p S-phase <br />                                                                            Mcd1 DNA double-strand break repair <br />                                                                            stimulates cohesion formation inhibiting activ- <br />                                                                            ity Wpl1 (Ben-Shahar et al., 2008; Unal et al., 2008; <br />                                                                            Zhang et al., 2008; Rowland et al., 2009; Sutani et al., <br />                                                                            2009). It interesting determine Arabidopsis <br />                                                                            CTF7 functions similar fashion. <br />                                                                               CTF7 proteins organisms contain additional <br />                                                                            diverse N-terminal domains. In fission yeast, the <br />                                                                            N-terminal two-thirds ESO1 similar DNA <br />                                                                            polymerase h budding yeast, deletion this <br />                                                                            domain increases sensitivity UV irradiation (Tanaka <br />                                                                            et al., 2000). Human ESCO1 contains N-terminal <br />                                                                            extension similarity linker histone proteins, <br />                                                                            facilitates binding chromosomes (Hou and <br />                                                                            Zou, 2005). Arabidopsis CTF7 significantly shorter <br />                                                                            Schizosaccharomyces pombe human pro- <br />                                                                            teins similar S. cerevisiae protein it <br />                                                                            lacks extended N-terminal domain. Predicted rice <br />                                                                            (Oryza sativa) maize (Zea mays) CTF7 proteins also <br />                                                                            lack N-terminal extension, suggesting a <br /> Figure 5. Embryo endosperm development wild-type ctf71/2        common feature plant CTF7 proteins. Arabidopsis <br /> siliques. Wild-type (A&#226;&#8364;&#8220;C, G&#226;&#8364;&#8220;I, M&#226;&#8364;&#8220;O, S&#226;&#8364;&#8220;U) ctf71/2 (D&#226;&#8364;&#8220;F, J&#226;&#8364;&#8220;L, P&#226;&#8364;&#8220; <br />                                                                            CTF7 able complement S. cerevisiae ctf7 temper- <br /> R, V&#226;&#8364;&#8220;X) embryo endosperm whole-mount images shown. <br /> A, Wild-type octant-stage embryo. The MCE NCD stage. B, <br /> CZE embryo sac A. C, PEN embryo sac A. D, ctf7 <br /> embryo showing aberrant cell division. The arrow indicates division    indicates abnormal cell shape cell organization. Q, CZE the <br /> plane. E, CZE embryo sac D. F, PEN embryo sac D. G,    embryo sac P. R, PEN embryo sac P. The arrowhead indicates <br /> Wild-type globular embryo. The MCE shows signs cellularization          NCDs starting cellularize. S, Wild-type torpedo-stage embryo. T, <br /> (arrowhead). H, CZE embryo sac G. Endosperm nodules          CZE embryo sac S. NCDs present. U, PEN the <br /> chalazal cyst visible. I, PEN embryo sac G. J, ctf7 embryo   embryo sac S. The arrowhead shows cells completed <br /> showing aberrant cell division. The arrow indicates division plane.    cellularization. V, ctf7 embryo abnormal shape disorga- <br /> K, CZE embryo sac J. L, PEN embryo sac J. M, Wild-     nized cell order. The arrow indicates abnormal cell organization. W, <br /> type early-heart-stage embryo. N, CZE embryo sac M. O, PEN       CZE embryo sac V. X, PEN embryo sac V. The <br /> of embryo sac M. The arrowhead indicates NCDs starting           arrowhead shows cells completed cellularization. Bars = <br /> cellularize. P, ctf7 embryo showing aberrant cell division. The arrow      10 mm. <br />  <br /> Plant Physiol. Vol. 154, 2010                                                                                                                 827 <br />  Jiang et al. <br />  <br />  <br />  <br />  <br />           Figure 6. Female gametophyte development revealed laser scanning confocal microscopy wild-type CTF7 <br />           overexpression plants. A H, Wild-type female gametophyte development. I L, Female gametophyte development in <br />           CTF7 overexpression lines. A, Female gametophyte stage 1 (FG1) ovule showing functional megaspore (M). A trace the <br />           degraded megaspore (DM) visible. Ch, Chalaza. B, FG2 ovule two-nucleate (N) embryo sac. The degraded <br />           megaspore visible cases. C, FG3 ovule showing late two-nucleate embryo sac enlarged central vacuole <br />           (V) small chalazal vacuole. D, Ovule four-nucleate embryo sac FG4. E, Ovule embryo sac FG5. <br />           Cellularization cell differentiation complete formation synergid nuclei (SN), egg nucleus (EN), three <br />           antipodal nuclei (AN), prominent polar nuclei (PN), fused. F, Ovule mature seven-celled <br />           embryo sac FG6. The polar nuclei fused form diploid central nucleus (CN). G, Ovule FG7 antipodal <br />           cells begun degenerate. H, Ovule fertilization. One synergid cell degraded endosperm completed <br />           rounds nuclear division. I, 35S-CTF7 ovule. No difference wild-type FG1 ovules apparent. The degraded <br />           megaspore visible. J, 35S-CTF7 ovule. The embryo sac enlarged female gametophyte remains FG1. K, 35S-CTF7 <br />           ovule. The female gametophyte completed round mitosis. L, 35S-CTF7 ovule. The female gametophyte degraded. <br />           Bars = 5 mm. <br />  <br />  <br /> ature-sensitive mutant, human protein                       pollen development appeared normal transgenic <br /> (Bellows et al., 2003). Taken together, results                        plants overexpressing CTF7. Therefore, changes in <br /> suggest CTF7 evolved additional dif-                           CTF7 levels mutation overexpression do <br /> ferent roles function somewhat differently                         dramatic effect pollen development or <br /> different organisms.                                                         function. <br />                                                                                 Inactivation CTF7 did did a <br /> Proper Levels CTF7 Are Essential Mitosis                       dramatic effect female gametophyte development, <br /> Female Gametophyte                                                           again, subtle alterations observed. Spe- <br />                                                                              cifically, inactivation CTF7 slowed progression <br />   Self-pollinated ctf71/2 plants consistently produce                        mitosis overall development female game- <br /> a 1.2:1 ratio heterozygous:wild-type progeny, in-                         tophytes (Table I); addition, antipodal cells <br /> dicating mutation effect gameto-                          did degenerate normally ctf7 embryo sacs (Fig. <br /> phyte development. Transmission pollen                           4, E, F, I). However, alterations appear <br /> reduced slightly ctf71/2 plants, pollen                          directly affect function embryo sac or <br /> exhibiting reduced germination frequencies. However,                         fertilization, mutation transmitted normally <br /> no dramatic alterations pollen development                           female gametophytes (Table I). These results <br /> observed generative nucleus sperm                            consistent number reports showing that <br /> nuclei present appeared normal. Likewise,                           inactivation essential genes, including in- <br /> 828                                                                                                                Plant Physiol. Vol. 154, 2010 <br />                                                                                                     Arabidopsis Cohesion Establishment Factor <br />  <br />  <br />  <br />  <br />          Figure 7. Cellular distribution CTF7 wild-type 35S-CTF7::YFP plants. A D, 35S-driven CTF7::YFP localization during <br />          female gametophyte development. A, FG1 ovule. The arrow indicates female gametophyte nucleus. B, Bright-field image of <br />          A. The position female gametophyte circled. C, FG2 ovule. The arrows indicate nuclei. D, FG7 ovule. The arrows <br />          indicate nuclei. E H, Aborted female gametophytes 35S-CTF7::YFP plants. E, Aborted FG1 ovule. The arrow indicates <br />          degraded nucleus. F, Bright-field image E. G, Aborted ovule early FG1. The arrow indicates degraded nucleus. H, Aborted <br />          ovule. The arrow indicates remnants degraded nucleus. I, Seed globular-stage embryo genomic CTF7::YFP plant. J, <br />          Seed heart-stage embryo genomic CTF7::YFP transgenic plant. K, Seed torpedo-stage embryo genomic <br />          CTF7::YFP transgenic plant. L, Seed wild-type, negative control plant. M P, CTF7 immunolocalization whole-mount <br />          cleared seeds. M, Torpedo-stage embryo. N, Single embryo early torpedo stage. O, Cotyledon-stage embryo. P, Negative <br />          control primary antibody. Arrows I K, M, P denote position embryo. Bars = 5 mm. <br />  <br />  <br /> volved formation release cohesion, does                        number aborted ovules silique ranging <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> not block gametophyte development cause                          5% 52%. We believe variability likely due <br /> alterations embryo development (Liu Meinke,                          relatively weak variable expression the <br /> 1998; Lam et al., 2005; Liu Makaroff, 2006). It                     35S promoter female gametophytes different <br /> been suggested gametophytes pool                         transgenic lines. Our results showing 35S <br /> mRNA protein utilized successful                     promoter active female gametophytes 35S- <br /> completion limited number nuclear divisions                       CTF7-YFP plants (Fig. 7, A&#226;&#8364;&#8220;H) support hypothesis <br /> required gametophyte development (Liu                               consistent report showing that <br /> Meinke, 1998). Evidence support hypothesis                       35S promoter active female gametophyte <br /> comes observation transgenic plants                           development (Liu et al., 2008). An alternative possi- <br /> expressing separase <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>RNA interference</span> construct                            bility, observed female gametophyte arrest <br /> from meiotic DMC1 promoter exhibit defects                              results chromosomal rearrangements associated <br /> much earlier T-DNA insertion lines (Liu                            T-DNA insertion (Ray et al., 1997; Curtis et al., <br /> Makaroff, 2006).                                                            2009), likely given absence pollen defects, <br />    In contrast, expression CTF7 CTF7:YFP                        results expression studies, multiple <br /> the 35S promoter significantly impacted female game-                        independent occurrences <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> the <br /> tophyte development, typically resulting arrest                       transgenic lines. <br /> FG1 FG2 (Fig. 6, I&#226;&#8364;&#8220;K). The arrested female gameto-                          At time, clear overexpression of <br /> phytes did exhibit obvious abnormalities                        CTF7 affects female gametophyte development. How- <br /> generally failed divide form two-nucleate                           ever, recently shown acetylation <br /> embryo sac. The observed effect variable,                          state SMC3 affects rate clamp-loader-depen- <br /> Plant Physiol. Vol. 154, 2010                                                                                                              829 <br />  Jiang et al. <br />  <br />  <br /> dent fork progression human cells (Terret et al.,           continued endosperm nuclear division ob- <br /> 2009) overexpression Ctf7 yeast bypasses        serve results different cell cycle checkpoints <br /> the requirement double-strand breaks generate           operating embryos endosperm. It possible <br /> cohesion G2/M (Heidinger-Pauli et al., 2009).           endosperm nuclei lack cell cycle check- <br /> Therefore, possible overexpression CFT7          points altered cell cycle checkpoints that <br /> may shift balance acetylated cohesin com-           tolerate missegregation chromosomes. Consis- <br /> plex, resulting slowdown blockage               tent hypothesis embryo-lethal <br /> replication fork. It possible noncohesin        mutants arrest one- two-cell zygote stage <br /> protein aberrantly acetylated, causing gametophyte          endosperm continues divide. However, <br /> arrest. Our observation CTF7 overexpression          mutants characterized date appear <br /> affects female gametophyte suggests female            exhibit normal endosperm development. For exam- <br /> gametophytes sensitive perturba-             ple, 70 MEE mutants arrest early in <br /> tions male gametophytes overexpression            embryo development, exhibit <br /> of CTF7 transgenic plants higher          normal endosperm development (Pagnussat et al., <br /> female gametophytes.                                           2005). Therefore, observation inactivation of <br />                                                                CTF7 blocks embryo development soon fertiliza- <br /> CTF7 Is Essential Mitosis Embryo But Not         tion, apparent effect endosperm devel- <br /> Pollen Endosperm                                        opment, quite unusual. <br />  <br />    CTF7 essential nuclear division embryo, <br /> with ctf7 embryos arresting soon fertiliza- <br />                                                                MATERIALS AND METHODS <br /> tion prior division (Fig. 5, J K). In <br /> contrast, syncytial nuclear divisions fertil-       Plant Material Growth Conditions <br /> ized central cell proceeded normally ctf7 seeds. <br />                                                                   Arabidopsis (Arabidopsis thaliana) plants grown Metro-Mix200 soil <br /> Furthermore, cellularization endosperm              (Scotts-Sierra Horticultural Products; http://www.scotts.com) germi- <br /> further cell division appeared similar situation        nation plates (Murashige Skoog; Caisson Laboratories; www.caissonlabs. <br /> in wild-type endosperm (Brown et al., 1999; Sorensen           peachhost.com) growth chamber 22&#194;&#176;C 16-h-light/8-h-dark cycle. <br /> et al., 2002; Ingouff et al., 2005). Therefore,      Arabidopsis ecotype Wassilewskija used transcript analysis plant <br />                                                                transformation studies. Crossing studies conducted Columbia <br /> zygote able undergo nuclear division          ecotype. The SALK_059500 SAIL_1214G06 T-DNA insertion lines were <br /> absence CTF7, multiple rounds endosperm nu-              obtained Arabidopsis Biological Resource Center. Seedlings and <br /> clear division occur relatively normal          roots collected 7-d-old plants grown plates. Leaves were <br /> absence CTF7.                                               harvested rosette-stage plants grown soil. Approximately 20 d after <br />                                                                germination, buds stems collected. All samples harvested, <br />    Our localization studies CTF7 is <br />                                                                frozen liquid N2, stored &#226;&#8364;&#8220;80&#194;&#176;C needed. <br /> present high levels embryo, absent/barely <br /> detectable endosperm (Fig. 7). These results, <br /> along phenotypic studies, sug-             Molecular Analysis CTF7 <br /> gest CTF7 required establishment            The Arabidopsis CTF7 (At4g31400) genomic DNA sequence identified <br /> sister chromatid cohesion endosperm. In S.              BLAST searching Arabidopsis Genome Initiative Proteins Database <br /> pombe, Eso1 dispensable Pds5 deleted             (http://www.arabidopsis.org/blast/) Saccharomyces cerevisiae Eco1, <br />                                                                Schizosaccharomyces pombe Eso1, human ECO1 proteins. <br /> (Tanaka et al., 2001), S. cerevisiae, lethality       A CTF7 cDNA obtained total RNA isolated using Trizol <br /> of Eco1 deletion suppressed inactivation           (Invitrogen) performing reverse transcription primer 695 followed <br /> Rad61/Wpl1 (Ben-Shahar et al., 2008). Furthermore, co-         PCR using primers 693 695 (Fig. 1). CTF7 transcript levels buds, <br /> hesion defects human cells depleted Esco1            leaves, shoots, stems, roots, seedlings wild-type plants analyzed <br />                                                                using RT-PCR. RT conducted total RNA (4 mg) using Thermoscript <br /> be rescued codepletion Wapl1 (Gandhi et al., <br />                                                                RT-PCR (Invitrogen) oligo(dT) primer followed 25 cycles of <br /> 2006). Therefore, possible endosperm            PCR primer pair 741/696 (Fig. 1). Amplification products analyzed <br /> lacks CTF7, WAPL1, PDS5 instead uses                 Southern blotting using [32P]dATP-labeled 741/696 PCR fragment. After <br /> different mechanism establish sister chromatid co-          hybridization washing, radioactivity detected using Molecular <br /> hesion.                                                        Dynamics PhosphorImager. RT-PCR conducted using primers ACTIN8 <br />                                                                standardize cDNA (An et al., 1996). <br />    The possibility exists CTF7-like protein            Genomic DNA T-DNA insertion plants genotyped PCR with <br /> may substitute CTF7 endosperm. Several              primer pairs specific T-DNA wild-type loci. Southern-blot analysis <br /> predicted proteins (At3g20350, At5g26190,                  ctf71/2 plants confirmed T-DNA insert line. <br /> At2g44140) exhibit low-level similarity yeast, hu-          A 4,275-bp genomic DNA fragment containing CTF7 coding region and <br />                                                                2,422 bp 5# flanking sequence pFGC5941 used complement ctf7-1 <br /> man, Arabidopsis CTF7. However,                plants using floral dip transformation method (Clough Bent, 1998). <br /> proteins contain elements conserved CTF7         Transgenic plants identified BASTA resistance PCR genotyped <br /> proteins (e.g. PIP box, C2H2 zinc finger, acetyl-          using primer pairs transgene, genomic CTF7 locus, T-DNA <br /> transferase domain), phylogenetic analyses              insert. <br /> the Arabidopsis CTF7, At3g20350, At5g26190,                    Two CTF7-YFP fusion protein constructs generated. The was <br />                                                                generated cloning 2,082-bp coding sequence fragment 35S::YFP <br /> At2g44140 proteins support conclusion Arab-           pFGC5941. In second construct, 4,275-bp genomic DNA fragment <br /> idopsis contains single CTF7 ortholog (Fig. 1D).             containing CTF7 coding region 2,422 bp 5# flanking sequence <br /> Finally, possible early embryo abortion         cloned pFGC5941 vector containing YFP. After confirmation by <br />  <br /> 830                                                                                                     Plant Physiol. Vol. 154, 2010 <br />                                                                                                                Arabidopsis Cohesion Establishment Factor <br />  <br />  <br /> DNA sequencing, clones introduced plants Agrobacterium              CTF7 Immunolocalization <br /> tumefaciens transformation screened BASTA. Positive plants were <br /> confirmed PCR analyzed confocal microscopy.                            Polyclonal antibodies raised Arabidopsis CTF7, over- <br />                                                                                      expressed E. coli affinity purified using standard procedures (Harlow <br />                                                                                      Lane, 1988). For immunolocalization experiments, Arabidopsis ovules <br /> Analysis CTF7 Acetyltransferase Activity                                          fixed overnight 4% paraformaldehyde 13 phosphate-buffered <br />                                                                                      saline (PBS) supplemented 0.1% Triton X-100. The material then <br />     The CTF7 cDNA cloned pET-24b NdeI/EcoRI fragment              squashed using cover glass, treated 1 h 2% Driselase, washed <br /> into pIADL14 NdeI/HindIII fragment generate CTF7-MBP fusion               briefly PBS PBS containing 3% IGEPAL CA-630 (Sigma) <br /> (CTF7-MBP). After DNA sequencing, pET-24b-CTF7 transformed                  plus 10% dimethyl sulfoxide 1 h. The slides washed times <br /> BL21(DE3) pLysS Escherichia coli cells, overexpression solubility     13 PBS, blocked 3% bovine serum albumin, treated with <br /> the protein determined. The solubility CTF7-MBP improved              primary antibody overnight 37&#194;&#176;C. The slides washed times with <br /> introducing pGroESL cell line (Goloubinoff et al., 1989). CTF7-MBP          13 PBS treated secondary antibody 3 h 37&#194;&#176;C. After washing, the <br /> was purified maltose-binding resin cell extracts cells       slides stained DAPI 30 min, washed briefly, treated with <br /> had grown overnight 30&#194;&#176;C. Fractions containing recombinant CTF7-             antifade, sealed cover glass. The slides analyzed using <br /> MBP analyzed SDS-PAGE confirmed matrix-assisted laser-                confocal microscopy described above. <br /> desorption ionization time flight-mass spectrometry. CTF7-MBP was <br /> assayed acetyltransferase activity described previously (Bellows et al.,          Sequence data article GenBank/EMBL data <br /> 2003).                                                                               libraries accession number EU077499. <br />  <br />  <br /> Complementation Yeast ctf7-203 Cells <br />    Yeast expression constructs prepared transferring CTF7 <br />                                                                                      ACKNOWLEDGMENTS <br /> protein-coding region pAS2 (Clontech Laboratories; http://www.                      We grateful Richard Edelmann Matthew L. Duley technical <br /> clontech.com). CTF7-pAS2 pAS2 transferred tempera-                 support microscopy, Robert Skibbens providing yeast strains <br /> ture-sensitive ctf7-203 line, YBS514, using lithium acetate transformation           YBS255 ctf7-203 (YBS514), Rich Jorgenson providing pFGC5941 <br /> (Skibbens et al., 1999). The ability constructs complement ctf7-203    plasmid, Michael Crowder providing MBP plasmid, John Hawes <br /> mutation tested growing YBS255 (wild-type) YBS514 cells contain-          providing pGroESL plasmid. We thank Meghan Holdorf Lara <br /> ing CTF7-pAS2 pAS2 appropriate dropout medium. After growth                Strittmatter helpful discussions comments manuscript. <br /> at 25&#194;&#176;C 48 h, 10 mL serial dilution YBS255, ctf7-203/YBS514, <br /> YBS514-PAS2, YBS514-CTF7-PAS2 plated selection plates             Received April 12, 2010; accepted July 28, 2010; published July 29, 2010. <br /> grown 25&#194;&#176;C, 28&#194;&#176;C, 33&#194;&#176;C, 38&#194;&#176;C 48 h. <br />  <br />  <br /> Morphological Analysis ctf71/2 35S-CTF7 Plants                                LITERATURE CITED <br />                                                  1/2 <br />     Ovule development analyzed ctf7            plants laser scanning       An Y, McDowell J, Huang S, McKinney E, Chambliss S, Meagher R (1996) <br /> confocal microscopy essentially described (Christensen et al., 1997). Inflo-         Strong, constitutive expression Arabidopsis ACT2/ACT8 actin <br /> rescences collected fixed 4% glutaraldehyde vacuum                    subclass vegetative tissues. Plant J 10: 107&#226;&#8364;&#8220;121 <br /> overnight, dehydrated graded ethanol series (20% steps 1 h each),       Bai X, Peirsion B, Dong F, Cai X, Makaroff C (1999) Isolation and <br /> cleared 2:1 mixture benzyl benzoate:benzyl alcohol. The pistils            characterization SYN1, RAD21-like gene essential meiosis in <br /> mounted sealed coverslips. Images collected projected               Arabidopsis. Plant Cell 11: 417&#226;&#8364;&#8220;430 <br /> Olympus Flouview 2.0 software (http://www.olympus-global.com/), ana-                 Bellows AM, Kenna MA, Cassimeris L, Skibbens RV (2003) Human <br /> lyzed Image Pro Plus (Media Cybernetics; http://www.mediacy.com),                  EFO1p exhibits acetyltransferase activity unique combination of <br /> and presented Photoshop version 7.0 (Adobe; http://www.adobe.com).                 linker histone Ctf7p/Eco1p chromatid cohesion establishment <br />     Endosperm development analyzed siliques individual                 domains. Nucleic Acids Res 31: 6334&#226;&#8364;&#8220;6343 <br /> ctf71/2 plants using laser scanning confocal microscopy essentially de-           Ben-Shahar TR, Heeger S, Lehane C, East P, Flynn H, Skehel M, Uhlmann <br /> scribed (Braselton et al., 1996). Inflorescences fixed 3:1 ethanol:glacial      F (2008) Eco1-dependent cohesin acetylation establishment of <br /> acetic acid 4&#194;&#176;C 48 h, hydrolyzed 5 mol L21 HCl 1 h, followed         sister chromatid cohesion. Science 321: 563&#226;&#8364;&#8220;566 <br /> staining using Schiff&#226;&#8364;&#8482;s reagent (Sigma; www.sigmaaldrich.com) 24 h. After        Bhatt AM, Lister C, Page T, Fransz P, Findlay K, Jones GH, Dickinson HG, <br /> dehydration serial ethanol series, individual seeds dissected              Dean C (1999) The DIF1 gene Arabidopsis required meiotic <br /> methyl salicylate viewed using Olympus IX-81 fluorescence deconvo-               chromosome segregation belongs REC8/RAD21 cohesin gene <br /> lution microscope system. Whole-mount clearing used determine                family. Plant J 19: 463&#226;&#8364;&#8220;472 <br /> embryo endosperm phenotypes. Siliques ctf7 heterozygous plants              Braselton JP, Wilkinson MJ, Clulow SA (1996) Feulgen staining intact <br /> were dissected cleared Herr&#226;&#8364;&#8482;s solution containing lactic acid:chloral            plant tissues confocal microscopy. Biotech Histochem 71: 84&#226;&#8364;&#8220;87 <br /> hydrate:phenol:clove oil:xylene (2:2:2:2:1, w/w). Embryo endosperm               Brown RC, Lemmon BE, Nguyen H, Olsen OA (1999) Development of <br /> development studied microscopically Nikon microscope equipped                endosperm Arabidopsis thaliana. Sex Plant Reprod 12: 32&#226;&#8364;&#8220;42 <br /> with differential interference contrast optics.                                      Chelysheva L, Diallo S, Vezon D, Gendrot G, Vrielynck N, Belcram K, <br />     Pollen morphology, viability, germination compared newly                Rocques N, Marquez-Lema A, Bhatt AM, Horlow C, et al (2005) <br /> opened flowers ctf71/2 plants wild-type siblings pot.              AtREC8 AtSCC3 essential monopolar orientation the <br /> Pollen morphology analyzed using SEM. Freshly opened flowers                   kinetochores meiosis. J Cell Sci 118: 4621&#226;&#8364;&#8220;4632 <br /> fixed formalin-acetic acid-alcohol overnight, rinsed times fixation      Christensen CA, King EJ, Jordan JR, Drews GN (1997) Megagameto- <br /> buffer, dehydrated graded ethanol series. Once 85% ethanol,             genesis Arabidopsis wild type Gf mutant. Sex Plant Reprod <br /> anthers dissected, dehydrated 100% ethanol, critical point           10: 49&#226;&#8364;&#8220;64 <br /> dried CO2, mounted SEM stubs carbon tabs, sputter coated           Christensen CA, Subramanian S, Drews GN (1998) Identification of <br /> 21-nm gold, viewed photographed using Supra 35 FEG-VP                         gametophytic mutations affecting female gametophyte development <br /> scanning electron microscope (Carl Zeiss; www.zeiss.com).                               Arabidopsis. Dev Biol 202: 136&#226;&#8364;&#8220;151 <br />     Pollen viability analyzed soaking grains 2,3,5-triphenyl           Ciosk R, Shirayama M, Shevchenko A, Tanaka T, Toth A, Nasmyth K <br /> tetrazolium chloride (1.0% weight 50% Suc) 37&#194;&#176;C 30 min followed            (2000) Cohesin&#226;&#8364;&#8482;s binding chromosomes depends separate com- <br /> by observation fluorescence microscopy (Huang et al., 2004). Pollen mitosis          plex consisting Scc2 Scc4 proteins. Mol Cell 5: 243&#226;&#8364;&#8220;254 <br /> was analyzed 4#,6-diamidino-2-phenylindole (DAPI) staining (2 mg L21)             Clough SJ, Bent AF (1998) Floral dip: simplified method Agro- <br /> followed observation fluorescence microscopy. Pollen germination              bacterium-mediated transformation Arabidopsis thaliana. Plant J 16: <br /> tested according Fan et al. (2001).                                                  735&#226;&#8364;&#8220;743 <br />  <br /> Plant Physiol. Vol. 154, 2010                                                                                                                                    831 <br />  Jiang et al. <br />  <br />  <br /> Curtis MJ, Belcram K, Bollmann SR, Tominey CM, Hoffman PD, Mercier             Nasmyth K, Haering CH (2009) Cohesin: roles mechanisms. Annu <br />    R, Hays JB (2009) Reciprocal chromosome translocation associated          Rev Genet 43: 525&#226;&#8364;&#8220;558 <br />    T-DNA-<span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>insertion mutation</span> Arabidopsis: genetic cytological            Olsen OA (2004) Nuclear endosperm development cereals Arabi- <br />    analyses consequences gametophyte development con-              dopsis thaliana. Plant Cell 16: 214&#226;&#8364;&#8220;227 <br />    struction doubly mutant lines. Planta 229: 731&#226;&#8364;&#8220;745                       Onn I, Guacci V, Koshland DE (2009) The zinc finger Eco1 enhances its <br /> da Costa-Nunes JA, Bhatt AM, O&#226;&#8364;&#8482;Shea S, West CE, Bray CM, Grossniklaus             acetyltransferase activity sister chromatid cohesion. Nucleic <br />    U, Dickinson HG (2006) Characterization Arabidopsis               Acids Res 37: 6126&#226;&#8364;&#8220;6134 <br />    thaliana RAD21 cohesins reveals differential responses ionizing radia-   Pagnussat GC, Yu HJ, Ngo QA, Rajani S, Mayalagu S, Johnson CS, <br />    tion. J Exp Bot 57: 971&#226;&#8364;&#8220;983                                                    Capron A, Xie LF, Ye D, Sundaresan V (2005) Genetic molecular <br /> Dong F, Cai X, Makaroff C (2001) Cloning characterization              identification genes required female gametophyte development <br />    Arabidopsis genes belong RAD21/REC8 family chromo-              function Arabidopsis. Development 132: 603&#226;&#8364;&#8220;614 <br />    cohesin proteins. Gene 271: 99&#226;&#8364;&#8220;108                                     Panizza S, Tanaka TU, Hochwagen A, Eisenhaber F, Nasmyth K (2000) <br /> Fan L, Wang Y, Wang H, Wu W (2001) In vitro Arabidopsis pollen                    Pds5 cooperates cohesin maintaining sister chromatid cohesion. <br />    germination characterization inward potassium currents           Curr Biol 10: 1557&#226;&#8364;&#8220;1564 <br />    Arabidopsis pollen grain protoplasts. J Exp Bot 52: 1603&#226;&#8364;&#8220;1614               Ray SM, Park SS, Ray A (1997) Pollen tube guidance female <br /> Gandhi R, Gillespie PJ, Hirano T (2006) Human Wapl cohesin-binding           gametophyte. Development 124: 2489&#226;&#8364;&#8220;2498 <br />    protein promotes sister-chromatid resolution mitotic prophase.      Rowland BD, Roig MB, Nishino T, Kurze A, Uluocak P, Mishra A, <br />    Curr Biol 16: 2406&#226;&#8364;&#8220;2417                                                        Beckouet F, Underwood P, Metson J, Imre R, et al (2009) Building sister <br /> Gillespie PJ, Hirano T (2004) Scc2 couples replication licensing sister        chromatid cohesion: Smc3 acetylation counteracts antiestablishment <br />    chromatid Xenopus egg extracts. Curr Biol 14: 1598&#226;&#8364;&#8220;1603                     activity. Mol Cell 33: 763&#226;&#8364;&#8220;774 <br /> Goloubinoff P, Gatenby AA, Lorimer GH (1989) GroE heat-shock proteins          Sanders PM, Bui AQ, Weterings K, McIntire KN, Hsu YC, Lee PY, Truong <br />    promote assembly foreign prokaryotic ribulose bisphosphate carbox-          MT, Beals TP, Goldberg RB (1999) Anther developmental defects in <br />    ylase oligomers Escherichia coli. Nature 337: 44&#226;&#8364;&#8220;47                         Arabidopsis thaliana male-sterile mutants. Sex Plant Reprod 11: 297&#226;&#8364;&#8220;322 <br /> Gruber S, Arumugam P, Katou Y, Kuglitsch D, Helmhart W, Shirahige K,           Shi DQ, Liu J, Xiang YH, Ye D, Sundaresan V, Yang WC (2005) SLOW <br />    Nasmyth K (2006) Evidence loading cohesin chromosomes             WALKER1, essential gametogenesis Arabidopsis, encodes WD40 <br />    involves opening SMC hinge. Cell 127: 523&#226;&#8364;&#8220;537                           protein involved 18S ribosomal RNA biogenesis. Plant Cell 17: <br /> Harlow E, Lane D (1988) Antibodies: A Laboratory Manual. Cold Spring              2340&#226;&#8364;&#8220;2354 <br />    Harbor Laboratory Press, Cold Spring Harbor, NY                             Skibbens R, Corson L, Koshland D, Hieter P (1999) Ctf7p essential for <br /> Heidinger-Pauli JM, Unal E, Koshland D (2009) Distinct targets             sister chromatid cohesion links mitotic chromosome structure the <br />    Eco1 acetyltransferase modulate cohesion S phase response         DNA replication machinery. Genes Dev 13: 307&#226;&#8364;&#8220;319 <br />    DNA damage. Mol Cell 34: 311&#226;&#8364;&#8220;321                                            Sorensen MB, Mayer U, Lukowitz W, Robert H, Chambrier P, Jurgens G, <br /> Hirano M, Hirano T (2006) Opening closed arms: long-distance activation           Somerville C, Lepiniec L, Berger F (2002) Cellularisation endo- <br />    SMC ATPase hinge-DNA interactions. Mol Cell 21: 175&#226;&#8364;&#8220;186                  sperm Arabidopsis thaliana coupled mitosis shares multiple <br /> Hou FJ, Zou H (2005) Two human orthologues Eco1/Ctf7 acetyltrans-              components cytokinesis. Development 129: 5567&#226;&#8364;&#8220;5576 <br />    ferases required proper sister-chromatid cohesion. Mol Biol    Sutani T, Kawaguchi T, Kanno R, Itoh T, Shirahige K (2009) Budding yeast <br />    Cell 16: 3908&#226;&#8364;&#8220;3918                                                             Wpl1 (Rad61)-Pds5 complex counteracts sister chromatid cohesion- <br /> Huang Z, Zhu J, Mu X, Lin J (2004) Pollen dispersion, pollen viability        establishing reaction. Curr Biol 19: 492&#226;&#8364;&#8220;497 <br />    pistil receptivity Lenmus chinensis. Ann Bot (Lond) 93: 295&#226;&#8364;&#8220;301          Tanaka K, Hao ZL, Kai M, Okayama H (2001) Establishment main- <br /> Ingouff M, Haseloff J, Berger F (2005) Polycomb group genes control               tenance sister chromatid cohesion fission yeast unique <br />    developmental timing endosperm. Plant J 42: 663&#226;&#8364;&#8220;674                         mechanism. EMBO J 20: 5779&#226;&#8364;&#8220;5790 <br /> Ivanov D, Schleiffer A, Eisenhaber F, Mechtler K, Haering CH, Nasmyth          Ta <br /> </body></html>