<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Escherichia coli-Salmonella typhimurium Hybrid nusA Genes: <br />              Identification of a Short Motif Required for Action of <br />                  the N Transcription Antitermination Protein <br />  <br />  <br />              The Escherichia coli nusA gene, nusA,E, encodes an essential protein that influences transcription elongation. <br />           Derivatives of E. coli in which the SalmoneUla typhimurium nusA gene, nusAs, has replaced nusA$4 are viable. <br />           Thus, NusAst can substitute for NusAEz in supporting essential bacterial activities. However, hybrid E. coli <br />           strains with the nus4st substitution do not effectively support transcription antitermination mediated by the N <br />           gene product of phage X. We report the DNA sequence of nusAst, showing that the derived amino acid sequence <br />           is 95% identical <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> the derived amino acid sequence of nus4,c. The alignment of the amino acid sequences <br />           reveals scattered single amino acid differences and one region of significant heterogeneity. In this region, called <br />           449, NusAF, has four amino acids and NusAst has nine amino acids. Functional studies of hybrid ns&amp;4 genes, <br />           constructed <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> nusAEc and nus4st, show that the 449 region of the NusAEc protein is important for X <br />           N-mediated transcription antitermination. A hybrid that has a substitution of the four E. coli codons for the <br />           nine S. typhimurium codons, but is otherwise nusAs, supports the action of the N antitermination protein. The <br />           449 region and, presumably, adjacent sequences appear to compose a functional domain of NusAE important <br />           for the action of the N transcription antitermination protein of phage X. <br />  <br />    The nusA gene of Escherichia coli encodes an essential                      terminators and proceeds into the Q gene. The Q gene <br /> 54,400-Mr protein that functions in transcription elongation                   product, which regulates late gene expression, is also an <br /> (reviewed in references 20, 29, and 83) and termination (16, 37,               antitermination factor that requires NusA (67). <br /> 49, 75, 81). Consistent with these roles, NusA has been shown                     Isolation of nonsense (80) as well as thermo- and cold- <br /> to influence pausing of RNA polymerase (16, 17, 47, 50, 75, 76,                sensitive (63, 73) mutations in nusA demonstrates that NusA <br /> 85). The demonstration of an association between NusA and                      provides an essential bacterial function. Although the essential <br /> RNA polymerase (31, 35, 43, 46, 74) suggests that NusA may                     role(s) for NusA in cellular growth has not been identified, <br /> influence transcription by a direct interaction with polymerase.               mutations in nusA reduce the effectiveness of some transcrip- <br />    NusA was identified through its role in the regulated expres-               tion terminators (62, 81). <br /> sion of phage X genes (reviewed in references 13, 20, 23, and                     In addition to RNA polymerase and N, NusA also has been <br /> 67). Early A transcription initiating at promoters PL and PR                   shown to bind the Rho termination protein (74) and RNA <br /> partially terminates at terminators tLl (15, 70) and tRl (10),                 (79). The facts that it binds a number of factors and is a <br /> respectively. The escaping transcripts in the latter case termi-               relatively large protein (36, 44, 68) suggest the possibility that <br /> nate completely at a collection of terminators in the nin region               NusA has multiple functional domains. There must be func- <br /> (8, 9, 11, 48, 52) (Fig. 1). The N gene product, encoded in the                tionally important regions of NusA located in the 5&apos; two-thirds <br /> PL operon, acts with host Nus proteins to modify RNA                           of the protein, since a plasmid-based nusA gene with its <br /> polymerase into a termination-resistant form at nut sites (1, 14,              3&apos;-terminal one-third deleted (nusAA324) complements nusA <br /> 32, 53-55, 64, 69). Mutations affecting N action have identified               mutations which fail to support either X N action or bacterial <br /> the E. coli nus genes (reviewed in references 20, 22, 26, and                  growth at low temperatures (73). However, the plasmid-based <br /> 67). For example, the nusAl <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> (19), a missense muta-                    nusAA324 gene fails to complement the nusAII(Ts) mutation. <br /> tion resulting in a single <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span> change (68), causes a failure in               Thus, the product of the nusAA&amp;324 gene is unable to support <br /> the support of N-mediated antitermination (14, 24, 42, 82). N                  bacterial growth at high temperatures (61). <br /> protein binds to NusA (34), and in vitro transcription studies                    Experiments with a hybrid bacterium, whose genome is <br /> have confirmed that Nus proteins are required for N action <br /> (14, 42, 54, 82). When properly modified by N and Nus                          derived primarily from E. coli, provide additional evidence <br /> proteins, RNA polymerase is resistant to many termination                      supporting a role for NusA in X N action. In this hybrid (2), a <br /> signals and thus proceeds through downstream terminators. In                   small exchange of DNA has resulted in the replacement of <br /> the case of PR, N-modified transcription passes through the nin                nusAE,, the nusA gene of E. coli, with nusAst, the nusA gene <br />                                                                                from the closely related enteric bacterium Salmonella typhi- <br />                                                                                murium (66, 72). In addition to the nusA gene, some additional <br />   * <br />     Corresponding author. Phone: (313) 763-3142. Fax: (313) 764- <br />                                                                                uncharacterized genetic information 5&apos; of nusA has also been <br /> 3562. Electronic mail address: David.Friedman@um.cc.umich.edu.                 replaced. The viability of the hybrid bacterium, a priori, <br />   t Present address: Roche Institute of Molecular Biology, Nutley, NJ          demonstrates that NusAst can substitute for NusAEc in sup- <br /> 07110-1199.                                                                    porting cellular processes in an E. coli genetic background. <br />   t Present address: Department of Microbiology, University of                 However, NusAst fails to support X N action (2, 21, 73). <br /> Texas, Austin, TX 78712-1095.                                                     The failure of nusAst to support X N action, under condi- <br />                                                                         1394 <br />  VOL. 176, 1994                                                                                                          HYBRID nusA GENES                  1395 <br />  <br />                                                                                                                                nin5 <br />                     att int                        dll        N                d          cro                   cllOP         ;Is         0   Q <br />                                                                                                 I          0   xs                   a   *         a <br />  <br />                                                                                                                                     a             I   4 <br />  <br />  <br />          ti                                          tLl nutL                                nutR         tRI                42 43 %4 qut tRI <br />                                                                     PL <br />                                                                          -N, nut                           I                  I <br />                                                                                                            i               I-- <br />                                                                           or Nus <br />        &apos;A~ ~~I                                                           +N, nut <br />                                                                          and Nus <br />   FIG. 1. Map of early region of the A genome. (Top line) Shown above are the locations of relevant genes as well as positions of the aut site and <br /> the nin region, and shown below are the relevant transcription signals, including promoters (p), terminators (t), and antitermination signals (nut <br /> and qut). (Middle line) Patterns of transcription from PL and PR in the absence of N. (Bottom line) Patterns of transcription from PR and PL <br /> following N modification of RNA polymerase. <br />  <br />  <br /> tions in which nusAEc does, suggested that there are structural                            EOP. Methods for determining efficiency of plating (EOP) <br /> differences between the two proteins that should be reflected                            and for the qualitative assessment of phage growth have been <br /> in differences in their primary amino acid sequences. We                                 described previously (3, 59). <br /> determined the DNA sequence of nusA, to compare its                                        Genetic techniques. (i) Transfer of the nusAhy3O2 hybrid <br /> derived amino acid sequence with that derived from the DNA                               gene from pNAX302 to the E. coli chromosome by homologous <br /> sequence of nusA,E (12, 45). This comparison, which revealed                             recombination. A rec+ nusAlO(Cs) strain, K1456, containing <br /> only small differences between the two proteins, provided the                            pNAX302 was passed through four rounds of growth in LB <br /> information enabling us to construct a series of hybrid nusA                             broth at 40&#194;&#176;C in the absence of ampicillin. Colonies that had a <br /> genes that have various amounts of genetic information from                              chromosomal nusAhy302 gene were selected by growth at 30&#194;&#176;C <br /> E. coli and S. typhimurium. On the basis of studies with these                           on LB plates in the presence of XcI - and Xc -h80; a colony <br /> hybrid genes, we have identified a region of the nusA gene that                          formed by such a recombinant would be expected to be cold <br /> is required for X N action.                                                              resistant and unable to support growth of X. Plasmid-free <br />                                                                                          bacteria were identified by screening for ampicillin sensitivity <br />                           MATERIALS AND METHODS                                          with Amp Screen (Bethesda Research Laboratories). P1 trans- <br />                                                                                          duction (77) was used to transfer the nusAhy3O2 allele from a <br />    Strains. Bacteria and phages are listed in Tables 1 and 2,                            colony with the appropriate <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> to K37, yielding strain <br /> respectively.                                                                            K4092. <br />    Media. All media have been described previously (56, 57).                                (ii) Creation of the chromosomal nusAhy3O2-449 allele. <br /> Bacterial strains with plasmids were cultivated in the presence                          Plasmid pSES28, a derivative of pUC18 with an 840-bp frag- <br /> of the appropriate antibiotic, either ampicillin at 30 ,ug/ml or                         ment composed of an internal portion of nusAs, in which the <br /> tetracycline at 15 ,ug/ml.                                                               nucleotides encoding the four amino acids of the nus4EC 449 <br />    Plasmids. Plasmid manipulations were essentially as de-                               sequence have replaced the nucleotides encoding the nine <br /> scribed by Sambrook et al. (71). Table 3 lists the plasmids used                         amino acids of the nusAst 449 sequence, was transformed into <br /> in this study as well as their sources.                                                  strain K6600. K6600 has the nusAhy3O2 gene and a defective <br />  <br />                                                                           TABLE 1. Bacterial strains <br />  Straina                                                 Relevant genotype                                Parent                        Source and/or reference <br /> K37                                    Wild type                                                                                    This laboratory (19) <br /> K95                                    nusAI                                                              K37                       This laboratory (19) <br /> K1102                                  nusAhyllO2                                                                                   L. S. Baron (21) <br /> K1456                                  argG::Tn5                                                          K37                       This laboratory (19, 73) <br /> K1914                                  argG::Tn5 nusAlO(Cs)                                               K1456                     This laboratory <br /> K2415                                  recA nusAlO(Cs)                                                    K1914                     L. Baron <br /> OR1150                                 XcI857pO-IS2-gal                                                                             S. Adhya (65) <br /> K4012                                  nusAll4(Ts)                                                        K37                       68 <br /> K4092                                  argG::Tn5 nusAhy3O2                                                K37                       This paper <br /> K6600                                  argG::Tn5 nusAhy3O2-449 XcI857pR-IS2-gal                                                     This paper <br /> K6906                                  argG::TnS nus4hy3O2-449                                            K37                       This paper <br /> K7346                                  nusA134(Am) recA::TnlO                                                                       Y. Nakamura (80) <br /> LT2                                    Wild type                                                                                    G. Jones <br />   a   All strains   are   E. coli except LT2, which is S. typhimurium. <br />  1396        CRAVEN ET AL.                                                                                                        J. BACT1E1RIOL. <br />  <br />                               TABLE 2. Phages                                   glucose plates. After 2 days, large haloed plaques could be <br />                                Relevant genotype                 Source or <br />                                                                                 observed on the thinly growing bacterial lawn. The bacteria <br />        Phage                                                                    could grow because the infecting phage carried and expressed <br />                                and/or phenotype                  reference <br />                                                                                 the argG gene of S. typhimurium. The centers of these plaques <br /> AcI857                   clts                                 NIHa collection   were picked, and the phage were purified on bacterial lawns <br /> XcI60                    cI-                                  NIH collection    formed under the conditions of the original isolation. Phage <br /> AcI857bio256sex          clts, altered N                      NIH collection    obtained from plaques identified in this second isolation were <br /> XcI -h80                 cI-, adsorption specificity          NIH collection <br />                             of 48O                                              used to make lysates. The resulting lysates were screened for <br /> XninS                    N independent                        NIH collection    nusAst by testing for complementation of both the argG::TnS <br /> M13mp8                   NAb                                  84                and nusAlO(Cs) mutations. The nusAst gene on the X1059 <br /> M13mp9                   NA                                   84                derivative was isolated from a partial Sau3AI digestion of <br /> Pi                       NA                                   NIH collection    the phage DNA that was cloned into the BamHI site of <br />   a NIH, National Institutes of Health.                                         pACYC184. A plasmid that complemented the nusAJO(Cs) <br />   b NA, not applicable.                                                         mutation was isolated; this plasmid, pNAS1000, contains a <br />                                                                                 2.9-kb insert that includes nusA5s, <br />                                                                                    DNA sequencing. The dideoxy chain termination method <br />                                                                                 described by Biggin et al. (4) was employed. Primers for <br /> XcI857 prophage fused to the bacterial gal operon so that the                   sequencing were obtained from the University of Michigan <br /> gal operon is expressed from the &apos;APR promoter. The presence                    Oligonucleotide Synthesis Facility and Bethesda Research <br /> of intervening termination signals between PR and gal as well                   Laboratories. <br /> as expression of X repressor from a cI gene with the cI857ts                       The nusA,st gene was sequenced by using a collection of <br /> allele results in expression of the gal operon being dependent                  overlapping subclones of pNAS1000 constructed by ligating <br /> upon both a functional phage N antitermination system to read                   sonication-generated DNA fragments into M13mp8. In those <br /> through these terminators and growth at a high temperature to                   regions where overlapping subclones were not obtained, the <br /> remove repression. Since the nusAhy302 allele fails to support                  sequence was determined directly from pNAS1000 by using <br /> N-mediated transcription antitermination, K6600 is Gal - at                     nusA-specific primers whose sequences were derived from the <br /> high temperatures. Gal&apos; segregants were isolated, and plating                   M13mp8 clones. <br /> of appropriate X derivatives identified those with the same                        PCR. Reactions were carried out in a thermocycler for 30 <br /> distinctive phenotype as the nusAhy449 gene on a plasmid.                       cycles. Cycle durations were 1 min at 94&#194;&#176;C, 1 min at 44&#194;&#176;C, and <br /> One derivative was chosen for further study. The putative                       1 min at 72&#194;&#176;C. The reaction sample consisted of 0.4 ,ug of the <br /> nusAhy3O2-449 allele was transduced to K37, creating strain                     DNA template (pNAS1000) and 80 pmol of the primers. <br /> K6906, and DNA sequencing of a PCR-amplified fragment                              Oligonucleotides used for priming PCRs were prepared by <br /> confirmed its structure.                                                        the University of Michigan Oligonucleotide Synthesis Facility. <br />    (iii) Isolation of the nusAst gene. A X1059-Salmonella clone                 The sequences of the primers and the nusAs, nucleotides to <br /> bank, constructed by R. Mauer and kindly provided by P.                         which they bind are as follows: SaI128, 5&apos; TCGATGTTCGT <br /> Youderian, was used to isolate the nusAs, gene according to a                   GTAGAA 3&apos; (binds to nucleotides 128 to 144 of the bottom <br /> method suggested by P. Youderian. The underlying assump-                        strand); Las971, 5&apos; ACGTTCTGACCATTACG 3&apos; (binds to <br /> tion was that the arrangement of the S. typhimurium genome in                   nucleotides 971 to 955 of the top strand); SaI483, 5&apos; GGATQT <br /> the nusA region would be the same as that found in the E. coli                  GGGCAACAACGCTGAAGCGGTGATTC[G 3&apos; (binds to <br /> genome; i.e., nusA would be adjacent to argG. A culture of E.                   nucleotides 504 to 484 of the bottom strand); and Las457, 5&apos; <br /> coli K-12 strain K1456 (K37 argG::TnS) was mixed with                           GITGCCCAGATCCAGAGAGATA1TGTCGCGGTTC 3&apos; <br /> different dilutions of the phage bank, and the infected bacteria                (binds to nucleotides 457 to 435 of the top strand). Primers <br /> were plated in 2.5 ml of top agar with M9 minimal glucose                       Sa1128 and Las971 perfectly match the nusAs, sequences. <br /> medium supplemented with 0.1 ml of LB broth on M9 minimal                       Primers Sal483 and Las457 were designed so that 13 bases at <br />                                                                                 their 5&apos; ends were complementary to each other and thus can <br />                                                                                 hybridize. Eleven (shown underlined) of the 13 bases corre- <br />                            TABLE 3. Plasmids                                    spond to the nusA,E nucleotides that replace nucleotides 458 <br />                                                                                 through 483 of the nusAst DNA sequence. <br />   Plasmid            Relevant marker(s)              Parent         Source or <br />                                                                     reference <br />                                                                                    Construction of hybrid nusA genes. The nusAhy449 gene was <br />                                                                                 created by PCR with the &quot;gene splicing by overlap extension&quot; <br /> pBR322   NA&quot;                                       NA              5            technique (41). The first two rounds of PCR used, in separate <br /> pACYC184 NA                                        NA              7            reactions, primers Sal128 and Las457 and primers Sal483 and <br /> pUC19    NA                                        NA              84           Las971 to amplify a portion of nusA st DNA, with pNAS1000 as <br /> pWR324   nusAA324                                  pBR322          73           the template. The two fragments were gel purified and com- <br /> pNAG15   nusAEc                                    pBR322          This work    bined with primers Sal128 and Las971, the primers in the third <br /> pNAG2010       nusAEc                              pBR322          This work    round of PCR. Since the two fragments overlap by 13 nucle- <br /> pNAS1000       nusAs,                              pACYC184        This work <br /> pNAS2001       nusA s                              pBR322          This work    otides, they could, at some frequency, anneal to each other. <br /> pN2/EuS        nusAA50L-nusA,s                     pBR322          This work    The annealed strands acted as templates for polymerase and <br /> pNAX2          nusAhy2                             pBR322          This work    generated the spliced recombinant DNA. The presence of <br /> pNAX3          nusAhy3                             pBR322          This work    Sal128 and Las971 allowed for amplification of the recombi- <br /> pNAX302        nusAhy3O2                           pBR322          This work    nant in subsequent cycles. The 840-bp fragment in which the 26 <br /> pSES28         840-bp fragment of nusA,s           pUC18           This work    bp of S. typhimurium DNA was replaced with 11 bp of E. coli <br />                  with 449 region of                                             DNA was digested with PvuI and used to replace the analo- <br />                  nusAEc                                                         gous fragment of the nusA st gene in pNAS1000. The resulting <br /> pNAX449        nusAhy449                           pBR322          This work    plasmid, pHSE19, was sequenced between the PvuI restriction <br />  a NA, not applicable.                                                          sites of the DNA insert to confirm that the inserted fragment <br />  VOL. 176, 1994 <br />  <br />  <br />  <br />  <br />     nusA <br />  <br />  <br />  <br />    nusAEC <br />  <br />  <br />  <br />  <br /> nusAhyllO2 <br />  <br />   nusAhy2 <br />  <br />  <br />   nusAhy3 <br />  <br />  <br />  nusAhy3O2 <br />  <br />  <br /> nusAhy3O2-449 <br />  <br />  <br />   nusAhy449 <br />  <br />   nusA A324 <br />                   v- <br />  <br />  <br />  <br />  <br />                     * ********(4/9) * * <br />  <br />  <br />  <br />  <br />                                    9 <br />  <br />  <br />  <br />                                    9 <br />  <br />  <br />                                    9 <br />  <br />  <br />  <br />  <br />                                   4li <br />                                         b&gt;&apos;, <br />                                         _S__ <br />  <br />                                         __ <br />  <br />  <br />  <br />  <br />                                         __ <br />                                              * <br />                                                  ~~~~~~~~~~-4 <br />                                                   8 <br />  <br />                                                       *** * ** <br />  <br />  <br />  <br />  <br />    FIG. 2. Structures of nusA genes. The bar at the top shows the <br />                                                                  * *** <br />  <br />  <br />  <br />  <br /> relative positions of relevant restriction sites shared by the nusAFs and <br /> nusAs, genes. The site marked 4/9 is the position of the 449 region of <br /> major heterology, and the asterisks indicate other regions where the <br /> two open reading frames encode different amino acids. Aligned below <br /> are schematic representations of hybrid genes constructed from <br /> nusAEc (solid bars) and nusAs, (hatched bars). A 4 or a 9 in the boxed <br /> area indicates the derivation of the heterologous region: a 4 indicates <br /> the four codons from E. coli, and a 9 indicates the nine codons from S. <br /> typhimurium. The four amino acids found at this position in the <br /> NusAE, protein are DLGN, and the nine amino acids in the NusAs, <br /> protein are EIKSEGMAG. The small region with parallel horizontal <br /> lines in the representation of nusAhy1lO2 indicates the region between <br /> the AvaI and BglII restriction sites in nusAst where one of the <br /> recombination events is assumed to have occurred. <br />                                                                             The hybrids constructed by this method are nusAhy2 and <br />                                                                             nusAhy3O2. <br />                                                                                                            HYBRID nusA GENES <br />  <br />  <br />  <br />                                                                                Method two, depicted in Fig. 4, relies on selection of <br />                                                                             functional hybrid nusA genes by complementation of <br />                                                                             nusAlO(Cs), a conditionally lethal mutation (73). Plasmid <br />                                                                             pN2/EuS has a tandem arrangement of the nusA4Ec and nusAs, <br />                                                                             genes. Since recognition sites for restriction enzyme PvuI are <br />                                                                             at the same positions in the two nusA genes, cleavage with this <br />                                                                             enzyme could be employed to delete the DNA between the two <br />                                                                             PvuI sites. Ligation of the remaining plasmid DNA generates <br />                                                                             a hybrid nusA gene. The nusAhy3 gene was constructed by this <br />                                                                             method. <br />                                                                                Southern and Western blot analyses. Southern blots were <br />                                                                             run essentially as outlined by Sambrook et al. (71). Western <br />                                                                             blots (immunoblots) were run essentially according to the <br />                                                                             method employed by Schauer et al. (73), except antibody was <br />                                                                             identified with the ECL kit from Amersham. <br />                                                                                Nucleotide sequence accession number. The DNA sequence <br />                                                                             of nusA,st has been deposited in GenBank and given accession <br />                                                                             number M61008. <br />  <br />  <br />  <br />  <br />                                                                             acid residues. <br />                                                                                                        RESULTS <br />                                                                                Nucleotide sequence of nusAst. The DNA sequence of <br />                                                                             nusAs, was determined as described in Materials and Methods. <br />                                                                             The nusAEc sequence used for comparison is based on the <br />                                                                             original sequencing data as well as subsequent corrections (12, <br />                                                                             44, 45, 68). The DNA sequences of the two nusA genes are <br />                                                                             extremely similar, varying at only 156 of 1,488 nucleotides <br />                                                                             (data not shown). The predicted primary sequences of the <br />                                                                             NusAs5 and NusAEc proteins are compared in Fig. 5A. As <br />                                                                             expected from the comparison of the DNA sequences, the two <br />                                                                             proteins are also very similar, varying at only 27 of 495 amino <br />                                                                                The alignment of the derived amino acid sequences of the <br />                                                                             two NusA proteins, shown in Fig. 5A, reveals one region of <br />  <br />                                                                             sequence has nine amino acids, EIKSEGMAG, while the <br />                                                                             amino acids, DLGN. This small region of NusA amino acid <br />                                                                                                                                         1397 <br />  <br />  <br />  <br />  <br />                                                                             significant heterogeneity that is located between amino acids <br />                                                                             152 and 162 of the nusA,s sequence. In this region, the NusAst <br />                                                                             corresponding site in the NusAEc sequence has four different <br />                                                                             mismatch is referred to as the 449 region to reflect the amino <br />                                                                             acid heterogeneity. A second distinguishing feature, a set of <br />                                                                             tandem repeats at the carboxy ends of both NusA proteins, was <br />                                                                             revealed by a dot plot comparison of the two amino acid <br />                                                                             sequences. The locations of the repeats are shown in Fig. 5A, <br />                                                                             and the alignment of the repeats is shown in Fig. SB. We have <br />                                                                             not identified any functional role for this repeat. However, a <br /> had the desired deletion-substitution and that no additional                repeat of an amino acid sequence has been implicated in <br /> changes were introduced by the PCR and cloning steps.                       TATA recognition by the eukaryotic transcription factor <br />    All other hybrid genes were constructed by one of two in                 TFIID (40, 60). <br /> vitro methods outlined below; both methods exploit restriction                 To determine whether amino acid sequence differences <br /> enzyme recognition sequences that occupy identical locations                relate to functional differences of the two NusA proteins, we <br /> in the reading frames of nusAEc and nusAst. The structures of               created a series of hybrid genes from nusAEc and nusA,s that <br /> the hybrid genes are shown in Fig. 2 and described below (the               express hybrid NusA proteins. The constructs used in this study <br /> plasmids containing these genes are described in Table 3).                  have sufficient genetic material of the nusA genes to include all <br />    Method one, depicted in Fig. 3, permits isolation of recom-              of the known promoters of the nusAEc operon or those <br /> binants without employing selective pressure for a functional               identified by sequence analysis for the nusA,s operon. Com- <br /> nusA gene. The recipient plasmid pNAS2001 has a tetR gene                   parison of the DNA sequence we obtained for the region <br /> adjacent to the cloned nusA gene. A portion of this nusA gene               upstream of nusA4s with that obtained for the analogous region <br /> is resected and replaced with a fragment from the donor                     upstream of nusAEc reveals a high degree of homology; the S. <br /> plasmid that contains the analogous portion of the heterolo-                typhimurium genome also has, upstream of nusA, an open <br /> gous nusA gene and downstream DNA that has replaced the                     reading frame corresponding to a protein of 15 kDa, an <br /> tetR gene up to the BamHI site. This cloning not only creates               fMet-tRNA, and a similar set of promoter sequences (32a, 33, <br /> a hybrid nusA gene but also results in the recombinant plasmid,             38, 72a). In each of the constructions, the DNA upstream of <br /> like the donor, containing only a 3&apos; portion of the tetR gene.              nusA derives from the same source as the 5&apos; end of the nusA <br />  1398      CRAVEN ET AL.                                                                                                              J. BAC-FERIOL. <br />  <br />                                  BomHI                                                                                      BomHI <br />                   EcoRI <br />  <br />                                                                                   BomHI <br />  <br />  <br />        Nco I                                          Stul <br />  <br />                                                     NcoI - <br />                           Eco RI <br />  <br />                      x Nco I                                                                                         x NcoI <br />                      x Bam HI                                             Eco RI                                     x Bom HI <br />                              i <br />  <br />                     purify f ragment                          -,combine       +   ligate <br />    FIG. 3. Construction of nusAhy2. Plasmid pNAG15 (containing nusAE) was digested with restriction enzymes BamHI and NcoI. The fragment <br /> containing the 3&apos; end of nusAE, was isolated and ligated with linearized pNAS2001 that had been completely digested with BamHI but only <br /> partially digested with NcoI. This construction created a hybrid nusA gene. Like all of the constructs employed in this study, the cloned nusA gene <br /> included the associated promoter of the nusA operon. See Materials and Methods for details. <br />  <br />  <br /> hybrid; e.g., the upstream region derives from S. typhimurium               action. All of the plasmid-cloned hybrid nusA genes tested <br /> when the 5&apos; end of nusA derives from S. typhimurium.                        supported the growth of bacteria with either thermosensitive <br />    Structures of the hybrid nus4 genes. In our initial studies,             or cold-sensitive conditionally lethal nusA alleles (data not <br /> hybrid nusA genes were constructed according to either of two               shown). Therefore, the resulting hybrid NusA proteins also <br /> strategies that relied on in vitro splicing, as detailed in Mate-           support essential bacterial functions that require NusA action. <br /> rials and Methods. The structures of the hybrid genes are                     Complementation of the nusAl mutation by the hybrid nusA <br /> shown in Fig. 2 and described below (the plasmids containing                genes. To determine whether a specific region of the NusAEc <br /> these genes are detailed in Table 3).                                       protein is necessary for N-mediated antitermination, the plas- <br />    The nusAhy2 gene contains nusAst sequences in the 5&apos;                     mid-based hybrid nusA genes were tested for support of N <br /> (N-terminal) two-thirds, and contains nusAEc sequences in the               action in a nusAl mutant host. The complementation test <br /> 3&apos; (C-terminal) one-third, of its reading frame. The nusAhy3                employed was based on two observations. First, at higher <br /> gene is virtually identical to nusAst, since the nusAEc DNA at              temperatures wild-type X grows poorly in an E. coli derivative <br /> its 5&apos; end encodes a sequence with single amino acid difference             carrying the nusAl mutation, because of a failure in N- <br /> from the amino acid sequence derived for nusA5s in that                     mediated antitermination (19). Second, the wild-type allele is <br /> region. However, the derivation of the associated 5&apos; regulatory             dominant in a nusA1/nusA+ heterozygote (21). <br /> region in the case of nusAhy3 is from E. coli, while that for                  Derivatives of X with mutations in various genes and nucleic <br /> nusAst is, obviously, from S. typhimurium.                                  acid sites involved in termination and antitermination of <br />    The nusAhy3O2 gene has nusA,E sequences at its 5&apos; and 3&apos;                 transcription provide a means for assaying host support of N <br /> ends but contains an internal segment that is derived from                  action (28). Three X derivatives that exhibit different degrees <br /> nusAs5. Because it has homologies with the E. coli chromosome               of dependence on host-encoded Nus factors were employed to <br /> on either side of the S. typhimurium sequences, nusAhy3O2                   assess N-mediated antitermination: (i) Abio256sex, which fails <br /> could be recombined into the E. coli chromosome. The result-                to form plaques on lawns formed from nus mutants at low as <br /> ing hybrid chromosome was E. coli except for the small                      well as high temperatures and exhibits maximal dependence <br /> replacement of nuSAEc with nusAst sequences. Thus, we were                  (73); (ii) wild-type X, which forms plaques on such lawns at low <br /> able to unambiguously distinguish effects due to the nusAst                 but not at high temperatures and thus is considered to have <br /> reading frame from any effects on expression that might be due              normal dependence (21); and (iii) Xnin, which forms plaques <br /> to upstream or downstream elements from the S. typhimurium                  on lawns formed from nus mutants at both low and high <br /> genome.                                                                     temperatures and therefore, in terms of N action, exhibits Nus <br />    Complementation of conditionally defective nusA mutations                independence. On the basis of these considerations, this group <br /> by the hybrid nus4 genes. Either the nusA,E or nusAst gene                  of phages was employed to test the abilities of nusA genes <br /> expressed from plasmids complements two different condition-                cloned on plasmids to support N-mediated antitermination in <br /> ally defective nusA mutations in E. coli, one cold sensitive                a host with a chromosomal nusAl allele. The growth of the <br /> [nusAlO(Cs)] and the other thermosensitive [nusAll(Ts)]                     tester phage in the heterozygote can be assessed at 40&#194;&#176;C, at <br /> (data not shown). Therefore, like their chromosomal progen-                 which the chromosomally encoded mutant NusA fails to <br /> itors, these cloned genes express products that are functional              support N action. We find that the restriction on X growth <br /> in supporting essential bacterial processes requiring NusA                  occurs at lower temperatures (40 rather than 42&#194;&#176;C) if the <br />  <br />    FIG. 4. Construction of nusAhy3. Plasmid pN 2/EuS contains one copy each of a carboxy-truncated nusAEc gene and a complete nusAs, gene <br /> oriented in the same direction and separated by a short intervening sequence. Cleavage at the PvuI sites (indicated by the arrows) followed by <br /> ligation generates a hybrid nusA gene. Following partial digestion with PvuI and ligation, functional hybrids were selected by complementation of <br /> the nusAlO(Cs) mutation. See Materials and Methods for details of the construction. <br />  <br />  <br />  <br /> nusAl mutant host carries a pBR322-based plasmid (Table 3                 failure to support N-mediated antitermination by E. coli <br /> and data not shown). Therefore, the restrictive temperature               derivatives containing some of the plasmid-based hybrid nusA <br /> used in this study was 40&#194;&#176;C.                                              genes could be explained by expression of low levels of NusA <br />    The growth of X derivatives was quantitatively assayed in              protein. To assess levels of NusA expression from the various <br /> nusAl/nusAhy heterozygotes by measuring EOP at 40&#194;&#176;C (Table                hybrid nusA constructs, we measured steady-state levels of <br /> 4). Although a nusA + bacterium supports growth of all three              NusA protein by employing immunoblots. Since E. coli re- <br /> phages, only Xnin5 is able to grow at 40&#194;&#176;C in a nusAl host that           quires NusA for growth and wild-type NusA expressed from <br /> carries pBR322, the vector employed in the cloning of the                 the chromosome would interfere with measuring NusA ex- <br /> various nusA genes. A plasmid containing nusAEc fully com-                pressed from plasmids, we employed a derivative of E. coli, <br /> plements the nusAl mutation; all phages tested grow as well in            K7346, with the nusA134(Am) allele (80). In the absence of an <br /> a nusAl/nusAEc heterozygote as in the nusA+ bacterium. A                  amber suppressor, this strain expresses a truncated NusA <br /> plasmid containing nusAst, however, fails to show complemen-              protein that supports bacterial growth at low temperatures. <br /> tation for X growth.                                                      Thus, K7346 provides a means for identifying NusA protein <br />    The plasmid with the truncated nusAA324 gene shows                     expressed from a plasmid, provided that the plasmid-encoded <br /> complementation when assessed for growth of the X derivative              NusA differs in size from the NusA amber fragment. The <br /> with wild-type Nus factor requirements, showing the impor-                immunoblot in Fig. 6 shows that most of the NusA protein in <br /> tance of the 5&apos; region of nusA for N action. However, unlike              K7346 runs lower in the gel than the wild-type protein at the <br /> the case of the complete nusA gene, when the plasmid has the              position expected for that amber fragment (lane 1). Therefore, <br /> truncated gene, the strain that is heterozygotic at the nusA              the protein identified by the anti-NusA antibody observed at <br /> locus fails to support growth of Xbio256sex. Plasmids with the            the position of the wild-type NusA protein in the other lanes <br /> cloned hybrid genes nusAhy2, nusAhy3, and nusAhy3O2 do not                identifies protein expressed from the plasmids. All of the <br /> complement nusAl (as shown in Table 4 for plasmids pNAX2,                 plasmid-based hybrid genes express roughly equal levels of <br /> pNAX3, and pNAX302). Growth of the tester phages in the                   their respective NusA proteins. Therefore, the functional <br /> heterozygotes with these plasmids is indistinguishable from               differences observed between the different nusA-bearing plas- <br /> that observed in the nusAlInusAst heterozygote. The central               mids are not likely to be due to differences in the levels of <br /> regions of these genes are derived from nusAst (Fig. 2).                  expression of their respective NusA proteins. <br />    Levels of expression by plasmid-based nusA genes. The                     Chromosomal hybrid nusA genes. Previous studies with <br />  1400         CRAVEN ET AL.                                                                                                                     J. BAC-rERIOL. <br />  <br />                                                                                 TABLE 4. Complementation of the nusAI mutation by hybrid <br />     A                                                                                                nusA genes <br />     S.t.    MNKEILAVVEAVSNEKALPREKIFEALESALATATKKKYEQEIDVRVEID 50 <br />     E. c. <br />             111 11111111111111111111111111111111111111111111:11 <br />             MNKEILAVVEAVSNEKALPREKIFEALESALATATKKKYEQEIDVRVQID 50 <br />                                                                                Plasmid             Episomal <br />                                                                                                   nusA gene               A             EOPa <br />                                                                                                                                        Anin5       Xbio256sex <br />     S.t.    RKSGDFDTFRRWLIVEEVTMPTKEITLEAARFEDESLNVGDYVEDQIESV 100           pBR322              None                  10-6             -1             &lt;lo-8 <br />             1111111111111:1:111 11111111111:111111:11111111111               pNAG2010            nusAE,,                -1              -1              -1 <br />     E. c.   RKSGDFDTFRRWLVVDEVTQPTKEITLEAARYEDESLNLGDYVEDQIESV 100           pNAS2001            nusAst               &lt;10-7             -1             &lt;10-8 <br />                                                                              pWR324              nusAA324                 -1            -1             &lt;10-8 <br />     S.t.    TFDRITTQTAKQVIVQKVREAERAMVVDQFRDQEGEIVTGVVKKVNRDNI 150           pNAX2               nusAhy2              &lt;10-7             -1             &lt;10-8 <br />              l l l ll llI 11l <br />                             11ll <br />                               ll ll ll ll ll l: : ll l:1l                    pNAX3               nusAhy3              &lt;10-7             -1             &lt;10-8 <br />     E. c.   TFDRITTQTAKQVIVQKVREAERAMVVDQFREHEGEIITGVVKKVNRDNI 150           pNAX302             nusAhy3O2            &lt;10-7             -1             &lt;10-8 <br />                 Nine <br />                                                                              pNAX449             nusAhy449                -1            -1             &lt;10-6 <br />     S.t.    SLEIKSEGMAGNAEAVILREDMLPRENFRPGDRIRGVLYAVRPEARGAQL 200             a Calculated by dividing the phage titer at 40&#194;&#176;C with the indicated strain as a <br />             I I:    : : . I II II II II II   I:1I  I.1I1     I               lawn by the titer of the same phage with the permissive strain K37 carrying <br />     E. c.   SL _      jNAEAVILREDMLPRENFRPGDRVRGVLYSVRPEARGAQL 195 <br />                 Four                                   InusAl                pBR322 as a lawn. <br />                                                        R <br />     S.t.    FVTRSKPEMLIELFRIEVPEIGEEVIEIKAAARDPGSRAKIAVKTNDKRI 250 <br />             11I111111111111111111111111111111111111111111111111 <br />     E. c.   FVTRSKPEMLIELFRIEVPEIGEEVIEIKAAARDPGSRAKIAVKTNDKRI 245           nusAs,  employed a hybrid bacterium, Ki 102, defective for <br />                                                                              support of N action as assayed by X growth (21, 73). The <br />     S.t.    DPVGACVGMRRARVQAVSTELGGERIDIVLWDDNPAQFVINAMAPADVAS 300           genome of K1102, derived primarily from E. coli, has a small <br />             111111I1111 111111111111111111111111111111111111111              substitution of S. typhimurium genetic information that was <br />     E. c.   DPVGACVGMRGARVQAVSTELGGERIDIVLWDDNPAQFVINAMAPADVAS 295           thought to include the entire nusAst gene (2, 21). However, <br />                                                                              there was no direct evidence as to the structure of the nusA <br />     S.t.    IVVDEDKHTMDIAVEAGNLAQAIGRNGQNVRLASQLSGWELNVMTVDDLQ 350           gene in this bacterium. Therefore, we employed Southern <br />             11111111111111111111111111111111111111111111111111               analysis of chromosomal DNA digested with restriction en- <br />     E. c.   IVVDEDKHTMDIAVEAGNLAQAIGRNGQNVRLASQLSGWELNVMTVDDLQ 345 <br />                                                                              zymes to determine the structure of this nusA gene. These <br />     S.t.    AKHQAEAHAAIEIFTKYLDIDEEFATVLVEEGFSTLEELAYVPMKELLEI 400 <br />                                                                              studies (data not shown) indicate that a crossover event <br />             11111111111:. 111111111:111111111111111111 111111111             generating one of the hybrid junctions must have occurred <br />     E. c.   AKHQAEAHAAIDTFTKYLDIDEDFATVLVEEGFSTLEELAYVPMKELLEI 395           within the 3&apos; segments of the nusA genes (Fig. 2). The position <br />                                 V                                  4         of the crossover is located between sites for AvaI (nucleotide <br />     S.t.    DGLDEPTVEALRERAKNALATLAQDQEASLGDNKPADDLLNLEGVDRDMA 450           1172 of nusAst), which is present in the nusA gene from K1102, <br />             :11111111111111111111:11.11.11111111111111111111:1               and BglI (nucleotide 1280 of nusAs,), which is absent in the <br />  E. c.      EGLDEPTVEALRERAKNALATIAQAQEESLGDNKPADDLLNLEGVDRDLA 445 <br />                                                         V <br />     S.t.    FKLAARGVCTLEDLAQQGIDHLADIEGLTDEKAGALIIAARNICWFGDEA 500 <br />             111111111II111111:1111.1111111111111111:111111111111                                                      1        2   3    4      5   6     7 <br />  E. c.      FKLAARGVCTLEDLAEQGIDDLADIEGLTDEKAGALIMAARNICWFGDEA 495 <br />                                                                                                 84kd - <br /> B <br /> Ec 364-414       DIDEDFATVLVEEGFSTLEELAYVPMKELLEIEGLDEPTVEALRERAKNAL <br />                   :1 11 1      1 111:11    : :1 :1111:   : 11   I I <br />                                                                                            Full sized NusA <br /> Ec 439-489       GVDRDLAFKLAARGVCTLEDLAEQGIDDLADIEGLTDEKAGALIMAARNIC <br />                   :1:  1  1    111:11      : :  :1:11 : : 11 <br /> St 369-489       DIDEEFATVLVEEGFSTLEELAYVPMKELLEIDGLDEPTVEALRERAKNAL                           58kd - <br />                   :1: 1 1      1 111:11    :    :1:11: : 11     I I <br /> St 444-494       GVDRDNAFKLAARGVCTLEDLAQQGIDHLADIEGLTDEKAGALIIAARNIC <br />                   I    D            D      I E   D E       D   A                        48.5kd - <br />                                                                               Amber fragment of NusA <br /> Consensus         ID-I-A--L---G--TLEILA---I-IL-JIIGL-I--l-AL---A-N--                                                  40M <br />                                                                                                                        q*M             to : <br />                  V     E            E      M D   E D       E   V                                                                        gz* <br />    FIG. 5. Analysis of NusA proteins. Single-letter abbreviations are <br /> used for amino acids. (A) Comparison of the S. typhimurium and E. coli <br /> NusA protein sequences. The optimal alignment requires one gap of five <br /> amino acids between residues 153 and 154 of the E. coli sequence. The                        36.5 kd - <br /> top line is the deduced S. typhimurium (S.t.) NusA protein sequence, <br /> and the bottom line is the deduced E. coli (E.c.) NusA protein sequence. <br /> The symbols placed between the two sequences indicate the relatedness <br /> of the amino acids: identical amino acids are indicated by a solid line, <br /> chemically similar amino acids are indicated by either two dots (very <br /> similar) or one dot (similar), and unrelated amino acids are indicated by       FIG. 6. Western blot analysis to assess NusA expression from <br /> a blank space. The 449 region is labeled, and its location is shown by       plasmids. Proteins were isolated and analyzed essentially as outlined by <br /> lines above and below the sequence. The amino acid change of the             Schauer et al. (73). The E. coli strain used in this study was K7346, <br /> nusAl mutation (68) is shown downstream of the 449 region. The               which has the nusA134(Am) mutation and expresses a truncated NusA <br /> positions of the repeated sequence are indicated by horizontal arrows at     protein. Positions of molecular mass markers, full-sized NusA protein, <br /> their starts and triangles at their endpoints. The junctions of the hybrid   and truncated NusA amber fragment are indicated to the left. Each of <br /> genes created with restriction enzyme sites shared by the two nusA genes     the lanes (numbered at the top) was loaded with a lysate prepared <br /> are indicated by vertical arrows ( T, PvuI; T T, NcoI; and T t I ,           from K7430 or one of its derivatives containing the following pBR322 <br /> BglII). (B) Alignment of the partial repeats at the carboxy ends of the      derivative, with the nusA gene listed in parentheses: 1, no plasmid <br /> two NusA proteins. Related amino acids are indicated by the same             (control); 2, pNAX2010 (nusAE); 3, pNAS2001 (nusAsj); 4, pNAX449 <br /> symbols described above. The consensus sequence derived from this            (nusAhy302-449); 5, pNAX2 (nusAhy2); 6, pNAX3 (nusAhy3); 7, <br /> alignment is shown in the bottom line. Positions with alternative but        pNAX302 (nusAhy302). Slight differences reflect differences in total <br /> related as well as unrelated (-) amino acids are indicated.                  protein loaded. <br />  VOL. 176, 1994                                                                                       HYBRID nusA GENES           1401 <br />  <br /> nusA gene from K1102. The other crossover creating this              a wide variety of conditions, which included temperatures <br /> hybrid gene occurred somewhere 5&apos; to the nusA gene. Because          ranging between 16 and 42&#194;&#176;C in both defined minimal and rich <br /> of the position of the crossover within the nusA gene, we have       media as well as anaerobiosis in rich medium and growth in the <br /> named this allele nusAhyllO2.                                        mouse gut (data not shown). <br />   A second chromosomal hybrid nusA gene was created by <br /> crossing the nusAhy302 allele from plasmid pNAX302 to the <br /> chromosome (see Materials and Methods for details) to gen-                                    DISCUSSION <br /> erate K4091. The portion of the chromosome of K4091 that is <br /> derived from S. typhimunium sequences can extend no further             Variations between closely related species have long been <br /> than from nucleotide 148 to 926, as this is the extent of S.         used as tools to assess the functional consequences of subtle <br /> typhimurium DNA in nusAhy302. The chromosomal                        genetic differences (58). In the present study, we have exam- <br /> nusAhy302 allele, like the nusAhyllO2 allele, fails to support       ined the structural basis of differences in the actions of nusA <br /> N-mediated antitermination (data not shown).                         gene products from two closely related species of enteric <br />   Defining the region of NusA required for N action. The             bacteria, E. coli (nusA,) and S. typhimurium (nusAst) (66, 72). <br /> nusAEc-nusAst hybrids that support action of N have nUSAEc           The demonstration that an E. coli hybrid with a small substi- <br /> sequences that include the 449 region, which encodes amino           tution of S. typhimurium DNA in the nusA region failed to <br /> acids at positions 153 to 156. In this region there are four         support N-mediated antitermination (21) suggested that if S. <br /> amino acids in NusAE, (DLGN) and nine amino acids in                 typhimurium has a nusA gene, its product is unable to support <br /> NusAst (EIKSEGMAG).                                                  N action. Since NusA provides an essential function, it seemed <br />    To assess the functional significance of the 449 region, we       likely that S. typhimurium has such a nusA gene. The basis for <br /> created a hybrid gene that is primarily nusAs, but has a             the current study was the cloning and sequencing of the nusA,s <br /> substitution in the 449 region; the nucleotides encoding the         gene as well as the comparison of the derived amino acid <br /> four amino acids of the nusAE, 449 region replaced the               sequence with that of nusAEc. This comparison revealed two <br /> nucleotides encoding the nine amino acids of NusAst. The             features that we consider significant: (i) the two sequences <br /> hybrid gene was named nusAhy449 to reflect this switch.              have extensive homologies, and (ii) alignment of the two amino <br />    The effectiveness of the nusAhy449 gene product in support        acid sequences reveals one region in which there is significant <br /> of the X N-mediated antitermination reaction was assessed by         variation between the two NusA proteins. <br /> using the previously described complementation assay. For this          Since NusAst functions in the genetic background of E. coli <br /> study we employed K6210, which has the chromosomal nusAl             K-12, it was not surprising to find that the derived amino acid <br /> allele and a pBR322 derivative with the nusAhy449 allele             sequences of the two nusA genes have extensive homologies. <br /> (pNAX449). A plates with an EOP of 1 on this heterozygotic           Both in vivo and in vitro recombination were employed to <br /> strain at 40&#194;&#176;C (Table 4). Bearing in mind that a host with the       create a series of hybrid nusA genes. These hybrid genes were <br /> nusAl allele fails to support A growth at 40&#194;&#176;C specifically          used to relate structure and function by examining their <br /> because of a failure in N action, this result establishes that the   effectiveness in supporting bacterial growth and N-mediated <br /> product of the plasmid-based nusAhy449 gene supports N               antitermination. <br /> action. However, the low EOP of Xbio256sex on this diploid at           All of the hybrids used in this study were constructed so as <br /> 40&#194;&#176;C   (less than 10-6) indicates that although the potency of <br /> the nusAhy449 allele in support of N action is significantly <br />                                                                      to include the upstream promoter regions. The observation <br />                                                                      that all of the plasmid constructs with hybrid nusA genes <br /> greater than that of nusAst, it is less than that of wild-type       complemented nusA mutations for bacterial growth demon- <br />  nusAEc.                                                             strates that these plasmid-based hybrid genes must be ex- <br />    We next determined whether expression of nusAhy449 in the         pressed. Moreover, Western blot analysis shows that all of the <br /> homozygous state is sufficient to support N action. This was         constructs express similar levels of NusA protein. <br /> accomplished by creating a chromosomal gene with the struc-             Support of N action. Hybrid nusA genes with the proximal <br /> ture of nusAhy449. To this end, the central region of                part of the central portion from nusAst fail to support X <br /> nusAhy449 was crossed to a chromosomally located nusAhy3O2           N-mediated antitermination. This portion of NusA, not sur- <br /> allele (see Materials and Methods). As previously discussed,         prisingly, includes the 449 region that was identified by the <br /> nusAhy3O2, like nusAst, fails to support X N action. The             alignment of the two NusA amino acid sequences. In this <br /> resulting hybrid gene, nusAhy302-449, differs from nusAhy449,        region NusAst has nine amino acids covering a region in which <br /> because the former has E. coli sequences on the 5&apos; and 3&apos; ends.      NusAEc has four different amino acids. To test the functional <br /> However, EOP studies showed that the chromosomally based             significance of this heterology, we created a hybrid nusA gene <br /> nusAhy3O2-449 allele, like the plasmid-based nusAhy449 allele,       whose encoded protein is NusAst except for the region of <br /> supports growth of X with a wild-type N but not growth of            heterogeneity, where the four amino acids in NusAEc have <br /> Xbio256sex (data not shown).                                         been substituted for the nine in NusAst. <br />    Effect of the nusA allele on bacterial growth. The observa-          This substitution of 449 regions, creating nusAhy449, was <br /> tion that X N functions with NusAEc but not with NusAst              sufficient to create a hybrid protein that supports N action. The <br /> indicates that the two proteins differ in at least one functional    diploid with a chromosomal nusA1 allele and a plasmid with <br /> domain. Such a domain might facilitate NusA interaction(s)           nusA449 supports growth of a X derivative with wild-type Nus <br /> with a bacterial factor(s), and thus there could conceivably be      dependence under conditions in which the nusAl haploid <br /> conditions in which NusAEc functions more effectively in E.          parent fails to support growth of this phage. However, the <br /> coli than NusAst. To assess the relative physiological effective-    heterozygote does not support growth of Abio256sex, a phage <br /> ness of the two nusA gene products in an E. coli background,         that expresses low levels of an altered N protein. It has been <br /> we compared the growth of two isogenic strains of E. coli that       proposed that because it has a significant impairment in N <br /> differ only in their nusA genes. These strains are K1456, which      activity, this phage requires more-effective interaction of the <br /> has nusAEc, and K4092, a derivative of K1456 which has the           other components that make up the antitermination complex <br /> nusAhy3O2 allele. Studies monitoring the growth of cultures of       (73). Thus, the inability of the nusAhy449 gene product to <br /> the two strains failed to reveal any significant differences under   support growth of Abio256sex indicates that this NusA protein <br />  1402      CRAVEN ET AL.                                                                                                           J. BACTERIOL. <br />  <br />  is not as effective in support of N action as the wild-type nus4E&quot;,   acids in the 449 region supports wild-type N action. Thus, the <br /> gene product.                                                          nusAhy3O2-449 allele is active in single copy, and its phenotype <br />     Lambdoid phages 21 and P22 have antitermination systems            does not vary from that observed when it is in multiple copies; <br />  analogous to the N system of X, with similarly located N genes        e.g., it supports growth of X but not growth of Xbio256sex. <br /> and nut target sites (18, 30, 39, 51). What is curious, at least for      Conclusion. Our studies identify the 449 region as important <br /> phage 21, an E. coli phage, is that these N products, unlike that      for NusA support of A N action but fail to provide sufficient <br /> of X, function with nusAs5. Thus, the N product of A uniquely          information to elucidate the specific role of that region in N <br /> requires the 449 region from E. coli. However, the requirement         action or to determine whether it has importance for NusA <br /> of the X N for a NusA protein with the E. coli 449 region is not       interactions in E. coli. Since the NusA structure probably did <br /> immutable; a X derivative with mutations in the N gene as well         not evolve to foster an interaction with X N, it is likely that the <br /> as in the nutR signal will grow in a hybrid E. coli strain with        449 region serves to enhance interactions within the bacterium <br /> nusAhyllO2 (25). The N mutations, punAl and punA133,                   that are yet to be identified. Perhaps results of current studies <br /> change, respectively, amino acids 45 (Lys to Arg) and 50 (Ser          on the effects of altering amino acids in the 449 region and <br /> to Asp) of the N protein (73). The nut mutations, either box,A4        surrounding regions will provide the information to formulate <br /> or boxA(Con), are changes in base pairs that create boxA               a definitive and unified model. <br /> sequences in nutR that conform more closely to the consensus <br /> box;A sequence (27).                                                                          <br />     Considering these findings in light of the in vitro studies <br /> demonstrating N binding to NusA (34), it is plausible to                 <br /> assume that the interaction with X N is influenced by the amino         <br /> acids in the 449 region. Perhaps the 449 region of NusAEr             <br /> positions X N so that proper contacts are made with NusA              <br /> and/or other proteins or nucleic acid sites. This may ensure the     <br /> formation of other contacts necessary to establish a function-          <br /> ing antitermination complex. Our finding that the 449 region             <br /> from NusAE, is not sufficient for full X N action is consistent       <br /> with the argument of additional contacts within NusA. Asso-             <br /> ciated with the 449 region is a run of amino acids immediately          <br /> downstream that appears to be functionally important. Within            <br /> 26 amino acids of the 449 region are found mutations influ- <br /> encing both NusA support of N action (nusAl) and bacterial                                         <br /> growth [nusAll(Ts)] phenotypes (63, 73). Moreover, this                  <br /> region in both nusA Ec and nusAs, is unusually rich in arginines;         <br /> there are 8 conserved arginines in a run of 35 amino acids.                <br /> Since arginines have been associated with regions of proteins            <br /> that interact with RNA (6,51, 78), it is conceivable that the 449         <br /> region and this associated arginine-rich region represent such          <br /> an active domain for NusA (79). The nusAl mutation results in              <br /> a change from a leucine to an arginine in the NusA protein.               <br /> This extra arginine residue in the run of arginines might cause            <br /> NusA to bind N or another Nus factor less effectively or                <br /> position it unfavorably in the antitermination complex.                    <br />     Chromosomally located nusA hybrids. The phenotype of                   <br /> nusAst in single copy was studied by using chromosomal hybrid           <br /> genes nusAhyllO2 and nusAhy3O2. Although these hybrid                     <br /> genes have different proportions of nusAEr and nusAst, they                <br /> both fail to support N action. The nusAhyllO2 allele had been           <br /> assumed to be a complete nusAst gene (21, 73). Our analysis                 <br /> demonstrates that one of the crossover events that substituted             <br /> S. typhimurium DNA for E. coli DNA in K1102 occurred                     <br /> within the 3&apos; regions of the nusA genes. The nusAst portion of             <br /> nusAhyl1O2, as well as that of nusAhy3O2, includes the 449           <br /> region. Therefore, we think it logical to assume that studies of       <br /> N action with either of these chromosomally based hybrid                   <br /> genes should reflect the phenotype expected of the complete                <br /> nusAst gene.                                                             <br />     Finally, we focus on nusAhy3O2. Since this hybrid is nusA,Ec           <br /> on both the 5&apos; and 3&apos; ends, the only S. typhimurium DNA in the             <br /> bacterium is internal to the nusA gene. Therefore, all regula-            <br /> tory signals (transcriptional and translational) that are 5&apos; as        <br /> well as 3&apos; of the nusA gene are from E. coli. The functional          <br /> features distinguishing nusAst from nusAEc should therefore                <br /> primarily reflect structural differences. The chromosomally                <br /> located nusAhy302 allele, like the plasmid-based hybrid, fails to          <br /> support N action. However, a derivative of nusAhy3O2 that has    <br /> the four NusAEC amino acids instead of the nine NusA5t amino               <br /> </body></html>