<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Biotechnology                                                                                                                        BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Plasmid selection Escherichia coli using endogenous essential <br /> gene marker <br /> Shan Goh1 Liam Good*1,2 <br />  <br /> Address: 1Department Cell Molecular Biology, Karolinska Institute, Stockholm, SE-17177, Sweden 2Department Pathology and <br /> Infectious Diseases, Royal Veterinary College, University London, AL9 7TF, UK <br /> Email: Shan Goh - shan.goh@ki.se; Liam Good* - lgood@rvc.ac.uk <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 11 August 2008                                                     Received: 4 April 2008 <br />                                                                               Accepted: 11 August 2008 <br /> BMC Biotechnology 2008, 8:61   doi:10.1186/1472-6750-8-61 <br /> This article available <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1472-6750/8/61 <br /> &#194;&#169; 2008 Goh Good; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Antibiotic resistance genes widely used selection recombinant bacteria, <br />                  use risks contributing spread antibiotic resistance. In particular, practice is <br />                  inappropriate intrinsically resistant bacteria vaccine production, costly for <br />                  industrial scale production. Non-antibiotic systems available, require mutant host strains, <br />                  defined media expensive reagents. An unexplored concept over-expression host essential <br />                  gene enable selection presence chemical inhibitor gene product. <span id='am-8' about='protege:TO' typeof='owl:Thing'>To</span> test this <br />                  idea E. coli, used growth essential target gene fabI plasmid-borne marker the <br />                  biocide triclosan selective agent. <br />                  Results: The new cloning vector, pFab, enabled selection triclosan 1 &#206;&#188;M. Interestingly, pFab <br />                  out-performed parent pUC19-ampicillin cell growth, plasmid stability plasmid <br />                  yield. Also, pFab toxic host cells way reversed triclosan. Therefore, pFab <br />                  triclosan toxic used combination enhance growth plasmid <br />                  production gene-inhibitor interaction. <br />                  Conclusion: The fabI-triclosan model provides alternative plasmid selection method <br />                  based essential gene over-expression, use antibiotic-resistance genes and <br />                  conventional antibiotics. <br />  <br />  <br />  <br />  <br /> Background                                                                    tem avoid antibiotic resistance markers thera- <br /> Antibiotic resistance marker genes commonly used                       peutic antibiotics, cost effective, convenient, <br /> select maintain recombinant bacteria presence                      robust flexible. <br /> of antibiotics. However, use antibiotics undesira- <br /> ble manufacturing gene therapy products [1,2]. Also,                      A variety strategies antibiotic-free selection have <br /> the introduction antibiotic resistance genes bio-                     developed, widely used bacteria. The <br /> hazardous strains recommended [3], antibiotic                      reported non-antibiotic involves auxo- <br /> selection fails bacteria naturally resistant [3,4].               trophic bacterial strain complementation using a <br /> Finally, use antibiotics costly industrial                   plasmid-encoded biosynthesis gene, trans- <br /> scale production, especially case enzymatic                         formants grow defined media lacking <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> [5]. <br /> depletion antibiotic culture. An alternative sys-                   Another termed repressor-titration, lac <br />  <br />  <br />                                                                                                                                          Page 1 9 <br />                                                                                                                  (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                   http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> operator functions vector-borne selection marker,          pBAD18s (National Institute Genetics, Japan). Media <br /> which de-represses modified chromosomal essential               SOC LB (GIBCOBRL) supplemented amp- <br /> gene [6]. A general approach alter expres-        icillin (LBA, 100 &#206;&#188;g/ml ampicillin, (Sigma)), triclosan <br /> sion growth essential genes using synthetic [7]             (LBT, 1 &#206;&#188;M triclosan; Ciba) arabinose (Sigma). Tri- <br /> expressed antisense sequences [8]. Unfortunately,           closan used 1 M stock DMSO. <br /> and existing non-antibiotic systems suffer a <br /> need mutant host strains expensive reagents,        Construction pFab, pUCFA pBFab <br /> some cases low efficiency.                                        An EagI site created nucleotide position 1621 of <br />                                                                   pUC19 (New England Biolabs), immediately downstream <br /> It observed over-expression growth             bla, PCR primers (5&apos;cgtcggccgttaccaatgcttaatcag <br /> essential gene results reduced susceptibility gene      5&apos;cgccggccggaccaagtttactcatat). The amplicon was <br /> product inhibitor [9]. Here considered strategy based        digested EagI (New England Biolabs), ligated T4 <br /> on essential gene plasmid-borne marker            DNA ligase transformed DH5&#206;&#177; propagation. <br /> specific protein level inhibitor selective agent. To       To construct pFab, bla gene excised pUC19 <br /> test idea, chose fabI-triclosan model combina-        SspI EagI, replaced fabI its <br /> tion well-studied gene-inhibitor pair: FabI       promoter, amplified K12 genomic DNA <br /> (enoyl ACP reductase) catalyzes fatty acid elongation [10]        using primers 5&apos;ccggatatcgtgctggagaatattcg and <br /> and confers reduced susceptibility triclosan over-        5&apos;gcgcggccgttatttcagttcgagttcgtt. The amplicon was <br /> expressed E. coli [11,12]; triclosan inhibits FabI     digested EcoRV EagI used create pFab <br /> binding ACP-enoyl substrate site, forming stable         vector. E. coli DH5&#206;&#177; transformed pFab and <br /> FabI/NAD+/triclosan ternary complex [13]. In addition,            plated LB 0.5 &#226;&#8364;&#8220; 5 &#206;&#188;M triclosan. Transformants <br /> triclosan biocide fulfils criteria non-anti-   subsequently maintained 1 &#206;&#188;M triclosan (LBT). <br /> biotic [14]. It stable, easy handle, inexpensive        The fabI gene cloned pUC19 SphI and <br /> approved use hygiene, household indus-            BamHI MCS (pUCFA) similar way using <br /> trial applications [15,16].                                       primers 5&apos;ccggcatgcgtgctggagaatattcg 5&apos;ccggatc- <br />                                                                   cgattatttcagttcgagt. Competent E. coli DH5&#206;&#177; trans- <br /> Although triclosan used systemic therapeutic,      formed pUCFA plated LBA LBT. <br /> is important consider possible risks associated with <br /> new applications. Most importantly, concern         To induce expression fabI PBAD, fabI amplicon <br /> triclosan use contribute antibiotic resistance [17];       generated primers (5&apos;cggaattcgaatgggttttctttccgg and <br /> indeed, resistant mutants generated labora-         5&apos;cctctagagattatttcagttcgagt) digested EcoRI and <br /> tory [18-20]. However, studies bacteria non-labora-         XbaI (New England Biolabs) cloned pBAD18s, <br /> tory conditions exposed biocide-concentrations              similarly digested, yield pBFab1. Expression <br /> triclosan did correlation antibiotic           fabI pBAD18 required Shine Dalgarno sequence, <br /> resistance decreased triclosan susceptibility [21-23].        predicted uaagga position -13 relative <br /> Regulatory agencies continue approve use tri-           start codon. Primers (5&apos;cggaattctcaacaataaggat- <br /> closan domestic clinical products [24,25],          taaagc 5&apos;cctctagagattatttcagttcgagt) used for <br /> appears comparatively safe choice selection          amplification fabI Shine Dalgarno sequence, <br /> agent.                                                            cloned pBAD18 yield pBFab6. E. coli DH5&#206;&#177; <br />                                                                   transformed pBFab plasmids plated <br /> In study, tested fabI-marker plasmid selection         LBA, LBT LBT 0.2% (w/v) arabinose. <br /> triclosan observed efficient stable selection. The <br /> new out-performed parent antibiotic          Plasmid transformant properties <br /> growth plasmid production presence tri-             Transformation efficiencies chemically competent <br /> closan. Cells containing fabI-marker plasmid dis-             DH5&#206;&#177; cells determined recommended the <br /> played toxic effects absence triclosan, suggesting      manufacturer (Invitrogen). <br /> an &quot;addictive&quot; effect, aid plasmid contain- <br /> ment.                                                             Plasmid yields determined clones of <br />                                                                   pUC19 pFab. Plasmids isolated ml <br /> Methods                                                           overnight cultures grown selection using mini- <br /> Bacterial strains, plasmids media                             prep kit (Qiagen) quantified OD260 readings. Plas- <br /> The E. coli strains used study DH5&#206;&#177; (Invitro-        mids (50 ng) digested BamHI and <br /> gen), XL1-Blue (Strategene), HB101, BL21 (Strategene)             electrophoresed 1% agarose gel. <br /> and K12 (Coli Genetics Stock Center, Yale U.). Plasmids <br /> were pUC19 (New England Biolabs), pBAD18 and <br />  <br />  <br />                                                                                                                        Page 2 9 <br />                                                                                                (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> Plasmid stability assayed presence        determined plasmid extraction, digestion 100 ng <br /> absence selection. Overnight cultures grown           DNA BamHI fractionation 1% agarose gel. <br /> selection 37&#194;&#176;C shaking diluted 1000 &#195;&#8212; 5 <br /> ml LB selection, aliquots time      Inducible expression fabI <br /> zero cultures diluted plated LB plates con-      Clones pBFab1 pBFab6 grown LBA 16 <br /> taining X-Gal (20 &#206;&#188;g/ml, Saveen). The time zero cultures       h, diluted approximately 5 &#226;&#8364;&#8220; 9 &#195;&#8212; 106 cfu/ml LBT and <br /> were grown diluted 24 48 h, aliq-      aliquots 180 &#206;&#188;l added wells 96 plate. <br /> uots 48 h cultures plated. The ratios blue      Arabinose added final concentration 0 &#226;&#8364;&#8220; 5% <br /> colonies total colonies LB plates X-Gal (20 &#206;&#188;g/     final volume 200 &#206;&#188;l with <br /> ml) determined 0 h 48 h, % plas-        water. Clones pBAD18 pBAD18s included as <br /> mid-bearing cells 48 h calculated. Five independ-      controls. Cultures grown 24 h VERSAmax <br /> ent clones pUC19 pFab used provide              spectrophotometer (Molecular Devices) agitation for <br /> replicates.                                                    5 s 5 min, followed OD550 reading. The growth <br />                                                                rate arabinose concentration calculated as <br /> Plasmid abundance determined ways. First,        described above. <br /> compare band intensities genomic (gDNA) plasmid <br /> DNA (pDNA) agarose gel, total genomic DNA            Cell viability <br /> extracted different clones pUC19 pFab         DH5&#206;&#177;/pFab grown overnight absence pres- <br /> clones grown selection 16 h. Five cultures        ence 0.5 &#226;&#8364;&#8220; 2 &#206;&#188;M triclosan subjected SYTOX green <br /> K12, derived single colonies, grown LB       staining flow cytometry, previously described [29]. <br /> for DNA extraction using Bacterial GenElute         DH5&#206;&#177;/pUC19 included control determine <br /> (Sigma) ml overnight culture. Total DNA (10         staining levels live heat-treated dead cells. Samples <br /> &#206;&#188;l) electrophoresed 1% agarose gel, stained      excited 488 nm air-cooled argon ion laser in <br /> ethidium bromide scanned using Typhoon 9400              CyFlow SL flow cytometer (Partec GmbH). Threshold <br /> (Amersham Biosciences). Band intensities deter-           settings enabled forward scatter exclude cell <br /> mined using ImageQuant software (Amersham Bio-              debris. The forward scatter dot plot used to <br /> sciences). Second, relative quantitative PCR (qPCR)        identify gate cell populations. Fluorescence meas- <br /> carried using plasmid lacZ&#206;&#177; gene target      ured 520 nm. Viable dead cell populations were <br /> gene single copy chromosomal dxs reference          counted using Partec, FloMax software version 2.4e. <br /> gene [26]. K12 gDNA containing single copy lacZ&#206;&#177; <br /> and dxs used calibrator. Primers amplifying       Growth competition <br /> target gene (5&apos;gtgctgcaaggcgattaagtt 5&apos; cactggccgtcgttt-   Growth competition DH5&#206;&#177; plasmid bearing <br /> tacaa), reference gene (5&apos;cgagaaactggcgatcctta         cells carried previously described, modifi- <br /> 5&apos;cttcatcaagcggtttcaca) validated similar amplifi-    cations [30]. Overnight (24 h) cultures DH5&#206;&#177;, DH5&#206;&#177;/ <br /> cation efficiencies. Real time data analyses carried      pUC19 DH5&#206;&#177;/pFab prepared LB, LBA and <br /> out 2-&#206;&#8221;&#206;&#8221;CT method relative qPCR [27]. Total         LBT, respectively. Equal volumes DH5&#206;&#177; plasmid <br /> DNA concentrations determined OD260 absorb-            bearing cultures mixed diluted 1:100 10 ml <br /> ance qPCR. Each 25 &#206;&#188;l PCR reaction contained 12.5       fresh LB growth. An aliquot diluted <br /> &#206;&#188;l SYBR Green PCR buffer (Eurogentec), 100 nM            mixed culture simultaneously plated selective <br /> each primer (Biomers) 5 ng total DNA.                      non-selective media cell counts. After mixed <br />                                                                culture incubated 24 h shaking 37&#194;&#176;C, it <br /> Growth rates calculated exponential phase        diluted 1:100 10 ml fresh LB growth <br /> of growth [28], monitored increased OD550         plated, above. This procedure repeated 5 times <br /> over time VERSAmax spectrophotometer (Molecu-           days. The numbers ampicillin triclosan <br /> lar Devices). An overnight culture (16 h), standardized     resistant colonies scored relative total CFUs. <br /> OD550 yield approximately 7 &#195;&#8212; 105 cfu/ml, grown <br /> in 200 &#206;&#188;l volumes 96-well plate 24 h    Results <br /> agitation 5 s 5 min, OD550 readings        Vectors containing fabI enable triclosan selection <br /> taken. Triclosan added 0 &#226;&#8364;&#8220; 2 &#206;&#188;M Triclosan      To test potential fabI selective marker clon- <br /> 1% DMSO final concentration.                                   ing, vectors derived pUC19 constructed. <br />                                                                First, constructed pFab, fabI its <br /> The host range pFab commonly used E. coli            native promoter replaces ampicillin resistance gene <br /> cloning strains tested transformation XL1-Blue       (bla) pUC19. Second, enable selection triclosan <br /> (Stratagene), HB101 BL21, followed selection         ampicillin, constructed pUCFA, contains the <br /> LBT plates. Plasmid DNA integrity abundance            fabI cassette cloned pUC19 MCS (Figure 1A). E. <br />  <br />  <br />                                                                                                                      Page 3 9 <br />                                                                                              (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                                                                                                                   http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />  A                                                                                                                                                                              Vector <br />                                                                                                                                                                                 Figure construction <br />                                                                                                                                                                                          1               triclosan selection <br />                                                                                                                                                                                 Vector construction triclosan selection. (A) The <br />                                                                                                                                                                                 bla gene pUC19, confers ampicillin resistance, was <br />                             EcoRV                                    AatII                                                           BamHI                                      replaced fabI promoter region (pFab). The <br />                                      Pbla <br />                                    SspI                                                                                                                                         pUC19 multiple cloning site (MCS) retained, Hin- <br />                                                                                                     lacZa <br />                                                                                                      BamHI                                                                      cII, HindIII PstI unique pFab. The fabI cassette in <br />    (triclosan resistance) <br />  <br />  <br />  <br />  <br />                                            (ampicillin resistance) <br />  <br />  <br />  <br />  <br />                                                                                                         MCS                                                                     pFab transferred pUC-derived plasmids using <br />  <br />  <br />  <br />  <br />                                                                                                                                                    fabI <br />                                                                                                        SphI <br />                                                                      pUC19 <br /> fabI <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 AatII AlwNI restriction sites. The fabI cassette was <br />                                      bla <br />  <br />  <br />  <br />  <br />                                                                                                        Plac <br />  <br />                                                                      2686 bp                                                                                                    inserted MCS pUC19 obtain pUCFA. All <br />                                                                                                                                                                                 plasmids available authors. (B) Growth pFab <br />                                                                                                                                    SphI                                         pUCFA clones LBT LBA plates. (C) Plasmids <br />                               EagI     EagI                                                  ori                                                                                propagated different E. coli hosts digested with <br />                                                                                           AlwNI <br />                                                                                                                                                                                 BamHI analyzed gel electrophoresis. <br />  <br />                                    AatII                                                                                                 AatII <br />                                                                                   lacZa                              Plac                                                       coli strain DH5&#206;&#177; transformed pFab, pUCFA or <br />                                                                                      BamHI                                                                         BamHI <br />                                                                                            MCS <br />                                                                                                                                                                                 pUC19 transformants selected LB plates con- <br />                                                                                                           bla <br />                                                                                            SphI                                                                                 taining triclosan (LBT) ampicillin (LBA). As antici- <br />                                 pFab                                                        Plac                               pUCFA                                            pated, fabI inserts enabled selection triclosan <br />   fabI <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          fabI <br />  <br />  <br />                                2912 bp                                                                                         3771 bp                                          containing plates. Colonies formed LBT more <br />                                                                                                                                                                                 variable size colonies LBA. However, triclosan- <br />                                                                                                          AlwNI <br />                                                                                                                                                                                 resistant colonies sizes maintained resistance (Figure <br />                                                                                ori                            ori                                                 SphI <br />                             EagI                                             AlwNI                                                                               Plac           1B) displayed uniform colony morphologies re- <br />                                                                                                                                                                                 plating. In contrast pUC19, did observe plas- <br />                                                                                                                                                                                 mid-free colonies satellite colonies using pFab <br /> B                                                                                                                                                                               pUCFA. Therefore, fabI-triclosan enables <br />                                                                                                                                                                                 non-antibiotic selection maintains stable recom- <br />                                                                      DH5&#206;&#177;/                                                                 DH5&#206;&#177;/ <br />                                                                                                                                             DH5&#206;&#177;/                               binant strains. <br />                             DH5&#206;&#177;                                     pUC19                               DH5&#206;&#177; <br />                                                                                                          DH5&#206;&#177;                              pUC19 <br />                                                                                                                                             pUC19 <br />                                                                                                                                                                                 To test fabI-triclosan selection function <br />                                                                                                                                                                                 E. coli strains, pFab transformed into <br />                                                                                                                                                                                 strains BL21, HB101 XL1-blue. Similar results <br />                                                                                                                                                                                 DH5&#206;&#177;, observed efficient selection high yield <br />                                                                                                                                                                                 plasmid production (Figure 1C). To test fabI can <br />                                                                                                                                                                                 enable triclosan selection vectors, pFab cas- <br />                             DH5&#206;&#177;/ DH5&#206;&#177;/                                                                  DH5&#206;&#177;/ DH5&#206;&#177;/                                                            sette inserted multiple cloning site pGEM- <br />                             pFab pUCFA                                                                   pFab pUCFA                                                             3Zf used replace ampicillin resistance gene <br />                                                                                                                                                                                 low copy number vector pBR322. In cases, tri- <br />                             Triclosan (LBT)                                                              Ampicillin (LBA)                                                       closan resistant colonies selected (data shown). <br />                                                                                                                                                                                 Therefore, fabI-triclosan enables efficient selec- <br />                                                                                                                                                                                 tion commonly used vectors E. coli strains. <br />  C <br />                                                                                                                                                  XL1-Blue/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 To confirm fabI expression main mechanism <br />                                                                              DH5&#206;&#177;/pUC19 <br />  <br />  <br />  <br />  <br />                                                                                                                             HB101/pFab <br />                                                                                              DH5&#206;&#177;/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 mediating triclosan resistance point mutations <br />                                                                                                          BL21/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 chromosomal fabI [12], expression fabI was <br />                                                                                                                                                                                 placed control PBAD promoter in <br />                             kb                                                                                                                                                  pBAD18s (pBFab1) pBAD18 (pBFab6) [31]. The <br />                                                                                                                                                                                 pBFab1 pBFab6 strains tested resistance fol- <br />                             3.5                                                                                                                                                 lowing fabI induction. In absence inducer arab- <br />                               3                                                                                                                                          2.9    inose, growth observed LBT broth, growth <br />                             2.5                                                                                                                                          2.7    rates increased increasing arabinose concentrations <br />                               2                                                                                                                                                 0.4% (Figure 2). Therefore, pFab-mediated resist- <br />                                                                                                                                                                                 ance triclosan expression fabI. <br /> Figure 1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         Page 4 9 <br />                                                                                                                                                                                                                 (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />                                                                A <br />                                                                       DH5&#206;&#177;/pUC19          DH5&#206;&#177;/pFab <br />                                                                 kb <br />                                                                                                                      genomic <br />                                                                 10 <br />                                                                  4                                                   DNA <br />                                                                  2                                                   plasmid <br />                                                                                                                      DNA <br />  <br />  <br />                                                                B <br />  <br /> Figureof2fabI induction triclosan resistance <br /> Effect <br /> Effect fabI induction triclosan resistance. Tri- <br /> closan resistance mediated arabinose-induced over- <br /> expression fabI. Exponential growth rates DH5&#206;&#177;/ <br /> pBFab1 DH5&#206;&#177;/pBFab6 relative DH5&#206;&#177;/pBAD18s and <br /> DH5&#206;&#177;/pBAD18, respectively. <br />  <br />  <br /> Characterization pFab <br /> After selection pFab transformants triclosan, we <br /> characterized general properties pFab cloning <br /> vector, pUC19 included comparison (Table 1).          Plasmid <br />                                                                Figure 3 production pFab transformants <br /> Plasmid preparation yield pFab 43% greater         Plasmid production pFab transformants. (A) Agar- <br /> pUC19 (Table 1) E. coli DH5&#206;&#177; transformants. Also,       ose gel electrophoresis total DNA isolated five <br /> copy number pFab 38% 40% greater          clones pUC19 pFab. Bands quantified using <br /> of pUC19, measured qPCR plasmid genomic           ImageQuant software calculate ratio pDNA gDNA <br /> DNA abundance (Table 1; Figure 3A &amp; B), respectively.          measure copy number pUC19 pFab. (B) Mean <br /> Therefore, pFab displayed higher yield copy number         ratios pDNA:gDNA pUC19 pFab (A). <br /> relative parent vector. <br />  <br /> To assess plasmid stability, scored number     observed selection (Figure 4A; Table 1). We next <br /> triclosan resistant colony forming units relative total     determined DH5&#206;&#177;/pFab culture growth rates pres- <br /> colony forming units. Surprisingly, observed col-      ence range triclosan concentrations (0 &#226;&#8364;&#8220; 2 &#206;&#188;M). As <br /> onies LBT LB plates. This indicated high plas-      pFab unstable triclosan, DH5&#206;&#177;/pUCFA <br /> mid stability presence triclosan, form      grown LBA included prevent growth com- <br /> of pFab-mediated toxicity absence triclosan.         peting plasmid-free cells. Growth rates pUCFA and <br /> Indeed, measurements plasmid stability using alter-      pFab clones lowest triclosan increased <br /> native &#206;&#177;-complementation assay revealed pFab          triclosan addition 1 &#206;&#188;M, indicating triclosan- <br /> more stable pUC19 selection (Table 1). There-       dependent fitness rescue cells carry pFab (Figure <br /> fore, pFab over-expression clearly conferred tri-        4B). Similarly, observed proportion dead <br /> closan resistance, appeared confer requirement    cells decreased increasing triclosan concentrations. <br /> for biocide.                                               Fluorescence microscopy SYTOX stained cells [29] <br />                                                                revealed elongated dead cells absence tri- <br /> Effects pFab triclosan E. coli growth, <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>survival</span>      closan, elongated viable cells 1 &#206;&#188;M triclosan <br /> and fitness                                                    (data shown). The ratio live dead cells indicated <br /> To investigate interaction fabI tri-   SYTOX staining quantified flow cytometry <br /> closan, examined culture growth profiles. While       [29]. The results showed triclosan reduced the <br /> ampicillin little effect DH5&#206;&#177;/pUC19 cultures,       number dead cells, indicating pFab transformants <br /> final optical density DH5&#206;&#177;/pFab highest          rescued triclosan (Figure 4C). Therefore, pFab is <br /> presence triclosan exponential phase growth         toxic toxicity suppressed triclosan. <br /> rates DH5&#206;&#177;/pFab cultures faster DH5&#206;&#177;/ <br /> pUC19 cultures selection, inverse was <br />  <br />  <br />                                                                                                                     Page 5 9 <br />                                                                                             (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> Table 1: Properties pFab pUC19 <br />  <br />  Parametersa                                                                    pUC19                                   pFab <br />  <br />  Transformation efficiencyb (CFU/&#206;&#188;g)                                     5.5 &#195;&#8212; 107 &#194;&#177; 1.2 &#195;&#8212; 107                  2.2 &#195;&#8212; 107 &#194;&#177; 3.6 &#195;&#8212; 106 <br />  Plasmid yieldc (&#206;&#188;g/ml)                                                        30 &#194;&#177; 2.1                              42.9 &#194;&#177; 3.5 <br />  Copy numberd                                                                  141 &#194;&#177; 25                               200 &#194;&#177; 33 <br />  Stability (%) <br />     With selectione                                                           85.3 &#194;&#177; 10.8                            99.5 &#194;&#177; 1.5 <br />     Without selectionf                                                        56.7 &#194;&#177; 17.7                            54.5 &#194;&#177; 14.9 <br />  Relative growth rateg (&#206;&#8221;OD/&#206;&#8221;t) <br />     With selection                                                                 1                                 1.2 &#194;&#177; 0.07 <br />     Without selection                                                         1.1 &#194;&#177; 0.05                             0.6 &#194;&#177; 0.02 <br />  <br />  Determined   replicate cultures. <br />  b In chemically competent DH5&#206;&#177; cells. <br />  c Plasmid yield 1 ml 18 h culture, determined OD <br />                                                                260 absorbance. <br />  d Determined copy ratio lacZ&#206;&#177; dxs qPCR. <br />  e Percentage plasmid-bearing cells 48 h cultures grown selection. <br />  f Percentage plasmid-bearing cells 48 h cultures grown selection. <br />  g Change OD <br />                   550 time DH5&#206;&#177;/pUC19 cultures LB DH5&#206;&#177;/pFab LB LBT, relative control DH5&#206;&#177;/pUC19 cultures LBA. <br />  <br />  <br /> To test triclosan resistance likely persist out-           practice [21-23,39,40]. In laboratory, <br /> side intended applications, assessed fitness                   spontaneous triclosan resistance E. coli resulting from <br /> DH5/pUC19 DH5/pFab strains mixed culture                     exposure low triclosan concentration been <br /> the plasmid-free parent strain. A competitive fitness assay,             observed, point mutations fabI increased <br /> conducted absence selection, showed pFab                  MIC 95 fold [12]. Such mutations arise dur- <br /> persistence weaker pUC19 co-cul-                 ing triclosan selection. However, experiments, tri- <br /> ture days (Figure 4D). In words, rate              closan resistance dependent expression of <br /> pFab loss faster pUC19 loss therefore,               plasmid-borne fabI (Figure 2), blue-white selection of <br /> absence selection, pFab stable competi-              recombinant E. coli indicated pFab-carrying cells <br /> tive pUC19.                                                         (blue) spontaneous resistant mutants (Table <br />                                                                          1). In environment, triclosan resistance slow <br /> Discussion                                                               emerge compared antibiotic resistance [21,23], pos- <br /> We over-expression growth essential gene                   sibly poor solubility triclosan [41], rapid degra- <br /> conferring resistance specific protein inhibitor               dation triclosan [22,42,43], low competitive fitness of <br /> plasmid selection E. coli, using fabI triclosan            FabI mutants [12], tripartite nature FabI/ <br /> as example. As avoiding use antibiotic                 NADPH/triclosan complex. Furthermore, spread of <br /> resistance genes antibiotics, pFab transformants                     pFab outside intended applications limited <br /> showed improved growth, yield gene containment.                      low plasmid stability, poor competitive fitness and <br /> These improvements appear to mechanism                     cell toxicity absence triclosan. Indeed, FabI is <br /> of inhibition [13] balance toxic gene-inhibitor                 stringently regulated fatty acid biosynthesis <br /> levels required <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>cell survival</span> [32] selection [33].               pathway [32], de-regulation inhibits cell growth and <br /> Similar interactions systems suggest gene-                 viability [44,45]. Nevertheless, possible pFab <br /> inhibitor reciprocal suppression common mech-                   transfer horizontally induce resistance to <br /> anism. For example, reciprocal mutual suppression                 triclosan wild-type bacteria, standard precautions <br /> been described genes [34] chemical                       handling genetically modified microorganisms <br /> inhibitors [35]. Also, antibiotic-dependent strains                 maintained. <br /> been described bacteria isolated laboratories <br /> [36,37] clinics [38].                                                In large scale production proteins, plasmid stability <br />                                                                          selection pre-requisite residual anti- <br /> The new selection relies endogenous E. coli                 biotics undesirable. In regard, finding that <br /> gene widely used biocide. Nevertheless, relative               pFab requires triclosan plasmid stability a <br /> safety triclosan fabI application               disadvantage. However, triclosan required <br /> considered. Triclosan approved regulatory authori-                 compared antibiotic selection (typ- <br /> ties EU USA applications [24,25]                 ically 2%), level residual triclosan permis- <br /> and association bacterial triclosan resistance                sible, given approval use non-prescription <br /> and antibiotic susceptibility, suggested,                 medicines hygiene products. Therefore, need for <br />  <br />  <br />                                                                                                                                    Page 6 9 <br />                                                                                                           (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                 http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />      A <br />                                formants <br />                                Effect <br />                                Figure  of4triclosan growth, viability fitness pFab trans- <br />                                Effect triclosan growth, viability fitness <br />                                pFab transformants. (A) Growth curves DH5&#206;&#177;/ <br />                                pUC19 DH5&#206;&#177;/pFab cultures grown without <br />                                selection. Growth rates determined exponen- <br />                                tial phase, indicated arrows. (B) Reduced fitness fabI <br />                                over-expression suppression triclosan. Growth <br />                                rates pFab pUCFA clones LB LBA, respectively, <br />                                calculated relative DH5&#206;&#177;/pUC19 LBA. (C) Flow <br />                                cytometric determination dead cell numbers, measure <br />                                cell toxicity, DH5&#206;&#177;/pFab cultures grown presence <br />                                range triclosan concentrations. (D) Growth competi- <br />                                tion DH5&#206;&#177; DH5&#206;&#177;/pUC19 DH5&#206;&#177;/pFab. The <br />                                log10 ratio plasmid-bearing cells total number cells <br />      B                         time represents rate plasmid loss mixed cell <br />                                populations. The data representative independent <br />                                experiments. <br />  <br />  <br />  <br />                                stability absence selection prove impor- <br />                                tant system. Nevertheless, pFab stability a <br />                                potential problem vector construct process adjust- <br />                                ments needed scale-up. <br />  <br />                                Conclusion <br />                                In summary, study provides example essen- <br />                                tial genes used combination non-antibiotic <br />       C                        inhibitors select maintain recombinant bacteria. It <br />                                possible apply triclosan selection bac- <br />                                teria Staphylococcus aureus [46] Mycobacterium <br />                                smegmatis [47], homologues FabI. Other <br />                                inhibitors FabI described [48,49] and, <br />                                depending bacterial resistance development, certain <br />                                triclosan analogues preferred use plas- <br />                                mid selection system. On hand, bacteria that <br />                                divergent enoyl-ACP reductase, Bacillus subti- <br />                                lis (FabL) [50] Streptococcus pneumoniae (FabK) [51], <br />                                species intrinsically resistant triclosan, <br />                                Pseudomonas aeruginosa [18], suitable for <br />                                fabI-triclosan plasmid selection. For species, would <br />                                interesting test additional essential gene inhibitor <br />       D                        combinations expand plasmid selection strategy <br />                                described here. The pFab triclosan poten- <br />                                tially attractive production recombinant proteins, <br />                                used increase plasmid copy number <br />                                yield. In addition, pFab derivatives be <br />                                suitable manufacturing biopharmaceuticals gene <br />                                therapy products, applications require <br />                                absence antibiotic resistance sequences antibiotic <br />                                residues. <br />  <br />                                Authors&apos; contributions <br />                                SG carried experimental work data analyses <br /> Figure 4 <br />                                flow cytometry, participated design study <br />                                drafted manuscript. LG conceived study, car- <br />  <br />  <br />                                                                                        Page 7 9 <br />                                                                (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                    http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> ried flow cytometry FACS analysis, helped                          19.   Braoudaki M, Hilton AC: Adaptive resistance biocides Sal- <br />                                                                                      monella enterica Escherichia coli O157 cross-resist- <br /> with design drafting manuscript. All authors                            ance antimicrobial agents. J Clin Microbiol 2004, 42:73-78. <br /> read approved final manuscript.                                        20.   Karatzas KA, Webber MA, Jorgensen F, Woodward MJ, Piddock LJ, <br />                                                                                      Humphrey TJ: Prolonged treatment Salmonella enterica <br />                                                                                      serovar Typhimurium commercial disinfectants selects <br /> Acknowledgements                                                                     multiple antibiotic resistance, increased efflux and <br /> We thank Swedish Institute Guest Scholarship SG                reduced invasiveness. J Antimicrob Chemother 2007, 60:947-955. <br /> Swedish Research Council University London Central Research             21.   Cole EC, Addison RM, Rubino JR, Leese KE, Dulaney PD, Newell MS, <br /> Fund support. We thank Thomas Bentin, Omid Faridani, Abbas                  Wilkins J, Gaber DJ, Wineinger T, Criger DA: Investigation anti- <br />                                                                                      biotic antibacterial agent cross-resistance target bac- <br /> Nikravesh, M&#195;&#179;nica Marta, Joshua Fink Rikard Dryselius help               teria homes antibacterial product users and <br /> comments.                                                                            nonusers. J Appl Microbiol 2003, 95:664-676. <br />                                                                                22.   McBain AJ, Bartolo RG, Catrenich CE, Charbonneau D, Ledder RG, <br />                                                                                      Price BB, Gilbert P: Exposure sink drain microcosms tri- <br /> References                                                                           closan: population dynamics antimicrobial susceptibility. <br /> 1.    Points consider production testing new drugs                  Appl Environ Microbiol 2003, 69:5433-5442. <br />       biologicals produced recombinant DNA technology                   23.   Beier RC, Duke SE, Ziprin RL, Harvey RB, Hume ME, Poole TL, Scott <br />       [http://www.fda.gov/cber/gdlns/ptcdna.htm]                                     HM, Highfield LD, Alali WQ, Andrews K, et al.: Antibiotic Dis- <br /> 2.    Validation growth-based rapid microbiological methods                       infectant Susceptibility Profiles Vancomycin-Resistant <br />       sterility testing cellular gene therapy products                    Enterococcus faecium (VRE) Isolated Community <br />       [http://www.fda.gov/cber/gdlns/stercgtp.htm]                                   Wastewater Texas. Bull Environ Contam Toxicol 2008. <br /> 3.    Titball RW, Sjostedt A, Pavelka MS Jr, Nano FE: Biosafety            24.   Opinion triclosan resistance             [http://www.europa.eu.int/ <br />       selectable markers. Ann N Y Acad Sci 2007, 1105:405-417.                       comm/food/fs/sc/ssc/out269_en.pdf] <br /> 4.    Fajardo A, Martinez-Martin N, Mercadillo M, Galan JC, Ghysels B,         25.   Drug approvals [http://www.fda.gov/cder/da/da1201.htm] <br />       Matthijs S, Cornelis P, Wiehlmann L, Tummler B, Baquero F, Martinez      26.   Lee C, Kim J, Shin SG, Hwang S: Absolute relative QPCR <br />       JL: The neglected intrinsic resistome bacterial pathogens.                  quantification plasmid copy number Escherichia coli. J <br />       PLoS ONE 2008, 3:e1619.                                                        Biotechnol 2006, 123:273-280. <br /> 5.    Gonzalez A, Davila G, Calva E: Cloning DNA sequence            27.   Livak KJ, Schmittgen TD: Analysis relative gene expression <br />       complements glutamine auxotrophy Saccharomyces cere-                        data using real-time quantitative PCR 2(-Delta Delta <br />       visiae. Gene 1985, 36:123-129.                                                 C(T)) Method. Methods 2001, 25:402-408. <br /> 6.    Cranenburgh RM, Hanak JA, Williams SG, Sherratt DJ: Escherichia          28.   Nilsson AI, Zorzet A, Kanth A, Dahlstrom S, Berg OG, Andersson DI: <br />       coli strains allow antibiotic-free plasmid selection                  Reducing fitness cost antibiotic resistance amplifi- <br />       maintenance repressor titration. Nucleic Acids Res 2001,                    cation initiator tRNA genes. Proc Natl Acad Sci USA 2006, <br />       29:E26.                                                                        103:6976-6981. <br /> 7.    Dryselius R, Nekhotiaeva N, Nielsen PE, Good L: Antibiotic-free          29.   Roth BL, Poot M, Yue ST, Millard PJ: Bacterial viability anti- <br />       bacterial strain selection using antisense peptide nucleic                     biotic susceptibility testing SYTOX green nucleic acid <br />       acid. Biotechniques 2003, 35:1060-1064.                                        stain. Appl Environ Microbiol 1997, 63:2421-2431. <br /> 8.    Mairhofer J, Pfaffenzeller I, Merz D, Grabherr R: A novel antibiotic     30.   Lenski RE, Simpson SC, Nguyen TT: Genetic analysis plas- <br />       free plasmid selection system: advances safe efficient                  mid-encoded, host genotype-specific enhancement bacte- <br />       DNA therapy. Biotechnol J 2008, 3:83-89.                                       rial fitness. J Bacteriol 1994, 176:3140-3147. <br /> 9.    Xu HH, Real L, Bailey MW: An array Escherichia coli clones            31.   Guzman LM, Belin D, Carson MJ, Beckwith J: Tight regulation, <br />       over-expressing essential proteins: new strategy identi-                  modulation, high-level expression vectors containing <br />       fying cellular targets potent antibacterial compounds. Bio-                 arabinose PBAD promoter. J Bacteriol 1995, 177:4121-4130. <br />       chem Biophys Res Commun 2006, 349:1250-1257.                             32.   Heath RJ, Rock CO: Regulation fatty acid elongation ini- <br /> 10.   Bergler H, Wallner P, Ebeling A, Leitinger B, Fuchsbichler S, Aschauer         tiation acyl-acyl carrier protein Escherichia coli. J Biol <br />       H, Kollenz G, Hogenauer G, Turnowsky F: Protein EnvM                    Chem 1996, 271:1833-1836. <br />       NADH-dependent enoyl-ACP reductase (FabI)                             33.   Ward WH, Holdgate GA, Rowsell S, McLean EG, Pauptit RA, Clayton <br />       Escherichia coli. J Biol Chem 1994, 269:5493-5496.                             E, Nichols WW, Colls JG, Minshull CA, Jude DA, et al.: Kinetic and <br /> 11.   Heath RJ, Yu YT, Shapiro MA, Olson E, Rock CO: Broad spectrum                  structural characteristics inhibition enoyl (acyl car- <br />       antimicrobial biocides target FabI component fatty                      rier protein) reductase triclosan.               Biochemistry 1999, <br />       acid synthesis. J Biol Chem 1998, 273:30316-30320.                             38:12514-12525. <br /> 12.   McMurry LM, Oethinger M, Levy SB: Triclosan targets lipid syn-           34.   Weitao T, Nordstrom K, Dasgupta S: Mutual suppression of <br />       thesis. Nature 1998, 394:531-532.                                              mukB seqA phenotypes arise opposing <br /> 13.   Heath RJ, Rubin JR, Holland DR, Zhang E, Snow ME, Rock CO: Mech-               influences Escherichia coli nucleoid structure. Mol <br />       anism triclosan inhibition bacterial fatty acid synthesis.               Microbiol 1999, 34:157-168. <br />       J Biol Chem 1999, 274:11110-11114.                                       35.   Yue TF, Dayton PG, Gutman AB: Mutual Suppression Uri- <br /> 14.   Russell AD: Biocide use antibiotic resistance: rele-                   cosuric Effects Sulfinpyrazone Salicylate: Study in <br />       vance laboratory findings clinical environmental                     Interactions Drugs. J Clin Invest 1963, 42:1330-1339. <br />       situations. Lancet Infect Dis 2003, 3:794-803.                           36.   Wild DG: Reversion erythromycin dependence in <br /> 15.   Panagakos FS, Volpe AR, Petrone ME, DeVizio W, Davies RM, Proskin              Escherichia coli: strains altered ribosomal sub-<span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> associa- <br />       HM: Advanced oral antibacterial/anti-inflammatory technol-                     tion ribosome assembly.                  J Gen Microbiol 1988, <br />       ogy: A comprehensive review clinical benefits tri-                 134:1251-1263. <br />       closan/copolymer/fluoride dentifrice.            J Clin Dent 2005,       37.   Goldstein F, Perutka J, Cuirolo A, Plata K, Faccone D, Morris J, Sour- <br />       16(Suppl):S1-19.                                                               nia A, Kitzis MD, Ly A, Archer G, Rosato AE: Identification and <br /> 16.   Wohlrab J, Jost G, Abeck D: Antiseptic efficacy low-dosed                 phenotypic characterization beta-lactam-dependent, <br />       topical triclosan/chlorhexidine combination therapy                         methicillin-resistant Staphylococcus aureus strain. Antimicrob <br />       atopic dermatitis. Skin Pharmacol Physiol 2007, 20:71-76.                      Agents Chemother 2007, 51:2514-2522. <br /> 17.   Aiello AE, Larson E: Antibacterial cleaning hygiene prod-            38.   Van Bambeke F, Chauvel M, Reynolds PE, Fraimow HS, Courvalin P: <br />       ucts emerging risk factor antibiotic resistance               Vancomycin-dependent Enterococcus faecalis clinical isolates <br />       community. Lancet Infect Dis 2003, 3:501-506.                                  revertant mutants. Antimicrob Agents Chemother 1999, <br /> 18.   Chuanchuen R, Beinlich K, Hoang TT, Becher A, Karkhoff-Schweizer               43:41-47. <br />       RR, Schweizer HP: Cross-resistance triclosan                 39.   Aiello AE, Marshall B, Levy SB, Della-Latta P, Larson E: Relationship <br />       antibiotics Pseudomonas aeruginosa mediated multid-                   triclosan susceptibilities bacteria isolated <br />       rug efflux pumps: exposure susceptible mutant strain                   hands community. Antimicrob Agents Chemother 2004, <br />       triclosan selects nfxB mutants overexpressing MexCD-OprJ.                      48:2973-2979. <br />       Antimicrob Agents Chemother 2001, 45:428-432. <br />  <br />  <br />                                                                                                                                             Page 8 9 <br />                                                                                                                     (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                  http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> 40.   Gilbert P, McBain A, Sreenivasan P: Common therapeutic <br />       approaches control oral biofilms: microbiological <br />       safety efficacy. Clin Microbiol Infect 2007, 13(Suppl 4):17-24. <br /> 41.   Orvos DR, Versteeg DJ, Inauen J, Capdevielle M, Rothenstein A, Cun- <br />       ningham V: Aquatic toxicity triclosan. Environ Toxicol Chem <br />       2002, 21:1338-1349. <br /> 42.   Federle TW, Kaiser SK, Nuck BA: Fate effects triclosan in <br />       activated sludge. Environ Toxicol Chem 2002, 21:1330-1337. <br /> 43.   Aranami K, Readman JW: Photolytic degradation triclosan in <br />       freshwater seawater. Chemosphere 2007, 66:1052-1056. <br /> 44.   Keating DH, Carey MR, Cronan JE Jr: The unmodified (apo) form <br />       Escherichia coli acyl carrier protein potent inhibitor of <br />       cell growth. J Biol Chem 1995, 270:22229-22235. <br /> 45.   Subrahmanyam S, Cronan JE Jr: Overproduction functional <br />       fatty acid biosynthetic enzyme blocks fatty acid synthesis in <br />       Escherichia coli. J Bacteriol 1998, 180:4596-4602. <br /> 46.   Heath RJ, Li J, Roland GE, Rock CO: Inhibition Staphyloco- <br />       ccus aureus NADPH-dependent enoyl-acyl carrier protein <br />       reductase triclosan hexachlorophene. J Biol Chem 2000, <br />       275:4654-4659. <br /> 47.   McMurry LM, McDermott PF, Levy SB: Genetic evidence that <br />       InhA Mycobacterium smegmatis target triclosan. <br />       Antimicrob Agents Chemother 1999, 43:711-713. <br /> 48.   Heerding DA, Chan G, DeWolf WE, Fosberry AP, Janson CA, Jawor- <br />       ski DD, McManus E, Miller WH, Moore TD, Payne DJ, et al.: 1,4-Dis- <br />       ubstituted imidazoles potential antibacterial agents <br />       functioning inhibitors enoyl acyl carrier protein reduct- <br />       ase (FabI). Bioorg Med Chem Lett 2001, 11:2061-2065. <br /> 49.   Lu H, Tonge PJ: Inhibitors FabI, Enzyme Drug Target in <br />       Bacterial Fatty Acid Biosynthesis Pathway. Acc Chem Res <br />       2008, 41:11-20. <br /> 50.   Heath RJ, Rock CO: A triclosan-resistant bacterial enzyme. <br />       Nature 2000, 406:145-146. <br /> 51.   Marrakchi H, Dewolf WE Jr, Quinn C, West <br /> </body></html>