<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Barnes et al. BMC Biochemistry 2010, 11:26 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br />     RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> Loss SIN3 transcriptional corepressor results <br /> Research article <br />  <br /> in aberrant mitochondrial function <br /> Valerie L Barnes1, Bethany S Strunk1,3, Icksoo Lee2, Maik H&#195;&#188;ttemann2 Lori A Pile*1 <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Background: SIN3 transcriptional repressor protein known regulate genes, including number those <br />     encode mitochondrial components. <br />     Results: By monitoring RNA levels, loss SIN3 Drosophila cultured cells results up-regulation not <br />     nuclear encoded mitochondrial genes, encoded mitochondrial genome. The up-regulation <br />     gene expression accompanied perturbation ATP levels SIN3-deficient cells, suggesting changes <br />     mitochondrial gene expression result altered mitochondrial activity. In support hypothesis SIN3 is <br />     necessary normal mitochondrial function, yeast sin3 <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>null mutants</span> exhibit poor growth non-fermentable <br />     carbon sources lower levels ATP reduced respiration rates. <br />     Conclusions: The findings yeast Drosophila SIN3 affect mitochondrial activity suggest evolutionarily <br />     conserved role SIN3 control cellular energy production. <br />  <br />  <br /> Background                                                                                  import mitochondrial factors mitochondrial <br /> Mitochondria dynamic organelles function                                          intermembrane space, inner membrane, the <br /> and total number responsive energy                                           mitochondrial matrix. The class comprised the <br /> demand cell. When cellular energy demand high,                                    enzymes additional proteins required replication <br /> expression genes encoding mitochondrial factors                                     translation mitochondrial genome are <br /> be up-regulated. Although mitochondrial genome                                          required mitochondrial biogenesis. Subsets of <br /> encodes components oxidative phosphorylation                                         mitochondrial genes shown increase or <br /> pathway, majority proteins required mito-                                    decrease expression variety tested condi- <br /> chondrial biogenesis function encoded                                        tions, suggesting groups genes co-regu- <br /> nuclear genome. While number transcription factors,                                    lated [3,4]. We previously determined nuclear <br /> including NRF1, NRF2, PPAR&#206;&#177; PPAR&#206;&#179;, required                                         encoded mitochondrial genes subject regulation by <br /> for activation nuclear encoded mitochondrial                                        SIN3 transcriptional corepressor [5]. Elimination of <br /> genes, little reported regarding repression                                     SIN3 Drosophila S2 tissue culture cells RNA inter- <br /> these genes [1].                                                                       ference (RNAi) led increase expression a <br />   Mitochondria contain important classes pro-                                      substantial number genes encoding mitochondrial pro- <br /> teins encoded nuclear genes [2]. The class                                         teins classes [5]. The SIN3-deficient cells were <br /> includes catalytic auxiliary proteins                                        increase mitochondrial mass. <br /> numerous enzyme systems. These include enzyme sys-                                            Although Drosophila SIN3 required viability dur- <br /> tems involved fatty acid oxidation, citric acid cycle,                               ing embryonic larval stages development, the <br /> oxidative phosphorylation, removal reactive                                      exact nature essential requirement SIN3 not <br /> oxygen species (ROS). A second class mitochondrial                                       understood [6-8]. SIN3 component SIN3 his- <br /> proteins includes components required protein                                           tone deacetylase complex, required general repression <br />                                                                                             transcription [9]. RPD3 catalytic component of <br /> * Correspondence: loripile@wayne.edu                                                        SIN3 complex [10-12]. Similar SIN3, loss Droso- <br /> 1Department Biological Sciences, Wayne State University, 5047 Gullen Mall,               phila RPD3 leads loss viability alterations <br /> Detroit, Michigan 48202, USA <br /> Full list author information available end article                      mitochondrial function gene activity [13-16]. Loss <br />                                             &#194;&#169; 2010 Barnes et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                             Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                             medium, provided original work properly cited. <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                      Page 2 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> of SIN3 RPD3 causes increase mitochon-         trans-factors putative link SIN3 or <br /> drial citrate synthase activity, indicating mitochon-     histone deacetylation. For example, mRpL19, EfTuM, fer- <br /> drial oxidative capacity affected following decreased       rochelatase, Tim10 shown bound by <br /> expression key SIN3 complex components [13].                members Max family network tran- <br />    Given finding SIN3 corepressor affects         scription factors [17]. Repression Max family <br /> expression nuclear genes encoding mitochondrial pro-        member Mad mediated interactions SIN3 <br /> teins, decided investigate relationship      [18]. Gapdh1 pyruvate kinase glycolytic genes <br /> between SIN3 mitochondrial gene activity func-         regulated HIF-1 [19]. HIF-1 associates with <br /> tion. We similar increase expression    FIH-1 mediates repression HIF-1 inter- <br /> the nuclear encoded mitochondrial genes,             action HDAC-1 [20]. Cytochrome c (Drosophila <br /> increase expression genes encoded mitochon-       CG2140) PHGPx regulated Sp1 that <br /> drial genome. This elevation increased tran-         shown interact SIN3 [21-23]. Thiolase <br /> scription number mitochondrial DNA genomes           acyl carnitine transporter (Drosophila CG3476) have <br /> is unchanged SIN3-deficient cells. Additionally,     regulated PPAR&#206;&#177; [24,25]. Multiple <br /> find increased expression genes encoding pro-      PPAR&#206;&#177;-regulated genes induced treatment with <br /> teins mitochondrial function accompanied         histone deacetylase inhibitors [26]. Thioredoxin <br /> alteration ATP levels. These results suggest       reductase activated following inhibition histone <br /> changes gene expression rise altered mitochon-      deacetylation [27]. <br /> drial function. In support findings yeast sin3 null     <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> extend previously published microarray data and <br /> mutants grow poorly media containing non-         determine relative difference expression, ana- <br /> fermentable carbon sources. The sin3 null strain          lyzed gene expression quantitative RT-PCR analysis. <br /> exhibits lowered levels ATP respiration rates rela-     To ensure changes gene expression due <br /> tive wild type. The data implicate SIN3 having role    loss SIN3 result target effect <br /> in control cellular energy production.                  [28], generated double stranded RNA (dsRNA) against <br />                                                                different regions SIN3 gene. The of <br /> Results                                                        SIN3 knockdown monitored Western blot analysis <br /> SIN3 regulates genes involved mitochondrial physiology      protein extracts prepared cells. SIN3 <br /> Previously published research authors          expression reduced similar levels cells incubated <br /> (LAP) revealed reduction SIN3 RNAi results       dsRNA targeting different sequences the <br /> up-regulation numerous genes involved multiple           SIN3 mRNA (Figure 1A). cDNA RNA isolated <br /> aspects mitochondrial biology, including activity       cell populations, control, SIN3-deficient <br /> biogenesis [5]. Using RNAi methodology cultured             1 (KD1), SIN3-deficient 2 (KD2), analyzed a <br /> Drosophila cells, tractable enables      quantitative real-time PCR (qPCR) assay. The fold change <br /> us induce group co-regulated genes loss      expression selected genes given Figure 1B. <br /> corepressor SIN3. This allows monitor         The majority genes selected (Additional file 1) were <br /> compare molecular events occur           previously identified SIN3 targets microarray analy- <br /> induction different classes mitochondrial genes      sis RNAi-induced SIN3-deficient cells [5]. We also <br /> including enzyme systems, protein import,         included genes, phosphoglycerate kinase aldo- <br /> and biogenesis. Because SIN3 regulates genes,          lase, did change expression the <br /> selected representative set analysis (Figure 1B, Addi-   microarray analysis, genes, microsomal <br /> tional file 1). Genes encoding mitochondrial protein      glutathione transferase 1 (Mgst1) CG11784 (glutathi- <br /> class represented up-regulated genes. We chose      transferase), showed down-regulation the <br /> at gene class. A number genes           microarray study. The increases gene expression fol- <br /> encoding proteins involved numerous enzyme sys-         lowing loss SIN3 using dsRNA (KD1) in <br /> tems selected. We able compare reg-       microrarray experiment ranged 1.6 4.1 fold, <br /> ulation genes distinct                   consistent microarray data [5]. Use dsRNA <br /> mitochondrial processes. We chose genes encoding pro-          targeting distinct sequence SIN3 (KD2) resulted <br /> teins individual energy pathways enzyme          up-regulation majority genes tested, although <br /> systems including fatty acid oxidation, citric acid cycle,     these, increase than <br /> oxidative phosphorylation, removal ROS.             1.5 fold. CG11784 (glutathione transferase), the <br /> Expression genes encoding proteins involved glycol-      down-regulated genes tested, showed decrease fol- <br /> ysis, takes place cytoplasm, produces         lowing SIN3 knockdown using dsRNAs. The <br /> pyruvate used mitochondria analyzed. A            gene, Mgst1, microarray be <br /> number selected genes known regulated         down-regulated, showed essentially change expres- <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                                      Page 3 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br />    A <br />  <br />  <br />  <br />  <br />                                       SIN3 <br />  <br />                                       E-tubulin <br />  <br />  <br />  <br />  <br />    B <br />                         Gapdh1 <br />            Phosphofructokinase <br />                         Thiolase <br />                 Pyruvate kinase <br />          Phosphoglycer kinase** <br />        I                Aldolase** <br />             Isocitrate dehydrog <br />        II         Malic enzyme <br />              Carn O-palm trans <br />        III       Acyl carn trans <br />            Cyt complex assemb <br />                  Cyt-c-proximal <br />        IV Electron transporter <br />                         mRpL19 <br />        V                  EfTuM <br />              Thioredoxin reduc <br />           Microsomal glut trans * <br />                        Catalase <br />               Glutathione perox <br />        VI      Glutathione trans * <br />        VII       Ferrochelatase <br />        VIII               Tim10 <br />  <br />                                         0.00      0.50       1.00       1.50      2.00       2.50      3.00       3.50       4.00      4.50       5.00 <br />                                                                       Fold Change (SIN3-deficient/Control) <br />  Figure 1 Expression nuclear encoded genes involved multiple mitochondrial processes up-regulated following loss SIN3. SIN3- <br />  deficient cells generated RNAi methodology using dsRNA generated different regions SIN3 transcript KD1 (black bars) KD2 <br />  (gray bars). (A) Western blot analysis cell extracts prepared control SIN3-deficient cells indicated. The blots probed an- <br />  tibody SIN3 &#206;&#178;-tubulin loading control. (B) qPCR performed using cDNA prepared control SIN3-deficient cells template <br />  primer pairs genes indicated Y-axis. Relative levels gene expression indicated. *Genes decreased expression mi- <br />  croarray. **Genes expression level change microarray. Genes grouped according process: I-glycolysis, II-citric acid cycle, III-fatty acid <br />  oxidation, IV-oxidative phosphorylation, V-mitochondrial biogenesis, VI-reactive oxygen species (ROS) pathways, VII-heme synthesis, VIII-trans- <br />  porter. Error bars represent standard error (n = 3-6 KD1 n = 3-5 KD2). p &#226;&#8240;&#164; 0.05 data. <br />  <br />  <br /> sion measured qPCR. The phosphoglycerate kinase                           confirm previous findings loss SIN3 results in <br /> and aldolase genes similar levels expression                      coordinated up-regulation genes encoding proteins <br /> control SIN3-deficient cells, consistent                           involved multiple mitochondrial functions. We saw <br /> microarray data. The results qPCR experiment                            similar patterns changes gene expression either <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                                Page 4 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> of SIN3 targeting dsRNAs, indicating the <br />                                                                        A <br /> observed effects likely loss SIN3 not <br />                                                                                                                   2.5 <br />  <br />  <br />  <br />  <br />                                                                            Fold Change (SIN3-deficient/Control) <br /> the result target effect. <br />                                                                                                                    2 <br /> Mitochondrial encoded genes up-regulated following <br /> loss SIN3                                                                                                      1.5 <br /> Because loss SIN3 resulted increased expression of <br /> nuclear encoded mitochondrial genes, examined                                                              1 <br /> expression genes encoded mitochondrial <br /> genome SIN3-deficient cells. We assayed RNA levels                                                             0.5 <br /> using qPCR analysis primers designed individu- <br />                                                                                                                    0 <br /> ally amplify distinct mitochondrial encoded genes. <br /> cDNA RNA isolated control SIN3- <br /> deficient S2 cells used template qPCR. Loss of <br /> SIN3 resulted increased expression mito- <br /> chondrial genes tested, mt: ATPase6, mt: CoI, mt: Cyt-b                B <br />                                                                                                                             2 <br /> and mt: ND1 (Figure 2A). Interestingly, increase in <br /> expression mitochondrial encoded genes                                                                                * * <br />  <br />  <br />  <br />  <br />                                                                                    Fold Change (SIN3- <br />                                                                                                                     1.5 <br />  <br />  <br />  <br />  <br />                                                                                     deficient/Control) <br /> SIN3-deficient cells similar level of <br /> increase expression nuclear encoded genes. The                                                                        * <br />                                                                                                                             1 <br /> changes range 1.4 2.0 fold. Mitochon- <br /> drial encoded genes regulated transcription factors <br /> that encoded nuclear genome. We determined                                                               0.5 <br /> whether nuclear encoded transcription factors, including <br /> TFAM, TFB1, TFB2, mitochondrial RNA polymerase                                                                              0 <br /> (mtRNA-pol) transcription termination factor <br /> DmTTF, showed change expression following loss of <br /> SIN3. While majority factors did a <br /> significant increase expression SIN3-deficient               C <br />                                                                                                                    1.5 <br /> cells, mtRNA-pol up-regulated 1.4 fold relative to <br />                                                                        (SIN3-deficient/Control) <br />                                                                        Relative Copy Number <br />  <br />  <br />  <br />  <br /> control cells (Figure 2B). Taken together, data suggest <br /> that increase levels tested mitochondrial                                                             1 <br /> transcripts possibly increase level of <br /> mtRNA-pol. The finding increases expression <br />                                                                                                                    0.5 <br /> of nuclear mitochondrial encoded genes in <br /> the range consistent numerous reports out- <br /> lining cross-talk nuclear mitochondrial                                                                 0 <br /> genomes [29]. <br />    The up-regulation mitochondrial ribosomal and <br /> translation elongation proteins suggests mitochon- <br /> drial biogenesis increased SIN3-deficient cells. In <br />                                                               Figure 2 Analysis mitochondrial gene expression genome <br /> addition, SIN3-deficient cells mitochondrial        copy number. Expression analysis mitochondrial encoded genes <br /> mass compared control cells [5]. Given findings,     (A) nuclear encoded mitochondrial transcription factors (B). Total <br /> we decided determine relative mitochondrial DNA        RNA isolated control SIN3-deficient cells (KD1, black bars, and <br /> copy number control SIN3-deficient cells. DNA          KD2, gray bars) used generate cDNA. qPCR performed us- <br /> extracted samples subjected           ing cDNA template primer pairs genes indicated <br />                                                               X-axis. Relative levels gene expression indicated. Error <br /> qPCR analysis using primer pairs specific mitochon-       bars represent standard error. A, n = 7 KD1 n = 5 KD2, p &lt; <br /> drial DNA. As control, determined relative      0.05 data. B, n = 4, *p &lt; 0.05. (C) Mitochondrial DNA copy number <br /> levels nuclear encoded genes. Based relative        unaltered following loss SIN3. DNA isolated control and <br /> levels mitochondrial genes, mito-       SIN3-deficient cells. qPCR performed using DNA template <br /> chondrial DNA copy number unchanged following             primer pairs nuclear mitochondrial genome se- <br />                                                               quences indicated X-axis. Pro-cyt-c-p amplifies pro- <br /> loss SIN3 (Figure 2C). The finding transcripts    moter region Cyt-c-p. Relative copy number indicated. The <br /> of mitochondrial encoded genes increased      nuclear gene glutathione transferase (CG11784) used normal- <br /> mitochondrial genome copy number unchanged sug-            ization. Error bars represent standard error (n = 3). <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                                                                          Page 5 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> gests that, SIN3-deficient cells, increase in <br />                                                                     A <br /> the mitochondrial gene transcription                                                         180 <br />  <br />  <br />  <br />  <br />                                                                     Relative ATP Levels (%) <br /> equivalent number genomes similar levels                                                         160 <br /> of transcription increased number mitochon-                                                          140 <br /> drial genomic templates.                                                                                        120 <br />                                                                                                                 100                                                     ** <br />                                                                                                                  80 <br /> Mitochondrial function altered SIN3-deficient cells                                                        60 <br /> Following loss SIN3, expression nuclear                                                            40 <br /> mitochondrial genes encoding proteins important gly-                                                         20 <br />                                                                                                                   0 <br /> colysis, citric acid cycle, fatty acid oxidation oxi- <br />  <br />  <br />  <br />  <br />                                                                                                                       Control <br />  <br />  <br />  <br />  <br />                                                                                                                                                             Control <br />                                                                                                                                     SIN3 KD1 <br />  <br />                                                                                                                                                 SIN3 KD2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          SIN3 KD1 <br />  <br />                                                                                                                                                                                        SIN3 KD2 <br /> dative phosphorylation increased (Figure 1B). <br /> Glycolysis, citric acid cycle fatty acid oxidation <br /> produce NADH FADH2 feed oxidative <br /> phosphorylation resulting ATP production. Since tran- <br /> scription mitochondrial encoded genes corre-                                                                            Normal                              Depleted <br /> lated ATP levels [30,31] asked ATP <br /> levels increased SIN3-deficient cells. The relative          B <br /> amount ATP present extracts control                                                              160 <br />                                                                                                                                                   * <br />                                                                               Relative Respiration Levels (%) <br /> SIN3-deficient S2 cells determined. We                                                           140 <br /> while ATP levels elevated SIN3-deficient cells rel-                                                     120 <br /> ative control samples, increase statisti-                                                    100                                                                     * <br />                                                                                                                  80                                                       * <br /> cally significant (Figure 3A). Proliferating cells grown in <br />                                                                                                                  60 <br /> culture rely heavily glycolysis                                                           40 <br /> than oxidative phosphorylation ATP production [32].                                                          20 <br /> In tissue culture cell lines, oxidative phosphoryla-                                                         0 <br /> tion shown inhibited excess glucose <br />                                                                                                                         Control <br />  <br />  <br />  <br />  <br />                                                                                                                                                             Control <br />                                                                                                                                      SIN3 KD1 <br />  <br />                                                                                                                                                  SIN3 KD2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          SIN3 KD1 <br />  <br />                                                                                                                                                                                        SIN3 KD2 <br /> [33]. In effort promote oxidative phosphorylation in <br /> Drosophila S2 cells, grew cells media diluted 1:4 <br /> with PBS (depleted) 24 hours prior ATP assay. <br /> Under conditions, SIN3-deficient cells exhibited a <br /> decrease ATP levels relative control cells (Figure                                                                      Normal                              Depleted <br /> 3A). These data suggest cells cultured <br />                                                                  Figure 3 ATP levels respiration affected S2 cells follow- <br /> under conditions favor ATP generation oxidative <br />                                                                  ing loss SIN3 <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> depleted conditions. (A) ATP levels S2 <br /> phosphorylation glycolysis, loss SIN3          cells grown normal depleted media SIN3. Error <br /> results lowered ATP production compared            bars represent standard error (n = 3-5). **p = 0.02. ATP present whole <br /> wild type cells.                                                 cell extracts prepared control SIN3-deficient cells quan- <br />   We monitored respiration rate SIN3-defi-           tified using bioluminescent method. (B) Relative respiration levels <br />                                                                  S2 cells grown normal depleted media SIN3 <br /> cient cells relative controls. Grown normal cul- <br />                                                                  (n = 3). Error bars represent standard error. *p &lt; 0.05. <br /> ture conditions, observed little change respiration <br /> using dsRNA target sequence increase using <br /> the sequence (Figure 3B). In depleted condition,      ing proteins function mitochondria, sin3 <br /> consistent lowered ATP levels, sets            mutant display altered ability grow media pre- <br /> SIN3-depleted cells lower respiration rates compared        pared non-fermentable carbon sources. To test our <br /> to control cells (Figure 3B). Taken together, ATP       hypothesis, monitored growth rates wild type <br /> respiration data suggest loss SIN3 leads changes     sin3 mutant cells glucose-containing fermentable <br /> in gene transcription affects mitochondrial function.      media (YPD) non-fermentable media. Glucose is <br />                                                                 preferred carbon source yeast, produce <br /> Yeast Sin3 mutants exhibit defects ATP production        ATP glycolysis fermentation. In absence <br /> respiration                                                     glucose presence carbon sources, <br /> Unlike Drosophila, Saccharomyces cerevisiae SIN3          including glycerol ethanol, ATP production only <br /> not essential gene [34]. Gene expression profiles         occur citric acid cycle oxidative phos- <br /> null mutants revealed yeast Sin3 (ySin3) regulates         phorylation requiring functional mitochondria [37]. The <br /> genes involved carbon metabolism [35,36]. We hypoth-         sin3 mutant displayed slightly delayed growth pheno- <br /> esized ySin3 regulates expression genes encod-       type cultured YPD (Figure 4A Additional file <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                               Page 6 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br />   A <br />                     18 <br />  <br />                     16 <br />  <br />                     14 <br />                                                                                                                   BY4741 (WT) - YPD <br />                     12 <br />       O.D. 550 nm <br />  <br />  <br />  <br />  <br />                                                                                                                   BY4741 (sin3) - YPD <br />                     10 <br />  <br />                      8                                                                                            BY4741 (WT) - YPE <br />  <br />                      6                                                                                            BY4741 (sin3) - YPE <br />  <br />                      4                                                                                            BY4741 (WT) - YPGE <br />  <br />                      2                                                                                            BY4741 (sin3) - YPGE <br />  <br />                      0 <br />                          0            10                    20                    30                    40 <br />                                                        Time (hr) <br />   B <br />                     12 <br />  <br />  <br />                     10 <br />                                                                                                                   BY4742 (WT) -YPD <br />  <br />                      8                                                                                            BY4742 (rpd3) -YPD <br />       O.D. 550 nm <br />  <br />  <br />  <br />  <br />                                                                                                                   BY4742 (WT) -YPE <br />                      6 <br />                                                                                                                   BY4742 (rpd3) -YPE <br />                      4 <br />                                                                                                                   BY4742 (WT) -YPGE <br />  <br />                      2                                                                                            BY4742 (rpd3) -YPGE <br />  <br />  <br />                      0 <br />                          0            10                   20                    30                    40 <br />                                                        Time (hr) <br />  Figure 4 S. cerevisiae SIN3 RPD3 critical growth media prepared non-fermentable carbon sources. Wild type (BY4741 WT), <br />  sin3 null mutant (BY4741 sin3) (A), wild type (BY4742 WT) rpd3 null mutant (BY4742 rpd3) (B) cells grown 5 ml YPD, washed twice with <br />  water, inoculated media containing glucose (YPD), ethanol (YPE) glycerol ethanol (YPGE) sole carbon sources incubated 30&#194;&#176;C. <br />  Absorbance 550 nm measured indicated times. Representative results single growth curve assays shown. A minimum three <br />  independent trials performed produced similar results. <br />  <br />  <br /> 2). This growth defect possibly cell cycle delay                data obtained glucose-containing media, sin3 <br /> in G2 phase cell cycle. SIN3 recently                  mutant grew poorly non-fermentable <br /> to important cell cycle progression percent-                  media tested (Figure 4A, Additional file 2). <br /> age cells G2 phase elevated null mutant                      As SIN3 previously identified global screens <br /> relative control strain [38]. In striking contrast                factors required respiratory activity [39-42], we <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                             Page 7 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> wished verify findings using sin3 mutant having a <br /> different genetic background. Two additional indepen-               A <br />                                                                                           2500 <br /> dent sin3 mutants grew poorly non-fermentable <br /> carbon sources compared wild type counter-                                    2000 <br />  <br />  <br />  <br />  <br />                                                                       ATP/PM/mL protein <br /> part strains (Additional file 3). Very poor growth the <br />                                                                                           1500 <br /> mutants noted strains spread <br /> onto solid agar plates containing respective carbon                                   1000 <br /> sources (Additional files 4, 5) Given poor growth on <br /> non-fermentable carbon sources, important verify                                      500 <br /> that sin3 mutant strains petite. To verify                                           0 <br /> the strains contain mitochondrial DNA, cells were <br />  <br />  <br />  <br />  <br />                                                                                                             WT <br />  <br />                                                                                                             WT <br />  <br />                                                                                                             WT <br />  <br />                                                                                                             WT <br />  <br />                                                                                                             WT <br />  <br />                                                                                                             WT <br />                                                                                                            sin3 <br />  <br />                                                                                                            sin3 <br />  <br />                                                                                                            sin3 <br />  <br />                                                                                                            sin3 <br />  <br />                                                                                                            sin3 <br />  <br />                                                                                                            sin3 <br /> stained DAPI. Mitochondrial genomes were <br /> observed mutants indicating defective growth <br /> on non-fermentable carbon strains                                                        YPD YPE YPGE YPD YPE YPGE <br /> becoming rhoo (Additional file 6). <br />    We tested deletion yRpd3, cata-                                                   Log Phase Growth Stationary Phase <br /> lytic component SIN3 histone deacetyase complex,                                                                          Growth <br /> would affect growth non-fermentable carbon. Like <br /> SIN3, RPD3 non-essential gene [43]. As expected,            B <br /> determined yRpd3 dispensable growth                       O2 consumption uM/min   140 <br /> glucose-containing YPD (Figure 4B, Additional file 7).                                               120 <br /> Similar sin3 mutants, rpd3 mutant grew                                               100 <br />                                                                                 mg protein <br />  <br /> poorly liquid media solid agar plates containing                                            80 <br /> non-fermentable carbon sources (Figure 4B, Additional                                                 60 <br /> file 7). Staining mitochondrial DNA rpd3 mutant                                             40 <br /> verified strain rhoo (Additional file 6).                                            20 <br /> Taken together, results clearly demonstrate                                                 0 <br />                                                                                                             sin3 <br />  <br />                                                                                                             sin3 <br />  <br />                                                                                                             sin3 <br />  <br />                                                                                                             sin3 <br />  <br />                                                                                                             sin3 <br />  <br />                                                                                                             sin3 <br />                                                                                                              WT <br />  <br />                                                                                                              WT <br />  <br />                                                                                                              WT <br />  <br />                                                                                                              WT <br />  <br />                                                                                                              WT <br />  <br />                                                                                                              WT <br /> yeast SIN3 RPD3 critical mitochondrial func- <br /> tion. <br />    To analyze yeast sin3 mutant mitochon-                                                   YPD YPE YPGE YPD YPE YPGE <br /> drial activity, monitored ATP levels respiration <br /> rates wild type sin3 strains. We performed these <br />                                                                                                            Log Phase Growth Stationary Phase <br /> assays using cells grown glucose non-fermentable                                                                          Growth <br /> media. Samples collected cells log and <br /> stationary phases growth. During log phase, yeast pro-          Figure 5 ATP levels respiration decreased yeast sin3 <br /> duce ATP primarily fermentation. As glucose             null mutants. (A) ATP levels yeast grown YPD non-ferment- <br /> the media exhausted, cells adapted               able carbon sources measured ATP/&#206;&#188;M/mg protein. Error bars rep- <br /> mitochondrial respiration, metabolic process known            resent standard error (n = 3). ATP present cell extracts <br />                                                                    prepared control SIN3-deficient yeast quantified using <br /> the diauxic shift [44]. The respiration rate wild type          bioluminescent method. (B) Respiration levels yeast grown in <br /> cells isolated stationary phase higher relative      YPD non-fermentable carbon sources measured O2 consump- <br /> rate cells log phase growth, indi-    tion &#206;&#188;M/min mg protein. Error bars represent standard devia- <br /> cating switch aerobic respiration seen               tion. <br /> diauxic shift (Figure 5B). In media phases <br /> of growth, sin3 null mutant showed lower levels            Discussion <br /> ATP (Figure 5A). Consistent lowered levels            We previously established expression numer- <br /> ATP, respiration rates decreased sin3        ous nuclear encoded mitochondrial genes up-regu- <br /> mutants relative wild type cells (Figure 5B). The find-        lated Drosophila tissue culture cells following loss <br /> ings yeast sin3 mutants unable grow non-           corepressor SIN3 [5]. In research described this <br /> fermentable carbon sources lowered ATP               current work, determined changes in <br /> reduced oxygen consumption indicate ySin3                 gene expression following loss SIN3 unlikely to <br /> important mitochondrial function. The observations            target effects SIN3 RNAi. Targeting different <br /> that yeast Drosophila SIN3 affect mitochondrial          regions SIN3 mRNA results similar changes in <br /> activity imply evolutionarily conserved role SIN3          gene expression, indicating changes likely due <br /> in control cellular energy production. <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                        Page 8 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> to loss SIN3 effect gene      SIN3 histone deactylase complex [9]. One major <br /> contains short similar sequence. In addition          reported difference yeast metazoan SIN3 is <br /> changes expression nuclear encoded genes, loss       requirement gene viability. ySin3 dis- <br /> of SIN3 results up-regulation mitochondrial           pensable growth metazoan SIN3 essential <br /> encoded genes. Although observe changes gene             gene [9,45]. Analysis conditional knockdown Droso- <br /> expression, mitochondrial genome copy             phila SIN3 mutants suggests SIN3-deficiency leads <br /> number unaltered. The changes gene expression            death defects cell cycle progression rather <br /> prompted examine mitochondrial function SIN3-         alteration specific developmental path- <br /> deficient cells. SIN3-deficient cells changes ATP      way [8]. In work, ySin3 required <br /> levels respiration rates, suggesting increases    cell growth mitochondrial function necessary <br /> in gene expression result altered oxidative phosphory-      ATP production. It possible that, than <br /> lation capacity. A necessary role SIN3 normal mito-     essential specific process development, <br /> chondrial function supported finding yeast      metazoan requirement SIN3 require- <br /> sin3 mutants unable grow non-fermentable car-        ment normal mitochondrial activity. We hypothesize <br /> bon sources, conditions require active mitochon-         loss SIN3 leads defect mitochondrial func- <br /> dria ATP production.                                       tion turn leads defect cell cycle progression. <br />    Consistent numerous published reports,         Links cell cycle control mitochondrial activ- <br /> determined loss ySin3 dispensable fermenta-     ity established. Myc: Max heterodimers positively <br /> tive growth [9]. In contrast, ySin3 critical growth     regulate cell cycle Rb-E2F complexes repress <br /> when respiration required (Figure 4A, Additional files      genes necessary cell cycle progression [46,47]. Both <br /> 2, 3, 4 5). Genome wide studies conducted        sets factors linked regulation NRF-1, a <br /> to identify yeast mutant strains ability      key transcription activator nuclear encoded mitochon- <br /> grow YPD exhibit reduced growth non-fer-             drial genes [48,49]. SIN3 linked Max <br /> mentable carbon sources failed identify SIN3 [39-       transcription network regulation Rb, support- <br /> 42]. Several possible explanations account false       ing hypothesis SIN3 regulation mitochondrial <br /> negatives global screens, discussed      biogenesis activity linked cell cycle control <br /> by authors global screen studies [40].       [18,50]. It recently shown mammalian cells that <br /> First, methods analysis growth scoring differ.      changes mitochondrial morphology crucial cell <br /> Second, phenotypic plasticity             cycle control. At G1-S checkpoint mitochondria are <br /> mutants different strain backgrounds. Third, yeast      organized large fused network higher mitochon- <br /> deletion collections known accumulate errors,       drial membrane potentials, increased ATP levels and <br /> it well-established deletion strains     increased respiration rates, stages the <br /> tested ensure deletion correct. Obviously,      cell cycle, mitochondria predominantly fission <br /> this carrying global screens. In       state [51]. Lower ATP levels respiration rates caused <br /> addition, global study Steinmetz et al. [39],    SIN3 knockdown (Figure 3, depleted condition) may <br /> sin3 mutant did grow cultured      interfere cell cycle progression G1-S tran- <br /> different carbon sources assay conditions            sition. <br /> tested, including YPD. Pools mutants cultured         In addition role gene regulation, SIN3 might <br /> various media growth individual mutant strains       directly regulate energy metabolism participant in <br /> the pool compared average growth culture.        cell signaling. There clear evidence signaling path- <br /> It possible sin3 strain able grow    ways regulate mitochondrial respiration energy lev- <br /> any assay conditions pools mutants. All      els, e.g., cAMP inflammatory pathways lead to <br /> of potential reasons SIN3      phosphorylation cytochrome c oxidase, terminal <br /> identified global screen genes required respi-    enzyme electron transport chain, followed by <br /> ratory activity. In study demonstrate that,       changes respiration ATP levels [52,53]. SIN3 might <br /> assayed individual strain, sin3 mutant grows         exert stimulatory signal required normal mitochon- <br /> very poorly non-fermentable media, indicating          drial function. The observation reduction SIN3 <br /> ySin3 required cells forced respire. Consis-    results decreased ATP levels respiration perhaps <br /> tent data, sin3 mutants lower levels ATP     surprising light finding expression many <br /> and decreased respiration rates.                               genes encoding proteins mitochondrial function are <br />    Work multiple laboratories indicated          up-regulated. Up-regulation nuclear encoded mito- <br /> there similarities yeast metazoan         chondrial genes SIN3 knockdown be <br /> SIN3, including fact SIN3 acts global regula-    compensatory response increase mitochon- <br /> tor transcription functions context   drial component dysfunctional energy metabo- <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                     Page 9 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> lism, seen yeast adapt mitochondrial        mocycler. Primers used listed Additional file 1. <br /> respiration dependence glycolysis (Figure 4,        Primers designed using PRIMER3 primer design <br /> YPD). Alternatively, mis-coordination mitochondrial         software [57], primers TFAM, TFB1, TFB2, <br /> gene expression SIN3-deficient cells result      TTF, CG4464 described previously by <br /> the observed altered mitochondrial function loss       Dubessay, et al. [58]. TAF1 used normalize cDNA <br /> of SIN3 leads increase expression     amounts comparative analysis. <br /> all genes involved mitochondrial processes [5]. As has <br /> been previously suggested, imbalance level        Western blot analysis <br /> the proteins present complexes important energy         Western blot analysis performed accordance with <br /> production result mitochondrial malfunction             standard protocols [59]. To prepare cell extracts, <br /> [48,54]. Finally, key links regulation acetyla-     cells pelleted centifugation lysed in <br /> tion metabolism highlighted recently pub-           Laemmli sample buffer (Bio-Rad) concentration of <br /> lished data indicating ATP-citrate lyase important     1.5 &#195;&#8212; 104 cell/&#206;&#188;l buffer. Protein concentration was <br /> for maintenance histone acetylation levels growing       determined using DC protein assay reagent (Bio-Rad) <br /> cells [55]. It possible SIN3 knockdown affects         according manufacturer&apos;s protocol. Protein extract <br /> acetylation resulting changes gene expres-      (15 20 &#206;&#188;g) fractionated sodium dodecyl sulfate <br /> sion altered pools key metabolites          (SDS) 8% polyacrylamide gel electrophoresis (PAGE), <br /> inappropriate response mitochondria            transferred polyvinylidene difluoride membrane, <br /> abnormal signals. It investi-   (PVDF; Pall), probed immunoglobulin G (IgG) <br /> gate relationship histone acetylation          purified polyclonal rabbit antibodies SIN3 <br /> mitochondrial function better understand control         (1:2000) [60], followed donkey anti-rabbit horseradish <br /> cellular metabolism.                                           peroxidase-conjugated IgG (1:3000) (GE Healthcare), and <br />                                                                detected enhanced chemiluminescence reagents <br /> Conclusion                                                     (GE Healthcare). The blots subsequently probed <br /> In summary, mitochondrial activity affected follow-        monoclonal mouse antibody &#206;&#178;-tubulin <br /> ing loss SIN3 Drosophila cultured cells      (1:1000) (Sigma), followed sheep anti-mouse horserad- <br /> yeast. This abnormal mitochondrial function lead        ish peroxidase-conjugated IgG (1:3000) (GE Healthcare) <br /> cell cycle defects affect cell organism viabil-   loading control. <br /> ity. <br />                                                                Genome copy number determination <br />                                                                Genomic DNA prepared tissue culture cells <br /> Methods <br />                                                                according standard protocol [61]. The isolated DNA <br /> Cell culture <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br />                                                                used template qPCR assay. The analysis was <br /> Drosophila S2 cell culture RNAi carried as <br />                                                                performed using ABsolute SYBR Green ROX master mix <br /> previously described [56]. RNAi performed using <br />                                                                (Fisher Scientific) carried Stratagene <br /> dsRNA corresponding distinct regions SIN3 <br />                                                                Mx3005P real-time thermocycler. Primers used amplified <br /> mRNA. SIN3 RNAi knockdown #1 dsRNA targeted <br />                                                                nuclear mitochondrial genomic regions <br /> against sequence encoded exon 3. The construction <br />                                                                listed Additional file 1. CG11784 (glutathione <br /> of targeting sequence #1 pCRII-Topo vector (Invitro- <br />                                                                transferase) genomic sequence used normalize the <br /> gen) described [56]. SIN3 RNAi knockdown #2 <br />                                                                DNA amounts reaction. <br /> dsRNA targeted exon 4. The sequences the <br /> pCRII-Topo vector generated using following           Determination cellular ATP levels <br /> primer set (oriented 5&apos; 3&apos;) TCACCAAGTTATGGCAT-              Drosophila S2 yeast cells grown culture yield <br /> TCC CTTAGGTACTGTGGACTGCG. Mock RNAi                        approximately 1 &#195;&#8212; 107 cells. Cells transferred 15 <br /> treated cells served control.                           ml Falcon tubes, pelleted quick spin 805 &#195;&#8212; g, rinsed <br /> RT-PCR analysis <br />                                                                1.5 ml PBS, re-pelleted 2 ml microcentri- <br />                                                                fuge tubes 16,100 &#195;&#8212; g 15 sec. After discarding PBS, <br /> Total RNA extracted tissue culture cells using <br />                                                                cells quick frozen liquid nitrogen stored - <br /> the RNeasy purification kit (Qiagen). cDNA gener- <br />                                                                80&#194;&#176;C measurement. Cell pellets supplemented <br /> ated total RNA using ImProm-II Reverse Tran- <br />                                                                300 &#206;&#188;l boiling buffer (100 mM Tris, pH 7.75, 4 <br /> scription System (Promega) random hexamers. The <br />                                                                mM EDTA) immediately transferred boiling <br /> cDNA used template quantitative real-time <br />                                                                water bath lysed 2 min, ice, solubilized <br /> PCR (qPCR) assay. The analysis performed using <br />                                                                ultrasonication. ATP concentration determined <br /> ABsolute SYBR Green ROX master mix (Fisher Scientific) <br />                                                                using ATP bioluminescence assay kit HS II (Roche <br /> and carried Stratagene Mx3005P real-time ther- <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                  Page 10 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br />  <br /> Applied Science) according manufacturer&apos;s proto-       Rpd3, TTCGGCATCCCTAGGTGGTGATT TCT- <br /> col. Data standardized protein concentration,     GCTGTCGTGTTACAGTGTGGT. <br /> as determined DC protein assay kit (Bio-           Strains used: <br /> Rad). For protein determination yeast samples, 100       BY4741 (WT) MAT a, 3&#206;&#8221;1, leu 2&#206;&#8221;0, met 15&#206;&#8221;0, <br /> &#206;&#188;l 20% SDS solution added ultrasonicated         ura3&#206;&#8221;0 <br /> again completely solubilize proteins.                        BY4741 sin3&#206;&#8221; MAT a, 3&#206;&#8221;1, leu 2&#206;&#8221;0, met 15&#206;&#8221;0, <br />                                                               ura3&#206;&#8221;0, sin3&#206;&#8221;::KanMX4 <br /> Respiration assay                                               FY23 (WT) MAT a, ura3-52, trp1&#206;&#8221;63, leu2&#206;&#8221;1 <br /> Cells collected protein concentration deter-       FY23 sin3&#206;&#8221; MAT a, ura3-52, trp1&#206;&#8221;63, leu2&#206;&#8221;1, <br /> mined DC protein assay kit (Bio-Rad). Cells     sin3&#206;&#8221;::KanMX4 <br /> then washed PBS, resuspended Schneider&apos;s            BY4733 (WT) MAT a, his3&#206;&#8221;200, trp1&#206;&#8221;63, leu2&#206;&#8221;0, <br /> Drosophila media S2 cells (GIBCO, Invitrogen)          met15&#206;&#8221;0, ura3&#206;&#8221;0 <br /> complex media liquid yeast cultures containing yeast        BY4733 sin3&#206;&#8221; MAT a, his3&#206;&#8221;200, trp1&#206;&#8221;63, leu2&#206;&#8221;0, <br /> extract (1%), peptone (2%), glucose (2%), etha-    met15&#206;&#8221;0, ura3&#206;&#8221;0, sin3&#206;&#8221;::KanMX4 <br /> nol (1%) ethanol (1%) plus glycerol (2%) (Difco, Fisher      BY4742 (WT) MAT&#206;&#8221;, his3&#206;&#8221;1, leu2&#206;&#8221;0, lys2&#206;&#8221;0, ura3&#206;&#8221;0 <br /> Scientific) 1 mg/ml final protein concentration. Oxy-        BY4742 rpd3&#206;&#8221; MAT&#206;&#8221;, his3&#206;&#8221;1, leu2&#206;&#8221;0, lys2&#206;&#8221;0, ura3&#206;&#8221;0, <br /> gen consumption cells measured closed         rpd3&#206;&#8221;::KanMX4 <br /> 200 &#206;&#188;l chamber equipped micro Clark-type oxygen <br /> electrode (Oxygraph system, Hansatech) 25&#194;&#176;C ana-       Yeast DAPI staining <br /> lyzed Oxygraph software. 500 &#206;&#188;M KCN added            To stain nuclear mitochondrial DNA, yeast cells were <br /> at end measurement inhibit cyto-           cultured mid-log phase, fixed 70% ethanol at <br /> chrome c oxidase. Non-cytochrome c oxidase-based res-         room temperature 30 min, washed ddH2O, <br /> piration subtracted determine oxygen                   stained 500 ng/ml DAPI (Sigma) 5 min. All <br /> consumption rates. Respiration defined oxygen          microscopy performed using Zeiss Axioscope 2 flu- <br /> consumed (&#206;&#188;M/min mg protein).                             orescence microscope. Images acquired using <br />                                                               Olympus QCapture2 software (1000&#195;&#8212; magnification). <br /> Yeast growth assay <br /> Glucose, yeast extract peptone obtained         Additional material <br /> Difco. Glycerol obtained Fisher Scientific. All <br /> chemicals used reagent grade better. Yeast cul-        Additional file 1 Primer pairs used study. This table lists all <br /> tures maintained 15% glycerol -80&#194;&#176;C long        primer pairs used qPCR gene expression analysis. <br /> term storage YPD plates short term storage.         Additional file 2 sin3 mutant cells grow poorly solid agar <br />                                                                plates containing non-fermentable carbon sources. This file contains <br /> Complex media liquid cultures contained yeast              images showing growth wild type sin3 mutant solid agar <br /> extract (1%), peptone (2%), glucose (2%), etha-     plates containing YPD non-fermentable carbon sources. <br /> nol (1%) ethanol (1%) plus glycerol (2%). For liquid    Additional file 3 Additional S. cerevisiae strains demonstrate ySin3 is <br />                                                                critical growth media prepared non-fermentable carbon <br /> growth curve assay, 5 ml YPD liquid media inocu-        sources. This file shows growth curves additional S. cerevisiae <br /> lated single colony solid agar YPD plate     strains, differing genetic backgrounds, media prepared YPD or <br /> the yeast strains grown stationary phase over-      non-fermentable carbon sources. <br /> night culture. The volume stationary culture needed      Additional file 4 A second S. cerevisiae strain demonstrates ySin3 is <br />                                                                critical growth media prepared non-fermentable carbon <br /> make 25 ml culture starting 0.05-0.06 OD (at 550 nm)        sources. This file contains images showing growth wild type a <br /> was transferred 1.5 ml tube cells pelleted     sin3 mutant different genetic background solid agar plates containing <br /> centrifugation. The yeast pellet washed 2 times       YPD non-fermentable carbon sources. <br />                                                                Additional file 5 A S. cerevisiae strain demonstrates ySin3 criti- <br /> 1 ml ddH2O resuspended 25 ml media           cal growth media prepared non-fermentable carbon <br /> containing different carbon sources. The cultures          sources. This file contains images showing growth wild type a <br /> were incubated 30&#194;&#176;C shaking 230-270 rpm. The        sin3 mutant different genetic background solid agar plates containing <br />                                                                YPD non-fermentable carbon sources. <br /> absorbance 550 nm determined indicated <br />                                                                Additional file 6 Mitochondrial genomes observed sin3 and <br /> times follow growth culture. For plate       rpd3 mutants. This file contains images DAPI staining nuclear and <br /> assay, strains grown stationary phase above,        mitochondrial DNA wild type sin3 rpd3 mutant cells. <br /> counted hemocytometer, washed 2 times 1 ml           Additional file 7 rpd3 mutant cells grow poorly solid agar <br />                                                                plates containing non-fermentable carbon sources. This file contains <br /> of ddH2O 300 cells spread respective agar        images showing growth wild type rpd3 mutant solid agar <br /> plates. The plates incubated 30&#194;&#176;C 3-6 days.        plates containing YPD non-fermentable carbon sources. <br /> The deletion mutants verified PCR using fol- <br /> lowing primers (oriented 5&apos; 3&apos;): Sin3, ATCACCC             Authors&apos; contributions <br />                                                               VLB carried RT-PCR, Western blot, growth curve DAPI staining <br /> TCGCCCTAAATC CCGGTGTGTCTATTGCCTG; <br />                                                               assays prepared samples respiration ATP analysis. BSS performed <br />  Barnes et al. BMC Biochemistry 2010, 11:26                                                                                                                                                                                                                                                                                                             Page 11 12 <br /> http://www.biomedcentral.com/1471-2091/11/26 <br />  <br />  <br />  <br /> RT-PCR genome copy number assays. IL analyzed respiration levels. MH                                                                                                                                                                                                                 16. Mottus R, Sobel RE, Grigliatti TA: Mutational analysis histone <br /> conceived respiration ATP assays, analyzed ATP levels, helped                                                                                                                                                                                                              deacetylase Drosophila melanogaster: missense mutations suppress <br /> draft manuscript. LAP conceived study participated design                                                                                                                                                                                                              gene silencing associated position effect variegation. Genetics <br /> and coordination drafted manuscript. All authors read approved                                                                                                                                                                                                                   2000, 154(2):657-668. <br /> the final manuscript.                                                                                                                                                                                                                                                                    17. Orian A, van Steensel B, Delrow J, Bussemaker HJ, Li L, Sawado T, Williams <br />                                                                                                                                                                                                                                                                                              E, Loo LW, Cowley SM, Yost C, et al.: Genomic binding Drosophila <br /> Acknowledgements                                                                                                                                                                                                                                                                             Myc, Max, Mad/Mnt transcription factor network. Genes Dev 2003, <br /> We thank Dr. Miriam Greenberg suggestion initiate studies                                                                                                                                                                                                                  17(9):1101-1114. <br /> yeast mutants, providing yeast strains thoughtful discussions                                                                                                                                                                                                                18. Ayer DE, Lawrence QA, Eisenman RN: Mad-Max transcriptional <br /> throughout course work. We thank Amit Joshi guidance                                                                                                                                                                                                                  repression mediated ternary complex formation mammalian <br /> the yeast growth assay. We thank Dr. Athar Ansari providing yeast                                                                                                                                                                                                                   homologs yeast repressor Sin3. Cell 1995, 80(5):767-776. <br /> strains. The work supported Research Scholar Grant Ameri-                                                                                                                                                                                                              19. Semenza GL: Regulation mammalian O2 homeostasis hypoxia- <br /> can Cancer Society LAP Center Molecular Medicine Genet-                                                                                                                                                                                                                   inducible factor 1. Annu Rev Cell Dev Biol 1999, 15:551-578. <br /> ics, Wayne State University School Medicine MH.                                                                                                                                                                                                                                    20. Mahon PC, Hirota K, Semenza GL: FIH-1: novel protein interacts <br />                                                                                                                                                                                                                                                                                              HIF-1alpha VHL mediate repression HIF-1 transcriptional <br /> Author Details                                                                                                                                                                                                                                                                               activity. Genes Dev 2001, 15(20):2675-2686. <br /> 1Department Biological Sciences, Wayne State University, 5047 Gullen Mall,                                                                                                                                                                                                            21. Evans MJ, Scarpulla RC: Interaction nuclear factors multiple sites <br />                                                                                                                                                                                                                                                                                              somatic cytochrome c promoter. Characterization upstream <br /> Detroit, Michigan 48202, USA, 2Center Molecular Medicine Genetics, <br />                                                                                                                                                                                                                                                                                              NRF-1, ATF, intron Sp1 recognition sequences. J Biol Chem 1989, <br /> Wayne State University School Medicine, 540 E. Canfield, Detroit, Michigan <br />                                                                                                                                                                                                                                                                                              264(24):14361-14368. <br /> 48201, USA 3Chemical Biology, University Michigan, 930 N. University <br />                                                                                                                                                                                                                                                                                          22. Zhang Y, Dufau ML: Silencing transcription human luteinizing <br /> Ave. Rm. 4250, Ann Arbor, Michigan 48109, USA <br />                                                                                                                                                                                                                                                                                              hormone receptor gene histone deacetylase-mSin3A complex. J <br /> Received: 5 May 2010 Accepted: 9 July 2010                                                                                                                                                                                                                                                   Biol Chem 2002, 277(36):33431-33438. <br /> Published: 9 July 2010                                                                                                                                                                                                                                                                   23. Ufer C, Borchert A, Kuhn H: Functional characterization cis- trans- <br /> BMC <br /> &#194;&#169; 2010 <br /> This is <br />      article <br />       Biochemistry <br />          Barnes <br />         Open  et al; <br />              available <br />                   Access <br />                      2010, <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span>: <br />                         licensee <br />                            articleBioMed <br />                                    distributed <br />                                            Central Ltd.the terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />                              11:26http://www.biomedcentral.com/1471-2091/11/26 <br />                                                under <br />                                                                                                                                                                                                                                                                                              regulatory elements involved expression phospholipid <br />                                                                                                                                                                                                                                                                                              hydroperoxide glutathione peroxidase. Nucleic Acids Res 2003, <br /> References <br />                                                                                                                                                                                                                                                                                              31(15):4293-4303. <br /> 1. Scarpulla RC: Nuclear control respiratory chain expression nuclear <br />                                                                                                                                                                                                                                                                                          24. Puigserver P, Spiegelman BM: Peroxisome proliferator-activated <br />     respiratory factors PGC-1-related coactivator. Ann N Y Acad Sci <br />                                                                                                                                                                                                                                                                                              receptor-gamma coactivator 1 alpha (PGC-1 alpha: transcriptional <br />     2008, 1147:321-334. <br />                                                                                                                                                                                                                                                                                              coactivator metabolic regulator. Endocr Rev 2003, 24(1):78-90. <br /> 2. Scarpulla RC: Nuclear activators coactivators mammalian <br />                                                                                                                                                                                                                                                                                          25. Mandard S, Muller M, Kersten S: Peroxisome proliferator-activated <br />     mitochondrial biogenesis. Biochim Biophys Acta 2002, 1576(1-2):1-14. <br />                                                                                                                                                                                                                                                                                              receptor alpha target genes. Cell Mol Life Sci 2004, 61(4):393-416. <br /> 3. Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, Armour CD, <br />                                                                                                                                                                                                                                                                                          26. Jones DC, Ding X, Daynes RA: Nuclear receptor peroxisome proliferator- <br />     Bennett HA, Coffey E, Dai H, He YD, et al.: Functional discovery a <br />                                                                                                                                                                                                                                                                                              activated receptor alpha (PPARalpha) expressed resting murine <br />     compendium expression profiles. Cell 2000, 102(1):109-126. <br />                                                                                                                                                                                                                                                                                              lymphocytes. The PPARalpha T B lymphocytes both <br /> 4. Mootha VK, Bunkenborg J, Olsen JV, Hjerrild M, Wisniewski JR, Stahl E, <br />                                                                                                                                                                                                                                                                                              transactivation transrepression competent. J Biol Chem 2002, <br />     Bolouri MS, Ray HN, Sihag S, Kamal M, et al.: Integrated analysis protein <br />                                                                                                                                                                                                                                                                                              277(9):6838-6845. <br />     composition, tissue diversity, gene regulation mouse <br />                                                                                                                                                                                                                                                                                          27. Deroo BJ, Hewitt SC, Peddada SD, Korach KS: Estradiol regulates the <br />     mitochondria. Cell 2003, 115(5):629-640. <br />                                                                                                                                                                                                                                                                                              thioredoxin antioxidant mouse uterus. Endocrinology <br /> 5. Pile LA, Spellman PT, Katzenberger RJ, Wassarman DA: The SIN3 <br />                                                                                                                                                                                                                                                                                              2004, 145(12):5485-5492. <br />     deacetylase complex represses genes encoding mitochondrial <br />                                                                                                                                                                                                                                                                                          28. Ma Y, Creanga A, Lum L, Beachy PA: Prevalence off-target effects in <br />     proteins: implications regulation energy metabolism. J Biol <br />                                                                                                                                                                                                                                                                                              Drosophila RNA interference screens. Nature 2006, 443(7109):359-363. <br />     Chem 2003, 278(39):37840-37848. <br />                                                                                                                                                                                                                                                                                          29. Garesse R, Vallejo CG: Animal mitochondrial biogenesis function: a <br /> 6. Pennetta G, Pauli D: The Drosophila Sin3 gene encodes widely <br />                                                                                                                                                                                                                                                                                              regulatory cross-talk genomes. Gene 2001, 263(1-2):1-16. <br />     distributed transcription factor essential embryonic viability. <br />                                                                                                                                                                                                                                                                                          30. DasGupta SF, Rapoport SI, Gerschenson M, Murphy E, Fiskum G, Russell SJ, <br />     Development genes evolution 1998, 208(9):531-536. <br />                                                                                                                                                                                                                                                                                              Chandrasekaran K: ATP synthesis coupled rat liver mitochondrial <br /> 7. Neufeld TP, Tang AH, Rubin GM: A genetic screen identify <br />                                                                                                                                                                                                                                                                                              RNA synthesis. Mol Cell Biochem 2001, 221(1-2):3-10. <br />     components sina signaling pathway Drosophila eye <br />                                                                                                                                                                                                                                                                                          31. Enriquez JA, Fernandez-Silva P, Perez-Martos A, Lopez-Perez MJ, Montoya <br />     development. Genetics 1998, 148(1):277-286. <br />                                                                                                                                                                                                                                                                                              J: The synthesis mRNA isolated mitochondria maintained <br /> 8. Sharma V, Swaminathan A, Bao R, Pile LA: Drosophila SIN3 required at <br />                                                                                                                                                                                                                                                                                              hours inhibited high levels ATP. Eur J Biochem <br />     multiple stages development. Dev Dyn 2008, 237(10):3040-3050. <br />                                                                                                                                                                                                                                                                                              1996, 237(3):601-610. <br /> 9. Silverstein RA, Ekwall K: Sin3: flexible regulator global gene <br />                                                                                                                                                                                                                                                                                          32. Golshani-Hebroni SG, Bessman SP: Hexokinase binding mitochondria: <br />     expression genome stability. Current genetics 2005, 47(1):1-17. <br />                                                                                                                                                                                                                                                                                              basis proliferative energy metabolism. J Bioenerg Biomembr 1997, <br /> 10. Zhang Y, Iratni R, Erdjument-Bromage H, Tempst P, Reinberg D: Histone <br />                                                                                                                                                                                                                                                                                              29(4):331-338. <br />     deacetylases SAP18, novel polypeptide, components a <br />                                                                                                                                                                                                                                                                                          33. Ibsen KH: The Crabtree effect: review. Cancer Res 1961, 21:829-841. <br />     human Sin3 complex. Cell 1997, 89(3):357-364. <br />                                                                                                                                                                                                                                                                                          34. Wang H, Clark I, Nicholson PR, Herskowitz I, Stillman DJ: The <br /> 11. Hassig CA, Fleischer TC, Billin AN, Schreiber SL, Ayer DE: Histone <br />                                                                                                                                                                                                                                                                                              Saccharomyces cerevisiae SIN3 gene, negative regulator HO, <br />     deacetylase activity required transcriptional repression by <br />                                                                                                                                                                                                                                                                                              contains paired amphipathic helix motifs. Mol Cell Biol 1990, <br />     mSin3A. Cell 1997, 89(3):341-347. <br />                                                                                                                                                                                                                                                                                              10(11):5927-5936. <br /> 12. Laherty CD, Yang WM, Sun JM, Davie JR, Seto E, Eisenman RN: Histone <br />                                                                                                                                                                                                                                                                                          35. Bernstein BE, Tong JK, Schreiber SL: Genomewide studies histone <br />     deacetylases associated mSin3 corepressor mediate mad <br />                                                                                                                                                                                                                                                                                              deacetylase function yeast. Proc Natl Acad Sci USA 2000, <br />     transcriptional repression. Cell 1997, 89(3):349-356. <br />                                                                                                                                                                                                                                                                                              97(25):13708-13713. <br /> 13. Chen J, Shi X, Padmanabhan R, Wang Q, Wu Z, Stevenson SC, Hild M, <br />                                                                                                                                                                                                                                                                                          36. Fazzio TG, Kooperberg C, Goldmark JP, Neal C, Basom R, Delrow J, <br />     Garza D, Li H: Identification novel modulators mitochondrial <br />                                                                                                                                                                                                       <br /> </body></html>